<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation
Authors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.
Score: 45.9, Published: 2023-08-09 DOI: 10.1101/2023.08.07.552219
Structural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation
Authors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.
Score: 45.9, Published: 2023-08-09 DOI: 10.1101/2023.08.07.552219
Structural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation
Authors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.
Score: 45.9, Published: 2023-08-09 DOI: 10.1101/2023.08.07.552219
Structural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation\nAuthors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.\nScore: 45.9, Published: 2023-08-09 DOI: 10.1101/2023.08.07.552219\nStructural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants.",
  "keywords": [
    
  ],
  "articleBody": " Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation\nAuthors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.\nScore: 45.9, Published: 2023-08-09 DOI: 10.1101/2023.08.07.552219\nStructural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants. However, structural data is still missing for most protein interactions because we lack computational and experimental tools at scale. We thoroughly assessed AlphaFold-Multimer accuracy for structure prediction of interactions involving folded domains binding to short linear motifs from the ELM database. The structure predictions were highly sensitive but not very specific when using small protein fragments. Sensitivity decreased substantially when using long protein fragments or full length proteins with intrinsically disordered regions. We delineated a fragmentation strategy to optimize sensitivity and applied it to interactions between proteins associated with neurodevelopmental disorders. This enabled prediction of highly confident and likely disease-related novel interfaces, but also resulted in many high scoring false positive predictions. Experiments supported predicted interfaces between CREBZF-HCFC1, FBXO23-STX1B, STX1B-VAMP2, ESRRG-PSMC5, PEX3-PEX19, PEX3-PEX16, and SNRPB-GIGYF1 providing novel molecular insights for diverse biological processes. Our work highlights exciting perspectives, but also reveals clear limitations and the need for future developments to maximize the power of Alphafold-Multimer for interface predictions.\nAmyloid β accelerates age-related proteome-wide protein insolubility.\nAuthors: Anderton, E.; Chamoli, M.; Bhaumik, D.; King, C. D.; Xie, X.; Foulger, A.; Andersen, J.; Schilling, B.; Lithgow, G.\nScore: 31.3, Published: 2023-07-16 DOI: 10.1101/2023.07.13.548937\nLoss of proteostasis is a highly conserved feature of aging across model organisms and typically results in the accumulation of insoluble protein aggregates. Protein insolubility is a central feature of major age-related neurodegenerative diseases including Alzheimers Disease (AD), where hundreds of insoluble proteins associate with aggregated amyloid beta (A{beta}) in senile plaques. Despite the established connection between aging and AD risk, therapeutic approaches to date have overlooked aging and proteome-wide protein insolubility as causal factors, instead focusing on A{beta} and Tau. Here, using an unbiased proteomics approach, we questioned the relationship between A{beta} and age-related protein insolubility. We demonstrate that, in C. elegans, A{beta} expression is sufficient to drive proteome-wide protein insolubility. The A{beta}-driven insoluble proteome bears a highly significant overlap with the aging-driven insoluble proteome, suggesting there exists a core, sub-proteome which is vulnerable to insolubility. Using human genome-wide association studies (GWAS) we show that this insoluble sub proteome is replete with biological processes implicated across not only neurodegenerative diseases but also across a broad array of chronic, age-related diseases (CARDs), providing suggestive evidence that age-related loss of proteostasis could play a role in general CARD risk. Finally, we show that the gut metabolite, Urolithin A, relieves A{beta} toxicity supporting its use in clinical trials for dementia and other age-related diseases.\nSystems-ecology designed bacterial consortium protects from severe Clostridioides difficile infection\nAuthors: Jenior, M. L.; Leslie, J. L.; Kolling, G. L.; Archbald-Pannone, L.; Powers, D. A.; Petri, W. A.; Papin, J. A.\nScore: 6.3, Published: 2023-08-10 DOI: 10.1101/2023.08.08.552483\nFecal Microbiota Transplant (FMT) is an emerging therapy that has had remarkable success in treatment and prevention of recurrent Clostridioides difficile infection (rCDI). FMT has recently been associated with adverse outcomes such as inadvertent transfer of antimicrobial resistance, necessitating development of more targeted bacteriotherapies. To address this challenge, we developed a novel systems biology pipeline to identify candidate probiotic strains that would be predicted to interrupt C. difficile pathogenesis. Utilizing metagenomic characterization of human FMT donor samples, we identified those metabolic pathways most associated with successful FMTs and reconstructed the metabolism of encoding species to simulate interactions with C. difficile. This analysis resulted in predictions of high levels of cross-feeding for amino acids in species most associated with FMT success. Guided by these in silico models, we assembled consortia of bacteria with increased amino acid cross-feeding which were then validated in vitro. We subsequently tested the consortia in a murine model of CDI, demonstrating total protection from severe CDI through decreased toxin levels, recovered gut microbiota, and increased intestinal eosinophils. These results support the novel framework that amino acid cross-feeding is likely a critical mechanism in the initial resolution of CDI by FMT. Importantly, we conclude that our predictive platform based on predicted and testable metabolic interactions between the microbiota and C. difficile led to a rationally designed biotherapeutic framework that may be extended to other enteric infections.\nProstate Cancer Reshapes the Secreted and Extracellular Vesicle Urinary Proteomes\nAuthors: Khoo, A.; Govindarajan, M.; Qui, Z.; Liu, L. Y.; Ignatchenko, V.; Waas, M.; Macklin, A.; Keszei, A.; Main, B. P.; Yang, L.; Lance, R. S.; Downes, M. R.; Semmes, O. J.; Vesprini, D.; Liu, S. K.; Nyalwidhe, J. O.; Boutros, P. C.; Kislinger, T.\nScore: 10.0, Published: 2023-07-25 DOI: 10.1101/2023.07.23.550214\nUrine is a complex biofluid that reflects both overall physiologic state and the state of the genitourinary tissues through which it passes. It contains both secreted proteins and proteins encapsulated in tissue-derived extracellular vesicles (EVs). To understand the population variability and clinical utility of urine, we quantified the secreted and EV proteomes from 190 men, including a subset with prostate cancer. We demonstrate that a simple protocol enriches prostatic proteins in urine. Secreted and EV proteins arise from different subcellular compartments. Urinary EVs are faithful surrogates of tissue proteomes, but secreted proteins in urine or cell line EVs are not. The urinary proteome is longitudinally stable over several years. It can accurately and non-invasively distinguish malignant from benign prostatic lesions, and can risk-stratify prostate tumors. This resource quantifies the complexity of the urinary proteome, and reveals the synergistic value of secreted and EV proteomes for translational and biomarker studies.\nBody weight, gonadectomy, and other risk factors for diagnosis of osteoarthritis in companion dogs\nAuthors: Graves, J. L.; McKenzie, B.; Koch, Z.; Naka, A. S.; Spofford, N.; Morrison, J.\nScore: 1.6, Published: 2023-08-08 DOI: 10.1101/2023.08.03.550998\nOBJECTIVETo evaluate age, sex, body weight, breed, neuter status, and age at neutering as risk factors for diagnosis of osteoarthritis in companion dogs ANIMALSDogs seen as patients at Banfield Pet Hospital in the United States from 1998-2019 with a date of death in 2019. The final cohort consisted of 131,140 dogs. METHODSIn this retrospective cohort study, Cox proportional hazard models were used to test for associations between osteoarthritis incidence and age at baseline, sex, maximum body weight, maximum body condition score, neuter status, and age at neutering. The same model was used to test these associations in 12 representative breeds, chosen based on breed weight and sample size. RESULTSOlder age, higher adult body weight, gonadectomy, and younger age at gonadectomy were significantly associated with higher risks of osteoarthritis in the total cohort and in all 12 breeds evaluated. Higher body condition score and sex were also significantly associated with osteoarthritis but with minimal effect sizes in the overall cohort, and these risk factors were not consistently significant in all breeds tested. CLINICAL RELEVANCEThese results will assist veterinarians in identifying dogs at higher risk for osteoarthritis and applying appropriate diagnostic, preventative, and treatment interventions. An understanding of potentially modifiable risk factors, such as body condition, and neutering, will support evidence-based discussions with dog owners about risk management in individual patients.\nAutomatic design of gene regulatory mechanisms for spatial pattern formation\nAuthors: Mousavi, R.; Lobo, D.\nScore: 4.5, Published: 2023-07-29 DOI: 10.1101/2023.07.26.550573\nGene regulatory mechanisms (GRMs) control the formation of spatial and temporal expression patterns that can serve as regulatory signals for the development of complex shapes. Synthetic developmental biology aims to engineer such mechanisms for understanding and producing desired multicellular patterns. However, the design of synthetic GRMs that can produce a target spatial gene expression pattern is a current challenge due to the nonlinear interactions and feedback loops present in genetic circuits. Here we present a methodology to automatically design GRMs that can produce any given spatial pattern. The proposed approach uses two orthogonal morphogen gradients acting as positional information signals in a multicellular tissue area or culture, which constitutes a continuous field of engineered cells implementing the same designed GRM. To efficiently design both the circuit network and the interaction mechanisms--including the number of genes required for the formation of the target pattern--we developed an automated algorithm based on high-performance evolutionary computation. The tolerance of the algorithm can be configured to design GRMs that are either simple to produce approximate patterns or complex to produce precise patterns. We demonstrate its performance by automatically designing GRMs for a diverse set of synthetic spatial expression patterns. The proposed method offers a versatile approach to systematically design and discover pattern-producing genetic circuits.\nSystematic exploration of prokaryotic form I rubisco maximal carboxylation rates\nAuthors: de Pins, B.; Greenspoon, L.; Bar-On, Y. M.; Shamshoum, M.; Ben-Nissan, R.; Milshtein, E.; Davidi, D.; Sharon, I.; Mueller-Cajar, O.; Noor, E.; Milo, R.\nScore: 19.3, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550689\nAutotrophy is the basis for complex life on Earth. Its physiological limits are largely constrained by rubisco - the enzyme that catalyzes almost all carbon fixation on the planet. Yet, with only a small fraction of rubisco diversity kinetically characterized so far, the underlying biological factors driving the evolution of fast rubiscos in nature remain unclear. We conducted a high-throughput kinetic characterization of over 100 prokaryotic form I rubiscos, the most ubiquitous group of rubisco sequences in nature, to uncover the determinants of rubiscos carboxylation velocity. We show that phototrophy and the presence of a carboxysome CO2 concentrating mechanism correlate with faster rubiscos (median rate difference of 3-5 fold). In contrast to previous assumptions, rubiscos originating from -cyanobacteria exhibit the highest carboxylation rates among form I enzymes ({approx}10 s-1 median versus \u003c7 s-1 in other groups). Our study ties biological and environmental insights associated with kinetic variation across rubiscos from nature.\n",
  "wordCount" : "1628",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552219" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552219">
        <p class="paperTitle">Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.</p>
        <p class="info">Score: 45.9, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552219' target='https://doi.org/10.1101/2023.08.07.552219'> 10.1101/2023.08.07.552219</a></p>
        <p class="abstract">Structural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants. However, structural data is still missing for most protein interactions because we lack computational and experimental tools at scale. We thoroughly assessed AlphaFold-Multimer accuracy for structure prediction of interactions involving folded domains binding to short linear motifs from the ELM database. The structure predictions were highly sensitive but not very specific when using small protein fragments. Sensitivity decreased substantially when using long protein fragments or full length proteins with intrinsically disordered regions. We delineated a fragmentation strategy to optimize sensitivity and applied it to interactions between proteins associated with neurodevelopmental disorders. This enabled prediction of highly confident and likely disease-related novel interfaces, but also resulted in many high scoring false positive predictions. Experiments supported predicted interfaces between CREBZF-HCFC1, FBXO23-STX1B, STX1B-VAMP2, ESRRG-PSMC5, PEX3-PEX19, PEX3-PEX16, and SNRPB-GIGYF1 providing novel molecular insights for diverse biological processes. Our work highlights exciting perspectives, but also reveals clear limitations and the need for future developments to maximize the power of Alphafold-Multimer for interface predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548937">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548937" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548937">
        <p class="paperTitle">Amyloid β accelerates age-related proteome-wide protein insolubility.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548937" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548937" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Anderton, E.; Chamoli, M.; Bhaumik, D.; King, C. D.; Xie, X.; Foulger, A.; Andersen, J.; Schilling, B.; Lithgow, G.</p>
        <p class="info">Score: 31.3, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548937' target='https://doi.org/10.1101/2023.07.13.548937'> 10.1101/2023.07.13.548937</a></p>
        <p class="abstract">Loss of proteostasis is a highly conserved feature of aging across model organisms and typically results in the accumulation of insoluble protein aggregates. Protein insolubility is a central feature of major age-related neurodegenerative diseases including Alzheimers Disease (AD), where hundreds of insoluble proteins associate with aggregated amyloid beta (A{beta}) in senile plaques. Despite the established connection between aging and AD risk, therapeutic approaches to date have overlooked aging and proteome-wide protein insolubility as causal factors, instead focusing on A{beta} and Tau. Here, using an unbiased proteomics approach, we questioned the relationship between A{beta} and age-related protein insolubility. We demonstrate that, in C. elegans, A{beta} expression is sufficient to drive proteome-wide protein insolubility. The A{beta}-driven insoluble proteome bears a highly significant overlap with the aging-driven insoluble proteome, suggesting there exists a core, sub-proteome which is vulnerable to insolubility. Using human genome-wide association studies (GWAS) we show that this insoluble sub proteome is replete with biological processes implicated across not only neurodegenerative diseases but also across a broad array of chronic, age-related diseases (CARDs), providing suggestive evidence that age-related loss of proteostasis could play a role in general CARD risk. Finally, we show that the gut metabolite, Urolithin A, relieves A{beta} toxicity supporting its use in clinical trials for dementia and other age-related diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552483" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552483">
        <p class="paperTitle">Systems-ecology designed bacterial consortium protects from severe Clostridioides difficile infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jenior, M. L.; Leslie, J. L.; Kolling, G. L.; Archbald-Pannone, L.; Powers, D. A.; Petri, W. A.; Papin, J. A.</p>
        <p class="info">Score: 6.3, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552483' target='https://doi.org/10.1101/2023.08.08.552483'> 10.1101/2023.08.08.552483</a></p>
        <p class="abstract">Fecal Microbiota Transplant (FMT) is an emerging therapy that has had remarkable success in treatment and prevention of recurrent Clostridioides difficile infection (rCDI). FMT has recently been associated with adverse outcomes such as inadvertent transfer of antimicrobial resistance, necessitating development of more targeted bacteriotherapies. To address this challenge, we developed a novel systems biology pipeline to identify candidate probiotic strains that would be predicted to interrupt C. difficile pathogenesis. Utilizing metagenomic characterization of human FMT donor samples, we identified those metabolic pathways most associated with successful FMTs and reconstructed the metabolism of encoding species to simulate interactions with C. difficile. This analysis resulted in predictions of high levels of cross-feeding for amino acids in species most associated with FMT success. Guided by these in silico models, we assembled consortia of bacteria with increased amino acid cross-feeding which were then validated in vitro. We subsequently tested the consortia in a murine model of CDI, demonstrating total protection from severe CDI through decreased toxin levels, recovered gut microbiota, and increased intestinal eosinophils. These results support the novel framework that amino acid cross-feeding is likely a critical mechanism in the initial resolution of CDI by FMT. Importantly, we conclude that our predictive platform based on predicted and testable metabolic interactions between the microbiota and C. difficile led to a rationally designed biotherapeutic framework that may be extended to other enteric infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.23.550214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.23.550214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.23.550214">
        <p class="paperTitle">Prostate Cancer Reshapes the Secreted and Extracellular Vesicle Urinary Proteomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.23.550214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.23.550214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khoo, A.; Govindarajan, M.; Qui, Z.; Liu, L. Y.; Ignatchenko, V.; Waas, M.; Macklin, A.; Keszei, A.; Main, B. P.; Yang, L.; Lance, R. S.; Downes, M. R.; Semmes, O. J.; Vesprini, D.; Liu, S. K.; Nyalwidhe, J. O.; Boutros, P. C.; Kislinger, T.</p>
        <p class="info">Score: 10.0, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.23.550214' target='https://doi.org/10.1101/2023.07.23.550214'> 10.1101/2023.07.23.550214</a></p>
        <p class="abstract">Urine is a complex biofluid that reflects both overall physiologic state and the state of the genitourinary tissues through which it passes. It contains both secreted proteins and proteins encapsulated in tissue-derived extracellular vesicles (EVs). To understand the population variability and clinical utility of urine, we quantified the secreted and EV proteomes from 190 men, including a subset with prostate cancer. We demonstrate that a simple protocol enriches prostatic proteins in urine. Secreted and EV proteins arise from different subcellular compartments. Urinary EVs are faithful surrogates of tissue proteomes, but secreted proteins in urine or cell line EVs are not. The urinary proteome is longitudinally stable over several years. It can accurately and non-invasively distinguish malignant from benign prostatic lesions, and can risk-stratify prostate tumors. This resource quantifies the complexity of the urinary proteome, and reveals the synergistic value of secreted and EV proteomes for translational and biomarker studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.550998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.550998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.550998">
        <p class="paperTitle">Body weight, gonadectomy, and other risk factors for diagnosis of osteoarthritis in companion dogs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.550998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.550998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Graves, J. L.; McKenzie, B.; Koch, Z.; Naka, A. S.; Spofford, N.; Morrison, J.</p>
        <p class="info">Score: 1.6, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.550998' target='https://doi.org/10.1101/2023.08.03.550998'> 10.1101/2023.08.03.550998</a></p>
        <p class="abstract">OBJECTIVETo evaluate age, sex, body weight, breed, neuter status, and age at neutering as risk factors for diagnosis of osteoarthritis in companion dogs

ANIMALSDogs seen as patients at Banfield Pet Hospital in the United States from 1998-2019 with a date of death in 2019. The final cohort consisted of 131,140 dogs.

METHODSIn this retrospective cohort study, Cox proportional hazard models were used to test for associations between osteoarthritis incidence and age at baseline, sex, maximum body weight, maximum body condition score, neuter status, and age at neutering. The same model was used to test these associations in 12 representative breeds, chosen based on breed weight and sample size.

RESULTSOlder age, higher adult body weight, gonadectomy, and younger age at gonadectomy were significantly associated with higher risks of osteoarthritis in the total cohort and in all 12 breeds evaluated. Higher body condition score and sex were also significantly associated with osteoarthritis but with minimal effect sizes in the overall cohort, and these risk factors were not consistently significant in all breeds tested.

CLINICAL RELEVANCEThese results will assist veterinarians in identifying dogs at higher risk for osteoarthritis and applying appropriate diagnostic, preventative, and treatment interventions. An understanding of potentially modifiable risk factors, such as body condition, and neutering, will support evidence-based discussions with dog owners about risk management in individual patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550573">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550573" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550573">
        <p class="paperTitle">Automatic design of gene regulatory mechanisms for spatial pattern formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550573" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550573" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mousavi, R.; Lobo, D.</p>
        <p class="info">Score: 4.5, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550573' target='https://doi.org/10.1101/2023.07.26.550573'> 10.1101/2023.07.26.550573</a></p>
        <p class="abstract">Gene regulatory mechanisms (GRMs) control the formation of spatial and temporal expression patterns that can serve as regulatory signals for the development of complex shapes. Synthetic developmental biology aims to engineer such mechanisms for understanding and producing desired multicellular patterns. However, the design of synthetic GRMs that can produce a target spatial gene expression pattern is a current challenge due to the nonlinear interactions and feedback loops present in genetic circuits. Here we present a methodology to automatically design GRMs that can produce any given spatial pattern. The proposed approach uses two orthogonal morphogen gradients acting as positional information signals in a multicellular tissue area or culture, which constitutes a continuous field of engineered cells implementing the same designed GRM. To efficiently design both the circuit network and the interaction mechanisms--including the number of genes required for the formation of the target pattern--we developed an automated algorithm based on high-performance evolutionary computation. The tolerance of the algorithm can be configured to design GRMs that are either simple to produce approximate patterns or complex to produce precise patterns. We demonstrate its performance by automatically designing GRMs for a diverse set of synthetic spatial expression patterns. The proposed method offers a versatile approach to systematically design and discover pattern-producing genetic circuits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550689">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550689" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550689">
        <p class="paperTitle">Systematic exploration of prokaryotic form I rubisco maximal carboxylation rates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550689" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550689" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Pins, B.; Greenspoon, L.; Bar-On, Y. M.; Shamshoum, M.; Ben-Nissan, R.; Milshtein, E.; Davidi, D.; Sharon, I.; Mueller-Cajar, O.; Noor, E.; Milo, R.</p>
        <p class="info">Score: 19.3, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550689' target='https://doi.org/10.1101/2023.07.27.550689'> 10.1101/2023.07.27.550689</a></p>
        <p class="abstract">Autotrophy is the basis for complex life on Earth. Its physiological limits are largely constrained by rubisco - the enzyme that catalyzes almost all carbon fixation on the planet. Yet, with only a small fraction of rubisco diversity kinetically characterized so far, the underlying biological factors driving the evolution of fast rubiscos in nature remain unclear. We conducted a high-throughput kinetic characterization of over 100 prokaryotic form I rubiscos, the most ubiquitous group of rubisco sequences in nature, to uncover the determinants of rubiscos carboxylation velocity. We show that phototrophy and the presence of a carboxysome CO2 concentrating mechanism correlate with faster rubiscos (median rate difference of 3-5 fold). In contrast to previous assumptions, rubiscos originating from -cyanobacteria exhibit the highest carboxylation rates among form I enzymes ({approx}10 s-1 median versus &lt;7 s-1 in other groups). Our study ties biological and environmental insights associated with kinetic variation across rubiscos from nature.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
