<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The interplay between lumen pressure and cell proliferation determines organoid morphology in a multicellular phase field model
Authors: Tanida, S.; Fuji, K.; Lu, L.; Guyomar, T.; Lee, B. H.; Honigmann, A.; Grapin-Botton, A.; Riveline, D.; Nonomura, M.; Sano, M.; Hiraiwa, T.
Score: 13.8, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553655
Organoids are ideal systems to predict the phenotypes of organs. However, there is currently a lack of understanding regarding the generalized rules that enable use of simple cellular principles to make morphological predictions of entire organoids.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="The interplay between lumen pressure and cell proliferation determines organoid morphology in a multicellular phase field model
Authors: Tanida, S.; Fuji, K.; Lu, L.; Guyomar, T.; Lee, B. H.; Honigmann, A.; Grapin-Botton, A.; Riveline, D.; Nonomura, M.; Sano, M.; Hiraiwa, T.
Score: 13.8, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553655
Organoids are ideal systems to predict the phenotypes of organs. However, there is currently a lack of understanding regarding the generalized rules that enable use of simple cellular principles to make morphological predictions of entire organoids." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-03T10:37:18+00:00" />
<meta property="article:modified_time" content="2023-09-03T10:37:18+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="The interplay between lumen pressure and cell proliferation determines organoid morphology in a multicellular phase field model
Authors: Tanida, S.; Fuji, K.; Lu, L.; Guyomar, T.; Lee, B. H.; Honigmann, A.; Grapin-Botton, A.; Riveline, D.; Nonomura, M.; Sano, M.; Hiraiwa, T.
Score: 13.8, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553655
Organoids are ideal systems to predict the phenotypes of organs. However, there is currently a lack of understanding regarding the generalized rules that enable use of simple cellular principles to make morphological predictions of entire organoids."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "The interplay between lumen pressure and cell proliferation determines organoid morphology in a multicellular phase field model\nAuthors: Tanida, S.; Fuji, K.; Lu, L.; Guyomar, T.; Lee, B. H.; Honigmann, A.; Grapin-Botton, A.; Riveline, D.; Nonomura, M.; Sano, M.; Hiraiwa, T.\nScore: 13.8, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553655\nOrganoids are ideal systems to predict the phenotypes of organs. However, there is currently a lack of understanding regarding the generalized rules that enable use of simple cellular principles to make morphological predictions of entire organoids.",
  "keywords": [
    
  ],
  "articleBody": " The interplay between lumen pressure and cell proliferation determines organoid morphology in a multicellular phase field model\nAuthors: Tanida, S.; Fuji, K.; Lu, L.; Guyomar, T.; Lee, B. H.; Honigmann, A.; Grapin-Botton, A.; Riveline, D.; Nonomura, M.; Sano, M.; Hiraiwa, T.\nScore: 13.8, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553655\nOrganoids are ideal systems to predict the phenotypes of organs. However, there is currently a lack of understanding regarding the generalized rules that enable use of simple cellular principles to make morphological predictions of entire organoids. Therefore, we employed a phase field model with the following basic components: the minimum conditions for the timing and volume of cell division, lumen nucleation rules, and lumenal pressure. Through our model, we could compute and generate a myriad of organoid phenotypes observed till date. We propose morphological indices necessary to characterize the shapes and construct phase diagrams and show their dependencies on proliferation time and lumen pressure. Additionally, we introduced the lumen-index parameter, which helped in examining the criteria to maintain organoids as spherical structures comprising a single layer of cells and enclosing an intact lumen. Finally, we predict a star-like organoid phenotype that did not undergo differentiation, suggesting that the volume constraint during cell division may determine the final phenotype. In summary, our approach provides researchers with guidelines to test the mechanisms of self-organization and predict the shape of organoid. Author summaryIn nature, a wide variety of organ morphologies are observed. Owing to the complexity of the process underlying the acquisition of organs morphology, it is challenging to investigate the mechanisms that lead to such variations. A promising approach to study these variations is the use of \"computational organoid\" study, which is the computational-based study of self-organizing shapes in multicellular assemblies and fluid-filled cavities called lumens that develop from a few proliferating cells. This study explores general mechanisms that dictate how various mechanical factors affect the growing self-organized multicellular assembly. We relied on computer simulations of the mathematical model called multicellular phase-field model with lumens and explored the mechanical factor effects, such as the lumen pressure while considering the time and volume conditions required for cell division. These simulations generated and categorized a wide range of organoid phenotypes based on the varying lumen pressure and cell division conditions. These phenotypes were characterized into seven distinct classes, based on the morphological index sets, including a cellular monolayer/multilayer surrounding single or multiple lumens and branch formation. These phenotypes were obtained without the assumption of differentiation. Our study elucidates the mechanisms underlying the organoid and organ formation with different shapes, thereby highlighting the significance of mechanical forces in shaping these complex biological structures.\nTranscription factors operate on a limited vocabulary of binding motifs in Arabidopsis thaliana\nAuthors: Zenker, S.; Wulf, D.; Meierhenrich, A.; Becker, S.; Eisenhut, M.; Stracke, R.; Weisshaar, B.; Braeutigam, A.\nScore: 8.7, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555073\nPredicting gene expression from promoter sequence requires understanding of the different signal integration points within a promoter. Sequence-specific transcription factors (TFs) binding to their cognate TF binding motifs control gene expression in eukaryotes by activating and repressing transcription. Their interplay generates complex expression patterns in reaction to environmental conditions and developmental cues. We hypothesized that signals are not only integrated by different TFs binding various positions in a promoter, but also by single TF binding motifs onto which multiple TFs can bind. Analyzing 2,190 binding motifs, we identified only 76 core TF binding motifs in plants. Twenty-one TF protein families act highly specific and bind a single conserved motif. Four TF families are classified as semi-conserved as they bind up to four motifs within a family, with divisions along phylogenetic groups. Five TF families bind diverse motifs. Expression analyses revealed high competition within TF families for the same binding motif. The results show that singular binding motifs act as signal integrators in plants where a combination of binding affinity and TF abundance likely determine the output.\nDynamics of bacterial operons during genome-wide stresses\nAuthors: Jagadeesan, R.; Dash, S.; Palma, C.; Baptista, I. S. C.; Chauhan, V.; Makela, J.; Ribeiro, A. S.\nScore: 8.2, Published: 2023-08-25 DOI: 10.1101/2023.08.24.554578\nBacterial gene networks have many operons, each coordinating the kinetics of a few genes in time and strength. In Escherichia coli, nearly half of these operons have internal promoters, which allow regulating the downstream genes, independently from upstream genes. We studied their role during genome-wide stresses targeting a few key elements of the transcription machinery, namely, RNAP and gyrase. We show that absolute differences in response strength between genes in the same operon follow a sinusoidal function of the distance between them. This results from the combined effect of elongation fall-offs and internal promoters. The frequency of this function is regulated by the positioning of internal promoters, while its amplitude differs with the perturbation strength. Moreover, we find that positive supercoiling buildup partially explains the RNAP fall-off rates. Finally, we report similar sinusoidal patterns in E. coli cells subject to several other stresses, as well as in Bacillus subtilis, Corynebacterium glutamicum, and Helicobacter pylori. Overall, our results suggest that the strength, number, and positioning of internal promoters in operons are influenced by a genome-wide need to compensate for RNAP fall-offs, and thus provide distal genes in the operon with similar response strength as genes proximal to the primary promoter.\nOn standardization of controls in lifespan studies\nAuthors: Spiridonova, O.; Kriukov, D.; Nemirovich-Danchenko, N.; Peshkin, L.\nScore: 9.9, Published: 2023-08-18 DOI: 10.1101/2023.08.17.552381\nThe search for interventions to slow down and even reverse aging is a burgeoning field. The literature cites hundreds of supposedly beneficial pharmacological and genetic interventions in model organisms: mice, rats, flies and worms, where research into physiology is routinely accompanied by lifespan data. Naturally the negative results are more frequent, yet scientifically quite valuable if analyzed systematically. Yet, there is a strong \"discovery bias\", i.e. results of interventions which turn out not to be beneficial remain unpublished. Theoretically, all lifespan data is ripe for re-analysis: we could contrast the molecular targets and pathways across studies and help focus the further search for interventions. Alas, the results of most longevity studies are difficult to compare. This is in part because there are no clear, universally accepted standards for conducting such experiments or even for reporting such data. The situation is worsened by the fact that the authors often do not describe experimental conditions completely. As a result, works on longevity make up a set of precedents, each of which might be interesting in its own right, yet incoherent and incomparable. Here we point out specific issues and propose solutions for quality control by checking both inter- and intra-study consistency of lifespan data.\nEvolution of hierarchical switching pattern in antigenic variation of Plasmodium falciparum under variable host immunity\nAuthors: Ghosh, B.; Priya, I. G.\nScore: 1.6, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555470\nThe var genes family encoding the variants of the erythrocyte membrane protein of Plasmodium falciparum is crucial for virulence of the parasite inside host. The transcriptional output of the var genes switches from one variant to other in a mutually exclusive fashion. It is proposed that a biased hierarchical switching pattern optimizes the growth and survival of the parasite inside the host. Apart from the hierarchical switching pattern, it is also well established that the intrinsic switching rates vary widely among the var genes. The centromeric protein like Var2csa is much more stable than the genes located at the telomeric and sub-telomeric regions of the chromosomes. In this study, we explored the evolutionary advantage achieved through selecting variable switching rates. Our theoretical analysis based on a mathematical model coupled with single cell RNA-seq data suggests that the variable switching rate is beneficial when cells expressing different variants are deferentially amenable to be cleared by the immune response. In fact, the variants which are cleared by the immune systems more efficiently are more stably expressed compared to a variant attacked by the immune system much less vigorously. The cells turn off expression of the variant quickly which is not cleared very efficiently. The evolutionary simulation shows that this strategy maximizes the growth of the parasite population under the presence of immune attack by the host. In corroboration with the result, we observed that stable variant has higher binding affinity to IgM from experimental data. Our study provides an evolutionary basis of widely variable switching rates of the var genes in Plasmodium falciparum.\nIntegration of graph neural networks and genome-scale metabolic models for predicting gene essentiality\nAuthors: Hasibi, R.; Michoel, T.; Oyarzun, D. A.\nScore: 1.5, Published: 2023-08-29 DOI: 10.1101/2023.08.25.554757\nGenome-scale metabolic models are powerful tools for understanding cellular physiology. Flux balance analysis (FBA), in particular, is a popular optimization approach for predicting metabolic phenotypes under genetic and environmental perturbations. In model microbes such as Escherichia coli, FBA has been successful at predicting essential genes, i.e. those genes that impair survival when deleted. A central assumption in this approach, however, is that both wild type and deletion strains optimize the same fitness objective. While the optimality assumption may hold for the wild type metabolic network, deletion strains are not subject to the same evolutionary pressures and knock-out mutants may steer their metabolism to meet other objectives for survival. Here, we present FlowGAT, a hybrid FBA-machine learning strategy for predicting essentiality directly from wild type metabolic phenotypes. The approach is based on graph-structured representation of metabolic fluxes predicted by FBA, where nodes correspond to enzymatic reactions and edges quantify the propagation of metabolite mass flow between a reaction and its neighbours. We integrate this information into a graph neural network that can be trained on knock-out fitness assay data. Comparisons across different model architectures reveal that FlowGAT predictions for E. coli are close to those of FBA for several growth conditions. This suggests that gene essentiality can be accurately predicted by exploiting the network structure of metabolism, without additional assumptions beyond optimality of the wild type. Our approach demonstrates the benefits of combining the mechanistic insights afforded by genome-scale models with the ability of deep learning models to extract patterns from complex data.\nLarge-scale analysis of the integration of enhancer-enhancer signals by promoters\nAuthors: Martinez-Ara, M.; Comoglio, F.; van Steensel, B.\nScore: 25.7, Published: 2023-08-14 DOI: 10.1101/2023.08.11.552995\nGenes are often regulated by multiple enhancers. It is poorly understood how the individual enhancer activities are combined to control promoter activity. Anecdotal evidence has shown that enhancers can combine sub-additively, additively, synergistically, or redundantly. However, it is not clear which of these modes are more frequent in mammalian genomes. Here, we systematically tested how pairs of enhancers activate promoters using a three-way combinatorial reporter assay in mouse cells. By assaying about 69,000 enhancer-enhancer-promoter combinations we found that enhancer pairs generally combine near-additively. This behaviour was conserved across seven developmental promoters tested. Surprisingly, these promoters scale the enhancer signals approximately following a power-law, but the exponent of this response varies between promoters. A housekeeping promoter showed an overall different response to enhancer pairs, and a smaller dynamic range. Thus, our data indicate that enhancers mostly act additively, but promoters transform their collective effect non-linearly.\nA trade-off in controlling upstream and downstream noise in signaling networks\nAuthors: Kong, K. K.; Luo, C.; Liu, F.\nScore: 1.0, Published: 2023-08-31 DOI: 10.1101/2023.08.29.555248\nSignal transduction, underpinning the function of a variety of biological systems, is inevitably affected by fluctuations. It remains intriguing how the timescale of a signaling network relates to its capability of noise control, specifically, whether long timescale can average out fluctuation or accumulate fluctuation. Here, we consider two noise components of the signaling system: the upstream noise from the fluctuation of the input signal and the downstream noise from the stochastic fluctuations of the network. We discover a fundamental trade-off in controlling the upstream and downstream noise: a longer timescale of the signaling network can buffer upstream noise, while accumulate downstream noise. Moreover, we confirm that this trade-off relation exists in real biological signaling networks such as a fold-change detection circuit and the p53 activation signaling system. Author SummaryInformation transmission is vital in biological systems, such as decoding the information regarding nutrient levels during chemotaxis or morphogen concentrations in tissue development. While fluctuations arising from the stochastic nature of biological processes inevitably affect information transmission, noise control mechanisms have been studied for decades. However, it remains controversial what the role of the slow dynamics (long timescale) is in noise control. On the one hand, it has been reported to attenuate noise by averaging out fluctuations. On the other hand, it is also proposed to amplify noise by accumulating fluctuations. Here we dissect the noise in signaling systems into two components: upstream noise originating from signal fluctuation, and downstream noise from the network stochasticity. Our analysis reveals that upstream noise negatively correlates with timescale, while downstream noise exhibits a positive correlation, indicating a fundamental trade-off in controlling the two noise components. Moreover, we provide an intuitive illustration to understand this phenomenon using the concept of landscape representation. Mathematically, we analytically derive a trade-off relation that agrees well with simulations. Our results uncover a new property of noise in signaling processes, deepening our understanding of noise control and proposing a new perspective in designing signaling network.\nA systems vaccinology resource to develop and test computational models of immunity\nAuthors: Shinde, P.; Soldevila, F.; Reyna, J.; Aoki, M.; Rasmussen, M.; Willemsen, L.; Kojima, M.; Ha, B.; Greenbaum, J. A.; Overton, J. A.; Guzman-Orozco, H.; Nili, S.; Orfield, S.; Gygi, J. P.; Antunes, R. d. S.; Sette, A.; Grant, B. J.; Olsen, L. R.; Konstorum, A.; Guan, L.; Ay, F.; Kleinstein, S. H.; Peters, B.\nScore: 1.0, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555193\nComputational models that predict an individuals response to a vaccine offer the potential for mechanistic insights and personalized vaccination strategies. These models are increasingly derived from systems vaccinology studies that generate immune profiles from human cohorts pre- and post-vaccination. Most of these studies involve relatively small cohorts and profile the response to a single vaccine. The ability to assess the performance of the resulting models would be improved by comparing their performance on independent datasets, as has been done with great success in other areas of biology such as protein structure predictions. To transfer this approach to system vaccinology studies, we established a prototype platform that focuses on the evaluation of Computational Models of Immunity to Pertussis Booster vaccinations (CMI-PB). A community resource, CMI-PB generates experimental data for the explicit purpose of model evaluation, which is performed through a series of annual data releases and associated contests. We here report on our experience with the first such dry run for a contest where the goal was to predict individual immune responses based on pre-vaccination multi-omic profiles. Over 30 models adopted from the literature were tested, but only one was predictive, and was based on age alone. The performance of new models built using CMI-PB training data was much better, but varied significantly based on the choice of pre-vaccination features used and the model building strategy. This suggests that previously published models developed for other vaccines do not generalize well to Pertussis Booster vaccination. Overall, these results reinforced the need for comparative analysis across models and datasets that CMI-PB aims to achieve. We are seeking wider community engagement for our first public prediction contest, which will open in early 2024.\nEvaluating single-cell variability in proteasomal decay\nAuthors: Das, S.; Singh, A.; Shah, P.\nScore: 6.1, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554358\nGene expression is a stochastic process that leads to variability in mRNA and protein abundances even within an isogenic population of cells grown in the same environment. This variation, often called gene-expression noise, has typically been attributed to transcriptional and translational processes while ignoring the contributions of protein decay variability across cells. Here we estimate the single-cell protein decay rates of two degron GFPs in Saccharomyces cerevisiae using time-lapse microscopy. We find substantial cell-to-cell variability in the decay rates of the degron GFPs. We evaluate cellular features that explain the variability in the proteasomal decay and find that the amount of 20s catalytic beta subunit of the proteasome marginally explains the observed variability in the degron GFP half-lives. We propose alternate hypotheses that might explain the observed variability in the decay of the two degron GFPs. Overall, our study highlights the importance of studying the kinetics of the decay process at single-cell resolution and that decay rates vary at the single-cell level, and that the decay process is stochastic. A complex model of decay dynamics must be included when modeling stochastic gene expression to estimate gene expression noise.\n",
  "wordCount" : "2744",
  "inLanguage": "en",
  "datePublished": "2023-09-03T10:37:18Z",
  "dateModified": "2023-09-03T10:37:18Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on September 3, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553655" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553655">
        <p class="paperTitle">The interplay between lumen pressure and cell proliferation determines organoid morphology in a multicellular phase field model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tanida, S.; Fuji, K.; Lu, L.; Guyomar, T.; Lee, B. H.; Honigmann, A.; Grapin-Botton, A.; Riveline, D.; Nonomura, M.; Sano, M.; Hiraiwa, T.</p>
        <p class="info">Score: 13.8, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553655' target='https://doi.org/10.1101/2023.08.17.553655'> 10.1101/2023.08.17.553655</a></p>
        <p class="abstract">Organoids are ideal systems to predict the phenotypes of organs. However, there is currently a lack of understanding regarding the generalized rules that enable use of simple cellular principles to make morphological predictions of entire organoids. Therefore, we employed a phase field model with the following basic components: the minimum conditions for the timing and volume of cell division, lumen nucleation rules, and lumenal pressure. Through our model, we could compute and generate a myriad of organoid phenotypes observed till date. We propose morphological indices necessary to characterize the shapes and construct phase diagrams and show their dependencies on proliferation time and lumen pressure. Additionally, we introduced the lumen-index parameter, which helped in examining the criteria to maintain organoids as spherical structures comprising a single layer of cells and enclosing an intact lumen. Finally, we predict a star-like organoid phenotype that did not undergo differentiation, suggesting that the volume constraint during cell division may determine the final phenotype. In summary, our approach provides researchers with guidelines to test the mechanisms of self-organization and predict the shape of organoid.

Author summaryIn nature, a wide variety of organ morphologies are observed. Owing to the complexity of the process underlying the acquisition of organs morphology, it is challenging to investigate the mechanisms that lead to such variations. A promising approach to study these variations is the use of &#34;computational organoid&#34; study, which is the computational-based study of self-organizing shapes in multicellular assemblies and fluid-filled cavities called lumens that develop from a few proliferating cells. This study explores general mechanisms that dictate how various mechanical factors affect the growing self-organized multicellular assembly. We relied on computer simulations of the mathematical model called multicellular phase-field model with lumens and explored the mechanical factor effects, such as the lumen pressure while considering the time and volume conditions required for cell division. These simulations generated and categorized a wide range of organoid phenotypes based on the varying lumen pressure and cell division conditions. These phenotypes were characterized into seven distinct classes, based on the morphological index sets, including a cellular monolayer/multilayer surrounding single or multiple lumens and branch formation. These phenotypes were obtained without the assumption of differentiation. Our study elucidates the mechanisms underlying the organoid and organ formation with different shapes, thereby highlighting the significance of mechanical forces in shaping these complex biological structures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555073">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555073" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555073">
        <p class="paperTitle">Transcription factors operate on a limited vocabulary of binding motifs in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555073" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555073" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zenker, S.; Wulf, D.; Meierhenrich, A.; Becker, S.; Eisenhut, M.; Stracke, R.; Weisshaar, B.; Braeutigam, A.</p>
        <p class="info">Score: 8.7, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555073' target='https://doi.org/10.1101/2023.08.28.555073'> 10.1101/2023.08.28.555073</a></p>
        <p class="abstract">Predicting gene expression from promoter sequence requires understanding of the different signal integration points within a promoter. Sequence-specific transcription factors (TFs) binding to their cognate TF binding motifs control gene expression in eukaryotes by activating and repressing transcription. Their interplay generates complex expression patterns in reaction to environmental conditions and developmental cues.

We hypothesized that signals are not only integrated by different TFs binding various positions in a promoter, but also by single TF binding motifs onto which multiple TFs can bind. Analyzing 2,190 binding motifs, we identified only 76 core TF binding motifs in plants. Twenty-one TF protein families act highly specific and bind a single conserved motif. Four TF families are classified as semi-conserved as they bind up to four motifs within a family, with divisions along phylogenetic groups. Five TF families bind diverse motifs. Expression analyses revealed high competition within TF families for the same binding motif. The results show that singular binding motifs act as signal integrators in plants where a combination of binding affinity and TF abundance likely determine the output.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554578">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554578" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554578">
        <p class="paperTitle">Dynamics of bacterial operons during genome-wide stresses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554578" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554578" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jagadeesan, R.; Dash, S.; Palma, C.; Baptista, I. S. C.; Chauhan, V.; Makela, J.; Ribeiro, A. S.</p>
        <p class="info">Score: 8.2, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554578' target='https://doi.org/10.1101/2023.08.24.554578'> 10.1101/2023.08.24.554578</a></p>
        <p class="abstract">Bacterial gene networks have many operons, each coordinating the kinetics of a few genes in time and strength. In Escherichia coli, nearly half of these operons have internal promoters, which allow regulating the downstream genes, independently from upstream genes. We studied their role during genome-wide stresses targeting a few key elements of the transcription machinery, namely, RNAP and gyrase. We show that absolute differences in response strength between genes in the same operon follow a sinusoidal function of the distance between them. This results from the combined effect of elongation fall-offs and internal promoters. The frequency of this function is regulated by the positioning of internal promoters, while its amplitude differs with the perturbation strength. Moreover, we find that positive supercoiling buildup partially explains the RNAP fall-off rates. Finally, we report similar sinusoidal patterns in E. coli cells subject to several other stresses, as well as in Bacillus subtilis, Corynebacterium glutamicum, and Helicobacter pylori. Overall, our results suggest that the strength, number, and positioning of internal promoters in operons are influenced by a genome-wide need to compensate for RNAP fall-offs, and thus provide distal genes in the operon with similar response strength as genes proximal to the primary promoter.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.552381">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.552381" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.552381">
        <p class="paperTitle">On standardization of controls in lifespan studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.552381" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.552381" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Spiridonova, O.; Kriukov, D.; Nemirovich-Danchenko, N.; Peshkin, L.</p>
        <p class="info">Score: 9.9, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.552381' target='https://doi.org/10.1101/2023.08.17.552381'> 10.1101/2023.08.17.552381</a></p>
        <p class="abstract">The search for interventions to slow down and even reverse aging is a burgeoning field. The literature cites hundreds of supposedly beneficial pharmacological and genetic interventions in model organisms: mice, rats, flies and worms, where research into physiology is routinely accompanied by lifespan data. Naturally the negative results are more frequent, yet scientifically quite valuable if analyzed systematically. Yet, there is a strong &#34;discovery bias&#34;, i.e. results of interventions which turn out not to be beneficial remain unpublished. Theoretically, all lifespan data is ripe for re-analysis: we could contrast the molecular targets and pathways across studies and help focus the further search for interventions. Alas, the results of most longevity studies are difficult to compare. This is in part because there are no clear, universally accepted standards for conducting such experiments or even for reporting such data. The situation is worsened by the fact that the authors often do not describe experimental conditions completely. As a result, works on longevity make up a set of precedents, each of which might be interesting in its own right, yet incoherent and incomparable. Here we point out specific issues and propose solutions for quality control by checking both inter- and intra-study consistency of lifespan data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555470">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555470" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555470">
        <p class="paperTitle">Evolution of hierarchical switching pattern in antigenic variation of Plasmodium falciparum under variable host immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555470" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555470" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghosh, B.; Priya, I. G.</p>
        <p class="info">Score: 1.6, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555470' target='https://doi.org/10.1101/2023.08.30.555470'> 10.1101/2023.08.30.555470</a></p>
        <p class="abstract">The var genes family encoding the variants of the erythrocyte membrane protein of Plasmodium falciparum is crucial for virulence of the parasite inside host. The transcriptional output of the var genes switches from one variant to other in a mutually exclusive fashion. It is proposed that a biased hierarchical switching pattern optimizes the growth and survival of the parasite inside the host. Apart from the hierarchical switching pattern, it is also well established that the intrinsic switching rates vary widely among the var genes. The centromeric protein like Var2csa is much more stable than the genes located at the telomeric and sub-telomeric regions of the chromosomes. In this study, we explored the evolutionary advantage achieved through selecting variable switching rates. Our theoretical analysis based on a mathematical model coupled with single cell RNA-seq data suggests that the variable switching rate is beneficial when cells expressing different variants are deferentially amenable to be cleared by the immune response. In fact, the variants which are cleared by the immune systems more efficiently are more stably expressed compared to a variant attacked by the immune system much less vigorously. The cells turn off expression of the variant quickly which is not cleared very efficiently. The evolutionary simulation shows that this strategy maximizes the growth of the parasite population under the presence of immune attack by the host. In corroboration with the result, we observed that stable variant has higher binding affinity to IgM from experimental data. Our study provides an evolutionary basis of widely variable switching rates of the var genes in Plasmodium falciparum.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554757" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554757">
        <p class="paperTitle">Integration of graph neural networks and genome-scale metabolic models for predicting gene essentiality</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hasibi, R.; Michoel, T.; Oyarzun, D. A.</p>
        <p class="info">Score: 1.5, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554757' target='https://doi.org/10.1101/2023.08.25.554757'> 10.1101/2023.08.25.554757</a></p>
        <p class="abstract">Genome-scale metabolic models are powerful tools for understanding cellular physiology. Flux balance analysis (FBA), in particular, is a popular optimization approach for predicting metabolic phenotypes under genetic and environmental perturbations. In model microbes such as Escherichia coli, FBA has been successful at predicting essential genes, i.e. those genes that impair survival when deleted. A central assumption in this approach, however, is that both wild type and deletion strains optimize the same fitness objective. While the optimality assumption may hold for the wild type metabolic network, deletion strains are not subject to the same evolutionary pressures and knock-out mutants may steer their metabolism to meet other objectives for survival. Here, we present FlowGAT, a hybrid FBA-machine learning strategy for predicting essentiality directly from wild type metabolic phenotypes. The approach is based on graph-structured representation of metabolic fluxes predicted by FBA, where nodes correspond to enzymatic reactions and edges quantify the propagation of metabolite mass flow between a reaction and its neighbours. We integrate this information into a graph neural network that can be trained on knock-out fitness assay data. Comparisons across different model architectures reveal that FlowGAT predictions for E. coli are close to those of FBA for several growth conditions. This suggests that gene essentiality can be accurately predicted by exploiting the network structure of metabolism, without additional assumptions beyond optimality of the wild type. Our approach demonstrates the benefits of combining the mechanistic insights afforded by genome-scale models with the ability of deep learning models to extract patterns from complex data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552995">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552995" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552995">
        <p class="paperTitle">Large-scale analysis of the integration of enhancer-enhancer signals by promoters</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552995" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552995" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Ara, M.; Comoglio, F.; van Steensel, B.</p>
        <p class="info">Score: 25.7, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552995' target='https://doi.org/10.1101/2023.08.11.552995'> 10.1101/2023.08.11.552995</a></p>
        <p class="abstract">Genes are often regulated by multiple enhancers. It is poorly understood how the individual enhancer activities are combined to control promoter activity. Anecdotal evidence has shown that enhancers can combine sub-additively, additively, synergistically, or redundantly. However, it is not clear which of these modes are more frequent in mammalian genomes. Here, we systematically tested how pairs of enhancers activate promoters using a three-way combinatorial reporter assay in mouse cells. By assaying about 69,000 enhancer-enhancer-promoter combinations we found that enhancer pairs generally combine near-additively. This behaviour was conserved across seven developmental promoters tested. Surprisingly, these promoters scale the enhancer signals approximately following a power-law, but the exponent of this response varies between promoters. A housekeeping promoter showed an overall different response to enhancer pairs, and a smaller dynamic range. Thus, our data indicate that enhancers mostly act additively, but promoters transform their collective effect non-linearly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555248">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555248" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555248">
        <p class="paperTitle">A trade-off in controlling upstream and downstream noise in signaling networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555248" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555248" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kong, K. K.; Luo, C.; Liu, F.</p>
        <p class="info">Score: 1.0, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555248' target='https://doi.org/10.1101/2023.08.29.555248'> 10.1101/2023.08.29.555248</a></p>
        <p class="abstract">Signal transduction, underpinning the function of a variety of biological systems, is inevitably affected by fluctuations. It remains intriguing how the timescale of a signaling network relates to its capability of noise control, specifically, whether long timescale can average out fluctuation or accumulate fluctuation. Here, we consider two noise components of the signaling system: the upstream noise from the fluctuation of the input signal and the downstream noise from the stochastic fluctuations of the network. We discover a fundamental trade-off in controlling the upstream and downstream noise: a longer timescale of the signaling network can buffer upstream noise, while accumulate downstream noise. Moreover, we confirm that this trade-off relation exists in real biological signaling networks such as a fold-change detection circuit and the p53 activation signaling system.

Author SummaryInformation transmission is vital in biological systems, such as decoding the information regarding nutrient levels during chemotaxis or morphogen concentrations in tissue development. While fluctuations arising from the stochastic nature of biological processes inevitably affect information transmission, noise control mechanisms have been studied for decades. However, it remains controversial what the role of the slow dynamics (long timescale) is in noise control. On the one hand, it has been reported to attenuate noise by averaging out fluctuations. On the other hand, it is also proposed to amplify noise by accumulating fluctuations. Here we dissect the noise in signaling systems into two components: upstream noise originating from signal fluctuation, and downstream noise from the network stochasticity. Our analysis reveals that upstream noise negatively correlates with timescale, while downstream noise exhibits a positive correlation, indicating a fundamental trade-off in controlling the two noise components. Moreover, we provide an intuitive illustration to understand this phenomenon using the concept of landscape representation. Mathematically, we analytically derive a trade-off relation that agrees well with simulations. Our results uncover a new property of noise in signaling processes, deepening our understanding of noise control and proposing a new perspective in designing signaling network.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555193" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555193">
        <p class="paperTitle">A systems vaccinology resource to develop and test computational models of immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shinde, P.; Soldevila, F.; Reyna, J.; Aoki, M.; Rasmussen, M.; Willemsen, L.; Kojima, M.; Ha, B.; Greenbaum, J. A.; Overton, J. A.; Guzman-Orozco, H.; Nili, S.; Orfield, S.; Gygi, J. P.; Antunes, R. d. S.; Sette, A.; Grant, B. J.; Olsen, L. R.; Konstorum, A.; Guan, L.; Ay, F.; Kleinstein, S. H.; Peters, B.</p>
        <p class="info">Score: 1.0, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555193' target='https://doi.org/10.1101/2023.08.28.555193'> 10.1101/2023.08.28.555193</a></p>
        <p class="abstract">Computational models that predict an individuals response to a vaccine offer the potential for mechanistic insights and personalized vaccination strategies. These models are increasingly derived from systems vaccinology studies that generate immune profiles from human cohorts pre- and post-vaccination. Most of these studies involve relatively small cohorts and profile the response to a single vaccine. The ability to assess the performance of the resulting models would be improved by comparing their performance on independent datasets, as has been done with great success in other areas of biology such as protein structure predictions. To transfer this approach to system vaccinology studies, we established a prototype platform that focuses on the evaluation of Computational Models of Immunity to Pertussis Booster vaccinations (CMI-PB). A community resource, CMI-PB generates experimental data for the explicit purpose of model evaluation, which is performed through a series of annual data releases and associated contests. We here report on our experience with the first such  dry run for a contest where the goal was to predict individual immune responses based on pre-vaccination multi-omic profiles. Over 30 models adopted from the literature were tested, but only one was predictive, and was based on age alone. The performance of new models built using CMI-PB training data was much better, but varied significantly based on the choice of pre-vaccination features used and the model building strategy. This suggests that previously published models developed for other vaccines do not generalize well to Pertussis Booster vaccination. Overall, these results reinforced the need for comparative analysis across models and datasets that CMI-PB aims to achieve. We are seeking wider community engagement for our first public prediction contest, which will open in early 2024.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554358">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554358" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554358">
        <p class="paperTitle">Evaluating single-cell variability in proteasomal decay</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554358" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554358" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Das, S.; Singh, A.; Shah, P.</p>
        <p class="info">Score: 6.1, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554358' target='https://doi.org/10.1101/2023.08.22.554358'> 10.1101/2023.08.22.554358</a></p>
        <p class="abstract">Gene expression is a stochastic process that leads to variability in mRNA and protein abundances even within an isogenic population of cells grown in the same environment. This variation, often called gene-expression noise, has typically been attributed to transcriptional and translational processes while ignoring the contributions of protein decay variability across cells. Here we estimate the single-cell protein decay rates of two degron GFPs in Saccharomyces cerevisiae using time-lapse microscopy. We find substantial cell-to-cell variability in the decay rates of the degron GFPs. We evaluate cellular features that explain the variability in the proteasomal decay and find that the amount of 20s catalytic beta subunit of the proteasome marginally explains the observed variability in the degron GFP half-lives. We propose alternate hypotheses that might explain the observed variability in the decay of the two degron GFPs. Overall, our study highlights the importance of studying the kinetics of the decay process at single-cell resolution and that decay rates vary at the single-cell level, and that the decay process is stochastic. A complex model of decay dynamics must be included when modeling stochastic gene expression to estimate gene expression noise.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
