<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput
Authors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.
Score: 16.4, Published: 2023-11-14 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput
Authors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.
Score: 16.4, Published: 2023-11-14 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput
Authors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.
Score: 16.4, Published: 2023-11-14 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput\nAuthors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.\nScore: 16.4, Published: 2023-11-14 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput\nAuthors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.\nScore: 16.4, Published: 2023-11-14 DOI: 10.1101/2023.11.11.566649\nSignaling pathways that drive gene expression are typically depicted as having a dozen or so landmark phosphorylation and transcriptional events. In reality, thousands of dynamic post-translational modifications (PTMs) orchestrate nearly every cellular function, and we lack technologies to find causal links between these vast biochemical pathways and genetic circuits at scale. Here, we describe \"signaling-to-transcription network\" mapping through the development of PTM-centric base editing coupled to phenotypic screens, directed by temporally-resolved phosphoproteomics. Using T cell activation as a model, we observe hundreds of unstudied phosphorylation sites that modulate NFAT transcriptional activity. We identify the phosphorylation-mediated nuclear localization of the phosphatase PHLPP1 which promotes NFAT but inhibits NF{kappa}B activity. We also find that specific phosphosite mutants can alter gene expression in subtle yet distinct patterns, demonstrating the potential for fine-tuning transcriptional responses. Overall, base editor screening of PTM sites provides a powerful platform to dissect PTM function within signaling pathways.\nShort-term Pre-operative Methionine Restriction Induces Browning of Perivascular Adipose Tissue and Improves Vein Graft Remodeling in Mice\nAuthors: Kip, P.; Sluiter, T. J.; MacArthur, M. R.; Tao, M.; Jung, J.; Mitchell, S. J.; Kooijman, S.; Kruit, N.; Gorham, J.; Seidman, J.; Quax, P. H. A.; Aikawa, M.; Ozaki, C. K.; Mitchell, J. R.; De Vries, M. R.\nScore: 4.1, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565269\nShort-term preoperative methionine restriction (MetR) shows promise as a translatable strategy to modulate the bodys response to surgical injury. Its application, however, to improve post-interventional vascular remodeling remains underexplored. Here, we find that MetR protects from arterial intimal hyperplasia in a focal stenosis model and adverse vascular remodeling after vein graft surgery. RNA sequencing reveals that MetR enhances the brown adipose tissue phenotype in arterial perivascular adipose tissue (PVAT) and induces it in venous PVAT. Specifically, PPAR- was highly upregulated in PVAT-adipocytes. Furthermore, MetR dampens the post-operative pro-inflammatory response to surgery in PVAT-macrophages in vivo and in vitro. This study shows for the first time that the detrimental effects of dysfunctional PVAT on vascular remodeling can be reversed by MetR, and identifies pathways involved in browning of PVAT. Furthermore, we demonstrate the potential of short-term pre-operative MetR as a simple intervention to ameliorate vascular remodeling after vascular surgery. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=109 SRC=\"FIGDIR/small/565269v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (38K): org.highwire.dtl.DTLVardef@137314corg.highwire.dtl.DTLVardef@858182org.highwire.dtl.DTLVardef@f9ba98org.highwire.dtl.DTLVardef@1496d5a_HPS_FORMAT_FIGEXP M_FIG C_FIG\nSpatial Metabolomics co-registered with Multiplex Phenotyping for the Evaluation of Human Kidney Tissue\nAuthors: Zickuhr, G. M.; Um, I. H.; Laird, A.; Harrison, D. J.; Dickson, A. L.\nScore: 3.1, Published: 2023-11-15 DOI: 10.1101/2023.11.10.566529\nA workflow has been evaluated that utilises a single tissue section to obtain spatially co-registered, molecular, and phenotypical information suitable for AI-enabled analysis. The impact of varying DESI-MSI conditions (e.g., temperature, scan rate, acquisition time) on the detection of small molecules and on tissue quality for integration into typical clinical pathology workflows assessed in human kidney.\nAn enzyme activation network provides evidence for extensive regulatory crosstalk between metabolic pathways\nAuthors: Alam, M. T.; Al Zubaidi, S. M.; Nasar, M. I.; Ralser, M.; Notebaart, R. A.\nScore: 2.8, Published: 2023-11-18 DOI: 10.1101/2023.11.16.567372\nEnzyme activation by cellular metabolites plays a pivotal role in regulating metabolic processes. Nevertheless, our comprehension of such activation events on a global network scale remains incomplete. In this study, we conducted a comprehensive investigation into the optimization of cell-intrinsic activation interactions within Saccharomyces cerevisiae. To achieve this, we integrated a genome-scale metabolic model with enzyme kinetic data sourced from the BRENDA database. Our objective was to map the distribution of enzyme activators throughout the cellular network. Our findings indicate that virtually all biochemical pathways encompass enzyme activators, frequently originating from disparate pathways, thus revealing extensive regulatory crosstalk between metabolic pathways. Indeed, activators have short pathway lengths, indicating they are activated quickly upon nutrient shifts, and in most instances, these activators target key enzymatic reactions to facilitate downstream metabolic processes. Interestingly, non-essential enzymes exhibit a significantly higher degree of activation compared to their essential counterparts. This observation suggests that cells employ enzyme activators to finely regulate secondary metabolic pathways that are only required under specific conditions. Conversely, the activator metabolites themselves are more likely to be essential components, and their activation levels surpass those of non-essential activators. In summary, our study unveils the widespread importance of enzymatic activators, and suggests that feed-forward activation of conditional metabolic pathways through essential metabolites mediates metabolic plasticity.\nFine-grained modelling of ATP dependence of decision-making capacity in genetic regulatory networks\nAuthors: Kumar, R.; Johnston, I.\nScore: 2.6, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567352\nCellular decision-making is fundamental to life, from developmental biology to environmental responses and antimicrobial resistance. Many regulatory processes that drive cellular decisions rely on gene expression, which requires energy in the form of ATP. As even genetically identical cells can have dramatically different ATP levels, bioenergetic status can be an important source of variability in cellular decision-making. Existing studies have investigated this energy dependence but often use coarse-grained modelling approaches (which are not always readily connected to the underlying molecular processes of gene regulation). Here we use a fine-grained mathematical model of gene expression in a two-gene decision-making regulatory network to explore cellular decision-making capacity as energy availability varies. We simulate both a deterministic model, to explore the emergence of different cell fate attractors as ATP levels vary, and a stochastic case to explore how ATP influences the noisy dynamics of stochastic cell decision-making. Higher energy levels typically support increased decision-making capacity (higher numbers of, and more separated, cell states that can be selected), and the fine-grained modelling reveals some differences in behaviour from previous coarse-grained modelling approaches.\nPulsed stimuli entrain p53 to synchronize single cells and modulate cell-fate determination\nAuthors: Venkatachalapathy, H.; Yang, Z.; Azarin, S. M.; Sarkar, C. A.; Batchelor, E.\nScore: 5.5, Published: 2023-11-15 DOI: 10.1101/2023.10.24.563786\nEntrainment to an external stimulus enables a synchronized oscillatory response across a population of cells, increasing coherent responses by reducing cell-to-cell heterogeneity. It is unclear whether the property of entrainability extends to systems where responses are intrinsic to the individual cell, rather than dependent on coherence across a population of cells. Using a combination of mathematical modeling, time-lapse fluorescence microscopy, and single-cell tracking, we demonstrated that p53 oscillations triggered by DNA double-strand breaks (DSBs) can be entrained with a periodic damage stimulus, despite such synchrony not known to function in effective DNA damage responses. Surprisingly, p53 oscillations were experimentally entrained over a wider range of DSB frequencies than predicted by an established computational model for the system. We determined that recapitulating the increased range of entrainment frequencies required, non-intuitively, a less robust oscillator and wider steady-state valley on the energy landscape. Further, we show that p53 entrainment can lead to altered expression dynamics of downstream targets responsible for cell fate in a manner dependent on target mRNA stability. Overall, this study demonstrates that entrainment can occur in a biological oscillator despite the apparent lack of an evolutionary advantage conferred through synchronized responses and highlights the potential of externally entraining p53 dynamics to reduce cellular variability and synchronize cell-fate responses for therapeutic outcomes.\nTrans-omic analysis reveals opposite metabolic dysregulation between feeding and fasting in liver associated with obesity\nAuthors: Bai, Y.; Morita, K.; Kokaji, T.; Hatano, A.; Ohno, S.; Egami, R.; Pan, Y.; Li, D.; Yugi, K.; Uematsu, S.; Inoue, H.; Inaba, Y.; Suzuki, Y.; Matsumoto, M.; Takahashi, M.; Yoshihiro, I.; Bamba, T.; Hirayama, A.; Soga, T.; Kuroda, S.\nScore: 1.0, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567316\nDysregulation of liver metabolism associated with obesity during feeding and fasting leads to the breakdown of metabolic homeostasis. However, the underlying mechanism remains unknown. Here, we measured multi-omics data in the liver of wild-type and leptin-deficient obese (ob/ob) mice at ad libitum feeding, and constructed a differential regulatory trans-omic network of metabolic reactions. We compared the trans-omic network at feeding with that at 16 h-fasting constructed in our previous study. Intermediate metabolites in glycolytic and nucleotide metabolism decreased in ob/ob mice at feeding but increased at fasting. Allosteric regulation reversely shifted between feeding and fasting, generally showing activation at feeding while inhibition at fasting in ob/ob mice. Transcriptional regulation was similar between feeding and fasting, generally showing inhibiting transcription factor regulations, activating enzyme protein regulations in ob/ob mice. The opposite metabolic dysregulation between feeding and fasting characterizes breakdown of metabolic homeostasis associated with obesity.\nSynthetic model ecosystem of 12 cryopreservable microbial species allowing for a noninvasive approach\nAuthors: Hosoda, K.; Seno, S.; Murakami, N.; Matsuda, H.; Osada, Y.; Kamiura, R.; Kondoh, M.\nScore: 30.3, Published: 2023-10-31 DOI: 10.1101/2020.10.23.351742\nSimultaneous understanding of both individual and ecosystem dynamics is crucial in an era marked by the degradation of ecosystem services. Herein, we present a high-throughput synthetic microcosm system comprising 12 functionally and phylogenetically diverse microbial species. These species are axenically culturable, cryopreservable, and can be measured noninvasively via microscopy, aided by machine learning. This system includes prokaryotic and eukaryotic producers and decomposers, and eukaryotic consumers to ensure functional redundancy. Our model system displayed both positive and negative interspecific interactions and higher-order interactions that surpassed the scope of any two-species interaction. Although complete species coexistence was not our primary objective, we identified several conditions under which at least one species from the producers, consumers, and decomposers groups, and one functionally redundant species, persisted for over six months. These conditions set the stage for detailed investigations in the future. Given its designability and experimental replicability, our model ecosystem offers a promising platform for deeper insights into both individual and ecosystem dynamics, including evolution and species interactions.\n",
  "wordCount" : "1633",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566649" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566649">
        <p class="paperTitle">Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.</p>
        <p class="info">Score: 16.4, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566649' target='https://doi.org/10.1101/2023.11.11.566649'> 10.1101/2023.11.11.566649</a></p>
        <p class="abstract">Signaling pathways that drive gene expression are typically depicted as having a dozen or so landmark phosphorylation and transcriptional events. In reality, thousands of dynamic post-translational modifications (PTMs) orchestrate nearly every cellular function, and we lack technologies to find causal links between these vast biochemical pathways and genetic circuits at scale. Here, we describe &#34;signaling-to-transcription network&#34; mapping through the development of PTM-centric base editing coupled to phenotypic screens, directed by temporally-resolved phosphoproteomics. Using T cell activation as a model, we observe hundreds of unstudied phosphorylation sites that modulate NFAT transcriptional activity. We identify the phosphorylation-mediated nuclear localization of the phosphatase PHLPP1 which promotes NFAT but inhibits NF{kappa}B activity. We also find that specific phosphosite mutants can alter gene expression in subtle yet distinct patterns, demonstrating the potential for fine-tuning transcriptional responses. Overall, base editor screening of PTM sites provides a powerful platform to dissect PTM function within signaling pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565269">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565269" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565269">
        <p class="paperTitle">Short-term Pre-operative Methionine Restriction Induces Browning of Perivascular Adipose Tissue and Improves Vein Graft Remodeling in Mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565269" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565269" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kip, P.; Sluiter, T. J.; MacArthur, M. R.; Tao, M.; Jung, J.; Mitchell, S. J.; Kooijman, S.; Kruit, N.; Gorham, J.; Seidman, J.; Quax, P. H. A.; Aikawa, M.; Ozaki, C. K.; Mitchell, J. R.; De Vries, M. R.</p>
        <p class="info">Score: 4.1, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565269' target='https://doi.org/10.1101/2023.11.02.565269'> 10.1101/2023.11.02.565269</a></p>
        <p class="abstract">Short-term preoperative methionine restriction (MetR) shows promise as a translatable strategy to modulate the bodys response to surgical injury. Its application, however, to improve post-interventional vascular remodeling remains underexplored. Here, we find that MetR protects from arterial intimal hyperplasia in a focal stenosis model and adverse vascular remodeling after vein graft surgery. RNA sequencing reveals that MetR enhances the brown adipose tissue phenotype in arterial perivascular adipose tissue (PVAT) and induces it in venous PVAT. Specifically, PPAR- was highly upregulated in PVAT-adipocytes. Furthermore, MetR dampens the post-operative pro-inflammatory response to surgery in PVAT-macrophages in vivo and in vitro. This study shows for the first time that the detrimental effects of dysfunctional PVAT on vascular remodeling can be reversed by MetR, and identifies pathways involved in browning of PVAT. Furthermore, we demonstrate the potential of short-term pre-operative MetR as a simple intervention to ameliorate vascular remodeling after vascular surgery.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=109 SRC=&#34;FIGDIR/small/565269v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (38K):
org.highwire.dtl.DTLVardef@137314corg.highwire.dtl.DTLVardef@858182org.highwire.dtl.DTLVardef@f9ba98org.highwire.dtl.DTLVardef@1496d5a_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566529">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566529" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566529">
        <p class="paperTitle">Spatial Metabolomics co-registered with Multiplex Phenotyping for the Evaluation of Human Kidney Tissue</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566529" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566529" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zickuhr, G. M.; Um, I. H.; Laird, A.; Harrison, D. J.; Dickson, A. L.</p>
        <p class="info">Score: 3.1, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566529' target='https://doi.org/10.1101/2023.11.10.566529'> 10.1101/2023.11.10.566529</a></p>
        <p class="abstract">A workflow has been evaluated that utilises a single tissue section to obtain spatially co-registered, molecular, and phenotypical information suitable for AI-enabled analysis. The impact of varying DESI-MSI conditions (e.g., temperature, scan rate, acquisition time) on the detection of small molecules and on tissue quality for integration into typical clinical pathology workflows assessed in human kidney.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567372">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567372" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567372">
        <p class="paperTitle">An enzyme activation network provides evidence for extensive regulatory crosstalk between metabolic pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567372" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567372" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alam, M. T.; Al Zubaidi, S. M.; Nasar, M. I.; Ralser, M.; Notebaart, R. A.</p>
        <p class="info">Score: 2.8, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567372' target='https://doi.org/10.1101/2023.11.16.567372'> 10.1101/2023.11.16.567372</a></p>
        <p class="abstract">Enzyme activation by cellular metabolites plays a pivotal role in regulating metabolic processes. Nevertheless, our comprehension of such activation events on a global network scale remains incomplete. In this study, we conducted a comprehensive investigation into the optimization of cell-intrinsic activation interactions within Saccharomyces cerevisiae. To achieve this, we integrated a genome-scale metabolic model with enzyme kinetic data sourced from the BRENDA database. Our objective was to map the distribution of enzyme activators throughout the cellular network. Our findings indicate that virtually all biochemical pathways encompass enzyme activators, frequently originating from disparate pathways, thus revealing extensive regulatory crosstalk between metabolic pathways. Indeed, activators have short pathway lengths, indicating they are activated quickly upon nutrient shifts, and in most instances, these activators target key enzymatic reactions to facilitate downstream metabolic processes. Interestingly, non-essential enzymes exhibit a significantly higher degree of activation compared to their essential counterparts. This observation suggests that cells employ enzyme activators to finely regulate secondary metabolic pathways that are only required under specific conditions. Conversely, the activator metabolites themselves are more likely to be essential components, and their activation levels surpass those of non-essential activators. In summary, our study unveils the widespread importance of enzymatic activators, and suggests that feed-forward activation of conditional metabolic pathways through essential metabolites mediates metabolic plasticity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567352" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567352">
        <p class="paperTitle">Fine-grained modelling of ATP dependence of decision-making capacity in genetic regulatory networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumar, R.; Johnston, I.</p>
        <p class="info">Score: 2.6, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567352' target='https://doi.org/10.1101/2023.11.16.567352'> 10.1101/2023.11.16.567352</a></p>
        <p class="abstract">Cellular decision-making is fundamental to life, from developmental biology to environmental responses and antimicrobial resistance. Many regulatory processes that drive cellular decisions rely on gene expression, which requires energy in the form of ATP. As even genetically identical cells can have dramatically different ATP levels, bioenergetic status can be an important source of variability in cellular decision-making. Existing studies have investigated this energy dependence but often use coarse-grained modelling approaches (which are not always readily connected to the underlying molecular processes of gene regulation). Here we use a fine-grained mathematical model of gene expression in a two-gene decision-making regulatory network to explore cellular decision-making capacity as energy availability varies. We simulate both a deterministic model, to explore the emergence of different cell fate attractors as ATP levels vary, and a stochastic case to explore how ATP influences the noisy dynamics of stochastic cell decision-making. Higher energy levels typically support increased decision-making capacity (higher numbers of, and more separated, cell states that can be selected), and the fine-grained modelling reveals some differences in behaviour from previous coarse-grained modelling approaches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563786">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563786" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563786">
        <p class="paperTitle">Pulsed stimuli entrain p53 to synchronize single cells and modulate cell-fate determination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563786" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563786" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venkatachalapathy, H.; Yang, Z.; Azarin, S. M.; Sarkar, C. A.; Batchelor, E.</p>
        <p class="info">Score: 5.5, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563786' target='https://doi.org/10.1101/2023.10.24.563786'> 10.1101/2023.10.24.563786</a></p>
        <p class="abstract">Entrainment to an external stimulus enables a synchronized oscillatory response across a population of cells, increasing coherent responses by reducing cell-to-cell heterogeneity. It is unclear whether the property of entrainability extends to systems where responses are intrinsic to the individual cell, rather than dependent on coherence across a population of cells. Using a combination of mathematical modeling, time-lapse fluorescence microscopy, and single-cell tracking, we demonstrated that p53 oscillations triggered by DNA double-strand breaks (DSBs) can be entrained with a periodic damage stimulus, despite such synchrony not known to function in effective DNA damage responses. Surprisingly, p53 oscillations were experimentally entrained over a wider range of DSB frequencies than predicted by an established computational model for the system. We determined that recapitulating the increased range of entrainment frequencies required, non-intuitively, a less robust oscillator and wider steady-state valley on the energy landscape. Further, we show that p53 entrainment can lead to altered expression dynamics of downstream targets responsible for cell fate in a manner dependent on target mRNA stability. Overall, this study demonstrates that entrainment can occur in a biological oscillator despite the apparent lack of an evolutionary advantage conferred through synchronized responses and highlights the potential of externally entraining p53 dynamics to reduce cellular variability and synchronize cell-fate responses for therapeutic outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567316">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567316" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567316">
        <p class="paperTitle">Trans-omic analysis reveals opposite metabolic dysregulation between feeding and fasting in liver associated with obesity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567316" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567316" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bai, Y.; Morita, K.; Kokaji, T.; Hatano, A.; Ohno, S.; Egami, R.; Pan, Y.; Li, D.; Yugi, K.; Uematsu, S.; Inoue, H.; Inaba, Y.; Suzuki, Y.; Matsumoto, M.; Takahashi, M.; Yoshihiro, I.; Bamba, T.; Hirayama, A.; Soga, T.; Kuroda, S.</p>
        <p class="info">Score: 1.0, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567316' target='https://doi.org/10.1101/2023.11.15.567316'> 10.1101/2023.11.15.567316</a></p>
        <p class="abstract">Dysregulation of liver metabolism associated with obesity during feeding and fasting leads to the breakdown of metabolic homeostasis. However, the underlying mechanism remains unknown. Here, we measured multi-omics data in the liver of wild-type and leptin-deficient obese (ob/ob) mice at ad libitum feeding, and constructed a differential regulatory trans-omic network of metabolic reactions. We compared the trans-omic network at feeding with that at 16 h-fasting constructed in our previous study. Intermediate metabolites in glycolytic and nucleotide metabolism decreased in ob/ob mice at feeding but increased at fasting. Allosteric regulation reversely shifted between feeding and fasting, generally showing activation at feeding while inhibition at fasting in ob/ob mice. Transcriptional regulation was similar between feeding and fasting, generally showing inhibiting transcription factor regulations, activating enzyme protein regulations in ob/ob mice. The opposite metabolic dysregulation between feeding and fasting characterizes breakdown of metabolic homeostasis associated with obesity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2020.10.23.351742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2020.10.23.351742" aria-expanded="false" aria-controls="flush-collapse10.1101/2020.10.23.351742">
        <p class="paperTitle">Synthetic model ecosystem of 12 cryopreservable microbial species allowing for a noninvasive approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2020.10.23.351742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2020.10.23.351742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hosoda, K.; Seno, S.; Murakami, N.; Matsuda, H.; Osada, Y.; Kamiura, R.; Kondoh, M.</p>
        <p class="info">Score: 30.3, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2020.10.23.351742' target='https://doi.org/10.1101/2020.10.23.351742'> 10.1101/2020.10.23.351742</a></p>
        <p class="abstract">Simultaneous understanding of both individual and ecosystem dynamics is crucial in an era marked by the degradation of ecosystem services. Herein, we present a high-throughput synthetic microcosm system comprising 12 functionally and phylogenetically diverse microbial species. These species are axenically culturable, cryopreservable, and can be measured noninvasively via microscopy, aided by machine learning. This system includes prokaryotic and eukaryotic producers and decomposers, and eukaryotic consumers to ensure functional redundancy. Our model system displayed both positive and negative interspecific interactions and higher-order interactions that surpassed the scope of any two-species interaction. Although complete species coexistence was not our primary objective, we identified several conditions under which at least one species from the producers, consumers, and decomposers groups, and one functionally redundant species, persisted for over six months. These conditions set the stage for detailed investigations in the future. Given its designability and experimental replicability, our model ecosystem offers a promising platform for deeper insights into both individual and ecosystem dynamics, including evolution and species interactions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
