<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer
Authors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.
Score: 25.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568149">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer
Authors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.
Score: 25.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568149" />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-26T10:37:33+00:00" />
<meta property="article:modified_time" content="2023-11-26T10:37:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer
Authors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.
Score: 25.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568149"/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer\nAuthors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.\nScore: 25.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568149",
  "keywords": [
    
  ],
  "articleBody": " One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer\nAuthors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.\nScore: 25.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568149\nAs a key mechanism for cellular signal transduction, protein phosphorylation plays critical roles in myriad cellular processes. This modification, however, is highly dynamic and occurs at sub-stoichiometric levels. Mass spectrometry is an essential tool for studying this modification on a global scale; however, the technologys impact suffers from four main limitations: need for site localization, dynamic range, reproducibility, and throughput. Here we describe the use of a novel mass spectrometer (Orbitrap Astral) coupled with data-independent acquisition (DIA) to achieve detection of close to 40,000 unique phosphorylation sites within one hour of analysis. We applied this approach to generate a phosphoproteome atlas of the mouse. Altogether, we detected 81,120 unique phosphorylation sites within 12 hours of measurement. With this unique dataset, we examine the sequence and structural context of protein phosphorylation. Finally, we highlight the discovery potential of this resource with multiple examples of novel phosphorylation events relevant to mitochondrial and brain biology.\nSystematic optimization of automated phosphopeptide enrichment for high-sensitivity phosphoproteomics\nAuthors: Bortel, P.; Piga, I.; Koenig, C.; Gerner, C.; Martinez del Val, A.; Olsen, J. V.\nScore: 23.6, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568418\nImproving coverage, robustness and sensitivity is crucial for routine phosphoproteomics analysis by single-shot liquid chromatography tandem mass spectrometry (LC-MS/MS) runs from minimal peptide inputs. Here, we systematically optimized key experimental parameters for automated on-beads phosphoproteomics sample preparation with focus on low input samples. Assessing the number of identified phosphopeptides, enrichment efficiency, site localization scores and relative enrichment of multiply-phosphorylated peptides pinpointed critical variables influencing the resulting phosphoproteome. Optimizing glycolic acid concentration in the loading buffer, percentage of ammonium hydroxide in the elution buffer, peptide-to-beads ratio, binding time, sample and loading buffer volumes, allowed us to confidently identify \u003e16,000 phosphopeptides in half-an-hour LC-MS/MS on an Orbitrap Exploris 480 using 30 ug of peptides as starting material. Furthermore, we evaluated how sequential enrichment can boost phosphoproteome coverage and showed that pooling fractions into a single LC-MS/MS analysis increased the depth. We also present an alternative phosphopeptide enrichment strategy based on stepwise addition of beads thereby boosting phosphoproteome coverage by 20%. Finally, we applied our optimized strategy to evaluate phosphoproteome depth with the Orbitrap Astral MS using a cell dilution series and were able to identify \u003e32,000 phosphopeptides from 0.5 million HeLa cells in half-an-hour LC-MS/MS using narrow-window data-independent acquisition (nDIA).\nSynthetic model ecosystem of 12 cryopreservable microbial species allowing for a noninvasive approach\nAuthors: Hosoda, K.; Seno, S.; Murakami, N.; Matsuda, H.; Osada, Y.; Kamiura, R.; Kondoh, M.\nScore: 39.8, Published: 2023-10-31 DOI: 10.1101/2020.10.23.351742\nSimultaneous understanding of both individual and ecosystem dynamics is crucial in an era marked by the degradation of ecosystem services. Herein, we present a high-throughput synthetic microcosm system comprising 12 functionally and phylogenetically diverse microbial species. These species are axenically culturable, cryopreservable, and can be measured noninvasively via microscopy, aided by machine learning. This system includes prokaryotic and eukaryotic producers and decomposers, and eukaryotic consumers to ensure functional redundancy. Our model system displayed both positive and negative interspecific interactions and higher-order interactions that surpassed the scope of any two-species interaction. Although complete species coexistence was not our primary objective, we identified several conditions under which at least one species from the producers, consumers, and decomposers groups, and one functionally redundant species, persisted for over six months. These conditions set the stage for detailed investigations in the future. Given its designability and experimental replicability, our model ecosystem offers a promising platform for deeper insights into both individual and ecosystem dynamics, including evolution and species interactions.\nAn enzyme activation network provides evidence for extensive regulatory crosstalk between metabolic pathways\nAuthors: Alam, M. T.; Al Zubaidi, S. M.; Nasar, M. I.; Ralser, M.; Notebaart, R. A.\nScore: 12.7, Published: 2023-11-18 DOI: 10.1101/2023.11.16.567372\nEnzyme activation by cellular metabolites plays a pivotal role in regulating metabolic processes. Nevertheless, our comprehension of such activation events on a global network scale remains incomplete. In this study, we conducted a comprehensive investigation into the optimization of cell-intrinsic activation interactions within Saccharomyces cerevisiae. To achieve this, we integrated a genome-scale metabolic model with enzyme kinetic data sourced from the BRENDA database. Our objective was to map the distribution of enzyme activators throughout the cellular network. Our findings indicate that virtually all biochemical pathways encompass enzyme activators, frequently originating from disparate pathways, thus revealing extensive regulatory crosstalk between metabolic pathways. Indeed, activators have short pathway lengths, indicating they are activated quickly upon nutrient shifts, and in most instances, these activators target key enzymatic reactions to facilitate downstream metabolic processes. Interestingly, non-essential enzymes exhibit a significantly higher degree of activation compared to their essential counterparts. This observation suggests that cells employ enzyme activators to finely regulate secondary metabolic pathways that are only required under specific conditions. Conversely, the activator metabolites themselves are more likely to be essential components, and their activation levels surpass those of non-essential activators. In summary, our study unveils the widespread importance of enzymatic activators, and suggests that feed-forward activation of conditional metabolic pathways through essential metabolites mediates metabolic plasticity.\nFeedback between stochastic gene networks and population dynamics enables cellular decision making\nAuthors: Piho, P.; Thomas, P.\nScore: 10.2, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568034\nPhenotypic selection is the process where genetically identical cells are subject to different re-productive abilities due to cellular noise. Such noise arises from fluctuations in the rates by which proteins are synthesised inside living cells and is hypothesised to play a crucial role in how cells make decisions and respond to stress or drugs. Nevertheless, the state-of-the-art modelling tools for quantifying cellular noise, such as the Gillespie algorithm and the chemical master equation, simply ignore selection. Here, we propose a general stochastic agent-based model for growing populations where the feedback of gene expression dynamics and cell cycle progression directly contribute to cell division dynamics. We devise a finite state projection method as a quantitative analysis tool and inference method for single-cell data obtained from mother machines and lineage trees. We use the theory to quantify selection in multi-stable gene expression networks and elucidate that the interplay between gene expression and phenotypic selection enables robust decision-making essential for developmental lineage decisions. Using live-cell data, we demonstrate that combining theory and inference provides quantitative insights into bet-hedging-like response to DNA damage, and adaption during antibiotic tolerance in Escherichia coli.\nSingle-cell imaging of protein dynamics of paralogs reveals mechanisms of gene retention\nAuthors: Dandage, R.; Papkov, M.; Greco, B. M.; Fishman, D.; Friesen, H.; Wang, K.; Styles, E.; Kraus, O.; Grys, B.; Boone, C.; Andrews, B. J.; Parts, L.; Kuzmin, E.\nScore: 8.8, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568466\nGene duplication is common across the tree of life, including yeast and humans, and contributes to genomic robustness. In this study, we examined changes in the subcellular localization and abundance of proteins in response to the deletion of their paralogs originating from the whole-genome duplication event, which is a largely unexplored mechanism of functional divergence. We performed a systematic single-cell imaging analysis of protein dynamics and screened subcellular redistribution of proteins, capturing their localization and abundance changes, providing insight into forces determining paralog retention. Paralogs showed dependency, whereby proteins required their paralog to maintain their native abundance or localization, more often than compensation. Network feature analysis suggested the importance of functional redundancy and rewiring of protein and genetic interactions underlying redistribution response of paralogs. Translation of non-canonical protein isoform emerged as a novel compensatory mechanism. This study provides new insights into paralog retention and evolutionary forces that shape genomes.\nVisual interpretability of image-based classification models by generative latent space disentanglement applied to in vitro fertilization\nAuthors: Rotem, O.; Schwartz, T.; Maor, R.; Tauber, Y.; Tsarfati Shapiro, M.; Meseguer, M.; Gilboa, D.; Seidman, D. S.; Zaritsky, A.\nScore: 8.8, Published: 2023-11-17 DOI: 10.1101/2023.11.15.566968\nThe success of deep learning in identifying complex patterns exceeding human intuition comes at the cost of interpretability. Non-linear entanglement of image features makes deep learning a \"black box\" lacking human meaningful explanations for the models decision. We present DISCOVER, a generative model designed to discover the underlying visual properties driving image-based classification models. DISCOVER learns disentangled latent representations, where each latent feature encodes a unique classification-driving visual property. This design enables \"human-in-the-loop\" interpretation by generating disentangled exaggerated counterfactual explanations. We apply DISCOVER to interpret classification of in-vitro fertilization embryo morphology quality. We quantitatively and systematically confirm the interpretation of known embryo properties, discover properties without previous explicit measurements, and quantitatively determine and empirically verify the classification decision of specific embryo instances. We show that DISCOVER provides human-interpretable understanding of \"black-box\" classification models, proposes hypotheses to decipher underlying biomedical mechanisms, and provides transparency for the classification of individual predictions.\nSpatial chromosome organization and adaptation of the radiation-resistant extremophile Deinococcus radiodurans\nAuthors: Qiu, Q.-T.; Zhang, C.-Y.; Gao, Z.-P.; Ma, B.-G.\nScore: 2.0, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567671\nRadiation-resistant Deinococcus radiodurans is an extremophilic microorganism capable of withstanding high levels of ionizing radiation and chemical mutagens. It possesses remarkable DNA repair capability and serves as a model organism for studying stress resistance mechanism. However, our understanding on the spatial chromosome organization of this species remains limited. In this study, we employed chromosome conformation capture (3C) technology to determine the 3D genome structure of D. radiodurans and to further investigate the changes of chromosome conformation induced by ultraviolet (UV) irradiation. We observed that UV irradiation reduced short-range chromosome interactions, and smaller chromosomal interaction domains (CIDs) merged to form larger CIDs. Integrating transcriptomic data analysis, we found that the majority of upregulated differentially expressed genes were significantly enriched near specific CID boundaries. Specially, we comprehensively elucidated that the nucleoid-associated protein Dr_ebfC may serve as a global regulator for gene expression by altering chromosome structure, thereby influencing the physiological state of the bacterium. Overall, our study revealed the chromosome conformations of D. radiodurans under different conditions, and offered valuable insights into the molecular responses of this extremophile to environmental stresses.\nSolving the time-dependent protein distributions for autoregulated bursty gene expression using spectral decomposition. Part I: Conventional models\nAuthors: Wu, B.; James, H.; Grima, R.; Jia, C.\nScore: 1.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568174\nIn this study, we obtain an exact time-dependent solution of the chemical master equation (CME) of an extension of the two-state telegraph model describing bursty or non-bursty protein expression in the presence of positive or negative autoregulation. Using the method of spectral decomposition, we show that the eigenfunctions of the generating function solution of the CME are Heun functions, while the eigenvalues can be determined by solving a continued fraction equation. Our solution generalizes and corrects a previous time-dependent solution for the CME of a gene circuit describing non-bursty protein expression in the presence of negative autoregulation [\"Exact time-dependent solutions for a self-regulating gene.\" Phys. Rev. E 83: 062902 (2011)]. In particular, we clarify that the eigenvalues are generally not real as previously claimed. We also investigate the relationship between different types of dynamic behavior and the type of feedback, the protein burst size, and the gene switching rate.\nSiderophore Piracy Promotes Dynamical Coexistence in Microbial Community\nAuthors: Shao, J.; Li, Y.; Lu, J.; Gu, S.; Li, Z.\nScore: 1.6, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568182\nThe game between cooperation and cheating in the microbial world is captivating. Microbes that acquire essential iron through the secretion of siderophores face the public good dilemma. In this game of iron, the specific recognition of different types of siderophores by corresponding receptors adds more complexity. Despite the prevalence of siderophore synthesis and piracy among microbes, the ecological consequences of siderophore-mediated interactions remain largely underexplored. This study introduced the framework of resource partition model for siderophore-mediated interactions, revealing conditions for stable coexistence in dual-species scenarios and uncovering surprising oscillatory and chaotic coexistence in systems with three or more species. Intriguingly, analytical analysis and simulations highlight the how cheating promotes biodiversity, emphasizing the non-monotonic changes in biodiversity as the level of siderophore piracy increases in producers. This work sheds light on the complex dynamics introduced by resource partition in microbial communities, elucidating the intricate balance between cooperation and cheating. HighlightsO_LIResource partition introduce complex dynamics C_LIO_LICheating play a main force in promoting coexistence C_LIO_LIIncreased proportion of cheating prefers dynamical behaviors C_LI\n",
  "wordCount" : "2074",
  "inLanguage": "en",
  "datePublished": "2023-11-26T10:37:33Z",
  "dateModified": "2023-11-26T10:37:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 26, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568149">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568149" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568149">
        <p class="paperTitle">One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568149" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568149" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.</p>
        <p class="info">Score: 25.8, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568149' target='https://doi.org/10.1101/2023.11.21.568149'> 10.1101/2023.11.21.568149</a></p>
        <p class="abstract">As a key mechanism for cellular signal transduction, protein phosphorylation plays critical roles in myriad cellular processes. This modification, however, is highly dynamic and occurs at sub-stoichiometric levels. Mass spectrometry is an essential tool for studying this modification on a global scale; however, the technologys impact suffers from four main limitations: need for site localization, dynamic range, reproducibility, and throughput. Here we describe the use of a novel mass spectrometer (Orbitrap Astral) coupled with data-independent acquisition (DIA) to achieve detection of close to 40,000 unique phosphorylation sites within one hour of analysis. We applied this approach to generate a phosphoproteome atlas of the mouse. Altogether, we detected 81,120 unique phosphorylation sites within 12 hours of measurement. With this unique dataset, we examine the sequence and structural context of protein phosphorylation. Finally, we highlight the discovery potential of this resource with multiple examples of novel phosphorylation events relevant to mitochondrial and brain biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568418">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568418" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568418">
        <p class="paperTitle">Systematic optimization of automated phosphopeptide enrichment for high-sensitivity phosphoproteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568418" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568418" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bortel, P.; Piga, I.; Koenig, C.; Gerner, C.; Martinez del Val, A.; Olsen, J. V.</p>
        <p class="info">Score: 23.6, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568418' target='https://doi.org/10.1101/2023.11.23.568418'> 10.1101/2023.11.23.568418</a></p>
        <p class="abstract">Improving coverage, robustness and sensitivity is crucial for routine phosphoproteomics analysis by single-shot liquid chromatography tandem mass spectrometry (LC-MS/MS) runs from minimal peptide inputs. Here, we systematically optimized key experimental parameters for automated on-beads phosphoproteomics sample preparation with focus on low input samples. Assessing the number of identified phosphopeptides, enrichment efficiency, site localization scores and relative enrichment of multiply-phosphorylated peptides pinpointed critical variables influencing the resulting phosphoproteome. Optimizing glycolic acid concentration in the loading buffer, percentage of ammonium hydroxide in the elution buffer, peptide-to-beads ratio, binding time, sample and loading buffer volumes, allowed us to confidently identify &gt;16,000 phosphopeptides in half-an-hour LC-MS/MS on an Orbitrap Exploris 480 using 30 ug of peptides as starting material. Furthermore, we evaluated how sequential enrichment can boost phosphoproteome coverage and showed that pooling fractions into a single LC-MS/MS analysis increased the depth. We also present an alternative phosphopeptide enrichment strategy based on stepwise addition of beads thereby boosting phosphoproteome coverage by 20%. Finally, we applied our optimized strategy to evaluate phosphoproteome depth with the Orbitrap Astral MS using a cell dilution series and were able to identify &gt;32,000 phosphopeptides from 0.5 million HeLa cells in half-an-hour LC-MS/MS using narrow-window data-independent acquisition (nDIA).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2020.10.23.351742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2020.10.23.351742" aria-expanded="false" aria-controls="flush-collapse10.1101/2020.10.23.351742">
        <p class="paperTitle">Synthetic model ecosystem of 12 cryopreservable microbial species allowing for a noninvasive approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2020.10.23.351742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2020.10.23.351742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hosoda, K.; Seno, S.; Murakami, N.; Matsuda, H.; Osada, Y.; Kamiura, R.; Kondoh, M.</p>
        <p class="info">Score: 39.8, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2020.10.23.351742' target='https://doi.org/10.1101/2020.10.23.351742'> 10.1101/2020.10.23.351742</a></p>
        <p class="abstract">Simultaneous understanding of both individual and ecosystem dynamics is crucial in an era marked by the degradation of ecosystem services. Herein, we present a high-throughput synthetic microcosm system comprising 12 functionally and phylogenetically diverse microbial species. These species are axenically culturable, cryopreservable, and can be measured noninvasively via microscopy, aided by machine learning. This system includes prokaryotic and eukaryotic producers and decomposers, and eukaryotic consumers to ensure functional redundancy. Our model system displayed both positive and negative interspecific interactions and higher-order interactions that surpassed the scope of any two-species interaction. Although complete species coexistence was not our primary objective, we identified several conditions under which at least one species from the producers, consumers, and decomposers groups, and one functionally redundant species, persisted for over six months. These conditions set the stage for detailed investigations in the future. Given its designability and experimental replicability, our model ecosystem offers a promising platform for deeper insights into both individual and ecosystem dynamics, including evolution and species interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567372">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567372" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567372">
        <p class="paperTitle">An enzyme activation network provides evidence for extensive regulatory crosstalk between metabolic pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567372" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567372" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alam, M. T.; Al Zubaidi, S. M.; Nasar, M. I.; Ralser, M.; Notebaart, R. A.</p>
        <p class="info">Score: 12.7, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567372' target='https://doi.org/10.1101/2023.11.16.567372'> 10.1101/2023.11.16.567372</a></p>
        <p class="abstract">Enzyme activation by cellular metabolites plays a pivotal role in regulating metabolic processes. Nevertheless, our comprehension of such activation events on a global network scale remains incomplete. In this study, we conducted a comprehensive investigation into the optimization of cell-intrinsic activation interactions within Saccharomyces cerevisiae. To achieve this, we integrated a genome-scale metabolic model with enzyme kinetic data sourced from the BRENDA database. Our objective was to map the distribution of enzyme activators throughout the cellular network. Our findings indicate that virtually all biochemical pathways encompass enzyme activators, frequently originating from disparate pathways, thus revealing extensive regulatory crosstalk between metabolic pathways. Indeed, activators have short pathway lengths, indicating they are activated quickly upon nutrient shifts, and in most instances, these activators target key enzymatic reactions to facilitate downstream metabolic processes. Interestingly, non-essential enzymes exhibit a significantly higher degree of activation compared to their essential counterparts. This observation suggests that cells employ enzyme activators to finely regulate secondary metabolic pathways that are only required under specific conditions. Conversely, the activator metabolites themselves are more likely to be essential components, and their activation levels surpass those of non-essential activators. In summary, our study unveils the widespread importance of enzymatic activators, and suggests that feed-forward activation of conditional metabolic pathways through essential metabolites mediates metabolic plasticity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568034">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568034" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568034">
        <p class="paperTitle">Feedback between stochastic gene networks and population dynamics enables cellular decision making</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568034" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568034" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Piho, P.; Thomas, P.</p>
        <p class="info">Score: 10.2, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568034' target='https://doi.org/10.1101/2023.11.21.568034'> 10.1101/2023.11.21.568034</a></p>
        <p class="abstract">Phenotypic selection is the process where genetically identical cells are subject to different re-productive abilities due to cellular noise. Such noise arises from fluctuations in the rates by which proteins are synthesised inside living cells and is hypothesised to play a crucial role in how cells make decisions and respond to stress or drugs. Nevertheless, the state-of-the-art modelling tools for quantifying cellular noise, such as the Gillespie algorithm and the chemical master equation, simply ignore selection. Here, we propose a general stochastic agent-based model for growing populations where the feedback of gene expression dynamics and cell cycle progression directly contribute to cell division dynamics. We devise a finite state projection method as a quantitative analysis tool and inference method for single-cell data obtained from mother machines and lineage trees. We use the theory to quantify selection in multi-stable gene expression networks and elucidate that the interplay between gene expression and phenotypic selection enables robust decision-making essential for developmental lineage decisions. Using live-cell data, we demonstrate that combining theory and inference provides quantitative insights into bet-hedging-like response to DNA damage, and adaption during antibiotic tolerance in Escherichia coli.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568466">
        <p class="paperTitle">Single-cell imaging of protein dynamics of paralogs reveals mechanisms of gene retention</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dandage, R.; Papkov, M.; Greco, B. M.; Fishman, D.; Friesen, H.; Wang, K.; Styles, E.; Kraus, O.; Grys, B.; Boone, C.; Andrews, B. J.; Parts, L.; Kuzmin, E.</p>
        <p class="info">Score: 8.8, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568466' target='https://doi.org/10.1101/2023.11.23.568466'> 10.1101/2023.11.23.568466</a></p>
        <p class="abstract">Gene duplication is common across the tree of life, including yeast and humans, and contributes to genomic robustness. In this study, we examined changes in the subcellular localization and abundance of proteins in response to the deletion of their paralogs originating from the whole-genome duplication event, which is a largely unexplored mechanism of functional divergence. We performed a systematic single-cell imaging analysis of protein dynamics and screened subcellular redistribution of proteins, capturing their localization and abundance changes, providing insight into forces determining paralog retention. Paralogs showed dependency, whereby proteins required their paralog to maintain their native abundance or localization, more often than compensation. Network feature analysis suggested the importance of functional redundancy and rewiring of protein and genetic interactions underlying redistribution response of paralogs. Translation of non-canonical protein isoform emerged as a novel compensatory mechanism. This study provides new insights into paralog retention and evolutionary forces that shape genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.566968">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.566968" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.566968">
        <p class="paperTitle">Visual interpretability of image-based classification models by generative latent space disentanglement applied to in vitro fertilization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.566968" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.566968" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rotem, O.; Schwartz, T.; Maor, R.; Tauber, Y.; Tsarfati Shapiro, M.; Meseguer, M.; Gilboa, D.; Seidman, D. S.; Zaritsky, A.</p>
        <p class="info">Score: 8.8, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.566968' target='https://doi.org/10.1101/2023.11.15.566968'> 10.1101/2023.11.15.566968</a></p>
        <p class="abstract">The success of deep learning in identifying complex patterns exceeding human intuition comes at the cost of interpretability. Non-linear entanglement of image features makes deep learning a &#34;black box&#34; lacking human meaningful explanations for the models decision. We present DISCOVER, a generative model designed to discover the underlying visual properties driving image-based classification models. DISCOVER learns disentangled latent representations, where each latent feature encodes a unique classification-driving visual property. This design enables &#34;human-in-the-loop&#34; interpretation by generating disentangled exaggerated counterfactual explanations. We apply DISCOVER to interpret classification of in-vitro fertilization embryo morphology quality. We quantitatively and systematically confirm the interpretation of known embryo properties, discover properties without previous explicit measurements, and quantitatively determine and empirically verify the classification decision of specific embryo instances. We show that DISCOVER provides human-interpretable understanding of &#34;black-box&#34; classification models, proposes hypotheses to decipher underlying biomedical mechanisms, and provides transparency for the classification of individual predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567671">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567671" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567671">
        <p class="paperTitle">Spatial chromosome organization and adaptation of the radiation-resistant extremophile Deinococcus radiodurans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567671" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567671" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Qiu, Q.-T.; Zhang, C.-Y.; Gao, Z.-P.; Ma, B.-G.</p>
        <p class="info">Score: 2.0, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567671' target='https://doi.org/10.1101/2023.11.19.567671'> 10.1101/2023.11.19.567671</a></p>
        <p class="abstract">Radiation-resistant Deinococcus radiodurans is an extremophilic microorganism capable of withstanding high levels of ionizing radiation and chemical mutagens. It possesses remarkable DNA repair capability and serves as a model organism for studying stress resistance mechanism. However, our understanding on the spatial chromosome organization of this species remains limited. In this study, we employed chromosome conformation capture (3C) technology to determine the 3D genome structure of D. radiodurans and to further investigate the changes of chromosome conformation induced by ultraviolet (UV) irradiation. We observed that UV irradiation reduced short-range chromosome interactions, and smaller chromosomal interaction domains (CIDs) merged to form larger CIDs. Integrating transcriptomic data analysis, we found that the majority of upregulated differentially expressed genes were significantly enriched near specific CID boundaries. Specially, we comprehensively elucidated that the nucleoid-associated protein Dr_ebfC may serve as a global regulator for gene expression by altering chromosome structure, thereby influencing the physiological state of the bacterium. Overall, our study revealed the chromosome conformations of D. radiodurans under different conditions, and offered valuable insights into the molecular responses of this extremophile to environmental stresses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568174">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568174" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568174">
        <p class="paperTitle">Solving the time-dependent protein distributions for autoregulated bursty gene expression using spectral decomposition. Part I: Conventional models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568174" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568174" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, B.; James, H.; Grima, R.; Jia, C.</p>
        <p class="info">Score: 1.6, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568174' target='https://doi.org/10.1101/2023.11.21.568174'> 10.1101/2023.11.21.568174</a></p>
        <p class="abstract">In this study, we obtain an exact time-dependent solution of the chemical master equation (CME) of an extension of the two-state telegraph model describing bursty or non-bursty protein expression in the presence of positive or negative autoregulation. Using the method of spectral decomposition, we show that the eigenfunctions of the generating function solution of the CME are Heun functions, while the eigenvalues can be determined by solving a continued fraction equation. Our solution generalizes and corrects a previous time-dependent solution for the CME of a gene circuit describing non-bursty protein expression in the presence of negative autoregulation [&#34;Exact time-dependent solutions for a self-regulating gene.&#34; Phys. Rev. E 83: 062902 (2011)]. In particular, we clarify that the eigenvalues are generally not real as previously claimed. We also investigate the relationship between different types of dynamic behavior and the type of feedback, the protein burst size, and the gene switching rate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568182">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568182" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568182">
        <p class="paperTitle">Siderophore Piracy Promotes Dynamical Coexistence in Microbial Community</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568182" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568182" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shao, J.; Li, Y.; Lu, J.; Gu, S.; Li, Z.</p>
        <p class="info">Score: 1.6, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568182' target='https://doi.org/10.1101/2023.11.21.568182'> 10.1101/2023.11.21.568182</a></p>
        <p class="abstract">The game between cooperation and cheating in the microbial world is captivating. Microbes that acquire essential iron through the secretion of siderophores face the public good dilemma. In this game of iron, the specific recognition of different types of siderophores by corresponding receptors adds more complexity. Despite the prevalence of siderophore synthesis and piracy among microbes, the ecological consequences of siderophore-mediated interactions remain largely underexplored. This study introduced the framework of resource partition model for siderophore-mediated interactions, revealing conditions for stable coexistence in dual-species scenarios and uncovering surprising oscillatory and chaotic coexistence in systems with three or more species. Intriguingly, analytical analysis and simulations highlight the how  cheating promotes biodiversity, emphasizing the non-monotonic changes in biodiversity as the level of siderophore piracy increases in producers. This work sheds light on the complex dynamics introduced by resource partition in microbial communities, elucidating the intricate balance between cooperation and cheating.

HighlightsO_LIResource partition introduce complex dynamics
C_LIO_LICheating play a main force in promoting coexistence
C_LIO_LIIncreased proportion of cheating prefers dynamical behaviors
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
