<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Digitize your Biology! Modeling multicellular systems through interpretable cell behavior
Authors: Johnson, J. A. I.; Stein-O&#39;Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Digitize your Biology! Modeling multicellular systems through interpretable cell behavior
Authors: Johnson, J. A. I.; Stein-O&#39;Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-04T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-10-04T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Digitize your Biology! Modeling multicellular systems through interpretable cell behavior
Authors: Johnson, J. A. I.; Stein-O&#39;Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Digitize your Biology! Modeling multicellular systems through interpretable cell behavior\nAuthors: Johnson, J. A. I.; Stein-O\u0026#39;Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N.",
  "keywords": [
    
  ],
  "articleBody": " Digitize your Biology! Modeling multicellular systems through interpretable cell behavior\nAuthors: Johnson, J. A. I.; Stein-O'Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N.; Zheng, L.; Zimmerman, J. W.; Jaffee, E. M.; Chang, Y. H.; Coussens, L.; Gray, J.; Heiser, L. M.; Fertig, E. J.; Macklin, P.\nScore: 26.1, Published: 2023-09-24 DOI: 10.1101/2023.09.17.557982\nCells are fundamental units of life. Recent technical advances have revolutionized our ability to quantify the state and identity of individual cells, and intercellular regulatory programs. However, these static measurements alone are limited in their ability to predict the complex collective behaviors that emerge from populations of many interacting cells over time. Mathematical models have a proven record of successfully predicting the behaviors of dynamic biological systems, e.g., therapeutic responses in cancer. Simulations from these models enable in silico visualization, examination, and refinement of biological models and can be used to generate new hypotheses about cells and their collective behavior. Agent-based modeling is particularly well-suited to studying communities of interacting cells, as it is intuitive to map a single cell to a single agent. Thus, we have developed a conceptual framing (with a reference implementation in the widely-used PhysiCell agent-based modeling framework) that can be initialized directly from single cell and spatial transcriptomic data, and that can be easily populated with interactive rules. Because the expert mathematical and computational knowledge required to build agent-based models has limited their widespread adoption in the biomedical research community, we engineered this framework to specify complex cellular responses to signals (or stimuli) using a single line of human-readable text. This plain language text encodes cellular phenotypes and regulatory mechanisms from high throughput data and published literature, using a novel concept of hypothesis grammar. We motivate and fully describe this grammar and its philosophy, and then present a series of five example reference models of tumor growth and response to immunotherapy. Biologically, these examples demonstrate how mathematical models can predict from single cell and spatial transcriptomic data the cellular phenotypes responsible for tumor cell invasion and the simulation of immunotherapy treatment to overcome tumor cell growth. Computationally, these examples are designed to demonstrate how this conceptual framing and software implementation empower interdisciplinary teams to build an agent-based model of their experimental system, leveraging prior biological knowledge alone or in combination with information from spatial multiomics technologies. Altogether, this approach provides an interface to bridge biological, clinical, and systems biology researchers for mathematical modeling of biological systems at scale, allowing the community to extrapolate from single-cell characterization to emergent multicellular behavior.\nModeling Cardiomyocyte Signaling and Metabolism Predicts Genotype to Phenotype Mechanisms in Hypertrophic Cardiomyopathy\nAuthors: Khalilimeybodi, A.; Rangamani, P.\nScore: 3.8, Published: 2023-09-26 DOI: 10.1101/2023.09.25.559356\nFamilial hypertrophic cardiomyopathy (HCM) is a significant precursor of heart failure and sudden cardiac death, primarily caused by mutations in sarcomeric and structural proteins. Despite the extensive research on the HCM genotype, the complex, context-specific nature of many signaling and metabolic pathways linking the HCM genotype to phenotype has hindered therapeutic advancements for patients. To address these challenges, here, we have developed a computational systems biology model of HCM at the cardiomyocyte level. Utilizing a stochastic logic-based ODE method, we integrate subcellular systems in cardiomyocytes that jointly modulate HCM genotype to phenotype, including cardiac signaling, metabolic, and gene regulatory networks, as well as posttranslational modifications linking these networks. After validating against existing experimental data, the model predicts significant changes in cardiomyocyte metabolic functions such as ATP synthase deficiency and a transition from fatty acids to carbohydrate metabolism in HCM. The model indicated major shifts in glutamine-related metabolism and increased apoptosis after HCM-induced ATP synthase deficiency. Aligned with prior experimental studies, we predicted that the transcription factors STAT, SRF, GATA4, TP53, and FoxO are the key regulators of cardiomyocyte hypertrophy and apoptosis in HCM. Using the model, we identified shared (e.g., activation of PGC1 by AMPK, and FHL1 by titin) and context-specific mechanisms (e.g., regulation of Ca2+ sensitivity by titin in HCM patients) that could control genotype to phenotype transition in HCM across different species or mutations. We also predicted potential combinatory drug targets for HCM (e.g., mavacamten paired with ROS inhibitors) preventing or reversing HCM phenotype (i.e., hypertrophic growth, apoptosis, and metabolic remodeling) in cardiomyocytes. This study provides new insights into mechanisms linking genotype to phenotype in familial hypertrophic cardiomyopathy and offers a framework for assessing new treatments and exploring variations in HCM experimental models.\nAPOLLO: A genome-scale metabolic reconstruction resource of 247,092 diverse human microbes spanning multiple continents, age groups, and body sites\nAuthors: Heinken, A.; Hulshof, T.; Nap, B.; Martinelli, F.; Basile, A.; O'Brolchain, A.; O'Sullivan, N. F.; Gallagher, C.; Magee, E.; McDonagh, F.; Lalor, I.; Bergin, M.; Evans, P.; Daly, R.; Farrell, R.; Delaney, R. M.; Hill, S.; McAuliffe, S. R.; Kilgannon, T.; Fleming, R. M.; Thinnes, C. C.; Thiele, I.\nScore: 2.8, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560573\nComputational modelling of microbiome metabolism has proved instrumental in catalysing our understanding of diet-host-microbiome-disease interactions through the interrogation of mechanistic, strain- and molecule-resolved metabolic models. We present APOLLO, a resource of 247,092 human microbial genome-scale metabolic reconstructions spanning 19 phyla and accounting for microbial genomes from 34 countries, all age groups, and five body sites. We explored the metabolic potential of the reconstructed strains and developed a machine learning classifier able to predict with high accuracy the taxonomic strain assignments. We also built 14,451 sample-specific microbial community models, which could be stratified by body site, age, and disease states. Finally, we predicted faecal metabolites enriched or depleted in gut microbiomes of people with Crohn's disease, Parkinson's disease, and undernourished children. APOLLO is compatible with the human whole-body models, and thus, provides unprecedented opportunities for systems-level modelling of personalised host-microbiome co-metabolism. APOLLO will be freely available under https://www.vmh.life/.\nWidespread transcriptional regulation from within transcribed regions in plants\nAuthors: Voichek, Y.; Hristova, G.; Molla-Morales, A.; Weigel, D.; Nordborg, M.\nScore: 110.7, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557872\nAnimals and plants have evolved separately for over 1.5 billion years and independently invented multicellularity, suggesting a possible divergence in their mode of transcription regulation. Here, we set out to elucidate fundamental features of transcription regulatory sequences in plants. Using a massively parallel reporter assay in four species, we show that sequences downstream of the transcription start site (TSS) play a major role in controlling transcription. Swapping regulatory sequences from one side of the TSS to the other yields different outcomes, unlike animal enhancers which act independently of their position. A GATC-containing DNA motif, positioned downstream of the TSS, was sufficient to enhance gene expression in a dose-dependent manner. Its effect on gene expression was tissue-dependent and conserved across vascular plants. These results identify a unique characteristic of transcriptional regulation in plants, and suggest fundamental differences in gene regulation might exist between plants and animals.\nMulti-modal transcriptomic analysis unravels enrichment of hybrid epithelial/mesenchymal state and enhanced phenotypic heterogeneity in basal breast cancer\nAuthors: Sahoo, S.; Ramu, S.; Nair, M. G.; Pillai, M.; Perez San Juan, B.; Milioli, H. Z.; Mandal, S.; Naidu, C. M.; Mavatkar, A. D.; Subramaniam, H.; Neogi, A. G.; Chaffer, C. L.; Prabhu, J. S.; Somarelli, J. A.; Jolly, M. K.\nScore: 2.0, Published: 2023-10-02 DOI: 10.1101/2023.09.30.558960\nIntra-tumoral phenotypic heterogeneity promotes tumor relapse and therapeutic resistance and remains an unsolved clinical challenge. It manifests along multiple phenotypic axes and decoding the interconnections among these different axes is crucial to understand its molecular origins and to develop novel therapeutic strategies to control it. Here, we use multi-modal transcriptomic data analysis - bulk, single-cell and spatial transcriptomics - from breast cancer cell lines and primary tumor samples, to identify associations between epithelial-mesenchymal transition (EMT) and luminal-basal plasticity - two key processes that enable heterogeneity. We show that luminal breast cancer strongly associates with an epithelial cell state, but basal breast cancer is associated with hybrid epithelial/mesenchymal phenotype(s) and higher phenotypic heterogeneity. These patterns were inherent in methylation profiles, suggesting an epigenetic crosstalk between EMT and lineage plasticity in breast cancer. Mathematical modelling of core underlying gene regulatory networks representative of the crosstalk between the luminal-basal and epithelial-mesenchymal axes recapitulate and thus elucidate mechanistic underpinnings of the observed associations from transcriptomic data. Our systems-based approach integrating multi-modal data analysis with mechanism-based modeling offers a predictive framework to characterize intra-tumor heterogeneity and to identify possible interventions to restrict it.\nPan-viral ORFs discovery using Massively Parallel Ribosome Profiling\nAuthors: Weingarten-Gabbay, S.; Bauer, M. R.; Stanton, A. C.; Klaeger, S.; Verzani, E. K.; Lopez, D.; Clauser, K. R.; Carr, S. A.; Abelin, J. G.; Rice, C. M.; Sabeti, P. C.\nScore: 1.8, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559641\nUnveiling the complete proteome of viruses is crucial to our understanding of the viral life cycle and interaction with the host. We developed Massively Parallel Ribosome Profiling (MPRP) to experimentally determine open reading frames (ORFs) in 20,170 designed oligonucleotides across 679 human-associated viral genomes. We identified 5,381 ORFs, including 4,208 non-canonical ORFs, and show successful detection of both annotated coding sequences (CDSs) and reported non-canonical ORFs. By examining immunopeptidome datasets of infected cells, we found class I human leukocyte antigen (HLA-I) peptides originating from non-canonical ORFs identified through MPRP. By inspecting ribosome occupancies on the 5UTR and CDS regions of annotated viral genes, we identified hundreds of upstream ORFs (uORFs) that negatively regulate the synthesis of canonical viral proteins. The unprecedented source of viral ORFs across a wide range of viral families, including highly pathogenic viruses, expands the repertoire of vaccine targets and exposes new cis-regulatory sequences in viral genomes.\nCardiac Specific Overexpression of Adenylyl Cyclase 8 Reprograms the Mouse Sinoatrial Node Transcriptome\nAuthors: Qu, J.; Tarasov, K. V.; Chakir, K.; Lakatta, E. G.\nScore: 1.1, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559627\nA coupled-clock system intrinsic to sinoatrial node (SAN) pacemaker cells that regulates the rate and rhythm of spontaneous action potential firing, is activated by Ca2+/calmodulin-stimulated Adenylyl Cyclase (AC) types 8 and 1. Our previous work in mice with cardiac specific overexpression of human AC8 gene (TGAC8) discovered that compared to its wild-type (WT) littermates, the heart rate (HR) of TGAC8 is elevated (by about 30%, 24 hours a day, 7days a week), and that the TGAC8 heart rhythm is markedly coherent, i.e., the HR Variability (HRV) lacks complexity, similar to that associated with aging or cardiac pathology. Reprogramming of molecular mechanisms, particularly SAN transcriptomic regulation that underlies the remarkable chronic shift in HR and HRV, however, has not been delineated. We conducted deep RNA sequencing (RNA-seq) in TGAC8 and WT SANs, using Mm10plus with human ADCY8 DNA sequence as the reference genome. Utilizing multiple bioinformatic techniques, we not only profiled the expression of marker genes related to SAN functions and AC-cAMP-PKA signaling, but also discovered negatively enriched hub pathways that differed in TGAC8 vs. WT, the top three being OXPHOS, ribosome, and cardiac muscle contraction. In contrast, signaling pathways related to inositol phosphate and its metabolism were positively enriched in TGAC8. Further, we identified two transcription regulators, KDM5A and PPARGC1A, that mediate effects of TGAC8 on ribosome and mitochondria. In summary, reprogrammed transcriptional regulation increases the HR of TGAC8 at the cost of impairment of other SAN cell functions, i.e., altered ribosome and inositol phosphate signaling and its crosstalk with mitochondria. Because these cell signaling alterations are associated with cardiac aging and age-associated CVDs, the TGAC8 mouse appears to be an ideal model in which to probe for potential therapeutic targets for cardiac aging and age-associated CVDs.\nHigh-volume, label-free imaging for quantifying single-cell dynamics in induced pluripotent stem cell colonies\nAuthors: Asmar, A. J.; Benson, Z.; Peskin, A. P.; Chalfoun, J.; Simon, M.; Halter, M.; Plant, A. L.\nScore: 1.0, Published: 2023-09-29 DOI: 10.1101/2023.09.29.558451\nTo facilitate the characterization of unlabeled induced pluripotent stem cells (iPSCs) during culture and expansion, we developed an AI pipeline for nuclear segmentation and mitosis detection from phase contrast images of individual cells within iPSC colonies. The analysis uses a 2D convolutional neural network (U-Net) plus a 3D U-Net applied on time lapse images to detect and segment nuclei, mitotic events, and daughter nuclei to enable tracking of large numbers of individual cells over long times in culture. The analysis uses fluorescence data to train models for segmenting nuclei in phase contrast images. The use of classical image processing routines to segment fluorescent nuclei precludes the need for manual annotation. We optimize and evaluate the accuracy of automated annotation to assure the reliability of the training. The model is generalizable in that it performs well on different datasets with an average F1 score of 0.94, on cells at different densities, and on cells from different pluripotent cell lines. The method allows us to assess, in a non-invasive manner, rates of mitosis and cell division which serve as indicators of cell state and cell health. We assess these parameters in up to hundreds of thousands of cells in culture for over 20 hours, at different locations in the colonies, and as a function of excitation light exposure.\nInvestigating the dose-dependency of the midgut escape barrier using a mechanistic model of within-mosquito dengue virus population dynamics\nAuthors: Johnson, R. M.; Stopard, I. J.; Byrne, H. M.; Armstrong, P. M.; Brackney, D. E.; Lambert, B.\nScore: 1.0, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559904\nFlaviviruses are arthropod-borne (arbo)viruses which can emerge rapidly and cause explosive epidemics of severe disease. Some of the most epidemiologically important flaviviruses, including dengue virus (DENV), Zika virus (ZIKV) and yellow fever virus (YFV), are transmitted by Aedes mosquitoes, most notably Aedes aegypti and Aedes albopictus. After a mosquito blood feeds on an infected host, virus enters the midgut and infects the midgut epithelium. The virus must then overcome a series of barriers before reaching the mosquito saliva and being transmitted to a new host. The virus must escape from the midgut (known as the midgut escape barrier; MEB), which is thought to be mediated by transient changes in the permeability of the midgut-surrounding basal lamina layer (BL) following blood feeding. Here, we present a mathematical model of the within-mosquito population dynamics of flaviviruses that includes the interaction of the midgut and BL which can account for the MEB. Our results indicate a dose-dependency of midgut establishment of infection as well as rate of escape from the midgut: collectively, these suggest that the extrinsic incubation period (EIP) -- the time taken for DENV virus to be transmissible after infection -- is shortened when mosquitoes imbibe more virus. Additionally, our experimental data indicates that multiple blood feeding events, which more closely mimic mosquito-feeding behavior in the wild, can hasten the course of infections, and our model predicts that this effect is sensitive to the amount of virus imbibed. Our model indicates that mutations to the virus which impact its replication rate in the midgut could lead to even shorter EIPs when double-feeding occurs. Mechanistic models of within-vector viral infection dynamics provide a quantitative understanding of infection dynamics and could be used to evaluate novel interventions that target the mosquito stages of the infection.\nDiscovering paracrine regulators of cellular composition from spatial transcriptomics data using SPER\nAuthors: Haber, A. L.; Zhao, T.\nScore: 1.0, Published: 2023-09-28 DOI: 10.1101/2023.09.27.559863\nA wide range of pathologies and chronic diseases are characterized by transformations in cell type composition within a tissue. Prominent examples are remodeling of barrier epithelia in response to external threats, immune cell infiltration in chronic inflammatory conditions, or proliferation of fibroblasts and stromal cells causing fibrosis. An extensive literature implicates paracrine signals secreted by the local tissue niche in regulating its cell type composition, and therefore dysregulation of these factors is frequently capable of inducing pathological changes. Emerging multi-omics methods, particularly spatial transcriptomic profiling, provide the ability to simultaneously assay cell type composition and gene expression, with the potential to discover the transcripts most strongly associated with compositional shifts. However, no method yet exists to detect such signals. Here, we develop Spatial Paired Expression Ratio (SPER), a computational approach to evaluate the spatial dependence between transcripts and cell types in spatial transcriptomics data. We demonstrate the ability of SPER to accurately detect paracrine drivers of cellular abundance using simulated datasets. Using publicly available spatial transcriptomic data from the adult mouse motor cortex, we show that genes identified by SPER are strongly statistically enriched for both transcripts known to be extracellularly secreted, and those known to participate in paracrine receptor-ligand pairs, providing evidence of the ability to detect compositional regulatory signals. SPER recovered known and novel interactions, such as a surprising spatial association between the Wnt ligand Rpso3 and its cognate receptor Lgr5, expressed in a specific neural subset, and not previously known to regulate neural cell type composition in the motor cortex. SPER thus presents a general approach to discovering paracrine drivers of cellular compositional changes.\n",
  "wordCount" : "2810",
  "inLanguage": "en",
  "datePublished": "2023-10-04T10:39:33Z",
  "dateModified": "2023-10-04T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on October 4, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.557982">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.557982" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.557982">
        <p class="paperTitle">Digitize your Biology! Modeling multicellular systems through interpretable cell behavior</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.557982" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.557982" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, J. A. I.; Stein-O&#39;Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N.; Zheng, L.; Zimmerman, J. W.; Jaffee, E. M.; Chang, Y. H.; Coussens, L.; Gray, J.; Heiser, L. M.; Fertig, E. J.; Macklin, P.</p>
        <p class="info">Score: 26.1, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.557982' target='https://doi.org/10.1101/2023.09.17.557982'> 10.1101/2023.09.17.557982</a></p>
        <p class="abstract">Cells are fundamental units of life. Recent technical advances have revolutionized our ability to quantify the state and identity of individual cells, and intercellular regulatory programs. However, these static measurements alone are limited in their ability to predict the complex collective behaviors that emerge from populations of many interacting cells over time. Mathematical models have a proven record of successfully predicting the behaviors of dynamic biological systems, e.g., therapeutic responses in cancer. Simulations from these models enable in silico visualization, examination, and refinement of biological models and can be used to generate new hypotheses about cells and their collective behavior. Agent-based modeling is particularly well-suited to studying communities of interacting cells, as it is intuitive to map a single cell to a single agent. Thus, we have developed a conceptual framing (with a reference implementation in the widely-used PhysiCell agent-based modeling framework) that can be initialized directly from single cell and spatial transcriptomic data, and that can be easily populated with interactive rules. Because the expert mathematical and computational knowledge required to build agent-based models has limited their widespread adoption in the biomedical research community, we engineered this framework to specify complex cellular responses to signals (or stimuli) using a single line of human-readable text. This plain language text encodes cellular phenotypes and regulatory mechanisms from high throughput data and published literature, using a novel concept of hypothesis grammar. We motivate and fully describe this grammar and its philosophy, and then present a series of five example reference models of tumor growth and response to immunotherapy. Biologically, these examples demonstrate how mathematical models can predict from single cell and spatial transcriptomic data the cellular phenotypes responsible for tumor cell invasion and the simulation of immunotherapy treatment to overcome tumor cell growth. Computationally, these examples are designed to demonstrate how this conceptual framing and software implementation empower interdisciplinary teams to build an agent-based model of their experimental system, leveraging prior biological knowledge alone or in combination with information from spatial multiomics technologies. Altogether, this approach provides an interface to bridge biological, clinical, and systems biology researchers for mathematical modeling of biological systems at scale, allowing the community to extrapolate from single-cell characterization to emergent multicellular behavior.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559356">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559356" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559356">
        <p class="paperTitle">Modeling Cardiomyocyte Signaling and Metabolism Predicts Genotype to Phenotype Mechanisms in Hypertrophic Cardiomyopathy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559356" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559356" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khalilimeybodi, A.; Rangamani, P.</p>
        <p class="info">Score: 3.8, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559356' target='https://doi.org/10.1101/2023.09.25.559356'> 10.1101/2023.09.25.559356</a></p>
        <p class="abstract">Familial hypertrophic cardiomyopathy (HCM) is a significant precursor of heart failure and sudden cardiac death, primarily caused by mutations in sarcomeric and structural proteins. Despite the extensive research on the HCM genotype, the complex, context-specific nature of many signaling and metabolic pathways linking the HCM genotype to phenotype has hindered therapeutic advancements for patients. To address these challenges, here, we have developed a computational systems biology model of HCM at the cardiomyocyte level. Utilizing a stochastic logic-based ODE method, we integrate subcellular systems in cardiomyocytes that jointly modulate HCM genotype to phenotype, including cardiac signaling, metabolic, and gene regulatory networks, as well as posttranslational modifications linking these networks. After validating against existing experimental data, the model predicts significant changes in cardiomyocyte metabolic functions such as ATP synthase deficiency and a transition from fatty acids to carbohydrate metabolism in HCM. The model indicated major shifts in glutamine-related metabolism and increased apoptosis after HCM-induced ATP synthase deficiency. Aligned with prior experimental studies, we predicted that the transcription factors STAT, SRF, GATA4, TP53, and FoxO are the key regulators of cardiomyocyte hypertrophy and apoptosis in HCM. Using the model, we identified shared (e.g., activation of PGC1 by AMPK, and FHL1 by titin) and context-specific mechanisms (e.g., regulation of Ca2&#43; sensitivity by titin in HCM patients) that could control genotype to phenotype transition in HCM across different species or mutations. We also predicted potential combinatory drug targets for HCM (e.g., mavacamten paired with ROS inhibitors) preventing or reversing HCM phenotype (i.e., hypertrophic growth, apoptosis, and metabolic remodeling) in cardiomyocytes. This study provides new insights into mechanisms linking genotype to phenotype in familial hypertrophic cardiomyopathy and offers a framework for assessing new treatments and exploring variations in HCM experimental models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560573">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560573" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560573">
        <p class="paperTitle">APOLLO: A genome-scale metabolic reconstruction resource of 247,092 diverse human microbes spanning multiple continents, age groups, and body sites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560573" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560573" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heinken, A.; Hulshof, T.; Nap, B.; Martinelli, F.; Basile, A.; O&#39;Brolchain, A.; O&#39;Sullivan, N. F.; Gallagher, C.; Magee, E.; McDonagh, F.; Lalor, I.; Bergin, M.; Evans, P.; Daly, R.; Farrell, R.; Delaney, R. M.; Hill, S.; McAuliffe, S. R.; Kilgannon, T.; Fleming, R. M.; Thinnes, C. C.; Thiele, I.</p>
        <p class="info">Score: 2.8, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560573' target='https://doi.org/10.1101/2023.10.02.560573'> 10.1101/2023.10.02.560573</a></p>
        <p class="abstract">Computational modelling of microbiome metabolism has proved instrumental in catalysing our understanding of diet-host-microbiome-disease interactions through the interrogation of mechanistic, strain- and molecule-resolved metabolic models. We present APOLLO, a resource of 247,092 human microbial genome-scale metabolic reconstructions spanning 19 phyla and accounting for microbial genomes from 34 countries, all age groups, and five body sites. We explored the metabolic potential of the reconstructed strains and developed a machine learning classifier able to predict with high accuracy the taxonomic strain assignments. We also built 14,451 sample-specific microbial community models, which could be stratified by body site, age, and disease states. Finally, we predicted faecal metabolites enriched or depleted in gut microbiomes of people with Crohn&#39;s disease, Parkinson&#39;s disease, and undernourished children. APOLLO is compatible with the human whole-body models, and thus, provides unprecedented opportunities for systems-level modelling of personalised host-microbiome co-metabolism. APOLLO will be freely available under https://www.vmh.life/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557872">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557872" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557872">
        <p class="paperTitle">Widespread transcriptional regulation from within transcribed regions in plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557872" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557872" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Voichek, Y.; Hristova, G.; Molla-Morales, A.; Weigel, D.; Nordborg, M.</p>
        <p class="info">Score: 110.7, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557872' target='https://doi.org/10.1101/2023.09.15.557872'> 10.1101/2023.09.15.557872</a></p>
        <p class="abstract">Animals and plants have evolved separately for over 1.5 billion years and independently invented multicellularity, suggesting a possible divergence in their mode of transcription regulation. Here, we set out to elucidate fundamental features of transcription regulatory sequences in plants. Using a massively parallel reporter assay in four species, we show that sequences downstream of the transcription start site (TSS) play a major role in controlling transcription. Swapping regulatory sequences from one side of the TSS to the other yields different outcomes, unlike animal enhancers which act independently of their position. A GATC-containing DNA motif, positioned downstream of the TSS, was sufficient to enhance gene expression in a dose-dependent manner. Its effect on gene expression was tissue-dependent and conserved across vascular plants. These results identify a unique characteristic of transcriptional regulation in plants, and suggest fundamental differences in gene regulation might exist between plants and animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.558960">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.558960" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.558960">
        <p class="paperTitle">Multi-modal transcriptomic analysis unravels enrichment of hybrid epithelial/mesenchymal state and enhanced phenotypic heterogeneity in basal breast cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.558960" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.558960" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sahoo, S.; Ramu, S.; Nair, M. G.; Pillai, M.; Perez San Juan, B.; Milioli, H. Z.; Mandal, S.; Naidu, C. M.; Mavatkar, A. D.; Subramaniam, H.; Neogi, A. G.; Chaffer, C. L.; Prabhu, J. S.; Somarelli, J. A.; Jolly, M. K.</p>
        <p class="info">Score: 2.0, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.558960' target='https://doi.org/10.1101/2023.09.30.558960'> 10.1101/2023.09.30.558960</a></p>
        <p class="abstract">Intra-tumoral phenotypic heterogeneity promotes tumor relapse and therapeutic resistance and remains an unsolved clinical challenge. It manifests along multiple phenotypic axes and decoding the interconnections among these different axes is crucial to understand its molecular origins and to develop novel therapeutic strategies to control it. Here, we use multi-modal transcriptomic data analysis - bulk, single-cell and spatial transcriptomics - from breast cancer cell lines and primary tumor samples, to identify associations between epithelial-mesenchymal transition (EMT) and luminal-basal plasticity - two key processes that enable heterogeneity. We show that luminal breast cancer strongly associates with an epithelial cell state, but basal breast cancer is associated with hybrid epithelial/mesenchymal phenotype(s) and higher phenotypic heterogeneity. These patterns were inherent in methylation profiles, suggesting an epigenetic crosstalk between EMT and lineage plasticity in breast cancer. Mathematical modelling of core underlying gene regulatory networks representative of the crosstalk between the luminal-basal and epithelial-mesenchymal axes recapitulate and thus elucidate mechanistic underpinnings of the observed associations from transcriptomic data. Our systems-based approach integrating multi-modal data analysis with mechanism-based modeling offers a predictive framework to characterize intra-tumor heterogeneity and to identify possible interventions to restrict it.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559641">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559641" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559641">
        <p class="paperTitle">Pan-viral ORFs discovery using Massively Parallel Ribosome Profiling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559641" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559641" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weingarten-Gabbay, S.; Bauer, M. R.; Stanton, A. C.; Klaeger, S.; Verzani, E. K.; Lopez, D.; Clauser, K. R.; Carr, S. A.; Abelin, J. G.; Rice, C. M.; Sabeti, P. C.</p>
        <p class="info">Score: 1.8, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559641' target='https://doi.org/10.1101/2023.09.26.559641'> 10.1101/2023.09.26.559641</a></p>
        <p class="abstract">Unveiling the complete proteome of viruses is crucial to our understanding of the viral life cycle and interaction with the host. We developed Massively Parallel Ribosome Profiling (MPRP) to experimentally determine open reading frames (ORFs) in 20,170 designed oligonucleotides across 679 human-associated viral genomes. We identified 5,381 ORFs, including 4,208 non-canonical ORFs, and show successful detection of both annotated coding sequences (CDSs) and reported non-canonical ORFs. By examining immunopeptidome datasets of infected cells, we found class I human leukocyte antigen (HLA-I) peptides originating from non-canonical ORFs identified through MPRP. By inspecting ribosome occupancies on the 5UTR and CDS regions of annotated viral genes, we identified hundreds of upstream ORFs (uORFs) that negatively regulate the synthesis of canonical viral proteins. The unprecedented source of viral ORFs across a wide range of viral families, including highly pathogenic viruses, expands the repertoire of vaccine targets and exposes new cis-regulatory sequences in viral genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559627">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559627" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559627">
        <p class="paperTitle">Cardiac Specific Overexpression of Adenylyl Cyclase 8 Reprograms the Mouse Sinoatrial Node Transcriptome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559627" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559627" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Qu, J.; Tarasov, K. V.; Chakir, K.; Lakatta, E. G.</p>
        <p class="info">Score: 1.1, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559627' target='https://doi.org/10.1101/2023.09.26.559627'> 10.1101/2023.09.26.559627</a></p>
        <p class="abstract">A coupled-clock system intrinsic to sinoatrial node (SAN) pacemaker cells that regulates the rate and rhythm of spontaneous action potential firing, is activated by Ca2&#43;/calmodulin-stimulated Adenylyl Cyclase (AC) types 8 and 1. Our previous work in mice with cardiac specific overexpression of human AC8 gene (TGAC8) discovered that compared to its wild-type (WT) littermates, the heart rate (HR) of TGAC8 is elevated (by about 30%, 24 hours a day, 7days a week), and that the TGAC8 heart rhythm is markedly coherent, i.e., the HR Variability (HRV) lacks complexity, similar to that associated with aging or cardiac pathology. Reprogramming of molecular mechanisms, particularly SAN transcriptomic regulation that underlies the remarkable chronic shift in HR and HRV, however, has not been delineated. We conducted deep RNA sequencing (RNA-seq) in TGAC8 and WT SANs, using Mm10plus with human ADCY8 DNA sequence as the reference genome. Utilizing multiple bioinformatic techniques, we not only profiled the expression of marker genes related to SAN functions and AC-cAMP-PKA signaling, but also discovered negatively enriched hub pathways that differed in TGAC8 vs. WT, the top three being OXPHOS, ribosome, and cardiac muscle contraction. In contrast, signaling pathways related to inositol phosphate and its metabolism were positively enriched in TGAC8. Further, we identified two transcription regulators, KDM5A and PPARGC1A, that mediate effects of TGAC8 on ribosome and mitochondria. In summary, reprogrammed transcriptional regulation increases the HR of TGAC8 at the cost of impairment of other SAN cell functions, i.e., altered ribosome and inositol phosphate signaling and its crosstalk with mitochondria. Because these cell signaling alterations are associated with cardiac aging and age-associated CVDs, the TGAC8 mouse appears to be an ideal model in which to probe for potential therapeutic targets for cardiac aging and age-associated CVDs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.558451">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.558451" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.558451">
        <p class="paperTitle">High-volume, label-free imaging for quantifying single-cell dynamics in induced pluripotent stem cell colonies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.558451" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.558451" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Asmar, A. J.; Benson, Z.; Peskin, A. P.; Chalfoun, J.; Simon, M.; Halter, M.; Plant, A. L.</p>
        <p class="info">Score: 1.0, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.558451' target='https://doi.org/10.1101/2023.09.29.558451'> 10.1101/2023.09.29.558451</a></p>
        <p class="abstract">To facilitate the characterization of unlabeled induced pluripotent stem cells (iPSCs) during culture and expansion, we developed an AI pipeline for nuclear segmentation and mitosis detection from phase contrast images of individual cells within iPSC colonies. The analysis uses a 2D convolutional neural network (U-Net) plus a 3D U-Net applied on time lapse images to detect and segment nuclei, mitotic events, and daughter nuclei to enable tracking of large numbers of individual cells over long times in culture. The analysis uses fluorescence data to train models for segmenting nuclei in phase contrast images. The use of classical image processing routines to segment fluorescent nuclei precludes the need for manual annotation. We optimize and evaluate the accuracy of automated annotation to assure the reliability of the training. The model is generalizable in that it performs well on different datasets with an average F1 score of 0.94, on cells at different densities, and on cells from different pluripotent cell lines. The method allows us to assess, in a non-invasive manner, rates of mitosis and cell division which serve as indicators of cell state and cell health. We assess these parameters in up to hundreds of thousands of cells in culture for over 20 hours, at different locations in the colonies, and as a function of excitation light exposure.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559904">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559904" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559904">
        <p class="paperTitle">Investigating the dose-dependency of the midgut escape barrier using a mechanistic model of within-mosquito dengue virus population dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559904" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559904" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, R. M.; Stopard, I. J.; Byrne, H. M.; Armstrong, P. M.; Brackney, D. E.; Lambert, B.</p>
        <p class="info">Score: 1.0, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559904' target='https://doi.org/10.1101/2023.09.28.559904'> 10.1101/2023.09.28.559904</a></p>
        <p class="abstract">Flaviviruses are arthropod-borne (arbo)viruses which can emerge rapidly and cause explosive epidemics of severe disease. Some of the most epidemiologically important flaviviruses, including dengue virus (DENV), Zika virus (ZIKV) and yellow fever virus (YFV), are transmitted by Aedes mosquitoes, most notably Aedes aegypti and Aedes albopictus. After a mosquito blood feeds on an infected host, virus enters the midgut and infects the midgut epithelium. The virus must then overcome a series of barriers before reaching the mosquito saliva and being transmitted to a new host. The virus must escape from the midgut (known as the midgut escape barrier; MEB), which is thought to be mediated by transient changes in the permeability of the midgut-surrounding basal lamina layer (BL) following blood feeding. Here, we present a mathematical model of the within-mosquito population dynamics of flaviviruses that includes the interaction of the midgut and BL which can account for the MEB. Our results indicate a dose-dependency of midgut establishment of infection as well as rate of escape from the midgut: collectively, these suggest that the extrinsic incubation period (EIP) -- the time taken for DENV virus to be transmissible after infection -- is shortened when mosquitoes imbibe more virus. Additionally, our experimental data indicates that multiple blood feeding events, which more closely mimic mosquito-feeding behavior in the wild, can hasten the course of infections, and our model predicts that this effect is sensitive to the amount of virus imbibed. Our model indicates that mutations to the virus which impact its replication rate in the midgut could lead to even shorter EIPs when double-feeding occurs. Mechanistic models of within-vector viral infection dynamics provide a quantitative understanding of infection dynamics and could be used to evaluate novel interventions that target the mosquito stages of the infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559863">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559863" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559863">
        <p class="paperTitle">Discovering paracrine regulators of cellular composition from spatial transcriptomics data using SPER</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559863" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559863" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Haber, A. L.; Zhao, T.</p>
        <p class="info">Score: 1.0, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559863' target='https://doi.org/10.1101/2023.09.27.559863'> 10.1101/2023.09.27.559863</a></p>
        <p class="abstract">A wide range of pathologies and chronic diseases are characterized by transformations in cell type composition within a tissue. Prominent examples are remodeling of barrier epithelia in response to external threats, immune cell infiltration in chronic inflammatory conditions, or proliferation of fibroblasts and stromal cells causing fibrosis. An extensive literature implicates paracrine signals secreted by the local tissue niche in regulating its cell type composition, and therefore dysregulation of these factors is frequently capable of inducing pathological changes. Emerging multi-omics methods, particularly spatial transcriptomic profiling, provide the ability to simultaneously assay cell type composition and gene expression, with the potential to discover the transcripts most strongly associated with compositional shifts. However, no method yet exists to detect such signals. Here, we develop Spatial Paired Expression Ratio (SPER), a computational approach to evaluate the spatial dependence between transcripts and cell types in spatial transcriptomics data. We demonstrate the ability of SPER to accurately detect paracrine drivers of cellular abundance using simulated datasets. Using publicly available spatial transcriptomic data from the adult mouse motor cortex, we show that genes identified by SPER are strongly statistically enriched for both transcripts known to be extracellularly secreted, and those known to participate in paracrine receptor-ligand pairs, providing evidence of the ability to detect compositional regulatory signals. SPER recovered known and novel interactions, such as a surprising spatial association between the Wnt ligand Rpso3 and its cognate receptor Lgr5, expressed in a specific neural subset, and not previously known to regulate neural cell type composition in the motor cortex. SPER thus presents a general approach to discovering paracrine drivers of cellular compositional changes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
