<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 86.7, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 86.7, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-14T10:38:40+00:00" />
<meta property="article:modified_time" content="2024-01-14T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 86.7, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Open-ST: High-resolution spatial transcriptomics in 3D\nAuthors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.\nScore: 86.7, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554\nSpatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods.",
  "keywords": [
    
  ],
  "articleBody": " Open-ST: High-resolution spatial transcriptomics in 3D\nAuthors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.\nScore: 86.7, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554\nSpatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods. Here, we report Open-ST, a sequencing-based, open-source experimental and computational resource to address these challenges and to study the molecular organization of tissues in 3D. In mouse brain, Open-ST captured transcripts at subcellular resolution and reconstructed cell types. In primary tumor and patient-matched healthy/metastatic lymph nodes, Open-ST captured the diversity of immune, stromal and tumor populations in space. Distinct cell states were organized around cell-cell communication hotspots in the tumor, but not the metastasis. Strikingly, the 3D reconstruction and multimodal analysis of the metastatic lymph node revealed spatially contiguous structures not visible in 2D and potential biomarkers precisely at the 3D tumor/lymph node boundary. We anticipate Open-ST to accelerate the identification of spatial molecular mechanisms in 2D and 3D.\nMetalinksDB: a flexible and contextualizable resource of metabolite-protein interactions\nAuthors: Farr, E. B.; Dimitrov, D.; Turei, D.; Schmidt, C.; Lobentanzer, S.; Dugourd, A.; Saez-Rodriguez, J.\nScore: 14.9, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573715\nInteractions between proteins and metabolites are key for cellular function, from the catalytic breakdown of nutrients to signaling. An important case is cell-cell communication, where cellular metabolites are secreted into the microenvironment and initiate a signaling cascade by binding to an intra- or extracellular receptor of another cell. While protein-protein mediated cell-cell communication is routinely inferred from transcriptomic data, for metabolite-protein interactions this is challenging due to the limitations of high-throughput single-cell and spatial metabolomics technologies, together with the absence of comprehensive prior knowledge resources that include metabolites. Here we report MetalinksDB, a comprehensive and flexible database of intercellular metabolite-protein interactions that is a magnitude larger than existing ones. MetalinksDB can be tailored to specific biological contexts such as diseases, pathways, or tissue/cellular locations by querying subsets of interactions using the web interface (https://metalinks.omnipathdb.org/) or the knowledge graph adapters. We showcase the use of MetalinksDB by identifying deregulated processes in renal cancer patients from multi-omics data as well as inferring metabolite-mediated cell-cell communication events driving acute kidney injury from spatial transcriptomic data. We anticipate that MetalinksDB will facilitate the study of metabolite-mediated communication processes. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=113 SRC=\"FIGDIR/small/573715v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (34K): org.highwire.dtl.DTLVardef@1bf2c72org.highwire.dtl.DTLVardef@1ff2cc2org.highwire.dtl.DTLVardef@1d3a2b6org.highwire.dtl.DTLVardef@15f9c4a_HPS_FORMAT_FIGEXP M_FIG C_FIG\nImpaired biogenesis of basic proteins impacts multiple hallmarks of the aging brain\nAuthors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.\nScore: 37.0, Published: 2024-01-09 DOI: 10.1101/2023.07.20.549210\nAging and neurodegeneration entail diverse cellular and molecular hallmarks. Here, we studied the effects of aging on the transcriptome, translatome, and multiple layers of the proteome in the brain of a short-lived killifish. We reveal that aging causes widespread reduction of proteins enriched in basic amino acids that is independent of mRNA regulation, and it is not due to impaired proteasome activity. Instead, we identify a cascade of events where aberrant translation pausing leads to reduced ribosome availability resulting in proteome remodeling independently of transcriptional regulation. Our research uncovers a vulnerable point in the aging brains biology - the biogenesis of basic DNA/RNA binding proteins. This vulnerability may represent a unifying principle that connects various aging hallmarks, encompassing genome integrity and the biosynthesis of macromolecules. One-Sentence SummaryTranslation pausing reshapes the aging brain proteome, revealing vulnerabilities in the biogenesis of nucleic-acid protein.\nA quantitative description of light-limited cyanobacterial growth using flux balance analysis\nAuthors: Hoeper, R.; Komkova, D.; Zavrel, T.; Steuer, R.\nScore: 7.7, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574812\nThe metabolism of phototrophic cyanobacterial is an integral part of global biogeochemical cycles, and the capability of cyanobacteria to assimilate atmospheric CO2 into organic carbon has manifold potential applications for a sustainable biotechnology. To elucidate the properties of cyanobacterial metabolism and growth, computational reconstructions of the genome-scale metabolic networks play an increasingly important role. Here, we present an updated reconstruction of the metabolic network of the cyanobacterium Synechocystis sp. PCC 6803 and its analysis using flux balance analysis (FBA). To overcome limitations of conventional FBA, and to allow for the integration of quantitative experimental analyses, we develop a novel approach to describe light absorption and light utilization. Our approach incorporates photoinhibition and a variable quantum yield into the constraint-based description of light-limited phototrophic growth. We show that the resulting model is capable to predict quantitative properties of cyanobacterial growth, including photosynthetic oxygen evolution and the ATP/NADPH ratio required for growth and cellular maintenance. Our approach retains the computational and conceptual simplicity of FBA and is readily applicable to other phototropic microorganisms.\nTranscriptome-wide analysis of cell cycle-dependent bursty gene expression from single-cell RNA-seq data using mechanistic model-based inference\nAuthors: Sukys, A.; Grima, R.\nScore: 6.1, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574820\nBursty gene expression is quantified by two intuitive parameters: the burst frequency and the burst size. While these parameters are known to be cell-cycle dependent for some genes, a transcriptome-wide picture remains missing. Here we address this question by fitting a suite of mechanistic models of gene expression to mRNA count data for thousands of mouse genes, obtained by sequencing of single cells for which the cell-cycle position has been inferred using a deep-learning approach. This leads to the estimation of the burst frequency and size per allele in the G1 and G2/M cell-cycle phases, hence providing insight into the global patterns of transcriptional regulation. In particular, we identify an interesting balancing mechanism: on average, upon DNA replication, the burst frequency decreases by{approx} 50%, while the burst size increases by the same amount. We also show that for accurate estimation of the ratio of burst parameters in the G1 and G2/M phases, mechanistic models must explicitly account for gene copy number differences between cells but, surprisingly, additional corrections for extrinsic noise due to the coupling of transcription to cell age within the cell cycle or technical noise due to imperfect capture of RNA molecules in sequencing experiments are unnecessary.\nEnhanced feature matching in single-cell proteomics characterizes response to IFN-gamma and reveals co-existence of different cell states\nAuthors: Krull, K. K.; Ali, S. A.; Krijgsveld, J.\nScore: 2.5, Published: 2024-01-10 DOI: 10.1101/2024.01.10.575010\nProteome analysis by data-independent acquisition (DIA) has become a powerful approach to obtain deep proteome coverage, and has gained recent traction for label-free analysis of single cells. However, optimal experimental design for DIA-based single-cell proteomics has not been fully explored, and performance metrics of subsequent data analysis tools remain to be evaluated. Therefore, we here present DIA-ME, a data analysis strategy that exploits the co-analysis of low-input samples with a so-called matching enhancer (ME) of higher input, to increase sensitivity, proteome coverage, and data completeness. We evaluate the matching specificity of DIA-ME by a two-proteome model, and demonstrate that false discovery and false transfer are maintained at low levels when using DIA-NN software, while preserving quantification accuracy. We apply DIA-ME to investigate the proteome response of U-2 OS cells to interferon gamma (IFN-{gamma}) in single cells, and recapitulate the time-resolved induction of IFN-{gamma} response proteins as observed in bulk material. Moreover, we observe co- and anti-correlating patterns of protein expression within the same cell, indicating mutually exclusive protein modules and the co-existence of different cell states. Collectively our data show that DIA-ME is a powerful, scalable, and easy-to- implement strategy for single-cell proteomics.\nA systematic analysis of the effects of splicing on the diversity of post-translational modifications in protein isoforms\nAuthors: Crowl, S.; Coleman, M. B.; Chaphiv, A.; Naegle, K. M.\nScore: 1.5, Published: 2024-01-12 DOI: 10.1101/2024.01.10.575062\nPost-translational modifications (PTMs) and splicing are known to be important regulatory processes for controlling protein function and activity. However, there have been limitations in analyzing the interplay of alternative splicing and PTMs, both from the standpoint of PTM presence and in the possible diversification of the regulatory windows of PTMs, which define the connection to regulatory enzymes and possible binding partners. Limitations stem from the deep differences in genomic and proteomic databases, where PTMs are predominantly identified by mass spectrometry and subsequently assigned to the canonical isoform of the protein in databases. In this work, we bridge the protein- and genome-centric world views to map PTMs to genomic locations for subsequent projection of PTMs onto alternative isoforms. We then perform a systematic analysis of the diversification of PTMs within all defined protein isoforms, focusing on the PTM-specific profiles that may differ across the various major modifications found in humans, including exploration of how often alternative splicing leads to diversification of the regulatory sequences directly flanking a PTM. We found the interplay between splicing and PTMs is PTM-specific across a range of behaviors, such as PTM inclusion rates across isoforms and tissues. Additionally we found that ~2% of prospective PTM sites exhibited altered regulatory sequences surrounding the modification site, suggesting that regulatory or binding interactions might be diversified in these proteoforms. In addition to exploring isoforms as defined by Ensembl, we applied this PTM-to-isoform mapping approach to explore the impacts of disease-related splicing in prostate cancer, identifying possible new hypotheses as to the variable mechanisms of ESRP1 expression in different cancers.\nAtrial Proteomic Profiling Reveals a Switch Towards Profibrotic Gene Expression Program in CREM-IbΔC-X Mice with Persistent Atrial Fibrillation\nAuthors: Zhao, S.; Hulsurkar, M.; Lahiri, S.; Aguilar-Sanchez, Y.; Munivez, E.; Muller, F.-U.; Jain, A.; Malovannaya, A.; Yiu, C. H. K.; Reilly, S.; Wehrens, X.\nScore: 1.5, Published: 2024-01-12 DOI: 10.1101/2024.01.10.575097\nBackgroundOverexpression of the CREM (cAMP response element-binding modulator) isoform CREM-Ib{Delta}C-X in transgenic mice (CREM-Tg) causes the age-dependent development of spontaneous AF. PurposeTo identify key proteome signatures and biological processes accompanying the development of persistent AF through integrated proteomics and bioinformatics analysis. MethodsAtrial tissue samples from three CREM-Tg mice and three wild-type littermates were subjected to unbiased mass spectrometry-based quantitative proteomics, differential expression and pathway enrichment analysis, and protein-protein interaction (PPI) network analysis. ResultsA total of 98 differentially expressed proteins were identified. Gene ontology analysis revealed enrichment for biological processes regulating actin cytoskeleton organization and extracellular matrix (ECM) dynamics. Changes in ITGAV, FBLN5, and LCP1 were identified as being relevant to atrial fibrosis and remodeling based on expression changes, co-expression patterns, and PPI network analysis. Comparative analysis with previously published datasets revealed a shift in protein expression patterns from ion-channel and metabolic regulators in young CREM-Tg mice to profibrotic remodeling factors in older CREM-Tg mice. Furthermore, older CREM-Tg mice exhibited protein expression patterns that resembled those of humans with persistent AF. ConclusionsThis study uncovered distinct temporal changes in atrial protein expression patterns with age in CREM-Tg mice consistent with the progressive evolution of AF. Future studies into the role of the key differentially abundant proteins identified in this study in AF progression may open new therapeutic avenues to control atrial fibrosis and substrate development in AF. Graphical abstractGraphical abstract summarizing key findings of this paper. The atrial proteome in 9-month-old CREM- Tg mice with chronic persistent AF (perAF) was compared with age-matched WT littermates. In addition, proteome changes in these old CREM-Tg mice were compared with proteome changes previously identified in young CREM-Tg mice with paroxysmal AF (pAF). Moreover, an interspecies comparison was performed between old CREM-Tg mice and human patients with perAF. The major findings are that in pAF, key changes were identified in proteins involved in metabolism, energy production, DNA synthesis, and cell proliferation and growth. On the other hand, in mice and humans with perAF, key changes were found in the expression of proteins involved in collagen production, extracellular matrix remodeling, actin cytoskeleton organization, and tissue repair. O_FIG O_LINKSMALLFIG WIDTH=159 HEIGHT=200 SRC=\"FIGDIR/small/575097v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (33K): org.highwire.dtl.DTLVardef@1558fc7org.highwire.dtl.DTLVardef@5281b0org.highwire.dtl.DTLVardef@c05856org.highwire.dtl.DTLVardef@1e7ade1_HPS_FORMAT_FIGEXP M_FIG C_FIG\nPhenylketonuria: modelling cerebral amino acid and neurotransmitter metabolism\nAuthors: Wegrzyn, A. B.; van Vliet, D.; van Eunen, K.; Heiner-Fokkema, M. R.; van der Zee, E. A.; van Spronsen, F.; Bakker, B. M.\nScore: 2.2, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574352\nAO_SCPLOWBSTRACTC_SCPLOWO_ST_ABSObjectiveC_ST_ABSPhenylketonuria (PKU) is caused by a deficiency of the hepatic enzyme phenylalanine hydroxylase, which primarily converts phenylalanine into tyrosine. Despite a phenylalanine-deprived diet, many adult PKU patients display deficits of executive functions. These are hypothesised to be caused by high cerebral phenylalanine and a shortage of monoaminergic neurotransmitters. MethodTo better understand the relationship between plasma and brain amino acid levels and monoaminergic neurotransmitter biochemistry, we constructed a computational model. The model comprises the transport of large neutral amino acids (LNAA) across the blood-brain barrier as well as cerebral amino acid and monoaminergic neurotransmitter metabolism and was validated by direct measurements of brain amino acid concentrations in PKU mice on various diets. ResultsThe model predicts that brain amino acids are positively controlled by the concentrations of the corresponding amino acids in the blood, and to a lesser extent negatively by other amino acids competing for the transport systems. The model suggests that brain levels of monoaminergic neurotransmitters are controlled more by phenylalanine, probably through non-competitive inhibition of the hydroxylases rather than by their precursor amino acids. Therefore, the model predicts that the decrease of neurotransmitters in PKU cannot be fully rescued by the addition of tyrosine and tryptophan alone, but also benefits from a reduction of the phenylalanine level. Thereby, the model substantiates the experimental data of the synergistic effect of both brain phenylalanine reduction and increased precursor amino acid availability to improve brain neurotransmitters in PKU mice. ConclusionWe present the first complete model of the LNAA transport through the blood-brain barrier and subsequent brain neurotransmitter metabolism in PKU. The model leads to a better understanding of the pathophysiological mechanisms and the influence of individual amino acids in the diet on the underlying brain dysfunction in PKU. Moreover, it identifies gaps in our current knowledge about the LNAA transport across the blood-brain barrier. Furthermore, the model can be readily applied in studies of other neurological disorders in which the relation between diet, gene activities, brain amino acids, and neurotransmitters is important.\nTransient power-law behaviour following induction distinguishes between competing models of stochastic gene expression\nAuthors: Nicoll, A. G.; Szavits-Nossan, J.; Evans, M. R.; Grima, R.\nScore: 4.3, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573521\nWhat features of transcription can be learnt by fitting mathematical models of gene expression to mRNA count data? Given a suite of models, fitting to data selects an optimal one, thus identifying a probable transcriptional mechanism. Whilst attractive, the utility of this methodology remains unclear. Here, we sample steady-state, single-cell mRNA count distributions from parameters in the physiological range, and show they cannot be used to confidently estimate the number of inactive gene states, i.e. the number of rate-limiting steps in transcriptional initiation. Distributions from over 99% of the parameter space generated using models with 2, 3, or 4 inactive states can be well fit by one with a single inactive state. However, we show that if the mRNA lifetime is hours long, then for many minutes following induction, the increase in the mean mRNA count obeys a power law whose exponent equals the sum of the number of states visited from the initial inactive state to the active state and the number of rate-limiting post-transcriptional processing steps. Our study shows that non-linear regression estimation of the exponent from eukaryotic data is sufficient to estimate the total number of regulatory steps in transcription initiation, splicing, and nuclear export.\n",
  "wordCount" : "2575",
  "inLanguage": "en",
  "datePublished": "2024-01-14T10:38:40Z",
  "dateModified": "2024-01-14T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572554">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572554" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572554">
        <p class="paperTitle">Open-ST: High-resolution spatial transcriptomics in 3D</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572554" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572554" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.</p>
        <p class="info">Score: 86.7, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572554' target='https://doi.org/10.1101/2023.12.22.572554'> 10.1101/2023.12.22.572554</a></p>
        <p class="abstract">Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods. Here, we report Open-ST, a sequencing-based, open-source experimental and computational resource to address these challenges and to study the molecular organization of tissues in 3D. In mouse brain, Open-ST captured transcripts at subcellular resolution and reconstructed cell types. In primary tumor and patient-matched healthy/metastatic lymph nodes, Open-ST captured the diversity of immune, stromal and tumor populations in space. Distinct cell states were organized around cell-cell communication hotspots in the tumor, but not the metastasis. Strikingly, the 3D reconstruction and multimodal analysis of the metastatic lymph node revealed spatially contiguous structures not visible in 2D and potential biomarkers precisely at the 3D tumor/lymph node boundary. We anticipate Open-ST to accelerate the identification of spatial molecular mechanisms in 2D and 3D.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573715">
        <p class="paperTitle">MetalinksDB: a flexible and contextualizable resource of metabolite-protein interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Farr, E. B.; Dimitrov, D.; Turei, D.; Schmidt, C.; Lobentanzer, S.; Dugourd, A.; Saez-Rodriguez, J.</p>
        <p class="info">Score: 14.9, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573715' target='https://doi.org/10.1101/2023.12.30.573715'> 10.1101/2023.12.30.573715</a></p>
        <p class="abstract">Interactions between proteins and metabolites are key for cellular function, from the catalytic breakdown of nutrients to signaling. An important case is cell-cell communication, where cellular metabolites are secreted into the microenvironment and initiate a signaling cascade by binding to an intra- or extracellular receptor of another cell. While protein-protein mediated cell-cell communication is routinely inferred from transcriptomic data, for metabolite-protein interactions this is challenging due to the limitations of high-throughput single-cell and spatial metabolomics technologies, together with the absence of comprehensive prior knowledge resources that include metabolites. Here we report MetalinksDB, a comprehensive and flexible database of intercellular metabolite-protein interactions that is a magnitude larger than existing ones. MetalinksDB can be tailored to specific biological contexts such as diseases, pathways, or tissue/cellular locations by querying subsets of interactions using the web interface (https://metalinks.omnipathdb.org/) or the knowledge graph adapters. We showcase the use of MetalinksDB by identifying deregulated processes in renal cancer patients from multi-omics data as well as inferring metabolite-mediated cell-cell communication events driving acute kidney injury from spatial transcriptomic data. We anticipate that MetalinksDB will facilitate the study of metabolite-mediated communication processes.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=113 SRC=&#34;FIGDIR/small/573715v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (34K):
org.highwire.dtl.DTLVardef@1bf2c72org.highwire.dtl.DTLVardef@1ff2cc2org.highwire.dtl.DTLVardef@1d3a2b6org.highwire.dtl.DTLVardef@15f9c4a_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549210">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549210" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549210">
        <p class="paperTitle">Impaired biogenesis of basic proteins impacts multiple hallmarks of the aging brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549210" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549210" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.</p>
        <p class="info">Score: 37.0, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549210' target='https://doi.org/10.1101/2023.07.20.549210'> 10.1101/2023.07.20.549210</a></p>
        <p class="abstract">Aging and neurodegeneration entail diverse cellular and molecular hallmarks. Here, we studied the effects of aging on the transcriptome, translatome, and multiple layers of the proteome in the brain of a short-lived killifish. We reveal that aging causes widespread reduction of proteins enriched in basic amino acids that is independent of mRNA regulation, and it is not due to impaired proteasome activity. Instead, we identify a cascade of events where aberrant translation pausing leads to reduced ribosome availability resulting in proteome remodeling independently of transcriptional regulation. Our research uncovers a vulnerable point in the aging brains biology - the biogenesis of basic DNA/RNA binding proteins. This vulnerability may represent a unifying principle that connects various aging hallmarks, encompassing genome integrity and the biosynthesis of macromolecules.

One-Sentence SummaryTranslation pausing reshapes the aging brain proteome, revealing vulnerabilities in the biogenesis of nucleic-acid protein.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574812">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574812" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574812">
        <p class="paperTitle">A quantitative description of light-limited cyanobacterial growth using flux balance analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574812" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574812" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoeper, R.; Komkova, D.; Zavrel, T.; Steuer, R.</p>
        <p class="info">Score: 7.7, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574812' target='https://doi.org/10.1101/2024.01.10.574812'> 10.1101/2024.01.10.574812</a></p>
        <p class="abstract">The metabolism of phototrophic cyanobacterial is an integral part of global biogeochemical cycles, and the capability of cyanobacteria to assimilate atmospheric CO2 into organic carbon has manifold potential applications for a sustainable biotechnology. To elucidate the properties of cyanobacterial metabolism and growth, computational reconstructions of the genome-scale metabolic networks play an increasingly important role. Here, we present an updated reconstruction of the metabolic network of the cyanobacterium Synechocystis sp. PCC 6803 and its analysis using flux balance analysis (FBA). To overcome limitations of conventional FBA, and to allow for the integration of quantitative experimental analyses, we develop a novel approach to describe light absorption and light utilization. Our approach incorporates photoinhibition and a variable quantum yield into the constraint-based description of light-limited phototrophic growth. We show that the resulting model is capable to predict quantitative properties of cyanobacterial growth, including photosynthetic oxygen evolution and the ATP/NADPH ratio required for growth and cellular maintenance. Our approach retains the computational and conceptual simplicity of FBA and is readily applicable to other phototropic microorganisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574820">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574820" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574820">
        <p class="paperTitle">Transcriptome-wide analysis of cell cycle-dependent bursty gene expression from single-cell RNA-seq data using mechanistic model-based inference</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574820" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574820" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sukys, A.; Grima, R.</p>
        <p class="info">Score: 6.1, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574820' target='https://doi.org/10.1101/2024.01.10.574820'> 10.1101/2024.01.10.574820</a></p>
        <p class="abstract">Bursty gene expression is quantified by two intuitive parameters: the burst frequency and the burst size. While these parameters are known to be cell-cycle dependent for some genes, a transcriptome-wide picture remains missing. Here we address this question by fitting a suite of mechanistic models of gene expression to mRNA count data for thousands of mouse genes, obtained by sequencing of single cells for which the cell-cycle position has been inferred using a deep-learning approach. This leads to the estimation of the burst frequency and size per allele in the G1 and G2/M cell-cycle phases, hence providing insight into the global patterns of transcriptional regulation. In particular, we identify an interesting balancing mechanism: on average, upon DNA replication, the burst frequency decreases by{approx} 50%, while the burst size increases by the same amount. We also show that for accurate estimation of the ratio of burst parameters in the G1 and G2/M phases, mechanistic models must explicitly account for gene copy number differences between cells but, surprisingly, additional corrections for extrinsic noise due to the coupling of transcription to cell age within the cell cycle or technical noise due to imperfect capture of RNA molecules in sequencing experiments are unnecessary.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575010">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575010" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575010">
        <p class="paperTitle">Enhanced feature matching in single-cell proteomics characterizes response to IFN-gamma and reveals co-existence of different cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575010" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575010" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krull, K. K.; Ali, S. A.; Krijgsveld, J.</p>
        <p class="info">Score: 2.5, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575010' target='https://doi.org/10.1101/2024.01.10.575010'> 10.1101/2024.01.10.575010</a></p>
        <p class="abstract">Proteome analysis by data-independent acquisition (DIA) has become a powerful approach to obtain deep proteome coverage, and has gained recent traction for label-free analysis of single cells. However, optimal experimental design for DIA-based single-cell proteomics has not been fully explored, and performance metrics of subsequent data analysis tools remain to be evaluated. Therefore, we here present DIA-ME, a data analysis strategy that exploits the co-analysis of low-input samples with a so-called matching enhancer (ME) of higher input, to increase sensitivity, proteome coverage, and data completeness. We evaluate the matching specificity of DIA-ME by a two-proteome model, and demonstrate that false discovery and false transfer are maintained at low levels when using DIA-NN software, while preserving quantification accuracy. We apply DIA-ME to investigate the proteome response of U-2 OS cells to interferon gamma (IFN-{gamma}) in single cells, and recapitulate the time-resolved induction of IFN-{gamma} response proteins as observed in bulk material. Moreover, we observe co- and anti-correlating patterns of protein expression within the same cell, indicating mutually exclusive protein modules and the co-existence of different cell states. Collectively our data show that DIA-ME is a powerful, scalable, and easy-to- implement strategy for single-cell proteomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575062">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575062" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575062">
        <p class="paperTitle">A systematic analysis of the effects of splicing on the diversity of post-translational modifications in protein isoforms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575062" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575062" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Crowl, S.; Coleman, M. B.; Chaphiv, A.; Naegle, K. M.</p>
        <p class="info">Score: 1.5, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575062' target='https://doi.org/10.1101/2024.01.10.575062'> 10.1101/2024.01.10.575062</a></p>
        <p class="abstract">Post-translational modifications (PTMs) and splicing are known to be important regulatory processes for controlling protein function and activity. However, there have been limitations in analyzing the interplay of alternative splicing and PTMs, both from the standpoint of PTM presence and in the possible diversification of the regulatory windows of PTMs, which define the connection to regulatory enzymes and possible binding partners. Limitations stem from the deep differences in genomic and proteomic databases, where PTMs are predominantly identified by mass spectrometry and subsequently assigned to the canonical isoform of the protein in databases. In this work, we bridge the protein- and genome-centric world views to map PTMs to genomic locations for subsequent projection of PTMs onto alternative isoforms. We then perform a systematic analysis of the diversification of PTMs within all defined protein isoforms, focusing on the PTM-specific profiles that may differ across the various major modifications found in humans, including exploration of how often alternative splicing leads to diversification of the regulatory sequences directly flanking a PTM. We found the interplay between splicing and PTMs is PTM-specific across a range of behaviors, such as PTM inclusion rates across isoforms and tissues. Additionally we found that ~2% of prospective PTM sites exhibited altered regulatory sequences surrounding the modification site, suggesting that regulatory or binding interactions might be diversified in these proteoforms. In addition to exploring isoforms as defined by Ensembl, we applied this PTM-to-isoform mapping approach to explore the impacts of disease-related splicing in prostate cancer, identifying possible new hypotheses as to the variable mechanisms of ESRP1 expression in different cancers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575097" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575097">
        <p class="paperTitle">Atrial Proteomic Profiling Reveals a Switch Towards Profibrotic Gene Expression Program in CREM-IbΔC-X Mice with Persistent Atrial Fibrillation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, S.; Hulsurkar, M.; Lahiri, S.; Aguilar-Sanchez, Y.; Munivez, E.; Muller, F.-U.; Jain, A.; Malovannaya, A.; Yiu, C. H. K.; Reilly, S.; Wehrens, X.</p>
        <p class="info">Score: 1.5, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575097' target='https://doi.org/10.1101/2024.01.10.575097'> 10.1101/2024.01.10.575097</a></p>
        <p class="abstract">BackgroundOverexpression of the CREM (cAMP response element-binding modulator) isoform CREM-Ib{Delta}C-X in transgenic mice (CREM-Tg) causes the age-dependent development of spontaneous AF.

PurposeTo identify key proteome signatures and biological processes accompanying the development of persistent AF through integrated proteomics and bioinformatics analysis.

MethodsAtrial tissue samples from three CREM-Tg mice and three wild-type littermates were subjected to unbiased mass spectrometry-based quantitative proteomics, differential expression and pathway enrichment analysis, and protein-protein interaction (PPI) network analysis.

ResultsA total of 98 differentially expressed proteins were identified. Gene ontology analysis revealed enrichment for biological processes regulating actin cytoskeleton organization and extracellular matrix (ECM) dynamics. Changes in ITGAV, FBLN5, and LCP1 were identified as being relevant to atrial fibrosis and remodeling based on expression changes, co-expression patterns, and PPI network analysis. Comparative analysis with previously published datasets revealed a shift in protein expression patterns from ion-channel and metabolic regulators in young CREM-Tg mice to profibrotic remodeling factors in older CREM-Tg mice. Furthermore, older CREM-Tg mice exhibited protein expression patterns that resembled those of humans with persistent AF.

ConclusionsThis study uncovered distinct temporal changes in atrial protein expression patterns with age in CREM-Tg mice consistent with the progressive evolution of AF. Future studies into the role of the key differentially abundant proteins identified in this study in AF progression may open new therapeutic avenues to control atrial fibrosis and substrate development in AF.

Graphical abstractGraphical abstract summarizing key findings of this paper. The atrial proteome in 9-month-old CREM- Tg mice with chronic persistent AF (perAF) was compared with age-matched WT littermates. In addition, proteome changes in these old CREM-Tg mice were compared with proteome changes previously identified in young CREM-Tg mice with paroxysmal AF (pAF). Moreover, an interspecies comparison was performed between old CREM-Tg mice and human patients with perAF. The major findings are that in pAF, key changes were identified in proteins involved in metabolism, energy production, DNA synthesis, and cell proliferation and growth. On the other hand, in mice and humans with perAF, key changes were found in the expression of proteins involved in collagen production, extracellular matrix remodeling, actin cytoskeleton organization, and tissue repair.



O_FIG O_LINKSMALLFIG WIDTH=159 HEIGHT=200 SRC=&#34;FIGDIR/small/575097v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (33K):
org.highwire.dtl.DTLVardef@1558fc7org.highwire.dtl.DTLVardef@5281b0org.highwire.dtl.DTLVardef@c05856org.highwire.dtl.DTLVardef@1e7ade1_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574352" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574352">
        <p class="paperTitle">Phenylketonuria: modelling cerebral amino acid and neurotransmitter metabolism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wegrzyn, A. B.; van Vliet, D.; van Eunen, K.; Heiner-Fokkema, M. R.; van der Zee, E. A.; van Spronsen, F.; Bakker, B. M.</p>
        <p class="info">Score: 2.2, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574352' target='https://doi.org/10.1101/2024.01.05.574352'> 10.1101/2024.01.05.574352</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWO_ST_ABSObjectiveC_ST_ABSPhenylketonuria (PKU) is caused by a deficiency of the hepatic enzyme phenylalanine hydroxylase, which primarily converts phenylalanine into tyrosine. Despite a phenylalanine-deprived diet, many adult PKU patients display deficits of executive functions. These are hypothesised to be caused by high cerebral phenylalanine and a shortage of monoaminergic neurotransmitters.

MethodTo better understand the relationship between plasma and brain amino acid levels and monoaminergic neurotransmitter biochemistry, we constructed a computational model. The model comprises the transport of large neutral amino acids (LNAA) across the blood-brain barrier as well as cerebral amino acid and monoaminergic neurotransmitter metabolism and was validated by direct measurements of brain amino acid concentrations in PKU mice on various diets.

ResultsThe model predicts that brain amino acids are positively controlled by the concentrations of the corresponding amino acids in the blood, and to a lesser extent negatively by other amino acids competing for the transport systems. The model suggests that brain levels of monoaminergic neurotransmitters are controlled more by phenylalanine, probably through non-competitive inhibition of the hydroxylases rather than by their precursor amino acids. Therefore, the model predicts that the decrease of neurotransmitters in PKU cannot be fully rescued by the addition of tyrosine and tryptophan alone, but also benefits from a reduction of the phenylalanine level. Thereby, the model substantiates the experimental data of the synergistic effect of both brain phenylalanine reduction and increased precursor amino acid availability to improve brain neurotransmitters in PKU mice.

ConclusionWe present the first complete model of the LNAA transport through the blood-brain barrier and subsequent brain neurotransmitter metabolism in PKU. The model leads to a better understanding of the pathophysiological mechanisms and the influence of individual amino acids in the diet on the underlying brain dysfunction in PKU. Moreover, it identifies gaps in our current knowledge about the LNAA transport across the blood-brain barrier. Furthermore, the model can be readily applied in studies of other neurological disorders in which the relation between diet, gene activities, brain amino acids, and neurotransmitters is important.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573521">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573521" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573521">
        <p class="paperTitle">Transient power-law behaviour following induction distinguishes between competing models of stochastic gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573521" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573521" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nicoll, A. G.; Szavits-Nossan, J.; Evans, M. R.; Grima, R.</p>
        <p class="info">Score: 4.3, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573521' target='https://doi.org/10.1101/2023.12.30.573521'> 10.1101/2023.12.30.573521</a></p>
        <p class="abstract">What features of transcription can be learnt by fitting mathematical models of gene expression to mRNA count data? Given a suite of models, fitting to data selects an optimal one, thus identifying a probable transcriptional mechanism. Whilst attractive, the utility of this methodology remains unclear. Here, we sample steady-state, single-cell mRNA count distributions from parameters in the physiological range, and show they cannot be used to confidently estimate the number of inactive gene states, i.e. the number of rate-limiting steps in transcriptional initiation. Distributions from over 99% of the parameter space generated using models with 2, 3, or 4 inactive states can be well fit by one with a single inactive state. However, we show that if the mRNA lifetime is hours long, then for many minutes following induction, the increase in the mean mRNA count obeys a power law whose exponent equals the sum of the number of states visited from the initial inactive state to the active state and the number of rate-limiting post-transcriptional processing steps. Our study shows that non-linear regression estimation of the exponent from eukaryotic data is sufficient to estimate the total number of regulatory steps in transcription initiation, splicing, and nuclear export.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
