<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="High-throughput and scalable single cell proteomics identifies over 5000 proteins per cell
Authors: Ye, Z.; Sabatier, P.; Hoeven, L.; Phlairaharn, T.; Hartlmayr, D.; Izaguirre, F.; Seth, A.; Joshi, H.; Bekker-Jensen, D.; Bache, N.; Olsen, J. V.
Score: 55.0, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568953
The emergence of mass spectrometry (MS)-based single-cell proteomics (SCP) promise to revolutionize the study of cellular biology and biomedicine by providing an unparalleled view of the proteome in individual cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="High-throughput and scalable single cell proteomics identifies over 5000 proteins per cell
Authors: Ye, Z.; Sabatier, P.; Hoeven, L.; Phlairaharn, T.; Hartlmayr, D.; Izaguirre, F.; Seth, A.; Joshi, H.; Bekker-Jensen, D.; Bache, N.; Olsen, J. V.
Score: 55.0, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568953
The emergence of mass spectrometry (MS)-based single-cell proteomics (SCP) promise to revolutionize the study of cellular biology and biomedicine by providing an unparalleled view of the proteome in individual cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-06T10:40:04+00:00" />
<meta property="article:modified_time" content="2023-12-06T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="High-throughput and scalable single cell proteomics identifies over 5000 proteins per cell
Authors: Ye, Z.; Sabatier, P.; Hoeven, L.; Phlairaharn, T.; Hartlmayr, D.; Izaguirre, F.; Seth, A.; Joshi, H.; Bekker-Jensen, D.; Bache, N.; Olsen, J. V.
Score: 55.0, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568953
The emergence of mass spectrometry (MS)-based single-cell proteomics (SCP) promise to revolutionize the study of cellular biology and biomedicine by providing an unparalleled view of the proteome in individual cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "High-throughput and scalable single cell proteomics identifies over 5000 proteins per cell\nAuthors: Ye, Z.; Sabatier, P.; Hoeven, L.; Phlairaharn, T.; Hartlmayr, D.; Izaguirre, F.; Seth, A.; Joshi, H.; Bekker-Jensen, D.; Bache, N.; Olsen, J. V.\nScore: 55.0, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568953\nThe emergence of mass spectrometry (MS)-based single-cell proteomics (SCP) promise to revolutionize the study of cellular biology and biomedicine by providing an unparalleled view of the proteome in individual cells.",
  "keywords": [
    
  ],
  "articleBody": " High-throughput and scalable single cell proteomics identifies over 5000 proteins per cell\nAuthors: Ye, Z.; Sabatier, P.; Hoeven, L.; Phlairaharn, T.; Hartlmayr, D.; Izaguirre, F.; Seth, A.; Joshi, H.; Bekker-Jensen, D.; Bache, N.; Olsen, J. V.\nScore: 55.0, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568953\nThe emergence of mass spectrometry (MS)-based single-cell proteomics (SCP) promise to revolutionize the study of cellular biology and biomedicine by providing an unparalleled view of the proteome in individual cells. Despite its groundbreaking potential, SCP is nascent and faces challenges including limited sequence depth, throughput, and reproducibility, which have constrained its broader utility. This study introduces key methodological advances, which considerably improve the sensitivity, coverage and dependability of protein identification from single cells. We developed an almost lossless SCP workflow encompassing sample preparation to MS analysis, doubling the number of identified proteins from roughly 2000 to over 5000 in individual HeLa cells. A comprehensive evaluation of analytical software tools, alongside strict false discovery rate (FDR) controls solidified the reliability of our results. These enhancements also facilitated the direct detection of post-translational modifications (PTMs) in single cells, negating the need for enrichment and thereby simplifying the analytical process. Although throughput in MS remains a challenge, our study demonstrates the feasibility of processing up to 80 label-free SCP samples per day. Moreover, an optimized tissue dissociation buffer enabled effective single cell disaggregation of drug-treated cancer cell spheroids, refining the overall proteomic analysis. Our workflow sets a new benchmark in SCP for sensitivity and throughput, with broad applications ranging from the study of cellular development to disease progression and the identification of cell type-specific markers and therapeutic targets.\nA foundational atlas of autism protein interactions reveals molecular convergence\nAuthors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.; Pham, P.; Arbalaez, J.; Pratt, D.; Chag, S.; Rolland, T.; Bourgeron, T.; Finkbeiner, S.; Bandyopadhay, S.; Ideker, T.; Beltrao, P.; Willsey, H. R.; Obernier, K.; Nowakowski, T. J.; Huttenhain, R.; State, M. W.; Willsey, A. J.; Krogan, N. J.\nScore: 50.9, Published: 2023-12-04 DOI: 10.1101/2023.12.03.569805\nDespite the identification of a large number of high-confidence (hc) autism spectrum disorder (ASD) risk genes, translation to viable treatment targets remains elusive. Using affinity purification-mass spectrometry and HEK293T cells, we constructed a protein-protein interaction (PPI) network involving 100 hcASD risk genes, revealing over 1,800 PPIs, the vast majority of which have not been previously described. Importantly, the interactors are highly expressed in the human brain and neural progenitor cells, and enriched for ASD, but not schizophrenia, genetic risk. The network is highly connected, converges on protein complexes, including those involved in neurogenesis, tubulin biology, transcriptional regulation, and chromatin modification. We also generated a PPI map using over 50 patient-derived missense variants to identify differential physical interactions. Using these data, we leveraged AlphaFold2 predictions to prioritize direct PPIs as well as specific variants for interrogation in Xenopus tropicalis and human forebrain organoids. One such mutation in the transcription factor FOXP1, which disrupts interaction with FOXP4, led to a reconfiguration of DNA binding sites and disruption of neurogenesis, with a particularly strong impact on deep layer cortical and subplate neurons,consistent with a large body of work in ASD. Overall, this work offers critical new insights into molecular mechanisms underlying ASD that results from large effect genetic variants and describes a powerful platform to develop and test novel therapeutic strategies for a wide range of genetically-defined conditions.\nAn automated and fast sample preparation workflow for laser microdissection guided ultrasensitive proteomics\nAuthors: Makhmut, A.; Qin, D.; Hartlmayr, D.; Seth, A.; Coscia, F.\nScore: 14.9, Published: 2023-12-01 DOI: 10.1101/2023.11.29.569257\nSpatial tissue proteomics integrating whole-slide imaging, laser microdissection and ultrasensitive mass spectrometry is a powerful approach to link cellular phenotypes to functional proteome states in (patho)physiology. To be applicable to large patient cohorts and low sample input amounts, including single-cell applications, loss-minimized and streamlined end-to-end workflows are key. We here introduce an automated sample preparation protocol for laser microdissected samples utilizing the cellenONE(R) robotic system, which has the capacity to process 192 samples in three hours. Following laser microdissection collection directly into the proteoCHIP LF 48 or EVO 96 chip, our optimized protocol facilitates lysis, formalin de-crosslinking and tryptic digest of low-input archival tissue samples. The seamless integration with the Evosep ONE LC system by centrifugation allows on-the-fly sample clean-up, particularly pertinent for laser microdissected workflows. We validate our method in human tonsil archival tissue, where we profile proteomes of spatially-defined B-cell, T-cell and epithelial microregions of 4,000 {micro}m2 to a depth of [~]2,000 proteins and with high cell type specificity. We finally provide detailed equipment templates and experimental guidelines for broad accessibility.\nRemodeling of the protein ubiquitylation landscape in the aging vertebrate brain\nAuthors: Marino, A.; Di Fraia, D.; Panfilova, D.; Sahu, A. K.; Ori, A.\nScore: 9.3, Published: 2023-12-04 DOI: 10.1101/2023.12.02.569713\nPost-translational modifications (PTMs) regulate protein homeostasis and function. How aging affects the landscape of PTMs remains largely elusive. Here, we reveal changes in hundreds of protein ubiquitylation, acetylation, and phosphorylation sites in the aging brain of mice. We show that aging has a major impact on protein ubiquitylation and that 29% of the ubiquitylation sites are affected independently of protein abundance, indicating altered PTM stoichiometry. We found a subset of these sites to be also affected in the brain of the short-lived killifish Nothobranchius furzeri, highlighting a conserved aging phenotype. Furthermore, we estimated that over 35% of ubiquitylation changes observed in old mouse brains derive from partial proteasome inhibition, a well-established hallmark of brain aging. Our findings provide evidence of an evolutionarily conserved ubiquitylation signature of the aging brain and establish a causal link between proteasome inhibition and age-related remodeling of the ubiquitylome.\nTranslation variation across genetic backgrounds reveals a post-transcriptional buffering signature in yeast\nAuthors: Teyssonniere, E.; Shichino, Y.; Friedrich, A.; Iwasaki, S.; Schacherer, J.\nScore: 9.3, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568778\nGene expression is known to vary among individuals, and this variability can impact the phenotypic diversity observed in natural populations. While the transcriptome and proteome have been extensively studied, little is known about the translation process itself. Here, we therefore performed ribosome and transcriptomic profiling on a genetically and ecologically diverse set of natural isolates of the Saccharomyces cerevisiae yeast. Interestingly, we found that the Euclidean distances between each profile and the expression fold changes in each pairwise isolate comparison were higher at the transcriptomic level. This observation clearly indicates that the transcriptional variation observed in the different isolates is buffered through a phenomenon known as post-transcriptional buffering at the translation level. Furthermore, this phenomenon seemed to have a specific signature by preferentially affecting essential genes as well as genes involved in complex-forming proteins, and low transcribed genes. We also explored the translation of the S. cerevisiae pangenome and found that the accessory genes related to introgression events displayed similar transcription and translation levels as the core genome. By contrast, genes acquired through horizontal gene transfer events tended to be less efficiently translated. Together, our results highlight both the extent and signature of the post-transcriptional buffering.\nEpistemic uncertainty challenges aging clock reliability in predicting rejuvenation effects\nAuthors: Kriukov, D.; Kuzmina, E. A.; Efimov, E.; Dylov, D. V.; Khrameeva, E.\nScore: 5.6, Published: 2023-12-04 DOI: 10.1101/2023.12.01.569529\nEpigenetic aging clocks have been widely used to validate rejuvenation effects during cellular reprogramming. However, these predictions are unfalsifiable, since the true biological age of reprogrammed cells remains inaccessible. We present a multifaceted analytical framework to consider rejuvenation predictions from the uncertainty perspective. We discover that DNA methylation profiles of reprogramming are not represented in the aging data used for clock training, which introduces high epistemic uncertainty in aging predictions. Moreover, predictions of different published clocks are poorly consistent with each other and suggest even zero or negative rejuvenation. We show that the high prediction uncertainty challenges the reliability of rejuvenation effects observed during in vitro reprogramming prior to pluripotency and throughout embryogenesis. Conversely, our method also reveals a significant age increase after in vivo reprogramming. We propose to include uncertainty estimation in future aging clocks to avoid the risk of misinterpreting the results of biological age prediction.\nProteome-wide base editor screens to assess phosphorylation site functionality in high-throughput\nAuthors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.\nScore: 19.8, Published: 2023-11-14 DOI: 10.1101/2023.11.11.566649\nSignaling pathways that drive gene expression are typically depicted as having a dozen or so landmark phosphorylation and transcriptional events. In reality, thousands of dynamic post-translational modifications (PTMs) orchestrate nearly every cellular function, and we lack technologies to find causal links between these vast biochemical pathways and genetic circuits at scale. Here, we describe \"signaling-to-transcription network\" mapping through the development of PTM-centric base editing coupled to phenotypic screens, directed by temporally-resolved phosphoproteomics. Using T cell activation as a model, we observe hundreds of unstudied phosphorylation sites that modulate NFAT transcriptional activity. We identify the phosphorylation-mediated nuclear localization of the phosphatase PHLPP1 which promotes NFAT but inhibits NF{kappa}B activity. We also find that specific phosphosite mutants can alter gene expression in subtle yet distinct patterns, demonstrating the potential for fine-tuning transcriptional responses. Overall, base editor screening of PTM sites provides a powerful platform to dissect PTM function within signaling pathways.\nChronic metabolic stress drives developmental programs and loss of tissue functions in non-transformed liver that mirror tumor states and stratify survival\nAuthors: Tzouanas, C. N.; Sherman, M. S.; Shay, J. E. S.; Rubin, A. J.; Mead, B. E.; Dao, T. T.; Butzlaff, T.; Mana, M. D.; Kolb, K. E.; Walesky, C.; Pepe-Mooney, B. J.; Smith, C. J.; Prakadan, S. M.; Ramseier, M. L.; Tong, E. Y.; Joung, J.; Chi, F.; McMahon-Skates, T.; Winston, C. L.; Jeong, W.-J.; Aney, K. J.; Chen, E.; Nissim, S.; Zhang, F.; Deshpande, V.; Lauer, G. M.; Yilmaz, O. H.; Goessling, W.; Shalek, A. K.\nScore: 2.5, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569407\nUnder chronic stress, cells must balance competing demands between cellular survival and tissue function. In metabolic dysfunction-associated steatotic liver disease (MASLD, formerly NAFLD/NASH), hepatocytes cooperate with structural and immune cells to perform crucial metabolic, synthetic, and detoxification functions despite nutrient imbalances. While prior work has emphasized stress-induced drivers of cell death, the dynamic adaptations of surviving cells and their functional repercussions remain unclear. Namely, we do not know which pathways and programs define cellular responses, what regulatory factors mediate (mal)adaptations, and how this aberrant activity connects to tissue-scale dysfunction and long-term disease outcomes. Here, by applying longitudinal single-cell multi-omics to a mouse model of chronic metabolic stress and extending to human cohorts, we show that stress drives survival-linked tradeoffs and metabolic rewiring, manifesting as shifts towards development-associated states in non-transformed hepatocytes with accompanying decreases in their professional functionality. Diet-induced adaptations occur significantly prior to tumorigenesis but parallel tumorigenesis-induced phenotypes and predict worsened human cancer survival. Through the development of a multi-omic computational gene regulatory inference framework and human in vitro and mouse in vivo genetic perturbations, we validate transcriptional (RELB, SOX4) and metabolic (HMGCS2) mediators that co-regulate and couple the balance between developmental state and hepatocyte functional identity programming. Our work defines cellular features of liver adaptation to chronic stress as well as their links to long-term disease outcomes and cancer hallmarks, unifying diverse axes of cellular dysfunction around core causal mechanisms.\nAn evolution-based framework for describing human gut bacteria\nAuthors: Doran, B. A.; Chen, R. Y.; Giba, H.; Behera, V.; Barat, B.; Sundararajan, A.; Lin, H.; Sidebottom, A.; Pamer, E. G.; Raman, A.\nScore: 2.2, Published: 2023-12-05 DOI: 10.1101/2023.12.04.569969\nThe human gut microbiome contains many bacterial strains of the same species (\"strain-level variants\"). Describing strains in a biologically meaningful way rather than purely taxonomically is an important goal but challenging due to the genetic complexity of strain-level variation. Here, we measured patterns of co-evolution across \u003e7,000 strains spanning the bacterial tree-of-life. Using these patterns as a prior for studying hundreds of gut commensal strains that we isolated, sequenced, and metabolically profiled revealed widespread structure beneath the phylogenetic level of species. Defining strains by their co-evolutionary signatures enabled predicting their metabolic phenotypes and engineering consortia from strain genome content alone. Our findings demonstrate a biologically relevant organization to strain-level variation and motivate a new schema for describing bacterial strains based on their evolutionary history.\nProneural - Mesenchymal antagonism dominates the patterns of phenotypic heterogeneity in Glioblastoma\nAuthors: BV, H.; Jolly, M. K.\nScore: 2.4, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568853\n1The aggressive nature of glioblastoma (GBM) - one of the deadliest forms of brain tumours - is majorly attributed to underlying phenotypic heterogeneity. Early attempts to classify this heterogeneity at a transcriptomic level in TCGA GBM cohort proposed the existence of four distinct molecular subtypes: Proneural, Neural, Classical and Mesenchymal. Further, a single-cell RNA-seq analysis of primary tumours also reported similar 4 subtypes mimicking neuro-developmental lineages. However, it remains unclear whether these 4 subtypes identified via bulk and single-cell transcriptomics are mutually exclusive or not. Here, we perform pairwise correlations among individual genes and gene signatures corresponding to these proposed subtypes, and show that the subtypes are not distinctly mutually antagonistic in either TCGA or single-cell RNA-sequencing data. We observed that the proneural (or neural progenitor-like) - mesenchymal axis is the most prominent antagonistic pair, with the other two subtypes lying on this spectrum. These results are reinforced through a meta-analysis of over 100 single-cell and bulk transcriptomic datasets as well as in terms of functional association with metabolic switching, cell cycle and immune evasion pathways. These results suggest rethinking GBM phenotypic characterization for more effective therapeutic targeting efforts.\n",
  "wordCount" : "2317",
  "inLanguage": "en",
  "datePublished": "2023-12-06T10:40:04Z",
  "dateModified": "2023-12-06T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 6, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568953">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568953" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568953">
        <p class="paperTitle">High-throughput and scalable single cell proteomics identifies over 5000 proteins per cell</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568953" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568953" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ye, Z.; Sabatier, P.; Hoeven, L.; Phlairaharn, T.; Hartlmayr, D.; Izaguirre, F.; Seth, A.; Joshi, H.; Bekker-Jensen, D.; Bache, N.; Olsen, J. V.</p>
        <p class="info">Score: 55.0, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568953' target='https://doi.org/10.1101/2023.11.27.568953'> 10.1101/2023.11.27.568953</a></p>
        <p class="abstract">The emergence of mass spectrometry (MS)-based single-cell proteomics (SCP) promise to revolutionize the study of cellular biology and biomedicine by providing an unparalleled view of the proteome in individual cells. Despite its groundbreaking potential, SCP is nascent and faces challenges including limited sequence depth, throughput, and reproducibility, which have constrained its broader utility. This study introduces key methodological advances, which considerably improve the sensitivity, coverage and dependability of protein identification from single cells. We developed an almost lossless SCP workflow encompassing sample preparation to MS analysis, doubling the number of identified proteins from roughly 2000 to over 5000 in individual HeLa cells. A comprehensive evaluation of analytical software tools, alongside strict false discovery rate (FDR) controls solidified the reliability of our results. These enhancements also facilitated the direct detection of post-translational modifications (PTMs) in single cells, negating the need for enrichment and thereby simplifying the analytical process. Although throughput in MS remains a challenge, our study demonstrates the feasibility of processing up to 80 label-free SCP samples per day. Moreover, an optimized tissue dissociation buffer enabled effective single cell disaggregation of drug-treated cancer cell spheroids, refining the overall proteomic analysis. Our workflow sets a new benchmark in SCP for sensitivity and throughput, with broad applications ranging from the study of cellular development to disease progression and the identification of cell type-specific markers and therapeutic targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569805">
        <p class="paperTitle">A foundational atlas of autism protein interactions reveals molecular convergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.; Pham, P.; Arbalaez, J.; Pratt, D.; Chag, S.; Rolland, T.; Bourgeron, T.; Finkbeiner, S.; Bandyopadhay, S.; Ideker, T.; Beltrao, P.; Willsey, H. R.; Obernier, K.; Nowakowski, T. J.; Huttenhain, R.; State, M. W.; Willsey, A. J.; Krogan, N. J.</p>
        <p class="info">Score: 50.9, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569805' target='https://doi.org/10.1101/2023.12.03.569805'> 10.1101/2023.12.03.569805</a></p>
        <p class="abstract">Despite the identification of a large number of high-confidence (hc) autism spectrum disorder (ASD) risk genes, translation to viable treatment targets remains elusive. Using affinity purification-mass spectrometry and HEK293T cells, we constructed a protein-protein interaction (PPI) network involving 100 hcASD risk genes, revealing over 1,800 PPIs, the vast majority of which have not been previously described. Importantly, the interactors are highly expressed in the human brain and neural progenitor cells, and enriched for ASD, but not schizophrenia, genetic risk. The network is highly connected, converges on protein complexes, including those involved in neurogenesis, tubulin biology, transcriptional regulation, and chromatin modification. We also generated a PPI map using over 50 patient-derived missense variants to identify differential physical interactions. Using these data, we leveraged AlphaFold2 predictions to prioritize direct PPIs as well as specific variants for interrogation in Xenopus tropicalis and human forebrain organoids. One such mutation in the transcription factor FOXP1, which disrupts interaction with FOXP4, led to a reconfiguration of DNA binding sites and disruption of neurogenesis, with a particularly strong impact on deep layer cortical and subplate neurons,consistent with a large body of work in ASD. Overall, this work offers critical new insights into molecular mechanisms underlying ASD that results from large effect genetic variants and describes a powerful platform to develop and test novel therapeutic strategies for a wide range of genetically-defined conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569257">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569257" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569257">
        <p class="paperTitle">An automated and fast sample preparation workflow for laser microdissection guided ultrasensitive proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569257" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569257" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Makhmut, A.; Qin, D.; Hartlmayr, D.; Seth, A.; Coscia, F.</p>
        <p class="info">Score: 14.9, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569257' target='https://doi.org/10.1101/2023.11.29.569257'> 10.1101/2023.11.29.569257</a></p>
        <p class="abstract">Spatial tissue proteomics integrating whole-slide imaging, laser microdissection and ultrasensitive mass spectrometry is a powerful approach to link cellular phenotypes to functional proteome states in (patho)physiology. To be applicable to large patient cohorts and low sample input amounts, including single-cell applications, loss-minimized and streamlined end-to-end workflows are key. We here introduce an automated sample preparation protocol for laser microdissected samples utilizing the cellenONE(R) robotic system, which has the capacity to process 192 samples in three hours. Following laser microdissection collection directly into the proteoCHIP LF 48 or EVO 96 chip, our optimized protocol facilitates lysis, formalin de-crosslinking and tryptic digest of low-input archival tissue samples. The seamless integration with the Evosep ONE LC system by centrifugation allows  on-the-fly sample clean-up, particularly pertinent for laser microdissected workflows. We validate our method in human tonsil archival tissue, where we profile proteomes of spatially-defined B-cell, T-cell and epithelial microregions of 4,000 {micro}m2 to a depth of [~]2,000 proteins and with high cell type specificity. We finally provide detailed equipment templates and experimental guidelines for broad accessibility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569713">
        <p class="paperTitle">Remodeling of the protein ubiquitylation landscape in the aging vertebrate brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marino, A.; Di Fraia, D.; Panfilova, D.; Sahu, A. K.; Ori, A.</p>
        <p class="info">Score: 9.3, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569713' target='https://doi.org/10.1101/2023.12.02.569713'> 10.1101/2023.12.02.569713</a></p>
        <p class="abstract">Post-translational modifications (PTMs) regulate protein homeostasis and function. How aging affects the landscape of PTMs remains largely elusive. Here, we reveal changes in hundreds of protein ubiquitylation, acetylation, and phosphorylation sites in the aging brain of mice. We show that aging has a major impact on protein ubiquitylation and that 29% of the ubiquitylation sites are affected independently of protein abundance, indicating altered PTM stoichiometry. We found a subset of these sites to be also affected in the brain of the short-lived killifish Nothobranchius furzeri, highlighting a conserved aging phenotype. Furthermore, we estimated that over 35% of ubiquitylation changes observed in old mouse brains derive from partial proteasome inhibition, a well-established hallmark of brain aging. Our findings provide evidence of an evolutionarily conserved ubiquitylation signature of the aging brain and establish a causal link between proteasome inhibition and age-related remodeling of the ubiquitylome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568778" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568778">
        <p class="paperTitle">Translation variation across genetic backgrounds reveals a post-transcriptional buffering signature in yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Teyssonniere, E.; Shichino, Y.; Friedrich, A.; Iwasaki, S.; Schacherer, J.</p>
        <p class="info">Score: 9.3, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568778' target='https://doi.org/10.1101/2023.11.28.568778'> 10.1101/2023.11.28.568778</a></p>
        <p class="abstract">Gene expression is known to vary among individuals, and this variability can impact the phenotypic diversity observed in natural populations. While the transcriptome and proteome have been extensively studied, little is known about the translation process itself. Here, we therefore performed ribosome and transcriptomic profiling on a genetically and ecologically diverse set of natural isolates of the Saccharomyces cerevisiae yeast. Interestingly, we found that the Euclidean distances between each profile and the expression fold changes in each pairwise isolate comparison were higher at the transcriptomic level. This observation clearly indicates that the transcriptional variation observed in the different isolates is buffered through a phenomenon known as post-transcriptional buffering at the translation level. Furthermore, this phenomenon seemed to have a specific signature by preferentially affecting essential genes as well as genes involved in complex-forming proteins, and low transcribed genes. We also explored the translation of the S. cerevisiae pangenome and found that the accessory genes related to introgression events displayed similar transcription and translation levels as the core genome. By contrast, genes acquired through horizontal gene transfer events tended to be less efficiently translated. Together, our results highlight both the extent and signature of the post-transcriptional buffering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569529">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569529" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569529">
        <p class="paperTitle">Epistemic uncertainty challenges aging clock reliability in predicting rejuvenation effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569529" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569529" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kriukov, D.; Kuzmina, E. A.; Efimov, E.; Dylov, D. V.; Khrameeva, E.</p>
        <p class="info">Score: 5.6, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569529' target='https://doi.org/10.1101/2023.12.01.569529'> 10.1101/2023.12.01.569529</a></p>
        <p class="abstract">Epigenetic aging clocks have been widely used to validate rejuvenation effects during cellular reprogramming. However, these predictions are unfalsifiable, since the true biological age of reprogrammed cells remains inaccessible. We present a multifaceted analytical framework to consider rejuvenation predictions from the uncertainty perspective. We discover that DNA methylation profiles of reprogramming are not represented in the aging data used for clock training, which introduces high epistemic uncertainty in aging predictions. Moreover, predictions of different published clocks are poorly consistent with each other and suggest even zero or negative rejuvenation. We show that the high prediction uncertainty challenges the reliability of rejuvenation effects observed during in vitro reprogramming prior to pluripotency and throughout embryogenesis. Conversely, our method also reveals a significant age increase after in vivo reprogramming. We propose to include uncertainty estimation in future aging clocks to avoid the risk of misinterpreting the results of biological age prediction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566649" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566649">
        <p class="paperTitle">Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.</p>
        <p class="info">Score: 19.8, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566649' target='https://doi.org/10.1101/2023.11.11.566649'> 10.1101/2023.11.11.566649</a></p>
        <p class="abstract">Signaling pathways that drive gene expression are typically depicted as having a dozen or so landmark phosphorylation and transcriptional events. In reality, thousands of dynamic post-translational modifications (PTMs) orchestrate nearly every cellular function, and we lack technologies to find causal links between these vast biochemical pathways and genetic circuits at scale. Here, we describe &#34;signaling-to-transcription network&#34; mapping through the development of PTM-centric base editing coupled to phenotypic screens, directed by temporally-resolved phosphoproteomics. Using T cell activation as a model, we observe hundreds of unstudied phosphorylation sites that modulate NFAT transcriptional activity. We identify the phosphorylation-mediated nuclear localization of the phosphatase PHLPP1 which promotes NFAT but inhibits NF{kappa}B activity. We also find that specific phosphosite mutants can alter gene expression in subtle yet distinct patterns, demonstrating the potential for fine-tuning transcriptional responses. Overall, base editor screening of PTM sites provides a powerful platform to dissect PTM function within signaling pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569407">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569407" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569407">
        <p class="paperTitle">Chronic metabolic stress drives developmental programs and loss of tissue functions in non-transformed liver that mirror tumor states and stratify survival</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569407" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569407" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tzouanas, C. N.; Sherman, M. S.; Shay, J. E. S.; Rubin, A. J.; Mead, B. E.; Dao, T. T.; Butzlaff, T.; Mana, M. D.; Kolb, K. E.; Walesky, C.; Pepe-Mooney, B. J.; Smith, C. J.; Prakadan, S. M.; Ramseier, M. L.; Tong, E. Y.; Joung, J.; Chi, F.; McMahon-Skates, T.; Winston, C. L.; Jeong, W.-J.; Aney, K. J.; Chen, E.; Nissim, S.; Zhang, F.; Deshpande, V.; Lauer, G. M.; Yilmaz, O. H.; Goessling, W.; Shalek, A. K.</p>
        <p class="info">Score: 2.5, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569407' target='https://doi.org/10.1101/2023.11.30.569407'> 10.1101/2023.11.30.569407</a></p>
        <p class="abstract">Under chronic stress, cells must balance competing demands between cellular survival and tissue function. In metabolic dysfunction-associated steatotic liver disease (MASLD, formerly NAFLD/NASH), hepatocytes cooperate with structural and immune cells to perform crucial metabolic, synthetic, and detoxification functions despite nutrient imbalances. While prior work has emphasized stress-induced drivers of cell death, the dynamic adaptations of surviving cells and their functional repercussions remain unclear. Namely, we do not know which pathways and programs define cellular responses, what regulatory factors mediate (mal)adaptations, and how this aberrant activity connects to tissue-scale dysfunction and long-term disease outcomes. Here, by applying longitudinal single-cell multi-omics to a mouse model of chronic metabolic stress and extending to human cohorts, we show that stress drives survival-linked tradeoffs and metabolic rewiring, manifesting as shifts towards development-associated states in non-transformed hepatocytes with accompanying decreases in their professional functionality. Diet-induced adaptations occur significantly prior to tumorigenesis but parallel tumorigenesis-induced phenotypes and predict worsened human cancer survival. Through the development of a multi-omic computational gene regulatory inference framework and human in vitro and mouse in vivo genetic perturbations, we validate transcriptional (RELB, SOX4) and metabolic (HMGCS2) mediators that co-regulate and couple the balance between developmental state and hepatocyte functional identity programming. Our work defines cellular features of liver adaptation to chronic stress as well as their links to long-term disease outcomes and cancer hallmarks, unifying diverse axes of cellular dysfunction around core causal mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569969">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569969" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569969">
        <p class="paperTitle">An evolution-based framework for describing human gut bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569969" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569969" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Doran, B. A.; Chen, R. Y.; Giba, H.; Behera, V.; Barat, B.; Sundararajan, A.; Lin, H.; Sidebottom, A.; Pamer, E. G.; Raman, A.</p>
        <p class="info">Score: 2.2, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569969' target='https://doi.org/10.1101/2023.12.04.569969'> 10.1101/2023.12.04.569969</a></p>
        <p class="abstract">The human gut microbiome contains many bacterial strains of the same species (&#34;strain-level variants&#34;). Describing strains in a biologically meaningful way rather than purely taxonomically is an important goal but challenging due to the genetic complexity of strain-level variation. Here, we measured patterns of co-evolution across &gt;7,000 strains spanning the bacterial tree-of-life. Using these patterns as a prior for studying hundreds of gut commensal strains that we isolated, sequenced, and metabolically profiled revealed widespread structure beneath the phylogenetic level of species. Defining strains by their co-evolutionary signatures enabled predicting their metabolic phenotypes and engineering consortia from strain genome content alone. Our findings demonstrate a biologically relevant organization to strain-level variation and motivate a new schema for describing bacterial strains based on their evolutionary history.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568853">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568853" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568853">
        <p class="paperTitle">Proneural - Mesenchymal antagonism dominates the patterns of phenotypic heterogeneity in Glioblastoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568853" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568853" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: BV, H.; Jolly, M. K.</p>
        <p class="info">Score: 2.4, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568853' target='https://doi.org/10.1101/2023.11.27.568853'> 10.1101/2023.11.27.568853</a></p>
        <p class="abstract">1The aggressive nature of glioblastoma (GBM) - one of the deadliest forms of brain tumours - is majorly attributed to underlying phenotypic heterogeneity. Early attempts to classify this heterogeneity at a transcriptomic level in TCGA GBM cohort proposed the existence of four distinct molecular subtypes: Proneural, Neural, Classical and Mesenchymal. Further, a single-cell RNA-seq analysis of primary tumours also reported similar 4 subtypes mimicking neuro-developmental lineages. However, it remains unclear whether these 4 subtypes identified via bulk and single-cell transcriptomics are mutually exclusive or not. Here, we perform pairwise correlations among individual genes and gene signatures corresponding to these proposed subtypes, and show that the subtypes are not distinctly mutually antagonistic in either TCGA or single-cell RNA-sequencing data. We observed that the proneural (or neural progenitor-like) - mesenchymal axis is the most prominent antagonistic pair, with the other two subtypes lying on this spectrum. These results are reinforced through a meta-analysis of over 100 single-cell and bulk transcriptomic datasets as well as in terms of functional association with metabolic switching, cell cycle and immune evasion pathways. These results suggest rethinking GBM phenotypic characterization for more effective therapeutic targeting efforts.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
