<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Widespread transcriptional regulation from within transcribed regions in plants
Authors: Voichek, Y.; Hristova, G.; Molla-Morales, A.; Weigel, D.; Nordborg, M.
Score: 108.5, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557872
Animals and plants have evolved separately for over 1.5 billion years and independently invented multicellularity, suggesting a possible divergence in their mode of transcription regulation. Here, we set out to elucidate fundamental features of transcription regulatory sequences in plants. Using a massively parallel reporter assay in four species, we show that sequences downstream of the transcription start site (TSS) play a major role in controlling transcription.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Widespread transcriptional regulation from within transcribed regions in plants
Authors: Voichek, Y.; Hristova, G.; Molla-Morales, A.; Weigel, D.; Nordborg, M.
Score: 108.5, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557872
Animals and plants have evolved separately for over 1.5 billion years and independently invented multicellularity, suggesting a possible divergence in their mode of transcription regulation. Here, we set out to elucidate fundamental features of transcription regulatory sequences in plants. Using a massively parallel reporter assay in four species, we show that sequences downstream of the transcription start site (TSS) play a major role in controlling transcription." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Widespread transcriptional regulation from within transcribed regions in plants
Authors: Voichek, Y.; Hristova, G.; Molla-Morales, A.; Weigel, D.; Nordborg, M.
Score: 108.5, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557872
Animals and plants have evolved separately for over 1.5 billion years and independently invented multicellularity, suggesting a possible divergence in their mode of transcription regulation. Here, we set out to elucidate fundamental features of transcription regulatory sequences in plants. Using a massively parallel reporter assay in four species, we show that sequences downstream of the transcription start site (TSS) play a major role in controlling transcription."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Widespread transcriptional regulation from within transcribed regions in plants\nAuthors: Voichek, Y.; Hristova, G.; Molla-Morales, A.; Weigel, D.; Nordborg, M.\nScore: 108.5, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557872\nAnimals and plants have evolved separately for over 1.5 billion years and independently invented multicellularity, suggesting a possible divergence in their mode of transcription regulation. Here, we set out to elucidate fundamental features of transcription regulatory sequences in plants. Using a massively parallel reporter assay in four species, we show that sequences downstream of the transcription start site (TSS) play a major role in controlling transcription.",
  "keywords": [
    
  ],
  "articleBody": " Widespread transcriptional regulation from within transcribed regions in plants\nAuthors: Voichek, Y.; Hristova, G.; Molla-Morales, A.; Weigel, D.; Nordborg, M.\nScore: 108.5, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557872\nAnimals and plants have evolved separately for over 1.5 billion years and independently invented multicellularity, suggesting a possible divergence in their mode of transcription regulation. Here, we set out to elucidate fundamental features of transcription regulatory sequences in plants. Using a massively parallel reporter assay in four species, we show that sequences downstream of the transcription start site (TSS) play a major role in controlling transcription. Swapping regulatory sequences from one side of the TSS to the other yields different outcomes, unlike animal enhancers which act independently of their position. A GATC-containing DNA motif, positioned downstream of the TSS, was sufficient to enhance gene expression in a dose-dependent manner. Its effect on gene expression was tissue-dependent and conserved across vascular plants. These results identify a unique characteristic of transcriptional regulation in plants, and suggest fundamental differences in gene regulation might exist between plants and animals.\nDigitize your Biology! Modeling multicellular systems through interpretable cell behavior\nAuthors: Johnson, J. A. I.; Stein-O'Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N.; Zheng, L.; Zimmerman, J. W.; Jaffee, E. M.; Chang, Y. H.; Coussens, L.; Gray, J.; Heiser, L. M.; Fertig, E. J.; Macklin, P.\nScore: 22.8, Published: 2023-09-17 DOI: 10.1101/2023.09.17.557982\nCells are fundamental units of life. Recent technical advances have revolutionized our ability to quantify the state and identity of individual cells, and intercellular regulatory programs. However, these static measurements alone are limited in their ability to predict the complex collective behaviors that emerge from populations of many interacting cells over time. Mathematical models have a proven record of successfully predicting the behaviors of dynamic biological systems, e.g., therapeutic responses in cancer. Simulations from these models enable in silico visualization, examination, and refinement of biological models and can be used to generate new hypotheses about cells and their collective behavior. Agent-based modeling is particularly well-suited to studying communities of interacting cells, as it is intuitive to map a single cell to a single agent. Thus, we have developed a conceptual framing (with a reference implementation in the widely-used PhysiCell agent-based modeling framework) that can be initialized directly from single cell and spatial transcriptomic data, and that can be easily populated with interactive rules. Because the expert mathematical and computational knowledge required to build agent-based models has limited their widespread adoption in the biomedical research community, we engineered this framework to specify complex cellular responses to signals (or stimuli) using a single line of human-readable text. This plain language text encodes cellular phenotypes and regulatory mechanisms from high throughput data and published literature, using a novel concept of hypothesis grammar. We motivate and fully describe this grammar and its philosophy, and then present a series of five example reference models of tumor growth and response to immunotherapy. Biologically, these examples demonstrate how mathematical models can predict from single cell and spatial transcriptomic data the cellular phenotypes responsible for tumor cell invasion and the simulation of immunotherapy treatment to overcome tumor cell growth. Computationally, these examples are designed to demonstrate how this conceptual framing and software implementation empower interdisciplinary teams to build an agent-based model of their experimental system, leveraging prior biological knowledge alone or in combination with information from spatial multiomics technologies. Altogether, this approach provides an interface to bridge biological, clinical, and systems biology researchers for mathematical modeling of biological systems at scale, allowing the community to extrapolate from single-cell characterization to emergent multicellular behavior.\nSystematic Profiling of Ale Yeast Protein Dynamics across Fermentation and Repitching\nAuthors: Garge, R. K.; Geck, R. C.; Armstrong, J. O.; Dunn, B.; Boutz, D. R.; Battenhouse, A.; Leutert, M.; Dang, V.; Jiang, P.; Kwiatkowski, D.; Peiser, T.; McElroy, H.; Marcotte, E. M.; Dunham, M. J.\nScore: 16.2, Published: 2023-09-22 DOI: 10.1101/2023.09.21.558736\nStudying the genetic and molecular characteristics of brewing yeast strains is crucial for understanding their domestication history and adaptations accumulated over time in fermentation environments, and for guiding optimizations to the brewing process itself. Saccharomyces cerevisiae (brewing yeast) is amongst the most profiled organisms on the planet, yet the temporal molecular changes that underlie industrial fermentation and beer brewing remain understudied. Here, we characterized the genomic makeup of a Saccharomyces cerevisiae ale yeast widely used in the production of Hefeweizen beers, and applied shotgun mass spectrometry to systematically measure the proteomic changes throughout two fermentation cycles which were separated by 14 rounds of serial repitching. The resulting brewing yeast proteomics resource includes 64,740 protein abundance measurements. We found that this strain possesses typical genetic characteristics of Saccharomyces cerevisiae ale strains and displayed progressive shifts in molecular processes during fermentation based on protein abundance changes. We observed protein abundance differences between early fermentation batches compared to those separated by 14 rounds of serial repitching. The observed abundance differences occurred mainly in proteins involved in the metabolism of ergosterol and isobutyraldehyde. Our systematic profiling serves as a starting point for deeper characterization of how the yeast proteome changes during commercial fermentations and additionally serves as a resource to guide fermentation protocols, strain handling, and engineering practices in commercial brewing and fermentation environments. Finally, we created a web interface (https://brewing-yeast-proteomics.ccbb.utexas.edu/) to serve as a valuable resource for yeast geneticists, brewers, and biochemists to provide insights into the global trends underlying commercial beer production.\nApplication of genome-scale models of metabolism and expression to the simulation and design of recombinant organisms\nAuthors: Oftadeh, O.; Hatzimanikatis, V.\nScore: 5.5, Published: 2023-09-15 DOI: 10.1101/2023.09.13.557522\nThe production of recombinant proteins in a host using synthetic constructs such as plasmids comes at the cost of detrimental effects such as reduced growth, energetic inefficiencies, and other stress responses, collectively known as metabolic stress. Increasing the number of copies of the foreign gene increases the metabolic load but increases the expression of the foreign protein. Thus, there is a trade-off between biomass and product yield in response to changes in heterologous gene copy number. This work proposes a computational method, rETFL (recombinant Expression and Thermodynamic Flux), for analyzing and predicting the responses of recombinant organisms to the introduction of synthetic constructs. rETFL is an extension to the ETFL formulations designed to reconstruct models of metabolism and expression (ME-models). We have illustrated the capabilities of the method in four studies to (i) capture the growth reduction in plasmid-containing E. coli and recombinant protein production; (ii) explore the trade-off between biomass and product yield as plasmid copy number is varied; (iii) predict the emergence of overflow metabolism in recombinant E. coli in agreement with experimental data; and (iv) investigate the individual pathways and enzymes affected by the presence of the plasmid. We anticipate that rETFL will serve as a comprehensive platform for integrating available omics data for recombinant organisms and making context-specific predictions that can help optimize recombinant expression systems for biopharmaceutical production and gene therapy.\nDesmosome mutations impact the tumor microenvironment to promote melanoma proliferation\nAuthors: Baron, M.; Tagore, M.; Wall, P.; Zheng, F.; Barkley, D.; Yanai, I.; Yang, J.; Kiuru, M.; White, R.; Ideker, T.\nScore: 4.3, Published: 2023-09-22 DOI: 10.1101/2023.09.19.558457\nDesmosomes are transmembrane protein complexes that contribute to cell-cell adhesion in epithelia and other tissues. Here, we report the discovery of frequent genetic alterations in the desmosome in human cancers, with the strongest signal seen in cutaneous melanoma where desmosomes are mutated in over 70% of cases. In primary but not metastatic melanoma biopsies, the burden of coding mutations on desmosome genes associates with a strong reduction in desmosome gene expression. Analysis by spatial transcriptomics suggests that these expression decreases occur in keratinocytes in the microenvironment rather than in primary melanoma tumor cells. In further support of a microenvironmental origin, we find that loss-of-function knockdowns of the desmosome in keratinocytes yield markedly increased proliferation of adjacent melanocytes in keratinocyte/melanocyte co-cultures. Thus, gradual accumulation of desmosome mutations in neighboring cells may prime melanocytes for neoplastic transformation.\nSweetwater: an underrated crude glycerol for sustainable lipid production in non-conventional yeasts\nAuthors: Malika Keita, V.; Lee, Y. Q.; Lakshmanan, M.; Ow, D. S.-W.; Staniland, P.; Staniland, J.; Savill, I.; Tee, K. L.; Wong, T. S.; Lee, D.-Y.\nScore: 2.4, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558176\nSweetwater, a methanol-free by-product of the fat splitting process, is an emerging alternative feedstock for microbial utilization of crude glycerol. Yeasts are promising candidates in this context due to their versatility in delivering a wide range of value-added products including lipids. To enhance sweetwater utilization, we investigated the growth and lipid production of 21 oleaginous yeast strains in sweetwater and found that nutrient limitation and the unique carbon composition of sweetwater boosted significant lipid accumulation in several strains, in particular Rhodosporidium toruloides NRRL Y-6987. To decipher the underlying mechanism, the transcriptomic changes of R. toluroides NRRL Y-6987 were further analyzed, indicating potential sugars and oligopeptides in sweetwater supporting growth and lipid accumulation as well as exogenous fatty acid uptake leading to the enhanced lipid accumulation. Our comparative study successfully demonstrated sweetwater as a cost-effective feedstock and suggested potential sweetwater type and strain engineering targets for increasing microbial lipid production. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=139 SRC=\"FIGDIR/small/558176v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (42K): org.highwire.dtl.DTLVardef@1514e2borg.highwire.dtl.DTLVardef@e2fb2aorg.highwire.dtl.DTLVardef@1d79a72org.highwire.dtl.DTLVardef@1dccf02_HPS_FORMAT_FIGEXP M_FIG C_FIG\nEscherichia coli sister cells dynamically organize developmental rosettes\nAuthors: Puri, D.; Allison, K. R.\nScore: 2.0, Published: 2023-09-18 DOI: 10.1101/2023.09.18.557913\nRosettes are self-organizing, circular multicellular communities that initiate the development of organs and embryos in higher organisms. Their formation results from the active repositioning of adhered sister cells and is thought to distinguish multicellular organisms form unicellular ones (1). Though biofilms have been argued to be the result of microbial development (2), bacterial rosettes are uncommon and largely uncharacterized (3-7). Here we report that Escherichia coli rosettes form by a consistent and novel mechanism. After division, sister cells \"folded\" to reposition themselves in parallel. After the second division, cells folded again to produce clonal 4-cell, quatrefoil rosettes. Analysis of folding revealed that it followed an angular random walk, comprised of [~]1-m angular strokes and directional randomization. We further showed that this motion was produced by the flagellum, the extracellular tail whose rotation generates swimming motility. Rosette formation was found to require de novo flagella synthesis and the balancing of sister-cell adhesion and propulsion. Moreover, by interrupting sister-cell folding, we demonstrated that rosette formation was required for E. colis multicellular-chain life cycle and the resulting clonal-chain biofilms. This study establishes that a unicellular bacterium self-organizes developmental rosettes by active sister-cell repositioning, and has implications for the evolution of multicellularity as well as E. colis roles as a commensal and pathogenic organism.\nMulti-SNP combination genome-scale metabolic models identify molecular mechanisms determining genetic interactions in a quantitative trait\nAuthors: Sasikumar, S.; S, P. K.; Bhatt, N.; Sinha, H.\nScore: 2.6, Published: 2023-09-20 DOI: 10.1101/2023.09.13.557398\nElucidating genotype-phenotype or variant-to-function relationships remains a pivotal challenge in genetics. Specifically, understanding how genetic interactions among causal quantitative trait nucleotides (QTNs) drive phenotypic variation remains to be determined. Employing genome-scale metabolic modelling, we analysed the metabolic consequences of four causal QTNs, individually and in all combinations, responsible for modulating sporulation efficiency in yeast. Differential gene expression and network analysis revealed QTN interactions dependent changes in the connectivity of crucial metabolic regulators. Furthermore, QTN-specific metabolic models and genome-scale differential flux analysis pinpointed variations in flux across multiple metabolic pathways that could explain the observed sporulation efficiency differences. Finally, the differential regulation of the pentose phosphate pathway in specific QTN combinations suggested autophagy as a pentose phosphate pathway-dependent compensatory mechanism for yeast sporulation. Our study contributes to the evolving functional understanding of variants, providing insightful information into the mechanisms governing phenotypic variation.\nCommunity metabolic modeling of host-microbiota interactions through multi-objective optimization\nAuthors: Lambert, A.; Budinich, M.; Mahe, M.; Chaffron, S.; Eveillard, D.\nScore: 1.8, Published: 2023-09-16 DOI: 10.1101/2023.09.15.557910\nThe human gut microbiota comprises various microorganisms engaged in intricate interactions among themselves and with the host, affecting its health. While advancements in omics technologies have led to the inference of clear associations between microbiome composition and health conditions, we usually lack a causal and mechanistic understanding of these associations. For modeling mechanisms driving the interactions, we simulated the organisms metabolism using in silico Genome-Scale Metabolic Models (GEMs). We used multi-objective optimization to predict and explain metabolic interactions among gut microbes and an intestinal epithelial cell. We developed a score integrating model simulation results to predict the type (competition, neutralism, mutualism) and quantify the interaction between several organisms. This framework uncovered a potential cross-feeding for choline, explaining the predicted mutualism between Lactobacillus rhamnosus GG and the epithelial cell. Finally, we analyzed a five-organism ecosystem, revealing that a minimal microbiota can favor the epithelial cells maintenance.\nSystems genetics analysis of human body fat distribution genes identifies Wnt signaling and mitochondrial activity in adipocytes\nAuthors: Reed, J. N.; Huang, J.; Li, Y.; Ma, L.; Banka, D.; Wabitsch, M.; Wang, T.; Ding, W.; Bjorkegren, J. L. M.; Civelek, M.\nScore: 2.5, Published: 2023-09-06 DOI: 10.1101/2023.09.06.556534\nBACKGROUNDExcess fat in the abdomen is a sexually dimorphic risk factor for cardio-metabolic disease. The relative storage between abdominal and lower-body subcutaneous adipose tissue depots is approximated by the waist-to-hip ratio adjusted for body mass index (WHRadjBMI). Genome-wide association studies (GWAS) identified 346 loci near 495 genes associated with WHRadjBMI. Most of these genes have unknown roles in fat distribution, but many are expressed and putatively act in adipose tissue. We aimed to identify novel sex- and depot-specific drivers of WHRadjBMI using a systems genetics approach. METHODSWe used two independent cohorts of adipose tissue gene expression with 362 - 444 males and 147 - 219 females, primarily of European ancestry. We constructed sex- and depot- specific Bayesian networks to model the gene-gene interactions from 8,492 adipose tissue genes. Key driver analysis identified genes that, in silico and putatively in vitro, regulate many others, including the 495 WHRadjBMI GWAS genes. Key driver gene function was determined by perturbing their expression in human subcutaneous pre-adipocytes using lenti-virus or siRNA. RESULTS51 - 119 key drivers in each network were replicated in both cohorts. We used single-cell expression data to select replicated key drivers expressed in adipocyte precursors and mature adipocytes, prioritized genes which have not been previously studied in adipose tissue, and used public human and mouse data to nominate 53 novel key driver genes (10 - 21 from each network) that may regulate fat distribution by altering adipocyte function. In other cell types, 23 of these genes are found in crucial adipocyte pathways: Wnt signaling or mitochondrial function. We selected seven genes whose expression is highly correlated with WHRadjBMI to further study their effects on adipogenesis/Wnt signaling (ANAPC2, PSME3, RSPO1, TYRO3) or mitochondrial function (C1QTNF3, MIGA1, PSME3, UBR1). Adipogenesis was inhibited in cells overexpressing ANAPC2 and RSPO1 compared to controls. RSPO1 results are consistent with a positive correlation between gene expression in the subcutaneous depot and WHRadjBMI, therefore lower relative storage in the subcutaneous depot. RSPO1 inhibited adipogenesis by increasing {beta}-catenin activation and Wnt-related transcription, thus repressing PPARG and CEBPA. PSME3 overexpression led to more adipogenesis than controls. In differentiated adipocytes, MIGA1 and UBR1 downregulation led to mitochondrial dysfunction, with lower oxygen consumption than controls; MIGA1 knockdown also lowered UCP1 expression. SUMMARYANAPC2, MIGA1, PSME3, RSPO1, and UBR1 affect adipocyte function and may drive body fat distribution.\n",
  "wordCount" : "2574",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557872">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557872" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557872">
        <p class="paperTitle">Widespread transcriptional regulation from within transcribed regions in plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557872" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557872" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Voichek, Y.; Hristova, G.; Molla-Morales, A.; Weigel, D.; Nordborg, M.</p>
        <p class="info">Score: 108.5, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557872' target='https://doi.org/10.1101/2023.09.15.557872'> 10.1101/2023.09.15.557872</a></p>
        <p class="abstract">Animals and plants have evolved separately for over 1.5 billion years and independently invented multicellularity, suggesting a possible divergence in their mode of transcription regulation. Here, we set out to elucidate fundamental features of transcription regulatory sequences in plants. Using a massively parallel reporter assay in four species, we show that sequences downstream of the transcription start site (TSS) play a major role in controlling transcription. Swapping regulatory sequences from one side of the TSS to the other yields different outcomes, unlike animal enhancers which act independently of their position. A GATC-containing DNA motif, positioned downstream of the TSS, was sufficient to enhance gene expression in a dose-dependent manner. Its effect on gene expression was tissue-dependent and conserved across vascular plants. These results identify a unique characteristic of transcriptional regulation in plants, and suggest fundamental differences in gene regulation might exist between plants and animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.557982">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.557982" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.557982">
        <p class="paperTitle">Digitize your Biology! Modeling multicellular systems through interpretable cell behavior</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.557982" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.557982" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, J. A. I.; Stein-O&#39;Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N.; Zheng, L.; Zimmerman, J. W.; Jaffee, E. M.; Chang, Y. H.; Coussens, L.; Gray, J.; Heiser, L. M.; Fertig, E. J.; Macklin, P.</p>
        <p class="info">Score: 22.8, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.557982' target='https://doi.org/10.1101/2023.09.17.557982'> 10.1101/2023.09.17.557982</a></p>
        <p class="abstract">Cells are fundamental units of life. Recent technical advances have revolutionized our ability to quantify the state and identity of individual cells, and intercellular regulatory programs. However, these static measurements alone are limited in their ability to predict the complex collective behaviors that emerge from populations of many interacting cells over time. Mathematical models have a proven record of successfully predicting the behaviors of dynamic biological systems, e.g., therapeutic responses in cancer. Simulations from these models enable in silico visualization, examination, and refinement of biological models and can be used to generate new hypotheses about cells and their collective behavior. Agent-based modeling is particularly well-suited to studying communities of interacting cells, as it is intuitive to map a single cell to a single agent. Thus, we have developed a conceptual framing (with a reference implementation in the widely-used PhysiCell agent-based modeling framework) that can be initialized directly from single cell and spatial transcriptomic data, and that can be easily populated with interactive rules. Because the expert mathematical and computational knowledge required to build agent-based models has limited their widespread adoption in the biomedical research community, we engineered this framework to specify complex cellular responses to signals (or stimuli) using a single line of human-readable text. This plain language text encodes cellular phenotypes and regulatory mechanisms from high throughput data and published literature, using a novel concept of hypothesis grammar. We motivate and fully describe this grammar and its philosophy, and then present a series of five example reference models of tumor growth and response to immunotherapy. Biologically, these examples demonstrate how mathematical models can predict from single cell and spatial transcriptomic data the cellular phenotypes responsible for tumor cell invasion and the simulation of immunotherapy treatment to overcome tumor cell growth. Computationally, these examples are designed to demonstrate how this conceptual framing and software implementation empower interdisciplinary teams to build an agent-based model of their experimental system, leveraging prior biological knowledge alone or in combination with information from spatial multiomics technologies. Altogether, this approach provides an interface to bridge biological, clinical, and systems biology researchers for mathematical modeling of biological systems at scale, allowing the community to extrapolate from single-cell characterization to emergent multicellular behavior.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558736">
        <p class="paperTitle">Systematic Profiling of Ale Yeast Protein Dynamics across Fermentation and Repitching</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garge, R. K.; Geck, R. C.; Armstrong, J. O.; Dunn, B.; Boutz, D. R.; Battenhouse, A.; Leutert, M.; Dang, V.; Jiang, P.; Kwiatkowski, D.; Peiser, T.; McElroy, H.; Marcotte, E. M.; Dunham, M. J.</p>
        <p class="info">Score: 16.2, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558736' target='https://doi.org/10.1101/2023.09.21.558736'> 10.1101/2023.09.21.558736</a></p>
        <p class="abstract">Studying the genetic and molecular characteristics of brewing yeast strains is crucial for understanding their domestication history and adaptations accumulated over time in fermentation environments, and for guiding optimizations to the brewing process itself. Saccharomyces cerevisiae (brewing yeast) is amongst the most profiled organisms on the planet, yet the temporal molecular changes that underlie industrial fermentation and beer brewing remain understudied. Here, we characterized the genomic makeup of a Saccharomyces cerevisiae ale yeast widely used in the production of Hefeweizen beers, and applied shotgun mass spectrometry to systematically measure the proteomic changes throughout two fermentation cycles which were separated by 14 rounds of serial repitching. The resulting brewing yeast proteomics resource includes 64,740 protein abundance measurements. We found that this strain possesses typical genetic characteristics of Saccharomyces cerevisiae ale strains and displayed progressive shifts in molecular processes during fermentation based on protein abundance changes. We observed protein abundance differences between early fermentation batches compared to those separated by 14 rounds of serial repitching. The observed abundance differences occurred mainly in proteins involved in the metabolism of ergosterol and isobutyraldehyde. Our systematic profiling serves as a starting point for deeper characterization of how the yeast proteome changes during commercial fermentations and additionally serves as a resource to guide fermentation protocols, strain handling, and engineering practices in commercial brewing and fermentation environments. Finally, we created a web interface (https://brewing-yeast-proteomics.ccbb.utexas.edu/) to serve as a valuable resource for yeast geneticists, brewers, and biochemists to provide insights into the global trends underlying commercial beer production.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557522">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557522" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557522">
        <p class="paperTitle">Application of genome-scale models of metabolism and expression to the simulation and design of recombinant organisms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557522" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557522" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oftadeh, O.; Hatzimanikatis, V.</p>
        <p class="info">Score: 5.5, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557522' target='https://doi.org/10.1101/2023.09.13.557522'> 10.1101/2023.09.13.557522</a></p>
        <p class="abstract">The production of recombinant proteins in a host using synthetic constructs such as plasmids comes at the cost of detrimental effects such as reduced growth, energetic inefficiencies, and other stress responses, collectively known as metabolic stress. Increasing the number of copies of the foreign gene increases the metabolic load but increases the expression of the foreign protein. Thus, there is a trade-off between biomass and product yield in response to changes in heterologous gene copy number. This work proposes a computational method, rETFL (recombinant Expression and Thermodynamic Flux), for analyzing and predicting the responses of recombinant organisms to the introduction of synthetic constructs. rETFL is an extension to the ETFL formulations designed to reconstruct models of metabolism and expression (ME-models). We have illustrated the capabilities of the method in four studies to (i) capture the growth reduction in plasmid-containing E. coli and recombinant protein production; (ii) explore the trade-off between biomass and product yield as plasmid copy number is varied; (iii) predict the emergence of overflow metabolism in recombinant E. coli in agreement with experimental data; and (iv) investigate the individual pathways and enzymes affected by the presence of the plasmid. We anticipate that rETFL will serve as a comprehensive platform for integrating available omics data for recombinant organisms and making context-specific predictions that can help optimize recombinant expression systems for biopharmaceutical production and gene therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558457">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558457" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558457">
        <p class="paperTitle">Desmosome mutations impact the tumor microenvironment to promote melanoma proliferation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558457" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558457" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baron, M.; Tagore, M.; Wall, P.; Zheng, F.; Barkley, D.; Yanai, I.; Yang, J.; Kiuru, M.; White, R.; Ideker, T.</p>
        <p class="info">Score: 4.3, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558457' target='https://doi.org/10.1101/2023.09.19.558457'> 10.1101/2023.09.19.558457</a></p>
        <p class="abstract">Desmosomes are transmembrane protein complexes that contribute to cell-cell adhesion in epithelia and other tissues. Here, we report the discovery of frequent genetic alterations in the desmosome in human cancers, with the strongest signal seen in cutaneous melanoma where desmosomes are mutated in over 70% of cases. In primary but not metastatic melanoma biopsies, the burden of coding mutations on desmosome genes associates with a strong reduction in desmosome gene expression. Analysis by spatial transcriptomics suggests that these expression decreases occur in keratinocytes in the microenvironment rather than in primary melanoma tumor cells. In further support of a microenvironmental origin, we find that loss-of-function knockdowns of the desmosome in keratinocytes yield markedly increased proliferation of adjacent melanocytes in keratinocyte/melanocyte co-cultures. Thus, gradual accumulation of desmosome mutations in neighboring cells may prime melanocytes for neoplastic transformation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558176">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558176" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558176">
        <p class="paperTitle">Sweetwater: an underrated crude glycerol for sustainable lipid production in non-conventional yeasts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558176" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558176" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Malika Keita, V.; Lee, Y. Q.; Lakshmanan, M.; Ow, D. S.-W.; Staniland, P.; Staniland, J.; Savill, I.; Tee, K. L.; Wong, T. S.; Lee, D.-Y.</p>
        <p class="info">Score: 2.4, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558176' target='https://doi.org/10.1101/2023.09.18.558176'> 10.1101/2023.09.18.558176</a></p>
        <p class="abstract">Sweetwater, a methanol-free by-product of the fat splitting process, is an emerging alternative feedstock for microbial utilization of crude glycerol. Yeasts are promising candidates in this context due to their versatility in delivering a wide range of value-added products including lipids. To enhance sweetwater utilization, we investigated the growth and lipid production of 21 oleaginous yeast strains in sweetwater and found that nutrient limitation and the unique carbon composition of sweetwater boosted significant lipid accumulation in several strains, in particular Rhodosporidium toruloides NRRL Y-6987. To decipher the underlying mechanism, the transcriptomic changes of R. toluroides NRRL Y-6987 were further analyzed, indicating potential sugars and oligopeptides in sweetwater supporting growth and lipid accumulation as well as exogenous fatty acid uptake leading to the enhanced lipid accumulation. Our comparative study successfully demonstrated sweetwater as a cost-effective feedstock and suggested potential sweetwater type and strain engineering targets for increasing microbial lipid production.

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=139 SRC=&#34;FIGDIR/small/558176v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (42K):
org.highwire.dtl.DTLVardef@1514e2borg.highwire.dtl.DTLVardef@e2fb2aorg.highwire.dtl.DTLVardef@1d79a72org.highwire.dtl.DTLVardef@1dccf02_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.557913">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.557913" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.557913">
        <p class="paperTitle">Escherichia coli sister cells dynamically organize developmental rosettes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.557913" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.557913" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Puri, D.; Allison, K. R.</p>
        <p class="info">Score: 2.0, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.557913' target='https://doi.org/10.1101/2023.09.18.557913'> 10.1101/2023.09.18.557913</a></p>
        <p class="abstract">Rosettes are self-organizing, circular multicellular communities that initiate the development of organs and embryos in higher organisms. Their formation results from the active repositioning of adhered sister cells and is thought to distinguish multicellular organisms form unicellular ones (1). Though biofilms have been argued to be the result of microbial development (2), bacterial rosettes are uncommon and largely uncharacterized (3-7). Here we report that Escherichia coli rosettes form by a consistent and novel mechanism. After division, sister cells &#34;folded&#34; to reposition themselves in parallel. After the second division, cells folded again to produce clonal 4-cell, quatrefoil rosettes. Analysis of folding revealed that it followed an angular random walk, comprised of [~]1-m angular strokes and directional randomization. We further showed that this motion was produced by the flagellum, the extracellular tail whose rotation generates swimming motility. Rosette formation was found to require de novo flagella synthesis and the balancing of sister-cell adhesion and propulsion. Moreover, by interrupting sister-cell folding, we demonstrated that rosette formation was required for E. colis multicellular-chain life cycle and the resulting clonal-chain biofilms. This study establishes that a unicellular bacterium self-organizes developmental rosettes by active sister-cell repositioning, and has implications for the evolution of multicellularity as well as E. colis roles as a commensal and pathogenic organism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557398">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557398" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557398">
        <p class="paperTitle">Multi-SNP combination genome-scale metabolic models identify molecular mechanisms determining genetic interactions in a quantitative trait</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557398" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557398" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sasikumar, S.; S, P. K.; Bhatt, N.; Sinha, H.</p>
        <p class="info">Score: 2.6, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557398' target='https://doi.org/10.1101/2023.09.13.557398'> 10.1101/2023.09.13.557398</a></p>
        <p class="abstract">Elucidating genotype-phenotype or variant-to-function relationships remains a pivotal challenge in genetics. Specifically, understanding how genetic interactions among causal quantitative trait nucleotides (QTNs) drive phenotypic variation remains to be determined. Employing genome-scale metabolic modelling, we analysed the metabolic consequences of four causal QTNs, individually and in all combinations, responsible for modulating sporulation efficiency in yeast. Differential gene expression and network analysis revealed QTN interactions dependent changes in the connectivity of crucial metabolic regulators. Furthermore, QTN-specific metabolic models and genome-scale differential flux analysis pinpointed variations in flux across multiple metabolic pathways that could explain the observed sporulation efficiency differences. Finally, the differential regulation of the pentose phosphate pathway in specific QTN combinations suggested autophagy as a pentose phosphate pathway-dependent compensatory mechanism for yeast sporulation. Our study contributes to the evolving functional understanding of variants, providing insightful information into the mechanisms governing phenotypic variation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557910">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557910" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557910">
        <p class="paperTitle">Community metabolic modeling of host-microbiota interactions through multi-objective optimization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557910" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557910" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lambert, A.; Budinich, M.; Mahe, M.; Chaffron, S.; Eveillard, D.</p>
        <p class="info">Score: 1.8, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557910' target='https://doi.org/10.1101/2023.09.15.557910'> 10.1101/2023.09.15.557910</a></p>
        <p class="abstract">The human gut microbiota comprises various microorganisms engaged in intricate interactions among themselves and with the host, affecting its health. While advancements in omics technologies have led to the inference of clear associations between microbiome composition and health conditions, we usually lack a causal and mechanistic understanding of these associations. For modeling mechanisms driving the interactions, we simulated the organisms metabolism using in silico Genome-Scale Metabolic Models (GEMs). We used multi-objective optimization to predict and explain metabolic interactions among gut microbes and an intestinal epithelial cell. We developed a score integrating model simulation results to predict the type (competition, neutralism, mutualism) and quantify the interaction between several organisms. This framework uncovered a potential cross-feeding for choline, explaining the predicted mutualism between Lactobacillus rhamnosus GG and the epithelial cell. Finally, we analyzed a five-organism ecosystem, revealing that a minimal microbiota can favor the epithelial cells maintenance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556534">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556534" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556534">
        <p class="paperTitle">Systems genetics analysis of human body fat distribution genes identifies Wnt signaling and mitochondrial activity in adipocytes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556534" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556534" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reed, J. N.; Huang, J.; Li, Y.; Ma, L.; Banka, D.; Wabitsch, M.; Wang, T.; Ding, W.; Bjorkegren, J. L. M.; Civelek, M.</p>
        <p class="info">Score: 2.5, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556534' target='https://doi.org/10.1101/2023.09.06.556534'> 10.1101/2023.09.06.556534</a></p>
        <p class="abstract">BACKGROUNDExcess fat in the abdomen is a sexually dimorphic risk factor for cardio-metabolic disease. The relative storage between abdominal and lower-body subcutaneous adipose tissue depots is approximated by the waist-to-hip ratio adjusted for body mass index (WHRadjBMI). Genome-wide association studies (GWAS) identified 346 loci near 495 genes associated with WHRadjBMI. Most of these genes have unknown roles in fat distribution, but many are expressed and putatively act in adipose tissue. We aimed to identify novel sex- and depot-specific drivers of WHRadjBMI using a systems genetics approach.

METHODSWe used two independent cohorts of adipose tissue gene expression with 362 - 444 males and 147 - 219 females, primarily of European ancestry. We constructed sex- and depot- specific Bayesian networks to model the gene-gene interactions from 8,492 adipose tissue genes. Key driver analysis identified genes that, in silico and putatively in vitro, regulate many others, including the 495 WHRadjBMI GWAS genes. Key driver gene function was determined by perturbing their expression in human subcutaneous pre-adipocytes using lenti-virus or siRNA.

RESULTS51 - 119 key drivers in each network were replicated in both cohorts. We used single-cell expression data to select replicated key drivers expressed in adipocyte precursors and mature adipocytes, prioritized genes which have not been previously studied in adipose tissue, and used public human and mouse data to nominate 53 novel key driver genes (10 - 21 from each network) that may regulate fat distribution by altering adipocyte function. In other cell types, 23 of these genes are found in crucial adipocyte pathways: Wnt signaling or mitochondrial function. We selected seven genes whose expression is highly correlated with WHRadjBMI to further study their effects on adipogenesis/Wnt signaling (ANAPC2, PSME3, RSPO1, TYRO3) or mitochondrial function (C1QTNF3, MIGA1, PSME3, UBR1).

Adipogenesis was inhibited in cells overexpressing ANAPC2 and RSPO1 compared to controls. RSPO1 results are consistent with a positive correlation between gene expression in the subcutaneous depot and WHRadjBMI, therefore lower relative storage in the subcutaneous depot. RSPO1 inhibited adipogenesis by increasing {beta}-catenin activation and Wnt-related transcription, thus repressing PPARG and CEBPA. PSME3 overexpression led to more adipogenesis than controls. In differentiated adipocytes, MIGA1 and UBR1 downregulation led to mitochondrial dysfunction, with lower oxygen consumption than controls; MIGA1 knockdown also lowered UCP1 expression.

SUMMARYANAPC2, MIGA1, PSME3, RSPO1, and UBR1 affect adipocyte function and may drive body fat distribution.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
