<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Escalating High-dimensional Imaging using Combinatorial Channel Multiplexing and Deep Learning
Authors: Ben-Uri, R.; Ben Shabat, L.; Bar-Tal, O.; Bussi, Y.; Maimon, N.; Keidar Haran, T.; Milo, I.; Elhanani, O.; Rochwarger, A.; Schürch, C. M.; Bagon, S.; Keren, L.
Score: 22.2, Published: 2023-09-12 DOI: 10.1101/2023.09.09.556962
Understanding tissue structure and function requires tools that quantify the expression of multiple proteins at single-cell resolution while preserving spatial information. Current imaging technologies use a separate channel for each individual protein, inherently limiting their throughput and scalability.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Escalating High-dimensional Imaging using Combinatorial Channel Multiplexing and Deep Learning
Authors: Ben-Uri, R.; Ben Shabat, L.; Bar-Tal, O.; Bussi, Y.; Maimon, N.; Keidar Haran, T.; Milo, I.; Elhanani, O.; Rochwarger, A.; Schürch, C. M.; Bagon, S.; Keren, L.
Score: 22.2, Published: 2023-09-12 DOI: 10.1101/2023.09.09.556962
Understanding tissue structure and function requires tools that quantify the expression of multiple proteins at single-cell resolution while preserving spatial information. Current imaging technologies use a separate channel for each individual protein, inherently limiting their throughput and scalability." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Escalating High-dimensional Imaging using Combinatorial Channel Multiplexing and Deep Learning
Authors: Ben-Uri, R.; Ben Shabat, L.; Bar-Tal, O.; Bussi, Y.; Maimon, N.; Keidar Haran, T.; Milo, I.; Elhanani, O.; Rochwarger, A.; Schürch, C. M.; Bagon, S.; Keren, L.
Score: 22.2, Published: 2023-09-12 DOI: 10.1101/2023.09.09.556962
Understanding tissue structure and function requires tools that quantify the expression of multiple proteins at single-cell resolution while preserving spatial information. Current imaging technologies use a separate channel for each individual protein, inherently limiting their throughput and scalability."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Escalating High-dimensional Imaging using Combinatorial Channel Multiplexing and Deep Learning\nAuthors: Ben-Uri, R.; Ben Shabat, L.; Bar-Tal, O.; Bussi, Y.; Maimon, N.; Keidar Haran, T.; Milo, I.; Elhanani, O.; Rochwarger, A.; Schürch, C. M.; Bagon, S.; Keren, L.\nScore: 22.2, Published: 2023-09-12 DOI: 10.1101/2023.09.09.556962\nUnderstanding tissue structure and function requires tools that quantify the expression of multiple proteins at single-cell resolution while preserving spatial information. Current imaging technologies use a separate channel for each individual protein, inherently limiting their throughput and scalability.",
  "keywords": [
    
  ],
  "articleBody": " Escalating High-dimensional Imaging using Combinatorial Channel Multiplexing and Deep Learning\nAuthors: Ben-Uri, R.; Ben Shabat, L.; Bar-Tal, O.; Bussi, Y.; Maimon, N.; Keidar Haran, T.; Milo, I.; Elhanani, O.; Rochwarger, A.; Schürch, C. M.; Bagon, S.; Keren, L.\nScore: 22.2, Published: 2023-09-12 DOI: 10.1101/2023.09.09.556962\nUnderstanding tissue structure and function requires tools that quantify the expression of multiple proteins at single-cell resolution while preserving spatial information. Current imaging technologies use a separate channel for each individual protein, inherently limiting their throughput and scalability. Here, we present CombPlex (COMBinatorial multiPLEXing), a combinatorial staining platform coupled with an algorithmic framework to exponentially increase the number of proteins that can be measured from C up to 2C - 1, and is applicable to any mass spectrometry-based or fluorescence-based microscopy platform. In CombPlex, every protein can be imaged in several channels, and every channel contains agglomerated images of several proteins. These combinatorically-compressed images are then decompressed to individual protein-images using deep learning and optimization. We perform feasibility experiments in silico and achieve accurate (F1=0.98, R=0.99) reconstruction for compressing the stains of twenty-two proteins to five imaging channels. We test our approach experimentally and obtain accurate (F1=0.97, R=0.93) images of seven proteins using three channels, both in fluorescence microscopy and in mass-based imaging. We demonstrate that combinatorial staining coupled with deep-learning decompression can serve to escalate the number of proteins measured using any imaging modality, without the need for specialized instrumentation. Coupling CombPlex with instruments for high-dimensional imaging could pave the way to image hundreds of proteins at single-cell resolution in intact tissue sections.\nLow dimensionality of phenotypic space as an emergent property of coordinated teams in biological regulatory networks\nAuthors: Hari, K.; Harlapur, P.; Saxena, A.; Haldar, K.; Girish, A.; Malpani, T.; Levine, H.; Jolly, M. K.\nScore: 17.6, Published: 2023-09-06 DOI: 10.1101/2023.02.03.526930\nBiological networks driving cell-fate decisions involve complex interactions, but they often give rise to only a few phenotypes, thus exhibiting low-dimensional dynamics. The network design principles that govern such cell-fate canalization remain unclear. Here, we investigate networks across diverse biological contexts- Epithelial-Mesenchymal Transition, Small Cell Lung Cancer, and Gonadal cell-fate determination - to reveal that the presence of two mutually antagonistic, well-coordinated teams of nodes leads to low-dimensional phenotypic space such that the first principal component (PC1) axis can capture most of the variance. Further analysis of artificial team-based networks and random counterparts of biological networks reveals that the principal component decomposition is determined by the team strength within these networks, demonstrating how the underlying network structure governs PC1 variance. The presence of low dimensionality in corresponding transcriptomic data confirms the applicability of our observations. We propose that team-based topology in biological networks are critical for generating a cell-fate canalization landscape.\nOn standardization of controls in lifespan studies\nAuthors: Spiridonova, O.; Kriukov, D.; Nemirovich-Danchenko, N.; Peshkin, L.\nScore: 24.8, Published: 2023-08-18 DOI: 10.1101/2023.08.17.552381\nThe search for interventions to slow down and even reverse aging is a burgeoning field. The literature cites hundreds of supposedly beneficial pharmacological and genetic interventions in model organisms: mice, rats, flies and worms, where research into physiology is routinely accompanied by lifespan data. Naturally the negative results are more frequent, yet scientifically quite valuable if analyzed systematically. Yet, there is a strong \"discovery bias\", i.e. results of interventions which turn out not to be beneficial remain unpublished. Theoretically, all lifespan data is ripe for re-analysis: we could contrast the molecular targets and pathways across studies and help focus the further search for interventions. Alas, the results of most longevity studies are difficult to compare. This is in part because there are no clear, universally accepted standards for conducting such experiments or even for reporting such data. The situation is worsened by the fact that the authors often do not describe experimental conditions completely. As a result, works on longevity make up a set of precedents, each of which might be interesting in its own right, yet incoherent and incomparable. Here we point out specific issues and propose solutions for quality control by checking both inter- and intra-study consistency of lifespan data.\nLongevity interventions in Titan mice attenuate frailty and senescence accumulation\nAuthors: Gille, B.; Muller-Eigner, A.; Gottschalk, S.; Wytrwat, E.; Langhammer, M.; Peleg, S.\nScore: 9.6, Published: 2023-09-12 DOI: 10.1101/2023.09.08.556694\nWild-type murine models for aging research have lifespans of several years, which results in long experimental duration and late output. Here we explore the short-lived non-inbred Titan mouse as a mouse model to test longevity interventions. We show that Titan mice exhibit increased frailty and cellular senescence at an early age. Dietary intervention attenuates the frailty progression of Titan mice. Additionally, cyclic administration of the senolytic drug Navitoclax at early age increases the lifespan and reduces cellular senescence. Our data suggests that Titan mice can serve as a cost-effective and timely model for longevity interventions in mammals.\nAgent-based modeling of tumor-immune interactions reveals determinants of final tumor states\nAuthors: Ahmidouch, M.; Tangella, N.; Finley, S. D.\nScore: 8.8, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556617\nInteractions between tumor and immune cells in the tumor microenvironment (TME) influence tumor growth and the tumors response to treatment. Excitingly, this complex landscape of tumor-immune interactions can be studied using computational modeling. Mathematical oncology can provide quantitative insights into the TME, serving as a framework for understanding tumor dynamics. Here, we use an agent-based model to simulate the interactions among cancer cells, macrophages (naive, M1, and M2), and T cells (active CD8+ and inactive) in a 2D representation of the TME. Key diffusible factors, IL-4 and IFN-{gamma}, are also incorporated. We apply the model to predict how cell-specific properties influence tumor progression. The model predictions and analyses revealed the relationships between different cell populations and highlighted the importance of macrophages and T cells in shaping the TME. Thus, we quantify how components of the TME influence the final tumor state and the effects of macrophage-based therapies. The findings emphasize the significant role of computational models in unraveling the intricate dynamics of tumor-immune interactions and their potential for guiding the development of tailored immunotherapeutic strategies. This study provides a foundation for future investigations aiming to refine and expand the model, validate predictions experimentally, and pave the way for improved cancer treatments.\nModelling count data with partial differential equation models in biology\nAuthors: Simpson, M. J.; Murphy, R. J.; Maclaren, O. J.\nScore: 6.5, Published: 2023-09-13 DOI: 10.1101/2023.09.09.556963\nPartial differential equation (PDE) models are often used to study biological phenomena involving movement-birth-death processes, including ecological population dynamics and the invasion of populations of biological cells. Count data, by definition, is non-negative, and count data relating to biological populations is often bounded above by some carrying capacity that arises through biological competition for space or nutrients. Parameter estimation, parameter identifiability, and model prediction usually involves working with a measurement error model explicitly relating experimental measurements with the solution of a mathematical model, and in biological applications this is typically achieved by assuming the data are normally distributed about the solution of the mathematical model. Despite the widespread use of the standard additive Gaussian measurement error model, the assumptions inherent in this approach are rarely tested or compared with other options. Here we interpret scratch assay data, involving migration, proliferation and delays in a population of cancer cells using a reaction-diffusion PDE model. Experimental measurements are related to the PDE solution using a standard additive Gaussian measurement error model and we compare results with a biologically motivated binomial measurement error model. While estimates of model parameters are relatively insensitive to the choice of measurement error model, we find that model predictions are very sensitive. The standard additive Gaussian measurement error model leads to biologically inconsistent predictions, such as negative counts as well as counts that exceed the carrying capacity across a relatively large spatial region within the experiment. Furthermore, the standard additive Gaussian measurement error model is more computationally expensive than the binomial measurement error model. In contrast, the binomial measurement error model is both simpler to implement becuase it involves one less parameter to estimate, and the binomial measurement error model leads to biologically plausible predictions. We provide open access Julia software on GitHub to replicate all calculations in this work, and we explain how to generalise our approach to deal with coupled PDE models with several dependent variables through a multinomial measurement error model.\nMixOmics Integration of Biological Datasets Identifies Highly Correlated Key Variables of COVID-19 severity.\nAuthors: Harriott, N. C.; Chimenti, M. S.; Ryan, A. L.\nScore: 1.8, Published: 2023-09-15 DOI: 10.1101/2023.09.14.557558\nBackground: Despite several years since the COVID-19 pandemic was declared, challenges remain in understanding the factors that can predict the severity of COVID-19 disease and complications of SARS-CoV-2 infection. While many large-scale Multiomic datasets have been published, integration of these datasets has the potential to substantially increase the biological insight gained allowing a more complex comprehension of the disease pathogenesis. Such insight may improve our ability to predict disease progression, detect severe cases more rapidly and develop effective therapeutics. Methods: In this study we have applied an innovative machine learning algorithm to delineate COVID-severity based on integration of paired samples of proteomic and transcriptomic data from a small cohort of patients testing positive for SARS-CoV-2 infection with differential disease severity. Targeted plasma proteomics and an onco-immune targeted transcriptomic panel was performed on sequential samples from a cohort of 23 severe, 21 moderate and 10 mild COVID-19 patients. We applied DIABLO, a new integrative method, to identify multi-omics biomarker panels that can discriminate between multiple phenotypic groups, such as the varied severity of disease in COVID-19 patients. Results: As COVID-19 severity is known among our sample group, we can train models using this as the outcome variable and calculate features that are important predictors of severe disease. In this study, we detect highly correlated key variables of severe COVID-19 using transcriptomic discriminant analysis and multi-omics integration methods. Conclusions: This approach highlights the power of data integration from a small cohort of patients offering a better biological understanding of the molecular mechanisms driving COVID-19 severity and an opportunity to improve prediction of disease trajectories and targeted therapeutics.\nLarge-scale computational modelling of the M1 and M2 synovial macrophages in Rheumatoid Arthritis\nAuthors: Zerrouk, N.; Alcraft, R.; Hall, B.; Auge, F.; Niarakis, A.\nScore: 1.5, Published: 2023-09-15 DOI: 10.1101/2023.09.11.556838\nMacrophages play an essential role in rheumatoid arthritis (RA). Depending on their phenotype (M1 or M2), they can play a role in the initiation or resolution of inflammation. The M1/M2 ratio in RA is higher than in healthy controls. Despite this, no treatment targeting specifically macrophages is currently used in clinics. Thus, devising strategies to selectively deplete proinflammatory macrophages and promote anti-inflammatory macrophages could be a promising therapeutic approach in RA. State-of-the-art molecular interaction maps of M1 and M2 macrophages in rheumatoid arthritis are available and represent a dense source of knowledge; however, these maps remain limited by their static nature. Discrete dynamic modelling can be employed to study the emergent behaviours of these systems. Nevertheless, handling such large-scale models is challenging. Due to their massive size, it is computationally demanding to identify biologically relevant states in a cell- and disease-specific context. In this work, we developed an efficient computational framework that converts molecular interaction maps into Boolean models using the CaSQ tool. Next, we use a newly developed BMA tool version deployed to a high-performance computing cluster to identify the models' steady states. The identified attractors are then validated using gene expression datasets and prior knowledge. We successfully applied our framework to generate and calibrate the first RA M1 and M2 macrophage Boolean models. Using single and double knockout simulations, we identified NFkB, JAK1/JAK2, and ERK1/Notch1 as potential targets that could selectively suppress proinflammatory macrophages, and GSK3B as a promising target that could promote anti-inflammatory macrophages in RA.\nDissecting Metabolic Landscape of Alveolar Macrophage\nAuthors: Malla, S.; Sajeevan, K. A.; Acharya, B.; Chowdhury, R.; Saha, R.\nScore: 1.5, Published: 2023-09-12 DOI: 10.1101/2023.09.08.556783\nThe highly plastic nature of Alveolar Macrophage (AM) plays a crucial role in the defense against inhaled particulates and pathogens in the lungs. Depending upon the signal, AM acquires either classically activated M1 phenotype or alternatively activated M2 phenotype. These phenotypes have specific functions and unique metabolic traits such as upregulated glycolysis and pentose phosphate pathway in M1 phase and enhanced oxidative phosphorylation and tricarboxylic acid cycle during M2 phase that help maintain the sterility of the lungs. In this study, we investigate the metabolic shift in the activated phases of AM (M1 and M2 phase) and highlight the roles of pathways other than the typical players of central carbon metabolism. Pathogenesis is a complex and elongated process where the heightened requirement for energy is matched by metabolic shifts that supplement immune response and maintain homeostasis. The first step of pathogenesis is fever; however, analyzing the role of physical parameters such as temperature is challenging. Here, we observe the effect of an increase in temperature on pathways such as glycolysis, pentose phosphate pathway, oxidative phosphorylation, tricarboxylic acid cycle, amino acid metabolism, and leukotriene metabolism. We report the role of temperature as a catalyst to the immune response of the cell. The activity of pathways such as pyruvate metabolism, arachidonic acid metabolism, chondroitin/heparan sulfate biosynthesis, and heparan sulfate degradation are found to be important driving forces in the M1/M2 phenotype. We have also identified a list of 34 reactions such as nitric oxide production from arginine and the conversion of glycogenin to UDP which play major roles in the metabolic models and prompt the shift of the M2 phenotype to M1 and vice versa. In future, these reactions could further be probed as major contributors in designing effective therapeutic targets against severe respiratory diseases. Author SummaryAlveolar macrophage (AM) is highly plastic in nature and has a wide range of functions including invasion/killing of bacteria to maintaining the homeostasis in the lungs. The regulatory mechanism involved in the alveolar macrophage polarization is essential to fight against severe respiratory conditions (pathogens and particulates). Over the years, experiments on mouse/rat models have been used to draw insightful inferences. However, recent advances have highlighted the lack of transmission from non-human models to successful in vivo human experiments. Hence using genome-scale metabolic (GSM) models to understand the unique metabolic traits of human alveolar macrophages and comprehend the complex metabolic underpinnings that govern the polarization can lead to novel therapeutic strategies. The GSM models of AMs thus far, has not incorporated the activated phases of AM. Here, we aim to exhaustively dissect the metabolic landscape and capabilities of AM in its healthy and activated stages. We carefully explore the changes in reaction fluxes under each of the conditions to understand the role and function of all the pathways with special attention to pathways away from central carbon metabolism. Understanding the characteristics of each phase of AM has applications that could help improve the therapeutic approaches against respiratory conditions.\nMechanisms of Cell Size Regulation in Slow-Growing Escherichia coli Cells: Discriminating Models Beyond the Adder\nAuthors: Nieto, C.; Pedraza, J. M.; Vargas-Garcia, C.; Singh, A.\nScore: 1.4, Published: 2023-09-15 DOI: 10.1101/2023.09.11.557238\nUnder ideal conditions, Escherichia coli cells divide after adding a fixed cell size, a strategy known as the adder. This concept applies to various microbes and is often explained as the division that occurs after a certain number of stages, associated with the accumulation of precursor proteins at a rate proportional to cell size. However, under poor media conditions, E. coli cells exhibit a different size regulation. They are smaller and follow a sizer-like division strategy where the added size is inversely proportional to the size at birth. We explore three potential causes for this deviation: precursor protein degradation, nonlinear accumulation rate, and a threshold size termed the commitment size. These models fit the mean trends but predict different distributions given the birth size. To validate these models, we used the Akaike information criterion and compared them to open datasets of slow-growing E. coli cells in different media. the degradation model could explain the division strategy for media where cells are larger, while the commitment size model could account for smaller cells. The power-law model, finally, better fits the data at intermediate regimes.\n",
  "wordCount" : "2679",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556962" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556962">
        <p class="paperTitle">Escalating High-dimensional Imaging using Combinatorial Channel Multiplexing and Deep Learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ben-Uri, R.; Ben Shabat, L.; Bar-Tal, O.; Bussi, Y.; Maimon, N.; Keidar Haran, T.; Milo, I.; Elhanani, O.; Rochwarger, A.; Schürch, C. M.; Bagon, S.; Keren, L.</p>
        <p class="info">Score: 22.2, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556962' target='https://doi.org/10.1101/2023.09.09.556962'> 10.1101/2023.09.09.556962</a></p>
        <p class="abstract">Understanding tissue structure and function requires tools that quantify the expression of multiple proteins at single-cell resolution while preserving spatial information. Current imaging technologies use a separate channel for each individual protein, inherently limiting their throughput and scalability. Here, we present CombPlex (COMBinatorial multiPLEXing), a combinatorial staining platform coupled with an algorithmic framework to exponentially increase the number of proteins that can be measured from C up to 2C - 1, and is applicable to any mass spectrometry-based or fluorescence-based microscopy platform. In CombPlex, every protein can be imaged in several channels, and every channel contains agglomerated images of several proteins. These combinatorically-compressed images are then decompressed to individual protein-images using deep learning and optimization. We perform feasibility experiments in silico and achieve accurate (F1=0.98, R=0.99) reconstruction for compressing the stains of twenty-two proteins to five imaging channels. We test our approach experimentally and obtain accurate (F1=0.97, R=0.93) images of seven proteins using three channels, both in fluorescence microscopy and in mass-based imaging. We demonstrate that combinatorial staining coupled with deep-learning decompression can serve to escalate the number of proteins measured using any imaging modality, without the need for specialized instrumentation. Coupling CombPlex with instruments for high-dimensional imaging could pave the way to image hundreds of proteins at single-cell resolution in intact tissue sections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.02.03.526930">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.02.03.526930" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.02.03.526930">
        <p class="paperTitle">Low dimensionality of phenotypic space as an emergent property of coordinated teams in biological regulatory networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.02.03.526930" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.02.03.526930" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hari, K.; Harlapur, P.; Saxena, A.; Haldar, K.; Girish, A.; Malpani, T.; Levine, H.; Jolly, M. K.</p>
        <p class="info">Score: 17.6, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.02.03.526930' target='https://doi.org/10.1101/2023.02.03.526930'> 10.1101/2023.02.03.526930</a></p>
        <p class="abstract">Biological networks driving cell-fate decisions involve complex interactions, but they often give rise to only a few phenotypes, thus exhibiting low-dimensional dynamics. The network design principles that govern such cell-fate canalization remain unclear. Here, we investigate networks across diverse biological contexts- Epithelial-Mesenchymal Transition, Small Cell Lung Cancer, and Gonadal cell-fate determination - to reveal that the presence of two mutually antagonistic, well-coordinated teams of nodes leads to low-dimensional phenotypic space such that the first principal component (PC1) axis can capture most of the variance. Further analysis of artificial team-based networks and random counterparts of biological networks reveals that the principal component decomposition is determined by the team strength within these networks, demonstrating how the underlying network structure governs PC1 variance. The presence of low dimensionality in corresponding transcriptomic data confirms the applicability of our observations. We propose that team-based topology in biological networks are critical for generating a cell-fate canalization landscape.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.552381">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.552381" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.552381">
        <p class="paperTitle">On standardization of controls in lifespan studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.552381" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.552381" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Spiridonova, O.; Kriukov, D.; Nemirovich-Danchenko, N.; Peshkin, L.</p>
        <p class="info">Score: 24.8, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.552381' target='https://doi.org/10.1101/2023.08.17.552381'> 10.1101/2023.08.17.552381</a></p>
        <p class="abstract">The search for interventions to slow down and even reverse aging is a burgeoning field. The literature cites hundreds of supposedly beneficial pharmacological and genetic interventions in model organisms: mice, rats, flies and worms, where research into physiology is routinely accompanied by lifespan data. Naturally the negative results are more frequent, yet scientifically quite valuable if analyzed systematically. Yet, there is a strong &#34;discovery bias&#34;, i.e. results of interventions which turn out not to be beneficial remain unpublished. Theoretically, all lifespan data is ripe for re-analysis: we could contrast the molecular targets and pathways across studies and help focus the further search for interventions. Alas, the results of most longevity studies are difficult to compare. This is in part because there are no clear, universally accepted standards for conducting such experiments or even for reporting such data. The situation is worsened by the fact that the authors often do not describe experimental conditions completely. As a result, works on longevity make up a set of precedents, each of which might be interesting in its own right, yet incoherent and incomparable. Here we point out specific issues and propose solutions for quality control by checking both inter- and intra-study consistency of lifespan data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556694">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556694" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556694">
        <p class="paperTitle">Longevity interventions in Titan mice attenuate frailty and senescence accumulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556694" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556694" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gille, B.; Muller-Eigner, A.; Gottschalk, S.; Wytrwat, E.; Langhammer, M.; Peleg, S.</p>
        <p class="info">Score: 9.6, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556694' target='https://doi.org/10.1101/2023.09.08.556694'> 10.1101/2023.09.08.556694</a></p>
        <p class="abstract">Wild-type murine models for aging research have lifespans of several years, which results in long experimental duration and late output. Here we explore the short-lived non-inbred Titan mouse as a mouse model to test longevity interventions. We show that Titan mice exhibit increased frailty and cellular senescence at an early age. Dietary intervention attenuates the frailty progression of Titan mice. Additionally, cyclic administration of the senolytic drug Navitoclax at early age increases the lifespan and reduces cellular senescence. Our data suggests that Titan mice can serve as a cost-effective and timely model for longevity interventions in mammals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556617">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556617" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556617">
        <p class="paperTitle">Agent-based modeling of tumor-immune interactions reveals determinants of final tumor states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556617" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556617" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahmidouch, M.; Tangella, N.; Finley, S. D.</p>
        <p class="info">Score: 8.8, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556617' target='https://doi.org/10.1101/2023.09.06.556617'> 10.1101/2023.09.06.556617</a></p>
        <p class="abstract">Interactions between tumor and immune cells in the tumor microenvironment (TME) influence tumor growth and the tumors response to treatment. Excitingly, this complex landscape of tumor-immune interactions can be studied using computational modeling. Mathematical oncology can provide quantitative insights into the TME, serving as a framework for understanding tumor dynamics. Here, we use an agent-based model to simulate the interactions among cancer cells, macrophages (naive, M1, and M2), and T cells (active CD8&#43; and inactive) in a 2D representation of the TME. Key diffusible factors, IL-4 and IFN-{gamma}, are also incorporated. We apply the model to predict how cell-specific properties influence tumor progression. The model predictions and analyses revealed the relationships between different cell populations and highlighted the importance of macrophages and T cells in shaping the TME. Thus, we quantify how components of the TME influence the final tumor state and the effects of macrophage-based therapies. The findings emphasize the significant role of computational models in unraveling the intricate dynamics of tumor-immune interactions and their potential for guiding the development of tailored immunotherapeutic strategies. This study provides a foundation for future investigations aiming to refine and expand the model, validate predictions experimentally, and pave the way for improved cancer treatments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556963" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556963">
        <p class="paperTitle">Modelling count data with partial differential equation models in biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Simpson, M. J.; Murphy, R. J.; Maclaren, O. J.</p>
        <p class="info">Score: 6.5, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556963' target='https://doi.org/10.1101/2023.09.09.556963'> 10.1101/2023.09.09.556963</a></p>
        <p class="abstract">Partial differential equation (PDE) models are often used to study biological phenomena involving movement-birth-death processes, including ecological population dynamics and the invasion of populations of biological cells. Count data, by definition, is non-negative, and count data relating to biological populations is often bounded above by some carrying capacity that arises through biological competition for space or nutrients. Parameter estimation, parameter identifiability, and model prediction usually involves working with a measurement error model explicitly relating experimental measurements with the solution of a mathematical model, and in biological applications this is typically achieved by assuming the data are normally distributed about the solution of the mathematical model. Despite the widespread use of the standard additive Gaussian measurement error model, the assumptions inherent in this approach are rarely tested or compared with other options. Here we interpret scratch assay data, involving migration, proliferation and delays in a population of cancer cells using a reaction-diffusion PDE model. Experimental measurements are related to the PDE solution using a standard additive Gaussian measurement error model and we compare results with a biologically motivated binomial measurement error model. While estimates of model parameters are relatively insensitive to the choice of measurement error model, we find that model predictions are very sensitive. The standard additive Gaussian measurement error model leads to biologically inconsistent predictions, such as negative counts as well as counts that exceed the carrying capacity across a relatively large spatial region within the experiment. Furthermore, the standard additive Gaussian measurement error model is more computationally expensive than the binomial measurement error model. In contrast, the binomial measurement error model is both simpler to implement becuase it involves one less parameter to estimate, and the binomial measurement error model leads to biologically plausible predictions. We provide open access Julia software on GitHub to replicate all calculations in this work, and we explain how to generalise our approach to deal with coupled PDE models with several dependent variables through a multinomial measurement error model.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557558">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557558" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557558">
        <p class="paperTitle">MixOmics Integration of Biological Datasets Identifies Highly Correlated Key Variables of COVID-19 severity.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557558" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557558" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harriott, N. C.; Chimenti, M. S.; Ryan, A. L.</p>
        <p class="info">Score: 1.8, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557558' target='https://doi.org/10.1101/2023.09.14.557558'> 10.1101/2023.09.14.557558</a></p>
        <p class="abstract">Background: Despite several years since the COVID-19 pandemic was declared, challenges remain in understanding the factors that can predict the severity of COVID-19 disease and complications of SARS-CoV-2 infection. While many large-scale Multiomic datasets have been published, integration of these datasets has the potential to substantially increase the biological insight gained allowing a more complex comprehension of the disease pathogenesis. Such insight may improve our ability to predict disease progression, detect severe cases more rapidly and develop effective therapeutics. Methods: In this study we have applied an innovative machine learning algorithm to delineate COVID-severity based on integration of paired samples of proteomic and transcriptomic data from a small cohort of patients testing positive for SARS-CoV-2 infection with differential disease severity. Targeted plasma proteomics and an onco-immune targeted transcriptomic panel was performed on sequential samples from a cohort of 23 severe, 21 moderate and 10 mild COVID-19 patients. We applied DIABLO, a new integrative method, to identify multi-omics biomarker panels that can discriminate between multiple phenotypic groups, such as the varied severity of disease in COVID-19 patients. Results: As COVID-19 severity is known among our sample group, we can train models using this as the outcome variable and calculate features that are important predictors of severe disease. In this study, we detect highly correlated key variables of severe COVID-19 using transcriptomic discriminant analysis and multi-omics integration methods. Conclusions: This approach highlights the power of data integration from a small cohort of patients offering a better biological understanding of the molecular mechanisms driving COVID-19 severity and an opportunity to improve prediction of disease trajectories and targeted therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.556838">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.556838" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.556838">
        <p class="paperTitle">Large-scale computational modelling of the M1 and M2 synovial macrophages in Rheumatoid Arthritis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.556838" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.556838" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zerrouk, N.; Alcraft, R.; Hall, B.; Auge, F.; Niarakis, A.</p>
        <p class="info">Score: 1.5, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.556838' target='https://doi.org/10.1101/2023.09.11.556838'> 10.1101/2023.09.11.556838</a></p>
        <p class="abstract">Macrophages play an essential role in rheumatoid arthritis (RA). Depending on their phenotype (M1 or M2), they can play a role in the initiation or resolution of inflammation. The M1/M2 ratio in RA is higher than in healthy controls. Despite this, no treatment targeting specifically macrophages is currently used in clinics. Thus, devising strategies to selectively deplete proinflammatory macrophages and promote anti-inflammatory macrophages could be a promising therapeutic approach in RA. State-of-the-art molecular interaction maps of M1 and M2 macrophages in rheumatoid arthritis are available and represent a dense source of knowledge; however, these maps remain limited by their static nature. Discrete dynamic modelling can be employed to study the emergent behaviours of these systems. Nevertheless, handling such large-scale models is challenging. Due to their massive size, it is computationally demanding to identify biologically relevant states in a cell- and disease-specific context. In this work, we developed an efficient computational framework that converts molecular interaction maps into Boolean models using the CaSQ tool. Next, we use a newly developed BMA tool version deployed to a high-performance computing cluster to identify the models&#39; steady states. The identified attractors are then validated using gene expression datasets and prior knowledge. We successfully applied our framework to generate and calibrate the first RA M1 and M2 macrophage Boolean models. Using single and double knockout simulations, we identified NFkB, JAK1/JAK2, and ERK1/Notch1 as potential targets that could selectively suppress proinflammatory macrophages, and GSK3B as a promising target that could promote anti-inflammatory macrophages in RA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556783">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556783" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556783">
        <p class="paperTitle">Dissecting Metabolic Landscape of Alveolar Macrophage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556783" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556783" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Malla, S.; Sajeevan, K. A.; Acharya, B.; Chowdhury, R.; Saha, R.</p>
        <p class="info">Score: 1.5, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556783' target='https://doi.org/10.1101/2023.09.08.556783'> 10.1101/2023.09.08.556783</a></p>
        <p class="abstract">The highly plastic nature of Alveolar Macrophage (AM) plays a crucial role in the defense against inhaled particulates and pathogens in the lungs. Depending upon the signal, AM acquires either classically activated M1 phenotype or alternatively activated M2 phenotype. These phenotypes have specific functions and unique metabolic traits such as upregulated glycolysis and pentose phosphate pathway in M1 phase and enhanced oxidative phosphorylation and tricarboxylic acid cycle during M2 phase that help maintain the sterility of the lungs. In this study, we investigate the metabolic shift in the activated phases of AM (M1 and M2 phase) and highlight the roles of pathways other than the typical players of central carbon metabolism. Pathogenesis is a complex and elongated process where the heightened requirement for energy is matched by metabolic shifts that supplement immune response and maintain homeostasis. The first step of pathogenesis is fever; however, analyzing the role of physical parameters such as temperature is challenging. Here, we observe the effect of an increase in temperature on pathways such as glycolysis, pentose phosphate pathway, oxidative phosphorylation, tricarboxylic acid cycle, amino acid metabolism, and leukotriene metabolism. We report the role of temperature as a catalyst to the immune response of the cell. The activity of pathways such as pyruvate metabolism, arachidonic acid metabolism, chondroitin/heparan sulfate biosynthesis, and heparan sulfate degradation are found to be important driving forces in the M1/M2 phenotype. We have also identified a list of 34 reactions such as nitric oxide production from arginine and the conversion of glycogenin to UDP which play major roles in the metabolic models and prompt the shift of the M2 phenotype to M1 and vice versa. In future, these reactions could further be probed as major contributors in designing effective therapeutic targets against severe respiratory diseases.

Author SummaryAlveolar macrophage (AM) is highly plastic in nature and has a wide range of functions including invasion/killing of bacteria to maintaining the homeostasis in the lungs. The regulatory mechanism involved in the alveolar macrophage polarization is essential to fight against severe respiratory conditions (pathogens and particulates). Over the years, experiments on mouse/rat models have been used to draw insightful inferences. However, recent advances have highlighted the lack of transmission from non-human models to successful in vivo human experiments. Hence using genome-scale metabolic (GSM) models to understand the unique metabolic traits of human alveolar macrophages and comprehend the complex metabolic underpinnings that govern the polarization can lead to novel therapeutic strategies. The GSM models of AMs thus far, has not incorporated the activated phases of AM. Here, we aim to exhaustively dissect the metabolic landscape and capabilities of AM in its healthy and activated stages. We carefully explore the changes in reaction fluxes under each of the conditions to understand the role and function of all the pathways with special attention to pathways away from central carbon metabolism. Understanding the characteristics of each phase of AM has applications that could help improve the therapeutic approaches against respiratory conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557238">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557238" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557238">
        <p class="paperTitle">Mechanisms of Cell Size Regulation in Slow-Growing Escherichia coli Cells: Discriminating Models Beyond the Adder</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557238" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557238" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nieto, C.; Pedraza, J. M.; Vargas-Garcia, C.; Singh, A.</p>
        <p class="info">Score: 1.4, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557238' target='https://doi.org/10.1101/2023.09.11.557238'> 10.1101/2023.09.11.557238</a></p>
        <p class="abstract">Under ideal conditions, Escherichia coli cells divide after adding a fixed cell size, a strategy known as the adder. This concept applies to various microbes and is often explained as the division that occurs after a certain number of stages, associated with the accumulation of precursor proteins at a rate proportional to cell size. However, under poor media conditions, E. coli cells exhibit a different size regulation. They are smaller and follow a sizer-like division strategy where the added size is inversely proportional to the size at birth. We explore three potential causes for this deviation: precursor protein degradation, nonlinear accumulation rate, and a threshold size termed the commitment size. These models fit the mean trends but predict different distributions given the birth size. To validate these models, we used the Akaike information criterion and compared them to open datasets of slow-growing E. coli cells in different media. the degradation model could explain the division strategy for media where cells are larger, while the commitment size model could account for smaller cells. The power-law model, finally, better fits the data at intermediate regimes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
