<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Exploiting fluctuations in gene expression to detect causal interactions between genes
Authors: Joly-Smith, E.; Talpur, M. M.; Allard, P.; Papazotos, F.; Potvin-Trottier, L.; Hilfinger, A.
Score: 10.1, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555799
Characterizing and manipulating cellular behaviour requires a mechanistic understanding of the causal interactions between cellular components. We present an approach that can detect causal interactions between genes without the need to perturb the physiological state of cells. This approach exploits naturally occurring cell-to-cell variability which is experimentally accessible from static population snapshots of genetically identical cells without the need to follow cells over time.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Exploiting fluctuations in gene expression to detect causal interactions between genes
Authors: Joly-Smith, E.; Talpur, M. M.; Allard, P.; Papazotos, F.; Potvin-Trottier, L.; Hilfinger, A.
Score: 10.1, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555799
Characterizing and manipulating cellular behaviour requires a mechanistic understanding of the causal interactions between cellular components. We present an approach that can detect causal interactions between genes without the need to perturb the physiological state of cells. This approach exploits naturally occurring cell-to-cell variability which is experimentally accessible from static population snapshots of genetically identical cells without the need to follow cells over time." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-06T10:39:07+00:00" />
<meta property="article:modified_time" content="2023-09-06T10:39:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Exploiting fluctuations in gene expression to detect causal interactions between genes
Authors: Joly-Smith, E.; Talpur, M. M.; Allard, P.; Papazotos, F.; Potvin-Trottier, L.; Hilfinger, A.
Score: 10.1, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555799
Characterizing and manipulating cellular behaviour requires a mechanistic understanding of the causal interactions between cellular components. We present an approach that can detect causal interactions between genes without the need to perturb the physiological state of cells. This approach exploits naturally occurring cell-to-cell variability which is experimentally accessible from static population snapshots of genetically identical cells without the need to follow cells over time."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Exploiting fluctuations in gene expression to detect causal interactions between genes\nAuthors: Joly-Smith, E.; Talpur, M. M.; Allard, P.; Papazotos, F.; Potvin-Trottier, L.; Hilfinger, A.\nScore: 10.1, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555799\nCharacterizing and manipulating cellular behaviour requires a mechanistic understanding of the causal interactions between cellular components. We present an approach that can detect causal interactions between genes without the need to perturb the physiological state of cells. This approach exploits naturally occurring cell-to-cell variability which is experimentally accessible from static population snapshots of genetically identical cells without the need to follow cells over time.",
  "keywords": [
    
  ],
  "articleBody": " Exploiting fluctuations in gene expression to detect causal interactions between genes\nAuthors: Joly-Smith, E.; Talpur, M. M.; Allard, P.; Papazotos, F.; Potvin-Trottier, L.; Hilfinger, A.\nScore: 10.1, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555799\nCharacterizing and manipulating cellular behaviour requires a mechanistic understanding of the causal interactions between cellular components. We present an approach that can detect causal interactions between genes without the need to perturb the physiological state of cells. This approach exploits naturally occurring cell-to-cell variability which is experimentally accessible from static population snapshots of genetically identical cells without the need to follow cells over time. Our main contribution is a simple mathematical relation that constrains the propagation of gene expression noise through biochemical reaction networks. This relation allows us to rigorously interpret fluctuation data even when only a small part of a complex gene regulatory process can be observed. This relation can be exploited to detect causal interactions by synthetically engineering a passive reporter of gene expression, akin to the established \"dual reporter assay\". While the focus of our contribution is theoretical, we also present an experimental proof-of-principle to illustrate the approach. Our data from synthetic gene regulatory networks in E. coli are not unequivocal but suggest that the method could prove useful in practice to identify causal interactions between genes from non-genetic cell-to-cell variability.\nThe interplay between lumen pressure and cell proliferation determines organoid morphology in a multicellular phase field model\nAuthors: Tanida, S.; Fuji, K.; Lu, L.; Guyomar, T.; Lee, B. H.; Honigmann, A.; Grapin-Botton, A.; Riveline, D.; Nonomura, M.; Sano, M.; Hiraiwa, T.\nScore: 13.8, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553655\nOrganoids are ideal systems to predict the phenotypes of organs. However, there is currently a lack of understanding regarding the generalized rules that enable use of simple cellular principles to make morphological predictions of entire organoids. Therefore, we employed a phase field model with the following basic components: the minimum conditions for the timing and volume of cell division, lumen nucleation rules, and lumenal pressure. Through our model, we could compute and generate a myriad of organoid phenotypes observed till date. We propose morphological indices necessary to characterize the shapes and construct phase diagrams and show their dependencies on proliferation time and lumen pressure. Additionally, we introduced the lumen-index parameter, which helped in examining the criteria to maintain organoids as spherical structures comprising a single layer of cells and enclosing an intact lumen. Finally, we predict a star-like organoid phenotype that did not undergo differentiation, suggesting that the volume constraint during cell division may determine the final phenotype. In summary, our approach provides researchers with guidelines to test the mechanisms of self-organization and predict the shape of organoid. Author summaryIn nature, a wide variety of organ morphologies are observed. Owing to the complexity of the process underlying the acquisition of organs morphology, it is challenging to investigate the mechanisms that lead to such variations. A promising approach to study these variations is the use of \"computational organoid\" study, which is the computational-based study of self-organizing shapes in multicellular assemblies and fluid-filled cavities called lumens that develop from a few proliferating cells. This study explores general mechanisms that dictate how various mechanical factors affect the growing self-organized multicellular assembly. We relied on computer simulations of the mathematical model called multicellular phase-field model with lumens and explored the mechanical factor effects, such as the lumen pressure while considering the time and volume conditions required for cell division. These simulations generated and categorized a wide range of organoid phenotypes based on the varying lumen pressure and cell division conditions. These phenotypes were characterized into seven distinct classes, based on the morphological index sets, including a cellular monolayer/multilayer surrounding single or multiple lumens and branch formation. These phenotypes were obtained without the assumption of differentiation. Our study elucidates the mechanisms underlying the organoid and organ formation with different shapes, thereby highlighting the significance of mechanical forces in shaping these complex biological structures.\nHuman subcutaneous and visceral adipocyte atlases uncover classical and specialized adipocytes and depot-specific patterns\nAuthors: Lazarescu, O.; Ziv Agam, M.; Haim, Y.; Hekselman, I.; Jubran, J.; Shneyour, A.; Kitsberg, D.; Levin, L.; Liberty, I. F.; Yoel, U.; Dukhno, O.; Adam, M.; Korner, A.; Murphy, R.; Bluher, M.; Habib, N.; Rudich, A.; Yeger-Lotem, E.\nScore: 8.2, Published: 2023-09-04 DOI: 10.1101/2023.09.04.555678\nHuman adipose depots are functionally distinct. Yet, recent single-nucleus RNA-sequencing (snRNA-seq) analyses largely uncovered overlapping/similar cell-type landscapes. We hypothesized that adipocytes subtypes, differentiation trajectories, and/or intercellular communication patterns could illuminate this depot similarity-difference gap. For this, we performed snRNA-seq of human subcutaneous and visceral adipose tissue. Whereas the majority of adipocytes in both depots were classical, namely enriched in lipid metabolism pathways, we also observed specialized adipocyte subtypes that were enriched in immune-related, extracellular matrix deposition (fibrosis), vascularization/angiogenesis, or ribosomal processes. Pseudo-temporal analysis suggested a developmental trajectory from adipose progenitor cells to classical adipocytes via specialized adipocytes, suggesting that the classical state stems from loss, rather than gain, of specialized functions. Lastly, intercellular communication routes were consistent with the different inflammatory tone of the two depots. Jointly, these findings provide a high-resolution view into the contribution of cellular composition, differentiation, and intercellular communication patterns to human fat depot differences.\nApproximate Bayesian Computation for Inferring Waddington Landscapes from Single Cell Data\nAuthors: Liu, Y.; Zhang, S. Y.; Kleijn, I. T.; Stumpf, M. P. H.\nScore: 7.9, Published: 2023-09-05 DOI: 10.1101/2023.09.03.556134\nSingle cell technologies allow us to gain insights into cellular processes at unprecedented resolution. In stem cell and developmental biology snap-shot data allows us to characterise how the transcriptional state of cells changes between successive cell types. Here we show how approximate Bayesian computation (ABC) can be employed to calibrate mathematical models against single cell data. In our simulation study we demonstrate the pivotal role of the adequate choice of distance measures appropriate for single cell data. We show that for good distance measures, notably optimal transport distances, we can infer parameters for mathematical models from simulated single cell data. We show that the ABC posteriors can be used to characterise parameter sensitivity and identify dependencies between different parameters, and to infer representations of the Waddington or epigenetic landscape, which forms a popular and interpretable representation of the developmental dynamics. In summary, these results pave the way for fitting mechanistic models of stem cell differentiation to single cell data.\nTranscription factors operate on a limited vocabulary of binding motifs in Arabidopsis thaliana\nAuthors: Zenker, S.; Wulf, D.; Meierhenrich, A.; Becker, S.; Eisenhut, M.; Stracke, R.; Weisshaar, B.; Braeutigam, A.\nScore: 8.7, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555073\nPredicting gene expression from promoter sequence requires understanding of the different signal integration points within a promoter. Sequence-specific transcription factors (TFs) binding to their cognate TF binding motifs control gene expression in eukaryotes by activating and repressing transcription. Their interplay generates complex expression patterns in reaction to environmental conditions and developmental cues. We hypothesized that signals are not only integrated by different TFs binding various positions in a promoter, but also by single TF binding motifs onto which multiple TFs can bind. Analyzing 2,190 binding motifs, we identified only 76 core TF binding motifs in plants. Twenty-one TF protein families act highly specific and bind a single conserved motif. Four TF families are classified as semi-conserved as they bind up to four motifs within a family, with divisions along phylogenetic groups. Five TF families bind diverse motifs. Expression analyses revealed high competition within TF families for the same binding motif. The results show that singular binding motifs act as signal integrators in plants where a combination of binding affinity and TF abundance likely determine the output.\nPTMNavigator: Interactive Visualization of Differentially Regulated Post-Translational Modifications in Cellular Signaling Pathways\nAuthors: Mueller, J.; Bayer, F. P.; Wilhelm, M.; Kuster, B.; The, M.\nScore: 2.0, Published: 2023-09-02 DOI: 10.1101/2023.08.31.555601\nPost-translational modifications (PTMs) play a governing role in regulating cellular signaling, fine-tuning protein function, and orchestrating complex biological processes. Despite their importance, the lack of comprehensive tools for studying PTMs from a pathway-centric perspective has limited our ability to understand how PTMs modulate cellular pathways on a molecular level. Here, we present PTMNavigator, a tool integrated into the ProteomicsDB platform, which offers an interactive interface for researchers to overlay experimental PTM data on top of ~3000 well-curated pathway diagrams. This offers a comprehensive view of the intricate relationship between PTMs and signaling pathways. To demonstrate the utility of PTMNavigator, we applied it to two phosphoproteomics perturbation datasets. First, PTMNavigator enhanced pathway enrichment analysis by showing how the regulated peptides are distributed in the pathways with high enrichment scores. Second, it visualized how drug treatments result in a discernable flow of PTM-driven signaling within pathways. Third, PTMNavigator aided in extending existing pathways by uncovering previously unknown relationships between PTMs and pathway components. By facilitating the discovery of novel PTM-pathway interactions and enhancing our understanding of cellular signaling dynamics, PTMNavigator advances our knowledge of PTM biology and its implications in health and disease.\nEvolution of hierarchical switching pattern in antigenic variation of Plasmodium falciparum under variable host immunity\nAuthors: Ghosh, B.; Priya, I. G.\nScore: 1.6, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555470\nThe var genes family encoding the variants of the erythrocyte membrane protein of Plasmodium falciparum is crucial for virulence of the parasite inside host. The transcriptional output of the var genes switches from one variant to other in a mutually exclusive fashion. It is proposed that a biased hierarchical switching pattern optimizes the growth and survival of the parasite inside the host. Apart from the hierarchical switching pattern, it is also well established that the intrinsic switching rates vary widely among the var genes. The centromeric protein like Var2csa is much more stable than the genes located at the telomeric and sub-telomeric regions of the chromosomes. In this study, we explored the evolutionary advantage achieved through selecting variable switching rates. Our theoretical analysis based on a mathematical model coupled with single cell RNA-seq data suggests that the variable switching rate is beneficial when cells expressing different variants are deferentially amenable to be cleared by the immune response. In fact, the variants which are cleared by the immune systems more efficiently are more stably expressed compared to a variant attacked by the immune system much less vigorously. The cells turn off expression of the variant quickly which is not cleared very efficiently. The evolutionary simulation shows that this strategy maximizes the growth of the parasite population under the presence of immune attack by the host. In corroboration with the result, we observed that stable variant has higher binding affinity to IgM from experimental data. Our study provides an evolutionary basis of widely variable switching rates of the var genes in Plasmodium falciparum.\nMIGRENE: The toolbox for microbial and individualized GEMs, reactobiome and community network modelling\nAuthors: Bidkhori, G.; Shoaie, S.\nScore: 1.0, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555866\nMotivation: Understanding microbial metabolism is crucial for evaluating shifts in human host-microbiome interactions during health and disease. However, the primary hurdle in the realm of constraint-based modelling and genome-scale metabolic models (GEMs) pertaining to host-microbiome interactions lies in the efficient utilization of metagenomic data for constructing GEMs that encompass unexplored and uncharacterized genomes. Challenges persist in effectively employing metagenomic data to address individualized microbial metabolism to investigate host-microbiome interactions. Result: To tackle this issue, we have created a computational framework designed for personalized microbiome metabolism. This framework takes into account factors such as microbiome composition, metagenomic species profiles and microbial gene catalogues. Subsequently, it generates GEMs at microbial level and individualized microbiome metabolism, including reaction richness, reaction abundance, reactobiome, individualized reaction set enrichment (iRSE), and community models. Availability and implementation: The MIGRENE toolbox is freely available at our GitHub repository https://github.com/sysbiomelab/MIGRENE. Comprehensive tutorials on the GitHub were provided for each step of the workflow.\nA trade-off in controlling upstream and downstream noise in signaling networks\nAuthors: Kong, K. K.; Luo, C.; Liu, F.\nScore: 1.0, Published: 2023-08-31 DOI: 10.1101/2023.08.29.555248\nSignal transduction, underpinning the function of a variety of biological systems, is inevitably affected by fluctuations. It remains intriguing how the timescale of a signaling network relates to its capability of noise control, specifically, whether long timescale can average out fluctuation or accumulate fluctuation. Here, we consider two noise components of the signaling system: the upstream noise from the fluctuation of the input signal and the downstream noise from the stochastic fluctuations of the network. We discover a fundamental trade-off in controlling the upstream and downstream noise: a longer timescale of the signaling network can buffer upstream noise, while accumulate downstream noise. Moreover, we confirm that this trade-off relation exists in real biological signaling networks such as a fold-change detection circuit and the p53 activation signaling system. Author SummaryInformation transmission is vital in biological systems, such as decoding the information regarding nutrient levels during chemotaxis or morphogen concentrations in tissue development. While fluctuations arising from the stochastic nature of biological processes inevitably affect information transmission, noise control mechanisms have been studied for decades. However, it remains controversial what the role of the slow dynamics (long timescale) is in noise control. On the one hand, it has been reported to attenuate noise by averaging out fluctuations. On the other hand, it is also proposed to amplify noise by accumulating fluctuations. Here we dissect the noise in signaling systems into two components: upstream noise originating from signal fluctuation, and downstream noise from the network stochasticity. Our analysis reveals that upstream noise negatively correlates with timescale, while downstream noise exhibits a positive correlation, indicating a fundamental trade-off in controlling the two noise components. Moreover, we provide an intuitive illustration to understand this phenomenon using the concept of landscape representation. Mathematically, we analytically derive a trade-off relation that agrees well with simulations. Our results uncover a new property of noise in signaling processes, deepening our understanding of noise control and proposing a new perspective in designing signaling network.\nA systems vaccinology resource to develop and test computational models of immunity\nAuthors: Shinde, P.; Soldevila, F.; Reyna, J.; Aoki, M.; Rasmussen, M.; Willemsen, L.; Kojima, M.; Ha, B.; Greenbaum, J. A.; Overton, J. A.; Guzman-Orozco, H.; Nili, S.; Orfield, S.; Gygi, J. P.; Antunes, R. d. S.; Sette, A.; Grant, B. J.; Olsen, L. R.; Konstorum, A.; Guan, L.; Ay, F.; Kleinstein, S. H.; Peters, B.\nScore: 1.0, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555193\nComputational models that predict an individuals response to a vaccine offer the potential for mechanistic insights and personalized vaccination strategies. These models are increasingly derived from systems vaccinology studies that generate immune profiles from human cohorts pre- and post-vaccination. Most of these studies involve relatively small cohorts and profile the response to a single vaccine. The ability to assess the performance of the resulting models would be improved by comparing their performance on independent datasets, as has been done with great success in other areas of biology such as protein structure predictions. To transfer this approach to system vaccinology studies, we established a prototype platform that focuses on the evaluation of Computational Models of Immunity to Pertussis Booster vaccinations (CMI-PB). A community resource, CMI-PB generates experimental data for the explicit purpose of model evaluation, which is performed through a series of annual data releases and associated contests. We here report on our experience with the first such dry run for a contest where the goal was to predict individual immune responses based on pre-vaccination multi-omic profiles. Over 30 models adopted from the literature were tested, but only one was predictive, and was based on age alone. The performance of new models built using CMI-PB training data was much better, but varied significantly based on the choice of pre-vaccination features used and the model building strategy. This suggests that previously published models developed for other vaccines do not generalize well to Pertussis Booster vaccination. Overall, these results reinforced the need for comparative analysis across models and datasets that CMI-PB aims to achieve. We are seeking wider community engagement for our first public prediction contest, which will open in early 2024.\n",
  "wordCount" : "2647",
  "inLanguage": "en",
  "datePublished": "2023-09-06T10:39:07Z",
  "dateModified": "2023-09-06T10:39:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on September 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555799">
        <p class="paperTitle">Exploiting fluctuations in gene expression to detect causal interactions between genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joly-Smith, E.; Talpur, M. M.; Allard, P.; Papazotos, F.; Potvin-Trottier, L.; Hilfinger, A.</p>
        <p class="info">Score: 10.1, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555799' target='https://doi.org/10.1101/2023.09.01.555799'> 10.1101/2023.09.01.555799</a></p>
        <p class="abstract">Characterizing and manipulating cellular behaviour requires a mechanistic understanding of the causal interactions between cellular components. We present an approach that can detect causal interactions between genes without the need to perturb the physiological state of cells. This approach exploits naturally occurring cell-to-cell variability which is experimentally accessible from static population snapshots of genetically identical cells without the need to follow cells over time. Our main contribution is a simple mathematical relation that constrains the propagation of gene expression noise through biochemical reaction networks. This relation allows us to rigorously interpret fluctuation data even when only a small part of a complex gene regulatory process can be observed. This relation can be exploited to detect causal interactions by synthetically engineering a passive reporter of gene expression, akin to the established &#34;dual reporter assay&#34;. While the focus of our contribution is theoretical, we also present an experimental proof-of-principle to illustrate the approach. Our data from synthetic gene regulatory networks in E. coli are not unequivocal but suggest that the method could prove useful in practice to identify causal interactions between genes from non-genetic cell-to-cell variability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553655" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553655">
        <p class="paperTitle">The interplay between lumen pressure and cell proliferation determines organoid morphology in a multicellular phase field model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tanida, S.; Fuji, K.; Lu, L.; Guyomar, T.; Lee, B. H.; Honigmann, A.; Grapin-Botton, A.; Riveline, D.; Nonomura, M.; Sano, M.; Hiraiwa, T.</p>
        <p class="info">Score: 13.8, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553655' target='https://doi.org/10.1101/2023.08.17.553655'> 10.1101/2023.08.17.553655</a></p>
        <p class="abstract">Organoids are ideal systems to predict the phenotypes of organs. However, there is currently a lack of understanding regarding the generalized rules that enable use of simple cellular principles to make morphological predictions of entire organoids. Therefore, we employed a phase field model with the following basic components: the minimum conditions for the timing and volume of cell division, lumen nucleation rules, and lumenal pressure. Through our model, we could compute and generate a myriad of organoid phenotypes observed till date. We propose morphological indices necessary to characterize the shapes and construct phase diagrams and show their dependencies on proliferation time and lumen pressure. Additionally, we introduced the lumen-index parameter, which helped in examining the criteria to maintain organoids as spherical structures comprising a single layer of cells and enclosing an intact lumen. Finally, we predict a star-like organoid phenotype that did not undergo differentiation, suggesting that the volume constraint during cell division may determine the final phenotype. In summary, our approach provides researchers with guidelines to test the mechanisms of self-organization and predict the shape of organoid.

Author summaryIn nature, a wide variety of organ morphologies are observed. Owing to the complexity of the process underlying the acquisition of organs morphology, it is challenging to investigate the mechanisms that lead to such variations. A promising approach to study these variations is the use of &#34;computational organoid&#34; study, which is the computational-based study of self-organizing shapes in multicellular assemblies and fluid-filled cavities called lumens that develop from a few proliferating cells. This study explores general mechanisms that dictate how various mechanical factors affect the growing self-organized multicellular assembly. We relied on computer simulations of the mathematical model called multicellular phase-field model with lumens and explored the mechanical factor effects, such as the lumen pressure while considering the time and volume conditions required for cell division. These simulations generated and categorized a wide range of organoid phenotypes based on the varying lumen pressure and cell division conditions. These phenotypes were characterized into seven distinct classes, based on the morphological index sets, including a cellular monolayer/multilayer surrounding single or multiple lumens and branch formation. These phenotypes were obtained without the assumption of differentiation. Our study elucidates the mechanisms underlying the organoid and organ formation with different shapes, thereby highlighting the significance of mechanical forces in shaping these complex biological structures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.555678">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.555678" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.555678">
        <p class="paperTitle">Human subcutaneous and visceral adipocyte atlases uncover classical and specialized adipocytes and depot-specific patterns</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.555678" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.555678" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lazarescu, O.; Ziv Agam, M.; Haim, Y.; Hekselman, I.; Jubran, J.; Shneyour, A.; Kitsberg, D.; Levin, L.; Liberty, I. F.; Yoel, U.; Dukhno, O.; Adam, M.; Korner, A.; Murphy, R.; Bluher, M.; Habib, N.; Rudich, A.; Yeger-Lotem, E.</p>
        <p class="info">Score: 8.2, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.555678' target='https://doi.org/10.1101/2023.09.04.555678'> 10.1101/2023.09.04.555678</a></p>
        <p class="abstract">Human adipose depots are functionally distinct. Yet, recent single-nucleus RNA-sequencing (snRNA-seq) analyses largely uncovered overlapping/similar cell-type landscapes. We hypothesized that adipocytes subtypes, differentiation trajectories, and/or intercellular communication patterns could illuminate this depot similarity-difference gap. For this, we performed snRNA-seq of human subcutaneous and visceral adipose tissue. Whereas the majority of adipocytes in both depots were classical, namely enriched in lipid metabolism pathways, we also observed specialized adipocyte subtypes that were enriched in immune-related, extracellular matrix deposition (fibrosis), vascularization/angiogenesis, or ribosomal processes. Pseudo-temporal analysis suggested a developmental trajectory from adipose progenitor cells to classical adipocytes via specialized adipocytes, suggesting that the classical state stems from loss, rather than gain, of specialized functions. Lastly, intercellular communication routes were consistent with the different inflammatory tone of the two depots. Jointly, these findings provide a high-resolution view into the contribution of cellular composition, differentiation, and intercellular communication patterns to human fat depot differences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556134">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556134" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556134">
        <p class="paperTitle">Approximate Bayesian Computation for Inferring Waddington Landscapes from Single Cell Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556134" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556134" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Y.; Zhang, S. Y.; Kleijn, I. T.; Stumpf, M. P. H.</p>
        <p class="info">Score: 7.9, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556134' target='https://doi.org/10.1101/2023.09.03.556134'> 10.1101/2023.09.03.556134</a></p>
        <p class="abstract">Single cell technologies allow us to gain insights into cellular processes at unprecedented resolution. In stem cell and developmental biology snap-shot data allows us to characterise how the transcriptional state of cells changes between successive cell types. Here we show how approximate Bayesian computation (ABC) can be employed to calibrate mathematical models against single cell data. In our simulation study we demonstrate the pivotal role of the adequate choice of distance measures appropriate for single cell data. We show that for good distance measures, notably optimal transport distances, we can infer parameters for mathematical models from simulated single cell data. We show that the ABC posteriors can be used to characterise parameter sensitivity and identify dependencies between different parameters, and to infer representations of the Waddington or epigenetic landscape, which forms a popular and interpretable representation of the developmental dynamics. In summary, these results pave the way for fitting mechanistic models of stem cell differentiation to single cell data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555073">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555073" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555073">
        <p class="paperTitle">Transcription factors operate on a limited vocabulary of binding motifs in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555073" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555073" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zenker, S.; Wulf, D.; Meierhenrich, A.; Becker, S.; Eisenhut, M.; Stracke, R.; Weisshaar, B.; Braeutigam, A.</p>
        <p class="info">Score: 8.7, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555073' target='https://doi.org/10.1101/2023.08.28.555073'> 10.1101/2023.08.28.555073</a></p>
        <p class="abstract">Predicting gene expression from promoter sequence requires understanding of the different signal integration points within a promoter. Sequence-specific transcription factors (TFs) binding to their cognate TF binding motifs control gene expression in eukaryotes by activating and repressing transcription. Their interplay generates complex expression patterns in reaction to environmental conditions and developmental cues.

We hypothesized that signals are not only integrated by different TFs binding various positions in a promoter, but also by single TF binding motifs onto which multiple TFs can bind. Analyzing 2,190 binding motifs, we identified only 76 core TF binding motifs in plants. Twenty-one TF protein families act highly specific and bind a single conserved motif. Four TF families are classified as semi-conserved as they bind up to four motifs within a family, with divisions along phylogenetic groups. Five TF families bind diverse motifs. Expression analyses revealed high competition within TF families for the same binding motif. The results show that singular binding motifs act as signal integrators in plants where a combination of binding affinity and TF abundance likely determine the output.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555601">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555601" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555601">
        <p class="paperTitle">PTMNavigator: Interactive Visualization of Differentially Regulated Post-Translational Modifications in Cellular Signaling Pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555601" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555601" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mueller, J.; Bayer, F. P.; Wilhelm, M.; Kuster, B.; The, M.</p>
        <p class="info">Score: 2.0, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555601' target='https://doi.org/10.1101/2023.08.31.555601'> 10.1101/2023.08.31.555601</a></p>
        <p class="abstract">Post-translational modifications (PTMs) play a governing role in regulating cellular signaling, fine-tuning protein function, and orchestrating complex biological processes. Despite their importance, the lack of comprehensive tools for studying PTMs from a pathway-centric perspective has limited our ability to understand how PTMs modulate cellular pathways on a molecular level. Here, we present PTMNavigator, a tool integrated into the ProteomicsDB platform, which offers an interactive interface for researchers to overlay experimental PTM data on top of ~3000 well-curated pathway diagrams. This offers a comprehensive view of the intricate relationship between PTMs and signaling pathways. To demonstrate the utility of PTMNavigator, we applied it to two phosphoproteomics perturbation datasets. First, PTMNavigator enhanced pathway enrichment analysis by showing how the regulated peptides are distributed in the pathways with high enrichment scores. Second, it visualized how drug treatments result in a discernable flow of PTM-driven signaling within pathways. Third, PTMNavigator aided in extending existing pathways by uncovering previously unknown relationships between PTMs and pathway components. By facilitating the discovery of novel PTM-pathway interactions and enhancing our understanding of cellular signaling dynamics, PTMNavigator advances our knowledge of PTM biology and its implications in health and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555470">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555470" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555470">
        <p class="paperTitle">Evolution of hierarchical switching pattern in antigenic variation of Plasmodium falciparum under variable host immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555470" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555470" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghosh, B.; Priya, I. G.</p>
        <p class="info">Score: 1.6, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555470' target='https://doi.org/10.1101/2023.08.30.555470'> 10.1101/2023.08.30.555470</a></p>
        <p class="abstract">The var genes family encoding the variants of the erythrocyte membrane protein of Plasmodium falciparum is crucial for virulence of the parasite inside host. The transcriptional output of the var genes switches from one variant to other in a mutually exclusive fashion. It is proposed that a biased hierarchical switching pattern optimizes the growth and survival of the parasite inside the host. Apart from the hierarchical switching pattern, it is also well established that the intrinsic switching rates vary widely among the var genes. The centromeric protein like Var2csa is much more stable than the genes located at the telomeric and sub-telomeric regions of the chromosomes. In this study, we explored the evolutionary advantage achieved through selecting variable switching rates. Our theoretical analysis based on a mathematical model coupled with single cell RNA-seq data suggests that the variable switching rate is beneficial when cells expressing different variants are deferentially amenable to be cleared by the immune response. In fact, the variants which are cleared by the immune systems more efficiently are more stably expressed compared to a variant attacked by the immune system much less vigorously. The cells turn off expression of the variant quickly which is not cleared very efficiently. The evolutionary simulation shows that this strategy maximizes the growth of the parasite population under the presence of immune attack by the host. In corroboration with the result, we observed that stable variant has higher binding affinity to IgM from experimental data. Our study provides an evolutionary basis of widely variable switching rates of the var genes in Plasmodium falciparum.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555866">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555866" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555866">
        <p class="paperTitle">MIGRENE: The toolbox for microbial and individualized GEMs, reactobiome and community network modelling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555866" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555866" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bidkhori, G.; Shoaie, S.</p>
        <p class="info">Score: 1.0, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555866' target='https://doi.org/10.1101/2023.09.01.555866'> 10.1101/2023.09.01.555866</a></p>
        <p class="abstract">Motivation: Understanding microbial metabolism is crucial for evaluating shifts in human host-microbiome interactions during health and disease. However, the primary hurdle in the realm of constraint-based modelling and genome-scale metabolic models (GEMs) pertaining to host-microbiome interactions lies in the efficient utilization of metagenomic data for constructing GEMs that encompass unexplored and uncharacterized genomes. Challenges persist in effectively employing metagenomic data to address individualized microbial metabolism to investigate host-microbiome interactions. Result: To tackle this issue, we have created a computational framework designed for personalized microbiome metabolism. This framework takes into account factors such as microbiome composition, metagenomic species profiles and microbial gene catalogues. Subsequently, it generates GEMs at microbial level and individualized microbiome metabolism, including reaction richness, reaction abundance, reactobiome, individualized reaction set enrichment (iRSE), and community models. Availability and implementation: The MIGRENE toolbox is freely available at our GitHub repository https://github.com/sysbiomelab/MIGRENE. Comprehensive tutorials on the GitHub were provided for each step of the workflow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555248">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555248" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555248">
        <p class="paperTitle">A trade-off in controlling upstream and downstream noise in signaling networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555248" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555248" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kong, K. K.; Luo, C.; Liu, F.</p>
        <p class="info">Score: 1.0, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555248' target='https://doi.org/10.1101/2023.08.29.555248'> 10.1101/2023.08.29.555248</a></p>
        <p class="abstract">Signal transduction, underpinning the function of a variety of biological systems, is inevitably affected by fluctuations. It remains intriguing how the timescale of a signaling network relates to its capability of noise control, specifically, whether long timescale can average out fluctuation or accumulate fluctuation. Here, we consider two noise components of the signaling system: the upstream noise from the fluctuation of the input signal and the downstream noise from the stochastic fluctuations of the network. We discover a fundamental trade-off in controlling the upstream and downstream noise: a longer timescale of the signaling network can buffer upstream noise, while accumulate downstream noise. Moreover, we confirm that this trade-off relation exists in real biological signaling networks such as a fold-change detection circuit and the p53 activation signaling system.

Author SummaryInformation transmission is vital in biological systems, such as decoding the information regarding nutrient levels during chemotaxis or morphogen concentrations in tissue development. While fluctuations arising from the stochastic nature of biological processes inevitably affect information transmission, noise control mechanisms have been studied for decades. However, it remains controversial what the role of the slow dynamics (long timescale) is in noise control. On the one hand, it has been reported to attenuate noise by averaging out fluctuations. On the other hand, it is also proposed to amplify noise by accumulating fluctuations. Here we dissect the noise in signaling systems into two components: upstream noise originating from signal fluctuation, and downstream noise from the network stochasticity. Our analysis reveals that upstream noise negatively correlates with timescale, while downstream noise exhibits a positive correlation, indicating a fundamental trade-off in controlling the two noise components. Moreover, we provide an intuitive illustration to understand this phenomenon using the concept of landscape representation. Mathematically, we analytically derive a trade-off relation that agrees well with simulations. Our results uncover a new property of noise in signaling processes, deepening our understanding of noise control and proposing a new perspective in designing signaling network.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555193">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555193" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555193">
        <p class="paperTitle">A systems vaccinology resource to develop and test computational models of immunity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555193" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555193" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shinde, P.; Soldevila, F.; Reyna, J.; Aoki, M.; Rasmussen, M.; Willemsen, L.; Kojima, M.; Ha, B.; Greenbaum, J. A.; Overton, J. A.; Guzman-Orozco, H.; Nili, S.; Orfield, S.; Gygi, J. P.; Antunes, R. d. S.; Sette, A.; Grant, B. J.; Olsen, L. R.; Konstorum, A.; Guan, L.; Ay, F.; Kleinstein, S. H.; Peters, B.</p>
        <p class="info">Score: 1.0, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555193' target='https://doi.org/10.1101/2023.08.28.555193'> 10.1101/2023.08.28.555193</a></p>
        <p class="abstract">Computational models that predict an individuals response to a vaccine offer the potential for mechanistic insights and personalized vaccination strategies. These models are increasingly derived from systems vaccinology studies that generate immune profiles from human cohorts pre- and post-vaccination. Most of these studies involve relatively small cohorts and profile the response to a single vaccine. The ability to assess the performance of the resulting models would be improved by comparing their performance on independent datasets, as has been done with great success in other areas of biology such as protein structure predictions. To transfer this approach to system vaccinology studies, we established a prototype platform that focuses on the evaluation of Computational Models of Immunity to Pertussis Booster vaccinations (CMI-PB). A community resource, CMI-PB generates experimental data for the explicit purpose of model evaluation, which is performed through a series of annual data releases and associated contests. We here report on our experience with the first such  dry run for a contest where the goal was to predict individual immune responses based on pre-vaccination multi-omic profiles. Over 30 models adopted from the literature were tested, but only one was predictive, and was based on age alone. The performance of new models built using CMI-PB training data was much better, but varied significantly based on the choice of pre-vaccination features used and the model building strategy. This suggests that previously published models developed for other vaccines do not generalize well to Pertussis Booster vaccination. Overall, these results reinforced the need for comparative analysis across models and datasets that CMI-PB aims to achieve. We are seeking wider community engagement for our first public prediction contest, which will open in early 2024.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
