<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Systematic mapping of antibiotic cross-resistance and collateral sensitivity with chemical genetics
Authors: Sakenova, N.; Cacace, E.; Orakov, A.; Huber, F.; Varik, V.; Kritikos, G.; Michiels, J.; Bork, P.; Cossart, P.; Goemans, C.; Typas, A.
Score: 45.7, Published: 2024-01-27 DOI: 10.1101/2024.01.25.576750
By acquiring or evolving resistance to one antibiotic, bacteria can become resistant to a second one, due to shared underlying mechanisms. This is called cross-resistance (XR) and further limits therapeutic choices.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Systematic mapping of antibiotic cross-resistance and collateral sensitivity with chemical genetics
Authors: Sakenova, N.; Cacace, E.; Orakov, A.; Huber, F.; Varik, V.; Kritikos, G.; Michiels, J.; Bork, P.; Cossart, P.; Goemans, C.; Typas, A.
Score: 45.7, Published: 2024-01-27 DOI: 10.1101/2024.01.25.576750
By acquiring or evolving resistance to one antibiotic, bacteria can become resistant to a second one, due to shared underlying mechanisms. This is called cross-resistance (XR) and further limits therapeutic choices." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-04T10:38:39+00:00" />
<meta property="article:modified_time" content="2024-02-04T10:38:39+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Systematic mapping of antibiotic cross-resistance and collateral sensitivity with chemical genetics
Authors: Sakenova, N.; Cacace, E.; Orakov, A.; Huber, F.; Varik, V.; Kritikos, G.; Michiels, J.; Bork, P.; Cossart, P.; Goemans, C.; Typas, A.
Score: 45.7, Published: 2024-01-27 DOI: 10.1101/2024.01.25.576750
By acquiring or evolving resistance to one antibiotic, bacteria can become resistant to a second one, due to shared underlying mechanisms. This is called cross-resistance (XR) and further limits therapeutic choices."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Systematic mapping of antibiotic cross-resistance and collateral sensitivity with chemical genetics\nAuthors: Sakenova, N.; Cacace, E.; Orakov, A.; Huber, F.; Varik, V.; Kritikos, G.; Michiels, J.; Bork, P.; Cossart, P.; Goemans, C.; Typas, A.\nScore: 45.7, Published: 2024-01-27 DOI: 10.1101/2024.01.25.576750\nBy acquiring or evolving resistance to one antibiotic, bacteria can become resistant to a second one, due to shared underlying mechanisms. This is called cross-resistance (XR) and further limits therapeutic choices.",
  "keywords": [
    
  ],
  "articleBody": " Systematic mapping of antibiotic cross-resistance and collateral sensitivity with chemical genetics\nAuthors: Sakenova, N.; Cacace, E.; Orakov, A.; Huber, F.; Varik, V.; Kritikos, G.; Michiels, J.; Bork, P.; Cossart, P.; Goemans, C.; Typas, A.\nScore: 45.7, Published: 2024-01-27 DOI: 10.1101/2024.01.25.576750\nBy acquiring or evolving resistance to one antibiotic, bacteria can become resistant to a second one, due to shared underlying mechanisms. This is called cross-resistance (XR) and further limits therapeutic choices. The opposite scenario, in which initial resistance leads to sensitivity to a second antibiotic, is termed collateral sensitivity (CS) and can inform cycling or combinatorial treatments. Despite their clinical relevance, our current knowledge of such interactions is limited, mostly due to experimental constraints in their assessment and lack of understanding of the underlying mechanisms. To fill this gap, we used published chemical genetic data on the impact of all Escherichia coli non-essential genes on resistance/sensitivity to 40 antibiotics, and devised a metric that robustly discriminates between known XR and CS antibiotic interactions. This metric, based on chemical genetic profile (dis)similarity between two drugs, allowed us to infer 404 XR and 267 CS interactions, thereby expanding the number of known interactions by more than 3-fold - including reclassifying 116 previously reported interactions. We benchmarked our results by validating 55 out of 59 inferred interactions via experimental evolution. By identifying mutants driving XR and CS interactions in chemical genetics, we recapitulated known and uncovered previously unknown mechanisms, and demonstrated that a given drug pair can exhibit both interactions depending on the resistance mechanism. Finally, we applied CS drug pairs in combination to reduce antibiotic resistance development in vitro. Altogether, our approach provides a systematic framework to map XR/CS interactions and their mechanisms, paving the way for the development of rationally-designed antibiotic combination treatments.\nPredicting microbial genome-scale metabolic networks directly from 16S rRNA gene sequences\nAuthors: Oyas, O.; Kobel, C. M.; Vik, J. O.; Pope, P. B.\nScore: 21.2, Published: 2024-01-27 DOI: 10.1101/2024.01.26.576649\nGenome-scale metabolic models are key biotechnology tools that can predict metabolic capabilities and growth for an organism. In particular, these models have become indispensable for metabolic analysis of microbial species and communities such as the gut microbiomes of humans and other animals. Accurate microbial models can be built automatically from genomes, but many microbes have only been observed through sequencing of marker genes such as 16S rRNA and thus remain inaccessible to genome-scale modeling. To extend the scope of genome-scale metabolic models to microbes that lack genomic information, we trained an artificial neural network to build microbial models from numeric representations of 16S rRNA gene sequences. Specifically, we built models and extracted 16S rRNA gene sequences from more than 15,000 reference and representative microbial genomes, computed multiple sequence alignments and large language model embeddings for the 16S rRNA gene sequences, and trained the neural network to predict metabolic reaction probabilities from sequences, alignments, or embeddings. Training was fast on a single graphics processing unit and trained networks predicted reaction probabilities accurately for unseen archaeal and bacterial sequences and species. This makes it possible to reconstruct microbial genome-scale metabolic networks from any 16S rRNA gene sequence and enables simulation of metabolism and growth for all observed microbial life.\nOrganellomics: AI-driven deep organellar phenotyping of human neurons\nAuthors: Molitor, L.; Krispin, S.; Van-Zuiden, W.; Danino, Y. M.; Rudberg, N.; Bar, C.; Amzallag, E.; Lubliner, J.; Siany, A.; Eitan, C.; Cohen, Y.; Yacovzada, N. S.; Hornstein, E.\nScore: 20.6, Published: 2024-01-31 DOI: 10.1101/2024.01.31.572110\nSystematic assessment of organelle architectures in cells, known as the organellome, could provide valuable insights into cellular states and disease pathologies but remains largely uncharted. Here, we devised a novel pipeline combining self-supervised deep learning and transfer learning to generate a Neuronal Organellomics Vision Atlas (NOVA). Analyzing over 1.5 million confocal images of 24 distinct membrane-bound and membrane-less organelles in human neurons, we enable a simultaneous evaluation of all organelles. We show that organellomics allows the study of cellular phenotypes by quantifying the localization and morphological properties embodied in multiple different organelles, using a unified score. We further developed a strategy to superimpose all organelles, which represents a new realization of cellular state. The value of our approach is demonstrated by characterizing specific organellar responses of human neurons to stress, cytoplasmic mislocalization of TDP-43, or disease-associated variations in ALS genes. Therefore, organellomics offers a novel approach to study the neuro-cellular biology of diseases. HighlightsAI-driven organellomics without cell segmentation or multiplexed imaging. Analysis of 24 membrane-bound and membrane-less organelles in more than 1.5 million images of human neurons. Quantitative organelle-level description of neuronal response to chemical and genetic perturbations. Organelles ranked on a single metric scale and integrated organellome view via superposition of multiple organelles.\nStructural determinants of co-translational protein complex assembly\nAuthors: Malik, S.; Venezian, J.; Lobov, A.; Heidenreich, M.; Garcia-Seisdedos, H.; Yeates, T. O.; Shiber, A.; Levy, E. D.\nScore: 27.9, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576408\nThe assembly of proteins into functional complexes is critical to lifes processes. While textbooks depict complex assembly as occurring between fully synthesized proteins, we know today that thousands of proteins in the human proteome assemble co-translationally during their synthesis. Why this process takes place, however, remains unknown. We show that co-translational assembly is governed by biophysical and structural characteristics of the protein complex, and involves mutually stabilized, intertwined subunits. Consequently, these subunits are also co-regulated across the central dogma, from transcription to protein degradation. Leveraging structural signatures with AlphaFold2-based predictions enables us to accurately predict co-translational assembly on a proteome-wide scale, which we validated by ribosome profiling, genetic perturbations, and smFISH experiments. Notably, the latter showed that co-translationally assembling subunits exhibit co-localized mRNAs. This work unveils a fundamental connection between protein structure and the translation process, highlighting the overarching impact of three-dimensional structure on gene expression, mRNA localization, and proteostasis. One Sentence SummaryProtein complexes with topologically intertwined subunits require co-translational assembly and synchronized proteostasis of subunits, with implications in protein stability, mRNA localization, and evolution. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=195 SRC=\"FIGDIR/small/576408v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (62K): org.highwire.dtl.DTLVardef@106b9a8org.highwire.dtl.DTLVardef@86f034org.highwire.dtl.DTLVardef@103fcc2org.highwire.dtl.DTLVardef@c74d15_HPS_FORMAT_FIGEXP M_FIG C_FIG\nA constraint-based framework to reconstruct interaction networks in microbial communities\nAuthors: Oftadeh, O.; Sahin, A.; Vayena, E.; Hatzimanikatis, V.\nScore: 7.9, Published: 2024-01-31 DOI: 10.1101/2024.01.30.577913\nMicrobial communities live in diverse habitats and significantly impact our health and the environment. However, the principles that govern their formation and evolution remain poorly understood. A crucial step in studying microbial communities is to identify the potential metabolic interactions between the community members, such as competition for nutrients or cross-feeding. Due to the size and complexity of the metabolic network of each organism, there may be a variety of connections between each pair of organisms, which poses a challenge to unraveling the metabolic interactions. Here, we present ReMIND, a computational framework to reconstruct the interaction networks in microbial communities based on the metabolic capabilities of individual organisms. We applied ReMIND to a well-studied uranium-reducing community and the honeybee gut microbiome. Our results provide new perspectives on the evolutionary forces that shape these ecosystems and the trade-off between metabolite exchange and biomass yield. By enumerating alternative interaction networks, we systematically identified the most likely metabolites to be exchanged and highlighted metabolites that could mediate competitive interactions. We envision that ReMIND will help characterize the metabolic capacity of individual members and elucidate metabolic interactions in diverse communities, thus holds the potential to guide many applications in precision medicine and synthetic ecology.\nNonreciprocal synchronization in embryonic oscillator ensembles\nAuthors: Ho, C.; Jutras-Dube, L.; Zhao, M.; Mönke, G.; Kiss, I. Z.; Francois, P.; Aulehla, A.\nScore: 7.5, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577856\nSynchronization of coupled oscillators is a universal phenomenon encountered across different scales and contexts e.g., chemical wave patterns, superconductors and the unison applause we witness in concert halls. The existence of common underlying coupling rules define universality classes, revealing a fundamental sameness between seemingly distinct systems. Identifying rules of synchronization in any particular setting is hence of paramount relevance. Here, we address the coupling rules within an embryonic oscillator ensemble linked to vertebrate embryo body axis segmentation. In vertebrates, the periodic segmentation of the body axis involves synchronized signaling oscillations in cells within the presomitic mesoderm (PSM), from which somites, the pre-vertebrae, form. At the molecular level, it is known that intact Notch-signaling and cell-to-cell contact is required for synchronization between PSM cells. However, an understanding of the coupling rules is still lacking. To identify these, we develop a novel experimental assay that enables direct quantification of synchronization dynamics within mixtures of oscillating cell ensembles, for which the initial input frequency and phase distribution are known. Our results reveal a \"winner-takes-it-all\" synchronization outcome i.e., the emerging collective rhythm matches one of the input rhythms. Using a combination of theory and experimental validation, we develop a new coupling model, the \"Rectified Kuramoto\" (ReKu) model, characterized by a phase-dependent, non-reciprocal interaction in the coupling of oscillatory cells. Such non-reciprocal synchronization rules reveal fundamental similarities between embryonic oscillators and a class of collective behaviours seen in neurons and fireflies, where higher level computations are performed and linked to non-reciprocal synchronization.\nSignalingProfiler 2.0: a network-based approach to bridge multi-omics data to phenotypic hallmarks\nAuthors: Venafra, V.; Sacco, F.; Perfetto, L.\nScore: 2.9, Published: 2024-01-29 DOI: 10.1101/2024.01.25.577229\nUnraveling the cellular signaling remodeling upon a perturbation is a fundamental challenge to understand disease mechanisms and to identify potential drug targets. In this pursuit, computational tools that generate mechanistic hypotheses from multi-omics data have invaluable potential. Here we present SignalingProfiler 2.0, a multi-step pipeline to systematically derive context-specific signaling models by integrating proteogenomic data with prior knowledge-causal networks. This is a freely accessible and flexible tool that incorporates statistical, footprint-based, and graph algorithms to accelerate the integration and interpretation of multi-omics data. Through benchmarking and rigorous parameter selection on a proof-of-concept study, performed in metformin-treated breast cancer cells, we demonstrate the tools ability to generate a hierarchical mechanistic network that recapitulates novel and known drug-perturbed signaling and phenotypic outcomes. In summary, SignalingProfiler 2.0 addresses the emergent need to derive biologically relevant information from complex multi-omics data by extracting interpretable networks.\nMultistability and predominant double-positive states in a four node mutually repressive network: a case study of Th1/Th2/Th17/T-reg differentiation\nAuthors: Duddu, A. S.; Andreas, E.; BV, H.; Grover, K.; Singh, V. R.; Hari, K.; Jhunjhunwala, S.; Cummins, B.; Gedeon, T.; Jolly, M. K.\nScore: 2.6, Published: 2024-02-02 DOI: 10.1101/2024.01.30.575880\nElucidating the emergent dynamics of complex regulatory networks enabling cellular differentiation is crucial to understand embryonic development and suggest strategies for synthetic circuit design. A well-studied network motif often driving cellular decisions is a toggle switch - a set of two mutually inhibitory lineage-specific transcription factors A and B. A toggle switch often enables two possible states mutually exclusive states - (high A, low B) and (low A, high B) - from a common progenitor cell. However, the dynamics of networks enabling differentiation of more than two cell types from a progenitor cell is not well-studied. Here, we investigate the dynamics of four master regulators A, B, C and D inhibiting each other, thus forming a toggle tetrahedron. Our simulations show that a toggle tetrahedron predominantly allows for co-existence of six 'double positive' or hybrid states where two of the nodes are expressed relatively high as compared to the remaining two - (high A, high B, low C, low D), (high A, low B, high C, low D), (high A, low B, low C, high D), (low A, high B, high C, low D), (low A, low B, high C, high D) and (low A, high B, low C, high D). Stochastic simulations showed state-switching among these phenotypes, indicating phenotypic plasticity. Finally, we apply our results to understand the differentiation of naive CD4+ T cells into Th1, Th2, Th17 and Treg subsets, suggesting Th1/Th2/Th17/Treg decision-making to be a two-step process. Our results reveal multistable dynamics and establish the stable co-existence of hybrid cell-states, offering a potential explanation for simultaneous differentiation of multipotent naive CD4+ T cells.\nThermodynamics determines the coupling between growth and byproduct production\nAuthors: Oftadeh, O.; Hatzimanikatis, V.\nScore: 2.2, Published: 2024-02-02 DOI: 10.1101/2024.01.31.578201\nGenetic manipulation of cells to couple byproduct production and growth rate is important in bioengineering and biotechnology. In this way, we can use growth rate as a selective pressure, where the mutants with higher growth have higher production capacity. Computational methods have been proposed to find knockouts that couple growth and byproduct production. However, none of these methods consider the energetic and thermodynamic feasibility of such knockout strategies. Furthermore, there is no computational study of how variations in metabolite concentrations affect the coupling between growth and byproduct formation. One of the computational methods to find knockouts that couple growth and byproduct formation is OptKnock. OptKnock is a bi-level optimization problem. Here, we integrated thermodynamic constraints into the bilevel formulation of OptKnock to create TOptKnock. We show that the computational efficiency of TOptKnock is comparable to that of OptKnock. TOptKnock can account for the thermodynamic viability of the knockouts and examine how variations in metabolite concentrations affect the coupling. We have shown that the coupling between growth and byproduct formation can change in response to variations in concentrations. Thus, a knockout strategy might be optimal for one intracellular condition but suboptimal for another. If metabolomics data are available, TOptKnock can search for optimal knockout interventions under the given condition. We also envision that the TOptKnock framework will help develop strategies for manipulating metabolite concentrations to couple growth and byproduct formation.\nUnraveling the oscillatory dynamics of mRNA metabolism and chromatin accessibility during the cell cycle through integration of single-cell multiomic data\nAuthors: Nariya, M. K.; Santiago-Algarra, D.; Tassy, O.; Cerciat, M.; Ye, T.; Riba, A.; Molina, N.\nScore: 16.3, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575159\nThe cell cycle is a highly regulated process that ensures the accurate replication and transmission of genetic information from one generation of cells to the next. It is a fundamental biological process and plays a crucial role in development, growth, and maintenance of all living organisms, and its dysregulation can lead to a number of pathologies such as autoimmune diseases, neurodegenerative diseases, and cancer. In this work we present a novel approach to study the gene expression and chromatin accessibility dynamics during the cell cycle in mouse embryonic stem cells. To achieve this, we combined high-depth single-cell multiome sequencing, biophysical modeling, and advanced deep learning techniques. First, we used DeepCycle, a deep learning tool that assigns a cell cycle phase to every cell based on its spliced and unspliced mRNA levels. We then developed a biophysical model that describes the dynamics of gene-specific mRNA transcription, nuclear export, and degradation rates during the cell cycle. Our model obtains genome-wide estimates of mRNA transcription, nuclear retention, and degradation half-lives of genes exhibiting oscillatory dynamics in mESCs. The key feature of single-cell multiome sequencing is that it simultaneously provides readouts for gene expression as well as chromatin accessibility in the same cells. By applying our approach to these data we showcase first-of-its-kind exploration of chromatin accessibility during the cell cycle at high temporal resolution. We believe that our work will be pivotal in formulating a coherent quantitative description of mRNA metabolism. HighlightsO_LIIntegration of single-cell and single-nucleus multiome profiling with deep learning and biophysical modeling uncovers cell cycle regulation of RNA metabolism C_LIO_LIGenome-wide estimation of mRNA transcription, nuclear export, and degradation rates during the cell cycle in mESCs C_LIO_LIIdentification of a prominent wave of transcription during early G1 in unperturbed asynchronously proliferating mESCs C_LIO_LICharacterization of temporal waves of post-transcriptional mRNA regulation controlling both mRNA degradation and nuclear export during the cell cycle C_LIO_LIChromatin accessibility dynamics within TF footprints reveal key cell cycle regulators C_LI\n",
  "wordCount" : "2589",
  "inLanguage": "en",
  "datePublished": "2024-02-04T10:38:39Z",
  "dateModified": "2024-02-04T10:38:39Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 4, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.576750">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.576750" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.576750">
        <p class="paperTitle">Systematic mapping of antibiotic cross-resistance and collateral sensitivity with chemical genetics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.576750" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.576750" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sakenova, N.; Cacace, E.; Orakov, A.; Huber, F.; Varik, V.; Kritikos, G.; Michiels, J.; Bork, P.; Cossart, P.; Goemans, C.; Typas, A.</p>
        <p class="info">Score: 45.7, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.576750' target='https://doi.org/10.1101/2024.01.25.576750'> 10.1101/2024.01.25.576750</a></p>
        <p class="abstract">By acquiring or evolving resistance to one antibiotic, bacteria can become resistant to a second one, due to shared underlying mechanisms. This is called cross-resistance (XR) and further limits therapeutic choices. The opposite scenario, in which initial resistance leads to sensitivity to a second antibiotic, is termed collateral sensitivity (CS) and can inform cycling or combinatorial treatments. Despite their clinical relevance, our current knowledge of such interactions is limited, mostly due to experimental constraints in their assessment and lack of understanding of the underlying mechanisms. To fill this gap, we used published chemical genetic data on the impact of all Escherichia coli non-essential genes on resistance/sensitivity to 40 antibiotics, and devised a metric that robustly discriminates between known XR and CS antibiotic interactions. This metric, based on chemical genetic profile (dis)similarity between two drugs, allowed us to infer 404 XR and 267 CS interactions, thereby expanding the number of known interactions by more than 3-fold - including reclassifying 116 previously reported interactions. We benchmarked our results by validating 55 out of 59 inferred interactions via experimental evolution. By identifying mutants driving XR and CS interactions in chemical genetics, we recapitulated known and uncovered previously unknown mechanisms, and demonstrated that a given drug pair can exhibit both interactions depending on the resistance mechanism. Finally, we applied CS drug pairs in combination to reduce antibiotic resistance development in vitro. Altogether, our approach provides a systematic framework to map XR/CS interactions and their mechanisms, paving the way for the development of rationally-designed antibiotic combination treatments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.576649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.576649" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.576649">
        <p class="paperTitle">Predicting microbial genome-scale metabolic networks directly from 16S rRNA gene sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.576649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.576649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oyas, O.; Kobel, C. M.; Vik, J. O.; Pope, P. B.</p>
        <p class="info">Score: 21.2, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.576649' target='https://doi.org/10.1101/2024.01.26.576649'> 10.1101/2024.01.26.576649</a></p>
        <p class="abstract">Genome-scale metabolic models are key biotechnology tools that can predict metabolic capabilities and growth for an organism. In particular, these models have become indispensable for metabolic analysis of microbial species and communities such as the gut microbiomes of humans and other animals. Accurate microbial models can be built automatically from genomes, but many microbes have only been observed through sequencing of marker genes such as 16S rRNA and thus remain inaccessible to genome-scale modeling. To extend the scope of genome-scale metabolic models to microbes that lack genomic information, we trained an artificial neural network to build microbial models from numeric representations of 16S rRNA gene sequences. Specifically, we built models and extracted 16S rRNA gene sequences from more than 15,000 reference and representative microbial genomes, computed multiple sequence alignments and large language model embeddings for the 16S rRNA gene sequences, and trained the neural network to predict metabolic reaction probabilities from sequences, alignments, or embeddings. Training was fast on a single graphics processing unit and trained networks predicted reaction probabilities accurately for unseen archaeal and bacterial sequences and species. This makes it possible to reconstruct microbial genome-scale metabolic networks from any 16S rRNA gene sequence and enables simulation of metabolism and growth for all observed microbial life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.572110">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.572110" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.572110">
        <p class="paperTitle">Organellomics: AI-driven deep organellar phenotyping of human neurons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.572110" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.572110" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Molitor, L.; Krispin, S.; Van-Zuiden, W.; Danino, Y. M.; Rudberg, N.; Bar, C.; Amzallag, E.; Lubliner, J.; Siany, A.; Eitan, C.; Cohen, Y.; Yacovzada, N. S.; Hornstein, E.</p>
        <p class="info">Score: 20.6, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.572110' target='https://doi.org/10.1101/2024.01.31.572110'> 10.1101/2024.01.31.572110</a></p>
        <p class="abstract">Systematic assessment of organelle architectures in cells, known as the organellome, could provide valuable insights into cellular states and disease pathologies but remains largely uncharted. Here, we devised a novel pipeline combining self-supervised deep learning and transfer learning to generate a Neuronal Organellomics Vision Atlas (NOVA). Analyzing over 1.5 million confocal images of 24 distinct membrane-bound and membrane-less organelles in human neurons, we enable a simultaneous evaluation of all organelles. We show that organellomics allows the study of cellular phenotypes by quantifying the localization and morphological properties embodied in multiple different organelles, using a unified score. We further developed a strategy to superimpose all organelles, which represents a new realization of cellular state. The value of our approach is demonstrated by characterizing specific organellar responses of human neurons to stress, cytoplasmic mislocalization of TDP-43, or disease-associated variations in ALS genes. Therefore, organellomics offers a novel approach to study the neuro-cellular biology of diseases.

HighlightsAI-driven organellomics without cell segmentation or multiplexed imaging.

Analysis of 24 membrane-bound and membrane-less organelles in more than 1.5 million images of human neurons.

Quantitative organelle-level description of neuronal response to chemical and genetic perturbations.

Organelles ranked on a single metric scale and integrated organellome view via superposition of multiple organelles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576408">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576408" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576408">
        <p class="paperTitle">Structural determinants of co-translational protein complex assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576408" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576408" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Malik, S.; Venezian, J.; Lobov, A.; Heidenreich, M.; Garcia-Seisdedos, H.; Yeates, T. O.; Shiber, A.; Levy, E. D.</p>
        <p class="info">Score: 27.9, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576408' target='https://doi.org/10.1101/2024.01.20.576408'> 10.1101/2024.01.20.576408</a></p>
        <p class="abstract">The assembly of proteins into functional complexes is critical to lifes processes. While textbooks depict complex assembly as occurring between fully synthesized proteins, we know today that thousands of proteins in the human proteome assemble co-translationally during their synthesis. Why this process takes place, however, remains unknown. We show that co-translational assembly is governed by biophysical and structural characteristics of the protein complex, and involves mutually stabilized, intertwined subunits. Consequently, these subunits are also co-regulated across the central dogma, from transcription to protein degradation. Leveraging structural signatures with AlphaFold2-based predictions enables us to accurately predict co-translational assembly on a proteome-wide scale, which we validated by ribosome profiling, genetic perturbations, and smFISH experiments. Notably, the latter showed that co-translationally assembling subunits exhibit co-localized mRNAs. This work unveils a fundamental connection between protein structure and the translation process, highlighting the overarching impact of three-dimensional structure on gene expression, mRNA localization, and proteostasis.

One Sentence SummaryProtein complexes with topologically intertwined subunits require co-translational assembly and synchronized proteostasis of subunits, with implications in protein stability, mRNA localization, and evolution.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=195 SRC=&#34;FIGDIR/small/576408v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (62K):
org.highwire.dtl.DTLVardef@106b9a8org.highwire.dtl.DTLVardef@86f034org.highwire.dtl.DTLVardef@103fcc2org.highwire.dtl.DTLVardef@c74d15_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577913">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577913" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577913">
        <p class="paperTitle">A constraint-based framework to reconstruct interaction networks in microbial communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577913" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577913" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oftadeh, O.; Sahin, A.; Vayena, E.; Hatzimanikatis, V.</p>
        <p class="info">Score: 7.9, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577913' target='https://doi.org/10.1101/2024.01.30.577913'> 10.1101/2024.01.30.577913</a></p>
        <p class="abstract">Microbial communities live in diverse habitats and significantly impact our health and the environment. However, the principles that govern their formation and evolution remain poorly understood. A crucial step in studying microbial communities is to identify the potential metabolic interactions between the community members, such as competition for nutrients or cross-feeding. Due to the size and complexity of the metabolic network of each organism, there may be a variety of connections between each pair of organisms, which poses a challenge to unraveling the metabolic interactions. Here, we present ReMIND, a computational framework to reconstruct the interaction networks in microbial communities based on the metabolic capabilities of individual organisms. We applied ReMIND to a well-studied uranium-reducing community and the honeybee gut microbiome. Our results provide new perspectives on the evolutionary forces that shape these ecosystems and the trade-off between metabolite exchange and biomass yield. By enumerating alternative interaction networks, we systematically identified the most likely metabolites to be exchanged and highlighted metabolites that could mediate competitive interactions. We envision that ReMIND will help characterize the metabolic capacity of individual members and elucidate metabolic interactions in diverse communities, thus holds the potential to guide many applications in precision medicine and synthetic ecology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577856">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577856" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577856">
        <p class="paperTitle">Nonreciprocal synchronization in embryonic oscillator ensembles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577856" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577856" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ho, C.; Jutras-Dube, L.; Zhao, M.; Mönke, G.; Kiss, I. Z.; Francois, P.; Aulehla, A.</p>
        <p class="info">Score: 7.5, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577856' target='https://doi.org/10.1101/2024.01.29.577856'> 10.1101/2024.01.29.577856</a></p>
        <p class="abstract">Synchronization of coupled oscillators is a universal phenomenon encountered across different scales and contexts e.g., chemical wave patterns, superconductors and the unison applause we witness in concert halls. The existence of common underlying coupling rules define universality classes, revealing a fundamental sameness between seemingly distinct systems. Identifying rules of synchronization in any particular setting is hence of paramount relevance. Here, we address the coupling rules within an embryonic oscillator ensemble linked to vertebrate embryo body axis segmentation. In vertebrates, the periodic segmentation of the body axis involves synchronized signaling oscillations in cells within the presomitic mesoderm (PSM), from which somites, the pre-vertebrae, form. At the molecular level, it is known that intact Notch-signaling and cell-to-cell contact is required for synchronization between PSM cells. However, an understanding of the coupling rules is still lacking. To identify these, we develop a novel experimental assay that enables direct quantification of synchronization dynamics within mixtures of oscillating cell ensembles, for which the initial input frequency and phase distribution are known. Our results reveal a &#34;winner-takes-it-all&#34; synchronization outcome i.e., the emerging collective rhythm matches one of the input rhythms. Using a combination of theory and experimental validation, we develop a new coupling model, the &#34;Rectified Kuramoto&#34; (ReKu) model, characterized by a phase-dependent, non-reciprocal interaction in the coupling of oscillatory cells. Such non-reciprocal synchronization rules reveal fundamental similarities between embryonic oscillators and a class of collective behaviours seen in neurons and fireflies, where higher level computations are performed and linked to non-reciprocal synchronization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577229">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577229" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577229">
        <p class="paperTitle">SignalingProfiler 2.0: a network-based approach to bridge multi-omics data to phenotypic hallmarks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577229" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577229" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venafra, V.; Sacco, F.; Perfetto, L.</p>
        <p class="info">Score: 2.9, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577229' target='https://doi.org/10.1101/2024.01.25.577229'> 10.1101/2024.01.25.577229</a></p>
        <p class="abstract">Unraveling the cellular signaling remodeling upon a perturbation is a fundamental challenge to understand disease mechanisms and to identify potential drug targets. In this pursuit, computational tools that generate mechanistic hypotheses from multi-omics data have invaluable potential. Here we present SignalingProfiler 2.0, a multi-step pipeline to systematically derive context-specific signaling models by integrating proteogenomic data with prior knowledge-causal networks. This is a freely accessible and flexible tool that incorporates statistical, footprint-based, and graph algorithms to accelerate the integration and interpretation of multi-omics data. Through benchmarking and rigorous parameter selection on a proof-of-concept study, performed in metformin-treated breast cancer cells, we demonstrate the tools ability to generate a hierarchical mechanistic network that recapitulates novel and known drug-perturbed signaling and phenotypic outcomes. In summary, SignalingProfiler 2.0 addresses the emergent need to derive biologically relevant information from complex multi-omics data by extracting interpretable networks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.575880">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.575880" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.575880">
        <p class="paperTitle">Multistability and predominant double-positive states in a four node mutually repressive network: a case study of Th1/Th2/Th17/T-reg differentiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.575880" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.575880" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duddu, A. S.; Andreas, E.; BV, H.; Grover, K.; Singh, V. R.; Hari, K.; Jhunjhunwala, S.; Cummins, B.; Gedeon, T.; Jolly, M. K.</p>
        <p class="info">Score: 2.6, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.575880' target='https://doi.org/10.1101/2024.01.30.575880'> 10.1101/2024.01.30.575880</a></p>
        <p class="abstract">Elucidating the emergent dynamics of complex regulatory networks enabling cellular differentiation is crucial to understand embryonic development and suggest strategies for synthetic circuit design. A well-studied network motif often driving cellular decisions is a toggle switch - a set of two mutually inhibitory lineage-specific transcription factors A and B. A toggle switch often enables two possible states mutually exclusive states - (high A, low B) and (low A, high B) - from a common progenitor cell. However, the dynamics of networks enabling differentiation of more than two cell types from a progenitor cell is not well-studied. Here, we investigate the dynamics of four master regulators A, B, C and D inhibiting each other, thus forming a toggle tetrahedron. Our simulations show that a toggle tetrahedron predominantly allows for co-existence of six &#39;double positive&#39; or hybrid states where two of the nodes are expressed relatively high as compared to the remaining two - (high A, high B, low C, low D), (high A, low B, high C, low D), (high A, low B, low C, high D), (low A, high B, high C, low D), (low A, low B, high C, high D) and (low A, high B, low C, high D). Stochastic simulations showed state-switching among these phenotypes, indicating phenotypic plasticity. Finally, we apply our results to understand the differentiation of naive CD4&#43; T cells into Th1, Th2, Th17 and Treg subsets, suggesting Th1/Th2/Th17/Treg decision-making to be a two-step process. Our results reveal multistable dynamics and establish the stable co-existence of hybrid cell-states, offering a potential explanation for simultaneous differentiation of multipotent naive CD4&#43; T cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578201">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578201" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578201">
        <p class="paperTitle">Thermodynamics determines the coupling between growth and byproduct production</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578201" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578201" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oftadeh, O.; Hatzimanikatis, V.</p>
        <p class="info">Score: 2.2, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578201' target='https://doi.org/10.1101/2024.01.31.578201'> 10.1101/2024.01.31.578201</a></p>
        <p class="abstract">Genetic manipulation of cells to couple byproduct production and growth rate is important in bioengineering and biotechnology. In this way, we can use growth rate as a selective pressure, where the mutants with higher growth have higher production capacity. Computational methods have been proposed to find knockouts that couple growth and byproduct production. However, none of these methods consider the energetic and thermodynamic feasibility of such knockout strategies. Furthermore, there is no computational study of how variations in metabolite concentrations affect the coupling between growth and byproduct formation. One of the computational methods to find knockouts that couple growth and byproduct formation is OptKnock. OptKnock is a bi-level optimization problem. Here, we integrated thermodynamic constraints into the bilevel formulation of OptKnock to create TOptKnock. We show that the computational efficiency of TOptKnock is comparable to that of OptKnock. TOptKnock can account for the thermodynamic viability of the knockouts and examine how variations in metabolite concentrations affect the coupling. We have shown that the coupling between growth and byproduct formation can change in response to variations in concentrations. Thus, a knockout strategy might be optimal for one intracellular condition but suboptimal for another. If metabolomics data are available, TOptKnock can search for optimal knockout interventions under the given condition. We also envision that the TOptKnock framework will help develop strategies for manipulating metabolite concentrations to couple growth and byproduct formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575159">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575159" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575159">
        <p class="paperTitle">Unraveling the oscillatory dynamics of mRNA metabolism and chromatin accessibility during the cell cycle through integration of single-cell multiomic data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575159" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575159" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nariya, M. K.; Santiago-Algarra, D.; Tassy, O.; Cerciat, M.; Ye, T.; Riba, A.; Molina, N.</p>
        <p class="info">Score: 16.3, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575159' target='https://doi.org/10.1101/2024.01.11.575159'> 10.1101/2024.01.11.575159</a></p>
        <p class="abstract">The cell cycle is a highly regulated process that ensures the accurate replication and transmission of genetic information from one generation of cells to the next. It is a fundamental biological process and plays a crucial role in development, growth, and maintenance of all living organisms, and its dysregulation can lead to a number of pathologies such as autoimmune diseases, neurodegenerative diseases, and cancer. In this work we present a novel approach to study the gene expression and chromatin accessibility dynamics during the cell cycle in mouse embryonic stem cells. To achieve this, we combined high-depth single-cell multiome sequencing, biophysical modeling, and advanced deep learning techniques. First, we used DeepCycle, a deep learning tool that assigns a cell cycle phase to every cell based on its spliced and unspliced mRNA levels. We then developed a biophysical model that describes the dynamics of gene-specific mRNA transcription, nuclear export, and degradation rates during the cell cycle. Our model obtains genome-wide estimates of mRNA transcription, nuclear retention, and degradation half-lives of genes exhibiting oscillatory dynamics in mESCs. The key feature of single-cell multiome sequencing is that it simultaneously provides readouts for gene expression as well as chromatin accessibility in the same cells. By applying our approach to these data we showcase first-of-its-kind exploration of chromatin accessibility during the cell cycle at high temporal resolution. We believe that our work will be pivotal in formulating a coherent quantitative description of mRNA metabolism.

HighlightsO_LIIntegration of single-cell and single-nucleus multiome profiling with deep learning and biophysical modeling uncovers cell cycle regulation of RNA metabolism
C_LIO_LIGenome-wide estimation of mRNA transcription, nuclear export, and degradation rates during the cell cycle in mESCs
C_LIO_LIIdentification of a prominent wave of transcription during early G1 in unperturbed asynchronously proliferating mESCs
C_LIO_LICharacterization of temporal waves of post-transcriptional mRNA regulation controlling both mRNA degradation and nuclear export during the cell cycle
C_LIO_LIChromatin accessibility dynamics within TF footprints reveal key cell cycle regulators
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
