<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The fitness cost of spurious phosphorylation
Authors: Bradley, D.; Hogrebe, A.; Dandage, R.; Dube, A. K.; Leutert, M.; Dionne, U.; Chang, A.; Villen, J.; Landry, C. R.
Score: 17.0, Published: 2023-10-10 DOI: 10.1101/2023.10.08.561337
The fidelity of signal transduction requires the binding of regulatory molecules to their cognate targets. However, the crowded cell interior risks off-target interactions between proteins that are functionally unrelated. How such off-target interactions impact fitness is not generally known, but quantifying this is required to understand the constraints faced by cell systems as they evolve.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="The fitness cost of spurious phosphorylation
Authors: Bradley, D.; Hogrebe, A.; Dandage, R.; Dube, A. K.; Leutert, M.; Dionne, U.; Chang, A.; Villen, J.; Landry, C. R.
Score: 17.0, Published: 2023-10-10 DOI: 10.1101/2023.10.08.561337
The fidelity of signal transduction requires the binding of regulatory molecules to their cognate targets. However, the crowded cell interior risks off-target interactions between proteins that are functionally unrelated. How such off-target interactions impact fitness is not generally known, but quantifying this is required to understand the constraints faced by cell systems as they evolve." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-18T10:38:41+00:00" />
<meta property="article:modified_time" content="2023-10-18T10:38:41+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="The fitness cost of spurious phosphorylation
Authors: Bradley, D.; Hogrebe, A.; Dandage, R.; Dube, A. K.; Leutert, M.; Dionne, U.; Chang, A.; Villen, J.; Landry, C. R.
Score: 17.0, Published: 2023-10-10 DOI: 10.1101/2023.10.08.561337
The fidelity of signal transduction requires the binding of regulatory molecules to their cognate targets. However, the crowded cell interior risks off-target interactions between proteins that are functionally unrelated. How such off-target interactions impact fitness is not generally known, but quantifying this is required to understand the constraints faced by cell systems as they evolve."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "The fitness cost of spurious phosphorylation\nAuthors: Bradley, D.; Hogrebe, A.; Dandage, R.; Dube, A. K.; Leutert, M.; Dionne, U.; Chang, A.; Villen, J.; Landry, C. R.\nScore: 17.0, Published: 2023-10-10 DOI: 10.1101/2023.10.08.561337\nThe fidelity of signal transduction requires the binding of regulatory molecules to their cognate targets. However, the crowded cell interior risks off-target interactions between proteins that are functionally unrelated. How such off-target interactions impact fitness is not generally known, but quantifying this is required to understand the constraints faced by cell systems as they evolve.",
  "keywords": [
    
  ],
  "articleBody": " The fitness cost of spurious phosphorylation\nAuthors: Bradley, D.; Hogrebe, A.; Dandage, R.; Dube, A. K.; Leutert, M.; Dionne, U.; Chang, A.; Villen, J.; Landry, C. R.\nScore: 17.0, Published: 2023-10-10 DOI: 10.1101/2023.10.08.561337\nThe fidelity of signal transduction requires the binding of regulatory molecules to their cognate targets. However, the crowded cell interior risks off-target interactions between proteins that are functionally unrelated. How such off-target interactions impact fitness is not generally known, but quantifying this is required to understand the constraints faced by cell systems as they evolve. Here, we use the model organism S. cerevisiae to inducibly express tyrosine kinases. Because yeast lacks bona fide tyrosine kinases, most of the resulting tyrosine phosphorylation is spurious. This provides a suitable system to measure the impact of artificial protein interactions on fitness. We engineered 44 yeast strains each expressing a tyrosine kinase, and quantitatively analysed their phosphoproteomes. This analysis resulted in [~]30,000 phosphosites mapping to [~]3,500 proteins. Examination of the fitness costs in each strain revealed a strong correlation between the number of spurious pY sites and decreased growth. Moreover, the analysis of pY effects on protein structure and on protein function revealed over 1000 pY events that we predict to be deleterious. However, we also find that a large number of the spurious pY sites have a negligible effect on fitness, possibly because of their low stoichiometry. This result is consistent with our evolutionary analyses demonstrating a lack of phosphotyrosine counter-selection in species with bona fide tyrosine kinases. Taken together, our results suggest that, alongside the risk for toxicity, the cell can tolerate a large degree of non-functional crosstalk as interaction networks evolve.\nImAge: an imaging approach to quantitate aging and rejuvenation.\nAuthors: Alvarez-Kuglen, M.; Rodriguez, D.; Qin, H.; Ninomiya, K.; Fiengo, L.; Farhy, C.; Havas, A.; Hsu, W.-M.; Feng, G.-S.; Roberts, A.; Anderson, R. M.; Serrano, M.; Adams, P. D.; Sharpee, T. O.; Terskikh, A.\nScore: 18.9, Published: 2023-09-24 DOI: 10.1101/2022.10.16.512441\nBiomarkers of biological age that predict the risk of disease and expected lifespan better than chronological age are key to efficient and cost-effective healthcare1-3. To advance a personalized approach to healthcare, such biomarkers must perform on the individual rather than population level, demonstrate single cell resolution, and provide scalable and cost-effective measurements. We developed a novel approach - image-based chromatin and epigenetic age (ImAge), that utilizes image texture features based on the patterns of chromatin and epigenetic marks in single nuclei. We observed the emergence of intrinsic trajectories of ImAge using dimensionality reduction without regression on chronological age. ImAge was correlated with chronological age in all tissues and organs examined and was consistent with the expected acceleration and/or deceleration of biological age in chronologically identical mice treated with chemotherapy or following a caloric restriction regimen, respectively. ImAge from chronologically identical mice inversely correlated with their locomotor activity (greater activity for younger ImAge), consistent with the essential role of locomotion as an aging biomarker. Finally, we demonstrated that ImAge is reduced upon partial reprogramming in vivo following transient expression of OSKM cassette in the liver and skeletal muscles of old mice and validated the power of ImAge to assess the heterogeneity of reprogramming. We propose that ImAge represents the first-in-class individual-level biomarker of aging and rejuvenation with single-cell resolution.\nSmurfness-based two-phase model of ageing helps deconvolve the ageing transcriptional signature\nAuthors: Zane, F.; Bouzid, H.; Sosa Marmol, S.; Brazane, M.; Besse, S.; Molina, J. L.; Cansell, C.; Aprahamian, F.; Durand, S.; Ayache, J.; Antoniewski, C.; CARRE, C.; Todd, N.; Rera, M.\nScore: 12.9, Published: 2023-09-27 DOI: 10.1101/2022.11.22.517330\nAgeing is characterised at the molecular level by six transcriptional hallmarks of ageing, that are commonly described as progressively affected as time passes. By contrast, the Smurf assay separates high-and-constant-mortality risk individuals from healthy, zero-mortality risk individuals, based on increased intestinal permeability. Performing whole body total RNA sequencing, we found that Smurfness distinguishes transcriptional changes associated with chronological age from those associated with biological age. We show that transcriptional heterogeneity increases with chronological age in non-Smurf individuals preceding the other five hallmarks of ageing, that are specifically associated with the Smurf state. Using this approach, we also devise targeted pro-longevity genetic interventions delaying entry in the Smurf state. We anticipate that increased attention to the evolutionary conserved Smurf phenotype will bring about significant advances in our understanding of the mechanisms of ageing. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=150 HEIGHT=200 SRC=\"FIGDIR/small/517330v5_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (35K): org.highwire.dtl.DTLVardef@1162afeorg.highwire.dtl.DTLVardef@60a6f9org.highwire.dtl.DTLVardef@17255d7org.highwire.dtl.DTLVardef@161a454_HPS_FORMAT_FIGEXP M_FIG The two-phase model of ageing allows to study separately the effect of chronological and physiological age. (A) Classic approaches for studying ageing tend to consider it as a black box affecting all individuals progressively from birth to death. Instead, the Smurf phenotype shows that life can be divided into two consecutive phases separated by an abrupt transition. (B) All individuals undergo this transition at a different moment in their life, prior to death. This allows us to switch from population based approaches, comparing bulks of age-matched individuals through time, to individuals-centred approaches relying on direct access to their transition status. (C) Such paradigm shift shows that hallmarks of ageing long thought to progressively change with age are actually mostly affected in a growing proportion of Smurfs, allowing for the identification of the chain of events accompanying ageing and death from natural causes. (D) By studying the behaviour of the ageing transcriptome as a function of chronological age and Smurfness separately, we demonstrate that the progressively changing transcriptional ageing signature, as described in Frenk \u0026 Houseley (2018), is in fact the convolution changes accompanying chronological age signature (increased transcriptional noise) and changes associated with Smurfness (or biological age) signature (increased stress response and inflammation, decreased expression of ribosomal and mitochondrial genes). We also identified a hallmark partially associated with only old Smurfs (ATH5), suggesting that chronological age can affect, late in life, the Smurf response. C_FIG\nVirus-host protein co-expression networks reveal temporal organization and strategies of viral infection\nAuthors: Guantes, R.; Aguirre, J.\nScore: 1.1, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561729\nViral replication is a complex dynamical process involving the global remodelling of the host cellular machinery across several stages. In this study, we provide a unified view of the virus-host interaction at the proteome level reconstructing protein co-expression networks from quantitative temporal data of four large DNA viruses. We take advantage of a formal framework, the theory of interacting networks, to describe the viral infection as a dynamical system taking place on a network of networks where perturbations induced by viral proteins spread to hijack the host proteome for the virus benefit. Our methodology demonstrates how the viral replication cycle can be effectively examined as a complex interaction between protein networks, providing useful insights into the viral and hosts temporal organization and strategies, key protein nodes targeted by the virus and dynamical bottlenecks during the course of the infection.\nComputational modeling establishes mechanotransduction as a potent entrainment cue for the mammalian circadian clock\nAuthors: Francis, E. A.; Rangamani, P.\nScore: 1.0, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561563\nMechanotransduction, which is the integration of mechanical signals from the cells external environment to changes in intracellular signaling, governs many cellular functions. Recent studies show that the mechanical state of the cell is also coupled to the cellular circadian clock. To investigate possible interactions between circadian rhythms and cellular mechanotransduction, we developed a computational model that integrates these two pathways. Combining previous models of mechan-otransduction and circadian clocks, we postulated that translocation of YAP/TAZ and MRTF into the nucleus leads to altered expression of circadian proteins. Simulations from our model predict that lower levels of cytoskeletal activity are associated with longer circadian oscillation periods and higher oscillation amplitudes, consistent with recent experimental observations. Furthermore, accumulation of YAP/TAZ and MRTF in the nucleus causes circadian oscillations to decay. These effects hold both at the single-cell level and within a population-level framework. Finally, we investigated the effects of mutations in YAP or lamin A, the latter of which lead to a class of diseases known as laminopathies. Oscillations in PER/CRY and BMAL1 are substantially weaker in populations of cells with in silico mutations in YAP and lamin A, suggesting that defects in mechanotransduction can disrupt the circadian clock in certain disease states. However, we find that normal oscillatory behavior can be rescued by lowering substrate stiffness, indicating a possible biological compensatory mechanism. Thus our study identifies that mechanotransduction could be a potent entrainment cue for cellular clocks and this crosstalk can be leveraged to rescue the circadian clock in disease states.\n",
  "wordCount" : "1407",
  "inLanguage": "en",
  "datePublished": "2023-10-18T10:38:41Z",
  "dateModified": "2023-10-18T10:38:41Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on October 18, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.561337">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.561337" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.561337">
        <p class="paperTitle">The fitness cost of spurious phosphorylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.561337" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.561337" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bradley, D.; Hogrebe, A.; Dandage, R.; Dube, A. K.; Leutert, M.; Dionne, U.; Chang, A.; Villen, J.; Landry, C. R.</p>
        <p class="info">Score: 17.0, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.561337' target='https://doi.org/10.1101/2023.10.08.561337'> 10.1101/2023.10.08.561337</a></p>
        <p class="abstract">The fidelity of signal transduction requires the binding of regulatory molecules to their cognate targets. However, the crowded cell interior risks off-target interactions between proteins that are functionally unrelated. How such off-target interactions impact fitness is not generally known, but quantifying this is required to understand the constraints faced by cell systems as they evolve. Here, we use the model organism S. cerevisiae to inducibly express tyrosine kinases. Because yeast lacks bona fide tyrosine kinases, most of the resulting tyrosine phosphorylation is spurious. This provides a suitable system to measure the impact of artificial protein interactions on fitness. We engineered 44 yeast strains each expressing a tyrosine kinase, and quantitatively analysed their phosphoproteomes. This analysis resulted in [~]30,000 phosphosites mapping to [~]3,500 proteins. Examination of the fitness costs in each strain revealed a strong correlation between the number of spurious pY sites and decreased growth. Moreover, the analysis of pY effects on protein structure and on protein function revealed over 1000 pY events that we predict to be deleterious. However, we also find that a large number of the spurious pY sites have a negligible effect on fitness, possibly because of their low stoichiometry. This result is consistent with our evolutionary analyses demonstrating a lack of phosphotyrosine counter-selection in species with bona fide tyrosine kinases. Taken together, our results suggest that, alongside the risk for toxicity, the cell can tolerate a large degree of non-functional crosstalk as interaction networks evolve.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.10.16.512441">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.10.16.512441" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.10.16.512441">
        <p class="paperTitle">ImAge: an imaging approach to quantitate aging and rejuvenation.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.10.16.512441" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.10.16.512441" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alvarez-Kuglen, M.; Rodriguez, D.; Qin, H.; Ninomiya, K.; Fiengo, L.; Farhy, C.; Havas, A.; Hsu, W.-M.; Feng, G.-S.; Roberts, A.; Anderson, R. M.; Serrano, M.; Adams, P. D.; Sharpee, T. O.; Terskikh, A.</p>
        <p class="info">Score: 18.9, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.10.16.512441' target='https://doi.org/10.1101/2022.10.16.512441'> 10.1101/2022.10.16.512441</a></p>
        <p class="abstract">Biomarkers of biological age that predict the risk of disease and expected lifespan better than chronological age are key to efficient and cost-effective healthcare1-3. To advance a personalized approach to healthcare, such biomarkers must perform on the individual rather than population level, demonstrate single cell resolution, and provide scalable and cost-effective measurements. We developed a novel approach - image-based chromatin and epigenetic age (ImAge), that utilizes image texture features based on the patterns of chromatin and epigenetic marks in single nuclei. We observed the emergence of intrinsic trajectories of ImAge using dimensionality reduction without regression on chronological age. ImAge was correlated with chronological age in all tissues and organs examined and was consistent with the expected acceleration and/or deceleration of biological age in chronologically identical mice treated with chemotherapy or following a caloric restriction regimen, respectively. ImAge from chronologically identical mice inversely correlated with their locomotor activity (greater activity for younger ImAge), consistent with the essential role of locomotion as an aging biomarker. Finally, we demonstrated that ImAge is reduced upon partial reprogramming in vivo following transient expression of OSKM cassette in the liver and skeletal muscles of old mice and validated the power of ImAge to assess the heterogeneity of reprogramming. We propose that ImAge represents the first-in-class individual-level biomarker of aging and rejuvenation with single-cell resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.11.22.517330">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.11.22.517330" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.11.22.517330">
        <p class="paperTitle">Smurfness-based two-phase model of ageing helps deconvolve the ageing transcriptional signature</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.11.22.517330" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.11.22.517330" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zane, F.; Bouzid, H.; Sosa Marmol, S.; Brazane, M.; Besse, S.; Molina, J. L.; Cansell, C.; Aprahamian, F.; Durand, S.; Ayache, J.; Antoniewski, C.; CARRE, C.; Todd, N.; Rera, M.</p>
        <p class="info">Score: 12.9, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.11.22.517330' target='https://doi.org/10.1101/2022.11.22.517330'> 10.1101/2022.11.22.517330</a></p>
        <p class="abstract">Ageing is characterised at the molecular level by six transcriptional  hallmarks of ageing, that are commonly described as progressively affected as time passes. By contrast, the  Smurf assay separates high-and-constant-mortality risk individuals from healthy, zero-mortality risk individuals, based on increased intestinal permeability. Performing whole body total RNA sequencing, we found that Smurfness distinguishes transcriptional changes associated with chronological age from those associated with biological age. We show that transcriptional heterogeneity increases with chronological age in non-Smurf individuals preceding the other five hallmarks of ageing, that are specifically associated with the Smurf state. Using this approach, we also devise targeted pro-longevity genetic interventions delaying entry in the Smurf state. We anticipate that increased attention to the evolutionary conserved Smurf phenotype will bring about significant advances in our understanding of the mechanisms of ageing.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=150 HEIGHT=200 SRC=&#34;FIGDIR/small/517330v5_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (35K):
org.highwire.dtl.DTLVardef@1162afeorg.highwire.dtl.DTLVardef@60a6f9org.highwire.dtl.DTLVardef@17255d7org.highwire.dtl.DTLVardef@161a454_HPS_FORMAT_FIGEXP  M_FIG The two-phase model of ageing allows to study separately the effect of chronological and physiological age.

(A) Classic approaches for studying ageing tend to consider it as a black box affecting all individuals progressively from birth to death. Instead, the Smurf phenotype shows that life can be divided into two consecutive phases separated by an abrupt transition. (B) All individuals undergo this transition at a different moment in their life, prior to death. This allows us to switch from population based approaches, comparing bulks of age-matched individuals through time, to individuals-centred approaches relying on direct access to their transition status. (C) Such paradigm shift shows that hallmarks of ageing long thought to progressively change with age are actually mostly affected in a growing proportion of Smurfs, allowing for the identification of the chain of events accompanying ageing and death from natural causes. (D) By studying the behaviour of the ageing transcriptome as a function of chronological age and Smurfness separately, we demonstrate that the progressively changing transcriptional ageing signature, as described in Frenk &amp; Houseley (2018), is in fact the convolution changes accompanying chronological age signature (increased transcriptional noise) and changes associated with Smurfness (or biological age) signature (increased stress response and inflammation, decreased expression of ribosomal and mitochondrial genes). We also identified a hallmark partially associated with only old Smurfs (ATH5), suggesting that chronological age can affect, late in life, the Smurf response.

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561729">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561729" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561729">
        <p class="paperTitle">Virus-host protein co-expression networks reveal temporal organization and strategies of viral infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561729" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561729" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guantes, R.; Aguirre, J.</p>
        <p class="info">Score: 1.1, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561729' target='https://doi.org/10.1101/2023.10.10.561729'> 10.1101/2023.10.10.561729</a></p>
        <p class="abstract">Viral replication is a complex dynamical process involving the global remodelling of the host cellular machinery across several stages. In this study, we provide a unified view of the virus-host interaction at the proteome level reconstructing protein co-expression networks from quantitative temporal data of four large DNA viruses. We take advantage of a formal framework, the theory of interacting networks, to describe the viral infection as a dynamical system taking place on a network of networks where perturbations induced by viral proteins spread to hijack the host proteome for the virus benefit. Our methodology demonstrates how the viral replication cycle can be effectively examined as a complex interaction between protein networks, providing useful insights into the viral and hosts temporal organization and strategies, key protein nodes targeted by the virus and dynamical bottlenecks during the course of the infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561563">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561563" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561563">
        <p class="paperTitle">Computational modeling establishes mechanotransduction as a potent entrainment cue for the mammalian circadian clock</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561563" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561563" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Francis, E. A.; Rangamani, P.</p>
        <p class="info">Score: 1.0, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561563' target='https://doi.org/10.1101/2023.10.09.561563'> 10.1101/2023.10.09.561563</a></p>
        <p class="abstract">Mechanotransduction, which is the integration of mechanical signals from the cells external environment to changes in intracellular signaling, governs many cellular functions. Recent studies show that the mechanical state of the cell is also coupled to the cellular circadian clock. To investigate possible interactions between circadian rhythms and cellular mechanotransduction, we developed a computational model that integrates these two pathways. Combining previous models of mechan-otransduction and circadian clocks, we postulated that translocation of YAP/TAZ and MRTF into the nucleus leads to altered expression of circadian proteins. Simulations from our model predict that lower levels of cytoskeletal activity are associated with longer circadian oscillation periods and higher oscillation amplitudes, consistent with recent experimental observations. Furthermore, accumulation of YAP/TAZ and MRTF in the nucleus causes circadian oscillations to decay. These effects hold both at the single-cell level and within a population-level framework. Finally, we investigated the effects of mutations in YAP or lamin A, the latter of which lead to a class of diseases known as laminopathies. Oscillations in PER/CRY and BMAL1 are substantially weaker in populations of cells with in silico mutations in YAP and lamin A, suggesting that defects in mechanotransduction can disrupt the circadian clock in certain disease states. However, we find that normal oscillatory behavior can be rescued by lowering substrate stiffness, indicating a possible biological compensatory mechanism. Thus our study identifies that mechanotransduction could be a potent entrainment cue for cellular clocks and this crosstalk can be leveraged to rescue the circadian clock in disease states.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
