<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 72.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 72.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-31T10:38:19+00:00" />
<meta property="article:modified_time" content="2023-12-31T10:38:19+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 72.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Open-ST: High-resolution spatial transcriptomics in 3D\nAuthors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.\nScore: 72.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554\nSpatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods.",
  "keywords": [
    
  ],
  "articleBody": " Open-ST: High-resolution spatial transcriptomics in 3D\nAuthors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.\nScore: 72.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554\nSpatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods. Here, we report Open-ST, a sequencing-based, open-source experimental and computational resource to address these challenges and to study the molecular organization of tissues in 3D. In mouse brain, Open-ST captured transcripts at subcellular resolution and reconstructed cell types. In primary tumor and patient-matched healthy/metastatic lymph nodes, Open-ST captured the diversity of immune, stromal and tumor populations in space. Distinct cell states were organized around cell-cell communication hotspots in the tumor, but not the metastasis. Strikingly, the 3D reconstruction and multimodal analysis of the metastatic lymph node revealed spatially contiguous structures not visible in 2D and potential biomarkers precisely at the 3D tumor/lymph node boundary. We anticipate Open-ST to accelerate the identification of spatial molecular mechanisms in 2D and 3D.\nHighly multiplexed, image-based pooled screens in primary cells and tissues with PerturbView\nAuthors: Kudo, T.; Meireles, A. M.; Moncada, R.; Chen, Y.; Wu, P.; Gould, J.; Hu, X.; Kornfeld, O.; Jesudason, R.; Foo, C.; Hockendorf, B.; Bravo, H. C.; Town, J. P.; Wei, R.; Rios, A.; Chandrasekar, V.; Heinlein, M.; Cai, S.; Lu, C. S.; Celen, C.; Kljavin, N.; Jiang, J.; Hleap, J. S.; Kayagaki, N.; Melo, F. d. S. e.; McGinnis, L.; Li, B.; Singh, A.; Garraway, L.; Rozenblatt-Rosen, O.; Regev, A.; Lubeck, E.\nScore: 41.7, Published: 2023-12-26 DOI: 10.1101/2023.12.26.573143\nOptical pooled screening (OPS) is a highly scalable method for linking image-based phenotypes with cellular perturbations. However, it has thus far been restricted to relatively low-plex phenotypic readouts in cancer cell lines in culture, due to limitations associated with in situ sequencing (ISS) of perturbation barcodes. Here, we developed PerturbView, an OPS technology that leverages in vitro transcription (IVT) to amplify barcodes prior to ISS, enabling screens with highly multiplexed phenotypic readouts across diverse systems, including primary cells and tissues. We demonstrate PerturbView in iPSC-derived neurons, primary immune cells, and tumor tissue sections from animal models. In a screen of immune signaling pathways in primary bone marrow-derived macrophages, PerturbView uncovered both known and novel regulators of NF{kappa}B signaling. Furthermore, we combined PerturbView with spatial transcriptomics in tissue sections from a mouse xenograft model, paving the way to in vivo screens with rich optical and transcriptomic phenotypes. PerturbView broadens the scope of OPS to a wide range of models and applications.\nDynamics of single-cell protein covariation during epithelial-mesenchymal transition\nAuthors: Khan, S.; Conover, R.; Asthagiri, A. R.; Slavov, N.\nScore: 29.7, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572913\nPhysiological processes, such as epithelial-mesenchymal transition (EMT), are mediated by changes in protein interactions. These changes may be better reflected in protein covariation within cellular cluster than in the temporal dynamics of cluster-average protein abundance. To explore this possibility, we quantified proteins in single human cells undergoing EMT. Covariation analysis of the data revealed that functionally coherent protein clusters dynamically changed their protein-protein correlations without concomitant changes in cluster-average protein abundance. These dynamics of protein-protein correlations were monotonic in time and delineated protein modules functioning in actin cytoskeleton organization, energy metabolism and protein transport. These protein modules are defined by protein covariation within the same time point and cluster and thus reflect biological regulation masked by the cluster-average protein dynamics. Thus, protein correlation dynamics across single cells offer a window into protein regulation during physiological transitions. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=132 SRC=\"FIGDIR/small/572913v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (79K): org.highwire.dtl.DTLVardef@151d03aorg.highwire.dtl.DTLVardef@3da81borg.highwire.dtl.DTLVardef@1e79733org.highwire.dtl.DTLVardef@177bc33_HPS_FORMAT_FIGEXP M_FIG C_FIG\nOncogenic PIK3CA corrupts growth factor signaling specificity\nAuthors: Madsen, R. R.; Le Marois, A.; Mruk, O.; Voliotis, M.; Yin, S.; Sufi, J.; Qin, X.; Zhao, S. J.; Gorczynska, J.; Morelli, D.; Davidson, L.; Sahai, E.; Korolchuk, V. I.; Tape, C. J.; Vanhaesebroeck, B.\nScore: 27.3, Published: 2023-12-26 DOI: 10.1101/2023.12.23.573207\nPathological activation of the PI3K/AKT pathway is among the most frequent defects in human cancer and is also the cause of rare overgrowth disorders. Yet, there is currently no systematic understanding of the quantitative flow of information within PI3K/AKT signaling and how it is perturbed by disease-causing mutations. Here, we develop scalable, single-cell approaches for systematic analyses of signal processing within the PI3K pathway, enabling precise calculations of its information transfer for different growth factors. Using genetically-engineered human cell models with allele dose-dependent expression of PIK3CAH1047R, we show that this oncogene is not a simple, constitutive pathway activator but a context-dependent modulator of extracellular signal transfer. PIK3CAH1047Rreduces information transmission downstream of IGF1 while selectively enhancing EGF-induced signaling and transcriptional responses. This leads to a gross reduction in signaling specificity, akin to \"blurred\" signal perception. The associated increase in signaling heterogeneity promotes phenotypic diversity in a human cervical cancer cell line model and in human induced pluripotent stem cells. Collectively, these findings and the accompanying methodological advances lay the foundations for a systematic mapping of the quantitative mechanisms of PI3K/AKT-dependent signal processing and phenotypic control in health and disease. One-sentence summarySingle-cell signaling and information theoretic analyses reveal that oncogenic PI3K/AKT activation leads to a gross reduction in signaling specificity, context-dependent EGF response amplification as well as increased phenotypic heterogeneity.\nCoupling Cell Size Regulation and Proliferation Dynamics of C.glutamicum Reveals Cell Division Based on Surface Area\nAuthors: Nieto, C.; Tauber, S.; Blobaum, L.; Vahdat, Z.; Grunberger, A.; Singh, A.\nScore: 4.3, Published: 2023-12-28 DOI: 10.1101/2023.12.26.573217\nSingle cells actively coordinate growth and division to regulate their size, yet how this size homeostasis at the single-cell level propagates over multiple generations to impact clonal expansion remains fundamentally unexplored. Classical timer models for cell proliferation (where the duration of the cell cycle is an independent variable) predict that the stochastic variation in colony size will increase monotonically over time. In stark contrast, implementing size control according to adder strategy (where on average a fixed size added from cell birth to division) leads to colony size variations that eventually decay to zero. While these results assume a fixed size of the colony-initiating progenitor cell, further analysis reveals that the magnitude of the intercolony variation in population number is sensitive to heterogeneity in the initial cell size. We validate these predictions by tracking the growth of isogenic microcolonies of Corynebacterium glutamicum in microfluidic chambers. Approximating their cell shape to a capsule, we observe that the degree of random variability in cell size is different depending on whether the cell size is quantified as per length, surface area, or volume, but size control remains an adder regardless of these size metrics. A comparison of the observed variability in the colony population with the predictions suggests that proliferation matches better with a cell division based on the cell surface. In summary, our integrated mathematical-experimental approach bridges the paradigms of single-cell size regulation and clonal expansion at the population levels. This innovative approach provides elucidation of the mechanisms of size homeostasis from the stochastic dynamics of colony size for rod-shaped microbes.\nBMI and BPH correlate with urinary microbiome diversity and lower urinary tract symptoms in men\nAuthors: Bowie, K.; Garzotto, M.; Orwoll, E.; Karstens, L.\nScore: 4.0, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571758\nSeveral studies have identified bacteria and other microbes in the bladder and lower urinary tract in the absence of infection. In women, the urinary microbiome has been associated with lower urinary tract symptoms (LUTS), however, similar studies have not been undertaken in large cohorts of men. Here we examine the urinary microbiome and its association with LUTS in a subset of 500 men aged 65 to 90 years from the Osteoporotic Fractures in Men (MrOS) study. We identified significant associations between benign prostatic hyperplasia (BPH), age, and body mass index (BMI) with several diversity metrics. Our analysis revealed complex relationships between BMI, BPH, LUTS, and alpha diversity which give insight into the intricate dynamics of the urinary microbiome. By beginning to uncover the interrelationships of BPH, BMI, LUTS, and the urinary microbiome, these results can inform future study design to better understand the heterogeneity of the male urinary microbiome.\nOptimization-Based Synthesis with Directed Cell Migration\nAuthors: Havenhill, E.; Ghosh, S.\nScore: 1.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573173\nCollective behavior of biological agents such as herds of organisms and cells is a fundamental feature in the systems biology and in the emergence of new phenomena in the biological environ-ment. Collective cell migration under a physical or chemical cue is an example of this fundamental phenomenon where individual cell migration is driven by the collective behavior of the neighboring cells and vice versa. The goal of this research is to discover the mathematical rules of collective cell migration using experimental data and testing the predictive nature of the models in independent experimental data. Such insight is made possible in this work with the hybrid use of dynamic mode decomposition (DMD) [1] and optimal control theory. Both single and multi-cellular systems are simulated, including obstacle courses, using this framework. The results of this work show how cells collectively behave during their migration and also, opens the possibility of designing robotic cells for possible therapeutic purpose where the cell trajectory can be controlled.\nDeregulation cascades in carcinogenesis\nAuthors: Perez, R. H.; Gomez, J. P.; Perez, G. J. G.; Gonzalez, A.\nScore: 1.8, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573550\nCarcinogenesis is one example of transition between two biological states involving strong gene expression rearrangements. Normal and tumor states are understood as two different states of a Gene Regulatory Network. In terms of the normal state characteristics, the transition may be thought as a deregulation process. According to a very abstract model for the transition, the available data on cancer risk support the idea of a single big jump in expression space, which may be associated to a deregulation cascade. Here, we use the measured frequency distribution of gene deregulations, and concepts from the probabilistic theory of causation in order to infer causal relations between pairs of genes and construct a deregulation causal network. Then, the deregulated genes in each sample are organized according to the network in such a way that they show the deregulation cascade behind the transition from the normal to the tumor state. The explicit analysis for prostate adenocarcinoma is given. In most cases, the cascade happens to be unique and initiated by a single deregulation event. Using results from a companion paper, we show that cascades conform classes which may be labeled by the deregulations in a predefined panel of 15 genes. The results of the paper may be checked in experiments with cellular lines or in animal models, and could have an impact on personalized cancer therapy.\nAutism genes converge on microtubule biology and RNA-binding proteins during excitatory neurogenesis\nAuthors: Sun, N.; Teyssier, N.; Wang, B.; Drake, S.; Seyler, M.; Zaltsman, Y.; Everitt, A.; Teerikorpi, N.; Willsey, H. R.; Goodarzi, H.; Tian, R.; Kampmann, M.; Willsey, J.\nScore: 1.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573108\nRecent studies have identified over one hundred high-confidence (hc) autism spectrum disorder (ASD) genes. Systems biological and functional analyses on smaller subsets of these genes have consistently implicated excitatory neurogenesis. However, the extent to which the broader set of hcASD genes are involved in this process has not been explored systematically nor have the biological pathways underlying this convergence been identified. Here, we leveraged CROP-Seq to repress 87 hcASD genes in a human in vitro model of cortical neurogenesis. We identified 17 hcASD genes whose repression significantly alters developmental trajectory and results in a common cellular state characterized by disruptions in proliferation, differentiation, cell cycle, microtubule biology, and RNA-binding proteins (RBPs). We also characterized over 3,000 differentially expressed genes, 286 of which had expression profiles correlated with changes in developmental trajectory. Overall, we uncovered transcriptional disruptions downstream of hcASD gene perturbations, correlated these disruptions with distinct differentiation phenotypes, and reinforced neurogenesis, microtubule biology, and RBPs as convergent points of disruption in ASD.\nGenome-scale metabolic model of Staphylococcus epidermidis ATCC 12228 matches in vitro conditions\nAuthors: Leonidou, N.; Renz, A.; Winnerling, B.; Grekova, A.; Grein, F.; Draeger, A.\nScore: 1.8, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572329\nStaphylococcus epidermidis, a commensal bacterium inhabiting collagen-rich areas, like human skin, has gained significance due to its probiotic potential in the nasal microbiome and as a leading cause of nosocomial infections. While infrequently leading to severe illnesses, S. epidermidis exerts a significant influence, particularly in its close association with implant-related infections and its role as a classic opportunistic biofilm former. Understanding its opportunistic nature is crucial for developing novel therapeutic strategies, addressing both its beneficial and pathogenic aspects, and alleviating the burdens it imposes on patients and healthcare systems. Here, we employ genome-scale metabolic modeling as a powerful tool to elucidate the lifestyle and capabilities of S. epidermidis. We created a comprehensive computational resource for understanding the organisms growth conditions within diverse habitats by reconstructing and analyzing a manually curated and experimentally validated metabolic model. The final network, iSep23, incorporates 1,415 reactions, 1,051 metabolites, and 705 genes, adhering to established community standards and modeling guidelines. Benchmarking with the MEMOTE test suite yields a high score, highlighting the models high semantic quality. Following the FAIR data principles, iSep23 becomes a valuable and publicly accessible asset for subsequent studies. Growth simulations and carbon source utilization predictions align with experimental results, showcasing the models predictive power. This metabolic model advances our understanding of S. epidermidis as a commensal and potential probiotic and enhances insights into its opportunistic pathogenicity against other microorganisms. Author summaryStaphylococcus epidermidis, a bacterium commonly found on human skin, has shown probiotic effects in the nasal microbiome and is a notable causative agent of hospital-acquired infections. While typically causing non-life-threatening diseases, the economic ramifications of S. epidermidis infections are substantial, with annual costs reaching billions of dollars in the United States. To unravel its opportunistic nature, we utilized genome-scale metabolic modeling, creating a detailed mathematical network that elucidates S. epidermidiss lifestyle and capabilities. This model, encompassing over a thousand reactions, metabolites, and genes, adheres rigorously to established standards and guidelines, evident in its commendable benchmarking scores. Adhering to the FAIR data principles (Findable, Accessible, Interoperable, and Reusable), the model stands as a valuable resource for subsequent investigations. Growth simulations and predictions align closely with experimental results, showcasing the models predictive accuracy. This metabolic model not only enhances our understanding of S. epidermidis as a skin commensal and potential probiotic but also sheds light on its opportunistic pathogenicity, particularly in competition with other microorganisms.\n",
  "wordCount" : "2420",
  "inLanguage": "en",
  "datePublished": "2023-12-31T10:38:19Z",
  "dateModified": "2023-12-31T10:38:19Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 31, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572554">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572554" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572554">
        <p class="paperTitle">Open-ST: High-resolution spatial transcriptomics in 3D</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572554" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572554" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.</p>
        <p class="info">Score: 72.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572554' target='https://doi.org/10.1101/2023.12.22.572554'> 10.1101/2023.12.22.572554</a></p>
        <p class="abstract">Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods. Here, we report Open-ST, a sequencing-based, open-source experimental and computational resource to address these challenges and to study the molecular organization of tissues in 3D. In mouse brain, Open-ST captured transcripts at subcellular resolution and reconstructed cell types. In primary tumor and patient-matched healthy/metastatic lymph nodes, Open-ST captured the diversity of immune, stromal and tumor populations in space. Distinct cell states were organized around cell-cell communication hotspots in the tumor, but not the metastasis. Strikingly, the 3D reconstruction and multimodal analysis of the metastatic lymph node revealed spatially contiguous structures not visible in 2D and potential biomarkers precisely at the 3D tumor/lymph node boundary. We anticipate Open-ST to accelerate the identification of spatial molecular mechanisms in 2D and 3D.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573143" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573143">
        <p class="paperTitle">Highly multiplexed, image-based pooled screens in primary cells and tissues with PerturbView</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kudo, T.; Meireles, A. M.; Moncada, R.; Chen, Y.; Wu, P.; Gould, J.; Hu, X.; Kornfeld, O.; Jesudason, R.; Foo, C.; Hockendorf, B.; Bravo, H. C.; Town, J. P.; Wei, R.; Rios, A.; Chandrasekar, V.; Heinlein, M.; Cai, S.; Lu, C. S.; Celen, C.; Kljavin, N.; Jiang, J.; Hleap, J. S.; Kayagaki, N.; Melo, F. d. S. e.; McGinnis, L.; Li, B.; Singh, A.; Garraway, L.; Rozenblatt-Rosen, O.; Regev, A.; Lubeck, E.</p>
        <p class="info">Score: 41.7, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573143' target='https://doi.org/10.1101/2023.12.26.573143'> 10.1101/2023.12.26.573143</a></p>
        <p class="abstract">Optical pooled screening (OPS) is a highly scalable method for linking image-based phenotypes with cellular perturbations. However, it has thus far been restricted to relatively low-plex phenotypic readouts in cancer cell lines in culture, due to limitations associated with in situ sequencing (ISS) of perturbation barcodes. Here, we developed PerturbView, an OPS technology that leverages in vitro transcription (IVT) to amplify barcodes prior to ISS, enabling screens with highly multiplexed phenotypic readouts across diverse systems, including primary cells and tissues. We demonstrate PerturbView in iPSC-derived neurons, primary immune cells, and tumor tissue sections from animal models. In a screen of immune signaling pathways in primary bone marrow-derived macrophages, PerturbView uncovered both known and novel regulators of NF{kappa}B signaling. Furthermore, we combined PerturbView with spatial transcriptomics in tissue sections from a mouse xenograft model, paving the way to in vivo screens with rich optical and transcriptomic phenotypes. PerturbView broadens the scope of OPS to a wide range of models and applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572913">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572913" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572913">
        <p class="paperTitle">Dynamics of single-cell protein covariation during epithelial-mesenchymal transition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572913" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572913" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khan, S.; Conover, R.; Asthagiri, A. R.; Slavov, N.</p>
        <p class="info">Score: 29.7, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572913' target='https://doi.org/10.1101/2023.12.21.572913'> 10.1101/2023.12.21.572913</a></p>
        <p class="abstract">Physiological processes, such as epithelial-mesenchymal transition (EMT), are mediated by changes in protein interactions. These changes may be better reflected in protein covariation within cellular cluster than in the temporal dynamics of cluster-average protein abundance. To explore this possibility, we quantified proteins in single human cells undergoing EMT. Covariation analysis of the data revealed that functionally coherent protein clusters dynamically changed their protein-protein correlations without concomitant changes in cluster-average protein abundance. These dynamics of protein-protein correlations were monotonic in time and delineated protein modules functioning in actin cytoskeleton organization, energy metabolism and protein transport. These protein modules are defined by protein covariation within the same time point and cluster and thus reflect biological regulation masked by the cluster-average protein dynamics. Thus, protein correlation dynamics across single cells offer a window into protein regulation during physiological transitions.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=132 SRC=&#34;FIGDIR/small/572913v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (79K):
org.highwire.dtl.DTLVardef@151d03aorg.highwire.dtl.DTLVardef@3da81borg.highwire.dtl.DTLVardef@1e79733org.highwire.dtl.DTLVardef@177bc33_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573207">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573207" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573207">
        <p class="paperTitle">Oncogenic PIK3CA corrupts growth factor signaling specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573207" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573207" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Madsen, R. R.; Le Marois, A.; Mruk, O.; Voliotis, M.; Yin, S.; Sufi, J.; Qin, X.; Zhao, S. J.; Gorczynska, J.; Morelli, D.; Davidson, L.; Sahai, E.; Korolchuk, V. I.; Tape, C. J.; Vanhaesebroeck, B.</p>
        <p class="info">Score: 27.3, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573207' target='https://doi.org/10.1101/2023.12.23.573207'> 10.1101/2023.12.23.573207</a></p>
        <p class="abstract">Pathological activation of the PI3K/AKT pathway is among the most frequent defects in human cancer and is also the cause of rare overgrowth disorders. Yet, there is currently no systematic understanding of the quantitative flow of information within PI3K/AKT signaling and how it is perturbed by disease-causing mutations. Here, we develop scalable, single-cell approaches for systematic analyses of signal processing within the PI3K pathway, enabling precise calculations of its information transfer for different growth factors. Using genetically-engineered human cell models with allele dose-dependent expression of PIK3CAH1047R, we show that this oncogene is not a simple, constitutive pathway activator but a context-dependent modulator of extracellular signal transfer. PIK3CAH1047Rreduces information transmission downstream of IGF1 while selectively enhancing EGF-induced signaling and transcriptional responses. This leads to a gross reduction in signaling specificity, akin to &#34;blurred&#34; signal perception. The associated increase in signaling heterogeneity promotes phenotypic diversity in a human cervical cancer cell line model and in human induced pluripotent stem cells. Collectively, these findings and the accompanying methodological advances lay the foundations for a systematic mapping of the quantitative mechanisms of PI3K/AKT-dependent signal processing and phenotypic control in health and disease.

One-sentence summarySingle-cell signaling and information theoretic analyses reveal that oncogenic PI3K/AKT activation leads to a gross reduction in signaling specificity, context-dependent EGF response amplification as well as increased phenotypic heterogeneity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573217" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573217">
        <p class="paperTitle">Coupling Cell Size Regulation and Proliferation Dynamics of C.glutamicum Reveals Cell Division Based on Surface Area</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nieto, C.; Tauber, S.; Blobaum, L.; Vahdat, Z.; Grunberger, A.; Singh, A.</p>
        <p class="info">Score: 4.3, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573217' target='https://doi.org/10.1101/2023.12.26.573217'> 10.1101/2023.12.26.573217</a></p>
        <p class="abstract">Single cells actively coordinate growth and division to regulate their size, yet how this size homeostasis at the single-cell level propagates over multiple generations to impact clonal expansion remains fundamentally unexplored. Classical timer models for cell proliferation (where the duration of the cell cycle is an independent variable) predict that the stochastic variation in colony size will increase monotonically over time. In stark contrast, implementing size control according to adder strategy (where on average a fixed size added from cell birth to division) leads to colony size variations that eventually decay to zero. While these results assume a fixed size of the colony-initiating progenitor cell, further analysis reveals that the magnitude of the intercolony variation in population number is sensitive to heterogeneity in the initial cell size. We validate these predictions by tracking the growth of isogenic microcolonies of Corynebacterium glutamicum in microfluidic chambers. Approximating their cell shape to a capsule, we observe that the degree of random variability in cell size is different depending on whether the cell size is quantified as per length, surface area, or volume, but size control remains an adder regardless of these size metrics. A comparison of the observed variability in the colony population with the predictions suggests that proliferation matches better with a cell division based on the cell surface. In summary, our integrated mathematical-experimental approach bridges the paradigms of single-cell size regulation and clonal expansion at the population levels. This innovative approach provides elucidation of the mechanisms of size homeostasis from the stochastic dynamics of colony size for rod-shaped microbes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571758">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571758" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571758">
        <p class="paperTitle">BMI and BPH correlate with urinary microbiome diversity and lower urinary tract symptoms in men</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571758" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571758" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bowie, K.; Garzotto, M.; Orwoll, E.; Karstens, L.</p>
        <p class="info">Score: 4.0, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571758' target='https://doi.org/10.1101/2023.12.14.571758'> 10.1101/2023.12.14.571758</a></p>
        <p class="abstract">Several studies have identified bacteria and other microbes in the bladder and lower urinary tract in the absence of infection. In women, the urinary microbiome has been associated with lower urinary tract symptoms (LUTS), however, similar studies have not been undertaken in large cohorts of men. Here we examine the urinary microbiome and its association with LUTS in a subset of 500 men aged 65 to 90 years from the Osteoporotic Fractures in Men (MrOS) study. We identified significant associations between benign prostatic hyperplasia (BPH), age, and body mass index (BMI) with several diversity metrics. Our analysis revealed complex relationships between BMI, BPH, LUTS, and alpha diversity which give insight into the intricate dynamics of the urinary microbiome. By beginning to uncover the interrelationships of BPH, BMI, LUTS, and the urinary microbiome, these results can inform future study design to better understand the heterogeneity of the male urinary microbiome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573173">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573173" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573173">
        <p class="paperTitle">Optimization-Based Synthesis with Directed Cell Migration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573173" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573173" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Havenhill, E.; Ghosh, S.</p>
        <p class="info">Score: 1.9, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573173' target='https://doi.org/10.1101/2023.12.22.573173'> 10.1101/2023.12.22.573173</a></p>
        <p class="abstract">Collective behavior of biological agents such as herds of organisms and cells is a fundamental feature in the systems biology and in the emergence of new phenomena in the biological environ-ment. Collective cell migration under a physical or chemical cue is an example of this fundamental phenomenon where individual cell migration is driven by the collective behavior of the neighboring cells and vice versa. The goal of this research is to discover the mathematical rules of collective cell migration using experimental data and testing the predictive nature of the models in independent experimental data. Such insight is made possible in this work with the hybrid use of dynamic mode decomposition (DMD) [1] and optimal control theory. Both single and multi-cellular systems are simulated, including obstacle courses, using this framework. The results of this work show how cells collectively behave during their migration and also, opens the possibility of designing robotic cells for possible therapeutic purpose where the cell trajectory can be controlled.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573550">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573550" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573550">
        <p class="paperTitle">Deregulation cascades in carcinogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573550" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573550" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perez, R. H.; Gomez, J. P.; Perez, G. J. G.; Gonzalez, A.</p>
        <p class="info">Score: 1.8, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573550' target='https://doi.org/10.1101/2023.12.28.573550'> 10.1101/2023.12.28.573550</a></p>
        <p class="abstract">Carcinogenesis is one example of transition between two biological states involving strong gene expression rearrangements. Normal and tumor states are understood as two different states of a Gene Regulatory Network. In terms of the normal state characteristics, the transition may be thought as a deregulation process. According to a very abstract model for the transition, the available data on cancer risk support the idea of a single big jump in expression space, which may be associated to a deregulation cascade. Here, we use the measured frequency distribution of gene deregulations, and concepts from the probabilistic theory of causation in order to infer causal relations between pairs of genes and construct a deregulation causal network. Then, the deregulated genes in each sample are organized according to the network in such a way that they show the deregulation cascade behind the transition from the normal to the tumor state. The explicit analysis for prostate adenocarcinoma is given. In most cases, the cascade happens to be unique and initiated by a single deregulation event. Using results from a companion paper, we show that cascades conform classes which may be labeled by the deregulations in a predefined panel of 15 genes. The results of the paper may be checked in experiments with cellular lines or in animal models, and could have an impact on personalized cancer therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573108">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573108" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573108">
        <p class="paperTitle">Autism genes converge on microtubule biology and RNA-binding proteins during excitatory neurogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573108" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573108" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, N.; Teyssier, N.; Wang, B.; Drake, S.; Seyler, M.; Zaltsman, Y.; Everitt, A.; Teerikorpi, N.; Willsey, H. R.; Goodarzi, H.; Tian, R.; Kampmann, M.; Willsey, J.</p>
        <p class="info">Score: 1.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573108' target='https://doi.org/10.1101/2023.12.22.573108'> 10.1101/2023.12.22.573108</a></p>
        <p class="abstract">Recent studies have identified over one hundred high-confidence (hc) autism spectrum disorder (ASD) genes. Systems biological and functional analyses on smaller subsets of these genes have consistently implicated excitatory neurogenesis. However, the extent to which the broader set of hcASD genes are involved in this process has not been explored systematically nor have the biological pathways underlying this convergence been identified. Here, we leveraged CROP-Seq to repress 87 hcASD genes in a human in vitro model of cortical neurogenesis. We identified 17 hcASD genes whose repression significantly alters developmental trajectory and results in a common cellular state characterized by disruptions in proliferation, differentiation, cell cycle, microtubule biology, and RNA-binding proteins (RBPs). We also characterized over 3,000 differentially expressed genes, 286 of which had expression profiles correlated with changes in developmental trajectory. Overall, we uncovered transcriptional disruptions downstream of hcASD gene perturbations, correlated these disruptions with distinct differentiation phenotypes, and reinforced neurogenesis, microtubule biology, and RBPs as convergent points of disruption in ASD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572329">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572329" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572329">
        <p class="paperTitle">Genome-scale metabolic model of Staphylococcus epidermidis ATCC 12228 matches in vitro conditions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572329" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572329" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leonidou, N.; Renz, A.; Winnerling, B.; Grekova, A.; Grein, F.; Draeger, A.</p>
        <p class="info">Score: 1.8, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572329' target='https://doi.org/10.1101/2023.12.19.572329'> 10.1101/2023.12.19.572329</a></p>
        <p class="abstract">Staphylococcus epidermidis, a commensal bacterium inhabiting collagen-rich areas, like human skin, has gained significance due to its probiotic potential in the nasal microbiome and as a leading cause of nosocomial infections. While infrequently leading to severe illnesses, S. epidermidis exerts a significant influence, particularly in its close association with implant-related infections and its role as a classic opportunistic biofilm former. Understanding its opportunistic nature is crucial for developing novel therapeutic strategies, addressing both its beneficial and pathogenic aspects, and alleviating the burdens it imposes on patients and healthcare systems. Here, we employ genome-scale metabolic modeling as a powerful tool to elucidate the lifestyle and capabilities of S. epidermidis. We created a comprehensive computational resource for understanding the organisms growth conditions within diverse habitats by reconstructing and analyzing a manually curated and experimentally validated metabolic model. The final network, iSep23, incorporates 1,415 reactions, 1,051 metabolites, and 705 genes, adhering to established community standards and modeling guidelines. Benchmarking with the MEMOTE test suite yields a high score, highlighting the models high semantic quality. Following the FAIR data principles, iSep23 becomes a valuable and publicly accessible asset for subsequent studies. Growth simulations and carbon source utilization predictions align with experimental results, showcasing the models predictive power. This metabolic model advances our understanding of S. epidermidis as a commensal and potential probiotic and enhances insights into its opportunistic pathogenicity against other microorganisms.

Author summaryStaphylococcus epidermidis, a bacterium commonly found on human skin, has shown probiotic effects in the nasal microbiome and is a notable causative agent of hospital-acquired infections. While typically causing non-life-threatening diseases, the economic ramifications of S. epidermidis infections are substantial, with annual costs reaching billions of dollars in the United States. To unravel its opportunistic nature, we utilized genome-scale metabolic modeling, creating a detailed mathematical network that elucidates S. epidermidiss lifestyle and capabilities. This model, encompassing over a thousand reactions, metabolites, and genes, adheres rigorously to established standards and guidelines, evident in its commendable benchmarking scores. Adhering to the FAIR data principles (Findable, Accessible, Interoperable, and Reusable), the model stands as a valuable resource for subsequent investigations. Growth simulations and predictions align closely with experimental results, showcasing the models predictive accuracy. This metabolic model not only enhances our understanding of S. epidermidis as a skin commensal and potential probiotic but also sheds light on its opportunistic pathogenicity, particularly in competition with other microorganisms.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
