<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DrugMap: A quantitative pan-cancer analysis of cysteine ligandability
Authors: Takahashi, M.; Chong, H. B.; Zhang, S.; Lazarov, M.; Harry, S.; Maynard, M.; White, R.; Hilbert, B.; Gohar, M.; Ge, M.; Zhang, J.; Durr, B. R.; Kryukov, G.; Tsou, C.-C.; Brooijmans, N.; Alghali, A.; Rubio, K.; Vilanueva, A.; Harrison, D.; Koglin, A.-S.; Ojeda, S.; Karakyriakou, B.; Healy, A.; Assaad, J.; Makram, F.; Rachimin, I.; Khandelwal, N.; Tien, P.-C.; Popoola, G.; Chen, N.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="DrugMap: A quantitative pan-cancer analysis of cysteine ligandability
Authors: Takahashi, M.; Chong, H. B.; Zhang, S.; Lazarov, M.; Harry, S.; Maynard, M.; White, R.; Hilbert, B.; Gohar, M.; Ge, M.; Zhang, J.; Durr, B. R.; Kryukov, G.; Tsou, C.-C.; Brooijmans, N.; Alghali, A.; Rubio, K.; Vilanueva, A.; Harrison, D.; Koglin, A.-S.; Ojeda, S.; Karakyriakou, B.; Healy, A.; Assaad, J.; Makram, F.; Rachimin, I.; Khandelwal, N.; Tien, P.-C.; Popoola, G.; Chen, N." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-29T10:37:40+00:00" />
<meta property="article:modified_time" content="2023-10-29T10:37:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="DrugMap: A quantitative pan-cancer analysis of cysteine ligandability
Authors: Takahashi, M.; Chong, H. B.; Zhang, S.; Lazarov, M.; Harry, S.; Maynard, M.; White, R.; Hilbert, B.; Gohar, M.; Ge, M.; Zhang, J.; Durr, B. R.; Kryukov, G.; Tsou, C.-C.; Brooijmans, N.; Alghali, A.; Rubio, K.; Vilanueva, A.; Harrison, D.; Koglin, A.-S.; Ojeda, S.; Karakyriakou, B.; Healy, A.; Assaad, J.; Makram, F.; Rachimin, I.; Khandelwal, N.; Tien, P.-C.; Popoola, G.; Chen, N."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "DrugMap: A quantitative pan-cancer analysis of cysteine ligandability\nAuthors: Takahashi, M.; Chong, H. B.; Zhang, S.; Lazarov, M.; Harry, S.; Maynard, M.; White, R.; Hilbert, B.; Gohar, M.; Ge, M.; Zhang, J.; Durr, B. R.; Kryukov, G.; Tsou, C.-C.; Brooijmans, N.; Alghali, A.; Rubio, K.; Vilanueva, A.; Harrison, D.; Koglin, A.-S.; Ojeda, S.; Karakyriakou, B.; Healy, A.; Assaad, J.; Makram, F.; Rachimin, I.; Khandelwal, N.; Tien, P.-C.; Popoola, G.; Chen, N.",
  "keywords": [
    
  ],
  "articleBody": " DrugMap: A quantitative pan-cancer analysis of cysteine ligandability\nAuthors: Takahashi, M.; Chong, H. B.; Zhang, S.; Lazarov, M.; Harry, S.; Maynard, M.; White, R.; Hilbert, B.; Gohar, M.; Ge, M.; Zhang, J.; Durr, B. R.; Kryukov, G.; Tsou, C.-C.; Brooijmans, N.; Alghali, A.; Rubio, K.; Vilanueva, A.; Harrison, D.; Koglin, A.-S.; Ojeda, S.; Karakyriakou, B.; Healy, A.; Assaad, J.; Makram, F.; Rachimin, I.; Khandelwal, N.; Tien, P.-C.; Popoola, G.; Chen, N.; Vordermark, K.; Richter, M.; Patel, H.; Yang, T.-y.; Griesshaber, H.; Hosp, T.; van den Ouweland, S.; Hara, T.; Bussema, L.; Shi, L.; Rasmussen, M.; Domingues, A. C.; Lawless, A.; Fang, J.; Yoda, S.; Nguyen, L. P.; R\nScore: 26.9, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563287\nCysteine-focused chemical proteomic platforms have accelerated the clinical development of covalent inhibitors of a wide-range of targets in cancer. However, how different oncogenic contexts influence cysteine targeting remains unknown. To address this question, we have developed DrugMap, an atlas of cysteine ligandability compiled across 416 cancer cell lines. We unexpectedly find that cysteine ligandability varies across cancer cell lines, and we attribute this to differences in cellular redox states, protein conformational changes, and genetic mutations. Leveraging these findings, we identify actionable cysteines in NF{kappa}B1 and SOX10 and develop corresponding covalent ligands that block the activity of these transcription factors. We demonstrate that the NF{kappa}B1 probe blocks DNA binding, whereas the SOX10 ligand increases SOX10-SOX10 interactions and disrupts melanoma transcriptional signaling. Our findings reveal heterogeneity in cysteine ligandability across cancers, pinpoint cell-intrinsic features driving cysteine targeting, and illustrate the use of covalent probes to disrupt oncogenic transcription factor activity.\nThe genetic architecture of protein interaction affinity and specificity\nAuthors: Bendel, A.; Faure, A.; Klein, D.; Shimada, K.; Kempf, G.; Cavadini, S.; Lehner, B.; Diss, G.\nScore: 24.5, Published: 2023-10-20 DOI: 10.1101/2023.10.17.562688\nProteins function in crowded cellular environments in which they must bind to specific target proteins but also avoid binding to many other off-target proteins. In large protein families this task is particularly challenging because many off-target proteins have very similar structures. How this specificity of physical protein-protein interactions in cellular networks is encoded and evolves is not very well understood. Here we address the question of specificity-encoding by comprehensively quantifying the effects of all mutations in one protein, JUN, on its binding to all other members of a protein family, the 54 human basic leucine zipper transcription factors. Fitting a global thermodynamic model to the data reveals that most affinity changing mutations equally affect JUNs propensity to bind to all its interaction partners. Mutations that alter the specificity of binding are much rarer. These specificity-altering mutations are, however, distributed throughout the JUN interaction interface. JUNs interaction specificity is encoded by both positive determinants that promote on-target interactions and negative determinants that prevent off-target interactions. Indeed, about half of the specificity-defining residues in JUN have dual functions and both promote on-target binding and prevent off-target binding. Whereas nearly all mutations that alter specificity are pleiotropic and also alter the affinity of binding to all interaction partners, the converse is not true with mutations outside of the interface able to tune affinity without affecting specificity. Our results provide the first global view of how mutations in a protein affect binding to all its potential interaction partners and reveal the distributed encoding of specificity and affinity in an interaction interface. They also show how the modular architecture of coiled-coils provides an elegant solution to the challenge of optimising specificity and affinity in a large protein family.\nPhysiMeSS - A New PhysiCell Addon for Extracellular Matrix Modelling\nAuthors: Noel, V.; Ruscone, M.; Shuttleworth, R.; MacNamara, C. K.\nScore: 8.6, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564365\nThe extracellular matrix is a complex assembly of macro-molecules, such as collagen fibres, which provides structural support for surrounding cells. In the context of cancer metastasis, it represents a barrier for the cells, that the migrating cells needs to degrade in order to leave the primary tumor and invade further tissues. Agent-based frameworks, such as PhysiCell, are often use to represent the spatial dynamics of tumor evolution. However, typically they only implement cells as agents, which are represented by either a circle (2D) or a sphere (3D). In order to accurately represent the extracellular matrix as a network of fibres, we require a new type of agent represented by a segment (2D) or a cylinder (3D). In this article, we present PhysiMeSS, an addon of PhysiCell, which introduces a new type of agent to describe fibres, and their physical interactions with cells and other fibres. PhysiMeSS implementation is publicly available at https://github.com/PhysiMeSS/PhysiMeSS, as well as in the official PhysiCell repository. We also provide simple examples to describe the extended possibilities of this new framework. We hope that this tool will serve to tackle important biological questions such as diseases linked to dis-regulation of the extracellular matrix, or the processes leading to cancer metastasis.\nRibosome demand links transcriptional bursts to protein expression noise\nAuthors: Pal, S.; Ray, U.; Dhar, R.\nScore: 7.0, Published: 2023-10-23 DOI: 10.1101/2023.10.19.563206\nStochastic variation in protein expression generates phenotypic heterogeneity in a cell population, and has important role in antibiotic persistence, mutation penetrance, cancer growth and therapy resistance. Studies investigating molecular origins of noise have predominantly focused on the transcription process. However, the noise generated during the transcription process is further modulated by the translation process, which eventually determines the expression noise at the protein level. Studies across different organisms have revealed a positive correlation between translational efficiency and protein noise. However, the molecular basis of this correlation has remained unknown. In this work, through stochastic modeling of translation in single mRNA molecules and empirical measurement of protein noise, we demonstrate that ribosome demand associated with high translational efficiency drives the correlation between translational efficiency and protein noise. We also show that this correlation is present only in genes with bursty transcription, where ribosome demand varies with stochastic fluctuations in mRNA numbers. Thus, our work reveals how transcriptional bursts are translated into protein noise, which has important implications for investigating protein noise and phenotypic heterogeneity across biological systems.\nA robust ultrasensitive transcriptional switch in noisy cellular environments\nAuthors: Jeong, E. M.; Kim, J. K.\nScore: 11.3, Published: 2023-10-18 DOI: 10.1101/2023.08.15.553401\nUltrasensitive transcriptional switches enable sharp transitions between on and off states of transcription and are essential for cells to respond to environmental cues precisely. However, conventional switches, relying on direct repressor-DNA binding, are extremely sensitive to noise. Here, we discovered an alternative design combining three indirect transcriptional repression mechanisms, sequestration, blocking, and displacement, to generate a noise-resilient ultrasensitive switch. Although sequestration alone can generate an ultrasensitive switch, it remains sensitive to noise because the unintended transcriptional state induced by noise can persist for long periods. However, by jointly utilizing blocking and displacement, these noise-induced transitions can be rapidly restored to the original transcriptional state. Because this transcriptional switch is effective in noisy cellular contexts, it goes beyond previous synthetic transcriptional switches utilizing direct repression mechanisms, making it particularly valuable for robust synthetic system design. Our findings also provide insights into the evolution of robust ultrasensitive switches in real cells. Specifically, the concurrent use of seemingly redundant indirect repression mechanisms in diverse biological systems appears to be a strategy to achieve noise-resilience of ultrasensitive switches. GRAPHICAL ABSTRACT O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=175 SRC=\"FIGDIR/small/553401v2_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (28K): org.highwire.dtl.DTLVardef@1608e6eorg.highwire.dtl.DTLVardef@270493org.highwire.dtl.DTLVardef@19d5106org.highwire.dtl.DTLVardef@132701a_HPS_FORMAT_FIGEXP M_FIG C_FIG\nNoise robustness and metabolic load determine the principles of central dogma regulation\nAuthors: Lo, T. W.; Choi, H. K. J.; Huang, D.; Wiggins, P. A.\nScore: 3.6, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563172\nProtein expression levels optimize cell fitness: Too low of an expression level of essential proteins slows growth by compromising the function of essential processes, whereas the overexpression of proteins slows growth by increasing the metabolic load. This trade-off naively predicts that the cell maximizes its fitness by a Goldilocks principle in which cells express just enough protein for function; however, this strategy neglects the significance of the inherent stochasticity of the gene expression process which leads to significant cell-to-cell variation in protein numbers. How does the cell ensure robust growth in the face of the inherent stochasticity of these central dogma processes? To explore the consequences of noise in the expression of hundreds of essential genes, we build a minimal model where the trade-off between metabolic cost and growth robustness can be analyzed analytically. The model predicts that growth-rate maximization leads to a highly-asymmetric cost-benefit analysis which drives the optimal protein expression levels far above what is required on average, with low-expression essential proteins expressed at more than 10-fold what is required for growth in the typical cell. This robustness mechanism naturally explains the surprisingly strong buffering effect observed in essential protein depletion experiments and leads to a second qualitative prediction: there is a lower floor on the transcription level of essential genes corresponding to one message per cell cycle. We show that nearly all essential, but not non-essential, genes obey this limit in three evolutionarily-divergent model organisms: Escherichia coli, yeast, and human. The model also predicts that the optimal translation efficiency is roughly proportional to message abundance, predicting the observed relation between proteome fraction and message abundance in eukaryotic cells. This optimal translation efficiency predicts a non-canonical scaling of gene expression noise with protein abundance, which we show is observed in yeast. Together, these results reveal that noise robustness and metabolic load determine the global regulatory principles that govern central dogma function.\nLogic-based mechanistic machine learning on high-content images reveals how drugs differentially regulate cardiac fibroblasts\nAuthors: Nelson, A. R.; Christiansen, S. L.; Naegle, K. M.; Saucerman, J. J.\nScore: 11.7, Published: 2023-10-23 DOI: 10.1101/2023.03.01.530599\nFibroblasts are essential regulators of extracellular matrix deposition following cardiac injury. These cells exhibit highly plastic responses in phenotype during fibrosis in response to environmental stimuli. Here, we test whether and how candidate anti-fibrotic drugs differentially regulate measures of cardiac fibroblast phenotype, which may help identify treatments for cardiac fibrosis. We conducted a high content microscopy screen of human cardiac fibroblasts treated with 13 clinically relevant drugs in the context of TGF{beta} and/or IL-1{beta}, measuring phenotype across 137 single-cell features. We used the phenotypic data from our high content imaging to train a logic-based mechanistic machine learning model (LogiMML) for fibroblast signaling. The model predicted how pirfenidone and Src inhibitor WH-4-023 reduce actin filament assembly and actin-myosin stress fiber formation, respectively. Validating the LogiMML model prediction that PI3K partially mediates the effects of Src inhibition, we found that PI3K inhibition reduces actin-myosin stress fiber formation and procollagen I production in human cardiac fibroblasts. In this study, we establish a modeling approach combining the strengths of logic-based network models and regularized regression models, apply this approach to predict mechanisms that mediate the differential effects of drugs on fibroblasts, revealing Src inhibition acting via PI3K as a potential therapy for cardiac fibrosis. SignificanceCardiac fibrosis is a dysregulation of the normal wound healing response, resulting in excessive scarring and cardiac dysfunction. As cardiac fibroblasts primarily regulate this process, we explored how candidate anti-fibrotic drugs alter the fibroblast phenotype. We identify a set of 137 phenotypic features that change in response to drug treatments. Using a new computational modeling approach termed logic-based mechanistic machine learning, we predict how pirfenidone and Src inhibition affect the regulation of the phenotypic features actin filament assembly and actin-myosin stress fiber formation. We also show that inhibition of PI3K reduces actin-myosin stress fiber formation and procollagen I production in human cardiac fibroblasts, supporting a role for PI3K as a mechanism by which Src inhibition may suppress fibrosis.\nA Hill type equation reveals the regulation principle of target protein expression led by p53 pulsing\nAuthors: Shi, X.\nScore: 1.6, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563713\nThe central dogma indicates the basic direction of gene expression pathway. For activated gene expression, the relationship between the various links that are from the binding of transcription factor (TF) DNA to synthesis of proteins is still unclear. There is a consensus that at steady state the protein levels are largely determined by mRNA level. How to find this steady state? What factors determine it? Taking TF p53 as an example, on the basis of the previously discovered Hill type equation that can characterize the average steady state of mRNA expression under p53 pulsing, and demonstrating the regulatory ability of duration and frequency beyond saturation. I proved that the average steady state of target protein expression driven by p53 pulsing can also be governed by this equation. Therefore, at steady state the average mRNA and protein expression under TF pulsing are exactly the same. This consensus has been successfully demonstrated. For gene BAX of p53 target, the observed fold changes of mRNA and protein are 1.72, 1.28, while the predictions for both mRNA and protein from the obtained equation are 1.35. The regulatory principle of gene expression revealed by this equation is that TF dynamics and binding affinity control the expression levels; and the half-life of mRNA and protein regulate the expression stability. By introducing two quantitative indicators, we can understand the degree of accumulation and stability of protein expression. Furthermore, this generalized equation broadens the application of classical biochemical theory.\nGlobal transcription regulation revealed from dynamical correlations in time-resolved single-cell RNA-sequencing\nAuthors: Volteras, D.; Shahrezaei, V.; Thomas, P.\nScore: 1.6, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563709\nSingle-cell transcriptomics reveals significant variations in the transcriptional activity across cells. Yet, it remains challenging to identify mechanisms of transcription dynamics from static snapshots. It is thus still unknown what drives global transcription dynamics in single cells. We present a stochastic model of gene expression in growing and dividing cells that harnesses temporal dimensions of single-cell RNA-sequencing through metabolic labelling protocols and cell cycle reporters. We develop a parallel and highly scalable Approximate Bayesian Computation method that corrects for technical variation and accurately quantifies absolute burst frequency, burst size and degradation rate at a transcriptome-wide scale. Using Bayesian model selection, we reveal scaling between transcription rates and cell size and unveil waves of gene regulation across the cell cycle-dependent transcriptome. Our study shows that stochastic modelling of dynamical correlations identifies global mechanisms of transcription regulation. We illustrate how mechanistic modelling and amortised inference can efficiently integrate distributions across time, modalities and conditions. Our framework thus provides a versatile tool to uncover how global transcriptional dynamics is orchestrated genome-wide in single cells.\nDevelopment of a genome scale metabolic model for the lager hybrid yeast S. pastorianus to understand evolution of metabolic pathways in industrial settings.\nAuthors: Timouma, S.; Balarezo-Cisneros, L. N.; Schwartz, J.-M.; Delneri, D.\nScore: 1.5, Published: 2023-10-28 DOI: 10.1101/2023.10.25.564032\nIn silico tools such as genome-scale metabolic models (GSMM) have shown to be powerful for metabolic engineering of microorganisms. Here, we created the iSP_1513 GSMM for the aneuploid hybrid S. pastorianus CBS1513 to allow top-down computational approaches to predict the evolution of metabolic pathways and to aid strain optimisation and media engineering in production processes. The iSP_1513 comprises 4062 reactions, 1808 alleles and 2747 metabolites, and takes into account the functional redundancy in the gene-protein-reaction rule caused by the presence of orthologous genes. Moreover, a universal algorithm to constrain GSMM reactions using transcriptome data was developed as a python library and enabled the integration of temperature as parameter. Essentiality datasets, growth data on various carbohydrates and volatile metabolites secretion were used to validate the model. Overall, the iSP_1513 GSMM represent an important step towards understanding the metabolic capabilities, evolutionary trajectories and adaptation potential of S. pastorianus in different industrial settings.\n",
  "wordCount" : "2561",
  "inLanguage": "en",
  "datePublished": "2023-10-29T10:37:40Z",
  "dateModified": "2023-10-29T10:37:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on October 29, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563287">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563287" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563287">
        <p class="paperTitle">DrugMap: A quantitative pan-cancer analysis of cysteine ligandability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563287" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563287" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Takahashi, M.; Chong, H. B.; Zhang, S.; Lazarov, M.; Harry, S.; Maynard, M.; White, R.; Hilbert, B.; Gohar, M.; Ge, M.; Zhang, J.; Durr, B. R.; Kryukov, G.; Tsou, C.-C.; Brooijmans, N.; Alghali, A.; Rubio, K.; Vilanueva, A.; Harrison, D.; Koglin, A.-S.; Ojeda, S.; Karakyriakou, B.; Healy, A.; Assaad, J.; Makram, F.; Rachimin, I.; Khandelwal, N.; Tien, P.-C.; Popoola, G.; Chen, N.; Vordermark, K.; Richter, M.; Patel, H.; Yang, T.-y.; Griesshaber, H.; Hosp, T.; van den Ouweland, S.; Hara, T.; Bussema, L.; Shi, L.; Rasmussen, M.; Domingues, A. C.; Lawless, A.; Fang, J.; Yoda, S.; Nguyen, L. P.; R</p>
        <p class="info">Score: 26.9, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563287' target='https://doi.org/10.1101/2023.10.20.563287'> 10.1101/2023.10.20.563287</a></p>
        <p class="abstract">Cysteine-focused chemical proteomic platforms have accelerated the clinical development of covalent inhibitors of a wide-range of targets in cancer. However, how different oncogenic contexts influence cysteine targeting remains unknown. To address this question, we have developed DrugMap, an atlas of cysteine ligandability compiled across 416 cancer cell lines. We unexpectedly find that cysteine ligandability varies across cancer cell lines, and we attribute this to differences in cellular redox states, protein conformational changes, and genetic mutations. Leveraging these findings, we identify actionable cysteines in NF{kappa}B1 and SOX10 and develop corresponding covalent ligands that block the activity of these transcription factors. We demonstrate that the NF{kappa}B1 probe blocks DNA binding, whereas the SOX10 ligand increases SOX10-SOX10 interactions and disrupts melanoma transcriptional signaling. Our findings reveal heterogeneity in cysteine ligandability across cancers, pinpoint cell-intrinsic features driving cysteine targeting, and illustrate the use of covalent probes to disrupt oncogenic transcription factor activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562688">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562688" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562688">
        <p class="paperTitle">The genetic architecture of protein interaction affinity and specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562688" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562688" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bendel, A.; Faure, A.; Klein, D.; Shimada, K.; Kempf, G.; Cavadini, S.; Lehner, B.; Diss, G.</p>
        <p class="info">Score: 24.5, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562688' target='https://doi.org/10.1101/2023.10.17.562688'> 10.1101/2023.10.17.562688</a></p>
        <p class="abstract">Proteins function in crowded cellular environments in which they must bind to specific target proteins but also avoid binding to many other off-target proteins. In large protein families this task is particularly challenging because many off-target proteins have very similar structures. How this specificity of physical protein-protein interactions in cellular networks is encoded and evolves is not very well understood. Here we address the question of specificity-encoding by comprehensively quantifying the effects of all mutations in one protein, JUN, on its binding to all other members of a protein family, the 54 human basic leucine zipper transcription factors. Fitting a global thermodynamic model to the data reveals that most affinity changing mutations equally affect JUNs propensity to bind to all its interaction partners. Mutations that alter the specificity of binding are much rarer. These specificity-altering mutations are, however, distributed throughout the JUN interaction interface. JUNs interaction specificity is encoded by both positive determinants that promote on-target interactions and negative determinants that prevent off-target interactions. Indeed, about half of the specificity-defining residues in JUN have dual functions and both promote on-target binding and prevent off-target binding. Whereas nearly all mutations that alter specificity are pleiotropic and also alter the affinity of binding to all interaction partners, the converse is not true with mutations outside of the interface able to tune affinity without affecting specificity. Our results provide the first global view of how mutations in a protein affect binding to all its potential interaction partners and reveal the distributed encoding of specificity and affinity in an interaction interface. They also show how the modular architecture of coiled-coils provides an elegant solution to the challenge of optimising specificity and affinity in a large protein family.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564365" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564365">
        <p class="paperTitle">PhysiMeSS - A New PhysiCell Addon for Extracellular Matrix Modelling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Noel, V.; Ruscone, M.; Shuttleworth, R.; MacNamara, C. K.</p>
        <p class="info">Score: 8.6, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564365' target='https://doi.org/10.1101/2023.10.27.564365'> 10.1101/2023.10.27.564365</a></p>
        <p class="abstract">The extracellular matrix is a complex assembly of macro-molecules, such as collagen fibres, which provides structural support for surrounding cells. In the context of cancer metastasis, it represents a barrier for the cells, that the migrating cells needs to degrade in order to leave the primary tumor and invade further tissues. Agent-based frameworks, such as PhysiCell, are often use to represent the spatial dynamics of tumor evolution. However, typically they only implement cells as agents, which are represented by either a circle (2D) or a sphere (3D). In order to accurately represent the extracellular matrix as a network of fibres, we require a new type of agent represented by a segment (2D) or a cylinder (3D). In this article, we present PhysiMeSS, an addon of PhysiCell, which introduces a new type of agent to describe fibres, and their physical interactions with cells and other fibres. PhysiMeSS implementation is publicly available at https://github.com/PhysiMeSS/PhysiMeSS, as well as in the official PhysiCell repository. We also provide simple examples to describe the extended possibilities of this new framework. We hope that this tool will serve to tackle important biological questions such as diseases linked to dis-regulation of the extracellular matrix, or the processes leading to cancer metastasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563206">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563206" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563206">
        <p class="paperTitle">Ribosome demand links transcriptional bursts to protein expression noise</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563206" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563206" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pal, S.; Ray, U.; Dhar, R.</p>
        <p class="info">Score: 7.0, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563206' target='https://doi.org/10.1101/2023.10.19.563206'> 10.1101/2023.10.19.563206</a></p>
        <p class="abstract">Stochastic variation in protein expression generates phenotypic heterogeneity in a cell population, and has important role in antibiotic persistence, mutation penetrance, cancer growth and therapy resistance. Studies investigating molecular origins of noise have predominantly focused on the transcription process. However, the noise generated during the transcription process is further modulated by the translation process, which eventually determines the expression noise at the protein level. Studies across different organisms have revealed a positive correlation between translational efficiency and protein noise. However, the molecular basis of this correlation has remained unknown. In this work, through stochastic modeling of translation in single mRNA molecules and empirical measurement of protein noise, we demonstrate that ribosome demand associated with high translational efficiency drives the correlation between translational efficiency and protein noise. We also show that this correlation is present only in genes with bursty transcription, where ribosome demand varies with stochastic fluctuations in mRNA numbers. Thus, our work reveals how transcriptional bursts are translated into protein noise, which has important implications for investigating protein noise and phenotypic heterogeneity across biological systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553401">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553401" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553401">
        <p class="paperTitle">A robust ultrasensitive transcriptional switch in noisy cellular environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553401" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553401" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jeong, E. M.; Kim, J. K.</p>
        <p class="info">Score: 11.3, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553401' target='https://doi.org/10.1101/2023.08.15.553401'> 10.1101/2023.08.15.553401</a></p>
        <p class="abstract">Ultrasensitive transcriptional switches enable sharp transitions between on and off states of transcription and are essential for cells to respond to environmental cues precisely. However, conventional switches, relying on direct repressor-DNA binding, are extremely sensitive to noise. Here, we discovered an alternative design combining three indirect transcriptional repression mechanisms, sequestration, blocking, and displacement, to generate a noise-resilient ultrasensitive switch. Although sequestration alone can generate an ultrasensitive switch, it remains sensitive to noise because the unintended transcriptional state induced by noise can persist for long periods. However, by jointly utilizing blocking and displacement, these noise-induced transitions can be rapidly restored to the original transcriptional state. Because this transcriptional switch is effective in noisy cellular contexts, it goes beyond previous synthetic transcriptional switches utilizing direct repression mechanisms, making it particularly valuable for robust synthetic system design. Our findings also provide insights into the evolution of robust ultrasensitive switches in real cells. Specifically, the concurrent use of seemingly redundant indirect repression mechanisms in diverse biological systems appears to be a strategy to achieve noise-resilience of ultrasensitive switches.

GRAPHICAL ABSTRACT

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=175 SRC=&#34;FIGDIR/small/553401v2_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (28K):
org.highwire.dtl.DTLVardef@1608e6eorg.highwire.dtl.DTLVardef@270493org.highwire.dtl.DTLVardef@19d5106org.highwire.dtl.DTLVardef@132701a_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563172">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563172" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563172">
        <p class="paperTitle">Noise robustness and metabolic load determine the principles of central dogma regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563172" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563172" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lo, T. W.; Choi, H. K. J.; Huang, D.; Wiggins, P. A.</p>
        <p class="info">Score: 3.6, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563172' target='https://doi.org/10.1101/2023.10.20.563172'> 10.1101/2023.10.20.563172</a></p>
        <p class="abstract">Protein expression levels optimize cell fitness: Too low of an expression level of essential proteins slows growth by compromising the function of essential processes, whereas the overexpression of proteins slows growth by increasing the metabolic load. This trade-off naively predicts that the cell maximizes its fitness by a Goldilocks principle in which cells express just enough protein for function; however, this strategy neglects the significance of the inherent stochasticity of the gene expression process which leads to significant cell-to-cell variation in protein numbers. How does the cell ensure robust growth in the face of the inherent stochasticity of these central dogma processes? To explore the consequences of noise in the expression of hundreds of essential genes, we build a minimal model where the trade-off between metabolic cost and growth robustness can be analyzed analytically. The model predicts that growth-rate maximization leads to a highly-asymmetric cost-benefit analysis which drives the optimal protein expression levels far above what is required on average, with low-expression essential proteins expressed at more than 10-fold what is required for growth in the typical cell. This robustness mechanism naturally explains the surprisingly strong buffering effect observed in essential protein depletion experiments and leads to a second qualitative prediction: there is a lower floor on the transcription level of essential genes corresponding to one message per cell cycle. We show that nearly all essential, but not non-essential, genes obey this limit in three evolutionarily-divergent model organisms: Escherichia coli, yeast, and human. The model also predicts that the optimal translation efficiency is roughly proportional to message abundance, predicting the observed relation between proteome fraction and message abundance in eukaryotic cells. This optimal translation efficiency predicts a non-canonical scaling of gene expression noise with protein abundance, which we show is observed in yeast. Together, these results reveal that noise robustness and metabolic load determine the global regulatory principles that govern central dogma function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.01.530599">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.01.530599" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.01.530599">
        <p class="paperTitle">Logic-based mechanistic machine learning on high-content images reveals how drugs differentially regulate cardiac fibroblasts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.01.530599" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.01.530599" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nelson, A. R.; Christiansen, S. L.; Naegle, K. M.; Saucerman, J. J.</p>
        <p class="info">Score: 11.7, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.01.530599' target='https://doi.org/10.1101/2023.03.01.530599'> 10.1101/2023.03.01.530599</a></p>
        <p class="abstract">Fibroblasts are essential regulators of extracellular matrix deposition following cardiac injury. These cells exhibit highly plastic responses in phenotype during fibrosis in response to environmental stimuli. Here, we test whether and how candidate anti-fibrotic drugs differentially regulate measures of cardiac fibroblast phenotype, which may help identify treatments for cardiac fibrosis. We conducted a high content microscopy screen of human cardiac fibroblasts treated with 13 clinically relevant drugs in the context of TGF{beta} and/or IL-1{beta}, measuring phenotype across 137 single-cell features. We used the phenotypic data from our high content imaging to train a logic-based mechanistic machine learning model (LogiMML) for fibroblast signaling. The model predicted how pirfenidone and Src inhibitor WH-4-023 reduce actin filament assembly and actin-myosin stress fiber formation, respectively. Validating the LogiMML model prediction that PI3K partially mediates the effects of Src inhibition, we found that PI3K inhibition reduces actin-myosin stress fiber formation and procollagen I production in human cardiac fibroblasts. In this study, we establish a modeling approach combining the strengths of logic-based network models and regularized regression models, apply this approach to predict mechanisms that mediate the differential effects of drugs on fibroblasts, revealing Src inhibition acting via PI3K as a potential therapy for cardiac fibrosis.

SignificanceCardiac fibrosis is a dysregulation of the normal wound healing response, resulting in excessive scarring and cardiac dysfunction. As cardiac fibroblasts primarily regulate this process, we explored how candidate anti-fibrotic drugs alter the fibroblast phenotype. We identify a set of 137 phenotypic features that change in response to drug treatments. Using a new computational modeling approach termed logic-based mechanistic machine learning, we predict how pirfenidone and Src inhibition affect the regulation of the phenotypic features actin filament assembly and actin-myosin stress fiber formation. We also show that inhibition of PI3K reduces actin-myosin stress fiber formation and procollagen I production in human cardiac fibroblasts, supporting a role for PI3K as a mechanism by which Src inhibition may suppress fibrosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563713">
        <p class="paperTitle">A Hill type equation reveals the regulation principle of target protein expression led by p53 pulsing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shi, X.</p>
        <p class="info">Score: 1.6, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563713' target='https://doi.org/10.1101/2023.10.24.563713'> 10.1101/2023.10.24.563713</a></p>
        <p class="abstract">The central dogma indicates the basic direction of gene expression pathway. For activated gene expression, the relationship between the various links that are from the binding of transcription factor (TF) DNA to synthesis of proteins is still unclear. There is a consensus that at steady state the protein levels are largely determined by mRNA level. How to find this steady state? What factors determine it? Taking TF p53 as an example, on the basis of the previously discovered Hill type equation that can characterize the average steady state of mRNA expression under p53 pulsing, and demonstrating the regulatory ability of duration and frequency beyond saturation. I proved that the average steady state of target protein expression driven by p53 pulsing can also be governed by this equation. Therefore, at steady state the average mRNA and protein expression under TF pulsing are exactly the same. This consensus has been successfully demonstrated. For gene BAX of p53 target, the observed fold changes of mRNA and protein are 1.72, 1.28, while the predictions for both mRNA and protein from the obtained equation are 1.35. The regulatory principle of gene expression revealed by this equation is that TF dynamics and binding affinity control the expression levels; and the half-life of mRNA and protein regulate the expression stability. By introducing two quantitative indicators, we can understand the degree of accumulation and stability of protein expression. Furthermore, this generalized equation broadens the application of classical biochemical theory.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563709">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563709" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563709">
        <p class="paperTitle">Global transcription regulation revealed from dynamical correlations in time-resolved single-cell RNA-sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563709" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563709" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Volteras, D.; Shahrezaei, V.; Thomas, P.</p>
        <p class="info">Score: 1.6, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563709' target='https://doi.org/10.1101/2023.10.24.563709'> 10.1101/2023.10.24.563709</a></p>
        <p class="abstract">Single-cell transcriptomics reveals significant variations in the transcriptional activity across cells. Yet, it remains challenging to identify mechanisms of transcription dynamics from static snapshots. It is thus still unknown what drives global transcription dynamics in single cells. We present a stochastic model of gene expression in growing and dividing cells that harnesses temporal dimensions of single-cell RNA-sequencing through metabolic labelling protocols and cell cycle reporters. We develop a parallel and highly scalable Approximate Bayesian Computation method that corrects for technical variation and accurately quantifies absolute burst frequency, burst size and degradation rate at a transcriptome-wide scale. Using Bayesian model selection, we reveal scaling between transcription rates and cell size and unveil waves of gene regulation across the cell cycle-dependent transcriptome. Our study shows that stochastic modelling of dynamical correlations identifies global mechanisms of transcription regulation. We illustrate how mechanistic modelling and amortised inference can efficiently integrate distributions across time, modalities and conditions. Our framework thus provides a versatile tool to uncover how global transcriptional dynamics is orchestrated genome-wide in single cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564032">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564032" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564032">
        <p class="paperTitle">Development of a genome scale metabolic model for the lager hybrid yeast S. pastorianus to understand evolution of metabolic pathways in industrial settings.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564032" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564032" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Timouma, S.; Balarezo-Cisneros, L. N.; Schwartz, J.-M.; Delneri, D.</p>
        <p class="info">Score: 1.5, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564032' target='https://doi.org/10.1101/2023.10.25.564032'> 10.1101/2023.10.25.564032</a></p>
        <p class="abstract">In silico tools such as genome-scale metabolic models (GSMM) have shown to be powerful for metabolic engineering of microorganisms. Here, we created the iSP_1513 GSMM for the aneuploid hybrid S. pastorianus CBS1513 to allow top-down computational approaches to predict the evolution of metabolic pathways and to aid strain optimisation and media engineering in production processes. The iSP_1513 comprises 4062 reactions, 1808 alleles and 2747 metabolites, and takes into account the functional redundancy in the gene-protein-reaction rule caused by the presence of orthologous genes. Moreover, a universal algorithm to constrain GSMM reactions using transcriptome data was developed as a python library and enabled the integration of temperature as parameter. Essentiality datasets, growth data on various carbohydrates and volatile metabolites secretion were used to validate the model. Overall, the iSP_1513 GSMM represent an important step towards understanding the metabolic capabilities, evolutionary trajectories and adaptation potential of S. pastorianus in different industrial settings.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
