<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Prevalence of and gene regulatory constraints on transcriptional adaptation in single cells
Authors: Mellis, I. A.; Bodkin, N.; Goyal, Y.
Score: 15.2, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553318
Cells and tissues have a remarkable ability to adapt to genetic perturbations via a variety of molecular mechanisms. Nonsense-induced transcriptional compensation, a form of transcriptional adaptation, has recently emerged as one such mechanism, in which nonsense mutations in a gene can trigger upregulation of related genes, possibly conferring robustness at cellular and organismal levels.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Prevalence of and gene regulatory constraints on transcriptional adaptation in single cells
Authors: Mellis, I. A.; Bodkin, N.; Goyal, Y.
Score: 15.2, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553318
Cells and tissues have a remarkable ability to adapt to genetic perturbations via a variety of molecular mechanisms. Nonsense-induced transcriptional compensation, a form of transcriptional adaptation, has recently emerged as one such mechanism, in which nonsense mutations in a gene can trigger upregulation of related genes, possibly conferring robustness at cellular and organismal levels." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-20T10:35:54+00:00" />
<meta property="article:modified_time" content="2023-08-20T10:35:54+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Prevalence of and gene regulatory constraints on transcriptional adaptation in single cells
Authors: Mellis, I. A.; Bodkin, N.; Goyal, Y.
Score: 15.2, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553318
Cells and tissues have a remarkable ability to adapt to genetic perturbations via a variety of molecular mechanisms. Nonsense-induced transcriptional compensation, a form of transcriptional adaptation, has recently emerged as one such mechanism, in which nonsense mutations in a gene can trigger upregulation of related genes, possibly conferring robustness at cellular and organismal levels."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Prevalence of and gene regulatory constraints on transcriptional adaptation in single cells\nAuthors: Mellis, I. A.; Bodkin, N.; Goyal, Y.\nScore: 15.2, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553318\nCells and tissues have a remarkable ability to adapt to genetic perturbations via a variety of molecular mechanisms. Nonsense-induced transcriptional compensation, a form of transcriptional adaptation, has recently emerged as one such mechanism, in which nonsense mutations in a gene can trigger upregulation of related genes, possibly conferring robustness at cellular and organismal levels.",
  "keywords": [
    
  ],
  "articleBody": " Prevalence of and gene regulatory constraints on transcriptional adaptation in single cells\nAuthors: Mellis, I. A.; Bodkin, N.; Goyal, Y.\nScore: 15.2, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553318\nCells and tissues have a remarkable ability to adapt to genetic perturbations via a variety of molecular mechanisms. Nonsense-induced transcriptional compensation, a form of transcriptional adaptation, has recently emerged as one such mechanism, in which nonsense mutations in a gene can trigger upregulation of related genes, possibly conferring robustness at cellular and organismal levels. However, beyond a handful of developmental contexts and curated sets of genes, to date, no comprehensive genome-wide investigation of this behavior has been undertaken for mammalian cell types and contexts. Moreover, how the regulatory-level effects of inherently stochastic compensatory gene networks contribute to phenotypic penetrance in single cells remains unclear. Here we combine computational analysis of existing datasets with stochastic mathematical modeling and machine learning to uncover the widespread prevalence of transcriptional adaptation in mammalian systems and the diverse single-cell manifestations of minimal compensatory gene networks. Regulon gene expression analysis of a pooled single-cell genetic perturbation dataset recapitulates our model predictions. Our integrative approach uncovers several putative hits-genes demonstrating possible transcriptional adaptation-to follow up on experimentally, and provides a formal quantitative framework to test and refine models of transcriptional adaptation.\nIntegration of Thermal Proteome Profiling with phosphoproteomic and transcriptomic data via mechanistic network models decodes the molecular response to PARP inhibition\nAuthors: Burtscher, M. L.; Gade, S.; Garrido-Rodriguez, M.; Rutkowska, A.; Werner, T.; Eberl, H. C.; Petretich, M.; Knopf, N.; Zirngibl, K.; Grandi, P.; Bergamini, G.; Bantscheff, M.; Faelth-Savitski, M.; Saez-Rodriguez, J.\nScore: 13.6, Published: 2023-08-16 DOI: 10.1101/2023.08.15.553354\nThe deregulation of complex diseases often spans multiple molecular processes. A multimodal functional characterization of these processes can shed light on the disease mechanisms and the effect of drugs. Thermal Proteome Profiling (TPP) is a mass-spectrometry based technique assessing changes in thermal protein stability that can serve as proxies of functional changes of the proteome. These unique insights of TPP can complement those obtained by other omics technologies. Here, we show how TPP can be integrated with phosphoproteomics and transcriptomics in a network-based approach using COSMOS, a framework for causal integration of multi-omics, to provide an integrated view of transcription factors, kinases and proteins with altered thermal stability. This allowed us to recover known mechanistic consequences of PARP inhibition in ovarian cancer cells on cell cycle and DNA damage response in detail and to uncover new insights into drug response mechanisms related to interferon and hippo signaling. We found that TPP complements the other omics data and allowed us to obtain a network model with higher coverage of the main underlying mechanisms. These results illustrate the added value of TPP, and more generally the power of network models to integrate the information provided by different omics technologies. We anticipate that this strategy can be used to broadly integrate functional proteomics with other omics to study complex molecular processes. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=63 SRC=\"FIGDIR/small/553354v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (18K): org.highwire.dtl.DTLVardef@12e9046org.highwire.dtl.DTLVardef@2178forg.highwire.dtl.DTLVardef@158d85forg.highwire.dtl.DTLVardef@1e570bf_HPS_FORMAT_FIGEXP M_FIG C_FIG\nRadiation necrosis after radiation therapy treatment of brain metastases: A computational approach\nAuthors: Ocana-Tienda, B.; Leon-Triana, O.; Perez-Beteta, J.; Perez-Garcia, V. M.\nScore: 7.6, Published: 2023-08-03 DOI: 10.1101/2023.08.01.551411\nMetastasis is the process through which cancer cells break away from a primary tumor, travel through the blood or lymph system, and form new tumors in distant tissues. One of the preferred sites for metastatic dissemination is the brain, affecting more than 20% of all cancer patients. This figure is increasing steadily due to improvements in treatments of primary tumors. Stereotactic radiosurgery (SRS) is one of the main treatment options for patients with a small or moderate number of brain metastases (BMs). A frequent adverse event of SRS is radiation necrosis (RN), an inflammatory condition caused by late normal tissue cell death. A major diagnostic problem is that RNs are difficult to distinguish from BM recurrences, due to their similarities on standard magnetic resonance images (MRIs). However, this distinction is key to choosing the best therapeutic approach since RNs resolve often without further interventions, while relapsing BMs may require open brain surgery. Recent research has shown that RNs have a faster growth dynamics than recurrent BMs, providing a way to differentiate the two entities, but no mechanistic explanation has been provided for those observations. In this study, computational frameworks were developed based on mathematical models of increasing complexity, providing mechanistic explanations for the differential growth dynamics of BMs relapse versus RN events and explaining the observed clinical phenomenology. Simulated tumor relapses were found to have growth exponents substantially smaller than the group in which there was inflammation due to damage induced by SRS to normal brain tissue adjacent to the BMs, thus leading to RN. ROC curves with the synthetic data had an optimal threshold that maximized the sensitivity and specificity values for a growth exponent {beta}* = 1.05, very close to that observed in patient datasets. Author summaryAfter treatment of brain metastases with radiation therapy, a fraction of patients experience tumor recurrences and others display radiation necrosis (RN). Clinical data shows that the growth of RNs is faster, as measured by the growth exponent, than that of recurrent brain metastases. This reflects the inflammatory nature of the former, and provides a method to distinguish RN in the clinics from relapsing metastatic lesions. A simple mathematical model for the inflammatory response and a more sophisticate discrete stochastic simulator with many biological details were been developed to provide a mechanistic explanation of the differential dynamics of tumor growth versus inflammatory responses after stereotactic radiation surgery of metastatic brain lesions.\nMetabolic Symbiosis between Oxygenated and Hypoxic Tumour Cells: An Agent-based Modelling Study\nAuthors: Jayathilake, P.; Victori, P.; Pavillet, C.; Voukantsis, D.; Miar Cuervo, A.; Arora, A.; Harris, A. L.; Morten, K.; Buffa, F. M.\nScore: 7.6, Published: 2023-07-24 DOI: 10.1101/2023.07.24.550289\nDeregulated metabolism is one of the hallmarks of cancer. It is well-known that tumour cells tend to metabolize glucose via glycolysis even when oxygen is available and mitochondrial respiration is functional. However, the lower energy efficiency of aerobic glycolysis with respect to mitochondrial respiration makes this behaviour, namely the Warburg effect, counter-intuitive, although it has now been recognized as a source of anabolic precursors. On the other hand, there is evidence that oxygenated tumour cells could be fuelled by exogenous lactate produced from glycolysis, and that glycolytic tumour cells export lactate to the environment. Using a multi-scale approach combining multi-agent modelling, diffusion-reaction and stoichiometric equations, we simulated these cell populations and their environment, and studied the metabolic co-operation between cells exposed to different oxygen concentration and nutrient abundance. The results show that the metabolic cooperation between cells reduces the depletion of environmental glucose, resulting in an overall advantage of using aerobic glycolysis. In addition, the environmental oxygen level was found to be decreased by the symbiosis, promoting a further shift towards anaerobic glycolysis. However, the oxygenated and hypoxic cell populations may gradually reach quasi-equilibrium. A sensitivity analysis based on Latin hypercube sampling and partial rank correlation shows that the symbiotic dynamics depends on properties of the specific cell such as the minimum glucose level needed for glycolysis. Our results suggest that strategies that block glucose transporters may be more effective than those blocking lactate intake transporters to reduce tumour growth by blocking lactate production. AUTHOR SUMMARYMetabolic alteration is one of the hallmarks of cancer and the well-known metabolic alteration of tumour cells is that cells prefer to do glycolysis over mitochondrial respiration even under well-oxygenated and functional mitochondrial conditions. On the other hand, there is evidence that oxygenated tumour cells could be fuelled by exogenous lactate produced from hypoxic glycolytic cells in which it can create a metabolic co-operation between oxygenated and hypoxic cell populations. This metabolic co-operation could allow tumour cells to economically share oxygen and glucose and promote tumour survival. Using a multi-scale approach combining multi-agent modelling, diffusion-reaction and stoichiometric equations, we studied this metabolic co-operation between different populations of cells, exposed to a changing microenvironment. We predict that the tumour environmental glucose depletion is decreased while the oxygen depletion is increased by this metabolic symbiosis, promoting a further shift towards glycolysis. Our results also show that blocking glucose transporters could be more effective than blocking lactate intake transporters, because the former would disrupt both glycolysis and lactate production, drastically reducing tumour growth.\nCURTAIN - A Unique Web-based tool for exploration and sharing of MS-based proteomics data\nAuthors: Phung, T. K.; Berndsen, K.; Phan, T. L.; Muqit, M. M.; Alessi, D. R.; Nirujogi, R. S.\nScore: 28.0, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550405\nTo facilitate analysis and sharing of mass spectrometry (MS)-based proteomics data we created tools called CURTAIN (https://curtain.proteo.info) and CURTAIN-PTM (https://curtainptm.proteo.info). These enable the non-MS expert to interactively peruse volcano plots; deconvolute primary experimental data to individual replicates that can be visualized in bar charts or violin plots allowing statistical analysis; and export of plots in SVG format. They also permit assessment of experimental quality by correlation matrix and profile plot. Within CURTAIN, the user can analyze domain structure, AlphaFold predicted structure, reported interactors, relative expression, disease and pharmaceutical links, and mutagenesis information on all selected hits. Moreover, CURTAIN-PTM permits the comparison of all identified PTM sites on protein(s) of interest with PTM information contained within selected databases. For phosphorylation site analysis CURTAIN-PTM links with the kinase library to predict upstream kinases that phosphorylate sites of interest. We provide examples of the utility of CURTAIN and CURTAIN-PTM in analyzing how targeted degradation of the PPM1H Rab phosphatase that counteracts the Parkinsons LRRK2 kinase impacts cellular protein levels and phosphorylation sites. We reanalyzed a ubiquitylation dataset, characterizing the PINK1-Parkin pathway activation in primary neurons, revealing new data of interest not highlighted previously. CURTAIN and CURTAIN-PTM are free to use and open-source and will enable researchers to share and maximize the analysis and impact of their proteomics data. We advocate that differential expression proteomic data should be published containing a shareable CURTAIN web-link, allowing readers to better explore their data. Significance StatementTo enable non-experts to better share and explore mass spectrometry data, we have generated using open-source software, interactive tools termed CURTAIN and CURTAIN-PTM. These tools enable users to save their analysis sessions with a sharable unique web-link, enabling other researchers to visualize and further analyze these datasets. These links can also be reported in publications allowing readers to further survey the reported data. We discuss benefits for the research community of publishing proteomic data containing a shareable web-link.\nTwo-way feedback between chromatin compaction and histone modification state explains S. cerevisiae heterochromatin bistability\nAuthors: Movilla Miangolarra, A.; Saxton, D. S.; Yan, Z.; Rine, J.; Howard, M.\nScore: 3.5, Published: 2023-08-15 DOI: 10.1101/2023.08.12.552948\nCompact chromatin is closely linked with gene silencing in part by sterically masking access to promoters, inhibiting transcription factor binding and preventing polymerase from efficiently transcribing a gene. Here, we propose a broader view: chromatin compaction can be both a cause and a consequence of the histone modification state, and this tight bidirectional interaction can underpin bistable transcriptional states. To test this theory, we developed a mathematical model for the dynamics of the HMR locus in S. cerevisiae, that incorporates activating histone modifications, silencing proteins and a dynamic, acetylation-dependent, three-dimensional locus size. Chromatin compaction enhances silencer protein binding, which in turn feeds back to remove activating histone modifications, leading to further compaction. The bistable output of the model was in good agreement with prior quantitative data, including switching rates from expressed to silent states, and vice versa, and protein binding levels within the locus. We then tested the model by predicting changes in switching rates as the genetic length of the locus was increased, which were then experimentally verified. This bidirectional feedback between chromatin compaction and the histone modification state may be an important regulatory mechanism at many loci. SignificanceChromatin is the complex formed by proteins, including histones, and DNA to form chromosomes. Specific chromatin structures and states are thought to be key factors regulating transcription. A common view proposes that histone modifications activate or inhibit transcription either via specific activation or inhibition of RNA polymerase binding/elongation at a locus, or by expanding/compacting the locus, thereby modulating its accessibility to many macromolecules. In this work, we elucidated a broader hypothesis that chromatin compaction may both inhibit transcription, and feedback via silencing proteins to remove histone modifications that further control chromatin compaction and correlate with gene activity. We developed a model incorporating these ideas and showed that it explains quantitative experimental data for a silent locus in budding yeast.\nDLTKcat: deep learning based prediction of temperature dependent enzyme turnover rates\nAuthors: Qiu, S.; Zhao, S.; Yang, A.\nScore: 2.6, Published: 2023-08-15 DOI: 10.1101/2023.08.10.552798\nThe enzyme turnover rate, kcat, quantifies enzyme kinetics by indicating the maximum efficiency of enzyme catalysis. Despite its importance, kcat values remain scarce in databases for most organisms, primarily due to the cost of experimental measurements. To predict kcat and account for its strong temperature dependence, DLTKcat was developed in this study and demonstrated superior performance (log10-scale RMSE = 0.88, R2 = 0.66) than previously published models. Through two case studies, DLTKcat showed its ability to predict the effect of protein sequence mutations and temperature changes on kcat values. Although its quantitative accuracy is not high enough yet to model the responses of cellular metabolism to temperature changes, DLTKcat has the potential to eventually become a computational tool to describe the temperature dependence of biological systems.\nRobust Ultrasensitive Transcriptional Switch in Noisy Cellular Environments\nAuthors: Jeong, E. M.; Kim, J. K.\nScore: 2.1, Published: 2023-08-17 DOI: 10.1101/2023.08.15.553401\nUltrasensitive transcriptional switches enable cells to make sharp transitions between on and off states of transcription, essential for cells to respond to environmental cues with high fidelity. However, conventional switches that rely on direct repressor-DNA binding are extremely sensitive to noise, leading to unintended changes in gene expression. This raises doubts about the effectiveness of these switches in noisy cellular environments. Here, we find an alternative avenue, where the combination of different indirect transcriptional regulations generates a noise-resilient ultrasensitive switch. Specifically, when a transcriptional repressor inhibits a transcriptional activator through the combination of sequestration, blocking, and displacement instead of directly binding to DNA, a robust ultrasensitive switch is achieved. Sequestration alone can generate an ultrasensitive transcriptional response, but the switch remains vulnerable to noise because the undesired on or off state induced by noise can persist for a long period. However, by jointly utilizing blocking and displacement, noise-induced transitions can be rapidly restored to the original desired state. This transcriptional switch is effective in noisy cellular contexts, which makes it suitable for robust synthetic system design. Our findings also provide insights into the evolution of ultrasensitive switches in real cells. The concurrent use of seemingly redundant multiple indirect transcriptional repression mechanisms in diverse biological systems appears to be a strategy to improve the noise-resilience of ultrasensitive switches.\nAgent-based model demonstrates the impact of nonlinear, complex interactions between cytokines on muscle regeneration\nAuthors: Haase, M.; Comlekoglu, T.; Petrucciani, A.; Peirce, S. M.; Blemker, S. S.\nScore: 1.8, Published: 2023-08-16 DOI: 10.1101/2023.08.14.553247\nMuscle regeneration is a complex process due to dynamic and multiscale biochemical and cellular interactions, making it difficult to determine optimal treatments for muscle injury using experimental approaches alone. To understand the degree to which individual cellular behaviors impact endogenous mechanisms of muscle recovery, we developed an agent-based model (ABM) using the Cellular Potts framework to simulate the dynamic microenvironment of a cross-section of murine skeletal muscle tissue. We referenced more than 100 published studies to define over 100 parameters and rules that dictate the behavior of muscle fibers, satellite stem cells (SSC), fibroblasts, neutrophils, macrophages, microvessels, and lymphatic vessels, as well as their interactions with each other and the microenvironment. We utilized parameter density estimation to calibrate the model to temporal biological datasets describing cross-sectional area (CSA) recovery, SSC, and fibroblast cell counts at multiple time points following injury. The calibrated model was validated by comparison of other model outputs (macrophage, neutrophil, and capillaries counts) to experimental observations. Predictions for eight model perturbations that varied cell or cytokine input conditions were compared to published experimental studies to validate model predictive capabilities. We used Latin hypercube sampling and partial rank correlation coefficient to identify in silico perturbations of cytokine diffusion coefficients and decay rates to enhance CSA recovery. This analysis suggests that combined alterations of specific cytokine decay and diffusion parameters result in greater fibroblast and SSC proliferation compared to individual perturbations with a 13% increase in CSA recovery compared to unaltered regeneration at 28 days. These results enable guided development of therapeutic strategies that similarly alter muscle physiology (i.e. converting ECM-bound cytokines into freely diffusible forms as studied in cancer therapeutics or delivery of exogenous cytokines) during regeneration to enhance muscle recovery after injury.\nMachine Learning-Driven Phenotype Predictions based on Genome Annotations\nAuthors: Edirisinghe, J. N.; Goyal, S.; Brace, A.; Colasanti, R.; Gu, T.; Sadhkin, B.; Zhang, Q.; Kamimura, R.; Henry, C. S.\nScore: 1.5, Published: 2023-08-14 DOI: 10.1101/2023.08.11.552879\nOver the past two decades, there has been a remarkable and exponential expansion in the availability of genome sequences, encompassing a vast number of isolate genomes, amounting to hundreds of thousands, and now extending to millions of metagenome-assembled genomes. The rapid and accurate interpretation of this data, along with the profiling of diverse phenotypes such as respiration type, antimicrobial resistance, or carbon utilization, is essential for a wide range of medical and research applications. Here, we leverage sequenced-based functional annotations obtained from the RAST annotation algorithm as predictors and employ six machine learning algorithms (K-Nearest Neighbors, Gaussian Naive Bayes, Support Vector Machines, Neural Networks, Logistic Regression, and Decision Trees) to generate classifiers that can accurately predict phenotypes of unclassified bacterial organisms. We apply this approach in two case studies focused on respiration types (aerobic, anaerobic, and facultative anaerobic) and Gram-stain types (Gram negative and Gram positive). We demonstrate that all six classifiers accurately classify the phenotypes of Gram stain and respiration type, and discuss the biological significance of the predicted outcomes. We also present four new applications that have been deployed in The Department of Energy Systems Biology Knowledgebase (KBase) that enable users to: (i) Upload high-quality data to train classifiers; (ii) Annotate genomes in the training set with the RAST annotation algorithm; (iii) Build six different genome classifiers; and (iv) Predict the phenotype of unclassified genomes. (https://narrative.kbase.us/#catalog/modules/kb_genomeclassification)\n",
  "wordCount" : "3047",
  "inLanguage": "en",
  "datePublished": "2023-08-20T10:35:54Z",
  "dateModified": "2023-08-20T10:35:54Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on August 20, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553318">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553318" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553318">
        <p class="paperTitle">Prevalence of and gene regulatory constraints on transcriptional adaptation in single cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553318" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553318" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mellis, I. A.; Bodkin, N.; Goyal, Y.</p>
        <p class="info">Score: 15.2, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553318' target='https://doi.org/10.1101/2023.08.14.553318'> 10.1101/2023.08.14.553318</a></p>
        <p class="abstract">Cells and tissues have a remarkable ability to adapt to genetic perturbations via a variety of molecular mechanisms. Nonsense-induced transcriptional compensation, a form of transcriptional adaptation, has recently emerged as one such mechanism, in which nonsense mutations in a gene can trigger upregulation of related genes, possibly conferring robustness at cellular and organismal levels. However, beyond a handful of developmental contexts and curated sets of genes, to date, no comprehensive genome-wide investigation of this behavior has been undertaken for mammalian cell types and contexts. Moreover, how the regulatory-level effects of inherently stochastic compensatory gene networks contribute to phenotypic penetrance in single cells remains unclear. Here we combine computational analysis of existing datasets with stochastic mathematical modeling and machine learning to uncover the widespread prevalence of transcriptional adaptation in mammalian systems and the diverse single-cell manifestations of minimal compensatory gene networks. Regulon gene expression analysis of a pooled single-cell genetic perturbation dataset recapitulates our model predictions. Our integrative approach uncovers several putative hits-genes demonstrating possible transcriptional adaptation-to follow up on experimentally, and provides a formal quantitative framework to test and refine models of transcriptional adaptation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553354" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553354">
        <p class="paperTitle">Integration of Thermal Proteome Profiling with phosphoproteomic and transcriptomic data via mechanistic network models decodes the molecular response to PARP inhibition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burtscher, M. L.; Gade, S.; Garrido-Rodriguez, M.; Rutkowska, A.; Werner, T.; Eberl, H. C.; Petretich, M.; Knopf, N.; Zirngibl, K.; Grandi, P.; Bergamini, G.; Bantscheff, M.; Faelth-Savitski, M.; Saez-Rodriguez, J.</p>
        <p class="info">Score: 13.6, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553354' target='https://doi.org/10.1101/2023.08.15.553354'> 10.1101/2023.08.15.553354</a></p>
        <p class="abstract">The deregulation of complex diseases often spans multiple molecular processes. A multimodal functional characterization of these processes can shed light on the disease mechanisms and the effect of drugs. Thermal Proteome Profiling (TPP) is a mass-spectrometry based technique assessing changes in thermal protein stability that can serve as proxies of functional changes of the proteome. These unique insights of TPP can complement those obtained by other omics technologies. Here, we show how TPP can be integrated with phosphoproteomics and transcriptomics in a network-based approach using COSMOS, a framework for causal integration of multi-omics, to provide an integrated view of transcription factors, kinases and proteins with altered thermal stability. This allowed us to recover known mechanistic consequences of PARP inhibition in ovarian cancer cells on cell cycle and DNA damage response in detail and to uncover new insights into drug response mechanisms related to interferon and hippo signaling. We found that TPP complements the other omics data and allowed us to obtain a network model with higher coverage of the main underlying mechanisms. These results illustrate the added value of TPP, and more generally the power of network models to integrate the information provided by different omics technologies. We anticipate that this strategy can be used to broadly integrate functional proteomics with other omics to study complex molecular processes.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=63 SRC=&#34;FIGDIR/small/553354v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (18K):
org.highwire.dtl.DTLVardef@12e9046org.highwire.dtl.DTLVardef@2178forg.highwire.dtl.DTLVardef@158d85forg.highwire.dtl.DTLVardef@1e570bf_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551411">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551411" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551411">
        <p class="paperTitle">Radiation necrosis after radiation therapy treatment of brain metastases: A computational approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551411" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551411" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ocana-Tienda, B.; Leon-Triana, O.; Perez-Beteta, J.; Perez-Garcia, V. M.</p>
        <p class="info">Score: 7.6, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551411' target='https://doi.org/10.1101/2023.08.01.551411'> 10.1101/2023.08.01.551411</a></p>
        <p class="abstract">Metastasis is the process through which cancer cells break away from a primary tumor, travel through the blood or lymph system, and form new tumors in distant tissues. One of the preferred sites for metastatic dissemination is the brain, affecting more than 20% of all cancer patients. This figure is increasing steadily due to improvements in treatments of primary tumors. Stereotactic radiosurgery (SRS) is one of the main treatment options for patients with a small or moderate number of brain metastases (BMs). A frequent adverse event of SRS is radiation necrosis (RN), an inflammatory condition caused by late normal tissue cell death. A major diagnostic problem is that RNs are difficult to distinguish from BM recurrences, due to their similarities on standard magnetic resonance images (MRIs). However, this distinction is key to choosing the best therapeutic approach since RNs resolve often without further interventions, while relapsing BMs may require open brain surgery. Recent research has shown that RNs have a faster growth dynamics than recurrent BMs, providing a way to differentiate the two entities, but no mechanistic explanation has been provided for those observations.

In this study, computational frameworks were developed based on mathematical models of increasing complexity, providing mechanistic explanations for the differential growth dynamics of BMs relapse versus RN events and explaining the observed clinical phenomenology. Simulated tumor relapses were found to have growth exponents substantially smaller than the group in which there was inflammation due to damage induced by SRS to normal brain tissue adjacent to the BMs, thus leading to RN. ROC curves with the synthetic data had an optimal threshold that maximized the sensitivity and specificity values for a growth exponent {beta}* = 1.05, very close to that observed in patient datasets.

Author summaryAfter treatment of brain metastases with radiation therapy, a fraction of patients experience tumor recurrences and others display radiation necrosis (RN). Clinical data shows that the growth of RNs is faster, as measured by the growth exponent, than that of recurrent brain metastases. This reflects the inflammatory nature of the former, and provides a method to distinguish RN in the clinics from relapsing metastatic lesions. A simple mathematical model for the inflammatory response and a more sophisticate discrete stochastic simulator with many biological details were been developed to provide a mechanistic explanation of the differential dynamics of tumor growth versus inflammatory responses after stereotactic radiation surgery of metastatic brain lesions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550289">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550289" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550289">
        <p class="paperTitle">Metabolic Symbiosis between Oxygenated and Hypoxic Tumour Cells: An Agent-based Modelling Study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550289" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550289" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jayathilake, P.; Victori, P.; Pavillet, C.; Voukantsis, D.; Miar Cuervo, A.; Arora, A.; Harris, A. L.; Morten, K.; Buffa, F. M.</p>
        <p class="info">Score: 7.6, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550289' target='https://doi.org/10.1101/2023.07.24.550289'> 10.1101/2023.07.24.550289</a></p>
        <p class="abstract">Deregulated metabolism is one of the hallmarks of cancer. It is well-known that tumour cells tend to metabolize glucose via glycolysis even when oxygen is available and mitochondrial respiration is functional. However, the lower energy efficiency of aerobic glycolysis with respect to mitochondrial respiration makes this behaviour, namely the Warburg effect, counter-intuitive, although it has now been recognized as a source of anabolic precursors. On the other hand, there is evidence that oxygenated tumour cells could be fuelled by exogenous lactate produced from glycolysis, and that glycolytic tumour cells export lactate to the environment. Using a multi-scale approach combining multi-agent modelling, diffusion-reaction and stoichiometric equations, we simulated these cell populations and their environment, and studied the metabolic co-operation between cells exposed to different oxygen concentration and nutrient abundance. The results show that the metabolic cooperation between cells reduces the depletion of environmental glucose, resulting in an overall advantage of using aerobic glycolysis. In addition, the environmental oxygen level was found to be decreased by the symbiosis, promoting a further shift towards anaerobic glycolysis. However, the oxygenated and hypoxic cell populations may gradually reach quasi-equilibrium. A sensitivity analysis based on Latin hypercube sampling and partial rank correlation shows that the symbiotic dynamics depends on properties of the specific cell such as the minimum glucose level needed for glycolysis. Our results suggest that strategies that block glucose transporters may be more effective than those blocking lactate intake transporters to reduce tumour growth by blocking lactate production.

AUTHOR SUMMARYMetabolic alteration is one of the hallmarks of cancer and the well-known metabolic alteration of tumour cells is that cells prefer to do glycolysis over mitochondrial respiration even under well-oxygenated and functional mitochondrial conditions. On the other hand, there is evidence that oxygenated tumour cells could be fuelled by exogenous lactate produced from hypoxic glycolytic cells in which it can create a metabolic co-operation between oxygenated and hypoxic cell populations. This metabolic co-operation could allow tumour cells to economically share oxygen and glucose and promote tumour survival. Using a multi-scale approach combining multi-agent modelling, diffusion-reaction and stoichiometric equations, we studied this metabolic co-operation between different populations of cells, exposed to a changing microenvironment. We predict that the tumour environmental glucose depletion is decreased while the oxygen depletion is increased by this metabolic symbiosis, promoting a further shift towards glycolysis. Our results also show that blocking glucose transporters could be more effective than blocking lactate intake transporters, because the former would disrupt both glycolysis and lactate production, drastically reducing tumour growth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550405">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550405" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550405">
        <p class="paperTitle">CURTAIN - A Unique Web-based tool for exploration and sharing of MS-based proteomics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550405" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550405" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Phung, T. K.; Berndsen, K.; Phan, T. L.; Muqit, M. M.; Alessi, D. R.; Nirujogi, R. S.</p>
        <p class="info">Score: 28.0, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550405' target='https://doi.org/10.1101/2023.07.25.550405'> 10.1101/2023.07.25.550405</a></p>
        <p class="abstract">To facilitate analysis and sharing of mass spectrometry (MS)-based proteomics data we created tools called CURTAIN (https://curtain.proteo.info) and CURTAIN-PTM (https://curtainptm.proteo.info). These enable the non-MS expert to interactively peruse volcano plots; deconvolute primary experimental data to individual replicates that can be visualized in bar charts or violin plots allowing statistical analysis; and export of plots in SVG format. They also permit assessment of experimental quality by correlation matrix and profile plot. Within CURTAIN, the user can analyze domain structure, AlphaFold predicted structure, reported interactors, relative expression, disease and pharmaceutical links, and mutagenesis information on all selected hits. Moreover, CURTAIN-PTM permits the comparison of all identified PTM sites on protein(s) of interest with PTM information contained within selected databases. For phosphorylation site analysis CURTAIN-PTM links with the kinase library to predict upstream kinases that phosphorylate sites of interest. We provide examples of the utility of CURTAIN and CURTAIN-PTM in analyzing how targeted degradation of the PPM1H Rab phosphatase that counteracts the Parkinsons LRRK2 kinase impacts cellular protein levels and phosphorylation sites. We reanalyzed a ubiquitylation dataset, characterizing the PINK1-Parkin pathway activation in primary neurons, revealing new data of interest not highlighted previously. CURTAIN and CURTAIN-PTM are free to use and open-source and will enable researchers to share and maximize the analysis and impact of their proteomics data. We advocate that differential expression proteomic data should be published containing a shareable CURTAIN web-link, allowing readers to better explore their data.

Significance StatementTo enable non-experts to better share and explore mass spectrometry data, we have generated using open-source software, interactive tools termed CURTAIN and CURTAIN-PTM. These tools enable users to save their analysis sessions with a sharable unique web-link, enabling other researchers to visualize and further analyze these datasets. These links can also be reported in publications allowing readers to further survey the reported data. We discuss benefits for the research community of publishing proteomic data containing a shareable web-link.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.552948">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.552948" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.552948">
        <p class="paperTitle">Two-way feedback between chromatin compaction and histone modification state explains S. cerevisiae heterochromatin bistability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.552948" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.552948" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Movilla Miangolarra, A.; Saxton, D. S.; Yan, Z.; Rine, J.; Howard, M.</p>
        <p class="info">Score: 3.5, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.552948' target='https://doi.org/10.1101/2023.08.12.552948'> 10.1101/2023.08.12.552948</a></p>
        <p class="abstract">Compact chromatin is closely linked with gene silencing in part by sterically masking access to promoters, inhibiting transcription factor binding and preventing polymerase from efficiently transcribing a gene. Here, we propose a broader view: chromatin compaction can be both a cause and a consequence of the histone modification state, and this tight bidirectional interaction can underpin bistable transcriptional states. To test this theory, we developed a mathematical model for the dynamics of the HMR locus in S. cerevisiae, that incorporates activating histone modifications, silencing proteins and a dynamic, acetylation-dependent, three-dimensional locus size. Chromatin compaction enhances silencer protein binding, which in turn feeds back to remove activating histone modifications, leading to further compaction. The bistable output of the model was in good agreement with prior quantitative data, including switching rates from expressed to silent states, and vice versa, and protein binding levels within the locus. We then tested the model by predicting changes in switching rates as the genetic length of the locus was increased, which were then experimentally verified. This bidirectional feedback between chromatin compaction and the histone modification state may be an important regulatory mechanism at many loci.

SignificanceChromatin is the complex formed by proteins, including histones, and DNA to form chromosomes. Specific chromatin structures and states are thought to be key factors regulating transcription. A common view proposes that histone modifications activate or inhibit transcription either via specific activation or inhibition of RNA polymerase binding/elongation at a locus, or by expanding/compacting the locus, thereby modulating its accessibility to many macromolecules. In this work, we elucidated a broader hypothesis that chromatin compaction may both inhibit transcription, and feedback via silencing proteins to remove histone modifications that further control chromatin compaction and correlate with gene activity. We developed a model incorporating these ideas and showed that it explains quantitative experimental data for a silent locus in budding yeast.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552798" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552798">
        <p class="paperTitle">DLTKcat: deep learning based prediction of temperature dependent enzyme turnover rates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Qiu, S.; Zhao, S.; Yang, A.</p>
        <p class="info">Score: 2.6, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552798' target='https://doi.org/10.1101/2023.08.10.552798'> 10.1101/2023.08.10.552798</a></p>
        <p class="abstract">The enzyme turnover rate, kcat, quantifies enzyme kinetics by indicating the maximum efficiency of enzyme catalysis. Despite its importance, kcat values remain scarce in databases for most organisms, primarily due to the cost of experimental measurements. To predict kcat and account for its strong temperature dependence, DLTKcat was developed in this study and demonstrated superior performance (log10-scale RMSE = 0.88, R2 = 0.66) than previously published models. Through two case studies, DLTKcat showed its ability to predict the effect of protein sequence mutations and temperature changes on kcat values. Although its quantitative accuracy is not high enough yet to model the responses of cellular metabolism to temperature changes, DLTKcat has the potential to eventually become a computational tool to describe the temperature dependence of biological systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553401">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553401" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553401">
        <p class="paperTitle">Robust Ultrasensitive Transcriptional Switch in Noisy Cellular Environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553401" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553401" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jeong, E. M.; Kim, J. K.</p>
        <p class="info">Score: 2.1, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553401' target='https://doi.org/10.1101/2023.08.15.553401'> 10.1101/2023.08.15.553401</a></p>
        <p class="abstract">Ultrasensitive transcriptional switches enable cells to make sharp transitions between on and off states of transcription, essential for cells to respond to environmental cues with high fidelity. However, conventional switches that rely on direct repressor-DNA binding are extremely sensitive to noise, leading to unintended changes in gene expression. This raises doubts about the effectiveness of these switches in noisy cellular environments. Here, we find an alternative avenue, where the combination of different indirect transcriptional regulations generates a noise-resilient ultrasensitive switch. Specifically, when a transcriptional repressor inhibits a transcriptional activator through the combination of sequestration, blocking, and displacement instead of directly binding to DNA, a robust ultrasensitive switch is achieved. Sequestration alone can generate an ultrasensitive transcriptional response, but the switch remains vulnerable to noise because the undesired on or off state induced by noise can persist for a long period. However, by jointly utilizing blocking and displacement, noise-induced transitions can be rapidly restored to the original desired state. This transcriptional switch is effective in noisy cellular contexts, which makes it suitable for robust synthetic system design. Our findings also provide insights into the evolution of ultrasensitive switches in real cells. The concurrent use of seemingly redundant multiple indirect transcriptional repression mechanisms in diverse biological systems appears to be a strategy to improve the noise-resilience of ultrasensitive switches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553247">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553247" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553247">
        <p class="paperTitle">Agent-based model demonstrates the impact of nonlinear, complex interactions between cytokines on muscle regeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553247" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553247" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Haase, M.; Comlekoglu, T.; Petrucciani, A.; Peirce, S. M.; Blemker, S. S.</p>
        <p class="info">Score: 1.8, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553247' target='https://doi.org/10.1101/2023.08.14.553247'> 10.1101/2023.08.14.553247</a></p>
        <p class="abstract">Muscle regeneration is a complex process due to dynamic and multiscale biochemical and cellular interactions, making it difficult to determine optimal treatments for muscle injury using experimental approaches alone. To understand the degree to which individual cellular behaviors impact endogenous mechanisms of muscle recovery, we developed an agent-based model (ABM) using the Cellular Potts framework to simulate the dynamic microenvironment of a cross-section of murine skeletal muscle tissue. We referenced more than 100 published studies to define over 100 parameters and rules that dictate the behavior of muscle fibers, satellite stem cells (SSC), fibroblasts, neutrophils, macrophages, microvessels, and lymphatic vessels, as well as their interactions with each other and the microenvironment. We utilized parameter density estimation to calibrate the model to temporal biological datasets describing cross-sectional area (CSA) recovery, SSC, and fibroblast cell counts at multiple time points following injury. The calibrated model was validated by comparison of other model outputs (macrophage, neutrophil, and capillaries counts) to experimental observations. Predictions for eight model perturbations that varied cell or cytokine input conditions were compared to published experimental studies to validate model predictive capabilities. We used Latin hypercube sampling and partial rank correlation coefficient to identify in silico perturbations of cytokine diffusion coefficients and decay rates to enhance CSA recovery. This analysis suggests that combined alterations of specific cytokine decay and diffusion parameters result in greater fibroblast and SSC proliferation compared to individual perturbations with a 13% increase in CSA recovery compared to unaltered regeneration at 28 days. These results enable guided development of therapeutic strategies that similarly alter muscle physiology (i.e. converting ECM-bound cytokines into freely diffusible forms as studied in cancer therapeutics or delivery of exogenous cytokines) during regeneration to enhance muscle recovery after injury.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552879">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552879" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552879">
        <p class="paperTitle">Machine Learning-Driven Phenotype Predictions based on Genome Annotations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552879" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552879" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Edirisinghe, J. N.; Goyal, S.; Brace, A.; Colasanti, R.; Gu, T.; Sadhkin, B.; Zhang, Q.; Kamimura, R.; Henry, C. S.</p>
        <p class="info">Score: 1.5, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552879' target='https://doi.org/10.1101/2023.08.11.552879'> 10.1101/2023.08.11.552879</a></p>
        <p class="abstract">Over the past two decades, there has been a remarkable and exponential expansion in the availability of genome sequences, encompassing a vast number of isolate genomes, amounting to hundreds of thousands, and now extending to millions of metagenome-assembled genomes. The rapid and accurate interpretation of this data, along with the profiling of diverse phenotypes such as respiration type, antimicrobial resistance, or carbon utilization, is essential for a wide range of medical and research applications.

Here, we leverage sequenced-based functional annotations obtained from the RAST annotation algorithm as predictors and employ six machine learning algorithms (K-Nearest Neighbors, Gaussian Naive Bayes, Support Vector Machines, Neural Networks, Logistic Regression, and Decision Trees) to generate classifiers that can accurately predict phenotypes of unclassified bacterial organisms. We apply this approach in two case studies focused on respiration types (aerobic, anaerobic, and facultative anaerobic) and Gram-stain types (Gram negative and Gram positive). We demonstrate that all six classifiers accurately classify the phenotypes of Gram stain and respiration type, and discuss the biological significance of the predicted outcomes. We also present four new applications that have been deployed in The Department of Energy Systems Biology Knowledgebase (KBase) that enable users to: (i) Upload high-quality data to train classifiers; (ii) Annotate genomes in the training set with the RAST annotation algorithm; (iii) Build six different genome classifiers; and (iv) Predict the phenotype of unclassified genomes. (https://narrative.kbase.us/#catalog/modules/kb_genomeclassification)</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
