<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Organellomics: AI-driven deep organellar phenotyping of human neurons
Authors: Molitor, L.; Krispin, S.; Van-Zuiden, W.; Danino, Y. M.; Rudberg, N.; Bar, C.; Amzallag, E.; Lubliner, J.; Siany, A.; Eitan, C.; Cohen, Y.; Yacovzada, N. S.; Hornstein, E.
Score: 22.3, Published: 2024-01-31 DOI: 10.1101/2024.01.31.572110
Systematic assessment of organelle architectures in cells, known as the organellome, could provide valuable insights into cellular states and disease pathologies but remains largely uncharted. Here, we devised a novel pipeline combining self-supervised deep learning and transfer learning to generate a Neuronal Organellomics Vision Atlas (NOVA).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Organellomics: AI-driven deep organellar phenotyping of human neurons
Authors: Molitor, L.; Krispin, S.; Van-Zuiden, W.; Danino, Y. M.; Rudberg, N.; Bar, C.; Amzallag, E.; Lubliner, J.; Siany, A.; Eitan, C.; Cohen, Y.; Yacovzada, N. S.; Hornstein, E.
Score: 22.3, Published: 2024-01-31 DOI: 10.1101/2024.01.31.572110
Systematic assessment of organelle architectures in cells, known as the organellome, could provide valuable insights into cellular states and disease pathologies but remains largely uncharted. Here, we devised a novel pipeline combining self-supervised deep learning and transfer learning to generate a Neuronal Organellomics Vision Atlas (NOVA)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-07T10:40:15+00:00" />
<meta property="article:modified_time" content="2024-02-07T10:40:15+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Organellomics: AI-driven deep organellar phenotyping of human neurons
Authors: Molitor, L.; Krispin, S.; Van-Zuiden, W.; Danino, Y. M.; Rudberg, N.; Bar, C.; Amzallag, E.; Lubliner, J.; Siany, A.; Eitan, C.; Cohen, Y.; Yacovzada, N. S.; Hornstein, E.
Score: 22.3, Published: 2024-01-31 DOI: 10.1101/2024.01.31.572110
Systematic assessment of organelle architectures in cells, known as the organellome, could provide valuable insights into cellular states and disease pathologies but remains largely uncharted. Here, we devised a novel pipeline combining self-supervised deep learning and transfer learning to generate a Neuronal Organellomics Vision Atlas (NOVA)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Organellomics: AI-driven deep organellar phenotyping of human neurons\nAuthors: Molitor, L.; Krispin, S.; Van-Zuiden, W.; Danino, Y. M.; Rudberg, N.; Bar, C.; Amzallag, E.; Lubliner, J.; Siany, A.; Eitan, C.; Cohen, Y.; Yacovzada, N. S.; Hornstein, E.\nScore: 22.3, Published: 2024-01-31 DOI: 10.1101/2024.01.31.572110\nSystematic assessment of organelle architectures in cells, known as the organellome, could provide valuable insights into cellular states and disease pathologies but remains largely uncharted. Here, we devised a novel pipeline combining self-supervised deep learning and transfer learning to generate a Neuronal Organellomics Vision Atlas (NOVA).",
  "keywords": [
    
  ],
  "articleBody": " Organellomics: AI-driven deep organellar phenotyping of human neurons\nAuthors: Molitor, L.; Krispin, S.; Van-Zuiden, W.; Danino, Y. M.; Rudberg, N.; Bar, C.; Amzallag, E.; Lubliner, J.; Siany, A.; Eitan, C.; Cohen, Y.; Yacovzada, N. S.; Hornstein, E.\nScore: 22.3, Published: 2024-01-31 DOI: 10.1101/2024.01.31.572110\nSystematic assessment of organelle architectures in cells, known as the organellome, could provide valuable insights into cellular states and disease pathologies but remains largely uncharted. Here, we devised a novel pipeline combining self-supervised deep learning and transfer learning to generate a Neuronal Organellomics Vision Atlas (NOVA). Analyzing over 1.5 million confocal images of 24 distinct membrane-bound and membrane-less organelles in human neurons, we enable a simultaneous evaluation of all organelles. We show that organellomics allows the study of cellular phenotypes by quantifying the localization and morphological properties embodied in multiple different organelles, using a unified score. We further developed a strategy to superimpose all organelles, which represents a new realization of cellular state. The value of our approach is demonstrated by characterizing specific organellar responses of human neurons to stress, cytoplasmic mislocalization of TDP-43, or disease-associated variations in ALS genes. Therefore, organellomics offers a novel approach to study the neuro-cellular biology of diseases. HighlightsAI-driven organellomics without cell segmentation or multiplexed imaging. Analysis of 24 membrane-bound and membrane-less organelles in more than 1.5 million images of human neurons. Quantitative organelle-level description of neuronal response to chemical and genetic perturbations. Organelles ranked on a single metric scale and integrated organellome view via superposition of multiple organelles.\nLarge-Scale Evaluation of Spatial Metabolomics Protocols and Technologies\nAuthors: Saharuka, V.; Vieira, L. M.; Stuart, L.; Ekelöf, M.; Molenaar, M. R.; Bailoni, A.; Ovchinnikova, K.; Soltwisch, J.; Bausbacher, T.; Jakob, D.; King, M.; Müller, M. A.; Oetjen, J.; Pace, C.; Pinto, F. E.; Strittmatter, N.; Velickovic, D.; Spengler, B.; Muddiman, D. C.; Liebeke, M.; Janfelt, C.; Goodwin, R.; Eberlin, L. S.; Anderton, C. R.; Hopf, C.; Dreisewerd, K.; Alexandrov, T.\nScore: 19.9, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577354\nSpatial metabolomics using imaging mass spectrometry (MS) enables untargeted and label-free metabolite mapping in biological samples. Despite the range of available imaging MS protocols and technologies, our understanding of metabolite detection under specific conditions is limited due to sparse empirical data and predictive theories. Consequently, challenges persist in designing new experiments, and accurately annotating and interpreting data. In this study, we systematically measured the detectability of 172 biologically-relevant metabolites across common imaging MS protocols using custom reference samples. We evaluated 24 MALDI-imaging MS protocols for untargeted metabolomics, and demonstrated the applicability of our findings to complex biological samples through comparison with animal tissue data. We showcased the potential for extending our results to further analytes by predicting metabolite detectability based on molecular properties. Additionally, our interlaboratory comparison of 10 imaging MS technologies, including MALDI, DESI, and IR-MALDESI, showed extensive metabolite coverage and comparable results, underscoring the broad applicability of our findings within the imaging MS community. We share our results and data through a new interactive web application integrated with METASPACE. This resource offers an extensive catalogue of detectable metabolite ions, facilitating protocol selection, supporting data annotation, and benefiting future untargeted spatial metabolomics studies.\nOrgan Boundary Circuits Regulate Sox9+ Alveolar Tuft Cells During Post-Pneumonectomy Lung Regeneration\nAuthors: Obata, T.; Mizoguchi, S.; Greaney, A. M.; Adams, T. S.; Yuan, Y.; Edelstein, S.; Leiby, K. L.; Rivero, R.; Wang, N.; Kim, H.; Yang, J.; Schupp, J. C.; Stitelman, D.; Tsuchiya, T.; Levchenko, A.; Kaminski, N.; Niklason, L. E.; Raredon, M. S. B.\nScore: 13.0, Published: 2024-01-08 DOI: 10.1101/2024.01.07.574469\nTissue homeostasis is controlled by cellular circuits governing cell growth, organization, and differentation. In this study we identify previously undescribed cell-to-cell communication that mediates information flow from mechanosensitive pleural mesothelial cells to alveolar-resident stem-like tuft cells in the lung. We find mesothelial cells to express a combination of mechanotransduction genes and lineage-restricted ligands which makes them uniquely capable of responding to tissue tension and producing paracrine cues acting on parenchymal populations. In parallel, we describe a large population of stem-like alveolar tuft cells that express the endodermal stem cell markers Sox9 and Lgr5 and a receptor profile making them uniquely sensitive to cues produced by pleural Mesothelium. We hypothesized that crosstalk from mesothelial cells to alveolar tuft cells might be central to the regulation of post-penumonectomy lung regeneration. Following pneumonectomy, we find that mesothelial cells display radically altered phenotype and ligand expression, in a pattern that closely tracks with parenchymal epithelial proliferation and alveolar tissue growth. During an initial pro-inflammatory stage of tissue regeneration, Mesothelium promotes epithelial proliferation via WNT ligand secretion, orchestrates an increase in microvascular permeability, and encourages immune extravasation via chemokine secretion. This stage is followed first by a tissue remodeling period, characterized by angiogenesis and BMP pathway sensitization, and then a stable return to homeostasis. Coupled with key changes in parenchymal structure and matrix production, the cumulative effect is a now larger organ including newly-grown, fully-functional tissue parenchyma. This study paints Mesothelial cells as a key orchestrating cell type that defines the boundary of the lung and exerts critical influence over the tissue-level signaling state regulating resident stem cell populations. The cellular circuits unearthed here suggest that human lung regeneration might be inducible through well-engineered approaches targeting the induction of tissue regeneration and safe return to homeostasis.\nA constraint-based framework to reconstruct interaction networks in microbial communities\nAuthors: Oftadeh, O.; Sahin, A.; Vayena, E.; Hatzimanikatis, V.\nScore: 7.9, Published: 2024-01-31 DOI: 10.1101/2024.01.30.577913\nMicrobial communities live in diverse habitats and significantly impact our health and the environment. However, the principles that govern their formation and evolution remain poorly understood. A crucial step in studying microbial communities is to identify the potential metabolic interactions between the community members, such as competition for nutrients or cross-feeding. Due to the size and complexity of the metabolic network of each organism, there may be a variety of connections between each pair of organisms, which poses a challenge to unraveling the metabolic interactions. Here, we present ReMIND, a computational framework to reconstruct the interaction networks in microbial communities based on the metabolic capabilities of individual organisms. We applied ReMIND to a well-studied uranium-reducing community and the honeybee gut microbiome. Our results provide new perspectives on the evolutionary forces that shape these ecosystems and the trade-off between metabolite exchange and biomass yield. By enumerating alternative interaction networks, we systematically identified the most likely metabolites to be exchanged and highlighted metabolites that could mediate competitive interactions. We envision that ReMIND will help characterize the metabolic capacity of individual members and elucidate metabolic interactions in diverse communities, thus holds the potential to guide many applications in precision medicine and synthetic ecology.\nPredicting microbial genome-scale metabolic networks directly from 16S rRNA gene sequences\nAuthors: Oyas, O.; Kobel, C. M.; Vik, J. O.; Pope, P. B.\nScore: 21.2, Published: 2024-01-27 DOI: 10.1101/2024.01.26.576649\nGenome-scale metabolic models are key biotechnology tools that can predict metabolic capabilities and growth for an organism. In particular, these models have become indispensable for metabolic analysis of microbial species and communities such as the gut microbiomes of humans and other animals. Accurate microbial models can be built automatically from genomes, but many microbes have only been observed through sequencing of marker genes such as 16S rRNA and thus remain inaccessible to genome-scale modeling. To extend the scope of genome-scale metabolic models to microbes that lack genomic information, we trained an artificial neural network to build microbial models from numeric representations of 16S rRNA gene sequences. Specifically, we built models and extracted 16S rRNA gene sequences from more than 15,000 reference and representative microbial genomes, computed multiple sequence alignments and large language model embeddings for the 16S rRNA gene sequences, and trained the neural network to predict metabolic reaction probabilities from sequences, alignments, or embeddings. Training was fast on a single graphics processing unit and trained networks predicted reaction probabilities accurately for unseen archaeal and bacterial sequences and species. This makes it possible to reconstruct microbial genome-scale metabolic networks from any 16S rRNA gene sequence and enables simulation of metabolism and growth for all observed microbial life.\nNonreciprocal synchronization in embryonic oscillator ensembles\nAuthors: Ho, C.; Jutras-Dube, L.; Zhao, M.; Mönke, G.; Kiss, I. Z.; Francois, P.; Aulehla, A.\nScore: 7.5, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577856\nSynchronization of coupled oscillators is a universal phenomenon encountered across different scales and contexts e.g., chemical wave patterns, superconductors and the unison applause we witness in concert halls. The existence of common underlying coupling rules define universality classes, revealing a fundamental sameness between seemingly distinct systems. Identifying rules of synchronization in any particular setting is hence of paramount relevance. Here, we address the coupling rules within an embryonic oscillator ensemble linked to vertebrate embryo body axis segmentation. In vertebrates, the periodic segmentation of the body axis involves synchronized signaling oscillations in cells within the presomitic mesoderm (PSM), from which somites, the pre-vertebrae, form. At the molecular level, it is known that intact Notch-signaling and cell-to-cell contact is required for synchronization between PSM cells. However, an understanding of the coupling rules is still lacking. To identify these, we develop a novel experimental assay that enables direct quantification of synchronization dynamics within mixtures of oscillating cell ensembles, for which the initial input frequency and phase distribution are known. Our results reveal a \"winner-takes-it-all\" synchronization outcome i.e., the emerging collective rhythm matches one of the input rhythms. Using a combination of theory and experimental validation, we develop a new coupling model, the \"Rectified Kuramoto\" (ReKu) model, characterized by a phase-dependent, non-reciprocal interaction in the coupling of oscillatory cells. Such non-reciprocal synchronization rules reveal fundamental similarities between embryonic oscillators and a class of collective behaviours seen in neurons and fireflies, where higher level computations are performed and linked to non-reciprocal synchronization.\nCombining systems and synthetic biology for in vivo enzymology\nAuthors: Castano-Cerezo, S.; Chamas, A.; Kulyk, H.; Treitz, C.; Bellvert, F.; Tholey, A.; Galeote, V.; Camarasa, C.; Heux, S.; Garcia-Alles, L. F.; Millard, P.; Truan, G.\nScore: 5.1, Published: 2024-02-04 DOI: 10.1101/2024.02.02.578620\nEnzymatic parameters are classically determined in vitro, under reaction conditions that are far from those encountered in cells, casting doubt on their physiological relevance. In this study, we developed a generic approach combining tools from synthetic and systems biology to measure enzymatic parameters in vivo. In the context of a synthetic pathway aiming to produce phytoene and lycopene in Saccharomyces cerevisiae, we focused on four membrane-associated enzymes, a phytoene synthase and three phytoene desaturases, which are difficult to study in vitro. We designed, built, and analyzed a collection of yeast strains mimicking substantial variations in substrate concentration (\u003e 200-fold change) by strategically manipulating the copy number and expression strength of the gene encoding geranyl-geranyl pyrophosphate (GGPP) synthase. We successfully determined in vivo-equivalent Michaelis-Menten parameters (KM, Vmax and kcat) for GGPP-converting phytoene synthase from quantitative metabolomics, fluxomics and proteomics data, highlighting differences between in vivo and in vitro parameters. Then, leveraging the versatility of the same set of strains to cover a wide range of phytoene concentrations, we successfully extracted enzymatic parameters for two of the three phytoene desaturases. Our approach demonstrates the feasibility of assessing enzymatic parameters along a pathway of interest directly in vivo, providing a novel perspective on the kinetic characteristics of enzymes in real cellular conditions.\nOptimal Performance Objectives in the Highly Conserved Bone Morphogenetic Protein Signaling Pathway\nAuthors: Shaikh, R.; Larson, N. J.; Hanjaya-Putra, D.; Zartman, J. J.; Umulis, D. M.; Li, L.; Reeves, G. T.\nScore: 4.0, Published: 2024-02-05 DOI: 10.1101/2024.02.01.578451\nThroughout development, complex networks of cell signaling pathways drive cellular decision-making across different tissues and contexts. The transforming growth factor {beta} (TGF- {beta}) pathways, including the BMP/Smad pathway, play crucial roles in these cellular responses. However, as the Smad pathway is used reiteratively throughout the life cycle of all animals, its systems-level behavior varies from one context to another, despite the pathway connectivity remaining nearly constant. For instance, some cellular systems require a rapid response, while others require high noise filtering. In this paper, we examine how the BMP-Smad pathway balances trade-offs among three such systems-level behaviors, or '' Performance Objectives (POs) '': response speed, noise amplification, and the sensitivity of pathway output to receptor input. Using a Smad pathway model fit to human cell data, we show that varying non-conserved parameters (NCPs) such as protein concentrations, the Smad pathway can be tuned to emphasize any of the three POs and that the concentration of nuclear phosphatase has the greatest effect on tuning the POs. However, due to competition among the POs, the pathway cannot simultaneously optimize all three, but at best must balance trade-offs among the POs. We applied the multi-objective optimization concept of the Pareto Front, a widely used concept in economics to identify optimal trade-offs among various requirements. We show that the BMP pathway efficiently balances competing POs across species and is largely Pareto optimal. Our findings reveal that varying the concentration of NCPs allows the Smad signaling pathway to generate a diverse range of POs. This insight identifies how signaling pathways can be optimally tuned for each context.\nUnraveling the oscillatory dynamics of mRNA metabolism and chromatin accessibility during the cell cycle through integration of single-cell multiomic data\nAuthors: Nariya, M. K.; Santiago-Algarra, D.; Tassy, O.; Cerciat, M.; Ye, T.; Riba, A.; Molina, N.\nScore: 16.8, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575159\nThe cell cycle is a highly regulated process that ensures the accurate replication and transmission of genetic information from one generation of cells to the next. It is a fundamental biological process and plays a crucial role in development, growth, and maintenance of all living organisms, and its dysregulation can lead to a number of pathologies such as autoimmune diseases, neurodegenerative diseases, and cancer. In this work we present a novel approach to study the gene expression and chromatin accessibility dynamics during the cell cycle in mouse embryonic stem cells. To achieve this, we combined high-depth single-cell multiome sequencing, biophysical modeling, and advanced deep learning techniques. First, we used DeepCycle, a deep learning tool that assigns a cell cycle phase to every cell based on its spliced and unspliced mRNA levels. We then developed a biophysical model that describes the dynamics of gene-specific mRNA transcription, nuclear export, and degradation rates during the cell cycle. Our model obtains genome-wide estimates of mRNA transcription, nuclear retention, and degradation half-lives of genes exhibiting oscillatory dynamics in mESCs. The key feature of single-cell multiome sequencing is that it simultaneously provides readouts for gene expression as well as chromatin accessibility in the same cells. By applying our approach to these data we showcase first-of-its-kind exploration of chromatin accessibility during the cell cycle at high temporal resolution. We believe that our work will be pivotal in formulating a coherent quantitative description of mRNA metabolism. HighlightsO_LIIntegration of single-cell and single-nucleus multiome profiling with deep learning and biophysical modeling uncovers cell cycle regulation of RNA metabolism C_LIO_LIGenome-wide estimation of mRNA transcription, nuclear export, and degradation rates during the cell cycle in mESCs C_LIO_LIIdentification of a prominent wave of transcription during early G1 in unperturbed asynchronously proliferating mESCs C_LIO_LICharacterization of temporal waves of post-transcriptional mRNA regulation controlling both mRNA degradation and nuclear export during the cell cycle C_LIO_LIChromatin accessibility dynamics within TF footprints reveal key cell cycle regulators C_LI\nMultistability and predominant double-positive states in a four node mutually repressive network: a case study of Th1/Th2/Th17/T-reg differentiation\nAuthors: Duddu, A. S.; Andreas, E.; BV, H.; Grover, K.; Singh, V. R.; Hari, K.; Jhunjhunwala, S.; Cummins, B.; Gedeon, T.; Jolly, M. K.\nScore: 2.6, Published: 2024-02-02 DOI: 10.1101/2024.01.30.575880\nElucidating the emergent dynamics of complex regulatory networks enabling cellular differentiation is crucial to understand embryonic development and suggest strategies for synthetic circuit design. A well-studied network motif often driving cellular decisions is a toggle switch - a set of two mutually inhibitory lineage-specific transcription factors A and B. A toggle switch often enables two possible mutually exclusive states - (high A, low B) and (low A, high B) - from a common progenitor cell. However, the dynamics of networks enabling differentiation of more than two cell types from a progenitor cell is not well-studied. Here, we investigate the dynamics of four master regulators A, B, C and D inhibiting each other, thus forming a toggle tetrahedron. Our simulations show that a toggle tetrahedron predominantly allows for co-existence of six double positive or hybrid states where two of the nodes are expressed relatively high as compared to the remaining two - (high A, high B, low C, low D), (high A, low B, high C, low D), (high A, low B, low C, high D), (low A, high B, high C, low D), (low A, low B, high C, high D) and (low A, high B, low C, high D). Stochastic simulations showed state-switching among these phenotypes, indicating phenotypic plasticity. Finally, we apply our results to understand the differentiation of naive CD4+ T cells into Th1, Th2, Th17 and Treg subsets, suggesting Th1/Th2/Th17/Treg decision-making to be a two-step process. Our results reveal multistable dynamics and establish the stable co-existence of hybrid cell-states, offering a potential explanation for simultaneous differentiation of multipotent naive CD4+ T cells.\n",
  "wordCount" : "2830",
  "inLanguage": "en",
  "datePublished": "2024-02-07T10:40:15Z",
  "dateModified": "2024-02-07T10:40:15Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.572110">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.572110" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.572110">
        <p class="paperTitle">Organellomics: AI-driven deep organellar phenotyping of human neurons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.572110" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.572110" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Molitor, L.; Krispin, S.; Van-Zuiden, W.; Danino, Y. M.; Rudberg, N.; Bar, C.; Amzallag, E.; Lubliner, J.; Siany, A.; Eitan, C.; Cohen, Y.; Yacovzada, N. S.; Hornstein, E.</p>
        <p class="info">Score: 22.3, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.572110' target='https://doi.org/10.1101/2024.01.31.572110'> 10.1101/2024.01.31.572110</a></p>
        <p class="abstract">Systematic assessment of organelle architectures in cells, known as the organellome, could provide valuable insights into cellular states and disease pathologies but remains largely uncharted. Here, we devised a novel pipeline combining self-supervised deep learning and transfer learning to generate a Neuronal Organellomics Vision Atlas (NOVA). Analyzing over 1.5 million confocal images of 24 distinct membrane-bound and membrane-less organelles in human neurons, we enable a simultaneous evaluation of all organelles. We show that organellomics allows the study of cellular phenotypes by quantifying the localization and morphological properties embodied in multiple different organelles, using a unified score. We further developed a strategy to superimpose all organelles, which represents a new realization of cellular state. The value of our approach is demonstrated by characterizing specific organellar responses of human neurons to stress, cytoplasmic mislocalization of TDP-43, or disease-associated variations in ALS genes. Therefore, organellomics offers a novel approach to study the neuro-cellular biology of diseases.

HighlightsAI-driven organellomics without cell segmentation or multiplexed imaging.

Analysis of 24 membrane-bound and membrane-less organelles in more than 1.5 million images of human neurons.

Quantitative organelle-level description of neuronal response to chemical and genetic perturbations.

Organelles ranked on a single metric scale and integrated organellome view via superposition of multiple organelles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577354" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577354">
        <p class="paperTitle">Large-Scale Evaluation of Spatial Metabolomics Protocols and Technologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saharuka, V.; Vieira, L. M.; Stuart, L.; Ekelöf, M.; Molenaar, M. R.; Bailoni, A.; Ovchinnikova, K.; Soltwisch, J.; Bausbacher, T.; Jakob, D.; King, M.; Müller, M. A.; Oetjen, J.; Pace, C.; Pinto, F. E.; Strittmatter, N.; Velickovic, D.; Spengler, B.; Muddiman, D. C.; Liebeke, M.; Janfelt, C.; Goodwin, R.; Eberlin, L. S.; Anderton, C. R.; Hopf, C.; Dreisewerd, K.; Alexandrov, T.</p>
        <p class="info">Score: 19.9, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577354' target='https://doi.org/10.1101/2024.01.29.577354'> 10.1101/2024.01.29.577354</a></p>
        <p class="abstract">Spatial metabolomics using imaging mass spectrometry (MS) enables untargeted and label-free metabolite mapping in biological samples. Despite the range of available imaging MS protocols and technologies, our understanding of metabolite detection under specific conditions is limited due to sparse empirical data and predictive theories. Consequently, challenges persist in designing new experiments, and accurately annotating and interpreting data. In this study, we systematically measured the detectability of 172 biologically-relevant metabolites across common imaging MS protocols using custom reference samples. We evaluated 24 MALDI-imaging MS protocols for untargeted metabolomics, and demonstrated the applicability of our findings to complex biological samples through comparison with animal tissue data. We showcased the potential for extending our results to further analytes by predicting metabolite detectability based on molecular properties. Additionally, our interlaboratory comparison of 10 imaging MS technologies, including MALDI, DESI, and IR-MALDESI, showed extensive metabolite coverage and comparable results, underscoring the broad applicability of our findings within the imaging MS community. We share our results and data through a new interactive web application integrated with METASPACE. This resource offers an extensive catalogue of detectable metabolite ions, facilitating protocol selection, supporting data annotation, and benefiting future untargeted spatial metabolomics studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.574469">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.574469" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.574469">
        <p class="paperTitle">Organ Boundary Circuits Regulate Sox9&#43; Alveolar Tuft Cells During Post-Pneumonectomy Lung Regeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.574469" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.574469" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Obata, T.; Mizoguchi, S.; Greaney, A. M.; Adams, T. S.; Yuan, Y.; Edelstein, S.; Leiby, K. L.; Rivero, R.; Wang, N.; Kim, H.; Yang, J.; Schupp, J. C.; Stitelman, D.; Tsuchiya, T.; Levchenko, A.; Kaminski, N.; Niklason, L. E.; Raredon, M. S. B.</p>
        <p class="info">Score: 13.0, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.574469' target='https://doi.org/10.1101/2024.01.07.574469'> 10.1101/2024.01.07.574469</a></p>
        <p class="abstract">Tissue homeostasis is controlled by cellular circuits governing cell growth, organization, and differentation. In this study we identify previously undescribed cell-to-cell communication that mediates information flow from mechanosensitive pleural mesothelial cells to alveolar-resident stem-like tuft cells in the lung. We find mesothelial cells to express a combination of mechanotransduction genes and lineage-restricted ligands which makes them uniquely capable of responding to tissue tension and producing paracrine cues acting on parenchymal populations. In parallel, we describe a large population of stem-like alveolar tuft cells that express the endodermal stem cell markers Sox9 and Lgr5 and a receptor profile making them uniquely sensitive to cues produced by pleural Mesothelium. We hypothesized that crosstalk from mesothelial cells to alveolar tuft cells might be central to the regulation of post-penumonectomy lung regeneration. Following pneumonectomy, we find that mesothelial cells display radically altered phenotype and ligand expression, in a pattern that closely tracks with parenchymal epithelial proliferation and alveolar tissue growth. During an initial pro-inflammatory stage of tissue regeneration, Mesothelium promotes epithelial proliferation via WNT ligand secretion, orchestrates an increase in microvascular permeability, and encourages immune extravasation via chemokine secretion. This stage is followed first by a tissue remodeling period, characterized by angiogenesis and BMP pathway sensitization, and then a stable return to homeostasis. Coupled with key changes in parenchymal structure and matrix production, the cumulative effect is a now larger organ including newly-grown, fully-functional tissue parenchyma. This study paints Mesothelial cells as a key orchestrating cell type that defines the boundary of the lung and exerts critical influence over the tissue-level signaling state regulating resident stem cell populations. The cellular circuits unearthed here suggest that human lung regeneration might be inducible through well-engineered approaches targeting the induction of tissue regeneration and safe return to homeostasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577913">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577913" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577913">
        <p class="paperTitle">A constraint-based framework to reconstruct interaction networks in microbial communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577913" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577913" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oftadeh, O.; Sahin, A.; Vayena, E.; Hatzimanikatis, V.</p>
        <p class="info">Score: 7.9, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577913' target='https://doi.org/10.1101/2024.01.30.577913'> 10.1101/2024.01.30.577913</a></p>
        <p class="abstract">Microbial communities live in diverse habitats and significantly impact our health and the environment. However, the principles that govern their formation and evolution remain poorly understood. A crucial step in studying microbial communities is to identify the potential metabolic interactions between the community members, such as competition for nutrients or cross-feeding. Due to the size and complexity of the metabolic network of each organism, there may be a variety of connections between each pair of organisms, which poses a challenge to unraveling the metabolic interactions. Here, we present ReMIND, a computational framework to reconstruct the interaction networks in microbial communities based on the metabolic capabilities of individual organisms. We applied ReMIND to a well-studied uranium-reducing community and the honeybee gut microbiome. Our results provide new perspectives on the evolutionary forces that shape these ecosystems and the trade-off between metabolite exchange and biomass yield. By enumerating alternative interaction networks, we systematically identified the most likely metabolites to be exchanged and highlighted metabolites that could mediate competitive interactions. We envision that ReMIND will help characterize the metabolic capacity of individual members and elucidate metabolic interactions in diverse communities, thus holds the potential to guide many applications in precision medicine and synthetic ecology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.576649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.576649" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.576649">
        <p class="paperTitle">Predicting microbial genome-scale metabolic networks directly from 16S rRNA gene sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.576649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.576649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oyas, O.; Kobel, C. M.; Vik, J. O.; Pope, P. B.</p>
        <p class="info">Score: 21.2, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.576649' target='https://doi.org/10.1101/2024.01.26.576649'> 10.1101/2024.01.26.576649</a></p>
        <p class="abstract">Genome-scale metabolic models are key biotechnology tools that can predict metabolic capabilities and growth for an organism. In particular, these models have become indispensable for metabolic analysis of microbial species and communities such as the gut microbiomes of humans and other animals. Accurate microbial models can be built automatically from genomes, but many microbes have only been observed through sequencing of marker genes such as 16S rRNA and thus remain inaccessible to genome-scale modeling. To extend the scope of genome-scale metabolic models to microbes that lack genomic information, we trained an artificial neural network to build microbial models from numeric representations of 16S rRNA gene sequences. Specifically, we built models and extracted 16S rRNA gene sequences from more than 15,000 reference and representative microbial genomes, computed multiple sequence alignments and large language model embeddings for the 16S rRNA gene sequences, and trained the neural network to predict metabolic reaction probabilities from sequences, alignments, or embeddings. Training was fast on a single graphics processing unit and trained networks predicted reaction probabilities accurately for unseen archaeal and bacterial sequences and species. This makes it possible to reconstruct microbial genome-scale metabolic networks from any 16S rRNA gene sequence and enables simulation of metabolism and growth for all observed microbial life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577856">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577856" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577856">
        <p class="paperTitle">Nonreciprocal synchronization in embryonic oscillator ensembles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577856" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577856" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ho, C.; Jutras-Dube, L.; Zhao, M.; Mönke, G.; Kiss, I. Z.; Francois, P.; Aulehla, A.</p>
        <p class="info">Score: 7.5, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577856' target='https://doi.org/10.1101/2024.01.29.577856'> 10.1101/2024.01.29.577856</a></p>
        <p class="abstract">Synchronization of coupled oscillators is a universal phenomenon encountered across different scales and contexts e.g., chemical wave patterns, superconductors and the unison applause we witness in concert halls. The existence of common underlying coupling rules define universality classes, revealing a fundamental sameness between seemingly distinct systems. Identifying rules of synchronization in any particular setting is hence of paramount relevance. Here, we address the coupling rules within an embryonic oscillator ensemble linked to vertebrate embryo body axis segmentation. In vertebrates, the periodic segmentation of the body axis involves synchronized signaling oscillations in cells within the presomitic mesoderm (PSM), from which somites, the pre-vertebrae, form. At the molecular level, it is known that intact Notch-signaling and cell-to-cell contact is required for synchronization between PSM cells. However, an understanding of the coupling rules is still lacking. To identify these, we develop a novel experimental assay that enables direct quantification of synchronization dynamics within mixtures of oscillating cell ensembles, for which the initial input frequency and phase distribution are known. Our results reveal a &#34;winner-takes-it-all&#34; synchronization outcome i.e., the emerging collective rhythm matches one of the input rhythms. Using a combination of theory and experimental validation, we develop a new coupling model, the &#34;Rectified Kuramoto&#34; (ReKu) model, characterized by a phase-dependent, non-reciprocal interaction in the coupling of oscillatory cells. Such non-reciprocal synchronization rules reveal fundamental similarities between embryonic oscillators and a class of collective behaviours seen in neurons and fireflies, where higher level computations are performed and linked to non-reciprocal synchronization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578620">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578620" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578620">
        <p class="paperTitle">Combining systems and synthetic biology for in vivo enzymology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578620" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578620" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Castano-Cerezo, S.; Chamas, A.; Kulyk, H.; Treitz, C.; Bellvert, F.; Tholey, A.; Galeote, V.; Camarasa, C.; Heux, S.; Garcia-Alles, L. F.; Millard, P.; Truan, G.</p>
        <p class="info">Score: 5.1, Published: 2024-02-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578620' target='https://doi.org/10.1101/2024.02.02.578620'> 10.1101/2024.02.02.578620</a></p>
        <p class="abstract">Enzymatic parameters are classically determined in vitro, under reaction conditions that are far from those encountered in cells, casting doubt on their physiological relevance. In this study, we developed a generic approach combining tools from synthetic and systems biology to measure enzymatic parameters in vivo. In the context of a synthetic pathway aiming to produce phytoene and lycopene in Saccharomyces cerevisiae, we focused on four membrane-associated enzymes, a phytoene synthase and three phytoene desaturases, which are difficult to study in vitro. We designed, built, and analyzed a collection of yeast strains mimicking substantial variations in substrate concentration (&gt; 200-fold change) by strategically manipulating the copy number and expression strength of the gene encoding geranyl-geranyl pyrophosphate (GGPP) synthase. We successfully determined in vivo-equivalent Michaelis-Menten parameters (KM, Vmax and kcat) for GGPP-converting phytoene synthase from quantitative metabolomics, fluxomics and proteomics data, highlighting differences between in vivo and in vitro parameters. Then, leveraging the versatility of the same set of strains to cover a wide range of phytoene concentrations, we successfully extracted enzymatic parameters for two of the three phytoene desaturases. Our approach demonstrates the feasibility of assessing enzymatic parameters along a pathway of interest directly in vivo, providing a novel perspective on the kinetic characteristics of enzymes in real cellular conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578451">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578451" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578451">
        <p class="paperTitle">Optimal Performance Objectives in the Highly Conserved Bone Morphogenetic Protein Signaling Pathway</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578451" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578451" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shaikh, R.; Larson, N. J.; Hanjaya-Putra, D.; Zartman, J. J.; Umulis, D. M.; Li, L.; Reeves, G. T.</p>
        <p class="info">Score: 4.0, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578451' target='https://doi.org/10.1101/2024.02.01.578451'> 10.1101/2024.02.01.578451</a></p>
        <p class="abstract">Throughout development, complex networks of cell signaling pathways drive cellular decision-making across different tissues and contexts. The transforming growth factor {beta} (TGF- {beta}) pathways, including the BMP/Smad pathway, play crucial roles in these cellular responses. However, as the Smad pathway is used reiteratively throughout the life cycle of all animals, its systems-level behavior varies from one context to another, despite the pathway connectivity remaining nearly constant. For instance, some cellular systems require a rapid response, while others require high noise filtering. In this paper, we examine how the BMP-Smad pathway balances trade-offs among three such systems-level behaviors, or &#39;&#39; Performance Objectives (POs) &#39;&#39;: response speed, noise amplification, and the sensitivity of pathway output to receptor input. Using a Smad pathway model fit to human cell data, we show that varying non-conserved parameters (NCPs) such as protein concentrations, the Smad pathway can be tuned to emphasize any of the three POs and that the concentration of nuclear phosphatase has the greatest effect on tuning the POs. However, due to competition among the POs, the pathway cannot simultaneously optimize all three, but at best must balance trade-offs among the POs. We applied the multi-objective optimization concept of the Pareto Front, a widely used concept in economics to identify optimal trade-offs among various requirements. We show that the BMP pathway efficiently balances competing POs across species and is largely Pareto optimal. Our findings reveal that varying the concentration of NCPs allows the Smad signaling pathway to generate a diverse range of POs. This insight identifies how signaling pathways can be optimally tuned for each context.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575159">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575159" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575159">
        <p class="paperTitle">Unraveling the oscillatory dynamics of mRNA metabolism and chromatin accessibility during the cell cycle through integration of single-cell multiomic data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575159" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575159" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nariya, M. K.; Santiago-Algarra, D.; Tassy, O.; Cerciat, M.; Ye, T.; Riba, A.; Molina, N.</p>
        <p class="info">Score: 16.8, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575159' target='https://doi.org/10.1101/2024.01.11.575159'> 10.1101/2024.01.11.575159</a></p>
        <p class="abstract">The cell cycle is a highly regulated process that ensures the accurate replication and transmission of genetic information from one generation of cells to the next. It is a fundamental biological process and plays a crucial role in development, growth, and maintenance of all living organisms, and its dysregulation can lead to a number of pathologies such as autoimmune diseases, neurodegenerative diseases, and cancer. In this work we present a novel approach to study the gene expression and chromatin accessibility dynamics during the cell cycle in mouse embryonic stem cells. To achieve this, we combined high-depth single-cell multiome sequencing, biophysical modeling, and advanced deep learning techniques. First, we used DeepCycle, a deep learning tool that assigns a cell cycle phase to every cell based on its spliced and unspliced mRNA levels. We then developed a biophysical model that describes the dynamics of gene-specific mRNA transcription, nuclear export, and degradation rates during the cell cycle. Our model obtains genome-wide estimates of mRNA transcription, nuclear retention, and degradation half-lives of genes exhibiting oscillatory dynamics in mESCs. The key feature of single-cell multiome sequencing is that it simultaneously provides readouts for gene expression as well as chromatin accessibility in the same cells. By applying our approach to these data we showcase first-of-its-kind exploration of chromatin accessibility during the cell cycle at high temporal resolution. We believe that our work will be pivotal in formulating a coherent quantitative description of mRNA metabolism.

HighlightsO_LIIntegration of single-cell and single-nucleus multiome profiling with deep learning and biophysical modeling uncovers cell cycle regulation of RNA metabolism
C_LIO_LIGenome-wide estimation of mRNA transcription, nuclear export, and degradation rates during the cell cycle in mESCs
C_LIO_LIIdentification of a prominent wave of transcription during early G1 in unperturbed asynchronously proliferating mESCs
C_LIO_LICharacterization of temporal waves of post-transcriptional mRNA regulation controlling both mRNA degradation and nuclear export during the cell cycle
C_LIO_LIChromatin accessibility dynamics within TF footprints reveal key cell cycle regulators
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.575880">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.575880" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.575880">
        <p class="paperTitle">Multistability and predominant double-positive states in a four node mutually repressive network: a case study of Th1/Th2/Th17/T-reg differentiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.575880" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.575880" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duddu, A. S.; Andreas, E.; BV, H.; Grover, K.; Singh, V. R.; Hari, K.; Jhunjhunwala, S.; Cummins, B.; Gedeon, T.; Jolly, M. K.</p>
        <p class="info">Score: 2.6, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.575880' target='https://doi.org/10.1101/2024.01.30.575880'> 10.1101/2024.01.30.575880</a></p>
        <p class="abstract">Elucidating the emergent dynamics of complex regulatory networks enabling cellular differentiation is crucial to understand embryonic development and suggest strategies for synthetic circuit design. A well-studied network motif often driving cellular decisions is a toggle switch - a set of two mutually inhibitory lineage-specific transcription factors A and B. A toggle switch often enables two possible mutually exclusive states - (high A, low B) and (low A, high B) - from a common progenitor cell. However, the dynamics of networks enabling differentiation of more than two cell types from a progenitor cell is not well-studied. Here, we investigate the dynamics of four master regulators A, B, C and D inhibiting each other, thus forming a toggle tetrahedron. Our simulations show that a toggle tetrahedron predominantly allows for co-existence of six  double positive or hybrid states where two of the nodes are expressed relatively high as compared to the remaining two - (high A, high B, low C, low D), (high A, low B, high C, low D), (high A, low B, low C, high D), (low A, high B, high C, low D), (low A, low B, high C, high D) and (low A, high B, low C, high D). Stochastic simulations showed state-switching among these phenotypes, indicating phenotypic plasticity. Finally, we apply our results to understand the differentiation of naive CD4&#43; T cells into Th1, Th2, Th17 and Treg subsets, suggesting Th1/Th2/Th17/Treg decision-making to be a two-step process. Our results reveal multistable dynamics and establish the stable co-existence of hybrid cell-states, offering a potential explanation for simultaneous differentiation of multipotent naive CD4&#43; T cells.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
