<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://yorks0n.github.io/TRxiv2" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/TRxiv2/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Storm: Incorporating transient dynamics to infer the RNA velocity with metabolic labeling information
Authors: Peng, Q.; Qiu, X.; Li, T.
Score: 14.8, Published: 2023-06-23 DOI: 10.1101/2023.06.21.545990
AO_SCPLOWBSTRACTC_SCPLOWThe time-resolved scRNA-seq (tscRNA-seq) provides the possibility to infer physically meaningful kinetic parameters, e.g., the transcription, splicing or RNA degradation rate constants with correct magnitudes, and RNA velocities by incorporating temporal information. Previous approaches utilizing the deterministic dynamics and steady-state assumption on gene expression states are insufficient to achieve favorable results for the data involving transient process.">
<meta name="author" content="">
<link rel="canonical" href="https://yorks0n.github.io/TRxiv2/posts/systems-biology/">
<link crossorigin="anonymous" href="/TRxiv2/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/TRxiv2/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://yorks0n.github.io/TRxiv2/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://yorks0n.github.io/TRxiv2/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://yorks0n.github.io/TRxiv2/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://yorks0n.github.io/TRxiv2/apple-touch-icon.png">
<link rel="mask-icon" href="https://yorks0n.github.io/TRxiv2/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Storm: Incorporating transient dynamics to infer the RNA velocity with metabolic labeling information
Authors: Peng, Q.; Qiu, X.; Li, T.
Score: 14.8, Published: 2023-06-23 DOI: 10.1101/2023.06.21.545990
AO_SCPLOWBSTRACTC_SCPLOWThe time-resolved scRNA-seq (tscRNA-seq) provides the possibility to infer physically meaningful kinetic parameters, e.g., the transcription, splicing or RNA degradation rate constants with correct magnitudes, and RNA velocities by incorporating temporal information. Previous approaches utilizing the deterministic dynamics and steady-state assumption on gene expression states are insufficient to achieve favorable results for the data involving transient process." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://yorks0n.github.io/TRxiv2/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T09:51:07+00:00" />
<meta property="article:modified_time" content="2023-06-28T09:51:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Storm: Incorporating transient dynamics to infer the RNA velocity with metabolic labeling information
Authors: Peng, Q.; Qiu, X.; Li, T.
Score: 14.8, Published: 2023-06-23 DOI: 10.1101/2023.06.21.545990
AO_SCPLOWBSTRACTC_SCPLOWThe time-resolved scRNA-seq (tscRNA-seq) provides the possibility to infer physically meaningful kinetic parameters, e.g., the transcription, splicing or RNA degradation rate constants with correct magnitudes, and RNA velocities by incorporating temporal information. Previous approaches utilizing the deterministic dynamics and steady-state assumption on gene expression states are insufficient to achieve favorable results for the data involving transient process."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://yorks0n.github.io/TRxiv2/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://yorks0n.github.io/TRxiv2/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Storm: Incorporating transient dynamics to infer the RNA velocity with metabolic labeling information\nAuthors: Peng, Q.; Qiu, X.; Li, T.\nScore: 14.8, Published: 2023-06-23 DOI: 10.1101/2023.06.21.545990\nAO_SCPLOWBSTRACTC_SCPLOWThe time-resolved scRNA-seq (tscRNA-seq) provides the possibility to infer physically meaningful kinetic parameters, e.g., the transcription, splicing or RNA degradation rate constants with correct magnitudes, and RNA velocities by incorporating temporal information. Previous approaches utilizing the deterministic dynamics and steady-state assumption on gene expression states are insufficient to achieve favorable results for the data involving transient process.",
  "keywords": [
    
  ],
  "articleBody": " Storm: Incorporating transient dynamics to infer the RNA velocity with metabolic labeling information\nAuthors: Peng, Q.; Qiu, X.; Li, T.\nScore: 14.8, Published: 2023-06-23 DOI: 10.1101/2023.06.21.545990\nAO_SCPLOWBSTRACTC_SCPLOWThe time-resolved scRNA-seq (tscRNA-seq) provides the possibility to infer physically meaningful kinetic parameters, e.g., the transcription, splicing or RNA degradation rate constants with correct magnitudes, and RNA velocities by incorporating temporal information. Previous approaches utilizing the deterministic dynamics and steady-state assumption on gene expression states are insufficient to achieve favorable results for the data involving transient process. We present a dynamical approach, Storm (Stochastic models of RNA metabolic-labeling), to overcome these limitations by solving stochastic differential equations of gene expression dynamics. The derivation reveals that the new mRNA sequencing data obeys different types of cell-specific Poisson distributions when jointly considering both biological and cell-specific technical noise. Storm deals with measured counts data directly and extends the RNA velocity methodology based on metabolic labeling scRNA-seq data to transient stochastic systems. Furthermore, we relax the constant parameter assumption over genes/cells to obtain gene-cell-specific transcription/splicing rates and gene-specific degradation rates, thus revealing time-dependent and cell-state specific transcriptional regulations. Storm will facilitate the study of the statistical properties of tscRNA-seq data, eventually advancing our understanding of the dynamic transcription regulation during development and disease.\nDual proteomic signature of immune cells and Yersinia pestis upon blood infection\nAuthors: Le-Bury, P.; Douche, T.; Giai Gianetto, Q.; Matondo, M.; Pizarro-Cerda, J.; Dussurget, O.\nScore: 7.0, Published: 2023-06-22 DOI: 10.1101/2023.06.19.545537\nEmerging and reemerging infectious diseases represent major public health concerns. The urgent need for infection control measures requires deep understanding of molecular pathogenesis. Global approaches to study biological systems such as mass-spectrometry based proteomics benefited from groundbreaking physical and bioinformatical technological developments over recent years. However, dual proteomic study of highly pathogenic microorganisms and their hosts in complex matrices encountered during infection remains challenging due to high protein dynamic range of samples and requirements imposed in biosafety level 3 or 4 laboratories. Here, we constructed a dual proteomic pipeline of Yersinia pestis in human blood and plasma, mirroring bacteremic phase of plague. We provide the most complete Y. pestis proteome revealing a major reshaping of important bacterial path-ways such as methionine biosynthesis and iron acquisition in human plasma. Remarkably, proteomic profiling in human blood highlights a greater Yersinia outer proteins intoxication of monocytes than neutrophils. Our study unravels global expression changes and points to a specific pathogenic signature during infection, paving the way for future exploration of proteomes in the complex context of host-pathogen interactions. Subject CategoriesMicrobiology, Virology and Host Pathogen Interaction, Proteomics\nThe genetic and dietary landscape of the muscle insulin signalling network\nAuthors: van Gerwen, J.; Masson, S. W. C.; Cutler, H. B.; Diaz-Vegas, A.; Potter, M.; Stockli, J.; Madsen, S.; Nelson, M. E.; Humphrey, S. J.; James, D. E.\nScore: 6.3, Published: 2023-06-22 DOI: 10.1101/2023.06.21.546001\nMetabolic disease is caused by a combination of genetic and environmental factors, yet few studies have examined how these factors influence signal transduction, a key mediator of metabolism. Using mass spectrometry-based phosphoproteomics, we quantified 23,126 phosphosites in skeletal muscle of five genetically distinct mouse strains in two dietary environments, with and without acute in vivo insulin stimulation. Almost half of the insulin-regulated phosphoproteome was modified by genetic background on an ordinary diet, and high-fat high-sugar feeding affected insulin signalling in a strain-dependent manner. Our data revealed coregulated subnetworks within the insulin signalling pathway, expanding our understanding of the pathways organisation. Furthermore, associating diverse signalling responses with insulin-stimulated glucose uptake uncovered regulators of muscle insulin responsiveness, including the regulatory phosphosite S469 on Pfkfb2, a key activator of glycolysis. Finally, we confirmed the role of glycolysis in modulating insulin action in insulin resistance. Our results underscore the significance of genetics in shaping global signalling responses and their adaptability to environmental changes, emphasizing the utility of studying biological diversity with phosphoproteomics to discover key regulatory mechanisms of complex traits.\nQuantifying and correcting estimation bias in transcriptional parameter inference from single-cell data\nAuthors: Grima, R.; Esmenjaud, P.-M.\nScore: 3.1, Published: 2023-06-28 DOI: 10.1101/2023.06.19.545536\nThe snapshot distribution of mRNA counts per cell can be readily measured using single molecule FISH or single-cell RNA sequencing. These distributions are often fit to the steady-state distribution of the two-state telegraph model to estimate the three transcriptional parameters for a gene of interest: the rate of transcription, the rate of switching to the active transcriptional state and the rate of switching to the inactive state. These parameters are often understood as being reflective of the average cell in a population. However it is currently unclear to what extent does the cell-to-cell variation of parameters (extrinsic noise) affect the accuracy of parameter estimation. Here we develop an analytical approach to understand the factors determining the size and sign of estimation bias. We find bias signatures that depend on the source of extrinsic noise and the mode of transcriptional activity. We estimate the size of extrinsic noise from the covariance matrix of scRNA-seq data which we use to correct published estimates of parameters for mammalian genes.\nHeritable epigenetic changes are constrained by the dynamics of regulatory architectures\nAuthors: Jose, A. M.\nScore: 4.1, Published: 2023-06-09 DOI: 10.1101/2023.06.07.544138\nInteracting molecules create regulatory architectures that can persist despite turnover of molecules. Although epigenetic changes occur within the context of such architectures, there is limited understanding of how they can influence the heritability of changes. Here I develop criteria for the heritability of regulatory architectures and use quantitative simulations of interacting regulators parsed as entities, their sensors and the sensed properties to analyze how architectures influence heritable epigenetic changes. Information contained in regulatory architectures grows rapidly with the number of interacting molecules and its transmission requires positive feedback loops. While these architectures can recover after many epigenetic perturbations, some resulting changes can become permanently heritable. Such stable changes can (1) alter steady-state levels while preserving the architecture, (2) induce different architectures that persist for many generations, or (3) collapse the entire architecture. Architectures that are otherwise unstable can become heritable through periodic interactions with external regulators, which suggests that the evolution of mortal somatic lineages with cells that reproducibly interact with the immortal germ lineage could make a wider variety of regulatory architectures heritable. Differential inhibition of the positive feedback loops that transmit regulatory architectures across generations can explain the gene-specific differences in heritable RNA silencing observed in the nematode C. elegans, which range from permanent silencing, to recovery from silencing within a few generations and subsequent resistance to silencing. More broadly, these results provide a foundation for analyzing the inheritance of epigenetic changes within the context of the regulatory architectures implemented using diverse molecules in different living systems. SIGNIFICANCERegulatory interactions in living systems are recreated in successive generations. Practical ways of analyzing how the information required for this recreation is transmitted across generations and how it can be changed are lacking. Simulating all heritable information by parsing regulatory interactions in terms of entities, their sensors, and the sensed properties reveals the minimal requirements for the heritability of regulatory interactions and how they influence the inheritance of epigenetic changes. Application of this approach can explain recent experimental results on the inheritance of RNA silencing across generations in the nematode C. elegans. Since all interactors can be abstracted as entity-sensor-property systems, similar analyses can be widely used to understand heritable epigenetic changes.\nSPIDR: a highly multiplexed method for mapping RNA-protein interactions uncovers a potential mechanism for selective translational suppression upon cellular stress\nAuthors: Wolin, E.; Guo, J. K.; Blanco, M. R.; Perez, A. A.; Goronzy, I. N.; Abdou, A. A.; Gorhe, D.; Guttman, M.; Jovanovic, M.\nScore: 115.6, Published: 2023-06-07 DOI: 10.1101/2023.06.05.543769\nRNA binding proteins (RBPs) play crucial roles in regulating every stage of the mRNA life cycle and mediating non-coding RNA functions. Despite their importance, the specific roles of most RBPs remain unexplored because we do not know what specific RNAs most RBPs bind. Current methods, such as crosslinking and immunoprecipitation followed by sequencing (CLIP-seq), have expanded our knowledge of RBP-RNA interactions but are generally limited by their ability to map only one RBP at a time. To address this limitation, we developed SPIDR (Split and Pool Identification of RBP targets), a massively multiplexed method to simultaneously profile global RNA binding sites of dozens to hundreds of RBPs in a single experiment. SPIDR employs split-pool barcoding coupled with antibody-bead barcoding to increase the throughput of current CLIP methods by two orders of magnitude. SPIDR reliably identifies precise, single-nucleotide RNA binding sites for diverse classes of RBPs simultaneously. Using SPIDR, we explored changes in RBP binding upon mTOR inhibition and identified that 4EBP1 acts as a dynamic RBP that selectively binds to 5-untranslated regions of specific translationally repressed mRNAs only upon mTOR inhibition. This observation provides a potential mechanism to explain the specificity of translational regulation controlled by mTOR signaling. SPIDR has the potential to revolutionize our understanding of RNA biology and both transcriptional and post-transcriptional gene regulation by enabling rapid, de novo discovery of RNA-protein interactions at an unprecedented scale.\nLarge-scale map of RNA binding protein interactomes across the mRNA life-cycle\nAuthors: Street, L.; Rothamel, K.; Brannan, K.; Jin, W.; Bokor, B.; Dong, K.; Rhine, K.; Madrigel, A.; Al-Azzam, N.; Kim, J.; Ma, Y.; Abdou, A.; Wolin, E.; Doron-Mandel, E. W.; Ahdout, J.; Mujumba, M.; Jovanovic, M.; Yeo, G. W.\nScore: 37.9, Published: 2023-06-08 DOI: 10.1101/2023.06.08.544225\nMessenger RNAs (mRNAs) interact with RNA-binding proteins (RBPs) in diverse ribonucleoprotein complexes (RNPs) during distinct life-cycle stages for their processing and maturation. While substantial attention has focused on understanding RNA regulation by assigning proteins, particularly RBPs, to specific RNA substrates, there has been considerably less exploration leveraging protein-protein interaction (PPI) methodologies to identify and study the role of proteins in mRNA life-cycle stages. To address this gap, we generated an RNA-aware RBP-centric PPI map across the mRNA life-cycle by immunopurification (IP-MS) of [~]100 endogenous RBPs across the life-cycle in the presence or absence of RNase, augmented by size exclusion chromatography (SEC-MS). Aside from confirming 8,700 known and discovering 20,359 novel interactions between 1125 proteins, we determined that 73% of our IP interactions are regulated by the presence of RNA. Our PPI data enables us to link proteins to life-cycle stage functions, highlighting that nearly half of the proteins participate in at least two distinct stages. We show that one of the most highly interconnected proteins, ERH, engages in multiple RNA processes, including via interactions with nuclear speckles and the mRNA export machinery. We also demonstrate that the spliceosomal protein SNRNP200 participates in distinct stress granule-associated RNPs and occupies different RNA target regions in the cytoplasm during stress. Our comprehensive RBP-focused PPI network is a novel resource for identifying multi-stage RBPs and exploring RBP complexes in RNA maturation. HIGHLIGHTSO_LIAn RBP-centric RNA-aware PPI network focuses on the mRNA life-cycle in human cells C_LIO_LIPrey-prey correlation analysis assigns prey proteins to life-cycle stages, of which 536 proteins (half of the network) interact with multiple steps C_LIO_LIERH is highly connected to multiple RNPs to affect nuclear speckle organization and mRNA export C_LIO_LISplicing factor SNRNP200 interacts with stress granule proteins and has distinct RNA occupancy in the cytoplasm C_LI\nDesigning function-specific minimal microbiomes from large microbial communities\nAuthors: Raghu, A. K.; Palanikumar, I.; Raman, K.\nScore: 6.5, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544531\nMotivationMicroorganisms thrive in large communities of diverse species, exhibiting various functionalities. The mammalian gut microbiome, for instance, has the functionality of digesting dietary fibre and producing different short-chain fatty acids. Not all microbes present in a community contribute to a given functionality; it is possible to find a minimal microbiome, which is a subset of the large microbiome, that is capable of performing the functionality while maintaining other community properties such as growth rate. Such a minimal microbiome will also contain keystone species for SCFA production in that community. In the wake of perturbations of the gut microbiome that result in disease conditions, cultivated minimal microbiomes can be administered to restore lost functionalities. ResultsIn this work, we present a systematic algorithm to design a minimal microbiome from a large community for a user-proposed function. We employ a top-down approach with sequential deletion followed by solving a mixed-integer linear programming problem with the objective of minimising the L1-norm of the membership vector. We demonstrate the utility of our algorithm by identifying the minimal microbiomes corresponding to model communities of the gut, and discuss their validity based on the presence of the keystone species in the community. Our approach is generic and finds application in studying a variety of microbial communities. AvailabilityThe algorithm is available from https://github.com/RamanLab/minMicrobiome Author SummaryMicroorganisms are ubiquitous in nature. They survive in communities by interacting with each other and influence the biosphere by carrying out specific functions. For instance, the mammalian digestive system is heavily dependent on microbial communities in the gut (known as gut microbiome) to digest dietary fibres which are otherwise indigestible. The capability of gut microbes to convert dietary fibres to short-chain fatty acids help the host by regulating the functionality of the gut epithelial barrier. Oftentimes, some members of a community have redundant functions. Hence, it is possible to find a smaller subset of organisms that is capable of a given functionality, while also maintaining the required growth rate. We call them a minimal microbiome. Knowledge of such function-specific minimal microbiomes is useful for constructing communities for laboratory study and for designing treatment strategies for medical conditions caused by microbiome disruption. We present an optimization algorithm for identifying function-specific minimal microbiomes from a large community. We also demonstrate the performance of the algorithm by analysing minimal microbiomes obtained from some known communities. Overall, our research work highlights the significance of function-specific minimal microbiomes and provides an efficient computational tool for their identification.\nMapping temperature-sensitive mutations at a genome-scale to engineer growth-switches in E. coli\nAuthors: Schramm, T.; Pahl, V.; Link, H.\nScore: 13.2, Published: 2023-06-02 DOI: 10.1101/2023.06.01.543195\nTemperature-sensitive (TS) mutants are a unique tool to perturb and engineer cellular systems. Here, we constructed a CRISPR library with 15,120 Escherichia coli mutants, each with a single amino acid change in one of 346 essential proteins. 1,269 of these mutants showed temperature-sensitive growth in a time-resolved competition assay. We reconstructed 94 TS mutants and measured their metabolism under growth arrest at 42{degrees}C using metabolomics. Metabolome changes were strong and mutant-specific, showing that metabolism of non-growing E. coli is perturbation-dependent. For example, 24 TS mutants of metabolic enzymes overproduced the direct substrate-metabolite due to a bottleneck in their associated pathway. A strain with TS homoserine kinase (ThrBF267D) produced homoserine for 24 hours, and production was tunable by temperature. Finally, we used a TS subunit of DNA polymerase III (DnaXL289Q) to decouple growth from arginine overproduction in engineered E. coli. These results provide a strategy to identify TS mutants en masse and demonstrate their large potential to produce bacterial metabolites with non-growing cells.\nBayesian analysis dissects kinetic modulation during non-stationary gene expression\nAuthors: Wildner, C.; Ball, D. A.; Mehta, G. D.; Karpova, T. S.; Koeppl, H.\nScore: 0.8, Published: 2023-06-23 DOI: 10.1101/2023.06.20.545522\nLabelling of nascent stem loops with fluorescent proteins has fostered the visualization of transcription in living cells. Quantitative analysis of recorded fluorescence traces can shed light on kinetic transcription parameters and regulatory mechanisms. However, existing methods typically focus on steady state dynamics. Here, we combine a stochastic process transcription model with a hierarchical Bayesian method to infer global as well locally shared parameters for groups of cells and recover unobserved quantities such as initiation times and polymerase loading of the gene. We apply our approach to the cyclic response of the yeast CUP1 locus to heavy metal stress. Within the previously described slow cycle of transcriptional activity on the scale of minutes, we discover fast time-modulated bursting on the scale of seconds. Model comparison suggests that slow oscillations of transcriptional output are regulated by the amplitude of the bursts. Several polymerases may initiate during a burst.\n",
  "wordCount" : "2633",
  "inLanguage": "en",
  "datePublished": "2023-06-28T09:51:07Z",
  "dateModified": "2023-06-28T09:51:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://yorks0n.github.io/TRxiv2/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://yorks0n.github.io/TRxiv2/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://yorks0n.github.io/TRxiv2" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545990">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545990" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545990">
        <p class="paperTitle">Storm: Incorporating transient dynamics to infer the RNA velocity with metabolic labeling information</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545990" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545990" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peng, Q.; Qiu, X.; Li, T.</p>
        <p class="info">Score: 14.8, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545990' target='https://doi.org/10.1101/2023.06.21.545990'> 10.1101/2023.06.21.545990</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWThe time-resolved scRNA-seq (tscRNA-seq) provides the possibility to infer physically meaningful kinetic parameters, e.g., the transcription, splicing or RNA degradation rate constants with correct magnitudes, and RNA velocities by incorporating temporal information. Previous approaches utilizing the deterministic dynamics and steady-state assumption on gene expression states are insufficient to achieve favorable results for the data involving transient process. We present a dynamical approach, Storm (Stochastic models of RNA metabolic-labeling), to overcome these limitations by solving stochastic differential equations of gene expression dynamics. The derivation reveals that the new mRNA sequencing data obeys different types of cell-specific Poisson distributions when jointly considering both biological and cell-specific technical noise. Storm deals with measured counts data directly and extends the RNA velocity methodology based on metabolic labeling scRNA-seq data to transient stochastic systems. Furthermore, we relax the constant parameter assumption over genes/cells to obtain gene-cell-specific transcription/splicing rates and gene-specific degradation rates, thus revealing time-dependent and cell-state specific transcriptional regulations. Storm will facilitate the study of the statistical properties of tscRNA-seq data, eventually advancing our understanding of the dynamic transcription regulation during development and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545537">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545537" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545537">
        <p class="paperTitle">Dual proteomic signature of immune cells and Yersinia pestis upon blood infection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545537" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545537" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Le-Bury, P.; Douche, T.; Giai Gianetto, Q.; Matondo, M.; Pizarro-Cerda, J.; Dussurget, O.</p>
        <p class="info">Score: 7.0, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545537' target='https://doi.org/10.1101/2023.06.19.545537'> 10.1101/2023.06.19.545537</a></p>
        <p class="abstract">Emerging and reemerging infectious diseases represent major public health concerns. The urgent need for infection control measures requires deep understanding of molecular pathogenesis. Global approaches to study biological systems such as mass-spectrometry based proteomics benefited from groundbreaking physical and bioinformatical technological developments over recent years. However, dual proteomic study of highly pathogenic microorganisms and their hosts in complex matrices encountered during infection remains challenging due to high protein dynamic range of samples and requirements imposed in biosafety level 3 or 4 laboratories. Here, we constructed a dual proteomic pipeline of Yersinia pestis in human blood and plasma, mirroring bacteremic phase of plague. We provide the most complete Y. pestis proteome revealing a major reshaping of important bacterial path-ways such as methionine biosynthesis and iron acquisition in human plasma. Remarkably, proteomic profiling in human blood highlights a greater Yersinia outer proteins intoxication of monocytes than neutrophils. Our study unravels global expression changes and points to a specific pathogenic signature during infection, paving the way for future exploration of proteomes in the complex context of host-pathogen interactions.

Subject CategoriesMicrobiology, Virology and Host Pathogen Interaction, Proteomics</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.546001">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.546001" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.546001">
        <p class="paperTitle">The genetic and dietary landscape of the muscle insulin signalling network</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.546001" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.546001" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van Gerwen, J.; Masson, S. W. C.; Cutler, H. B.; Diaz-Vegas, A.; Potter, M.; Stockli, J.; Madsen, S.; Nelson, M. E.; Humphrey, S. J.; James, D. E.</p>
        <p class="info">Score: 6.3, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.546001' target='https://doi.org/10.1101/2023.06.21.546001'> 10.1101/2023.06.21.546001</a></p>
        <p class="abstract">Metabolic disease is caused by a combination of genetic and environmental factors, yet few studies have examined how these factors influence signal transduction, a key mediator of metabolism. Using mass spectrometry-based phosphoproteomics, we quantified 23,126 phosphosites in skeletal muscle of five genetically distinct mouse strains in two dietary environments, with and without acute in vivo insulin stimulation. Almost half of the insulin-regulated phosphoproteome was modified by genetic background on an ordinary diet, and high-fat high-sugar feeding affected insulin signalling in a strain-dependent manner. Our data revealed coregulated subnetworks within the insulin signalling pathway, expanding our understanding of the pathways organisation. Furthermore, associating diverse signalling responses with insulin-stimulated glucose uptake uncovered regulators of muscle insulin responsiveness, including the regulatory phosphosite S469 on Pfkfb2, a key activator of glycolysis. Finally, we confirmed the role of glycolysis in modulating insulin action in insulin resistance. Our results underscore the significance of genetics in shaping global signalling responses and their adaptability to environmental changes, emphasizing the utility of studying biological diversity with phosphoproteomics to discover key regulatory mechanisms of complex traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545536">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545536" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545536">
        <p class="paperTitle">Quantifying and correcting estimation bias in transcriptional parameter inference from single-cell data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545536" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545536" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grima, R.; Esmenjaud, P.-M.</p>
        <p class="info">Score: 3.1, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545536' target='https://doi.org/10.1101/2023.06.19.545536'> 10.1101/2023.06.19.545536</a></p>
        <p class="abstract">The snapshot distribution of mRNA counts per cell can be readily measured using single molecule FISH or single-cell RNA sequencing. These distributions are often fit to the steady-state distribution of the two-state telegraph model to estimate the three transcriptional parameters for a gene of interest: the rate of transcription, the rate of switching to the active transcriptional state and the rate of switching to the inactive state. These parameters are often understood as being reflective of the average cell in a population. However it is currently unclear to what extent does the cell-to-cell variation of parameters (extrinsic noise) affect the accuracy of parameter estimation. Here we develop an analytical approach to understand the factors determining the size and sign of estimation bias. We find bias signatures that depend on the source of extrinsic noise and the mode of transcriptional activity. We estimate the size of extrinsic noise from the covariance matrix of scRNA-seq data which we use to correct published estimates of parameters for mammalian genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.07.544138">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.07.544138" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.07.544138">
        <p class="paperTitle">Heritable epigenetic changes are constrained by the dynamics of regulatory architectures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.07.544138" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.07.544138" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jose, A. M.</p>
        <p class="info">Score: 4.1, Published: 2023-06-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.07.544138' target='https://doi.org/10.1101/2023.06.07.544138'> 10.1101/2023.06.07.544138</a></p>
        <p class="abstract">Interacting molecules create regulatory architectures that can persist despite turnover of molecules. Although epigenetic changes occur within the context of such architectures, there is limited understanding of how they can influence the heritability of changes. Here I develop criteria for the heritability of regulatory architectures and use quantitative simulations of interacting regulators parsed as entities, their sensors and the sensed properties to analyze how architectures influence heritable epigenetic changes. Information contained in regulatory architectures grows rapidly with the number of interacting molecules and its transmission requires positive feedback loops. While these architectures can recover after many epigenetic perturbations, some resulting changes can become permanently heritable. Such stable changes can (1) alter steady-state levels while preserving the architecture, (2) induce different architectures that persist for many generations, or (3) collapse the entire architecture. Architectures that are otherwise unstable can become heritable through periodic interactions with external regulators, which suggests that the evolution of mortal somatic lineages with cells that reproducibly interact with the immortal germ lineage could make a wider variety of regulatory architectures heritable. Differential inhibition of the positive feedback loops that transmit regulatory architectures across generations can explain the gene-specific differences in heritable RNA silencing observed in the nematode C. elegans, which range from permanent silencing, to recovery from silencing within a few generations and subsequent resistance to silencing. More broadly, these results provide a foundation for analyzing the inheritance of epigenetic changes within the context of the regulatory architectures implemented using diverse molecules in different living systems.

SIGNIFICANCERegulatory interactions in living systems are recreated in successive generations. Practical ways of analyzing how the information required for this recreation is transmitted across generations and how it can be changed are lacking. Simulating all heritable information by parsing regulatory interactions in terms of entities, their sensors, and the sensed properties reveals the minimal requirements for the heritability of regulatory interactions and how they influence the inheritance of epigenetic changes. Application of this approach can explain recent experimental results on the inheritance of RNA silencing across generations in the nematode C. elegans. Since all interactors can be abstracted as entity-sensor-property systems, similar analyses can be widely used to understand heritable epigenetic changes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.05.543769">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.05.543769" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.05.543769">
        <p class="paperTitle">SPIDR: a highly multiplexed method for mapping RNA-protein interactions uncovers a potential mechanism for selective translational suppression upon cellular stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.05.543769" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.05.543769" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wolin, E.; Guo, J. K.; Blanco, M. R.; Perez, A. A.; Goronzy, I. N.; Abdou, A. A.; Gorhe, D.; Guttman, M.; Jovanovic, M.</p>
        <p class="info">Score: 115.6, Published: 2023-06-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.05.543769' target='https://doi.org/10.1101/2023.06.05.543769'> 10.1101/2023.06.05.543769</a></p>
        <p class="abstract">RNA binding proteins (RBPs) play crucial roles in regulating every stage of the mRNA life cycle and mediating non-coding RNA functions. Despite their importance, the specific roles of most RBPs remain unexplored because we do not know what specific RNAs most RBPs bind. Current methods, such as crosslinking and immunoprecipitation followed by sequencing (CLIP-seq), have expanded our knowledge of RBP-RNA interactions but are generally limited by their ability to map only one RBP at a time. To address this limitation, we developed SPIDR (Split and Pool Identification of RBP targets), a massively multiplexed method to simultaneously profile global RNA binding sites of dozens to hundreds of RBPs in a single experiment. SPIDR employs split-pool barcoding coupled with antibody-bead barcoding to increase the throughput of current CLIP methods by two orders of magnitude. SPIDR reliably identifies precise, single-nucleotide RNA binding sites for diverse classes of RBPs simultaneously. Using SPIDR, we explored changes in RBP binding upon mTOR inhibition and identified that 4EBP1 acts as a dynamic RBP that selectively binds to 5-untranslated regions of specific translationally repressed mRNAs only upon mTOR inhibition. This observation provides a potential mechanism to explain the specificity of translational regulation controlled by mTOR signaling. SPIDR has the potential to revolutionize our understanding of RNA biology and both transcriptional and post-transcriptional gene regulation by enabling rapid, de novo discovery of RNA-protein interactions at an unprecedented scale.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.08.544225">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.08.544225" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.08.544225">
        <p class="paperTitle">Large-scale map of RNA binding protein interactomes across the mRNA life-cycle</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.08.544225" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.08.544225" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Street, L.; Rothamel, K.; Brannan, K.; Jin, W.; Bokor, B.; Dong, K.; Rhine, K.; Madrigel, A.; Al-Azzam, N.; Kim, J.; Ma, Y.; Abdou, A.; Wolin, E.; Doron-Mandel, E. W.; Ahdout, J.; Mujumba, M.; Jovanovic, M.; Yeo, G. W.</p>
        <p class="info">Score: 37.9, Published: 2023-06-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.08.544225' target='https://doi.org/10.1101/2023.06.08.544225'> 10.1101/2023.06.08.544225</a></p>
        <p class="abstract">Messenger RNAs (mRNAs) interact with RNA-binding proteins (RBPs) in diverse ribonucleoprotein complexes (RNPs) during distinct life-cycle stages for their processing and maturation. While substantial attention has focused on understanding RNA regulation by assigning proteins, particularly RBPs, to specific RNA substrates, there has been considerably less exploration leveraging protein-protein interaction (PPI) methodologies to identify and study the role of proteins in mRNA life-cycle stages. To address this gap, we generated an RNA-aware RBP-centric PPI map across the mRNA life-cycle by immunopurification (IP-MS) of [~]100 endogenous RBPs across the life-cycle in the presence or absence of RNase, augmented by size exclusion chromatography (SEC-MS). Aside from confirming 8,700 known and discovering 20,359 novel interactions between 1125 proteins, we determined that 73% of our IP interactions are regulated by the presence of RNA. Our PPI data enables us to link proteins to life-cycle stage functions, highlighting that nearly half of the proteins participate in at least two distinct stages. We show that one of the most highly interconnected proteins, ERH, engages in multiple RNA processes, including via interactions with nuclear speckles and the mRNA export machinery. We also demonstrate that the spliceosomal protein SNRNP200 participates in distinct stress granule-associated RNPs and occupies different RNA target regions in the cytoplasm during stress. Our comprehensive RBP-focused PPI network is a novel resource for identifying multi-stage RBPs and exploring RBP complexes in RNA maturation.

HIGHLIGHTSO_LIAn RBP-centric RNA-aware PPI network focuses on the mRNA life-cycle in human cells
C_LIO_LIPrey-prey correlation analysis assigns prey proteins to life-cycle stages, of which 536 proteins (half of the network) interact with multiple steps
C_LIO_LIERH is highly connected to multiple RNPs to affect nuclear speckle organization and mRNA export
C_LIO_LISplicing factor SNRNP200 interacts with stress granule proteins and has distinct RNA occupancy in the cytoplasm
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.12.544531">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.12.544531" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.12.544531">
        <p class="paperTitle">Designing function-specific minimal microbiomes from large microbial communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.12.544531" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.12.544531" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Raghu, A. K.; Palanikumar, I.; Raman, K.</p>
        <p class="info">Score: 6.5, Published: 2023-06-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.12.544531' target='https://doi.org/10.1101/2023.06.12.544531'> 10.1101/2023.06.12.544531</a></p>
        <p class="abstract">MotivationMicroorganisms thrive in large communities of diverse species, exhibiting various functionalities. The mammalian gut microbiome, for instance, has the functionality of digesting dietary fibre and producing different short-chain fatty acids. Not all microbes present in a community contribute to a given functionality; it is possible to find a minimal microbiome, which is a subset of the large microbiome, that is capable of performing the functionality while maintaining other community properties such as growth rate. Such a minimal microbiome will also contain keystone species for SCFA production in that community. In the wake of perturbations of the gut microbiome that result in disease conditions, cultivated minimal microbiomes can be administered to restore lost functionalities.

ResultsIn this work, we present a systematic algorithm to design a minimal microbiome from a large community for a user-proposed function. We employ a top-down approach with sequential deletion followed by solving a mixed-integer linear programming problem with the objective of minimising the L1-norm of the membership vector. We demonstrate the utility of our algorithm by identifying the minimal microbiomes corresponding to model communities of the gut, and discuss their validity based on the presence of the keystone species in the community. Our approach is generic and finds application in studying a variety of microbial communities.

AvailabilityThe algorithm is available from https://github.com/RamanLab/minMicrobiome

Author SummaryMicroorganisms are ubiquitous in nature. They survive in communities by interacting with each other and influence the biosphere by carrying out specific functions. For instance, the mammalian digestive system is heavily dependent on microbial communities in the gut (known as gut microbiome) to digest dietary fibres which are otherwise indigestible. The capability of gut microbes to convert dietary fibres to short-chain fatty acids help the host by regulating the functionality of the gut epithelial barrier. Oftentimes, some members of a community have redundant functions. Hence, it is possible to find a smaller subset of organisms that is capable of a given functionality, while also maintaining the required growth rate. We call them a minimal microbiome. Knowledge of such function-specific minimal microbiomes is useful for constructing communities for laboratory study and for designing treatment strategies for medical conditions caused by microbiome disruption. We present an optimization algorithm for identifying function-specific minimal microbiomes from a large community. We also demonstrate the performance of the algorithm by analysing minimal microbiomes obtained from some known communities. Overall, our research work highlights the significance of function-specific minimal microbiomes and provides an efficient computational tool for their identification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.01.543195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.01.543195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.01.543195">
        <p class="paperTitle">Mapping temperature-sensitive mutations at a genome-scale to engineer growth-switches in E. coli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.01.543195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.01.543195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schramm, T.; Pahl, V.; Link, H.</p>
        <p class="info">Score: 13.2, Published: 2023-06-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.01.543195' target='https://doi.org/10.1101/2023.06.01.543195'> 10.1101/2023.06.01.543195</a></p>
        <p class="abstract">Temperature-sensitive (TS) mutants are a unique tool to perturb and engineer cellular systems. Here, we constructed a CRISPR library with 15,120 Escherichia coli mutants, each with a single amino acid change in one of 346 essential proteins. 1,269 of these mutants showed temperature-sensitive growth in a time-resolved competition assay. We reconstructed 94 TS mutants and measured their metabolism under growth arrest at 42{degrees}C using metabolomics. Metabolome changes were strong and mutant-specific, showing that metabolism of non-growing E. coli is perturbation-dependent. For example, 24 TS mutants of metabolic enzymes overproduced the direct substrate-metabolite due to a bottleneck in their associated pathway. A strain with TS homoserine kinase (ThrBF267D) produced homoserine for 24 hours, and production was tunable by temperature. Finally, we used a TS subunit of DNA polymerase III (DnaXL289Q) to decouple growth from arginine overproduction in engineered E. coli. These results provide a strategy to identify TS mutants en masse and demonstrate their large potential to produce bacterial metabolites with non-growing cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545522">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545522" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545522">
        <p class="paperTitle">Bayesian analysis dissects kinetic modulation during non-stationary gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545522" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545522" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wildner, C.; Ball, D. A.; Mehta, G. D.; Karpova, T. S.; Koeppl, H.</p>
        <p class="info">Score: 0.8, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545522' target='https://doi.org/10.1101/2023.06.20.545522'> 10.1101/2023.06.20.545522</a></p>
        <p class="abstract">Labelling of nascent stem loops with fluorescent proteins has fostered the visualization of transcription in living cells. Quantitative analysis of recorded fluorescence traces can shed light on kinetic transcription parameters and regulatory mechanisms. However, existing methods typically focus on steady state dynamics. Here, we combine a stochastic process transcription model with a hierarchical Bayesian method to infer global as well locally shared parameters for groups of cells and recover unobserved quantities such as initiation times and polymerase loading of the gene. We apply our approach to the cyclic response of the yeast CUP1 locus to heavy metal stress. Within the previously described slow cycle of transcriptional activity on the scale of minutes, we discover fast time-modulated bursting on the scale of seconds. Model comparison suggests that slow oscillations of transcriptional output are regulated by the amplitude of the bursts. Several polymerases may initiate during a burst.</p>
      </div>
    </div>
  </div>
</div>









<script src="/TRxiv2/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://yorks0n.github.io/TRxiv2">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
