<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 87.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 87.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 87.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Open-ST: High-resolution spatial transcriptomics in 3D\nAuthors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.\nScore: 87.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554\nSpatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods.",
  "keywords": [
    
  ],
  "articleBody": " Open-ST: High-resolution spatial transcriptomics in 3D\nAuthors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.\nScore: 87.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554\nSpatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods. Here, we report Open-ST, a sequencing-based, open-source experimental and computational resource to address these challenges and to study the molecular organization of tissues in 3D. In mouse brain, Open-ST captured transcripts at subcellular resolution and reconstructed cell types. In primary tumor and patient-matched healthy/metastatic lymph nodes, Open-ST captured the diversity of immune, stromal and tumor populations in space. Distinct cell states were organized around cell-cell communication hotspots in the tumor, but not the metastasis. Strikingly, the 3D reconstruction and multimodal analysis of the metastatic lymph node revealed spatially contiguous structures not visible in 2D and potential biomarkers precisely at the 3D tumor/lymph node boundary. We anticipate Open-ST to accelerate the identification of spatial molecular mechanisms in 2D and 3D.\nMetalinksDB: a flexible and contextualizable resource of metabolite-protein interactions\nAuthors: Farr, E. B.; Dimitrov, D.; Turei, D.; Schmidt, C.; Lobentanzer, S.; Dugourd, A.; Saez-Rodriguez, J.\nScore: 14.9, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573715\nInteractions between proteins and metabolites are key for cellular function, from the catalytic breakdown of nutrients to signaling. An important case is cell-cell communication, where cellular metabolites are secreted into the microenvironment and initiate a signaling cascade by binding to an intra- or extracellular receptor of another cell. While protein-protein mediated cell-cell communication is routinely inferred from transcriptomic data, for metabolite-protein interactions this is challenging due to the limitations of high-throughput single-cell and spatial metabolomics technologies, together with the absence of comprehensive prior knowledge resources that include metabolites. Here we report MetalinksDB, a comprehensive and flexible database of intercellular metabolite-protein interactions that is a magnitude larger than existing ones. MetalinksDB can be tailored to specific biological contexts such as diseases, pathways, or tissue/cellular locations by querying subsets of interactions using the web interface (https://metalinks.omnipathdb.org/) or the knowledge graph adapters. We showcase the use of MetalinksDB by identifying deregulated processes in renal cancer patients from multi-omics data as well as inferring metabolite-mediated cell-cell communication events driving acute kidney injury from spatial transcriptomic data. We anticipate that MetalinksDB will facilitate the study of metabolite-mediated communication processes. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=113 SRC=\"FIGDIR/small/573715v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (34K): org.highwire.dtl.DTLVardef@18eeb45org.highwire.dtl.DTLVardef@1f82cf2org.highwire.dtl.DTLVardef@4edf8corg.highwire.dtl.DTLVardef@367d35_HPS_FORMAT_FIGEXP M_FIG C_FIG\nUnraveling the oscillatory dynamics of mRNA metabolism and chromatin accessibility during the cell cycle through integration of single-cell multiomic data\nAuthors: Nariya, M. K.; Santiago-Algarra, D.; Tassy, O.; Cerciat, M.; Ye, T.; Riba, A.; Molina, N.\nScore: 11.2, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575159\nThe cell cycle is a highly regulated process that ensures the accurate replication and transmission of genetic information from one generation of cells to the next. It is a fundamental biological process and plays a crucial role in development, growth, and maintenance of all living organisms, and its dysregulation can lead to a number of pathologies such as autoimmune diseases, neurodegenerative diseases, and cancer. In this work we present a novel approach to study the gene expression and chromatin accessibility dynamics during the cell cycle in mouse embryonic stem cells. To achieve this, we combined high-depth single-cell multiome sequencing, biophysical modeling, and advanced deep learning techniques. First, we used DeepCycle, a deep learning tool that assigns a cell cycle phase to every cell based on its spliced and unspliced mRNA levels. We then developed a biophysical model that describes the dynamics of gene-specific mRNA transcription, nuclear export, and degradation rates during the cell cycle. Our model obtains genome-wide estimates of mRNA transcription, nuclear retention, and degradation half-lives of genes exhibiting oscillatory dynamics in mESCs. The key feature of single-cell multiome sequencing is that it simultaneously provides readouts for gene expression as well as chromatin accessibility in the same cells. By applying our approach to these data we showcase first-of-its-kind exploration of chromatin accessibility during the cell cycle at high temporal resolution. We believe that our work will be pivotal in formulating a coherent quantitative description of mRNA metabolism. HighlightsO_LIIntegration of single-cell and single-nucleus multiome profiling with deep learning and biophysical modeling uncovers cell cycle regulation of RNA metabolism C_LIO_LIGenome-wide estimation of mRNA transcription, nuclear export, and degradation rates during the cell cycle in mESCs C_LIO_LIIdentification of a prominent wave of transcription during early G1 in unperturbed asynchronously proliferating mESCs C_LIO_LICharacterization of temporal waves of post-transcriptional mRNA regulation controlling both mRNA degradation and nuclear export during the cell cycle C_LIO_LIChromatin accessibility dynamics within TF footprints reveal key cell cycle regulators C_LI\nImpaired biogenesis of basic proteins impacts multiple hallmarks of the aging brain\nAuthors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.\nScore: 37.0, Published: 2024-01-09 DOI: 10.1101/2023.07.20.549210\nAging and neurodegeneration entail diverse cellular and molecular hallmarks. Here, we studied the effects of aging on the transcriptome, translatome, and multiple layers of the proteome in the brain of a short-lived killifish. We reveal that aging causes widespread reduction of proteins enriched in basic amino acids that is independent of mRNA regulation, and it is not due to impaired proteasome activity. Instead, we identify a cascade of events where aberrant translation pausing leads to reduced ribosome availability resulting in proteome remodeling independently of transcriptional regulation. Our research uncovers a vulnerable point in the aging brains biology - the biogenesis of basic DNA/RNA binding proteins. This vulnerability may represent a unifying principle that connects various aging hallmarks, encompassing genome integrity and the biosynthesis of macromolecules. One-Sentence SummaryTranslation pausing reshapes the aging brain proteome, revealing vulnerabilities in the biogenesis of nucleic-acid protein.\nA quantitative description of light-limited cyanobacterial growth using flux balance analysis\nAuthors: Hoeper, R.; Komkova, D.; Zavrel, T.; Steuer, R.\nScore: 7.9, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574812\nThe metabolism of phototrophic cyanobacterial is an integral part of global biogeochemical cycles, and the capability of cyanobacteria to assimilate atmospheric CO2 into organic carbon has manifold potential applications for a sustainable biotechnology. To elucidate the properties of cyanobacterial metabolism and growth, computational reconstructions of the genome-scale metabolic networks play an increasingly important role. Here, we present an updated reconstruction of the metabolic network of the cyanobacterium Synechocystis sp. PCC 6803 and its analysis using flux balance analysis (FBA). To overcome limitations of conventional FBA, and to allow for the integration of quantitative experimental analyses, we develop a novel approach to describe light absorption and light utilization. Our approach incorporates photoinhibition and a variable quantum yield into the constraint-based description of light-limited phototrophic growth. We show that the resulting model is capable to predict quantitative properties of cyanobacterial growth, including photosynthetic oxygen evolution and the ATP/NADPH ratio required for growth and cellular maintenance. Our approach retains the computational and conceptual simplicity of FBA and is readily applicable to other phototropic microorganisms.\nBeyond microtubules: The cellular environment at the endoplasmic reticulum attracts proteins to the nucleus, enabling nuclear transport\nAuthors: Chae, S. J.; Kim, D. W.; Igoshin, O. A.; Lee, S.; Kim, J. K.\nScore: 7.5, Published: 2024-01-14 DOI: 10.1101/2024.01.12.575351\nAll proteins are translated in the cytoplasm, yet many, including transcription factors, play vital roles in the nucleus. While previous research has concentrated on molecular motors for the transport of these proteins to the nucleus, recent observations reveal perinuclear accumulation even in the absence of an energy source, hinting at alternative mechanisms. Here, we propose that structural properties of the cellular environment, specifically the endoplasmic reticulum (ER), can promote molecular transport to the perinucleus without requiring additional energy expenditure. Specifically, physical interaction between proteins and the ER impedes their diffusion and leads to their accumulation near the nucleus. This result explains why larger proteins, more frequently interacting with the ER membrane, tend to accumulate at the perinucleus. Interestingly, such diffusion in a heterogeneous environment follows Chapman law rather than the popular Fick law. Our findings suggest a novel protein transport mechanism arising solely from characteristics of the intracellular environment.\nTranscriptome-wide analysis of cell cycle-dependent bursty gene expression from single-cell RNA-seq data using mechanistic model-based inference\nAuthors: Sukys, A.; Grima, R.\nScore: 6.1, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574820\nBursty gene expression is quantified by two intuitive parameters: the burst frequency and the burst size. While these parameters are known to be cell-cycle dependent for some genes, a transcriptome-wide picture remains missing. Here we address this question by fitting a suite of mechanistic models of gene expression to mRNA count data for thousands of mouse genes, obtained by sequencing of single cells for which the cell-cycle position has been inferred using a deep-learning approach. This leads to the estimation of the burst frequency and size per allele in the G1 and G2/M cell-cycle phases, hence providing insight into the global patterns of transcriptional regulation. In particular, we identify an interesting balancing mechanism: on average, upon DNA replication, the burst frequency decreases by{approx} 50%, while the burst size increases by the same amount. We also show that for accurate estimation of the ratio of burst parameters in the G1 and G2/M phases, mechanistic models must explicitly account for gene copy number differences between cells but, surprisingly, additional corrections for extrinsic noise due to the coupling of transcription to cell age within the cell cycle or technical noise due to imperfect capture of RNA molecules in sequencing experiments are unnecessary.\nEnhanced feature matching in single-cell proteomics characterizes response to IFN-gamma and reveals co-existence of different cell states\nAuthors: Krull, K. K.; Ali, S. A.; Krijgsveld, J.\nScore: 2.5, Published: 2024-01-10 DOI: 10.1101/2024.01.10.575010\nProteome analysis by data-independent acquisition (DIA) has become a powerful approach to obtain deep proteome coverage, and has gained recent traction for label-free analysis of single cells. However, optimal experimental design for DIA-based single-cell proteomics has not been fully explored, and performance metrics of subsequent data analysis tools remain to be evaluated. Therefore, we here present DIA-ME, a data analysis strategy that exploits the co-analysis of low-input samples with a so-called matching enhancer (ME) of higher input, to increase sensitivity, proteome coverage, and data completeness. We evaluate the matching specificity of DIA-ME by a two-proteome model, and demonstrate that false discovery and false transfer are maintained at low levels when using DIA-NN software, while preserving quantification accuracy. We apply DIA-ME to investigate the proteome response of U-2 OS cells to interferon gamma (IFN-{gamma}) in single cells, and recapitulate the time-resolved induction of IFN-{gamma} response proteins as observed in bulk material. Moreover, we observe co- and anti-correlating patterns of protein expression within the same cell, indicating mutually exclusive protein modules and the co-existence of different cell states. Collectively our data show that DIA-ME is a powerful, scalable, and easy-to- implement strategy for single-cell proteomics.\nTargeted Perturb-seq Reveals EGR1 and FOS as Key Regulators of the Transcriptional RAF-MAPK Response\nAuthors: El Kassem, G.; Sieber, A.; Klinger, B.; Uhlitz, F.; Steinbrecht, D.; van Bentum, M.; Hillmer, J.; von Schlichting, J.; Schäfer, R.; Blüthgen, N.; Boettcher, M.\nScore: 1.9, Published: 2024-01-15 DOI: 10.1101/2024.01.13.575500\nThe MAPK pathway is an important cellular signaling cascade whose dysregulation causes a variety of diseases. While the upstream regulators of this cascade have been extensively characterized, the understanding of how its activation translates into different transcriptional responses remains poorly understood. This study attempts to fill this knowledge gap by using targeted Perturb-seq against 22 transcription factors in an inducible model system for RAF-MAPK signaling. A topology-based modeling approach is applied to the obtained data to construct a directional interaction network. By removing coherent feed-forward loops and integrating the expression kinetics of transcription factors, a parsimonious network structure is derived that distinguishes direct from indirect interactions between the investigated transcription factors and their targets. In particular, EGR1 and FOS are found to act as orthogonal upstream regulators of the RAF-MAPK response. The results presented here provide valuable insights into the organization of the transcriptional network downstream of RAF-MAPK signaling and thus provide a basis for a better understanding of this complex process.\nEpistemic uncertainty challenges aging clock reliability in predicting rejuvenation effects\nAuthors: Kriukov, D.; Kuzmina, E. A.; Efimov, E.; Dylov, D. V.; Khrameeva, E.\nScore: 6.9, Published: 2023-12-30 DOI: 10.1101/2023.12.01.569529\nEpigenetic aging clocks have been widely used to validate rejuvenation effects during cellular reprogramming. However, these predictions are unverifiable because the true biological age of reprogrammed cells remains unknown. We present an analytical framework to consider rejuvenation predictions from the uncertainty perspective. Our analysis reveals that the DNA methylation profiles across reprogramming are poorly represented in the aging data used to train clock models, thus introducing high epistemic uncertainty in age estimations. Moreover, predictions of different published clocks are inconsistent, with some even suggesting zero or negative rejuvenation. While not questioning the possibility of age reversal, we show that the high clock uncertainty challenges the reliability of rejuvenation effects observed during in vitro reprogramming prior to pluripotency and throughout embryogenesis. Conversely, our method reveals a significant age increase after in vivo reprogramming. We recommend including uncertainty estimation in future aging clock models to avoid the risk of misinterpreting the results of biological age prediction.\n",
  "wordCount" : "2213",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572554">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572554" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572554">
        <p class="paperTitle">Open-ST: High-resolution spatial transcriptomics in 3D</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572554" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572554" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.</p>
        <p class="info">Score: 87.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572554' target='https://doi.org/10.1101/2023.12.22.572554'> 10.1101/2023.12.22.572554</a></p>
        <p class="abstract">Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods. Here, we report Open-ST, a sequencing-based, open-source experimental and computational resource to address these challenges and to study the molecular organization of tissues in 3D. In mouse brain, Open-ST captured transcripts at subcellular resolution and reconstructed cell types. In primary tumor and patient-matched healthy/metastatic lymph nodes, Open-ST captured the diversity of immune, stromal and tumor populations in space. Distinct cell states were organized around cell-cell communication hotspots in the tumor, but not the metastasis. Strikingly, the 3D reconstruction and multimodal analysis of the metastatic lymph node revealed spatially contiguous structures not visible in 2D and potential biomarkers precisely at the 3D tumor/lymph node boundary. We anticipate Open-ST to accelerate the identification of spatial molecular mechanisms in 2D and 3D.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573715">
        <p class="paperTitle">MetalinksDB: a flexible and contextualizable resource of metabolite-protein interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Farr, E. B.; Dimitrov, D.; Turei, D.; Schmidt, C.; Lobentanzer, S.; Dugourd, A.; Saez-Rodriguez, J.</p>
        <p class="info">Score: 14.9, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573715' target='https://doi.org/10.1101/2023.12.30.573715'> 10.1101/2023.12.30.573715</a></p>
        <p class="abstract">Interactions between proteins and metabolites are key for cellular function, from the catalytic breakdown of nutrients to signaling. An important case is cell-cell communication, where cellular metabolites are secreted into the microenvironment and initiate a signaling cascade by binding to an intra- or extracellular receptor of another cell. While protein-protein mediated cell-cell communication is routinely inferred from transcriptomic data, for metabolite-protein interactions this is challenging due to the limitations of high-throughput single-cell and spatial metabolomics technologies, together with the absence of comprehensive prior knowledge resources that include metabolites. Here we report MetalinksDB, a comprehensive and flexible database of intercellular metabolite-protein interactions that is a magnitude larger than existing ones. MetalinksDB can be tailored to specific biological contexts such as diseases, pathways, or tissue/cellular locations by querying subsets of interactions using the web interface (https://metalinks.omnipathdb.org/) or the knowledge graph adapters. We showcase the use of MetalinksDB by identifying deregulated processes in renal cancer patients from multi-omics data as well as inferring metabolite-mediated cell-cell communication events driving acute kidney injury from spatial transcriptomic data. We anticipate that MetalinksDB will facilitate the study of metabolite-mediated communication processes.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=113 SRC=&#34;FIGDIR/small/573715v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (34K):
org.highwire.dtl.DTLVardef@18eeb45org.highwire.dtl.DTLVardef@1f82cf2org.highwire.dtl.DTLVardef@4edf8corg.highwire.dtl.DTLVardef@367d35_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575159">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575159" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575159">
        <p class="paperTitle">Unraveling the oscillatory dynamics of mRNA metabolism and chromatin accessibility during the cell cycle through integration of single-cell multiomic data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575159" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575159" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nariya, M. K.; Santiago-Algarra, D.; Tassy, O.; Cerciat, M.; Ye, T.; Riba, A.; Molina, N.</p>
        <p class="info">Score: 11.2, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575159' target='https://doi.org/10.1101/2024.01.11.575159'> 10.1101/2024.01.11.575159</a></p>
        <p class="abstract">The cell cycle is a highly regulated process that ensures the accurate replication and transmission of genetic information from one generation of cells to the next. It is a fundamental biological process and plays a crucial role in development, growth, and maintenance of all living organisms, and its dysregulation can lead to a number of pathologies such as autoimmune diseases, neurodegenerative diseases, and cancer. In this work we present a novel approach to study the gene expression and chromatin accessibility dynamics during the cell cycle in mouse embryonic stem cells. To achieve this, we combined high-depth single-cell multiome sequencing, biophysical modeling, and advanced deep learning techniques. First, we used DeepCycle, a deep learning tool that assigns a cell cycle phase to every cell based on its spliced and unspliced mRNA levels. We then developed a biophysical model that describes the dynamics of gene-specific mRNA transcription, nuclear export, and degradation rates during the cell cycle. Our model obtains genome-wide estimates of mRNA transcription, nuclear retention, and degradation half-lives of genes exhibiting oscillatory dynamics in mESCs. The key feature of single-cell multiome sequencing is that it simultaneously provides readouts for gene expression as well as chromatin accessibility in the same cells. By applying our approach to these data we showcase first-of-its-kind exploration of chromatin accessibility during the cell cycle at high temporal resolution. We believe that our work will be pivotal in formulating a coherent quantitative description of mRNA metabolism.

HighlightsO_LIIntegration of single-cell and single-nucleus multiome profiling with deep learning and biophysical modeling uncovers cell cycle regulation of RNA metabolism
C_LIO_LIGenome-wide estimation of mRNA transcription, nuclear export, and degradation rates during the cell cycle in mESCs
C_LIO_LIIdentification of a prominent wave of transcription during early G1 in unperturbed asynchronously proliferating mESCs
C_LIO_LICharacterization of temporal waves of post-transcriptional mRNA regulation controlling both mRNA degradation and nuclear export during the cell cycle
C_LIO_LIChromatin accessibility dynamics within TF footprints reveal key cell cycle regulators
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549210">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549210" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549210">
        <p class="paperTitle">Impaired biogenesis of basic proteins impacts multiple hallmarks of the aging brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549210" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549210" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.</p>
        <p class="info">Score: 37.0, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549210' target='https://doi.org/10.1101/2023.07.20.549210'> 10.1101/2023.07.20.549210</a></p>
        <p class="abstract">Aging and neurodegeneration entail diverse cellular and molecular hallmarks. Here, we studied the effects of aging on the transcriptome, translatome, and multiple layers of the proteome in the brain of a short-lived killifish. We reveal that aging causes widespread reduction of proteins enriched in basic amino acids that is independent of mRNA regulation, and it is not due to impaired proteasome activity. Instead, we identify a cascade of events where aberrant translation pausing leads to reduced ribosome availability resulting in proteome remodeling independently of transcriptional regulation. Our research uncovers a vulnerable point in the aging brains biology - the biogenesis of basic DNA/RNA binding proteins. This vulnerability may represent a unifying principle that connects various aging hallmarks, encompassing genome integrity and the biosynthesis of macromolecules.

One-Sentence SummaryTranslation pausing reshapes the aging brain proteome, revealing vulnerabilities in the biogenesis of nucleic-acid protein.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574812">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574812" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574812">
        <p class="paperTitle">A quantitative description of light-limited cyanobacterial growth using flux balance analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574812" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574812" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoeper, R.; Komkova, D.; Zavrel, T.; Steuer, R.</p>
        <p class="info">Score: 7.9, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574812' target='https://doi.org/10.1101/2024.01.10.574812'> 10.1101/2024.01.10.574812</a></p>
        <p class="abstract">The metabolism of phototrophic cyanobacterial is an integral part of global biogeochemical cycles, and the capability of cyanobacteria to assimilate atmospheric CO2 into organic carbon has manifold potential applications for a sustainable biotechnology. To elucidate the properties of cyanobacterial metabolism and growth, computational reconstructions of the genome-scale metabolic networks play an increasingly important role. Here, we present an updated reconstruction of the metabolic network of the cyanobacterium Synechocystis sp. PCC 6803 and its analysis using flux balance analysis (FBA). To overcome limitations of conventional FBA, and to allow for the integration of quantitative experimental analyses, we develop a novel approach to describe light absorption and light utilization. Our approach incorporates photoinhibition and a variable quantum yield into the constraint-based description of light-limited phototrophic growth. We show that the resulting model is capable to predict quantitative properties of cyanobacterial growth, including photosynthetic oxygen evolution and the ATP/NADPH ratio required for growth and cellular maintenance. Our approach retains the computational and conceptual simplicity of FBA and is readily applicable to other phototropic microorganisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575351">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575351" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575351">
        <p class="paperTitle">Beyond microtubules: The cellular environment at the endoplasmic reticulum attracts proteins to the nucleus, enabling nuclear transport</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575351" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575351" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chae, S. J.; Kim, D. W.; Igoshin, O. A.; Lee, S.; Kim, J. K.</p>
        <p class="info">Score: 7.5, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575351' target='https://doi.org/10.1101/2024.01.12.575351'> 10.1101/2024.01.12.575351</a></p>
        <p class="abstract">All proteins are translated in the cytoplasm, yet many, including transcription factors, play vital roles in the nucleus. While previous research has concentrated on molecular motors for the transport of these proteins to the nucleus, recent observations reveal perinuclear accumulation even in the absence of an energy source, hinting at alternative mechanisms. Here, we propose that structural properties of the cellular environment, specifically the endoplasmic reticulum (ER), can promote molecular transport to the perinucleus without requiring additional energy expenditure. Specifically, physical interaction between proteins and the ER impedes their diffusion and leads to their accumulation near the nucleus. This result explains why larger proteins, more frequently interacting with the ER membrane, tend to accumulate at the perinucleus. Interestingly, such diffusion in a heterogeneous environment follows Chapman law rather than the popular Fick law. Our findings suggest a novel protein transport mechanism arising solely from characteristics of the intracellular environment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574820">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574820" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574820">
        <p class="paperTitle">Transcriptome-wide analysis of cell cycle-dependent bursty gene expression from single-cell RNA-seq data using mechanistic model-based inference</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574820" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574820" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sukys, A.; Grima, R.</p>
        <p class="info">Score: 6.1, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574820' target='https://doi.org/10.1101/2024.01.10.574820'> 10.1101/2024.01.10.574820</a></p>
        <p class="abstract">Bursty gene expression is quantified by two intuitive parameters: the burst frequency and the burst size. While these parameters are known to be cell-cycle dependent for some genes, a transcriptome-wide picture remains missing. Here we address this question by fitting a suite of mechanistic models of gene expression to mRNA count data for thousands of mouse genes, obtained by sequencing of single cells for which the cell-cycle position has been inferred using a deep-learning approach. This leads to the estimation of the burst frequency and size per allele in the G1 and G2/M cell-cycle phases, hence providing insight into the global patterns of transcriptional regulation. In particular, we identify an interesting balancing mechanism: on average, upon DNA replication, the burst frequency decreases by{approx} 50%, while the burst size increases by the same amount. We also show that for accurate estimation of the ratio of burst parameters in the G1 and G2/M phases, mechanistic models must explicitly account for gene copy number differences between cells but, surprisingly, additional corrections for extrinsic noise due to the coupling of transcription to cell age within the cell cycle or technical noise due to imperfect capture of RNA molecules in sequencing experiments are unnecessary.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575010">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575010" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575010">
        <p class="paperTitle">Enhanced feature matching in single-cell proteomics characterizes response to IFN-gamma and reveals co-existence of different cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575010" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575010" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krull, K. K.; Ali, S. A.; Krijgsveld, J.</p>
        <p class="info">Score: 2.5, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575010' target='https://doi.org/10.1101/2024.01.10.575010'> 10.1101/2024.01.10.575010</a></p>
        <p class="abstract">Proteome analysis by data-independent acquisition (DIA) has become a powerful approach to obtain deep proteome coverage, and has gained recent traction for label-free analysis of single cells. However, optimal experimental design for DIA-based single-cell proteomics has not been fully explored, and performance metrics of subsequent data analysis tools remain to be evaluated. Therefore, we here present DIA-ME, a data analysis strategy that exploits the co-analysis of low-input samples with a so-called matching enhancer (ME) of higher input, to increase sensitivity, proteome coverage, and data completeness. We evaluate the matching specificity of DIA-ME by a two-proteome model, and demonstrate that false discovery and false transfer are maintained at low levels when using DIA-NN software, while preserving quantification accuracy. We apply DIA-ME to investigate the proteome response of U-2 OS cells to interferon gamma (IFN-{gamma}) in single cells, and recapitulate the time-resolved induction of IFN-{gamma} response proteins as observed in bulk material. Moreover, we observe co- and anti-correlating patterns of protein expression within the same cell, indicating mutually exclusive protein modules and the co-existence of different cell states. Collectively our data show that DIA-ME is a powerful, scalable, and easy-to- implement strategy for single-cell proteomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.575500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.575500" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.575500">
        <p class="paperTitle">Targeted Perturb-seq Reveals EGR1 and FOS as Key Regulators of the Transcriptional RAF-MAPK Response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.575500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.575500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: El Kassem, G.; Sieber, A.; Klinger, B.; Uhlitz, F.; Steinbrecht, D.; van Bentum, M.; Hillmer, J.; von Schlichting, J.; Schäfer, R.; Blüthgen, N.; Boettcher, M.</p>
        <p class="info">Score: 1.9, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.575500' target='https://doi.org/10.1101/2024.01.13.575500'> 10.1101/2024.01.13.575500</a></p>
        <p class="abstract">The MAPK pathway is an important cellular signaling cascade whose dysregulation causes a variety of diseases. While the upstream regulators of this cascade have been extensively characterized, the understanding of how its activation translates into different transcriptional responses remains poorly understood. This study attempts to fill this knowledge gap by using targeted Perturb-seq against 22 transcription factors in an inducible model system for RAF-MAPK signaling. A topology-based modeling approach is applied to the obtained data to construct a directional interaction network. By removing coherent feed-forward loops and integrating the expression kinetics of transcription factors, a parsimonious network structure is derived that distinguishes direct from indirect interactions between the investigated transcription factors and their targets. In particular, EGR1 and FOS are found to act as orthogonal upstream regulators of the RAF-MAPK response. The results presented here provide valuable insights into the organization of the transcriptional network downstream of RAF-MAPK signaling and thus provide a basis for a better understanding of this complex process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569529">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569529" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569529">
        <p class="paperTitle">Epistemic uncertainty challenges aging clock reliability in predicting rejuvenation effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569529" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569529" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kriukov, D.; Kuzmina, E. A.; Efimov, E.; Dylov, D. V.; Khrameeva, E.</p>
        <p class="info">Score: 6.9, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569529' target='https://doi.org/10.1101/2023.12.01.569529'> 10.1101/2023.12.01.569529</a></p>
        <p class="abstract">Epigenetic aging clocks have been widely used to validate rejuvenation effects during cellular reprogramming. However, these predictions are unverifiable because the true biological age of reprogrammed cells remains unknown. We present an analytical framework to consider rejuvenation predictions from the uncertainty perspective. Our analysis reveals that the DNA methylation profiles across reprogramming are poorly represented in the aging data used to train clock models, thus introducing high epistemic uncertainty in age estimations. Moreover, predictions of different published clocks are inconsistent, with some even suggesting zero or negative rejuvenation. While not questioning the possibility of age reversal, we show that the high clock uncertainty challenges the reliability of rejuvenation effects observed during in vitro reprogramming prior to pluripotency and throughout embryogenesis. Conversely, our method reveals a significant age increase after in vivo reprogramming. We recommend including uncertainty estimation in future aging clock models to avoid the risk of misinterpreting the results of biological age prediction.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
