<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Unraveling the oscillatory dynamics of mRNA metabolism and chromatin accessibility during the cell cycle through integration of single-cell multiomic data
Authors: Nariya, M. K.; Santiago-Algarra, D.; Tassy, O.; Cerciat, M.; Ye, T.; Riba, A.; Molina, N.
Score: 14.5, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575159
The cell cycle is a highly regulated process that ensures the accurate replication and transmission of genetic information from one generation of cells to the next. It is a fundamental biological process and plays a crucial role in development, growth, and maintenance of all living organisms, and its dysregulation can lead to a number of pathologies such as autoimmune diseases, neurodegenerative diseases, and cancer.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Unraveling the oscillatory dynamics of mRNA metabolism and chromatin accessibility during the cell cycle through integration of single-cell multiomic data
Authors: Nariya, M. K.; Santiago-Algarra, D.; Tassy, O.; Cerciat, M.; Ye, T.; Riba, A.; Molina, N.
Score: 14.5, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575159
The cell cycle is a highly regulated process that ensures the accurate replication and transmission of genetic information from one generation of cells to the next. It is a fundamental biological process and plays a crucial role in development, growth, and maintenance of all living organisms, and its dysregulation can lead to a number of pathologies such as autoimmune diseases, neurodegenerative diseases, and cancer." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-21T10:39:08+00:00" />
<meta property="article:modified_time" content="2024-01-21T10:39:08+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Unraveling the oscillatory dynamics of mRNA metabolism and chromatin accessibility during the cell cycle through integration of single-cell multiomic data
Authors: Nariya, M. K.; Santiago-Algarra, D.; Tassy, O.; Cerciat, M.; Ye, T.; Riba, A.; Molina, N.
Score: 14.5, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575159
The cell cycle is a highly regulated process that ensures the accurate replication and transmission of genetic information from one generation of cells to the next. It is a fundamental biological process and plays a crucial role in development, growth, and maintenance of all living organisms, and its dysregulation can lead to a number of pathologies such as autoimmune diseases, neurodegenerative diseases, and cancer."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Unraveling the oscillatory dynamics of mRNA metabolism and chromatin accessibility during the cell cycle through integration of single-cell multiomic data\nAuthors: Nariya, M. K.; Santiago-Algarra, D.; Tassy, O.; Cerciat, M.; Ye, T.; Riba, A.; Molina, N.\nScore: 14.5, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575159\nThe cell cycle is a highly regulated process that ensures the accurate replication and transmission of genetic information from one generation of cells to the next. It is a fundamental biological process and plays a crucial role in development, growth, and maintenance of all living organisms, and its dysregulation can lead to a number of pathologies such as autoimmune diseases, neurodegenerative diseases, and cancer.",
  "keywords": [
    
  ],
  "articleBody": " Unraveling the oscillatory dynamics of mRNA metabolism and chromatin accessibility during the cell cycle through integration of single-cell multiomic data\nAuthors: Nariya, M. K.; Santiago-Algarra, D.; Tassy, O.; Cerciat, M.; Ye, T.; Riba, A.; Molina, N.\nScore: 14.5, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575159\nThe cell cycle is a highly regulated process that ensures the accurate replication and transmission of genetic information from one generation of cells to the next. It is a fundamental biological process and plays a crucial role in development, growth, and maintenance of all living organisms, and its dysregulation can lead to a number of pathologies such as autoimmune diseases, neurodegenerative diseases, and cancer. In this work we present a novel approach to study the gene expression and chromatin accessibility dynamics during the cell cycle in mouse embryonic stem cells. To achieve this, we combined high-depth single-cell multiome sequencing, biophysical modeling, and advanced deep learning techniques. First, we used DeepCycle, a deep learning tool that assigns a cell cycle phase to every cell based on its spliced and unspliced mRNA levels. We then developed a biophysical model that describes the dynamics of gene-specific mRNA transcription, nuclear export, and degradation rates during the cell cycle. Our model obtains genome-wide estimates of mRNA transcription, nuclear retention, and degradation half-lives of genes exhibiting oscillatory dynamics in mESCs. The key feature of single-cell multiome sequencing is that it simultaneously provides readouts for gene expression as well as chromatin accessibility in the same cells. By applying our approach to these data we showcase first-of-its-kind exploration of chromatin accessibility during the cell cycle at high temporal resolution. We believe that our work will be pivotal in formulating a coherent quantitative description of mRNA metabolism. HighlightsO_LIIntegration of single-cell and single-nucleus multiome profiling with deep learning and biophysical modeling uncovers cell cycle regulation of RNA metabolism C_LIO_LIGenome-wide estimation of mRNA transcription, nuclear export, and degradation rates during the cell cycle in mESCs C_LIO_LIIdentification of a prominent wave of transcription during early G1 in unperturbed asynchronously proliferating mESCs C_LIO_LICharacterization of temporal waves of post-transcriptional mRNA regulation controlling both mRNA degradation and nuclear export during the cell cycle C_LIO_LIChromatin accessibility dynamics within TF footprints reveal key cell cycle regulators C_LI\nBeyond microtubules: The cellular environment at the endoplasmic reticulum attracts proteins to the nucleus, enabling nuclear transport\nAuthors: Chae, S. J.; Kim, D. W.; Igoshin, O. A.; Lee, S.; Kim, J. K.\nScore: 7.7, Published: 2024-01-14 DOI: 10.1101/2024.01.12.575351\nAll proteins are translated in the cytoplasm, yet many, including transcription factors, play vital roles in the nucleus. While previous research has concentrated on molecular motors for the transport of these proteins to the nucleus, recent observations reveal perinuclear accumulation even in the absence of an energy source, hinting at alternative mechanisms. Here, we propose that structural properties of the cellular environment, specifically the endoplasmic reticulum (ER), can promote molecular transport to the perinucleus without requiring additional energy expenditure. Specifically, physical interaction between proteins and the ER impedes their diffusion and leads to their accumulation near the nucleus. This result explains why larger proteins, more frequently interacting with the ER membrane, tend to accumulate at the perinucleus. Interestingly, such diffusion in a heterogeneous environment follows Chapmans law rather than the popular Ficks law. Our findings suggest a novel protein transport mechanism arising solely from characteristics of the intracellular environment. HighlightsO_LIThe interaction of proteins with ER slows down their diffusion at the perinucleus. C_LIO_LIThis leads proteins to migrate toward the perinucleus without ATP consumption. C_LIO_LIFrequent ER interaction of larger proteins promotes perinuclear accumulation. C_LIO_LIDiffusion with the physical interaction can be described by Chapmans law. C_LI Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=194 SRC=\"FIGDIR/small/575351v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (41K): org.highwire.dtl.DTLVardef@79d915org.highwire.dtl.DTLVardef@19b0675org.highwire.dtl.DTLVardef@c51842org.highwire.dtl.DTLVardef@1bb9679_HPS_FORMAT_FIGEXP M_FIG C_FIG\nTargeted Perturb-seq Reveals EGR1 and FOS as Key Regulators of the Transcriptional RAF-MAPK Response\nAuthors: El Kassem, G.; Sieber, A.; Klinger, B.; Uhlitz, F.; Steinbrecht, D.; van Bentum, M.; Hillmer, J.; von Schlichting, J.; Schäfer, R.; Blüthgen, N.; Boettcher, M.\nScore: 5.0, Published: 2024-01-15 DOI: 10.1101/2024.01.13.575500\nThe MAPK pathway is an important cellular signaling cascade whose dysregulation causes a variety of diseases. While the upstream regulators of this cascade have been extensively characterized, the understanding of how its activation translates into different transcriptional responses remains poorly understood. This study attempts to fill this knowledge gap by using targeted Perturb-seq against 22 transcription factors in an inducible model system for RAF-MAPK signaling. A topology-based modeling approach is applied to the obtained data to construct a directional interaction network. By removing coherent feed-forward loops and integrating the expression kinetics of transcription factors, a parsimonious network structure is derived that distinguishes direct from indirect interactions between the investigated transcription factors and their targets. In particular, EGR1 and FOS are found to act as orthogonal upstream regulators of the RAF-MAPK response. The results presented here provide valuable insights into the organization of the transcriptional network downstream of RAF-MAPK signaling and thus provide a basis for a better understanding of this complex process.\nUnraveling principles of thermodynamics for genome-scale metabolic networks using graph neural networks\nAuthors: Fan, W.; Ding, C.; Huang, D.; Zheng, W.; Dai, Z.\nScore: 3.8, Published: 2024-01-17 DOI: 10.1101/2024.01.15.575679\nThe fundamental laws of thermodynamics determine the feasibility of all natural processes including metabolism. Although several algorithms have been developed to predict the most important thermodynamic parameter, the standard Gibbs free energy, for metabolic reactions and metabolites, their application to genome-scale metabolic networks (GEMs) with thousands of metabolites and reactions is still limited. Here, we develop a graph neural network (GNN)- based model dGbyG for predicting Gibbs energy for metabolites and metabolic reactions which outperforms all existing methods in accuracy, versatility, robustness, and generalization ability. By applying dGbyG to the human GEM, Recon3D, we identify a critical subset of reactions with substantial negative values of the standard Gibbs free energy change, which we name thermodynamic driver reactions. These reactions exhibit distinctive network topological characteristics akin to driver nodes defined in control theory and remarkable heterogeneity in enzyme abundance, implying evolutionary constraints on the selection of thermodynamic parameters of metabolic networks. We also develop a thermodynamics-based flux balance analysis (TFBA) approach to integrate reaction thermodynamics with GEMs to curate these models. Our work not only transcends the augmentation of accessible thermodynamic data to facilitate an enriched understanding of metabolism, but also enables refinement of metabolic reconstructions from a thermodynamic perspective, thereby underscoring the critical role of thermodynamics in the accurate modeling of biological systems.\nLizardNet: A mobile hybrid deep learning tool for classification of 3D representations of Amazonian lizards\nAuthors: da Silva, A. G.; de Oliveira, R. P.; de Carvalho, E. A.; Bastos, C. d. O.; Gomes, B. D.\nScore: 2.8, Published: 2024-01-17 DOI: 10.1101/2024.01.15.575627\nImage classification is a highly significant field in machine learning (ML), especially when applied to address longstanding and challenging issues in the biological sciences. In this study, we present the development of a hybrid deep learning-based tool suitable for deployment on mobile devices. This tool is aimed at processing and classifying three-dimensional samples of endemic lizard species from the Amazon rainforest. The dataset used in our experiment was collected at the Museu Paraense Emilio Goeldi (MPEG), Belem-PA, Brazil, and comprises three species: a) Anolis fuscoauratus; b) Hoplocercus spinosus; and c) Polychrus marmoratus. We compared the effectiveness of four artificial neural networks (ANN) for feature extraction: a) MobileNet; b) MobileNetV2; c) MobileNetV3Small; and d) MobileNetV3Large. Additionally, we evaluated five classical ML models for classifying the extracted patterns: a) Support Vector Machine (SVM); b) GaussianNB (GNB); c) AdaBoost (ADB); d) K-Nearest Neighbors (KNN); and e) Random Forest (RF). Our most effective model, MobileNetV3-Small + Linear SVM, achieved an accuracy of 0.948 and a f1-score of 0.955. Notably, it not only proved to be the least complex model among all combinations but also demonstrated the best performance after a statistical comparison. These results indicate that the combination of deep learning (DL) models with less complex classical ML algorithms, which have a lower error propensity, emerges as a viable and efficient technique for classifying three-dimensional lizard species samples. Such an approach facilitates taxonomic identification work for professionals in the field and provides a tool adaptable for integration into mobile data recording equipment, such as smartphones. Author summaryThe taxonomic classification of lizards requires an exceptional level of knowledge and attention to minute details beyond the ordinary to accurately categorize specimens. Such tasks impose significant mental and visual costs on humans, unlike computer vision algorithms capable of extracting visual patterns from images imperceptible to the human eye. In this research, we utilized a dataset from the herpetarium of the Emilio Goeldi Museum in Belem-PA, Brazil. The data were self-captured, with each sample comprised of three photos: dorsal, lateral, and ventral views of each specimen. The sample size was constrained by the quality and abundance of preserved specimens, necessitating the application of a data augmentation method on the pre-separated training and validation sets. This augmentation led to a considerable increase in the number of samples per species, from a few dozen to several hundred. Our experimental approach involved utilizing pre-trained neural networks to extract 3D sample characteristics, subsequently classified using classical machine learning algorithms. This hybrid strategy was adopted due to the nature of data collection and synthetic data augmentation. Our method enables specimen identification through three-dimensional representations, allowing for a more comprehensive utilization of morphological information by the model.\nEpistemic uncertainty challenges aging clock reliability in predicting rejuvenation effects\nAuthors: Kriukov, D.; Kuzmina, E. A.; Efimov, E.; Dylov, D. V.; Khrameeva, E.\nScore: 6.9, Published: 2023-12-30 DOI: 10.1101/2023.12.01.569529\nEpigenetic aging clocks have been widely used to validate rejuvenation effects during cellular reprogramming. However, these predictions are unverifiable because the true biological age of reprogrammed cells remains unknown. We present an analytical framework to consider rejuvenation predictions from the uncertainty perspective. Our analysis reveals that the DNA methylation profiles across reprogramming are poorly represented in the aging data used to train clock models, thus introducing high epistemic uncertainty in age estimations. Moreover, predictions of different published clocks are inconsistent, with some even suggesting zero or negative rejuvenation. While not questioning the possibility of age reversal, we show that the high clock uncertainty challenges the reliability of rejuvenation effects observed during in vitro reprogramming prior to pluripotency and throughout embryogenesis. Conversely, our method reveals a significant age increase after in vivo reprogramming. We recommend including uncertainty estimation in future aging clock models to avoid the risk of misinterpreting the results of biological age prediction.\nProteome-scale tagging and functional screening in mammalian cells by ORFtag\nAuthors: Nemcko, F.; Himmelsbach, M.; Loubiere, V.; Yelagandula, R.; Pagani, M.; Brennecke, J.; Elling, U.; Stark, A.; Ameres, S. L.\nScore: 1.0, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575827\nDetermining protein function in a systematic manner is a key goal of modern biology, but remains challenging with current approaches. Here, we present ORFtag, a versatile, cost-effective and highly efficient method for the massively-parallel tagging and functional interrogation of proteins at proteome scale. Using mouse embryonic stem cells, we showcase ORFtag's utility through screens for transcriptional activators, repressors and post-transcriptional regulators. Each screen finds known and novel regulators, including long ORFs not accessible to other methods, revealing that Zfp574 is a highly selective transcriptional activator and that oncogenic fusions frequently function as transactivators.\nA quantitative description of light-limited cyanobacterial growth using flux balance analysis\nAuthors: Hoeper, R.; Komkova, D.; Zavrel, T.; Steuer, R.\nScore: 8.7, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574812\nThe metabolism of phototrophic cyanobacterial is an integral part of global biogeochemical cycles, and the capability of cyanobacteria to assimilate atmospheric CO2 into organic carbon has manifold potential applications for a sustainable biotechnology. To elucidate the properties of cyanobacterial metabolism and growth, computational reconstructions of the genome-scale metabolic networks play an increasingly important role. Here, we present an updated reconstruction of the metabolic network of the cyanobacterium Synechocystis sp. PCC 6803 and its analysis using flux balance analysis (FBA). To overcome limitations of conventional FBA, and to allow for the integration of quantitative experimental analyses, we develop a novel approach to describe light absorption and light utilization. Our approach incorporates photoinhibition and a variable quantum yield into the constraint-based description of light-limited phototrophic growth. We show that the resulting model is capable to predict quantitative properties of cyanobacterial growth, including photosynthetic oxygen evolution and the ATP/NADPH ratio required for growth and cellular maintenance. Our approach retains the computational and conceptual simplicity of FBA and is readily applicable to other phototropic microorganisms.\nTranscriptome-wide analysis of cell cycle-dependent bursty gene expression from single-cell RNA-seq data using mechanistic model-based inference\nAuthors: Sukys, A.; Grima, R.\nScore: 6.1, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574820\nBursty gene expression is quantified by two intuitive parameters: the burst frequency and the burst size. While these parameters are known to be cell-cycle dependent for some genes, a transcriptome-wide picture remains missing. Here we address this question by fitting a suite of mechanistic models of gene expression to mRNA count data for thousands of mouse genes, obtained by sequencing of single cells for which the cell-cycle position has been inferred using a deep-learning approach. This leads to the estimation of the burst frequency and size per allele in the G1 and G2/M cell-cycle phases, hence providing insight into the global patterns of transcriptional regulation. In particular, we identify an interesting balancing mechanism: on average, upon DNA replication, the burst frequency decreases by{approx} 50%, while the burst size increases by the same amount. We also show that for accurate estimation of the ratio of burst parameters in the G1 and G2/M phases, mechanistic models must explicitly account for gene copy number differences between cells but, surprisingly, additional corrections for extrinsic noise due to the coupling of transcription to cell age within the cell cycle or technical noise due to imperfect capture of RNA molecules in sequencing experiments are unnecessary.\nExploring human rare disease variants from a multidimensional perspective illuminates receptor - G protein coupling diversity\nAuthors: Redfern-Nichols, T. B.; O'Brien, S. L.; Huang, X.; Medel-Lacruz, B.; Calebiro, D.; Selent, J.; Ladds, G.; Marti Solano, M.\nScore: 0.8, Published: 2024-01-17 DOI: 10.1101/2024.01.16.575841\nG protein-coupled receptors (GPCRs) are transmembrane proteins capable of detecting signals as diverse as odours, neurotransmitters, and hormones. Upon activation, receptor signalling converges onto four G protein subtypes to regulate intracellular responses. Therefore, variation in a single G protein gene can potentially impact the function of numerous receptors. In this work, we have performed a multidimensional study of rare disease mutations in Gs, a prototypical G protein. By integrating data from 3D structures, GPCR / G protein functional pairings, transcriptomics, biophysics, and molecular dynamics with systems pharmacology modelling, our results reveal why mutations impairing receptor / Gs coupling result in highly specific context-based signalling defects. Furthermore, we show that mutations leading to the same rare disease can alter different signal transduction steps, highlighting the importance of patient-specific treatment strategies. By closely dissecting G protein coupling, our study provides a blueprint to interrogate GPCR pathway signalling diversity in different (patho)physiological contexts.\n",
  "wordCount" : "2471",
  "inLanguage": "en",
  "datePublished": "2024-01-21T10:39:08Z",
  "dateModified": "2024-01-21T10:39:08Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 21, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575159">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575159" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575159">
        <p class="paperTitle">Unraveling the oscillatory dynamics of mRNA metabolism and chromatin accessibility during the cell cycle through integration of single-cell multiomic data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575159" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575159" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nariya, M. K.; Santiago-Algarra, D.; Tassy, O.; Cerciat, M.; Ye, T.; Riba, A.; Molina, N.</p>
        <p class="info">Score: 14.5, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575159' target='https://doi.org/10.1101/2024.01.11.575159'> 10.1101/2024.01.11.575159</a></p>
        <p class="abstract">The cell cycle is a highly regulated process that ensures the accurate replication and transmission of genetic information from one generation of cells to the next. It is a fundamental biological process and plays a crucial role in development, growth, and maintenance of all living organisms, and its dysregulation can lead to a number of pathologies such as autoimmune diseases, neurodegenerative diseases, and cancer. In this work we present a novel approach to study the gene expression and chromatin accessibility dynamics during the cell cycle in mouse embryonic stem cells. To achieve this, we combined high-depth single-cell multiome sequencing, biophysical modeling, and advanced deep learning techniques. First, we used DeepCycle, a deep learning tool that assigns a cell cycle phase to every cell based on its spliced and unspliced mRNA levels. We then developed a biophysical model that describes the dynamics of gene-specific mRNA transcription, nuclear export, and degradation rates during the cell cycle. Our model obtains genome-wide estimates of mRNA transcription, nuclear retention, and degradation half-lives of genes exhibiting oscillatory dynamics in mESCs. The key feature of single-cell multiome sequencing is that it simultaneously provides readouts for gene expression as well as chromatin accessibility in the same cells. By applying our approach to these data we showcase first-of-its-kind exploration of chromatin accessibility during the cell cycle at high temporal resolution. We believe that our work will be pivotal in formulating a coherent quantitative description of mRNA metabolism.

HighlightsO_LIIntegration of single-cell and single-nucleus multiome profiling with deep learning and biophysical modeling uncovers cell cycle regulation of RNA metabolism
C_LIO_LIGenome-wide estimation of mRNA transcription, nuclear export, and degradation rates during the cell cycle in mESCs
C_LIO_LIIdentification of a prominent wave of transcription during early G1 in unperturbed asynchronously proliferating mESCs
C_LIO_LICharacterization of temporal waves of post-transcriptional mRNA regulation controlling both mRNA degradation and nuclear export during the cell cycle
C_LIO_LIChromatin accessibility dynamics within TF footprints reveal key cell cycle regulators
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575351">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575351" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575351">
        <p class="paperTitle">Beyond microtubules: The cellular environment at the endoplasmic reticulum attracts proteins to the nucleus, enabling nuclear transport</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575351" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575351" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chae, S. J.; Kim, D. W.; Igoshin, O. A.; Lee, S.; Kim, J. K.</p>
        <p class="info">Score: 7.7, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575351' target='https://doi.org/10.1101/2024.01.12.575351'> 10.1101/2024.01.12.575351</a></p>
        <p class="abstract">All proteins are translated in the cytoplasm, yet many, including transcription factors, play vital roles in the nucleus. While previous research has concentrated on molecular motors for the transport of these proteins to the nucleus, recent observations reveal perinuclear accumulation even in the absence of an energy source, hinting at alternative mechanisms. Here, we propose that structural properties of the cellular environment, specifically the endoplasmic reticulum (ER), can promote molecular transport to the perinucleus without requiring additional energy expenditure. Specifically, physical interaction between proteins and the ER impedes their diffusion and leads to their accumulation near the nucleus. This result explains why larger proteins, more frequently interacting with the ER membrane, tend to accumulate at the perinucleus. Interestingly, such diffusion in a heterogeneous environment follows Chapmans law rather than the popular Ficks law. Our findings suggest a novel protein transport mechanism arising solely from characteristics of the intracellular environment.

HighlightsO_LIThe interaction of proteins with ER slows down their diffusion at the perinucleus.
C_LIO_LIThis leads proteins to migrate toward the perinucleus without ATP consumption.
C_LIO_LIFrequent ER interaction of larger proteins promotes perinuclear accumulation.
C_LIO_LIDiffusion with the physical interaction can be described by Chapmans law.
C_LI

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=194 SRC=&#34;FIGDIR/small/575351v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (41K):
org.highwire.dtl.DTLVardef@79d915org.highwire.dtl.DTLVardef@19b0675org.highwire.dtl.DTLVardef@c51842org.highwire.dtl.DTLVardef@1bb9679_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.575500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.575500" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.575500">
        <p class="paperTitle">Targeted Perturb-seq Reveals EGR1 and FOS as Key Regulators of the Transcriptional RAF-MAPK Response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.575500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.575500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: El Kassem, G.; Sieber, A.; Klinger, B.; Uhlitz, F.; Steinbrecht, D.; van Bentum, M.; Hillmer, J.; von Schlichting, J.; Schäfer, R.; Blüthgen, N.; Boettcher, M.</p>
        <p class="info">Score: 5.0, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.575500' target='https://doi.org/10.1101/2024.01.13.575500'> 10.1101/2024.01.13.575500</a></p>
        <p class="abstract">The MAPK pathway is an important cellular signaling cascade whose dysregulation causes a variety of diseases. While the upstream regulators of this cascade have been extensively characterized, the understanding of how its activation translates into different transcriptional responses remains poorly understood. This study attempts to fill this knowledge gap by using targeted Perturb-seq against 22 transcription factors in an inducible model system for RAF-MAPK signaling. A topology-based modeling approach is applied to the obtained data to construct a directional interaction network. By removing coherent feed-forward loops and integrating the expression kinetics of transcription factors, a parsimonious network structure is derived that distinguishes direct from indirect interactions between the investigated transcription factors and their targets. In particular, EGR1 and FOS are found to act as orthogonal upstream regulators of the RAF-MAPK response. The results presented here provide valuable insights into the organization of the transcriptional network downstream of RAF-MAPK signaling and thus provide a basis for a better understanding of this complex process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575679">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575679" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575679">
        <p class="paperTitle">Unraveling principles of thermodynamics for genome-scale metabolic networks using graph neural networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575679" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575679" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fan, W.; Ding, C.; Huang, D.; Zheng, W.; Dai, Z.</p>
        <p class="info">Score: 3.8, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575679' target='https://doi.org/10.1101/2024.01.15.575679'> 10.1101/2024.01.15.575679</a></p>
        <p class="abstract">The fundamental laws of thermodynamics determine the feasibility of all natural processes including metabolism. Although several algorithms have been developed to predict the most important thermodynamic parameter, the standard Gibbs free energy, for metabolic reactions and metabolites, their application to genome-scale metabolic networks (GEMs) with thousands of metabolites and reactions is still limited. Here, we develop a graph neural network (GNN)- based model dGbyG for predicting Gibbs energy for metabolites and metabolic reactions which outperforms all existing methods in accuracy, versatility, robustness, and generalization ability. By applying dGbyG to the human GEM, Recon3D, we identify a critical subset of reactions with substantial negative values of the standard Gibbs free energy change, which we name thermodynamic driver reactions. These reactions exhibit distinctive network topological characteristics akin to driver nodes defined in control theory and remarkable heterogeneity in enzyme abundance, implying evolutionary constraints on the selection of thermodynamic parameters of metabolic networks. We also develop a thermodynamics-based flux balance analysis (TFBA) approach to integrate reaction thermodynamics with GEMs to curate these models. Our work not only transcends the augmentation of accessible thermodynamic data to facilitate an enriched understanding of metabolism, but also enables refinement of metabolic reconstructions from a thermodynamic perspective, thereby underscoring the critical role of thermodynamics in the accurate modeling of biological systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575627">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575627" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575627">
        <p class="paperTitle">LizardNet: A mobile hybrid deep learning tool for classification of 3D representations of Amazonian lizards</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575627" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575627" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: da Silva, A. G.; de Oliveira, R. P.; de Carvalho, E. A.; Bastos, C. d. O.; Gomes, B. D.</p>
        <p class="info">Score: 2.8, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575627' target='https://doi.org/10.1101/2024.01.15.575627'> 10.1101/2024.01.15.575627</a></p>
        <p class="abstract">Image classification is a highly significant field in machine learning (ML), especially when applied to address longstanding and challenging issues in the biological sciences. In this study, we present the development of a hybrid deep learning-based tool suitable for deployment on mobile devices. This tool is aimed at processing and classifying three-dimensional samples of endemic lizard species from the Amazon rainforest. The dataset used in our experiment was collected at the Museu Paraense Emilio Goeldi (MPEG), Belem-PA, Brazil, and comprises three species: a) Anolis fuscoauratus; b) Hoplocercus spinosus; and c) Polychrus marmoratus. We compared the effectiveness of four artificial neural networks (ANN) for feature extraction: a) MobileNet; b) MobileNetV2; c) MobileNetV3Small; and d) MobileNetV3Large. Additionally, we evaluated five classical ML models for classifying the extracted patterns: a) Support Vector Machine (SVM); b) GaussianNB (GNB); c) AdaBoost (ADB); d) K-Nearest Neighbors (KNN); and e) Random Forest (RF). Our most effective model, MobileNetV3-Small &#43; Linear SVM, achieved an accuracy of 0.948 and a f1-score of 0.955. Notably, it not only proved to be the least complex model among all combinations but also demonstrated the best performance after a statistical comparison. These results indicate that the combination of deep learning (DL) models with less complex classical ML algorithms, which have a lower error propensity, emerges as a viable and efficient technique for classifying three-dimensional lizard species samples. Such an approach facilitates taxonomic identification work for professionals in the field and provides a tool adaptable for integration into mobile data recording equipment, such as smartphones.

Author summaryThe taxonomic classification of lizards requires an exceptional level of knowledge and attention to minute details beyond the ordinary to accurately categorize specimens. Such tasks impose significant mental and visual costs on humans, unlike computer vision algorithms capable of extracting visual patterns from images imperceptible to the human eye. In this research, we utilized a dataset from the herpetarium of the Emilio Goeldi Museum in Belem-PA, Brazil. The data were self-captured, with each sample comprised of three photos: dorsal, lateral, and ventral views of each specimen. The sample size was constrained by the quality and abundance of preserved specimens, necessitating the application of a data augmentation method on the pre-separated training and validation sets. This augmentation led to a considerable increase in the number of samples per species, from a few dozen to several hundred. Our experimental approach involved utilizing pre-trained neural networks to extract 3D sample characteristics, subsequently classified using classical machine learning algorithms. This hybrid strategy was adopted due to the nature of data collection and synthetic data augmentation. Our method enables specimen identification through three-dimensional representations, allowing for a more comprehensive utilization of morphological information by the model.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569529">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569529" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569529">
        <p class="paperTitle">Epistemic uncertainty challenges aging clock reliability in predicting rejuvenation effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569529" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569529" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kriukov, D.; Kuzmina, E. A.; Efimov, E.; Dylov, D. V.; Khrameeva, E.</p>
        <p class="info">Score: 6.9, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569529' target='https://doi.org/10.1101/2023.12.01.569529'> 10.1101/2023.12.01.569529</a></p>
        <p class="abstract">Epigenetic aging clocks have been widely used to validate rejuvenation effects during cellular reprogramming. However, these predictions are unverifiable because the true biological age of reprogrammed cells remains unknown. We present an analytical framework to consider rejuvenation predictions from the uncertainty perspective. Our analysis reveals that the DNA methylation profiles across reprogramming are poorly represented in the aging data used to train clock models, thus introducing high epistemic uncertainty in age estimations. Moreover, predictions of different published clocks are inconsistent, with some even suggesting zero or negative rejuvenation. While not questioning the possibility of age reversal, we show that the high clock uncertainty challenges the reliability of rejuvenation effects observed during in vitro reprogramming prior to pluripotency and throughout embryogenesis. Conversely, our method reveals a significant age increase after in vivo reprogramming. We recommend including uncertainty estimation in future aging clock models to avoid the risk of misinterpreting the results of biological age prediction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575827">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575827" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575827">
        <p class="paperTitle">Proteome-scale tagging and functional screening in mammalian cells by ORFtag</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575827" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575827" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nemcko, F.; Himmelsbach, M.; Loubiere, V.; Yelagandula, R.; Pagani, M.; Brennecke, J.; Elling, U.; Stark, A.; Ameres, S. L.</p>
        <p class="info">Score: 1.0, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575827' target='https://doi.org/10.1101/2024.01.16.575827'> 10.1101/2024.01.16.575827</a></p>
        <p class="abstract">Determining protein function in a systematic manner is a key goal of modern biology, but remains challenging with current approaches. Here, we present ORFtag, a versatile, cost-effective and highly efficient method for the massively-parallel tagging and functional interrogation of proteins at proteome scale. Using mouse embryonic stem cells, we showcase ORFtag&#39;s utility through screens for transcriptional activators, repressors and post-transcriptional regulators. Each screen finds known and novel regulators, including long ORFs not accessible to other methods, revealing that Zfp574 is a highly selective transcriptional activator and that oncogenic fusions frequently function as transactivators.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574812">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574812" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574812">
        <p class="paperTitle">A quantitative description of light-limited cyanobacterial growth using flux balance analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574812" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574812" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hoeper, R.; Komkova, D.; Zavrel, T.; Steuer, R.</p>
        <p class="info">Score: 8.7, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574812' target='https://doi.org/10.1101/2024.01.10.574812'> 10.1101/2024.01.10.574812</a></p>
        <p class="abstract">The metabolism of phototrophic cyanobacterial is an integral part of global biogeochemical cycles, and the capability of cyanobacteria to assimilate atmospheric CO2 into organic carbon has manifold potential applications for a sustainable biotechnology. To elucidate the properties of cyanobacterial metabolism and growth, computational reconstructions of the genome-scale metabolic networks play an increasingly important role. Here, we present an updated reconstruction of the metabolic network of the cyanobacterium Synechocystis sp. PCC 6803 and its analysis using flux balance analysis (FBA). To overcome limitations of conventional FBA, and to allow for the integration of quantitative experimental analyses, we develop a novel approach to describe light absorption and light utilization. Our approach incorporates photoinhibition and a variable quantum yield into the constraint-based description of light-limited phototrophic growth. We show that the resulting model is capable to predict quantitative properties of cyanobacterial growth, including photosynthetic oxygen evolution and the ATP/NADPH ratio required for growth and cellular maintenance. Our approach retains the computational and conceptual simplicity of FBA and is readily applicable to other phototropic microorganisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574820">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574820" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574820">
        <p class="paperTitle">Transcriptome-wide analysis of cell cycle-dependent bursty gene expression from single-cell RNA-seq data using mechanistic model-based inference</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574820" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574820" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sukys, A.; Grima, R.</p>
        <p class="info">Score: 6.1, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574820' target='https://doi.org/10.1101/2024.01.10.574820'> 10.1101/2024.01.10.574820</a></p>
        <p class="abstract">Bursty gene expression is quantified by two intuitive parameters: the burst frequency and the burst size. While these parameters are known to be cell-cycle dependent for some genes, a transcriptome-wide picture remains missing. Here we address this question by fitting a suite of mechanistic models of gene expression to mRNA count data for thousands of mouse genes, obtained by sequencing of single cells for which the cell-cycle position has been inferred using a deep-learning approach. This leads to the estimation of the burst frequency and size per allele in the G1 and G2/M cell-cycle phases, hence providing insight into the global patterns of transcriptional regulation. In particular, we identify an interesting balancing mechanism: on average, upon DNA replication, the burst frequency decreases by{approx} 50%, while the burst size increases by the same amount. We also show that for accurate estimation of the ratio of burst parameters in the G1 and G2/M phases, mechanistic models must explicitly account for gene copy number differences between cells but, surprisingly, additional corrections for extrinsic noise due to the coupling of transcription to cell age within the cell cycle or technical noise due to imperfect capture of RNA molecules in sequencing experiments are unnecessary.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575841">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575841" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575841">
        <p class="paperTitle">Exploring human rare disease variants from a multidimensional perspective illuminates receptor - G protein coupling diversity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575841" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575841" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Redfern-Nichols, T. B.; O&#39;Brien, S. L.; Huang, X.; Medel-Lacruz, B.; Calebiro, D.; Selent, J.; Ladds, G.; Marti Solano, M.</p>
        <p class="info">Score: 0.8, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575841' target='https://doi.org/10.1101/2024.01.16.575841'> 10.1101/2024.01.16.575841</a></p>
        <p class="abstract">G protein-coupled receptors (GPCRs) are transmembrane proteins capable of detecting signals as diverse as odours, neurotransmitters, and hormones. Upon activation, receptor signalling converges onto four G protein subtypes to regulate intracellular responses. Therefore, variation in a single G protein gene can potentially impact the function of numerous receptors. In this work, we have performed a multidimensional study of rare disease mutations in Gs, a prototypical G protein. By integrating data from 3D structures, GPCR / G protein functional pairings, transcriptomics, biophysics, and molecular dynamics with systems pharmacology modelling, our results reveal why mutations impairing receptor / Gs coupling result in highly specific context-based signalling defects. Furthermore, we show that mutations leading to the same rare disease can alter different signal transduction steps, highlighting the importance of patient-specific treatment strategies. By closely dissecting G protein coupling, our study provides a blueprint to interrogate GPCR pathway signalling diversity in different (patho)physiological contexts.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
