<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)
Authors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.
Score: 22.3, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546628
Combined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)
Authors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.
Score: 22.3, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546628
Combined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-05T10:42:43+00:00" />
<meta property="article:modified_time" content="2023-07-05T10:42:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)
Authors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.
Score: 22.3, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546628
Combined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)\nAuthors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.\nScore: 22.3, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546628\nCombined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts.",
  "keywords": [
    
  ],
  "articleBody": " Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)\nAuthors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.\nScore: 22.3, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546628\nCombined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts. Here, we present an innovative strategy for simultaneous multi-omics analysis by direct infusion (SMAD) using one single injection without liquid chromatography. SMAD allows quantification of over 9,000 metabolite m/z features and over 1,300 proteins from the same sample in less than five minutes. We validated the efficiency and reliability of this method and then present two practical applications: mouse macrophage M1/M2 polarization and high throughput drug screening in human 293T cells. Finally, we demonstrate relationships between proteomic and metabolomic data are discovered by machine learning.\nMassively integrated coexpression analysis reveals transcriptional regulation, evolution and cellular implications of the noncanonical translatome\nAuthors: Rich, A.; Acar, O.; Carvunis, A.-R.\nScore: 17.3, Published: 2023-06-22 DOI: 10.1101/2023.03.16.533058\nBackgroundRecent studies uncovered pervasive transcription and translation of thousands of noncanonical open reading frames (nORFs) outside of annotated genes. The contribution of nORFs to cellular phenotypes is difficult to infer using conventional approaches because nORFs tend to be short, of recent de novo origins, and lowly expressed. Here we develop a dedicated coexpression analysis framework that accounts for low expression to investigate the transcriptional regulation, evolution, and potential cellular roles of nORFs in Saccharomyces cerevisiae. ResultsOur results reveal that nORFs tend to be preferentially coexpressed with genes involved in cellular transport or homeostasis but rarely with genes involved in RNA processing. Mechanistically, we discover that young de novo nORFs located downstream of conserved genes tend to leverage their neighbors promoters through transcription readthrough, resulting in high coexpression and high expression levels. Transcriptional piggybacking also influences the coexpression profiles of young de novo nORFs located upstream of genes, but to a lesser extent and without detectable impact on expression levels. Transcriptional piggybacking influences, but does not determine, the transcription profiles of de novo nORFs emerging nearby genes. About 40% of nORFs are not strongly coexpressed with any gene but are transcriptionally regulated nonetheless and tend to form entirely new transcription modules. We offer a web browser interface (https://carvunislab.csb.pitt.edu/shiny/coexpression/) to efficiently query, visualize and download our coexpression inferences. ConclusionsOur results suggest that nORF transcription is highly regulated. Our coexpression dataset serves as an unprecedented resource for unraveling how nORFs integrate into cellular networks, contribute to cellular phenotypes, and evolve.\nOne thousand samples per day capillary-flow LC/MS/MS for high-speed, high-sensitivity and in-depth proteomics\nAuthors: Tomioka, A.; Tomioka, R.; Mori, I.; Isobe, Y.; Arita, M.; Imami, K.; Kanao, E.; Ogata, K.; Ishihama, Y.\nScore: 22.0, Published: 2023-06-07 DOI: 10.1101/2023.06.05.543682\nWe developed a capillary-flow LC/MS/MS system with ultrahigh speed, enabling a throughput of 1,000 samples per day while maintaining high sensitivity and depth of analysis. In targeted LC/MS mode, 36 endogenous phosphopeptides in HeLa cells, including EphA2- derived phosphopeptide isomers, were successfully quantified with high selectivity and linearity by combining ion mobility separation. When 500 ng of HeLa cell digest was measured 100 times repeatedly in data-dependent acquisition mode, the coefficient of variation of retention time, peak intensity and number of identified peptides were on average 3.4%, 19.8%, and 6.0%, respectively. In data-independent acquisition mode, this system achieved the identification and quantification of 3,139 protein groups from a 100 ng HeLa cell digest and 2,145 protein groups from a sample of only 10 ng. The coefficient of variation of protein commonly quantified in the triplicate analysis ranged from 12 to 24% for HeLa digest samples ranging from 10 to 1000 ng. Finally, we applied this high-speed system to the spatial proteomics of the mouse brain, and succeeded in capturing the proteome distribution along a 96-sectioned brain structure in 135 minutes. This is the first LC/MS/MS system to achieve both more than 500 samples per day and more than 3000 identified protein groups ID with less than 100 ng human cultured cells simultaneously.\nQuantifying and correcting estimation bias in transcriptional parameter inference from single-cell data\nAuthors: Grima, R.; Esmenjaud, P.-M.\nScore: 10.5, Published: 2023-06-28 DOI: 10.1101/2023.06.19.545536\nThe snapshot distribution of mRNA counts per cell can be readily measured using single molecule FISH or single-cell RNA sequencing. These distributions are often fit to the steady-state distribution of the two-state telegraph model to estimate the three transcriptional parameters for a gene of interest: the rate of transcription, the rate of switching to the active transcriptional state and the rate of switching to the inactive state. These parameters are often understood as being reflective of the average cell in a population. However it is currently unclear to what extent does the cell-to-cell variation of parameters (extrinsic noise) affect the accuracy of parameter estimation. Here we develop an analytical approach to understand the factors determining the size and sign of estimation bias. We find bias signatures that depend on the source of extrinsic noise and the mode of transcriptional activity. We estimate the size of extrinsic noise from the covariance matrix of scRNA-seq data which we use to correct published estimates of parameters for mammalian genes.\nRobustness of biomolecular networks suggests functional modules far from the edge of chaos\nAuthors: Park, K. H.; Costa, F. X.; Rocha, L. M.; Albert, R.; Rozum, J. C.\nScore: 5.5, Published: 2023-07-02 DOI: 10.1101/2023.06.30.547297\nA common feature of complex systems is their ability to balance the flexibility needed to adapt to their environment with the rigidity required for robust function. It has been conjectured that living systems accomplish this by existing at the \"edge of chaos\", i.e., the critical boundary between ordered and disordered dynamics. Simple toy models of gene regulatory networks lend support to this idea, and mathematical tools developed for these toy models yield similar results when applied to experimentally-supported models of specific cellular regulatory mechanisms (functional modules). Here, however, we demonstrate that a deeper inspection of 72 experimentally-supported discrete dynamical models of functional modules reveals previously unobserved order in these systems on long time scales, suggesting greater rigidity in these systems than was previously conjectured. Our analysis relies on new measures that quantify the tendency of perturbations to spread through a discrete dynamical system. A benefit of our new approach is that it accounts for how system trajectories are mapped to phenotypes in practice. Because these measures are computationally expensive to estimate, existing tools were insufficient for the ensemble of models considered here. To simulate the tens of millions of trajectories required for convergence, we developed a multi-purpose CUDA-based simulation tool, which we have made available as the open-source Python library cubewalkers. We find that in experimentally-supported models of biomolecular functional modules, perturbation propagation is more transitory than previously thought, and that even in cases where large perturbation cascades persist, their phenotypic effects are often minimal. Moreover, by examining the impact of update scheme on experimentally-supported models, we find evidence that stochasticity and desynchronization can lead to increased recovery from regulatory perturbation cascades in functional modules and uncover previously unreported population-level robustness to even timing perturbations in these systems. We identify specific biological mechanisms underlying these dynamical behaviors and highlight them in experimentally-supported regulatory networks from the systems biology literature. Based on novel measures and simulations, our results suggest that--contrary to current theory--functional modules of biological systems are ordered and far from the edge of chaos.\nRegulation of bacterial stringent response by an evolutionary conserved ribosomal protein L11 methylation\nAuthors: Walukiewicz, H. E.; Farris, Y.; Burnet, M. C.; Feid, S. C.; You, Y.; Christensen, D.; Payne, S. H.; Wolfe, A.; Rao, C.; Nakayasu, E. S.\nScore: 3.4, Published: 2023-06-30 DOI: 10.1101/2023.06.29.546327\nLysine and arginine methylation is an important regulator of enzyme activity and transcription in eukaryotes. However, little is known about this covalent modification in bacteria. In this work, we investigated the role of methylation in bacteria. By reanalyzing a large phyloproteomics dataset from 48 bacterial strains representing 6 phyla, we found that almost a quarter of the bacterial proteome is methylated. Many of these methylated proteins are conserved across diverse bacterial lineages, including those involved in central carbon metabolism and translation. Among the proteins with the most conserved methylation sites is ribosomal protein L11 (bL11). bL11 methylation has been a mystery for five decades, as the deletion of its methyltransferase PrmA causes no cell growth defects. A comparative proteomics analysis combined with a guanosine polyphosphate assay of the {Delta}prmA mutant in Escherichia coli revealed that bL11 methylation is important for stringent response signaling. Moreover, we show that the {Delta}prmA mutant has an abnormal polysome profile, suggesting a role in ribosomal homeostasis during stationary growth phase. Overall, our investigation demonstrates that the evolutionary conserved bL11 methylation is important for stringent response signaling and ribosomal homeostasis.\nModeling of the Network Mediated by IL-36 Involved inPsoriasis\nAuthors: Pandey, S.; Tiwari, S.; Basu, S.; Mishra, R. K.; Pandey, R.\nScore: 3.0, Published: 2023-07-04 DOI: 10.1101/2023.07.03.547538\nPathogenesis of inflammatory, chronic and common skin disease Psoriasis involves immune cells, skin cells (keratinocytes) and cytokines secreted by them. Hyperproliferation and abnormal differentiation of keratinocytes are believed to be a hallmark of it. Roles of several cytokines such as TNFalpha, IL-15, IL-17 and IL-23 in Psoriasis have been explored through mathematical/computational models as well as experimentally. However, the role of pro-inflammatory cytokine IL-36 is still elusive, especially in the case of General Pustular Psoriasis, a prevalent type of Psoriasis. To explore the role of that here, we construct a network embodying indirect cell-cell interactions of a few immune and skin cells mediated by IL-36 based on the existing knowledge. Further, we develop a mathematical model for the network and study the steady-state behaviour of that. Our results demonstrate that an increase in the level of IL-36 could lead to the hyper-proliferation of keratinocytes and, thus, Psoriasis. In addition, the analysis suggests that the plaque formation and progression of Psoriasis could occur via a gradual or switch-like increase in the population of keratinocytes. The switch-like increase would be due to the bistable behaviour of the network and could be used as a novel treatment strategy, as proposed and demonstrated earlier.\nThe alternative sigma factor RpoD4 pulses at division and links the circadian clock and the cell cycle in Synechococcus elongatus\nAuthors: Ye, C.; Saez, T.; Das, A. K.; Martins, B. M. C.; Locke, J. C. W.\nScore: 2.1, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546855\nThe cyanobacterial circadian clock is an extensive regulator of gene expression, generating a 24-hour rhythm in the majority of genes in Synechococcus elongatus PCC 7942. This raises the question of how the cyanobacterial clock regulates, and receives input from, the diverse cellular processes it controls. Alternative sigma factors, which bind to the core RNA polymerase enzyme and direct the synthesis of specific sets of transcripts, are one mechanism for the clock to set the time of its outputs. In this work, we use single-cell time-lapse microscopy to reveal the transcriptional dynamics of RpoD4, an alternative sigma factor that was previously reported as either arrhythmic or circadian. We find instead that RpoD4 pulses at cell division, dynamics missed by previous bulk averaging. The circadian clock modulates the amplitude of RpoD4 expression pulses, as well as the timing of pulses through its control of division timing. In turn, a rpoD4 mutation causes a reduction in clock period. Further, a rpoD4 mutant results in smaller cell size, and increasing expression of RpoD4 results in larger cell sizes in a dose-response manner, allowing tunable control of cell size. Thus, our single-cell analysis has revealed pulsing gene expression dynamics in cyanobacteria, linking the clock, RpoD4 and the cell cycle.\nIntegrative analysis of CAKUT multi-omics data\nAuthors: Bayjanov, J. R.; Doornbos, C.; Ozisik, O.; Shin, W.; Queralt-Rosinach, N.; Wijnbergen, D.; Saulnier Blache, J.-S.; Schanstra, J. P.; Fernandez, J. M.; Kaliyaperumal, R.; Baudot, A.; t Hoen, P. A. C.; Ehrhart, F.\nScore: 1.4, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547015\nCongenital Anomalies of the Kidney and Urinary Tract (CAKUT) is the leading cause of childhood end-stage renal disease and a significant cause of chronic kidney disease in adults. Genetic and environmental factors are known to influence CAKUT development, but the currently known disease mechanism remains incomplete. Our goal is to identify affected pathways and networks in CAKUT, and thereby aid in getting a better understanding of its pathophysiology. Multi-omics experiments, including amniotic fluid miRNome, peptidome, and proteome analyses, can shed light on foetal kidney development in non-severe CAKUT patients compared to severe CAKUT cases. We performed FAIRification of these omics data sets to facilitate their integration with external data resources. Furthermore, we analysed and integrated the omics data sets using three different bioinformatics strategies. The three bioinformatics analyses provided complementary features, but all pointed towards an important role for collagen in CAKUT development. We published the three analysis strategies as containerized workflows. These workflows can be applied to other FAIR data sets and help gaining knowledge on other rare diseases.\nSPIDR: a highly multiplexed method for mapping RNA-protein interactions uncovers a potential mechanism for selective translational suppression upon cellular stress\nAuthors: Wolin, E.; Guo, J. K.; Blanco, M. R.; Perez, A. A.; Goronzy, I. N.; Abdou, A. A.; Gorhe, D.; Guttman, M.; Jovanovic, M.\nScore: 115.0, Published: 2023-06-07 DOI: 10.1101/2023.06.05.543769\nRNA binding proteins (RBPs) play crucial roles in regulating every stage of the mRNA life cycle and mediating non-coding RNA functions. Despite their importance, the specific roles of most RBPs remain unexplored because we do not know what specific RNAs most RBPs bind. Current methods, such as crosslinking and immunoprecipitation followed by sequencing (CLIP-seq), have expanded our knowledge of RBP-RNA interactions but are generally limited by their ability to map only one RBP at a time. To address this limitation, we developed SPIDR (Split and Pool Identification of RBP targets), a massively multiplexed method to simultaneously profile global RNA binding sites of dozens to hundreds of RBPs in a single experiment. SPIDR employs split-pool barcoding coupled with antibody-bead barcoding to increase the throughput of current CLIP methods by two orders of magnitude. SPIDR reliably identifies precise, single-nucleotide RNA binding sites for diverse classes of RBPs simultaneously. Using SPIDR, we explored changes in RBP binding upon mTOR inhibition and identified that 4EBP1 acts as a dynamic RBP that selectively binds to 5-untranslated regions of specific translationally repressed mRNAs only upon mTOR inhibition. This observation provides a potential mechanism to explain the specificity of translational regulation controlled by mTOR signaling. SPIDR has the potential to revolutionize our understanding of RNA biology and both transcriptional and post-transcriptional gene regulation by enabling rapid, de novo discovery of RNA-protein interactions at an unprecedented scale.\n",
  "wordCount" : "2438",
  "inLanguage": "en",
  "datePublished": "2023-07-05T10:42:43Z",
  "dateModified": "2023-07-05T10:42:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on July 5, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546628">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546628" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546628">
        <p class="paperTitle">Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546628" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546628" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.</p>
        <p class="info">Score: 22.3, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546628' target='https://doi.org/10.1101/2023.06.26.546628'> 10.1101/2023.06.26.546628</a></p>
        <p class="abstract">Combined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts. Here, we present an innovative strategy for simultaneous multi-omics analysis by direct infusion (SMAD) using one single injection without liquid chromatography. SMAD allows quantification of over 9,000 metabolite m/z features and over 1,300 proteins from the same sample in less than five minutes. We validated the efficiency and reliability of this method and then present two practical applications: mouse macrophage M1/M2 polarization and high throughput drug screening in human 293T cells. Finally, we demonstrate relationships between proteomic and metabolomic data are discovered by machine learning.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.03.16.533058">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.03.16.533058" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.03.16.533058">
        <p class="paperTitle">Massively integrated coexpression analysis reveals transcriptional regulation, evolution and cellular implications of the noncanonical translatome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.03.16.533058" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.03.16.533058" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rich, A.; Acar, O.; Carvunis, A.-R.</p>
        <p class="info">Score: 17.3, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.03.16.533058' target='https://doi.org/10.1101/2023.03.16.533058'> 10.1101/2023.03.16.533058</a></p>
        <p class="abstract">BackgroundRecent studies uncovered pervasive transcription and translation of thousands of noncanonical open reading frames (nORFs) outside of annotated genes. The contribution of nORFs to cellular phenotypes is difficult to infer using conventional approaches because nORFs tend to be short, of recent de novo origins, and lowly expressed. Here we develop a dedicated coexpression analysis framework that accounts for low expression to investigate the transcriptional regulation, evolution, and potential cellular roles of nORFs in Saccharomyces cerevisiae.

ResultsOur results reveal that nORFs tend to be preferentially coexpressed with genes involved in cellular transport or homeostasis but rarely with genes involved in RNA processing. Mechanistically, we discover that young de novo nORFs located downstream of conserved genes tend to leverage their neighbors promoters through transcription readthrough, resulting in high coexpression and high expression levels. Transcriptional piggybacking also influences the coexpression profiles of young de novo nORFs located upstream of genes, but to a lesser extent and without detectable impact on expression levels. Transcriptional piggybacking influences, but does not determine, the transcription profiles of de novo nORFs emerging nearby genes. About 40% of nORFs are not strongly coexpressed with any gene but are transcriptionally regulated nonetheless and tend to form entirely new transcription modules. We offer a web browser interface (https://carvunislab.csb.pitt.edu/shiny/coexpression/) to efficiently query, visualize and download our coexpression inferences.

ConclusionsOur results suggest that nORF transcription is highly regulated. Our coexpression dataset serves as an unprecedented resource for unraveling how nORFs integrate into cellular networks, contribute to cellular phenotypes, and evolve.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.05.543682">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.05.543682" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.05.543682">
        <p class="paperTitle">One thousand samples per day capillary-flow LC/MS/MS for high-speed, high-sensitivity and in-depth proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.05.543682" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.05.543682" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tomioka, A.; Tomioka, R.; Mori, I.; Isobe, Y.; Arita, M.; Imami, K.; Kanao, E.; Ogata, K.; Ishihama, Y.</p>
        <p class="info">Score: 22.0, Published: 2023-06-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.05.543682' target='https://doi.org/10.1101/2023.06.05.543682'> 10.1101/2023.06.05.543682</a></p>
        <p class="abstract">We developed a capillary-flow LC/MS/MS system with ultrahigh speed, enabling a throughput of 1,000 samples per day while maintaining high sensitivity and depth of analysis. In targeted LC/MS mode, 36 endogenous phosphopeptides in HeLa cells, including EphA2- derived phosphopeptide isomers, were successfully quantified with high selectivity and linearity by combining ion mobility separation. When 500 ng of HeLa cell digest was measured 100 times repeatedly in data-dependent acquisition mode, the coefficient of variation of retention time, peak intensity and number of identified peptides were on average 3.4%, 19.8%, and 6.0%, respectively. In data-independent acquisition mode, this system achieved the identification and quantification of 3,139 protein groups from a 100 ng HeLa cell digest and 2,145 protein groups from a sample of only 10 ng. The coefficient of variation of protein commonly quantified in the triplicate analysis ranged from 12 to 24% for HeLa digest samples ranging from 10 to 1000 ng. Finally, we applied this high-speed system to the spatial proteomics of the mouse brain, and succeeded in capturing the proteome distribution along a 96-sectioned brain structure in 135 minutes. This is the first LC/MS/MS system to achieve both more than 500 samples per day and more than 3000 identified protein groups ID with less than 100 ng human cultured cells simultaneously.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.19.545536">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.19.545536" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.19.545536">
        <p class="paperTitle">Quantifying and correcting estimation bias in transcriptional parameter inference from single-cell data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.19.545536" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.19.545536" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grima, R.; Esmenjaud, P.-M.</p>
        <p class="info">Score: 10.5, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.19.545536' target='https://doi.org/10.1101/2023.06.19.545536'> 10.1101/2023.06.19.545536</a></p>
        <p class="abstract">The snapshot distribution of mRNA counts per cell can be readily measured using single molecule FISH or single-cell RNA sequencing. These distributions are often fit to the steady-state distribution of the two-state telegraph model to estimate the three transcriptional parameters for a gene of interest: the rate of transcription, the rate of switching to the active transcriptional state and the rate of switching to the inactive state. These parameters are often understood as being reflective of the average cell in a population. However it is currently unclear to what extent does the cell-to-cell variation of parameters (extrinsic noise) affect the accuracy of parameter estimation. Here we develop an analytical approach to understand the factors determining the size and sign of estimation bias. We find bias signatures that depend on the source of extrinsic noise and the mode of transcriptional activity. We estimate the size of extrinsic noise from the covariance matrix of scRNA-seq data which we use to correct published estimates of parameters for mammalian genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547297">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547297" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547297">
        <p class="paperTitle">Robustness of biomolecular networks suggests functional modules far from the edge of chaos</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547297" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547297" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Park, K. H.; Costa, F. X.; Rocha, L. M.; Albert, R.; Rozum, J. C.</p>
        <p class="info">Score: 5.5, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547297' target='https://doi.org/10.1101/2023.06.30.547297'> 10.1101/2023.06.30.547297</a></p>
        <p class="abstract">A common feature of complex systems is their ability to balance the flexibility needed to adapt to their environment with the rigidity required for robust function. It has been conjectured that living systems accomplish this by existing at the &#34;edge of chaos&#34;, i.e., the critical boundary between ordered and disordered dynamics. Simple toy models of gene regulatory networks lend support to this idea, and mathematical tools developed for these toy models yield similar results when applied to experimentally-supported models of specific cellular regulatory mechanisms (functional modules). Here, however, we demonstrate that a deeper inspection of 72 experimentally-supported discrete dynamical models of functional modules reveals previously unobserved order in these systems on long time scales, suggesting greater rigidity in these systems than was previously conjectured. Our analysis relies on new measures that quantify the tendency of perturbations to spread through a discrete dynamical system. A benefit of our new approach is that it accounts for how system trajectories are mapped to phenotypes in practice. Because these measures are computationally expensive to estimate, existing tools were insufficient for the ensemble of models considered here. To simulate the tens of millions of trajectories required for convergence, we developed a multi-purpose CUDA-based simulation tool, which we have made available as the open-source Python library cubewalkers. We find that in experimentally-supported models of biomolecular functional modules, perturbation propagation is more transitory than previously thought, and that even in cases where large perturbation cascades persist, their phenotypic effects are often minimal. Moreover, by examining the impact of update scheme on experimentally-supported models, we find evidence that stochasticity and desynchronization can lead to increased recovery from regulatory perturbation cascades in functional modules and uncover previously unreported population-level robustness to even timing perturbations in these systems. We identify specific biological mechanisms underlying these dynamical behaviors and highlight them in experimentally-supported regulatory networks from the systems biology literature. Based on novel measures and simulations, our results suggest that--contrary to current theory--functional modules of biological systems are ordered and far from the edge of chaos.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546327">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546327" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546327">
        <p class="paperTitle">Regulation of bacterial stringent response by an evolutionary conserved ribosomal protein L11 methylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546327" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546327" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Walukiewicz, H. E.; Farris, Y.; Burnet, M. C.; Feid, S. C.; You, Y.; Christensen, D.; Payne, S. H.; Wolfe, A.; Rao, C.; Nakayasu, E. S.</p>
        <p class="info">Score: 3.4, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546327' target='https://doi.org/10.1101/2023.06.29.546327'> 10.1101/2023.06.29.546327</a></p>
        <p class="abstract">Lysine and arginine methylation is an important regulator of enzyme activity and transcription in eukaryotes. However, little is known about this covalent modification in bacteria. In this work, we investigated the role of methylation in bacteria. By reanalyzing a large phyloproteomics dataset from 48 bacterial strains representing 6 phyla, we found that almost a quarter of the bacterial proteome is methylated. Many of these methylated proteins are conserved across diverse bacterial lineages, including those involved in central carbon metabolism and translation. Among the proteins with the most conserved methylation sites is ribosomal protein L11 (bL11). bL11 methylation has been a mystery for five decades, as the deletion of its methyltransferase PrmA causes no cell growth defects. A comparative proteomics analysis combined with a guanosine polyphosphate assay of the {Delta}prmA mutant in Escherichia coli revealed that bL11 methylation is important for stringent response signaling. Moreover, we show that the {Delta}prmA mutant has an abnormal polysome profile, suggesting a role in ribosomal homeostasis during stationary growth phase. Overall, our investigation demonstrates that the evolutionary conserved bL11 methylation is important for stringent response signaling and ribosomal homeostasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547538">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547538" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547538">
        <p class="paperTitle">Modeling of the Network Mediated by IL-36 Involved inPsoriasis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547538" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547538" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pandey, S.; Tiwari, S.; Basu, S.; Mishra, R. K.; Pandey, R.</p>
        <p class="info">Score: 3.0, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547538' target='https://doi.org/10.1101/2023.07.03.547538'> 10.1101/2023.07.03.547538</a></p>
        <p class="abstract">Pathogenesis of inflammatory, chronic and common skin disease Psoriasis involves immune cells, skin cells (keratinocytes) and cytokines secreted by them. Hyperproliferation and abnormal differentiation of keratinocytes are believed to be a hallmark of it. Roles of several cytokines such as TNFalpha, IL-15, IL-17 and IL-23 in Psoriasis have been explored through mathematical/computational models as well as experimentally. However, the role of pro-inflammatory cytokine IL-36 is still elusive, especially in the case of General Pustular Psoriasis, a prevalent type of Psoriasis. To explore the role of that here, we construct a network embodying indirect cell-cell interactions of a few immune and skin cells mediated by IL-36 based on the existing knowledge. Further, we develop a mathematical model for the network and study the steady-state behaviour of that. Our results demonstrate that an increase in the level of IL-36 could lead to the hyper-proliferation of keratinocytes and, thus, Psoriasis. In addition, the analysis suggests that the plaque formation and progression of Psoriasis could occur via a gradual or switch-like increase in the population of keratinocytes. The switch-like increase would be due to the bistable behaviour of the network and could be used as a novel treatment strategy, as proposed and demonstrated earlier.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546855">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546855" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546855">
        <p class="paperTitle">The alternative sigma factor RpoD4 pulses at division and links the circadian clock and the cell cycle in Synechococcus elongatus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546855" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546855" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ye, C.; Saez, T.; Das, A. K.; Martins, B. M. C.; Locke, J. C. W.</p>
        <p class="info">Score: 2.1, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546855' target='https://doi.org/10.1101/2023.06.28.546855'> 10.1101/2023.06.28.546855</a></p>
        <p class="abstract">The cyanobacterial circadian clock is an extensive regulator of gene expression, generating a 24-hour rhythm in the majority of genes in Synechococcus elongatus PCC 7942. This raises the question of how the cyanobacterial clock regulates, and receives input from, the diverse cellular processes it controls. Alternative sigma factors, which bind to the core RNA polymerase enzyme and direct the synthesis of specific sets of transcripts, are one mechanism for the clock to set the time of its outputs. In this work, we use single-cell time-lapse microscopy to reveal the transcriptional dynamics of RpoD4, an alternative sigma factor that was previously reported as either arrhythmic or circadian. We find instead that RpoD4 pulses at cell division, dynamics missed by previous bulk averaging. The circadian clock modulates the amplitude of RpoD4 expression pulses, as well as the timing of pulses through its control of division timing. In turn, a rpoD4 mutation causes a reduction in clock period. Further, a rpoD4 mutant results in smaller cell size, and increasing expression of RpoD4 results in larger cell sizes in a dose-response manner, allowing tunable control of cell size. Thus, our single-cell analysis has revealed pulsing gene expression dynamics in cyanobacteria, linking the clock, RpoD4 and the cell cycle.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547015">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547015" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547015">
        <p class="paperTitle">Integrative analysis of CAKUT multi-omics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547015" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547015" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bayjanov, J. R.; Doornbos, C.; Ozisik, O.; Shin, W.; Queralt-Rosinach, N.; Wijnbergen, D.; Saulnier Blache, J.-S.; Schanstra, J. P.; Fernandez, J. M.; Kaliyaperumal, R.; Baudot, A.; t Hoen, P. A. C.; Ehrhart, F.</p>
        <p class="info">Score: 1.4, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547015' target='https://doi.org/10.1101/2023.06.29.547015'> 10.1101/2023.06.29.547015</a></p>
        <p class="abstract">Congenital Anomalies of the Kidney and Urinary Tract (CAKUT) is the leading cause of childhood end-stage renal disease and a significant cause of chronic kidney disease in adults. Genetic and environmental factors are known to influence CAKUT development, but the currently known disease mechanism remains incomplete. Our goal is to identify affected pathways and networks in CAKUT, and thereby aid in getting a better understanding of its pathophysiology. Multi-omics experiments, including amniotic fluid miRNome, peptidome, and proteome analyses, can shed light on foetal kidney development in non-severe CAKUT patients compared to severe CAKUT cases. We performed FAIRification of these omics data sets to facilitate their integration with external data resources. Furthermore, we analysed and integrated the omics data sets using three different bioinformatics strategies. The three bioinformatics analyses provided complementary features, but all pointed towards an important role for collagen in CAKUT development. We published the three analysis strategies as containerized workflows. These workflows can be applied to other FAIR data sets and help gaining knowledge on other rare diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.05.543769">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.05.543769" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.05.543769">
        <p class="paperTitle">SPIDR: a highly multiplexed method for mapping RNA-protein interactions uncovers a potential mechanism for selective translational suppression upon cellular stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.05.543769" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.05.543769" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wolin, E.; Guo, J. K.; Blanco, M. R.; Perez, A. A.; Goronzy, I. N.; Abdou, A. A.; Gorhe, D.; Guttman, M.; Jovanovic, M.</p>
        <p class="info">Score: 115.0, Published: 2023-06-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.05.543769' target='https://doi.org/10.1101/2023.06.05.543769'> 10.1101/2023.06.05.543769</a></p>
        <p class="abstract">RNA binding proteins (RBPs) play crucial roles in regulating every stage of the mRNA life cycle and mediating non-coding RNA functions. Despite their importance, the specific roles of most RBPs remain unexplored because we do not know what specific RNAs most RBPs bind. Current methods, such as crosslinking and immunoprecipitation followed by sequencing (CLIP-seq), have expanded our knowledge of RBP-RNA interactions but are generally limited by their ability to map only one RBP at a time. To address this limitation, we developed SPIDR (Split and Pool Identification of RBP targets), a massively multiplexed method to simultaneously profile global RNA binding sites of dozens to hundreds of RBPs in a single experiment. SPIDR employs split-pool barcoding coupled with antibody-bead barcoding to increase the throughput of current CLIP methods by two orders of magnitude. SPIDR reliably identifies precise, single-nucleotide RNA binding sites for diverse classes of RBPs simultaneously. Using SPIDR, we explored changes in RBP binding upon mTOR inhibition and identified that 4EBP1 acts as a dynamic RBP that selectively binds to 5-untranslated regions of specific translationally repressed mRNAs only upon mTOR inhibition. This observation provides a potential mechanism to explain the specificity of translational regulation controlled by mTOR signaling. SPIDR has the potential to revolutionize our understanding of RNA biology and both transcriptional and post-transcriptional gene regulation by enabling rapid, de novo discovery of RNA-protein interactions at an unprecedented scale.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
