<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Does ribosome composition maximize growth rate? The role of RNA instability
Authors: Szeliova, D.; Müller, S.; Zanghellini, J.
Score: 14.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548114
Ribosomes are protein synthesis machines that are central to cellular self-fabrication, and the synthesis time of a ribosome places an upper bound on growth rate. While most cellular enzymes are proteins, ribosomes consist of 1/3 protein and 2/3 RNA (in E. coli). Recent research suggests that ribosome composition arises from a trade-off between two &#34;autocatalytic loops&#34;, ribosomal protein and RNA polymerase synthesis, respectively.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Does ribosome composition maximize growth rate? The role of RNA instability
Authors: Szeliova, D.; Müller, S.; Zanghellini, J.
Score: 14.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548114
Ribosomes are protein synthesis machines that are central to cellular self-fabrication, and the synthesis time of a ribosome places an upper bound on growth rate. While most cellular enzymes are proteins, ribosomes consist of 1/3 protein and 2/3 RNA (in E. coli). Recent research suggests that ribosome composition arises from a trade-off between two &#34;autocatalytic loops&#34;, ribosomal protein and RNA polymerase synthesis, respectively." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-12T10:41:10+00:00" />
<meta property="article:modified_time" content="2023-07-12T10:41:10+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Does ribosome composition maximize growth rate? The role of RNA instability
Authors: Szeliova, D.; Müller, S.; Zanghellini, J.
Score: 14.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548114
Ribosomes are protein synthesis machines that are central to cellular self-fabrication, and the synthesis time of a ribosome places an upper bound on growth rate. While most cellular enzymes are proteins, ribosomes consist of 1/3 protein and 2/3 RNA (in E. coli). Recent research suggests that ribosome composition arises from a trade-off between two &#34;autocatalytic loops&#34;, ribosomal protein and RNA polymerase synthesis, respectively."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Does ribosome composition maximize growth rate? The role of RNA instability\nAuthors: Szeliova, D.; Müller, S.; Zanghellini, J.\nScore: 14.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548114\nRibosomes are protein synthesis machines that are central to cellular self-fabrication, and the synthesis time of a ribosome places an upper bound on growth rate. While most cellular enzymes are proteins, ribosomes consist of 1/3 protein and 2/3 RNA (in E. coli). Recent research suggests that ribosome composition arises from a trade-off between two \u0026#34;autocatalytic loops\u0026#34;, ribosomal protein and RNA polymerase synthesis, respectively.",
  "keywords": [
    
  ],
  "articleBody": " Does ribosome composition maximize growth rate? The role of RNA instability\nAuthors: Szeliova, D.; Müller, S.; Zanghellini, J.\nScore: 14.0, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548114\nRibosomes are protein synthesis machines that are central to cellular self-fabrication, and the synthesis time of a ribosome places an upper bound on growth rate. While most cellular enzymes are proteins, ribosomes consist of 1/3 protein and 2/3 RNA (in E. coli). Recent research suggests that ribosome composition arises from a trade-off between two \"autocatalytic loops\", ribosomal protein and RNA polymerase synthesis, respectively. In this work, we develop a (coarse-grained) mechanistic model of a self-fabricating cell, validate it under different growth conditions, and use resource balance analysis (RBA) to study maximum growth rate as a function of ribosome composition. Our model highlights the importance of RNA instability. If we neglect it, RNA synthesis is always \"cheaper\" than protein synthesis, leading to an RNA-only ribosome at maximum growth rate. To account for RNA turnover, we explore two scenarios regarding the activity of RNases. In (a) degradation is proportional to RNA content, whereas in (b) ribosomal proteins cooperatively mitigate RNA instability by protecting it from misfolding and subsequent degradation. In both cases, an increase in ribosomal protein content raises protein synthesis costs while reducing RNA turnover costs. This leads to a mixed ribosome composed of RNA and proteins. However, only in scenario (b), where we consider the cooperative protection of ribosomal RNA by proteins, our model predictions are in qualitative agreement with experimental data under different growth conditions. Our research offers new mechanistic insights into ribosome biogenesis and evolution. Furthermore, it paves the way for understanding the protein-rich ribosome composition found in archaea and mitochondria.\nBasal MET Phosphorylation is an Indicator of Hepatocyte Dysregulation in Liver Disease\nAuthors: Burbano de Lara, S.; Kemmer, S.; Biermayer, I.; Feiler, S.; Vlasov, A.; D'Alessandro, L. A.; Helm, B.; Dieter, Y.; Ghallab, A.; Hengstler, J. G.; Hoffmann, K.; Schilling, M.; Timmer, J.; Klingmueller, U.\nScore: 3.9, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547655\nChronic liver diseases are worldwide on the rise. Due to the rapidly increasing incidence, in particular in Western countries, Non-alcoholic fatty liver disease (NAFLD) is gaining importance as the disease can develop into hepatocellular carcinoma. Lipid accumulation in hepatocytes has been identified as the characteristic structural change in NAFLD development, but molecular mechanisms responsible for disease progression remained unresolved. Here, we uncover in primary hepatocytes from a preclinical model fed with a Western diet (WD) a strong downregulation of the PI3K-AKT pathway and an upregulation of the MAPK pathway. Dynamic pathway modeling of hepatocyte growth factor (HGF) signal transduction combined with global proteomics identifies that an elevated basal MET phosphorylation rate is the main driver of altered signaling leading to increased proliferation of WD-hepatocytes. Model-adaptation to patient-derived hepatocytes reveal patient-specific variability in basal MET phosphorylation, which correlates with patient outcome after liver surgery. Thus, dysregulated basal MET phosphorylation could be an indicator for the health status of the liver and thereby inform on the risk of a patient to suffer from liver failure after surgery.\nA systems-level analysis of the mutually antagonistic roles of RKIP and BACH1 in dynamics of cancer cell plasticity\nAuthors: Shyam, S.; R, S.; Sehgal, M.; Jolly, M. K.\nScore: 3.1, Published: 2023-07-08 DOI: 10.1101/2023.07.08.547338\nPhenotypic plasticity is a hallmark of cancer metastasis. Epithelial-mesenchymal transition (EMT) is an important axis of phenotypic plasticity. Raf kinase-B inhibitor protein (RKIP) and BTB and CNC homology 1 (BACH1) are two proteins reported to influence EMT. In breast cancer, they act antagonistically, but the exact nature of their roles in mediating EMT and associated other axes of plasticity remains unclear. Here, analysing transcriptomic data, we reveal their antagonistic trends in a pan-cancer manner, in terms of association with EMT, metabolic reprogramming and immune evasion via PD-L1. Next, we developed and simulated a mechanism-based gene regulatory network that captures how RKIP and BACH1 engage in feedback loops with drivers of EMT and stemness. We found that RKIP and BACH1 belong to two separate 'teams' of players - while BACH1 belonged to the one that drove pro-EMT, stem-like and therapy-resistant cell-states, RKIP is a member of a team that enables pro-epithelial, less stem-like and therapy-sensitive phenotypes. Finally, we observed that low RKIP levels and concomitant upregulated BACH1 levels associated with worse clinical outcomes in many cancer types. Together, our systems-level analysis indicates that the emergent dynamics of underlying regulatory network underlie the antagonistic patterns of RKIP and BACH1 with various axes of cancer cell plasticity, as well as with patient survival data.\nRobustness of biomolecular networks suggests functional modules far from the edge of chaos\nAuthors: Park, K. H.; Costa, F. X.; Rocha, L. M.; Albert, R.; Rozum, J. C.\nScore: 8.1, Published: 2023-07-02 DOI: 10.1101/2023.06.30.547297\nA common feature of complex systems is their ability to balance the flexibility needed to adapt to their environment with the rigidity required for robust function. It has been conjectured that living systems accomplish this by existing at the \"edge of chaos\", i.e., the critical boundary between ordered and disordered dynamics. Simple toy models of gene regulatory networks lend support to this idea, and mathematical tools developed for these toy models yield similar results when applied to experimentally-supported models of specific cellular regulatory mechanisms (functional modules). Here, however, we demonstrate that a deeper inspection of 72 experimentally-supported discrete dynamical models of functional modules reveals previously unobserved order in these systems on long time scales, suggesting greater rigidity in these systems than was previously conjectured. Our analysis relies on new measures that quantify the tendency of perturbations to spread through a discrete dynamical system. A benefit of our new approach is that it accounts for how system trajectories are mapped to phenotypes in practice. Because these measures are computationally expensive to estimate, existing tools were insufficient for the ensemble of models considered here. To simulate the tens of millions of trajectories required for convergence, we developed a multipurpose CUDA-based simulation tool, which we have made available as the open-source Python library cubewalkers. We find that in experimentally-supported models of biomolecular functional modules, perturbation propagation is more transitory than previously thought, and that even in cases where large perturbation cascades persist, their phenotypic effects are often minimal. Moreover, by examining the impact of update scheme on experimentally-supported models, we find evidence that stochasticity and desynchronization can lead to increased recovery from regulatory perturbation cascades in functional modules and uncover previously unreported population-level robustness to even timing perturbations in these systems. We identify specific biological mechanisms underlying these dynamical behaviors and highlight them in experimentally-supported regulatory networks from the systems biology literature. Based on novel measures and simulations, our results suggest that-contrary to current theory-functional modules of biological systems are ordered and far from the edge of chaos.\nThe PP2A-like phosphatase Ppg1 mediates assembly of the Far complex to balance gluconeogenic outputs and adapt to glucose depletion\nAuthors: Niphadkar, S.; Karinje, L.; Laxman, S.\nScore: 3.1, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547494\nTo sustain growth in changing nutrient conditions, cells reorganize outputs of metabolic networks and appropriately reallocate resources. Signaling by reversible protein phosphorylation can control such metabolic adaptations. In contrast to kinases, the functions of phosphatases that enable metabolic adaptation as glucose depletes are poorly studied. Using a Saccharomyces cerevisiae deletion screen, we identified the requirement of PP2A-like phosphatase Ppg1 for appropriate carbon allocations towards gluconeogenic outputs - trehalose, glycogen, UDP-glucose, UDP-GlcNAc - specifically after glucose depletion. This homeostatic Ppg1 function is mediated via regulation of the assembly of the Far complex - a multi-subunit complex that tethers to the ER and mitochondrial outer membranes as localized signaling hubs. We show that the Far complex assembly is Ppg1 catalytic activity-dependent. The assembled Far complex is required to maintain gluconeogenic outputs after glucose depletion. Glucose in turn regulates Far complex abundance. This Ppg1-mediated Far complex assembly, and dependent control of gluconeogenic outputs enhances adaptive growth under glucose depletion. Our study illustrates how protein dephosphorylation is required for the assembly of a multi-protein scaffold present in localized cytosolic pools, to thereby enable cells metabolically adapt to nutrient fluctuations.\nSingle transcript-level metabolic responsive landscape of human liver transcriptome\nAuthors: Jiang, C.; Li, P.; Ma, Y.; Yoneda, N.; Nishiwaki, M.; Ohnishi, Y.; Suemizu, H.; Cao, H.\nScore: 1.5, Published: 2023-07-06 DOI: 10.1101/2023.07.06.544842\nDirect knowledge of gene regulation in human liver by metabolic stimuli could fundamentally advance our understanding of metabolic physiology. This information, however, is largely unknown, and this void is deeply rooted in a paradox that is caused by the inaccessibility of human liver to treatments and the insufficient annotation of liver transcriptome. Recent advances have uncovered immense complexity of transcriptome, i.e., multiple transcripts produced from one gene and extensive RNA modifications, which are all highly regulated and often condition-dependent. Establishing an inclusive annotation to study liver transcriptome dynamics thus requires human liver samples of diverse conditions, which are paradoxically unavailable due to the inaccessibility of human liver to treatments. In this work, we addressed these challenges by coupling an isogenic humanized mouse model with Nanopore single-molecule direct RNA sequencing (DRS). We first generated mice that carried humanized livers of identical genetic background, which were equivalent to clones of a single human liver, and then subjected the mice to representative metabolic treatments. We then analyzed the humanized livers with Nanopore DRS, which directly reads full-length native RNAs to determine the expression level, m6A modification and poly(A) tail length of all RNA transcript isoforms. Thus, our system allows for constructing a de novo annotation of human liver transcriptomes reflecting metabolic responses and studying transcriptome dynamics in conjunction. Our analysis uncovered a vast number of novel genes and transcripts that have not been previously reported. Our transcript-level analysis of human liver transcriptomes also identified a multitude of regulated metabolic pathways that were otherwise invisible using conventional short read RNA-seq. We also revealed for the first time the dynamic changes in m6A and poly(A) tail length of human liver transcripts many of which are transcribed from key metabolic genes. Furthermore, we performed comparative analyses of gene regulation between human and mouse and between two individuals using the liver-specific humanized mice. This revealed that transcriptome dynamics are highly species- and genetic background-dependent, which may only be faithfully studied in a humanized system that entails clones of the same human liver. Hence our work revealed a complex metabolic responsive landscape of human liver transcriptome and also provided a framework to understand transcriptome dynamics of human liver in response to physiologically relevant metabolic stimuli.\nSimultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)\nAuthors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.\nScore: 23.3, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546628\nCombined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts. Here, we present an innovative strategy for simultaneous multi-omics analysis by direct infusion (SMAD) using one single injection without liquid chromatography. SMAD allows quantification of over 9,000 metabolite m/z features and over 1,300 proteins from the same sample in less than five minutes. We validated the efficiency and reliability of this method and then present two practical applications: mouse macrophage M1/M2 polarization and high throughput drug screening in human 293T cells. Finally, we demonstrate relationships between proteomic and metabolomic data are discovered by machine learning.\n",
  "wordCount" : "1891",
  "inLanguage": "en",
  "datePublished": "2023-07-12T10:41:10Z",
  "dateModified": "2023-07-12T10:41:10Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on July 12, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548114">
        <p class="paperTitle">Does ribosome composition maximize growth rate? The role of RNA instability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Szeliova, D.; Müller, S.; Zanghellini, J.</p>
        <p class="info">Score: 14.0, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548114' target='https://doi.org/10.1101/2023.07.07.548114'> 10.1101/2023.07.07.548114</a></p>
        <p class="abstract">Ribosomes are protein synthesis machines that are central to cellular self-fabrication, and the synthesis time of a ribosome places an upper bound on growth rate. While most cellular enzymes are proteins, ribosomes consist of 1/3 protein and 2/3 RNA (in E. coli). Recent research suggests that ribosome composition arises from a trade-off between two &#34;autocatalytic loops&#34;, ribosomal protein and RNA polymerase synthesis, respectively. In this work, we develop a (coarse-grained) mechanistic model of a self-fabricating cell, validate it under different growth conditions, and use resource balance analysis (RBA) to study maximum growth rate as a function of ribosome composition. Our model highlights the importance of RNA instability. If we neglect it, RNA synthesis is always &#34;cheaper&#34; than protein synthesis, leading to an RNA-only ribosome at maximum growth rate. To account for RNA turnover, we explore two scenarios regarding the activity of RNases. In (a) degradation is proportional to RNA content, whereas in (b) ribosomal proteins cooperatively mitigate RNA instability by protecting it from misfolding and subsequent degradation. In both cases, an increase in ribosomal protein content raises protein synthesis costs while reducing RNA turnover costs. This leads to a mixed ribosome composed of RNA and proteins. However, only in scenario (b), where we consider the cooperative protection of ribosomal RNA by proteins, our model predictions are in qualitative agreement with experimental data under different growth conditions. Our research offers new mechanistic insights into ribosome biogenesis and evolution. Furthermore, it paves the way for understanding the protein-rich ribosome composition found in archaea and mitochondria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547655" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547655">
        <p class="paperTitle">Basal MET Phosphorylation is an Indicator of Hepatocyte Dysregulation in Liver Disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burbano de Lara, S.; Kemmer, S.; Biermayer, I.; Feiler, S.; Vlasov, A.; D&#39;Alessandro, L. A.; Helm, B.; Dieter, Y.; Ghallab, A.; Hengstler, J. G.; Hoffmann, K.; Schilling, M.; Timmer, J.; Klingmueller, U.</p>
        <p class="info">Score: 3.9, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547655' target='https://doi.org/10.1101/2023.07.04.547655'> 10.1101/2023.07.04.547655</a></p>
        <p class="abstract">Chronic liver diseases are worldwide on the rise. Due to the rapidly increasing incidence, in particular in Western countries, Non-alcoholic fatty liver disease (NAFLD) is gaining importance as the disease can develop into hepatocellular carcinoma. Lipid accumulation in hepatocytes has been identified as the characteristic structural change in NAFLD development, but molecular mechanisms responsible for disease progression remained unresolved. Here, we uncover in primary hepatocytes from a preclinical model fed with a Western diet (WD) a strong downregulation of the PI3K-AKT pathway and an upregulation of the MAPK pathway. Dynamic pathway modeling of hepatocyte growth factor (HGF) signal transduction combined with global proteomics identifies that an elevated basal MET phosphorylation rate is the main driver of altered signaling leading to increased proliferation of WD-hepatocytes. Model-adaptation to patient-derived hepatocytes reveal patient-specific variability in basal MET phosphorylation, which correlates with patient outcome after liver surgery. Thus, dysregulated basal MET phosphorylation could be an indicator for the health status of the liver and thereby inform on the risk of a patient to suffer from liver failure after surgery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.08.547338">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.08.547338" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.08.547338">
        <p class="paperTitle">A systems-level analysis of the mutually antagonistic roles of RKIP and BACH1 in dynamics of cancer cell plasticity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.08.547338" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.08.547338" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shyam, S.; R, S.; Sehgal, M.; Jolly, M. K.</p>
        <p class="info">Score: 3.1, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.08.547338' target='https://doi.org/10.1101/2023.07.08.547338'> 10.1101/2023.07.08.547338</a></p>
        <p class="abstract">Phenotypic plasticity is a hallmark of cancer metastasis. Epithelial-mesenchymal transition (EMT) is an important axis of phenotypic plasticity. Raf kinase-B inhibitor protein (RKIP) and BTB and CNC homology 1 (BACH1) are two proteins reported to influence EMT. In breast cancer, they act antagonistically, but the exact nature of their roles in mediating EMT and associated other axes of plasticity remains unclear. Here, analysing transcriptomic data, we reveal their antagonistic trends in a pan-cancer manner, in terms of association with EMT, metabolic reprogramming and immune evasion via PD-L1. Next, we developed and simulated a mechanism-based gene regulatory network that captures how RKIP and BACH1 engage in feedback loops with drivers of EMT and stemness. We found that RKIP and BACH1 belong to two separate &#39;teams&#39; of players - while BACH1 belonged to the one that drove pro-EMT, stem-like and therapy-resistant cell-states, RKIP is a member of a team that enables pro-epithelial, less stem-like and therapy-sensitive phenotypes. Finally, we observed that low RKIP levels and concomitant upregulated BACH1 levels associated with worse clinical outcomes in many cancer types. Together, our systems-level analysis indicates that the emergent dynamics of underlying regulatory network underlie the antagonistic patterns of RKIP and BACH1 with various axes of cancer cell plasticity, as well as with patient survival data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547297">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547297" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547297">
        <p class="paperTitle">Robustness of biomolecular networks suggests functional modules far from the edge of chaos</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547297" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547297" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Park, K. H.; Costa, F. X.; Rocha, L. M.; Albert, R.; Rozum, J. C.</p>
        <p class="info">Score: 8.1, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547297' target='https://doi.org/10.1101/2023.06.30.547297'> 10.1101/2023.06.30.547297</a></p>
        <p class="abstract">A common feature of complex systems is their ability to balance the flexibility needed to adapt to their environment with the rigidity required for robust function. It has been conjectured that living systems accomplish this by existing at the &#34;edge of chaos&#34;, i.e., the critical boundary between ordered and disordered dynamics. Simple toy models of gene regulatory networks lend support to this idea, and mathematical tools developed for these toy models yield similar results when applied to experimentally-supported models of specific cellular regulatory mechanisms (functional modules). Here, however, we demonstrate that a deeper inspection of 72 experimentally-supported discrete dynamical models of functional modules reveals previously unobserved order in these systems on long time scales, suggesting greater rigidity in these systems than was previously conjectured. Our analysis relies on new measures that quantify the tendency of perturbations to spread through a discrete dynamical system. A benefit of our new approach is that it accounts for how system trajectories are mapped to phenotypes in practice. Because these measures are computationally expensive to estimate, existing tools were insufficient for the ensemble of models considered here. To simulate the tens of millions of trajectories required for convergence, we developed a multipurpose CUDA-based simulation tool, which we have made available as the open-source Python library cubewalkers. We find that in experimentally-supported models of biomolecular functional modules, perturbation propagation is more transitory than previously thought, and that even in cases where large perturbation cascades persist, their phenotypic effects are often minimal. Moreover, by examining the impact of update scheme on experimentally-supported models, we find evidence that stochasticity and desynchronization can lead to increased recovery from regulatory perturbation cascades in functional modules and uncover previously unreported population-level robustness to even timing perturbations in these systems. We identify specific biological mechanisms underlying these dynamical behaviors and highlight them in experimentally-supported regulatory networks from the systems biology literature. Based on novel measures and simulations, our results suggest that-contrary to current theory-functional modules of biological systems are ordered and far from the edge of chaos.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547494">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547494" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547494">
        <p class="paperTitle">The PP2A-like phosphatase Ppg1 mediates assembly of the Far complex to balance gluconeogenic outputs and adapt to glucose depletion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547494" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547494" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Niphadkar, S.; Karinje, L.; Laxman, S.</p>
        <p class="info">Score: 3.1, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547494' target='https://doi.org/10.1101/2023.07.03.547494'> 10.1101/2023.07.03.547494</a></p>
        <p class="abstract">To sustain growth in changing nutrient conditions, cells reorganize outputs of metabolic networks and appropriately reallocate resources. Signaling by reversible protein phosphorylation can control such metabolic adaptations. In contrast to kinases, the functions of phosphatases that enable metabolic adaptation as glucose depletes are poorly studied. Using a Saccharomyces cerevisiae deletion screen, we identified the requirement of PP2A-like phosphatase Ppg1 for appropriate carbon allocations towards gluconeogenic outputs - trehalose, glycogen, UDP-glucose, UDP-GlcNAc - specifically after glucose depletion. This homeostatic Ppg1 function is mediated via regulation of the assembly of the Far complex - a multi-subunit complex that tethers to the ER and mitochondrial outer membranes as localized signaling hubs. We show that the Far complex assembly is Ppg1 catalytic activity-dependent. The assembled Far complex is required to maintain gluconeogenic outputs after glucose depletion. Glucose in turn regulates Far complex abundance. This Ppg1-mediated Far complex assembly, and dependent control of gluconeogenic outputs enhances adaptive growth under glucose depletion. Our study illustrates how protein dephosphorylation is required for the assembly of a multi-protein scaffold present in localized cytosolic pools, to thereby enable cells metabolically adapt to nutrient fluctuations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.544842">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.544842" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.544842">
        <p class="paperTitle">Single transcript-level metabolic responsive landscape of human liver transcriptome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.544842" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.544842" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, C.; Li, P.; Ma, Y.; Yoneda, N.; Nishiwaki, M.; Ohnishi, Y.; Suemizu, H.; Cao, H.</p>
        <p class="info">Score: 1.5, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.544842' target='https://doi.org/10.1101/2023.07.06.544842'> 10.1101/2023.07.06.544842</a></p>
        <p class="abstract">Direct knowledge of gene regulation in human liver by metabolic stimuli could fundamentally advance our understanding of metabolic physiology. This information, however, is largely unknown, and this void is deeply rooted in a paradox that is caused by the inaccessibility of human liver to treatments and the insufficient annotation of liver transcriptome. Recent advances have uncovered immense complexity of transcriptome, i.e., multiple transcripts produced from one gene and extensive RNA modifications, which are all highly regulated and often condition-dependent. Establishing an inclusive annotation to study liver transcriptome dynamics thus requires human liver samples of diverse conditions, which are paradoxically unavailable due to the inaccessibility of human liver to treatments. In this work, we addressed these challenges by coupling an isogenic humanized mouse model with Nanopore single-molecule direct RNA sequencing (DRS). We first generated mice that carried humanized livers of identical genetic background, which were equivalent to clones of a single human liver, and then subjected the mice to representative metabolic treatments. We then analyzed the humanized livers with Nanopore DRS, which directly reads full-length native RNAs to determine the expression level, m6A modification and poly(A) tail length of all RNA transcript isoforms. Thus, our system allows for constructing a de novo annotation of human liver transcriptomes reflecting metabolic responses and studying transcriptome dynamics in conjunction. Our analysis uncovered a vast number of novel genes and transcripts that have not been previously reported. Our transcript-level analysis of human liver transcriptomes also identified a multitude of regulated metabolic pathways that were otherwise invisible using conventional short read RNA-seq. We also revealed for the first time the dynamic changes in m6A and poly(A) tail length of human liver transcripts many of which are transcribed from key metabolic genes. Furthermore, we performed comparative analyses of gene regulation between human and mouse and between two individuals using the liver-specific humanized mice. This revealed that transcriptome dynamics are highly species- and genetic background-dependent, which may only be faithfully studied in a humanized system that entails clones of the same human liver. Hence our work revealed a complex metabolic responsive landscape of human liver transcriptome and also provided a framework to understand transcriptome dynamics of human liver in response to physiologically relevant metabolic stimuli.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546628">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546628" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546628">
        <p class="paperTitle">Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546628" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546628" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G.</p>
        <p class="info">Score: 23.3, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546628' target='https://doi.org/10.1101/2023.06.26.546628'> 10.1101/2023.06.26.546628</a></p>
        <p class="abstract">Combined multi-omics analysis of proteomics, polar metabolomics, and lipidomics requires separate liquid chromatography-mass spectrometry (LC-MS) platforms for each omics layer. This requirement for different platforms limits throughput and increases costs, preventing the application of mass spectrometry-based multi-omics to large scale drug discovery or clinical cohorts. Here, we present an innovative strategy for simultaneous multi-omics analysis by direct infusion (SMAD) using one single injection without liquid chromatography. SMAD allows quantification of over 9,000 metabolite m/z features and over 1,300 proteins from the same sample in less than five minutes. We validated the efficiency and reliability of this method and then present two practical applications: mouse macrophage M1/M2 polarization and high throughput drug screening in human 293T cells. Finally, we demonstrate relationships between proteomic and metabolomic data are discovered by machine learning.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
