<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Systematic exploration of prokaryotic form I rubisco maximal carboxylation rates
Authors: de Pins, B.; Greenspoon, L.; Bar-On, Y. M.; Shamshoum, M.; Ben-Nissan, R.; Milshtein, E.; Davidi, D.; Sharon, I.; Mueller-Cajar, O.; Noor, E.; Milo, R.
Score: 18.4, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550689
Autotrophy is the basis for complex life on Earth. Its physiological limits are largely constrained by rubisco - the enzyme that catalyzes almost all carbon fixation on the planet.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Systematic exploration of prokaryotic form I rubisco maximal carboxylation rates
Authors: de Pins, B.; Greenspoon, L.; Bar-On, Y. M.; Shamshoum, M.; Ben-Nissan, R.; Milshtein, E.; Davidi, D.; Sharon, I.; Mueller-Cajar, O.; Noor, E.; Milo, R.
Score: 18.4, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550689
Autotrophy is the basis for complex life on Earth. Its physiological limits are largely constrained by rubisco - the enzyme that catalyzes almost all carbon fixation on the planet." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-06T10:36:48+00:00" />
<meta property="article:modified_time" content="2023-08-06T10:36:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Systematic exploration of prokaryotic form I rubisco maximal carboxylation rates
Authors: de Pins, B.; Greenspoon, L.; Bar-On, Y. M.; Shamshoum, M.; Ben-Nissan, R.; Milshtein, E.; Davidi, D.; Sharon, I.; Mueller-Cajar, O.; Noor, E.; Milo, R.
Score: 18.4, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550689
Autotrophy is the basis for complex life on Earth. Its physiological limits are largely constrained by rubisco - the enzyme that catalyzes almost all carbon fixation on the planet."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Systematic exploration of prokaryotic form I rubisco maximal carboxylation rates\nAuthors: de Pins, B.; Greenspoon, L.; Bar-On, Y. M.; Shamshoum, M.; Ben-Nissan, R.; Milshtein, E.; Davidi, D.; Sharon, I.; Mueller-Cajar, O.; Noor, E.; Milo, R.\nScore: 18.4, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550689\nAutotrophy is the basis for complex life on Earth. Its physiological limits are largely constrained by rubisco - the enzyme that catalyzes almost all carbon fixation on the planet.",
  "keywords": [
    
  ],
  "articleBody": " Systematic exploration of prokaryotic form I rubisco maximal carboxylation rates\nAuthors: de Pins, B.; Greenspoon, L.; Bar-On, Y. M.; Shamshoum, M.; Ben-Nissan, R.; Milshtein, E.; Davidi, D.; Sharon, I.; Mueller-Cajar, O.; Noor, E.; Milo, R.\nScore: 18.4, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550689\nAutotrophy is the basis for complex life on Earth. Its physiological limits are largely constrained by rubisco - the enzyme that catalyzes almost all carbon fixation on the planet. Yet, with only a small fraction of rubisco diversity kinetically characterized so far, the underlying biological factors driving the evolution of fast rubiscos in nature remain unclear. We conducted a high-throughput kinetic characterization of over 100 prokaryotic form I rubiscos, the most ubiquitous group of rubisco sequences in nature, to uncover the determinants of rubiscos carboxylation velocity. We show that phototrophy and the presence of a carboxysome CO2 concentrating mechanism correlate with faster rubiscos (median rate difference of 3-5 fold). In contrast to previous assumptions, rubiscos originating from -cyanobacteria exhibit the highest carboxylation rates among form I enzymes ({approx}10 s-1 median versus \u003c7 s-1 in other groups). Our study ties biological and environmental insights associated with kinetic variation across rubiscos from nature.\nPathogenic mutations of human phosphorylation sites affect protein-protein interactions\nAuthors: Rrustemi, T.; Meyer, K.; Roske, Y.; Uyar, B.; Akalin, A.; Imami, K.; Ishihama, Y.; Daumke, O.; Selbach, M.\nScore: 8.2, Published: 2023-08-03 DOI: 10.1101/2023.08.01.551433\nDespite their lack of a defined 3D structure, intrinsically disordered regions (IDRs) of proteins play important biological roles. Many IDRs contain short linear motifs (SLiMs) that mediate protein-protein interactions (PPIs), which can be regulated by post-translational modifications like phosphorylation. 20% of pathogenic missense mutations are found in IDRs, and understanding how such mutations affect PPIs is essential for unraveling disease mechanisms. Here, we employed peptide-based interaction proteomics to investigate 36 disease-causing mutations affecting phosphorylation sites. Our results unveiled significant differences in interactomes between phosphorylated and non-phosphorylated peptides, often due to disrupted phosphorylation-dependent SLiMs. We focused on a mutation of a serine phosphorylation site in the transcription factor GATAD1, which causes dilated cardiomyopathy. We found that this phosphorylation site mediates interaction with 14-3-3 family proteins. Follow-up experiments revealed the structural basis of this interaction and suggest that 14-3-3 binding affects GATAD1 nucleocytoplasmic transport by masking a nuclear localisation signal. Our results demonstrate that pathogenic mutations of human phosphorylation sites can significantly impact protein-protein interactions, offering fresh insights into potential molecular mechanisms underlying pathogenesis.\nDensity Physics-Informed Neural Network reveals sources of cell heterogeneity in signal transduction\nAuthors: Jo, H.; Hong, H.; Hwang, H. J.; Chang, W.; Kim, J. K.\nScore: 3.6, Published: 2023-08-02 DOI: 10.1101/2023.07.31.551393\nThe transduction time between signal initiation and final response provides valuable information on the underlying signaling pathway, including its speed and precision. Furthermore, multimodality in transduction-time distribution informs that the response is regulated by multiple pathways with different transduction speeds. Here, we developed Density physics-informed neural network (Density-PINN) to infer the transduction-time distribution, challenging to measure, from measurable final stress response time traces. We applied Density-PINN to single-cell gene expression data from 16 promoters regulated by unknown pathways in response to antibiotic stresses. We found that promoters with slower signaling initiation and transduction exhibit larger cell-to-cell heterogeneity in response intensity. However, this heterogeneity was greatly reduced when the response was regulated by slow and fast pathways together. This suggests a strategy for identifying effective signaling pathways for consistent cellular responses to disease treatments. Density-PINN can also be applied to understand various time delay systems, including infectious diseases.\nInterpreting roles of mutations associated with the emergence of S. aureus USA300 strains using transcriptional regulatory network reconstruction\nAuthors: Poudel, S.; Hyun, J. C.; Hefner, Y.; Monk, J.; Nizet, V.; Palsson, B. O.\nScore: 3.4, Published: 2023-07-29 DOI: 10.1101/2023.07.26.550757\nThe Staphylococcus aureus clonal complex 8 (CC8) is made up of several subtypes with varying levels of clinical burden; from community-associated methicillin resistant S. aureus (CA-MRSA) USA300 strains to hospital-associated (HA-MRSA) USA500 strains and basal methicillin susceptible (MSSA) strains. This phenotypic distribution within a single clonal complex makes CC8 an ideal clade to study the emergence of mutations important for antibiotic resistance and community spread. Gene level analysis comparing USA300 against MSSA and HA-MRSA strains have revealed key horizontally acquired genes important for its rapid spread in the community. However, efforts to define the contributions of point mutations and indels have been confounded by strong linkage disequilibrium resulting from clonal propagation. To break down this confounding effect, we combined genetic association testing with a model of the transcriptional regulatory network (TRN) to find candidate mutations that may have led to changes in gene regulation. First, we used a De Bruijn graph genome-wide association study (DBGWAS) to enrich mutations unique to the USA300 lineages within CC8. Next, we reconstructed the TRN by using Independent Component Analysis on 670 RNA sequencing samples from USA300 and non-USA300 CC8 strains which predicted several genes with strain-specific altered expression patterns. Examination of the regulatory region of one of the genes enriched by both approaches, isdH, revealed a 38 base pair deletion containing a Fur binding site and a conserved SNP which likely led to the altered expression levels in USA300 strains. Taken together, our results demonstrate the utility of reconstructed TRNs to address the limits of genetic approaches when studying emerging pathogenic strains.\nComputational prediction of protein interactions on single cells by proximity sequencing\nAuthors: Xia, J.; Phan, H. V.; Vistain, L.; Chen, M.; Khan, A. A.; Tay, S.\nScore: 3.1, Published: 2023-07-30 DOI: 10.1101/2023.07.27.550388\nProximity sequencing (Prox-seq) measures gene expression, protein expression, and protein complexes at the single cell level, using information from dual-antibody binding events and a single cell sequencing readout. Prox-seq provides multi-dimensional phenotyping of single cells and was recently used to track the formation of receptor complexes during inflammatory signaling in macrophages and to discover a new interaction between CD9/CD8 proteins on naive T cells. The distribution of protein abundance affects identification of protein complexes in a complicated manner in dual-binding assays like Prox-seq. These effects are difficult to explore with experiments, yet important for accurate quantification of protein complexes. Here, we introduce a physical model for protein dimer formation on single cells and computationally evaluate several different methods for reducing background noise when quantifying protein complexes. Furthermore, we developed an improved method for analysis of Prox-seq single-cell data, which resulted in more accurate and robust quantification of protein complexes. Finally, our model offers a simple way to investigate the behavior of Prox-seq under various biological conditions and guide users toward selecting the best analysis method for their data.\nMachine Learning identifies conserved traits that influence lifespan and healthspan responses to dietary restriction\nAuthors: Hilsabeck, T. A.; Narayan, V. P.; Wilson, K. A.; Carrera, E.; Raftery, D.; Promislow, D. E.; Brem, R. B.; Campisi, J. A.; Kapahi, P.\nScore: 5.3, Published: 2023-07-13 DOI: 10.1101/2023.07.09.548232\nDietary restriction (DR) is the most robust means to extend lifespan and healthspan across species, but factors such as genetic variation affect how an individual will respond to DR. Additionally, it is unclear how cumulative variations in metabolism and the metabolome influence longevity and health. We utilized metabolomic, phenotypic, and genome-wide data from Drosophila Genetic Reference Panel strains raised under ad libitum and DR conditions to identify factors which influence longevity and health in response to dietary modulation. We found multiple intra-dataset correlations (e.g., metabolites with metabolites) but few inter-dataset correlations (e.g., metabolites with health-related phenotypes). Through random forest modeling across all traits and Mendelian Randomization, we found key translatable traits that influence lifespan or healthspan determination and validated the role of multiple metabolites in regulating lifespan. Through these approaches, we utilized data from flies and humans to elucidate potential therapeutic pathways and metabolomic targets for diet response, lifespan, and healthspan.\nAutomatic design of gene regulatory mechanisms for spatial pattern formation\nAuthors: Mousavi, R.; Lobo, D.\nScore: 3.5, Published: 2023-07-29 DOI: 10.1101/2023.07.26.550573\nGene regulatory mechanisms (GRMs) control the formation of spatial and temporal expression patterns that can serve as regulatory signals for the development of complex shapes. Synthetic developmental biology aims to engineer such mechanisms for understanding and producing desired multicellular patterns. However, the design of synthetic GRMs that can produce a target spatial gene expression pattern is a current challenge due to the nonlinear interactions and feedback loops present in genetic circuits. Here we present a methodology to automatically design GRMs that can produce any given spatial pattern. The proposed approach uses two orthogonal morphogen gradients acting as positional information signals in a multicellular tissue area or culture, which constitutes a continuous field of engineered cells implementing the same designed GRM. To efficiently design both the circuit network and the interaction mechanisms--including the number of genes required for the formation of the target pattern--we developed an automated algorithm based on high-performance evolutionary computation. The tolerance of the algorithm can be configured to design GRMs that are either simple to produce approximate patterns or complex to produce precise patterns. We demonstrate its performance by automatically designing GRMs for a diverse set of synthetic spatial expression patterns. The proposed method offers a versatile approach to systematically design and discover pattern-producing genetic circuits.\nThe epigenetic response of mouse brown adipose tissue to cold stress: histone proteoform, DNA methylation, and RNA expression\nAuthors: Taylor, B. C.; Steinthal, L. H.; Dias, M.; Yalamanchili, H. K.; Ochsner, S. A.; Zapata, G. E.; Mehta, N. R.; McKenna, N. J.; Young, N. L.; Nuotio-Antar, A. M.\nScore: 2.5, Published: 2023-07-30 DOI: 10.1101/2023.07.30.551059\nRegulation of the thermogenic response by brown adipose tissue (BAT) is an important component of energy homeostasis with implications for the treatment of obesity and diabetes. Motivated to understand how BAT function is regulated epigenetically, we reanalyzed publicly available RNA-seq data and uncovered many nodes representing epigenetic modifiers that are altered in BAT in response to chronic thermogenic activation. Thus, we hypothesized that chronic thermogenic activation broadly alters epigenetic modifications of DNA and histones in BAT. To test our hypothesis, wildtype male C57BL/6J mice were housed under chronic conditions of thermoneutral temperature (TN, 28.8{degrees}C), mild cold/room temperature (RT, 22{degrees}C), or severe cold (SC, 8{degrees}C). BAT and reference tissue liver were subsequently dissected from each mouse. Reduced representation bisulfite sequencing (RRBS) reveals decreased methylation of promoters and intragenic regions in BAT genomic DNA in response to varying degrees of chronic cold exposure. Integration of our RRBS and the RNA-Seq dataset suggests a role for epigenetic modification of DNA in gene regulation in response to cold. To analyze histone modifications in BAT, we develop a robust method for the isolation of histones and report the first quantitation of histone H3.2 and H4 proteoforms. We initially observe that BAT and liver exhibit different histone proteoforms. Next, we report housing temperature-dependent changes in histone proteoforms in BAT. We observe differences in bivalent proteoforms such as H3{K9me2,K23ac,K36me1}, H3{K9me3,K23ac,K36me1} and H4. Taken together, our results provide novel findings supporting global epigenetic modification in murine BAT in response to varying degrees of chronic cold stimuli and establish a methodology to quantitatively study histones in BAT, allowing for direct comparisons to decipher mechanistic changes during the thermogenic response.\nQuantifying mutational synergy using computational models predicts survival in haematological cancers\nAuthors: Norris, R.; Jones, J.; Mancini, E.; Chevassut, T.; Pepper, C.; Pepper, A.; Mitchell, S.\nScore: 5.9, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548371\nGenetic heterogeneity and co-occurring driver mutations contribute to poor clinical outcomes in cancer. However, the impact of multiple mutations on complex signalling networks is not easily predicted. We found that, by placing mutations into their cellular context, multi-scale agent- based mathematical models could predict how genetic events combine in haematological malignancies. Simulations of lymphoma and myeloma predicted co-occurring mutations synergised to increase tumour cell expansion beyond what would be expected from the impact of the individual mutations alone. Mutational synergy between MYC and BCL2 was consistent with the more aggressive disease course of patients with double-hit lymphoma, and mutational synergy between MCL1 and CKS1B was predictive of outcome in patients with gain 1q multiple myeloma. Incorporating patient-specific mutational profiles into personalised models of lymphoma patients with the worst clinical outcomes revealed a correlation between simulated mutational synergy and overall survival, which outperformed widely used classifications of lymphoma informed by gene expression or mutational data alone. Our results demonstrated that mutational synergy scores enabled prediction of the impact of co-occurring mutations and may improve personalised prognostic predictions.\nEnhanced cell segmentation with limited annotated data using generative adversarial networks\nAuthors: Zargari, A.; Mashhadi, N.; Shariati, A.\nScore: 3.7, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550715\nThe application of deep learning is rapidly transforming the field of bioimage analysis. While deep learning has shown great promise in complex microscopy tasks such as single-cell segmentation, the development of generalizable foundation deep learning segmentation models is hampered by the scarcity of large and diverse annotated datasets of cell images for training purposes. Generative Adversarial Networks (GANs) can generate realistic images that can potentially be easily used to train deep learning models without the generation of large manually annotated microscopy images. Here, we propose a customized CycleGAN architecture to train an enhanced cell segmentation model with limited annotated cell images, effectively addressing the challenge of paucity of annotated data in microscopy imaging. Our customized CycleGAN model can generate realistic synthetic images of cells with morphological details and nuances very similar to that of real images. This method not only increases the variability seen during training but also enhances the authenticity of synthetic samples, thereby enhancing the overall predictive accuracy and robustness of the cell segmentation model. Our experimental results show that our CycleGAN-based method significantly improves the performance of the segmentation model compared to conventional training techniques. Interestingly, we demonstrate that our model can extrapolate its knowledge by synthesizing imaging scenarios that were not seen during the training process. Our proposed customized CycleGAN method will accelerate the development of foundation models for cell segmentation in microscopy images.\n",
  "wordCount" : "2293",
  "inLanguage": "en",
  "datePublished": "2023-08-06T10:36:48Z",
  "dateModified": "2023-08-06T10:36:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on August 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550689">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550689" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550689">
        <p class="paperTitle">Systematic exploration of prokaryotic form I rubisco maximal carboxylation rates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550689" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550689" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Pins, B.; Greenspoon, L.; Bar-On, Y. M.; Shamshoum, M.; Ben-Nissan, R.; Milshtein, E.; Davidi, D.; Sharon, I.; Mueller-Cajar, O.; Noor, E.; Milo, R.</p>
        <p class="info">Score: 18.4, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550689' target='https://doi.org/10.1101/2023.07.27.550689'> 10.1101/2023.07.27.550689</a></p>
        <p class="abstract">Autotrophy is the basis for complex life on Earth. Its physiological limits are largely constrained by rubisco - the enzyme that catalyzes almost all carbon fixation on the planet. Yet, with only a small fraction of rubisco diversity kinetically characterized so far, the underlying biological factors driving the evolution of fast rubiscos in nature remain unclear. We conducted a high-throughput kinetic characterization of over 100 prokaryotic form I rubiscos, the most ubiquitous group of rubisco sequences in nature, to uncover the determinants of rubiscos carboxylation velocity. We show that phototrophy and the presence of a carboxysome CO2 concentrating mechanism correlate with faster rubiscos (median rate difference of 3-5 fold). In contrast to previous assumptions, rubiscos originating from -cyanobacteria exhibit the highest carboxylation rates among form I enzymes ({approx}10 s-1 median versus &lt;7 s-1 in other groups). Our study ties biological and environmental insights associated with kinetic variation across rubiscos from nature.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551433">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551433" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551433">
        <p class="paperTitle">Pathogenic mutations of human phosphorylation sites affect protein-protein interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551433" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551433" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rrustemi, T.; Meyer, K.; Roske, Y.; Uyar, B.; Akalin, A.; Imami, K.; Ishihama, Y.; Daumke, O.; Selbach, M.</p>
        <p class="info">Score: 8.2, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551433' target='https://doi.org/10.1101/2023.08.01.551433'> 10.1101/2023.08.01.551433</a></p>
        <p class="abstract">Despite their lack of a defined 3D structure, intrinsically disordered regions (IDRs) of proteins play important biological roles. Many IDRs contain short linear motifs (SLiMs) that mediate protein-protein interactions (PPIs), which can be regulated by post-translational modifications like phosphorylation. 20% of pathogenic missense mutations are found in IDRs, and understanding how such mutations affect PPIs is essential for unraveling disease mechanisms. Here, we employed peptide-based interaction proteomics to investigate 36 disease-causing mutations affecting phosphorylation sites. Our results unveiled significant differences in interactomes between phosphorylated and non-phosphorylated peptides, often due to disrupted phosphorylation-dependent SLiMs. We focused on a mutation of a serine phosphorylation site in the transcription factor GATAD1, which causes dilated cardiomyopathy. We found that this phosphorylation site mediates interaction with 14-3-3 family proteins. Follow-up experiments revealed the structural basis of this interaction and suggest that 14-3-3 binding affects GATAD1 nucleocytoplasmic transport by masking a nuclear localisation signal. Our results demonstrate that pathogenic mutations of human phosphorylation sites can significantly impact protein-protein interactions, offering fresh insights into potential molecular mechanisms underlying pathogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551393">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551393" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551393">
        <p class="paperTitle">Density Physics-Informed Neural Network reveals sources of cell heterogeneity in signal transduction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551393" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551393" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jo, H.; Hong, H.; Hwang, H. J.; Chang, W.; Kim, J. K.</p>
        <p class="info">Score: 3.6, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551393' target='https://doi.org/10.1101/2023.07.31.551393'> 10.1101/2023.07.31.551393</a></p>
        <p class="abstract">The transduction time between signal initiation and final response provides valuable information on the underlying signaling pathway, including its speed and precision. Furthermore, multimodality in transduction-time distribution informs that the response is regulated by multiple pathways with different transduction speeds. Here, we developed Density physics-informed neural network (Density-PINN) to infer the transduction-time distribution, challenging to measure, from measurable final stress response time traces. We applied Density-PINN to single-cell gene expression data from 16 promoters regulated by unknown pathways in response to antibiotic stresses. We found that promoters with slower signaling initiation and transduction exhibit larger cell-to-cell heterogeneity in response intensity. However, this heterogeneity was greatly reduced when the response was regulated by slow and fast pathways together. This suggests a strategy for identifying effective signaling pathways for consistent cellular responses to disease treatments. Density-PINN can also be applied to understand various time delay systems, including infectious diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550757">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550757" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550757">
        <p class="paperTitle">Interpreting roles of mutations associated with the emergence of S. aureus USA300 strains using transcriptional regulatory network reconstruction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550757" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550757" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Poudel, S.; Hyun, J. C.; Hefner, Y.; Monk, J.; Nizet, V.; Palsson, B. O.</p>
        <p class="info">Score: 3.4, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550757' target='https://doi.org/10.1101/2023.07.26.550757'> 10.1101/2023.07.26.550757</a></p>
        <p class="abstract">The Staphylococcus aureus clonal complex 8 (CC8) is made up of several subtypes with varying levels of clinical burden; from community-associated methicillin resistant S. aureus (CA-MRSA) USA300 strains to hospital-associated (HA-MRSA) USA500 strains and basal methicillin susceptible (MSSA) strains. This phenotypic distribution within a single clonal complex makes CC8 an ideal clade to study the emergence of mutations important for antibiotic resistance and community spread. Gene level analysis comparing USA300 against MSSA and HA-MRSA strains have revealed key horizontally acquired genes important for its rapid spread in the community. However, efforts to define the contributions of point mutations and indels have been confounded by strong linkage disequilibrium resulting from clonal propagation. To break down this confounding effect, we combined genetic association testing with a model of the transcriptional regulatory network (TRN) to find candidate mutations that may have led to changes in gene regulation. First, we used a De Bruijn graph genome-wide association study (DBGWAS) to enrich mutations unique to the USA300 lineages within CC8. Next, we reconstructed the TRN by using Independent Component Analysis on 670 RNA sequencing samples from USA300 and non-USA300 CC8 strains which predicted several genes with strain-specific altered expression patterns. Examination of the regulatory region of one of the genes enriched by both approaches, isdH, revealed a 38 base pair deletion containing a Fur binding site and a conserved SNP which likely led to the altered expression levels in USA300 strains. Taken together, our results demonstrate the utility of reconstructed TRNs to address the limits of genetic approaches when studying emerging pathogenic strains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550388">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550388" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550388">
        <p class="paperTitle">Computational prediction of protein interactions on single cells by proximity sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550388" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550388" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xia, J.; Phan, H. V.; Vistain, L.; Chen, M.; Khan, A. A.; Tay, S.</p>
        <p class="info">Score: 3.1, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550388' target='https://doi.org/10.1101/2023.07.27.550388'> 10.1101/2023.07.27.550388</a></p>
        <p class="abstract">Proximity sequencing (Prox-seq) measures gene expression, protein expression, and protein complexes at the single cell level, using information from dual-antibody binding events and a single cell sequencing readout. Prox-seq provides multi-dimensional phenotyping of single cells and was recently used to track the formation of receptor complexes during inflammatory signaling in macrophages and to discover a new interaction between CD9/CD8 proteins on naive T cells. The distribution of protein abundance affects identification of protein complexes in a complicated manner in dual-binding assays like Prox-seq. These effects are difficult to explore with experiments, yet important for accurate quantification of protein complexes. Here, we introduce a physical model for protein dimer formation on single cells and computationally evaluate several different methods for reducing background noise when quantifying protein complexes. Furthermore, we developed an improved method for analysis of Prox-seq single-cell data, which resulted in more accurate and robust quantification of protein complexes. Finally, our model offers a simple way to investigate the behavior of Prox-seq under various biological conditions and guide users toward selecting the best analysis method for their data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.09.548232">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.09.548232" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.09.548232">
        <p class="paperTitle">Machine Learning identifies conserved traits that influence lifespan and healthspan responses to dietary restriction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.09.548232" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.09.548232" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hilsabeck, T. A.; Narayan, V. P.; Wilson, K. A.; Carrera, E.; Raftery, D.; Promislow, D. E.; Brem, R. B.; Campisi, J. A.; Kapahi, P.</p>
        <p class="info">Score: 5.3, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.09.548232' target='https://doi.org/10.1101/2023.07.09.548232'> 10.1101/2023.07.09.548232</a></p>
        <p class="abstract">Dietary restriction (DR) is the most robust means to extend lifespan and healthspan across species, but factors such as genetic variation affect how an individual will respond to DR. Additionally, it is unclear how cumulative variations in metabolism and the metabolome influence longevity and health. We utilized metabolomic, phenotypic, and genome-wide data from Drosophila Genetic Reference Panel strains raised under ad libitum and DR conditions to identify factors which influence longevity and health in response to dietary modulation. We found multiple intra-dataset correlations (e.g., metabolites with metabolites) but few inter-dataset correlations (e.g., metabolites with health-related phenotypes). Through random forest modeling across all traits and Mendelian Randomization, we found key translatable traits that influence lifespan or healthspan determination and validated the role of multiple metabolites in regulating lifespan. Through these approaches, we utilized data from flies and humans to elucidate potential therapeutic pathways and metabolomic targets for diet response, lifespan, and healthspan.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550573">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550573" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550573">
        <p class="paperTitle">Automatic design of gene regulatory mechanisms for spatial pattern formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550573" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550573" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mousavi, R.; Lobo, D.</p>
        <p class="info">Score: 3.5, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550573' target='https://doi.org/10.1101/2023.07.26.550573'> 10.1101/2023.07.26.550573</a></p>
        <p class="abstract">Gene regulatory mechanisms (GRMs) control the formation of spatial and temporal expression patterns that can serve as regulatory signals for the development of complex shapes. Synthetic developmental biology aims to engineer such mechanisms for understanding and producing desired multicellular patterns. However, the design of synthetic GRMs that can produce a target spatial gene expression pattern is a current challenge due to the nonlinear interactions and feedback loops present in genetic circuits. Here we present a methodology to automatically design GRMs that can produce any given spatial pattern. The proposed approach uses two orthogonal morphogen gradients acting as positional information signals in a multicellular tissue area or culture, which constitutes a continuous field of engineered cells implementing the same designed GRM. To efficiently design both the circuit network and the interaction mechanisms--including the number of genes required for the formation of the target pattern--we developed an automated algorithm based on high-performance evolutionary computation. The tolerance of the algorithm can be configured to design GRMs that are either simple to produce approximate patterns or complex to produce precise patterns. We demonstrate its performance by automatically designing GRMs for a diverse set of synthetic spatial expression patterns. The proposed method offers a versatile approach to systematically design and discover pattern-producing genetic circuits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.551059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.551059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.551059">
        <p class="paperTitle">The epigenetic response of mouse brown adipose tissue to cold stress: histone proteoform, DNA methylation, and RNA expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.551059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.551059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, B. C.; Steinthal, L. H.; Dias, M.; Yalamanchili, H. K.; Ochsner, S. A.; Zapata, G. E.; Mehta, N. R.; McKenna, N. J.; Young, N. L.; Nuotio-Antar, A. M.</p>
        <p class="info">Score: 2.5, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.551059' target='https://doi.org/10.1101/2023.07.30.551059'> 10.1101/2023.07.30.551059</a></p>
        <p class="abstract">Regulation of the thermogenic response by brown adipose tissue (BAT) is an important component of energy homeostasis with implications for the treatment of obesity and diabetes. Motivated to understand how BAT function is regulated epigenetically, we reanalyzed publicly available RNA-seq data and uncovered many nodes representing epigenetic modifiers that are altered in BAT in response to chronic thermogenic activation. Thus, we hypothesized that chronic thermogenic activation broadly alters epigenetic modifications of DNA and histones in BAT. To test our hypothesis, wildtype male C57BL/6J mice were housed under chronic conditions of thermoneutral temperature (TN, 28.8{degrees}C), mild cold/room temperature (RT, 22{degrees}C), or severe cold (SC, 8{degrees}C). BAT and reference tissue liver were subsequently dissected from each mouse. Reduced representation bisulfite sequencing (RRBS) reveals decreased methylation of promoters and intragenic regions in BAT genomic DNA in response to varying degrees of chronic cold exposure. Integration of our RRBS and the RNA-Seq dataset suggests a role for epigenetic modification of DNA in gene regulation in response to cold. To analyze histone modifications in BAT, we develop a robust method for the isolation of histones and report the first quantitation of histone H3.2 and H4 proteoforms. We initially observe that BAT and liver exhibit different histone proteoforms. Next, we report housing temperature-dependent changes in histone proteoforms in BAT. We observe differences in bivalent proteoforms such as H3{K9me2,K23ac,K36me1}, H3{K9me3,K23ac,K36me1} and H4&lt;K8ACK16ACK20ME3&gt;. Taken together, our results provide novel findings supporting global epigenetic modification in murine BAT in response to varying degrees of chronic cold stimuli and establish a methodology to quantitatively study histones in BAT, allowing for direct comparisons to decipher mechanistic changes during the thermogenic response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548371">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548371" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548371">
        <p class="paperTitle">Quantifying mutational synergy using computational models predicts survival in haematological cancers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548371" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548371" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Norris, R.; Jones, J.; Mancini, E.; Chevassut, T.; Pepper, C.; Pepper, A.; Mitchell, S.</p>
        <p class="info">Score: 5.9, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548371' target='https://doi.org/10.1101/2023.07.10.548371'> 10.1101/2023.07.10.548371</a></p>
        <p class="abstract">Genetic heterogeneity and co-occurring driver mutations contribute to poor clinical outcomes in cancer. However, the impact of multiple mutations on complex signalling networks is not easily predicted. We found that, by placing mutations into their cellular context, multi-scale agent- based mathematical models could predict how genetic events combine in haematological malignancies. Simulations of lymphoma and myeloma predicted co-occurring mutations synergised to increase tumour cell expansion beyond what would be expected from the impact of the individual mutations alone. Mutational synergy between MYC and BCL2 was consistent with the more aggressive disease course of patients with double-hit lymphoma, and mutational synergy between MCL1 and CKS1B was predictive of outcome in patients with gain 1q multiple myeloma. Incorporating patient-specific mutational profiles into personalised models of lymphoma patients with the worst clinical outcomes revealed a correlation between simulated mutational synergy and overall survival, which outperformed widely used classifications of lymphoma informed by gene expression or mutational data alone. Our results demonstrated that mutational synergy scores enabled prediction of the impact of co-occurring mutations and may improve personalised prognostic predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550715">
        <p class="paperTitle">Enhanced cell segmentation with limited annotated data using generative adversarial networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zargari, A.; Mashhadi, N.; Shariati, A.</p>
        <p class="info">Score: 3.7, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550715' target='https://doi.org/10.1101/2023.07.26.550715'> 10.1101/2023.07.26.550715</a></p>
        <p class="abstract">The application of deep learning is rapidly transforming the field of bioimage analysis. While deep learning has shown great promise in complex microscopy tasks such as single-cell segmentation, the development of generalizable foundation deep learning segmentation models is hampered by the scarcity of large and diverse annotated datasets of cell images for training purposes. Generative Adversarial Networks (GANs) can generate realistic images that can potentially be easily used to train deep learning models without the generation of large manually annotated microscopy images. Here, we propose a customized CycleGAN architecture to train an enhanced cell segmentation model with limited annotated cell images, effectively addressing the challenge of paucity of annotated data in microscopy imaging. Our customized CycleGAN model can generate realistic synthetic images of cells with morphological details and nuances very similar to that of real images. This method not only increases the variability seen during training but also enhances the authenticity of synthetic samples, thereby enhancing the overall predictive accuracy and robustness of the cell segmentation model. Our experimental results show that our CycleGAN-based method significantly improves the performance of the segmentation model compared to conventional training techniques. Interestingly, we demonstrate that our model can extrapolate its knowledge by synthesizing imaging scenarios that were not seen during the training process. Our proposed customized CycleGAN method will accelerate the development of foundation models for cell segmentation in microscopy images.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
