<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="LIANA&#43;: an all-in-one cell-cell communication framework
Authors: Dimitrov, D.; Schäfer, P. S. L.; Farr, E.; Rodriguez Mier, P.; Lobentanzer, S.; Dugourd, A.; Tanevski, J.; Ramirez Flores, R. O.; Saez-Rodriguez, J.
Score: 41.4, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553863
The growing availability of single-cell and spatially-resolved transcriptomics has led to the rapidly growing popularity of methods to infer cell-cell communication. Many approaches have emerged, each capturing only a partial view of the complex landscape of cell-cell communication.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="LIANA&#43;: an all-in-one cell-cell communication framework
Authors: Dimitrov, D.; Schäfer, P. S. L.; Farr, E.; Rodriguez Mier, P.; Lobentanzer, S.; Dugourd, A.; Tanevski, J.; Ramirez Flores, R. O.; Saez-Rodriguez, J.
Score: 41.4, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553863
The growing availability of single-cell and spatially-resolved transcriptomics has led to the rapidly growing popularity of methods to infer cell-cell communication. Many approaches have emerged, each capturing only a partial view of the complex landscape of cell-cell communication." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="LIANA&#43;: an all-in-one cell-cell communication framework
Authors: Dimitrov, D.; Schäfer, P. S. L.; Farr, E.; Rodriguez Mier, P.; Lobentanzer, S.; Dugourd, A.; Tanevski, J.; Ramirez Flores, R. O.; Saez-Rodriguez, J.
Score: 41.4, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553863
The growing availability of single-cell and spatially-resolved transcriptomics has led to the rapidly growing popularity of methods to infer cell-cell communication. Many approaches have emerged, each capturing only a partial view of the complex landscape of cell-cell communication."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "LIANA\u0026#43;: an all-in-one cell-cell communication framework\nAuthors: Dimitrov, D.; Schäfer, P. S. L.; Farr, E.; Rodriguez Mier, P.; Lobentanzer, S.; Dugourd, A.; Tanevski, J.; Ramirez Flores, R. O.; Saez-Rodriguez, J.\nScore: 41.4, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553863\nThe growing availability of single-cell and spatially-resolved transcriptomics has led to the rapidly growing popularity of methods to infer cell-cell communication. Many approaches have emerged, each capturing only a partial view of the complex landscape of cell-cell communication.",
  "keywords": [
    
  ],
  "articleBody": " LIANA+: an all-in-one cell-cell communication framework\nAuthors: Dimitrov, D.; Schäfer, P. S. L.; Farr, E.; Rodriguez Mier, P.; Lobentanzer, S.; Dugourd, A.; Tanevski, J.; Ramirez Flores, R. O.; Saez-Rodriguez, J.\nScore: 41.4, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553863\nThe growing availability of single-cell and spatially-resolved transcriptomics has led to the rapidly growing popularity of methods to infer cell-cell communication. Many approaches have emerged, each capturing only a partial view of the complex landscape of cell-cell communication. Here, we present LIANA+, a scalable framework to decode coordinated inter- and intracellular signalling events from single- and multi-condition datasets in both single-cell and spatially-resolved data. Beyond integrating and extending established methodologies and a rich knowledge base, LIANA+ enables novel analyses using diverse molecular mediators, including those measured in multi-omics data. Accessible as an open-source Python package at https://github.com/saezlab/liana-py, LIANA+ provides a comprehensive set of synergistic components to study cell-cell communication.\nIntegration of Thermal Proteome Profiling with phosphoproteomic and transcriptomic data via mechanistic network models decodes the molecular response to PARP inhibition\nAuthors: Burtscher, M. L.; Gade, S.; Garrido-Rodriguez, M.; Rutkowska, A.; Werner, T.; Eberl, H. C.; Petretich, M.; Knopf, N.; Zirngibl, K.; Grandi, P.; Bergamini, G.; Bantscheff, M.; Faelth-Savitski, M.; Saez-Rodriguez, J.\nScore: 18.1, Published: 2023-08-16 DOI: 10.1101/2023.08.15.553354\nThe deregulation of complex diseases often spans multiple molecular processes. A multimodal functional characterization of these processes can shed light on the disease mechanisms and the effect of drugs. Thermal Proteome Profiling (TPP) is a mass-spectrometry based technique assessing changes in thermal protein stability that can serve as proxies of functional changes of the proteome. These unique insights of TPP can complement those obtained by other omics technologies. Here, we show how TPP can be integrated with phosphoproteomics and transcriptomics in a network-based approach using COSMOS, a framework for causal integration of multi-omics, to provide an integrated view of transcription factors, kinases and proteins with altered thermal stability. This allowed us to recover known mechanistic consequences of PARP inhibition in ovarian cancer cells on cell cycle and DNA damage response in detail and to uncover new insights into drug response mechanisms related to interferon and hippo signaling. We found that TPP complements the other omics data and allowed us to obtain a network model with higher coverage of the main underlying mechanisms. These results illustrate the added value of TPP, and more generally the power of network models to integrate the information provided by different omics technologies. We anticipate that this strategy can be used to broadly integrate functional proteomics with other omics to study complex molecular processes. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=63 SRC=\"FIGDIR/small/553354v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (18K): org.highwire.dtl.DTLVardef@7fac35org.highwire.dtl.DTLVardef@2d71dorg.highwire.dtl.DTLVardef@78bd3borg.highwire.dtl.DTLVardef@cf9efe_HPS_FORMAT_FIGEXP M_FIG C_FIG\nPrevalence of and gene regulatory constraints on transcriptional adaptation in single cells\nAuthors: Mellis, I. A.; Bodkin, N.; Goyal, Y.\nScore: 17.9, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553318\nCells and tissues have a remarkable ability to adapt to genetic perturbations via a variety of molecular mechanisms. Nonsense-induced transcriptional compensation, a form of transcriptional adaptation, has recently emerged as one such mechanism, in which nonsense mutations in a gene can trigger upregulation of related genes, possibly conferring robustness at cellular and organismal levels. However, beyond a handful of developmental contexts and curated sets of genes, to date, no comprehensive genome-wide investigation of this behavior has been undertaken for mammalian cell types and contexts. Moreover, how the regulatory-level effects of inherently stochastic compensatory gene networks contribute to phenotypic penetrance in single cells remains unclear. Here we combine computational analysis of existing datasets with stochastic mathematical modeling and machine learning to uncover the widespread prevalence of transcriptional adaptation in mammalian systems and the diverse single-cell manifestations of minimal compensatory gene networks. Regulon gene expression analysis of a pooled single-cell genetic perturbation dataset recapitulates our model predictions. Our integrative approach uncovers several putative hits-genes demonstrating possible transcriptional adaptation-to follow up on experimentally, and provides a formal quantitative framework to test and refine models of transcriptional adaptation.\nSystematic discovery of protein interaction interfaces using AlphaFold and experimental validation\nAuthors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.\nScore: 60.9, Published: 2023-08-09 DOI: 10.1101/2023.08.07.552219\nStructural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants. However, structural data is still missing for most protein interactions because we lack computational and experimental tools at scale. We thoroughly assessed AlphaFold-Multimer accuracy for structure prediction of interactions involving folded domains binding to short linear motifs from the ELM database. The structure predictions were highly sensitive but not very specific when using small protein fragments. Sensitivity decreased substantially when using long protein fragments or full length proteins with intrinsically disordered regions. We delineated a fragmentation strategy to optimize sensitivity and applied it to interactions between proteins associated with neurodevelopmental disorders. This enabled prediction of highly confident and likely disease-related novel interfaces, but also resulted in many high scoring false positive predictions. Experiments supported predicted interfaces between CREBZF-HCFC1, FBXO23-STX1B, STX1B-VAMP2, ESRRG-PSMC5, PEX3-PEX19, PEX3-PEX16, and SNRPB-GIGYF1 providing novel molecular insights for diverse biological processes. Our work highlights exciting perspectives, but also reveals clear limitations and the need for future developments to maximize the power of Alphafold-Multimer for interface predictions.\nLarge-scale analysis of the integration of enhancer-enhancer signals by promoters\nAuthors: Martinez-Ara, M.; Comoglio, F.; van Steensel, B.\nScore: 9.4, Published: 2023-08-14 DOI: 10.1101/2023.08.11.552995\nGenes are often regulated by multiple enhancers. It is poorly understood how the individual enhancer activities are combined to control promoter activity. Anecdotal evidence has shown that enhancers can combine sub-additively, additively, synergistically, or redundantly. However, it is not clear which of these modes are more frequent in mammalian genomes. Here, we systematically tested how pairs of enhancers activate promoters using a three-way combinatorial reporter assay in mouse cells. By assaying about 69,000 enhancer-enhancer-promoter combinations we found that enhancer pairs generally combine near-additively. This behaviour was conserved across seven developmental promoters tested. Surprisingly, these promoters scale the enhancer signals approximately following a power-law, but the exponent of this response varies between promoters. A housekeeping promoter showed an overall different response to enhancer pairs, and a smaller dynamic range. Thus, our data indicate that enhancers mostly act additively, but promoters transform their collective effect non-linearly.\nRadiation necrosis after radiation therapy treatment of brain metastases: A computational approach\nAuthors: Ocana-Tienda, B.; Leon-Triana, O.; Perez-Beteta, J.; Perez-Garcia, V. M.\nScore: 7.6, Published: 2023-08-03 DOI: 10.1101/2023.08.01.551411\nMetastasis is the process through which cancer cells break away from a primary tumor, travel through the blood or lymph system, and form new tumors in distant tissues. One of the preferred sites for metastatic dissemination is the brain, affecting more than 20% of all cancer patients. This figure is increasing steadily due to improvements in treatments of primary tumors. Stereotactic radiosurgery (SRS) is one of the main treatment options for patients with a small or moderate number of brain metastases (BMs). A frequent adverse event of SRS is radiation necrosis (RN), an inflammatory condition caused by late normal tissue cell death. A major diagnostic problem is that RNs are difficult to distinguish from BM recurrences, due to their similarities on standard magnetic resonance images (MRIs). However, this distinction is key to choosing the best therapeutic approach since RNs resolve often without further interventions, while relapsing BMs may require open brain surgery. Recent research has shown that RNs have a faster growth dynamics than recurrent BMs, providing a way to differentiate the two entities, but no mechanistic explanation has been provided for those observations. In this study, computational frameworks were developed based on mathematical models of increasing complexity, providing mechanistic explanations for the differential growth dynamics of BMs relapse versus RN events and explaining the observed clinical phenomenology. Simulated tumor relapses were found to have growth exponents substantially smaller than the group in which there was inflammation due to damage induced by SRS to normal brain tissue adjacent to the BMs, thus leading to RN. ROC curves with the synthetic data had an optimal threshold that maximized the sensitivity and specificity values for a growth exponent {beta}* = 1.05, very close to that observed in patient datasets. Author summaryAfter treatment of brain metastases with radiation therapy, a fraction of patients experience tumor recurrences and others display radiation necrosis (RN). Clinical data shows that the growth of RNs is faster, as measured by the growth exponent, than that of recurrent brain metastases. This reflects the inflammatory nature of the former, and provides a method to distinguish RN in the clinics from relapsing metastatic lesions. A simple mathematical model for the inflammatory response and a more sophisticate discrete stochastic simulator with many biological details were been developed to provide a mechanistic explanation of the differential dynamics of tumor growth versus inflammatory responses after stereotactic radiation surgery of metastatic brain lesions.\nMetabolic Symbiosis between Oxygenated and Hypoxic Tumour Cells: An Agent-based Modelling Study\nAuthors: Jayathilake, P.; Victori, P.; Pavillet, C.; Voukantsis, D.; Miar Cuervo, A.; Arora, A.; Harris, A. L.; Morten, K.; Buffa, F. M.\nScore: 7.6, Published: 2023-07-24 DOI: 10.1101/2023.07.24.550289\nDeregulated metabolism is one of the hallmarks of cancer. It is well-known that tumour cells tend to metabolize glucose via glycolysis even when oxygen is available and mitochondrial respiration is functional. However, the lower energy efficiency of aerobic glycolysis with respect to mitochondrial respiration makes this behaviour, namely the Warburg effect, counter-intuitive, although it has now been recognized as a source of anabolic precursors. On the other hand, there is evidence that oxygenated tumour cells could be fuelled by exogenous lactate produced from glycolysis, and that glycolytic tumour cells export lactate to the environment. Using a multi-scale approach combining multi-agent modelling, diffusion-reaction and stoichiometric equations, we simulated these cell populations and their environment, and studied the metabolic co-operation between cells exposed to different oxygen concentration and nutrient abundance. The results show that the metabolic cooperation between cells reduces the depletion of environmental glucose, resulting in an overall advantage of using aerobic glycolysis. In addition, the environmental oxygen level was found to be decreased by the symbiosis, promoting a further shift towards anaerobic glycolysis. However, the oxygenated and hypoxic cell populations may gradually reach quasi-equilibrium. A sensitivity analysis based on Latin hypercube sampling and partial rank correlation shows that the symbiotic dynamics depends on properties of the specific cell such as the minimum glucose level needed for glycolysis. Our results suggest that strategies that block glucose transporters may be more effective than those blocking lactate intake transporters to reduce tumour growth by blocking lactate production. AUTHOR SUMMARYMetabolic alteration is one of the hallmarks of cancer and the well-known metabolic alteration of tumour cells is that cells prefer to do glycolysis over mitochondrial respiration even under well-oxygenated and functional mitochondrial conditions. On the other hand, there is evidence that oxygenated tumour cells could be fuelled by exogenous lactate produced from hypoxic glycolytic cells in which it can create a metabolic co-operation between oxygenated and hypoxic cell populations. This metabolic co-operation could allow tumour cells to economically share oxygen and glucose and promote tumour survival. Using a multi-scale approach combining multi-agent modelling, diffusion-reaction and stoichiometric equations, we studied this metabolic co-operation between different populations of cells, exposed to a changing microenvironment. We predict that the tumour environmental glucose depletion is decreased while the oxygen depletion is increased by this metabolic symbiosis, promoting a further shift towards glycolysis. Our results also show that blocking glucose transporters could be more effective than blocking lactate intake transporters, because the former would disrupt both glycolysis and lactate production, drastically reducing tumour growth.\nReverse engineering morphogenesis through Bayesian optimization of physics-based models\nAuthors: Kumar, N.; Dowling, A.; Zartman, J. J.\nScore: 3.9, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553928\nMorphogenetic programs direct the cell signaling and nonlinear mechanical interactions between multiple cell types and tissue layers to define organ shape and size. A key challenge for systems and synthetic biology is determining optimal combinations of intra- and inter-cellular interactions to predict an organ\\'s shape, size, and function. Physics-based mechanistic models that define the subcellular force distribution facilitate this, but it is extremely challenging to calibrate parameters in these models from data. To solve this inverse problem, we created a Bayesian optimization framework to determine the optimal cellular force distribution such that the predicted organ shapes match the desired organ shapes observed within the experimental imaging data. This integrative framework employs Gaussian Process Regression (GPR), a non-parametric kernel-based probabilistic machine learning modeling paradigm, to learn the mapping functions relating to the morphogenetic programs that generate and maintain the final organ shape. We calibrated and tested the method on cross-sections of Drosophila wing imaginal discs, a highly informative model organ system, to study mechanisms that regulate epithelial processes that range from development to cancer. As a specific test case, the parameter estimation framework successfully infers the underlying changes in core parameters needed to match simulation data with time series imaging data of wing discs perturbed with collagenase. Unexpectedly, the framework also identifies multiple distinct parameter sets that generate shapes similar to wild-type organ shapes. This platform enables an efficient, global sensitivity analysis to support the necessity of both actomyosin contractility and basal ECM stiffness to generate and maintain the curved shape of the wing imaginal disc. The optimization framework, combined with fixed tissue imaging, identified that Piezo, a mechanosensitive ion channel, impacts fold formation by regulating the apical-basal balance of actomyosin contractility and elasticity of ECM. This framework is extensible toward reverse-engineering the morphogenesis of any organ system and can be utilized in real-time control of complex multicellular systems.\nThe interplay between lumen pressure and cell proliferation determines organoid morphology in a multicellular phase field model\nAuthors: Tanida, S.; Fuji, K.; Lu, L.; Guyomar, T.; Lee, B. H.; Honigmann, A.; Grapin-Botton, A.; Riveline, D.; Nonomura, M.; Sano, M.; Hiraiwa, T.\nScore: 3.4, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553655\nOrganoids are ideal systems to predict the phenotypes of organs. However, there is currently a lack of understanding regarding the generalized rules that enable use of simple cellular principles to make morphological predictions of entire organoids. Therefore, we employed a phase field model with the following basic components: the minimum conditions for the timing and volume of cell division, lumen nucleation rules, and lumenal pressure. Through our model, we could compute and generate a myriad of organoid phenotypes observed till date. We propose morphological indices necessary to characterize the shapes and construct phase diagrams and show their dependencies on proliferation time and lumen pressure. Additionally, we introduced the lumen-index parameter, which helped in examining the criteria to maintain organoids as spherical structures comprising a single layer of cells and enclosing an intact lumen. Finally, we predict a star-like organoid phenotype that did not undergo differentiation, suggesting that the volume constraint during cell division may determine the final phenotype. In summary, our approach provides researchers with guidelines to test the mechanisms of self-organization and predict the shape of organoid. Author summaryIn nature, a wide variety of organ morphologies are observed. Owing to the complexity of the process underlying the acquisition of organs morphology, it is challenging to investigate the mechanisms that lead to such variations. A promising approach to study these variations is the use of \"computational organoid\" study, which is the computational-based study of self-organizing shapes in multicellular assemblies and fluid-filled cavities called lumens that develop from a few proliferating cells. This study explores general mechanisms that dictate how various mechanical factors affect the growing self-organized multicellular assembly. We relied on computer simulations of the mathematical model called multicellular phase-field model with lumens and explored the mechanical factor effects, such as the lumen pressure while considering the time and volume conditions required for cell division. These simulations generated and categorized a wide range of organoid phenotypes based on the varying lumen pressure and cell division conditions. These phenotypes were characterized into seven distinct classes, based on the morphological index sets, including a cellular monolayer/multilayer surrounding single or multiple lumens and branch formation. These phenotypes were obtained without the assumption of differentiation. Our study elucidates the mechanisms underlying the organoid and organ formation with different shapes, thereby highlighting the significance of mechanical forces in shaping these complex biological structures.\nTwo-way feedback between chromatin compaction and histone modification state explains S. cerevisiae heterochromatin bistability\nAuthors: Movilla Miangolarra, A.; Saxton, D. S.; Yan, Z.; Rine, J.; Howard, M.\nScore: 3.5, Published: 2023-08-15 DOI: 10.1101/2023.08.12.552948\nCompact chromatin is closely linked with gene silencing in part by sterically masking access to promoters, inhibiting transcription factor binding and preventing polymerase from efficiently transcribing a gene. Here, we propose a broader view: chromatin compaction can be both a cause and a consequence of the histone modification state, and this tight bidirectional interaction can underpin bistable transcriptional states. To test this theory, we developed a mathematical model for the dynamics of the HMR locus in S. cerevisiae, that incorporates activating histone modifications, silencing proteins and a dynamic, acetylation-dependent, three-dimensional locus size. Chromatin compaction enhances silencer protein binding, which in turn feeds back to remove activating histone modifications, leading to further compaction. The bistable output of the model was in good agreement with prior quantitative data, including switching rates from expressed to silent states, and vice versa, and protein binding levels within the locus. We then tested the model by predicting changes in switching rates as the genetic length of the locus was increased, which were then experimentally verified. This bidirectional feedback between chromatin compaction and the histone modification state may be an important regulatory mechanism at many loci. SignificanceChromatin is the complex formed by proteins, including histones, and DNA to form chromosomes. Specific chromatin structures and states are thought to be key factors regulating transcription. A common view proposes that histone modifications activate or inhibit transcription either via specific activation or inhibition of RNA polymerase binding/elongation at a locus, or by expanding/compacting the locus, thereby modulating its accessibility to many macromolecules. In this work, we elucidated a broader hypothesis that chromatin compaction may both inhibit transcription, and feedback via silencing proteins to remove histone modifications that further control chromatin compaction and correlate with gene activity. We developed a model incorporating these ideas and showed that it explains quantitative experimental data for a silent locus in budding yeast.\n",
  "wordCount" : "3035",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.553863">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.553863" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.553863">
        <p class="paperTitle">LIANA&#43;: an all-in-one cell-cell communication framework</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.553863" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.553863" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dimitrov, D.; Schäfer, P. S. L.; Farr, E.; Rodriguez Mier, P.; Lobentanzer, S.; Dugourd, A.; Tanevski, J.; Ramirez Flores, R. O.; Saez-Rodriguez, J.</p>
        <p class="info">Score: 41.4, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.553863' target='https://doi.org/10.1101/2023.08.19.553863'> 10.1101/2023.08.19.553863</a></p>
        <p class="abstract">The growing availability of single-cell and spatially-resolved transcriptomics has led to the rapidly growing popularity of methods to infer cell-cell communication. Many approaches have emerged, each capturing only a partial view of the complex landscape of cell-cell communication. Here, we present LIANA&#43;, a scalable framework to decode coordinated inter- and intracellular signalling events from single- and multi-condition datasets in both single-cell and spatially-resolved data. Beyond integrating and extending established methodologies and a rich knowledge base, LIANA&#43; enables novel analyses using diverse molecular mediators, including those measured in multi-omics data. Accessible as an open-source Python package at https://github.com/saezlab/liana-py, LIANA&#43; provides a comprehensive set of synergistic components to study cell-cell communication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553354" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553354">
        <p class="paperTitle">Integration of Thermal Proteome Profiling with phosphoproteomic and transcriptomic data via mechanistic network models decodes the molecular response to PARP inhibition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burtscher, M. L.; Gade, S.; Garrido-Rodriguez, M.; Rutkowska, A.; Werner, T.; Eberl, H. C.; Petretich, M.; Knopf, N.; Zirngibl, K.; Grandi, P.; Bergamini, G.; Bantscheff, M.; Faelth-Savitski, M.; Saez-Rodriguez, J.</p>
        <p class="info">Score: 18.1, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553354' target='https://doi.org/10.1101/2023.08.15.553354'> 10.1101/2023.08.15.553354</a></p>
        <p class="abstract">The deregulation of complex diseases often spans multiple molecular processes. A multimodal functional characterization of these processes can shed light on the disease mechanisms and the effect of drugs. Thermal Proteome Profiling (TPP) is a mass-spectrometry based technique assessing changes in thermal protein stability that can serve as proxies of functional changes of the proteome. These unique insights of TPP can complement those obtained by other omics technologies. Here, we show how TPP can be integrated with phosphoproteomics and transcriptomics in a network-based approach using COSMOS, a framework for causal integration of multi-omics, to provide an integrated view of transcription factors, kinases and proteins with altered thermal stability. This allowed us to recover known mechanistic consequences of PARP inhibition in ovarian cancer cells on cell cycle and DNA damage response in detail and to uncover new insights into drug response mechanisms related to interferon and hippo signaling. We found that TPP complements the other omics data and allowed us to obtain a network model with higher coverage of the main underlying mechanisms. These results illustrate the added value of TPP, and more generally the power of network models to integrate the information provided by different omics technologies. We anticipate that this strategy can be used to broadly integrate functional proteomics with other omics to study complex molecular processes.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=63 SRC=&#34;FIGDIR/small/553354v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (18K):
org.highwire.dtl.DTLVardef@7fac35org.highwire.dtl.DTLVardef@2d71dorg.highwire.dtl.DTLVardef@78bd3borg.highwire.dtl.DTLVardef@cf9efe_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553318">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553318" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553318">
        <p class="paperTitle">Prevalence of and gene regulatory constraints on transcriptional adaptation in single cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553318" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553318" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mellis, I. A.; Bodkin, N.; Goyal, Y.</p>
        <p class="info">Score: 17.9, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553318' target='https://doi.org/10.1101/2023.08.14.553318'> 10.1101/2023.08.14.553318</a></p>
        <p class="abstract">Cells and tissues have a remarkable ability to adapt to genetic perturbations via a variety of molecular mechanisms. Nonsense-induced transcriptional compensation, a form of transcriptional adaptation, has recently emerged as one such mechanism, in which nonsense mutations in a gene can trigger upregulation of related genes, possibly conferring robustness at cellular and organismal levels. However, beyond a handful of developmental contexts and curated sets of genes, to date, no comprehensive genome-wide investigation of this behavior has been undertaken for mammalian cell types and contexts. Moreover, how the regulatory-level effects of inherently stochastic compensatory gene networks contribute to phenotypic penetrance in single cells remains unclear. Here we combine computational analysis of existing datasets with stochastic mathematical modeling and machine learning to uncover the widespread prevalence of transcriptional adaptation in mammalian systems and the diverse single-cell manifestations of minimal compensatory gene networks. Regulon gene expression analysis of a pooled single-cell genetic perturbation dataset recapitulates our model predictions. Our integrative approach uncovers several putative hits-genes demonstrating possible transcriptional adaptation-to follow up on experimentally, and provides a formal quantitative framework to test and refine models of transcriptional adaptation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552219" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552219">
        <p class="paperTitle">Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, C. Y.; Hubrich, D.; Varga, J. K.; Schäfer, C.; Welzel, M.; Schumbera, E.; Đokic, M.; Strom, J. M.; Schönfeld, J.; Geist, J. L.; Polat, F.; Gibson, T. J.; Keller Valsecchi, C. I.; Kumar, M.; Schueler-Furman, O.; Luck, K.</p>
        <p class="info">Score: 60.9, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552219' target='https://doi.org/10.1101/2023.08.07.552219'> 10.1101/2023.08.07.552219</a></p>
        <p class="abstract">Structural resolution of protein interactions enables mechanistic and functional studies as well as interpretation of disease variants. However, structural data is still missing for most protein interactions because we lack computational and experimental tools at scale. We thoroughly assessed AlphaFold-Multimer accuracy for structure prediction of interactions involving folded domains binding to short linear motifs from the ELM database. The structure predictions were highly sensitive but not very specific when using small protein fragments. Sensitivity decreased substantially when using long protein fragments or full length proteins with intrinsically disordered regions. We delineated a fragmentation strategy to optimize sensitivity and applied it to interactions between proteins associated with neurodevelopmental disorders. This enabled prediction of highly confident and likely disease-related novel interfaces, but also resulted in many high scoring false positive predictions. Experiments supported predicted interfaces between CREBZF-HCFC1, FBXO23-STX1B, STX1B-VAMP2, ESRRG-PSMC5, PEX3-PEX19, PEX3-PEX16, and SNRPB-GIGYF1 providing novel molecular insights for diverse biological processes. Our work highlights exciting perspectives, but also reveals clear limitations and the need for future developments to maximize the power of Alphafold-Multimer for interface predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552995">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552995" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552995">
        <p class="paperTitle">Large-scale analysis of the integration of enhancer-enhancer signals by promoters</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552995" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552995" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Ara, M.; Comoglio, F.; van Steensel, B.</p>
        <p class="info">Score: 9.4, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552995' target='https://doi.org/10.1101/2023.08.11.552995'> 10.1101/2023.08.11.552995</a></p>
        <p class="abstract">Genes are often regulated by multiple enhancers. It is poorly understood how the individual enhancer activities are combined to control promoter activity. Anecdotal evidence has shown that enhancers can combine sub-additively, additively, synergistically, or redundantly. However, it is not clear which of these modes are more frequent in mammalian genomes. Here, we systematically tested how pairs of enhancers activate promoters using a three-way combinatorial reporter assay in mouse cells. By assaying about 69,000 enhancer-enhancer-promoter combinations we found that enhancer pairs generally combine near-additively. This behaviour was conserved across seven developmental promoters tested. Surprisingly, these promoters scale the enhancer signals approximately following a power-law, but the exponent of this response varies between promoters. A housekeeping promoter showed an overall different response to enhancer pairs, and a smaller dynamic range. Thus, our data indicate that enhancers mostly act additively, but promoters transform their collective effect non-linearly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551411">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551411" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551411">
        <p class="paperTitle">Radiation necrosis after radiation therapy treatment of brain metastases: A computational approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551411" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551411" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ocana-Tienda, B.; Leon-Triana, O.; Perez-Beteta, J.; Perez-Garcia, V. M.</p>
        <p class="info">Score: 7.6, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551411' target='https://doi.org/10.1101/2023.08.01.551411'> 10.1101/2023.08.01.551411</a></p>
        <p class="abstract">Metastasis is the process through which cancer cells break away from a primary tumor, travel through the blood or lymph system, and form new tumors in distant tissues. One of the preferred sites for metastatic dissemination is the brain, affecting more than 20% of all cancer patients. This figure is increasing steadily due to improvements in treatments of primary tumors. Stereotactic radiosurgery (SRS) is one of the main treatment options for patients with a small or moderate number of brain metastases (BMs). A frequent adverse event of SRS is radiation necrosis (RN), an inflammatory condition caused by late normal tissue cell death. A major diagnostic problem is that RNs are difficult to distinguish from BM recurrences, due to their similarities on standard magnetic resonance images (MRIs). However, this distinction is key to choosing the best therapeutic approach since RNs resolve often without further interventions, while relapsing BMs may require open brain surgery. Recent research has shown that RNs have a faster growth dynamics than recurrent BMs, providing a way to differentiate the two entities, but no mechanistic explanation has been provided for those observations.

In this study, computational frameworks were developed based on mathematical models of increasing complexity, providing mechanistic explanations for the differential growth dynamics of BMs relapse versus RN events and explaining the observed clinical phenomenology. Simulated tumor relapses were found to have growth exponents substantially smaller than the group in which there was inflammation due to damage induced by SRS to normal brain tissue adjacent to the BMs, thus leading to RN. ROC curves with the synthetic data had an optimal threshold that maximized the sensitivity and specificity values for a growth exponent {beta}* = 1.05, very close to that observed in patient datasets.

Author summaryAfter treatment of brain metastases with radiation therapy, a fraction of patients experience tumor recurrences and others display radiation necrosis (RN). Clinical data shows that the growth of RNs is faster, as measured by the growth exponent, than that of recurrent brain metastases. This reflects the inflammatory nature of the former, and provides a method to distinguish RN in the clinics from relapsing metastatic lesions. A simple mathematical model for the inflammatory response and a more sophisticate discrete stochastic simulator with many biological details were been developed to provide a mechanistic explanation of the differential dynamics of tumor growth versus inflammatory responses after stereotactic radiation surgery of metastatic brain lesions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550289">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550289" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550289">
        <p class="paperTitle">Metabolic Symbiosis between Oxygenated and Hypoxic Tumour Cells: An Agent-based Modelling Study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550289" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550289" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jayathilake, P.; Victori, P.; Pavillet, C.; Voukantsis, D.; Miar Cuervo, A.; Arora, A.; Harris, A. L.; Morten, K.; Buffa, F. M.</p>
        <p class="info">Score: 7.6, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550289' target='https://doi.org/10.1101/2023.07.24.550289'> 10.1101/2023.07.24.550289</a></p>
        <p class="abstract">Deregulated metabolism is one of the hallmarks of cancer. It is well-known that tumour cells tend to metabolize glucose via glycolysis even when oxygen is available and mitochondrial respiration is functional. However, the lower energy efficiency of aerobic glycolysis with respect to mitochondrial respiration makes this behaviour, namely the Warburg effect, counter-intuitive, although it has now been recognized as a source of anabolic precursors. On the other hand, there is evidence that oxygenated tumour cells could be fuelled by exogenous lactate produced from glycolysis, and that glycolytic tumour cells export lactate to the environment. Using a multi-scale approach combining multi-agent modelling, diffusion-reaction and stoichiometric equations, we simulated these cell populations and their environment, and studied the metabolic co-operation between cells exposed to different oxygen concentration and nutrient abundance. The results show that the metabolic cooperation between cells reduces the depletion of environmental glucose, resulting in an overall advantage of using aerobic glycolysis. In addition, the environmental oxygen level was found to be decreased by the symbiosis, promoting a further shift towards anaerobic glycolysis. However, the oxygenated and hypoxic cell populations may gradually reach quasi-equilibrium. A sensitivity analysis based on Latin hypercube sampling and partial rank correlation shows that the symbiotic dynamics depends on properties of the specific cell such as the minimum glucose level needed for glycolysis. Our results suggest that strategies that block glucose transporters may be more effective than those blocking lactate intake transporters to reduce tumour growth by blocking lactate production.

AUTHOR SUMMARYMetabolic alteration is one of the hallmarks of cancer and the well-known metabolic alteration of tumour cells is that cells prefer to do glycolysis over mitochondrial respiration even under well-oxygenated and functional mitochondrial conditions. On the other hand, there is evidence that oxygenated tumour cells could be fuelled by exogenous lactate produced from hypoxic glycolytic cells in which it can create a metabolic co-operation between oxygenated and hypoxic cell populations. This metabolic co-operation could allow tumour cells to economically share oxygen and glucose and promote tumour survival. Using a multi-scale approach combining multi-agent modelling, diffusion-reaction and stoichiometric equations, we studied this metabolic co-operation between different populations of cells, exposed to a changing microenvironment. We predict that the tumour environmental glucose depletion is decreased while the oxygen depletion is increased by this metabolic symbiosis, promoting a further shift towards glycolysis. Our results also show that blocking glucose transporters could be more effective than blocking lactate intake transporters, because the former would disrupt both glycolysis and lactate production, drastically reducing tumour growth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553928">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553928" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553928">
        <p class="paperTitle">Reverse engineering morphogenesis through Bayesian optimization of physics-based models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553928" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553928" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumar, N.; Dowling, A.; Zartman, J. J.</p>
        <p class="info">Score: 3.9, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553928' target='https://doi.org/10.1101/2023.08.21.553928'> 10.1101/2023.08.21.553928</a></p>
        <p class="abstract">Morphogenetic programs direct the cell signaling and nonlinear mechanical interactions between multiple cell types and tissue layers to define organ shape and size. A key challenge for systems and synthetic biology is determining optimal combinations of intra- and inter-cellular interactions to predict an organ\&#39;s shape, size, and function. Physics-based mechanistic models that define the subcellular force distribution facilitate this, but it is extremely challenging to calibrate parameters in these models from data. To solve this inverse problem, we created a Bayesian optimization framework to determine the optimal cellular force distribution such that the predicted organ shapes match the desired organ shapes observed within the experimental imaging data. This integrative framework employs Gaussian Process Regression (GPR), a non-parametric kernel-based probabilistic machine learning modeling paradigm, to learn the mapping functions relating to the morphogenetic programs that generate and maintain the final organ shape. We calibrated and tested the method on cross-sections of Drosophila wing imaginal discs, a highly informative model organ system, to study mechanisms that regulate epithelial processes that range from development to cancer. As a specific test case, the parameter estimation framework successfully infers the underlying changes in core parameters needed to match simulation data with time series imaging data of wing discs perturbed with collagenase. Unexpectedly, the framework also identifies multiple distinct parameter sets that generate shapes similar to wild-type organ shapes. This platform enables an efficient, global sensitivity analysis to support the necessity of both actomyosin contractility and basal ECM stiffness to generate and maintain the curved shape of the wing imaginal disc. The optimization framework, combined with fixed tissue imaging, identified that Piezo, a mechanosensitive ion channel, impacts fold formation by regulating the apical-basal balance of actomyosin contractility and elasticity of ECM. This framework is extensible toward reverse-engineering the morphogenesis of any organ system and can be utilized in real-time control of complex multicellular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553655" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553655">
        <p class="paperTitle">The interplay between lumen pressure and cell proliferation determines organoid morphology in a multicellular phase field model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tanida, S.; Fuji, K.; Lu, L.; Guyomar, T.; Lee, B. H.; Honigmann, A.; Grapin-Botton, A.; Riveline, D.; Nonomura, M.; Sano, M.; Hiraiwa, T.</p>
        <p class="info">Score: 3.4, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553655' target='https://doi.org/10.1101/2023.08.17.553655'> 10.1101/2023.08.17.553655</a></p>
        <p class="abstract">Organoids are ideal systems to predict the phenotypes of organs. However, there is currently a lack of understanding regarding the generalized rules that enable use of simple cellular principles to make morphological predictions of entire organoids. Therefore, we employed a phase field model with the following basic components: the minimum conditions for the timing and volume of cell division, lumen nucleation rules, and lumenal pressure. Through our model, we could compute and generate a myriad of organoid phenotypes observed till date. We propose morphological indices necessary to characterize the shapes and construct phase diagrams and show their dependencies on proliferation time and lumen pressure. Additionally, we introduced the lumen-index parameter, which helped in examining the criteria to maintain organoids as spherical structures comprising a single layer of cells and enclosing an intact lumen. Finally, we predict a star-like organoid phenotype that did not undergo differentiation, suggesting that the volume constraint during cell division may determine the final phenotype. In summary, our approach provides researchers with guidelines to test the mechanisms of self-organization and predict the shape of organoid.

Author summaryIn nature, a wide variety of organ morphologies are observed. Owing to the complexity of the process underlying the acquisition of organs morphology, it is challenging to investigate the mechanisms that lead to such variations. A promising approach to study these variations is the use of &#34;computational organoid&#34; study, which is the computational-based study of self-organizing shapes in multicellular assemblies and fluid-filled cavities called lumens that develop from a few proliferating cells. This study explores general mechanisms that dictate how various mechanical factors affect the growing self-organized multicellular assembly. We relied on computer simulations of the mathematical model called multicellular phase-field model with lumens and explored the mechanical factor effects, such as the lumen pressure while considering the time and volume conditions required for cell division. These simulations generated and categorized a wide range of organoid phenotypes based on the varying lumen pressure and cell division conditions. These phenotypes were characterized into seven distinct classes, based on the morphological index sets, including a cellular monolayer/multilayer surrounding single or multiple lumens and branch formation. These phenotypes were obtained without the assumption of differentiation. Our study elucidates the mechanisms underlying the organoid and organ formation with different shapes, thereby highlighting the significance of mechanical forces in shaping these complex biological structures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.552948">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.552948" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.552948">
        <p class="paperTitle">Two-way feedback between chromatin compaction and histone modification state explains S. cerevisiae heterochromatin bistability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.552948" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.552948" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Movilla Miangolarra, A.; Saxton, D. S.; Yan, Z.; Rine, J.; Howard, M.</p>
        <p class="info">Score: 3.5, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.552948' target='https://doi.org/10.1101/2023.08.12.552948'> 10.1101/2023.08.12.552948</a></p>
        <p class="abstract">Compact chromatin is closely linked with gene silencing in part by sterically masking access to promoters, inhibiting transcription factor binding and preventing polymerase from efficiently transcribing a gene. Here, we propose a broader view: chromatin compaction can be both a cause and a consequence of the histone modification state, and this tight bidirectional interaction can underpin bistable transcriptional states. To test this theory, we developed a mathematical model for the dynamics of the HMR locus in S. cerevisiae, that incorporates activating histone modifications, silencing proteins and a dynamic, acetylation-dependent, three-dimensional locus size. Chromatin compaction enhances silencer protein binding, which in turn feeds back to remove activating histone modifications, leading to further compaction. The bistable output of the model was in good agreement with prior quantitative data, including switching rates from expressed to silent states, and vice versa, and protein binding levels within the locus. We then tested the model by predicting changes in switching rates as the genetic length of the locus was increased, which were then experimentally verified. This bidirectional feedback between chromatin compaction and the histone modification state may be an important regulatory mechanism at many loci.

SignificanceChromatin is the complex formed by proteins, including histones, and DNA to form chromosomes. Specific chromatin structures and states are thought to be key factors regulating transcription. A common view proposes that histone modifications activate or inhibit transcription either via specific activation or inhibition of RNA polymerase binding/elongation at a locus, or by expanding/compacting the locus, thereby modulating its accessibility to many macromolecules. In this work, we elucidated a broader hypothesis that chromatin compaction may both inhibit transcription, and feedback via silencing proteins to remove histone modifications that further control chromatin compaction and correlate with gene activity. We developed a model incorporating these ideas and showed that it explains quantitative experimental data for a silent locus in budding yeast.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
