<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Systematic mapping of antibiotic cross-resistance and collateral sensitivity with chemical genetics
Authors: Sakenova, N.; Cacace, E.; Orakov, A.; Huber, F.; Varik, V.; Kritikos, G.; Michiels, J.; Bork, P.; Cossart, P.; Goemans, C.; Typas, A.
Score: 45.2, Published: 2024-01-27 DOI: 10.1101/2024.01.25.576750
By acquiring or evolving resistance to one antibiotic, bacteria can become resistant to a second one, due to shared underlying mechanisms. This is called cross-resistance (XR) and further limits therapeutic choices.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Systematic mapping of antibiotic cross-resistance and collateral sensitivity with chemical genetics
Authors: Sakenova, N.; Cacace, E.; Orakov, A.; Huber, F.; Varik, V.; Kritikos, G.; Michiels, J.; Bork, P.; Cossart, P.; Goemans, C.; Typas, A.
Score: 45.2, Published: 2024-01-27 DOI: 10.1101/2024.01.25.576750
By acquiring or evolving resistance to one antibiotic, bacteria can become resistant to a second one, due to shared underlying mechanisms. This is called cross-resistance (XR) and further limits therapeutic choices." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Systematic mapping of antibiotic cross-resistance and collateral sensitivity with chemical genetics
Authors: Sakenova, N.; Cacace, E.; Orakov, A.; Huber, F.; Varik, V.; Kritikos, G.; Michiels, J.; Bork, P.; Cossart, P.; Goemans, C.; Typas, A.
Score: 45.2, Published: 2024-01-27 DOI: 10.1101/2024.01.25.576750
By acquiring or evolving resistance to one antibiotic, bacteria can become resistant to a second one, due to shared underlying mechanisms. This is called cross-resistance (XR) and further limits therapeutic choices."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Systematic mapping of antibiotic cross-resistance and collateral sensitivity with chemical genetics\nAuthors: Sakenova, N.; Cacace, E.; Orakov, A.; Huber, F.; Varik, V.; Kritikos, G.; Michiels, J.; Bork, P.; Cossart, P.; Goemans, C.; Typas, A.\nScore: 45.2, Published: 2024-01-27 DOI: 10.1101/2024.01.25.576750\nBy acquiring or evolving resistance to one antibiotic, bacteria can become resistant to a second one, due to shared underlying mechanisms. This is called cross-resistance (XR) and further limits therapeutic choices.",
  "keywords": [
    
  ],
  "articleBody": " Systematic mapping of antibiotic cross-resistance and collateral sensitivity with chemical genetics\nAuthors: Sakenova, N.; Cacace, E.; Orakov, A.; Huber, F.; Varik, V.; Kritikos, G.; Michiels, J.; Bork, P.; Cossart, P.; Goemans, C.; Typas, A.\nScore: 45.2, Published: 2024-01-27 DOI: 10.1101/2024.01.25.576750\nBy acquiring or evolving resistance to one antibiotic, bacteria can become resistant to a second one, due to shared underlying mechanisms. This is called cross-resistance (XR) and further limits therapeutic choices. The opposite scenario, in which initial resistance leads to sensitivity to a second antibiotic, is termed collateral sensitivity (CS) and can inform cycling or combinatorial treatments. Despite their clinical relevance, our current knowledge of such interactions is limited, mostly due to experimental constraints in their assessment and lack of understanding of the underlying mechanisms. To fill this gap, we used published chemical genetic data on the impact of all Escherichia coli non-essential genes on resistance/sensitivity to 40 antibiotics, and devised a metric that robustly discriminates between known XR and CS antibiotic interactions. This metric, based on chemical genetic profile (dis)similarity between two drugs, allowed us to infer 404 XR and 267 CS interactions, thereby expanding the number of known interactions by more than 3-fold - including reclassifying 116 previously reported interactions. We benchmarked our results by validating 55 out of 59 inferred interactions via experimental evolution. By identifying mutants driving XR and CS interactions in chemical genetics, we recapitulated known and uncovered previously unknown mechanisms, and demonstrated that a given drug pair can exhibit both interactions depending on the resistance mechanism. Finally, we applied CS drug pairs in combination to reduce antibiotic resistance development in vitro. Altogether, our approach provides a systematic framework to map XR/CS interactions and their mechanisms, paving the way for the development of rationally-designed antibiotic combination treatments.\nStructural determinants of co-translational protein complex assembly\nAuthors: Malik, S.; Venezian, J.; Lobov, A.; Heidenreich, M.; Garcia-Seisdedos, H.; Yeates, T. O.; Shiber, A.; Levy, E. D.\nScore: 26.9, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576408\nThe assembly of proteins into functional complexes is critical to lifes processes. While textbooks depict complex assembly as occurring between fully synthesized proteins, we know today that thousands of proteins in the human proteome assemble co-translationally during their synthesis. Why this process takes place, however, remains unknown. We show that co-translational assembly is governed by biophysical and structural characteristics of the protein complex, and involves mutually stabilized, intertwined subunits. Consequently, these subunits are also co-regulated across the central dogma, from transcription to protein degradation. Leveraging structural signatures with AlphaFold2-based predictions enables us to accurately predict co-translational assembly on a proteome-wide scale, which we validated by ribosome profiling, genetic perturbations, and smFISH experiments. Notably, the latter showed that co-translationally assembling subunits exhibit co-localized mRNAs. This work unveils a fundamental connection between protein structure and the translation process, highlighting the overarching impact of three-dimensional structure on gene expression, mRNA localization, and proteostasis. One Sentence SummaryProtein complexes with topologically intertwined subunits require co-translational assembly and synchronized proteostasis of subunits, with implications in protein stability, mRNA localization, and evolution. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=195 SRC=\"FIGDIR/small/576408v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (62K): org.highwire.dtl.DTLVardef@9ef8d4org.highwire.dtl.DTLVardef@e25aeeorg.highwire.dtl.DTLVardef@bfe7b7org.highwire.dtl.DTLVardef@105d82e_HPS_FORMAT_FIGEXP M_FIG C_FIG\nPredicting microbial genome-scale metabolic networks directly from 16S rRNA gene sequences\nAuthors: Oyas, O.; Kobel, C. M.; Vik, J. O.; Pope, P. B.\nScore: 20.7, Published: 2024-01-27 DOI: 10.1101/2024.01.26.576649\nGenome-scale metabolic models are key biotechnology tools that can predict metabolic capabilities and growth for an organism. In particular, these models have become indispensable for metabolic analysis of microbial species and communities such as the gut microbiomes of humans and other animals. Accurate microbial models can be built automatically from genomes, but many microbes have only been observed through sequencing of marker genes such as 16S rRNA and thus remain inaccessible to genome-scale modeling. To extend the scope of genome-scale metabolic models to microbes that lack genomic information, we trained an artificial neural network to build microbial models from numeric representations of 16S rRNA gene sequences. Specifically, we built models and extracted 16S rRNA gene sequences from more than 15,000 reference and representative microbial genomes, computed multiple sequence alignments and large language model embeddings for the 16S rRNA gene sequences, and trained the neural network to predict metabolic reaction probabilities from sequences, alignments, or embeddings. Training was fast on a single graphics processing unit and trained networks predicted reaction probabilities accurately for unseen archaeal and bacterial sequences and species. This makes it possible to reconstruct microbial genome-scale metabolic networks from any 16S rRNA gene sequence and enables simulation of metabolism and growth for all observed microbial life.\nMoCHI: neural networks to fit interpretable models and quantify energies, energetic couplings, epistasis and allostery from deep mutational scanning data\nAuthors: Faure, A. J.; Lehner, B.\nScore: 10.9, Published: 2024-01-28 DOI: 10.1101/2024.01.21.575681\nThe massively parallel nature of deep mutational scanning (DMS) allows the quantification of the phenotypic effects of thousands of perturbations in a single experiment. We have developed MoCHI, a software tool that allows the parameterisation of arbitrarily complex models using DMS data. MoCHI simplifies the task of building custom models from measurements of mutant effects on any number of phenotypes. It allows the inference of free energy changes, as well as pairwise and higher-order interaction terms (energetic couplings) for specified biophysical models. When a suitable user-specified mechanistic model is not available, global nonlinearities (epistasis) can be estimated directly from the data. MoCHI also builds upon and leverages theory on ensemble (or background-averaged) epistasis to learn sparse predictive models that can incorporate higher-order epistatic terms and are informative of the genetic architecture of the underlying biological system. The combination of DMS and MoCHI allows biophysical measurements to be performed at scale, including the construction of complete allosteric maps of proteins. MoCHI is freely available (https://github.com/lehner-lab/MoCHI) and implemented as an easy-to-use python package relying on the PyTorch machine learning framework.\nStatistical inference with a manifold-constrained RNA velocity model uncovers cell cycle speed modulations\nAuthors: Lederer, A. R.; Leonardi, M.; Talamanca, L.; Herrera, A.; Droin, C.; Khven, I.; Carvalho, H. J.; Valente, A.; Dominguez Mantes, A.; Mulet Arabi, P.; Pinello, L.; Naef, F.; La Manno, G.\nScore: 39.9, Published: 2024-01-22 DOI: 10.1101/2024.01.18.576093\nAcross a range of biological processes, cells undergo coordinated changes in gene expression, resulting in transcriptome dynamics that unfold within a low-dimensional manifold. Single-cell RNA-sequencing (scRNA-seq) only measures temporal snapshots of gene expression. However, information on the underlying low-dimensional dynamics can be extracted using RNA velocity, which models unspliced and spliced RNA abundances to estimate the rate of change of gene expression. Available RNA velocity algorithms can be fragile and rely on heuristics that lack statistical control. Moreover, the estimated vector field is not dynamically consistent with the traversed gene expression manifold. Here, we develop a generative model of RNA velocity and a Bayesian inference approach that solves these problems. Our model couples velocity field and manifold estimation in a reformulated, unified framework, so as to coherently identify the parameters of an autonomous dynamical system. Focusing on the cell cycle, we implemented VeloCycle to study gene regulation dynamics on one-dimensional periodic manifolds and validated using live-imaging its ability to infer actual cell cycle periods. We benchmarked RNA velocity inference with sensitivity analyses and demonstrated one- and multiple-sample testing. We also conducted Markov chain Monte Carlo inference on the model, uncovering key relationships between gene-specific kinetics and our gene-independent velocity estimate. Finally, we applied VeloCycle to in vivo samples and in vitro genome-wide Perturb-seq, revealing regionally-defined proliferation modes in neural progenitors and the effect of gene knockdowns on cell cycle speed. Ultimately, VeloCycle expands the scRNA-seq analysis toolkit with a modular and statistically rigorous RNA velocity inference framework.\nEnhanced feature matching in single-cell proteomics characterizes response to IFN-gamma and reveals co-existence of different cell states\nAuthors: Krull, K. K.; Ali, S. A.; Krijgsveld, J.\nScore: 6.6, Published: 2024-01-10 DOI: 10.1101/2024.01.10.575010\nProteome analysis by data-independent acquisition (DIA) has become a powerful approach to obtain deep proteome coverage, and has gained recent traction for label-free analysis of single cells. However, optimal experimental design for DIA-based single-cell proteomics has not been fully explored, and performance metrics of subsequent data analysis tools remain to be evaluated. Therefore, we here present DIA-ME, a data analysis strategy that exploits the co-analysis of low-input samples with a so-called matching enhancer (ME) of higher input, to increase sensitivity, proteome coverage, and data completeness. We evaluate the matching specificity of DIA-ME by a two-proteome model, and demonstrate that false discovery and false transfer are maintained at low levels when using DIA-NN software, while preserving quantification accuracy. We apply DIA-ME to investigate the proteome response of U-2 OS cells to interferon gamma (IFN-{gamma}) in single cells, and recapitulate the time-resolved induction of IFN-{gamma} response proteins as observed in bulk material. Moreover, we observe co- and anti-correlating patterns of protein expression within the same cell, indicating mutually exclusive protein modules and the co-existence of different cell states. Collectively our data show that DIA-ME is a powerful, scalable, and easy-to- implement strategy for single-cell proteomics.\nSignalingProfiler 2.0: a network-based approach to bridge multi-omics data to phenotypic hallmarks\nAuthors: Venafra, V.; Sacco, F.; Perfetto, L.\nScore: 2.9, Published: 2024-01-29 DOI: 10.1101/2024.01.25.577229\nUnraveling the cellular signaling remodeling upon a perturbation is a fundamental challenge to understand disease mechanisms and to identify potential drug targets. In this pursuit, computational tools that generate mechanistic hypotheses from multi-omics data have invaluable potential. Here we present SignalingProfiler 2.0, a multi-step pipeline to systematically derive context-specific signaling models by integrating proteogenomic data with prior knowledge-causal networks. This is a freely accessible and flexible tool that incorporates statistical, footprint-based, and graph algorithms to accelerate the integration and interpretation of multi-omics data. Through benchmarking and rigorous parameter selection on a proof-of-concept study, performed in metformin-treated breast cancer cells, we demonstrate the tool's ability to generate a hierarchical mechanistic network that recapitulates novel and known drug-perturbed signaling and phenotypic outcomes. In summary, SignalingProfiler 2.0 addresses the emergent need to derive biologically relevant information from complex multi-omics data by extracting interpretable networks.\nA rule-based multiscale model of hepatic stellate cell plasticity: critical role of the inactivation loop in fibrosis progression\nAuthors: Bougueon, M.; Legagneux, V.; Hazard, O.; Bomo, J.; Siegel, A.; Feret, J.; Theret, N.\nScore: 1.5, Published: 2024-01-26 DOI: 10.1101/2024.01.24.577157\nHepatic stellate cells (HSC) are the source of extracellular matrix (ECM) whose overproduction leads to fibrosis, a condition that impairs liver functions in chronic liver diseases. Understanding the dynamics of HSCs will provide insights needed to develop new therapeutic approaches. Few models of hepatic fibrosis have been proposed, and none of them include the heterogeneity of HSC phenotypes recently highlighted by single-cell RNA sequencing analyses. Here, we developed rule-based models to study HSC dynamics during fibrosis progression and reversion. We used the Kappa graph rewriting language, for which we used tokens and counters to overcome temporal explosion. HSCs are modeled as agents that present seven physiological cellular states and that interact with (TGF{beta}1) molecules which regulate HSC activation and the secretion of type I collagen, the main component of the ECM. Simulation studies revealed the critical role of the HSC inactivation process during fibrosis progression and reversion. While inactivation allows elimination of activated HSCs during reversion steps, reactivation loops of inactivated HSCs (iHSCs) are required to sustain fibrosis. Furthermore, we demonstrated the models sensitivity to (TGF{beta}1) parameters, suggesting its adaptability to a variety of pathophysiological conditions for which levels of (TGF{beta}1) production associated with the inflammatory response differ. Using new experimental data from a mouse model of CCl4-induced liver fibrosis, we validated the predicted ECM dynamics. Our model also predicts the accumulation of iHSCs during chronic liver disease. By analyzing RNA sequencing data from patients with non-alcoholic steatohepatitis (NASH) associated with liver fibrosis, we confirmed this accumulation, identifying iHSCs as novel markers of fibrosis progression. Overall, our study provides the first model of HSC dynamics in chronic liver disease that can be used to explore the regulatory role of iHSCs in liver homeostasis. Moreover our model can also be generalized to fibroblasts during repair and fibrosis in other tissues. Author summaryChronic liver diseases (CLDs) are associated with the development of fibrosis which is characterized by an abnormal deposition of extracellular matrix (ECM) leading to severe liver dysfunction. Hepatic stellate cells (HSCs) are key players in liver fibrosis driving ECM remodeling. However numerous biological processes are involved including HSC activation, proliferation, differentiation and inactivation and novel computational modeling is necessary to integrate such complex dynamics. Here, we used the Kappa graph rewriting language to develop the first rule-based model describing the HSCs dynamics during liver fibrosis and its reversion. Simulation analyses enabled us to demonstrate the critical role of the HSC inactivation loop in the development of liver fibrosis, and to identify inactivated HSCs as potential new markers of fibrosis progression.\nMacrophage Polarizations in the Placenta and Lung are Associated with Bronchopulmonary Dysplasia\nAuthors: Mestan, K. K.; Sharma, A.; Lazar, S.; Pandey, S.; Parast, M. M.; Laurent, L. C.; Prince, L. S.; Sahoo, D.\nScore: 1.2, Published: 2024-01-30 DOI: 10.1101/2024.01.26.577443\nThe intricate interplay between macrophage polarization and placenta vascular dysfunction has garnered increasing attention in the context of placental inflammatory diseases. This study delves into the complex relationship between macrophage polarization within the placenta and its potential impact on the development of vascular dysfunction and inflammatory conditions. The placenta, a crucial organ in fetal development, relies on a finely tuned balance of immune responses for proper functioning. Disruptions in this delicate equilibrium can lead to pathological conditions, including inflammatory diseases affecting the fetus and newborn infant. We explored the interconnectedness between placental macrophage polarization and its relevance to lung macrophages, particularly in the context of early life lung development. Bronchopulmonary dysplasia (BPD), the most common chronic lung disease of prematurity, has been associated with abnormal immune responses, and understanding the role of macrophages in this context is pivotal. The investigation aims to shed light on how alterations in placental macrophage polarization may contribute to lung macrophage behavior and, consequently, influence the development of BPD. By unraveling the intricate mechanisms linking macrophage polarization, placental dysfunction and BPD, this research seeks to provide insights that could pave the way for targeted therapeutic interventions. The findings may offer novel perspectives on preventing and managing placental and lung-related pathologies, ultimately contributing to improved maternal and neonatal health outcomes.\nC1QA is an invariant biomarker for tissue macrophages\nAuthors: Horowitz, A.; Yu, H.; Pandey, S.; Mishra, B.; Sahoo, D.\nScore: 1.2, Published: 2024-01-28 DOI: 10.1101/2024.01.26.577475\nMacrophages play a pivotal role in immune responses, particularly in the context of combating microbial threats within tissues. The identification of reliable biomarkers associated with macrophage function is essential for understanding their diverse roles in host defense. This study investigates the potential of C1QA as an invariant biomarker for tissue macrophages, focusing on its correlation with the anti-microbial pathway. C1QA, a component of the complement system, has been previously implicated in various immune functions. Our research delves into the specific association of C1QA with tissue-resident macrophages and its implications in the context of anti-microbial responses. Through comprehensive systems biology and Boolean analysis of gene expression, we aim to establish C1QA as a consistent and reliable marker for identifying tissue macrophages. Furthermore, we explore the functional significance of C1QA in the anti-microbial pathway. This research seeks to provide valuable insights into the molecular mechanisms underlying the anti-microbial functions of tissue macrophages, with C1QA emerging as a potential key player in this intricate regulatory network. Understanding the relationship between C1QA, tissue macrophages, and the anti-microbial pathway could pave the way for the development of targeted therapeutic strategies aimed at enhancing the host's ability to combat infections. Ultimately, our findings contribute to the expanding knowledge of macrophage biology and may have implications for the diagnosis and treatment of infectious diseases.\n",
  "wordCount" : "2604",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.576750">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.576750" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.576750">
        <p class="paperTitle">Systematic mapping of antibiotic cross-resistance and collateral sensitivity with chemical genetics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.576750" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.576750" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sakenova, N.; Cacace, E.; Orakov, A.; Huber, F.; Varik, V.; Kritikos, G.; Michiels, J.; Bork, P.; Cossart, P.; Goemans, C.; Typas, A.</p>
        <p class="info">Score: 45.2, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.576750' target='https://doi.org/10.1101/2024.01.25.576750'> 10.1101/2024.01.25.576750</a></p>
        <p class="abstract">By acquiring or evolving resistance to one antibiotic, bacteria can become resistant to a second one, due to shared underlying mechanisms. This is called cross-resistance (XR) and further limits therapeutic choices. The opposite scenario, in which initial resistance leads to sensitivity to a second antibiotic, is termed collateral sensitivity (CS) and can inform cycling or combinatorial treatments. Despite their clinical relevance, our current knowledge of such interactions is limited, mostly due to experimental constraints in their assessment and lack of understanding of the underlying mechanisms. To fill this gap, we used published chemical genetic data on the impact of all Escherichia coli non-essential genes on resistance/sensitivity to 40 antibiotics, and devised a metric that robustly discriminates between known XR and CS antibiotic interactions. This metric, based on chemical genetic profile (dis)similarity between two drugs, allowed us to infer 404 XR and 267 CS interactions, thereby expanding the number of known interactions by more than 3-fold - including reclassifying 116 previously reported interactions. We benchmarked our results by validating 55 out of 59 inferred interactions via experimental evolution. By identifying mutants driving XR and CS interactions in chemical genetics, we recapitulated known and uncovered previously unknown mechanisms, and demonstrated that a given drug pair can exhibit both interactions depending on the resistance mechanism. Finally, we applied CS drug pairs in combination to reduce antibiotic resistance development in vitro. Altogether, our approach provides a systematic framework to map XR/CS interactions and their mechanisms, paving the way for the development of rationally-designed antibiotic combination treatments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576408">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576408" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576408">
        <p class="paperTitle">Structural determinants of co-translational protein complex assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576408" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576408" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Malik, S.; Venezian, J.; Lobov, A.; Heidenreich, M.; Garcia-Seisdedos, H.; Yeates, T. O.; Shiber, A.; Levy, E. D.</p>
        <p class="info">Score: 26.9, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576408' target='https://doi.org/10.1101/2024.01.20.576408'> 10.1101/2024.01.20.576408</a></p>
        <p class="abstract">The assembly of proteins into functional complexes is critical to lifes processes. While textbooks depict complex assembly as occurring between fully synthesized proteins, we know today that thousands of proteins in the human proteome assemble co-translationally during their synthesis. Why this process takes place, however, remains unknown. We show that co-translational assembly is governed by biophysical and structural characteristics of the protein complex, and involves mutually stabilized, intertwined subunits. Consequently, these subunits are also co-regulated across the central dogma, from transcription to protein degradation. Leveraging structural signatures with AlphaFold2-based predictions enables us to accurately predict co-translational assembly on a proteome-wide scale, which we validated by ribosome profiling, genetic perturbations, and smFISH experiments. Notably, the latter showed that co-translationally assembling subunits exhibit co-localized mRNAs. This work unveils a fundamental connection between protein structure and the translation process, highlighting the overarching impact of three-dimensional structure on gene expression, mRNA localization, and proteostasis.

One Sentence SummaryProtein complexes with topologically intertwined subunits require co-translational assembly and synchronized proteostasis of subunits, with implications in protein stability, mRNA localization, and evolution.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=195 SRC=&#34;FIGDIR/small/576408v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (62K):
org.highwire.dtl.DTLVardef@9ef8d4org.highwire.dtl.DTLVardef@e25aeeorg.highwire.dtl.DTLVardef@bfe7b7org.highwire.dtl.DTLVardef@105d82e_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.576649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.576649" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.576649">
        <p class="paperTitle">Predicting microbial genome-scale metabolic networks directly from 16S rRNA gene sequences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.576649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.576649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oyas, O.; Kobel, C. M.; Vik, J. O.; Pope, P. B.</p>
        <p class="info">Score: 20.7, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.576649' target='https://doi.org/10.1101/2024.01.26.576649'> 10.1101/2024.01.26.576649</a></p>
        <p class="abstract">Genome-scale metabolic models are key biotechnology tools that can predict metabolic capabilities and growth for an organism. In particular, these models have become indispensable for metabolic analysis of microbial species and communities such as the gut microbiomes of humans and other animals. Accurate microbial models can be built automatically from genomes, but many microbes have only been observed through sequencing of marker genes such as 16S rRNA and thus remain inaccessible to genome-scale modeling. To extend the scope of genome-scale metabolic models to microbes that lack genomic information, we trained an artificial neural network to build microbial models from numeric representations of 16S rRNA gene sequences. Specifically, we built models and extracted 16S rRNA gene sequences from more than 15,000 reference and representative microbial genomes, computed multiple sequence alignments and large language model embeddings for the 16S rRNA gene sequences, and trained the neural network to predict metabolic reaction probabilities from sequences, alignments, or embeddings. Training was fast on a single graphics processing unit and trained networks predicted reaction probabilities accurately for unseen archaeal and bacterial sequences and species. This makes it possible to reconstruct microbial genome-scale metabolic networks from any 16S rRNA gene sequence and enables simulation of metabolism and growth for all observed microbial life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.575681">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.575681" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.575681">
        <p class="paperTitle">MoCHI: neural networks to fit interpretable models and quantify energies, energetic couplings, epistasis and allostery from deep mutational scanning data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.575681" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.575681" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faure, A. J.; Lehner, B.</p>
        <p class="info">Score: 10.9, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.575681' target='https://doi.org/10.1101/2024.01.21.575681'> 10.1101/2024.01.21.575681</a></p>
        <p class="abstract">The massively parallel nature of deep mutational scanning (DMS) allows the quantification of the phenotypic effects of thousands of perturbations in a single experiment. We have developed MoCHI, a software tool that allows the parameterisation of arbitrarily complex models using DMS data. MoCHI simplifies the task of building custom models from measurements of mutant effects on any number of phenotypes. It allows the inference of free energy changes, as well as pairwise and higher-order interaction terms (energetic couplings) for specified biophysical models. When a suitable user-specified mechanistic model is not available, global nonlinearities (epistasis) can be estimated directly from the data. MoCHI also builds upon and leverages theory on ensemble (or background-averaged) epistasis to learn sparse predictive models that can incorporate higher-order epistatic terms and are informative of the genetic architecture of the underlying biological system. The combination of DMS and MoCHI allows biophysical measurements to be performed at scale, including the construction of complete allosteric maps of proteins. MoCHI is freely available (https://github.com/lehner-lab/MoCHI) and implemented as an easy-to-use python package relying on the PyTorch machine learning framework.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576093">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576093" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576093">
        <p class="paperTitle">Statistical inference with a manifold-constrained RNA velocity model uncovers cell cycle speed modulations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576093" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576093" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lederer, A. R.; Leonardi, M.; Talamanca, L.; Herrera, A.; Droin, C.; Khven, I.; Carvalho, H. J.; Valente, A.; Dominguez Mantes, A.; Mulet Arabi, P.; Pinello, L.; Naef, F.; La Manno, G.</p>
        <p class="info">Score: 39.9, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576093' target='https://doi.org/10.1101/2024.01.18.576093'> 10.1101/2024.01.18.576093</a></p>
        <p class="abstract">Across a range of biological processes, cells undergo coordinated changes in gene expression, resulting in transcriptome dynamics that unfold within a low-dimensional manifold. Single-cell RNA-sequencing (scRNA-seq) only measures temporal snapshots of gene expression. However, information on the underlying low-dimensional dynamics can be extracted using RNA velocity, which models unspliced and spliced RNA abundances to estimate the rate of change of gene expression. Available RNA velocity algorithms can be fragile and rely on heuristics that lack statistical control. Moreover, the estimated vector field is not dynamically consistent with the traversed gene expression manifold. Here, we develop a generative model of RNA velocity and a Bayesian inference approach that solves these problems. Our model couples velocity field and manifold estimation in a reformulated, unified framework, so as to coherently identify the parameters of an autonomous dynamical system. Focusing on the cell cycle, we implemented VeloCycle to study gene regulation dynamics on one-dimensional periodic manifolds and validated using live-imaging its ability to infer actual cell cycle periods. We benchmarked RNA velocity inference with sensitivity analyses and demonstrated one- and multiple-sample testing. We also conducted Markov chain Monte Carlo inference on the model, uncovering key relationships between gene-specific kinetics and our gene-independent velocity estimate. Finally, we applied VeloCycle to in vivo samples and in vitro genome-wide Perturb-seq, revealing regionally-defined proliferation modes in neural progenitors and the effect of gene knockdowns on cell cycle speed. Ultimately, VeloCycle expands the scRNA-seq analysis toolkit with a modular and statistically rigorous RNA velocity inference framework.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575010">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575010" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575010">
        <p class="paperTitle">Enhanced feature matching in single-cell proteomics characterizes response to IFN-gamma and reveals co-existence of different cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575010" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575010" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krull, K. K.; Ali, S. A.; Krijgsveld, J.</p>
        <p class="info">Score: 6.6, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575010' target='https://doi.org/10.1101/2024.01.10.575010'> 10.1101/2024.01.10.575010</a></p>
        <p class="abstract">Proteome analysis by data-independent acquisition (DIA) has become a powerful approach to obtain deep proteome coverage, and has gained recent traction for label-free analysis of single cells. However, optimal experimental design for DIA-based single-cell proteomics has not been fully explored, and performance metrics of subsequent data analysis tools remain to be evaluated. Therefore, we here present DIA-ME, a data analysis strategy that exploits the co-analysis of low-input samples with a so-called matching enhancer (ME) of higher input, to increase sensitivity, proteome coverage, and data completeness. We evaluate the matching specificity of DIA-ME by a two-proteome model, and demonstrate that false discovery and false transfer are maintained at low levels when using DIA-NN software, while preserving quantification accuracy. We apply DIA-ME to investigate the proteome response of U-2 OS cells to interferon gamma (IFN-{gamma}) in single cells, and recapitulate the time-resolved induction of IFN-{gamma} response proteins as observed in bulk material. Moreover, we observe co- and anti-correlating patterns of protein expression within the same cell, indicating mutually exclusive protein modules and the co-existence of different cell states. Collectively our data show that DIA-ME is a powerful, scalable, and easy-to- implement strategy for single-cell proteomics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577229">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577229" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577229">
        <p class="paperTitle">SignalingProfiler 2.0: a network-based approach to bridge multi-omics data to phenotypic hallmarks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577229" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577229" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venafra, V.; Sacco, F.; Perfetto, L.</p>
        <p class="info">Score: 2.9, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577229' target='https://doi.org/10.1101/2024.01.25.577229'> 10.1101/2024.01.25.577229</a></p>
        <p class="abstract">Unraveling the cellular signaling remodeling upon a perturbation is a fundamental challenge to understand disease mechanisms and to identify potential drug targets. In this pursuit, computational tools that generate mechanistic hypotheses from multi-omics data have invaluable potential. Here we present SignalingProfiler 2.0, a multi-step pipeline to systematically derive context-specific signaling models by integrating proteogenomic data with prior knowledge-causal networks. This is a freely accessible and flexible tool that incorporates statistical, footprint-based, and graph algorithms to accelerate the integration and interpretation of multi-omics data. Through benchmarking and rigorous parameter selection on a proof-of-concept study, performed in metformin-treated breast cancer cells, we demonstrate the tool&#39;s ability to generate a hierarchical mechanistic network that recapitulates novel and known drug-perturbed signaling and phenotypic outcomes. In summary, SignalingProfiler 2.0 addresses the emergent need to derive biologically relevant information from complex multi-omics data by extracting interpretable networks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577157" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577157">
        <p class="paperTitle">A rule-based multiscale model of hepatic stellate cell plasticity: critical role of the inactivation loop in fibrosis progression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bougueon, M.; Legagneux, V.; Hazard, O.; Bomo, J.; Siegel, A.; Feret, J.; Theret, N.</p>
        <p class="info">Score: 1.5, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577157' target='https://doi.org/10.1101/2024.01.24.577157'> 10.1101/2024.01.24.577157</a></p>
        <p class="abstract">Hepatic stellate cells (HSC) are the source of extracellular matrix (ECM) whose overproduction leads to fibrosis, a condition that impairs liver functions in chronic liver diseases. Understanding the dynamics of HSCs will provide insights needed to develop new therapeutic approaches. Few models of hepatic fibrosis have been proposed, and none of them include the heterogeneity of HSC phenotypes recently highlighted by single-cell RNA sequencing analyses. Here, we developed rule-based models to study HSC dynamics during fibrosis progression and reversion. We used the Kappa graph rewriting language, for which we used tokens and counters to overcome temporal explosion. HSCs are modeled as agents that present seven physiological cellular states and that interact with (TGF{beta}1) molecules which regulate HSC activation and the secretion of type I collagen, the main component of the ECM. Simulation studies revealed the critical role of the HSC inactivation process during fibrosis progression and reversion. While inactivation allows elimination of activated HSCs during reversion steps, reactivation loops of inactivated HSCs (iHSCs) are required to sustain fibrosis. Furthermore, we demonstrated the models sensitivity to (TGF{beta}1) parameters, suggesting its adaptability to a variety of pathophysiological conditions for which levels of (TGF{beta}1) production associated with the inflammatory response differ. Using new experimental data from a mouse model of CCl4-induced liver fibrosis, we validated the predicted ECM dynamics. Our model also predicts the accumulation of iHSCs during chronic liver disease. By analyzing RNA sequencing data from patients with non-alcoholic steatohepatitis (NASH) associated with liver fibrosis, we confirmed this accumulation, identifying iHSCs as novel markers of fibrosis progression. Overall, our study provides the first model of HSC dynamics in chronic liver disease that can be used to explore the regulatory role of iHSCs in liver homeostasis. Moreover our model can also be generalized to fibroblasts during repair and fibrosis in other tissues.

Author summaryChronic liver diseases (CLDs) are associated with the development of fibrosis which is characterized by an abnormal deposition of extracellular matrix (ECM) leading to severe liver dysfunction. Hepatic stellate cells (HSCs) are key players in liver fibrosis driving ECM remodeling. However numerous biological processes are involved including HSC activation, proliferation, differentiation and inactivation and novel computational modeling is necessary to integrate such complex dynamics. Here, we used the Kappa graph rewriting language to develop the first rule-based model describing the HSCs dynamics during liver fibrosis and its reversion. Simulation analyses enabled us to demonstrate the critical role of the HSC inactivation loop in the development of liver fibrosis, and to identify inactivated HSCs as potential new markers of fibrosis progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577443">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577443" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577443">
        <p class="paperTitle">Macrophage Polarizations in the Placenta and Lung are Associated with Bronchopulmonary Dysplasia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577443" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577443" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mestan, K. K.; Sharma, A.; Lazar, S.; Pandey, S.; Parast, M. M.; Laurent, L. C.; Prince, L. S.; Sahoo, D.</p>
        <p class="info">Score: 1.2, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577443' target='https://doi.org/10.1101/2024.01.26.577443'> 10.1101/2024.01.26.577443</a></p>
        <p class="abstract">The intricate interplay between macrophage polarization and placenta vascular dysfunction has garnered increasing attention in the context of placental inflammatory diseases. This study delves into the complex relationship between macrophage polarization within the placenta and its potential impact on the development of vascular dysfunction and inflammatory conditions. The placenta, a crucial organ in fetal development, relies on a finely tuned balance of immune responses for proper functioning. Disruptions in this delicate equilibrium can lead to pathological conditions, including inflammatory diseases affecting the fetus and newborn infant. We explored the interconnectedness between placental macrophage polarization and its relevance to lung macrophages, particularly in the context of early life lung development. Bronchopulmonary dysplasia (BPD), the most common chronic lung disease of prematurity, has been associated with abnormal immune responses, and understanding the role of macrophages in this context is pivotal. The investigation aims to shed light on how alterations in placental macrophage polarization may contribute to lung macrophage behavior and, consequently, influence the development of BPD. By unraveling the intricate mechanisms linking macrophage polarization, placental dysfunction and BPD, this research seeks to provide insights that could pave the way for targeted therapeutic interventions. The findings may offer novel perspectives on preventing and managing placental and lung-related pathologies, ultimately contributing to improved maternal and neonatal health outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577475">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577475" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577475">
        <p class="paperTitle">C1QA is an invariant biomarker for tissue macrophages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577475" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577475" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Horowitz, A.; Yu, H.; Pandey, S.; Mishra, B.; Sahoo, D.</p>
        <p class="info">Score: 1.2, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577475' target='https://doi.org/10.1101/2024.01.26.577475'> 10.1101/2024.01.26.577475</a></p>
        <p class="abstract">Macrophages play a pivotal role in immune responses, particularly in the context of combating microbial threats within tissues. The identification of reliable biomarkers associated with macrophage function is essential for understanding their diverse roles in host defense. This study investigates the potential of C1QA as an invariant biomarker for tissue macrophages, focusing on its correlation with the anti-microbial pathway. C1QA, a component of the complement system, has been previously implicated in various immune functions. Our research delves into the specific association of C1QA with tissue-resident macrophages and its implications in the context of anti-microbial responses. Through comprehensive systems biology and Boolean analysis of gene expression, we aim to establish C1QA as a consistent and reliable marker for identifying tissue macrophages. Furthermore, we explore the functional significance of C1QA in the anti-microbial pathway. This research seeks to provide valuable insights into the molecular mechanisms underlying the anti-microbial functions of tissue macrophages, with C1QA emerging as a potential key player in this intricate regulatory network. Understanding the relationship between C1QA, tissue macrophages, and the anti-microbial pathway could pave the way for the development of targeted therapeutic strategies aimed at enhancing the host&#39;s ability to combat infections. Ultimately, our findings contribute to the expanding knowledge of macrophage biology and may have implications for the diagnosis and treatment of infectious diseases.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
