<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer
Authors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.
Score: 25.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568149">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer
Authors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.
Score: 25.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568149" />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-29T10:40:14+00:00" />
<meta property="article:modified_time" content="2023-11-29T10:40:14+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer
Authors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.
Score: 25.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568149"/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer\nAuthors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.\nScore: 25.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568149",
  "keywords": [
    
  ],
  "articleBody": " One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer\nAuthors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.\nScore: 25.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568149\nAs a key mechanism for cellular signal transduction, protein phosphorylation plays critical roles in myriad cellular processes. This modification, however, is highly dynamic and occurs at sub-stoichiometric levels. Mass spectrometry is an essential tool for studying this modification on a global scale; however, the technologys impact suffers from four main limitations: need for site localization, dynamic range, reproducibility, and throughput. Here we describe the use of a novel mass spectrometer (Orbitrap Astral) coupled with data-independent acquisition (DIA) to achieve detection of close to 40,000 unique phosphorylation sites within one hour of analysis. We applied this approach to generate a phosphoproteome atlas of the mouse. Altogether, we detected 81,120 unique phosphorylation sites within 12 hours of measurement. With this unique dataset, we examine the sequence and structural context of protein phosphorylation. Finally, we highlight the discovery potential of this resource with multiple examples of novel phosphorylation events relevant to mitochondrial and brain biology.\nSystematic optimization of automated phosphopeptide enrichment for high-sensitivity phosphoproteomics\nAuthors: Bortel, P.; Piga, I.; Koenig, C.; Gerner, C.; Martinez del Val, A.; Olsen, J. V.\nScore: 23.6, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568418\nImproving coverage, robustness and sensitivity is crucial for routine phosphoproteomics analysis by single-shot liquid chromatography tandem mass spectrometry (LC-MS/MS) runs from minimal peptide inputs. Here, we systematically optimized key experimental parameters for automated on-beads phosphoproteomics sample preparation with focus on low input samples. Assessing the number of identified phosphopeptides, enrichment efficiency, site localization scores and relative enrichment of multiply-phosphorylated peptides pinpointed critical variables influencing the resulting phosphoproteome. Optimizing glycolic acid concentration in the loading buffer, percentage of ammonium hydroxide in the elution buffer, peptide-to-beads ratio, binding time, sample and loading buffer volumes, allowed us to confidently identify \u003e16,000 phosphopeptides in half-an-hour LC-MS/MS on an Orbitrap Exploris 480 using 30 {micro}g of peptides as starting material. Furthermore, we evaluated how sequential enrichment can boost phosphoproteome coverage and showed that pooling fractions into a single LC-MS/MS analysis increased the depth. We also present an alternative phosphopeptide enrichment strategy based on stepwise addition of beads thereby boosting phosphoproteome coverage by 20%. Finally, we applied our optimized strategy to evaluate phosphoproteome depth with the Orbitrap Astral MS using a cell dilution series and were able to identify \u003e32,000 phosphopeptides from 0.5 million HeLa cells in half-an-hour LC-MS/MS using narrow-window data-independent acquisition (nDIA). Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=159 HEIGHT=200 SRC=\"FIGDIR/small/568418v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (34K): org.highwire.dtl.DTLVardef@149b840org.highwire.dtl.DTLVardef@dfade8org.highwire.dtl.DTLVardef@112788aorg.highwire.dtl.DTLVardef@1a7c638_HPS_FORMAT_FIGEXP M_FIG C_FIG\nHigh-throughput and scalable single cell proteomics identifies over 5000 proteins per cell\nAuthors: Ye, Z.; Sabatier, P.; Hoeven, L.; Phlairaharn, T.; Hartlmayr, D.; Izaguirre, F.; Seth, A.; Joshi, H.; Bekker-Jensen, D.; Bache, N.; Olsen, J. V.\nScore: 20.2, Published: 2023-11-28 DOI: 10.1101/2023.11.27.568953\nThe emergence of mass spectrometry (MS)-based single-cell proteomics (SCP) promise to revolutionize the study of cellular biology and biomedicine by providing an unparalleled view of the proteome in individual cells. Despite its groundbreaking potential, SCP is nascent and faces challenges including limited sequence depth, throughput, and reproducibility, which have constrained its broader utility. This study introduces key methodological advances, which considerably improve the sensitivity, coverage and dependability of protein identification from single cells. We developed an almost lossless SCP workflow encompassing sample preparation to MS analysis, doubling the number of identified proteins from roughly 2000 to over 5000 in individual HeLa cells. A comprehensive evaluation of analytical software tools, alongside strict false discovery rate (FDR) controls solidified the reliability of our results. These enhancements also facilitated the direct detection of post-translational modifications (PTMs) in single cells, negating the need for enrichment and thereby simplifying the analytical process. Although throughput in MS remains a challenge, our study demonstrates the feasibility of processing up to 80 label-free SCP samples per day. Moreover, an optimized tissue dissociation buffer enabled effective single cell disaggregation of drug-treated cancer cell spheroids, refining the overall proteomic analysis. Our workflow sets a new benchmark in SCP for sensitivity and throughput, with broad applications ranging from the study of cellular development to disease progression and the identification of cell type-specific markers and therapeutic targets.\nFeedback between stochastic gene networks and population dynamics enables cellular decision making\nAuthors: Piho, P.; Thomas, P.\nScore: 10.7, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568034\nPhenotypic selection is the process where genetically identical cells are subject to different re-productive abilities due to cellular noise. Such noise arises from fluctuations in the rates by which proteins are synthesised inside living cells and is hypothesised to play a crucial role in how cells make decisions and respond to stress or drugs. Nevertheless, the state-of-the-art modelling tools for quantifying cellular noise, such as the Gillespie algorithm and the chemical master equation, simply ignore selection. Here, we propose a general stochastic agent-based model for growing populations where the feedback of gene expression dynamics and cell cycle progression directly contribute to cell division dynamics. We devise a finite state projection method as a quantitative analysis tool and inference method for single-cell data obtained from mother machines and lineage trees. We use the theory to quantify selection in multi-stable gene expression networks and elucidate that the interplay between gene expression and phenotypic selection enables robust decision-making essential for developmental lineage decisions. Using live-cell data, we demonstrate that combining theory and inference provides quantitative insights into bet-hedging-like response to DNA damage, and adaption during antibiotic tolerance in Escherichia coli.\nSingle-cell imaging of protein dynamics of paralogs reveals mechanisms of gene retention\nAuthors: Dandage, R.; Papkov, M.; Greco, B. M.; Fishman, D.; Friesen, H.; Wang, K.; Styles, E.; Kraus, O.; Grys, B.; Boone, C.; Andrews, B. J.; Parts, L.; Kuzmin, E.\nScore: 10.1, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568466\nGene duplication is common across the tree of life, including yeast and humans, and contributes to genomic robustness. In this study, we examined changes in the subcellular localization and abundance of proteins in response to the deletion of their paralogs originating from the whole-genome duplication event, which is a largely unexplored mechanism of functional divergence. We performed a systematic single-cell imaging analysis of protein dynamics and screened subcellular redistribution of proteins, capturing their localization and abundance changes, providing insight into forces determining paralog retention. Paralogs showed dependency, whereby proteins required their paralog to maintain their native abundance or localization, more often than compensation. Network feature analysis suggested the importance of functional redundancy and rewiring of protein and genetic interactions underlying redistribution response of paralogs. Translation of non-canonical protein isoform emerged as a novel compensatory mechanism. This study provides new insights into paralog retention and evolutionary forces that shape genomes.\nSiderophore Piracy Promotes Dynamical Coexistence in Microbial Community\nAuthors: Shao, J.; Li, Y.; Lu, J.; Gu, S.; Li, Z.\nScore: 4.3, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568182\nThe game between cooperation and cheating in the microbial world is captivating. Microbes that acquire essential iron through the secretion of siderophores face the public good dilemma. In this game of iron, the specific recognition of different types of siderophores by corresponding receptors adds more complexity. Despite the prevalence of siderophore synthesis and piracy among microbes, the ecological consequences of siderophore-mediated interactions remain largely underexplored. This study introduced the framework of resource partition model for siderophore-mediated interactions, revealing conditions for stable coexistence in dual-species scenarios and uncovering surprising oscillatory and chaotic coexistence in systems with three or more species. Intriguingly, analytical analysis and simulations highlight the how cheating promotes biodiversity, emphasizing the non-monotonic changes in biodiversity as the level of siderophore piracy increases in producers. This work sheds light on the complex dynamics introduced by resource partition in microbial communities, elucidating the intricate balance between cooperation and cheating. HighlightsO_LIResource partition introduce complex dynamics C_LIO_LICheating play a main force in promoting coexistence C_LIO_LIIncreased proportion of cheating prefers dynamical behaviors C_LI\nA modular model integrating metabolism, growth, and cell cycle predicts that fermentation is required to modulate cell size in yeast populations\nAuthors: Vanoni, M.; Palumbo, P.; Papa, F.; Busti, S.; Gotti, L.; Wortel, M. T.; Teusink, B.; Orlandi, I.; Pessina, A.; Airoldi, C.; Brambilla, L.; Vai, M.; Alberghina, L.\nScore: 4.0, Published: 2023-11-25 DOI: 10.1101/2023.11.25.568635\nFor unicellular organisms, the reproduction rate and growth are crucial determinants of fitness and, therefore, essential functional manifestations of the organism genotype. Using the budding yeast Saccharomyces cerevisiae as a model organism, we integrated metabolism, which provides energy and building blocks for growth, with cell mass growth and cell cycle progression into a low-granularity, multiscale (from cell to population) computational model. This model predicted that cells with constitutive respiration do not modulate cell size according to the growth conditions. We experimentally validated the model predictions using mutants with defects in the upper part of glycolysis or glucose transport. Plugging in molecular details of cellular subsystems allowed us to refine predictions from the cellular to the molecular level. Our hybrid multiscale modeling approach provides a framework for structuring molecular knowledge and predicting cell phenotypes under various genetic and environmental conditions.\nTranslation variation across genetic backgrounds reveals a post-transcriptional buffering signature in yeast\nAuthors: Teyssonniere, E.; Shichino, Y.; Friedrich, A.; Iwasaki, S.; Schacherer, J.\nScore: 3.8, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568778\nGene expression is known to vary among individuals, and this variability can impact the phenotypic diversity observed in natural populations. While the transcriptome and proteome have been extensively studied, little is known about the translation process itself. Here, we therefore performed ribosome and transcriptomic profiling on a genetically and ecologically diverse set of natural isolates of the Saccharomyces cerevisiae yeast. Interestingly, we found that the Euclidean distances between each profile and the expression fold changes in each pairwise isolate comparison were higher at the transcriptomic level. This observation clearly indicates that the transcriptional variation observed in the different isolates is buffered through a phenomenon known as post-transcriptional buffering at the translation level. Furthermore, this phenomenon seemed to have a specific signature by preferentially affecting essential genes as well as genes involved in complex-forming proteins, and low transcribed genes. We also explored the translation of the S. cerevisiae pangenome and found that the accessory genes related to introgression events displayed similar transcription and translation levels as the core genome. By contrast, genes acquired through horizontal gene transfer events tended to be less efficiently translated. Together, our results highlight both the extent and signature of the post-transcriptional buffering.\nAn enzyme activation network provides evidence for extensive regulatory crosstalk between metabolic pathways\nAuthors: Alam, M. T.; Al Zubaidi, S. M.; Nasar, M. I.; Ralser, M.; Notebaart, R. A.\nScore: 12.7, Published: 2023-11-18 DOI: 10.1101/2023.11.16.567372\nEnzyme activation by cellular metabolites plays a pivotal role in regulating metabolic processes. Nevertheless, our comprehension of such activation events on a global network scale remains incomplete. In this study, we conducted a comprehensive investigation into the optimization of cell-intrinsic activation interactions within Saccharomyces cerevisiae. To achieve this, we integrated a genome-scale metabolic model with enzyme kinetic data sourced from the BRENDA database. Our objective was to map the distribution of enzyme activators throughout the cellular network. Our findings indicate that virtually all biochemical pathways encompass enzyme activators, frequently originating from disparate pathways, thus revealing extensive regulatory crosstalk between metabolic pathways. Indeed, activators have short pathway lengths, indicating they are activated quickly upon nutrient shifts, and in most instances, these activators target key enzymatic reactions to facilitate downstream metabolic processes. Interestingly, non-essential enzymes exhibit a significantly higher degree of activation compared to their essential counterparts. This observation suggests that cells employ enzyme activators to finely regulate secondary metabolic pathways that are only required under specific conditions. Conversely, the activator metabolites themselves are more likely to be essential components, and their activation levels surpass those of non-essential activators. In summary, our study unveils the widespread importance of enzymatic activators, and suggests that feed-forward activation of conditional metabolic pathways through essential metabolites mediates metabolic plasticity.\nProteome-wide base editor screens to assess phosphorylation site functionality in high-throughput\nAuthors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.\nScore: 18.1, Published: 2023-11-14 DOI: 10.1101/2023.11.11.566649\nSignaling pathways that drive gene expression are typically depicted as having a dozen or so landmark phosphorylation and transcriptional events. In reality, thousands of dynamic post-translational modifications (PTMs) orchestrate nearly every cellular function, and we lack technologies to find causal links between these vast biochemical pathways and genetic circuits at scale. Here, we describe \"signaling-to-transcription network\" mapping through the development of PTM-centric base editing coupled to phenotypic screens, directed by temporally-resolved phosphoproteomics. Using T cell activation as a model, we observe hundreds of unstudied phosphorylation sites that modulate NFAT transcriptional activity. We identify the phosphorylation-mediated nuclear localization of the phosphatase PHLPP1 which promotes NFAT but inhibits NF{kappa}B activity. We also find that specific phosphosite mutants can alter gene expression in subtle yet distinct patterns, demonstrating the potential for fine-tuning transcriptional responses. Overall, base editor screening of PTM sites provides a powerful platform to dissect PTM function within signaling pathways.\n",
  "wordCount" : "2220",
  "inLanguage": "en",
  "datePublished": "2023-11-29T10:40:14Z",
  "dateModified": "2023-11-29T10:40:14Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 29, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568149">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568149" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568149">
        <p class="paperTitle">One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568149" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568149" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.</p>
        <p class="info">Score: 25.8, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568149' target='https://doi.org/10.1101/2023.11.21.568149'> 10.1101/2023.11.21.568149</a></p>
        <p class="abstract">As a key mechanism for cellular signal transduction, protein phosphorylation plays critical roles in myriad cellular processes. This modification, however, is highly dynamic and occurs at sub-stoichiometric levels. Mass spectrometry is an essential tool for studying this modification on a global scale; however, the technologys impact suffers from four main limitations: need for site localization, dynamic range, reproducibility, and throughput. Here we describe the use of a novel mass spectrometer (Orbitrap Astral) coupled with data-independent acquisition (DIA) to achieve detection of close to 40,000 unique phosphorylation sites within one hour of analysis. We applied this approach to generate a phosphoproteome atlas of the mouse. Altogether, we detected 81,120 unique phosphorylation sites within 12 hours of measurement. With this unique dataset, we examine the sequence and structural context of protein phosphorylation. Finally, we highlight the discovery potential of this resource with multiple examples of novel phosphorylation events relevant to mitochondrial and brain biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568418">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568418" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568418">
        <p class="paperTitle">Systematic optimization of automated phosphopeptide enrichment for high-sensitivity phosphoproteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568418" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568418" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bortel, P.; Piga, I.; Koenig, C.; Gerner, C.; Martinez del Val, A.; Olsen, J. V.</p>
        <p class="info">Score: 23.6, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568418' target='https://doi.org/10.1101/2023.11.23.568418'> 10.1101/2023.11.23.568418</a></p>
        <p class="abstract">Improving coverage, robustness and sensitivity is crucial for routine phosphoproteomics analysis by single-shot liquid chromatography tandem mass spectrometry (LC-MS/MS) runs from minimal peptide inputs. Here, we systematically optimized key experimental parameters for automated on-beads phosphoproteomics sample preparation with focus on low input samples. Assessing the number of identified phosphopeptides, enrichment efficiency, site localization scores and relative enrichment of multiply-phosphorylated peptides pinpointed critical variables influencing the resulting phosphoproteome. Optimizing glycolic acid concentration in the loading buffer, percentage of ammonium hydroxide in the elution buffer, peptide-to-beads ratio, binding time, sample and loading buffer volumes, allowed us to confidently identify &gt;16,000 phosphopeptides in half-an-hour LC-MS/MS on an Orbitrap Exploris 480 using 30 {micro}g of peptides as starting material. Furthermore, we evaluated how sequential enrichment can boost phosphoproteome coverage and showed that pooling fractions into a single LC-MS/MS analysis increased the depth. We also present an alternative phosphopeptide enrichment strategy based on stepwise addition of beads thereby boosting phosphoproteome coverage by 20%. Finally, we applied our optimized strategy to evaluate phosphoproteome depth with the Orbitrap Astral MS using a cell dilution series and were able to identify &gt;32,000 phosphopeptides from 0.5 million HeLa cells in half-an-hour LC-MS/MS using narrow-window data-independent acquisition (nDIA).

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=159 HEIGHT=200 SRC=&#34;FIGDIR/small/568418v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (34K):
org.highwire.dtl.DTLVardef@149b840org.highwire.dtl.DTLVardef@dfade8org.highwire.dtl.DTLVardef@112788aorg.highwire.dtl.DTLVardef@1a7c638_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568953">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568953" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568953">
        <p class="paperTitle">High-throughput and scalable single cell proteomics identifies over 5000 proteins per cell</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568953" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568953" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ye, Z.; Sabatier, P.; Hoeven, L.; Phlairaharn, T.; Hartlmayr, D.; Izaguirre, F.; Seth, A.; Joshi, H.; Bekker-Jensen, D.; Bache, N.; Olsen, J. V.</p>
        <p class="info">Score: 20.2, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568953' target='https://doi.org/10.1101/2023.11.27.568953'> 10.1101/2023.11.27.568953</a></p>
        <p class="abstract">The emergence of mass spectrometry (MS)-based single-cell proteomics (SCP) promise to revolutionize the study of cellular biology and biomedicine by providing an unparalleled view of the proteome in individual cells. Despite its groundbreaking potential, SCP is nascent and faces challenges including limited sequence depth, throughput, and reproducibility, which have constrained its broader utility. This study introduces key methodological advances, which considerably improve the sensitivity, coverage and dependability of protein identification from single cells. We developed an almost lossless SCP workflow encompassing sample preparation to MS analysis, doubling the number of identified proteins from roughly 2000 to over 5000 in individual HeLa cells. A comprehensive evaluation of analytical software tools, alongside strict false discovery rate (FDR) controls solidified the reliability of our results. These enhancements also facilitated the direct detection of post-translational modifications (PTMs) in single cells, negating the need for enrichment and thereby simplifying the analytical process. Although throughput in MS remains a challenge, our study demonstrates the feasibility of processing up to 80 label-free SCP samples per day. Moreover, an optimized tissue dissociation buffer enabled effective single cell disaggregation of drug-treated cancer cell spheroids, refining the overall proteomic analysis. Our workflow sets a new benchmark in SCP for sensitivity and throughput, with broad applications ranging from the study of cellular development to disease progression and the identification of cell type-specific markers and therapeutic targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568034">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568034" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568034">
        <p class="paperTitle">Feedback between stochastic gene networks and population dynamics enables cellular decision making</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568034" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568034" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Piho, P.; Thomas, P.</p>
        <p class="info">Score: 10.7, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568034' target='https://doi.org/10.1101/2023.11.21.568034'> 10.1101/2023.11.21.568034</a></p>
        <p class="abstract">Phenotypic selection is the process where genetically identical cells are subject to different re-productive abilities due to cellular noise. Such noise arises from fluctuations in the rates by which proteins are synthesised inside living cells and is hypothesised to play a crucial role in how cells make decisions and respond to stress or drugs. Nevertheless, the state-of-the-art modelling tools for quantifying cellular noise, such as the Gillespie algorithm and the chemical master equation, simply ignore selection. Here, we propose a general stochastic agent-based model for growing populations where the feedback of gene expression dynamics and cell cycle progression directly contribute to cell division dynamics. We devise a finite state projection method as a quantitative analysis tool and inference method for single-cell data obtained from mother machines and lineage trees. We use the theory to quantify selection in multi-stable gene expression networks and elucidate that the interplay between gene expression and phenotypic selection enables robust decision-making essential for developmental lineage decisions. Using live-cell data, we demonstrate that combining theory and inference provides quantitative insights into bet-hedging-like response to DNA damage, and adaption during antibiotic tolerance in Escherichia coli.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568466">
        <p class="paperTitle">Single-cell imaging of protein dynamics of paralogs reveals mechanisms of gene retention</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dandage, R.; Papkov, M.; Greco, B. M.; Fishman, D.; Friesen, H.; Wang, K.; Styles, E.; Kraus, O.; Grys, B.; Boone, C.; Andrews, B. J.; Parts, L.; Kuzmin, E.</p>
        <p class="info">Score: 10.1, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568466' target='https://doi.org/10.1101/2023.11.23.568466'> 10.1101/2023.11.23.568466</a></p>
        <p class="abstract">Gene duplication is common across the tree of life, including yeast and humans, and contributes to genomic robustness. In this study, we examined changes in the subcellular localization and abundance of proteins in response to the deletion of their paralogs originating from the whole-genome duplication event, which is a largely unexplored mechanism of functional divergence. We performed a systematic single-cell imaging analysis of protein dynamics and screened subcellular redistribution of proteins, capturing their localization and abundance changes, providing insight into forces determining paralog retention. Paralogs showed dependency, whereby proteins required their paralog to maintain their native abundance or localization, more often than compensation. Network feature analysis suggested the importance of functional redundancy and rewiring of protein and genetic interactions underlying redistribution response of paralogs. Translation of non-canonical protein isoform emerged as a novel compensatory mechanism. This study provides new insights into paralog retention and evolutionary forces that shape genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568182">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568182" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568182">
        <p class="paperTitle">Siderophore Piracy Promotes Dynamical Coexistence in Microbial Community</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568182" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568182" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shao, J.; Li, Y.; Lu, J.; Gu, S.; Li, Z.</p>
        <p class="info">Score: 4.3, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568182' target='https://doi.org/10.1101/2023.11.21.568182'> 10.1101/2023.11.21.568182</a></p>
        <p class="abstract">The game between cooperation and cheating in the microbial world is captivating. Microbes that acquire essential iron through the secretion of siderophores face the public good dilemma. In this game of iron, the specific recognition of different types of siderophores by corresponding receptors adds more complexity. Despite the prevalence of siderophore synthesis and piracy among microbes, the ecological consequences of siderophore-mediated interactions remain largely underexplored. This study introduced the framework of resource partition model for siderophore-mediated interactions, revealing conditions for stable coexistence in dual-species scenarios and uncovering surprising oscillatory and chaotic coexistence in systems with three or more species. Intriguingly, analytical analysis and simulations highlight the how  cheating promotes biodiversity, emphasizing the non-monotonic changes in biodiversity as the level of siderophore piracy increases in producers. This work sheds light on the complex dynamics introduced by resource partition in microbial communities, elucidating the intricate balance between cooperation and cheating.

HighlightsO_LIResource partition introduce complex dynamics
C_LIO_LICheating play a main force in promoting coexistence
C_LIO_LIIncreased proportion of cheating prefers dynamical behaviors
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.568635">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.568635" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.568635">
        <p class="paperTitle">A modular model integrating metabolism, growth, and cell cycle predicts that fermentation is required to modulate cell size in yeast populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.568635" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.568635" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vanoni, M.; Palumbo, P.; Papa, F.; Busti, S.; Gotti, L.; Wortel, M. T.; Teusink, B.; Orlandi, I.; Pessina, A.; Airoldi, C.; Brambilla, L.; Vai, M.; Alberghina, L.</p>
        <p class="info">Score: 4.0, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.568635' target='https://doi.org/10.1101/2023.11.25.568635'> 10.1101/2023.11.25.568635</a></p>
        <p class="abstract">For unicellular organisms, the reproduction rate and growth are crucial determinants of fitness and, therefore, essential functional manifestations of the organism genotype. Using the budding yeast Saccharomyces cerevisiae as a model organism, we integrated metabolism, which provides energy and building blocks for growth, with cell mass growth and cell cycle progression into a low-granularity, multiscale (from cell to population) computational model. This model predicted that cells with constitutive respiration do not modulate cell size according to the growth conditions. We experimentally validated the model predictions using mutants with defects in the upper part of glycolysis or glucose transport. Plugging in molecular details of cellular subsystems allowed us to refine predictions from the cellular to the molecular level. Our hybrid multiscale modeling approach provides a framework for structuring molecular knowledge and predicting cell phenotypes under various genetic and environmental conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568778" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568778">
        <p class="paperTitle">Translation variation across genetic backgrounds reveals a post-transcriptional buffering signature in yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Teyssonniere, E.; Shichino, Y.; Friedrich, A.; Iwasaki, S.; Schacherer, J.</p>
        <p class="info">Score: 3.8, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568778' target='https://doi.org/10.1101/2023.11.28.568778'> 10.1101/2023.11.28.568778</a></p>
        <p class="abstract">Gene expression is known to vary among individuals, and this variability can impact the phenotypic diversity observed in natural populations. While the transcriptome and proteome have been extensively studied, little is known about the translation process itself. Here, we therefore performed ribosome and transcriptomic profiling on a genetically and ecologically diverse set of natural isolates of the Saccharomyces cerevisiae yeast. Interestingly, we found that the Euclidean distances between each profile and the expression fold changes in each pairwise isolate comparison were higher at the transcriptomic level. This observation clearly indicates that the transcriptional variation observed in the different isolates is buffered through a phenomenon known as post-transcriptional buffering at the translation level. Furthermore, this phenomenon seemed to have a specific signature by preferentially affecting essential genes as well as genes involved in complex-forming proteins, and low transcribed genes. We also explored the translation of the S. cerevisiae pangenome and found that the accessory genes related to introgression events displayed similar transcription and translation levels as the core genome. By contrast, genes acquired through horizontal gene transfer events tended to be less efficiently translated. Together, our results highlight both the extent and signature of the post-transcriptional buffering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567372">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567372" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567372">
        <p class="paperTitle">An enzyme activation network provides evidence for extensive regulatory crosstalk between metabolic pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567372" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567372" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alam, M. T.; Al Zubaidi, S. M.; Nasar, M. I.; Ralser, M.; Notebaart, R. A.</p>
        <p class="info">Score: 12.7, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567372' target='https://doi.org/10.1101/2023.11.16.567372'> 10.1101/2023.11.16.567372</a></p>
        <p class="abstract">Enzyme activation by cellular metabolites plays a pivotal role in regulating metabolic processes. Nevertheless, our comprehension of such activation events on a global network scale remains incomplete. In this study, we conducted a comprehensive investigation into the optimization of cell-intrinsic activation interactions within Saccharomyces cerevisiae. To achieve this, we integrated a genome-scale metabolic model with enzyme kinetic data sourced from the BRENDA database. Our objective was to map the distribution of enzyme activators throughout the cellular network. Our findings indicate that virtually all biochemical pathways encompass enzyme activators, frequently originating from disparate pathways, thus revealing extensive regulatory crosstalk between metabolic pathways. Indeed, activators have short pathway lengths, indicating they are activated quickly upon nutrient shifts, and in most instances, these activators target key enzymatic reactions to facilitate downstream metabolic processes. Interestingly, non-essential enzymes exhibit a significantly higher degree of activation compared to their essential counterparts. This observation suggests that cells employ enzyme activators to finely regulate secondary metabolic pathways that are only required under specific conditions. Conversely, the activator metabolites themselves are more likely to be essential components, and their activation levels surpass those of non-essential activators. In summary, our study unveils the widespread importance of enzymatic activators, and suggests that feed-forward activation of conditional metabolic pathways through essential metabolites mediates metabolic plasticity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566649" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566649">
        <p class="paperTitle">Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.</p>
        <p class="info">Score: 18.1, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566649' target='https://doi.org/10.1101/2023.11.11.566649'> 10.1101/2023.11.11.566649</a></p>
        <p class="abstract">Signaling pathways that drive gene expression are typically depicted as having a dozen or so landmark phosphorylation and transcriptional events. In reality, thousands of dynamic post-translational modifications (PTMs) orchestrate nearly every cellular function, and we lack technologies to find causal links between these vast biochemical pathways and genetic circuits at scale. Here, we describe &#34;signaling-to-transcription network&#34; mapping through the development of PTM-centric base editing coupled to phenotypic screens, directed by temporally-resolved phosphoproteomics. Using T cell activation as a model, we observe hundreds of unstudied phosphorylation sites that modulate NFAT transcriptional activity. We identify the phosphorylation-mediated nuclear localization of the phosphatase PHLPP1 which promotes NFAT but inhibits NF{kappa}B activity. We also find that specific phosphosite mutants can alter gene expression in subtle yet distinct patterns, demonstrating the potential for fine-tuning transcriptional responses. Overall, base editor screening of PTM sites provides a powerful platform to dissect PTM function within signaling pathways.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
