<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="CURTAIN - A Unique Web-based tool for exploration and sharing of MS-based proteomics data
Authors: Phung, T. K.; Berndsen, K.; Phan, T. L.; Muqit, M. M.; Alessi, D. R.; Nirujogi, R. S.
Score: 26.3, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550405
To facilitate analysis and sharing of mass spectrometry (MS)-based proteomics data we created tools called CURTAIN (https://curtain.proteo.info) and CURTAIN-PTM (https://curtainptm.proteo.info). These enable the non-MS expert to interactively peruse volcano plots; deconvolute primary experimental data to individual replicates that can be visualized in bar charts or violin plots allowing statistical analysis; and export of plots in SVG format.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="CURTAIN - A Unique Web-based tool for exploration and sharing of MS-based proteomics data
Authors: Phung, T. K.; Berndsen, K.; Phan, T. L.; Muqit, M. M.; Alessi, D. R.; Nirujogi, R. S.
Score: 26.3, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550405
To facilitate analysis and sharing of mass spectrometry (MS)-based proteomics data we created tools called CURTAIN (https://curtain.proteo.info) and CURTAIN-PTM (https://curtainptm.proteo.info). These enable the non-MS expert to interactively peruse volcano plots; deconvolute primary experimental data to individual replicates that can be visualized in bar charts or violin plots allowing statistical analysis; and export of plots in SVG format." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-02T10:38:09+00:00" />
<meta property="article:modified_time" content="2023-08-02T10:38:09+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="CURTAIN - A Unique Web-based tool for exploration and sharing of MS-based proteomics data
Authors: Phung, T. K.; Berndsen, K.; Phan, T. L.; Muqit, M. M.; Alessi, D. R.; Nirujogi, R. S.
Score: 26.3, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550405
To facilitate analysis and sharing of mass spectrometry (MS)-based proteomics data we created tools called CURTAIN (https://curtain.proteo.info) and CURTAIN-PTM (https://curtainptm.proteo.info). These enable the non-MS expert to interactively peruse volcano plots; deconvolute primary experimental data to individual replicates that can be visualized in bar charts or violin plots allowing statistical analysis; and export of plots in SVG format."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "CURTAIN - A Unique Web-based tool for exploration and sharing of MS-based proteomics data\nAuthors: Phung, T. K.; Berndsen, K.; Phan, T. L.; Muqit, M. M.; Alessi, D. R.; Nirujogi, R. S.\nScore: 26.3, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550405\nTo facilitate analysis and sharing of mass spectrometry (MS)-based proteomics data we created tools called CURTAIN (https://curtain.proteo.info) and CURTAIN-PTM (https://curtainptm.proteo.info). These enable the non-MS expert to interactively peruse volcano plots; deconvolute primary experimental data to individual replicates that can be visualized in bar charts or violin plots allowing statistical analysis; and export of plots in SVG format.",
  "keywords": [
    
  ],
  "articleBody": " CURTAIN - A Unique Web-based tool for exploration and sharing of MS-based proteomics data\nAuthors: Phung, T. K.; Berndsen, K.; Phan, T. L.; Muqit, M. M.; Alessi, D. R.; Nirujogi, R. S.\nScore: 26.3, Published: 2023-07-25 DOI: 10.1101/2023.07.25.550405\nTo facilitate analysis and sharing of mass spectrometry (MS)-based proteomics data we created tools called CURTAIN (https://curtain.proteo.info) and CURTAIN-PTM (https://curtainptm.proteo.info). These enable the non-MS expert to interactively peruse volcano plots; deconvolute primary experimental data to individual replicates that can be visualized in bar charts or violin plots allowing statistical analysis; and export of plots in SVG format. They also permit assessment of experimental quality by correlation matrix and profile plot. Within CURTAIN, the user can analyze domain structure, AlphaFold predicted structure, reported interactors, relative expression, disease and pharmaceutical links, and mutagenesis information on all selected hits. Moreover, CURTAIN-PTM permits the comparison of all identified PTM sites on protein(s) of interest with PTM information contained within selected databases. For phosphorylation site analysis CURTAIN-PTM links with the kinase library to predict upstream kinases that phosphorylate sites of interest. We provide examples of the utility of CURTAIN and CURTAIN-PTM in analyzing how targeted degradation of the PPM1H Rab phosphatase that counteracts the Parkinsons LRRK2 kinase impacts cellular protein levels and phosphorylation sites. We reanalyzed a ubiquitylation dataset, characterizing the PINK1-Parkin pathway activation in primary neurons, revealing new data of interest not highlighted previously. CURTAIN and CURTAIN-PTM are free to use and open-source and will enable researchers to share and maximize the analysis and impact of their proteomics data. We advocate that differential expression proteomic data should be published containing a shareable CURTAIN web-link, allowing readers to better explore their data. Significance StatementTo enable non-experts to better share and explore mass spectrometry data, we have generated using open-source software, interactive tools termed CURTAIN and CURTAIN-PTM. These tools enable users to save their analysis sessions with a sharable unique web-link, enabling other researchers to visualize and further analyze these datasets. These links can also be reported in publications allowing readers to further survey the reported data. We discuss benefits for the research community of publishing proteomic data containing a shareable web-link.\nA comprehensive atlas of the aging vertebrate brain reveals signatures of progressive proteostasis dysfunction\nAuthors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.\nScore: 34.0, Published: 2023-07-24 DOI: 10.1101/2023.07.20.549210\nProtein homeostasis is disrupted in aging and neurodegenerative diseases, yet, the specific impact of aging on brain proteostasis remains poorly understood. Here, we measured and integrated the effects of aging on the transcriptome, translatome, and multiple layers of the proteome in the brain of a short-lived killifish. We find that aging causes a decoupling between transcriptome and proteome. This leads to decreased abundance of proteins enriched in basic amino acids such as DNA/RNA-binding proteins and increased levels of others, independent of mRNA changes. Chronic proteasome impairment in vivo induces aging signatures in lysosomes and mitochondria. However, it does not recapitulate the age-related decoupling between transcripts and proteins. Instead, aberrant translation pausing and ensuing reduced ribosome availability reprogram the proteome independently of transcription. The age-linked changes in protein biogenesis likely enhance aggregation and reduce availability of key protein complexes, thus contributing to proteome dysfunction and aging hallmarks in older brains. HighlightsO_LIA resource of protein solubility, organelle composition and PTMs in the aging vertebrate brain C_LIO_LIProteostasis alterations converge on mitochondria, ribosomes and biosynthetic pathways C_LIO_LIBasic proteins, e.g., DNA/RNA binding, are reduced due to translation pausing C_LIO_LIDecreased ribosome availability reprograms protein synthesis in old brains C_LI O_FIG O_LINKSMALLFIG WIDTH=199 HEIGHT=200 SRC=\"FIGDIR/small/549210v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (46K): org.highwire.dtl.DTLVardef@9580d5org.highwire.dtl.DTLVardef@13d58aborg.highwire.dtl.DTLVardef@884f7forg.highwire.dtl.DTLVardef@11b203f_HPS_FORMAT_FIGEXP M_FIG C_FIG\nProstate Cancer Reshapes the Secreted and Extracellular Vesicle Urinary Proteomes\nAuthors: Khoo, A.; Govindarajan, M.; Qui, Z.; Liu, L. Y.; Ignatchenko, V.; Waas, M.; Macklin, A.; Keszei, A.; Main, B. P.; Yang, L.; Lance, R. S.; Downes, M. R.; Semmes, O. J.; Vesprini, D.; Liu, S. K.; Nyalwidhe, J. O.; Boutros, P. C.; Kislinger, T.\nScore: 9.0, Published: 2023-07-25 DOI: 10.1101/2023.07.23.550214\nUrine is a complex biofluid that reflects both overall physiologic state and the state of the genitourinary tissues through which it passes. It contains both secreted proteins and proteins encapsulated in tissue-derived extracellular vesicles (EVs). To understand the population variability and clinical utility of urine, we quantified the secreted and EV proteomes from 190 men, including a subset with prostate cancer. We demonstrate that a simple protocol enriches prostatic proteins in urine. Secreted and EV proteins arise from different subcellular compartments. Urinary EVs are faithful surrogates of tissue proteomes, but secreted proteins in urine or cell line EVs are not. The urinary proteome is longitudinally stable over several years. It can accurately and non-invasively distinguish malignant from benign prostatic lesions, and can risk-stratify prostate tumors. This resource quantifies the complexity of the urinary proteome, and reveals the synergistic value of secreted and EV proteomes for translational and biomarker studies.\nMachine Learning identifies conserved traits that influence lifespan and healthspan responses to dietary restriction\nAuthors: Hilsabeck, T. A.; Narayan, V. P.; Wilson, K. A.; Carrera, E.; Raftery, D.; Promislow, D. E.; Brem, R. B.; Campisi, J. A.; Kapahi, P.\nScore: 5.3, Published: 2023-07-13 DOI: 10.1101/2023.07.09.548232\nDietary restriction (DR) is the most robust means to extend lifespan and healthspan across species, but factors such as genetic variation affect how an individual will respond to DR. Additionally, it is unclear how cumulative variations in metabolism and the metabolome influence longevity and health. We utilized metabolomic, phenotypic, and genome-wide data from Drosophila Genetic Reference Panel strains raised under ad libitum and DR conditions to identify factors which influence longevity and health in response to dietary modulation. We found multiple intra-dataset correlations (e.g., metabolites with metabolites) but few inter-dataset correlations (e.g., metabolites with health-related phenotypes). Through random forest modeling across all traits and Mendelian Randomization, we found key translatable traits that influence lifespan or healthspan determination and validated the role of multiple metabolites in regulating lifespan. Through these approaches, we utilized data from flies and humans to elucidate potential therapeutic pathways and metabolomic targets for diet response, lifespan, and healthspan.\nEnhanced cell segmentation with limited annotated data using generative adversarial networks\nAuthors: Zargari, A.; Mashhadi, N.; Shariati, A.\nScore: 3.7, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550715\nThe application of deep learning is rapidly transforming the field of bioimage analysis. While deep learning has shown great promise in complex microscopy tasks such as single-cell segmentation, the development of generalizable foundation deep learning segmentation models is hampered by the scarcity of large and diverse annotated datasets of cell images for training purposes. Generative Adversarial Networks (GANs) can generate realistic images that can potentially be easily used to train deep learning models without the generation of large manually annotated microscopy images. Here, we propose a customized CycleGAN architecture to train an enhanced cell segmentation model with limited annotated cell images, effectively addressing the challenge of paucity of annotated data in microscopy imaging. Our customized CycleGAN model can generate realistic synthetic images of cells with morphological details and nuances very similar to that of real images. This method not only increases the variability seen during training but also enhances the authenticity of synthetic samples, thereby enhancing the overall predictive accuracy and robustness of the cell segmentation model. Our experimental results show that our CycleGAN-based method significantly improves the performance of the segmentation model compared to conventional training techniques. Interestingly, we demonstrate that our model can extrapolate its knowledge by synthesizing imaging scenarios that were not seen during the training process. Our proposed customized CycleGAN method will accelerate the development of foundation models for cell segmentation in microscopy images.\nSystems Age: A single blood methylation test to quantify aging heterogeneity across 11 physiological systems\nAuthors: Sehgal, R.; Meer, M.; Shadyab, A. H.; Casanova, R.; Manson, J. E.; Bhatti, P.; Crimmins, E. M.; Assimes, T. L.; Whitsel, E. A.; Higgins-Chen, A. T.; Levine, M.\nScore: 42.4, Published: 2023-07-17 DOI: 10.1101/2023.07.13.548904\nIndividuals, organs, tissues, and cells age in diverse ways throughout the lifespan. Epigenetic clocks attempt to quantify differential aging between individuals, but they typically summarize aging as a single measure, ignoring within-person heterogeneity. Our aim was to develop novel systems-based methylation clocks that, when assessed in blood, capture aging in distinct physiological systems. We combined supervised and unsupervised machine learning methods to link DNA methylation, system-specific clinical chemistry and functional measures, and mortality risk. This yielded a panel of 11 system-specific scores- Heart, Lung, Kidney, Liver, Brain, Immune, Inflammatory, Blood, Musculoskeletal, Hormone, and Metabolic. Each system score predicted a wide variety of outcomes, aging phenotypes, and conditions specific to the respective system, and often did so more strongly than existing epigenetic clocks that report single global measures. We also combined the system scores into a composite Systems Age clock that is predictive of aging across physiological systems in an unbiased manner. Finally, we showed that the system scores clustered individuals into unique aging subtypes that had different patterns of age-related disease and decline. Overall, our biological systems based epigenetic framework captures aging in multiple physiological systems using a single blood draw and assay and may inform the development of more personalized clinical approaches for improving age-related quality of life.\nComputational prediction of protein interactions on single cells by proximity sequencing\nAuthors: Xia, J.; Phan, H. V.; Vistain, L.; Chen, M.; Khan, A. A.; Tay, S.\nScore: 2.9, Published: 2023-07-30 DOI: 10.1101/2023.07.27.550388\nProximity sequencing (Prox-seq) measures gene expression, protein expression, and protein complexes at the single cell level, using information from dual-antibody binding events and a single cell sequencing readout. Prox-seq provides multi-dimensional phenotyping of single cells and was recently used to track the formation of receptor complexes during inflammatory signaling in macrophages and to discover a new interaction between CD9/CD8 proteins on naive T cells. The distribution of protein abundance affects identification of protein complexes in a complicated manner in dual-binding assays like Prox-seq. These effects are difficult to explore with experiments, yet important for accurate quantification of protein complexes. Here, we introduce a physical model for protein dimer formation on single cells and computationally evaluate several different methods for reducing background noise when quantifying protein complexes. Furthermore, we developed an improved method for analysis of Prox-seq single-cell data, which resulted in more accurate and robust quantification of protein complexes. Finally, our model offers a simple way to investigate the behavior of Prox-seq under various biological conditions and guide users toward selecting the best analysis method for their data.\nAutomatic design of gene regulatory mechanisms for spatial pattern formation\nAuthors: Mousavi, R.; Lobo, D.\nScore: 2.6, Published: 2023-07-29 DOI: 10.1101/2023.07.26.550573\nGene regulatory mechanisms (GRMs) control the formation of spatial and temporal expression patterns that can serve as regulatory signals for the development of complex shapes. Synthetic developmental biology aims to engineer such mechanisms for understanding and producing desired multicellular patterns. However, the design of synthetic GRMs that can produce a target spatial gene expression pattern is a current challenge due to the nonlinear interactions and feedback loops present in genetic circuits. Here we present a methodology to automatically design GRMs that can produce any given spatial pattern. The proposed approach uses two orthogonal morphogen gradients acting as positional information signals in a multicellular tissue area or culture, which constitutes a continuous field of engineered cells implementing the same designed GRM. To efficiently design both the circuit network and the interaction mechanisms--including the number of genes required for the formation of the target pattern--we developed an automated algorithm based on high-performance evolutionary computation. The tolerance of the algorithm can be configured to design GRMs that are either simple to produce approximate patterns or complex to produce precise patterns. We demonstrate its performance by automatically designing GRMs for a diverse set of synthetic spatial expression patterns. The proposed method offers a versatile approach to systematically design and discover pattern-producing genetic circuits.\nAmyloid β accelerates age-related proteome-wide protein insolubility.\nAuthors: Anderton, E.; Chamoli, M.; Bhaumik, D.; King, C. D.; Xie, X.; Foulger, A.; Andersen, J.; Schilling, B.; Lithgow, G.\nScore: 15.7, Published: 2023-07-16 DOI: 10.1101/2023.07.13.548937\nLoss of proteostasis is a highly conserved feature of aging across model organisms and typically results in the accumulation of insoluble protein aggregates. Protein insolubility is a central feature of major age-related neurodegenerative diseases including Alzheimer's Disease (AD), where hundreds of insoluble proteins associate with aggregated amyloid beta (A{beta}) in senile plaques. Despite the established connection between aging and AD risk, therapeutic approaches to date have overlooked aging and proteome-wide protein insolubility as causal factors, instead focusing on A{beta} and Tau. Here, using an unbiased proteomics approach, we questioned the relationship between A{beta} and age-related protein insolubility. We demonstrate that, in C. elegans, A{beta} expression is sufficient to drive proteome-wide protein insolubility. The A{beta}-driven insoluble proteome bears a highly significant overlap with the aging-driven insoluble proteome, suggesting there exists a core, sub-proteome which is vulnerable to insolubility. Using human genome-wide association studies (GWAS) we show that this insoluble sub proteome is replete with biological processes implicated across not only neurodegenerative diseases but also across a broad array of chronic, age-related diseases (CARDs), providing suggestive evidence that age-related loss of proteostasis could play a role in general CARD risk. Finally, we show that the gut metabolite, Urolithin A, relieves A{beta} toxicity supporting its use in clinical trials for dementia and other age-related diseases.\nSystematic exploration of prokaryotic form I rubisco maximal carboxylation rates\nAuthors: de Pins, B.; Greenspoon, L.; Bar-On, Y. M.; Shamshoum, M.; Ben-Nissan, R.; Milshtein, E.; Davidi, D.; Sharon, I.; Mueller-Cajar, O.; Noor, E.; Milo, R.\nScore: 2.5, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550689\nAutotrophy is the basis for complex life on Earth. Its physiological limits are largely constrained by rubisco - the enzyme that catalyzes almost all carbon fixation on the planet. Yet, with only a small fraction of rubisco diversity kinetically characterized so far, the underlying biological factors driving the evolution of fast rubiscos in nature remain unclear. We conducted a high-throughput kinetic characterization of over 100 prokaryotic form I rubiscos, the most ubiquitous group of rubisco sequences in nature, to uncover the determinants of rubiscos carboxylation velocity. We show that phototrophy and the presence of a carboxysome CO2 concentrating mechanism correlate with faster rubiscos (median rate difference of 3-5 fold). In contrast to previous assumptions, rubiscos originating from -cyanobacteria exhibit the highest carboxylation rates among form I enzymes ({approx}10 s-1 median versus \u003c7 s-1 in other groups). Our study ties biological and environmental insights associated with kinetic variation across rubiscos from nature.\n",
  "wordCount" : "2410",
  "inLanguage": "en",
  "datePublished": "2023-08-02T10:38:09Z",
  "dateModified": "2023-08-02T10:38:09Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on August 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550405">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550405" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550405">
        <p class="paperTitle">CURTAIN - A Unique Web-based tool for exploration and sharing of MS-based proteomics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550405" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550405" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Phung, T. K.; Berndsen, K.; Phan, T. L.; Muqit, M. M.; Alessi, D. R.; Nirujogi, R. S.</p>
        <p class="info">Score: 26.3, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550405' target='https://doi.org/10.1101/2023.07.25.550405'> 10.1101/2023.07.25.550405</a></p>
        <p class="abstract">To facilitate analysis and sharing of mass spectrometry (MS)-based proteomics data we created tools called CURTAIN (https://curtain.proteo.info) and CURTAIN-PTM (https://curtainptm.proteo.info). These enable the non-MS expert to interactively peruse volcano plots; deconvolute primary experimental data to individual replicates that can be visualized in bar charts or violin plots allowing statistical analysis; and export of plots in SVG format. They also permit assessment of experimental quality by correlation matrix and profile plot. Within CURTAIN, the user can analyze domain structure, AlphaFold predicted structure, reported interactors, relative expression, disease and pharmaceutical links, and mutagenesis information on all selected hits. Moreover, CURTAIN-PTM permits the comparison of all identified PTM sites on protein(s) of interest with PTM information contained within selected databases. For phosphorylation site analysis CURTAIN-PTM links with the kinase library to predict upstream kinases that phosphorylate sites of interest. We provide examples of the utility of CURTAIN and CURTAIN-PTM in analyzing how targeted degradation of the PPM1H Rab phosphatase that counteracts the Parkinsons LRRK2 kinase impacts cellular protein levels and phosphorylation sites. We reanalyzed a ubiquitylation dataset, characterizing the PINK1-Parkin pathway activation in primary neurons, revealing new data of interest not highlighted previously. CURTAIN and CURTAIN-PTM are free to use and open-source and will enable researchers to share and maximize the analysis and impact of their proteomics data. We advocate that differential expression proteomic data should be published containing a shareable CURTAIN web-link, allowing readers to better explore their data.

Significance StatementTo enable non-experts to better share and explore mass spectrometry data, we have generated using open-source software, interactive tools termed CURTAIN and CURTAIN-PTM. These tools enable users to save their analysis sessions with a sharable unique web-link, enabling other researchers to visualize and further analyze these datasets. These links can also be reported in publications allowing readers to further survey the reported data. We discuss benefits for the research community of publishing proteomic data containing a shareable web-link.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549210">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549210" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549210">
        <p class="paperTitle">A comprehensive atlas of the aging vertebrate brain reveals signatures of progressive proteostasis dysfunction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549210" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549210" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.</p>
        <p class="info">Score: 34.0, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549210' target='https://doi.org/10.1101/2023.07.20.549210'> 10.1101/2023.07.20.549210</a></p>
        <p class="abstract">Protein homeostasis is disrupted in aging and neurodegenerative diseases, yet, the specific impact of aging on brain proteostasis remains poorly understood. Here, we measured and integrated the effects of aging on the transcriptome, translatome, and multiple layers of the proteome in the brain of a short-lived killifish. We find that aging causes a decoupling between transcriptome and proteome. This leads to decreased abundance of proteins enriched in basic amino acids such as DNA/RNA-binding proteins and increased levels of others, independent of mRNA changes. Chronic proteasome impairment in vivo induces aging signatures in lysosomes and mitochondria. However, it does not recapitulate the age-related decoupling between transcripts and proteins. Instead, aberrant translation pausing and ensuing reduced ribosome availability reprogram the proteome independently of transcription. The age-linked changes in protein biogenesis likely enhance aggregation and reduce availability of key protein complexes, thus contributing to proteome dysfunction and aging hallmarks in older brains.

HighlightsO_LIA resource of protein solubility, organelle composition and PTMs in the aging vertebrate brain
C_LIO_LIProteostasis alterations converge on mitochondria, ribosomes and biosynthetic pathways
C_LIO_LIBasic proteins, e.g., DNA/RNA binding, are reduced due to translation pausing
C_LIO_LIDecreased ribosome availability reprograms protein synthesis in old brains
C_LI



O_FIG O_LINKSMALLFIG WIDTH=199 HEIGHT=200 SRC=&#34;FIGDIR/small/549210v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (46K):
org.highwire.dtl.DTLVardef@9580d5org.highwire.dtl.DTLVardef@13d58aborg.highwire.dtl.DTLVardef@884f7forg.highwire.dtl.DTLVardef@11b203f_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.23.550214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.23.550214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.23.550214">
        <p class="paperTitle">Prostate Cancer Reshapes the Secreted and Extracellular Vesicle Urinary Proteomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.23.550214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.23.550214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khoo, A.; Govindarajan, M.; Qui, Z.; Liu, L. Y.; Ignatchenko, V.; Waas, M.; Macklin, A.; Keszei, A.; Main, B. P.; Yang, L.; Lance, R. S.; Downes, M. R.; Semmes, O. J.; Vesprini, D.; Liu, S. K.; Nyalwidhe, J. O.; Boutros, P. C.; Kislinger, T.</p>
        <p class="info">Score: 9.0, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.23.550214' target='https://doi.org/10.1101/2023.07.23.550214'> 10.1101/2023.07.23.550214</a></p>
        <p class="abstract">Urine is a complex biofluid that reflects both overall physiologic state and the state of the genitourinary tissues through which it passes. It contains both secreted proteins and proteins encapsulated in tissue-derived extracellular vesicles (EVs). To understand the population variability and clinical utility of urine, we quantified the secreted and EV proteomes from 190 men, including a subset with prostate cancer. We demonstrate that a simple protocol enriches prostatic proteins in urine. Secreted and EV proteins arise from different subcellular compartments. Urinary EVs are faithful surrogates of tissue proteomes, but secreted proteins in urine or cell line EVs are not. The urinary proteome is longitudinally stable over several years. It can accurately and non-invasively distinguish malignant from benign prostatic lesions, and can risk-stratify prostate tumors. This resource quantifies the complexity of the urinary proteome, and reveals the synergistic value of secreted and EV proteomes for translational and biomarker studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.09.548232">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.09.548232" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.09.548232">
        <p class="paperTitle">Machine Learning identifies conserved traits that influence lifespan and healthspan responses to dietary restriction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.09.548232" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.09.548232" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hilsabeck, T. A.; Narayan, V. P.; Wilson, K. A.; Carrera, E.; Raftery, D.; Promislow, D. E.; Brem, R. B.; Campisi, J. A.; Kapahi, P.</p>
        <p class="info">Score: 5.3, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.09.548232' target='https://doi.org/10.1101/2023.07.09.548232'> 10.1101/2023.07.09.548232</a></p>
        <p class="abstract">Dietary restriction (DR) is the most robust means to extend lifespan and healthspan across species, but factors such as genetic variation affect how an individual will respond to DR. Additionally, it is unclear how cumulative variations in metabolism and the metabolome influence longevity and health. We utilized metabolomic, phenotypic, and genome-wide data from Drosophila Genetic Reference Panel strains raised under ad libitum and DR conditions to identify factors which influence longevity and health in response to dietary modulation. We found multiple intra-dataset correlations (e.g., metabolites with metabolites) but few inter-dataset correlations (e.g., metabolites with health-related phenotypes). Through random forest modeling across all traits and Mendelian Randomization, we found key translatable traits that influence lifespan or healthspan determination and validated the role of multiple metabolites in regulating lifespan. Through these approaches, we utilized data from flies and humans to elucidate potential therapeutic pathways and metabolomic targets for diet response, lifespan, and healthspan.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550715">
        <p class="paperTitle">Enhanced cell segmentation with limited annotated data using generative adversarial networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zargari, A.; Mashhadi, N.; Shariati, A.</p>
        <p class="info">Score: 3.7, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550715' target='https://doi.org/10.1101/2023.07.26.550715'> 10.1101/2023.07.26.550715</a></p>
        <p class="abstract">The application of deep learning is rapidly transforming the field of bioimage analysis. While deep learning has shown great promise in complex microscopy tasks such as single-cell segmentation, the development of generalizable foundation deep learning segmentation models is hampered by the scarcity of large and diverse annotated datasets of cell images for training purposes. Generative Adversarial Networks (GANs) can generate realistic images that can potentially be easily used to train deep learning models without the generation of large manually annotated microscopy images. Here, we propose a customized CycleGAN architecture to train an enhanced cell segmentation model with limited annotated cell images, effectively addressing the challenge of paucity of annotated data in microscopy imaging. Our customized CycleGAN model can generate realistic synthetic images of cells with morphological details and nuances very similar to that of real images. This method not only increases the variability seen during training but also enhances the authenticity of synthetic samples, thereby enhancing the overall predictive accuracy and robustness of the cell segmentation model. Our experimental results show that our CycleGAN-based method significantly improves the performance of the segmentation model compared to conventional training techniques. Interestingly, we demonstrate that our model can extrapolate its knowledge by synthesizing imaging scenarios that were not seen during the training process. Our proposed customized CycleGAN method will accelerate the development of foundation models for cell segmentation in microscopy images.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548904">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548904" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548904">
        <p class="paperTitle">Systems Age: A single blood methylation test to quantify aging heterogeneity across 11 physiological systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548904" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548904" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sehgal, R.; Meer, M.; Shadyab, A. H.; Casanova, R.; Manson, J. E.; Bhatti, P.; Crimmins, E. M.; Assimes, T. L.; Whitsel, E. A.; Higgins-Chen, A. T.; Levine, M.</p>
        <p class="info">Score: 42.4, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548904' target='https://doi.org/10.1101/2023.07.13.548904'> 10.1101/2023.07.13.548904</a></p>
        <p class="abstract">Individuals, organs, tissues, and cells age in diverse ways throughout the lifespan. Epigenetic clocks attempt to quantify differential aging between individuals, but they typically summarize aging as a single measure, ignoring within-person heterogeneity. Our aim was to develop novel systems-based methylation clocks that, when assessed in blood, capture aging in distinct physiological systems. We combined supervised and unsupervised machine learning methods to link DNA methylation, system-specific clinical chemistry and functional measures, and mortality risk. This yielded a panel of 11 system-specific scores- Heart, Lung, Kidney, Liver, Brain, Immune, Inflammatory, Blood, Musculoskeletal, Hormone, and Metabolic. Each system score predicted a wide variety of outcomes, aging phenotypes, and conditions specific to the respective system, and often did so more strongly than existing epigenetic clocks that report single global measures. We also combined the system scores into a composite Systems Age clock that is predictive of aging across physiological systems in an unbiased manner. Finally, we showed that the system scores clustered individuals into unique aging subtypes that had different patterns of age-related disease and decline. Overall, our biological systems based epigenetic framework captures aging in multiple physiological systems using a single blood draw and assay and may inform the development of more personalized clinical approaches for improving age-related quality of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550388">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550388" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550388">
        <p class="paperTitle">Computational prediction of protein interactions on single cells by proximity sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550388" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550388" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xia, J.; Phan, H. V.; Vistain, L.; Chen, M.; Khan, A. A.; Tay, S.</p>
        <p class="info">Score: 2.9, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550388' target='https://doi.org/10.1101/2023.07.27.550388'> 10.1101/2023.07.27.550388</a></p>
        <p class="abstract">Proximity sequencing (Prox-seq) measures gene expression, protein expression, and protein complexes at the single cell level, using information from dual-antibody binding events and a single cell sequencing readout. Prox-seq provides multi-dimensional phenotyping of single cells and was recently used to track the formation of receptor complexes during inflammatory signaling in macrophages and to discover a new interaction between CD9/CD8 proteins on naive T cells. The distribution of protein abundance affects identification of protein complexes in a complicated manner in dual-binding assays like Prox-seq. These effects are difficult to explore with experiments, yet important for accurate quantification of protein complexes. Here, we introduce a physical model for protein dimer formation on single cells and computationally evaluate several different methods for reducing background noise when quantifying protein complexes. Furthermore, we developed an improved method for analysis of Prox-seq single-cell data, which resulted in more accurate and robust quantification of protein complexes. Finally, our model offers a simple way to investigate the behavior of Prox-seq under various biological conditions and guide users toward selecting the best analysis method for their data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550573">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550573" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550573">
        <p class="paperTitle">Automatic design of gene regulatory mechanisms for spatial pattern formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550573" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550573" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mousavi, R.; Lobo, D.</p>
        <p class="info">Score: 2.6, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550573' target='https://doi.org/10.1101/2023.07.26.550573'> 10.1101/2023.07.26.550573</a></p>
        <p class="abstract">Gene regulatory mechanisms (GRMs) control the formation of spatial and temporal expression patterns that can serve as regulatory signals for the development of complex shapes. Synthetic developmental biology aims to engineer such mechanisms for understanding and producing desired multicellular patterns. However, the design of synthetic GRMs that can produce a target spatial gene expression pattern is a current challenge due to the nonlinear interactions and feedback loops present in genetic circuits. Here we present a methodology to automatically design GRMs that can produce any given spatial pattern. The proposed approach uses two orthogonal morphogen gradients acting as positional information signals in a multicellular tissue area or culture, which constitutes a continuous field of engineered cells implementing the same designed GRM. To efficiently design both the circuit network and the interaction mechanisms--including the number of genes required for the formation of the target pattern--we developed an automated algorithm based on high-performance evolutionary computation. The tolerance of the algorithm can be configured to design GRMs that are either simple to produce approximate patterns or complex to produce precise patterns. We demonstrate its performance by automatically designing GRMs for a diverse set of synthetic spatial expression patterns. The proposed method offers a versatile approach to systematically design and discover pattern-producing genetic circuits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548937">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548937" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548937">
        <p class="paperTitle">Amyloid β accelerates age-related proteome-wide protein insolubility.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548937" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548937" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Anderton, E.; Chamoli, M.; Bhaumik, D.; King, C. D.; Xie, X.; Foulger, A.; Andersen, J.; Schilling, B.; Lithgow, G.</p>
        <p class="info">Score: 15.7, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548937' target='https://doi.org/10.1101/2023.07.13.548937'> 10.1101/2023.07.13.548937</a></p>
        <p class="abstract">Loss of proteostasis is a highly conserved feature of aging across model organisms and typically results in the accumulation of insoluble protein aggregates. Protein insolubility is a central feature of major age-related neurodegenerative diseases including Alzheimer&#39;s Disease (AD), where hundreds of insoluble proteins associate with aggregated amyloid beta (A{beta}) in senile plaques. Despite the established connection between aging and AD risk, therapeutic approaches to date have overlooked aging and proteome-wide protein insolubility as causal factors, instead focusing on A{beta} and Tau. Here, using an unbiased proteomics approach, we questioned the relationship between A{beta} and age-related protein insolubility. We demonstrate that, in C. elegans, A{beta} expression is sufficient to drive proteome-wide protein insolubility. The A{beta}-driven insoluble proteome bears a highly significant overlap with the aging-driven insoluble proteome, suggesting there exists a core, sub-proteome which is vulnerable to insolubility. Using human genome-wide association studies (GWAS) we show that this insoluble sub proteome is replete with biological processes implicated across not only neurodegenerative diseases but also across a broad array of chronic, age-related diseases (CARDs), providing suggestive evidence that age-related loss of proteostasis could play a role in general CARD risk. Finally, we show that the gut metabolite, Urolithin A, relieves A{beta} toxicity supporting its use in clinical trials for dementia and other age-related diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550689">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550689" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550689">
        <p class="paperTitle">Systematic exploration of prokaryotic form I rubisco maximal carboxylation rates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550689" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550689" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Pins, B.; Greenspoon, L.; Bar-On, Y. M.; Shamshoum, M.; Ben-Nissan, R.; Milshtein, E.; Davidi, D.; Sharon, I.; Mueller-Cajar, O.; Noor, E.; Milo, R.</p>
        <p class="info">Score: 2.5, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550689' target='https://doi.org/10.1101/2023.07.27.550689'> 10.1101/2023.07.27.550689</a></p>
        <p class="abstract">Autotrophy is the basis for complex life on Earth. Its physiological limits are largely constrained by rubisco - the enzyme that catalyzes almost all carbon fixation on the planet. Yet, with only a small fraction of rubisco diversity kinetically characterized so far, the underlying biological factors driving the evolution of fast rubiscos in nature remain unclear. We conducted a high-throughput kinetic characterization of over 100 prokaryotic form I rubiscos, the most ubiquitous group of rubisco sequences in nature, to uncover the determinants of rubiscos carboxylation velocity. We show that phototrophy and the presence of a carboxysome CO2 concentrating mechanism correlate with faster rubiscos (median rate difference of 3-5 fold). In contrast to previous assumptions, rubiscos originating from -cyanobacteria exhibit the highest carboxylation rates among form I enzymes ({approx}10 s-1 median versus &lt;7 s-1 in other groups). Our study ties biological and environmental insights associated with kinetic variation across rubiscos from nature.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
