<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 58.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 58.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Open-ST: High-resolution spatial transcriptomics in 3D
Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.
Score: 58.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554
Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Open-ST: High-resolution spatial transcriptomics in 3D\nAuthors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.\nScore: 58.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554\nSpatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods.",
  "keywords": [
    
  ],
  "articleBody": " Open-ST: High-resolution spatial transcriptomics in 3D\nAuthors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.\nScore: 58.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572554\nSpatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods. Here, we report Open-ST, a sequencing-based, open-source experimental and computational resource to address these challenges and to study the molecular organization of tissues in 3D. In mouse brain, Open-ST captured transcripts at subcellular resolution and reconstructed cell types. In primary tumor and patient-matched healthy/metastatic lymph nodes, Open-ST captured the diversity of immune, stromal and tumor populations in space. Distinct cell states were organized around cell-cell communication hotspots in the tumor, but not the metastasis. Strikingly, the 3D reconstruction and multimodal analysis of the metastatic lymph node revealed spatially contiguous structures not visible in 2D and potential biomarkers precisely at the 3D tumor/lymph node boundary. We anticipate Open-ST to accelerate the identification of spatial molecular mechanisms in 2D and 3D.\nOncogenic PIK3CA corrupts growth factor signaling specificity\nAuthors: Madsen, R. R.; Le Marois, A.; Mruk, O.; Voliotis, M.; Yin, S.; Sufi, J.; Qin, X.; Zhao, S. J.; Gorczynska, J.; Morelli, D.; Davidson, L.; Sahai, E.; Korolchuk, V. I.; Tape, C. J.; Vanhaesebroeck, B.\nScore: 21.4, Published: 2023-12-26 DOI: 10.1101/2023.12.23.573207\nPathological activation of the PI3K/AKT pathway is among the most frequent defects in human cancer and is also the cause of rare overgrowth disorders. Yet, there is currently no systematic understanding of the quantitative flow of information within PI3K/AKT signaling and how it is perturbed by disease-causing mutations. Here, we develop scalable, single-cell approaches for systematic analyses of signal processing within the PI3K pathway, enabling precise calculations of its information transfer for different growth factors. Using genetically-engineered human cell models with allele dose-dependent expression of PIK3CA H1047R , we show that this oncogene is not a simple, constitutive pathway activator but a context-dependent modulator of extracellular signal transfer. PIK3CA H1047R reduces information transmission downstream of IGF1 while selectively enhancing EGF-induced signaling and transcriptional responses. This leads to a gross reduction in signaling specificity, akin to \"blurred\" signal perception. The associated increase in signaling heterogeneity promotes phenotypic diversity in a human cervical cancer cell line model and in human induced pluripotent stem cells. Collectively, these findings and the accompanying methodological advances lay the foundations for a systematic mapping of the quantitative mechanisms of PI3K/AKT-dependent signal processing and phenotypic control in health and disease.\nDynamics of single-cell protein covariation during epithelial-mesenchymal transition\nAuthors: Khan, S.; Conover, R.; Asthagiri, A. R.; Slavov, N.\nScore: 19.1, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572913\nPhysiological processes, such as epithelial-mesenchymal transition (EMT), are mediated by changes in protein interactions. These changes may be better reflected in protein covariation within cellular cluster than in the temporal dynamics of cluster-average protein abundance. To explore this possibility, we quantified proteins in single human cells undergoing EMT. Covariation analysis of the data revealed that functionally coherent protein clusters dynamically changed their protein-protein correlations without concomitant changes in cluster-average protein abundance. These dynamics of protein-protein correlations were monotonic in time and delineated protein modules functioning in actin cytoskeleton organization, energy metabolism and protein transport. These protein modules are defined by protein covariation within the same time point and cluster and thus reflect biological regulation masked by the cluster-average protein dynamics. Thus, protein correlation dynamics across single cells offer a window into protein regulation during physiological transitions.\nHighly multiplexed, image-based pooled screens in primary cells and tissues with PerturbView\nAuthors: Kudo, T.; Meireles, A. M.; Moncada, R.; Chen, Y.; Wu, P.; Gould, J.; Hu, X.; Kornfeld, O.; Jesudason, R.; Foo, C.; Hockendorf, B.; Bravo, H. C.; Town, J. P.; Wei, R.; Rios, A.; Chandrasekar, V.; Heinlein, M.; Cai, S.; Lu, C. S.; Celen, C.; Kljavin, N.; Jiang, J.; Hleap, J. S.; Kayagaki, N.; Melo, F. d. S. e.; McGinnis, L.; Li, B.; Singh, A.; Garraway, L.; Rozenblatt-Rosen, O.; Regev, A.; Lubeck, E.\nScore: 3.2, Published: 2023-12-26 DOI: 10.1101/2023.12.26.573143\nOptical pooled screening (OPS) is a highly scalable method for linking image-based phenotypes with cellular perturbations. However, it has thus far been restricted to relatively low-plex phenotypic readouts in cancer cell lines in culture, due to limitations associated with in situ sequencing (ISS) of perturbation barcodes. Here, we developed PerturbView, an OPS technology that leverages in vitro transcription (IVT) to amplify barcodes prior to ISS, enabling screens with highly multiplexed phenotypic readouts across diverse systems, including primary cells and tissues. We demonstrate PerturbView in iPSC-derived neurons, primary immune cells, and tumor tissue sections from animal models. In a screen of immune signaling pathways in primary bone marrow-derived macrophages, PerturbView uncovered both known and novel regulators of NFKB signaling. Furthermore, we combined PerturbView with spatial transcriptomics in tissue sections from a mouse xenograft model, paving the way to in vivo screens with rich optical and transcriptomic phenotypes. PerturbView broadens the scope of OPS to a wide range of models and applications.\nKinAce: a web portal for exploring kinase-substrate interactions\nAuthors: Sekar, J. A. P.; Li, Y. C.; Schlessinger, A.; Pandey, G.\nScore: 3.0, Published: 2023-12-10 DOI: 10.1101/2023.12.08.570875\nInteractions between protein kinases and their substrates are critical for the modulation of complex signaling pathways. Currently, there is a large amount of information available about kinases and their substrates in disparate public databases. However, these data are difficult to interpret in the context of cellular systems, which can be facilitated by examining interactions among multiple proteins at once, such as the network of interactions that constitute a signaling pathway. We present KinAce, a user-friendly web portal that integrates and shares information about kinase-substrate interactions from multiple databases of post-translational modifications. KinAce enables the visual exploration of these interactions in systems contexts, such as pathways, domain families, and custom protein set inputs, in an interactive fashion. We expect KinAce to be useful as a knowledge discovery tool for kinase-substrate interactions, and the aggregated KinAce dataset to be useful for protein kinase studies and systems-level analyses. The portal is available at https://kinace.kinametrix.com/.\nMechanistic computational modeling of monospecific and bispecific antibodies targeting interleukin-6/8 receptors\nAuthors: Ray, C. M.; Yang, H.; Spangler, J. B.; Mac Gabhann, F.\nScore: 3.4, Published: 2023-12-19 DOI: 10.1101/2023.12.18.570445\nThe spread of cancer from organ to organ (metastasis) is responsible for the vast majority of cancer deaths; however, most current anti-cancer drugs are designed to arrest or reverse tumor growth without directly addressing disease spread. It was recently discovered that tumor cell-secreted interleukin-6 (IL-6) and interleukin-8 (IL-8) synergize to enhance cancer metastasis in a cell-density dependent manner, and blockade of the IL-6 and IL-8 receptors (IL-6R and IL-8R) with a novel bispecific antibody, BS1, significantly reduced metastatic burden in multiple preclinical mouse models of cancer. Bispecific antibodies (BsAbs), which combine two different antigen-binding sites into one molecule, are a promising modality for drug development due to their enhanced avidity and dual targeting effects. However, while BsAbs have tremendous therapeutic potential, elucidating the mechanisms underlying their binding and inhibition will be critical for maximizing the efficacy of new BsAb treatments. Here, we describe a quantitative, computational model of the BS1 BsAb, exhibiting how modeling multivalent binding provides key insights into antibody affinity and avidity effects and can guide therapeutic design. We present detailed simulations of the monovalent and bivalent binding interactions between different antibody constructs and the IL-6 and IL-8 receptors to establish how antibody properties and system conditions impact the formation of binary (antibody-receptor) and ternary (receptor-antibody-receptor) complexes. Model results demonstrate how the balance of these complex types drives receptor inhibition, providing important and generalizable predictions for effective therapeutic design.\nA distinctive evolution of alveolar T cell responses is associated with clinical outcomes in unvaccinated patients with SARS-CoV-2 pneumonia\nAuthors: Markov, N. S.; Ren, Z.; Senkow, K. J.; Grant, R. A.; Gao, C. A.; Malsin, E. S.; Sichizya, L.; Kihshen, H.; Helmin, K. A.; Jovisic, M.; Arnold, J. M.; Perez-Leonor, X. G.; Abdala-Valencia, H.; Swaminathan, S.; Nwaezeapu, J.; Kang, M.; Rasmussen, L.; Ozer, E. A.; Lorenzo-Redondo, R.; Hultquist, J. F.; Simons, L.; Rios-Guzman, E.; Misharin, A. V.; Wunderink, R. G.; Budinger, G. S.; Singer, B. D.; Morales-Nebreda, L.; The NU SCRIPT Investigators, Score: 8.1, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571479\nPathogen clearance and resolution of inflammation in patients with pneumonia require an effective local T cell response. Nevertheless, local T cell activation may drive lung injury, particularly during prolonged episodes of respiratory failure characteristic of severe SARS-CoV-2 pneumonia. While T cell responses in the peripheral blood are well described, the evolution of T cell phenotypes and molecular signatures in the distal lung of patients with severe pneumonia caused by SARS-CoV-2 or other pathogens is understudied. Accordingly, we serially obtained 432 bronchoalveolar lavage fluid samples from 273 patients with severe pneumonia and respiratory failure, including 74 unvaccinated patients with COVID-19, and performed flow cytometry, transcriptional, and T cell receptor profiling on sorted CD8+ and CD4+ T cell subsets. In patients with COVID-19 but not pneumonia secondary to other pathogens, we found that early and persistent enrichment in CD8+ and CD4+ T cell subsets correlated with survival to hospital discharge. Activation of interferon signaling pathways early after intubation for COVID-19 was associated with favorable outcomes, while activation of NF-{kappa}B-driven programs late in disease was associated with poor outcomes. Patients with SARS-CoV-2 pneumonia whose alveolar T cells preferentially targeted the Spike and Nucleocapsid proteins tended to experience more favorable outcomes than patients whose T cells predominantly targeted the ORF1ab polyprotein complex. These results suggest that in patients with severe SARS-CoV-2 pneumonia, alveolar T cell interferon responses targeting structural SARS-CoV-2 proteins characterize patients who recover, yet these responses progress to NF-{kappa}B activation against non-structural proteins in patients who go on to experience poor clinical outcomes.\nOptimization-Based Synthesis with Directed Cell Migration\nAuthors: Havenhill, E.; Ghosh, S.\nScore: 1.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573173\nCollective behavior of biological agents such as herds of organisms and cells is a fundamental feature in the systems biology and in the emergence of new phenomena in the biological environment. Collective cell migration under a physical or chemical cue is an example of this fundamental phenomenon where individual cell migration is driven by the collective behavior of the neighboring cells and vice versa. The goal of this research is to discover the mathematical rules of collective cell migration using experimental data and testing the predictive nature of the models in independent experimental data. Such insight is made possible in this work with the hybrid use of dynamic mode decomposition (DMD) and optimal control theory. Both single and multi-cellular systems are simulated, including obstacle courses, using this framework. The results of this work show how cells collectively behave during their migration and also, opens the possibility of designing robotic cells for possible therapeutic purpose where the cell trajectory can be controlled.\nMachine Learning and Metabolic Model Guided CRISPRi Reveals a Central Role for Phosphoglycerate Mutase in Chlamydia trachomatis Persistence\nAuthors: Chowdhury, N. B.; Pokorzynski, N. D.; Rucks, E. A.; Ouellette, S. P.; CARABEO, R.; Saha, R.\nScore: 3.6, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572198\nUpon nutrient starvation, Chlamydia trachomatis serovar L2 (CTL) shifts from its normal growth to a non-replicating form, termed persistence. It is unclear if persistence is an adaptive response or lack of it. To understand that transcriptomics data were collected for nutrient-sufficient and nutrient-starved CTL. Applying machine learning approaches on transcriptomics data revealed a global transcriptomic rewiring of CTL under stress conditions without having any global stress regulator. This indicated that CTLs stress response is due to lack of an adaptive response mechanism. To investigate the impact of this on CTL metabolism, we reconstructed a genome-scale metabolic model of CTL (iCTL278) and contextualized it with the collected transcriptomics data. Using the metabolic bottleneck analysis on contextualized iCTL278, we observed phosphoglycerate mutase (pgm) regulates the entry of CTL to the persistence. Later, pgm was found to have the highest thermodynamics driving force and lowest enzymatic cost. Furthermore, CRISPRi-driven knockdown of pgm and tryptophan starvation experiments revealed the importance of this gene in inducing persistence. Hence, this work, for the first time, introduced thermodynamics and enzyme-cost as tools to gain deeper understanding on CTL persistence.\nAdvancing Reef Monitoring Techniques through Exometabolomics: Quantification of Labile Dissolved Organic Metabolites on Coral Reefs\nAuthors: Garcia, B. M.; Becker, C. C.; Weber, L.; Swarr, G. J.; Kido Soule, M. C.; Apprill, A.; Kujawinski, E. B.\nScore: 1.8, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572630\nExtracellular chemical cues constitute much of the language of life among marine organisms from microbes to mammals. These chemical cues shape foraging and feeding strategies, symbiotic interactions, selection of mates and habitats, and the transfer of energy and nutrients within and among ecosystems. Changes in the chemical pool flux within these ecosystems serve as invisible signals of overall ecosystem health and, conversely, disruption to this finely tuned equilibrium. In coral reef systems, the full scope and magnitude of the chemicals involved in maintaining reef equilibria are largely unknown. In particular, processes involving small, polar molecules, which form the majority components of labile dissolved organic carbon (DOC), are often poorly captured using traditional techniques and therefore remain unconstrained. In this study, we employed a recently developed chemical derivatization method and mass spectrometry-based targeted exometabolomics to capture and quantify polar dissolved phase metabolites on five coral reefs in the U.S. Virgin Islands. We detected and quantified 45 polar exometabolites, and further demonstrate their variability across the investigated geographic landscape and contextualize these findings in terms of geographic and benthic cover differences. By comparing our results to previously published coral reef exometabolomes that did not employ chemical derivatization, we show the novel quantification of 23 metabolites which were previously undetected in the coral reef exometabolome, including compounds involved in central carbon metabolism (e.g., glutamate) and novel metabolites such as homoserine betaine. We highlight the immense potential of chemical derivatization-based exometabolomics for quantifying labile chemical cues on coral reefs and measuring molecular level responses to environmental stressors. Overall, improving our understanding of the composition and dynamics of reef exometabolites is vital for effective ecosystem monitoring and management strategies. SynopsisMinimal research exists on the polar exometabolites on coral reefs, an essential component of reef health and the majority component of labile DOC. This study reports 23 newly quantified polar metabolites on coral reefs and highlights significant differences in the exometabolome composition across geographically distinct reefs and bays.\n",
  "wordCount" : "2403",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572554">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572554" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572554">
        <p class="paperTitle">Open-ST: High-resolution spatial transcriptomics in 3D</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572554" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572554" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schott, M.; Leon-Perinan, D.; Splendiani, E.; Strenger, L.; Licha, J. R.; Pentimalli, T. M.; Schallenberg, S.; Alles, J.; Samut Tagliaferro, S.; Boltengagen, A.; Ehrig, S.; Abbiati, S.; Dommerich, S.; Pagani, M.; Ferretti, E.; Macino, G.; Karaiskos, N.; Rajewsky, N.</p>
        <p class="info">Score: 58.9, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572554' target='https://doi.org/10.1101/2023.12.22.572554'> 10.1101/2023.12.22.572554</a></p>
        <p class="abstract">Spatial transcriptomics (ST) methods have been developed to unlock molecular mechanisms underlying tissue development, homeostasis, or disease. However, there is a need for easy-to-use, high-resolution, cost-efficient, and 3D-scalable methods. Here, we report Open-ST, a sequencing-based, open-source experimental and computational resource to address these challenges and to study the molecular organization of tissues in 3D. In mouse brain, Open-ST captured transcripts at subcellular resolution and reconstructed cell types. In primary tumor and patient-matched healthy/metastatic lymph nodes, Open-ST captured the diversity of immune, stromal and tumor populations in space. Distinct cell states were organized around cell-cell communication hotspots in the tumor, but not the metastasis. Strikingly, the 3D reconstruction and multimodal analysis of the metastatic lymph node revealed spatially contiguous structures not visible in 2D and potential biomarkers precisely at the 3D tumor/lymph node boundary. We anticipate Open-ST to accelerate the identification of spatial molecular mechanisms in 2D and 3D.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573207">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573207" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573207">
        <p class="paperTitle">Oncogenic PIK3CA corrupts growth factor signaling specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573207" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573207" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Madsen, R. R.; Le Marois, A.; Mruk, O.; Voliotis, M.; Yin, S.; Sufi, J.; Qin, X.; Zhao, S. J.; Gorczynska, J.; Morelli, D.; Davidson, L.; Sahai, E.; Korolchuk, V. I.; Tape, C. J.; Vanhaesebroeck, B.</p>
        <p class="info">Score: 21.4, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573207' target='https://doi.org/10.1101/2023.12.23.573207'> 10.1101/2023.12.23.573207</a></p>
        <p class="abstract">Pathological activation of the PI3K/AKT pathway is among the most frequent defects in human cancer and is also the cause of rare overgrowth disorders. Yet, there is currently no systematic understanding of the quantitative flow of information within PI3K/AKT signaling and how it is perturbed by disease-causing mutations. Here, we develop scalable, single-cell approaches for systematic analyses of signal processing within the PI3K pathway, enabling precise calculations of its information transfer for different growth factors. Using genetically-engineered human cell models with allele dose-dependent expression of PIK3CA  H1047R , we show that this oncogene is not a simple, constitutive pathway activator but a context-dependent modulator of extracellular signal transfer. PIK3CA  H1047R  reduces information transmission downstream of IGF1 while selectively enhancing EGF-induced signaling and transcriptional responses. This leads to a gross reduction in signaling specificity, akin to &#34;blurred&#34; signal perception. The associated increase in signaling heterogeneity promotes phenotypic diversity in a human cervical cancer cell line model and in human induced pluripotent stem cells. Collectively, these findings and the accompanying methodological advances lay the foundations for a systematic mapping of the quantitative mechanisms of PI3K/AKT-dependent signal processing and phenotypic control in health and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572913">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572913" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572913">
        <p class="paperTitle">Dynamics of single-cell protein covariation during epithelial-mesenchymal transition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572913" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572913" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khan, S.; Conover, R.; Asthagiri, A. R.; Slavov, N.</p>
        <p class="info">Score: 19.1, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572913' target='https://doi.org/10.1101/2023.12.21.572913'> 10.1101/2023.12.21.572913</a></p>
        <p class="abstract">Physiological processes, such as epithelial-mesenchymal transition (EMT), are mediated by changes in protein interactions. These changes may be better reflected in protein covariation within cellular cluster than in the temporal dynamics of cluster-average protein abundance. To explore this possibility, we quantified proteins in single human cells undergoing EMT. Covariation analysis of the data revealed that functionally coherent protein clusters dynamically changed their protein-protein correlations without concomitant changes in cluster-average protein abundance. These dynamics of protein-protein correlations were monotonic in time and delineated protein modules functioning in actin cytoskeleton organization, energy metabolism and protein transport. These protein modules are defined by protein covariation within the same time point and cluster and thus reflect biological regulation masked by the cluster-average protein dynamics. Thus, protein correlation dynamics across single cells offer a window into protein regulation during physiological transitions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573143">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573143" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573143">
        <p class="paperTitle">Highly multiplexed, image-based pooled screens in primary cells and tissues with PerturbView</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573143" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573143" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kudo, T.; Meireles, A. M.; Moncada, R.; Chen, Y.; Wu, P.; Gould, J.; Hu, X.; Kornfeld, O.; Jesudason, R.; Foo, C.; Hockendorf, B.; Bravo, H. C.; Town, J. P.; Wei, R.; Rios, A.; Chandrasekar, V.; Heinlein, M.; Cai, S.; Lu, C. S.; Celen, C.; Kljavin, N.; Jiang, J.; Hleap, J. S.; Kayagaki, N.; Melo, F. d. S. e.; McGinnis, L.; Li, B.; Singh, A.; Garraway, L.; Rozenblatt-Rosen, O.; Regev, A.; Lubeck, E.</p>
        <p class="info">Score: 3.2, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573143' target='https://doi.org/10.1101/2023.12.26.573143'> 10.1101/2023.12.26.573143</a></p>
        <p class="abstract">Optical pooled screening (OPS) is a highly scalable method for linking image-based phenotypes with cellular perturbations. However, it has thus far been restricted to relatively low-plex phenotypic readouts in cancer cell lines in culture, due to limitations associated with in situ sequencing (ISS) of perturbation barcodes. Here, we developed PerturbView, an OPS technology that leverages in vitro transcription (IVT) to amplify barcodes prior to ISS, enabling screens with highly multiplexed phenotypic readouts across diverse systems, including primary cells and tissues. We demonstrate PerturbView in iPSC-derived neurons, primary immune cells, and tumor tissue sections from animal models. In a screen of immune signaling pathways in primary bone marrow-derived macrophages, PerturbView uncovered both known and novel regulators of NFKB signaling. Furthermore, we combined PerturbView with spatial transcriptomics in tissue sections from a mouse xenograft model, paving the way to in vivo screens with rich optical and transcriptomic phenotypes. PerturbView broadens the scope of OPS to a wide range of models and applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570875">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570875" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570875">
        <p class="paperTitle">KinAce: a web portal for exploring kinase-substrate interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570875" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570875" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sekar, J. A. P.; Li, Y. C.; Schlessinger, A.; Pandey, G.</p>
        <p class="info">Score: 3.0, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570875' target='https://doi.org/10.1101/2023.12.08.570875'> 10.1101/2023.12.08.570875</a></p>
        <p class="abstract">Interactions between protein kinases and their substrates are critical for the modulation of complex signaling pathways. Currently, there is a large amount of information available about kinases and their substrates in disparate public databases. However, these data are difficult to interpret in the context of cellular systems, which can be facilitated by examining interactions among multiple proteins at once, such as the network of interactions that constitute a signaling pathway. We present KinAce, a user-friendly web portal that integrates and shares information about kinase-substrate interactions from multiple databases of post-translational modifications. KinAce enables the visual exploration of these interactions in systems contexts, such as pathways, domain families, and custom protein set inputs, in an interactive fashion. We expect KinAce to be useful as a knowledge discovery tool for kinase-substrate interactions, and the aggregated KinAce dataset to be useful for protein kinase studies and systems-level analyses. The portal is available at https://kinace.kinametrix.com/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.570445">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.570445" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.570445">
        <p class="paperTitle">Mechanistic computational modeling of monospecific and bispecific antibodies targeting interleukin-6/8 receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.570445" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.570445" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ray, C. M.; Yang, H.; Spangler, J. B.; Mac Gabhann, F.</p>
        <p class="info">Score: 3.4, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.570445' target='https://doi.org/10.1101/2023.12.18.570445'> 10.1101/2023.12.18.570445</a></p>
        <p class="abstract">The spread of cancer from organ to organ (metastasis) is responsible for the vast majority of cancer deaths; however, most current anti-cancer drugs are designed to arrest or reverse tumor growth without directly addressing disease spread. It was recently discovered that tumor cell-secreted interleukin-6 (IL-6) and interleukin-8 (IL-8) synergize to enhance cancer metastasis in a cell-density dependent manner, and blockade of the IL-6 and IL-8 receptors (IL-6R and IL-8R) with a novel bispecific antibody, BS1, significantly reduced metastatic burden in multiple preclinical mouse models of cancer. Bispecific antibodies (BsAbs), which combine two different antigen-binding sites into one molecule, are a promising modality for drug development due to their enhanced avidity and dual targeting effects. However, while BsAbs have tremendous therapeutic potential, elucidating the mechanisms underlying their binding and inhibition will be critical for maximizing the efficacy of new BsAb treatments. Here, we describe a quantitative, computational model of the BS1 BsAb, exhibiting how modeling multivalent binding provides key insights into antibody affinity and avidity effects and can guide therapeutic design. We present detailed simulations of the monovalent and bivalent binding interactions between different antibody constructs and the IL-6 and IL-8 receptors to establish how antibody properties and system conditions impact the formation of binary (antibody-receptor) and ternary (receptor-antibody-receptor) complexes. Model results demonstrate how the balance of these complex types drives receptor inhibition, providing important and generalizable predictions for effective therapeutic design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571479">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571479" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571479">
        <p class="paperTitle">A distinctive evolution of alveolar T cell responses is associated with clinical outcomes in unvaccinated patients with SARS-CoV-2 pneumonia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571479" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571479" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Markov, N. S.; Ren, Z.; Senkow, K. J.; Grant, R. A.; Gao, C. A.; Malsin, E. S.; Sichizya, L.; Kihshen, H.; Helmin, K. A.; Jovisic, M.; Arnold, J. M.; Perez-Leonor, X. G.; Abdala-Valencia, H.; Swaminathan, S.; Nwaezeapu, J.; Kang, M.; Rasmussen, L.; Ozer, E. A.; Lorenzo-Redondo, R.; Hultquist, J. F.; Simons, L.; Rios-Guzman, E.; Misharin, A. V.; Wunderink, R. G.; Budinger, G. S.; Singer, B. D.; Morales-Nebreda, L.; The NU SCRIPT Investigators,  </p>
        <p class="info">Score: 8.1, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571479' target='https://doi.org/10.1101/2023.12.13.571479'> 10.1101/2023.12.13.571479</a></p>
        <p class="abstract">Pathogen clearance and resolution of inflammation in patients with pneumonia require an effective local T cell response. Nevertheless, local T cell activation may drive lung injury, particularly during prolonged episodes of respiratory failure characteristic of severe SARS-CoV-2 pneumonia. While T cell responses in the peripheral blood are well described, the evolution of T cell phenotypes and molecular signatures in the distal lung of patients with severe pneumonia caused by SARS-CoV-2 or other pathogens is understudied. Accordingly, we serially obtained 432 bronchoalveolar lavage fluid samples from 273 patients with severe pneumonia and respiratory failure, including 74 unvaccinated patients with COVID-19, and performed flow cytometry, transcriptional, and T cell receptor profiling on sorted CD8&#43; and CD4&#43; T cell subsets. In patients with COVID-19 but not pneumonia secondary to other pathogens, we found that early and persistent enrichment in CD8&#43; and CD4&#43; T cell subsets correlated with survival to hospital discharge. Activation of interferon signaling pathways early after intubation for COVID-19 was associated with favorable outcomes, while activation of NF-{kappa}B-driven programs late in disease was associated with poor outcomes. Patients with SARS-CoV-2 pneumonia whose alveolar T cells preferentially targeted the Spike and Nucleocapsid proteins tended to experience more favorable outcomes than patients whose T cells predominantly targeted the ORF1ab polyprotein complex. These results suggest that in patients with severe SARS-CoV-2 pneumonia, alveolar T cell interferon responses targeting structural SARS-CoV-2 proteins characterize patients who recover, yet these responses progress to NF-{kappa}B activation against non-structural proteins in patients who go on to experience poor clinical outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573173">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573173" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573173">
        <p class="paperTitle">Optimization-Based Synthesis with Directed Cell Migration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573173" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573173" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Havenhill, E.; Ghosh, S.</p>
        <p class="info">Score: 1.9, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573173' target='https://doi.org/10.1101/2023.12.22.573173'> 10.1101/2023.12.22.573173</a></p>
        <p class="abstract">Collective behavior of biological agents such as herds of organisms and cells is a fundamental feature in the systems biology and in the emergence of new phenomena in the biological environment. Collective cell migration under a physical or chemical cue is an example of this fundamental phenomenon where individual cell migration is driven by the collective behavior of the neighboring cells and vice versa. The goal of this research is to discover the mathematical rules of collective cell migration using experimental data and testing the predictive nature of the models in independent experimental data. Such insight is made possible in this work with the hybrid use of dynamic mode decomposition (DMD) and optimal control theory. Both single and multi-cellular systems are simulated, including obstacle courses, using this framework. The results of this work show how cells collectively behave during their migration and also, opens the possibility of designing robotic cells for possible therapeutic purpose where the cell trajectory can be controlled.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572198">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572198" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572198">
        <p class="paperTitle">Machine Learning and Metabolic Model Guided CRISPRi Reveals a Central Role for Phosphoglycerate Mutase in Chlamydia trachomatis Persistence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572198" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572198" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chowdhury, N. B.; Pokorzynski, N. D.; Rucks, E. A.; Ouellette, S. P.; CARABEO, R.; Saha, R.</p>
        <p class="info">Score: 3.6, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572198' target='https://doi.org/10.1101/2023.12.18.572198'> 10.1101/2023.12.18.572198</a></p>
        <p class="abstract">Upon nutrient starvation, Chlamydia trachomatis serovar L2 (CTL) shifts from its normal growth to a non-replicating form, termed persistence. It is unclear if persistence is an adaptive response or lack of it. To understand that transcriptomics data were collected for nutrient-sufficient and nutrient-starved CTL. Applying machine learning approaches on transcriptomics data revealed a global transcriptomic rewiring of CTL under stress conditions without having any global stress regulator. This indicated that CTLs stress response is due to lack of an adaptive response mechanism. To investigate the impact of this on CTL metabolism, we reconstructed a genome-scale metabolic model of CTL (iCTL278) and contextualized it with the collected transcriptomics data. Using the metabolic bottleneck analysis on contextualized iCTL278, we observed phosphoglycerate mutase (pgm) regulates the entry of CTL to the persistence. Later, pgm was found to have the highest thermodynamics driving force and lowest enzymatic cost. Furthermore, CRISPRi-driven knockdown of pgm and tryptophan starvation experiments revealed the importance of this gene in inducing persistence. Hence, this work, for the first time, introduced thermodynamics and enzyme-cost as tools to gain deeper understanding on CTL persistence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572630">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572630" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572630">
        <p class="paperTitle">Advancing Reef Monitoring Techniques through Exometabolomics: Quantification of Labile Dissolved Organic Metabolites on Coral Reefs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572630" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572630" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garcia, B. M.; Becker, C. C.; Weber, L.; Swarr, G. J.; Kido Soule, M. C.; Apprill, A.; Kujawinski, E. B.</p>
        <p class="info">Score: 1.8, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572630' target='https://doi.org/10.1101/2023.12.20.572630'> 10.1101/2023.12.20.572630</a></p>
        <p class="abstract">Extracellular chemical cues constitute much of the language of life among marine organisms from microbes to mammals. These chemical cues shape foraging and feeding strategies, symbiotic interactions, selection of mates and habitats, and the transfer of energy and nutrients within and among ecosystems. Changes in the chemical pool flux within these ecosystems serve as invisible signals of overall ecosystem health and, conversely, disruption to this finely tuned equilibrium. In coral reef systems, the full scope and magnitude of the chemicals involved in maintaining reef equilibria are largely unknown. In particular, processes involving small, polar molecules, which form the majority components of labile dissolved organic carbon (DOC), are often poorly captured using traditional techniques and therefore remain unconstrained. In this study, we employed a recently developed chemical derivatization method and mass spectrometry-based targeted exometabolomics to capture and quantify polar dissolved phase metabolites on five coral reefs in the U.S. Virgin Islands. We detected and quantified 45 polar exometabolites, and further demonstrate their variability across the investigated geographic landscape and contextualize these findings in terms of geographic and benthic cover differences. By comparing our results to previously published coral reef exometabolomes that did not employ chemical derivatization, we show the novel quantification of 23 metabolites which were previously undetected in the coral reef exometabolome, including compounds involved in central carbon metabolism (e.g., glutamate) and novel metabolites such as homoserine betaine. We highlight the immense potential of chemical derivatization-based exometabolomics for quantifying labile chemical cues on coral reefs and measuring molecular level responses to environmental stressors. Overall, improving our understanding of the composition and dynamics of reef exometabolites is vital for effective ecosystem monitoring and management strategies.

SynopsisMinimal research exists on the polar exometabolites on coral reefs, an essential component of reef health and the majority component of labile DOC. This study reports 23 newly quantified polar metabolites on coral reefs and highlights significant differences in the exometabolome composition across geographically distinct reefs and bays.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
