<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Correlation of myeloid-derived suppressor cell expansion with upregulated transposable elements in severe COVID-19 unveiled in single-cell RNA sequencing reanalysis
Authors: Farahmandnejad, M.; Mosadeghi, P.; Dorvash, M.; Sakhteman, A.; Faridi, P.; Negahdaripour, M.
Score: 19.2, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556192
Some studies investigated the potential role of transposable elements (TEs) in COVID-19 pathogenesis and complications. However, to the best of our knowledge, there is no study to examine the possible association of TEs expression in cell functions and its potential role in COVID-19 immune response at the single-cell level.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Correlation of myeloid-derived suppressor cell expansion with upregulated transposable elements in severe COVID-19 unveiled in single-cell RNA sequencing reanalysis
Authors: Farahmandnejad, M.; Mosadeghi, P.; Dorvash, M.; Sakhteman, A.; Faridi, P.; Negahdaripour, M.
Score: 19.2, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556192
Some studies investigated the potential role of transposable elements (TEs) in COVID-19 pathogenesis and complications. However, to the best of our knowledge, there is no study to examine the possible association of TEs expression in cell functions and its potential role in COVID-19 immune response at the single-cell level." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-13T10:39:03+00:00" />
<meta property="article:modified_time" content="2023-09-13T10:39:03+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Correlation of myeloid-derived suppressor cell expansion with upregulated transposable elements in severe COVID-19 unveiled in single-cell RNA sequencing reanalysis
Authors: Farahmandnejad, M.; Mosadeghi, P.; Dorvash, M.; Sakhteman, A.; Faridi, P.; Negahdaripour, M.
Score: 19.2, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556192
Some studies investigated the potential role of transposable elements (TEs) in COVID-19 pathogenesis and complications. However, to the best of our knowledge, there is no study to examine the possible association of TEs expression in cell functions and its potential role in COVID-19 immune response at the single-cell level."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Correlation of myeloid-derived suppressor cell expansion with upregulated transposable elements in severe COVID-19 unveiled in single-cell RNA sequencing reanalysis\nAuthors: Farahmandnejad, M.; Mosadeghi, P.; Dorvash, M.; Sakhteman, A.; Faridi, P.; Negahdaripour, M.\nScore: 19.2, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556192\nSome studies investigated the potential role of transposable elements (TEs) in COVID-19 pathogenesis and complications. However, to the best of our knowledge, there is no study to examine the possible association of TEs expression in cell functions and its potential role in COVID-19 immune response at the single-cell level.",
  "keywords": [
    
  ],
  "articleBody": " Correlation of myeloid-derived suppressor cell expansion with upregulated transposable elements in severe COVID-19 unveiled in single-cell RNA sequencing reanalysis\nAuthors: Farahmandnejad, M.; Mosadeghi, P.; Dorvash, M.; Sakhteman, A.; Faridi, P.; Negahdaripour, M.\nScore: 19.2, Published: 2023-09-05 DOI: 10.1101/2023.09.04.556192\nSome studies investigated the potential role of transposable elements (TEs) in COVID-19 pathogenesis and complications. However, to the best of our knowledge, there is no study to examine the possible association of TEs expression in cell functions and its potential role in COVID-19 immune response at the single-cell level. In this study, we reanalyzed single-cell RNA seq data of bronchoalveolar lavage (BAL) samples obtained from six severe COVID-19 patients and three healthy donors to assess the probable correlation of TE expression with the immune responses induced by the SARS-CoV-2 virus in COVID-19 patients. Our findings indicated that the expansion of myeloid-derived suppressor cells (MDSCs) may be a characteristic feature of COVID-19. Additionally, a significant increase in TEs expression in MDSCs was observed. This upregulation of TEs in COVID-19 may be linked to the adaptability of these cells in response to their microenvironments. Furthermore, it appears that the identification of overexpressed TEs by Pattern recognition receptors (PRRs) in MDSCs may enhance the suppressive capacity of these cells. Thus, this study emphasizes the crucial role of TEs in the functionality of MDSCs during COVID-19.\nEfficient gene knockout and genetic interactions: the IN4MER CRISPR/Cas12a multiplex knockout platform\nAuthors: Esmaeili Anvar, N.; Lin, C.; Ma, X.; Wilson, L. L.; Steger, R.; Sangree, A. K.; Colic, M.; Wang, S. H.; Doench, J. G.; Hart, T.\nScore: 32.1, Published: 2023-09-05 DOI: 10.1101/2023.01.03.522655\nGenetic interactions mediate the emergence of phenotype from genotype, but initial technologies for combinatorial genetic perturbation in mammalian cells suffer from inefficiency and are challenging to scale. Recent focus on paralog synthetic lethality in cancer cells offers an opportunity to evaluate different approaches and improve on the state of the art. Here we report a meta-analysis of CRISPR genetic interactions screens, identifying a candidate set of background-independent paralog synthetic lethals, and find that the Cas12a platform provides superior sensitivity and assay replicability. We demonstrate that Cas12a can independently target up to four genes from a single guide array, and we build on this knowledge by constructing a genome-scale library that expresses arrays of four guides per clone, a platform we call in4mer. Our genome-scale human library, with only 49k clones, is substantially smaller than a typical CRISPR/Cas9 monogenic library while also targeting more than four thousand paralog pairs, triples, and quads. Proof of concept screens in four cell lines demonstrate discrimination of core and context-dependent essential genes similar to that of state-of-the-art CRISPR/Cas9 libraries, as well as detection of synthetic lethal and masking/buffering genetic interactions between paralogs of various family sizes, a capability not offered by any extant library. Importantly, the in4mer platform offers a fivefold reduction in the number of clones required to assay genetic interactions, dramatically improving the cost and effort required for these studies.\nA hierarchy of metabolite exchanges in metabolic models of microbial species and communities\nAuthors: Wedmark, Y. K.; Vik, J. O.; Oyas, O.\nScore: 7.6, Published: 2023-09-07 DOI: 10.1101/2023.09.05.556413\nMotivationThe metabolic network of an organism can be analyzed as a constraint-based model (CBM). CBM analyses can be biased, optimizing an objective such as growth rate, or unbiased, attempting to describe the feasible flux space through pathway analysis or random flux sampling. Biased methods scale to genome-scale models (GEMs) and beyond, but unbiased methods suffer from combinatorial explosion. One promising approach to improve the scalability of unbiased analyses is to focus on cells metabolite exchanges with each other and the environment rather than the internal details of their metabolic networks. However, methods for unbiased analysis of metabolite exchanges have not been evaluated or compared. ResultsUsing CBMs ranging in size from core models to GEMs, we applied pathway enumeration and random flux sampling to metabolite exchanges in microbial species and a microbial community. We focused on pathway definitions that allow direct targeting of metabolite exchanges: elementary conversion modes (ECMs), elementary flux patterns (EFPs), and minimal pathways (MPs). Enumerating growth-supporting ECMs, EFPs, and MPs for the same models, we found that the MPs were always a subset of the ECMs and EFPs, specifically the subset of support-minimal pathways. The EFPs were always a subset of the ECMs, which in turn were always a subset of flux patterns from elementary flux modes (EFMs). We argue that this hierarchical relationship between pathway definitions holds more generally and that it helps explain the relative scalability of ECM, EFP, and MP enumeration. Metabolite exchange frequencies in enumerated pathways were similar across methods with a few specific exchanges explaining large differences in pathway counts. Thus, biological interpretation was largely robust to the choice of pathway definition. Random flux sampling scaled to GEMs and complemented pathway enumeration. AvailabilityData and code are available at https://gitlab.com/YlvaKaW/exchange-enumeration.\nLow dimensionality of phenotypic space as an emergent property of coordinated teams in biological regulatory networks\nAuthors: Hari, K.; Harlapur, P.; Saxena, A.; Haldar, K.; Girish, A.; Malpani, T.; Levine, H.; Jolly, M. K.\nScore: 8.3, Published: 2023-09-06 DOI: 10.1101/2023.02.03.526930\nBiological networks driving cell-fate decisions involve complex interactions, but they often give rise to only a few phenotypes, thus exhibiting low-dimensional dynamics. The network design principles that govern such cell-fate canalization remain unclear. Here, we investigate networks across diverse biological contexts- Epithelial-Mesenchymal Transition, Small Cell Lung Cancer, and Gonadal cell-fate determination - to reveal that the presence of two mutually antagonistic, well-coordinated teams of nodes leads to low-dimensional phenotypic space such that the first principal component (PC1) axis can capture most of the variance. Further analysis of artificial team-based networks and random counterparts of biological networks reveals that the principal component decomposition is determined by the team strength within these networks, demonstrating how the underlying network structure governs PC1 variance. The presence of low dimensionality in corresponding transcriptomic data confirms the applicability of our observations. We propose that team-based topology in biological networks are critical for generating a cell-fate canalization landscape.\nLongevity interventions in Titan mice attenuate frailty and senescence accumulation\nAuthors: Gille, B.; Muller-Eigner, A.; Gottschalk, S.; Wytrwat, E.; Langhammer, M.; Peleg, S.\nScore: 5.7, Published: 2023-09-12 DOI: 10.1101/2023.09.08.556694\nWild-type murine models for aging research have lifespans of several years, which results in long experimental duration and late output. Here we explore the short-lived non-inbred Titan mouse as a mouse model to test longevity interventions. We show that Titan mice exhibit increased frailty and cellular senescence at an early age. Dietary intervention attenuates the frailty progression of Titan mice. Additionally, cyclic administration of the senolytic drug Navitoclax at early age increases the lifespan and reduces cellular senescence. Our data suggests that Titan mice can serve as a cost-effective and timely model for longevity interventions in mammals.\nEscalating High-dimensional Imaging using Combinatorial Channel Multiplexing and Deep Learning\nAuthors: Ben-Uri, R.; Ben Shabat, L.; Bar-Tal, O.; Bussi, Y.; Maimon, N.; Keidar Haran, T.; Milo, I.; Elhanani, O.; Rochwarger, A.; Schürch, C. M.; Bagon, S.; Keren, L.\nScore: 5.6, Published: 2023-09-12 DOI: 10.1101/2023.09.09.556962\nUnderstanding tissue structure and function requires tools that quantify the expression of multiple proteins at single-cell resolution while preserving spatial information. Current imaging technologies use a separate channel for each individual protein, inherently limiting their throughput and scalability. Here, we present CombPlex (COMBinatorial multiPLEXing), a combinatorial staining platform coupled with an algorithmic framework to exponentially increase the number of proteins that can be measured from C up to 2C - 1, and is applicable to any mass spectrometry-based or fluorescence-based microscopy platform. In CombPlex, every protein can be imaged in several channels, and every channel contains agglomerated images of several proteins. These combinatorically-compressed images are then decompressed to individual protein-images using deep learning and optimization. We perform feasibility experiments in silico and achieve accurate (F1=0.98, R=0.99) reconstruction for compressing the stains of twenty-two proteins to five imaging channels. We test our approach experimentally and obtain accurate(F1=0.97, R=0.93) images of seven proteins using three channels, both in fluorescence microscopy and in mass-based imaging. We demonstrate that combinatorial staining coupled with deep-learning decompression can serve to escalate the number of proteins measured using any imaging modality, without the need for specialized instrumentation. Coupling CombPlex with instruments for high dimensional imaging could pave the way to image hundreds of proteins at single-cell resolution in intact tissue sections.\nA dynamic Boolean model of molecular and cellular interactions represents psoriasis development and predicts drug candidates.\nAuthors: Tsirvouli, E.; Noël, V.; Flobak, A.; Calzone, L.; Kuiper, M.\nScore: 5.0, Published: 2023-09-06 DOI: 10.1101/2023.09.03.556147\nPsoriasis is a chronic skin disease affecting 2-3% of the global population. Psoriasis arises from complex interactions between keratinocytes and immune cells, leading to uncontrolled inflammation, immune hyperactivation and perturbed keratinocyte life cycle. Although the latest generation of drugs have greatly improved psoriasis management, the disease remains incurable, and the substantial variability in treatment response calls for novel approaches to comprehend the intricate mechanisms underlying disease development and to discover potential drug targets. In this study, we present a multiscale population model that captures the dynamics of cell-specific phenotypes in psoriasis, integrating discrete logical formalism and population dynamics simulations. Through simulations and network metrics, we identify potential pairwise interventions as alternative treatment options. Specifically, The model predictions suggest that targeting neutrophil activation in conjunction with either PGE2 production or STAT3 signaling shows promise comparable to IL-17 inhibition, which is currently the most used treatment option for moderate and severe cases of psoriasis. Our findings underscore the significance of considering complex intercellular interactions and intracellular signaling cascades in psoriasis, and highlight the importance of computational approaches in unraveling complex biological systems for drug target identification. Author summaryIn our study, we aimed to uncover the complex mechanisms underlying psoriasis and identify potential treatment options. By utilizing a computational model, we simulated the dynamic interactions between different cell types involved in psoriasis, such as immune cells and keratinocytes. Our model predicts that targeting neutrophil activation, combined with either PGE2 production or STAT3 signaling, may yield comparable effectiveness to the current standard treatment for moderate or severe psoriasis, namely IL-17 inhibition. Our study underscores the importance of computational modeling in unraveling the complexities of disease systems and provides a foundation for identifying new candidate treatment options in psoriasis that should be tested in the lab.\nExploiting fluctuations in gene expression to detect causal interactions between genes\nAuthors: Joly-Smith, E.; Talpur, M. M.; Allard, P.; Papazotos, F.; Potvin-Trottier, L.; Hilfinger, A.\nScore: 11.2, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555799\nCharacterizing and manipulating cellular behaviour requires a mechanistic understanding of the causal interactions between cellular components. We present an approach that can detect causal interactions between genes without the need to perturb the physiological state of cells. This approach exploits naturally occurring cell-to-cell variability which is experimentally accessible from static population snapshots of genetically identical cells without the need to follow cells over time. Our main contribution is a simple mathematical relation that constrains the propagation of gene expression noise through biochemical reaction networks. This relation allows us to rigorously interpret fluctuation data even when only a small part of a complex gene regulatory process can be observed. This relation can be exploited to detect causal interactions by synthetically engineering a passive reporter of gene expression, akin to the established \"dual reporter assay\". While the focus of our contribution is theoretical, we also present an experimental proof-of-principle to illustrate the approach. Our data from synthetic gene regulatory networks in E. coli are not unequivocal but suggest that the method could prove useful in practice to identify causal interactions between genes from non-genetic cell-to-cell variability.\nApproximate Bayesian Computation for Inferring Waddington Landscapes from Single Cell Data\nAuthors: Liu, Y.; Zhang, S. Y.; Kleijn, I. T.; Stumpf, M. P. H.\nScore: 8.7, Published: 2023-09-05 DOI: 10.1101/2023.09.03.556134\nSingle cell technologies allow us to gain insights into cellular processes at unprecedented resolution. In stem cell and developmental biology snapshot data allows us to characterise how the transcriptional state of cells changes between successive cell types. Here we show how approximate Bayesian computation (ABC) can be employed to calibrate mathematical models against single cell data. In our simulation study we demonstrate the pivotal role of the adequate choice of distance measures appropriate for single cell data. We show that for good distance measures, notably optimal transport distances, we can infer parameters for mathematical models from simulated single cell data. We show that the ABC posteriors can be used to characterise parameter sensitivity and identify dependencies between different parameters, and to infer representations of the Waddington or epigenetic landscape, which forms a popular and interpretable representation of the developmental dynamics. In summary, these results pave the way for fitting mechanistic models of stem cell differentiation to single cell data.\nAgent-based modeling of tumor-immune interactions reveals determinants of final tumor states\nAuthors: Ahmidouch, M.; Tangella, N.; Finley, S. D.\nScore: 1.2, Published: 2023-09-07 DOI: 10.1101/2023.09.06.556617\nInteractions between tumor and immune cells in the tumor microenvironment (TME) influence tumor growth and the tumors response to treatment. Excitingly, this complex landscape of tumor-immune interactions can be studied using computational modeling. Mathematical oncology can provide quantitative insights into the TME, serving as a framework for understanding tumor dynamics. Here, we use an agent-based model to simulate the interactions among cancer cells, macrophages (naive, M1, and M2), and T cells (active CD8+ and inactive) in a 2D representation of the TME. Key diffusible factors, IL-4 and IFN-{gamma}, are also incorporated. We apply the model to predict how cell-specific properties influence tumor progression. The model predictions and analyses revealed the relationships between different cell populations and highlighted the importance of macrophages and T cells in shaping the TME. Thus, we quantify how components of the TME influence the final tumor state and the effects of macrophage-based therapies. The findings emphasize the significant role of computational models in unraveling the intricate dynamics of tumor-immune interactions and their potential for guiding the development of tailored immunotherapeutic strategies. This study provides a foundation for future investigations aiming to refine and expand the model, validate predictions experimentally, and pave the way for improved cancer treatments.\n",
  "wordCount" : "2334",
  "inLanguage": "en",
  "datePublished": "2023-09-13T10:39:03Z",
  "dateModified": "2023-09-13T10:39:03Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on September 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556192">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556192" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556192">
        <p class="paperTitle">Correlation of myeloid-derived suppressor cell expansion with upregulated transposable elements in severe COVID-19 unveiled in single-cell RNA sequencing reanalysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556192" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556192" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Farahmandnejad, M.; Mosadeghi, P.; Dorvash, M.; Sakhteman, A.; Faridi, P.; Negahdaripour, M.</p>
        <p class="info">Score: 19.2, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556192' target='https://doi.org/10.1101/2023.09.04.556192'> 10.1101/2023.09.04.556192</a></p>
        <p class="abstract">Some studies investigated the potential role of transposable elements (TEs) in COVID-19 pathogenesis and complications. However, to the best of our knowledge, there is no study to examine the possible association of TEs expression in cell functions and its potential role in COVID-19 immune response at the single-cell level.

In this study, we reanalyzed single-cell RNA seq data of bronchoalveolar lavage (BAL) samples obtained from six severe COVID-19 patients and three healthy donors to assess the probable correlation of TE expression with the immune responses induced by the SARS-CoV-2 virus in COVID-19 patients.

Our findings indicated that the expansion of myeloid-derived suppressor cells (MDSCs) may be a characteristic feature of COVID-19. Additionally, a significant increase in TEs expression in MDSCs was observed. This upregulation of TEs in COVID-19 may be linked to the adaptability of these cells in response to their microenvironments. Furthermore, it appears that the identification of overexpressed TEs by Pattern recognition receptors (PRRs) in MDSCs may enhance the suppressive capacity of these cells. Thus, this study emphasizes the crucial role of TEs in the functionality of MDSCs during COVID-19.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.03.522655">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.03.522655" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.03.522655">
        <p class="paperTitle">Efficient gene knockout and genetic interactions: the IN4MER CRISPR/Cas12a multiplex knockout platform</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.03.522655" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.03.522655" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Esmaeili Anvar, N.; Lin, C.; Ma, X.; Wilson, L. L.; Steger, R.; Sangree, A. K.; Colic, M.; Wang, S. H.; Doench, J. G.; Hart, T.</p>
        <p class="info">Score: 32.1, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.03.522655' target='https://doi.org/10.1101/2023.01.03.522655'> 10.1101/2023.01.03.522655</a></p>
        <p class="abstract">Genetic interactions mediate the emergence of phenotype from genotype, but initial technologies for combinatorial genetic perturbation in mammalian cells suffer from inefficiency and are challenging to scale. Recent focus on paralog synthetic lethality in cancer cells offers an opportunity to evaluate different approaches and improve on the state of the art. Here we report a meta-analysis of CRISPR genetic interactions screens, identifying a candidate set of background-independent paralog synthetic lethals, and find that the Cas12a platform provides superior sensitivity and assay replicability. We demonstrate that Cas12a can independently target up to four genes from a single guide array, and we build on this knowledge by constructing a genome-scale library that expresses arrays of four guides per clone, a platform we call  in4mer. Our genome-scale human library, with only 49k clones, is substantially smaller than a typical CRISPR/Cas9 monogenic library while also targeting more than four thousand paralog pairs, triples, and quads. Proof of concept screens in four cell lines demonstrate discrimination of core and context-dependent essential genes similar to that of state-of-the-art CRISPR/Cas9 libraries, as well as detection of synthetic lethal and masking/buffering genetic interactions between paralogs of various family sizes, a capability not offered by any extant library. Importantly, the in4mer platform offers a fivefold reduction in the number of clones required to assay genetic interactions, dramatically improving the cost and effort required for these studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556413">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556413" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556413">
        <p class="paperTitle">A hierarchy of metabolite exchanges in metabolic models of microbial species and communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556413" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556413" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wedmark, Y. K.; Vik, J. O.; Oyas, O.</p>
        <p class="info">Score: 7.6, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556413' target='https://doi.org/10.1101/2023.09.05.556413'> 10.1101/2023.09.05.556413</a></p>
        <p class="abstract">MotivationThe metabolic network of an organism can be analyzed as a constraint-based model (CBM). CBM analyses can be biased, optimizing an objective such as growth rate, or unbiased, attempting to describe the feasible flux space through pathway analysis or random flux sampling. Biased methods scale to genome-scale models (GEMs) and beyond, but unbiased methods suffer from combinatorial explosion. One promising approach to improve the scalability of unbiased analyses is to focus on cells metabolite exchanges with each other and the environment rather than the internal details of their metabolic networks. However, methods for unbiased analysis of metabolite exchanges have not been evaluated or compared.

ResultsUsing CBMs ranging in size from core models to GEMs, we applied pathway enumeration and random flux sampling to metabolite exchanges in microbial species and a microbial community. We focused on pathway definitions that allow direct targeting of metabolite exchanges: elementary conversion modes (ECMs), elementary flux patterns (EFPs), and minimal pathways (MPs). Enumerating growth-supporting ECMs, EFPs, and MPs for the same models, we found that the MPs were always a subset of the ECMs and EFPs, specifically the subset of support-minimal pathways. The EFPs were always a subset of the ECMs, which in turn were always a subset of flux patterns from elementary flux modes (EFMs). We argue that this hierarchical relationship between pathway definitions holds more generally and that it helps explain the relative scalability of ECM, EFP, and MP enumeration. Metabolite exchange frequencies in enumerated pathways were similar across methods with a few specific exchanges explaining large differences in pathway counts. Thus, biological interpretation was largely robust to the choice of pathway definition. Random flux sampling scaled to GEMs and complemented pathway enumeration.

AvailabilityData and code are available at https://gitlab.com/YlvaKaW/exchange-enumeration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.02.03.526930">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.02.03.526930" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.02.03.526930">
        <p class="paperTitle">Low dimensionality of phenotypic space as an emergent property of coordinated teams in biological regulatory networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.02.03.526930" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.02.03.526930" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hari, K.; Harlapur, P.; Saxena, A.; Haldar, K.; Girish, A.; Malpani, T.; Levine, H.; Jolly, M. K.</p>
        <p class="info">Score: 8.3, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.02.03.526930' target='https://doi.org/10.1101/2023.02.03.526930'> 10.1101/2023.02.03.526930</a></p>
        <p class="abstract">Biological networks driving cell-fate decisions involve complex interactions, but they often give rise to only a few phenotypes, thus exhibiting low-dimensional dynamics. The network design principles that govern such cell-fate canalization remain unclear. Here, we investigate networks across diverse biological contexts- Epithelial-Mesenchymal Transition, Small Cell Lung Cancer, and Gonadal cell-fate determination - to reveal that the presence of two mutually antagonistic, well-coordinated teams of nodes leads to low-dimensional phenotypic space such that the first principal component (PC1) axis can capture most of the variance. Further analysis of artificial team-based networks and random counterparts of biological networks reveals that the principal component decomposition is determined by the team strength within these networks, demonstrating how the underlying network structure governs PC1 variance. The presence of low dimensionality in corresponding transcriptomic data confirms the applicability of our observations. We propose that team-based topology in biological networks are critical for generating a cell-fate canalization landscape.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556694">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556694" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556694">
        <p class="paperTitle">Longevity interventions in Titan mice attenuate frailty and senescence accumulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556694" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556694" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gille, B.; Muller-Eigner, A.; Gottschalk, S.; Wytrwat, E.; Langhammer, M.; Peleg, S.</p>
        <p class="info">Score: 5.7, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556694' target='https://doi.org/10.1101/2023.09.08.556694'> 10.1101/2023.09.08.556694</a></p>
        <p class="abstract">Wild-type murine models for aging research have lifespans of several years, which results in long experimental duration and late output. Here we explore the short-lived non-inbred Titan mouse as a mouse model to test longevity interventions. We show that Titan mice exhibit increased frailty and cellular senescence at an early age. Dietary intervention attenuates the frailty progression of Titan mice. Additionally, cyclic administration of the senolytic drug Navitoclax at early age increases the lifespan and reduces cellular senescence. Our data suggests that Titan mice can serve as a cost-effective and timely model for longevity interventions in mammals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.09.556962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.09.556962" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.09.556962">
        <p class="paperTitle">Escalating High-dimensional Imaging using Combinatorial Channel Multiplexing and Deep Learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.09.556962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.09.556962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ben-Uri, R.; Ben Shabat, L.; Bar-Tal, O.; Bussi, Y.; Maimon, N.; Keidar Haran, T.; Milo, I.; Elhanani, O.; Rochwarger, A.; Schürch, C. M.; Bagon, S.; Keren, L.</p>
        <p class="info">Score: 5.6, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.09.556962' target='https://doi.org/10.1101/2023.09.09.556962'> 10.1101/2023.09.09.556962</a></p>
        <p class="abstract">Understanding tissue structure and function requires tools that quantify the expression of multiple proteins at single-cell resolution while preserving spatial information. Current imaging technologies use a separate channel for each individual protein, inherently limiting their throughput and scalability. Here, we present CombPlex (COMBinatorial multiPLEXing), a combinatorial staining platform coupled with an algorithmic framework to exponentially increase the number of proteins that can be measured from C up to 2C - 1, and is applicable to any mass spectrometry-based or fluorescence-based microscopy platform. In CombPlex, every protein can be imaged in several channels, and every channel contains agglomerated images of several proteins. These combinatorically-compressed images are then decompressed to individual protein-images using deep learning and optimization. We perform feasibility experiments in silico and achieve accurate (F1=0.98, R=0.99) reconstruction for compressing the stains of twenty-two proteins to five imaging channels. We test our approach experimentally and obtain accurate(F1=0.97, R=0.93) images of seven proteins using three channels, both in fluorescence microscopy and in mass-based imaging. We demonstrate that combinatorial staining coupled with deep-learning decompression can serve to escalate the number of proteins measured using any imaging modality, without the need for specialized instrumentation. Coupling CombPlex with instruments for high dimensional imaging could pave the way to image hundreds of proteins at single-cell resolution in intact tissue sections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556147" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556147">
        <p class="paperTitle">A dynamic Boolean model of molecular and cellular interactions represents psoriasis development and predicts drug candidates.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsirvouli, E.; Noël, V.; Flobak, A.; Calzone, L.; Kuiper, M.</p>
        <p class="info">Score: 5.0, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556147' target='https://doi.org/10.1101/2023.09.03.556147'> 10.1101/2023.09.03.556147</a></p>
        <p class="abstract">Psoriasis is a chronic skin disease affecting 2-3% of the global population. Psoriasis arises from complex interactions between keratinocytes and immune cells, leading to uncontrolled inflammation, immune hyperactivation and perturbed keratinocyte life cycle. Although the latest generation of drugs have greatly improved psoriasis management, the disease remains incurable, and the substantial variability in treatment response calls for novel approaches to comprehend the intricate mechanisms underlying disease development and to discover potential drug targets. In this study, we present a multiscale population model that captures the dynamics of cell-specific phenotypes in psoriasis, integrating discrete logical formalism and population dynamics simulations. Through simulations and network metrics, we identify potential pairwise interventions as alternative treatment options. Specifically, The model predictions suggest that targeting neutrophil activation in conjunction with either PGE2 production or STAT3 signaling shows promise comparable to IL-17 inhibition, which is currently the most used treatment option for moderate and severe cases of psoriasis. Our findings underscore the significance of considering complex intercellular interactions and intracellular signaling cascades in psoriasis, and highlight the importance of computational approaches in unraveling complex biological systems for drug target identification.

Author summaryIn our study, we aimed to uncover the complex mechanisms underlying psoriasis and identify potential treatment options. By utilizing a computational model, we simulated the dynamic interactions between different cell types involved in psoriasis, such as immune cells and keratinocytes. Our model predicts that targeting neutrophil activation, combined with either PGE2 production or STAT3 signaling, may yield comparable effectiveness to the current standard treatment for moderate or severe psoriasis, namely IL-17 inhibition. Our study underscores the importance of computational modeling in unraveling the complexities of disease systems and provides a foundation for identifying new candidate treatment options in psoriasis that should be tested in the lab.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555799">
        <p class="paperTitle">Exploiting fluctuations in gene expression to detect causal interactions between genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joly-Smith, E.; Talpur, M. M.; Allard, P.; Papazotos, F.; Potvin-Trottier, L.; Hilfinger, A.</p>
        <p class="info">Score: 11.2, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555799' target='https://doi.org/10.1101/2023.09.01.555799'> 10.1101/2023.09.01.555799</a></p>
        <p class="abstract">Characterizing and manipulating cellular behaviour requires a mechanistic understanding of the causal interactions between cellular components. We present an approach that can detect causal interactions between genes without the need to perturb the physiological state of cells. This approach exploits naturally occurring cell-to-cell variability which is experimentally accessible from static population snapshots of genetically identical cells without the need to follow cells over time. Our main contribution is a simple mathematical relation that constrains the propagation of gene expression noise through biochemical reaction networks. This relation allows us to rigorously interpret fluctuation data even when only a small part of a complex gene regulatory process can be observed. This relation can be exploited to detect causal interactions by synthetically engineering a passive reporter of gene expression, akin to the established &#34;dual reporter assay&#34;. While the focus of our contribution is theoretical, we also present an experimental proof-of-principle to illustrate the approach. Our data from synthetic gene regulatory networks in E. coli are not unequivocal but suggest that the method could prove useful in practice to identify causal interactions between genes from non-genetic cell-to-cell variability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.556134">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.556134" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.556134">
        <p class="paperTitle">Approximate Bayesian Computation for Inferring Waddington Landscapes from Single Cell Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.556134" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.556134" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, Y.; Zhang, S. Y.; Kleijn, I. T.; Stumpf, M. P. H.</p>
        <p class="info">Score: 8.7, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.556134' target='https://doi.org/10.1101/2023.09.03.556134'> 10.1101/2023.09.03.556134</a></p>
        <p class="abstract">Single cell technologies allow us to gain insights into cellular processes at unprecedented resolution. In stem cell and developmental biology snapshot data allows us to characterise how the transcriptional state of cells changes between successive cell types. Here we show how approximate Bayesian computation (ABC) can be employed to calibrate mathematical models against single cell data. In our simulation study we demonstrate the pivotal role of the adequate choice of distance measures appropriate for single cell data. We show that for good distance measures, notably optimal transport distances, we can infer parameters for mathematical models from simulated single cell data. We show that the ABC posteriors can be used to characterise parameter sensitivity and identify dependencies between different parameters, and to infer representations of the Waddington or epigenetic landscape, which forms a popular and interpretable representation of the developmental dynamics. In summary, these results pave the way for fitting mechanistic models of stem cell differentiation to single cell data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556617">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556617" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556617">
        <p class="paperTitle">Agent-based modeling of tumor-immune interactions reveals determinants of final tumor states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556617" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556617" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahmidouch, M.; Tangella, N.; Finley, S. D.</p>
        <p class="info">Score: 1.2, Published: 2023-09-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556617' target='https://doi.org/10.1101/2023.09.06.556617'> 10.1101/2023.09.06.556617</a></p>
        <p class="abstract">Interactions between tumor and immune cells in the tumor microenvironment (TME) influence tumor growth and the tumors response to treatment. Excitingly, this complex landscape of tumor-immune interactions can be studied using computational modeling. Mathematical oncology can provide quantitative insights into the TME, serving as a framework for understanding tumor dynamics. Here, we use an agent-based model to simulate the interactions among cancer cells, macrophages (naive, M1, and M2), and T cells (active CD8&#43; and inactive) in a 2D representation of the TME. Key diffusible factors, IL-4 and IFN-{gamma}, are also incorporated. We apply the model to predict how cell-specific properties influence tumor progression. The model predictions and analyses revealed the relationships between different cell populations and highlighted the importance of macrophages and T cells in shaping the TME. Thus, we quantify how components of the TME influence the final tumor state and the effects of macrophage-based therapies. The findings emphasize the significant role of computational models in unraveling the intricate dynamics of tumor-immune interactions and their potential for guiding the development of tailored immunotherapeutic strategies. This study provides a foundation for future investigations aiming to refine and expand the model, validate predictions experimentally, and pave the way for improved cancer treatments.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
