<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Remodeling of the protein ubiquitylation landscape in the aging vertebrate brain
Authors: Marino, A.; Di Fraia, D.; Panfilova, D.; Sahu, A. K.; Ori, A.
Score: 18.6, Published: 2023-12-04 DOI: 10.1101/2023.12.02.569713
Post-translational modifications (PTMs) regulate protein homeostasis and function. How aging affects the landscape of PTMs remains largely elusive. Here, we reveal changes in hundreds of protein ubiquitylation, acetylation, and phosphorylation sites in the aging brain of mice. We show that aging has a major impact on protein ubiquitylation and that 29% of the ubiquitylation sites are affected independently of protein abundance, indicating altered PTM stoichiometry.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Remodeling of the protein ubiquitylation landscape in the aging vertebrate brain
Authors: Marino, A.; Di Fraia, D.; Panfilova, D.; Sahu, A. K.; Ori, A.
Score: 18.6, Published: 2023-12-04 DOI: 10.1101/2023.12.02.569713
Post-translational modifications (PTMs) regulate protein homeostasis and function. How aging affects the landscape of PTMs remains largely elusive. Here, we reveal changes in hundreds of protein ubiquitylation, acetylation, and phosphorylation sites in the aging brain of mice. We show that aging has a major impact on protein ubiquitylation and that 29% of the ubiquitylation sites are affected independently of protein abundance, indicating altered PTM stoichiometry." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Remodeling of the protein ubiquitylation landscape in the aging vertebrate brain
Authors: Marino, A.; Di Fraia, D.; Panfilova, D.; Sahu, A. K.; Ori, A.
Score: 18.6, Published: 2023-12-04 DOI: 10.1101/2023.12.02.569713
Post-translational modifications (PTMs) regulate protein homeostasis and function. How aging affects the landscape of PTMs remains largely elusive. Here, we reveal changes in hundreds of protein ubiquitylation, acetylation, and phosphorylation sites in the aging brain of mice. We show that aging has a major impact on protein ubiquitylation and that 29% of the ubiquitylation sites are affected independently of protein abundance, indicating altered PTM stoichiometry."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Remodeling of the protein ubiquitylation landscape in the aging vertebrate brain\nAuthors: Marino, A.; Di Fraia, D.; Panfilova, D.; Sahu, A. K.; Ori, A.\nScore: 18.6, Published: 2023-12-04 DOI: 10.1101/2023.12.02.569713\nPost-translational modifications (PTMs) regulate protein homeostasis and function. How aging affects the landscape of PTMs remains largely elusive. Here, we reveal changes in hundreds of protein ubiquitylation, acetylation, and phosphorylation sites in the aging brain of mice. We show that aging has a major impact on protein ubiquitylation and that 29% of the ubiquitylation sites are affected independently of protein abundance, indicating altered PTM stoichiometry.",
  "keywords": [
    
  ],
  "articleBody": " Remodeling of the protein ubiquitylation landscape in the aging vertebrate brain\nAuthors: Marino, A.; Di Fraia, D.; Panfilova, D.; Sahu, A. K.; Ori, A.\nScore: 18.6, Published: 2023-12-04 DOI: 10.1101/2023.12.02.569713\nPost-translational modifications (PTMs) regulate protein homeostasis and function. How aging affects the landscape of PTMs remains largely elusive. Here, we reveal changes in hundreds of protein ubiquitylation, acetylation, and phosphorylation sites in the aging brain of mice. We show that aging has a major impact on protein ubiquitylation and that 29% of the ubiquitylation sites are affected independently of protein abundance, indicating altered PTM stoichiometry. We found a subset of these sites to be also affected in the brain of the short-lived killifish Nothobranchius furzeri, highlighting a conserved aging phenotype. Furthermore, we estimated that over 35% of ubiquitylation changes observed in old mouse brains derive from partial proteasome inhibition, a well-established hallmark of brain aging. Our findings provide evidence of an evolutionarily conserved ubiquitylation signature of the aging brain and establish a causal link between proteasome inhibition and age-related remodeling of the ubiquitylome.\npyRBDome: A comprehensive computational platform for enhancing and interpreting RNA-binding proteome data\nAuthors: Chu, L.-C.; Christopoulou, N.; McCaughan, H.; Winterbourne, S.; Cazzola, D.; Wang, S.; Litvin, U.; Brunon, S.; Harker, P. J. B.; McNae, I.; Granneman, S.\nScore: 10.0, Published: 2023-12-08 DOI: 10.1101/2023.12.08.570608\nHigh-throughput proteomics approaches have revolutionised the identification of RNA-binding proteins (RBPome) and RNA-binding sequences (RBDome) across organisms. Many novel putative RNA-binding proteins (RBPs) were discovered, including those that lack recognisable RNA-binding domains. Yet the extent of noise, including false-positive identifications associated with these methodologies, is difficult to quantify as experimental approaches for validating the results are generally low throughput. To address this, we introduce pyRBDome, a pipeline for in-depth in silico enhancement of RNA-binding proteome data. It does so by comparing experimental results with RNA-binding site (RBS) predictions from multiple and distinct machine learning tools and integrates high-resolution structural data of protein-RNA complexes when available. By providing a statistical evaluation of RBDome data, users can rapidly identify protein sequences from the experimental data most likely to be bona fide RNA-binders. Furthermore, by leveraging the predictions collated by pyRBDome, we have enhanced the sensitivity and specificity of RBS detection through training new ensemble machine learning models. We describe a pyRBDome analysis of a large human RBDome dataset and conducted a comparison with known structural data. These analyses reinforced the significance of stacking interactions in UV cross-linking of protein-RNA interactions. Surprisingly, our analyses revealed two contrasting findings: While UV cross-linked amino acids were more likely to contain predicted RBSs, they infrequently bind RNA in high-resolution structures. Given the known limitations of structural data as benchmarks, these findings highlight the utility of pyRBDome as a valuable alternative approach for enhancing confidence in RBDome datasets. Finally, our comprehensive analysis of hundreds of (putative) RBPs offers a valuable resource for RBP enthusiasts\nDifferential proteome diversification in yeast populations: modes of short-term adaptation and fitness outcomes\nAuthors: Knafo, M.; Rezenman, S.; Tsigalnitski, I.; Nevo, R.; Reich, Z.; Kapon, R.\nScore: 10.0, Published: 2023-12-07 DOI: 10.1101/2023.11.28.568995\nShort-term proteomic adaptations serve as an initial line of defence, allowing populations to cope with environmental changes before long-term genetic alterations occur. Using a representative set of genes, we examined how stress affects gene expression variability for different types and levels of abiotic stresses and how this influences population-level adaptation. Our data reveal that, depending on the nature of the stress, two distinct modes of response can be employed. In one, the levels of most proteins vary between individuals, leading to varied fitness levels in the population. In the other, a more limited range of expression is seen, and fitness is more even. This suggests different levels of complexity and plasticity in adaptation to different types of stress.\nA foundational atlas of autism protein interactions reveals molecular convergence\nAuthors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.; Pham, P.; Arbalaez, J.; Pratt, D.; Chag, S.; Rolland, T.; Bourgeron, T.; Finkbeiner, S.; Bandyopadhay, S.; Ideker, T.; Beltrao, P.; Willsey, H. R.; Obernier, K.; Nowakowski, T. J.; Huttenhain, R.; State, M. W.; Willsey, A. J.; Krogan, N. J.\nScore: 57.2, Published: 2023-12-09 DOI: 10.1101/2023.12.03.569805\nDespite the identification of a large number of high-confidence (hc) autism spectrum disorder (ASD) risk genes, translation to viable treatment targets remains elusive. Using affinity purification-mass spectrometry and HEK293T cells, we constructed a protein-protein interaction (PPI) network involving 100 hcASD risk genes, revealing over 1,800 PPIs, the vast majority of which have not been previously described. Importantly, the interactors are highly expressed in the human brain and neural progenitor cells, and enriched for ASD, but not schizophrenia, genetic risk. The network is highly connected, converges on protein complexes, including those involved in neurogenesis, tubulin biology, transcriptional regulation, and chromatin modification. We also generated a PPI map using over 50 patient-derived missense variants to identify differential physical interactions. Using these data, we leveraged AlphaFold2 predictions to prioritize direct PPIs as well as specific variants for interrogation in Xenopus tropicalis and human forebrain organoids. One such mutation in the transcription factor FOXP1, which disrupts interaction with FOXP4, led to a reconfiguration of DNA binding sites and disruption of neurogenesis, with a particularly strong impact on deep layer cortical and subplate neurons,consistent with a large body of work in ASD. Overall, this work offers critical new insights into molecular mechanisms underlying ASD that results from large effect genetic variants and describes a powerful platform to develop and test novel therapeutic strategies for a wide range of genetically-defined conditions.\nBroad H3K4me3 Domain Is Associated with Spatial Coherence during Mammalian Embryonic Development\nAuthors: Cao, X.; Ma, T.; Fan, R.; Yuan, G.-C.\nScore: 8.9, Published: 2023-12-12 DOI: 10.1101/2023.12.11.570452\nIt is well known that the chromatin states play a major role in cell-fate decision and cell-identity maintenance; however, the spatial variation of chromatin states in situ remains poorly characterized. Here, by leveraging recently available spatial-CUT\u0026Tag data, we systematically characterized the global spatial organization of the H3K4me3 profiles in a mouse embryo. Our analysis identified a subset of genes with spatially coherent H3K4me3 patterns, which together delineate the tissue boundaries. The spatially coherent genes are strongly enriched with tissue-specific transcriptional regulators. Remarkably, their corresponding genomic loci are marked by broad H3K4me3 domains, which is distinct from the typical H3K4me3 signature. Spatial transition across tissue boundaries is associated with continuous shortening of the broad H3K4me3 domains as well as expansion of H3K27me3 domains. Our analysis reveals a strong connection between the genomic and spatial variation of chromatin states, which may play an important role in embryonic development.\nSignalling-state dependent drug-tolerance in head and neck squamous cell carcinoma.\nAuthors: Karjosukarso, D. W.; Dini, A.; Wingens, L. J. A.; Liu, R.; Joosten, L. A. B.; Bussink, J.; Mulder, K. W.\nScore: 8.4, Published: 2023-12-07 DOI: 10.1101/2023.12.05.570063\nIntratumor heterogeneity negatively impacts therapeutic response and patient prognosis. Besides the established role of genetic heterogeneity, non-genetic mechanisms of persistence to drug treatment are emerging. Here, we characterise cells selected for their persistence to control, epidermal growth factor inhibition (EGFRi), radiation and combined treatment from low passage head and neck squamous cell carcinoma (HNSCC) cultures. Using a panel of 70 (phospho-)specific DNA-conjugated antibodies we measured activities of 8 signalling pathways, self-renewal, differentiation, DNA damage and cell-cycle, in conjunction with the transcriptional output in single cells, using our RNA and Immuno-Detection (RAID) technology. Six recurrent transcriptional programs reflecting processes including proliferation, differentiation and metabolic activity, as well as protein-based signalling-states, were associated with drug persistence, while copy number variation inference indicated involvement of non-genetic tolerance mechanisms. Projecting RNA velocity onto the antibody-derived signalling-states suggested a key role for integrin-mediated focal-adhesion signalling in drug-persistence in our cell system. Using machine-learning we derived a core transcriptional signature connected to adhesion-based drug-persistence, which was predictive of poor prognosis in a TGCA HNSCC cohort (hazard-ratio 1.87, p\u003c10-5). Furthermore, functional analyses confirmed that cells expressing high levels of integrin alpha-6 (ITGA6) were tolerant to EGFRi treatment, and that forcing cells out of this cell-state through transient targeted inhibition of Focal Adhesion Kinase activity re-instated EGFRi sensitivity in drug persistent cells. Taken together, our single-cell multi-omics analysis identified an actionable adhesion-signalling mediated cell-state driving drug tolerance in HNSCC.\nCell-state transitions and frequency-dependent interactions among subpopulations together explain the dynamics of spontaneous epithelial-mesenchymal heterogeneity in breast cancer\nAuthors: Jain, P.; Kizhuttil, R.; Nair, M. B.; Bhatia, S.; Thompson, E. W.; George, J. T.; Jolly, M. K.\nScore: 4.0, Published: 2023-12-08 DOI: 10.1101/2023.12.07.567986\nIndividual cells in a tumour can be distributed among Epithelial (E) and Mesenchymal (M) cell-states, as characterised by the levels of canonical E and M markers. Even after E and M (E-M) subpopulations are isolated and then cultured independently, E-M heterogeneity can re-equilibrate in each population over time, sometimes regaining the initial distribution of the parental cell population. However, it remains unclear which population-level processes give rise to the dynamical changes in E-M heterogeneity observed experimentally, including 1) differential growth, 2) cell-state switching, and 3) frequency-dependent growth or state-transition rates. Here, we analyse the necessity of these three processes in explaining the dynamics of E-M population distributions as observed in PMC42-LA and HCC38 breast cancer cells. We find that growth differences among E and M subpopulations, with and without any frequency-dependent interactions (cooperation or suppression) among E-M sub-populations, are insufficient to explain the observed population dynamics. This insufficiency is ameliorated by including cell-state transitions, albeit at slow rates, in explaining both PMC42-LA and HCC38 cells data. Further, our models predict that treatment of HCC38 cells with TGF{beta} signalling and JAK2/3 inhibitors could significantly enhance the transition rates from M state to E state, but does not prevent transitions from E to M. Finally, we devise a selection criterion to identify the next most informative time points for which future experimental data can optimally improve the identifiability of our estimated best fit model parameters. Overall, our study identifies the necessary population-level processes shaping the dynamics of E-M heterogeneity in breast cancer cells.\nPredicting Phenotypic Traits Using a Massive RNA-seq Dataset\nAuthors: Hadish, J. A.; Honaas, L. A.; Ficklin, S. P.\nScore: 3.6, Published: 2023-12-07 DOI: 10.1101/2023.12.05.570195\nTranscriptomic data can be used to predict environmentally impacted phenotypic traits. This type of prediction is particularly useful for monitoring difficult-to-measure phenotypic traits and has become increasingly popular for monitoring high-value agricultural crops and in precision medicine. Despite this increase in popularity, little research has been done on how many samples are required for these models to be accurate, and which normalization should be used. Here we create a massive RNA-seq dataset from publicly available Arabidopsis thaliana data with corresponding measurements for age and tissue type. We use this dataset to determine how many samples are required for accurate model prediction and which normalization method is required. We find that Median Ratios Normalization significantly increases performance when predicting age. We also find that in the case of our dataset, only a few hundred samples are required to predict tissue types, and only a few thousand samples are necessary to accurately predict age. Researchers should consider these results when choosing the number of samples in a transcriptomic experiment and during data-processing. Author SummaryLarge datasets have become ubiquitous in both research and industry, with thousands and sometimes millions of samples being collected for a single project. In biology a prominent new technology is RNA-seq, which can be used to measure the expression level of thousands of genes for a single sample. These measurements are used for a variety of downstream applications, including predicting phenotypic traits (i.e. height, disease, etc.). A number of experiments have attempted to use RNA-seq data to make phenotype predictions with varying success. This is partially due to the small sample size of their experiments. RNA-seq datasets are currently relatively small--only a dozen to a few hundred samples--due to the cost per sample. This is expected to change as the cost of sequencing decreases. In this paper we create a massive conglomerate RNA-seq dataset from publicly available Arabidopsis thaliana RNA-seq data. We use this dataset to determine how many samples are required to accurately predict plant age and tissue type using machine learning models. We also explore the best way to normalize large datasets. Our results show the potential of massive RNA-seq datasets, and can be used to inform experimental design for phenotype prediction.\nA new universal system of tree shape indices\nAuthors: Noble, R. J.; Verity, K.\nScore: 13.9, Published: 2023-11-27 DOI: 10.1101/2023.07.17.549219\nAO_SCPLOWBSTRACTC_SCPLOWThe comparison and categorization of tree diagrams is fundamental to large parts of biology, linguistics, computer science, and other fields, yet the indices currently applied to describing tree shape have important flaws that complicate their interpretation and limit their scope. Here we introduce a new system of indices with no such shortcomings. Our indices account for node sizes and branch lengths and are robust to small changes in either attribute. Unlike currently popular phylogenetic diversity, phylogenetic entropy, and tree balance indices, our definitions assign interpretable values to all rooted trees and enable meaningful comparison of any pair of trees. Our self-consistent definitions further unite measures of diversity, richness, balance, symmetry, effective height, effective outdegree, and effective branch count in a coherent system, and we derive numerous simple relationships between these indices. The main practical advantages of our indices are in 1) quantifying diversity in non-ultrametric trees; 2) assessing the balance of trees that have non-uniform branch lengths or node sizes; 3) comparing the balance of trees with different leaf counts or outdegrees; 4) obtaining a coherent, generic, multidimensional quantification of tree shape that is robust to sampling error and inferential error. We illustrate these features by comparing the shapes of trees representing the evolution of HIV and of Uralic languages, and trees generated by computational models of tumour evolution. Given the ubiquity of tree structures, we identify a wide range of applications across diverse domains.\nAn evolution-based framework for describing human gut bacteria\nAuthors: Doran, B. A.; Chen, R. Y.; Giba, H.; Behera, V.; Barat, B.; Sundararajan, A.; Lin, H.; Sidebottom, A.; Pamer, E. G.; Raman, A.\nScore: 3.0, Published: 2023-12-05 DOI: 10.1101/2023.12.04.569969\nThe human gut microbiome contains many bacterial strains of the same species ( strain-level variants). Describing strains in a biologically meaningful way rather than purely taxonomically is an important goal but challenging due to the genetic complexity of strain-level variation. Here, we measured patterns of co-evolution across \u003e7,000 strains spanning the bacterial tree-of-life. Using these patterns as a prior for studying hundreds of gut commensal strains that we isolated, sequenced, and metabolically profiled revealed widespread structure beneath the phylogenetic level of species. Defining strains by their co-evolutionary signatures enabled predicting their metabolic phenotypes and engineering consortia from strain genome content alone. Our findings demonstrate a biologically relevant organization to strain-level variation and motivate a new schema for describing bacterial strains based on their evolutionary history. One Sentence SummaryDescribing bacterial strains in the human gut by a statistical model that captures their evolutionary history provides insight into their biology.\n",
  "wordCount" : "2528",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569713">
        <p class="paperTitle">Remodeling of the protein ubiquitylation landscape in the aging vertebrate brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marino, A.; Di Fraia, D.; Panfilova, D.; Sahu, A. K.; Ori, A.</p>
        <p class="info">Score: 18.6, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569713' target='https://doi.org/10.1101/2023.12.02.569713'> 10.1101/2023.12.02.569713</a></p>
        <p class="abstract">Post-translational modifications (PTMs) regulate protein homeostasis and function. How aging affects the landscape of PTMs remains largely elusive. Here, we reveal changes in hundreds of protein ubiquitylation, acetylation, and phosphorylation sites in the aging brain of mice. We show that aging has a major impact on protein ubiquitylation and that 29% of the ubiquitylation sites are affected independently of protein abundance, indicating altered PTM stoichiometry. We found a subset of these sites to be also affected in the brain of the short-lived killifish Nothobranchius furzeri, highlighting a conserved aging phenotype. Furthermore, we estimated that over 35% of ubiquitylation changes observed in old mouse brains derive from partial proteasome inhibition, a well-established hallmark of brain aging. Our findings provide evidence of an evolutionarily conserved ubiquitylation signature of the aging brain and establish a causal link between proteasome inhibition and age-related remodeling of the ubiquitylome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570608">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570608" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570608">
        <p class="paperTitle">pyRBDome: A comprehensive computational platform for enhancing and interpreting RNA-binding proteome data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570608" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570608" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chu, L.-C.; Christopoulou, N.; McCaughan, H.; Winterbourne, S.; Cazzola, D.; Wang, S.; Litvin, U.; Brunon, S.; Harker, P. J. B.; McNae, I.; Granneman, S.</p>
        <p class="info">Score: 10.0, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570608' target='https://doi.org/10.1101/2023.12.08.570608'> 10.1101/2023.12.08.570608</a></p>
        <p class="abstract">High-throughput proteomics approaches have revolutionised the identification of RNA-binding proteins (RBPome) and RNA-binding sequences (RBDome) across organisms. Many novel putative RNA-binding proteins (RBPs) were discovered, including those that lack recognisable RNA-binding domains. Yet the extent of noise, including false-positive identifications associated with these methodologies, is difficult to quantify as experimental approaches for validating the results are generally low throughput. To address this, we introduce pyRBDome, a pipeline for in-depth in silico enhancement of RNA-binding proteome data. It does so by comparing experimental results with RNA-binding site (RBS) predictions from multiple and distinct machine learning tools and integrates high-resolution structural data of protein-RNA complexes when available. By providing a statistical evaluation of RBDome data, users can rapidly identify protein sequences from the experimental data most likely to be bona fide RNA-binders. Furthermore, by leveraging the predictions collated by pyRBDome, we have enhanced the sensitivity and specificity of RBS detection through training new ensemble machine learning models. We describe a pyRBDome analysis of a large human RBDome dataset and conducted a comparison with known structural data. These analyses reinforced the significance of stacking interactions in UV cross-linking of protein-RNA interactions. Surprisingly, our analyses revealed two contrasting findings: While UV cross-linked amino acids were more likely to contain predicted RBSs, they infrequently bind RNA in high-resolution structures. Given the known limitations of structural data as benchmarks, these findings highlight the utility of pyRBDome as a valuable alternative approach for enhancing confidence in RBDome datasets. Finally, our comprehensive analysis of hundreds of (putative) RBPs offers a valuable resource for RBP enthusiasts</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568995">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568995" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568995">
        <p class="paperTitle">Differential proteome diversification in yeast populations: modes of short-term adaptation and fitness outcomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568995" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568995" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Knafo, M.; Rezenman, S.; Tsigalnitski, I.; Nevo, R.; Reich, Z.; Kapon, R.</p>
        <p class="info">Score: 10.0, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568995' target='https://doi.org/10.1101/2023.11.28.568995'> 10.1101/2023.11.28.568995</a></p>
        <p class="abstract">Short-term proteomic adaptations serve as an initial line of defence, allowing populations to cope with environmental changes before long-term genetic alterations occur. Using a representative set of genes, we examined how stress affects gene expression variability for different types and levels of abiotic stresses and how this influences population-level adaptation. Our data reveal that, depending on the nature of the stress, two distinct modes of response can be employed. In one, the levels of most proteins vary between individuals, leading to varied fitness levels in the population. In the other, a more limited range of expression is seen, and fitness is more even. This suggests different levels of complexity and plasticity in adaptation to different types of stress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569805">
        <p class="paperTitle">A foundational atlas of autism protein interactions reveals molecular convergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, B.; Vartak, R.; Zaltsman, Y.; Naing, Z. Z. C.; Hennick, K. M.; Polacco, B. J.; Bashir, A.; Eckhardt, M.; Bouhaddou, M.; Xu, J.; Sun, N.; Lasser, M.; Zhou, Y.; Guiley, K. Z.; Chan, U.; Kaye, J. A.; Khare, P.; Drake, S.; Drury, V.; Burke, D. F.; Gonzalez, S.; Alkhairy, S.; Morris, M.; Baum, T.; Krasnoff, R.; Wang, S.; Pham, P.; Arbalaez, J.; Pratt, D.; Chag, S.; Rolland, T.; Bourgeron, T.; Finkbeiner, S.; Bandyopadhay, S.; Ideker, T.; Beltrao, P.; Willsey, H. R.; Obernier, K.; Nowakowski, T. J.; Huttenhain, R.; State, M. W.; Willsey, A. J.; Krogan, N. J.</p>
        <p class="info">Score: 57.2, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569805' target='https://doi.org/10.1101/2023.12.03.569805'> 10.1101/2023.12.03.569805</a></p>
        <p class="abstract">Despite the identification of a large number of high-confidence (hc) autism spectrum disorder (ASD) risk genes, translation to viable treatment targets remains elusive. Using affinity purification-mass spectrometry and HEK293T cells, we constructed a protein-protein interaction (PPI) network involving 100 hcASD risk genes, revealing over 1,800 PPIs, the vast majority of which have not been previously described. Importantly, the interactors are highly expressed in the human brain and neural progenitor cells, and enriched for ASD, but not schizophrenia, genetic risk. The network is highly connected, converges on protein complexes, including those involved in neurogenesis, tubulin biology, transcriptional regulation, and chromatin modification. We also generated a PPI map using over 50 patient-derived missense variants to identify differential physical interactions. Using these data, we leveraged AlphaFold2 predictions to prioritize direct PPIs as well as specific variants for interrogation in Xenopus tropicalis and human forebrain organoids. One such mutation in the transcription factor FOXP1, which disrupts interaction with FOXP4, led to a reconfiguration of DNA binding sites and disruption of neurogenesis, with a particularly strong impact on deep layer cortical and subplate neurons,consistent with a large body of work in ASD. Overall, this work offers critical new insights into molecular mechanisms underlying ASD that results from large effect genetic variants and describes a powerful platform to develop and test novel therapeutic strategies for a wide range of genetically-defined conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.570452">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.570452" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.570452">
        <p class="paperTitle">Broad H3K4me3 Domain Is Associated with Spatial Coherence during Mammalian Embryonic Development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.570452" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.570452" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cao, X.; Ma, T.; Fan, R.; Yuan, G.-C.</p>
        <p class="info">Score: 8.9, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.570452' target='https://doi.org/10.1101/2023.12.11.570452'> 10.1101/2023.12.11.570452</a></p>
        <p class="abstract">It is well known that the chromatin states play a major role in cell-fate decision and cell-identity maintenance; however, the spatial variation of chromatin states in situ remains poorly characterized. Here, by leveraging recently available spatial-CUT&amp;Tag data, we systematically characterized the global spatial organization of the H3K4me3 profiles in a mouse embryo. Our analysis identified a subset of genes with spatially coherent H3K4me3 patterns, which together delineate the tissue boundaries. The spatially coherent genes are strongly enriched with tissue-specific transcriptional regulators. Remarkably, their corresponding genomic loci are marked by broad H3K4me3 domains, which is distinct from the typical H3K4me3 signature. Spatial transition across tissue boundaries is associated with continuous shortening of the broad H3K4me3 domains as well as expansion of H3K27me3 domains. Our analysis reveals a strong connection between the genomic and spatial variation of chromatin states, which may play an important role in embryonic development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570063">
        <p class="paperTitle">Signalling-state dependent drug-tolerance in head and neck squamous cell carcinoma.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karjosukarso, D. W.; Dini, A.; Wingens, L. J. A.; Liu, R.; Joosten, L. A. B.; Bussink, J.; Mulder, K. W.</p>
        <p class="info">Score: 8.4, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570063' target='https://doi.org/10.1101/2023.12.05.570063'> 10.1101/2023.12.05.570063</a></p>
        <p class="abstract">Intratumor heterogeneity negatively impacts therapeutic response and patient prognosis. Besides the established role of genetic heterogeneity, non-genetic mechanisms of persistence to drug treatment are emerging. Here, we characterise cells selected for their persistence to control, epidermal growth factor inhibition (EGFRi), radiation and combined treatment from low passage head and neck squamous cell carcinoma (HNSCC) cultures. Using a panel of 70 (phospho-)specific DNA-conjugated antibodies we measured activities of 8 signalling pathways, self-renewal, differentiation, DNA damage and cell-cycle, in conjunction with the transcriptional output in single cells, using our RNA and Immuno-Detection (RAID) technology. Six recurrent transcriptional programs reflecting processes including proliferation, differentiation and metabolic activity, as well as protein-based signalling-states, were associated with drug persistence, while copy number variation inference indicated involvement of non-genetic tolerance mechanisms. Projecting RNA velocity onto the antibody-derived signalling-states suggested a key role for integrin-mediated focal-adhesion signalling in drug-persistence in our cell system. Using machine-learning we derived a core transcriptional signature connected to adhesion-based drug-persistence, which was predictive of poor prognosis in a TGCA HNSCC cohort (hazard-ratio 1.87, p&lt;10-5). Furthermore, functional analyses confirmed that cells expressing high levels of integrin alpha-6 (ITGA6) were tolerant to EGFRi treatment, and that forcing cells out of this cell-state through transient targeted inhibition of Focal Adhesion Kinase activity re-instated EGFRi sensitivity in drug persistent cells. Taken together, our single-cell multi-omics analysis identified an actionable adhesion-signalling mediated cell-state driving drug tolerance in HNSCC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.567986">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.567986" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.567986">
        <p class="paperTitle">Cell-state transitions and frequency-dependent interactions among subpopulations together explain the dynamics of spontaneous epithelial-mesenchymal heterogeneity in breast cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.567986" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.567986" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jain, P.; Kizhuttil, R.; Nair, M. B.; Bhatia, S.; Thompson, E. W.; George, J. T.; Jolly, M. K.</p>
        <p class="info">Score: 4.0, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.567986' target='https://doi.org/10.1101/2023.12.07.567986'> 10.1101/2023.12.07.567986</a></p>
        <p class="abstract">Individual cells in a tumour can be distributed among Epithelial (E) and Mesenchymal (M) cell-states, as characterised by the levels of canonical E and M markers. Even after E and M (E-M) subpopulations are isolated and then cultured independently, E-M heterogeneity can re-equilibrate in each population over time, sometimes regaining the initial distribution of the parental cell population. However, it remains unclear which population-level processes give rise to the dynamical changes in E-M heterogeneity observed experimentally, including 1) differential growth, 2) cell-state switching, and 3) frequency-dependent growth or state-transition rates. Here, we analyse the necessity of these three processes in explaining the dynamics of E-M population distributions as observed in PMC42-LA and HCC38 breast cancer cells. We find that growth differences among E and M subpopulations, with and without any frequency-dependent interactions (cooperation or suppression) among E-M sub-populations, are insufficient to explain the observed population dynamics. This insufficiency is ameliorated by including cell-state transitions, albeit at slow rates, in explaining both PMC42-LA and HCC38 cells data. Further, our models predict that treatment of HCC38 cells with TGF{beta} signalling and JAK2/3 inhibitors could significantly enhance the transition rates from M state to E state, but does not prevent transitions from E to M. Finally, we devise a selection criterion to identify the next most informative time points for which future experimental data can optimally improve the identifiability of our estimated best fit model parameters. Overall, our study identifies the necessary population-level processes shaping the dynamics of E-M heterogeneity in breast cancer cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570195">
        <p class="paperTitle">Predicting Phenotypic Traits Using a Massive RNA-seq Dataset</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hadish, J. A.; Honaas, L. A.; Ficklin, S. P.</p>
        <p class="info">Score: 3.6, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570195' target='https://doi.org/10.1101/2023.12.05.570195'> 10.1101/2023.12.05.570195</a></p>
        <p class="abstract">Transcriptomic data can be used to predict environmentally impacted phenotypic traits. This type of prediction is particularly useful for monitoring difficult-to-measure phenotypic traits and has become increasingly popular for monitoring high-value agricultural crops and in precision medicine. Despite this increase in popularity, little research has been done on how many samples are required for these models to be accurate, and which normalization should be used. Here we create a massive RNA-seq dataset from publicly available Arabidopsis thaliana data with corresponding measurements for age and tissue type. We use this dataset to determine how many samples are required for accurate model prediction and which normalization method is required. We find that Median Ratios Normalization significantly increases performance when predicting age. We also find that in the case of our dataset, only a few hundred samples are required to predict tissue types, and only a few thousand samples are necessary to accurately predict age. Researchers should consider these results when choosing the number of samples in a transcriptomic experiment and during data-processing.

Author SummaryLarge datasets have become ubiquitous in both research and industry, with thousands and sometimes millions of samples being collected for a single project. In biology a prominent new technology is RNA-seq, which can be used to measure the expression level of thousands of genes for a single sample. These measurements are used for a variety of downstream applications, including predicting phenotypic traits (i.e. height, disease, etc.). A number of experiments have attempted to use RNA-seq data to make phenotype predictions with varying success. This is partially due to the small sample size of their experiments. RNA-seq datasets are currently relatively small--only a dozen to a few hundred samples--due to the cost per sample. This is expected to change as the cost of sequencing decreases. In this paper we create a massive conglomerate RNA-seq dataset from publicly available Arabidopsis thaliana RNA-seq data. We use this dataset to determine how many samples are required to accurately predict plant age and tissue type using machine learning models. We also explore the best way to normalize large datasets. Our results show the potential of massive RNA-seq datasets, and can be used to inform experimental design for phenotype prediction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549219" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549219">
        <p class="paperTitle">A new universal system of tree shape indices</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Noble, R. J.; Verity, K.</p>
        <p class="info">Score: 13.9, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549219' target='https://doi.org/10.1101/2023.07.17.549219'> 10.1101/2023.07.17.549219</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWThe comparison and categorization of tree diagrams is fundamental to large parts of biology, linguistics, computer science, and other fields, yet the indices currently applied to describing tree shape have important flaws that complicate their interpretation and limit their scope. Here we introduce a new system of indices with no such shortcomings. Our indices account for node sizes and branch lengths and are robust to small changes in either attribute. Unlike currently popular phylogenetic diversity, phylogenetic entropy, and tree balance indices, our definitions assign interpretable values to all rooted trees and enable meaningful comparison of any pair of trees. Our self-consistent definitions further unite measures of diversity, richness, balance, symmetry, effective height, effective outdegree, and effective branch count in a coherent system, and we derive numerous simple relationships between these indices. The main practical advantages of our indices are in 1) quantifying diversity in non-ultrametric trees; 2) assessing the balance of trees that have non-uniform branch lengths or node sizes; 3) comparing the balance of trees with different leaf counts or outdegrees; 4) obtaining a coherent, generic, multidimensional quantification of tree shape that is robust to sampling error and inferential error. We illustrate these features by comparing the shapes of trees representing the evolution of HIV and of Uralic languages, and trees generated by computational models of tumour evolution. Given the ubiquity of tree structures, we identify a wide range of applications across diverse domains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569969">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569969" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569969">
        <p class="paperTitle">An evolution-based framework for describing human gut bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569969" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569969" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Doran, B. A.; Chen, R. Y.; Giba, H.; Behera, V.; Barat, B.; Sundararajan, A.; Lin, H.; Sidebottom, A.; Pamer, E. G.; Raman, A.</p>
        <p class="info">Score: 3.0, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569969' target='https://doi.org/10.1101/2023.12.04.569969'> 10.1101/2023.12.04.569969</a></p>
        <p class="abstract">The human gut microbiome contains many bacterial strains of the same species ( strain-level variants). Describing strains in a biologically meaningful way rather than purely taxonomically is an important goal but challenging due to the genetic complexity of strain-level variation. Here, we measured patterns of co-evolution across &gt;7,000 strains spanning the bacterial tree-of-life. Using these patterns as a prior for studying hundreds of gut commensal strains that we isolated, sequenced, and metabolically profiled revealed widespread structure beneath the phylogenetic level of species. Defining strains by their co-evolutionary signatures enabled predicting their metabolic phenotypes and engineering consortia from strain genome content alone. Our findings demonstrate a biologically relevant organization to strain-level variation and motivate a new schema for describing bacterial strains based on their evolutionary history.

One Sentence SummaryDescribing bacterial strains in the human gut by a statistical model that captures their evolutionary history provides insight into their biology.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
