<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput
Authors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.
Score: 16.4, Published: 2023-11-14 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput
Authors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.
Score: 16.4, Published: 2023-11-14 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-22T10:41:31+00:00" />
<meta property="article:modified_time" content="2023-11-22T10:41:31+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput
Authors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.
Score: 16.4, Published: 2023-11-14 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput\nAuthors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.\nScore: 16.4, Published: 2023-11-14 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput\nAuthors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.\nScore: 16.4, Published: 2023-11-14 DOI: 10.1101/2023.11.11.566649\nSignaling pathways that drive gene expression are typically depicted as having a dozen or so landmark phosphorylation and transcriptional events. In reality, thousands of dynamic post-translational modifications (PTMs) orchestrate nearly every cellular function, and we lack technologies to find causal links between these vast biochemical pathways and genetic circuits at scale. Here, we describe \"signaling-to-transcription network\" mapping through the development of PTM-centric base editing coupled to phenotypic screens, directed by temporally-resolved phosphoproteomics. Using T cell activation as a model, we observe hundreds of unstudied phosphorylation sites that modulate NFAT transcriptional activity. We identify the phosphorylation-mediated nuclear localization of the phosphatase PHLPP1 which promotes NFAT but inhibits NF{kappa}B activity. We also find that specific phosphosite mutants can alter gene expression in subtle yet distinct patterns, demonstrating the potential for fine-tuning transcriptional responses. Overall, base editor screening of PTM sites provides a powerful platform to dissect PTM function within signaling pathways.\nSynthetic model ecosystem of 12 cryopreservable microbial species allowing for a noninvasive approach\nAuthors: Hosoda, K.; Seno, S.; Murakami, N.; Matsuda, H.; Osada, Y.; Kamiura, R.; Kondoh, M.\nScore: 38.0, Published: 2023-10-31 DOI: 10.1101/2020.10.23.351742\nSimultaneous understanding of both individual and ecosystem dynamics is crucial in an era marked by the degradation of ecosystem services. Herein, we present a high-throughput synthetic microcosm system comprising 12 functionally and phylogenetically diverse microbial species. These species are axenically culturable, cryopreservable, and can be measured noninvasively via microscopy, aided by machine learning. This system includes prokaryotic and eukaryotic producers and decomposers, and eukaryotic consumers to ensure functional redundancy. Our model system displayed both positive and negative interspecific interactions and higher-order interactions that surpassed the scope of any two-species interaction. Although complete species coexistence was not our primary objective, we identified several conditions under which at least one species from the producers, consumers, and decomposers groups, and one functionally redundant species, persisted for over six months. These conditions set the stage for detailed investigations in the future. Given its designability and experimental replicability, our model ecosystem offers a promising platform for deeper insights into both individual and ecosystem dynamics, including evolution and species interactions.\nAn enzyme activation network provides evidence for extensive regulatory crosstalk between metabolic pathways\nAuthors: Alam, M. T.; Al Zubaidi, S. M.; Nasar, M. I.; Ralser, M.; Notebaart, R. A.\nScore: 11.5, Published: 2023-11-18 DOI: 10.1101/2023.11.16.567372\nEnzyme activation by cellular metabolites plays a pivotal role in regulating metabolic processes. Nevertheless, our comprehension of such activation events on a global network scale remains incomplete. In this study, we conducted a comprehensive investigation into the optimization of cell-intrinsic activation interactions within Saccharomyces cerevisiae. To achieve this, we integrated a genome-scale metabolic model with enzyme kinetic data sourced from the BRENDA database. Our objective was to map the distribution of enzyme activators throughout the cellular network. Our findings indicate that virtually all biochemical pathways encompass enzyme activators, frequently originating from disparate pathways, thus revealing extensive regulatory crosstalk between metabolic pathways. Indeed, activators have short pathway lengths, indicating they are activated quickly upon nutrient shifts, and in most instances, these activators target key enzymatic reactions to facilitate downstream metabolic processes. Interestingly, non-essential enzymes exhibit a significantly higher degree of activation compared to their essential counterparts. This observation suggests that cells employ enzyme activators to finely regulate secondary metabolic pathways that are only required under specific conditions. Conversely, the activator metabolites themselves are more likely to be essential components, and their activation levels surpass those of non-essential activators. In summary, our study unveils the widespread importance of enzymatic activators, and suggests that feed-forward activation of conditional metabolic pathways through essential metabolites mediates metabolic plasticity.\nVisual interpretability of image-based classification models by generative latent space disentanglement applied to in vitro fertilization\nAuthors: Rotem, O.; Schwartz, T.; Maor, R.; Tauber, Y.; Tsarfati Shapiro, M.; Meseguer, M.; Gilboa, D.; Seidman, D. S.; Zaritsky, A.\nScore: 8.8, Published: 2023-11-17 DOI: 10.1101/2023.11.15.566968\nThe success of deep learning in identifying complex patterns exceeding human intuition comes at the cost of interpretability. Non-linear entanglement of image features makes deep learning a \"black box\" lacking human meaningful explanations for the models decision. We present DISCOVER, a generative model designed to discover the underlying visual properties driving image-based classification models. DISCOVER learns disentangled latent representations, where each latent feature encodes a unique classification-driving visual property. This design enables \"human-in-the-loop\" interpretation by generating disentangled exaggerated counterfactual explanations. We apply DISCOVER to interpret classification of in-vitro fertilization embryo morphology quality. We quantitatively and systematically confirm the interpretation of known embryo properties, discover properties without previous explicit measurements, and quantitatively determine and empirically verify the classification decision of specific embryo instances. We show that DISCOVER provides human-interpretable understanding of \"black-box\" classification models, proposes hypotheses to decipher underlying biomedical mechanisms, and provides transparency for the classification of individual predictions.\nFine-grained modelling of ATP dependence of decision-making capacity in genetic regulatory networks\nAuthors: Kumar, R.; Johnston, I.\nScore: 4.0, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567352\nCellular decision-making is fundamental to life, from developmental biology to environmental responses and antimicrobial resistance. Many regulatory processes that drive cellular decisions rely on gene expression, which requires energy in the form of ATP. As even genetically identical cells can have dramatically different ATP levels, bioenergetic status can be an important source of variability in cellular decision-making. Existing studies have investigated this energy dependence but often use coarse-grained modelling approaches (which are not always readily connected to the underlying molecular processes of gene regulation). Here we use a fine-grained mathematical model of gene expression in a two-gene decision-making regulatory network to explore cellular decision-making capacity as energy availability varies. We simulate both a deterministic model, to explore the emergence of different cell fate attractors as ATP levels vary, and a stochastic case to explore how ATP influences the noisy dynamics of stochastic cell decision-making. Higher energy levels typically support increased decision-making capacity (higher numbers of, and more separated, cell states that can be selected), and the fine-grained modelling reveals some differences in behaviour from previous coarse-grained modelling approaches.\nSpatial Metabolomics co-registered with Multiplex Phenotyping for the Evaluation of Human Kidney Tissue\nAuthors: Zickuhr, G. M.; Um, I. H.; Laird, A.; Harrison, D. J.; Dickson, A. L.\nScore: 3.2, Published: 2023-11-15 DOI: 10.1101/2023.11.10.566529\nA workflow has been evaluated that utilises a single tissue section to obtain spatially co-registered, molecular, and phenotypical information suitable for AI-enabled analysis. The impact of varying DESI-MSI conditions (e.g., temperature, scan rate, acquisition time) on the detection of small molecules and on tissue quality for integration into typical clinical pathology workflows assessed in human kidney.\nOne-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer\nAuthors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.\nScore: 2.5, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568149\nAs a key mechanism for cellular signal transduction, protein phosphorylation plays critical roles in myriad cellular processes. This modification, however, is highly dynamic and occurs at sub-stoichiometric levels. Mass spectrometry is an essential tool for studying this modification on a global scale; however, the technology's impact suffers from four main limitations: need for site localization, dynamic range, reproducibility, and throughput. Here we describe the use of a novel mass spectrometer (Orbitrap Astral) coupled with data-independent acquisition (DIA) to achieve detection of close to 40,000 unique phosphorylation sites within one hour of analysis. We applied this approach to generate a phosphoproteome atlas of the mouse. Altogether, we detected 81,120 unique phosphorylation sites within 12 hours of measurement. With this unique dataset, we examine the sequence and structural context of protein phosphorylation. Finally, we highlight the discovery potential of this resource with multiple examples of novel phosphorylation events relevant to mitochondrial and brain biology.\nPulsed stimuli entrain p53 to synchronize single cells and modulate cell-fate determination\nAuthors: Venkatachalapathy, H.; Yang, Z.; Azarin, S. M.; Sarkar, C. A.; Batchelor, E.\nScore: 5.5, Published: 2023-11-15 DOI: 10.1101/2023.10.24.563786\nEntrainment to an external stimulus enables a synchronized oscillatory response across a population of cells, increasing coherent responses by reducing cell-to-cell heterogeneity. It is unclear whether the property of entrainability extends to systems where responses are intrinsic to the individual cell, rather than dependent on coherence across a population of cells. Using a combination of mathematical modeling, time-lapse fluorescence microscopy, and single-cell tracking, we demonstrated that p53 oscillations triggered by DNA double-strand breaks (DSBs) can be entrained with a periodic damage stimulus, despite such synchrony not known to function in effective DNA damage responses. Surprisingly, p53 oscillations were experimentally entrained over a wider range of DSB frequencies than predicted by an established computational model for the system. We determined that recapitulating the increased range of entrainment frequencies required, non-intuitively, a less robust oscillator and wider steady-state valley on the energy landscape. Further, we show that p53 entrainment can lead to altered expression dynamics of downstream targets responsible for cell fate in a manner dependent on target mRNA stability. Overall, this study demonstrates that entrainment can occur in a biological oscillator despite the apparent lack of an evolutionary advantage conferred through synchronized responses and highlights the potential of externally entraining p53 dynamics to reduce cellular variability and synchronize cell-fate responses for therapeutic outcomes.\nActive learning for medium optimization for selective bacterial culture\nAuthors: Zhang, S.; Aida, H.; YING, B.-W.\nScore: 1.9, Published: 2023-11-17 DOI: 10.1101/2023.11.15.567258\nMedium optimization and development for selective bacterial culture are essential for isolating and functionalizing individual bacteria in microbial communities; nevertheless, it remains challenging due to the unknown mechanisms between bacterial growth and medium components. The present study first tried combining machine learning (ML) with active learning to finetune the medium components for the selective culture of two divergent bacteria, i.e., Lactobacillus plantarum and Escherichia coli. ML models considering multiple growth parameters of the two bacterial strains were constructed to predict the finetuned medium combinations for higher specificity of bacterial growth. The growth parameters were designed as the exponential growth rate (r) and maximal growth yield (K), which were calculated according to the growth curves. The eleven chemical components in the commercially available medium MRS were subjected to medium optimization and specialization. High-throughput growth assays of both strains grown separately were performed to obtain thousands of growth curves in more than one hundred medium combinations, and the resultant datasets linking the growth parameters to the medium combinations were used for the ML training. Repeated rounds of active learning (i.e., ML model construction, medium prediction, and experimental verification) successfully improved the specific growth of a single strain out of the two. Both r and K showed maximized differentiation between the two strains. Further analysis of all data accumulated in active learning identified the decision-making medium components for growth specificity and the differentiated determinative manner of growth decision of the two strains. In summary, this study demonstrated the efficiency and practicality of active learning in medium optimization for selective culture and offered novel insights into the contribution of the chemical components to specific bacterial growth.\nDifferential responses of primary neuron-secreted MCP-1 and IL-9 to type 2 diabetes and Alzheimer's disease-associated metabolites\nAuthors: Ball, B. K.; Kuhn, M. K.; Fleeman, R. M.; Proctor, E. A.; Brubaker, D. K.\nScore: 1.2, Published: 2023-11-17 DOI: 10.1101/2023.11.17.567595\nType 2 diabetes (T2D) is implicated as a risk factor for Alzheimers disease (AD), the most common form of dementia. In this work, we investigated neuroinflammatory responses of primary neurons to potentially circulating, blood-brain barrier (BBB) permeable metabolites associated with AD, T2D, or both. We identified nine metabolites associated with protective or detrimental properties of AD and T2D in literature (lauric acid, asparagine, fructose, arachidonic acid, aminoadipic acid, sorbitol, retinol, tryptophan, niacinamide) and stimulated primary mouse neuron cultures with each metabolite before quantifying cytokine secretion via Luminex. We employed unsupervised clustering, inferential statistics, and partial least squares discriminant analysis to identify relationships between cytokine concentration and disease-associations of metabolites. We identified MCP-1, a cytokine associated with monocyte recruitment, as differentially abundant between neurons stimulated by metabolites associated with protective and detrimental properties of AD and T2D. We also identified IL-9, a cytokine that promotes mast cell growth, to be differentially associated with T2D. Indeed, cytokines, such as MCP-1 and IL-9, released from neurons in response to BBB-permeable metabolites associated with T2D may contribute to AD development by downstream effects of neuroinflammation.\n",
  "wordCount" : "2126",
  "inLanguage": "en",
  "datePublished": "2023-11-22T10:41:31Z",
  "dateModified": "2023-11-22T10:41:31Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 22, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566649" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566649">
        <p class="paperTitle">Proteome-wide base editor screens to assess phosphorylation site functionality in high-throughput</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kennedy, P. H.; Deh Sheikh, A. A.; Balakar, M.; Jones, A. C.; Olive, M. E.; Hedge, M.; Matias, M. I.; Pirete, N.; Burt, R.; Levy, J.; Little, T.; Hogan, P. G.; Liu, D. R.; Doench, J. G.; Newton, A. C.; Gottschalk, R. A.; de Boer, C.; Alarcon, S.; Newby, G.; Myers, S. A.</p>
        <p class="info">Score: 16.4, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566649' target='https://doi.org/10.1101/2023.11.11.566649'> 10.1101/2023.11.11.566649</a></p>
        <p class="abstract">Signaling pathways that drive gene expression are typically depicted as having a dozen or so landmark phosphorylation and transcriptional events. In reality, thousands of dynamic post-translational modifications (PTMs) orchestrate nearly every cellular function, and we lack technologies to find causal links between these vast biochemical pathways and genetic circuits at scale. Here, we describe &#34;signaling-to-transcription network&#34; mapping through the development of PTM-centric base editing coupled to phenotypic screens, directed by temporally-resolved phosphoproteomics. Using T cell activation as a model, we observe hundreds of unstudied phosphorylation sites that modulate NFAT transcriptional activity. We identify the phosphorylation-mediated nuclear localization of the phosphatase PHLPP1 which promotes NFAT but inhibits NF{kappa}B activity. We also find that specific phosphosite mutants can alter gene expression in subtle yet distinct patterns, demonstrating the potential for fine-tuning transcriptional responses. Overall, base editor screening of PTM sites provides a powerful platform to dissect PTM function within signaling pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2020.10.23.351742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2020.10.23.351742" aria-expanded="false" aria-controls="flush-collapse10.1101/2020.10.23.351742">
        <p class="paperTitle">Synthetic model ecosystem of 12 cryopreservable microbial species allowing for a noninvasive approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2020.10.23.351742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2020.10.23.351742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hosoda, K.; Seno, S.; Murakami, N.; Matsuda, H.; Osada, Y.; Kamiura, R.; Kondoh, M.</p>
        <p class="info">Score: 38.0, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2020.10.23.351742' target='https://doi.org/10.1101/2020.10.23.351742'> 10.1101/2020.10.23.351742</a></p>
        <p class="abstract">Simultaneous understanding of both individual and ecosystem dynamics is crucial in an era marked by the degradation of ecosystem services. Herein, we present a high-throughput synthetic microcosm system comprising 12 functionally and phylogenetically diverse microbial species. These species are axenically culturable, cryopreservable, and can be measured noninvasively via microscopy, aided by machine learning. This system includes prokaryotic and eukaryotic producers and decomposers, and eukaryotic consumers to ensure functional redundancy. Our model system displayed both positive and negative interspecific interactions and higher-order interactions that surpassed the scope of any two-species interaction. Although complete species coexistence was not our primary objective, we identified several conditions under which at least one species from the producers, consumers, and decomposers groups, and one functionally redundant species, persisted for over six months. These conditions set the stage for detailed investigations in the future. Given its designability and experimental replicability, our model ecosystem offers a promising platform for deeper insights into both individual and ecosystem dynamics, including evolution and species interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567372">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567372" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567372">
        <p class="paperTitle">An enzyme activation network provides evidence for extensive regulatory crosstalk between metabolic pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567372" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567372" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alam, M. T.; Al Zubaidi, S. M.; Nasar, M. I.; Ralser, M.; Notebaart, R. A.</p>
        <p class="info">Score: 11.5, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567372' target='https://doi.org/10.1101/2023.11.16.567372'> 10.1101/2023.11.16.567372</a></p>
        <p class="abstract">Enzyme activation by cellular metabolites plays a pivotal role in regulating metabolic processes. Nevertheless, our comprehension of such activation events on a global network scale remains incomplete. In this study, we conducted a comprehensive investigation into the optimization of cell-intrinsic activation interactions within Saccharomyces cerevisiae. To achieve this, we integrated a genome-scale metabolic model with enzyme kinetic data sourced from the BRENDA database. Our objective was to map the distribution of enzyme activators throughout the cellular network. Our findings indicate that virtually all biochemical pathways encompass enzyme activators, frequently originating from disparate pathways, thus revealing extensive regulatory crosstalk between metabolic pathways. Indeed, activators have short pathway lengths, indicating they are activated quickly upon nutrient shifts, and in most instances, these activators target key enzymatic reactions to facilitate downstream metabolic processes. Interestingly, non-essential enzymes exhibit a significantly higher degree of activation compared to their essential counterparts. This observation suggests that cells employ enzyme activators to finely regulate secondary metabolic pathways that are only required under specific conditions. Conversely, the activator metabolites themselves are more likely to be essential components, and their activation levels surpass those of non-essential activators. In summary, our study unveils the widespread importance of enzymatic activators, and suggests that feed-forward activation of conditional metabolic pathways through essential metabolites mediates metabolic plasticity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.566968">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.566968" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.566968">
        <p class="paperTitle">Visual interpretability of image-based classification models by generative latent space disentanglement applied to in vitro fertilization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.566968" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.566968" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rotem, O.; Schwartz, T.; Maor, R.; Tauber, Y.; Tsarfati Shapiro, M.; Meseguer, M.; Gilboa, D.; Seidman, D. S.; Zaritsky, A.</p>
        <p class="info">Score: 8.8, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.566968' target='https://doi.org/10.1101/2023.11.15.566968'> 10.1101/2023.11.15.566968</a></p>
        <p class="abstract">The success of deep learning in identifying complex patterns exceeding human intuition comes at the cost of interpretability. Non-linear entanglement of image features makes deep learning a &#34;black box&#34; lacking human meaningful explanations for the models decision. We present DISCOVER, a generative model designed to discover the underlying visual properties driving image-based classification models. DISCOVER learns disentangled latent representations, where each latent feature encodes a unique classification-driving visual property. This design enables &#34;human-in-the-loop&#34; interpretation by generating disentangled exaggerated counterfactual explanations. We apply DISCOVER to interpret classification of in-vitro fertilization embryo morphology quality. We quantitatively and systematically confirm the interpretation of known embryo properties, discover properties without previous explicit measurements, and quantitatively determine and empirically verify the classification decision of specific embryo instances. We show that DISCOVER provides human-interpretable understanding of &#34;black-box&#34; classification models, proposes hypotheses to decipher underlying biomedical mechanisms, and provides transparency for the classification of individual predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567352" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567352">
        <p class="paperTitle">Fine-grained modelling of ATP dependence of decision-making capacity in genetic regulatory networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumar, R.; Johnston, I.</p>
        <p class="info">Score: 4.0, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567352' target='https://doi.org/10.1101/2023.11.16.567352'> 10.1101/2023.11.16.567352</a></p>
        <p class="abstract">Cellular decision-making is fundamental to life, from developmental biology to environmental responses and antimicrobial resistance. Many regulatory processes that drive cellular decisions rely on gene expression, which requires energy in the form of ATP. As even genetically identical cells can have dramatically different ATP levels, bioenergetic status can be an important source of variability in cellular decision-making. Existing studies have investigated this energy dependence but often use coarse-grained modelling approaches (which are not always readily connected to the underlying molecular processes of gene regulation). Here we use a fine-grained mathematical model of gene expression in a two-gene decision-making regulatory network to explore cellular decision-making capacity as energy availability varies. We simulate both a deterministic model, to explore the emergence of different cell fate attractors as ATP levels vary, and a stochastic case to explore how ATP influences the noisy dynamics of stochastic cell decision-making. Higher energy levels typically support increased decision-making capacity (higher numbers of, and more separated, cell states that can be selected), and the fine-grained modelling reveals some differences in behaviour from previous coarse-grained modelling approaches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566529">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566529" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566529">
        <p class="paperTitle">Spatial Metabolomics co-registered with Multiplex Phenotyping for the Evaluation of Human Kidney Tissue</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566529" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566529" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zickuhr, G. M.; Um, I. H.; Laird, A.; Harrison, D. J.; Dickson, A. L.</p>
        <p class="info">Score: 3.2, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566529' target='https://doi.org/10.1101/2023.11.10.566529'> 10.1101/2023.11.10.566529</a></p>
        <p class="abstract">A workflow has been evaluated that utilises a single tissue section to obtain spatially co-registered, molecular, and phenotypical information suitable for AI-enabled analysis. The impact of varying DESI-MSI conditions (e.g., temperature, scan rate, acquisition time) on the detection of small molecules and on tissue quality for integration into typical clinical pathology workflows assessed in human kidney.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568149">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568149" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568149">
        <p class="paperTitle">One-Hour Phosphoproteome Analysis with the Orbitrap Astral Mass Spectrometer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568149" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568149" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lancaster, N. M.; Sinitcyn, P.; Forny, P.; Peters-Clarke, T. M.; Fecher, C.; Smith, A. J.; Shishkova, E.; Arrey, T. N.; Pashkova, A.; Robinson, M. L.; Arp, N.; Fan, J.; Hansen, J.; Galmozzi, A.; Serrano, L. R.; Westphall, M. S.; Stewart, H.; Hock, C.; Damoc, E.; Pagliarini, D. J.; Zabrouskov, V.; Coon, J. J.</p>
        <p class="info">Score: 2.5, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568149' target='https://doi.org/10.1101/2023.11.21.568149'> 10.1101/2023.11.21.568149</a></p>
        <p class="abstract">As a key mechanism for cellular signal transduction, protein phosphorylation plays critical roles in myriad cellular processes. This modification, however, is highly dynamic and occurs at sub-stoichiometric levels. Mass spectrometry is an essential tool for studying this modification on a global scale; however, the technology&#39;s impact suffers from four main limitations: need for site localization, dynamic range, reproducibility, and throughput. Here we describe the use of a novel mass spectrometer (Orbitrap Astral) coupled with data-independent acquisition (DIA) to achieve detection of close to 40,000 unique phosphorylation sites within one hour of analysis. We applied this approach to generate a phosphoproteome atlas of the mouse. Altogether, we detected 81,120 unique phosphorylation sites within 12 hours of measurement. With this unique dataset, we examine the sequence and structural context of protein phosphorylation. Finally, we highlight the discovery potential of this resource with multiple examples of novel phosphorylation events relevant to mitochondrial and brain biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563786">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563786" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563786">
        <p class="paperTitle">Pulsed stimuli entrain p53 to synchronize single cells and modulate cell-fate determination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563786" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563786" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venkatachalapathy, H.; Yang, Z.; Azarin, S. M.; Sarkar, C. A.; Batchelor, E.</p>
        <p class="info">Score: 5.5, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563786' target='https://doi.org/10.1101/2023.10.24.563786'> 10.1101/2023.10.24.563786</a></p>
        <p class="abstract">Entrainment to an external stimulus enables a synchronized oscillatory response across a population of cells, increasing coherent responses by reducing cell-to-cell heterogeneity. It is unclear whether the property of entrainability extends to systems where responses are intrinsic to the individual cell, rather than dependent on coherence across a population of cells. Using a combination of mathematical modeling, time-lapse fluorescence microscopy, and single-cell tracking, we demonstrated that p53 oscillations triggered by DNA double-strand breaks (DSBs) can be entrained with a periodic damage stimulus, despite such synchrony not known to function in effective DNA damage responses. Surprisingly, p53 oscillations were experimentally entrained over a wider range of DSB frequencies than predicted by an established computational model for the system. We determined that recapitulating the increased range of entrainment frequencies required, non-intuitively, a less robust oscillator and wider steady-state valley on the energy landscape. Further, we show that p53 entrainment can lead to altered expression dynamics of downstream targets responsible for cell fate in a manner dependent on target mRNA stability. Overall, this study demonstrates that entrainment can occur in a biological oscillator despite the apparent lack of an evolutionary advantage conferred through synchronized responses and highlights the potential of externally entraining p53 dynamics to reduce cellular variability and synchronize cell-fate responses for therapeutic outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567258">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567258" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567258">
        <p class="paperTitle">Active learning for medium optimization for selective bacterial culture</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567258" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567258" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, S.; Aida, H.; YING, B.-W.</p>
        <p class="info">Score: 1.9, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567258' target='https://doi.org/10.1101/2023.11.15.567258'> 10.1101/2023.11.15.567258</a></p>
        <p class="abstract">Medium optimization and development for selective bacterial culture are essential for isolating and functionalizing individual bacteria in microbial communities; nevertheless, it remains challenging due to the unknown mechanisms between bacterial growth and medium components. The present study first tried combining machine learning (ML) with active learning to finetune the medium components for the selective culture of two divergent bacteria, i.e., Lactobacillus plantarum and Escherichia coli. ML models considering multiple growth parameters of the two bacterial strains were constructed to predict the finetuned medium combinations for higher specificity of bacterial growth. The growth parameters were designed as the exponential growth rate (r) and maximal growth yield (K), which were calculated according to the growth curves. The eleven chemical components in the commercially available medium MRS were subjected to medium optimization and specialization. High-throughput growth assays of both strains grown separately were performed to obtain thousands of growth curves in more than one hundred medium combinations, and the resultant datasets linking the growth parameters to the medium combinations were used for the ML training. Repeated rounds of active learning (i.e., ML model construction, medium prediction, and experimental verification) successfully improved the specific growth of a single strain out of the two. Both r and K showed maximized differentiation between the two strains. Further analysis of all data accumulated in active learning identified the decision-making medium components for growth specificity and the differentiated determinative manner of growth decision of the two strains. In summary, this study demonstrated the efficiency and practicality of active learning in medium optimization for selective culture and offered novel insights into the contribution of the chemical components to specific bacterial growth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567595">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567595" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567595">
        <p class="paperTitle">Differential responses of primary neuron-secreted MCP-1 and IL-9 to type 2 diabetes and Alzheimer&#39;s disease-associated metabolites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567595" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567595" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ball, B. K.; Kuhn, M. K.; Fleeman, R. M.; Proctor, E. A.; Brubaker, D. K.</p>
        <p class="info">Score: 1.2, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567595' target='https://doi.org/10.1101/2023.11.17.567595'> 10.1101/2023.11.17.567595</a></p>
        <p class="abstract">Type 2 diabetes (T2D) is implicated as a risk factor for Alzheimers disease (AD), the most common form of dementia. In this work, we investigated neuroinflammatory responses of primary neurons to potentially circulating, blood-brain barrier (BBB) permeable metabolites associated with AD, T2D, or both. We identified nine metabolites associated with protective or detrimental properties of AD and T2D in literature (lauric acid, asparagine, fructose, arachidonic acid, aminoadipic acid, sorbitol, retinol, tryptophan, niacinamide) and stimulated primary mouse neuron cultures with each metabolite before quantifying cytokine secretion via Luminex. We employed unsupervised clustering, inferential statistics, and partial least squares discriminant analysis to identify relationships between cytokine concentration and disease-associations of metabolites. We identified MCP-1, a cytokine associated with monocyte recruitment, as differentially abundant between neurons stimulated by metabolites associated with protective and detrimental properties of AD and T2D. We also identified IL-9, a cytokine that promotes mast cell growth, to be differentially associated with T2D. Indeed, cytokines, such as MCP-1 and IL-9, released from neurons in response to BBB-permeable metabolites associated with T2D may contribute to AD development by downstream effects of neuroinflammation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
