<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="APOLLO: A genome-scale metabolic reconstruction resource of 247,092 diverse human microbes spanning multiple continents, age groups, and body sites
Authors: Heinken, A.; Hulshof, T.; Nap, B.; Martinelli, F.; Basile, A.; O&#39;Brolchain, A.; O&#39;Sullivan, N. F.; Gallagher, C.; Magee, E.; McDonagh, F.; Lalor, I.; Bergin, M.; Evans, P.; Daly, R.; Farrell, R.; Delaney, R. M.; Hill, S.; McAuliffe, S. R.; Kilgannon, T.; Fleming, R. M.; Thinnes, C. C.; Thiele, I.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="APOLLO: A genome-scale metabolic reconstruction resource of 247,092 diverse human microbes spanning multiple continents, age groups, and body sites
Authors: Heinken, A.; Hulshof, T.; Nap, B.; Martinelli, F.; Basile, A.; O&#39;Brolchain, A.; O&#39;Sullivan, N. F.; Gallagher, C.; Magee, E.; McDonagh, F.; Lalor, I.; Bergin, M.; Evans, P.; Daly, R.; Farrell, R.; Delaney, R. M.; Hill, S.; McAuliffe, S. R.; Kilgannon, T.; Fleming, R. M.; Thinnes, C. C.; Thiele, I." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="APOLLO: A genome-scale metabolic reconstruction resource of 247,092 diverse human microbes spanning multiple continents, age groups, and body sites
Authors: Heinken, A.; Hulshof, T.; Nap, B.; Martinelli, F.; Basile, A.; O&#39;Brolchain, A.; O&#39;Sullivan, N. F.; Gallagher, C.; Magee, E.; McDonagh, F.; Lalor, I.; Bergin, M.; Evans, P.; Daly, R.; Farrell, R.; Delaney, R. M.; Hill, S.; McAuliffe, S. R.; Kilgannon, T.; Fleming, R. M.; Thinnes, C. C.; Thiele, I."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "APOLLO: A genome-scale metabolic reconstruction resource of 247,092 diverse human microbes spanning multiple continents, age groups, and body sites\nAuthors: Heinken, A.; Hulshof, T.; Nap, B.; Martinelli, F.; Basile, A.; O\u0026#39;Brolchain, A.; O\u0026#39;Sullivan, N. F.; Gallagher, C.; Magee, E.; McDonagh, F.; Lalor, I.; Bergin, M.; Evans, P.; Daly, R.; Farrell, R.; Delaney, R. M.; Hill, S.; McAuliffe, S. R.; Kilgannon, T.; Fleming, R. M.; Thinnes, C. C.; Thiele, I.",
  "keywords": [
    
  ],
  "articleBody": " APOLLO: A genome-scale metabolic reconstruction resource of 247,092 diverse human microbes spanning multiple continents, age groups, and body sites\nAuthors: Heinken, A.; Hulshof, T.; Nap, B.; Martinelli, F.; Basile, A.; O'Brolchain, A.; O'Sullivan, N. F.; Gallagher, C.; Magee, E.; McDonagh, F.; Lalor, I.; Bergin, M.; Evans, P.; Daly, R.; Farrell, R.; Delaney, R. M.; Hill, S.; McAuliffe, S. R.; Kilgannon, T.; Fleming, R. M.; Thinnes, C. C.; Thiele, I.\nScore: 18.6, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560573\nComputational modelling of microbiome metabolism has proved instrumental to catalyse our understanding of diet-host-microbiome-disease interactions through the interrogation of mechanistic, strain- and molecule-resolved metabolic models. We present APOLLO, a resource of 247,092 human microbial genome-scale metabolic reconstructions spanning 19 phyla and accounting for microbial genomes from 34 countries, all age groups, and five body sites. We explored the metabolic potential of the reconstructed strains and developed a machine learning classifier able to predict with high accuracy the taxonomic strain assignments. We also built 14,451 sample-specific microbial community models, which could be stratified by body site, age, and disease states. Finally, we predicted faecal metabolites enriched or depleted in gut microbiomes of people with Crohns disease, Parkinson disease, and undernourished children. APOLLO is compatible with the human whole-body models, and thus, provide unprecedented opportunities for systems-level modelling of personalised host-microbiome co-metabolism. APOLLO will be freely available under https://www.vmh.life/.\nModelSEED v2: High-throughput genome-scale metabolic model reconstruction with enhanced energy biosynthesis pathway prediction\nAuthors: Faria, J. P.; Liu, F.; Edirisinghe, J. N.; Gupta, N.; Seaver, S. M. D.; Freiburger, A. P.; Zhang, Q.; Weisenhorn, P.; Conrad, N.; Zarecki, R.; Song, H.-S.; DeJongh, M.; Best, A. A.; Cottingham, R. W.; Arkin, A. P.; Henry, C. S.\nScore: 5.7, Published: 2023-10-06 DOI: 10.1101/2023.10.04.556561\nSince the release of ModelSEED in 2010, the systems biology research community has used the ModelSEED genome-scale metabolic model reconstruction pipeline to build over 200,000 draft metabolic reconstructions that support hundreds of publications. Here we describe the first comprehensive update to this reconstruction tool, with new features such as (i) a dramatically improved representation of energy metabolism, which ensures that models produce accurate amounts of ATP per mol of nutrient consumed; (ii) a new template for Archaea model reconstruction; and (iii) a significantly improved curation of all metabolic pathways with mappings to RAST subsystems annotations. We applied the improved pipeline to build and analyze core and genome-scale models for Archaea and Bacteria genomes in KEGG. The new ModelSEED pipeline generates larger models that possess more reactions and genes and require fewer gapfilled reactions. In addition, we see conserved patterns in the ATP biosynthesis mechanism across phylogeny, and identify clades where our understanding of energy biosynthesis is still poor. The ModelSEED v2 pipeline is currently available only as new reconstruction and gap-filling Appsin the KBase platform.\nA new universal system of tree shape indices\nAuthors: Noble, R. J.; Verity, K.\nScore: 9.4, Published: 2023-09-19 DOI: 10.1101/2023.07.17.549219\nThe comparison and categorization of tree diagrams is fundamental to large parts of biology, linguistics, computer science, and other fields, yet the indices currently applied to describing tree shape have important flaws that complicate their interpretation and limit their scope. Here we introduce a new system of indices with no such shortcomings. Our indices account for node sizes and branch lengths and are robust to small changes in either attribute. Unlike currently popular phylogenetic diversity, phylogenetic entropy, and tree balance indices, our definitions assign interpretable values to all rooted trees and enable meaningful comparison of any pair of trees. Our self-consistent definitions further unite measures of diversity, richness, balance, symmetry, effective height, effective outdegree, and effective branch count in a coherent system, and we derive numerous simple relationships between these indices. The main practical advantages of our indices are in 1) quantifying diversity in non-ultrametric trees; 2) assessing the balance of trees that have non-uniform branch lengths or node sizes; 3) comparing the balance of trees with different leaf counts or outdegrees; 4) obtaining a coherent, generic, multidimensional quantification of tree shape that is robust to sampling error and inferential error. We illustrate these features by comparing the shapes of trees representing the evolution of HIV and of Uralic languages. Given the ubiquity of tree structures, we identify a wide range of applications across diverse domains.\nWidespread transcriptional regulation from within transcribed regions in plants\nAuthors: Voichek, Y.; Hristova, G.; Molla-Morales, A.; Weigel, D.; Nordborg, M.\nScore: 112.8, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557872\nAnimals and plants have evolved separately for over 1.5 billion years and independently invented multicellularity, suggesting a possible divergence in their mode of transcription regulation. Here, we set out to elucidate fundamental features of transcription regulatory sequences in plants. Using a massively parallel reporter assay in four species, we show that sequences downstream of the transcription start site (TSS) play a major role in controlling transcription. Swapping regulatory sequences from one side of the TSS to the other yields different outcomes, unlike animal enhancers which act independently of their position. A GATC-containing DNA motif, positioned downstream of the TSS, was sufficient to enhance gene expression in a dose-dependent manner. Its effect on gene expression was tissue-dependent and conserved across vascular plants. These results identify a unique characteristic of transcriptional regulation in plants, and suggest fundamental differences in gene regulation might exist between plants and animals.\nSimultaneous estimation of gene regulatory network structure and RNA kinetics from single cell gene expression\nAuthors: Jackson, C. A.; Beheler-Amass, M.; Tjärnberg, A.; Suresh, I.; Hickey, A. S.-m.; Bonneau, R.; Gresham, D. J.\nScore: 3.0, Published: 2023-09-23 DOI: 10.1101/2023.09.21.558277\nCells respond to environmental and developmental stimuli by remodeling their transcriptomes through regulation of both mRNA transcription and mRNA decay. A central goal of biology is identifying the global set of regulatory relationships between factors that control mRNA production and degradation and their target transcripts and construct a predictive model of gene expression. Regulatory relationships are typically identified using transcriptome measurements and causal inference algorithms. RNA kinetic parameters are determined experimentally by employing run-on or metabolic labeling (e.g. 4-thiouracil) methods that allow transcription and decay rates to be separately measured. Here, we develop a deep learning model, trained with single-cell RNA-seq data, that both infers causal regulatory relationships and estimates RNA kinetic parameters. The resulting in silico model predicts future gene expression states and can be perturbed to simulate the effect of transcription factor changes. We acquired model training data by sequencing the transcriptomes of 175,000 individual Saccharomyces cerevisiae cells that were subject to an external perturbation and continuously sampled over a one hour period. The rate of change for each transcript was calculated on a per-cell basis to estimate RNA velocity. We then trained a deep learning model with transcriptome and RNA velocity data to calculate time-dependent estimates of mRNA production and decay rates. By separating RNA velocity into transcription and decay rates, we show that rapamycin treatment causes existing ribosomal protein transcripts to be rapidly destabilized, while production of new transcripts gradually slows over the course of an hour. The neural network framework we present is designed to explicitly model causal regulatory relationships between transcription factors and their genes, and shows superior performance to existing models on the basis of recovery of known regulatory relationships. We validated the predictive power of the model by perturbing transcription factors in silico and comparing transcriptome-wide effects with experimental data. Our study represents the first step in constructing a complete, predictive, biophysical model of gene expression regulation.\nMulti-modal transcriptomic analysis unravels enrichment of hybrid epithelial/mesenchymal state and enhanced phenotypic heterogeneity in basal breast cancer\nAuthors: Sahoo, S.; Ramu, S.; Nair, M. G.; Pillai, M.; Perez San Juan, B.; Milioli, H. Z.; Mandal, S.; Naidu, C. M.; Mavatkar, A. D.; Subramaniam, H.; Neogi, A. G.; Chaffer, C. L.; Prabhu, J. S.; Somarelli, J. A.; Jolly, M. K.\nScore: 2.0, Published: 2023-10-02 DOI: 10.1101/2023.09.30.558960\nIntra-tumoral phenotypic heterogeneity promotes tumor relapse and therapeutic resistance and remains an unsolved clinical challenge. It manifests along multiple phenotypic axes and decoding the interconnections among these different axes is crucial to understand its molecular origins and to develop novel therapeutic strategies to control it. Here, we use multi-modal transcriptomic data analysis - bulk, single-cell and spatial transcriptomics - from breast cancer cell lines and primary tumor samples, to identify associations between epithelial-mesenchymal transition (EMT) and luminal-basal plasticity - two key processes that enable heterogeneity. We show that luminal breast cancer strongly associates with an epithelial cell state, but basal breast cancer is associated with hybrid epithelial/mesenchymal phenotype(s) and higher phenotypic heterogeneity. These patterns were inherent in methylation profiles, suggesting an epigenetic crosstalk between EMT and lineage plasticity in breast cancer. Mathematical modelling of core underlying gene regulatory networks representative of the crosstalk between the luminal-basal and epithelial-mesenchymal axes recapitulate and thus elucidate mechanistic underpinnings of the observed associations from transcriptomic data. Our systems-based approach integrating multi-modal data analysis with mechanism-based modeling offers a predictive framework to characterize intra-tumor heterogeneity and to identify possible interventions to restrict it.\nThe EFFECT benchmark suite: measuring cancer sensitivity prediction performance - without the bias\nAuthors: Szalai, B.; Gaspar, I.; Kaszas, V.; Mero, L.; Sztilkovics, M.; Szalay, K. Z.\nScore: 1.9, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560281\nCreating computational biology models applicable to industry is much more difficult than it appears. There is a major gap between a model that looks good on paper and a model that performs well in the drug discovery process. We are trying to shrink this gap by introducing the Evaluation Framework For predicting Efficiency of Cancer Treatment (EFFECT) benchmark suite based on the DepMap and GDSC data sets to facilitate the creation of well-applicable machine learning models capable of predicting gene essentiality and/or drug sensitivity on in vitro cancer cell lines. We show that standard evaluation metrics like Pearson correlation are easily misled by inherent biases in the data. Thus, to assess the performance of models properly, we propose the use of cell line/perturbation exclusive data splits, perturbation-wise evaluation, and the application of our Bias Detector framework, which can identify model predictions not explicable by data bias alone. Testing the EFFECT suite on a few popular ML models showed that while library-standard non-linear models have measurable performance in splits representing precision medicine and target identification tasks, the actual corrected correlations are rather low, showing that even simple knock-out/drug sensitivity prediction is a yet unsolved task. For this reason, we aim our proposed framework to be a unified test and evaluation pipeline for machine learning models predicting cancer sensitivity data, facilitating unbiased benchmarking to support teams to improve on the state of the art.\nIntegrating microbial abundance time series with fermentation dynamics of the rumen microbiome via mathematical modelling\nAuthors: Davoudkhani, M.; Rubino, F.; Creevey, C. J.; Ahvenjarvi, S.; Bayat, A. R.; Tapio, I.; Belanche, A.; Munoz-Tamayo, R.\nScore: 1.8, Published: 2023-10-06 DOI: 10.1101/2023.10.05.560454\nThe rumen represents a dynamic microbial ecosystem where fermentation metabolites and microbial concentrations change over time in response to dietary changes. The integration of microbial genomic knowledge and dynamic modelling can enhance our system-level understanding of rumen ecosystem's function. However, such an integration between dynamic models and rumen microbiota data is lacking. The objective of this work was to integrate rumen microbiota time series determined by 16S rRNA gene amplicon sequencing into a dynamic modelling framework to link microbial data to the dynamics of the volatile fatty acids (VFA) production during fermentation. For that, we used the theory of state observers to develop a model that estimates the dynamics of VFA from the data of microbial functional proxies associated with the specific production of each VFA. We determined the microbial proxies using CowPi to infer the functional potential of the rumen microbiota and extrapolate their functional modules from KEGG (Kyoto Encyclopedia of Genes and Genomes). The approach was challenged using data from an in vitro RUSITEC experiment and from an in vivo experiment with four cows. The model performance was evaluated by the coefficient of variation of the root mean square error (CRMSE). For the in vitro case study, the mean CVRMSE were 9.8% for acetate, 14% for butyrate and 14.5% for propionate. For the in vivo case study, the mean CVRMSE were 16.4% for acetate, 15.8% for butyrate and 19.8% for propionate. The mean CVRMSE for the VFA molar fractions were 3.1% for acetate, 3.8% for butyrate and 8.9% for propionate. Ours results show the promising application of state observers integrated with microbiota time series data for predicting rumen microbial metabolism.\nCompression of sickspan by interventions that steepen the survival curve\nAuthors: Yang, Y.; Mayo, A.; Levy, T.; Jarosz, D. F.; Alon, U.\nScore: 1.2, Published: 2023-10-05 DOI: 10.1101/2023.10.04.560871\nLongevity research aims to enhance the period of healthy life, known as the healthspan, while minimizing the duration of disability and morbidity, known as the sickspan. Various interventions extend lifespan in model organisms, but it is not known which of these are likely to compress the relative sickspan. Here, we present a theory that predicts which interventions effectively compress the sickspan. The theory is based on the shape of the survival curve - the fraction of organisms surviving as a function of age. Interventions that lengthen mean lifespan while preserving the shape of the survival curve, known as scaling, are hypothesized to extend the sickspan proportionally, without compressing it. Such interventions include caloric restriction and most other known longevity interventions. Conversely, a small subset of interventions that extend lifespan and alter the shape of the survival curve, particularly by steepening it, are predicted to compress the sickspan. We provide intuitive explanations for this theory and demonstrate its origin from the saturated removal model, a recent mathematical aging model calibrated in mice. We discuss specific longevity interventions in mice that steepen the survival curves, including senolytics, ketogenic diet, and agents that reduce glucose spikes and protect blood vessels, as potential candidates for compressing the sickspan. This hypothesis offers insights into the potential strategies for achieving compressed morbidity and extending healthspan.\nRepresentational Learning from Healthy Multi-Tissue Human RNA-seq Data such that Latent Space Arithmetics Extracts Disease Modules\nAuthors: de Weerd, H. A.; Guala, D.; Gustafsson, M.; Synnergren, J.; Tegner, J.; Lubovac-Pilav, Z.; Magnusson, R.\nScore: 1.0, Published: 2023-10-05 DOI: 10.1101/2023.10.03.560661\nDeveloping computational analyses of transcriptomic data has dramatically improved our understanding of complex multifactorial diseases. However, such approaches are limited to small sample sets of disease-affected material, thus being sensitive to statistical biases and noise. Here, we ask if a variational autoencoder (VAE) trained on large groups of healthy, human RNA-seq data of multiple tissues can capture the fundamental healthy gene regulation system such that the learned representation generalizes to account for unseen disease changes. To this end, we trained a multi-scale representation to encode cellular processes ranging from cell types to gene-gene interactions. Importantly, we found that the learned healthy representations could predict unseen gene expression changes from 25 independent disease datasets. We extracted and decoded disease-specific signals from the VAE latent space to dissect this finding. Interestingly, the gene modules corresponding to this signal contained more disease-specific genes than the respective differential expression analysis in 20 of 25 cases. Finally, we matched genes related to the disease signals to known drug targets. We could extract sets of known and potential pharmaceutical candidates from this analysis and demonstrate the utility in three use cases. In summary, our study showcases how data-driven representation learning using a VAE as a foundational model allows an arithmetic deconstruction of the latent space such that biological insights enable the dissection of disease mechanisms and drug targets. Our model is available at https://github.com/ddeweerd/VAE Transcriptomics/.\n",
  "wordCount" : "2574",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560573">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560573" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560573">
        <p class="paperTitle">APOLLO: A genome-scale metabolic reconstruction resource of 247,092 diverse human microbes spanning multiple continents, age groups, and body sites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560573" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560573" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heinken, A.; Hulshof, T.; Nap, B.; Martinelli, F.; Basile, A.; O&#39;Brolchain, A.; O&#39;Sullivan, N. F.; Gallagher, C.; Magee, E.; McDonagh, F.; Lalor, I.; Bergin, M.; Evans, P.; Daly, R.; Farrell, R.; Delaney, R. M.; Hill, S.; McAuliffe, S. R.; Kilgannon, T.; Fleming, R. M.; Thinnes, C. C.; Thiele, I.</p>
        <p class="info">Score: 18.6, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560573' target='https://doi.org/10.1101/2023.10.02.560573'> 10.1101/2023.10.02.560573</a></p>
        <p class="abstract">Computational modelling of microbiome metabolism has proved instrumental to catalyse our understanding of diet-host-microbiome-disease interactions through the interrogation of mechanistic, strain- and molecule-resolved metabolic models. We present APOLLO, a resource of 247,092 human microbial genome-scale metabolic reconstructions spanning 19 phyla and accounting for microbial genomes from 34 countries, all age groups, and five body sites. We explored the metabolic potential of the reconstructed strains and developed a machine learning classifier able to predict with high accuracy the taxonomic strain assignments. We also built 14,451 sample-specific microbial community models, which could be stratified by body site, age, and disease states. Finally, we predicted faecal metabolites enriched or depleted in gut microbiomes of people with Crohns disease, Parkinson disease, and undernourished children. APOLLO is compatible with the human whole-body models, and thus, provide unprecedented opportunities for systems-level modelling of personalised host-microbiome co-metabolism. APOLLO will be freely available under https://www.vmh.life/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.556561">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.556561" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.556561">
        <p class="paperTitle">ModelSEED v2: High-throughput genome-scale metabolic model reconstruction with enhanced energy biosynthesis pathway prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.556561" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.556561" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faria, J. P.; Liu, F.; Edirisinghe, J. N.; Gupta, N.; Seaver, S. M. D.; Freiburger, A. P.; Zhang, Q.; Weisenhorn, P.; Conrad, N.; Zarecki, R.; Song, H.-S.; DeJongh, M.; Best, A. A.; Cottingham, R. W.; Arkin, A. P.; Henry, C. S.</p>
        <p class="info">Score: 5.7, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.556561' target='https://doi.org/10.1101/2023.10.04.556561'> 10.1101/2023.10.04.556561</a></p>
        <p class="abstract">Since the release of ModelSEED in 2010, the systems biology research community has used the ModelSEED genome-scale metabolic model reconstruction pipeline to build over 200,000 draft metabolic reconstructions that support hundreds of publications. Here we describe the first comprehensive update to this reconstruction tool, with new features such as (i) a dramatically improved representation of energy metabolism, which ensures that models produce accurate amounts of ATP per mol of nutrient consumed; (ii) a new template for Archaea model reconstruction; and (iii) a significantly improved curation of all metabolic pathways with mappings to RAST subsystems annotations. We applied the improved pipeline to build and analyze core and genome-scale models for Archaea and Bacteria genomes in KEGG. The new ModelSEED pipeline generates larger models that possess more reactions and genes and require fewer gapfilled reactions. In addition, we see conserved patterns in the ATP biosynthesis mechanism across phylogeny, and identify clades where our understanding of energy biosynthesis is still poor. The ModelSEED v2 pipeline is currently available only as new reconstruction and gap-filling Appsin the KBase platform.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549219" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549219">
        <p class="paperTitle">A new universal system of tree shape indices</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Noble, R. J.; Verity, K.</p>
        <p class="info">Score: 9.4, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549219' target='https://doi.org/10.1101/2023.07.17.549219'> 10.1101/2023.07.17.549219</a></p>
        <p class="abstract">The comparison and categorization of tree diagrams is fundamental to large parts of biology, linguistics, computer science, and other fields, yet the indices currently applied to describing tree shape have important flaws that complicate their interpretation and limit their scope. Here we introduce a new system of indices with no such shortcomings. Our indices account for node sizes and branch lengths and are robust to small changes in either attribute. Unlike currently popular phylogenetic diversity, phylogenetic entropy, and tree balance indices, our definitions assign interpretable values to all rooted trees and enable meaningful comparison of any pair of trees. Our self-consistent definitions further unite measures of diversity, richness, balance, symmetry, effective height, effective outdegree, and effective branch count in a coherent system, and we derive numerous simple relationships between these indices. The main practical advantages of our indices are in 1) quantifying diversity in non-ultrametric trees; 2) assessing the balance of trees that have non-uniform branch lengths or node sizes; 3) comparing the balance of trees with different leaf counts or outdegrees; 4) obtaining a coherent, generic, multidimensional quantification of tree shape that is robust to sampling error and inferential error. We illustrate these features by comparing the shapes of trees representing the evolution of HIV and of Uralic languages. Given the ubiquity of tree structures, we identify a wide range of applications across diverse domains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557872">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557872" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557872">
        <p class="paperTitle">Widespread transcriptional regulation from within transcribed regions in plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557872" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557872" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Voichek, Y.; Hristova, G.; Molla-Morales, A.; Weigel, D.; Nordborg, M.</p>
        <p class="info">Score: 112.8, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557872' target='https://doi.org/10.1101/2023.09.15.557872'> 10.1101/2023.09.15.557872</a></p>
        <p class="abstract">Animals and plants have evolved separately for over 1.5 billion years and independently invented multicellularity, suggesting a possible divergence in their mode of transcription regulation. Here, we set out to elucidate fundamental features of transcription regulatory sequences in plants. Using a massively parallel reporter assay in four species, we show that sequences downstream of the transcription start site (TSS) play a major role in controlling transcription. Swapping regulatory sequences from one side of the TSS to the other yields different outcomes, unlike animal enhancers which act independently of their position. A GATC-containing DNA motif, positioned downstream of the TSS, was sufficient to enhance gene expression in a dose-dependent manner. Its effect on gene expression was tissue-dependent and conserved across vascular plants. These results identify a unique characteristic of transcriptional regulation in plants, and suggest fundamental differences in gene regulation might exist between plants and animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558277">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558277" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558277">
        <p class="paperTitle">Simultaneous estimation of gene regulatory network structure and RNA kinetics from single cell gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558277" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558277" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jackson, C. A.; Beheler-Amass, M.; Tjärnberg, A.; Suresh, I.; Hickey, A. S.-m.; Bonneau, R.; Gresham, D. J.</p>
        <p class="info">Score: 3.0, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558277' target='https://doi.org/10.1101/2023.09.21.558277'> 10.1101/2023.09.21.558277</a></p>
        <p class="abstract">Cells respond to environmental and developmental stimuli by remodeling their transcriptomes through regulation of both mRNA transcription and mRNA decay. A central goal of biology is identifying the global set of regulatory relationships between factors that control mRNA production and degradation and their target transcripts and construct a predictive model of gene expression. Regulatory relationships are typically identified using transcriptome measurements and causal inference algorithms. RNA kinetic parameters are determined experimentally by employing run-on or metabolic labeling (e.g. 4-thiouracil) methods that allow transcription and decay rates to be separately measured. Here, we develop a deep learning model, trained with single-cell RNA-seq data, that both infers causal regulatory relationships and estimates RNA kinetic parameters. The resulting in silico model predicts future gene expression states and can be perturbed to simulate the effect of transcription factor changes.

We acquired model training data by sequencing the transcriptomes of 175,000 individual Saccharomyces cerevisiae cells that were subject to an external perturbation and continuously sampled over a one hour period. The rate of change for each transcript was calculated on a per-cell basis to estimate RNA velocity. We then trained a deep learning model with transcriptome and RNA velocity data to calculate time-dependent estimates of mRNA production and decay rates. By separating RNA velocity into transcription and decay rates, we show that rapamycin treatment causes existing ribosomal protein transcripts to be rapidly destabilized, while production of new transcripts gradually slows over the course of an hour.

The neural network framework we present is designed to explicitly model causal regulatory relationships between transcription factors and their genes, and shows superior performance to existing models on the basis of recovery of known regulatory relationships. We validated the predictive power of the model by perturbing transcription factors in silico and comparing transcriptome-wide effects with experimental data. Our study represents the first step in constructing a complete, predictive, biophysical model of gene expression regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.558960">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.558960" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.558960">
        <p class="paperTitle">Multi-modal transcriptomic analysis unravels enrichment of hybrid epithelial/mesenchymal state and enhanced phenotypic heterogeneity in basal breast cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.558960" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.558960" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sahoo, S.; Ramu, S.; Nair, M. G.; Pillai, M.; Perez San Juan, B.; Milioli, H. Z.; Mandal, S.; Naidu, C. M.; Mavatkar, A. D.; Subramaniam, H.; Neogi, A. G.; Chaffer, C. L.; Prabhu, J. S.; Somarelli, J. A.; Jolly, M. K.</p>
        <p class="info">Score: 2.0, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.558960' target='https://doi.org/10.1101/2023.09.30.558960'> 10.1101/2023.09.30.558960</a></p>
        <p class="abstract">Intra-tumoral phenotypic heterogeneity promotes tumor relapse and therapeutic resistance and remains an unsolved clinical challenge. It manifests along multiple phenotypic axes and decoding the interconnections among these different axes is crucial to understand its molecular origins and to develop novel therapeutic strategies to control it. Here, we use multi-modal transcriptomic data analysis - bulk, single-cell and spatial transcriptomics - from breast cancer cell lines and primary tumor samples, to identify associations between epithelial-mesenchymal transition (EMT) and luminal-basal plasticity - two key processes that enable heterogeneity. We show that luminal breast cancer strongly associates with an epithelial cell state, but basal breast cancer is associated with hybrid epithelial/mesenchymal phenotype(s) and higher phenotypic heterogeneity. These patterns were inherent in methylation profiles, suggesting an epigenetic crosstalk between EMT and lineage plasticity in breast cancer. Mathematical modelling of core underlying gene regulatory networks representative of the crosstalk between the luminal-basal and epithelial-mesenchymal axes recapitulate and thus elucidate mechanistic underpinnings of the observed associations from transcriptomic data. Our systems-based approach integrating multi-modal data analysis with mechanism-based modeling offers a predictive framework to characterize intra-tumor heterogeneity and to identify possible interventions to restrict it.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560281">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560281" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560281">
        <p class="paperTitle">The EFFECT benchmark suite: measuring cancer sensitivity prediction performance - without the bias</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560281" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560281" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Szalai, B.; Gaspar, I.; Kaszas, V.; Mero, L.; Sztilkovics, M.; Szalay, K. Z.</p>
        <p class="info">Score: 1.9, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560281' target='https://doi.org/10.1101/2023.10.02.560281'> 10.1101/2023.10.02.560281</a></p>
        <p class="abstract">Creating computational biology models applicable to industry is much more difficult than it appears. There is a major gap between a model that looks good on paper and a model that performs well in the drug discovery process. We are trying to shrink this gap by introducing the Evaluation Framework For predicting Efficiency of Cancer Treatment (EFFECT) benchmark suite based on the DepMap and GDSC data sets to facilitate the creation of well-applicable machine learning models capable of predicting gene essentiality and/or drug sensitivity on in vitro cancer cell lines. We show that standard evaluation metrics like Pearson correlation are easily misled by inherent biases in the data. Thus, to assess the performance of models properly, we propose the use of cell line/perturbation exclusive data splits, perturbation-wise evaluation, and the application of our Bias Detector framework, which can identify model predictions not explicable by data bias alone. Testing the EFFECT suite on a few popular ML models showed that while library-standard non-linear models have measurable performance in splits representing precision medicine and target identification tasks, the actual corrected correlations are rather low, showing that even simple knock-out/drug sensitivity prediction is a yet unsolved task. For this reason, we aim our proposed framework to be a unified test and evaluation pipeline for machine learning models predicting cancer sensitivity data, facilitating unbiased benchmarking to support teams to improve on the state of the art.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.560454">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.560454" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.560454">
        <p class="paperTitle">Integrating microbial abundance time series with fermentation dynamics of the rumen microbiome via mathematical modelling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.560454" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.560454" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davoudkhani, M.; Rubino, F.; Creevey, C. J.; Ahvenjarvi, S.; Bayat, A. R.; Tapio, I.; Belanche, A.; Munoz-Tamayo, R.</p>
        <p class="info">Score: 1.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.560454' target='https://doi.org/10.1101/2023.10.05.560454'> 10.1101/2023.10.05.560454</a></p>
        <p class="abstract">The rumen represents a dynamic microbial ecosystem where fermentation metabolites and microbial concentrations change over time in response to dietary changes. The integration of microbial genomic knowledge and dynamic modelling can enhance our system-level understanding of rumen ecosystem&#39;s function. However, such an integration between dynamic models and rumen microbiota data is lacking. The objective of this work was to integrate rumen microbiota time series determined by 16S rRNA gene amplicon sequencing into a dynamic modelling framework to link microbial data to the dynamics of the volatile fatty acids (VFA) production during fermentation. For that, we used the theory of state observers to develop a model that estimates the dynamics of VFA from the data of microbial functional proxies associated with the specific production of each VFA. We determined the microbial proxies using CowPi to infer the functional potential of the rumen microbiota and extrapolate their functional modules from KEGG (Kyoto Encyclopedia of Genes and Genomes). The approach was challenged using data from an in vitro RUSITEC experiment and from an in vivo experiment with four cows. The model performance was evaluated by the coefficient of variation of the root mean square error (CRMSE). For the in vitro case study, the mean CVRMSE were 9.8% for acetate, 14% for butyrate and 14.5% for propionate. For the in vivo case study, the mean CVRMSE were 16.4% for acetate, 15.8% for butyrate and 19.8% for propionate. The mean CVRMSE for the VFA molar fractions were 3.1% for acetate, 3.8% for butyrate and 8.9% for propionate. Ours results show the promising application of state observers integrated with microbiota time series data for predicting rumen microbial metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560871">
        <p class="paperTitle">Compression of sickspan by interventions that steepen the survival curve</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, Y.; Mayo, A.; Levy, T.; Jarosz, D. F.; Alon, U.</p>
        <p class="info">Score: 1.2, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560871' target='https://doi.org/10.1101/2023.10.04.560871'> 10.1101/2023.10.04.560871</a></p>
        <p class="abstract">Longevity research aims to enhance the period of healthy life, known as the healthspan, while minimizing the duration of disability and morbidity, known as the sickspan. Various interventions extend lifespan in model organisms, but it is not known which of these are likely to compress the relative sickspan. Here, we present a theory that predicts which interventions effectively compress the sickspan. The theory is based on the shape of the survival curve - the fraction of organisms surviving as a function of age. Interventions that lengthen mean lifespan while preserving the shape of the survival curve, known as scaling, are hypothesized to extend the sickspan proportionally, without compressing it. Such interventions include caloric restriction and most other known longevity interventions. Conversely, a small subset of interventions that extend lifespan and alter the shape of the survival curve, particularly by steepening it, are predicted to compress the sickspan. We provide intuitive explanations for this theory and demonstrate its origin from the saturated removal model, a recent mathematical aging model calibrated in mice. We discuss specific longevity interventions in mice that steepen the survival curves, including senolytics, ketogenic diet, and agents that reduce glucose spikes and protect blood vessels, as potential candidates for compressing the sickspan. This hypothesis offers insights into the potential strategies for achieving compressed morbidity and extending healthspan.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560661">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560661" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560661">
        <p class="paperTitle">Representational Learning from Healthy Multi-Tissue Human RNA-seq Data such that Latent Space Arithmetics Extracts Disease Modules</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560661" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560661" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: de Weerd, H. A.; Guala, D.; Gustafsson, M.; Synnergren, J.; Tegner, J.; Lubovac-Pilav, Z.; Magnusson, R.</p>
        <p class="info">Score: 1.0, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560661' target='https://doi.org/10.1101/2023.10.03.560661'> 10.1101/2023.10.03.560661</a></p>
        <p class="abstract">Developing computational analyses of transcriptomic data has dramatically improved our understanding of complex multifactorial diseases. However, such approaches are limited to small sample sets of disease-affected material, thus being sensitive to statistical biases and noise. Here, we ask if a variational autoencoder (VAE) trained on large groups of healthy, human RNA-seq data of multiple tissues can capture the fundamental healthy gene regulation system such that the learned representation generalizes to account for unseen disease changes. To this end, we trained a multi-scale representation to encode cellular processes ranging from cell types to gene-gene interactions. Importantly, we found that the learned healthy representations could predict unseen gene expression changes from 25 independent disease datasets. We extracted and decoded disease-specific signals from the VAE latent space to dissect this finding. Interestingly, the gene modules corresponding to this signal contained more disease-specific genes than the respective differential expression analysis in 20 of 25 cases. Finally, we matched genes related to the disease signals to known drug targets. We could extract sets of known and potential pharmaceutical candidates from this analysis and demonstrate the utility in three use cases. In summary, our study showcases how data-driven representation learning using a VAE as a foundational model allows an arithmetic deconstruction of the latent space such that biological insights enable the dissection of disease mechanisms and drug targets. Our model is available at https://github.com/ddeweerd/VAE Transcriptomics/.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
