<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Allometry of cell types in planarians by single cell transcriptomics
Authors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.
Score: 64.1, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140
Allometry explores the relationship between an organisms body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Allometry of cell types in planarians by single cell transcriptomics
Authors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.
Score: 64.1, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140
Allometry explores the relationship between an organisms body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-12T10:37:43+00:00" />
<meta property="article:modified_time" content="2023-11-12T10:37:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Allometry of cell types in planarians by single cell transcriptomics
Authors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.
Score: 64.1, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140
Allometry explores the relationship between an organisms body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Allometry of cell types in planarians by single cell transcriptomics\nAuthors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.\nScore: 64.1, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140\nAllometry explores the relationship between an organisms body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry.",
  "keywords": [
    
  ],
  "articleBody": " Allometry of cell types in planarians by single cell transcriptomics\nAuthors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.\nScore: 64.1, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140\nAllometry explores the relationship between an organisms body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry. Planarians, capable of growing and degrowing following allometric scaling rules, serve as an excellent model for such studies. We used scRNA-seq to examine cell type allometry in asexual planarians of different sizes, revealing that they consist of the same basic cell types but in varying proportions. Notably, the gut basal cells are the most responsive to changes in size, suggesting a role in energy storage. We capture the gene regulatory programs of distinct cell types in response to size. This research sheds light on the molecular and cellular aspects of cell type allometry in planarians and underscores the utility of scRNA-seq in such investigations.\nPrinciples of Computation by Competitive Protein Dimerization Networks\nAuthors: Parres-Gold, J.; Levine, M.; Emert, B.; Stuart, A.; Elowitz, M.\nScore: 50.1, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564854\nMany biological signaling pathways employ proteins that competitively dimerize in diverse combinations. These dimerization networks can perform biochemical computations, in which the concentrations of monomers (inputs) determine the concentrations of dimers (outputs). Despite their prevalence, little is known about the range of input-output computations that dimerization networks can perform (their \"expressivity\") and how it depends on network size and connectivity. Using a systematic computational approach, we demonstrate that even small dimerization networks (3-6 monomers) are expressive, performing diverse multi-input computations. Further, dimerization networks are versatile, performing different computations when their protein components are expressed at different levels, such as in different cell types. Remarkably, individual networks with random interaction affinities, when large enough ([\u0026ge;]8 proteins), can perform nearly all ([~]90%) potential one-input network computations merely by tuning their monomer expression levels. Thus, even the simple process of competitive dimerization provides a powerful architecture for multi-input, cell-type-specific signal processing. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=192 SRC=\"FIGDIR/small/564854v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (35K): org.highwire.dtl.DTLVardef@1c906faorg.highwire.dtl.DTLVardef@e948a0org.highwire.dtl.DTLVardef@19f3528org.highwire.dtl.DTLVardef@15b051c_HPS_FORMAT_FIGEXP M_FIG C_FIG\nMicrobial functional pathways based on metatranscriptomic profiling enable effective saliva-based health assessments for precision wellness\nAuthors: Patridge, E.; Gorakshakar, A.; Molusky, M. M.; Ogundijo, O.; Janevski, A.; Julian, C.; Hu, L.; Vuyisich, M.; Banavar, G.\nScore: 2.1, Published: 2023-11-07 DOI: 10.1101/2023.11.01.565122\nIt is increasingly recognized that an important step towards improving overall health is to accurately measure biomarkers of health from the molecular activities prevalent in the oral cavity. We present a general methodology for computationally quantifying the activity of microbial functional pathways using metatranscriptomic data. We describe their implementation as a collection of eight oral pathway scores using a large salivary sample dataset (n=9,350), and we evaluate score associations with oropharyngeal disease phenotypes within an unseen independent cohort (n=14,129). As clinical validation, we show that the relevant oral pathway scores are significantly worse in individuals with periodontal disease, acid reflux, and nicotine addiction, compared with controls. Given these associations, we make the case to use these oral pathway scores to provide molecular health insights from simple, non-invasive saliva samples, and as molecular endpoints for actionable interventions to address the associated conditions. HighlightsO_LIMicrobial functional pathways in the oral cavity are quantified as eight oral scores C_LIO_LIScores are significantly worse for individuals with oropharyngeal disease phenotypes C_LIO_LIThis methodology may be generalized to other pathways and other sample types C_LIO_LIThese scores provide longitudinal health insights in a precision wellness application C_LI Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=72 SRC=\"FIGDIR/small/565122v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (16K): org.highwire.dtl.DTLVardef@f28a09org.highwire.dtl.DTLVardef@1445d90org.highwire.dtl.DTLVardef@15aab06org.highwire.dtl.DTLVardef@19246cb_HPS_FORMAT_FIGEXP M_FIG C_FIG\nModel-based inference of a plant-specific dual role for HOPS in regulating guard cell vacuole fusion\nAuthors: Hodgens, C.; Flaherty, D.; Pullen, A.-M.; Khan, I.; English, N. J.; Gillan, L.; Rojas-Pierce, M.; Akpa, B.\nScore: 1.5, Published: 2023-11-09 DOI: 10.1101/2023.11.07.565947\nStomata are the pores on a leaf surface that regulate gas exchange. Each stoma consists of two guard cells whose movements regulate pore opening and thereby control CO2 fixation and water loss. Guard cell movements depend in part on the remodeling of vacuoles, which have been observed to change from a highly fragmented state to a fused morphology during stomata opening. This change in morphology requires a membrane fusion mechanism that responds rapidly to environmental signals, allowing plants to respond to diurnal and stress cues. With guard cell vacuoles being both large and responsive to external signals, stomata represent a unique system in which to delineate mechanisms of membrane fusion. Fusion of vacuole membranes is a highly conserved process in eukaryotes, with key roles played by two multi-subunit complexes: HOPS (homotypic fusion and vacuolar protein sorting) and SNARE (soluble NSF attachment protein receptor). HOPS is a vacuole tethering factor that is thought to chaperone SNAREs from apposing vacuole membranes into a fusion-competent complex capable of rearranging membranes. To resolve a counter-intuitive observation regarding the role of HOPS in regulating plant vacuole morphology, we derived a quantitative model of vacuole fusion dynamics and used it to generate testable predictions about HOPS-SNARE interactions. We derived our model by applying simulation-based inference to integrate prior knowledge about molecular interactions with limited, qualitative observations of emergent vacuole phenotypes. By constraining the model parameters to yield the emergent outcomes observed for stoma opening - as induced by two distinct chemical treatments - we predicted a dual role for HOPS and identified a stalled form of the SNARE complex that differs from phenomena reported in yeast. We predict that HOPS has contradictory actions at different points in the fusion signaling pathway, promoting the formation of SNARE complexes, but limiting their activity.\nDetermination of Metabolic Fluxes by Deep Learning of Isotope Labeling Patterns\nAuthors: Law, R. C.; O'Keeffe, S.; Nurwono, G.; Ki, R.; Lakhani, A.; Lai, P.-K.; Park, J. O.\nScore: 1.5, Published: 2023-11-08 DOI: 10.1101/2023.11.06.565907\nFluxomics offers a direct readout of metabolic state but relies on indirect measurement. Stable isotope tracers imprint flux-dependent isotope labeling patterns on metabolites we measure; however, the relationship between labeling patterns and fluxes remains elusive. Here we innovate a two-stage machine learning framework termed ML-Flux that streamlines metabolic flux quantitation from isotope tracing. We train machine learning models by simulating atom transitions across five universal metabolic models starting from 26 13C-glucose, 2H-glucose, and 13C-glutamine tracers within feasible flux space. ML-Flux employs deep-learning-based imputation to take variable measurements of labeling patterns as input and successive neural networks to convert the ensuing comprehensive labeling information into metabolic fluxes. Using ML-Flux with multi-isotope tracing, we obtain fluxes through central carbon metabolism that are comparable to those from a least-squares method but orders-of-magnitude faster. ML-Flux is deployed as a webtool to expand the accessibility of metabolic flux quantitation and afford actionable information on metabolism.\nMutational biases contribute to the neutral complexification of protein interaction networks following gene duplication\nAuthors: Cisneros, A. F.; Nielly-Thibault, L.; Mallik, S.; Levy, E. D.; Landry, C. R.\nScore: 2.0, Published: 2023-11-02 DOI: 10.1101/2023.10.31.565025\nBiological systems can gain complexity over time. While some of these transitions are likely driven by natural selection, the extent to which they occur without providing an adaptive benefit is unknown. At the molecular level, one example is heteromeric complexes replacing homomeric ones following gene duplication. Here, we build a biophysical model and simulate the evolution of homodimers and heterodimers following gene duplication using distributions of mutational effects inferred from available protein structures. We keep the specific activity of each dimer identical, so their concentrations drift neutrally in the absence of new functions. We show that for more than 60% of tested dimer structures, the relative concentration of the heteromer increases over time due to mutational biases that favor the heterodimer. However, allowing mutational effects on synthesis rates and differences in the specific activity of homodimers and heterodimers can limit or reverse the observed bias toward heterodimers. Our results show that the accumulation of more complex protein quaternary structures is likely under neutral evolution, and that natural selection would be needed to reverse this tendency.\nMulticellular growth as a dynamic network of cells\nAuthors: Nanda, P.; Barrere, J.; LaBar, T.; Murray, A. W.\nScore: 3.4, Published: 2023-11-03 DOI: 10.1101/2023.11.02.565242\nCell division without cell separation produces multicellular clusters in budding yeast. Two fundamental characteristics of these clusters are their size (the number of cells per cluster) and cellular composition: the fractions of cells with different phenotypes. However, we do not understand how different cellular features quantitatively influence these two phenotypes. Using cells as nodes and links between mother and daughter cells as edges, we model cluster growth and breakage by varying three parameters: the cell division rate, the rate at which intercellular connections break, and the kissing number (the maximum number of connections to one cell). We find that the kissing number sets the maximum possible cluster size. Below this limit, the ratio of the cell division rate to the connection breaking rate determines the cluster size. If links have a constant probability of breaking per unit time, the probability that a link survives decreases exponentially with its age. Modeling this behavior recapitulates experimental data. We then use this framework to examine synthetic, differentiating clusters with two cell types, faster-growing germ cells and their somatic derivatives. The fraction of clusters that contain both cell types increases as either of two parameters increase: the kissing number and difference between the growth rate of germ and somatic cells. In a population of clusters, the variation in cellular composition is inversely correlated (r2=0.87) with the average fraction of somatic cells in clusters. Our results show how a small number of cellular features can control the phenotypes of multicellular clusters that were potentially the ancestors of more complex forms of multicellular development, organization, and reproduction.\nSynthetic model ecosystem of 12 cryopreservable microbial species allowing for a noninvasive approach\nAuthors: Hosoda, K.; Seno, S.; Murakami, N.; Matsuda, H.; Osada, Y.; Kamiura, R.; Kondoh, M.\nScore: 30.3, Published: 2023-10-31 DOI: 10.1101/2020.10.23.351742\nSimultaneous understanding of both individual and ecosystem dynamics is crucial in an era marked by the degradation of ecosystem services. Herein, we present a high-throughput synthetic microcosm system comprising 12 functionally and phylogenetically diverse microbial species. These species are axenically culturable, cryopreservable, and can be measured noninvasively via microscopy, aided by machine learning. This system includes prokaryotic and eukaryotic producers and decomposers, and eukaryotic consumers to ensure functional redundancy. Our model system displayed both positive and negative interspecific interactions and higher-order interactions that surpassed the scope of any two-species interaction. Although complete species coexistence was not our primary objective, we identified several conditions under which at least one species from the producers, consumers, and decomposers groups, and one functionally redundant species, persisted for over six months. These conditions set the stage for detailed investigations in the future. Given its designability and experimental replicability, our model ecosystem offers a promising platform for deeper insights into both individual and ecosystem dynamics, including evolution and species interactions.\nIdentification and Pharmacological Targeting of Treatment-Resistant, Stem-like Breast Cancer Cells for Combination Therapy\nAuthors: Worley, J.; Noh, H.; You, D.; Turunen, M.; Ding, H.; Paull, E. O.; Griffen, A.; Grunn, A.; Zhang, M.; Guillan, K.; Bush, E.; Brosius, S.; Hibshoosh, H.; Mundi, P. S.; Sims, P.; Dalerba, P.; Dela Cruz, F. S.; Kung, A. L.; Califano, A.\nScore: 1.0, Published: 2023-11-10 DOI: 10.1101/2023.11.08.562798\nTumors frequently harbor isogenic yet epigenetically distinct subpopulations of multi-potent cells with high tumor-initiating potential-often called Cancer Stem-Like Cells (CSLCs). These can display preferential resistance to standard-of-care chemotherapy. Single-cell analyses can help elucidate Master Regulator (MR) proteins responsible for governing the transcriptional state of these cells, thus revealing complementary dependencies that may be leveraged via combination therapy. Interrogation of single-cell RNA sequencing profiles from seven metastatic breast cancer patients, using perturbational profiles of clinically relevant drugs, identified drugs predicted to invert the activity of MR proteins governing the transcriptional state of chemoresistant CSLCs, which were then validated by CROP-seq assays. The top drug, the anthelmintic albendazole, depleted this subpopulation in vivo without noticeable cytotoxicity. Moreover, sequential cycles of albendazole and paclitaxel-a commonly used chemotherapeutic-displayed significant synergy in a patient-derived xenograft (PDX) from a TNBC patient, suggesting that network-based approaches can help develop mechanism-based combinatorial therapies targeting complementary subpopulations.\nShort-term Pre-operative Methionine Restriction Induces Browning of Perivascular Adipose Tissue and Improves Vein Graft Remodeling in Mice\nAuthors: Kip, P.; Sluiter, T. J.; MacArthur, M. R.; Tao, M.; Jung, J.; Mitchell, S. J.; Kooijman, S.; Kruit, N.; Gorham, J.; Seidman, J.; Quax, P. H. A.; Aikawa, M.; Ozaki, C. K.; Mitchell, J. R.; De Vries, M. R.\nScore: 1.0, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565269\nShort-term preoperative methionine restriction (MetR) shows promise as a translatable strategy to modulate the bodys response to surgical injury. Its application, however, to improve post-interventional vascular remodeling remains underexplored. Here, we find that MetR protects from arterial intimal hyperplasia in a focal stenosis model and adverse vascular remodeling after vein graft surgery. RNA sequencing reveals that MetR enhances the brown adipose tissue phenotype in arterial perivascular adipose tissue (PVAT) and induces it in venous PVAT. Specifically, PPAR- was highly upregulated in PVAT-adipocytes. Furthermore, MetR dampens the post-operative pro-inflammatory response to surgery in PVAT-macrophages in vivo and in vitro. This study shows for the first time that the detrimental effects of dysfunctional PVAT on vascular remodeling can be reversed by MetR, and identifies pathways involved in browning of PVAT. Furthermore, we demonstrate the potential of short-term pre-operative MetR as a simple intervention to ameliorate vascular remodeling after vascular surgery. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=109 SRC=\"FIGDIR/small/565269v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (38K): org.highwire.dtl.DTLVardef@24d9b6org.highwire.dtl.DTLVardef@1514715org.highwire.dtl.DTLVardef@111db5borg.highwire.dtl.DTLVardef@1583d24_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2266",
  "inLanguage": "en",
  "datePublished": "2023-11-12T10:37:43Z",
  "dateModified": "2023-11-12T10:37:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 12, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565140">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565140" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565140">
        <p class="paperTitle">Allometry of cell types in planarians by single cell transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565140" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565140" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.</p>
        <p class="info">Score: 64.1, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565140' target='https://doi.org/10.1101/2023.11.01.565140'> 10.1101/2023.11.01.565140</a></p>
        <p class="abstract">Allometry explores the relationship between an organisms body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry. Planarians, capable of growing and degrowing following allometric scaling rules, serve as an excellent model for such studies. We used scRNA-seq to examine cell type allometry in asexual planarians of different sizes, revealing that they consist of the same basic cell types but in varying proportions. Notably, the gut basal cells are the most responsive to changes in size, suggesting a role in energy storage. We capture the gene regulatory programs of distinct cell types in response to size. This research sheds light on the molecular and cellular aspects of cell type allometry in planarians and underscores the utility of scRNA-seq in such investigations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564854">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564854" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564854">
        <p class="paperTitle">Principles of Computation by Competitive Protein Dimerization Networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564854" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564854" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parres-Gold, J.; Levine, M.; Emert, B.; Stuart, A.; Elowitz, M.</p>
        <p class="info">Score: 50.1, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564854' target='https://doi.org/10.1101/2023.10.30.564854'> 10.1101/2023.10.30.564854</a></p>
        <p class="abstract">Many biological signaling pathways employ proteins that competitively dimerize in diverse combinations. These dimerization networks can perform biochemical computations, in which the concentrations of monomers (inputs) determine the concentrations of dimers (outputs). Despite their prevalence, little is known about the range of input-output computations that dimerization networks can perform (their &#34;expressivity&#34;) and how it depends on network size and connectivity. Using a systematic computational approach, we demonstrate that even small dimerization networks (3-6 monomers) are expressive, performing diverse multi-input computations. Further, dimerization networks are versatile, performing different computations when their protein components are expressed at different levels, such as in different cell types. Remarkably, individual networks with random interaction affinities, when large enough ([&amp;ge;]8 proteins), can perform nearly all ([~]90%) potential one-input network computations merely by tuning their monomer expression levels. Thus, even the simple process of competitive dimerization provides a powerful architecture for multi-input, cell-type-specific signal processing.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=192 SRC=&#34;FIGDIR/small/564854v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (35K):
org.highwire.dtl.DTLVardef@1c906faorg.highwire.dtl.DTLVardef@e948a0org.highwire.dtl.DTLVardef@19f3528org.highwire.dtl.DTLVardef@15b051c_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565122">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565122" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565122">
        <p class="paperTitle">Microbial functional pathways based on metatranscriptomic profiling enable effective saliva-based health assessments for precision wellness</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565122" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565122" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Patridge, E.; Gorakshakar, A.; Molusky, M. M.; Ogundijo, O.; Janevski, A.; Julian, C.; Hu, L.; Vuyisich, M.; Banavar, G.</p>
        <p class="info">Score: 2.1, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565122' target='https://doi.org/10.1101/2023.11.01.565122'> 10.1101/2023.11.01.565122</a></p>
        <p class="abstract">It is increasingly recognized that an important step towards improving overall health is to accurately measure biomarkers of health from the molecular activities prevalent in the oral cavity. We present a general methodology for computationally quantifying the activity of microbial functional pathways using metatranscriptomic data. We describe their implementation as a collection of eight oral pathway scores using a large salivary sample dataset (n=9,350), and we evaluate score associations with oropharyngeal disease phenotypes within an unseen independent cohort (n=14,129). As clinical validation, we show that the relevant oral pathway scores are significantly worse in individuals with periodontal disease, acid reflux, and nicotine addiction, compared with controls. Given these associations, we make the case to use these oral pathway scores to provide molecular health insights from simple, non-invasive saliva samples, and as molecular endpoints for actionable interventions to address the associated conditions.

HighlightsO_LIMicrobial functional pathways in the oral cavity are quantified as eight oral scores
C_LIO_LIScores are significantly worse for individuals with oropharyngeal disease phenotypes
C_LIO_LIThis methodology may be generalized to other pathways and other sample types
C_LIO_LIThese scores provide longitudinal health insights in a precision wellness application
C_LI

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=72 SRC=&#34;FIGDIR/small/565122v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (16K):
org.highwire.dtl.DTLVardef@f28a09org.highwire.dtl.DTLVardef@1445d90org.highwire.dtl.DTLVardef@15aab06org.highwire.dtl.DTLVardef@19246cb_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565947">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565947" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565947">
        <p class="paperTitle">Model-based inference of a plant-specific dual role for HOPS in regulating guard cell vacuole fusion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565947" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565947" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hodgens, C.; Flaherty, D.; Pullen, A.-M.; Khan, I.; English, N. J.; Gillan, L.; Rojas-Pierce, M.; Akpa, B.</p>
        <p class="info">Score: 1.5, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565947' target='https://doi.org/10.1101/2023.11.07.565947'> 10.1101/2023.11.07.565947</a></p>
        <p class="abstract">Stomata are the pores on a leaf surface that regulate gas exchange. Each stoma consists of two guard cells whose movements regulate pore opening and thereby control CO2 fixation and water loss. Guard cell movements depend in part on the remodeling of vacuoles, which have been observed to change from a highly fragmented state to a fused morphology during stomata opening. This change in morphology requires a membrane fusion mechanism that responds rapidly to environmental signals, allowing plants to respond to diurnal and stress cues. With guard cell vacuoles being both large and responsive to external signals, stomata represent a unique system in which to delineate mechanisms of membrane fusion. Fusion of vacuole membranes is a highly conserved process in eukaryotes, with key roles played by two multi-subunit complexes: HOPS (homotypic fusion and vacuolar protein sorting) and SNARE (soluble NSF attachment protein receptor). HOPS is a vacuole tethering factor that is thought to chaperone SNAREs from apposing vacuole membranes into a fusion-competent complex capable of rearranging membranes. To resolve a counter-intuitive observation regarding the role of HOPS in regulating plant vacuole morphology, we derived a quantitative model of vacuole fusion dynamics and used it to generate testable predictions about HOPS-SNARE interactions. We derived our model by applying simulation-based inference to integrate prior knowledge about molecular interactions with limited, qualitative observations of emergent vacuole phenotypes. By constraining the model parameters to yield the emergent outcomes observed for stoma opening - as induced by two distinct chemical treatments - we predicted a dual role for HOPS and identified a stalled form of the SNARE complex that differs from phenomena reported in yeast. We predict that HOPS has contradictory actions at different points in the fusion signaling pathway, promoting the formation of SNARE complexes, but limiting their activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565907">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565907" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565907">
        <p class="paperTitle">Determination of Metabolic Fluxes by Deep Learning of Isotope Labeling Patterns</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565907" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565907" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Law, R. C.; O&#39;Keeffe, S.; Nurwono, G.; Ki, R.; Lakhani, A.; Lai, P.-K.; Park, J. O.</p>
        <p class="info">Score: 1.5, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565907' target='https://doi.org/10.1101/2023.11.06.565907'> 10.1101/2023.11.06.565907</a></p>
        <p class="abstract">Fluxomics offers a direct readout of metabolic state but relies on indirect measurement. Stable isotope tracers imprint flux-dependent isotope labeling patterns on metabolites we measure; however, the relationship between labeling patterns and fluxes remains elusive. Here we innovate a two-stage machine learning framework termed ML-Flux that streamlines metabolic flux quantitation from isotope tracing. We train machine learning models by simulating atom transitions across five universal metabolic models starting from 26 13C-glucose, 2H-glucose, and 13C-glutamine tracers within feasible flux space. ML-Flux employs deep-learning-based imputation to take variable measurements of labeling patterns as input and successive neural networks to convert the ensuing comprehensive labeling information into metabolic fluxes. Using ML-Flux with multi-isotope tracing, we obtain fluxes through central carbon metabolism that are comparable to those from a least-squares method but orders-of-magnitude faster. ML-Flux is deployed as a webtool to expand the accessibility of metabolic flux quantitation and afford actionable information on metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565025" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565025">
        <p class="paperTitle">Mutational biases contribute to the neutral complexification of protein interaction networks following gene duplication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cisneros, A. F.; Nielly-Thibault, L.; Mallik, S.; Levy, E. D.; Landry, C. R.</p>
        <p class="info">Score: 2.0, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565025' target='https://doi.org/10.1101/2023.10.31.565025'> 10.1101/2023.10.31.565025</a></p>
        <p class="abstract">Biological systems can gain complexity over time. While some of these transitions are likely driven by natural selection, the extent to which they occur without providing an adaptive benefit is unknown. At the molecular level, one example is heteromeric complexes replacing homomeric ones following gene duplication. Here, we build a biophysical model and simulate the evolution of homodimers and heterodimers following gene duplication using distributions of mutational effects inferred from available protein structures. We keep the specific activity of each dimer identical, so their concentrations drift neutrally in the absence of new functions. We show that for more than 60% of tested dimer structures, the relative concentration of the heteromer increases over time due to mutational biases that favor the heterodimer. However, allowing mutational effects on synthesis rates and differences in the specific activity of homodimers and heterodimers can limit or reverse the observed bias toward heterodimers. Our results show that the accumulation of more complex protein quaternary structures is likely under neutral evolution, and that natural selection would be needed to reverse this tendency.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565242">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565242" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565242">
        <p class="paperTitle">Multicellular growth as a dynamic network of cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565242" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565242" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nanda, P.; Barrere, J.; LaBar, T.; Murray, A. W.</p>
        <p class="info">Score: 3.4, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565242' target='https://doi.org/10.1101/2023.11.02.565242'> 10.1101/2023.11.02.565242</a></p>
        <p class="abstract">Cell division without cell separation produces multicellular clusters in budding yeast. Two fundamental characteristics of these clusters are their size (the number of cells per cluster) and cellular composition: the fractions of cells with different phenotypes. However, we do not understand how different cellular features quantitatively influence these two phenotypes. Using cells as nodes and links between mother and daughter cells as edges, we model cluster growth and breakage by varying three parameters: the cell division rate, the rate at which intercellular connections break, and the kissing number (the maximum number of connections to one cell). We find that the kissing number sets the maximum possible cluster size. Below this limit, the ratio of the cell division rate to the connection breaking rate determines the cluster size. If links have a constant probability of breaking per unit time, the probability that a link survives decreases exponentially with its age. Modeling this behavior recapitulates experimental data. We then use this framework to examine synthetic, differentiating clusters with two cell types, faster-growing germ cells and their somatic derivatives. The fraction of clusters that contain both cell types increases as either of two parameters increase: the kissing number and difference between the growth rate of germ and somatic cells. In a population of clusters, the variation in cellular composition is inversely correlated (r2=0.87) with the average fraction of somatic cells in clusters. Our results show how a small number of cellular features can control the phenotypes of multicellular clusters that were potentially the ancestors of more complex forms of multicellular development, organization, and reproduction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2020.10.23.351742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2020.10.23.351742" aria-expanded="false" aria-controls="flush-collapse10.1101/2020.10.23.351742">
        <p class="paperTitle">Synthetic model ecosystem of 12 cryopreservable microbial species allowing for a noninvasive approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2020.10.23.351742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2020.10.23.351742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hosoda, K.; Seno, S.; Murakami, N.; Matsuda, H.; Osada, Y.; Kamiura, R.; Kondoh, M.</p>
        <p class="info">Score: 30.3, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2020.10.23.351742' target='https://doi.org/10.1101/2020.10.23.351742'> 10.1101/2020.10.23.351742</a></p>
        <p class="abstract">Simultaneous understanding of both individual and ecosystem dynamics is crucial in an era marked by the degradation of ecosystem services. Herein, we present a high-throughput synthetic microcosm system comprising 12 functionally and phylogenetically diverse microbial species. These species are axenically culturable, cryopreservable, and can be measured noninvasively via microscopy, aided by machine learning. This system includes prokaryotic and eukaryotic producers and decomposers, and eukaryotic consumers to ensure functional redundancy. Our model system displayed both positive and negative interspecific interactions and higher-order interactions that surpassed the scope of any two-species interaction. Although complete species coexistence was not our primary objective, we identified several conditions under which at least one species from the producers, consumers, and decomposers groups, and one functionally redundant species, persisted for over six months. These conditions set the stage for detailed investigations in the future. Given its designability and experimental replicability, our model ecosystem offers a promising platform for deeper insights into both individual and ecosystem dynamics, including evolution and species interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.562798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.562798" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.562798">
        <p class="paperTitle">Identification and Pharmacological Targeting of Treatment-Resistant, Stem-like Breast Cancer Cells for Combination Therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.562798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.562798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Worley, J.; Noh, H.; You, D.; Turunen, M.; Ding, H.; Paull, E. O.; Griffen, A.; Grunn, A.; Zhang, M.; Guillan, K.; Bush, E.; Brosius, S.; Hibshoosh, H.; Mundi, P. S.; Sims, P.; Dalerba, P.; Dela Cruz, F. S.; Kung, A. L.; Califano, A.</p>
        <p class="info">Score: 1.0, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.562798' target='https://doi.org/10.1101/2023.11.08.562798'> 10.1101/2023.11.08.562798</a></p>
        <p class="abstract">Tumors frequently harbor isogenic yet epigenetically distinct subpopulations of multi-potent cells with high tumor-initiating potential-often called Cancer Stem-Like Cells (CSLCs). These can display preferential resistance to standard-of-care chemotherapy. Single-cell analyses can help elucidate Master Regulator (MR) proteins responsible for governing the transcriptional state of these cells, thus revealing complementary dependencies that may be leveraged via combination therapy. Interrogation of single-cell RNA sequencing profiles from seven metastatic breast cancer patients, using perturbational profiles of clinically relevant drugs, identified drugs predicted to invert the activity of MR proteins governing the transcriptional state of chemoresistant CSLCs, which were then validated by CROP-seq assays. The top drug, the anthelmintic albendazole, depleted this subpopulation in vivo without noticeable cytotoxicity. Moreover, sequential cycles of albendazole and paclitaxel-a commonly used chemotherapeutic-displayed significant synergy in a patient-derived xenograft (PDX) from a TNBC patient, suggesting that network-based approaches can help develop mechanism-based combinatorial therapies targeting complementary subpopulations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565269">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565269" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565269">
        <p class="paperTitle">Short-term Pre-operative Methionine Restriction Induces Browning of Perivascular Adipose Tissue and Improves Vein Graft Remodeling in Mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565269" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565269" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kip, P.; Sluiter, T. J.; MacArthur, M. R.; Tao, M.; Jung, J.; Mitchell, S. J.; Kooijman, S.; Kruit, N.; Gorham, J.; Seidman, J.; Quax, P. H. A.; Aikawa, M.; Ozaki, C. K.; Mitchell, J. R.; De Vries, M. R.</p>
        <p class="info">Score: 1.0, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565269' target='https://doi.org/10.1101/2023.11.02.565269'> 10.1101/2023.11.02.565269</a></p>
        <p class="abstract">Short-term preoperative methionine restriction (MetR) shows promise as a translatable strategy to modulate the bodys response to surgical injury. Its application, however, to improve post-interventional vascular remodeling remains underexplored. Here, we find that MetR protects from arterial intimal hyperplasia in a focal stenosis model and adverse vascular remodeling after vein graft surgery. RNA sequencing reveals that MetR enhances the brown adipose tissue phenotype in arterial perivascular adipose tissue (PVAT) and induces it in venous PVAT. Specifically, PPAR- was highly upregulated in PVAT-adipocytes. Furthermore, MetR dampens the post-operative pro-inflammatory response to surgery in PVAT-macrophages in vivo and in vitro. This study shows for the first time that the detrimental effects of dysfunctional PVAT on vascular remodeling can be reversed by MetR, and identifies pathways involved in browning of PVAT. Furthermore, we demonstrate the potential of short-term pre-operative MetR as a simple intervention to ameliorate vascular remodeling after vascular surgery.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=109 SRC=&#34;FIGDIR/small/565269v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (38K):
org.highwire.dtl.DTLVardef@24d9b6org.highwire.dtl.DTLVardef@1514715org.highwire.dtl.DTLVardef@111db5borg.highwire.dtl.DTLVardef@1583d24_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
