<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="ImAge: an imaging approach to quantitate aging and rejuvenation.
Authors: Alvarez-Kuglen, M.; Rodriguez, D.; Qin, H.; Ninomiya, K.; Fiengo, L.; Farhy, C.; Havas, A.; Hsu, W.-M.; Feng, G.-S.; Roberts, A.; Anderson, R. M.; Serrano, M.; Adams, P. D.; Sharpee, T. O.; Terskikh, A.
Score: 19.1, Published: 2023-09-24 DOI: 10.1101/2022.10.16.512441
Biomarkers of biological age that predict the risk of disease and expected lifespan better than chronological age are key to efficient and cost-effective healthcare1-3.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="ImAge: an imaging approach to quantitate aging and rejuvenation.
Authors: Alvarez-Kuglen, M.; Rodriguez, D.; Qin, H.; Ninomiya, K.; Fiengo, L.; Farhy, C.; Havas, A.; Hsu, W.-M.; Feng, G.-S.; Roberts, A.; Anderson, R. M.; Serrano, M.; Adams, P. D.; Sharpee, T. O.; Terskikh, A.
Score: 19.1, Published: 2023-09-24 DOI: 10.1101/2022.10.16.512441
Biomarkers of biological age that predict the risk of disease and expected lifespan better than chronological age are key to efficient and cost-effective healthcare1-3." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-22T10:37:23+00:00" />
<meta property="article:modified_time" content="2023-10-22T10:37:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="ImAge: an imaging approach to quantitate aging and rejuvenation.
Authors: Alvarez-Kuglen, M.; Rodriguez, D.; Qin, H.; Ninomiya, K.; Fiengo, L.; Farhy, C.; Havas, A.; Hsu, W.-M.; Feng, G.-S.; Roberts, A.; Anderson, R. M.; Serrano, M.; Adams, P. D.; Sharpee, T. O.; Terskikh, A.
Score: 19.1, Published: 2023-09-24 DOI: 10.1101/2022.10.16.512441
Biomarkers of biological age that predict the risk of disease and expected lifespan better than chronological age are key to efficient and cost-effective healthcare1-3."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "ImAge: an imaging approach to quantitate aging and rejuvenation.\nAuthors: Alvarez-Kuglen, M.; Rodriguez, D.; Qin, H.; Ninomiya, K.; Fiengo, L.; Farhy, C.; Havas, A.; Hsu, W.-M.; Feng, G.-S.; Roberts, A.; Anderson, R. M.; Serrano, M.; Adams, P. D.; Sharpee, T. O.; Terskikh, A.\nScore: 19.1, Published: 2023-09-24 DOI: 10.1101/2022.10.16.512441\nBiomarkers of biological age that predict the risk of disease and expected lifespan better than chronological age are key to efficient and cost-effective healthcare1-3.",
  "keywords": [
    
  ],
  "articleBody": " ImAge: an imaging approach to quantitate aging and rejuvenation.\nAuthors: Alvarez-Kuglen, M.; Rodriguez, D.; Qin, H.; Ninomiya, K.; Fiengo, L.; Farhy, C.; Havas, A.; Hsu, W.-M.; Feng, G.-S.; Roberts, A.; Anderson, R. M.; Serrano, M.; Adams, P. D.; Sharpee, T. O.; Terskikh, A.\nScore: 19.1, Published: 2023-09-24 DOI: 10.1101/2022.10.16.512441\nBiomarkers of biological age that predict the risk of disease and expected lifespan better than chronological age are key to efficient and cost-effective healthcare1-3. To advance a personalized approach to healthcare, such biomarkers must perform on the individual rather than population level, demonstrate single cell resolution, and provide scalable and cost-effective measurements. We developed a novel approach - image-based chromatin and epigenetic age (ImAge), that utilizes image texture features based on the patterns of chromatin and epigenetic marks in single nuclei. We observed the emergence of intrinsic trajectories of ImAge using dimensionality reduction without regression on chronological age. ImAge was correlated with chronological age in all tissues and organs examined and was consistent with the expected acceleration and/or deceleration of biological age in chronologically identical mice treated with chemotherapy or following a caloric restriction regimen, respectively. ImAge from chronologically identical mice inversely correlated with their locomotor activity (greater activity for younger ImAge), consistent with the essential role of locomotion as an aging biomarker. Finally, we demonstrated that ImAge is reduced upon partial reprogramming in vivo following transient expression of OSKM cassette in the liver and skeletal muscles of old mice and validated the power of ImAge to assess the heterogeneity of reprogramming. We propose that ImAge represents the first-in-class individual-level biomarker of aging and rejuvenation with single-cell resolution.\nPerturbation-response analysis of in silico metabolic dynamics: Hard-coded responsiveness in the cofactor and network sparsity\nAuthors: Himeoka, Y.; Furusawa, C.\nScore: 10.7, Published: 2023-10-20 DOI: 10.1101/2023.10.18.562862\nHomeostasis is a fundamental characteristic of living systems. Unlike rigidity, homeostasis necessitates that systems respond flexibly to diverse environments. Understanding the dynamics of biochemical systems when subjected to perturbations is essential for the development of a quantitative theory of homeostasis. In this study, we analyze the response of bacterial metabolism to externally imposed perturbations using kinetic models of Escherichia coli's central carbon metabolism. Our findings indicate that three distinct kinetic models consistently display strong responses to perturbations, wherein minor initial discrepancies in metabolite concentrations from steady-state values amplify over time, resulting in significant deviations. Subsequent numerical studies show that adenyl cofactors consistently influence the responsiveness of the metabolic systems across models. Additionally, we examine the impact of network structure on metabolic dynamics, demonstrating that as the metabolic network becomes denser, the perturbation response diminishes-a trend observed commonly in the models. To confirm the significance of cofactors and network structure, we constructed a simplified metabolic network model, underscoring their importance. By identifying the structural determinants of responsiveness, our findings offer implications for bacterial physiology, the evolution of metabolic networks, and the design principles for robust artificial metabolism in synthetic biology and bioengineering.\nThe genetic architecture of protein interaction affinity and specificity\nAuthors: Bendel, A.; Faure, A.; Klein, D.; Shimada, K.; Kempf, G.; Cavadini, S.; Lehner, B.; Diss, G.\nScore: 10.4, Published: 2023-10-20 DOI: 10.1101/2023.10.17.562688\nProteins function in crowded cellular environments in which they must bind to specific target proteins but also avoid binding to many other off-target proteins. In large protein families this task is particularly challenging because many off-target proteins have very similar structures. How this specificity of physical protein-protein interactions in cellular networks is encoded and evolves is not very well understood. Here we address the question of specificity-encoding by comprehensively quantifying the effects of all mutations in one protein, JUN, on its binding to all other members of a protein family, the 54 human basic leucine zipper transcription factors. Fitting a global thermodynamic model to the data reveals that most affinity changing mutations equally affect JUN's propensity to bind to all its interaction partners. Mutations that alter the specificity of binding are much rarer. These specificity-altering mutations are, however, distributed throughout the JUN interaction interface. JUN's interaction specificity is encoded by both positive determinants that promote on-target interactions and negative determinants that prevent off-target interactions. Indeed, about half of the specificity-defining residues in JUN have dual functions and both promote on-target binding and prevent off-target binding. Whereas nearly all mutations that alter specificity are pleiotropic and also alter the affinity of binding to all interaction partners, the converse is not true with mutations outside of the interface able to tune affinity without affecting specificity. Our results provide the first global view of how mutations in a protein affect binding to all its potential interaction partners and reveal the distributed encoding of specificity and affinity in an interaction interface. They also show how the modular architecture of coiled-coils provides an elegant solution to the challenge of optimising specificity and affinity in a large protein family.\nThe type of carbon source not the growth rate it supports can determine diauxie\nAuthors: Huo, Y.; Danecka, W.; Farquhar, I.; Mailliet, K.; Moses, T.; Wallace, E. W.; Swain, P. S.\nScore: 4.1, Published: 2023-10-20 DOI: 10.1101/2023.10.18.562896\nHow cells choose between potential carbon sources is a classic example of cellular decision-making, and we know that many organisms prioritise glucose. Yet there has been little investigation of whether other sugars are also preferred, blinkering our view of carbon sensing. Here we study eukaryotic budding yeast and its growth on mixtures of palatinose, an isomer of sucrose, with other sugars. We find that yeast prioritise galactose over palatinose, but not sucrose or fructose, despite all three of these sugars being able to support faster growth than palatinose. Our results therefore disfavour carbon flux-sensing as the sole mechanism. By using genetic perturbations and transcriptomics, we show that repression is active and through Gal4, the master regulator of the GAL regulon. Cells enforce their preference for galactose over palatinose by preventing runaway positive feedback in the MAL regulon, whose genes enable palatinose catabolism. They do so both by repressing MAL11, the gene encoding the palatinose transporter, and by first expressing the isomaltases, IMA1 and IMA5, which cleave palatinose and so prevent its intracellular concentration becoming enough to induce further MAL expression. Our results demonstrate that budding yeast actively maintain a preference for carbon sources other than glucose and that such preferences have been selected by more than differences in growth rates. They imply that carbon-sensing strategies even in unicellular organisms are more complex than previously thought.\nA robust ultrasensitive transcriptional switch in noisy cellular environments\nAuthors: Jeong, E. M.; Kim, J. K.\nScore: 4.5, Published: 2023-10-18 DOI: 10.1101/2023.08.15.553401\nUltrasensitive transcriptional switches enable sharp transitions between on and off states of transcription and are essential for cells to respond to environmental cues precisely. However, conventional switches, relying on direct repressor-DNA binding, are extremely sensitive to noise. Here, we discovered an alternative design combining three indirect transcriptional repression mechanisms, sequestration, blocking, and displacement, to generate a noise-resilient ultrasensitive switch. Although sequestration alone can generate an ultrasensitive switch, it remains sensitive to noise because the unintended transcriptional state induced by noise can persist for long periods. However, by jointly utilizing blocking and displacement, these noise-induced transitions can be rapidly restored to the original transcriptional state. Because this transcriptional switch is effective in noisy cellular contexts, it goes beyond previous synthetic transcriptional switches utilizing direct repression mechanisms, making it particularly valuable for robust synthetic system design. Our findings also provide insights into the evolution of robust ultrasensitive switches in real cells. Specifically, the concurrent use of seemingly redundant indirect repression mechanisms in diverse biological systems appears to be a strategy to achieve noise-resilience of ultrasensitive switches. GRAPHICAL ABSTRACT O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=175 SRC=\"FIGDIR/small/553401v2_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (28K): org.highwire.dtl.DTLVardef@a36772org.highwire.dtl.DTLVardef@1dc8f8borg.highwire.dtl.DTLVardef@1542b20org.highwire.dtl.DTLVardef@db74d2_HPS_FORMAT_FIGEXP M_FIG C_FIG\nModeling alternative translation initiation sites in plants reveals evolutionarily conserved cis-regulatory codes in eukaryotes\nAuthors: Wu, T.-Y.; Li, Y.-R.; Chang, K.-J.; Urano, D.; Liu, M.-J.\nScore: 2.1, Published: 2023-10-21 DOI: 10.1101/2023.10.20.563379\nmRNA translation relies on identifying translation initiation sites (TISs) in mRNAs. Alternative TISs are prevalent across plant transcriptomes, but the mechanisms for their recognition are unclear. Using ribosome profiling and machine learning, we developed models for predicting alternative TISs in Arabidopsis thaliana and tomato (Solanum lycopersicum). Distinct feature sets were predictive of AUG and non-AUG TISs in 5' untranslated regions and coding sequences, including a novel CU-rich sequence that promoted plant TIS activity, a translational enhancer found across dicots and monocots and also in humans and viruses. Our results elucidate the mechanistic and evolutionary basis of TIS recognition, whereby cis-regulatory RNA signatures affect start site selection. The TIS prediction model provides global estimates of TISs to discover neglected protein-coding genes across plant genomes. The prevalence of cis-regulatory signatures across eukaryotes and viruses suggests their broad, critical roles in reprogramming the translational landscape in the plant-virus arms race.\nMachine learning predicts system-wide metabolic flux control in cyanobacteria\nAuthors: Kugler, A.; Stensjö, K.\nScore: 2.0, Published: 2023-10-20 DOI: 10.1101/2023.10.18.562898\nMetabolic fluxes and their control mechanisms are fundamental in cellular metabolism, offering insights for the study of biological systems and biotechnological applications. However, quantitative and predictive understanding of controlling biochemical reactions in microbial cell factories, especially at the system level, is limited. In this work, we present ARCTICA, a computational framework that integrates constraint-based modelling with machine learning tools to address this challenge. Using the model cyanobacterium Synechocystis sp. PCC 6803 as chassis, we demonstrate that ARCTICA effectively simulates global-scale metabolic flux control. Key findings are that (i) the photosynthetic bioproduction is mainly governed by enzymes from the Calvin-Benson-Bassham (CBB) cycle, rather than by those involve in the biosynthesis of the end-product, (ii) the catalytic capacity of the CBB cycle limits the photosynthetic activity and downstream pathways and (iii) ribulose-1,5-bisphosphate carboxylase/oxygenase (RuBisCO) is a major, but not the most, limiting step in the CBB cycle. Predicted metabolic reactions qualitatively align with prior experimental observations, validating our modelling approach. ARCTICA serves as a valuable pipeline for understanding cellular physiology and predicting rate-limiting steps in genome-scale metabolic networks, providing guidance for bioengineering of cyanobacteria.\nSystems-level analyses dissociate genetic regulators of reactive oxygen species and energy production\nAuthors: Bennett, N. K.; Lee, M.; Orr, A. L.; Nakamura, K.\nScore: 1.8, Published: 2023-10-18 DOI: 10.1101/2023.10.14.562276\nRespiratory chain dysfunction can decrease ATP and increase reactive oxygen species (ROS) levels. Despite the importance of these metabolic parameters to a wide range of cellular functions and disease, we lack an integrated understanding of how they are differentially regulated. To address this question, we adapted a CRISPRi- and FACS-based platform to compare the effects of respiratory gene knockdown on ROS to their effects on ATP. Focusing on genes whose knockdown is known to decrease mitochondria-derived ATP, we showed that knockdown of genes in specific respiratory chain complexes (I, III and CoQ10 biosynthesis) increased ROS, whereas knockdown of other low ATP hits either had no impact (mitochondrial ribosomal proteins) or actually decreased ROS (complex IV). Moreover, although shifting metabolic conditions profoundly altered mitochondria-derived ATP levels, it had little impact on mitochondrial or cytosolic ROS. In addition, knockdown of a subset of complex I subunits--including NDUFA8, NDUFB4, and NDUFS8--decreased complex I activity, mitochondria-derived ATP and supercomplex level, but knockdown of these genes had differential effects on ROS. Conversely, we found an essential role for ether lipids in the dynamic regulation of mitochondrial ROS levels independent of ATP. Thus, our results identify specific metabolic regulators of cellular ATP and ROS balance that may help dissect the roles of these processes in disease and identify therapeutic strategies to independently target energy failure and oxidative stress. SignificanceMitochondrial respiration generates both energy (ATP) and reactive oxygen species (ROS). Insufficient energy and increased ROS from respiratory chain dysfunction may be central to the pathophysiology of neurodegenerative diseases and aging. We established a screening platform using CRISPR and fluorescent-cell sorting to compare the impact of decreasing respiratory chain proteins on ROS and ATP levels. The results provide the first systems-level analysis of how ROS and ATP are differentially regulated, and identify genes and respiratory chain complexes that can manipulate each independently. These findings advance our understanding of the relative contributions of ATP and ROS to disease pathophysiology, and guide the development of therapies to preserve energy while minimizing ROS.\nMinimally Sufficient Experimental Design using Identifiability Analysis\nAuthors: Kareva, I.; Gevertz, J. L.\nScore: 1.5, Published: 2023-10-18 DOI: 10.1101/2023.10.14.562348\nMathematical models are increasingly being developed and calibrated in tandem with data collection, empowering scientists to intervene in real time based on quantitative model predictions. Well-designed experiments can help augment the predictive power of a mathematical model but the question of when to collect data to maximize its utility for a model is non-trivial. Here we define data as model-informative if it results in a unique parametrization, assessed through the lens of practical identifiability. The framework we propose identifies an optimal experimental design (how much data to collect and when to collect it) that ensures parameter identifiability (permitting confidence in model predictions), while minimizing experimental time and costs. We demonstrate the power of the method by applying it to a modified version of a classic site-of-action pharmacokinetic/pharmacodynamic model that describes distribution of a drug into the tumor microenvironment (TME), where its efficacy is dependent on the level of target occupancy in the TME. In this context, we identify a minimal set of time points when data needs to be collected that robustly ensures practical identifiability of model parameters. The proposed methodology can be applied broadly to any mathematical model, allowing for the identification of a minimally sufficient experimental design that collects the most informative data.\nInvestigating RNA splicing as a source of cellular diversity using a binomial mixture model\nAuthors: Isaev, K.; Knowles, D. A.\nScore: 1.2, Published: 2023-10-20 DOI: 10.1101/2023.10.17.562774\nAlternative splicing (AS) contributes significantly to RNA and protein variability yet its role in defining cellular diversity is not fully understood. While Smart-seq2 offers enhanced coverage across transcripts compared to 10X single cell RNA-sequencing (scRNA-seq), current computational methods often miss the full complexity of AS. Most approaches for single cell based differential splicing analysis focus on simple AS events such as exon skipping, and rely on predefined cell type labels or low-dimensional gene expression representations. This limits their ability to detect more complex AS events and makes them dependent on prior knowledge of cell classifications. Here, we present Leaflet, a splice junction centric approach inspired by Leafcutter, our tool for quantifying RNA splicing variation with bulk RNA-seq. Leaflet is a probabilistic mixture model designed to infer AS-driven cell states without the need for cell type labels. We detail Leaflet's generative model, inference methodology, and its efficiency in detecting differentially spliced junctions. By applying Leaflet to the Tabula Muris brain cell dataset, we highlight cell-state specific splicing patterns, offering a deeper insight into cellular diversity beyond that captured by gene expression alone.\n",
  "wordCount" : "2445",
  "inLanguage": "en",
  "datePublished": "2023-10-22T10:37:23Z",
  "dateModified": "2023-10-22T10:37:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on October 22, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.10.16.512441">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.10.16.512441" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.10.16.512441">
        <p class="paperTitle">ImAge: an imaging approach to quantitate aging and rejuvenation.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.10.16.512441" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.10.16.512441" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alvarez-Kuglen, M.; Rodriguez, D.; Qin, H.; Ninomiya, K.; Fiengo, L.; Farhy, C.; Havas, A.; Hsu, W.-M.; Feng, G.-S.; Roberts, A.; Anderson, R. M.; Serrano, M.; Adams, P. D.; Sharpee, T. O.; Terskikh, A.</p>
        <p class="info">Score: 19.1, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.10.16.512441' target='https://doi.org/10.1101/2022.10.16.512441'> 10.1101/2022.10.16.512441</a></p>
        <p class="abstract">Biomarkers of biological age that predict the risk of disease and expected lifespan better than chronological age are key to efficient and cost-effective healthcare1-3. To advance a personalized approach to healthcare, such biomarkers must perform on the individual rather than population level, demonstrate single cell resolution, and provide scalable and cost-effective measurements. We developed a novel approach - image-based chromatin and epigenetic age (ImAge), that utilizes image texture features based on the patterns of chromatin and epigenetic marks in single nuclei. We observed the emergence of intrinsic trajectories of ImAge using dimensionality reduction without regression on chronological age. ImAge was correlated with chronological age in all tissues and organs examined and was consistent with the expected acceleration and/or deceleration of biological age in chronologically identical mice treated with chemotherapy or following a caloric restriction regimen, respectively. ImAge from chronologically identical mice inversely correlated with their locomotor activity (greater activity for younger ImAge), consistent with the essential role of locomotion as an aging biomarker. Finally, we demonstrated that ImAge is reduced upon partial reprogramming in vivo following transient expression of OSKM cassette in the liver and skeletal muscles of old mice and validated the power of ImAge to assess the heterogeneity of reprogramming. We propose that ImAge represents the first-in-class individual-level biomarker of aging and rejuvenation with single-cell resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562862">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562862" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562862">
        <p class="paperTitle">Perturbation-response analysis of in silico metabolic dynamics: Hard-coded responsiveness in the cofactor and network sparsity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562862" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562862" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Himeoka, Y.; Furusawa, C.</p>
        <p class="info">Score: 10.7, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562862' target='https://doi.org/10.1101/2023.10.18.562862'> 10.1101/2023.10.18.562862</a></p>
        <p class="abstract">Homeostasis is a fundamental characteristic of living systems. Unlike rigidity, homeostasis necessitates that systems respond flexibly to diverse environments. Understanding the dynamics of biochemical systems when subjected to perturbations is essential for the development of a quantitative theory of homeostasis. In this study, we analyze the response of bacterial metabolism to externally imposed perturbations using kinetic models of Escherichia coli&#39;s central carbon metabolism. Our findings indicate that three distinct kinetic models consistently display strong responses to perturbations, wherein minor initial discrepancies in metabolite concentrations from steady-state values amplify over time, resulting in significant deviations. Subsequent numerical studies show that adenyl cofactors consistently influence the responsiveness of the metabolic systems across models. Additionally, we examine the impact of network structure on metabolic dynamics, demonstrating that as the metabolic network becomes denser, the perturbation response diminishes-a trend observed commonly in the models. To confirm the significance of cofactors and network structure, we constructed a simplified metabolic network model, underscoring their importance. By identifying the structural determinants of responsiveness, our findings offer implications for bacterial physiology, the evolution of metabolic networks, and the design principles for robust artificial metabolism in synthetic biology and bioengineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562688">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562688" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562688">
        <p class="paperTitle">The genetic architecture of protein interaction affinity and specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562688" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562688" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bendel, A.; Faure, A.; Klein, D.; Shimada, K.; Kempf, G.; Cavadini, S.; Lehner, B.; Diss, G.</p>
        <p class="info">Score: 10.4, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562688' target='https://doi.org/10.1101/2023.10.17.562688'> 10.1101/2023.10.17.562688</a></p>
        <p class="abstract">Proteins function in crowded cellular environments in which they must bind to specific target proteins but also avoid binding to many other off-target proteins. In large protein families this task is particularly challenging because many off-target proteins have very similar structures. How this specificity of physical protein-protein interactions in cellular networks is encoded and evolves is not very well understood. Here we address the question of specificity-encoding by comprehensively quantifying the effects of all mutations in one protein, JUN, on its binding to all other members of a protein family, the 54 human basic leucine zipper transcription factors. Fitting a global thermodynamic model to the data reveals that most affinity changing mutations equally affect JUN&#39;s propensity to bind to all its interaction partners. Mutations that alter the specificity of binding are much rarer. These specificity-altering mutations are, however, distributed throughout the JUN interaction interface. JUN&#39;s interaction specificity is encoded by both positive determinants that promote on-target interactions and negative determinants that prevent off-target interactions. Indeed, about half of the specificity-defining residues in JUN have dual functions and both promote on-target binding and prevent off-target binding. Whereas nearly all mutations that alter specificity are pleiotropic and also alter the affinity of binding to all interaction partners, the converse is not true with mutations outside of the interface able to tune affinity without affecting specificity. Our results provide the first global view of how mutations in a protein affect binding to all its potential interaction partners and reveal the distributed encoding of specificity and affinity in an interaction interface. They also show how the modular architecture of coiled-coils provides an elegant solution to the challenge of optimising specificity and affinity in a large protein family.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562896">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562896" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562896">
        <p class="paperTitle">The type of carbon source not the growth rate it supports can determine diauxie</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562896" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562896" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huo, Y.; Danecka, W.; Farquhar, I.; Mailliet, K.; Moses, T.; Wallace, E. W.; Swain, P. S.</p>
        <p class="info">Score: 4.1, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562896' target='https://doi.org/10.1101/2023.10.18.562896'> 10.1101/2023.10.18.562896</a></p>
        <p class="abstract">How cells choose between potential carbon sources is a classic example of cellular decision-making, and we know that many organisms prioritise glucose. Yet there has been little investigation of whether other sugars are also preferred, blinkering our view of carbon sensing. Here we study eukaryotic budding yeast and its growth on mixtures of palatinose, an isomer of sucrose, with other sugars. We find that yeast prioritise galactose over palatinose, but not sucrose or fructose, despite all three of these sugars being able to support faster growth than palatinose. Our results therefore disfavour carbon flux-sensing as the sole mechanism. By using genetic perturbations and transcriptomics, we show that repression is active and through Gal4, the master regulator of the GAL regulon. Cells enforce their preference for galactose over palatinose by preventing runaway positive feedback in the MAL regulon, whose genes enable palatinose catabolism. They do so both by repressing MAL11, the gene encoding the palatinose transporter, and by first expressing the isomaltases, IMA1 and IMA5, which cleave palatinose and so prevent its intracellular concentration becoming enough to induce further MAL expression. Our results demonstrate that budding yeast actively maintain a preference for carbon sources other than glucose and that such preferences have been selected by more than differences in growth rates. They imply that carbon-sensing strategies even in unicellular organisms are more complex than previously thought.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553401">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553401" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553401">
        <p class="paperTitle">A robust ultrasensitive transcriptional switch in noisy cellular environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553401" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553401" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jeong, E. M.; Kim, J. K.</p>
        <p class="info">Score: 4.5, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553401' target='https://doi.org/10.1101/2023.08.15.553401'> 10.1101/2023.08.15.553401</a></p>
        <p class="abstract">Ultrasensitive transcriptional switches enable sharp transitions between on and off states of transcription and are essential for cells to respond to environmental cues precisely. However, conventional switches, relying on direct repressor-DNA binding, are extremely sensitive to noise. Here, we discovered an alternative design combining three indirect transcriptional repression mechanisms, sequestration, blocking, and displacement, to generate a noise-resilient ultrasensitive switch. Although sequestration alone can generate an ultrasensitive switch, it remains sensitive to noise because the unintended transcriptional state induced by noise can persist for long periods. However, by jointly utilizing blocking and displacement, these noise-induced transitions can be rapidly restored to the original transcriptional state. Because this transcriptional switch is effective in noisy cellular contexts, it goes beyond previous synthetic transcriptional switches utilizing direct repression mechanisms, making it particularly valuable for robust synthetic system design. Our findings also provide insights into the evolution of robust ultrasensitive switches in real cells. Specifically, the concurrent use of seemingly redundant indirect repression mechanisms in diverse biological systems appears to be a strategy to achieve noise-resilience of ultrasensitive switches.

GRAPHICAL ABSTRACT

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=175 SRC=&#34;FIGDIR/small/553401v2_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (28K):
org.highwire.dtl.DTLVardef@a36772org.highwire.dtl.DTLVardef@1dc8f8borg.highwire.dtl.DTLVardef@1542b20org.highwire.dtl.DTLVardef@db74d2_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563379">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563379" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563379">
        <p class="paperTitle">Modeling alternative translation initiation sites in plants reveals evolutionarily conserved cis-regulatory codes in eukaryotes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563379" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563379" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, T.-Y.; Li, Y.-R.; Chang, K.-J.; Urano, D.; Liu, M.-J.</p>
        <p class="info">Score: 2.1, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563379' target='https://doi.org/10.1101/2023.10.20.563379'> 10.1101/2023.10.20.563379</a></p>
        <p class="abstract">mRNA translation relies on identifying translation initiation sites (TISs) in mRNAs. Alternative TISs are prevalent across plant transcriptomes, but the mechanisms for their recognition are unclear. Using ribosome profiling and machine learning, we developed models for predicting alternative TISs in Arabidopsis thaliana and tomato (Solanum lycopersicum). Distinct feature sets were predictive of AUG and non-AUG TISs in 5&#39; untranslated regions and coding sequences, including a novel CU-rich sequence that promoted plant TIS activity, a translational enhancer found across dicots and monocots and also in humans and viruses. Our results elucidate the mechanistic and evolutionary basis of TIS recognition, whereby cis-regulatory RNA signatures affect start site selection. The TIS prediction model provides global estimates of TISs to discover neglected protein-coding genes across plant genomes. The prevalence of cis-regulatory signatures across eukaryotes and viruses suggests their broad, critical roles in reprogramming the translational landscape in the plant-virus arms race.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562898">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562898" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562898">
        <p class="paperTitle">Machine learning predicts system-wide metabolic flux control in cyanobacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562898" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562898" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kugler, A.; Stensjö, K.</p>
        <p class="info">Score: 2.0, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562898' target='https://doi.org/10.1101/2023.10.18.562898'> 10.1101/2023.10.18.562898</a></p>
        <p class="abstract">Metabolic fluxes and their control mechanisms are fundamental in cellular metabolism, offering insights for the study of biological systems and biotechnological applications. However, quantitative and predictive understanding of controlling biochemical reactions in microbial cell factories, especially at the system level, is limited. In this work, we present ARCTICA, a computational framework that integrates constraint-based modelling with machine learning tools to address this challenge. Using the model cyanobacterium Synechocystis sp. PCC 6803 as chassis, we demonstrate that ARCTICA effectively simulates global-scale metabolic flux control. Key findings are that (i) the photosynthetic bioproduction is mainly governed by enzymes from the Calvin-Benson-Bassham (CBB) cycle, rather than by those involve in the biosynthesis of the end-product, (ii) the catalytic capacity of the CBB cycle limits the photosynthetic activity and downstream pathways and (iii) ribulose-1,5-bisphosphate carboxylase/oxygenase (RuBisCO) is a major, but not the most, limiting step in the CBB cycle. Predicted metabolic reactions qualitatively align with prior experimental observations, validating our modelling approach. ARCTICA serves as a valuable pipeline for understanding cellular physiology and predicting rate-limiting steps in genome-scale metabolic networks, providing guidance for bioengineering of cyanobacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562276">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562276" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562276">
        <p class="paperTitle">Systems-level analyses dissociate genetic regulators of reactive oxygen species and energy production</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562276" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562276" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bennett, N. K.; Lee, M.; Orr, A. L.; Nakamura, K.</p>
        <p class="info">Score: 1.8, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562276' target='https://doi.org/10.1101/2023.10.14.562276'> 10.1101/2023.10.14.562276</a></p>
        <p class="abstract">Respiratory chain dysfunction can decrease ATP and increase reactive oxygen species (ROS) levels. Despite the importance of these metabolic parameters to a wide range of cellular functions and disease, we lack an integrated understanding of how they are differentially regulated. To address this question, we adapted a CRISPRi- and FACS-based platform to compare the effects of respiratory gene knockdown on ROS to their effects on ATP. Focusing on genes whose knockdown is known to decrease mitochondria-derived ATP, we showed that knockdown of genes in specific respiratory chain complexes (I, III and CoQ10 biosynthesis) increased ROS, whereas knockdown of other low ATP hits either had no impact (mitochondrial ribosomal proteins) or actually decreased ROS (complex IV). Moreover, although shifting metabolic conditions profoundly altered mitochondria-derived ATP levels, it had little impact on mitochondrial or cytosolic ROS. In addition, knockdown of a subset of complex I subunits--including NDUFA8, NDUFB4, and NDUFS8--decreased complex I activity, mitochondria-derived ATP and supercomplex level, but knockdown of these genes had differential effects on ROS. Conversely, we found an essential role for ether lipids in the dynamic regulation of mitochondrial ROS levels independent of ATP. Thus, our results identify specific metabolic regulators of cellular ATP and ROS balance that may help dissect the roles of these processes in disease and identify therapeutic strategies to independently target energy failure and oxidative stress.

SignificanceMitochondrial respiration generates both energy (ATP) and reactive oxygen species (ROS). Insufficient energy and increased ROS from respiratory chain dysfunction may be central to the pathophysiology of neurodegenerative diseases and aging. We established a screening platform using CRISPR and fluorescent-cell sorting to compare the impact of decreasing respiratory chain proteins on ROS and ATP levels. The results provide the first systems-level analysis of how ROS and ATP are differentially regulated, and identify genes and respiratory chain complexes that can manipulate each independently. These findings advance our understanding of the relative contributions of ATP and ROS to disease pathophysiology, and guide the development of therapies to preserve energy while minimizing ROS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562348">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562348" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562348">
        <p class="paperTitle">Minimally Sufficient Experimental Design using Identifiability Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562348" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562348" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kareva, I.; Gevertz, J. L.</p>
        <p class="info">Score: 1.5, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562348' target='https://doi.org/10.1101/2023.10.14.562348'> 10.1101/2023.10.14.562348</a></p>
        <p class="abstract">Mathematical models are increasingly being developed and calibrated in tandem with data collection, empowering scientists to intervene in real time based on quantitative model predictions. Well-designed experiments can help augment the predictive power of a mathematical model but the question of when to collect data to maximize its utility for a model is non-trivial. Here we define data as model-informative if it results in a unique parametrization, assessed through the lens of practical identifiability. The framework we propose identifies an optimal experimental design (how much data to collect and when to collect it) that ensures parameter identifiability (permitting confidence in model predictions), while minimizing experimental time and costs. We demonstrate the power of the method by applying it to a modified version of a classic site-of-action pharmacokinetic/pharmacodynamic model that describes distribution of a drug into the tumor microenvironment (TME), where its efficacy is dependent on the level of target occupancy in the TME. In this context, we identify a minimal set of time points when data needs to be collected that robustly ensures practical identifiability of model parameters. The proposed methodology can be applied broadly to any mathematical model, allowing for the identification of a minimally sufficient experimental design that collects the most informative data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562774">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562774" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562774">
        <p class="paperTitle">Investigating RNA splicing as a source of cellular diversity using a binomial mixture model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562774" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562774" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Isaev, K.; Knowles, D. A.</p>
        <p class="info">Score: 1.2, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562774' target='https://doi.org/10.1101/2023.10.17.562774'> 10.1101/2023.10.17.562774</a></p>
        <p class="abstract">Alternative splicing (AS) contributes significantly to RNA and protein variability yet its role in defining cellular diversity is not fully understood. While Smart-seq2 offers enhanced coverage across transcripts compared to 10X single cell RNA-sequencing (scRNA-seq), current computational methods often miss the full complexity of AS. Most approaches for single cell based differential splicing analysis focus on simple AS events such as exon skipping, and rely on predefined cell type labels or low-dimensional gene expression representations. This limits their ability to detect more complex AS events and makes them dependent on prior knowledge of cell classifications. Here, we present Leaflet, a splice junction centric approach inspired by Leafcutter, our tool for quantifying RNA splicing variation with bulk RNA-seq. Leaflet is a probabilistic mixture model designed to infer AS-driven cell states without the need for cell type labels. We detail Leaflet&#39;s generative model, inference methodology, and its efficiency in detecting differentially spliced junctions. By applying Leaflet to the Tabula Muris brain cell dataset, we highlight cell-state specific splicing patterns, offering a deeper insight into cellular diversity beyond that captured by gene expression alone.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
