<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A comprehensive atlas of the aging vertebrate brain reveals signatures of progressive proteostasis dysfunction
Authors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.
Score: 25.5, Published: 2023-07-24 DOI: 10.1101/2023.07.20.549210
Protein homeostasis is disrupted in aging and neurodegenerative diseases, yet, the specific impact of aging on brain proteostasis remains poorly understood.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="A comprehensive atlas of the aging vertebrate brain reveals signatures of progressive proteostasis dysfunction
Authors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.
Score: 25.5, Published: 2023-07-24 DOI: 10.1101/2023.07.20.549210
Protein homeostasis is disrupted in aging and neurodegenerative diseases, yet, the specific impact of aging on brain proteostasis remains poorly understood." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="A comprehensive atlas of the aging vertebrate brain reveals signatures of progressive proteostasis dysfunction
Authors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.
Score: 25.5, Published: 2023-07-24 DOI: 10.1101/2023.07.20.549210
Protein homeostasis is disrupted in aging and neurodegenerative diseases, yet, the specific impact of aging on brain proteostasis remains poorly understood."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "A comprehensive atlas of the aging vertebrate brain reveals signatures of progressive proteostasis dysfunction\nAuthors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.\nScore: 25.5, Published: 2023-07-24 DOI: 10.1101/2023.07.20.549210\nProtein homeostasis is disrupted in aging and neurodegenerative diseases, yet, the specific impact of aging on brain proteostasis remains poorly understood.",
  "keywords": [
    
  ],
  "articleBody": " A comprehensive atlas of the aging vertebrate brain reveals signatures of progressive proteostasis dysfunction\nAuthors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.\nScore: 25.5, Published: 2023-07-24 DOI: 10.1101/2023.07.20.549210\nProtein homeostasis is disrupted in aging and neurodegenerative diseases, yet, the specific impact of aging on brain proteostasis remains poorly understood. Here, we measured and integrated the effects of aging on the transcriptome, translatome, and multiple layers of the proteome in the brain of a short-lived killifish. We find that aging causes a decoupling between transcriptome and proteome. This leads to decreased abundance of proteins enriched in basic amino acids such as DNA/RNA-binding proteins and increased levels of others, independent of mRNA changes. Chronic proteasome impairment in vivo induces aging signatures in lysosomes and mitochondria. However, it does not recapitulate the age-related decoupling between transcripts and proteins. Instead, aberrant translation pausing and ensuing reduced ribosome availability reprogram the proteome independently of transcription. The age-linked changes in protein biogenesis likely enhance aggregation and reduce availability of key protein complexes, thus contributing to proteome dysfunction and aging hallmarks in older brains. HighlightsO_LIA resource of protein solubility, organelle composition and PTMs in the aging vertebrate brain C_LIO_LIProteostasis alterations converge on mitochondria, ribosomes and biosynthetic pathways C_LIO_LIBasic proteins, e.g., DNA/RNA binding, are reduced due to translation pausing C_LIO_LIDecreased ribosome availability reprograms protein synthesis in old brains C_LI O_FIG O_LINKSMALLFIG WIDTH=199 HEIGHT=200 SRC=\"FIGDIR/small/549210v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (46K): org.highwire.dtl.DTLVardef@b10be3org.highwire.dtl.DTLVardef@1839aceorg.highwire.dtl.DTLVardef@1285150org.highwire.dtl.DTLVardef@1f27987_HPS_FORMAT_FIGEXP M_FIG C_FIG\nSystems Age: A single blood methylation test to quantify aging heterogeneity across 11 physiological systems\nAuthors: Sehgal, R.; Meer, M.; Shadyab, A. H.; Casanova, R.; Manson, J. E.; Bhatti, P.; Crimmins, E. M.; Assimes, T. L.; Whitsel, E. A.; Higgins-Chen, A. T.; Levine, M.\nScore: 40.4, Published: 2023-07-17 DOI: 10.1101/2023.07.13.548904\nIndividuals, organs, tissues, and cells age in diverse ways throughout the lifespan. Epigenetic clocks attempt to quantify differential aging between individuals, but they typically summarize aging as a single measure, ignoring within-person heterogeneity. Our aim was to develop novel systems-based methylation clocks that, when assessed in blood, capture aging in distinct physiological systems. We combined supervised and unsupervised machine learning methods to link DNA methylation, system-specific clinical chemistry and functional measures, and mortality risk. This yielded a panel of 11 system-specific scores- Heart, Lung, Kidney, Liver, Brain, Immune, Inflammatory, Blood, Musculoskeletal, Hormone, and Metabolic. Each system score predicted a wide variety of outcomes, aging phenotypes, and conditions specific to the respective system, and often did so more strongly than existing epigenetic clocks that report single global measures. We also combined the system scores into a composite Systems Age clock that is predictive of aging across physiological systems in an unbiased manner. Finally, we showed that the system scores clustered individuals into unique aging subtypes that had different patterns of age-related disease and decline. Overall, our biological systems based epigenetic framework captures aging in multiple physiological systems using a single blood draw and assay and may inform the development of more personalized clinical approaches for improving age-related quality of life.\nThe timing of cellular events: a stochastic vs deterministic perspective\nAuthors: Ham, L.; Coomer, M. A.; Öcal, K.; Grima, R.; Stumpf, M. P. H.\nScore: 8.4, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549956\nChanges in cell state are driven by key molecular events whose timing can often be measured experimentally. Of particular interest is the time taken for the levels of RNA or protein molecules to reach a critical threshold defining the triggering of a cellular event. While this mean trigger time can be estimated by numerical integration of deterministic models, these ignore intrinsic noise and hence their predictions may be inaccurate. Here we study the differences between deterministic and stochastic model predictions for the mean trigger times using simple models of gene expression, post-transcriptional feedback control, and enzyme-mediated catalysis. By comparison of the two predictions, we show that when promoter switching is present there exists a transition from a parameter regime where deterministic models predict a longer trigger time than stochastic models to a regime where the opposite occurs. Furthermore, the ratio of the trigger times of the two models can be large, particularly for auto-regulatory genetic feedback loops. Our theory provides intuitive insight into the origin of these effects and shows that deterministic predictions for cellular event timing can be highly inaccurate when molecule numbers are within the range known for many cells.\nSpatial Metabolome Lipidome and Glycome from a Single brain Section\nAuthors: Clarke, H. A.; Ma, X.; shedlock, c. J.; medina, t.; Hawkinson, T. R.; Wu, L.; Robas, R. A.; Ravi, S.; Bizon, J. L. L.; Burke, S. N.; Abisambra, J. F.; Merritt, M. E.; Prentice, B. M.; Vander Kooi, C. W.; Gentry, m. S.; Chen, L.; sun, r. c.\nScore: 7.8, Published: 2023-07-25 DOI: 10.1101/2023.07.22.550155\nMetabolites, lipids, and glycans are fundamental biomolecules involved in complex biological systems. They are metabolically channeled through a myriad of pathways and molecular processes that define the physiology and pathology of an organism. Here, we present a blueprint for the simultaneous analysis of spatial metabolome, lipidome, and glycome from a single tissue section using mass spectrometry imaging. Complimenting an original experimental protocol, our workflow includes a computational framework called Spatial Augmented Multiomics Interface (Sami) that offers multiomics integration, high dimensionality clustering, spatial anatomical mapping with matched multiomics features, and metabolic pathway enrichment to providing unprecedented insights into the spatial distribution and interaction of these biomolecules in mammalian tissue biology.\nBiologically meaningful regulatory logic enhances the convergence rate in Boolean networks and bushiness of their state transition graph\nAuthors: Sil, P.; Subbaroyan, A.; Kulkarni, S.; Martin, O. C.; Samal, A.\nScore: 5.3, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549398\nBoolean network (BN) models of gene regulatory networks (GRNs) have gained widespread traction as they can easily recapitulate cellular phenotypes via their attractor states. The overall dynamics of such models are embodied in the systems state transition graph (STG) which is highly informative. Indeed, even if two BN models have the same network structure and recover the same attractors, their STGs can be drastically different depending on the type of regulatory logic rules or Boolean functions (BFs) employed. A key objective of the present work is to systematically delineate the effects of different classes of regulatory logic rules on the structural features of the STG of reconstructed Boolean GRNs, while keeping BN structure and biological attractors fixed. Furthermore, we ask how such global features might be driven by characteristics of the underlying BFs. For that, we draw from ideas and concepts proposed in cellular automata for both the structural features and their associated proxies. We use the network of 10 reconstructed Boolean GRNs to generate ensembles that differ in the type of logic used while keeping their structure fixed and recovering their biological attractors, and compute quantities associated with the structural features of the STG: bushiness and convergence, that are based on the number of garden-of-Eden (GoE) states and transient times to reach attractor states when originating at them. We find that ensembles employing biologically meaningful BFs have higher bushiness and convergence than those employing random ones. Computing these global measures gets expensive with larger network sizes, stressing the need for more feasible proxies. We thus adapt Wuensches Z-parameter to BFs in BNs and provide 4 natural variants, which along with the network sensitivity, comprise our descriptors of local dynamics. One variant of the network Z-parameter as well as the network sensitivity correlate particularly very well with the bushiness, serving as a good proxy for the same. Finally, we provide an excellent proxy for the convergence based on computing transient lengths originating at random states rather than GoE states.\nProstate Cancer Reshapes the Secreted and Extracellular Vesicle Urinary Proteomes\nAuthors: Khoo, A.; Govindarajan, M.; Qui, Z.; Liu, L. Y.; Ignatchenko, V.; Waas, M.; Macklin, A.; Keszei, A.; Main, B. P.; Yang, L.; Lance, R. S.; Downes, M. R.; Semmes, O. J.; Vesprini, D.; Liu, S. K.; Nyalwidhe, J. O.; Boutros, P. C.; Kislinger, T.\nScore: 5.1, Published: 2023-07-25 DOI: 10.1101/2023.07.23.550214\nUrine is a complex biofluid that reflects both overall physiologic state and the state of the genitourinary tissues through which it passes. It contains both secreted proteins and proteins encapsulated in tissue-derived extracellular vesicles (EVs). To understand the population variability and clinical utility of urine, we quantified the secreted and EV proteomes from 190 men, including a subset with prostate cancer. We demonstrate that a simple protocol enriches prostatic proteins in urine. Secreted and EV proteins arise from different subcellular compartments. Urinary EVs are faithful surrogates of tissue proteomes, but secreted proteins in urine or cell line EVs are not. The urinary proteome is longitudinally stable over several years. It can accurately and non-invasively distinguish malignant from benign prostatic lesions, and can risk-stratify prostate tumors. This resource quantifies the complexity of the urinary proteome, and reveals the synergistic value of secreted and EV proteomes for translational and biomarker studies.\nA systems-level analysis of the mutually antagonistic roles of RKIP and BACH1 in dynamics of cancer cell plasticity\nAuthors: Shyam, S.; R, S.; Sehgal, M.; Jolly, M. K.\nScore: 10.2, Published: 2023-07-08 DOI: 10.1101/2023.07.08.547338\nPhenotypic plasticity is a hallmark of cancer metastasis. Epithelial-mesenchymal transition (EMT) is an important axis of phenotypic plasticity. Raf kinase-B inhibitor protein (RKIP) and BTB and CNC homology 1 (BACH1) are two proteins reported to influence EMT. In breast cancer, they act antagonistically, but the exact nature of their roles in mediating EMT and associated other axes of plasticity remains unclear. Here, analysing transcriptomic data, we reveal their antagonistic trends in a pan-cancer manner, in terms of association with EMT, metabolic reprogramming and immune evasion via PD-L1. Next, we developed and simulated a mechanism-based gene regulatory network that captures how RKIP and BACH1 engage in feedback loops with drivers of EMT and stemness. We found that RKIP and BACH1 belong to two separate \"teams\" of players - while BACH1 belonged to the one that drove pro-EMT, stem-like and therapy-resistant cell-states, RKIP is a member of a team that enables pro-epithelial, less stem-like and therapy-sensitive phenotypes. Finally, we observed that low RKIP levels and concomitant upregulated BACH1 levels associated with worse clinical outcomes in many cancer types. Together, our systems-level analysis indicates that the emergent dynamics of underlying regulatory network underlie the antagonistic patterns of RKIP and BACH1 with various axes of cancer cell plasticity, as well as with patient survival data.\nQuantifying mutational synergy using computational models predicts survival in haematological cancers\nAuthors: Norris, R.; Jones, J.; Mancini, E.; Chevassut, T.; Pepper, C.; Pepper, A.; Mitchell, S.\nScore: 4.1, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548371\nGenetic heterogeneity and co-occurring driver mutations contribute to poor clinical outcomes in cancer. However, the impact of multiple mutations on complex signalling networks is not easily predicted. We found that, by placing mutations into their cellular context, multi-scale agent- based mathematical models could predict how genetic events combine in haematological malignancies. Simulations of lymphoma and myeloma predicted co-occurring mutations synergised to increase tumour cell expansion beyond what would be expected from the impact of the individual mutations alone. Mutational synergy between MYC and BCL2 was consistent with the more aggressive disease course of patients with double-hit lymphoma, and mutational synergy between MCL1 and CKS1B was predictive of outcome in patients with gain 1q multiple myeloma. Incorporating patient-specific mutational profiles into personalised models of lymphoma patients with the worst clinical outcomes revealed a correlation between simulated mutational synergy and overall survival, which outperformed widely used classifications of lymphoma informed by gene expression or mutational data alone. Our results demonstrated that mutational synergy scores enabled prediction of the impact of co-occurring mutations and may improve personalised prognostic predictions.\nRegulatory start-stop elements in 5' untranslated regions pervasively modulate translation\nAuthors: Rendleman, J.; Haizel, S.; Wu, S.; Ge, X.; Liu, J.; Zou, H.; Mohammad, M. P.; Pressler, M.; Maity, S.; Hronova, V.; Gao, Z.; Herrmannova, A.; Lei, A.; Allgoewer, K.; Sultanov, D.; Hinckley, W. E.; Szkop, K.; Topisirovic, I.; Larsson, O.; Hatzoglou, M.; Valasek, L. S.; Vogel, C.\nScore: 7.5, Published: 2023-07-18 DOI: 10.1101/2021.07.26.453809\nVarious mechanisms controlling ribosome scanning and initiation have been identified within the 5 untranslated region (UTR) of eukaryotic genes, including upstream open reading frames, specific sequence motifs, and secondary structures. We describe the impact and distinct characteristics of a translation regulatory element consisting of a start codon immediately followed by a stop codon. While to date this start-stop element has been described only for specific cases, we identified them in the 5UTR of hundreds of human genes whereby their general impact on translation control in mammalian cells has been unnoticed. We show that the element is evolutionarily conserved and represses translation of the downstream open reading frames. While having no apparent impact on translation termination or RNA stability, start-stops efficiently halt ribosomes and act as a barrier for the scanning of the small ribosomal subunit. Start-stop element occurs in many transcription factors and key signaling genes, and we highlight routes in which they can affect cellular fate. We investigate the start-stop element in several genes, i.e. MORF4L1, IFNAR1, SLC39A1, and PSPC1, and delineate the role of the start-stop in translation regulation of ATF4 mRNA. Thus, start-stops uniquely modulate translation through elongation-less ribosome stalling.\nTranscriptomic signatures of progressive and regressive liver fibrosis and portal hypertension\nAuthors: Petrenko, O.; Koenigshofer, P.; Brusilovskaya, K.; Hofer, B. S.; Bareiner, K.; Simbrunner, B.; Trauner, M.; Kauschke, S. G.; Pfisterer, L.; Simon, E.; Rendeiro, A.; de Rooij, L.; Schwabl, P.; Reiberger, T.\nScore: 1.2, Published: 2023-07-25 DOI: 10.1101/2023.07.22.550133\nPersistent liver injury triggers a fibrogenic program that causes pathologic remodelling of the hepatic microenvironment (i.e., liver fibrosis) and portal hypertension. The dynamics of gene regulation during liver disease progression and regression remain understudied. Here, we generated hepatic transcriptome profiles in two well-established liver disease models at peak fibrosis and during spontaneous regression after the removal of the inducing agents. We linked the dynamics of key liver disease readouts, such as portal pressure, collagen proportionate area, and transaminase serum levels, to most differentially expressed genes, enabling the identification of transcriptomic signatures of progressive vs. regressive liver fibrosis and portal hypertension. These candidate biomarkers (e.g., Scube1, Tcf4, Src, Hmga1, Trem2, Mafk, Mmp7) were also validated in RNA-seq datasets of patients with cirrhosis and portal hypertension. Finally, deconvolution analysis identified major cell types and suggested an association of macrophage and portal hepatocyte signatures with portal hypertension and fibrosis area in both models.\n",
  "wordCount" : "2401",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549210">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549210" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549210">
        <p class="paperTitle">A comprehensive atlas of the aging vertebrate brain reveals signatures of progressive proteostasis dysfunction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549210" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549210" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Di Fraia, D.; Marino, A.; Lee, J. H.; Kelmer Sacramento, E.; Baumgart, M.; Bagnoli, S.; Tomaz da Silva, P.; Kumar Sahu, A.; Siano, G.; Tiessen, M.; Gagneur, J.; Frydman, J.; Cellerino, A.; Ori, A.</p>
        <p class="info">Score: 25.5, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549210' target='https://doi.org/10.1101/2023.07.20.549210'> 10.1101/2023.07.20.549210</a></p>
        <p class="abstract">Protein homeostasis is disrupted in aging and neurodegenerative diseases, yet, the specific impact of aging on brain proteostasis remains poorly understood. Here, we measured and integrated the effects of aging on the transcriptome, translatome, and multiple layers of the proteome in the brain of a short-lived killifish. We find that aging causes a decoupling between transcriptome and proteome. This leads to decreased abundance of proteins enriched in basic amino acids such as DNA/RNA-binding proteins and increased levels of others, independent of mRNA changes. Chronic proteasome impairment in vivo induces aging signatures in lysosomes and mitochondria. However, it does not recapitulate the age-related decoupling between transcripts and proteins. Instead, aberrant translation pausing and ensuing reduced ribosome availability reprogram the proteome independently of transcription. The age-linked changes in protein biogenesis likely enhance aggregation and reduce availability of key protein complexes, thus contributing to proteome dysfunction and aging hallmarks in older brains.

HighlightsO_LIA resource of protein solubility, organelle composition and PTMs in the aging vertebrate brain
C_LIO_LIProteostasis alterations converge on mitochondria, ribosomes and biosynthetic pathways
C_LIO_LIBasic proteins, e.g., DNA/RNA binding, are reduced due to translation pausing
C_LIO_LIDecreased ribosome availability reprograms protein synthesis in old brains
C_LI



O_FIG O_LINKSMALLFIG WIDTH=199 HEIGHT=200 SRC=&#34;FIGDIR/small/549210v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (46K):
org.highwire.dtl.DTLVardef@b10be3org.highwire.dtl.DTLVardef@1839aceorg.highwire.dtl.DTLVardef@1285150org.highwire.dtl.DTLVardef@1f27987_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548904">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548904" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548904">
        <p class="paperTitle">Systems Age: A single blood methylation test to quantify aging heterogeneity across 11 physiological systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548904" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548904" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sehgal, R.; Meer, M.; Shadyab, A. H.; Casanova, R.; Manson, J. E.; Bhatti, P.; Crimmins, E. M.; Assimes, T. L.; Whitsel, E. A.; Higgins-Chen, A. T.; Levine, M.</p>
        <p class="info">Score: 40.4, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548904' target='https://doi.org/10.1101/2023.07.13.548904'> 10.1101/2023.07.13.548904</a></p>
        <p class="abstract">Individuals, organs, tissues, and cells age in diverse ways throughout the lifespan. Epigenetic clocks attempt to quantify differential aging between individuals, but they typically summarize aging as a single measure, ignoring within-person heterogeneity. Our aim was to develop novel systems-based methylation clocks that, when assessed in blood, capture aging in distinct physiological systems. We combined supervised and unsupervised machine learning methods to link DNA methylation, system-specific clinical chemistry and functional measures, and mortality risk. This yielded a panel of 11 system-specific scores- Heart, Lung, Kidney, Liver, Brain, Immune, Inflammatory, Blood, Musculoskeletal, Hormone, and Metabolic. Each system score predicted a wide variety of outcomes, aging phenotypes, and conditions specific to the respective system, and often did so more strongly than existing epigenetic clocks that report single global measures. We also combined the system scores into a composite Systems Age clock that is predictive of aging across physiological systems in an unbiased manner. Finally, we showed that the system scores clustered individuals into unique aging subtypes that had different patterns of age-related disease and decline. Overall, our biological systems based epigenetic framework captures aging in multiple physiological systems using a single blood draw and assay and may inform the development of more personalized clinical approaches for improving age-related quality of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549956">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549956" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549956">
        <p class="paperTitle">The timing of cellular events: a stochastic vs deterministic perspective</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549956" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549956" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ham, L.; Coomer, M. A.; Öcal, K.; Grima, R.; Stumpf, M. P. H.</p>
        <p class="info">Score: 8.4, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549956' target='https://doi.org/10.1101/2023.07.20.549956'> 10.1101/2023.07.20.549956</a></p>
        <p class="abstract">Changes in cell state are driven by key molecular events whose timing can often be measured experimentally. Of particular interest is the time taken for the levels of RNA or protein molecules to reach a critical threshold defining the triggering of a cellular event. While this mean trigger time can be estimated by numerical integration of deterministic models, these ignore intrinsic noise and hence their predictions may be inaccurate. Here we study the differences between deterministic and stochastic model predictions for the mean trigger times using simple models of gene expression, post-transcriptional feedback control, and enzyme-mediated catalysis. By comparison of the two predictions, we show that when promoter switching is present there exists a transition from a parameter regime where deterministic models predict a longer trigger time than stochastic models to a regime where the opposite occurs. Furthermore, the ratio of the trigger times of the two models can be large, particularly for auto-regulatory genetic feedback loops. Our theory provides intuitive insight into the origin of these effects and shows that deterministic predictions for cellular event timing can be highly inaccurate when molecule numbers are within the range known for many cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.22.550155">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.22.550155" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.22.550155">
        <p class="paperTitle">Spatial Metabolome Lipidome and Glycome from a Single brain Section</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.22.550155" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.22.550155" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Clarke, H. A.; Ma, X.; shedlock, c. J.; medina, t.; Hawkinson, T. R.; Wu, L.; Robas, R. A.; Ravi, S.; Bizon, J. L. L.; Burke, S. N.; Abisambra, J. F.; Merritt, M. E.; Prentice, B. M.; Vander Kooi, C. W.; Gentry, m. S.; Chen, L.; sun, r. c.</p>
        <p class="info">Score: 7.8, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.22.550155' target='https://doi.org/10.1101/2023.07.22.550155'> 10.1101/2023.07.22.550155</a></p>
        <p class="abstract">Metabolites, lipids, and glycans are fundamental biomolecules involved in complex biological systems. They are metabolically channeled through a myriad of pathways and molecular processes that define the physiology and pathology of an organism. Here, we present a blueprint for the simultaneous analysis of spatial metabolome, lipidome, and glycome from a single tissue section using mass spectrometry imaging. Complimenting an original experimental protocol, our workflow includes a computational framework called Spatial Augmented Multiomics Interface (Sami) that offers multiomics integration, high dimensionality clustering, spatial anatomical mapping with matched multiomics features, and metabolic pathway enrichment to providing unprecedented insights into the spatial distribution and interaction of these biomolecules in mammalian tissue biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549398">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549398" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549398">
        <p class="paperTitle">Biologically meaningful regulatory logic enhances the convergence rate in Boolean networks and bushiness of their state transition graph</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549398" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549398" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sil, P.; Subbaroyan, A.; Kulkarni, S.; Martin, O. C.; Samal, A.</p>
        <p class="info">Score: 5.3, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549398' target='https://doi.org/10.1101/2023.07.17.549398'> 10.1101/2023.07.17.549398</a></p>
        <p class="abstract">Boolean network (BN) models of gene regulatory networks (GRNs) have gained widespread traction as they can easily recapitulate cellular phenotypes via their attractor states. The overall dynamics of such models are embodied in the systems state transition graph (STG) which is highly informative. Indeed, even if two BN models have the same network structure and recover the same attractors, their STGs can be drastically different depending on the type of regulatory logic rules or Boolean functions (BFs) employed. A key objective of the present work is to systematically delineate the effects of different classes of regulatory logic rules on the structural features of the STG of reconstructed Boolean GRNs, while keeping BN structure and biological attractors fixed. Furthermore, we ask how such global features might be driven by characteristics of the underlying BFs. For that, we draw from ideas and concepts proposed in cellular automata for both the structural features and their associated proxies. We use the network of 10 reconstructed Boolean GRNs to generate ensembles that differ in the type of logic used while keeping their structure fixed and recovering their biological attractors, and compute quantities associated with the structural features of the STG:  bushiness and  convergence, that are based on the number of garden-of-Eden (GoE) states and transient times to reach attractor states when originating at them. We find that ensembles employing biologically meaningful BFs have higher  bushiness and  convergence than those employing random ones. Computing these  global measures gets expensive with larger network sizes, stressing the need for more feasible proxies. We thus adapt Wuensches Z-parameter to BFs in BNs and provide 4 natural variants, which along with the network sensitivity, comprise our descriptors of local dynamics. One variant of the network Z-parameter as well as the network sensitivity correlate particularly very well with the bushiness, serving as a good proxy for the same. Finally, we provide an excellent proxy for the  convergence based on computing transient lengths originating at random states rather than GoE states.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.23.550214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.23.550214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.23.550214">
        <p class="paperTitle">Prostate Cancer Reshapes the Secreted and Extracellular Vesicle Urinary Proteomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.23.550214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.23.550214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khoo, A.; Govindarajan, M.; Qui, Z.; Liu, L. Y.; Ignatchenko, V.; Waas, M.; Macklin, A.; Keszei, A.; Main, B. P.; Yang, L.; Lance, R. S.; Downes, M. R.; Semmes, O. J.; Vesprini, D.; Liu, S. K.; Nyalwidhe, J. O.; Boutros, P. C.; Kislinger, T.</p>
        <p class="info">Score: 5.1, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.23.550214' target='https://doi.org/10.1101/2023.07.23.550214'> 10.1101/2023.07.23.550214</a></p>
        <p class="abstract">Urine is a complex biofluid that reflects both overall physiologic state and the state of the genitourinary tissues through which it passes. It contains both secreted proteins and proteins encapsulated in tissue-derived extracellular vesicles (EVs). To understand the population variability and clinical utility of urine, we quantified the secreted and EV proteomes from 190 men, including a subset with prostate cancer. We demonstrate that a simple protocol enriches prostatic proteins in urine. Secreted and EV proteins arise from different subcellular compartments. Urinary EVs are faithful surrogates of tissue proteomes, but secreted proteins in urine or cell line EVs are not. The urinary proteome is longitudinally stable over several years. It can accurately and non-invasively distinguish malignant from benign prostatic lesions, and can risk-stratify prostate tumors. This resource quantifies the complexity of the urinary proteome, and reveals the synergistic value of secreted and EV proteomes for translational and biomarker studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.08.547338">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.08.547338" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.08.547338">
        <p class="paperTitle">A systems-level analysis of the mutually antagonistic roles of RKIP and BACH1 in dynamics of cancer cell plasticity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.08.547338" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.08.547338" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shyam, S.; R, S.; Sehgal, M.; Jolly, M. K.</p>
        <p class="info">Score: 10.2, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.08.547338' target='https://doi.org/10.1101/2023.07.08.547338'> 10.1101/2023.07.08.547338</a></p>
        <p class="abstract">Phenotypic plasticity is a hallmark of cancer metastasis. Epithelial-mesenchymal transition (EMT) is an important axis of phenotypic plasticity. Raf kinase-B inhibitor protein (RKIP) and BTB and CNC homology 1 (BACH1) are two proteins reported to influence EMT. In breast cancer, they act antagonistically, but the exact nature of their roles in mediating EMT and associated other axes of plasticity remains unclear. Here, analysing transcriptomic data, we reveal their antagonistic trends in a pan-cancer manner, in terms of association with EMT, metabolic reprogramming and immune evasion via PD-L1. Next, we developed and simulated a mechanism-based gene regulatory network that captures how RKIP and BACH1 engage in feedback loops with drivers of EMT and stemness. We found that RKIP and BACH1 belong to two separate &#34;teams&#34; of players - while BACH1 belonged to the one that drove pro-EMT, stem-like and therapy-resistant cell-states, RKIP is a member of a team that enables pro-epithelial, less stem-like and therapy-sensitive phenotypes. Finally, we observed that low RKIP levels and concomitant upregulated BACH1 levels associated with worse clinical outcomes in many cancer types. Together, our systems-level analysis indicates that the emergent dynamics of underlying regulatory network underlie the antagonistic patterns of RKIP and BACH1 with various axes of cancer cell plasticity, as well as with patient survival data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548371">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548371" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548371">
        <p class="paperTitle">Quantifying mutational synergy using computational models predicts survival in haematological cancers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548371" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548371" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Norris, R.; Jones, J.; Mancini, E.; Chevassut, T.; Pepper, C.; Pepper, A.; Mitchell, S.</p>
        <p class="info">Score: 4.1, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548371' target='https://doi.org/10.1101/2023.07.10.548371'> 10.1101/2023.07.10.548371</a></p>
        <p class="abstract">Genetic heterogeneity and co-occurring driver mutations contribute to poor clinical outcomes in cancer. However, the impact of multiple mutations on complex signalling networks is not easily predicted. We found that, by placing mutations into their cellular context, multi-scale agent- based mathematical models could predict how genetic events combine in haematological malignancies. Simulations of lymphoma and myeloma predicted co-occurring mutations synergised to increase tumour cell expansion beyond what would be expected from the impact of the individual mutations alone. Mutational synergy between MYC and BCL2 was consistent with the more aggressive disease course of patients with double-hit lymphoma, and mutational synergy between MCL1 and CKS1B was predictive of outcome in patients with gain 1q multiple myeloma. Incorporating patient-specific mutational profiles into personalised models of lymphoma patients with the worst clinical outcomes revealed a correlation between simulated mutational synergy and overall survival, which outperformed widely used classifications of lymphoma informed by gene expression or mutational data alone. Our results demonstrated that mutational synergy scores enabled prediction of the impact of co-occurring mutations and may improve personalised prognostic predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2021.07.26.453809">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2021.07.26.453809" aria-expanded="false" aria-controls="flush-collapse10.1101/2021.07.26.453809">
        <p class="paperTitle">Regulatory start-stop elements in 5&#39; untranslated regions pervasively modulate translation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2021.07.26.453809" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2021.07.26.453809" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rendleman, J.; Haizel, S.; Wu, S.; Ge, X.; Liu, J.; Zou, H.; Mohammad, M. P.; Pressler, M.; Maity, S.; Hronova, V.; Gao, Z.; Herrmannova, A.; Lei, A.; Allgoewer, K.; Sultanov, D.; Hinckley, W. E.; Szkop, K.; Topisirovic, I.; Larsson, O.; Hatzoglou, M.; Valasek, L. S.; Vogel, C.</p>
        <p class="info">Score: 7.5, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2021.07.26.453809' target='https://doi.org/10.1101/2021.07.26.453809'> 10.1101/2021.07.26.453809</a></p>
        <p class="abstract">Various mechanisms controlling ribosome scanning and initiation have been identified within the 5 untranslated region (UTR) of eukaryotic genes, including upstream open reading frames, specific sequence motifs, and secondary structures. We describe the impact and distinct characteristics of a translation regulatory element consisting of a start codon immediately followed by a stop codon. While to date this start-stop element has been described only for specific cases, we identified them in the 5UTR of hundreds of human genes whereby their general impact on translation control in mammalian cells has been unnoticed. We show that the element is evolutionarily conserved and represses translation of the downstream open reading frames. While having no apparent impact on translation termination or RNA stability, start-stops efficiently halt ribosomes and act as a barrier for the scanning of the small ribosomal subunit. Start-stop element occurs in many transcription factors and key signaling genes, and we highlight routes in which they can affect cellular fate. We investigate the start-stop element in several genes, i.e. MORF4L1, IFNAR1, SLC39A1, and PSPC1, and delineate the role of the start-stop in translation regulation of ATF4 mRNA. Thus, start-stops uniquely modulate translation through elongation-less ribosome stalling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.22.550133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.22.550133" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.22.550133">
        <p class="paperTitle">Transcriptomic signatures of progressive and regressive liver fibrosis and portal hypertension</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.22.550133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.22.550133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Petrenko, O.; Koenigshofer, P.; Brusilovskaya, K.; Hofer, B. S.; Bareiner, K.; Simbrunner, B.; Trauner, M.; Kauschke, S. G.; Pfisterer, L.; Simon, E.; Rendeiro, A.; de Rooij, L.; Schwabl, P.; Reiberger, T.</p>
        <p class="info">Score: 1.2, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.22.550133' target='https://doi.org/10.1101/2023.07.22.550133'> 10.1101/2023.07.22.550133</a></p>
        <p class="abstract">Persistent liver injury triggers a fibrogenic program that causes pathologic remodelling of the hepatic microenvironment (i.e., liver fibrosis) and portal hypertension. The dynamics of gene regulation during liver disease progression and regression remain understudied. Here, we generated hepatic transcriptome profiles in two well-established liver disease models at peak fibrosis and during spontaneous regression after the removal of the inducing agents. We linked the dynamics of key liver disease readouts, such as portal pressure, collagen proportionate area, and transaminase serum levels, to most differentially expressed genes, enabling the identification of transcriptomic signatures of progressive vs. regressive liver fibrosis and portal hypertension. These candidate biomarkers (e.g., Scube1, Tcf4, Src, Hmga1, Trem2, Mafk, Mmp7) were also validated in RNA-seq datasets of patients with cirrhosis and portal hypertension. Finally, deconvolution analysis identified major cell types and suggested an association of macrophage and portal hepatocyte signatures with portal hypertension and fibrosis area in both models.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
