<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Digitize your Biology! Modeling multicellular systems through interpretable cell behavior
Authors: Johnson, J. A. I.; Stein-O&#39;Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Digitize your Biology! Modeling multicellular systems through interpretable cell behavior
Authors: Johnson, J. A. I.; Stein-O&#39;Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-27T10:39:11+00:00" />
<meta property="article:modified_time" content="2023-09-27T10:39:11+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Digitize your Biology! Modeling multicellular systems through interpretable cell behavior
Authors: Johnson, J. A. I.; Stein-O&#39;Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Digitize your Biology! Modeling multicellular systems through interpretable cell behavior\nAuthors: Johnson, J. A. I.; Stein-O\u0026#39;Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N.",
  "keywords": [
    
  ],
  "articleBody": " Digitize your Biology! Modeling multicellular systems through interpretable cell behavior\nAuthors: Johnson, J. A. I.; Stein-O'Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N.; Zheng, L.; Zimmerman, J. W.; Jaffee, E. M.; Chang, Y. H.; Coussens, L.; Gray, J.; Heiser, L. M.; Fertig, E. J.; Macklin, P.\nScore: 25.3, Published: 2023-09-24 DOI: 10.1101/2023.09.17.557982\nCells are fundamental units of life. Recent technical advances have revolutionized our ability to quantify the state and identity of individual cells, and intercellular regulatory programs. However, these static measurements alone are limited in their ability to predict the complex collective behaviors that emerge from populations of many interacting cells over time. Mathematical models have a proven record of successfully predicting the behaviors of dynamic biological systems, e.g., therapeutic responses in cancer. Simulations from these models enable in silico visualization, examination, and refinement of biological models and can be used to generate new hypotheses about cells and their collective behavior. Agent-based modeling is particularly well-suited to studying communities of interacting cells, as it is intuitive to map a single cell to a single agent. Thus, we have developed a conceptual framing (with a reference implementation in the widely-used PhysiCell agent-based modeling framework) that can be initialized directly from single cell and spatial transcriptomic data, and that can be easily populated with interactive rules. Because the expert mathematical and computational knowledge required to build agent-based models has limited their widespread adoption in the biomedical research community, we engineered this framework to specify complex cellular responses to signals (or stimuli) using a single line of human-readable text. This plain language text encodes cellular phenotypes and regulatory mechanisms from high throughput data and published literature, using a novel concept of hypothesis grammar. We motivate and fully describe this grammar and its philosophy, and then present a series of five example reference models of tumor growth and response to immunotherapy. Biologically, these examples demonstrate how mathematical models can predict from single cell and spatial transcriptomic data the cellular phenotypes responsible for tumor cell invasion and the simulation of immunotherapy treatment to overcome tumor cell growth. Computationally, these examples are designed to demonstrate how this conceptual framing and software implementation empower interdisciplinary teams to build an agent-based model of their experimental system, leveraging prior biological knowledge alone or in combination with information from spatial multiomics technologies. Altogether, this approach provides an interface to bridge biological, clinical, and systems biology researchers for mathematical modeling of biological systems at scale, allowing the community to extrapolate from single-cell characterization to emergent multicellular behavior.\nSystematic Profiling of Ale Yeast Protein Dynamics across Fermentation and Repitching\nAuthors: Garge, R. K.; Geck, R. C.; Armstrong, J. O.; Dunn, B.; Boutz, D. R.; Battenhouse, A.; Leutert, M.; Dang, V.; Jiang, P.; Kwiatkowski, D.; Peiser, T.; McElroy, H.; Marcotte, E. M.; Dunham, M. J.\nScore: 17.9, Published: 2023-09-22 DOI: 10.1101/2023.09.21.558736\nStudying the genetic and molecular characteristics of brewing yeast strains is crucial for understanding their domestication history and adaptations accumulated over time in fermentation environments, and for guiding optimizations to the brewing process itself. Saccharomyces cerevisiae (brewing yeast) is amongst the most profiled organisms on the planet, yet the temporal molecular changes that underlie industrial fermentation and beer brewing remain understudied. Here, we characterized the genomic makeup of a Saccharomyces cerevisiae ale yeast widely used in the production of Hefeweizen beers, and applied shotgun mass spectrometry to systematically measure the proteomic changes throughout two fermentation cycles which were separated by 14 rounds of serial repitching. The resulting brewing yeast proteomics resource includes 64,740 protein abundance measurements. We found that this strain possesses typical genetic characteristics of Saccharomyces cerevisiae ale strains and displayed progressive shifts in molecular processes during fermentation based on protein abundance changes. We observed protein abundance differences between early fermentation batches compared to those separated by 14 rounds of serial repitching. The observed abundance differences occurred mainly in proteins involved in the metabolism of ergosterol and isobutyraldehyde. Our systematic profiling serves as a starting point for deeper characterization of how the yeast proteome changes during commercial fermentations and additionally serves as a resource to guide fermentation protocols, strain handling, and engineering practices in commercial brewing and fermentation environments. Finally, we created a web interface (https://brewing-yeast-proteomics.ccbb.utexas.edu/) to serve as a valuable resource for yeast geneticists, brewers, and biochemists to provide insights into the global trends underlying commercial beer production.\nWidespread transcriptional regulation from within transcribed regions in plants\nAuthors: Voichek, Y.; Hristova, G.; Molla-Morales, A.; Weigel, D.; Nordborg, M.\nScore: 109.0, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557872\nAnimals and plants have evolved separately for over 1.5 billion years and independently invented multicellularity, suggesting a possible divergence in their mode of transcription regulation. Here, we set out to elucidate fundamental features of transcription regulatory sequences in plants. Using a massively parallel reporter assay in four species, we show that sequences downstream of the transcription start site (TSS) play a major role in controlling transcription. Swapping regulatory sequences from one side of the TSS to the other yields different outcomes, unlike animal enhancers which act independently of their position. A GATC-containing DNA motif, positioned downstream of the TSS, was sufficient to enhance gene expression in a dose-dependent manner. Its effect on gene expression was tissue-dependent and conserved across vascular plants. These results identify a unique characteristic of transcriptional regulation in plants, and suggest fundamental differences in gene regulation might exist between plants and animals.\nMulti-SNP combination genome-scale metabolic models identify molecular mechanisms determining genetic interactions in a quantitative trait\nAuthors: Sasikumar, S.; S, P. K.; Bhatt, N.; Sinha, H.\nScore: 11.7, Published: 2023-09-20 DOI: 10.1101/2023.09.13.557398\nElucidating genotype-phenotype or variant-to-function relationships remains a pivotal challenge in genetics. Specifically, understanding how genetic interactions among causal quantitative trait nucleotides (QTNs) drive phenotypic variation remains to be determined. Employing genome-scale metabolic modelling, we analysed the metabolic consequences of four causal QTNs, individually and in all combinations, responsible for modulating sporulation efficiency in yeast. Differential gene expression and network analysis revealed QTN interactions dependent changes in the connectivity of crucial metabolic regulators. Furthermore, QTN-specific metabolic models and genome-scale differential flux analysis pinpointed variations in flux across multiple metabolic pathways that could explain the observed sporulation efficiency differences. Finally, the differential regulation of the pentose phosphate pathway in specific QTN combinations suggested autophagy as a pentose phosphate pathway-dependent compensatory mechanism for yeast sporulation. Our study contributes to the evolving functional understanding of variants, providing insightful information into the mechanisms governing phenotypic variation.\nReconstruction of single cell lineage trajectories and identification of diversity in fates during the epithelial-to-mesenchymal transition\nAuthors: Cheng, Y.-C.; Zhang, Y.; Tripathi, S.; BV, H.; Jolly, M. K.; Schiebinger, G.; Levine, H.; McDonald, T. O.; Michor, F.\nScore: 5.1, Published: 2023-09-22 DOI: 10.1101/2023.09.19.558325\nThe epithelial-to-mesenchymal transition (EMT) gives rise to a variety of different cell states, yet it remains unclear whether these distinct states represent varying degrees of progression along a single trajectory or serve as endpoints of multiple potential trajectories that individual cells might pursue. Understanding the intricacies of these diverse EMT trajectories and cell fates, especially using lineage tracing at the single cell level using single cell RNA sequencing (scRNAseq) data, remains challenging. In this study, we employed optimal-transport analysis to reconstruct the most probable past trajectories of different cell fates during TGF-beta-induced EMT in the MCF10A cell line. Our analysis revealed three distinct temporal processes: those resulting in a low EMT, partial EMT, and high EMT state. The partial EMT trajectory exhibited substantial variations in the EMT signature and displayed unique cellular characteristics, including high stemness, low proliferation scores, and a potent cellular response to hypoxia. Along the partial EMT trajectory, we observed downregulation of the EED and EZH2 genes together with early high expression of TGFBI, KRT8, and CDH1--a finding that was validated by recent CRISPR-associated knock-out screening studies. Through our early differential gene expression analysis, we comprehensively explored genes linked to proliferation and stemness, identifying ITGB4, LAMA3, and LAMB3 as differentially expressed in the early stages of the partial versus high EMT trajectory. Our analysis thus elucidates novel potential targets for modulating EMT trajectories.\nDesmosome mutations impact the tumor microenvironment to promote melanoma proliferation\nAuthors: Baron, M.; Tagore, M.; Wall, P.; Zheng, F.; Barkley, D.; Yanai, I.; Yang, J.; Kiuru, M.; White, R.; Ideker, T.\nScore: 4.6, Published: 2023-09-22 DOI: 10.1101/2023.09.19.558457\nDesmosomes are transmembrane protein complexes that contribute to cell-cell adhesion in epithelia and other tissues. Here, we report the discovery of frequent genetic alterations in the desmosome in human cancers, with the strongest signal seen in cutaneous melanoma where desmosomes are mutated in over 70% of cases. In primary but not metastatic melanoma biopsies, the burden of coding mutations on desmosome genes associates with a strong reduction in desmosome gene expression. Analysis by spatial transcriptomics suggests that these expression decreases occur in keratinocytes in the microenvironment rather than in primary melanoma tumor cells. In further support of a microenvironmental origin, we find that loss-of-function knockdowns of the desmosome in keratinocytes yield markedly increased proliferation of adjacent melanocytes in keratinocyte/melanocyte co-cultures. Thus, gradual accumulation of desmosome mutations in neighboring cells may prime melanocytes for neoplastic transformation.\nA new universal system of tree shape indices\nAuthors: Noble, R. J.; Verity, K.\nScore: 2.4, Published: 2023-09-19 DOI: 10.1101/2023.07.17.549219\nThe comparison and categorization of tree diagrams is fundamental to large parts of biology, linguistics, computer science, and other fields, yet the indices currently applied to describing tree shape have important flaws that complicate their interpretation and limit their scope. Here we introduce a new system of indices with no such shortcomings. Our indices account for node sizes and branch lengths and are robust to small changes in either attribute. Unlike currently popular phylogenetic diversity, phylogenetic entropy, and tree balance indices, our definitions assign interpretable values to all rooted trees and enable meaningful comparison of any pair of trees. Our self-consistent definitions further unite measures of diversity, richness, balance, symmetry, effective height, effective outdegree, and effective branch count in a coherent system, and we derive numerous simple relationships between these indices. The main practical advantages of our indices are in 1) quantifying diversity in non-ultrametric trees; 2) assessing the balance of trees that have non-uniform branch lengths or node sizes; 3) comparing the balance of trees with different leaf counts or outdegrees; 4) obtaining a coherent, generic, multidimensional quantification of tree shape that is robust to sampling error and inferential error. We illustrate these features by comparing the shapes of trees representing the evolution of HIV and of Uralic languages. Given the ubiquity of tree structures, we identify a wide range of applications across diverse domains.\nModeling Cardiomyocyte Signaling and Metabolism Predicts Genotype to Phenotype Mechanisms in Hypertrophic Cardiomyopathy\nAuthors: Khalilimeybodi, A.; Rangamani, P.\nScore: 1.8, Published: 2023-09-26 DOI: 10.1101/2023.09.25.559356\nFamilial hypertrophic cardiomyopathy (HCM) is a significant precursor of heart failure and sudden cardiac death, primarily caused by mutations in sarcomeric and structural proteins. Despite the extensive research on the HCM genotype, the complex, context-specific nature of many signaling and metabolic pathways linking the HCM genotype to phenotype has hindered therapeutic advancements for patients. To address these challenges, here, we have developed a computational systems biology model of HCM at the cardiomyocyte level. Utilizing a stochastic logic-based ODE method, we integrate subcellular systems in cardiomyocytes that jointly modulate HCM genotype to phenotype, including cardiac signaling, metabolic, and gene regulatory networks, as well as posttranslational modifications linking these networks. After validating against existing experimental data, the model predicts significant changes in cardiomyocyte metabolic functions such as ATP synthase deficiency and a transition from fatty acids to carbohydrate metabolism in HCM. The model indicated major shifts in glutamine-related metabolism and increased apoptosis after HCM-induced ATP synthase deficiency. Aligned with prior experimental studies, we predicted that the transcription factors STAT, SRF, GATA4, TP53, and FoxO are the key regulators of cardiomyocyte hypertrophy and apoptosis in HCM. Using the model, we identified shared (e.g., activation of PGC1 by AMPK, and FHL1 by titin) and context-specific mechanisms (e.g., regulation of calcium sensitivity by titin in HCM patients) that could control genotype to phenotype transition in HCM across different species or mutations. We also predicted potential combinatory drug targets for HCM (e.g., mavacamten paired with ROS inhibitors) preventing or reversing HCM phenotype (i.e., hypertrophic growth, apoptosis, and metabolic remodeling) in cardiomyocytes. This study provides new insights into mechanisms linking genotype to phenotype in familial hypertrophic cardiomyopathy and offers a framework for assessing new treatments and exploring variations in HCM experimental models.\nSystems genetics analysis of human body fat distribution genes identifies Wnt signaling and mitochondrial activity in adipocytes\nAuthors: Reed, J. N.; Huang, J.; Li, Y.; Ma, L.; Banka, D.; Wabitsch, M.; Wang, T.; Ding, W.; Bjorkegren, J. L. M.; Civelek, M.\nScore: 2.5, Published: 2023-09-06 DOI: 10.1101/2023.09.06.556534\nBACKGROUNDExcess fat in the abdomen is a sexually dimorphic risk factor for cardio-metabolic disease. The relative storage between abdominal and lower-body subcutaneous adipose tissue depots is approximated by the waist-to-hip ratio adjusted for body mass index (WHRadjBMI). Genome-wide association studies (GWAS) identified 346 loci near 495 genes associated with WHRadjBMI. Most of these genes have unknown roles in fat distribution, but many are expressed and putatively act in adipose tissue. We aimed to identify novel sex- and depot-specific drivers of WHRadjBMI using a systems genetics approach. METHODSWe used two independent cohorts of adipose tissue gene expression with 362 - 444 males and 147 - 219 females, primarily of European ancestry. We constructed sex- and depot- specific Bayesian networks to model the gene-gene interactions from 8,492 adipose tissue genes. Key driver analysis identified genes that, in silico and putatively in vitro, regulate many others, including the 495 WHRadjBMI GWAS genes. Key driver gene function was determined by perturbing their expression in human subcutaneous pre-adipocytes using lenti-virus or siRNA. RESULTS51 - 119 key drivers in each network were replicated in both cohorts. We used single-cell expression data to select replicated key drivers expressed in adipocyte precursors and mature adipocytes, prioritized genes which have not been previously studied in adipose tissue, and used public human and mouse data to nominate 53 novel key driver genes (10 - 21 from each network) that may regulate fat distribution by altering adipocyte function. In other cell types, 23 of these genes are found in crucial adipocyte pathways: Wnt signaling or mitochondrial function. We selected seven genes whose expression is highly correlated with WHRadjBMI to further study their effects on adipogenesis/Wnt signaling (ANAPC2, PSME3, RSPO1, TYRO3) or mitochondrial function (C1QTNF3, MIGA1, PSME3, UBR1). Adipogenesis was inhibited in cells overexpressing ANAPC2 and RSPO1 compared to controls. RSPO1 results are consistent with a positive correlation between gene expression in the subcutaneous depot and WHRadjBMI, therefore lower relative storage in the subcutaneous depot. RSPO1 inhibited adipogenesis by increasing {beta}-catenin activation and Wnt-related transcription, thus repressing PPARG and CEBPA. PSME3 overexpression led to more adipogenesis than controls. In differentiated adipocytes, MIGA1 and UBR1 downregulation led to mitochondrial dysfunction, with lower oxygen consumption than controls; MIGA1 knockdown also lowered UCP1 expression. SUMMARYANAPC2, MIGA1, PSME3, RSPO1, and UBR1 affect adipocyte function and may drive body fat distribution.\nEscherichia coli sister cells dynamically organize developmental rosettes\nAuthors: Puri, D.; Allison, K. R.\nScore: 2.0, Published: 2023-09-18 DOI: 10.1101/2023.09.18.557913\nRosettes are self-organizing, circular multicellular communities that initiate the development of organs and embryos in higher organisms. Their formation results from the active repositioning of adhered sister cells and is thought to distinguish multicellular organisms form unicellular ones (1). Though biofilms have been argued to be the result of microbial development (2), bacterial rosettes are uncommon and largely uncharacterized (3-7). Here we report that Escherichia coli rosettes form by a consistent and novel mechanism. After division, sister cells \"folded\" to reposition themselves in parallel. After the second division, cells folded again to produce clonal 4-cell, quatrefoil rosettes. Analysis of folding revealed that it followed an angular random walk, comprised of [~]1-m angular strokes and directional randomization. We further showed that this motion was produced by the flagellum, the extracellular tail whose rotation generates swimming motility. Rosette formation was found to require de novo flagella synthesis and the balancing of sister-cell adhesion and propulsion. Moreover, by interrupting sister-cell folding, we demonstrated that rosette formation was required for E. colis multicellular-chain life cycle and the resulting clonal-chain biofilms. This study establishes that a unicellular bacterium self-organizes developmental rosettes by active sister-cell repositioning, and has implications for the evolution of multicellularity as well as E. colis roles as a commensal and pathogenic organism.\n",
  "wordCount" : "2767",
  "inLanguage": "en",
  "datePublished": "2023-09-27T10:39:11Z",
  "dateModified": "2023-09-27T10:39:11Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta"><span>updated on September 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.557982">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.557982" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.557982">
        <p class="paperTitle">Digitize your Biology! Modeling multicellular systems through interpretable cell behavior</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.557982" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.557982" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, J. A. I.; Stein-O&#39;Brien, G. L.; Booth, M.; Heiland, R.; Kurtoglu, F.; Bergman, D.; Bucher, E.; Deshpande, A.; Forjaz, A.; Getz, M.; Godet, I.; Lyman, M.; Metzcar, J.; Mitchell, J.; Raddatz, A.; Rocha, H. L.; Solorzano, J.; Sundus, A.; Wang, Y.; Gilkes, D. M.; Kagohara, L. T.; Kiemen, A. L.; Thompson, E. D.; Wirtz, D.; Wu, P.-H.; Zaidi, N.; Zheng, L.; Zimmerman, J. W.; Jaffee, E. M.; Chang, Y. H.; Coussens, L.; Gray, J.; Heiser, L. M.; Fertig, E. J.; Macklin, P.</p>
        <p class="info">Score: 25.3, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.557982' target='https://doi.org/10.1101/2023.09.17.557982'> 10.1101/2023.09.17.557982</a></p>
        <p class="abstract">Cells are fundamental units of life. Recent technical advances have revolutionized our ability to quantify the state and identity of individual cells, and intercellular regulatory programs. However, these static measurements alone are limited in their ability to predict the complex collective behaviors that emerge from populations of many interacting cells over time. Mathematical models have a proven record of successfully predicting the behaviors of dynamic biological systems, e.g., therapeutic responses in cancer. Simulations from these models enable in silico visualization, examination, and refinement of biological models and can be used to generate new hypotheses about cells and their collective behavior. Agent-based modeling is particularly well-suited to studying communities of interacting cells, as it is intuitive to map a single cell to a single agent. Thus, we have developed a conceptual framing (with a reference implementation in the widely-used PhysiCell agent-based modeling framework) that can be initialized directly from single cell and spatial transcriptomic data, and that can be easily populated with interactive rules. Because the expert mathematical and computational knowledge required to build agent-based models has limited their widespread adoption in the biomedical research community, we engineered this framework to specify complex cellular responses to signals (or stimuli) using a single line of human-readable text. This plain language text encodes cellular phenotypes and regulatory mechanisms from high throughput data and published literature, using a novel concept of hypothesis grammar. We motivate and fully describe this grammar and its philosophy, and then present a series of five example reference models of tumor growth and response to immunotherapy. Biologically, these examples demonstrate how mathematical models can predict from single cell and spatial transcriptomic data the cellular phenotypes responsible for tumor cell invasion and the simulation of immunotherapy treatment to overcome tumor cell growth. Computationally, these examples are designed to demonstrate how this conceptual framing and software implementation empower interdisciplinary teams to build an agent-based model of their experimental system, leveraging prior biological knowledge alone or in combination with information from spatial multiomics technologies. Altogether, this approach provides an interface to bridge biological, clinical, and systems biology researchers for mathematical modeling of biological systems at scale, allowing the community to extrapolate from single-cell characterization to emergent multicellular behavior.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558736">
        <p class="paperTitle">Systematic Profiling of Ale Yeast Protein Dynamics across Fermentation and Repitching</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garge, R. K.; Geck, R. C.; Armstrong, J. O.; Dunn, B.; Boutz, D. R.; Battenhouse, A.; Leutert, M.; Dang, V.; Jiang, P.; Kwiatkowski, D.; Peiser, T.; McElroy, H.; Marcotte, E. M.; Dunham, M. J.</p>
        <p class="info">Score: 17.9, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558736' target='https://doi.org/10.1101/2023.09.21.558736'> 10.1101/2023.09.21.558736</a></p>
        <p class="abstract">Studying the genetic and molecular characteristics of brewing yeast strains is crucial for understanding their domestication history and adaptations accumulated over time in fermentation environments, and for guiding optimizations to the brewing process itself. Saccharomyces cerevisiae (brewing yeast) is amongst the most profiled organisms on the planet, yet the temporal molecular changes that underlie industrial fermentation and beer brewing remain understudied. Here, we characterized the genomic makeup of a Saccharomyces cerevisiae ale yeast widely used in the production of Hefeweizen beers, and applied shotgun mass spectrometry to systematically measure the proteomic changes throughout two fermentation cycles which were separated by 14 rounds of serial repitching. The resulting brewing yeast proteomics resource includes 64,740 protein abundance measurements. We found that this strain possesses typical genetic characteristics of Saccharomyces cerevisiae ale strains and displayed progressive shifts in molecular processes during fermentation based on protein abundance changes. We observed protein abundance differences between early fermentation batches compared to those separated by 14 rounds of serial repitching. The observed abundance differences occurred mainly in proteins involved in the metabolism of ergosterol and isobutyraldehyde. Our systematic profiling serves as a starting point for deeper characterization of how the yeast proteome changes during commercial fermentations and additionally serves as a resource to guide fermentation protocols, strain handling, and engineering practices in commercial brewing and fermentation environments. Finally, we created a web interface (https://brewing-yeast-proteomics.ccbb.utexas.edu/) to serve as a valuable resource for yeast geneticists, brewers, and biochemists to provide insights into the global trends underlying commercial beer production.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557872">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557872" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557872">
        <p class="paperTitle">Widespread transcriptional regulation from within transcribed regions in plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557872" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557872" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Voichek, Y.; Hristova, G.; Molla-Morales, A.; Weigel, D.; Nordborg, M.</p>
        <p class="info">Score: 109.0, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557872' target='https://doi.org/10.1101/2023.09.15.557872'> 10.1101/2023.09.15.557872</a></p>
        <p class="abstract">Animals and plants have evolved separately for over 1.5 billion years and independently invented multicellularity, suggesting a possible divergence in their mode of transcription regulation. Here, we set out to elucidate fundamental features of transcription regulatory sequences in plants. Using a massively parallel reporter assay in four species, we show that sequences downstream of the transcription start site (TSS) play a major role in controlling transcription. Swapping regulatory sequences from one side of the TSS to the other yields different outcomes, unlike animal enhancers which act independently of their position. A GATC-containing DNA motif, positioned downstream of the TSS, was sufficient to enhance gene expression in a dose-dependent manner. Its effect on gene expression was tissue-dependent and conserved across vascular plants. These results identify a unique characteristic of transcriptional regulation in plants, and suggest fundamental differences in gene regulation might exist between plants and animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557398">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557398" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557398">
        <p class="paperTitle">Multi-SNP combination genome-scale metabolic models identify molecular mechanisms determining genetic interactions in a quantitative trait</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557398" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557398" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sasikumar, S.; S, P. K.; Bhatt, N.; Sinha, H.</p>
        <p class="info">Score: 11.7, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557398' target='https://doi.org/10.1101/2023.09.13.557398'> 10.1101/2023.09.13.557398</a></p>
        <p class="abstract">Elucidating genotype-phenotype or variant-to-function relationships remains a pivotal challenge in genetics. Specifically, understanding how genetic interactions among causal quantitative trait nucleotides (QTNs) drive phenotypic variation remains to be determined. Employing genome-scale metabolic modelling, we analysed the metabolic consequences of four causal QTNs, individually and in all combinations, responsible for modulating sporulation efficiency in yeast. Differential gene expression and network analysis revealed QTN interactions dependent changes in the connectivity of crucial metabolic regulators. Furthermore, QTN-specific metabolic models and genome-scale differential flux analysis pinpointed variations in flux across multiple metabolic pathways that could explain the observed sporulation efficiency differences. Finally, the differential regulation of the pentose phosphate pathway in specific QTN combinations suggested autophagy as a pentose phosphate pathway-dependent compensatory mechanism for yeast sporulation. Our study contributes to the evolving functional understanding of variants, providing insightful information into the mechanisms governing phenotypic variation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558325">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558325" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558325">
        <p class="paperTitle">Reconstruction of single cell lineage trajectories and identification of diversity in fates during the epithelial-to-mesenchymal transition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558325" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558325" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cheng, Y.-C.; Zhang, Y.; Tripathi, S.; BV, H.; Jolly, M. K.; Schiebinger, G.; Levine, H.; McDonald, T. O.; Michor, F.</p>
        <p class="info">Score: 5.1, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558325' target='https://doi.org/10.1101/2023.09.19.558325'> 10.1101/2023.09.19.558325</a></p>
        <p class="abstract">The epithelial-to-mesenchymal transition (EMT) gives rise to a variety of different cell states, yet it remains unclear whether these distinct states represent varying degrees of progression along a single trajectory or serve as endpoints of multiple potential trajectories that individual cells might pursue. Understanding the intricacies of these diverse EMT trajectories and cell fates, especially using lineage tracing at the single cell level using single cell RNA sequencing (scRNAseq) data, remains challenging. In this study, we employed optimal-transport analysis to reconstruct the most probable past trajectories of different cell fates during TGF-beta-induced EMT in the MCF10A cell line. Our analysis revealed three distinct temporal processes: those resulting in a low EMT, partial EMT, and high EMT state. The partial EMT trajectory exhibited substantial variations in the EMT signature and displayed unique cellular characteristics, including high stemness, low proliferation scores, and a potent cellular response to hypoxia. Along the partial EMT trajectory, we observed downregulation of the EED and EZH2 genes together with early high expression of TGFBI, KRT8, and CDH1--a finding that was validated by recent CRISPR-associated knock-out screening studies. Through our early differential gene expression analysis, we comprehensively explored genes linked to proliferation and stemness, identifying ITGB4, LAMA3, and LAMB3 as differentially expressed in the early stages of the partial versus high EMT trajectory. Our analysis thus elucidates novel potential targets for modulating EMT trajectories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558457">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558457" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558457">
        <p class="paperTitle">Desmosome mutations impact the tumor microenvironment to promote melanoma proliferation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558457" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558457" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baron, M.; Tagore, M.; Wall, P.; Zheng, F.; Barkley, D.; Yanai, I.; Yang, J.; Kiuru, M.; White, R.; Ideker, T.</p>
        <p class="info">Score: 4.6, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558457' target='https://doi.org/10.1101/2023.09.19.558457'> 10.1101/2023.09.19.558457</a></p>
        <p class="abstract">Desmosomes are transmembrane protein complexes that contribute to cell-cell adhesion in epithelia and other tissues. Here, we report the discovery of frequent genetic alterations in the desmosome in human cancers, with the strongest signal seen in cutaneous melanoma where desmosomes are mutated in over 70% of cases. In primary but not metastatic melanoma biopsies, the burden of coding mutations on desmosome genes associates with a strong reduction in desmosome gene expression. Analysis by spatial transcriptomics suggests that these expression decreases occur in keratinocytes in the microenvironment rather than in primary melanoma tumor cells. In further support of a microenvironmental origin, we find that loss-of-function knockdowns of the desmosome in keratinocytes yield markedly increased proliferation of adjacent melanocytes in keratinocyte/melanocyte co-cultures. Thus, gradual accumulation of desmosome mutations in neighboring cells may prime melanocytes for neoplastic transformation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549219" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549219">
        <p class="paperTitle">A new universal system of tree shape indices</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Noble, R. J.; Verity, K.</p>
        <p class="info">Score: 2.4, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549219' target='https://doi.org/10.1101/2023.07.17.549219'> 10.1101/2023.07.17.549219</a></p>
        <p class="abstract">The comparison and categorization of tree diagrams is fundamental to large parts of biology, linguistics, computer science, and other fields, yet the indices currently applied to describing tree shape have important flaws that complicate their interpretation and limit their scope. Here we introduce a new system of indices with no such shortcomings. Our indices account for node sizes and branch lengths and are robust to small changes in either attribute. Unlike currently popular phylogenetic diversity, phylogenetic entropy, and tree balance indices, our definitions assign interpretable values to all rooted trees and enable meaningful comparison of any pair of trees. Our self-consistent definitions further unite measures of diversity, richness, balance, symmetry, effective height, effective outdegree, and effective branch count in a coherent system, and we derive numerous simple relationships between these indices. The main practical advantages of our indices are in 1) quantifying diversity in non-ultrametric trees; 2) assessing the balance of trees that have non-uniform branch lengths or node sizes; 3) comparing the balance of trees with different leaf counts or outdegrees; 4) obtaining a coherent, generic, multidimensional quantification of tree shape that is robust to sampling error and inferential error. We illustrate these features by comparing the shapes of trees representing the evolution of HIV and of Uralic languages. Given the ubiquity of tree structures, we identify a wide range of applications across diverse domains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559356">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559356" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559356">
        <p class="paperTitle">Modeling Cardiomyocyte Signaling and Metabolism Predicts Genotype to Phenotype Mechanisms in Hypertrophic Cardiomyopathy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559356" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559356" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khalilimeybodi, A.; Rangamani, P.</p>
        <p class="info">Score: 1.8, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559356' target='https://doi.org/10.1101/2023.09.25.559356'> 10.1101/2023.09.25.559356</a></p>
        <p class="abstract">Familial hypertrophic cardiomyopathy (HCM) is a significant precursor of heart failure and sudden cardiac death, primarily caused by mutations in sarcomeric and structural proteins. Despite the extensive research on the HCM genotype, the complex, context-specific nature of many signaling and metabolic pathways linking the HCM genotype to phenotype has hindered therapeutic advancements for patients. To address these challenges, here, we have developed a computational systems biology model of HCM at the cardiomyocyte level. Utilizing a stochastic logic-based ODE method, we integrate subcellular systems in cardiomyocytes that jointly modulate HCM genotype to phenotype, including cardiac signaling, metabolic, and gene regulatory networks, as well as posttranslational modifications linking these networks. After validating against existing experimental data, the model predicts significant changes in cardiomyocyte metabolic functions such as ATP synthase deficiency and a transition from fatty acids to carbohydrate metabolism in HCM. The model indicated major shifts in glutamine-related metabolism and increased apoptosis after HCM-induced ATP synthase deficiency. Aligned with prior experimental studies, we predicted that the transcription factors STAT, SRF, GATA4, TP53, and FoxO are the key regulators of cardiomyocyte hypertrophy and apoptosis in HCM. Using the model, we identified shared (e.g., activation of PGC1 by AMPK, and FHL1 by titin) and context-specific mechanisms (e.g., regulation of calcium sensitivity by titin in HCM patients) that could control genotype to phenotype transition in HCM across different species or mutations. We also predicted potential combinatory drug targets for HCM (e.g., mavacamten paired with ROS inhibitors) preventing or reversing HCM phenotype (i.e., hypertrophic growth, apoptosis, and metabolic remodeling) in cardiomyocytes. This study provides new insights into mechanisms linking genotype to phenotype in familial hypertrophic cardiomyopathy and offers a framework for assessing new treatments and exploring variations in HCM experimental models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556534">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556534" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556534">
        <p class="paperTitle">Systems genetics analysis of human body fat distribution genes identifies Wnt signaling and mitochondrial activity in adipocytes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556534" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556534" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reed, J. N.; Huang, J.; Li, Y.; Ma, L.; Banka, D.; Wabitsch, M.; Wang, T.; Ding, W.; Bjorkegren, J. L. M.; Civelek, M.</p>
        <p class="info">Score: 2.5, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556534' target='https://doi.org/10.1101/2023.09.06.556534'> 10.1101/2023.09.06.556534</a></p>
        <p class="abstract">BACKGROUNDExcess fat in the abdomen is a sexually dimorphic risk factor for cardio-metabolic disease. The relative storage between abdominal and lower-body subcutaneous adipose tissue depots is approximated by the waist-to-hip ratio adjusted for body mass index (WHRadjBMI). Genome-wide association studies (GWAS) identified 346 loci near 495 genes associated with WHRadjBMI. Most of these genes have unknown roles in fat distribution, but many are expressed and putatively act in adipose tissue. We aimed to identify novel sex- and depot-specific drivers of WHRadjBMI using a systems genetics approach.

METHODSWe used two independent cohorts of adipose tissue gene expression with 362 - 444 males and 147 - 219 females, primarily of European ancestry. We constructed sex- and depot- specific Bayesian networks to model the gene-gene interactions from 8,492 adipose tissue genes. Key driver analysis identified genes that, in silico and putatively in vitro, regulate many others, including the 495 WHRadjBMI GWAS genes. Key driver gene function was determined by perturbing their expression in human subcutaneous pre-adipocytes using lenti-virus or siRNA.

RESULTS51 - 119 key drivers in each network were replicated in both cohorts. We used single-cell expression data to select replicated key drivers expressed in adipocyte precursors and mature adipocytes, prioritized genes which have not been previously studied in adipose tissue, and used public human and mouse data to nominate 53 novel key driver genes (10 - 21 from each network) that may regulate fat distribution by altering adipocyte function. In other cell types, 23 of these genes are found in crucial adipocyte pathways: Wnt signaling or mitochondrial function. We selected seven genes whose expression is highly correlated with WHRadjBMI to further study their effects on adipogenesis/Wnt signaling (ANAPC2, PSME3, RSPO1, TYRO3) or mitochondrial function (C1QTNF3, MIGA1, PSME3, UBR1).

Adipogenesis was inhibited in cells overexpressing ANAPC2 and RSPO1 compared to controls. RSPO1 results are consistent with a positive correlation between gene expression in the subcutaneous depot and WHRadjBMI, therefore lower relative storage in the subcutaneous depot. RSPO1 inhibited adipogenesis by increasing {beta}-catenin activation and Wnt-related transcription, thus repressing PPARG and CEBPA. PSME3 overexpression led to more adipogenesis than controls. In differentiated adipocytes, MIGA1 and UBR1 downregulation led to mitochondrial dysfunction, with lower oxygen consumption than controls; MIGA1 knockdown also lowered UCP1 expression.

SUMMARYANAPC2, MIGA1, PSME3, RSPO1, and UBR1 affect adipocyte function and may drive body fat distribution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.557913">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.557913" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.557913">
        <p class="paperTitle">Escherichia coli sister cells dynamically organize developmental rosettes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.557913" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.557913" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Puri, D.; Allison, K. R.</p>
        <p class="info">Score: 2.0, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.557913' target='https://doi.org/10.1101/2023.09.18.557913'> 10.1101/2023.09.18.557913</a></p>
        <p class="abstract">Rosettes are self-organizing, circular multicellular communities that initiate the development of organs and embryos in higher organisms. Their formation results from the active repositioning of adhered sister cells and is thought to distinguish multicellular organisms form unicellular ones (1). Though biofilms have been argued to be the result of microbial development (2), bacterial rosettes are uncommon and largely uncharacterized (3-7). Here we report that Escherichia coli rosettes form by a consistent and novel mechanism. After division, sister cells &#34;folded&#34; to reposition themselves in parallel. After the second division, cells folded again to produce clonal 4-cell, quatrefoil rosettes. Analysis of folding revealed that it followed an angular random walk, comprised of [~]1-m angular strokes and directional randomization. We further showed that this motion was produced by the flagellum, the extracellular tail whose rotation generates swimming motility. Rosette formation was found to require de novo flagella synthesis and the balancing of sister-cell adhesion and propulsion. Moreover, by interrupting sister-cell folding, we demonstrated that rosette formation was required for E. colis multicellular-chain life cycle and the resulting clonal-chain biofilms. This study establishes that a unicellular bacterium self-organizes developmental rosettes by active sister-cell repositioning, and has implications for the evolution of multicellularity as well as E. colis roles as a commensal and pathogenic organism.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
