<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>systems biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>systems biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Allometry of cell types in planarians by single cell transcriptomics
Authors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.
Score: 56.8, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140
Allometry explores the relationship between an organism&#39;s body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/systems-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="systems biology" />
<meta property="og:description" content="Allometry of cell types in planarians by single cell transcriptomics
Authors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.
Score: 56.8, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140
Allometry explores the relationship between an organism&#39;s body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/systems-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="systems biology"/>
<meta name="twitter:description" content="Allometry of cell types in planarians by single cell transcriptomics
Authors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.
Score: 56.8, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140
Allometry explores the relationship between an organism&#39;s body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "systems biology",
      "item": "https://trxiv.yorks0n.com/posts/systems-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "systems biology",
  "name": "systems biology",
  "description": "Allometry of cell types in planarians by single cell transcriptomics\nAuthors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.\nScore: 56.8, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140\nAllometry explores the relationship between an organism\u0026#39;s body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry.",
  "keywords": [
    
  ],
  "articleBody": " Allometry of cell types in planarians by single cell transcriptomics\nAuthors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.\nScore: 56.8, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565140\nAllometry explores the relationship between an organism's body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry. Planarians, capable of growing and degrowing following allometric scaling rules, serve as an excellent model for such studies. We used scRNA-seq to examine cell type allometry in asexual planarians of different sizes, revealing that they consist of the same basic cell types but in varying proportions. Notably, the gut basal cells are the most responsive to changes in size, suggesting a role in energy storage. We capture the gene regulatory programs of distinct cell types in response to size. This research sheds light on the molecular and cellular aspects of cell type allometry in planarians and underscores the utility of scRNA-seq in such investigations.\nPrinciples of Computation by Competitive Protein Dimerization Networks\nAuthors: Parres-Gold, J.; Levine, M.; Emert, B.; Stuart, A.; Elowitz, M.\nScore: 49.2, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564854\nMany biological signaling pathways employ proteins that competitively dimerize in diverse combinations. These dimerization networks can perform biochemical computations, in which the concentrations of monomers (inputs) determine the concentrations of dimers (outputs). Despite their prevalence, little is known about the range of input-output computations that dimerization networks can perform (their \"expressivity\") and how it depends on network size and connectivity. Using a systematic computational approach, we demonstrate that even small dimerization networks (3-6 monomers) are expressive, performing diverse multi-input computations. Further, dimerization networks are versatile, performing different computations when their protein components are expressed at different levels, such as in different cell types. Remarkably, individual networks with random interaction affinities, when large enough ([\u0026ge;]8 proteins), can perform nearly all ([~]90%) potential one-input network computations merely by tuning their monomer expression levels. Thus, even the simple process of competitive dimerization provides a powerful architecture for multi-input, cell-type-specific signal processing. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=192 SRC=\"FIGDIR/small/564854v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (35K): org.highwire.dtl.DTLVardef@5e43forg.highwire.dtl.DTLVardef@cbfbb4org.highwire.dtl.DTLVardef@1f9e8e1org.highwire.dtl.DTLVardef@284328_HPS_FORMAT_FIGEXP M_FIG C_FIG\nSynthetic model ecosystem of 12 cryopreservable microbial species allowing for a noninvasive approach\nAuthors: Hosoda, K.; Seno, S.; Murakami, N.; Matsuda, H.; Osada, Y.; Kamiura, R.; Kondoh, M.\nScore: 29.2, Published: 2023-10-31 DOI: 10.1101/2020.10.23.351742\nSimultaneous understanding of both individual and ecosystem dynamics is crucial in an era marked by the degradation of ecosystem services. Herein, we present a high-throughput synthetic microcosm system comprising 12 functionally and phylogenetically diverse microbial species. These species are axenically culturable, cryopreservable, and can be measured noninvasively via microscopy, aided by machine learning. This system includes prokaryotic and eukaryotic producers and decomposers, and eukaryotic consumers to ensure functional redundancy. Our model system displayed both positive and negative interspecific interactions and higher-order interactions that surpassed the scope of any two-species interaction. Although complete species coexistence was not our primary objective, we identified several conditions under which at least one species from the producers, consumers, and decomposers groups, and one functionally redundant species, persisted for over six months. These conditions set the stage for detailed investigations in the future. Given its designability and experimental replicability, our model ecosystem offers a promising platform for deeper insights into both individual and ecosystem dynamics, including evolution and species interactions.\nStory of an infection: viral dynamics and host responses in the Caenorhabditis elegans-Orsay virus pathosystem\nAuthors: Castiglioni, V. G.; Olmo-Uceda, M. J.; Villena-Jimenez, A.; Munoz-Sanchez, J. C.; Elena, S. F.\nScore: 5.2, Published: 2023-11-03 DOI: 10.1101/2023.10.31.564947\nOrsay virus (OrV) is the only known natural virus affecting Caenorhabditis elegans, with minimal impact on the worms fitness due to its robust innate immune response. This study aimed to understand the interactions between C. elegans and OrV by tracking the infections progression during larval development. Four distinct stages of infection were identified based on viral load, with a peak in capsid- encoding RNA2 coinciding with the first signs of viral egression. Transcriptomic analysis revealed temporal changes in gene expression and functions induced by the infection. A specific set of up- regulated genes remained active throughout the infection, and genes correlated and anticorrelated with virus accumulation were identified. Responses to OrV mirrored reactions to other biotic stressors, distinguishing between virus-specific responses and broader immune responses. Additionally, mutants of early response genes and defense-related processes showed altered viral load progression, uncovering new players in the antiviral defense response.\nMinimally Sufficient Experimental Design using Identifiability Analysis\nAuthors: Kareva, I.; Gevertz, J. L.\nScore: 8.6, Published: 2023-10-18 DOI: 10.1101/2023.10.14.562348\nMathematical models are increasingly being developed and calibrated in tandem with data collection, empowering scientists to intervene in real time based on quantitative model predictions. Well-designed experiments can help augment the predictive power of a mathematical model but the question of when to collect data to maximize its utility for a model is non-trivial. Here we define data as model-informative if it results in a unique parametrization, assessed through the lens of practical identifiability. The framework we propose identifies an optimal experimental design (how much data to collect and when to collect it) that ensures parameter identifiability (permitting confidence in model predictions), while minimizing experimental time and costs. We demonstrate the power of the method by applying it to a modified version of a classic site-of-action pharmacokinetic/pharmacodynamic model that describes distribution of a drug into the tumor microenvironment (TME), where its efficacy is dependent on the level of target occupancy in the TME. In this context, we identify a minimal set of time points when data needs to be collected that robustly ensures practical identifiability of model parameters. The proposed methodology can be applied broadly to any mathematical model, allowing for the identification of a minimally sufficient experimental design that collects the most informative data.\nExamination of the quantitative relationship between proteome- and transcriptome-level responses in larval guts of Frankliniella occidentalis infected with an orthotospovirus.\nAuthors: Han, J.; Rotenberg, D.\nScore: 4.8, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564998\nThe western flower thrips, Frankliniella occidentalis, is the principal thrips vector of Orthotospovirus tomatomaculae (order Bunyavirales, family Tospoviridae), a devastating plant-pathogenic virus commonly referred to as tomato spotted wilt virus (TSWV). The larval gut is the gateway for virus transmission by F. occidentalis adults to plants. In a previous report, gut expression at the transcriptome-level was subtle but significant in response to TSWV in L1s. Since it has been well documented that the relationship between the expression of mRNA and associated protein products in eukaryotic cells is often discordant, we performed identical, replicated experiments to identify and quantify virus-responsive larval gut proteins (427) to expand our understanding of insect host response to TSWV. While we documented statistically-significant, positive correlations between mRNA and protein abundance for genes (4,192) expressed in first and second instar guts, there was virtually no alignment of individual genes identified to be differentially modulated by virus infection at the transcriptome and proteome levels. Predicted protein-protein interaction networks associated with clusters of co-expressed proteins revealed wide variation in correlation strength between protein and cognate transcript abundance, which appeared to be associated with the type of cellular processes, cellular compartments, and network connectivity represented by the proteins. In total, our findings indicate distinct and dynamic regulatory mechanisms of transcript and protein abundance (expression, modifications, and/or turnover) in gut tissues regardless of virus-infection. This study provides molecular candidates for future functional analysis of thrips vector competence, and underscores the necessity of examining complex virus-vector interactions at a systems level.\nGlobal transcription regulation revealed from dynamical correlations in time-resolved single-cell RNA-sequencing\nAuthors: Volteras, D.; Shahrezaei, V.; Thomas, P.\nScore: 7.0, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563709\nSingle-cell transcriptomics reveals significant variations in the transcriptional activity across cells. Yet, it remains challenging to identify mechanisms of transcription dynamics from static snapshots. It is thus still unknown what drives global transcription dynamics in single cells. We present a stochastic model of gene expression in growing and dividing cells that harnesses temporal dimensions of single-cell RNA-sequencing through metabolic labelling protocols and cell cycle reporters. We develop a parallel and highly scalable Approximate Bayesian Computation method that corrects for technical variation and accurately quantifies absolute burst frequency, burst size and degradation rate at a transcriptome-wide scale. Using Bayesian model selection, we reveal scaling between transcription rates and cell size and unveil waves of gene regulation across the cell cycle-dependent transcriptome. Our study shows that stochastic modelling of dynamical correlations identifies global mechanisms of transcription regulation. We illustrate how mechanistic modelling and amortised inference can efficiently integrate distributions across time, modalities and conditions. Our framework thus provides a versatile tool to uncover how global transcriptional dynamics is orchestrated genome-wide in single cells.\nMulticellular growth as a dynamic network of cells\nAuthors: Nanda, P.; Barrere, J.; LaBar, T.; Murray, A. W.\nScore: 3.4, Published: 2023-11-03 DOI: 10.1101/2023.11.02.565242\nCell division without cell separation produces multicellular clusters in budding yeast. Two fundamental characteristics of these clusters are their size (the number of cells per cluster) and cellular composition: the fractions of cells with different phenotypes. However, we do not understand how different cellular features quantitatively influence these two phenotypes. Using cells as nodes and links between mother and daughter cells as edges, we model cluster growth and breakage by varying three parameters: the cell division rate, the rate at which intercellular connections break, and the kissing number (the maximum number of connections to one cell). We find that the kissing number sets the maximum possible cluster size. Below this limit, the ratio of the cell division rate to the connection breaking rate determines the cluster size. If links have a constant probability of breaking per unit time, the probability that a link survives decreases exponentially with its age. Modeling this behavior recapitulates experimental data. We then use this framework to examine synthetic, differentiating clusters with two cell types, faster-growing germ cells and their somatic derivatives. The fraction of clusters that contain both cell types increases as either of two parameters increase: the kissing number and difference between the growth rate of germ and somatic cells. In a population of clusters, the variation in cellular composition is inversely correlated (r2=0.87) with the average fraction of somatic cells in clusters. Our results show how a small number of cellular features can control the phenotypes of multicellular clusters that were potentially the ancestors of more complex forms of multicellular development, organization, and reproduction.\nMicrobial functional pathways based on metatranscriptomic profiling enable effective saliva-based health assessments for precision wellness\nAuthors: Patridge, E.; Gorakshakar, A.; Molusky, M. M.; Ogundijo, O.; Janevski, A.; Julian, C.; Hu, L.; Vuyisich, M.; Banavar, G.\nScore: 2.1, Published: 2023-11-07 DOI: 10.1101/2023.11.01.565122\nIt is increasingly recognized that an important step towards improving overall health is to accurately measure biomarkers of health from the molecular activities prevalent in the oral cavity. We present a general methodology for computationally quantifying the activity of microbial functional pathways using metatranscriptomic data. We describe their implementation as a collection of eight oral pathway scores using a large salivary sample dataset (n=9,350), and we evaluate score associations with oropharyngeal disease phenotypes within an unseen independent cohort (n=14,129). As clinical validation, we show that the relevant oral pathway scores are significantly worse in individuals with periodontal disease, acid reflux, and nicotine addiction, compared with controls. Given these associations, we make the case to use these oral pathway scores to provide molecular health insights from simple, non-invasive saliva samples, and as molecular endpoints for actionable interventions to address the associated conditions.\nMutational biases contribute to the neutral complexification of protein interaction networks following gene duplication\nAuthors: Cisneros, A. F.; Nielly-Thibault, L.; Mallik, S.; Levy, E. D.; Landry, C. R.\nScore: 2.0, Published: 2023-11-02 DOI: 10.1101/2023.10.31.565025\nBiological systems can gain complexity over time. While some of these transitions are likely driven by natural selection, the extent to which they occur without providing an adaptive benefit is unknown. At the molecular level, one example is heteromeric complexes replacing homomeric ones following gene duplication. Here, we build a biophysical model and simulate the evolution of homodimers and heterodimers following gene duplication using distributions of mutational effects inferred from available protein structures. We keep the specific activity of each dimer identical, so their concentrations drift neutrally in the absence of new functions. We show that for more than 60% of tested dimer structures, the relative concentration of the heteromer increases over time due to mutational biases that favor the heterodimer. However, allowing mutational effects on synthesis rates and differences in the specific activity of homodimers and heterodimers can limit or reverse the observed bias toward heterodimers. Our results show that the accumulation of more complex protein quaternary structures is likely under neutral evolution, and that natural selection would be needed to reverse this tendency.\n",
  "wordCount" : "2129",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/systems-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      systems biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565140">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565140" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565140">
        <p class="paperTitle">Allometry of cell types in planarians by single cell transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565140" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565140" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Emili, E.; Perez-Posada, A.; Christodoulou, M. D.; Solana, J.</p>
        <p class="info">Score: 56.8, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565140' target='https://doi.org/10.1101/2023.11.01.565140'> 10.1101/2023.11.01.565140</a></p>
        <p class="abstract">Allometry explores the relationship between an organism&#39;s body size and its various components, offering insights into ecology, physiology, metabolism, and disease. The cell is the basic unit of biological systems, and yet, the study of cell type allometry remains relatively unexplored. Single-cell RNA sequencing (scRNA-seq) provides a promising tool for investigating cell type allometry. Planarians, capable of growing and degrowing following allometric scaling rules, serve as an excellent model for such studies. We used scRNA-seq to examine cell type allometry in asexual planarians of different sizes, revealing that they consist of the same basic cell types but in varying proportions. Notably, the gut basal cells are the most responsive to changes in size, suggesting a role in energy storage. We capture the gene regulatory programs of distinct cell types in response to size. This research sheds light on the molecular and cellular aspects of cell type allometry in planarians and underscores the utility of scRNA-seq in such investigations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564854">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564854" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564854">
        <p class="paperTitle">Principles of Computation by Competitive Protein Dimerization Networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564854" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564854" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parres-Gold, J.; Levine, M.; Emert, B.; Stuart, A.; Elowitz, M.</p>
        <p class="info">Score: 49.2, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564854' target='https://doi.org/10.1101/2023.10.30.564854'> 10.1101/2023.10.30.564854</a></p>
        <p class="abstract">Many biological signaling pathways employ proteins that competitively dimerize in diverse combinations. These dimerization networks can perform biochemical computations, in which the concentrations of monomers (inputs) determine the concentrations of dimers (outputs). Despite their prevalence, little is known about the range of input-output computations that dimerization networks can perform (their &#34;expressivity&#34;) and how it depends on network size and connectivity. Using a systematic computational approach, we demonstrate that even small dimerization networks (3-6 monomers) are expressive, performing diverse multi-input computations. Further, dimerization networks are versatile, performing different computations when their protein components are expressed at different levels, such as in different cell types. Remarkably, individual networks with random interaction affinities, when large enough ([&amp;ge;]8 proteins), can perform nearly all ([~]90%) potential one-input network computations merely by tuning their monomer expression levels. Thus, even the simple process of competitive dimerization provides a powerful architecture for multi-input, cell-type-specific signal processing.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=192 SRC=&#34;FIGDIR/small/564854v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (35K):
org.highwire.dtl.DTLVardef@5e43forg.highwire.dtl.DTLVardef@cbfbb4org.highwire.dtl.DTLVardef@1f9e8e1org.highwire.dtl.DTLVardef@284328_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2020.10.23.351742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2020.10.23.351742" aria-expanded="false" aria-controls="flush-collapse10.1101/2020.10.23.351742">
        <p class="paperTitle">Synthetic model ecosystem of 12 cryopreservable microbial species allowing for a noninvasive approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2020.10.23.351742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2020.10.23.351742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hosoda, K.; Seno, S.; Murakami, N.; Matsuda, H.; Osada, Y.; Kamiura, R.; Kondoh, M.</p>
        <p class="info">Score: 29.2, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2020.10.23.351742' target='https://doi.org/10.1101/2020.10.23.351742'> 10.1101/2020.10.23.351742</a></p>
        <p class="abstract">Simultaneous understanding of both individual and ecosystem dynamics is crucial in an era marked by the degradation of ecosystem services. Herein, we present a high-throughput synthetic microcosm system comprising 12 functionally and phylogenetically diverse microbial species. These species are axenically culturable, cryopreservable, and can be measured noninvasively via microscopy, aided by machine learning. This system includes prokaryotic and eukaryotic producers and decomposers, and eukaryotic consumers to ensure functional redundancy. Our model system displayed both positive and negative interspecific interactions and higher-order interactions that surpassed the scope of any two-species interaction. Although complete species coexistence was not our primary objective, we identified several conditions under which at least one species from the producers, consumers, and decomposers groups, and one functionally redundant species, persisted for over six months. These conditions set the stage for detailed investigations in the future. Given its designability and experimental replicability, our model ecosystem offers a promising platform for deeper insights into both individual and ecosystem dynamics, including evolution and species interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564947">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564947" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564947">
        <p class="paperTitle">Story of an infection: viral dynamics and host responses in the Caenorhabditis elegans-Orsay virus pathosystem</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564947" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564947" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Castiglioni, V. G.; Olmo-Uceda, M. J.; Villena-Jimenez, A.; Munoz-Sanchez, J. C.; Elena, S. F.</p>
        <p class="info">Score: 5.2, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564947' target='https://doi.org/10.1101/2023.10.31.564947'> 10.1101/2023.10.31.564947</a></p>
        <p class="abstract">Orsay virus (OrV) is the only known natural virus affecting Caenorhabditis elegans, with minimal impact on the worms fitness due to its robust innate immune response. This study aimed to understand the interactions between C. elegans and OrV by tracking the infections progression during larval development. Four distinct stages of infection were identified based on viral load, with a peak in capsid- encoding RNA2 coinciding with the first signs of viral egression. Transcriptomic analysis revealed temporal changes in gene expression and functions induced by the infection. A specific set of up- regulated genes remained active throughout the infection, and genes correlated and anticorrelated with virus accumulation were identified. Responses to OrV mirrored reactions to other biotic stressors, distinguishing between virus-specific responses and broader immune responses. Additionally, mutants of early response genes and defense-related processes showed altered viral load progression, uncovering new players in the antiviral defense response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562348">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562348" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562348">
        <p class="paperTitle">Minimally Sufficient Experimental Design using Identifiability Analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562348" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562348" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kareva, I.; Gevertz, J. L.</p>
        <p class="info">Score: 8.6, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562348' target='https://doi.org/10.1101/2023.10.14.562348'> 10.1101/2023.10.14.562348</a></p>
        <p class="abstract">Mathematical models are increasingly being developed and calibrated in tandem with data collection, empowering scientists to intervene in real time based on quantitative model predictions. Well-designed experiments can help augment the predictive power of a mathematical model but the question of when to collect data to maximize its utility for a model is non-trivial. Here we define data as model-informative if it results in a unique parametrization, assessed through the lens of practical identifiability. The framework we propose identifies an optimal experimental design (how much data to collect and when to collect it) that ensures parameter identifiability (permitting confidence in model predictions), while minimizing experimental time and costs. We demonstrate the power of the method by applying it to a modified version of a classic site-of-action pharmacokinetic/pharmacodynamic model that describes distribution of a drug into the tumor microenvironment (TME), where its efficacy is dependent on the level of target occupancy in the TME. In this context, we identify a minimal set of time points when data needs to be collected that robustly ensures practical identifiability of model parameters. The proposed methodology can be applied broadly to any mathematical model, allowing for the identification of a minimally sufficient experimental design that collects the most informative data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564998">
        <p class="paperTitle">Examination of the quantitative relationship between proteome- and transcriptome-level responses in larval guts of Frankliniella occidentalis infected with an orthotospovirus.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Han, J.; Rotenberg, D.</p>
        <p class="info">Score: 4.8, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564998' target='https://doi.org/10.1101/2023.10.31.564998'> 10.1101/2023.10.31.564998</a></p>
        <p class="abstract">The western flower thrips, Frankliniella occidentalis, is the principal thrips vector of Orthotospovirus tomatomaculae (order Bunyavirales, family Tospoviridae), a devastating plant-pathogenic virus commonly referred to as tomato spotted wilt virus (TSWV). The larval gut is the gateway for virus transmission by F. occidentalis adults to plants. In a previous report, gut expression at the transcriptome-level was subtle but significant in response to TSWV in L1s. Since it has been well documented that the relationship between the expression of mRNA and associated protein products in eukaryotic cells is often discordant, we performed identical, replicated experiments to identify and quantify virus-responsive larval gut proteins (427) to expand our understanding of insect host response to TSWV. While we documented statistically-significant, positive correlations between mRNA and protein abundance for genes (4,192) expressed in first and second instar guts, there was virtually no alignment of individual genes identified to be differentially modulated by virus infection at the transcriptome and proteome levels. Predicted protein-protein interaction networks associated with clusters of co-expressed proteins revealed wide variation in correlation strength between protein and cognate transcript abundance, which appeared to be associated with the type of cellular processes, cellular compartments, and network connectivity represented by the proteins. In total, our findings indicate distinct and dynamic regulatory mechanisms of transcript and protein abundance (expression, modifications, and/or turnover) in gut tissues regardless of virus-infection. This study provides molecular candidates for future functional analysis of thrips vector competence, and underscores the necessity of examining complex virus-vector interactions at a systems level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563709">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563709" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563709">
        <p class="paperTitle">Global transcription regulation revealed from dynamical correlations in time-resolved single-cell RNA-sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563709" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563709" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Volteras, D.; Shahrezaei, V.; Thomas, P.</p>
        <p class="info">Score: 7.0, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563709' target='https://doi.org/10.1101/2023.10.24.563709'> 10.1101/2023.10.24.563709</a></p>
        <p class="abstract">Single-cell transcriptomics reveals significant variations in the transcriptional activity across cells. Yet, it remains challenging to identify mechanisms of transcription dynamics from static snapshots. It is thus still unknown what drives global transcription dynamics in single cells. We present a stochastic model of gene expression in growing and dividing cells that harnesses temporal dimensions of single-cell RNA-sequencing through metabolic labelling protocols and cell cycle reporters. We develop a parallel and highly scalable Approximate Bayesian Computation method that corrects for technical variation and accurately quantifies absolute burst frequency, burst size and degradation rate at a transcriptome-wide scale. Using Bayesian model selection, we reveal scaling between transcription rates and cell size and unveil waves of gene regulation across the cell cycle-dependent transcriptome. Our study shows that stochastic modelling of dynamical correlations identifies global mechanisms of transcription regulation. We illustrate how mechanistic modelling and amortised inference can efficiently integrate distributions across time, modalities and conditions. Our framework thus provides a versatile tool to uncover how global transcriptional dynamics is orchestrated genome-wide in single cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565242">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565242" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565242">
        <p class="paperTitle">Multicellular growth as a dynamic network of cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565242" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565242" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nanda, P.; Barrere, J.; LaBar, T.; Murray, A. W.</p>
        <p class="info">Score: 3.4, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565242' target='https://doi.org/10.1101/2023.11.02.565242'> 10.1101/2023.11.02.565242</a></p>
        <p class="abstract">Cell division without cell separation produces multicellular clusters in budding yeast. Two fundamental characteristics of these clusters are their size (the number of cells per cluster) and cellular composition: the fractions of cells with different phenotypes. However, we do not understand how different cellular features quantitatively influence these two phenotypes. Using cells as nodes and links between mother and daughter cells as edges, we model cluster growth and breakage by varying three parameters: the cell division rate, the rate at which intercellular connections break, and the kissing number (the maximum number of connections to one cell). We find that the kissing number sets the maximum possible cluster size. Below this limit, the ratio of the cell division rate to the connection breaking rate determines the cluster size. If links have a constant probability of breaking per unit time, the probability that a link survives decreases exponentially with its age. Modeling this behavior recapitulates experimental data. We then use this framework to examine synthetic, differentiating clusters with two cell types, faster-growing germ cells and their somatic derivatives. The fraction of clusters that contain both cell types increases as either of two parameters increase: the kissing number and difference between the growth rate of germ and somatic cells. In a population of clusters, the variation in cellular composition is inversely correlated (r2=0.87) with the average fraction of somatic cells in clusters. Our results show how a small number of cellular features can control the phenotypes of multicellular clusters that were potentially the ancestors of more complex forms of multicellular development, organization, and reproduction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565122">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565122" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565122">
        <p class="paperTitle">Microbial functional pathways based on metatranscriptomic profiling enable effective saliva-based health assessments for precision wellness</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565122" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565122" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Patridge, E.; Gorakshakar, A.; Molusky, M. M.; Ogundijo, O.; Janevski, A.; Julian, C.; Hu, L.; Vuyisich, M.; Banavar, G.</p>
        <p class="info">Score: 2.1, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565122' target='https://doi.org/10.1101/2023.11.01.565122'> 10.1101/2023.11.01.565122</a></p>
        <p class="abstract">It is increasingly recognized that an important step towards improving overall health is to accurately measure biomarkers of health from the molecular activities prevalent in the oral cavity. We present a general methodology for computationally quantifying the activity of microbial functional pathways using metatranscriptomic data. We describe their implementation as a collection of eight oral pathway scores using a large salivary sample dataset (n=9,350), and we evaluate score associations with oropharyngeal disease phenotypes within an unseen independent cohort (n=14,129). As clinical validation, we show that the relevant oral pathway scores are significantly worse in individuals with periodontal disease, acid reflux, and nicotine addiction, compared with controls. Given these associations, we make the case to use these oral pathway scores to provide molecular health insights from simple, non-invasive saliva samples, and as molecular endpoints for actionable interventions to address the associated conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565025" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565025">
        <p class="paperTitle">Mutational biases contribute to the neutral complexification of protein interaction networks following gene duplication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cisneros, A. F.; Nielly-Thibault, L.; Mallik, S.; Levy, E. D.; Landry, C. R.</p>
        <p class="info">Score: 2.0, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565025' target='https://doi.org/10.1101/2023.10.31.565025'> 10.1101/2023.10.31.565025</a></p>
        <p class="abstract">Biological systems can gain complexity over time. While some of these transitions are likely driven by natural selection, the extent to which they occur without providing an adaptive benefit is unknown. At the molecular level, one example is heteromeric complexes replacing homomeric ones following gene duplication. Here, we build a biophysical model and simulate the evolution of homodimers and heterodimers following gene duplication using distributions of mutational effects inferred from available protein structures. We keep the specific activity of each dimer identical, so their concentrations drift neutrally in the absence of new functions. We show that for more than 60% of tested dimer structures, the relative concentration of the heteromer increases over time due to mutational biases that favor the heterodimer. However, allowing mutational effects on synthesis rates and differences in the specific activity of homodimers and heterodimers can limit or reverse the observed bias toward heterodimers. Our results show that the accumulation of more complex protein quaternary structures is likely under neutral evolution, and that natural selection would be needed to reverse this tendency.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
