<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Accurate inference of population history in the presence of background selection
Authors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.
Score: 34.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576291
1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Accurate inference of population history in the presence of background selection
Authors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.
Score: 34.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576291
1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Accurate inference of population history in the presence of background selection
Authors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.
Score: 34.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576291
1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Accurate inference of population history in the presence of background selection\nAuthors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.\nScore: 34.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576291\n1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species.",
  "keywords": [
    
  ],
  "articleBody": " Accurate inference of population history in the presence of background selection\nAuthors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.\nScore: 34.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576291\n1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species. Simulations and theoretical work have shown that methods to infer changes in effective population size over time (Ne(t)) become increasingly inaccurate as the strength of linked selection increases. Here, we introduce an extension to the Pairwise Sequentially Markovian Coalescent (PSMC) algorithm, PSMC+, which explicitly co-models demographic history and natural selection. We benchmark our method using forward-in-time simulations with BGS and find that our approach improves the accuracy of effective population size inference. Leveraging a high resolution map of BGS in humans, we infer considerable changes in the magnitude of inferred effective population size relative to previous reports. Finally, we separately infer Ne(t) on the X chromosome and on the autosomes in diverse great apes without making a correction for selection, and find that the inferred ratio fluctuates substantially through time in a way that differs across species, showing that uncorrected selection may be an important driver of signals of genetic difference on the X chromosome and autosomes.\nDiencephalic and Neuropeptidergic Dysfunction in Zebrafish with Autism Risk Mutations\nAuthors: Capps, M. E. S.; Moyer, A. J.; Conklin, C. L.; Martina, V.; Torija-Olson, E. G.; Klein, M. C.; Gannaway, W. C.; Calhoun, C. C. S.; Vivian, M. D.; Thyme, S. B.\nScore: 10.8, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576309\nHundreds of human mutations are linked to autism and related disorders, yet the functions of many of these mutated genes during vertebrate neurodevelopment are unclear. We generated 27 zebrafish mutants with presumptive protein-truncating mutations or specific missense variants corresponding to autism-risk alleles in 17 human genes. We observed baseline and stimulus-driven behavioral changes at larval stages, as well as social behavior differences in lines tested as juveniles. Imaging whole-brain activity revealed a near identical activity map for mutations in the unrelated genes kmt5b and hdlbpa, defined by increased activity mainly in the diencephalon. Mutating 7 of the 17 risk genes resulted in substantial brain size differences. Using RNA sequencing, we further defined molecular drivers of the observed phenotypes, identifying targetable disruptions in neuropeptide signaling, neuronal maturation, and cell proliferation. This multi-modal screen nominated brain regions, cell types, and molecular pathways that may contribute to autism susceptibility. TeaserZebrafish screen uncovers diencephalon, social interaction, and neuropeptidergic signaling phenotypes in ASD risk mutants.\nBridge RNAs direct modular and programmable recombination of target and donor DNA\nAuthors: Durrant, M. G.; Perry, N. T.; Pai, J. J.; Jangid, A. R.; Athukoralage, J. S.; Hiraizumi, M.; McSpedon, J. P.; Pawluk, A.; Nishimasu, H.; Konermann, S.; Hsu, P. D.\nScore: 8.5, Published: 2024-01-26 DOI: 10.1101/2024.01.24.577089\nGenomic rearrangements, encompassing mutational changes in the genome such as insertions, deletions, or inversions, are essential for genetic diversity. These rearrangements are typically orchestrated by enzymes involved in fundamental DNA repair processes such as homologous recombination or in the transposition of foreign genetic material by viruses and mobile genetic elements (MGEs). We report that IS110 insertion sequences, a family of minimal and autonomous MGEs, express a structured non-coding RNA that binds specifically to their encoded recombinase. This bridge RNA contains two internal loops encoding nucleotide stretches that base-pair with the target DNA and donor DNA, which is the IS110 element itself. We demonstrate that the target-binding and donor-binding loops can be independently reprogrammed to direct sequence-specific recombination between two DNA molecules. This modularity enables DNA insertion into genomic target sites as well as programmable DNA excision and inversion. The IS110 bridge system expands the diversity of nucleic acid-guided systems beyond CRISPR and RNA interference, offering a unified mechanism for the three fundamental DNA rearrangements required for genome design.\nThe neocortical infrastructure for language involves region-specific patterns of laminar gene expression\nAuthors: Wong, M. M. K.; Sha, Z.; Lutje, L.; Kong, X.-Z.; Velthuijs, N.; van Heukelum, S.; van de Berg, W. D. J.; Jonkman, L. E.; Fisher, S. E.; Francks, C.\nScore: 11.4, Published: 2024-01-20 DOI: 10.1101/2024.01.17.575815\nThe language network of the human brain has core components in the inferior frontal cortex and superior/middle temporal cortex, with left-hemisphere dominance in most people. Functional specialization and interconnectivity of these neocortical regions is likely to be reflected in their molecular and cellular profiles. Excitatory connections between cortical regions arise and innervate according to layer-specific patterns. Here we generated a new gene expression dataset from human postmortem cortical tissue samples from core language network regions, using spatial transcriptomics to discriminate gene expression across cortical layers. Integration of these data with existing single-cell expression data identified 56 genes that showed differences in laminar expression profiles between frontal and temporal language cortex together with upregulation in layer II/III and/or layer V/VI excitatory neurons. Based on data from large-scale genome-wide screening in the population, DNA variants within these 56 genes showed set-level associations with inter-individual variation in structural connectivity between left-hemisphere frontal and temporal language cortex, and with predisposition to dyslexia. The axon guidance genes SLIT1 and SLIT2 were consistently implicated. These findings identify region-specific patterns of laminar gene expression as a feature of the brains language network.\nDeep learning predicts DNA methylation regulatory variants in specific brain cell types and enhances fine mapping for brain disorders\nAuthors: Zhou, J.; Weinberger, D.; Han, S.\nScore: 3.7, Published: 2024-01-21 DOI: 10.1101/2024.01.18.576319\nDNA methylation (DNAm) is essential for brain development and function and potentially mediates the effects of genetic risk variants underlying brain disorders. We present INTERACT, a transformer-based deep learning model to predict regulatory variants impacting DNAm levels in specific brain cell types, leveraging existing single-nucleus DNAm data from the human brain. We show that INTERACT accurately predicts cell type-specific DNAm profiles, achieving an average area under the Receiver Operating Characteristic curve of 0.98 across cell types. Furthermore, INTERACT predicts cell type-specific DNAm regulatory variants, which reflect cellular context and enrich the heritability of brain-related traits in relevant cell types. Importantly, we demonstrate that incorporating predicted variant effects and DNAm levels of CpG sites enhances the fine mapping for three brain disorders--schizophrenia, depression, and Alzheimers disease--and facilitates mapping causal genes to particular cell types. Our study highlights the power of deep learning in identifying cell type-specific regulatory variants, which will enhance our understanding of the genetics of complex traits. TeaserDeep learning reveals genetic variations impacting brain cell type-specific DNA methylation and illuminates genetic bases of brain disorders\nEvolution, diversity, function, and marker-based elimination of the disease susceptibility gene Snn1 in wheat\nAuthors: Seneviratne, S.; Shi, G.; Szabo-Hever, A.; Zhang, Z.; Peters Haugrud, A.; Running, K. L. D.; Singh, G.; Nandety, R. S.; Fiedler, J. D.; McClean, P. E.; Xu, S.; Friesen, T. L.; Faris, J. D.\nScore: 1.4, Published: 2024-01-26 DOI: 10.1101/2024.01.24.577084\nSeptoria nodorum blotch (SNB), caused by Parastagonospora nodorum, is a disease of durum and common wheat initiated by the recognition of pathogen-produced necrotrophic effectors (NEs) by specific wheat genes. The wheat gene Snn1 encodes a wall-associated kinase that directly interacts with the NE SnTox1 leading to the development of SNB. Here, sequence analysis of Snn1 from 114 accessions including diploid, tetraploid and hexaploid wheat species revealed that some wheat lines possess two copies of Snn1 (designated Snn1-B1 and Snn1-B2) approximately 120 kb apart. Snn1-B2 evolved relatively recently as a paralog of Snn1-B1, and both genes have undergone diversifying selection. Three point mutations associated with the formation of the first SnTox1-sensitive Snn1-B1 allele from a primitive wild wheat were identified. Four subsequent and independent SNPs, three in Snn1-B1 and one in Snn1-B2, converted the sensitive alleles to insensitive forms. Protein modeling indicated these four mutations could abolish Snn1-SnTox1 compatibility either through destabilization of the Snn1 protein or direct disruption of the protein-protein interaction. High-throughput markers were developed for the causal mutations and evaluated on panels of durum and common wheat. The markers were able to correctly identify 96.9 % of SnTox1-sensitive durum wheat accessions, and a marker for the null allele was 100% accurate at predicting SnTox1-insensitive lines in both durum and spring wheat. Results of this study increase our understanding of the evolution, diversity, and function of Snn1-B1 and Snn1-B2 genes and will be useful for marker-assisted elimination of these genes for better host resistance.\nLarge-Scale COI Gene Data Analysis for the Development of a Gel-Based Multiplex PCR Test Capable of Identifying Biting Midge Vector Species and Haplotypes (Diptera: Ceratopogonidae) of the Culicoides subgenus Avaritia Fox, 1955\nAuthors: Dähn, O.; Werner, D.; Mathieu, B.; Kampen, H.\nScore: 1.0, Published: 2024-01-26 DOI: 10.1101/2024.01.23.576915\nThe emergence of culicoid-transmitted bluetongue and Schmallenberg viruses in several European countries demonstrated the ability of indigenous biting midge species to transmit pathogens. Entomologic research programmes identified members of the Obsoletus Group (Culicoides subgenus Avaritia) as keyplayers in disease epidemiology in Europe. However, morphological identification of potential vectors to species level is challenging due to the existence of isomorphic species. PCR tests developed to facilitate genetic species determination have been dismantled by the discovery of new genetic variants (haplotypes) of C. obsoletus sensu stricto (s.s.), forming distinct clades. In this study, 4,422 GenBank entries of the mitochondrial cytochrome c oxidase subunit I (COI) gene of subgenus Avaritia members of the genus Culicoides were analyzed to develop a conventional multiplex PCR, capable of detecting all vector species and recently described clades of the western Palaearctic in this subgenus. Numerous GenBank entries incorrectly assigned to a species were identified, analyzed and reassigned. The results suggest that the three clades of C. obsoletus represent independent species, whereas C. montanus should rather be regarded as a genetic variant of C. obsoletus s.s.. Based on these findings, specific primers were designed and validated with DNA material from field-caught biting midges which achieved very high diagnostic sensitivity (100%) when compared to an established reference PCR (82.6%). Hence, the newly developed multiplex PCR represents the first molecular tool which enables both the identification of accepted species and of the three clades of C. obsoletus s.s. and could provide new insights into the ecology of the latter.\nHigher-order epistasis within Pol II trigger loop haplotypes\nAuthors: Duan, B.; Qiu, C.; Lockless, S. W.; Sze, S.-H.; Kaplan, C. D.\nScore: 1.0, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576280\nRNA polymerase II (Pol II) has a highly conserved domain, the trigger loop (TL), that controls transcription fidelity and speed. We previously probed pairwise genetic interactions between residues within and surrounding the TL and identified widespread incompatibility between TLs of different species when placed in the Saccharomyces cerevisiae Pol II context, indicating epistasis between the TL and its surrounding context. We sought to understand the nature of this incompatibility and probe higher order epistasis internal to the TL. We have employed deep mutational scanning with selected natural TL variants (\"haplotypes\"), and all possible intermediate substitution combinations between them and the yeast Pol II TL. We identified both positive and negative higher-order residue interactions within example TL haplotypes. Intricate higher-order epistasis formed by TL residues was sometimes only apparent from analysis of intermediate genotypes, emphasizing complexity of epistatic interactions. Furthermore, we distinguished TL substitutions with distinct classes of epistatic patterns, suggesting specific TL residues that potentially influence TL evolution. Our examples of complex residue interactions suggest possible pathways for epistasis to facilitate Pol II evolution.\nSingle-step generation of homozygous knock-out/knock-in individuals in an extremotolerant parthenogenetic tardigrade using DIPA-CRISPR\nAuthors: Kondo, K.; Tanaka, A.; Kunieda, T.\nScore: 16.6, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575120\nTardigrades are small aquatic invertebrates known for their remarkable tolerance to diverse extreme stresses. To elucidate the in vivo mechanisms underlying this extraordinary resilience, the genetic manipulation methods in tardigrades have long been desired. Despite our prior success in somatic cell gene-editing by microinjecting Cas9 ribonucleoproteins (RNPs) into the body cavity of tardigrades, the generation of gene-edited individuals remained elusive. In this study, employing an extremotolerant parthenogenetic tardigrade species, Ramazzottius varieornatus, we established conditions conductive to generating gene-edited tardigrade individuals. Drawing inspiration from the direct parental CRIPSR (DIPA-CRISPR) technique employed in several insects, we simply injected a concentrated Cas9 RNP solution into the body cavity of parental females shortly before their initial oviposition. This approach yielded gene-edited G0 progeny. Notably, only a single allele was predominantly detected at the target locus for each G0 individual, indicative of homozygous mutations. Through co-injecting single-stranded oligodeoxynucleotides (ssODNs) with Cas9 RNPs, we achieved the generation of homozygously knocked-in G0 progeny and these edited-alleles were inherited by G1/G2 progeny. This establishment of a simple method for generating homozygous knock-out/knock-in individuals not only facilitates in vivo analyses of molecular mechanisms underpinning extreme tolerance but also opens avenues for exploring various topics, including Evo-Devo, in tardigrades.\nA critical role for Pol II CTD phosphorylation in heterochromatic gene activation\nAuthors: Kainth, A. S.; Zhang, H.; Gross, D. S.\nScore: 0.8, Published: 2024-01-27 DOI: 10.1101/2024.01.26.577526\nHow gene activation works in heterochromatin, and how the mechanism might differ from the one used in euchromatin, has been largely unexplored. Previous work has shown that in SIR-regulated heterochromatin of Saccharomyces cerevisiae, gene activation occurs in the absence of covalent histone modifications and other alterations of chromatin commonly associated with transcription. Here we demonstrate that such activation occurs in a substantial fraction of cells, and this raises the possibility that an alternative activation pathway might be used. We address one such possibility, Pol II CTD phosphorylation, and explore this idea using a natural telomere-linked gene, YFR057w, as a model. Unlike covalent histone modifications, CTD phosphorylation at Ser2, Ser5 and Ser7 is abundant at the drug-induced heterochromatic gene. Particularly enriched relative to the euchromatic state is Ser2 phosphorylation. Consistent with a functional role for Ser2P, YFR057w is negligibly activated in cells deficient in the Ser2 CTD kinases Ctk1 and Bur1 even though the gene is strongly stimulated in ctk1{Delta} bur1 cells when it is placed in a euchromatic context. Collectively, our results are consistent with a critical role for Ser2 CTD phosphorylation in driving Pol II recruitment and transcription of a natural heterochromatic gene - an activity that may help supplant the need for histone epigenetic modifications.\n",
  "wordCount" : "2369",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576291">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576291" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576291">
        <p class="paperTitle">Accurate inference of population history in the presence of background selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576291" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576291" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.</p>
        <p class="info">Score: 34.3, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576291' target='https://doi.org/10.1101/2024.01.18.576291'> 10.1101/2024.01.18.576291</a></p>
        <p class="abstract">1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species. Simulations and theoretical work have shown that methods to infer changes in effective population size over time (Ne(t)) become increasingly inaccurate as the strength of linked selection increases. Here, we introduce an extension to the Pairwise Sequentially Markovian Coalescent (PSMC) algorithm, PSMC&#43;, which explicitly co-models demographic history and natural selection. We benchmark our method using forward-in-time simulations with BGS and find that our approach improves the accuracy of effective population size inference. Leveraging a high resolution map of BGS in humans, we infer considerable changes in the magnitude of inferred effective population size relative to previous reports. Finally, we separately infer Ne(t) on the X chromosome and on the autosomes in diverse great apes without making a correction for selection, and find that the inferred ratio fluctuates substantially through time in a way that differs across species, showing that uncorrected selection may be an important driver of signals of genetic difference on the X chromosome and autosomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576309">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576309" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576309">
        <p class="paperTitle">Diencephalic and Neuropeptidergic Dysfunction in Zebrafish with Autism Risk Mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576309" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576309" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Capps, M. E. S.; Moyer, A. J.; Conklin, C. L.; Martina, V.; Torija-Olson, E. G.; Klein, M. C.; Gannaway, W. C.; Calhoun, C. C. S.; Vivian, M. D.; Thyme, S. B.</p>
        <p class="info">Score: 10.8, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576309' target='https://doi.org/10.1101/2024.01.18.576309'> 10.1101/2024.01.18.576309</a></p>
        <p class="abstract">Hundreds of human mutations are linked to autism and related disorders, yet the functions of many of these mutated genes during vertebrate neurodevelopment are unclear. We generated 27 zebrafish mutants with presumptive protein-truncating mutations or specific missense variants corresponding to autism-risk alleles in 17 human genes. We observed baseline and stimulus-driven behavioral changes at larval stages, as well as social behavior differences in lines tested as juveniles. Imaging whole-brain activity revealed a near identical activity map for mutations in the unrelated genes kmt5b and hdlbpa, defined by increased activity mainly in the diencephalon. Mutating 7 of the 17 risk genes resulted in substantial brain size differences. Using RNA sequencing, we further defined molecular drivers of the observed phenotypes, identifying targetable disruptions in neuropeptide signaling, neuronal maturation, and cell proliferation. This multi-modal screen nominated brain regions, cell types, and molecular pathways that may contribute to autism susceptibility.

TeaserZebrafish screen uncovers diencephalon, social interaction, and neuropeptidergic signaling phenotypes in ASD risk mutants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577089">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577089" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577089">
        <p class="paperTitle">Bridge RNAs direct modular and programmable recombination of target and donor DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577089" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577089" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Durrant, M. G.; Perry, N. T.; Pai, J. J.; Jangid, A. R.; Athukoralage, J. S.; Hiraizumi, M.; McSpedon, J. P.; Pawluk, A.; Nishimasu, H.; Konermann, S.; Hsu, P. D.</p>
        <p class="info">Score: 8.5, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577089' target='https://doi.org/10.1101/2024.01.24.577089'> 10.1101/2024.01.24.577089</a></p>
        <p class="abstract">Genomic rearrangements, encompassing mutational changes in the genome such as insertions, deletions, or inversions, are essential for genetic diversity. These rearrangements are typically orchestrated by enzymes involved in fundamental DNA repair processes such as homologous recombination or in the transposition of foreign genetic material by viruses and mobile genetic elements (MGEs). We report that IS110 insertion sequences, a family of minimal and autonomous MGEs, express a structured non-coding RNA that binds specifically to their encoded recombinase. This bridge RNA contains two internal loops encoding nucleotide stretches that base-pair with the target DNA and donor DNA, which is the IS110 element itself. We demonstrate that the target-binding and donor-binding loops can be independently reprogrammed to direct sequence-specific recombination between two DNA molecules. This modularity enables DNA insertion into genomic target sites as well as programmable DNA excision and inversion. The IS110 bridge system expands the diversity of nucleic acid-guided systems beyond CRISPR and RNA interference, offering a unified mechanism for the three fundamental DNA rearrangements required for genome design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.575815">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.575815" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.575815">
        <p class="paperTitle">The neocortical infrastructure for language involves region-specific patterns of laminar gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.575815" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.575815" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wong, M. M. K.; Sha, Z.; Lutje, L.; Kong, X.-Z.; Velthuijs, N.; van Heukelum, S.; van de Berg, W. D. J.; Jonkman, L. E.; Fisher, S. E.; Francks, C.</p>
        <p class="info">Score: 11.4, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.575815' target='https://doi.org/10.1101/2024.01.17.575815'> 10.1101/2024.01.17.575815</a></p>
        <p class="abstract">The language network of the human brain has core components in the inferior frontal cortex and superior/middle temporal cortex, with left-hemisphere dominance in most people. Functional specialization and interconnectivity of these neocortical regions is likely to be reflected in their molecular and cellular profiles. Excitatory connections between cortical regions arise and innervate according to layer-specific patterns. Here we generated a new gene expression dataset from human postmortem cortical tissue samples from core language network regions, using spatial transcriptomics to discriminate gene expression across cortical layers. Integration of these data with existing single-cell expression data identified 56 genes that showed differences in laminar expression profiles between frontal and temporal language cortex together with upregulation in layer II/III and/or layer V/VI excitatory neurons. Based on data from large-scale genome-wide screening in the population, DNA variants within these 56 genes showed set-level associations with inter-individual variation in structural connectivity between left-hemisphere frontal and temporal language cortex, and with predisposition to dyslexia. The axon guidance genes SLIT1 and SLIT2 were consistently implicated. These findings identify region-specific patterns of laminar gene expression as a feature of the brains language network.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576319">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576319" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576319">
        <p class="paperTitle">Deep learning predicts DNA methylation regulatory variants in specific brain cell types and enhances fine mapping for brain disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576319" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576319" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, J.; Weinberger, D.; Han, S.</p>
        <p class="info">Score: 3.7, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576319' target='https://doi.org/10.1101/2024.01.18.576319'> 10.1101/2024.01.18.576319</a></p>
        <p class="abstract">DNA methylation (DNAm) is essential for brain development and function and potentially mediates the effects of genetic risk variants underlying brain disorders. We present INTERACT, a transformer-based deep learning model to predict regulatory variants impacting DNAm levels in specific brain cell types, leveraging existing single-nucleus DNAm data from the human brain. We show that INTERACT accurately predicts cell type-specific DNAm profiles, achieving an average area under the Receiver Operating Characteristic curve of 0.98 across cell types. Furthermore, INTERACT predicts cell type-specific DNAm regulatory variants, which reflect cellular context and enrich the heritability of brain-related traits in relevant cell types. Importantly, we demonstrate that incorporating predicted variant effects and DNAm levels of CpG sites enhances the fine mapping for three brain disorders--schizophrenia, depression, and Alzheimers disease--and facilitates mapping causal genes to particular cell types. Our study highlights the power of deep learning in identifying cell type-specific regulatory variants, which will enhance our understanding of the genetics of complex traits.

TeaserDeep learning reveals genetic variations impacting brain cell type-specific DNA methylation and illuminates genetic bases of brain disorders</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577084">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577084" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577084">
        <p class="paperTitle">Evolution, diversity, function, and marker-based elimination of the disease susceptibility gene Snn1 in wheat</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577084" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577084" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seneviratne, S.; Shi, G.; Szabo-Hever, A.; Zhang, Z.; Peters Haugrud, A.; Running, K. L. D.; Singh, G.; Nandety, R. S.; Fiedler, J. D.; McClean, P. E.; Xu, S.; Friesen, T. L.; Faris, J. D.</p>
        <p class="info">Score: 1.4, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577084' target='https://doi.org/10.1101/2024.01.24.577084'> 10.1101/2024.01.24.577084</a></p>
        <p class="abstract">Septoria nodorum blotch (SNB), caused by Parastagonospora nodorum, is a disease of durum and common wheat initiated by the recognition of pathogen-produced necrotrophic effectors (NEs) by specific wheat genes. The wheat gene Snn1 encodes a wall-associated kinase that directly interacts with the NE SnTox1 leading to the development of SNB. Here, sequence analysis of Snn1 from 114 accessions including diploid, tetraploid and hexaploid wheat species revealed that some wheat lines possess two copies of Snn1 (designated Snn1-B1 and Snn1-B2) approximately 120 kb apart. Snn1-B2 evolved relatively recently as a paralog of Snn1-B1, and both genes have undergone diversifying selection. Three point mutations associated with the formation of the first SnTox1-sensitive Snn1-B1 allele from a primitive wild wheat were identified. Four subsequent and independent SNPs, three in Snn1-B1 and one in Snn1-B2, converted the sensitive alleles to insensitive forms. Protein modeling indicated these four mutations could abolish Snn1-SnTox1 compatibility either through destabilization of the Snn1 protein or direct disruption of the protein-protein interaction. High-throughput markers were developed for the causal mutations and evaluated on panels of durum and common wheat. The markers were able to correctly identify 96.9 % of SnTox1-sensitive durum wheat accessions, and a marker for the null allele was 100% accurate at predicting SnTox1-insensitive lines in both durum and spring wheat. Results of this study increase our understanding of the evolution, diversity, and function of Snn1-B1 and Snn1-B2 genes and will be useful for marker-assisted elimination of these genes for better host resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576915">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576915" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576915">
        <p class="paperTitle">Large-Scale COI Gene Data Analysis for the Development of a Gel-Based Multiplex PCR Test Capable of Identifying Biting Midge Vector Species and Haplotypes (Diptera: Ceratopogonidae) of the Culicoides subgenus Avaritia Fox, 1955</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576915" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576915" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dähn, O.; Werner, D.; Mathieu, B.; Kampen, H.</p>
        <p class="info">Score: 1.0, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576915' target='https://doi.org/10.1101/2024.01.23.576915'> 10.1101/2024.01.23.576915</a></p>
        <p class="abstract">The emergence of culicoid-transmitted bluetongue and Schmallenberg viruses in several European countries demonstrated the ability of indigenous biting midge species to transmit pathogens. Entomologic research programmes identified members of the Obsoletus Group (Culicoides subgenus Avaritia) as keyplayers in disease epidemiology in Europe. However, morphological identification of potential vectors to species level is challenging due to the existence of isomorphic species. PCR tests developed to facilitate genetic species determination have been dismantled by the discovery of new genetic variants (haplotypes) of C. obsoletus sensu stricto (s.s.), forming distinct clades. In this study, 4,422 GenBank entries of the mitochondrial cytochrome c oxidase subunit I (COI) gene of subgenus Avaritia members of the genus Culicoides were analyzed to develop a conventional multiplex PCR, capable of detecting all vector species and recently described clades of the western Palaearctic in this subgenus. Numerous GenBank entries incorrectly assigned to a species were identified, analyzed and reassigned. The results suggest that the three clades of C. obsoletus represent independent species, whereas C. montanus should rather be regarded as a genetic variant of C. obsoletus s.s.. Based on these findings, specific primers were designed and validated with DNA material from field-caught biting midges which achieved very high diagnostic sensitivity (100%) when compared to an established reference PCR (82.6%). Hence, the newly developed multiplex PCR represents the first molecular tool which enables both the identification of accepted species and of the three clades of C. obsoletus s.s. and could provide new insights into the ecology of the latter.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576280">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576280" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576280">
        <p class="paperTitle">Higher-order epistasis within Pol II trigger loop haplotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576280" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576280" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duan, B.; Qiu, C.; Lockless, S. W.; Sze, S.-H.; Kaplan, C. D.</p>
        <p class="info">Score: 1.0, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576280' target='https://doi.org/10.1101/2024.01.20.576280'> 10.1101/2024.01.20.576280</a></p>
        <p class="abstract">RNA polymerase II (Pol II) has a highly conserved domain, the trigger loop (TL), that controls transcription fidelity and speed. We previously probed pairwise genetic interactions between residues within and surrounding the TL and identified widespread incompatibility between TLs of different species when placed in the Saccharomyces cerevisiae Pol II context, indicating epistasis between the TL and its surrounding context. We sought to understand the nature of this incompatibility and probe higher order epistasis internal to the TL. We have employed deep mutational scanning with selected natural TL variants (&#34;haplotypes&#34;), and all possible intermediate substitution combinations between them and the yeast Pol II TL. We identified both positive and negative higher-order residue interactions within example TL haplotypes. Intricate higher-order epistasis formed by TL residues was sometimes only apparent from analysis of intermediate genotypes, emphasizing complexity of epistatic interactions. Furthermore, we distinguished TL substitutions with distinct classes of epistatic patterns, suggesting specific TL residues that potentially influence TL evolution. Our examples of complex residue interactions suggest possible pathways for epistasis to facilitate Pol II evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575120" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575120">
        <p class="paperTitle">Single-step generation of homozygous knock-out/knock-in individuals in an extremotolerant parthenogenetic tardigrade using DIPA-CRISPR</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kondo, K.; Tanaka, A.; Kunieda, T.</p>
        <p class="info">Score: 16.6, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575120' target='https://doi.org/10.1101/2024.01.10.575120'> 10.1101/2024.01.10.575120</a></p>
        <p class="abstract">Tardigrades are small aquatic invertebrates known for their remarkable tolerance to diverse extreme stresses. To elucidate the in vivo mechanisms underlying this extraordinary resilience, the genetic manipulation methods in tardigrades have long been desired. Despite our prior success in somatic cell gene-editing by microinjecting Cas9 ribonucleoproteins (RNPs) into the body cavity of tardigrades, the generation of gene-edited individuals remained elusive.

In this study, employing an extremotolerant parthenogenetic tardigrade species, Ramazzottius varieornatus, we established conditions conductive to generating gene-edited tardigrade individuals. Drawing inspiration from the direct parental CRIPSR (DIPA-CRISPR) technique employed in several insects, we simply injected a concentrated Cas9 RNP solution into the body cavity of parental females shortly before their initial oviposition. This approach yielded gene-edited G0 progeny. Notably, only a single allele was predominantly detected at the target locus for each G0 individual, indicative of homozygous mutations. Through co-injecting single-stranded oligodeoxynucleotides (ssODNs) with Cas9 RNPs, we achieved the generation of homozygously knocked-in G0 progeny and these edited-alleles were inherited by G1/G2 progeny.

This establishment of a simple method for generating homozygous knock-out/knock-in individuals not only facilitates in vivo analyses of molecular mechanisms underpinning extreme tolerance but also opens avenues for exploring various topics, including Evo-Devo, in tardigrades.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577526">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577526" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577526">
        <p class="paperTitle">A critical role for Pol II CTD phosphorylation in heterochromatic gene activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577526" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577526" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kainth, A. S.; Zhang, H.; Gross, D. S.</p>
        <p class="info">Score: 0.8, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577526' target='https://doi.org/10.1101/2024.01.26.577526'> 10.1101/2024.01.26.577526</a></p>
        <p class="abstract">How gene activation works in heterochromatin, and how the mechanism might differ from the one used in euchromatin, has been largely unexplored. Previous work has shown that in SIR-regulated heterochromatin of Saccharomyces cerevisiae, gene activation occurs in the absence of covalent histone modifications and other alterations of chromatin commonly associated with transcription. Here we demonstrate that such activation occurs in a substantial fraction of cells, and this raises the possibility that an alternative activation pathway might be used. We address one such possibility, Pol II CTD phosphorylation, and explore this idea using a natural telomere-linked gene, YFR057w, as a model. Unlike covalent histone modifications, CTD phosphorylation at Ser2, Ser5 and Ser7 is abundant at the drug-induced heterochromatic gene. Particularly enriched relative to the euchromatic state is Ser2 phosphorylation. Consistent with a functional role for Ser2P, YFR057w is negligibly activated in cells deficient in the Ser2 CTD kinases Ctk1 and Bur1 even though the gene is strongly stimulated in ctk1{Delta} bur1 cells when it is placed in a euchromatic context. Collectively, our results are consistent with a critical role for Ser2 CTD phosphorylation in driving Pol II recruitment and transcription of a natural heterochromatic gene - an activity that may help supplant the need for histone epigenetic modifications.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
