<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency
Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.
Score: 35.2, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808
Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency
Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.
Score: 35.2, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808
Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-29T10:37:40+00:00" />
<meta property="article:modified_time" content="2023-10-29T10:37:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency
Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.
Score: 35.2, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808
Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency\nAuthors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.\nScore: 35.2, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808\nEctopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming.",
  "keywords": [
    
  ],
  "articleBody": " Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency\nAuthors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.\nScore: 35.2, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808\nEctopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming. Using neural networks that map DNA sequence to ATAC-seq profiles at base-resolution, we annotated cell-state-specific predictive transcription factor (TF) motif syntax in regulatory elements, inferred affinity- and concentration-dependent dynamics of Tn5-bias corrected TF footprints, linked peaks to putative target genes, and elucidated rewiring of TF- to-gene cis-regulatory networks. Our models reveal that early in reprogramming, OSK, at supraphysiological concentrations, rapidly open transient regulatory elements by occupying non-canonical low-affinity binding sites. As OSK concentration falls, the accessibility of these transient elements decays as a function of motif affinity. We find that these OSK-dependent transient elements sequester the somatic TF AP-1. This redistribution is strongly associated with the silencing of fibroblast-specific genes within individual nuclei. Together, our integrated single-cell resource and models reveal insights into the cis-regulatory code of reprogramming at unprecedented resolution, connect TF stoichiometry and motif syntax to diversification of cell fate trajectories, and provide new perspectives on the dynamics and role of transient regulatory elements in somatic silencing.\nNucleotide metabolism constrains prime editing in hematopoietic stem and progenitor cells\nAuthors: Levesque, S.; Verma, A.; Bauer, D. E.\nScore: 15.9, Published: 2023-10-22 DOI: 10.1101/2023.10.22.563434\nTherapeutic prime editing of hematopoietic stem and progenitor cells (HSPCs) holds great potential to remedy blood disorders. Since quiescent cells have low nucleotide levels and resist retroviral infection, we hypothesized that nucleotide metabolism could limit reverse transcription mediated prime editing in HSPCs. We demonstrate that deoxynucleoside supplementation and Vpx-mediated degradation of SAMHD1 improve prime editing efficiency in HSPCs, especially when coupled with editing approaches that evade mismatch repair.\nGene-specific reactivation of X-linked genes upon Xist loss is linked to the chromatin states in extraembryonic endoderm and epiblast stem cells\nAuthors: Arava, M.; Majumdar, S.; Bammidi, L. S.; Naik, H.; Baro, R.; GAYEN, S.\nScore: 14.9, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563299\nIn eutherian mammals, X-chromosome dosage between sexes is balanced through the inactivation of one of the two X-chromosomes in female cells. In mouse, X-inactivation initiates at [~]4-8 cell stages of embryogenesis, where paternal-X undergoes imprinted X-inactivation. Subsequently, it switches to random X-inactivation in post-iplantation epiblast. The initiation of XCI is orchestrated by Xist. However, the role of Xist in the maintenance of X-chromosome inactivation remains underexplored. Here, we have explored the role of Xist in the maintenance of X-inactivation in extraembryonic endoderm stem cells (XEN) and epiblast stem cells (EpiSC), which undergo imprinted and random form of X-inactivation respectively. We show that removal of Xist leads to the partial reactivation of inactive-X chromosome. Intriguingly, many reactivated genes were found to be common between XEN and EpiSC, indicating these genes require Xist to maintain their silent state irrespective of the lineages or forms of X-inactivation. Notably, despite Xist ablation and the subsequent removal of DNA methylation, several X-linked genes remained resistant to reactivation, indicating the involvement of other factors in maintaining the silencing of these genes. On the other hand, we show that genes on the inactive-X with low levels of H3K9me3 and high levels of H3K27me3 are more susceptible to reactivation upon the loss of Xist. Interestingly, active-X homolog of the reactivated genes was found to be enriched with H3K4me3 and H3K27ac. Taken together, our study sheds light on the role of chromatin states in the reactivation of X-linked genes following the loss of Xist in XEN and EpiSC.\nAlphaMissense is better correlated with functional assays of missense impact than earlier prediction algorithms\nAuthors: Ljungdahl, A.; Kohani, S.; Page, N. F.; Wells, E. S.; Wigdor, E. M.; Dong, S.; Sanders, S. J.\nScore: 11.8, Published: 2023-10-27 DOI: 10.1101/2023.10.24.562294\nMissense variants that alter a single amino acid in the encoded protein contribute to many human disorders but pose a substantial challenge in interpretation. Though these variants can be reliably identified through sequencing, distinguishing the clinically significant ones remains difficult, such that \"Variants of Unknown Significance\" outnumber those classified as \"Pathogenic\" or \"Likely Pathogenic\". Numerous in silico approaches have been developed to predict the functional impact of missense variants to inform clinical interpretation, the latest being AlphaMissense, which uses artificial intelligence methods trained on predicted protein structure. To independently assess the performance of AlphaMissense and 38 other predictors of missense severity, we compared predictions to data from multiplexed assays of variant effect (MAVE). MAVE experiments generate almost every possible individual amino acid change in a gene and measure their functional impact using a high-throughput assay. Assessing 17,696 variants across five genes (DDX3X, MSH2, PTEN, KCNQ4, and BRCA1), we find that AlphaMissense is consistently one of the top five algorithms based on correlation with functional impact and is the best-correlated algorithm for two genes. We conclude that AlphaMissense represents the current best-in-class predictor by this metric, however, the improvement over other algorithms is modest. We note that multiple missense predictors, including AlphaMissense, appear to overcall variants as pathogenic despite minimal functional impact and that substantially more high-quality training data, including consistently analyzed patient cohorts and MAVE analyses, are required to improve accuracy.\nGenotype prediction of 336,463 samples from public expression data\nAuthors: Razi, A.; Lo, C. C.; Wang, S.; Leek, J. T.; Hansen, K. D.\nScore: 10.7, Published: 2023-10-22 DOI: 10.1101/2023.10.21.562237\nTens of thousands of RNA-sequencing experiments comprising hundreds of thousands of individual samples have now been performed. These data represent a broad range of experimental conditions, sequencing technologies, and hypotheses under study. The Recount project has aggregated and uniformly processed hundreds of thousands of publicly available RNA-seq samples. Most of these samples only include RNA expression measurements; genotype data for these same samples would enable a wide range of analyses including variant prioritization, eQTL analysis, and studies of allele specific expression. Here, we developed a statistical model based on the existing reference and alternative read counts from the RNA-seq experiments available through Recount3 to predict genotypes at autosomal biallelic loci in coding regions. We demonstrate the accuracy of our model using large-scale studies that measured both gene expression and genotype genome-wide. We show that our predictive model is highly accurate with 99.5% overall accuracy, 99.6% major allele accuracy, and 90.4% minor allele accuracy. Our model is robust to tissue and study effects, provided the coverage is high enough. We applied this model to genotype all the samples in Recount3 and provide the largest ready-to-use expression repository containing genotype information. We illustrate that the predicted genotype from RNA-seq data is sufficient to unravel the underlying population structure of samples in Recount3 using Principal Component Analysis.\nA comparative approach for selecting orthologous candidate genes underlying signal in genome-wide association studies across multiple species\nAuthors: Whitt, L.; Mahood, E. H.; Ziegler, G.; Luebbert, C.; Gillman, J. D.; Norton, G. J.; Price, A. H.; Salt, D. E.; Dilkes, B.; Baxter, I.\nScore: 7.8, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561051\nAdvances in quantitative genetics have enabled researchers to identify genomic regions associated with changes in phenotype. However, genomic regions can contain hundreds to thousands of genes, and progressing from genomic regions to candidate genes is still challenging. In genome-wide association studies (GWAS) measuring elemental accumulation (ionomic) traits, a mere 5% of loci are associated with a known ionomic gene - indicating that many causal genes are still unknown. To select candidates for the remaining 95% of loci, we developed a method to identify conserved genes underlying GWAS loci in multiple species. For 19 ionomic traits, we identified 14,336 candidates across Arabidopsis, soybean, rice, maize, and sorghum. We calculated the likelihood of candidates with random permutations of the data and determined that most of the top 10% of candidates were orthologous genes linked to GWAS loci across all five species. The candidate list also includes orthologous genes with previously established ionomic functions in Arabidopsis and rice. Our methods highlight the conserved nature of ionomic genetic regulators and enable the identification of previously unknown ionomic genes. Author summaryIdentifying the genes contributing to changes in a given trait is challenging. Many genes can be near the region of interest, but proximity does not always translate to causality. We use other methods to narrow our focus to the genes most likely involved in our trait of interest before confirming their involvement through confirmation experiments. However, these other methods are often time, labor, and resource-consuming. We developed an approach to narrow these gene lists before these laborious methods are required. Through testing with genetic markers for elemental (i.e., calcium, iron, zinc) uptake, we found that comparing markers across multiple species for nearby evolutionarily conserved genes is a successful approach. Weve produced a list of candidate genes likely to be involved in elemental uptake traits, including previously known elemental uptake genes and genes whose potential elemental uptake function has yet to be observed. Some of these genes would not have been considered according to the significance threshold within a single GWAS, but combining comparable datasets across species has collectively boosted their signal. Methods like our approach are useful for reducing candidate lists to conserve resources spent in functional characterization experiments and encouraging the discovery of new functional roles of these genes.\nDiscovery of a locus associated with susceptibility to esca dieback in grapevine\nAuthors: Arnold, G.; Prado, E.; Dumas, V.; Butterlin, G.; Duchene, E.; Avia, K.; Merdinoglu, D.\nScore: 4.2, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563213\nEsca is the most destructive and predominant grapevine trunk diseases. The chronic infections and vine mortality caused by esca syndrome lead to huge economic losses and threatens the sustainability of vineyards worldwide. Although shown as associated with the presence of wood fungi, the etiology of esca remains still unclear and putatively involves multifactorial causes, which makes the development of effective control methods challenging. As differences in esca susceptibility had already been observed among grapevine varieties, we investigated in a biparental population the presence of genetic factors that can explain theses variations. Thanks to the destructive phenotyping of a 16-year-old vineyard plot, we discovered that the Gewurztraminer variety carries on chromosome 1 a locus linked to variations in trunk necrosis associated with esca, which we have named ENS1. Our study also suggests that there is a partial link between trunk vigor and necrosis due to esca. To our best knowledge, ENS1 is the first instance of genetic factor identified as involved in the limitation of necrosis associated to grapevine esca. While the identification of ENS1 alone may not provide a complete resolution of the esca issue, this discovery represents nonetheless a first step towards a genetic solution and paves the way for broader genetic investigations in the future.\nA natural bacterial pathogen of C. elegans uses a small RNA to induce transgenerational inheritance of learned avoidance\nAuthors: Sengupta, T.; Ange, J. S.; Moore, R. S.; Kaletsky, R. T.; Marogi, J. T.; Myhrvold, C.; Gitai, Z.; Murphy, C. T.\nScore: 18.3, Published: 2023-10-17 DOI: 10.1101/2023.07.20.549962\nPreviously, we discovered that a small RNA from a clinical isolate of Pseudomonas aeruginosa, PA14, induces learned avoidance and its transgenerational inheritance in C. elegans. Pseudomonas aeruginosa is an important human pathogen, and there are other Pseudomonads in C. elegans natural habitat, but it is unclear whether C. elegans ever encounters PA14-like bacteria in the wild. Thus, it is not known if small RNAs from bacteria found in C. elegans natural habitat can also regulate host behavior and produce heritable behavioral effects. Here we found that a pathogenic Pseudomonas vranovensis strain isolated from the C. elegans microbiota, GRb0427, like PA14, regulates worm behavior: worms learn to avoid this pathogenic bacterium following exposure to GRb0427, and this learned avoidance is inherited for four generations. The learned response is entirely mediated by bacterially-produced small RNAs, which induce avoidance and transgenerational inheritance, providing further support that such mechanisms of learning and inheritance exist in the wild. Using bacterial small RNA sequencing, we identified Pv1, a small RNA from GRb0427, that matches the sequence of C. elegans maco-1. We find that Pv1 is both necessary and sufficient to induce learned avoidance of Grb0427. However, Pv1 also results in avoidance of a beneficial microbiome strain, P. mendocina; this potentially maladaptive response may favor reversal of the transgenerational memory after a few generations. Our findings suggest that bacterial small RNA-mediated regulation of host behavior and its transgenerational inheritance are functional in C. elegans natural environment, and that different bacterial small RNA-mediated regulation systems evolved independently but define shared molecular features of bacterial small RNAs that produce transgenerationally-inherited effects.\nIntegrator-mediated clustering of poised RNA polymerase II synchronizes histone transcription\nAuthors: Lu, F.; Park, B. J.; Fujiwara, R.; Wilusz, J. E.; Gilmour, D. S.; Lehmann, R.; Lionnet, T.\nScore: 41.1, Published: 2023-10-08 DOI: 10.1101/2023.10.07.561364\nDynamic clustering of transcription players, including RNA polymerase II (Pol II), is thought to rely on multivalent interactions of their intrinsically disordered regions, thereby enhancing active transcription. Using the histone locus bodies (HLBs) of Drosophila nurse cells as a model, we find that Pol II forms long-lived, transcriptionally poised clusters distinct from liquid droplets, which contain unbound and paused Pol II. Depletion of the Integrator complex endonuclease module, but not its phosphatase module or pausing factors disperses these Pol II clusters, leaving HLBs intact. Consequently, histone transcription fails to reach peak levels during S-phase and aberrantly spills over throughout the cell cycle. We propose that clustering sequesters numerous poised Pol II molecules near gene promoters to ensure synchronous and efficient gene activation at desired times.\nA gene with a thousand alleles: the HYPer-variable effectors of plant-parasitic nematodes\nAuthors: Sonawala, U.; Beasley, H.; Thorpe, P. J.; Varypatakis, K.; Senatori, B.; Jones, J. T.; Derevnina, L.; Eves-van den Akker, S.\nScore: 38.2, Published: 2023-10-18 DOI: 10.1101/2023.10.16.561705\nPathogens are engaged in a fierce evolutionary arms race with their host. The genes at the forefront of the engagement between kingdoms, including effectors and immune receptors, are part of diverse and highly mutable gene families. Even in this context, we discovered unprecedented variation in the HYPer-variable (HYP) effectors of plant-parasitic nematodes. We discovered single effector gene loci that can harbour potentially thousands of allelic variants. These alleles vary in the number, but principally in the organisation, of motifs within a central Hyper Variable Domain (HVD). Using targeted long-read sequencing, we dramatically expand the HYP repertoire of two plant-parasitic nematodes, Globodera pallida and G. rostochiensis, such that we can define distinct sets of species-specific \"rules\" underlying the apparently flawless genetic rearrangements. Finally, by analysing the HYP complement of 68 individual nematodes we made the unexpected finding that despite the huge number of alleles, most individuals are homozygous. Taken together, these data point to a novel mechanism of programmed genetic variation, termed HVD-editing, where alterations are locus-specific, strictly governed by rules, and can theoretically produce thousands of variants without errors.\n",
  "wordCount" : "2503",
  "inLanguage": "en",
  "datePublished": "2023-10-29T10:37:40Z",
  "dateModified": "2023-10-29T10:37:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on October 29, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560808">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560808" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560808">
        <p class="paperTitle">Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560808" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560808" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.</p>
        <p class="info">Score: 35.2, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560808' target='https://doi.org/10.1101/2023.10.04.560808'> 10.1101/2023.10.04.560808</a></p>
        <p class="abstract">Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming. Using neural networks that map DNA sequence to ATAC-seq profiles at base-resolution, we annotated cell-state-specific predictive transcription factor (TF) motif syntax in regulatory elements, inferred affinity- and concentration-dependent dynamics of Tn5-bias corrected TF footprints, linked peaks to putative target genes, and elucidated rewiring of TF- to-gene cis-regulatory networks. Our models reveal that early in reprogramming, OSK, at supraphysiological concentrations, rapidly open transient regulatory elements by occupying non-canonical low-affinity binding sites. As OSK concentration falls, the accessibility of these transient elements decays as a function of motif affinity. We find that these OSK-dependent transient elements sequester the somatic TF AP-1. This redistribution is strongly associated with the silencing of fibroblast-specific genes within individual nuclei. Together, our integrated single-cell resource and models reveal insights into the cis-regulatory code of reprogramming at unprecedented resolution, connect TF stoichiometry and motif syntax to diversification of cell fate trajectories, and provide new perspectives on the dynamics and role of transient regulatory elements in somatic silencing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.22.563434">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.22.563434" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.22.563434">
        <p class="paperTitle">Nucleotide metabolism constrains prime editing in hematopoietic stem and progenitor cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.22.563434" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.22.563434" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levesque, S.; Verma, A.; Bauer, D. E.</p>
        <p class="info">Score: 15.9, Published: 2023-10-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.22.563434' target='https://doi.org/10.1101/2023.10.22.563434'> 10.1101/2023.10.22.563434</a></p>
        <p class="abstract">Therapeutic prime editing of hematopoietic stem and progenitor cells (HSPCs) holds great potential to remedy blood disorders. Since quiescent cells have low nucleotide levels and resist retroviral infection, we hypothesized that nucleotide metabolism could limit reverse transcription mediated prime editing in HSPCs. We demonstrate that deoxynucleoside supplementation and Vpx-mediated degradation of SAMHD1 improve prime editing efficiency in HSPCs, especially when coupled with editing approaches that evade mismatch repair.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563299">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563299" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563299">
        <p class="paperTitle">Gene-specific reactivation of X-linked genes upon Xist loss is linked to the chromatin states in extraembryonic endoderm and epiblast stem cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563299" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563299" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arava, M.; Majumdar, S.; Bammidi, L. S.; Naik, H.; Baro, R.; GAYEN, S.</p>
        <p class="info">Score: 14.9, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563299' target='https://doi.org/10.1101/2023.10.20.563299'> 10.1101/2023.10.20.563299</a></p>
        <p class="abstract">In eutherian mammals, X-chromosome dosage between sexes is balanced through the inactivation of one of the two X-chromosomes in female cells. In mouse, X-inactivation initiates at [~]4-8 cell stages of embryogenesis, where paternal-X undergoes imprinted X-inactivation. Subsequently, it switches to random X-inactivation in post-iplantation epiblast. The initiation of XCI is orchestrated by Xist. However, the role of Xist in the maintenance of X-chromosome inactivation remains underexplored. Here, we have explored the role of Xist in the maintenance of X-inactivation in extraembryonic endoderm stem cells (XEN) and epiblast stem cells (EpiSC), which undergo imprinted and random form of X-inactivation respectively. We show that removal of Xist leads to the partial reactivation of inactive-X chromosome. Intriguingly, many reactivated genes were found to be common between XEN and EpiSC, indicating these genes require Xist to maintain their silent state irrespective of the lineages or forms of X-inactivation. Notably, despite Xist ablation and the subsequent removal of DNA methylation, several X-linked genes remained resistant to reactivation, indicating the involvement of other factors in maintaining the silencing of these genes. On the other hand, we show that genes on the inactive-X with low levels of H3K9me3 and high levels of H3K27me3 are more susceptible to reactivation upon the loss of Xist. Interestingly, active-X homolog of the reactivated genes was found to be enriched with H3K4me3 and H3K27ac. Taken together, our study sheds light on the role of chromatin states in the reactivation of X-linked genes following the loss of Xist in XEN and EpiSC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.562294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.562294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.562294">
        <p class="paperTitle">AlphaMissense is better correlated with functional assays of missense impact than earlier prediction algorithms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.562294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.562294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ljungdahl, A.; Kohani, S.; Page, N. F.; Wells, E. S.; Wigdor, E. M.; Dong, S.; Sanders, S. J.</p>
        <p class="info">Score: 11.8, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.562294' target='https://doi.org/10.1101/2023.10.24.562294'> 10.1101/2023.10.24.562294</a></p>
        <p class="abstract">Missense variants that alter a single amino acid in the encoded protein contribute to many human disorders but pose a substantial challenge in interpretation. Though these variants can be reliably identified through sequencing, distinguishing the clinically significant ones remains difficult, such that &#34;Variants of Unknown Significance&#34; outnumber those classified as &#34;Pathogenic&#34; or &#34;Likely Pathogenic&#34;. Numerous in silico approaches have been developed to predict the functional impact of missense variants to inform clinical interpretation, the latest being AlphaMissense, which uses artificial intelligence methods trained on predicted protein structure. To independently assess the performance of AlphaMissense and 38 other predictors of missense severity, we compared predictions to data from multiplexed assays of variant effect (MAVE). MAVE experiments generate almost every possible individual amino acid change in a gene and measure their functional impact using a high-throughput assay. Assessing 17,696 variants across five genes (DDX3X, MSH2, PTEN, KCNQ4, and BRCA1), we find that AlphaMissense is consistently one of the top five algorithms based on correlation with functional impact and is the best-correlated algorithm for two genes. We conclude that AlphaMissense represents the current best-in-class predictor by this metric, however, the improvement over other algorithms is modest. We note that multiple missense predictors, including AlphaMissense, appear to overcall variants as pathogenic despite minimal functional impact and that substantially more high-quality training data, including consistently analyzed patient cohorts and MAVE analyses, are required to improve accuracy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.21.562237">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.21.562237" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.21.562237">
        <p class="paperTitle">Genotype prediction of 336,463 samples from public expression data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.21.562237" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.21.562237" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Razi, A.; Lo, C. C.; Wang, S.; Leek, J. T.; Hansen, K. D.</p>
        <p class="info">Score: 10.7, Published: 2023-10-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.21.562237' target='https://doi.org/10.1101/2023.10.21.562237'> 10.1101/2023.10.21.562237</a></p>
        <p class="abstract">Tens of thousands of RNA-sequencing experiments comprising hundreds of thousands of individual samples have now been performed. These data represent a broad range of experimental conditions, sequencing technologies, and hypotheses under study. The Recount project has aggregated and uniformly processed hundreds of thousands of publicly available RNA-seq samples. Most of these samples only include RNA expression measurements; genotype data for these same samples would enable a wide range of analyses including variant prioritization, eQTL analysis, and studies of allele specific expression. Here, we developed a statistical model based on the existing reference and alternative read counts from the RNA-seq experiments available through Recount3 to predict genotypes at autosomal biallelic loci in coding regions. We demonstrate the accuracy of our model using large-scale studies that measured both gene expression and genotype genome-wide. We show that our predictive model is highly accurate with 99.5% overall accuracy, 99.6% major allele accuracy, and 90.4% minor allele accuracy. Our model is robust to tissue and study effects, provided the coverage is high enough. We applied this model to genotype all the samples in Recount3 and provide the largest ready-to-use expression repository containing genotype information. We illustrate that the predicted genotype from RNA-seq data is sufficient to unravel the underlying population structure of samples in Recount3 using Principal Component Analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561051">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561051" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561051">
        <p class="paperTitle">A comparative approach for selecting orthologous candidate genes underlying signal in genome-wide association studies across multiple species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561051" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561051" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Whitt, L.; Mahood, E. H.; Ziegler, G.; Luebbert, C.; Gillman, J. D.; Norton, G. J.; Price, A. H.; Salt, D. E.; Dilkes, B.; Baxter, I.</p>
        <p class="info">Score: 7.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561051' target='https://doi.org/10.1101/2023.10.05.561051'> 10.1101/2023.10.05.561051</a></p>
        <p class="abstract">Advances in quantitative genetics have enabled researchers to identify genomic regions associated with changes in phenotype. However, genomic regions can contain hundreds to thousands of genes, and progressing from genomic regions to candidate genes is still challenging. In genome-wide association studies (GWAS) measuring elemental accumulation (ionomic) traits, a mere 5% of loci are associated with a known ionomic gene - indicating that many causal genes are still unknown. To select candidates for the remaining 95% of loci, we developed a method to identify conserved genes underlying GWAS loci in multiple species. For 19 ionomic traits, we identified 14,336 candidates across Arabidopsis, soybean, rice, maize, and sorghum. We calculated the likelihood of candidates with random permutations of the data and determined that most of the top 10% of candidates were orthologous genes linked to GWAS loci across all five species. The candidate list also includes orthologous genes with previously established ionomic functions in Arabidopsis and rice. Our methods highlight the conserved nature of ionomic genetic regulators and enable the identification of previously unknown ionomic genes.

Author summaryIdentifying the genes contributing to changes in a given trait is challenging. Many genes can be near the region of interest, but proximity does not always translate to causality. We use other methods to narrow our focus to the genes most likely involved in our trait of interest before confirming their involvement through confirmation experiments. However, these other methods are often time, labor, and resource-consuming. We developed an approach to narrow these gene lists before these laborious methods are required. Through testing with genetic markers for elemental (i.e., calcium, iron, zinc) uptake, we found that comparing markers across multiple species for nearby evolutionarily conserved genes is a successful approach. Weve produced a list of candidate genes likely to be involved in elemental uptake traits, including previously known elemental uptake genes and genes whose potential elemental uptake function has yet to be observed. Some of these genes would not have been considered according to the significance threshold within a single GWAS, but combining comparable datasets across species has collectively boosted their signal. Methods like our approach are useful for reducing candidate lists to conserve resources spent in functional characterization experiments and encouraging the discovery of new functional roles of these genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563213">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563213" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563213">
        <p class="paperTitle">Discovery of a locus associated with susceptibility to esca dieback in grapevine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563213" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563213" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arnold, G.; Prado, E.; Dumas, V.; Butterlin, G.; Duchene, E.; Avia, K.; Merdinoglu, D.</p>
        <p class="info">Score: 4.2, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563213' target='https://doi.org/10.1101/2023.10.20.563213'> 10.1101/2023.10.20.563213</a></p>
        <p class="abstract">Esca is the most destructive and predominant grapevine trunk diseases. The chronic infections and vine mortality caused by esca syndrome lead to huge economic losses and threatens the sustainability of vineyards worldwide. Although shown as associated with the presence of wood fungi, the etiology of esca remains still unclear and putatively involves multifactorial causes, which makes the development of effective control methods challenging. As differences in esca susceptibility had already been observed among grapevine varieties, we investigated in a biparental population the presence of genetic factors that can explain theses variations. Thanks to the destructive phenotyping of a 16-year-old vineyard plot, we discovered that the Gewurztraminer variety carries on chromosome 1 a locus linked to variations in trunk necrosis associated with esca, which we have named ENS1. Our study also suggests that there is a partial link between trunk vigor and necrosis due to esca. To our best knowledge, ENS1 is the first instance of genetic factor identified as involved in the limitation of necrosis associated to grapevine esca. While the identification of ENS1 alone may not provide a complete resolution of the esca issue, this discovery represents nonetheless a first step towards a genetic solution and paves the way for broader genetic investigations in the future.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549962" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549962">
        <p class="paperTitle">A natural bacterial pathogen of C. elegans uses a small RNA to induce transgenerational inheritance of learned avoidance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sengupta, T.; Ange, J. S.; Moore, R. S.; Kaletsky, R. T.; Marogi, J. T.; Myhrvold, C.; Gitai, Z.; Murphy, C. T.</p>
        <p class="info">Score: 18.3, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549962' target='https://doi.org/10.1101/2023.07.20.549962'> 10.1101/2023.07.20.549962</a></p>
        <p class="abstract">Previously, we discovered that a small RNA from a clinical isolate of Pseudomonas aeruginosa, PA14, induces learned avoidance and its transgenerational inheritance in C. elegans. Pseudomonas aeruginosa is an important human pathogen, and there are other Pseudomonads in C. elegans natural habitat, but it is unclear whether C. elegans ever encounters PA14-like bacteria in the wild. Thus, it is not known if small RNAs from bacteria found in C. elegans natural habitat can also regulate host behavior and produce heritable behavioral effects. Here we found that a pathogenic Pseudomonas vranovensis strain isolated from the C. elegans microbiota, GRb0427, like PA14, regulates worm behavior: worms learn to avoid this pathogenic bacterium following exposure to GRb0427, and this learned avoidance is inherited for four generations. The learned response is entirely mediated by bacterially-produced small RNAs, which induce avoidance and transgenerational inheritance, providing further support that such mechanisms of learning and inheritance exist in the wild. Using bacterial small RNA sequencing, we identified Pv1, a small RNA from GRb0427, that matches the sequence of C. elegans maco-1. We find that Pv1 is both necessary and sufficient to induce learned avoidance of Grb0427. However, Pv1 also results in avoidance of a beneficial microbiome strain, P. mendocina; this potentially maladaptive response may favor reversal of the transgenerational memory after a few generations. Our findings suggest that bacterial small RNA-mediated regulation of host behavior and its transgenerational inheritance are functional in C. elegans natural environment, and that different bacterial small RNA-mediated regulation systems evolved independently but define shared molecular features of bacterial small RNAs that produce transgenerationally-inherited effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.07.561364">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.07.561364" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.07.561364">
        <p class="paperTitle">Integrator-mediated clustering of poised RNA polymerase II synchronizes histone transcription</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.07.561364" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.07.561364" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, F.; Park, B. J.; Fujiwara, R.; Wilusz, J. E.; Gilmour, D. S.; Lehmann, R.; Lionnet, T.</p>
        <p class="info">Score: 41.1, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.07.561364' target='https://doi.org/10.1101/2023.10.07.561364'> 10.1101/2023.10.07.561364</a></p>
        <p class="abstract">Dynamic clustering of transcription players, including RNA polymerase II (Pol II), is thought to rely on multivalent interactions of their intrinsically disordered regions, thereby enhancing active transcription. Using the histone locus bodies (HLBs) of Drosophila nurse cells as a model, we find that Pol II forms long-lived, transcriptionally poised clusters distinct from liquid droplets, which contain unbound and paused Pol II. Depletion of the Integrator complex endonuclease module, but not its phosphatase module or pausing factors disperses these Pol II clusters, leaving HLBs intact. Consequently, histone transcription fails to reach peak levels during S-phase and aberrantly spills over throughout the cell cycle. We propose that clustering sequesters numerous poised Pol II molecules near gene promoters to ensure synchronous and efficient gene activation at desired times.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.561705">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.561705" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.561705">
        <p class="paperTitle">A gene with a thousand alleles: the HYPer-variable effectors of plant-parasitic nematodes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.561705" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.561705" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sonawala, U.; Beasley, H.; Thorpe, P. J.; Varypatakis, K.; Senatori, B.; Jones, J. T.; Derevnina, L.; Eves-van den Akker, S.</p>
        <p class="info">Score: 38.2, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.561705' target='https://doi.org/10.1101/2023.10.16.561705'> 10.1101/2023.10.16.561705</a></p>
        <p class="abstract">Pathogens are engaged in a fierce evolutionary arms race with their host. The genes at the forefront of the engagement between kingdoms, including effectors and immune receptors, are part of diverse and highly mutable gene families. Even in this context, we discovered unprecedented variation in the HYPer-variable (HYP) effectors of plant-parasitic nematodes.

We discovered single effector gene loci that can harbour potentially thousands of allelic variants. These alleles vary in the number, but principally in the organisation, of motifs within a central Hyper Variable Domain (HVD). Using targeted long-read sequencing, we dramatically expand the HYP repertoire of two plant-parasitic nematodes, Globodera pallida and G. rostochiensis, such that we can define distinct sets of species-specific &#34;rules&#34; underlying the apparently flawless genetic rearrangements. Finally, by analysing the HYP complement of 68 individual nematodes we made the unexpected finding that despite the huge number of alleles, most individuals are homozygous. Taken together, these data point to a novel mechanism of programmed genetic variation, termed HVD-editing, where alterations are locus-specific, strictly governed by rules, and can theoretically produce thousands of variants without errors.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
