<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Misexpression of inactive genes in whole blood is associated with nearby rare structural variants
Authors: Vanderstichele, T.; Burnham, K. L.; de Klein, N.; Tardaguila, M.; Howell, B.; Walter, K.; Kundu, K.; Koeppel, J.; Lee, W.; Tokolyi, A.; Persyn, E.; Nath, A. P.; Marten, J.; Petrovski, S.; Roberts, D. J.; Di Angelantonio, E.; Danesh, J.; Berton, A.; Platt, A.; Butterworth, A. S.; Soranzo, N.; Parts, L.; Inouye, M.; Paul, D. S.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Misexpression of inactive genes in whole blood is associated with nearby rare structural variants
Authors: Vanderstichele, T.; Burnham, K. L.; de Klein, N.; Tardaguila, M.; Howell, B.; Walter, K.; Kundu, K.; Koeppel, J.; Lee, W.; Tokolyi, A.; Persyn, E.; Nath, A. P.; Marten, J.; Petrovski, S.; Roberts, D. J.; Di Angelantonio, E.; Danesh, J.; Berton, A.; Platt, A.; Butterworth, A. S.; Soranzo, N.; Parts, L.; Inouye, M.; Paul, D. S." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-26T10:37:33+00:00" />
<meta property="article:modified_time" content="2023-11-26T10:37:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Misexpression of inactive genes in whole blood is associated with nearby rare structural variants
Authors: Vanderstichele, T.; Burnham, K. L.; de Klein, N.; Tardaguila, M.; Howell, B.; Walter, K.; Kundu, K.; Koeppel, J.; Lee, W.; Tokolyi, A.; Persyn, E.; Nath, A. P.; Marten, J.; Petrovski, S.; Roberts, D. J.; Di Angelantonio, E.; Danesh, J.; Berton, A.; Platt, A.; Butterworth, A. S.; Soranzo, N.; Parts, L.; Inouye, M.; Paul, D. S."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Misexpression of inactive genes in whole blood is associated with nearby rare structural variants\nAuthors: Vanderstichele, T.; Burnham, K. L.; de Klein, N.; Tardaguila, M.; Howell, B.; Walter, K.; Kundu, K.; Koeppel, J.; Lee, W.; Tokolyi, A.; Persyn, E.; Nath, A. P.; Marten, J.; Petrovski, S.; Roberts, D. J.; Di Angelantonio, E.; Danesh, J.; Berton, A.; Platt, A.; Butterworth, A. S.; Soranzo, N.; Parts, L.; Inouye, M.; Paul, D. S.",
  "keywords": [
    
  ],
  "articleBody": " Misexpression of inactive genes in whole blood is associated with nearby rare structural variants\nAuthors: Vanderstichele, T.; Burnham, K. L.; de Klein, N.; Tardaguila, M.; Howell, B.; Walter, K.; Kundu, K.; Koeppel, J.; Lee, W.; Tokolyi, A.; Persyn, E.; Nath, A. P.; Marten, J.; Petrovski, S.; Roberts, D. J.; Di Angelantonio, E.; Danesh, J.; Berton, A.; Platt, A.; Butterworth, A. S.; Soranzo, N.; Parts, L.; Inouye, M.; Paul, D. S.; Davenport, E. E.\nScore: 30.9, Published: 2023-11-19 DOI: 10.1101/2023.11.17.567537\nGene misexpression is the aberrant transcription of a gene in a context where it is usually inactive. Despite its known pathological consequences in specific rare diseases, we have a limited understanding of its wider prevalence and mechanisms in humans. To address this, we analyzed gene misexpression in 4,568 whole blood bulk RNA sequencing samples from INTERVAL study blood donors. We found that while individual misexpression events occur rarely, in aggregate they were found in almost all samples and over half of inactive genes. Using 2,821 paired whole genome and RNA sequencing samples, we identified that misexpression events are enriched in cis for rare structural variants. We established putative mechanisms through which a subset of SVs lead to gene misexpression, including transcriptional readthrough, transcript fusions and gene inversion. Overall, we develop misexpression as a novel type of transcriptomic outlier analysis and extend our understanding of the variety of mechanisms by which genetic variants can influence gene expression.\nWhole genome association testing in 333,100 individuals across three biobanks identifies rare non-coding single variant and genomic aggregate associations with height\nAuthors: Hawkes, G.; Beaumont, R. N.; Li, Z.; Mandla, R.; Li, X.; Albert, C. M.; Arnett, D. K.; Ashley-Koch, A. E.; Ashrani, A. A.; Barnes, K. C.; Boerwinkle, E.; Brody, J. A.; Carson, A. P.; Chami, N.; Chen, Y.-D. I.; Chung, M. K.; Curran, J. E.; Darbar, D.; Ellinor, P. T.; Fornage, M.; Gordeuk, V. R.; Guo, X.; He, J.; Hwu, C.-M.; Kalyani, R. R.; Kaplan, R.; Kardia, S. L. R.; Kooperberg, C.; Loos, R. J. F.; Lubitz, S. A.; Minster, R. L.; Mitchell, B. D.; Murabito, J. M.; Palmer, N. D.; Psaty, B. M.; Redline, S.; Shoemaker, M. B.; Silverman, E. K.; Telen, M. J.; Weiss, S. T.; Yanek, L. R.; Zhou, H.; NH\nScore: 27.8, Published: 2023-11-20 DOI: 10.1101/2023.11.19.566520\nThe role of rare non-coding variation in complex human phenotypes is still largely unknown. To elucidate the impact of rare variants in regulatory elements, we performed a whole-genome sequencing association analysis for height using 333,100 individuals from three datasets: UK Biobank (N=200,003), TOPMed (N=87,652) and All of Us (N=45,445). We performed rare (\u003c0.1% minor-allele-frequency) single-variant and aggregate testing of non-coding variants in regulatory regions based on proximal, intergenic and deep-intronic annotation. We observed 29 independent variants associated with height at P \u003c 6 x 10-10 after conditioning on previously reported variants, with effect sizes ranging from -7cm to +4.7cm. We also identified and replicated non-coding aggregate-based associations proximal to HMGA1 containing variants associated with a 5cm taller height and of highly-conserved variants in MIR497HG on chromosome 17. We have developed a novel approach for identifying non-coding rare variants in regulatory regions with large effects from whole-genome sequencing data associated with complex traits.\nModeling recent positive selection in Americans of European ancestry\nAuthors: Temple, S. D.; Waples, R. D.; Browning, S. R.\nScore: 22.7, Published: 2023-11-16 DOI: 10.1101/2023.11.13.566947\nRecent positive selection can result in an excess of long identity-by-descent (IBD) haplotype segments. The statistical methods that we propose here address three major objectives in studying classical selective sweeps: scanning for regions of interest, identifying possible sweeping alleles, and estimating a selection coefficient s. First, we implement a selection scan to locate regions of excess IBD rate. Second, we develop a statistic to rank alleles in strong linkage disequilibrium with a putative sweeping allele. We aggregate these scores to estimate the allele frequency of the sweeping allele, even if it is not genotyped. Lastly, we propose an estimator for the selection coefficient and quantify uncertainty using the parametric bootstrap. Comparing against state-of-the-art methods in extensive simulations, we show that our methods are better at pinpointing alleles at low frequency and estimating s [\u0026ge;] 0.015. We apply these methods to study positive selection in inferred European ancestry samples from the TOPMed project. We find twelve loci where their IBD rates exceed four standard deviations above the population median. The excess IBD rate at LCT is thirty-five standard deviations above the population median, and our estimates of its selection coefficient imply strong selection within the past 120 to 163 generations. Overall, we present robust and accurate new approaches to study very recent adaptive evolution under mild assumptions.\nHigh prevalence of Prdm9-independentrecombination hotspots in placental mammals\nAuthors: Joseph, J.; Prentout, D.; Laverre, A.; Tricou, T.; Duret, L.\nScore: 13.2, Published: 2023-11-17 DOI: 10.1101/2023.11.17.567540\nIn many mammals, recombination events are concentrated into hotspots directed by a sequence specific DNA-binding protein named Prdm9. This protein facilitates chromosome pairing and its inactivation has been shown to induce fertility losses in mice and rats. Intriguingly, Prdm9 has been lost several times in vertebrates, and notably among mammals, it has been pseudogenized in the ancestor of canids (dogs, wolves foxes). When this gene is inactive, either naturally in dogs, or through knock-out experiments in mice, recombination hotspots still exist, but they tend to occur in promoter-like features such as CpG islands. It has thus been proposed that one role of Prdm9 could be to direct recombination away from those Prdm9-independent hotspots. However, the ability of Prdm9 to direct recombination hotspots has been assessed only in a handful of species, and a clear picture of how much recombination occurs outside of Prdm9-directed hotspots in mammals is still lacking. In this study, we derived an estimator of past recombination activity based on signatures of GC-biased gene conversion in substitution patterns. We applied it to quantify recombination activity in Prdm9-independent hotspots in 52 species of boreoeutherian mammals. We observed a wide range of recombination rate at these loci: several species (such as mice, humans, some felids or cetaceans) show a deficit of recombination, while a majority of mammals display a clear peak of recombination. Our results demonstrate that Prdm9-directed and Prdm9-independent hotspots can co-exist in mammals, and that their co-existence seem to be the rule rather than an exception.\nAssessing the impact of 20th century internal migrations on the genetic structure of Estonia\nAuthors: Kuznetsov, I. A.; Estonian Biobank Research Team, ; Metspalu, M.; Vainik, U.; Pagani, L.; Montinaro, F.; Pankratov, V.\nScore: 31.9, Published: 2023-10-29 DOI: 10.1101/2023.10.25.564036\nSpatial genetic structure observed in many human populations is in large part attributed to past demographic events and isolation by distance. However, how intensifying migration affects this structure remains understudied. Here we harness a sample of more than 180 thousand individuals to explore the genetic correlates and consequences of contemporary migrations in Estonia. While we show that migration smoothens the genome-wide genetic structure, it intensifies inter-regional differences in polygenic scores (PGS) for certain traits, derived both from population as well as within-sibship studies. The strongest effect is observed for educational attainment which is consistent with previous observations in the UK and suggests this to be a general pattern. We explore those regional differences in PGS in terms of the driving forces behind them and from a temporal perspective, and suggest urbanisation as a major driver for this pattern in Estonia from at least the first half of the 20th century.\nmtDNA \"Nomenclutter\" and its Consequences on the Interpretation of Genetic Data\nAuthors: Bajic, V.; Schulmann, V. H.; Nowick, K.\nScore: 7.2, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567721\nPopulation-based studies of mitochondrial genetic diversity often require the classification of mitochondrial DNA (mtDNA) haplotypes into more than 2000 described haplogroups, and further grouping those into hierarchically higher haplogroups. Such secondary haplogroup groupings (e.g. \"macro-haplogroups\") vary across studies, as they depend on the sample quality, technical factors of haplogroup calling, the aims of the study, and the researchers understanding of the mtDNA haplogroup nomenclature. Retention of historical nomenclature coupled with a growing number of newly described mtDNA lineages results in increasingly complex and inconsistent nomenclature that does not reflect phylogeny well. This \"clutter\" leaves room for grouping errors and inconsistencies between scientific publications, especially when the haplogroup names are used as a proxy for secondary groupings, and represents a source for scientific misinterpretation. Here we explore the effects of phylogenetically insensitive secondary mtDNA haplogroup groupings, and the lack of standardized secondary haplogroup groupings on downstream analyses and interpretation of genetic data. We demonstrate that frequency-based analyses produce inconsistent results when different secondary mtDNA groupings are applied, and thus allow for vastly different interpretations of the same genetic data. The lack of guidelines and recommendations on how to choose appropriate secondary haplogroup groupings presents an issue for the interpretation of results, as well as their comparison and reproducibility across studies. To reduce biases introduced by arbitrarily defined secondary nomenclature-based groupings, we suggest the implementation of phylogenetically meaningful algorithm-based groupings to define a standardized set of \"macro-haplogroups\", \"meso-haplogroups\", and \"micro-haplogroups\". Such phylogenetically informative levels of haplogroup groupings can be easily implemented into haplogroup callers such as HaploGrep3. This would foster reproducibility across studies, provide a grouping standard for population-based studies, and reduce errors associated with haplogroup nomenclatures in future studies.\nPolyglucosan body density in the aged mouse hippocampus is controlled by a novel modifier locus on chromosome 1\nAuthors: Gomez-Pascual, A.; Glikman, D. M.; Ng, H. X.; Tomkins, J. E.; Lu, L.; Xu, Y.; Ashbrook, D. G.; Kaczorowski, C.; Kempermann, G.; Killmar, J.; Mozhui, K.; Aebersold, R.; Williams, E. G.; Williams, R. W.; Overall, R. W.; Jucker, M.; de Bakker, D. E. M.\nScore: 6.3, Published: 2023-11-22 DOI: 10.1101/2023.11.22.567373\nAging can be associated with the accumulation of hypobranched glycogen molecules (polyglucosan bodies, PGBs), particularly in astrocytes of the hippocampus. While PGBs have a detrimental effect on cognition in diseases such as adult polyglucosan body disease and Lafora disease, the underlying mechanism and clinical relevance of age-related PGB accumulation remains unknown. Here, we have investigated the genetic basis and functional impact of age-related PGB accumulation in 32 fully sequenced BXD-type strains of mice which exhibit a 400-fold variation in PGB burden in 16-18 month old females. We mapped a major locus controlling PGB density in the hippocampus to chromosome 1 at 72-75 Mb (linkage of 4.9 -logP), which we defined as the Pgb1 locus. To identify potentially causal gene variants within Pgb1, we generated extensive hippocampal transcriptome datasets and identified two strong candidate genes for which mRNA correlates with PGB density--Smarcal1 and Usp37. In addition, both Smarcal1 and Usp37 contain non-synonymous allele variations likely to impact protein function. A phenome-wide association analysis highlighted a trans-regulatory effect of the Pgb1 locus on expression of Hp1bp3, a gene known to play a role in age-related changes in learning and memory. To investigate the potential impact of PGBs on cognition, we performed conditioned fear memory testing on strains displaying varying degrees of PGB burden, and a phenome-wide association scan of ~12,000 traits. Importantly, we did not find any evidence suggesting a negative impact of PGB burden on cognitive capacity. Taken together, we have identified a major modifier locus controlling PGB burden in the hippocampus and shed light on the genetic architecture and clinical relevance of this strikingly heterogeneous hippocampal phenotype.\nCell morphology QTL reveal gene by environment interactions in a genetically diverse cell population\nAuthors: O'Connor, C.; Keele, G. R.; Martin, W.; Stodola, T.; Gatti, D.; Hoffman, B. R.; Korstanje, R.; Churchill, G. A.; Reinholdt, L. G.\nScore: 5.7, Published: 2023-11-18 DOI: 10.1101/2023.11.18.567597\nGenetically heterogenous cell lines from laboratory mice are promising tools for population-based screening as they offer power for genetic mapping, and potentially, predictive value for in vivo experimentation in genetically matched individuals. To explore this further, we derived a panel of fibroblast lines from a genetic reference population of laboratory mice (the Diversity Outbred, DO). We then used high-content imaging to capture hundreds of cell morphology traits in cells exposed to the oxidative stress-inducing arsenic metabolite monomethylarsonous acid (MMAIII). We employed dose-response modeling to capture latent parameters of response and we then used these parameters to identify several hundred cell morphology quantitative trait loci (cmQTL). Response cmQTL encompass genes with established associations with cellular responses to arsenic exposure, including Abcc4 and Txnrd1, as well as novel gene candidates like Xrcc2. Moreover, baseline trait cmQTL highlight the influence of natural variation on fundamental aspects of nuclear morphology. We show that the natural variants influencing response include both coding and non-coding variation, and that cmQTL haplotypes can be used to predict response in orthogonal cell lines. Our study sheds light on the major molecular initiating events of oxidative stress that are under genetic regulation, including the NRF2-mediated antioxidant response, cellular detoxification pathways, DNA damage repair response, and cell death trajectories.\nMechanisms of transcriptional regulation in Anopheles gambiae revealed by allele specific expression\nAuthors: Dyer, N. A.; Lucas, E. R.; Nagi, S. C.; McDermott, D. P.; Brenas, J. H.; Miles, A.; Clarkson, C. S.; Mawejje, H. D.; Wilding, C. S.; Halfon, M. S.; Asma, H.; Heinz, E.; Donnelly, M. J.\nScore: 4.3, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568226\nMalaria control relies on insecticides targeting the mosquito vector, but this is increasingly compromised by insecticide resistance, which can be achieved by elevated expression of detoxifying enzymes that metabolize the insecticide. In diploid organisms, gene expression is regulated both in cis, by regulatory sequences on the same chromosome, and by trans acting factors, affecting both alleles equally. Differing levels of transcription can be caused by mutations in cis-regulatory modules (CRM), but few of these have been identified in mosquitoes. We crossed bendiocarb resistant and susceptible Anopheles gambiae strains to identify cis-regulated genes that might be responsible for the resistant phenotype using RNAseq, and cis-regulatory module sequences controlling gene expression in insecticide resistance relevant tissues were predicted using machine learning. We found 115 genes showing allele specific expression in hybrids of insecticide susceptible and resistant strains, suggesting cis regulation is an important mechanism of gene expression regulation in Anopheles gambiae. The genes showing allele specific expression included a higher proportion of Anopheles-specific genes on average younger than genes those with balanced allelic expression.\nAccounting for nuclear and mito genome in dairy cattle breeding - a simulation study\nAuthors: Mafra Fortuna, G.; Zumbach, B. J.; Johnsson, M.; Pocrnic, I.; Gorjanc, G.\nScore: 3.6, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567907\nMitochondria play a significant role in numerous cellular processes through proteins encoded by both nuclear genome (nDNA) and mito genome (mDNA). While the variation in nDNA is influenced by mutations and recombination of parental genomes, the variation in mDNA is solely driven by mutations. In addition, mDNA is inherited in a haploid form, from the dam. Cattle populations show significant variation in mDNA between and within breeds. Past research suggests that variation in mDNA accounts for 1-5% of the phenotypic variation in dairy traits. Here we simulated a dairy cattle breeding program to assess the impact of accounting for mDNA variation in pedigree-based and genome-based genetic evaluations on the accuracy of estimated breeding values for mDNA and nDNA components. We also examined the impact of alternative definitions of breeding values on genetic gain, including nDNA and mDNA components that both impact phenotype expression, but mDNA is inherited only maternally. We found that accounting for mDNA variation increased accuracy between +0.01 and +0.05 for different categories of animals, especially for young bulls (+0.05) and females without genotype data (between +0.01 and +0.03). Different scenarios of modelling and breeding value definition impacted genetic gain. The standard approach of ignoring mDNA variation achieved competitive genetic gain. Modelling, but not selecting on mDNA expectedly reduced genetic gain, while optimal use of mDNA variation recovered the genetic gain.\n",
  "wordCount" : "2605",
  "inLanguage": "en",
  "datePublished": "2023-11-26T10:37:33Z",
  "dateModified": "2023-11-26T10:37:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 26, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567537">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567537" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567537">
        <p class="paperTitle">Misexpression of inactive genes in whole blood is associated with nearby rare structural variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567537" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567537" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vanderstichele, T.; Burnham, K. L.; de Klein, N.; Tardaguila, M.; Howell, B.; Walter, K.; Kundu, K.; Koeppel, J.; Lee, W.; Tokolyi, A.; Persyn, E.; Nath, A. P.; Marten, J.; Petrovski, S.; Roberts, D. J.; Di Angelantonio, E.; Danesh, J.; Berton, A.; Platt, A.; Butterworth, A. S.; Soranzo, N.; Parts, L.; Inouye, M.; Paul, D. S.; Davenport, E. E.</p>
        <p class="info">Score: 30.9, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567537' target='https://doi.org/10.1101/2023.11.17.567537'> 10.1101/2023.11.17.567537</a></p>
        <p class="abstract">Gene misexpression is the aberrant transcription of a gene in a context where it is usually inactive. Despite its known pathological consequences in specific rare diseases, we have a limited understanding of its wider prevalence and mechanisms in humans. To address this, we analyzed gene misexpression in 4,568 whole blood bulk RNA sequencing samples from INTERVAL study blood donors. We found that while individual misexpression events occur rarely, in aggregate they were found in almost all samples and over half of inactive genes. Using 2,821 paired whole genome and RNA sequencing samples, we identified that misexpression events are enriched in cis for rare structural variants. We established putative mechanisms through which a subset of SVs lead to gene misexpression, including transcriptional readthrough, transcript fusions and gene inversion. Overall, we develop misexpression as a novel type of transcriptomic outlier analysis and extend our understanding of the variety of mechanisms by which genetic variants can influence gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.566520">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.566520" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.566520">
        <p class="paperTitle">Whole genome association testing in 333,100 individuals across three biobanks identifies rare non-coding single variant and genomic aggregate associations with height</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.566520" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.566520" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hawkes, G.; Beaumont, R. N.; Li, Z.; Mandla, R.; Li, X.; Albert, C. M.; Arnett, D. K.; Ashley-Koch, A. E.; Ashrani, A. A.; Barnes, K. C.; Boerwinkle, E.; Brody, J. A.; Carson, A. P.; Chami, N.; Chen, Y.-D. I.; Chung, M. K.; Curran, J. E.; Darbar, D.; Ellinor, P. T.; Fornage, M.; Gordeuk, V. R.; Guo, X.; He, J.; Hwu, C.-M.; Kalyani, R. R.; Kaplan, R.; Kardia, S. L. R.; Kooperberg, C.; Loos, R. J. F.; Lubitz, S. A.; Minster, R. L.; Mitchell, B. D.; Murabito, J. M.; Palmer, N. D.; Psaty, B. M.; Redline, S.; Shoemaker, M. B.; Silverman, E. K.; Telen, M. J.; Weiss, S. T.; Yanek, L. R.; Zhou, H.; NH</p>
        <p class="info">Score: 27.8, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.566520' target='https://doi.org/10.1101/2023.11.19.566520'> 10.1101/2023.11.19.566520</a></p>
        <p class="abstract">The role of rare non-coding variation in complex human phenotypes is still largely unknown. To elucidate the impact of rare variants in regulatory elements, we performed a whole-genome sequencing association analysis for height using 333,100 individuals from three datasets: UK Biobank (N=200,003), TOPMed (N=87,652) and All of Us (N=45,445). We performed rare (&lt;0.1% minor-allele-frequency) single-variant and aggregate testing of non-coding variants in regulatory regions based on proximal, intergenic and deep-intronic annotation. We observed 29 independent variants associated with height at P &lt; 6 x 10-10 after conditioning on previously reported variants, with effect sizes ranging from -7cm to &#43;4.7cm. We also identified and replicated non-coding aggregate-based associations proximal to HMGA1 containing variants associated with a 5cm taller height and of highly-conserved variants in MIR497HG on chromosome 17. We have developed a novel approach for identifying non-coding rare variants in regulatory regions with large effects from whole-genome sequencing data associated with complex traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566947">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566947" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566947">
        <p class="paperTitle">Modeling recent positive selection in Americans of European ancestry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566947" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566947" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Temple, S. D.; Waples, R. D.; Browning, S. R.</p>
        <p class="info">Score: 22.7, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566947' target='https://doi.org/10.1101/2023.11.13.566947'> 10.1101/2023.11.13.566947</a></p>
        <p class="abstract">Recent positive selection can result in an excess of long identity-by-descent (IBD) haplotype segments. The statistical methods that we propose here address three major objectives in studying classical selective sweeps: scanning for regions of interest, identifying possible sweeping alleles, and estimating a selection coefficient s. First, we implement a selection scan to locate regions of excess IBD rate. Second, we develop a statistic to rank alleles in strong linkage disequilibrium with a putative sweeping allele. We aggregate these scores to estimate the allele frequency of the sweeping allele, even if it is not genotyped. Lastly, we propose an estimator for the selection coefficient and quantify uncertainty using the parametric bootstrap. Comparing against state-of-the-art methods in extensive simulations, we show that our methods are better at pinpointing alleles at low frequency and estimating s [&amp;ge;] 0.015. We apply these methods to study positive selection in inferred European ancestry samples from the TOPMed project. We find twelve loci where their IBD rates exceed four standard deviations above the population median. The excess IBD rate at LCT is thirty-five standard deviations above the population median, and our estimates of its selection coefficient imply strong selection within the past 120 to 163 generations. Overall, we present robust and accurate new approaches to study very recent adaptive evolution under mild assumptions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567540">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567540" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567540">
        <p class="paperTitle">High prevalence of Prdm9-independentrecombination hotspots in placental mammals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567540" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567540" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joseph, J.; Prentout, D.; Laverre, A.; Tricou, T.; Duret, L.</p>
        <p class="info">Score: 13.2, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567540' target='https://doi.org/10.1101/2023.11.17.567540'> 10.1101/2023.11.17.567540</a></p>
        <p class="abstract">In many mammals, recombination events are concentrated into hotspots directed by a sequence specific DNA-binding protein named Prdm9. This protein facilitates chromosome pairing and its inactivation has been shown to induce fertility losses in mice and rats. Intriguingly, Prdm9 has been lost several times in vertebrates, and notably among mammals, it has been pseudogenized in the ancestor of canids (dogs, wolves foxes). When this gene is inactive, either naturally in dogs, or through knock-out experiments in mice, recombination hotspots still exist, but they tend to occur in promoter-like features such as CpG islands. It has thus been proposed that one role of Prdm9 could be to direct recombination away from those Prdm9-independent hotspots. However, the ability of Prdm9 to direct recombination hotspots has been assessed only in a handful of species, and a clear picture of how much recombination occurs outside of Prdm9-directed hotspots in mammals is still lacking. In this study, we derived an estimator of past recombination activity based on signatures of GC-biased gene conversion in substitution patterns. We applied it to quantify recombination activity in Prdm9-independent hotspots in 52 species of boreoeutherian mammals. We observed a wide range of recombination rate at these loci: several species (such as mice, humans, some felids or cetaceans) show a deficit of recombination, while a majority of mammals display a clear peak of recombination. Our results demonstrate that Prdm9-directed and Prdm9-independent hotspots can co-exist in mammals, and that their co-existence seem to be the rule rather than an exception.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564036">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564036" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564036">
        <p class="paperTitle">Assessing the impact of 20th century internal migrations on the genetic structure of Estonia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564036" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564036" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kuznetsov, I. A.; Estonian Biobank Research Team,  ; Metspalu, M.; Vainik, U.; Pagani, L.; Montinaro, F.; Pankratov, V.</p>
        <p class="info">Score: 31.9, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564036' target='https://doi.org/10.1101/2023.10.25.564036'> 10.1101/2023.10.25.564036</a></p>
        <p class="abstract">Spatial genetic structure observed in many human populations is in large part attributed to past demographic events and isolation by distance. However, how intensifying migration affects this structure remains understudied. Here we harness a sample of more than 180 thousand individuals to explore the genetic correlates and consequences of contemporary migrations in Estonia. While we show that migration smoothens the genome-wide genetic structure, it intensifies inter-regional differences in polygenic scores (PGS) for certain traits, derived both from population as well as within-sibship studies. The strongest effect is observed for educational attainment which is consistent with previous observations in the UK and suggests this to be a general pattern. We explore those regional differences in PGS in terms of the driving forces behind them and from a temporal perspective, and suggest urbanisation as a major driver for this pattern in Estonia from at least the first half of the 20th century.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567721">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567721" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567721">
        <p class="paperTitle">mtDNA &#34;Nomenclutter&#34; and its Consequences on the Interpretation of Genetic Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567721" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567721" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bajic, V.; Schulmann, V. H.; Nowick, K.</p>
        <p class="info">Score: 7.2, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567721' target='https://doi.org/10.1101/2023.11.19.567721'> 10.1101/2023.11.19.567721</a></p>
        <p class="abstract">Population-based studies of mitochondrial genetic diversity often require the classification of mitochondrial DNA (mtDNA) haplotypes into more than 2000 described haplogroups, and further grouping those into hierarchically higher haplogroups. Such secondary haplogroup groupings (e.g. &#34;macro-haplogroups&#34;) vary across studies, as they depend on the sample quality, technical factors of haplogroup calling, the aims of the study, and the researchers understanding of the mtDNA haplogroup nomenclature. Retention of historical nomenclature coupled with a growing number of newly described mtDNA lineages results in increasingly complex and inconsistent nomenclature that does not reflect phylogeny well. This &#34;clutter&#34; leaves room for grouping errors and inconsistencies between scientific publications, especially when the haplogroup names are used as a proxy for secondary groupings, and represents a source for scientific misinterpretation.

Here we explore the effects of phylogenetically insensitive secondary mtDNA haplogroup groupings, and the lack of standardized secondary haplogroup groupings on downstream analyses and interpretation of genetic data. We demonstrate that frequency-based analyses produce inconsistent results when different secondary mtDNA groupings are applied, and thus allow for vastly different interpretations of the same genetic data. The lack of guidelines and recommendations on how to choose appropriate secondary haplogroup groupings presents an issue for the interpretation of results, as well as their comparison and reproducibility across studies.

To reduce biases introduced by arbitrarily defined secondary nomenclature-based groupings, we suggest the implementation of phylogenetically meaningful algorithm-based groupings to define a standardized set of &#34;macro-haplogroups&#34;, &#34;meso-haplogroups&#34;, and &#34;micro-haplogroups&#34;. Such phylogenetically informative levels of haplogroup groupings can be easily implemented into haplogroup callers such as HaploGrep3. This would foster reproducibility across studies, provide a grouping standard for population-based studies, and reduce errors associated with haplogroup nomenclatures in future studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.567373">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.567373" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.567373">
        <p class="paperTitle">Polyglucosan body density in the aged mouse hippocampus is controlled by a novel modifier locus on chromosome 1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.567373" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.567373" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gomez-Pascual, A.; Glikman, D. M.; Ng, H. X.; Tomkins, J. E.; Lu, L.; Xu, Y.; Ashbrook, D. G.; Kaczorowski, C.; Kempermann, G.; Killmar, J.; Mozhui, K.; Aebersold, R.; Williams, E. G.; Williams, R. W.; Overall, R. W.; Jucker, M.; de Bakker, D. E. M.</p>
        <p class="info">Score: 6.3, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.567373' target='https://doi.org/10.1101/2023.11.22.567373'> 10.1101/2023.11.22.567373</a></p>
        <p class="abstract">Aging can be associated with the accumulation of hypobranched glycogen molecules (polyglucosan bodies, PGBs), particularly in astrocytes of the hippocampus. While PGBs have a detrimental effect on cognition in diseases such as adult polyglucosan body disease and Lafora disease, the underlying mechanism and clinical relevance of age-related PGB accumulation remains unknown. Here, we have investigated the genetic basis and functional impact of age-related PGB accumulation in 32 fully sequenced BXD-type strains of mice which exhibit a 400-fold variation in PGB burden in 16-18 month old females. We mapped a major locus controlling PGB density in the hippocampus to chromosome 1 at 72-75 Mb (linkage of 4.9 -logP), which we defined as the Pgb1 locus. To identify potentially causal gene variants within Pgb1, we generated extensive hippocampal transcriptome datasets and identified two strong candidate genes for which mRNA correlates with PGB density--Smarcal1 and Usp37. In addition, both Smarcal1 and Usp37 contain non-synonymous allele variations likely to impact protein function. A phenome-wide association analysis highlighted a trans-regulatory effect of the Pgb1 locus on expression of Hp1bp3, a gene known to play a role in age-related changes in learning and memory. To investigate the potential impact of PGBs on cognition, we performed conditioned fear memory testing on strains displaying varying degrees of PGB burden, and a phenome-wide association scan of ~12,000 traits. Importantly, we did not find any evidence suggesting a negative impact of PGB burden on cognitive capacity. Taken together, we have identified a major modifier locus controlling PGB burden in the hippocampus and shed light on the genetic architecture and clinical relevance of this strikingly heterogeneous hippocampal phenotype.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.18.567597">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.18.567597" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.18.567597">
        <p class="paperTitle">Cell morphology QTL reveal gene by environment interactions in a genetically diverse cell population</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.18.567597" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.18.567597" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Connor, C.; Keele, G. R.; Martin, W.; Stodola, T.; Gatti, D.; Hoffman, B. R.; Korstanje, R.; Churchill, G. A.; Reinholdt, L. G.</p>
        <p class="info">Score: 5.7, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.18.567597' target='https://doi.org/10.1101/2023.11.18.567597'> 10.1101/2023.11.18.567597</a></p>
        <p class="abstract">Genetically heterogenous cell lines from laboratory mice are promising tools for population-based screening as they offer power for genetic mapping, and potentially, predictive value for in vivo experimentation in genetically matched individuals. To explore this further, we derived a panel of fibroblast lines from a genetic reference population of laboratory mice (the Diversity Outbred, DO). We then used high-content imaging to capture hundreds of cell morphology traits in cells exposed to the oxidative stress-inducing arsenic metabolite monomethylarsonous acid (MMAIII). We employed dose-response modeling to capture latent parameters of response and we then used these parameters to identify several hundred cell morphology quantitative trait loci (cmQTL). Response cmQTL encompass genes with established associations with cellular responses to arsenic exposure, including Abcc4 and Txnrd1, as well as novel gene candidates like Xrcc2. Moreover, baseline trait cmQTL highlight the influence of natural variation on fundamental aspects of nuclear morphology. We show that the natural variants influencing response include both coding and non-coding variation, and that cmQTL haplotypes can be used to predict response in orthogonal cell lines. Our study sheds light on the major molecular initiating events of oxidative stress that are under genetic regulation, including the NRF2-mediated antioxidant response, cellular detoxification pathways, DNA damage repair response, and cell death trajectories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568226">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568226" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568226">
        <p class="paperTitle">Mechanisms of transcriptional regulation in Anopheles gambiae revealed by allele specific expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568226" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568226" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dyer, N. A.; Lucas, E. R.; Nagi, S. C.; McDermott, D. P.; Brenas, J. H.; Miles, A.; Clarkson, C. S.; Mawejje, H. D.; Wilding, C. S.; Halfon, M. S.; Asma, H.; Heinz, E.; Donnelly, M. J.</p>
        <p class="info">Score: 4.3, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568226' target='https://doi.org/10.1101/2023.11.22.568226'> 10.1101/2023.11.22.568226</a></p>
        <p class="abstract">Malaria control relies on insecticides targeting the mosquito vector, but this is increasingly compromised by insecticide resistance, which can be achieved by elevated expression of detoxifying enzymes that metabolize the insecticide. In diploid organisms, gene expression is regulated both in cis, by regulatory sequences on the same chromosome, and by trans acting factors, affecting both alleles equally. Differing levels of transcription can be caused by mutations in cis-regulatory modules (CRM), but few of these have been identified in mosquitoes. We crossed bendiocarb resistant and susceptible Anopheles gambiae strains to identify cis-regulated genes that might be responsible for the resistant phenotype using RNAseq, and cis-regulatory module sequences controlling gene expression in insecticide resistance relevant tissues were predicted using machine learning. We found 115 genes showing allele specific expression in hybrids of insecticide susceptible and resistant strains, suggesting cis regulation is an important mechanism of gene expression regulation in Anopheles gambiae. The genes showing allele specific expression included a higher proportion of Anopheles-specific genes on average younger than genes those with balanced allelic expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567907">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567907" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567907">
        <p class="paperTitle">Accounting for nuclear and mito genome in dairy cattle breeding - a simulation study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567907" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567907" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mafra Fortuna, G.; Zumbach, B. J.; Johnsson, M.; Pocrnic, I.; Gorjanc, G.</p>
        <p class="info">Score: 3.6, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567907' target='https://doi.org/10.1101/2023.11.20.567907'> 10.1101/2023.11.20.567907</a></p>
        <p class="abstract">Mitochondria play a significant role in numerous cellular processes through proteins encoded by both nuclear genome (nDNA) and mito genome (mDNA). While the variation in nDNA is influenced by mutations and recombination of parental genomes, the variation in mDNA is solely driven by mutations. In addition, mDNA is inherited in a haploid form, from the dam. Cattle populations show significant variation in mDNA between and within breeds. Past research suggests that variation in mDNA accounts for 1-5% of the phenotypic variation in dairy traits. Here we simulated a dairy cattle breeding program to assess the impact of accounting for mDNA variation in pedigree-based and genome-based genetic evaluations on the accuracy of estimated breeding values for mDNA and nDNA components. We also examined the impact of alternative definitions of breeding values on genetic gain, including nDNA and mDNA components that both impact phenotype expression, but mDNA is inherited only maternally. We found that accounting for mDNA variation increased accuracy between &#43;0.01 and &#43;0.05 for different categories of animals, especially for young bulls (&#43;0.05) and females without genotype data (between &#43;0.01 and &#43;0.03). Different scenarios of modelling and breeding value definition impacted genetic gain. The standard approach of ignoring mDNA variation achieved competitive genetic gain. Modelling, but not selecting on mDNA expectedly reduced genetic gain, while optimal use of mDNA variation recovered the genetic gain.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
