<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Nucleus-Independent Transgenerational Small RNA Inheritance in C. elegans
Authors: Rieger, I.; Weintraub, G.; Lev, I.; Goldstein, K.; Bar-Zvi, D.; Anava, S.; Gingold, H.; Shaham, S.; Rechavi, O.
Score: 155.9, Published: 2023-06-21 DOI: 10.1101/2023.06.20.545749
Studies using C. elegans nematodes demonstrated that, against the dogma, animals can transmit epigenetic information transgenerationally. While it is now clear that in these worms ancestral RNA interference (RNAi) responses continue to regulate gene expression for many generations, it is still debated whether the primary agent that perpetuates heritable silencing is RNA or chromatin, and whether the information is communicated to the next generation inside or outside of the nucleus.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Nucleus-Independent Transgenerational Small RNA Inheritance in C. elegans
Authors: Rieger, I.; Weintraub, G.; Lev, I.; Goldstein, K.; Bar-Zvi, D.; Anava, S.; Gingold, H.; Shaham, S.; Rechavi, O.
Score: 155.9, Published: 2023-06-21 DOI: 10.1101/2023.06.20.545749
Studies using C. elegans nematodes demonstrated that, against the dogma, animals can transmit epigenetic information transgenerationally. While it is now clear that in these worms ancestral RNA interference (RNAi) responses continue to regulate gene expression for many generations, it is still debated whether the primary agent that perpetuates heritable silencing is RNA or chromatin, and whether the information is communicated to the next generation inside or outside of the nucleus." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T11:45:43+00:00" />
<meta property="article:modified_time" content="2023-06-28T11:45:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Nucleus-Independent Transgenerational Small RNA Inheritance in C. elegans
Authors: Rieger, I.; Weintraub, G.; Lev, I.; Goldstein, K.; Bar-Zvi, D.; Anava, S.; Gingold, H.; Shaham, S.; Rechavi, O.
Score: 155.9, Published: 2023-06-21 DOI: 10.1101/2023.06.20.545749
Studies using C. elegans nematodes demonstrated that, against the dogma, animals can transmit epigenetic information transgenerationally. While it is now clear that in these worms ancestral RNA interference (RNAi) responses continue to regulate gene expression for many generations, it is still debated whether the primary agent that perpetuates heritable silencing is RNA or chromatin, and whether the information is communicated to the next generation inside or outside of the nucleus."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Nucleus-Independent Transgenerational Small RNA Inheritance in C. elegans\nAuthors: Rieger, I.; Weintraub, G.; Lev, I.; Goldstein, K.; Bar-Zvi, D.; Anava, S.; Gingold, H.; Shaham, S.; Rechavi, O.\nScore: 155.9, Published: 2023-06-21 DOI: 10.1101/2023.06.20.545749\nStudies using C. elegans nematodes demonstrated that, against the dogma, animals can transmit epigenetic information transgenerationally. While it is now clear that in these worms ancestral RNA interference (RNAi) responses continue to regulate gene expression for many generations, it is still debated whether the primary agent that perpetuates heritable silencing is RNA or chromatin, and whether the information is communicated to the next generation inside or outside of the nucleus.",
  "keywords": [
    
  ],
  "articleBody": " Nucleus-Independent Transgenerational Small RNA Inheritance in C. elegans\nAuthors: Rieger, I.; Weintraub, G.; Lev, I.; Goldstein, K.; Bar-Zvi, D.; Anava, S.; Gingold, H.; Shaham, S.; Rechavi, O.\nScore: 155.9, Published: 2023-06-21 DOI: 10.1101/2023.06.20.545749\nStudies using C. elegans nematodes demonstrated that, against the dogma, animals can transmit epigenetic information transgenerationally. While it is now clear that in these worms ancestral RNA interference (RNAi) responses continue to regulate gene expression for many generations, it is still debated whether the primary agent that perpetuates heritable silencing is RNA or chromatin, and whether the information is communicated to the next generation inside or outside of the nucleus. Here we take advantage of the tractability of gene-specific double stranded RNA-induced heritable silencing to answer these questions. We demonstrate that RNAi can be inherited independently of any changes to the chromatin or any other nuclear factors via mothers that are genetically engineered to transmit only their ooplasm but not the oocytes nuclei to the next generation. Nucleus-independent RNA inheritance depends on ZNFX-1, an RNA-binding germ granule resident protein. We find that upon manipulation of normal germ granules functions (in pptr-1 mutants) nucleus-independent RNA inheritance becomes stronger, and can occur even in znfx-1 mutants. Utilizing RNA sequencing, chimeric worms, and sequence polymorphism between different C. elegans isolates, we identify endogenous small RNAs which, similarly to exogenous siRNAs, are inherited in a nucleus-independent manner. From an historical perspective, nucleus-independent inheritance of small RNAs might be regarded as partial vindication of discredited cytoplasmic inheritance theories from the 19th century, such as Darwins \"pangenesis\" theory.\nFetally-encoded GDF15 and maternal GDF15 sensitivity are major determinants of nausea and vomiting in human pregnancy\nAuthors: Fejzo, M.; Rocha, N.; Cimino, I.; Lockhart, S. M.; Petry, C.; Kay, R. G.; Burling, K.; Barker, P.; George, A. L.; Yasara, N.; Premawardhena, A.; Gong, S.; Cook, E.; Rainbow, K.; Withers, D.; Cortessis, V.; Mullin, P.; MacGibbon, K.; Jin, E.; Kam, A.; Campbell, A.; Polasek, O.; Tzoneva, G.; Gribble, F.; Yeo, G.; Lam, B.; Saudek, V.; Hughes, I.; Ong, K.; Perry, J.; Sutton-Cole, A.; Baumgarten, M.; Welsh, P.; Sattar, N.; Smith, G. C.; Charnock-Jones, D. S.; Coll, A. P.; Meek, C. L.; Mettananda, S.; Hayward, C.; Mancuso, N.; O'Rahilly, S.\nScore: 211.2, Published: 2023-06-04 DOI: 10.1101/2023.06.02.542661\nHuman pregnancy is frequently accompanied by nausea and vomiting that may become severe and life-threatening, as in hyperemesis gravidarum (HG), the cause of which is unknown. Growth Differentiation Factor-15 (GDF15), a hormone known to act on the hindbrain to cause emesis, is highly expressed in the placenta and its levels in maternal blood rise rapidly in pregnancy. Variants in the maternal GDF15 gene are associated with HG. Here we report that fetal production of GDF15, and maternal sensitivity to it, both contribute substantially to the risk of HG. We found that the great majority of GDF15 in maternal circulation is derived from the feto-placental unit and that higher GDF15 levels in maternal blood are associated with vomiting and are further elevated in patients with HG. Conversely, we found that lower levels of GDF15 in the non-pregnant state predispose women to HG. A rare C211G variant in GDF15 which strongly predisposes mothers to HG, particularly when the fetus is wild-type, was found to markedly impair cellular secretion of GDF15 and associate with low circulating levels of GDF15 in the non-pregnant state. Consistent with this, two common GDF15 haplotypes which predispose to HG were associated with lower circulating levels outside pregnancy. The administration of a long-acting form of GDF15 to wild-type mice markedly reduced subsequent responses to an acute dose, establishing that desensitisation is a feature of this system. GDF15 levels are known to be highly and chronically elevated in patients with beta thalassemia. In women with this disorder, reports of symptoms of nausea or vomiting in pregnancy were strikingly diminished. Our findings support a causal role for fetal derived GDF15 in the nausea and vomiting of human pregnancy, with maternal sensitivity, at least partly determined by pre-pregnancy exposure to GDF15, being a major influence on its severity. They also suggest mechanism-based approaches to the treatment and prevention of HG.\nAncient DNA reveals the origins of the Albanians\nAuthors: Davranoglou, L. R.; Aristodemou, A.; Wesolowski, D.; Heraclides, A.\nScore: 189.9, Published: 2023-06-07 DOI: 10.1101/2023.06.05.543790\nThe origins of the Albanian people have vexed linguists and historians for centuries, as Albanians first appear in the historical record in the 11th century CE, while their language is one of the most enigmatic branches of the Indo-European family. To identify the populations that contributed to the ancestry of Albanians, we undertake a genomic transect of the Balkans over the last 8000 years, where we analyse more than 6000 previously published ancient genomes using state-of-the-art bioinformatics tools and algorithms that quantify spatiotemporal human mobility. We find that modern Albanians descend from Roman era western Balkan populations, with additional admixture from Slavic-related groups. Remarkably, Albanian paternal ancestry shows continuity from Bronze Age Balkan populations, including those known as Illyrians. Our results provide an unprecedented understanding of the historical and demographic processes that led to the formation of modern Albanians and help locate the area where the Albanian language developed.\nThe human Y and inactive X chromosomes similarly modulate autosomal gene expression\nAuthors: San Roman, A. K.; Skaletsky, H.; Godfrey, A. K.; Bokil, N. V.; Teitz, L.; Singh, I.; Blanton, L. V.; Bellott, D. W.; Pyntikova, T.; Lange, J.; Koutseva, N.; Hughes, J. F.; Brown, L.; Phou, S.; Buscetta, A.; Kruszka, P.; Banks, N.; Dutra, A.; Pak, E.; Lasutschinkow, P. C.; Keen, C.; Davis, S. M.; Lin, A. E.; Tartaglia, N. R.; Samango-Sprouse, C.; Muenke, M.; Page, D. C.\nScore: 16.9, Published: 2023-06-07 DOI: 10.1101/2023.06.05.543763\nSomatic cells of human males and females have 45 chromosomes in common, including the \"active\" X chromosome. In males the 46th chromosome is a Y; in females it is an \"inactive\" X (Xi). Through linear modeling of autosomal gene expression in cells from individuals with zero to three Xi and zero to four Y chromosomes, we found that Xi and Y impact autosomal expression broadly and with remarkably similar effects. Studying sex-chromosome structural anomalies, promoters of Xi- and Y-responsive genes, and CRISPR inhibition, we traced part of this shared effect to homologous transcription factors - ZFX and ZFY - encoded by Chr X and Y. This demonstrates sex-shared mechanisms by which Xi and Y modulate autosomal expression. Combined with earlier analyses of sex-linked gene expression, our studies show that 21% of all genes expressed in lymphoblastoid cells or fibroblasts change expression significantly in response to Xi or Y chromosomes.\nInteraction molecular QTL mapping discovers cellular and environmental modifiers of genetic regulatory effects\nAuthors: Kasela, S.; Aguet, F.; Kim-Hellmuth, S.; Brown, B. C.; Nachun, D. C.; Tracy, R. P.; Durda, P.; Liu, Y.; Taylor, K. D.; Johnson, W. C.; Van Den Berg, D.; Gabriel, S.; Gupta, N.; Smith, J. D.; Blackwell, T. W.; Rotter, J. I.; Ardlie, K. G.; Manichaikul, A.; Rich, S. E.; Barr, R. G.; Lappalainen, T.\nScore: 15.2, Published: 2023-06-27 DOI: 10.1101/2023.06.26.546528\nBulk tissue molecular quantitative trait loci (QTLs) have been the starting point for interpreting disease-associated variants, while context-specific QTLs show particular relevance for disease. Here, we present the results of mapping interaction QTLs (iQTLs) for cell type, age, and other phenotypic variables in multi-omic, longitudinal data from blood of individuals of diverse ancestries. By modeling the interaction between genotype and estimated cell type proportions, we demonstrate that cell type iQTLs could be considered as proxies for cell type-specific QTL effects. The interpretation of age iQTLs, however, warrants caution as the moderation effect of age on the genotype and molecular phenotype association may be mediated by changes in cell type composition. Finally, we show that cell type iQTLs contribute to cell type-specific enrichment of diseases that, in combination with additional functional data, may guide future functional studies. Overall, this study highlights iQTLs to gain insights into the context-specificity of regulatory effects.\nGenetic population structure across Brittany and the downstream Loire basin provides new insights on the demographic history of Western Europe\nAuthors: Alves, I.; Giemza, J.; Blum, M.; Bernhardsson, C.; Chatel, S.; Karakachoff, M.; Saint Pierre, A.; Herzig, A. F.; Olaso, R.; Monteil, M.; Gallien, V.; Cabot, E.; Svensson, E.; Bacq Daian, D.; Baron, E.; Berthellier, C.; Besse, C.; Blanche, H.; Bocher, O.; Boland, A.; Bonnaud, S.; Charpentier, E.; Dandine-Roulland, C.; Ferec, C.; Fruchet, C.; Lecointe, S.; Le Floch, E.; Ludwig, T. E.; Marenne, G.; Meyer, V.; Quellery, E.; Racimo, F.; Rouault, K.; Sandron, F.; Schott, J.-J.; Velo Suarez, L.; Violleau, J.; Willerslev, E.; Coativy, Y.; Jezequel, M.; Le Bris, D.; Nicolas, C.; Pailler, Y.; Goldberg, Score: 41.2, Published: 2023-06-25 DOI: 10.1101/2022.02.03.478491\nEuropean genetic ancestry originates from three main ancestral populations - Western hunter-gatherers, early European farmers and Yamnaya Eurasian herders - whose edges geographically met in present-day France. Despite its central role to our understanding of how the ancestral populations interacted and gave rise to modern population structure, the population history of France has remained largely understudied. Here, we analysed the high-coverage whole-genome sequences and genome-wide genotype profiles of respectively 856 and 3,234 present-day individuals from the northern half of France, and merged them with publicly available present-day and ancient Europe-wide genotype datasets. We also explored, for the first time, the whole-genome sequences of six mediaeval individuals (300-1100 CE) from Western France to gain insights into the genetic impact of what is commonly known as the Migration Period in Europe. We found extensive fine-scale population structure across Brittany and the downstream Loire basin, emphasising the need for investigating local populations to better understand the distribution of rare and putatively deleterious variants across space. Overall, we observed an increased population differentiation between the northern and southern sides of the river Loire, which are characterised by different proportions of steppe vs. Neolithic-related ancestry. Samples from Western Brittany carry the largest levels of steppe ancestry and show high levels of allele sharing with individuals associated with the Bell Beaker complex, levels that are only comparable with those found in populations lying on the northwestern edges of Europe. Together, our results imply that present-day individuals from Western Brittany retain substantial legacy of the genetic changes that occurred in Northwestern Europe following the arrival of the Bell Beaker people c. 2500 BCE. Such genetic legacy may explain the sharing of disease-related alleles with other present-day populations from Western Britain and Ireland.\nA whole organism map of X-inactivation reveals tissue-specific escape as an exceptionally rare event in humans\nAuthors: Gylemo, B.; Nestor, C. E.\nScore: 9.6, Published: 2023-06-27 DOI: 10.1101/2023.06.26.546519\nAs females are mosaic for X-inactivation, direct determination of X-linked allelic expression in bulk tissues is typically unfeasible. Using females that are non-mosaic for X-inactivation (nmXCI) as a powerful and natural genetic system we directly determined X-inactivation status of 401 X-linked genes across 32 normal tissues. We reveal that nmXCI is common in the female population and that contrary to previous reports, variable escape is an exceedingly rare event in humans. As the most complete atlas of XCI in humans to date, our findings represent a major advance in our understanding of human X-inactivation and will serve as a reference for dissecting the genetic origin of sex-bias in human traits.\nTwo independent DNA repair pathways cause mutagenesis in template switching deficient Saccharomyces cerevisiae\nAuthors: Jiang, Y. K.; Medley, E. A.; Brown, G. W.\nScore: 7.4, Published: 2023-06-26 DOI: 10.1101/2023.06.25.546467\nUpon DNA replication stress, cells utilize the post-replication repair pathway to repair single-stranded DNA and maintain genome integrity. Post-replication repair is divided into two branches: error-prone translesion synthesis, signaled by PCNA mono-ubiquitination, and error-free template switching, signaled by PCNA poly-ubiquitination. In Saccharomyces cerevisiae, Rad5 is involved in both branches of repair during DNA replication stress. When the PCNA poly-ubiquitination function of Rad5 is disrupted, Rad5 recruits translesion synthesis polymerases to stalled replication forks, resulting in mutagenic repair. Details of how mutagenic repair is carried out, as well as the relationship between Rad5-mediated mutagenic repair and the canonical PCNA-mediated mutagenic repair, remain to be understood. We find that Rad5-mediated mutagenic repair requires the translesion synthesis polymerase {zeta} but does not require other yeast translesion polymerase activities. Furthermore, we show that Rad5-mediated mutagenic repair is independent of PCNA binding by Rev1 and so is separable from canonical mutagenic repair. In the absence of error-free template switching, both modes of mutagenic repair contribute additively to replication stress response in a replication timing-independent manner. Cellular contexts where error-free template switching is compromised are not simply laboratory phenomena, as we find that a natural variant in RAD5 is defective in PCNA poly-ubiquitination and therefore defective in error-free repair, resulting in Rad5- and PCNA-mediated mutagenic repair. Our results highlight the importance of Rad5 in regulating spontaneous mutagenesis and genetic diversity in S. cerevisiae through different modes of post-replication repair.\nGenome reduction occurred in early Prochlorococcus with an unusually low effective population size\nAuthors: Zhang, H.; Hellweger, F. L.; Luo, H.\nScore: 5.8, Published: 2023-06-26 DOI: 10.1101/2023.06.25.546417\nIn the oligotrophic sunlit ocean, the most abundant free-living planktonic bacterial lineages evolve convergently through genome reduction. The cyanobacterium Prochlorococcus responsible for 10% global oxygen production is a prominent example. The dominant theory known as 'genome streamlining' posits that they have extremely large effective population sizes (Ne) such that selection for metabolic efficiency acts to drive genome reduction. Because genome reduction largely took place anciently, this theory builds on the assumption that their ancestors' Ne was similarly large. Constraining Ne for ancient ancestors is challenging because experimental measurements of extinct organisms are impossible and alternatively reconstructing ancestral Ne with phylogenetic models gives large uncertainties. Here, we develop a new strategy that leverages agent-based modeling to simulate the change of Ne proxy for ancient ancestors, the genome-wide ratio of radical to conservative nonsynonymous nucleotide substitution rate (dR/dC), in response to the change of Ne. Surprisingly, this proxy shows expected increases with decreases of Ne only when Ne falls to about 10k - 100k or lower, magnitudes characteristic of Ne of obligate endosymbiont species where drift drives genome reduction. We therefore conclude that drift, rather than selection, is the primary force that drove Prochlorococcus genome reduction.\nThe haplolethal gene wupA of Drosophila exhibits potential as a target for an X-poisoning gene drive\nAuthors: Lawler, C. D.; Hernandes, N.; Parra Nunez, A. K.; Bhide, S.; Baxter, S.; Robin, C.\nScore: 2.9, Published: 2023-06-26 DOI: 10.1101/2023.06.23.546292\nA synthetic gene drive that targets haplolethal genes on the X-chromosome can skew the sex ratio towards males. Like an X-shredder strategy it does not involve homing and that has advantages including the reduction in gene drive resistance allele formation. We examine this X-poisoning strategy by targeting four of the 11 known X-linked haplolethal/haplosterile genes of Drosophila melanogaster with CRISPR/cas9. We find that targeting the wupA gene during spermatogenesis skews the sex ratio so fewer than 14% of progeny are daughters. That is unless we cross the mutagenic males to X^XY female flies that bear attached-X chromosomes, which reverses the inheritance of the poisoned X chromosome so that sons inherit it from their mother; in which case only 2% of the progeny are sons. These sex ratio biases suggests that most of the CRISPR/cas9 mutants we induced in the wupA gene are haplolethal but some are recessive lethal. We have used simulations to model release strategies of a gene drive where an X-poisoning Y chromosome is introduced to invasive populations of the pest Drosophila suzukii. While the reproductive output of carriers of X-poisoning drives is less than that of X-shredding drives due to their post-zygotic mode of action, the threshold dynamics of an X-poisoning drive result in more predictable outcomes than X-shredders that tend to exhibit chasing more often. Like D. melanogaster and D. suzukii, the wupA orthologs in Anopheles mosquitos are found on X chromosomes making it a viable target in multiple species.\n",
  "wordCount" : "2618",
  "inLanguage": "en",
  "datePublished": "2023-06-28T11:45:43Z",
  "dateModified": "2023-06-28T11:45:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545749">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545749" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545749">
        <p class="paperTitle">Nucleus-Independent Transgenerational Small RNA Inheritance in C. elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545749" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545749" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rieger, I.; Weintraub, G.; Lev, I.; Goldstein, K.; Bar-Zvi, D.; Anava, S.; Gingold, H.; Shaham, S.; Rechavi, O.</p>
        <p class="info">Score: 155.9, Published: 2023-06-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545749' target='https://doi.org/10.1101/2023.06.20.545749'> 10.1101/2023.06.20.545749</a></p>
        <p class="abstract">Studies using C. elegans nematodes demonstrated that, against the dogma, animals can transmit epigenetic information transgenerationally. While it is now clear that in these worms ancestral RNA interference (RNAi) responses continue to regulate gene expression for many generations, it is still debated whether the primary agent that perpetuates heritable silencing is RNA or chromatin, and whether the information is communicated to the next generation inside or outside of the nucleus. Here we take advantage of the tractability of gene-specific double stranded RNA-induced heritable silencing to answer these questions. We demonstrate that RNAi can be inherited independently of any changes to the chromatin or any other nuclear factors via mothers that are genetically engineered to transmit only their ooplasm but not the oocytes nuclei to the next generation. Nucleus-independent RNA inheritance depends on ZNFX-1, an RNA-binding germ granule resident protein. We find that upon manipulation of normal germ granules functions (in pptr-1 mutants) nucleus-independent RNA inheritance becomes stronger, and can occur even in znfx-1 mutants. Utilizing RNA sequencing, chimeric worms, and sequence polymorphism between different C. elegans isolates, we identify endogenous small RNAs which, similarly to exogenous siRNAs, are inherited in a nucleus-independent manner. From an historical perspective, nucleus-independent inheritance of small RNAs might be regarded as partial vindication of discredited cytoplasmic inheritance theories from the 19th century, such as Darwins &#34;pangenesis&#34; theory.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.02.542661">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.02.542661" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.02.542661">
        <p class="paperTitle">Fetally-encoded GDF15 and maternal GDF15 sensitivity are major determinants of nausea and vomiting in human pregnancy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.02.542661" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.02.542661" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fejzo, M.; Rocha, N.; Cimino, I.; Lockhart, S. M.; Petry, C.; Kay, R. G.; Burling, K.; Barker, P.; George, A. L.; Yasara, N.; Premawardhena, A.; Gong, S.; Cook, E.; Rainbow, K.; Withers, D.; Cortessis, V.; Mullin, P.; MacGibbon, K.; Jin, E.; Kam, A.; Campbell, A.; Polasek, O.; Tzoneva, G.; Gribble, F.; Yeo, G.; Lam, B.; Saudek, V.; Hughes, I.; Ong, K.; Perry, J.; Sutton-Cole, A.; Baumgarten, M.; Welsh, P.; Sattar, N.; Smith, G. C.; Charnock-Jones, D. S.; Coll, A. P.; Meek, C. L.; Mettananda, S.; Hayward, C.; Mancuso, N.; O&#39;Rahilly, S.</p>
        <p class="info">Score: 211.2, Published: 2023-06-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.02.542661' target='https://doi.org/10.1101/2023.06.02.542661'> 10.1101/2023.06.02.542661</a></p>
        <p class="abstract">Human pregnancy is frequently accompanied by nausea and vomiting that may become severe and life-threatening, as in hyperemesis gravidarum (HG), the cause of which is unknown. Growth Differentiation Factor-15 (GDF15), a hormone known to act on the hindbrain to cause emesis, is highly expressed in the placenta and its levels in maternal blood rise rapidly in pregnancy. Variants in the maternal GDF15 gene are associated with HG. Here we report that fetal production of GDF15, and maternal sensitivity to it, both contribute substantially to the risk of HG. We found that the great majority of GDF15 in maternal circulation is derived from the feto-placental unit and that higher GDF15 levels in maternal blood are associated with vomiting and are further elevated in patients with HG. Conversely, we found that lower levels of GDF15 in the non-pregnant state predispose women to HG. A rare C211G variant in GDF15 which strongly predisposes mothers to HG, particularly when the fetus is wild-type, was found to markedly impair cellular secretion of GDF15 and associate with low circulating levels of GDF15 in the non-pregnant state. Consistent with this, two common GDF15 haplotypes which predispose to HG were associated with lower circulating levels outside pregnancy. The administration of a long-acting form of GDF15 to wild-type mice markedly reduced subsequent responses to an acute dose, establishing that desensitisation is a feature of this system. GDF15 levels are known to be highly and chronically elevated in patients with beta thalassemia. In women with this disorder, reports of symptoms of nausea or vomiting in pregnancy were strikingly diminished. Our findings support a causal role for fetal derived GDF15 in the nausea and vomiting of human pregnancy, with maternal sensitivity, at least partly determined by pre-pregnancy exposure to GDF15, being a major influence on its severity. They also suggest mechanism-based approaches to the treatment and prevention of HG.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.05.543790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.05.543790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.05.543790">
        <p class="paperTitle">Ancient DNA reveals the origins of the Albanians</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.05.543790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.05.543790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davranoglou, L. R.; Aristodemou, A.; Wesolowski, D.; Heraclides, A.</p>
        <p class="info">Score: 189.9, Published: 2023-06-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.05.543790' target='https://doi.org/10.1101/2023.06.05.543790'> 10.1101/2023.06.05.543790</a></p>
        <p class="abstract">The origins of the Albanian people have vexed linguists and historians for centuries, as Albanians first appear in the historical record in the 11th century CE, while their language is one of the most enigmatic branches of the Indo-European family. To identify the populations that contributed to the ancestry of Albanians, we undertake a genomic transect of the Balkans over the last 8000 years, where we analyse more than 6000 previously published ancient genomes using state-of-the-art bioinformatics tools and algorithms that quantify spatiotemporal human mobility. We find that modern Albanians descend from Roman era western Balkan populations, with additional admixture from Slavic-related groups. Remarkably, Albanian paternal ancestry shows continuity from Bronze Age Balkan populations, including those known as Illyrians. Our results provide an unprecedented understanding of the historical and demographic processes that led to the formation of modern Albanians and help locate the area where the Albanian language developed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.05.543763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.05.543763" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.05.543763">
        <p class="paperTitle">The human Y and inactive X chromosomes similarly modulate autosomal gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.05.543763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.05.543763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: San Roman, A. K.; Skaletsky, H.; Godfrey, A. K.; Bokil, N. V.; Teitz, L.; Singh, I.; Blanton, L. V.; Bellott, D. W.; Pyntikova, T.; Lange, J.; Koutseva, N.; Hughes, J. F.; Brown, L.; Phou, S.; Buscetta, A.; Kruszka, P.; Banks, N.; Dutra, A.; Pak, E.; Lasutschinkow, P. C.; Keen, C.; Davis, S. M.; Lin, A. E.; Tartaglia, N. R.; Samango-Sprouse, C.; Muenke, M.; Page, D. C.</p>
        <p class="info">Score: 16.9, Published: 2023-06-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.05.543763' target='https://doi.org/10.1101/2023.06.05.543763'> 10.1101/2023.06.05.543763</a></p>
        <p class="abstract">Somatic cells of human males and females have 45 chromosomes in common, including the &#34;active&#34; X chromosome. In males the 46th chromosome is a Y; in females it is an &#34;inactive&#34; X (Xi). Through linear modeling of autosomal gene expression in cells from individuals with zero to three Xi and zero to four Y chromosomes, we found that Xi and Y impact autosomal expression broadly and with remarkably similar effects. Studying sex-chromosome structural anomalies, promoters of Xi- and Y-responsive genes, and CRISPR inhibition, we traced part of this shared effect to homologous transcription factors - ZFX and ZFY - encoded by Chr X and Y. This demonstrates sex-shared mechanisms by which Xi and Y modulate autosomal expression. Combined with earlier analyses of sex-linked gene expression, our studies show that 21% of all genes expressed in lymphoblastoid cells or fibroblasts change expression significantly in response to Xi or Y chromosomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546528">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546528" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546528">
        <p class="paperTitle">Interaction molecular QTL mapping discovers cellular and environmental modifiers of genetic regulatory effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546528" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546528" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kasela, S.; Aguet, F.; Kim-Hellmuth, S.; Brown, B. C.; Nachun, D. C.; Tracy, R. P.; Durda, P.; Liu, Y.; Taylor, K. D.; Johnson, W. C.; Van Den Berg, D.; Gabriel, S.; Gupta, N.; Smith, J. D.; Blackwell, T. W.; Rotter, J. I.; Ardlie, K. G.; Manichaikul, A.; Rich, S. E.; Barr, R. G.; Lappalainen, T.</p>
        <p class="info">Score: 15.2, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546528' target='https://doi.org/10.1101/2023.06.26.546528'> 10.1101/2023.06.26.546528</a></p>
        <p class="abstract">Bulk tissue molecular quantitative trait loci (QTLs) have been the starting point for interpreting disease-associated variants, while context-specific QTLs show particular relevance for disease. Here, we present the results of mapping interaction QTLs (iQTLs) for cell type, age, and other phenotypic variables in multi-omic, longitudinal data from blood of individuals of diverse ancestries. By modeling the interaction between genotype and estimated cell type proportions, we demonstrate that cell type iQTLs could be considered as proxies for cell type-specific QTL effects. The interpretation of age iQTLs, however, warrants caution as the moderation effect of age on the genotype and molecular phenotype association may be mediated by changes in cell type composition. Finally, we show that cell type iQTLs contribute to cell type-specific enrichment of diseases that, in combination with additional functional data, may guide future functional studies. Overall, this study highlights iQTLs to gain insights into the context-specificity of regulatory effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.02.03.478491">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.02.03.478491" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.02.03.478491">
        <p class="paperTitle">Genetic population structure across Brittany and the downstream Loire basin provides new insights on the demographic history of Western Europe</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.02.03.478491" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.02.03.478491" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alves, I.; Giemza, J.; Blum, M.; Bernhardsson, C.; Chatel, S.; Karakachoff, M.; Saint Pierre, A.; Herzig, A. F.; Olaso, R.; Monteil, M.; Gallien, V.; Cabot, E.; Svensson, E.; Bacq Daian, D.; Baron, E.; Berthellier, C.; Besse, C.; Blanche, H.; Bocher, O.; Boland, A.; Bonnaud, S.; Charpentier, E.; Dandine-Roulland, C.; Ferec, C.; Fruchet, C.; Lecointe, S.; Le Floch, E.; Ludwig, T. E.; Marenne, G.; Meyer, V.; Quellery, E.; Racimo, F.; Rouault, K.; Sandron, F.; Schott, J.-J.; Velo Suarez, L.; Violleau, J.; Willerslev, E.; Coativy, Y.; Jezequel, M.; Le Bris, D.; Nicolas, C.; Pailler, Y.; Goldberg, </p>
        <p class="info">Score: 41.2, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.02.03.478491' target='https://doi.org/10.1101/2022.02.03.478491'> 10.1101/2022.02.03.478491</a></p>
        <p class="abstract">European genetic ancestry originates from three main ancestral populations - Western hunter-gatherers, early European farmers and Yamnaya Eurasian herders - whose edges geographically met in present-day France. Despite its central role to our understanding of how the ancestral populations interacted and gave rise to modern population structure, the population history of France has remained largely understudied. Here, we analysed the high-coverage whole-genome sequences and genome-wide genotype profiles of respectively 856 and 3,234 present-day individuals from the northern half of France, and merged them with publicly available present-day and ancient Europe-wide genotype datasets. We also explored, for the first time, the whole-genome sequences of six mediaeval individuals (300-1100 CE) from Western France to gain insights into the genetic impact of what is commonly known as the Migration Period in Europe. We found extensive fine-scale population structure across Brittany and the downstream Loire basin, emphasising the need for investigating local populations to better understand the distribution of rare and putatively deleterious variants across space. Overall, we observed an increased population differentiation between the northern and southern sides of the river Loire, which are characterised by different proportions of steppe vs. Neolithic-related ancestry. Samples from Western Brittany carry the largest levels of steppe ancestry and show high levels of allele sharing with individuals associated with the Bell Beaker complex, levels that are only comparable with those found in populations lying on the northwestern edges of Europe. Together, our results imply that present-day individuals from Western Brittany retain substantial legacy of the genetic changes that occurred in Northwestern Europe following the arrival of the Bell Beaker people c. 2500 BCE. Such genetic legacy may explain the sharing of disease-related alleles with other present-day populations from Western Britain and Ireland.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546519" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546519">
        <p class="paperTitle">A whole organism map of X-inactivation reveals tissue-specific escape as an exceptionally rare event in humans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gylemo, B.; Nestor, C. E.</p>
        <p class="info">Score: 9.6, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546519' target='https://doi.org/10.1101/2023.06.26.546519'> 10.1101/2023.06.26.546519</a></p>
        <p class="abstract">As females are mosaic for X-inactivation, direct determination of X-linked allelic expression in bulk tissues is typically unfeasible. Using females that are non-mosaic for X-inactivation (nmXCI) as a powerful and natural genetic system we directly determined X-inactivation status of 401 X-linked genes across 32 normal tissues. We reveal that nmXCI is common in the female population and that contrary to previous reports, variable escape is an exceedingly rare event in humans. As the most complete atlas of XCI in humans to date, our findings represent a major advance in our understanding of human X-inactivation and will serve as a reference for dissecting the genetic origin of sex-bias in human traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.25.546467">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.25.546467" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.25.546467">
        <p class="paperTitle">Two independent DNA repair pathways cause mutagenesis in template switching deficient Saccharomyces cerevisiae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.25.546467" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.25.546467" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, Y. K.; Medley, E. A.; Brown, G. W.</p>
        <p class="info">Score: 7.4, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.25.546467' target='https://doi.org/10.1101/2023.06.25.546467'> 10.1101/2023.06.25.546467</a></p>
        <p class="abstract">Upon DNA replication stress, cells utilize the post-replication repair pathway to repair single-stranded DNA and maintain genome integrity. Post-replication repair is divided into two branches: error-prone translesion synthesis, signaled by PCNA mono-ubiquitination, and error-free template switching, signaled by PCNA poly-ubiquitination. In Saccharomyces cerevisiae, Rad5 is involved in both branches of repair during DNA replication stress. When the PCNA poly-ubiquitination function of Rad5 is disrupted, Rad5 recruits translesion synthesis polymerases to stalled replication forks, resulting in mutagenic repair. Details of how mutagenic repair is carried out, as well as the relationship between Rad5-mediated mutagenic repair and the canonical PCNA-mediated mutagenic repair, remain to be understood. We find that Rad5-mediated mutagenic repair requires the translesion synthesis polymerase {zeta} but does not require other yeast translesion polymerase activities. Furthermore, we show that Rad5-mediated mutagenic repair is independent of PCNA binding by Rev1 and so is separable from canonical mutagenic repair. In the absence of error-free template switching, both modes of mutagenic repair contribute additively to replication stress response in a replication timing-independent manner. Cellular contexts where error-free template switching is compromised are not simply laboratory phenomena, as we find that a natural variant in RAD5 is defective in PCNA poly-ubiquitination and therefore defective in error-free repair, resulting in Rad5- and PCNA-mediated mutagenic repair. Our results highlight the importance of Rad5 in regulating spontaneous mutagenesis and genetic diversity in S. cerevisiae through different modes of post-replication repair.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.25.546417">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.25.546417" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.25.546417">
        <p class="paperTitle">Genome reduction occurred in early Prochlorococcus with an unusually low effective population size</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.25.546417" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.25.546417" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, H.; Hellweger, F. L.; Luo, H.</p>
        <p class="info">Score: 5.8, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.25.546417' target='https://doi.org/10.1101/2023.06.25.546417'> 10.1101/2023.06.25.546417</a></p>
        <p class="abstract">In the oligotrophic sunlit ocean, the most abundant free-living planktonic bacterial lineages evolve convergently through genome reduction. The cyanobacterium Prochlorococcus responsible for 10% global oxygen production is a prominent example. The dominant theory known as &#39;genome streamlining&#39; posits that they have extremely large effective population sizes (Ne) such that selection for metabolic efficiency acts to drive genome reduction. Because genome reduction largely took place anciently, this theory builds on the assumption that their ancestors&#39; Ne was similarly large. Constraining Ne for ancient ancestors is challenging because experimental measurements of extinct organisms are impossible and alternatively reconstructing ancestral Ne with phylogenetic models gives large uncertainties. Here, we develop a new strategy that leverages agent-based modeling to simulate the change of Ne proxy for ancient ancestors, the genome-wide ratio of radical to conservative nonsynonymous nucleotide substitution rate (dR/dC), in response to the change of Ne. Surprisingly, this proxy shows expected increases with decreases of Ne only when Ne falls to about 10k - 100k or lower, magnitudes characteristic of Ne of obligate endosymbiont species where drift drives genome reduction. We therefore conclude that drift, rather than selection, is the primary force that drove Prochlorococcus genome reduction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546292">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546292" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546292">
        <p class="paperTitle">The haplolethal gene wupA of Drosophila exhibits potential as a target for an X-poisoning gene drive</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546292" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546292" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lawler, C. D.; Hernandes, N.; Parra Nunez, A. K.; Bhide, S.; Baxter, S.; Robin, C.</p>
        <p class="info">Score: 2.9, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546292' target='https://doi.org/10.1101/2023.06.23.546292'> 10.1101/2023.06.23.546292</a></p>
        <p class="abstract">A synthetic gene drive that targets haplolethal genes on the X-chromosome can skew the sex ratio towards males. Like an X-shredder strategy it does not involve homing and that has advantages including the reduction in gene drive resistance allele formation. We examine this X-poisoning strategy by targeting four of the 11 known X-linked haplolethal/haplosterile genes of Drosophila melanogaster with CRISPR/cas9. We find that targeting the wupA gene during spermatogenesis skews the sex ratio so fewer than 14% of progeny are daughters. That is unless we cross the mutagenic males to X^XY female flies that bear attached-X chromosomes, which reverses the inheritance of the poisoned X chromosome so that sons inherit it from their mother; in which case only 2% of the progeny are sons. These sex ratio biases suggests that most of the CRISPR/cas9 mutants we induced in the wupA gene are haplolethal but some are recessive lethal. We have used simulations to model release strategies of a gene drive where an X-poisoning Y chromosome is introduced to invasive populations of the pest Drosophila suzukii. While the reproductive output of carriers of X-poisoning drives is less than that of X-shredding drives due to their post-zygotic mode of action, the threshold dynamics of an X-poisoning drive result in more predictable outcomes than X-shredders that tend to exhibit chasing more often. Like D. melanogaster and D. suzukii, the wupA orthologs in Anopheles mosquitos are found on X chromosomes making it a viable target in multiple species.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
