<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Rewriting regulatory DNA to dissect and reprogram gene expression
Authors: Martyn, G. E.; Montgomery, M. T.; Jones, H.; Guo, K.; Doughty, B. R.; Linder, J.; Chen, Z.; Cochran, K.; Lawrence, K. A.; Munson, G.; Pampari, A.; Fulco, C. P.; Kelley, D. R.; Lander, E. S.; Kundaje, A.; Engreitz, J. M.
Score: 50.7, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572268
Regulatory DNA sequences within enhancers and promoters bind transcription factors to encode cell type-specific patterns of gene expression.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Rewriting regulatory DNA to dissect and reprogram gene expression
Authors: Martyn, G. E.; Montgomery, M. T.; Jones, H.; Guo, K.; Doughty, B. R.; Linder, J.; Chen, Z.; Cochran, K.; Lawrence, K. A.; Munson, G.; Pampari, A.; Fulco, C. P.; Kelley, D. R.; Lander, E. S.; Kundaje, A.; Engreitz, J. M.
Score: 50.7, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572268
Regulatory DNA sequences within enhancers and promoters bind transcription factors to encode cell type-specific patterns of gene expression." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Rewriting regulatory DNA to dissect and reprogram gene expression
Authors: Martyn, G. E.; Montgomery, M. T.; Jones, H.; Guo, K.; Doughty, B. R.; Linder, J.; Chen, Z.; Cochran, K.; Lawrence, K. A.; Munson, G.; Pampari, A.; Fulco, C. P.; Kelley, D. R.; Lander, E. S.; Kundaje, A.; Engreitz, J. M.
Score: 50.7, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572268
Regulatory DNA sequences within enhancers and promoters bind transcription factors to encode cell type-specific patterns of gene expression."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Rewriting regulatory DNA to dissect and reprogram gene expression\nAuthors: Martyn, G. E.; Montgomery, M. T.; Jones, H.; Guo, K.; Doughty, B. R.; Linder, J.; Chen, Z.; Cochran, K.; Lawrence, K. A.; Munson, G.; Pampari, A.; Fulco, C. P.; Kelley, D. R.; Lander, E. S.; Kundaje, A.; Engreitz, J. M.\nScore: 50.7, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572268\nRegulatory DNA sequences within enhancers and promoters bind transcription factors to encode cell type-specific patterns of gene expression.",
  "keywords": [
    
  ],
  "articleBody": " Rewriting regulatory DNA to dissect and reprogram gene expression\nAuthors: Martyn, G. E.; Montgomery, M. T.; Jones, H.; Guo, K.; Doughty, B. R.; Linder, J.; Chen, Z.; Cochran, K.; Lawrence, K. A.; Munson, G.; Pampari, A.; Fulco, C. P.; Kelley, D. R.; Lander, E. S.; Kundaje, A.; Engreitz, J. M.\nScore: 50.7, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572268\nRegulatory DNA sequences within enhancers and promoters bind transcription factors to encode cell type-specific patterns of gene expression. However, the regulatory effects and programmability of such DNA sequences remain difficult to map or predict because we have lacked scalable methods to precisely edit regulatory DNA and quantify the effects in an endogenous genomic context. Here we present an approach to measure the quantitative effects of hundreds of designed DNA sequence variants on gene expression, by combining pooled CRISPR prime editing with RNA fluorescence in situ hybridization and cell sorting (Variant-FlowFISH). We apply this method to mutagenize and rewrite regulatory DNA sequences in an enhancer and the promoter of PPIF in two immune cell lines. Of 672 variant-cell type pairs, we identify 497 that affect PPIF expression. These variants appear to act through a variety of mechanisms including disruption or optimization of existing transcription factor binding sites, as well as creation of de novo sites. Disrupting a single endogenous transcription factor binding site often led to large changes in expression (up to -40% in the enhancer, and -50% in the promoter). The same variant often had different effects across cell types and states, demonstrating a highly tunable regulatory landscape. We use these data to benchmark performance of sequence-based predictive models of gene regulation, and find that certain types of variants are not accurately predicted by existing models. Finally, we computationally design 185 small sequence variants ([\u0026le;]10 bp) and optimize them for specific effects on expression in silico. 84% of these rationally designed edits showed the intended direction of effect, and some had dramatic effects on expression (-100% to +202%). Variant-FlowFISH thus provides a powerful tool to map the effects of variants and transcription factor binding sites on gene expression, test and improve computational models of gene regulation, and reprogram regulatory DNA.\nBetween-group heritability and the status of hereditarianism as an evolutionary science\nAuthors: Roseman, C. C.; Bird, K.\nScore: 26.9, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572247\nHereditarianism is a school of thought that contends there are substantial evolved cognitive and behavioral differences among groups of humans which are both resistant to environmental intervention and are a root cause of differential social outcomes across groups. The relationship of betweengroup heritability [Formula] to within-group heritability [Formula] is one of the key theoretical components of hereditarianism and forms one of the bases for its claim to be an evolutionary science. Here, we examine the relationship between [Formula]and [Formula] and its application to problems in the hereditarian literature from an evolutionary genetic perspective. We demonstrate that the formulation of the relationship between [Formula] and [Formula]used in the hereditarian literature has no evolutionary content. By re-writing the relationship between [Formula] and[Formula] in a novel evolutionary framework, we demonstrate that there is no way to predict[Formula] using[Formula] without considerable additional theory that is absent from the hereditarian literature. Furthermore, we demonstrate that the hereditarian technique that uses[Formula] and [Formula] as a means of judging whether a given difference between groups may be plausibly ameliorated through environmental intervention is mathematically flawed. Lastly, we fill a gap in the hereditarian literature by writing out a means of using [Formula] to predict the absolute difference between groups under a neutral evolutionary model and find that it is much smaller than claimed by hereditarians. In conclusion, we propose a path forward for the study of human variation that moves us past the ill-conceived nature vs. nurture question and allows us to focus on more productive issues.\nThe germline regulates longevity and somatic repair in a sex-specific manner\nAuthors: Moses, E.; Atlan, T.; Sun, X.; Franek, R.; Siddiqui, A.; Marinov, G. K.; Shifman, S.; Zucker, D. M.; Oron-Gottesman, A.; Greenleaf, W. J.; Cohen, E.; Ram, O.; Harel, I.\nScore: 24.7, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572041\nClassical evolutionary theories propose tradeoffs between reproduction, damage repair, and lifespan. However, the specific role of the germline in shaping vertebrate aging remains largely unknown. Here, we use the turquoise killifish (N. furzeri) to genetically arrest germline differentiation at discrete stages, and examine how different flavors of infertility impact life-history. We first constructed a comprehensive single-cell gonadal atlas, providing cell-type-specific markers for downstream phenotypic analysis. Next, investigating our genetic models revealed that only germline depletion enhanced female damage repair, while arresting germline differentiation did not. Conversely, germline-depleted males were significantly long-lived, indicating that the mere presence of the germline can negatively affect lifespan. Transcriptomic analysis highlighted enrichment of pro-longevity pathways and genes, with functional conservation in germline-depleted C. elegans. Finally, germline depletion extended male healthspan through rejuvenated metabolic functions. Our results suggest that different germline manipulation paradigms can yield pronounced sexually dimorphic phenotypes, implying alternative mechanisms to classical evolutionary tradeoffs.\nKNL1 and NDC80 represent new universal markers for the detection of functional centromeres in plants\nAuthors: Oliveira, L.; Neumann, P.; Mata-Sucre, Y.; Kuo, Y.-T.; Marques, A.; Schubert, V.; Macas, J.\nScore: 14.4, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572763\nCentromere is the chromosomal site of kinetochore assembly and microtubule attachment for chromosome segregation. Given its importance, markers that allow specific labeling of centromeric chromatin throughout the cell cycle and across all chromosome types are sought for facilitating various centromere studies. Antibodies against the N-terminal region of CENH3 are commonly used for this purpose, since CENH3 is the near-universal marker of functional centromeres. However, because the N-terminal region of CENH3 is highly variable among plant species, antibodies directed against this region usually function only in a small group of closely related species. As a more versatile alternative, we present here antibodies targeted to the conserved domains of two outer kinetochore proteins, KNL1 and NDC80. Sequence comparison of these domains across more than 350 plant species revealed a high degree of conservation, particularly within a six amino acid motif, FFGPVS in KNL1, suggesting that both antibodies would function in a wide range of plant species. This assumption was confirmed by immunolabeling experiments in angiosperm (monocot and dicot) and gymnosperm species, including those with mono-, holo-, and meta-polycentric chromosomes. In addition to centromere labeling on condensed chromosomes during cell division, both antibodies detected the corresponding regions in the interphase nuclei of most species tested. These results demonstrated that KNL1 and NDC80 are better suited for immunolabeling centromeres than CENH3, because antibodies against these proteins offer incomparably greater versatility across different plant species which is particularly convenient for studying the organization and function of the centromere in non-model species.\nGenetic and environmental determinants of multicellular-like phenotypes in fission yeast\nAuthors: Köver, B.; Cohen, C. E.; Ralser, M.; Heineike, B. M.; Bähler, J.\nScore: 12.7, Published: 2023-12-19 DOI: 10.1101/2023.12.15.571870\nMulticellular fungi have repeatedly given rise to primarily unicellular yeast species. Some of these, including Schizosaccharomyces pombe, are able to revert to multicellular-like phenotypes (MLP). Our bioinformatic analysis of existing data suggested that, besides some regulatory proteins, most proteins involved in MLP formation are not functionally conserved between S. pombe and budding yeast. We developed high-throughput assays for two types of MLP in S. pombe: flocculation and surface adhesion, which correlated in minimal medium, suggesting a common mechanism. Using a library of 57 natural S. pombe isolates, we found MLP formation to widely vary across different nutrient and drug conditions. Next, in a segregant S. pombe library generated from an adhesive natural isolate and the standard laboratory strain, MLP formation correlated with expression levels of the transcription-factor gene mbx2 and several flocculins. Quantitative trait locus mapping of MLP formation located a causal frameshift mutation in the srb11 gene encoding cyclin C, a part of the Cdk8 kinase module (CKM) of the Mediator complex. Other CKM deletions also resulted in MLP formation, consistently through upregulation of mbx2, and only in minimal media. We screened a library of 3721 gene-deletion strains, uncovering additional genes involved in surface adhesion on minimal media. We identified 31 high-confidence hits, including 19 genes that have not been associated with MLPs in fission or budding yeast. Notably, deletion of srb11, unlike deletions of the 31 hits, did not compromise cell growth, which might explain its natural occurrence as a QTL for MLP formation.\nModelling the age-related deceleration of clonal haematopoiesis in UK Biobank\nAuthors: MacGregor, H. A.; Easton, D.; Blundell, J. R.\nScore: 8.2, Published: 2023-12-21 DOI: 10.1101/2023.12.21.572706\nSomatic mutations acquired in haematopoietic stem cells can drive clonal expansions, a phenomenon called clonal haematopoiesis (CH). CH has been associated with higher risk of haematological cancer, cardiovascular disease and lower life expectancy and is strongly age-dependent, becoming almost ubiquitous in older adults. However, the dynamics of CH are not yet fully understood, and an accurate quantitative model of clonal expansion in healthy people could inform early detection strategies. Here, we analyse the age-dependence of the variant allele frequency (VAF) distributions of common CH driver variants from the exomes of 400,000 cancer-free UK Biobank participants. We find evidence of a substantial deceleration of clonal expansion with age taking place in younger people. This deceleration appears to be general: different CH hotspot variants all displayed a [~] 3-fold reduction in growth rate between young and old. We develop several alternative models of clonal dynamics that might explain this age-related deceleration. First we show that variation in fitness of variants among individuals (e.g., due to genetic susceptibility) is unable to produce the required deceleration. Second, we consider a clonal competition model in which clonal expansion in later life is inhibited by competition with an unobserved background of other expanding clones. Finally, we consider an ageing model where HSC division slows down with increasing age. The observed age-specific VAF distributions are consistent with the ageing model with plausible parameters. To further interrogate these models, we consider a large data set of longitudinal growth rates obtained from serial samples of blood. These data suggest that a general slowdown of HSC division rates plays a key role in shaping the genetic diversity of ageing blood.\nIntegrative genomics reveals the polygenic basis of seedlessness in grapevine\nAuthors: Wang, X.; Liu, Z.; Zhang, F.; Xiao, H.; Cao, S.; Xue, H.; Liu, W.; Su, Y.; Liu, Z.; Zhong, H.; Zhang, F.; Ahmad, B.; Long, Q.; Zhang, Y.; Liu, Y.; Gan, Y.; Hou, T.; Jin, Z.; Wu, X.; Wang, Y.; Peng, Y.; Zhou, Y.\nScore: 6.4, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573032\nSeedlessness is a crucial quality trait in table grape (Vitis vinifera L.) breeding. However, the development of seeds involved intricate regulations, while the polygenic basis of seed abortion remains unclear. Here, we combine comparative genomics, population genetics, quantitative genetics, and integrative genomics to unravel the evolution and polygenic basis of seedlessness in grapes. We generated four haplotype-resolved telomere-to-telomere (T2T) genomes for two seedless grape cultivars, Thompson Seedless (TS, syn. Sultania) and Black Monukka (BM). Comparative genomics identified a ~4.25 Mb hemizygous inversion on Chr10 specific in seedless cultivars, with seedless-associated genes VvTT16 and VvSUS2 located at breakpoints. Population genomic analyses of 548 grapevine accessions revealed two distinct clusters of seedless cultivars, tracing the origin of the seedlessness trait back to Sultania. Introgression, rather than convergent selection, shaped the evolutionary history of seedlessness in grape improvement. Genome-wide association study (GWAS) analysis identified 110 quantitative trait loci (QTLs) associated with 634 candidate genes, including novel candidate genes, such as three 11S GLOBULIN SEED STORAGE PROTEIN and two CYTOCHROME P450 genes, and well-known genes like VviAGL11. Integrative genomic analyses resulted in 339 core candidate genes categorized into 13 groups related to seed development. Machine learning based genomic selection achieved a remarkable 99% precision in predicting grapevine seedlessness. Our findings highlight the polygenic nature of seedless and provide novel candidate genes for molecular genetics and an effective prediction for seedlessness in grape genomic breeding.\nEmergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases\nAuthors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.\nScore: 127.5, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569447\nTransposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6. We set out to test the hypothesis that TnpB nucleases may have also been repurposed for novel, unexpected functions other than CRIS-PR-Cas. Here, using phylogenetics, structural predictions, comparative genomics, and functional assays, we uncover multiple instances of programmable transcription factors that we name TnpB-like nuclease-dead repressors (TldR). These proteins employ naturally occurring guide RNAs to specifically target conserved promoter regions of the genome, leading to potent gene repression in a mechanism akin to CRISPRi technologies invented by humans7. Focusing on a TldR clade found broadly in Enterobacteriaceae, we discover that bacteriophages exploit the combined action of TldR and an adjacently encoded phage gene to alter the expression and composition of the host flagellar assembly, a transformation with the potential to impact motility8, phage susceptibility9, and host immunity10. Col-lectively, this work showcases the diverse molecular innovations that were enabled through repeated exaptation of genes encoded by transposable elements, and reveals that RNA-guided transcription factors emerged long before the development of dCas9-based editors.\nThe balance between ATR and DDK activities controls TopBP1-mediated locking of dormant origins at the pre-IC stage\nAuthors: Koundrioukoff, S.; Kim, S.-J.; Alary, N.; Toffano, A.; Melendez-Garcia, R.; Wu, X.; Liu, Y.; Gnan, S.; El-Hilali, S.; Brison, O.; Rosselli, F.; Chen, C.-L.; Debatisse, M.\nScore: 9.6, Published: 2023-12-03 DOI: 10.1101/2023.11.29.569233\nReplication stress, a major hallmark of cancers, and ensuing genome instability source from impaired progression of replication forks. The first line of defense against fork slowing is compensation, a long-described process that elicits firing of otherwise dormant origins. It remains unclear whether compensation requires activation of the DNA replication checkpoint or passively results from lengthening of the window of time during which dormant origins can fire when fork progression slows, or both. Using molecular DNA combing we show here that a linear relationship ties inter-origin distances to fork speeds, independently of the checkpoint status. We called this line \"stressline\" and further show that its slope enables precise quantification of the compensation efficiency. Comparison of the slopes in different genetic backgrounds reveals that compensation requires ATR, not CHK1, while TopBP1 and CDC7/DBF4 repress dormant origin activation. These results strongly suggest that TopBP1 locks dormant origins at the pre-IC stage and that ATR and DDK oppose to control the conversion of dormant pre-ICs into functional salvage origins. Both passive and active processes thus contribute to compensation. Moreover, Repli-seq and OK-seq analyses confirm the activating role of ATR and permit development of ATRAP-seq, a new procedure allowing mapping of early constitutive origins.\nZelda is dispensable for Drosophila melanogaster histone gene regulation\nAuthors: O'Haren, T. E.; Aoki, T.; Rieder, L. E.\nScore: 4.0, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572383\nTo ensure that the embryo can package exponentially increasing amounts of DNA, replication-dependent histones are some of the earliest transcribed genes from the zygotic genome. However, how the histone genes are identified is not known. The pioneer factors Zelda and CLAMP collaborate at a subset of genes to regulate zygotic genome activation in Drosophila melanogaster and target early activated genes to induce transcription. CLAMP also regulates the embryonic histone genes and helps establish the histone locus body, a suite of factors that controls histone mRNA biosynthesis. The relationship between Zelda and CLAMP led us to hypothesize that Zelda helps identify histone genes for early embryonic expression. We found that Zelda targets the histone locus early during embryogenesis, prior to histone gene expression. However, depletion of zelda in the early embryo does not affect histone mRNA levels or histone locus body formation. While surprising, these results concur with other investigations into Zeldas role in the early embryo, suggesting the earliest factors responsible for specifying the zygotic histone genes remain undiscovered.\n",
  "wordCount" : "2653",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572268">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572268" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572268">
        <p class="paperTitle">Rewriting regulatory DNA to dissect and reprogram gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572268" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572268" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martyn, G. E.; Montgomery, M. T.; Jones, H.; Guo, K.; Doughty, B. R.; Linder, J.; Chen, Z.; Cochran, K.; Lawrence, K. A.; Munson, G.; Pampari, A.; Fulco, C. P.; Kelley, D. R.; Lander, E. S.; Kundaje, A.; Engreitz, J. M.</p>
        <p class="info">Score: 50.7, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572268' target='https://doi.org/10.1101/2023.12.20.572268'> 10.1101/2023.12.20.572268</a></p>
        <p class="abstract">Regulatory DNA sequences within enhancers and promoters bind transcription factors to encode cell type-specific patterns of gene expression. However, the regulatory effects and programmability of such DNA sequences remain difficult to map or predict because we have lacked scalable methods to precisely edit regulatory DNA and quantify the effects in an endogenous genomic context. Here we present an approach to measure the quantitative effects of hundreds of designed DNA sequence variants on gene expression, by combining pooled CRISPR prime editing with RNA fluorescence in situ hybridization and cell sorting (Variant-FlowFISH). We apply this method to mutagenize and rewrite regulatory DNA sequences in an enhancer and the promoter of PPIF in two immune cell lines. Of 672 variant-cell type pairs, we identify 497 that affect PPIF expression. These variants appear to act through a variety of mechanisms including disruption or optimization of existing transcription factor binding sites, as well as creation of de novo sites. Disrupting a single endogenous transcription factor binding site often led to large changes in expression (up to -40% in the enhancer, and -50% in the promoter). The same variant often had different effects across cell types and states, demonstrating a highly tunable regulatory landscape. We use these data to benchmark performance of sequence-based predictive models of gene regulation, and find that certain types of variants are not accurately predicted by existing models. Finally, we computationally design 185 small sequence variants ([&amp;le;]10 bp) and optimize them for specific effects on expression in silico. 84% of these rationally designed edits showed the intended direction of effect, and some had dramatic effects on expression (-100% to &#43;202%). Variant-FlowFISH thus provides a powerful tool to map the effects of variants and transcription factor binding sites on gene expression, test and improve computational models of gene regulation, and reprogram regulatory DNA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572247">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572247" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572247">
        <p class="paperTitle">Between-group heritability and the status of hereditarianism as an evolutionary science</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572247" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572247" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roseman, C. C.; Bird, K.</p>
        <p class="info">Score: 26.9, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572247' target='https://doi.org/10.1101/2023.12.18.572247'> 10.1101/2023.12.18.572247</a></p>
        <p class="abstract">Hereditarianism is a school of thought that contends there are substantial evolved cognitive and behavioral differences among groups of humans which are both resistant to environmental intervention and are a root cause of differential social outcomes across groups. The relationship of betweengroup heritability [Formula] to within-group heritability [Formula] is one of the key theoretical components of hereditarianism and forms one of the bases for its claim to be an evolutionary science. Here, we examine the relationship between [Formula]and [Formula] and its application to problems in the hereditarian literature from an evolutionary genetic perspective. We demonstrate that the formulation of the relationship between [Formula] and [Formula]used in the hereditarian literature has no evolutionary content. By re-writing the relationship between [Formula] and[Formula] in a novel evolutionary framework, we demonstrate that there is no way to predict[Formula] using[Formula] without considerable additional theory that is absent from the hereditarian literature. Furthermore, we demonstrate that the hereditarian technique that uses[Formula] and [Formula] as a means of judging whether a given difference between groups may be plausibly ameliorated through environmental intervention is mathematically flawed. Lastly, we fill a gap in the hereditarian literature by writing out a means of using [Formula] to predict the absolute difference between groups under a neutral evolutionary model and find that it is much smaller than claimed by hereditarians. In conclusion, we propose a path forward for the study of human variation that moves us past the ill-conceived nature vs. nurture question and allows us to focus on more productive issues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572041">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572041" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572041">
        <p class="paperTitle">The germline regulates longevity and somatic repair in a sex-specific manner</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572041" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572041" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moses, E.; Atlan, T.; Sun, X.; Franek, R.; Siddiqui, A.; Marinov, G. K.; Shifman, S.; Zucker, D. M.; Oron-Gottesman, A.; Greenleaf, W. J.; Cohen, E.; Ram, O.; Harel, I.</p>
        <p class="info">Score: 24.7, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572041' target='https://doi.org/10.1101/2023.12.18.572041'> 10.1101/2023.12.18.572041</a></p>
        <p class="abstract">Classical evolutionary theories propose tradeoffs between reproduction, damage repair, and lifespan. However, the specific role of the germline in shaping vertebrate aging remains largely unknown. Here, we use the turquoise killifish (N. furzeri) to genetically arrest germline differentiation at discrete stages, and examine how different  flavors of infertility impact life-history. We first constructed a comprehensive single-cell gonadal atlas, providing cell-type-specific markers for downstream phenotypic analysis. Next, investigating our genetic models revealed that only germline depletion enhanced female damage repair, while arresting germline differentiation did not. Conversely, germline-depleted males were significantly long-lived, indicating that the mere presence of the germline can negatively affect lifespan. Transcriptomic analysis highlighted enrichment of pro-longevity pathways and genes, with functional conservation in germline-depleted C. elegans. Finally, germline depletion extended male healthspan through rejuvenated metabolic functions. Our results suggest that different germline manipulation paradigms can yield pronounced sexually dimorphic phenotypes, implying alternative mechanisms to classical evolutionary tradeoffs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572763" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572763">
        <p class="paperTitle">KNL1 and NDC80 represent new universal markers for the detection of functional centromeres in plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oliveira, L.; Neumann, P.; Mata-Sucre, Y.; Kuo, Y.-T.; Marques, A.; Schubert, V.; Macas, J.</p>
        <p class="info">Score: 14.4, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572763' target='https://doi.org/10.1101/2023.12.21.572763'> 10.1101/2023.12.21.572763</a></p>
        <p class="abstract">Centromere is the chromosomal site of kinetochore assembly and microtubule attachment for chromosome segregation. Given its importance, markers that allow specific labeling of centromeric chromatin throughout the cell cycle and across all chromosome types are sought for facilitating various centromere studies. Antibodies against the N-terminal region of CENH3 are commonly used for this purpose, since CENH3 is the near-universal marker of functional centromeres. However, because the N-terminal region of CENH3 is highly variable among plant species, antibodies directed against this region usually function only in a small group of closely related species. As a more versatile alternative, we present here antibodies targeted to the conserved domains of two outer kinetochore proteins, KNL1 and NDC80. Sequence comparison of these domains across more than 350 plant species revealed a high degree of conservation, particularly within a six amino acid motif, FFGPVS in KNL1, suggesting that both antibodies would function in a wide range of plant species. This assumption was confirmed by immunolabeling experiments in angiosperm (monocot and dicot) and gymnosperm species, including those with mono-, holo-, and meta-polycentric chromosomes. In addition to centromere labeling on condensed chromosomes during cell division, both antibodies detected the corresponding regions in the interphase nuclei of most species tested. These results demonstrated that KNL1 and NDC80 are better suited for immunolabeling centromeres than CENH3, because antibodies against these proteins offer incomparably greater versatility across different plant species which is particularly convenient for studying the organization and function of the centromere in non-model species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571870">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571870" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571870">
        <p class="paperTitle">Genetic and environmental determinants of multicellular-like phenotypes in fission yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571870" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571870" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Köver, B.; Cohen, C. E.; Ralser, M.; Heineike, B. M.; Bähler, J.</p>
        <p class="info">Score: 12.7, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571870' target='https://doi.org/10.1101/2023.12.15.571870'> 10.1101/2023.12.15.571870</a></p>
        <p class="abstract">Multicellular fungi have repeatedly given rise to primarily unicellular yeast species. Some of these, including Schizosaccharomyces pombe, are able to revert to multicellular-like phenotypes (MLP). Our bioinformatic analysis of existing data suggested that, besides some regulatory proteins, most proteins involved in MLP formation are not functionally conserved between S. pombe and budding yeast. We developed high-throughput assays for two types of MLP in S. pombe: flocculation and surface adhesion, which correlated in minimal medium, suggesting a common mechanism. Using a library of 57 natural S. pombe isolates, we found MLP formation to widely vary across different nutrient and drug conditions. Next, in a segregant S. pombe library generated from an adhesive natural isolate and the standard laboratory strain, MLP formation correlated with expression levels of the transcription-factor gene mbx2 and several flocculins. Quantitative trait locus mapping of MLP formation located a causal frameshift mutation in the srb11 gene encoding cyclin C, a part of the Cdk8 kinase module (CKM) of the Mediator complex. Other CKM deletions also resulted in MLP formation, consistently through upregulation of mbx2, and only in minimal media. We screened a library of 3721 gene-deletion strains, uncovering additional genes involved in surface adhesion on minimal media. We identified 31 high-confidence hits, including 19 genes that have not been associated with MLPs in fission or budding yeast. Notably, deletion of srb11, unlike deletions of the 31 hits, did not compromise cell growth, which might explain its natural occurrence as a QTL for MLP formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572706" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572706">
        <p class="paperTitle">Modelling the age-related deceleration of clonal haematopoiesis in UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: MacGregor, H. A.; Easton, D.; Blundell, J. R.</p>
        <p class="info">Score: 8.2, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572706' target='https://doi.org/10.1101/2023.12.21.572706'> 10.1101/2023.12.21.572706</a></p>
        <p class="abstract">Somatic mutations acquired in haematopoietic stem cells can drive clonal expansions, a phenomenon called clonal haematopoiesis (CH). CH has been associated with higher risk of haematological cancer, cardiovascular disease and lower life expectancy and is strongly age-dependent, becoming almost ubiquitous in older adults. However, the dynamics of CH are not yet fully understood, and an accurate quantitative model of clonal expansion in healthy people could inform early detection strategies. Here, we analyse the age-dependence of the variant allele frequency (VAF) distributions of common CH driver variants from the exomes of 400,000 cancer-free UK Biobank participants. We find evidence of a substantial deceleration of clonal expansion with age taking place in younger people. This deceleration appears to be general: different CH hotspot variants all displayed a [~] 3-fold reduction in growth rate between young and old. We develop several alternative models of clonal dynamics that might explain this age-related deceleration. First we show that variation in fitness of variants among individuals (e.g., due to genetic susceptibility) is unable to produce the required deceleration. Second, we consider a clonal competition model in which clonal expansion in later life is inhibited by competition with an unobserved background of other expanding clones. Finally, we consider an ageing model where HSC division slows down with increasing age. The observed age-specific VAF distributions are consistent with the ageing model with plausible parameters. To further interrogate these models, we consider a large data set of longitudinal growth rates obtained from serial samples of blood. These data suggest that a general slowdown of HSC division rates plays a key role in shaping the genetic diversity of ageing blood.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573032">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573032" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573032">
        <p class="paperTitle">Integrative genomics reveals the polygenic basis of seedlessness in grapevine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573032" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573032" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, X.; Liu, Z.; Zhang, F.; Xiao, H.; Cao, S.; Xue, H.; Liu, W.; Su, Y.; Liu, Z.; Zhong, H.; Zhang, F.; Ahmad, B.; Long, Q.; Zhang, Y.; Liu, Y.; Gan, Y.; Hou, T.; Jin, Z.; Wu, X.; Wang, Y.; Peng, Y.; Zhou, Y.</p>
        <p class="info">Score: 6.4, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573032' target='https://doi.org/10.1101/2023.12.22.573032'> 10.1101/2023.12.22.573032</a></p>
        <p class="abstract">Seedlessness is a crucial quality trait in table grape (Vitis vinifera L.) breeding. However, the development of seeds involved intricate regulations, while the polygenic basis of seed abortion remains unclear. Here, we combine comparative genomics, population genetics, quantitative genetics, and integrative genomics to unravel the evolution and polygenic basis of seedlessness in grapes. We generated four haplotype-resolved telomere-to-telomere (T2T) genomes for two seedless grape cultivars, Thompson Seedless (TS, syn. Sultania) and Black Monukka (BM). Comparative genomics identified a ~4.25 Mb hemizygous inversion on Chr10 specific in seedless cultivars, with seedless-associated genes VvTT16 and VvSUS2 located at breakpoints. Population genomic analyses of 548 grapevine accessions revealed two distinct clusters of seedless cultivars, tracing the origin of the seedlessness trait back to Sultania. Introgression, rather than convergent selection, shaped the evolutionary history of seedlessness in grape improvement. Genome-wide association study (GWAS) analysis identified 110 quantitative trait loci (QTLs) associated with 634 candidate genes, including novel candidate genes, such as three 11S GLOBULIN SEED STORAGE PROTEIN and two CYTOCHROME P450 genes, and well-known genes like VviAGL11. Integrative genomic analyses resulted in 339 core candidate genes categorized into 13 groups related to seed development. Machine learning based genomic selection achieved a remarkable 99% precision in predicting grapevine seedlessness. Our findings highlight the polygenic nature of seedless and provide novel candidate genes for molecular genetics and an effective prediction for seedlessness in grape genomic breeding.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569447">
        <p class="paperTitle">Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.</p>
        <p class="info">Score: 127.5, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569447' target='https://doi.org/10.1101/2023.11.30.569447'> 10.1101/2023.11.30.569447</a></p>
        <p class="abstract">Transposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6. We set out to test the hypothesis that TnpB nucleases may have also been repurposed for novel, unexpected functions other than CRIS-PR-Cas. Here, using phylogenetics, structural predictions, comparative genomics, and functional assays, we uncover multiple instances of programmable transcription factors that we name TnpB-like nuclease-dead repressors (TldR). These proteins employ naturally occurring guide RNAs to specifically target conserved promoter regions of the genome, leading to potent gene repression in a mechanism akin to CRISPRi technologies invented by humans7. Focusing on a TldR clade found broadly in Enterobacteriaceae, we discover that bacteriophages exploit the combined action of TldR and an adjacently encoded phage gene to alter the expression and composition of the host flagellar assembly, a transformation with the potential to impact motility8, phage susceptibility9, and host immunity10. Col-lectively, this work showcases the diverse molecular innovations that were enabled through repeated exaptation of genes encoded by transposable elements, and reveals that RNA-guided transcription factors emerged long before the development of dCas9-based editors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569233">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569233" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569233">
        <p class="paperTitle">The balance between ATR and DDK activities controls TopBP1-mediated locking of dormant origins at the pre-IC stage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569233" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569233" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koundrioukoff, S.; Kim, S.-J.; Alary, N.; Toffano, A.; Melendez-Garcia, R.; Wu, X.; Liu, Y.; Gnan, S.; El-Hilali, S.; Brison, O.; Rosselli, F.; Chen, C.-L.; Debatisse, M.</p>
        <p class="info">Score: 9.6, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569233' target='https://doi.org/10.1101/2023.11.29.569233'> 10.1101/2023.11.29.569233</a></p>
        <p class="abstract">Replication stress, a major hallmark of cancers, and ensuing genome instability source from impaired progression of replication forks. The first line of defense against fork slowing is compensation, a long-described process that elicits firing of otherwise dormant origins. It remains unclear whether compensation requires activation of the DNA replication checkpoint or passively results from lengthening of the window of time during which dormant origins can fire when fork progression slows, or both. Using molecular DNA combing we show here that a linear relationship ties inter-origin distances to fork speeds, independently of the checkpoint status. We called this line &#34;stressline&#34; and further show that its slope enables precise quantification of the compensation efficiency. Comparison of the slopes in different genetic backgrounds reveals that compensation requires ATR, not CHK1, while TopBP1 and CDC7/DBF4 repress dormant origin activation. These results strongly suggest that TopBP1 locks dormant origins at the pre-IC stage and that ATR and DDK oppose to control the conversion of dormant pre-ICs into functional salvage origins. Both passive and active processes thus contribute to compensation. Moreover, Repli-seq and OK-seq analyses confirm the activating role of ATR and permit development of ATRAP-seq, a new procedure allowing mapping of early constitutive origins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572383">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572383" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572383">
        <p class="paperTitle">Zelda is dispensable for Drosophila melanogaster histone gene regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572383" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572383" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Haren, T. E.; Aoki, T.; Rieder, L. E.</p>
        <p class="info">Score: 4.0, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572383' target='https://doi.org/10.1101/2023.12.19.572383'> 10.1101/2023.12.19.572383</a></p>
        <p class="abstract">To ensure that the embryo can package exponentially increasing amounts of DNA, replication-dependent histones are some of the earliest transcribed genes from the zygotic genome. However, how the histone genes are identified is not known. The pioneer factors Zelda and CLAMP collaborate at a subset of genes to regulate zygotic genome activation in Drosophila melanogaster and target early activated genes to induce transcription. CLAMP also regulates the embryonic histone genes and helps establish the histone locus body, a suite of factors that controls histone mRNA biosynthesis. The relationship between Zelda and CLAMP led us to hypothesize that Zelda helps identify histone genes for early embryonic expression. We found that Zelda targets the histone locus early during embryogenesis, prior to histone gene expression. However, depletion of zelda in the early embryo does not affect histone mRNA levels or histone locus body formation. While surprising, these results concur with other investigations into Zeldas role in the early embryo, suggesting the earliest factors responsible for specifying the zygotic histone genes remain undiscovered.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
