<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Expression Level Analysis of ACE2 Receptor Gene in African-American and Non-African-American COVID-19 Patients
Authors: Nyamari, M. N.; Omar, K. M.; Fayehun, A. F.; Dachi, O.; Bwana, B. K.; Awe, O. I.
Score: 19.5, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557129
BackgroundThe COVID-19 pandemic caused by SARS-CoV-2 has spread rapidly across the continents. While the incidence of COVID-19 has been reported to be higher among African-American individuals, the rate of mortality has been lower compared to that of non-African-Americans.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Expression Level Analysis of ACE2 Receptor Gene in African-American and Non-African-American COVID-19 Patients
Authors: Nyamari, M. N.; Omar, K. M.; Fayehun, A. F.; Dachi, O.; Bwana, B. K.; Awe, O. I.
Score: 19.5, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557129
BackgroundThe COVID-19 pandemic caused by SARS-CoV-2 has spread rapidly across the continents. While the incidence of COVID-19 has been reported to be higher among African-American individuals, the rate of mortality has been lower compared to that of non-African-Americans." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Expression Level Analysis of ACE2 Receptor Gene in African-American and Non-African-American COVID-19 Patients
Authors: Nyamari, M. N.; Omar, K. M.; Fayehun, A. F.; Dachi, O.; Bwana, B. K.; Awe, O. I.
Score: 19.5, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557129
BackgroundThe COVID-19 pandemic caused by SARS-CoV-2 has spread rapidly across the continents. While the incidence of COVID-19 has been reported to be higher among African-American individuals, the rate of mortality has been lower compared to that of non-African-Americans."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Expression Level Analysis of ACE2 Receptor Gene in African-American and Non-African-American COVID-19 Patients\nAuthors: Nyamari, M. N.; Omar, K. M.; Fayehun, A. F.; Dachi, O.; Bwana, B. K.; Awe, O. I.\nScore: 19.5, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557129\nBackgroundThe COVID-19 pandemic caused by SARS-CoV-2 has spread rapidly across the continents. While the incidence of COVID-19 has been reported to be higher among African-American individuals, the rate of mortality has been lower compared to that of non-African-Americans.",
  "keywords": [
    
  ],
  "articleBody": " Expression Level Analysis of ACE2 Receptor Gene in African-American and Non-African-American COVID-19 Patients\nAuthors: Nyamari, M. N.; Omar, K. M.; Fayehun, A. F.; Dachi, O.; Bwana, B. K.; Awe, O. I.\nScore: 19.5, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557129\nBackgroundThe COVID-19 pandemic caused by SARS-CoV-2 has spread rapidly across the continents. While the incidence of COVID-19 has been reported to be higher among African-American individuals, the rate of mortality has been lower compared to that of non-African-Americans. ACE2 is involved in COVID-19 as SARS-CoV-2 uses the ACE2 enzyme to enter host cells. Although the difference in COVID-19 incidence can be explained by many factors such as low accessibility of health insurance among the African-American community, little is known about ACE2 expression in African-American COVID-19 patients compared to non-African-American COVID-19 patients. The variable expression of genes can contribute to this observed phenomenon. MethodologyIn this study, transcriptomes from African-American and non-African-American COVID-19 patients were retrieved from the sequence read archive and analyzed for ACE2 gene expression. HISAT2 was used to align the reads to the human reference genome, and HTseq-count was used to get raw gene counts. EdgeR was utilized for differential gene expression analysis, and enrichR was employed for gene enrichment analysis. ResultsThe datasets included 14 and 33 transcriptome sequences from COVID-19 patients of African-American and non-African-American descent, respectively. There were 24,092 differentially expressed genes, with 7,718 upregulated (log fold change \u003e 1 and FDR 0.05) and 16,374 downregulated (log fold change -1 and FDR 0.05). The ACE2 mRNA level was found to be considerably downregulated in the African-American cohort (p-value = 0.0242, p-adjusted value = 0.038). ConclusionThe downregulation of ACE2 in the African-American cohort could indicate a correlation to the low COVID-19 severity observed among the African-American community.\nSystematic identification of transcriptional activator domains from non-transcription factor proteins in plants and yeast\nAuthors: Hummel, N.; Markel, K.; Stefani, J.; Staller, M. V.; Shih, P.\nScore: 13.9, Published: 2023-09-12 DOI: 10.1101/2023.09.12.557247\nTranscription factors promote gene expression via trans-regulatory activation domains. Although whole genome scale screens in model organisms (e.g. human, yeast, fly) have helped identify activation domains from transcription factors, such screens have been less extensively used to explore the occurrence of activation domains in non-transcription factor proteins, such as transcriptional coactivators, chromatin regulators and some cytosolic proteins, leaving a blind spot on what role activation domains in these proteins could play in regulating transcription. We utilized the activation domain predictor PADDLE to mine the entire proteomes of two model eukaryotes, Arabidopsis thaliana and Saccharomyces cerevisiae (1). We characterized 18,000 fragments covering predicted activation domains from \u003e800 non-transcription factor genes in both species, and experimentally validated that 89% of proteins contained fragments capable of activating transcription in yeast. Peptides with similar sequence composition show a broad range of activities, which is explained by the arrangement of key amino acids. We also annotated hundreds of nuclear proteins with activation domains as putative coactivators; many of which have never been ascribed any function in plants. Furthermore, our library contains \u003e250 non-nuclear proteins containing peptides with activation domain function across both eukaryotic lineages, suggesting that there are unknown biological roles of these peptides beyond transcription. Finally, we identify and validate short, universal eukaryotic activation domains that activate transcription in both yeast and plants with comparable or stronger performance to state-of-the-art activation domains. Overall, our dual host screen provides a blueprint on how to systematically discover novel genetic parts for synthetic biology that function across a wide diversity of eukaryotes. Significance StatementActivation domains promote transcription and play a critical role in regulating gene expression. Although the mapping of activation domains from transcription factors has been carried out in previous genome-wide screens, their occurrence in non-transcription factors has been less explored. We utilize an activation domain predictor to mine the entire proteomes of Arabidopsis thaliana and Saccharomyces cerevisiae for new activation domains on non-transcription factor proteins. We validate peptides derived from \u003e750 non-transcription factor proteins capable of activating transcription, discovering many potentially new coactivators in plants. Importantly, we identify novel genetic parts that can function across both species, representing unique synthetic biology tools.\nGenetic manipulation of an Ixodes scapularis cell line\nAuthors: Singh, N.; Rolandelli, A.; ONeal, A. J.; Butler, L. R.; Samaddar, S.; Laukaitis-Yousey, H.; Butnaru, M.; Mohr, S. E.; Perrimon, N.; Pedra, J. H.\nScore: 10.5, Published: 2023-09-10 DOI: 10.1101/2023.09.08.556855\nAlthough genetic manipulation is one of the hallmarks in model organisms, its applicability to non-model species has remained difficult due to our limited understanding of their fundamental biology. For instance, manipulation of a cell line originated from the blacklegged tick Ixodes scapularis, an arthropod that serves as a vector of several human pathogens, has yet to be established. Here, we demonstrate the successful genetic modification of the commonly used tick ISE6 line through ectopic expression and clustered regularly interspaced palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9) genome editing. We performed ectopic expression using nucleofection and attained CRISPR-Cas9 editing via homology dependent recombination. Targeting the E3 ubiquitin ligase X-linked inhibitor of apoptosis (xiap) and its substrate p47 led to alteration in molecular signaling within the immune deficiency (IMD) network and increased infection of the rickettsial agent Anaplasma phagocytophilum in I. scapularis ISE6 cells. Collectively, our findings complement techniques for genetic engineering of ticks in vivo and aid in circumventing the long-life cycle of I. scapularis, of which limits efficient and scalable molecular genetic screens. ImportanceGenetic engineering in arachnids has lagged compared to insects, largely because of substantial differences in their biology. This study unveils the implementation of ectopic expression and CRISPR-Cas9 gene editing in a tick cell line. We introduced fluorescently tagged proteins in ISE6 cells and edited its genome via homology dependent recombination. We ablated the expression of xiap and p47, two signaling molecules present in the immune deficiency (IMD) pathway of I. scapularis. Impairment of the tick IMD pathway, an analogous network of the tumor necrosis factor receptor in mammals, led to enhanced infection of the rickettsial agent A. phagocytophilum. Altogether, our findings provide a critical technical resource to the scientific community to enable a deeper understanding of biological circuits in the blacklegged tick Ixodes scapularis.\nDisentangling sources of clock-like mutations in germline and soma\nAuthors: Spisak, N.; de Manuel, M.; Milligan, W. R.; Sella, G.; Przeworski, M.\nScore: 10.0, Published: 2023-09-12 DOI: 10.1101/2023.09.07.556720\nThe rates of mutations vary across cell types. To identify causes of this variation, mutations are often decomposed into a combination of the single base substitution (SBS) \"signatures\" observed in germline, soma and tumors, with the idea that each signature corresponds to one or a small number of underlying mutagenic processes. Two such signatures turn out to be ubiquitous across cell types: SBS signature 1, which consists primarily of transitions at methylated CpG sites caused by spontaneous deamination, and the more diffuse SBS signature 5, which is of unknown etiology. In cancers, the number of mutations attributed to these two signatures accumulates linearly with age of diagnosis, and thus the signatures have been termed \"clock-like.\" To better understand this clocklike behavior, we develop a mathematical model that includes DNA replication errors, unrepaired damage, and damage repaired incorrectly. We show that mutational signatures can exhibit clocklike behavior because cell divisions occur at a constant rate and/or because damage rates remain constant over time, and that these distinct sources can be teased apart by comparing cell lineages that divide at different rates. With this goal in mind, we analyze the rate of accumulation of mutations in multiple cell types, including soma as well as male and female germline. We find no detectable increase in SBS signature 1 mutations in neurons and only a very weak increase in mutations assigned to the female germline, but a significant increase with time in rapidly-dividing cells, suggesting that SBS signature 1 is driven by rounds of DNA replication occurring at a relatively fixed rate. In contrast, SBS signature 5 increases with time in all cell types, including post-mitotic ones, indicating that it accumulates independently of cell divisions; this observation points to errors in DNA repair as the key underlying mechanism. Thus, the two \"clock-like\" signatures observed across cell types likely have distinct origins, one set by rates of cell division, the other by damage rates.\nSeptoria tritici blotch resistance gene Stb15 encodes a lectin receptor-like kinase\nAuthors: Hafeez, A. N.; Chartrain, L.; Cong, F.; Cambon, F.; Clarke, M.; Griffiths, S.; Hayta, S.; Jiang, M.; Keller, B.; Kirby, R.; Kolodziej, M. C.; Powell, O. R.; Smedley, M.; Steuernagel, B.; Xian, W.; Wingen, L. U.; Cheng, S.; Saintenac, C.; Wulff, B. B. H.; Brown, J. K. M.\nScore: 9.1, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557217\nSeptoria tritici blotch (STB), caused by the Dothideomycete fungus Zymoseptoria tritici, is of one of the most damaging diseases of bread wheat (Triticum aestivum)1 and the target of costly fungicide applications2. In line with the fungus apoplastic lifestyle, STB resistance genes isolated to date encode receptor-like kinases (RLKs) including a wall-associated kinase (Stb6) and a cysteine-rich kinase (Stb16q)3,4. Here, we used genome-wide association studies (GWAS) on a panel of 300 whole-genome shotgun-sequenced diverse wheat landraces (WatSeq consortium) to identify a 99 kb region containing six candidates for the Stb15 resistance gene. Mutagenesis and transgenesis confirmed a gene encoding an intronless G-type lectin RLK (LecRK) as Stb15. The characterisation of Stb15 exemplifies the unexpected diversity of RLKs conferring Z. tritici resistance in wheat.\nIdentification of genome safe harbor loci for human gene therapy based on evolutionary biology and comparative genomics\nAuthors: Quezada-Ramirez, M. A.; Loncar, S.; Campbell, M. A.; Gifford, R. J.; Kotin, R. M.\nScore: 6.7, Published: 2023-09-10 DOI: 10.1101/2023.09.08.556857\nGene transfer into hematopoietic stem and progenitor cells (HSPCs) involving indiscriminately integrating viral vectors has unpredictable outcomes including potential adverse events such as leukemogenesis, resulting from insertional mutagenesis. Therefore, identifying and characterizing genomic safe harbor (GSH) sites where exogenous genetic information can be integrated safely into adult progenitor and stem cells is critically important for therapeutic gene addition. Here, we present a novel approach to identify new GSH sites based on a proven system of stable transgene insertion: the evolutionarily conserved integration of parvovirus DNA into the germlines of host species. From a dataset of 199 unique endogenous parvovirus element (EPV) integration events identified in host genomes, 102 loci were mapped to the human genome with 17 being evaluated for potential use as GSHs in primary human CD34+ HSPCs. Six of the edited loci resulted in sustained transgene expression in both erythroid and myeloid phenotypes while three exhibited myeloid specific-expression and the remaining four loci resulted in a subset of myeloid and erythroid lineages. Following this approach, a minimum of nine promising GSH sites suitable for gene therapy of blood disorders were identified and additional GSH sites are likely to emerge from the remaining mapped loci which may be suitable for gene addition in stem and progenitor cells other than hematopoietic tissues. This novel approach extends the options for gene knock-ins while reducing the risks of insertional mutagenesis, unpredictable expression profiles, and increasing the safety and therapeutic effects.\nSlave Trade and Colonial Expansion Resulted in Strong Sex-Biased Admixture in South Africa\nAuthors: Reynolds, A.; Schurz, H.; Meeks, G.; Gravel, S.; Myrick, J. W.; Mendez, F. L.; Werely, C.; van Helden, P. D.; Hoal, E. G.; Poznik, G. D.; Kim, M.; Uren, C.; Underhill, P. A.; Möller, M.; Henn, B. M.\nScore: 6.6, Published: 2023-09-11 DOI: 10.1101/2023.09.06.556626\nThe colonial-period arrival of Europeans in southern Africa is associated with strong sex-biased migration by which male settlers displaced indigenous Khoekhoe and San men. Simultaneously, the importation of South Asian, Indonesian and Eastern African slaves may have contributed female-biased migration to Cape Town and surrounding areas. We examine the spatial and temporal spread of sex-biased migration from the Cape northward into Namaqualand and the southern Kalahari using genetic data from more than 1,400 individuals. In all regions, admixture patterns were sex-biased, with evidence of a greater male contribution of European ancestry and greater female contribution of Khoe-San ancestry. While admixture among Khoe-San, European, equatorial African, and Asian groups has likely been continuous from the founding of Cape Town to present-day, we find that Khoe-San groups further north experienced a single pulse of European admixture 6-8 generations ago. European admixture was followed by additional Khoe-San gene flow, potentially reflecting an aggregation of indigenous groups due to disruption by colonial interlopers. Male migration into the northern frontier territories was not a homogenous group of expanding Afrikaners and slaves. The Nama show evidence of distinct founder effects and derive 15% of their male lineages from Asian men, a pattern absent in the =Khomani San. Khoe-San ancestry from the paternal line is greatly diminished in populations from Cape Town, the Cederberg Mountains and Upington, but remains more frequent in self-identified ethnically indigenous groups. Strikingly, we estimate that Khoe-San Y-chromosomes were experiencing unprecedented population growth at the time of European arrival. Our findings shed light on the patterns of admixture and the population history of South Africa as the colonial frontier expanded.\nEngineering heterothallic strains in fission yeast\nAuthors: Garcia-Ruano, D.; Hsu, I.; Leray, B.; Billard, B.; Liti, G.; Coudreuse, D.\nScore: 5.3, Published: 2023-09-14 DOI: 10.1101/2023.09.13.557521\nIn poor nitrogen conditions, fission yeast cells mate, undergo meiosis and form spores that are resistant to deleterious environments. Natural isolates of Schizosaccharomyces pombe are homothallic. This allows them to naturally switch between the two h- and h+ mating types with a high frequency, thereby ensuring the presence of both mating partners in a population of cells. However, alteration of the mating type locus can abolish mating type switching or reduce it to a very low frequency. Such heterothallic strains have been isolated and are common in research laboratories due to the simplicity of their use for Mendelian genetics. In addition to the standard laboratory strains, a large collection of natural S. pombe isolates is now available, representing a powerful resource for investigating the genetic diversity and biology of fission yeast. However, most of these strains are homothallic, and only tedious or mutagenic strategies have been described to obtain heterothallic cells from a homothallic parent. Here, we describe a simple approach to generate heterothallic strains. It takes advantage of an alteration of the mating type locus that was previously identified in a mating type switching-deficient strain and the CRISPR-Cas9 editing tool, allowing for a one-step engineering of heterothallic cells with high efficiency.\nIdentifying latent genetic interactions in genome-wide association studies using multiple traits\nAuthors: Bass, A. J.; Bian, S.; Wingo, A.; Wingo, T. S.; Cutler, D.; Epstein, M.\nScore: 4.1, Published: 2023-09-13 DOI: 10.1101/2023.09.11.557155\nGenome-wide association studies of complex traits frequently find that SNP-based estimates of heritability are considerably smaller than estimates from classic family-based studies. This missing heritability may be partly explained by genetic variants interacting with other genes or environments that are difficult to specify, observe, and detect. To circumvent these challenges, we propose a new method to detect genetic interactions that leverages pleiotropy from multiple related traits without requiring the interacting variable to be specified or observed. Our approach, Latent Interaction Testing (LIT), uses the observation that correlated traits with shared latent genetic interactions have trait variance and covariance patterns that differ by genotype. LIT examines the relationship between trait variance/covariance patterns and genotype using a flexible kernel-based framework that is computationally scalable for biobank-sized datasets with a large number of traits. We first use simulated data to demonstrate that LIT substantially increases power to detect latent genetic interactions compared to a trait-by-trait univariate method. We then apply LIT to four obesity-related traits in the UK Biobank and detect genetic variants with interactive effects near known obesity-related genes. Overall, we show that LIT, implemented in the R package lit, uses shared information across traits to improve detection of latent genetic interactions compared to standard approaches.\nHigh throughput evaluation of genetic variants with prime editing sensor libraries\nAuthors: Gould, S. I.; Wuest, A. N.; Dong, K.; Johnson, G. A.; Hsu, A.; Narendra, V. K.; Levine, S. S.; Liu, D. R.; Sanchez-Rivera, F. J.\nScore: 6.2, Published: 2023-09-12 DOI: 10.1101/2022.10.26.513842\nMany human diseases have a strong association with diverse types of genetic alterations. These diseases include cancer, in which tumor genomes often harbor a complex spectrum of single-nucleotide alterations and chromosomal rearrangements that can perturb gene function in ways that remain poorly understood. Some cancer-associated genes exhibit a tremendous degree of mutational heterogeneity, which may impact disease initiation, progression, and therapy responses. For example, TP53, the most frequently mutated gene in cancer, shows extensive allelic variation that leads to the generation of altered proteins that can produce functionally distinct phenotypes. Whether distinct variants of TP53 and other genes encode proteins with loss-of-function, gain-of-function, or otherwise neomorphic phenotypes remains both controversial and technically challenging to assess, particularly at the endogenous level. Here, we present a high-throughput prime editing \"sensor\" strategy to quantitatively assess the functional impact of diverse types of endogenous genetic variants. We used this strategy to screen the largest collection of endogenous cancer-associated TP53 variants assembled to date, identifying both known and novel alleles that impact p53 function in mechanistically diverse ways. Intriguingly, we find that certain types of endogenous TP53 variants, particularly those in the p53 oligomerization domain, display opposite phenotypes in exogenous overexpression systems. These include disease-relevant variants found in humans with cancer predisposition syndromes that encode altered proteins with unique molecular properties. Our results emphasize the physiological importance of gene dosage in shaping native protein stoichiometry and protein-protein interactions, highlight the dangers of using exogenous overexpression systems to interpret pathogenic alleles, and establish a powerful computational and experimental framework for studying diverse types of genetic variants in their endogenous sequence context at scale.\n",
  "wordCount" : "2919",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557129">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557129" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557129">
        <p class="paperTitle">Expression Level Analysis of ACE2 Receptor Gene in African-American and Non-African-American COVID-19 Patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557129" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557129" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nyamari, M. N.; Omar, K. M.; Fayehun, A. F.; Dachi, O.; Bwana, B. K.; Awe, O. I.</p>
        <p class="info">Score: 19.5, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557129' target='https://doi.org/10.1101/2023.09.11.557129'> 10.1101/2023.09.11.557129</a></p>
        <p class="abstract">BackgroundThe COVID-19 pandemic caused by SARS-CoV-2 has spread rapidly across the continents. While the incidence of COVID-19 has been reported to be higher among African-American individuals, the rate of mortality has been lower compared to that of non-African-Americans. ACE2 is involved in COVID-19 as SARS-CoV-2 uses the ACE2 enzyme to enter host cells. Although the difference in COVID-19 incidence can be explained by many factors such as low accessibility of health insurance among the African-American community, little is known about ACE2 expression in African-American COVID-19 patients compared to non-African-American COVID-19 patients. The variable expression of genes can contribute to this observed phenomenon.

MethodologyIn this study, transcriptomes from African-American and non-African-American COVID-19 patients were retrieved from the sequence read archive and analyzed for ACE2 gene expression. HISAT2 was used to align the reads to the human reference genome, and HTseq-count was used to get raw gene counts. EdgeR was utilized for differential gene expression analysis, and enrichR was employed for gene enrichment analysis.

ResultsThe datasets included 14 and 33 transcriptome sequences from COVID-19 patients of African-American and non-African-American descent, respectively. There were 24,092 differentially expressed genes, with 7,718 upregulated (log fold change &gt; 1 and FDR 0.05) and 16,374 downregulated (log fold change -1 and FDR 0.05). The ACE2 mRNA level was found to be considerably downregulated in the African-American cohort (p-value = 0.0242, p-adjusted value = 0.038).

ConclusionThe downregulation of ACE2 in the African-American cohort could indicate a correlation to the low COVID-19 severity observed among the African-American community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557247">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557247" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557247">
        <p class="paperTitle">Systematic identification of transcriptional activator domains from non-transcription factor proteins in plants and yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557247" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557247" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hummel, N.; Markel, K.; Stefani, J.; Staller, M. V.; Shih, P.</p>
        <p class="info">Score: 13.9, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557247' target='https://doi.org/10.1101/2023.09.12.557247'> 10.1101/2023.09.12.557247</a></p>
        <p class="abstract">Transcription factors promote gene expression via trans-regulatory activation domains. Although whole genome scale screens in model organisms (e.g. human, yeast, fly) have helped identify activation domains from transcription factors, such screens have been less extensively used to explore the occurrence of activation domains in non-transcription factor proteins, such as transcriptional coactivators, chromatin regulators and some cytosolic proteins, leaving a blind spot on what role activation domains in these proteins could play in regulating transcription. We utilized the activation domain predictor PADDLE to mine the entire proteomes of two model eukaryotes, Arabidopsis thaliana and Saccharomyces cerevisiae (1). We characterized 18,000 fragments covering predicted activation domains from &gt;800 non-transcription factor genes in both species, and experimentally validated that 89% of proteins contained fragments capable of activating transcription in yeast. Peptides with similar sequence composition show a broad range of activities, which is explained by the arrangement of key amino acids. We also annotated hundreds of nuclear proteins with activation domains as putative coactivators; many of which have never been ascribed any function in plants. Furthermore, our library contains &gt;250 non-nuclear proteins containing peptides with activation domain function across both eukaryotic lineages, suggesting that there are unknown biological roles of these peptides beyond transcription. Finally, we identify and validate short,  universal eukaryotic activation domains that activate transcription in both yeast and plants with comparable or stronger performance to state-of-the-art activation domains. Overall, our dual host screen provides a blueprint on how to systematically discover novel genetic parts for synthetic biology that function across a wide diversity of eukaryotes.

Significance StatementActivation domains promote transcription and play a critical role in regulating gene expression. Although the mapping of activation domains from transcription factors has been carried out in previous genome-wide screens, their occurrence in non-transcription factors has been less explored. We utilize an activation domain predictor to mine the entire proteomes of Arabidopsis thaliana and Saccharomyces cerevisiae for new activation domains on non-transcription factor proteins. We validate peptides derived from &gt;750 non-transcription factor proteins capable of activating transcription, discovering many potentially new coactivators in plants. Importantly, we identify novel genetic parts that can function across both species, representing unique synthetic biology tools.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556855">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556855" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556855">
        <p class="paperTitle">Genetic manipulation of an Ixodes scapularis cell line</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556855" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556855" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, N.; Rolandelli, A.; ONeal, A. J.; Butler, L. R.; Samaddar, S.; Laukaitis-Yousey, H.; Butnaru, M.; Mohr, S. E.; Perrimon, N.; Pedra, J. H.</p>
        <p class="info">Score: 10.5, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556855' target='https://doi.org/10.1101/2023.09.08.556855'> 10.1101/2023.09.08.556855</a></p>
        <p class="abstract">Although genetic manipulation is one of the hallmarks in model organisms, its applicability to non-model species has remained difficult due to our limited understanding of their fundamental biology. For instance, manipulation of a cell line originated from the blacklegged tick Ixodes scapularis, an arthropod that serves as a vector of several human pathogens, has yet to be established. Here, we demonstrate the successful genetic modification of the commonly used tick ISE6 line through ectopic expression and clustered regularly interspaced palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9) genome editing. We performed ectopic expression using nucleofection and attained CRISPR-Cas9 editing via homology dependent recombination. Targeting the E3 ubiquitin ligase X-linked inhibitor of apoptosis (xiap) and its substrate p47 led to alteration in molecular signaling within the immune deficiency (IMD) network and increased infection of the rickettsial agent Anaplasma phagocytophilum in I. scapularis ISE6 cells. Collectively, our findings complement techniques for genetic engineering of ticks in vivo and aid in circumventing the long-life cycle of I. scapularis, of which limits efficient and scalable molecular genetic screens.

ImportanceGenetic engineering in arachnids has lagged compared to insects, largely because of substantial differences in their biology. This study unveils the implementation of ectopic expression and CRISPR-Cas9 gene editing in a tick cell line. We introduced fluorescently tagged proteins in ISE6 cells and edited its genome via homology dependent recombination. We ablated the expression of xiap and p47, two signaling molecules present in the immune deficiency (IMD) pathway of I. scapularis. Impairment of the tick IMD pathway, an analogous network of the tumor necrosis factor receptor in mammals, led to enhanced infection of the rickettsial agent A. phagocytophilum. Altogether, our findings provide a critical technical resource to the scientific community to enable a deeper understanding of biological circuits in the blacklegged tick Ixodes scapularis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556720">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556720" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556720">
        <p class="paperTitle">Disentangling sources of clock-like mutations in germline and soma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556720" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556720" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Spisak, N.; de Manuel, M.; Milligan, W. R.; Sella, G.; Przeworski, M.</p>
        <p class="info">Score: 10.0, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556720' target='https://doi.org/10.1101/2023.09.07.556720'> 10.1101/2023.09.07.556720</a></p>
        <p class="abstract">The rates of mutations vary across cell types. To identify causes of this variation, mutations are often decomposed into a combination of the single base substitution (SBS) &#34;signatures&#34; observed in germline, soma and tumors, with the idea that each signature corresponds to one or a small number of underlying mutagenic processes. Two such signatures turn out to be ubiquitous across cell types: SBS signature 1, which consists primarily of transitions at methylated CpG sites caused by spontaneous deamination, and the more diffuse SBS signature 5, which is of unknown etiology. In cancers, the number of mutations attributed to these two signatures accumulates linearly with age of diagnosis, and thus the signatures have been termed &#34;clock-like.&#34; To better understand this clocklike behavior, we develop a mathematical model that includes DNA replication errors, unrepaired damage, and damage repaired incorrectly. We show that mutational signatures can exhibit clocklike behavior because cell divisions occur at a constant rate and/or because damage rates remain constant over time, and that these distinct sources can be teased apart by comparing cell lineages that divide at different rates. With this goal in mind, we analyze the rate of accumulation of mutations in multiple cell types, including soma as well as male and female germline. We find no detectable increase in SBS signature 1 mutations in neurons and only a very weak increase in mutations assigned to the female germline, but a significant increase with time in rapidly-dividing cells, suggesting that SBS signature 1 is driven by rounds of DNA replication occurring at a relatively fixed rate. In contrast, SBS signature 5 increases with time in all cell types, including post-mitotic ones, indicating that it accumulates independently of cell divisions; this observation points to errors in DNA repair as the key underlying mechanism. Thus, the two &#34;clock-like&#34; signatures observed across cell types likely have distinct origins, one set by rates of cell division, the other by damage rates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557217" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557217">
        <p class="paperTitle">Septoria tritici blotch resistance gene Stb15 encodes a lectin receptor-like kinase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hafeez, A. N.; Chartrain, L.; Cong, F.; Cambon, F.; Clarke, M.; Griffiths, S.; Hayta, S.; Jiang, M.; Keller, B.; Kirby, R.; Kolodziej, M. C.; Powell, O. R.; Smedley, M.; Steuernagel, B.; Xian, W.; Wingen, L. U.; Cheng, S.; Saintenac, C.; Wulff, B. B. H.; Brown, J. K. M.</p>
        <p class="info">Score: 9.1, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557217' target='https://doi.org/10.1101/2023.09.11.557217'> 10.1101/2023.09.11.557217</a></p>
        <p class="abstract">Septoria tritici blotch (STB), caused by the Dothideomycete fungus Zymoseptoria tritici, is of one of the most damaging diseases of bread wheat (Triticum aestivum)1 and the target of costly fungicide applications2. In line with the fungus apoplastic lifestyle, STB resistance genes isolated to date encode receptor-like kinases (RLKs) including a wall-associated kinase (Stb6) and a cysteine-rich kinase (Stb16q)3,4. Here, we used genome-wide association studies (GWAS) on a panel of 300 whole-genome shotgun-sequenced diverse wheat landraces (WatSeq consortium) to identify a 99 kb region containing six candidates for the Stb15 resistance gene. Mutagenesis and transgenesis confirmed a gene encoding an intronless G-type lectin RLK (LecRK) as Stb15. The characterisation of Stb15 exemplifies the unexpected diversity of RLKs conferring Z. tritici resistance in wheat.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556857">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556857" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556857">
        <p class="paperTitle">Identification of genome safe harbor loci for human gene therapy based on evolutionary biology and comparative genomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556857" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556857" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Quezada-Ramirez, M. A.; Loncar, S.; Campbell, M. A.; Gifford, R. J.; Kotin, R. M.</p>
        <p class="info">Score: 6.7, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556857' target='https://doi.org/10.1101/2023.09.08.556857'> 10.1101/2023.09.08.556857</a></p>
        <p class="abstract">Gene transfer into hematopoietic stem and progenitor cells (HSPCs) involving indiscriminately integrating viral vectors has unpredictable outcomes including potential adverse events such as leukemogenesis, resulting from insertional mutagenesis. Therefore, identifying and characterizing genomic safe harbor (GSH) sites where exogenous genetic information can be integrated safely into adult progenitor and stem cells is critically important for therapeutic gene addition. Here, we present a novel approach to identify new GSH sites based on a proven system of stable transgene insertion: the evolutionarily conserved integration of parvovirus DNA into the germlines of host species. From a dataset of 199 unique endogenous parvovirus element (EPV) integration events identified in host genomes, 102 loci were mapped to the human genome with 17 being evaluated for potential use as GSHs in primary human CD34&#43; HSPCs. Six of the edited loci resulted in sustained transgene expression in both erythroid and myeloid phenotypes while three exhibited myeloid specific-expression and the remaining four loci resulted in a subset of myeloid and erythroid lineages. Following this approach, a minimum of nine promising GSH sites suitable for gene therapy of blood disorders were identified and additional GSH sites are likely to emerge from the remaining mapped loci which may be suitable for gene addition in stem and progenitor cells other than hematopoietic tissues. This novel approach extends the options for gene knock-ins while reducing the risks of insertional mutagenesis, unpredictable expression profiles, and increasing the safety and therapeutic effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556626">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556626" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556626">
        <p class="paperTitle">Slave Trade and Colonial Expansion Resulted in Strong Sex-Biased Admixture in South Africa</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556626" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556626" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reynolds, A.; Schurz, H.; Meeks, G.; Gravel, S.; Myrick, J. W.; Mendez, F. L.; Werely, C.; van Helden, P. D.; Hoal, E. G.; Poznik, G. D.; Kim, M.; Uren, C.; Underhill, P. A.; Möller, M.; Henn, B. M.</p>
        <p class="info">Score: 6.6, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556626' target='https://doi.org/10.1101/2023.09.06.556626'> 10.1101/2023.09.06.556626</a></p>
        <p class="abstract">The colonial-period arrival of Europeans in southern Africa is associated with strong sex-biased migration by which male settlers displaced indigenous Khoekhoe and San men. Simultaneously, the importation of South Asian, Indonesian and Eastern African slaves may have contributed female-biased migration to Cape Town and surrounding areas. We examine the spatial and temporal spread of sex-biased migration from the Cape northward into Namaqualand and the southern Kalahari using genetic data from more than 1,400 individuals. In all regions, admixture patterns were sex-biased, with evidence of a greater male contribution of European ancestry and greater female contribution of Khoe-San ancestry. While admixture among Khoe-San, European, equatorial African, and Asian groups has likely been continuous from the founding of Cape Town to present-day, we find that Khoe-San groups further north experienced a single pulse of European admixture 6-8 generations ago. European admixture was followed by additional Khoe-San gene flow, potentially reflecting an aggregation of indigenous groups due to disruption by colonial interlopers. Male migration into the northern frontier territories was not a homogenous group of expanding Afrikaners and slaves. The Nama show evidence of distinct founder effects and derive 15% of their male lineages from Asian men, a pattern absent in the =Khomani San. Khoe-San ancestry from the paternal line is greatly diminished in populations from Cape Town, the Cederberg Mountains and Upington, but remains more frequent in self-identified ethnically indigenous groups. Strikingly, we estimate that Khoe-San Y-chromosomes were experiencing unprecedented population growth at the time of European arrival. Our findings shed light on the patterns of admixture and the population history of South Africa as the colonial frontier expanded.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557521">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557521" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557521">
        <p class="paperTitle">Engineering heterothallic strains in fission yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557521" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557521" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garcia-Ruano, D.; Hsu, I.; Leray, B.; Billard, B.; Liti, G.; Coudreuse, D.</p>
        <p class="info">Score: 5.3, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557521' target='https://doi.org/10.1101/2023.09.13.557521'> 10.1101/2023.09.13.557521</a></p>
        <p class="abstract">In poor nitrogen conditions, fission yeast cells mate, undergo meiosis and form spores that are resistant to deleterious environments. Natural isolates of Schizosaccharomyces pombe are homothallic. This allows them to naturally switch between the two h- and h&#43; mating types with a high frequency, thereby ensuring the presence of both mating partners in a population of cells. However, alteration of the mating type locus can abolish mating type switching or reduce it to a very low frequency. Such heterothallic strains have been isolated and are common in research laboratories due to the simplicity of their use for Mendelian genetics. In addition to the standard laboratory strains, a large collection of natural S. pombe isolates is now available, representing a powerful resource for investigating the genetic diversity and biology of fission yeast. However, most of these strains are homothallic, and only tedious or mutagenic strategies have been described to obtain heterothallic cells from a homothallic parent. Here, we describe a simple approach to generate heterothallic strains. It takes advantage of an alteration of the mating type locus that was previously identified in a mating type switching-deficient strain and the CRISPR-Cas9 editing tool, allowing for a one-step engineering of heterothallic cells with high efficiency.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557155">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557155" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557155">
        <p class="paperTitle">Identifying latent genetic interactions in genome-wide association studies using multiple traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557155" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557155" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bass, A. J.; Bian, S.; Wingo, A.; Wingo, T. S.; Cutler, D.; Epstein, M.</p>
        <p class="info">Score: 4.1, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557155' target='https://doi.org/10.1101/2023.09.11.557155'> 10.1101/2023.09.11.557155</a></p>
        <p class="abstract">Genome-wide association studies of complex traits frequently find that SNP-based estimates of heritability are considerably smaller than estimates from classic family-based studies. This  missing heritability may be partly explained by genetic variants interacting with other genes or environments that are difficult to specify, observe, and detect. To circumvent these challenges, we propose a new method to detect genetic interactions that leverages pleiotropy from multiple related traits without requiring the interacting variable to be specified or observed. Our approach, Latent Interaction Testing (LIT), uses the observation that correlated traits with shared latent genetic interactions have trait variance and covariance patterns that differ by genotype. LIT examines the relationship between trait variance/covariance patterns and genotype using a flexible kernel-based framework that is computationally scalable for biobank-sized datasets with a large number of traits. We first use simulated data to demonstrate that LIT substantially increases power to detect latent genetic interactions compared to a trait-by-trait univariate method. We then apply LIT to four obesity-related traits in the UK Biobank and detect genetic variants with interactive effects near known obesity-related genes. Overall, we show that LIT, implemented in the R package lit, uses shared information across traits to improve detection of latent genetic interactions compared to standard approaches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.10.26.513842">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.10.26.513842" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.10.26.513842">
        <p class="paperTitle">High throughput evaluation of genetic variants with prime editing sensor libraries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.10.26.513842" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.10.26.513842" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gould, S. I.; Wuest, A. N.; Dong, K.; Johnson, G. A.; Hsu, A.; Narendra, V. K.; Levine, S. S.; Liu, D. R.; Sanchez-Rivera, F. J.</p>
        <p class="info">Score: 6.2, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.10.26.513842' target='https://doi.org/10.1101/2022.10.26.513842'> 10.1101/2022.10.26.513842</a></p>
        <p class="abstract">Many human diseases have a strong association with diverse types of genetic alterations. These diseases include cancer, in which tumor genomes often harbor a complex spectrum of single-nucleotide alterations and chromosomal rearrangements that can perturb gene function in ways that remain poorly understood. Some cancer-associated genes exhibit a tremendous degree of mutational heterogeneity, which may impact disease initiation, progression, and therapy responses. For example, TP53, the most frequently mutated gene in cancer, shows extensive allelic variation that leads to the generation of altered proteins that can produce functionally distinct phenotypes. Whether distinct variants of TP53 and other genes encode proteins with loss-of-function, gain-of-function, or otherwise neomorphic phenotypes remains both controversial and technically challenging to assess, particularly at the endogenous level. Here, we present a high-throughput prime editing &#34;sensor&#34; strategy to quantitatively assess the functional impact of diverse types of endogenous genetic variants. We used this strategy to screen the largest collection of endogenous cancer-associated TP53 variants assembled to date, identifying both known and novel alleles that impact p53 function in mechanistically diverse ways. Intriguingly, we find that certain types of endogenous TP53 variants, particularly those in the p53 oligomerization domain, display opposite phenotypes in exogenous overexpression systems. These include disease-relevant variants found in humans with cancer predisposition syndromes that encode altered proteins with unique molecular properties. Our results emphasize the physiological importance of gene dosage in shaping native protein stoichiometry and protein-protein interactions, highlight the dangers of using exogenous overexpression systems to interpret pathogenic alleles, and establish a powerful computational and experimental framework for studying diverse types of genetic variants in their endogenous sequence context at scale.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
