<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Interpreting SNP heritability in admixed populations
Authors: Huang, J.; Basu, S.; Shriver, M. D.; Zaidi, A. A.
Score: 19.9, Published: 2023-08-04 DOI: 10.1101/2023.08.04.551959
SNP heritability (h2snp) is defined as the proportion of phenotypic variance explained by genotyped SNPs and is believed to be a lower bound of heritability (h2), being equal to it if all causal variants are known. Despite the simple intuition behind h2snp, its interpretation and equivalence to h2 is unclear, particularly in the presence of population structure and assortative mating.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Interpreting SNP heritability in admixed populations
Authors: Huang, J.; Basu, S.; Shriver, M. D.; Zaidi, A. A.
Score: 19.9, Published: 2023-08-04 DOI: 10.1101/2023.08.04.551959
SNP heritability (h2snp) is defined as the proportion of phenotypic variance explained by genotyped SNPs and is believed to be a lower bound of heritability (h2), being equal to it if all causal variants are known. Despite the simple intuition behind h2snp, its interpretation and equivalence to h2 is unclear, particularly in the presence of population structure and assortative mating." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-06T10:36:48+00:00" />
<meta property="article:modified_time" content="2023-08-06T10:36:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Interpreting SNP heritability in admixed populations
Authors: Huang, J.; Basu, S.; Shriver, M. D.; Zaidi, A. A.
Score: 19.9, Published: 2023-08-04 DOI: 10.1101/2023.08.04.551959
SNP heritability (h2snp) is defined as the proportion of phenotypic variance explained by genotyped SNPs and is believed to be a lower bound of heritability (h2), being equal to it if all causal variants are known. Despite the simple intuition behind h2snp, its interpretation and equivalence to h2 is unclear, particularly in the presence of population structure and assortative mating."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Interpreting SNP heritability in admixed populations\nAuthors: Huang, J.; Basu, S.; Shriver, M. D.; Zaidi, A. A.\nScore: 19.9, Published: 2023-08-04 DOI: 10.1101/2023.08.04.551959\nSNP heritability (h2snp) is defined as the proportion of phenotypic variance explained by genotyped SNPs and is believed to be a lower bound of heritability (h2), being equal to it if all causal variants are known. Despite the simple intuition behind h2snp, its interpretation and equivalence to h2 is unclear, particularly in the presence of population structure and assortative mating.",
  "keywords": [
    
  ],
  "articleBody": " Interpreting SNP heritability in admixed populations\nAuthors: Huang, J.; Basu, S.; Shriver, M. D.; Zaidi, A. A.\nScore: 19.9, Published: 2023-08-04 DOI: 10.1101/2023.08.04.551959\nSNP heritability (h2snp) is defined as the proportion of phenotypic variance explained by genotyped SNPs and is believed to be a lower bound of heritability (h2), being equal to it if all causal variants are known. Despite the simple intuition behind h2snp, its interpretation and equivalence to h2 is unclear, particularly in the presence of population structure and assortative mating. It is well known that population structure can lead to inflation in h2snp estimates. Here we use analytical theory and simulations to demonstrate that estimates of h2snp are not guaranteed to be equal to h2 in admixed populations, even in the absence of confounding and even if the causal variants are known. We interpret this discrepancy arising not because the estimate is biased, but because the estimand itself as defined under the random effects model may not be equal to h2. The model assumes that SNP effects are uncorrelated which may not be true, even for unlinked loci in admixed and structured populations, leading to over- or under-estimates of h2snp relative to h2. For the same reason, local ancestry heritability (hgamma) may also not be equal to the variance explained by local ancestry in admixed populations. We describe the quantitative behavior of h2snp and h2gamma as a function of admixture history and the genetic architecture of the trait and discuss its implications for genome-wide association and polygenic prediction.\nLocal population structure in Cambridgeshire during the Roman occupation\nAuthors: Scheib, C. L.; Hui, R.; Rose, A. K.; Solnik, A.; D'Atanasio, E.; Inskip, S. A.; Cessford, C.; Griffith, S. J.; Wiseman, R.; Neil, B.; Biers, T.; Harknett, S.-J.; Sasso, S.; Biagini, S. A.; Runfeldt, G.; Duhig, C.; Evans, C.; O'Connell, T. C.; Metspalu, M.; Millett, M. J.; Robb, J. E.; Kivisild, T.\nScore: 18.0, Published: 2023-08-02 DOI: 10.1101/2023.07.31.551265\nThe Roman period saw the empire expand across Europe and the Mediterranean, including much of what is today the United Kingdom. While there is written evidence of high mobility into and out of Britain for administrators, traders and the military, the impact of imperialism on local population structure is invisible in the textual record. The extent of genetic change that occurred in Britain before the Early Medieval Period and how closely linked by genetic kinship the local populations were, remains underexplored. Here, using genome-wide data from 52 ancient individuals from Cambridgeshire, we show low levels of genetic ancestry differentiation between Romano-British sites and lower levels of runs of homozygosity over 4 centimorgans (cM than in the Bronze Age and Neolithic. We find fourteen cases of genetic relatedness within and one between sites without evidence of patrilineal dominance and one case of temporary mobility within a family unit during the Late Romano-British period. We also show that the modern patterns of genetic ancestry composition in Modern Britain emerged after the Roman period.\nMitochondrial Genome Diversity across the Subphylum Saccharomycotina\nAuthors: Wolters, J. F.; LaBella, A.; Opulente, D.; Rokas, A.; Hittinger, C. T.\nScore: 14.9, Published: 2023-07-31 DOI: 10.1101/2023.07.28.551029\nEukaryotic life depends on the functional elements encoded by both the nuclear genome and organellar genomes, such as those contained within the mitochondria. The content, size, and structure of the mitochondrial genome varies across organisms with potentially large implications for phenotypic variance and resulting evolutionary trajectories. Among yeasts in the subphylum Saccharomycotina, extensive differences have been observed in various species relative to the model yeast Saccharomyces cerevisiae, but mitochondrial genome sampling across many groups has been scarce, even as hundreds of nuclear genomes have become available. By extracting mitochondrial assemblies from existing short-read genome sequence datasets, we have greatly expanded both the number of available genomes and the coverage across sparsely sampled clades. Comparison of 353 yeast mitochondrial genomes revealed that, while size and GC content were fairly consistent across species, those in the genera Metschnikowia and Saccharomyces trended larger, while several species in the order Saccharomycetales, which includes S. cerevisiae, exhibited lower GC content. Extreme examples for both size and GC content were scattered throughout the subphylum. All mitochondrial genomes shared a core set of protein-coding genes for Complexes III, IV, and V, but they varied in the presence or absence of mitochondrially-encoded canonical Complex I genes. We traced the loss of Complex I genes to a major event in the ancestor of the orders Saccharomycetales and Saccharomycodales, but we also observed several independent losses in the orders Phaffomycetales, Pichiales, and Dipodascales. In contrast to prior hypotheses based on smaller-scale datasets, comparison of evolutionary rates in protein-coding genes showed no bias towards elevated rates among aerobically fermenting (Crabtree/Warburg-positive) yeasts. Mitochondrial introns were widely distributed, but they were highly enriched in some groups. The majority of mitochondrial introns were poorly conserved within groups, but several were shared within groups, between groups, and even across taxonomic orders, which is consistent with horizontal gene transfer, likely involving homing endonucleases acting as selfish elements. As the number of available fungal nuclear genomes continues to expand, the methods described here to retrieve mitochondrial genome sequences from these datasets will prove invaluable to ensuring that studies of fungal mitochondrial genomes keep pace with their nuclear counterparts.\nisGWAS: ultra-high-throughput, scalable and equitable inference of genetic associations with disease\nAuthors: Foley, C. N.; Kuncheva, Z.; Marioni, R.; Runz, H.; Sun, B.\nScore: 39.2, Published: 2023-07-27 DOI: 10.1101/2023.07.21.550074\nGenome-wide association studies (GWAS) have proven a powerful tool for human geneticists to generate biological insights or hypotheses for drug discovery. Nevertheless, a dependency on sensitive individual-level data together with ever-increasing cohort sample sizes, numbers of variants and phenotypes studied put a strain on existing algorithms, limiting the GWAS approach from maximising potential. Here we present in-silico GWAS (isGWAS), a uniquely scalable algorithm to infer regression parameters in case-control GWAS from cohort-level summary data. For any sample size, isGWAS computes a variant-disease association parameter in [~]1 millisecond, or [~]11m variants in UK-Biobank within [~]4 minutes ([~]1500-fold faster than state-of-the-art). Extensive simulations and empirical tests demonstrate that isGWAS results are highly comparable to traditional regression-based approaches. We further introduce a heuristic re-sampling algorithm, leapfrog re-sampler (LRS), to extrapolate association results to semi-virtually enlarged cohorts. Owing to significant computational gains we anticipate a broad use of isGWAS and LRS which are customizable on a web interface.\nThe Interferon γ Pathway Enhances Pluripotency and X-Chromosome Reactivation in iPSC Reprogramming\nAuthors: Barrero, M.; Lopez-Rubio, A. V.; Lazarenkov, A.; Blanco, E.; Bauer, M.; Palma, L. G.; Bigas, A.; Di Croce, L.; Sardina, J. L.; Payer, B.\nScore: 7.9, Published: 2023-08-03 DOI: 10.1101/2023.07.31.551297\nReprogramming somatic cells into induced pluripotent stem cells (iPSCs) requires activation of the pluripotency network and resetting of the epigenome by erasing the epigenetic memory of the somatic state. In female mouse cells, a critical epigenetic reprogramming step is the reactivation of the inactive X chromosome. Despite its importance, a systematic understanding of the regulatory networks linking pluripotency and X-reactivation is missing. Here we reveal the pathways important for iPSC reprogramming and X-reactivation using a genome-wide CRISPR screen. In particular, we discover that activation of the interferon {gamma} (IFN{gamma}) pathway early during reprogramming accelerates pluripotency acquisition and X-reactivation. IFN{gamma} stimulates STAT3 signaling and the pluripotency network and leads to enhanced TET-mediated DNA demethylation, which consequently boosts X-reactivation. We therefore gain a mechanistic understanding of the role of IFN{gamma} in reprogramming and X-reactivation and provide a comprehensive resource of the molecular networks involved in these processes.\nPax3 lineage-specific deletion of Gpr161 is associated with spinal neural tube and craniofacial malformations during embryonic development.\nAuthors: Kim, S. E.; Chothani, P.; Shaik, R.; Pollard, W.; Finnell, R. H.\nScore: 8.7, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548129\nShh signaling is the morphogen signaling that regulates embryonic craniofacial and neural tube development. G protein-coupled receptor 161 (Gpr161) is a negative regulator of Shh signaling, and its inactivation in mice results in embryo lethality with craniofacial and neural tube defects (NTDs). However, the structural defects of later embryonic stages in Gpr161 null mice and cell lineages underlying abnormalities were not well characterized due to their limited lifespan. We found the Pax3 lineage-specific deletion of Gpr161 in mice presented with tectal hypertrophy (anterior dorsal neuroepithelium), cranial vault and facial bone hypoplasia (cranial neural crest (CNC)), vertebral abnormalities (somite), and the closed form of spina bifida (posterior dorsal neuroepithelium). In particular, the closed form of spina bifida is partly due to the reduced Pax3 and Cdx4 gene expression of the posterior dorsal neural tubes of Gpr161 mutant embryos involving decreased Wnt signaling whereas Shh signaling was increased. This study provides the novel role of Gpr161 in the posterior neural tube development and confirms its role on CNC- and somite-derived skeletogenesis and midbrain morphogenesis in mice.\nR2ROC: An efficient method of comparing two or more correlated AUC from out-of-sample prediction using polygenic scores\nAuthors: Momin, M. M.; Wray, N. R.; Lee, S. H.\nScore: 4.9, Published: 2023-08-03 DOI: 10.1101/2023.08.01.551571\nPolygenic risk scores (PRSs) enable early prediction of disease risk. Evaluating PRS performance for binary traits commonly relies on the area under the receiver operating characteristic curve (AUC). However, the widely used DeLongs method for comparative significance tests suffer from limitations, including computational time and the lack of a one-to-one mapping between test statistics based on AUC and R2. To overcome these limitations, we propose a novel approach that leverages the Delta method to derive the variance and covariance of AUC values, enabling a comprehensive and efficient comparative significance test. Our approach offers notable advantages over DeLongs method, including reduced computation time (up to 150-fold), making it suitable for large-scale analyses and ideal for integration into machine learning frameworks. Furthermore, our method allows for a direct one-to-one mapping between AUC and R2 values for comparative significance tests, providing enhanced insights into the relationship between these measures and facilitating their interpretation. We validated our proposed approach through simulations and applied it to real data comparing PRSs for diabetes and coronary artery disease (CAD) prediction in a cohort of 28,880 European individuals. The PRSs were derived using genome-wide association study summary statistics from two distinct sources. Our approach enabled a comprehensive and informative comparison of the PRSs, shedding light on their respective predictive abilities for diabetes and CAD. This advancement contributes to the assessment of genetic risk factors and personalized disease prediction, supporting better healthcare decision-making.\nZebrafish models of Mucopolysaccharidosis types IIIA, B, \u0026 C show hyperactivity and changes in oligodendrocyte state\nAuthors: Gerken, E.; Ahmad, S.; Barthelson, K.; Lardelli, M.\nScore: 4.1, Published: 2023-08-03 DOI: 10.1101/2023.08.02.550904\nSanfilippo syndrome childhood dementia, also known as mucopolysaccharidosis type III (MPS III), is a rare inherited lysosomal storage disorder. Subtypes of MPS III are caused by deficiencies in one of four enzymes required for degradation of the glycosaminoglycan heparan sulfate (HS). An inability to degrade HS leads to progressive neurodegeneration and death in the second or third decades of life. Knowledge of MPS III pathogenesis is incomplete, and no effective therapies exist. We generated the hypomorphic mutations sgshS387Lfs, nagluA603Efs and hgsnatG577Sfsin the endogenous zebrafish genes orthologous to human SGSH, NAGLU, and HGSNAT that are loci for mutations causing MPS III subtypes MPS IIIA, B and C respectively. Our models display the primary MPS III disease signature of significant brain accumulation of HS, while behavioural analyses support hyperactivity phenotypes. Brain transcriptome analysis revealed changes related to lysosomal, glycosaminoglycan, immune system and iron homeostasis biology in all three models but also distinct differences in brain transcriptome state between models. The transcriptome analysis also indicated marked disturbance of the oligodendrocyte cell state in the brains of MPS IIIA, B and C zebrafish, supporting that effects on this cell type are an early and consistent characteristic of MPS III. Overall, our zebrafish models recapture key characteristics of the human disease and phenotypes seen in mouse models. Our models will allow exploitation of the zebrafishs extreme fecundity and accessible anatomy to dissect the pathological mechanisms both common and divergent between the MPS IIIA, B, and C subtypes.\nThe neurotrophic factor MANF regulates autophagic flux and lysosome function to promote proteostasis in C. elegans\nAuthors: Taylor, S. K. B.; Gupta, B. P.\nScore: 3.4, Published: 2023-08-02 DOI: 10.1101/2023.07.31.551399\nThe neurotrophic factor, Mesencephalic Astrocyte Derived Neurotrophic Factor (MANF), promotes neuronal health and survival. MANF is conserved in metazoans and necessary for the protection of dopaminergic neurons. The neuroprotective role of MANF involves regulation of the ER unfolded protein response (ER-UPR). Recent work has shown that MANF also functions in many other tissues and affects autophagy genes. Previously, we reported that C. elegans manf-1 mutants show defects including increased ER stress, -Synuclein aggregation, and age-dependent dopaminergic neurodegeneration. In this study, we have further investigated manf-1s function and describe its mechanism of action in regulating stress response, protein aggregation and aging. Our results showed that exposure to ER stress-inducing agents caused manf-1 transcription and protein expression to be upregulated. Consistent with the protective role of manf-1, mutant animals showed greater sensitivity to ER stress and died prematurely. We generated manf-1p::manf-1::mCherry (MANF-1OE) and hsp::manf-1 transgenic strains overexpressing MANF and found that these animals had an extended lifespan, reduced protein aggregation, and increased neuronal health. The in vivo analysis of MANF-1 expression revealed that the protein was secreted and present in many tissues. Subcellular studies showed that MANF-1 was localized to lysosomes. We also observed that MANF-1OE worms had reduced LGG-1 and SQST-1/p62 levels, and increased expression and nuclear localization of the autophagy transcription factor hlh-30/TFEB. Thus, MANF-1 regulates protein homeostasis through increased lysosome activity and autophagic flux. Collectively, our work provides a novel understanding of MANF-1 function in promoting neuronal health, stress response, proteostasis, and lifespan of animals.\nMultiplexed amplicon sequencing reveals the heterogeneous spatial distribution of pyrethroid resistance mutations in Aedes albopictus mosquito populations in Southern France.\nAuthors: Fontaine, A.; Mignotte, A.; Lacour, G.; Nguyen, A.; Gomez, N.; Chanaud, L.; L'Ambert, G.; Briolant, S.\nScore: 3.1, Published: 2023-08-05 DOI: 10.1101/2023.08.05.552101\nThe risk of mosquito-borne diseases transmission is moving fast toward temperate climates with the colonization and proliferation of the Asian tiger mosquito vector Aedes albopictus and the rapid and mass transport of passengers returning from tropical regions where the viruses are endemic. The prevention of major Aedes-borne viruses heavily relies on the use of insecticides for vector control, mainly pyrethroids In Europe. High-throughput molecular assays can provide a cost-effective surrogate to phenotypic insecticide resistance assays when mutations have been previously linked to a resistance phenotype. Here, we screened for the spatial distribution of kdr mutations at a large scale using a two-step approach based on multiplexed amplicon sequencing and an unprecedented collection of field-derived mosquitoes in South of France. We identified the presence of the V1016G allele in 14 sites. The V1016G allele was predominantly found in South-East France close to the Italian border with two additional isolated sites close to Bordeaux and Marmande. All mosquitoes were heterozygous for this mutation and should not be phenotypically resistant to pyrethroid insecticide. Four other mutations were identified in our targeted genomic sequence: I1532T, M1006L, M1586L, M995L. Sequencing a section of maternally inherited mitochondrial genome confirmed that the spread of Ae. albopictus in France originated from founders with haplogroup A1. These findings contribute to the broader understanding of resistance dynamics in Europe and can inform targeted approaches to mitigate the impact of resistance on vector control.\n",
  "wordCount" : "2540",
  "inLanguage": "en",
  "datePublished": "2023-08-06T10:36:48Z",
  "dateModified": "2023-08-06T10:36:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on August 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.551959">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.551959" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.551959">
        <p class="paperTitle">Interpreting SNP heritability in admixed populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.551959" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.551959" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, J.; Basu, S.; Shriver, M. D.; Zaidi, A. A.</p>
        <p class="info">Score: 19.9, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.551959' target='https://doi.org/10.1101/2023.08.04.551959'> 10.1101/2023.08.04.551959</a></p>
        <p class="abstract">SNP heritability (h2snp) is defined as the proportion of phenotypic variance explained by genotyped SNPs and is believed to be a lower bound of heritability (h2), being equal to it if all causal variants are known. Despite the simple intuition behind h2snp, its interpretation and equivalence to h2 is unclear, particularly in the presence of population structure and assortative mating. It is well known that population structure can lead to inflation in h2snp estimates. Here we use analytical theory and simulations to demonstrate that estimates of h2snp are not guaranteed to be equal to h2 in admixed populations, even in the absence of confounding and even if the causal variants are known. We interpret this discrepancy arising not because the estimate is biased, but because the estimand itself as defined under the random effects model may not be equal to h2. The model assumes that SNP effects are uncorrelated which may not be true, even for unlinked loci in admixed and structured populations, leading to over- or under-estimates of h2snp relative to h2. For the same reason, local ancestry heritability (hgamma) may also not be equal to the variance explained by local ancestry in admixed populations. We describe the quantitative behavior of h2snp and h2gamma as a function of admixture history and the genetic architecture of the trait and discuss its implications for genome-wide association and polygenic prediction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551265" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551265">
        <p class="paperTitle">Local population structure in Cambridgeshire during the Roman occupation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scheib, C. L.; Hui, R.; Rose, A. K.; Solnik, A.; D&#39;Atanasio, E.; Inskip, S. A.; Cessford, C.; Griffith, S. J.; Wiseman, R.; Neil, B.; Biers, T.; Harknett, S.-J.; Sasso, S.; Biagini, S. A.; Runfeldt, G.; Duhig, C.; Evans, C.; O&#39;Connell, T. C.; Metspalu, M.; Millett, M. J.; Robb, J. E.; Kivisild, T.</p>
        <p class="info">Score: 18.0, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551265' target='https://doi.org/10.1101/2023.07.31.551265'> 10.1101/2023.07.31.551265</a></p>
        <p class="abstract">The Roman period saw the empire expand across Europe and the Mediterranean, including much of what is today the United Kingdom. While there is written evidence of high mobility into and out of Britain for administrators, traders and the military, the impact of imperialism on local population structure is invisible in the textual record. The extent of genetic change that occurred in Britain before the Early Medieval Period and how closely linked by genetic kinship the local populations were, remains underexplored. Here, using genome-wide data from 52 ancient individuals from Cambridgeshire, we show low levels of genetic ancestry differentiation between Romano-British sites and lower levels of runs of homozygosity over 4 centimorgans (cM than in the Bronze Age and Neolithic. We find fourteen cases of genetic relatedness within and one between sites without evidence of patrilineal dominance and one case of temporary mobility within a family unit during the Late Romano-British period. We also show that the modern patterns of genetic ancestry composition in Modern Britain emerged after the Roman period.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.551029">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.551029" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.551029">
        <p class="paperTitle">Mitochondrial Genome Diversity across the Subphylum Saccharomycotina</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.551029" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.551029" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wolters, J. F.; LaBella, A.; Opulente, D.; Rokas, A.; Hittinger, C. T.</p>
        <p class="info">Score: 14.9, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.551029' target='https://doi.org/10.1101/2023.07.28.551029'> 10.1101/2023.07.28.551029</a></p>
        <p class="abstract">Eukaryotic life depends on the functional elements encoded by both the nuclear genome and organellar genomes, such as those contained within the mitochondria. The content, size, and structure of the mitochondrial genome varies across organisms with potentially large implications for phenotypic variance and resulting evolutionary trajectories. Among yeasts in the subphylum Saccharomycotina, extensive differences have been observed in various species relative to the model yeast Saccharomyces cerevisiae, but mitochondrial genome sampling across many groups has been scarce, even as hundreds of nuclear genomes have become available. By extracting mitochondrial assemblies from existing short-read genome sequence datasets, we have greatly expanded both the number of available genomes and the coverage across sparsely sampled clades. Comparison of 353 yeast mitochondrial genomes revealed that, while size and GC content were fairly consistent across species, those in the genera Metschnikowia and Saccharomyces trended larger, while several species in the order Saccharomycetales, which includes S. cerevisiae, exhibited lower GC content. Extreme examples for both size and GC content were scattered throughout the subphylum. All mitochondrial genomes shared a core set of protein-coding genes for Complexes III, IV, and V, but they varied in the presence or absence of mitochondrially-encoded canonical Complex I genes. We traced the loss of Complex I genes to a major event in the ancestor of the orders Saccharomycetales and Saccharomycodales, but we also observed several independent losses in the orders Phaffomycetales, Pichiales, and Dipodascales. In contrast to prior hypotheses based on smaller-scale datasets, comparison of evolutionary rates in protein-coding genes showed no bias towards elevated rates among aerobically fermenting (Crabtree/Warburg-positive) yeasts. Mitochondrial introns were widely distributed, but they were highly enriched in some groups. The majority of mitochondrial introns were poorly conserved within groups, but several were shared within groups, between groups, and even across taxonomic orders, which is consistent with horizontal gene transfer, likely involving homing endonucleases acting as selfish elements. As the number of available fungal nuclear genomes continues to expand, the methods described here to retrieve mitochondrial genome sequences from these datasets will prove invaluable to ensuring that studies of fungal mitochondrial genomes keep pace with their nuclear counterparts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550074">
        <p class="paperTitle">isGWAS: ultra-high-throughput, scalable and equitable inference of genetic associations with disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Foley, C. N.; Kuncheva, Z.; Marioni, R.; Runz, H.; Sun, B.</p>
        <p class="info">Score: 39.2, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550074' target='https://doi.org/10.1101/2023.07.21.550074'> 10.1101/2023.07.21.550074</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have proven a powerful tool for human geneticists to generate biological insights or hypotheses for drug discovery. Nevertheless, a dependency on sensitive individual-level data together with ever-increasing cohort sample sizes, numbers of variants and phenotypes studied put a strain on existing algorithms, limiting the GWAS approach from maximising potential. Here we present in-silico GWAS (isGWAS), a uniquely scalable algorithm to infer regression parameters in case-control GWAS from cohort-level summary data. For any sample size, isGWAS computes a variant-disease association parameter in [~]1 millisecond, or [~]11m variants in UK-Biobank within [~]4 minutes ([~]1500-fold faster than state-of-the-art). Extensive simulations and empirical tests demonstrate that isGWAS results are highly comparable to traditional regression-based approaches. We further introduce a heuristic re-sampling algorithm, leapfrog re-sampler (LRS), to extrapolate association results to semi-virtually enlarged cohorts. Owing to significant computational gains we anticipate a broad use of isGWAS and LRS which are customizable on a web interface.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551297">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551297" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551297">
        <p class="paperTitle">The Interferon γ Pathway Enhances Pluripotency and X-Chromosome Reactivation in iPSC Reprogramming</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551297" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551297" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Barrero, M.; Lopez-Rubio, A. V.; Lazarenkov, A.; Blanco, E.; Bauer, M.; Palma, L. G.; Bigas, A.; Di Croce, L.; Sardina, J. L.; Payer, B.</p>
        <p class="info">Score: 7.9, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551297' target='https://doi.org/10.1101/2023.07.31.551297'> 10.1101/2023.07.31.551297</a></p>
        <p class="abstract">Reprogramming somatic cells into induced pluripotent stem cells (iPSCs) requires activation of the pluripotency network and resetting of the epigenome by erasing the epigenetic memory of the somatic state. In female mouse cells, a critical epigenetic reprogramming step is the reactivation of the inactive X chromosome. Despite its importance, a systematic understanding of the regulatory networks linking pluripotency and X-reactivation is missing. Here we reveal the pathways important for iPSC reprogramming and X-reactivation using a genome-wide CRISPR screen. In particular, we discover that activation of the interferon {gamma} (IFN{gamma}) pathway early during reprogramming accelerates pluripotency acquisition and X-reactivation. IFN{gamma} stimulates STAT3 signaling and the pluripotency network and leads to enhanced TET-mediated DNA demethylation, which consequently boosts X-reactivation. We therefore gain a mechanistic understanding of the role of IFN{gamma} in reprogramming and X-reactivation and provide a comprehensive resource of the molecular networks involved in these processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548129">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548129" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548129">
        <p class="paperTitle">Pax3 lineage-specific deletion of Gpr161 is associated with spinal neural tube and craniofacial malformations during embryonic development.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548129" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548129" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, S. E.; Chothani, P.; Shaik, R.; Pollard, W.; Finnell, R. H.</p>
        <p class="info">Score: 8.7, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548129' target='https://doi.org/10.1101/2023.07.07.548129'> 10.1101/2023.07.07.548129</a></p>
        <p class="abstract">Shh signaling is the morphogen signaling that regulates embryonic craniofacial and neural tube development. G protein-coupled receptor 161 (Gpr161) is a negative regulator of Shh signaling, and its inactivation in mice results in embryo lethality with craniofacial and neural tube defects (NTDs). However, the structural defects of later embryonic stages in Gpr161 null mice and cell lineages underlying abnormalities were not well characterized due to their limited lifespan. We found the Pax3 lineage-specific deletion of Gpr161 in mice presented with tectal hypertrophy (anterior dorsal neuroepithelium), cranial vault and facial bone hypoplasia (cranial neural crest (CNC)), vertebral abnormalities (somite), and the closed form of spina bifida (posterior dorsal neuroepithelium). In particular, the closed form of spina bifida is partly due to the reduced Pax3 and Cdx4 gene expression of the posterior dorsal neural tubes of Gpr161 mutant embryos involving decreased Wnt signaling whereas Shh signaling was increased. This study provides the novel role of Gpr161 in the posterior neural tube development and confirms its role on CNC- and somite-derived skeletogenesis and midbrain morphogenesis in mice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551571">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551571" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551571">
        <p class="paperTitle">R2ROC: An efficient method of comparing two or more correlated AUC from out-of-sample prediction using polygenic scores</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551571" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551571" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Momin, M. M.; Wray, N. R.; Lee, S. H.</p>
        <p class="info">Score: 4.9, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551571' target='https://doi.org/10.1101/2023.08.01.551571'> 10.1101/2023.08.01.551571</a></p>
        <p class="abstract">Polygenic risk scores (PRSs) enable early prediction of disease risk. Evaluating PRS performance for binary traits commonly relies on the area under the receiver operating characteristic curve (AUC). However, the widely used DeLongs method for comparative significance tests suffer from limitations, including computational time and the lack of a one-to-one mapping between test statistics based on AUC and R2. To overcome these limitations, we propose a novel approach that leverages the Delta method to derive the variance and covariance of AUC values, enabling a comprehensive and efficient comparative significance test. Our approach offers notable advantages over DeLongs method, including reduced computation time (up to 150-fold), making it suitable for large-scale analyses and ideal for integration into machine learning frameworks. Furthermore, our method allows for a direct one-to-one mapping between AUC and R2 values for comparative significance tests, providing enhanced insights into the relationship between these measures and facilitating their interpretation. We validated our proposed approach through simulations and applied it to real data comparing PRSs for diabetes and coronary artery disease (CAD) prediction in a cohort of 28,880 European individuals. The PRSs were derived using genome-wide association study summary statistics from two distinct sources. Our approach enabled a comprehensive and informative comparison of the PRSs, shedding light on their respective predictive abilities for diabetes and CAD. This advancement contributes to the assessment of genetic risk factors and personalized disease prediction, supporting better healthcare decision-making.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.550904">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.550904" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.550904">
        <p class="paperTitle">Zebrafish models of Mucopolysaccharidosis types IIIA, B, &amp; C show hyperactivity and changes in oligodendrocyte state</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.550904" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.550904" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gerken, E.; Ahmad, S.; Barthelson, K.; Lardelli, M.</p>
        <p class="info">Score: 4.1, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.550904' target='https://doi.org/10.1101/2023.08.02.550904'> 10.1101/2023.08.02.550904</a></p>
        <p class="abstract">Sanfilippo syndrome childhood dementia, also known as mucopolysaccharidosis type III (MPS III), is a rare inherited lysosomal storage disorder. Subtypes of MPS III are caused by deficiencies in one of four enzymes required for degradation of the glycosaminoglycan heparan sulfate (HS). An inability to degrade HS leads to progressive neurodegeneration and death in the second or third decades of life. Knowledge of MPS III pathogenesis is incomplete, and no effective therapies exist. We generated the hypomorphic mutations sgshS387Lfs, nagluA603Efs and hgsnatG577Sfsin the endogenous zebrafish genes orthologous to human SGSH, NAGLU, and HGSNAT that are loci for mutations causing MPS III subtypes MPS IIIA, B and C respectively. Our models display the primary MPS III disease signature of significant brain accumulation of HS, while behavioural analyses support hyperactivity phenotypes. Brain transcriptome analysis revealed changes related to lysosomal, glycosaminoglycan, immune system and iron homeostasis biology in all three models but also distinct differences in brain transcriptome state between models. The transcriptome analysis also indicated marked disturbance of the oligodendrocyte cell state in the brains of MPS IIIA, B and C zebrafish, supporting that effects on this cell type are an early and consistent characteristic of MPS III. Overall, our zebrafish models recapture key characteristics of the human disease and phenotypes seen in mouse models. Our models will allow exploitation of the zebrafishs extreme fecundity and accessible anatomy to dissect the pathological mechanisms both common and divergent between the MPS IIIA, B, and C subtypes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551399">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551399" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551399">
        <p class="paperTitle">The neurotrophic factor MANF regulates autophagic flux and lysosome function to promote proteostasis in C. elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551399" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551399" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, S. K. B.; Gupta, B. P.</p>
        <p class="info">Score: 3.4, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551399' target='https://doi.org/10.1101/2023.07.31.551399'> 10.1101/2023.07.31.551399</a></p>
        <p class="abstract">The neurotrophic factor, Mesencephalic Astrocyte Derived Neurotrophic Factor (MANF), promotes neuronal health and survival. MANF is conserved in metazoans and necessary for the protection of dopaminergic neurons. The neuroprotective role of MANF involves regulation of the ER unfolded protein response (ER-UPR). Recent work has shown that MANF also functions in many other tissues and affects autophagy genes. Previously, we reported that C. elegans manf-1 mutants show defects including increased ER stress, -Synuclein aggregation, and age-dependent dopaminergic neurodegeneration. In this study, we have further investigated manf-1s function and describe its mechanism of action in regulating stress response, protein aggregation and aging. Our results showed that exposure to ER stress-inducing agents caused manf-1 transcription and protein expression to be upregulated. Consistent with the protective role of manf-1, mutant animals showed greater sensitivity to ER stress and died prematurely. We generated manf-1p::manf-1::mCherry (MANF-1OE) and hsp::manf-1 transgenic strains overexpressing MANF and found that these animals had an extended lifespan, reduced protein aggregation, and increased neuronal health. The in vivo analysis of MANF-1 expression revealed that the protein was secreted and present in many tissues. Subcellular studies showed that MANF-1 was localized to lysosomes. We also observed that MANF-1OE worms had reduced LGG-1 and SQST-1/p62 levels, and increased expression and nuclear localization of the autophagy transcription factor hlh-30/TFEB. Thus, MANF-1 regulates protein homeostasis through increased lysosome activity and autophagic flux. Collectively, our work provides a novel understanding of MANF-1 function in promoting neuronal health, stress response, proteostasis, and lifespan of animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552101">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552101" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552101">
        <p class="paperTitle">Multiplexed amplicon sequencing reveals the heterogeneous spatial distribution of pyrethroid resistance mutations in Aedes albopictus mosquito populations in Southern France.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552101" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552101" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fontaine, A.; Mignotte, A.; Lacour, G.; Nguyen, A.; Gomez, N.; Chanaud, L.; L&#39;Ambert, G.; Briolant, S.</p>
        <p class="info">Score: 3.1, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552101' target='https://doi.org/10.1101/2023.08.05.552101'> 10.1101/2023.08.05.552101</a></p>
        <p class="abstract">The risk of mosquito-borne diseases transmission is moving fast toward temperate climates with the colonization and proliferation of the Asian tiger mosquito vector Aedes albopictus and the rapid and mass transport of passengers returning from tropical regions where the viruses are endemic. The prevention of major Aedes-borne viruses heavily relies on the use of insecticides for vector control, mainly pyrethroids In Europe. High-throughput molecular assays can provide a cost-effective surrogate to phenotypic insecticide resistance assays when mutations have been previously linked to a resistance phenotype. Here, we screened for the spatial distribution of kdr mutations at a large scale using a two-step approach based on multiplexed amplicon sequencing and an unprecedented collection of field-derived mosquitoes in South of France. We identified the presence of the V1016G allele in 14 sites. The V1016G allele was predominantly found in South-East France close to the Italian border with two additional isolated sites close to Bordeaux and Marmande. All mosquitoes were heterozygous for this mutation and should not be phenotypically resistant to pyrethroid insecticide. Four other mutations were identified in our targeted genomic sequence: I1532T, M1006L, M1586L, M995L. Sequencing a section of maternally inherited mitochondrial genome confirmed that the spread of Ae. albopictus in France originated from founders with haplogroup A1. These findings contribute to the broader understanding of resistance dynamics in Europe and can inform targeted approaches to mitigate the impact of resistance on vector control.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
