<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Gene-environment correlation: The role of family environment in academic development
Authors: Zhou, Q.; Gidziela, A.; Allegrini, A. G.; Cheesman, R.; Wertz, J.; Maxwell, J.; Plomin, R.; Rimfeld, K.; Malanchini, M.
Score: 16.4, Published: 2024-01-08 DOI: 10.1101/2024.01.05.574339
Academic achievement is partly heritable and highly polygenic. However, genetic effects on academic achievement are not independent of environmental processes. We investigated whether aspects of the family environment mediated genetic effects on academic achievement across development.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Gene-environment correlation: The role of family environment in academic development
Authors: Zhou, Q.; Gidziela, A.; Allegrini, A. G.; Cheesman, R.; Wertz, J.; Maxwell, J.; Plomin, R.; Rimfeld, K.; Malanchini, M.
Score: 16.4, Published: 2024-01-08 DOI: 10.1101/2024.01.05.574339
Academic achievement is partly heritable and highly polygenic. However, genetic effects on academic achievement are not independent of environmental processes. We investigated whether aspects of the family environment mediated genetic effects on academic achievement across development." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Gene-environment correlation: The role of family environment in academic development
Authors: Zhou, Q.; Gidziela, A.; Allegrini, A. G.; Cheesman, R.; Wertz, J.; Maxwell, J.; Plomin, R.; Rimfeld, K.; Malanchini, M.
Score: 16.4, Published: 2024-01-08 DOI: 10.1101/2024.01.05.574339
Academic achievement is partly heritable and highly polygenic. However, genetic effects on academic achievement are not independent of environmental processes. We investigated whether aspects of the family environment mediated genetic effects on academic achievement across development."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Gene-environment correlation: The role of family environment in academic development\nAuthors: Zhou, Q.; Gidziela, A.; Allegrini, A. G.; Cheesman, R.; Wertz, J.; Maxwell, J.; Plomin, R.; Rimfeld, K.; Malanchini, M.\nScore: 16.4, Published: 2024-01-08 DOI: 10.1101/2024.01.05.574339\nAcademic achievement is partly heritable and highly polygenic. However, genetic effects on academic achievement are not independent of environmental processes. We investigated whether aspects of the family environment mediated genetic effects on academic achievement across development.",
  "keywords": [
    
  ],
  "articleBody": " Gene-environment correlation: The role of family environment in academic development\nAuthors: Zhou, Q.; Gidziela, A.; Allegrini, A. G.; Cheesman, R.; Wertz, J.; Maxwell, J.; Plomin, R.; Rimfeld, K.; Malanchini, M.\nScore: 16.4, Published: 2024-01-08 DOI: 10.1101/2024.01.05.574339\nAcademic achievement is partly heritable and highly polygenic. However, genetic effects on academic achievement are not independent of environmental processes. We investigated whether aspects of the family environment mediated genetic effects on academic achievement across development. Our sample included 5,151 children who participated in the Twins Early Development Study, as well as their parents and teachers. Data on academic achievement and family environments were available at ages 7, 9, 12 and 16. We computed educational attainment polygenic scores (PGS) and further separated genetic effects into cognitive and noncognitive PGS. Three core findings emerged. First, aspects of the family environment, but not the wider neighbourhood context, consistently mediated the PGS effects on achievement across development, accounting for up to 34.3% of the total effect. Family characteristics mattered beyond socio-economic status. Second, family environments were more robustly linked to noncognitive PGS effects on academic achievement than cognitive PGS effects. Third, when we investigated whether environmental mediation effects could also be observed when considering differences between siblings, adjusting for family fixed effects, we found that environmental mediation was nearly exclusively observed between families. This is consistent with the proposition that family environmental contexts contribute to academic development via passive gene-environment correlation processes. Our results show how parents shape environments that foster their children's academic development partly based on their own genetic disposition, particularly towards noncognitive skills.\npIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish\nAuthors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.\nScore: 57.6, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570868\nStandard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis. However, while the phiC31 system functions in zebrafish, validated loci carrying attP-based landing or safe harbor sites suitable for universal transgenesis applications in zebrafish have not been established. Here, using CRISPR-Cas9, we converted two well-validated single insertion Tol2-based zebrafish transgenes with long-standing genetic stability into two attP landing sites, called phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET). Generating fluorescent reporters, loxP-based Switch lines, CreERT2 drivers, and gene-regulatory variant reporters in the pIGLET14a and pIGLET24b landing site alleles, we document their suitability for transgenesis applications across cell types and developmental stages. For both landing sites, we routinely achieve 25-50% germline transmission of targeted transgene integrations, drastically reducing the number of required animals and necessary resources to generate individual transgenic lines. We document that phiC31 integrase-based transgenesis into pIGLET14a and pIGLET24b reproducibly results in representative reporter expression patterns in injected F0 zebrafish embryos suitable for enhancer discovery and qualitative and quantitative comparison of gene-regulatory element variants. Taken together, our new phiC31 integrase-based transgene landing sites establish reproducible, targeted zebrafish transgenesis for numerous applications while greatly reducing the workload of generating new transgenic zebrafish lines. SHORT ABSTRACTTargeted transgenesis into pre-established, \"safe harbor,\" landing sites remains a missing technique in zebrafish. Here, we established phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET) by CRISPR-Cas9-based conversion of two previously validated Tol2 transgenes into attP sites. phiC31-mediated transgenesis into our pIGLET14a and pIGLET24b attP landing sites results in 25-50% germline transmission efficiency and quantifiable transgene activity. Our landing sites are suitable for reproducible, routine zebrafish transgenesis for diverse applications.\nRare genetic variation in Fibronectin 1 (FN1) protects against APOEe4 in Alzheimer's disease\nAuthors: Bhattarai, P.; Gunasekaran, T. I.; Reyes-Dumeyer, D.; Juelich, D.; Tayran, H.; Yilmaz, E.; Flaherty, D.; Rafael, L.; Martin, M.; Diones, R.; Patricia, R.; Ertekin-Taner, N.; Teich, A. F.; Dennis, D. W.; Scott, H.; Mayeux, R.; Kizil, C.; Vardarajan, B. N.\nScore: 14.7, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573895\nThe risk of developing Alzheimers disease (AD) significantly increases in individuals carrying the APOE{varepsilon}4 allele. Elderly cognitively healthy individuals with APOE{varepsilon}4 also exist, suggesting the presence of cellular mechanisms that counteract the pathological effects of APOE{varepsilon}4; however, these mechanisms are unknown. We hypothesized that APOE{varepsilon}4 carriers without dementia might carry genetic variations that could protect them from developing APOE{varepsilon}4-mediated AD pathology. To test this, we leveraged whole genome sequencing (WGS) data in National Institute on Aging Alzheimers Disease Family Based Study (NIA-AD FBS), Washington Heights/Inwood Columbia Aging Project (WHICAP), and Estudio Familiar de Influencia Genetica en Alzheimer (EFIGA) cohorts and identified potentially protective variants segregating exclusively among unaffected APOE{varepsilon}4 carriers. In homozygous unaffected carriers above 70 years old, we identified 510 rare coding variants. Pathway analysis of the genes harboring these variants showed significant enrichment in extracellular matrix (ECM)-related processes, suggesting protective effects of functional modifications in ECM proteins. We prioritized two genes that were highly represented in the ECM-related gene ontology terms, (FN1) and collagen type VI alpha 2 chain (COL6A2) and are known to be expressed at the blood-brain barrier (BBB), for postmortem validation and in vivo functional studies. The FN1 and COL6A2 protein levels were increased at the BBB in APOE{varepsilon}4 carriers with AD. Brain expression of cognitively unaffected homozygous APOE{varepsilon}4 carriers had significantly lower FN1 deposition and less reactive gliosis compared to homozygous APOE{varepsilon}4 carriers with AD, suggesting that FN1 might be a downstream driver of APOE{varepsilon}4-mediated AD-related pathology and cognitive decline. To validate our findings, we used zebrafish models with loss-of-function (LOF) mutations in fn1b - the ortholog for human FN1. We found that fibronectin LOF reduced gliosis, enhanced gliovascular remodeling and potentiated the microglial response, suggesting that pathological accumulation of FN1 could impair toxic protein clearance, which is ameliorated with FN1 LOF. Our study suggests vascular deposition of FN1 is related to the pathogenicity of APOE{varepsilon}4, LOF variants in FN1 may reduce APOE{varepsilon}4-related AD risk, providing novel clues to potential therapeutic interventions targeting the ECM to mitigate AD risk.\nMultiple DNA repair pathways prevent acetaldehyde-induced mutagenesis in yeast\nAuthors: Porcher, L.; Vijayraghavan, S.; McCollum, J.; Mieczkowski, P. A.; Saini, N.\nScore: 10.4, Published: 2024-01-08 DOI: 10.1101/2024.01.07.574575\nAcetaldehyde is the primary metabolite of alcohol and is present in many environmental sources including tobacco smoke. Acetaldehyde is genotoxic, whereby it can form DNA adducts and lead to mutagenesis. Individuals with defects in acetaldehyde clearance pathways have increased susceptibility to alcohol-associated cancers. Moreover, a mutation signature specific to acetaldehyde exposure is widespread in alcohol and smoking-associated cancers. However, the pathways that repair acetaldehyde-induced DNA damage and thus prevent mutagenesis are vaguely understood. Here, we used Saccharomycescerevisiae to systematically delete genes in each of the major DNA repair pathways to identify those that alter acetaldehyde-induced mutagenesis. We found that deletion of the nucleotide excision repair (NER) genes, RAD1 or RAD14, led to an increase in mutagenesis upon acetaldehyde exposure. Acetaldehyde-induced mutations were dependent on translesion synthesis as well as DNA inter-strand crosslink (ICL) repair in Drad1 strains. Moreover, whole genome sequencing of the mutated isolates demonstrated an increase in C?A changes coupled with an enrichment of gCn?A changes in the acetaldehyde-treated Drad1 isolates. The gCn?A mutation signature has been shown to be diagnostic of acetaldehyde exposure in yeast and in human cancers. We also demonstrated that the deletion of the two DNA-protein crosslink (DPC) repair proteases, WSS1 and DDI1, also led to increased acetaldehyde-induced mutagenesis. Defects in base excision repair (BER) led to a mild increase in mutagenesis, while defects in mismatch repair (MMR), homologous recombination repair (HR) and post replicative repair pathways did not impact mutagenesis upon acetaldehyde exposure. Our results in yeast were further corroborated upon analysis of whole exome sequenced liver cancers, wherein, tumors with defects in ERCC1 and ERCC4 (NER), FANCD2 (ICL repair) or SPRTN (DPC repair) carried a higher gCn?A mutation load than tumors with no deleterious mutations in these genes. Our findings demonstrate that multiple DNA repair pathways protect against acetaldehyde-induced mutagenesis.\nDetailed mechanisms for unintended large DNA deletions with CRISPR, base editors, and prime editors\nAuthors: Hwang, G.-h.; Kim, S.-H.; Oh, M.; Kim, S.; Habib, O.; Jang, H.-K.; Kim, H. S.; Kim, C. H.; Kim, S.; Bae, S.\nScore: 9.0, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574288\nCRISPR-Cas9 nucleases are versatile tools for genetic engineering cells and function by producing targeted double-strand breaks (DSBs) in the DNA sequence. However, the unintended production of large deletions (\u003e100 bp) represents a challenge to the effective application of this genome-editing system. We optimized a long-range amplicon sequencing system and developed a k-mer sequence-alignment algorithm to simultaneously detect small DNA alteration events and large DNA deletions. With this workflow, we determined that CRISPR-Cas9 induced large deletions at varying frequencies in cancer cell lines, stem cells, and primary T cells. With CRISPR interference screening, we determined that end resection and the subsequent TMEJ [DNA polymerase theta-mediated end joining] repair process produce most large deletions. Furthermore, base editors and prime editors also generated large deletions despite employing mutated Cas9 \"nickases\" that produce single-strand breaks. Our findings reveal an important limitation of current genome-editing tools and identify strategies for mitigating unwanted large deletion events.\nRewriting regulatory DNA to dissect and reprogram gene expression\nAuthors: Martyn, G. E.; Montgomery, M. T.; Jones, H.; Guo, K.; Doughty, B. R.; Linder, J.; Chen, Z.; Cochran, K.; Lawrence, K. A.; Munson, G.; Pampari, A.; Fulco, C. P.; Kelley, D. R.; Lander, E. S.; Kundaje, A.; Engreitz, J. M.\nScore: 54.4, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572268\nRegulatory DNA sequences within enhancers and promoters bind transcription factors to encode cell type-specific patterns of gene expression. However, the regulatory effects and programmability of such DNA sequences remain difficult to map or predict because we have lacked scalable methods to precisely edit regulatory DNA and quantify the effects in an endogenous genomic context. Here we present an approach to measure the quantitative effects of hundreds of designed DNA sequence variants on gene expression, by combining pooled CRISPR prime editing with RNA fluorescence in situ hybridization and cell sorting (Variant-FlowFISH). We apply this method to mutagenize and rewrite regulatory DNA sequences in an enhancer and the promoter of PPIF in two immune cell lines. Of 672 variant-cell type pairs, we identify 497 that affect PPIF expression. These variants appear to act through a variety of mechanisms including disruption or optimization of existing transcription factor binding sites, as well as creation of de novo sites. Disrupting a single endogenous transcription factor binding site often led to large changes in expression (up to -40% in the enhancer, and -50% in the promoter). The same variant often had different effects across cell types and states, demonstrating a highly tunable regulatory landscape. We use these data to benchmark performance of sequence-based predictive models of gene regulation, and find that certain types of variants are not accurately predicted by existing models. Finally, we computationally design 185 small sequence variants ([\u0026le;]10 bp) and optimize them for specific effects on expression in silico. 84% of these rationally designed edits showed the intended direction of effect, and some had dramatic effects on expression (-100% to +202%). Variant-FlowFISH thus provides a powerful tool to map the effects of variants and transcription factor binding sites on gene expression, test and improve computational models of gene regulation, and reprogram regulatory DNA.\nComplementation testing identifies causal genes at quantitative trait loci underlying fear related behavior\nAuthors: Chen, P. B.; Chen, R.; LaPierre, N. R. S.; Chen, Z.; Mefford, J.; Marcus, E.; Heffel, M.; Soto, D. C.; Ernst, J.; Luo, C.; Flint, J.\nScore: 2.1, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574060\nKnowing the genes involved in quantitative traits provides a critical entry point to understanding the biological bases of behavior, but there are very few examples where the pathway from genetic locus to behavioral change is known. Here we address a key step towards that goal by deploying a test that directly queries whether a gene mediates the effect of a quantitative trait locus (QTL). To explore the role of specific genes in fear behavior, we mapped three fear-related traits, tested fourteen genes at six QTLs, and identified six genes. Four genes, Lsamp, Ptprd, Nptx2 and Sh3gl, have known roles in synapse function; the fifth gene, Psip1, is a transcriptional co-activator not previously implicated in behavior; the sixth is a long non-coding RNA 4933413L06Rik with no known function. Single nucleus transcriptomic and epigenetic analyses implicated excitatory neurons as likely mediating the genetic effects. Surprisingly, variation in transcriptome and epigenetic modalities between inbred strains occurred preferentially in excitatory neurons, suggesting that genetic variation is more permissible in excitatory than inhibitory neuronal circuits. Our results open a bottleneck in using genetic mapping of QTLs to find novel biology underlying behavior and prompt a reconsideration of expected relationships between genetic and functional variation.\nDNA-PKcs Suppresses Illegitimate Chromosome Rearrangements\nAuthors: Wang, J.; Sadeghi, C.; Frock, R. L.\nScore: 3.1, Published: 2024-01-01 DOI: 10.1101/2023.12.30.573736\nTwo DNA repair pathways, non-homologous end joining (NHEJ) and alternative end joining (A-EJ), are involved in V(D)J recombination and chromosome translocation. Previous studies reported distinct repair mechanisms for chromosome translocation, with NHEJ involved in human and A-EJ in mice predominantly. NHEJ depends on DNA-PKcs, a critical partner in synapsis formation and downstream component activation. While DNA-PKcs inhibition promotes chromosome translocations harboring microhomologies in mice, its synonymous effect in humans is not known. We find partial DNA-PKcs inhibition in human cells leads to increased translocations and the continued involvement of a dampened NHEJ. In contrast, complete DNA-PKcs inhibition substantially increased microhomology-mediated end joining (MMEJ), thus bridging the two different translocation mechanisms between human and mice. Similar to a previous study on Ku70 deletion, DNA-PKcs deletion in G1/G0-phase mouse progenitor B cell lines, significantly impairs V(D)J recombination and generated higher rates of translocations as a consequence of dysregulated coding and signal end joining. Genetic DNA-PKcs inhibition suppresses NHEJ involvement entirely, with repair phenotypically resembling Ku70-deficient A-EJ. In contrast, we find DNA-PKcs necessary in generating the near-excusive MMEJ associated with Lig4 deficiency. Our study underscores DNA-PKcs in suppressing illegitimate chromosome rearrangement while also contributing to MMEJ in both species.\nExtreme restructuring of cis-regulatory regions controlling a deeply conserved plant stem cell regulator\nAuthors: Ciren, D.; Zebell, S.; Lippman, Z.\nScore: 24.8, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572550\nA striking paradox is that genes with conserved protein sequence, function and expression pattern over deep time often exhibit extremely divergent cis-regulatory sequences. It remains unclear how such drastic cis-regulatory evolution across species allows preservation of gene function, and to what extent these differences influence how cis-regulatory variation arising within species impacts phenotypic change. Here, we investigated these questions using a plant stem cell regulator conserved in expression pattern and function over [~]125 million years. Using in-vivo genome editing in two distantly related models, Arabidopsis thaliana (Arabidopsis) and Solanum lycopersicum (tomato), we generated over 70 deletion alleles in the upstream and downstream regions of the stem cell repressor gene CLAVATA3 (CLV3) and compared their individual and combined effects on a shared phenotype, the number of carpels that make fruits. We found that sequences upstream of tomato CLV3 are highly sensitive to even small perturbations compared to its downstream region. In contrast, Arabidopsis CLV3 function is tolerant to severe disruptions both upstream and downstream of the coding sequence. Combining upstream and downstream deletions also revealed a different regulatory outcome. Whereas phenotypic enhancement from adding downstream mutations was predominantly weak and additive in tomato, mutating both regions of Arabidopsis CLV3 caused substantial and synergistic effects, demonstrating distinct distribution and redundancy of functional cis-regulatory sequences. Our results demonstrate remarkable malleability in cis-regulatory structural organization of a deeply conserved plant stem cell regulator and suggest that major reconfiguration of cis-regulatory sequence space is a common yet cryptic evolutionary force altering genotype-to-phenotype relationships from regulatory variation in conserved genes. Finally, our findings underscore the need for lineage-specific dissection of the spatial architecture of cis-regulation to effectively engineer trait variation from conserved productivity genes in crops. Author summaryWe investigated the evolution of cis-regulatory elements (CREs) and their interactions in the regulation of a plant stem cell regulator gene, CLAVATA3 (CLV3), in Arabidopsis and tomato. Despite diverging [~]125 million years ago, the function and expression of CLV3 is conserved in these species; however, cis-regulatory sequences upstream and downstream have drastically diverged, preventing identification of conserved non-coding sequences between them. We used CRISPR-Cas9 to engineer dozens of mutations within the cis-regulatory regions of Arabidopsis and tomato CLV3. In tomato, our results show that tomato CLV3 function primarily relies on interactions among CREs in the 5 non-coding region, unlike Arabidopsis CLV3, which depends on a more balanced distribution of functional CREs between the 5 and 3 regions. Therefore, despite a high degree of functional conservation, our study demonstrates divergent regulatory strategies between two distantly related CLV3 orthologs, with substantial alterations in regulatory sequences, their spatial arrangement, and their relative effects on CLV3 regulation. These results suggest that regulatory regions are not only extremely robust to mutagenesis, but also that the sequences underlying this robustness can be lineage-specific for conserved genes, due to the complex and often redundant interactions among CREs that ensure proper gene function amidst large-scale sequence turnover.\nImputation of 3D genome structure by genetic-epigenetic interaction modeling in mice\nAuthors: Kuffler, L.; Fortin, H. J.; Skelly, D. A.; Czechanski, A.; Munger, S. C.; Reinholdt, L. G.; Baker, C. L.; Carter, G. W.\nScore: 2.6, Published: 2024-01-02 DOI: 10.1101/2022.02.07.479436\nGene expression is known to be affected by interactions between local genetic variation and DNA accessibility, with the latter organized into three-dimensional chromatin structures. Analyses of these interactions has previously been limited, obscuring their regulatory context, and the extent to which they occur throughout the genome. Here we undertake a genome-scale analysis of these interactions in a genetically diverse population to systematically identify global genetic-epigenetic interaction, and reveal constraints imposed by chromatin structure. We establish the extent and structure of genotype-by-epigenotype interaction using embryonic stem cells derived from Diversity Outbred mice. This mouse population segregates millions of variants from eight inbred founders, enabling precision genetic mapping with extensive genotypic and phenotypic diversity. With 176 samples profiled for genotype, gene expression, and open chromatin, we used regression modeling to infer genetic-epigenetic interactions on a genome-wide scale. Our results demonstrate that statistical interactions between genetic variants and chromatin accessibility are common throughout the genome. We found that these interactions occur within the local area of the affected gene, and that this locality corresponds to topologically associated domains (TADs). The likelihood of interaction was most strongly defined by the three-dimensional (3D) domain structure rather than linear DNA sequence. We show that stable 3D genome structure is an effective tool to guide searches for regulatory elements and, conversely, that regulatory elements in genetically diverse populations provide a means to infer 3D genome structure. We confirmed this finding with CTCF ChIP-seq that revealed strain-specific binding in the inbred founder mice. In stem cells, open chromatin participating in the most significant regression models demonstrated an enrichment for developmental genes and the TAD-forming CTCF binding complex, providing an opportunity for statistical inference of shifting TAD boundaries operating during early development. These findings provide evidence that genetic and epigenetic factors operate within the context of three- dimensional chromatin structure. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=196 SRC=\"FIGDIR/small/479436v3_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (34K): org.highwire.dtl.DTLVardef@155bda5org.highwire.dtl.DTLVardef@15c039org.highwire.dtl.DTLVardef@1ff063borg.highwire.dtl.DTLVardef@1f24802_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "3179",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574339">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574339" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574339">
        <p class="paperTitle">Gene-environment correlation: The role of family environment in academic development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574339" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574339" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, Q.; Gidziela, A.; Allegrini, A. G.; Cheesman, R.; Wertz, J.; Maxwell, J.; Plomin, R.; Rimfeld, K.; Malanchini, M.</p>
        <p class="info">Score: 16.4, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574339' target='https://doi.org/10.1101/2024.01.05.574339'> 10.1101/2024.01.05.574339</a></p>
        <p class="abstract">Academic achievement is partly heritable and highly polygenic. However, genetic effects on academic achievement are not independent of environmental processes. We investigated whether aspects of the family environment mediated genetic effects on academic achievement across development. Our sample included 5,151 children who participated in the Twins Early Development Study, as well as their parents and teachers. Data on academic achievement and family environments were available at ages 7, 9, 12 and 16. We computed educational attainment polygenic scores (PGS) and further separated genetic effects into cognitive and noncognitive PGS. Three core findings emerged. First, aspects of the family environment, but not the wider neighbourhood context, consistently mediated the PGS effects on achievement across development, accounting for up to 34.3% of the total effect. Family characteristics mattered beyond socio-economic status. Second, family environments were more robustly linked to noncognitive PGS effects on academic achievement than cognitive PGS effects. Third, when we investigated whether environmental mediation effects could also be observed when considering differences between siblings, adjusting for family fixed effects, we found that environmental mediation was nearly exclusively observed between families. This is consistent with the proposition that family environmental contexts contribute to academic development via passive gene-environment correlation processes. Our results show how parents shape environments that foster their children&#39;s academic development partly based on their own genetic disposition, particularly towards noncognitive skills.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570868">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570868" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570868">
        <p class="paperTitle">pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570868" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570868" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.</p>
        <p class="info">Score: 57.6, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570868' target='https://doi.org/10.1101/2023.12.08.570868'> 10.1101/2023.12.08.570868</a></p>
        <p class="abstract">Standard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis. However, while the phiC31 system functions in zebrafish, validated loci carrying attP-based landing or safe harbor sites suitable for universal transgenesis applications in zebrafish have not been established. Here, using CRISPR-Cas9, we converted two well-validated single insertion Tol2-based zebrafish transgenes with long-standing genetic stability into two attP landing sites, called phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET). Generating fluorescent reporters, loxP-based Switch lines, CreERT2 drivers, and gene-regulatory variant reporters in the pIGLET14a and pIGLET24b landing site alleles, we document their suitability for transgenesis applications across cell types and developmental stages. For both landing sites, we routinely achieve 25-50% germline transmission of targeted transgene integrations, drastically reducing the number of required animals and necessary resources to generate individual transgenic lines. We document that phiC31 integrase-based transgenesis into pIGLET14a and pIGLET24b reproducibly results in representative reporter expression patterns in injected F0 zebrafish embryos suitable for enhancer discovery and qualitative and quantitative comparison of gene-regulatory element variants. Taken together, our new phiC31 integrase-based transgene landing sites establish reproducible, targeted zebrafish transgenesis for numerous applications while greatly reducing the workload of generating new transgenic zebrafish lines.

SHORT ABSTRACTTargeted transgenesis into pre-established, &#34;safe harbor,&#34; landing sites remains a missing technique in zebrafish. Here, we established phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET) by CRISPR-Cas9-based conversion of two previously validated Tol2 transgenes into attP sites. phiC31-mediated transgenesis into our pIGLET14a and pIGLET24b attP landing sites results in 25-50% germline transmission efficiency and quantifiable transgene activity. Our landing sites are suitable for reproducible, routine zebrafish transgenesis for diverse applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573895">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573895" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573895">
        <p class="paperTitle">Rare genetic variation in Fibronectin 1 (FN1) protects against APOEe4 in Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573895" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573895" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhattarai, P.; Gunasekaran, T. I.; Reyes-Dumeyer, D.; Juelich, D.; Tayran, H.; Yilmaz, E.; Flaherty, D.; Rafael, L.; Martin, M.; Diones, R.; Patricia, R.; Ertekin-Taner, N.; Teich, A. F.; Dennis, D. W.; Scott, H.; Mayeux, R.; Kizil, C.; Vardarajan, B. N.</p>
        <p class="info">Score: 14.7, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573895' target='https://doi.org/10.1101/2024.01.02.573895'> 10.1101/2024.01.02.573895</a></p>
        <p class="abstract">The risk of developing Alzheimers disease (AD) significantly increases in individuals carrying the APOE{varepsilon}4 allele. Elderly cognitively healthy individuals with APOE{varepsilon}4 also exist, suggesting the presence of cellular mechanisms that counteract the pathological effects of APOE{varepsilon}4; however, these mechanisms are unknown. We hypothesized that APOE{varepsilon}4 carriers without dementia might carry genetic variations that could protect them from developing APOE{varepsilon}4-mediated AD pathology. To test this, we leveraged whole genome sequencing (WGS) data in National Institute on Aging Alzheimers Disease Family Based Study (NIA-AD FBS), Washington Heights/Inwood Columbia Aging Project (WHICAP), and Estudio Familiar de Influencia Genetica en Alzheimer (EFIGA) cohorts and identified potentially protective variants segregating exclusively among unaffected APOE{varepsilon}4 carriers. In homozygous unaffected carriers above 70 years old, we identified 510 rare coding variants. Pathway analysis of the genes harboring these variants showed significant enrichment in extracellular matrix (ECM)-related processes, suggesting protective effects of functional modifications in ECM proteins. We prioritized two genes that were highly represented in the ECM-related gene ontology terms, (FN1) and collagen type VI alpha 2 chain (COL6A2) and are known to be expressed at the blood-brain barrier (BBB), for postmortem validation and in vivo functional studies. The FN1 and COL6A2 protein levels were increased at the BBB in APOE{varepsilon}4 carriers with AD. Brain expression of cognitively unaffected homozygous APOE{varepsilon}4 carriers had significantly lower FN1 deposition and less reactive gliosis compared to homozygous APOE{varepsilon}4 carriers with AD, suggesting that FN1 might be a downstream driver of APOE{varepsilon}4-mediated AD-related pathology and cognitive decline. To validate our findings, we used zebrafish models with loss-of-function (LOF) mutations in fn1b - the ortholog for human FN1. We found that fibronectin LOF reduced gliosis, enhanced gliovascular remodeling and potentiated the microglial response, suggesting that pathological accumulation of FN1 could impair toxic protein clearance, which is ameliorated with FN1 LOF. Our study suggests vascular deposition of FN1 is related to the pathogenicity of APOE{varepsilon}4, LOF variants in FN1 may reduce APOE{varepsilon}4-related AD risk, providing novel clues to potential therapeutic interventions targeting the ECM to mitigate AD risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.574575">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.574575" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.574575">
        <p class="paperTitle">Multiple DNA repair pathways prevent acetaldehyde-induced mutagenesis in yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.574575" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.574575" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Porcher, L.; Vijayraghavan, S.; McCollum, J.; Mieczkowski, P. A.; Saini, N.</p>
        <p class="info">Score: 10.4, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.574575' target='https://doi.org/10.1101/2024.01.07.574575'> 10.1101/2024.01.07.574575</a></p>
        <p class="abstract">Acetaldehyde is the primary metabolite of alcohol and is present in many environmental sources including tobacco smoke. Acetaldehyde is genotoxic, whereby it can form DNA adducts and lead to mutagenesis. Individuals with defects in acetaldehyde clearance pathways have increased susceptibility to alcohol-associated cancers. Moreover, a mutation signature specific to acetaldehyde exposure is widespread in alcohol and smoking-associated cancers. However, the pathways that repair acetaldehyde-induced DNA damage and thus prevent mutagenesis are vaguely understood. Here, we used Saccharomycescerevisiae to systematically delete genes in each of the major DNA repair pathways to identify those that alter acetaldehyde-induced mutagenesis. We found that deletion of the nucleotide excision repair (NER) genes, RAD1 or RAD14, led to an increase in mutagenesis upon acetaldehyde exposure. Acetaldehyde-induced mutations were dependent on translesion synthesis as well as DNA inter-strand crosslink (ICL) repair in Drad1 strains. Moreover, whole genome sequencing of the mutated isolates demonstrated an increase in C?A changes coupled with an enrichment of gCn?A changes in the acetaldehyde-treated Drad1 isolates. The gCn?A mutation signature has been shown to be diagnostic of acetaldehyde exposure in yeast and in human cancers. We also demonstrated that the deletion of the two DNA-protein crosslink (DPC) repair proteases, WSS1 and DDI1, also led to increased acetaldehyde-induced mutagenesis. Defects in base excision repair (BER) led to a mild increase in mutagenesis, while defects in mismatch repair (MMR), homologous recombination repair (HR) and post replicative repair pathways did not impact mutagenesis upon acetaldehyde exposure. Our results in yeast were further corroborated upon analysis of whole exome sequenced liver cancers, wherein, tumors with defects in ERCC1 and ERCC4 (NER), FANCD2 (ICL repair) or SPRTN (DPC repair) carried a higher gCn?A mutation load than tumors with no deleterious mutations in these genes. Our findings demonstrate that multiple DNA repair pathways protect against acetaldehyde-induced mutagenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574288">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574288" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574288">
        <p class="paperTitle">Detailed mechanisms for unintended large DNA deletions with CRISPR, base editors, and prime editors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574288" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574288" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hwang, G.-h.; Kim, S.-H.; Oh, M.; Kim, S.; Habib, O.; Jang, H.-K.; Kim, H. S.; Kim, C. H.; Kim, S.; Bae, S.</p>
        <p class="info">Score: 9.0, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574288' target='https://doi.org/10.1101/2024.01.04.574288'> 10.1101/2024.01.04.574288</a></p>
        <p class="abstract">CRISPR-Cas9 nucleases are versatile tools for genetic engineering cells and function by producing targeted double-strand breaks (DSBs) in the DNA sequence. However, the unintended production of large deletions (&gt;100 bp) represents a challenge to the effective application of this genome-editing system. We optimized a long-range amplicon sequencing system and developed a k-mer sequence-alignment algorithm to simultaneously detect small DNA alteration events and large DNA deletions. With this workflow, we determined that CRISPR-Cas9 induced large deletions at varying frequencies in cancer cell lines, stem cells, and primary T cells. With CRISPR interference screening, we determined that end resection and the subsequent TMEJ [DNA polymerase theta-mediated end joining] repair process produce most large deletions. Furthermore, base editors and prime editors also generated large deletions despite employing mutated Cas9 &#34;nickases&#34; that produce single-strand breaks. Our findings reveal an important limitation of current genome-editing tools and identify strategies for mitigating unwanted large deletion events.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572268">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572268" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572268">
        <p class="paperTitle">Rewriting regulatory DNA to dissect and reprogram gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572268" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572268" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martyn, G. E.; Montgomery, M. T.; Jones, H.; Guo, K.; Doughty, B. R.; Linder, J.; Chen, Z.; Cochran, K.; Lawrence, K. A.; Munson, G.; Pampari, A.; Fulco, C. P.; Kelley, D. R.; Lander, E. S.; Kundaje, A.; Engreitz, J. M.</p>
        <p class="info">Score: 54.4, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572268' target='https://doi.org/10.1101/2023.12.20.572268'> 10.1101/2023.12.20.572268</a></p>
        <p class="abstract">Regulatory DNA sequences within enhancers and promoters bind transcription factors to encode cell type-specific patterns of gene expression. However, the regulatory effects and programmability of such DNA sequences remain difficult to map or predict because we have lacked scalable methods to precisely edit regulatory DNA and quantify the effects in an endogenous genomic context. Here we present an approach to measure the quantitative effects of hundreds of designed DNA sequence variants on gene expression, by combining pooled CRISPR prime editing with RNA fluorescence in situ hybridization and cell sorting (Variant-FlowFISH). We apply this method to mutagenize and rewrite regulatory DNA sequences in an enhancer and the promoter of PPIF in two immune cell lines. Of 672 variant-cell type pairs, we identify 497 that affect PPIF expression. These variants appear to act through a variety of mechanisms including disruption or optimization of existing transcription factor binding sites, as well as creation of de novo sites. Disrupting a single endogenous transcription factor binding site often led to large changes in expression (up to -40% in the enhancer, and -50% in the promoter). The same variant often had different effects across cell types and states, demonstrating a highly tunable regulatory landscape. We use these data to benchmark performance of sequence-based predictive models of gene regulation, and find that certain types of variants are not accurately predicted by existing models. Finally, we computationally design 185 small sequence variants ([&amp;le;]10 bp) and optimize them for specific effects on expression in silico. 84% of these rationally designed edits showed the intended direction of effect, and some had dramatic effects on expression (-100% to &#43;202%). Variant-FlowFISH thus provides a powerful tool to map the effects of variants and transcription factor binding sites on gene expression, test and improve computational models of gene regulation, and reprogram regulatory DNA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574060">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574060" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574060">
        <p class="paperTitle">Complementation testing identifies causal genes at quantitative trait loci underlying fear related behavior</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574060" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574060" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, P. B.; Chen, R.; LaPierre, N. R. S.; Chen, Z.; Mefford, J.; Marcus, E.; Heffel, M.; Soto, D. C.; Ernst, J.; Luo, C.; Flint, J.</p>
        <p class="info">Score: 2.1, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574060' target='https://doi.org/10.1101/2024.01.03.574060'> 10.1101/2024.01.03.574060</a></p>
        <p class="abstract">Knowing the genes involved in quantitative traits provides a critical entry point to understanding the biological bases of behavior, but there are very few examples where the pathway from genetic locus to behavioral change is known. Here we address a key step towards that goal by deploying a test that directly queries whether a gene mediates the effect of a quantitative trait locus (QTL). To explore the role of specific genes in fear behavior, we mapped three fear-related traits, tested fourteen genes at six QTLs, and identified six genes. Four genes, Lsamp, Ptprd, Nptx2 and Sh3gl, have known roles in synapse function; the fifth gene, Psip1, is a transcriptional co-activator not previously implicated in behavior; the sixth is a long non-coding RNA 4933413L06Rik with no known function. Single nucleus transcriptomic and epigenetic analyses implicated excitatory neurons as likely mediating the genetic effects. Surprisingly, variation in transcriptome and epigenetic modalities between inbred strains occurred preferentially in excitatory neurons, suggesting that genetic variation is more permissible in excitatory than inhibitory neuronal circuits. Our results open a bottleneck in using genetic mapping of QTLs to find novel biology underlying behavior and prompt a reconsideration of expected relationships between genetic and functional variation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573736">
        <p class="paperTitle">DNA-PKcs Suppresses Illegitimate Chromosome Rearrangements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, J.; Sadeghi, C.; Frock, R. L.</p>
        <p class="info">Score: 3.1, Published: 2024-01-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573736' target='https://doi.org/10.1101/2023.12.30.573736'> 10.1101/2023.12.30.573736</a></p>
        <p class="abstract">Two DNA repair pathways, non-homologous end joining (NHEJ) and alternative end joining (A-EJ), are involved in V(D)J recombination and chromosome translocation. Previous studies reported distinct repair mechanisms for chromosome translocation, with NHEJ involved in human and A-EJ in mice predominantly. NHEJ depends on DNA-PKcs, a critical partner in synapsis formation and downstream component activation. While DNA-PKcs inhibition promotes chromosome translocations harboring microhomologies in mice, its synonymous effect in humans is not known. We find partial DNA-PKcs inhibition in human cells leads to increased translocations and the continued involvement of a dampened NHEJ. In contrast, complete DNA-PKcs inhibition substantially increased microhomology-mediated end joining (MMEJ), thus bridging the two different translocation mechanisms between human and mice. Similar to a previous study on Ku70 deletion, DNA-PKcs deletion in G1/G0-phase mouse progenitor B cell lines, significantly impairs V(D)J recombination and generated higher rates of translocations as a consequence of dysregulated coding and signal end joining. Genetic DNA-PKcs inhibition suppresses NHEJ involvement entirely, with repair phenotypically resembling Ku70-deficient A-EJ. In contrast, we find DNA-PKcs necessary in generating the near-excusive MMEJ associated with Lig4 deficiency. Our study underscores DNA-PKcs in suppressing illegitimate chromosome rearrangement while also contributing to MMEJ in both species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572550">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572550" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572550">
        <p class="paperTitle">Extreme restructuring of cis-regulatory regions controlling a deeply conserved plant stem cell regulator</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572550" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572550" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ciren, D.; Zebell, S.; Lippman, Z.</p>
        <p class="info">Score: 24.8, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572550' target='https://doi.org/10.1101/2023.12.20.572550'> 10.1101/2023.12.20.572550</a></p>
        <p class="abstract">A striking paradox is that genes with conserved protein sequence, function and expression pattern over deep time often exhibit extremely divergent cis-regulatory sequences. It remains unclear how such drastic cis-regulatory evolution across species allows preservation of gene function, and to what extent these differences influence how cis-regulatory variation arising within species impacts phenotypic change. Here, we investigated these questions using a plant stem cell regulator conserved in expression pattern and function over [~]125 million years. Using in-vivo genome editing in two distantly related models, Arabidopsis thaliana (Arabidopsis) and Solanum lycopersicum (tomato), we generated over 70 deletion alleles in the upstream and downstream regions of the stem cell repressor gene CLAVATA3 (CLV3) and compared their individual and combined effects on a shared phenotype, the number of carpels that make fruits. We found that sequences upstream of tomato CLV3 are highly sensitive to even small perturbations compared to its downstream region. In contrast, Arabidopsis CLV3 function is tolerant to severe disruptions both upstream and downstream of the coding sequence. Combining upstream and downstream deletions also revealed a different regulatory outcome. Whereas phenotypic enhancement from adding downstream mutations was predominantly weak and additive in tomato, mutating both regions of Arabidopsis CLV3 caused substantial and synergistic effects, demonstrating distinct distribution and redundancy of functional cis-regulatory sequences. Our results demonstrate remarkable malleability in cis-regulatory structural organization of a deeply conserved plant stem cell regulator and suggest that major reconfiguration of cis-regulatory sequence space is a common yet cryptic evolutionary force altering genotype-to-phenotype relationships from regulatory variation in conserved genes. Finally, our findings underscore the need for lineage-specific dissection of the spatial architecture of cis-regulation to effectively engineer trait variation from conserved productivity genes in crops.

Author summaryWe investigated the evolution of cis-regulatory elements (CREs) and their interactions in the regulation of a plant stem cell regulator gene, CLAVATA3 (CLV3), in Arabidopsis and tomato. Despite diverging [~]125 million years ago, the function and expression of CLV3 is conserved in these species; however, cis-regulatory sequences upstream and downstream have drastically diverged, preventing identification of conserved non-coding sequences between them. We used CRISPR-Cas9 to engineer dozens of mutations within the cis-regulatory regions of Arabidopsis and tomato CLV3. In tomato, our results show that tomato CLV3 function primarily relies on interactions among CREs in the 5 non-coding region, unlike Arabidopsis CLV3, which depends on a more balanced distribution of functional CREs between the 5 and 3 regions. Therefore, despite a high degree of functional conservation, our study demonstrates divergent regulatory strategies between two distantly related CLV3 orthologs, with substantial alterations in regulatory sequences, their spatial arrangement, and their relative effects on CLV3 regulation. These results suggest that regulatory regions are not only extremely robust to mutagenesis, but also that the sequences underlying this robustness can be lineage-specific for conserved genes, due to the complex and often redundant interactions among CREs that ensure proper gene function amidst large-scale sequence turnover.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.02.07.479436">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.02.07.479436" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.02.07.479436">
        <p class="paperTitle">Imputation of 3D genome structure by genetic-epigenetic interaction modeling in mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.02.07.479436" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.02.07.479436" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kuffler, L.; Fortin, H. J.; Skelly, D. A.; Czechanski, A.; Munger, S. C.; Reinholdt, L. G.; Baker, C. L.; Carter, G. W.</p>
        <p class="info">Score: 2.6, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.02.07.479436' target='https://doi.org/10.1101/2022.02.07.479436'> 10.1101/2022.02.07.479436</a></p>
        <p class="abstract">Gene expression is known to be affected by interactions between local genetic variation and DNA accessibility, with the latter organized into three-dimensional chromatin structures.

Analyses of these interactions has previously been limited, obscuring their regulatory context, and the extent to which they occur throughout the genome. Here we undertake a genome-scale analysis of these interactions in a genetically diverse population to systematically identify global genetic-epigenetic interaction, and reveal constraints imposed by chromatin structure. We establish the extent and structure of genotype-by-epigenotype interaction using embryonic stem cells derived from Diversity Outbred mice. This mouse population segregates millions of variants from eight inbred founders, enabling precision genetic mapping with extensive genotypic and phenotypic diversity. With 176 samples profiled for genotype, gene expression, and open chromatin, we used regression modeling to infer genetic-epigenetic interactions on a genome-wide scale. Our results demonstrate that statistical interactions between genetic variants and chromatin accessibility are common throughout the genome. We found that these interactions occur within the local area of the affected gene, and that this locality corresponds to topologically associated domains (TADs). The likelihood of interaction was most strongly defined by the three-dimensional (3D) domain structure rather than linear DNA sequence. We show that stable 3D genome structure is an effective tool to guide searches for regulatory elements and, conversely, that regulatory elements in genetically diverse populations provide a means to infer 3D genome structure. We confirmed this finding with CTCF ChIP-seq that revealed strain-specific binding in the inbred founder mice. In stem cells, open chromatin participating in the most significant regression models demonstrated an enrichment for developmental genes and the TAD-forming CTCF binding complex, providing an opportunity for statistical inference of shifting TAD boundaries operating during early development. These findings provide evidence that genetic and epigenetic factors operate within the context of three- dimensional chromatin structure.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=196 SRC=&#34;FIGDIR/small/479436v3_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (34K):
org.highwire.dtl.DTLVardef@155bda5org.highwire.dtl.DTLVardef@15c039org.highwire.dtl.DTLVardef@1ff063borg.highwire.dtl.DTLVardef@1f24802_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
