<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genetic similarity between relatives provides evidence on the presence and history of assortative mating
Authors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.
Score: 35.4, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546663
Assortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives1-7.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Genetic similarity between relatives provides evidence on the presence and history of assortative mating
Authors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.
Score: 35.4, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546663
Assortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives1-7." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-05T10:42:43+00:00" />
<meta property="article:modified_time" content="2023-07-05T10:42:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Genetic similarity between relatives provides evidence on the presence and history of assortative mating
Authors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.
Score: 35.4, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546663
Assortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives1-7."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Genetic similarity between relatives provides evidence on the presence and history of assortative mating\nAuthors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.\nScore: 35.4, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546663\nAssortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives1-7.",
  "keywords": [
    
  ],
  "articleBody": " Genetic similarity between relatives provides evidence on the presence and history of assortative mating\nAuthors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.\nScore: 35.4, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546663\nAssortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives1-7. However, empirical evidence is limited for many traits, and the implications hinge on whether assortative mating has started recently or many generations ago. Here we show theoretically and empirically that genetic similarity between relatives can provide evidence on the presence and history of assortative mating. First, we employed path analysis to understand how assortative mating affects genetic similarity between family members across generations, finding that similarity between distant relatives is more affected than close relatives. Next, we correlated polygenic indices of 47,135 co-parents from the Norwegian Mother, Father, and Child Cohort Study (MoBa) and found genetic evidence of assortative mating in eight out of fifteen examined traits. The same traits show elevated similarity between relatives, especially distant relatives. Five of the eight traits, including educational attainment, showed greater genetic variance among offspring, which is inconsistent with stable assortative mating over many generations. These results suggest an ongoing increase in familial similarity for these traits. The implications of this research extend to genetic methodology and the understanding of social and economic disparities.\nPolygenic risk score prediction accuracy convergence\nAuthors: Henches, L.; Kim, J.; Yang, Z.; Rubinacci, S.; Pires, G.; Albinana, C.; Boetto, C.; Julienne, H.; Frouin, A.; Auvergne, A.; Suzuki, Y.; Djebali, S.; Delaneau, O.; Ganna, A.; Vilhjalmsson, B.; Prive, F.; Aschard, H.\nScore: 25.9, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546518\nPolygenic risk scores (PRS) trained from genome-wide association study (GWAS) results are set to play a pivotal role in biomedical research addressing multifactorial human diseases. The prospect of using these risk scores in clinical care and public health is generating both enthusiasm and controversy, with varying opinions about strengths and limitations across experts1. The performances of existing polygenic scores are still limited, and although it is expected to improve with increasing sample size of GWAS and the development of new powerful methods, it remains unclear how much prediction can be ultimately achieved. Here, we conducted a retrospective analysis to assess the progress in PRS prediction accuracy since the publication of the first large-scale GWASs using six common human diseases with sufficient GWAS data. We show that while PRS accuracy has grown rapidly for years, the improvement pace from recent GWAS has decreased substantially, suggesting that further increasing GWAS sample size may translate into very modest risk discrimination improvement. We next investigated the factors influencing the maximum achievable prediction using recently released whole genome-sequencing data from 125K UK Biobank participants, and state-of-the-art modeling of polygenic outcomes. Our analyses point toward increasing the variant coverage of PRS, using either more imputed variants or sequencing data, as a key component for future improvement in prediction accuracy.\nAccounting for Isoform Expression in eQTL Mapping Substantially Increases Power\nAuthors: LaPierre, N.; Pimentel, H.\nScore: 20.5, Published: 2023-06-30 DOI: 10.1101/2023.06.28.546921\nA core problem in genetics is eQTL mapping, in which genetic variants associated with changes in expression of genes are identified. It is common in eQTL mapping to compute gene expression by aggregating the expression levels of individual isoforms from the same gene and then performing linear regression between SNPs and this aggregated gene expression level. However, SNPs may regulate isoforms from the same gene in different directions due to alternative splicing, or only regulate the expression level of one isoform, causing this approach to lose power. In this study, we provide a systematic evaluation of methods for accounting for individual isoform expression levels based on generative isoform expression heritability models and real data. Over a range of conditions, we show that these approaches substantially increase the power to map eQTLs in both simulations and commonly analyzed large data sets. We identify settings in which different approaches yield an inflated number of false discoveries or lose power. In particular, we show that calling an eGene if there is a significant association between a SNP and any isoform fails to control False Discovery Rate, even when applying standard False Discovery Rate correction. We show that similar trends are observed in real data from the GEUVADIS and GTEx studies, suggesting the possibility that similar effects are present in these consortia.\nAncient DNA reveals the origins of the Albanians\nAuthors: Davranoglou, L. R.; Aristodemou, A.; Wesolowski, D.; Heraclides, A.\nScore: 193.9, Published: 2023-06-07 DOI: 10.1101/2023.06.05.543790\nThe origins of the Albanian people have vexed linguists and historians for centuries, as Albanians first appear in the historical record in the 11th century CE, while their language is one of the most enigmatic branches of the Indo-European family. To identify the populations that contributed to the ancestry of Albanians, we undertake a genomic transect of the Balkans over the last 8000 years, where we analyse more than 6000 previously published ancient genomes using state-of-the-art bioinformatics tools and algorithms that quantify spatiotemporal human mobility. We find that modern Albanians descend from Roman era western Balkan populations, with additional admixture from Slavic-related groups. Remarkably, Albanian paternal ancestry shows continuity from Bronze Age Balkan populations, including those known as Illyrians. Our results provide an unprecedented understanding of the historical and demographic processes that led to the formation of modern Albanians and help locate the area where the Albanian language developed.\nSex peptide receptor is not required for refractoriness to remating or induction of egg laying in Aedes aegypti\nAuthors: Amaro, I. A.; Wohl, M. P.; Pitcher, S.; Alfonso-Parra, C.; Avila, F. W.; Paige, A. S.; Helinski, M.; Duvall, L. B.; Harrington, L. C.; Wolfner, M. F.; McMeniman, C. J.\nScore: 14.6, Published: 2023-06-30 DOI: 10.1101/2023.06.28.546954\nAcross diverse insect taxa, the behavior and physiology of females dramatically changes after mating - processes largely triggered by the transfer of seminal proteins from their mates. In the vinegar fly Drosophila melanogaster, the seminal protein sex peptide (SP) decreases the likelihood of female flies remating and causes additional behavioral and physiological changes that promote fertility including increasing egg production. Although SP is only found in the Drosophila genus, its receptor, sex peptide receptor (SPR), is the widely-conserved myoinhibitory peptide (MIP) receptor. To test the functional role of SPR in mediating post-mating responses in a non-Drosophila dipteran, we generated two independent Spr-knockout alleles in the yellow fever mosquito Aedes aegypti. Although SPR is needed for post-mating responses in Drosophila and the cotton bollworm Helicoverpa armigera, Spr mutant Ae. aegypti show completely normal post-mating decreases in remating propensity and increases in egg laying. In addition, injection of synthetic SP or accessory gland homogenate from D. melanogaster into virgin female mosquitoes did not elicit these post-mating responses. Our results indicate that Spr is not required for these canonical post-mating responses in Ae. aegypti, indicating that unknown signaling pathways are likely responsible for these behavioral switches in this disease vector.\nIs a Picture Worth 1,000 SNPs? Effects of User-Submitted Photographs on Ancestry Estimates from Direct-to-Consumer Canine Genetic Tests\nAuthors: Rando, H.; Graim, K.; Hampikian, G.; Greene, C. S.\nScore: 11.6, Published: 2023-06-29 DOI: 10.1101/2023.06.28.546898\nObjectiveTo evaluate whether the breed ancestry predictions of direct-to-consumer (DTC) genetic tests for dogs are influenced by the user-provided photograph. AnimalsTwelve pet dogs considered purebred (i.e., registered with a breed organization) representing twelve different breeds. MethodsSix buccal swabs per dog were collected by the owners and submitted to six DTC genetic testing companies. The experimenters registered each sample with the company. For half of the dogs, the registration included a photograph of the DNA donor. For the other half of the dogs, photographs were swapped between dogs. Analysis of the DNA and breed ancestry prediction was conducted by each company. Each companys breed predictions were evaluated to assess whether the condition (i.e., matching versus shuffled photograph) affected the odds of identifying the DNA donors registered breed. A convolutional neural network was also used to predict breed based solely on the photograph as a positive control. ResultsFive of the six tests always produced results that included the registered breed. One test and the convolutional neural network were unlikely to identify the registered breed and frequently returned results that included the breed in the photograph. This result suggests that one test on the market is relying on the photograph more than the DNA sample. Additionally, differences in the predictions made across all tests underscore the challenge of identifying breed ancestry, even in purebred dogs. Clinical RelevanceVeterinarians are likely to encounter patients who have conducted DTC genetic testing and may find themselves in the position of explaining genetic test results that they did not order. This systematic comparison of tests on the market provides context for interpreting unexpected results from consumer-grade DTC genetic testing kits.\nThe ChickenGTEx pilot analysis: a reference of regulatory variants across 28 chicken tissues\nAuthors: Guan, D.; Bai, Z.; Zhu, X.; Zhong, C.; Hou, Y.; Lan, F.; Diao, S.; Yao, Y.; Zhao, B.; Zhu, D.; Li, X.; Pan, Z.; Gao, Y.; Wang, Y.; Zou, D.; Wang, R.; Xu, T.; Sun, C.; Yin, H.; Teng, J.; Xu, Z.; Lin, Q.; Shi, S.; Shao, D.; Degalez, F.; Lagarrigue, S.; Wang, Y.; Wang, M.; Peng, M.; Rocha, D.; Charles, M.; Smith, J.; Watson, K.; Buitenhuis, A. J.; Sahana, G.; Lund, M. S.; Warren, W.; Frantz, L.; Larson, G.; Lamont, S. J.; Si, W.; Zhao, X.; Li, B.; Zhang, H.; Luo, C.; Shu, D.; Qu, H.; Luo, W.; Li, Z.; Nie, Q.; Zhang, X.; Zhang, Z.; Zhang, Z.; Liu, G. E.; Cheng, H.; Yang, N.; Hu, X.; Zhou, H.; Fang\nScore: 10.3, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546670\nChicken is a valuable model for understanding fundamental biology, vertebrate evolution and diseases, as well as a major source of nutrient-dense and lean-protein-enriched food globally. Although it is the first non-mammalian amniote genome to be sequenced, the chicken genome still lacks a systematic characterization of functional impacts of genetic variants. Here, through integrating 7,015 RNA-Seq and 2,869 whole-genome sequence data, the Chicken Genotype- Tissue Expression (ChickenGTEx) project presents the pilot reference of regulatory variants in 28 chicken tissue transcriptomes, including millions of regulatory effects on primary expression (including protein-coding genes, lncRNA and exon) and post-transcriptional modifications (alternative splicing and 3 untranslated region alternative polyadenylation). We explored the tissue-sharing and context-specificity of these regulatory variants, their underlying molecular mechanisms of action, and their utility in interpreting adaptation and genome-wide associations of 108 chicken complex traits. Finally, we illustrated shared and lineage-specific features of gene regulation between chickens and mammals, and demonstrated how the ChickenGTEx resource can further assist with translating genetic findings across species. One-Sentence SummaryThe ChickenGTEx provides a multi-tissue reference of regulatory variants for chicken genetics and genomics, functional genomics, precision breeding, veterinary medicine, vertebrate evolution and even human biomedicine.\nA whole organism map of X-inactivation reveals tissue-specific escape as an exceptionally rare event in humans\nAuthors: Gylemo, B.; Nestor, C. E.\nScore: 11.6, Published: 2023-06-27 DOI: 10.1101/2023.06.26.546519\nAs females are mosaic for X-inactivation, direct determination of X-linked allelic expression in bulk tissues is typically unfeasible. Using females that are non-mosaic for X-inactivation (nmXCI) as a powerful and natural genetic system we directly determined X-inactivation status of 401 X-linked genes across 32 normal tissues. We reveal that nmXCI is common in the female population and that contrary to previous reports, variable escape is an exceedingly rare event in humans. As the most complete atlas of XCI in humans to date, our findings represent a major advance in our understanding of human X-inactivation and will serve as a reference for dissecting the genetic origin of sex-bias in human traits.\nMulti-trait GWAS for diverse ancestries: Mapping the knowledge gap\nAuthors: Troubat, L.; Fettahoglu, D.; Aschard, H.; Julienne, H.\nScore: 7.9, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546248\nBackgroundApproximately 95% of samples analyzed in univariate genome-wide association studies (GWAS) are of European ancestry. This bias toward European ancestry populations in association screening also exists for other analyses and methods that are often developed and tested on European ancestry only. However, existing data in non-European populations, which are often of modest sample size, could benefit from innovative approaches as recently illustrated in the context of polygenic risk scores. MethodsHere, we extend and assess the potential limitations and gain of our multi-trait GWAS pipeline, JASS (Joint Analysis of Summary Statistics), for the analysis of non-European ancestries. To this end, we conducted the joint GWAS of 19 hematological traits and glycemic traits accross five ancestries (European (EUR), admixed American (AMR), African (AFR), East Asian (EAS), South-East Asian (SAS)). ResultsWe detected 367 new genome-wide significant associations in non-European populations (15 in Admixed American (AMR), 72 in African (AFR) and 280 in East Asian (EAS)). New associations detected represent 7%, 25% and 21% of associations in the AFR, AMR and EAS populations, respectively. Overall, multi-trait testing increases the replication of European associated loci in non-European ancestry by 15%. Pleiotropic effects were highly similar at significant loci across ancestries (e.g. the mean correlation between multi-trait genetic effects of EUR and EAS ancestries was 0.88). For hematological traits, strong discrepancies in multi-trait genetic effects are tied to known evolutionary divergences: the ARKC1 loci, which is adaptive to overcome the p.vivax induced malaria. ConclusionsMulti-trait GWAS can be a valuable tool to narrow the genetic knowledge gap between European and non-European populations.\nInteraction molecular QTL mapping discovers cellular and environmental modifiers of genetic regulatory effects\nAuthors: Kasela, S.; Aguet, F.; Kim-Hellmuth, S.; Brown, B. C.; Nachun, D. C.; Tracy, R. P.; Durda, P.; Liu, Y.; Taylor, K. D.; Johnson, W. C.; Van Den Berg, D.; Gabriel, S.; Gupta, N.; Smith, J. D.; Blackwell, T. W.; Rotter, J. I.; Ardlie, K. G.; Manichaikul, A.; Rich, S. S.; Barr, R. G.; Lappalainen, T.\nScore: 19.4, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546528\nBulk tissue molecular quantitative trait loci (QTLs) have been the starting point for interpreting disease-associated variants, while context-specific QTLs show particular relevance for disease. Here, we present the results of mapping interaction QTLs (iQTLs) for cell type, age, and other phenotypic variables in multi-omic, longitudinal data from blood of individuals of diverse ancestries. By modeling the interaction between genotype and estimated cell type proportions, we demonstrate that cell type iQTLs could be considered as proxies for cell type-specific QTL effects. The interpretation of age iQTLs, however, warrants caution as the moderation effect of age on the genotype and molecular phenotype association may be mediated by changes in cell type composition. Finally, we show that cell type iQTLs contribute to cell type-specific enrichment of diseases that, in combination with additional functional data, may guide future functional studies. Overall, this study highlights iQTLs to gain insights into the context-specificity of regulatory effects.\n",
  "wordCount" : "2444",
  "inLanguage": "en",
  "datePublished": "2023-07-05T10:42:43Z",
  "dateModified": "2023-07-05T10:42:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on July 5, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546663">
        <p class="paperTitle">Genetic similarity between relatives provides evidence on the presence and history of assortative mating</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.</p>
        <p class="info">Score: 35.4, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546663' target='https://doi.org/10.1101/2023.06.27.546663'> 10.1101/2023.06.27.546663</a></p>
        <p class="abstract">Assortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives1-7. However, empirical evidence is limited for many traits, and the implications hinge on whether assortative mating has started recently or many generations ago. Here we show theoretically and empirically that genetic similarity between relatives can provide evidence on the presence and history of assortative mating. First, we employed path analysis to understand how assortative mating affects genetic similarity between family members across generations, finding that similarity between distant relatives is more affected than close relatives. Next, we correlated polygenic indices of 47,135 co-parents from the Norwegian Mother, Father, and Child Cohort Study (MoBa) and found genetic evidence of assortative mating in eight out of fifteen examined traits. The same traits show elevated similarity between relatives, especially distant relatives. Five of the eight traits, including educational attainment, showed greater genetic variance among offspring, which is inconsistent with stable assortative mating over many generations. These results suggest an ongoing increase in familial similarity for these traits. The implications of this research extend to genetic methodology and the understanding of social and economic disparities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546518">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546518" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546518">
        <p class="paperTitle">Polygenic risk score prediction accuracy convergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546518" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546518" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Henches, L.; Kim, J.; Yang, Z.; Rubinacci, S.; Pires, G.; Albinana, C.; Boetto, C.; Julienne, H.; Frouin, A.; Auvergne, A.; Suzuki, Y.; Djebali, S.; Delaneau, O.; Ganna, A.; Vilhjalmsson, B.; Prive, F.; Aschard, H.</p>
        <p class="info">Score: 25.9, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546518' target='https://doi.org/10.1101/2023.06.27.546518'> 10.1101/2023.06.27.546518</a></p>
        <p class="abstract">Polygenic risk scores (PRS) trained from genome-wide association study (GWAS) results are set to play a pivotal role in biomedical research addressing multifactorial human diseases. The prospect of using these risk scores in clinical care and public health is generating both enthusiasm and controversy, with varying opinions about strengths and limitations across experts1. The performances of existing polygenic scores are still limited, and although it is expected to improve with increasing sample size of GWAS and the development of new powerful methods, it remains unclear how much prediction can be ultimately achieved. Here, we conducted a retrospective analysis to assess the progress in PRS prediction accuracy since the publication of the first large-scale GWASs using six common human diseases with sufficient GWAS data. We show that while PRS accuracy has grown rapidly for years, the improvement pace from recent GWAS has decreased substantially, suggesting that further increasing GWAS sample size may translate into very modest risk discrimination improvement. We next investigated the factors influencing the maximum achievable prediction using recently released whole genome-sequencing data from 125K UK Biobank participants, and state-of-the-art modeling of polygenic outcomes. Our analyses point toward increasing the variant coverage of PRS, using either more imputed variants or sequencing data, as a key component for future improvement in prediction accuracy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546921" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546921">
        <p class="paperTitle">Accounting for Isoform Expression in eQTL Mapping Substantially Increases Power</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: LaPierre, N.; Pimentel, H.</p>
        <p class="info">Score: 20.5, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546921' target='https://doi.org/10.1101/2023.06.28.546921'> 10.1101/2023.06.28.546921</a></p>
        <p class="abstract">A core problem in genetics is eQTL mapping, in which genetic variants associated with changes in expression of genes are identified. It is common in eQTL mapping to compute gene expression by aggregating the expression levels of individual isoforms from the same gene and then performing linear regression between SNPs and this aggregated gene expression level. However, SNPs may regulate isoforms from the same gene in different directions due to alternative splicing, or only regulate the expression level of one isoform, causing this approach to lose power. In this study, we provide a systematic evaluation of methods for accounting for individual isoform expression levels based on generative isoform expression heritability models and real data. Over a range of conditions, we show that these approaches substantially increase the power to map eQTLs in both simulations and commonly analyzed large data sets. We identify settings in which different approaches yield an inflated number of false discoveries or lose power. In particular, we show that calling an eGene if there is a significant association between a SNP and any isoform fails to control False Discovery Rate, even when applying standard False Discovery Rate correction. We show that similar trends are observed in real data from the GEUVADIS and GTEx studies, suggesting the possibility that similar effects are present in these consortia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.05.543790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.05.543790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.05.543790">
        <p class="paperTitle">Ancient DNA reveals the origins of the Albanians</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.05.543790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.05.543790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davranoglou, L. R.; Aristodemou, A.; Wesolowski, D.; Heraclides, A.</p>
        <p class="info">Score: 193.9, Published: 2023-06-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.05.543790' target='https://doi.org/10.1101/2023.06.05.543790'> 10.1101/2023.06.05.543790</a></p>
        <p class="abstract">The origins of the Albanian people have vexed linguists and historians for centuries, as Albanians first appear in the historical record in the 11th century CE, while their language is one of the most enigmatic branches of the Indo-European family. To identify the populations that contributed to the ancestry of Albanians, we undertake a genomic transect of the Balkans over the last 8000 years, where we analyse more than 6000 previously published ancient genomes using state-of-the-art bioinformatics tools and algorithms that quantify spatiotemporal human mobility. We find that modern Albanians descend from Roman era western Balkan populations, with additional admixture from Slavic-related groups. Remarkably, Albanian paternal ancestry shows continuity from Bronze Age Balkan populations, including those known as Illyrians. Our results provide an unprecedented understanding of the historical and demographic processes that led to the formation of modern Albanians and help locate the area where the Albanian language developed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546954">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546954" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546954">
        <p class="paperTitle">Sex peptide receptor is not required for refractoriness to remating or induction of egg laying in Aedes aegypti</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546954" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546954" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Amaro, I. A.; Wohl, M. P.; Pitcher, S.; Alfonso-Parra, C.; Avila, F. W.; Paige, A. S.; Helinski, M.; Duvall, L. B.; Harrington, L. C.; Wolfner, M. F.; McMeniman, C. J.</p>
        <p class="info">Score: 14.6, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546954' target='https://doi.org/10.1101/2023.06.28.546954'> 10.1101/2023.06.28.546954</a></p>
        <p class="abstract">Across diverse insect taxa, the behavior and physiology of females dramatically changes after mating - processes largely triggered by the transfer of seminal proteins from their mates. In the vinegar fly Drosophila melanogaster, the seminal protein sex peptide (SP) decreases the likelihood of female flies remating and causes additional behavioral and physiological changes that promote fertility including increasing egg production. Although SP is only found in the Drosophila genus, its receptor, sex peptide receptor (SPR), is the widely-conserved myoinhibitory peptide (MIP) receptor. To test the functional role of SPR in mediating post-mating responses in a non-Drosophila dipteran, we generated two independent Spr-knockout alleles in the yellow fever mosquito Aedes aegypti. Although SPR is needed for post-mating responses in Drosophila and the cotton bollworm Helicoverpa armigera, Spr mutant Ae. aegypti show completely normal post-mating decreases in remating propensity and increases in egg laying. In addition, injection of synthetic SP or accessory gland homogenate from D. melanogaster into virgin female mosquitoes did not elicit these post-mating responses. Our results indicate that Spr is not required for these canonical post-mating responses in Ae. aegypti, indicating that unknown signaling pathways are likely responsible for these behavioral switches in this disease vector.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546898">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546898" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546898">
        <p class="paperTitle">Is a Picture Worth 1,000 SNPs? Effects of User-Submitted Photographs on Ancestry Estimates from Direct-to-Consumer Canine Genetic Tests</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546898" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546898" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rando, H.; Graim, K.; Hampikian, G.; Greene, C. S.</p>
        <p class="info">Score: 11.6, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546898' target='https://doi.org/10.1101/2023.06.28.546898'> 10.1101/2023.06.28.546898</a></p>
        <p class="abstract">ObjectiveTo evaluate whether the breed ancestry predictions of direct-to-consumer (DTC) genetic tests for dogs are influenced by the user-provided photograph.

AnimalsTwelve pet dogs considered purebred (i.e., registered with a breed organization) representing twelve different breeds.

MethodsSix buccal swabs per dog were collected by the owners and submitted to six DTC genetic testing companies. The experimenters registered each sample with the company. For half of the dogs, the registration included a photograph of the DNA donor. For the other half of the dogs, photographs were swapped between dogs. Analysis of the DNA and breed ancestry prediction was conducted by each company. Each companys breed predictions were evaluated to assess whether the condition (i.e., matching versus shuffled photograph) affected the odds of identifying the DNA donors registered breed. A convolutional neural network was also used to predict breed based solely on the photograph as a positive control.

ResultsFive of the six tests always produced results that included the registered breed. One test and the convolutional neural network were unlikely to identify the registered breed and frequently returned results that included the breed in the photograph. This result suggests that one test on the market is relying on the photograph more than the DNA sample. Additionally, differences in the predictions made across all tests underscore the challenge of identifying breed ancestry, even in purebred dogs.

Clinical RelevanceVeterinarians are likely to encounter patients who have conducted DTC genetic testing and may find themselves in the position of explaining genetic test results that they did not order. This systematic comparison of tests on the market provides context for interpreting unexpected results from consumer-grade DTC genetic testing kits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546670">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546670" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546670">
        <p class="paperTitle">The ChickenGTEx pilot analysis: a reference of regulatory variants across 28 chicken tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546670" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546670" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guan, D.; Bai, Z.; Zhu, X.; Zhong, C.; Hou, Y.; Lan, F.; Diao, S.; Yao, Y.; Zhao, B.; Zhu, D.; Li, X.; Pan, Z.; Gao, Y.; Wang, Y.; Zou, D.; Wang, R.; Xu, T.; Sun, C.; Yin, H.; Teng, J.; Xu, Z.; Lin, Q.; Shi, S.; Shao, D.; Degalez, F.; Lagarrigue, S.; Wang, Y.; Wang, M.; Peng, M.; Rocha, D.; Charles, M.; Smith, J.; Watson, K.; Buitenhuis, A. J.; Sahana, G.; Lund, M. S.; Warren, W.; Frantz, L.; Larson, G.; Lamont, S. J.; Si, W.; Zhao, X.; Li, B.; Zhang, H.; Luo, C.; Shu, D.; Qu, H.; Luo, W.; Li, Z.; Nie, Q.; Zhang, X.; Zhang, Z.; Zhang, Z.; Liu, G. E.; Cheng, H.; Yang, N.; Hu, X.; Zhou, H.; Fang</p>
        <p class="info">Score: 10.3, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546670' target='https://doi.org/10.1101/2023.06.27.546670'> 10.1101/2023.06.27.546670</a></p>
        <p class="abstract">Chicken is a valuable model for understanding fundamental biology, vertebrate evolution and diseases, as well as a major source of nutrient-dense and lean-protein-enriched food globally. Although it is the first non-mammalian amniote genome to be sequenced, the chicken genome still lacks a systematic characterization of functional impacts of genetic variants. Here, through integrating 7,015 RNA-Seq and 2,869 whole-genome sequence data, the Chicken Genotype- Tissue Expression (ChickenGTEx) project presents the pilot reference of regulatory variants in 28 chicken tissue transcriptomes, including millions of regulatory effects on primary expression (including protein-coding genes, lncRNA and exon) and post-transcriptional modifications (alternative splicing and 3 untranslated region alternative polyadenylation). We explored the tissue-sharing and context-specificity of these regulatory variants, their underlying molecular mechanisms of action, and their utility in interpreting adaptation and genome-wide associations of 108 chicken complex traits. Finally, we illustrated shared and lineage-specific features of gene regulation between chickens and mammals, and demonstrated how the ChickenGTEx resource can further assist with translating genetic findings across species.

One-Sentence SummaryThe ChickenGTEx provides a multi-tissue reference of regulatory variants for chicken genetics and genomics, functional genomics, precision breeding, veterinary medicine, vertebrate evolution and even human biomedicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546519" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546519">
        <p class="paperTitle">A whole organism map of X-inactivation reveals tissue-specific escape as an exceptionally rare event in humans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gylemo, B.; Nestor, C. E.</p>
        <p class="info">Score: 11.6, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546519' target='https://doi.org/10.1101/2023.06.26.546519'> 10.1101/2023.06.26.546519</a></p>
        <p class="abstract">As females are mosaic for X-inactivation, direct determination of X-linked allelic expression in bulk tissues is typically unfeasible. Using females that are non-mosaic for X-inactivation (nmXCI) as a powerful and natural genetic system we directly determined X-inactivation status of 401 X-linked genes across 32 normal tissues. We reveal that nmXCI is common in the female population and that contrary to previous reports, variable escape is an exceedingly rare event in humans. As the most complete atlas of XCI in humans to date, our findings represent a major advance in our understanding of human X-inactivation and will serve as a reference for dissecting the genetic origin of sex-bias in human traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546248">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546248" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546248">
        <p class="paperTitle">Multi-trait GWAS for diverse ancestries: Mapping the knowledge gap</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546248" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546248" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Troubat, L.; Fettahoglu, D.; Aschard, H.; Julienne, H.</p>
        <p class="info">Score: 7.9, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546248' target='https://doi.org/10.1101/2023.06.23.546248'> 10.1101/2023.06.23.546248</a></p>
        <p class="abstract">BackgroundApproximately 95% of samples analyzed in univariate genome-wide association studies (GWAS) are of European ancestry. This bias toward European ancestry populations in association screening also exists for other analyses and methods that are often developed and tested on European ancestry only. However, existing data in non-European populations, which are often of modest sample size, could benefit from innovative approaches as recently illustrated in the context of polygenic risk scores.

MethodsHere, we extend and assess the potential limitations and gain of our multi-trait GWAS pipeline, JASS (Joint Analysis of Summary Statistics), for the analysis of non-European ancestries. To this end, we conducted the joint GWAS of 19 hematological traits and glycemic traits accross five ancestries (European (EUR), admixed American (AMR), African (AFR), East Asian (EAS), South-East Asian (SAS)).

ResultsWe detected 367 new genome-wide significant associations in non-European populations (15 in Admixed American (AMR), 72 in African (AFR) and 280 in East Asian (EAS)). New associations detected represent 7%, 25% and 21% of associations in the AFR, AMR and EAS populations, respectively. Overall, multi-trait testing increases the replication of European associated loci in non-European ancestry by 15%. Pleiotropic effects were highly similar at significant loci across ancestries (e.g. the mean correlation between multi-trait genetic effects of EUR and EAS ancestries was 0.88). For hematological traits, strong discrepancies in multi-trait genetic effects are tied to known evolutionary divergences: the ARKC1 loci, which is adaptive to overcome the p.vivax induced malaria.

ConclusionsMulti-trait GWAS can be a valuable tool to narrow the genetic knowledge gap between European and non-European populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546528">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546528" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546528">
        <p class="paperTitle">Interaction molecular QTL mapping discovers cellular and environmental modifiers of genetic regulatory effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546528" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546528" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kasela, S.; Aguet, F.; Kim-Hellmuth, S.; Brown, B. C.; Nachun, D. C.; Tracy, R. P.; Durda, P.; Liu, Y.; Taylor, K. D.; Johnson, W. C.; Van Den Berg, D.; Gabriel, S.; Gupta, N.; Smith, J. D.; Blackwell, T. W.; Rotter, J. I.; Ardlie, K. G.; Manichaikul, A.; Rich, S. S.; Barr, R. G.; Lappalainen, T.</p>
        <p class="info">Score: 19.4, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546528' target='https://doi.org/10.1101/2023.06.26.546528'> 10.1101/2023.06.26.546528</a></p>
        <p class="abstract">Bulk tissue molecular quantitative trait loci (QTLs) have been the starting point for interpreting disease-associated variants, while context-specific QTLs show particular relevance for disease. Here, we present the results of mapping interaction QTLs (iQTLs) for cell type, age, and other phenotypic variables in multi-omic, longitudinal data from blood of individuals of diverse ancestries. By modeling the interaction between genotype and estimated cell type proportions, we demonstrate that cell type iQTLs could be considered as proxies for cell type-specific QTL effects. The interpretation of age iQTLs, however, warrants caution as the moderation effect of age on the genotype and molecular phenotype association may be mediated by changes in cell type composition. Finally, we show that cell type iQTLs contribute to cell type-specific enrichment of diseases that, in combination with additional functional data, may guide future functional studies. Overall, this study highlights iQTLs to gain insights into the context-specificity of regulatory effects.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
