<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Synergistic insulation of regulatory domains by developmental genes and clusters of CTCF sites
Authors: Ealo, T.; Sanchez-Gaya, V.; Respuela, P.; Munoz-San Martin, M.; Martin-Batista, E.; Haro, E.; Rada-Iglesias, A.
Score: 18.6, Published: 2023-12-16 DOI: 10.1101/2023.12.15.571760
The specificity of gene expression during development requires the insulation of regulatory domains to avoid inappropriate enhancer-gene interactions. In vertebrates, this insulator function is mostly attributed to clusters of CTCF sites located at topologically associating domain (TAD) boundaries.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Synergistic insulation of regulatory domains by developmental genes and clusters of CTCF sites
Authors: Ealo, T.; Sanchez-Gaya, V.; Respuela, P.; Munoz-San Martin, M.; Martin-Batista, E.; Haro, E.; Rada-Iglesias, A.
Score: 18.6, Published: 2023-12-16 DOI: 10.1101/2023.12.15.571760
The specificity of gene expression during development requires the insulation of regulatory domains to avoid inappropriate enhancer-gene interactions. In vertebrates, this insulator function is mostly attributed to clusters of CTCF sites located at topologically associating domain (TAD) boundaries." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Synergistic insulation of regulatory domains by developmental genes and clusters of CTCF sites
Authors: Ealo, T.; Sanchez-Gaya, V.; Respuela, P.; Munoz-San Martin, M.; Martin-Batista, E.; Haro, E.; Rada-Iglesias, A.
Score: 18.6, Published: 2023-12-16 DOI: 10.1101/2023.12.15.571760
The specificity of gene expression during development requires the insulation of regulatory domains to avoid inappropriate enhancer-gene interactions. In vertebrates, this insulator function is mostly attributed to clusters of CTCF sites located at topologically associating domain (TAD) boundaries."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Synergistic insulation of regulatory domains by developmental genes and clusters of CTCF sites\nAuthors: Ealo, T.; Sanchez-Gaya, V.; Respuela, P.; Munoz-San Martin, M.; Martin-Batista, E.; Haro, E.; Rada-Iglesias, A.\nScore: 18.6, Published: 2023-12-16 DOI: 10.1101/2023.12.15.571760\nThe specificity of gene expression during development requires the insulation of regulatory domains to avoid inappropriate enhancer-gene interactions. In vertebrates, this insulator function is mostly attributed to clusters of CTCF sites located at topologically associating domain (TAD) boundaries.",
  "keywords": [
    
  ],
  "articleBody": " Synergistic insulation of regulatory domains by developmental genes and clusters of CTCF sites\nAuthors: Ealo, T.; Sanchez-Gaya, V.; Respuela, P.; Munoz-San Martin, M.; Martin-Batista, E.; Haro, E.; Rada-Iglesias, A.\nScore: 18.6, Published: 2023-12-16 DOI: 10.1101/2023.12.15.571760\nThe specificity of gene expression during development requires the insulation of regulatory domains to avoid inappropriate enhancer-gene interactions. In vertebrates, this insulator function is mostly attributed to clusters of CTCF sites located at topologically associating domain (TAD) boundaries. However, TAD boundaries allow a certain level of physical crosstalk across regulatory domains, which is at odds with the highly specific and precise expression of developmental genes. Here we show that developmental genes and nearby clusters of CTCF sites synergistically foster the robust insulation of regulatory domains. Firstly, we found that the TADs containing developmental genes have distinctive features, including the sequential organization of developmental genes and CTCF clusters near TAD boundaries. Most importantly, by genetically dissecting representative loci in mouse embryonic stem cells, we showed that developmental genes and CTCF sites synergistically strengthened the insulation capacity of nearby boundaries through different mechanisms. Namely, while CTCF sites prevent undesirable enhancer-gene contacts (i.e. physical insulation), developmental genes preferentially contribute to regulatory insulation through non-structural mechanisms involving promoter competition rather than enhancer blocking. Overall, our work provides important insights into the specificity of gene regulation, which in turn might help interpreting the pathological consequences of certain structural variants.\nBetween-group heritability and the status of hereditarianism as an evolutionary science\nAuthors: Roseman, C. C.; Bird, K.\nScore: 13.9, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572247\nHereditarianism is a school of thought that contends there are substantial evolved cognitive and behavioral differences among groups of humans which are both resistant to environmental intervention and are a root cause of differential social outcomes across groups. The relationship of between- group heritability (h2B) to within-group heritability (h2W) is one of the key theoretical components of hereditarianism and forms one of the bases for its claim to be an evolutionary science. Here, we examine the relationship between h2B and h2W and its application to problems in the hereditarian literature from an evolutionary genetic perspective. We demonstrate that the formulation of the relationship between h2B and h2W used in the hereditarian literature has no evolutionary content. By re-writing the relationship between h2B and h2W in a novel evolutionary framework, we demonstrate that there is no way to predict h2B using h2W without considerable additional theory that is absent from the hereditarian literature. Furthermore, we demonstrate that the hereditarian technique that uses h2B and h2W as a means of judging whether a given difference between groups may be plausibly ameliorated through environmental intervention is mathematically flawed. Lastly, we fill a gap in the hereditarian literature by writing out a means of using h2B to predict the absolute difference between groups under a neutral evolutionary model and find that it is much smaller than claimed by hereditarians. In conclusion, we propose a path forward for the study of human variation that moves us past the ill-conceived nature vs. nurture question and allows us to focus on more productive issues.\nThe germline regulates longevity and somatic repair in a sex-specific manner\nAuthors: Moses, E.; Atlan, T.; Sun, X.; Franek, R.; Siddiqui, A.; Marinov, G. K.; Shifman, S.; Zucker, D. M.; Oron-Gottesman, A.; Greenleaf, W. J.; Cohen, E.; Ram, O.; Harel, I.\nScore: 12.9, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572041\nClassical evolutionary theories propose tradeoffs between reproduction, damage repair, and lifespan. However, the specific role of the germline in shaping vertebrate aging remains largely unknown. Here, we use the turquoise killifish (N. furzeri) to genetically arrest germline differentiation at discrete stages, and examine how different flavors of infertility impact life-history. We first constructed a comprehensive single-cell gonadal atlas, providing cell-type-specific markers for downstream phenotypic analysis. Next, investigating our genetic models revealed that only germline depletion enhanced female damage repair, while arresting germline differentiation did not. Conversely, germline-depleted males were significantly long-lived, indicating that the mere presence of the germline can negatively affect lifespan. Transcriptomic analysis highlighted enrichment of pro-longevity pathways and genes, with functional conservation in germline-depleted C. elegans. Finally, germline depletion extended male healthspan through rejuvenated metabolic functions. Our results suggest that different germline manipulation paradigms can yield pronounced sexually dimorphic phenotypes, implying alternative mechanisms to classical evolutionary tradeoffs.\nThe balance between ATR and DDK activities controls TopBP1-mediated locking of dormant origins at the pre-IC stage\nAuthors: Koundrioukoff, S.; Kim, S.-J.; Alary, N.; Toffano, A.; Melendez-Garcia, R.; Wu, X.; Liu, Y.; Gnan, S.; El-Hilali, S.; Brison, O.; Rosselli, F.; Chen, C.-L.; Debatisse, M.\nScore: 7.8, Published: 2023-12-03 DOI: 10.1101/2023.11.29.569233\nReplication stress, a major hallmark of cancers, and ensuing genome instability source from impaired progression of replication forks. The first line of defense against fork slowing is compensation, a long-described process that elicits firing of otherwise dormant origins. It remains unclear whether compensation requires activation of the DNA replication checkpoint or passively results from lengthening of the window of time during which dormant origins can fire when fork progression slows, or both. Using molecular DNA combing we show here that a linear relationship ties inter-origin distances to fork speeds, independently of the checkpoint status. We called this line \"stressline\" and further show that its slope enables precise quantification of the compensation efficiency. Comparison of the slopes in different genetic backgrounds reveals that compensation requires ATR, not CHK1, while TopBP1 and CDC7/DBF4 repress dormant origin activation. These results strongly suggest that TopBP1 locks dormant origins at the pre-IC stage and that ATR and DDK oppose to control the conversion of dormant pre-ICs into functional salvage origins. Both passive and active processes thus contribute to compensation. Moreover, Repli-seq and OK-seq analyses confirm the activating role of ATR and permit development of ATRAP-seq, a new procedure allowing mapping of early constitutive origins.\npIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish\nAuthors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.\nScore: 52.2, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570868\nStandard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis. However, while the phiC31 system functions in zebrafish, validated loci carrying attP-based landing or safe harbor sites for transgene integration in zebrafish have not been established. Here, using CRISPR-Cas9, we converted two well-validated single insertion Tol2-based zebrafish transgenes with long-standing genetic stability into two attP landing sites, called phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET). Generating fluorescent reporters, loxP-based Switch lines, CreERT2 effector transgenes, and gene-regulatory variant reporters in the pIGLET14a and pIGLET24b landing site alleles, we document their suitability for transgenesis applications across cell types and developmental stages. For both landing sites, we routinely achieve 25-50% germline transmission of targeted transgene integrations, drastically reducing the number of required animals and necessary resources to generate individual transgenic lines. We document that phiC31 integrase-based transgenesis into pIGLET14a and pIGLET24b reproducibly results in representative reporter expression patterns in injected F0 zebrafish embryos for enhancer discovery and enables qualitative and quantitative comparison of gene-regulatory element variants. Taken together, our new phiC31 integrase-based transgene landing sites establish reproducible, targeted zebrafish transgenesis for numerous applications including quantitative experiments while greatly reducing the workload of generating new transgenic zebrafish lines. SHORT ABSTRACTTargeted transgenesis into pre-established, \"safe harbor,\" landing sites remains a missing technique in zebrafish. Here, we established phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET) by CRISPR-Cas9-based conversion of two previously validated Tol2 transgenes into attP sites. phiC31-mediated transgenesis into our pIGLET14a and pIGLET24b attP landing sites results in 25-50% germline transmission efficiency and quantifiable transgene activity. Our landing sites are suitable for reproducible, routine zebrafish transgenesis for diverse applications.\nTesting SIPA1L2 as a modifier of CMT1A using mouse models\nAuthors: Murray, G. C.; Hines, T. J.; Tadenev, A. L. D.; Xu, I.; Züchner, S.; Burgess, R. W.\nScore: 5.4, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569428\nCharcot-Marie-Tooth 1A is a demyelinating peripheral neuropathy caused by the duplication of peripheral myelin protein 22 (PMP22), which produces muscle weakness and loss of sensation in the hands and feet. A recent case-only genome wide association study by the Inherited Neuropathy Consortium identified a strong association between variants in signal induced proliferation associated 1 like 2 (SIPA1L2) and strength of foot dorsiflexion. To validate SIPA1L2 as a candidate modifier, and to assess its potential as a therapeutic target, we engineered mice with a deletion in SIPA1L2 and crossed them to the C3-PMP22 mouse model of CMT1A. We performed neuromuscular phenotyping and identified an interaction between Sipa1l2 deletion and muscular endurance decrements assayed by wire-hang duration in C3-PMP22 mice, as well as several interactions in femoral nerve axon morphometrics such as myelin thickness. Gene expression changes suggested an involvement of Sipa1l2 in cholesterol biosynthesis, which was also implicated in C3-PMP22 mice. Though several interactions between Sipa1l2 deletion and CMT1A-associated phenotypes were identified, validating a genetic interaction, the overall effect on neuropathy was small.\nIdentification of methylation markers for age and Bovine Respiratory Disease in dairy cattle\nAuthors: Attree, E.; Griffiths, B.; Panchal, K.; Xia, D.; Werling, D.; Banos, G.; Oikonomou, G.; Psifidi, A.\nScore: 4.0, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572169\nMethylation profiles of animals is known to differ by age and disease status. Bovine respiratory disease (BRD), a complex infectious disease, primarily affects calves and has significant impact on animal welfare and the cattle industry, predominantly from production losses. BRD susceptibility is multifactorial, influenced by both environmental and genetic factors. We investigated the epigenetic profile of BRD susceptibility in calves and age-related methylation differences between healthy calves and adult dairy cows using Reduced Representation Bisulfite Sequencing (RRBS). We identified 3,452 genes within differentially methylated regions between calves and adults. Functional analysis revealed enrichment of developmental pathways including cell fate commitment and tissue morphogenesis. Between healthy and BRD affected calves, 964 genes were identified within differentially methylated regions. Immune and vasculature regulatory pathways were enriched and key candidates in BRD susceptibility involved in complement cascade regulation, vasoconstriction and respiratory cilia structure and function were identified.\nA transient but very intense mutational burst occurs during the normal development of yeast colonies\nAuthors: Agier, N.; Vittorelli, N.; Chaux, F.; Gillet-Markowska, A.; O'Donnell, S.; Fischer, G.; Delmas, S.\nScore: 10.3, Published: 2023-12-11 DOI: 10.1101/2023.12.11.571082\nCharacterizing the pace of mutation accumulation is crucial not only for understanding how populations adapt to their environment but also for unraveling the intricate dynamics between gradual mutational processes and more sudden burst-like events occurring during cancer development. We engineered the genome of Saccharomyces cerevisiae to measure the rates of single and double mutations, including point mutations, segmental duplications and reciprocal translocations. We found that during the development of wild-type yeast colonies grown in absence of external stresses, double mutations occur at rates that are four to 17-fold higher than those expected on the basis of single mutation rates. We showed that double mutants retained wild-type mutation rates, revealing that they resulted from transient mutator episodes. Our results also indicated that the excess of double mutations would partly originate from the ELG1-dependent recombination salvage pathway. Additionally, simulating the tempo of mutation accumulation using the experimentally defined single and double mutation rates revealed that the transient mutator subpopulation would be limited to less than few thousands of cells and last for less than five generations, yet undergoes a remarkably intense mutational burst with mutation rates multiplied by hundreds or thousands. The majority of double mutations would accumulate sequentially, the simultaneous acquisition of both mutations during the same cell cycle being rare. This rarity might be linked to systemic genomic instability, aligning well with our experimental findings. In conclusion, our results suggest that transient hypermutators are major contributors to the mutational load and genomic instability accumulating in isogenic cell populations.\nCRISPR/Cas9-mediated genome editing of Frankliniella occidentalis, the western flower thrips, via embryonic microinjection\nAuthors: Han, J.; Klobasa, W.; de Oliveira, L.; Rotenberg, D.; Whitfield, A. E.; Lorenzen, M. D.\nScore: 15.7, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569456\nThe western flower thrips, Frankliniella occidentalis, poses a significant challenge in global agriculture as a notorious pest and a vector of economically significant orthotospoviruses. However, the limited availability of genetic tools for F. occidentalis hampers the advancement of functional genomics and the development of innovative pest control strategies. In this study, we present a robust methodology for generating heritable mutations in F. occidentalis using the CRISPR/Cas9 genome editing system. Two eye-color genes, white (Fo-w) and cinnabar (Fo-cn), frequently used to assess Cas9 function in insects were identified in the F. occidentalis genome and targeted for knockout through embryonic microinjection of Cas9 complexed with Fo-w or Fo-cn specific guide RNAs. Homozygous Fo-w and Fo-cn knockout lines were established by crossing mutant females and males. The Fo-w knockout line revealed an age-dependent modification of eye-color phenotype. Specifically, while young larvae exhibit ivory-colored eyes, the color transitions to bright red as they age. Unexpectedly, loss of Fo-w function also altered body color, with Fo-w mutants having a lighter colored body than wild type, suggesting a dual role for Fo-w in thrips. In contrast, individuals from the Fo-cn knockout line consistently displayed bright red eyes throughout all life stages. Molecular analyses validated precise editing of both target genes. This study offers a powerful tool to investigate thrips gene functions and paves the way for the development of genetic technologies for population suppression and/or population replacement as a means of mitigating virus transmission by this vector.\nStrawberry phenotypic plasticity in flowering time is driven by interaction between genetic loci and temperature\nAuthors: Prohaska, A.; Petit, A.; Lesemann, S.; Rey-Serra, P.; Mazzoni, L.; Masny, A.; Sanchez-Sevilla, J. F.; Potier, A.; Gaston, A.; Klamkowski, K.; Rothan, C.; Mezzetti, B.; Amaya, I.; Olbricht, K.; Denoyes, B.\nScore: 4.3, Published: 2023-12-01 DOI: 10.1101/2023.11.29.569202\nThe flowering time, which determines when the fruits or seeds can be harvested, is known to be sensitive to plasticity, i.e. the ability of a genotype to display different phenotypes in response to environmental variations. In the context of climate change, strawberry breeding can take advantage of phenotypic plasticity to create high-performing varieties adapted either to local conditions or to a wide range of climates. To decipher how the environment affects the genetic architecture of flowering time in cultivated strawberry (Fragaria xananassa) and modify its QTL effects, we used a bi-parental segregating population grown for two years at widely divergent latitudes (5 European countries) and combined climatic variables with genomic data (Affymetrix(R) SNP array). We detected 10 unique flowering time QTL and demonstrated that temperature modulates the effect of plasticity-related QTL. We propose candidate genes for the three main plasticity QTL, including FaTFL1 which is the most relevant candidate in the interval of the major temperature-sensitive QTL (6D_M). We further designed and validated a genetic marker for the 6D_M QTL which offers great potential for breeding programs, for example for selecting of early-flowering strawberry varieties well adapted to different environmental conditions. HighlightsA GXE study of a segregating strawberry population in Europe showed that temperature is the main driver of flowering time plasticity. A genetic marker was designed for the main QTL.\n",
  "wordCount" : "2505",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571760">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571760" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571760">
        <p class="paperTitle">Synergistic insulation of regulatory domains by developmental genes and clusters of CTCF sites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571760" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571760" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ealo, T.; Sanchez-Gaya, V.; Respuela, P.; Munoz-San Martin, M.; Martin-Batista, E.; Haro, E.; Rada-Iglesias, A.</p>
        <p class="info">Score: 18.6, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571760' target='https://doi.org/10.1101/2023.12.15.571760'> 10.1101/2023.12.15.571760</a></p>
        <p class="abstract">The specificity of gene expression during development requires the insulation of regulatory domains to avoid inappropriate enhancer-gene interactions. In vertebrates, this insulator function is mostly attributed to clusters of CTCF sites located at topologically associating domain (TAD) boundaries. However, TAD boundaries allow a certain level of physical crosstalk across regulatory domains, which is at odds with the highly specific and precise expression of developmental genes. Here we show that developmental genes and nearby clusters of CTCF sites synergistically foster the robust insulation of regulatory domains. Firstly, we found that the TADs containing developmental genes have distinctive features, including the sequential organization of developmental genes and CTCF clusters near TAD boundaries. Most importantly, by genetically dissecting representative loci in mouse embryonic stem cells, we showed that developmental genes and CTCF sites synergistically strengthened the insulation capacity of nearby boundaries through different mechanisms. Namely, while CTCF sites prevent undesirable enhancer-gene contacts (i.e. physical insulation), developmental genes preferentially contribute to regulatory insulation through non-structural mechanisms involving promoter competition rather than enhancer blocking. Overall, our work provides important insights into the specificity of gene regulation, which in turn might help interpreting the pathological consequences of certain structural variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572247">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572247" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572247">
        <p class="paperTitle">Between-group heritability and the status of hereditarianism as an evolutionary science</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572247" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572247" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roseman, C. C.; Bird, K.</p>
        <p class="info">Score: 13.9, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572247' target='https://doi.org/10.1101/2023.12.18.572247'> 10.1101/2023.12.18.572247</a></p>
        <p class="abstract">Hereditarianism is a school of thought that contends there are substantial evolved cognitive and behavioral differences among groups of humans which are both resistant to environmental intervention and are a root cause of differential social outcomes across groups. The relationship of between- group heritability (h2B) to within-group heritability (h2W) is one of the key theoretical components of hereditarianism and forms one of the bases for its claim to be an evolutionary science. Here, we examine the relationship between h2B and h2W and its application to problems in the hereditarian literature from an evolutionary genetic perspective. We demonstrate that the formulation of the relationship between h2B and h2W used in the hereditarian literature has no evolutionary content. By re-writing the relationship between h2B and h2W in a novel evolutionary framework, we demonstrate that there is no way to predict h2B using h2W without considerable additional theory that is absent from the hereditarian literature. Furthermore, we demonstrate that the hereditarian technique that uses h2B and h2W as a means of judging whether a given difference between groups may be plausibly ameliorated through environmental intervention is mathematically flawed. Lastly, we fill a gap in the hereditarian literature by writing out a means of using h2B to predict the absolute difference between groups under a neutral evolutionary model and find that it is much smaller than claimed by hereditarians. In conclusion, we propose a path forward for the study of human variation that moves us past the ill-conceived nature vs. nurture question and allows us to focus on more productive issues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572041">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572041" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572041">
        <p class="paperTitle">The germline regulates longevity and somatic repair in a sex-specific manner</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572041" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572041" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moses, E.; Atlan, T.; Sun, X.; Franek, R.; Siddiqui, A.; Marinov, G. K.; Shifman, S.; Zucker, D. M.; Oron-Gottesman, A.; Greenleaf, W. J.; Cohen, E.; Ram, O.; Harel, I.</p>
        <p class="info">Score: 12.9, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572041' target='https://doi.org/10.1101/2023.12.18.572041'> 10.1101/2023.12.18.572041</a></p>
        <p class="abstract">Classical evolutionary theories propose tradeoffs between reproduction, damage repair, and lifespan. However, the specific role of the germline in shaping vertebrate aging remains largely unknown. Here, we use the turquoise killifish (N. furzeri) to genetically arrest germline differentiation at discrete stages, and examine how different flavors of infertility impact life-history. We first constructed a comprehensive single-cell gonadal atlas, providing cell-type-specific markers for downstream phenotypic analysis. Next, investigating our genetic models revealed that only germline depletion enhanced female damage repair, while arresting germline differentiation did not. Conversely, germline-depleted males were significantly long-lived, indicating that the mere presence of the germline can negatively affect lifespan. Transcriptomic analysis highlighted enrichment of pro-longevity pathways and genes, with functional conservation in germline-depleted C. elegans. Finally, germline depletion extended male healthspan through rejuvenated metabolic functions. Our results suggest that different germline manipulation paradigms can yield pronounced sexually dimorphic phenotypes, implying alternative mechanisms to classical evolutionary tradeoffs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569233">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569233" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569233">
        <p class="paperTitle">The balance between ATR and DDK activities controls TopBP1-mediated locking of dormant origins at the pre-IC stage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569233" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569233" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koundrioukoff, S.; Kim, S.-J.; Alary, N.; Toffano, A.; Melendez-Garcia, R.; Wu, X.; Liu, Y.; Gnan, S.; El-Hilali, S.; Brison, O.; Rosselli, F.; Chen, C.-L.; Debatisse, M.</p>
        <p class="info">Score: 7.8, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569233' target='https://doi.org/10.1101/2023.11.29.569233'> 10.1101/2023.11.29.569233</a></p>
        <p class="abstract">Replication stress, a major hallmark of cancers, and ensuing genome instability source from impaired progression of replication forks. The first line of defense against fork slowing is compensation, a long-described process that elicits firing of otherwise dormant origins. It remains unclear whether compensation requires activation of the DNA replication checkpoint or passively results from lengthening of the window of time during which dormant origins can fire when fork progression slows, or both. Using molecular DNA combing we show here that a linear relationship ties inter-origin distances to fork speeds, independently of the checkpoint status. We called this line &#34;stressline&#34; and further show that its slope enables precise quantification of the compensation efficiency. Comparison of the slopes in different genetic backgrounds reveals that compensation requires ATR, not CHK1, while TopBP1 and CDC7/DBF4 repress dormant origin activation. These results strongly suggest that TopBP1 locks dormant origins at the pre-IC stage and that ATR and DDK oppose to control the conversion of dormant pre-ICs into functional salvage origins. Both passive and active processes thus contribute to compensation. Moreover, Repli-seq and OK-seq analyses confirm the activating role of ATR and permit development of ATRAP-seq, a new procedure allowing mapping of early constitutive origins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570868">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570868" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570868">
        <p class="paperTitle">pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570868" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570868" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.</p>
        <p class="info">Score: 52.2, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570868' target='https://doi.org/10.1101/2023.12.08.570868'> 10.1101/2023.12.08.570868</a></p>
        <p class="abstract">Standard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis. However, while the phiC31 system functions in zebrafish, validated loci carrying attP-based landing or safe harbor sites for transgene integration in zebrafish have not been established. Here, using CRISPR-Cas9, we converted two well-validated single insertion Tol2-based zebrafish transgenes with long-standing genetic stability into two attP landing sites, called phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET). Generating fluorescent reporters, loxP-based Switch lines, CreERT2 effector transgenes, and gene-regulatory variant reporters in the pIGLET14a and pIGLET24b landing site alleles, we document their suitability for transgenesis applications across cell types and developmental stages. For both landing sites, we routinely achieve 25-50% germline transmission of targeted transgene integrations, drastically reducing the number of required animals and necessary resources to generate individual transgenic lines. We document that phiC31 integrase-based transgenesis into pIGLET14a and pIGLET24b reproducibly results in representative reporter expression patterns in injected F0 zebrafish embryos for enhancer discovery and enables qualitative and quantitative comparison of gene-regulatory element variants. Taken together, our new phiC31 integrase-based transgene landing sites establish reproducible, targeted zebrafish transgenesis for numerous applications including quantitative experiments while greatly reducing the workload of generating new transgenic zebrafish lines.

SHORT ABSTRACTTargeted transgenesis into pre-established, &#34;safe harbor,&#34; landing sites remains a missing technique in zebrafish. Here, we established phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET) by CRISPR-Cas9-based conversion of two previously validated Tol2 transgenes into attP sites. phiC31-mediated transgenesis into our pIGLET14a and pIGLET24b attP landing sites results in 25-50% germline transmission efficiency and quantifiable transgene activity. Our landing sites are suitable for reproducible, routine zebrafish transgenesis for diverse applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569428">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569428" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569428">
        <p class="paperTitle">Testing SIPA1L2 as a modifier of CMT1A using mouse models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569428" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569428" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Murray, G. C.; Hines, T. J.; Tadenev, A. L. D.; Xu, I.; Züchner, S.; Burgess, R. W.</p>
        <p class="info">Score: 5.4, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569428' target='https://doi.org/10.1101/2023.11.30.569428'> 10.1101/2023.11.30.569428</a></p>
        <p class="abstract">Charcot-Marie-Tooth 1A is a demyelinating peripheral neuropathy caused by the duplication of peripheral myelin protein 22 (PMP22), which produces muscle weakness and loss of sensation in the hands and feet. A recent case-only genome wide association study by the Inherited Neuropathy Consortium identified a strong association between variants in signal induced proliferation associated 1 like 2 (SIPA1L2) and strength of foot dorsiflexion. To validate SIPA1L2 as a candidate modifier, and to assess its potential as a therapeutic target, we engineered mice with a deletion in SIPA1L2 and crossed them to the C3-PMP22 mouse model of CMT1A. We performed neuromuscular phenotyping and identified an interaction between Sipa1l2 deletion and muscular endurance decrements assayed by wire-hang duration in C3-PMP22 mice, as well as several interactions in femoral nerve axon morphometrics such as myelin thickness. Gene expression changes suggested an involvement of Sipa1l2 in cholesterol biosynthesis, which was also implicated in C3-PMP22 mice. Though several interactions between Sipa1l2 deletion and CMT1A-associated phenotypes were identified, validating a genetic interaction, the overall effect on neuropathy was small.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572169">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572169" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572169">
        <p class="paperTitle">Identification of methylation markers for age and Bovine Respiratory Disease in dairy cattle</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572169" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572169" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Attree, E.; Griffiths, B.; Panchal, K.; Xia, D.; Werling, D.; Banos, G.; Oikonomou, G.; Psifidi, A.</p>
        <p class="info">Score: 4.0, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572169' target='https://doi.org/10.1101/2023.12.18.572169'> 10.1101/2023.12.18.572169</a></p>
        <p class="abstract">Methylation profiles of animals is known to differ by age and disease status. Bovine respiratory disease (BRD), a complex infectious disease, primarily affects calves and has significant impact on animal welfare and the cattle industry, predominantly from production losses. BRD susceptibility is multifactorial, influenced by both environmental and genetic factors. We investigated the epigenetic profile of BRD susceptibility in calves and age-related methylation differences between healthy calves and adult dairy cows using Reduced Representation Bisulfite Sequencing (RRBS). We identified 3,452 genes within differentially methylated regions between calves and adults. Functional analysis revealed enrichment of developmental pathways including cell fate commitment and tissue morphogenesis. Between healthy and BRD affected calves, 964 genes were identified within differentially methylated regions. Immune and vasculature regulatory pathways were enriched and key candidates in BRD susceptibility involved in complement cascade regulation, vasoconstriction and respiratory cilia structure and function were identified.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.571082">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.571082" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.571082">
        <p class="paperTitle">A transient but very intense mutational burst occurs during the normal development of yeast colonies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.571082" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.571082" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Agier, N.; Vittorelli, N.; Chaux, F.; Gillet-Markowska, A.; O&#39;Donnell, S.; Fischer, G.; Delmas, S.</p>
        <p class="info">Score: 10.3, Published: 2023-12-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.571082' target='https://doi.org/10.1101/2023.12.11.571082'> 10.1101/2023.12.11.571082</a></p>
        <p class="abstract">Characterizing the pace of mutation accumulation is crucial not only for understanding how populations adapt to their environment but also for unraveling the intricate dynamics between gradual mutational processes and more sudden burst-like events occurring during cancer development. We engineered the genome of Saccharomyces cerevisiae to measure the rates of single and double mutations, including point mutations, segmental duplications and reciprocal translocations. We found that during the development of wild-type yeast colonies grown in absence of external stresses, double mutations occur at rates that are four to 17-fold higher than those expected on the basis of single mutation rates. We showed that double mutants retained wild-type mutation rates, revealing that they resulted from transient mutator episodes. Our results also indicated that the excess of double mutations would partly originate from the ELG1-dependent recombination salvage pathway. Additionally, simulating the tempo of mutation accumulation using the experimentally defined single and double mutation rates revealed that the transient mutator subpopulation would be limited to less than few thousands of cells and last for less than five generations, yet undergoes a remarkably intense mutational burst with mutation rates multiplied by hundreds or thousands. The majority of double mutations would accumulate sequentially, the simultaneous acquisition of both mutations during the same cell cycle being rare. This rarity might be linked to systemic genomic instability, aligning well with our experimental findings. In conclusion, our results suggest that transient hypermutators are major contributors to the mutational load and genomic instability accumulating in isogenic cell populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569456">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569456" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569456">
        <p class="paperTitle">CRISPR/Cas9-mediated genome editing of Frankliniella occidentalis, the western flower thrips, via embryonic microinjection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569456" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569456" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Han, J.; Klobasa, W.; de Oliveira, L.; Rotenberg, D.; Whitfield, A. E.; Lorenzen, M. D.</p>
        <p class="info">Score: 15.7, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569456' target='https://doi.org/10.1101/2023.11.30.569456'> 10.1101/2023.11.30.569456</a></p>
        <p class="abstract">The western flower thrips, Frankliniella occidentalis, poses a significant challenge in global agriculture as a notorious pest and a vector of economically significant orthotospoviruses. However, the limited availability of genetic tools for F. occidentalis hampers the advancement of functional genomics and the development of innovative pest control strategies. In this study, we present a robust methodology for generating heritable mutations in F. occidentalis using the CRISPR/Cas9 genome editing system. Two eye-color genes, white (Fo-w) and cinnabar (Fo-cn), frequently used to assess Cas9 function in insects were identified in the F. occidentalis genome and targeted for knockout through embryonic microinjection of Cas9 complexed with Fo-w or Fo-cn specific guide RNAs. Homozygous Fo-w and Fo-cn knockout lines were established by crossing mutant females and males. The Fo-w knockout line revealed an age-dependent modification of eye-color phenotype. Specifically, while young larvae exhibit ivory-colored eyes, the color transitions to bright red as they age. Unexpectedly, loss of Fo-w function also altered body color, with Fo-w mutants having a lighter colored body than wild type, suggesting a dual role for Fo-w in thrips. In contrast, individuals from the Fo-cn knockout line consistently displayed bright red eyes throughout all life stages. Molecular analyses validated precise editing of both target genes. This study offers a powerful tool to investigate thrips gene functions and paves the way for the development of genetic technologies for population suppression and/or population replacement as a means of mitigating virus transmission by this vector.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569202">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569202" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569202">
        <p class="paperTitle">Strawberry phenotypic plasticity in flowering time is driven by interaction between genetic loci and temperature</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569202" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569202" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prohaska, A.; Petit, A.; Lesemann, S.; Rey-Serra, P.; Mazzoni, L.; Masny, A.; Sanchez-Sevilla, J. F.; Potier, A.; Gaston, A.; Klamkowski, K.; Rothan, C.; Mezzetti, B.; Amaya, I.; Olbricht, K.; Denoyes, B.</p>
        <p class="info">Score: 4.3, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569202' target='https://doi.org/10.1101/2023.11.29.569202'> 10.1101/2023.11.29.569202</a></p>
        <p class="abstract">The flowering time, which determines when the fruits or seeds can be harvested, is known to be sensitive to plasticity, i.e. the ability of a genotype to display different phenotypes in response to environmental variations. In the context of climate change, strawberry breeding can take advantage of phenotypic plasticity to create high-performing varieties adapted either to local conditions or to a wide range of climates. To decipher how the environment affects the genetic architecture of flowering time in cultivated strawberry (Fragaria xananassa) and modify its QTL effects, we used a bi-parental segregating population grown for two years at widely divergent latitudes (5 European countries) and combined climatic variables with genomic data (Affymetrix(R) SNP array). We detected 10 unique flowering time QTL and demonstrated that temperature modulates the effect of plasticity-related QTL. We propose candidate genes for the three main plasticity QTL, including FaTFL1 which is the most relevant candidate in the interval of the major temperature-sensitive QTL (6D_M). We further designed and validated a genetic marker for the 6D_M QTL which offers great potential for breeding programs, for example for selecting of early-flowering strawberry varieties well adapted to different environmental conditions.

HighlightsA GXE study of a segregating strawberry population in Europe showed that temperature is the main driver of flowering time plasticity. A genetic marker was designed for the main QTL.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
