<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish
Authors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.
Score: 47.3, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570868
Standard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish
Authors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.
Score: 47.3, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570868
Standard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish
Authors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.
Score: 47.3, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570868
Standard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish\nAuthors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.\nScore: 47.3, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570868\nStandard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis.",
  "keywords": [
    
  ],
  "articleBody": " pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish\nAuthors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.\nScore: 47.3, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570868\nStandard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis. However, while the phiC31 system functions in zebrafish, validated loci carrying attP-based landing or safe harbor sites for transgene integration in zebrafish have not been established. Here, using CRISPR-Cas9, we converted two well-validated single insertion Tol2-based zebrafish transgenes with long-standing genetic stability into two attP landing sites, called phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET). Generating fluorescent reporters, loxP-based Switch lines, CreERT2 effector transgenes, and gene-regulatory variant reporters in the pIGLET14a and pIGLET24b landing site alleles, we document their suitability for transgenesis applications across cell types and developmental stages. For both landing sites, we routinely achieve 25-50% germline transmission of targeted transgene integrations, drastically reducing the number of required animals and necessary resources to generate individual transgenic lines. We document that phiC31 integrase-based transgenesis into pIGLET14a and pIGLET24b reproducibly results in representative reporter expression patterns in injected F0 zebrafish embryos for enhancer discovery and enables qualitative and quantitative comparison of gene-regulatory element variants. Taken together, our new phiC31 integrase-based transgene landing sites establish reproducible, targeted zebrafish transgenesis for numerous applications including quantitative experiments while greatly reducing the workload of generating new transgenic zebrafish lines. SHORT ABSTRACTTargeted transgenesis into pre-established, \"safe harbor,\" landing sites remains a missing technique in zebrafish. Here, we established phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET) by CRISPR-Cas9-based conversion of two previously validated Tol2 transgenes into attP sites. phiC31-mediated transgenesis into our pIGLET14a and pIGLET24b attP landing sites results in 25-50% germline transmission efficiency and quantifiable transgene activity. Our landing sites are suitable for reproducible, routine zebrafish transgenesis for diverse applications.\nEmergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases\nAuthors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.\nScore: 127.0, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569447\nTransposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6. We set out to test the hypothesis that TnpB nucleases may have also been repurposed for novel, unexpected functions other than CRIS-PR-Cas. Here, using phylogenetics, structural predictions, comparative genomics, and functional assays, we uncover multiple instances of programmable transcription factors that we name TnpB-like nuclease-dead repressors (TldR). These proteins employ naturally occurring guide RNAs to specifically target conserved promoter regions of the genome, leading to potent gene repression in a mechanism akin to CRISPRi technologies invented by humans7. Focusing on a TldR clade found broadly in Enterobacteriaceae, we discover that bacteriophages exploit the combined action of TldR and an adjacently encoded phage gene to alter the expression and composition of the host flagellar assembly, a transformation with the potential to impact motility8, phage susceptibility9, and host immunity10. Col-lectively, this work showcases the diverse molecular innovations that were enabled through repeated exaptation of genes encoded by transposable elements, and reveals that RNA-guided transcription factors emerged long before the development of dCas9-based editors.\nCRISPR/Cas9-mediated genome editing of Frankliniella occidentalis, the western flower thrips, via embryonic microinjection\nAuthors: Han, J.; Klobasa, W.; de Oliveira, L.; Rotenberg, D.; Whitfield, A. E.; Lorenzen, M. D.\nScore: 13.2, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569456\nThe western flower thrips, Frankliniella occidentalis, poses a significant challenge in global agriculture as a notorious pest and a vector of economically significant orthotospoviruses. However, the limited availability of genetic tools for F. occidentalis hampers the advancement of functional genomics and the development of innovative pest control strategies. In this study, we present a robust methodology for generating heritable mutations in F. occidentalis using the CRISPR/Cas9 genome editing system. Two eye-color genes, white (Fo-w) and cinnabar (Fo-cn), frequently used to assess Cas9 function in insects were identified in the F. occidentalis genome and targeted for knockout through embryonic microinjection of Cas9 complexed with Fo-w or Fo-cn specific guide RNAs. Homozygous Fo-w and Fo-cn knockout lines were established by crossing mutant females and males. The Fo-w knockout line revealed an age-dependent modification of eye-color phenotype. Specifically, while young larvae exhibit ivory-colored eyes, the color transitions to bright red as they age. Unexpectedly, loss of Fo-w function also altered body color, with Fo-w mutants having a lighter colored body than wild type, suggesting a dual role for Fo-w in thrips. In contrast, individuals from the Fo-cn knockout line consistently displayed bright red eyes throughout all life stages. Molecular analyses validated precise editing of both target genes. This study offers a powerful tool to investigate thrips gene functions and paves the way for the development of genetic technologies for population suppression and/or population replacement as a means of mitigating virus transmission by this vector.\nSingle-cell eQTL mapping in yeast reveals a tradeoff between growth and reproduction\nAuthors: Boocock, J.; Alexander, N.; Tapia, L. A.; Walter-McNeill, L.; Munugala, C.; Bloom, J. S.; Kruglyak, L.\nScore: 10.7, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570640\nExpression quantitative trait loci (eQTLs) provide a key bridge between noncoding DNA sequence variants and organismal traits. The effects of eQTLs can differ among tissues, cell types, and cellular states, but these differences are obscured by gene expression measurements in bulk populations. We developed a one-pot approach to map eQTLs in Saccharomyces cerevisiae by single-cell RNA sequencing (scRNA-seq) and applied it to over 100,000 single cells from three crosses. We used scRNA-seq data to genotype each cell, measure gene expression, and classify the cells by cell-cycle stage. We mapped thousands of local and distant eQTLs and identified interactions between eQTL effects and cell-cycle stages. We took advantage of single-cell expression information to identify hundreds of genes with allele-specific effects on expression noise. We used cell-cycle stage classification to map 20 loci that influence cell-cycle progression. One of these loci influenced the expression of genes involved in the mating response. We showed that the effects of this locus arise from a common variant (W82R) in the gene GPA1, which encodes a signaling protein that negatively regulates the mating pathway. The 82R allele increases mating efficiency at the cost of slower cell-cycle progression and is associated with a higher rate of outcrossing in nature. Our results provide a more granular picture of the effects of genetic variants on gene expression and downstream traits.\nA transient but very intense mutational burst occurs during the normal development of yeast colonies\nAuthors: Agier, N.; Vittorelli, N.; Chaux, F.; Gillet-Markowska, A.; O'Donnell, S.; Fischer, G.; Delmas, S.\nScore: 10.3, Published: 2023-12-11 DOI: 10.1101/2023.12.11.571082\nCharacterizing the pace of mutation accumulation is crucial not only for understanding how populations adapt to their environment but also for unraveling the intricate dynamics between gradual mutational processes and more sudden burst-like events occurring during cancer development. We engineered the genome of Saccharomyces cerevisiae to measure the rates of single and double mutations, including point mutations, segmental duplications and reciprocal translocations. We found that during the development of wild-type yeast colonies grown in absence of external stresses, double mutations occur at rates that are four to 17-fold higher than those expected on the basis of single mutation rates. We showed that double mutants retained wild-type mutation rates, revealing that they resulted from transient mutator episodes. Our results also indicated that the excess of double mutations would partly originate from the ELG1-dependent recombination salvage pathway. Additionally, simulating the tempo of mutation accumulation using the experimentally defined single and double mutation rates revealed that the transient mutator subpopulation would be limited to less than few thousands of cells and last for less than five generations, yet undergoes a remarkably intense mutational burst with mutation rates multiplied by hundreds or thousands. The majority of double mutations would accumulate sequentially, the simultaneous acquisition of both mutations during the same cell cycle being rare. This rarity might be linked to systemic genomic instability, aligning well with our experimental findings. In conclusion, our results suggest that transient hypermutators are major contributors to the mutational load and genomic instability accumulating in isogenic cell populations.\nSomatic Mosaicism in Amyotrophic Lateral Sclerosis and Frontotemporal Dementia Reveals Widespread Degeneration from Focal Mutations\nAuthors: Zhou, Z.; Kim, J.; Huang, A. Y.; Nolan, M.; Park, J.; Doan, R.; Shin, T.; Miller, M. B.; Chhouk, B.; Morillo, K.; Yeh, R. C.; Kenny, C.; Neil, J. E.; Lee, C.-Z.; Ohkubo, T.; Ravits, J.; Ansorge, O.; Ostrow, L. W.; Lagier-Tourenne, C.; Lee, E. A.; Walsh, C. A.\nScore: 23.9, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569436\nAlthough mutations in dozens of genes have been implicated in familial forms of amyotrophic lateral sclerosis (fALS) and frontotemporal degeneration (fFTD), most cases of these conditions are sporadic (sALS and sFTD), with no family history, and their etiology remains obscure. We tested the hypothesis that somatic mosaic mutations, present in some but not all cells, might contribute in these cases, by performing ultra-deep, targeted sequencing of 88 genes associated with neurodegenerative diseases in postmortem brain and spinal cord samples from 404 individuals with sALS or sFTD and 144 controls. Known pathogenic germline mutations were found in 20.6% of ALS, and 26.5% of FTD cases. Predicted pathogenic somatic mutations in ALS/FTD genes were observed in 2.7% of sALS and sFTD cases that did not carry known pathogenic or novel germline mutations. Somatic mutations showed low variant allele fraction (typically \u003c2%) and were often restricted to the region of initial discovery, preventing detection through genetic screening in peripheral tissues. Damaging somatic mutations were preferentially enriched in primary motor cortex of sALS and prefrontal cortex of sFTD, mirroring regions most severely affected in each disease. Somatic mutation analysis of bulk RNA-seq data from brain and spinal cord from an additional 143 sALS cases and 23 controls confirmed an overall enrichment of somatic mutations in sALS. Two adult sALS cases were identified bearing pathogenic somatic mutations in DYNC1H1 and LMNA, two genes associated with pediatric motor neuron degeneration. Our study suggests that somatic mutations in fALS/fFTD genes, and in genes associated with more severe diseases in the germline state, contribute to sALS and sFTD, and that mosaic mutations in a small fraction of cells in focal regions of the nervous system can ultimately result in widespread degeneration.\nDigital telomere measurement by long-read sequencing distinguishes healthy aging from disease\nAuthors: Sanchez, S. E.; Gu, J.; Golla, A.; Martin, A.; Shomali, W.; Hockemeyer, D.; Savage, S.; Artandi, S.\nScore: 17.9, Published: 2023-12-01 DOI: 10.1101/2023.11.29.569263\nTelomere length is an important biomarker of organismal aging and cellular replicative potential, but existing measurement methods are limited in resolution and accuracy. Here, we deploy digital telomere measurement by nanopore sequencing to understand how distributions of human telomere length change with age and disease. We measure telomere attrition and de novo elongation with unprecedented resolution in genetically defined populations of human cells, in blood cells from healthy donors and in blood cells from patients with genetic defects in telomere maintenance. We find that human aging is accompanied by a progressive loss of long telomeres and an accumulation of shorter telomeres. In patients with defects in telomere maintenance, the accumulation of short telomeres is more pronounced and correlates with phenotypic severity. We apply machine learning to train a binary classification model that distinguishes healthy individuals from those with telomere biology disorders. This sequencing and bioinformatic pipeline will advance our understanding of telomere maintenance mechanisms and the use of telomere length as a clinical biomarker of aging and disease.\nImproving the suppressive power of homing gene drive by co-targeting a distant-site female fertility gene\nAuthors: Faber, N. R.; Xu, X.; Chen, J.; Hou, S.; Du, J.; Pannebakker, B. A.; Zwaan, B. J.; van den Heuvel, J.; Champer, J.\nScore: 6.7, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570117\nGene drive technology has the potential to address major biological challenges, including the management of disease vectors, invasive species, and agricultural pests. After releasing individuals carrying the gene drive in the target population, suppression gene drives are designed to spread at a rapid rate and carry a recessive fitness cost, thus bringing about a decline in population size or even complete suppression. Well-studied homing suppression drives have been shown to be highly efficient in Anopheles mosquitoes and were successful in eliminating large cage populations. However, for other organisms, including Aedes mosquitoes, homing gene drives are so far too inefficient to achieve complete population suppression, mainly due to lower rates of drive conversion, which is the rate at which wild type alleles are converted into drive alleles. Low drive conversion is also a major issue in vertebrates, as indicated by experiments in mice. To tackle this issue, we propose a novel gene drive design that has two targets: a homing site where the drive is located and drive conversion takes place (with rescue for an essential gene), and a distant site for providing the fitness cost for population suppression (preferably a female fertility gene, for which no rescue is provided). We modeled this design and found that the two-target system allows suppression to occur over a much wider range of drive conversion efficiency. Specifically, in the new design, the suppressive power depends mostly on total gRNA cutting efficiency instead of just drive conversion efficiency, which is advantageous because cut rates are often substantially higher than drive conversion rates. We constructed a proof of concept in Drosophila melanogaster and show that both components of the gene drive function successfully. However, embryo drive activity from maternally deposited Cas9 as well as fitness costs for female drive heterozygotes both remain significant challenges for two-target and standard suppression drives. Overall, our improved gene drive design eases the development of strong homing suppression gene drives for many species where drive conversion is less efficient.\nA mouse model of ZTTK syndrome reveals indispensable SON functions in organ development and hematopoiesis\nAuthors: Vukadin, L.; Park, B.; Mohamed, M.; Li, H.; Elkholy, A.; Torrelli-Diljohn, A.; Kim, J.-H.; Jeong, K.; Murphy, J. M.; Harvey, C. A.; Dunlap, S.; Gehrs, L.; Lee, H.; Kim, H.-G.; Lee, S. N.; Stanford, D.; Barrington, R. A.; Foote, J.; Sorace, A. G.; Welner, R. S.; Hildreth, B. E.; Lim, S.-T. S.; Ahn, E.-Y. E.\nScore: 11.5, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567732\nRare diseases are underrepresented in biomedical research, leading to insufficient awareness. Zhu-Tokita-Takenouchi-Kim (ZTTK) syndrome is a rare disease caused by genetic alterations that result in heterozygous loss-of-function of SON. While ZTTK syndrome patients suffer from numerous symptoms, the lack of model organisms hamper our understanding of both SON and this complex syndrome. Here, we developed Son haploinsufficiency (Son+/-) mice as a model of ZTTK syndrome and identified the indispensable roles of Son in organ development and hematopoiesis. Son+/- mice recapitulated clinical symptoms of ZTTK syndrome, including growth retardation, cognitive impairment, skeletal abnormalities, and kidney agenesis. Furthermore, we identified hematopoietic abnormalities in Son+/- mice, similar to those observed in human patients. Surface marker analyses and single-cell transcriptome profiling of hematopoietic stem and progenitor cells revealed that Son haploinsufficiency inclines cell fate toward the myeloid lineage but compromises lymphoid lineage development by reducing key genes required for lymphoid and B cell lineage specification. Additionally, Son haploinsufficiency causes inappropriate activation of erythroid genes and impaired erythroid maturation. These findings highlight the importance of the full gene dosage of Son in organ development and hematopoiesis. Our model serves as an invaluable research tool for this rare disease and related disorders associated with SON dysfunction.\nMechanism of homology search expansion during recombinational DNA break repair\nAuthors: Dumont, A.; Mendiboure, N.; Savocco, J.; Anani, L.; Moreau, P.; Thierry, A.; Modolo, L.; Jost, D.; Piazza, A.\nScore: 25.5, Published: 2023-12-01 DOI: 10.1101/2023.12.01.569403\nHomology search catalyzed by a RecA/Rad51 nucleoprotein filament (NPF) is a central step of DNA double-strand break (DSB) repair by homologous recombination. How it operates in cells remains elusive. Here we developed a Hi-C-based methodology to map single-stranded DNA (ssDNA) contacts genome-wide in S. cerevisiae, which revealed two main homology search phases. Initial search conducted by short NPFs is confined in cis by cohesin-mediated chromatin loop folding. Progressive growth of stiff NPFs enables exploration of distant genomic sites. Long-range resection by Exo1 drives this transition from local to genome-wide search by providing ssDNA substrates for assembly of extensive NPFs. DSB end-tethering promotes coordinated homology search by NPFs formed on the two DSB ends. Finally, an autonomous genetic element on chromosome III engages the NPF and stimulates homology search in its vicinity. This work reveals the mechanism of the progressive and uneven expansion of homology search orchestrated by chromatin organizers, long-range resection, end-tethering, specialized genetic elements, and that exploits the stiff NPF structure conferred by Rad51 oligomerization. Highlights- Cohesin-mediated chromatin loops constrain homology search in cis for NPF regions close to the resection front - Stiffening of ssDNA by Rad51 enables genome-wide homology search by DSB-proximal sites - Exo1-mediated long-range resection promotes genome-wide homology search - DSB end-tethering promotes coordinated homology search by NPFs formed on both DSB ends - The recombination enhancer focuses homology search in its vicinity\n",
  "wordCount" : "2858",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570868">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570868" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570868">
        <p class="paperTitle">pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570868" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570868" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.</p>
        <p class="info">Score: 47.3, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570868' target='https://doi.org/10.1101/2023.12.08.570868'> 10.1101/2023.12.08.570868</a></p>
        <p class="abstract">Standard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis. However, while the phiC31 system functions in zebrafish, validated loci carrying attP-based landing or safe harbor sites for transgene integration in zebrafish have not been established. Here, using CRISPR-Cas9, we converted two well-validated single insertion Tol2-based zebrafish transgenes with long-standing genetic stability into two attP landing sites, called phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET). Generating fluorescent reporters, loxP-based Switch lines, CreERT2 effector transgenes, and gene-regulatory variant reporters in the pIGLET14a and pIGLET24b landing site alleles, we document their suitability for transgenesis applications across cell types and developmental stages. For both landing sites, we routinely achieve 25-50% germline transmission of targeted transgene integrations, drastically reducing the number of required animals and necessary resources to generate individual transgenic lines. We document that phiC31 integrase-based transgenesis into pIGLET14a and pIGLET24b reproducibly results in representative reporter expression patterns in injected F0 zebrafish embryos for enhancer discovery and enables qualitative and quantitative comparison of gene-regulatory element variants. Taken together, our new phiC31 integrase-based transgene landing sites establish reproducible, targeted zebrafish transgenesis for numerous applications including quantitative experiments while greatly reducing the workload of generating new transgenic zebrafish lines.

SHORT ABSTRACTTargeted transgenesis into pre-established, &#34;safe harbor,&#34; landing sites remains a missing technique in zebrafish. Here, we established phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET) by CRISPR-Cas9-based conversion of two previously validated Tol2 transgenes into attP sites. phiC31-mediated transgenesis into our pIGLET14a and pIGLET24b attP landing sites results in 25-50% germline transmission efficiency and quantifiable transgene activity. Our landing sites are suitable for reproducible, routine zebrafish transgenesis for diverse applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569447">
        <p class="paperTitle">Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.</p>
        <p class="info">Score: 127.0, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569447' target='https://doi.org/10.1101/2023.11.30.569447'> 10.1101/2023.11.30.569447</a></p>
        <p class="abstract">Transposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6. We set out to test the hypothesis that TnpB nucleases may have also been repurposed for novel, unexpected functions other than CRIS-PR-Cas. Here, using phylogenetics, structural predictions, comparative genomics, and functional assays, we uncover multiple instances of programmable transcription factors that we name TnpB-like nuclease-dead repressors (TldR). These proteins employ naturally occurring guide RNAs to specifically target conserved promoter regions of the genome, leading to potent gene repression in a mechanism akin to CRISPRi technologies invented by humans7. Focusing on a TldR clade found broadly in Enterobacteriaceae, we discover that bacteriophages exploit the combined action of TldR and an adjacently encoded phage gene to alter the expression and composition of the host flagellar assembly, a transformation with the potential to impact motility8, phage susceptibility9, and host immunity10. Col-lectively, this work showcases the diverse molecular innovations that were enabled through repeated exaptation of genes encoded by transposable elements, and reveals that RNA-guided transcription factors emerged long before the development of dCas9-based editors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569456">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569456" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569456">
        <p class="paperTitle">CRISPR/Cas9-mediated genome editing of Frankliniella occidentalis, the western flower thrips, via embryonic microinjection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569456" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569456" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Han, J.; Klobasa, W.; de Oliveira, L.; Rotenberg, D.; Whitfield, A. E.; Lorenzen, M. D.</p>
        <p class="info">Score: 13.2, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569456' target='https://doi.org/10.1101/2023.11.30.569456'> 10.1101/2023.11.30.569456</a></p>
        <p class="abstract">The western flower thrips, Frankliniella occidentalis, poses a significant challenge in global agriculture as a notorious pest and a vector of economically significant orthotospoviruses. However, the limited availability of genetic tools for F. occidentalis hampers the advancement of functional genomics and the development of innovative pest control strategies. In this study, we present a robust methodology for generating heritable mutations in F. occidentalis using the CRISPR/Cas9 genome editing system. Two eye-color genes, white (Fo-w) and cinnabar (Fo-cn), frequently used to assess Cas9 function in insects were identified in the F. occidentalis genome and targeted for knockout through embryonic microinjection of Cas9 complexed with Fo-w or Fo-cn specific guide RNAs. Homozygous Fo-w and Fo-cn knockout lines were established by crossing mutant females and males. The Fo-w knockout line revealed an age-dependent modification of eye-color phenotype. Specifically, while young larvae exhibit ivory-colored eyes, the color transitions to bright red as they age. Unexpectedly, loss of Fo-w function also altered body color, with Fo-w mutants having a lighter colored body than wild type, suggesting a dual role for Fo-w in thrips. In contrast, individuals from the Fo-cn knockout line consistently displayed bright red eyes throughout all life stages. Molecular analyses validated precise editing of both target genes. This study offers a powerful tool to investigate thrips gene functions and paves the way for the development of genetic technologies for population suppression and/or population replacement as a means of mitigating virus transmission by this vector.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570640">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570640" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570640">
        <p class="paperTitle">Single-cell eQTL mapping in yeast reveals a tradeoff between growth and reproduction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570640" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570640" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boocock, J.; Alexander, N.; Tapia, L. A.; Walter-McNeill, L.; Munugala, C.; Bloom, J. S.; Kruglyak, L.</p>
        <p class="info">Score: 10.7, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570640' target='https://doi.org/10.1101/2023.12.07.570640'> 10.1101/2023.12.07.570640</a></p>
        <p class="abstract">Expression quantitative trait loci (eQTLs) provide a key bridge between noncoding DNA sequence variants and organismal traits. The effects of eQTLs can differ among tissues, cell types, and cellular states, but these differences are obscured by gene expression measurements in bulk populations. We developed a one-pot approach to map eQTLs in Saccharomyces cerevisiae by single-cell RNA sequencing (scRNA-seq) and applied it to over 100,000 single cells from three crosses. We used scRNA-seq data to genotype each cell, measure gene expression, and classify the cells by cell-cycle stage. We mapped thousands of local and distant eQTLs and identified interactions between eQTL effects and cell-cycle stages. We took advantage of single-cell expression information to identify hundreds of genes with allele-specific effects on expression noise. We used cell-cycle stage classification to map 20 loci that influence cell-cycle progression. One of these loci influenced the expression of genes involved in the mating response. We showed that the effects of this locus arise from a common variant (W82R) in the gene GPA1, which encodes a signaling protein that negatively regulates the mating pathway. The 82R allele increases mating efficiency at the cost of slower cell-cycle progression and is associated with a higher rate of outcrossing in nature. Our results provide a more granular picture of the effects of genetic variants on gene expression and downstream traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.571082">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.571082" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.571082">
        <p class="paperTitle">A transient but very intense mutational burst occurs during the normal development of yeast colonies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.571082" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.571082" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Agier, N.; Vittorelli, N.; Chaux, F.; Gillet-Markowska, A.; O&#39;Donnell, S.; Fischer, G.; Delmas, S.</p>
        <p class="info">Score: 10.3, Published: 2023-12-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.571082' target='https://doi.org/10.1101/2023.12.11.571082'> 10.1101/2023.12.11.571082</a></p>
        <p class="abstract">Characterizing the pace of mutation accumulation is crucial not only for understanding how populations adapt to their environment but also for unraveling the intricate dynamics between gradual mutational processes and more sudden burst-like events occurring during cancer development. We engineered the genome of Saccharomyces cerevisiae to measure the rates of single and double mutations, including point mutations, segmental duplications and reciprocal translocations. We found that during the development of wild-type yeast colonies grown in absence of external stresses, double mutations occur at rates that are four to 17-fold higher than those expected on the basis of single mutation rates. We showed that double mutants retained wild-type mutation rates, revealing that they resulted from transient mutator episodes. Our results also indicated that the excess of double mutations would partly originate from the ELG1-dependent recombination salvage pathway. Additionally, simulating the tempo of mutation accumulation using the experimentally defined single and double mutation rates revealed that the transient mutator subpopulation would be limited to less than few thousands of cells and last for less than five generations, yet undergoes a remarkably intense mutational burst with mutation rates multiplied by hundreds or thousands. The majority of double mutations would accumulate sequentially, the simultaneous acquisition of both mutations during the same cell cycle being rare. This rarity might be linked to systemic genomic instability, aligning well with our experimental findings. In conclusion, our results suggest that transient hypermutators are major contributors to the mutational load and genomic instability accumulating in isogenic cell populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569436">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569436" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569436">
        <p class="paperTitle">Somatic Mosaicism in Amyotrophic Lateral Sclerosis and Frontotemporal Dementia Reveals Widespread Degeneration from Focal Mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569436" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569436" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, Z.; Kim, J.; Huang, A. Y.; Nolan, M.; Park, J.; Doan, R.; Shin, T.; Miller, M. B.; Chhouk, B.; Morillo, K.; Yeh, R. C.; Kenny, C.; Neil, J. E.; Lee, C.-Z.; Ohkubo, T.; Ravits, J.; Ansorge, O.; Ostrow, L. W.; Lagier-Tourenne, C.; Lee, E. A.; Walsh, C. A.</p>
        <p class="info">Score: 23.9, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569436' target='https://doi.org/10.1101/2023.11.30.569436'> 10.1101/2023.11.30.569436</a></p>
        <p class="abstract">Although mutations in dozens of genes have been implicated in familial forms of amyotrophic lateral sclerosis (fALS) and frontotemporal degeneration (fFTD), most cases of these conditions are sporadic (sALS and sFTD), with no family history, and their etiology remains obscure. We tested the hypothesis that somatic mosaic mutations, present in some but not all cells, might contribute in these cases, by performing ultra-deep, targeted sequencing of 88 genes associated with neurodegenerative diseases in postmortem brain and spinal cord samples from 404 individuals with sALS or sFTD and 144 controls. Known pathogenic germline mutations were found in 20.6% of ALS, and 26.5% of FTD cases. Predicted pathogenic somatic mutations in ALS/FTD genes were observed in 2.7% of sALS and sFTD cases that did not carry known pathogenic or novel germline mutations. Somatic mutations showed low variant allele fraction (typically &lt;2%) and were often restricted to the region of initial discovery, preventing detection through genetic screening in peripheral tissues. Damaging somatic mutations were preferentially enriched in primary motor cortex of sALS and prefrontal cortex of sFTD, mirroring regions most severely affected in each disease. Somatic mutation analysis of bulk RNA-seq data from brain and spinal cord from an additional 143 sALS cases and 23 controls confirmed an overall enrichment of somatic mutations in sALS. Two adult sALS cases were identified bearing pathogenic somatic mutations in DYNC1H1 and LMNA, two genes associated with pediatric motor neuron degeneration. Our study suggests that somatic mutations in fALS/fFTD genes, and in genes associated with more severe diseases in the germline state, contribute to sALS and sFTD, and that mosaic mutations in a small fraction of cells in focal regions of the nervous system can ultimately result in widespread degeneration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569263">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569263" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569263">
        <p class="paperTitle">Digital telomere measurement by long-read sequencing distinguishes healthy aging from disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569263" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569263" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez, S. E.; Gu, J.; Golla, A.; Martin, A.; Shomali, W.; Hockemeyer, D.; Savage, S.; Artandi, S.</p>
        <p class="info">Score: 17.9, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569263' target='https://doi.org/10.1101/2023.11.29.569263'> 10.1101/2023.11.29.569263</a></p>
        <p class="abstract">Telomere length is an important biomarker of organismal aging and cellular replicative potential, but existing measurement methods are limited in resolution and accuracy. Here, we deploy digital telomere measurement by nanopore sequencing to understand how distributions of human telomere length change with age and disease. We measure telomere attrition and de novo elongation with unprecedented resolution in genetically defined populations of human cells, in blood cells from healthy donors and in blood cells from patients with genetic defects in telomere maintenance. We find that human aging is accompanied by a progressive loss of long telomeres and an accumulation of shorter telomeres. In patients with defects in telomere maintenance, the accumulation of short telomeres is more pronounced and correlates with phenotypic severity. We apply machine learning to train a binary classification model that distinguishes healthy individuals from those with telomere biology disorders. This sequencing and bioinformatic pipeline will advance our understanding of telomere maintenance mechanisms and the use of telomere length as a clinical biomarker of aging and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570117">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570117" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570117">
        <p class="paperTitle">Improving the suppressive power of homing gene drive by co-targeting a distant-site female fertility gene</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570117" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570117" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faber, N. R.; Xu, X.; Chen, J.; Hou, S.; Du, J.; Pannebakker, B. A.; Zwaan, B. J.; van den Heuvel, J.; Champer, J.</p>
        <p class="info">Score: 6.7, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570117' target='https://doi.org/10.1101/2023.12.07.570117'> 10.1101/2023.12.07.570117</a></p>
        <p class="abstract">Gene drive technology has the potential to address major biological challenges, including the management of disease vectors, invasive species, and agricultural pests. After releasing individuals carrying the gene drive in the target population, suppression gene drives are designed to spread at a rapid rate and carry a recessive fitness cost, thus bringing about a decline in population size or even complete suppression. Well-studied homing suppression drives have been shown to be highly efficient in Anopheles mosquitoes and were successful in eliminating large cage populations. However, for other organisms, including Aedes mosquitoes, homing gene drives are so far too inefficient to achieve complete population suppression, mainly due to lower rates of drive conversion, which is the rate at which wild type alleles are converted into drive alleles. Low drive conversion is also a major issue in vertebrates, as indicated by experiments in mice. To tackle this issue, we propose a novel gene drive design that has two targets: a homing site where the drive is located and drive conversion takes place (with rescue for an essential gene), and a distant site for providing the fitness cost for population suppression (preferably a female fertility gene, for which no rescue is provided). We modeled this design and found that the two-target system allows suppression to occur over a much wider range of drive conversion efficiency. Specifically, in the new design, the suppressive power depends mostly on total gRNA cutting efficiency instead of just drive conversion efficiency, which is advantageous because cut rates are often substantially higher than drive conversion rates. We constructed a proof of concept in Drosophila melanogaster and show that both components of the gene drive function successfully. However, embryo drive activity from maternally deposited Cas9 as well as fitness costs for female drive heterozygotes both remain significant challenges for two-target and standard suppression drives. Overall, our improved gene drive design eases the development of strong homing suppression gene drives for many species where drive conversion is less efficient.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567732">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567732" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567732">
        <p class="paperTitle">A mouse model of ZTTK syndrome reveals indispensable SON functions in organ development and hematopoiesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567732" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567732" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vukadin, L.; Park, B.; Mohamed, M.; Li, H.; Elkholy, A.; Torrelli-Diljohn, A.; Kim, J.-H.; Jeong, K.; Murphy, J. M.; Harvey, C. A.; Dunlap, S.; Gehrs, L.; Lee, H.; Kim, H.-G.; Lee, S. N.; Stanford, D.; Barrington, R. A.; Foote, J.; Sorace, A. G.; Welner, R. S.; Hildreth, B. E.; Lim, S.-T. S.; Ahn, E.-Y. E.</p>
        <p class="info">Score: 11.5, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567732' target='https://doi.org/10.1101/2023.11.19.567732'> 10.1101/2023.11.19.567732</a></p>
        <p class="abstract">Rare diseases are underrepresented in biomedical research, leading to insufficient awareness. Zhu-Tokita-Takenouchi-Kim (ZTTK) syndrome is a rare disease caused by genetic alterations that result in heterozygous loss-of-function of SON. While ZTTK syndrome patients suffer from numerous symptoms, the lack of model organisms hamper our understanding of both SON and this complex syndrome. Here, we developed Son haploinsufficiency (Son&#43;/-) mice as a model of ZTTK syndrome and identified the indispensable roles of Son in organ development and hematopoiesis. Son&#43;/- mice recapitulated clinical symptoms of ZTTK syndrome, including growth retardation, cognitive impairment, skeletal abnormalities, and kidney agenesis. Furthermore, we identified hematopoietic abnormalities in Son&#43;/- mice, similar to those observed in human patients. Surface marker analyses and single-cell transcriptome profiling of hematopoietic stem and progenitor cells revealed that Son haploinsufficiency inclines cell fate toward the myeloid lineage but compromises lymphoid lineage development by reducing key genes required for lymphoid and B cell lineage specification. Additionally, Son haploinsufficiency causes inappropriate activation of erythroid genes and impaired erythroid maturation. These findings highlight the importance of the full gene dosage of Son in organ development and hematopoiesis. Our model serves as an invaluable research tool for this rare disease and related disorders associated with SON dysfunction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569403" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569403">
        <p class="paperTitle">Mechanism of homology search expansion during recombinational DNA break repair</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dumont, A.; Mendiboure, N.; Savocco, J.; Anani, L.; Moreau, P.; Thierry, A.; Modolo, L.; Jost, D.; Piazza, A.</p>
        <p class="info">Score: 25.5, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569403' target='https://doi.org/10.1101/2023.12.01.569403'> 10.1101/2023.12.01.569403</a></p>
        <p class="abstract">Homology search catalyzed by a RecA/Rad51 nucleoprotein filament (NPF) is a central step of DNA double-strand break (DSB) repair by homologous recombination. How it operates in cells remains elusive. Here we developed a Hi-C-based methodology to map single-stranded DNA (ssDNA) contacts genome-wide in S. cerevisiae, which revealed two main homology search phases. Initial search conducted by short NPFs is confined in cis by cohesin-mediated chromatin loop folding. Progressive growth of stiff NPFs enables exploration of distant genomic sites. Long-range resection by Exo1 drives this transition from local to genome-wide search by providing ssDNA substrates for assembly of extensive NPFs. DSB end-tethering promotes coordinated homology search by NPFs formed on the two DSB ends. Finally, an autonomous genetic element on chromosome III engages the NPF and stimulates homology search in its vicinity. This work reveals the mechanism of the progressive and uneven expansion of homology search orchestrated by chromatin organizers, long-range resection, end-tethering, specialized genetic elements, and that exploits the stiff NPF structure conferred by Rad51 oligomerization.

Highlights- Cohesin-mediated chromatin loops constrain homology search in cis for NPF regions close to the resection front
- Stiffening of ssDNA by Rad51 enables genome-wide homology search by DSB-proximal sites
- Exo1-mediated long-range resection promotes genome-wide homology search
- DSB end-tethering promotes coordinated homology search by NPFs formed on both DSB ends
- The recombination enhancer focuses homology search in its vicinity</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
