<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Extreme restructuring of cis-regulatory regions controlling a deeply conserved plant stem cell regulator
Authors: Ciren, D.; Zebell, S.; Lippman, Z.
Score: 23.0, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572550
A striking paradox is that genes with conserved protein sequence, function and expression pattern over deep time often exhibit extremely divergent cis-regulatory sequences. It remains unclear how such drastic cis-regulatory evolution across species allows preservation of gene function, and to what extent these differences influence how cis-regulatory variation arising within species impacts phenotypic change.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Extreme restructuring of cis-regulatory regions controlling a deeply conserved plant stem cell regulator
Authors: Ciren, D.; Zebell, S.; Lippman, Z.
Score: 23.0, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572550
A striking paradox is that genes with conserved protein sequence, function and expression pattern over deep time often exhibit extremely divergent cis-regulatory sequences. It remains unclear how such drastic cis-regulatory evolution across species allows preservation of gene function, and to what extent these differences influence how cis-regulatory variation arising within species impacts phenotypic change." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-31T10:38:19+00:00" />
<meta property="article:modified_time" content="2023-12-31T10:38:19+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Extreme restructuring of cis-regulatory regions controlling a deeply conserved plant stem cell regulator
Authors: Ciren, D.; Zebell, S.; Lippman, Z.
Score: 23.0, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572550
A striking paradox is that genes with conserved protein sequence, function and expression pattern over deep time often exhibit extremely divergent cis-regulatory sequences. It remains unclear how such drastic cis-regulatory evolution across species allows preservation of gene function, and to what extent these differences influence how cis-regulatory variation arising within species impacts phenotypic change."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Extreme restructuring of cis-regulatory regions controlling a deeply conserved plant stem cell regulator\nAuthors: Ciren, D.; Zebell, S.; Lippman, Z.\nScore: 23.0, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572550\nA striking paradox is that genes with conserved protein sequence, function and expression pattern over deep time often exhibit extremely divergent cis-regulatory sequences. It remains unclear how such drastic cis-regulatory evolution across species allows preservation of gene function, and to what extent these differences influence how cis-regulatory variation arising within species impacts phenotypic change.",
  "keywords": [
    
  ],
  "articleBody": " Extreme restructuring of cis-regulatory regions controlling a deeply conserved plant stem cell regulator\nAuthors: Ciren, D.; Zebell, S.; Lippman, Z.\nScore: 23.0, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572550\nA striking paradox is that genes with conserved protein sequence, function and expression pattern over deep time often exhibit extremely divergent cis-regulatory sequences. It remains unclear how such drastic cis-regulatory evolution across species allows preservation of gene function, and to what extent these differences influence how cis-regulatory variation arising within species impacts phenotypic change. Here, we investigated these questions using a plant stem cell regulator conserved in expression pattern and function over [~]125 million years. Using in-vivo genome editing in two distantly related models, Arabidopsis thaliana (Arabidopsis) and Solanum lycopersicum (tomato), we generated over 70 deletion alleles in the upstream and downstream regions of the stem cell repressor gene CLAVATA3 (CLV3) and compared their individual and combined effects on a shared phenotype, the number of carpels that make fruits. We found that sequences upstream of tomato CLV3 are highly sensitive to even small perturbations compared to its downstream region. In contrast, Arabidopsis CLV3 function is tolerant to severe disruptions both upstream and downstream of the coding sequence. Combining upstream and downstream deletions also revealed a different regulatory outcome. Whereas phenotypic enhancement from adding downstream mutations was predominantly weak and additive in tomato, mutating both regions of Arabidopsis CLV3 caused substantial and synergistic effects, demonstrating distinct distribution and redundancy of functional cis-regulatory sequences. Our results demonstrate remarkable malleability in cis-regulatory structural organization of a deeply conserved plant stem cell regulator and suggest that major reconfiguration of cis-regulatory sequence space is a common yet cryptic evolutionary force altering genotype-to-phenotype relationships from regulatory variation in conserved genes. Finally, our findings underscore the need for lineage-specific dissection of the spatial architecture of cis-regulation to effectively engineer trait variation from conserved productivity genes in crops. Author summaryWe investigated the evolution of cis-regulatory elements (CREs) and their interactions in the regulation of a plant stem cell regulator gene, CLAVATA3 (CLV3), in Arabidopsis and tomato. Despite diverging [~]125 million years ago, the function and expression of CLV3 is conserved in these species; however, cis-regulatory sequences upstream and downstream have drastically diverged, preventing identification of conserved non-coding sequences between them. We used CRISPR-Cas9 to engineer dozens of mutations within the cis-regulatory regions of Arabidopsis and tomato CLV3. In tomato, our results show that tomato CLV3 function primarily relies on interactions among CREs in the 5 non-coding region, unlike Arabidopsis CLV3, which depends on a more balanced distribution of functional CREs between the 5 and 3 regions. Therefore, despite a high degree of functional conservation, our study demonstrates divergent regulatory strategies between two distantly related CLV3 orthologs, with substantial alterations in regulatory sequences, their spatial arrangement, and their relative effects on CLV3 regulation. These results suggest that regulatory regions are not only extremely robust to mutagenesis, but also that the sequences underlying this robustness can be lineage-specific for conserved genes, due to the complex and often redundant interactions among CREs that ensure proper gene function amidst large-scale sequence turnover.\nKNL1 and NDC80 represent new universal markers for the detection of functional centromeres in plants\nAuthors: Oliveira, L.; Neumann, P.; Mata-Sucre, Y.; Kuo, Y.-T.; Marques, A.; Schubert, V.; Macas, J.\nScore: 14.4, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572763\nCentromere is the chromosomal site of kinetochore assembly and microtubule attachment for chromosome segregation. Given its importance, markers that allow specific labeling of centromeric chromatin throughout the cell cycle and across all chromosome types are sought for facilitating various centromere studies. Antibodies against the N-terminal region of CENH3 are commonly used for this purpose, since CENH3 is the near-universal marker of functional centromeres. However, because the N-terminal region of CENH3 is highly variable among plant species, antibodies directed against this region usually function only in a small group of closely related species. As a more versatile alternative, we present here antibodies targeted to the conserved domains of two outer kinetochore proteins, KNL1 and NDC80. Sequence comparison of these domains across more than 350 plant species revealed a high degree of conservation, particularly within a six amino acid motif, FFGPVS in KNL1, suggesting that both antibodies would function in a wide range of plant species. This assumption was confirmed by immunolabeling experiments in angiosperm (monocot and dicot) and gymnosperm species, including those with mono-, holo-, and meta-polycentric chromosomes. In addition to centromere labeling on condensed chromosomes during cell division, both antibodies detected the corresponding regions in the interphase nuclei of most species tested. These results demonstrated that KNL1 and NDC80 are better suited for immunolabeling centromeres than CENH3, because antibodies against these proteins offer incomparably greater versatility across different plant species which is particularly convenient for studying the organization and function of the centromere in non-model species.\nRewriting regulatory DNA to dissect and reprogram gene expression\nAuthors: Martyn, G. E.; Montgomery, M. T.; Jones, H.; Guo, K.; Doughty, B. R.; Linder, J.; Chen, Z.; Cochran, K.; Lawrence, K. A.; Munson, G.; Pampari, A.; Fulco, C. P.; Kelley, D. R.; Lander, E. S.; Kundaje, A.; Engreitz, J. M.\nScore: 52.8, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572268\nRegulatory DNA sequences within enhancers and promoters bind transcription factors to encode cell type-specific patterns of gene expression. However, the regulatory effects and programmability of such DNA sequences remain difficult to map or predict because we have lacked scalable methods to precisely edit regulatory DNA and quantify the effects in an endogenous genomic context. Here we present an approach to measure the quantitative effects of hundreds of designed DNA sequence variants on gene expression, by combining pooled CRISPR prime editing with RNA fluorescence in situ hybridization and cell sorting (Variant-FlowFISH). We apply this method to mutagenize and rewrite regulatory DNA sequences in an enhancer and the promoter of PPIF in two immune cell lines. Of 672 variant-cell type pairs, we identify 497 that affect PPIF expression. These variants appear to act through a variety of mechanisms including disruption or optimization of existing transcription factor binding sites, as well as creation of de novo sites. Disrupting a single endogenous transcription factor binding site often led to large changes in expression (up to -40% in the enhancer, and -50% in the promoter). The same variant often had different effects across cell types and states, demonstrating a highly tunable regulatory landscape. We use these data to benchmark performance of sequence-based predictive models of gene regulation, and find that certain types of variants are not accurately predicted by existing models. Finally, we computationally design 185 small sequence variants ([\u0026le;]10 bp) and optimize them for specific effects on expression in silico. 84% of these rationally designed edits showed the intended direction of effect, and some had dramatic effects on expression (-100% to +202%). Variant-FlowFISH thus provides a powerful tool to map the effects of variants and transcription factor binding sites on gene expression, test and improve computational models of gene regulation, and reprogram regulatory DNA.\nMapping cis- and trans-regulatory target genes of human-specific deletions\nAuthors: Fair, T.; Pavlovic, B. J.; Schaefer, N. K.; Pollen, A. A.\nScore: 9.0, Published: 2023-12-28 DOI: 10.1101/2023.12.27.573461\nDeletion of functional sequence is predicted to represent a fundamental mechanism of molecular evolution1,2. Comparative genetic studies of primates2,3 have identified thousands of human-specific deletions (hDels), and the cis-regulatory potential of short ([\u0026le;]31 base pairs) hDels has been assessed using reporter assays4. However, how structural variant-sized ([\u0026ge;]50 base pairs) hDels influence molecular and cellular processes in their native genomic contexts remains unexplored. Here, we design genome-scale libraries of single-guide RNAs targeting 7.2 megabases of sequence in 6,358 hDels and present a systematic CRISPR interference (CRISPRi) screening approach to identify hDels that modify cellular proliferation in chimpanzee pluripotent stem cells. By intersecting hDels with chromatin state features and performing single-cell CRISPRi (Perturb-seq) to identify their cis- and trans-regulatory target genes, we discovered 19 hDels controlling gene expression. We highlight two hDels, hDel_2247 and hDel_585, with tissue-specific activity in the liver and brain, respectively. Our findings reveal a molecular and cellular role for sequences lost in the human lineage and establish a framework for functionally interrogating human-specific genetic variants.\nIntegrative genomics reveals the polygenic basis of seedlessness in grapevine\nAuthors: Wang, X.; Liu, Z.; Zhang, F.; Xiao, H.; Cao, S.; Xue, H.; Liu, W.; Su, Y.; Liu, Z.; Zhong, H.; Zhang, F.; Ahmad, B.; Long, Q.; Zhang, Y.; Liu, Y.; Gan, Y.; Hou, T.; Jin, Z.; Wu, X.; Wang, Y.; Peng, Y.; Zhou, Y.\nScore: 7.3, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573032\nSeedlessness is a crucial quality trait in table grape (Vitis vinifera L.) breeding. However, the development of seeds involved intricate regulations, while the polygenic basis of seed abortion remains unclear. Here, we combine comparative genomics, population genetics, quantitative genetics, and integrative genomics to unravel the evolution and polygenic basis of seedlessness in grapes. We generated four haplotype-resolved telomere-to-telomere (T2T) genomes for two seedless grape cultivars, Thompson Seedless (TS, syn. Sultania) and Black Monukka (BM). Comparative genomics identified a [~]4.25 Mb hemizygous inversion on Chr10 specific in seedless cultivars, with seedless-associated genes VvTT16 and VvSUS2 located at breakpoints. Population genomic analyses of 548 grapevine accessions revealed two distinct clusters of seedless cultivars, tracing the origin of the seedlessness trait back to Sultania. Introgression, rather than convergent selection, shaped the evolutionary history of seedlessness in grape improvement. Genome-wide association study (GWAS) analysis identified 110 quantitative trait loci (QTLs) associated with 634 candidate genes, including novel candidate genes, such as three 11S GLOBULIN SEED STORAGE PROTEIN and two CYTOCHROME P450 genes, and well-known genes like VviAGL11. Integrative genomic analyses resulted in 339 core candidate genes categorized into 13 groups related to seed development. Machine learning based genomic selection achieved a remarkable 99% precision in predicting grapevine seedlessness. Our findings highlight the polygenic nature of seedless and provide novel candidate genes for molecular genetics and an effective prediction for seedlessness in grape genomic breeding.\nAVENGERS: Analysis of Variant Effects using Next Generation sequencing to Enhance BRCA2 Stratification\nAuthors: Sahu, S.; Galloux, M.; Southon, E.; Caylor, D.; Sullivan, T.; Arnaudi, M.; Geh, J.; Chari, R.; Papaleo, E.; Sharan, S. K.\nScore: 6.9, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571713\nAccurate interpretation of genetic variation is a critical step towards realizing the potential of precision medicine. Sequencing-based genetic tests have uncovered a vast array of BRCA2 sequence variants. Due to limited clinical, familial and/or epidemiological data, thousands of variants are considered to be variants of uncertain significance (VUS). To determine the functional impact of VUSs, here we develop AVENGERS: Analysis of Variant Effects using NGs to Enhance BRCA2 Stratification, utilizing CRISPR-Cas9-based saturation genome editing (SGE) in a humanized-mouse embryonic stem cell line. We have categorized nearly all possible missense single nucleotide variants (SNVs) encompassing the C-terminal DNA binding domain of BRCA2. We have generated the function scores for 6270 SNVs, covering 95.5% of possible SNVs in exons 15-26 spanning residues 2479-3216, including 1069 unique missense VUS, with 81% functional and 14% found to be nonfunctional. Our classification aligns strongly with pathogenicity data from ClinVar, orthogonal functional assays and computational meta predictors. Our statistical classifier exhibits 92.2% sensitivity and 96% specificity in distinguishing clinically benign and pathogenic variants recorded in ClinVar. Furthermore, we offer proactive evidence for 617 SNVs being non-functional and 3396 SNVs being functional demonstrated by impact on cell growth and response to DNA damaging drugs like cisplatin and olaparib. This classification serves as a valuable resource for interpreting unidentified variants in the population and for physicians and genetic counselors assessing BRCA2 VUSs in patients.\nBetween-group heritability and the status of hereditarianism as an evolutionary science\nAuthors: Roseman, C. C.; Bird, K.\nScore: 30.2, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572247\nHereditarianism is a school of thought that contends there are substantial evolved cognitive and behavioral differences among groups of humans which are both resistant to environmental intervention and are a root cause of differential social outcomes across groups. The relationship of betweengroup heritability [Formula] to within-group heritability [Formula] is one of the key theoretical components of hereditarianism and forms one of the bases for its claim to be an evolutionary science. Here, we examine the relationship between [Formula]and [Formula] and its application to problems in the hereditarian literature from an evolutionary genetic perspective. We demonstrate that the formulation of the relationship between [Formula] and [Formula]used in the hereditarian literature has no evolutionary content. By re-writing the relationship between [Formula] and[Formula] in a novel evolutionary framework, we demonstrate that there is no way to predict[Formula] using[Formula] without considerable additional theory that is absent from the hereditarian literature. Furthermore, we demonstrate that the hereditarian technique that uses[Formula] and [Formula] as a means of judging whether a given difference between groups may be plausibly ameliorated through environmental intervention is mathematically flawed. Lastly, we fill a gap in the hereditarian literature by writing out a means of using [Formula] to predict the absolute difference between groups under a neutral evolutionary model and find that it is much smaller than claimed by hereditarians. In conclusion, we propose a path forward for the study of human variation that moves us past the ill-conceived nature vs. nurture question and allows us to focus on more productive issues.\nCommon genetic variation impacts molecular stress response in the brain\nAuthors: Seah, C.; Singer, R.; Deans, P. M.; Bader, H.; Rusielewicz, T.; Hicks, E. M.; Young, H.; Cote, A.; Townsley, K.; Xu, C.; Hunter, C. J.; McCarthy, B.; Goldberg, J.; Dobariya, S.; Holtzherimer, P. E.; Young, K. A.; NYSCF Global Stem Cell Array Team, ; Traumatic Stress Brain Research Group, ; Noggle, S. A.; Krystal, J. H.; Paull, D.; Girgenti, M. J.; Yehuda, R.; Brennand, K.; Huckins, L.\nScore: 3.1, Published: 2023-12-28 DOI: 10.1101/2023.12.27.573459\nTo explain why individuals exposed to identical stressors experience divergent clinical outcomes, we determine how molecular encoding of stress modifies genetic risk for brain disorders. Analysis of post-mortem brain (n=304) revealed 8557 stress-interactive expression quantitative trait loci (eQTLs) that dysregulate expression of 915 eGenes in response to stress, and lie in stress-related transcription factor binding sites. Response to stress is robust across experimental paradigms: up to 50% of stress-interactive eGenes validate in glucocorticoid treated hiPSC-derived neurons (n=39 donors). Stress-interactive eGenes show brain region- and cell type-specificity, and, in post-mortem brain, implicate glial and endothelial mechanisms. Stress dysregulates long-term expression of disorder risk genes in a genotype-dependent manner; stress-interactive transcriptomic imputation uncovered 139 novel genes conferring brain disorder risk only in the context of traumatic stress. Molecular stress-encoding explains individualized responses to traumatic stress; incorporating trauma into genomic studies of brain disorders is likely to improve diagnosis, prognosis, and drug discovery. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=71 SRC=\"FIGDIR/small/573459v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (25K): org.highwire.dtl.DTLVardef@12f7dfdorg.highwire.dtl.DTLVardef@f8853eorg.highwire.dtl.DTLVardef@1f3d60dorg.highwire.dtl.DTLVardef@16a1b90_HPS_FORMAT_FIGEXP M_FIG C_FIG\nParallel genetic screens identify nuclear envelope homeostasis as a key determinant of telomere entanglement resolution in fission yeast\nAuthors: Nageshan, R. K.; Krogan, N. J.; Cooper, J. P.\nScore: 1.5, Published: 2023-12-26 DOI: 10.1101/2023.12.26.573359\nIn fission yeast lacking the telomere binding protein, Taz1, replication stalls at telomeres, triggering deleterious downstream events. Strand invasion from one taz1{Delta} telomeric stalled fork to another on a separate (non-sister) chromosome leads to telomere entanglements, which are resolved in mitosis at 32{degrees}C; however, entanglement resolution fails at [\u0026le;]20{degrees}C, leading to cold-specific cell lethality. Previously, we found that loss of the mitotic function of Rif1, a conserved DNA replication and repair factor, suppresses cold sensitivity by promoting resolution of entanglements without affecting entanglement formation. To understand the underlying pathways of mitotic entanglement resolution, we performed a series of genomewide synthetic genetic array screens to generate a comprehensive list of genetic interactors of taz1{Delta} and rif1{Delta}. We modified a previously described screening method to ensure that the queried cells were kept in log phase growth. In addition to recapitulating previously identified genetic interactions, we find that loss of genes encoding components of nuclear pore complexes (NPCs) promotes telomere disentanglement and suppresses taz1{Delta} cold sensitivity; we attribute this to more rapid anaphase midregion nuclear envelope (NE) breakdown in the absence of these NPC components. Moreover, loss of genes involved in lipid metabolism reverses the ability of rif1+ deletion to suppress taz1{Delta} cold sensitivity, again pinpointing NE modulation. A rif1+ separation-of-function mutant that specifically loses Rif1s mitotic functions yields similar genetic interactions. Genes promoting membrane fluidity were enriched in a parallel taz1+ synthetic lethal screen at permissive temperature, cementing the idea that the cold specificity of taz1{Delta} lethality stems from altered NE homeostasis.\nThe balance between ATR and DDK activities controls TopBP1-mediated locking of dormant origins at the pre-IC stage\nAuthors: Koundrioukoff, S.; Kim, S.-J.; Alary, N.; Toffano, A.; Melendez-Garcia, R.; Wu, X.; Liu, Y.; Gnan, S.; El-Hilali, S.; Brison, O.; Rosselli, F.; Chen, C.-L.; Debatisse, M.\nScore: 11.0, Published: 2023-12-03 DOI: 10.1101/2023.11.29.569233\nReplication stress, a major hallmark of cancers, and ensuing genome instability source from impaired progression of replication forks. The first line of defense against fork slowing is compensation, a long-described process that elicits firing of otherwise dormant origins. It remains unclear whether compensation requires activation of the DNA replication checkpoint or passively results from lengthening of the window of time during which dormant origins can fire when fork progression slows, or both. Using molecular DNA combing we show here that a linear relationship ties inter-origin distances to fork speeds, independently of the checkpoint status. We called this line \"stressline\" and further show that its slope enables precise quantification of the compensation efficiency. Comparison of the slopes in different genetic backgrounds reveals that compensation requires ATR, not CHK1, while TopBP1 and CDC7/DBF4 repress dormant origin activation. These results strongly suggest that TopBP1 locks dormant origins at the pre-IC stage and that ATR and DDK oppose to control the conversion of dormant pre-ICs into functional salvage origins. Both passive and active processes thus contribute to compensation. Moreover, Repli-seq and OK-seq analyses confirm the activating role of ATR and permit development of ATRAP-seq, a new procedure allowing mapping of early constitutive origins.\n",
  "wordCount" : "2952",
  "inLanguage": "en",
  "datePublished": "2023-12-31T10:38:19Z",
  "dateModified": "2023-12-31T10:38:19Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 31, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572550">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572550" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572550">
        <p class="paperTitle">Extreme restructuring of cis-regulatory regions controlling a deeply conserved plant stem cell regulator</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572550" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572550" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ciren, D.; Zebell, S.; Lippman, Z.</p>
        <p class="info">Score: 23.0, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572550' target='https://doi.org/10.1101/2023.12.20.572550'> 10.1101/2023.12.20.572550</a></p>
        <p class="abstract">A striking paradox is that genes with conserved protein sequence, function and expression pattern over deep time often exhibit extremely divergent cis-regulatory sequences. It remains unclear how such drastic cis-regulatory evolution across species allows preservation of gene function, and to what extent these differences influence how cis-regulatory variation arising within species impacts phenotypic change. Here, we investigated these questions using a plant stem cell regulator conserved in expression pattern and function over [~]125 million years. Using in-vivo genome editing in two distantly related models, Arabidopsis thaliana (Arabidopsis) and Solanum lycopersicum (tomato), we generated over 70 deletion alleles in the upstream and downstream regions of the stem cell repressor gene CLAVATA3 (CLV3) and compared their individual and combined effects on a shared phenotype, the number of carpels that make fruits. We found that sequences upstream of tomato CLV3 are highly sensitive to even small perturbations compared to its downstream region. In contrast, Arabidopsis CLV3 function is tolerant to severe disruptions both upstream and downstream of the coding sequence. Combining upstream and downstream deletions also revealed a different regulatory outcome. Whereas phenotypic enhancement from adding downstream mutations was predominantly weak and additive in tomato, mutating both regions of Arabidopsis CLV3 caused substantial and synergistic effects, demonstrating distinct distribution and redundancy of functional cis-regulatory sequences. Our results demonstrate remarkable malleability in cis-regulatory structural organization of a deeply conserved plant stem cell regulator and suggest that major reconfiguration of cis-regulatory sequence space is a common yet cryptic evolutionary force altering genotype-to-phenotype relationships from regulatory variation in conserved genes. Finally, our findings underscore the need for lineage-specific dissection of the spatial architecture of cis-regulation to effectively engineer trait variation from conserved productivity genes in crops.

Author summaryWe investigated the evolution of cis-regulatory elements (CREs) and their interactions in the regulation of a plant stem cell regulator gene, CLAVATA3 (CLV3), in Arabidopsis and tomato. Despite diverging [~]125 million years ago, the function and expression of CLV3 is conserved in these species; however, cis-regulatory sequences upstream and downstream have drastically diverged, preventing identification of conserved non-coding sequences between them. We used CRISPR-Cas9 to engineer dozens of mutations within the cis-regulatory regions of Arabidopsis and tomato CLV3. In tomato, our results show that tomato CLV3 function primarily relies on interactions among CREs in the 5 non-coding region, unlike Arabidopsis CLV3, which depends on a more balanced distribution of functional CREs between the 5 and 3 regions. Therefore, despite a high degree of functional conservation, our study demonstrates divergent regulatory strategies between two distantly related CLV3 orthologs, with substantial alterations in regulatory sequences, their spatial arrangement, and their relative effects on CLV3 regulation. These results suggest that regulatory regions are not only extremely robust to mutagenesis, but also that the sequences underlying this robustness can be lineage-specific for conserved genes, due to the complex and often redundant interactions among CREs that ensure proper gene function amidst large-scale sequence turnover.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572763" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572763">
        <p class="paperTitle">KNL1 and NDC80 represent new universal markers for the detection of functional centromeres in plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oliveira, L.; Neumann, P.; Mata-Sucre, Y.; Kuo, Y.-T.; Marques, A.; Schubert, V.; Macas, J.</p>
        <p class="info">Score: 14.4, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572763' target='https://doi.org/10.1101/2023.12.21.572763'> 10.1101/2023.12.21.572763</a></p>
        <p class="abstract">Centromere is the chromosomal site of kinetochore assembly and microtubule attachment for chromosome segregation. Given its importance, markers that allow specific labeling of centromeric chromatin throughout the cell cycle and across all chromosome types are sought for facilitating various centromere studies. Antibodies against the N-terminal region of CENH3 are commonly used for this purpose, since CENH3 is the near-universal marker of functional centromeres. However, because the N-terminal region of CENH3 is highly variable among plant species, antibodies directed against this region usually function only in a small group of closely related species. As a more versatile alternative, we present here antibodies targeted to the conserved domains of two outer kinetochore proteins, KNL1 and NDC80. Sequence comparison of these domains across more than 350 plant species revealed a high degree of conservation, particularly within a six amino acid motif, FFGPVS in KNL1, suggesting that both antibodies would function in a wide range of plant species. This assumption was confirmed by immunolabeling experiments in angiosperm (monocot and dicot) and gymnosperm species, including those with mono-, holo-, and meta-polycentric chromosomes. In addition to centromere labeling on condensed chromosomes during cell division, both antibodies detected the corresponding regions in the interphase nuclei of most species tested. These results demonstrated that KNL1 and NDC80 are better suited for immunolabeling centromeres than CENH3, because antibodies against these proteins offer incomparably greater versatility across different plant species which is particularly convenient for studying the organization and function of the centromere in non-model species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572268">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572268" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572268">
        <p class="paperTitle">Rewriting regulatory DNA to dissect and reprogram gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572268" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572268" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martyn, G. E.; Montgomery, M. T.; Jones, H.; Guo, K.; Doughty, B. R.; Linder, J.; Chen, Z.; Cochran, K.; Lawrence, K. A.; Munson, G.; Pampari, A.; Fulco, C. P.; Kelley, D. R.; Lander, E. S.; Kundaje, A.; Engreitz, J. M.</p>
        <p class="info">Score: 52.8, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572268' target='https://doi.org/10.1101/2023.12.20.572268'> 10.1101/2023.12.20.572268</a></p>
        <p class="abstract">Regulatory DNA sequences within enhancers and promoters bind transcription factors to encode cell type-specific patterns of gene expression. However, the regulatory effects and programmability of such DNA sequences remain difficult to map or predict because we have lacked scalable methods to precisely edit regulatory DNA and quantify the effects in an endogenous genomic context. Here we present an approach to measure the quantitative effects of hundreds of designed DNA sequence variants on gene expression, by combining pooled CRISPR prime editing with RNA fluorescence in situ hybridization and cell sorting (Variant-FlowFISH). We apply this method to mutagenize and rewrite regulatory DNA sequences in an enhancer and the promoter of PPIF in two immune cell lines. Of 672 variant-cell type pairs, we identify 497 that affect PPIF expression. These variants appear to act through a variety of mechanisms including disruption or optimization of existing transcription factor binding sites, as well as creation of de novo sites. Disrupting a single endogenous transcription factor binding site often led to large changes in expression (up to -40% in the enhancer, and -50% in the promoter). The same variant often had different effects across cell types and states, demonstrating a highly tunable regulatory landscape. We use these data to benchmark performance of sequence-based predictive models of gene regulation, and find that certain types of variants are not accurately predicted by existing models. Finally, we computationally design 185 small sequence variants ([&amp;le;]10 bp) and optimize them for specific effects on expression in silico. 84% of these rationally designed edits showed the intended direction of effect, and some had dramatic effects on expression (-100% to &#43;202%). Variant-FlowFISH thus provides a powerful tool to map the effects of variants and transcription factor binding sites on gene expression, test and improve computational models of gene regulation, and reprogram regulatory DNA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573461">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573461" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573461">
        <p class="paperTitle">Mapping cis- and trans-regulatory target genes of human-specific deletions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573461" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573461" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fair, T.; Pavlovic, B. J.; Schaefer, N. K.; Pollen, A. A.</p>
        <p class="info">Score: 9.0, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573461' target='https://doi.org/10.1101/2023.12.27.573461'> 10.1101/2023.12.27.573461</a></p>
        <p class="abstract">Deletion of functional sequence is predicted to represent a fundamental mechanism of molecular evolution1,2. Comparative genetic studies of primates2,3 have identified thousands of human-specific deletions (hDels), and the cis-regulatory potential of short ([&amp;le;]31 base pairs) hDels has been assessed using reporter assays4. However, how structural variant-sized ([&amp;ge;]50 base pairs) hDels influence molecular and cellular processes in their native genomic contexts remains unexplored. Here, we design genome-scale libraries of single-guide RNAs targeting 7.2 megabases of sequence in 6,358 hDels and present a systematic CRISPR interference (CRISPRi) screening approach to identify hDels that modify cellular proliferation in chimpanzee pluripotent stem cells. By intersecting hDels with chromatin state features and performing single-cell CRISPRi (Perturb-seq) to identify their cis- and trans-regulatory target genes, we discovered 19 hDels controlling gene expression. We highlight two hDels, hDel_2247 and hDel_585, with tissue-specific activity in the liver and brain, respectively. Our findings reveal a molecular and cellular role for sequences lost in the human lineage and establish a framework for functionally interrogating human-specific genetic variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573032">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573032" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573032">
        <p class="paperTitle">Integrative genomics reveals the polygenic basis of seedlessness in grapevine</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573032" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573032" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, X.; Liu, Z.; Zhang, F.; Xiao, H.; Cao, S.; Xue, H.; Liu, W.; Su, Y.; Liu, Z.; Zhong, H.; Zhang, F.; Ahmad, B.; Long, Q.; Zhang, Y.; Liu, Y.; Gan, Y.; Hou, T.; Jin, Z.; Wu, X.; Wang, Y.; Peng, Y.; Zhou, Y.</p>
        <p class="info">Score: 7.3, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573032' target='https://doi.org/10.1101/2023.12.22.573032'> 10.1101/2023.12.22.573032</a></p>
        <p class="abstract">Seedlessness is a crucial quality trait in table grape (Vitis vinifera L.) breeding. However, the development of seeds involved intricate regulations, while the polygenic basis of seed abortion remains unclear. Here, we combine comparative genomics, population genetics, quantitative genetics, and integrative genomics to unravel the evolution and polygenic basis of seedlessness in grapes. We generated four haplotype-resolved telomere-to-telomere (T2T) genomes for two seedless grape cultivars,  Thompson Seedless (TS, syn.  Sultania) and  Black Monukka (BM). Comparative genomics identified a [~]4.25 Mb hemizygous inversion on Chr10 specific in seedless cultivars, with seedless-associated genes VvTT16 and VvSUS2 located at breakpoints. Population genomic analyses of 548 grapevine accessions revealed two distinct clusters of seedless cultivars, tracing the origin of the seedlessness trait back to  Sultania. Introgression, rather than convergent selection, shaped the evolutionary history of seedlessness in grape improvement. Genome-wide association study (GWAS) analysis identified 110 quantitative trait loci (QTLs) associated with 634 candidate genes, including novel candidate genes, such as three 11S GLOBULIN SEED STORAGE PROTEIN and two CYTOCHROME P450 genes, and well-known genes like VviAGL11. Integrative genomic analyses resulted in 339 core candidate genes categorized into 13 groups related to seed development. Machine learning based genomic selection achieved a remarkable 99% precision in predicting grapevine seedlessness. Our findings highlight the polygenic nature of seedless and provide novel candidate genes for molecular genetics and an effective prediction for seedlessness in grape genomic breeding.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571713">
        <p class="paperTitle">AVENGERS: Analysis of Variant Effects using Next Generation sequencing to Enhance BRCA2 Stratification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sahu, S.; Galloux, M.; Southon, E.; Caylor, D.; Sullivan, T.; Arnaudi, M.; Geh, J.; Chari, R.; Papaleo, E.; Sharan, S. K.</p>
        <p class="info">Score: 6.9, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571713' target='https://doi.org/10.1101/2023.12.14.571713'> 10.1101/2023.12.14.571713</a></p>
        <p class="abstract">Accurate interpretation of genetic variation is a critical step towards realizing the potential of precision medicine. Sequencing-based genetic tests have uncovered a vast array of BRCA2 sequence variants. Due to limited clinical, familial and/or epidemiological data, thousands of variants are considered to be variants of uncertain significance (VUS). To determine the functional impact of VUSs, here we develop AVENGERS: Analysis of Variant Effects using NGs to Enhance BRCA2 Stratification, utilizing CRISPR-Cas9-based saturation genome editing (SGE) in a humanized-mouse embryonic stem cell line. We have categorized nearly all possible missense single nucleotide variants (SNVs) encompassing the C-terminal DNA binding domain of BRCA2. We have generated the function scores for 6270 SNVs, covering 95.5% of possible SNVs in exons 15-26 spanning residues 2479-3216, including 1069 unique missense VUS, with 81% functional and 14% found to be nonfunctional. Our classification aligns strongly with pathogenicity data from ClinVar, orthogonal functional assays and computational meta predictors. Our statistical classifier exhibits 92.2% sensitivity and 96% specificity in distinguishing clinically benign and pathogenic variants recorded in ClinVar. Furthermore, we offer proactive evidence for 617 SNVs being non-functional and 3396 SNVs being functional demonstrated by impact on cell growth and response to DNA damaging drugs like cisplatin and olaparib. This classification serves as a valuable resource for interpreting unidentified variants in the population and for physicians and genetic counselors assessing BRCA2 VUSs in patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572247">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572247" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572247">
        <p class="paperTitle">Between-group heritability and the status of hereditarianism as an evolutionary science</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572247" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572247" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roseman, C. C.; Bird, K.</p>
        <p class="info">Score: 30.2, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572247' target='https://doi.org/10.1101/2023.12.18.572247'> 10.1101/2023.12.18.572247</a></p>
        <p class="abstract">Hereditarianism is a school of thought that contends there are substantial evolved cognitive and behavioral differences among groups of humans which are both resistant to environmental intervention and are a root cause of differential social outcomes across groups. The relationship of betweengroup heritability [Formula] to within-group heritability [Formula] is one of the key theoretical components of hereditarianism and forms one of the bases for its claim to be an evolutionary science. Here, we examine the relationship between [Formula]and [Formula] and its application to problems in the hereditarian literature from an evolutionary genetic perspective. We demonstrate that the formulation of the relationship between [Formula] and [Formula]used in the hereditarian literature has no evolutionary content. By re-writing the relationship between [Formula] and[Formula] in a novel evolutionary framework, we demonstrate that there is no way to predict[Formula] using[Formula] without considerable additional theory that is absent from the hereditarian literature. Furthermore, we demonstrate that the hereditarian technique that uses[Formula] and [Formula] as a means of judging whether a given difference between groups may be plausibly ameliorated through environmental intervention is mathematically flawed. Lastly, we fill a gap in the hereditarian literature by writing out a means of using [Formula] to predict the absolute difference between groups under a neutral evolutionary model and find that it is much smaller than claimed by hereditarians. In conclusion, we propose a path forward for the study of human variation that moves us past the ill-conceived nature vs. nurture question and allows us to focus on more productive issues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573459">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573459" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573459">
        <p class="paperTitle">Common genetic variation impacts molecular stress response in the brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573459" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573459" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seah, C.; Singer, R.; Deans, P. M.; Bader, H.; Rusielewicz, T.; Hicks, E. M.; Young, H.; Cote, A.; Townsley, K.; Xu, C.; Hunter, C. J.; McCarthy, B.; Goldberg, J.; Dobariya, S.; Holtzherimer, P. E.; Young, K. A.; NYSCF Global Stem Cell Array Team,  ; Traumatic Stress Brain Research Group,  ; Noggle, S. A.; Krystal, J. H.; Paull, D.; Girgenti, M. J.; Yehuda, R.; Brennand, K.; Huckins, L.</p>
        <p class="info">Score: 3.1, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573459' target='https://doi.org/10.1101/2023.12.27.573459'> 10.1101/2023.12.27.573459</a></p>
        <p class="abstract">To explain why individuals exposed to identical stressors experience divergent clinical outcomes, we determine how molecular encoding of stress modifies genetic risk for brain disorders. Analysis of post-mortem brain (n=304) revealed 8557 stress-interactive expression quantitative trait loci (eQTLs) that dysregulate expression of 915 eGenes in response to stress, and lie in stress-related transcription factor binding sites. Response to stress is robust across experimental paradigms: up to 50% of stress-interactive eGenes validate in glucocorticoid treated hiPSC-derived neurons (n=39 donors). Stress-interactive eGenes show brain region- and cell type-specificity, and, in post-mortem brain, implicate glial and endothelial mechanisms. Stress dysregulates long-term expression of disorder risk genes in a genotype-dependent manner; stress-interactive transcriptomic imputation uncovered 139 novel genes conferring brain disorder risk only in the context of traumatic stress. Molecular stress-encoding explains individualized responses to traumatic stress; incorporating trauma into genomic studies of brain disorders is likely to improve diagnosis, prognosis, and drug discovery.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=71 SRC=&#34;FIGDIR/small/573459v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (25K):
org.highwire.dtl.DTLVardef@12f7dfdorg.highwire.dtl.DTLVardef@f8853eorg.highwire.dtl.DTLVardef@1f3d60dorg.highwire.dtl.DTLVardef@16a1b90_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573359">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573359" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573359">
        <p class="paperTitle">Parallel genetic screens identify nuclear envelope homeostasis as a key determinant of telomere entanglement resolution in fission yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573359" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573359" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nageshan, R. K.; Krogan, N. J.; Cooper, J. P.</p>
        <p class="info">Score: 1.5, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573359' target='https://doi.org/10.1101/2023.12.26.573359'> 10.1101/2023.12.26.573359</a></p>
        <p class="abstract">In fission yeast lacking the telomere binding protein, Taz1, replication stalls at telomeres, triggering deleterious downstream events. Strand invasion from one taz1{Delta} telomeric stalled fork to another on a separate (non-sister) chromosome leads to telomere entanglements, which are resolved in mitosis at 32{degrees}C; however, entanglement resolution fails at [&amp;le;]20{degrees}C, leading to cold-specific cell lethality. Previously, we found that loss of the mitotic function of Rif1, a conserved DNA replication and repair factor, suppresses cold sensitivity by promoting resolution of entanglements without affecting entanglement formation. To understand the underlying pathways of mitotic entanglement resolution, we performed a series of genomewide synthetic genetic array screens to generate a comprehensive list of genetic interactors of taz1{Delta} and rif1{Delta}. We modified a previously described screening method to ensure that the queried cells were kept in log phase growth. In addition to recapitulating previously identified genetic interactions, we find that loss of genes encoding components of nuclear pore complexes (NPCs) promotes telomere disentanglement and suppresses taz1{Delta} cold sensitivity; we attribute this to more rapid anaphase midregion nuclear envelope (NE) breakdown in the absence of these NPC components. Moreover, loss of genes involved in lipid metabolism reverses the ability of rif1&#43; deletion to suppress taz1{Delta} cold sensitivity, again pinpointing NE modulation. A rif1&#43; separation-of-function mutant that specifically loses Rif1s mitotic functions yields similar genetic interactions. Genes promoting membrane fluidity were enriched in a parallel taz1&#43; synthetic lethal screen at permissive temperature, cementing the idea that the cold specificity of taz1{Delta} lethality stems from altered NE homeostasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569233">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569233" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569233">
        <p class="paperTitle">The balance between ATR and DDK activities controls TopBP1-mediated locking of dormant origins at the pre-IC stage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569233" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569233" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koundrioukoff, S.; Kim, S.-J.; Alary, N.; Toffano, A.; Melendez-Garcia, R.; Wu, X.; Liu, Y.; Gnan, S.; El-Hilali, S.; Brison, O.; Rosselli, F.; Chen, C.-L.; Debatisse, M.</p>
        <p class="info">Score: 11.0, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569233' target='https://doi.org/10.1101/2023.11.29.569233'> 10.1101/2023.11.29.569233</a></p>
        <p class="abstract">Replication stress, a major hallmark of cancers, and ensuing genome instability source from impaired progression of replication forks. The first line of defense against fork slowing is compensation, a long-described process that elicits firing of otherwise dormant origins. It remains unclear whether compensation requires activation of the DNA replication checkpoint or passively results from lengthening of the window of time during which dormant origins can fire when fork progression slows, or both. Using molecular DNA combing we show here that a linear relationship ties inter-origin distances to fork speeds, independently of the checkpoint status. We called this line &#34;stressline&#34; and further show that its slope enables precise quantification of the compensation efficiency. Comparison of the slopes in different genetic backgrounds reveals that compensation requires ATR, not CHK1, while TopBP1 and CDC7/DBF4 repress dormant origin activation. These results strongly suggest that TopBP1 locks dormant origins at the pre-IC stage and that ATR and DDK oppose to control the conversion of dormant pre-ICs into functional salvage origins. Both passive and active processes thus contribute to compensation. Moreover, Repli-seq and OK-seq analyses confirm the activating role of ATR and permit development of ATRAP-seq, a new procedure allowing mapping of early constitutive origins.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
