<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genetic similarity between relatives provides evidence on the presence and history of assortative mating
Authors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.
Score: 31.8, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546663
Assortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Genetic similarity between relatives provides evidence on the presence and history of assortative mating
Authors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.
Score: 31.8, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546663
Assortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Genetic similarity between relatives provides evidence on the presence and history of assortative mating
Authors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.
Score: 31.8, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546663
Assortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Genetic similarity between relatives provides evidence on the presence and history of assortative mating\nAuthors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.\nScore: 31.8, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546663\nAssortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives.",
  "keywords": [
    
  ],
  "articleBody": " Genetic similarity between relatives provides evidence on the presence and history of assortative mating\nAuthors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.\nScore: 31.8, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546663\nAssortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives. However, empirical evidence is limited for many traits, and the implications hinge on whether assortative mating has started recently or many generations ago. Here we show theoretically and empirically that genetic similarity between relatives can provide evidence on the presence and history of assortative mating. First, we employed path analysis to understand how assortative mating affects genetic similarity between family members across generations, finding that similarity between distant relatives is more affected than close relatives. Next, we correlated polygenic indices of 47,135 co-parents from the Norwegian Mother, Father, and Child Cohort Study (MoBa) and found genetic evidence of assortative mating in eight out of fifteen examined traits. The same traits show elevated similarity between relatives, especially distant relatives. Five of the eight traits, including educational attainment, showed greater genetic variance among offspring, which is inconsistent with stable assortative mating over many generations. These results suggest an ongoing increase in familial similarity for these traits. The implications of this research extend to genetic methodology and the understanding of social and economic disparities.\nAncient DNA reveals the origins of the Albanians\nAuthors: Davranoglou, L. R.; Aristodemou, A.; Wesolowski, D.; Heraclides, A.\nScore: 193.1, Published: 2023-06-07 DOI: 10.1101/2023.06.05.543790\nThe origins of the Albanian people have vexed linguists and historians for centuries, as Albanians first appear in the historical record in the 11th century CE, while their language is one of the most enigmatic branches of the Indo-European family. To identify the populations that contributed to the ancestry of Albanians, we undertake a genomic transect of the Balkans over the last 8000 years, where we analyse more than 6000 previously published ancient genomes using state-of-the-art bioinformatics tools and algorithms that quantify spatiotemporal human mobility. We find that modern Albanians descend from Roman era western Balkan populations, with additional admixture from Slavic-related groups. Remarkably, Albanian paternal ancestry shows continuity from Bronze Age Balkan populations, including those known as Illyrians. Our results provide an unprecedented understanding of the historical and demographic processes that led to the formation of modern Albanians and help locate the area where the Albanian language developed.\nInteraction molecular QTL mapping discovers cellular and environmental modifiers of genetic regulatory effects\nAuthors: Kasela, S.; Aguet, F.; Kim-Hellmuth, S.; Brown, B. C.; Nachun, D. C.; Tracy, R. P.; Durda, P.; Liu, Y.; Taylor, K. D.; Johnson, W. C.; Van Den Berg, D.; Gabriel, S.; Gupta, N.; Smith, J. D.; Blackwell, T. W.; Rotter, J. I.; Ardlie, K. G.; Manichaikul, A.; Rich, S. S.; Barr, R. G.; Lappalainen, T.\nScore: 18.9, Published: 2023-06-29 DOI: 10.1101/2023.06.26.546528\nBulk tissue molecular quantitative trait loci (QTLs) have been the starting point for interpreting disease-associated variants, while context-specific QTLs show particular relevance for disease. Here, we present the results of mapping interaction QTLs (iQTLs) for cell type, age, and other phenotypic variables in multi-omic, longitudinal data from blood of individuals of diverse ancestries. By modeling the interaction between genotype and estimated cell type proportions, we demonstrate that cell type iQTLs could be considered as proxies for cell type-specific QTL effects. The interpretation of age iQTLs, however, warrants caution as the moderation effect of age on the genotype and molecular phenotype association may be mediated by changes in cell type composition. Finally, we show that cell type iQTLs contribute to cell type-specific enrichment of diseases that, in combination with additional functional data, may guide future functional studies. Overall, this study highlights iQTLs to gain insights into the context-specificity of regulatory effects.\nThe human Y and inactive X chromosomes similarly modulate autosomal gene expression\nAuthors: San Roman, A. K.; Skaletsky, H.; Godfrey, A. K.; Bokil, N. V.; Teitz, L.; Singh, I.; Blanton, L. V.; Bellott, D. W.; Pyntikova, T.; Lange, J.; Koutseva, N.; Hughes, J. F.; Brown, L.; Phou, S.; Buscetta, A.; Kruszka, P.; Banks, N.; Dutra, A.; Pak, E.; Lasutschinkow, P. C.; Keen, C.; Davis, S. M.; Lin, A. E.; Tartaglia, N. R.; Samango-Sprouse, C.; Muenke, M.; Page, D. C.\nScore: 20.2, Published: 2023-06-07 DOI: 10.1101/2023.06.05.543763\nSomatic cells of human males and females have 45 chromosomes in common, including the \"active\" X chromosome. In males the 46th chromosome is a Y; in females it is an \"inactive\" X (Xi). Through linear modeling of autosomal gene expression in cells from individuals with zero to three Xi and zero to four Y chromosomes, we found that Xi and Y impact autosomal expression broadly and with remarkably similar effects. Studying sex-chromosome structural anomalies, promoters of Xi- and Y-responsive genes, and CRISPR inhibition, we traced part of this shared effect to homologous transcription factors - ZFX and ZFY - encoded by Chr X and Y. This demonstrates sex-shared mechanisms by which Xi and Y modulate autosomal expression. Combined with earlier analyses of sex-linked gene expression, our studies show that 21% of all genes expressed in lymphoblastoid cells or fibroblasts change expression significantly in response to Xi or Y chromosomes.\nFetally-encoded GDF15 and maternal GDF15 sensitivity are major determinants of nausea and vomiting in human pregnancy\nAuthors: Fejzo, M.; Rocha, N.; Cimino, I.; Lockhart, S. M.; Petry, C.; Kay, R. G.; Burling, K.; Barker, P.; George, A. L.; Yasara, N.; Premawardhena, A.; Gong, S.; Cook, E.; Rainbow, K.; Withers, D.; Cortessis, V.; Mullin, P.; MacGibbon, K.; Jin, E.; Kam, A.; Campbell, A.; Polasek, O.; Tzoneva, G.; Gribble, F.; Yeo, G.; Lam, B.; Saudek, V.; Hughes, I.; Ong, K.; Perry, J.; Sutton-Cole, A.; Baumgarten, M.; Welsh, P.; Sattar, N.; Smith, G. C.; Charnock-Jones, D. S.; Coll, A. P.; Meek, C. L.; Mettananda, S.; Hayward, C.; Mancuso, N.; O'Rahilly, S.\nScore: 209.8, Published: 2023-06-04 DOI: 10.1101/2023.06.02.542661\nHuman pregnancy is frequently accompanied by nausea and vomiting that may become severe and life-threatening, as in hyperemesis gravidarum (HG), the cause of which is unknown. Growth Differentiation Factor-15 (GDF15), a hormone known to act on the hindbrain to cause emesis, is highly expressed in the placenta and its levels in maternal blood rise rapidly in pregnancy. Variants in the maternal GDF15 gene are associated with HG. Here we report that fetal production of GDF15, and maternal sensitivity to it, both contribute substantially to the risk of HG. We found that the great majority of GDF15 in maternal circulation is derived from the feto-placental unit and that higher GDF15 levels in maternal blood are associated with vomiting and are further elevated in patients with HG. Conversely, we found that lower levels of GDF15 in the non-pregnant state predispose women to HG. A rare C211G variant in GDF15 which strongly predisposes mothers to HG, particularly when the fetus is wild-type, was found to markedly impair cellular secretion of GDF15 and associate with low circulating levels of GDF15 in the non-pregnant state. Consistent with this, two common GDF15 haplotypes which predispose to HG were associated with lower circulating levels outside pregnancy. The administration of a long-acting form of GDF15 to wild-type mice markedly reduced subsequent responses to an acute dose, establishing that desensitisation is a feature of this system. GDF15 levels are known to be highly and chronically elevated in patients with beta thalassemia. In women with this disorder, reports of symptoms of nausea or vomiting in pregnancy were strikingly diminished. Our findings support a causal role for fetal derived GDF15 in the nausea and vomiting of human pregnancy, with maternal sensitivity, at least partly determined by pre-pregnancy exposure to GDF15, being a major influence on its severity. They also suggest mechanism-based approaches to the treatment and prevention of HG.\nIs a Picture Worth 1,000 SNPs? Effects of User-Submitted Photographs on Ancestry Estimates from Direct-to-Consumer Canine Genetic Tests\nAuthors: Rando, H.; Graim, K.; Hampikian, G.; Greene, C. S.\nScore: 11.6, Published: 2023-06-29 DOI: 10.1101/2023.06.28.546898\nObjective: To evaluate whether the breed ancestry predictions of direct-to-consumer (DTC) genetic tests for dogs are influenced by the user-provided photograph. Animals: Twelve pet dogs considered purebred (i.e., registered with a breed organization) representing twelve different breeds. Methods: Six buccal swabs per dog were collected by the owners and submitted to six DTC genetic testing companies. The experimenters registered each sample with the company. For half of the dogs, the registration included a photograph of the DNA donor. For the other half of the dogs, photographs were swapped between dogs. Analysis of the DNA and breed ancestry prediction was conducted by each company. Each company's breed predictions were evaluated to assess whether the condition (i.e., matching versus shuffled photograph) affected the odds of identifying the DNA donor's registered breed. A convolutional neural network was also used to predict breed based solely on the photograph as a positive control. Results: Five of the six tests always produced results that included the registered breed. One test and the convolutional neural network were unlikely to identify the registered breed and frequently returned results that included the breed in the photograph. This result suggests that one test on the market is relying on the photograph more than the DNA sample. Additionally, differences in the predictions made across all tests underscore the challenge of identifying breed ancestry, even in purebred dogs. Clinical Relevance: Veterinarians are likely to encounter patients who have conducted DTC genetic testing and may find themselves in the position of explaining genetic test results that they did not order. This systematic comparison of tests on the market provides context for interpreting unexpected results from consumer-grade DTC genetic testing kits.\nA whole organism map of X-inactivation reveals tissue-specific escape as an exceptionally rare event in humans\nAuthors: Gylemo, B.; Nestor, C. E.\nScore: 11.6, Published: 2023-06-27 DOI: 10.1101/2023.06.26.546519\nAs females are mosaic for X-inactivation, direct determination of X-linked allelic expression in bulk tissues is typically unfeasible. Using females that are non-mosaic for X-inactivation (nmXCI) as a powerful and natural genetic system we directly determined X-inactivation status of 401 X-linked genes across 32 normal tissues. We reveal that nmXCI is common in the female population and that contrary to previous reports, variable escape is an exceedingly rare event in humans. As the most complete atlas of XCI in humans to date, our findings represent a major advance in our understanding of human X-inactivation and will serve as a reference for dissecting the genetic origin of sex-bias in human traits.\nGenetic population structure across Brittany and the downstream Loire basin provides new insights on the demographic history of Western Europe\nAuthors: Alves, I.; Giemza, J.; Blum, M.; Bernhardsson, C.; Chatel, S.; Karakachoff, M.; Saint Pierre, A.; Herzig, A. F.; Olaso, R.; Monteil, M.; Gallien, V.; Cabot, E.; Svensson, E.; Bacq Daian, D.; Baron, E.; Berthellier, C.; Besse, C.; Blanche, H.; Bocher, O.; Boland, A.; Bonnaud, S.; Charpentier, E.; Dandine-Roulland, C.; Ferec, C.; Fruchet, C.; Lecointe, S.; Le Floch, E.; Ludwig, T. E.; Marenne, G.; Meyer, V.; Quellery, E.; Racimo, F.; Rouault, K.; Sandron, F.; Schott, J.-J.; Velo Suarez, L.; Violleau, J.; Willerslev, E.; Coativy, Y.; Jezequel, M.; Le Bris, D.; Nicolas, C.; Pailler, Y.; Goldberg, Score: 40.9, Published: 2023-06-25 DOI: 10.1101/2022.02.03.478491\nEuropean genetic ancestry originates from three main ancestral populations - Western hunter-gatherers, early European farmers and Yamnaya Eurasian herders - whose edges geographically met in present-day France. Despite its central role to our understanding of how the ancestral populations interacted and gave rise to modern population structure, the population history of France has remained largely understudied. Here, we analysed the high-coverage whole-genome sequences and genome-wide genotype profiles of respectively 856 and 3,234 present-day individuals from the northern half of France, and merged them with publicly available present-day and ancient Europe-wide genotype datasets. We also explored, for the first time, the whole-genome sequences of six mediaeval individuals (300-1100 CE) from Western France to gain insights into the genetic impact of what is commonly known as the Migration Period in Europe. We found extensive fine-scale population structure across Brittany and the downstream Loire basin, emphasising the need for investigating local populations to better understand the distribution of rare and putatively deleterious variants across space. Overall, we observed an increased population differentiation between the northern and southern sides of the river Loire, which are characterised by different proportions of steppe vs. Neolithic-related ancestry. Samples from Western Brittany carry the largest levels of steppe ancestry and show high levels of allele sharing with individuals associated with the Bell Beaker complex, levels that are only comparable with those found in populations lying on the northwestern edges of Europe. Together, our results imply that present-day individuals from Western Brittany retain substantial legacy of the genetic changes that occurred in Northwestern Europe following the arrival of the Bell Beaker people c. 2500 BCE. Such genetic legacy may explain the sharing of disease-related alleles with other present-day populations from Western Britain and Ireland.\nSex peptide receptor is not required for refractoriness to remating or induction of egg laying in Aedes aegypti\nAuthors: Amaro, I. A.; Wohl, M. P.; Pitcher, S.; Alfonso-Parra, C.; Avila, F. W.; Paige, A. S.; Helinski, M.; Duvall, L. B.; Harrington, L. C.; Wolfner, M. F.; McMeniman, C. J.\nScore: 10.8, Published: 2023-06-30 DOI: 10.1101/2023.06.28.546954\nAcross diverse insect taxa, the behavior and physiology of females dramatically changes after mating processes largely triggered by the transfer of seminal proteins from their mates. In the vinegar fly Drosophila melanogaster, the seminal protein sex peptide (SP) decreases the likelihood of female flies remating and causes additional behavioral and physiological changes that promote fertility including increasing egg production. Although SP is only found in the Drosophila genus, its receptor, sex peptide receptor (SPR), is the widely-conserved myoinhibitory peptide (MIP) receptor. To test the functional role of SPR in mediating post-mating responses in a non-Drosophila dipteran, we generated two independent Spr-knockout alleles in the yellow fever mosquito Aedes aegypti. Although SPR is needed for post-mating responses in Drosophila and the cotton bollworm Helicoverpa armigera, Spr mutant Ae. aegypti show completely normal post-mating decreases in remating propensity and increases in egg laying. In addition, injection of synthetic SP or accessory gland homogenate from D. melanogaster into virgin female mosquitoes did not elicit these post-mating responses. Our results indicate that Spr is not required for these canonical post-mating responses in Ae. aegypti, indicating that unknown signaling pathways are likely responsible for these behavioral switches in this disease vector.\nThe ChickenGTEx pilot analysis: a reference of regulatory variants across 28 chicken tissues\nAuthors: Guan, D.; Bai, Z.; Zhu, X.; Zhong, C.; Hou, Y.; Lan, F.; Diao, S.; Yao, Y.; Zhao, B.; Zhu, D.; Li, X.; Pan, Z.; Gao, Y.; Wang, Y.; Zou, D.; Wang, R.; Xu, T.; Sun, C.; Yin, H.; Teng, J.; Xu, Z.; Lin, Q.; Shi, S.; Shao, D.; Degalez, F.; Lagarrigue, S.; Wang, Y.; Wang, M.; Peng, M.; Rocha, D.; Charles, M.; Smith, J.; Watson, K.; Buitenhuis, A. J.; Sahana, G.; Lund, M. S.; Warren, W.; Frantz, L.; Larson, G.; Lamont, S. J.; Si, W.; Zhao, X.; Li, B.; Zhang, H.; Luo, C.; Shu, D.; Qu, H.; Luo, W.; Li, Z.; Nie, Q.; Zhang, X.; Zhang, Z.; Zhang, Z.; Liu, G. E.; Cheng, H.; Yang, N.; Hu, X.; Zhou, H.; Fang\nScore: 10.3, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546670\nChicken is a valuable model for understanding fundamental biology, vertebrate evolution and diseases, as well as a major source of nutrient-dense and lean-protein-enriched food globally. Although it is the first non-mammalian amniote genome to be sequenced, the chicken genome still lacks a systematic characterization of functional impacts of genetic variants. Here, through integrating 7,015 RNA-Seq and 2,869 whole-genome sequence data, the Chicken Genotype-Tissue Expression (ChickenGTEx) project presents the pilot reference of regulatory variants in 28 chicken tissue transcriptomes, including millions of regulatory effects on primary expression (including protein-coding genes, lncRNA and exon) and post-transcriptional modifications (alternative splicing and 3 untranslated region alternative polyadenylation). We explored the tissue-sharing and context-specificity of these regulatory variants, their underlying molecular mechanisms of action, and their utility in interpreting adaptation and genome-wide associations of 108 chicken complex traits. Finally, we illustrated shared and lineage-specific features of gene regulation between chickens and mammals, and demonstrated how the ChickenGTEx resource can further assist with translating genetic findings across species.\n",
  "wordCount" : "2704",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546663">
        <p class="paperTitle">Genetic similarity between relatives provides evidence on the presence and history of assortative mating</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.</p>
        <p class="info">Score: 31.8, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546663' target='https://doi.org/10.1101/2023.06.27.546663'> 10.1101/2023.06.27.546663</a></p>
        <p class="abstract">Assortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives. However, empirical evidence is limited for many traits, and the implications hinge on whether assortative mating has started recently or many generations ago. Here we show theoretically and empirically that genetic similarity between relatives can provide evidence on the presence and history of assortative mating. First, we employed path analysis to understand how assortative mating affects genetic similarity between family members across generations, finding that similarity between distant relatives is more affected than close relatives. Next, we correlated polygenic indices of 47,135 co-parents from the Norwegian Mother, Father, and Child Cohort Study (MoBa) and found genetic evidence of assortative mating in eight out of fifteen examined traits. The same traits show elevated similarity between relatives, especially distant relatives. Five of the eight traits, including educational attainment, showed greater genetic variance among offspring, which is inconsistent with stable assortative mating over many generations. These results suggest an ongoing increase in familial similarity for these traits. The implications of this research extend to genetic methodology and the understanding of social and economic disparities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.05.543790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.05.543790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.05.543790">
        <p class="paperTitle">Ancient DNA reveals the origins of the Albanians</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.05.543790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.05.543790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davranoglou, L. R.; Aristodemou, A.; Wesolowski, D.; Heraclides, A.</p>
        <p class="info">Score: 193.1, Published: 2023-06-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.05.543790' target='https://doi.org/10.1101/2023.06.05.543790'> 10.1101/2023.06.05.543790</a></p>
        <p class="abstract">The origins of the Albanian people have vexed linguists and historians for centuries, as Albanians first appear in the historical record in the 11th century CE, while their language is one of the most enigmatic branches of the Indo-European family. To identify the populations that contributed to the ancestry of Albanians, we undertake a genomic transect of the Balkans over the last 8000 years, where we analyse more than 6000 previously published ancient genomes using state-of-the-art bioinformatics tools and algorithms that quantify spatiotemporal human mobility. We find that modern Albanians descend from Roman era western Balkan populations, with additional admixture from Slavic-related groups. Remarkably, Albanian paternal ancestry shows continuity from Bronze Age Balkan populations, including those known as Illyrians. Our results provide an unprecedented understanding of the historical and demographic processes that led to the formation of modern Albanians and help locate the area where the Albanian language developed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546528">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546528" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546528">
        <p class="paperTitle">Interaction molecular QTL mapping discovers cellular and environmental modifiers of genetic regulatory effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546528" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546528" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kasela, S.; Aguet, F.; Kim-Hellmuth, S.; Brown, B. C.; Nachun, D. C.; Tracy, R. P.; Durda, P.; Liu, Y.; Taylor, K. D.; Johnson, W. C.; Van Den Berg, D.; Gabriel, S.; Gupta, N.; Smith, J. D.; Blackwell, T. W.; Rotter, J. I.; Ardlie, K. G.; Manichaikul, A.; Rich, S. S.; Barr, R. G.; Lappalainen, T.</p>
        <p class="info">Score: 18.9, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546528' target='https://doi.org/10.1101/2023.06.26.546528'> 10.1101/2023.06.26.546528</a></p>
        <p class="abstract">Bulk tissue molecular quantitative trait loci (QTLs) have been the starting point for interpreting disease-associated variants, while context-specific QTLs show particular relevance for disease. Here, we present the results of mapping interaction QTLs (iQTLs) for cell type, age, and other phenotypic variables in multi-omic, longitudinal data from blood of individuals of diverse ancestries. By modeling the interaction between genotype and estimated cell type proportions, we demonstrate that cell type iQTLs could be considered as proxies for cell type-specific QTL effects. The interpretation of age iQTLs, however, warrants caution as the moderation effect of age on the genotype and molecular phenotype association may be mediated by changes in cell type composition. Finally, we show that cell type iQTLs contribute to cell type-specific enrichment of diseases that, in combination with additional functional data, may guide future functional studies. Overall, this study highlights iQTLs to gain insights into the context-specificity of regulatory effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.05.543763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.05.543763" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.05.543763">
        <p class="paperTitle">The human Y and inactive X chromosomes similarly modulate autosomal gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.05.543763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.05.543763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: San Roman, A. K.; Skaletsky, H.; Godfrey, A. K.; Bokil, N. V.; Teitz, L.; Singh, I.; Blanton, L. V.; Bellott, D. W.; Pyntikova, T.; Lange, J.; Koutseva, N.; Hughes, J. F.; Brown, L.; Phou, S.; Buscetta, A.; Kruszka, P.; Banks, N.; Dutra, A.; Pak, E.; Lasutschinkow, P. C.; Keen, C.; Davis, S. M.; Lin, A. E.; Tartaglia, N. R.; Samango-Sprouse, C.; Muenke, M.; Page, D. C.</p>
        <p class="info">Score: 20.2, Published: 2023-06-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.05.543763' target='https://doi.org/10.1101/2023.06.05.543763'> 10.1101/2023.06.05.543763</a></p>
        <p class="abstract">Somatic cells of human males and females have 45 chromosomes in common, including the &#34;active&#34; X chromosome. In males the 46th chromosome is a Y; in females it is an &#34;inactive&#34; X (Xi). Through linear modeling of autosomal gene expression in cells from individuals with zero to three Xi and zero to four Y chromosomes, we found that Xi and Y impact autosomal expression broadly and with remarkably similar effects. Studying sex-chromosome structural anomalies, promoters of Xi- and Y-responsive genes, and CRISPR inhibition, we traced part of this shared effect to homologous transcription factors - ZFX and ZFY - encoded by Chr X and Y. This demonstrates sex-shared mechanisms by which Xi and Y modulate autosomal expression. Combined with earlier analyses of sex-linked gene expression, our studies show that 21% of all genes expressed in lymphoblastoid cells or fibroblasts change expression significantly in response to Xi or Y chromosomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.02.542661">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.02.542661" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.02.542661">
        <p class="paperTitle">Fetally-encoded GDF15 and maternal GDF15 sensitivity are major determinants of nausea and vomiting in human pregnancy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.02.542661" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.02.542661" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fejzo, M.; Rocha, N.; Cimino, I.; Lockhart, S. M.; Petry, C.; Kay, R. G.; Burling, K.; Barker, P.; George, A. L.; Yasara, N.; Premawardhena, A.; Gong, S.; Cook, E.; Rainbow, K.; Withers, D.; Cortessis, V.; Mullin, P.; MacGibbon, K.; Jin, E.; Kam, A.; Campbell, A.; Polasek, O.; Tzoneva, G.; Gribble, F.; Yeo, G.; Lam, B.; Saudek, V.; Hughes, I.; Ong, K.; Perry, J.; Sutton-Cole, A.; Baumgarten, M.; Welsh, P.; Sattar, N.; Smith, G. C.; Charnock-Jones, D. S.; Coll, A. P.; Meek, C. L.; Mettananda, S.; Hayward, C.; Mancuso, N.; O&#39;Rahilly, S.</p>
        <p class="info">Score: 209.8, Published: 2023-06-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.02.542661' target='https://doi.org/10.1101/2023.06.02.542661'> 10.1101/2023.06.02.542661</a></p>
        <p class="abstract">Human pregnancy is frequently accompanied by nausea and vomiting that may become severe and life-threatening, as in hyperemesis gravidarum (HG), the cause of which is unknown. Growth Differentiation Factor-15 (GDF15), a hormone known to act on the hindbrain to cause emesis, is highly expressed in the placenta and its levels in maternal blood rise rapidly in pregnancy. Variants in the maternal GDF15 gene are associated with HG. Here we report that fetal production of GDF15, and maternal sensitivity to it, both contribute substantially to the risk of HG. We found that the great majority of GDF15 in maternal circulation is derived from the feto-placental unit and that higher GDF15 levels in maternal blood are associated with vomiting and are further elevated in patients with HG. Conversely, we found that lower levels of GDF15 in the non-pregnant state predispose women to HG. A rare C211G variant in GDF15 which strongly predisposes mothers to HG, particularly when the fetus is wild-type, was found to markedly impair cellular secretion of GDF15 and associate with low circulating levels of GDF15 in the non-pregnant state. Consistent with this, two common GDF15 haplotypes which predispose to HG were associated with lower circulating levels outside pregnancy. The administration of a long-acting form of GDF15 to wild-type mice markedly reduced subsequent responses to an acute dose, establishing that desensitisation is a feature of this system. GDF15 levels are known to be highly and chronically elevated in patients with beta thalassemia. In women with this disorder, reports of symptoms of nausea or vomiting in pregnancy were strikingly diminished. Our findings support a causal role for fetal derived GDF15 in the nausea and vomiting of human pregnancy, with maternal sensitivity, at least partly determined by pre-pregnancy exposure to GDF15, being a major influence on its severity. They also suggest mechanism-based approaches to the treatment and prevention of HG.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546898">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546898" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546898">
        <p class="paperTitle">Is a Picture Worth 1,000 SNPs? Effects of User-Submitted Photographs on Ancestry Estimates from Direct-to-Consumer Canine Genetic Tests</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546898" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546898" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rando, H.; Graim, K.; Hampikian, G.; Greene, C. S.</p>
        <p class="info">Score: 11.6, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546898' target='https://doi.org/10.1101/2023.06.28.546898'> 10.1101/2023.06.28.546898</a></p>
        <p class="abstract">Objective: To evaluate whether the breed ancestry predictions of direct-to-consumer (DTC) genetic tests for dogs are influenced by the user-provided photograph. Animals: Twelve pet dogs considered purebred (i.e., registered with a breed organization) representing twelve different breeds. Methods: Six buccal swabs per dog were collected by the owners and submitted to six DTC genetic testing companies. The experimenters registered each sample with the company. For half of the dogs, the registration included a photograph of the DNA donor. For the other half of the dogs, photographs were swapped between dogs. Analysis of the DNA and breed ancestry prediction was conducted by each company. Each company&#39;s breed predictions were evaluated to assess whether the condition (i.e., matching versus shuffled photograph) affected the odds of identifying the DNA donor&#39;s registered breed. A convolutional neural network was also used to predict breed based solely on the photograph as a positive control. Results: Five of the six tests always produced results that included the registered breed. One test and the convolutional neural network were unlikely to identify the registered breed and frequently returned results that included the breed in the photograph. This result suggests that one test on the market is relying on the photograph more than the DNA sample. Additionally, differences in the predictions made across all tests underscore the challenge of identifying breed ancestry, even in purebred dogs. Clinical Relevance: Veterinarians are likely to encounter patients who have conducted DTC genetic testing and may find themselves in the position of explaining genetic test results that they did not order. This systematic comparison of tests on the market provides context for interpreting unexpected results from consumer-grade DTC genetic testing kits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546519" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546519">
        <p class="paperTitle">A whole organism map of X-inactivation reveals tissue-specific escape as an exceptionally rare event in humans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gylemo, B.; Nestor, C. E.</p>
        <p class="info">Score: 11.6, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546519' target='https://doi.org/10.1101/2023.06.26.546519'> 10.1101/2023.06.26.546519</a></p>
        <p class="abstract">As females are mosaic for X-inactivation, direct determination of X-linked allelic expression in bulk tissues is typically unfeasible. Using females that are non-mosaic for X-inactivation (nmXCI) as a powerful and natural genetic system we directly determined X-inactivation status of 401 X-linked genes across 32 normal tissues. We reveal that nmXCI is common in the female population and that contrary to previous reports, variable escape is an exceedingly rare event in humans. As the most complete atlas of XCI in humans to date, our findings represent a major advance in our understanding of human X-inactivation and will serve as a reference for dissecting the genetic origin of sex-bias in human traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.02.03.478491">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.02.03.478491" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.02.03.478491">
        <p class="paperTitle">Genetic population structure across Brittany and the downstream Loire basin provides new insights on the demographic history of Western Europe</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.02.03.478491" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.02.03.478491" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alves, I.; Giemza, J.; Blum, M.; Bernhardsson, C.; Chatel, S.; Karakachoff, M.; Saint Pierre, A.; Herzig, A. F.; Olaso, R.; Monteil, M.; Gallien, V.; Cabot, E.; Svensson, E.; Bacq Daian, D.; Baron, E.; Berthellier, C.; Besse, C.; Blanche, H.; Bocher, O.; Boland, A.; Bonnaud, S.; Charpentier, E.; Dandine-Roulland, C.; Ferec, C.; Fruchet, C.; Lecointe, S.; Le Floch, E.; Ludwig, T. E.; Marenne, G.; Meyer, V.; Quellery, E.; Racimo, F.; Rouault, K.; Sandron, F.; Schott, J.-J.; Velo Suarez, L.; Violleau, J.; Willerslev, E.; Coativy, Y.; Jezequel, M.; Le Bris, D.; Nicolas, C.; Pailler, Y.; Goldberg, </p>
        <p class="info">Score: 40.9, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.02.03.478491' target='https://doi.org/10.1101/2022.02.03.478491'> 10.1101/2022.02.03.478491</a></p>
        <p class="abstract">European genetic ancestry originates from three main ancestral populations - Western hunter-gatherers, early European farmers and Yamnaya Eurasian herders - whose edges geographically met in present-day France. Despite its central role to our understanding of how the ancestral populations interacted and gave rise to modern population structure, the population history of France has remained largely understudied. Here, we analysed the high-coverage whole-genome sequences and genome-wide genotype profiles of respectively 856 and 3,234 present-day individuals from the northern half of France, and merged them with publicly available present-day and ancient Europe-wide genotype datasets. We also explored, for the first time, the whole-genome sequences of six mediaeval individuals (300-1100 CE) from Western France to gain insights into the genetic impact of what is commonly known as the Migration Period in Europe. We found extensive fine-scale population structure across Brittany and the downstream Loire basin, emphasising the need for investigating local populations to better understand the distribution of rare and putatively deleterious variants across space. Overall, we observed an increased population differentiation between the northern and southern sides of the river Loire, which are characterised by different proportions of steppe vs. Neolithic-related ancestry. Samples from Western Brittany carry the largest levels of steppe ancestry and show high levels of allele sharing with individuals associated with the Bell Beaker complex, levels that are only comparable with those found in populations lying on the northwestern edges of Europe. Together, our results imply that present-day individuals from Western Brittany retain substantial legacy of the genetic changes that occurred in Northwestern Europe following the arrival of the Bell Beaker people c. 2500 BCE. Such genetic legacy may explain the sharing of disease-related alleles with other present-day populations from Western Britain and Ireland.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546954">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546954" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546954">
        <p class="paperTitle">Sex peptide receptor is not required for refractoriness to remating or induction of egg laying in Aedes aegypti</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546954" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546954" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Amaro, I. A.; Wohl, M. P.; Pitcher, S.; Alfonso-Parra, C.; Avila, F. W.; Paige, A. S.; Helinski, M.; Duvall, L. B.; Harrington, L. C.; Wolfner, M. F.; McMeniman, C. J.</p>
        <p class="info">Score: 10.8, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546954' target='https://doi.org/10.1101/2023.06.28.546954'> 10.1101/2023.06.28.546954</a></p>
        <p class="abstract">Across diverse insect taxa, the behavior and physiology of females dramatically changes after mating processes largely triggered by the transfer of seminal proteins from their mates. In the vinegar fly Drosophila melanogaster, the seminal protein sex peptide (SP) decreases the likelihood of female flies remating and causes additional behavioral and physiological changes that promote fertility including increasing egg production. Although SP is only found in the Drosophila genus, its receptor, sex peptide receptor (SPR), is the widely-conserved myoinhibitory peptide (MIP) receptor. To test the functional role of SPR in mediating post-mating responses in a non-Drosophila dipteran, we generated two independent Spr-knockout alleles in the yellow fever mosquito Aedes aegypti. Although SPR is needed for post-mating responses in Drosophila and the cotton bollworm Helicoverpa armigera, Spr mutant Ae. aegypti show completely normal post-mating decreases in remating propensity and increases in egg laying. In addition, injection of synthetic SP or accessory gland homogenate from D. melanogaster into virgin female mosquitoes did not elicit these post-mating responses. Our results indicate that Spr is not required for these canonical post-mating responses in Ae. aegypti, indicating that unknown signaling pathways are likely responsible for these behavioral switches in this disease vector.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546670">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546670" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546670">
        <p class="paperTitle">The ChickenGTEx pilot analysis: a reference of regulatory variants across 28 chicken tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546670" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546670" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guan, D.; Bai, Z.; Zhu, X.; Zhong, C.; Hou, Y.; Lan, F.; Diao, S.; Yao, Y.; Zhao, B.; Zhu, D.; Li, X.; Pan, Z.; Gao, Y.; Wang, Y.; Zou, D.; Wang, R.; Xu, T.; Sun, C.; Yin, H.; Teng, J.; Xu, Z.; Lin, Q.; Shi, S.; Shao, D.; Degalez, F.; Lagarrigue, S.; Wang, Y.; Wang, M.; Peng, M.; Rocha, D.; Charles, M.; Smith, J.; Watson, K.; Buitenhuis, A. J.; Sahana, G.; Lund, M. S.; Warren, W.; Frantz, L.; Larson, G.; Lamont, S. J.; Si, W.; Zhao, X.; Li, B.; Zhang, H.; Luo, C.; Shu, D.; Qu, H.; Luo, W.; Li, Z.; Nie, Q.; Zhang, X.; Zhang, Z.; Zhang, Z.; Liu, G. E.; Cheng, H.; Yang, N.; Hu, X.; Zhou, H.; Fang</p>
        <p class="info">Score: 10.3, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546670' target='https://doi.org/10.1101/2023.06.27.546670'> 10.1101/2023.06.27.546670</a></p>
        <p class="abstract">Chicken is a valuable model for understanding fundamental biology, vertebrate evolution and diseases, as well as a major source of nutrient-dense and lean-protein-enriched food globally. Although it is the first non-mammalian amniote genome to be sequenced, the chicken genome still lacks a systematic characterization of functional impacts of genetic variants. Here, through integrating 7,015 RNA-Seq and 2,869 whole-genome sequence data, the Chicken Genotype-Tissue Expression (ChickenGTEx) project presents the pilot reference of regulatory variants in 28 chicken tissue transcriptomes, including millions of regulatory effects on primary expression (including protein-coding genes, lncRNA and exon) and post-transcriptional modifications (alternative splicing and 3 untranslated region alternative polyadenylation). We explored the tissue-sharing and context-specificity of these regulatory variants, their underlying molecular mechanisms of action, and their utility in interpreting adaptation and genome-wide associations of 108 chicken complex traits. Finally, we illustrated shared and lineage-specific features of gene regulation between chickens and mammals, and demonstrated how the ChickenGTEx resource can further assist with translating genetic findings across species.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
