<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Accurate inference of population history in the presence of background selection
Authors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.
Score: 32.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576291
1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Accurate inference of population history in the presence of background selection
Authors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.
Score: 32.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576291
1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-24T10:40:04+00:00" />
<meta property="article:modified_time" content="2024-01-24T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Accurate inference of population history in the presence of background selection
Authors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.
Score: 32.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576291
1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Accurate inference of population history in the presence of background selection\nAuthors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.\nScore: 32.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576291\n1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species.",
  "keywords": [
    
  ],
  "articleBody": " Accurate inference of population history in the presence of background selection\nAuthors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.\nScore: 32.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576291\n1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species. Simulations and theoretical work have shown that methods to infer changes in effective population size over time (Ne(t)) become increasingly inaccurate as the strength of linked selection increases. Here, we introduce an extension to the Pairwise Sequentially Markovian Coalescent (PSMC) algorithm, PSMC+, which explicitly co-models demographic history and natural selection. We benchmark our method using forward-in-time simulations with BGS and find that our approach improves the accuracy of effective population size inference. Leveraging a high resolution map of BGS in humans, we infer considerable changes in the magnitude of inferred effective population size relative to previous reports. Finally, we separately infer Ne(t) on the X chromosome and on the autosomes in diverse great apes without making a correction for selection, and find that the inferred ratio fluctuates substantially through time in a way that differs across species, showing that uncorrected selection may be an important driver of signals of genetic difference on the X chromosome and autosomes.\nDiencephalic and Neuropeptidergic Dysfunction in Zebrafish with Autism Risk Mutations\nAuthors: Capps, M. E. S.; Moyer, A. J.; Conklin, C. L.; Martina, V.; Torija-Olson, E. G.; Klein, M. C.; Gannaway, W. C.; Calhoun, C. C. S.; Vivian, M. D.; Thyme, S. B.\nScore: 9.9, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576309\nHundreds of human mutations are linked to autism and related disorders, yet the functions of many of these mutated genes during vertebrate neurodevelopment are unclear. We generated 27 zebrafish mutants with presumptive protein-truncating mutations or specific missense variants corresponding to autism-risk alleles in 17 human genes. We observed baseline and stimulus-driven behavioral changes at larval stages, as well as social behavior differences in lines tested as juveniles. Imaging whole-brain activity revealed a near identical activity map for mutations in the unrelated genes kmt5b and hdlbpa, defined by increased activity mainly in the diencephalon. Mutating 7 of the 17 risk genes resulted in substantial brain size differences. Using RNA sequencing, we further defined molecular drivers of the observed phenotypes, identifying targetable disruptions in neuropeptide signaling, neuronal maturation, and cell proliferation. This multi-modal screen nominated brain regions, cell types, and molecular pathways that may contribute to autism susceptibility. TeaserZebrafish screen uncovers diencephalon, social interaction, and neuropeptidergic signaling phenotypes in ASD risk mutants.\nThe neocortical infrastructure for language involves region-specific patterns of laminar gene expression\nAuthors: Wong, M. M. K.; Sha, Z.; Lutje, L.; Kong, X.-Z.; Velthuijs, N.; van Heukelum, S.; van de Berg, W. D. J.; Jonkman, L. E.; Fisher, S. E.; Francks, C.\nScore: 9.5, Published: 2024-01-20 DOI: 10.1101/2024.01.17.575815\nThe language network of the human brain has core components in the inferior frontal cortex and superior/middle temporal cortex, with left-hemisphere dominance in most people. Functional specialization and interconnectivity of these neocortical regions is likely to be reflected in their molecular and cellular profiles. Excitatory connections between cortical regions arise and innervate according to layer-specific patterns. Here we generated a new gene expression dataset from human postmortem cortical tissue samples from core language network regions, using spatial transcriptomics to discriminate gene expression across cortical layers. Integration of these data with existing single-cell expression data identified 56 genes that showed differences in laminar expression profiles between frontal and temporal language cortex together with upregulation in layer II/III and/or layer V/VI excitatory neurons. Based on data from large-scale genome-wide screening in the population, DNA variants within these 56 genes showed set-level associations with inter-individual variation in structural connectivity between left-hemisphere frontal and temporal language cortex, and with predisposition to dyslexia. The axon guidance genes SLIT1 and SLIT2 were consistently implicated. These findings identify region-specific patterns of laminar gene expression as a feature of the brains language network.\nDeep learning predicts DNA methylation regulatory variants in specific brain cell types and enhances fine mapping for brain disorders\nAuthors: Zhou, J.; Weinberger, D.; Han, S.\nScore: 4.1, Published: 2024-01-21 DOI: 10.1101/2024.01.18.576319\nDNA methylation (DNAm) is essential for brain development and function and potentially mediates the effects of genetic risk variants underlying brain disorders. We present INTERACT, a transformer-based deep learning model to predict regulatory variants impacting DNAm levels in specific brain cell types, leveraging existing single-nucleus DNAm data from the human brain. We show that INTERACT accurately predicts cell type-specific DNAm profiles, achieving an average area under the Receiver Operating Characteristic curve of 0.98 across cell types. Furthermore, INTERACT predicts cell type-specific DNAm regulatory variants, which reflect cellular context and enrich the heritability of brain-related traits in relevant cell types. Importantly, we demonstrate that incorporating predicted variant effects and DNAm levels of CpG sites enhances the fine mapping for three brain disorders (schizophrenia, depression, and Alzheimer disease) and facilitates mapping causal genes to particular cell types. Our study highlights the power of deep learning in identifying cell type-specific regulatory variants, which will enhance our understanding of the genetics of complex traits.\nSAM-DNMT3A, a strategy for induction of genome-wide DNA methylation, identifies DNA methylation as a vulnerability in ER-positive breast cancers\nAuthors: Hosseinpour, M.; Malaver-Ortega, L.; Perlaza-Jimenez, L.; Joo, J. E.; Liu, L.; Caldon, C. E.; Dugue, P.-A.; Dowty, J. G.; Southey, M. C.; Rosenbluh, J.\nScore: 2.8, Published: 2024-01-16 DOI: 10.1101/2024.01.16.575955\nDNA methylation is an epigenetic mark that plays a critical role in regulation of gene expression. DNA methylase (DNMT) inhibitors, inhibit global DNA methylation, and have been a key tool in studies of DNA methylation in healthy or disease conditions. A major bottleneck is the lack of tools to induce global DNA methylation. Here, we engineered a CRISPR based approach, that was initially designed, to enable site specific DNA methylation. Using the synergistic activation mediator (SAM) system, we unexpectedly found that regardless of the targeted sequence any sgRNA induced global genome-wide DNA methylation. We termed this new method SAM-DNMT3A and show that induction of global DNA methylation is a unique vulnerability in ER-positive breast cancer suggesting a therapeutic approach. Our findings highlight the need of caution when using CRISPR based approaches for inducing DNA methylation and demonstrate a new method for global induction of DNA methylation.\nCreation and Validation of the First Infinium DNA Methylation Array for the Human Imprintome\nAuthors: Carreras-Gallo, N.; Dwaraka, V. B.; Jima, D. D.; Skaar, D. A.; Mendez, T. L.; Planchart, A.; Zhou, W.; Jirtle, R. L.; Smith, R.; Hoyo, C.\nScore: 2.4, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575646\nBackgroundDifferentially methylated imprint control regions (ICRs) regulate the monoallelic expression of imprinted genes. Their epigenetic dysregulation by environmental exposures throughout life results in the formation of common chronic diseases. Unfortunately, existing Infinium methylation arrays lack the ability to profile these regions adequately. Whole genome bisulfite sequencing (WGBS) is the unique method able to profile these regions, but it is very expensive and it requires not only a high coverage but it is also computationally intensive to assess those regions. FindingsTo address this deficiency, we developed a custom methylation array containing 22,819 probes. Among them, 9,757 probes map to 1,088 out of the 1,488 candidate ICRs recently described. To assess the performance of the array, we created matched samples processed with the Human Imprintome array and WGBS, which is the current standard method for assessing the methylation of the Human Imprintome. We compared the methylation levels from the shared CpG sites and obtained a mean R2 = 0.569. We also created matched samples processed with the Human Imprintome array and the Infinium Methylation EPIC v2 array and obtained a mean R2 = 0.796. Furthermore, replication experiments demonstrated high reliability (ICC: 0.799-0.945). ConclusionsOur custom array will be useful for replicable and accurate assessment, mechanistic insight, and targeted investigation of ICRs. This tool should accelerate the discovery of ICRs associated with a wide range of diseases and exposures, and advance our understanding of genomic imprinting and its relevance in development and disease formation throughout the life course.\nHigher-order epistasis within Pol II trigger loop haplotypes\nAuthors: Duan, B.; Qiu, C.; Lockless, S. W.; Sze, S.-H.; Kaplan, C. D.\nScore: 1.0, Published: 2024-01-21 DOI: 10.1101/2024.01.20.576280\nRNA polymerase II (Pol II) has a highly conserved domain, the trigger loop (TL), that controls transcription fidelity and speed. We previously probed pairwise genetic interactions between residues within and surrounding the TL and identified widespread incompatibility between TLs of different species when placed in the Saccharomyces cerevisiae Pol II context, indicating epistasis between the TL and its surrounding context. We sought to understand the nature of this incompatibility and probe higher order epistasis internal to the TL. We have employed deep mutational scanning with selected natural TL variants (\"haplotypes\"), and all possible intermediate substitution combinations between them and the yeast Pol II TL. We identified both positive and negative higher-order residue interactions within example TL haplotypes. Intricate higher-order epistasis formed by TL residues was sometimes only apparent from analysis of intermediate genotypes, emphasizing complexity of epistatic interactions. Furthermore, we distinguished TL substitutions with distinct classes of epistatic patterns, suggesting specific TL residues that potentially influence TL evolution. Our examples of complex residue interactions suggest possible pathways for epistasis to facilitate Pol II evolution.\nThe cytidine deaminase APOBEC3C has unique sequence and genome feature preferences\nAuthors: Brown, G. W.\nScore: 1.0, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575754\nAPOBEC proteins are cytidine deaminases that restrict the replication of viruses and transposable elements. Several members of the APOBEC3 family, APOBEC3A, APOBEC3B, and APOBEC3H-I, can access the nucleus and cause what is thought to be indiscriminate deamination of the genome, resulting in mutagenesis and genome instability. Although APOBEC3C is also present in the nucleus, its deamination target preferences are unknown. By expressing human APOBEC3C in a yeast model system, I have defined the APOBEC3C mutation signature, as well as the preferred genome features of APOBEC3C targets. The APOBEC3C mutation signature is distinct from those of the known cancer genome mutators APOBEC3A and APOBEC3B. APOBEC3C produces DNA strand-coordinated mutation clusters, and APOBEC3C mutations are enriched near the transcription start sites of active genes. Surprisingly, APOBEC3C lacks the bias for the lagging strand of DNA replication that is seen for APOBEC3A and APOBEC3B. The unique preferences of APOBEC3C constitute a mutation profile that will be useful in defining sites of APOBEC3C mutagenesis in human genomes.\nSingle-step generation of homozygous knock-out/knock-in individuals in an extremotolerant parthenogenetic tardigrade using DIPA-CRISPR\nAuthors: Kondo, K.; Tanaka, A.; Kunieda, T.\nScore: 16.9, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575120\nTardigrades are small aquatic invertebrates known for their remarkable tolerance to diverse extreme stresses. To elucidate the in vivo mechanisms underlying this extraordinary resilience, the genetic manipulation methods in tardigrades have long been desired. Despite our prior success in somatic cell gene-editing by microinjecting Cas9 ribonucleoproteins (RNPs) into the body cavity of tardigrades, the generation of gene-edited individuals remained elusive. In this study, employing an extremotolerant parthenogenetic tardigrade species, Ramazzottius varieornatus, we established conditions conductive to generating gene-edited tardigrade individuals. Drawing inspiration from the direct parental CRIPSR (DIPA-CRISPR) technique employed in several insects, we simply injected a concentrated Cas9 RNP solution into the body cavity of parental females shortly before their initial oviposition. This approach yielded gene-edited G0 progeny. Notably, only a single allele was predominantly detected at the target locus for each G0 individual, indicative of homozygous mutations. Through co-injecting single-stranded oligodeoxynucleotides (ssODNs) with Cas9 RNPs, we achieved the generation of homozygously knocked-in G0 progeny and these edited-alleles were inherited by G1/G2 progeny. This establishment of a simple method for generating homozygous knock-out/knock-in individuals not only facilitates in vivo analyses of molecular mechanisms underpinning extreme tolerance but also opens avenues for exploring various topics, including Evo-Devo, in tardigrades.\nGene-environment correlation: The role of family environment in academic development\nAuthors: Zhou, Q.; Gidziela, A.; Allegrini, A. G.; Cheesman, R.; Wertz, J.; Maxwell, J.; Plomin, R.; Rimfeld, K.; Malanchini, M.\nScore: 21.9, Published: 2024-01-08 DOI: 10.1101/2024.01.05.574339\nAcademic achievement is partly heritable and highly polygenic. However, genetic effects on academic achievement are not independent of environmental processes. We investigated whether aspects of the family environment mediated genetic effects on academic achievement across development. Our sample included 5,151 children who participated in the Twins Early Development Study, as well as their parents and teachers. Data on academic achievement and family environments were available at ages 7, 9, 12 and 16. We computed educational attainment polygenic scores (PGS) and further separated genetic effects into cognitive and noncognitive PGS. Three core findings emerged. First, aspects of the family environment, but not the wider neighbourhood context, consistently mediated the PGS effects on achievement across development, accounting for up to 34.3% of the total effect. Family characteristics mattered beyond socio-economic status. Second, family environments were more robustly linked to noncognitive PGS effects on academic achievement than cognitive PGS effects. Third, when we investigated whether environmental mediation effects could also be observed when considering differences between siblings, adjusting for family fixed effects, we found that environmental mediation was nearly exclusively observed between families. This is consistent with the proposition that family environmental contexts contribute to academic development via passive gene-environment correlation processes. Our results show how parents shape environments that foster their childrens academic development partly based on their own genetic disposition, particularly towards noncognitive skills.\n",
  "wordCount" : "2236",
  "inLanguage": "en",
  "datePublished": "2024-01-24T10:40:04Z",
  "dateModified": "2024-01-24T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 24, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576291">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576291" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576291">
        <p class="paperTitle">Accurate inference of population history in the presence of background selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576291" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576291" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.</p>
        <p class="info">Score: 32.3, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576291' target='https://doi.org/10.1101/2024.01.18.576291'> 10.1101/2024.01.18.576291</a></p>
        <p class="abstract">1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species. Simulations and theoretical work have shown that methods to infer changes in effective population size over time (Ne(t)) become increasingly inaccurate as the strength of linked selection increases. Here, we introduce an extension to the Pairwise Sequentially Markovian Coalescent (PSMC) algorithm, PSMC&#43;, which explicitly co-models demographic history and natural selection. We benchmark our method using forward-in-time simulations with BGS and find that our approach improves the accuracy of effective population size inference. Leveraging a high resolution map of BGS in humans, we infer considerable changes in the magnitude of inferred effective population size relative to previous reports. Finally, we separately infer Ne(t) on the X chromosome and on the autosomes in diverse great apes without making a correction for selection, and find that the inferred ratio fluctuates substantially through time in a way that differs across species, showing that uncorrected selection may be an important driver of signals of genetic difference on the X chromosome and autosomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576309">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576309" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576309">
        <p class="paperTitle">Diencephalic and Neuropeptidergic Dysfunction in Zebrafish with Autism Risk Mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576309" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576309" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Capps, M. E. S.; Moyer, A. J.; Conklin, C. L.; Martina, V.; Torija-Olson, E. G.; Klein, M. C.; Gannaway, W. C.; Calhoun, C. C. S.; Vivian, M. D.; Thyme, S. B.</p>
        <p class="info">Score: 9.9, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576309' target='https://doi.org/10.1101/2024.01.18.576309'> 10.1101/2024.01.18.576309</a></p>
        <p class="abstract">Hundreds of human mutations are linked to autism and related disorders, yet the functions of many of these mutated genes during vertebrate neurodevelopment are unclear. We generated 27 zebrafish mutants with presumptive protein-truncating mutations or specific missense variants corresponding to autism-risk alleles in 17 human genes. We observed baseline and stimulus-driven behavioral changes at larval stages, as well as social behavior differences in lines tested as juveniles. Imaging whole-brain activity revealed a near identical activity map for mutations in the unrelated genes kmt5b and hdlbpa, defined by increased activity mainly in the diencephalon. Mutating 7 of the 17 risk genes resulted in substantial brain size differences. Using RNA sequencing, we further defined molecular drivers of the observed phenotypes, identifying targetable disruptions in neuropeptide signaling, neuronal maturation, and cell proliferation. This multi-modal screen nominated brain regions, cell types, and molecular pathways that may contribute to autism susceptibility.

TeaserZebrafish screen uncovers diencephalon, social interaction, and neuropeptidergic signaling phenotypes in ASD risk mutants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.575815">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.575815" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.575815">
        <p class="paperTitle">The neocortical infrastructure for language involves region-specific patterns of laminar gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.575815" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.575815" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wong, M. M. K.; Sha, Z.; Lutje, L.; Kong, X.-Z.; Velthuijs, N.; van Heukelum, S.; van de Berg, W. D. J.; Jonkman, L. E.; Fisher, S. E.; Francks, C.</p>
        <p class="info">Score: 9.5, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.575815' target='https://doi.org/10.1101/2024.01.17.575815'> 10.1101/2024.01.17.575815</a></p>
        <p class="abstract">The language network of the human brain has core components in the inferior frontal cortex and superior/middle temporal cortex, with left-hemisphere dominance in most people. Functional specialization and interconnectivity of these neocortical regions is likely to be reflected in their molecular and cellular profiles. Excitatory connections between cortical regions arise and innervate according to layer-specific patterns. Here we generated a new gene expression dataset from human postmortem cortical tissue samples from core language network regions, using spatial transcriptomics to discriminate gene expression across cortical layers. Integration of these data with existing single-cell expression data identified 56 genes that showed differences in laminar expression profiles between frontal and temporal language cortex together with upregulation in layer II/III and/or layer V/VI excitatory neurons. Based on data from large-scale genome-wide screening in the population, DNA variants within these 56 genes showed set-level associations with inter-individual variation in structural connectivity between left-hemisphere frontal and temporal language cortex, and with predisposition to dyslexia. The axon guidance genes SLIT1 and SLIT2 were consistently implicated. These findings identify region-specific patterns of laminar gene expression as a feature of the brains language network.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576319">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576319" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576319">
        <p class="paperTitle">Deep learning predicts DNA methylation regulatory variants in specific brain cell types and enhances fine mapping for brain disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576319" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576319" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, J.; Weinberger, D.; Han, S.</p>
        <p class="info">Score: 4.1, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576319' target='https://doi.org/10.1101/2024.01.18.576319'> 10.1101/2024.01.18.576319</a></p>
        <p class="abstract">DNA methylation (DNAm) is essential for brain development and function and potentially mediates the effects of genetic risk variants underlying brain disorders. We present INTERACT, a transformer-based deep learning model to predict regulatory variants impacting DNAm levels in specific brain cell types, leveraging existing single-nucleus DNAm data from the human brain. We show that INTERACT accurately predicts cell type-specific DNAm profiles, achieving an average area under the Receiver Operating Characteristic curve of 0.98 across cell types. Furthermore, INTERACT predicts cell type-specific DNAm regulatory variants, which reflect cellular context and enrich the heritability of brain-related traits in relevant cell types. Importantly, we demonstrate that incorporating predicted variant effects and DNAm levels of CpG sites enhances the fine mapping for three brain disorders (schizophrenia, depression, and Alzheimer disease) and facilitates mapping causal genes to particular cell types. Our study highlights the power of deep learning in identifying cell type-specific regulatory variants, which will enhance our understanding of the genetics of complex traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575955" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575955">
        <p class="paperTitle">SAM-DNMT3A, a strategy for induction of genome-wide DNA methylation, identifies DNA methylation as a vulnerability in ER-positive breast cancers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hosseinpour, M.; Malaver-Ortega, L.; Perlaza-Jimenez, L.; Joo, J. E.; Liu, L.; Caldon, C. E.; Dugue, P.-A.; Dowty, J. G.; Southey, M. C.; Rosenbluh, J.</p>
        <p class="info">Score: 2.8, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575955' target='https://doi.org/10.1101/2024.01.16.575955'> 10.1101/2024.01.16.575955</a></p>
        <p class="abstract">DNA methylation is an epigenetic mark that plays a critical role in regulation of gene expression. DNA methylase (DNMT) inhibitors, inhibit global DNA methylation, and have been a key tool in studies of DNA methylation in healthy or disease conditions. A major bottleneck is the lack of tools to induce global DNA methylation. Here, we engineered a CRISPR based approach, that was initially designed, to enable site specific DNA methylation. Using the synergistic activation mediator (SAM) system, we unexpectedly found that regardless of the targeted sequence any sgRNA induced global genome-wide DNA methylation. We termed this new method SAM-DNMT3A and show that induction of global DNA methylation is a unique vulnerability in ER-positive breast cancer suggesting a therapeutic approach. Our findings highlight the need of caution when using CRISPR based approaches for inducing DNA methylation and demonstrate a new method for global induction of DNA methylation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575646">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575646" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575646">
        <p class="paperTitle">Creation and Validation of the First Infinium DNA Methylation Array for the Human Imprintome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575646" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575646" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carreras-Gallo, N.; Dwaraka, V. B.; Jima, D. D.; Skaar, D. A.; Mendez, T. L.; Planchart, A.; Zhou, W.; Jirtle, R. L.; Smith, R.; Hoyo, C.</p>
        <p class="info">Score: 2.4, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575646' target='https://doi.org/10.1101/2024.01.15.575646'> 10.1101/2024.01.15.575646</a></p>
        <p class="abstract">BackgroundDifferentially methylated imprint control regions (ICRs) regulate the monoallelic expression of imprinted genes. Their epigenetic dysregulation by environmental exposures throughout life results in the formation of common chronic diseases. Unfortunately, existing Infinium methylation arrays lack the ability to profile these regions adequately. Whole genome bisulfite sequencing (WGBS) is the unique method able to profile these regions, but it is very expensive and it requires not only a high coverage but it is also computationally intensive to assess those regions.

FindingsTo address this deficiency, we developed a custom methylation array containing 22,819 probes. Among them, 9,757 probes map to 1,088 out of the 1,488 candidate ICRs recently described. To assess the performance of the array, we created matched samples processed with the Human Imprintome array and WGBS, which is the current standard method for assessing the methylation of the Human Imprintome. We compared the methylation levels from the shared CpG sites and obtained a mean R2 = 0.569. We also created matched samples processed with the Human Imprintome array and the Infinium Methylation EPIC v2 array and obtained a mean R2 = 0.796. Furthermore, replication experiments demonstrated high reliability (ICC: 0.799-0.945).

ConclusionsOur custom array will be useful for replicable and accurate assessment, mechanistic insight, and targeted investigation of ICRs. This tool should accelerate the discovery of ICRs associated with a wide range of diseases and exposures, and advance our understanding of genomic imprinting and its relevance in development and disease formation throughout the life course.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576280">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576280" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576280">
        <p class="paperTitle">Higher-order epistasis within Pol II trigger loop haplotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576280" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576280" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duan, B.; Qiu, C.; Lockless, S. W.; Sze, S.-H.; Kaplan, C. D.</p>
        <p class="info">Score: 1.0, Published: 2024-01-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576280' target='https://doi.org/10.1101/2024.01.20.576280'> 10.1101/2024.01.20.576280</a></p>
        <p class="abstract">RNA polymerase II (Pol II) has a highly conserved domain, the trigger loop (TL), that controls transcription fidelity and speed. We previously probed pairwise genetic interactions between residues within and surrounding the TL and identified widespread incompatibility between TLs of different species when placed in the Saccharomyces cerevisiae Pol II context, indicating epistasis between the TL and its surrounding context. We sought to understand the nature of this incompatibility and probe higher order epistasis internal to the TL. We have employed deep mutational scanning with selected natural TL variants (&#34;haplotypes&#34;), and all possible intermediate substitution combinations between them and the yeast Pol II TL. We identified both positive and negative higher-order residue interactions within example TL haplotypes. Intricate higher-order epistasis formed by TL residues was sometimes only apparent from analysis of intermediate genotypes, emphasizing complexity of epistatic interactions. Furthermore, we distinguished TL substitutions with distinct classes of epistatic patterns, suggesting specific TL residues that potentially influence TL evolution. Our examples of complex residue interactions suggest possible pathways for epistasis to facilitate Pol II evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575754">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575754" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575754">
        <p class="paperTitle">The cytidine deaminase APOBEC3C has unique sequence and genome feature preferences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575754" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575754" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brown, G. W.</p>
        <p class="info">Score: 1.0, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575754' target='https://doi.org/10.1101/2024.01.15.575754'> 10.1101/2024.01.15.575754</a></p>
        <p class="abstract">APOBEC proteins are cytidine deaminases that restrict the replication of viruses and transposable elements. Several members of the APOBEC3 family, APOBEC3A, APOBEC3B, and APOBEC3H-I, can access the nucleus and cause what is thought to be indiscriminate deamination of the genome, resulting in mutagenesis and genome instability. Although APOBEC3C is also present in the nucleus, its deamination target preferences are unknown. By expressing human APOBEC3C in a yeast model system, I have defined the APOBEC3C mutation signature, as well as the preferred genome features of APOBEC3C targets. The APOBEC3C mutation signature is distinct from those of the known cancer genome mutators APOBEC3A and APOBEC3B. APOBEC3C produces DNA strand-coordinated mutation clusters, and APOBEC3C mutations are enriched near the transcription start sites of active genes. Surprisingly, APOBEC3C lacks the bias for the lagging strand of DNA replication that is seen for APOBEC3A and APOBEC3B. The unique preferences of APOBEC3C constitute a mutation profile that will be useful in defining sites of APOBEC3C mutagenesis in human genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575120" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575120">
        <p class="paperTitle">Single-step generation of homozygous knock-out/knock-in individuals in an extremotolerant parthenogenetic tardigrade using DIPA-CRISPR</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kondo, K.; Tanaka, A.; Kunieda, T.</p>
        <p class="info">Score: 16.9, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575120' target='https://doi.org/10.1101/2024.01.10.575120'> 10.1101/2024.01.10.575120</a></p>
        <p class="abstract">Tardigrades are small aquatic invertebrates known for their remarkable tolerance to diverse extreme stresses. To elucidate the in vivo mechanisms underlying this extraordinary resilience, the genetic manipulation methods in tardigrades have long been desired. Despite our prior success in somatic cell gene-editing by microinjecting Cas9 ribonucleoproteins (RNPs) into the body cavity of tardigrades, the generation of gene-edited individuals remained elusive.

In this study, employing an extremotolerant parthenogenetic tardigrade species, Ramazzottius varieornatus, we established conditions conductive to generating gene-edited tardigrade individuals. Drawing inspiration from the direct parental CRIPSR (DIPA-CRISPR) technique employed in several insects, we simply injected a concentrated Cas9 RNP solution into the body cavity of parental females shortly before their initial oviposition. This approach yielded gene-edited G0 progeny. Notably, only a single allele was predominantly detected at the target locus for each G0 individual, indicative of homozygous mutations. Through co-injecting single-stranded oligodeoxynucleotides (ssODNs) with Cas9 RNPs, we achieved the generation of homozygously knocked-in G0 progeny and these edited-alleles were inherited by G1/G2 progeny.

This establishment of a simple method for generating homozygous knock-out/knock-in individuals not only facilitates in vivo analyses of molecular mechanisms underpinning extreme tolerance but also opens avenues for exploring various topics, including Evo-Devo, in tardigrades.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574339">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574339" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574339">
        <p class="paperTitle">Gene-environment correlation: The role of family environment in academic development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574339" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574339" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, Q.; Gidziela, A.; Allegrini, A. G.; Cheesman, R.; Wertz, J.; Maxwell, J.; Plomin, R.; Rimfeld, K.; Malanchini, M.</p>
        <p class="info">Score: 21.9, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574339' target='https://doi.org/10.1101/2024.01.05.574339'> 10.1101/2024.01.05.574339</a></p>
        <p class="abstract">Academic achievement is partly heritable and highly polygenic. However, genetic effects on academic achievement are not independent of environmental processes. We investigated whether aspects of the family environment mediated genetic effects on academic achievement across development. Our sample included 5,151 children who participated in the Twins Early Development Study, as well as their parents and teachers. Data on academic achievement and family environments were available at ages 7, 9, 12 and 16. We computed educational attainment polygenic scores (PGS) and further separated genetic effects into cognitive and noncognitive PGS. Three core findings emerged. First, aspects of the family environment, but not the wider neighbourhood context, consistently mediated the PGS effects on achievement across development, accounting for up to 34.3% of the total effect. Family characteristics mattered beyond socio-economic status. Second, family environments were more robustly linked to noncognitive PGS effects on academic achievement than cognitive PGS effects. Third, when we investigated whether environmental mediation effects could also be observed when considering differences between siblings, adjusting for family fixed effects, we found that environmental mediation was nearly exclusively observed between families. This is consistent with the proposition that family environmental contexts contribute to academic development via passive gene-environment correlation processes. Our results show how parents shape environments that foster their childrens academic development partly based on their own genetic disposition, particularly towards noncognitive skills.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
