<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Helicase-assisted continuous editing for programmable mutagenesis of endogenous genomes
Authors: Chen, X. D.; Chen, Z.; Wythes, G.; Zhang, Y.; Orr, B. C.; Sun, G.; Thao, K.; Vallurupalli, M.; Sun, J.; Borji, M.; Tkacik, E.; Chen, H.; Bernstein, B. E.; Chen, F.
Score: 27.8, Published: 2024-02-01 DOI: 10.1101/2024.02.01.577593
A major challenge in human genomics is to decipher the context specific relationship of sequence to function. However, existing tools for locus specific hypermutation and evolution in the native genome context are limited.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Helicase-assisted continuous editing for programmable mutagenesis of endogenous genomes
Authors: Chen, X. D.; Chen, Z.; Wythes, G.; Zhang, Y.; Orr, B. C.; Sun, G.; Thao, K.; Vallurupalli, M.; Sun, J.; Borji, M.; Tkacik, E.; Chen, H.; Bernstein, B. E.; Chen, F.
Score: 27.8, Published: 2024-02-01 DOI: 10.1101/2024.02.01.577593
A major challenge in human genomics is to decipher the context specific relationship of sequence to function. However, existing tools for locus specific hypermutation and evolution in the native genome context are limited." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-04T10:38:39+00:00" />
<meta property="article:modified_time" content="2024-02-04T10:38:39+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Helicase-assisted continuous editing for programmable mutagenesis of endogenous genomes
Authors: Chen, X. D.; Chen, Z.; Wythes, G.; Zhang, Y.; Orr, B. C.; Sun, G.; Thao, K.; Vallurupalli, M.; Sun, J.; Borji, M.; Tkacik, E.; Chen, H.; Bernstein, B. E.; Chen, F.
Score: 27.8, Published: 2024-02-01 DOI: 10.1101/2024.02.01.577593
A major challenge in human genomics is to decipher the context specific relationship of sequence to function. However, existing tools for locus specific hypermutation and evolution in the native genome context are limited."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Helicase-assisted continuous editing for programmable mutagenesis of endogenous genomes\nAuthors: Chen, X. D.; Chen, Z.; Wythes, G.; Zhang, Y.; Orr, B. C.; Sun, G.; Thao, K.; Vallurupalli, M.; Sun, J.; Borji, M.; Tkacik, E.; Chen, H.; Bernstein, B. E.; Chen, F.\nScore: 27.8, Published: 2024-02-01 DOI: 10.1101/2024.02.01.577593\nA major challenge in human genomics is to decipher the context specific relationship of sequence to function. However, existing tools for locus specific hypermutation and evolution in the native genome context are limited.",
  "keywords": [
    
  ],
  "articleBody": " Helicase-assisted continuous editing for programmable mutagenesis of endogenous genomes\nAuthors: Chen, X. D.; Chen, Z.; Wythes, G.; Zhang, Y.; Orr, B. C.; Sun, G.; Thao, K.; Vallurupalli, M.; Sun, J.; Borji, M.; Tkacik, E.; Chen, H.; Bernstein, B. E.; Chen, F.\nScore: 27.8, Published: 2024-02-01 DOI: 10.1101/2024.02.01.577593\nA major challenge in human genomics is to decipher the context specific relationship of sequence to function. However, existing tools for locus specific hypermutation and evolution in the native genome context are limited. Here we present a novel programmable platform for long-range, locus-specific hypermutation called helicase-assisted continuous editing (HACE). HACE leverages CRISPR-Cas9 to target a processive helicase-deaminase fusion that incurs mutations across large (\u003e1000 bp) genomic intervals. We applied HACE to identify mutations in MEK1 that confer kinase inhibitor resistance, to dissect the impact of individual variants in SF3B1-dependent mis-splicing, and to evaluate noncoding variants in a stimulation-dependent immune enhancer of CD69. HACE provides a powerful tool for investigating coding and noncoding variants, uncovering combinatorial sequence-to-function relationships, and evolving new biological functions.\nRepairing a deleterious domestication variant in a floral regulator of tomato by base editing\nAuthors: Glaus, A. N.; Brechet, M.; Lebeigle, L.; Iwaszkiewicz, J.; Ambrosini, G.; Julca, I.; Zhang, J.; Roberts, R.; Iseli, C.; Guex, N.; Jimenez-Gomez, J. M.; Glover, N.; Martin, G. B.; Strickler, S. R.; Soyk, S.\nScore: 19.7, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577624\nCrop genomes accumulated deleterious mutations, a symptom known as the cost of domestication. Precision genome editing has been proposed to eliminate such potentially harmful mutations, however, experimental demonstration is lacking. Here, we identified a deleterious mutation in the tomato transcription factor SUPPRESSOR OF SP2 (SSP2), which became prevalent in the domesticated germplasm and diminished DNA-binding to genome-wide targets. We found that SSP2 acts partially redundant with its paralog SSP to regulate shoot and inflorescence architecture. However, redundancy was compromised during tomato domestication and completely lost in the closely-related species Physalis grisea, in which a single ortholog regulates shoot branching. We applied base editing to directly repair the deleterious mutation in cultivated tomato and obtained plants with compact growth that provide an early fruit yield. Our work shows how deleterious variants sensitized modern genotypes for phenotypic tuning and illustrates how repairing deleterious mutations with genome editing allows for predictable crop improvement.\nPatient derived model of UBA5-associated encephalopathy identifies defects in neurodevelopment and highlights potential therapies\nAuthors: Chen, H.; Wang, Y.-D.; Blan, A. W.; Almanza-Fuerte, E. P.; Bonkowski, E. S.; Bajpai, R.; Pruett-Miller, S. M.; Mefford, H. C.\nScore: 14.0, Published: 2024-01-27 DOI: 10.1101/2024.01.25.577254\nUBA5 encodes for the E1 enzyme of the UFMylation cascade, which plays an essential role in ER homeostasis. The clinical phenotypes of UBA5-associated encephalopathy include developmental delays, epilepsy and intellectual disability. To date, there is no humanized neuronal model to study the cellular and molecular consequences of UBA5 pathogenic variants. We developed and characterized patient-derived cortical organoid cultures and identified defects in GABAergic interneuron development. We demonstrated aberrant neuronal firing and microcephaly phenotypes in patient-derived organoids. Mechanistically, we show that ER homeostasis is perturbed along with exacerbated unfolded protein response pathway in cells and organoids expressing UBA5 pathogenic variants. We also assessed two gene expression modalities that augmented UBA5 expression to rescue aberrant molecular and cellular phenotypes. Our study provides a novel humanized model that allows further investigations of UBA5 variants in the brain and highlights novel systemic approaches to alleviate cellular aberrations for this rare, developmental disorder. One-sentence summaryPatient derived model of UBA5-assoicated DEE recapitulated disease phenotype, revealed defects in neurodevelopment, and highlighted potential therapies.\nHistone variant H2BE enhances chromatin accessibility in neurons to promote synaptic gene expression and long-term memory\nAuthors: Feierman, E. R.; Louzon, S.; Prescott, N. A.; Biaco, T.; Gao, Q.; Qiu, Q.; Choi, K.; Palozola, K. C.; Voss, A. J.; Mehta, S.; Quaye, C.; Lynch, K.; Fuccillo, M. V.; Wu, H.; David, Y.; Korb, E.\nScore: 5.3, Published: 2024-01-30 DOI: 10.1101/2024.01.29.575103\nRegulation of histones occurs through multiple mechanisms including exchange with histone variants. Unlike canonical histones, variants are replication-independent and therefore accumulate in post-mitotic cells such as neurons. While recent findings link variants to neurological and neuropsychiatric disorders, few are well studied in the context of the brain. H2BE is the single H2B variant found outside germline tissues, yet its expression and effects on chromatin remained unclear. We applied new tools including novel antibodies, biochemical assays, and sequencing approaches to reveal broad H2BE expression in the brain and its role in regulating chromatin structure, neuronal transcription, and mouse behavior. H2BE is enriched at promoters and enhances chromatin accessibility. We further identify a single amino acid driving these accessibility changes. Lastly, we show that H2BE is critical for synaptic gene expression and long-term memory. Together, these data reveal a novel mechanism linking histone variants to chromatin regulation and neuronal function underlying memory.\nHighly parameterized polygenic scores tend to overfit to population stratification via random effects\nAuthors: Aw, A. J.; McRae, J.; Rahmani, E.; Song, Y. S.\nScore: 5.1, Published: 2024-01-29 DOI: 10.1101/2024.01.27.577589\nPolygenic scores (PGSs), increasingly used in clinical settings, frequently include many genetic variants, with performance typically peaking at thousands of variants. Such highly parameterized PGSs often include variants that do not pass a genome-wide significance threshold. We propose a mathematical perspective that renders the effects of many of these nonsignificant variants random rather than causal, with the randomness capturing population structure. We devise methods to assess variant effect randomness and population stratification bias. Applying these methods to 141 traits from the UK Biobank, we find that, for many PGSs, the effects of non-significant variants are considerably random, with the extent of randomness associated with the degree of overfitting to population structure of the discovery cohort. Our findings explain why highly parameterized PGSs simultaneously have superior cohort-specific performance and limited generalizability, suggesting the critical need for variant randomness tests in PGS evaluation. Supporting code and a dashboard are available at https://github.com/songlab-cal/StratPGS.\nDissecting quantitative trait nucleotides by saturation genome editing\nAuthors: Roy, K. R.; Smith, J. D.; Li, S.; Vonesch, S. C.; Nguyen, M.; Burnett, W. T.; Orsley, K. M.; Lee, C.-S.; Haber, J. E.; St.Onge, R. P.; Steinmetz, L. M.\nScore: 5.0, Published: 2024-02-02 DOI: 10.1101/2024.02.02.577784\nGenome editing technologies have the potential to transform our understanding of how genetic variation gives rise to complex traits through the systematic engineering and phenotypic characterization of genetic variants. However, there has yet to be a system with sufficient efficiency, fidelity, and throughput to comprehensively identify causal variants at the genome scale. Here we explored the ability of templated CRISPR editing systems to install natural variants genome-wide in budding yeast. We optimized several approaches to enhance homology-directed repair (HDR) with donor DNA templates, including donor recruitment to target sites, single-stranded donor production by bacterial retrons, and in vivo plasmid assembly. We uncovered unique advantages of each system that we integrated into a single superior system named MAGESTIC 3.0. We used MAGESTIC 3.0 to dissect causal variants residing in 112 quantitative trait loci across 32 environmental conditions, revealing an enrichment for missense variants and loci with multiple causal variants. MAGESTIC 3.0 will facilitate the functional analysis of the genome at single-nucleotide resolution and provides a roadmap for improving template-based genome editing systems in other organisms.\nAssessing the effects of acute temperature changes of sea water temperature due to climate change on Scottish farmed Atlantic salmon and investigating genetic mitigation\nAuthors: Tsairidou, S.; van den Berg, J.; Tapping, S.; Sobolewska, H.; Hamilton, A.\nScore: 4.1, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577726\nChanges in growth, survival, maturation and health in Atlantic salmon post-smolts were observed under moderate and more extreme heat-wave conditions for the west coast of Scotland. A Scottish Atlantic salmon breeding programme population of 518 salmon of age 13-14 months, was observed for 4 weeks in summer, in tanks with ambient water temperature or [~] 4 degrees and [~] 8 degrees above ambient temperature. Data were recorded for the fish before and after the challenge. All fish were genotyped using a custom SNP (single nucleotide polymorphism) array (8,978 SNP genotypes after quality control). Temperature-dependant genotype-by-environment interactions and the potential for selective breeding to improve resilience were investigated. Statistical analyses revealed significant differences between thermal environments for body weight, average daily weight gain, and survival, while higher temperature induced earlier maturation, and an increase of gill health scores. QPCR (quantitative polymerase chain reaction) analyses revealed the presence of Aeromonas hydrophila, a pathogen typically more prevalent in warmer climates. Using genomic relationships and animal mixed models, body weight and average daily weight gain provided moderate heritabilities, while between-tank genetic correlations were close to 1, indicating no significant re-ranking of genotypes between the different thermal environments. These findings suggest that even short-term exposure to heat stress may be sufficient to negatively impact survival and gill health, and induce earlier maturation. However, observations took place within a commercial farm where having replicates for each thermal environment was not possible, hence further experiments with larger populations, exposed to more prolonged heat stress are needed.\nBridge RNAs direct modular and programmable recombination of target and donor DNA\nAuthors: Durrant, M. G.; Perry, N. T.; Pai, J. J.; Jangid, A. R.; Athukoralage, J. S.; Hiraizumi, M.; McSpedon, J. P.; Pawluk, A.; Nishimasu, H.; Konermann, S.; Hsu, P. D.\nScore: 11.1, Published: 2024-01-26 DOI: 10.1101/2024.01.24.577089\nGenomic rearrangements, encompassing mutational changes in the genome such as insertions, deletions, or inversions, are essential for genetic diversity. These rearrangements are typically orchestrated by enzymes involved in fundamental DNA repair processes such as homologous recombination or in the transposition of foreign genetic material by viruses and mobile genetic elements (MGEs). We report that IS110 insertion sequences, a family of minimal and autonomous MGEs, express a structured non-coding RNA that binds specifically to their encoded recombinase. This bridge RNA contains two internal loops encoding nucleotide stretches that base-pair with the target DNA and donor DNA, which is the IS110 element itself. We demonstrate that the target-binding and donor-binding loops can be independently reprogrammed to direct sequence-specific recombination between two DNA molecules. This modularity enables DNA insertion into genomic target sites as well as programmable DNA excision and inversion. The IS110 bridge system expands the diversity of nucleic acid-guided systems beyond CRISPR and RNA interference, offering a unified mechanism for the three fundamental DNA rearrangements required for genome design.\nAssessing the potential of germplasm collections for the management of genetic diversity: the case of the French National Cryobank\nAuthors: Jacques, A.; Duclos, D.; Danchin-Burge, C.; Mercat, M.-J.; Tixier-Boichard, M.; Restoux, G.\nScore: 5.2, Published: 2024-01-30 DOI: 10.1101/2023.07.19.549644\nThrough a combination of selective pressure and genetic drift, there has been a notable erosion of genetic diversity in domesticated animal populations. In response, many countries, including France, have developed gene banks in order to conserve reproductive genetic material. Cryopreserved resources can potentially be used to manage the genetic diversity of populations, but this opportunity is seldom exploited. As an initial step toward this goal, we describe here a methodology for the characterization of the genetic diversity of cryopreserved collections. Using the example of the French National Cryobank, this study employs newly proposed biodiversity metrics to conduct a detailed assessment of the status of collections for six livestock species: cattle, sheep, goat, horse, donkey, and pig. Both the amount of resources available and their characteristics varied among species and/or breeds. Overall, breeds with a large commercial distribution had more donors in the collection than local breeds, while the number of doses available was mainly determined by the physiology of the species. An adapted version of the Gini-Simpson equitability index revealed an unbalanced number of donors between breeds for some species. Similarly, estimates of effective donor numbers (De) highlighted the unequal distribution of donors within a breed. Finally, we developed a new index of diversity impact (IDI) to assess the potential of a collection to reintroduce genetic diversity in contemporary populations. The IDI was calculated on the basis of pedigree data for 17 breeds of three livestock species, pig, sheep, and cattle, which differed in both population size and management program. IDI values are negative when the use of cryoconserved sires would decrease the overall kinship of the current population and positive when it would increase it, enabling the most interesting donors to be chosen for immediate use. Negative (favorable) IDI values 41 were found for both local breeds as well as for commercial populations. In general, older collections exhibited better IDI values but recently collected donors could also be useful for populations undergoing strong selection. Within a breed, IDI can be computed individually and thus be used to select the best sires for a given objective. In the absence of pedigree data, IDI values could also be calculated on the basis of marker genotypes. Overall, this study proposes a framework for the assessment of germplasm collections in the service of various objectives. Compared to FAO indicators motivated by breed reconstitution, the Gini-Simpson and De indices can help to plan sampling more efficiently, whereas IDI can guide donor selection in order to manage the diversity of existing populations. These indicators can be calculated at regular intervals to support the planning and management of collections at national and international levels and help population managers to exploit the resources currently available.\nImpact of inter-species hybridisation on antifungal drug response in the Saccharomyces genus.\nAuthors: Visinoni, F.; Royle, W.; Scholey, R.; Hu, Y.; Timouma, S.; Zeef, L.; Louis, E. J.; Delneri, D.\nScore: 2.9, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577396\nAntifungal drug resistance across fungal and yeast pathogens presents one of the major concerns for global public health. Understanding the interactions between genetic background and environment is important for the development of new, effective treatments of infections. Allelic variation within populations of Ascomycota as well as hybridisation impacts the phenotype in response to stressful conditions, including to antifungal drugs. We exploited recent advances in multigenerational breeding of Saccharomyces interspecies hybrids to study the impact of hybridisation on antifungal resistance and identify quantitative trait loci (QTL) responsible for the phenotypes observed. A library of Saccharomyces cerevisiae x S. kudriavzevii hybrid offspring was screened in the presence of sub-lethal concentrations of six antifungal drugs and revealed a broad phenotypic diversity across the progeny. QTL analysis was carried out comparing alleles between the pools of high and low fitness offspring, identifying hybrid-specific genetic regions involved in resistance to fluconazole, micafungin and flucytosine. We found both drug specific and pleiotropic regions, and through gene ontology and SIFT analysis we identify potential causal genes, such as BCK2 and DNF1 that were validated via reciprocal hemizygosity analysis. We highlight 41 regions that contain genes not previously associated with resistance phenotypes in the literature. The results of this screening will help identify new pathways contributing to drug resistance, and lead to greater understanding of how allelic variation, hybridisation and evolution affect antifungal drug resistance in yeast and fungi.\n",
  "wordCount" : "2452",
  "inLanguage": "en",
  "datePublished": "2024-02-04T10:38:39Z",
  "dateModified": "2024-02-04T10:38:39Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 4, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.577593">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.577593" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.577593">
        <p class="paperTitle">Helicase-assisted continuous editing for programmable mutagenesis of endogenous genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.577593" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.577593" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, X. D.; Chen, Z.; Wythes, G.; Zhang, Y.; Orr, B. C.; Sun, G.; Thao, K.; Vallurupalli, M.; Sun, J.; Borji, M.; Tkacik, E.; Chen, H.; Bernstein, B. E.; Chen, F.</p>
        <p class="info">Score: 27.8, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.577593' target='https://doi.org/10.1101/2024.02.01.577593'> 10.1101/2024.02.01.577593</a></p>
        <p class="abstract">A major challenge in human genomics is to decipher the context specific relationship of sequence to function. However, existing tools for locus specific hypermutation and evolution in the native genome context are limited. Here we present a novel programmable platform for long-range, locus-specific hypermutation called helicase-assisted continuous editing (HACE). HACE leverages CRISPR-Cas9 to target a processive helicase-deaminase fusion that incurs mutations across large (&gt;1000 bp) genomic intervals. We applied HACE to identify mutations in MEK1 that confer kinase inhibitor resistance, to dissect the impact of individual variants in SF3B1-dependent mis-splicing, and to evaluate noncoding variants in a stimulation-dependent immune enhancer of CD69. HACE provides a powerful tool for investigating coding and noncoding variants, uncovering combinatorial sequence-to-function relationships, and evolving new biological functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577624">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577624" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577624">
        <p class="paperTitle">Repairing a deleterious domestication variant in a floral regulator of tomato by base editing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577624" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577624" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Glaus, A. N.; Brechet, M.; Lebeigle, L.; Iwaszkiewicz, J.; Ambrosini, G.; Julca, I.; Zhang, J.; Roberts, R.; Iseli, C.; Guex, N.; Jimenez-Gomez, J. M.; Glover, N.; Martin, G. B.; Strickler, S. R.; Soyk, S.</p>
        <p class="info">Score: 19.7, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577624' target='https://doi.org/10.1101/2024.01.29.577624'> 10.1101/2024.01.29.577624</a></p>
        <p class="abstract">Crop genomes accumulated deleterious mutations, a symptom known as the cost of domestication. Precision genome editing has been proposed to eliminate such potentially harmful mutations, however, experimental demonstration is lacking. Here, we identified a deleterious mutation in the tomato transcription factor SUPPRESSOR OF SP2 (SSP2), which became prevalent in the domesticated germplasm and diminished DNA-binding to genome-wide targets. We found that SSP2 acts partially redundant with its paralog SSP to regulate shoot and inflorescence architecture. However, redundancy was compromised during tomato domestication and completely lost in the closely-related species Physalis grisea, in which a single ortholog regulates shoot branching. We applied base editing to directly repair the deleterious mutation in cultivated tomato and obtained plants with compact growth that provide an early fruit yield. Our work shows how deleterious variants sensitized modern genotypes for phenotypic tuning and illustrates how repairing deleterious mutations with genome editing allows for predictable crop improvement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577254">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577254" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577254">
        <p class="paperTitle">Patient derived model of UBA5-associated encephalopathy identifies defects in neurodevelopment and highlights potential therapies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577254" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577254" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, H.; Wang, Y.-D.; Blan, A. W.; Almanza-Fuerte, E. P.; Bonkowski, E. S.; Bajpai, R.; Pruett-Miller, S. M.; Mefford, H. C.</p>
        <p class="info">Score: 14.0, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577254' target='https://doi.org/10.1101/2024.01.25.577254'> 10.1101/2024.01.25.577254</a></p>
        <p class="abstract">UBA5 encodes for the E1 enzyme of the UFMylation cascade, which plays an essential role in ER homeostasis. The clinical phenotypes of UBA5-associated encephalopathy include developmental delays, epilepsy and intellectual disability. To date, there is no humanized neuronal model to study the cellular and molecular consequences of UBA5 pathogenic variants. We developed and characterized patient-derived cortical organoid cultures and identified defects in GABAergic interneuron development. We demonstrated aberrant neuronal firing and microcephaly phenotypes in patient-derived organoids. Mechanistically, we show that ER homeostasis is perturbed along with exacerbated unfolded protein response pathway in cells and organoids expressing UBA5 pathogenic variants. We also assessed two gene expression modalities that augmented UBA5 expression to rescue aberrant molecular and cellular phenotypes. Our study provides a novel humanized model that allows further investigations of UBA5 variants in the brain and highlights novel systemic approaches to alleviate cellular aberrations for this rare, developmental disorder.

One-sentence summaryPatient derived model of UBA5-assoicated DEE recapitulated disease phenotype, revealed defects in neurodevelopment, and highlighted potential therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.575103">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.575103" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.575103">
        <p class="paperTitle">Histone variant H2BE enhances chromatin accessibility in neurons to promote synaptic gene expression and long-term memory</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.575103" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.575103" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Feierman, E. R.; Louzon, S.; Prescott, N. A.; Biaco, T.; Gao, Q.; Qiu, Q.; Choi, K.; Palozola, K. C.; Voss, A. J.; Mehta, S.; Quaye, C.; Lynch, K.; Fuccillo, M. V.; Wu, H.; David, Y.; Korb, E.</p>
        <p class="info">Score: 5.3, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.575103' target='https://doi.org/10.1101/2024.01.29.575103'> 10.1101/2024.01.29.575103</a></p>
        <p class="abstract">Regulation of histones occurs through multiple mechanisms including exchange with histone variants. Unlike canonical histones, variants are replication-independent and therefore accumulate in post-mitotic cells such as neurons. While recent findings link variants to neurological and neuropsychiatric disorders, few are well studied in the context of the brain. H2BE is the single H2B variant found outside germline tissues, yet its expression and effects on chromatin remained unclear. We applied new tools including novel antibodies, biochemical assays, and sequencing approaches to reveal broad H2BE expression in the brain and its role in regulating chromatin structure, neuronal transcription, and mouse behavior. H2BE is enriched at promoters and enhances chromatin accessibility. We further identify a single amino acid driving these accessibility changes. Lastly, we show that H2BE is critical for synaptic gene expression and long-term memory. Together, these data reveal a novel mechanism linking histone variants to chromatin regulation and neuronal function underlying memory.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.27.577589">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.27.577589" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.27.577589">
        <p class="paperTitle">Highly parameterized polygenic scores tend to overfit to population stratification via random effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.27.577589" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.27.577589" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aw, A. J.; McRae, J.; Rahmani, E.; Song, Y. S.</p>
        <p class="info">Score: 5.1, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.27.577589' target='https://doi.org/10.1101/2024.01.27.577589'> 10.1101/2024.01.27.577589</a></p>
        <p class="abstract">Polygenic scores (PGSs), increasingly used in clinical settings, frequently include many genetic variants, with performance typically peaking at thousands of variants. Such highly parameterized PGSs often include variants that do not pass a genome-wide significance threshold. We propose a mathematical perspective that renders the effects of many of these nonsignificant variants random rather than causal, with the randomness capturing population structure. We devise methods to assess variant effect randomness and population stratification bias. Applying these methods to 141 traits from the UK Biobank, we find that, for many PGSs, the effects of non-significant variants are considerably random, with the extent of randomness associated with the degree of overfitting to population structure of the discovery cohort. Our findings explain why highly parameterized PGSs simultaneously have superior cohort-specific performance and limited generalizability, suggesting the critical need for variant randomness tests in PGS evaluation. Supporting code and a dashboard are available at https://github.com/songlab-cal/StratPGS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.577784">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.577784" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.577784">
        <p class="paperTitle">Dissecting quantitative trait nucleotides by saturation genome editing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.577784" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.577784" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roy, K. R.; Smith, J. D.; Li, S.; Vonesch, S. C.; Nguyen, M.; Burnett, W. T.; Orsley, K. M.; Lee, C.-S.; Haber, J. E.; St.Onge, R. P.; Steinmetz, L. M.</p>
        <p class="info">Score: 5.0, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.577784' target='https://doi.org/10.1101/2024.02.02.577784'> 10.1101/2024.02.02.577784</a></p>
        <p class="abstract">Genome editing technologies have the potential to transform our understanding of how genetic variation gives rise to complex traits through the systematic engineering and phenotypic characterization of genetic variants. However, there has yet to be a system with sufficient efficiency, fidelity, and throughput to comprehensively identify causal variants at the genome scale. Here we explored the ability of templated CRISPR editing systems to install natural variants genome-wide in budding yeast. We optimized several approaches to enhance homology-directed repair (HDR) with donor DNA templates, including donor recruitment to target sites, single-stranded donor production by bacterial retrons, and in vivo plasmid assembly. We uncovered unique advantages of each system that we integrated into a single superior system named MAGESTIC 3.0. We used MAGESTIC 3.0 to dissect causal variants residing in 112 quantitative trait loci across 32 environmental conditions, revealing an enrichment for missense variants and loci with multiple causal variants. MAGESTIC 3.0 will facilitate the functional analysis of the genome at single-nucleotide resolution and provides a roadmap for improving template-based genome editing systems in other organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577726">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577726" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577726">
        <p class="paperTitle">Assessing the effects of acute temperature changes of sea water temperature due to climate change on Scottish farmed Atlantic salmon and investigating genetic mitigation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577726" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577726" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsairidou, S.; van den Berg, J.; Tapping, S.; Sobolewska, H.; Hamilton, A.</p>
        <p class="info">Score: 4.1, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577726' target='https://doi.org/10.1101/2024.01.29.577726'> 10.1101/2024.01.29.577726</a></p>
        <p class="abstract">Changes in growth, survival, maturation and health in Atlantic salmon post-smolts were observed under moderate and more extreme heat-wave conditions for the west coast of Scotland. A Scottish Atlantic salmon breeding programme population of 518 salmon of age 13-14 months, was observed for 4 weeks in summer, in tanks with ambient water temperature or [~] 4 degrees and [~] 8 degrees above ambient temperature. Data were recorded for the fish before and after the challenge. All fish were genotyped using a custom SNP (single nucleotide polymorphism) array (8,978 SNP genotypes after quality control). Temperature-dependant genotype-by-environment interactions and the potential for selective breeding to improve resilience were investigated. Statistical analyses revealed significant differences between thermal environments for body weight, average daily weight gain, and survival, while higher temperature induced earlier maturation, and an increase of gill health scores. QPCR (quantitative polymerase chain reaction) analyses revealed the presence of Aeromonas hydrophila, a pathogen typically more prevalent in warmer climates. Using genomic relationships and animal mixed models, body weight and average daily weight gain provided moderate heritabilities, while between-tank genetic correlations were close to 1, indicating no significant re-ranking of genotypes between the different thermal environments. These findings suggest that even short-term exposure to heat stress may be sufficient to negatively impact survival and gill health, and induce earlier maturation. However, observations took place within a commercial farm where having replicates for each thermal environment was not possible, hence further experiments with larger populations, exposed to more prolonged heat stress are needed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577089">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577089" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577089">
        <p class="paperTitle">Bridge RNAs direct modular and programmable recombination of target and donor DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577089" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577089" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Durrant, M. G.; Perry, N. T.; Pai, J. J.; Jangid, A. R.; Athukoralage, J. S.; Hiraizumi, M.; McSpedon, J. P.; Pawluk, A.; Nishimasu, H.; Konermann, S.; Hsu, P. D.</p>
        <p class="info">Score: 11.1, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577089' target='https://doi.org/10.1101/2024.01.24.577089'> 10.1101/2024.01.24.577089</a></p>
        <p class="abstract">Genomic rearrangements, encompassing mutational changes in the genome such as insertions, deletions, or inversions, are essential for genetic diversity. These rearrangements are typically orchestrated by enzymes involved in fundamental DNA repair processes such as homologous recombination or in the transposition of foreign genetic material by viruses and mobile genetic elements (MGEs). We report that IS110 insertion sequences, a family of minimal and autonomous MGEs, express a structured non-coding RNA that binds specifically to their encoded recombinase. This bridge RNA contains two internal loops encoding nucleotide stretches that base-pair with the target DNA and donor DNA, which is the IS110 element itself. We demonstrate that the target-binding and donor-binding loops can be independently reprogrammed to direct sequence-specific recombination between two DNA molecules. This modularity enables DNA insertion into genomic target sites as well as programmable DNA excision and inversion. The IS110 bridge system expands the diversity of nucleic acid-guided systems beyond CRISPR and RNA interference, offering a unified mechanism for the three fundamental DNA rearrangements required for genome design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549644">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549644" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549644">
        <p class="paperTitle">Assessing the potential of germplasm collections for the management of genetic diversity: the case of the French National Cryobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549644" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549644" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jacques, A.; Duclos, D.; Danchin-Burge, C.; Mercat, M.-J.; Tixier-Boichard, M.; Restoux, G.</p>
        <p class="info">Score: 5.2, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549644' target='https://doi.org/10.1101/2023.07.19.549644'> 10.1101/2023.07.19.549644</a></p>
        <p class="abstract">Through a combination of selective pressure and genetic drift, there has been a notable erosion of genetic diversity in domesticated animal populations. In response, many countries, including France, have developed gene banks in order to conserve reproductive genetic material. Cryopreserved resources can potentially be used to manage the genetic diversity of populations, but this opportunity is seldom exploited. As an initial step toward this goal, we describe here a methodology for the characterization of the genetic diversity of cryopreserved collections. Using the example of the French National Cryobank, this study employs newly proposed biodiversity metrics to conduct a detailed assessment of the status of collections for six livestock species: cattle, sheep, goat, horse, donkey, and pig.

Both the amount of resources available and their characteristics varied among species and/or breeds. Overall, breeds with a large commercial distribution had more donors in the collection than local breeds, while the number of doses available was mainly determined by the physiology of the species. An adapted version of the Gini-Simpson equitability index revealed an unbalanced number of donors between breeds for some species. Similarly, estimates of effective donor numbers (De) highlighted the unequal distribution of donors within a breed. Finally, we developed a new index of diversity impact (IDI) to assess the potential of a collection to reintroduce genetic diversity in contemporary populations. The IDI was calculated on the basis of pedigree data for 17 breeds of three livestock species, pig, sheep, and cattle, which differed in both population size and management program. IDI values are negative when the use of cryoconserved sires would decrease the overall kinship of the current population and positive when it would increase it, enabling the most interesting donors to be chosen for immediate use. Negative (favorable) IDI values 41 were found for both local breeds as well as for commercial populations. In general, older collections exhibited better IDI values but recently collected donors could also be useful for populations undergoing strong selection. Within a breed, IDI can be computed individually and thus be used to select the best sires for a given objective. In the absence of pedigree data, IDI values could also be calculated on the basis of marker genotypes.

Overall, this study proposes a framework for the assessment of germplasm collections in the service of various objectives. Compared to FAO indicators motivated by breed reconstitution, the Gini-Simpson and De indices can help to plan sampling more efficiently, whereas IDI can guide donor selection in order to manage the diversity of existing populations. These indicators can be calculated at regular intervals to support the planning and management of collections at national and international levels and help population managers to exploit the resources currently available.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577396">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577396" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577396">
        <p class="paperTitle">Impact of inter-species hybridisation on antifungal drug response in the Saccharomyces genus.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577396" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577396" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Visinoni, F.; Royle, W.; Scholey, R.; Hu, Y.; Timouma, S.; Zeef, L.; Louis, E. J.; Delneri, D.</p>
        <p class="info">Score: 2.9, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577396' target='https://doi.org/10.1101/2024.01.29.577396'> 10.1101/2024.01.29.577396</a></p>
        <p class="abstract">Antifungal drug resistance across fungal and yeast pathogens presents one of the major concerns for global public health. Understanding the interactions between genetic background and environment is important for the development of new, effective treatments of infections. Allelic variation within populations of Ascomycota as well as hybridisation impacts the phenotype in response to stressful conditions, including to antifungal drugs. We exploited recent advances in multigenerational breeding of Saccharomyces interspecies hybrids to study the impact of hybridisation on antifungal resistance and identify quantitative trait loci (QTL) responsible for the phenotypes observed. A library of Saccharomyces cerevisiae x S. kudriavzevii hybrid offspring was screened in the presence of sub-lethal concentrations of six antifungal drugs and revealed a broad phenotypic diversity across the progeny. QTL analysis was carried out comparing alleles between the pools of high and low fitness offspring, identifying hybrid-specific genetic regions involved in resistance to fluconazole, micafungin and flucytosine. We found both drug specific and pleiotropic regions, and through gene ontology and SIFT analysis we identify potential causal genes, such as BCK2 and DNF1 that were validated via reciprocal hemizygosity analysis. We highlight 41 regions that contain genes not previously associated with resistance phenotypes in the literature. The results of this screening will help identify new pathways contributing to drug resistance, and lead to greater understanding of how allelic variation, hybridisation and evolution affect antifungal drug resistance in yeast and fungi.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
