<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Admix-kit: An Integrated Toolkit and Pipeline for Genetic Analyses of Admixed Populations
Authors: Hou, K.; Gogarten, S.; Kim, J.; Hua, X.; Dias, J.-A.; Sun, Q.; Wang, Y.; Tan, T.; Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium Methods Working Group, ; Atkinson, E. G.; Martin, A.; Shortt, J.; Hirbo, J.; Li, Y.; Pasaniuc, B.; Zhang, H.
Score: 16.9, Published: 2023-10-02 DOI: 10.1101/2023.09.30.560263
SummaryAdmixed populations, with their unique and diverse genetic backgrounds, are often underrepresented in genetic studies.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Admix-kit: An Integrated Toolkit and Pipeline for Genetic Analyses of Admixed Populations
Authors: Hou, K.; Gogarten, S.; Kim, J.; Hua, X.; Dias, J.-A.; Sun, Q.; Wang, Y.; Tan, T.; Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium Methods Working Group, ; Atkinson, E. G.; Martin, A.; Shortt, J.; Hirbo, J.; Li, Y.; Pasaniuc, B.; Zhang, H.
Score: 16.9, Published: 2023-10-02 DOI: 10.1101/2023.09.30.560263
SummaryAdmixed populations, with their unique and diverse genetic backgrounds, are often underrepresented in genetic studies." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-11T10:39:46+00:00" />
<meta property="article:modified_time" content="2023-10-11T10:39:46+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Admix-kit: An Integrated Toolkit and Pipeline for Genetic Analyses of Admixed Populations
Authors: Hou, K.; Gogarten, S.; Kim, J.; Hua, X.; Dias, J.-A.; Sun, Q.; Wang, Y.; Tan, T.; Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium Methods Working Group, ; Atkinson, E. G.; Martin, A.; Shortt, J.; Hirbo, J.; Li, Y.; Pasaniuc, B.; Zhang, H.
Score: 16.9, Published: 2023-10-02 DOI: 10.1101/2023.09.30.560263
SummaryAdmixed populations, with their unique and diverse genetic backgrounds, are often underrepresented in genetic studies."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Admix-kit: An Integrated Toolkit and Pipeline for Genetic Analyses of Admixed Populations\nAuthors: Hou, K.; Gogarten, S.; Kim, J.; Hua, X.; Dias, J.-A.; Sun, Q.; Wang, Y.; Tan, T.; Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium Methods Working Group, ; Atkinson, E. G.; Martin, A.; Shortt, J.; Hirbo, J.; Li, Y.; Pasaniuc, B.; Zhang, H.\nScore: 16.9, Published: 2023-10-02 DOI: 10.1101/2023.09.30.560263\nSummaryAdmixed populations, with their unique and diverse genetic backgrounds, are often underrepresented in genetic studies.",
  "keywords": [
    
  ],
  "articleBody": " Admix-kit: An Integrated Toolkit and Pipeline for Genetic Analyses of Admixed Populations\nAuthors: Hou, K.; Gogarten, S.; Kim, J.; Hua, X.; Dias, J.-A.; Sun, Q.; Wang, Y.; Tan, T.; Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium Methods Working Group, ; Atkinson, E. G.; Martin, A.; Shortt, J.; Hirbo, J.; Li, Y.; Pasaniuc, B.; Zhang, H.\nScore: 16.9, Published: 2023-10-02 DOI: 10.1101/2023.09.30.560263\nSummaryAdmixed populations, with their unique and diverse genetic backgrounds, are often underrepresented in genetic studies. This oversight not only limits our understanding but also exacerbates existing health disparities. One major barrier has been the lack of efficient tools tailored for the special challenges of genetic study of admixed populations. Here, we present admix-kit, an integrated toolkit and pipeline for genetic analyses of admixed populations. Admix-kit implements a suite of methods to facilitate genotype and phenotype simulation, association testing, genetic architecture inference, and polygenic scoring in admixed populations. Availability and implementationAdmix-kit package is open-source and available at https://github.com/KangchengHou/admix-kit. Additionally, users can use the pipeline designed for admixed genotype simulation available at https://github.com/UW-GAC/admix-kit_workflow.\nThe DNA helicase FANCJ (BRIP1) functions in Double Strand Break repair processing, but not crossover formation during Prophase I of meiosis in male mice\nAuthors: Horan, T. S.; Ascencao, C. F. R.; Mellor, C. A.; Wang, M.; Smolka, M. B.; Cohen, P. E.\nScore: 12.2, Published: 2023-10-08 DOI: 10.1101/2023.10.06.561296\nDuring meiotic prophase I, recombination between homologous parental chromosomes is initiated by the formation of hundreds of programmed double-strand breaks (DSBs), each of which must be repaired with absolute fidelity to ensure genome stability of the germline. One outcome of these DSB events is the formation of Crossovers (COs), the sites of physical DNA exchange between homologs that are critical to ensure the correct segregation of parental chromosomes. However, COs account for only a small (~10%) proportion of all DSB repair events; the remaining 90% are repaired as non-crossovers (NCOs), most by synthesis dependent strand annealing. Virtually all COs are formed by coordinated efforts of the MSH4/MSH5 and MLH1/MLH3 heterodimers. The number and positioning of COs is exquisitely controlled via mechanisms that remain poorly understood, but which undoubtedly require the coordinated action of multiple repair pathways downstream of the initiating DSB. In a previous report we found evidence suggesting that the DNA helicase and Fanconi Anemia repair protein, FANCJ (BRIP1/BACH1), functions to regulate meiotic recombination in mouse. A gene-trap disruption of Fancj showed an elevated number of MLH1 foci and COs. FANCJ is known to interact with numerous DNA repair proteins in somatic cell repair contexts, including MLH1, BLM, BRCA1, and TOPBP1, and we hypothesized that FANCJ regulates CO formation through a direct interaction with MLH1 to suppress the major CO pathway. To further elucidate the function of FANCJ in meiosis, we produced three new Fancj mutant mouse lines via CRISPR/Cas9 gene editing: a full-gene deletion, a mutant line lacking the MLH1 interaction site and the N-terminal region of the Helicase domain, and a C-terminal 6xHIS-HA dual-tagged allele of Fancj. We also generated an antibody against the C-terminus of the mouse FANCJ protein. Surprisingly, while Fanconi-like phenotypes are observed within the somatic cell lineages of the full deletion Fancj line, none of the Fancj mutants show any change in either MLH1 focus counts during pachynema or total CO number at diakinesis of prophase I of meiosis. We find evidence that FANCJ and MLH1 do not interact in meiosis; further, FANCJ does not co-localize with MSH4, MLH1, or MLH3 during late prophase I. Instead, FANCJ forms discrete foci along the chromosome cores beginning in early meiotic prophase I, occasionally co-localizing with MSH4, and then becomes densely localized on unsynapsed chromosome axes in late zygonema and to the XY chromosomes in early pachynema. Strikingly, this localization strongly overlaps with BRCA1 and TOPBP1. Fancj mutants also exhibit a subtle persistence of DSBs in pachynema. Collectively, these data suggest a role for FANCJ in early DSB repair events, and possibly in the formation of NCOs, but they rule out a role for FANCJ in MLH1-mediated CO events. Thus, the role of FANCJ in meiotic cells involves different pathways and different interactors to those described in somatic cell lineages.\nThe overlapping genetic architecture of psychiatric disorders and cortical brain structure\nAuthors: Sha, Z.; Warrier, V.; Bethlehem, R. A. I.; Schultz, L. M.; Merikangas, A. K.; Sun, K. Y.; Gur, R. C.; Gur, R. E.; Shinohara, R. T.; Seidlitz, J.; Almasy, L.; Andreassen, O.; Alexander-Bloch, A.\nScore: 11.4, Published: 2023-10-05 DOI: 10.1101/2023.10.05.561040\nBoth psychiatric vulnerability and cortical structure are shaped by the cumulative effect of common genetic variants across the genome. However, the shared genetic underpinnings between psychiatric disorders and brain structural phenotypes, such as thickness and surface area of the cerebral cortex, remains elusive. In this study, we employed pleiotropy-informed conjunctional false discovery rate analysis to investigate shared loci across genome-wide association scans of regional cortical thickness, surface area, and seven psychiatric disorders in approximately 700,000 individuals of European ancestry. Aggregating regional measures, we identified 50 genetic loci shared between psychiatric disorders and surface area, as well as 26 genetic loci shared with cortical thickness. Risk alleles exhibited bidirectional effects on both cortical thickness and surface area, such that some risk alleles for each disorder increased regional brain size while other risk alleles decreased regional brain size. Due to bidirectional effects, in many cases we observed extensive pleiotropy between an imaging phenotype and a psychiatric disorder even in the absence of a significant genetic correlation between them. The impact of genetic risk for psychiatric disorders on regional brain structure did exhibit a consistent pattern across highly comorbid psychiatric disorders, with 80% of the genetic loci shared across multiple disorders displaying consistent directions of effect. Cortical patterning of genetic overlap revealed a hierarchical genetic architecture, with the association cortex and sensorimotor cortex representing two extremes of shared genetic influence on psychiatric disorders and brain structural variation. Integrating multi-scale functional annotations and transcriptomic profiles, we observed that shared genetic loci were enriched in active genomic regions, converged on neurobiological and metabolic pathways, and showed differential expression in postmortem brain tissue from individuals with psychiatric disorders. Cumulatively, these findings provide a significant advance in our understanding of the overlapping polygenic architecture between psychopathology and cortical brain structure.\nNAMERS: a purpose-built reference DNA sequence database to support applied eDNA metabarcoding\nAuthors: Westfall, K. M.; Singer, G. A. C.; Kaushal, M.; Gilmore, S. R.; Fahner, N.; Hajibabaei, M.; Abbott, C. L.\nScore: 9.5, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561210\nApplied eDNA metabarcoding is increasingly being used to generate actionable results to inform management decisions, regulations, or policy development. Because of these important downstream considerations, optimizing workflow elements is now essential to increasing standardization, efficiency, and confidence of metabarcoding results. Reference DNA sequences are critical workflow elements that currently lack consistent approaches to generating, curating, or publishing. Here we present a complete (mitochondrial genome and nuclear ribosomal DNA cistron) and high quality reference DNA sequence library for the freshwater fishes of British Columbia, Canada. This resource is published as the Novel Applied eDNA Metabarcoding Reference Sequences (NAMERS) repository (https://namers.ca), a user-friendly and interactive website for specialists and non-specialists alike to explore and generate custom reference libraries for taxa and genes of interest. We demonstrate the power of NAMERS to optimize applied eDNA metabarcoding workflows at the study design stage by analyzing the number of primer mismatches and resolution power of existing metabarcoding markers. To meet the increasing demand for actionable eDNA metabarcoding applications, NAMERS demonstrates that high quality curated genomic information is within a reasonable reach. It is timely to establish this framework as the new gold standard and coordinate our efforts to generate this type of reference data at scale.\nBronze Age Northern Eurasian Genetics in the Context of Development of Metallurgy and Siberian Ancestry\nAuthors: Childebayeva, A.; Fricke, F.; Rohrlach, A. B.; Huang, L.; Schiffels, S.; Vesakoski, O.; Semerau, L.; Aron, F.; Moiseyev, V.; Khartanovich, V.; Kovtun, I.; Krause, J.; Kuzminykh, S.; Haak, W.\nScore: 19.7, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560195\nThe Eurasian Bronze Age (BA) has been described as a period of substantial human migrations, the emergence of pastoralism, horse domestication, and development of metallurgy. This study focuses on individuals associated with BA metallurgical production, specifically the Seima-Turbino (ST) phenomenon ([~]2,200-1,900 BCE) associated with elaborate metal objects found across Northern Eurasia. The genetic profiles of nine ST-associated individuals vary widely ranging between ancestries maximized in individuals from the Eastern Siberian Late Neolithic/BA, and those of the Western Steppe Middle Late BA. The genetic heterogeneity observed is consistent with the current understanding of the ST metallurgical network as a transcultural phenomenon. The new data also shed light on the temporal and spatial range of an ancient Siberian genetic ancestry component, which is shared across many Uralic-speaking populations, and which we explore further via demographic modeling using additional genome-wide (2 individuals) and whole genome data (5 individuals, including a [~]30x genome) from northwestern Russia.\nImprinting but not cytonuclear interactions affects parent-of-origin effect on seed size in Arabidopsis hybrids\nAuthors: June, V.; Song, X.; Chen, Z. J.\nScore: 8.2, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557997\nThe parent-of-origin effect on seed size can result from imprinting or a combinational effect between cytoplasmic and nuclear genomes, but their relative contributions remain unknown. To discern these confounding effects, we generated cytoplasmic-nuclear substitution (CNS) lines using recurrent backcrossing in the Arabidopsis thaliana ecotypes Col-0 and C24. These CNS lines differ only in the nuclear genome (imprinting) or in the cytoplasm. The CNS reciprocal hybrids with the same cytoplasm display a [~]20% seed size difference as observed in the conventional hybrids. However, seed size is similar between the reciprocal cybrids with fixed imprinting. Transcriptome analyses in the endosperm of CNS hybrids using laser-capture microdissection have identified 104 maternally expressed genes (MEGs) and 90 paternally-expressed genes (PEGs). These imprinted genes are involved in pectin catabolism and cell wall modification in the endosperm. HDG9, an epiallele and one of 11 cross-specific imprinted genes, controls seed size. In the embryo, a handful of imprinted genes is found in the CNS hybrids but only one is expressed higher in the embryo than endosperm. AT4G13495 encodes a long-noncoding RNA (lncRNA), but no obvious seed phenotype is observed in the lncRNA knockout lines. NRPD1, encoding the largest subunit of RNA Pol IV, is involved in the biogenesis of small interfering RNAs. Seed size and embryo is larger in the cross using nrpd1 as the maternal parent than in the reciprocal cross. In spite of limited ecotypes tested, these results suggest potential roles of imprinting and NRPD1-mediated small RNA pathway in seed size variation in hybrids.\nHLA-Based Banking of Human Induced Pluripotent Stem Cells in Saudi Arabia\nAuthors: Alowaysi, M.; Lehmann, R.; Al-Shehri, M.; Baadheim, M.; Alzahrani, H.; Aboalola, D.; Zia, A.; Malibari, D.; Daghestani, M.; Alghamdi, K.; Haneef, A.; Jawdat, D.; Hakami, F.; Cabrero, D. G.; Tegner, J.; Alsayegh, K.\nScore: 8.8, Published: 2023-09-17 DOI: 10.1101/2023.09.16.557826\nHuman iPSCs derivation and use in clinical studies are transforming medicine. Yet, there is a high cost and long waiting time for autologous iPS-based cellular therapy, and the genetic engineering of hypo-immunogenic iPS cell lines is hampered with numerous hurdles. Therefore, it is increasingly interesting to create cell stocks based on HLA haplotype distribution in a given population. In this study, we assessed the potential of HLA-based iPS banking for the Saudi population. First, we analyzed the HLA database of the Saudi Stem Cell Donor Registry (SSCDR), which contains high-resolution HLA genotype data of 64,315 registered Saudi donors at the time of analysis. We found that only 13 iPS lines would be required to cover 30% of the Saudi population, 39 iPS lines would offer 50% coverage and 596 for more than 90% coverage. Next, As a proof-of-concept, we launched the first HLA-based banking of iPSCs in Saudi Arabia. Using clinically relevant methods, we generated the first iPSC line from a homozygous donor for the most common HLA haplotype in Saudi. The two generated clones expressed pluripotency markers, could be differentiated into all three germ layers, beating cardiomyocytes and neuronal progenitors. To ensure that our reprogramming method generates genetically stable iPSCs, we assessed the mutational burden in the generated clones and the original blood sample from which the iPSCs were derived using whole-genome sequencing. All detected variants were found in the original donor sample and were classified as benign according to current guidelines of the American College of Medical Genetics and Genomics (ACMG). This study sets a road map for introducing iPS-based cell therapy in the Kingdom of Saudi Arabia.\nSynchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition\nAuthors: Vollger, M. R.; Korlach, J.; Eldred, K. C.; Swanson, E.; Underwood, J. G.; Munson, K. M.; Cheng, Y.-H. H.; Ranchalis, J.; Mao, Y.; Blue, E. E.; Schwarze, U.; Saunders, C. T.; Wenger, A. M.; Allworth, A.; Chanprasert, S.; Duerden, B. L.; Glass, I.; Horike-Pyne, M.; Kim, M.; Leppig, K. A.; McLaughlin, I. J.; Ogawa, J.; Rosenthal, E. A.; Sheppeard, S.; Sherman, S. M.; Strohbehn, S.; Yuen, A. L.; Reh, T. A.; Byers, P. H.; Bamshad, M. J.; Hisama, F. M.; Jarvik, G. P.; Sancak, Y.; Dipple, K. M.; Stergachis, A. B.\nScore: 38.3, Published: 2023-09-27 DOI: 10.1101/2023.09.26.559521\nResolving the molecular basis of a Mendelian condition (MC) remains challenging owing to the diverse mechanisms by which genetic variants cause disease. To address this, we developed a synchronized long-read genome, methylome, epigenome, and transcriptome sequencing approach, which enables accurate single-nucleotide, insertion-deletion, and structural variant calling and diploid de novo genome assembly, and permits the simultaneous elucidation of haplotype-resolved CpG methylation, chromatin accessibility, and full-length transcript information in a single long-read sequencing run. Application of this approach to an Undiagnosed Diseases Network (UDN) participant with a chromosome X;13 balanced translocation of uncertain significance revealed that this translocation disrupted the functioning of four separate genes (NBEA, PDK3, MAB21L1, and RB1) previously associated with single-gene MCs. Notably, the function of each gene was disrupted via a distinct mechanism that required integration of the four omes to resolve. These included nonsense-mediated decay, fusion transcript formation, enhancer adoption, transcriptional readthrough silencing, and inappropriate X chromosome inactivation of autosomal genes. Overall, this highlights the utility of synchronized long-read multi-omic profiling for mechanistically resolving complex phenotypes.\nSingle-cell chromatin state transitions during epigenetic memory formation\nAuthors: Fujimori, T.; Rios-Martinez, C.; Thurm, A. R.; Hinks, M. M.; Doughty, B. R.; Sinha, J.; Le, D.; Hafner, A.; Greenleaf, W. J.; Boettiger, A. N.; Bintu, L.\nScore: 4.5, Published: 2023-10-05 DOI: 10.1101/2023.10.03.560616\nRepressive chromatin modifications are thought to compact chromatin to silence transcription. However, it is unclear how chromatin structure changes during silencing and epigenetic memory formation. We measured gene expression and chromatin structure in single cells after recruitment and release of repressors at a reporter gene. Chromatin structure is heterogeneous, with open and compact conformations present in both active and silent states. Recruitment of repressors associated with epigenetic memory produces chromatin compaction across 10-20 kilobases, while reversible silencing does not cause compaction at this scale. Chromatin compaction is inherited, but changes molecularly over time from histone methylation (H3K9me3) to DNA methylation. The level of compaction at the end of silencing quantitatively predicts epigenetic memory weeks later. Similarly, chromatin compaction at the Nanog locus predicts the degree of stem-cell fate commitment. These findings suggest that the chromatin state across tens of kilobases, beyond the gene itself, is important for epigenetic memory formation.\nFunctional coherence of the Xist and RSX protein interactomes: X chromosome inactivation marsupials\nAuthors: McIntyre, K. L.; Waters, S.; Zhong, L.; Hart-Smith, G.; Raftery, M.; Graves, J. A. M.; Waters, P. D.\nScore: 3.1, Published: 2023-10-08 DOI: 10.1101/2023.10.08.560959\nLong range epigenetic silencing is epitomised by X chromosome inactivation (XCI) in mammals. It is mediated by independently evolved, non-homologous long noncoding RNAs in eutherian and marsupial mammals (XIST and RSX). The Xist interactome, comprising proteins that mediate the silencing process, is well documented in mouse studies. Here we interrogate proteins that interact with RSX using chromatin isolation by RNA purification followed by mass spectrometry (ChIRP-MS) in a marsupial representative, Monodelphis domestica. We identify 135 proteins that interact with RSX, of which 56 have orthologues in the Xist interactome. Remarkably, nearly 90% of the combined Xist and RSX interactomes are within the same protein-protein association network. This network clustered into three major groups with distinctive functional enrichments, including RNA splicing and processing, regulation of translation and ribosomal biogenesis, and epigenetic transcriptional silencing. The proteins of the RSX interactome were enriched for regions of intrinsic disorder in common with the Xist interactome, identifying this as a feature of ribonucleoprotein complexes associated with XCI. We also show that RNAi knockdown of representative RSX interactors, HNRNPK and CKAP4, led to reactivation of transcription from the inactive X chromosome, indicating a role for each in marsupial XCI. Thus, despite the absence of linear sequence homology between Xist and RSX, they exhibit extraordinary functional coherence that indicates potential for post-transcriptional regulation, a feature not previously associated with the molecular machinery of XCI.\n",
  "wordCount" : "2808",
  "inLanguage": "en",
  "datePublished": "2023-10-11T10:39:46Z",
  "dateModified": "2023-10-11T10:39:46Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on October 11, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560263">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560263" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560263">
        <p class="paperTitle">Admix-kit: An Integrated Toolkit and Pipeline for Genetic Analyses of Admixed Populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560263" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560263" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hou, K.; Gogarten, S.; Kim, J.; Hua, X.; Dias, J.-A.; Sun, Q.; Wang, Y.; Tan, T.; Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium Methods Working Group,  ; Atkinson, E. G.; Martin, A.; Shortt, J.; Hirbo, J.; Li, Y.; Pasaniuc, B.; Zhang, H.</p>
        <p class="info">Score: 16.9, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560263' target='https://doi.org/10.1101/2023.09.30.560263'> 10.1101/2023.09.30.560263</a></p>
        <p class="abstract">SummaryAdmixed populations, with their unique and diverse genetic backgrounds, are often underrepresented in genetic studies. This oversight not only limits our understanding but also exacerbates existing health disparities. One major barrier has been the lack of efficient tools tailored for the special challenges of genetic study of admixed populations. Here, we present admix-kit, an integrated toolkit and pipeline for genetic analyses of admixed populations. Admix-kit implements a suite of methods to facilitate genotype and phenotype simulation, association testing, genetic architecture inference, and polygenic scoring in admixed populations.

Availability and implementationAdmix-kit package is open-source and available at https://github.com/KangchengHou/admix-kit. Additionally, users can use the pipeline designed for admixed genotype simulation available at https://github.com/UW-GAC/admix-kit_workflow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561296">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561296" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561296">
        <p class="paperTitle">The DNA helicase FANCJ (BRIP1) functions in Double Strand Break repair processing, but not crossover formation during Prophase I of meiosis in male mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561296" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561296" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Horan, T. S.; Ascencao, C. F. R.; Mellor, C. A.; Wang, M.; Smolka, M. B.; Cohen, P. E.</p>
        <p class="info">Score: 12.2, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561296' target='https://doi.org/10.1101/2023.10.06.561296'> 10.1101/2023.10.06.561296</a></p>
        <p class="abstract">During meiotic prophase I, recombination between homologous parental chromosomes is initiated by the formation of hundreds of programmed double-strand breaks (DSBs), each of which must be repaired with absolute fidelity to ensure genome stability of the germline. One outcome of these DSB events is the formation of Crossovers (COs), the sites of physical DNA exchange between homologs that are critical to ensure the correct segregation of parental chromosomes. However, COs account for only a small (~10%) proportion of all DSB repair events; the remaining 90% are repaired as non-crossovers (NCOs), most by synthesis dependent strand annealing. Virtually all COs are formed by coordinated efforts of the MSH4/MSH5 and MLH1/MLH3 heterodimers. The number and positioning of COs is exquisitely controlled via mechanisms that remain poorly understood, but which undoubtedly require the coordinated action of multiple repair pathways downstream of the initiating DSB. In a previous report we found evidence suggesting that the DNA helicase and Fanconi Anemia repair protein, FANCJ (BRIP1/BACH1), functions to regulate meiotic recombination in mouse. A gene-trap disruption of Fancj showed an elevated number of MLH1 foci and COs. FANCJ is known to interact with numerous DNA repair proteins in somatic cell repair contexts, including MLH1, BLM, BRCA1, and TOPBP1, and we hypothesized that FANCJ regulates CO formation through a direct interaction with MLH1 to suppress the major CO pathway. To further elucidate the function of FANCJ in meiosis, we produced three new Fancj mutant mouse lines via CRISPR/Cas9 gene editing: a full-gene deletion, a mutant line lacking the MLH1 interaction site and the N-terminal region of the Helicase domain, and a C-terminal 6xHIS-HA dual-tagged allele of Fancj. We also generated an antibody against the C-terminus of the mouse FANCJ protein. Surprisingly, while Fanconi-like phenotypes are observed within the somatic cell lineages of the full deletion Fancj line, none of the Fancj mutants show any change in either MLH1 focus counts during pachynema or total CO number at diakinesis of prophase I of meiosis. We find evidence that FANCJ and MLH1 do not interact in meiosis; further, FANCJ does not co-localize with MSH4, MLH1, or MLH3 during late prophase I. Instead, FANCJ forms discrete foci along the chromosome cores beginning in early meiotic prophase I, occasionally co-localizing with MSH4, and then becomes densely localized on unsynapsed chromosome axes in late zygonema and to the XY chromosomes in early pachynema. Strikingly, this localization strongly overlaps with BRCA1 and TOPBP1. Fancj mutants also exhibit a subtle persistence of DSBs in pachynema. Collectively, these data suggest a role for FANCJ in early DSB repair events, and possibly in the formation of NCOs, but they rule out a role for FANCJ in MLH1-mediated CO events. Thus, the role of FANCJ in meiotic cells involves different pathways and different interactors to those described in somatic cell lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561040">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561040" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561040">
        <p class="paperTitle">The overlapping genetic architecture of psychiatric disorders and cortical brain structure</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561040" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561040" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sha, Z.; Warrier, V.; Bethlehem, R. A. I.; Schultz, L. M.; Merikangas, A. K.; Sun, K. Y.; Gur, R. C.; Gur, R. E.; Shinohara, R. T.; Seidlitz, J.; Almasy, L.; Andreassen, O.; Alexander-Bloch, A.</p>
        <p class="info">Score: 11.4, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561040' target='https://doi.org/10.1101/2023.10.05.561040'> 10.1101/2023.10.05.561040</a></p>
        <p class="abstract">Both psychiatric vulnerability and cortical structure are shaped by the cumulative effect of common genetic variants across the genome. However, the shared genetic underpinnings between psychiatric disorders and brain structural phenotypes, such as thickness and surface area of the cerebral cortex, remains elusive. In this study, we employed pleiotropy-informed conjunctional false discovery rate analysis to investigate shared loci across genome-wide association scans of regional cortical thickness, surface area, and seven psychiatric disorders in approximately 700,000 individuals of European ancestry. Aggregating regional measures, we identified 50 genetic loci shared between psychiatric disorders and surface area, as well as 26 genetic loci shared with cortical thickness. Risk alleles exhibited bidirectional effects on both cortical thickness and surface area, such that some risk alleles for each disorder increased regional brain size while other risk alleles decreased regional brain size. Due to bidirectional effects, in many cases we observed extensive pleiotropy between an imaging phenotype and a psychiatric disorder even in the absence of a significant genetic correlation between them. The impact of genetic risk for psychiatric disorders on regional brain structure did exhibit a consistent pattern across highly comorbid psychiatric disorders, with 80% of the genetic loci shared across multiple disorders displaying consistent directions of effect. Cortical patterning of genetic overlap revealed a hierarchical genetic architecture, with the association cortex and sensorimotor cortex representing two extremes of shared genetic influence on psychiatric disorders and brain structural variation. Integrating multi-scale functional annotations and transcriptomic profiles, we observed that shared genetic loci were enriched in active genomic regions, converged on neurobiological and metabolic pathways, and showed differential expression in postmortem brain tissue from individuals with psychiatric disorders. Cumulatively, these findings provide a significant advance in our understanding of the overlapping polygenic architecture between psychopathology and cortical brain structure.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561210">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561210" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561210">
        <p class="paperTitle">NAMERS: a purpose-built reference DNA sequence database to support applied eDNA metabarcoding</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561210" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561210" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Westfall, K. M.; Singer, G. A. C.; Kaushal, M.; Gilmore, S. R.; Fahner, N.; Hajibabaei, M.; Abbott, C. L.</p>
        <p class="info">Score: 9.5, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561210' target='https://doi.org/10.1101/2023.10.06.561210'> 10.1101/2023.10.06.561210</a></p>
        <p class="abstract">Applied eDNA metabarcoding is increasingly being used to generate actionable results to inform management decisions, regulations, or policy development. Because of these important downstream considerations, optimizing workflow elements is now essential to increasing standardization, efficiency, and confidence of metabarcoding results. Reference DNA sequences are critical workflow elements that currently lack consistent approaches to generating, curating, or publishing. Here we present a complete (mitochondrial genome and nuclear ribosomal DNA cistron) and high quality reference DNA sequence library for the freshwater fishes of British Columbia, Canada. This resource is published as the Novel Applied eDNA Metabarcoding Reference Sequences (NAMERS) repository (https://namers.ca), a user-friendly and interactive website for specialists and non-specialists alike to explore and generate custom reference libraries for taxa and genes of interest. We demonstrate the power of NAMERS to optimize applied eDNA metabarcoding workflows at the study design stage by analyzing the number of primer mismatches and resolution power of existing metabarcoding markers. To meet the increasing demand for actionable eDNA metabarcoding applications, NAMERS demonstrates that high quality curated genomic information is within a reasonable reach. It is timely to establish this framework as the new gold standard and coordinate our efforts to generate this type of reference data at scale.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560195">
        <p class="paperTitle">Bronze Age Northern Eurasian Genetics in the Context of Development of Metallurgy and Siberian Ancestry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Childebayeva, A.; Fricke, F.; Rohrlach, A. B.; Huang, L.; Schiffels, S.; Vesakoski, O.; Semerau, L.; Aron, F.; Moiseyev, V.; Khartanovich, V.; Kovtun, I.; Krause, J.; Kuzminykh, S.; Haak, W.</p>
        <p class="info">Score: 19.7, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560195' target='https://doi.org/10.1101/2023.10.01.560195'> 10.1101/2023.10.01.560195</a></p>
        <p class="abstract">The Eurasian Bronze Age (BA) has been described as a period of substantial human migrations, the emergence of pastoralism, horse domestication, and development of metallurgy. This study focuses on individuals associated with BA metallurgical production, specifically the Seima-Turbino (ST) phenomenon ([~]2,200-1,900 BCE) associated with elaborate metal objects found across Northern Eurasia. The genetic profiles of nine ST-associated individuals vary widely ranging between ancestries maximized in individuals from the Eastern Siberian Late Neolithic/BA, and those of the Western Steppe Middle Late BA. The genetic heterogeneity observed is consistent with the current understanding of the ST metallurgical network as a transcultural phenomenon. The new data also shed light on the temporal and spatial range of an ancient Siberian genetic ancestry component, which is shared across many Uralic-speaking populations, and which we explore further via demographic modeling using additional genome-wide (2 individuals) and whole genome data (5 individuals, including a [~]30x genome) from northwestern Russia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557997">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557997" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557997">
        <p class="paperTitle">Imprinting but not cytonuclear interactions affects parent-of-origin effect on seed size in Arabidopsis hybrids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557997" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557997" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: June, V.; Song, X.; Chen, Z. J.</p>
        <p class="info">Score: 8.2, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557997' target='https://doi.org/10.1101/2023.09.15.557997'> 10.1101/2023.09.15.557997</a></p>
        <p class="abstract">The parent-of-origin effect on seed size can result from imprinting or a combinational effect between cytoplasmic and nuclear genomes, but their relative contributions remain unknown. To discern these confounding effects, we generated cytoplasmic-nuclear substitution (CNS) lines using recurrent backcrossing in the Arabidopsis thaliana ecotypes Col-0 and C24. These CNS lines differ only in the nuclear genome (imprinting) or in the cytoplasm. The CNS reciprocal hybrids with the same cytoplasm display a [~]20% seed size difference as observed in the conventional hybrids. However, seed size is similar between the reciprocal cybrids with fixed imprinting. Transcriptome analyses in the endosperm of CNS hybrids using laser-capture microdissection have identified 104 maternally expressed genes (MEGs) and 90 paternally-expressed genes (PEGs). These imprinted genes are involved in pectin catabolism and cell wall modification in the endosperm. HDG9, an epiallele and one of 11 cross-specific imprinted genes, controls seed size. In the embryo, a handful of imprinted genes is found in the CNS hybrids but only one is expressed higher in the embryo than endosperm. AT4G13495 encodes a long-noncoding RNA (lncRNA), but no obvious seed phenotype is observed in the lncRNA knockout lines. NRPD1, encoding the largest subunit of RNA Pol IV, is involved in the biogenesis of small interfering RNAs. Seed size and embryo is larger in the cross using nrpd1 as the maternal parent than in the reciprocal cross. In spite of limited ecotypes tested, these results suggest potential roles of imprinting and NRPD1-mediated small RNA pathway in seed size variation in hybrids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.16.557826">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.16.557826" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.16.557826">
        <p class="paperTitle">HLA-Based Banking of Human Induced Pluripotent Stem Cells in Saudi Arabia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.16.557826" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.16.557826" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alowaysi, M.; Lehmann, R.; Al-Shehri, M.; Baadheim, M.; Alzahrani, H.; Aboalola, D.; Zia, A.; Malibari, D.; Daghestani, M.; Alghamdi, K.; Haneef, A.; Jawdat, D.; Hakami, F.; Cabrero, D. G.; Tegner, J.; Alsayegh, K.</p>
        <p class="info">Score: 8.8, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.16.557826' target='https://doi.org/10.1101/2023.09.16.557826'> 10.1101/2023.09.16.557826</a></p>
        <p class="abstract">Human iPSCs derivation and use in clinical studies are transforming medicine. Yet, there is a high cost and long waiting time for autologous iPS-based cellular therapy, and the genetic engineering of hypo-immunogenic iPS cell lines is hampered with numerous hurdles. Therefore, it is increasingly interesting to create cell stocks based on HLA haplotype distribution in a given population. In this study, we assessed the potential of HLA-based iPS banking for the Saudi population. First, we analyzed the HLA database of the Saudi Stem Cell Donor Registry (SSCDR), which contains high-resolution HLA genotype data of 64,315 registered Saudi donors at the time of analysis. We found that only 13 iPS lines would be required to cover 30% of the Saudi population, 39 iPS lines would offer 50% coverage and 596 for more than 90% coverage.

Next, As a proof-of-concept, we launched the first HLA-based banking of iPSCs in Saudi Arabia. Using clinically relevant methods, we generated the first iPSC line from a homozygous donor for the most common HLA haplotype in Saudi. The two generated clones expressed pluripotency markers, could be differentiated into all three germ layers, beating cardiomyocytes and neuronal progenitors. To ensure that our reprogramming method generates genetically stable iPSCs, we assessed the mutational burden in the generated clones and the original blood sample from which the iPSCs were derived using whole-genome sequencing. All detected variants were found in the original donor sample and were classified as benign according to current guidelines of the American College of Medical Genetics and Genomics (ACMG).

This study sets a road map for introducing iPS-based cell therapy in the Kingdom of Saudi Arabia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559521">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559521" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559521">
        <p class="paperTitle">Synchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559521" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559521" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vollger, M. R.; Korlach, J.; Eldred, K. C.; Swanson, E.; Underwood, J. G.; Munson, K. M.; Cheng, Y.-H. H.; Ranchalis, J.; Mao, Y.; Blue, E. E.; Schwarze, U.; Saunders, C. T.; Wenger, A. M.; Allworth, A.; Chanprasert, S.; Duerden, B. L.; Glass, I.; Horike-Pyne, M.; Kim, M.; Leppig, K. A.; McLaughlin, I. J.; Ogawa, J.; Rosenthal, E. A.; Sheppeard, S.; Sherman, S. M.; Strohbehn, S.; Yuen, A. L.; Reh, T. A.; Byers, P. H.; Bamshad, M. J.; Hisama, F. M.; Jarvik, G. P.; Sancak, Y.; Dipple, K. M.; Stergachis, A. B.</p>
        <p class="info">Score: 38.3, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559521' target='https://doi.org/10.1101/2023.09.26.559521'> 10.1101/2023.09.26.559521</a></p>
        <p class="abstract">Resolving the molecular basis of a Mendelian condition (MC) remains challenging owing to the diverse mechanisms by which genetic variants cause disease. To address this, we developed a synchronized long-read genome, methylome, epigenome, and transcriptome sequencing approach, which enables accurate single-nucleotide, insertion-deletion, and structural variant calling and diploid de novo genome assembly, and permits the simultaneous elucidation of haplotype-resolved CpG methylation, chromatin accessibility, and full-length transcript information in a single long-read sequencing run. Application of this approach to an Undiagnosed Diseases Network (UDN) participant with a chromosome X;13 balanced translocation of uncertain significance revealed that this translocation disrupted the functioning of four separate genes (NBEA, PDK3, MAB21L1, and RB1) previously associated with single-gene MCs. Notably, the function of each gene was disrupted via a distinct mechanism that required integration of the four  omes to resolve. These included nonsense-mediated decay, fusion transcript formation, enhancer adoption, transcriptional readthrough silencing, and inappropriate X chromosome inactivation of autosomal genes. Overall, this highlights the utility of synchronized long-read multi-omic profiling for mechanistically resolving complex phenotypes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560616">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560616" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560616">
        <p class="paperTitle">Single-cell chromatin state transitions during epigenetic memory formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560616" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560616" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fujimori, T.; Rios-Martinez, C.; Thurm, A. R.; Hinks, M. M.; Doughty, B. R.; Sinha, J.; Le, D.; Hafner, A.; Greenleaf, W. J.; Boettiger, A. N.; Bintu, L.</p>
        <p class="info">Score: 4.5, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560616' target='https://doi.org/10.1101/2023.10.03.560616'> 10.1101/2023.10.03.560616</a></p>
        <p class="abstract">Repressive chromatin modifications are thought to compact chromatin to silence transcription. However, it is unclear how chromatin structure changes during silencing and epigenetic memory formation. We measured gene expression and chromatin structure in single cells after recruitment and release of repressors at a reporter gene. Chromatin structure is heterogeneous, with open and compact conformations present in both active and silent states. Recruitment of repressors associated with epigenetic memory produces chromatin compaction across 10-20 kilobases, while reversible silencing does not cause compaction at this scale. Chromatin compaction is inherited, but changes molecularly over time from histone methylation (H3K9me3) to DNA methylation. The level of compaction at the end of silencing quantitatively predicts epigenetic memory weeks later. Similarly, chromatin compaction at the Nanog locus predicts the degree of stem-cell fate commitment. These findings suggest that the chromatin state across tens of kilobases, beyond the gene itself, is important for epigenetic memory formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.560959">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.560959" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.560959">
        <p class="paperTitle">Functional coherence of the Xist and RSX protein interactomes: X chromosome inactivation marsupials</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.560959" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.560959" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McIntyre, K. L.; Waters, S.; Zhong, L.; Hart-Smith, G.; Raftery, M.; Graves, J. A. M.; Waters, P. D.</p>
        <p class="info">Score: 3.1, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.560959' target='https://doi.org/10.1101/2023.10.08.560959'> 10.1101/2023.10.08.560959</a></p>
        <p class="abstract">Long range epigenetic silencing is epitomised by X chromosome inactivation (XCI) in mammals. It is mediated by independently evolved, non-homologous long noncoding RNAs in eutherian and marsupial mammals (XIST and RSX). The Xist interactome, comprising proteins that mediate the silencing process, is well documented in mouse studies. Here we interrogate proteins that interact with RSX using chromatin isolation by RNA purification followed by mass spectrometry (ChIRP-MS) in a marsupial representative, Monodelphis domestica. We identify 135 proteins that interact with RSX, of which 56 have orthologues in the Xist interactome. Remarkably, nearly 90% of the combined Xist and RSX interactomes are within the same protein-protein association network. This network clustered into three major groups with distinctive functional enrichments, including RNA splicing and processing, regulation of translation and ribosomal biogenesis, and epigenetic transcriptional silencing. The proteins of the RSX interactome were enriched for regions of intrinsic disorder in common with the Xist interactome, identifying this as a feature of ribonucleoprotein complexes associated with XCI. We also show that RNAi knockdown of representative RSX interactors, HNRNPK and CKAP4, led to reactivation of transcription from the inactive X chromosome, indicating a role for each in marsupial XCI. Thus, despite the absence of linear sequence homology between Xist and RSX, they exhibit extraordinary functional coherence that indicates potential for post-transcriptional regulation, a feature not previously associated with the molecular machinery of XCI.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
