<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The simplicity of protein sequence-function relationships
Authors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.
Score: 29.8, Published: 2023-09-05 DOI: 10.1101/2023.09.02.556057
How complicated is the relationship between a proteins sequence and its function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most prior studies, however, used methods that misinterpret measurement errors, small local idiosyncracies around a designated wild-type sequence, and global nonlinearity in the sequence-function relationship as rampant high-order interactions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="The simplicity of protein sequence-function relationships
Authors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.
Score: 29.8, Published: 2023-09-05 DOI: 10.1101/2023.09.02.556057
How complicated is the relationship between a proteins sequence and its function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most prior studies, however, used methods that misinterpret measurement errors, small local idiosyncracies around a designated wild-type sequence, and global nonlinearity in the sequence-function relationship as rampant high-order interactions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-10T10:36:49+00:00" />
<meta property="article:modified_time" content="2023-09-10T10:36:49+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="The simplicity of protein sequence-function relationships
Authors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.
Score: 29.8, Published: 2023-09-05 DOI: 10.1101/2023.09.02.556057
How complicated is the relationship between a proteins sequence and its function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most prior studies, however, used methods that misinterpret measurement errors, small local idiosyncracies around a designated wild-type sequence, and global nonlinearity in the sequence-function relationship as rampant high-order interactions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "The simplicity of protein sequence-function relationships\nAuthors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.\nScore: 29.8, Published: 2023-09-05 DOI: 10.1101/2023.09.02.556057\nHow complicated is the relationship between a proteins sequence and its function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most prior studies, however, used methods that misinterpret measurement errors, small local idiosyncracies around a designated wild-type sequence, and global nonlinearity in the sequence-function relationship as rampant high-order interactions.",
  "keywords": [
    
  ],
  "articleBody": " The simplicity of protein sequence-function relationships\nAuthors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.\nScore: 29.8, Published: 2023-09-05 DOI: 10.1101/2023.09.02.556057\nHow complicated is the relationship between a proteins sequence and its function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most prior studies, however, used methods that misinterpret measurement errors, small local idiosyncracies around a designated wild-type sequence, and global nonlinearity in the sequence-function relationship as rampant high-order interactions. Here we present a simple new method to jointly estimate global nonlinearity and specific epistatic interactions across a proteins genotype-phenotype map. Our reference-free approach calculates the effect of each amino acid state or combination by averaging over all genotypes that contain it relative to the global average. We show that this method is more accurate than any alternative approach and is robust to measurement error and partial sampling. We reanalyze 20 combinatorial mutagenesis experiments and find that main and pairwise effects, together with a simple form of global nonlinearity, account for a median of 96% of total variance in the measured phenotype (and \u003e 92% in every case), and only a tiny fraction of genotypes are strongly affected by epistasis at third or higher orders. The genetic architecture is also sparse: the number of model terms required to explain the vast majority of phenotypic variance is smaller than the number of genotypes by many orders of magnitude. The sequence-function relationship in most proteins is therefore far simpler than previously thought, and new, more tractable experimental approaches, combined with reference-free analysis, may be sufficient to explain it in most cases. Significance StatementIt is widely thought that a proteins function depends on complex interactions among amino acids. If so, it would be virtually impossible to predict the function of new variants, and understanding how proteins work genetically and biochemically would require huge combinatorial experiments. We show that prior studies overestimated complexity because they analyzed sequence-function relationships from the perspective of a single reference genotype and/or misinterpreted global phenotypic nonlinearities as complex amino acid interactions. By developing a new reference-free approach and using it to reanalyze 20 experimental datasets, we show that additive effects and pairwise interactions alone, along with a simple global nonlinearity, explain the vast majority of functional variation. Higher-order interactions are weak or rare, and a minuscule fraction of possible interactions shape each proteins function. Our work reveals that protein sequence-function relationships are surprisingly simple and suggests new strategies that are far more tractable than the massive experiments currently used.\nFunctional characterization of gene regulatory elements and neuropsychiatric disease-associated risk loci in iPSCs and iPSC-derived neurons\nAuthors: Yang, X.; Jones, I. R.; Chen, P. B.; Yang, H.; Ren, X.; Zheng, L.; Li, B.; Li, Y. E.; Sun, Q.; Wen, J.; Beaman, C.; Cui, X.; Li, Y.; Wang, W.; Hu, M.; Ren, B.; Shen, Y.\nScore: 27.0, Published: 2023-08-30 DOI: 10.1101/2023.08.30.555359\nGenome-wide association studies (GWAS) have identified thousands of non-coding variants that contribute to psychiatric disease risks, likely by perturbing cis-regulatory elements (CREs). However, our ability to interpret and explore their mechanisms of action is hampered by a lack of annotation of functional CREs (fCREs) in neural cell types. Here, through genome-scale CRISPR screens of 22,000 candidate CREs (cCREs) in human induced pluripotent stem cells (iPSCs) undergoing differentiation to excitatory neurons, we identify 2,847 and 5,540 fCREs essential for iPSC fitness and neuronal differentiation, respectively. These fCREs display dynamic epigenomic features and exhibit increased numbers and genomic spans of chromatin interactions following terminal neuronal differentiation. Furthermore, fCREs essential for neuronal differentiation show significantly greater enrichment of genetic heritability for neurodevelopmental diseases including schizophrenia (SCZ), attention deficit hyperactivity disorder (ADHD), and autism spectrum disorders (ASD) than cCREs. Using high-throughput prime editing screens we experimentally confirm 45 SCZ risk variants that act by affecting the function of fCREs. The extensive and in-depth functional annotation of cCREs in neuronal types therefore provides a crucial resource for interpreting non-coding risk variants of neuropsychiatric disorders.\nNatural genetic variation quantitatively regulates heart-rate and -dimension\nAuthors: Gierten, J.; Welz, B.; Fitzgerald, T. W.; Thumberger, T.; Hummel, O.; Leger, A.; Weber, P.; Hassel, D.; Hubner, N.; Birney, E.; Wittbrodt, J.\nScore: 7.9, Published: 2023-09-02 DOI: 10.1101/2023.09.01.555906\nDevelopment and function of the heart are crucially controlled by individual genetic factors determining the continuum from health to disease. A prime example of a quantitative pathological phenotype is left ventricular hypoplasia, affecting the left ventricle (LV) and its associated inflow and outflow structures and reducing its size to varying degrees1. Current models suggest polygenic contributions on heart development and disease, but we are just beginning to understand the impact of naturally occurring sequence variations. Here we show how natural genetic variants exert cumulative, environment-sensitive effects on both embryonic heart development as well as adult heart function. We addressed this in embryos of five inbred strains of the Japanese rice fish medaka (Oryzias latipes2 and Oryzias sakaizumii3), analyzing heart rates as a functional proxy to capture combined characteristics of the conduction system and structural morphology. We identified one strain, HO52, with embryonic hypoplastic ventricles, fast heart rates, and significantly impaired cardiac function and overall fitness in adults. We created an interpopulation cross of the divergent HO5 strain to HdrR, which exhibits a morphologically normal heart and comparably slow heart rates, to leverage segregation variance in the second filial generation (F2) for genetic mapping. The correlation of 1192 individually phenotyped F2 embryos (heart rate) with the corresponding whole-genome sequences established a single nucleotide polymorphism-based linkage map. Integrated analysis of parental cardiac transcriptomes highlighted 59 loci containing genes prominently linked to human cardiovascular phenotypes. In vivo knockout models of the top 12 candidate genes demonstrated a causal and differential impact on heart development, ventricular size, and heart rate, evidencing the polygenic nature of heart phenotypes and establishing new loci for genetic diagnostics of human congenital heart disease. The controllable and scalable genetics and experimental validation in inbred vertebrate models complement and expand human association studies to provide novel and unique insights into the role of natural genetic variation in the mechanism of cardiac disease and allow to identify genetic susceptibility in individual human subjects.\nProteome-scale prediction of molecular mechanisms underlying dominant genetic diseases\nAuthors: Badonyi, M.; Marsh, J. A.\nScore: 7.3, Published: 2023-09-08 DOI: 10.1101/2023.09.08.556798\nMany dominant genetic disorders result from protein-altering mutations, acting primarily through dominant-negative (DN), gain-of-function (GOF), and loss-of-function (LOF) mechanisms. Deciphering the mechanisms by which dominant diseases exert their effects is often experimentally challenging and resource intensive, but is essential for developing appropriate therapeutic approaches. Diseases that arise via a LOF mechanism are more amenable to be treated by conventional gene therapy, whereas DN and GOF mechanisms may require gene editing or targeting by small molecules. Moreover, pathogenic missense mutations that act via DN and GOF mechanisms are more difficult to identify than those that act via LOF using nearly all currently available variant effect predictors. Here, we introduce a tripartite statistical model made up of support vector machine binary classifiers trained to predict whether human protein-coding genes are likely to be associated with DN, GOF, or LOF molecular disease mechanisms. We test the utility of the predictions by examining biologically and clinically meaningful properties known to be associated with the mechanisms. Our results strongly support that the models are able to generalise on unseen data and offer insight into the functional attributes of proteins associated with different mechanisms. We hope that our predictions will serve as a springboard for researchers studying novel variants and those of uncertain clinical significance, guiding variant interpretation strategies and experimental characterisation. Predictions for the 2023_02 UniProt reference proteome are available at https://osf.io/z4dcp/.\nEvolutionary diversity of the control of the azole response by Tra1 across yeast species\nAuthors: Librais, G. M.; Jiang, Y.; Razzaq, I.; Brandl, C. J.; Shapiro, R. S.; Lajoie, P.\nScore: 6.4, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556458\nTra1 is an essential co-activator protein of the yeast SAGA and NuA4 acetyltransferase complexes that regulate gene expression through multiple mechanisms including the acetylation of histone proteins. Tra1 is a pseudokinase of the PIKK family characterized by a C-terminal PI3K domain with no known kinase activity. However, mutations of specific arginine residues to glutamine in the PI3K domains (an allele termed tra1Q3) result in reduced growth and increased sensitivity to multiple stresses. In the opportunistic fungal pathogen Candida albicans, the tra1Q3 allele reduces pathogenicity and increases sensitivity to the echinocandin antifungal drug caspofungin, which disrupts the fungal cell wall. Here, we found that loss of Tra1 function, in contrast to what is seen with caspofungin, increases tolerance to the azole class of antifungal drugs, which inhibits ergosterol synthesis. In C. albicans, tra1Q3 increases expression of genes linked to azole resistance, such as ERG11 and CDR1. CDR1 encodes a multidrug ABC transporter associated with efflux of multiple xenobiotics, including azoles. Consequently, cells carrying tra1Q3 show reduced intracellular accumulation of fluconazole. In contrast, a tra1Q3 S. cerevisiae strain displayed opposite phenotypes: decreased tolerance to azole, decreased expression of the efflux pump PDR5 and increased intracellular accumulation of fluconazole. Therefore, our data provide evidence that Tra1 differentially regulates the antifungal response across yeast species.\nGeneration of isogenic models of Angelman syndrome and Prader-Willi syndrome in CRISPR/Cas9-engineered human embryonic stem cells\nAuthors: Gilmore, R. B.; Gorka, D.; Stoddard, C.; Cotney, J.; Chamberlain, S.\nScore: 8.8, Published: 2023-08-30 DOI: 10.1101/2023.08.30.555563\nAngelman Syndrome (AS) and Prader-Willi Syndrome (PWS), two distinct neurodevelopmental disorders, result from loss of expression from imprinted genes in the chromosome 15q11-13 locus most commonly caused by a megabase-scale deletion on either the maternal or paternal allele, respectively. Each occurs at an approximate incidence of 1/15,000 to 1/30,000 live births and has a range of debilitating phenotypes. Patient-derived induced pluripotent stem cells (iPSCs) have been valuable tools to understand human-relevant gene regulation at this locus and have contributed to the development of therapeutic approaches for AS. Nonetheless, gaps remain in our understanding of how these deletions contribute to dysregulation and phenotypes of AS and PWS. Variability across cell lines due to donor differences, reprogramming methods, and genetic background make it challenging to fill these gaps in knowledge without substantially increasing the number of cell lines used in the analyses. Isogenic cell lines that differ only by the genetic mutation causing the disease can ease this burden without requiring such a large number of cell lines. Here, we describe the development of isogenic human embryonic stem cell (hESC) lines modeling the most common genetic subtypes of AS and PWS. These lines allow for a facile interrogation of allele-specific gene regulation at the chromosome 15q11-q13 locus. Additionally, these lines are an important resource to identify and test targeted therapeutic approaches for patients with AS and PWS.\nFemale germline expression of OVO transcription factor bridges Drosophila generations\nAuthors: Benner, L.; Muron, S.; Oliver, B.\nScore: 7.7, Published: 2023-08-26 DOI: 10.1101/2023.08.25.554887\nOVO is required for karyotypically female germ cell viability but has no known function in the male germline in Drosophila. ovo is autoregulated by two antagonistic isoforms, OVO-A and OVO-B. All ovo- alleles were created as partial revertants of the antimorphic ovoD1allele. Creation of new targeted alleles in an ovo+ background indicated that disrupting the germline-specific exon extension of ovo-B leads to an arrested egg chamber phenotype, rather than germ cell death. RNA-seq analysis, including \u003e1K full length cDNAs, indicates that ovo utilizes a number of unannotated splice variations in the extended exon and a minor population of ovo-B transcripts utilizes an alternative splice. This indicates that classical ovo alleles such as ovoD1rv23, are not truly null for ovo, and are likely to be weak antimorphs. To generate bonafide nulls, we deleted the ovo-A and ovo-B promoters showing that only ovo-B is required for female germ cell viability and there is an early and polyphasic developmental requirement for ovo-B in the female germline. To visualize OVO expression and localization, we endogenously tagged ovo and found nuclear OVO in all differentiating female germ cells throughout oogenesis in adults. We also found that OVO is maternally deposited into the embryo, where it showed nuclear localization in newly formed pole cells. Maternal OVO persisted in embryonic germ cells until zygotic OVO expression was detectable, suggesting that there is continuous nuclear OVO expression in the female germline in the transition from one generation to the next. Article Summaryovo has long been considered to be at the top of the female germline sex determination pathway. We utilized updated genetic methods to determine OVO expression, localization, and requirement in the embryonic and adult germline. Our results indicate that OVO is always present, and likely required, in the Drosophila female germline.\nThe role of heterochronic gene expression and regulatory architecture in early developmental divergence\nAuthors: Harry, N. D.; Zakas, C.\nScore: 7.9, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553877\nNew developmental programs can evolve through adaptive changes to gene expression. The annelid S. benedicti has a developmental dimorphism, which provides a unique intraspecific framework for understanding the earliest genetic changes that take place during developmental divergence. Using comparative RNAseq through ontogeny, we find that only a small proportion of genes are differentially expressed at any time, despite major differences in larval development and life-history. These genes shift expression profiles across morphs by either turning off any expression in one morph or changing the timing or amount of gene expression. We directly connect the contributions of these mechanisms to differences in developmental processes. We examine F1 offspring-- using reciprocal crosses-- to determine maternal mRNA inheritance and the regulatory architecture of gene expression. These results highlight the importance of both novel gene expression and heterochronic shifts in developmental evolution, as well as the trans-acting regulatory factors in initiating divergence.\nGetPrimers: a generalized PCR-based genetic targeting primer designer enabling easy and standardized targeted gene modification across multiple systems\nAuthors: Miao, Z.; Wang, H.; Tu, X.; Huang, Z.; Huang, S.; Zhang, X.; Wang, F.; Huang, Z.; Li, H.; Jiao, Y.; Gao, S.; Zhou, Z.; Shan, C.-M.; Li, J.; Yue, J.-X.\nScore: 12.8, Published: 2023-08-14 DOI: 10.1101/2023.08.12.552711\nGenetic targeting (e.g., gene knockout and tagging) based on polymerase chain reaction (PCR) is a simple yet powerful approach for studying gene functions. Although originally developed in classic budding and fission yeast models, the same principle applies to other eukaryotic systems with efficient homologous recombination. One-step-PCR-based genetic targeting is conventionally used but the sizes of the homologous arms that it generates for recombination-mediated genetic targeting are usually limited. Alternatively, gene targeting can also be performed via fusion PCR, which can create homologous arms that are orders of magnitude larger, therefore substantially increases the efficiency of recombination-mediated genetic targeting. Here we present GetPrimers (https://www.evomicslab.org/app/getprimers/), a generalized computational framework and web tool to assist automatic targeting and verification primer design for both one-step-PCR-based and fusion-PCR-based genetic targeting experiments. Moreover, GetPrimers by design runs for any given genetic background of any species with full genome scalability. Therefore, GetPrimers is capable of empowering high-throughput functional genomics assays at multi-population and multi-species levels. Comprehensive experimental validations have been performed for targeting and verification primers designed by GetPrimers across multiple organism systems and experimental setups. We anticipate GetPrimers to become a highly useful and popular tool to facilitate easy and standardized gene modification across multiple systems. KeypointsDeveloped the web tool GetPrimers to assist automatic primer design for PCR-based genetic targeting. It runs for any given locus on any genetic background of any given species with full genome scalability. Experimental validations have been performed across multiple organism systems and experimental setups.\nAncient farmer and steppe pastoralist-related founding lineages contributed to the complex landscape of episodes in the diversification of Chinese paternal lineages\nAuthors: Wang, M.; Huang, Y.; Liu, K.; Yuan, H.; Duan, S.; Wang, Z.; Wei, L.; Yao, H.; Sun, Q.; Zhong, J.; Tang, R.; Chen, J.; Sun, Y.; Li, X.; Su, H.; Yang, Q.; Hu, L.; Yun, L.; Yang, J.; Nie, S.; Cai, Y.; Yan, J.; Zhou, K.; 10K_CPGDP Consortium, ; Wang, C.-C.; Zhu, B.; Liu, C.; He, G.\nScore: 28.0, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555114\nAncient DNA advances have reported the complex genetic history of Eurasians, but how the knowledge of ancient subsistence strategy shifts and population movements influenced the fine-scale paternal genetic structure in East Asia has not been assessed. Here, we reported one integrated Y-chromosome genomic database of 15,530 people, including 1753 ancient people and newly-reported 919 individuals genotyped using our recently-developed targeted sequencing YHSeqY3000 panel, to explore Chinese genomic diversity, population evolutionary tracts and their genetic formation mechanism. We identified four major ancient technological innovations and population movements that shaped the landscape of Chinese paternal lineages. First, the expansion of millet farmers and early East Asians from the Yellow River Basin carrying the major O2/D subclades promoted the formation of the Sino-Tibetan peoples major composition and accelerated the Tibetan Plateaus permanent occupation. Second, rice farmers dispersal from the Yangtze River Valley carrying O1 and some sublineages of O2 contributed significantly to Tai-Kadai, Austronesian, Hmong-Mien, Austroasiatic people and southern Han Chinese. Third, Siberian-related paternal lineages of Q and C originated and boomed from Neolithic hunter-gatherers from the Mongolian Plateau and the Amur River Basin and significantly influenced the gene pools of northern Chinese. Fourth, western Eurasian-derived J, G and R lineages initially spread with Yamnaya steppe pastoralists and other proto-Indo-European people and further widely dispersed via the trans-Eurasian cultural communication along the Eurasian Steppe and the ancient Silk Road, remaining genetic trajectories in northwestern Chinese. Our work provided comprehensive modern and ancient genetic evidence to illuminate the impact of population interaction from the ancient farmer or herder-based societies on the genetic diversity patterns of modern people, revised our understandings of ancestral sources of Chinese paternal lineages, underscored the scientific imperative of the large-scale genomic resources of dense spatiotemporal underrepresented sampling populations to understand human evolutionary history.\n",
  "wordCount" : "2945",
  "inLanguage": "en",
  "datePublished": "2023-09-10T10:36:49Z",
  "dateModified": "2023-09-10T10:36:49Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on September 10, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.02.556057">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.02.556057" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.02.556057">
        <p class="paperTitle">The simplicity of protein sequence-function relationships</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.02.556057" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.02.556057" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.</p>
        <p class="info">Score: 29.8, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.02.556057' target='https://doi.org/10.1101/2023.09.02.556057'> 10.1101/2023.09.02.556057</a></p>
        <p class="abstract">How complicated is the relationship between a proteins sequence and its function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most prior studies, however, used methods that misinterpret measurement errors, small local idiosyncracies around a designated wild-type sequence, and global nonlinearity in the sequence-function relationship as rampant high-order interactions. Here we present a simple new method to jointly estimate global nonlinearity and specific epistatic interactions across a proteins genotype-phenotype map. Our reference-free approach calculates the effect of each amino acid state or combination by averaging over all genotypes that contain it relative to the global average. We show that this method is more accurate than any alternative approach and is robust to measurement error and partial sampling. We reanalyze 20 combinatorial mutagenesis experiments and find that main and pairwise effects, together with a simple form of global nonlinearity, account for a median of 96% of total variance in the measured phenotype (and &gt; 92% in every case), and only a tiny fraction of genotypes are strongly affected by epistasis at third or higher orders. The genetic architecture is also sparse: the number of model terms required to explain the vast majority of phenotypic variance is smaller than the number of genotypes by many orders of magnitude. The sequence-function relationship in most proteins is therefore far simpler than previously thought, and new, more tractable experimental approaches, combined with reference-free analysis, may be sufficient to explain it in most cases.

Significance StatementIt is widely thought that a proteins function depends on complex interactions among amino acids. If so, it would be virtually impossible to predict the function of new variants, and understanding how proteins work genetically and biochemically would require huge combinatorial experiments. We show that prior studies overestimated complexity because they analyzed sequence-function relationships from the perspective of a single reference genotype and/or misinterpreted global phenotypic nonlinearities as complex amino acid interactions. By developing a new reference-free approach and using it to reanalyze 20 experimental datasets, we show that additive effects and pairwise interactions alone, along with a simple global nonlinearity, explain the vast majority of functional variation. Higher-order interactions are weak or rare, and a minuscule fraction of possible interactions shape each proteins function. Our work reveals that protein sequence-function relationships are surprisingly simple and suggests new strategies that are far more tractable than the massive experiments currently used.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555359">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555359" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555359">
        <p class="paperTitle">Functional characterization of gene regulatory elements and neuropsychiatric disease-associated risk loci in iPSCs and iPSC-derived neurons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555359" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555359" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, X.; Jones, I. R.; Chen, P. B.; Yang, H.; Ren, X.; Zheng, L.; Li, B.; Li, Y. E.; Sun, Q.; Wen, J.; Beaman, C.; Cui, X.; Li, Y.; Wang, W.; Hu, M.; Ren, B.; Shen, Y.</p>
        <p class="info">Score: 27.0, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555359' target='https://doi.org/10.1101/2023.08.30.555359'> 10.1101/2023.08.30.555359</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have identified thousands of non-coding variants that contribute to psychiatric disease risks, likely by perturbing cis-regulatory elements (CREs). However, our ability to interpret and explore their mechanisms of action is hampered by a lack of annotation of functional CREs (fCREs) in neural cell types. Here, through genome-scale CRISPR screens of 22,000 candidate CREs (cCREs) in human induced pluripotent stem cells (iPSCs) undergoing differentiation to excitatory neurons, we identify 2,847 and 5,540 fCREs essential for iPSC fitness and neuronal differentiation, respectively. These fCREs display dynamic epigenomic features and exhibit increased numbers and genomic spans of chromatin interactions following terminal neuronal differentiation. Furthermore, fCREs essential for neuronal differentiation show significantly greater enrichment of genetic heritability for neurodevelopmental diseases including schizophrenia (SCZ), attention deficit hyperactivity disorder (ADHD), and autism spectrum disorders (ASD) than cCREs. Using high-throughput prime editing screens we experimentally confirm 45 SCZ risk variants that act by affecting the function of fCREs. The extensive and in-depth functional annotation of cCREs in neuronal types therefore provides a crucial resource for interpreting non-coding risk variants of neuropsychiatric disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555906" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555906">
        <p class="paperTitle">Natural genetic variation quantitatively regulates heart-rate and -dimension</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gierten, J.; Welz, B.; Fitzgerald, T. W.; Thumberger, T.; Hummel, O.; Leger, A.; Weber, P.; Hassel, D.; Hubner, N.; Birney, E.; Wittbrodt, J.</p>
        <p class="info">Score: 7.9, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555906' target='https://doi.org/10.1101/2023.09.01.555906'> 10.1101/2023.09.01.555906</a></p>
        <p class="abstract">Development and function of the heart are crucially controlled by individual genetic factors determining the continuum from health to disease. A prime example of a quantitative pathological phenotype is left ventricular hypoplasia, affecting the left ventricle (LV) and its associated inflow and outflow structures and reducing its size to varying degrees1. Current models suggest polygenic contributions on heart development and disease, but we are just beginning to understand the impact of naturally occurring sequence variations. Here we show how natural genetic variants exert cumulative, environment-sensitive effects on both embryonic heart development as well as adult heart function. We addressed this in embryos of five inbred strains of the Japanese rice fish medaka (Oryzias latipes2 and Oryzias sakaizumii3), analyzing heart rates as a functional proxy to capture combined characteristics of the conduction system and structural morphology. We identified one strain, HO52, with embryonic hypoplastic ventricles, fast heart rates, and significantly impaired cardiac function and overall fitness in adults. We created an interpopulation cross of the divergent HO5 strain to HdrR, which exhibits a morphologically normal heart and comparably slow heart rates, to leverage segregation variance in the second filial generation (F2) for genetic mapping. The correlation of 1192 individually phenotyped F2 embryos (heart rate) with the corresponding whole-genome sequences established a single nucleotide polymorphism-based linkage map. Integrated analysis of parental cardiac transcriptomes highlighted 59 loci containing genes prominently linked to human cardiovascular phenotypes. In vivo knockout models of the top 12 candidate genes demonstrated a causal and differential impact on heart development, ventricular size, and heart rate, evidencing the polygenic nature of heart phenotypes and establishing new loci for genetic diagnostics of human congenital heart disease. The controllable and scalable genetics and experimental validation in inbred vertebrate models complement and expand human association studies to provide novel and unique insights into the role of natural genetic variation in the mechanism of cardiac disease and allow to identify genetic susceptibility in individual human subjects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556798" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556798">
        <p class="paperTitle">Proteome-scale prediction of molecular mechanisms underlying dominant genetic diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Badonyi, M.; Marsh, J. A.</p>
        <p class="info">Score: 7.3, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556798' target='https://doi.org/10.1101/2023.09.08.556798'> 10.1101/2023.09.08.556798</a></p>
        <p class="abstract">Many dominant genetic disorders result from protein-altering mutations, acting primarily through dominant-negative (DN), gain-of-function (GOF), and loss-of-function (LOF) mechanisms. Deciphering the mechanisms by which dominant diseases exert their effects is often experimentally challenging and resource intensive, but is essential for developing appropriate therapeutic approaches. Diseases that arise via a LOF mechanism are more amenable to be treated by conventional gene therapy, whereas DN and GOF mechanisms may require gene editing or targeting by small molecules. Moreover, pathogenic missense mutations that act via DN and GOF mechanisms are more difficult to identify than those that act via LOF using nearly all currently available variant effect predictors. Here, we introduce a tripartite statistical model made up of support vector machine binary classifiers trained to predict whether human protein-coding genes are likely to be associated with DN, GOF, or LOF molecular disease mechanisms. We test the utility of the predictions by examining biologically and clinically meaningful properties known to be associated with the mechanisms. Our results strongly support that the models are able to generalise on unseen data and offer insight into the functional attributes of proteins associated with different mechanisms. We hope that our predictions will serve as a springboard for researchers studying novel variants and those of uncertain clinical significance, guiding variant interpretation strategies and experimental characterisation. Predictions for the 2023_02 UniProt reference proteome are available at https://osf.io/z4dcp/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556458">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556458" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556458">
        <p class="paperTitle">Evolutionary diversity of the control of the azole response by Tra1 across yeast species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556458" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556458" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Librais, G. M.; Jiang, Y.; Razzaq, I.; Brandl, C. J.; Shapiro, R. S.; Lajoie, P.</p>
        <p class="info">Score: 6.4, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556458' target='https://doi.org/10.1101/2023.09.05.556458'> 10.1101/2023.09.05.556458</a></p>
        <p class="abstract">Tra1 is an essential co-activator protein of the yeast SAGA and NuA4 acetyltransferase complexes that regulate gene expression through multiple mechanisms including the acetylation of histone proteins. Tra1 is a pseudokinase of the PIKK family characterized by a C-terminal PI3K domain with no known kinase activity. However, mutations of specific arginine residues to glutamine in the PI3K domains (an allele termed tra1Q3) result in reduced growth and increased sensitivity to multiple stresses. In the opportunistic fungal pathogen Candida albicans, the tra1Q3 allele reduces pathogenicity and increases sensitivity to the echinocandin antifungal drug caspofungin, which disrupts the fungal cell wall. Here, we found that loss of Tra1 function, in contrast to what is seen with caspofungin, increases tolerance to the azole class of antifungal drugs, which inhibits ergosterol synthesis. In C. albicans, tra1Q3 increases expression of genes linked to azole resistance, such as ERG11 and CDR1. CDR1 encodes a multidrug ABC transporter associated with efflux of multiple xenobiotics, including azoles. Consequently, cells carrying tra1Q3 show reduced intracellular accumulation of fluconazole. In contrast, a tra1Q3 S. cerevisiae strain displayed opposite phenotypes: decreased tolerance to azole, decreased expression of the efflux pump PDR5 and increased intracellular accumulation of fluconazole. Therefore, our data provide evidence that Tra1 differentially regulates the antifungal response across yeast species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555563">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555563" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555563">
        <p class="paperTitle">Generation of isogenic models of Angelman syndrome and Prader-Willi syndrome in CRISPR/Cas9-engineered human embryonic stem cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555563" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555563" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gilmore, R. B.; Gorka, D.; Stoddard, C.; Cotney, J.; Chamberlain, S.</p>
        <p class="info">Score: 8.8, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555563' target='https://doi.org/10.1101/2023.08.30.555563'> 10.1101/2023.08.30.555563</a></p>
        <p class="abstract">Angelman Syndrome (AS) and Prader-Willi Syndrome (PWS), two distinct neurodevelopmental disorders, result from loss of expression from imprinted genes in the chromosome 15q11-13 locus most commonly caused by a megabase-scale deletion on either the maternal or paternal allele, respectively. Each occurs at an approximate incidence of 1/15,000 to 1/30,000 live births and has a range of debilitating phenotypes. Patient-derived induced pluripotent stem cells (iPSCs) have been valuable tools to understand human-relevant gene regulation at this locus and have contributed to the development of therapeutic approaches for AS. Nonetheless, gaps remain in our understanding of how these deletions contribute to dysregulation and phenotypes of AS and PWS. Variability across cell lines due to donor differences, reprogramming methods, and genetic background make it challenging to fill these gaps in knowledge without substantially increasing the number of cell lines used in the analyses. Isogenic cell lines that differ only by the genetic mutation causing the disease can ease this burden without requiring such a large number of cell lines. Here, we describe the development of isogenic human embryonic stem cell (hESC) lines modeling the most common genetic subtypes of AS and PWS. These lines allow for a facile interrogation of allele-specific gene regulation at the chromosome 15q11-q13 locus. Additionally, these lines are an important resource to identify and test targeted therapeutic approaches for patients with AS and PWS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554887">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554887" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554887">
        <p class="paperTitle">Female germline expression of OVO transcription factor bridges Drosophila generations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554887" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554887" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benner, L.; Muron, S.; Oliver, B.</p>
        <p class="info">Score: 7.7, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554887' target='https://doi.org/10.1101/2023.08.25.554887'> 10.1101/2023.08.25.554887</a></p>
        <p class="abstract">OVO is required for karyotypically female germ cell viability but has no known function in the male germline in Drosophila. ovo is autoregulated by two antagonistic isoforms, OVO-A and OVO-B. All ovo- alleles were created as partial revertants of the antimorphic ovoD1allele. Creation of new targeted alleles in an ovo&#43; background indicated that disrupting the germline-specific exon extension of ovo-B leads to an arrested egg chamber phenotype, rather than germ cell death. RNA-seq analysis, including &gt;1K full length cDNAs, indicates that ovo utilizes a number of unannotated splice variations in the extended exon and a minor population of ovo-B transcripts utilizes an alternative splice. This indicates that classical ovo alleles such as ovoD1rv23, are not truly null for ovo, and are likely to be weak antimorphs. To generate bonafide nulls, we deleted the ovo-A and ovo-B promoters showing that only ovo-B is required for female germ cell viability and there is an early and polyphasic developmental requirement for ovo-B in the female germline. To visualize OVO expression and localization, we endogenously tagged ovo and found nuclear OVO in all differentiating female germ cells throughout oogenesis in adults. We also found that OVO is maternally deposited into the embryo, where it showed nuclear localization in newly formed pole cells. Maternal OVO persisted in embryonic germ cells until zygotic OVO expression was detectable, suggesting that there is continuous nuclear OVO expression in the female germline in the transition from one generation to the next.

Article Summaryovo has long been considered to be at the top of the female germline sex determination pathway. We utilized updated genetic methods to determine OVO expression, localization, and requirement in the embryonic and adult germline. Our results indicate that OVO is always present, and likely required, in the Drosophila female germline.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553877">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553877" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553877">
        <p class="paperTitle">The role of heterochronic gene expression and regulatory architecture in early developmental divergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553877" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553877" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harry, N. D.; Zakas, C.</p>
        <p class="info">Score: 7.9, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553877' target='https://doi.org/10.1101/2023.08.18.553877'> 10.1101/2023.08.18.553877</a></p>
        <p class="abstract">New developmental programs can evolve through adaptive changes to gene expression. The annelid S. benedicti has a developmental dimorphism, which provides a unique intraspecific framework for understanding the earliest genetic changes that take place during developmental divergence. Using comparative RNAseq through ontogeny, we find that only a small proportion of genes are differentially expressed at any time, despite major differences in larval development and life-history. These genes shift expression profiles across morphs by either turning off any expression in one morph or changing the timing or amount of gene expression. We directly connect the contributions of these mechanisms to differences in developmental processes. We examine F1 offspring-- using reciprocal crosses-- to determine maternal mRNA inheritance and the regulatory architecture of gene expression. These results highlight the importance of both novel gene expression and heterochronic shifts in developmental evolution, as well as the trans-acting regulatory factors in initiating divergence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.552711">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.552711" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.552711">
        <p class="paperTitle">GetPrimers: a generalized PCR-based genetic targeting primer designer enabling easy and standardized targeted gene modification across multiple systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.552711" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.552711" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miao, Z.; Wang, H.; Tu, X.; Huang, Z.; Huang, S.; Zhang, X.; Wang, F.; Huang, Z.; Li, H.; Jiao, Y.; Gao, S.; Zhou, Z.; Shan, C.-M.; Li, J.; Yue, J.-X.</p>
        <p class="info">Score: 12.8, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.552711' target='https://doi.org/10.1101/2023.08.12.552711'> 10.1101/2023.08.12.552711</a></p>
        <p class="abstract">Genetic targeting (e.g., gene knockout and tagging) based on polymerase chain reaction (PCR) is a simple yet powerful approach for studying gene functions. Although originally developed in classic budding and fission yeast models, the same principle applies to other eukaryotic systems with efficient homologous recombination. One-step-PCR-based genetic targeting is conventionally used but the sizes of the homologous arms that it generates for recombination-mediated genetic targeting are usually limited. Alternatively, gene targeting can also be performed via fusion PCR, which can create homologous arms that are orders of magnitude larger, therefore substantially increases the efficiency of recombination-mediated genetic targeting. Here we present GetPrimers (https://www.evomicslab.org/app/getprimers/), a generalized computational framework and web tool to assist automatic targeting and verification primer design for both one-step-PCR-based and fusion-PCR-based genetic targeting experiments. Moreover, GetPrimers by design runs for any given genetic background of any species with full genome scalability. Therefore, GetPrimers is capable of empowering high-throughput functional genomics assays at multi-population and multi-species levels. Comprehensive experimental validations have been performed for targeting and verification primers designed by GetPrimers across multiple organism systems and experimental setups. We anticipate GetPrimers to become a highly useful and popular tool to facilitate easy and standardized gene modification across multiple systems.

KeypointsDeveloped the web tool GetPrimers to assist automatic primer design for PCR-based genetic targeting.

It runs for any given locus on any genetic background of any given species with full genome scalability.

Experimental validations have been performed across multiple organism systems and experimental setups.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555114">
        <p class="paperTitle">Ancient farmer and steppe pastoralist-related founding lineages contributed to the complex landscape of episodes in the diversification of Chinese paternal lineages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, M.; Huang, Y.; Liu, K.; Yuan, H.; Duan, S.; Wang, Z.; Wei, L.; Yao, H.; Sun, Q.; Zhong, J.; Tang, R.; Chen, J.; Sun, Y.; Li, X.; Su, H.; Yang, Q.; Hu, L.; Yun, L.; Yang, J.; Nie, S.; Cai, Y.; Yan, J.; Zhou, K.; 10K_CPGDP Consortium,  ; Wang, C.-C.; Zhu, B.; Liu, C.; He, G.</p>
        <p class="info">Score: 28.0, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555114' target='https://doi.org/10.1101/2023.08.28.555114'> 10.1101/2023.08.28.555114</a></p>
        <p class="abstract">Ancient DNA advances have reported the complex genetic history of Eurasians, but how the knowledge of ancient subsistence strategy shifts and population movements influenced the fine-scale paternal genetic structure in East Asia has not been assessed. Here, we reported one integrated Y-chromosome genomic database of 15,530 people, including 1753 ancient people and newly-reported 919 individuals genotyped using our recently-developed targeted sequencing YHSeqY3000 panel, to explore Chinese genomic diversity, population evolutionary tracts and their genetic formation mechanism. We identified four major ancient technological innovations and population movements that shaped the landscape of Chinese paternal lineages. First, the expansion of millet farmers and early East Asians from the Yellow River Basin carrying the major O2/D subclades promoted the formation of the Sino-Tibetan peoples major composition and accelerated the Tibetan Plateaus permanent occupation. Second, rice farmers dispersal from the Yangtze River Valley carrying O1 and some sublineages of O2 contributed significantly to Tai-Kadai, Austronesian, Hmong-Mien, Austroasiatic people and southern Han Chinese. Third, Siberian-related paternal lineages of Q and C originated and boomed from Neolithic hunter-gatherers from the Mongolian Plateau and the Amur River Basin and significantly influenced the gene pools of northern Chinese. Fourth, western Eurasian-derived J, G and R lineages initially spread with Yamnaya steppe pastoralists and other proto-Indo-European people and further widely dispersed via the trans-Eurasian cultural communication along the Eurasian Steppe and the ancient Silk Road, remaining genetic trajectories in northwestern Chinese. Our work provided comprehensive modern and ancient genetic evidence to illuminate the impact of population interaction from the ancient farmer or herder-based societies on the genetic diversity patterns of modern people, revised our understandings of ancestral sources of Chinese paternal lineages, underscored the scientific imperative of the large-scale genomic resources of dense spatiotemporal underrepresented sampling populations to understand human evolutionary history.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
