<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="DNA methylation modulates nucleosome retention in sperm and H3K4 methylation deposition in early mouse embryos.
Authors: Fanourgakis, G.; Gaspa-Toneu, L.; Komarov, P. A.; Ozonov, E. A.; Smallwood, S. A.; Peters, A. H. F. M.
Score: 18.9, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579069
DNA methylation (DNAme) serves a stable gene regulatory function in somatic cells (1). In the germ line and during early embryogenesis, however, DNAme undergoes global erasure and re-establishment to support germ cell and embryonic development (2).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="DNA methylation modulates nucleosome retention in sperm and H3K4 methylation deposition in early mouse embryos.
Authors: Fanourgakis, G.; Gaspa-Toneu, L.; Komarov, P. A.; Ozonov, E. A.; Smallwood, S. A.; Peters, A. H. F. M.
Score: 18.9, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579069
DNA methylation (DNAme) serves a stable gene regulatory function in somatic cells (1). In the germ line and during early embryogenesis, however, DNAme undergoes global erasure and re-establishment to support germ cell and embryonic development (2)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-11T10:38:55+00:00" />
<meta property="article:modified_time" content="2024-02-11T10:38:55+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="DNA methylation modulates nucleosome retention in sperm and H3K4 methylation deposition in early mouse embryos.
Authors: Fanourgakis, G.; Gaspa-Toneu, L.; Komarov, P. A.; Ozonov, E. A.; Smallwood, S. A.; Peters, A. H. F. M.
Score: 18.9, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579069
DNA methylation (DNAme) serves a stable gene regulatory function in somatic cells (1). In the germ line and during early embryogenesis, however, DNAme undergoes global erasure and re-establishment to support germ cell and embryonic development (2)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "DNA methylation modulates nucleosome retention in sperm and H3K4 methylation deposition in early mouse embryos.\nAuthors: Fanourgakis, G.; Gaspa-Toneu, L.; Komarov, P. A.; Ozonov, E. A.; Smallwood, S. A.; Peters, A. H. F. M.\nScore: 18.9, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579069\nDNA methylation (DNAme) serves a stable gene regulatory function in somatic cells (1). In the germ line and during early embryogenesis, however, DNAme undergoes global erasure and re-establishment to support germ cell and embryonic development (2).",
  "keywords": [
    
  ],
  "articleBody": " DNA methylation modulates nucleosome retention in sperm and H3K4 methylation deposition in early mouse embryos.\nAuthors: Fanourgakis, G.; Gaspa-Toneu, L.; Komarov, P. A.; Ozonov, E. A.; Smallwood, S. A.; Peters, A. H. F. M.\nScore: 18.9, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579069\nDNA methylation (DNAme) serves a stable gene regulatory function in somatic cells (1). In the germ line and during early embryogenesis, however, DNAme undergoes global erasure and re-establishment to support germ cell and embryonic development (2). While de novo DNAme acquisition during male germ cell development is essential for setting genomic DNA methylation imprints, other intergenerational roles for paternal DNAme in defining embryonic chromatin after fertilization are unknown. To approach this question, we reduced levels of DNAme in developing male germ cells through conditional gene deletion of the de novo DNA methyltransferases DNMT3A and DNMT3B in undifferentiated spermatogonia. We observed that DNMT3A serves a DNAme maintenance function in undifferentiated spermatogonia while DNMT3B catalyzes de novo DNAme during spermatogonial differentiation. Mutant male germ cells nevertheless completed their differentiation to sperm. Failing de novo DNAme in Dnmt3a/Dnmt3b double deficient spermatogonia is associated with increased nucleosome occupancy in mature sperm, preferentially at sites with higher CpG content, supporting the model that DNAme modulates nucleosome retention in sperm (3). To assess the impact of altered sperm chromatin in the formation of embryonic chromatin, we measured H3K4me3 occupancy at paternal and maternal alleles in 2-cell embryos using a newly developed transposon-based tagging assay for modified chromatin. Our data show that reduced DNAme in sperm renders paternal alleles permissive for H3K4me3 establishment in early embryos, independently of possible paternal inheritance of sperm born H3K4me3. Together, this study provides first evidence that paternally inherited DNAme directs chromatin formation during early embryonic development.\nDevelopmental disruption of the mitochondrial fissiongene drp-1 extends the longevity of daf-2 insulin/IGF-1 receptor mutant\nAuthors: Traa, A.; Van Raamsdonk, J.\nScore: 18.2, Published: 2024-02-08 DOI: 10.1101/2024.02.07.579360\nThe dynamic nature of the mitochondrial network is regulated by mitochondrial fission and fusion, allowing for re-organization of mitochondria to adapt to the cell's ever-changing needs. As organisms age, mitochondrial fission and fusion become dysregulated and mitochondrial networks become increasingly fragmented. Modulation of mitochondrial dynamics has been shown to affect longevity in fungi, yeast, Drosophila and C. elegans. While disruption of the mitochondrial fission gene drp-1 only mildly increases wild-type lifespan, it drastically increases the already long lifespan of daf-2 insulin/IGF-1 signaling (IIS) mutants. In this work, we determined the conditions required for drp-1 disruption to extend daf-2 longevity and explored the molecular mechanisms involved. We found that knockdown of drp-1 during development is sufficient to extend daf-2 lifespan, while tissue-specific knockdown of drp-1 in neurons, intestine or muscle failed to increase daf-2 longevity. Disruption of other genes involved in mitochondrial fission also increased daf-2 lifespan as did treatment with a number of different RNAi clones that decrease mitochondrial fragmentation. In exploring potential mechanisms involved, we found that deletion of drp-1 increases resistance to chronic stresses and slows physiologic rates in daf-2 worms. In addition, we found that disruption of drp-1 increased mitochondrial and peroxisomal connectedness in daf-2 worms, increased oxidative phosphorylation and ATP levels, and increased mitophagy in daf-2 worms, but did not affect their ROS levels or mitochondrial membrane potential. Overall, this work defined the conditions under which drp-1 disruption increases daf-2 lifespan and has identified multiple changes in daf-2;drp-1 mutants that may contribute to their lifespan extension.\nA Genetic Screen for regulators of CNS morphology in Drosophila\nAuthors: Lacin, H.; Zhu, Y.; DiPaola, J.; Wilson, B. A.; Zhu, Y.; Skeath, J. B.\nScore: 8.9, Published: 2024-02-03 DOI: 10.1101/2024.01.09.574922\nDespite increasing in mass approximately 100-fold during larval life, the Drosophila CNS maintains its characteristic form during this rapid growth phase. Dynamic interactions between the overlying basement membrane and underlying surface glia are known to regulate CNS structure in Drosophila, but the genes and pathways that establish and maintain CNS morphology during development remain poorly characterized. To identify genes that regulate CNS shape in Drosophila, we conducted an EMS-based, forward genetic screen of the second chromosome, uncovered 50 mutations that disrupt CNS structure, and mapped these alleles to 17 genes. Whole genome sequencing revealed the affected gene for all but one gene. Identified genes include well characterized regulators of tissue shape, like Laminin B1, viking, and Collagen 4a1, as well as characterized genes, such as Toll-2 and Rme-8, with no known role in regulating CNS structure. We also uncovered that papilin and dC1GalTA likely act in the same pathway to regulate CNS structure and found that a glucuronosyltransferase that regulates Dystroglycan function in mammals is required to maintain CNS shape in Drosophila. Finally, we show that the senseless-2 transcription factor is expressed and functions specifically in surface glia found on peripheral nerves but not those on the CNS proper to govern CNS structure, identifying the first gene that functionally subdivides a glial subtype along the peripheral-central axis. Future work on these genes should help clarify the genetic mechanisms that ensure the homeostasis of CNS shape and form during development.\nLimitations of the inference of the distribution of fitness effects of new mutations in partially selfing populations with linkage\nAuthors: Daigle, A.; Johri, P.\nScore: 7.8, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579142\nThe accurate estimation of the distribution of fitness effects (DFE) of new mutations is critical for population genetic inference but remains a challenging task. While various methods have been developed for DFE inference using the site frequency spectrum of putatively neutral and selected sites, their applicability in species with diverse life history traits and complex demographic scenarios is not well understood. Selfing is common among eukaryotic species and can lead to decreased effective recombination rates in such populations, increasing the effects of selection at linked sites, including interference between selected alleles. We employ forward simulations to investigate the limitations of current DFE estimation approaches in the presence of selfing and linked effects of selection. We find that distortions of the site frequency spectrum due to Hill-Robertson interference in highly selfing populations lead to mis-inference of the deleterious DFE of new mutations. While accounting for the decrease in the effective population size due to linked effects of selection largely accounts for the observed bias in populations with moderate levels of selfing, this correction is unable to accurately estimate the DFE in highly selfing populations. In addition, the presence of cryptic population structure and uneven sampling across subpopulations leads to the false inference of a deleterious DFE skewed towards effectively neutral/mildly deleterious mutations. Finally, the proportion of adaptive substitutions estimated at high rates of selfing is substantially overestimated. Our results clarify the parameter space where current DFE methods might be problematic and where they remain robust in the presence of selfing and other model violations, such as departures from semidominance, population structure, and uneven sampling. Our observations apply broadly to species and genomic regions with little/no recombination.\nHelicase-assisted continuous editing for programmable mutagenesis of endogenous genomes\nAuthors: Chen, X. D.; Chen, Z.; Wythes, G.; Zhang, Y.; Orr, B. C.; Sun, G.; Thao, K.; Vallurupalli, M.; Sun, J.; Borji, M.; Tkacik, E.; Chen, H.; Bernstein, B. E.; Chen, F.\nScore: 30.4, Published: 2024-02-01 DOI: 10.1101/2024.02.01.577593\nA major challenge in human genomics is to decipher the context specific relationship of sequence to function. However, existing tools for locus specific hypermutation and evolution in the native genome context are limited. Here we present a novel programmable platform for long-range, locus-specific hypermutation called helicase-assisted continuous editing (HACE). HACE leverages CRISPR-Cas9 to target a processive helicase-deaminase fusion that incurs mutations across large (\u003e1000 bp) genomic intervals. We applied HACE to identify mutations in MEK1 that confer kinase inhibitor resistance, to dissect the impact of individual variants in SF3B1-dependent mis-splicing, and to evaluate noncoding variants in a stimulation-dependent immune enhancer of CD69. HACE provides a powerful tool for investigating coding and noncoding variants, uncovering combinatorial sequence-to-function relationships, and evolving new biological functions. One Sentence SummaryWe developed a tool for continuous, long-range, targeted diversification of endogenous mammalian genomes and used it to explore the function of genetic variants in both coding and non-coding regions.\nThe genetic landscape, gene flow and adaptation in Ladakh highlanders of trans-Himalayas\nAuthors: Kumar, L.; Rajpal, R.; Ahlawat, B.; Pasupuleti, N.; Konar, S.; Dwivedi, A.; Kumar, S.; Spalzin, S.; Rabyang, S.; Thangaraj, K.; Rai, N.\nScore: 6.6, Published: 2024-02-08 DOI: 10.1101/2024.02.05.579041\nThe Ladakh in the trans-Himalayas has witnessed complex cultural movements, genetic assimilation and demographic changes since the Neolithic period, which continue despite the harsh, inhospitable and cold climate. Although geographically isolated from mainland South Asia, Ladakh has historic trade routes (the Silk Road) and is well connected and accessible to travelers from Tibet and Central Asia. Despite its rich cultural heritage, a detailed description of the genetic landscape of the Ladakh region is completely lacking, particularly with regard to genome-wide markers and larger sample sizes. Therefore, in our current study, we genotyped 80 individuals from Kargil and Leh districts of the Union Territory of Ladakh, India. Here, we performed a comprehensive genetic analysis based on allele frequency and haplotype sharing. Our analysis revealed the presence of two distinct genetic lineages in the region with quite different genetic composition. The population of the Leh region is more similar to East Asian and Southeast Asian populations with population drift. In contrast, the population of the Kargil region is more similar to the Indo-European population. The former group shares a genetic history with Tibetans and Sherpas, while the latter group from Kargil associates with Kashmiri Muslims, Gujjars and Nepalese Brahmins. For both groups, there was a founder event at about 18th to 22th generations ago and duration coincided with the Mughal invasion. The genome-wide scan for a signal of positive selection revealed genetic signatures of altitude adaptation in the Leh population, while in the Kargil population the key gene signatures were associated with immunity and female fertility.\nGenetic and bioinformatic analyses reveal transcriptional networks underlying dual genomic coordination of mitochondrial biogenesis\nAuthors: Xu, H.\nScore: 7.7, Published: 2024-01-29 DOI: 10.1101/2024.01.25.577217\nMitochondrial genome encodes handful genes of respiratory chain complexes, whereas all the remaining mitochondrial proteins are encoded on the nuclear genome. However, the mechanisms coordinating these two genomes to control mitochondrial biogenesis remain largely unknown. To identify transcription circuits involved in these processes, we performed a candidate RNAi screen in developing eyes that had reduced mitochondrial DNA contents. We reasoned that impaired mitochondrial biogenesis would synergistically interact with mtDNA deficiency in disrupting tissue development. Over 638 transcription factors annotated in the fly genome, we identified 77 transcription factors that may be involved in mitochondrial genome maintenance and gene expression. Additional genetic and genomic analyses revealed that a novel transcription factor, CG1603, and its upstream factor YL-1 are essential for mitochondrial biogenesis. We constructed a regulator network among positive hits using the published CHIP-seq data. The network analysis revealed extensive connections, and complex hierarchical organization underlying the transcription regulation of mitochondrial biogenesis.\nSomatic epigenetic drift during shoot branching: a cell lineage-based model\nAuthors: Chen, Y.; Burian, A.; Johannes, F.\nScore: 9.0, Published: 2024-01-29 DOI: 10.1101/2024.01.24.577071\nPlant architecture is shaped by the continuous production of new organs, most of which emerge post-embryonically. This process includes the formation of new lateral branches along existing shoots. Shoot branching is fundamental to plant development, plant-environment interactions, and vegetative propagation. Current empirical evidence supports a \"detached meristem\" model as the cellular basis of lateral shoot initiation. In this model, a small number of undifferentiated cells are \"sampled\" from the periphery of the shoot apical meristem (SAM) to act as precursors for axillary buds, which eventually develop into new shoots. Repeated branching thus creates a series of cellular bottlenecks (i.e. somatic drift) that affect how de novo genetic and epigenetic mutations propagate through the plant body during development. Somatic drift could be particularly relevant for epigenetic changes in the form of stochastic DNA methylation gains and losses (i.e. spontaneous epimutations), as they have been shown to arise rapidly with each cell division. Here, we formalize a special case of the \"detached meristem\" model, where pre-cursor cells are randomly sampled from the SAM periphery in a way that maximizes cell lineage independence. By following a population of SAM cells through repeated branching processes, we show that somatic drift gives rise to a complex mixture of cellular phylogenies, which shape the evolution of cell-to-cell DNA methylation heterogeneity within the SAM over time. This process is dependent on the number of branch points, the strength of somatic drift as well as the epimutation rate. For many realistic cell biological settings, our model predicts that cell-to-cell DNA methylation heterogeneity in the SAM converges to non-zero states during development, suggesting that epigenetic variation is an inherent property of the SAM cell population. Our insights have direct implications for empirical studies of somatic (epi)genomic diversity in long-lived perennial and clonal species using bulk or single-cell sequencing approaches.\nPatient derived model of UBA5-associated encephalopathy identifies defects in neurodevelopment and highlights potential therapies\nAuthors: Chen, H.; Wang, Y.-D.; Blan, A. W.; Almanza-Fuerte, E. P.; Bonkowski, E. S.; Bajpai, R.; Pruett-Miller, S. M.; Mefford, H. C.\nScore: 21.2, Published: 2024-01-27 DOI: 10.1101/2024.01.25.577254\nUBA5 encodes for the E1 enzyme of the UFMylation cascade, which plays an essential role in ER homeostasis. The clinical phenotypes of UBA5-associated encephalopathy include developmental delays, epilepsy and intellectual disability. To date, there is no humanized neuronal model to study the cellular and molecular consequences of UBA5 pathogenic variants. We developed and characterized patient-derived cortical organoid cultures and identified defects in GABAergic interneuron development. We demonstrated aberrant neuronal firing and microcephaly phenotypes in patient-derived organoids. Mechanistically, we show that ER homeostasis is perturbed along with exacerbated unfolded protein response pathway in cells and organoids expressing UBA5 pathogenic variants. We also assessed two gene expression modalities that augmented UBA5 expression to rescue aberrant molecular and cellular phenotypes. Our study provides a novel humanized model that allows further investigations of UBA5 variants in the brain and highlights novel systemic approaches to alleviate cellular aberrations for this rare, developmental disorder. One-sentence summaryPatient derived model of UBA5-assoicated DEE recapitulated disease phenotype, revealed defects in neurodevelopment, and highlighted potential therapies.\nThe novel linkage between Fuz and Gpr161 genes regulates sonic hedgehog signaling during mouse embryonic development.\nAuthors: Kim, S. E.; Kim, H. Y.; Wlodarczyk, B. J.; Finnell, R. H.\nScore: 8.2, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575263\nSonic hedgehog (Shh) signaling regulates embryonic morphogenesis utilizing primary cilia, the cell antenna acting as a signaling hub. Fuz, an effector of planar cell polarity (PCP) signaling, involves Shh signaling via cilia formation, while the G protein-coupled receptor 161 (Gpr161) is a negative regulator of Shh signaling. The range of phenotypic malformations observed in mice bearing mutations in either of these two genes is similar; however, their functional relations have not been previously explored. This study identified the genetic and biochemical link between Fuz and Gpr161 in mouse embryonic development. Fuz was genetically epistatic to Gpr161 via Shh signaling during mouse embryonic development. The FUZ biochemically interacted with GPR161, and Fuz regulated Gpr161 ciliary trafficking via {beta}-arrestin2. Our study suggested the novel Gpr161-Fuz axis that regulates Shh signaling during mouse embryonic development. Summary statementThis study illuminates the novel genetic and biochemical linkages between Fuz and Gpr161 to regulate sonic hedgehog signaling during mouse embryonic development.\n",
  "wordCount" : "2527",
  "inLanguage": "en",
  "datePublished": "2024-02-11T10:38:55Z",
  "dateModified": "2024-02-11T10:38:55Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 11, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579069">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579069" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579069">
        <p class="paperTitle">DNA methylation modulates nucleosome retention in sperm and H3K4 methylation deposition in early mouse embryos.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579069" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579069" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fanourgakis, G.; Gaspa-Toneu, L.; Komarov, P. A.; Ozonov, E. A.; Smallwood, S. A.; Peters, A. H. F. M.</p>
        <p class="info">Score: 18.9, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579069' target='https://doi.org/10.1101/2024.02.06.579069'> 10.1101/2024.02.06.579069</a></p>
        <p class="abstract">DNA methylation (DNAme) serves a stable gene regulatory function in somatic cells (1). In the germ line and during early embryogenesis, however, DNAme undergoes global erasure and re-establishment to support germ cell and embryonic development (2). While de novo DNAme acquisition during male germ cell development is essential for setting genomic DNA methylation imprints, other intergenerational roles for paternal DNAme in defining embryonic chromatin after fertilization are unknown. To approach this question, we reduced levels of DNAme in developing male germ cells through conditional gene deletion of the de novo DNA methyltransferases DNMT3A and DNMT3B in undifferentiated spermatogonia. We observed that DNMT3A serves a DNAme maintenance function in undifferentiated spermatogonia while DNMT3B catalyzes de novo DNAme during spermatogonial differentiation. Mutant male germ cells nevertheless completed their differentiation to sperm. Failing de novo DNAme in Dnmt3a/Dnmt3b double deficient spermatogonia is associated with increased nucleosome occupancy in mature sperm, preferentially at sites with higher CpG content, supporting the model that DNAme modulates nucleosome retention in sperm (3). To assess the impact of altered sperm chromatin in the formation of embryonic chromatin, we measured H3K4me3 occupancy at paternal and maternal alleles in 2-cell embryos using a newly developed transposon-based tagging assay for modified chromatin. Our data show that reduced DNAme in sperm renders paternal alleles permissive for H3K4me3 establishment in early embryos, independently of possible paternal inheritance of sperm born H3K4me3. Together, this study provides first evidence that paternally inherited DNAme directs chromatin formation during early embryonic development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579360">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579360" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579360">
        <p class="paperTitle">Developmental disruption of the mitochondrial fissiongene drp-1 extends the longevity of daf-2 insulin/IGF-1 receptor mutant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579360" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579360" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Traa, A.; Van Raamsdonk, J.</p>
        <p class="info">Score: 18.2, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579360' target='https://doi.org/10.1101/2024.02.07.579360'> 10.1101/2024.02.07.579360</a></p>
        <p class="abstract">The dynamic nature of the mitochondrial network is regulated by mitochondrial fission and fusion, allowing for re-organization of mitochondria to adapt to the cell&#39;s ever-changing needs. As organisms age, mitochondrial fission and fusion become dysregulated and mitochondrial networks become increasingly fragmented. Modulation of mitochondrial dynamics has been shown to affect longevity in fungi, yeast, Drosophila and C. elegans. While disruption of the mitochondrial fission gene drp-1 only mildly increases wild-type lifespan, it drastically increases the already long lifespan of daf-2 insulin/IGF-1 signaling (IIS) mutants. In this work, we determined the conditions required for drp-1 disruption to extend daf-2 longevity and explored the molecular mechanisms involved. We found that knockdown of drp-1 during development is sufficient to extend daf-2 lifespan, while tissue-specific knockdown of drp-1 in neurons, intestine or muscle failed to increase daf-2 longevity. Disruption of other genes involved in mitochondrial fission also increased daf-2 lifespan as did treatment with a number of different RNAi clones that decrease mitochondrial fragmentation. In exploring potential mechanisms involved, we found that deletion of drp-1 increases resistance to chronic stresses and slows physiologic rates in daf-2 worms. In addition, we found that disruption of drp-1 increased mitochondrial and peroxisomal connectedness in daf-2 worms, increased oxidative phosphorylation and ATP levels, and increased mitophagy in daf-2 worms, but did not affect their ROS levels or mitochondrial membrane potential. Overall, this work defined the conditions under which drp-1 disruption increases daf-2 lifespan and has identified multiple changes in daf-2;drp-1 mutants that may contribute to their lifespan extension.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574922">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574922" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574922">
        <p class="paperTitle">A Genetic Screen for regulators of CNS morphology in Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574922" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574922" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lacin, H.; Zhu, Y.; DiPaola, J.; Wilson, B. A.; Zhu, Y.; Skeath, J. B.</p>
        <p class="info">Score: 8.9, Published: 2024-02-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574922' target='https://doi.org/10.1101/2024.01.09.574922'> 10.1101/2024.01.09.574922</a></p>
        <p class="abstract">Despite increasing in mass approximately 100-fold during larval life, the Drosophila CNS maintains its characteristic form during this rapid growth phase. Dynamic interactions between the overlying basement membrane and underlying surface glia are known to regulate CNS structure in Drosophila, but the genes and pathways that establish and maintain CNS morphology during development remain poorly characterized. To identify genes that regulate CNS shape in Drosophila, we conducted an EMS-based, forward genetic screen of the second chromosome, uncovered 50 mutations that disrupt CNS structure, and mapped these alleles to 17 genes. Whole genome sequencing revealed the affected gene for all but one gene. Identified genes include well characterized regulators of tissue shape, like Laminin B1, viking, and Collagen 4a1, as well as characterized genes, such as Toll-2 and Rme-8, with no known role in regulating CNS structure. We also uncovered that papilin and dC1GalTA likely act in the same pathway to regulate CNS structure and found that a glucuronosyltransferase that regulates Dystroglycan function in mammals is required to maintain CNS shape in Drosophila. Finally, we show that the senseless-2 transcription factor is expressed and functions specifically in surface glia found on peripheral nerves but not those on the CNS proper to govern CNS structure, identifying the first gene that functionally subdivides a glial subtype along the peripheral-central axis. Future work on these genes should help clarify the genetic mechanisms that ensure the homeostasis of CNS shape and form during development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579142">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579142" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579142">
        <p class="paperTitle">Limitations of the inference of the distribution of fitness effects of new mutations in partially selfing populations with linkage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579142" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579142" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daigle, A.; Johri, P.</p>
        <p class="info">Score: 7.8, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579142' target='https://doi.org/10.1101/2024.02.06.579142'> 10.1101/2024.02.06.579142</a></p>
        <p class="abstract">The accurate estimation of the distribution of fitness effects (DFE) of new mutations is critical for population genetic inference but remains a challenging task. While various methods have been developed for DFE inference using the site frequency spectrum of putatively neutral and selected sites, their applicability in species with diverse life history traits and complex demographic scenarios is not well understood. Selfing is common among eukaryotic species and can lead to decreased effective recombination rates in such populations, increasing the effects of selection at linked sites, including interference between selected alleles. We employ forward simulations to investigate the limitations of current DFE estimation approaches in the presence of selfing and linked effects of selection. We find that distortions of the site frequency spectrum due to Hill-Robertson interference in highly selfing populations lead to mis-inference of the deleterious DFE of new mutations. While accounting for the decrease in the effective population size due to linked effects of selection largely accounts for the observed bias in populations with moderate levels of selfing, this correction is unable to accurately estimate the DFE in highly selfing populations. In addition, the presence of cryptic population structure and uneven sampling across subpopulations leads to the false inference of a deleterious DFE skewed towards effectively neutral/mildly deleterious mutations. Finally, the proportion of adaptive substitutions estimated at high rates of selfing is substantially overestimated. Our results clarify the parameter space where current DFE methods might be problematic and where they remain robust in the presence of selfing and other model violations, such as departures from semidominance, population structure, and uneven sampling. Our observations apply broadly to species and genomic regions with little/no recombination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.577593">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.577593" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.577593">
        <p class="paperTitle">Helicase-assisted continuous editing for programmable mutagenesis of endogenous genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.577593" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.577593" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, X. D.; Chen, Z.; Wythes, G.; Zhang, Y.; Orr, B. C.; Sun, G.; Thao, K.; Vallurupalli, M.; Sun, J.; Borji, M.; Tkacik, E.; Chen, H.; Bernstein, B. E.; Chen, F.</p>
        <p class="info">Score: 30.4, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.577593' target='https://doi.org/10.1101/2024.02.01.577593'> 10.1101/2024.02.01.577593</a></p>
        <p class="abstract">A major challenge in human genomics is to decipher the context specific relationship of sequence to function. However, existing tools for locus specific hypermutation and evolution in the native genome context are limited. Here we present a novel programmable platform for long-range, locus-specific hypermutation called helicase-assisted continuous editing (HACE). HACE leverages CRISPR-Cas9 to target a processive helicase-deaminase fusion that incurs mutations across large (&gt;1000 bp) genomic intervals. We applied HACE to identify mutations in MEK1 that confer kinase inhibitor resistance, to dissect the impact of individual variants in SF3B1-dependent mis-splicing, and to evaluate noncoding variants in a stimulation-dependent immune enhancer of CD69. HACE provides a powerful tool for investigating coding and noncoding variants, uncovering combinatorial sequence-to-function relationships, and evolving new biological functions.

One Sentence SummaryWe developed a tool for continuous, long-range, targeted diversification of endogenous mammalian genomes and used it to explore the function of genetic variants in both coding and non-coding regions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.579041">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.579041" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.579041">
        <p class="paperTitle">The genetic landscape, gene flow and adaptation in Ladakh highlanders of trans-Himalayas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.579041" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.579041" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumar, L.; Rajpal, R.; Ahlawat, B.; Pasupuleti, N.; Konar, S.; Dwivedi, A.; Kumar, S.; Spalzin, S.; Rabyang, S.; Thangaraj, K.; Rai, N.</p>
        <p class="info">Score: 6.6, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.579041' target='https://doi.org/10.1101/2024.02.05.579041'> 10.1101/2024.02.05.579041</a></p>
        <p class="abstract">The Ladakh in the trans-Himalayas has witnessed complex cultural movements, genetic assimilation and demographic changes since the Neolithic period, which continue despite the harsh, inhospitable and cold climate. Although geographically isolated from mainland South Asia, Ladakh has historic trade routes (the Silk Road) and is well connected and accessible to travelers from Tibet and Central Asia. Despite its rich cultural heritage, a detailed description of the genetic landscape of the Ladakh region is completely lacking, particularly with regard to genome-wide markers and larger sample sizes. Therefore, in our current study, we genotyped 80 individuals from Kargil and Leh districts of the Union Territory of Ladakh, India. Here, we performed a comprehensive genetic analysis based on allele frequency and haplotype sharing. Our analysis revealed the presence of two distinct genetic lineages in the region with quite different genetic composition. The population of the Leh region is more similar to East Asian and Southeast Asian populations with population drift. In contrast, the population of the Kargil region is more similar to the Indo-European population. The former group shares a genetic history with Tibetans and Sherpas, while the latter group from Kargil associates with Kashmiri Muslims, Gujjars and Nepalese Brahmins. For both groups, there was a founder event at about 18th to 22th generations ago and duration coincided with the Mughal invasion. The genome-wide scan for a signal of positive selection revealed genetic signatures of altitude adaptation in the Leh population, while in the Kargil population the key gene signatures were associated with immunity and female fertility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577217" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577217">
        <p class="paperTitle">Genetic and bioinformatic analyses reveal transcriptional networks underlying dual genomic coordination of mitochondrial biogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, H.</p>
        <p class="info">Score: 7.7, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577217' target='https://doi.org/10.1101/2024.01.25.577217'> 10.1101/2024.01.25.577217</a></p>
        <p class="abstract">Mitochondrial genome encodes handful genes of respiratory chain complexes, whereas all the remaining mitochondrial proteins are encoded on the nuclear genome. However, the mechanisms coordinating these two genomes to control mitochondrial biogenesis remain largely unknown. To identify transcription circuits involved in these processes, we performed a candidate RNAi screen in developing eyes that had reduced mitochondrial DNA contents. We reasoned that impaired mitochondrial biogenesis would synergistically interact with mtDNA deficiency in disrupting tissue development. Over 638 transcription factors annotated in the fly genome, we identified 77 transcription factors that may be involved in mitochondrial genome maintenance and gene expression. Additional genetic and genomic analyses revealed that a novel transcription factor, CG1603, and its upstream factor YL-1 are essential for mitochondrial biogenesis. We constructed a regulator network among positive hits using the published CHIP-seq data. The network analysis revealed extensive connections, and complex hierarchical organization underlying the transcription regulation of mitochondrial biogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577071">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577071" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577071">
        <p class="paperTitle">Somatic epigenetic drift during shoot branching: a cell lineage-based model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577071" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577071" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Y.; Burian, A.; Johannes, F.</p>
        <p class="info">Score: 9.0, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577071' target='https://doi.org/10.1101/2024.01.24.577071'> 10.1101/2024.01.24.577071</a></p>
        <p class="abstract">Plant architecture is shaped by the continuous production of new organs, most of which emerge post-embryonically. This process includes the formation of new lateral branches along existing shoots. Shoot branching is fundamental to plant development, plant-environment interactions, and vegetative propagation. Current empirical evidence supports a &#34;detached meristem&#34; model as the cellular basis of lateral shoot initiation. In this model, a small number of undifferentiated cells are &#34;sampled&#34; from the periphery of the shoot apical meristem (SAM) to act as precursors for axillary buds, which eventually develop into new shoots. Repeated branching thus creates a series of cellular bottlenecks (i.e. somatic drift) that affect how de novo genetic and epigenetic mutations propagate through the plant body during development. Somatic drift could be particularly relevant for epigenetic changes in the form of stochastic DNA methylation gains and losses (i.e. spontaneous epimutations), as they have been shown to arise rapidly with each cell division.

Here, we formalize a special case of the &#34;detached meristem&#34; model, where pre-cursor cells are randomly sampled from the SAM periphery in a way that maximizes cell lineage independence. By following a population of SAM cells through repeated branching processes, we show that somatic drift gives rise to a complex mixture of cellular phylogenies, which shape the evolution of cell-to-cell DNA methylation heterogeneity within the SAM over time. This process is dependent on the number of branch points, the strength of somatic drift as well as the epimutation rate. For many realistic cell biological settings, our model predicts that cell-to-cell DNA methylation heterogeneity in the SAM converges to non-zero states during development, suggesting that epigenetic variation is an inherent property of the SAM cell population.

Our insights have direct implications for empirical studies of somatic (epi)genomic diversity in long-lived perennial and clonal species using bulk or single-cell sequencing approaches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577254">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577254" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577254">
        <p class="paperTitle">Patient derived model of UBA5-associated encephalopathy identifies defects in neurodevelopment and highlights potential therapies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577254" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577254" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, H.; Wang, Y.-D.; Blan, A. W.; Almanza-Fuerte, E. P.; Bonkowski, E. S.; Bajpai, R.; Pruett-Miller, S. M.; Mefford, H. C.</p>
        <p class="info">Score: 21.2, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577254' target='https://doi.org/10.1101/2024.01.25.577254'> 10.1101/2024.01.25.577254</a></p>
        <p class="abstract">UBA5 encodes for the E1 enzyme of the UFMylation cascade, which plays an essential role in ER homeostasis. The clinical phenotypes of UBA5-associated encephalopathy include developmental delays, epilepsy and intellectual disability. To date, there is no humanized neuronal model to study the cellular and molecular consequences of UBA5 pathogenic variants. We developed and characterized patient-derived cortical organoid cultures and identified defects in GABAergic interneuron development. We demonstrated aberrant neuronal firing and microcephaly phenotypes in patient-derived organoids. Mechanistically, we show that ER homeostasis is perturbed along with exacerbated unfolded protein response pathway in cells and organoids expressing UBA5 pathogenic variants. We also assessed two gene expression modalities that augmented UBA5 expression to rescue aberrant molecular and cellular phenotypes. Our study provides a novel humanized model that allows further investigations of UBA5 variants in the brain and highlights novel systemic approaches to alleviate cellular aberrations for this rare, developmental disorder.

One-sentence summaryPatient derived model of UBA5-assoicated DEE recapitulated disease phenotype, revealed defects in neurodevelopment, and highlighted potential therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575263">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575263" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575263">
        <p class="paperTitle">The novel linkage between Fuz and Gpr161 genes regulates sonic hedgehog signaling during mouse embryonic development.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575263" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575263" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, S. E.; Kim, H. Y.; Wlodarczyk, B. J.; Finnell, R. H.</p>
        <p class="info">Score: 8.2, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575263' target='https://doi.org/10.1101/2024.01.11.575263'> 10.1101/2024.01.11.575263</a></p>
        <p class="abstract">Sonic hedgehog (Shh) signaling regulates embryonic morphogenesis utilizing primary cilia, the cell antenna acting as a signaling hub. Fuz, an effector of planar cell polarity (PCP) signaling, involves Shh signaling via cilia formation, while the G protein-coupled receptor 161 (Gpr161) is a negative regulator of Shh signaling. The range of phenotypic malformations observed in mice bearing mutations in either of these two genes is similar; however, their functional relations have not been previously explored. This study identified the genetic and biochemical link between Fuz and Gpr161 in mouse embryonic development. Fuz was genetically epistatic to Gpr161 via Shh signaling during mouse embryonic development. The FUZ biochemically interacted with GPR161, and Fuz regulated Gpr161 ciliary trafficking via {beta}-arrestin2. Our study suggested the novel Gpr161-Fuz axis that regulates Shh signaling during mouse embryonic development.

Summary statementThis study illuminates the novel genetic and biochemical linkages between Fuz and Gpr161 to regulate sonic hedgehog signaling during mouse embryonic development.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
