<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Pervasive biases in proxy GWAS based on parental history of Alzheimer&#39;s disease
Authors: Wu, Y.; Sun, Z.; Zheng, Q.; Miao, J.; Dorn, S.; Mukherjee, S.; Fletcher, J.; Lu, Q.
Score: 26.9, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562272
Almost every recent Alzheimer&#39;s disease (AD) genome-wide association study (GWAS) has performed meta-analysis to combine studies with clinical diagnosis of AD with studies that use proxy phenotypes based on parental disease history. Here, we report major limitations in current GWAS-by-proxy (GWAX) practices due to uncorrected survival bias and non-random participation of parental illness survey, which cause substantial discrepancies between AD GWAS and GWAX results.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Pervasive biases in proxy GWAS based on parental history of Alzheimer&#39;s disease
Authors: Wu, Y.; Sun, Z.; Zheng, Q.; Miao, J.; Dorn, S.; Mukherjee, S.; Fletcher, J.; Lu, Q.
Score: 26.9, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562272
Almost every recent Alzheimer&#39;s disease (AD) genome-wide association study (GWAS) has performed meta-analysis to combine studies with clinical diagnosis of AD with studies that use proxy phenotypes based on parental disease history. Here, we report major limitations in current GWAS-by-proxy (GWAX) practices due to uncorrected survival bias and non-random participation of parental illness survey, which cause substantial discrepancies between AD GWAS and GWAX results." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-18T10:38:41+00:00" />
<meta property="article:modified_time" content="2023-10-18T10:38:41+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Pervasive biases in proxy GWAS based on parental history of Alzheimer&#39;s disease
Authors: Wu, Y.; Sun, Z.; Zheng, Q.; Miao, J.; Dorn, S.; Mukherjee, S.; Fletcher, J.; Lu, Q.
Score: 26.9, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562272
Almost every recent Alzheimer&#39;s disease (AD) genome-wide association study (GWAS) has performed meta-analysis to combine studies with clinical diagnosis of AD with studies that use proxy phenotypes based on parental disease history. Here, we report major limitations in current GWAS-by-proxy (GWAX) practices due to uncorrected survival bias and non-random participation of parental illness survey, which cause substantial discrepancies between AD GWAS and GWAX results."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Pervasive biases in proxy GWAS based on parental history of Alzheimer\u0026#39;s disease\nAuthors: Wu, Y.; Sun, Z.; Zheng, Q.; Miao, J.; Dorn, S.; Mukherjee, S.; Fletcher, J.; Lu, Q.\nScore: 26.9, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562272\nAlmost every recent Alzheimer\u0026#39;s disease (AD) genome-wide association study (GWAS) has performed meta-analysis to combine studies with clinical diagnosis of AD with studies that use proxy phenotypes based on parental disease history. Here, we report major limitations in current GWAS-by-proxy (GWAX) practices due to uncorrected survival bias and non-random participation of parental illness survey, which cause substantial discrepancies between AD GWAS and GWAX results.",
  "keywords": [
    
  ],
  "articleBody": " Pervasive biases in proxy GWAS based on parental history of Alzheimer's disease\nAuthors: Wu, Y.; Sun, Z.; Zheng, Q.; Miao, J.; Dorn, S.; Mukherjee, S.; Fletcher, J.; Lu, Q.\nScore: 26.9, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562272\nAlmost every recent Alzheimer's disease (AD) genome-wide association study (GWAS) has performed meta-analysis to combine studies with clinical diagnosis of AD with studies that use proxy phenotypes based on parental disease history. Here, we report major limitations in current GWAS-by-proxy (GWAX) practices due to uncorrected survival bias and non-random participation of parental illness survey, which cause substantial discrepancies between AD GWAS and GWAX results. We demonstrate that current AD GWAX provide highly misleading genetic correlations between AD risk and higher education which subsequently affects a variety of genetic epidemiologic applications involving AD and cognition. Our study sheds important light on the design and analysis of mid-aged biobank cohorts and underscores the need for caution when interpreting genetic association results based on proxy-reported parental disease history.\nA likelihood-based framework for demographic inference from genealogical trees\nAuthors: Fan, C.; Cahoon, J. L.; Dinh, B. L.; Ortega-Del Vecchyo, D.; Huber, C. D.; Edge, M. D.; Mancuso, N.; Chiang, C. W. K.\nScore: 18.0, Published: 2023-10-13 DOI: 10.1101/2023.10.10.561787\nThe demographic history of a population drives the pattern of genetic variation and is encoded in the gene-genealogical trees of the sampled alleles. However, existing methods to infer demographic history from genetic data tend to use relatively low-dimensional summaries of the genealogy, such as allele frequency spectra. As a step toward capturing more of the information encoded in the genome-wide sequence of genealogical trees, here we propose a novel framework called the genealogical likelihood (gLike), which derives the full likelihood of a genealogical tree under any hypothesized demographic history. Employing a graph-based structure, gLike summarizes across independent trees the relationships among all lineages in a tree with all possible trajectories of population memberships through time and efficiently computes the exact marginal probability under a parameterized demographic model. Through extensive simulations and empirical applications on populations that have experienced multiple admixtures, we showed that gLike can accurately estimate dozens of demographic parameters when the true genealogy is known, including ancestral population sizes, admixture timing, and admixture proportions. Moreover, when using genealogical trees inferred from genetic data, we showed that gLike outperformed conventional demographic inference methods that leverage only the allele-frequency spectrum and yielded parameter estimates that align with established historical knowledge of the past demographic histories for populations like Latino Americans and Native Hawaiians. Furthermore, our framework can trace ancestral histories by analyzing a sample from the admixed population without proxies for its source populations, removing the need to sample ancestral populations that may no longer exist. Taken together, our proposed gLike framework harnesses underutilized genealogical information to offer exceptional sensitivity and accuracy in inferring complex demographies for humans and other species, particularly as estimation of genome-wide genealogies improves.\nSingle-cell chromatin state transitions during epigenetic memory formation\nAuthors: Fujimori, T.; Rios-Martinez, C.; Thurm, A. R.; Hinks, M. M.; Doughty, B. R.; Sinha, J.; Le, D.; Hafner, A.; Greenleaf, W. J.; Boettiger, A. N.; Bintu, L.\nScore: 19.3, Published: 2023-10-05 DOI: 10.1101/2023.10.03.560616\nRepressive chromatin modifications are thought to compact chromatin to silence transcription. However, it is unclear how chromatin structure changes during silencing and epigenetic memory formation. We measured gene expression and chromatin structure in single cells after recruitment and release of repressors at a reporter gene. Chromatin structure is heterogeneous, with open and compact conformations present in both active and silent states. Recruitment of repressors associated with epigenetic memory produces chromatin compaction across 10-20 kilobases, while reversible silencing does not cause compaction at this scale. Chromatin compaction is inherited, but changes molecularly over time from histone methylation (H3K9me3) to DNA methylation. The level of compaction at the end of silencing quantitatively predicts epigenetic memory weeks later. Similarly, chromatin compaction at the Nanog locus predicts the degree of stem-cell fate commitment. These findings suggest that the chromatin state across tens of kilobases, beyond the gene itself, is important for epigenetic memory formation.\ndiplo-locus: A lightweight toolkit for inference and simulation of time-series genetic data under general diploid selection\nAuthors: Cheng, X.; Steinruecken, M.\nScore: 7.1, Published: 2023-10-17 DOI: 10.1101/2023.10.12.562101\nSummary: Whole-genome time-series allele frequency data are becoming more prevalent as ancient DNA (aDNA) sequences and data from evolve-and-resequence (E\u0026R) experiments are generated at a rapid pace. Such data presents unprecedented opportunities to elucidate the dynamics of adaptative genetic variation. However, despite many methods to infer parameters of selection models from allele frequency trajectories available in the literature, few provide user-friendly implementations for large-scale empirical applications. Here, we present diplo-locus, an open-source Python package that provides functionality to simulate and perform inference from time-series under the Wright-Fisher diffusion with general diploid selection. The package includes Python modules as well as command-line tools. Availability: Python package and command-line tool available at: https://github.com/steinrue/ diplo_locus or https://pypi.org/project/diplo-locus/.\nCLN3 transcript complexity revealed by long-read RNA sequencing analysis\nAuthors: Zhang, H.-Y.; Minnis, C.; Gustavsson, E. K.; Ryten, M.; Mole, S. E.\nScore: 3.7, Published: 2023-10-13 DOI: 10.1101/2023.10.12.562062\nBackgroundBatten disease is a group of rare inherited neurodegenerative diseases. Juvenile CLN3 disease is the most prevalent type, and the most common mutation shared by most patients is the \"1-kb\" deletion which removes two internal coding exons (7 and 8) in CLN3. Previously, we identified two transcripts in patient fibroblasts homozygous for the \"1-kb\" deletion: the \"major\" and \"minor\" transcripts. To understand the full variety of disease transcripts and their role in disease pathogenesis, it is necessary to first investigate CLN3 transcription in \"healthy\" samples without juvenile CLN3 disease. MethodsWe leveraged PacBio long-read RNA sequencing datasets from ENCODE to investigate the full range of CLN3 transcripts across various tissues and cell types in human control samples. Then we sought to validate their existence using data from different sources. ResultsWe found that a readthrough gene affects the quantification and annotation of CLN3. After taking this into account, we detected over 100 novel CLN3 transcripts, with no dominantly expressed CLN3 transcript. The most abundant transcript has median usage of 42.9%. Surprisingly, the known disease-associated \"major\" transcripts are detected. Together, they have median usage of 1.51% across 22 samples. Furthermore, we identified 48 CLN3 ORFs, of which 26 are novel. The predominant ORF that encodes the canonical CLN3 protein isoform has median usage of 66.7%, meaning around one-third of CLN3 transcripts encode protein isoforms with different stretches of amino acids. The same ORFs could be found with alternative UTRs. Moreover, we were able to validate the translational potential of certain transcripts using public mass spectrometry data. ConclusionOverall, these findings provide valuable insights into the complexity of CLN3 transcription, highlighting the importance of studying both canonical and non-canonical CLN3 protein isoforms as well as the regulatory role of UTRs to fully comprehend the regulation and function(s) of CLN3. This knowledge is essential for investigating the impact of the \"1-kb\" deletion and rare mutations on CLN3 transcription and disease pathogenesis.\nBronze Age Northern Eurasian Genetics in the Context of Development of Metallurgy and Siberian Ancestry\nAuthors: Childebayeva, A.; Fricke, F.; Rohrlach, A. B.; Huang, L.; Schiffels, S.; Vesakoski, O.; Semerau, L.; Aron, F.; Moiseyev, V.; Khartanovich, V.; Kovtun, I.; Krause, J.; Kuzminykh, S.; Haak, W.\nScore: 21.6, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560195\nThe Eurasian Bronze Age (BA) has been described as a period of substantial human migrations, the emergence of pastoralism, horse domestication, and development of metallurgy. This study focuses on individuals associated with BA metallurgical production, specifically the Seima-Turbino (ST) phenomenon ([~]2,200-1,900 BCE) associated with elaborate metal objects found across Northern Eurasia. The genetic profiles of nine ST-associated individuals vary widely ranging between ancestries maximized in individuals from the Eastern Siberian Late Neolithic/BA, and those of the Western Steppe Middle Late BA. The genetic heterogeneity observed is consistent with the current understanding of the ST metallurgical network as a transcultural phenomenon. The new data also shed light on the temporal and spatial range of an ancient Siberian genetic ancestry component, which is shared across many Uralic-speaking populations, and which we explore further via demographic modeling using additional genome-wide (2 individuals) and whole genome data (5 individuals, including a [~]30x genome) from northwestern Russia.\nMale gonad-enriched microRNAs function to control sperm production in C. elegans\nAuthors: Lu, L.; Abbott, A. L.\nScore: 2.8, Published: 2023-10-12 DOI: 10.1101/2023.10.10.561762\nGerm cell development and gamete production in animals require small RNA pathways. While studies indicate that microRNAs (miRNAs) are necessary for normal sperm production and function, the specific roles for individual miRNAs are largely unknown. Here, we use small RNA sequencing of dissected gonads and functional analysis of new loss of function alleles to identify functions for miRNAs in the control of fecundity and sperm production in Caenorhabditis elegans males and hermaphrodites. We describe a set of 29 male gonad-enriched miRNAs and identify a set of 3 individual miRNAs (mir-58.1, mir-83, and mir-235) and a miRNA cluster (mir-4807-4810.1) that are required for optimal sperm production at 20{degrees}C and 5 additional miRNAs (mir-49, mir-57, mir-261, and mir-357/358) that are required for sperm production at 25{degrees}C. We observed defects in meiotic progression in mir-58.1, mir-83, mir-235, and mir-4807-4810.1 mutants that may contribute to the reduced number of sperm. Further, analysis of multiple mutants of these miRNAs suggested complex genetic interactions between these miRNAs for sperm production. This study provides insights on the regulatory roles of miRNAs that promote optimal sperm production and fecundity in males and hermaphrodites. Article SummaryMicroRNAs are small non-coding RNAs that are required for the normal production of sperm but the roles of individual microRNAs in the process of spermatogenesis are not well understood. Here, we use the nematode Caenorhabditis elegans to identify microRNAs that are enriched in the male gonad to identify specific microRNAs that regulate male fertility. We generated new loss of function mutants for functional analysis to identify a set of microRNAs that are necessary for optimal fertility and fecundity in males.\nSynchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition\nAuthors: Vollger, M. R.; Korlach, J.; Eldred, K. C.; Swanson, E.; Underwood, J. G.; Munson, K. M.; Cheng, Y.-H. H.; Ranchalis, J.; Mao, Y.; Blue, E. E.; Schwarze, U.; Saunders, C. T.; Wenger, A. M.; Allworth, A.; Chanprasert, S.; Duerden, B. L.; Glass, I.; Horike-Pyne, M.; Kim, M.; Leppig, K. A.; McLaughlin, I. J.; Ogawa, J.; Rosenthal, E. A.; Sheppeard, S.; Sherman, S. M.; Strohbehn, S.; Yuen, A. L.; Reh, T. A.; Byers, P. H.; Bamshad, M. J.; Hisama, F. M.; Jarvik, G. P.; Sancak, Y.; Dipple, K. M.; Stergachis, A. B.\nScore: 39.4, Published: 2023-09-27 DOI: 10.1101/2023.09.26.559521\nResolving the molecular basis of a Mendelian condition (MC) remains challenging owing to the diverse mechanisms by which genetic variants cause disease. To address this, we developed a synchronized long-read genome, methylome, epigenome, and transcriptome sequencing approach, which enables accurate single-nucleotide, insertion-deletion, and structural variant calling and diploid de novo genome assembly, and permits the simultaneous elucidation of haplotype-resolved CpG methylation, chromatin accessibility, and full-length transcript information in a single long-read sequencing run. Application of this approach to an Undiagnosed Diseases Network (UDN) participant with a chromosome X;13 balanced translocation of uncertain significance revealed that this translocation disrupted the functioning of four separate genes (NBEA, PDK3, MAB21L1, and RB1) previously associated with single-gene MCs. Notably, the function of each gene was disrupted via a distinct mechanism that required integration of the four omes to resolve. These included nonsense-mediated decay, fusion transcript formation, enhancer adoption, transcriptional readthrough silencing, and inappropriate X chromosome inactivation of autosomal genes. Overall, this highlights the utility of synchronized long-read multi-omic profiling for mechanistically resolving complex phenotypes.\nCleave and Rescue gamete killers create conditions for gene drive in plants\nAuthors: Oberhofer, G.; Johnson, M.; Ivy, T.; Hay, B.\nScore: 2.2, Published: 2023-10-14 DOI: 10.1101/2023.10.13.562303\nGene drive elements promote the spread of linked traits, even when their presence confers a fitness cost to carriers, and can be used to change the composition or fate of wild populations. Cleave and Rescue (ClvR) drive elements sit at a fixed chromosomal position and include a DNA sequence-modifying enzyme such as Cas9/gRNAs (the Cleaver/Toxin) that disrupts endogenous versions of an essential gene, and a recoded version of the essential gene resistant to cleavage (the Rescue/Antidote). ClvR spreads by creating conditions in which those lacking ClvR die because they lack functional versions of the essential gene. We demonstrate the essential features of ClvR gene drive in the plant Arabidopsis thaliana through killing of gametes that fail to inherit a ClvR that targets the essential gene YKT61, whose expression is required in male and female gametes for their survival. Resistant (uncleavable but functional) alleles, which can slow or prevent drive, were not observed. Modeling shows plant ClvRs can be used to rapidly drive population modification or suppression. Possible applications in weed control, plant breeding and conservation are discussed.\nStable population structure in Europe since the Iron Age, despite high mobility\nAuthors: Antonio, M. L.; Weiss, C. L.; Gao, Z.; Sawyer, S.; Oberreiter, V.; Moots, H. M.; Spence, J. P.; Cheronet, O.; Zagorc, B.; Praxmarer, E.; Ozdogan, K. T.; Demetz, L.; Gelabert, P.; Fernandes, D.; Lucci, M.; Alihodzic, T.; Amrani, S.; Avetisyan, P.; Baillif-Ducros, C.; Bedic, Z.; Bertrand, A.; Bilic, M.; Bondioli, L.; Borowka, P.; Botte, E.; Burmaz, J.; Buzanic, D.; Candilio, F.; Cvetko, M.; De Angelis, D.; Drnic, I.; Elschek, K.; Fantar, M.; Gaspari, A.; Gasperetti, G.; Genchi, F.; Golubovic, S.; Hukelova, Z.; Jankauskas, R.; Vuckovic, K. J.; Jeremic, G.; Kaic, I.; Kazek, K.; Khachatryan, H.; Kh\nScore: 173.2, Published: 2023-09-21 DOI: 10.1101/2022.05.15.491973\nAncient DNA research in the past decade has revealed that European population structure changed dramatically in the prehistoric period (14,000-3,000 years before present, YBP), reflecting the widespread introduction of Neolithic farmer and Bronze Age Steppe ancestries. However, little is known about how population structure changed from the historical period onward (3,000 YBP - present). To address this, we collected whole genomes from 204 individuals from Europe and the Mediterranean, many of which are the first historical period genomes from their region (e.g. Armenia and France). We found that most regions show remarkable inter-individual heterogeneity. At least 7% of historical individuals carry ancestry uncommon in the region where they were sampled, some indicating cross-Mediterranean contacts. Despite this high level of mobility, overall population structure across western Eurasia is relatively stable through the historical period up to the present, mirroring geography. We show that, under standard population genetics models with local panmixia, the observed level of dispersal would lead to a collapse of population structure. Persistent population structure thus suggests a lower effective migration rate than indicated by the observed dispersal. We hypothesize that this phenomenon can be explained by extensive transient dispersal arising from drastically improved transportation networks and the Roman Empires mobilization of people for trade, labor, and military. This work highlights the utility of ancient DNA in elucidating finer scale human population dynamics in recent history.\n",
  "wordCount" : "2471",
  "inLanguage": "en",
  "datePublished": "2023-10-18T10:38:41Z",
  "dateModified": "2023-10-18T10:38:41Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on October 18, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562272">
        <p class="paperTitle">Pervasive biases in proxy GWAS based on parental history of Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, Y.; Sun, Z.; Zheng, Q.; Miao, J.; Dorn, S.; Mukherjee, S.; Fletcher, J.; Lu, Q.</p>
        <p class="info">Score: 26.9, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562272' target='https://doi.org/10.1101/2023.10.13.562272'> 10.1101/2023.10.13.562272</a></p>
        <p class="abstract">Almost every recent Alzheimer&#39;s disease (AD) genome-wide association study (GWAS) has performed meta-analysis to combine studies with clinical diagnosis of AD with studies that use proxy phenotypes based on parental disease history. Here, we report major limitations in current GWAS-by-proxy (GWAX) practices due to uncorrected survival bias and non-random participation of parental illness survey, which cause substantial discrepancies between AD GWAS and GWAX results. We demonstrate that current AD GWAX provide highly misleading genetic correlations between AD risk and higher education which subsequently affects a variety of genetic epidemiologic applications involving AD and cognition. Our study sheds important light on the design and analysis of mid-aged biobank cohorts and underscores the need for caution when interpreting genetic association results based on proxy-reported parental disease history.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561787">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561787" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561787">
        <p class="paperTitle">A likelihood-based framework for demographic inference from genealogical trees</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561787" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561787" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fan, C.; Cahoon, J. L.; Dinh, B. L.; Ortega-Del Vecchyo, D.; Huber, C. D.; Edge, M. D.; Mancuso, N.; Chiang, C. W. K.</p>
        <p class="info">Score: 18.0, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561787' target='https://doi.org/10.1101/2023.10.10.561787'> 10.1101/2023.10.10.561787</a></p>
        <p class="abstract">The demographic history of a population drives the pattern of genetic variation and is encoded in the gene-genealogical trees of the sampled alleles. However, existing methods to infer demographic history from genetic data tend to use relatively low-dimensional summaries of the genealogy, such as allele frequency spectra. As a step toward capturing more of the information encoded in the genome-wide sequence of genealogical trees, here we propose a novel framework called the genealogical likelihood (gLike), which derives the full likelihood of a genealogical tree under any hypothesized demographic history. Employing a graph-based structure, gLike summarizes across independent trees the relationships among all lineages in a tree with all possible trajectories of population memberships through time and efficiently computes the exact marginal probability under a parameterized demographic model. Through extensive simulations and empirical applications on populations that have experienced multiple admixtures, we showed that gLike can accurately estimate dozens of demographic parameters when the true genealogy is known, including ancestral population sizes, admixture timing, and admixture proportions. Moreover, when using genealogical trees inferred from genetic data, we showed that gLike outperformed conventional demographic inference methods that leverage only the allele-frequency spectrum and yielded parameter estimates that align with established historical knowledge of the past demographic histories for populations like Latino Americans and Native Hawaiians. Furthermore, our framework can trace ancestral histories by analyzing a sample from the admixed population without proxies for its source populations, removing the need to sample ancestral populations that may no longer exist. Taken together, our proposed gLike framework harnesses underutilized genealogical information to offer exceptional sensitivity and accuracy in inferring complex demographies for humans and other species, particularly as estimation of genome-wide genealogies improves.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560616">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560616" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560616">
        <p class="paperTitle">Single-cell chromatin state transitions during epigenetic memory formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560616" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560616" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fujimori, T.; Rios-Martinez, C.; Thurm, A. R.; Hinks, M. M.; Doughty, B. R.; Sinha, J.; Le, D.; Hafner, A.; Greenleaf, W. J.; Boettiger, A. N.; Bintu, L.</p>
        <p class="info">Score: 19.3, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560616' target='https://doi.org/10.1101/2023.10.03.560616'> 10.1101/2023.10.03.560616</a></p>
        <p class="abstract">Repressive chromatin modifications are thought to compact chromatin to silence transcription. However, it is unclear how chromatin structure changes during silencing and epigenetic memory formation. We measured gene expression and chromatin structure in single cells after recruitment and release of repressors at a reporter gene. Chromatin structure is heterogeneous, with open and compact conformations present in both active and silent states. Recruitment of repressors associated with epigenetic memory produces chromatin compaction across 10-20 kilobases, while reversible silencing does not cause compaction at this scale. Chromatin compaction is inherited, but changes molecularly over time from histone methylation (H3K9me3) to DNA methylation. The level of compaction at the end of silencing quantitatively predicts epigenetic memory weeks later. Similarly, chromatin compaction at the Nanog locus predicts the degree of stem-cell fate commitment. These findings suggest that the chromatin state across tens of kilobases, beyond the gene itself, is important for epigenetic memory formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.562101">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.562101" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.562101">
        <p class="paperTitle">diplo-locus: A lightweight toolkit for inference and simulation of time-series genetic data under general diploid selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.562101" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.562101" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cheng, X.; Steinruecken, M.</p>
        <p class="info">Score: 7.1, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.562101' target='https://doi.org/10.1101/2023.10.12.562101'> 10.1101/2023.10.12.562101</a></p>
        <p class="abstract">Summary: Whole-genome time-series allele frequency data are becoming more prevalent as ancient DNA (aDNA) sequences and data from evolve-and-resequence (E&amp;R) experiments are generated at a rapid pace. Such data presents unprecedented opportunities to elucidate the dynamics of adaptative genetic variation. However, despite many methods to infer parameters of selection models from allele frequency trajectories available in the literature, few provide user-friendly implementations for large-scale empirical applications. Here, we present diplo-locus, an open-source Python package that provides functionality to simulate and perform inference from time-series under the Wright-Fisher diffusion with general diploid selection. The package includes Python modules as well as command-line tools. Availability: Python package and command-line tool available at: https://github.com/steinrue/ diplo_locus or https://pypi.org/project/diplo-locus/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.562062">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.562062" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.562062">
        <p class="paperTitle">CLN3 transcript complexity revealed by long-read RNA sequencing analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.562062" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.562062" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, H.-Y.; Minnis, C.; Gustavsson, E. K.; Ryten, M.; Mole, S. E.</p>
        <p class="info">Score: 3.7, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.562062' target='https://doi.org/10.1101/2023.10.12.562062'> 10.1101/2023.10.12.562062</a></p>
        <p class="abstract">BackgroundBatten disease is a group of rare inherited neurodegenerative diseases. Juvenile CLN3 disease is the most prevalent type, and the most common mutation shared by most patients is the &#34;1-kb&#34; deletion which removes two internal coding exons (7 and 8) in CLN3. Previously, we identified two transcripts in patient fibroblasts homozygous for the &#34;1-kb&#34; deletion: the &#34;major&#34; and &#34;minor&#34; transcripts. To understand the full variety of disease transcripts and their role in disease pathogenesis, it is necessary to first investigate CLN3 transcription in &#34;healthy&#34; samples without juvenile CLN3 disease.

MethodsWe leveraged PacBio long-read RNA sequencing datasets from ENCODE to investigate the full range of CLN3 transcripts across various tissues and cell types in human control samples. Then we sought to validate their existence using data from different sources.

ResultsWe found that a readthrough gene affects the quantification and annotation of CLN3. After taking this into account, we detected over 100 novel CLN3 transcripts, with no dominantly expressed CLN3 transcript. The most abundant transcript has median usage of 42.9%. Surprisingly, the known disease-associated &#34;major&#34; transcripts are detected. Together, they have median usage of 1.51% across 22 samples. Furthermore, we identified 48 CLN3 ORFs, of which 26 are novel. The predominant ORF that encodes the canonical CLN3 protein isoform has median usage of 66.7%, meaning around one-third of CLN3 transcripts encode protein isoforms with different stretches of amino acids. The same ORFs could be found with alternative UTRs. Moreover, we were able to validate the translational potential of certain transcripts using public mass spectrometry data.

ConclusionOverall, these findings provide valuable insights into the complexity of CLN3 transcription, highlighting the importance of studying both canonical and non-canonical CLN3 protein isoforms as well as the regulatory role of UTRs to fully comprehend the regulation and function(s) of CLN3. This knowledge is essential for investigating the impact of the &#34;1-kb&#34; deletion and rare mutations on CLN3 transcription and disease pathogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560195">
        <p class="paperTitle">Bronze Age Northern Eurasian Genetics in the Context of Development of Metallurgy and Siberian Ancestry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Childebayeva, A.; Fricke, F.; Rohrlach, A. B.; Huang, L.; Schiffels, S.; Vesakoski, O.; Semerau, L.; Aron, F.; Moiseyev, V.; Khartanovich, V.; Kovtun, I.; Krause, J.; Kuzminykh, S.; Haak, W.</p>
        <p class="info">Score: 21.6, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560195' target='https://doi.org/10.1101/2023.10.01.560195'> 10.1101/2023.10.01.560195</a></p>
        <p class="abstract">The Eurasian Bronze Age (BA) has been described as a period of substantial human migrations, the emergence of pastoralism, horse domestication, and development of metallurgy. This study focuses on individuals associated with BA metallurgical production, specifically the Seima-Turbino (ST) phenomenon ([~]2,200-1,900 BCE) associated with elaborate metal objects found across Northern Eurasia. The genetic profiles of nine ST-associated individuals vary widely ranging between ancestries maximized in individuals from the Eastern Siberian Late Neolithic/BA, and those of the Western Steppe Middle Late BA. The genetic heterogeneity observed is consistent with the current understanding of the ST metallurgical network as a transcultural phenomenon. The new data also shed light on the temporal and spatial range of an ancient Siberian genetic ancestry component, which is shared across many Uralic-speaking populations, and which we explore further via demographic modeling using additional genome-wide (2 individuals) and whole genome data (5 individuals, including a [~]30x genome) from northwestern Russia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561762">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561762" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561762">
        <p class="paperTitle">Male gonad-enriched microRNAs function to control sperm production in C. elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561762" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561762" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, L.; Abbott, A. L.</p>
        <p class="info">Score: 2.8, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561762' target='https://doi.org/10.1101/2023.10.10.561762'> 10.1101/2023.10.10.561762</a></p>
        <p class="abstract">Germ cell development and gamete production in animals require small RNA pathways. While studies indicate that microRNAs (miRNAs) are necessary for normal sperm production and function, the specific roles for individual miRNAs are largely unknown. Here, we use small RNA sequencing of dissected gonads and functional analysis of new loss of function alleles to identify functions for miRNAs in the control of fecundity and sperm production in Caenorhabditis elegans males and hermaphrodites. We describe a set of 29 male gonad-enriched miRNAs and identify a set of 3 individual miRNAs (mir-58.1, mir-83, and mir-235) and a miRNA cluster (mir-4807-4810.1) that are required for optimal sperm production at 20{degrees}C and 5 additional miRNAs (mir-49, mir-57, mir-261, and mir-357/358) that are required for sperm production at 25{degrees}C. We observed defects in meiotic progression in mir-58.1, mir-83, mir-235, and mir-4807-4810.1 mutants that may contribute to the reduced number of sperm. Further, analysis of multiple mutants of these miRNAs suggested complex genetic interactions between these miRNAs for sperm production. This study provides insights on the regulatory roles of miRNAs that promote optimal sperm production and fecundity in males and hermaphrodites.

Article SummaryMicroRNAs are small non-coding RNAs that are required for the normal production of sperm but the roles of individual microRNAs in the process of spermatogenesis are not well understood. Here, we use the nematode Caenorhabditis elegans to identify microRNAs that are enriched in the male gonad to identify specific microRNAs that regulate male fertility. We generated new loss of function mutants for functional analysis to identify a set of microRNAs that are necessary for optimal fertility and fecundity in males.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559521">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559521" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559521">
        <p class="paperTitle">Synchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559521" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559521" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vollger, M. R.; Korlach, J.; Eldred, K. C.; Swanson, E.; Underwood, J. G.; Munson, K. M.; Cheng, Y.-H. H.; Ranchalis, J.; Mao, Y.; Blue, E. E.; Schwarze, U.; Saunders, C. T.; Wenger, A. M.; Allworth, A.; Chanprasert, S.; Duerden, B. L.; Glass, I.; Horike-Pyne, M.; Kim, M.; Leppig, K. A.; McLaughlin, I. J.; Ogawa, J.; Rosenthal, E. A.; Sheppeard, S.; Sherman, S. M.; Strohbehn, S.; Yuen, A. L.; Reh, T. A.; Byers, P. H.; Bamshad, M. J.; Hisama, F. M.; Jarvik, G. P.; Sancak, Y.; Dipple, K. M.; Stergachis, A. B.</p>
        <p class="info">Score: 39.4, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559521' target='https://doi.org/10.1101/2023.09.26.559521'> 10.1101/2023.09.26.559521</a></p>
        <p class="abstract">Resolving the molecular basis of a Mendelian condition (MC) remains challenging owing to the diverse mechanisms by which genetic variants cause disease. To address this, we developed a synchronized long-read genome, methylome, epigenome, and transcriptome sequencing approach, which enables accurate single-nucleotide, insertion-deletion, and structural variant calling and diploid de novo genome assembly, and permits the simultaneous elucidation of haplotype-resolved CpG methylation, chromatin accessibility, and full-length transcript information in a single long-read sequencing run. Application of this approach to an Undiagnosed Diseases Network (UDN) participant with a chromosome X;13 balanced translocation of uncertain significance revealed that this translocation disrupted the functioning of four separate genes (NBEA, PDK3, MAB21L1, and RB1) previously associated with single-gene MCs. Notably, the function of each gene was disrupted via a distinct mechanism that required integration of the four  omes to resolve. These included nonsense-mediated decay, fusion transcript formation, enhancer adoption, transcriptional readthrough silencing, and inappropriate X chromosome inactivation of autosomal genes. Overall, this highlights the utility of synchronized long-read multi-omic profiling for mechanistically resolving complex phenotypes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562303">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562303" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562303">
        <p class="paperTitle">Cleave and Rescue gamete killers create conditions for gene drive in plants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562303" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562303" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oberhofer, G.; Johnson, M.; Ivy, T.; Hay, B.</p>
        <p class="info">Score: 2.2, Published: 2023-10-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562303' target='https://doi.org/10.1101/2023.10.13.562303'> 10.1101/2023.10.13.562303</a></p>
        <p class="abstract">Gene drive elements promote the spread of linked traits, even when their presence confers a fitness cost to carriers, and can be used to change the composition or fate of wild populations. Cleave and Rescue (ClvR) drive elements sit at a fixed chromosomal position and include a DNA sequence-modifying enzyme such as Cas9/gRNAs (the Cleaver/Toxin) that disrupts endogenous versions of an essential gene, and a recoded version of the essential gene resistant to cleavage (the Rescue/Antidote). ClvR spreads by creating conditions in which those lacking ClvR die because they lack functional versions of the essential gene. We demonstrate the essential features of ClvR gene drive in the plant Arabidopsis thaliana through killing of gametes that fail to inherit a ClvR that targets the essential gene YKT61, whose expression is required in male and female gametes for their survival. Resistant (uncleavable but functional) alleles, which can slow or prevent drive, were not observed. Modeling shows plant ClvRs can be used to rapidly drive population modification or suppression. Possible applications in weed control, plant breeding and conservation are discussed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.05.15.491973">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.05.15.491973" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.05.15.491973">
        <p class="paperTitle">Stable population structure in Europe since the Iron Age, despite high mobility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.05.15.491973" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.05.15.491973" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Antonio, M. L.; Weiss, C. L.; Gao, Z.; Sawyer, S.; Oberreiter, V.; Moots, H. M.; Spence, J. P.; Cheronet, O.; Zagorc, B.; Praxmarer, E.; Ozdogan, K. T.; Demetz, L.; Gelabert, P.; Fernandes, D.; Lucci, M.; Alihodzic, T.; Amrani, S.; Avetisyan, P.; Baillif-Ducros, C.; Bedic, Z.; Bertrand, A.; Bilic, M.; Bondioli, L.; Borowka, P.; Botte, E.; Burmaz, J.; Buzanic, D.; Candilio, F.; Cvetko, M.; De Angelis, D.; Drnic, I.; Elschek, K.; Fantar, M.; Gaspari, A.; Gasperetti, G.; Genchi, F.; Golubovic, S.; Hukelova, Z.; Jankauskas, R.; Vuckovic, K. J.; Jeremic, G.; Kaic, I.; Kazek, K.; Khachatryan, H.; Kh</p>
        <p class="info">Score: 173.2, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.05.15.491973' target='https://doi.org/10.1101/2022.05.15.491973'> 10.1101/2022.05.15.491973</a></p>
        <p class="abstract">Ancient DNA research in the past decade has revealed that European population structure changed dramatically in the prehistoric period (14,000-3,000 years before present, YBP), reflecting the widespread introduction of Neolithic farmer and Bronze Age Steppe ancestries. However, little is known about how population structure changed from the historical period onward (3,000 YBP - present). To address this, we collected whole genomes from 204 individuals from Europe and the Mediterranean, many of which are the first historical period genomes from their region (e.g. Armenia and France). We found that most regions show remarkable inter-individual heterogeneity. At least 7% of historical individuals carry ancestry uncommon in the region where they were sampled, some indicating cross-Mediterranean contacts. Despite this high level of mobility, overall population structure across western Eurasia is relatively stable through the historical period up to the present, mirroring geography. We show that, under standard population genetics models with local panmixia, the observed level of dispersal would lead to a collapse of population structure. Persistent population structure thus suggests a lower effective migration rate than indicated by the observed dispersal. We hypothesize that this phenomenon can be explained by extensive transient dispersal arising from drastically improved transportation networks and the Roman Empires mobilization of people for trade, labor, and military. This work highlights the utility of ancient DNA in elucidating finer scale human population dynamics in recent history.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
