<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Associations between common genetic variants and income provide insights about the socioeconomic health gradient
Authors: Kweon, H.; Burik, C. A. P.; Ning, Y.; Ahlskog, R.; Xia, C.; Abner, E.; Bao, Y.; Bhatta, L.; Faquih, T.; Feijter, M.; Fisher, P.; Gelemanovic, A.; Giannelis, A.; Hottenga, J.-J.; Khalili, B.; Lee, Y.; Li-Gao, R.; Masso, J.; Myhre, R.; Palviainen, T.; Rietveld, C. A. P.; Teumer, A.; Verweij, R. M.; Willoughby, E. A.; Agerbo, E.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Associations between common genetic variants and income provide insights about the socioeconomic health gradient
Authors: Kweon, H.; Burik, C. A. P.; Ning, Y.; Ahlskog, R.; Xia, C.; Abner, E.; Bao, Y.; Bhatta, L.; Faquih, T.; Feijter, M.; Fisher, P.; Gelemanovic, A.; Giannelis, A.; Hottenga, J.-J.; Khalili, B.; Lee, Y.; Li-Gao, R.; Masso, J.; Myhre, R.; Palviainen, T.; Rietveld, C. A. P.; Teumer, A.; Verweij, R. M.; Willoughby, E. A.; Agerbo, E." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Associations between common genetic variants and income provide insights about the socioeconomic health gradient
Authors: Kweon, H.; Burik, C. A. P.; Ning, Y.; Ahlskog, R.; Xia, C.; Abner, E.; Bao, Y.; Bhatta, L.; Faquih, T.; Feijter, M.; Fisher, P.; Gelemanovic, A.; Giannelis, A.; Hottenga, J.-J.; Khalili, B.; Lee, Y.; Li-Gao, R.; Masso, J.; Myhre, R.; Palviainen, T.; Rietveld, C. A. P.; Teumer, A.; Verweij, R. M.; Willoughby, E. A.; Agerbo, E."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Associations between common genetic variants and income provide insights about the socioeconomic health gradient\nAuthors: Kweon, H.; Burik, C. A. P.; Ning, Y.; Ahlskog, R.; Xia, C.; Abner, E.; Bao, Y.; Bhatta, L.; Faquih, T.; Feijter, M.; Fisher, P.; Gelemanovic, A.; Giannelis, A.; Hottenga, J.-J.; Khalili, B.; Lee, Y.; Li-Gao, R.; Masso, J.; Myhre, R.; Palviainen, T.; Rietveld, C. A. P.; Teumer, A.; Verweij, R. M.; Willoughby, E. A.; Agerbo, E.",
  "keywords": [
    
  ],
  "articleBody": " Associations between common genetic variants and income provide insights about the socioeconomic health gradient\nAuthors: Kweon, H.; Burik, C. A. P.; Ning, Y.; Ahlskog, R.; Xia, C.; Abner, E.; Bao, Y.; Bhatta, L.; Faquih, T.; Feijter, M.; Fisher, P.; Gelemanovic, A.; Giannelis, A.; Hottenga, J.-J.; Khalili, B.; Lee, Y.; Li-Gao, R.; Masso, J.; Myhre, R.; Palviainen, T.; Rietveld, C. A. P.; Teumer, A.; Verweij, R. M.; Willoughby, E. A.; Agerbo, E.; Bergmann, S.; Boomsma, D. I.; Borglum, A. D.; Brumpton, B. M.; Davies, N. M.; Esko, T.; Gordon, S. D.; Homuth, G.; Ikram, A.; Johannesson, M.; Kaprio, J.; Kidd, M. P.; Kutalik, Z.; Kwong, A. S. F.; Lee, J.; Luik, A.; Mook-Kanamori, D. O.; Magnus, P.; Marques-Vidal, P.; M\nScore: 50.2, Published: 2024-01-10 DOI: 10.1101/2024.01.09.574865\nWe conducted a genome-wide association study (GWAS) on income among individuals of European descent and leveraged the results to investigate the socio-economic health gradient (N=668,288). We found 162 genomic loci associated with a common genetic factor underlying various income measures, all with small effect sizes. Our GWAS-derived polygenic index captures 1 - 4% of income variance, with only one-fourth attributed to direct genetic effects. A phenome-wide association study using this polygenic index showed reduced risks for a broad spectrum of diseases, including hypertension, obesity, type 2 diabetes, coronary atherosclerosis, depression, asthma, and back pain. The income factor showed a substantial genetic correlation (0.92, s.e. = .006) with educational attainment (EA). Accounting for EAs genetic overlap with income revealed that the remaining genetic signal for higher income related to better mental health but reduced physical health benefits and increased participation in risky behaviours such as drinking and smoking.\nSingle-step generation of homozygous knock-out/knock-in individuals in an extremotolerant parthenogenetic tardigrade using DIPA-CRISPR\nAuthors: Kondo, K.; Tanaka, A.; Kunieda, T.\nScore: 16.7, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575120\nTardigrades are small aquatic invertebrates known for their remarkable tolerance to diverse extreme stresses. To elucidate the in vivo mechanisms underlying this extraordinary resilience, the genetic manipulation methods in tardigrades have long been desired. Despite our prior success in somatic cell gene-editing by microinjecting Cas9 ribonucleoproteins (RNPs) into the body cavity of tardigrades, the generation of gene-edited individuals remained elusive. In this study, employing an extremotolerant parthenogenetic tardigrade species, Ramazzottius varieornatus, we established conditions conductive to generating gene-edited tardigrade individuals. Drawing inspiration from the direct parental CRIPSR (DIPA-CRISPR) technique employed in several insects, we simply injected a concentrated Cas9 RNP solution into the body cavity of parental females shortly before their initial oviposition. This approach yielded gene-edited G0 progeny. Notably, only a single allele was predominantly detected at the target locus for each G0 individual, indicative of homozygous mutations. Through co-injecting single-stranded oligodeoxynucleotides (ssODNs) with Cas9 RNPs, we achieved the generation of homozygously knocked-in G0 progeny and these edited-alleles were inherited by G1/G2 progeny. This establishment of a simple method for generating homozygous knock-out/knock-in individuals not only facilitates in vivo analyses of molecular mechanisms underpinning extreme tolerance but also opens avenues for exploring various topics, including Evo-Devo, in tardigrades.\nGene-environment correlation: The role of family environment in academic development\nAuthors: Zhou, Q.; Gidziela, A.; Allegrini, A. G.; Cheesman, R.; Wertz, J.; Maxwell, J.; Plomin, R.; Rimfeld, K.; Malanchini, M.\nScore: 21.0, Published: 2024-01-08 DOI: 10.1101/2024.01.05.574339\nAcademic achievement is partly heritable and highly polygenic. However, genetic effects on academic achievement are not independent of environmental processes. We investigated whether aspects of the family environment mediated genetic effects on academic achievement across development. Our sample included 5,151 children who participated in the Twins Early Development Study, as well as their parents and teachers. Data on academic achievement and family environments were available at ages 7, 9, 12 and 16. We computed educational attainment polygenic scores (PGS) and further separated genetic effects into cognitive and noncognitive PGS. Three core findings emerged. First, aspects of the family environment, but not the wider neighbourhood context, consistently mediated the PGS effects on achievement across development, accounting for up to 34.3% of the total effect. Family characteristics mattered beyond socio-economic status. Second, family environments were more robustly linked to noncognitive PGS effects on academic achievement than cognitive PGS effects. Third, when we investigated whether environmental mediation effects could also be observed when considering differences between siblings, adjusting for family fixed effects, we found that environmental mediation was nearly exclusively observed between families. This is consistent with the proposition that family environmental contexts contribute to academic development via passive gene-environment correlation processes. Our results show how parents shape environments that foster their childrens academic development partly based on their own genetic disposition, particularly towards noncognitive skills.\nGenome scale CRISPR screens identify actin capping proteins as key modulators of therapeutic responses to radiation and immunotherapy\nAuthors: Verma, N.; Renauer, P. A.; Dong, C.; Xin, S.; Lin, Q.; Zhang, F.; Glazer, P. M.; Chen, S.\nScore: 3.9, Published: 2024-01-15 DOI: 10.1101/2024.01.14.575614\nRadiotherapy (RT), is a fundamental treatment for malignant tumors and is used in over half of cancer patients. As radiation can promote anti-tumor immune effects, a promising therapeutic strategy is to combine radiation with immune checkpoint inhibitors (ICIs). However, the genetic determinants that impact therapeutic response in the context of combination therapy with radiation and ICI have not been systematically investigated. To unbiasedly identify the tumor intrinsic genetic factors governing such responses, we perform a set of genome-scale CRISPR screens in melanoma cells for cancer survival in response to low-dose genotoxic radiation treatment, in the context of CD8 T cell co-culture and with anti-PD1 checkpoint blockade antibody. Two actin capping proteins, Capza3 and Capg, emerge as top hits that upon inactivation promote the survival of melanoma cells in such settings. Capza3 and Capg knockouts (KOs) in mouse and human cancer cells display persistent DNA damage due to impaired homology directed repair (HDR); along with increased radiation, chemotherapy, and DNA repair inhibitor sensitivity. However, when cancer cells with these genes inactivated were exposed to sublethal radiation, inactivation of such actin capping protein promotes activation of the STING pathway, induction of inhibitory CEACAM1 ligand expression and resistance to CD8 T cell killing. Patient cancer genomics analysis reveals an increased mutational burden in patients with inactivating mutations in CAPG and/or CAPZA3, at levels comparable to other HDR associated genes. There is also a positive correlation between CAPG expression and activation of immune related pathways and CD8 T cell tumor infiltration. Our results unveil the critical roles of actin binding proteins for efficient HDR within cancer cells and demonstrate a previously unrecognized regulatory mechanism of therapeutic response to radiation and immunotherapy.\nSAM-DNMT3A, a strategy for induction of genome-wide DNA methylation, identifies DNA methylation as a vulnerability in ER-positive breast cancers\nAuthors: Hosseinpour, M.; Malaver-Ortega, L.; Perlaza-Jimenez, L.; Joo, J. E.; Liu, L.; Caldon, C. E.; Dugue, P.-A.; Dowty, J. G.; Southey, M. C.; Rosenbluh, J.\nScore: 1.8, Published: 2024-01-16 DOI: 10.1101/2024.01.16.575955\nDNA methylation is an epigenetic mark that plays a critical role in regulation of gene expression. DNA methylase (DNMT) inhibitors, inhibit global DNA methylation, and have been a key tool in studies of DNA methylation in healthy or disease conditions. A major bottleneck is the lack of tools to induce global DNA methylation. Here, we engineered a CRISPR based approach, that was initially designed, to enable site specific DNA methylation. Using the synergistic activation mediator (SAM) system, we unexpectedly found that regardless of the targeted sequence any sgRNA induced global genome-wide DNA methylation. We termed this new method SAM-DNMT3A and show that induction of global DNA methylation is a unique vulnerability in ER-positive breast cancer suggesting a therapeutic approach. Our findings highlight the need of caution when using CRISPR based approaches for inducing DNA methylation and demonstrate a new method for global induction of DNA methylation.\nImpact of Unc-51 Like Kinase 4 (ULK4) on the Reactivity of the Extended Reward System in Response to Conditioned Stimuli\nAuthors: Treutlein, J.; Loehlein, S.; Einenkel, K. E.; Diekhof, E. K.; Gruber, O.\nScore: 1.5, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575326\nObjectives: ULK4 is an established candidate gene for mental disorders and antipsychotic treatment response and codes for a serine/threonine kinase that regulates the neural stem cell pool and controls cortex development. We investigated the effects of functional genetic variation at the ULK4 locus on the human extended dopaminergic reward system using functional magnetic resonance imaging (fMRI) during performance of a well-established reward paradigm. Methods: 234 study participants with functional neuroimaging (fMRI) data of the extended reward system and with ULK4 genotype data were included in this study. Effects of genetic variation in the ULK4 gene on reward system functioning were determined using the Desire-Reason-Dilemma (DRD) paradigm which allows to assess brain activation in response to conditioned reward stimuli (Diekhof et al. 2010). Results: Among common missense variants of the ULK4 gene, variant prioritization revealed strongest functional signatures for variant rs17215589, coding for amino acid exchange Ala715Thr. For rs17215589 minor allele carriers, we detected increased activation responses to conditioned reward stimuli in the ventral tegmental area, the nucleus accumbens and several cortical brain regions of the extended reward system. Conclusions: Our findings provide further evidence in humans that genetic variation in ULK4 may increase the vulnerability to mental disorders by modulating the function of the extended reward system. Future studies are needed to confirm the functional modulation of the extended reward system by ULK4 and to specify the role of this mechanism in the pathogenesis of psychiatric disorders.\nPicturing plant biodiversity from airborne environmental DNA.\nAuthors: Thuillet, A.-C.; Morisot, D.; Renno, J.-F.; Scarcelli, N.; Serret, J.; Mariac, C.\nScore: 1.5, Published: 2024-01-13 DOI: 10.1101/2024.01.11.571706\nWhile eDNA approaches have gained interest over the past decades all types of organisms have not been addressed evenly. In particular terrestrial plants have been the subject of less attention. Here we address the possibility to represent plant biodiversity from airborne environmental DNA (eDNA) sampling and metabarcoding. We collected air using a biological air sampler in the Botanical Garden of Montpellier (France) and compared the list of revealed plant species to the botanical inventory of the Garden. Ninety-two plant species could be detected from three sampling points across the 4,6 ha of the Garden, after one hour sampling allowing to filter 9 m3 of air. We recorded the plants carrying flowers at the time of the experiment, which allowed us to estimate that plants flowering at the time of the sampling could be detected 10 times more easily than plants that were not, given the number of plants carrying flowers. However, flowering is far from being required as a vast majority of plants still was detected without flowering. We also show that not all species orders are detected with the same probability, tree species being better represented in the sample than herbal plants, given the number of trees present in the garden. Finally using diagnostic species, present only once in the garden, we estimate that the maximum sampling distance allowed by the biological air sampler is at least 110 m. Our study underlines that air sampling is a promising method for monitoring terrestrial plant biodiversity and highlights the parameters that should be adjusted to optimize the approach.\nThe cytidine deaminase APOBEC3C has unique sequence and genome feature preferences\nAuthors: Brown, G. W.\nScore: 1.2, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575754\nAPOBEC proteins are cytidine deaminases that restrict the replication of viruses and transposable elements. Several members of the APOBEC3 family, APOBEC3A, APOBEC3B, and APOBEC3H-I, can access the nucleus and cause what is thought to be indiscriminate deamination of the genome, resulting in mutagenesis and genome instability. Although APOBEC3C is also present in the nucleus, its deamination target preferences are unknown. By expressing human APOBEC3C in a yeast model system, I have defined the APOBEC3C mutation signature, as well as the preferred genome features of APOBEC3C targets. The APOBEC3C mutation signature is distinct from those of the known cancer genome mutators APOBEC3A and APOBEC3B. APOBEC3C produces DNA strand-coordinated mutation clusters, and APOBEC3C mutations are enriched near the transcription start sites of active genes. Surprisingly, APOBEC3C lacks the bias for the lagging strand of DNA replication that is seen for APOBEC3A and APOBEC3B. The unique preferences of APOBEC3C constitute a mutation profile that will be useful in defining sites of APOBEC3C mutagenesis in human genomes.\nCreation and Validation of the First Infinium DNA Methylation Array for the Human Imprintome\nAuthors: Carreras-Gallo, N.; Dwaraka, V. B.; Jima, D. D.; Skaar, D. A.; Mendez, T. L.; Planchart, A.; Zhou, W.; Jirtle, R. L.; Smith, R.; Hoyo, C.\nScore: 1.0, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575646\nBackground: Differentially methylated imprint control regions (ICRs) regulate the monoallelic expression of imprinted genes. Their epigenetic dysregulation by environmental exposures throughout life results in the formation of common chronic diseases. Unfortunately, existing Infinium methylation arrays lack the ability to profile these regions adequately. Whole genome bisulfite sequencing (WGBS) is the unique method able to profile these regions, but it is very expensive and it requires not only a high coverage but it is also computationally intensive to assess those regions. Findings: To address this deficiency, we developed a custom methylation array containing 22,819 probes. Among them, 9,757 probes map to 1,088 out of the 1,488 candidate ICRs recently described. To assess the performance of the array, we created matched samples processed with the Human Imprintome array and WGBS, which is the current standard method for assessing the methylation of the Human Imprintome. We compared the methylation levels from the shared CpG sites and obtained a mean R2 = 0.569. We also created matched samples processed with the Human Imprintome array and the Infinium Methylation EPIC v2 array and obtained a mean R2 = 0.796. Furthermore, replication experiments demonstrated high reliability (ICC: 0.799-0.945). Conclusions: Our custom array will be useful for replicable and accurate assessment, mechanistic insight, and targeted investigation of ICRs. This tool should accelerate the discovery of ICRs associated with a wide range of diseases and exposures, and advance our understanding of genomic imprinting and its relevance in development and disease formation throughout the life course.\nShort-chain fatty acids propionate and butyrate control growth and differentiation linked to cellular metabolism\nAuthors: Nshanian, M.; Geller, B. S.; Gruber, J. J.; Chleilat, F.; Camarillo, J. M.; Kelleher, N. L.; Zhao, Y.; Snyder, M. P.\nScore: 1.0, Published: 2024-01-15 DOI: 10.1101/2024.01.11.575111\nThe short-chain fatty acids (SCFA) propionate and butyrate are produced in large amounts by microbial metabolism and have been identified as unique acyl lysine histone marks. In order to better understand the function of these modifications we used ChIP-Seq to map the genome-wide location of four short-chain acyl histone marks H3K18pr/bu and H4K12pr/bu in treated and untreated colorectal cancer (CRC) and normal cells, as well as in mouse intestines in vivo. We correlate these marks with open chromatin regions along with gene expression to access the function of the target regions. Our data demonstrate that propionate and butyrate act as promoters of growth, differentiation as well as ion transport. We propose a mechanism involving direct modification of specific genomic regions, resulting in increased chromatin accessibility, and in case of butyrate, opposing effects on the proliferation of normal versus CRC cells.\n",
  "wordCount" : "2490",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574865">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574865" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574865">
        <p class="paperTitle">Associations between common genetic variants and income provide insights about the socioeconomic health gradient</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574865" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574865" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kweon, H.; Burik, C. A. P.; Ning, Y.; Ahlskog, R.; Xia, C.; Abner, E.; Bao, Y.; Bhatta, L.; Faquih, T.; Feijter, M.; Fisher, P.; Gelemanovic, A.; Giannelis, A.; Hottenga, J.-J.; Khalili, B.; Lee, Y.; Li-Gao, R.; Masso, J.; Myhre, R.; Palviainen, T.; Rietveld, C. A. P.; Teumer, A.; Verweij, R. M.; Willoughby, E. A.; Agerbo, E.; Bergmann, S.; Boomsma, D. I.; Borglum, A. D.; Brumpton, B. M.; Davies, N. M.; Esko, T.; Gordon, S. D.; Homuth, G.; Ikram, A.; Johannesson, M.; Kaprio, J.; Kidd, M. P.; Kutalik, Z.; Kwong, A. S. F.; Lee, J.; Luik, A.; Mook-Kanamori, D. O.; Magnus, P.; Marques-Vidal, P.; M</p>
        <p class="info">Score: 50.2, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574865' target='https://doi.org/10.1101/2024.01.09.574865'> 10.1101/2024.01.09.574865</a></p>
        <p class="abstract">We conducted a genome-wide association study (GWAS) on income among individuals of European descent and leveraged the results to investigate the socio-economic health gradient (N=668,288). We found 162 genomic loci associated with a common genetic factor underlying various income measures, all with small effect sizes. Our GWAS-derived polygenic index captures 1 - 4% of income variance, with only one-fourth attributed to direct genetic effects. A phenome-wide association study using this polygenic index showed reduced risks for a broad spectrum of diseases, including hypertension, obesity, type 2 diabetes, coronary atherosclerosis, depression, asthma, and back pain. The income factor showed a substantial genetic correlation (0.92, s.e. = .006) with educational attainment (EA). Accounting for EAs genetic overlap with income revealed that the remaining genetic signal for higher income related to better mental health but reduced physical health benefits and increased participation in risky behaviours such as drinking and smoking.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575120" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575120">
        <p class="paperTitle">Single-step generation of homozygous knock-out/knock-in individuals in an extremotolerant parthenogenetic tardigrade using DIPA-CRISPR</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kondo, K.; Tanaka, A.; Kunieda, T.</p>
        <p class="info">Score: 16.7, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575120' target='https://doi.org/10.1101/2024.01.10.575120'> 10.1101/2024.01.10.575120</a></p>
        <p class="abstract">Tardigrades are small aquatic invertebrates known for their remarkable tolerance to diverse extreme stresses. To elucidate the in vivo mechanisms underlying this extraordinary resilience, the genetic manipulation methods in tardigrades have long been desired. Despite our prior success in somatic cell gene-editing by microinjecting Cas9 ribonucleoproteins (RNPs) into the body cavity of tardigrades, the generation of gene-edited individuals remained elusive.

In this study, employing an extremotolerant parthenogenetic tardigrade species, Ramazzottius varieornatus, we established conditions conductive to generating gene-edited tardigrade individuals. Drawing inspiration from the direct parental CRIPSR (DIPA-CRISPR) technique employed in several insects, we simply injected a concentrated Cas9 RNP solution into the body cavity of parental females shortly before their initial oviposition. This approach yielded gene-edited G0 progeny. Notably, only a single allele was predominantly detected at the target locus for each G0 individual, indicative of homozygous mutations. Through co-injecting single-stranded oligodeoxynucleotides (ssODNs) with Cas9 RNPs, we achieved the generation of homozygously knocked-in G0 progeny and these edited-alleles were inherited by G1/G2 progeny.

This establishment of a simple method for generating homozygous knock-out/knock-in individuals not only facilitates in vivo analyses of molecular mechanisms underpinning extreme tolerance but also opens avenues for exploring various topics, including Evo-Devo, in tardigrades.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574339">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574339" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574339">
        <p class="paperTitle">Gene-environment correlation: The role of family environment in academic development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574339" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574339" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, Q.; Gidziela, A.; Allegrini, A. G.; Cheesman, R.; Wertz, J.; Maxwell, J.; Plomin, R.; Rimfeld, K.; Malanchini, M.</p>
        <p class="info">Score: 21.0, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574339' target='https://doi.org/10.1101/2024.01.05.574339'> 10.1101/2024.01.05.574339</a></p>
        <p class="abstract">Academic achievement is partly heritable and highly polygenic. However, genetic effects on academic achievement are not independent of environmental processes. We investigated whether aspects of the family environment mediated genetic effects on academic achievement across development. Our sample included 5,151 children who participated in the Twins Early Development Study, as well as their parents and teachers. Data on academic achievement and family environments were available at ages 7, 9, 12 and 16. We computed educational attainment polygenic scores (PGS) and further separated genetic effects into cognitive and noncognitive PGS. Three core findings emerged. First, aspects of the family environment, but not the wider neighbourhood context, consistently mediated the PGS effects on achievement across development, accounting for up to 34.3% of the total effect. Family characteristics mattered beyond socio-economic status. Second, family environments were more robustly linked to noncognitive PGS effects on academic achievement than cognitive PGS effects. Third, when we investigated whether environmental mediation effects could also be observed when considering differences between siblings, adjusting for family fixed effects, we found that environmental mediation was nearly exclusively observed between families. This is consistent with the proposition that family environmental contexts contribute to academic development via passive gene-environment correlation processes. Our results show how parents shape environments that foster their childrens academic development partly based on their own genetic disposition, particularly towards noncognitive skills.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575614">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575614" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575614">
        <p class="paperTitle">Genome scale CRISPR screens identify actin capping proteins as key modulators of therapeutic responses to radiation and immunotherapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575614" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575614" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Verma, N.; Renauer, P. A.; Dong, C.; Xin, S.; Lin, Q.; Zhang, F.; Glazer, P. M.; Chen, S.</p>
        <p class="info">Score: 3.9, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575614' target='https://doi.org/10.1101/2024.01.14.575614'> 10.1101/2024.01.14.575614</a></p>
        <p class="abstract">Radiotherapy (RT), is a fundamental treatment for malignant tumors and is used in over half of cancer patients. As radiation can promote anti-tumor immune effects, a promising therapeutic strategy is to combine radiation with immune checkpoint inhibitors (ICIs). However, the genetic determinants that impact therapeutic response in the context of combination therapy with radiation and ICI have not been systematically investigated. To unbiasedly identify the tumor intrinsic genetic factors governing such responses, we perform a set of genome-scale CRISPR screens in melanoma cells for cancer survival in response to low-dose genotoxic radiation treatment, in the context of CD8 T cell co-culture and with anti-PD1 checkpoint blockade antibody. Two actin capping proteins, Capza3 and Capg, emerge as top hits that upon inactivation promote the survival of melanoma cells in such settings. Capza3 and Capg knockouts (KOs) in mouse and human cancer cells display persistent DNA damage due to impaired homology directed repair (HDR); along with increased radiation, chemotherapy, and DNA repair inhibitor sensitivity. However, when cancer cells with these genes inactivated were exposed to sublethal radiation, inactivation of such actin capping protein promotes activation of the STING pathway, induction of inhibitory CEACAM1 ligand expression and resistance to CD8 T cell killing. Patient cancer genomics analysis reveals an increased mutational burden in patients with inactivating mutations in CAPG and/or CAPZA3, at levels comparable to other HDR associated genes. There is also a positive correlation between CAPG expression and activation of immune related pathways and CD8 T cell tumor infiltration. Our results unveil the critical roles of actin binding proteins for efficient HDR within cancer cells and demonstrate a previously unrecognized regulatory mechanism of therapeutic response to radiation and immunotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575955">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575955" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575955">
        <p class="paperTitle">SAM-DNMT3A, a strategy for induction of genome-wide DNA methylation, identifies DNA methylation as a vulnerability in ER-positive breast cancers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575955" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575955" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hosseinpour, M.; Malaver-Ortega, L.; Perlaza-Jimenez, L.; Joo, J. E.; Liu, L.; Caldon, C. E.; Dugue, P.-A.; Dowty, J. G.; Southey, M. C.; Rosenbluh, J.</p>
        <p class="info">Score: 1.8, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575955' target='https://doi.org/10.1101/2024.01.16.575955'> 10.1101/2024.01.16.575955</a></p>
        <p class="abstract">DNA methylation is an epigenetic mark that plays a critical role in regulation of gene expression. DNA methylase (DNMT) inhibitors, inhibit global DNA methylation, and have been a key tool in studies of DNA methylation in healthy or disease conditions. A major bottleneck is the lack of tools to induce global DNA methylation. Here, we engineered a CRISPR based approach, that was initially designed, to enable site specific DNA methylation. Using the synergistic activation mediator (SAM) system, we unexpectedly found that regardless of the targeted sequence any sgRNA induced global genome-wide DNA methylation. We termed this new method SAM-DNMT3A and show that induction of global DNA methylation is a unique vulnerability in ER-positive breast cancer suggesting a therapeutic approach. Our findings highlight the need of caution when using CRISPR based approaches for inducing DNA methylation and demonstrate a new method for global induction of DNA methylation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575326">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575326" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575326">
        <p class="paperTitle">Impact of Unc-51 Like Kinase 4 (ULK4) on the Reactivity of the Extended Reward System in Response to Conditioned Stimuli</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575326" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575326" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Treutlein, J.; Loehlein, S.; Einenkel, K. E.; Diekhof, E. K.; Gruber, O.</p>
        <p class="info">Score: 1.5, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575326' target='https://doi.org/10.1101/2024.01.15.575326'> 10.1101/2024.01.15.575326</a></p>
        <p class="abstract">Objectives: ULK4 is an established candidate gene for mental disorders and antipsychotic treatment response and codes for a serine/threonine kinase that regulates the neural stem cell pool and controls cortex development. We investigated the effects of functional genetic variation at the ULK4 locus on the human extended dopaminergic reward system using functional magnetic resonance imaging (fMRI) during performance of a well-established reward paradigm. Methods: 234 study participants with functional neuroimaging (fMRI) data of the extended reward system and with ULK4 genotype data were included in this study. Effects of genetic variation in the ULK4 gene on reward system functioning were determined using the Desire-Reason-Dilemma (DRD) paradigm which allows to assess brain activation in response to conditioned reward stimuli (Diekhof et al. 2010). Results: Among common missense variants of the ULK4 gene, variant prioritization revealed strongest functional signatures for variant rs17215589, coding for amino acid exchange Ala715Thr. For rs17215589 minor allele carriers, we detected increased activation responses to conditioned reward stimuli in the ventral tegmental area, the nucleus accumbens and several cortical brain regions of the extended reward system. Conclusions: Our findings provide further evidence in humans that genetic variation in ULK4 may increase the vulnerability to mental disorders by modulating the function of the extended reward system. Future studies are needed to confirm the functional modulation of the extended reward system by ULK4 and to specify the role of this mechanism in the pathogenesis of psychiatric disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.571706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.571706" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.571706">
        <p class="paperTitle">Picturing plant biodiversity from airborne environmental DNA.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.571706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.571706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thuillet, A.-C.; Morisot, D.; Renno, J.-F.; Scarcelli, N.; Serret, J.; Mariac, C.</p>
        <p class="info">Score: 1.5, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.571706' target='https://doi.org/10.1101/2024.01.11.571706'> 10.1101/2024.01.11.571706</a></p>
        <p class="abstract">While eDNA approaches have gained interest over the past decades all types of organisms have not been addressed evenly. In particular terrestrial plants have been the subject of less attention. Here we address the possibility to represent plant biodiversity from airborne environmental DNA (eDNA) sampling and metabarcoding. We collected air using a biological air sampler in the Botanical Garden of Montpellier (France) and compared the list of revealed plant species to the botanical inventory of the Garden. Ninety-two plant species could be detected from three sampling points across the 4,6 ha of the Garden, after one hour sampling allowing to filter 9 m3 of air. We recorded the plants carrying flowers at the time of the experiment, which allowed us to estimate that plants flowering at the time of the sampling could be detected 10 times more easily than plants that were not, given the number of plants carrying flowers. However, flowering is far from being required as a vast majority of plants still was detected without flowering. We also show that not all species orders are detected with the same probability, tree species being better represented in the sample than herbal plants, given the number of trees present in the garden. Finally using diagnostic species, present only once in the garden, we estimate that the maximum sampling distance allowed by the biological air sampler is at least 110 m. Our study underlines that air sampling is a promising method for monitoring terrestrial plant biodiversity and highlights the parameters that should be adjusted to optimize the approach.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575754">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575754" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575754">
        <p class="paperTitle">The cytidine deaminase APOBEC3C has unique sequence and genome feature preferences</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575754" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575754" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brown, G. W.</p>
        <p class="info">Score: 1.2, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575754' target='https://doi.org/10.1101/2024.01.15.575754'> 10.1101/2024.01.15.575754</a></p>
        <p class="abstract">APOBEC proteins are cytidine deaminases that restrict the replication of viruses and transposable elements. Several members of the APOBEC3 family, APOBEC3A, APOBEC3B, and APOBEC3H-I, can access the nucleus and cause what is thought to be indiscriminate deamination of the genome, resulting in mutagenesis and genome instability. Although APOBEC3C is also present in the nucleus, its deamination target preferences are unknown. By expressing human APOBEC3C in a yeast model system, I have defined the APOBEC3C mutation signature, as well as the preferred genome features of APOBEC3C targets. The APOBEC3C mutation signature is distinct from those of the known cancer genome mutators APOBEC3A and APOBEC3B. APOBEC3C produces DNA strand-coordinated mutation clusters, and APOBEC3C mutations are enriched near the transcription start sites of active genes. Surprisingly, APOBEC3C lacks the bias for the lagging strand of DNA replication that is seen for APOBEC3A and APOBEC3B. The unique preferences of APOBEC3C constitute a mutation profile that will be useful in defining sites of APOBEC3C mutagenesis in human genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575646">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575646" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575646">
        <p class="paperTitle">Creation and Validation of the First Infinium DNA Methylation Array for the Human Imprintome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575646" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575646" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carreras-Gallo, N.; Dwaraka, V. B.; Jima, D. D.; Skaar, D. A.; Mendez, T. L.; Planchart, A.; Zhou, W.; Jirtle, R. L.; Smith, R.; Hoyo, C.</p>
        <p class="info">Score: 1.0, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575646' target='https://doi.org/10.1101/2024.01.15.575646'> 10.1101/2024.01.15.575646</a></p>
        <p class="abstract">Background: Differentially methylated imprint control regions (ICRs) regulate the monoallelic expression of imprinted genes. Their epigenetic dysregulation by environmental exposures throughout life results in the formation of common chronic diseases. Unfortunately, existing Infinium methylation arrays lack the ability to profile these regions adequately. Whole genome bisulfite sequencing (WGBS) is the unique method able to profile these regions, but it is very expensive and it requires not only a high coverage but it is also computationally intensive to assess those regions. Findings: To address this deficiency, we developed a custom methylation array containing 22,819 probes. Among them, 9,757 probes map to 1,088 out of the 1,488 candidate ICRs recently described. To assess the performance of the array, we created matched samples processed with the Human Imprintome array and WGBS, which is the current standard method for assessing the methylation of the Human Imprintome. We compared the methylation levels from the shared CpG sites and obtained a mean R2 = 0.569. We also created matched samples processed with the Human Imprintome array and the Infinium Methylation EPIC v2 array and obtained a mean R2 = 0.796. Furthermore, replication experiments demonstrated high reliability (ICC: 0.799-0.945). Conclusions: Our custom array will be useful for replicable and accurate assessment, mechanistic insight, and targeted investigation of ICRs. This tool should accelerate the discovery of ICRs associated with a wide range of diseases and exposures, and advance our understanding of genomic imprinting and its relevance in development and disease formation throughout the life course.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575111">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575111" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575111">
        <p class="paperTitle">Short-chain fatty acids propionate and butyrate control growth and differentiation linked to cellular metabolism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575111" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575111" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nshanian, M.; Geller, B. S.; Gruber, J. J.; Chleilat, F.; Camarillo, J. M.; Kelleher, N. L.; Zhao, Y.; Snyder, M. P.</p>
        <p class="info">Score: 1.0, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575111' target='https://doi.org/10.1101/2024.01.11.575111'> 10.1101/2024.01.11.575111</a></p>
        <p class="abstract">The short-chain fatty acids (SCFA) propionate and butyrate are produced in large amounts by microbial metabolism and have been identified as unique acyl lysine histone marks. In order to better understand the function of these modifications we used ChIP-Seq to map the genome-wide location of four short-chain acyl histone marks H3K18pr/bu and H4K12pr/bu in treated and untreated colorectal cancer (CRC) and normal cells, as well as in mouse intestines in vivo. We correlate these marks with open chromatin regions along with gene expression to access the function of the target regions. Our data demonstrate that propionate and butyrate act as promoters of growth, differentiation as well as ion transport. We propose a mechanism involving direct modification of specific genomic regions, resulting in increased chromatin accessibility, and in case of butyrate, opposing effects on the proliferation of normal versus CRC cells.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
