<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Leveraging fine-scale population structure reveals conservation in genetic effect sizes between human populations across a range of human phenotypes
Authors: Hu, S.; Ferreira, L. A. F.; Shi, S.; Hellenthal, G.; Marchini, J.; Lawson, D. J.; Myers, S. R.
Score: 56.3, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552281
An understanding of genetic differences between populations is essential for avoiding confounding in genome-wide association studies (GWAS) and understanding the evolution of human traits. Polygenic risk scores constructed in one group perform poorly in highly genetically-differentiated populations, for reasons which remain controversial.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Leveraging fine-scale population structure reveals conservation in genetic effect sizes between human populations across a range of human phenotypes
Authors: Hu, S.; Ferreira, L. A. F.; Shi, S.; Hellenthal, G.; Marchini, J.; Lawson, D. J.; Myers, S. R.
Score: 56.3, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552281
An understanding of genetic differences between populations is essential for avoiding confounding in genome-wide association studies (GWAS) and understanding the evolution of human traits. Polygenic risk scores constructed in one group perform poorly in highly genetically-differentiated populations, for reasons which remain controversial." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-06T10:39:07+00:00" />
<meta property="article:modified_time" content="2023-09-06T10:39:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Leveraging fine-scale population structure reveals conservation in genetic effect sizes between human populations across a range of human phenotypes
Authors: Hu, S.; Ferreira, L. A. F.; Shi, S.; Hellenthal, G.; Marchini, J.; Lawson, D. J.; Myers, S. R.
Score: 56.3, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552281
An understanding of genetic differences between populations is essential for avoiding confounding in genome-wide association studies (GWAS) and understanding the evolution of human traits. Polygenic risk scores constructed in one group perform poorly in highly genetically-differentiated populations, for reasons which remain controversial."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Leveraging fine-scale population structure reveals conservation in genetic effect sizes between human populations across a range of human phenotypes\nAuthors: Hu, S.; Ferreira, L. A. F.; Shi, S.; Hellenthal, G.; Marchini, J.; Lawson, D. J.; Myers, S. R.\nScore: 56.3, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552281\nAn understanding of genetic differences between populations is essential for avoiding confounding in genome-wide association studies (GWAS) and understanding the evolution of human traits. Polygenic risk scores constructed in one group perform poorly in highly genetically-differentiated populations, for reasons which remain controversial.",
  "keywords": [
    
  ],
  "articleBody": " Leveraging fine-scale population structure reveals conservation in genetic effect sizes between human populations across a range of human phenotypes\nAuthors: Hu, S.; Ferreira, L. A. F.; Shi, S.; Hellenthal, G.; Marchini, J.; Lawson, D. J.; Myers, S. R.\nScore: 56.3, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552281\nAn understanding of genetic differences between populations is essential for avoiding confounding in genome-wide association studies (GWAS) and understanding the evolution of human traits. Polygenic risk scores constructed in one group perform poorly in highly genetically-differentiated populations, for reasons which remain controversial. We developed a statistical ancestry inference pipeline able to decompose ancestry both within and between countries, and applied it to the UK Biobank data. This identifies fine-scale patterns of genetic relatedness not captured by standard and widely used principal components (PCs), and allows fine-scale population stratification correction that removes both false positive and false negative associations for traits with geographic correlations. We also develop and apply ANCHOR, an approach leveraging segments of distinct ancestries within individuals to estimate similarity in underlying causal effect sizes between groups, using an existing PGS. Applying ANCHOR to \u003e8000 people of mixed African and European ancestry, we demonstrate that estimated causal effect sizes are highly similar across these ancestries for 26 of 29 quantitative molecular and non-molecular phenotypes (mean correlation 0.98 +/-0.08), providing evidence that gene-environment and gene-gene interactions do not play major roles in the poor prediction of European-ancestry PRS scores in African populations for these traits, contradicting previous findings. Instead our results provide optimism that shared causal mutations operate similarly in different groups, focussing the challenge of improving GWAS \"portability\" between groups on joint fine-mapping.\nAncient farmer and steppe pastoralist-related founding lineages contributed to the complex landscape of episodes in the diversification of Chinese paternal lineages\nAuthors: Wang, M.; Huang, Y.; Liu, K.; Yuan, H.; Duan, S.; Wang, Z.; Wei, L.; Yao, H.; Sun, Q.; Zhong, J.; Tang, R.; Chen, J.; Sun, Y.; Li, X.; Su, H.; Yang, Q.; Hu, L.; Yun, L.; Yang, J.; Nie, S.; Cai, Y.; Yan, J.; Zhou, K.; 10K_CPGDP Consortium, ; Wang, C.-C.; Zhu, B.; Liu, C.; He, G.\nScore: 29.0, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555114\nAncient DNA advances have reported the complex genetic history of Eurasians, but how the knowledge of ancient subsistence strategy shifts and population movements influenced the fine-scale paternal genetic structure in East Asia has not been assessed. Here, we reported one integrated Y-chromosome genomic database of 15,530 people, including 1753 ancient people and newly-reported 919 individuals genotyped using our recently-developed targeted sequencing YHSeqY3000 panel, to explore Chinese genomic diversity, population evolutionary tracts and their genetic formation mechanism. We identified four major ancient technological innovations and population movements that shaped the landscape of Chinese paternal lineages. First, the expansion of millet farmers and early East Asians from the Yellow River Basin carrying the major O2/D subclades promoted the formation of the Sino-Tibetan peoples major composition and accelerated the Tibetan Plateaus permanent occupation. Second, rice farmers dispersal from the Yangtze River Valley carrying O1 and some sublineages of O2 contributed significantly to Tai-Kadai, Austronesian, Hmong-Mien, Austroasiatic people and southern Han Chinese. Third, Siberian-related paternal lineages of Q and C originated and boomed from Neolithic hunter-gatherers from the Mongolian Plateau and the Amur River Basin and significantly influenced the gene pools of northern Chinese. Fourth, western Eurasian-derived J, G and R lineages initially spread with Yamnaya steppe pastoralists and other proto-Indo-European people and further widely dispersed via the trans-Eurasian cultural communication along the Eurasian Steppe and the ancient Silk Road, remaining genetic trajectories in northwestern Chinese. Our work provided comprehensive modern and ancient genetic evidence to illuminate the impact of population interaction from the ancient farmer or herder-based societies on the genetic diversity patterns of modern people, revised our understandings of ancestral sources of Chinese paternal lineages, underscored the scientific imperative of the large-scale genomic resources of dense spatiotemporal underrepresented sampling populations to understand human evolutionary history.\nFunctional characterization of gene regulatory elements and neuropsychiatric disease-associated risk loci in iPSCs and iPSC-derived neurons\nAuthors: Yang, X.; Jones, I. R.; Chen, P. B.; Yang, H.; Ren, X.; Zheng, L.; Li, B.; Li, Y. E.; Sun, Q.; Wen, J.; Beaman, C.; Cui, X.; Li, Y.; Wang, W.; Hu, M.; Ren, B.; Shen, Y.\nScore: 27.0, Published: 2023-08-30 DOI: 10.1101/2023.08.30.555359\nGenome-wide association studies (GWAS) have identified thousands of non-coding variants that contribute to psychiatric disease risks, likely by perturbing cis-regulatory elements (CREs). However, our ability to interpret and explore their mechanisms of action is hampered by a lack of annotation of functional CREs (fCREs) in neural cell types. Here, through genome-scale CRISPR screens of 22,000 candidate CREs (cCREs) in human induced pluripotent stem cells (iPSCs) undergoing differentiation to excitatory neurons, we identify 2,847 and 5,540 fCREs essential for iPSC fitness and neuronal differentiation, respectively. These fCREs display dynamic epigenomic features and exhibit increased numbers and genomic spans of chromatin interactions following terminal neuronal differentiation. Furthermore, fCREs essential for neuronal differentiation show significantly greater enrichment of genetic heritability for neurodevelopmental diseases including schizophrenia (SCZ), attention deficit hyperactivity disorder (ADHD), and autism spectrum disorders (ASD) than cCREs. Using high-throughput prime editing screens we experimentally confirm 45 SCZ risk variants that act by affecting the function of fCREs. The extensive and in-depth functional annotation of cCREs in neuronal types therefore provides a crucial resource for interpreting non-coding risk variants of neuropsychiatric disorders.\nA defined diet for pre-adult Drosophila melanogaster\nAuthors: Martelli, F.; Quig, A.; Mele, S.; Lin, J.; Fulton, T. L.; Wansbrough, M.; Barlow, C. K.; Schittenhelm, R. B.; Johnson, T. K.; Piper, M. D.\nScore: 14.4, Published: 2023-08-30 DOI: 10.1101/2023.08.28.555220\nDrosophila melanogaster is unique among animal models because it has a fully defined synthetic diet available to study nutrient-gene interactions. However, use of this diet is limited to adult studies due to impaired larval development and survival. Here, we provide an adjusted formula that improves larval growth, development rate, and rescues survival. We demonstrate its use for exploring pre-adult diet compositions of therapeutic potential in a model of an inherited metabolic disorder.\nAbsence of chromosome axis proteins recruitment prevents meiotic recombination chromosome-wide in the budding yeast Lachancea kluyveri\nAuthors: Legrand, S.; Saifudeen, A.; Bordelet, H.; Vernerey, J.; Guille, A.; Bignaut, A.; Thierry, A.; Acquaviva, L.; Gaudin, M.; Sanchez, A.; Johnson, D.; Friedrcih, A.; Schacherer, J.; Neale, M.; Borde, V.; Koszul, R.; llorente, b.\nScore: 13.2, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555118\nMeiotic recombination shows broad variations across species and along chromosomes, and is often suppressed at and around genomic regions determining sexual compatibility such as mating type loci in fungi. Here we show that the absence of Spo11-DSBs and meiotic recombination on Lakl0C-left, the chromosome arm containing the sex locus of the Lachancea kluyveri budding yeast, results from the absence of recruitment of the two chromosome axis proteins Red1 and Hop1, essential for proper Spo11-DSBs formation. Furthermore, cytological observation of spread pachytene meiotic chromosomes reveals that Lakl0C-left does not undergo synapsis. However, we show that the behavior of Lakl0C-left is independent of its particularly early replication timing and is not accompanied by any peculiar chromosome structure as detectable by Hi-C in this yet poorly studied yeast. Finally, we observed an accumulation of heterozygous mutations on Lakl0C-left and a sexual dimorphism of the haploid meiotic offspring, supporting a direct effect of this absence of meiotic recombination on L. kluyveri genome evolution and fitness. Because suppression of meiotic recombination on sex chromosomes is widely observed across eukaryotes, the novel mechanism for recombination suppression described here may apply to other species, with the potential to impact sex chromosome evolution.\nGeneration of isogenic models of Angelman syndrome and Prader-Willi syndrome in CRISPR/Cas9-engineered human embryonic stem cells\nAuthors: Gilmore, R. B.; Gorka, D.; Stoddard, C.; Cotney, J.; Chamberlain, S.\nScore: 8.8, Published: 2023-08-30 DOI: 10.1101/2023.08.30.555563\nAngelman Syndrome (AS) and Prader-Willi Syndrome (PWS), two distinct neurodevelopmental disorders, result from loss of expression from imprinted genes in the chromosome 15q11-13 locus most commonly caused by a megabase-scale deletion on either the maternal or paternal allele, respectively. Each occurs at an approximate incidence of 1/15,000 to 1/30,000 live births and has a range of debilitating phenotypes. Patient-derived induced pluripotent stem cells (iPSCs) have been valuable tools to understand human-relevant gene regulation at this locus and have contributed to the development of therapeutic approaches for AS. Nonetheless, gaps remain in our understanding of how these deletions contribute to dysregulation and phenotypes of AS and PWS. Variability across cell lines due to donor differences, reprogramming methods, and genetic background make it challenging to fill these gaps in knowledge without substantially increasing the number of cell lines used in the analyses. Isogenic cell lines that differ only by the genetic mutation causing the disease can ease this burden without requiring such a large number of cell lines. Here, we describe the development of isogenic human embryonic stem cell (hESC) lines modeling the most common genetic subtypes of AS and PWS. These lines allow for a facile interrogation of allele-specific gene regulation at the chromosome 15q11-q13 locus. Additionally, these lines are an important resource to identify and test targeted therapeutic approaches for patients with AS and PWS.\nNatural genetic variation quantitatively regulates heart-rate and -dimension\nAuthors: Gierten, J.; Welz, B.; Fitzgerald, T. W.; Thumberger, T.; Hummel, O.; Leger, A.; Weber, P.; Hassel, D.; Hubner, N.; Birney, E.; Wittbrodt, J.\nScore: 7.9, Published: 2023-09-02 DOI: 10.1101/2023.09.01.555906\nDevelopment and function of the heart are crucially controlled by individual genetic factors determining the continuum from health to disease. A prime example of a quantitative pathological phenotype is left ventricular hypoplasia, affecting the left ventricle (LV) and its associated inflow and outflow structures and reducing its size to varying degrees. Current models suggest polygenic contributions on heart development and disease, but we are just beginning to understand the impact of naturally occurring sequence variations. Here we show how natural genetic variants exert cumulative, environment-sensitive effects on both embryonic heart development as well as adult heart function. We addressed this in embryos of five inbred strains of the Japanese rice fish medaka (Oryzias latipes and Oryzias sakaizumii), analyzing heart rates as a functional proxy to capture combined characteristics of the conduction system and structural morphology. We identified one strain, HO5, with embryonic hypoplastic ventricles, fast heart rates, and significantly impaired cardiac function and overall fitness in adults. We created an interpopulation cross of the divergent HO5 strain to HdrR, which exhibits a morphologically normal heart and comparably slow heart rates, to leverage segregation variance in the second filial generation (F2) for genetic mapping. The correlation of 1192 individually phenotyped F2 embryos (heart rate) with the corresponding whole-genome sequences established a single nucleotide polymorphism-based linkage map. Integrated analysis of parental cardiac transcriptomes highlighted 59 loci containing genes prominently linked to human cardiovascular phenotypes. In vivo knockout models of the top 12 candidate genes demonstrated a causal and differential impact on heart development, ventricular size, and heart rate, evidencing the polygenic nature of heart phenotypes and establishing new loci for genetic diagnostics of human congenital heart disease. The controllable and scalable genetics and experimental validation in inbred vertebrate models complement and expand human association studies to provide novel and unique insights into the role of natural genetic variation in the mechanism of cardiac disease and allow to identify genetic susceptibility in individual human subjects.\nIdentification and functional characterisation of a locus for target site integration in Fusarium graminearum\nAuthors: Darino, M.; Urban, M.; Kaur, N.; Machado-Wood, A.; Grimwade-Mann, M.; Smith, D.; Beacham, A.; Hammond-kosack, K.\nScore: 10.4, Published: 2023-08-21 DOI: 10.1101/2023.08.20.553861\nBackgroundFusarium Head Blight is a destructive floral disease of different cereal crops. The Ascomycete fungus Fusarium graminearum (Fg) is one of the main causal agents of FHB in wheat and barley. The role(s) in virulence of Fg genes include genetic studies that involve the transformation of the fungus with different expression cassettes. We have observed in several studies where Fg genes functions were characterised that integration of expression cassettes occurred randomly. Random insertion of a cassette may disrupt gene expression and/or protein functions and hence the overall conclusion of the study. Target site integration (TSI) is an approach that consists in identifying a chromosomal region where the cassette can be inserted. The identification of a suitable locus for TSI in Fg would avert the potential risks of ectopic integration. ResultsHere, we identified a highly conserved intergenic region on chromosome 1 suitable for TSI. We named this intergenic region the TSI locus 1. We developed an efficient cloning vector system based on the Golden Gate method to clone different expression cassettes for use in combination with TSI locus 1. We present evidence that integrations in the TSI locus 1 affects neither fungal virulence nor fungal growth under different stress conditions. Integrations at the TSI locus 1 resulted in the expression of different gene fusions. In addition, the activities of Fg native promoters were not altered by integration into the TSI locus 1. We have developed a bespoke bioinformatic pipeline to analyse the existence of ectopic integrations and tandem insertions of the cassette that may occurred during the transformation process. Finally, we established a protocol to study protein secretion in wheat coleoptiles using confocal microscopy and the TSI locus 1. ConclusionThe TSI locus 1 can be used in Fg and potentially other cereal infecting Fusarium species for diverse studies including promoter activity analysis, secretion, protein localisation studies and gene complementation. The bespoke bioinformatic pipeline developed in this work can be an alternative to southern blotting, the gold standard technique to identify ectopic integration and tandem insertions in fungal transformation.\nFemale germline expression of OVO transcription factor bridges Drosophila generations\nAuthors: Benner, L.; Muron, S.; Oliver, B.\nScore: 7.7, Published: 2023-08-26 DOI: 10.1101/2023.08.25.554887\nOVO is required for karyotypically female germ cell viability but has no known function in the male germline in Drosophila. ovo is autoregulated by two antagonistic isoforms, OVO-A and OVO-B. All ovo- alleles were created as partial revertants of the antimorphic ovoD1allele. Creation of new targeted alleles in an ovo+ background indicated that disrupting the germline-specific exon extension of ovo-B leads to an arrested egg chamber phenotype, rather than germ cell death. RNA-seq analysis, including \u003e1K full length cDNAs, indicates that ovo utilizes a number of unannotated splice variations in the extended exon and a minor population of ovo-B transcripts utilizes an alternative splice. This indicates that classical ovo alleles such as ovoD1rv23, are not truly null for ovo, and are likely to be weak antimorphs. To generate bonafide nulls, we deleted the ovo-A and ovo-B promoters showing that only ovo-B is required for female germ cell viability and there is an early and polyphasic developmental requirement for ovo-B in the female germline. To visualize OVO expression and localization, we endogenously tagged ovo and found nuclear OVO in all differentiating female germ cells throughout oogenesis in adults. We also found that OVO is maternally deposited into the embryo, where it showed nuclear localization in newly formed pole cells. Maternal OVO persisted in embryonic germ cells until zygotic OVO expression was detectable, suggesting that there is continuous nuclear OVO expression in the female germline in the transition from one generation to the next. Article Summaryovo has long been considered to be at the top of the female germline sex determination pathway. We utilized updated genetic methods to determine OVO expression, localization, and requirement in the embryonic and adult germline. Our results indicate that OVO is always present, and likely required, in the Drosophila female germline.\nThe role of heterochronic gene expression and regulatory architecture in early developmental divergence\nAuthors: Harry, N. D.; Zakas, C.\nScore: 7.9, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553877\nNew developmental programs can evolve through adaptive changes to gene expression. The annelid S. benedicti has a developmental dimorphism, which provides a unique intraspecific framework for understanding the earliest genetic changes that take place during developmental divergence. Using comparative RNAseq through ontogeny, we find that only a small proportion of genes are differentially expressed at any time, despite major differences in larval development and life-history. These genes shift expression profiles across morphs by either turning off any expression in one morph or changing the timing or amount of gene expression. We directly connect the contributions of these mechanisms to differences in developmental processes. We examine F1 offspring-- using reciprocal crosses-- to determine maternal mRNA inheritance and the regulatory architecture of gene expression. These results highlight the importance of both novel gene expression and heterochronic shifts in developmental evolution, as well as the trans-acting regulatory factors in initiating divergence.\n",
  "wordCount" : "2731",
  "inLanguage": "en",
  "datePublished": "2023-09-06T10:39:07Z",
  "dateModified": "2023-09-06T10:39:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on September 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552281">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552281" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552281">
        <p class="paperTitle">Leveraging fine-scale population structure reveals conservation in genetic effect sizes between human populations across a range of human phenotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552281" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552281" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, S.; Ferreira, L. A. F.; Shi, S.; Hellenthal, G.; Marchini, J.; Lawson, D. J.; Myers, S. R.</p>
        <p class="info">Score: 56.3, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552281' target='https://doi.org/10.1101/2023.08.08.552281'> 10.1101/2023.08.08.552281</a></p>
        <p class="abstract">An understanding of genetic differences between populations is essential for avoiding confounding in genome-wide association studies (GWAS) and understanding the evolution of human traits. Polygenic risk scores constructed in one group perform poorly in highly genetically-differentiated populations, for reasons which remain controversial. We developed a statistical ancestry inference pipeline able to decompose ancestry both within and between countries, and applied it to the UK Biobank data. This identifies fine-scale patterns of genetic relatedness not captured by standard and widely used principal components (PCs), and allows fine-scale population stratification correction that removes both false positive and false negative associations for traits with geographic correlations. We also develop and apply ANCHOR, an approach leveraging segments of distinct ancestries within individuals to estimate similarity in underlying causal effect sizes between groups, using an existing PGS. Applying ANCHOR to &gt;8000 people of mixed African and European ancestry, we demonstrate that estimated causal effect sizes are highly similar across these ancestries for 26 of 29 quantitative molecular and non-molecular phenotypes (mean correlation 0.98 &#43;/-0.08), providing evidence that gene-environment and gene-gene interactions do not play major roles in the poor prediction of European-ancestry PRS scores in African populations for these traits, contradicting previous findings. Instead our results provide optimism that shared causal mutations operate similarly in different groups, focussing the challenge of improving GWAS &#34;portability&#34; between groups on joint fine-mapping.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555114">
        <p class="paperTitle">Ancient farmer and steppe pastoralist-related founding lineages contributed to the complex landscape of episodes in the diversification of Chinese paternal lineages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, M.; Huang, Y.; Liu, K.; Yuan, H.; Duan, S.; Wang, Z.; Wei, L.; Yao, H.; Sun, Q.; Zhong, J.; Tang, R.; Chen, J.; Sun, Y.; Li, X.; Su, H.; Yang, Q.; Hu, L.; Yun, L.; Yang, J.; Nie, S.; Cai, Y.; Yan, J.; Zhou, K.; 10K_CPGDP Consortium,  ; Wang, C.-C.; Zhu, B.; Liu, C.; He, G.</p>
        <p class="info">Score: 29.0, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555114' target='https://doi.org/10.1101/2023.08.28.555114'> 10.1101/2023.08.28.555114</a></p>
        <p class="abstract">Ancient DNA advances have reported the complex genetic history of Eurasians, but how the knowledge of ancient subsistence strategy shifts and population movements influenced the fine-scale paternal genetic structure in East Asia has not been assessed. Here, we reported one integrated Y-chromosome genomic database of 15,530 people, including 1753 ancient people and newly-reported 919 individuals genotyped using our recently-developed targeted sequencing YHSeqY3000 panel, to explore Chinese genomic diversity, population evolutionary tracts and their genetic formation mechanism. We identified four major ancient technological innovations and population movements that shaped the landscape of Chinese paternal lineages. First, the expansion of millet farmers and early East Asians from the Yellow River Basin carrying the major O2/D subclades promoted the formation of the Sino-Tibetan peoples major composition and accelerated the Tibetan Plateaus permanent occupation. Second, rice farmers dispersal from the Yangtze River Valley carrying O1 and some sublineages of O2 contributed significantly to Tai-Kadai, Austronesian, Hmong-Mien, Austroasiatic people and southern Han Chinese. Third, Siberian-related paternal lineages of Q and C originated and boomed from Neolithic hunter-gatherers from the Mongolian Plateau and the Amur River Basin and significantly influenced the gene pools of northern Chinese. Fourth, western Eurasian-derived J, G and R lineages initially spread with Yamnaya steppe pastoralists and other proto-Indo-European people and further widely dispersed via the trans-Eurasian cultural communication along the Eurasian Steppe and the ancient Silk Road, remaining genetic trajectories in northwestern Chinese. Our work provided comprehensive modern and ancient genetic evidence to illuminate the impact of population interaction from the ancient farmer or herder-based societies on the genetic diversity patterns of modern people, revised our understandings of ancestral sources of Chinese paternal lineages, underscored the scientific imperative of the large-scale genomic resources of dense spatiotemporal underrepresented sampling populations to understand human evolutionary history.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555359">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555359" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555359">
        <p class="paperTitle">Functional characterization of gene regulatory elements and neuropsychiatric disease-associated risk loci in iPSCs and iPSC-derived neurons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555359" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555359" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, X.; Jones, I. R.; Chen, P. B.; Yang, H.; Ren, X.; Zheng, L.; Li, B.; Li, Y. E.; Sun, Q.; Wen, J.; Beaman, C.; Cui, X.; Li, Y.; Wang, W.; Hu, M.; Ren, B.; Shen, Y.</p>
        <p class="info">Score: 27.0, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555359' target='https://doi.org/10.1101/2023.08.30.555359'> 10.1101/2023.08.30.555359</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have identified thousands of non-coding variants that contribute to psychiatric disease risks, likely by perturbing cis-regulatory elements (CREs). However, our ability to interpret and explore their mechanisms of action is hampered by a lack of annotation of functional CREs (fCREs) in neural cell types. Here, through genome-scale CRISPR screens of 22,000 candidate CREs (cCREs) in human induced pluripotent stem cells (iPSCs) undergoing differentiation to excitatory neurons, we identify 2,847 and 5,540 fCREs essential for iPSC fitness and neuronal differentiation, respectively. These fCREs display dynamic epigenomic features and exhibit increased numbers and genomic spans of chromatin interactions following terminal neuronal differentiation. Furthermore, fCREs essential for neuronal differentiation show significantly greater enrichment of genetic heritability for neurodevelopmental diseases including schizophrenia (SCZ), attention deficit hyperactivity disorder (ADHD), and autism spectrum disorders (ASD) than cCREs. Using high-throughput prime editing screens we experimentally confirm 45 SCZ risk variants that act by affecting the function of fCREs. The extensive and in-depth functional annotation of cCREs in neuronal types therefore provides a crucial resource for interpreting non-coding risk variants of neuropsychiatric disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555220">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555220" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555220">
        <p class="paperTitle">A defined diet for pre-adult Drosophila melanogaster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555220" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555220" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martelli, F.; Quig, A.; Mele, S.; Lin, J.; Fulton, T. L.; Wansbrough, M.; Barlow, C. K.; Schittenhelm, R. B.; Johnson, T. K.; Piper, M. D.</p>
        <p class="info">Score: 14.4, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555220' target='https://doi.org/10.1101/2023.08.28.555220'> 10.1101/2023.08.28.555220</a></p>
        <p class="abstract">Drosophila melanogaster is unique among animal models because it has a fully defined synthetic diet available to study nutrient-gene interactions. However, use of this diet is limited to adult studies due to impaired larval development and survival. Here, we provide an adjusted formula that improves larval growth, development rate, and rescues survival. We demonstrate its use for exploring pre-adult diet compositions of therapeutic potential in a model of an inherited metabolic disorder.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555118">
        <p class="paperTitle">Absence of chromosome axis proteins recruitment prevents meiotic recombination chromosome-wide in the budding yeast Lachancea kluyveri</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Legrand, S.; Saifudeen, A.; Bordelet, H.; Vernerey, J.; Guille, A.; Bignaut, A.; Thierry, A.; Acquaviva, L.; Gaudin, M.; Sanchez, A.; Johnson, D.; Friedrcih, A.; Schacherer, J.; Neale, M.; Borde, V.; Koszul, R.; llorente, b.</p>
        <p class="info">Score: 13.2, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555118' target='https://doi.org/10.1101/2023.08.28.555118'> 10.1101/2023.08.28.555118</a></p>
        <p class="abstract">Meiotic recombination shows broad variations across species and along chromosomes, and is often suppressed at and around genomic regions determining sexual compatibility such as mating type loci in fungi. Here we show that the absence of Spo11-DSBs and meiotic recombination on Lakl0C-left, the chromosome arm containing the sex locus of the Lachancea kluyveri budding yeast, results from the absence of recruitment of the two chromosome axis proteins Red1 and Hop1, essential for proper Spo11-DSBs formation. Furthermore, cytological observation of spread pachytene meiotic chromosomes reveals that Lakl0C-left does not undergo synapsis. However, we show that the behavior of Lakl0C-left is independent of its particularly early replication timing and is not accompanied by any peculiar chromosome structure as detectable by Hi-C in this yet poorly studied yeast. Finally, we observed an accumulation of heterozygous mutations on Lakl0C-left and a sexual dimorphism of the haploid meiotic offspring, supporting a direct effect of this absence of meiotic recombination on L. kluyveri genome evolution and fitness. Because suppression of meiotic recombination on sex chromosomes is widely observed across eukaryotes, the novel mechanism for recombination suppression described here may apply to other species, with the potential to impact sex chromosome evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555563">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555563" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555563">
        <p class="paperTitle">Generation of isogenic models of Angelman syndrome and Prader-Willi syndrome in CRISPR/Cas9-engineered human embryonic stem cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555563" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555563" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gilmore, R. B.; Gorka, D.; Stoddard, C.; Cotney, J.; Chamberlain, S.</p>
        <p class="info">Score: 8.8, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555563' target='https://doi.org/10.1101/2023.08.30.555563'> 10.1101/2023.08.30.555563</a></p>
        <p class="abstract">Angelman Syndrome (AS) and Prader-Willi Syndrome (PWS), two distinct neurodevelopmental disorders, result from loss of expression from imprinted genes in the chromosome 15q11-13 locus most commonly caused by a megabase-scale deletion on either the maternal or paternal allele, respectively. Each occurs at an approximate incidence of 1/15,000 to 1/30,000 live births and has a range of debilitating phenotypes. Patient-derived induced pluripotent stem cells (iPSCs) have been valuable tools to understand human-relevant gene regulation at this locus and have contributed to the development of therapeutic approaches for AS. Nonetheless, gaps remain in our understanding of how these deletions contribute to dysregulation and phenotypes of AS and PWS. Variability across cell lines due to donor differences, reprogramming methods, and genetic background make it challenging to fill these gaps in knowledge without substantially increasing the number of cell lines used in the analyses. Isogenic cell lines that differ only by the genetic mutation causing the disease can ease this burden without requiring such a large number of cell lines. Here, we describe the development of isogenic human embryonic stem cell (hESC) lines modeling the most common genetic subtypes of AS and PWS. These lines allow for a facile interrogation of allele-specific gene regulation at the chromosome 15q11-q13 locus. Additionally, these lines are an important resource to identify and test targeted therapeutic approaches for patients with AS and PWS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555906">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555906" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555906">
        <p class="paperTitle">Natural genetic variation quantitatively regulates heart-rate and -dimension</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555906" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555906" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gierten, J.; Welz, B.; Fitzgerald, T. W.; Thumberger, T.; Hummel, O.; Leger, A.; Weber, P.; Hassel, D.; Hubner, N.; Birney, E.; Wittbrodt, J.</p>
        <p class="info">Score: 7.9, Published: 2023-09-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555906' target='https://doi.org/10.1101/2023.09.01.555906'> 10.1101/2023.09.01.555906</a></p>
        <p class="abstract">Development and function of the heart are crucially controlled by individual genetic factors determining the continuum from health to disease. A prime example of a quantitative pathological phenotype is left ventricular hypoplasia, affecting the left ventricle (LV) and its associated inflow and outflow structures and reducing its size to varying degrees. Current models suggest polygenic contributions on heart development and disease, but we are just beginning to understand the impact of naturally occurring sequence variations. Here we show how natural genetic variants exert cumulative, environment-sensitive effects on both embryonic heart development as well as adult heart function. We addressed this in embryos of five inbred strains of the Japanese rice fish medaka (Oryzias latipes and Oryzias sakaizumii), analyzing heart rates as a functional proxy to capture combined characteristics of the conduction system and structural morphology. We identified one strain, HO5, with embryonic hypoplastic ventricles, fast heart rates, and significantly impaired cardiac function and overall fitness in adults. We created an interpopulation cross of the divergent HO5 strain to HdrR, which exhibits a morphologically normal heart and comparably slow heart rates, to leverage segregation variance in the second filial generation (F2) for genetic mapping. The correlation of 1192 individually phenotyped F2 embryos (heart rate) with the corresponding whole-genome sequences established a single nucleotide polymorphism-based linkage map. Integrated analysis of parental cardiac transcriptomes highlighted 59 loci containing genes prominently linked to human cardiovascular phenotypes. In vivo knockout models of the top 12 candidate genes demonstrated a causal and differential impact on heart development, ventricular size, and heart rate, evidencing the polygenic nature of heart phenotypes and establishing new loci for genetic diagnostics of human congenital heart disease. The controllable and scalable genetics and experimental validation in inbred vertebrate models complement and expand human association studies to provide novel and unique insights into the role of natural genetic variation in the mechanism of cardiac disease and allow to identify genetic susceptibility in individual human subjects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.553861">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.553861" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.553861">
        <p class="paperTitle">Identification and functional characterisation of a locus for target site integration in Fusarium graminearum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.553861" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.553861" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Darino, M.; Urban, M.; Kaur, N.; Machado-Wood, A.; Grimwade-Mann, M.; Smith, D.; Beacham, A.; Hammond-kosack, K.</p>
        <p class="info">Score: 10.4, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.553861' target='https://doi.org/10.1101/2023.08.20.553861'> 10.1101/2023.08.20.553861</a></p>
        <p class="abstract">BackgroundFusarium Head Blight is a destructive floral disease of different cereal crops. The Ascomycete fungus Fusarium graminearum (Fg) is one of the main causal agents of FHB in wheat and barley. The role(s) in virulence of Fg genes include genetic studies that involve the transformation of the fungus with different expression cassettes. We have observed in several studies where Fg genes functions were characterised that integration of expression cassettes occurred randomly. Random insertion of a cassette may disrupt gene expression and/or protein functions and hence the overall conclusion of the study. Target site integration (TSI) is an approach that consists in identifying a chromosomal region where the cassette can be inserted. The identification of a suitable locus for TSI in Fg would avert the potential risks of ectopic integration.

ResultsHere, we identified a highly conserved intergenic region on chromosome 1 suitable for TSI. We named this intergenic region the TSI locus 1. We developed an efficient cloning vector system based on the Golden Gate method to clone different expression cassettes for use in combination with TSI locus 1. We present evidence that integrations in the TSI locus 1 affects neither fungal virulence nor fungal growth under different stress conditions. Integrations at the TSI locus 1 resulted in the expression of different gene fusions. In addition, the activities of Fg native promoters were not altered by integration into the TSI locus 1. We have developed a bespoke bioinformatic pipeline to analyse the existence of ectopic integrations and tandem insertions of the cassette that may occurred during the transformation process. Finally, we established a protocol to study protein secretion in wheat coleoptiles using confocal microscopy and the TSI locus 1.

ConclusionThe TSI locus 1 can be used in Fg and potentially other cereal infecting Fusarium species for diverse studies including promoter activity analysis, secretion, protein localisation studies and gene complementation. The bespoke bioinformatic pipeline developed in this work can be an alternative to southern blotting, the gold standard technique to identify ectopic integration and tandem insertions in fungal transformation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554887">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554887" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554887">
        <p class="paperTitle">Female germline expression of OVO transcription factor bridges Drosophila generations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554887" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554887" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benner, L.; Muron, S.; Oliver, B.</p>
        <p class="info">Score: 7.7, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554887' target='https://doi.org/10.1101/2023.08.25.554887'> 10.1101/2023.08.25.554887</a></p>
        <p class="abstract">OVO is required for karyotypically female germ cell viability but has no known function in the male germline in Drosophila. ovo is autoregulated by two antagonistic isoforms, OVO-A and OVO-B. All ovo- alleles were created as partial revertants of the antimorphic ovoD1allele. Creation of new targeted alleles in an ovo&#43; background indicated that disrupting the germline-specific exon extension of ovo-B leads to an arrested egg chamber phenotype, rather than germ cell death. RNA-seq analysis, including &gt;1K full length cDNAs, indicates that ovo utilizes a number of unannotated splice variations in the extended exon and a minor population of ovo-B transcripts utilizes an alternative splice. This indicates that classical ovo alleles such as ovoD1rv23, are not truly null for ovo, and are likely to be weak antimorphs. To generate bonafide nulls, we deleted the ovo-A and ovo-B promoters showing that only ovo-B is required for female germ cell viability and there is an early and polyphasic developmental requirement for ovo-B in the female germline. To visualize OVO expression and localization, we endogenously tagged ovo and found nuclear OVO in all differentiating female germ cells throughout oogenesis in adults. We also found that OVO is maternally deposited into the embryo, where it showed nuclear localization in newly formed pole cells. Maternal OVO persisted in embryonic germ cells until zygotic OVO expression was detectable, suggesting that there is continuous nuclear OVO expression in the female germline in the transition from one generation to the next.

Article Summaryovo has long been considered to be at the top of the female germline sex determination pathway. We utilized updated genetic methods to determine OVO expression, localization, and requirement in the embryonic and adult germline. Our results indicate that OVO is always present, and likely required, in the Drosophila female germline.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553877">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553877" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553877">
        <p class="paperTitle">The role of heterochronic gene expression and regulatory architecture in early developmental divergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553877" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553877" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harry, N. D.; Zakas, C.</p>
        <p class="info">Score: 7.9, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553877' target='https://doi.org/10.1101/2023.08.18.553877'> 10.1101/2023.08.18.553877</a></p>
        <p class="abstract">New developmental programs can evolve through adaptive changes to gene expression. The annelid S. benedicti has a developmental dimorphism, which provides a unique intraspecific framework for understanding the earliest genetic changes that take place during developmental divergence. Using comparative RNAseq through ontogeny, we find that only a small proportion of genes are differentially expressed at any time, despite major differences in larval development and life-history. These genes shift expression profiles across morphs by either turning off any expression in one morph or changing the timing or amount of gene expression. We directly connect the contributions of these mechanisms to differences in developmental processes. We examine F1 offspring-- using reciprocal crosses-- to determine maternal mRNA inheritance and the regulatory architecture of gene expression. These results highlight the importance of both novel gene expression and heterochronic shifts in developmental evolution, as well as the trans-acting regulatory factors in initiating divergence.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
