<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deletion of Xist upstream sequences alters TAD interactions and leads to defects in Xist coating and expression
Authors: Majumdar, S.; Bammidi, L. S.; Naik, H.; M, A.; Baro, R.; Kalita, A.; Sundarraj, N.; Bariha, G.; Notani, D.; GAYEN, S.
Score: 17.7, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553118
The topological organization of the genome plays an important role in regulating gene expression. However, the connection between the two remains poorly understood. X-chromosome inactivation is a unique model system to explore the interlink between topologically associated domains (TADs) and gene expression.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Deletion of Xist upstream sequences alters TAD interactions and leads to defects in Xist coating and expression
Authors: Majumdar, S.; Bammidi, L. S.; Naik, H.; M, A.; Baro, R.; Kalita, A.; Sundarraj, N.; Bariha, G.; Notani, D.; GAYEN, S.
Score: 17.7, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553118
The topological organization of the genome plays an important role in regulating gene expression. However, the connection between the two remains poorly understood. X-chromosome inactivation is a unique model system to explore the interlink between topologically associated domains (TADs) and gene expression." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-20T10:35:54+00:00" />
<meta property="article:modified_time" content="2023-08-20T10:35:54+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Deletion of Xist upstream sequences alters TAD interactions and leads to defects in Xist coating and expression
Authors: Majumdar, S.; Bammidi, L. S.; Naik, H.; M, A.; Baro, R.; Kalita, A.; Sundarraj, N.; Bariha, G.; Notani, D.; GAYEN, S.
Score: 17.7, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553118
The topological organization of the genome plays an important role in regulating gene expression. However, the connection between the two remains poorly understood. X-chromosome inactivation is a unique model system to explore the interlink between topologically associated domains (TADs) and gene expression."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Deletion of Xist upstream sequences alters TAD interactions and leads to defects in Xist coating and expression\nAuthors: Majumdar, S.; Bammidi, L. S.; Naik, H.; M, A.; Baro, R.; Kalita, A.; Sundarraj, N.; Bariha, G.; Notani, D.; GAYEN, S.\nScore: 17.7, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553118\nThe topological organization of the genome plays an important role in regulating gene expression. However, the connection between the two remains poorly understood. X-chromosome inactivation is a unique model system to explore the interlink between topologically associated domains (TADs) and gene expression.",
  "keywords": [
    
  ],
  "articleBody": " Deletion of Xist upstream sequences alters TAD interactions and leads to defects in Xist coating and expression\nAuthors: Majumdar, S.; Bammidi, L. S.; Naik, H.; M, A.; Baro, R.; Kalita, A.; Sundarraj, N.; Bariha, G.; Notani, D.; GAYEN, S.\nScore: 17.7, Published: 2023-08-15 DOI: 10.1101/2023.08.14.553118\nThe topological organization of the genome plays an important role in regulating gene expression. However, the connection between the two remains poorly understood. X-chromosome inactivation is a unique model system to explore the interlink between topologically associated domains (TADs) and gene expression. TADs are largely lost upon X-inactivation, and the inactive-X gets bipartitely reorganized into two large mega domains. However, the X-inactivation center (XIC) harbors two TADs - at the locus of long non-coding RNA Xist (Xist-TAD) and Tsix (Tsix-TAD). Xist is the master regulator of X-inactivation, which coat the inactive-X and facilitates heterochromatinization. Here, we deleted Xist upstream sequences ([~]6 kb) near the Xist TADs boundary in extraembryonic endoderm stem cells (XEN), which undergo imprinted X-inactivation. This deletion led to the major rearrangement of TADs and affected the expression of genes located within Xist and Tsix TAD, specially the expression of Xist was upregulated, suggesting TADs are essential for proper transcriptional regulation. On the other hand, Xist-upstream deletion on the inactive-X resulted in dispersal of Xist coating and loss of enrichment of repressive chromatin marks on the inactive-X but no effect on X-linked gene silencing. However, we found that autosomal genes were dysregulated in Xist-upstream deleted cells, probably because of misregulation of genes located in Xist and Tsix-TAD, specially Xist. We conclude that Xist upstream sequences are necessary for proper organization of the TADs at the XIC, maintenance of Xist coating/expression and autosomal gene expression.\nExpanding the Drosophila toolkit for dual control of gene expression\nAuthors: Zirin, J.; Jusiak, B.; Lopes, R.; Ewen-Campen, B.; Bosch, J.; Villalta, C.; Forman, C.; Hu, Y.; Mohr, S.; Perrimon, N.\nScore: 15.5, Published: 2023-08-16 DOI: 10.1101/2023.08.15.553399\nSummaryHaving tools and resources for independent control of gene expression in two different tissues in the same animal is emerging as a major need, especially in the context of inter-organ communication studies. This type of study is made possible by technologies combining the GAL4/UAS and a second binary expression system such as LexA/LexAop or QF/QUAS. Here, we describe a resource of reagents that facilitate combined use of the GAL4/UAS and a second binary systems in various tissues. Focusing on genes with well-characterized GAL4 expression patterns, we generated a set of more than 40 LexA-GAD and QF2 insertions by CRISPR knock-in and verified their tissue-specificity in larvae. We also built constructs that encode QF2 and LexA-GAD transcription factors in a single vector, with each coding sequence flanked by FRT sites. Following successful integration into the fly genome, the vector generates a single polypeptide, which is separated into the individual drivers by the inclusion of T2A. If desired, one of the two coding regions can then be excised with Flp, resulting in flies that express only QF2 or LexA-GAD. We evaluated both QF2 and LexA-GAD systems for in vivo gene knockdown and are generating a compatible library of transgenic shRNA lines in our custom QUAS and LexAop vectors as a community resource. Together, these QF2/LexA-GAD and QUAS/LexAop vectors and fly lines will provide a new set of tools for researchers who need to activate or repress two different genes in an orthogonal manner in the same animal.\nShared and distinct molecular effects of regulatory genomic variants provide insight into mechanisms of distal enhancer-promoter communication\nAuthors: Ray-Jones, H.; Song, Z.; Haglund, A.; Artemov, P.; Della Rosa, M.; Burden, F.; Kreuzhuber, R.; Litovskikh, A.; Tan, V. X. H.; Chan, L. T.; Frontini, M.; Wallace, C.; Malysheva, V.; Bottolo, L.; Vigorito, E.; Spivakov, M.\nScore: 15.1, Published: 2023-08-15 DOI: 10.1101/2023.08.04.551251\nTranscriptional enhancers control spatiotemporal gene expression and commonly engage in long-range chromosomal contacts with gene promoters. However, the relationship between enhancer activity, enhancer-promoter contact and gene expression is not completely understood. Here, we leverage human genetic variation as a \"natural perturbation\" to dissect this relationship. Focusing on enhancers that harbour genetic variants showing allelic associations with the expression levels of distal genes (expression quantitative trait loci, eQTLs), we profiled eQTL-promoter contacts using high-resolution Capture Hi-C, as well as chromatin activity by ATAC-sequencing, in primary monocytes isolated from 34 healthy donors. To detect the associations of genetic variants with these molecular phenotypes, we extended a recently proposed Bayesian method that leverages both intra- and interindividual variation. We identified 19 eQTLs associated with differential promoter contacts, most of which also showed an association with chromatin activity. Capitalising on this overlap, we next adapted a Bayesian mapping strategy for detecting variants that exert genetic control across multiple phenotypes, revealing hundreds of putative \"trimodal QTL\" variants jointly associated with enhancer activity, connectivity, and gene expression. A subset of these variants was predicted to affect the binding of a diverse range of transcription factors, in part through disruption of the binding motifs for either these factors or their binding partners. In addition, we identified a variant with opposite effects on promoter contact and expression levels of its target gene, PCK2. We show that this variant disrupts the canonical binding motif of the architectural protein CTCF and likely perturbs the chromatin insulation of the PCK2 promoter from more distal enhancers. Finally, multiple identified QTLs overlapped known disease susceptibility loci, pointing to the potential role of enhancer-promoter communication in mediating the pathological effects of non-coding genetic variation. Jointly, our findings suggest an inherent functional link between the activity and connectivity of enhancers with relevance for human disease and highlight the role of genetically-determined chromatin boundaries in gene expression control.\nGetPrimers: a generalized PCR-based genetic targeting primer designer enabling easy and standardized targeted gene modification across multiple systems\nAuthors: Miao, Z.; Wang, H.; Tu, X.; Huang, Z.; Huang, S.; Zhang, X.; Wang, F.; Huang, Z.; Li, H.; Jiao, Y.; Gao, S.; Zhou, Z.; Shan, C.-M.; Li, J.; Yue, J.-X.\nScore: 8.3, Published: 2023-08-14 DOI: 10.1101/2023.08.12.552711\nGenetic targeting (e.g., gene knockout and tagging) based on polymerase chain reaction (PCR) is a simple yet powerful approach for studying gene functions. Although originally developed in classic budding and fission yeast models, the same principle applies to other eukaryotic systems with efficient homologous recombination. One-step-PCR-based genetic targeting is conventionally used but the sizes of the homologous arms that it generates for recombination-mediated genetic targeting are usually limited. Alternatively, gene targeting can also be performed via fusion PCR, which can create homologous arms that are orders of magnitude larger, therefore substantially increases the efficiency of recombination-mediated genetic targeting. Here we present GetPrimers (https://www.evomicslab.org/app/getprimers/), a generalized computational framework and web tool to assist automatic targeting and verification primer design for both one-step-PCR-based and fusion-PCR-based genetic targeting experiments. Moreover, GetPrimers by design runs for any given genetic background of any species with full genome scalability. Therefore, GetPrimers is capable of empowering high-throughput functional genomics assays at multi-population and multi-species levels. Comprehensive experimental validations have been performed for targeting and verification primers designed by GetPrimers across multiple organism systems and experimental setups. We anticipate GetPrimers to become a highly useful and popular tool to facilitate easy and standardized gene modification across multiple systems. KeypointsDeveloped the web tool GetPrimers to assist automatic primer design for PCR-based genetic targeting. It runs for any given locus on any genetic background of any given species with full genome scalability. Experimental validations have been performed across multiple organism systems and experimental setups.\nMethods and techniques enabling multi-kilobase long-range genomic rewrite/replace editing\nAuthors: Piggott, C. A.; Tjang, L. V.; Chen, S.; Tatu, U.; Hirou, H.; Jaroszewska, A.; McIntyre, E.; Chen, J.; Faybishenko, J. V.; Gavan, R.; Hackley, C.\nScore: 11.8, Published: 2023-08-06 DOI: 10.1101/2023.08.05.551844\nCRISPR enabled cell and gene therapies have the potential to revolutionize the field of genetic medicine. However, the vast majority of rare diseases remain untreatable due to the limitations of current tools and techniques. To date, most corrective therapeutic approaches have been restricted to mutation-by-mutation approaches, where either HDR, or newer techniques such as base or prime editing, rewrite small regions of DNA at a time ([~]1-100 bp). While these approaches are powerful, short editing windows (relative to the size of human genes) are financially and/or technically incompatible with most rare-disease mutation profiles. Here, we demonstrate for the first time that CRISPR/Cas9 can be used to \"rewrite\" 7kb+ sections of the human genome simultaneously via a selection-free process we have named \"long-range rewriting\". Long-range rewriting approaches are compatible with multiple nucleases, cell types and genomic loci, and can be used with both double-strand break (DSB) and non-DSB based approaches.\nCryptoCEN: A Co-Expression Network for Cryptococcus neoformans reveals novel proteins involved in DNA damage repair\nAuthors: O'Meara, M. J.; Rapala, J. R.; Nichols, C. B.; Alexandre, C.; Billmyre, R. B.; Steenwyk, J. L.; Alspaugh, J. A.; O'Meara, T. R.\nScore: 5.3, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553567\nElucidating gene function is a major goal in biology, especially among non-model organisms. However, doing so is complicated by the fact that molecular conservation does not always mirror functional conservation, and that complex relationships among genes are responsible for encoding pathways and higher-order biological processes. Co-expression, a promising approach for predicting gene function, relies on the general principal that genes with similar expression patterns across multiple conditions will likely be involved in the same biological process. For Cryptococcus neoformans, a prevalent human fungal pathogen greatly diverged from model yeasts, approximately 60% of the predicted genes in the genome lack functional annotations. Here, we leveraged a large amount of publicly available transcriptomic data to generate a C. neoformans Co-Expression Network (CryptoCEN), successfully recapitulating known protein networks, predicting gene function, and enabling insights into the principles influencing co-expression. With 100% predictive accuracy, we used CryptoCEN to identify 13 new DNA damage response genes, underscoring the utility of guilt-by-association for determining gene function. Overall, co-expression is a powerful tool for uncovering gene function, and decreases the experimental tests needed to identify functions for currently under-annotated genes.\nA fluorescent sex-sorting technique for insects with the demonstration in Drosophila melanogaster\nAuthors: Liu, J.; Rayes, D.; Akbari, O.\nScore: 4.0, Published: 2023-08-14 DOI: 10.1101/2023.08.11.553026\nRecent advances in insect genetic engineering offer alternative genetic biocontrol solutions to control populations of pests and disease vectors. While success has been achieved, sex-sorting remains problematic for scaling many genetic biocontrol interventions. Here we describe the development of a sex-sorting technique for female and male selection with a proof-of-concept in D. melanogaster termed SEPARATOR (Sexing Element Produced by Alternative RNA-splicing of A Transgenic Observable Reporter). This approach utilizes dominant fluorescent proteins and differentially spliced introns to ensure sex-specific expression. The system has the potential for adaptability to various insect species and application for high-throughput insect sex-sorting.\nLeveraging fine-scale population structure reveals conservation in genetic effect sizes between human populations across a range of human phenotypes\nAuthors: Hu, S.; Ferreira, L. A. F.; Shi, S.; Hellenthal, G.; Marchini, J.; Lawson, D. J.; Myers, S. R.\nScore: 28.4, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552281\nAn understanding of genetic differences between populations is essential for avoiding confounding in genome-wide association studies (GWAS) and understanding the evolution of human traits. Polygenic risk scores constructed in one group perform poorly in highly genetically-differentiated populations, for reasons which remain controversial. We developed a statistical ancestry inference pipeline able to decompose ancestry both within and between countries, and applied it to the UK Biobank data. This identifies fine-scale patterns of genetic relatedness not captured by standard and widely used principal components (PCs), and allows fine-scale population stratification correction that removes both false positive and false negative associations for traits with geographic correlations. We also develop and apply ANCHOR, an approach leveraging segments of distinct ancestries within individuals to estimate similarity in underlying causal effect sizes between groups, using an existing PGS. Applying ANCHOR to \u003e8000 people of mixed African and European ancestry, we demonstrate that estimated causal effect sizes are highly similar across these ancestries for 26 of 29 quantitative molecular and non-molecular phenotypes (mean correlation 0.98 +/-0.08), providing evidence that gene-environment and gene-gene interactions do not play major roles in the poor prediction of European-ancestry PRS scores in African populations for these traits, contradicting previous findings. Instead our results provide optimism that shared causal mutations operate similarly in different groups, focussing the challenge of improving GWAS \"portability\" between groups on joint fine-mapping.\nMitochondrial Genome Diversity across the Subphylum Saccharomycotina\nAuthors: Wolters, J. F.; LaBella, A.; Opulente, D.; Rokas, A.; Hittinger, C. T.\nScore: 16.4, Published: 2023-07-31 DOI: 10.1101/2023.07.28.551029\nEukaryotic life depends on the functional elements encoded by both the nuclear genome and organellar genomes, such as those contained within the mitochondria. The content, size, and structure of the mitochondrial genome varies across organisms with potentially large implications for phenotypic variance and resulting evolutionary trajectories. Among yeasts in the subphylum Saccharomycotina, extensive differences have been observed in various species relative to the model yeast Saccharomyces cerevisiae, but mitochondrial genome sampling across many groups has been scarce, even as hundreds of nuclear genomes have become available. By extracting mitochondrial assemblies from existing short-read genome sequence datasets, we have greatly expanded both the number of available genomes and the coverage across sparsely sampled clades. Comparison of 353 yeast mitochondrial genomes revealed that, while size and GC content were fairly consistent across species, those in the genera Metschnikowia and Saccharomyces trended larger, while several species in the order Saccharomycetales, which includes S. cerevisiae, exhibited lower GC content. Extreme examples for both size and GC content were scattered throughout the subphylum. All mitochondrial genomes shared a core set of protein-coding genes for Complexes III, IV, and V, but they varied in the presence or absence of mitochondrially-encoded canonical Complex I genes. We traced the loss of Complex I genes to a major event in the ancestor of the orders Saccharomycetales and Saccharomycodales, but we also observed several independent losses in the orders Phaffomycetales, Pichiales, and Dipodascales. In contrast to prior hypotheses based on smaller-scale datasets, comparison of evolutionary rates in protein-coding genes showed no bias towards elevated rates among aerobically fermenting (Crabtree/Warburg-positive) yeasts. Mitochondrial introns were widely distributed, but they were highly enriched in some groups. The majority of mitochondrial introns were poorly conserved within groups, but several were shared within groups, between groups, and even across taxonomic orders, which is consistent with horizontal gene transfer, likely involving homing endonucleases acting as selfish elements. As the number of available fungal nuclear genomes continues to expand, the methods described here to retrieve mitochondrial genome sequences from these datasets will prove invaluable to ensuring that studies of fungal mitochondrial genomes keep pace with their nuclear counterparts.\nPostzygotic barriers persist despite ongoing introgression in hybridizing Mimulus species\nAuthors: Mantel, S. J.; Sweigart, A. L.\nScore: 6.2, Published: 2023-08-06 DOI: 10.1101/2023.08.05.552095\nThe evolution of postzygotic isolation is thought to be a key step in maintaining species boundaries upon secondary contact. Yet the dynamics and persistence of hybrid incompatibilities in naturally hybridizing species is not well understood. Here, we explore these issues with genetic mapping in three independent populations of recombinant inbred lines between naturally hybridizing monkeyflowers Mimulus guttatus and M. nasutus from the sympatric Catherine Creek population. We discovered that the three M. guttatus founders differ dramatically in admixture history, with nearly a quarter of one founders genome introgressed from M. nasutus. Comparative genetic mapping in the three RIL populations revealed three new putative inversions, each one segregating among the M. guttatus founders, two due to admixture. We also found strong, genome-wide transmission ratio distortion in all RILs, but patterns were highly variable among the three populations. At least some of this distortion appears to be explained by epistatic selection favoring parental genotypes, but tests of inter-chromosomal linkage disequilibrium also revealed multiple candidate Dobzhansky-Muller incompatibilities. We also mapped several genetic loci for hybrid pollen viability, including two interacting pairs that coincide with peaks of distortion. Remarkably, even with this limited sample of three M. guttatus lines, we discovered abundant segregating variation for hybrid incompatibilities with M. nasutus, suggesting this population harbors diverse contributors to postzygotic isolation. Moreover, even with substantial admixture, hybrid incompatibilities between Mimulus species persist, suggesting postzygotic isolation might be a potent force in maintaining species barriers in this system.\n",
  "wordCount" : "2683",
  "inLanguage": "en",
  "datePublished": "2023-08-20T10:35:54Z",
  "dateModified": "2023-08-20T10:35:54Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on August 20, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.14.553118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.14.553118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.14.553118">
        <p class="paperTitle">Deletion of Xist upstream sequences alters TAD interactions and leads to defects in Xist coating and expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.14.553118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.14.553118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Majumdar, S.; Bammidi, L. S.; Naik, H.; M, A.; Baro, R.; Kalita, A.; Sundarraj, N.; Bariha, G.; Notani, D.; GAYEN, S.</p>
        <p class="info">Score: 17.7, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.14.553118' target='https://doi.org/10.1101/2023.08.14.553118'> 10.1101/2023.08.14.553118</a></p>
        <p class="abstract">The topological organization of the genome plays an important role in regulating gene expression. However, the connection between the two remains poorly understood. X-chromosome inactivation is a unique model system to explore the interlink between topologically associated domains (TADs) and gene expression. TADs are largely lost upon X-inactivation, and the inactive-X gets bipartitely reorganized into two large mega domains. However, the X-inactivation center (XIC) harbors two TADs - at the locus of long non-coding RNA Xist (Xist-TAD) and Tsix (Tsix-TAD). Xist is the master regulator of X-inactivation, which coat the inactive-X and facilitates heterochromatinization. Here, we deleted Xist upstream sequences ([~]6 kb) near the Xist TADs boundary in extraembryonic endoderm stem cells (XEN), which undergo imprinted X-inactivation. This deletion led to the major rearrangement of TADs and affected the expression of genes located within Xist and Tsix TAD, specially the expression of Xist was upregulated, suggesting TADs are essential for proper transcriptional regulation. On the other hand, Xist-upstream deletion on the inactive-X resulted in dispersal of Xist coating and loss of enrichment of repressive chromatin marks on the inactive-X but no effect on X-linked gene silencing. However, we found that autosomal genes were dysregulated in Xist-upstream deleted cells, probably because of misregulation of genes located in Xist and Tsix-TAD, specially Xist. We conclude that Xist upstream sequences are necessary for proper organization of the TADs at the XIC, maintenance of Xist coating/expression and autosomal gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553399">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553399" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553399">
        <p class="paperTitle">Expanding the Drosophila toolkit for dual control of gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553399" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553399" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zirin, J.; Jusiak, B.; Lopes, R.; Ewen-Campen, B.; Bosch, J.; Villalta, C.; Forman, C.; Hu, Y.; Mohr, S.; Perrimon, N.</p>
        <p class="info">Score: 15.5, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553399' target='https://doi.org/10.1101/2023.08.15.553399'> 10.1101/2023.08.15.553399</a></p>
        <p class="abstract">SummaryHaving tools and resources for independent control of gene expression in two different tissues in the same animal is emerging as a major need, especially in the context of inter-organ communication studies. This type of study is made possible by technologies combining the GAL4/UAS and a second binary expression system such as LexA/LexAop or QF/QUAS. Here, we describe a resource of reagents that facilitate combined use of the GAL4/UAS and a second binary systems in various tissues. Focusing on genes with well-characterized GAL4 expression patterns, we generated a set of more than 40 LexA-GAD and QF2 insertions by CRISPR knock-in and verified their tissue-specificity in larvae. We also built constructs that encode QF2 and LexA-GAD transcription factors in a single vector, with each coding sequence flanked by FRT sites. Following successful integration into the fly genome, the vector generates a single polypeptide, which is separated into the individual drivers by the inclusion of T2A. If desired, one of the two coding regions can then be excised with Flp, resulting in flies that express only QF2 or LexA-GAD. We evaluated both QF2 and LexA-GAD systems for in vivo gene knockdown and are generating a compatible library of transgenic shRNA lines in our custom QUAS and LexAop vectors as a community resource. Together, these QF2/LexA-GAD and QUAS/LexAop vectors and fly lines will provide a new set of tools for researchers who need to activate or repress two different genes in an orthogonal manner in the same animal.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.551251">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.551251" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.551251">
        <p class="paperTitle">Shared and distinct molecular effects of regulatory genomic variants provide insight into mechanisms of distal enhancer-promoter communication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.551251" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.551251" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ray-Jones, H.; Song, Z.; Haglund, A.; Artemov, P.; Della Rosa, M.; Burden, F.; Kreuzhuber, R.; Litovskikh, A.; Tan, V. X. H.; Chan, L. T.; Frontini, M.; Wallace, C.; Malysheva, V.; Bottolo, L.; Vigorito, E.; Spivakov, M.</p>
        <p class="info">Score: 15.1, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.551251' target='https://doi.org/10.1101/2023.08.04.551251'> 10.1101/2023.08.04.551251</a></p>
        <p class="abstract">Transcriptional enhancers control spatiotemporal gene expression and commonly engage in long-range chromosomal contacts with gene promoters. However, the relationship between enhancer activity, enhancer-promoter contact and gene expression is not completely understood. Here, we leverage human genetic variation as a &#34;natural perturbation&#34; to dissect this relationship. Focusing on enhancers that harbour genetic variants showing allelic associations with the expression levels of distal genes (expression quantitative trait loci, eQTLs), we profiled eQTL-promoter contacts using high-resolution Capture Hi-C, as well as chromatin activity by ATAC-sequencing, in primary monocytes isolated from 34 healthy donors. To detect the associations of genetic variants with these molecular phenotypes, we extended a recently proposed Bayesian method that leverages both intra- and interindividual variation. We identified 19 eQTLs associated with differential promoter contacts, most of which also showed an association with chromatin activity. Capitalising on this overlap, we next adapted a Bayesian mapping strategy for detecting variants that exert genetic control across multiple phenotypes, revealing hundreds of putative &#34;trimodal QTL&#34; variants jointly associated with enhancer activity, connectivity, and gene expression. A subset of these variants was predicted to affect the binding of a diverse range of transcription factors, in part through disruption of the binding motifs for either these factors or their binding partners. In addition, we identified a variant with opposite effects on promoter contact and expression levels of its target gene, PCK2. We show that this variant disrupts the canonical binding motif of the architectural protein CTCF and likely perturbs the chromatin insulation of the PCK2 promoter from more distal enhancers. Finally, multiple identified QTLs overlapped known disease susceptibility loci, pointing to the potential role of enhancer-promoter communication in mediating the pathological effects of non-coding genetic variation. Jointly, our findings suggest an inherent functional link between the activity and connectivity of enhancers with relevance for human disease and highlight the role of genetically-determined chromatin boundaries in gene expression control.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.552711">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.552711" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.552711">
        <p class="paperTitle">GetPrimers: a generalized PCR-based genetic targeting primer designer enabling easy and standardized targeted gene modification across multiple systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.552711" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.552711" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miao, Z.; Wang, H.; Tu, X.; Huang, Z.; Huang, S.; Zhang, X.; Wang, F.; Huang, Z.; Li, H.; Jiao, Y.; Gao, S.; Zhou, Z.; Shan, C.-M.; Li, J.; Yue, J.-X.</p>
        <p class="info">Score: 8.3, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.552711' target='https://doi.org/10.1101/2023.08.12.552711'> 10.1101/2023.08.12.552711</a></p>
        <p class="abstract">Genetic targeting (e.g., gene knockout and tagging) based on polymerase chain reaction (PCR) is a simple yet powerful approach for studying gene functions. Although originally developed in classic budding and fission yeast models, the same principle applies to other eukaryotic systems with efficient homologous recombination. One-step-PCR-based genetic targeting is conventionally used but the sizes of the homologous arms that it generates for recombination-mediated genetic targeting are usually limited. Alternatively, gene targeting can also be performed via fusion PCR, which can create homologous arms that are orders of magnitude larger, therefore substantially increases the efficiency of recombination-mediated genetic targeting. Here we present GetPrimers (https://www.evomicslab.org/app/getprimers/), a generalized computational framework and web tool to assist automatic targeting and verification primer design for both one-step-PCR-based and fusion-PCR-based genetic targeting experiments. Moreover, GetPrimers by design runs for any given genetic background of any species with full genome scalability. Therefore, GetPrimers is capable of empowering high-throughput functional genomics assays at multi-population and multi-species levels. Comprehensive experimental validations have been performed for targeting and verification primers designed by GetPrimers across multiple organism systems and experimental setups. We anticipate GetPrimers to become a highly useful and popular tool to facilitate easy and standardized gene modification across multiple systems.

KeypointsDeveloped the web tool GetPrimers to assist automatic primer design for PCR-based genetic targeting.

It runs for any given locus on any genetic background of any given species with full genome scalability.

Experimental validations have been performed across multiple organism systems and experimental setups.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.551844">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.551844" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.551844">
        <p class="paperTitle">Methods and techniques enabling multi-kilobase long-range genomic rewrite/replace editing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.551844" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.551844" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Piggott, C. A.; Tjang, L. V.; Chen, S.; Tatu, U.; Hirou, H.; Jaroszewska, A.; McIntyre, E.; Chen, J.; Faybishenko, J. V.; Gavan, R.; Hackley, C.</p>
        <p class="info">Score: 11.8, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.551844' target='https://doi.org/10.1101/2023.08.05.551844'> 10.1101/2023.08.05.551844</a></p>
        <p class="abstract">CRISPR enabled cell and gene therapies have the potential to revolutionize the field of genetic medicine. However, the vast majority of rare diseases remain untreatable due to the limitations of current tools and techniques. To date, most corrective therapeutic approaches have been restricted to mutation-by-mutation approaches, where either HDR, or newer techniques such as base or prime editing, rewrite small regions of DNA at a time ([~]1-100 bp). While these approaches are powerful, short editing windows (relative to the size of human genes) are financially and/or technically incompatible with most rare-disease mutation profiles. Here, we demonstrate for the first time that CRISPR/Cas9 can be used to &#34;rewrite&#34; 7kb&#43; sections of the human genome simultaneously via a selection-free process we have named &#34;long-range rewriting&#34;. Long-range rewriting approaches are compatible with multiple nucleases, cell types and genomic loci, and can be used with both double-strand break (DSB) and non-DSB based approaches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553567">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553567" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553567">
        <p class="paperTitle">CryptoCEN: A Co-Expression Network for Cryptococcus neoformans reveals novel proteins involved in DNA damage repair</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553567" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553567" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Meara, M. J.; Rapala, J. R.; Nichols, C. B.; Alexandre, C.; Billmyre, R. B.; Steenwyk, J. L.; Alspaugh, J. A.; O&#39;Meara, T. R.</p>
        <p class="info">Score: 5.3, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553567' target='https://doi.org/10.1101/2023.08.17.553567'> 10.1101/2023.08.17.553567</a></p>
        <p class="abstract">Elucidating gene function is a major goal in biology, especially among non-model organisms. However, doing so is complicated by the fact that molecular conservation does not always mirror functional conservation, and that complex relationships among genes are responsible for encoding pathways and higher-order biological processes. Co-expression, a promising approach for predicting gene function, relies on the general principal that genes with similar expression patterns across multiple conditions will likely be involved in the same biological process. For Cryptococcus neoformans, a prevalent human fungal pathogen greatly diverged from model yeasts, approximately 60% of the predicted genes in the genome lack functional annotations. Here, we leveraged a large amount of publicly available transcriptomic data to generate a C. neoformans Co-Expression Network (CryptoCEN), successfully recapitulating known protein networks, predicting gene function, and enabling insights into the principles influencing co-expression. With 100% predictive accuracy, we used CryptoCEN to identify 13 new DNA damage response genes, underscoring the utility of guilt-by-association for determining gene function. Overall, co-expression is a powerful tool for uncovering gene function, and decreases the experimental tests needed to identify functions for currently under-annotated genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.553026">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.553026" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.553026">
        <p class="paperTitle">A fluorescent sex-sorting technique for insects with the demonstration in Drosophila melanogaster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.553026" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.553026" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, J.; Rayes, D.; Akbari, O.</p>
        <p class="info">Score: 4.0, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.553026' target='https://doi.org/10.1101/2023.08.11.553026'> 10.1101/2023.08.11.553026</a></p>
        <p class="abstract">Recent advances in insect genetic engineering offer alternative genetic biocontrol solutions to control populations of pests and disease vectors. While success has been achieved, sex-sorting remains problematic for scaling many genetic biocontrol interventions. Here we describe the development of a sex-sorting technique for female and male selection with a proof-of-concept in D. melanogaster termed SEPARATOR (Sexing Element Produced by Alternative RNA-splicing of A Transgenic Observable Reporter). This approach utilizes dominant fluorescent proteins and differentially spliced introns to ensure sex-specific expression. The system has the potential for adaptability to various insect species and application for high-throughput insect sex-sorting.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552281">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552281" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552281">
        <p class="paperTitle">Leveraging fine-scale population structure reveals conservation in genetic effect sizes between human populations across a range of human phenotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552281" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552281" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, S.; Ferreira, L. A. F.; Shi, S.; Hellenthal, G.; Marchini, J.; Lawson, D. J.; Myers, S. R.</p>
        <p class="info">Score: 28.4, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552281' target='https://doi.org/10.1101/2023.08.08.552281'> 10.1101/2023.08.08.552281</a></p>
        <p class="abstract">An understanding of genetic differences between populations is essential for avoiding confounding in genome-wide association studies (GWAS) and understanding the evolution of human traits. Polygenic risk scores constructed in one group perform poorly in highly genetically-differentiated populations, for reasons which remain controversial. We developed a statistical ancestry inference pipeline able to decompose ancestry both within and between countries, and applied it to the UK Biobank data. This identifies fine-scale patterns of genetic relatedness not captured by standard and widely used principal components (PCs), and allows fine-scale population stratification correction that removes both false positive and false negative associations for traits with geographic correlations. We also develop and apply ANCHOR, an approach leveraging segments of distinct ancestries within individuals to estimate similarity in underlying causal effect sizes between groups, using an existing PGS. Applying ANCHOR to &gt;8000 people of mixed African and European ancestry, we demonstrate that estimated causal effect sizes are highly similar across these ancestries for 26 of 29 quantitative molecular and non-molecular phenotypes (mean correlation 0.98 &#43;/-0.08), providing evidence that gene-environment and gene-gene interactions do not play major roles in the poor prediction of European-ancestry PRS scores in African populations for these traits, contradicting previous findings. Instead our results provide optimism that shared causal mutations operate similarly in different groups, focussing the challenge of improving GWAS &#34;portability&#34; between groups on joint fine-mapping.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.551029">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.551029" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.551029">
        <p class="paperTitle">Mitochondrial Genome Diversity across the Subphylum Saccharomycotina</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.551029" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.551029" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wolters, J. F.; LaBella, A.; Opulente, D.; Rokas, A.; Hittinger, C. T.</p>
        <p class="info">Score: 16.4, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.551029' target='https://doi.org/10.1101/2023.07.28.551029'> 10.1101/2023.07.28.551029</a></p>
        <p class="abstract">Eukaryotic life depends on the functional elements encoded by both the nuclear genome and organellar genomes, such as those contained within the mitochondria. The content, size, and structure of the mitochondrial genome varies across organisms with potentially large implications for phenotypic variance and resulting evolutionary trajectories. Among yeasts in the subphylum Saccharomycotina, extensive differences have been observed in various species relative to the model yeast Saccharomyces cerevisiae, but mitochondrial genome sampling across many groups has been scarce, even as hundreds of nuclear genomes have become available. By extracting mitochondrial assemblies from existing short-read genome sequence datasets, we have greatly expanded both the number of available genomes and the coverage across sparsely sampled clades. Comparison of 353 yeast mitochondrial genomes revealed that, while size and GC content were fairly consistent across species, those in the genera Metschnikowia and Saccharomyces trended larger, while several species in the order Saccharomycetales, which includes S. cerevisiae, exhibited lower GC content. Extreme examples for both size and GC content were scattered throughout the subphylum. All mitochondrial genomes shared a core set of protein-coding genes for Complexes III, IV, and V, but they varied in the presence or absence of mitochondrially-encoded canonical Complex I genes. We traced the loss of Complex I genes to a major event in the ancestor of the orders Saccharomycetales and Saccharomycodales, but we also observed several independent losses in the orders Phaffomycetales, Pichiales, and Dipodascales. In contrast to prior hypotheses based on smaller-scale datasets, comparison of evolutionary rates in protein-coding genes showed no bias towards elevated rates among aerobically fermenting (Crabtree/Warburg-positive) yeasts. Mitochondrial introns were widely distributed, but they were highly enriched in some groups. The majority of mitochondrial introns were poorly conserved within groups, but several were shared within groups, between groups, and even across taxonomic orders, which is consistent with horizontal gene transfer, likely involving homing endonucleases acting as selfish elements. As the number of available fungal nuclear genomes continues to expand, the methods described here to retrieve mitochondrial genome sequences from these datasets will prove invaluable to ensuring that studies of fungal mitochondrial genomes keep pace with their nuclear counterparts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552095">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552095" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552095">
        <p class="paperTitle">Postzygotic barriers persist despite ongoing introgression in hybridizing Mimulus species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552095" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552095" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mantel, S. J.; Sweigart, A. L.</p>
        <p class="info">Score: 6.2, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552095' target='https://doi.org/10.1101/2023.08.05.552095'> 10.1101/2023.08.05.552095</a></p>
        <p class="abstract">The evolution of postzygotic isolation is thought to be a key step in maintaining species boundaries upon secondary contact. Yet the dynamics and persistence of hybrid incompatibilities in naturally hybridizing species is not well understood. Here, we explore these issues with genetic mapping in three independent populations of recombinant inbred lines between naturally hybridizing monkeyflowers Mimulus guttatus and M. nasutus from the sympatric Catherine Creek population. We discovered that the three M. guttatus founders differ dramatically in admixture history, with nearly a quarter of one founders genome introgressed from M. nasutus. Comparative genetic mapping in the three RIL populations revealed three new putative inversions, each one segregating among the M. guttatus founders, two due to admixture. We also found strong, genome-wide transmission ratio distortion in all RILs, but patterns were highly variable among the three populations. At least some of this distortion appears to be explained by epistatic selection favoring parental genotypes, but tests of inter-chromosomal linkage disequilibrium also revealed multiple candidate Dobzhansky-Muller incompatibilities. We also mapped several genetic loci for hybrid pollen viability, including two interacting pairs that coincide with peaks of distortion. Remarkably, even with this limited sample of three M. guttatus lines, we discovered abundant segregating variation for hybrid incompatibilities with M. nasutus, suggesting this population harbors diverse contributors to postzygotic isolation. Moreover, even with substantial admixture, hybrid incompatibilities between Mimulus species persist, suggesting postzygotic isolation might be a potent force in maintaining species barriers in this system.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
