<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley
Authors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.
Score: 29.1, Published: 2023-09-22 DOI: 10.1101/2023.09.22.557807
Direct observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley
Authors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.
Score: 29.1, Published: 2023-09-22 DOI: 10.1101/2023.09.22.557807
Direct observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-01T10:37:47+00:00" />
<meta property="article:modified_time" content="2023-10-01T10:37:47+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley
Authors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.
Score: 29.1, Published: 2023-09-22 DOI: 10.1101/2023.09.22.557807
Direct observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley\nAuthors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.\nScore: 29.1, Published: 2023-09-22 DOI: 10.1101/2023.09.22.557807\nDirect observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII).",
  "keywords": [
    
  ],
  "articleBody": " Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley\nAuthors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.\nScore: 29.1, Published: 2023-09-22 DOI: 10.1101/2023.09.22.557807\nDirect observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII). CCII was founded in 1929 with tens of thousands of unique genotypes and has been adapted to local conditions in Davis, CA, USA for 58 generations. We find that natural selection has massively reduced genetic diversity leading to a single clonal lineage constituting most of the population by generation F50. Selection favored alleles originating from similar climates to that of Davis, and targeted genes regulating reproductive development, including some of the most well-characterized barley diversification loci, Vrs1, HvCEN, and Ppd-H1. We chronicle the dynamic evolution of reproductive timing in the population and uncover how parallel molecular pathways are targeted by stabilizing selection to optimize this trait. Our findings point to selection as the predominant force shaping genomic variation in one of the worlds oldest ongoing biological experiments. One-Sentence SummaryWholesale genetic restructuring of an experimental population is a consequence of rapid environmental adaptation.\nSynchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition\nAuthors: Vollger, M. R.; Korlach, J.; Eldred, K. C.; Swanson, E.; Underwood, J. G.; Munson, K. M.; Cheng, Y.-H. H.; Ranchalis, J.; Mao, Y.; Blue, E. E.; Schwarze, U.; Saunders, C. T.; Wenger, A. M.; Allworth, A.; Chanprasert, S.; Duerden, B. L.; Glass, I.; Horike-Pyne, M.; Kim, M.; Leppig, K. A.; McLaughlin, I. J.; Ogawa, J.; Rosenthal, E. A.; Sheppeard, S.; Sherman, S. M.; Strohbehn, S.; Yuen, A. L.; Reh, T. A.; Byers, P. H.; Bamshad, M. J.; Hisama, F. M.; Jarvik, G. P.; Sancak, Y.; Dipple, K. M.; Stergachis, A. B.\nScore: 27.7, Published: 2023-09-27 DOI: 10.1101/2023.09.26.559521\nResolving the molecular basis of a Mendelian condition (MC) remains challenging owing to the diverse mechanisms by which genetic variants cause disease. To address this, we developed a synchronized long-read genome, methylome, epigenome, and transcriptome sequencing approach, which enables accurate single-nucleotide, insertion-deletion, and structural variant calling and diploid de novo genome assembly, and permits the simultaneous elucidation of haplotype-resolved CpG methylation, chromatin accessibility, and full-length transcript information in a single long-read sequencing run. Application of this approach to an Undiagnosed Diseases Network (UDN) participant with a chromosome X;13 balanced translocation of uncertain significance revealed that this translocation disrupted the functioning of four separate genes (NBEA, PDK3, MAB21L1, and RB1) previously associated with single-gene MCs. Notably, the function of each gene was disrupted via a distinct mechanism that required integration of the four omes to resolve. These included nonsense-mediated decay, fusion transcript formation, enhancer adoption, transcriptional readthrough silencing, and inappropriate X chromosome inactivation of autosomal genes. Overall, this highlights the utility of synchronized long-read multi-omic profiling for mechanistically resolving complex phenotypes.\nCyclin N-terminal domain-containing 1 (CNTD1) is critical for crossover designation during meiotic prophase I and for maintenance of ovarian reserve in mammalian oocytes\nAuthors: Wood, A.; Ahmed, R. M.; Bradley, R. A.; Wolff, I. D.; Cohen, P. E.\nScore: 17.6, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559118\nIn meiotic prophase I, hundreds of double-strand breaks (DSB) are induced to initiate homologous recombination, resulting in the formation of non-crossovers (NCO) and crossovers (CO). Of the 250-300 DSBs that form in mice, only [~]10% are resolved as COs, with all remaining DSB events resolving as NCO. Thus, both the frequency and distribution of COs must be tightly regulated to ensure their appropriate localization across the genome. Errors in crossover designation and maturation can result in nondisjunction, leading to aneuploidy, germ cell death, birth defects, or infertility. That these errors are more evident in female meiosis compared to males suggests that the events of meiotic prophase I are sexually dimorphic with respect to DSB processing, resolution, and/or surveillance. Therefore, it is essential to investigate how COs are designated and regulated during prophase I in the female germline. Previous studies in our lab showed that Cyclin N-Terminal Domain Containing 1 (CNTD1) protein plays a critical role in CO designation during male meiosis, co-localizing with other essential CO proteins such as MutL{gamma} (MLH1/MLH3) and crossover site-associated cyclin-dependent kinase-2 (CDK2) during pachynema. In the current study, we investigated whether analogous roles for CNTD1 existed in female meiosis. Similar to Cntd1-/- males, Cntd1-/- females are sterile, with no gross morphological ovarian defects in adults albeit with a depletion in ovarian follicle numbers. During prophase I, early DSB induction and repair occur normally, but oocytes from Cntd1-/- females fail to accumulate MutL{gamma} in pachynema. Phenotypically, meiotic prophase I disruption in Cntd1-/- females is similar to that seen in Cntd1-/- males. Importantly, whereas the Caernohabditis elegans ortholog of CNTD1, COSA-1, appears to interact with CDK-2 during pachynema in oocytes, CNTD1 in mouse oocytes lacks the CDK-interacting motif, suggesting that its function in oocytes does not involve a cyclin-like induction of CDK2 phosphorylation activity. In contradiction to the timing of prophase I disruption in Cntd1-/- females at pachynema, we observe a dramatic depletion of primordial follicles, beginning shortly after birth and increasing dramatically through to adult ages. Those primordial follicles that persist, however, are able to develop through dictyate arrest into primary and larger follicular structures in pre-pubertal females, and only by adulthood were significant declines observed in all follicles. The early loss of oocytes in Cntd1-/- females shortly after birth, considerably earlier than that observed for mice lacking MLH1 or MLH3, suggests a novel early role for CNTD1 in CO designation and/or in ensuring progression of oocytes into dictyate arrest at late pachynema.\nEvolutionary adaptation of the chromodomain of the HP1-protein Rhino allows the integration of chromatin and DNA sequence signals\nAuthors: Baumgartner, L.; Ipsaro, J. J.; Hohmann, U.; Handler, D.; Schleiffer, A.; Duchek, P.; Brennecke, J.\nScore: 14.3, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560096\nMembers of the diverse heterochromatin protein 1 (HP1) family of proteins play crucial roles in heterochromatin formation and maintenance. Despite the similar affinities of their chromodomains for di- and tri-methylated histone H3 lysine 9 (H3K9me2/3), different HP1 proteins exhibit distinct chromatin binding patterns, presumably due to their interactions with various specificity factors. Here, we elucidate the molecular basis of the protein-protein interaction between the HP1 protein Rhino, a critical factor of the Drosophila piRNA pathway, and Kipferl, a DNA sequence-specific C2H2 zinc finger protein and Rhino guidance factor. Through phylogenetic analyses, structure prediction, and in vivo genetics, we identify a single amino acid change within Rhino's chromodomain, G31D, that does not affect H3K9me2/3 binding but abolishes the specific interaction between Rhino and Kipferl. Flies carrying the rhinoG31D mutation phenocopy kipferl mutant flies, with Rhino redistributing from piRNA clusters to satellite repeats, causing pronounced changes in the ovarian piRNA profile of rhinoG31D flies. Thus, Rhino's chromodomain serves as a dual-specificity module, facilitating interactions with both a histone mark and a DNA-binding protein.\nEnsembled best subset selection using summary statistics for polygenic risk prediction\nAuthors: Chen, T.; Zhang, H.; Mazumder, R.; Lin, X.\nScore: 8.0, Published: 2023-09-27 DOI: 10.1101/2023.09.25.559307\nPolygenic risk scores (PRS) enhance population risk stratification and advance personalized medicine, yet existing methods face a tradeoff between predictive power and computational efficiency. We introduce ALL-Sum, an easy-to-implement PRS method that combines an efficient summary statistic-based L0L2 penalized regression algorithm with an ensembling step that aggregates estimates from different tuning parameters for improved prediction performance. In extensive large-scale simulations across a wide range of polygenicity and genome-wide association studies (GWAS) sample sizes, ALL-Sum consistently outperforms popular alternative methods in terms of prediction accuracy, runtime, and memory usage. We analyze 27 published GWAS summary statistics for 11 complex traits from 9 reputable data sources, including the Global Lipids Genetics Consortium, Breast Cancer Association Consortium, and FinnGen, evaluated using individual-level UKBB data. ALL-Sum achieves the highest accuracy for most traits, particularly for GWAS with large sample sizes. We provide ALL-Sum as a user-friendly command-line software with pre-computed reference data for streamlined user-end analysis.\nTranscriptome-based host epistasis and pathogen co-expression in barley-powdery mildew interactions\nAuthors: Velasquez-Zapata, V.; Smith, S. D.; Surana, P.; Chapman, A. V. E.; Wise, R. P.\nScore: 6.7, Published: 2023-09-23 DOI: 10.1101/2023.09.18.558274\nMildew locus a (Mla) from the Triticeae grain crop barley (Hordeum vulgare L.) encodes a multi-allelic series of nucleotide-binding leucine-rich repeat (NLR) immune receptors. These variable NLRs recognize complementary secreted effectors from the powdery mildew fungus, Blumeria hordei (Bh), to block disease progression. We used a dynamic time-course transcriptome of barley infected with Bh to infer gene effects and epistatic relationships governed by the Mla6 NLR, two other host loci critical to the interaction, Blufensin1 (Bln1) and Required for Mla6 resistance3 (Rar3), and genes that interact with them. Bln1 is an R-gene independent regulator of immunity and the resistant bln1 mutant exhibits enhanced basal defense to compatible Bh. Rar3 is required for MLA6-mediated generation of H2O2 and the hypersensitive reaction; the rar3 mutant contains an in-frame Lys-Leu deletion in the SGT1-specific domain that compromises immunity by a subset of Mla alleles. Interactions of Mla6 and Bln1 resulted in symmetric, suppression and masked epistasis on the Bh-induced barley transcriptome. Likewise, dominant or equal effects were caused by Mla6 and Sgt1. Of a total of 468 barley NLRs, 366 were expressed in our dataset and 115 of those were grouped under different gene effect models, which localized to several chromosome hotspots. The corresponding Bh infection transcriptome was classified into nine co-expressed modules, linking differential expression with pathogen development. Expression of most of 517 Bh effectors exhibited dependence on disease phenotype and was associated with appressorial or haustorial structures, suggesting that disease is regulated by a host-pathogen intercommunication network that diversifies the response.\nAutosomal suppression of meiotic drive can prevent sex chromosome cycling\nAuthors: Gupta, A.; Unckless, R.\nScore: 4.9, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559847\nSex-ratio meiotic drivers are selfish genes or gene complexes that bias the transmission of chromosomes resulting in skewed sex ratios. While existing theoretical models have suggested the maintenance of a four-chromosome equilibrium (X, XSR, Y, YSUP; SR: sex-ratio driver, SUP: suppressor) in a cyclic dynamic, studies of natural populations have failed to capture this pattern. Although there are several plausible explanations for this lack of cycling, interference from autosomal suppressors has not been studied using a population genetic framework even though autosomal suppressors and Y-linked suppressors both exist in populations of some species. In this study, we use a simulations-based approach to investigate the influence of autosomal suppressors on the cycling of sex chromosomes. Our findings demonstrate that the presence of an autosomal suppressor can hinder the invasion of a Y-linked suppressor under some parameter space, thereby impeding the cyclic dynamics. Even when a Y-linked suppressor invades, the presence of an autosomal suppressor can prevent cycling. Our study demonstrates the potential role of autosomal suppressors in preventing sex chromosome cycling and provides insights into the conditions and consequences of the maintenance of both Y-linked and autosomal suppressors.\nIdentification of Genetic Suppressors for a Berardinelli-Seip Congenital Generalized Lipodystrophy Type 2 (BSCL2) Pathogenic Variant in C. elegans\nAuthors: Bai, X.; Smith, H. E.; Golden, A.\nScore: 3.6, Published: 2023-09-24 DOI: 10.1101/2023.09.22.559059\nMaintaining the metabolic homeostasis of fatty acids is crucial for human health. Excess fatty acids are stored in lipid droplets (LDs), the primary energy reservoir that helps regulate fat and lipid homeostasis in nearly all cell types. Seipin (BSCL2), a conserved endoplasmic reticulum protein, plays a critical role in LD biogenesis and regulating LD morphology. Pathogenic variants of seipin are associated with multiple human genetic diseases, including Berardinelli-Seip Congenital Generalized Lipodystrophy Type 2 (BSCL2). However, the cellular and molecular mechanisms by which dysfunctional seipin leads to these diseases remain unclear. To model BSCL2 disease, we generated an orthologous BSCL2 pathogenic variant seip-1(A185P) using CRISPR/Cas9 genome editing in Caenorhabditis elegans. This variant led to severe developmental and cellular defects, including embryonic lethality, impaired eggshell formation, and abnormally enlarged LDs. We set out to identify genetic determinants that could suppress these defective phenotypes in the seip-1(A185P) mutant background. To this end, we conducted an unbiased chemical mutagenesis screen to identify genetic suppressors that restore embryonic viability in the seip-1(A185P) mutant background. A total of five suppressor lines were isolated and recovered from the screen. The defective phenotypes of seip-1(A185P), including embryonic lethality and impaired eggshell formation, were significantly suppressed in each suppressor line. Two of the five suppressor lines also alleviated the enlarged LDs in the oocytes. We then mapped a suppressor candidate gene, R05D3.2 (renamed as lmbr-1), which is an ortholog of human LMBR1 (limb development membrane protein 1). The CRISPR/Cas9 edited lmbr-1 suppressor alleles, lmbr-1(Ser647Phe) and lmbr-1(Pro314Leu), both significantly suppressed embryonic lethality and defective eggshell formation in the seip-1(A185P) background. The newly identified suppressor lines offer valuable insights into potential genetic interactors and pathways that may regulate seipin in the lipodystrophy model.\nStable population structure in Europe since the Iron Age, despite high mobility\nAuthors: Antonio, M. L.; Weiss, C. L.; Gao, Z.; Sawyer, S.; Oberreiter, V.; Moots, H. M.; Spence, J. P.; Cheronet, O.; Zagorc, B.; Praxmarer, E.; Ozdogan, K. T.; Demetz, L.; Gelabert, P.; Fernandes, D.; Lucci, M.; Alihodzic, T.; Amrani, S.; Avetisyan, P.; Baillif-Ducros, C.; Bedic, Z.; Bertrand, A.; Bilic, M.; Bondioli, L.; Borowka, P.; Botte, E.; Burmaz, J.; Buzanic, D.; Candilio, F.; Cvetko, M.; De Angelis, D.; Drnic, I.; Elschek, K.; Fantar, M.; Gaspari, A.; Gasperetti, G.; Genchi, F.; Golubovic, S.; Hukelova, Z.; Jankauskas, R.; Vuckovic, K. J.; Jeremic, G.; Kaic, I.; Kazek, K.; Khachatryan, H.; Kh\nScore: 171.0, Published: 2023-09-21 DOI: 10.1101/2022.05.15.491973\nAncient DNA research in the past decade has revealed that European population structure changed dramatically in the prehistoric period (14,000-3,000 years before present, YBP), reflecting the widespread introduction of Neolithic farmer and Bronze Age Steppe ancestries. However, little is known about how population structure changed from the historical period onward (3,000 YBP - present). To address this, we collected whole genomes from 204 individuals from Europe and the Mediterranean, many of which are the first historical period genomes from their region (e.g. Armenia and France). We found that most regions show remarkable inter-individual heterogeneity. At least 7% of historical individuals carry ancestry uncommon in the region where they were sampled, some indicating cross-Mediterranean contacts. Despite this high level of mobility, overall population structure across western Eurasia is relatively stable through the historical period up to the present, mirroring geography. We show that, under standard population genetics models with local panmixia, the observed level of dispersal would lead to a collapse of population structure. Persistent population structure thus suggests a lower effective migration rate than indicated by the observed dispersal. We hypothesize that this phenomenon can be explained by extensive transient dispersal arising from drastically improved transportation networks and the Roman Empires mobilization of people for trade, labor, and military. This work highlights the utility of ancient DNA in elucidating finer scale human population dynamics in recent history.\nDetection of short identity by descent segments using low-frequency variants\nAuthors: Wang, C.; Veldsman, W. P.; Zhang, L.\nScore: 2.8, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559464\nIdentity by descent (IBD) segments are inherited genomic sequences from the same common ancestors that serve as biomarkers during lineage analysis, disease mutation mapping, and within broader fields of population genetics. Current computational tools used for the detection of IBD segments focus on identifying long IBD segments, with short IBD segments (i.e. shorter than 2 cM) often going undetected. Here, we present SILO, a method with a remarkably improved ability to detect short IBD segments as compared to state-of-the-art IBD detection tools. SILO detects IBD segments by considering both common and low-frequency variants (LFVs), where the probability of the presence of a LFV minor allele is assumed to follow a Bernoulli distribution with a Beta prior. SILO was benchmarked against GERMLINE2, hap-IBD, HapFABIA, Parente, and TRUFFLE on simulated data and on a real pedigree from the 1,000 Genomes Project. The results show that SILO has an unrivaled ability to detect short IBD segments, while its ability to detect long IBD segments is similar to that of contemporary tools in the field.\n",
  "wordCount" : "2683",
  "inLanguage": "en",
  "datePublished": "2023-10-01T10:37:47Z",
  "dateModified": "2023-10-01T10:37:47Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on October 1, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.557807">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.557807" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.557807">
        <p class="paperTitle">Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.557807" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.557807" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.</p>
        <p class="info">Score: 29.1, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.557807' target='https://doi.org/10.1101/2023.09.22.557807'> 10.1101/2023.09.22.557807</a></p>
        <p class="abstract">Direct observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII). CCII was founded in 1929 with tens of thousands of unique genotypes and has been adapted to local conditions in Davis, CA, USA for 58 generations. We find that natural selection has massively reduced genetic diversity leading to a single clonal lineage constituting most of the population by generation F50. Selection favored alleles originating from similar climates to that of Davis, and targeted genes regulating reproductive development, including some of the most well-characterized barley diversification loci, Vrs1, HvCEN, and Ppd-H1. We chronicle the dynamic evolution of reproductive timing in the population and uncover how parallel molecular pathways are targeted by stabilizing selection to optimize this trait. Our findings point to selection as the predominant force shaping genomic variation in one of the worlds oldest ongoing biological experiments.

One-Sentence SummaryWholesale genetic restructuring of an experimental population is a consequence of rapid environmental adaptation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559521">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559521" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559521">
        <p class="paperTitle">Synchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559521" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559521" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vollger, M. R.; Korlach, J.; Eldred, K. C.; Swanson, E.; Underwood, J. G.; Munson, K. M.; Cheng, Y.-H. H.; Ranchalis, J.; Mao, Y.; Blue, E. E.; Schwarze, U.; Saunders, C. T.; Wenger, A. M.; Allworth, A.; Chanprasert, S.; Duerden, B. L.; Glass, I.; Horike-Pyne, M.; Kim, M.; Leppig, K. A.; McLaughlin, I. J.; Ogawa, J.; Rosenthal, E. A.; Sheppeard, S.; Sherman, S. M.; Strohbehn, S.; Yuen, A. L.; Reh, T. A.; Byers, P. H.; Bamshad, M. J.; Hisama, F. M.; Jarvik, G. P.; Sancak, Y.; Dipple, K. M.; Stergachis, A. B.</p>
        <p class="info">Score: 27.7, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559521' target='https://doi.org/10.1101/2023.09.26.559521'> 10.1101/2023.09.26.559521</a></p>
        <p class="abstract">Resolving the molecular basis of a Mendelian condition (MC) remains challenging owing to the diverse mechanisms by which genetic variants cause disease. To address this, we developed a synchronized long-read genome, methylome, epigenome, and transcriptome sequencing approach, which enables accurate single-nucleotide, insertion-deletion, and structural variant calling and diploid de novo genome assembly, and permits the simultaneous elucidation of haplotype-resolved CpG methylation, chromatin accessibility, and full-length transcript information in a single long-read sequencing run. Application of this approach to an Undiagnosed Diseases Network (UDN) participant with a chromosome X;13 balanced translocation of uncertain significance revealed that this translocation disrupted the functioning of four separate genes (NBEA, PDK3, MAB21L1, and RB1) previously associated with single-gene MCs. Notably, the function of each gene was disrupted via a distinct mechanism that required integration of the four  omes to resolve. These included nonsense-mediated decay, fusion transcript formation, enhancer adoption, transcriptional readthrough silencing, and inappropriate X chromosome inactivation of autosomal genes. Overall, this highlights the utility of synchronized long-read multi-omic profiling for mechanistically resolving complex phenotypes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559118">
        <p class="paperTitle">Cyclin N-terminal domain-containing 1 (CNTD1) is critical for crossover designation during meiotic prophase I and for maintenance of ovarian reserve in mammalian oocytes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wood, A.; Ahmed, R. M.; Bradley, R. A.; Wolff, I. D.; Cohen, P. E.</p>
        <p class="info">Score: 17.6, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559118' target='https://doi.org/10.1101/2023.09.25.559118'> 10.1101/2023.09.25.559118</a></p>
        <p class="abstract">In meiotic prophase I, hundreds of double-strand breaks (DSB) are induced to initiate homologous recombination, resulting in the formation of non-crossovers (NCO) and crossovers (CO). Of the 250-300 DSBs that form in mice, only [~]10% are resolved as COs, with all remaining DSB events resolving as NCO. Thus, both the frequency and distribution of COs must be tightly regulated to ensure their appropriate localization across the genome. Errors in crossover designation and maturation can result in nondisjunction, leading to aneuploidy, germ cell death, birth defects, or infertility. That these errors are more evident in female meiosis compared to males suggests that the events of meiotic prophase I are sexually dimorphic with respect to DSB processing, resolution, and/or surveillance. Therefore, it is essential to investigate how COs are designated and regulated during prophase I in the female germline. Previous studies in our lab showed that Cyclin N-Terminal Domain Containing 1 (CNTD1) protein plays a critical role in CO designation during male meiosis, co-localizing with other essential CO proteins such as MutL{gamma} (MLH1/MLH3) and crossover site-associated cyclin-dependent kinase-2 (CDK2) during pachynema. In the current study, we investigated whether analogous roles for CNTD1 existed in female meiosis. Similar to Cntd1-/- males, Cntd1-/- females are sterile, with no gross morphological ovarian defects in adults albeit with a depletion in ovarian follicle numbers. During prophase I, early DSB induction and repair occur normally, but oocytes from Cntd1-/- females fail to accumulate MutL{gamma} in pachynema. Phenotypically, meiotic prophase I disruption in Cntd1-/- females is similar to that seen in Cntd1-/- males. Importantly, whereas the Caernohabditis elegans ortholog of CNTD1, COSA-1, appears to interact with CDK-2 during pachynema in oocytes, CNTD1 in mouse oocytes lacks the CDK-interacting motif, suggesting that its function in oocytes does not involve a cyclin-like induction of CDK2 phosphorylation activity. In contradiction to the timing of prophase I disruption in Cntd1-/- females at pachynema, we observe a dramatic depletion of primordial follicles, beginning shortly after birth and increasing dramatically through to adult ages. Those primordial follicles that persist, however, are able to develop through dictyate arrest into primary and larger follicular structures in pre-pubertal females, and only by adulthood were significant declines observed in all follicles. The early loss of oocytes in Cntd1-/- females shortly after birth, considerably earlier than that observed for mice lacking MLH1 or MLH3, suggests a novel early role for CNTD1 in CO designation and/or in ensuring progression of oocytes into dictyate arrest at late pachynema.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560096">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560096" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560096">
        <p class="paperTitle">Evolutionary adaptation of the chromodomain of the HP1-protein Rhino allows the integration of chromatin and DNA sequence signals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560096" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560096" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baumgartner, L.; Ipsaro, J. J.; Hohmann, U.; Handler, D.; Schleiffer, A.; Duchek, P.; Brennecke, J.</p>
        <p class="info">Score: 14.3, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560096' target='https://doi.org/10.1101/2023.09.29.560096'> 10.1101/2023.09.29.560096</a></p>
        <p class="abstract">Members of the diverse heterochromatin protein 1 (HP1) family of proteins play crucial roles in heterochromatin formation and maintenance. Despite the similar affinities of their chromodomains for di- and tri-methylated histone H3 lysine 9 (H3K9me2/3), different HP1 proteins exhibit distinct chromatin binding patterns, presumably due to their interactions with various specificity factors. Here, we elucidate the molecular basis of the protein-protein interaction between the HP1 protein Rhino, a critical factor of the Drosophila piRNA pathway, and Kipferl, a DNA sequence-specific C2H2 zinc finger protein and Rhino guidance factor. Through phylogenetic analyses, structure prediction, and in vivo genetics, we identify a single amino acid change within Rhino&#39;s chromodomain, G31D, that does not affect H3K9me2/3 binding but abolishes the specific interaction between Rhino and Kipferl. Flies carrying the rhinoG31D mutation phenocopy kipferl mutant flies, with Rhino redistributing from piRNA clusters to satellite repeats, causing pronounced changes in the ovarian piRNA profile of rhinoG31D flies. Thus, Rhino&#39;s chromodomain serves as a dual-specificity module, facilitating interactions with both a histone mark and a DNA-binding protein.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559307">
        <p class="paperTitle">Ensembled best subset selection using summary statistics for polygenic risk prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, T.; Zhang, H.; Mazumder, R.; Lin, X.</p>
        <p class="info">Score: 8.0, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559307' target='https://doi.org/10.1101/2023.09.25.559307'> 10.1101/2023.09.25.559307</a></p>
        <p class="abstract">Polygenic risk scores (PRS) enhance population risk stratification and advance personalized medicine, yet existing methods face a tradeoff between predictive power and computational efficiency. We introduce ALL-Sum, an easy-to-implement PRS method that combines an efficient summary statistic-based L0L2 penalized regression algorithm with an ensembling step that aggregates estimates from different tuning parameters for improved prediction performance. In extensive large-scale simulations across a wide range of polygenicity and genome-wide association studies (GWAS) sample sizes, ALL-Sum consistently outperforms popular alternative methods in terms of prediction accuracy, runtime, and memory usage. We analyze 27 published GWAS summary statistics for 11 complex traits from 9 reputable data sources, including the Global Lipids Genetics Consortium, Breast Cancer Association Consortium, and FinnGen, evaluated using individual-level UKBB data. ALL-Sum achieves the highest accuracy for most traits, particularly for GWAS with large sample sizes. We provide ALL-Sum as a user-friendly command-line software with pre-computed reference data for streamlined user-end analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558274">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558274" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558274">
        <p class="paperTitle">Transcriptome-based host epistasis and pathogen co-expression in barley-powdery mildew interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558274" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558274" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Velasquez-Zapata, V.; Smith, S. D.; Surana, P.; Chapman, A. V. E.; Wise, R. P.</p>
        <p class="info">Score: 6.7, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558274' target='https://doi.org/10.1101/2023.09.18.558274'> 10.1101/2023.09.18.558274</a></p>
        <p class="abstract">Mildew locus a (Mla) from the Triticeae grain crop barley (Hordeum vulgare L.) encodes a multi-allelic series of nucleotide-binding leucine-rich repeat (NLR) immune receptors. These variable NLRs recognize complementary secreted effectors from the powdery mildew fungus, Blumeria hordei (Bh), to block disease progression. We used a dynamic time-course transcriptome of barley infected with Bh to infer gene effects and epistatic relationships governed by the Mla6 NLR, two other host loci critical to the interaction, Blufensin1 (Bln1) and Required for Mla6 resistance3 (Rar3), and genes that interact with them. Bln1 is an R-gene independent regulator of immunity and the resistant bln1 mutant exhibits enhanced basal defense to compatible Bh. Rar3 is required for MLA6-mediated generation of H2O2 and the hypersensitive reaction; the rar3 mutant contains an in-frame Lys-Leu deletion in the SGT1-specific domain that compromises immunity by a subset of Mla alleles. Interactions of Mla6 and Bln1 resulted in symmetric, suppression and masked epistasis on the Bh-induced barley transcriptome. Likewise, dominant or equal effects were caused by Mla6 and Sgt1. Of a total of 468 barley NLRs, 366 were expressed in our dataset and 115 of those were grouped under different gene effect models, which localized to several chromosome hotspots. The corresponding Bh infection transcriptome was classified into nine co-expressed modules, linking differential expression with pathogen development. Expression of most of 517 Bh effectors exhibited dependence on disease phenotype and was associated with appressorial or haustorial structures, suggesting that disease is regulated by a host-pathogen intercommunication network that diversifies the response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559847">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559847" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559847">
        <p class="paperTitle">Autosomal suppression of meiotic drive can prevent sex chromosome cycling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559847" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559847" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, A.; Unckless, R.</p>
        <p class="info">Score: 4.9, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559847' target='https://doi.org/10.1101/2023.09.28.559847'> 10.1101/2023.09.28.559847</a></p>
        <p class="abstract">Sex-ratio meiotic drivers are selfish genes or gene complexes that bias the transmission of chromosomes resulting in skewed sex ratios. While existing theoretical models have suggested the maintenance of a four-chromosome equilibrium (X, XSR, Y, YSUP; SR: sex-ratio driver, SUP: suppressor) in a cyclic dynamic, studies of natural populations have failed to capture this pattern. Although there are several plausible explanations for this lack of cycling, interference from autosomal suppressors has not been studied using a population genetic framework even though autosomal suppressors and Y-linked suppressors both exist in populations of some species. In this study, we use a simulations-based approach to investigate the influence of autosomal suppressors on the cycling of sex chromosomes. Our findings demonstrate that the presence of an autosomal suppressor can hinder the invasion of a Y-linked suppressor under some parameter space, thereby impeding the cyclic dynamics. Even when a Y-linked suppressor invades, the presence of an autosomal suppressor can prevent cycling. Our study demonstrates the potential role of autosomal suppressors in preventing sex chromosome cycling and provides insights into the conditions and consequences of the maintenance of both Y-linked and autosomal suppressors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559059">
        <p class="paperTitle">Identification of Genetic Suppressors for a Berardinelli-Seip Congenital Generalized Lipodystrophy Type 2 (BSCL2) Pathogenic Variant in  C. elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bai, X.; Smith, H. E.; Golden, A.</p>
        <p class="info">Score: 3.6, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559059' target='https://doi.org/10.1101/2023.09.22.559059'> 10.1101/2023.09.22.559059</a></p>
        <p class="abstract">Maintaining the metabolic homeostasis of fatty acids is crucial for human health. Excess fatty acids are stored in lipid droplets (LDs), the primary energy reservoir that helps regulate fat and lipid homeostasis in nearly all cell types. Seipin (BSCL2), a conserved endoplasmic reticulum protein, plays a critical role in LD biogenesis and regulating LD morphology. Pathogenic variants of seipin are associated with multiple human genetic diseases, including Berardinelli-Seip Congenital Generalized Lipodystrophy Type 2 (BSCL2). However, the cellular and molecular mechanisms by which dysfunctional seipin leads to these diseases remain unclear. To model BSCL2 disease, we generated an orthologous BSCL2 pathogenic variant seip-1(A185P) using CRISPR/Cas9 genome editing in Caenorhabditis elegans. This variant led to severe developmental and cellular defects, including embryonic lethality, impaired eggshell formation, and abnormally enlarged LDs. We set out to identify genetic determinants that could suppress these defective phenotypes in the seip-1(A185P) mutant background. To this end, we conducted an unbiased chemical mutagenesis screen to identify genetic suppressors that restore embryonic viability in the seip-1(A185P) mutant background. A total of five suppressor lines were isolated and recovered from the screen. The defective phenotypes of seip-1(A185P), including embryonic lethality and impaired eggshell formation, were significantly suppressed in each suppressor line. Two of the five suppressor lines also alleviated the enlarged LDs in the oocytes. We then mapped a suppressor candidate gene, R05D3.2 (renamed as lmbr-1), which is an ortholog of human LMBR1 (limb development membrane protein 1). The CRISPR/Cas9 edited lmbr-1 suppressor alleles, lmbr-1(Ser647Phe) and lmbr-1(Pro314Leu), both significantly suppressed embryonic lethality and defective eggshell formation in the seip-1(A185P) background. The newly identified suppressor lines offer valuable insights into potential genetic interactors and pathways that may regulate seipin in the lipodystrophy model.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.05.15.491973">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.05.15.491973" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.05.15.491973">
        <p class="paperTitle">Stable population structure in Europe since the Iron Age, despite high mobility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.05.15.491973" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.05.15.491973" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Antonio, M. L.; Weiss, C. L.; Gao, Z.; Sawyer, S.; Oberreiter, V.; Moots, H. M.; Spence, J. P.; Cheronet, O.; Zagorc, B.; Praxmarer, E.; Ozdogan, K. T.; Demetz, L.; Gelabert, P.; Fernandes, D.; Lucci, M.; Alihodzic, T.; Amrani, S.; Avetisyan, P.; Baillif-Ducros, C.; Bedic, Z.; Bertrand, A.; Bilic, M.; Bondioli, L.; Borowka, P.; Botte, E.; Burmaz, J.; Buzanic, D.; Candilio, F.; Cvetko, M.; De Angelis, D.; Drnic, I.; Elschek, K.; Fantar, M.; Gaspari, A.; Gasperetti, G.; Genchi, F.; Golubovic, S.; Hukelova, Z.; Jankauskas, R.; Vuckovic, K. J.; Jeremic, G.; Kaic, I.; Kazek, K.; Khachatryan, H.; Kh</p>
        <p class="info">Score: 171.0, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.05.15.491973' target='https://doi.org/10.1101/2022.05.15.491973'> 10.1101/2022.05.15.491973</a></p>
        <p class="abstract">Ancient DNA research in the past decade has revealed that European population structure changed dramatically in the prehistoric period (14,000-3,000 years before present, YBP), reflecting the widespread introduction of Neolithic farmer and Bronze Age Steppe ancestries. However, little is known about how population structure changed from the historical period onward (3,000 YBP - present). To address this, we collected whole genomes from 204 individuals from Europe and the Mediterranean, many of which are the first historical period genomes from their region (e.g. Armenia and France). We found that most regions show remarkable inter-individual heterogeneity. At least 7% of historical individuals carry ancestry uncommon in the region where they were sampled, some indicating cross-Mediterranean contacts. Despite this high level of mobility, overall population structure across western Eurasia is relatively stable through the historical period up to the present, mirroring geography. We show that, under standard population genetics models with local panmixia, the observed level of dispersal would lead to a collapse of population structure. Persistent population structure thus suggests a lower effective migration rate than indicated by the observed dispersal. We hypothesize that this phenomenon can be explained by extensive transient dispersal arising from drastically improved transportation networks and the Roman Empires mobilization of people for trade, labor, and military. This work highlights the utility of ancient DNA in elucidating finer scale human population dynamics in recent history.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559464">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559464" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559464">
        <p class="paperTitle">Detection of short identity by descent segments using low-frequency variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559464" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559464" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, C.; Veldsman, W. P.; Zhang, L.</p>
        <p class="info">Score: 2.8, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559464' target='https://doi.org/10.1101/2023.09.26.559464'> 10.1101/2023.09.26.559464</a></p>
        <p class="abstract">Identity by descent (IBD) segments are inherited genomic sequences from the same common ancestors that serve as biomarkers during lineage analysis, disease mutation mapping, and within broader fields of population genetics. Current computational tools used for the detection of IBD segments focus on identifying long IBD segments, with short IBD segments (i.e. shorter than 2 cM) often going undetected. Here, we present SILO, a method with a remarkably improved ability to detect short IBD segments as compared to state-of-the-art IBD detection tools. SILO detects IBD segments by considering both common and low-frequency variants (LFVs), where the probability of the presence of a LFV minor allele is assumed to follow a Bernoulli distribution with a Beta prior. SILO was benchmarked against GERMLINE2, hap-IBD, HapFABIA, Parente, and TRUFFLE on simulated data and on a real pedigree from the 1,000 Genomes Project. The results show that SILO has an unrivaled ability to detect short IBD segments, while its ability to detect long IBD segments is similar to that of contemporary tools in the field.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
