<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Estimation of indirect genetic effects and heritability under assortative mating
Authors: Young, A. S.
Score: 34.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458
Both direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects -- effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) -- can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Estimation of indirect genetic effects and heritability under assortative mating
Authors: Young, A. S.
Score: 34.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458
Both direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects -- effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) -- can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-19T10:38:24+00:00" />
<meta property="article:modified_time" content="2023-07-19T10:38:24+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Estimation of indirect genetic effects and heritability under assortative mating
Authors: Young, A. S.
Score: 34.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458
Both direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects -- effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) -- can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Estimation of indirect genetic effects and heritability under assortative mating\nAuthors: Young, A. S.\nScore: 34.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458\nBoth direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects -- effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) -- can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium.",
  "keywords": [
    
  ],
  "articleBody": " Estimation of indirect genetic effects and heritability under assortative mating\nAuthors: Young, A. S.\nScore: 34.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458\nBoth direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects -- effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) -- can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium. We generalize classical theory to derive a decomposition of the equilibrium phenotypic variance in terms of direct and indirect genetic effect components. We extend this theory to show that popular methods for estimating indirect genetic effects or genetic nurture through analysis of parental and offspring polygenic predictors (called polygenic indices or scores -- PGIs or PGSs) are substantially biased by assortative mating. We propose an improved method for estimating indirect genetic effects while accounting for assortative mating that can also correct heritability estimates for bias due to assortative mating. We validate our method in simulations and apply it to PGIs for height and educational attainment (EA), estimating that the equilibrium heritability of height is 0.699 (S.E. = 0.075) and finding no evidence for indirect genetic effects on height. We estimate a very high correlation between parents underlying genetic components for EA, 0.755 (S.E. = 0.035), which is inconsistent with twin based estimates of the heritability of EA, possibly due to confounding in the EA PGI and/or in twin studies. We implement our method in the software package snipar, enabling researchers to apply the method to data including observed and/or imputed parental genotypes. We provide a theoretical framework for understanding the results of PGI analyses and a practical methodology for estimating heritability and indirect genetic effects while accounting for assortative mating.\nSocial and genetic diversity in first farmers of central Europe\nAuthors: Gelabert, P.; Bickle, P.; Hofmann, D.; Teschler-Nicola, M.; Anders, A.; Huang, X.; Olalde, I.; Fournier, R.; Ringbauer, H.; Akbari, A.; Cheronet, O.; Lazaridis, I.; Broomandkhoshbacht, N.; Fernandes, D. M.; Buttinger, K.; Callan, K.; Candilio, F.; Bravo, G.; Curtis, E.; Ferry, M.; Keating, D.; Freilich, S.; Kearns, A.; Harney, E.; Lawson, A. M.; Mandl, K.; Michel, M.; Oberreiter, V.; Oppenheimer, J.; Sawyer, S.; Schattke, C.; Ozdogan, K. T. T.; Qiu, L.; Workman, N.; Zalzala, F.; Mallick, S.; Mah, M.; Micco, A.; Pieler, F.; Pavuk, J.; Lazar, C.; Skrivanko, M. K.; Slaus, M.; Bedic, Z.; Novotny, Score: 33.7, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548126\nThe Linearbandkeramik (LBK) Neolithic communities were the first to spread farming across large parts of central Europe, settling fertile regions from Ukraine to France during the second half of the 6th millennium BCE. The LBK had a high degree of material culture uniformity, albeit with regional differences in settlement patterns, subsistence, and mortuary practices. To date, ancient DNA data from LBK individuals have been generated for a limited number of locations and often in small sample sizes, making it challenging to study variation within and across sites. We report genome-wide data for 178 LBK individuals, from the Alfold Linearbankeramik Culture (ALPC) eastern LBK site of Polgar-Ferenci-hat in Hungary, the western LBK site of Nitra in Slovakia, and the enclosed western LBK settlement and massacre site of Schletz in Austria, as well as 42 LBK individuals from 18 other sites. We also report genome-wide data for 28 Early Neolithic Koros and Star[c]evo individuals from 13 sites, viewed as the predecessors of the LBK. We observe a higher percentage of western hunter-gatherer (WHG) admixture among individuals in the eastern LBK than in the far more widely distributed western LBK, showing that these two archaeologically distinct cultures also had different genetic trajectories. Most WHG-farmer mixture occurred just before the dawn of the LBK culture and there is no evidence that the WHG ancestry came systematically more from males or females. However, we do find strong genetic evidence for patrilocality among the LBK, extending previous findings based on isotopic analysis, with more genetic structure across sites on the male than on the female line, and a higher rate of within-site relatives for males. At Schletz we detect almost no first-degree relatives despite reporting data from almost every skeleton present at the site, showing that this massacre involved people from a large population, not a small community.\nCRIMP: A CRISPR/Cas9 Insertional Mutagenesis Protocol and the CRIMP Toolkit\nAuthors: Miles, L. B.; Calcinotto, V.; Sonntag, C.; Lee, C.; Bryson-Richardson, R. J.\nScore: 23.4, Published: 2023-07-14 DOI: 10.1101/2023.07.13.548929\nWe developed a highly efficient targeted insertional mutagenesis system, CRIMP, and an associated plasmid toolkit, CRIMPkit, that disrupts native gene expression by inducing complete transcriptional termination to produce null mutant alleles without inducing genetic compensation. The CRIMPkit contains over 30 ready-to-use plasmid vectors allowing easy and complete mutagenesis of any gene in any reading frame without requiring custom sequences, modification, or subcloning, and also provides a fluorescent readout of successfully mutagenised fish.\nChromatin states contribute to coordinated allelic transcriptional bursting to drive iPSC reprogramming\nAuthors: Parichitran, A.; Naik, H.; Naskar, A.; Bammidi, L. S.; GAYEN, S.\nScore: 17.1, Published: 2023-07-13 DOI: 10.1101/2023.07.13.548864\nMolecular mechanisms behind the reprogramming of somatic cells to induced pluripotent stem cells (iPSC) remain poorly understood. While dynamic changes in gene expression are considered to drive reprogramming, the contribution of individual alleles of genes to reprogramming remains unexplored. It is thought that two alleles of a gene can transcribe independently or coordinatedly, which in turn can lead to temporal expression heterogeneity with potentially distinct impacts on cell fate. Here, we profiled genome-wide transcriptional burst kinetics with an allelic resolution during the reprogramming of mouse embryonic fibroblast (MEF) to iPSC. We show that many genes involved in iPSC reprogramming pathways exhibit bursty expression and contribute to dynamic autosomal random monoallelic expression (aRME). Moreover, we find that the degree of coordination of allelic bursting differs among genes and changes dynamically during iPSC reprogramming. Importantly, we show that alleles of many reprogramming-related genes burst in a highly coordinated fashion. ATAC-seq analysis revealed that coordination of allelic bursting is linked to allelic chromatin accessibility. Consistently, we show that highly coordinated genes are enriched with chromatin accessibility regulators such as H3K36me3, H3K27ac, histone variant H3.3 and BRD4. Collectively, our study demonstrates that chromatin states contribute to coordinated allelic bursting to fine-tune the expression of genes involved in iPSC reprogramming and provides insights into the implications of allelic bursting coordination in cell fate specification. O_FIG O_LINKSMALLFIG WIDTH=195 HEIGHT=200 SRC=\"FIGDIR/small/548864v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (45K): org.highwire.dtl.DTLVardef@143059dorg.highwire.dtl.DTLVardef@125c49dorg.highwire.dtl.DTLVardef@8f2f64org.highwire.dtl.DTLVardef@7083dd_HPS_FORMAT_FIGEXP M_FIG C_FIG\nVirus replication in the honey bee parasite, Varroa destructor\nAuthors: Damayo, J. E.; McKee, R. C.; Buchmann, G.; Norton, A. M.; Ashe, A.; Remnant, E. J.\nScore: 12.7, Published: 2023-07-16 DOI: 10.1101/2023.07.16.549232\nArthropod vectors such as mites and ticks introduce an alternative viral transmission route between their hosts. The ectoparasitic mite Varroa destructor is the leading threat to the health of Western honey bees (Apis mellifera) primarily through its action as a vector of viruses. However, it is unclear whether viruses transmitted by V. destructor actively infect and replicate in mites, which could facilitate increased transmission and select for more virulent strains. To better understand the role of V. destructor as a vector, we took advantage of differences between bee and mite antiviral RNA interference pathways to identify the host specificity of replicating viruses. We used small RNA sequencing of individual V. destructor mites to examine viral small interfering RNA (vsiRNA) profiles of Deformed wing virus genotypes (DWV-A and DWV-B), associated with colony declines, as well as nine other viruses present in our samples. We found active replication of six V. destructor-associated viruses, including a novel virus, Varroa destructor virus 9 (VDV-9), and replication of two honey bee associated viruses, including both DWV-A and -B genotypes, suggesting that mites are biological vectors for important bee pathogens. We show that the antiviral RNAi response can be used define the host range of viruses in host-parasite interactions, such as honey bees and their parasites, enabling a better understanding of the role of a vector in the evolution and spread of honey bee pathogens.\nsciMET-cap: High-throughput single-cell methylation analysis with a reduced sequencing burden\nAuthors: Acharya, S.; Nichols, R. V.; Rylaarsdam, L. E.; O'Connell, B. L.; Braun, T. P.; Adey, A. C.\nScore: 11.0, Published: 2023-07-14 DOI: 10.1101/2023.07.12.548718\nDNA methylation is a key component of the mammalian epigenome, playing a regulatory role in development, disease, and other processes. Robust, high-throughput single-cell DNA methylation assays are now possible (sciMET); however, the genome-wide nature of DNA methylation results in a high sequencing burden per cell. Here, we leverage target enrichment with sciMET to capture sufficient information per cell for cell type assignment using substantially fewer sequence reads (sciMET-cap). Sufficient off-target coverage further enables the production of near-complete methylomes for individual cell types. We characterize sciMET-cap on human PBMCs and brain (middle frontal gyrus).\nRecombination map tailored to Native Hawaiians reduces false positive findings in genomic scans of positive selection\nAuthors: Dinh, B. L.; Tang, E.; Taparra, K.; Nakatsuka, N.; Chen, F.; Chiang, C. W. K.\nScore: 9.9, Published: 2023-07-14 DOI: 10.1101/2023.07.12.548735\nRecombination events establish the patterns of haplotypic structure in a population and estimates of recombination rates are used in several downstream population and statistical genetic analyses. However, the recombination landscape of populations not represented in the 1000 Genomes Project (1KGP) is poorly understood. Using suboptimal maps from distantly related populations reduces the efficacy of extending genomic analyses to underrepresented populations, including Native Hawaiians. To overcome this challenge, we constructed linkage disequilibrium (LD)- and Identity-by-Descent (IBD)-based recombination maps specifically for Native Hawaiians. We created and characterized two recombination maps from our study sample: one based on a random subset of our sample reflecting the current admixed state of Native Hawaiians (NH map), and one based on a subset of individuals with enriched Polynesian ancestries (PNS map) with potential to be used for less admixed Polynesian populations such as the Samoans. We found the recombination landscape to be less correlated with those from other 1KGP populations (e.g. Spearmans rho = 0.79 between PNS and CEU (Utah residents with Northern and Western European ancestry) compared to 0.92 between YRI (Yoruba in Ibadan, Nigeria) and CEU at 50 kb resolution. PNS shared the fewest recombination hotspots with other populations from 1KGP (e.g. 8% of hotspots shared between PNS and CEU compared to 27% of hotspots shared between YRI and CEU). We found that downstream analysis in the Native Hawaiian population, such as local ancestry inference, imputation, and IBD segment and relatedness detections, would achieve similar efficacy when using the NH map compared to an omnibus map. However, for genome scans of adaptive loci using integrated haplotype scores, we found several loci with apparent genome-wide significant signals (|Z-score| \u003e 4) in Native Hawaiians that would not have been significant when analyzed using NH-specific maps. Therefore, population-specific recombination maps improve the robustness of haplotype-based statistics and help us better characterize the evolutionary history that may underlie Native Hawaiian-specific health conditions that persist today.\nValidation of human telomere length trans-ancestry meta-analysis association signals identifies POP5 and KBTBD6 as novel human telomere length regulation genes\nAuthors: Keener, R.; Chhetri, S. B.; Connelly, C. J.; Taub, M. A.; Conomos, M. P.; Weinstock, J. S.; Ni, B.; Strober, B. J.; Aslibekyan, S.; Auer, P. L.; Barwick, L.; Becker, L. C.; Blangero, J.; Bleecker, E. R.; Brody, J. A.; Cade, B. E.; Celedon, J. C.; Chang, Y.-C.; Cupples, L. A.; Custer, B.; Freedman, B. I.; Gladwin, M. T.; Heckbert, S. R.; Hou, L.; Irvin, M. R.; Isasi, C. R.; Johnsen, J. M.; Kenny, E. E.; Kooperberg, C.; Minster, R. L.; Nekhai, S.; Pankratz, N.; Peyser, P. A.; Rotter, J. I.; Taylor, K. D.; Telen, M. J.; Wu, B.; Yanek, L. R.; Yang, I. V.; Albert, C.; Arnett, D. K.; Ashley-Koch, A.\nScore: 9.2, Published: 2023-07-14 DOI: 10.1101/2023.07.12.548702\nTelomere length genome-wide association studies (GWAS) have become well-powered to detect novel genes in telomere length regulation. However, no prior work has validated these putative novel genes to confirm the contribution of GWAS loci to telomere length regulation. We conducted a trans-ancestry meta-analysis of 211,369 individuals. Through enrichment analyses of chromatin state and cell-type heritability we identified blood and immune cells as the most relevant cell type to examine telomere length association signals. We validated specific GWAS associations by overexpressing KBTBD6, a component of an E3 ubiquitin ligase complex, and POP5, a component of the Ribonuclease P/MRP complex, and demonstrating that both lengthened telomeres as predicted by our statistical analyses. CRISPR/Cas9 deletion of the predicted causal regions of these association peaks in K562 immortalized blood cells reduced expression of these genes, demonstrating that these loci are related to transcriptional regulation of KBTBD6 and POP5, respectively. Together our results demonstrate the utility of telomere length GWAS in the identification of novel telomere length regulation mechanisms and highlight the importance of the proteasome-ubiquitin pathway in telomere length regulation.\nPopulation suppression by release of insects carrying a dominant sterile homing gene drive targeting doublesex in Drosophila\nAuthors: Chen, W.; Guo, J.; Liu, Y.; Champer, J.\nScore: 3.8, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549342\nGene drive alleles, which bias their own inheritance and increase in frequency, show great promise for blocking disease transmission or directly suppressing pest populations. The most common engineered drive system is the CRISPR homing drive, which converts wild-type alleles to drive alleles in the germline of drive heterozygotes by homology-directed repair after CRISPR cleavage. One successful homing drive example targets a female-specific exon in doublesex in Anopheles mosquitos, suppressing the population by inducing recessive sterility in female drive homozygotes. We found that in Drosophila melanogaster, a 3-gRNA drive disrupting the doublesex female exon resulted in a masculine phenotype and dominant female sterility. Resistance alleles formed by end-joining repair were also dominant sterile. This was likely caused by expression of male-specific transcripts in females with drive and resistance alleles, disrupting sex development. Based on this construct, we proposed a new pest suppression system called Release of Insects carrying a Dominant-sterile Drive (RIDD). This entails continuously releasing drive heterozygous males, with drive and resistance alleles causing sterility in females. The drive remains at high frequency longer than currently used dominant female-lethal alleles (RIDL) due to drive conversion in males, and drive alleles also cause sterility based on resistance, both substantial advantages. With weekly releases of drive males into a cage population with overlapping generations, our RIDD system targeting dsx reached 100% prevalence within 27 weeks, progressively reducing egg production and eventually causing total population collapse. RIDD combines the merits of homing gene drive and RIDL. It is powerful but self-limiting, unlike unconfined standard homing drives, allowing for targeted population suppression.\nNew germline Cas9 promoters show improved performance for homing gene drive\nAuthors: Du, J.; Chen, W.; Jia, X.; Xu, X.; Yang, E.; Zhou, R.; Zhang, Y.; Metzloff, M.; Messer, P. W.; Champer, J.\nScore: 3.8, Published: 2023-07-16 DOI: 10.1101/2023.07.16.549205\nGene drive systems could be a viable strategy to prevent pathogen transmission or suppress vector populations by propagating drive alleles with super-Mendelian inheritance. CRISPR-based homing gene drives, perhaps the most powerful gene drive strategy, convert wild type alleles into drive alleles in heterozygotes with the help of Cas9 and gRNA. However, achieving successful outcomes with these drives often requires high performance. Specifically, it is desirable to identify Cas9 promoters that yield high drive conversion rates, minimize the formation rate of resistance alleles in both the germline and the early embryo, and limit somatic Cas9 expression. Thus far, high-performance promoters have only been discovered in Anopheles species. In Drosophila, the nanos promoter avoids leaky somatic expression, but at the cost of high embryo resistance from maternally deposited Cas9. To improve drive efficiency, we tested eleven Drosophila melanogaster germline promoters in several configurations. Some of the new promoters achieved higher drive conversion efficiency with minimal embryo resistance, but none could completely avoid somatic expression like nanos. However, such somatic expression often did not carry detectable fitness costs when the promoter-Cas9 elements supported a rescue homing drive targeting a haplolethal gene, suggesting somatic drive conversion. Based on our findings, we selected two Cas9 promoter lines for cage experiments with a 4-gRNA suppression drive. While one promoter exhibited substantial somatic effects, leading to a low drive equilibrium frequency, the other outperformed nanos, resulting in the successful suppression of the cage population. Overall, these novel Cas9 promoters hold potential advantages for homing drives in Drosophila species and may also possess valuable homologs in other organisms.\n",
  "wordCount" : "2722",
  "inLanguage": "en",
  "datePublished": "2023-07-19T10:38:24Z",
  "dateModified": "2023-07-19T10:38:24Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on July 19, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548458">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548458" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548458">
        <p class="paperTitle">Estimation of indirect genetic effects and heritability under assortative mating</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548458" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548458" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Young, A. S.</p>
        <p class="info">Score: 34.5, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548458' target='https://doi.org/10.1101/2023.07.10.548458'> 10.1101/2023.07.10.548458</a></p>
        <p class="abstract">Both direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects -- effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) -- can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium. We generalize classical theory to derive a decomposition of the equilibrium phenotypic variance in terms of direct and indirect genetic effect components. We extend this theory to show that popular methods for estimating indirect genetic effects or  genetic nurture through analysis of parental and offspring polygenic predictors (called polygenic indices or scores -- PGIs or PGSs) are substantially biased by assortative mating. We propose an improved method for estimating indirect genetic effects while accounting for assortative mating that can also correct heritability estimates for bias due to assortative mating. We validate our method in simulations and apply it to PGIs for height and educational attainment (EA), estimating that the equilibrium heritability of height is 0.699 (S.E. = 0.075) and finding no evidence for indirect genetic effects on height. We estimate a very high correlation between parents underlying genetic components for EA, 0.755 (S.E. = 0.035), which is inconsistent with twin based estimates of the heritability of EA, possibly due to confounding in the EA PGI and/or in twin studies. We implement our method in the software package snipar, enabling researchers to apply the method to data including observed and/or imputed parental genotypes. We provide a theoretical framework for understanding the results of PGI analyses and a practical methodology for estimating heritability and indirect genetic effects while accounting for assortative mating.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548126">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548126" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548126">
        <p class="paperTitle">Social and genetic diversity in first farmers of central Europe</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548126" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548126" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gelabert, P.; Bickle, P.; Hofmann, D.; Teschler-Nicola, M.; Anders, A.; Huang, X.; Olalde, I.; Fournier, R.; Ringbauer, H.; Akbari, A.; Cheronet, O.; Lazaridis, I.; Broomandkhoshbacht, N.; Fernandes, D. M.; Buttinger, K.; Callan, K.; Candilio, F.; Bravo, G.; Curtis, E.; Ferry, M.; Keating, D.; Freilich, S.; Kearns, A.; Harney, E.; Lawson, A. M.; Mandl, K.; Michel, M.; Oberreiter, V.; Oppenheimer, J.; Sawyer, S.; Schattke, C.; Ozdogan, K. T. T.; Qiu, L.; Workman, N.; Zalzala, F.; Mallick, S.; Mah, M.; Micco, A.; Pieler, F.; Pavuk, J.; Lazar, C.; Skrivanko, M. K.; Slaus, M.; Bedic, Z.; Novotny, </p>
        <p class="info">Score: 33.7, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548126' target='https://doi.org/10.1101/2023.07.07.548126'> 10.1101/2023.07.07.548126</a></p>
        <p class="abstract">The Linearbandkeramik (LBK) Neolithic communities were the first to spread farming across large parts of central Europe, settling fertile regions from Ukraine to France during the second half of the 6th millennium BCE. The LBK had a high degree of material culture uniformity, albeit with regional differences in settlement patterns, subsistence, and mortuary practices. To date, ancient DNA data from LBK individuals have been generated for a limited number of locations and often in small sample sizes, making it challenging to study variation within and across sites. We report genome-wide data for 178 LBK individuals, from the Alfold Linearbankeramik Culture (ALPC) eastern LBK site of Polgar-Ferenci-hat in Hungary, the western LBK site of Nitra in Slovakia, and the enclosed western LBK settlement and massacre site of Schletz in Austria, as well as 42 LBK individuals from 18 other sites. We also report genome-wide data for 28 Early Neolithic Koros and Star[c]evo individuals from 13 sites, viewed as the predecessors of the LBK. We observe a higher percentage of western hunter-gatherer (WHG) admixture among individuals in the eastern LBK than in the far more widely distributed western LBK, showing that these two archaeologically distinct cultures also had different genetic trajectories. Most WHG-farmer mixture occurred just before the dawn of the LBK culture and there is no evidence that the WHG ancestry came systematically more from males or females. However, we do find strong genetic evidence for patrilocality among the LBK, extending previous findings based on isotopic analysis, with more genetic structure across sites on the male than on the female line, and a higher rate of within-site relatives for males. At Schletz we detect almost no first-degree relatives despite reporting data from almost every skeleton present at the site, showing that this massacre involved people from a large population, not a small community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548929">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548929" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548929">
        <p class="paperTitle">CRIMP: A CRISPR/Cas9 Insertional Mutagenesis Protocol and the CRIMP Toolkit</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548929" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548929" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miles, L. B.; Calcinotto, V.; Sonntag, C.; Lee, C.; Bryson-Richardson, R. J.</p>
        <p class="info">Score: 23.4, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548929' target='https://doi.org/10.1101/2023.07.13.548929'> 10.1101/2023.07.13.548929</a></p>
        <p class="abstract">We developed a highly efficient targeted insertional mutagenesis system, CRIMP, and an associated plasmid toolkit, CRIMPkit, that disrupts native gene expression by inducing complete transcriptional termination to produce null mutant alleles without inducing genetic compensation. The CRIMPkit contains over 30 ready-to-use plasmid vectors allowing easy and complete mutagenesis of any gene in any reading frame without requiring custom sequences, modification, or subcloning, and also provides a fluorescent readout of successfully mutagenised fish.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548864">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548864" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548864">
        <p class="paperTitle">Chromatin states contribute to coordinated allelic transcriptional bursting to drive iPSC reprogramming</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548864" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548864" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parichitran, A.; Naik, H.; Naskar, A.; Bammidi, L. S.; GAYEN, S.</p>
        <p class="info">Score: 17.1, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548864' target='https://doi.org/10.1101/2023.07.13.548864'> 10.1101/2023.07.13.548864</a></p>
        <p class="abstract">Molecular mechanisms behind the reprogramming of somatic cells to induced pluripotent stem cells (iPSC) remain poorly understood. While dynamic changes in gene expression are considered to drive reprogramming, the contribution of individual alleles of genes to reprogramming remains unexplored. It is thought that two alleles of a gene can transcribe independently or coordinatedly, which in turn can lead to temporal expression heterogeneity with potentially distinct impacts on cell fate. Here, we profiled genome-wide transcriptional burst kinetics with an allelic resolution during the reprogramming of mouse embryonic fibroblast (MEF) to iPSC. We show that many genes involved in iPSC reprogramming pathways exhibit bursty expression and contribute to dynamic autosomal random monoallelic expression (aRME). Moreover, we find that the degree of coordination of allelic bursting differs among genes and changes dynamically during iPSC reprogramming. Importantly, we show that alleles of many reprogramming-related genes burst in a highly coordinated fashion. ATAC-seq analysis revealed that coordination of allelic bursting is linked to allelic chromatin accessibility. Consistently, we show that highly coordinated genes are enriched with chromatin accessibility regulators such as H3K36me3, H3K27ac, histone variant H3.3 and BRD4. Collectively, our study demonstrates that chromatin states contribute to coordinated allelic bursting to fine-tune the expression of genes involved in iPSC reprogramming and provides insights into the implications of allelic bursting coordination in cell fate specification.



O_FIG O_LINKSMALLFIG WIDTH=195 HEIGHT=200 SRC=&#34;FIGDIR/small/548864v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (45K):
org.highwire.dtl.DTLVardef@143059dorg.highwire.dtl.DTLVardef@125c49dorg.highwire.dtl.DTLVardef@8f2f64org.highwire.dtl.DTLVardef@7083dd_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549232">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549232" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549232">
        <p class="paperTitle">Virus replication in the honey bee parasite, Varroa destructor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549232" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549232" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Damayo, J. E.; McKee, R. C.; Buchmann, G.; Norton, A. M.; Ashe, A.; Remnant, E. J.</p>
        <p class="info">Score: 12.7, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549232' target='https://doi.org/10.1101/2023.07.16.549232'> 10.1101/2023.07.16.549232</a></p>
        <p class="abstract">Arthropod vectors such as mites and ticks introduce an alternative viral transmission route between their hosts. The ectoparasitic mite Varroa destructor is the leading threat to the health of Western honey bees (Apis mellifera) primarily through its action as a vector of viruses. However, it is unclear whether viruses transmitted by V. destructor actively infect and replicate in mites, which could facilitate increased transmission and select for more virulent strains. To better understand the role of V. destructor as a vector, we took advantage of differences between bee and mite antiviral RNA interference pathways to identify the host specificity of replicating viruses. We used small RNA sequencing of individual V. destructor mites to examine viral small interfering RNA (vsiRNA) profiles of Deformed wing virus genotypes (DWV-A and DWV-B), associated with colony declines, as well as nine other viruses present in our samples. We found active replication of six V. destructor-associated viruses, including a novel virus, Varroa destructor virus 9 (VDV-9), and replication of two honey bee associated viruses, including both DWV-A and -B genotypes, suggesting that mites are biological vectors for important bee pathogens. We show that the antiviral RNAi response can be used define the host range of viruses in host-parasite interactions, such as honey bees and their parasites, enabling a better understanding of the role of a vector in the evolution and spread of honey bee pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548718">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548718" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548718">
        <p class="paperTitle">sciMET-cap: High-throughput single-cell methylation analysis with a reduced sequencing burden</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548718" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548718" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Acharya, S.; Nichols, R. V.; Rylaarsdam, L. E.; O&#39;Connell, B. L.; Braun, T. P.; Adey, A. C.</p>
        <p class="info">Score: 11.0, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548718' target='https://doi.org/10.1101/2023.07.12.548718'> 10.1101/2023.07.12.548718</a></p>
        <p class="abstract">DNA methylation is a key component of the mammalian epigenome, playing a regulatory role in development, disease, and other processes. Robust, high-throughput single-cell DNA methylation assays are now possible (sciMET); however, the genome-wide nature of DNA methylation results in a high sequencing burden per cell. Here, we leverage target enrichment with sciMET to capture sufficient information per cell for cell type assignment using substantially fewer sequence reads (sciMET-cap). Sufficient off-target coverage further enables the production of near-complete methylomes for individual cell types. We characterize sciMET-cap on human PBMCs and brain (middle frontal gyrus).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548735" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548735">
        <p class="paperTitle">Recombination map tailored to Native Hawaiians reduces false positive findings in genomic scans of positive selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dinh, B. L.; Tang, E.; Taparra, K.; Nakatsuka, N.; Chen, F.; Chiang, C. W. K.</p>
        <p class="info">Score: 9.9, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548735' target='https://doi.org/10.1101/2023.07.12.548735'> 10.1101/2023.07.12.548735</a></p>
        <p class="abstract">Recombination events establish the patterns of haplotypic structure in a population and estimates of recombination rates are used in several downstream population and statistical genetic analyses. However, the recombination landscape of populations not represented in the 1000 Genomes Project (1KGP) is poorly understood. Using suboptimal maps from distantly related populations reduces the efficacy of extending genomic analyses to underrepresented populations, including Native Hawaiians. To overcome this challenge, we constructed linkage disequilibrium (LD)- and Identity-by-Descent (IBD)-based recombination maps specifically for Native Hawaiians. We created and characterized two recombination maps from our study sample: one based on a random subset of our sample reflecting the current admixed state of Native Hawaiians (NH map), and one based on a subset of individuals with enriched Polynesian ancestries (PNS map) with potential to be used for less admixed Polynesian populations such as the Samoans. We found the recombination landscape to be less correlated with those from other 1KGP populations (e.g. Spearmans rho = 0.79 between PNS and CEU (Utah residents with Northern and Western European ancestry) compared to 0.92 between YRI (Yoruba in Ibadan, Nigeria) and CEU at 50 kb resolution. PNS shared the fewest recombination hotspots with other populations from 1KGP (e.g. 8% of hotspots shared between PNS and CEU compared to 27% of hotspots shared between YRI and CEU). We found that downstream analysis in the Native Hawaiian population, such as local ancestry inference, imputation, and IBD segment and relatedness detections, would achieve similar efficacy when using the NH map compared to an omnibus map. However, for genome scans of adaptive loci using integrated haplotype scores, we found several loci with apparent genome-wide significant signals (|Z-score| &gt; 4) in Native Hawaiians that would not have been significant when analyzed using NH-specific maps. Therefore, population-specific recombination maps improve the robustness of haplotype-based statistics and help us better characterize the evolutionary history that may underlie Native Hawaiian-specific health conditions that persist today.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548702">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548702" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548702">
        <p class="paperTitle">Validation of human telomere length trans-ancestry meta-analysis association signals identifies POP5 and KBTBD6 as novel human telomere length regulation genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548702" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548702" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Keener, R.; Chhetri, S. B.; Connelly, C. J.; Taub, M. A.; Conomos, M. P.; Weinstock, J. S.; Ni, B.; Strober, B. J.; Aslibekyan, S.; Auer, P. L.; Barwick, L.; Becker, L. C.; Blangero, J.; Bleecker, E. R.; Brody, J. A.; Cade, B. E.; Celedon, J. C.; Chang, Y.-C.; Cupples, L. A.; Custer, B.; Freedman, B. I.; Gladwin, M. T.; Heckbert, S. R.; Hou, L.; Irvin, M. R.; Isasi, C. R.; Johnsen, J. M.; Kenny, E. E.; Kooperberg, C.; Minster, R. L.; Nekhai, S.; Pankratz, N.; Peyser, P. A.; Rotter, J. I.; Taylor, K. D.; Telen, M. J.; Wu, B.; Yanek, L. R.; Yang, I. V.; Albert, C.; Arnett, D. K.; Ashley-Koch, A.</p>
        <p class="info">Score: 9.2, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548702' target='https://doi.org/10.1101/2023.07.12.548702'> 10.1101/2023.07.12.548702</a></p>
        <p class="abstract">Telomere length genome-wide association studies (GWAS) have become well-powered to detect novel genes in telomere length regulation. However, no prior work has validated these putative novel genes to confirm the contribution of GWAS loci to telomere length regulation. We conducted a trans-ancestry meta-analysis of 211,369 individuals. Through enrichment analyses of chromatin state and cell-type heritability we identified blood and immune cells as the most relevant cell type to examine telomere length association signals. We validated specific GWAS associations by overexpressing KBTBD6, a component of an E3 ubiquitin ligase complex, and POP5, a component of the Ribonuclease P/MRP complex, and demonstrating that both lengthened telomeres as predicted by our statistical analyses. CRISPR/Cas9 deletion of the predicted causal regions of these association peaks in K562 immortalized blood cells reduced expression of these genes, demonstrating that these loci are related to transcriptional regulation of KBTBD6 and POP5, respectively. Together our results demonstrate the utility of telomere length GWAS in the identification of novel telomere length regulation mechanisms and highlight the importance of the proteasome-ubiquitin pathway in telomere length regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549342">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549342" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549342">
        <p class="paperTitle">Population suppression by release of insects carrying a dominant sterile homing gene drive targeting doublesex in Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549342" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549342" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, W.; Guo, J.; Liu, Y.; Champer, J.</p>
        <p class="info">Score: 3.8, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549342' target='https://doi.org/10.1101/2023.07.17.549342'> 10.1101/2023.07.17.549342</a></p>
        <p class="abstract">Gene drive alleles, which bias their own inheritance and increase in frequency, show great promise for blocking disease transmission or directly suppressing pest populations. The most common engineered drive system is the CRISPR homing drive, which converts wild-type alleles to drive alleles in the germline of drive heterozygotes by homology-directed repair after CRISPR cleavage. One successful homing drive example targets a female-specific exon in doublesex in Anopheles mosquitos, suppressing the population by inducing recessive sterility in female drive homozygotes. We found that in Drosophila melanogaster, a 3-gRNA drive disrupting the doublesex female exon resulted in a masculine phenotype and dominant female sterility. Resistance alleles formed by end-joining repair were also dominant sterile. This was likely caused by expression of male-specific transcripts in females with drive and resistance alleles, disrupting sex development. Based on this construct, we proposed a new pest suppression system called Release of Insects carrying a Dominant-sterile Drive (RIDD). This entails continuously releasing drive heterozygous males, with drive and resistance alleles causing sterility in females. The drive remains at high frequency longer than currently used dominant female-lethal alleles (RIDL) due to drive conversion in males, and drive alleles also cause sterility based on resistance, both substantial advantages. With weekly releases of drive males into a cage population with overlapping generations, our RIDD system targeting dsx reached 100% prevalence within 27 weeks, progressively reducing egg production and eventually causing total population collapse. RIDD combines the merits of homing gene drive and RIDL. It is powerful but self-limiting, unlike unconfined standard homing drives, allowing for targeted population suppression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549205">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549205" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549205">
        <p class="paperTitle">New germline Cas9 promoters show improved performance for homing gene drive</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549205" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549205" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Du, J.; Chen, W.; Jia, X.; Xu, X.; Yang, E.; Zhou, R.; Zhang, Y.; Metzloff, M.; Messer, P. W.; Champer, J.</p>
        <p class="info">Score: 3.8, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549205' target='https://doi.org/10.1101/2023.07.16.549205'> 10.1101/2023.07.16.549205</a></p>
        <p class="abstract">Gene drive systems could be a viable strategy to prevent pathogen transmission or suppress vector populations by propagating drive alleles with super-Mendelian inheritance. CRISPR-based homing gene drives, perhaps the most powerful gene drive strategy, convert wild type alleles into drive alleles in heterozygotes with the help of Cas9 and gRNA. However, achieving successful outcomes with these drives often requires high performance. Specifically, it is desirable to identify Cas9 promoters that yield high drive conversion rates, minimize the formation rate of resistance alleles in both the germline and the early embryo, and limit somatic Cas9 expression. Thus far, high-performance promoters have only been discovered in Anopheles species. In Drosophila, the nanos promoter avoids leaky somatic expression, but at the cost of high embryo resistance from maternally deposited Cas9. To improve drive efficiency, we tested eleven Drosophila melanogaster germline promoters in several configurations. Some of the new promoters achieved higher drive conversion efficiency with minimal embryo resistance, but none could completely avoid somatic expression like nanos. However, such somatic expression often did not carry detectable fitness costs when the promoter-Cas9 elements supported a rescue homing drive targeting a haplolethal gene, suggesting somatic drive conversion. Based on our findings, we selected two Cas9 promoter lines for cage experiments with a 4-gRNA suppression drive. While one promoter exhibited substantial somatic effects, leading to a low drive equilibrium frequency, the other outperformed nanos, resulting in the successful suppression of the cage population. Overall, these novel Cas9 promoters hold potential advantages for homing drives in Drosophila species and may also possess valuable homologs in other organisms.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
