<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Estimation of indirect genetic effects and heritability under assortative mating
Authors: Young, A. S.
Score: 12.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458
Both direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects - effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) - can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Estimation of indirect genetic effects and heritability under assortative mating
Authors: Young, A. S.
Score: 12.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458
Both direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects - effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) - can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-12T10:41:10+00:00" />
<meta property="article:modified_time" content="2023-07-12T10:41:10+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Estimation of indirect genetic effects and heritability under assortative mating
Authors: Young, A. S.
Score: 12.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458
Both direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects - effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) - can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Estimation of indirect genetic effects and heritability under assortative mating\nAuthors: Young, A. S.\nScore: 12.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458\nBoth direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects - effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) - can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium.",
  "keywords": [
    
  ],
  "articleBody": " Estimation of indirect genetic effects and heritability under assortative mating\nAuthors: Young, A. S.\nScore: 12.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458\nBoth direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects - effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) - can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium. We generalize classical theory to derive a decomposition of the equilibrium phenotypic variance in terms of direct and indirect genetic effect components. We extend this theory to show that popular methods for estimating indirect genetic effects (genetic nurture) through analysis of parental and offspring polygenic predictors (called polygenic indices or scores - PGIs or PGSs) are substantially biased by assortative mating. We propose an improved method for estimating indirect genetic effects while accounting for assortative mating that can also correct heritability estimates for bias due to assortative mating. We validate our method in simulations and apply it to PGIs for height and educational attainment (EA), estimating that the equilibrium heritability of height is 0.699 (S.E. = 0.075) and finding no evidence for indirect genetic effects on height. We estimate a very high correlation between parents' underlying genetic components for EA, 0.755 (S.E. = 0.035), which is inconsistent with twin based estimates of the heritability of EA, possibly due to confounding in the EA PGI and/or in twin studies. We implement our method in the software package snipar, enabling researchers to apply the method to data including observed and/or imputed parental genotypes. We provide a theoretical framework for understanding the results of PGI analyses and a practical methodology for estimating heritability and indirect genetic effects while accounting for assortative mating.\nPolygenic risk score prediction accuracy convergence\nAuthors: Henches, L.; Kim, J.; Yang, Z.; Rubinacci, S.; Pires, G.; Albinana, C.; Boetto, C.; Julienne, H.; Frouin, A.; Auvergne, A.; Suzuki, Y.; Djebali, S.; Delaneau, O.; Ganna, A.; Vilhjalmsson, B.; Prive, F.; Aschard, H.\nScore: 33.5, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546518\nPolygenic risk scores (PRS) trained from genome-wide association study (GWAS) results are set to play a pivotal role in biomedical research addressing multifactorial human diseases. The prospect of using these risk scores in clinical care and public health is generating both enthusiasm and controversy, with varying opinions about strengths and limitations across experts1. The performances of existing polygenic scores are still limited, and although it is expected to improve with increasing sample size of GWAS and the development of new powerful methods, it remains unclear how much prediction can be ultimately achieved. Here, we conducted a retrospective analysis to assess the progress in PRS prediction accuracy since the publication of the first large-scale GWASs using six common human diseases with sufficient GWAS data. We show that while PRS accuracy has grown rapidly for years, the improvement pace from recent GWAS has decreased substantially, suggesting that further increasing GWAS sample size may translate into very modest risk discrimination improvement. We next investigated the factors influencing the maximum achievable prediction using recently released whole genome-sequencing data from 125K UK Biobank participants, and state-of-the-art modeling of polygenic outcomes. Our analyses point toward increasing the variant coverage of PRS, using either more imputed variants or sequencing data, as a key component for future improvement in prediction accuracy.\nGenetic similarity between relatives provides evidence on the presence and history of assortative mating\nAuthors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.\nScore: 40.5, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546663\nAssortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives1-7. However, empirical evidence is limited for many traits, and the implications hinge on whether assortative mating has started recently or many generations ago. Here we show theoretically and empirically that genetic similarity between relatives can provide evidence on the presence and history of assortative mating. First, we employed path analysis to understand how assortative mating affects genetic similarity between family members across generations, finding that similarity between distant relatives is more affected than close relatives. Next, we correlated polygenic indices of 47,135 co-parents from the Norwegian Mother, Father, and Child Cohort Study (MoBa) and found genetic evidence of assortative mating in eight out of fifteen examined traits. The same traits show elevated similarity between relatives, especially distant relatives. Five of the eight traits, including educational attainment, showed greater genetic variance among offspring, which is inconsistent with stable assortative mating over many generations. These results suggest an ongoing increase in familial similarity for these traits. The implications of this research extend to genetic methodology and the understanding of social and economic disparities.\nAccounting for Isoform Expression in eQTL Mapping Substantially Increases Power\nAuthors: LaPierre, N.; Pimentel, H.\nScore: 23.8, Published: 2023-06-30 DOI: 10.1101/2023.06.28.546921\nA core problem in genetics is eQTL mapping, in which genetic variants associated with changes in expression of genes are identified. It is common in eQTL mapping to compute gene expression by aggregating the expression levels of individual isoforms from the same gene and then performing linear regression between SNPs and this aggregated gene expression level. However, SNPs may regulate isoforms from the same gene in different directions due to alternative splicing, or only regulate the expression level of one isoform, causing this approach to lose power. In this study, we provide a systematic evaluation of methods for accounting for individual isoform expression levels based on generative isoform expression heritability models and real data. Over a range of conditions, we show that these approaches substantially increase the power to map eQTLs in both simulations and commonly analyzed large data sets. We identify settings in which different approaches yield an inflated number of false discoveries or lose power. In particular, we show that calling an eGene if there is a significant association between a SNP and any isoform fails to control False Discovery Rate, even when applying standard False Discovery Rate correction. We show that similar trends are observed in real data from the GEUVADIS and GTEx studies, suggesting the possibility that similar effects are present in these consortia.\nSocial and genetic diversity in first farmers of central Europe\nAuthors: Gelabert, P.; Bickle, P.; Hofmann, D.; Teschler-Nicola, M.; Anders, A.; Huang, X.; Olalde, I.; Fournier, R.; Ringbauer, H.; Akbari, A.; Cheronet, O.; Lazaridis, I.; Broomandkhoshbacht, N.; Fernandes, D. M.; Buttinger, K.; Callan, K.; Candilio, F.; Bravo, G.; Curtis, E.; Ferry, M.; Keating, D.; Freilich, S.; Kearns, A.; Harney, E.; Lawson, A. M.; Mandl, K.; Michel, M.; Oberreiter, V.; Oppenheimer, J.; Sawyer, S.; Schattke, C.; Ozdogan, K. T. T.; Qiu, L.; Workman, N.; Zalzala, F.; Mallick, S.; Mah, M.; Micco, A.; Pieler, F.; Pavuk, J.; Lazar, C.; Skrivanko, M. K.; Slaus, M.; Bedic, Z.; Novotny, Score: 6.2, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548126\nThe Linearbandkeramik (LBK) Neolithic communities were the first to spread farming across large parts of central Europe, settling fertile regions from Ukraine to France during the second half of the 6th millennium BCE. The LBK had a high degree of material culture uniformity, albeit with regional differences in settlement patterns, subsistence, and mortuary practices. To date, ancient DNA data from LBK individuals have been generated for a limited number of locations and often in small sample sizes, making it challenging to study variation within and across sites. We report genome-wide data for 178 LBK individuals, from the Alfold Linearbankeramik Culture (ALPC) eastern LBK site of Polgar-Ferenci-hat in Hungary, the western LBK site of Nitra in Slovakia, and the enclosed western LBK settlement and massacre site of Schletz in Austria, as well as 42 LBK individuals from 18 other sites. We also report genome-wide data for 28 Early Neolithic Koros and Starcevo individuals from 13 sites, viewed as the predecessors of the LBK. We observe a higher percentage of western hunter-gatherer (WHG) admixture among individuals in the eastern LBK than in the far more widely distributed western LBK, showing that these two archaeologically distinct cultures also had different genetic trajectories. Most WHG-farmer mixture occurred just before the dawn of the LBK culture and there is no evidence that the WHG ancestry came systematically more from males or females. However, we do find strong genetic evidence for patrilocality among the LBK, extending previous findings based on isotopic analysis, with more genetic structure across sites on the male than on the female line, and a higher rate of within-site relatives for males. At Schletz we detect almost no first-degree relatives despite reporting data from almost every skeleton present at the site, showing that this massacre involved people from a large population, not a small community.\nDISCOVERY AND VALIDATION OF GENES DRIVING DRUG-INTAKE AND RELATED BEHAVIORAL TRAITS IN MICE\nAuthors: Roy, T.; Bubier, J. A.; Dickson, P. E.; Wilcox, T.; Ndukum, J.; Clark, J. W.; Rizzo, S. J. S.; Crabbe, J. C.; Denegre, J.; Svenson, K. L.; Braun, R. E.; Kumar, V.; Murray, S. A.; White, J. K.; Philip, V.; Chesler, E.\nScore: 6.1, Published: 2023-07-10 DOI: 10.1101/2023.07.09.548280\nSubstance use disorders (SUDs) are heritable disorders characterized by compulsive drug use, but the biological mechanisms driving addiction remain largely unknown. Genetic correlations reveal that predisposing drug-naive phenotypes, including anxiety, depression, novelty preference, and sensation seeking, are predictive of drug-use phenotypes, implicating shared genetic mechanisms. Because of this relationship, high-throughput behavioral screening of predictive phenotypes in knockout (KO) mice allows efficient discovery of genes likely to be involved in drug use. We used this strategy in two rounds of screening in which we identified 33 drug-use candidate genes and ultimately validated the perturbation of 22 of these genes as causal drivers of substance intake. In our initial round of screening, we employed the two-bottle-choice paradigms to assess alcohol, methamphetamine, and nicotine intake. We identified 19 KO strains that were extreme responders on at least one predictive phenotype. Thirteen of the 19 gene deletions (68%) significantly affected alcohol use three methamphetamine use, and two both. In the second round of screening, we employed a multivariate approach to identify outliers and performed validation using methamphetamine two-bottle choice and ethanol drinking-in-the-dark protocols. We identified 15 KO strains that were extreme responders across the predisposing drug-naive phenotypes. Eight of the 15 gene deletions (53%) significantly affected intake or preference for three alcohol, eight methamphetamine or three both (3). We observed multiple relations between predisposing behaviors and drug intake, revealing many distinct biobehavioral processes underlying these relationships. The set of mouse models identified in this study can be used to characterize these addiction-related processes further.\nMulti-trait GWAS for diverse ancestries: Mapping the knowledge gap\nAuthors: Troubat, L.; Fettahoglu, D.; Aschard, H.; Julienne, H.\nScore: 10.7, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546248\nBackgroundApproximately 95% of samples analyzed in univariate genome-wide association studies (GWAS) are of European ancestry. This bias toward European ancestry populations in association screening also exists for other analyses and methods that are often developed and tested on European ancestry only. However, existing data in non-European populations, which are often of modest sample size, could benefit from innovative approaches as recently illustrated in the context of polygenic risk scores. MethodsHere, we extend and assess the potential limitations and gain of our multi-trait GWAS pipeline, JASS (Joint Analysis of Summary Statistics), for the analysis of non-European ancestries. To this end, we conducted the joint GWAS of 19 hematological traits and glycemic traits accross five ancestries (European (EUR), admixed American (AMR), African (AFR), East Asian (EAS), South-East Asian (SAS)). ResultsWe detected 367 new genome-wide significant associations in non-European populations (15 in Admixed American (AMR), 72 in African (AFR) and 280 in East Asian (EAS)). New associations detected represent 7%, 25% and 21% of associations in the AFR, AMR and EAS populations, respectively. Overall, multi-trait testing increases the replication of European associated loci in non-European ancestry by 15%. Pleiotropic effects were highly similar at significant loci across ancestries (e.g. the mean correlation between multi-trait genetic effects of EUR and EAS ancestries was 0.88). For hematological traits, strong discrepancies in multi-trait genetic effects are tied to known evolutionary divergences: the ARKC1 loci, which is adaptive to overcome the p.vivax induced malaria. ConclusionsMulti-trait GWAS can be a valuable tool to narrow the genetic knowledge gap between European and non-European populations.\nPolygenic Risk Score Associates with Atherosclerotic Plaque Characteristics at Autopsy\nAuthors: Cornelissen, A.; Gadhoke, N.; Ryan, K. A.; Hodonsky, C. J.; Mitchell, R. N.; Bihlmeyer, N. A.; Duong, T.; Chen, Z.; Dikongue, A.; Sakamoto, A.; Sato, Y.; Kawakami, R.; Mori, M.; Kawai, K.; Fernandez, R.; Ghosh, S. K. B.; Braumann, R.; Abebe, B.; Kutys, R.; Kutyna, M. D.; Romero, M. E.; Kolodgie, F.; Miller, C. L.; Hong, C. C.; Grove, M. L.; Brody, J. A.; Sotoodehnia, N.; Arking, D. E.; Schunkert, H.; Mitchell, B. D.; Guo, L.; Virmani, R.; Finn, A. V.\nScore: 5.4, Published: 2023-07-09 DOI: 10.1101/2023.07.05.547891\nBackground: Polygenic risk scores (PRS) for coronary artery disease (CAD) potentially improve cardiovascular risk prediction. However, their relationship with histopathologic features of CAD has never been examined systematically. Methods: From 4,327 subjects referred to CVPath by the State of Maryland Office Chief Medical Examiner (OCME) for sudden death between 1994 and 2015, 2,455 cases were randomly selected for genotyping. We generated PRS from 291 known CAD risk loci. Detailed histopathologic examination of the coronary arteries was performed in all subjects. The primary study outcome measurements were histopathologic plaque features determining severity of atherosclerosis, including %stenosis, calcification, thin-cap fibroatheromas (TCFA), and thrombotic CAD. Results: After exclusion of cases with insufficient DNA sample quality or with missing data, 954 cases (mean age 48.8{+/-}14.7; 75.7% men) remained in the final study cohort. Subjects in the highest PRS quintile exhibited more severe atherosclerosis compared to subjects in the lowest quintile, with greater %stenosis (80.3%{+/-}27.0% vs. 50.4%{+/-}38.7%; adjusted p\u003c0.001) and a higher frequency of calcification (69.6% vs. 35.8%; adjusted p=0.004) and TCFAs (26.7% vs. 9.5%; adjusted p=0.007). Even after adjustment for traditional CAD risk factors subjects within the highest PRS quintile had higher odds of severe atherosclerosis (i.e., ?75% stenosis; adjusted OR 3.77; 95%CI 2.10-6.78; p\u003c0.001) and plaque rupture (adjusted OR 4.05; 95%CI 2.26-7.24; p\u003c0.001). Moreover, subjects within the highest quintile had higher odds of CAD-associated cause of death, especially among those aged 50 years and younger (adjusted OR 4.08; 95%CI 2.01-8.30; p\u003c0.001). No associations were observed with plaque erosion. Conclusions: This is the first autopsy study investigating associations between PRS and atherosclerosis severity at the histopathologic level in subjects with sudden death. Our pathological analysis suggests PRS correlates with plaque burden and features of advanced atherosclerosis and may be useful as a method for CAD risk stratification, especially in younger subjects.\nModeling soybean growth: A mixed model approach\nAuthors: DELATTRE, M.; Toda, Y.; Tressou, J.; Iwata, H.\nScore: 8.4, Published: 2023-06-13 DOI: 10.1101/2023.06.13.544713\nThe evaluation of plant and animal growth, separately for genetic and environmental effects, is necessary for genetic understanding and genetic improvement of environmental responses of plants and animals. We propose an original approach that combines nonlinear mixed-effects model (NLME) and the stochastic approximation of the Expectation-Maximization algorithm (SAEM) to analyze genetic and environmental effects on plant growth. These tools are widely used in many fields but very rarely in plant biology. During model formulation, a nonlinear mechanistic function describes the shape of growth, and random effects describe genetic and environmental effects and their variability. Genetic relationships among the varieties were also integrated into the model using a kinship matrix. The SAEM algorithm was chosen as an efficient alternative to MCMC methods, which are more commonly used in the domain. It was implemented to infer the expected growth patterns in the analyzed population and the expected curves for each variety through a maximum-likelihood and a maximum-a-posteriori approaches, respectively. The obtained estimates can be used to predict the growth curves for each variety. We illustrate the strengths of the proposed approach using simulated data and soybean plant growth data obtained from a soybean cultivation experiment conducted at the Arid Land Research Center, Tottori University. In this experiment, plant height was measured daily using drones, and the growth was monitored for approximately 200 soybean cultivars for which whole-genome sequence data were available. The NLME approach improved our understanding of the determinants of soybean growth and can be successfully used for the genomic prediction of growth pattern characteristics. Author summaryNonlinear mechanistic models are useful for modeling animal and plant growth; however, their parameters are influenced by both genetic and environmental factors. If the same model can be applied to data with different genetic and environmental factors by allowing parameter variations, it can be used to understand, predict, and control the genetic and environmental influences of growth models based on parameter variation. In this study, we propose a statistical method for integrating a nonlinear mixed-effects model with a nonlinear mechanistic model. The simulation and real data analysis results show that the proposed method was effective in modeling the growth of genetically different soybean varieties under different drought conditions. The usefulness of the proposed method is expected to increase, as high-throughput measurements provide growth data for a large number of genotypes in various environments.\nHigh throughput functional profiling of genes at intraocular pressure loci reveals distinct networks for glaucoma\nAuthors: Greatbatch, C. J.; Lu, Q.; Hung, S.; Wing, K.; Liang, H.; Han, X.; Zhou, T.; Siggs, O.; Mackey, D.; Cook, A.; Senabouth, A.; Liu, G.-S.; Craig, J.; MacGregor, S.; Powell, J.; Hewitt, A.\nScore: 2.8, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548340\nINTRODUCTION: Primary open angle glaucoma (POAG) is a leading cause of blindness globally. Characterised by progressive retinal ganglion cell degeneration, the precise pathogenesis remains unknown. Genome-wide association studies (GWAS) have uncovered many genetic variants associated with elevated intraocular pressure (IOP), one of the key risk factors for POAG. This study sought to investigate the morphological and transcriptional consequences of perturbation of key genes at IOP loci in trabecular meshwork cell (TMC); the cellular regulators of IOP. We aimed to identify genetic and morphological variation that can be attributed to TMC dysfunction and raised IOP in POAG. METHODS: 62 genes across 55 loci were knocked-out in a primary human TMC line. Each knockout group, including five non-targeting control groups, underwent single-cell RNA-sequencing (scRNA-seq) for differentially-expressed gene (DEG) analysis. Multiplexed fluorescent staining of key organelles, was coupled with high-throughput microscopy for single-cell morphological analysis using CellProfiler image analysis. RESULTS: Across many of the individual gene knockouts scRNA-seq highlighted genes relating to matrix metalloproteinases and interferon-induced proteins. Our work has prioritised genes at four loci of interest to identify gene knockouts that may contribute to the pathogenesis of POAG, including ANGPTL2, LMX1B, CAV1, and KREMEN1. Three genetic networks of gene knockouts with similar transcriptomic profiles were identified (ABO / CAV1 / MYOC, ANGPT2 / PKHD1 / TNS1 / TXNRD2, and CAPZA1 / KALRN / LMO7 / PLEKHA7 / GNB1L / TEX41), suggesting a synergistic function in trabecular meshwork cell physiology. TEK knockout caused significant upregulation of nuclear granularity on morphological analysis, whilst knockout of TRIOBP, TMCO1 and PLEKHA7 increased granularity and intensity of actin and the cell-membrane. CONCLUSION: High throughput analysis of cellular structure and function through multiplex fluorescent single-cell analysis and scRNA-seq assays enabled the direct study of genetic perturbations at the single-cell resolution. This work provides a framework for investigating the role of genes in the pathogenesis of glaucoma and heterogenous diseases with a strong genetic basis.\n",
  "wordCount" : "3217",
  "inLanguage": "en",
  "datePublished": "2023-07-12T10:41:10Z",
  "dateModified": "2023-07-12T10:41:10Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on July 12, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548458">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548458" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548458">
        <p class="paperTitle">Estimation of indirect genetic effects and heritability under assortative mating</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548458" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548458" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Young, A. S.</p>
        <p class="info">Score: 12.5, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548458' target='https://doi.org/10.1101/2023.07.10.548458'> 10.1101/2023.07.10.548458</a></p>
        <p class="abstract">Both direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects - effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) - can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium. We generalize classical theory to derive a decomposition of the equilibrium phenotypic variance in terms of direct and indirect genetic effect components. We extend this theory to show that popular methods for estimating indirect genetic effects (genetic nurture) through analysis of parental and offspring polygenic predictors (called polygenic indices or scores - PGIs or PGSs) are substantially biased by assortative mating. We propose an improved method for estimating indirect genetic effects while accounting for assortative mating that can also correct heritability estimates for bias due to assortative mating. We validate our method in simulations and apply it to PGIs for height and educational attainment (EA), estimating that the equilibrium heritability of height is 0.699 (S.E. = 0.075) and finding no evidence for indirect genetic effects on height. We estimate a very high correlation between parents&#39; underlying genetic components for EA, 0.755 (S.E. = 0.035), which is inconsistent with twin based estimates of the heritability of EA, possibly due to confounding in the EA PGI and/or in twin studies. We implement our method in the software package snipar, enabling researchers to apply the method to data including observed and/or imputed parental genotypes. We provide a theoretical framework for understanding the results of PGI analyses and a practical methodology for estimating heritability and indirect genetic effects while accounting for assortative mating.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546518">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546518" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546518">
        <p class="paperTitle">Polygenic risk score prediction accuracy convergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546518" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546518" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Henches, L.; Kim, J.; Yang, Z.; Rubinacci, S.; Pires, G.; Albinana, C.; Boetto, C.; Julienne, H.; Frouin, A.; Auvergne, A.; Suzuki, Y.; Djebali, S.; Delaneau, O.; Ganna, A.; Vilhjalmsson, B.; Prive, F.; Aschard, H.</p>
        <p class="info">Score: 33.5, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546518' target='https://doi.org/10.1101/2023.06.27.546518'> 10.1101/2023.06.27.546518</a></p>
        <p class="abstract">Polygenic risk scores (PRS) trained from genome-wide association study (GWAS) results are set to play a pivotal role in biomedical research addressing multifactorial human diseases. The prospect of using these risk scores in clinical care and public health is generating both enthusiasm and controversy, with varying opinions about strengths and limitations across experts1. The performances of existing polygenic scores are still limited, and although it is expected to improve with increasing sample size of GWAS and the development of new powerful methods, it remains unclear how much prediction can be ultimately achieved. Here, we conducted a retrospective analysis to assess the progress in PRS prediction accuracy since the publication of the first large-scale GWASs using six common human diseases with sufficient GWAS data. We show that while PRS accuracy has grown rapidly for years, the improvement pace from recent GWAS has decreased substantially, suggesting that further increasing GWAS sample size may translate into very modest risk discrimination improvement. We next investigated the factors influencing the maximum achievable prediction using recently released whole genome-sequencing data from 125K UK Biobank participants, and state-of-the-art modeling of polygenic outcomes. Our analyses point toward increasing the variant coverage of PRS, using either more imputed variants or sequencing data, as a key component for future improvement in prediction accuracy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546663">
        <p class="paperTitle">Genetic similarity between relatives provides evidence on the presence and history of assortative mating</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.</p>
        <p class="info">Score: 40.5, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546663' target='https://doi.org/10.1101/2023.06.27.546663'> 10.1101/2023.06.27.546663</a></p>
        <p class="abstract">Assortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives1-7. However, empirical evidence is limited for many traits, and the implications hinge on whether assortative mating has started recently or many generations ago. Here we show theoretically and empirically that genetic similarity between relatives can provide evidence on the presence and history of assortative mating. First, we employed path analysis to understand how assortative mating affects genetic similarity between family members across generations, finding that similarity between distant relatives is more affected than close relatives. Next, we correlated polygenic indices of 47,135 co-parents from the Norwegian Mother, Father, and Child Cohort Study (MoBa) and found genetic evidence of assortative mating in eight out of fifteen examined traits. The same traits show elevated similarity between relatives, especially distant relatives. Five of the eight traits, including educational attainment, showed greater genetic variance among offspring, which is inconsistent with stable assortative mating over many generations. These results suggest an ongoing increase in familial similarity for these traits. The implications of this research extend to genetic methodology and the understanding of social and economic disparities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546921" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546921">
        <p class="paperTitle">Accounting for Isoform Expression in eQTL Mapping Substantially Increases Power</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: LaPierre, N.; Pimentel, H.</p>
        <p class="info">Score: 23.8, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546921' target='https://doi.org/10.1101/2023.06.28.546921'> 10.1101/2023.06.28.546921</a></p>
        <p class="abstract">A core problem in genetics is eQTL mapping, in which genetic variants associated with changes in expression of genes are identified. It is common in eQTL mapping to compute gene expression by aggregating the expression levels of individual isoforms from the same gene and then performing linear regression between SNPs and this aggregated gene expression level. However, SNPs may regulate isoforms from the same gene in different directions due to alternative splicing, or only regulate the expression level of one isoform, causing this approach to lose power. In this study, we provide a systematic evaluation of methods for accounting for individual isoform expression levels based on generative isoform expression heritability models and real data. Over a range of conditions, we show that these approaches substantially increase the power to map eQTLs in both simulations and commonly analyzed large data sets. We identify settings in which different approaches yield an inflated number of false discoveries or lose power. In particular, we show that calling an eGene if there is a significant association between a SNP and any isoform fails to control False Discovery Rate, even when applying standard False Discovery Rate correction. We show that similar trends are observed in real data from the GEUVADIS and GTEx studies, suggesting the possibility that similar effects are present in these consortia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548126">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548126" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548126">
        <p class="paperTitle">Social and genetic diversity in first farmers of central Europe</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548126" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548126" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gelabert, P.; Bickle, P.; Hofmann, D.; Teschler-Nicola, M.; Anders, A.; Huang, X.; Olalde, I.; Fournier, R.; Ringbauer, H.; Akbari, A.; Cheronet, O.; Lazaridis, I.; Broomandkhoshbacht, N.; Fernandes, D. M.; Buttinger, K.; Callan, K.; Candilio, F.; Bravo, G.; Curtis, E.; Ferry, M.; Keating, D.; Freilich, S.; Kearns, A.; Harney, E.; Lawson, A. M.; Mandl, K.; Michel, M.; Oberreiter, V.; Oppenheimer, J.; Sawyer, S.; Schattke, C.; Ozdogan, K. T. T.; Qiu, L.; Workman, N.; Zalzala, F.; Mallick, S.; Mah, M.; Micco, A.; Pieler, F.; Pavuk, J.; Lazar, C.; Skrivanko, M. K.; Slaus, M.; Bedic, Z.; Novotny, </p>
        <p class="info">Score: 6.2, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548126' target='https://doi.org/10.1101/2023.07.07.548126'> 10.1101/2023.07.07.548126</a></p>
        <p class="abstract">The Linearbandkeramik (LBK) Neolithic communities were the first to spread farming across large parts of central Europe, settling fertile regions from Ukraine to France during the second half of the 6th millennium BCE. The LBK had a high degree of material culture uniformity, albeit with regional differences in settlement patterns, subsistence, and mortuary practices. To date, ancient DNA data from LBK individuals have been generated for a limited number of locations and often in small sample sizes, making it challenging to study variation within and across sites. We report genome-wide data for 178 LBK individuals, from the Alfold Linearbankeramik Culture (ALPC) eastern LBK site of Polgar-Ferenci-hat in Hungary, the western LBK site of Nitra in Slovakia, and the enclosed western LBK settlement and massacre site of Schletz in Austria, as well as 42 LBK individuals from 18 other sites. We also report genome-wide data for 28 Early Neolithic Koros and Starcevo individuals from 13 sites, viewed as the predecessors of the LBK. We observe a higher percentage of western hunter-gatherer (WHG) admixture among individuals in the eastern LBK than in the far more widely distributed western LBK, showing that these two archaeologically distinct cultures also had different genetic trajectories. Most WHG-farmer mixture occurred just before the dawn of the LBK culture and there is no evidence that the WHG ancestry came systematically more from males or females. However, we do find strong genetic evidence for patrilocality among the LBK, extending previous findings based on isotopic analysis, with more genetic structure across sites on the male than on the female line, and a higher rate of within-site relatives for males. At Schletz we detect almost no first-degree relatives despite reporting data from almost every skeleton present at the site, showing that this massacre involved people from a large population, not a small community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.09.548280">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.09.548280" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.09.548280">
        <p class="paperTitle">DISCOVERY AND VALIDATION OF GENES DRIVING DRUG-INTAKE AND RELATED BEHAVIORAL TRAITS IN MICE</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.09.548280" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.09.548280" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roy, T.; Bubier, J. A.; Dickson, P. E.; Wilcox, T.; Ndukum, J.; Clark, J. W.; Rizzo, S. J. S.; Crabbe, J. C.; Denegre, J.; Svenson, K. L.; Braun, R. E.; Kumar, V.; Murray, S. A.; White, J. K.; Philip, V.; Chesler, E.</p>
        <p class="info">Score: 6.1, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.09.548280' target='https://doi.org/10.1101/2023.07.09.548280'> 10.1101/2023.07.09.548280</a></p>
        <p class="abstract">Substance use disorders (SUDs) are heritable disorders characterized by compulsive drug use, but the biological mechanisms driving addiction remain largely unknown. Genetic correlations reveal that predisposing drug-naive phenotypes, including anxiety, depression, novelty preference, and sensation seeking, are predictive of drug-use phenotypes, implicating shared genetic mechanisms. Because of this relationship, high-throughput behavioral screening of predictive phenotypes in knockout (KO) mice allows efficient discovery of genes likely to be involved in drug use. We used this strategy in two rounds of screening in which we identified 33 drug-use candidate genes and ultimately validated the perturbation of 22 of these genes as causal drivers of substance intake. In our initial round of screening, we employed the two-bottle-choice paradigms to assess alcohol, methamphetamine, and nicotine intake. We identified 19 KO strains that were extreme responders on at least one predictive phenotype. Thirteen of the 19 gene deletions (68%) significantly affected alcohol use three methamphetamine use, and two both. In the second round of screening, we employed a multivariate approach to identify outliers and performed validation using methamphetamine two-bottle choice and ethanol drinking-in-the-dark protocols. We identified 15 KO strains that were extreme responders across the predisposing drug-naive phenotypes. Eight of the 15 gene deletions (53%) significantly affected intake or preference for three alcohol, eight methamphetamine or three both (3). We observed multiple relations between predisposing behaviors and drug intake, revealing many distinct biobehavioral processes underlying these relationships. The set of mouse models identified in this study can be used to characterize these addiction-related processes further.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546248">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546248" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546248">
        <p class="paperTitle">Multi-trait GWAS for diverse ancestries: Mapping the knowledge gap</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546248" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546248" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Troubat, L.; Fettahoglu, D.; Aschard, H.; Julienne, H.</p>
        <p class="info">Score: 10.7, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546248' target='https://doi.org/10.1101/2023.06.23.546248'> 10.1101/2023.06.23.546248</a></p>
        <p class="abstract">BackgroundApproximately 95% of samples analyzed in univariate genome-wide association studies (GWAS) are of European ancestry. This bias toward European ancestry populations in association screening also exists for other analyses and methods that are often developed and tested on European ancestry only. However, existing data in non-European populations, which are often of modest sample size, could benefit from innovative approaches as recently illustrated in the context of polygenic risk scores.

MethodsHere, we extend and assess the potential limitations and gain of our multi-trait GWAS pipeline, JASS (Joint Analysis of Summary Statistics), for the analysis of non-European ancestries. To this end, we conducted the joint GWAS of 19 hematological traits and glycemic traits accross five ancestries (European (EUR), admixed American (AMR), African (AFR), East Asian (EAS), South-East Asian (SAS)).

ResultsWe detected 367 new genome-wide significant associations in non-European populations (15 in Admixed American (AMR), 72 in African (AFR) and 280 in East Asian (EAS)). New associations detected represent 7%, 25% and 21% of associations in the AFR, AMR and EAS populations, respectively. Overall, multi-trait testing increases the replication of European associated loci in non-European ancestry by 15%. Pleiotropic effects were highly similar at significant loci across ancestries (e.g. the mean correlation between multi-trait genetic effects of EUR and EAS ancestries was 0.88). For hematological traits, strong discrepancies in multi-trait genetic effects are tied to known evolutionary divergences: the ARKC1 loci, which is adaptive to overcome the p.vivax induced malaria.

ConclusionsMulti-trait GWAS can be a valuable tool to narrow the genetic knowledge gap between European and non-European populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547891">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547891" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547891">
        <p class="paperTitle">Polygenic Risk Score Associates with Atherosclerotic Plaque Characteristics at Autopsy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547891" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547891" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cornelissen, A.; Gadhoke, N.; Ryan, K. A.; Hodonsky, C. J.; Mitchell, R. N.; Bihlmeyer, N. A.; Duong, T.; Chen, Z.; Dikongue, A.; Sakamoto, A.; Sato, Y.; Kawakami, R.; Mori, M.; Kawai, K.; Fernandez, R.; Ghosh, S. K. B.; Braumann, R.; Abebe, B.; Kutys, R.; Kutyna, M. D.; Romero, M. E.; Kolodgie, F.; Miller, C. L.; Hong, C. C.; Grove, M. L.; Brody, J. A.; Sotoodehnia, N.; Arking, D. E.; Schunkert, H.; Mitchell, B. D.; Guo, L.; Virmani, R.; Finn, A. V.</p>
        <p class="info">Score: 5.4, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547891' target='https://doi.org/10.1101/2023.07.05.547891'> 10.1101/2023.07.05.547891</a></p>
        <p class="abstract">Background: Polygenic risk scores (PRS) for coronary artery disease (CAD) potentially improve cardiovascular risk prediction. However, their relationship with histopathologic features of CAD has never been examined systematically. Methods: From 4,327 subjects referred to CVPath by the State of Maryland Office Chief Medical Examiner (OCME) for sudden death between 1994 and 2015, 2,455 cases were randomly selected for genotyping. We generated PRS from 291 known CAD risk loci. Detailed histopathologic examination of the coronary arteries was performed in all subjects. The primary study outcome measurements were histopathologic plaque features determining severity of atherosclerosis, including %stenosis, calcification, thin-cap fibroatheromas (TCFA), and thrombotic CAD. Results: After exclusion of cases with insufficient DNA sample quality or with missing data, 954 cases (mean age 48.8{&#43;/-}14.7; 75.7% men) remained in the final study cohort. Subjects in the highest PRS quintile exhibited more severe atherosclerosis compared to subjects in the lowest quintile, with greater %stenosis (80.3%{&#43;/-}27.0% vs. 50.4%{&#43;/-}38.7%; adjusted p&lt;0.001) and a higher frequency of calcification (69.6% vs. 35.8%; adjusted p=0.004) and TCFAs (26.7% vs. 9.5%; adjusted p=0.007). Even after adjustment for traditional CAD risk factors subjects within the highest PRS quintile had higher odds of severe atherosclerosis (i.e., ?75% stenosis; adjusted OR 3.77; 95%CI 2.10-6.78; p&lt;0.001) and plaque rupture (adjusted OR 4.05; 95%CI 2.26-7.24; p&lt;0.001). Moreover, subjects within the highest quintile had higher odds of CAD-associated cause of death, especially among those aged 50 years and younger (adjusted OR 4.08; 95%CI 2.01-8.30; p&lt;0.001). No associations were observed with plaque erosion. Conclusions: This is the first autopsy study investigating associations between PRS and atherosclerosis severity at the histopathologic level in subjects with sudden death. Our pathological analysis suggests PRS correlates with plaque burden and features of advanced atherosclerosis and may be useful as a method for CAD risk stratification, especially in younger subjects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.13.544713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.13.544713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.13.544713">
        <p class="paperTitle">Modeling soybean growth: A mixed model approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.13.544713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.13.544713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: DELATTRE, M.; Toda, Y.; Tressou, J.; Iwata, H.</p>
        <p class="info">Score: 8.4, Published: 2023-06-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.13.544713' target='https://doi.org/10.1101/2023.06.13.544713'> 10.1101/2023.06.13.544713</a></p>
        <p class="abstract">The evaluation of plant and animal growth, separately for genetic and environmental effects, is necessary for genetic understanding and genetic improvement of environmental responses of plants and animals. We propose an original approach that combines nonlinear mixed-effects model (NLME) and the stochastic approximation of the Expectation-Maximization algorithm (SAEM) to analyze genetic and environmental effects on plant growth. These tools are widely used in many fields but very rarely in plant biology. During model formulation, a nonlinear mechanistic function describes the shape of growth, and random effects describe genetic and environmental effects and their variability. Genetic relationships among the varieties were also integrated into the model using a kinship matrix. The SAEM algorithm was chosen as an efficient alternative to MCMC methods, which are more commonly used in the domain. It was implemented to infer the expected growth patterns in the analyzed population and the expected curves for each variety through a maximum-likelihood and a maximum-a-posteriori approaches, respectively. The obtained estimates can be used to predict the growth curves for each variety. We illustrate the strengths of the proposed approach using simulated data and soybean plant growth data obtained from a soybean cultivation experiment conducted at the Arid Land Research Center, Tottori University. In this experiment, plant height was measured daily using drones, and the growth was monitored for approximately 200 soybean cultivars for which whole-genome sequence data were available. The NLME approach improved our understanding of the determinants of soybean growth and can be successfully used for the genomic prediction of growth pattern characteristics.

Author summaryNonlinear mechanistic models are useful for modeling animal and plant growth; however, their parameters are influenced by both genetic and environmental factors. If the same model can be applied to data with different genetic and environmental factors by allowing parameter variations, it can be used to understand, predict, and control the genetic and environmental influences of growth models based on parameter variation. In this study, we propose a statistical method for integrating a nonlinear mixed-effects model with a nonlinear mechanistic model. The simulation and real data analysis results show that the proposed method was effective in modeling the growth of genetically different soybean varieties under different drought conditions. The usefulness of the proposed method is expected to increase, as high-throughput measurements provide growth data for a large number of genotypes in various environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548340">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548340" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548340">
        <p class="paperTitle">High throughput functional profiling of genes at intraocular pressure loci reveals distinct networks for glaucoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548340" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548340" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Greatbatch, C. J.; Lu, Q.; Hung, S.; Wing, K.; Liang, H.; Han, X.; Zhou, T.; Siggs, O.; Mackey, D.; Cook, A.; Senabouth, A.; Liu, G.-S.; Craig, J.; MacGregor, S.; Powell, J.; Hewitt, A.</p>
        <p class="info">Score: 2.8, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548340' target='https://doi.org/10.1101/2023.07.10.548340'> 10.1101/2023.07.10.548340</a></p>
        <p class="abstract">INTRODUCTION: Primary open angle glaucoma (POAG) is a leading cause of blindness globally. Characterised by progressive retinal ganglion cell degeneration, the precise pathogenesis remains unknown. Genome-wide association studies (GWAS) have uncovered many genetic variants associated with elevated intraocular pressure (IOP), one of the key risk factors for POAG. This study sought to investigate the morphological and transcriptional consequences of perturbation of key genes at IOP loci in trabecular meshwork cell (TMC); the cellular regulators of IOP. We aimed to identify genetic and morphological variation that can be attributed to TMC dysfunction and raised IOP in POAG. METHODS: 62 genes across 55 loci were knocked-out in a primary human TMC line. Each knockout group, including five non-targeting control groups, underwent single-cell RNA-sequencing (scRNA-seq) for differentially-expressed gene (DEG) analysis. Multiplexed fluorescent staining of key organelles, was coupled with high-throughput microscopy for single-cell morphological analysis using CellProfiler image analysis. RESULTS: Across many of the individual gene knockouts scRNA-seq highlighted genes relating to matrix metalloproteinases and interferon-induced proteins. Our work has prioritised genes at four loci of interest to identify gene knockouts that may contribute to the pathogenesis of POAG, including ANGPTL2, LMX1B, CAV1, and KREMEN1. Three genetic networks of gene knockouts with similar transcriptomic profiles were identified (ABO / CAV1 / MYOC, ANGPT2 / PKHD1 / TNS1 / TXNRD2, and CAPZA1 / KALRN / LMO7 / PLEKHA7 / GNB1L / TEX41), suggesting a synergistic function in trabecular meshwork cell physiology. TEK knockout caused significant upregulation of nuclear granularity on morphological analysis, whilst knockout of TRIOBP, TMCO1 and PLEKHA7 increased granularity and intensity of actin and the cell-membrane. CONCLUSION: High throughput analysis of cellular structure and function through multiplex fluorescent single-cell analysis and scRNA-seq assays enabled the direct study of genetic perturbations at the single-cell resolution. This work provides a framework for investigating the role of genes in the pathogenesis of glaucoma and heterogenous diseases with a strong genetic basis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
