<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency
Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.
Score: 63.6, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808
Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency
Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.
Score: 63.6, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808
Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-15T10:40:26+00:00" />
<meta property="article:modified_time" content="2023-11-15T10:40:26+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency
Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.
Score: 63.6, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808
Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency\nAuthors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.\nScore: 63.6, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808\nEctopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming.",
  "keywords": [
    
  ],
  "articleBody": " Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency\nAuthors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.\nScore: 63.6, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808\nEctopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming. Using neural networks that map DNA sequence to ATAC-seq profiles at base-resolution, we annotated cell-state-specific predictive transcription factor (TF) motif syntax in regulatory elements, inferred affinity- and concentration-dependent dynamics of Tn5-bias corrected TF footprints, linked peaks to putative target genes, and elucidated rewiring of TF- to-gene cis-regulatory networks. Our models reveal that early in reprogramming, OSK, at supraphysiological concentrations, rapidly open transient regulatory elements by occupying non-canonical low-affinity binding sites. As OSK concentration falls, the accessibility of these transient elements decays as a function of motif affinity. We find that these OSK-dependent transient elements sequester the somatic TF AP-1. This redistribution is strongly associated with the silencing of fibroblast-specific genes within individual nuclei. Together, our integrated single-cell resource and models reveal insights into the cis-regulatory code of reprogramming at unprecedented resolution, connect TF stoichiometry and motif syntax to diversification of cell fate trajectories, and provide new perspectives on the dynamics and role of transient regulatory elements in somatic silencing.\nHeritability within groups is uninformative about differences among groups: cases from behavioral, evolutionary, and statistical genetics\nAuthors: Schraiber, J. G.; Edge, M. D.\nScore: 75.2, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565864\nWithout the ability to control or randomize environments (or genotypes), it is difficult to determine the degree to which observed phenotypic difference between two groups of individuals are due to genetic vs. environmental differences. However, some have suggested that these concerns may be limited to pathological cases, and methods have appeared that seem to give--directly or indirectly--some support to claims that aggregate heritable variation within groups can be related to heritable variation among groups. We consider three families of approaches: the \"between-group heritability\" sometimes invoked in behavior genetics, the statistic PST used in empirical work in evolutionary quantitative genetics, and methods based on variation in ancestry in an admixed population, used in anthropological and statistical genetics. In each case, we show mathematically that information on within-group genetic and phenotypic information in the aggregate cannot separate among-group differences into genetic and environmental components, and we provide simulation results that support our claims. We discuss these results in terms of the long-running debate on this topic.\nMilo2.0 unlocks population genetic analyses of cell state abundance using a count-based mixed model\nAuthors: Kluzer, A.; Marioni, J. C.; Morgan, M. D.\nScore: 21.0, Published: 2023-11-11 DOI: 10.1101/2023.11.08.566176\nCell type proportions vary between individuals and are heritable, as demonstrated by statistical genetic analysis of flow cytometry data [1,2]. Higher-resolution cell states can be identified by single-cell RNA-sequencing, the scalability of which now makes it applicable to population-scale cohorts. However, the integration of statistical genetic analysis of cell states using cohort-scale single-cell data requires appropriate algorithms to account for and model the genetic relationships and complex batch-processing inherent to these studies. We describe Milo2.0, which enables the discovery of cell state quantitative trait loci (csQTL), scaling to millions of cells across hundreds of individuals. We identify \u003e 500 csQTLs across peripheral blood immune states and investigate their relationship with the genetic regulation of gene expression. Moreover, we colocalise immune csQTLs with human traits and identify links between immune regulators, cell state abundance and immune-mediated disease.\nGenome assembly of the dyeing poison frog provides insights into the dynamics of transposable element and genome-size evolution\nAuthors: Dittrich, C.; Hoelzl, F.; Smith, S.; Fouilloux, C. A.; Parker, D.; O'Connell, L. A.; Knowles, L. S.; Hughes, M.; Fewings, A.; Morgan, R.; Rojas, B.; Comeault, A. A.\nScore: 21.4, Published: 2023-11-07 DOI: 10.1101/2023.11.06.565769\nGenome size varies greatly across the tree of life and transposable elements are an important contributor to this variation. Among vertebrates, amphibians display the greatest variation in genome size, making them ideal models to explore the causes and consequences of genome size variation. However, high-quality genome assemblies for amphibians have, until recently, been rare. Here, we generate a high-quality genome assembly for the dyeing poison frog, Dendrobates tinctorius. We compare this assembly to publicly-available frog genomes and find evidence for both large-scale conserved synteny and widespread rearrangements between frog lineages. Comparing conserved orthologs annotated in these genomes revealed a strong correlation between genome size and gene size. To explore the cause of gene-size variation, we quantified the location of transposable elements relative to gene features and find that the accumulation of transposable elements in introns has played an important role in the evolution of gene size in D. tinctorius, while estimates of insertion times suggest that many insertion events are recent and species-specific. Finally, we show that the diversity and abundance of transposable elements in poison frog genomes can complicate genotyping efforts that rely on repetitive elements as sequence anchors. Our results show that transposable elements have clearly played an important role in the evolution of large genome size in D. tinctorius. Future studies are needed to fully understand the dynamics of transposable element evolution and to optimise primer or bait design for cost-effective population-level genotyping in species with large, repetitive genomes. SignificanceAmphibians display more variation in genome size than any other vertebrate lineage. Complexities associated with large genomes frequently hamper genome assembly and population genetic studies. Here we use long-read HiFi sequences to generate a high-quality 6.3 Gbp genome assembly of the poison frog Dendrobates tinctorius. We use this genome and leverage comparative genomics and de novo annotations to quantify aspects of genome evolution driven by repetitive transposable genetic elements. Our results provide support for the dynamic role that transposable elements play in driving the evolution of \"genomic gigantism\" in amphibians. We also show how transposable elements can be leveraged for cost-efficient population genetic studies using limited input material.\nGenetic Discovery Enabled by A Large Language Model\nAuthors: Tu, T.; Fang, Z.; Cheng, Z.; Spasic, S.; Palepu, A.; Stankovic, K.; Natarajan, V.; Peltz, G.\nScore: 15.2, Published: 2023-11-12 DOI: 10.1101/2023.11.09.566468\nArtificial intelligence (AI) has been used in many areas of medicine, and recently large language models (LLMs) have shown potential utility for clinical applications. However, since we do not know if the use of LLMs can accelerate the pace of genetic discovery, we used data generated from mouse genetic models to investigate this possibility. We examined whether a recently developed specialized LLM (Med-PaLM 2) could analyze sets of candidate genes generated from analysis of murine models of biomedical traits. In response to free-text input, Med-PaLM 2 correctly identified the murine genes that contained experimentally verified causative genetic factors for six biomedical traits, which included susceptibility to diabetes and cataracts. Med-PaLM 2 was also able to analyze a list of genes with high impact alleles, which were identified by comparative analysis of murine genomic sequence data, and it identified a causative murine genetic factor for spontaneous hearing loss. Based upon this Med-PaLM 2 finding, a novel bigenic model for susceptibility to spontaneous hearing loss was developed. These results demonstrate Med-PaLM 2 can analyze gene-phenotype relationships and generate novel hypotheses, which can facilitate genetic discovery.\nDistinct and interdependent functions of three RING proteins regulate recombination during mammalian meiosis\nAuthors: Ito, M.; Yun, Y.; Kulkarni, D. S.; Sandhu, S.; Nunez, B.; Hu, L.; Lee, K.; Lim, N.; Hirota, R.; Prendergast, R.; Huang, C.; Huang, I.; Hunter, N.\nScore: 12.6, Published: 2023-11-10 DOI: 10.1101/2023.11.07.566091\nCrossing-over between homologous chromosomes is essential for accurate segregation during meiosis. In mammals, factors required for crossing over include RNF212 and HEI10, RING-finger domain proteins implicated in modification by SUMO and ubiquitin, respectively. Here we show that a third RING protein, RNF212B, is essential for crossing over in mouse and characterize its relationships with RNF212 and HEI10. Mutant analysis indicates that, analogous to RNF212, RNF212B regulates recombination occurring in the context of synapsed chromosomes by stabilizing pro-crossover factors, regulating the progression of recombination, and enabling the differentiation and maturation of crossover-specific recombination complexes. Also, like RNF212, RNF212B localizes between synapsed chromosomes, initially as numerous foci along synaptonemal complexes (SCs) before undergoing HEI10-dependent redistribution to accumulate at prospective crossover sites. Despite these similarities, accumulation of RNF212B at nascent crossover sites is much greater than RNF212 and occurs earlier, coincident with the appearance of HEI10 foci, implicating concerted action of RNF212B and HEI10 in the differentiation of crossover sites. RNF212B localization also shows greater dependence on DNA double-strand breaks and SC formation than RNF212. Together, our analysis delineates three distinct but interdependent RING proteins that regulate meiotic recombination by integrating signals from DSBs, synapsis, the cell-cycle, and developing crossover sites.\nWhole genome sequencing analysis identifies rare, large-effect non-coding variants and regions associated with circulating protein levels\nAuthors: Hawkes, G.; Chundru, K.; Jackson, L.; Patel, K. A.; Murray, A.; Wood, A. R.; Wright, C. R.; Weedon, M. N.; Frayling, T. M.; Beaumont, R. N.\nScore: 30.1, Published: 2023-11-05 DOI: 10.1101/2023.11.04.565589\nThe role of non-coding rare variation in common phenotypes is largely unknown, due to a lack of whole-genome sequence data, and the difficulty of categorising non-coding variants into biologically meaningful regulatory units. To begin addressing these challenges, we performed a cis association analysis using whole-genome sequence data, consisting of 391 million variants and 1,450 circulating protein levels in [~]20,000 UK Biobank participants. We identified 777 independent rare non-coding single variants associated with circulating protein levels (P\u003c1x10-9), after conditioning on protein-coding and common associated variants. Rare non-coding aggregate testing identified 108 conditionally independent regulatory regions. Unlike protein-coding variation, rare non-coding genetic variation was almost as likely to increase as decrease protein levels. The regions we identified overlapped predicted tissue-specific enhancers more than promoters, suggesting they represent tissue-specific regulatory regions. Our results have important implications for the identification, and role, of rare non-coding variation associated with common human phenotypes.\nBenchmarking kinship estimation tools for ancient genomes using pedigree simulations\nAuthors: Aktürk, S.; Mapelli, I.; Güler, M. N.; Gürün, K.; Katırcıoglu, B.; Vural, K. B.; Saglıcan, E.; Cetin, M.; Yaka, R.; Sürer, E.; Atag, G.; Cokoglu, S. S.; Sevkar, A.; Altınısık, N. E.; Koptekin, D.; Somel, M.\nScore: 11.8, Published: 2023-11-12 DOI: 10.1101/2023.11.08.566300\nThere is growing interest in uncovering genetic kinship patterns in past societies using low-coverage paleogenomes. Here, we benchmark four tools for kinship estimation with such data: lcMLkin, NgsRelate, KIN, and READ, which differ in their input, IBD-estimation methods and statistical approaches. We used pedigree and ancient genome sequence simulations to evaluate these tools when only a limited number (1K to 50K) of shared SNPs (with minor allele frequency [\u0026ge;]0.01) are available. The performance of all four tools was comparable using [\u0026ge;]20K SNPs. We found that first-degree related pairs can be accurately classified even with 1K SNPs, with 85% F1 scores using READ and 96% using NgsRelate or lcMLkin. Distinguishing third-degree relatives from unrelated pairs or second-degree relatives was also possible with high accuracy (F1 \u003e90%) with 5K SNPs using NgsRelate and lcMLkin, while READ and KIN showed lower success (69% and 79%, respectively). Meanwhile, noise in population allele frequencies and inbreeding (first cousin mating) led to deviations in kinship coefficients, with different sensitivities across tools. We conclude that using multiple tools in parallel might be an effective approach to achieve robust estimates on ultra-low coverage genomes.\nIncreasing equity in science requires better ethics training: a course by trainees, for trainees\nAuthors: Patel, R. A.; Ungar, R. A.; Pyke, A. L.; Adimoelja, A.; Chakraborty, M.; Cotter, D. J.; Freund, M.; Goddard, P.; Gomez-Stafford, J.; Greenwald, E.; Higgs, E.; Hunter, N.; MacKenzie, T. M. G.; Narain, A.; Martschenko, D. O.\nScore: 32.0, Published: 2023-11-06 DOI: 10.1101/2023.11.03.565577\nDespite the profound impacts of scientific research, few scientists have received the necessary training to productively discuss the ethical and societal implications of their work. To address this critical gap, we - a group of predominantly human genetics trainees - developed a course on genetics, ethics, and society. We intend for this course to serve as a template for other institutions and scientific disciplines. Our curriculum positions human genetics within its historical and societal context and encourages students to evaluate how societal norms and structures impact the conduct of scientific research. We demonstrate the utility of this course via surveys of enrolled students and provide resources and strategies for others hoping to teach a similar course. We conclude by arguing that if we are to work towards rectifying the inequities and injustices produced by our field, we must first learn to view our own research as impacting and being impacted by society.\nAssessing the impact of 20th century internal migrations on the genetic structure of Estonia\nAuthors: Kuznetsov, I. A.; Estonian Biobank Research Team, ; Metspalu, M.; Vainik, U.; Pagani, L.; Montinaro, F.; Pankratov, V.\nScore: 30.3, Published: 2023-10-29 DOI: 10.1101/2023.10.25.564036\nSpatial genetic structure observed in many human populations is in large part attributed to past demographic events and isolation by distance. However, how intensifying migration affects this structure remains understudied. Here we harness a sample of more than 180 thousand individuals to explore the genetic correlates and consequences of contemporary migrations in Estonia. While we show that migration smoothens the genome-wide genetic structure, it intensifies inter-regional differences in polygenic scores (PGS) for certain traits, derived both from population as well as within-sibship studies. The strongest effect is observed for educational attainment which is consistent with previous observations in the UK and suggests this to be a general pattern. We explore those regional differences in PGS in terms of the driving forces behind them and from a temporal perspective, and suggest urbanisation as a major driver for this pattern in Estonia from at least the first half of the 20th century.\n",
  "wordCount" : "2312",
  "inLanguage": "en",
  "datePublished": "2023-11-15T10:40:26Z",
  "dateModified": "2023-11-15T10:40:26Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 15, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560808">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560808" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560808">
        <p class="paperTitle">Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560808" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560808" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.</p>
        <p class="info">Score: 63.6, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560808' target='https://doi.org/10.1101/2023.10.04.560808'> 10.1101/2023.10.04.560808</a></p>
        <p class="abstract">Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming. Using neural networks that map DNA sequence to ATAC-seq profiles at base-resolution, we annotated cell-state-specific predictive transcription factor (TF) motif syntax in regulatory elements, inferred affinity- and concentration-dependent dynamics of Tn5-bias corrected TF footprints, linked peaks to putative target genes, and elucidated rewiring of TF- to-gene cis-regulatory networks. Our models reveal that early in reprogramming, OSK, at supraphysiological concentrations, rapidly open transient regulatory elements by occupying non-canonical low-affinity binding sites. As OSK concentration falls, the accessibility of these transient elements decays as a function of motif affinity. We find that these OSK-dependent transient elements sequester the somatic TF AP-1. This redistribution is strongly associated with the silencing of fibroblast-specific genes within individual nuclei. Together, our integrated single-cell resource and models reveal insights into the cis-regulatory code of reprogramming at unprecedented resolution, connect TF stoichiometry and motif syntax to diversification of cell fate trajectories, and provide new perspectives on the dynamics and role of transient regulatory elements in somatic silencing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565864">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565864" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565864">
        <p class="paperTitle">Heritability within groups is uninformative about differences among groups: cases from behavioral, evolutionary, and statistical genetics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565864" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565864" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schraiber, J. G.; Edge, M. D.</p>
        <p class="info">Score: 75.2, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565864' target='https://doi.org/10.1101/2023.11.06.565864'> 10.1101/2023.11.06.565864</a></p>
        <p class="abstract">Without the ability to control or randomize environments (or genotypes), it is difficult to determine the degree to which observed phenotypic difference between two groups of individuals are due to genetic vs. environmental differences. However, some have suggested that these concerns may be limited to pathological cases, and methods have appeared that seem to give--directly or indirectly--some support to claims that aggregate heritable variation within groups can be related to heritable variation among groups. We consider three families of approaches: the &#34;between-group heritability&#34; sometimes invoked in behavior genetics, the statistic PST used in empirical work in evolutionary quantitative genetics, and methods based on variation in ancestry in an admixed population, used in anthropological and statistical genetics. In each case, we show mathematically that information on within-group genetic and phenotypic information in the aggregate cannot separate among-group differences into genetic and environmental components, and we provide simulation results that support our claims. We discuss these results in terms of the long-running debate on this topic.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566176">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566176" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566176">
        <p class="paperTitle">Milo2.0 unlocks population genetic analyses of cell state abundance using a count-based mixed model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566176" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566176" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kluzer, A.; Marioni, J. C.; Morgan, M. D.</p>
        <p class="info">Score: 21.0, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566176' target='https://doi.org/10.1101/2023.11.08.566176'> 10.1101/2023.11.08.566176</a></p>
        <p class="abstract">Cell type proportions vary between individuals and are heritable, as demonstrated by statistical genetic analysis of flow cytometry data [1,2]. Higher-resolution cell states can be identified by single-cell RNA-sequencing, the scalability of which now makes it applicable to population-scale cohorts. However, the integration of statistical genetic analysis of cell states using cohort-scale single-cell data requires appropriate algorithms to account for and model the genetic relationships and complex batch-processing inherent to these studies. We describe Milo2.0, which enables the discovery of cell state quantitative trait loci (csQTL), scaling to millions of cells across hundreds of individuals. We identify &gt; 500 csQTLs across peripheral blood immune states and investigate their relationship with the genetic regulation of gene expression. Moreover, we colocalise immune csQTLs with human traits and identify links between immune regulators, cell state abundance and immune-mediated disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565769">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565769" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565769">
        <p class="paperTitle">Genome assembly of the dyeing poison frog provides insights into the dynamics of transposable element and genome-size evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565769" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565769" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dittrich, C.; Hoelzl, F.; Smith, S.; Fouilloux, C. A.; Parker, D.; O&#39;Connell, L. A.; Knowles, L. S.; Hughes, M.; Fewings, A.; Morgan, R.; Rojas, B.; Comeault, A. A.</p>
        <p class="info">Score: 21.4, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565769' target='https://doi.org/10.1101/2023.11.06.565769'> 10.1101/2023.11.06.565769</a></p>
        <p class="abstract">Genome size varies greatly across the tree of life and transposable elements are an important contributor to this variation. Among vertebrates, amphibians display the greatest variation in genome size, making them ideal models to explore the causes and consequences of genome size variation. However, high-quality genome assemblies for amphibians have, until recently, been rare. Here, we generate a high-quality genome assembly for the dyeing poison frog, Dendrobates tinctorius. We compare this assembly to publicly-available frog genomes and find evidence for both large-scale conserved synteny and widespread rearrangements between frog lineages. Comparing conserved orthologs annotated in these genomes revealed a strong correlation between genome size and gene size. To explore the cause of gene-size variation, we quantified the location of transposable elements relative to gene features and find that the accumulation of transposable elements in introns has played an important role in the evolution of gene size in D. tinctorius, while estimates of insertion times suggest that many insertion events are recent and species-specific. Finally, we show that the diversity and abundance of transposable elements in poison frog genomes can complicate genotyping efforts that rely on repetitive elements as sequence anchors. Our results show that transposable elements have clearly played an important role in the evolution of large genome size in D. tinctorius. Future studies are needed to fully understand the dynamics of transposable element evolution and to optimise primer or bait design for cost-effective population-level genotyping in species with large, repetitive genomes.

SignificanceAmphibians display more variation in genome size than any other vertebrate lineage. Complexities associated with large genomes frequently hamper genome assembly and population genetic studies. Here we use long-read HiFi sequences to generate a high-quality 6.3 Gbp genome assembly of the poison frog Dendrobates tinctorius. We use this genome and leverage comparative genomics and de novo annotations to quantify aspects of genome evolution driven by repetitive transposable genetic elements. Our results provide support for the dynamic role that transposable elements play in driving the evolution of &#34;genomic gigantism&#34; in amphibians. We also show how transposable elements can be leveraged for cost-efficient population genetic studies using limited input material.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566468">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566468" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566468">
        <p class="paperTitle">Genetic Discovery Enabled by A Large Language Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566468" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566468" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tu, T.; Fang, Z.; Cheng, Z.; Spasic, S.; Palepu, A.; Stankovic, K.; Natarajan, V.; Peltz, G.</p>
        <p class="info">Score: 15.2, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566468' target='https://doi.org/10.1101/2023.11.09.566468'> 10.1101/2023.11.09.566468</a></p>
        <p class="abstract">Artificial intelligence (AI) has been used in many areas of medicine, and recently large language models (LLMs) have shown potential utility for clinical applications. However, since we do not know if the use of LLMs can accelerate the pace of genetic discovery, we used data generated from mouse genetic models to investigate this possibility. We examined whether a recently developed specialized LLM (Med-PaLM 2) could analyze sets of candidate genes generated from analysis of murine models of biomedical traits. In response to free-text input, Med-PaLM 2 correctly identified the murine genes that contained experimentally verified causative genetic factors for six biomedical traits, which included susceptibility to diabetes and cataracts. Med-PaLM 2 was also able to analyze a list of genes with high impact alleles, which were identified by comparative analysis of murine genomic sequence data, and it identified a causative murine genetic factor for spontaneous hearing loss. Based upon this Med-PaLM 2 finding, a novel bigenic model for susceptibility to spontaneous hearing loss was developed. These results demonstrate Med-PaLM 2 can analyze gene-phenotype relationships and generate novel hypotheses, which can facilitate genetic discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566091">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566091" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566091">
        <p class="paperTitle">Distinct and interdependent functions of three RING proteins regulate recombination during mammalian meiosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566091" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566091" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ito, M.; Yun, Y.; Kulkarni, D. S.; Sandhu, S.; Nunez, B.; Hu, L.; Lee, K.; Lim, N.; Hirota, R.; Prendergast, R.; Huang, C.; Huang, I.; Hunter, N.</p>
        <p class="info">Score: 12.6, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566091' target='https://doi.org/10.1101/2023.11.07.566091'> 10.1101/2023.11.07.566091</a></p>
        <p class="abstract">Crossing-over between homologous chromosomes is essential for accurate segregation during meiosis. In mammals, factors required for crossing over include RNF212 and HEI10, RING-finger domain proteins implicated in modification by SUMO and ubiquitin, respectively. Here we show that a third RING protein, RNF212B, is essential for crossing over in mouse and characterize its relationships with RNF212 and HEI10. Mutant analysis indicates that, analogous to RNF212, RNF212B regulates recombination occurring in the context of synapsed chromosomes by stabilizing pro-crossover factors, regulating the progression of recombination, and enabling the differentiation and maturation of crossover-specific recombination complexes. Also, like RNF212, RNF212B localizes between synapsed chromosomes, initially as numerous foci along synaptonemal complexes (SCs) before undergoing HEI10-dependent redistribution to accumulate at prospective crossover sites. Despite these similarities, accumulation of RNF212B at nascent crossover sites is much greater than RNF212 and occurs earlier, coincident with the appearance of HEI10 foci, implicating concerted action of RNF212B and HEI10 in the differentiation of crossover sites. RNF212B localization also shows greater dependence on DNA double-strand breaks and SC formation than RNF212. Together, our analysis delineates three distinct but interdependent RING proteins that regulate meiotic recombination by integrating signals from DSBs, synapsis, the cell-cycle, and developing crossover sites.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.04.565589">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.04.565589" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.04.565589">
        <p class="paperTitle">Whole genome sequencing analysis identifies rare, large-effect non-coding variants and regions associated with circulating protein levels</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.04.565589" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.04.565589" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hawkes, G.; Chundru, K.; Jackson, L.; Patel, K. A.; Murray, A.; Wood, A. R.; Wright, C. R.; Weedon, M. N.; Frayling, T. M.; Beaumont, R. N.</p>
        <p class="info">Score: 30.1, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.04.565589' target='https://doi.org/10.1101/2023.11.04.565589'> 10.1101/2023.11.04.565589</a></p>
        <p class="abstract">The role of non-coding rare variation in common phenotypes is largely unknown, due to a lack of whole-genome sequence data, and the difficulty of categorising non-coding variants into biologically meaningful regulatory units. To begin addressing these challenges, we performed a cis association analysis using whole-genome sequence data, consisting of 391 million variants and 1,450 circulating protein levels in [~]20,000 UK Biobank participants. We identified 777 independent rare non-coding single variants associated with circulating protein levels (P&lt;1x10-9), after conditioning on protein-coding and common associated variants. Rare non-coding aggregate testing identified 108 conditionally independent regulatory regions. Unlike protein-coding variation, rare non-coding genetic variation was almost as likely to increase as decrease protein levels. The regions we identified overlapped predicted tissue-specific enhancers more than promoters, suggesting they represent tissue-specific regulatory regions. Our results have important implications for the identification, and role, of rare non-coding variation associated with common human phenotypes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566300">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566300" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566300">
        <p class="paperTitle">Benchmarking kinship estimation tools for ancient genomes using pedigree simulations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566300" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566300" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aktürk, S.; Mapelli, I.; Güler, M. N.; Gürün, K.; Katırcıoglu, B.; Vural, K. B.; Saglıcan, E.; Cetin, M.; Yaka, R.; Sürer, E.; Atag, G.; Cokoglu, S. S.; Sevkar, A.; Altınısık, N. E.; Koptekin, D.; Somel, M.</p>
        <p class="info">Score: 11.8, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566300' target='https://doi.org/10.1101/2023.11.08.566300'> 10.1101/2023.11.08.566300</a></p>
        <p class="abstract">There is growing interest in uncovering genetic kinship patterns in past societies using low-coverage paleogenomes. Here, we benchmark four tools for kinship estimation with such data: lcMLkin, NgsRelate, KIN, and READ, which differ in their input, IBD-estimation methods and statistical approaches. We used pedigree and ancient genome sequence simulations to evaluate these tools when only a limited number (1K to 50K) of shared SNPs (with minor allele frequency [&amp;ge;]0.01) are available. The performance of all four tools was comparable using [&amp;ge;]20K SNPs. We found that first-degree related pairs can be accurately classified even with 1K SNPs, with 85% F1 scores using READ and 96% using NgsRelate or lcMLkin. Distinguishing third-degree relatives from unrelated pairs or second-degree relatives was also possible with high accuracy (F1 &gt;90%) with 5K SNPs using NgsRelate and lcMLkin, while READ and KIN showed lower success (69% and 79%, respectively). Meanwhile, noise in population allele frequencies and inbreeding (first cousin mating) led to deviations in kinship coefficients, with different sensitivities across tools. We conclude that using multiple tools in parallel might be an effective approach to achieve robust estimates on ultra-low coverage genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565577">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565577" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565577">
        <p class="paperTitle">Increasing equity in science requires better ethics training: a course by trainees, for trainees</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565577" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565577" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Patel, R. A.; Ungar, R. A.; Pyke, A. L.; Adimoelja, A.; Chakraborty, M.; Cotter, D. J.; Freund, M.; Goddard, P.; Gomez-Stafford, J.; Greenwald, E.; Higgs, E.; Hunter, N.; MacKenzie, T. M. G.; Narain, A.; Martschenko, D. O.</p>
        <p class="info">Score: 32.0, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565577' target='https://doi.org/10.1101/2023.11.03.565577'> 10.1101/2023.11.03.565577</a></p>
        <p class="abstract">Despite the profound impacts of scientific research, few scientists have received the necessary training to productively discuss the ethical and societal implications of their work. To address this critical gap, we - a group of predominantly human genetics trainees - developed a course on genetics, ethics, and society. We intend for this course to serve as a template for other institutions and scientific disciplines. Our curriculum positions human genetics within its historical and societal context and encourages students to evaluate how societal norms and structures impact the conduct of scientific research. We demonstrate the utility of this course via surveys of enrolled students and provide resources and strategies for others hoping to teach a similar course. We conclude by arguing that if we are to work towards rectifying the inequities and injustices produced by our field, we must first learn to view our own research as impacting and being impacted by society.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564036">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564036" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564036">
        <p class="paperTitle">Assessing the impact of 20th century internal migrations on the genetic structure of Estonia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564036" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564036" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kuznetsov, I. A.; Estonian Biobank Research Team,  ; Metspalu, M.; Vainik, U.; Pagani, L.; Montinaro, F.; Pankratov, V.</p>
        <p class="info">Score: 30.3, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564036' target='https://doi.org/10.1101/2023.10.25.564036'> 10.1101/2023.10.25.564036</a></p>
        <p class="abstract">Spatial genetic structure observed in many human populations is in large part attributed to past demographic events and isolation by distance. However, how intensifying migration affects this structure remains understudied. Here we harness a sample of more than 180 thousand individuals to explore the genetic correlates and consequences of contemporary migrations in Estonia. While we show that migration smoothens the genome-wide genetic structure, it intensifies inter-regional differences in polygenic scores (PGS) for certain traits, derived both from population as well as within-sibship studies. The strongest effect is observed for educational attainment which is consistent with previous observations in the UK and suggests this to be a general pattern. We explore those regional differences in PGS in terms of the driving forces behind them and from a temporal perspective, and suggest urbanisation as a major driver for this pattern in Estonia from at least the first half of the 20th century.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
