<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases
Authors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.
Score: 97.1, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569447
Transposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases
Authors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.
Score: 97.1, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569447
Transposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-03T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-12-03T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases
Authors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.
Score: 97.1, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569447
Transposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases\nAuthors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.\nScore: 97.1, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569447\nTransposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6.",
  "keywords": [
    
  ],
  "articleBody": " Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases\nAuthors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.\nScore: 97.1, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569447\nTransposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6. We set out to test the hypothesis that TnpB nucleases may have also been repurposed for novel, unexpected functions other than CRIS-PR-Cas. Here, using phylogenetics, structural predictions, comparative genomics, and functional assays, we uncover multiple instances of programmable transcription factors that we name TnpB-like nuclease-dead repressors (TldR). These proteins employ naturally occurring guide RNAs to specifically target conserved promoter regions of the genome, leading to potent gene repression in a mechanism akin to CRISPRi technologies invented by humans7. Focusing on a TldR clade found broadly in Enterobacteriaceae, we discover that bacteriophages exploit the combined action of TldR and an adjacently encoded phage gene to alter the expression and composition of the host flagellar assembly, a transformation with the potential to impact motility8, phage susceptibility9, and host immunity10. Col-lectively, this work showcases the diverse molecular innovations that were enabled through repeated exaptation of genes encoded by transposable elements, and reveals that RNA-guided transcription factors emerged long before the development of dCas9-based editors.\nRegulators of health and lifespan extension in genetically diverse mice on dietary restriction\nAuthors: Di Francesco, A.; Deighan, A. G.; Litichevskiy, L.; Chen, Z.; Luciano, A.; Robinson, L.; Garland, G.; Donato, H.; Schott, W.; Wright, K. M.; Raj, A.; Prateek, G. V.; Mullis, M.; Hill, W.; Zediel, M.; Peters, L.; Harding, F.; Botstein, D.; Korstanje, R.; Thaiss, C.; Freund, A.; Churchill, G. A.\nScore: 42.9, Published: 2023-11-29 DOI: 10.1101/2023.11.28.568901\nCaloric restriction (CR) delays aging and extends healthy lifespan in multiple species. Alternative forms of dietary restriction (DR) such as intermittent fasting (IF) have drawn significant interest as a more sustainable regimen, but the landscape of longevity-promoting dietary interventions remains largely unexplored. Identifying the most robust, efficacious, and experimentally tractable modes of DR is key to better understanding and implementing effective longevity interventions for human healthspan. To that end, we have performed an extensive assessment of DR interventions, investigating the effects of graded levels of CR (20% and 40%) and IF (1 day and 2 days of fasting per week) on the health and survival of 960 genetically diverse female mice. All interventions extended lifespan, although only CR significantly reduced the mortality doubling time. Notably, IF did not extend lifespan in mice with high pre-intervention bodyweight. We carried out extensive phenotyping to determine the health effects of long-term DR and to better understand the mechanisms driving within-diet heterogeneity in lifespan. The top within-diet predictor of lifespan was the ability of mice to maintain bodyweight through periods of handling, an indicator of stress resilience. Additional predictors of long lifespan include specific changes in immune cells, red blood cell distribution width (RDW), and retention of adiposity in late life. We found that lifespan is heritable (h2 = 0.24), and that genetic background has a larger influence on lifespan than dietary interventions. We identified a significant association for lifespan and RDW on chromosome 18 that explained 4.3% of the diet-adjusted variation in lifespan. Diet-induced changes on metabolic traits, although beneficial, were relatively poor predictors of lifespan, arguing against the long-standing notion that DR works by counteracting the negative effects of obesity. These findings indicate that improving health and extending lifespan are not synonymous and that metabolic parameters may be inappropriate endpoints for evaluating aging interventions in preclinical models and clinical trials.\nMulti-Step Control of Homologous Recombination by Mec1/ATR Ensures Robust Suppression of Gross Chromosomal Rearrangements\nAuthors: Xie, B.; Sanford, E. J.; Hung, S.-H.; Wagner, M. M.; Heyer, W.-D.; Smolka, M. B.\nScore: 17.0, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568146\nThe Mec1/ATR kinase is crucial for genome stability, yet the mechanism by which it prevents gross chromosomal rearrangements (GCRs) remains unknown. Here we find that in cells with deficient Mec1 signaling, GCRs accumulate due to the deregulation of multiple steps in homologous recombination (HR). Mec1 primarily suppresses GCRs through its role in activating the canonical checkpoint kinase Rad53, which ensures the proper control of DNA end resection. Upon loss of Rad53 signaling and resection control, Mec1 becomes hyperactivated and triggers a salvage pathway in which the Sgs1 helicase is recruited to sites of DNA lesions via the 911-Dpb11 scaffolds to favor heteroduplex rejection and limit HR-driven GCR accumulation. Fusing an ssDNA recognition domain to Sgs1 bypasses the requirement of Mec1 signaling for GCR suppression and nearly eliminates D-loop formation, thus preventing non-allelic recombination events. We propose that Mec1 regulates multiple steps of HR to prevent GCRs while ensuring balanced HR usage when needed for promoting tolerance to replication stress.\nRepeat mediated excision of gene drive elements for restoring wild-type populations\nAuthors: Chennuri, P. R.; Zapletal, J.; Monfardini, R. D.; Ndeffo-Mbah, M. L.; Adelman, Z.; Myles, K. M. M.\nScore: 4.6, Published: 2023-11-26 DOI: 10.1101/2023.11.23.568397\nWe demonstrate here that single strand annealing (SSA) repair can be co-opted for the precise autocatalytic excision of a drive element. Although SSA is not the predominant form of DNA repair in eukaryotic organisms, we increased the likelihood of its use by engineering direct repeats at sites flanking the drive allele, and then introducing a double-strand DNA break (DSB) at a second endonuclease target site encoded within the drive allele. We have termed this technology Repeat Mediated Excision of a Drive Element (ReMEDE). Incorporation of ReMEDE into the previously described mutagenic chain reaction (MCR) gene drive, targeting the yellow gene of Drosophila melanogaster, replaced drive alleles with wild-type alleles demonstrating proof-of-principle. Although the ReMEDE system requires further research and development, the technology has a number of attractive features as a gene drive mitigation strategy, chief among these the potential to restore a wild-type population without releasing additional transgenic organisms or large-scale environmental engineering efforts.\nA genome-wide association study of neonatal metabolites\nAuthors: He, Q.; Liu, H.; Lu, L.; Zhang, Q.; Wang, Q.; Wang, B.; Wu, X.; Guan, L.; Mao, J.; Xue, Y.; Zhang, C.; He, Y.; Peng, X.; Peng, H.; Zhao, K.; Li, H.; Jin, X.; Zhao, L.; Zhang, J.; Wang, T.\nScore: 4.0, Published: 2023-11-27 DOI: 10.1101/2023.11.25.568687\nThe hereditary component significantly influences the concentration of metabolites in adults. Nevertheless, the precise influence of genetic factors on neonatal metabolites remains uncertain. To bridge this gap, we employed genotype imputation techniques on large-scale low-pass genome data obtained from non-invasive prenatal testing. Subsequently, we conducted association studies on a total of 75 metabolic components in neonates. The study identified a total 17 previous reported associations and 13 novel discovered associations between single nucleotide polymorphisms and metabolic components. These associations were initially found in the discovery cohort (8,744 participants) and subsequently confirmed in a replication cohort (19,041 participants). The average heritability of metabolic components was calculated to be 76.2%, with a range of 69-78.8%. The aforementioned findings offer valuable insights pertaining to the genetic architecture of neonatal metabolism. In BriefLarge-scale genomes of maternal non-invasive prenatal testing provide insights into the genetic contribution to neonatal metabolism. HighlightsGWAS of 27,785 low-pass genomes revealed 13 novel associations of neonatal metabolic components. Estimated an average of 76.2% heritability of neonatal metabolic components and showed the individual concentration can be accurately predicted from polygenic risk scores. A total of 17 established relationships have been observed, providing evidence that maternal genomes can be utilized in neonatal metabolite GWAS.\nAccounting for nuclear and mito genome in dairy cattle breeding - a simulation study\nAuthors: Mafra Fortuna, G.; Zumbach, B. J.; Johnsson, M.; Pocrnic, I.; Gorjanc, G.\nScore: 5.9, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567907\nMitochondria play a significant role in numerous cellular processes through proteins encoded by both nuclear genome (nDNA) and mito genome (mDNA). While the variation in nDNA is influenced by mutations and recombination of parental genomes, the variation in mDNA is solely driven by mutations. In addition, mDNA is inherited in a haploid form, from the dam. Cattle populations show significant variation in mDNA between and within breeds. Past research suggests that variation in mDNA accounts for 1-5% of the phenotypic variation in dairy traits. Here we simulated a dairy cattle breeding program to assess the impact of accounting for mDNA variation in pedigree-based and genome-based genetic evaluations on the accuracy of estimated breeding values for mDNA and nDNA components. We also examined the impact of alternative definitions of breeding values on genetic gain, including nDNA and mDNA components that both impact phenotype expression, but mDNA is inherited only maternally. We found that accounting for mDNA variation increased accuracy between +0.01 and +0.05 for different categories of animals, especially for young bulls (+0.05) and females without genotype data (between +0.01 and +0.03). Different scenarios of modelling and breeding value definition impacted genetic gain. The standard approach of ignoring mDNA variation achieved competitive genetic gain. Modelling, but not selecting on mDNA expectedly reduced genetic gain, while optimal use of mDNA variation recovered the genetic gain.\nMycobacterium smegmatis NucS-promoted DNA Mismatch Repair involves limited resection by a 5'-3' exonuclease and is independent of homologous recombination and NHEJ\nAuthors: Rivera-Flores, I. V.; Wang, E. X.; Murphy, K. C.\nScore: 2.2, Published: 2023-11-26 DOI: 10.1101/2023.11.26.568737\nThe MutSL mismatch repair system in bacteria and eukaryotes correct mismatches made at the replication fork to maintain genome stability. Despite lacking the mutSL genes, most actinobacterial species, including Mycobacterium tuberculosis, exhibit spontaneous mutation rates similar to MutSL-bearing bacteria, suggesting the existence of an alternative mismatch repair (MMR) system. It has been shown previously that the EndoMS/NucS protein from Actinobacterium Corynebacterium glutamicum is a mismatch-specific endonuclease that recognizes and creates dsDNA breaks in vitro. Additionally, the structure of the NucS protein from the archaeal species P. abyssi reveals high similarity to the structures of restriction endonucleases, leading to a model that NucS mismatch repair involves a dsDNA break in the repair process. In this report, a genetic analysis shows that an M. smegmatis 11nucS strain could be complemented by expression of wild type NucS protein, but not by one deleted of its last five amino acids, a region predicted to be critical for binding to the {beta}-clamp at the replication fork. Oligo-recombineering promoted by Che9 phage RecT annealase was then leveraged to directly deliver defined mismatches to a defective hygromycin resistance gene on the M. smegmatis chromosome. We find that NucS recognizes and repairs G-G, G-T, and T-T mismatches in vivo, consistent with the recognition of these same mismatches in C. glutamicum in vitro, as well as mutation accumulation studies done in M. smegmatis. Three residues in M. smegmatis NucS, predicted to be catalytic residues based on the restriction enzyme-like structure of P. abyssi NucS, were mutated to alanine; all three mutants were defective for mismatch repair. Finally, an assay the employs one oligo that promotes the generation of two mismatches in close proximity on the chromosome shows that a NucS-promoted cut in dsDNA is processed by a 5 - 3 exonuclease, but is independent of both RecA, RadA, and NHEJ functions Ku and LigD, suggesting that the mechanism of DNA mismatch repair does not involve a homologous recombination event or non-homologous end-joining.\nWhole genome association testing in 333,100 individuals across three biobanks identifies rare non-coding single variant and genomic aggregate associations with height\nAuthors: Hawkes, G.; Beaumont, R. N.; Li, Z.; Mandla, R.; Li, X.; Albert, C. M.; Arnett, D. K.; Ashley-Koch, A. E.; Ashrani, A. A.; Barnes, K. C.; Boerwinkle, E.; Brody, J. A.; Carson, A. P.; Chami, N.; Chen, Y.-D. I.; Chung, M. K.; Curran, J. E.; Darbar, D.; Ellinor, P. T.; Fornage, M.; Gordeuk, V. R.; Guo, X.; He, J.; Hwu, C.-M.; Kalyani, R. R.; Kaplan, R.; Kardia, S. L. R.; Kooperberg, C.; Loos, R. J. F.; Lubitz, S. A.; Minster, R. L.; Mitchell, B. D.; Murabito, J. M.; Palmer, N. D.; Psaty, B. M.; Redline, S.; Shoemaker, M. B.; Silverman, E. K.; Telen, M. J.; Weiss, S. T.; Yanek, L. R.; Zhou, H.; NH\nScore: 29.6, Published: 2023-11-20 DOI: 10.1101/2023.11.19.566520\nThe role of rare non-coding variation in complex human phenotypes is still largely unknown. To elucidate the impact of rare variants in regulatory elements, we performed a whole-genome sequencing association analysis for height using 333,100 individuals from three datasets: UK Biobank (N=200,003), TOPMed (N=87,652) and All of Us (N=45,445). We performed rare (\u003c0.1% minor-allele-frequency) single-variant and aggregate testing of non-coding variants in regulatory regions based on proximal, intergenic and deep-intronic annotation. We observed 29 independent variants associated with height at P \u003c 6 x 10-10 after conditioning on previously reported variants, with effect sizes ranging from -7cm to +4.7cm. We also identified and replicated non-coding aggregate-based associations proximal to HMGA1 containing variants associated with a 5cm taller height and of highly-conserved variants in MIR497HG on chromosome 17. We have developed a novel approach for identifying non-coding rare variants in regulatory regions with large effects from whole-genome sequencing data associated with complex traits.\nIdentifying genetic variants that influence the abundance of cell states in single-cell data\nAuthors: Rumker, L.; Sakaue, S.; Reshef, Y.; Kang, J. B.; Yazar, S.; Alquicira-Hernandez, J.; Valencia, C.; Lagattuta, K. A.; Mah-Som, A.; Nathan, A.; Powell, J. E.; Loh, P.-R.; Raychaudhuri, S.\nScore: 22.9, Published: 2023-11-15 DOI: 10.1101/2023.11.13.566919\nIntroductory ParagraphTo understand genetic mechanisms driving disease, it is essential but difficult to map how risk alleles affect the composition of cells present in the body. Single-cell profiling quantifies granular information about tissues, but variant-associated cell states may reflect diverse combinations of the profiled cell features that are challenging to predefine. We introduce GeNA (Genotype-Neighborhood Associations), a statistical tool to identify cell state abundance quantitative trait loci (csaQTLs) in high-dimensional single-cell datasets. Instead of testing associations to predefined cell states, GeNA flexibly identifies the cell states whose abundance is most associated with genetic variants. In a genome-wide survey of scRNA-seq peripheral blood profiling from 969 individuals,1 GeNA identifies five independent loci associated with shifts in the relative abundance of immune cell states. For example, rs3003-T (p=1.96x10-11) associates with increased abundance of NK cells expressing TNF- response programs. This csaQTL colocalizes with increased risk for psoriasis, an autoimmune disease that responds to anti-TNF treatments. Flexibly characterizing csaQTLs for granular cell states may help illuminate how genetic background alters cellular composition to confer disease risk.\nThe C. elegans Myc-family of transcription factors coordinate a dynamic adaptive response to dietary restriction.\nAuthors: Cornwell, A. B.; Zhang, Y.; Thondamal, M.; Johnson, D. W.; Thakar, J.; Samuelson, A. V.\nScore: 2.1, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568222\nDietary restriction (DR), the process of decreasing overall food consumption over an extended period of time, has been shown to increase longevity across evolutionarily diverse species and delay the onset of age-associated diseases in humans. In Caenorhabditis elegans, the Myc-family transcription factors (TFs) MXL-2 (Mlx) and MML-1 (MondoA/ChREBP), which function as obligate heterodimers, and PHA-4 (orthologous to forkhead box transcription factor A) are both necessary for the full physiological benefits of DR. However, the adaptive transcriptional response to DR and the role of MML-1::MXL-2 and PHA-4 remains elusive. We identified the transcriptional signature of C. elegans DR, using the eat-2 genetic model, and demonstrate broad changes in metabolic gene expression in eat-2 DR animals, which requires both mxl-2 and pha-4. While the requirement for these factors in DR gene expression overlaps, we found many of the DR genes exhibit an opposing change in relative gene expression in eat-2;mxl-2 animals compared to wild-type, which was not observed in eat-2 animals with pha-4 loss. We further show functional deficiencies of the mxl-2 loss in DR outside of lifespan, as eat-2;mxl-2 animals exhibit substantially smaller brood sizes and lay a proportion of dead eggs, indicating that MML-1::MXL-2 has a role in maintaining the balance between resource allocation to the soma and to reproduction under conditions of chronic food scarcity. While eat-2 animals do not show a significantly different metabolic rate compared to wild-type, we also find that loss of mxl-2 in DR does not affect the rate of oxygen consumption in young animals. The gene expression signature of eat-2 mutant animals is consistent with optimization of energy utilization and resource allocation, rather than induction of canonical gene expression changes associated with acute metabolic stress -such as induction of autophagy after TORC1 inhibition. Consistently, eat-2 animals are not substantially resistant to stress, providing further support to the idea that chronic DR may benefit healthspan and lifespan through efficient use of limited resources rather than broad upregulation of stress responses, and also indicates that MML-1::MXL-2 and PHA-4 may have different roles in promotion of benefits in response to different pro-longevity stimuli.\n",
  "wordCount" : "2773",
  "inLanguage": "en",
  "datePublished": "2023-12-03T10:38:01Z",
  "dateModified": "2023-12-03T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 3, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569447">
        <p class="paperTitle">Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.</p>
        <p class="info">Score: 97.1, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569447' target='https://doi.org/10.1101/2023.11.30.569447'> 10.1101/2023.11.30.569447</a></p>
        <p class="abstract">Transposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6. We set out to test the hypothesis that TnpB nucleases may have also been repurposed for novel, unexpected functions other than CRIS-PR-Cas. Here, using phylogenetics, structural predictions, comparative genomics, and functional assays, we uncover multiple instances of programmable transcription factors that we name TnpB-like nuclease-dead repressors (TldR). These proteins employ naturally occurring guide RNAs to specifically target conserved promoter regions of the genome, leading to potent gene repression in a mechanism akin to CRISPRi technologies invented by humans7. Focusing on a TldR clade found broadly in Enterobacteriaceae, we discover that bacteriophages exploit the combined action of TldR and an adjacently encoded phage gene to alter the expression and composition of the host flagellar assembly, a transformation with the potential to impact motility8, phage susceptibility9, and host immunity10. Col-lectively, this work showcases the diverse molecular innovations that were enabled through repeated exaptation of genes encoded by transposable elements, and reveals that RNA-guided transcription factors emerged long before the development of dCas9-based editors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568901">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568901" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568901">
        <p class="paperTitle">Regulators of health and lifespan extension in genetically diverse mice on dietary restriction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568901" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568901" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Di Francesco, A.; Deighan, A. G.; Litichevskiy, L.; Chen, Z.; Luciano, A.; Robinson, L.; Garland, G.; Donato, H.; Schott, W.; Wright, K. M.; Raj, A.; Prateek, G. V.; Mullis, M.; Hill, W.; Zediel, M.; Peters, L.; Harding, F.; Botstein, D.; Korstanje, R.; Thaiss, C.; Freund, A.; Churchill, G. A.</p>
        <p class="info">Score: 42.9, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568901' target='https://doi.org/10.1101/2023.11.28.568901'> 10.1101/2023.11.28.568901</a></p>
        <p class="abstract">Caloric restriction (CR) delays aging and extends healthy lifespan in multiple species. Alternative forms of dietary restriction (DR) such as intermittent fasting (IF) have drawn significant interest as a more sustainable regimen, but the landscape of longevity-promoting dietary interventions remains largely unexplored. Identifying the most robust, efficacious, and experimentally tractable modes of DR is key to better understanding and implementing effective longevity interventions for human healthspan. To that end, we have performed an extensive assessment of DR interventions, investigating the effects of graded levels of CR (20% and 40%) and IF (1 day and 2 days of fasting per week) on the health and survival of 960 genetically diverse female mice. All interventions extended lifespan, although only CR significantly reduced the mortality doubling time. Notably, IF did not extend lifespan in mice with high pre-intervention bodyweight. We carried out extensive phenotyping to determine the health effects of long-term DR and to better understand the mechanisms driving within-diet heterogeneity in lifespan. The top within-diet predictor of lifespan was the ability of mice to maintain bodyweight through periods of handling, an indicator of stress resilience. Additional predictors of long lifespan include specific changes in immune cells, red blood cell distribution width (RDW), and retention of adiposity in late life. We found that lifespan is heritable (h2 = 0.24), and that genetic background has a larger influence on lifespan than dietary interventions. We identified a significant association for lifespan and RDW on chromosome 18 that explained 4.3% of the diet-adjusted variation in lifespan. Diet-induced changes on metabolic traits, although beneficial, were relatively poor predictors of lifespan, arguing against the long-standing notion that DR works by counteracting the negative effects of obesity. These findings indicate that improving health and extending lifespan are not synonymous and that metabolic parameters may be inappropriate endpoints for evaluating aging interventions in preclinical models and clinical trials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568146">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568146" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568146">
        <p class="paperTitle">Multi-Step Control of Homologous Recombination by Mec1/ATR Ensures Robust Suppression of Gross Chromosomal Rearrangements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568146" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568146" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xie, B.; Sanford, E. J.; Hung, S.-H.; Wagner, M. M.; Heyer, W.-D.; Smolka, M. B.</p>
        <p class="info">Score: 17.0, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568146' target='https://doi.org/10.1101/2023.11.21.568146'> 10.1101/2023.11.21.568146</a></p>
        <p class="abstract">The Mec1/ATR kinase is crucial for genome stability, yet the mechanism by which it prevents gross chromosomal rearrangements (GCRs) remains unknown. Here we find that in cells with deficient Mec1 signaling, GCRs accumulate due to the deregulation of multiple steps in homologous recombination (HR). Mec1 primarily suppresses GCRs through its role in activating the canonical checkpoint kinase Rad53, which ensures the proper control of DNA end resection. Upon loss of Rad53 signaling and resection control, Mec1 becomes hyperactivated and triggers a salvage pathway in which the Sgs1 helicase is recruited to sites of DNA lesions via the 911-Dpb11 scaffolds to favor heteroduplex rejection and limit HR-driven GCR accumulation. Fusing an ssDNA recognition domain to Sgs1 bypasses the requirement of Mec1 signaling for GCR suppression and nearly eliminates D-loop formation, thus preventing non-allelic recombination events. We propose that Mec1 regulates multiple steps of HR to prevent GCRs while ensuring balanced HR usage when needed for promoting tolerance to replication stress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568397">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568397" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568397">
        <p class="paperTitle">Repeat mediated excision of gene drive elements for restoring wild-type populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568397" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568397" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chennuri, P. R.; Zapletal, J.; Monfardini, R. D.; Ndeffo-Mbah, M. L.; Adelman, Z.; Myles, K. M. M.</p>
        <p class="info">Score: 4.6, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568397' target='https://doi.org/10.1101/2023.11.23.568397'> 10.1101/2023.11.23.568397</a></p>
        <p class="abstract">We demonstrate here that single strand annealing (SSA) repair can be co-opted for the precise autocatalytic excision of a drive element. Although SSA is not the predominant form of DNA repair in eukaryotic organisms, we increased the likelihood of its use by engineering direct repeats at sites flanking the drive allele, and then introducing a double-strand DNA break (DSB) at a second endonuclease target site encoded within the drive allele. We have termed this technology Repeat Mediated Excision of a Drive Element (ReMEDE). Incorporation of ReMEDE into the previously described mutagenic chain reaction (MCR) gene drive, targeting the yellow gene of Drosophila melanogaster, replaced drive alleles with wild-type alleles demonstrating proof-of-principle. Although the ReMEDE system requires further research and development, the technology has a number of attractive features as a gene drive mitigation strategy, chief among these the potential to restore a wild-type population without releasing additional transgenic organisms or large-scale environmental engineering efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.568687">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.568687" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.568687">
        <p class="paperTitle">A genome-wide association study of neonatal metabolites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.568687" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.568687" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, Q.; Liu, H.; Lu, L.; Zhang, Q.; Wang, Q.; Wang, B.; Wu, X.; Guan, L.; Mao, J.; Xue, Y.; Zhang, C.; He, Y.; Peng, X.; Peng, H.; Zhao, K.; Li, H.; Jin, X.; Zhao, L.; Zhang, J.; Wang, T.</p>
        <p class="info">Score: 4.0, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.568687' target='https://doi.org/10.1101/2023.11.25.568687'> 10.1101/2023.11.25.568687</a></p>
        <p class="abstract">The hereditary component significantly influences the concentration of metabolites in adults. Nevertheless, the precise influence of genetic factors on neonatal metabolites remains uncertain. To bridge this gap, we employed genotype imputation techniques on large-scale low-pass genome data obtained from non-invasive prenatal testing. Subsequently, we conducted association studies on a total of 75 metabolic components in neonates. The study identified a total 17 previous reported associations and 13 novel discovered associations between single nucleotide polymorphisms and metabolic components. These associations were initially found in the discovery cohort (8,744 participants) and subsequently confirmed in a replication cohort (19,041 participants). The average heritability of metabolic components was calculated to be 76.2%, with a range of 69-78.8%. The aforementioned findings offer valuable insights pertaining to the genetic architecture of neonatal metabolism.

In BriefLarge-scale genomes of maternal non-invasive prenatal testing provide insights into the genetic contribution to neonatal metabolism.

HighlightsGWAS of 27,785 low-pass genomes revealed 13 novel associations of neonatal metabolic components.

Estimated an average of 76.2% heritability of neonatal metabolic components and showed the individual concentration can be accurately predicted from polygenic risk scores.

A total of 17 established relationships have been observed, providing evidence that maternal genomes can be utilized in neonatal metabolite GWAS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567907">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567907" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567907">
        <p class="paperTitle">Accounting for nuclear and mito genome in dairy cattle breeding - a simulation study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567907" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567907" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mafra Fortuna, G.; Zumbach, B. J.; Johnsson, M.; Pocrnic, I.; Gorjanc, G.</p>
        <p class="info">Score: 5.9, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567907' target='https://doi.org/10.1101/2023.11.20.567907'> 10.1101/2023.11.20.567907</a></p>
        <p class="abstract">Mitochondria play a significant role in numerous cellular processes through proteins encoded by both nuclear genome (nDNA) and mito genome (mDNA). While the variation in nDNA is influenced by mutations and recombination of parental genomes, the variation in mDNA is solely driven by mutations. In addition, mDNA is inherited in a haploid form, from the dam. Cattle populations show significant variation in mDNA between and within breeds. Past research suggests that variation in mDNA accounts for 1-5% of the phenotypic variation in dairy traits. Here we simulated a dairy cattle breeding program to assess the impact of accounting for mDNA variation in pedigree-based and genome-based genetic evaluations on the accuracy of estimated breeding values for mDNA and nDNA components. We also examined the impact of alternative definitions of breeding values on genetic gain, including nDNA and mDNA components that both impact phenotype expression, but mDNA is inherited only maternally. We found that accounting for mDNA variation increased accuracy between &#43;0.01 and &#43;0.05 for different categories of animals, especially for young bulls (&#43;0.05) and females without genotype data (between &#43;0.01 and &#43;0.03). Different scenarios of modelling and breeding value definition impacted genetic gain. The standard approach of ignoring mDNA variation achieved competitive genetic gain. Modelling, but not selecting on mDNA expectedly reduced genetic gain, while optimal use of mDNA variation recovered the genetic gain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568737">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568737" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568737">
        <p class="paperTitle">Mycobacterium smegmatis NucS-promoted DNA Mismatch Repair involves limited resection by a 5&#39;-3&#39; exonuclease and is independent of homologous recombination and NHEJ</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568737" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568737" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rivera-Flores, I. V.; Wang, E. X.; Murphy, K. C.</p>
        <p class="info">Score: 2.2, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568737' target='https://doi.org/10.1101/2023.11.26.568737'> 10.1101/2023.11.26.568737</a></p>
        <p class="abstract">The MutSL mismatch repair system in bacteria and eukaryotes correct mismatches made at the replication fork to maintain genome stability. Despite lacking the mutSL genes, most actinobacterial species, including Mycobacterium tuberculosis, exhibit spontaneous mutation rates similar to MutSL-bearing bacteria, suggesting the existence of an alternative mismatch repair (MMR) system. It has been shown previously that the EndoMS/NucS protein from Actinobacterium Corynebacterium glutamicum is a mismatch-specific endonuclease that recognizes and creates dsDNA breaks in vitro. Additionally, the structure of the NucS protein from the archaeal species P. abyssi reveals high similarity to the structures of restriction endonucleases, leading to a model that NucS mismatch repair involves a dsDNA break in the repair process. In this report, a genetic analysis shows that an M. smegmatis 11nucS strain could be complemented by expression of wild type NucS protein, but not by one deleted of its last five amino acids, a region predicted to be critical for binding to the {beta}-clamp at the replication fork. Oligo-recombineering promoted by Che9 phage RecT annealase was then leveraged to directly deliver defined mismatches to a defective hygromycin resistance gene on the M. smegmatis chromosome. We find that NucS recognizes and repairs G-G, G-T, and T-T mismatches in vivo, consistent with the recognition of these same mismatches in C. glutamicum in vitro, as well as mutation accumulation studies done in M. smegmatis. Three residues in M. smegmatis NucS, predicted to be catalytic residues based on the restriction enzyme-like structure of P. abyssi NucS, were mutated to alanine; all three mutants were defective for mismatch repair. Finally, an assay the employs one oligo that promotes the generation of two mismatches in close proximity on the chromosome shows that a NucS-promoted cut in dsDNA is processed by a 5 - 3 exonuclease, but is independent of both RecA, RadA, and NHEJ functions Ku and LigD, suggesting that the mechanism of DNA mismatch repair does not involve a homologous recombination event or non-homologous end-joining.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.566520">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.566520" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.566520">
        <p class="paperTitle">Whole genome association testing in 333,100 individuals across three biobanks identifies rare non-coding single variant and genomic aggregate associations with height</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.566520" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.566520" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hawkes, G.; Beaumont, R. N.; Li, Z.; Mandla, R.; Li, X.; Albert, C. M.; Arnett, D. K.; Ashley-Koch, A. E.; Ashrani, A. A.; Barnes, K. C.; Boerwinkle, E.; Brody, J. A.; Carson, A. P.; Chami, N.; Chen, Y.-D. I.; Chung, M. K.; Curran, J. E.; Darbar, D.; Ellinor, P. T.; Fornage, M.; Gordeuk, V. R.; Guo, X.; He, J.; Hwu, C.-M.; Kalyani, R. R.; Kaplan, R.; Kardia, S. L. R.; Kooperberg, C.; Loos, R. J. F.; Lubitz, S. A.; Minster, R. L.; Mitchell, B. D.; Murabito, J. M.; Palmer, N. D.; Psaty, B. M.; Redline, S.; Shoemaker, M. B.; Silverman, E. K.; Telen, M. J.; Weiss, S. T.; Yanek, L. R.; Zhou, H.; NH</p>
        <p class="info">Score: 29.6, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.566520' target='https://doi.org/10.1101/2023.11.19.566520'> 10.1101/2023.11.19.566520</a></p>
        <p class="abstract">The role of rare non-coding variation in complex human phenotypes is still largely unknown. To elucidate the impact of rare variants in regulatory elements, we performed a whole-genome sequencing association analysis for height using 333,100 individuals from three datasets: UK Biobank (N=200,003), TOPMed (N=87,652) and All of Us (N=45,445). We performed rare (&lt;0.1% minor-allele-frequency) single-variant and aggregate testing of non-coding variants in regulatory regions based on proximal, intergenic and deep-intronic annotation. We observed 29 independent variants associated with height at P &lt; 6 x 10-10 after conditioning on previously reported variants, with effect sizes ranging from -7cm to &#43;4.7cm. We also identified and replicated non-coding aggregate-based associations proximal to HMGA1 containing variants associated with a 5cm taller height and of highly-conserved variants in MIR497HG on chromosome 17. We have developed a novel approach for identifying non-coding rare variants in regulatory regions with large effects from whole-genome sequencing data associated with complex traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566919">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566919" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566919">
        <p class="paperTitle">Identifying genetic variants that influence the abundance of cell states in single-cell data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566919" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566919" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rumker, L.; Sakaue, S.; Reshef, Y.; Kang, J. B.; Yazar, S.; Alquicira-Hernandez, J.; Valencia, C.; Lagattuta, K. A.; Mah-Som, A.; Nathan, A.; Powell, J. E.; Loh, P.-R.; Raychaudhuri, S.</p>
        <p class="info">Score: 22.9, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566919' target='https://doi.org/10.1101/2023.11.13.566919'> 10.1101/2023.11.13.566919</a></p>
        <p class="abstract">Introductory ParagraphTo understand genetic mechanisms driving disease, it is essential but difficult to map how risk alleles affect the composition of cells present in the body. Single-cell profiling quantifies granular information about tissues, but variant-associated cell states may reflect diverse combinations of the profiled cell features that are challenging to predefine. We introduce GeNA (Genotype-Neighborhood Associations), a statistical tool to identify cell state abundance quantitative trait loci (csaQTLs) in high-dimensional single-cell datasets. Instead of testing associations to predefined cell states, GeNA flexibly identifies the cell states whose abundance is most associated with genetic variants. In a genome-wide survey of scRNA-seq peripheral blood profiling from 969 individuals,1 GeNA identifies five independent loci associated with shifts in the relative abundance of immune cell states. For example, rs3003-T (p=1.96x10-11) associates with increased abundance of NK cells expressing TNF- response programs. This csaQTL colocalizes with increased risk for psoriasis, an autoimmune disease that responds to anti-TNF treatments. Flexibly characterizing csaQTLs for granular cell states may help illuminate how genetic background alters cellular composition to confer disease risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568222">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568222" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568222">
        <p class="paperTitle">The C. elegans Myc-family of transcription factors coordinate a dynamic adaptive response to dietary restriction.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568222" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568222" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cornwell, A. B.; Zhang, Y.; Thondamal, M.; Johnson, D. W.; Thakar, J.; Samuelson, A. V.</p>
        <p class="info">Score: 2.1, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568222' target='https://doi.org/10.1101/2023.11.22.568222'> 10.1101/2023.11.22.568222</a></p>
        <p class="abstract">Dietary restriction (DR), the process of decreasing overall food consumption over an extended period of time, has been shown to increase longevity across evolutionarily diverse species and delay the onset of age-associated diseases in humans. In Caenorhabditis elegans, the Myc-family transcription factors (TFs) MXL-2 (Mlx) and MML-1 (MondoA/ChREBP), which function as obligate heterodimers, and PHA-4 (orthologous to forkhead box transcription factor A) are both necessary for the full physiological benefits of DR. However, the adaptive transcriptional response to DR and the role of MML-1::MXL-2 and PHA-4 remains elusive. We identified the transcriptional signature of C. elegans DR, using the eat-2 genetic model, and demonstrate broad changes in metabolic gene expression in eat-2 DR animals, which requires both mxl-2 and pha-4. While the requirement for these factors in DR gene expression overlaps, we found many of the DR genes exhibit an opposing change in relative gene expression in eat-2;mxl-2 animals compared to wild-type, which was not observed in eat-2 animals with pha-4 loss. We further show functional deficiencies of the mxl-2 loss in DR outside of lifespan, as eat-2;mxl-2 animals exhibit substantially smaller brood sizes and lay a proportion of dead eggs, indicating that MML-1::MXL-2 has a role in maintaining the balance between resource allocation to the soma and to reproduction under conditions of chronic food scarcity. While eat-2 animals do not show a significantly different metabolic rate compared to wild-type, we also find that loss of mxl-2 in DR does not affect the rate of oxygen consumption in young animals. The gene expression signature of eat-2 mutant animals is consistent with optimization of energy utilization and resource allocation, rather than induction of canonical gene expression changes associated with acute metabolic stress -such as induction of autophagy after TORC1 inhibition. Consistently, eat-2 animals are not substantially resistant to stress, providing further support to the idea that chronic DR may benefit healthspan and lifespan through efficient use of limited resources rather than broad upregulation of stress responses, and also indicates that MML-1::MXL-2 and PHA-4 may have different roles in promotion of benefits in response to different pro-longevity stimuli.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
