<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Ancient farmer and steppe pastoralist-related founding lineages contributed to the complex landscape of episodes in the diversification of Chinese paternal lineages
Authors: Wang, M.; Huang, Y.; Liu, K.; Yuan, H.; Duan, S.; Wang, Z.; Wei, L.; Yao, H.; Sun, Q.; Zhong, J.; Tang, R.; Chen, J.; Sun, Y.; Li, X.; Su, H.; Yang, Q.; Hu, L.; Yun, L.; Yang, J.; Nie, S.; Cai, Y.; Yan, J.; Zhou, K.; 10K_CPGDP Consortium, ; Wang, C.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Ancient farmer and steppe pastoralist-related founding lineages contributed to the complex landscape of episodes in the diversification of Chinese paternal lineages
Authors: Wang, M.; Huang, Y.; Liu, K.; Yuan, H.; Duan, S.; Wang, Z.; Wei, L.; Yao, H.; Sun, Q.; Zhong, J.; Tang, R.; Chen, J.; Sun, Y.; Li, X.; Su, H.; Yang, Q.; Hu, L.; Yun, L.; Yang, J.; Nie, S.; Cai, Y.; Yan, J.; Zhou, K.; 10K_CPGDP Consortium, ; Wang, C." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-03T10:37:18+00:00" />
<meta property="article:modified_time" content="2023-09-03T10:37:18+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Ancient farmer and steppe pastoralist-related founding lineages contributed to the complex landscape of episodes in the diversification of Chinese paternal lineages
Authors: Wang, M.; Huang, Y.; Liu, K.; Yuan, H.; Duan, S.; Wang, Z.; Wei, L.; Yao, H.; Sun, Q.; Zhong, J.; Tang, R.; Chen, J.; Sun, Y.; Li, X.; Su, H.; Yang, Q.; Hu, L.; Yun, L.; Yang, J.; Nie, S.; Cai, Y.; Yan, J.; Zhou, K.; 10K_CPGDP Consortium, ; Wang, C."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Ancient farmer and steppe pastoralist-related founding lineages contributed to the complex landscape of episodes in the diversification of Chinese paternal lineages\nAuthors: Wang, M.; Huang, Y.; Liu, K.; Yuan, H.; Duan, S.; Wang, Z.; Wei, L.; Yao, H.; Sun, Q.; Zhong, J.; Tang, R.; Chen, J.; Sun, Y.; Li, X.; Su, H.; Yang, Q.; Hu, L.; Yun, L.; Yang, J.; Nie, S.; Cai, Y.; Yan, J.; Zhou, K.; 10K_CPGDP Consortium, ; Wang, C.",
  "keywords": [
    
  ],
  "articleBody": " Ancient farmer and steppe pastoralist-related founding lineages contributed to the complex landscape of episodes in the diversification of Chinese paternal lineages\nAuthors: Wang, M.; Huang, Y.; Liu, K.; Yuan, H.; Duan, S.; Wang, Z.; Wei, L.; Yao, H.; Sun, Q.; Zhong, J.; Tang, R.; Chen, J.; Sun, Y.; Li, X.; Su, H.; Yang, Q.; Hu, L.; Yun, L.; Yang, J.; Nie, S.; Cai, Y.; Yan, J.; Zhou, K.; 10K_CPGDP Consortium, ; Wang, C.-C.; Zhu, B.; Liu, C.; He, G.\nScore: 25.3, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555114\nAncient DNA advances have reported the complex genetic history of Eurasians, but how the knowledge of ancient subsistence strategy shifts and population movements influenced the fine-scale paternal genetic structure in East Asia has not been assessed. Here, we reported one integrated Y-chromosome genomic database of 15,530 people, including 1753 ancient people and newly-reported 919 individuals genotyped using our recently-developed targeted sequencing YHSeqY3000 panel, to explore Chinese genomic diversity, population evolutionary tracts and their genetic formation mechanism. We identified four major ancient technological innovations and population movements that shaped the landscape of Chinese paternal lineages. First, the expansion of millet farmers and early East Asians from the Yellow River Basin carrying the major O2/D subclades promoted the formation of the Sino-Tibetan peoples major composition and accelerated the Tibetan Plateaus permanent occupation. Second, rice farmers dispersal from the Yangtze River Valley carrying O1 and some sublineages of O2 contributed significantly to Tai-Kadai, Austronesian, Hmong-Mien, Austroasiatic people and southern Han Chinese. Third, Siberian-related paternal lineages of Q and C originated and boomed from Neolithic hunter-gatherers from the Mongolian Plateau and the Amur River Basin and significantly influenced the gene pools of northern Chinese. Fourth, western Eurasian-derived J, G and R lineages initially spread with Yamnaya steppe pastoralists and other proto-Indo-European people and further widely dispersed via the trans-Eurasian cultural communication along the Eurasian Steppe and the ancient Silk Road, remaining genetic trajectories in northwestern Chinese. Our work provided comprehensive modern and ancient genetic evidence to illuminate the impact of population interaction from the ancient farmer or herder-based societies on the genetic diversity patterns of modern people, revised our understandings of ancestral sources of Chinese paternal lineages, underscored the scientific imperative of the large-scale genomic resources of dense spatiotemporal underrepresented sampling populations to understand human evolutionary history.\nFunctional characterization of gene regulatory elements and neuropsychiatric disease-associated risk loci in iPSCs and iPSC-derived neurons\nAuthors: Yang, X.; Jones, I. R.; Chen, P. B.; Yang, H.; Ren, X.; Zheng, L.; Li, B.; Li, Y. E.; Sun, Q.; Wen, J.; Beaman, C.; Cui, X.; Li, Y.; Wang, W.; Hu, M.; Ren, B.; Shen, Y.\nScore: 19.6, Published: 2023-08-30 DOI: 10.1101/2023.08.30.555359\nGenome-wide association studies (GWAS) have identified thousands of non-coding variants that contribute to psychiatric disease risks, likely by perturbing cis-regulatory elements (CREs). However, our ability to interpret and explore their mechanisms of action is hampered by a lack of annotation of functional CREs (fCREs) in neural cell types. Here, through genome-scale CRISPR screens of 22,000 candidate CREs (cCREs) in human induced pluripotent stem cells (iPSCs) undergoing differentiation to excitatory neurons, we identify 2,847 and 5,540 fCREs essential for iPSC fitness and neuronal differentiation, respectively. These fCREs display dynamic epigenomic features and exhibit increased numbers and genomic spans of chromatin interactions following terminal neuronal differentiation. Furthermore, fCREs essential for neuronal differentiation show significantly greater enrichment of genetic heritability for neurodevelopmental diseases including schizophrenia (SCZ), attention deficit hyperactivity disorder (ADHD), and autism spectrum disorders (ASD) than cCREs. Using high-throughput prime editing screens we experimentally confirm 45 SCZ risk variants that act by affecting the function of fCREs. The extensive and in-depth functional annotation of cCREs in neuronal types therefore provides a crucial resource for interpreting non-coding risk variants of neuropsychiatric disorders.\nLeveraging fine-scale population structure reveals conservation in genetic effect sizes between human populations across a range of human phenotypes\nAuthors: Hu, S.; Ferreira, L. A. F.; Shi, S.; Hellenthal, G.; Marchini, J.; Lawson, D. J.; Myers, S. R.\nScore: 47.0, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552281\nAn understanding of genetic differences between populations is essential for avoiding confounding in genome-wide association studies (GWAS) and understanding the evolution of human traits. Polygenic risk scores constructed in one group perform poorly in highly genetically-differentiated populations, for reasons which remain controversial. We developed a statistical ancestry inference pipeline able to decompose ancestry both within and between countries, and applied it to the UK Biobank data. This identifies fine-scale patterns of genetic relatedness not captured by standard and widely used principal components (PCs), and allows fine-scale population stratification correction that removes both false positive and false negative associations for traits with geographic correlations. We also develop and apply ANCHOR, an approach leveraging segments of distinct ancestries within individuals to estimate similarity in underlying causal effect sizes between groups, using an existing PGS. Applying ANCHOR to \u003e8000 people of mixed African and European ancestry, we demonstrate that estimated causal effect sizes are highly similar across these ancestries for 26 of 29 quantitative molecular and non-molecular phenotypes (mean correlation 0.98 +/-0.08), providing evidence that gene-environment and gene-gene interactions do not play major roles in the poor prediction of European-ancestry PRS scores in African populations for these traits, contradicting previous findings. Instead our results provide optimism that shared causal mutations operate similarly in different groups, focussing the challenge of improving GWAS \"portability\" between groups on joint fine-mapping.\nA defined diet for pre-adult Drosophila melanogaster\nAuthors: Martelli, F.; Quig, A.; Mele, S.; Lin, J.; Fulton, T. L.; Wansbrough, M.; Barlow, C. K.; Schittenhelm, R. B.; Johnson, T. K.; Piper, M. D.\nScore: 14.4, Published: 2023-08-30 DOI: 10.1101/2023.08.28.555220\nDrosophila melanogaster is unique among animal models because it has a fully defined synthetic diet available to study nutrient-gene interactions. However, use of this diet is limited to adult studies due to impaired larval development and survival. Here, we provide an adjusted formula that improves larval growth, development rate, and rescues survival. We demonstrate its use for exploring pre-adult diet compositions of therapeutic potential in a model of an inherited metabolic disorder.\nAdmixture as a source for HLA variation in Neolithic European farming communities\nAuthors: da Silva, N. A.; Ozer, O.; Haller, M.; Chen, Y.-R.; Kolbe, D.; Schade-Linding, S.; Wahl, J.; Berszin, C.; Francken, M.; Gorner, I.; Schierhold, K.; Pechtl, J.; Grupe, G.; Rinne, C.; Muller, J.; Lenz, T. L.; Nebel, A.; Krause-Kyora, B.\nScore: 15.6, Published: 2023-08-23 DOI: 10.1101/2023.08.23.554285\nThe northern European Neolithic is characterized by two major demographic events: immigration of early farmers (EF) from Anatolia (5500 BCE) and their admixture (from [~]4200 BCE) with western hunter-gatherers (WHG) forming late farmers (LF). The influence of this admixture event on variation in the immune-relevant human leukocyte antigen (HLA) region is understudied. Here, we conducted population and immunogenetic analyses on 83 individuals from six EF and LF sites located in present-day Germany. We observed significant shifts in HLA allele frequencies from EF to LF. The HLA diversity increased from EF to LF, likely due to admixture with WHG. However, it was considerably lower than in modern populations. Both EF and LF exhibited a relatively narrow HLA allele spectrum compared to today. This coincides with sparse traces of pathogen DNA, potentially indicating a lower pathogen pressure at the time. We additionally noted that LF resulted from sex-biased admixture from male WHG. TEASERMore restricted HLA allele spectrum and lower diversity in Neolithic farmers than in modern populations\nAbsence of chromosome axis proteins recruitment prevents meiotic recombination chromosome-wide in the budding yeast Lachancea kluyveri\nAuthors: Legrand, S.; Saifudeen, A.; Bordelet, H.; Vernerey, J.; Guille, A.; Bignaut, A.; Thierry, A.; Acquaviva, L.; Gaudin, M.; Sanchez, A.; Johnson, D.; Friedrcih, A.; Schacherer, J.; Neale, M.; Borde, V.; Koszul, R.; llorente, b.\nScore: 12.4, Published: 2023-08-29 DOI: 10.1101/2023.08.28.555118\nMeiotic recombination shows broad variations across species and along chromosomes, and is often suppressed at and around genomic regions determining sexual compatibility such as mating type loci in fungi. Here we show that the absence of Spo11-DSBs and meiotic recombination on Lakl0C-left, the chromosome arm containing the sex locus of the Lachancea kluyveri budding yeast, results from the absence of recruitment of the two chromosome axis proteins Red1 and Hop1, essential for proper Spo11-DSBs formation. Furthermore, cytological observation of spread pachytene meiotic chromosomes reveals that Lakl0C-left does not undergo synapsis. However, we show that the behavior of Lakl0C-left is independent of its particularly early replication timing and is not accompanied by any peculiar chromosome structure as detectable by Hi-C in this yet poorly studied yeast. Finally, we observed an accumulation of heterozygous mutations on Lakl0C-left and a sexual dimorphism of the haploid meiotic offspring, supporting a direct effect of this absence of meiotic recombination on L. kluyveri genome evolution and fitness. Because suppression of meiotic recombination on sex chromosomes is widely observed across eukaryotes, the novel mechanism for recombination suppression described here may apply to other species, with the potential to impact sex chromosome evolution.\nIdentification and functional characterisation of a locus for target site integration in Fusarium graminearum\nAuthors: Darino, M.; Urban, M.; Kaur, N.; Machado-Wood, A.; Grimwade-Mann, M.; Smith, D.; Beacham, A.; Hammond-kosack, K.\nScore: 10.4, Published: 2023-08-21 DOI: 10.1101/2023.08.20.553861\nBackgroundFusarium Head Blight is a destructive floral disease of different cereal crops. The Ascomycete fungus Fusarium graminearum (Fg) is one of the main causal agents of FHB in wheat and barley. The role(s) in virulence of Fg genes include genetic studies that involve the transformation of the fungus with different expression cassettes. We have observed in several studies where Fg genes functions were characterised that integration of expression cassettes occurred randomly. Random insertion of a cassette may disrupt gene expression and/or protein functions and hence the overall conclusion of the study. Target site integration (TSI) is an approach that consists in identifying a chromosomal region where the cassette can be inserted. The identification of a suitable locus for TSI in Fg would avert the potential risks of ectopic integration. ResultsHere, we identified a highly conserved intergenic region on chromosome 1 suitable for TSI. We named this intergenic region the TSI locus 1. We developed an efficient cloning vector system based on the Golden Gate method to clone different expression cassettes for use in combination with TSI locus 1. We present evidence that integrations in the TSI locus 1 affects neither fungal virulence nor fungal growth under different stress conditions. Integrations at the TSI locus 1 resulted in the expression of different gene fusions. In addition, the activities of Fg native promoters were not altered by integration into the TSI locus 1. We have developed a bespoke bioinformatic pipeline to analyse the existence of ectopic integrations and tandem insertions of the cassette that may occurred during the transformation process. Finally, we established a protocol to study protein secretion in wheat coleoptiles using confocal microscopy and the TSI locus 1. ConclusionThe TSI locus 1 can be used in Fg and potentially other cereal infecting Fusarium species for diverse studies including promoter activity analysis, secretion, protein localisation studies and gene complementation. The bespoke bioinformatic pipeline developed in this work can be an alternative to southern blotting, the gold standard technique to identify ectopic integration and tandem insertions in fungal transformation.\nDecreased accuracy of forensic DNA mixture analysis for groups with lower genetic diversity\nAuthors: Flores, M.; Ly, C.; Ho, E.; Ceberio, N.; Felix, K.; Thorner, H. M.; Guardado, M.; Paunovich, M.; Godek, C.; Kalaydjian, C.; Rohlfs, R.\nScore: 4.9, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554311\nForensic investigation of DNA samples from multiple contributors has become commonplace. These complex analyses use statistical frameworks accounting for multiple levels of uncertainty in allelic contributions from different individuals, particularly for samples containing few molecules of DNA. These methods have been thoroughly tested along some axes of variation, but less attention has been paid to accuracy across human genetic variation. Here, we quantify the accuracy of DNA mixture analysis over 244 human groups. We find higher false inclusion rates for mixtures with more contributors, and for groups with lower genetic diversity. Even for two-contributor mixtures where one contributor is known and the reference group is correctly specified, false inclusion rates are 1e-5 or higher for 56 out of 244 groups. This means that, depending on multiple testing, some false inclusions may be expected. These false positives could be lessened with more selective and conservative use of DNA mixture analysis. HIGHLIGHTSO_LIGroups with lower genetic diversity have higher mixture analysis false positive rates. C_LIO_LIAnalyses with mis-specified references have somewhat higher false positive rates. C_LIO_LIMixture analysis accuracy decreases with more mixture contributors. C_LI GRAPHICAL ABSTRACT O_FIG O_LINKSMALLFIG WIDTH=198 HEIGHT=200 SRC=\"FIGDIR/small/554311v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (51K): org.highwire.dtl.DTLVardef@2b8c0forg.highwire.dtl.DTLVardef@f86a6borg.highwire.dtl.DTLVardef@112d1c5org.highwire.dtl.DTLVardef@89e64b_HPS_FORMAT_FIGEXP M_FIG C_FIG\nComplete correction of murine phenylketonuria by selection-enhanced hepatocyte transplantation\nAuthors: Vonada, A.; Wakefield, L.; Martinez, M.; Harding, C. O.; Grompe, M.; Tiyaboonchai, A.\nScore: 4.2, Published: 2023-08-28 DOI: 10.1101/2023.08.27.554228\nHepatocyte transplantation for genetic liver diseases has several potential advantages over gene therapy. However, low efficiency of cell engraftment has limited its clinical implementation. This problem could be overcome by selectively expanding transplanted donor cells until they replace enough of the liver mass to achieve therapeutic benefit. We previously described a gene therapy method to selectively expand hepatocytes deficient in cytochrome p450 reductase (Cypor) using acetaminophen (APAP). Because Cypor is required for the transformation of APAP to a hepatotoxic metabolite, Cypor deficient cells are protected from toxicity and are able to expand following APAP-induced liver injury. Here, we apply this selection system to correct a mouse model of phenylketonuria (PKU) by cell transplantation. Hepatocytes from a wildtype donor animal were edited in vitro to create Cypor deficiency and then transplanted into PKU animals. Following selection with APAP, blood phenylalanine concentrations were fully normalized and remained stable following APAP withdrawal. Cypor-deficient hepatocytes expanded from \u003c1% to [~]14% in corrected animals, and they showed no abnormalities in blood chemistries, liver histology, or drug metabolism. We conclude that APAP-mediated selection of transplanted hepatocytes is a potential therapeutic for PKU with long-term efficacy and a favorable safety profile.\nVitamin C activates young LINE-1 elements in mouse embryonic stem cells via H3K9me3 demethylation\nAuthors: Cheng, K. C. L.; Frost, J. M.; Sanchez-Luque, F. J.; Garcia-Canadas, M.; Taylor, D.; Yang, W. R.; Irayanar, B.; Sampath, S.; Patani, H.; Agger, K.; Helin, K.; Ficz, G.; Burns, K. H.; Ewing, A. D.; Garcia-Perez, J. L.; Branco, M. R.\nScore: 12.3, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552254\nVitamin C (vitC) enhances the activity of 2-oxoglutarate-dependent dioxygenases, including TET enzymes, which catalyse DNA demethylation, and Jumonji-domain histone demethylases. The epigenetic remodelling promoted by vitC improves the efficiency of induced pluripotent stem cell derivation, and is required to attain a ground-state of pluripotency in embryonic stem cells (ESCs) that closely mimics the inner cell mass of the early blastocyst. However, genome-wide DNA and histone demethylation can lead to upregulation of transposable elements (TEs), and it is not known how vitC addition in culture media affects TE expression in pluripotent stem cells. Here we show that vitC increases the expression of evolutionarily young LINE-1 (L1) elements in mouse ESCs. We find that TET activity is dispensable for these effects, and that instead L1 upregulation occurs largely as a result of H3K9me3 loss mediated by KDM4A/C histone demethylases. Despite increased L1 levels, we did not detect increased somatic insertion rates in vitC-treated cells. Notably, treatment of human ESCs with vitC also increases L1 protein levels, which could impact the genetic and epigenetic stability of human pluripotent stem cells.\n",
  "wordCount" : "2546",
  "inLanguage": "en",
  "datePublished": "2023-09-03T10:37:18Z",
  "dateModified": "2023-09-03T10:37:18Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on September 3, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555114">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555114" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555114">
        <p class="paperTitle">Ancient farmer and steppe pastoralist-related founding lineages contributed to the complex landscape of episodes in the diversification of Chinese paternal lineages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555114" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555114" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, M.; Huang, Y.; Liu, K.; Yuan, H.; Duan, S.; Wang, Z.; Wei, L.; Yao, H.; Sun, Q.; Zhong, J.; Tang, R.; Chen, J.; Sun, Y.; Li, X.; Su, H.; Yang, Q.; Hu, L.; Yun, L.; Yang, J.; Nie, S.; Cai, Y.; Yan, J.; Zhou, K.; 10K_CPGDP Consortium,  ; Wang, C.-C.; Zhu, B.; Liu, C.; He, G.</p>
        <p class="info">Score: 25.3, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555114' target='https://doi.org/10.1101/2023.08.28.555114'> 10.1101/2023.08.28.555114</a></p>
        <p class="abstract">Ancient DNA advances have reported the complex genetic history of Eurasians, but how the knowledge of ancient subsistence strategy shifts and population movements influenced the fine-scale paternal genetic structure in East Asia has not been assessed. Here, we reported one integrated Y-chromosome genomic database of 15,530 people, including 1753 ancient people and newly-reported 919 individuals genotyped using our recently-developed targeted sequencing YHSeqY3000 panel, to explore Chinese genomic diversity, population evolutionary tracts and their genetic formation mechanism. We identified four major ancient technological innovations and population movements that shaped the landscape of Chinese paternal lineages. First, the expansion of millet farmers and early East Asians from the Yellow River Basin carrying the major O2/D subclades promoted the formation of the Sino-Tibetan peoples major composition and accelerated the Tibetan Plateaus permanent occupation. Second, rice farmers dispersal from the Yangtze River Valley carrying O1 and some sublineages of O2 contributed significantly to Tai-Kadai, Austronesian, Hmong-Mien, Austroasiatic people and southern Han Chinese. Third, Siberian-related paternal lineages of Q and C originated and boomed from Neolithic hunter-gatherers from the Mongolian Plateau and the Amur River Basin and significantly influenced the gene pools of northern Chinese. Fourth, western Eurasian-derived J, G and R lineages initially spread with Yamnaya steppe pastoralists and other proto-Indo-European people and further widely dispersed via the trans-Eurasian cultural communication along the Eurasian Steppe and the ancient Silk Road, remaining genetic trajectories in northwestern Chinese. Our work provided comprehensive modern and ancient genetic evidence to illuminate the impact of population interaction from the ancient farmer or herder-based societies on the genetic diversity patterns of modern people, revised our understandings of ancestral sources of Chinese paternal lineages, underscored the scientific imperative of the large-scale genomic resources of dense spatiotemporal underrepresented sampling populations to understand human evolutionary history.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555359">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555359" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555359">
        <p class="paperTitle">Functional characterization of gene regulatory elements and neuropsychiatric disease-associated risk loci in iPSCs and iPSC-derived neurons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555359" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555359" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, X.; Jones, I. R.; Chen, P. B.; Yang, H.; Ren, X.; Zheng, L.; Li, B.; Li, Y. E.; Sun, Q.; Wen, J.; Beaman, C.; Cui, X.; Li, Y.; Wang, W.; Hu, M.; Ren, B.; Shen, Y.</p>
        <p class="info">Score: 19.6, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555359' target='https://doi.org/10.1101/2023.08.30.555359'> 10.1101/2023.08.30.555359</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have identified thousands of non-coding variants that contribute to psychiatric disease risks, likely by perturbing cis-regulatory elements (CREs). However, our ability to interpret and explore their mechanisms of action is hampered by a lack of annotation of functional CREs (fCREs) in neural cell types. Here, through genome-scale CRISPR screens of 22,000 candidate CREs (cCREs) in human induced pluripotent stem cells (iPSCs) undergoing differentiation to excitatory neurons, we identify 2,847 and 5,540 fCREs essential for iPSC fitness and neuronal differentiation, respectively. These fCREs display dynamic epigenomic features and exhibit increased numbers and genomic spans of chromatin interactions following terminal neuronal differentiation. Furthermore, fCREs essential for neuronal differentiation show significantly greater enrichment of genetic heritability for neurodevelopmental diseases including schizophrenia (SCZ), attention deficit hyperactivity disorder (ADHD), and autism spectrum disorders (ASD) than cCREs. Using high-throughput prime editing screens we experimentally confirm 45 SCZ risk variants that act by affecting the function of fCREs. The extensive and in-depth functional annotation of cCREs in neuronal types therefore provides a crucial resource for interpreting non-coding risk variants of neuropsychiatric disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552281">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552281" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552281">
        <p class="paperTitle">Leveraging fine-scale population structure reveals conservation in genetic effect sizes between human populations across a range of human phenotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552281" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552281" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, S.; Ferreira, L. A. F.; Shi, S.; Hellenthal, G.; Marchini, J.; Lawson, D. J.; Myers, S. R.</p>
        <p class="info">Score: 47.0, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552281' target='https://doi.org/10.1101/2023.08.08.552281'> 10.1101/2023.08.08.552281</a></p>
        <p class="abstract">An understanding of genetic differences between populations is essential for avoiding confounding in genome-wide association studies (GWAS) and understanding the evolution of human traits. Polygenic risk scores constructed in one group perform poorly in highly genetically-differentiated populations, for reasons which remain controversial. We developed a statistical ancestry inference pipeline able to decompose ancestry both within and between countries, and applied it to the UK Biobank data. This identifies fine-scale patterns of genetic relatedness not captured by standard and widely used principal components (PCs), and allows fine-scale population stratification correction that removes both false positive and false negative associations for traits with geographic correlations. We also develop and apply ANCHOR, an approach leveraging segments of distinct ancestries within individuals to estimate similarity in underlying causal effect sizes between groups, using an existing PGS. Applying ANCHOR to &gt;8000 people of mixed African and European ancestry, we demonstrate that estimated causal effect sizes are highly similar across these ancestries for 26 of 29 quantitative molecular and non-molecular phenotypes (mean correlation 0.98 &#43;/-0.08), providing evidence that gene-environment and gene-gene interactions do not play major roles in the poor prediction of European-ancestry PRS scores in African populations for these traits, contradicting previous findings. Instead our results provide optimism that shared causal mutations operate similarly in different groups, focussing the challenge of improving GWAS &#34;portability&#34; between groups on joint fine-mapping.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555220">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555220" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555220">
        <p class="paperTitle">A defined diet for pre-adult Drosophila melanogaster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555220" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555220" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martelli, F.; Quig, A.; Mele, S.; Lin, J.; Fulton, T. L.; Wansbrough, M.; Barlow, C. K.; Schittenhelm, R. B.; Johnson, T. K.; Piper, M. D.</p>
        <p class="info">Score: 14.4, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555220' target='https://doi.org/10.1101/2023.08.28.555220'> 10.1101/2023.08.28.555220</a></p>
        <p class="abstract">Drosophila melanogaster is unique among animal models because it has a fully defined synthetic diet available to study nutrient-gene interactions. However, use of this diet is limited to adult studies due to impaired larval development and survival. Here, we provide an adjusted formula that improves larval growth, development rate, and rescues survival. We demonstrate its use for exploring pre-adult diet compositions of therapeutic potential in a model of an inherited metabolic disorder.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554285">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554285" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554285">
        <p class="paperTitle">Admixture as a source for HLA variation in Neolithic European farming communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554285" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554285" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: da Silva, N. A.; Ozer, O.; Haller, M.; Chen, Y.-R.; Kolbe, D.; Schade-Linding, S.; Wahl, J.; Berszin, C.; Francken, M.; Gorner, I.; Schierhold, K.; Pechtl, J.; Grupe, G.; Rinne, C.; Muller, J.; Lenz, T. L.; Nebel, A.; Krause-Kyora, B.</p>
        <p class="info">Score: 15.6, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554285' target='https://doi.org/10.1101/2023.08.23.554285'> 10.1101/2023.08.23.554285</a></p>
        <p class="abstract">The northern European Neolithic is characterized by two major demographic events: immigration of early farmers (EF) from Anatolia (5500 BCE) and their admixture (from [~]4200 BCE) with western hunter-gatherers (WHG) forming late farmers (LF). The influence of this admixture event on variation in the immune-relevant human leukocyte antigen (HLA) region is understudied. Here, we conducted population and immunogenetic analyses on 83 individuals from six EF and LF sites located in present-day Germany. We observed significant shifts in HLA allele frequencies from EF to LF. The HLA diversity increased from EF to LF, likely due to admixture with WHG. However, it was considerably lower than in modern populations. Both EF and LF exhibited a relatively narrow HLA allele spectrum compared to today. This coincides with sparse traces of pathogen DNA, potentially indicating a lower pathogen pressure at the time. We additionally noted that LF resulted from sex-biased admixture from male WHG.

TEASERMore restricted HLA allele spectrum and lower diversity in Neolithic farmers than in modern populations</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555118">
        <p class="paperTitle">Absence of chromosome axis proteins recruitment prevents meiotic recombination chromosome-wide in the budding yeast Lachancea kluyveri</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Legrand, S.; Saifudeen, A.; Bordelet, H.; Vernerey, J.; Guille, A.; Bignaut, A.; Thierry, A.; Acquaviva, L.; Gaudin, M.; Sanchez, A.; Johnson, D.; Friedrcih, A.; Schacherer, J.; Neale, M.; Borde, V.; Koszul, R.; llorente, b.</p>
        <p class="info">Score: 12.4, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555118' target='https://doi.org/10.1101/2023.08.28.555118'> 10.1101/2023.08.28.555118</a></p>
        <p class="abstract">Meiotic recombination shows broad variations across species and along chromosomes, and is often suppressed at and around genomic regions determining sexual compatibility such as mating type loci in fungi. Here we show that the absence of Spo11-DSBs and meiotic recombination on Lakl0C-left, the chromosome arm containing the sex locus of the Lachancea kluyveri budding yeast, results from the absence of recruitment of the two chromosome axis proteins Red1 and Hop1, essential for proper Spo11-DSBs formation. Furthermore, cytological observation of spread pachytene meiotic chromosomes reveals that Lakl0C-left does not undergo synapsis. However, we show that the behavior of Lakl0C-left is independent of its particularly early replication timing and is not accompanied by any peculiar chromosome structure as detectable by Hi-C in this yet poorly studied yeast. Finally, we observed an accumulation of heterozygous mutations on Lakl0C-left and a sexual dimorphism of the haploid meiotic offspring, supporting a direct effect of this absence of meiotic recombination on L. kluyveri genome evolution and fitness. Because suppression of meiotic recombination on sex chromosomes is widely observed across eukaryotes, the novel mechanism for recombination suppression described here may apply to other species, with the potential to impact sex chromosome evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.553861">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.553861" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.553861">
        <p class="paperTitle">Identification and functional characterisation of a locus for target site integration in Fusarium graminearum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.553861" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.553861" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Darino, M.; Urban, M.; Kaur, N.; Machado-Wood, A.; Grimwade-Mann, M.; Smith, D.; Beacham, A.; Hammond-kosack, K.</p>
        <p class="info">Score: 10.4, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.553861' target='https://doi.org/10.1101/2023.08.20.553861'> 10.1101/2023.08.20.553861</a></p>
        <p class="abstract">BackgroundFusarium Head Blight is a destructive floral disease of different cereal crops. The Ascomycete fungus Fusarium graminearum (Fg) is one of the main causal agents of FHB in wheat and barley. The role(s) in virulence of Fg genes include genetic studies that involve the transformation of the fungus with different expression cassettes. We have observed in several studies where Fg genes functions were characterised that integration of expression cassettes occurred randomly. Random insertion of a cassette may disrupt gene expression and/or protein functions and hence the overall conclusion of the study. Target site integration (TSI) is an approach that consists in identifying a chromosomal region where the cassette can be inserted. The identification of a suitable locus for TSI in Fg would avert the potential risks of ectopic integration.

ResultsHere, we identified a highly conserved intergenic region on chromosome 1 suitable for TSI. We named this intergenic region the TSI locus 1. We developed an efficient cloning vector system based on the Golden Gate method to clone different expression cassettes for use in combination with TSI locus 1. We present evidence that integrations in the TSI locus 1 affects neither fungal virulence nor fungal growth under different stress conditions. Integrations at the TSI locus 1 resulted in the expression of different gene fusions. In addition, the activities of Fg native promoters were not altered by integration into the TSI locus 1. We have developed a bespoke bioinformatic pipeline to analyse the existence of ectopic integrations and tandem insertions of the cassette that may occurred during the transformation process. Finally, we established a protocol to study protein secretion in wheat coleoptiles using confocal microscopy and the TSI locus 1.

ConclusionThe TSI locus 1 can be used in Fg and potentially other cereal infecting Fusarium species for diverse studies including promoter activity analysis, secretion, protein localisation studies and gene complementation. The bespoke bioinformatic pipeline developed in this work can be an alternative to southern blotting, the gold standard technique to identify ectopic integration and tandem insertions in fungal transformation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554311">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554311" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554311">
        <p class="paperTitle">Decreased accuracy of forensic DNA mixture analysis for groups with lower genetic diversity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554311" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554311" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Flores, M.; Ly, C.; Ho, E.; Ceberio, N.; Felix, K.; Thorner, H. M.; Guardado, M.; Paunovich, M.; Godek, C.; Kalaydjian, C.; Rohlfs, R.</p>
        <p class="info">Score: 4.9, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554311' target='https://doi.org/10.1101/2023.08.25.554311'> 10.1101/2023.08.25.554311</a></p>
        <p class="abstract">Forensic investigation of DNA samples from multiple contributors has become commonplace. These complex analyses use statistical frameworks accounting for multiple levels of uncertainty in allelic contributions from different individuals, particularly for samples containing few molecules of DNA. These methods have been thoroughly tested along some axes of variation, but less attention has been paid to accuracy across human genetic variation. Here, we quantify the accuracy of DNA mixture analysis over 244 human groups. We find higher false inclusion rates for mixtures with more contributors, and for groups with lower genetic diversity. Even for two-contributor mixtures where one contributor is known and the reference group is correctly specified, false inclusion rates are 1e-5 or higher for 56 out of 244 groups. This means that, depending on multiple testing, some false inclusions may be expected. These false positives could be lessened with more selective and conservative use of DNA mixture analysis.

HIGHLIGHTSO_LIGroups with lower genetic diversity have higher mixture analysis false positive rates.
C_LIO_LIAnalyses with mis-specified references have somewhat higher false positive rates.
C_LIO_LIMixture analysis accuracy decreases with more mixture contributors.
C_LI

GRAPHICAL ABSTRACT

O_FIG O_LINKSMALLFIG WIDTH=198 HEIGHT=200 SRC=&#34;FIGDIR/small/554311v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (51K):
org.highwire.dtl.DTLVardef@2b8c0forg.highwire.dtl.DTLVardef@f86a6borg.highwire.dtl.DTLVardef@112d1c5org.highwire.dtl.DTLVardef@89e64b_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.554228">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.554228" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.554228">
        <p class="paperTitle">Complete correction of murine phenylketonuria by selection-enhanced hepatocyte transplantation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.554228" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.554228" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vonada, A.; Wakefield, L.; Martinez, M.; Harding, C. O.; Grompe, M.; Tiyaboonchai, A.</p>
        <p class="info">Score: 4.2, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.554228' target='https://doi.org/10.1101/2023.08.27.554228'> 10.1101/2023.08.27.554228</a></p>
        <p class="abstract">Hepatocyte transplantation for genetic liver diseases has several potential advantages over gene therapy. However, low efficiency of cell engraftment has limited its clinical implementation. This problem could be overcome by selectively expanding transplanted donor cells until they replace enough of the liver mass to achieve therapeutic benefit. We previously described a gene therapy method to selectively expand hepatocytes deficient in cytochrome p450 reductase (Cypor) using acetaminophen (APAP). Because Cypor is required for the transformation of APAP to a hepatotoxic metabolite, Cypor deficient cells are protected from toxicity and are able to expand following APAP-induced liver injury. Here, we apply this selection system to correct a mouse model of phenylketonuria (PKU) by cell transplantation. Hepatocytes from a wildtype donor animal were edited in vitro to create Cypor deficiency and then transplanted into PKU animals. Following selection with APAP, blood phenylalanine concentrations were fully normalized and remained stable following APAP withdrawal. Cypor-deficient hepatocytes expanded from &lt;1% to [~]14% in corrected animals, and they showed no abnormalities in blood chemistries, liver histology, or drug metabolism. We conclude that APAP-mediated selection of transplanted hepatocytes is a potential therapeutic for PKU with long-term efficacy and a favorable safety profile.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552254">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552254" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552254">
        <p class="paperTitle">Vitamin C activates young LINE-1 elements in mouse embryonic stem cells via H3K9me3 demethylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552254" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552254" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cheng, K. C. L.; Frost, J. M.; Sanchez-Luque, F. J.; Garcia-Canadas, M.; Taylor, D.; Yang, W. R.; Irayanar, B.; Sampath, S.; Patani, H.; Agger, K.; Helin, K.; Ficz, G.; Burns, K. H.; Ewing, A. D.; Garcia-Perez, J. L.; Branco, M. R.</p>
        <p class="info">Score: 12.3, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552254' target='https://doi.org/10.1101/2023.08.07.552254'> 10.1101/2023.08.07.552254</a></p>
        <p class="abstract">Vitamin C (vitC) enhances the activity of 2-oxoglutarate-dependent dioxygenases, including TET enzymes, which catalyse DNA demethylation, and Jumonji-domain histone demethylases. The epigenetic remodelling promoted by vitC improves the efficiency of induced pluripotent stem cell derivation, and is required to attain a ground-state of pluripotency in embryonic stem cells (ESCs) that closely mimics the inner cell mass of the early blastocyst. However, genome-wide DNA and histone demethylation can lead to upregulation of transposable elements (TEs), and it is not known how vitC addition in culture media affects TE expression in pluripotent stem cells. Here we show that vitC increases the expression of evolutionarily young LINE-1 (L1) elements in mouse ESCs. We find that TET activity is dispensable for these effects, and that instead L1 upregulation occurs largely as a result of H3K9me3 loss mediated by KDM4A/C histone demethylases. Despite increased L1 levels, we did not detect increased somatic insertion rates in vitC-treated cells. Notably, treatment of human ESCs with vitC also increases L1 protein levels, which could impact the genetic and epigenetic stability of human pluripotent stem cells.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
