<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish
Authors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.
Score: 58.7, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570868
Standard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish
Authors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.
Score: 58.7, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570868
Standard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-07T10:38:59+00:00" />
<meta property="article:modified_time" content="2024-01-07T10:38:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish
Authors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.
Score: 58.7, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570868
Standard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish\nAuthors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.\nScore: 58.7, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570868\nStandard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis.",
  "keywords": [
    
  ],
  "articleBody": " pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish\nAuthors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.\nScore: 58.7, Published: 2023-12-12 DOI: 10.1101/2023.12.08.570868\nStandard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis. However, while the phiC31 system functions in zebrafish, validated loci carrying attP-based landing or safe harbor sites suitable for universal transgenesis applications in zebrafish have not been established. Here, using CRISPR-Cas9, we converted two well-validated single insertion Tol2-based zebrafish transgenes with long-standing genetic stability into two attP landing sites, called phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET). Generating fluorescent reporters, loxP-based Switch lines, CreERT2 drivers, and gene-regulatory variant reporters in the pIGLET14a and pIGLET24b landing site alleles, we document their suitability for transgenesis applications across cell types and developmental stages. For both landing sites, we routinely achieve 25-50% germline transmission of targeted transgene integrations, drastically reducing the number of required animals and necessary resources to generate individual transgenic lines. We document that phiC31 integrase-based transgenesis into pIGLET14a and pIGLET24b reproducibly results in representative reporter expression patterns in injected F0 zebrafish embryos suitable for enhancer discovery and qualitative and quantitative comparison of gene-regulatory element variants. Taken together, our new phiC31 integrase-based transgene landing sites establish reproducible, targeted zebrafish transgenesis for numerous applications while greatly reducing the workload of generating new transgenic zebrafish lines. SHORT ABSTRACTTargeted transgenesis into pre-established, \"safe harbor,\" landing sites remains a missing technique in zebrafish. Here, we established phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET) by CRISPR-Cas9-based conversion of two previously validated Tol2 transgenes into attP sites. phiC31-mediated transgenesis into our pIGLET14a and pIGLET24b attP landing sites results in 25-50% germline transmission efficiency and quantifiable transgene activity. Our landing sites are suitable for reproducible, routine zebrafish transgenesis for diverse applications.\nPlant breeding simulations with AlphaSimR\nAuthors: Bancic, J.; Greenspoon, P.; Gaynor, C. R.; Gorjanc, G.\nScore: 10.7, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573724\nPlant breeding plays a crucial role in the development of high-performing crop varieties that meet the demands of society. Emerging breeding techniques offer the potential to improve the precision and efficiency of plant breeding programs; however, their optimal implementation requires refinement of existing breeding programs or the design of new ones. Stochastic simulations are a cost-effective solution for testing and optimizing new breeding strategies. The aim of this paper is to provide an introduction to stochastic simulation with software AlphaSimR for plant breeding students, researchers, and experienced breeders. We present an overview of how to use the software and provide an introductory AlphaSimR vignette as well as complete AlphaSimR scripts of breeding programs for self-pollinated, clonal, and cross-pollinated plants, including relevant breeding techniques, such as backcrossing, speed breeding, genomic selection, index selection, and others. Our objective is to provide a foundation for understanding and utilizing simulation software, enabling readers to adapt the provided scripts for their own use or even develop completely new plant breeding programs. By incorporating simulation software into plant breeding education and practice, the next generation of plant breeders will have a valuable tool in their quest to provide sustainable and nutritious food sources for a growing population.\nDetailed mechanisms for unintended large DNA deletions with CRISPR, base editors, and prime editors\nAuthors: Hwang, G.-h.; Kim, S.-H.; Oh, M.; Kim, S.; Habib, O.; Jang, H.-K.; Kim, H. S.; Kim, C. H.; Kim, S.; Bae, S.\nScore: 7.2, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574288\nCRISPR-Cas9 nucleases are versatile tools for genetic engineering cells and function by producing targeted double-strand breaks (DSBs) in the DNA sequence. However, the unintended production of large deletions (\u003e100 bp) represents a challenge to the effective application of this genome-editing system. We optimized a long-range amplicon sequencing system and developed a k-mer sequence-alignment algorithm to simultaneously detect small DNA alteration events and large DNA deletions. With this workflow, we determined that CRISPR-Cas9 induced large deletions at varying frequencies in cancer cell lines, stem cells, and primary T cells. With CRISPR interference screening, we determined that end resection and the subsequent TMEJ [DNA polymerase theta-mediated end joining] repair process produce most large deletions. Furthermore, base editors and prime editors also generated large deletions despite employing mutated Cas9 nickases that produce single-strand breaks. Our findings reveal an important limitation of current genome-editing tools and identify strategies for mitigating unwanted large deletion events.\nRare genetic variation in Fibronectin 1 (FN1) protects against APOEe4 in Alzheimer's disease\nAuthors: Bhattarai, P.; Gunasekaran, T. I.; Reyes-Dumeyer, D.; Juelich, D.; Tayran, H.; Yilmaz, E.; Flaherty, D.; Rafael, L.; Martin, M.; Diones, R.; Patricia, R.; Ertekin-Taner, N.; Teich, A. F.; Dennis, D. W.; Scott, H.; Mayeux, R.; Kizil, C.; Vardarajan, B. N.\nScore: 4.8, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573895\nThe risk of developing Alzheimers disease (AD) significantly increases in individuals carrying the APOE{varepsilon}4 allele. Elderly cognitively healthy individuals with APOE{varepsilon}4 also exist, suggesting the presence of cellular mechanisms that counteract the pathological effects of APOE{varepsilon}4; however, these mechanisms are unknown. We hypothesized that APOE{varepsilon}4 carriers without dementia might carry genetic variations that could protect them from developing APOE{varepsilon}4-mediated AD pathology. To test this, we leveraged whole genome sequencing (WGS) data in National Institute on Aging Alzheimers Disease Family Based Study (NIA-AD FBS), Washington Heights/Inwood Columbia Aging Project (WHICAP), and Estudio Familiar de Influencia Genetica en Alzheimer (EFIGA) cohorts and identified potentially protective variants segregating exclusively among unaffected APOE{varepsilon}4 carriers. In homozygous unaffected carriers above 70 years old, we identified 510 rare coding variants. Pathway analysis of the genes harboring these variants showed significant enrichment in extracellular matrix (ECM)-related processes, suggesting protective effects of functional modifications in ECM proteins. We prioritized two genes that were highly represented in the ECM-related gene ontology terms, (FN1) and collagen type VI alpha 2 chain (COL6A2) and are known to be expressed at the blood-brain barrier (BBB), for postmortem validation and in vivo functional studies. The FN1 and COL6A2 protein levels were increased at the BBB in APOE{varepsilon}4 carriers with AD. Brain expression of cognitively unaffected homozygous APOE{varepsilon}4 carriers had significantly lower FN1 deposition and less reactive gliosis compared to homozygous APOE{varepsilon}4 carriers with AD, suggesting that FN1 might be a downstream driver of APOE{varepsilon}4-mediated AD-related pathology and cognitive decline. To validate our findings, we used zebrafish models with loss-of-function (LOF) mutations in fn1b - the ortholog for human FN1. We found that fibronectin LOF reduced gliosis, enhanced gliovascular remodeling and potentiated the microglial response, suggesting that pathological accumulation of FN1 could impair toxic protein clearance, which is ameliorated with FN1 LOF. Our study suggests vascular deposition of FN1 is related to the pathogenicity of APOE{varepsilon}4, LOF variants in FN1 may reduce APOE{varepsilon}4-related AD risk, providing novel clues to potential therapeutic interventions targeting the ECM to mitigate AD risk.\nBetween-group heritability and the status of hereditarianism as an evolutionary science\nAuthors: Roseman, C. C.; Bird, K.\nScore: 31.7, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572247\nHereditarianism is a school of thought that contends there are substantial evolved cognitive and behavioral differences among groups of humans which are both resistant to environmental intervention and are a root cause of differential social outcomes across groups. The relationship of betweengroup heritability [Formula] to within-group heritability [Formula] is one of the key theoretical components of hereditarianism and forms one of the bases for its claim to be an evolutionary science. Here, we examine the relationship between [Formula]and [Formula] and its application to problems in the hereditarian literature from an evolutionary genetic perspective. We demonstrate that the formulation of the relationship between [Formula] and [Formula]used in the hereditarian literature has no evolutionary content. By re-writing the relationship between [Formula] and[Formula] in a novel evolutionary framework, we demonstrate that there is no way to predict[Formula] using[Formula] without considerable additional theory that is absent from the hereditarian literature. Furthermore, we demonstrate that the hereditarian technique that uses[Formula] and [Formula] as a means of judging whether a given difference between groups may be plausibly ameliorated through environmental intervention is mathematically flawed. Lastly, we fill a gap in the hereditarian literature by writing out a means of using [Formula] to predict the absolute difference between groups under a neutral evolutionary model and find that it is much smaller than claimed by hereditarians. In conclusion, we propose a path forward for the study of human variation that moves us past the ill-conceived nature vs. nurture question and allows us to focus on more productive issues.\nThe pattern of genetic variability in a core collection of 2,021 cowpea accessions\nAuthors: Fiscus, C. J.; Herniter, I. A.; Tchamba, M.; Paliwal, R.; Munoz-Amatriain, M.; Roberts, P. A.; Abberton, M.; Alaba, O.; Close, T. J.; Oyatomi, O.; Koenig, D.\nScore: 4.5, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572659\nCowpea is a highly drought-adapted leguminous crop with great promise for improving agricultural sustainability and food security. Here, we report analyses derived from array-based genotyping of 2,021 accessions constituting a core subset of the worlds largest cowpea collection, held at the International Institute of Tropical Agriculture (IITA) in Ibadan, Nigeria. We used this dataset to examine genetic variation and population structure in worldwide cowpea. We confirm that the primary pattern of population structure is two geographically defined subpopulations origining in West and East Africa, respectively, and that population structure is associated with shifts in phenotypic distribution. Furthermore, we establish the cowpea core collection as a resource for genome-wide association studies by mapping the genetic basis of several phenotypes, with a focus on seed coat pigmentation patterning and color. We anticipate that the genotyped IITA cowpea core collection will serve as a powerful tool for mapping complex traits, facilitating the acceleration of breeding programs to enhance the resilience of this crop in the face of rapid global climate change.\nDNA-PKcs Suppresses Illegitimate Chromosome Rearrangements\nAuthors: Wang, J.; Sadeghi, C.; Frock, R. L.\nScore: 3.1, Published: 2024-01-01 DOI: 10.1101/2023.12.30.573736\nTwo DNA repair pathways, non-homologous end joining (NHEJ) and alternative end joining (A-EJ), are involved in V(D)J recombination and chromosome translocation. Previous studies reported distinct repair mechanisms for chromosome translocation, with NHEJ involved in human and A-EJ in mice predominantly. NHEJ depends on DNA-PKcs, a critical partner in synapsis formation and downstream component activation. While DNA-PKcs inhibition promotes chromosome translocations harboring microhomologies in mice, its synonymous effect in humans is not known. We find partial DNA-PKcs inhibition in human cells leads to increased translocations and the continued involvement of a dampened NHEJ. In contrast, complete DNA-PKcs inhibition substantially increased microhomology-mediated end joining (MMEJ), thus bridging the two different translocation mechanisms between human and mice. Similar to a previous study on Ku70 deletion, DNA-PKcs deletion in G1/G0-phase mouse progenitor B cell lines, significantly impairs V(D)J recombination and generated higher rates of translocations as a consequence of dysregulated coding and signal end joining. Genetic DNA-PKcs inhibition suppresses NHEJ involvement entirely, with repair phenotypically resembling Ku70-deficient A-EJ. In contrast, we find DNA-PKcs necessary in generating the near-excusive MMEJ associated with Lig4 deficiency. Our study underscores DNA-PKcs in suppressing illegitimate chromosome rearrangement while also contributing to MMEJ in both species.\nA Comprehensive Allele Specific Expression Resource for the Equine Transcriptome\nAuthors: Heath, H. D.; Peng, S.; Szmatola, T.; Bellone, R. R.; Kalbfleish, T.; Petersen, J. L.; Finno, C. J.\nScore: 2.6, Published: 2024-01-02 DOI: 10.1101/2023.12.31.573798\nBackgroundAllele-specific expression (ASE) analysis provides a nuanced view of cis-regulatory mechanisms affecting gene expression. ResultsIn this work, we introduce and highlight the significance of an equine ASE analysis, containing integrated long- and short-read RNA sequencing data, along with insight from histone modification data, from four healthy Thoroughbreds (2 mares and 2 stallions) across 9 tissues. ConclusionsThis valuable publicly accessible resource is poised to facilitate investigations into regulatory variation in equine tissues and foster a deeper understanding of the impact of allelic imbalance in equine health and disease at the molecular level.\nComplementation testing identifies causal genes at quantitative trait loci underlying fear related behavior\nAuthors: Chen, P. B.; Chen, R.; LaPierre, N. R. S.; Chen, Z.; Mefford, J.; Marcus, E.; Heffel, M.; Soto, D. C.; Ernst, J.; Luo, C.; Flint, J.\nScore: 2.1, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574060\nKnowing the genes involved in quantitative traits provides a critical entry point to understanding the biological bases of behavior, but there are very few examples where the pathway from genetic locus to behavioral change is known. Here we address a key step towards that goal by deploying a test that directly queries whether a gene mediates the effect of a quantitative trait locus (QTL). To explore the role of specific genes in fear behavior, we mapped three fear-related traits, tested fourteen genes at six QTLs, and identified six genes. Four genes, Lsamp, Ptprd, Nptx2 and Sh3gl, have known roles in synapse function; the fifth gene, Psip1, is a transcriptional co-activator not previously implicated in behavior; the sixth is a long non-coding RNA 4933413L06Rik with no known function. Single nucleus transcriptomic and epigenetic analyses implicated excitatory neurons as likely mediating the genetic effects. Surprisingly, variation in transcriptome and epigenetic modalities between inbred strains occurred preferentially in excitatory neurons, suggesting that genetic variation is more permissible in excitatory than inhibitory neuronal circuits. Our results open a bottleneck in using genetic mapping of QTLs to find novel biology underlying behavior and prompt a reconsideration of expected relationships between genetic and functional variation.\nRewriting regulatory DNA to dissect and reprogram gene expression\nAuthors: Martyn, G. E.; Montgomery, M. T.; Jones, H.; Guo, K.; Doughty, B. R.; Linder, J.; Chen, Z.; Cochran, K.; Lawrence, K. A.; Munson, G.; Pampari, A.; Fulco, C. P.; Kelley, D. R.; Lander, E. S.; Kundaje, A.; Engreitz, J. M.\nScore: 53.9, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572268\nRegulatory DNA sequences within enhancers and promoters bind transcription factors to encode cell type-specific patterns of gene expression. However, the regulatory effects and programmability of such DNA sequences remain difficult to map or predict because we have lacked scalable methods to precisely edit regulatory DNA and quantify the effects in an endogenous genomic context. Here we present an approach to measure the quantitative effects of hundreds of designed DNA sequence variants on gene expression, by combining pooled CRISPR prime editing with RNA fluorescence in situ hybridization and cell sorting (Variant-FlowFISH). We apply this method to mutagenize and rewrite regulatory DNA sequences in an enhancer and the promoter of PPIF in two immune cell lines. Of 672 variant-cell type pairs, we identify 497 that affect PPIF expression. These variants appear to act through a variety of mechanisms including disruption or optimization of existing transcription factor binding sites, as well as creation of de novo sites. Disrupting a single endogenous transcription factor binding site often led to large changes in expression (up to -40% in the enhancer, and -50% in the promoter). The same variant often had different effects across cell types and states, demonstrating a highly tunable regulatory landscape. We use these data to benchmark performance of sequence-based predictive models of gene regulation, and find that certain types of variants are not accurately predicted by existing models. Finally, we computationally design 185 small sequence variants ([\u0026le;]10 bp) and optimize them for specific effects on expression in silico. 84% of these rationally designed edits showed the intended direction of effect, and some had dramatic effects on expression (-100% to +202%). Variant-FlowFISH thus provides a powerful tool to map the effects of variants and transcription factor binding sites on gene expression, test and improve computational models of gene regulation, and reprogram regulatory DNA.\n",
  "wordCount" : "2567",
  "inLanguage": "en",
  "datePublished": "2024-01-07T10:38:59Z",
  "dateModified": "2024-01-07T10:38:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570868">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570868" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570868">
        <p class="paperTitle">pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570868" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570868" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.</p>
        <p class="info">Score: 58.7, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570868' target='https://doi.org/10.1101/2023.12.08.570868'> 10.1101/2023.12.08.570868</a></p>
        <p class="abstract">Standard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis. However, while the phiC31 system functions in zebrafish, validated loci carrying attP-based landing or safe harbor sites suitable for universal transgenesis applications in zebrafish have not been established. Here, using CRISPR-Cas9, we converted two well-validated single insertion Tol2-based zebrafish transgenes with long-standing genetic stability into two attP landing sites, called phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET). Generating fluorescent reporters, loxP-based Switch lines, CreERT2 drivers, and gene-regulatory variant reporters in the pIGLET14a and pIGLET24b landing site alleles, we document their suitability for transgenesis applications across cell types and developmental stages. For both landing sites, we routinely achieve 25-50% germline transmission of targeted transgene integrations, drastically reducing the number of required animals and necessary resources to generate individual transgenic lines. We document that phiC31 integrase-based transgenesis into pIGLET14a and pIGLET24b reproducibly results in representative reporter expression patterns in injected F0 zebrafish embryos suitable for enhancer discovery and qualitative and quantitative comparison of gene-regulatory element variants. Taken together, our new phiC31 integrase-based transgene landing sites establish reproducible, targeted zebrafish transgenesis for numerous applications while greatly reducing the workload of generating new transgenic zebrafish lines.

SHORT ABSTRACTTargeted transgenesis into pre-established, &#34;safe harbor,&#34; landing sites remains a missing technique in zebrafish. Here, we established phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET) by CRISPR-Cas9-based conversion of two previously validated Tol2 transgenes into attP sites. phiC31-mediated transgenesis into our pIGLET14a and pIGLET24b attP landing sites results in 25-50% germline transmission efficiency and quantifiable transgene activity. Our landing sites are suitable for reproducible, routine zebrafish transgenesis for diverse applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573724">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573724" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573724">
        <p class="paperTitle">Plant breeding simulations with AlphaSimR</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573724" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573724" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bancic, J.; Greenspoon, P.; Gaynor, C. R.; Gorjanc, G.</p>
        <p class="info">Score: 10.7, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573724' target='https://doi.org/10.1101/2023.12.30.573724'> 10.1101/2023.12.30.573724</a></p>
        <p class="abstract">Plant breeding plays a crucial role in the development of high-performing crop varieties that meet the demands of society. Emerging breeding techniques offer the potential to improve the precision and efficiency of plant breeding programs; however, their optimal implementation requires refinement of existing breeding programs or the design of new ones. Stochastic simulations are a cost-effective solution for testing and optimizing new breeding strategies. The aim of this paper is to provide an introduction to stochastic simulation with software AlphaSimR for plant breeding students, researchers, and experienced breeders. We present an overview of how to use the software and provide an introductory AlphaSimR vignette as well as complete AlphaSimR scripts of breeding programs for self-pollinated, clonal, and cross-pollinated plants, including relevant breeding techniques, such as backcrossing, speed breeding, genomic selection, index selection, and others. Our objective is to provide a foundation for understanding and utilizing simulation software, enabling readers to adapt the provided scripts for their own use or even develop completely new plant breeding programs. By incorporating simulation software into plant breeding education and practice, the next generation of plant breeders will have a valuable tool in their quest to provide sustainable and nutritious food sources for a growing population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574288">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574288" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574288">
        <p class="paperTitle">Detailed mechanisms for unintended large DNA deletions with CRISPR, base editors, and prime editors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574288" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574288" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hwang, G.-h.; Kim, S.-H.; Oh, M.; Kim, S.; Habib, O.; Jang, H.-K.; Kim, H. S.; Kim, C. H.; Kim, S.; Bae, S.</p>
        <p class="info">Score: 7.2, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574288' target='https://doi.org/10.1101/2024.01.04.574288'> 10.1101/2024.01.04.574288</a></p>
        <p class="abstract">CRISPR-Cas9 nucleases are versatile tools for genetic engineering cells and function by producing targeted double-strand breaks (DSBs) in the DNA sequence. However, the unintended production of large deletions (&gt;100 bp) represents a challenge to the effective application of this genome-editing system. We optimized a long-range amplicon sequencing system and developed a k-mer sequence-alignment algorithm to simultaneously detect small DNA alteration events and large DNA deletions. With this workflow, we determined that CRISPR-Cas9 induced large deletions at varying frequencies in cancer cell lines, stem cells, and primary T cells. With CRISPR interference screening, we determined that end resection and the subsequent TMEJ [DNA polymerase theta-mediated end joining] repair process produce most large deletions. Furthermore, base editors and prime editors also generated large deletions despite employing mutated Cas9 nickases that produce single-strand breaks. Our findings reveal an important limitation of current genome-editing tools and identify strategies for mitigating unwanted large deletion events.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573895">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573895" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573895">
        <p class="paperTitle">Rare genetic variation in Fibronectin 1 (FN1) protects against APOEe4 in Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573895" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573895" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhattarai, P.; Gunasekaran, T. I.; Reyes-Dumeyer, D.; Juelich, D.; Tayran, H.; Yilmaz, E.; Flaherty, D.; Rafael, L.; Martin, M.; Diones, R.; Patricia, R.; Ertekin-Taner, N.; Teich, A. F.; Dennis, D. W.; Scott, H.; Mayeux, R.; Kizil, C.; Vardarajan, B. N.</p>
        <p class="info">Score: 4.8, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573895' target='https://doi.org/10.1101/2024.01.02.573895'> 10.1101/2024.01.02.573895</a></p>
        <p class="abstract">The risk of developing Alzheimers disease (AD) significantly increases in individuals carrying the APOE{varepsilon}4 allele. Elderly cognitively healthy individuals with APOE{varepsilon}4 also exist, suggesting the presence of cellular mechanisms that counteract the pathological effects of APOE{varepsilon}4; however, these mechanisms are unknown. We hypothesized that APOE{varepsilon}4 carriers without dementia might carry genetic variations that could protect them from developing APOE{varepsilon}4-mediated AD pathology. To test this, we leveraged whole genome sequencing (WGS) data in National Institute on Aging Alzheimers Disease Family Based Study (NIA-AD FBS), Washington Heights/Inwood Columbia Aging Project (WHICAP), and Estudio Familiar de Influencia Genetica en Alzheimer (EFIGA) cohorts and identified potentially protective variants segregating exclusively among unaffected APOE{varepsilon}4 carriers. In homozygous unaffected carriers above 70 years old, we identified 510 rare coding variants. Pathway analysis of the genes harboring these variants showed significant enrichment in extracellular matrix (ECM)-related processes, suggesting protective effects of functional modifications in ECM proteins. We prioritized two genes that were highly represented in the ECM-related gene ontology terms, (FN1) and collagen type VI alpha 2 chain (COL6A2) and are known to be expressed at the blood-brain barrier (BBB), for postmortem validation and in vivo functional studies. The FN1 and COL6A2 protein levels were increased at the BBB in APOE{varepsilon}4 carriers with AD. Brain expression of cognitively unaffected homozygous APOE{varepsilon}4 carriers had significantly lower FN1 deposition and less reactive gliosis compared to homozygous APOE{varepsilon}4 carriers with AD, suggesting that FN1 might be a downstream driver of APOE{varepsilon}4-mediated AD-related pathology and cognitive decline. To validate our findings, we used zebrafish models with loss-of-function (LOF) mutations in fn1b - the ortholog for human FN1. We found that fibronectin LOF reduced gliosis, enhanced gliovascular remodeling and potentiated the microglial response, suggesting that pathological accumulation of FN1 could impair toxic protein clearance, which is ameliorated with FN1 LOF. Our study suggests vascular deposition of FN1 is related to the pathogenicity of APOE{varepsilon}4, LOF variants in FN1 may reduce APOE{varepsilon}4-related AD risk, providing novel clues to potential therapeutic interventions targeting the ECM to mitigate AD risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572247">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572247" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572247">
        <p class="paperTitle">Between-group heritability and the status of hereditarianism as an evolutionary science</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572247" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572247" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roseman, C. C.; Bird, K.</p>
        <p class="info">Score: 31.7, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572247' target='https://doi.org/10.1101/2023.12.18.572247'> 10.1101/2023.12.18.572247</a></p>
        <p class="abstract">Hereditarianism is a school of thought that contends there are substantial evolved cognitive and behavioral differences among groups of humans which are both resistant to environmental intervention and are a root cause of differential social outcomes across groups. The relationship of betweengroup heritability [Formula] to within-group heritability [Formula] is one of the key theoretical components of hereditarianism and forms one of the bases for its claim to be an evolutionary science. Here, we examine the relationship between [Formula]and [Formula] and its application to problems in the hereditarian literature from an evolutionary genetic perspective. We demonstrate that the formulation of the relationship between [Formula] and [Formula]used in the hereditarian literature has no evolutionary content. By re-writing the relationship between [Formula] and[Formula] in a novel evolutionary framework, we demonstrate that there is no way to predict[Formula] using[Formula] without considerable additional theory that is absent from the hereditarian literature. Furthermore, we demonstrate that the hereditarian technique that uses[Formula] and [Formula] as a means of judging whether a given difference between groups may be plausibly ameliorated through environmental intervention is mathematically flawed. Lastly, we fill a gap in the hereditarian literature by writing out a means of using [Formula] to predict the absolute difference between groups under a neutral evolutionary model and find that it is much smaller than claimed by hereditarians. In conclusion, we propose a path forward for the study of human variation that moves us past the ill-conceived nature vs. nurture question and allows us to focus on more productive issues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572659">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572659" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572659">
        <p class="paperTitle">The pattern of genetic variability in a core collection of 2,021 cowpea accessions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572659" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572659" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fiscus, C. J.; Herniter, I. A.; Tchamba, M.; Paliwal, R.; Munoz-Amatriain, M.; Roberts, P. A.; Abberton, M.; Alaba, O.; Close, T. J.; Oyatomi, O.; Koenig, D.</p>
        <p class="info">Score: 4.5, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572659' target='https://doi.org/10.1101/2023.12.21.572659'> 10.1101/2023.12.21.572659</a></p>
        <p class="abstract">Cowpea is a highly drought-adapted leguminous crop with great promise for improving agricultural sustainability and food security. Here, we report analyses derived from array-based genotyping of 2,021 accessions constituting a core subset of the worlds largest cowpea collection, held at the International Institute of Tropical Agriculture (IITA) in Ibadan, Nigeria. We used this dataset to examine genetic variation and population structure in worldwide cowpea. We confirm that the primary pattern of population structure is two geographically defined subpopulations origining in West and East Africa, respectively, and that population structure is associated with shifts in phenotypic distribution. Furthermore, we establish the cowpea core collection as a resource for genome-wide association studies by mapping the genetic basis of several phenotypes, with a focus on seed coat pigmentation patterning and color. We anticipate that the genotyped IITA cowpea core collection will serve as a powerful tool for mapping complex traits, facilitating the acceleration of breeding programs to enhance the resilience of this crop in the face of rapid global climate change.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573736">
        <p class="paperTitle">DNA-PKcs Suppresses Illegitimate Chromosome Rearrangements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, J.; Sadeghi, C.; Frock, R. L.</p>
        <p class="info">Score: 3.1, Published: 2024-01-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573736' target='https://doi.org/10.1101/2023.12.30.573736'> 10.1101/2023.12.30.573736</a></p>
        <p class="abstract">Two DNA repair pathways, non-homologous end joining (NHEJ) and alternative end joining (A-EJ), are involved in V(D)J recombination and chromosome translocation. Previous studies reported distinct repair mechanisms for chromosome translocation, with NHEJ involved in human and A-EJ in mice predominantly. NHEJ depends on DNA-PKcs, a critical partner in synapsis formation and downstream component activation. While DNA-PKcs inhibition promotes chromosome translocations harboring microhomologies in mice, its synonymous effect in humans is not known. We find partial DNA-PKcs inhibition in human cells leads to increased translocations and the continued involvement of a dampened NHEJ. In contrast, complete DNA-PKcs inhibition substantially increased microhomology-mediated end joining (MMEJ), thus bridging the two different translocation mechanisms between human and mice. Similar to a previous study on Ku70 deletion, DNA-PKcs deletion in G1/G0-phase mouse progenitor B cell lines, significantly impairs V(D)J recombination and generated higher rates of translocations as a consequence of dysregulated coding and signal end joining. Genetic DNA-PKcs inhibition suppresses NHEJ involvement entirely, with repair phenotypically resembling Ku70-deficient A-EJ. In contrast, we find DNA-PKcs necessary in generating the near-excusive MMEJ associated with Lig4 deficiency. Our study underscores DNA-PKcs in suppressing illegitimate chromosome rearrangement while also contributing to MMEJ in both species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.573798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.573798" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.573798">
        <p class="paperTitle">A Comprehensive Allele Specific Expression Resource for the Equine Transcriptome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.573798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.573798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heath, H. D.; Peng, S.; Szmatola, T.; Bellone, R. R.; Kalbfleish, T.; Petersen, J. L.; Finno, C. J.</p>
        <p class="info">Score: 2.6, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.573798' target='https://doi.org/10.1101/2023.12.31.573798'> 10.1101/2023.12.31.573798</a></p>
        <p class="abstract">BackgroundAllele-specific expression (ASE) analysis provides a nuanced view of cis-regulatory mechanisms affecting gene expression.

ResultsIn this work, we introduce and highlight the significance of an equine ASE analysis, containing integrated long- and short-read RNA sequencing data, along with insight from histone modification data, from four healthy Thoroughbreds (2 mares and 2 stallions) across 9 tissues.

ConclusionsThis valuable publicly accessible resource is poised to facilitate investigations into regulatory variation in equine tissues and foster a deeper understanding of the impact of allelic imbalance in equine health and disease at the molecular level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574060">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574060" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574060">
        <p class="paperTitle">Complementation testing identifies causal genes at quantitative trait loci underlying fear related behavior</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574060" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574060" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, P. B.; Chen, R.; LaPierre, N. R. S.; Chen, Z.; Mefford, J.; Marcus, E.; Heffel, M.; Soto, D. C.; Ernst, J.; Luo, C.; Flint, J.</p>
        <p class="info">Score: 2.1, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574060' target='https://doi.org/10.1101/2024.01.03.574060'> 10.1101/2024.01.03.574060</a></p>
        <p class="abstract">Knowing the genes involved in quantitative traits provides a critical entry point to understanding the biological bases of behavior, but there are very few examples where the pathway from genetic locus to behavioral change is known. Here we address a key step towards that goal by deploying a test that directly queries whether a gene mediates the effect of a quantitative trait locus (QTL). To explore the role of specific genes in fear behavior, we mapped three fear-related traits, tested fourteen genes at six QTLs, and identified six genes. Four genes, Lsamp, Ptprd, Nptx2 and Sh3gl, have known roles in synapse function; the fifth gene, Psip1, is a transcriptional co-activator not previously implicated in behavior; the sixth is a long non-coding RNA 4933413L06Rik with no known function. Single nucleus transcriptomic and epigenetic analyses implicated excitatory neurons as likely mediating the genetic effects. Surprisingly, variation in transcriptome and epigenetic modalities between inbred strains occurred preferentially in excitatory neurons, suggesting that genetic variation is more permissible in excitatory than inhibitory neuronal circuits. Our results open a bottleneck in using genetic mapping of QTLs to find novel biology underlying behavior and prompt a reconsideration of expected relationships between genetic and functional variation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572268">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572268" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572268">
        <p class="paperTitle">Rewriting regulatory DNA to dissect and reprogram gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572268" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572268" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martyn, G. E.; Montgomery, M. T.; Jones, H.; Guo, K.; Doughty, B. R.; Linder, J.; Chen, Z.; Cochran, K.; Lawrence, K. A.; Munson, G.; Pampari, A.; Fulco, C. P.; Kelley, D. R.; Lander, E. S.; Kundaje, A.; Engreitz, J. M.</p>
        <p class="info">Score: 53.9, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572268' target='https://doi.org/10.1101/2023.12.20.572268'> 10.1101/2023.12.20.572268</a></p>
        <p class="abstract">Regulatory DNA sequences within enhancers and promoters bind transcription factors to encode cell type-specific patterns of gene expression. However, the regulatory effects and programmability of such DNA sequences remain difficult to map or predict because we have lacked scalable methods to precisely edit regulatory DNA and quantify the effects in an endogenous genomic context. Here we present an approach to measure the quantitative effects of hundreds of designed DNA sequence variants on gene expression, by combining pooled CRISPR prime editing with RNA fluorescence in situ hybridization and cell sorting (Variant-FlowFISH). We apply this method to mutagenize and rewrite regulatory DNA sequences in an enhancer and the promoter of PPIF in two immune cell lines. Of 672 variant-cell type pairs, we identify 497 that affect PPIF expression. These variants appear to act through a variety of mechanisms including disruption or optimization of existing transcription factor binding sites, as well as creation of de novo sites. Disrupting a single endogenous transcription factor binding site often led to large changes in expression (up to -40% in the enhancer, and -50% in the promoter). The same variant often had different effects across cell types and states, demonstrating a highly tunable regulatory landscape. We use these data to benchmark performance of sequence-based predictive models of gene regulation, and find that certain types of variants are not accurately predicted by existing models. Finally, we computationally design 185 small sequence variants ([&amp;le;]10 bp) and optimize them for specific effects on expression in silico. 84% of these rationally designed edits showed the intended direction of effect, and some had dramatic effects on expression (-100% to &#43;202%). Variant-FlowFISH thus provides a powerful tool to map the effects of variants and transcription factor binding sites on gene expression, test and improve computational models of gene regulation, and reprogram regulatory DNA.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
