<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Estimation of indirect genetic effects and heritability under assortative mating
Authors: Young, A. S.
Score: 34.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458
Both direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects -- effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) -- can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Estimation of indirect genetic effects and heritability under assortative mating
Authors: Young, A. S.
Score: 34.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458
Both direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects -- effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) -- can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-16T10:39:40+00:00" />
<meta property="article:modified_time" content="2023-07-16T10:39:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Estimation of indirect genetic effects and heritability under assortative mating
Authors: Young, A. S.
Score: 34.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458
Both direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects -- effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) -- can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Estimation of indirect genetic effects and heritability under assortative mating\nAuthors: Young, A. S.\nScore: 34.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458\nBoth direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects -- effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) -- can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium.",
  "keywords": [
    
  ],
  "articleBody": " Estimation of indirect genetic effects and heritability under assortative mating\nAuthors: Young, A. S.\nScore: 34.5, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548458\nBoth direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects -- effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) -- can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium. We generalize classical theory to derive a decomposition of the equilibrium phenotypic variance in terms of direct and indirect genetic effect components. We extend this theory to show that popular methods for estimating indirect genetic effects or genetic nurture through analysis of parental and offspring polygenic predictors (called polygenic indices or scores -- PGIs or PGSs) are substantially biased by assortative mating. We propose an improved method for estimating indirect genetic effects while accounting for assortative mating that can also correct heritability estimates for bias due to assortative mating. We validate our method in simulations and apply it to PGIs for height and educational attainment (EA), estimating that the equilibrium heritability of height is 0.699 (S.E. = 0.075) and finding no evidence for indirect genetic effects on height. We estimate a very high correlation between parents underlying genetic components for EA, 0.755 (S.E. = 0.035), which is inconsistent with twin based estimates of the heritability of EA, possibly due to confounding in the EA PGI and/or in twin studies. We implement our method in the software package snipar, enabling researchers to apply the method to data including observed and/or imputed parental genotypes. We provide a theoretical framework for understanding the results of PGI analyses and a practical methodology for estimating heritability and indirect genetic effects while accounting for assortative mating.\nSocial and genetic diversity in first farmers of central Europe\nAuthors: Gelabert, P.; Bickle, P.; Hofmann, D.; Teschler-Nicola, M.; Anders, A.; Huang, X.; Olalde, I.; Fournier, R.; Ringbauer, H.; Akbari, A.; Cheronet, O.; Lazaridis, I.; Broomandkhoshbacht, N.; Fernandes, D. M.; Buttinger, K.; Callan, K.; Candilio, F.; Bravo, G.; Curtis, E.; Ferry, M.; Keating, D.; Freilich, S.; Kearns, A.; Harney, E.; Lawson, A. M.; Mandl, K.; Michel, M.; Oberreiter, V.; Oppenheimer, J.; Sawyer, S.; Schattke, C.; Ozdogan, K. T. T.; Qiu, L.; Workman, N.; Zalzala, F.; Mallick, S.; Mah, M.; Micco, A.; Pieler, F.; Pavuk, J.; Lazar, C.; Skrivanko, M. K.; Slaus, M.; Bedic, Z.; Novotny, Score: 28.6, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548126\nThe Linearbandkeramik (LBK) Neolithic communities were the first to spread farming across large parts of central Europe, settling fertile regions from Ukraine to France during the second half of the 6th millennium BCE. The LBK had a high degree of material culture uniformity, albeit with regional differences in settlement patterns, subsistence, and mortuary practices. To date, ancient DNA data from LBK individuals have been generated for a limited number of locations and often in small sample sizes, making it challenging to study variation within and across sites. We report genome-wide data for 178 LBK individuals, from the Alfold Linearbankeramik Culture (ALPC) eastern LBK site of Polgar-Ferenci-hat in Hungary, the western LBK site of Nitra in Slovakia, and the enclosed western LBK settlement and massacre site of Schletz in Austria, as well as 42 LBK individuals from 18 other sites. We also report genome-wide data for 28 Early Neolithic Koros and Star[c]evo individuals from 13 sites, viewed as the predecessors of the LBK. We observe a higher percentage of western hunter-gatherer (WHG) admixture among individuals in the eastern LBK than in the far more widely distributed western LBK, showing that these two archaeologically distinct cultures also had different genetic trajectories. Most WHG-farmer mixture occurred just before the dawn of the LBK culture and there is no evidence that the WHG ancestry came systematically more from males or females. However, we do find strong genetic evidence for patrilocality among the LBK, extending previous findings based on isotopic analysis, with more genetic structure across sites on the male than on the female line, and a higher rate of within-site relatives for males. At Schletz we detect almost no first-degree relatives despite reporting data from almost every skeleton present at the site, showing that this massacre involved people from a large population, not a small community.\nChromatin states contribute to coordinated allelic transcriptional bursting to drive iPSC reprogramming\nAuthors: Parichitran, A.; Naik, H.; Naskar, A.; Bammidi, L. S.; GAYEN, S.\nScore: 16.1, Published: 2023-07-13 DOI: 10.1101/2023.07.13.548864\nMolecular mechanisms behind the reprogramming of somatic cells to induced pluripotent stem cells (iPSC) remain poorly understood. While dynamic changes in gene expression are considered to drive reprogramming, the contribution of individual alleles of genes to reprogramming remains unexplored. It is thought that two alleles of a gene can transcribe independently or coordinatedly, which in turn can lead to temporal expression heterogeneity with potentially distinct impacts on cell fate. Here, we profiled genome-wide transcriptional burst kinetics with an allelic resolution during the reprogramming of mouse embryonic fibroblast (MEF) to iPSC. We show that many genes involved in iPSC reprogramming pathways exhibit bursty expression and contribute to dynamic autosomal random monoallelic expression (aRME). Moreover, we find that the degree of coordination of allelic bursting differs among genes and changes dynamically during iPSC reprogramming. Importantly, we show that alleles of many reprogramming-related genes burst in a highly coordinated fashion. ATAC-seq analysis revealed that coordination of allelic bursting is linked to allelic chromatin accessibility. Consistently, we show that highly coordinated genes are enriched with chromatin accessibility regulators such as H3K36me3, H3K27ac, histone variant H3.3 and BRD4. Collectively, our study demonstrates that chromatin states contribute to coordinated allelic bursting to fine-tune the expression of genes involved in iPSC reprogramming and provides insights into the implications of allelic bursting coordination in cell fate specification.\nsciMET-cap: High-throughput single-cell methylation analysis with a reduced sequencing burden\nAuthors: Acharya, S.; Nichols, R. V.; Rylaarsdam, L. E.; O'Connell, B. L.; Braun, T. P.; Adey, A. C.\nScore: 9.9, Published: 2023-07-14 DOI: 10.1101/2023.07.12.548718\nDNA methylation is a key component of the mammalian epigenome, playing a regulatory role in development, disease, and other processes. Robust, high-throughput single-cell DNA methylation assays are now possible (sciMET); however, the genome-wide nature of DNA methylation results in a high sequencing burden per cell. Here, we leverage target enrichment with sciMET to capture sufficient information per cell for cell type assignment using substantially fewer sequence reads (sciMET-cap). Sufficient off-target coverage further enables the production of near-complete methylomes for individual cell types. We characterize sciMET-cap on human PBMCs and brain (middle frontal gyrus).\nCRIMP: A CRISPR/Cas9 Insertional Mutagenesis Protocol and the CRIMP Toolkit\nAuthors: Miles, L. B.; Calcinotto, V.; Sonntag, C.; Lee, C.; Bryson-Richardson, R. J.\nScore: 9.0, Published: 2023-07-14 DOI: 10.1101/2023.07.13.548929\nWe developed a highly efficient targeted insertional mutagenesis system, CRIMP, and an associated plasmid toolkit, CRIMPkit, that disrupts native gene expression by inducing complete transcriptional termination to produce null mutant alleles without inducing genetic compensation. The CRIMPkit contains over 30 ready-to-use plasmid vectors to allow easy and complete mutagenesis of any gene in any reading frame without requiring custom sequences, modification, or subcloning, and also provides a fluorescent readout of successfully mutagenised fish.\nDISCOVERY AND VALIDATION OF GENES DRIVING DRUG-INTAKE AND RELATED BEHAVIORAL TRAITS IN MICE\nAuthors: Roy, T.; Bubier, J. A.; Dickson, P. E.; Wilcox, T.; Ndukum, J.; Clark, J. W.; Rizzo, S. J. S.; Crabbe, J. C.; Denegre, J.; Svenson, K. L.; Braun, R. E.; Kumar, V.; Murray, S. A.; White, J. K.; Philip, V.; Chesler, E.\nScore: 6.1, Published: 2023-07-10 DOI: 10.1101/2023.07.09.548280\nSubstance use disorders (SUDs) are heritable disorders characterized by compulsive drug use, but the biological mechanisms driving addiction remain largely unknown. Genetic correlations reveal that predisposing drug-naive phenotypes, including anxiety, depression, novelty preference, and sensation seeking, are predictive of drug-use phenotypes, implicating shared genetic mechanisms. Because of this relationship, high-throughput behavioral screening of predictive phenotypes in knockout (KO) mice allows efficient discovery of genes likely to be involved in drug use. We used this strategy in two rounds of screening in which we identified 33 drug-use candidate genes and ultimately validated the perturbation of 22 of these genes as causal drivers of substance intake. In our initial round of screening, we employed the two-bottle-choice paradigms to assess alcohol, methamphetamine, and nicotine intake. We identified 19 KO strains that were extreme responders on at least one predictive phenotype. Thirteen of the 19 gene deletions (68%) significantly affected alcohol use three methamphetamine use, and two both. In the second round of screening, we employed a multivariate approach to identify outliers and performed validation using methamphetamine two-bottle choice and ethanol drinking-in-the-dark protocols. We identified 15 KO strains that were extreme responders across the predisposing drug-naive phenotypes. Eight of the 15 gene deletions (53%) significantly affected intake or preference for three alcohol, eight methamphetamine or three both (3). We observed multiple relations between predisposing behaviors and drug intake, revealing many distinct biobehavioral processes underlying these relationships. The set of mouse models identified in this study can be used to characterize these addiction-related processes further.\nPolygenic Risk Score Associates with Atherosclerotic Plaque Characteristics at Autopsy\nAuthors: Cornelissen, A.; Gadhoke, N.; Ryan, K. A.; Hodonsky, C. J.; Mitchell, R. N.; Bihlmeyer, N. A.; Duong, T.; Chen, Z.; Dikongue, A.; Sakamoto, A.; Sato, Y.; Kawakami, R.; Mori, M.; Kawai, K.; Fernandez, R.; Ghosh, S. K. B.; Braumann, R.; Abebe, B.; Kutys, R.; Kutyna, M. D.; Romero, M. E.; Kolodgie, F.; Miller, C. L.; Hong, C. C.; Grove, M. L.; Brody, J. A.; Sotoodehnia, N.; Arking, D. E.; Schunkert, H.; Mitchell, B. D.; Guo, L.; Virmani, R.; Finn, A. V.\nScore: 5.4, Published: 2023-07-09 DOI: 10.1101/2023.07.05.547891\nBackgroundPolygenic risk scores (PRS) for coronary artery disease (CAD) potentially improve cardiovascular risk prediction. However, their relationship with histopathologic features of CAD has never been examined systematically. MethodsFrom 4,327 subjects referred to CVPath by the State of Maryland Office Chief Medical Examiner (OCME) for sudden death between 1994 and 2015, 2,455 cases were randomly selected for genotyping. We generated PRS from 291 known CAD risk loci. Detailed histopathologic examination of the coronary arteries was performed in all subjects. The primary study outcome measurements were histopathologic plaque features determining severity of atherosclerosis, including %stenosis, calcification, thin-cap fibroatheromas (TCFA), and thrombotic CAD. ResultsAfter exclusion of cases with insufficient DNA sample quality or with missing data, 954 cases (mean age 48.8{+/-}14.7; 75.7% men) remained in the final study cohort. Subjects in the highest PRS quintile exhibited more severe atherosclerosis compared to subjects in the lowest quintile, with greater %stenosis (80.3%{+/-}27.0% vs. 50.4%{+/-}38.7%; adjusted p\u003c0.001) and a higher frequency of calcification (69.6% vs. 35.8%; adjusted p=0.004) and TCFAs (26.7% vs. 9.5%; adjusted p=0.007). Even after adjustment for traditional CAD risk factors subjects within the highest PRS quintile had higher odds of severe atherosclerosis (i.e., [\u0026ge;]75% stenosis; adjusted OR 3.77; 95%CI 2.10-6.78; p\u003c0.001) and plaque rupture (adjusted OR 4.05; 95%CI 2.26-7.24; p\u003c0.001). Moreover, subjects within the highest quintile had higher odds of CAD-associated cause of death, especially among those aged 50 years and younger (adjusted OR 4.08; 95%CI 2.01-8.30; p\u003c0.001). No associations were observed with plaque erosion. ConclusionsThis is the first autopsy study investigating associations between PRS and atherosclerosis severity at the histopathologic level in subjects with sudden death. Our pathological analysis suggests PRS correlates with plaque burden and features of advanced atherosclerosis and may be useful as a method for CAD risk stratification, especially in younger subjects. HighlightsO_LIIn this autopsy study including 954 subjects within the CVPath Sudden Death Registry, high PRS correlated with plaque burden and atherosclerosis severity. C_LIO_LIThe PRS showed differential associations with plaque rupture and plaque erosion, suggesting different etiologies to these two causes of thrombotic CAD. C_LIO_LIPRS may be useful for risk stratification, particularly in the young. Further examination of individual risk loci and their association with plaque morphology may help understand molecular mechanisms of atherosclerosis, potentially revealing new therapy targets of CAD. C_LI Graphic AbstractA polygenic risk score, generated from 291 known CAD risk loci, was assessed in 954 subjects within the CVPath Sudden Death Registry. Histopathologic examination of the coronary arteries was performed in all subjects. Subjects in the highest PRS quintile exhibited more severe atherosclerosis as compared to subjects in the lowest quintile, with a greater plaque burden, more calcification, and a higher frequency of plaque rupture.\nValidation of human telomere length trans-ancestry meta-analysis association signals identifies POP5 and KBTBD6 as novel human telomere length regulation genes\nAuthors: Keener, R.; Chhetri, S. B.; Connelly, C. J.; Taub, M. A.; Conomos, M. P.; Weinstock, J. S.; Ni, B.; Strober, B. J.; Aslibekyan, S.; Auer, P. L.; Barwick, L.; Becker, L. C.; Blangero, J.; Bleecker, E. R.; Brody, J. A.; Cade, B. E.; Celedon, J. C.; Chang, Y.-C.; Cupples, L. A.; Custer, B.; Freedman, B. I.; Gladwin, M. T.; Heckbert, S. R.; Hou, L.; Irvin, M. R.; Isasi, C. R.; Johnsen, J. M.; Kenny, E. E.; Kooperberg, C.; Minster, R. L.; Nekhai, S.; Pankratz, N.; Peyser, P. A.; Rotter, J. I.; Taylor, K. D.; Telen, M. J.; Wu, B.; Yanek, L. R.; Yang, I. V.; Albert, C.; Arnett, D. K.; Ashley-Koch, A.\nScore: 5.2, Published: 2023-07-14 DOI: 10.1101/2023.07.12.548702\nTelomere length genome-wide association studies (GWAS) have become well-powered to detect novel genes in telomere length regulation. However, no prior work has validated these putative novel genes to confirm the contribution of GWAS loci to telomere length regulation. We conducted a trans-ancestry meta-analysis of 211,369 individuals. Through enrichment analyses of chromatin state and cell-type heritability we identified blood and immune cells as the most relevant cell type to examine telomere length association signals. We validated specific GWAS associations by overexpressing KBTBD6, a component of an E3 ubiquitin ligase complex, and POP5, a component of the Ribonuclease P/MRP complex, and demonstrating that both lengthened telomeres as predicted by our statistical analyses. CRISPR/Cas9 deletion of the predicted causal regions of these association peaks in K562 immortalized blood cells reduced expression of these genes, demonstrating that these loci are related to transcriptional regulation of KBTBD6 and POP5, respectively. Together our results demonstrate the utility of telomere length GWAS in the identification of novel telomere length regulation mechanisms and highlight the importance of the proteasome-ubiquitin pathway in telomere length regulation.\nHigh throughput functional profiling of genes at intraocular pressure loci reveals distinct networks for glaucoma\nAuthors: Greatbatch, C. J.; Lu, Q.; Hung, S.; Wing, K.; Liang, H.; Han, X.; Zhou, T.; Siggs, O.; Mackey, D.; Cook, A.; Senabouth, A.; Liu, G.-S.; Craig, J.; MacGregor, S.; Powell, J.; Hewitt, A.\nScore: 2.8, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548340\nINTRODUCTIONPrimary open angle glaucoma (POAG) is a leading cause of blindness globally. Characterised by progressive retinal ganglion cell degeneration, the precise pathogenesis remains unknown. Genome-wide association studies (GWAS) have uncovered many genetic variants associated with elevated intraocular pressure (IOP), one of the key risk factors for POAG. This study sought to investigate the morphological and transcriptional consequences of perturbation of key genes at IOP loci in trabecular meshwork cell (TMC); the cellular regulators of IOP. We aimed to identify genetic and morphological variation that can be attributed to TMC dysfunction and raised IOP in POAG. METHODS62 genes across 55 loci were knocked-out in a primary human TMC line. Each knockout group, including five non-targeting control groups, underwent single-cell RNA-sequencing (scRNA-seq) for differentially-expressed gene (DEG) analysis. Multiplexed fluorescent staining of key organelles, was coupled with high-throughput microscopy for single-cell morphological analysis using CellProfiler image analysis. RESULTSAcross many of the individual gene knockouts scRNA-seq highlighted genes relating to matrix metalloproteinases and interferon-induced proteins. Our work has prioritised genes at four loci of interest to identify gene knockouts that may contribute to the pathogenesis of POAG, including ANGPTL2, LMX1B, CAV1, and KREMEN1. Three genetic networks of gene knockouts with similar transcriptomic profiles were identified (ABO / CAV1 / MYOC, ANGPT2 / PKHD1 / TNS1 / TXNRD2, and CAPZA1 / KALRN / LMO7 / PLEKHA7 / GNB1L / TEX41), suggesting a synergistic function in trabecular meshwork cell physiology. TEK knockout caused significant upregulation of nuclear granularity on morphological analysis, whilst knockout of TRIOBP, TMCO1 and PLEKHA7 increased granularity and intensity of actin and the cell-membrane. CONCLUSIONHigh throughput analysis of cellular structure and function through multiplex fluorescent single-cell analysis and scRNA-seq assays enabled the direct study of genetic perturbations at the single-cell resolution. This work provides a framework for investigating the role of genes in the pathogenesis of glaucoma and heterogenous diseases with a strong genetic basis.\nRecombination map tailored to Native Hawaiians reduces false positive findings in genomic scans of positive selection\nAuthors: Dinh, B. L.; Tang, E.; Taparra, K.; Nakatsuka, N.; Chen, F.; Chiang, C. W. K.\nScore: 2.2, Published: 2023-07-14 DOI: 10.1101/2023.07.12.548735\nRecombination events establish the patterns of haplotypic structure in a population and estimates of recombination rates are used in several downstream population and statistical genetic analyses. However, the recombination landscape of populations not represented in the 1000 Genomes Project (1KGP) is poorly understood. Using suboptimal maps from distantly related populations reduces the efficacy of extending genomic analyses to underrepresented populations, including Native Hawaiians. To overcome this challenge, we constructed linkage disequilibrium (LD)- and Identity-by-Descent (IBD)-based recombination maps specifically for Native Hawaiians. We created and characterized two recombination maps from our study sample: one based on a random subset of our sample reflecting the current admixed state of Native Hawaiians (NH map), and one based on a subset of individuals with enriched Polynesian ancestries (PNS map) with potential to be used for less admixed Polynesian populations such as the Samoans. We found the recombination landscape to be less correlated with those from other 1KGP populations (e.g. Spearman's rho = 0.79 between PNS and CEU (Utah residents with Northern and Western European ancestry) compared to 0.92 between YRI (Yoruba in Ibadan, Nigeria) and CEU at 50 kb resolution. PNS shared the fewest recombination hotspots with other populations from 1KGP (e.g. 8% of hotspots shared between PNS and CEU compared to 27% of hotspots shared between YRI and CEU). We found that downstream analysis in the Native Hawaiian population, such as local ancestry inference, imputation, and IBD segment and relatedness detections, would achieve similar efficacy when using the NH map compared to an omnibus map. However, for genome scans of adaptive loci using integrated haplotype scores, we found several loci with apparent genome-wide significant signals (|Z-score| \u003e 4) in Native Hawaiians that would not have been significant when analyzed using NH-specific maps. Therefore, population-specific recombination maps improve the robustness of haplotype-based statistics and help us better characterize the evolutionary history that may underlie Native Hawaiian-specific health conditions that persist today.\n",
  "wordCount" : "3076",
  "inLanguage": "en",
  "datePublished": "2023-07-16T10:39:40Z",
  "dateModified": "2023-07-16T10:39:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on July 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548458">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548458" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548458">
        <p class="paperTitle">Estimation of indirect genetic effects and heritability under assortative mating</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548458" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548458" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Young, A. S.</p>
        <p class="info">Score: 34.5, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548458' target='https://doi.org/10.1101/2023.07.10.548458'> 10.1101/2023.07.10.548458</a></p>
        <p class="abstract">Both direct genetic effects (effects of alleles in an individual on that individual) and indirect genetic effects -- effects of alleles in an individual (e.g. parents) on another individual (e.g. offspring) -- can contribute to phenotypic variation and genotype-phenotype associations. Here, we consider a phenotype affected by direct and parental indirect genetic effects under assortative mating at equilibrium. We generalize classical theory to derive a decomposition of the equilibrium phenotypic variance in terms of direct and indirect genetic effect components. We extend this theory to show that popular methods for estimating indirect genetic effects or  genetic nurture through analysis of parental and offspring polygenic predictors (called polygenic indices or scores -- PGIs or PGSs) are substantially biased by assortative mating. We propose an improved method for estimating indirect genetic effects while accounting for assortative mating that can also correct heritability estimates for bias due to assortative mating. We validate our method in simulations and apply it to PGIs for height and educational attainment (EA), estimating that the equilibrium heritability of height is 0.699 (S.E. = 0.075) and finding no evidence for indirect genetic effects on height. We estimate a very high correlation between parents underlying genetic components for EA, 0.755 (S.E. = 0.035), which is inconsistent with twin based estimates of the heritability of EA, possibly due to confounding in the EA PGI and/or in twin studies. We implement our method in the software package snipar, enabling researchers to apply the method to data including observed and/or imputed parental genotypes. We provide a theoretical framework for understanding the results of PGI analyses and a practical methodology for estimating heritability and indirect genetic effects while accounting for assortative mating.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548126">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548126" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548126">
        <p class="paperTitle">Social and genetic diversity in first farmers of central Europe</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548126" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548126" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gelabert, P.; Bickle, P.; Hofmann, D.; Teschler-Nicola, M.; Anders, A.; Huang, X.; Olalde, I.; Fournier, R.; Ringbauer, H.; Akbari, A.; Cheronet, O.; Lazaridis, I.; Broomandkhoshbacht, N.; Fernandes, D. M.; Buttinger, K.; Callan, K.; Candilio, F.; Bravo, G.; Curtis, E.; Ferry, M.; Keating, D.; Freilich, S.; Kearns, A.; Harney, E.; Lawson, A. M.; Mandl, K.; Michel, M.; Oberreiter, V.; Oppenheimer, J.; Sawyer, S.; Schattke, C.; Ozdogan, K. T. T.; Qiu, L.; Workman, N.; Zalzala, F.; Mallick, S.; Mah, M.; Micco, A.; Pieler, F.; Pavuk, J.; Lazar, C.; Skrivanko, M. K.; Slaus, M.; Bedic, Z.; Novotny, </p>
        <p class="info">Score: 28.6, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548126' target='https://doi.org/10.1101/2023.07.07.548126'> 10.1101/2023.07.07.548126</a></p>
        <p class="abstract">The Linearbandkeramik (LBK) Neolithic communities were the first to spread farming across large parts of central Europe, settling fertile regions from Ukraine to France during the second half of the 6th millennium BCE. The LBK had a high degree of material culture uniformity, albeit with regional differences in settlement patterns, subsistence, and mortuary practices. To date, ancient DNA data from LBK individuals have been generated for a limited number of locations and often in small sample sizes, making it challenging to study variation within and across sites. We report genome-wide data for 178 LBK individuals, from the Alfold Linearbankeramik Culture (ALPC) eastern LBK site of Polgar-Ferenci-hat in Hungary, the western LBK site of Nitra in Slovakia, and the enclosed western LBK settlement and massacre site of Schletz in Austria, as well as 42 LBK individuals from 18 other sites. We also report genome-wide data for 28 Early Neolithic Koros and Star[c]evo individuals from 13 sites, viewed as the predecessors of the LBK. We observe a higher percentage of western hunter-gatherer (WHG) admixture among individuals in the eastern LBK than in the far more widely distributed western LBK, showing that these two archaeologically distinct cultures also had different genetic trajectories. Most WHG-farmer mixture occurred just before the dawn of the LBK culture and there is no evidence that the WHG ancestry came systematically more from males or females. However, we do find strong genetic evidence for patrilocality among the LBK, extending previous findings based on isotopic analysis, with more genetic structure across sites on the male than on the female line, and a higher rate of within-site relatives for males. At Schletz we detect almost no first-degree relatives despite reporting data from almost every skeleton present at the site, showing that this massacre involved people from a large population, not a small community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548864">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548864" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548864">
        <p class="paperTitle">Chromatin states contribute to coordinated allelic transcriptional bursting to drive iPSC reprogramming</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548864" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548864" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parichitran, A.; Naik, H.; Naskar, A.; Bammidi, L. S.; GAYEN, S.</p>
        <p class="info">Score: 16.1, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548864' target='https://doi.org/10.1101/2023.07.13.548864'> 10.1101/2023.07.13.548864</a></p>
        <p class="abstract">Molecular mechanisms behind the reprogramming of somatic cells to induced pluripotent stem cells (iPSC) remain poorly understood. While dynamic changes in gene expression are considered to drive reprogramming, the contribution of individual alleles of genes to reprogramming remains unexplored. It is thought that two alleles of a gene can transcribe independently or coordinatedly, which in turn can lead to temporal expression heterogeneity with potentially distinct impacts on cell fate. Here, we profiled genome-wide transcriptional burst kinetics with an allelic resolution during the reprogramming of mouse embryonic fibroblast (MEF) to iPSC. We show that many genes involved in iPSC reprogramming pathways exhibit bursty expression and contribute to dynamic autosomal random monoallelic expression (aRME). Moreover, we find that the degree of coordination of allelic bursting differs among genes and changes dynamically during iPSC reprogramming. Importantly, we show that alleles of many reprogramming-related genes burst in a highly coordinated fashion. ATAC-seq analysis revealed that coordination of allelic bursting is linked to allelic chromatin accessibility. Consistently, we show that highly coordinated genes are enriched with chromatin accessibility regulators such as H3K36me3, H3K27ac, histone variant H3.3 and BRD4. Collectively, our study demonstrates that chromatin states contribute to coordinated allelic bursting to fine-tune the expression of genes involved in iPSC reprogramming and provides insights into the implications of allelic bursting coordination in cell fate specification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548718">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548718" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548718">
        <p class="paperTitle">sciMET-cap: High-throughput single-cell methylation analysis with a reduced sequencing burden</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548718" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548718" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Acharya, S.; Nichols, R. V.; Rylaarsdam, L. E.; O&#39;Connell, B. L.; Braun, T. P.; Adey, A. C.</p>
        <p class="info">Score: 9.9, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548718' target='https://doi.org/10.1101/2023.07.12.548718'> 10.1101/2023.07.12.548718</a></p>
        <p class="abstract">DNA methylation is a key component of the mammalian epigenome, playing a regulatory role in development, disease, and other processes. Robust, high-throughput single-cell DNA methylation assays are now possible (sciMET); however, the genome-wide nature of DNA methylation results in a high sequencing burden per cell. Here, we leverage target enrichment with sciMET to capture sufficient information per cell for cell type assignment using substantially fewer sequence reads (sciMET-cap). Sufficient off-target coverage further enables the production of near-complete methylomes for individual cell types. We characterize sciMET-cap on human PBMCs and brain (middle frontal gyrus).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548929">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548929" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548929">
        <p class="paperTitle">CRIMP: A CRISPR/Cas9 Insertional Mutagenesis Protocol and the CRIMP Toolkit</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548929" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548929" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miles, L. B.; Calcinotto, V.; Sonntag, C.; Lee, C.; Bryson-Richardson, R. J.</p>
        <p class="info">Score: 9.0, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548929' target='https://doi.org/10.1101/2023.07.13.548929'> 10.1101/2023.07.13.548929</a></p>
        <p class="abstract">We developed a highly efficient targeted insertional mutagenesis system, CRIMP, and an associated plasmid toolkit, CRIMPkit, that disrupts native gene expression by inducing complete transcriptional termination to produce null mutant alleles without inducing genetic compensation. The CRIMPkit contains over 30 ready-to-use plasmid vectors to allow easy and complete mutagenesis of any gene in any reading frame without requiring custom sequences, modification, or subcloning, and also provides a fluorescent readout of successfully mutagenised fish.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.09.548280">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.09.548280" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.09.548280">
        <p class="paperTitle">DISCOVERY AND VALIDATION OF GENES DRIVING DRUG-INTAKE AND RELATED BEHAVIORAL TRAITS IN MICE</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.09.548280" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.09.548280" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roy, T.; Bubier, J. A.; Dickson, P. E.; Wilcox, T.; Ndukum, J.; Clark, J. W.; Rizzo, S. J. S.; Crabbe, J. C.; Denegre, J.; Svenson, K. L.; Braun, R. E.; Kumar, V.; Murray, S. A.; White, J. K.; Philip, V.; Chesler, E.</p>
        <p class="info">Score: 6.1, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.09.548280' target='https://doi.org/10.1101/2023.07.09.548280'> 10.1101/2023.07.09.548280</a></p>
        <p class="abstract">Substance use disorders (SUDs) are heritable disorders characterized by compulsive drug use, but the biological mechanisms driving addiction remain largely unknown. Genetic correlations reveal that predisposing drug-naive phenotypes, including anxiety, depression, novelty preference, and sensation seeking, are predictive of drug-use phenotypes, implicating shared genetic mechanisms. Because of this relationship, high-throughput behavioral screening of predictive phenotypes in knockout (KO) mice allows efficient discovery of genes likely to be involved in drug use. We used this strategy in two rounds of screening in which we identified 33 drug-use candidate genes and ultimately validated the perturbation of 22 of these genes as causal drivers of substance intake. In our initial round of screening, we employed the two-bottle-choice paradigms to assess alcohol, methamphetamine, and nicotine intake. We identified 19 KO strains that were extreme responders on at least one predictive phenotype. Thirteen of the 19 gene deletions (68%) significantly affected alcohol use three methamphetamine use, and two both. In the second round of screening, we employed a multivariate approach to identify outliers and performed validation using methamphetamine two-bottle choice and ethanol drinking-in-the-dark protocols. We identified 15 KO strains that were extreme responders across the predisposing drug-naive phenotypes. Eight of the 15 gene deletions (53%) significantly affected intake or preference for three alcohol, eight methamphetamine or three both (3). We observed multiple relations between predisposing behaviors and drug intake, revealing many distinct biobehavioral processes underlying these relationships. The set of mouse models identified in this study can be used to characterize these addiction-related processes further.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547891">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547891" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547891">
        <p class="paperTitle">Polygenic Risk Score Associates with Atherosclerotic Plaque Characteristics at Autopsy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547891" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547891" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cornelissen, A.; Gadhoke, N.; Ryan, K. A.; Hodonsky, C. J.; Mitchell, R. N.; Bihlmeyer, N. A.; Duong, T.; Chen, Z.; Dikongue, A.; Sakamoto, A.; Sato, Y.; Kawakami, R.; Mori, M.; Kawai, K.; Fernandez, R.; Ghosh, S. K. B.; Braumann, R.; Abebe, B.; Kutys, R.; Kutyna, M. D.; Romero, M. E.; Kolodgie, F.; Miller, C. L.; Hong, C. C.; Grove, M. L.; Brody, J. A.; Sotoodehnia, N.; Arking, D. E.; Schunkert, H.; Mitchell, B. D.; Guo, L.; Virmani, R.; Finn, A. V.</p>
        <p class="info">Score: 5.4, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547891' target='https://doi.org/10.1101/2023.07.05.547891'> 10.1101/2023.07.05.547891</a></p>
        <p class="abstract">BackgroundPolygenic risk scores (PRS) for coronary artery disease (CAD) potentially improve cardiovascular risk prediction. However, their relationship with histopathologic features of CAD has never been examined systematically.

MethodsFrom 4,327 subjects referred to CVPath by the State of Maryland Office Chief Medical Examiner (OCME) for sudden death between 1994 and 2015, 2,455 cases were randomly selected for genotyping. We generated PRS from 291 known CAD risk loci. Detailed histopathologic examination of the coronary arteries was performed in all subjects. The primary study outcome measurements were histopathologic plaque features determining severity of atherosclerosis, including %stenosis, calcification, thin-cap fibroatheromas (TCFA), and thrombotic CAD.

ResultsAfter exclusion of cases with insufficient DNA sample quality or with missing data, 954 cases (mean age 48.8{&#43;/-}14.7; 75.7% men) remained in the final study cohort. Subjects in the highest PRS quintile exhibited more severe atherosclerosis compared to subjects in the lowest quintile, with greater %stenosis (80.3%{&#43;/-}27.0% vs. 50.4%{&#43;/-}38.7%; adjusted p&lt;0.001) and a higher frequency of calcification (69.6% vs. 35.8%; adjusted p=0.004) and TCFAs (26.7% vs. 9.5%; adjusted p=0.007). Even after adjustment for traditional CAD risk factors subjects within the highest PRS quintile had higher odds of severe atherosclerosis (i.e., [&amp;ge;]75% stenosis; adjusted OR 3.77; 95%CI 2.10-6.78; p&lt;0.001) and plaque rupture (adjusted OR 4.05; 95%CI 2.26-7.24; p&lt;0.001). Moreover, subjects within the highest quintile had higher odds of CAD-associated cause of death, especially among those aged 50 years and younger (adjusted OR 4.08; 95%CI 2.01-8.30; p&lt;0.001). No associations were observed with plaque erosion.

ConclusionsThis is the first autopsy study investigating associations between PRS and atherosclerosis severity at the histopathologic level in subjects with sudden death. Our pathological analysis suggests PRS correlates with plaque burden and features of advanced atherosclerosis and may be useful as a method for CAD risk stratification, especially in younger subjects.

HighlightsO_LIIn this autopsy study including 954 subjects within the CVPath Sudden Death Registry, high PRS correlated with plaque burden and atherosclerosis severity.
C_LIO_LIThe PRS showed differential associations with plaque rupture and plaque erosion, suggesting different etiologies to these two causes of thrombotic CAD.
C_LIO_LIPRS may be useful for risk stratification, particularly in the young. Further examination of individual risk loci and their association with plaque morphology may help understand molecular mechanisms of atherosclerosis, potentially revealing new therapy targets of CAD.
C_LI

Graphic AbstractA polygenic risk score, generated from 291 known CAD risk loci, was assessed in 954 subjects within the CVPath Sudden Death Registry. Histopathologic examination of the coronary arteries was performed in all subjects. Subjects in the highest PRS quintile exhibited more severe atherosclerosis as compared to subjects in the lowest quintile, with a greater plaque burden, more calcification, and a higher frequency of plaque rupture.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548702">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548702" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548702">
        <p class="paperTitle">Validation of human telomere length trans-ancestry meta-analysis association signals identifies POP5 and KBTBD6 as novel human telomere length regulation genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548702" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548702" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Keener, R.; Chhetri, S. B.; Connelly, C. J.; Taub, M. A.; Conomos, M. P.; Weinstock, J. S.; Ni, B.; Strober, B. J.; Aslibekyan, S.; Auer, P. L.; Barwick, L.; Becker, L. C.; Blangero, J.; Bleecker, E. R.; Brody, J. A.; Cade, B. E.; Celedon, J. C.; Chang, Y.-C.; Cupples, L. A.; Custer, B.; Freedman, B. I.; Gladwin, M. T.; Heckbert, S. R.; Hou, L.; Irvin, M. R.; Isasi, C. R.; Johnsen, J. M.; Kenny, E. E.; Kooperberg, C.; Minster, R. L.; Nekhai, S.; Pankratz, N.; Peyser, P. A.; Rotter, J. I.; Taylor, K. D.; Telen, M. J.; Wu, B.; Yanek, L. R.; Yang, I. V.; Albert, C.; Arnett, D. K.; Ashley-Koch, A.</p>
        <p class="info">Score: 5.2, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548702' target='https://doi.org/10.1101/2023.07.12.548702'> 10.1101/2023.07.12.548702</a></p>
        <p class="abstract">Telomere length genome-wide association studies (GWAS) have become well-powered to detect novel genes in telomere length regulation. However, no prior work has validated these putative novel genes to confirm the contribution of GWAS loci to telomere length regulation. We conducted a trans-ancestry meta-analysis of 211,369 individuals. Through enrichment analyses of chromatin state and cell-type heritability we identified blood and immune cells as the most relevant cell type to examine telomere length association signals. We validated specific GWAS associations by overexpressing KBTBD6, a component of an E3 ubiquitin ligase complex, and POP5, a component of the Ribonuclease P/MRP complex, and demonstrating that both lengthened telomeres as predicted by our statistical analyses. CRISPR/Cas9 deletion of the predicted causal regions of these association peaks in K562 immortalized blood cells reduced expression of these genes, demonstrating that these loci are related to transcriptional regulation of KBTBD6 and POP5, respectively. Together our results demonstrate the utility of telomere length GWAS in the identification of novel telomere length regulation mechanisms and highlight the importance of the proteasome-ubiquitin pathway in telomere length regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548340">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548340" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548340">
        <p class="paperTitle">High throughput functional profiling of genes at intraocular pressure loci reveals distinct networks for glaucoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548340" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548340" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Greatbatch, C. J.; Lu, Q.; Hung, S.; Wing, K.; Liang, H.; Han, X.; Zhou, T.; Siggs, O.; Mackey, D.; Cook, A.; Senabouth, A.; Liu, G.-S.; Craig, J.; MacGregor, S.; Powell, J.; Hewitt, A.</p>
        <p class="info">Score: 2.8, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548340' target='https://doi.org/10.1101/2023.07.10.548340'> 10.1101/2023.07.10.548340</a></p>
        <p class="abstract">INTRODUCTIONPrimary open angle glaucoma (POAG) is a leading cause of blindness globally. Characterised by progressive retinal ganglion cell degeneration, the precise pathogenesis remains unknown. Genome-wide association studies (GWAS) have uncovered many genetic variants associated with elevated intraocular pressure (IOP), one of the key risk factors for POAG. This study sought to investigate the morphological and transcriptional consequences of perturbation of key genes at IOP loci in trabecular meshwork cell (TMC); the cellular regulators of IOP. We aimed to identify genetic and morphological variation that can be attributed to TMC dysfunction and raised IOP in POAG.

METHODS62 genes across 55 loci were knocked-out in a primary human TMC line. Each knockout group, including five non-targeting control groups, underwent single-cell RNA-sequencing (scRNA-seq) for differentially-expressed gene (DEG) analysis. Multiplexed fluorescent staining of key organelles, was coupled with high-throughput microscopy for single-cell morphological analysis using CellProfiler image analysis.

RESULTSAcross many of the individual gene knockouts scRNA-seq highlighted genes relating to matrix metalloproteinases and interferon-induced proteins. Our work has prioritised genes at four loci of interest to identify gene knockouts that may contribute to the pathogenesis of POAG, including ANGPTL2, LMX1B, CAV1, and KREMEN1. Three genetic networks of gene knockouts with similar transcriptomic profiles were identified (ABO / CAV1 / MYOC, ANGPT2 / PKHD1 / TNS1 / TXNRD2, and CAPZA1 / KALRN / LMO7 / PLEKHA7 / GNB1L / TEX41), suggesting a synergistic function in trabecular meshwork cell physiology. TEK knockout caused significant upregulation of nuclear granularity on morphological analysis, whilst knockout of TRIOBP, TMCO1 and PLEKHA7 increased granularity and intensity of actin and the cell-membrane.

CONCLUSIONHigh throughput analysis of cellular structure and function through multiplex fluorescent single-cell analysis and scRNA-seq assays enabled the direct study of genetic perturbations at the single-cell resolution. This work provides a framework for investigating the role of genes in the pathogenesis of glaucoma and heterogenous diseases with a strong genetic basis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548735" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548735">
        <p class="paperTitle">Recombination map tailored to Native Hawaiians reduces false positive findings in genomic scans of positive selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dinh, B. L.; Tang, E.; Taparra, K.; Nakatsuka, N.; Chen, F.; Chiang, C. W. K.</p>
        <p class="info">Score: 2.2, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548735' target='https://doi.org/10.1101/2023.07.12.548735'> 10.1101/2023.07.12.548735</a></p>
        <p class="abstract">Recombination events establish the patterns of haplotypic structure in a population and estimates of recombination rates are used in several downstream population and statistical genetic analyses. However, the recombination landscape of populations not represented in the 1000 Genomes Project (1KGP) is poorly understood. Using suboptimal maps from distantly related populations reduces the efficacy of extending genomic analyses to underrepresented populations, including Native Hawaiians. To overcome this challenge, we constructed linkage disequilibrium (LD)- and Identity-by-Descent (IBD)-based recombination maps specifically for Native Hawaiians. We created and characterized two recombination maps from our study sample: one based on a random subset of our sample reflecting the current admixed state of Native Hawaiians (NH map), and one based on a subset of individuals with enriched Polynesian ancestries (PNS map) with potential to be used for less admixed Polynesian populations such as the Samoans. We found the recombination landscape to be less correlated with those from other 1KGP populations (e.g. Spearman&#39;s rho = 0.79 between PNS and CEU (Utah residents with Northern and Western European ancestry) compared to 0.92 between YRI (Yoruba in Ibadan, Nigeria) and CEU at 50 kb resolution. PNS shared the fewest recombination hotspots with other populations from 1KGP (e.g. 8% of hotspots shared between PNS and CEU compared to 27% of hotspots shared between YRI and CEU). We found that downstream analysis in the Native Hawaiian population, such as local ancestry inference, imputation, and IBD segment and relatedness detections, would achieve similar efficacy when using the NH map compared to an omnibus map. However, for genome scans of adaptive loci using integrated haplotype scores, we found several loci with apparent genome-wide significant signals (|Z-score| &gt; 4) in Native Hawaiians that would not have been significant when analyzed using NH-specific maps. Therefore, population-specific recombination maps improve the robustness of haplotype-based statistics and help us better characterize the evolutionary history that may underlie Native Hawaiian-specific health conditions that persist today.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
