<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Clark (2023) and the Persistence of Hereditarian Fallacies
Authors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.
Score: 103.1, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565061
Clark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people &#34;remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England&#34;; and (3) social mobility has remained static over this time period due to strong assortative mating on a &#34;social genotype.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Clark (2023) and the Persistence of Hereditarian Fallacies
Authors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.
Score: 103.1, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565061
Clark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people &#34;remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England&#34;; and (3) social mobility has remained static over this time period due to strong assortative mating on a &#34;social genotype." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Clark (2023) and the Persistence of Hereditarian Fallacies
Authors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.
Score: 103.1, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565061
Clark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people &#34;remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England&#34;; and (3) social mobility has remained static over this time period due to strong assortative mating on a &#34;social genotype."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Clark (2023) and the Persistence of Hereditarian Fallacies\nAuthors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.\nScore: 103.1, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565061\nClark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people \u0026#34;remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England\u0026#34;; and (3) social mobility has remained static over this time period due to strong assortative mating on a \u0026#34;social genotype.",
  "keywords": [
    
  ],
  "articleBody": " Clark (2023) and the Persistence of Hereditarian Fallacies\nAuthors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.\nScore: 103.1, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565061\nClark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people \"remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England\"; and (3) social mobility has remained static over this time period due to strong assortative mating on a \"social genotype.\" These conclusions are based on a misconstrual of model parameters, which conflates genetic and non-genetic transmission (e.g. of wealth) within families. As we show, there is strong confounding of genetic and non-genetic sources of similarity in these data. Inconsistent with claims (2) and (3), we show that familial correlations in status are variable--generally decreasing--through the time period analyzed. Lastly, we find that statistical artifacts substantially bias estimates of familial correlations in the paper. Overall, Clark (2023) provides no information about the relative contribution of genetic and non-genetic factors to social status.\nHeritability within groups is uninformative about differences among groups: cases from behavioral, evolutionary, and statistical genetics\nAuthors: Schraiber, J. G.; Edge, M. D.\nScore: 59.7, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565864\nWithout the ability to control or randomize environments (or genotypes), it is difficult to determine the degree to which observed phenotypic difference between two groups of individuals are due to genetic vs. environmental differences. However, some have suggested that these concerns may be limited to pathological cases, and methods have appeared that seem to give--directly or indirectly--some support to claims that aggregate heritable variation within groups can be related to heritable variation among groups. We consider three families of approaches: the \"between-group heritability\" sometimes invoked in behavior genetics, the statistic PST used in empirical work in evolutionary quantitative genetics, and methods based on variation in ancestry in an admixed population, used in anthropological and statistical genetics. In each case, we show mathematically that information on within-group genetic and phenotypic information in the aggregate cannot separate among-group differences into genetic and environmental components, and we provide simulation results that support our claims. We discuss these results in terms of the long-running debate on this topic.\nA general and efficient representation of ancestral recombination graphs\nAuthors: Wong, Y.; Ignatieva, A.; Koskela, J.; Gorjanc, G.; Wohns, A. W.; Kelleher, J.\nScore: 32.1, Published: 2023-11-04 DOI: 10.1101/2023.11.03.565466\nAs a result of recombination, adjacent nucleotides can have different paths of genetic inheri-tance and therefore the genealogical trees for a sample of DNA sequences vary along the genome. The structure capturing the details of these intricately interwoven paths of inheritance is referred to as an ancestral recombination graph (ARG). New developments have made it possible to infer ARGs at scale, enabling many new applications in population and statistical genetics. This rapid progress, however, has led to a substantial gap opening between theory and practice. Standard mathematical formalisms, based on exhaustively detailing the \"events\" that occur in the history of a sample, are insufficient to describe the outputs of current methods. Moreover, we argue that the underlying assumption that all events can be known and precisely estimated is fundamentally unsuited to the realities of modern, population-scale datasets. We propose an alternative mathematical formulation that encompasses the outputs of recent methods and can capture the full richness of modern large-scale datasets. By defining this ARG encoding in terms of specific genomes and their intervals of genetic inheritance, we avoid the need to exhaustively list (and estimate) all events. The effects of multiple events can be aggregated in different ways, providing a natural way to express many forms of approximate and partial knowledge about the recombinant ancestry of a sample.\nWhole genome sequencing analysis identifies rare, large-effect non-coding variants and regions associated with circulating protein levels\nAuthors: Hawkes, G.; Chundru, K.; Jackson, L.; Patel, K. A.; Murray, A.; Wood, A. R.; Wright, C. R.; Weedon, M. N.; Frayling, T. M.; Beaumont, R. N.\nScore: 28.4, Published: 2023-11-05 DOI: 10.1101/2023.11.04.565589\nThe role of non-coding rare variation in common phenotypes is largely unknown, due to a lack of whole-genome sequence data, and the difficulty of categorising non-coding variants into biologically meaningful regulatory units. To begin addressing these challenges, we performed a cis association analysis using whole-genome sequence data, consisting of 391 million variants and 1,450 circulating protein levels in ~20,000 UK Biobank participants. We identified 777 independent rare non-coding single variants associated with circulating protein levels (P\u003c1x10-9), after conditioning on protein-coding and common associated variants. Rare non-coding aggregate testing identified 108 conditionally independent regulatory regions. Unlike protein-coding variation, rare non-coding genetic variation was almost as likely to increase as decrease protein levels. The regions we identified overlapped predicted tissue-specific enhancers more than promoters, suggesting they represent tissue-specific regulatory regions. Our results have important implications for the identification, and role, of rare non-coding variation associated with common human phenotypes.\nIncreasing equity in science requires better ethics training: a course by trainees, for trainees\nAuthors: Patel, R. A.; Ungar, R. A.; Pyke, A. L.; Adimoelja, A.; Chakraborty, M.; Cotter, D. J.; Freund, M.; Goddard, P.; Gomez-Stafford, J.; Greenwald, E.; Higgs, E.; Hunter, N.; MacKenzie, T. M. G.; Narain, A.; Martschenko, D. O.\nScore: 28.0, Published: 2023-11-06 DOI: 10.1101/2023.11.03.565577\nDespite the profound impacts of scientific research, few scientists have received the necessary training to productively discuss the ethical and societal implications of their work. To address this critical gap, we - a group of predominantly human genetics trainees - developed a course on genetics, ethics, and society. We intend for this course to serve as a template for other institutions and scientific disciplines. Our curriculum positions human genetics within its historical and societal context and encourages students to evaluate how societal norms and structures impact the conduct of scientific research. We demonstrate the utility of this course via surveys of enrolled students and provide resources and strategies for others hoping to teach a similar course. We conclude by arguing that if we are to work towards rectifying the inequities and injustices produced by our field, we must first learn to view our own research as impacting and being impacted by society.\nTranscription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency\nAuthors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.\nScore: 53.0, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808\nEctopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming. Using neural networks that map DNA sequence to ATAC-seq profiles at base-resolution, we annotated cell-state-specific predictive transcription factor (TF) motif syntax in regulatory elements, inferred affinity- and concentration-dependent dynamics of Tn5-bias corrected TF footprints, linked peaks to putative target genes, and elucidated rewiring of TF- to-gene cis-regulatory networks. Our models reveal that early in reprogramming, OSK, at supraphysiological concentrations, rapidly open transient regulatory elements by occupying non-canonical low-affinity binding sites. As OSK concentration falls, the accessibility of these transient elements decays as a function of motif affinity. We find that these OSK-dependent transient elements sequester the somatic TF AP-1. This redistribution is strongly associated with the silencing of fibroblast-specific genes within individual nuclei. Together, our integrated single-cell resource and models reveal insights into the cis-regulatory code of reprogramming at unprecedented resolution, connect TF stoichiometry and motif syntax to diversification of cell fate trajectories, and provide new perspectives on the dynamics and role of transient regulatory elements in somatic silencing.\nA statistical framework for powerful multi-trait rare variant analysis in large-scale whole-genome sequencing studies\nAuthors: Li, X.; Chen, H.; Selvaraj, M. S.; Van Buren, E.; Zhou, H.; Wang, Y.; Sun, R.; McCaw, Z. R.; Yu, Z.; Arnett, D. K.; Bis, J. C.; Blangero, J.; Boerwinkle, E.; Bowden, D. W.; Brody, J. A.; Cade, B. E.; Carson, A. P.; Carlson, J. C.; Chami, N.; Chen, Y.-D. I.; Curran, J. E.; de Vries, P. S.; Fornage, M.; Franceschini, N.; Freedman, B. I.; Gu, C.; Heard-Costa, N. L.; He, J.; Hou, L.; Hung, Y.-J.; Irvin, M. R.; Kaplan, R. C.; Kardia, S. L. R.; Kelly, T.; Konigsberg, I.; Kooperberg, C.; Kral, B. G.; Li, C.; Loos, R. J. F.; Mahaney, M. C.; Martin, L. W.; Mathias, R. A.; Minster, R. L.; Mitchell, B. D\nScore: 14.8, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564764\nLarge-scale whole-genome sequencing (WGS) studies have improved our understanding of the contributions of coding and noncoding rare variants to complex human traits. Leveraging association effect sizes across multiple traits in WGS rare variant association analysis can improve statistical power over single-trait analysis, and also detect pleiotropic genes and regions. Existing multi-trait methods have limited ability to perform rare variant analysis of large-scale WGS data. We propose MultiSTAAR, a statistical framework and computationally-scalable analytical pipeline for functionally-informed multi-trait rare variant analysis in large-scale WGS studies. MultiSTAAR accounts for relatedness, population structure and correlation among phenotypes by jointly analyzing multiple traits, and further empowers rare variant association analysis by incorporating multiple functional annotations. We applied MultiSTAAR to jointly analyze three lipid traits (low-density lipoprotein cholesterol, high-density lipoprotein cholesterol and triglycerides) in 61,861 multi-ethnic samples from the Trans-Omics for Precision Medicine (TOPMed) Program. We discovered new associations with lipid traits missed by single-trait analysis, including rare variants within an enhancer of NIPSNAP3A and an intergenic region on chromosome 1.\nPitfalls in performing genome-wide association studies on ratio traits\nAuthors: McCaw, Z. R.; Dey, R.; Somineni, H.; Amar, D.; Mukherjee, S.; Sandor, K.; insitro Research Team, ; Karaletsos, T.; Koller, D.; Davey Smith, G.; MacArthur, D.; O'Dushlaine, C.; Soare, T. W.\nScore: 13.8, Published: 2023-11-01 DOI: 10.1101/2023.10.27.564385\nGenome-wide association studies (GWAS) are often performed on ratios composed of a numerator trait divided by a denominator trait. Examples include body mass index (BMI) and the waist-to-hip ratio, among many others. Explicitly or implicitly, the goal of forming the ratio is typically to adjust the numerator for the denominator. While forming ratios may be clinically expedient, there are several important issues with performing GWAS on ratios. Forming a ratio does not \"adjust\" for the denominator in the sense of holding it constant, and it is unclear whether associations with ratios are attributable to the numerator, the denominator, or both. Here we demonstrate that associations arising in ratio GWAS can be entirely denominator-driven, implying that at least some associations uncovered by ratio GWAS may be due solely to a putative adjustment variable. In a survey of 10 exemplar ratios, we find that the ratio model disagrees with the adjusted model (performing GWAS on the numerator while conditioning on the denominator) at around 1/3 of loci. Using BMI as an example, we show that variants detected by only the ratio model are more strongly associated with the denominator (height), while variants detected by only the adjusted model are more strongly associated with the numerator (weight). Although the adjusted model provides effect sizes with a clearer interpretation, it is susceptible to collider bias. We propose and validate a simple method of correcting for the genetic collider bias via leave-one-chromosome-out polygenic scoring.\nPolycomb protein binding and looping mediated by Polycomb Response Elements in the ON transcriptional state\nAuthors: Brown, J. L.; Zhang, L.; Rocha, P. P.; Kassis, J. A.; Sun, M.-a.\nScore: 9.2, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565256\nPolycomb group proteins (PcG) mediate epigenetic silencing of important developmental genes and other targets. In Drosophila, canonical PcG-target genes contain Polycomb Response Elements (PREs) that recruit PcG protein complexes including PRC2 that tri-methylates H3K27 forming large H3K27me3 domains. In the OFF transcriptional state, PREs loop with each other and this looping strengthens silencing. Here we address the question of what PcG proteins bind to PREs when canonical PcG target genes are expressed, and whether PREs loop when these genes are ON. Our data show that the answer to this question is PRE-specific but general conclusions can be made. First, within a PcG-target gene, some regulatory DNA can remain covered with H3K27me3 and PcG proteins remain bound to PREs in these regions. Second, when PREs are within H3K27ac domains, PcG-binding decreases, however, this depends on the protein and PRE. The DNA binding protein GAF, and the PcG protein Ph remain at PREs even when other PcG proteins are greatly depleted. In the ON state, PREs can still loop with each other, but also form loops with presumptive enhancers. These data support the model that, in addition to their role in PcG silencing, PREs can act as \"promoter-tethering elements\" mediating interactions between promoter proximal PREs and distant enhancers.\nH3K9me regulates heterochromatin silencing through incoherent feedforward loops\nAuthors: Yabe, K.; Asuka, K.; Oya, S.; Kakutani, T.; Hirayama, M.; Tanaka, Y.; Inagaki, S.\nScore: 6.3, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565278\nHistone H3 lysine-9 methylation (H3K9me) is associated with condensed and transcriptionally inactive heterochromatin. Although it has long been known that H3K9me silences transcription to control a wide variety of biological phenomena in many eukaryotic species, how the silencing is regulated under the control of H3K9me is still largely unclear. Moreover, how cells delimit regions with H3K9me to avoid silencing essential genes remains unexplored. Here, using Arabidopsis genetic systems that induce H3K9me2 and its associated non-CG DNA methylation (mCH) in the transcribed region of genes de novo, we show that the accumulation of H3K9me2/mCH paradoxically also leads to the deposition of the euchromatic mark H3K36me3. This induction of H3K36me3 depends on a SET domain methyltransferase, ASHH3, and brings about anti-silencing by preventing the demethylation of H3K4me1 by LDL2, which mediates transcriptional silencing downstream of H3K9me2/mCH. H3K9me2/mCH-driven antagonistic actions of ASHH3-H3K36me3 and LDL2-H3K4me1-loss also regulate the de novo silencing of reactivated transposable elements (TEs). These results demonstrate that H3K9me2 both facilitates and impedes silencing, and the incoherent feedforward loops fine-tune the fate of genes and TEs. Our results illuminate a novel elaborate mechanism for partitioning chromatin domains and provide insights into the molecular basis underlying natural epigenetic variation.\n",
  "wordCount" : "2368",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565061">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565061" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565061">
        <p class="paperTitle">Clark (2023) and the Persistence of Hereditarian Fallacies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565061" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565061" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.</p>
        <p class="info">Score: 103.1, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565061' target='https://doi.org/10.1101/2023.11.01.565061'> 10.1101/2023.11.01.565061</a></p>
        <p class="abstract">Clark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people &#34;remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England&#34;; and (3) social mobility has remained static over this time period due to strong assortative mating on a &#34;social genotype.&#34; These conclusions are based on a misconstrual of model parameters, which conflates genetic and non-genetic transmission (e.g. of wealth) within families. As we show, there is strong confounding of genetic and non-genetic sources of similarity in these data. Inconsistent with claims (2) and (3), we show that familial correlations in status are variable--generally decreasing--through the time period analyzed. Lastly, we find that statistical artifacts substantially bias estimates of familial correlations in the paper. Overall, Clark (2023) provides no information about the relative contribution of genetic and non-genetic factors to social status.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565864">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565864" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565864">
        <p class="paperTitle">Heritability within groups is uninformative about differences among groups: cases from behavioral, evolutionary, and statistical genetics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565864" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565864" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schraiber, J. G.; Edge, M. D.</p>
        <p class="info">Score: 59.7, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565864' target='https://doi.org/10.1101/2023.11.06.565864'> 10.1101/2023.11.06.565864</a></p>
        <p class="abstract">Without the ability to control or randomize environments (or genotypes), it is difficult to determine the degree to which observed phenotypic difference between two groups of individuals are due to genetic vs. environmental differences. However, some have suggested that these concerns may be limited to pathological cases, and methods have appeared that seem to give--directly or indirectly--some support to claims that aggregate heritable variation within groups can be related to heritable variation among groups. We consider three families of approaches: the &#34;between-group heritability&#34; sometimes invoked in behavior genetics, the statistic PST used in empirical work in evolutionary quantitative genetics, and methods based on variation in ancestry in an admixed population, used in anthropological and statistical genetics. In each case, we show mathematically that information on within-group genetic and phenotypic information in the aggregate cannot separate among-group differences into genetic and environmental components, and we provide simulation results that support our claims. We discuss these results in terms of the long-running debate on this topic.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565466">
        <p class="paperTitle">A general and efficient representation of ancestral recombination graphs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wong, Y.; Ignatieva, A.; Koskela, J.; Gorjanc, G.; Wohns, A. W.; Kelleher, J.</p>
        <p class="info">Score: 32.1, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565466' target='https://doi.org/10.1101/2023.11.03.565466'> 10.1101/2023.11.03.565466</a></p>
        <p class="abstract">As a result of recombination, adjacent nucleotides can have different paths of genetic inheri-tance and therefore the genealogical trees for a sample of DNA sequences vary along the genome. The structure capturing the details of these intricately interwoven paths of inheritance is referred to as an ancestral recombination graph (ARG). New developments have made it possible to infer ARGs at scale, enabling many new applications in population and statistical genetics. This rapid progress, however, has led to a substantial gap opening between theory and practice. Standard mathematical formalisms, based on exhaustively detailing the &#34;events&#34; that occur in the history of a sample, are insufficient to describe the outputs of current methods. Moreover, we argue that the underlying assumption that all events can be known and precisely estimated is fundamentally unsuited to the realities of modern, population-scale datasets. We propose an alternative mathematical formulation that encompasses the outputs of recent methods and can capture the full richness of modern large-scale datasets. By defining this ARG encoding in terms of specific genomes and their intervals of genetic inheritance, we avoid the need to exhaustively list (and estimate) all events. The effects of multiple events can be aggregated in different ways, providing a natural way to express many forms of approximate and partial knowledge about the recombinant ancestry of a sample.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.04.565589">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.04.565589" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.04.565589">
        <p class="paperTitle">Whole genome sequencing analysis identifies rare, large-effect non-coding variants and regions associated with circulating protein levels</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.04.565589" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.04.565589" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hawkes, G.; Chundru, K.; Jackson, L.; Patel, K. A.; Murray, A.; Wood, A. R.; Wright, C. R.; Weedon, M. N.; Frayling, T. M.; Beaumont, R. N.</p>
        <p class="info">Score: 28.4, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.04.565589' target='https://doi.org/10.1101/2023.11.04.565589'> 10.1101/2023.11.04.565589</a></p>
        <p class="abstract">The role of non-coding rare variation in common phenotypes is largely unknown, due to a lack of whole-genome sequence data, and the difficulty of categorising non-coding variants into biologically meaningful regulatory units. To begin addressing these challenges, we performed a cis association analysis using whole-genome sequence data, consisting of 391 million variants and 1,450 circulating protein levels in ~20,000 UK Biobank participants. We identified 777 independent rare non-coding single variants associated with circulating protein levels (P&lt;1x10-9), after conditioning on protein-coding and common associated variants. Rare non-coding aggregate testing identified 108 conditionally independent regulatory regions. Unlike protein-coding variation, rare non-coding genetic variation was almost as likely to increase as decrease protein levels. The regions we identified overlapped predicted tissue-specific enhancers more than promoters, suggesting they represent tissue-specific regulatory regions. Our results have important implications for the identification, and role, of rare non-coding variation associated with common human phenotypes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565577">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565577" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565577">
        <p class="paperTitle">Increasing equity in science requires better ethics training: a course by trainees, for trainees</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565577" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565577" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Patel, R. A.; Ungar, R. A.; Pyke, A. L.; Adimoelja, A.; Chakraborty, M.; Cotter, D. J.; Freund, M.; Goddard, P.; Gomez-Stafford, J.; Greenwald, E.; Higgs, E.; Hunter, N.; MacKenzie, T. M. G.; Narain, A.; Martschenko, D. O.</p>
        <p class="info">Score: 28.0, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565577' target='https://doi.org/10.1101/2023.11.03.565577'> 10.1101/2023.11.03.565577</a></p>
        <p class="abstract">Despite the profound impacts of scientific research, few scientists have received the necessary training to productively discuss the ethical and societal implications of their work. To address this critical gap, we - a group of predominantly human genetics trainees - developed a course on genetics, ethics, and society. We intend for this course to serve as a template for other institutions and scientific disciplines. Our curriculum positions human genetics within its historical and societal context and encourages students to evaluate how societal norms and structures impact the conduct of scientific research. We demonstrate the utility of this course via surveys of enrolled students and provide resources and strategies for others hoping to teach a similar course. We conclude by arguing that if we are to work towards rectifying the inequities and injustices produced by our field, we must first learn to view our own research as impacting and being impacted by society.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560808">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560808" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560808">
        <p class="paperTitle">Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560808" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560808" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.</p>
        <p class="info">Score: 53.0, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560808' target='https://doi.org/10.1101/2023.10.04.560808'> 10.1101/2023.10.04.560808</a></p>
        <p class="abstract">Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming. Using neural networks that map DNA sequence to ATAC-seq profiles at base-resolution, we annotated cell-state-specific predictive transcription factor (TF) motif syntax in regulatory elements, inferred affinity- and concentration-dependent dynamics of Tn5-bias corrected TF footprints, linked peaks to putative target genes, and elucidated rewiring of TF- to-gene cis-regulatory networks. Our models reveal that early in reprogramming, OSK, at supraphysiological concentrations, rapidly open transient regulatory elements by occupying non-canonical low-affinity binding sites. As OSK concentration falls, the accessibility of these transient elements decays as a function of motif affinity. We find that these OSK-dependent transient elements sequester the somatic TF AP-1. This redistribution is strongly associated with the silencing of fibroblast-specific genes within individual nuclei. Together, our integrated single-cell resource and models reveal insights into the cis-regulatory code of reprogramming at unprecedented resolution, connect TF stoichiometry and motif syntax to diversification of cell fate trajectories, and provide new perspectives on the dynamics and role of transient regulatory elements in somatic silencing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564764" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564764">
        <p class="paperTitle">A statistical framework for powerful multi-trait rare variant analysis in large-scale whole-genome sequencing studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, X.; Chen, H.; Selvaraj, M. S.; Van Buren, E.; Zhou, H.; Wang, Y.; Sun, R.; McCaw, Z. R.; Yu, Z.; Arnett, D. K.; Bis, J. C.; Blangero, J.; Boerwinkle, E.; Bowden, D. W.; Brody, J. A.; Cade, B. E.; Carson, A. P.; Carlson, J. C.; Chami, N.; Chen, Y.-D. I.; Curran, J. E.; de Vries, P. S.; Fornage, M.; Franceschini, N.; Freedman, B. I.; Gu, C.; Heard-Costa, N. L.; He, J.; Hou, L.; Hung, Y.-J.; Irvin, M. R.; Kaplan, R. C.; Kardia, S. L. R.; Kelly, T.; Konigsberg, I.; Kooperberg, C.; Kral, B. G.; Li, C.; Loos, R. J. F.; Mahaney, M. C.; Martin, L. W.; Mathias, R. A.; Minster, R. L.; Mitchell, B. D</p>
        <p class="info">Score: 14.8, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564764' target='https://doi.org/10.1101/2023.10.30.564764'> 10.1101/2023.10.30.564764</a></p>
        <p class="abstract">Large-scale whole-genome sequencing (WGS) studies have improved our understanding of the contributions of coding and noncoding rare variants to complex human traits. Leveraging association effect sizes across multiple traits in WGS rare variant association analysis can improve statistical power over single-trait analysis, and also detect pleiotropic genes and regions. Existing multi-trait methods have limited ability to perform rare variant analysis of large-scale WGS data. We propose MultiSTAAR, a statistical framework and computationally-scalable analytical pipeline for functionally-informed multi-trait rare variant analysis in large-scale WGS studies. MultiSTAAR accounts for relatedness, population structure and correlation among phenotypes by jointly analyzing multiple traits, and further empowers rare variant association analysis by incorporating multiple functional annotations. We applied MultiSTAAR to jointly analyze three lipid traits (low-density lipoprotein cholesterol, high-density lipoprotein cholesterol and triglycerides) in 61,861 multi-ethnic samples from the Trans-Omics for Precision Medicine (TOPMed) Program. We discovered new associations with lipid traits missed by single-trait analysis, including rare variants within an enhancer of NIPSNAP3A and an intergenic region on chromosome 1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564385">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564385" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564385">
        <p class="paperTitle">Pitfalls in performing genome-wide association studies on ratio traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564385" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564385" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCaw, Z. R.; Dey, R.; Somineni, H.; Amar, D.; Mukherjee, S.; Sandor, K.; insitro Research Team,  ; Karaletsos, T.; Koller, D.; Davey Smith, G.; MacArthur, D.; O&#39;Dushlaine, C.; Soare, T. W.</p>
        <p class="info">Score: 13.8, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564385' target='https://doi.org/10.1101/2023.10.27.564385'> 10.1101/2023.10.27.564385</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) are often performed on ratios composed of a numerator trait divided by a denominator trait. Examples include body mass index (BMI) and the waist-to-hip ratio, among many others. Explicitly or implicitly, the goal of forming the ratio is typically to adjust the numerator for the denominator. While forming ratios may be clinically expedient, there are several important issues with performing GWAS on ratios. Forming a ratio does not &#34;adjust&#34; for the denominator in the sense of holding it constant, and it is unclear whether associations with ratios are attributable to the numerator, the denominator, or both. Here we demonstrate that associations arising in ratio GWAS can be entirely denominator-driven, implying that at least some associations uncovered by ratio GWAS may be due solely to a putative adjustment variable. In a survey of 10 exemplar ratios, we find that the ratio model disagrees with the adjusted model (performing GWAS on the numerator while conditioning on the denominator) at around 1/3 of loci. Using BMI as an example, we show that variants detected by only the ratio model are more strongly associated with the denominator (height), while variants detected by only the adjusted model are more strongly associated with the numerator (weight). Although the adjusted model provides effect sizes with a clearer interpretation, it is susceptible to collider bias. We propose and validate a simple method of correcting for the genetic collider bias via leave-one-chromosome-out polygenic scoring.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565256">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565256" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565256">
        <p class="paperTitle">Polycomb protein binding and looping mediated by Polycomb Response Elements in the ON transcriptional state</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565256" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565256" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brown, J. L.; Zhang, L.; Rocha, P. P.; Kassis, J. A.; Sun, M.-a.</p>
        <p class="info">Score: 9.2, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565256' target='https://doi.org/10.1101/2023.11.02.565256'> 10.1101/2023.11.02.565256</a></p>
        <p class="abstract">Polycomb group proteins (PcG) mediate epigenetic silencing of important developmental genes and other targets. In Drosophila, canonical PcG-target genes contain Polycomb Response Elements (PREs) that recruit PcG protein complexes including PRC2 that tri-methylates H3K27 forming large H3K27me3 domains. In the OFF transcriptional state, PREs loop with each other and this looping strengthens silencing. Here we address the question of what PcG proteins bind to PREs when canonical PcG target genes are expressed, and whether PREs loop when these genes are ON. Our data show that the answer to this question is PRE-specific but general conclusions can be made. First, within a PcG-target gene, some regulatory DNA can remain covered with H3K27me3 and PcG proteins remain bound to PREs in these regions. Second, when PREs are within H3K27ac domains, PcG-binding decreases, however, this depends on the protein and PRE. The DNA binding protein GAF, and the PcG protein Ph remain at PREs even when other PcG proteins are greatly depleted. In the ON state, PREs can still loop with each other, but also form loops with presumptive enhancers. These data support the model that, in addition to their role in PcG silencing, PREs can act as &#34;promoter-tethering elements&#34; mediating interactions between promoter proximal PREs and distant enhancers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565278">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565278" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565278">
        <p class="paperTitle">H3K9me regulates heterochromatin silencing through incoherent feedforward loops</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565278" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565278" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yabe, K.; Asuka, K.; Oya, S.; Kakutani, T.; Hirayama, M.; Tanaka, Y.; Inagaki, S.</p>
        <p class="info">Score: 6.3, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565278' target='https://doi.org/10.1101/2023.11.02.565278'> 10.1101/2023.11.02.565278</a></p>
        <p class="abstract">Histone H3 lysine-9 methylation (H3K9me) is associated with condensed and transcriptionally inactive heterochromatin. Although it has long been known that H3K9me silences transcription to control a wide variety of biological phenomena in many eukaryotic species, how the silencing is regulated under the control of H3K9me is still largely unclear. Moreover, how cells delimit regions with H3K9me to avoid silencing essential genes remains unexplored. Here, using Arabidopsis genetic systems that induce H3K9me2 and its associated non-CG DNA methylation (mCH) in the transcribed region of genes de novo, we show that the accumulation of H3K9me2/mCH paradoxically also leads to the deposition of the euchromatic mark H3K36me3. This induction of H3K36me3 depends on a SET domain methyltransferase, ASHH3, and brings about anti-silencing by preventing the demethylation of H3K4me1 by LDL2, which mediates transcriptional silencing downstream of H3K9me2/mCH. H3K9me2/mCH-driven antagonistic actions of ASHH3-H3K36me3 and LDL2-H3K4me1-loss also regulate the de novo silencing of reactivated transposable elements (TEs). These results demonstrate that H3K9me2 both facilitates and impedes silencing, and the incoherent feedforward loops fine-tune the fate of genes and TEs. Our results illuminate a novel elaborate mechanism for partitioning chromatin domains and provide insights into the molecular basis underlying natural epigenetic variation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
