<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Associations between common genetic variants and income provide insights about the socioeconomic health gradient
Authors: Kweon, H.; Burik, C. A. P.; Ning, Y.; Ahlskog, R.; Xia, C.; Abner, E.; Bao, Y.; Bhatta, L.; Faquih, T.; Feijter, M.; Fisher, P.; Gelemanovic, A.; Giannelis, A.; Hottenga, J.-J.; Khalili, B.; Lee, Y.; Li-Gao, R.; Masso, J.; Myhre, R.; Palviainen, T.; Rietveld, C. A. P.; Teumer, A.; Verweij, R. M.; Willoughby, E. A.; Agerbo, E.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Associations between common genetic variants and income provide insights about the socioeconomic health gradient
Authors: Kweon, H.; Burik, C. A. P.; Ning, Y.; Ahlskog, R.; Xia, C.; Abner, E.; Bao, Y.; Bhatta, L.; Faquih, T.; Feijter, M.; Fisher, P.; Gelemanovic, A.; Giannelis, A.; Hottenga, J.-J.; Khalili, B.; Lee, Y.; Li-Gao, R.; Masso, J.; Myhre, R.; Palviainen, T.; Rietveld, C. A. P.; Teumer, A.; Verweij, R. M.; Willoughby, E. A.; Agerbo, E." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-14T10:38:40+00:00" />
<meta property="article:modified_time" content="2024-01-14T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Associations between common genetic variants and income provide insights about the socioeconomic health gradient
Authors: Kweon, H.; Burik, C. A. P.; Ning, Y.; Ahlskog, R.; Xia, C.; Abner, E.; Bao, Y.; Bhatta, L.; Faquih, T.; Feijter, M.; Fisher, P.; Gelemanovic, A.; Giannelis, A.; Hottenga, J.-J.; Khalili, B.; Lee, Y.; Li-Gao, R.; Masso, J.; Myhre, R.; Palviainen, T.; Rietveld, C. A. P.; Teumer, A.; Verweij, R. M.; Willoughby, E. A.; Agerbo, E."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Associations between common genetic variants and income provide insights about the socioeconomic health gradient\nAuthors: Kweon, H.; Burik, C. A. P.; Ning, Y.; Ahlskog, R.; Xia, C.; Abner, E.; Bao, Y.; Bhatta, L.; Faquih, T.; Feijter, M.; Fisher, P.; Gelemanovic, A.; Giannelis, A.; Hottenga, J.-J.; Khalili, B.; Lee, Y.; Li-Gao, R.; Masso, J.; Myhre, R.; Palviainen, T.; Rietveld, C. A. P.; Teumer, A.; Verweij, R. M.; Willoughby, E. A.; Agerbo, E.",
  "keywords": [
    
  ],
  "articleBody": " Associations between common genetic variants and income provide insights about the socioeconomic health gradient\nAuthors: Kweon, H.; Burik, C. A. P.; Ning, Y.; Ahlskog, R.; Xia, C.; Abner, E.; Bao, Y.; Bhatta, L.; Faquih, T.; Feijter, M.; Fisher, P.; Gelemanovic, A.; Giannelis, A.; Hottenga, J.-J.; Khalili, B.; Lee, Y.; Li-Gao, R.; Masso, J.; Myhre, R.; Palviainen, T.; Rietveld, C. A. P.; Teumer, A.; Verweij, R. M.; Willoughby, E. A.; Agerbo, E.; Bergmann, S.; Boomsma, D. I.; Borglum, A. D.; Brumpton, B. M.; Davies, N. M.; Esko, T.; Gordon, S. D.; Homuth, G.; Ikram, A.; Johannesson, M.; Kaprio, J.; Kidd, M. P.; Kutalik, Z.; Kwong, A. S. F.; Lee, J.; Luik, A.; Mook-Kanamori, D. O.; Magnus, P.; Marques-Vidal, P.; M\nScore: 48.9, Published: 2024-01-10 DOI: 10.1101/2024.01.09.574865\nWe conducted a genome-wide association study (GWAS) on income among individuals of European descent and leveraged the results to investigate the socio-economic health gradient (N=668,288). We found 162 genomic loci associated with a common genetic factor underlying various income measures, all with small effect sizes. Our GWAS-derived polygenic index captures 1 - 4% of income variance, with only one-fourth attributed to direct genetic effects. A phenome-wide association study using this polygenic index showed reduced risks for a broad spectrum of diseases, including hypertension, obesity, type 2 diabetes, coronary atherosclerosis, depression, asthma, and back pain. The income factor showed a substantial genetic correlation (0.92, s.e. = .006) with educational attainment (EA). Accounting for EAs genetic overlap with income revealed that the remaining genetic signal for higher income related to better mental health but reduced physical health benefits and increased participation in risky behaviours such as drinking and smoking.\nGene-environment correlation: The role of family environment in academic development\nAuthors: Zhou, Q.; Gidziela, A.; Allegrini, A. G.; Cheesman, R.; Wertz, J.; Maxwell, J.; Plomin, R.; Rimfeld, K.; Malanchini, M.\nScore: 21.4, Published: 2024-01-08 DOI: 10.1101/2024.01.05.574339\nAcademic achievement is partly heritable and highly polygenic. However, genetic effects on academic achievement are not independent of environmental processes. We investigated whether aspects of the family environment mediated genetic effects on academic achievement across development. Our sample included 5,151 children who participated in the Twins Early Development Study, as well as their parents and teachers. Data on academic achievement and family environments were available at ages 7, 9, 12 and 16. We computed educational attainment polygenic scores (PGS) and further separated genetic effects into cognitive and noncognitive PGS. Three core findings emerged. First, aspects of the family environment, but not the wider neighbourhood context, consistently mediated the PGS effects on achievement across development, accounting for up to 34.3% of the total effect. Family characteristics mattered beyond socio-economic status. Second, family environments were more robustly linked to noncognitive PGS effects on academic achievement than cognitive PGS effects. Third, when we investigated whether environmental mediation effects could also be observed when considering differences between siblings, adjusting for family fixed effects, we found that environmental mediation was nearly exclusively observed between families. This is consistent with the proposition that family environmental contexts contribute to academic development via passive gene-environment correlation processes. Our results show how parents shape environments that foster their childrens academic development partly based on their own genetic disposition, particularly towards noncognitive skills.\nSingle-step generation of homozygous knock-out/knock-in individuals in an extremotolerant parthenogenetic tardigrade using DIPA-CRISPR\nAuthors: Kondo, K.; Tanaka, A.; Kunieda, T.\nScore: 15.8, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575120\nTardigrades are small aquatic invertebrates known for their remarkable tolerance to diverse extreme stresses. To elucidate the in vivo mechanisms underlying this extraordinary resilience, the genetic manipulation methods in tardigrades have long been desired. Despite our prior success in somatic cell gene-editing by microinjecting Cas9 ribonucleoproteins (RNPs) into the body cavity of tardigrades, the generation of gene-edited individuals remained elusive. In this study, employing an extremotolerant parthenogenetic tardigrade species, Ramazzottius varieornatus, we established conditions conductive to generating gene-edited tardigrade individuals. Drawing inspiration from the direct parental CRIPSR (DIPA-CRISPR) technique employed in several insects, we simply injected a concentrated Cas9 RNP solution into the body cavity of parental females shortly before their initial oviposition. This approach yielded gene-edited G0 progeny. Notably, only a single allele was predominantly detected at the target locus for each G0 individual, indicative of homozygous mutations. Through co-injecting single-stranded oligodeoxynucleotides (ssODNs) with Cas9 RNPs, we achieved the generation of homozygously knocked-in G0 progeny and these edited-alleles were inherited by G1/G2 progeny. This establishment of a simple method for generating homozygous knock-out/knock-in individuals not only facilitates in vivo analyses of molecular mechanisms underpinning extreme tolerance but also opens avenues for exploring various topics, including Evo-Devo, in tardigrades.\nMultiple DNA repair pathways prevent acetaldehyde-induced mutagenesis in yeast\nAuthors: Porcher, L.; Vijayraghavan, S.; McCollum, J.; Mieczkowski, P. A.; Saini, N.\nScore: 10.0, Published: 2024-01-08 DOI: 10.1101/2024.01.07.574575\nAcetaldehyde is the primary metabolite of alcohol and is present in many environmental sources including tobacco smoke. Acetaldehyde is genotoxic, whereby it can form DNA adducts and lead to mutagenesis. Individuals with defects in acetaldehyde clearance pathways have increased susceptibility to alcohol-associated cancers. Moreover, a mutation signature specific to acetaldehyde exposure is widespread in alcohol and smoking-associated cancers. However, the pathways that repair acetaldehyde-induced DNA damage and thus prevent mutagenesis are vaguely understood. Here, we used Saccharomyces cerevisiae to systematically delete genes in each of the major DNA repair pathways to identify those that alter acetaldehyde-induced mutagenesis. We found that deletion of the nucleotide excision repair (NER) genes, RAD1 or RAD14, led to an increase in mutagenesis upon acetaldehyde exposure. Acetaldehyde-induced mutations were dependent on translesion synthesis as well as DNA inter-strand crosslink (ICL) repair in{Delta} rad1 strains. Moreover, whole genome sequencing of the mutated isolates demonstrated an increase in C[-\u0026gt;]A changes coupled with an enrichment of gCn[-\u0026gt;]A changes in the acetaldehyde-treated{Delta} rad1 isolates. The gCn[-\u0026gt;]A mutation signature has been shown to be diagnostic of acetaldehyde exposure in yeast and in human cancers. We also demonstrated that the deletion of the two DNA-protein crosslink (DPC) repair proteases, WSS1 and DDI1, also led to increased acetaldehyde-induced mutagenesis. Defects in base excision repair (BER) led to a mild increase in mutagenesis, while defects in mismatch repair (MMR), homologous recombination repair (HR) and post replicative repair pathways did not impact mutagenesis upon acetaldehyde exposure. Our results in yeast were further corroborated upon analysis of whole exome sequenced liver cancers, wherein, tumors with defects in ERCC1 and ERCC4 (NER), FANCD2 (ICL repair) or SPRTN (DPC repair) carried a higher gCn[-\u0026gt;]A mutation load than tumors with no deleterious mutations in these genes. Our findings demonstrate that multiple DNA repair pathways protect against acetaldehyde-induced mutagenesis.\nRare genetic variation in Fibronectin 1 (FN1) protects against APOEe4 in Alzheimer's disease\nAuthors: Bhattarai, P.; Gunasekaran, T. I.; Reyes-Dumeyer, D.; Juelich, D.; Tayran, H.; Yilmaz, E.; Flaherty, D.; Rafael, L.; Martin, M.; Diones, R.; Patricia, R.; Ertekin-Taner, N.; Teich, A. F.; Dennis, D. W.; Scott, H.; Mayeux, R.; Kizil, C.; Vardarajan, B. N.\nScore: 14.7, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573895\nThe risk of developing Alzheimers disease (AD) significantly increases in individuals carrying the APOE{varepsilon}4 allele. Elderly cognitively healthy individuals with APOE{varepsilon}4 also exist, suggesting the presence of cellular mechanisms that counteract the pathological effects of APOE{varepsilon}4; however, these mechanisms are unknown. We hypothesized that APOE{varepsilon}4 carriers without dementia might carry genetic variations that could protect them from developing APOE{varepsilon}4-mediated AD pathology. To test this, we leveraged whole genome sequencing (WGS) data in National Institute on Aging Alzheimers Disease Family Based Study (NIA-AD FBS), Washington Heights/Inwood Columbia Aging Project (WHICAP), and Estudio Familiar de Influencia Genetica en Alzheimer (EFIGA) cohorts and identified potentially protective variants segregating exclusively among unaffected APOE{varepsilon}4 carriers. In homozygous unaffected carriers above 70 years old, we identified 510 rare coding variants. Pathway analysis of the genes harboring these variants showed significant enrichment in extracellular matrix (ECM)-related processes, suggesting protective effects of functional modifications in ECM proteins. We prioritized two genes that were highly represented in the ECM-related gene ontology terms, (FN1) and collagen type VI alpha 2 chain (COL6A2) and are known to be expressed at the blood-brain barrier (BBB), for postmortem validation and in vivo functional studies. The FN1 and COL6A2 protein levels were increased at the BBB in APOE{varepsilon}4 carriers with AD. Brain expression of cognitively unaffected homozygous APOE{varepsilon}4 carriers had significantly lower FN1 deposition and less reactive gliosis compared to homozygous APOE{varepsilon}4 carriers with AD, suggesting that FN1 might be a downstream driver of APOE{varepsilon}4-mediated AD-related pathology and cognitive decline. To validate our findings, we used zebrafish models with loss-of-function (LOF) mutations in fn1b - the ortholog for human FN1. We found that fibronectin LOF reduced gliosis, enhanced gliovascular remodeling and potentiated the microglial response, suggesting that pathological accumulation of FN1 could impair toxic protein clearance, which is ameliorated with FN1 LOF. Our study suggests vascular deposition of FN1 is related to the pathogenicity of APOE{varepsilon}4, LOF variants in FN1 may reduce APOE{varepsilon}4-related AD risk, providing novel clues to potential therapeutic interventions targeting the ECM to mitigate AD risk.\nDetailed mechanisms for unintended large DNA deletions with CRISPR, base editors, and prime editors\nAuthors: Hwang, G.-h.; Kim, S.-H.; Oh, M.; Kim, S.; Habib, O.; Jang, H.-K.; Kim, H. S.; Kim, C. H.; Kim, S.; Bae, S.\nScore: 9.8, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574288\nCRISPR-Cas9 nucleases are versatile tools for genetic engineering cells and function by producing targeted double-strand breaks (DSBs) in the DNA sequence. However, the unintended production of large deletions (\u003e100 bp) represents a challenge to the effective application of this genome-editing system. We optimized a long-range amplicon sequencing system and developed a k-mer sequence-alignment algorithm to simultaneously detect small DNA alteration events and large DNA deletions. With this workflow, we determined that CRISPR-Cas9 induced large deletions at varying frequencies in cancer cell lines, stem cells, and primary T cells. With CRISPR interference screening, we determined that end resection and the subsequent TMEJ [DNA polymerase theta-mediated end joining] repair process produce most large deletions. Furthermore, base editors and prime editors also generated large deletions despite employing mutated Cas9 \"nickases\" that produce single-strand breaks. Our findings reveal an important limitation of current genome-editing tools and identify strategies for mitigating unwanted large deletion events.\nPicturing plant biodiversity from airborne environmental DNA.\nAuthors: Thuillet, A.-C.; Morisot, D.; Renno, J.-F.; Scarcelli, N.; Serret, J.; Mariac, C.\nScore: 1.5, Published: 2024-01-13 DOI: 10.1101/2024.01.11.571706\nWhile eDNA approaches have gained interest over the past decades all types of organisms have not been addressed evenly. In particular terrestrial plants have been the subject of less attention. Here we address the possibility to represent plant biodiversity from airborne environmental DNA (eDNA) sampling and metabarcoding. We collected air using a biological air sampler in the Botanical Garden of Montpellier (France) and compared the list of revealed plant species to the botanical inventory of the Garden. Ninety-two plant species could be detected from three sampling points across the 4,6 ha of the Garden, after one hour sampling allowing to filter 9 m3 of air. We recorded the plants carrying flowers at the time of the experiment, which allowed us to estimate that plants flowering at the time of the sampling could be detected 10 times more easily than plants that were not, given the number of plants carrying flowers. However, flowering is far from being required as a vast majority of plants still was detected without flowering. We also show that not all species orders are detected with the same probability, tree species being better represented in the sample than herbal plants, given the number of trees present in the garden. Finally using diagnostic species, present only once in the garden, we estimate that the maximum sampling distance allowed by the biological air sampler is at least 110 m. Our study underlines that air sampling is a promising method for monitoring terrestrial plant biodiversity and highlights the parameters that should be adjusted to optimize the approach.\nExtreme restructuring of cis-regulatory regions controlling a deeply conserved plant stem cell regulator\nAuthors: Ciren, D.; Zebell, S.; Lippman, Z.\nScore: 24.2, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572550\nA striking paradox is that genes with conserved protein sequence, function and expression pattern over deep time often exhibit extremely divergent cis-regulatory sequences. It remains unclear how such drastic cis-regulatory evolution across species allows preservation of gene function, and to what extent these differences influence how cis-regulatory variation arising within species impacts phenotypic change. Here, we investigated these questions using a plant stem cell regulator conserved in expression pattern and function over [~]125 million years. Using in-vivo genome editing in two distantly related models, Arabidopsis thaliana (Arabidopsis) and Solanum lycopersicum (tomato), we generated over 70 deletion alleles in the upstream and downstream regions of the stem cell repressor gene CLAVATA3 (CLV3) and compared their individual and combined effects on a shared phenotype, the number of carpels that make fruits. We found that sequences upstream of tomato CLV3 are highly sensitive to even small perturbations compared to its downstream region. In contrast, Arabidopsis CLV3 function is tolerant to severe disruptions both upstream and downstream of the coding sequence. Combining upstream and downstream deletions also revealed a different regulatory outcome. Whereas phenotypic enhancement from adding downstream mutations was predominantly weak and additive in tomato, mutating both regions of Arabidopsis CLV3 caused substantial and synergistic effects, demonstrating distinct distribution and redundancy of functional cis-regulatory sequences. Our results demonstrate remarkable malleability in cis-regulatory structural organization of a deeply conserved plant stem cell regulator and suggest that major reconfiguration of cis-regulatory sequence space is a common yet cryptic evolutionary force altering genotype-to-phenotype relationships from regulatory variation in conserved genes. Finally, our findings underscore the need for lineage-specific dissection of the spatial architecture of cis-regulation to effectively engineer trait variation from conserved productivity genes in crops. Author summaryWe investigated the evolution of cis-regulatory elements (CREs) and their interactions in the regulation of a plant stem cell regulator gene, CLAVATA3 (CLV3), in Arabidopsis and tomato. Despite diverging [~]125 million years ago, the function and expression of CLV3 is conserved in these species; however, cis-regulatory sequences upstream and downstream have drastically diverged, preventing identification of conserved non-coding sequences between them. We used CRISPR-Cas9 to engineer dozens of mutations within the cis-regulatory regions of Arabidopsis and tomato CLV3. In tomato, our results show that tomato CLV3 function primarily relies on interactions among CREs in the 5 non-coding region, unlike Arabidopsis CLV3, which depends on a more balanced distribution of functional CREs between the 5 and 3 regions. Therefore, despite a high degree of functional conservation, our study demonstrates divergent regulatory strategies between two distantly related CLV3 orthologs, with substantial alterations in regulatory sequences, their spatial arrangement, and their relative effects on CLV3 regulation. These results suggest that regulatory regions are not only extremely robust to mutagenesis, but also that the sequences underlying this robustness can be lineage-specific for conserved genes, due to the complex and often redundant interactions among CREs that ensure proper gene function amidst large-scale sequence turnover.\nIdentification of arginine-vasopressin receptor 1a (Avpr1a/AVPR1A) as a novel candidate gene for chronic visceral pain.\nAuthors: Kader, L.; Willits, A.; Meriano, S. A.; Christianson, J. A.; La, J.-H.; Feng, B.; Knight, B.; Kosova, G.; Deberry, J.; Coates, M.; Hyams, J.; Baumbauer, K.; Young, E. E.\nScore: 1.6, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572390\nChronic abdominal pain in the absence of ongoing disease is the hallmark of disorders of gut-brain interaction (DGBIs), including irritable bowel syndrome (IBS). While the etiology of DGBIs remains poorly understood, there is evidence that both genetic and environmental factors play a role. In this study, we report the identification and validation of Avpr1a as a novel candidate gene for visceral hypersensitivity (VH), a primary peripheral mechanism underlying abdominal pain in DGBI/IBS. Comparing two C57BL/6 (BL/6) substrains (C57BL/6NTac and C57BL/6J) revealed differential susceptibility to the development of chronic VH following intrarectal zymosan (ZYM) instillation, a validated preclinical model for post-inflammatory IBS. Using whole genome sequencing, we identified a SNP differentiating the two strains in the 5 intergenic region upstream of Avpr1a, encoding the protein arginine-vasopressin receptor 1A (AVPR1A). We used behavioral, histological, and molecular approaches to identify distal colon- specific gene expression differences and neuronal hyperresponsiveness covarying with Avpr1a genotype and VH susceptibility. While the two BL/6 substrains did not differ across other gastrointestinal (GI) phenotypes (e.g., GI motility), VH-susceptible BL/6NTac mice had higher colonic Avpr1a mRNA and protein expression. Moreover, neurons of the enteric nervous system were hyperresponsive to the AVPR1A agonist AVP, suggesting a role for enteric neurons in the pathology underlying VH. These results parallel our findings that patients colonic Avpr1a mRNA expression was higher in patients with higher pain ratings. Taken together, these findings implicate differential regulation of Avpr1a as a novel mechanism of VH-susceptibility as well as a potential therapeutic target specific to VH. SummaryA combination of approaches, from genomic analysis to functional analyses, confirm Avpr1a as a high priority candidate gene for visceral pain.\nComplementary approaches to dissect late leaf rust resistance in an interspecific raspberry population\nAuthors: Prado, M.; Silva, A. V. d.; Campos, G. R.; Borges, K. L. R.; Yassue, R. M.; Husein, G.; Akens, F. F.; Sposito, M. B.; Amorim, L.; Behrouzi, P.; Bustos-Korts, D.; Fritsche Neto, R.\nScore: 1.2, Published: 2024-01-07 DOI: 10.1101/2024.01.04.574234\nOver the last ten years, global raspberry production has increased by 47.89%, based on the red species (Rubus idaeus). However, the black raspberry species (Rubus occidentalis), although less consumed, is resistant to one of the most important diseases for the crop, the late leaf rust caused by Acculeastrum americanum fungus, to which the red ones are susceptible. In this context, genetic resistance is the most sustainable way to control the disease, mainly because there are no registered fungicides for late leaf rust in the crop in Brazil. Therefore, the aim was to understand the genetic architecture that controls resistance to late rust in raspberries. For that, we used an interspecific diversity panel between the cited above species, two different statistical approaches to associate the phenotypes to the markers (GWAS and copula graphical models), and two phenotyping methodologies from the first to the seventeenth day after inoculation (high-throughput phenotyping with a multispectral camera and traditional phenotyping by disease severity scores). Our findings indicate that a locus of higher effect possibly controls the resistance to late leaf rust, as both GWAS and the network suggested the same marker. Furthermore, a candidate defense-related gene cluster is close to this marker. Finally, the best stage to evaluate for disease severity is thirteen days after inoculation, confirmed by both traditional and high-throughput phenotyping. Although the network and GWAS indicated the same higher effect genomic region, the network identified other different regions complementing the genetic control comprehension.\n",
  "wordCount" : "3074",
  "inLanguage": "en",
  "datePublished": "2024-01-14T10:38:40Z",
  "dateModified": "2024-01-14T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574865">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574865" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574865">
        <p class="paperTitle">Associations between common genetic variants and income provide insights about the socioeconomic health gradient</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574865" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574865" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kweon, H.; Burik, C. A. P.; Ning, Y.; Ahlskog, R.; Xia, C.; Abner, E.; Bao, Y.; Bhatta, L.; Faquih, T.; Feijter, M.; Fisher, P.; Gelemanovic, A.; Giannelis, A.; Hottenga, J.-J.; Khalili, B.; Lee, Y.; Li-Gao, R.; Masso, J.; Myhre, R.; Palviainen, T.; Rietveld, C. A. P.; Teumer, A.; Verweij, R. M.; Willoughby, E. A.; Agerbo, E.; Bergmann, S.; Boomsma, D. I.; Borglum, A. D.; Brumpton, B. M.; Davies, N. M.; Esko, T.; Gordon, S. D.; Homuth, G.; Ikram, A.; Johannesson, M.; Kaprio, J.; Kidd, M. P.; Kutalik, Z.; Kwong, A. S. F.; Lee, J.; Luik, A.; Mook-Kanamori, D. O.; Magnus, P.; Marques-Vidal, P.; M</p>
        <p class="info">Score: 48.9, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574865' target='https://doi.org/10.1101/2024.01.09.574865'> 10.1101/2024.01.09.574865</a></p>
        <p class="abstract">We conducted a genome-wide association study (GWAS) on income among individuals of European descent and leveraged the results to investigate the socio-economic health gradient (N=668,288). We found 162 genomic loci associated with a common genetic factor underlying various income measures, all with small effect sizes. Our GWAS-derived polygenic index captures 1 - 4% of income variance, with only one-fourth attributed to direct genetic effects. A phenome-wide association study using this polygenic index showed reduced risks for a broad spectrum of diseases, including hypertension, obesity, type 2 diabetes, coronary atherosclerosis, depression, asthma, and back pain. The income factor showed a substantial genetic correlation (0.92, s.e. = .006) with educational attainment (EA). Accounting for EAs genetic overlap with income revealed that the remaining genetic signal for higher income related to better mental health but reduced physical health benefits and increased participation in risky behaviours such as drinking and smoking.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574339">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574339" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574339">
        <p class="paperTitle">Gene-environment correlation: The role of family environment in academic development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574339" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574339" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, Q.; Gidziela, A.; Allegrini, A. G.; Cheesman, R.; Wertz, J.; Maxwell, J.; Plomin, R.; Rimfeld, K.; Malanchini, M.</p>
        <p class="info">Score: 21.4, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574339' target='https://doi.org/10.1101/2024.01.05.574339'> 10.1101/2024.01.05.574339</a></p>
        <p class="abstract">Academic achievement is partly heritable and highly polygenic. However, genetic effects on academic achievement are not independent of environmental processes. We investigated whether aspects of the family environment mediated genetic effects on academic achievement across development. Our sample included 5,151 children who participated in the Twins Early Development Study, as well as their parents and teachers. Data on academic achievement and family environments were available at ages 7, 9, 12 and 16. We computed educational attainment polygenic scores (PGS) and further separated genetic effects into cognitive and noncognitive PGS. Three core findings emerged. First, aspects of the family environment, but not the wider neighbourhood context, consistently mediated the PGS effects on achievement across development, accounting for up to 34.3% of the total effect. Family characteristics mattered beyond socio-economic status. Second, family environments were more robustly linked to noncognitive PGS effects on academic achievement than cognitive PGS effects. Third, when we investigated whether environmental mediation effects could also be observed when considering differences between siblings, adjusting for family fixed effects, we found that environmental mediation was nearly exclusively observed between families. This is consistent with the proposition that family environmental contexts contribute to academic development via passive gene-environment correlation processes. Our results show how parents shape environments that foster their childrens academic development partly based on their own genetic disposition, particularly towards noncognitive skills.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575120" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575120">
        <p class="paperTitle">Single-step generation of homozygous knock-out/knock-in individuals in an extremotolerant parthenogenetic tardigrade using DIPA-CRISPR</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kondo, K.; Tanaka, A.; Kunieda, T.</p>
        <p class="info">Score: 15.8, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575120' target='https://doi.org/10.1101/2024.01.10.575120'> 10.1101/2024.01.10.575120</a></p>
        <p class="abstract">Tardigrades are small aquatic invertebrates known for their remarkable tolerance to diverse extreme stresses. To elucidate the in vivo mechanisms underlying this extraordinary resilience, the genetic manipulation methods in tardigrades have long been desired. Despite our prior success in somatic cell gene-editing by microinjecting Cas9 ribonucleoproteins (RNPs) into the body cavity of tardigrades, the generation of gene-edited individuals remained elusive. In this study, employing an extremotolerant parthenogenetic tardigrade species, Ramazzottius varieornatus, we established conditions conductive to generating gene-edited tardigrade individuals. Drawing inspiration from the direct parental CRIPSR (DIPA-CRISPR) technique employed in several insects, we simply injected a concentrated Cas9 RNP solution into the body cavity of parental females shortly before their initial oviposition. This approach yielded gene-edited G0 progeny. Notably, only a single allele was predominantly detected at the target locus for each G0 individual, indicative of homozygous mutations. Through co-injecting single-stranded oligodeoxynucleotides (ssODNs) with Cas9 RNPs, we achieved the generation of homozygously knocked-in G0 progeny and these edited-alleles were inherited by G1/G2 progeny. This establishment of a simple method for generating homozygous knock-out/knock-in individuals not only facilitates in vivo analyses of molecular mechanisms underpinning extreme tolerance but also opens avenues for exploring various topics, including Evo-Devo, in tardigrades.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.574575">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.574575" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.574575">
        <p class="paperTitle">Multiple DNA repair pathways prevent acetaldehyde-induced mutagenesis in yeast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.574575" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.574575" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Porcher, L.; Vijayraghavan, S.; McCollum, J.; Mieczkowski, P. A.; Saini, N.</p>
        <p class="info">Score: 10.0, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.574575' target='https://doi.org/10.1101/2024.01.07.574575'> 10.1101/2024.01.07.574575</a></p>
        <p class="abstract">Acetaldehyde is the primary metabolite of alcohol and is present in many environmental sources including tobacco smoke. Acetaldehyde is genotoxic, whereby it can form DNA adducts and lead to mutagenesis. Individuals with defects in acetaldehyde clearance pathways have increased susceptibility to alcohol-associated cancers. Moreover, a mutation signature specific to acetaldehyde exposure is widespread in alcohol and smoking-associated cancers. However, the pathways that repair acetaldehyde-induced DNA damage and thus prevent mutagenesis are vaguely understood. Here, we used Saccharomyces cerevisiae to systematically delete genes in each of the major DNA repair pathways to identify those that alter acetaldehyde-induced mutagenesis. We found that deletion of the nucleotide excision repair (NER) genes, RAD1 or RAD14, led to an increase in mutagenesis upon acetaldehyde exposure. Acetaldehyde-induced mutations were dependent on translesion synthesis as well as DNA inter-strand crosslink (ICL) repair in{Delta} rad1 strains. Moreover, whole genome sequencing of the mutated isolates demonstrated an increase in C[-&amp;gt;]A changes coupled with an enrichment of gCn[-&amp;gt;]A changes in the acetaldehyde-treated{Delta} rad1 isolates. The gCn[-&amp;gt;]A mutation signature has been shown to be diagnostic of acetaldehyde exposure in yeast and in human cancers. We also demonstrated that the deletion of the two DNA-protein crosslink (DPC) repair proteases, WSS1 and DDI1, also led to increased acetaldehyde-induced mutagenesis. Defects in base excision repair (BER) led to a mild increase in mutagenesis, while defects in mismatch repair (MMR), homologous recombination repair (HR) and post replicative repair pathways did not impact mutagenesis upon acetaldehyde exposure. Our results in yeast were further corroborated upon analysis of whole exome sequenced liver cancers, wherein, tumors with defects in ERCC1 and ERCC4 (NER), FANCD2 (ICL repair) or SPRTN (DPC repair) carried a higher gCn[-&amp;gt;]A mutation load than tumors with no deleterious mutations in these genes. Our findings demonstrate that multiple DNA repair pathways protect against acetaldehyde-induced mutagenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573895">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573895" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573895">
        <p class="paperTitle">Rare genetic variation in Fibronectin 1 (FN1) protects against APOEe4 in Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573895" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573895" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhattarai, P.; Gunasekaran, T. I.; Reyes-Dumeyer, D.; Juelich, D.; Tayran, H.; Yilmaz, E.; Flaherty, D.; Rafael, L.; Martin, M.; Diones, R.; Patricia, R.; Ertekin-Taner, N.; Teich, A. F.; Dennis, D. W.; Scott, H.; Mayeux, R.; Kizil, C.; Vardarajan, B. N.</p>
        <p class="info">Score: 14.7, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573895' target='https://doi.org/10.1101/2024.01.02.573895'> 10.1101/2024.01.02.573895</a></p>
        <p class="abstract">The risk of developing Alzheimers disease (AD) significantly increases in individuals carrying the APOE{varepsilon}4 allele. Elderly cognitively healthy individuals with APOE{varepsilon}4 also exist, suggesting the presence of cellular mechanisms that counteract the pathological effects of APOE{varepsilon}4; however, these mechanisms are unknown. We hypothesized that APOE{varepsilon}4 carriers without dementia might carry genetic variations that could protect them from developing APOE{varepsilon}4-mediated AD pathology. To test this, we leveraged whole genome sequencing (WGS) data in National Institute on Aging Alzheimers Disease Family Based Study (NIA-AD FBS), Washington Heights/Inwood Columbia Aging Project (WHICAP), and Estudio Familiar de Influencia Genetica en Alzheimer (EFIGA) cohorts and identified potentially protective variants segregating exclusively among unaffected APOE{varepsilon}4 carriers. In homozygous unaffected carriers above 70 years old, we identified 510 rare coding variants. Pathway analysis of the genes harboring these variants showed significant enrichment in extracellular matrix (ECM)-related processes, suggesting protective effects of functional modifications in ECM proteins. We prioritized two genes that were highly represented in the ECM-related gene ontology terms, (FN1) and collagen type VI alpha 2 chain (COL6A2) and are known to be expressed at the blood-brain barrier (BBB), for postmortem validation and in vivo functional studies. The FN1 and COL6A2 protein levels were increased at the BBB in APOE{varepsilon}4 carriers with AD. Brain expression of cognitively unaffected homozygous APOE{varepsilon}4 carriers had significantly lower FN1 deposition and less reactive gliosis compared to homozygous APOE{varepsilon}4 carriers with AD, suggesting that FN1 might be a downstream driver of APOE{varepsilon}4-mediated AD-related pathology and cognitive decline. To validate our findings, we used zebrafish models with loss-of-function (LOF) mutations in fn1b - the ortholog for human FN1. We found that fibronectin LOF reduced gliosis, enhanced gliovascular remodeling and potentiated the microglial response, suggesting that pathological accumulation of FN1 could impair toxic protein clearance, which is ameliorated with FN1 LOF. Our study suggests vascular deposition of FN1 is related to the pathogenicity of APOE{varepsilon}4, LOF variants in FN1 may reduce APOE{varepsilon}4-related AD risk, providing novel clues to potential therapeutic interventions targeting the ECM to mitigate AD risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574288">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574288" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574288">
        <p class="paperTitle">Detailed mechanisms for unintended large DNA deletions with CRISPR, base editors, and prime editors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574288" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574288" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hwang, G.-h.; Kim, S.-H.; Oh, M.; Kim, S.; Habib, O.; Jang, H.-K.; Kim, H. S.; Kim, C. H.; Kim, S.; Bae, S.</p>
        <p class="info">Score: 9.8, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574288' target='https://doi.org/10.1101/2024.01.04.574288'> 10.1101/2024.01.04.574288</a></p>
        <p class="abstract">CRISPR-Cas9 nucleases are versatile tools for genetic engineering cells and function by producing targeted double-strand breaks (DSBs) in the DNA sequence. However, the unintended production of large deletions (&gt;100 bp) represents a challenge to the effective application of this genome-editing system. We optimized a long-range amplicon sequencing system and developed a k-mer sequence-alignment algorithm to simultaneously detect small DNA alteration events and large DNA deletions. With this workflow, we determined that CRISPR-Cas9 induced large deletions at varying frequencies in cancer cell lines, stem cells, and primary T cells. With CRISPR interference screening, we determined that end resection and the subsequent TMEJ [DNA polymerase theta-mediated end joining] repair process produce most large deletions. Furthermore, base editors and prime editors also generated large deletions despite employing mutated Cas9 &#34;nickases&#34; that produce single-strand breaks. Our findings reveal an important limitation of current genome-editing tools and identify strategies for mitigating unwanted large deletion events.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.571706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.571706" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.571706">
        <p class="paperTitle">Picturing plant biodiversity from airborne environmental DNA.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.571706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.571706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thuillet, A.-C.; Morisot, D.; Renno, J.-F.; Scarcelli, N.; Serret, J.; Mariac, C.</p>
        <p class="info">Score: 1.5, Published: 2024-01-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.571706' target='https://doi.org/10.1101/2024.01.11.571706'> 10.1101/2024.01.11.571706</a></p>
        <p class="abstract">While eDNA approaches have gained interest over the past decades all types of organisms have not been addressed evenly. In particular terrestrial plants have been the subject of less attention. Here we address the possibility to represent plant biodiversity from airborne environmental DNA (eDNA) sampling and metabarcoding. We collected air using a biological air sampler in the Botanical Garden of Montpellier (France) and compared the list of revealed plant species to the botanical inventory of the Garden. Ninety-two plant species could be detected from three sampling points across the 4,6 ha of the Garden, after one hour sampling allowing to filter 9 m3 of air. We recorded the plants carrying flowers at the time of the experiment, which allowed us to estimate that plants flowering at the time of the sampling could be detected 10 times more easily than plants that were not, given the number of plants carrying flowers. However, flowering is far from being required as a vast majority of plants still was detected without flowering. We also show that not all species orders are detected with the same probability, tree species being better represented in the sample than herbal plants, given the number of trees present in the garden. Finally using diagnostic species, present only once in the garden, we estimate that the maximum sampling distance allowed by the biological air sampler is at least 110 m. Our study underlines that air sampling is a promising method for monitoring terrestrial plant biodiversity and highlights the parameters that should be adjusted to optimize the approach.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572550">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572550" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572550">
        <p class="paperTitle">Extreme restructuring of cis-regulatory regions controlling a deeply conserved plant stem cell regulator</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572550" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572550" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ciren, D.; Zebell, S.; Lippman, Z.</p>
        <p class="info">Score: 24.2, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572550' target='https://doi.org/10.1101/2023.12.20.572550'> 10.1101/2023.12.20.572550</a></p>
        <p class="abstract">A striking paradox is that genes with conserved protein sequence, function and expression pattern over deep time often exhibit extremely divergent cis-regulatory sequences. It remains unclear how such drastic cis-regulatory evolution across species allows preservation of gene function, and to what extent these differences influence how cis-regulatory variation arising within species impacts phenotypic change. Here, we investigated these questions using a plant stem cell regulator conserved in expression pattern and function over [~]125 million years. Using in-vivo genome editing in two distantly related models, Arabidopsis thaliana (Arabidopsis) and Solanum lycopersicum (tomato), we generated over 70 deletion alleles in the upstream and downstream regions of the stem cell repressor gene CLAVATA3 (CLV3) and compared their individual and combined effects on a shared phenotype, the number of carpels that make fruits. We found that sequences upstream of tomato CLV3 are highly sensitive to even small perturbations compared to its downstream region. In contrast, Arabidopsis CLV3 function is tolerant to severe disruptions both upstream and downstream of the coding sequence. Combining upstream and downstream deletions also revealed a different regulatory outcome. Whereas phenotypic enhancement from adding downstream mutations was predominantly weak and additive in tomato, mutating both regions of Arabidopsis CLV3 caused substantial and synergistic effects, demonstrating distinct distribution and redundancy of functional cis-regulatory sequences. Our results demonstrate remarkable malleability in cis-regulatory structural organization of a deeply conserved plant stem cell regulator and suggest that major reconfiguration of cis-regulatory sequence space is a common yet cryptic evolutionary force altering genotype-to-phenotype relationships from regulatory variation in conserved genes. Finally, our findings underscore the need for lineage-specific dissection of the spatial architecture of cis-regulation to effectively engineer trait variation from conserved productivity genes in crops.

Author summaryWe investigated the evolution of cis-regulatory elements (CREs) and their interactions in the regulation of a plant stem cell regulator gene, CLAVATA3 (CLV3), in Arabidopsis and tomato. Despite diverging [~]125 million years ago, the function and expression of CLV3 is conserved in these species; however, cis-regulatory sequences upstream and downstream have drastically diverged, preventing identification of conserved non-coding sequences between them. We used CRISPR-Cas9 to engineer dozens of mutations within the cis-regulatory regions of Arabidopsis and tomato CLV3. In tomato, our results show that tomato CLV3 function primarily relies on interactions among CREs in the 5 non-coding region, unlike Arabidopsis CLV3, which depends on a more balanced distribution of functional CREs between the 5 and 3 regions. Therefore, despite a high degree of functional conservation, our study demonstrates divergent regulatory strategies between two distantly related CLV3 orthologs, with substantial alterations in regulatory sequences, their spatial arrangement, and their relative effects on CLV3 regulation. These results suggest that regulatory regions are not only extremely robust to mutagenesis, but also that the sequences underlying this robustness can be lineage-specific for conserved genes, due to the complex and often redundant interactions among CREs that ensure proper gene function amidst large-scale sequence turnover.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572390">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572390" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572390">
        <p class="paperTitle">Identification of arginine-vasopressin receptor 1a (Avpr1a/AVPR1A) as a novel candidate gene for chronic visceral pain.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572390" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572390" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kader, L.; Willits, A.; Meriano, S. A.; Christianson, J. A.; La, J.-H.; Feng, B.; Knight, B.; Kosova, G.; Deberry, J.; Coates, M.; Hyams, J.; Baumbauer, K.; Young, E. E.</p>
        <p class="info">Score: 1.6, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572390' target='https://doi.org/10.1101/2023.12.19.572390'> 10.1101/2023.12.19.572390</a></p>
        <p class="abstract">Chronic abdominal pain in the absence of ongoing disease is the hallmark of disorders of gut-brain interaction (DGBIs), including irritable bowel syndrome (IBS). While the etiology of DGBIs remains poorly understood, there is evidence that both genetic and environmental factors play a role. In this study, we report the identification and validation of Avpr1a as a novel candidate gene for visceral hypersensitivity (VH), a primary peripheral mechanism underlying abdominal pain in DGBI/IBS. Comparing two C57BL/6 (BL/6) substrains (C57BL/6NTac and C57BL/6J) revealed differential susceptibility to the development of chronic VH following intrarectal zymosan (ZYM) instillation, a validated preclinical model for post-inflammatory IBS. Using whole genome sequencing, we identified a SNP differentiating the two strains in the 5 intergenic region upstream of Avpr1a, encoding the protein arginine-vasopressin receptor 1A (AVPR1A). We used behavioral, histological, and molecular approaches to identify distal colon- specific gene expression differences and neuronal hyperresponsiveness covarying with Avpr1a genotype and VH susceptibility. While the two BL/6 substrains did not differ across other gastrointestinal (GI) phenotypes (e.g., GI motility), VH-susceptible BL/6NTac mice had higher colonic Avpr1a mRNA and protein expression. Moreover, neurons of the enteric nervous system were hyperresponsive to the AVPR1A agonist AVP, suggesting a role for enteric neurons in the pathology underlying VH. These results parallel our findings that patients colonic Avpr1a mRNA expression was higher in patients with higher pain ratings. Taken together, these findings implicate differential regulation of Avpr1a as a novel mechanism of VH-susceptibility as well as a potential therapeutic target specific to VH.

SummaryA combination of approaches, from genomic analysis to functional analyses, confirm Avpr1a as a high priority candidate gene for visceral pain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574234">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574234" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574234">
        <p class="paperTitle">Complementary approaches to dissect late leaf rust resistance in an interspecific raspberry population</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574234" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574234" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prado, M.; Silva, A. V. d.; Campos, G. R.; Borges, K. L. R.; Yassue, R. M.; Husein, G.; Akens, F. F.; Sposito, M. B.; Amorim, L.; Behrouzi, P.; Bustos-Korts, D.; Fritsche Neto, R.</p>
        <p class="info">Score: 1.2, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574234' target='https://doi.org/10.1101/2024.01.04.574234'> 10.1101/2024.01.04.574234</a></p>
        <p class="abstract">Over the last ten years, global raspberry production has increased by 47.89%, based on the red species (Rubus idaeus). However, the black raspberry species (Rubus occidentalis), although less consumed, is resistant to one of the most important diseases for the crop, the late leaf rust caused by Acculeastrum americanum fungus, to which the red ones are susceptible. In this context, genetic resistance is the most sustainable way to control the disease, mainly because there are no registered fungicides for late leaf rust in the crop in Brazil. Therefore, the aim was to understand the genetic architecture that controls resistance to late rust in raspberries. For that, we used an interspecific diversity panel between the cited above species, two different statistical approaches to associate the phenotypes to the markers (GWAS and copula graphical models), and two phenotyping methodologies from the first to the seventeenth day after inoculation (high-throughput phenotyping with a multispectral camera and traditional phenotyping by disease severity scores). Our findings indicate that a locus of higher effect possibly controls the resistance to late leaf rust, as both GWAS and the network suggested the same marker. Furthermore, a candidate defense-related gene cluster is close to this marker. Finally, the best stage to evaluate for disease severity is thirteen days after inoculation, confirmed by both traditional and high-throughput phenotyping. Although the network and GWAS indicated the same higher effect genomic region, the network identified other different regions complementing the genetic control comprehension.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
