<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization
Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.
Score: 397.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478
Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization
Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.
Score: 397.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478
Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization
Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.
Score: 397.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478
Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization\nAuthors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.\nScore: 397.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478\nAging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging.",
  "keywords": [
    
  ],
  "articleBody": " Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization\nAuthors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.\nScore: 397.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478\nAging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging. Epigenetic clocks have been shown to be valuable tools for measuring biological age in a variety of tissues and samples. As such, these epigenetic clocks can determine accelerated biological aging and time-to-mortality across various tissues. Previous reports have shown accelerated biological aging and telomere attrition acceleration following SARS-CoV-2 infection. However, the effect of accelerated epigenetic aging on outcome (death/recovery) in COVID-19 patients with Acute Respiratory Distress Syndrome (ARDS) has not been well investigated. In this study, we measured DNA methylation age and telomere attrition in 87 severe COVID-19 cases with ARDS under mechanical ventilation. Furthermore, we compared dynamic changes in epigenetic aging across multiples time-points until recovery or death. Epigenetic age was measured using the Horvath, Hannum, DNAm skin and blood, GrimAge, and PhenoAge clocks, whereas telomere length was calculated using the surrogate marker DNAmTL. Our analysis revealed significant accelerated epigenetic aging but no telomere attrition acceleration in severe COVID-19 cases. In addition, we observed epigenetic age deceleration at inclusion vs end of follow-up in recovered but not in deceased COVID-19 cases using certain clocks. When comparing dynamic changes in epigenetic age acceleration (EAA), we detected higher EAA using both the Horvath and PhenoAge clocks in deceased vs recovered patients. The DNAmTL measurements revealed telomere attrition acceleration in deceased COVID19 patients between inclusion and end of follow-up as well as a significant change in dynamic telomere attrition acceleration when comparing patients who recovered vs those who died. In conclusion, EAA and telomere attrition acceleration was associated with treatment outcome in hospitalized COVID-19 Patients with ARDS. A better understanding of the long-term effects of EAA in COVID19 patients and how they might contribute to Long COVID symptoms in recovered individuals is urgently needed.\nisGWAS: ultra-high-throughput, scalable and equitable inference of genetic associations with disease\nAuthors: Foley, C. N.; Kuncheva, Z.; Marioni, R.; Runz, H.; Sun, B.\nScore: 18.4, Published: 2023-07-25 DOI: 10.1101/2023.07.21.550074\nGenome-wide association studies (GWAS) have proven a powerful tool for human geneticists to generate biological insights or hypotheses for drug discovery. Nevertheless, a dependency on sensitive individual-level data together with ever-increasing cohort sample sizes, numbers of variants and phenotypes studied put a strain on existing algorithms, limiting the GWAS approach from maximising potential. Here we present in-silico GWAS (isGWAS), a uniquely scalable algorithm to infer regression parameters in case-control GWAS from cohort-level summary data. For any sample size, isGWAS computes a variant-disease association parameter in ~1 millisecond, or ~11m variants in UK-Biobank within ~4 minutes (~1500-fold faster than state-of-the-art). Extensive simulations and empirical tests demonstrate that isGWAS results are highly comparable to traditional regression-based approaches. We further introduce a heuristic re-sampling algorithm, leapfrog re-sampler (LRS), to extrapolate association results to semi-virtually enlarged cohorts. Owing to significant computational gains we anticipate a broad use of isGWAS and LRS which are customizable on a web interface.\nA small RNA from a natural bacterial pathogen of C. elegans induces transgenerational inheritance of learned avoidance\nAuthors: Sengupta, T.; Ange, J. S.; Moore, R. S.; Kaletsky, R. T.; Marogi, J. T.; Myhrvold, C.; Gitai, Z.; Murphy, C. T.\nScore: 12.5, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549962\nA small RNA from a clinical isolate of Pseudomonas aeruginosa, PA14, induces learned avoidance and its transgenerational inheritance in C. elegans. However, it is not known if small RNAs from bacteria found in C. elegans natural habitat can regulate host behavior and produce heritable behavioral effects. Here we found that GRb0427, a pathogenic Pseudomonas vranovensis strain isolated from the C. elegans microbiota, similarly affects worm behavior: worms learn to avoid this pathogenic bacterium following exposure, and this learned avoidance is inherited for four generations. The learned response is entirely mediated by small RNAs, which induce avoidance and transgenerational inheritance in both the laboratory N2 strain and a natural isolate of C. elegans, JU1580, providing further support that such mechanisms of learning and inheritance exist in the wild. A small RNA, Pv1, matches the sequence of C. elegans maco-1, and is both necessary and sufficient to induce learned avoidance in worms. However, Pv1 also results in avoidance of a beneficial microbiome strain, P. mendocina; this maladaptive response may favor reversal of the transgenerational memory after a few generations. Our findings suggest that bacterial small RNA-mediated regulation of host behavior and its transgenerational inheritance are functional in C. elegans natural environment, and that different bacterial small RNA-mediated regulation systems evolved independently but define shared molecular features of bacterial small RNAs that produce transgenerationally-inherited effects.\nGenetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions\nAuthors: Suhre, K.\nScore: 11.9, Published: 2023-07-20 DOI: 10.1101/2023.07.19.549734\nProtein quantitative trait loci (pQTLs) are an invaluable source of information for drug target development as they provide genetic evidence to support protein function, suggest relationships between cis- and trans-associated proteins, and link proteins to disease where they collocate with genetic risk loci for clinical endpoints. Using the recently released Olink proteomics data for 1,463 proteins measured in over 54,000 samples of the UK Biobank we identified and replicated 4,248 associations with 2,821 ratios between protein levels (rQTLs) where the strengths of association at known pQTL loci increased by up to several hundred orders of magnitude. We attribute this increase in statistical power (p-gain) to accounting for genetic and non-genetic variance shared by the two proteins in the ratio pair. Protein pairs with a significant p-gain were 7.6-fold enriched in known protein-protein interactions, suggesting that their ratios reflect biological links between the implicated proteins. We then conducted a GWAS on the 2,821 ratios and identified 2,527 novel rQTLs, increasing the number of discovered genetic signals compared to the original protein-only GWAS by 24.7%. At examples we demonstrate that this approach can identify novel loci of clinical relevance, support causal gene identification, and reveal complex networks of interacting proteins. Taken together, our study adds significant value to the genetic insights that can be derived from the UKB proteomics data and motivates the wider use of ratios in large scale GWAS.\nCost-effectiveness of Precision Guided SIT for Control of Anopheles gambiae in the Upper River Region, The Gambia.\nAuthors: Gendron, W. A. C.; Raban, R.; Mondal, A.; Sanchez C., H. M.; Smidler, A.; Zilberman, D.; Ilboudo, P. G.; D'Alessandro, U.; Marshall, J. M.; Akbari, O. S.\nScore: 11.7, Published: 2023-07-24 DOI: 10.1101/2023.07.20.549762\nPrecision-guided sterile insect technique (pgSIT) is an extremely promising vector control intervention that can reduce and potentially eliminate the unacceptable malaria burden, particularly in sub-Saharan Africa. Here we explore the cost effectiveness of using this approach in Africa using mathematical modeling and economical analysis. Overall, we find that pgSIT represents a cost-effective and promising approach to A. gambiae control in The Gambia, with the potential to deliver significant economic and social benefits.\nGxE PRS: Genotype-environment interaction in polygenic risk score models for quantitative and binary traits\nAuthors: Jayasinghe, D.; Momin, M.; Beckmann, K.; Hypponen, E.; Benyamin, B.; Lee, S. H.\nScore: 7.6, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549816\nThe use of polygenic risk score (PRS) models has transformed the field of genetics by enabling the prediction of complex traits and diseases based on an individuals genetic profile. However, the impact of genotype-environment interaction (GxE) on the performance and applicability of PRS models remains a crucial aspect to be explored. Currently, existing GxE PRS models are often inappropriately used, which can result in inflated type 1 error rates and compromised results. In this study, we propose a novel GxE PRS model that correctly incorporates the GxE component to analyze complex traits and diseases. Through extensive simulations, we demonstrate that our proposed model outperforms existing models in terms of controlling type 1 error rates and enhancing statistical power. Furthermore, we apply the proposed model to real data, and report significant GxE effects. Specifically, we highlight the impact of our model on both quantitative and binary traits. For quantitative traits, we uncover the GxE modulation of genetic effects on body mass index (BMI) by alcohol intake frequency (ALC). In the case of binary traits, we identify the GxE modulation of genetic effects on hypertension (HYP) by waist-to-hip ratio (WHR). These findings underscore the importance of employing a robust model that effectively controls type 1 error rates, thus preventing the occurrence of spurious GxE signals. To facilitate the implementation of our approach, we have developed an innovative R software package called GxE PRS, specifically designed to detect and estimate GxE effects. Overall, our study highlights the importance of accurate GxE modeling and its implications for genetic risk prediction, while providing a practical tool to support further research in this area.\nCRIMP: A CRISPR/Cas9 Insertional Mutagenesis Protocol and the CRIMP Toolkit\nAuthors: Miles, L. B.; Calcinotto, V.; Sonntag, C.; Lee, C.; Bryson-Richardson, R. J.\nScore: 24.7, Published: 2023-07-14 DOI: 10.1101/2023.07.13.548929\nWe developed a highly efficient targeted insertional mutagenesis system, CRIMP, and an associated plasmid toolkit, CRIMPkit, that disrupts native gene expression by inducing complete transcriptional termination to produce null mutant alleles without inducing genetic compensation. The CRIMPkit contains over 30 ready-to-use plasmid vectors allowing easy and complete mutagenesis of any gene in any reading frame without requiring custom sequences, modification, or subcloning, and also provides a fluorescent readout of successfully mutagenised fish.\nT2T-YAO: a Telomere-to-telomere Assembled Diploid Reference Genome for Han Chinese\nAuthors: He, Y.; Chu, Y.; Guo, S.; Hu, J.; Li, R.; Zheng, Y.; Ma, X.; Du, Z.; Zhao, L.; Yu, W.; Xue, J.; Bian, W.; Yang, F.; Chen, X.; Zhang, P.; Wu, R.; Ma, Y.; Shao, C.; Chen, J.; Wang, J.; Li, J.; Wu, J.; Hu, X.; Long, Q.; Jiang, M.; Ye, H.; Song, S.; Li, G.; Wei, Y.; Xu, Y.; Ma, Y.; Chen, Y.; Wang, K.; Bao, J.; Xi, W.; Wang, F.; Ni, W.; Zhang, M.; Yu, Y.; Li, S.; Kang, Y.; Gao, Z.\nScore: 4.8, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549286\nSince its initial release in 2001, the human reference genome has been continuously improved in both continuity and accuracy, and the recently-released telomere-to-telomere version--T2T-CHM13--reaches its top quality after 20 years of effort. However, T2T-CHM13 does not represent an authentic diploid human genome, but rather one derived from a simplified, nearly homozygous genome of a hydatidiform mole cell line. To address this limitation and provide an alternative pertinent to the Chinese population, the largest ethnic group in the world, we have assembled a complete diploid human genome of a male Han Chinese, T2T-YAO, which includes telomere-to-telomere assemblies for all the 22+X+M and 22+Y chromosomes in his two haploids inherited separately from his parents. Both haplotypes contain no artificial sequences or model nucleotides and possess a high quality comparable to CHM13, with fewer than one error per [~]14 Mb. Derived from the individual who lives in the aboriginal region of Han Chinese, T2T-YAO shows clear ancestry and potential genetic continuity from the ancient ancestors of the Han population. Each haplotype of T2T-YAO possesses [~]340 Mb exclusive sequences and [~]3100 unique genes as compared to CHM13, and their genome sequences show greater genetic distance to CHM13 than to each other in terms of nucleotide polymorphism and structural variations. The construction of T2T-YAO would serve as a high-quality diploid reference that enables precise delineation of genomic variations in a haplotype-sensitive manner, which could advance our understandings in human evolution, hereditability of diseases and phenotypes, especially within the context of the unique variations of the Chinese population.\nIdentification of the yeast mannoprotein gene HZY1\tas a key genetic determinant for yeast-derived haze in beer\nAuthors: Lacy, K. M.; Mormando, R.; Smith, J. R.; Gibney, P. A.; Shaner, L. M.; Burns, L. T.\nScore: 6.6, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548400\nWith the sustained popularity of hazy IPAs, brewers have explored multiple approaches to maximizing stable haze that will remain in suspension throughout the shelf life of the beer. Our recent investigations into yeast-dependent haze have uncovered specific brewing yeast strains that promote the formation of haze in heavily dry-hopped beer styles. These brewing strains have been termed \"haze-positive\" and furthermore, the timing of dry hop additions has been found to be another key factor in producing this stable haze. Classical genetics have identified YIL169C (herein referred to as HZY1) as both necessary and sufficient for the haze-positive phenotype in the yeast strain most widely used for Hazy IPAs. HZY1 encodes a candidate glycoprotein and our recent findings suggest it is localized to the cell wall through a GPI anchor. Surprisingly, using long-read sequencing data we uncovered extensive genetic variation in HZY1 across brewing strains. The haze-positive phenotype correlates with an expansion in the N-terminal serine-rich region. We propose that the Hzy1 glycoprotein is a critical component to yeast-dependent colloidal haze and the genetic variation in this locus contributes the range of haze phenotypes observed across industrial brewing strains.\nrec-1 loss of function is insufficient to homogenize crossover distribution in Caenorhabditis elegans\nAuthors: Paree, T.; Noble, L. M.; Goncalves, J. F.; Teotonio, H.\nScore: 4.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549456\nMeiotic control of crossover (CO) position is critical for proper homologous chromosome segregation and organismal fertility, recombination of parental genotypes, and the generation of novel genetic combinations. We here characterize the recombination rate landscape of a loss of function genetic modifier of CO position in Caernorhabditis elegans. By averaging CO position across hermaphrodite and male meioses and by genotyping 203 single-nucleotide variants covering about 95% of the genome, we find that the characteristic chromosomal arm-center recombination rate domain structure is lost in a loss of function rec-1 mutant. The rec-1 loss of function mutant smooths the recombination rate landscape but is insufficient to eliminate the non-uniform position of CO. We further find that the rec-1 mutant is of little consequence for organismal fertility and embryo hatchability and thus for rates of autosomal non-disjunction. However, it specifically increases X chromosome non-disjunction rates and males appearance. Our findings question the maintenance of genetic diversity among C. elegans natural populations, and they further suggest that manipulating genetic modifiers of CO position will help map quantitative trait loci in low-recombining genomic regions.\n",
  "wordCount" : "2379",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549478">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549478" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549478">
        <p class="paperTitle">Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549478" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549478" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.</p>
        <p class="info">Score: 397.0, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549478' target='https://doi.org/10.1101/2023.07.18.549478'> 10.1101/2023.07.18.549478</a></p>
        <p class="abstract">Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging. Epigenetic clocks have been shown to be valuable tools for measuring biological age in a variety of tissues and samples. As such, these epigenetic clocks can determine accelerated biological aging and time-to-mortality across various tissues. Previous reports have shown accelerated biological aging and telomere attrition acceleration following SARS-CoV-2 infection. However, the effect of accelerated epigenetic aging on outcome (death/recovery) in COVID-19 patients with Acute Respiratory Distress Syndrome (ARDS) has not been well investigated. In this study, we measured DNA methylation age and telomere attrition in 87 severe COVID-19 cases with ARDS under mechanical ventilation. Furthermore, we compared dynamic changes in epigenetic aging across multiples time-points until recovery or death. Epigenetic age was measured using the Horvath, Hannum, DNAm skin and blood, GrimAge, and PhenoAge clocks, whereas telomere length was calculated using the surrogate marker DNAmTL. Our analysis revealed significant accelerated epigenetic aging but no telomere attrition acceleration in severe COVID-19 cases. In addition, we observed epigenetic age deceleration at inclusion vs end of follow-up in recovered but not in deceased COVID-19 cases using certain clocks. When comparing dynamic changes in epigenetic age acceleration (EAA), we detected higher EAA using both the Horvath and PhenoAge clocks in deceased vs recovered patients. The DNAmTL measurements revealed telomere attrition acceleration in deceased COVID19 patients between inclusion and end of follow-up as well as a significant change in dynamic telomere attrition acceleration when comparing patients who recovered vs those who died. In conclusion, EAA and telomere attrition acceleration was associated with treatment outcome in hospitalized COVID-19 Patients with ARDS. A better understanding of the long-term effects of EAA in COVID19 patients and how they might contribute to Long COVID symptoms in recovered individuals is urgently needed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550074">
        <p class="paperTitle">isGWAS: ultra-high-throughput, scalable and equitable inference of genetic associations with disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Foley, C. N.; Kuncheva, Z.; Marioni, R.; Runz, H.; Sun, B.</p>
        <p class="info">Score: 18.4, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550074' target='https://doi.org/10.1101/2023.07.21.550074'> 10.1101/2023.07.21.550074</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have proven a powerful tool for human geneticists to generate biological insights or hypotheses for drug discovery. Nevertheless, a dependency on sensitive individual-level data together with ever-increasing cohort sample sizes, numbers of variants and phenotypes studied put a strain on existing algorithms, limiting the GWAS approach from maximising potential. Here we present in-silico GWAS (isGWAS), a uniquely scalable algorithm to infer regression parameters in case-control GWAS from cohort-level summary data. For any sample size, isGWAS computes a variant-disease association parameter in ~1 millisecond, or ~11m variants in UK-Biobank within ~4 minutes (~1500-fold faster than state-of-the-art). Extensive simulations and empirical tests demonstrate that isGWAS results are highly comparable to traditional regression-based approaches. We further introduce a heuristic re-sampling algorithm, leapfrog re-sampler (LRS), to extrapolate association results to semi-virtually enlarged cohorts. Owing to significant computational gains we anticipate a broad use of isGWAS and LRS which are customizable on a web interface.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549962" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549962">
        <p class="paperTitle">A small RNA from a natural bacterial pathogen of C. elegans induces transgenerational inheritance of learned avoidance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sengupta, T.; Ange, J. S.; Moore, R. S.; Kaletsky, R. T.; Marogi, J. T.; Myhrvold, C.; Gitai, Z.; Murphy, C. T.</p>
        <p class="info">Score: 12.5, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549962' target='https://doi.org/10.1101/2023.07.20.549962'> 10.1101/2023.07.20.549962</a></p>
        <p class="abstract">A small RNA from a clinical isolate of Pseudomonas aeruginosa, PA14, induces learned avoidance and its transgenerational inheritance in C. elegans. However, it is not known if small RNAs from bacteria found in C. elegans natural habitat can regulate host behavior and produce heritable behavioral effects. Here we found that GRb0427, a pathogenic Pseudomonas vranovensis strain isolated from the C. elegans microbiota, similarly affects worm behavior: worms learn to avoid this pathogenic bacterium following exposure, and this learned avoidance is inherited for four generations. The learned response is entirely mediated by small RNAs, which induce avoidance and transgenerational inheritance in both the laboratory N2 strain and a natural isolate of C. elegans, JU1580, providing further support that such mechanisms of learning and inheritance exist in the wild. A small RNA, Pv1, matches the sequence of C. elegans maco-1, and is both necessary and sufficient to induce learned avoidance in worms. However, Pv1 also results in avoidance of a beneficial microbiome strain, P. mendocina; this maladaptive response may favor reversal of the transgenerational memory after a few generations. Our findings suggest that bacterial small RNA-mediated regulation of host behavior and its transgenerational inheritance are functional in C. elegans natural environment, and that different bacterial small RNA-mediated regulation systems evolved independently but define shared molecular features of bacterial small RNAs that produce transgenerationally-inherited effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549734">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549734" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549734">
        <p class="paperTitle">Genetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549734" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549734" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suhre, K.</p>
        <p class="info">Score: 11.9, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549734' target='https://doi.org/10.1101/2023.07.19.549734'> 10.1101/2023.07.19.549734</a></p>
        <p class="abstract">Protein quantitative trait loci (pQTLs) are an invaluable source of information for drug target development as they provide genetic evidence to support protein function, suggest relationships between cis- and trans-associated proteins, and link proteins to disease where they collocate with genetic risk loci for clinical endpoints. Using the recently released Olink proteomics data for 1,463 proteins measured in over 54,000 samples of the UK Biobank we identified and replicated 4,248 associations with 2,821 ratios between protein levels (rQTLs) where the strengths of association at known pQTL loci increased by up to several hundred orders of magnitude. We attribute this increase in statistical power (p-gain) to accounting for genetic and non-genetic variance shared by the two proteins in the ratio pair. Protein pairs with a significant p-gain were 7.6-fold enriched in known protein-protein interactions, suggesting that their ratios reflect biological links between the implicated proteins. We then conducted a GWAS on the 2,821 ratios and identified 2,527 novel rQTLs, increasing the number of discovered genetic signals compared to the original protein-only GWAS by 24.7%. At examples we demonstrate that this approach can identify novel loci of clinical relevance, support causal gene identification, and reveal complex networks of interacting proteins. Taken together, our study adds significant value to the genetic insights that can be derived from the UKB proteomics data and motivates the wider use of ratios in large scale GWAS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549762">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549762" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549762">
        <p class="paperTitle">Cost-effectiveness of Precision Guided SIT for Control of Anopheles gambiae in the Upper River Region, The Gambia.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549762" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549762" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gendron, W. A. C.; Raban, R.; Mondal, A.; Sanchez C., H. M.; Smidler, A.; Zilberman, D.; Ilboudo, P. G.; D&#39;Alessandro, U.; Marshall, J. M.; Akbari, O. S.</p>
        <p class="info">Score: 11.7, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549762' target='https://doi.org/10.1101/2023.07.20.549762'> 10.1101/2023.07.20.549762</a></p>
        <p class="abstract">Precision-guided sterile insect technique (pgSIT) is an extremely promising vector control intervention that can reduce and potentially eliminate the unacceptable malaria burden, particularly in sub-Saharan Africa. Here we explore the cost effectiveness of using this approach in Africa using mathematical modeling and economical analysis. Overall, we find that pgSIT represents a cost-effective and promising approach to A. gambiae control in The Gambia, with the potential to deliver significant economic and social benefits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549816">
        <p class="paperTitle">GxE PRS: Genotype-environment interaction in polygenic risk score models for quantitative and binary traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jayasinghe, D.; Momin, M.; Beckmann, K.; Hypponen, E.; Benyamin, B.; Lee, S. H.</p>
        <p class="info">Score: 7.6, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549816' target='https://doi.org/10.1101/2023.07.20.549816'> 10.1101/2023.07.20.549816</a></p>
        <p class="abstract">The use of polygenic risk score (PRS) models has transformed the field of genetics by enabling the prediction of complex traits and diseases based on an individuals genetic profile. However, the impact of genotype-environment interaction (GxE) on the performance and applicability of PRS models remains a crucial aspect to be explored. Currently, existing GxE PRS models are often inappropriately used, which can result in inflated type 1 error rates and compromised results. In this study, we propose a novel GxE PRS model that correctly incorporates the GxE component to analyze complex traits and diseases. Through extensive simulations, we demonstrate that our proposed model outperforms existing models in terms of controlling type 1 error rates and enhancing statistical power. Furthermore, we apply the proposed model to real data, and report significant GxE effects. Specifically, we highlight the impact of our model on both quantitative and binary traits. For quantitative traits, we uncover the GxE modulation of genetic effects on body mass index (BMI) by alcohol intake frequency (ALC). In the case of binary traits, we identify the GxE modulation of genetic effects on hypertension (HYP) by waist-to-hip ratio (WHR). These findings underscore the importance of employing a robust model that effectively controls type 1 error rates, thus preventing the occurrence of spurious GxE signals. To facilitate the implementation of our approach, we have developed an innovative R software package called GxE PRS, specifically designed to detect and estimate GxE effects. Overall, our study highlights the importance of accurate GxE modeling and its implications for genetic risk prediction, while providing a practical tool to support further research in this area.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548929">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548929" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548929">
        <p class="paperTitle">CRIMP: A CRISPR/Cas9 Insertional Mutagenesis Protocol and the CRIMP Toolkit</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548929" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548929" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miles, L. B.; Calcinotto, V.; Sonntag, C.; Lee, C.; Bryson-Richardson, R. J.</p>
        <p class="info">Score: 24.7, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548929' target='https://doi.org/10.1101/2023.07.13.548929'> 10.1101/2023.07.13.548929</a></p>
        <p class="abstract">We developed a highly efficient targeted insertional mutagenesis system, CRIMP, and an associated plasmid toolkit, CRIMPkit, that disrupts native gene expression by inducing complete transcriptional termination to produce null mutant alleles without inducing genetic compensation. The CRIMPkit contains over 30 ready-to-use plasmid vectors allowing easy and complete mutagenesis of any gene in any reading frame without requiring custom sequences, modification, or subcloning, and also provides a fluorescent readout of successfully mutagenised fish.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549286">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549286" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549286">
        <p class="paperTitle">T2T-YAO: a Telomere-to-telomere Assembled Diploid Reference Genome for Han Chinese</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549286" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549286" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, Y.; Chu, Y.; Guo, S.; Hu, J.; Li, R.; Zheng, Y.; Ma, X.; Du, Z.; Zhao, L.; Yu, W.; Xue, J.; Bian, W.; Yang, F.; Chen, X.; Zhang, P.; Wu, R.; Ma, Y.; Shao, C.; Chen, J.; Wang, J.; Li, J.; Wu, J.; Hu, X.; Long, Q.; Jiang, M.; Ye, H.; Song, S.; Li, G.; Wei, Y.; Xu, Y.; Ma, Y.; Chen, Y.; Wang, K.; Bao, J.; Xi, W.; Wang, F.; Ni, W.; Zhang, M.; Yu, Y.; Li, S.; Kang, Y.; Gao, Z.</p>
        <p class="info">Score: 4.8, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549286' target='https://doi.org/10.1101/2023.07.17.549286'> 10.1101/2023.07.17.549286</a></p>
        <p class="abstract">Since its initial release in 2001, the human reference genome has been continuously improved in both continuity and accuracy, and the recently-released telomere-to-telomere version--T2T-CHM13--reaches its top quality after 20 years of effort. However, T2T-CHM13 does not represent an authentic diploid human genome, but rather one derived from a simplified, nearly homozygous genome of a hydatidiform mole cell line. To address this limitation and provide an alternative pertinent to the Chinese population, the largest ethnic group in the world, we have assembled a complete diploid human genome of a male Han Chinese, T2T-YAO, which includes telomere-to-telomere assemblies for all the 22&#43;X&#43;M and 22&#43;Y chromosomes in his two haploids inherited separately from his parents. Both haplotypes contain no artificial sequences or model nucleotides and possess a high quality comparable to CHM13, with fewer than one error per [~]14 Mb. Derived from the individual who lives in the aboriginal region of Han Chinese, T2T-YAO shows clear ancestry and potential genetic continuity from the ancient ancestors of the Han population. Each haplotype of T2T-YAO possesses [~]340 Mb exclusive sequences and [~]3100 unique genes as compared to CHM13, and their genome sequences show greater genetic distance to CHM13 than to each other in terms of nucleotide polymorphism and structural variations. The construction of T2T-YAO would serve as a high-quality diploid reference that enables precise delineation of genomic variations in a haplotype-sensitive manner, which could advance our understandings in human evolution, hereditability of diseases and phenotypes, especially within the context of the unique variations of the Chinese population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548400">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548400" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548400">
        <p class="paperTitle">Identification of the yeast mannoprotein gene HZY1	 as a key genetic determinant for yeast-derived haze in beer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548400" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548400" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lacy, K. M.; Mormando, R.; Smith, J. R.; Gibney, P. A.; Shaner, L. M.; Burns, L. T.</p>
        <p class="info">Score: 6.6, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548400' target='https://doi.org/10.1101/2023.07.10.548400'> 10.1101/2023.07.10.548400</a></p>
        <p class="abstract">With the sustained popularity of hazy IPAs, brewers have explored multiple approaches to maximizing stable haze that will remain in suspension throughout the shelf life of the beer. Our recent investigations into yeast-dependent haze have uncovered specific brewing yeast strains that promote the formation of haze in heavily dry-hopped beer styles. These brewing strains have been termed &#34;haze-positive&#34; and furthermore, the timing of dry hop additions has been found to be another key factor in producing this stable haze. Classical genetics have identified YIL169C (herein referred to as HZY1) as both necessary and sufficient for the haze-positive phenotype in the yeast strain most widely used for Hazy IPAs. HZY1 encodes a candidate glycoprotein and our recent findings suggest it is localized to the cell wall through a GPI anchor. Surprisingly, using long-read sequencing data we uncovered extensive genetic variation in HZY1 across brewing strains. The haze-positive phenotype correlates with an expansion in the N-terminal serine-rich region. We propose that the Hzy1 glycoprotein is a critical component to yeast-dependent colloidal haze and the genetic variation in this locus contributes the range of haze phenotypes observed across industrial brewing strains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549456">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549456" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549456">
        <p class="paperTitle">rec-1 loss of function is insufficient to homogenize crossover distribution in Caenorhabditis elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549456" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549456" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paree, T.; Noble, L. M.; Goncalves, J. F.; Teotonio, H.</p>
        <p class="info">Score: 4.0, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549456' target='https://doi.org/10.1101/2023.07.18.549456'> 10.1101/2023.07.18.549456</a></p>
        <p class="abstract">Meiotic control of crossover (CO) position is critical for proper homologous chromosome segregation and organismal fertility, recombination of parental genotypes, and the generation of novel genetic combinations. We here characterize the recombination rate landscape of a loss of function genetic modifier of CO position in Caernorhabditis elegans. By averaging CO position across hermaphrodite and male meioses and by genotyping 203 single-nucleotide variants covering about 95% of the genome, we find that the characteristic chromosomal arm-center recombination rate domain structure is lost in a loss of function rec-1 mutant. The rec-1 loss of function mutant smooths the recombination rate landscape but is insufficient to eliminate the non-uniform position of CO. We further find that the rec-1 mutant is of little consequence for organismal fertility and embryo hatchability and thus for rates of autosomal non-disjunction. However, it specifically increases X chromosome non-disjunction rates and males appearance. Our findings question the maintenance of genetic diversity among C. elegans natural populations, and they further suggest that manipulating genetic modifiers of CO position will help map quantitative trait loci in low-recombining genomic regions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
