<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Synchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition
Authors: Vollger, M. R.; Korlach, J.; Eldred, K. C.; Swanson, E.; Underwood, J. G.; Munson, K. M.; Cheng, Y.-H. H.; Ranchalis, J.; Mao, Y.; Blue, E. E.; Schwarze, U.; Saunders, C. T.; Wenger, A. M.; Allworth, A.; Chanprasert, S.; Duerden, B. L.; Glass, I.; Horike-Pyne, M.; Kim, M.; Leppig, K. A.; McLaughlin, I. J.; Ogawa, J.; Rosenthal, E.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Synchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition
Authors: Vollger, M. R.; Korlach, J.; Eldred, K. C.; Swanson, E.; Underwood, J. G.; Munson, K. M.; Cheng, Y.-H. H.; Ranchalis, J.; Mao, Y.; Blue, E. E.; Schwarze, U.; Saunders, C. T.; Wenger, A. M.; Allworth, A.; Chanprasert, S.; Duerden, B. L.; Glass, I.; Horike-Pyne, M.; Kim, M.; Leppig, K. A.; McLaughlin, I. J.; Ogawa, J.; Rosenthal, E." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-04T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-10-04T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Synchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition
Authors: Vollger, M. R.; Korlach, J.; Eldred, K. C.; Swanson, E.; Underwood, J. G.; Munson, K. M.; Cheng, Y.-H. H.; Ranchalis, J.; Mao, Y.; Blue, E. E.; Schwarze, U.; Saunders, C. T.; Wenger, A. M.; Allworth, A.; Chanprasert, S.; Duerden, B. L.; Glass, I.; Horike-Pyne, M.; Kim, M.; Leppig, K. A.; McLaughlin, I. J.; Ogawa, J.; Rosenthal, E."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Synchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition\nAuthors: Vollger, M. R.; Korlach, J.; Eldred, K. C.; Swanson, E.; Underwood, J. G.; Munson, K. M.; Cheng, Y.-H. H.; Ranchalis, J.; Mao, Y.; Blue, E. E.; Schwarze, U.; Saunders, C. T.; Wenger, A. M.; Allworth, A.; Chanprasert, S.; Duerden, B. L.; Glass, I.; Horike-Pyne, M.; Kim, M.; Leppig, K. A.; McLaughlin, I. J.; Ogawa, J.; Rosenthal, E.",
  "keywords": [
    
  ],
  "articleBody": " Synchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition\nAuthors: Vollger, M. R.; Korlach, J.; Eldred, K. C.; Swanson, E.; Underwood, J. G.; Munson, K. M.; Cheng, Y.-H. H.; Ranchalis, J.; Mao, Y.; Blue, E. E.; Schwarze, U.; Saunders, C. T.; Wenger, A. M.; Allworth, A.; Chanprasert, S.; Duerden, B. L.; Glass, I.; Horike-Pyne, M.; Kim, M.; Leppig, K. A.; McLaughlin, I. J.; Ogawa, J.; Rosenthal, E. A.; Sheppeard, S.; Sherman, S. M.; Strohbehn, S.; Yuen, A. L.; Reh, T. A.; Byers, P. H.; Bamshad, M. J.; Hisama, F. M.; Jarvik, G. P.; Sancak, Y.; Dipple, K. M.; Stergachis, A. B.\nScore: 33.8, Published: 2023-09-27 DOI: 10.1101/2023.09.26.559521\nResolving the molecular basis of a Mendelian condition (MC) remains challenging owing to the diverse mechanisms by which genetic variants cause disease. To address this, we developed a synchronized long-read genome, methylome, epigenome, and transcriptome sequencing approach, which enables accurate single-nucleotide, insertion-deletion, and structural variant calling and diploid de novo genome assembly, and permits the simultaneous elucidation of haplotype-resolved CpG methylation, chromatin accessibility, and full-length transcript information in a single long-read sequencing run. Application of this approach to an Undiagnosed Diseases Network (UDN) participant with a chromosome X;13 balanced translocation of uncertain significance revealed that this translocation disrupted the functioning of four separate genes (NBEA, PDK3, MAB21L1, and RB1) previously associated with single-gene MCs. Notably, the function of each gene was disrupted via a distinct mechanism that required integration of the four omes to resolve. These included nonsense-mediated decay, fusion transcript formation, enhancer adoption, transcriptional readthrough silencing, and inappropriate X chromosome inactivation of autosomal genes. Overall, this highlights the utility of synchronized long-read multi-omic profiling for mechanistically resolving complex phenotypes.\nExtensive co-regulation of neighbouring genes complicates the use of eQTLs in target gene prioritisation\nAuthors: Tambets, R.; Kolde, A.; Kolberg, P.; Love, M. I.; Alasoo, K.\nScore: 19.7, Published: 2023-09-30 DOI: 10.1101/2023.09.29.560109\nIdentifying causal genes underlying genome-wide association studies (GWAS) is a fundamental problem in human genetics. Although colocalisation with gene expression quantitative trait loci (eQTLs) is often used to prioritise GWAS target genes, systematic benchmarking has been limited due to unavailability of large ground truth datasets. Here, we re-analysed plasma protein QTL data from 3,301 individuals of the INTERVAL cohort together with 131 eQTL Catalogue datasets. Focusing on variants located within or close to the affected protein identified 793 proteins with at least one cis-pQTL where we could assume that the most likely causal gene was the gene coding for the protein. We then benchmarked the ability of cis-eQTLs to recover these causal genes by comparing three Bayesian colocalisation methods (coloc.susie, coloc.abf and CLPP) and five Mendelian randomisation (MR) approaches (three varieties of inverse-variance weighted MR, MR-RAPS, and MRLocus). We found that assigning fine-mapped pQTLs to their closest protein coding genes outperformed all colocalisation methods regarding both precision (71.9%) and recall (76.9%). Furthermore, the colocalisation method with the highest recall (coloc.susie - 46.3%) also had the lowest precision (45.1%). Combining evidence from multiple conditionally distinct colocalising QTLs with MR increased precision to 81%, but this was accompanied by a large reduction in recall to 7.1%. Furthermore, the choice of the MR method greatly affected performance, with the standard inverse-variance weighted MR often producing many false positives. Our results highlight that linking GWAS variants to target genes remains challenging with eQTL evidence alone, and prioritising novel targets requires triangulation of evidence from multiple sources.\nEvolutionary adaptation of the chromodomain of the HP1-protein Rhino allows the integration of chromatin and DNA sequence signals\nAuthors: Baumgartner, L.; Ipsaro, J. J.; Hohmann, U.; Handler, D.; Schleiffer, A.; Duchek, P.; Brennecke, J.\nScore: 18.5, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560096\nMembers of the diverse heterochromatin protein 1 (HP1) family of proteins play crucial roles in heterochromatin formation and maintenance. Despite the similar affinities of their chromodomains for di- and tri-methylated histone H3 lysine 9 (H3K9me2/3), different HP1 proteins exhibit distinct chromatin binding patterns, presumably due to their interactions with various specificity factors. Here, we elucidate the molecular basis of the protein-protein interaction between the HP1 protein Rhino, a critical factor of the Drosophila piRNA pathway, and Kipferl, a DNA sequence-specific C2H2 zinc finger protein and Rhino guidance factor. Through phylogenetic analyses, structure prediction, and in vivo genetics, we identify a single amino acid change within Rhino's chromodomain, G31D, that does not affect H3K9me2/3 binding but abolishes the specific interaction between Rhino and Kipferl. Flies carrying the rhinoG31D mutation phenocopy kipferl mutant flies, with Rhino redistributing from piRNA clusters to satellite repeats, causing pronounced changes in the ovarian piRNA profile of rhinoG31D flies. Thus, Rhino's chromodomain serves as a dual-specificity module, facilitating interactions with both a histone mark and a DNA-binding protein.\nBronze Age Northern Eurasian Genetics in the Context of Development of Metallurgy and Siberian Ancestry\nAuthors: Childebayeva, A.; Fricke, F.; Rohrlach, A. B.; Huang, L.; Schiffels, S.; Vesakoski, O.; Semerau, L.; Aron, F.; Moiseyev, V.; Khartanovich, V.; Kovtun, I.; Krause, J.; Kuzminykh, S.; Haak, W.\nScore: 13.0, Published: 2023-10-01 DOI: 10.1101/2023.10.01.560195\nThe Eurasian Bronze Age (BA) has been described as a period of substantial human migrations, the emergence of pastoralism, horse domestication, and development of metallurgy. This study focuses on individuals associated with BA metallurgical production, specifically the Seima-Turbino (ST) phenomenon (~2,200-1,900 BCE) associated with elaborate metal objects found across Northern Eurasia. The genetic profiles of nine ST-associated individuals vary widely ranging between ancestries maximized in individuals from the Eastern Siberian Late Neolithic/BA, and those of the Western Steppe Middle Late BA. The genetic heterogeneity observed is consistent with the current understanding of the ST metallurgical network as a transcultural phenomenon. The new data also shed light on the temporal and spatial range of an ancient Siberian genetic ancestry component, which is shared across many Uralic-speaking populations, and which we explore further via demographic modeling using additional genome-wide (2 individuals) and whole genome data (5 individuals, including a ~30x genome) from northwestern Russia.\nEnsembled best subset selection using summary statistics for polygenic risk prediction\nAuthors: Chen, T.; Zhang, H.; Mazumder, R.; Lin, X.\nScore: 9.9, Published: 2023-09-27 DOI: 10.1101/2023.09.25.559307\nPolygenic risk scores (PRS) enhance population risk stratification and advance personalized medicine, yet existing methods face a tradeoff between predictive power and computational efficiency. We introduce ALL-Sum, an easy-to-implement PRS method that combines an efficient summary statistic-based L0L2 penalized regression algorithm with an ensembling step that aggregates estimates from different tuning parameters for improved prediction performance. In extensive large-scale simulations across a wide range of polygenicity and genome-wide association studies (GWAS) sample sizes, ALL-Sum consistently outperforms popular alternative methods in terms of prediction accuracy, runtime, and memory usage. We analyze 27 published GWAS summary statistics for 11 complex traits from 9 reputable data sources, including the Global Lipids Genetics Consortium, Breast Cancer Association Consortium, and FinnGen, evaluated using individual-level UKBB data. ALL-Sum achieves the highest accuracy for most traits, particularly for GWAS with large sample sizes. We provide ALL-Sum as a user-friendly command-line software with pre-computed reference data for streamlined user-end analysis.\nAdmix-kit: An Integrated Toolkit and Pipeline for Genetic Analyses of Admixed Populations\nAuthors: Hou, K.; Gogarten, S.; Kim, J.; Hua, X.; Dias, J.-A.; Sun, Q.; Wang, Y.; Tan, T.; Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium Methods Working Group, ; Atkinson, E. G.; Martin, A.; Shortt, J.; Hirbo, J.; Li, Y.; Pasaniuc, B.; Zhang, H.\nScore: 8.2, Published: 2023-10-02 DOI: 10.1101/2023.09.30.560263\nAdmixed populations, with their unique and diverse genetic backgrounds, are often underrepresented in genetic studies. This oversight not only limits our understanding but also exacerbates existing health disparities. One major barrier has been the lack of efficient tools tailored for the special challenges of genetic study of admixed populations. Here, we present admix-kit, an integrated toolkit and pipeline for genetic analyses of admixed populations. Admix-kit implements a suite of methods to facilitate genotype and phenotype simulation, association testing, genetic architecture inference, and polygenic scoring in admixed populations.\nNatural selection drives emergent genetic homogeneity in a century-scale experiment with barley\nAuthors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.\nScore: 29.1, Published: 2023-09-22 DOI: 10.1101/2023.09.22.557807\nDirect observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII). CCII was founded in 1929 with tens of thousands of unique genotypes and has been adapted to local conditions in Davis, CA, USA for 58 generations. We find that natural selection has massively reduced genetic diversity leading to a single clonal lineage constituting most of the population by generation F50. Selection favored alleles originating from similar climates to that of Davis, and targeted genes regulating reproductive development, including some of the most well-characterized barley diversification loci, Vrs1, HvCEN, and Ppd-H1. We chronicle the dynamic evolution of reproductive timing in the population and uncover how parallel molecular pathways are targeted by stabilizing selection to optimize this trait. Our findings point to selection as the predominant force shaping genomic variation in one of the worlds oldest ongoing biological experiments. One-Sentence SummaryWholesale genetic restructuring of an experimental population is a consequence of rapid environmental adaptation.\nAutosomal suppression of meiotic drive can prevent sex chromosome cycling\nAuthors: Gupta, A.; Unckless, R.\nScore: 6.3, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559847\nSex-ratio meiotic drivers are selfish genes or gene complexes that bias the transmission of chromosomes resulting in skewed sex ratios. While existing theoretical models have suggested the maintenance of a four-chromosome equilibrium (X, XSR, Y, YSUP; SR: sex-ratio driver, SUP: suppressor) in a cyclic dynamic, studies of natural populations have failed to capture this pattern. Although there are several plausible explanations for this lack of cycling, interference from autosomal suppressors has not been studied using a population genetic framework even though autosomal suppressors and Y-linked suppressors both exist in populations of some species. In this study, we use a simulations-based approach to investigate the influence of autosomal suppressors on the cycling of sex chromosomes. Our findings demonstrate that the presence of an autosomal suppressor can hinder the invasion of a Y-linked suppressor under some parameter space, thereby impeding the cyclic dynamics. Even when a Y-linked suppressor invades, the presence of an autosomal suppressor can prevent cycling. Our study demonstrates the potential role of autosomal suppressors in preventing sex chromosome cycling and provides insights into the conditions and consequences of the maintenance of both Y-linked and autosomal suppressors.\nIn vivo dissection of the mouse tyrosine catabolic pathway with CRISPR-Cas9 identifies modifier genes affecting hereditary tyrosinemia type 1\nAuthors: Rivest, J.-F.; Carter, S.; Goupil, C.; Cyr, D.; Ung, R.-V.; Dal Soglio, D.; Mac-Way, F.; Waters, P. J.; Paganelli, M.; Doyon, Y.\nScore: 4.1, Published: 2023-09-29 DOI: 10.1101/2023.09.29.559947\nHereditary tyrosinemia type 1 is an autosomal recessive disorder caused by mutations (pathogenic variants) in fumarylacetoacetate hydrolase, an enzyme involved in tyrosine degradation. Its loss results in the accumulation of toxic metabolites that mainly affect the liver and kidneys and can lead to severe liver disease and liver cancer. Tyrosinemia type 1 has a global prevalence of approximately 1 in 100,000 births but can reach up to 1 in 1,500 births in some regions of Quebec, Canada. Mutating functionally related modifier genes (i.e., genes that, when mutated, affect the phenotypic impacts of mutations in other genes) is an emerging strategy for treating human genetic diseases. In vivo somatic genome editing in animal models of these diseases is a powerful means to identify modifier genes and fuel treatment development. In this study, we demonstrate that mutating additional enzymes in the tyrosine catabolic pathway through liver-specific genome editing can relieve or worsen the phenotypic severity of a murine model of tyrosinemia type 1. Neonatal gene delivery using recombinant adeno-associated viral vectors expressing Staphylococcus aureus Cas9 under the control of a liver-specific promoter led to efficient gene disruption and metabolic rewiring of the pathway, with systemic effects that were distinct from the phenotypes observed in whole-body knockout models. Our work illustrates the value of using in vivo genome editing in model organisms to study the direct effects of combining pathological mutations with modifier gene mutations in isogenic settings.\nGenetic diversity of domestic cat hepadnavirus in Taiwan\nAuthors: Silva, B. B. I.; Chen, J.-Y.; Villanueva, B. H. A.; Lu, Z.-Y.; Zhen, H.-H.; Montecillo, A. D.; Shofa, M.; Minh, H.; Chuang, J.-P.; Huang, H.-Y.; Saito, A.; Chuang, K.-P.\nScore: 3.7, Published: 2023-09-29 DOI: 10.1101/2023.09.29.560097\nDomestic cat hepadnavirus (DCH) is an infectious disease associated with chronic hepatitis in cats, suggesting a similarity with hepatitis B virus infection in humans. Since its first identification in Australia in 2018, DCH has been reported in several countries with varying prevalence rates, but its prevalence in Taiwan has not yet been investigated. Here, we aimed to identify the presence and prevalence of DCH infections in Taiwan. Among 71 samples tested, eight (11.27%) were positive for DCH. Of these positive cases, three cats had elevated levels of alanine transaminase (ALT) and aspartate transaminase (AST), suggesting an association between DCH infection and chronic hepatitis. Four DCH-positive samples were also tested for feline immunodeficiency virus (FIV) and feline leukemia virus (FeLV) co-infection, one (25%) was positive for FIV while none for FeLV (0%). In addition, we performed whole genome sequencing of six samples to determine the viral genome sequences. Phylogenetic analyses identified a distinct lineage compared with previously reported sequences. Considering the recent findings suggesting the potential risk of DCH for interspecies or zoonotic transmission, this study suggests the importance of continuous surveillance of DCH and further research to elucidate the pathophysiology and transmission route of DCH.\n",
  "wordCount" : "2258",
  "inLanguage": "en",
  "datePublished": "2023-10-04T10:39:33Z",
  "dateModified": "2023-10-04T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on October 4, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559521">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559521" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559521">
        <p class="paperTitle">Synchronized long-read genome, methylome, epigenome, and transcriptome for resolving a Mendelian condition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559521" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559521" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vollger, M. R.; Korlach, J.; Eldred, K. C.; Swanson, E.; Underwood, J. G.; Munson, K. M.; Cheng, Y.-H. H.; Ranchalis, J.; Mao, Y.; Blue, E. E.; Schwarze, U.; Saunders, C. T.; Wenger, A. M.; Allworth, A.; Chanprasert, S.; Duerden, B. L.; Glass, I.; Horike-Pyne, M.; Kim, M.; Leppig, K. A.; McLaughlin, I. J.; Ogawa, J.; Rosenthal, E. A.; Sheppeard, S.; Sherman, S. M.; Strohbehn, S.; Yuen, A. L.; Reh, T. A.; Byers, P. H.; Bamshad, M. J.; Hisama, F. M.; Jarvik, G. P.; Sancak, Y.; Dipple, K. M.; Stergachis, A. B.</p>
        <p class="info">Score: 33.8, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559521' target='https://doi.org/10.1101/2023.09.26.559521'> 10.1101/2023.09.26.559521</a></p>
        <p class="abstract">Resolving the molecular basis of a Mendelian condition (MC) remains challenging owing to the diverse mechanisms by which genetic variants cause disease. To address this, we developed a synchronized long-read genome, methylome, epigenome, and transcriptome sequencing approach, which enables accurate single-nucleotide, insertion-deletion, and structural variant calling and diploid de novo genome assembly, and permits the simultaneous elucidation of haplotype-resolved CpG methylation, chromatin accessibility, and full-length transcript information in a single long-read sequencing run. Application of this approach to an Undiagnosed Diseases Network (UDN) participant with a chromosome X;13 balanced translocation of uncertain significance revealed that this translocation disrupted the functioning of four separate genes (NBEA, PDK3, MAB21L1, and RB1) previously associated with single-gene MCs. Notably, the function of each gene was disrupted via a distinct mechanism that required integration of the four  omes to resolve. These included nonsense-mediated decay, fusion transcript formation, enhancer adoption, transcriptional readthrough silencing, and inappropriate X chromosome inactivation of autosomal genes. Overall, this highlights the utility of synchronized long-read multi-omic profiling for mechanistically resolving complex phenotypes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560109">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560109" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560109">
        <p class="paperTitle">Extensive co-regulation of neighbouring genes complicates the use of eQTLs in target gene prioritisation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560109" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560109" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tambets, R.; Kolde, A.; Kolberg, P.; Love, M. I.; Alasoo, K.</p>
        <p class="info">Score: 19.7, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560109' target='https://doi.org/10.1101/2023.09.29.560109'> 10.1101/2023.09.29.560109</a></p>
        <p class="abstract">Identifying causal genes underlying genome-wide association studies (GWAS) is a fundamental problem in human genetics. Although colocalisation with gene expression quantitative trait loci (eQTLs) is often used to prioritise GWAS target genes, systematic benchmarking has been limited due to unavailability of large ground truth datasets. Here, we re-analysed plasma protein QTL data from 3,301 individuals of the INTERVAL cohort together with 131 eQTL Catalogue datasets. Focusing on variants located within or close to the affected protein identified 793 proteins with at least one cis-pQTL where we could assume that the most likely causal gene was the gene coding for the protein. We then benchmarked the ability of cis-eQTLs to recover these causal genes by comparing three Bayesian colocalisation methods (coloc.susie, coloc.abf and CLPP) and five Mendelian randomisation (MR) approaches (three varieties of inverse-variance weighted MR, MR-RAPS, and MRLocus). We found that assigning fine-mapped pQTLs to their closest protein coding genes outperformed all colocalisation methods regarding both precision (71.9%) and recall (76.9%). Furthermore, the colocalisation method with the highest recall (coloc.susie - 46.3%) also had the lowest precision (45.1%). Combining evidence from multiple conditionally distinct colocalising QTLs with MR increased precision to 81%, but this was accompanied by a large reduction in recall to 7.1%. Furthermore, the choice of the MR method greatly affected performance, with the standard inverse-variance weighted MR often producing many false positives. Our results highlight that linking GWAS variants to target genes remains challenging with eQTL evidence alone, and prioritising novel targets requires triangulation of evidence from multiple sources.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560096">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560096" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560096">
        <p class="paperTitle">Evolutionary adaptation of the chromodomain of the HP1-protein Rhino allows the integration of chromatin and DNA sequence signals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560096" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560096" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baumgartner, L.; Ipsaro, J. J.; Hohmann, U.; Handler, D.; Schleiffer, A.; Duchek, P.; Brennecke, J.</p>
        <p class="info">Score: 18.5, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560096' target='https://doi.org/10.1101/2023.09.29.560096'> 10.1101/2023.09.29.560096</a></p>
        <p class="abstract">Members of the diverse heterochromatin protein 1 (HP1) family of proteins play crucial roles in heterochromatin formation and maintenance. Despite the similar affinities of their chromodomains for di- and tri-methylated histone H3 lysine 9 (H3K9me2/3), different HP1 proteins exhibit distinct chromatin binding patterns, presumably due to their interactions with various specificity factors. Here, we elucidate the molecular basis of the protein-protein interaction between the HP1 protein Rhino, a critical factor of the Drosophila piRNA pathway, and Kipferl, a DNA sequence-specific C2H2 zinc finger protein and Rhino guidance factor. Through phylogenetic analyses, structure prediction, and in vivo genetics, we identify a single amino acid change within Rhino&#39;s chromodomain, G31D, that does not affect H3K9me2/3 binding but abolishes the specific interaction between Rhino and Kipferl. Flies carrying the rhinoG31D mutation phenocopy kipferl mutant flies, with Rhino redistributing from piRNA clusters to satellite repeats, causing pronounced changes in the ovarian piRNA profile of rhinoG31D flies. Thus, Rhino&#39;s chromodomain serves as a dual-specificity module, facilitating interactions with both a histone mark and a DNA-binding protein.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560195">
        <p class="paperTitle">Bronze Age Northern Eurasian Genetics in the Context of Development of Metallurgy and Siberian Ancestry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Childebayeva, A.; Fricke, F.; Rohrlach, A. B.; Huang, L.; Schiffels, S.; Vesakoski, O.; Semerau, L.; Aron, F.; Moiseyev, V.; Khartanovich, V.; Kovtun, I.; Krause, J.; Kuzminykh, S.; Haak, W.</p>
        <p class="info">Score: 13.0, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560195' target='https://doi.org/10.1101/2023.10.01.560195'> 10.1101/2023.10.01.560195</a></p>
        <p class="abstract">The Eurasian Bronze Age (BA) has been described as a period of substantial human migrations, the emergence of pastoralism, horse domestication, and development of metallurgy. This study focuses on individuals associated with BA metallurgical production, specifically the Seima-Turbino (ST) phenomenon (~2,200-1,900 BCE) associated with elaborate metal objects found across Northern Eurasia. The genetic profiles of nine ST-associated individuals vary widely ranging between ancestries maximized in individuals from the Eastern Siberian Late Neolithic/BA, and those of the Western Steppe Middle Late BA. The genetic heterogeneity observed is consistent with the current understanding of the ST metallurgical network as a transcultural phenomenon. The new data also shed light on the temporal and spatial range of an ancient Siberian genetic ancestry component, which is shared across many Uralic-speaking populations, and which we explore further via demographic modeling using additional genome-wide (2 individuals) and whole genome data (5 individuals, including a ~30x genome) from northwestern Russia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559307">
        <p class="paperTitle">Ensembled best subset selection using summary statistics for polygenic risk prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, T.; Zhang, H.; Mazumder, R.; Lin, X.</p>
        <p class="info">Score: 9.9, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559307' target='https://doi.org/10.1101/2023.09.25.559307'> 10.1101/2023.09.25.559307</a></p>
        <p class="abstract">Polygenic risk scores (PRS) enhance population risk stratification and advance personalized medicine, yet existing methods face a tradeoff between predictive power and computational efficiency. We introduce ALL-Sum, an easy-to-implement PRS method that combines an efficient summary statistic-based L0L2 penalized regression algorithm with an ensembling step that aggregates estimates from different tuning parameters for improved prediction performance. In extensive large-scale simulations across a wide range of polygenicity and genome-wide association studies (GWAS) sample sizes, ALL-Sum consistently outperforms popular alternative methods in terms of prediction accuracy, runtime, and memory usage. We analyze 27 published GWAS summary statistics for 11 complex traits from 9 reputable data sources, including the Global Lipids Genetics Consortium, Breast Cancer Association Consortium, and FinnGen, evaluated using individual-level UKBB data. ALL-Sum achieves the highest accuracy for most traits, particularly for GWAS with large sample sizes. We provide ALL-Sum as a user-friendly command-line software with pre-computed reference data for streamlined user-end analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560263">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560263" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560263">
        <p class="paperTitle">Admix-kit: An Integrated Toolkit and Pipeline for Genetic Analyses of Admixed Populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560263" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560263" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hou, K.; Gogarten, S.; Kim, J.; Hua, X.; Dias, J.-A.; Sun, Q.; Wang, Y.; Tan, T.; Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium Methods Working Group,  ; Atkinson, E. G.; Martin, A.; Shortt, J.; Hirbo, J.; Li, Y.; Pasaniuc, B.; Zhang, H.</p>
        <p class="info">Score: 8.2, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560263' target='https://doi.org/10.1101/2023.09.30.560263'> 10.1101/2023.09.30.560263</a></p>
        <p class="abstract">Admixed populations, with their unique and diverse genetic backgrounds, are often underrepresented in genetic studies. This oversight not only limits our understanding but also exacerbates existing health disparities. One major barrier has been the lack of efficient tools tailored for the special challenges of genetic study of admixed populations. Here, we present admix-kit, an integrated toolkit and pipeline for genetic analyses of admixed populations. Admix-kit implements a suite of methods to facilitate genotype and phenotype simulation, association testing, genetic architecture inference, and polygenic scoring in admixed populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.557807">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.557807" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.557807">
        <p class="paperTitle">Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.557807" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.557807" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.</p>
        <p class="info">Score: 29.1, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.557807' target='https://doi.org/10.1101/2023.09.22.557807'> 10.1101/2023.09.22.557807</a></p>
        <p class="abstract">Direct observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII). CCII was founded in 1929 with tens of thousands of unique genotypes and has been adapted to local conditions in Davis, CA, USA for 58 generations. We find that natural selection has massively reduced genetic diversity leading to a single clonal lineage constituting most of the population by generation F50. Selection favored alleles originating from similar climates to that of Davis, and targeted genes regulating reproductive development, including some of the most well-characterized barley diversification loci, Vrs1, HvCEN, and Ppd-H1. We chronicle the dynamic evolution of reproductive timing in the population and uncover how parallel molecular pathways are targeted by stabilizing selection to optimize this trait. Our findings point to selection as the predominant force shaping genomic variation in one of the worlds oldest ongoing biological experiments.

One-Sentence SummaryWholesale genetic restructuring of an experimental population is a consequence of rapid environmental adaptation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559847">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559847" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559847">
        <p class="paperTitle">Autosomal suppression of meiotic drive can prevent sex chromosome cycling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559847" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559847" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, A.; Unckless, R.</p>
        <p class="info">Score: 6.3, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559847' target='https://doi.org/10.1101/2023.09.28.559847'> 10.1101/2023.09.28.559847</a></p>
        <p class="abstract">Sex-ratio meiotic drivers are selfish genes or gene complexes that bias the transmission of chromosomes resulting in skewed sex ratios. While existing theoretical models have suggested the maintenance of a four-chromosome equilibrium (X, XSR, Y, YSUP; SR: sex-ratio driver, SUP: suppressor) in a cyclic dynamic, studies of natural populations have failed to capture this pattern. Although there are several plausible explanations for this lack of cycling, interference from autosomal suppressors has not been studied using a population genetic framework even though autosomal suppressors and Y-linked suppressors both exist in populations of some species. In this study, we use a simulations-based approach to investigate the influence of autosomal suppressors on the cycling of sex chromosomes. Our findings demonstrate that the presence of an autosomal suppressor can hinder the invasion of a Y-linked suppressor under some parameter space, thereby impeding the cyclic dynamics. Even when a Y-linked suppressor invades, the presence of an autosomal suppressor can prevent cycling. Our study demonstrates the potential role of autosomal suppressors in preventing sex chromosome cycling and provides insights into the conditions and consequences of the maintenance of both Y-linked and autosomal suppressors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.559947">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.559947" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.559947">
        <p class="paperTitle">In vivo dissection of the mouse tyrosine catabolic pathway with CRISPR-Cas9 identifies modifier genes affecting hereditary tyrosinemia type 1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.559947" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.559947" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rivest, J.-F.; Carter, S.; Goupil, C.; Cyr, D.; Ung, R.-V.; Dal Soglio, D.; Mac-Way, F.; Waters, P. J.; Paganelli, M.; Doyon, Y.</p>
        <p class="info">Score: 4.1, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.559947' target='https://doi.org/10.1101/2023.09.29.559947'> 10.1101/2023.09.29.559947</a></p>
        <p class="abstract">Hereditary tyrosinemia type 1 is an autosomal recessive disorder caused by mutations (pathogenic variants) in fumarylacetoacetate hydrolase, an enzyme involved in tyrosine degradation. Its loss results in the accumulation of toxic metabolites that mainly affect the liver and kidneys and can lead to severe liver disease and liver cancer. Tyrosinemia type 1 has a global prevalence of approximately 1 in 100,000 births but can reach up to 1 in 1,500 births in some regions of Quebec, Canada. Mutating functionally related modifier genes (i.e., genes that, when mutated, affect the phenotypic impacts of mutations in other genes) is an emerging strategy for treating human genetic diseases. In vivo somatic genome editing in animal models of these diseases is a powerful means to identify modifier genes and fuel treatment development. In this study, we demonstrate that mutating additional enzymes in the tyrosine catabolic pathway through liver-specific genome editing can relieve or worsen the phenotypic severity of a murine model of tyrosinemia type 1. Neonatal gene delivery using recombinant adeno-associated viral vectors expressing Staphylococcus aureus Cas9 under the control of a liver-specific promoter led to efficient gene disruption and metabolic rewiring of the pathway, with systemic effects that were distinct from the phenotypes observed in whole-body knockout models. Our work illustrates the value of using in vivo genome editing in model organisms to study the direct effects of combining pathological mutations with modifier gene mutations in isogenic settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560097" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560097">
        <p class="paperTitle">Genetic diversity of domestic cat hepadnavirus in Taiwan</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Silva, B. B. I.; Chen, J.-Y.; Villanueva, B. H. A.; Lu, Z.-Y.; Zhen, H.-H.; Montecillo, A. D.; Shofa, M.; Minh, H.; Chuang, J.-P.; Huang, H.-Y.; Saito, A.; Chuang, K.-P.</p>
        <p class="info">Score: 3.7, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560097' target='https://doi.org/10.1101/2023.09.29.560097'> 10.1101/2023.09.29.560097</a></p>
        <p class="abstract">Domestic cat hepadnavirus (DCH) is an infectious disease associated with chronic hepatitis in cats, suggesting a similarity with hepatitis B virus infection in humans. Since its first identification in Australia in 2018, DCH has been reported in several countries with varying prevalence rates, but its prevalence in Taiwan has not yet been investigated. Here, we aimed to identify the presence and prevalence of DCH infections in Taiwan. Among 71 samples tested, eight (11.27%) were positive for DCH. Of these positive cases, three cats had elevated levels of alanine transaminase (ALT) and aspartate transaminase (AST), suggesting an association between DCH infection and chronic hepatitis. Four DCH-positive samples were also tested for feline immunodeficiency virus (FIV) and feline leukemia virus (FeLV) co-infection, one (25%) was positive for FIV while none for FeLV (0%). In addition, we performed whole genome sequencing of six samples to determine the viral genome sequences. Phylogenetic analyses identified a distinct lineage compared with previously reported sequences. Considering the recent findings suggesting the potential risk of DCH for interspecies or zoonotic transmission, this study suggests the importance of continuous surveillance of DCH and further research to elucidate the pathophysiology and transmission route of DCH.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
