<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization
Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.
Score: 400.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478
Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization
Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.
Score: 400.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478
Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-30T10:36:57+00:00" />
<meta property="article:modified_time" content="2023-07-30T10:36:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization
Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.
Score: 400.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478
Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization\nAuthors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.\nScore: 400.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478\nAging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging.",
  "keywords": [
    
  ],
  "articleBody": " Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization\nAuthors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.\nScore: 400.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478\nAging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging. Epigenetic clocks have been shown to be valuable tools for measuring biological age in a variety of tissues and samples. As such, these epigenetic clocks can determine accelerated biological aging and time-to-mortality across various tissues. Previous reports have shown accelerated biological aging and telomere attrition acceleration following SARS-CoV-2 infection. However, the effect of accelerated epigenetic aging on outcome (death/recovery) in COVID-19 patients with Acute Respiratory Distress Syndrome (ARDS) has not been well investigated. In this study, we measured DNA methylation age and telomere attrition in 87 severe COVID-19 cases with ARDS under mechanical ventilation. Furthermore, we compared dynamic changes in epigenetic aging across multiples time-points until recovery or death. Epigenetic age was measured using the Horvath, Hannum, DNAm skin and blood, GrimAge, and PhenoAge clocks, whereas telomere length was calculated using the surrogate marker DNAmTL. Our analysis revealed significant accelerated epigenetic aging but no telomere attrition acceleration in severe COVID-19 cases. In addition, we observed epigenetic age deceleration at inclusion vs end of follow-up in recovered but not in deceased COVID-19 cases using certain clocks. When comparing dynamic changes in epigenetic age acceleration (EAA), we detected higher EAA using both the Horvath and PhenoAge clocks in deceased vs recovered patients. The DNAmTL measurements revealed telomere attrition acceleration in deceased COVID19 patients between inclusion and end of follow-up as well as a significant change in dynamic telomere attrition acceleration when comparing patients who recovered vs those who died. In conclusion, EAA and telomere attrition acceleration was associated with treatment outcome in hospitalized COVID-19 Patients with ARDS. A better understanding of the long-term effects of EAA in COVID19 patients and how they might contribute to Long COVID symptoms in recovered individuals is urgently needed.\nisGWAS: ultra-high-throughput, scalable and equitable inference of genetic associations with disease\nAuthors: Foley, C. N.; Kuncheva, Z.; Marioni, R.; Runz, H.; Sun, B.\nScore: 26.2, Published: 2023-07-27 DOI: 10.1101/2023.07.21.550074\nGenome-wide association studies (GWAS) have proven a powerful tool for human geneticists to generate biological insights or hypotheses for drug discovery. Nevertheless, a dependency on sensitive individual-level data together with ever-increasing cohort sample sizes, numbers of variants and phenotypes studied put a strain on existing algorithms, limiting the GWAS approach from maximising potential. Here we present in-silico GWAS (isGWAS), a uniquely scalable algorithm to infer regression parameters in case-control GWAS from cohort-level summary data. For any sample size, isGWAS computes a variant-disease association parameter in [~]1 millisecond, or [~]11m variants in UK-Biobank within [~]4 minutes ([~]1500-fold faster than state-of-the-art). Extensive simulations and empirical tests demonstrate that isGWAS results are highly comparable to traditional regression-based approaches. We further introduce a heuristic re-sampling algorithm, leapfrog re-sampler (LRS), to extrapolate association results to semi-virtually enlarged cohorts. Owing to significant computational gains we anticipate a broad use of isGWAS and LRS which are customizable on a web interface.\nA small RNA from a natural bacterial pathogen of C. elegans induces transgenerational inheritance of learned avoidance\nAuthors: Sengupta, T.; Ange, J. S.; Moore, R. S.; Kaletsky, R. T.; Marogi, J. T.; Myhrvold, C.; Gitai, Z.; Murphy, C. T.\nScore: 15.7, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549962\nA small RNA from a clinical isolate of Pseudomonas aeruginosa, PA14, induces learned avoidance and its transgenerational inheritance in C. elegans. However, it is not known if small RNAs from bacteria found in C. elegans natural habitat can regulate host behavior and produce heritable behavioral effects. Here we found that GRb0427, a pathogenic Pseudomonas vranovensis strain isolated from the C. elegans microbiota, similarly affects worm behavior: worms learn to avoid this pathogenic bacterium following exposure, and this learned avoidance is inherited for four generations. The learned response is entirely mediated by small RNAs, which induce avoidance and transgenerational inheritance in both the laboratory N2 strain and a natural isolate of C. elegans, JU1580, providing further support that such mechanisms of learning and inheritance exist in the wild. A small RNA, Pv1, matches the sequence of C. elegans maco-1, and is both necessary and sufficient to induce learned avoidance in worms. However, Pv1 also results in avoidance of a beneficial microbiome strain, P. mendocina; this maladaptive response may favor reversal of the transgenerational memory after a few generations. Our findings suggest that bacterial small RNA-mediated regulation of host behavior and its transgenerational inheritance are functional in C. elegans natural environment, and that different bacterial small RNA-mediated regulation systems evolved independently but define shared molecular features of bacterial small RNAs that produce transgenerationally-inherited effects.\nGenetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions\nAuthors: Suhre, K.\nScore: 15.5, Published: 2023-07-20 DOI: 10.1101/2023.07.19.549734\nProtein quantitative trait loci (pQTLs) are an invaluable source of information for drug target development as they provide genetic evidence to support protein function, suggest relationships between cis- and trans-associated proteins, and link proteins to disease where they collocate with genetic risk loci for clinical endpoints. Using the recently released Olink proteomics data for 1,463 proteins measured in over 54,000 samples of the UK Biobank we identified and replicated 4,248 associations with 2,821 ratios between protein levels (rQTLs) where the strengths of association at known pQTL loci increased by up to several hundred orders of magnitude. We attribute this increase in statistical power (p-gain) to accounting for genetic and non-genetic variance shared by the two proteins in the ratio pair. Protein pairs with a significant p-gain were 7.6-fold enriched in known protein-protein interactions, suggesting that their ratios reflect biological links between the implicated proteins. We then conducted a GWAS on the 2,821 ratios and identified 2,527 novel rQTLs, increasing the number of discovered genetic signals compared to the original protein-only GWAS by 24.7%. At examples we demonstrate that this approach can identify novel loci of clinical relevance, support causal gene identification, and reveal complex networks of interacting proteins. Taken together, our study adds significant value to the genetic insights that can be derived from the UKB proteomics data and motivates the wider use of ratios in large scale GWAS.\nCost-effectiveness of Precision Guided SIT for Control of Anopheles gambiae in the Upper River Region, The Gambia.\nAuthors: Gendron, W. A. C.; Raban, R.; Mondal, A.; Sanchez C., H. M.; Smidler, A.; Zilberman, D.; Ilboudo, P. G.; D'Alessandro, U.; Marshall, J. M.; Akbari, O. S.\nScore: 11.7, Published: 2023-07-24 DOI: 10.1101/2023.07.20.549762\nPrecision-guided sterile insect technique (pgSIT) is an extremely promising vector control intervention that can reduce and potentially eliminate the unacceptable malaria burden, particularly in sub-Saharan Africa. Here we explore the cost effectiveness of using this approach in Africa using mathematical modeling and economical analysis. Overall, we find that pgSIT represents a cost-effective and promising approach to A. gambiae control in The Gambia, with the potential to deliver significant economic and social benefits. SummaryPrecision-guided sterile insect technique (pgSIT) is an extremely promising vector control intervention that can reduce and potentially eliminate the unacceptable malaria burden, particularly in sub-Saharan Africa. pgSIT is a safe, innovative, and highly targeted approach to mosquito control that combines the principles of the sterile insect technique (SIT) with advanced state-of-the-art technologies of genetic engineering (Akbari et al. 2023; M. Li et al. 2021; Kandul, Liu, and Akbari 2021; Kandul et al. 2019, 2022). The technique involves inundative releases of genetically sterile male mosquitoes into the environment to mate with their wild counterparts, sterilizing them in the process. After multiple releases, this method can suppress, and even eradicate pest populations without the use of chemical pesticides or other less specific agents. The use of pgSIT has the potential to be a lasting, safe, cost-effective, sustainable, and environmentally friendly method to suppress a target species, acting as a chemical-free, species-specific insecticide. Before this work is considered for field application, however, we need to have a robust, data-driven modeling framework that will accurately predict the outcome of pgSIT release scenarios for malaria control, and we need an assessment of the costs and health benefits of implementing this technology in a region in Africa. This cost assessment evaluates the economic feasibility of both capacity building and establishing the infrastructure for a pgSIT facility in The Gambia to control the deadly malaria mosquito vector Anopheles gambiae. We focused on the Upper River region (URR) for three key reasons: (1) this area is known to have the highest per capita malaria rates in The Gambia (2) this region has more comprehensive historical and current data on malaria incidence and prevalence, malaria-associated healthcare and prevention costs, and human demographic data and (3) its location could feasibly demonstrate that a single pgSIT production facility can support phased suppression on a regional scale and then upon local extinction be repurposed to support vector suppression on a country or continent-wide scale. The pgSIT treatment of the URR ([~]2069 km2) is predicted to prevent approximately 230 deaths and over 48,000 sick days per year. This estimate is based on a model for localized extinction of A. gambiae, reaching full epidemiological impact by the third year. There are multiple ways to calculate the value of this intervention monetarily, from the value of statistical life (VSL) to quality adjusted life years (QALY), which ranges from 334 million to 784 million USD saved in the first ten years of the facility being active. Other metrics such as willingness-to-pay (WTP), estimates the willingness of locals to contribute to malaria prevention financially, and estimates based on gross domestic product (GDP) growth predict this model to save either 47 million or 551 million USD, respectively, in the first ten years of intervention. This model assumes localized extinction of A. gambiae by the second year of intervention with repeated releases to maintain extinction despite seasonal mosquito migration from beyond the treatment area. Localized extinction, however, is expected to have a year-to-year suppression effect making it easier to suppress mosquito populations with reduced sterile male releases in subsequent years. It is, therefore, likely an underestimate of the costs and benefits of pgSIT sterile male releases. In later years, the release of sterile males from this facility could be redirected to new areas to expand the suppression region. Additionally, this facility would have a significant off-season where the facility is not producing sterile male A. gambiae. This off-season could, therefore, produce sterile males to suppress A. gambiae populations in other regions with seasonal malaria during the off-season in The Gambiae. It could even be used to mass produce pgSIT sterile males for other mosquito species, such as the dengue vector, Aedes aegypti, which have eggs that can be stored for many months and consequently can be stockpiled to aid in the suppression of dengue outbreaks, which are also common in the area. Initially, however, the off-season can be used to build local capacity for genetically engineered (GE) mosquitos, which would consist of training staff, optimizing procedures, and troubleshooting any issues that arose during the higher production phases. The costs for this approach will vary as there are some unknowns and variability in the expected efficiencies of the facility, equipment, and procedures. In particular, mosquito survival rates and fecundity may vary more widely at scale and selection of the rearing equipment and protocols, and the mosquito sorting technology will dictate production levels and procedures. This variability is factored into the cost, and, therefore, a range of technologies and costs are considered. The expected start-up costs range from 8.4 to 20 million USD, which includes all necessary development, field trials, equipment, facility construction, staffing, and other establishment costs. The major upfront costs vary by facility size and capital equipment, much of which is dependent on mosquito rearing efficiencies. The annual estimated operational costs range from approximately 780,000 to 1,300,000 USD, depending on equipment selection and the size of the facility. Annually, this intervention costs a little over 3 USD per person to suppress mosquitoes in the URR and prevent malaria transmission, which is about 21-36% of a malaria prevention WTP based on current interventions or 0.7-3.3% when using the VSL or QALY metrics. WTP is ten-fold less than the predicted benefits demonstrated by other metrics but shows that this facility could be made locally sustainable long term by local funding. Overall, the estimates of the capital and operational costs associated with the production of pgSIT sterile males, and the construction and management of the production facility indicate the cost savings associated with the annual decrease in morbidity and mortality (value of life) resulting from the use of pgSIT are significantly higher than the implementation costs. These estimates suggest that pgSIT represents a cost-effective and promising approach to A. gambiae control in The Gambia, with the potential to deliver significant economic and social benefits.\nHuman Plasma Proteomic Profile of Clonal Hematopoiesis\nAuthors: Yu, Z.; Vromman, A.; Nguyen, N. Q. H.; Vellarikkal, S. K.; Uddin, M. M.; Niroula, A.; Griffin, G.; Honigberg, M. C.; Lin, A. E.; Gibson, C. J.; Katz, D. H.; Tahir, U.; Haidermota, S.; Ganesh, S.; Antoine, T.; Weinstock, J.; Austin, T. R.; Vasan, R. S.; peloso, G. M.; Hornsby, W.; Ganz, P.; Manson, J. E.; Haring, B.; Kooperberg, C. L.; Reiner, A. P.; Bis, J. C.; Psaty, B. M.; Min, Y.-I.; Correa, A.; Lange, L. A.; Post, W.; Rotter, J. I.; Rich, S. S.; Wilson, J. G.; Ebert, B. L.; Yu, B.; Ballantyne, C. M.; Coresh, J.; Sankaran, V. G.; Bick, A. G.; Jaiswal, S.; Libby, P.; Gerszten, R. E.; NHLBI T\nScore: 11.3, Published: 2023-07-27 DOI: 10.1101/2023.07.25.550557\nPlasma proteomic profiles associated with subclinical somatic mutations in blood cells may offer novel insights in downstream clinical consequences. Here, we explore such patterns in clonal hematopoiesis of indeterminate potential (CHIP), which links to several cancer and non-cancer outcomes, including coronary artery disease. Among 12,911 ancestrally diverse participants (682 with CHIP) from NHLBI TOPMed with blood-based DNA sequencing and 1,148 common proteins measured by SomaScan, we identified 32 unique proteins associated with the most prevalent driver genes (DNMT3A, TET2, and ASXL1) after multiple testing corrections. These associations showed substantial heterogeneity by driver genes, sex, and race, were enriched for immune response and inflammation pathways, and were moderately replicated in UK Biobank (N=48,922) that used Olink for proteomics measurement. Murine single-cell RNA-sequencing data from aortic arch cells, inclusive of resident hematologic cells, in mice with Tet2-/- bone marrow and wild-type mice revealed corroborating differential expression of TET2-associated protein-encoding genes. Lastly, we apply these observations to identify 68 plasma proteins shared between CHIP and coronary artery disease.\nPathogenic CANVAS (AAGGG)n repeats stall DNA replication due to the formation of alternative DNA structures\nAuthors: Hisey, J. A.; Radchenko, E. A.; Ceschi, S.; Rastokina, A.; Mandel, N. H.; McGinty, R. J.; Matos-Rodrigues, G.; Hernandez, A.; Nussenzweig, A.; Mirkin, S. M.\nScore: 10.0, Published: 2023-07-26 DOI: 10.1101/2023.07.25.550509\nCANVAS is a recently characterized repeat expansion disease, most commonly caused by homozygous expansions of an intronic (A2G3)n repeat in the RFC1 gene. There are a multitude of repeat motifs found in the human population at this locus, some of which are pathogenic and others benign. In this study, we conducted structure-functional analyses of the main pathogenic (A2G3)n and the main nonpathogenic (A4G)n repeats. We found that the pathogenic, but not the nonpathogenic, repeat presents a potent, orientation-dependent impediment to DNA polymerization in vitro. The pattern of the polymerization blockage is consistent with triplex or quadruplex formation in the presence of magnesium or potassium ions, respectively. Chemical probing of both repeats in supercoiled DNA reveals triplex H-DNA formation by the pathogenic repeat. Consistently, bioinformatic analysis of the S1-END-seq data from human cell lines shows preferential H-DNA formation genome-wide by (A2G3)n motifs over (A4G)n motifs in vivo. Finally, the pathogenic, but not the non-pathogenic, repeat stalls replication fork progression in yeast and human cells. We hypothesize that CANVAS-causing (A2G3)n repeat represents a challenge to genome stability by folding into alternative DNA structures that stall DNA replication.\nThe influence of HLA genetic variation on plasma protein expression\nAuthors: Krishna, C.; Chiou, J.; Lee, I.; Kim, H. I.; Atalar Aksit, M.; Sakaue, S.; Von Schack, D.; Raychaudhuri, S.; Ziemek, D.; Hu, X.\nScore: 9.6, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550394\nPolymorphism in the human leukocyte antigen class I (HLA-I) and class II (HLA-II) genes is strongly implicated in susceptibility to immune-mediated diseases. However, the molecular effects of HLA genetic variation, including and beyond antigen presentation, remain unclear. Here we examined the effect of HLA genetic variation on the expression of 2940 plasma proteins using imputed HLA variants in 45,330 Europeans in the UK Biobank. We detected 504 proteins (17.1% of all proteins tested) affected by HLA genetic variation (HLA-pQTL), including widespread trans regulation of protein expression by autoimmune disease risk alleles. HLA-pQTL were enriched in gene families related to antigen presentation (e.g. B2M), T cell fate (CD8A; CD4), chemokines (CCL19; CCL21), and NK and macrophage receptors (KIR; LILRA/B), suggesting that HLA polymorphism affects both adaptive and innate immunity. HLA-pQTL also affected expression of diverse proteins with unclear roles in the immune response (e.g. SFTPD, LRPAP1, ENPP6, NPTX1), as well as drug targets for immune-mediated diseases, suggesting complex regulatory roles of the HLA loci. Among trans HLA-pQTL, HLA variants explained 0.1-42.9% of the protein expression variance. Fine-mapping revealed that most HLA-pQTL implicated amino acid positions within the peptide binding groove, suggesting that trans regulation of plasma protein expression by the HLA loci is primarily a consequence of antigen presentation. We also show that HLA-I and II uniquely affect different proteins and biological mechanisms. Altogether, our data reveal the effects of HLA genetic variation on protein expression and aid the interpretation of associations between HLA alleles and immune-mediated diseases.\nGxE PRS: Genotype-environment interaction in polygenic risk score models for quantitative and binary traits\nAuthors: Jayasinghe, D.; Momin, M.; Beckmann, K.; Hypponen, E.; Benyamin, B.; Lee, S. H.\nScore: 10.2, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549816\nThe use of polygenic risk score (PRS) models has transformed the field of genetics by enabling the prediction of complex traits and diseases based on an individuals genetic profile. However, the impact of genotype-environment interaction (GxE) on the performance and applicability of PRS models remains a crucial aspect to be explored. Currently, existing GxE PRS models are often inappropriately used, which can result in inflated type 1 error rates and compromised results. In this study, we propose a novel GxE PRS model that correctly incorporates the GxE component to analyze complex traits and diseases. Through extensive simulations, we demonstrate that our proposed model outperforms existing models in terms of controlling type 1 error rates and enhancing statistical power. Furthermore, we apply the proposed model to real data, and report significant GxE effects. Specifically, we highlight the impact of our model on both quantitative and binary traits. For quantitative traits, we uncover the GxE modulation of genetic effects on body mass index (BMI) by alcohol intake frequency (ALC). In the case of binary traits, we identify the GxE modulation of genetic effects on hypertension (HYP) by waist-to-hip ratio (WHR). These findings underscore the importance of employing a robust model that effectively controls type 1 error rates, thus preventing the occurrence of spurious GxE signals. To facilitate the implementation of our approach, we have developed an innovative R software package called GxE PRS, specifically designed to detect and estimate GxE effects. Overall, our study highlights the importance of accurate GxE modeling and its implications for genetic risk prediction, while providing a practical tool to support further research in this area.\nThe density of regulatory information is a major determinant of evolutionary constraint on non-coding DNA in Drosophila\nAuthors: Sabaris, G.; Ortiz, D. M.; Laiker, I.; Mayansky, I.; Naik, S.; Cavalli, G.; Stern, D. L.; Preger-Ben Noon, E.; Frankel, N.\nScore: 6.8, Published: 2023-07-22 DOI: 10.1101/2023.07.21.550043\nThe density and distribution of regulatory information in non-coding DNA of eukaryotic genomes is largely unknown. Evolutionary analyses have estimated that [~]60% of nucleotides in intergenic regions of the D. melanogaster genome is functionally relevant. This estimate is difficult to reconcile with the commonly accepted idea that enhancers are compact regulatory elements that generally encompass less than 1 kilobase of DNA. Here, we approached this issue through a functional dissection of the regulatory region of the gene shavenbaby (svb). Most of the [~]90 kilobases of this large regulatory region is highly conserved in the genus Drosophila, though characterized enhancers occupy a small fraction of this region. By analyzing the regulation of svb in different contexts of Drosophila development, we found that the regulatory architecture that drives svb expression in the abdominal pupal epidermis is organized in a dramatically different way than the information that drives svb expression in the embryonic epidermis. While in the embryonic epidermis svb is activated by compact and dispersed enhancers, svb expression in the pupal epidermis is driven by large regions with enhancer activity, which occupy a great portion of the svb cis-regulatory DNA. We observed that other developmental genes also display a dense distribution of putative regulatory elements in their regulatory regions. Furthermore, we found that a large percentage of conserved non-coding DNA of the Drosophila genome is contained within putative regulatory DNA. These results suggest that part of the evolutionary constraint on non-coding DNA of Drosophila is explained by the density of regulatory information.\n",
  "wordCount" : "3566",
  "inLanguage": "en",
  "datePublished": "2023-07-30T10:36:57Z",
  "dateModified": "2023-07-30T10:36:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on July 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549478">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549478" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549478">
        <p class="paperTitle">Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549478" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549478" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.</p>
        <p class="info">Score: 400.9, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549478' target='https://doi.org/10.1101/2023.07.18.549478'> 10.1101/2023.07.18.549478</a></p>
        <p class="abstract">Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging. Epigenetic clocks have been shown to be valuable tools for measuring biological age in a variety of tissues and samples. As such, these epigenetic clocks can determine accelerated biological aging and time-to-mortality across various tissues. Previous reports have shown accelerated biological aging and telomere attrition acceleration following SARS-CoV-2 infection. However, the effect of accelerated epigenetic aging on outcome (death/recovery) in COVID-19 patients with Acute Respiratory Distress Syndrome (ARDS) has not been well investigated. In this study, we measured DNA methylation age and telomere attrition in 87 severe COVID-19 cases with ARDS under mechanical ventilation. Furthermore, we compared dynamic changes in epigenetic aging across multiples time-points until recovery or death. Epigenetic age was measured using the Horvath, Hannum, DNAm skin and blood, GrimAge, and PhenoAge clocks, whereas telomere length was calculated using the surrogate marker DNAmTL. Our analysis revealed significant accelerated epigenetic aging but no telomere attrition acceleration in severe COVID-19 cases. In addition, we observed epigenetic age deceleration at inclusion vs end of follow-up in recovered but not in deceased COVID-19 cases using certain clocks. When comparing dynamic changes in epigenetic age acceleration (EAA), we detected higher EAA using both the Horvath and PhenoAge clocks in deceased vs recovered patients. The DNAmTL measurements revealed telomere attrition acceleration in deceased COVID19 patients between inclusion and end of follow-up as well as a significant change in dynamic telomere attrition acceleration when comparing patients who recovered vs those who died. In conclusion, EAA and telomere attrition acceleration was associated with treatment outcome in hospitalized COVID-19 Patients with ARDS. A better understanding of the long-term effects of EAA in COVID19 patients and how they might contribute to Long COVID symptoms in recovered individuals is urgently needed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550074">
        <p class="paperTitle">isGWAS: ultra-high-throughput, scalable and equitable inference of genetic associations with disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Foley, C. N.; Kuncheva, Z.; Marioni, R.; Runz, H.; Sun, B.</p>
        <p class="info">Score: 26.2, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550074' target='https://doi.org/10.1101/2023.07.21.550074'> 10.1101/2023.07.21.550074</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have proven a powerful tool for human geneticists to generate biological insights or hypotheses for drug discovery. Nevertheless, a dependency on sensitive individual-level data together with ever-increasing cohort sample sizes, numbers of variants and phenotypes studied put a strain on existing algorithms, limiting the GWAS approach from maximising potential. Here we present in-silico GWAS (isGWAS), a uniquely scalable algorithm to infer regression parameters in case-control GWAS from cohort-level summary data. For any sample size, isGWAS computes a variant-disease association parameter in [~]1 millisecond, or [~]11m variants in UK-Biobank within [~]4 minutes ([~]1500-fold faster than state-of-the-art). Extensive simulations and empirical tests demonstrate that isGWAS results are highly comparable to traditional regression-based approaches. We further introduce a heuristic re-sampling algorithm, leapfrog re-sampler (LRS), to extrapolate association results to semi-virtually enlarged cohorts. Owing to significant computational gains we anticipate a broad use of isGWAS and LRS which are customizable on a web interface.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549962" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549962">
        <p class="paperTitle">A small RNA from a natural bacterial pathogen of C. elegans induces transgenerational inheritance of learned avoidance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sengupta, T.; Ange, J. S.; Moore, R. S.; Kaletsky, R. T.; Marogi, J. T.; Myhrvold, C.; Gitai, Z.; Murphy, C. T.</p>
        <p class="info">Score: 15.7, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549962' target='https://doi.org/10.1101/2023.07.20.549962'> 10.1101/2023.07.20.549962</a></p>
        <p class="abstract">A small RNA from a clinical isolate of Pseudomonas aeruginosa, PA14, induces learned avoidance and its transgenerational inheritance in C. elegans. However, it is not known if small RNAs from bacteria found in C. elegans natural habitat can regulate host behavior and produce heritable behavioral effects. Here we found that GRb0427, a pathogenic Pseudomonas vranovensis strain isolated from the C. elegans microbiota, similarly affects worm behavior: worms learn to avoid this pathogenic bacterium following exposure, and this learned avoidance is inherited for four generations. The learned response is entirely mediated by small RNAs, which induce avoidance and transgenerational inheritance in both the laboratory N2 strain and a natural isolate of C. elegans, JU1580, providing further support that such mechanisms of learning and inheritance exist in the wild. A small RNA, Pv1, matches the sequence of C. elegans maco-1, and is both necessary and sufficient to induce learned avoidance in worms. However, Pv1 also results in avoidance of a beneficial microbiome strain, P. mendocina; this maladaptive response may favor reversal of the transgenerational memory after a few generations. Our findings suggest that bacterial small RNA-mediated regulation of host behavior and its transgenerational inheritance are functional in C. elegans natural environment, and that different bacterial small RNA-mediated regulation systems evolved independently but define shared molecular features of bacterial small RNAs that produce transgenerationally-inherited effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549734">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549734" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549734">
        <p class="paperTitle">Genetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549734" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549734" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suhre, K.</p>
        <p class="info">Score: 15.5, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549734' target='https://doi.org/10.1101/2023.07.19.549734'> 10.1101/2023.07.19.549734</a></p>
        <p class="abstract">Protein quantitative trait loci (pQTLs) are an invaluable source of information for drug target development as they provide genetic evidence to support protein function, suggest relationships between cis- and trans-associated proteins, and link proteins to disease where they collocate with genetic risk loci for clinical endpoints. Using the recently released Olink proteomics data for 1,463 proteins measured in over 54,000 samples of the UK Biobank we identified and replicated 4,248 associations with 2,821 ratios between protein levels (rQTLs) where the strengths of association at known pQTL loci increased by up to several hundred orders of magnitude. We attribute this increase in statistical power (p-gain) to accounting for genetic and non-genetic variance shared by the two proteins in the ratio pair. Protein pairs with a significant p-gain were 7.6-fold enriched in known protein-protein interactions, suggesting that their ratios reflect biological links between the implicated proteins. We then conducted a GWAS on the 2,821 ratios and identified 2,527 novel rQTLs, increasing the number of discovered genetic signals compared to the original protein-only GWAS by 24.7%. At examples we demonstrate that this approach can identify novel loci of clinical relevance, support causal gene identification, and reveal complex networks of interacting proteins. Taken together, our study adds significant value to the genetic insights that can be derived from the UKB proteomics data and motivates the wider use of ratios in large scale GWAS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549762">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549762" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549762">
        <p class="paperTitle">Cost-effectiveness of Precision Guided SIT for Control of Anopheles gambiae in the Upper River Region, The Gambia.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549762" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549762" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gendron, W. A. C.; Raban, R.; Mondal, A.; Sanchez C., H. M.; Smidler, A.; Zilberman, D.; Ilboudo, P. G.; D&#39;Alessandro, U.; Marshall, J. M.; Akbari, O. S.</p>
        <p class="info">Score: 11.7, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549762' target='https://doi.org/10.1101/2023.07.20.549762'> 10.1101/2023.07.20.549762</a></p>
        <p class="abstract">Precision-guided sterile insect technique (pgSIT) is an extremely promising vector control intervention that can reduce and potentially eliminate the unacceptable malaria burden, particularly in sub-Saharan Africa. Here we explore the cost effectiveness of using this approach in Africa using mathematical modeling and economical analysis. Overall, we find that pgSIT represents a cost-effective and promising approach to A. gambiae control in The Gambia, with the potential to deliver significant economic and social benefits.

SummaryPrecision-guided sterile insect technique (pgSIT) is an extremely promising vector control intervention that can reduce and potentially eliminate the unacceptable malaria burden, particularly in sub-Saharan Africa. pgSIT is a safe, innovative, and highly targeted approach to mosquito control that combines the principles of the sterile insect technique (SIT) with advanced state-of-the-art technologies of genetic engineering (Akbari et al. 2023; M. Li et al. 2021; Kandul, Liu, and Akbari 2021; Kandul et al. 2019, 2022). The technique involves inundative releases of genetically sterile male mosquitoes into the environment to mate with their wild counterparts, sterilizing them in the process. After multiple releases, this method can suppress, and even eradicate pest populations without the use of chemical pesticides or other less specific agents. The use of pgSIT has the potential to be a lasting, safe, cost-effective, sustainable, and environmentally friendly method to suppress a target species, acting as a chemical-free, species-specific insecticide.

Before this work is considered for field application, however, we need to have a robust, data-driven modeling framework that will accurately predict the outcome of pgSIT release scenarios for malaria control, and we need an assessment of the costs and health benefits of implementing this technology in a region in Africa. This cost assessment evaluates the economic feasibility of both capacity building and establishing the infrastructure for a pgSIT facility in The Gambia to control the deadly malaria mosquito vector Anopheles gambiae. We focused on the Upper River region (URR) for three key reasons: (1) this area is known to have the highest per capita malaria rates in The Gambia (2) this region has more comprehensive historical and current data on malaria incidence and prevalence, malaria-associated healthcare and prevention costs, and human demographic data and (3) its location could feasibly demonstrate that a single pgSIT production facility can support phased suppression on a regional scale and then upon local extinction be repurposed to support vector suppression on a country or continent-wide scale.

The pgSIT treatment of the URR ([~]2069 km2) is predicted to prevent approximately 230 deaths and over 48,000 sick days per year. This estimate is based on a model for localized extinction of A. gambiae, reaching full epidemiological impact by the third year. There are multiple ways to calculate the value of this intervention monetarily, from the value of statistical life (VSL) to quality adjusted life years (QALY), which ranges from 334 million to 784 million USD saved in the first ten years of the facility being active. Other metrics such as willingness-to-pay (WTP), estimates the willingness of locals to contribute to malaria prevention financially, and estimates based on gross domestic product (GDP) growth predict this model to save either 47 million or 551 million USD, respectively, in the first ten years of intervention. This model assumes localized extinction of A. gambiae by the second year of intervention with repeated releases to maintain extinction despite seasonal mosquito migration from beyond the treatment area. Localized extinction, however, is expected to have a year-to-year suppression effect making it easier to suppress mosquito populations with reduced sterile male releases in subsequent years. It is, therefore, likely an underestimate of the costs and benefits of pgSIT sterile male releases. In later years, the release of sterile males from this facility could be redirected to new areas to expand the suppression region. Additionally, this facility would have a significant off-season where the facility is not producing sterile male A. gambiae. This off-season could, therefore, produce sterile males to suppress A. gambiae populations in other regions with seasonal malaria during the off-season in The Gambiae. It could even be used to mass produce pgSIT sterile males for other mosquito species, such as the dengue vector, Aedes aegypti, which have eggs that can be stored for many months and consequently can be stockpiled to aid in the suppression of dengue outbreaks, which are also common in the area. Initially, however, the off-season can be used to build local capacity for genetically engineered (GE) mosquitos, which would consist of training staff, optimizing procedures, and troubleshooting any issues that arose during the higher production phases.

The costs for this approach will vary as there are some unknowns and variability in the expected efficiencies of the facility, equipment, and procedures. In particular, mosquito survival rates and fecundity may vary more widely at scale and selection of the rearing equipment and protocols, and the mosquito sorting technology will dictate production levels and procedures. This variability is factored into the cost, and, therefore, a range of technologies and costs are considered. The expected start-up costs range from 8.4 to 20 million USD, which includes all necessary development, field trials, equipment, facility construction, staffing, and other establishment costs. The major upfront costs vary by facility size and capital equipment, much of which is dependent on mosquito rearing efficiencies. The annual estimated operational costs range from approximately 780,000 to 1,300,000 USD, depending on equipment selection and the size of the facility. Annually, this intervention costs a little over 3 USD per person to suppress mosquitoes in the URR and prevent malaria transmission, which is about 21-36% of a malaria prevention WTP based on current interventions or 0.7-3.3% when using the VSL or QALY metrics. WTP is ten-fold less than the predicted benefits demonstrated by other metrics but shows that this facility could be made locally sustainable long term by local funding.

Overall, the estimates of the capital and operational costs associated with the production of pgSIT sterile males, and the construction and management of the production facility indicate the cost savings associated with the annual decrease in morbidity and mortality (value of life) resulting from the use of pgSIT are significantly higher than the implementation costs. These estimates suggest that pgSIT represents a cost-effective and promising approach to A. gambiae control in The Gambia, with the potential to deliver significant economic and social benefits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550557">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550557" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550557">
        <p class="paperTitle">Human Plasma Proteomic Profile of Clonal Hematopoiesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550557" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550557" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, Z.; Vromman, A.; Nguyen, N. Q. H.; Vellarikkal, S. K.; Uddin, M. M.; Niroula, A.; Griffin, G.; Honigberg, M. C.; Lin, A. E.; Gibson, C. J.; Katz, D. H.; Tahir, U.; Haidermota, S.; Ganesh, S.; Antoine, T.; Weinstock, J.; Austin, T. R.; Vasan, R. S.; peloso, G. M.; Hornsby, W.; Ganz, P.; Manson, J. E.; Haring, B.; Kooperberg, C. L.; Reiner, A. P.; Bis, J. C.; Psaty, B. M.; Min, Y.-I.; Correa, A.; Lange, L. A.; Post, W.; Rotter, J. I.; Rich, S. S.; Wilson, J. G.; Ebert, B. L.; Yu, B.; Ballantyne, C. M.; Coresh, J.; Sankaran, V. G.; Bick, A. G.; Jaiswal, S.; Libby, P.; Gerszten, R. E.; NHLBI T</p>
        <p class="info">Score: 11.3, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550557' target='https://doi.org/10.1101/2023.07.25.550557'> 10.1101/2023.07.25.550557</a></p>
        <p class="abstract">Plasma proteomic profiles associated with subclinical somatic mutations in blood cells may offer novel insights in downstream clinical consequences. Here, we explore such patterns in clonal hematopoiesis of indeterminate potential (CHIP), which links to several cancer and non-cancer outcomes, including coronary artery disease. Among 12,911 ancestrally diverse participants (682 with CHIP) from NHLBI TOPMed with blood-based DNA sequencing and 1,148 common proteins measured by SomaScan, we identified 32 unique proteins associated with the most prevalent driver genes (DNMT3A, TET2, and ASXL1) after multiple testing corrections. These associations showed substantial heterogeneity by driver genes, sex, and race, were enriched for immune response and inflammation pathways, and were moderately replicated in UK Biobank (N=48,922) that used Olink for proteomics measurement. Murine single-cell RNA-sequencing data from aortic arch cells, inclusive of resident hematologic cells, in mice with Tet2-/- bone marrow and wild-type mice revealed corroborating differential expression of TET2-associated protein-encoding genes. Lastly, we apply these observations to identify 68 plasma proteins shared between CHIP and coronary artery disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550509">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550509" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550509">
        <p class="paperTitle">Pathogenic CANVAS (AAGGG)n repeats stall DNA replication due to the formation of alternative DNA structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550509" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550509" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hisey, J. A.; Radchenko, E. A.; Ceschi, S.; Rastokina, A.; Mandel, N. H.; McGinty, R. J.; Matos-Rodrigues, G.; Hernandez, A.; Nussenzweig, A.; Mirkin, S. M.</p>
        <p class="info">Score: 10.0, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550509' target='https://doi.org/10.1101/2023.07.25.550509'> 10.1101/2023.07.25.550509</a></p>
        <p class="abstract">CANVAS is a recently characterized repeat expansion disease, most commonly caused by homozygous expansions of an intronic (A2G3)n repeat in the RFC1 gene. There are a multitude of repeat motifs found in the human population at this locus, some of which are pathogenic and others benign. In this study, we conducted structure-functional analyses of the main pathogenic (A2G3)n and the main nonpathogenic (A4G)n repeats. We found that the pathogenic, but not the nonpathogenic, repeat presents a potent, orientation-dependent impediment to DNA polymerization in vitro. The pattern of the polymerization blockage is consistent with triplex or quadruplex formation in the presence of magnesium or potassium ions, respectively. Chemical probing of both repeats in supercoiled DNA reveals triplex H-DNA formation by the pathogenic repeat. Consistently, bioinformatic analysis of the S1-END-seq data from human cell lines shows preferential H-DNA formation genome-wide by (A2G3)n motifs over (A4G)n motifs in vivo. Finally, the pathogenic, but not the non-pathogenic, repeat stalls replication fork progression in yeast and human cells. We hypothesize that CANVAS-causing (A2G3)n repeat represents a challenge to genome stability by folding into alternative DNA structures that stall DNA replication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550394">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550394" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550394">
        <p class="paperTitle">The influence of HLA genetic variation on plasma protein expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550394" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550394" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishna, C.; Chiou, J.; Lee, I.; Kim, H. I.; Atalar Aksit, M.; Sakaue, S.; Von Schack, D.; Raychaudhuri, S.; Ziemek, D.; Hu, X.</p>
        <p class="info">Score: 9.6, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550394' target='https://doi.org/10.1101/2023.07.24.550394'> 10.1101/2023.07.24.550394</a></p>
        <p class="abstract">Polymorphism in the human leukocyte antigen class I (HLA-I) and class II (HLA-II) genes is strongly implicated in susceptibility to immune-mediated diseases. However, the molecular effects of HLA genetic variation, including and beyond antigen presentation, remain unclear. Here we examined the effect of HLA genetic variation on the expression of 2940 plasma proteins using imputed HLA variants in 45,330 Europeans in the UK Biobank. We detected 504 proteins (17.1% of all proteins tested) affected by HLA genetic variation (HLA-pQTL), including widespread trans regulation of protein expression by autoimmune disease risk alleles. HLA-pQTL were enriched in gene families related to antigen presentation (e.g. B2M), T cell fate (CD8A; CD4), chemokines (CCL19; CCL21), and NK and macrophage receptors (KIR; LILRA/B), suggesting that HLA polymorphism affects both adaptive and innate immunity. HLA-pQTL also affected expression of diverse proteins with unclear roles in the immune response (e.g. SFTPD, LRPAP1, ENPP6, NPTX1), as well as drug targets for immune-mediated diseases, suggesting complex regulatory roles of the HLA loci. Among trans HLA-pQTL, HLA variants explained 0.1-42.9% of the protein expression variance. Fine-mapping revealed that most HLA-pQTL implicated amino acid positions within the peptide binding groove, suggesting that trans regulation of plasma protein expression by the HLA loci is primarily a consequence of antigen presentation. We also show that HLA-I and II uniquely affect different proteins and biological mechanisms. Altogether, our data reveal the effects of HLA genetic variation on protein expression and aid the interpretation of associations between HLA alleles and immune-mediated diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549816">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549816" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549816">
        <p class="paperTitle">GxE PRS: Genotype-environment interaction in polygenic risk score models for quantitative and binary traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549816" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549816" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jayasinghe, D.; Momin, M.; Beckmann, K.; Hypponen, E.; Benyamin, B.; Lee, S. H.</p>
        <p class="info">Score: 10.2, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549816' target='https://doi.org/10.1101/2023.07.20.549816'> 10.1101/2023.07.20.549816</a></p>
        <p class="abstract">The use of polygenic risk score (PRS) models has transformed the field of genetics by enabling the prediction of complex traits and diseases based on an individuals genetic profile. However, the impact of genotype-environment interaction (GxE) on the performance and applicability of PRS models remains a crucial aspect to be explored. Currently, existing GxE PRS models are often inappropriately used, which can result in inflated type 1 error rates and compromised results. In this study, we propose a novel GxE PRS model that correctly incorporates the GxE component to analyze complex traits and diseases. Through extensive simulations, we demonstrate that our proposed model outperforms existing models in terms of controlling type 1 error rates and enhancing statistical power. Furthermore, we apply the proposed model to real data, and report significant GxE effects. Specifically, we highlight the impact of our model on both quantitative and binary traits. For quantitative traits, we uncover the GxE modulation of genetic effects on body mass index (BMI) by alcohol intake frequency (ALC). In the case of binary traits, we identify the GxE modulation of genetic effects on hypertension (HYP) by waist-to-hip ratio (WHR). These findings underscore the importance of employing a robust model that effectively controls type 1 error rates, thus preventing the occurrence of spurious GxE signals. To facilitate the implementation of our approach, we have developed an innovative R software package called GxE PRS, specifically designed to detect and estimate GxE effects. Overall, our study highlights the importance of accurate GxE modeling and its implications for genetic risk prediction, while providing a practical tool to support further research in this area.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550043">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550043" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550043">
        <p class="paperTitle">The density of regulatory information is a major determinant of evolutionary constraint on non-coding DNA in Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550043" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550043" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sabaris, G.; Ortiz, D. M.; Laiker, I.; Mayansky, I.; Naik, S.; Cavalli, G.; Stern, D. L.; Preger-Ben Noon, E.; Frankel, N.</p>
        <p class="info">Score: 6.8, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550043' target='https://doi.org/10.1101/2023.07.21.550043'> 10.1101/2023.07.21.550043</a></p>
        <p class="abstract">The density and distribution of regulatory information in non-coding DNA of eukaryotic genomes is largely unknown. Evolutionary analyses have estimated that [~]60% of nucleotides in intergenic regions of the D. melanogaster genome is functionally relevant. This estimate is difficult to reconcile with the commonly accepted idea that enhancers are compact regulatory elements that generally encompass less than 1 kilobase of DNA. Here, we approached this issue through a functional dissection of the regulatory region of the gene shavenbaby (svb). Most of the [~]90 kilobases of this large regulatory region is highly conserved in the genus Drosophila, though characterized enhancers occupy a small fraction of this region. By analyzing the regulation of svb in different contexts of Drosophila development, we found that the regulatory architecture that drives svb expression in the abdominal pupal epidermis is organized in a dramatically different way than the information that drives svb expression in the embryonic epidermis. While in the embryonic epidermis svb is activated by compact and dispersed enhancers, svb expression in the pupal epidermis is driven by large regions with enhancer activity, which occupy a great portion of the svb cis-regulatory DNA. We observed that other developmental genes also display a dense distribution of putative regulatory elements in their regulatory regions. Furthermore, we found that a large percentage of conserved non-coding DNA of the Drosophila genome is contained within putative regulatory DNA. These results suggest that part of the evolutionary constraint on non-coding DNA of Drosophila is explained by the density of regulatory information.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
