<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency
Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.
Score: 37.7, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808
Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency
Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.
Score: 37.7, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808
Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-01T10:38:40+00:00" />
<meta property="article:modified_time" content="2023-11-01T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency
Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.
Score: 37.7, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808
Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency\nAuthors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.\nScore: 37.7, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808\nEctopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming.",
  "keywords": [
    
  ],
  "articleBody": " Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency\nAuthors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.\nScore: 37.7, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808\nEctopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming. Using neural networks that map DNA sequence to ATAC-seq profiles at base-resolution, we annotated cell-state-specific predictive transcription factor (TF) motif syntax in regulatory elements, inferred affinity- and concentration-dependent dynamics of Tn5-bias corrected TF footprints, linked peaks to putative target genes, and elucidated rewiring of TF- to-gene cis-regulatory networks. Our models reveal that early in reprogramming, OSK, at supraphysiological concentrations, rapidly open transient regulatory elements by occupying non-canonical low-affinity binding sites. As OSK concentration falls, the accessibility of these transient elements decays as a function of motif affinity. We find that these OSK-dependent transient elements sequester the somatic TF AP-1. This redistribution is strongly associated with the silencing of fibroblast-specific genes within individual nuclei. Together, our integrated single-cell resource and models reveal insights into the cis-regulatory code of reprogramming at unprecedented resolution, connect TF stoichiometry and motif syntax to diversification of cell fate trajectories, and provide new perspectives on the dynamics and role of transient regulatory elements in somatic silencing.\nAssessing the impact of 20th century internal migrations on the genetic structure of Estonia\nAuthors: Kuznetsov, I. A.; Estonian Biobank Research Team, ; Metspalu, M.; Vainik, U.; Pagani, L.; Montinaro, F.; Pankratov, V.\nScore: 21.6, Published: 2023-10-29 DOI: 10.1101/2023.10.25.564036\nSpatial genetic structure observed in many human populations is in large part attributed to past demographic events and isolation by distance. However, how intensifying migration affects this structure remains understudied. Here we harness a sample of more than 180 thousand individuals to explore the genetic correlates and consequences of contemporary migrations in Estonia. While we show that migration smoothens the genome-wide genetic structure, it intensifies inter-regional differences in polygenic scores (PGS) for certain traits, derived both from population as well as within-sibship studies. The strongest effect is observed for educational attainment which is consistent with previous observations in the UK and suggests this to be a general pattern. We explore those regional differences in PGS in terms of the driving forces behind them and from a temporal perspective, and suggest urbanisation as a major driver for this pattern in Estonia from at least the first half of the 20th century.\nAlphaMissense is better correlated with functional assays of missense impact than earlier prediction algorithms\nAuthors: Ljungdahl, A.; Kohani, S.; Page, N. F.; Wells, E. S.; Wigdor, E. M.; Dong, S.; Sanders, S. J.\nScore: 14.2, Published: 2023-10-27 DOI: 10.1101/2023.10.24.562294\nMissense variants that alter a single amino acid in the encoded protein contribute to many human disorders but pose a substantial challenge in interpretation. Though these variants can be reliably identified through sequencing, distinguishing the clinically significant ones remains difficult, such that \"Variants of Unknown Significance\" outnumber those classified as \"Pathogenic\" or \"Likely Pathogenic.\" Numerous in silico approaches have been developed to predict the functional impact of missense variants to inform clinical interpretation, the latest being AlphaMissense, which uses artificial intelligence methods trained on predicted protein structure. To independently assess the performance of AlphaMissense and 38 other predictors of missense severity, we compared predictions to data from multiplexed assays of variant effect (MAVE). MAVE experiments generate almost every possible individual amino acid change in a gene and measure their functional impact using a high-throughput assay. Assessing 17,696 variants across five genes (DDX3X, MSH2, PTEN, KCNQ4, and BRCA1), we find that AlphaMissense is consistently one of the top five algorithms based on correlation with functional impact and is the best-correlated algorithm for two genes. We conclude that AlphaMissense represents the current best-in-class predictor by this metric; however, the improvement over other algorithms is modest. We note that multiple missense predictors, including AlphaMissense, appear to overcall variants as pathogenic despite minimal functional impact and that substantially more high-quality training data, including consistently analyzed patient cohorts and MAVE analyses, are required to improve accuracy.\nGenotype prediction of 336,463 samples from public expression data\nAuthors: Razi, A.; Lo, C. C.; Wang, S.; Leek, J. T.; Hansen, K. D.\nScore: 15.5, Published: 2023-10-22 DOI: 10.1101/2023.10.21.562237\nTens of thousands of RNA-sequencing experiments comprising hundreds of thousands of individual samples have now been performed. These data represent a broad range of experimental conditions, sequencing technologies, and hypotheses under study. The Recount project has aggregated and uniformly processed hundreds of thousands of publicly available RNA-seq samples. Most of these samples only include RNA expression measurements; genotype data for these same samples would enable a wide range of analyses including variant prioritization, eQTL analysis, and studies of allele specific expression. Here, we developed a statistical model based on the existing reference and alternative read counts from the RNA-seq experiments available through Recount3 to predict genotypes at autosomal biallelic loci in coding regions. We demonstrate the accuracy of our model using large-scale studies that measured both gene expression and genotype genome-wide. We show that our predictive model is highly accurate with 99.5% overall accuracy, 99.6% major allele accuracy, and 90.4% minor allele accuracy. Our model is robust to tissue and study effects, provided the coverage is high enough. We applied this model to genotype all the samples in Recount3 and provide the largest ready-to-use expression repository containing genotype information. We illustrate that the predicted genotype from RNA-seq data is sufficient to unravel the underlying population structure of samples in Recount3 using Principal Component Analysis.\nTrait selection strategy in multi-trait GWAS: Boosting SNPs discoverability\nAuthors: Suzuki, Y.; Menager, H.; Brancotte, B.; Vernet, R.; Nerin, C.; Boetto, C.; Auvergne, A.; Linhard, C.; Torchet, R.; Lechat, P.; Troubat, L.; Cho, M. H.; Bouzigon, E.; Aschard, H.; Julienne, H.\nScore: 9.1, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564319\nSince the first Genome-Wide Association Studies (GWAS), thousands of variant-trait associations have been discovered. However, the sample size required to detect additional variants using standard univariate association screening is increasingly prohibitive. Multi-trait GWAS offers a relevant alternative: it can improve statistical power and lead to new insights about gene function and the joint genetic architecture of human phenotypes. Although many methodological hurdles of multi-trait testing have been discussed, the strategy to select trait, among overwhelming possibilities, has been overlooked. In this study, we conducted extensive multi-trait tests using JASS (Joint Analysis of Summary Statistics) and assessed which genetic features of the analysed sets were associated with an increased detection of variants as compared to univariate screening. Our analyses identified multiple factors associated with the gain in the association detection in multi-trait tests. Together, these factors of the analysed sets are predictive of the gain of the multi-trait test (Pearson's rho equal to 0.43 between the observed and predicted gain, P \u003c 1.6 x 10-60). Applying an alternative multi-trait approach (MTAG, multi-trait analysis of GWAS), we found that in most scenarios but particularly those with larger numbers of traits, JASS outperformed MTAG. Finally, we benchmark several strategies to select set of traits including the prevalent strategy of selecting clinically similar traits, which systematically underperformed selecting clinically heterogenous traits or selecting sets that issued from our data-driven models. This work provides a unique picture of the determinant of multi-trait GWAS statistical power and outline practical strategies for multi-trait testing.\nDrosophila models of PIGA-CDG mirror patient phenotypes\nAuthors: Thorpe, H. J.; Owings, K. G.; Aziz, M. C.; Haller, M.; Coelho, E.; Chow, C. Y.\nScore: 6.2, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564441\nMutations in the phosphatidylinositol glycan biosynthesis class A (PIGA) gene cause a rare, X-linked recessive congenital disorder of glycosylation (CDG). PIGA-CDG is characterized by seizures, intellectual and developmental delay, and congenital malformations. The PIGA gene encodes an enzyme involved in the first step of GPI anchor biosynthesis. There are over 100 GPI anchored proteins that attach to the cell surface and are involved in cell signaling, immunity, and adhesion. Little is known about the pathophysiology of PIGA-CDG. Here we describe the first Drosophila model of PIGA-CDG and demonstrate that loss of PIG-A function in Drosophila accurately models the human disease. As expected, complete loss of PIG-A function is larval lethal. Heterozygous null animals appear healthy, but when challenged, have a seizure phenotype similar to what is observed in patients. To identify the cell-type specific contributions to disease, we generated neuron- and glia-specific knockdown of PIG-A. Neuron-specific knockdown resulted in reduced lifespan and a number of neurological phenotypes, but no seizure phenotype. Glia-knockdown also reduced lifespan and, notably, resulted in a very strong seizure phenotype. RNAseq analyses demonstrated that there are fundamentally different molecular processes that are disrupted when PIG-A function is eliminated in different cell types. In particular, loss of PIG-A in neurons resulted in upregulation of glycolysis, but loss of PIG-A in glia resulted in upregulation of protein translation machinery. Here we demonstrate that Drosophila is a good model of PIGA-CDG and provide new data resources for future study of PIGA-CDG and other GPI anchor disorders. Article SummaryPIGA-CDG is a rare genetic disorder. In order to study this rare disease, we generated and characterized several Drosophila models of PIGA-CDG. These models faithfully recapitulate different patient phenotypes, including movement disorder and seizures. Drosophila is a good model for PIGA-CDG and other GPI anchor disorders.\nLarge-scale causal discovery using interventional data sheds light on the regulatory network architecture of blood traits\nAuthors: Brown, B. C.; Morris, J. A.; Lappalainen, T.; Knowles, D. A.\nScore: 6.2, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562293\nInference of directed biological networks is an important but notoriously challenging problem. We introduce inverse sparse regression (inspre), an approach to learning causal networks that leverages large-scale intervention-response data. Applied to 788 genes from the genome-wide perturb-seq dataset, inspre helps elucidate the network architecture of blood traits.\nA comparative approach for selecting orthologous candidate genes underlying signal in genome-wide association studies across multiple species\nAuthors: Whitt, L.; Mahood, E. H.; Ziegler, G.; Luebbert, C.; Gillman, J. D.; Norton, G. J.; Price, A. H.; Salt, D. E.; Dilkes, B.; Baxter, I.\nScore: 7.8, Published: 2023-10-06 DOI: 10.1101/2023.10.05.561051\nAdvances in quantitative genetics have enabled researchers to identify genomic regions associated with changes in phenotype. However, genomic regions can contain hundreds to thousands of genes, and progressing from genomic regions to candidate genes is still challenging. In genome-wide association studies (GWAS) measuring elemental accumulation (ionomic) traits, a mere 5% of loci are associated with a known ionomic gene - indicating that many causal genes are still unknown. To select candidates for the remaining 95% of loci, we developed a method to identify conserved genes underlying GWAS loci in multiple species. For 19 ionomic traits, we identified 14,336 candidates across Arabidopsis, soybean, rice, maize, and sorghum. We calculated the likelihood of candidates with random permutations of the data and determined that most of the top 10% of candidates were orthologous genes linked to GWAS loci across all five species. The candidate list also includes orthologous genes with previously established ionomic functions in Arabidopsis and rice. Our methods highlight the conserved nature of ionomic genetic regulators and enable the identification of previously unknown ionomic genes. Author summaryIdentifying the genes contributing to changes in a given trait is challenging. Many genes can be near the region of interest, but proximity does not always translate to causality. We use other methods to narrow our focus to the genes most likely involved in our trait of interest before confirming their involvement through confirmation experiments. However, these other methods are often time, labor, and resource-consuming. We developed an approach to narrow these gene lists before these laborious methods are required. Through testing with genetic markers for elemental (i.e., calcium, iron, zinc) uptake, we found that comparing markers across multiple species for nearby evolutionarily conserved genes is a successful approach. Weve produced a list of candidate genes likely to be involved in elemental uptake traits, including previously known elemental uptake genes and genes whose potential elemental uptake function has yet to be observed. Some of these genes would not have been considered according to the significance threshold within a single GWAS, but combining comparable datasets across species has collectively boosted their signal. Methods like our approach are useful for reducing candidate lists to conserve resources spent in functional characterization experiments and encouraging the discovery of new functional roles of these genes.\nA natural bacterial pathogen of C. elegans uses a small RNA to induce transgenerational inheritance of learned avoidance\nAuthors: Sengupta, T.; Ange, J. S.; Moore, R. S.; Kaletsky, R. T.; Marogi, J. T.; Myhrvold, C.; Gitai, Z.; Murphy, C. T.\nScore: 18.3, Published: 2023-10-17 DOI: 10.1101/2023.07.20.549962\nPreviously, we discovered that a small RNA from a clinical isolate of Pseudomonas aeruginosa, PA14, induces learned avoidance and its transgenerational inheritance in C. elegans. Pseudomonas aeruginosa is an important human pathogen, and there are other Pseudomonads in C. elegans natural habitat, but it is unclear whether C. elegans ever encounters PA14-like bacteria in the wild. Thus, it is not known if small RNAs from bacteria found in C. elegans natural habitat can also regulate host behavior and produce heritable behavioral effects. Here we found that a pathogenic Pseudomonas vranovensis strain isolated from the C. elegans microbiota, GRb0427, like PA14, regulates worm behavior: worms learn to avoid this pathogenic bacterium following exposure to GRb0427, and this learned avoidance is inherited for four generations. The learned response is entirely mediated by bacterially-produced small RNAs, which induce avoidance and transgenerational inheritance, providing further support that such mechanisms of learning and inheritance exist in the wild. Using bacterial small RNA sequencing, we identified Pv1, a small RNA from GRb0427, that matches the sequence of C. elegans maco-1. We find that Pv1 is both necessary and sufficient to induce learned avoidance of Grb0427. However, Pv1 also results in avoidance of a beneficial microbiome strain, P. mendocina; this potentially maladaptive response may favor reversal of the transgenerational memory after a few generations. Our findings suggest that bacterial small RNA-mediated regulation of host behavior and its transgenerational inheritance are functional in C. elegans natural environment, and that different bacterial small RNA-mediated regulation systems evolved independently but define shared molecular features of bacterial small RNAs that produce transgenerationally-inherited effects.\nAssigning credit where it's due: An information content score to capture the clinical value of Multiplexed Assays of Variant Effect\nAuthors: Ranola, J. M.; Horton, C.; Pesaran, T.; Fayer, S.; Starita, L. M.; Shirts, B. H.\nScore: 3.9, Published: 2023-10-20 DOI: 10.1101/2023.10.20.562794\nBackgroundA variant can be pathogenic or benign with relation to a human disease. Current classification categories from benign to pathogenic reflect a probabilistic summary of current understanding. A primary metric of clinical utility for multiplexed assays of variant effect (MAVE) is the number of variants that can be reclassified from uncertain significance (VUS). However, we hypothesized that this measure of utility underrepresents the information gained from MAVEs and that an information theory approach which includes data that does not reclassify variants will better reflect true information gain. We used this information theory approach to evaluate the information gain, in bits, for MAVEs of BRCA1, PTEN, and TP53. Here, one bit represents the amount of information required to completely classify a single variant starting from no information. ResultsBRCA1 MAVEs produced a total of 831.2 bits of information, 6.58% of the total missense information in BRCA1 and a 22-fold increase over the information that only contributed to VUS reclassification. PTEN MAVEs produced 2059.6 bits of information which represents 32.8% of the total missense information in PTEN and an 85-fold increase over the information that contributed to VUS reclassification. TP53 MAVEs produced 277.8 bits of information which represents 6.22% of the total missense information in TP53 and a 3.5-fold increase over the information that contributed to VUS reclassification. ConclusionsAn information content approach will more accurately portray information gained through MAVE mapping efforts than counting the number of variants reclassified. This information content approach may also help define the impact of modifying information definitions used to classify many variants, such as guideline rule changes.\n",
  "wordCount" : "2693",
  "inLanguage": "en",
  "datePublished": "2023-11-01T10:38:40Z",
  "dateModified": "2023-11-01T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 1, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560808">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560808" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560808">
        <p class="paperTitle">Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560808" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560808" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.</p>
        <p class="info">Score: 37.7, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560808' target='https://doi.org/10.1101/2023.10.04.560808'> 10.1101/2023.10.04.560808</a></p>
        <p class="abstract">Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming. Using neural networks that map DNA sequence to ATAC-seq profiles at base-resolution, we annotated cell-state-specific predictive transcription factor (TF) motif syntax in regulatory elements, inferred affinity- and concentration-dependent dynamics of Tn5-bias corrected TF footprints, linked peaks to putative target genes, and elucidated rewiring of TF- to-gene cis-regulatory networks. Our models reveal that early in reprogramming, OSK, at supraphysiological concentrations, rapidly open transient regulatory elements by occupying non-canonical low-affinity binding sites. As OSK concentration falls, the accessibility of these transient elements decays as a function of motif affinity. We find that these OSK-dependent transient elements sequester the somatic TF AP-1. This redistribution is strongly associated with the silencing of fibroblast-specific genes within individual nuclei. Together, our integrated single-cell resource and models reveal insights into the cis-regulatory code of reprogramming at unprecedented resolution, connect TF stoichiometry and motif syntax to diversification of cell fate trajectories, and provide new perspectives on the dynamics and role of transient regulatory elements in somatic silencing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564036">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564036" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564036">
        <p class="paperTitle">Assessing the impact of 20th century internal migrations on the genetic structure of Estonia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564036" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564036" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kuznetsov, I. A.; Estonian Biobank Research Team,  ; Metspalu, M.; Vainik, U.; Pagani, L.; Montinaro, F.; Pankratov, V.</p>
        <p class="info">Score: 21.6, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564036' target='https://doi.org/10.1101/2023.10.25.564036'> 10.1101/2023.10.25.564036</a></p>
        <p class="abstract">Spatial genetic structure observed in many human populations is in large part attributed to past demographic events and isolation by distance. However, how intensifying migration affects this structure remains understudied. Here we harness a sample of more than 180 thousand individuals to explore the genetic correlates and consequences of contemporary migrations in Estonia. While we show that migration smoothens the genome-wide genetic structure, it intensifies inter-regional differences in polygenic scores (PGS) for certain traits, derived both from population as well as within-sibship studies. The strongest effect is observed for educational attainment which is consistent with previous observations in the UK and suggests this to be a general pattern. We explore those regional differences in PGS in terms of the driving forces behind them and from a temporal perspective, and suggest urbanisation as a major driver for this pattern in Estonia from at least the first half of the 20th century.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.562294">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.562294" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.562294">
        <p class="paperTitle">AlphaMissense is better correlated with functional assays of missense impact than earlier prediction algorithms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.562294" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.562294" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ljungdahl, A.; Kohani, S.; Page, N. F.; Wells, E. S.; Wigdor, E. M.; Dong, S.; Sanders, S. J.</p>
        <p class="info">Score: 14.2, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.562294' target='https://doi.org/10.1101/2023.10.24.562294'> 10.1101/2023.10.24.562294</a></p>
        <p class="abstract">Missense variants that alter a single amino acid in the encoded protein contribute to many human disorders but pose a substantial challenge in interpretation. Though these variants can be reliably identified through sequencing, distinguishing the clinically significant ones remains difficult, such that &#34;Variants of Unknown Significance&#34; outnumber those classified as &#34;Pathogenic&#34; or &#34;Likely Pathogenic.&#34; Numerous in silico approaches have been developed to predict the functional impact of missense variants to inform clinical interpretation, the latest being AlphaMissense, which uses artificial intelligence methods trained on predicted protein structure. To independently assess the performance of AlphaMissense and 38 other predictors of missense severity, we compared predictions to data from multiplexed assays of variant effect (MAVE). MAVE experiments generate almost every possible individual amino acid change in a gene and measure their functional impact using a high-throughput assay. Assessing 17,696 variants across five genes (DDX3X, MSH2, PTEN, KCNQ4, and BRCA1), we find that AlphaMissense is consistently one of the top five algorithms based on correlation with functional impact and is the best-correlated algorithm for two genes. We conclude that AlphaMissense represents the current best-in-class predictor by this metric; however, the improvement over other algorithms is modest. We note that multiple missense predictors, including AlphaMissense, appear to overcall variants as pathogenic despite minimal functional impact and that substantially more high-quality training data, including consistently analyzed patient cohorts and MAVE analyses, are required to improve accuracy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.21.562237">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.21.562237" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.21.562237">
        <p class="paperTitle">Genotype prediction of 336,463 samples from public expression data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.21.562237" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.21.562237" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Razi, A.; Lo, C. C.; Wang, S.; Leek, J. T.; Hansen, K. D.</p>
        <p class="info">Score: 15.5, Published: 2023-10-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.21.562237' target='https://doi.org/10.1101/2023.10.21.562237'> 10.1101/2023.10.21.562237</a></p>
        <p class="abstract">Tens of thousands of RNA-sequencing experiments comprising hundreds of thousands of individual samples have now been performed. These data represent a broad range of experimental conditions, sequencing technologies, and hypotheses under study. The Recount project has aggregated and uniformly processed hundreds of thousands of publicly available RNA-seq samples. Most of these samples only include RNA expression measurements; genotype data for these same samples would enable a wide range of analyses including variant prioritization, eQTL analysis, and studies of allele specific expression. Here, we developed a statistical model based on the existing reference and alternative read counts from the RNA-seq experiments available through Recount3 to predict genotypes at autosomal biallelic loci in coding regions. We demonstrate the accuracy of our model using large-scale studies that measured both gene expression and genotype genome-wide. We show that our predictive model is highly accurate with 99.5% overall accuracy, 99.6% major allele accuracy, and 90.4% minor allele accuracy. Our model is robust to tissue and study effects, provided the coverage is high enough. We applied this model to genotype all the samples in Recount3 and provide the largest ready-to-use expression repository containing genotype information. We illustrate that the predicted genotype from RNA-seq data is sufficient to unravel the underlying population structure of samples in Recount3 using Principal Component Analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564319">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564319" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564319">
        <p class="paperTitle">Trait selection strategy in multi-trait GWAS: Boosting SNPs discoverability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564319" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564319" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suzuki, Y.; Menager, H.; Brancotte, B.; Vernet, R.; Nerin, C.; Boetto, C.; Auvergne, A.; Linhard, C.; Torchet, R.; Lechat, P.; Troubat, L.; Cho, M. H.; Bouzigon, E.; Aschard, H.; Julienne, H.</p>
        <p class="info">Score: 9.1, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564319' target='https://doi.org/10.1101/2023.10.27.564319'> 10.1101/2023.10.27.564319</a></p>
        <p class="abstract">Since the first Genome-Wide Association Studies (GWAS), thousands of variant-trait associations have been discovered. However, the sample size required to detect additional variants using standard univariate association screening is increasingly prohibitive. Multi-trait GWAS offers a relevant alternative: it can improve statistical power and lead to new insights about gene function and the joint genetic architecture of human phenotypes. Although many methodological hurdles of multi-trait testing have been discussed, the strategy to select trait, among overwhelming possibilities, has been overlooked. In this study, we conducted extensive multi-trait tests using JASS (Joint Analysis of Summary Statistics) and assessed which genetic features of the analysed sets were associated with an increased detection of variants as compared to univariate screening. Our analyses identified multiple factors associated with the gain in the association detection in multi-trait tests. Together, these factors of the analysed sets are predictive of the gain of the multi-trait test (Pearson&#39;s rho equal to 0.43 between the observed and predicted gain, P &lt; 1.6 x 10-60). Applying an alternative multi-trait approach (MTAG, multi-trait analysis of GWAS), we found that in most scenarios but particularly those with larger numbers of traits, JASS outperformed MTAG. Finally, we benchmark several strategies to select set of traits including the prevalent strategy of selecting clinically similar traits, which systematically underperformed selecting clinically heterogenous traits or selecting sets that issued from our data-driven models. This work provides a unique picture of the determinant of multi-trait GWAS statistical power and outline practical strategies for multi-trait testing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564441">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564441" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564441">
        <p class="paperTitle">Drosophila models of PIGA-CDG mirror patient phenotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564441" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564441" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thorpe, H. J.; Owings, K. G.; Aziz, M. C.; Haller, M.; Coelho, E.; Chow, C. Y.</p>
        <p class="info">Score: 6.2, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564441' target='https://doi.org/10.1101/2023.10.27.564441'> 10.1101/2023.10.27.564441</a></p>
        <p class="abstract">Mutations in the phosphatidylinositol glycan biosynthesis class A (PIGA) gene cause a rare, X-linked recessive congenital disorder of glycosylation (CDG). PIGA-CDG is characterized by seizures, intellectual and developmental delay, and congenital malformations. The PIGA gene encodes an enzyme involved in the first step of GPI anchor biosynthesis. There are over 100 GPI anchored proteins that attach to the cell surface and are involved in cell signaling, immunity, and adhesion. Little is known about the pathophysiology of PIGA-CDG. Here we describe the first Drosophila model of PIGA-CDG and demonstrate that loss of PIG-A function in Drosophila accurately models the human disease. As expected, complete loss of PIG-A function is larval lethal. Heterozygous null animals appear healthy, but when challenged, have a seizure phenotype similar to what is observed in patients. To identify the cell-type specific contributions to disease, we generated neuron- and glia-specific knockdown of PIG-A. Neuron-specific knockdown resulted in reduced lifespan and a number of neurological phenotypes, but no seizure phenotype. Glia-knockdown also reduced lifespan and, notably, resulted in a very strong seizure phenotype. RNAseq analyses demonstrated that there are fundamentally different molecular processes that are disrupted when PIG-A function is eliminated in different cell types. In particular, loss of PIG-A in neurons resulted in upregulation of glycolysis, but loss of PIG-A in glia resulted in upregulation of protein translation machinery. Here we demonstrate that Drosophila is a good model of PIGA-CDG and provide new data resources for future study of PIGA-CDG and other GPI anchor disorders.

Article SummaryPIGA-CDG is a rare genetic disorder. In order to study this rare disease, we generated and characterized several Drosophila models of PIGA-CDG. These models faithfully recapitulate different patient phenotypes, including movement disorder and seizures. Drosophila is a good model for PIGA-CDG and other GPI anchor disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562293" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562293">
        <p class="paperTitle">Large-scale causal discovery using interventional data sheds light on the regulatory network architecture of blood traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brown, B. C.; Morris, J. A.; Lappalainen, T.; Knowles, D. A.</p>
        <p class="info">Score: 6.2, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562293' target='https://doi.org/10.1101/2023.10.13.562293'> 10.1101/2023.10.13.562293</a></p>
        <p class="abstract">Inference of directed biological networks is an important but notoriously challenging problem. We introduce inverse sparse regression (inspre), an approach to learning causal networks that leverages large-scale intervention-response data. Applied to 788 genes from the genome-wide perturb-seq dataset, inspre helps elucidate the network architecture of blood traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.561051">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.561051" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.561051">
        <p class="paperTitle">A comparative approach for selecting orthologous candidate genes underlying signal in genome-wide association studies across multiple species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.561051" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.561051" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Whitt, L.; Mahood, E. H.; Ziegler, G.; Luebbert, C.; Gillman, J. D.; Norton, G. J.; Price, A. H.; Salt, D. E.; Dilkes, B.; Baxter, I.</p>
        <p class="info">Score: 7.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.561051' target='https://doi.org/10.1101/2023.10.05.561051'> 10.1101/2023.10.05.561051</a></p>
        <p class="abstract">Advances in quantitative genetics have enabled researchers to identify genomic regions associated with changes in phenotype. However, genomic regions can contain hundreds to thousands of genes, and progressing from genomic regions to candidate genes is still challenging. In genome-wide association studies (GWAS) measuring elemental accumulation (ionomic) traits, a mere 5% of loci are associated with a known ionomic gene - indicating that many causal genes are still unknown. To select candidates for the remaining 95% of loci, we developed a method to identify conserved genes underlying GWAS loci in multiple species. For 19 ionomic traits, we identified 14,336 candidates across Arabidopsis, soybean, rice, maize, and sorghum. We calculated the likelihood of candidates with random permutations of the data and determined that most of the top 10% of candidates were orthologous genes linked to GWAS loci across all five species. The candidate list also includes orthologous genes with previously established ionomic functions in Arabidopsis and rice. Our methods highlight the conserved nature of ionomic genetic regulators and enable the identification of previously unknown ionomic genes.

Author summaryIdentifying the genes contributing to changes in a given trait is challenging. Many genes can be near the region of interest, but proximity does not always translate to causality. We use other methods to narrow our focus to the genes most likely involved in our trait of interest before confirming their involvement through confirmation experiments. However, these other methods are often time, labor, and resource-consuming. We developed an approach to narrow these gene lists before these laborious methods are required. Through testing with genetic markers for elemental (i.e., calcium, iron, zinc) uptake, we found that comparing markers across multiple species for nearby evolutionarily conserved genes is a successful approach. Weve produced a list of candidate genes likely to be involved in elemental uptake traits, including previously known elemental uptake genes and genes whose potential elemental uptake function has yet to be observed. Some of these genes would not have been considered according to the significance threshold within a single GWAS, but combining comparable datasets across species has collectively boosted their signal. Methods like our approach are useful for reducing candidate lists to conserve resources spent in functional characterization experiments and encouraging the discovery of new functional roles of these genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549962" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549962">
        <p class="paperTitle">A natural bacterial pathogen of C. elegans uses a small RNA to induce transgenerational inheritance of learned avoidance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sengupta, T.; Ange, J. S.; Moore, R. S.; Kaletsky, R. T.; Marogi, J. T.; Myhrvold, C.; Gitai, Z.; Murphy, C. T.</p>
        <p class="info">Score: 18.3, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549962' target='https://doi.org/10.1101/2023.07.20.549962'> 10.1101/2023.07.20.549962</a></p>
        <p class="abstract">Previously, we discovered that a small RNA from a clinical isolate of Pseudomonas aeruginosa, PA14, induces learned avoidance and its transgenerational inheritance in C. elegans. Pseudomonas aeruginosa is an important human pathogen, and there are other Pseudomonads in C. elegans natural habitat, but it is unclear whether C. elegans ever encounters PA14-like bacteria in the wild. Thus, it is not known if small RNAs from bacteria found in C. elegans natural habitat can also regulate host behavior and produce heritable behavioral effects. Here we found that a pathogenic Pseudomonas vranovensis strain isolated from the C. elegans microbiota, GRb0427, like PA14, regulates worm behavior: worms learn to avoid this pathogenic bacterium following exposure to GRb0427, and this learned avoidance is inherited for four generations. The learned response is entirely mediated by bacterially-produced small RNAs, which induce avoidance and transgenerational inheritance, providing further support that such mechanisms of learning and inheritance exist in the wild. Using bacterial small RNA sequencing, we identified Pv1, a small RNA from GRb0427, that matches the sequence of C. elegans maco-1. We find that Pv1 is both necessary and sufficient to induce learned avoidance of Grb0427. However, Pv1 also results in avoidance of a beneficial microbiome strain, P. mendocina; this potentially maladaptive response may favor reversal of the transgenerational memory after a few generations. Our findings suggest that bacterial small RNA-mediated regulation of host behavior and its transgenerational inheritance are functional in C. elegans natural environment, and that different bacterial small RNA-mediated regulation systems evolved independently but define shared molecular features of bacterial small RNAs that produce transgenerationally-inherited effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.562794">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.562794" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.562794">
        <p class="paperTitle">Assigning credit where it&#39;s due: An information content score to capture the clinical value of Multiplexed Assays of Variant Effect</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.562794" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.562794" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ranola, J. M.; Horton, C.; Pesaran, T.; Fayer, S.; Starita, L. M.; Shirts, B. H.</p>
        <p class="info">Score: 3.9, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.562794' target='https://doi.org/10.1101/2023.10.20.562794'> 10.1101/2023.10.20.562794</a></p>
        <p class="abstract">BackgroundA variant can be pathogenic or benign with relation to a human disease. Current classification categories from benign to pathogenic reflect a probabilistic summary of current understanding. A primary metric of clinical utility for multiplexed assays of variant effect (MAVE) is the number of variants that can be reclassified from uncertain significance (VUS). However, we hypothesized that this measure of utility underrepresents the information gained from MAVEs and that an information theory approach which includes data that does not reclassify variants will better reflect true information gain. We used this information theory approach to evaluate the information gain, in bits, for MAVEs of BRCA1, PTEN, and TP53. Here, one bit represents the amount of information required to completely classify a single variant starting from no information.

ResultsBRCA1 MAVEs produced a total of 831.2 bits of information, 6.58% of the total missense information in BRCA1 and a 22-fold increase over the information that only contributed to VUS reclassification. PTEN MAVEs produced 2059.6 bits of information which represents 32.8% of the total missense information in PTEN and an 85-fold increase over the information that contributed to VUS reclassification. TP53 MAVEs produced 277.8 bits of information which represents 6.22% of the total missense information in TP53 and a 3.5-fold increase over the information that contributed to VUS reclassification.

ConclusionsAn information content approach will more accurately portray information gained through MAVE mapping efforts than counting the number of variants reclassified. This information content approach may also help define the impact of modifying information definitions used to classify many variants, such as guideline rule changes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
