<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="isGWAS: ultra-high-throughput, scalable and equitable inference of genetic associations with disease
Authors: Foley, C. N.; Kuncheva, Z.; Marioni, R.; Runz, H.; Sun, B.
Score: 39.2, Published: 2023-07-27 DOI: 10.1101/2023.07.21.550074
Genome-wide association studies (GWAS) have proven a powerful tool for human geneticists to generate biological insights or hypotheses for drug discovery. Nevertheless, a dependency on sensitive individual-level data together with ever-increasing cohort sample sizes, numbers of variants and phenotypes studied put a strain on existing algorithms, limiting the GWAS approach from maximising potential.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="isGWAS: ultra-high-throughput, scalable and equitable inference of genetic associations with disease
Authors: Foley, C. N.; Kuncheva, Z.; Marioni, R.; Runz, H.; Sun, B.
Score: 39.2, Published: 2023-07-27 DOI: 10.1101/2023.07.21.550074
Genome-wide association studies (GWAS) have proven a powerful tool for human geneticists to generate biological insights or hypotheses for drug discovery. Nevertheless, a dependency on sensitive individual-level data together with ever-increasing cohort sample sizes, numbers of variants and phenotypes studied put a strain on existing algorithms, limiting the GWAS approach from maximising potential." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-02T10:38:09+00:00" />
<meta property="article:modified_time" content="2023-08-02T10:38:09+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="isGWAS: ultra-high-throughput, scalable and equitable inference of genetic associations with disease
Authors: Foley, C. N.; Kuncheva, Z.; Marioni, R.; Runz, H.; Sun, B.
Score: 39.2, Published: 2023-07-27 DOI: 10.1101/2023.07.21.550074
Genome-wide association studies (GWAS) have proven a powerful tool for human geneticists to generate biological insights or hypotheses for drug discovery. Nevertheless, a dependency on sensitive individual-level data together with ever-increasing cohort sample sizes, numbers of variants and phenotypes studied put a strain on existing algorithms, limiting the GWAS approach from maximising potential."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "isGWAS: ultra-high-throughput, scalable and equitable inference of genetic associations with disease\nAuthors: Foley, C. N.; Kuncheva, Z.; Marioni, R.; Runz, H.; Sun, B.\nScore: 39.2, Published: 2023-07-27 DOI: 10.1101/2023.07.21.550074\nGenome-wide association studies (GWAS) have proven a powerful tool for human geneticists to generate biological insights or hypotheses for drug discovery. Nevertheless, a dependency on sensitive individual-level data together with ever-increasing cohort sample sizes, numbers of variants and phenotypes studied put a strain on existing algorithms, limiting the GWAS approach from maximising potential.",
  "keywords": [
    
  ],
  "articleBody": " isGWAS: ultra-high-throughput, scalable and equitable inference of genetic associations with disease\nAuthors: Foley, C. N.; Kuncheva, Z.; Marioni, R.; Runz, H.; Sun, B.\nScore: 39.2, Published: 2023-07-27 DOI: 10.1101/2023.07.21.550074\nGenome-wide association studies (GWAS) have proven a powerful tool for human geneticists to generate biological insights or hypotheses for drug discovery. Nevertheless, a dependency on sensitive individual-level data together with ever-increasing cohort sample sizes, numbers of variants and phenotypes studied put a strain on existing algorithms, limiting the GWAS approach from maximising potential. Here we present in-silico GWAS (isGWAS), a uniquely scalable algorithm to infer regression parameters in case-control GWAS from cohort-level summary data. For any sample size, isGWAS computes a variant-disease association parameter in [~]1 millisecond, or [~]11m variants in UK-Biobank within [~]4 minutes ([~]1500-fold faster than state-of-the-art). Extensive simulations and empirical tests demonstrate that isGWAS results are highly comparable to traditional regression-based approaches. We further introduce a heuristic re-sampling algorithm, leapfrog re-sampler (LRS), to extrapolate association results to semi-virtually enlarged cohorts. Owing to significant computational gains we anticipate a broad use of isGWAS and LRS which are customizable on a web interface.\nHuman Plasma Proteomic Profile of Clonal Hematopoiesis\nAuthors: Yu, Z.; Vromman, A.; Nguyen, N. Q. H.; Vellarikkal, S. K.; Uddin, M. M.; Niroula, A.; Griffin, G.; Honigberg, M. C.; Lin, A. E.; Gibson, C. J.; Katz, D. H.; Tahir, U.; Haidermota, S.; Ganesh, S.; Antoine, T.; Weinstock, J.; Austin, T. R.; Vasan, R. S.; peloso, G. M.; Hornsby, W.; Ganz, P.; Manson, J. E.; Haring, B.; Kooperberg, C. L.; Reiner, A. P.; Bis, J. C.; Psaty, B. M.; Min, Y.-I.; Correa, A.; Lange, L. A.; Post, W.; Rotter, J. I.; Rich, S. S.; Wilson, J. G.; Ebert, B. L.; Yu, B.; Ballantyne, C. M.; Coresh, J.; Sankaran, V. G.; Bick, A. G.; Jaiswal, S.; Libby, P.; Gerszten, R. E.; NHLBI T\nScore: 12.2, Published: 2023-07-27 DOI: 10.1101/2023.07.25.550557\nPlasma proteomic profiles associated with subclinical somatic mutations in blood cells may offer novel insights in downstream clinical consequences. Here, we explore such patterns in clonal hematopoiesis of indeterminate potential (CHIP), which links to several cancer and non-cancer outcomes, including coronary artery disease. Among 12,911 ancestrally diverse participants (682 with CHIP) from NHLBI TOPMed with blood-based DNA sequencing and 1,148 common proteins measured by SomaScan, we identified 32 unique proteins associated with the most prevalent driver genes (DNMT3A, TET2, and ASXL1) after multiple testing corrections. These associations showed substantial heterogeneity by driver genes, sex, and race, were enriched for immune response and inflammation pathways, and were moderately replicated in UK Biobank (N=48,922) that used Olink for proteomics measurement. Murine single-cell RNA-sequencing data from aortic arch cells, inclusive of resident hematologic cells, in mice with Tet2-/-bone marrow and wild-type mice revealed corroborating differential expression of TET2-associated protein-encoding genes. Lastly, we apply these observations to identify 68 plasma proteins shared between CHIP and coronary artery disease.\nPathogenic CANVAS (AAGGG)n repeats stall DNA replication due to the formation of alternative DNA structures\nAuthors: Hisey, J. A.; Radchenko, E. A.; Ceschi, S.; Rastokina, A.; Mandel, N. H.; McGinty, R. J.; Matos-Rodrigues, G.; Hernandez, A.; Nussenzweig, A.; Mirkin, S. M.\nScore: 10.0, Published: 2023-07-26 DOI: 10.1101/2023.07.25.550509\nCANVAS is a recently characterized repeat expansion disease, most commonly caused by homozygous expansions of an intronic (A2G3)n repeat in the RFC1 gene. There are a multitude of repeat motifs found in the human population at this locus, some of which are pathogenic and others benign. In this study, we conducted structure-functional analyses of the main pathogenic (A2G3)n and the main nonpathogenic (A4G)n repeats. We found that the pathogenic, but not the nonpathogenic, repeat presents a potent, orientation-dependent impediment to DNA polymerization in vitro. The pattern of the polymerization blockage is consistent with triplex or quadruplex formation in the presence of magnesium or potassium ions, respectively. Chemical probing of both repeats in supercoiled DNA reveals triplex H-DNA formation by the pathogenic repeat. Consistently, bioinformatic analysis of the S1-END-seq data from human cell lines shows preferential H-DNA formation genome-wide by (A2G3)n motifs over (A4G)n motifs in vivo. Finally, the pathogenic, but not the non-pathogenic, repeat stalls replication fork progression in yeast and human cells. We hypothesize that CANVAS-causing (A2G3)n repeat represents a challenge to genome stability by folding into alternative DNA structures that stall DNA replication.\nThe influence of HLA genetic variation on plasma protein expression\nAuthors: Krishna, C.; Chiou, J.; Lee, I.; Kim, H. I.; Atalar Aksit, M.; Sakaue, S.; Von Schack, D.; Raychaudhuri, S.; Ziemek, D.; Hu, X.\nScore: 9.6, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550394\nPolymorphism in the human leukocyte antigen class I (HLA-I) and class II (HLA-II) genes is strongly implicated in susceptibility to immune-mediated diseases. However, the molecular effects of HLA genetic variation, including and beyond antigen presentation, remain unclear. Here we examined the effect of HLA genetic variation on the expression of 2940 plasma proteins using imputed HLA variants in 45,330 Europeans in the UK Biobank. We detected 504 proteins (17.1% of all proteins tested) affected by HLA genetic variation (HLA-pQTL), including widespread trans regulation of protein expression by autoimmune disease risk alleles. HLA-pQTL were enriched in gene families related to antigen presentation (e.g. B2M), T cell fate (CD8A; CD4), chemokines (CCL19; CCL21), and NK and macrophage receptors (KIR; LILRA/B), suggesting that HLA polymorphism affects both adaptive and innate immunity. HLA-pQTL also affected expression of diverse proteins with unclear roles in the immune response (e.g. SFTPD, LRPAP1, ENPP6, NPTX1), as well as drug targets for immune-mediated diseases, suggesting complex regulatory roles of the HLA loci. Among trans HLA-pQTL, HLA variants explained 0.1-42.9% of the protein expression variance. Fine-mapping revealed that most HLA-pQTL implicated amino acid positions within the peptide binding groove, suggesting that trans regulation of plasma protein expression by the HLA loci is primarily a consequence of antigen presentation. We also show that HLA-I and II uniquely affect different proteins and biological mechanisms. Altogether, our data reveal the effects of HLA genetic variation on protein expression and aid the interpretation of associations between HLA alleles and immune-mediated diseases.\nEpigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization\nAuthors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.\nScore: 399.3, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478\nAging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging. Epigenetic clocks have been shown to be valuable tools for measuring biological age in a variety of tissues and samples. As such, these epigenetic clocks can determine accelerated biological aging and time-to-mortality across various tissues. Previous reports have shown accelerated biological aging and telomere attrition acceleration following SARS-CoV-2 infection. However, the effect of accelerated epigenetic aging on outcome (death/recovery) in COVID-19 patients with Acute Respiratory Distress Syndrome (ARDS) has not been well investigated. In this study, we measured DNA methylation age and telomere attrition in 87 severe COVID-19 cases with ARDS under mechanical ventilation. Furthermore, we compared dynamic changes in epigenetic aging across multiples time-points until recovery or death. Epigenetic age was measured using the Horvath, Hannum, DNAm skin and blood, GrimAge, and PhenoAge clocks, whereas telomere length was calculated using the surrogate marker DNAmTL. Our analysis revealed significant accelerated epigenetic aging but no telomere attrition acceleration in severe COVID-19 cases. In addition, we observed epigenetic age deceleration at inclusion vs end of follow-up in recovered but not in deceased COVID-19 cases using certain clocks. When comparing dynamic changes in epigenetic age acceleration (EAA), we detected higher EAA using both the Horvath and PhenoAge clocks in deceased vs recovered patients. The DNAmTL measurements revealed telomere attrition acceleration in deceased COVID19 patients between inclusion and end of follow-up as well as a significant change in dynamic telomere attrition acceleration when comparing patients who recovered vs those who died. In conclusion, EAA and telomere attrition acceleration was associated with treatment outcome in hospitalized COVID-19 Patients with ARDS. A better understanding of the long-term effects of EAA in COVID19 patients and how they might contribute to Long COVID symptoms in recovered individuals is urgently needed.\nMitochondrial Genome Diversity across the Subphylum Saccharomycotina\nAuthors: Wolters, J. F.; LaBella, A.; Opulente, D.; Rokas, A.; Hittinger, C. T.\nScore: 7.1, Published: 2023-07-31 DOI: 10.1101/2023.07.28.551029\nEukaryotic life depends on the functional elements encoded by both the nuclear genome and organellar genomes, such as those contained within the mitochondria. The content, size, and structure of the mitochondrial genome varies across organisms with potentially large implications for phenotypic variance and resulting evolutionary trajectories. Among yeasts in the subphylum Saccharomycotina, extensive differences have been observed in various species relative to the model yeast Saccharomyces cerevisiae, but mitochondrial genome sampling across many groups has been scarce, even as hundreds of nuclear genomes have become available. By extracting mitochondrial assemblies from existing short-read genome sequence datasets, we have greatly expanded both the number of available genomes and the coverage across sparsely sampled clades. Comparison of 353 yeast mitochondrial genomes revealed that, while size and GC content were fairly consistent across species, those in the genera Metschnikowia and Saccharomyces trended larger, while several species in the order Saccharomycetales, which includes S. cerevisiae, exhibited lower GC content. Extreme examples for both size and GC content were scattered throughout the subphylum. All mitochondrial genomes shared a core set of protein-coding genes for Complexes III, IV, and V, but they varied in the presence or absence of mitochondrially-encoded canonical Complex I genes. We traced the loss of Complex I genes to a major event in the ancestor of the orders Saccharomycetales and Saccharomycodales, but we also observed several independent losses in the orders Phaffomycetales, Pichiales, and Dipodascales. In contrast to prior hypotheses based on smaller-scale datasets, comparison of evolutionary rates in protein-coding genes showed no bias towards elevated rates among aerobically fermenting (Crabtree/Warburg-positive) yeasts. Mitochondrial introns were widely distributed, but they were highly enriched in some groups. The majority of mitochondrial introns were poorly conserved within groups, but several were shared within groups, between groups, and even across taxonomic orders, which is consistent with horizontal gene transfer, likely involving homing endonucleases acting as selfish elements. As the number of available fungal nuclear genomes continues to expand, the methods described here to retrieve mitochondrial genome sequences from these datasets will prove invaluable to ensuring that studies of fungal mitochondrial genomes keep pace with their nuclear counterparts.\nGenetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions\nAuthors: Suhre, K.\nScore: 15.3, Published: 2023-07-20 DOI: 10.1101/2023.07.19.549734\nProtein quantitative trait loci (pQTLs) are an invaluable source of information for drug target development as they provide genetic evidence to support protein function, suggest relationships between cis- and trans-associated proteins, and link proteins to disease where they collocate with genetic risk loci for clinical endpoints. Using the recently released Olink proteomics data for 1,463 proteins measured in over 54,000 samples of the UK Biobank we identified and replicated 4,248 associations with 2,821 ratios between protein levels (rQTLs) where the strengths of association at known pQTL loci increased by up to several hundred orders of magnitude. We attribute this increase in statistical power (p-gain) to accounting for genetic and non-genetic variance shared by the two proteins in the ratio pair. Protein pairs with a significant p-gain were 7.6-fold enriched in known protein-protein interactions, suggesting that their ratios reflect biological links between the implicated proteins. We then conducted a GWAS on the 2,821 ratios and identified 2,527 novel rQTLs, increasing the number of discovered genetic signals compared to the original protein-only GWAS by 24.7%. At examples we demonstrate that this approach can identify novel loci of clinical relevance, support causal gene identification, and reveal complex networks of interacting proteins. Taken together, our study adds significant value to the genetic insights that can be derived from the UKB proteomics data and motivates the wider use of ratios in large scale GWAS.\nPax3 lineage-specific deletion of Gpr161 is associated with spinal neural tube and craniofacial malformations during embryonic development.\nAuthors: Kim, S. E.; Chothani, P.; Shaik, R.; Pollard, W.; Finnell, R. H.\nScore: 8.7, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548129\nShh signaling is the morphogen signaling that regulates embryonic craniofacial and neural tube development. G protein-coupled receptor 161 (Gpr161) is a negative regulator of Shh signaling, and its inactivation in mice results in embryo lethality with craniofacial and neural tube defects (NTDs). However, the structural defects of later embryonic stages in Gpr161 null mice and cell lineages underlying abnormalities were not well characterized due to their limited lifespan. We found the Pax3 lineage-specific deletion of Gpr161 in mice presented with tectal hypertrophy (anterior dorsal neuroepithelium), cranial vault and facial bone hypoplasia (cranial neural crest (CNC)), vertebral abnormalities (somite), and the closed form of spina bifida (posterior dorsal neuroepithelium). In particular, the closed form of spina bifida is partly due to the reduced Pax3 and Cdx4 gene expression of the posterior dorsal neural tubes of Gpr161 mutant embryos involving decreased Wnt signaling whereas Shh signaling was increased. This study provides the novel role of Gpr161 in the posterior neural tube development and confirms its role on CNC- and somite-derived skeletogenesis and midbrain morphogenesis in mice.\nCalcineurin contributes to RNAi-mediated transgene silencing and small interfering RNA production in the human fungal pathogen Cryptococcus neoformans\nAuthors: Yadav, V.; Mohan, R.; Sun, S.; Heitman, J.\nScore: 5.1, Published: 2023-07-26 DOI: 10.1101/2023.07.25.550548\nAdaptation to external environmental challenges at the cellular level requires rapid responses and involves relay of information to the nucleus to drive key gene expression changes through downstream transcription factors. Here, we describe an alternative route of adaptation through a direct role for cellular signaling components in governing gene expression via RNA interference-mediated small RNA production. Calcium-calcineurin signaling is a highly conserved signaling cascade that plays central roles in stress adaptation and virulence of eukaryotic pathogens, including the human fungal pathogen Cryptococcus neoformans. Upon activation in C. neoformans, calcineurin localizes to P-bodies, membrane-less organelles that are also the site for RNA processing. Here, we studied the role of calcineurin and its substrates in RNAi-mediated transgene silencing. Our results reveal that calcineurin regulates both the onset and the reversion of transgene silencing. We found that some calcineurin substrates that localize to P-bodies also regulate transgene silencing but in opposing directions. Small RNA sequencing in mutants lacking calcineurin or its targets revealed a role for calcineurin in small RNA production. Interestingly, the impact of calcineurin and its substrates was found to be different in genome-wide analysis, suggesting that calcineurin may regulate small RNA production in C. neoformans through additional pathways. Overall, these findings define a mechanism by which signaling machinery induced by external stimuli can directly alter gene expression to accelerate adaptative responses and contribute to genome defense.\nA small RNA from a natural bacterial pathogen of C. elegans induces transgenerational inheritance of learned avoidance\nAuthors: Sengupta, T.; Ange, J. S.; Moore, R. S.; Kaletsky, R. T.; Marogi, J. T.; Myhrvold, C.; Gitai, Z.; Murphy, C. T.\nScore: 15.8, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549962\nA small RNA from a clinical isolate of Pseudomonas aeruginosa, PA14, induces learned avoidance and its transgenerational inheritance in C. elegans. However, it is not known if small RNAs from bacteria found in C. elegans natural habitat can regulate host behavior and produce heritable behavioral effects. Here we found that GRb0427, a pathogenic Pseudomonas vranovensis strain isolated from the C. elegans microbiota, similarly affects worm behavior: worms learn to avoid this pathogenic bacterium following exposure, and this learned avoidance is inherited for four generations. The learned response is entirely mediated by small RNAs, which induce avoidance and transgenerational inheritance in both the laboratory N2 strain and a natural isolate of C. elegans, JU1580, providing further support that such mechanisms of learning and inheritance exist in the wild. A small RNA, Pv1, matches the sequence of C. elegans maco-1, and is both necessary and sufficient to induce learned avoidance in worms. However, Pv1 also results in avoidance of a beneficial microbiome strain, P. mendocina; this maladaptive response may favor reversal of the transgenerational memory after a few generations. Our findings suggest that bacterial small RNA-mediated regulation of host behavior and its transgenerational inheritance are functional in C. elegans natural environment, and that different bacterial small RNA-mediated regulation systems evolved independently but define shared molecular features of bacterial small RNAs that produce transgenerationally-inherited effects.\n",
  "wordCount" : "2719",
  "inLanguage": "en",
  "datePublished": "2023-08-02T10:38:09Z",
  "dateModified": "2023-08-02T10:38:09Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on August 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550074">
        <p class="paperTitle">isGWAS: ultra-high-throughput, scalable and equitable inference of genetic associations with disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Foley, C. N.; Kuncheva, Z.; Marioni, R.; Runz, H.; Sun, B.</p>
        <p class="info">Score: 39.2, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550074' target='https://doi.org/10.1101/2023.07.21.550074'> 10.1101/2023.07.21.550074</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have proven a powerful tool for human geneticists to generate biological insights or hypotheses for drug discovery. Nevertheless, a dependency on sensitive individual-level data together with ever-increasing cohort sample sizes, numbers of variants and phenotypes studied put a strain on existing algorithms, limiting the GWAS approach from maximising potential. Here we present in-silico GWAS (isGWAS), a uniquely scalable algorithm to infer regression parameters in case-control GWAS from cohort-level summary data. For any sample size, isGWAS computes a variant-disease association parameter in [~]1 millisecond, or [~]11m variants in UK-Biobank within [~]4 minutes ([~]1500-fold faster than state-of-the-art). Extensive simulations and empirical tests demonstrate that isGWAS results are highly comparable to traditional regression-based approaches. We further introduce a heuristic re-sampling algorithm, leapfrog re-sampler (LRS), to extrapolate association results to semi-virtually enlarged cohorts. Owing to significant computational gains we anticipate a broad use of isGWAS and LRS which are customizable on a web interface.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550557">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550557" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550557">
        <p class="paperTitle">Human Plasma Proteomic Profile of Clonal Hematopoiesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550557" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550557" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, Z.; Vromman, A.; Nguyen, N. Q. H.; Vellarikkal, S. K.; Uddin, M. M.; Niroula, A.; Griffin, G.; Honigberg, M. C.; Lin, A. E.; Gibson, C. J.; Katz, D. H.; Tahir, U.; Haidermota, S.; Ganesh, S.; Antoine, T.; Weinstock, J.; Austin, T. R.; Vasan, R. S.; peloso, G. M.; Hornsby, W.; Ganz, P.; Manson, J. E.; Haring, B.; Kooperberg, C. L.; Reiner, A. P.; Bis, J. C.; Psaty, B. M.; Min, Y.-I.; Correa, A.; Lange, L. A.; Post, W.; Rotter, J. I.; Rich, S. S.; Wilson, J. G.; Ebert, B. L.; Yu, B.; Ballantyne, C. M.; Coresh, J.; Sankaran, V. G.; Bick, A. G.; Jaiswal, S.; Libby, P.; Gerszten, R. E.; NHLBI T</p>
        <p class="info">Score: 12.2, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550557' target='https://doi.org/10.1101/2023.07.25.550557'> 10.1101/2023.07.25.550557</a></p>
        <p class="abstract">Plasma proteomic profiles associated with subclinical somatic mutations in blood cells may offer novel insights in downstream clinical consequences. Here, we explore such patterns in clonal hematopoiesis of indeterminate potential (CHIP), which links to several cancer and non-cancer outcomes, including coronary artery disease. Among 12,911 ancestrally diverse participants (682 with CHIP) from NHLBI TOPMed with blood-based DNA sequencing and 1,148 common proteins measured by SomaScan, we identified 32 unique proteins associated with the most prevalent driver genes (DNMT3A, TET2, and ASXL1) after multiple testing corrections. These associations showed substantial heterogeneity by driver genes, sex, and race, were enriched for immune response and inflammation pathways, and were moderately replicated in UK Biobank (N=48,922) that used Olink for proteomics measurement. Murine single-cell RNA-sequencing data from aortic arch cells, inclusive of resident hematologic cells, in mice with Tet2-/-bone marrow and wild-type mice revealed corroborating differential expression of TET2-associated protein-encoding genes. Lastly, we apply these observations to identify 68 plasma proteins shared between CHIP and coronary artery disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550509">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550509" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550509">
        <p class="paperTitle">Pathogenic CANVAS (AAGGG)n repeats stall DNA replication due to the formation of alternative DNA structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550509" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550509" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hisey, J. A.; Radchenko, E. A.; Ceschi, S.; Rastokina, A.; Mandel, N. H.; McGinty, R. J.; Matos-Rodrigues, G.; Hernandez, A.; Nussenzweig, A.; Mirkin, S. M.</p>
        <p class="info">Score: 10.0, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550509' target='https://doi.org/10.1101/2023.07.25.550509'> 10.1101/2023.07.25.550509</a></p>
        <p class="abstract">CANVAS is a recently characterized repeat expansion disease, most commonly caused by homozygous expansions of an intronic (A2G3)n repeat in the RFC1 gene. There are a multitude of repeat motifs found in the human population at this locus, some of which are pathogenic and others benign. In this study, we conducted structure-functional analyses of the main pathogenic (A2G3)n and the main nonpathogenic (A4G)n repeats. We found that the pathogenic, but not the nonpathogenic, repeat presents a potent, orientation-dependent impediment to DNA polymerization in vitro. The pattern of the polymerization blockage is consistent with triplex or quadruplex formation in the presence of magnesium or potassium ions, respectively. Chemical probing of both repeats in supercoiled DNA reveals triplex H-DNA formation by the pathogenic repeat. Consistently, bioinformatic analysis of the S1-END-seq data from human cell lines shows preferential H-DNA formation genome-wide by (A2G3)n motifs over (A4G)n motifs in vivo. Finally, the pathogenic, but not the non-pathogenic, repeat stalls replication fork progression in yeast and human cells. We hypothesize that CANVAS-causing (A2G3)n repeat represents a challenge to genome stability by folding into alternative DNA structures that stall DNA replication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550394">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550394" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550394">
        <p class="paperTitle">The influence of HLA genetic variation on plasma protein expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550394" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550394" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishna, C.; Chiou, J.; Lee, I.; Kim, H. I.; Atalar Aksit, M.; Sakaue, S.; Von Schack, D.; Raychaudhuri, S.; Ziemek, D.; Hu, X.</p>
        <p class="info">Score: 9.6, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550394' target='https://doi.org/10.1101/2023.07.24.550394'> 10.1101/2023.07.24.550394</a></p>
        <p class="abstract">Polymorphism in the human leukocyte antigen class I (HLA-I) and class II (HLA-II) genes is strongly implicated in susceptibility to immune-mediated diseases. However, the molecular effects of HLA genetic variation, including and beyond antigen presentation, remain unclear. Here we examined the effect of HLA genetic variation on the expression of 2940 plasma proteins using imputed HLA variants in 45,330 Europeans in the UK Biobank. We detected 504 proteins (17.1% of all proteins tested) affected by HLA genetic variation (HLA-pQTL), including widespread trans regulation of protein expression by autoimmune disease risk alleles. HLA-pQTL were enriched in gene families related to antigen presentation (e.g. B2M), T cell fate (CD8A; CD4), chemokines (CCL19; CCL21), and NK and macrophage receptors (KIR; LILRA/B), suggesting that HLA polymorphism affects both adaptive and innate immunity. HLA-pQTL also affected expression of diverse proteins with unclear roles in the immune response (e.g. SFTPD, LRPAP1, ENPP6, NPTX1), as well as drug targets for immune-mediated diseases, suggesting complex regulatory roles of the HLA loci. Among trans HLA-pQTL, HLA variants explained 0.1-42.9% of the protein expression variance. Fine-mapping revealed that most HLA-pQTL implicated amino acid positions within the peptide binding groove, suggesting that trans regulation of plasma protein expression by the HLA loci is primarily a consequence of antigen presentation. We also show that HLA-I and II uniquely affect different proteins and biological mechanisms. Altogether, our data reveal the effects of HLA genetic variation on protein expression and aid the interpretation of associations between HLA alleles and immune-mediated diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549478">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549478" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549478">
        <p class="paperTitle">Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549478" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549478" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.</p>
        <p class="info">Score: 399.3, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549478' target='https://doi.org/10.1101/2023.07.18.549478'> 10.1101/2023.07.18.549478</a></p>
        <p class="abstract">Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging. Epigenetic clocks have been shown to be valuable tools for measuring biological age in a variety of tissues and samples. As such, these epigenetic clocks can determine accelerated biological aging and time-to-mortality across various tissues. Previous reports have shown accelerated biological aging and telomere attrition acceleration following SARS-CoV-2 infection. However, the effect of accelerated epigenetic aging on outcome (death/recovery) in COVID-19 patients with Acute Respiratory Distress Syndrome (ARDS) has not been well investigated. In this study, we measured DNA methylation age and telomere attrition in 87 severe COVID-19 cases with ARDS under mechanical ventilation. Furthermore, we compared dynamic changes in epigenetic aging across multiples time-points until recovery or death. Epigenetic age was measured using the Horvath, Hannum, DNAm skin and blood, GrimAge, and PhenoAge clocks, whereas telomere length was calculated using the surrogate marker DNAmTL. Our analysis revealed significant accelerated epigenetic aging but no telomere attrition acceleration in severe COVID-19 cases. In addition, we observed epigenetic age deceleration at inclusion vs end of follow-up in recovered but not in deceased COVID-19 cases using certain clocks. When comparing dynamic changes in epigenetic age acceleration (EAA), we detected higher EAA using both the Horvath and PhenoAge clocks in deceased vs recovered patients. The DNAmTL measurements revealed telomere attrition acceleration in deceased COVID19 patients between inclusion and end of follow-up as well as a significant change in dynamic telomere attrition acceleration when comparing patients who recovered vs those who died. In conclusion, EAA and telomere attrition acceleration was associated with treatment outcome in hospitalized COVID-19 Patients with ARDS. A better understanding of the long-term effects of EAA in COVID19 patients and how they might contribute to Long COVID symptoms in recovered individuals is urgently needed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.551029">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.551029" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.551029">
        <p class="paperTitle">Mitochondrial Genome Diversity across the Subphylum Saccharomycotina</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.551029" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.551029" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wolters, J. F.; LaBella, A.; Opulente, D.; Rokas, A.; Hittinger, C. T.</p>
        <p class="info">Score: 7.1, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.551029' target='https://doi.org/10.1101/2023.07.28.551029'> 10.1101/2023.07.28.551029</a></p>
        <p class="abstract">Eukaryotic life depends on the functional elements encoded by both the nuclear genome and organellar genomes, such as those contained within the mitochondria. The content, size, and structure of the mitochondrial genome varies across organisms with potentially large implications for phenotypic variance and resulting evolutionary trajectories. Among yeasts in the subphylum Saccharomycotina, extensive differences have been observed in various species relative to the model yeast Saccharomyces cerevisiae, but mitochondrial genome sampling across many groups has been scarce, even as hundreds of nuclear genomes have become available. By extracting mitochondrial assemblies from existing short-read genome sequence datasets, we have greatly expanded both the number of available genomes and the coverage across sparsely sampled clades. Comparison of 353 yeast mitochondrial genomes revealed that, while size and GC content were fairly consistent across species, those in the genera Metschnikowia and Saccharomyces trended larger, while several species in the order Saccharomycetales, which includes S. cerevisiae, exhibited lower GC content. Extreme examples for both size and GC content were scattered throughout the subphylum. All mitochondrial genomes shared a core set of protein-coding genes for Complexes III, IV, and V, but they varied in the presence or absence of mitochondrially-encoded canonical Complex I genes. We traced the loss of Complex I genes to a major event in the ancestor of the orders Saccharomycetales and Saccharomycodales, but we also observed several independent losses in the orders Phaffomycetales, Pichiales, and Dipodascales. In contrast to prior hypotheses based on smaller-scale datasets, comparison of evolutionary rates in protein-coding genes showed no bias towards elevated rates among aerobically fermenting (Crabtree/Warburg-positive) yeasts. Mitochondrial introns were widely distributed, but they were highly enriched in some groups. The majority of mitochondrial introns were poorly conserved within groups, but several were shared within groups, between groups, and even across taxonomic orders, which is consistent with horizontal gene transfer, likely involving homing endonucleases acting as selfish elements. As the number of available fungal nuclear genomes continues to expand, the methods described here to retrieve mitochondrial genome sequences from these datasets will prove invaluable to ensuring that studies of fungal mitochondrial genomes keep pace with their nuclear counterparts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549734">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549734" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549734">
        <p class="paperTitle">Genetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549734" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549734" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suhre, K.</p>
        <p class="info">Score: 15.3, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549734' target='https://doi.org/10.1101/2023.07.19.549734'> 10.1101/2023.07.19.549734</a></p>
        <p class="abstract">Protein quantitative trait loci (pQTLs) are an invaluable source of information for drug target development as they provide genetic evidence to support protein function, suggest relationships between cis- and trans-associated proteins, and link proteins to disease where they collocate with genetic risk loci for clinical endpoints. Using the recently released Olink proteomics data for 1,463 proteins measured in over 54,000 samples of the UK Biobank we identified and replicated 4,248 associations with 2,821 ratios between protein levels (rQTLs) where the strengths of association at known pQTL loci increased by up to several hundred orders of magnitude. We attribute this increase in statistical power (p-gain) to accounting for genetic and non-genetic variance shared by the two proteins in the ratio pair. Protein pairs with a significant p-gain were 7.6-fold enriched in known protein-protein interactions, suggesting that their ratios reflect biological links between the implicated proteins. We then conducted a GWAS on the 2,821 ratios and identified 2,527 novel rQTLs, increasing the number of discovered genetic signals compared to the original protein-only GWAS by 24.7%. At examples we demonstrate that this approach can identify novel loci of clinical relevance, support causal gene identification, and reveal complex networks of interacting proteins. Taken together, our study adds significant value to the genetic insights that can be derived from the UKB proteomics data and motivates the wider use of ratios in large scale GWAS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548129">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548129" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548129">
        <p class="paperTitle">Pax3 lineage-specific deletion of Gpr161 is associated with spinal neural tube and craniofacial malformations during embryonic development.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548129" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548129" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, S. E.; Chothani, P.; Shaik, R.; Pollard, W.; Finnell, R. H.</p>
        <p class="info">Score: 8.7, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548129' target='https://doi.org/10.1101/2023.07.07.548129'> 10.1101/2023.07.07.548129</a></p>
        <p class="abstract">Shh signaling is the morphogen signaling that regulates embryonic craniofacial and neural tube development. G protein-coupled receptor 161 (Gpr161) is a negative regulator of Shh signaling, and its inactivation in mice results in embryo lethality with craniofacial and neural tube defects (NTDs). However, the structural defects of later embryonic stages in Gpr161 null mice and cell lineages underlying abnormalities were not well characterized due to their limited lifespan. We found the Pax3 lineage-specific deletion of Gpr161 in mice presented with tectal hypertrophy (anterior dorsal neuroepithelium), cranial vault and facial bone hypoplasia (cranial neural crest (CNC)), vertebral abnormalities (somite), and the closed form of spina bifida (posterior dorsal neuroepithelium). In particular, the closed form of spina bifida is partly due to the reduced Pax3 and Cdx4 gene expression of the posterior dorsal neural tubes of Gpr161 mutant embryos involving decreased Wnt signaling whereas Shh signaling was increased. This study provides the novel role of Gpr161 in the posterior neural tube development and confirms its role on CNC- and somite-derived skeletogenesis and midbrain morphogenesis in mice.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550548">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550548" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550548">
        <p class="paperTitle">Calcineurin contributes to RNAi-mediated transgene silencing and small interfering RNA production in the human fungal pathogen Cryptococcus neoformans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550548" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550548" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yadav, V.; Mohan, R.; Sun, S.; Heitman, J.</p>
        <p class="info">Score: 5.1, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550548' target='https://doi.org/10.1101/2023.07.25.550548'> 10.1101/2023.07.25.550548</a></p>
        <p class="abstract">Adaptation to external environmental challenges at the cellular level requires rapid responses and involves relay of information to the nucleus to drive key gene expression changes through downstream transcription factors. Here, we describe an alternative route of adaptation through a direct role for cellular signaling components in governing gene expression via RNA interference-mediated small RNA production. Calcium-calcineurin signaling is a highly conserved signaling cascade that plays central roles in stress adaptation and virulence of eukaryotic pathogens, including the human fungal pathogen Cryptococcus neoformans. Upon activation in C. neoformans, calcineurin localizes to P-bodies, membrane-less organelles that are also the site for RNA processing. Here, we studied the role of calcineurin and its substrates in RNAi-mediated transgene silencing. Our results reveal that calcineurin regulates both the onset and the reversion of transgene silencing. We found that some calcineurin substrates that localize to P-bodies also regulate transgene silencing but in opposing directions. Small RNA sequencing in mutants lacking calcineurin or its targets revealed a role for calcineurin in small RNA production. Interestingly, the impact of calcineurin and its substrates was found to be different in genome-wide analysis, suggesting that calcineurin may regulate small RNA production in C. neoformans through additional pathways. Overall, these findings define a mechanism by which signaling machinery induced by external stimuli can directly alter gene expression to accelerate adaptative responses and contribute to genome defense.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549962" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549962">
        <p class="paperTitle">A small RNA from a natural bacterial pathogen of C. elegans induces transgenerational inheritance of learned avoidance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sengupta, T.; Ange, J. S.; Moore, R. S.; Kaletsky, R. T.; Marogi, J. T.; Myhrvold, C.; Gitai, Z.; Murphy, C. T.</p>
        <p class="info">Score: 15.8, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549962' target='https://doi.org/10.1101/2023.07.20.549962'> 10.1101/2023.07.20.549962</a></p>
        <p class="abstract">A small RNA from a clinical isolate of Pseudomonas aeruginosa, PA14, induces learned avoidance and its transgenerational inheritance in C. elegans. However, it is not known if small RNAs from bacteria found in C. elegans natural habitat can regulate host behavior and produce heritable behavioral effects. Here we found that GRb0427, a pathogenic Pseudomonas vranovensis strain isolated from the C. elegans microbiota, similarly affects worm behavior: worms learn to avoid this pathogenic bacterium following exposure, and this learned avoidance is inherited for four generations. The learned response is entirely mediated by small RNAs, which induce avoidance and transgenerational inheritance in both the laboratory N2 strain and a natural isolate of C. elegans, JU1580, providing further support that such mechanisms of learning and inheritance exist in the wild. A small RNA, Pv1, matches the sequence of C. elegans maco-1, and is both necessary and sufficient to induce learned avoidance in worms. However, Pv1 also results in avoidance of a beneficial microbiome strain, P. mendocina; this maladaptive response may favor reversal of the transgenerational memory after a few generations. Our findings suggest that bacterial small RNA-mediated regulation of host behavior and its transgenerational inheritance are functional in C. elegans natural environment, and that different bacterial small RNA-mediated regulation systems evolved independently but define shared molecular features of bacterial small RNAs that produce transgenerationally-inherited effects.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
