<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Integrator-mediated clustering of poised RNA polymerase II synchronizes histone transcription
Authors: Lu, F.; Park, B. J.; Fujiwara, R.; Wilusz, J. E.; Gilmour, D. S.; Lehmann, R.; Lionnet, T.
Score: 38.5, Published: 2023-10-08 DOI: 10.1101/2023.10.07.561364
Dynamic clustering of transcription players, including RNA polymerase II (Pol II), is thought to rely on multivalent interactions of their intrinsically disordered regions, thereby enhancing active transcription. Using the histone locus bodies (HLBs) of Drosophila nurse cells as a model, we find that Pol II forms long-lived, transcriptionally poised clusters distinct from liquid droplets, which contain unbound and paused Pol II.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Integrator-mediated clustering of poised RNA polymerase II synchronizes histone transcription
Authors: Lu, F.; Park, B. J.; Fujiwara, R.; Wilusz, J. E.; Gilmour, D. S.; Lehmann, R.; Lionnet, T.
Score: 38.5, Published: 2023-10-08 DOI: 10.1101/2023.10.07.561364
Dynamic clustering of transcription players, including RNA polymerase II (Pol II), is thought to rely on multivalent interactions of their intrinsically disordered regions, thereby enhancing active transcription. Using the histone locus bodies (HLBs) of Drosophila nurse cells as a model, we find that Pol II forms long-lived, transcriptionally poised clusters distinct from liquid droplets, which contain unbound and paused Pol II." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-22T10:37:23+00:00" />
<meta property="article:modified_time" content="2023-10-22T10:37:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Integrator-mediated clustering of poised RNA polymerase II synchronizes histone transcription
Authors: Lu, F.; Park, B. J.; Fujiwara, R.; Wilusz, J. E.; Gilmour, D. S.; Lehmann, R.; Lionnet, T.
Score: 38.5, Published: 2023-10-08 DOI: 10.1101/2023.10.07.561364
Dynamic clustering of transcription players, including RNA polymerase II (Pol II), is thought to rely on multivalent interactions of their intrinsically disordered regions, thereby enhancing active transcription. Using the histone locus bodies (HLBs) of Drosophila nurse cells as a model, we find that Pol II forms long-lived, transcriptionally poised clusters distinct from liquid droplets, which contain unbound and paused Pol II."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Integrator-mediated clustering of poised RNA polymerase II synchronizes histone transcription\nAuthors: Lu, F.; Park, B. J.; Fujiwara, R.; Wilusz, J. E.; Gilmour, D. S.; Lehmann, R.; Lionnet, T.\nScore: 38.5, Published: 2023-10-08 DOI: 10.1101/2023.10.07.561364\nDynamic clustering of transcription players, including RNA polymerase II (Pol II), is thought to rely on multivalent interactions of their intrinsically disordered regions, thereby enhancing active transcription. Using the histone locus bodies (HLBs) of Drosophila nurse cells as a model, we find that Pol II forms long-lived, transcriptionally poised clusters distinct from liquid droplets, which contain unbound and paused Pol II.",
  "keywords": [
    
  ],
  "articleBody": " Integrator-mediated clustering of poised RNA polymerase II synchronizes histone transcription\nAuthors: Lu, F.; Park, B. J.; Fujiwara, R.; Wilusz, J. E.; Gilmour, D. S.; Lehmann, R.; Lionnet, T.\nScore: 38.5, Published: 2023-10-08 DOI: 10.1101/2023.10.07.561364\nDynamic clustering of transcription players, including RNA polymerase II (Pol II), is thought to rely on multivalent interactions of their intrinsically disordered regions, thereby enhancing active transcription. Using the histone locus bodies (HLBs) of Drosophila nurse cells as a model, we find that Pol II forms long-lived, transcriptionally poised clusters distinct from liquid droplets, which contain unbound and paused Pol II. Depletion of the Integrator complex endonuclease module, but not its phosphatase module or pausing factors disperses these Pol II clusters, leaving HLBs intact. Consequently, histone transcription fails to reach peak levels during S-phase and aberrantly spills over throughout the cell cycle. We propose that clustering sequesters numerous poised Pol II molecules near gene promoters to ensure synchronous and efficient gene activation at desired times.\nPervasive biases in proxy GWAS based on parental history of Alzheimer's disease\nAuthors: Wu, Y.; Sun, Z.; Zheng, Q.; Miao, J.; Dorn, S.; Mukherjee, S.; Fletcher, J.; Lu, Q.\nScore: 36.9, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562272\nAlmost every recent Alzheimers disease (AD) genome-wide association study (GWAS) has performed meta-analysis to combine studies with clinical diagnosis of AD with studies that use proxy phenotypes based on parental disease history. Here, we report major limitations in current GWAS-by-proxy (GWAX) practices due to uncorrected survival bias and non-random participation of parental illness survey, which cause substantial discrepancies between AD GWAS and GWAX results. We demonstrate that current AD GWAX provide highly misleading genetic correlations between AD risk and higher education which subsequently affects a variety of genetic epidemiologic applications involving AD and cognition. Our study sheds important light on the design and analysis of mid-aged biobank cohorts and underscores the need for caution when interpreting genetic association results based on proxy-reported parental disease history.\nA gene with a thousand alleles: the HYPer-variable effectors of plant-parasitic nematodes\nAuthors: Sonawala, U.; Beasley, H.; Thorpe, P. J.; Varypatakis, K.; Senatori, B.; Jones, J. T.; Derevnina, L.; Eves-van den Akker, S.\nScore: 36.0, Published: 2023-10-18 DOI: 10.1101/2023.10.16.561705\nPathogens are engaged in a fierce evolutionary arms race with their host. The genes at the forefront of the engagement between kingdoms, including effectors and immune receptors, are part of diverse and highly mutable gene families. Even in this context, we discovered unprecedented variation in the HYPer-variable (HYP) effectors of plant-parasitic nematodes. We discovered single effector gene loci that can harbour potentially thousands of allelic variants. These alleles vary in the number, but principally in the organisation, of motifs within a central Hyper Variable Domain (HVD). Using targeted long-read sequencing, we dramatically expand the HYP repertoire of two plant-parasitic nematodes, Globodera pallida and G. rostochiensis, such that we can define distinct sets of species-specific rules underlying the apparently flawless genetic rearrangements. Finally, by analysing the HYP complement of 68 individual nematodes we made the unexpected finding that despite the huge number of alleles, most individuals are homozygous. Taken together, these data point to a novel mechanism of programmed genetic variation, termed HVD-editing, where alterations are locus-specific, strictly governed by rules, and can theoretically produce thousands of variants without errors.\nSingle-cell chromatin state transitions during epigenetic memory formation\nAuthors: Fujimori, T.; Rios-Martinez, C.; Thurm, A. R.; Hinks, M. M.; Doughty, B. R.; Sinha, J.; Le, D.; Hafner, A.; Greenleaf, W. J.; Boettiger, A. N.; Bintu, L.\nScore: 29.9, Published: 2023-10-05 DOI: 10.1101/2023.10.03.560616\nRepressive chromatin modifications are thought to compact chromatin to silence transcription. However, it is unclear how chromatin structure changes during silencing and epigenetic memory formation. We measured gene expression and chromatin structure in single cells after recruitment and release of repressors at a reporter gene. Chromatin structure is heterogeneous, with open and compact conformations present in both active and silent states. Recruitment of repressors associated with epigenetic memory produces chromatin compaction across 10-20 kilobases, while reversible silencing does not cause compaction at this scale. Chromatin compaction is inherited, but changes molecularly over time from histone methylation (H3K9me3) to DNA methylation. The level of compaction at the end of silencing quantitatively predicts epigenetic memory weeks later. Similarly, chromatin compaction at the Nanog locus predicts the degree of stem-cell fate commitment. These findings suggest that the chromatin state across tens of kilobases, beyond the gene itself, is important for epigenetic memory formation.\ndiplo-locus: A lightweight toolkit for inference and simulation of time-series genetic data under general diploid selection\nAuthors: Cheng, X.; Steinruecken, M.\nScore: 13.3, Published: 2023-10-17 DOI: 10.1101/2023.10.12.562101\nSummaryWhole-genome time-series allele frequency data are becoming more prevalent as ancient DNA (aDNA) sequences and data from evolve-and-resequence (E\u0026R) experiments are generated at a rapid pace. Such data presents unprecedented opportunities to elucidate the dynamics of adaptative genetic variation. However, despite many methods to infer parameters of selection models from allele frequency trajectories available in the literature, few provide user-friendly implementations for large-scale empirical applications. Here, we present diplo-locus, an open-source Python package that provides functionality to simulate and perform inference from time-series under the Wright-Fisher diffusion with general diploid selection. The package includes Python modules as well as command-line tools. AvailabilityPython package and command-line tool avilable at: https://github.com/steinrue/diplo_locus or https://pypi.org/project/diplo-locus/.\nFalse discovery rates of qpAdm-based screens for genetic admixture\nAuthors: Yüncü, E.; Isıldak, U.; Williams, M. P.; Huber, C. D.; Flegontova, O.; Vyazov, L. A.; Changmai, P.; Flegontov, P.\nScore: 26.0, Published: 2023-10-18 DOI: 10.1101/2023.04.25.538339\nAlthough a broad range of methods exists for reconstructing population history from genome-wide single nucleotide polymorphism data, just a few methods gained popularity in archaeogenetics: principal component analysis (PCA); ADMIXTURE, an algorithm that models individuals as mixtures of multiple ancestral sources represented by actual or inferred populations; formal tests for admixture such as f3-statistics and D/f4-statistics; and qpAdm, a tool for fitting two-component and more complex admixture models to groups or individuals. Despite their popularity in archaeogenetics, which is explained by modest computational requirements and ability to analyze data of various types and qualities, protocols relying on qpAdm that screen numerous alternative models of varying complexity and find \"fitting\" models (often considering both estimated admixture proportions and p-values as a composite criterion of model fit) remain untested on complex simulated population histories in the form of admixture graphs of random topology. We analyzed genotype data extracted from such simulations and tested various types of high-throughput qpAdm protocols (\"rotating\" and \"non-rotating\", with or without temporal stratification of target groups and proxy ancestry sources, and with or without a \"model competition\" step). We caution that high-throughput qpAdm protocols may be inappropriate for exploratory analyses in poorly studied regions/periods since their false discovery rates varied between 12% and 68% depending on the details of the protocol and on the amount and quality of simulated data (i.e., \u003e12% of fitting two-way admixture models imply gene flows that were not simulated). We demonstrate that for reducing false discovery rates of qpAdm protocols to nearly 0% it is advisable to use large SNP sets with low missing data rates, the rotating qpAdm protocol with a strictly enforced rule that target groups do not pre-date their proxy sources, and an unsupervised ADMIXTURE analysis as a way to verify feasible qpAdm models. Our study has a number of limitations: for instance, these recommendations depend on the assumption that the underlying genetic history is a complex admixture graph and not a stepping-stone model.\nA likelihood-based framework for demographic inference from genealogical trees\nAuthors: Fan, C.; Cahoon, J. L.; Dinh, B. L.; Ortega-Del Vecchyo, D.; Huber, C. D.; Edge, M. D.; Mancuso, N.; Chiang, C. W. K.\nScore: 22.2, Published: 2023-10-13 DOI: 10.1101/2023.10.10.561787\nThe demographic history of a population drives the pattern of genetic variation and is encoded in the gene-genealogical trees of the sampled alleles. However, existing methods to infer demographic history from genetic data tend to use relatively low-dimensional summaries of the genealogy, such as allele frequency spectra. As a step toward capturing more of the information encoded in the genome-wide sequence of genealogical trees, here we propose a novel framework called the genealogical likelihood (gLike), which derives the full likelihood of a genealogical tree under any hypothesized demographic history. Employing a graph-based structure, gLike summarizes across independent trees the relationships among all lineages in a tree with all possible trajectories of population memberships through time and efficiently computes the exact marginal probability under a parameterized demographic model. Through extensive simulations and empirical applications on populations that have experienced multiple admixtures, we showed that gLike can accurately estimate dozens of demographic parameters when the true genealogy is known, including ancestral population sizes, admixture timing, and admixture proportions. Moreover, when using genealogical trees inferred from genetic data, we showed that gLike outperformed conventional demographic inference methods that leverage only the allele-frequency spectrum and yielded parameter estimates that align with established historical knowledge of the past demographic histories for populations like Latino Americans and Native Hawaiians. Furthermore, our framework can trace ancestral histories by analyzing a sample from the admixed population without proxies for its source populations, removing the need to sample ancestral populations that may no longer exist. Taken together, our proposed gLike framework harnesses underutilized genealogical information to offer exceptional sensitivity and accuracy in inferring complex demographies for humans and other species, particularly as estimation of genome-wide genealogies improves.\nMolecular Time Capsules Enable Transcriptomic Recording in Living Cells\nAuthors: Parker, M. L.; Rubien, J.; McCormick, D.; Li, G.-W.\nScore: 9.0, Published: 2023-10-16 DOI: 10.1101/2023.10.12.562053\nLive-cell transcriptomic recording can help reveal hidden cellular states that precede phenotypic transformation. Here we demonstrate the use of protein-based encapsulation for preserving samples of cytoplasmic RNAs inside living cells. These molecular time capsules (MTCs) can be induced to create time-stamped transcriptome snapshots, preserve RNAs after cellular transitions, and enable retrospective investigations of gene expression programs that drive distinct developmental trajectories. MTCs also open the possibility to uncover transcriptomes in difficult-to-reach conditions.\nCRISPR/Cas9 ribonucleoprotein-mediated mutagenesis in Sporisorium reilianum\nAuthors: Werner, J.; Zuo, W.; Doehlemann, G.\nScore: 7.9, Published: 2023-10-19 DOI: 10.1101/2023.10.17.562663\nClustered Regularly Interspaced Short Palindromic Repeats/CRISPR-associated protein 9 (CRISPR/Cas9) has become the state of art for mutagenesis in filamentous fungi. Here, we describe a RNP-mediated CRISPR/Cas9 for mutagenesis in Sporisorium reilianum. The efficiency of the method was tested in vitro with a cleavage assay as well as in vivo with a GFP expressing S. reilianum strain. We applied this method to generate frameshift-, knock-in- and knock-out mutants in S. reilianum without a resistance marker by using an auto-replicating plasmid for selection. The RNP-mediated CRISPR/Cas9 increased the mutagenesis efficiency and firstly enables a marker-free genome editing in S. reilianum.\nAn RNAi screen for conserved kinases that enhance microRNA activity after dauer in C. elegans\nAuthors: Roka Pun, H.; Karp, X.\nScore: 6.1, Published: 2023-10-18 DOI: 10.1101/2023.10.17.562753\nGene regulation in changing environments is critical for maintaining homeostasis. Some animals undergo a stress-resistant diapause stage to withstand harsh environmental conditions encountered during development. MicroRNAs (miRNAs) are one mechanism for regulating gene expression during and after diapause. MicroRNAs downregulate target genes post-transcriptionally through the activity of the miRNA-Induced Silencing Complex (miRISC). Argonaute is the core miRISC protein that binds to both the miRNA and to other miRISC proteins. The two major miRNA Argonautes in the C. elegans soma are ALG-1 and ALG-2, which function partially redundantly. Loss of alg-1 (alg-1(0)) causes penetrant developmental phenotypes including vulval defects and the reiteration of larval cell programs in hypodermal cells. However, these phenotypes are essentially absent if alg-1(0) animals undergo a diapause stage called dauer. Levels of the relevant miRNAs are not higher during or after dauer, suggesting that activity of the miRISC may be enhanced in this context. To identify genes that are required for alg-1(0) mutants to develop without vulval defects after dauer, we performed an RNAi screen of genes encoding conserved kinases. We focused on kinases because of their known role in modulating miRISC activity. We found RNAi knockdown of four kinase-encoding genes, air-2, bub-1, chk-1, and nekl-3, caused vulval defects and reiterative phenotypes in alg-1(0) mutants after dauer, and that these defects were more penetrant in an alg-1(0) background than in wild type. Our results implicate these kinases as potential regulators of miRISC activity during post-dauer development in C. elegans.\n",
  "wordCount" : "2026",
  "inLanguage": "en",
  "datePublished": "2023-10-22T10:37:23Z",
  "dateModified": "2023-10-22T10:37:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on October 22, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.07.561364">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.07.561364" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.07.561364">
        <p class="paperTitle">Integrator-mediated clustering of poised RNA polymerase II synchronizes histone transcription</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.07.561364" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.07.561364" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, F.; Park, B. J.; Fujiwara, R.; Wilusz, J. E.; Gilmour, D. S.; Lehmann, R.; Lionnet, T.</p>
        <p class="info">Score: 38.5, Published: 2023-10-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.07.561364' target='https://doi.org/10.1101/2023.10.07.561364'> 10.1101/2023.10.07.561364</a></p>
        <p class="abstract">Dynamic clustering of transcription players, including RNA polymerase II (Pol II), is thought to rely on multivalent interactions of their intrinsically disordered regions, thereby enhancing active transcription. Using the histone locus bodies (HLBs) of Drosophila nurse cells as a model, we find that Pol II forms long-lived, transcriptionally poised clusters distinct from liquid droplets, which contain unbound and paused Pol II. Depletion of the Integrator complex endonuclease module, but not its phosphatase module or pausing factors disperses these Pol II clusters, leaving HLBs intact. Consequently, histone transcription fails to reach peak levels during S-phase and aberrantly spills over throughout the cell cycle. We propose that clustering sequesters numerous poised Pol II molecules near gene promoters to ensure synchronous and efficient gene activation at desired times.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562272">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562272" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562272">
        <p class="paperTitle">Pervasive biases in proxy GWAS based on parental history of Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562272" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562272" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wu, Y.; Sun, Z.; Zheng, Q.; Miao, J.; Dorn, S.; Mukherjee, S.; Fletcher, J.; Lu, Q.</p>
        <p class="info">Score: 36.9, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562272' target='https://doi.org/10.1101/2023.10.13.562272'> 10.1101/2023.10.13.562272</a></p>
        <p class="abstract">Almost every recent Alzheimers disease (AD) genome-wide association study (GWAS) has performed meta-analysis to combine studies with clinical diagnosis of AD with studies that use proxy phenotypes based on parental disease history. Here, we report major limitations in current GWAS-by-proxy (GWAX) practices due to uncorrected survival bias and non-random participation of parental illness survey, which cause substantial discrepancies between AD GWAS and GWAX results. We demonstrate that current AD GWAX provide highly misleading genetic correlations between AD risk and higher education which subsequently affects a variety of genetic epidemiologic applications involving AD and cognition. Our study sheds important light on the design and analysis of mid-aged biobank cohorts and underscores the need for caution when interpreting genetic association results based on proxy-reported parental disease history.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.561705">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.561705" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.561705">
        <p class="paperTitle">A gene with a thousand alleles: the HYPer-variable effectors of plant-parasitic nematodes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.561705" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.561705" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sonawala, U.; Beasley, H.; Thorpe, P. J.; Varypatakis, K.; Senatori, B.; Jones, J. T.; Derevnina, L.; Eves-van den Akker, S.</p>
        <p class="info">Score: 36.0, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.561705' target='https://doi.org/10.1101/2023.10.16.561705'> 10.1101/2023.10.16.561705</a></p>
        <p class="abstract">Pathogens are engaged in a fierce evolutionary arms race with their host. The genes at the forefront of the engagement between kingdoms, including effectors and immune receptors, are part of diverse and highly mutable gene families. Even in this context, we discovered unprecedented variation in the HYPer-variable (HYP) effectors of plant-parasitic nematodes. We discovered single effector gene loci that can harbour potentially thousands of allelic variants. These alleles vary in the number, but principally in the organisation, of motifs within a central Hyper Variable Domain (HVD). Using targeted long-read sequencing, we dramatically expand the HYP repertoire of two plant-parasitic nematodes, Globodera pallida and G. rostochiensis, such that we can define distinct sets of species-specific rules underlying the apparently flawless genetic rearrangements. Finally, by analysing the HYP complement of 68 individual nematodes we made the unexpected finding that despite the huge number of alleles, most individuals are homozygous. Taken together, these data point to a novel mechanism of programmed genetic variation, termed HVD-editing, where alterations are locus-specific, strictly governed by rules, and can theoretically produce thousands of variants without errors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560616">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560616" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560616">
        <p class="paperTitle">Single-cell chromatin state transitions during epigenetic memory formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560616" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560616" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fujimori, T.; Rios-Martinez, C.; Thurm, A. R.; Hinks, M. M.; Doughty, B. R.; Sinha, J.; Le, D.; Hafner, A.; Greenleaf, W. J.; Boettiger, A. N.; Bintu, L.</p>
        <p class="info">Score: 29.9, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560616' target='https://doi.org/10.1101/2023.10.03.560616'> 10.1101/2023.10.03.560616</a></p>
        <p class="abstract">Repressive chromatin modifications are thought to compact chromatin to silence transcription. However, it is unclear how chromatin structure changes during silencing and epigenetic memory formation. We measured gene expression and chromatin structure in single cells after recruitment and release of repressors at a reporter gene. Chromatin structure is heterogeneous, with open and compact conformations present in both active and silent states. Recruitment of repressors associated with epigenetic memory produces chromatin compaction across 10-20 kilobases, while reversible silencing does not cause compaction at this scale. Chromatin compaction is inherited, but changes molecularly over time from histone methylation (H3K9me3) to DNA methylation. The level of compaction at the end of silencing quantitatively predicts epigenetic memory weeks later. Similarly, chromatin compaction at the Nanog locus predicts the degree of stem-cell fate commitment. These findings suggest that the chromatin state across tens of kilobases, beyond the gene itself, is important for epigenetic memory formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.562101">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.562101" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.562101">
        <p class="paperTitle">diplo-locus: A lightweight toolkit for inference and simulation of time-series genetic data under general diploid selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.562101" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.562101" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cheng, X.; Steinruecken, M.</p>
        <p class="info">Score: 13.3, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.562101' target='https://doi.org/10.1101/2023.10.12.562101'> 10.1101/2023.10.12.562101</a></p>
        <p class="abstract">SummaryWhole-genome time-series allele frequency data are becoming more prevalent as ancient DNA (aDNA) sequences and data from evolve-and-resequence (E&amp;R) experiments are generated at a rapid pace. Such data presents unprecedented opportunities to elucidate the dynamics of adaptative genetic variation. However, despite many methods to infer parameters of selection models from allele frequency trajectories available in the literature, few provide user-friendly implementations for large-scale empirical applications. Here, we present diplo-locus, an open-source Python package that provides functionality to simulate and perform inference from time-series under the Wright-Fisher diffusion with general diploid selection. The package includes Python modules as well as command-line tools.

AvailabilityPython package and command-line tool avilable at: https://github.com/steinrue/diplo_locus or https://pypi.org/project/diplo-locus/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.04.25.538339">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.04.25.538339" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.04.25.538339">
        <p class="paperTitle">False discovery rates of qpAdm-based screens for genetic admixture</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.04.25.538339" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.04.25.538339" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yüncü, E.; Isıldak, U.; Williams, M. P.; Huber, C. D.; Flegontova, O.; Vyazov, L. A.; Changmai, P.; Flegontov, P.</p>
        <p class="info">Score: 26.0, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.04.25.538339' target='https://doi.org/10.1101/2023.04.25.538339'> 10.1101/2023.04.25.538339</a></p>
        <p class="abstract">Although a broad range of methods exists for reconstructing population history from genome-wide single nucleotide polymorphism data, just a few methods gained popularity in archaeogenetics: principal component analysis (PCA); ADMIXTURE, an algorithm that models individuals as mixtures of multiple ancestral sources represented by actual or inferred populations; formal tests for admixture such as f3-statistics and D/f4-statistics; and qpAdm, a tool for fitting two-component and more complex admixture models to groups or individuals. Despite their popularity in archaeogenetics, which is explained by modest computational requirements and ability to analyze data of various types and qualities, protocols relying on qpAdm that screen numerous alternative models of varying complexity and find &#34;fitting&#34; models (often considering both estimated admixture proportions and p-values as a composite criterion of model fit) remain untested on complex simulated population histories in the form of admixture graphs of random topology. We analyzed genotype data extracted from such simulations and tested various types of high-throughput qpAdm protocols (&#34;rotating&#34; and &#34;non-rotating&#34;, with or without temporal stratification of target groups and proxy ancestry sources, and with or without a &#34;model competition&#34; step). We caution that high-throughput qpAdm protocols may be inappropriate for exploratory analyses in poorly studied regions/periods since their false discovery rates varied between 12% and 68% depending on the details of the protocol and on the amount and quality of simulated data (i.e., &gt;12% of fitting two-way admixture models imply gene flows that were not simulated). We demonstrate that for reducing false discovery rates of qpAdm protocols to nearly 0% it is advisable to use large SNP sets with low missing data rates, the rotating qpAdm protocol with a strictly enforced rule that target groups do not pre-date their proxy sources, and an unsupervised ADMIXTURE analysis as a way to verify feasible qpAdm models. Our study has a number of limitations: for instance, these recommendations depend on the assumption that the underlying genetic history is a complex admixture graph and not a stepping-stone model.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561787">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561787" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561787">
        <p class="paperTitle">A likelihood-based framework for demographic inference from genealogical trees</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561787" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561787" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fan, C.; Cahoon, J. L.; Dinh, B. L.; Ortega-Del Vecchyo, D.; Huber, C. D.; Edge, M. D.; Mancuso, N.; Chiang, C. W. K.</p>
        <p class="info">Score: 22.2, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561787' target='https://doi.org/10.1101/2023.10.10.561787'> 10.1101/2023.10.10.561787</a></p>
        <p class="abstract">The demographic history of a population drives the pattern of genetic variation and is encoded in the gene-genealogical trees of the sampled alleles. However, existing methods to infer demographic history from genetic data tend to use relatively low-dimensional summaries of the genealogy, such as allele frequency spectra. As a step toward capturing more of the information encoded in the genome-wide sequence of genealogical trees, here we propose a novel framework called the genealogical likelihood (gLike), which derives the full likelihood of a genealogical tree under any hypothesized demographic history. Employing a graph-based structure, gLike summarizes across independent trees the relationships among all lineages in a tree with all possible trajectories of population memberships through time and efficiently computes the exact marginal probability under a parameterized demographic model. Through extensive simulations and empirical applications on populations that have experienced multiple admixtures, we showed that gLike can accurately estimate dozens of demographic parameters when the true genealogy is known, including ancestral population sizes, admixture timing, and admixture proportions. Moreover, when using genealogical trees inferred from genetic data, we showed that gLike outperformed conventional demographic inference methods that leverage only the allele-frequency spectrum and yielded parameter estimates that align with established historical knowledge of the past demographic histories for populations like Latino Americans and Native Hawaiians. Furthermore, our framework can trace ancestral histories by analyzing a sample from the admixed population without proxies for its source populations, removing the need to sample ancestral populations that may no longer exist. Taken together, our proposed gLike framework harnesses underutilized genealogical information to offer exceptional sensitivity and accuracy in inferring complex demographies for humans and other species, particularly as estimation of genome-wide genealogies improves.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.562053">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.562053" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.562053">
        <p class="paperTitle">Molecular Time Capsules Enable Transcriptomic Recording in Living Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.562053" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.562053" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parker, M. L.; Rubien, J.; McCormick, D.; Li, G.-W.</p>
        <p class="info">Score: 9.0, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.562053' target='https://doi.org/10.1101/2023.10.12.562053'> 10.1101/2023.10.12.562053</a></p>
        <p class="abstract">Live-cell transcriptomic recording can help reveal hidden cellular states that precede phenotypic transformation. Here we demonstrate the use of protein-based encapsulation for preserving samples of cytoplasmic RNAs inside living cells. These molecular time capsules (MTCs) can be induced to create time-stamped transcriptome snapshots, preserve RNAs after cellular transitions, and enable retrospective investigations of gene expression programs that drive distinct developmental trajectories. MTCs also open the possibility to uncover transcriptomes in difficult-to-reach conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562663">
        <p class="paperTitle">CRISPR/Cas9 ribonucleoprotein-mediated mutagenesis in Sporisorium reilianum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Werner, J.; Zuo, W.; Doehlemann, G.</p>
        <p class="info">Score: 7.9, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562663' target='https://doi.org/10.1101/2023.10.17.562663'> 10.1101/2023.10.17.562663</a></p>
        <p class="abstract">Clustered Regularly Interspaced Short Palindromic Repeats/CRISPR-associated protein 9 (CRISPR/Cas9) has become the state of art for mutagenesis in filamentous fungi. Here, we describe a RNP-mediated CRISPR/Cas9 for mutagenesis in Sporisorium reilianum. The efficiency of the method was tested in vitro with a cleavage assay as well as in vivo with a GFP expressing S. reilianum strain. We applied this method to generate frameshift-, knock-in- and knock-out mutants in S. reilianum without a resistance marker by using an auto-replicating plasmid for selection. The RNP-mediated CRISPR/Cas9 increased the mutagenesis efficiency and firstly enables a marker-free genome editing in S. reilianum.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562753">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562753" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562753">
        <p class="paperTitle">An RNAi screen for conserved kinases that enhance microRNA activity after dauer in C. elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562753" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562753" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roka Pun, H.; Karp, X.</p>
        <p class="info">Score: 6.1, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562753' target='https://doi.org/10.1101/2023.10.17.562753'> 10.1101/2023.10.17.562753</a></p>
        <p class="abstract">Gene regulation in changing environments is critical for maintaining homeostasis. Some animals undergo a stress-resistant diapause stage to withstand harsh environmental conditions encountered during development. MicroRNAs (miRNAs) are one mechanism for regulating gene expression during and after diapause. MicroRNAs downregulate target genes post-transcriptionally through the activity of the miRNA-Induced Silencing Complex (miRISC). Argonaute is the core miRISC protein that binds to both the miRNA and to other miRISC proteins. The two major miRNA Argonautes in the C. elegans soma are ALG-1 and ALG-2, which function partially redundantly. Loss of alg-1 (alg-1(0)) causes penetrant developmental phenotypes including vulval defects and the reiteration of larval cell programs in hypodermal cells. However, these phenotypes are essentially absent if alg-1(0) animals undergo a diapause stage called dauer. Levels of the relevant miRNAs are not higher during or after dauer, suggesting that activity of the miRISC may be enhanced in this context. To identify genes that are required for alg-1(0) mutants to develop without vulval defects after dauer, we performed an RNAi screen of genes encoding conserved kinases. We focused on kinases because of their known role in modulating miRISC activity. We found RNAi knockdown of four kinase-encoding genes, air-2, bub-1, chk-1, and nekl-3, caused vulval defects and reiterative phenotypes in alg-1(0) mutants after dauer, and that these defects were more penetrant in an alg-1(0) background than in wild type. Our results implicate these kinases as potential regulators of miRISC activity during post-dauer development in C. elegans.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
