<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Multi-Step Control of Homologous Recombination by Mec1/ATR Ensures Robust Suppression of Gross Chromosomal Rearrangements
Authors: Xie, B.; Sanford, E. J.; Hung, S.-H.; Wagner, M. M.; Heyer, W.-D.; Smolka, M. B.
Score: 15.5, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568146
The Mec1/ATR kinase is crucial for genome stability, yet the mechanism by which it prevents gross chromosomal rearrangements (GCRs) remains unknown. Here we find that in cells with deficient Mec1 signaling, GCRs accumulate due to the deregulation of multiple steps in homologous recombination (HR).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Multi-Step Control of Homologous Recombination by Mec1/ATR Ensures Robust Suppression of Gross Chromosomal Rearrangements
Authors: Xie, B.; Sanford, E. J.; Hung, S.-H.; Wagner, M. M.; Heyer, W.-D.; Smolka, M. B.
Score: 15.5, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568146
The Mec1/ATR kinase is crucial for genome stability, yet the mechanism by which it prevents gross chromosomal rearrangements (GCRs) remains unknown. Here we find that in cells with deficient Mec1 signaling, GCRs accumulate due to the deregulation of multiple steps in homologous recombination (HR)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-29T10:40:14+00:00" />
<meta property="article:modified_time" content="2023-11-29T10:40:14+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Multi-Step Control of Homologous Recombination by Mec1/ATR Ensures Robust Suppression of Gross Chromosomal Rearrangements
Authors: Xie, B.; Sanford, E. J.; Hung, S.-H.; Wagner, M. M.; Heyer, W.-D.; Smolka, M. B.
Score: 15.5, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568146
The Mec1/ATR kinase is crucial for genome stability, yet the mechanism by which it prevents gross chromosomal rearrangements (GCRs) remains unknown. Here we find that in cells with deficient Mec1 signaling, GCRs accumulate due to the deregulation of multiple steps in homologous recombination (HR)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Multi-Step Control of Homologous Recombination by Mec1/ATR Ensures Robust Suppression of Gross Chromosomal Rearrangements\nAuthors: Xie, B.; Sanford, E. J.; Hung, S.-H.; Wagner, M. M.; Heyer, W.-D.; Smolka, M. B.\nScore: 15.5, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568146\nThe Mec1/ATR kinase is crucial for genome stability, yet the mechanism by which it prevents gross chromosomal rearrangements (GCRs) remains unknown. Here we find that in cells with deficient Mec1 signaling, GCRs accumulate due to the deregulation of multiple steps in homologous recombination (HR).",
  "keywords": [
    
  ],
  "articleBody": " Multi-Step Control of Homologous Recombination by Mec1/ATR Ensures Robust Suppression of Gross Chromosomal Rearrangements\nAuthors: Xie, B.; Sanford, E. J.; Hung, S.-H.; Wagner, M. M.; Heyer, W.-D.; Smolka, M. B.\nScore: 15.5, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568146\nThe Mec1/ATR kinase is crucial for genome stability, yet the mechanism by which it prevents gross chromosomal rearrangements (GCRs) remains unknown. Here we find that in cells with deficient Mec1 signaling, GCRs accumulate due to the deregulation of multiple steps in homologous recombination (HR). Mec1 primarily suppresses GCRs through its role in activating the canonical checkpoint kinase Rad53, which ensures the proper control of DNA end resection. Upon loss of Rad53 signaling and resection control, Mec1 becomes hyperactivated and triggers a salvage pathway in which the Sgs1 helicase is recruited to sites of DNA lesions via the 911-Dpb11 scaffolds to favor heteroduplex rejection and limit HR-driven GCR accumulation. Fusing an ssDNA recognition domain to Sgs1 bypasses the requirement of Mec1 signaling for GCR suppression and nearly eliminates D-loop formation, thus preventing non-allelic recombination events. We propose that Mec1 regulates multiple steps of HR to prevent GCRs while ensuring balanced HR usage when needed for promoting tolerance to replication stress.\nWhole genome association testing in 333,100 individuals across three biobanks identifies rare non-coding single variant and genomic aggregate associations with height\nAuthors: Hawkes, G.; Beaumont, R. N.; Li, Z.; Mandla, R.; Li, X.; Albert, C. M.; Arnett, D. K.; Ashley-Koch, A. E.; Ashrani, A. A.; Barnes, K. C.; Boerwinkle, E.; Brody, J. A.; Carson, A. P.; Chami, N.; Chen, Y.-D. I.; Chung, M. K.; Curran, J. E.; Darbar, D.; Ellinor, P. T.; Fornage, M.; Gordeuk, V. R.; Guo, X.; He, J.; Hwu, C.-M.; Kalyani, R. R.; Kaplan, R.; Kardia, S. L. R.; Kooperberg, C.; Loos, R. J. F.; Lubitz, S. A.; Minster, R. L.; Mitchell, B. D.; Murabito, J. M.; Palmer, N. D.; Psaty, B. M.; Redline, S.; Shoemaker, M. B.; Silverman, E. K.; Telen, M. J.; Weiss, S. T.; Yanek, L. R.; Zhou, H.; NH\nScore: 28.0, Published: 2023-11-20 DOI: 10.1101/2023.11.19.566520\nThe role of rare non-coding variation in complex human phenotypes is still largely unknown. To elucidate the impact of rare variants in regulatory elements, we performed a whole-genome sequencing association analysis for height using 333,100 individuals from three datasets: UK Biobank (N=200,003), TOPMed (N=87,652) and All of Us (N=45,445). We performed rare (\u003c0.1% minor-allele-frequency) single-variant and aggregate testing of non-coding variants in regulatory regions based on proximal, intergenic and deep-intronic annotation. We observed 29 independent variants associated with height at P \u003c 6 x 10-10 after conditioning on previously reported variants, with effect sizes ranging from -7cm to +4.7cm. We also identified and replicated non-coding aggregate-based associations proximal to HMGA1 containing variants associated with a 5cm taller height and of highly-conserved variants in MIR497HG on chromosome 17. We have developed a novel approach for identifying non-coding rare variants in regulatory regions with large effects from whole-genome sequencing data associated with complex traits.\nPolyglucosan body density in the aged mouse hippocampus is controlled by a novel modifier locus on chromosome 1\nAuthors: Gomez-Pascual, A.; Glikman, D. M.; Ng, H. X.; Tomkins, J. E.; Lu, L.; Xu, Y.; Ashbrook, D. G.; Kaczorowski, C.; Kempermann, G.; Killmar, J.; Mozhui, K.; Aebersold, R.; Williams, E. G.; Williams, R. W.; Overall, R. W.; Jucker, M.; de Bakker, D. E. M.\nScore: 6.5, Published: 2023-11-22 DOI: 10.1101/2023.11.22.567373\nAging can be associated with the accumulation of hypobranched glycogen molecules (polyglucosan bodies, PGBs), particularly in astrocytes of the hippocampus. While PGBs have a detrimental effect on cognition in diseases such as adult polyglucosan body disease and Lafora disease, the underlying mechanism and clinical relevance of age-related PGB accumulation remains unknown. Here, we have investigated the genetic basis and functional impact of age-related PGB accumulation in 32 fully sequenced BXD-type strains of mice which exhibit a 400-fold variation in PGB burden in 16-18 month old females. We mapped a major locus controlling PGB density in the hippocampus to chromosome 1 at 72-75 Mb (linkage of 4.9 -logP), which we defined as the Pgb1 locus. To identify potentially causal gene variants within Pgb1, we generated extensive hippocampal transcriptome datasets and identified two strong candidate genes for which mRNA correlates with PGB density--Smarcal1 and Usp37. In addition, both Smarcal1 and Usp37 contain non-synonymous allele variations likely to impact protein function. A phenome-wide association analysis highlighted a trans-regulatory effect of the Pgb1 locus on expression of Hp1bp3, a gene known to play a role in age-related changes in learning and memory. To investigate the potential impact of PGBs on cognition, we performed conditioned fear memory testing on strains displaying varying degrees of PGB burden, and a phenome-wide association scan of ~12,000 traits. Importantly, we did not find any evidence suggesting a negative impact of PGB burden on cognitive capacity. Taken together, we have identified a major modifier locus controlling PGB burden in the hippocampus and shed light on the genetic architecture and clinical relevance of this strikingly heterogeneous hippocampal phenotype.\nAccounting for nuclear and mito genome in dairy cattle breeding - a simulation study\nAuthors: Mafra Fortuna, G.; Zumbach, B. J.; Johnsson, M.; Pocrnic, I.; Gorjanc, G.\nScore: 5.9, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567907\nMitochondria play a significant role in numerous cellular processes through proteins encoded by both nuclear genome (nDNA) and mito genome (mDNA). While the variation in nDNA is influenced by mutations and recombination of parental genomes, the variation in mDNA is solely driven by mutations. In addition, mDNA is inherited in a haploid form, from the dam. Cattle populations show significant variation in mDNA between and within breeds. Past research suggests that variation in mDNA accounts for 1-5% of the phenotypic variation in dairy traits. Here we simulated a dairy cattle breeding program to assess the impact of accounting for mDNA variation in pedigree-based and genome-based genetic evaluations on the accuracy of estimated breeding values for mDNA and nDNA components. We also examined the impact of alternative definitions of breeding values on genetic gain, including nDNA and mDNA components that both impact phenotype expression, but mDNA is inherited only maternally. We found that accounting for mDNA variation increased accuracy between +0.01 and +0.05 for different categories of animals, especially for young bulls (+0.05) and females without genotype data (between +0.01 and +0.03). Different scenarios of modelling and breeding value definition impacted genetic gain. The standard approach of ignoring mDNA variation achieved competitive genetic gain. Modelling, but not selecting on mDNA expectedly reduced genetic gain, while optimal use of mDNA variation recovered the genetic gain.\nMisexpression of inactive genes in whole blood is associated with nearby rare structural variants\nAuthors: Vanderstichele, T.; Burnham, K. L.; de Klein, N.; Tardaguila, M.; Howell, B.; Walter, K.; Kundu, K.; Koeppel, J.; Lee, W.; Tokolyi, A.; Persyn, E.; Nath, A. P.; Marten, J.; Petrovski, S.; Roberts, D. J.; Di Angelantonio, E.; Danesh, J.; Berton, A.; Platt, A.; Butterworth, A. S.; Soranzo, N.; Parts, L.; Inouye, M.; Paul, D. S.; Davenport, E. E.\nScore: 32.6, Published: 2023-11-19 DOI: 10.1101/2023.11.17.567537\nGene misexpression is the aberrant transcription of a gene in a context where it is usually inactive. Despite its known pathological consequences in specific rare diseases, we have a limited understanding of its wider prevalence and mechanisms in humans. To address this, we analyzed gene misexpression in 4,568 whole blood bulk RNA sequencing samples from INTERVAL study blood donors. We found that while individual misexpression events occur rarely, in aggregate they were found in almost all samples and over half of inactive genes. Using 2,821 paired whole genome and RNA sequencing samples, we identified that misexpression events are enriched in cis for rare structural variants. We established putative mechanisms through which a subset of SVs lead to gene misexpression, including transcriptional readthrough, transcript fusions and gene inversion. Overall, we develop misexpression as a novel type of transcriptomic outlier analysis and extend our understanding of the variety of mechanisms by which genetic variants can influence gene expression.\nRepeat mediated excision of gene drive elements for restoring wild-type populations\nAuthors: Chennuri, P. R.; Zapletal, J.; Monfardini, R. D.; Ndeffo-Mbah, M. L.; Adelman, Z.; Myles, K. M. M.\nScore: 4.3, Published: 2023-11-26 DOI: 10.1101/2023.11.23.568397\nWe demonstrate here that single strand annealing (SSA) repair can be co-opted for the precise autocatalytic excision of a drive element. Although SSA is not the predominant form of DNA repair in eukaryotic organisms, we increased the likelihood of its use by engineering direct repeats at sites flanking the drive allele, and then introducing a double-strand DNA break (DSB) at a second endonuclease target site encoded within the drive allele. We have termed this technology Repeat Mediated Excision of a Drive Element (ReMEDE). Incorporation of ReMEDE into the previously described mutagenic chain reaction (MCR) gene drive, targeting the yellow gene of Drosophila melanogaster, replaced drive alleles with wild-type alleles demonstrating proof-of-principle. Although the ReMEDE system requires further research and development, the technology has a number of attractive features as a gene drive mitigation strategy, chief among these the potential to restore a wild-type population without releasing additional transgenic organisms or large-scale environmental engineering efforts.\nMechanisms of transcriptional regulation in Anopheles gambiae revealed by allele specific expression\nAuthors: Dyer, N. A.; Lucas, E. R.; Nagi, S. C.; McDermott, D. P.; Brenas, J. H.; Miles, A.; Clarkson, C. S.; Mawejje, H. D.; Wilding, C. S.; Halfon, M. S.; Asma, H.; Heinz, E.; Donnelly, M. J.\nScore: 4.3, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568226\nMalaria control relies on insecticides targeting the mosquito vector, but this is increasingly compromised by insecticide resistance, which can be achieved by elevated expression of detoxifying enzymes that metabolize the insecticide. In diploid organisms, gene expression is regulated both in cis, by regulatory sequences on the same chromosome, and by trans acting factors, affecting both alleles equally. Differing levels of transcription can be caused by mutations in cis-regulatory modules (CRM), but few of these have been identified in mosquitoes. We crossed bendiocarb resistant and susceptible Anopheles gambiae strains to identify cis-regulated genes that might be responsible for the resistant phenotype using RNAseq, and cis-regulatory module sequences controlling gene expression in insecticide resistance relevant tissues were predicted using machine learning. We found 115 genes showing allele specific expression in hybrids of insecticide susceptible and resistant strains, suggesting cis regulation is an important mechanism of gene expression regulation in Anopheles gambiae. The genes showing allele specific expression included a higher proportion of Anopheles specific genes on average younger than genes those with balanced allelic expression.\nGenetic Discovery Enabled by A Large Language Model\nAuthors: Tu, T.; Fang, Z.; Cheng, Z.; Spasic, S.; Palepu, A.; Stankovic, K.; Natarajan, V.; Peltz, G.\nScore: 29.4, Published: 2023-11-12 DOI: 10.1101/2023.11.09.566468\nArtificial intelligence (AI) has been used in many areas of medicine, and recently large language models (LLMs) have shown potential utility for clinical applications. However, since we do not know if the use of LLMs can accelerate the pace of genetic discovery, we used data generated from mouse genetic models to investigate this possibility. We examined whether a recently developed specialized LLM (Med-PaLM 2) could analyze sets of candidate genes generated from analysis of murine models of biomedical traits. In response to free-text input, Med-PaLM 2 correctly identified the murine genes that contained experimentally verified causative genetic factors for six biomedical traits, which included susceptibility to diabetes and cataracts. Med-PaLM 2 was also able to analyze a list of genes with high impact alleles, which were identified by comparative analysis of murine genomic sequence data, and it identified a causative murine genetic factor for spontaneous hearing loss. Based upon this Med-PaLM 2 finding, a novel bigenic model for susceptibility to spontaneous hearing loss was developed. These results demonstrate Med-PaLM 2 can analyze gene-phenotype relationships and generate novel hypotheses, which can facilitate genetic discovery.\nA genome-wide association study of neonatal metabolites\nAuthors: He, Q.; Liu, H.; Lu, L.; Zhang, Q.; Wang, Q.; Wang, B.; Wu, X.; Guan, L.; Mao, J.; Xue, Y.; Zhang, C.; He, Y.; Peng, X.; Peng, H.; Zhao, K.; Li, H.; Jin, X.; Zhao, L.; Zhang, J.; Wang, T.\nScore: 3.8, Published: 2023-11-27 DOI: 10.1101/2023.11.25.568687\nThe hereditary component significantly influences the concentration of metabolites in adults. Nevertheless, the precise influence of genetic factors on neonatal metabolites remains uncertain. To bridge this gap, we employed genotype imputation techniques on large-scale low-pass genome data obtained from non-invasive prenatal testing. Subsequently, we conducted association studies on a total of 75 metabolic components in neonates. The study identified a total 17 previous reported associations and 13 novel discovered associations between single nucleotide polymorphisms and metabolic components. These associations were initially found in the discovery cohort (8,744 participants) and subsequently confirmed in a replication cohort (19,041 participants). The average heritability of metabolic components was calculated to be 76.2%, with a range of 69-78.8%. The aforementioned findings offer valuable insights pertaining to the genetic architecture of neonatal metabolism.\nHeritability of complex traits in sub-populations experiencing bottlenecks and growth\nAuthors: Taylor, C. S.; Lawson, D. J.\nScore: 3.5, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568467\n1Populations that have experienced a bottleneck are regularly used in Genome Wide Association Studies (GWAS) to investigate variants associated with complex traits. It is generally understood that these isolated sub-populations may experience high frequency of otherwise rare variants with large effect size, and therefore provide a unique opportunity to study said trait. However, the demographic history of the population under investigation affects all SNPs that determine the complex trait genome-wide, changing its heritability and genetic architecture. We use a simulation-based approach to identify the impact of the demographic processes of drift, expansion, and migration on the heritability of complex trait. We show that demography has considerable impact on complex traits. We then investigate the power to resolve heritability of complex traits in GWAS studies subjected to demographic effects. We find that demography is an important component for interpreting inference of complex traits and has a nuanced impact on the power of GWAS. We conclude that demographic histories need to be explicitly modelled in order to properly quantify the history of selection on a complex trait.\n",
  "wordCount" : "2326",
  "inLanguage": "en",
  "datePublished": "2023-11-29T10:40:14Z",
  "dateModified": "2023-11-29T10:40:14Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 29, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568146">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568146" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568146">
        <p class="paperTitle">Multi-Step Control of Homologous Recombination by Mec1/ATR Ensures Robust Suppression of Gross Chromosomal Rearrangements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568146" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568146" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xie, B.; Sanford, E. J.; Hung, S.-H.; Wagner, M. M.; Heyer, W.-D.; Smolka, M. B.</p>
        <p class="info">Score: 15.5, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568146' target='https://doi.org/10.1101/2023.11.21.568146'> 10.1101/2023.11.21.568146</a></p>
        <p class="abstract">The Mec1/ATR kinase is crucial for genome stability, yet the mechanism by which it prevents gross chromosomal rearrangements (GCRs) remains unknown. Here we find that in cells with deficient Mec1 signaling, GCRs accumulate due to the deregulation of multiple steps in homologous recombination (HR). Mec1 primarily suppresses GCRs through its role in activating the canonical checkpoint kinase Rad53, which ensures the proper control of DNA end resection. Upon loss of Rad53 signaling and resection control, Mec1 becomes hyperactivated and triggers a salvage pathway in which the Sgs1 helicase is recruited to sites of DNA lesions via the 911-Dpb11 scaffolds to favor heteroduplex rejection and limit HR-driven GCR accumulation. Fusing an ssDNA recognition domain to Sgs1 bypasses the requirement of Mec1 signaling for GCR suppression and nearly eliminates D-loop formation, thus preventing non-allelic recombination events. We propose that Mec1 regulates multiple steps of HR to prevent GCRs while ensuring balanced HR usage when needed for promoting tolerance to replication stress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.566520">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.566520" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.566520">
        <p class="paperTitle">Whole genome association testing in 333,100 individuals across three biobanks identifies rare non-coding single variant and genomic aggregate associations with height</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.566520" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.566520" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hawkes, G.; Beaumont, R. N.; Li, Z.; Mandla, R.; Li, X.; Albert, C. M.; Arnett, D. K.; Ashley-Koch, A. E.; Ashrani, A. A.; Barnes, K. C.; Boerwinkle, E.; Brody, J. A.; Carson, A. P.; Chami, N.; Chen, Y.-D. I.; Chung, M. K.; Curran, J. E.; Darbar, D.; Ellinor, P. T.; Fornage, M.; Gordeuk, V. R.; Guo, X.; He, J.; Hwu, C.-M.; Kalyani, R. R.; Kaplan, R.; Kardia, S. L. R.; Kooperberg, C.; Loos, R. J. F.; Lubitz, S. A.; Minster, R. L.; Mitchell, B. D.; Murabito, J. M.; Palmer, N. D.; Psaty, B. M.; Redline, S.; Shoemaker, M. B.; Silverman, E. K.; Telen, M. J.; Weiss, S. T.; Yanek, L. R.; Zhou, H.; NH</p>
        <p class="info">Score: 28.0, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.566520' target='https://doi.org/10.1101/2023.11.19.566520'> 10.1101/2023.11.19.566520</a></p>
        <p class="abstract">The role of rare non-coding variation in complex human phenotypes is still largely unknown. To elucidate the impact of rare variants in regulatory elements, we performed a whole-genome sequencing association analysis for height using 333,100 individuals from three datasets: UK Biobank (N=200,003), TOPMed (N=87,652) and All of Us (N=45,445). We performed rare (&lt;0.1% minor-allele-frequency) single-variant and aggregate testing of non-coding variants in regulatory regions based on proximal, intergenic and deep-intronic annotation. We observed 29 independent variants associated with height at P &lt; 6 x 10-10 after conditioning on previously reported variants, with effect sizes ranging from -7cm to &#43;4.7cm. We also identified and replicated non-coding aggregate-based associations proximal to HMGA1 containing variants associated with a 5cm taller height and of highly-conserved variants in MIR497HG on chromosome 17. We have developed a novel approach for identifying non-coding rare variants in regulatory regions with large effects from whole-genome sequencing data associated with complex traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.567373">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.567373" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.567373">
        <p class="paperTitle">Polyglucosan body density in the aged mouse hippocampus is controlled by a novel modifier locus on chromosome 1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.567373" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.567373" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gomez-Pascual, A.; Glikman, D. M.; Ng, H. X.; Tomkins, J. E.; Lu, L.; Xu, Y.; Ashbrook, D. G.; Kaczorowski, C.; Kempermann, G.; Killmar, J.; Mozhui, K.; Aebersold, R.; Williams, E. G.; Williams, R. W.; Overall, R. W.; Jucker, M.; de Bakker, D. E. M.</p>
        <p class="info">Score: 6.5, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.567373' target='https://doi.org/10.1101/2023.11.22.567373'> 10.1101/2023.11.22.567373</a></p>
        <p class="abstract">Aging can be associated with the accumulation of hypobranched glycogen molecules (polyglucosan bodies, PGBs), particularly in astrocytes of the hippocampus. While PGBs have a detrimental effect on cognition in diseases such as adult polyglucosan body disease and Lafora disease, the underlying mechanism and clinical relevance of age-related PGB accumulation remains unknown. Here, we have investigated the genetic basis and functional impact of age-related PGB accumulation in 32 fully sequenced BXD-type strains of mice which exhibit a 400-fold variation in PGB burden in 16-18 month old females. We mapped a major locus controlling PGB density in the hippocampus to chromosome 1 at 72-75 Mb (linkage of 4.9 -logP), which we defined as the Pgb1 locus. To identify potentially causal gene variants within Pgb1, we generated extensive hippocampal transcriptome datasets and identified two strong candidate genes for which mRNA correlates with PGB density--Smarcal1 and Usp37. In addition, both Smarcal1 and Usp37 contain non-synonymous allele variations likely to impact protein function. A phenome-wide association analysis highlighted a trans-regulatory effect of the Pgb1 locus on expression of Hp1bp3, a gene known to play a role in age-related changes in learning and memory. To investigate the potential impact of PGBs on cognition, we performed conditioned fear memory testing on strains displaying varying degrees of PGB burden, and a phenome-wide association scan of ~12,000 traits. Importantly, we did not find any evidence suggesting a negative impact of PGB burden on cognitive capacity. Taken together, we have identified a major modifier locus controlling PGB burden in the hippocampus and shed light on the genetic architecture and clinical relevance of this strikingly heterogeneous hippocampal phenotype.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567907">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567907" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567907">
        <p class="paperTitle">Accounting for nuclear and mito genome in dairy cattle breeding - a simulation study</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567907" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567907" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mafra Fortuna, G.; Zumbach, B. J.; Johnsson, M.; Pocrnic, I.; Gorjanc, G.</p>
        <p class="info">Score: 5.9, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567907' target='https://doi.org/10.1101/2023.11.20.567907'> 10.1101/2023.11.20.567907</a></p>
        <p class="abstract">Mitochondria play a significant role in numerous cellular processes through proteins encoded by both nuclear genome (nDNA) and mito genome (mDNA). While the variation in nDNA is influenced by mutations and recombination of parental genomes, the variation in mDNA is solely driven by mutations. In addition, mDNA is inherited in a haploid form, from the dam. Cattle populations show significant variation in mDNA between and within breeds. Past research suggests that variation in mDNA accounts for 1-5% of the phenotypic variation in dairy traits. Here we simulated a dairy cattle breeding program to assess the impact of accounting for mDNA variation in pedigree-based and genome-based genetic evaluations on the accuracy of estimated breeding values for mDNA and nDNA components. We also examined the impact of alternative definitions of breeding values on genetic gain, including nDNA and mDNA components that both impact phenotype expression, but mDNA is inherited only maternally. We found that accounting for mDNA variation increased accuracy between &#43;0.01 and &#43;0.05 for different categories of animals, especially for young bulls (&#43;0.05) and females without genotype data (between &#43;0.01 and &#43;0.03). Different scenarios of modelling and breeding value definition impacted genetic gain. The standard approach of ignoring mDNA variation achieved competitive genetic gain. Modelling, but not selecting on mDNA expectedly reduced genetic gain, while optimal use of mDNA variation recovered the genetic gain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567537">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567537" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567537">
        <p class="paperTitle">Misexpression of inactive genes in whole blood is associated with nearby rare structural variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567537" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567537" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vanderstichele, T.; Burnham, K. L.; de Klein, N.; Tardaguila, M.; Howell, B.; Walter, K.; Kundu, K.; Koeppel, J.; Lee, W.; Tokolyi, A.; Persyn, E.; Nath, A. P.; Marten, J.; Petrovski, S.; Roberts, D. J.; Di Angelantonio, E.; Danesh, J.; Berton, A.; Platt, A.; Butterworth, A. S.; Soranzo, N.; Parts, L.; Inouye, M.; Paul, D. S.; Davenport, E. E.</p>
        <p class="info">Score: 32.6, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567537' target='https://doi.org/10.1101/2023.11.17.567537'> 10.1101/2023.11.17.567537</a></p>
        <p class="abstract">Gene misexpression is the aberrant transcription of a gene in a context where it is usually inactive. Despite its known pathological consequences in specific rare diseases, we have a limited understanding of its wider prevalence and mechanisms in humans. To address this, we analyzed gene misexpression in 4,568 whole blood bulk RNA sequencing samples from INTERVAL study blood donors. We found that while individual misexpression events occur rarely, in aggregate they were found in almost all samples and over half of inactive genes. Using 2,821 paired whole genome and RNA sequencing samples, we identified that misexpression events are enriched in cis for rare structural variants. We established putative mechanisms through which a subset of SVs lead to gene misexpression, including transcriptional readthrough, transcript fusions and gene inversion. Overall, we develop misexpression as a novel type of transcriptomic outlier analysis and extend our understanding of the variety of mechanisms by which genetic variants can influence gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568397">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568397" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568397">
        <p class="paperTitle">Repeat mediated excision of gene drive elements for restoring wild-type populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568397" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568397" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chennuri, P. R.; Zapletal, J.; Monfardini, R. D.; Ndeffo-Mbah, M. L.; Adelman, Z.; Myles, K. M. M.</p>
        <p class="info">Score: 4.3, Published: 2023-11-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568397' target='https://doi.org/10.1101/2023.11.23.568397'> 10.1101/2023.11.23.568397</a></p>
        <p class="abstract">We demonstrate here that single strand annealing (SSA) repair can be co-opted for the precise autocatalytic excision of a drive element. Although SSA is not the predominant form of DNA repair in eukaryotic organisms, we increased the likelihood of its use by engineering direct repeats at sites flanking the drive allele, and then introducing a double-strand DNA break (DSB) at a second endonuclease target site encoded within the drive allele. We have termed this technology Repeat Mediated Excision of a Drive Element (ReMEDE). Incorporation of ReMEDE into the previously described mutagenic chain reaction (MCR) gene drive, targeting the yellow gene of Drosophila melanogaster, replaced drive alleles with wild-type alleles demonstrating proof-of-principle. Although the ReMEDE system requires further research and development, the technology has a number of attractive features as a gene drive mitigation strategy, chief among these the potential to restore a wild-type population without releasing additional transgenic organisms or large-scale environmental engineering efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568226">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568226" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568226">
        <p class="paperTitle">Mechanisms of transcriptional regulation in Anopheles gambiae revealed by allele specific expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568226" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568226" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dyer, N. A.; Lucas, E. R.; Nagi, S. C.; McDermott, D. P.; Brenas, J. H.; Miles, A.; Clarkson, C. S.; Mawejje, H. D.; Wilding, C. S.; Halfon, M. S.; Asma, H.; Heinz, E.; Donnelly, M. J.</p>
        <p class="info">Score: 4.3, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568226' target='https://doi.org/10.1101/2023.11.22.568226'> 10.1101/2023.11.22.568226</a></p>
        <p class="abstract">Malaria control relies on insecticides targeting the mosquito vector, but this is increasingly compromised by insecticide resistance, which can be achieved by elevated expression of detoxifying enzymes that metabolize the insecticide. In diploid organisms, gene expression is regulated both in cis, by regulatory sequences on the same chromosome, and by trans acting factors, affecting both alleles equally. Differing levels of transcription can be caused by mutations in cis-regulatory modules (CRM), but few of these have been identified in mosquitoes. We crossed bendiocarb resistant and susceptible Anopheles gambiae strains to identify cis-regulated genes that might be responsible for the resistant phenotype using RNAseq, and cis-regulatory module sequences controlling gene expression in insecticide resistance relevant tissues were predicted using machine learning. We found 115 genes showing allele specific expression in hybrids of insecticide susceptible and resistant strains, suggesting cis regulation is an important mechanism of gene expression regulation in Anopheles gambiae. The genes showing allele specific expression included a higher proportion of Anopheles specific genes on average younger than genes those with balanced allelic expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566468">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566468" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566468">
        <p class="paperTitle">Genetic Discovery Enabled by A Large Language Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566468" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566468" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tu, T.; Fang, Z.; Cheng, Z.; Spasic, S.; Palepu, A.; Stankovic, K.; Natarajan, V.; Peltz, G.</p>
        <p class="info">Score: 29.4, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566468' target='https://doi.org/10.1101/2023.11.09.566468'> 10.1101/2023.11.09.566468</a></p>
        <p class="abstract">Artificial intelligence (AI) has been used in many areas of medicine, and recently large language models (LLMs) have shown potential utility for clinical applications. However, since we do not know if the use of LLMs can accelerate the pace of genetic discovery, we used data generated from mouse genetic models to investigate this possibility. We examined whether a recently developed specialized LLM (Med-PaLM 2) could analyze sets of candidate genes generated from analysis of murine models of biomedical traits. In response to free-text input, Med-PaLM 2 correctly identified the murine genes that contained experimentally verified causative genetic factors for six biomedical traits, which included susceptibility to diabetes and cataracts. Med-PaLM 2 was also able to analyze a list of genes with high impact alleles, which were identified by comparative analysis of murine genomic sequence data, and it identified a causative murine genetic factor for spontaneous hearing loss. Based upon this Med-PaLM 2 finding, a novel bigenic model for susceptibility to spontaneous hearing loss was developed. These results demonstrate Med-PaLM 2 can analyze gene-phenotype relationships and generate novel hypotheses, which can facilitate genetic discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.568687">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.568687" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.568687">
        <p class="paperTitle">A genome-wide association study of neonatal metabolites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.568687" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.568687" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, Q.; Liu, H.; Lu, L.; Zhang, Q.; Wang, Q.; Wang, B.; Wu, X.; Guan, L.; Mao, J.; Xue, Y.; Zhang, C.; He, Y.; Peng, X.; Peng, H.; Zhao, K.; Li, H.; Jin, X.; Zhao, L.; Zhang, J.; Wang, T.</p>
        <p class="info">Score: 3.8, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.568687' target='https://doi.org/10.1101/2023.11.25.568687'> 10.1101/2023.11.25.568687</a></p>
        <p class="abstract">The hereditary component significantly influences the concentration of metabolites in adults. Nevertheless, the precise influence of genetic factors on neonatal metabolites remains uncertain. To bridge this gap, we employed genotype imputation techniques on large-scale low-pass genome data obtained from non-invasive prenatal testing. Subsequently, we conducted association studies on a total of 75 metabolic components in neonates. The study identified a total 17 previous reported associations and 13 novel discovered associations between single nucleotide polymorphisms and metabolic components. These associations were initially found in the discovery cohort (8,744 participants) and subsequently confirmed in a replication cohort (19,041 participants). The average heritability of metabolic components was calculated to be 76.2%, with a range of 69-78.8%. The aforementioned findings offer valuable insights pertaining to the genetic architecture of neonatal metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568467">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568467" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568467">
        <p class="paperTitle">Heritability of complex traits in sub-populations experiencing bottlenecks and growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568467" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568467" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, C. S.; Lawson, D. J.</p>
        <p class="info">Score: 3.5, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568467' target='https://doi.org/10.1101/2023.11.23.568467'> 10.1101/2023.11.23.568467</a></p>
        <p class="abstract">1Populations that have experienced a bottleneck are regularly used in Genome Wide Association Studies (GWAS) to investigate variants associated with complex traits. It is generally understood that these isolated sub-populations may experience high frequency of otherwise rare variants with large effect size, and therefore provide a unique opportunity to study said trait. However, the demographic history of the population under investigation affects all SNPs that determine the complex trait genome-wide, changing its heritability and genetic architecture. We use a simulation-based approach to identify the impact of the demographic processes of drift, expansion, and migration on the heritability of complex trait. We show that demography has considerable impact on complex traits. We then investigate the power to resolve heritability of complex traits in GWAS studies subjected to demographic effects. We find that demography is an important component for interpreting inference of complex traits and has a nuanced impact on the power of GWAS. We conclude that demographic histories need to be explicitly modelled in order to properly quantify the history of selection on a complex trait.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
