<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The simplicity of protein sequence-function relationships
Authors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.
Score: 30.5, Published: 2023-09-05 DOI: 10.1101/2023.09.02.556057
How complicated is the relationship between a proteins sequence and its function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most prior studies, however, used methods that misinterpret measurement errors, small local idiosyncracies around a designated wild-type sequence, and global nonlinearity in the sequence-function relationship as rampant high-order interactions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="The simplicity of protein sequence-function relationships
Authors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.
Score: 30.5, Published: 2023-09-05 DOI: 10.1101/2023.09.02.556057
How complicated is the relationship between a proteins sequence and its function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most prior studies, however, used methods that misinterpret measurement errors, small local idiosyncracies around a designated wild-type sequence, and global nonlinearity in the sequence-function relationship as rampant high-order interactions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-13T10:39:03+00:00" />
<meta property="article:modified_time" content="2023-09-13T10:39:03+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="The simplicity of protein sequence-function relationships
Authors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.
Score: 30.5, Published: 2023-09-05 DOI: 10.1101/2023.09.02.556057
How complicated is the relationship between a proteins sequence and its function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most prior studies, however, used methods that misinterpret measurement errors, small local idiosyncracies around a designated wild-type sequence, and global nonlinearity in the sequence-function relationship as rampant high-order interactions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "The simplicity of protein sequence-function relationships\nAuthors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.\nScore: 30.5, Published: 2023-09-05 DOI: 10.1101/2023.09.02.556057\nHow complicated is the relationship between a proteins sequence and its function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most prior studies, however, used methods that misinterpret measurement errors, small local idiosyncracies around a designated wild-type sequence, and global nonlinearity in the sequence-function relationship as rampant high-order interactions.",
  "keywords": [
    
  ],
  "articleBody": " The simplicity of protein sequence-function relationships\nAuthors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.\nScore: 30.5, Published: 2023-09-05 DOI: 10.1101/2023.09.02.556057\nHow complicated is the relationship between a proteins sequence and its function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most prior studies, however, used methods that misinterpret measurement errors, small local idiosyncracies around a designated wild-type sequence, and global nonlinearity in the sequence-function relationship as rampant high-order interactions. Here we present a simple new method to jointly estimate global nonlinearity and specific epistatic interactions across a proteins genotype-phenotype map. Our reference-free approach calculates the effect of each amino acid state or combination by averaging over all genotypes that contain it relative to the global average. We show that this method is more accurate than any alternative approach and is robust to measurement error and partial sampling. We reanalyze 20 combinatorial mutagenesis experiments and find that main and pairwise effects, together with a simple form of global nonlinearity, account for a median of 96% of total variance in the measured phenotype (and \u003e 92% in every case), and only a tiny fraction of genotypes are strongly affected by epistasis at third or higher orders. The genetic architecture is also sparse: the number of model terms required to explain the vast majority of phenotypic variance is smaller than the number of genotypes by many orders of magnitude. The sequence-function relationship in most proteins is therefore far simpler than previously thought, and new, more tractable experimental approaches, combined with reference-free analysis, may be sufficient to explain it in most cases. Significance StatementIt is widely thought that a proteins function depends on complex interactions among amino acids. If so, it would be virtually impossible to predict the function of new variants, and understanding how proteins work genetically and biochemically would require huge combinatorial experiments. We show that prior studies overestimated complexity because they analyzed sequence-function relationships from the perspective of a single reference genotype and/or misinterpreted global phenotypic nonlinearities as complex amino acid interactions. By developing a new reference-free approach and using it to reanalyze 20 experimental datasets, we show that additive effects and pairwise interactions alone, along with a simple global nonlinearity, explain the vast majority of functional variation. Higher-order interactions are weak or rare, and a minuscule fraction of possible interactions shape each proteins function. Our work reveals that protein sequence-function relationships are surprisingly simple and suggests new strategies that are far more tractable than the massive experiments currently used.\nExpression Level Analysis of ACE2 Receptor Gene in African-American and Non-African-American COVID-19 Patients\nAuthors: Nyamari, M. N.; Omar, K. M.; Fayehun, A. F.; Dachi, O.; Bwana, B. K.; Awe, O. I.\nScore: 13.2, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557129\nBackground: The COVID-19 pandemic caused by SARS-CoV-2 has spread rapidly across the continents. While the incidence of COVID-19 has been reported to be higher among African-American individuals, the rate of mortality has been lower compared to that of non-African-Americans. ACE2 is involved in COVID-19 as SARS-CoV-2 uses the ACE2 enzyme to enter host cells. Although the difference in COVID-19 incidence can be explained by many factors such as low accessibility of health insurance among the African-American community, little is known about ACE2 expression in African-American COVID-19 patients compared to non-African-American COVID-19 patients. The variable expression of genes can contribute to this observed phenomenon. Methodology: In this study, transcriptomes from African-American and non-African-American COVID-19 patients were retrieved from the sequence read archive and analyzed for ACE2 gene expression. HISAT2 was used to align the reads to the human reference genome, and HTseq-count was used to get raw gene counts. EdgeR was utilized for differential gene expression analysis, and enrichR was employed for gene enrichment analysis. Results: The datasets included 14 and 33 transcriptome sequences from COVID-19 patients of African-American and non-African-American descent, respectively. There were 24,092 differentially expressed genes, with 7,718 upregulated (log fold change \u003e 1 and FDR 0.05) and 16,374 downregulated (log fold change -1 and FDR 0.05). The ACE2 mRNA level was found to be considerably downregulated in the African-American cohort (p-value = 0.0242, p-adjusted value = 0.038). Conclusion: The downregulation of ACE2 in the African-American cohort could indicate a correlation to the low COVID-19 severity observed among the African-American community.\nGenetic manipulation of an Ixodes scapularis cell line\nAuthors: Singh, N.; Rolandelli, A.; ONeal, A. J.; Butler, L. R.; Samaddar, S.; Laukaitis-Yousey, H.; Butnaru, M.; Mohr, S. E.; Perrimon, N.; Pedra, J. H.\nScore: 11.0, Published: 2023-09-10 DOI: 10.1101/2023.09.08.556855\nAlthough genetic manipulation is one of the hallmarks in model organisms, its applicability to non-model species has remained difficult due to our limited understanding of their fundamental biology. For instance, manipulation of a cell line originated from the blacklegged tick Ixodes scapularis, an arthropod that serves as a vector of several human pathogens, has yet to be established. Here, we demonstrate the successful genetic modification of the commonly used tick ISE6 line through ectopic expression and clustered regularly interspaced palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9) genome editing. We performed ectopic expression using nucleofection and attained CRISPR-Cas9 editing via homology dependent recombination. Targeting the E3 ubiquitin ligase X-linked inhibitor of apoptosis (xiap) and its substrate p47 led to alteration in molecular signaling within the immune deficiency (IMD) network and increased infection of the rickettsial agent Anaplasma phagocytophilum in I. scapularis ISE6 cells. Collectively, our findings complement techniques for genetic engineering of ticks in vivo and aid in circumventing the long-life cycle of I. scapularis, of which limits efficient and scalable molecular genetic screens.\nProteome-scale prediction of molecular mechanisms underlying dominant genetic diseases\nAuthors: Badonyi, M.; Marsh, J. A.\nScore: 9.5, Published: 2023-09-08 DOI: 10.1101/2023.09.08.556798\nMany dominant genetic disorders result from protein-altering mutations, acting primarily through dominant-negative (DN), gain-of-function (GOF), and loss-of-function (LOF) mechanisms. Deciphering the mechanisms by which dominant diseases exert their effects is often experimentally challenging and resource intensive, but is essential for developing appropriate therapeutic approaches. Diseases that arise via a LOF mechanism are more amenable to be treated by conventional gene therapy, whereas DN and GOF mechanisms may require gene editing or targeting by small molecules. Moreover, pathogenic missense mutations that act via DN and GOF mechanisms are more difficult to identify than those that act via LOF using nearly all currently available variant effect predictors. Here, we introduce a tripartite statistical model made up of support vector machine binary classifiers trained to predict whether human protein-coding genes are likely to be associated with DN, GOF, or LOF molecular disease mechanisms. We test the utility of the predictions by examining biologically and clinically meaningful properties known to be associated with the mechanisms. Our results strongly support that the models are able to generalise on unseen data and offer insight into the functional attributes of proteins associated with different mechanisms. We hope that our predictions will serve as a springboard for researchers studying novel variants and those of uncertain clinical significance, guiding variant interpretation strategies and experimental characterisation. Predictions for the 2023_02 UniProt reference proteome are available at https://osf.io/z4dcp/.\nDisentangling sources of clock-like mutations in germline and soma\nAuthors: Spisak, N.; de Manuel, M.; Milligan, W. R.; Sella, G.; Przeworski, M.\nScore: 8.7, Published: 2023-09-12 DOI: 10.1101/2023.09.07.556720\nThe rates of mutations vary across cell types. To identify causes of this variation, mutations are often decomposed into a combination of the single base substitution (SBS) \"signatures\" observed in germline, soma and tumors, with the idea that each signature corresponds to one or a small number of underlying mutagenic processes. Two such signatures turn out to be ubiquitous across cell types: SBS signature 1, which consists primarily of transitions at methylated CpG sites caused by spontaneous deamination, and the more diffuse SBS signature 5, which is of unknown etiology. In cancers, the number of mutations attributed to these two signatures accumulates linearly with age of diagnosis, and thus the signatures have been termed \"clock-like.\" To better understand this clock-like behavior, we develop a mathematical model that includes DNA replication errors, unrepaired damage, and damage repaired incorrectly. We show that mutational signatures can exhibit clock-like behavior because cell divisions occur at a constant rate and/or because damage rates remain constant over time, and that these distinct sources can be teased apart by comparing cell lineages that divide at different rates. With this goal in mind, we analyze the rate of accumulation of mutations in multiple cell types, including soma as well as male and female germline. We find no detectable increase in SBS signature 1 mutations in neurons and only a very weak increase in mutations assigned to the female germline, but a significant increase with time in rapidly-dividing cells, suggesting that SBS signature 1 is driven by rounds of DNA replication occurring at a relatively fixed rate. In contrast, SBS signature 5 increases with time in all cell types, including post-mitotic ones, indicating that it accumulates independently of cell divisions; this observation points to errors in DNA repair as the key underlying mechanism. Thus, the two \"clock-like\" signatures observed across cell types likely have distinct origins, one set by rates of cell division, the other by damage rates.\nEvolutionary diversity of the control of the azole response by Tra1 across yeast species\nAuthors: Librais, G. M.; Jiang, Y.; Razzaq, I.; Brandl, C. J.; Shapiro, R. S.; Lajoie, P.\nScore: 6.4, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556458\nTra1 is an essential co-activator protein of the yeast SAGA and NuA4 acetyltransferase complexes that regulate gene expression through multiple mechanisms including the acetylation of histone proteins. Tra1 is a pseudokinase of the PIKK family characterized by a C-terminal PI3K domain with no known kinase activity. However, mutations of specific arginine residues to glutamine in the PI3K domains (an allele termed tra1Q3) result in reduced growth and increased sensitivity to multiple stresses. In the opportunistic fungal pathogen Candida albicans, the tra1Q3 allele reduces pathogenicity and increases sensitivity to the echinocandin antifungal drug caspofungin, which disrupts the fungal cell wall. Here, we found that loss of Tra1 function, in contrast to what is seen with caspofungin, increases tolerance to the azole class of antifungal drugs, which inhibits ergosterol synthesis. In C. albicans, tra1Q3 increases expression of genes linked to azole resistance, such as ERG11 and CDR1. CDR1 encodes a multidrug ABC transporter associated with efflux of multiple xenobiotics, including azoles. Consequently, cells carrying tra1Q3 show reduced intracellular accumulation of fluconazole. In contrast, a tra1Q3 S. cerevisiae strain displayed opposite phenotypes: decreased tolerance to azole, decreased expression of the efflux pump PDR5 and increased intracellular accumulation of fluconazole. Therefore, our data provide evidence that Tra1 differentially regulates the antifungal response across yeast species.\nSlave Trade and Colonial Expansion Resulted in Strong Sex-Biased Admixture in South Africa\nAuthors: Reynolds, A.; Schurz, H.; Meeks, G.; Gravel, S.; Myrick, J. W.; Mendez, F. L.; Werely, C.; van Helden, P. D.; Hoal, E. G.; Poznik, G. D.; Kim, M.; Uren, C.; Underhill, P. A.; Möller, M.; Henn, B. M.\nScore: 5.8, Published: 2023-09-11 DOI: 10.1101/2023.09.06.556626\nThe colonial-period arrival of Europeans in southern Africa is associated with strong sex-biased migration by which male settlers displaced indigenous Khoekhoe and San men. Simultaneously, the importation of South Asian, Indonesian and Eastern African slaves may have contributed female-biased migration to Cape Town and surrounding areas. We examine the spatial and temporal spread of sex-biased migration from the Cape northward into Namaqualand and the southern Kalahari using genetic data from more than 1,400 individuals. In all regions, admixture patterns were sex-biased, with evidence of a greater male contribution of European ancestry and greater female contribution of Khoe-San ancestry. While admixture among Khoe-San, European, equatorial African, and Asian groups has likely been continuous from the founding of Cape Town to present-day, we find that Khoe-San groups further north experienced a single pulse of European admixture 6-8 generations ago. European admixture was followed by additional Khoe-San gene flow, potentially reflecting an aggregation of indigenous groups due to disruption by colonial interlopers. Male migration into the northern frontier territories was not a homogenous group of expanding Afrikaners and slaves. The Nama show evidence of distinct founder effects and derive 15% of their male lineages from Asian men, a pattern absent in the =Khomani San. Khoe-San ancestry from the paternal line is greatly diminished in populations from Cape Town, the Cederberg Mountains and Upington, but remains more frequent in self-identified ethnically indigenous groups. Strikingly, we estimate that Khoe-San Y-chromosomes were experiencing unprecedented population growth at the time of European arrival. Our findings shed light on the patterns of admixture and the population history of South Africa as the colonial frontier expanded.\nFunctional characterization of gene regulatory elements and neuropsychiatric disease-associated risk loci in iPSCs and iPSC-derived neurons\nAuthors: Yang, X.; Jones, I. R.; Chen, P. B.; Yang, H.; Ren, X.; Zheng, L.; Li, B.; Li, Y. E.; Sun, Q.; Wen, J.; Beaman, C.; Cui, X.; Li, Y.; Wang, W.; Hu, M.; Ren, B.; Shen, Y.\nScore: 27.0, Published: 2023-08-30 DOI: 10.1101/2023.08.30.555359\nGenome-wide association studies (GWAS) have identified thousands of non-coding variants that contribute to psychiatric disease risks, likely by perturbing cis-regulatory elements (CREs). However, our ability to interpret and explore their mechanisms of action is hampered by a lack of annotation of functional CREs (fCREs) in neural cell types. Here, through genome-scale CRISPR screens of 22,000 candidate CREs (cCREs) in human induced pluripotent stem cells (iPSCs) undergoing differentiation to excitatory neurons, we identify 2,847 and 5,540 fCREs essential for iPSC fitness and neuronal differentiation, respectively. These fCREs display dynamic epigenomic features and exhibit increased numbers and genomic spans of chromatin interactions following terminal neuronal differentiation. Furthermore, fCREs essential for neuronal differentiation show significantly greater enrichment of genetic heritability for neurodevelopmental diseases including schizophrenia (SCZ), attention deficit hyperactivity disorder (ADHD), and autism spectrum disorders (ASD) than cCREs. Using high-throughput prime editing screens we experimentally confirm 45 SCZ risk variants that act by affecting the function of fCREs. The extensive and in-depth functional annotation of cCREs in neuronal types therefore provides a crucial resource for interpreting non-coding risk variants of neuropsychiatric disorders.\nGeneration of isogenic models of Angelman syndrome and Prader-Willi syndrome in CRISPR/Cas9-engineered human embryonic stem cells\nAuthors: Gilmore, R. B.; Gorka, D.; Stoddard, C.; Cotney, J.; Chamberlain, S.\nScore: 8.8, Published: 2023-08-30 DOI: 10.1101/2023.08.30.555563\nAngelman Syndrome (AS) and Prader-Willi Syndrome (PWS), two distinct neurodevelopmental disorders, result from loss of expression from imprinted genes in the chromosome 15q11-13 locus most commonly caused by a megabase-scale deletion on either the maternal or paternal allele, respectively. Each occurs at an approximate incidence of 1/15,000 to 1/30,000 live births and has a range of debilitating phenotypes. Patient-derived induced pluripotent stem cells (iPSCs) have been valuable tools to understand human-relevant gene regulation at this locus and have contributed to the development of therapeutic approaches for AS. Nonetheless, gaps remain in our understanding of how these deletions contribute to dysregulation and phenotypes of AS and PWS. Variability across cell lines due to donor differences, reprogramming methods, and genetic background make it challenging to fill these gaps in knowledge without substantially increasing the number of cell lines used in the analyses. Isogenic cell lines that differ only by the genetic mutation causing the disease can ease this burden without requiring such a large number of cell lines. Here, we describe the development of isogenic human embryonic stem cell (hESC) lines modeling the most common genetic subtypes of AS and PWS. These lines allow for a facile interrogation of allele-specific gene regulation at the chromosome 15q11-q13 locus. Additionally, these lines are an important resource to identify and test targeted therapeutic approaches for patients with AS and PWS.\nFemale germline expression of OVO transcription factor bridges Drosophila generations\nAuthors: Benner, L.; Muron, S.; Oliver, B.\nScore: 7.7, Published: 2023-08-26 DOI: 10.1101/2023.08.25.554887\nOVO is required for karyotypically female germ cell viability but has no known function in the male germline in Drosophila. ovo is autoregulated by two antagonistic isoforms, OVO-A and OVO-B. All ovo- alleles were created as partial revertants of the antimorphic ovoD1allele. Creation of new targeted alleles in an ovo+ background indicated that disrupting the germline-specific exon extension of ovo-B leads to an arrested egg chamber phenotype, rather than germ cell death. RNA-seq analysis, including \u003e1K full length cDNAs, indicates that ovo utilizes a number of unannotated splice variations in the extended exon and a minor population of ovo-B transcripts utilizes an alternative splice. This indicates that classical ovo alleles such as ovoD1rv23, are not truly null for ovo, and are likely to be weak antimorphs. To generate bonafide nulls, we deleted the ovo-A and ovo-B promoters showing that only ovo-B is required for female germ cell viability and there is an early and polyphasic developmental requirement for ovo-B in the female germline. To visualize OVO expression and localization, we endogenously tagged ovo and found nuclear OVO in all differentiating female germ cells throughout oogenesis in adults. We also found that OVO is maternally deposited into the embryo, where it showed nuclear localization in newly formed pole cells. Maternal OVO persisted in embryonic germ cells until zygotic OVO expression was detectable, suggesting that there is continuous nuclear OVO expression in the female germline in the transition from one generation to the next. Article Summaryovo has long been considered to be at the top of the female germline sex determination pathway. We utilized updated genetic methods to determine OVO expression, localization, and requirement in the embryonic and adult germline. Our results indicate that OVO is always present, and likely required, in the Drosophila female germline.\n",
  "wordCount" : "2908",
  "inLanguage": "en",
  "datePublished": "2023-09-13T10:39:03Z",
  "dateModified": "2023-09-13T10:39:03Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on September 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.02.556057">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.02.556057" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.02.556057">
        <p class="paperTitle">The simplicity of protein sequence-function relationships</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.02.556057" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.02.556057" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.</p>
        <p class="info">Score: 30.5, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.02.556057' target='https://doi.org/10.1101/2023.09.02.556057'> 10.1101/2023.09.02.556057</a></p>
        <p class="abstract">How complicated is the relationship between a proteins sequence and its function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most prior studies, however, used methods that misinterpret measurement errors, small local idiosyncracies around a designated wild-type sequence, and global nonlinearity in the sequence-function relationship as rampant high-order interactions. Here we present a simple new method to jointly estimate global nonlinearity and specific epistatic interactions across a proteins genotype-phenotype map. Our reference-free approach calculates the effect of each amino acid state or combination by averaging over all genotypes that contain it relative to the global average. We show that this method is more accurate than any alternative approach and is robust to measurement error and partial sampling. We reanalyze 20 combinatorial mutagenesis experiments and find that main and pairwise effects, together with a simple form of global nonlinearity, account for a median of 96% of total variance in the measured phenotype (and &gt; 92% in every case), and only a tiny fraction of genotypes are strongly affected by epistasis at third or higher orders. The genetic architecture is also sparse: the number of model terms required to explain the vast majority of phenotypic variance is smaller than the number of genotypes by many orders of magnitude. The sequence-function relationship in most proteins is therefore far simpler than previously thought, and new, more tractable experimental approaches, combined with reference-free analysis, may be sufficient to explain it in most cases.

Significance StatementIt is widely thought that a proteins function depends on complex interactions among amino acids. If so, it would be virtually impossible to predict the function of new variants, and understanding how proteins work genetically and biochemically would require huge combinatorial experiments. We show that prior studies overestimated complexity because they analyzed sequence-function relationships from the perspective of a single reference genotype and/or misinterpreted global phenotypic nonlinearities as complex amino acid interactions. By developing a new reference-free approach and using it to reanalyze 20 experimental datasets, we show that additive effects and pairwise interactions alone, along with a simple global nonlinearity, explain the vast majority of functional variation. Higher-order interactions are weak or rare, and a minuscule fraction of possible interactions shape each proteins function. Our work reveals that protein sequence-function relationships are surprisingly simple and suggests new strategies that are far more tractable than the massive experiments currently used.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557129">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557129" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557129">
        <p class="paperTitle">Expression Level Analysis of ACE2 Receptor Gene in African-American and Non-African-American COVID-19 Patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557129" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557129" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nyamari, M. N.; Omar, K. M.; Fayehun, A. F.; Dachi, O.; Bwana, B. K.; Awe, O. I.</p>
        <p class="info">Score: 13.2, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557129' target='https://doi.org/10.1101/2023.09.11.557129'> 10.1101/2023.09.11.557129</a></p>
        <p class="abstract">Background: The COVID-19 pandemic caused by SARS-CoV-2 has spread rapidly across the continents. While the incidence of COVID-19 has been reported to be higher among African-American individuals, the rate of mortality has been lower compared to that of non-African-Americans. ACE2 is involved in COVID-19 as SARS-CoV-2 uses the ACE2 enzyme to enter host cells. Although the difference in COVID-19 incidence can be explained by many factors such as low accessibility of health insurance among the African-American community, little is known about ACE2 expression in African-American COVID-19 patients compared to non-African-American COVID-19 patients. The variable expression of genes can contribute to this observed phenomenon. Methodology: In this study, transcriptomes from African-American and non-African-American COVID-19 patients were retrieved from the sequence read archive and analyzed for ACE2 gene expression. HISAT2 was used to align the reads to the human reference genome, and HTseq-count was used to get raw gene counts. EdgeR was utilized for differential gene expression analysis, and enrichR was employed for gene enrichment analysis. Results: The datasets included 14 and 33 transcriptome sequences from COVID-19 patients of African-American and non-African-American descent, respectively. There were 24,092 differentially expressed genes, with 7,718 upregulated (log fold change &gt; 1 and FDR 0.05) and 16,374 downregulated (log fold change -1 and FDR 0.05). The ACE2 mRNA level was found to be considerably downregulated in the African-American cohort (p-value = 0.0242, p-adjusted value = 0.038). Conclusion: The downregulation of ACE2 in the African-American cohort could indicate a correlation to the low COVID-19 severity observed among the African-American community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556855">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556855" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556855">
        <p class="paperTitle">Genetic manipulation of an Ixodes scapularis cell line</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556855" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556855" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Singh, N.; Rolandelli, A.; ONeal, A. J.; Butler, L. R.; Samaddar, S.; Laukaitis-Yousey, H.; Butnaru, M.; Mohr, S. E.; Perrimon, N.; Pedra, J. H.</p>
        <p class="info">Score: 11.0, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556855' target='https://doi.org/10.1101/2023.09.08.556855'> 10.1101/2023.09.08.556855</a></p>
        <p class="abstract">Although genetic manipulation is one of the hallmarks in model organisms, its applicability to non-model species has remained difficult due to our limited understanding of their fundamental biology. For instance, manipulation of a cell line originated from the blacklegged tick Ixodes scapularis, an arthropod that serves as a vector of several human pathogens, has yet to be established. Here, we demonstrate the successful genetic modification of the commonly used tick ISE6 line through ectopic expression and clustered regularly interspaced palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9) genome editing. We performed ectopic expression using nucleofection and attained CRISPR-Cas9 editing via homology dependent recombination. Targeting the E3 ubiquitin ligase X-linked inhibitor of apoptosis (xiap) and its substrate p47 led to alteration in molecular signaling within the immune deficiency (IMD) network and increased infection of the rickettsial agent Anaplasma phagocytophilum in I. scapularis ISE6 cells. Collectively, our findings complement techniques for genetic engineering of ticks in vivo and aid in circumventing the long-life cycle of I. scapularis, of which limits efficient and scalable molecular genetic screens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556798" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556798">
        <p class="paperTitle">Proteome-scale prediction of molecular mechanisms underlying dominant genetic diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Badonyi, M.; Marsh, J. A.</p>
        <p class="info">Score: 9.5, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556798' target='https://doi.org/10.1101/2023.09.08.556798'> 10.1101/2023.09.08.556798</a></p>
        <p class="abstract">Many dominant genetic disorders result from protein-altering mutations, acting primarily through dominant-negative (DN), gain-of-function (GOF), and loss-of-function (LOF) mechanisms. Deciphering the mechanisms by which dominant diseases exert their effects is often experimentally challenging and resource intensive, but is essential for developing appropriate therapeutic approaches. Diseases that arise via a LOF mechanism are more amenable to be treated by conventional gene therapy, whereas DN and GOF mechanisms may require gene editing or targeting by small molecules. Moreover, pathogenic missense mutations that act via DN and GOF mechanisms are more difficult to identify than those that act via LOF using nearly all currently available variant effect predictors. Here, we introduce a tripartite statistical model made up of support vector machine binary classifiers trained to predict whether human protein-coding genes are likely to be associated with DN, GOF, or LOF molecular disease mechanisms. We test the utility of the predictions by examining biologically and clinically meaningful properties known to be associated with the mechanisms. Our results strongly support that the models are able to generalise on unseen data and offer insight into the functional attributes of proteins associated with different mechanisms. We hope that our predictions will serve as a springboard for researchers studying novel variants and those of uncertain clinical significance, guiding variant interpretation strategies and experimental characterisation. Predictions for the 2023_02 UniProt reference proteome are available at https://osf.io/z4dcp/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556720">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556720" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556720">
        <p class="paperTitle">Disentangling sources of clock-like mutations in germline and soma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556720" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556720" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Spisak, N.; de Manuel, M.; Milligan, W. R.; Sella, G.; Przeworski, M.</p>
        <p class="info">Score: 8.7, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556720' target='https://doi.org/10.1101/2023.09.07.556720'> 10.1101/2023.09.07.556720</a></p>
        <p class="abstract">The rates of mutations vary across cell types. To identify causes of this variation, mutations are often decomposed into a combination of the single base substitution (SBS) &#34;signatures&#34; observed in germline, soma and tumors, with the idea that each signature corresponds to one or a small number of underlying mutagenic processes. Two such signatures turn out to be ubiquitous across cell types: SBS signature 1, which consists primarily of transitions at methylated CpG sites caused by spontaneous deamination, and the more diffuse SBS signature 5, which is of unknown etiology. In cancers, the number of mutations attributed to these two signatures accumulates linearly with age of diagnosis, and thus the signatures have been termed &#34;clock-like.&#34; To better understand this clock-like behavior, we develop a mathematical model that includes DNA replication errors, unrepaired damage, and damage repaired incorrectly. We show that mutational signatures can exhibit clock-like behavior because cell divisions occur at a constant rate and/or because damage rates remain constant over time, and that these distinct sources can be teased apart by comparing cell lineages that divide at different rates. With this goal in mind, we analyze the rate of accumulation of mutations in multiple cell types, including soma as well as male and female germline. We find no detectable increase in SBS signature 1 mutations in neurons and only a very weak increase in mutations assigned to the female germline, but a significant increase with time in rapidly-dividing cells, suggesting that SBS signature 1 is driven by rounds of DNA replication occurring at a relatively fixed rate. In contrast, SBS signature 5 increases with time in all cell types, including post-mitotic ones, indicating that it accumulates independently of cell divisions; this observation points to errors in DNA repair as the key underlying mechanism. Thus, the two &#34;clock-like&#34; signatures observed across cell types likely have distinct origins, one set by rates of cell division, the other by damage rates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556458">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556458" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556458">
        <p class="paperTitle">Evolutionary diversity of the control of the azole response by Tra1 across yeast species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556458" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556458" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Librais, G. M.; Jiang, Y.; Razzaq, I.; Brandl, C. J.; Shapiro, R. S.; Lajoie, P.</p>
        <p class="info">Score: 6.4, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556458' target='https://doi.org/10.1101/2023.09.05.556458'> 10.1101/2023.09.05.556458</a></p>
        <p class="abstract">Tra1 is an essential co-activator protein of the yeast SAGA and NuA4 acetyltransferase complexes that regulate gene expression through multiple mechanisms including the acetylation of histone proteins. Tra1 is a pseudokinase of the PIKK family characterized by a C-terminal PI3K domain with no known kinase activity. However, mutations of specific arginine residues to glutamine in the PI3K domains (an allele termed tra1Q3) result in reduced growth and increased sensitivity to multiple stresses. In the opportunistic fungal pathogen Candida albicans, the tra1Q3 allele reduces pathogenicity and increases sensitivity to the echinocandin antifungal drug caspofungin, which disrupts the fungal cell wall. Here, we found that loss of Tra1 function, in contrast to what is seen with caspofungin, increases tolerance to the azole class of antifungal drugs, which inhibits ergosterol synthesis. In C. albicans, tra1Q3 increases expression of genes linked to azole resistance, such as ERG11 and CDR1. CDR1 encodes a multidrug ABC transporter associated with efflux of multiple xenobiotics, including azoles. Consequently, cells carrying tra1Q3 show reduced intracellular accumulation of fluconazole. In contrast, a tra1Q3 S. cerevisiae strain displayed opposite phenotypes: decreased tolerance to azole, decreased expression of the efflux pump PDR5 and increased intracellular accumulation of fluconazole. Therefore, our data provide evidence that Tra1 differentially regulates the antifungal response across yeast species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556626">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556626" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556626">
        <p class="paperTitle">Slave Trade and Colonial Expansion Resulted in Strong Sex-Biased Admixture in South Africa</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556626" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556626" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reynolds, A.; Schurz, H.; Meeks, G.; Gravel, S.; Myrick, J. W.; Mendez, F. L.; Werely, C.; van Helden, P. D.; Hoal, E. G.; Poznik, G. D.; Kim, M.; Uren, C.; Underhill, P. A.; Möller, M.; Henn, B. M.</p>
        <p class="info">Score: 5.8, Published: 2023-09-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556626' target='https://doi.org/10.1101/2023.09.06.556626'> 10.1101/2023.09.06.556626</a></p>
        <p class="abstract">The colonial-period arrival of Europeans in southern Africa is associated with strong sex-biased migration by which male settlers displaced indigenous Khoekhoe and San men. Simultaneously, the importation of South Asian, Indonesian and Eastern African slaves may have contributed female-biased migration to Cape Town and surrounding areas. We examine the spatial and temporal spread of sex-biased migration from the Cape northward into Namaqualand and the southern Kalahari using genetic data from more than 1,400 individuals. In all regions, admixture patterns were sex-biased, with evidence of a greater male contribution of European ancestry and greater female contribution of Khoe-San ancestry. While admixture among Khoe-San, European, equatorial African, and Asian groups has likely been continuous from the founding of Cape Town to present-day, we find that Khoe-San groups further north experienced a single pulse of European admixture 6-8 generations ago. European admixture was followed by additional Khoe-San gene flow, potentially reflecting an aggregation of indigenous groups due to disruption by colonial interlopers. Male migration into the northern frontier territories was not a homogenous group of expanding Afrikaners and slaves. The Nama show evidence of distinct founder effects and derive 15% of their male lineages from Asian men, a pattern absent in the =Khomani San. Khoe-San ancestry from the paternal line is greatly diminished in populations from Cape Town, the Cederberg Mountains and Upington, but remains more frequent in self-identified ethnically indigenous groups. Strikingly, we estimate that Khoe-San Y-chromosomes were experiencing unprecedented population growth at the time of European arrival. Our findings shed light on the patterns of admixture and the population history of South Africa as the colonial frontier expanded.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555359">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555359" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555359">
        <p class="paperTitle">Functional characterization of gene regulatory elements and neuropsychiatric disease-associated risk loci in iPSCs and iPSC-derived neurons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555359" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555359" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, X.; Jones, I. R.; Chen, P. B.; Yang, H.; Ren, X.; Zheng, L.; Li, B.; Li, Y. E.; Sun, Q.; Wen, J.; Beaman, C.; Cui, X.; Li, Y.; Wang, W.; Hu, M.; Ren, B.; Shen, Y.</p>
        <p class="info">Score: 27.0, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555359' target='https://doi.org/10.1101/2023.08.30.555359'> 10.1101/2023.08.30.555359</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have identified thousands of non-coding variants that contribute to psychiatric disease risks, likely by perturbing cis-regulatory elements (CREs). However, our ability to interpret and explore their mechanisms of action is hampered by a lack of annotation of functional CREs (fCREs) in neural cell types. Here, through genome-scale CRISPR screens of 22,000 candidate CREs (cCREs) in human induced pluripotent stem cells (iPSCs) undergoing differentiation to excitatory neurons, we identify 2,847 and 5,540 fCREs essential for iPSC fitness and neuronal differentiation, respectively. These fCREs display dynamic epigenomic features and exhibit increased numbers and genomic spans of chromatin interactions following terminal neuronal differentiation. Furthermore, fCREs essential for neuronal differentiation show significantly greater enrichment of genetic heritability for neurodevelopmental diseases including schizophrenia (SCZ), attention deficit hyperactivity disorder (ADHD), and autism spectrum disorders (ASD) than cCREs. Using high-throughput prime editing screens we experimentally confirm 45 SCZ risk variants that act by affecting the function of fCREs. The extensive and in-depth functional annotation of cCREs in neuronal types therefore provides a crucial resource for interpreting non-coding risk variants of neuropsychiatric disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555563">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555563" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555563">
        <p class="paperTitle">Generation of isogenic models of Angelman syndrome and Prader-Willi syndrome in CRISPR/Cas9-engineered human embryonic stem cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555563" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555563" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gilmore, R. B.; Gorka, D.; Stoddard, C.; Cotney, J.; Chamberlain, S.</p>
        <p class="info">Score: 8.8, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555563' target='https://doi.org/10.1101/2023.08.30.555563'> 10.1101/2023.08.30.555563</a></p>
        <p class="abstract">Angelman Syndrome (AS) and Prader-Willi Syndrome (PWS), two distinct neurodevelopmental disorders, result from loss of expression from imprinted genes in the chromosome 15q11-13 locus most commonly caused by a megabase-scale deletion on either the maternal or paternal allele, respectively. Each occurs at an approximate incidence of 1/15,000 to 1/30,000 live births and has a range of debilitating phenotypes. Patient-derived induced pluripotent stem cells (iPSCs) have been valuable tools to understand human-relevant gene regulation at this locus and have contributed to the development of therapeutic approaches for AS. Nonetheless, gaps remain in our understanding of how these deletions contribute to dysregulation and phenotypes of AS and PWS. Variability across cell lines due to donor differences, reprogramming methods, and genetic background make it challenging to fill these gaps in knowledge without substantially increasing the number of cell lines used in the analyses. Isogenic cell lines that differ only by the genetic mutation causing the disease can ease this burden without requiring such a large number of cell lines. Here, we describe the development of isogenic human embryonic stem cell (hESC) lines modeling the most common genetic subtypes of AS and PWS. These lines allow for a facile interrogation of allele-specific gene regulation at the chromosome 15q11-q13 locus. Additionally, these lines are an important resource to identify and test targeted therapeutic approaches for patients with AS and PWS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554887">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554887" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554887">
        <p class="paperTitle">Female germline expression of OVO transcription factor bridges Drosophila generations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554887" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554887" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benner, L.; Muron, S.; Oliver, B.</p>
        <p class="info">Score: 7.7, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554887' target='https://doi.org/10.1101/2023.08.25.554887'> 10.1101/2023.08.25.554887</a></p>
        <p class="abstract">OVO is required for karyotypically female germ cell viability but has no known function in the male germline in Drosophila. ovo is autoregulated by two antagonistic isoforms, OVO-A and OVO-B. All ovo- alleles were created as partial revertants of the antimorphic ovoD1allele. Creation of new targeted alleles in an ovo&#43; background indicated that disrupting the germline-specific exon extension of ovo-B leads to an arrested egg chamber phenotype, rather than germ cell death. RNA-seq analysis, including &gt;1K full length cDNAs, indicates that ovo utilizes a number of unannotated splice variations in the extended exon and a minor population of ovo-B transcripts utilizes an alternative splice. This indicates that classical ovo alleles such as ovoD1rv23, are not truly null for ovo, and are likely to be weak antimorphs. To generate bonafide nulls, we deleted the ovo-A and ovo-B promoters showing that only ovo-B is required for female germ cell viability and there is an early and polyphasic developmental requirement for ovo-B in the female germline. To visualize OVO expression and localization, we endogenously tagged ovo and found nuclear OVO in all differentiating female germ cells throughout oogenesis in adults. We also found that OVO is maternally deposited into the embryo, where it showed nuclear localization in newly formed pole cells. Maternal OVO persisted in embryonic germ cells until zygotic OVO expression was detectable, suggesting that there is continuous nuclear OVO expression in the female germline in the transition from one generation to the next.

Article Summaryovo has long been considered to be at the top of the female germline sex determination pathway. We utilized updated genetic methods to determine OVO expression, localization, and requirement in the embryonic and adult germline. Our results indicate that OVO is always present, and likely required, in the Drosophila female germline.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
