<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Polygenic risk score prediction accuracy convergence
Authors: Henches, L.; Kim, J.; Yang, Z.; Rubinacci, S.; Pires, G.; Albinana, C.; Boetto, C.; Julienne, H.; Frouin, A.; Auvergne, A.; Suzuki, Y.; Djebali, S.; Delaneau, O.; Ganna, A.; Vilhjalmsson, B.; Prive, F.; Aschard, H.
Score: 33.2, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546518
Polygenic risk scores (PRS) trained from genome-wide association study (GWAS) results are set to play a pivotal role in biomedical research addressing multifactorial human diseases.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Polygenic risk score prediction accuracy convergence
Authors: Henches, L.; Kim, J.; Yang, Z.; Rubinacci, S.; Pires, G.; Albinana, C.; Boetto, C.; Julienne, H.; Frouin, A.; Auvergne, A.; Suzuki, Y.; Djebali, S.; Delaneau, O.; Ganna, A.; Vilhjalmsson, B.; Prive, F.; Aschard, H.
Score: 33.2, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546518
Polygenic risk scores (PRS) trained from genome-wide association study (GWAS) results are set to play a pivotal role in biomedical research addressing multifactorial human diseases." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-09T10:40:17+00:00" />
<meta property="article:modified_time" content="2023-07-09T10:40:17+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Polygenic risk score prediction accuracy convergence
Authors: Henches, L.; Kim, J.; Yang, Z.; Rubinacci, S.; Pires, G.; Albinana, C.; Boetto, C.; Julienne, H.; Frouin, A.; Auvergne, A.; Suzuki, Y.; Djebali, S.; Delaneau, O.; Ganna, A.; Vilhjalmsson, B.; Prive, F.; Aschard, H.
Score: 33.2, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546518
Polygenic risk scores (PRS) trained from genome-wide association study (GWAS) results are set to play a pivotal role in biomedical research addressing multifactorial human diseases."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Polygenic risk score prediction accuracy convergence\nAuthors: Henches, L.; Kim, J.; Yang, Z.; Rubinacci, S.; Pires, G.; Albinana, C.; Boetto, C.; Julienne, H.; Frouin, A.; Auvergne, A.; Suzuki, Y.; Djebali, S.; Delaneau, O.; Ganna, A.; Vilhjalmsson, B.; Prive, F.; Aschard, H.\nScore: 33.2, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546518\nPolygenic risk scores (PRS) trained from genome-wide association study (GWAS) results are set to play a pivotal role in biomedical research addressing multifactorial human diseases.",
  "keywords": [
    
  ],
  "articleBody": " Polygenic risk score prediction accuracy convergence\nAuthors: Henches, L.; Kim, J.; Yang, Z.; Rubinacci, S.; Pires, G.; Albinana, C.; Boetto, C.; Julienne, H.; Frouin, A.; Auvergne, A.; Suzuki, Y.; Djebali, S.; Delaneau, O.; Ganna, A.; Vilhjalmsson, B.; Prive, F.; Aschard, H.\nScore: 33.2, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546518\nPolygenic risk scores (PRS) trained from genome-wide association study (GWAS) results are set to play a pivotal role in biomedical research addressing multifactorial human diseases. The prospect of using these risk scores in clinical care and public health is generating both enthusiasm and controversy, with varying opinions about strengths and limitations across experts1. The performances of existing polygenic scores are still limited, and although it is expected to improve with increasing sample size of GWAS and the development of new powerful methods, it remains unclear how much prediction can be ultimately achieved. Here, we conducted a retrospective analysis to assess the progress in PRS prediction accuracy since the publication of the first large-scale GWASs using six common human diseases with sufficient GWAS data. We show that while PRS accuracy has grown rapidly for years, the improvement pace from recent GWAS has decreased substantially, suggesting that further increasing GWAS sample size may translate into very modest risk discrimination improvement. We next investigated the factors influencing the maximum achievable prediction using recently released whole genome-sequencing data from 125K UK Biobank participants, and state-of-the-art modeling of polygenic outcomes. Our analyses point toward increasing the variant coverage of PRS, using either more imputed variants or sequencing data, as a key component for future improvement in prediction accuracy.\nAccounting for Isoform Expression in eQTL Mapping Substantially Increases Power\nAuthors: LaPierre, N.; Pimentel, H.\nScore: 23.6, Published: 2023-06-30 DOI: 10.1101/2023.06.28.546921\nA core problem in genetics is eQTL mapping, in which genetic variants associated with changes in expression of genes are identified. It is common in eQTL mapping to compute gene expression by aggregating the expression levels of individual isoforms from the same gene and then performing linear regression between SNPs and this aggregated gene expression level. However, SNPs may regulate isoforms from the same gene in different directions due to alternative splicing, or only regulate the expression level of one isoform, causing this approach to lose power. In this study, we provide a systematic evaluation of methods for accounting for individual isoform expression levels based on generative isoform expression heritability models and real data. Over a range of conditions, we show that these approaches substantially increase the power to map eQTLs in both simulations and commonly analyzed large data sets. We identify settings in which different approaches yield an inflated number of false discoveries or lose power. In particular, we show that calling an eGene if there is a significant association between a SNP and any isoform fails to control False Discovery Rate, even when applying standard False Discovery Rate correction. We show that similar trends are observed in real data from the GEUVADIS and GTEx studies, suggesting the possibility that similar effects are present in these consortia.\nGenetic similarity between relatives provides evidence on the presence and history of assortative mating\nAuthors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.\nScore: 41.3, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546663\nAssortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives1-7. However, empirical evidence is limited for many traits, and the implications hinge on whether assortative mating has started recently or many generations ago. Here we show theoretically and empirically that genetic similarity between relatives can provide evidence on the presence and history of assortative mating. First, we employed path analysis to understand how assortative mating affects genetic similarity between family members across generations, finding that similarity between distant relatives is more affected than close relatives. Next, we correlated polygenic indices of 47,135 co-parents from the Norwegian Mother, Father, and Child Cohort Study (MoBa) and found genetic evidence of assortative mating in eight out of fifteen examined traits. The same traits show elevated similarity between relatives, especially distant relatives. Five of the eight traits, including educational attainment, showed greater genetic variance among offspring, which is inconsistent with stable assortative mating over many generations. These results suggest an ongoing increase in familial similarity for these traits. The implications of this research extend to genetic methodology and the understanding of social and economic disparities.\nMulti-trait GWAS for diverse ancestries: Mapping the knowledge gap\nAuthors: Troubat, L.; Fettahoglu, D.; Aschard, H.; Julienne, H.\nScore: 11.0, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546248\nBackgroundApproximately 95% of samples analyzed in univariate genome-wide association studies (GWAS) are of European ancestry. This bias toward European ancestry populations in association screening also exists for other analyses and methods that are often developed and tested on European ancestry only. However, existing data in non-European populations, which are often of modest sample size, could benefit from innovative approaches as recently illustrated in the context of polygenic risk scores. MethodsHere, we extend and assess the potential limitations and gain of our multi-trait GWAS pipeline, JASS (Joint Analysis of Summary Statistics), for the analysis of non-European ancestries. To this end, we conducted the joint GWAS of 19 hematological traits and glycemic traits accross five ancestries (European (EUR), admixed American (AMR), African (AFR), East Asian (EAS), South-East Asian (SAS)). ResultsWe detected 367 new genome-wide significant associations in non-European populations (15 in Admixed American (AMR), 72 in African (AFR) and 280 in East Asian (EAS)). New associations detected represent 7%, 25% and 21% of associations in the AFR, AMR and EAS populations, respectively. Overall, multi-trait testing increases the replication of European associated loci in non-European ancestry by 15%. Pleiotropic effects were highly similar at significant loci across ancestries (e.g. the mean correlation between multi-trait genetic effects of EUR and EAS ancestries was 0.88). For hematological traits, strong discrepancies in multi-trait genetic effects are tied to known evolutionary divergences: the ARKC1 loci, which is adaptive to overcome the p.vivax induced malaria. ConclusionsMulti-trait GWAS can be a valuable tool to narrow the genetic knowledge gap between European and non-European populations.\nTranscriptome brings variations of gene expression, alternative splicing, and structural variations into gene-scale trait dissection in soybean\nAuthors: Li, D.; Wang, Q.; Tian, Y.; Lyu, X.; Zhang, H.; Sun, Y.; Hong, H.; Gao, H.; Li, Y.-F.; Zhao, C.; Wang, J.; Wang, R.; Yang, J.; Liu, B.; Schnable, P. S.; Schnable, J. C.; Li, Y.; Qiu, L.\nScore: 5.5, Published: 2023-07-04 DOI: 10.1101/2023.07.03.545230\nGenome-wide association study (GWAS) identifies trait-associated loci, but due in part to slow decay of linkage disequilibrium (LD), identifying the causal genes can be a bottleneck. Transcriptome-wide association study (TWAS) addresses this by identifying gene expression-phenotype associations or integrating gene expression quantitative trait loci (eQTLs) with GWAS results. Here, we used self-pollinated soybean as a model to evaluate the application of TWAS in the genetic dissection of traits in plant species that exhibit slow LD decay. The first RNA-Seq analysis of a soybean diversity panel was conducted, which identified the genetic regulation of 29,286 genes. Different TWAS solutions were less affected by LD and robust with source of expression that identified known genes related to traits from different development stages and tissues. A novel gene named pod color L2 was identified via TWAS and functionally validated by genome editing. Our introduction of the new exon proportion feature significantly improves the capture of expression variations resulting from structural variations and alternative splicing. As a result, the genes identified by our TWAS approach exhibited a diverse range of causal variations, including SNP, insertion/deletion, gene fusion, copy number variation, and alternative splicing. Using our TWAS approach, we identified genes associated with flowering time, including both previously known candidates and novel genes that had not been linked to this trait before, providing complementary insights with GWAS. In summary, this study supports the application of TWAS for candidate gene identification in species with low rates of LD decay.\nModeling soybean growth: A mixed model approach\nAuthors: DELATTRE, M.; Toda, Y.; Tressou, J.; Iwata, H.\nScore: 8.4, Published: 2023-06-13 DOI: 10.1101/2023.06.13.544713\nThe evaluation of plant and animal growth, separately for genetic and environmental effects, is necessary for genetic understanding and genetic improvement of environmental responses of plants and animals. We propose an original approach that combines nonlinear mixed-effects model (NLME) and the stochastic approximation of the Expectation-Maximization algorithm (SAEM) to analyze genetic and environmental effects on plant growth. These tools are widely used in many fields but very rarely in plant biology. During model formulation, a nonlinear mechanistic function describes the shape of growth, and random effects describe genetic and environmental effects and their variability. Genetic relationships among the varieties were also integrated into the model using a kinship matrix. The SAEM algorithm was chosen as an efficient alternative to MCMC methods, which are more commonly used in the domain. It was implemented to infer the expected growth patterns in the analyzed population and the expected curves for each variety through a maximum-likelihood and a maximum-a-posteriori approaches, respectively. The obtained estimates can be used to predict the growth curves for each variety. We illustrate the strengths of the proposed approach using simulated data and soybean plant growth data obtained from a soybean cultivation experiment conducted at the Arid Land Research Center, Tottori University. In this experiment, plant height was measured daily using drones, and the growth was monitored for approximately 200 soybean cultivars for which whole-genome sequence data were available. The NLME approach improved our understanding of the determinants of soybean growth and can be successfully used for the genomic prediction of growth pattern characteristics. Author summaryNonlinear mechanistic models are useful for modeling animal and plant growth; however, their parameters are influenced by both genetic and environmental factors. If the same model can be applied to data with different genetic and environmental factors by allowing parameter variations, it can be used to understand, predict, and control the genetic and environmental influences of growth models based on parameter variation. In this study, we propose a statistical method for integrating a nonlinear mixed-effects model with a nonlinear mechanistic model. The simulation and real data analysis results show that the proposed method was effective in modeling the growth of genetically different soybean varieties under different drought conditions. The usefulness of the proposed method is expected to increase, as high-throughput measurements provide growth data for a large number of genotypes in various environments.\nIncomplete Reprogramming of DNA Replication Timing in Induced Pluripotent Stem Cells\nAuthors: Edwards, M. M.; Wang, N.; Massey, D.; Egli, D.; Koren, A.\nScore: 7.7, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544654\nInduced pluripotent stem cells (iPSC) are a widely used cell system and a foundation for cell therapy. Differences in gene expression, DNA methylation, and chromatin conformation, which have the potential to affect differentiation capacity, have been identified between iPSCs and embryonic stem cells (ESCs). Less is known about whether DNA replication timing - a process linked to both genome regulation and genome stability - is efficiently reprogrammed to the embryonic state. To answer this, we profiled and compared genome-wide replication timing between ESCs, iPSCs, and cells reprogrammed by somatic cell nuclear transfer (NT-ESCs). While NT-ESCs replicated their DNA in a manner indistinguishable from ESCs, a subset of iPSCs exhibit delayed replication at heterochromatic regions containing genes downregulated in iPSC with incompletely reprogrammed DNA methylation. DNA replication delays were not the result of gene expression and DNA methylation aberrations and persisted after differentiating cells to neuronal precursors. Thus, DNA replication timing can be resistant to reprogramming and lead to undesirable phenotypes in iPSCs, establishing it as an important genomic feature to consider when evaluating iPSC lines.\nThe Janelia Atalanta plasmids provide a simple and efficient CRISPR/Cas9-mediated homology directed repair platform for Drosophila\nAuthors: Stern, D. L.; Kim, E.; Berhman, E. L.\nScore: 15.4, Published: 2023-06-17 DOI: 10.1101/2023.06.17.545412\nHomology-directed repair (HDR) is a powerful tool for modifying genomes in precise ways to address many biological questions. Use of Clustered Regularly Interspersed Short Palindromic Repeats (CRISPR)-Cas9 induced targeted DNA double-strand breakage has substantially simplified use of homology-directed repair to introduce specific perturbations in Drosophila, but existing platforms for CRISPR-Cas9-mediated HDR in Drosophila involve multiple cloning steps and have low efficiency. To simplify cloning of HDR plasmids, we designed a new plasmid platform, the Janelia Atalanta (pJAT) series, that exploits recent advances in dsDNA synthesis to facilitate Gateway cloning of gRNA sequences and homology arms in one step. Surprisingly, the pJAT plasmids yielded considerably higher HDR efficiency (approximately 25%) than we have observed with other approaches. pJAT plasmids work in multiple Drosophila species and exhibited such high efficiency that previously impossible experiments in Drosophila, such as driving targeted chromosomal inversions, were made possible. We provide pJAT plasmids for a range of commonly performed experiments including targeted insertional mutagenesis, insertion of phiC31-mediated attP landing sites, generation of strains carrying a germ-line source of Cas9, and induction of chromosomal rearrangements. We also provide \"empty\" pJAT plasmids with multiple cloning sites to simplify construction of plasmids with new functionality. The pJAT platform is generic and may facilitate improved efficiency CRISPR-Cas9 HDR in a wide range of model and non-model organisms.\nSex peptide receptor is not required for refractoriness to remating or induction of egg laying in Aedes aegypti\nAuthors: Amaro, I. A.; Wohl, M. P.; Pitcher, S.; Alfonso-Parra, C.; Avila, F. W.; Paige, A. S.; Helinski, M.; Duvall, L. B.; Harrington, L. C.; Wolfner, M. F.; McMeniman, C. J.\nScore: 14.6, Published: 2023-06-30 DOI: 10.1101/2023.06.28.546954\nAcross diverse insect taxa, the behavior and physiology of females dramatically changes after mating - processes largely triggered by the transfer of seminal proteins from their mates. In the vinegar fly Drosophila melanogaster, the seminal protein sex peptide (SP) decreases the likelihood of female flies remating and causes additional behavioral and physiological changes that promote fertility including increasing egg production. Although SP is only found in the Drosophila genus, its receptor, sex peptide receptor (SPR), is the widely-conserved myoinhibitory peptide (MIP) receptor. To test the functional role of SPR in mediating post-mating responses in a non-Drosophila dipteran, we generated two independent Spr-knockout alleles in the yellow fever mosquito Aedes aegypti. Although SPR is needed for post-mating responses in Drosophila and the cotton bollworm Helicoverpa armigera, Spr mutant Ae. aegypti show completely normal post-mating decreases in remating propensity and increases in egg laying. In addition, injection of synthetic SP or accessory gland homogenate from D. melanogaster into virgin female mosquitoes did not elicit these post-mating responses. Our results indicate that Spr is not required for these canonical post-mating responses in Ae. aegypti, indicating that unknown signaling pathways are likely responsible for these behavioral switches in this disease vector.\nAllelic variation in the wheat homolog of topoisomerase II is associated with crossover rate\nAuthors: Jordan, K. W.; Fernandez-de Soto, M.; Korol, A.; Akhunova, A.; Akhunov, E.\nScore: 1.8, Published: 2023-07-07 DOI: 10.1101/2023.07.06.548017\nMeiotic recombination is a fundamental biological process that impacts genetic diversity and response of variants to selection. The increase in crossover (CO) frequency by modulating the activity of meiotic genes has been suggested as potential means to improve the efficiency of selection and mitigate the negative impact of linkage drag in crop improvement programs. In this study, we used the wheat nested association mapping (NAM) population to map QTL associated with CO rate and harboring TaTOPII-A1, a gene encoding topoisomerase II that is involved in the regulation of meiosis. An amino acid changing SNP (T/C) located in exon 19 of TaTOPII-A1 with a predicted functional effect showed an association with CO rate. The T-allele of TaTOPII-A1 was associated with a statistically significant 5.9 % increase in COs across several NAM families. The involvement of TaTOPII-A1 in processes affecting CO rate in wheat was confirmed using a TaTOPII-A1 mutant. A strong-effect missense mutation in the TaTOPII-A1 coding region resulted in 1.4-fold increase in the genetic map length and 53% increase in CO rate compared to the wild-type allele. These results demonstrate that functional mutations in TaTOPII-A1 can lead to increased CO frequency. The mutant and natural alleles of TaTOPII-A1 identified in our study could serve as new sources of recombination-promoting genes to manipulate CO rate in wheat and possibly other crops.\n",
  "wordCount" : "2691",
  "inLanguage": "en",
  "datePublished": "2023-07-09T10:40:17Z",
  "dateModified": "2023-07-09T10:40:17Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on July 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546518">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546518" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546518">
        <p class="paperTitle">Polygenic risk score prediction accuracy convergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546518" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546518" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Henches, L.; Kim, J.; Yang, Z.; Rubinacci, S.; Pires, G.; Albinana, C.; Boetto, C.; Julienne, H.; Frouin, A.; Auvergne, A.; Suzuki, Y.; Djebali, S.; Delaneau, O.; Ganna, A.; Vilhjalmsson, B.; Prive, F.; Aschard, H.</p>
        <p class="info">Score: 33.2, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546518' target='https://doi.org/10.1101/2023.06.27.546518'> 10.1101/2023.06.27.546518</a></p>
        <p class="abstract">Polygenic risk scores (PRS) trained from genome-wide association study (GWAS) results are set to play a pivotal role in biomedical research addressing multifactorial human diseases. The prospect of using these risk scores in clinical care and public health is generating both enthusiasm and controversy, with varying opinions about strengths and limitations across experts1. The performances of existing polygenic scores are still limited, and although it is expected to improve with increasing sample size of GWAS and the development of new powerful methods, it remains unclear how much prediction can be ultimately achieved. Here, we conducted a retrospective analysis to assess the progress in PRS prediction accuracy since the publication of the first large-scale GWASs using six common human diseases with sufficient GWAS data. We show that while PRS accuracy has grown rapidly for years, the improvement pace from recent GWAS has decreased substantially, suggesting that further increasing GWAS sample size may translate into very modest risk discrimination improvement. We next investigated the factors influencing the maximum achievable prediction using recently released whole genome-sequencing data from 125K UK Biobank participants, and state-of-the-art modeling of polygenic outcomes. Our analyses point toward increasing the variant coverage of PRS, using either more imputed variants or sequencing data, as a key component for future improvement in prediction accuracy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546921">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546921" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546921">
        <p class="paperTitle">Accounting for Isoform Expression in eQTL Mapping Substantially Increases Power</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546921" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546921" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: LaPierre, N.; Pimentel, H.</p>
        <p class="info">Score: 23.6, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546921' target='https://doi.org/10.1101/2023.06.28.546921'> 10.1101/2023.06.28.546921</a></p>
        <p class="abstract">A core problem in genetics is eQTL mapping, in which genetic variants associated with changes in expression of genes are identified. It is common in eQTL mapping to compute gene expression by aggregating the expression levels of individual isoforms from the same gene and then performing linear regression between SNPs and this aggregated gene expression level. However, SNPs may regulate isoforms from the same gene in different directions due to alternative splicing, or only regulate the expression level of one isoform, causing this approach to lose power. In this study, we provide a systematic evaluation of methods for accounting for individual isoform expression levels based on generative isoform expression heritability models and real data. Over a range of conditions, we show that these approaches substantially increase the power to map eQTLs in both simulations and commonly analyzed large data sets. We identify settings in which different approaches yield an inflated number of false discoveries or lose power. In particular, we show that calling an eGene if there is a significant association between a SNP and any isoform fails to control False Discovery Rate, even when applying standard False Discovery Rate correction. We show that similar trends are observed in real data from the GEUVADIS and GTEx studies, suggesting the possibility that similar effects are present in these consortia.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546663">
        <p class="paperTitle">Genetic similarity between relatives provides evidence on the presence and history of assortative mating</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sunde, H. F.; Eftedal, N. H.; Cheesman, R.; Corfield, E. C.; Kleppesto, T. H.; Seierstad, A. C.; Ystrom, E.; Eilertsen, E. M.; Torvik, F. A.</p>
        <p class="info">Score: 41.3, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546663' target='https://doi.org/10.1101/2023.06.27.546663'> 10.1101/2023.06.27.546663</a></p>
        <p class="abstract">Assortative mating - the non-random mating of individuals with similar traits - is known to increase trait-specific genetic variance and genetic similarity between relatives1-7. However, empirical evidence is limited for many traits, and the implications hinge on whether assortative mating has started recently or many generations ago. Here we show theoretically and empirically that genetic similarity between relatives can provide evidence on the presence and history of assortative mating. First, we employed path analysis to understand how assortative mating affects genetic similarity between family members across generations, finding that similarity between distant relatives is more affected than close relatives. Next, we correlated polygenic indices of 47,135 co-parents from the Norwegian Mother, Father, and Child Cohort Study (MoBa) and found genetic evidence of assortative mating in eight out of fifteen examined traits. The same traits show elevated similarity between relatives, especially distant relatives. Five of the eight traits, including educational attainment, showed greater genetic variance among offspring, which is inconsistent with stable assortative mating over many generations. These results suggest an ongoing increase in familial similarity for these traits. The implications of this research extend to genetic methodology and the understanding of social and economic disparities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546248">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546248" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546248">
        <p class="paperTitle">Multi-trait GWAS for diverse ancestries: Mapping the knowledge gap</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546248" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546248" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Troubat, L.; Fettahoglu, D.; Aschard, H.; Julienne, H.</p>
        <p class="info">Score: 11.0, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546248' target='https://doi.org/10.1101/2023.06.23.546248'> 10.1101/2023.06.23.546248</a></p>
        <p class="abstract">BackgroundApproximately 95% of samples analyzed in univariate genome-wide association studies (GWAS) are of European ancestry. This bias toward European ancestry populations in association screening also exists for other analyses and methods that are often developed and tested on European ancestry only. However, existing data in non-European populations, which are often of modest sample size, could benefit from innovative approaches as recently illustrated in the context of polygenic risk scores.

MethodsHere, we extend and assess the potential limitations and gain of our multi-trait GWAS pipeline, JASS (Joint Analysis of Summary Statistics), for the analysis of non-European ancestries. To this end, we conducted the joint GWAS of 19 hematological traits and glycemic traits accross five ancestries (European (EUR), admixed American (AMR), African (AFR), East Asian (EAS), South-East Asian (SAS)).

ResultsWe detected 367 new genome-wide significant associations in non-European populations (15 in Admixed American (AMR), 72 in African (AFR) and 280 in East Asian (EAS)). New associations detected represent 7%, 25% and 21% of associations in the AFR, AMR and EAS populations, respectively. Overall, multi-trait testing increases the replication of European associated loci in non-European ancestry by 15%. Pleiotropic effects were highly similar at significant loci across ancestries (e.g. the mean correlation between multi-trait genetic effects of EUR and EAS ancestries was 0.88). For hematological traits, strong discrepancies in multi-trait genetic effects are tied to known evolutionary divergences: the ARKC1 loci, which is adaptive to overcome the p.vivax induced malaria.

ConclusionsMulti-trait GWAS can be a valuable tool to narrow the genetic knowledge gap between European and non-European populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.545230">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.545230" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.545230">
        <p class="paperTitle">Transcriptome brings variations of gene expression, alternative splicing, and structural variations into gene-scale trait dissection in soybean</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.545230" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.545230" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, D.; Wang, Q.; Tian, Y.; Lyu, X.; Zhang, H.; Sun, Y.; Hong, H.; Gao, H.; Li, Y.-F.; Zhao, C.; Wang, J.; Wang, R.; Yang, J.; Liu, B.; Schnable, P. S.; Schnable, J. C.; Li, Y.; Qiu, L.</p>
        <p class="info">Score: 5.5, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.545230' target='https://doi.org/10.1101/2023.07.03.545230'> 10.1101/2023.07.03.545230</a></p>
        <p class="abstract">Genome-wide association study (GWAS) identifies trait-associated loci, but due in part to slow decay of linkage disequilibrium (LD), identifying the causal genes can be a bottleneck. Transcriptome-wide association study (TWAS) addresses this by identifying gene expression-phenotype associations or integrating gene expression quantitative trait loci (eQTLs) with GWAS results. Here, we used self-pollinated soybean as a model to evaluate the application of TWAS in the genetic dissection of traits in plant species that exhibit slow LD decay. The first RNA-Seq analysis of a soybean diversity panel was conducted, which identified the genetic regulation of 29,286 genes. Different TWAS solutions were less affected by LD and robust with source of expression that identified known genes related to traits from different development stages and tissues. A novel gene named pod color L2 was identified via TWAS and functionally validated by genome editing. Our introduction of the new exon proportion feature significantly improves the capture of expression variations resulting from structural variations and alternative splicing. As a result, the genes identified by our TWAS approach exhibited a diverse range of causal variations, including SNP, insertion/deletion, gene fusion, copy number variation, and alternative splicing. Using our TWAS approach, we identified genes associated with flowering time, including both previously known candidates and novel genes that had not been linked to this trait before, providing complementary insights with GWAS. In summary, this study supports the application of TWAS for candidate gene identification in species with low rates of LD decay.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.13.544713">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.13.544713" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.13.544713">
        <p class="paperTitle">Modeling soybean growth: A mixed model approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.13.544713" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.13.544713" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: DELATTRE, M.; Toda, Y.; Tressou, J.; Iwata, H.</p>
        <p class="info">Score: 8.4, Published: 2023-06-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.13.544713' target='https://doi.org/10.1101/2023.06.13.544713'> 10.1101/2023.06.13.544713</a></p>
        <p class="abstract">The evaluation of plant and animal growth, separately for genetic and environmental effects, is necessary for genetic understanding and genetic improvement of environmental responses of plants and animals. We propose an original approach that combines nonlinear mixed-effects model (NLME) and the stochastic approximation of the Expectation-Maximization algorithm (SAEM) to analyze genetic and environmental effects on plant growth. These tools are widely used in many fields but very rarely in plant biology. During model formulation, a nonlinear mechanistic function describes the shape of growth, and random effects describe genetic and environmental effects and their variability. Genetic relationships among the varieties were also integrated into the model using a kinship matrix. The SAEM algorithm was chosen as an efficient alternative to MCMC methods, which are more commonly used in the domain. It was implemented to infer the expected growth patterns in the analyzed population and the expected curves for each variety through a maximum-likelihood and a maximum-a-posteriori approaches, respectively. The obtained estimates can be used to predict the growth curves for each variety. We illustrate the strengths of the proposed approach using simulated data and soybean plant growth data obtained from a soybean cultivation experiment conducted at the Arid Land Research Center, Tottori University. In this experiment, plant height was measured daily using drones, and the growth was monitored for approximately 200 soybean cultivars for which whole-genome sequence data were available. The NLME approach improved our understanding of the determinants of soybean growth and can be successfully used for the genomic prediction of growth pattern characteristics.

Author summaryNonlinear mechanistic models are useful for modeling animal and plant growth; however, their parameters are influenced by both genetic and environmental factors. If the same model can be applied to data with different genetic and environmental factors by allowing parameter variations, it can be used to understand, predict, and control the genetic and environmental influences of growth models based on parameter variation. In this study, we propose a statistical method for integrating a nonlinear mixed-effects model with a nonlinear mechanistic model. The simulation and real data analysis results show that the proposed method was effective in modeling the growth of genetically different soybean varieties under different drought conditions. The usefulness of the proposed method is expected to increase, as high-throughput measurements provide growth data for a large number of genotypes in various environments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.12.544654">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.12.544654" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.12.544654">
        <p class="paperTitle">Incomplete Reprogramming of DNA Replication Timing in Induced Pluripotent Stem Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.12.544654" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.12.544654" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Edwards, M. M.; Wang, N.; Massey, D.; Egli, D.; Koren, A.</p>
        <p class="info">Score: 7.7, Published: 2023-06-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.12.544654' target='https://doi.org/10.1101/2023.06.12.544654'> 10.1101/2023.06.12.544654</a></p>
        <p class="abstract">Induced pluripotent stem cells (iPSC) are a widely used cell system and a foundation for cell therapy. Differences in gene expression, DNA methylation, and chromatin conformation, which have the potential to affect differentiation capacity, have been identified between iPSCs and embryonic stem cells (ESCs). Less is known about whether DNA replication timing - a process linked to both genome regulation and genome stability - is efficiently reprogrammed to the embryonic state. To answer this, we profiled and compared genome-wide replication timing between ESCs, iPSCs, and cells reprogrammed by somatic cell nuclear transfer (NT-ESCs). While NT-ESCs replicated their DNA in a manner indistinguishable from ESCs, a subset of iPSCs exhibit delayed replication at heterochromatic regions containing genes downregulated in iPSC with incompletely reprogrammed DNA methylation. DNA replication delays were not the result of gene expression and DNA methylation aberrations and persisted after differentiating cells to neuronal precursors. Thus, DNA replication timing can be resistant to reprogramming and lead to undesirable phenotypes in iPSCs, establishing it as an important genomic feature to consider when evaluating iPSC lines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.17.545412">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.17.545412" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.17.545412">
        <p class="paperTitle">The Janelia Atalanta plasmids provide a simple and efficient CRISPR/Cas9-mediated homology directed repair platform for Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.17.545412" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.17.545412" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stern, D. L.; Kim, E.; Berhman, E. L.</p>
        <p class="info">Score: 15.4, Published: 2023-06-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.17.545412' target='https://doi.org/10.1101/2023.06.17.545412'> 10.1101/2023.06.17.545412</a></p>
        <p class="abstract">Homology-directed repair (HDR) is a powerful tool for modifying genomes in precise ways to address many biological questions. Use of Clustered Regularly Interspersed Short Palindromic Repeats (CRISPR)-Cas9 induced targeted DNA double-strand breakage has substantially simplified use of homology-directed repair to introduce specific perturbations in Drosophila, but existing platforms for CRISPR-Cas9-mediated HDR in Drosophila involve multiple cloning steps and have low efficiency. To simplify cloning of HDR plasmids, we designed a new plasmid platform, the Janelia Atalanta (pJAT) series, that exploits recent advances in dsDNA synthesis to facilitate Gateway cloning of gRNA sequences and homology arms in one step. Surprisingly, the pJAT plasmids yielded considerably higher HDR efficiency (approximately 25%) than we have observed with other approaches. pJAT plasmids work in multiple Drosophila species and exhibited such high efficiency that previously impossible experiments in Drosophila, such as driving targeted chromosomal inversions, were made possible. We provide pJAT plasmids for a range of commonly performed experiments including targeted insertional mutagenesis, insertion of phiC31-mediated attP landing sites, generation of strains carrying a germ-line source of Cas9, and induction of chromosomal rearrangements. We also provide &#34;empty&#34; pJAT plasmids with multiple cloning sites to simplify construction of plasmids with new functionality. The pJAT platform is generic and may facilitate improved efficiency CRISPR-Cas9 HDR in a wide range of model and non-model organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546954">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546954" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546954">
        <p class="paperTitle">Sex peptide receptor is not required for refractoriness to remating or induction of egg laying in Aedes aegypti</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546954" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546954" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Amaro, I. A.; Wohl, M. P.; Pitcher, S.; Alfonso-Parra, C.; Avila, F. W.; Paige, A. S.; Helinski, M.; Duvall, L. B.; Harrington, L. C.; Wolfner, M. F.; McMeniman, C. J.</p>
        <p class="info">Score: 14.6, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546954' target='https://doi.org/10.1101/2023.06.28.546954'> 10.1101/2023.06.28.546954</a></p>
        <p class="abstract">Across diverse insect taxa, the behavior and physiology of females dramatically changes after mating - processes largely triggered by the transfer of seminal proteins from their mates. In the vinegar fly Drosophila melanogaster, the seminal protein sex peptide (SP) decreases the likelihood of female flies remating and causes additional behavioral and physiological changes that promote fertility including increasing egg production. Although SP is only found in the Drosophila genus, its receptor, sex peptide receptor (SPR), is the widely-conserved myoinhibitory peptide (MIP) receptor. To test the functional role of SPR in mediating post-mating responses in a non-Drosophila dipteran, we generated two independent Spr-knockout alleles in the yellow fever mosquito Aedes aegypti. Although SPR is needed for post-mating responses in Drosophila and the cotton bollworm Helicoverpa armigera, Spr mutant Ae. aegypti show completely normal post-mating decreases in remating propensity and increases in egg laying. In addition, injection of synthetic SP or accessory gland homogenate from D. melanogaster into virgin female mosquitoes did not elicit these post-mating responses. Our results indicate that Spr is not required for these canonical post-mating responses in Ae. aegypti, indicating that unknown signaling pathways are likely responsible for these behavioral switches in this disease vector.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.548017">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.548017" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.548017">
        <p class="paperTitle">Allelic variation in the wheat homolog of topoisomerase II is associated with crossover rate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.548017" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.548017" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jordan, K. W.; Fernandez-de Soto, M.; Korol, A.; Akhunova, A.; Akhunov, E.</p>
        <p class="info">Score: 1.8, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.548017' target='https://doi.org/10.1101/2023.07.06.548017'> 10.1101/2023.07.06.548017</a></p>
        <p class="abstract">Meiotic recombination is a fundamental biological process that impacts genetic diversity and response of variants to selection. The increase in crossover (CO) frequency by modulating the activity of meiotic genes has been suggested as potential means to improve the efficiency of selection and mitigate the negative impact of linkage drag in crop improvement programs. In this study, we used the wheat nested association mapping (NAM) population to map QTL associated with CO rate and harboring TaTOPII-A1, a gene encoding topoisomerase II that is involved in the regulation of meiosis. An amino acid changing SNP (T/C) located in exon 19 of TaTOPII-A1 with a predicted functional effect showed an association with CO rate. The T-allele of TaTOPII-A1 was associated with a statistically significant 5.9 % increase in COs across several NAM families. The involvement of TaTOPII-A1 in processes affecting CO rate in wheat was confirmed using a TaTOPII-A1 mutant. A strong-effect missense mutation in the TaTOPII-A1 coding region resulted in 1.4-fold increase in the genetic map length and 53% increase in CO rate compared to the wild-type allele. These results demonstrate that functional mutations in TaTOPII-A1 can lead to increased CO frequency. The mutant and natural alleles of TaTOPII-A1 identified in our study could serve as new sources of recombination-promoting genes to manipulate CO rate in wheat and possibly other crops.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
