<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Causal interpretations of family GWAS in the presence of heterogeneous effects
Authors: Veller, C.; Przeworski, M.; Coop, G.
Score: 51.0, Published: 2023-11-16 DOI: 10.1101/2023.11.13.566950
Family-based genome-wide association studies (GWAS) have emerged as a gold standard for assessing causal effects of alleles and polygenic scores. Notably, family studies are often claimed to provide an unbiased estimate of the average causal effect (or average treatment effect; ATE) of an allele, on the basis of an analogy between the random transmission of alleles from parents to children and a randomized controlled trial.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Causal interpretations of family GWAS in the presence of heterogeneous effects
Authors: Veller, C.; Przeworski, M.; Coop, G.
Score: 51.0, Published: 2023-11-16 DOI: 10.1101/2023.11.13.566950
Family-based genome-wide association studies (GWAS) have emerged as a gold standard for assessing causal effects of alleles and polygenic scores. Notably, family studies are often claimed to provide an unbiased estimate of the average causal effect (or average treatment effect; ATE) of an allele, on the basis of an analogy between the random transmission of alleles from parents to children and a randomized controlled trial." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-22T10:41:31+00:00" />
<meta property="article:modified_time" content="2023-11-22T10:41:31+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Causal interpretations of family GWAS in the presence of heterogeneous effects
Authors: Veller, C.; Przeworski, M.; Coop, G.
Score: 51.0, Published: 2023-11-16 DOI: 10.1101/2023.11.13.566950
Family-based genome-wide association studies (GWAS) have emerged as a gold standard for assessing causal effects of alleles and polygenic scores. Notably, family studies are often claimed to provide an unbiased estimate of the average causal effect (or average treatment effect; ATE) of an allele, on the basis of an analogy between the random transmission of alleles from parents to children and a randomized controlled trial."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Causal interpretations of family GWAS in the presence of heterogeneous effects\nAuthors: Veller, C.; Przeworski, M.; Coop, G.\nScore: 51.0, Published: 2023-11-16 DOI: 10.1101/2023.11.13.566950\nFamily-based genome-wide association studies (GWAS) have emerged as a gold standard for assessing causal effects of alleles and polygenic scores. Notably, family studies are often claimed to provide an unbiased estimate of the average causal effect (or average treatment effect; ATE) of an allele, on the basis of an analogy between the random transmission of alleles from parents to children and a randomized controlled trial.",
  "keywords": [
    
  ],
  "articleBody": " Causal interpretations of family GWAS in the presence of heterogeneous effects\nAuthors: Veller, C.; Przeworski, M.; Coop, G.\nScore: 51.0, Published: 2023-11-16 DOI: 10.1101/2023.11.13.566950\nFamily-based genome-wide association studies (GWAS) have emerged as a gold standard for assessing causal effects of alleles and polygenic scores. Notably, family studies are often claimed to provide an unbiased estimate of the average causal effect (or average treatment effect; ATE) of an allele, on the basis of an analogy between the random transmission of alleles from parents to children and a randomized controlled trial. Here, we show that this interpretation does not hold in general. Because Mendelian segregation only randomizes alleles among children of heterozygotes, the effects of alleles in the children of homozygotes are not observable. Consequently, if an allele has different average effects in the children of homozygotes and heterozygotes, as can arise in the presence of gene-by-environment interactions, gene-by-gene interactions, or differences in LD patterns, family studies provide a biased estimate of the average effect in the sample. At a single locus, family-based association studies can be thought of as providing an unbiased estimate of the average effect in the children of heterozygotes (i.e., a local average treatment effect; LATE). This interpretation does not extend to polygenic scores, however, because different sets of SNPs are heterozygous in each family. Therefore, other than under specific conditions, the within-family regression slope of a PGS cannot be assumed to provide an un-biased estimate for any subset or weighted average of families. Instead, family-based studies can be reinterpreted as enabling an unbiased estimate of the extent to which Mendelian segregation at loci in the PGS contributes to the population-level variance in the trait. Because this estimate does not include the between-family variance, however, this interpretation applies to only (roughly) half of the sample PGS variance. In practice, the potential biases of a family-based GWAS are likely smaller than those arising from confounding in a standard, population-based GWAS, and so family studies remain important for the dissection of genetic contributions to phenotypic variation. Nonetheless, the causal interpretation of family-based GWAS estimates is less straightforward than has been widely appreciated.\nMisexpression of inactive genes in whole blood is associated with nearby rare structural variants\nAuthors: Vanderstichele, T.; Burnham, K. L.; de Klein, N.; Tardaguila, M.; Howell, B.; Walter, K.; Kundu, K.; Koeppel, J.; Lee, W.; Tokolyi, A.; Persyn, E.; Nath, A. P.; Marten, J.; Petrovski, S.; Roberts, D. J.; Di Angelantonio, E.; Danesh, J.; Berton, A.; Platt, A.; Butterworth, A. S.; Soranzo, N.; Parts, L.; Inouye, M.; Paul, D. S.; Davenport, E. E.\nScore: 28.2, Published: 2023-11-19 DOI: 10.1101/2023.11.17.567537\nGene misexpression is the aberrant transcription of a gene in a context where it is usually inactive. Despite its known pathological consequences in specific rare diseases, we have a limited understanding of its wider prevalence and mechanisms in humans. To address this, we analyzed gene misexpression in 4,568 whole blood bulk RNA sequencing samples from INTERVAL study blood donors. We found that while individual misexpression events occur rarely, in aggregate they were found in almost all samples and over half of inactive genes. Using 2,821 paired whole genome and RNA sequencing samples, we identified that misexpression events are enriched in cis for rare structural variants. We established putative mechanisms through which a subset of SVs lead to gene misexpression, including transcriptional readthrough, transcript fusions and gene inversion. Overall, we develop misexpression as a novel type of transcriptomic outlier analysis and extend our understanding of the variety of mechanisms by which genetic variants can influence gene expression.\nModeling recent positive selection in Americans of European ancestry\nAuthors: Temple, S. D.; Waples, R. D.; Browning, S. R.\nScore: 22.7, Published: 2023-11-16 DOI: 10.1101/2023.11.13.566947\nRecent positive selection can result in an excess of long identity-by-descent (IBD) haplotype segments. The statistical methods that we propose here address three major objectives in studying classical selective sweeps: scanning for regions of interest, identifying possible sweeping alleles, and estimating a selection coefficient s. First, we implement a selection scan to locate regions of excess IBD rate. Second, we develop a statistic to rank alleles in strong linkage disequilibrium with a putative sweeping allele. We aggregate these scores to estimate the allele frequency of the sweeping allele, even if it is not genotyped. Lastly, we propose an estimator for the selection coefficient and quantify uncertainty using the parametric bootstrap. Comparing against state-of-the-art methods in extensive simulations, we show that our methods are better at pinpointing alleles at low frequency and estimating s [\u0026ge;] 0.015. We apply these methods to study positive selection in inferred European ancestry samples from the TOPMed project. We find twelve loci where their IBD rates exceed four standard deviations above the population median. The excess IBD rate at LCT is thirty-five standard deviations above the population median, and our estimates of its selection coefficient imply strong selection within the past 120 to 163 generations. Overall, we present robust and accurate new approaches to study very recent adaptive evolution under mild assumptions.\nClark (2023) and the Persistence of Hereditarian Fallacies\nAuthors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.\nScore: 118.4, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565061\nClark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people \"remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England\"; and (3) social mobility has remained static over this time period due to strong assortative mating on a \"social genotype.\" These conclusions are based on a misconstrual of model parameters, which conflates genetic and non-genetic transmission (e.g. of wealth) within families. As we show, there is strong confounding of genetic and non-genetic sources of similarity in these data. Inconsistent with claims (2) and (3), we show that familial correlations in status are variable--generally decreasing--through the time period analyzed. Lastly, we find that statistical artifacts substantially bias estimates of familial correlations in the paper. Overall, Clark (2023) provides no information about the relative contribution of genetic and non-genetic factors to social status.\nIdentifying genetic variants that influence the abundance of cell states in single-cell data\nAuthors: Rumker, L.; Sakaue, S.; Reshef, Y.; Kang, J. B.; Yazar, S.; Alquicira-Hernandez, J.; Valencia, C.; Lagattuta, K. A.; Mah-Som, A.; Nathan, A.; Powell, J. E.; Loh, P.-R.; Raychaudhuri, S.\nScore: 20.9, Published: 2023-11-15 DOI: 10.1101/2023.11.13.566919\nIntroductory ParagraphTo understand genetic mechanisms driving disease, it is essential but difficult to map how risk alleles affect the composition of cells present in the body. Single-cell profiling quantifies granular information about tissues, but variant-associated cell states may reflect diverse combinations of the profiled cell features that are challenging to predefine. We introduce GeNA (Genotype-Neighborhood Associations), a statistical tool to identify cell state abundance quantitative trait loci (csaQTLs) in high-dimensional single-cell datasets. Instead of testing associations to predefined cell states, GeNA flexibly identifies the cell states whose abundance is most associated with genetic variants. In a genome-wide survey of scRNA-seq peripheral blood profiling from 969 individuals,1 GeNA identifies five independent loci associated with shifts in the relative abundance of immune cell states. For example, rs3003-T (p=1.96x10-11) associates with increased abundance of NK cells expressing TNF- response programs. This csaQTL colocalizes with increased risk for psoriasis, an autoimmune disease that responds to anti-TNF treatments. Flexibly characterizing csaQTLs for granular cell states may help illuminate how genetic background alters cellular composition to confer disease risk.\nGenetic Discovery Enabled by A Large Language Model\nAuthors: Tu, T.; Fang, Z.; Cheng, Z.; Spasic, S.; Palepu, A.; Stankovic, K.; Natarajan, V.; Peltz, G.\nScore: 26.7, Published: 2023-11-12 DOI: 10.1101/2023.11.09.566468\nArtificial intelligence (AI) has been used in many areas of medicine, and recently large language models (LLMs) have shown potential utility for clinical applications. However, since we do not know if the use of LLMs can accelerate the pace of genetic discovery, we used data generated from mouse genetic models to investigate this possibility. We examined whether a recently developed specialized LLM (Med-PaLM 2) could analyze sets of candidate genes generated from analysis of murine models of biomedical traits. In response to free-text input, Med-PaLM 2 correctly identified the murine genes that contained experimentally verified causative genetic factors for six biomedical traits, which included susceptibility to diabetes and cataracts. Med-PaLM 2 was also able to analyze a list of genes with high impact alleles, which were identified by comparative analysis of murine genomic sequence data, and it identified a causative murine genetic factor for spontaneous hearing loss. Based upon this Med-PaLM 2 finding, a novel bigenic model for susceptibility to spontaneous hearing loss was developed. These results demonstrate Med-PaLM 2 can analyze gene-phenotype relationships and generate novel hypotheses, which can facilitate genetic discovery.\nWhole genome association testing in 333,100 individuals across three biobanks identifies rare non-coding single variant and genomic aggregate associations with height\nAuthors: Hawkes, G.; Beaumont, R. N.; Li, Z.; Mandla, R.; Li, X.; Albert, C. M.; Arnett, D. K.; Ashley-Koch, A. E.; Ashrani, A. A.; Barnes, K. C.; Boerwinkle, E.; Brody, J. A.; Carson, A. P.; Chami, N.; Chen, Y.-D. I.; Chung, M. K.; Curran, J. E.; Darbar, D.; Ellinor, P. T.; Fornage, M.; Gordeuk, V. R.; Guo, X.; He, J.; Hwu, C.-M.; Kalyani, R. R.; Kaplan, R.; Kardia, S. L. R.; Kooperberg, C.; Loos, R. J. F.; Lubitz, S. A.; Minster, R. L.; Mitchell, B. D.; Murabito, J. M.; Palmer, N. D.; Psaty, B. M.; Redline, S.; Shoemaker, M. B.; Silverman, E. K.; Telen, M. J.; Weiss, S. T.; Yanek, L. R.; Zhou, H.; NH\nScore: 18.7, Published: 2023-11-20 DOI: 10.1101/2023.11.19.566520\nThe role of rare non-coding variation in complex human phenotypes is still largely unknown. To elucidate the impact of rare variants in regulatory elements, we performed a whole-genome sequencing association analysis for height using 333,100 individuals from three datasets: UK Biobank (N=200,003), TOPMed (N=87,652) and All of Us (N=45,445). We performed rare (\u003c0.1% minor-allele-frequency) single-variant and aggregate testing of non-coding variants in regulatory regions based on proximal, intergenic and deep-intronic annotation. We observed 29 independent variants associated with height at P\u003c6x10-10 after conditioning on previously reported variants, with effect sizes ranging from -7cm to +4.7cm. We also identified and replicated non-coding aggregate-based associations proximal to HMGA1 containing variants associated with a 5cm taller height and of highly-conserved variants in MIR497HG on chromosome 17. We have developed a novel approach for identifying non-coding rare variants in regulatory regions with large effects from whole-genome sequencing data associated with complex traits.\nHigh prevalence of Prdm9-independentrecombination hotspots in placental mammals\nAuthors: Joseph, J.; Prentout, D.; Laverre, A.; Tricou, T.; Duret, L.\nScore: 13.2, Published: 2023-11-17 DOI: 10.1101/2023.11.17.567540\nIn many mammals, recombination events are concentrated into hotspots directed by a sequence specific DNA-binding protein named Prdm9. This protein facilitates chromosome pairing and its inactivation has been shown to induce fertility losses in mice and rats. Intriguingly, Prdm9 has been lost several times in vertebrates, and notably among mammals, it has been pseudogenized in the ancestor of canids (dogs, wolves foxes). When this gene is inactive, either naturally in dogs, or through knock-out experiments in mice, recombination hotspots still exist, but they tend to occur in promoter-like features such as CpG islands. It has thus been proposed that one role of Prdm9 could be to direct recombination away from those Prdm9-independent hotspots. However, the ability of Prdm9 to direct recombination hotspots has been assessed only in a handful of species, and a clear picture of how much recombination occurs outside of Prdm9-directed hotspots in mammals is still lacking. In this study, we derived an estimator of past recombination activity based on signatures of GC-biased gene conversion in substitution patterns. We applied it to quantify recombination activity in Prdm9-independent hotspots in 52 species of boreoeutherian mammals. We observed a wide range of recombination rate at these loci: several species (such as mice, humans, some felids or cetaceans) show a deficit of recombination, while a majority of mammals display a clear peak of recombination. Our results demonstrate that Prdm9-directed and Prdm9-independent hotspots can co-exist in mammals, and that their co-existence seem to be the rule rather than an exception.\nExpansion of human centromeric arrays in cells undergoing break-induced replication\nAuthors: Showman, S.; Talbert, P. B.; Xu, Y.; Adeyemi, R. O.; Henikoff, S.\nScore: 9.6, Published: 2023-11-15 DOI: 10.1101/2023.11.11.566714\nHuman centromeres are located within -satellite arrays and evolve rapidly, which can lead to individual variation in array lengths. Proposed mechanisms for such alterations in lengths are unequal cross-over between sister chromatids, gene conversion, and break-induced replication. However, the underlying molecular mechanisms responsible for the massive, complex, and homogeneous organization of centromeric arrays have not been experimentally validated. Here, we use droplet digital PCR assays to demonstrate that centromeric arrays can expand and contract within ~20 somatic cell divisions of a cell line. We find that the frequency of array variation among single-cell-derived subclones ranges from a minimum of ~7% to a maximum of ~100%. Further clonal evolution revealed that centromere expansion is favored over contraction. We find that the homologous recombination protein RAD52 and the helicase PIF1 are required for extensive array change, suggesting that centromere sequence evolution can occur via break-induced replication.\nTrans-eQTL hotspots shape complex traits by modulating cellular states\nAuthors: Renganaath, K.; Albert, F. W.\nScore: 9.5, Published: 2023-11-15 DOI: 10.1101/2023.11.14.567054\nRegulatory genetic variation shapes gene expression, providing an important mechanism connecting DNA variation and complex traits. The causal relationships between gene expression and complex traits remain poorly understood. Here, we integrated transcriptomes and 46 genetically complex growth traits in a large cross between two strains of the yeast Saccharomyces cerevisiae. We discovered thousands of genetic correlations between gene expression and growth, suggesting functional connections. Local regulatory variation was a minor source of these genetic correlations. Instead, genetic correlations tended to arise from multiple independent trans-acting regulatory loci. Trans-acting hotspots that affect the expression of numerous genes accounted for particularly large fractions of genetic growth variation and of genetic correlations between gene expression and growth. Genes with genetic correlations were enriched for similar biological processes across traits, but with heterogeneous direction of effect. Our results reveal how trans-acting regulatory hotspots shape complex traits by altering cellular states.\n",
  "wordCount" : "2376",
  "inLanguage": "en",
  "datePublished": "2023-11-22T10:41:31Z",
  "dateModified": "2023-11-22T10:41:31Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 22, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566950">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566950" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566950">
        <p class="paperTitle">Causal interpretations of family GWAS in the presence of heterogeneous effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566950" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566950" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Veller, C.; Przeworski, M.; Coop, G.</p>
        <p class="info">Score: 51.0, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566950' target='https://doi.org/10.1101/2023.11.13.566950'> 10.1101/2023.11.13.566950</a></p>
        <p class="abstract">Family-based genome-wide association studies (GWAS) have emerged as a gold standard for assessing causal effects of alleles and polygenic scores. Notably, family studies are often claimed to provide an unbiased estimate of the average causal effect (or average treatment effect; ATE) of an allele, on the basis of an analogy between the random transmission of alleles from parents to children and a randomized controlled trial. Here, we show that this interpretation does not hold in general. Because Mendelian segregation only randomizes alleles among children of heterozygotes, the effects of alleles in the children of homozygotes are not observable. Consequently, if an allele has different average effects in the children of homozygotes and heterozygotes, as can arise in the presence of gene-by-environment interactions, gene-by-gene interactions, or differences in LD patterns, family studies provide a biased estimate of the average effect in the sample. At a single locus, family-based association studies can be thought of as providing an unbiased estimate of the average effect in the children of heterozygotes (i.e., a local average treatment effect; LATE). This interpretation does not extend to polygenic scores, however, because different sets of SNPs are heterozygous in each family. Therefore, other than under specific conditions, the within-family regression slope of a PGS cannot be assumed to provide an un-biased estimate for any subset or weighted average of families. Instead, family-based studies can be reinterpreted as enabling an unbiased estimate of the extent to which Mendelian segregation at loci in the PGS contributes to the population-level variance in the trait. Because this estimate does not include the between-family variance, however, this interpretation applies to only (roughly) half of the sample PGS variance. In practice, the potential biases of a family-based GWAS are likely smaller than those arising from confounding in a standard, population-based GWAS, and so family studies remain important for the dissection of genetic contributions to phenotypic variation. Nonetheless, the causal interpretation of family-based GWAS estimates is less straightforward than has been widely appreciated.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567537">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567537" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567537">
        <p class="paperTitle">Misexpression of inactive genes in whole blood is associated with nearby rare structural variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567537" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567537" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vanderstichele, T.; Burnham, K. L.; de Klein, N.; Tardaguila, M.; Howell, B.; Walter, K.; Kundu, K.; Koeppel, J.; Lee, W.; Tokolyi, A.; Persyn, E.; Nath, A. P.; Marten, J.; Petrovski, S.; Roberts, D. J.; Di Angelantonio, E.; Danesh, J.; Berton, A.; Platt, A.; Butterworth, A. S.; Soranzo, N.; Parts, L.; Inouye, M.; Paul, D. S.; Davenport, E. E.</p>
        <p class="info">Score: 28.2, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567537' target='https://doi.org/10.1101/2023.11.17.567537'> 10.1101/2023.11.17.567537</a></p>
        <p class="abstract">Gene misexpression is the aberrant transcription of a gene in a context where it is usually inactive. Despite its known pathological consequences in specific rare diseases, we have a limited understanding of its wider prevalence and mechanisms in humans. To address this, we analyzed gene misexpression in 4,568 whole blood bulk RNA sequencing samples from INTERVAL study blood donors. We found that while individual misexpression events occur rarely, in aggregate they were found in almost all samples and over half of inactive genes. Using 2,821 paired whole genome and RNA sequencing samples, we identified that misexpression events are enriched in cis for rare structural variants. We established putative mechanisms through which a subset of SVs lead to gene misexpression, including transcriptional readthrough, transcript fusions and gene inversion. Overall, we develop misexpression as a novel type of transcriptomic outlier analysis and extend our understanding of the variety of mechanisms by which genetic variants can influence gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566947">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566947" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566947">
        <p class="paperTitle">Modeling recent positive selection in Americans of European ancestry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566947" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566947" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Temple, S. D.; Waples, R. D.; Browning, S. R.</p>
        <p class="info">Score: 22.7, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566947' target='https://doi.org/10.1101/2023.11.13.566947'> 10.1101/2023.11.13.566947</a></p>
        <p class="abstract">Recent positive selection can result in an excess of long identity-by-descent (IBD) haplotype segments. The statistical methods that we propose here address three major objectives in studying classical selective sweeps: scanning for regions of interest, identifying possible sweeping alleles, and estimating a selection coefficient s. First, we implement a selection scan to locate regions of excess IBD rate. Second, we develop a statistic to rank alleles in strong linkage disequilibrium with a putative sweeping allele. We aggregate these scores to estimate the allele frequency of the sweeping allele, even if it is not genotyped. Lastly, we propose an estimator for the selection coefficient and quantify uncertainty using the parametric bootstrap. Comparing against state-of-the-art methods in extensive simulations, we show that our methods are better at pinpointing alleles at low frequency and estimating s [&amp;ge;] 0.015. We apply these methods to study positive selection in inferred European ancestry samples from the TOPMed project. We find twelve loci where their IBD rates exceed four standard deviations above the population median. The excess IBD rate at LCT is thirty-five standard deviations above the population median, and our estimates of its selection coefficient imply strong selection within the past 120 to 163 generations. Overall, we present robust and accurate new approaches to study very recent adaptive evolution under mild assumptions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565061">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565061" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565061">
        <p class="paperTitle">Clark (2023) and the Persistence of Hereditarian Fallacies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565061" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565061" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.</p>
        <p class="info">Score: 118.4, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565061' target='https://doi.org/10.1101/2023.11.01.565061'> 10.1101/2023.11.01.565061</a></p>
        <p class="abstract">Clark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people &#34;remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England&#34;; and (3) social mobility has remained static over this time period due to strong assortative mating on a &#34;social genotype.&#34; These conclusions are based on a misconstrual of model parameters, which conflates genetic and non-genetic transmission (e.g. of wealth) within families. As we show, there is strong confounding of genetic and non-genetic sources of similarity in these data. Inconsistent with claims (2) and (3), we show that familial correlations in status are variable--generally decreasing--through the time period analyzed. Lastly, we find that statistical artifacts substantially bias estimates of familial correlations in the paper. Overall, Clark (2023) provides no information about the relative contribution of genetic and non-genetic factors to social status.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566919">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566919" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566919">
        <p class="paperTitle">Identifying genetic variants that influence the abundance of cell states in single-cell data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566919" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566919" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rumker, L.; Sakaue, S.; Reshef, Y.; Kang, J. B.; Yazar, S.; Alquicira-Hernandez, J.; Valencia, C.; Lagattuta, K. A.; Mah-Som, A.; Nathan, A.; Powell, J. E.; Loh, P.-R.; Raychaudhuri, S.</p>
        <p class="info">Score: 20.9, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566919' target='https://doi.org/10.1101/2023.11.13.566919'> 10.1101/2023.11.13.566919</a></p>
        <p class="abstract">Introductory ParagraphTo understand genetic mechanisms driving disease, it is essential but difficult to map how risk alleles affect the composition of cells present in the body. Single-cell profiling quantifies granular information about tissues, but variant-associated cell states may reflect diverse combinations of the profiled cell features that are challenging to predefine. We introduce GeNA (Genotype-Neighborhood Associations), a statistical tool to identify cell state abundance quantitative trait loci (csaQTLs) in high-dimensional single-cell datasets. Instead of testing associations to predefined cell states, GeNA flexibly identifies the cell states whose abundance is most associated with genetic variants. In a genome-wide survey of scRNA-seq peripheral blood profiling from 969 individuals,1 GeNA identifies five independent loci associated with shifts in the relative abundance of immune cell states. For example, rs3003-T (p=1.96x10-11) associates with increased abundance of NK cells expressing TNF- response programs. This csaQTL colocalizes with increased risk for psoriasis, an autoimmune disease that responds to anti-TNF treatments. Flexibly characterizing csaQTLs for granular cell states may help illuminate how genetic background alters cellular composition to confer disease risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566468">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566468" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566468">
        <p class="paperTitle">Genetic Discovery Enabled by A Large Language Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566468" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566468" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tu, T.; Fang, Z.; Cheng, Z.; Spasic, S.; Palepu, A.; Stankovic, K.; Natarajan, V.; Peltz, G.</p>
        <p class="info">Score: 26.7, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566468' target='https://doi.org/10.1101/2023.11.09.566468'> 10.1101/2023.11.09.566468</a></p>
        <p class="abstract">Artificial intelligence (AI) has been used in many areas of medicine, and recently large language models (LLMs) have shown potential utility for clinical applications. However, since we do not know if the use of LLMs can accelerate the pace of genetic discovery, we used data generated from mouse genetic models to investigate this possibility. We examined whether a recently developed specialized LLM (Med-PaLM 2) could analyze sets of candidate genes generated from analysis of murine models of biomedical traits. In response to free-text input, Med-PaLM 2 correctly identified the murine genes that contained experimentally verified causative genetic factors for six biomedical traits, which included susceptibility to diabetes and cataracts. Med-PaLM 2 was also able to analyze a list of genes with high impact alleles, which were identified by comparative analysis of murine genomic sequence data, and it identified a causative murine genetic factor for spontaneous hearing loss. Based upon this Med-PaLM 2 finding, a novel bigenic model for susceptibility to spontaneous hearing loss was developed. These results demonstrate Med-PaLM 2 can analyze gene-phenotype relationships and generate novel hypotheses, which can facilitate genetic discovery.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.566520">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.566520" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.566520">
        <p class="paperTitle">Whole genome association testing in 333,100 individuals across three biobanks identifies rare non-coding single variant and genomic aggregate associations with height</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.566520" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.566520" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hawkes, G.; Beaumont, R. N.; Li, Z.; Mandla, R.; Li, X.; Albert, C. M.; Arnett, D. K.; Ashley-Koch, A. E.; Ashrani, A. A.; Barnes, K. C.; Boerwinkle, E.; Brody, J. A.; Carson, A. P.; Chami, N.; Chen, Y.-D. I.; Chung, M. K.; Curran, J. E.; Darbar, D.; Ellinor, P. T.; Fornage, M.; Gordeuk, V. R.; Guo, X.; He, J.; Hwu, C.-M.; Kalyani, R. R.; Kaplan, R.; Kardia, S. L. R.; Kooperberg, C.; Loos, R. J. F.; Lubitz, S. A.; Minster, R. L.; Mitchell, B. D.; Murabito, J. M.; Palmer, N. D.; Psaty, B. M.; Redline, S.; Shoemaker, M. B.; Silverman, E. K.; Telen, M. J.; Weiss, S. T.; Yanek, L. R.; Zhou, H.; NH</p>
        <p class="info">Score: 18.7, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.566520' target='https://doi.org/10.1101/2023.11.19.566520'> 10.1101/2023.11.19.566520</a></p>
        <p class="abstract">The role of rare non-coding variation in complex human phenotypes is still largely unknown. To elucidate the impact of rare variants in regulatory elements, we performed a whole-genome sequencing association analysis for height using 333,100 individuals from three datasets: UK Biobank (N=200,003), TOPMed (N=87,652) and All of Us (N=45,445). We performed rare (&lt;0.1% minor-allele-frequency) single-variant and aggregate testing of non-coding variants in regulatory regions based on proximal, intergenic and deep-intronic annotation. We observed 29 independent variants associated with height at P&lt;6x10-10 after conditioning on previously reported variants, with effect sizes ranging from -7cm to &#43;4.7cm. We also identified and replicated non-coding aggregate-based associations proximal to HMGA1 containing variants associated with a 5cm taller height and of highly-conserved variants in MIR497HG on chromosome 17. We have developed a novel approach for identifying non-coding rare variants in regulatory regions with large effects from whole-genome sequencing data associated with complex traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.17.567540">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.17.567540" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.17.567540">
        <p class="paperTitle">High prevalence of Prdm9-independentrecombination hotspots in placental mammals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.17.567540" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.17.567540" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joseph, J.; Prentout, D.; Laverre, A.; Tricou, T.; Duret, L.</p>
        <p class="info">Score: 13.2, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.17.567540' target='https://doi.org/10.1101/2023.11.17.567540'> 10.1101/2023.11.17.567540</a></p>
        <p class="abstract">In many mammals, recombination events are concentrated into hotspots directed by a sequence specific DNA-binding protein named Prdm9. This protein facilitates chromosome pairing and its inactivation has been shown to induce fertility losses in mice and rats. Intriguingly, Prdm9 has been lost several times in vertebrates, and notably among mammals, it has been pseudogenized in the ancestor of canids (dogs, wolves foxes). When this gene is inactive, either naturally in dogs, or through knock-out experiments in mice, recombination hotspots still exist, but they tend to occur in promoter-like features such as CpG islands. It has thus been proposed that one role of Prdm9 could be to direct recombination away from those Prdm9-independent hotspots. However, the ability of Prdm9 to direct recombination hotspots has been assessed only in a handful of species, and a clear picture of how much recombination occurs outside of Prdm9-directed hotspots in mammals is still lacking. In this study, we derived an estimator of past recombination activity based on signatures of GC-biased gene conversion in substitution patterns. We applied it to quantify recombination activity in Prdm9-independent hotspots in 52 species of boreoeutherian mammals. We observed a wide range of recombination rate at these loci: several species (such as mice, humans, some felids or cetaceans) show a deficit of recombination, while a majority of mammals display a clear peak of recombination. Our results demonstrate that Prdm9-directed and Prdm9-independent hotspots can co-exist in mammals, and that their co-existence seem to be the rule rather than an exception.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.11.566714">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.11.566714" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.11.566714">
        <p class="paperTitle">Expansion of human centromeric arrays in cells undergoing break-induced replication</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.11.566714" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.11.566714" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Showman, S.; Talbert, P. B.; Xu, Y.; Adeyemi, R. O.; Henikoff, S.</p>
        <p class="info">Score: 9.6, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.11.566714' target='https://doi.org/10.1101/2023.11.11.566714'> 10.1101/2023.11.11.566714</a></p>
        <p class="abstract">Human centromeres are located within -satellite arrays and evolve rapidly, which can lead to individual variation in array lengths. Proposed mechanisms for such alterations in lengths are unequal cross-over between sister chromatids, gene conversion, and break-induced replication. However, the underlying molecular mechanisms responsible for the massive, complex, and homogeneous organization of centromeric arrays have not been experimentally validated. Here, we use droplet digital PCR assays to demonstrate that centromeric arrays can expand and contract within ~20 somatic cell divisions of a cell line. We find that the frequency of array variation among single-cell-derived subclones ranges from a minimum of ~7% to a maximum of ~100%. Further clonal evolution revealed that centromere expansion is favored over contraction. We find that the homologous recombination protein RAD52 and the helicase PIF1 are required for extensive array change, suggesting that centromere sequence evolution can occur via break-induced replication.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567054">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567054" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567054">
        <p class="paperTitle">Trans-eQTL hotspots shape complex traits by modulating cellular states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567054" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567054" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Renganaath, K.; Albert, F. W.</p>
        <p class="info">Score: 9.5, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567054' target='https://doi.org/10.1101/2023.11.14.567054'> 10.1101/2023.11.14.567054</a></p>
        <p class="abstract">Regulatory genetic variation shapes gene expression, providing an important mechanism connecting DNA variation and complex traits. The causal relationships between gene expression and complex traits remain poorly understood. Here, we integrated transcriptomes and 46 genetically complex growth traits in a large cross between two strains of the yeast Saccharomyces cerevisiae. We discovered thousands of genetic correlations between gene expression and growth, suggesting functional connections. Local regulatory variation was a minor source of these genetic correlations. Instead, genetic correlations tended to arise from multiple independent trans-acting regulatory loci. Trans-acting hotspots that affect the expression of numerous genes accounted for particularly large fractions of genetic growth variation and of genetic correlations between gene expression and growth. Genes with genetic correlations were enriched for similar biological processes across traits, but with heterogeneous direction of effect. Our results reveal how trans-acting regulatory hotspots shape complex traits by altering cellular states.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
