<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Enhancing In Vitro SMN Protein Expression and Cell Viability through Xeno-Nucleic Acid-Based ASOs in Spinal Muscular Atrophy
Authors: Kilic, O.; Babar, H. A.; Inci, C. C.; Odabas, S. P.; Yelgen, G.; Oltan, S.; Kulac, S.; Tastan, C.
Score: 6.9, Published: 2023-08-21 DOI: 10.1101/2023.08.21.552929
Spinal Muscular Atrophy (SMA) stands as a devastating ailment arising from the dearth of functional SMN (Survival Motor Neuron) protein due to genetic anomalies within the SMN1 gene.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Enhancing In Vitro SMN Protein Expression and Cell Viability through Xeno-Nucleic Acid-Based ASOs in Spinal Muscular Atrophy
Authors: Kilic, O.; Babar, H. A.; Inci, C. C.; Odabas, S. P.; Yelgen, G.; Oltan, S.; Kulac, S.; Tastan, C.
Score: 6.9, Published: 2023-08-21 DOI: 10.1101/2023.08.21.552929
Spinal Muscular Atrophy (SMA) stands as a devastating ailment arising from the dearth of functional SMN (Survival Motor Neuron) protein due to genetic anomalies within the SMN1 gene." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-27T10:37:02+00:00" />
<meta property="article:modified_time" content="2023-08-27T10:37:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Enhancing In Vitro SMN Protein Expression and Cell Viability through Xeno-Nucleic Acid-Based ASOs in Spinal Muscular Atrophy
Authors: Kilic, O.; Babar, H. A.; Inci, C. C.; Odabas, S. P.; Yelgen, G.; Oltan, S.; Kulac, S.; Tastan, C.
Score: 6.9, Published: 2023-08-21 DOI: 10.1101/2023.08.21.552929
Spinal Muscular Atrophy (SMA) stands as a devastating ailment arising from the dearth of functional SMN (Survival Motor Neuron) protein due to genetic anomalies within the SMN1 gene."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Enhancing In Vitro SMN Protein Expression and Cell Viability through Xeno-Nucleic Acid-Based ASOs in Spinal Muscular Atrophy\nAuthors: Kilic, O.; Babar, H. A.; Inci, C. C.; Odabas, S. P.; Yelgen, G.; Oltan, S.; Kulac, S.; Tastan, C.\nScore: 6.9, Published: 2023-08-21 DOI: 10.1101/2023.08.21.552929\nSpinal Muscular Atrophy (SMA) stands as a devastating ailment arising from the dearth of functional SMN (Survival Motor Neuron) protein due to genetic anomalies within the SMN1 gene.",
  "keywords": [
    
  ],
  "articleBody": " Enhancing In Vitro SMN Protein Expression and Cell Viability through Xeno-Nucleic Acid-Based ASOs in Spinal Muscular Atrophy\nAuthors: Kilic, O.; Babar, H. A.; Inci, C. C.; Odabas, S. P.; Yelgen, G.; Oltan, S.; Kulac, S.; Tastan, C.\nScore: 6.9, Published: 2023-08-21 DOI: 10.1101/2023.08.21.552929\nSpinal Muscular Atrophy (SMA) stands as a devastating ailment arising from the dearth of functional SMN (Survival Motor Neuron) protein due to genetic anomalies within the SMN1 gene. This condition is marked by the consequential attrition of motor neurons, precipitating a progressive decline in muscular strength and culminating in the disruption of neuromuscular junctions. Existing therapeutic approaches encompassing Zolgensma, Nursinersen, and Evrysdi employ innovative genetic therapeutic strategies involving transgene delivery, Antisense Oligonucleotide (ASO) technology, and modulation of pre-mRNA processing to enhance functional SMN protein expression. However, the ASO therapeutics remain suboptimal in establishing a sustained panacea for SMA, as they inadequately maintain consistent levels of functional SMN protein expression. In this study, we present a discerning inquiry into focusing on XNA-DNA-ASO products that exhibit enhanced safety and stability compared to conventional DNA/RNA-ASO sequences. Through precise targeting of the ISSN-1 region within SMN2 genes intron 7, our approach seeks to amplify SMN protein expression. Employing Xeno Nucleic Acid (XNA) bases, known for their augmented hydrophobicity and stability, our strategy surmounts previous limitations associated with chemical modifications, showcasing heightened endonuclease resistance. Comparative analyses with conventional DNA/RNA-ASO products substantiate the superiority of XNA-DNA-ASO sequences, underscoring elevated SMN protein expression and reduced toxicity. In a comprehensive evaluation, our gene therapy paradigm is scrutinized within a type 1 SMA fibroblast cell line. Utilizing diverse analytical methodologies, encompassing Annexin V-PI analysis for cytotoxicity, MTT assays for mitochondrial activity, and flow cytometry for SMN protein expression profile, we gauge therapeutic impact and potential toxicity. In conclusion, our investigation not only spotlights the promise of XNA-DNA-ASO sequences but also holds implications for refining SMA treatment strategies, converging on minimized dosages, lowered toxicity, and heightened therapeutic efficacy, thus shaping the landscape of gene therapy for SMA.\nVariant-to-function analysis of the childhood obesity chr12q13 locus implicates rs7132908 as a causal variant within the 3' UTR of FAIM2\nAuthors: Littleton, S. H.; Trang, K. B.; Volpe, C. M.; Cook, K.; DeBruyne, N.; Maguire, J. A.; Weidekamp, M. A.; Boehm, K.; Chesi, A.; Pippin, J. A.; Anderson, S. A.; Wells, A. D.; Pahl, M. C.; Grant, S. F.\nScore: 6.8, Published: 2023-08-22 DOI: 10.1101/2023.08.21.553157\nThe ch12q13 obesity locus is among the most significant childhood obesity loci identified in genome-wide association studies. This locus resides in a non-coding region within FAIM2; thus, the underlying causal variant(s) presumably influence disease susceptibility via an influence on cis-regulation within the genomic region. We implicated rs7132908 as a putative causal variant at this locus leveraging a combination of our inhouse 3D genomic data, public domain datasets, and several computational approaches. Using a luciferase reporter assay in human primary astrocytes, we observed allele-specific cis-regulatory activity of the immediate region harboring rs7132908. Motivated by this finding, we went on to generate isogenic human embryonic stem cell lines homozygous for either rs7132908 allele with CRISPR-Cas9 homology-directed repair to assess changes in gene expression due to genotype and chromatin accessibility throughout a differentiation to hypothalamic neurons, a key cell type known to regulate feeding behavior. We observed that the rs7132908 obesity risk allele influenced the expression of FAIM2 along with other genes, decreased the proportion of neurons produced during differentiation, up-regulated cell death gene sets, and conversely down-regulated neuron differentiation gene sets. We have therefore functionally validated rs7132908 as a causal obesity variant which temporally regulates nearby effector genes at the ch12q13 locus and influences neurodevelopment and survival.\nHemin decreases cellular aging and enhances healthspan and lifespan through the AMPK pathway\nAuthors: Zhang, Y.; Naaz, A.; Faidzinn, N. A.; Yogasundaram, S.; Cheng, T. Y. N.; Jing, J. L.; Morel Gan, I. W.-H. J.; Junqi, C.; Alfatah, M.\nScore: 7.3, Published: 2023-08-16 DOI: 10.1101/2023.08.15.553367\nThe quest to understand and manipulate the mechanisms of cellular aging has far-reaching implications for improving human health and longevity. Our comprehensive effort has led to the discovery of the intriguing anti-aging potential of hemin, an FDA-approved drug primarily used for the treatment of acute intermittent porphyria. Leveraging both yeast and human cell models, we investigate the multifaceted effects of hemin on extending cellular lifespan. Intriguingly, the involvement of the AMPK pathway emerges as a pivotal mechanism underlying hemins anti-aging effects. The exploration of hemins impact on cellular functionality further uncovers its influence on mitochondrial processes. Notably, both mitochondrial-dependent and -independent mechanisms are implicated in hemins ability to extend cellular lifespan, with autophagy playing a significant role in the latter. Additionally, a striking synergy between hemin and the TORC1 inhibitor rapamycin is unveiled, underlining the complexity of cellular signaling networks involved in lifespan extension. Translating these findings to human cells, hemin demonstrates an analogous ability to induce mitochondrial biogenesis, reduce proinflammatory cytokine expression, and enhance antioxidant response. The conservation of hemins anti-aging effects across species holds promise for therapeutic applications in addressing age-related diseases and promoting healthier aging.\nGenetic variation at 11q23.1 confers colorectal cancer risk by dysregulation of colonic tuft cell transcriptional activator POU2AF2\nAuthors: Rajasekaran, V.; Harris, B. T.; Osborn, R. T.; Smillie, C.; Donnelly, K.; Bacou, M.; Esiri-Bloom, E.; Ooi, L. Y.; Allan, M.; Walker, M.; Reid, S.; Meynert, A.; Grimes, G.; Blackmur, J. P.; Vaughan-Shaw, P. G.; Law, P. J.; Fernandez-Rozadilla, C.; Tomlinson, I. P.; Houlston, R.; Myant, K. B.; Din, F. V. N.; Dunlop, M. G.; Farrington, S. M.\nScore: 6.3, Published: 2023-08-25 DOI: 10.1101/2023.08.24.554659\nCommon genetic variation at 11q23.1 is associated with colorectal cancer (CRC) risk, and exerts local (cis) expression quantitative trait locus (cis-eQTL) effects on POU2AF2, COLCA1 and POU2AF3 genes. However, complex linkage disequilibrium and correlated expression at the 11q23.1 locus has thus far hindered elucidation of the mechanisms by which genetic variants impart CRC risk. Here, we establish that rs3087967 is the likely causal eQTL at this locus, co-localising with expression of POU2AF2 and CRC risk. Furthermore, we show trans-eQTL effects on 21 distant target genes, which are highly enriched for Tuft cell markers. Analysis of available scRNAseq, ChIPseq and scATACseq data implicates POU2AF2 as the primary controller of the tuft cell specific trans-genes through POU2F3-correlated genetic regulation. Immunofluorescence demonstrates that the rs3087967 risk genotype (T) is associated with lower tuft cell abundance in human colonic epithelium. CRISPR-mediated deletion of the 11q23.1 risk locus in the mouse germline exacerbated the ApcMin/+ mouse phenotype upon abrogation of Pou2af2 expression specifically. Taken together, we implicate a key protective role of tuft cells in the large bowel and the importance of mis-regulation of POU2AF2 as the prime tuft cell transcriptional activator at this locus.\nCryptoCEN: A Co-Expression Network for Cryptococcus neoformans reveals novel proteins involved in DNA damage repair\nAuthors: O'Meara, M. J.; Rapala, J. R.; Nichols, C. B.; Alexandre, C.; Billmyre, R. B.; Steenwyk, J. L.; Alspaugh, J. A.; O'Meara, T. R.\nScore: 9.0, Published: 2023-08-18 DOI: 10.1101/2023.08.17.553567\nElucidating gene function is a major goal in biology, especially among non-model organisms. However, doing so is complicated by the fact that molecular conservation does not always mirror functional conservation, and that complex relationships among genes are responsible for encoding pathways and higher-order biological processes. Co-expression, a promising approach for predicting gene function, relies on the general principal that genes with similar expression patterns across multiple conditions will likely be involved in the same biological process. For Cryptococcus neoformans, a prevalent human fungal pathogen greatly diverged from model yeasts, approximately 60% of the predicted genes in the genome lack functional annotations. Here, we leveraged a large amount of publicly available transcriptomic data to generate a C. neoformans Co-Expression Network (CryptoCEN), successfully recapitulating known protein networks, predicting gene function, and enabling insights into the principles influencing co-expression. With 100% predictive accuracy, we used CryptoCEN to identify 13 new DNA damage response genes, underscoring the utility of guilt-by-association for determining gene function. Overall, co-expression is a powerful tool for uncovering gene function, and decreases the experimental tests needed to identify functions for currently under-annotated genes.\nGlial-derived mitochondrial signals impact neuronal proteostasis and aging.\nAuthors: Bar-Ziv, R.; Dutta, N.; Hruby, A.; Sukarto, E.; Henderson, H. R.; Durieux, J.; Tronnes, S. U.; Averbukh, M.; Alcala, A.; Ahmad, Q.; Bolas, T.; Perez, J.; Dishart, J. G.; Vega, M.; Garcia, G.; Higuchi-Sanabria, R.; Dillin, A.\nScore: 7.1, Published: 2023-08-08 DOI: 10.1101/2023.07.20.549924\nThe nervous system plays a critical role in maintaining whole-organism homeostasis; neurons experiencing mitochondrial stress can coordinate the induction of protective cellular pathways, such as the mitochondrial unfolded protein response (UPRMT), between tissues. However, these studies largely ignored non-neuronal cells of the nervous system. Here, we found that UPRMT activation in four, astrocyte-like glial cells in the nematode, C. elegans, can promote protein homeostasis by alleviating protein aggregation in neurons. Surprisingly, we find that glial cells utilize small clear vesicles (SCVs) to signal to neurons, which then relay the signal to the periphery using dense-core vesicles (DCVs). This work underlines the importance of glia in establishing and regulating protein homeostasis within the nervous system, which can then impact neuron-mediated effects in organismal homeostasis and longevity. One-Sentence SummaryGlial cells sense mitochondrial stress and signal a beneficial stress signal to promote neuronal health and longevity.\nIdentification and functional characterisation of a locus for target site integration in Fusarium graminearum\nAuthors: Darino, M.; Urban, M.; Kaur, N.; Machado-Wood, A.; Grimwade-Mann, M.; Smith, D.; Beacham, A.; Hammond-kosack, K.\nScore: 2.9, Published: 2023-08-21 DOI: 10.1101/2023.08.20.553861\nBackgroundFusarium Head Blight is a destructive floral disease of different cereal crops. The Ascomycete fungus Fusarium graminearum (Fg) is one of the main causal agents of FHB in wheat and barley. The role(s) in virulence of Fg genes include genetic studies that involve the transformation of the fungus with different expression cassettes. We have observed in several studies where Fg genes functions were characterised that integration of expression cassettes occurred randomly. Random insertion of a cassette may disrupt gene expression and/or protein functions and hence the overall conclusion of the study. Target site integration (TSI) is an approach that consists in identifying a chromosomal region where the cassette can be inserted. The identification of a suitable locus for TSI in Fg would avert the potential risks of ectopic integration. ResultsHere, we identified a highly conserved intergenic region on chromosome 1 suitable for TSI. We named this intergenic region the TSI locus 1. We developed an efficient cloning vector system based on the Golden Gate method to clone different expression cassettes for use in combination with TSI locus 1. We present evidence that integrations in the TSI locus 1 affects neither fungal virulence nor fungal growth under different stress conditions. Integrations at the TSI locus 1 resulted in the expression of different gene fusions. In addition, the activities of Fg native promoters were not altered by integration into the TSI locus 1. We have developed a bespoke bioinformatic pipeline to analyse the existence of ectopic integrations and tandem insertions of the cassette that may occurred during the transformation process. Finally, we established a protocol to study protein secretion in wheat coleoptiles using confocal microscopy and the TSI locus 1. ConclusionThe TSI locus 1 can be used in Fg and potentially other cereal infecting Fusarium species for diverse studies including promoter activity analysis, secretion, protein localisation studies and gene complementation. The bespoke bioinformatic pipeline developed in this work can be an alternative to southern blotting, the gold standard technique to identify ectopic integration and tandem insertions in fungal transformation.\nAdmixture as a source for HLA variation in Neolithic European farming communities\nAuthors: da Silva, N. A.; Ozer, O.; Haller, M.; Chen, Y.-R.; Kolbe, D.; Schade-Linding, S.; Wahl, J.; Berszin, C.; Francken, M.; Gorner, I.; Schierhold, K.; Pechtl, J.; Grupe, G.; Rinne, C.; Muller, J.; Lenz, T. L.; Nebel, A.; Krause-Kyora, B.\nScore: 2.4, Published: 2023-08-23 DOI: 10.1101/2023.08.23.554285\nThe northern European Neolithic is characterized by two major demographic events: immigration of early farmers (EF) from Anatolia (5500 BCE) and their admixture (from ~4200 BCE) with western hunter-gatherers (WHG) forming late farmers (LF). The influence of this admixture event on variation in the immune-relevant human leukocyte antigen (HLA) region is understudied. Here, we conducted population and immunogenetic analyses on 83 individuals from six EF and LF sites located in present-day Germany. We observed significant shifts in HLA allele frequencies from EF to LF. The HLA diversity increased from EF to LF, likely due to admixture with WHG. However, it was considerably lower than in modern populations. Both EF and LF exhibited a relatively narrow HLA allele spectrum compared to today. This coincides with sparse traces of pathogen DNA, potentially indicating a lower pathogen pressure at the time. We additionally noted that LF resulted from sex-biased admixture from male WHG.\nEZH2 specifically regulates ISL1 during embryonic urinary tract formation\nAuthors: Mingrado, E.; Kalanithy, J.; Dworschak, G.; Ishorst, N.; Yilmaz, O.; Lindenberg, T.; Hollstein, R.; Felger, T.; Angrand, P.-O.; Reutter, H.; Odermatt, B.\nScore: 1.8, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554100\nIsl1 has been described as an embryonic master control gene expressed in the pericloacal mesenchyme. Deletion of Isl1 from the genital mesenchyme in mice leads to an ectopic urethral opening and epispadias-like phenotype. Using genome wide association methods, we identified ISL1 as the key susceptibility gene for classic bladder exstrophy (CBE), comprising epispadias and exstrophy of the urinary bladder. The most significant marker (rs6874700) identified in our recent GWAS meta-analysis achieved a p value of 1.48 x 10-24 within the ISL1 region. In silico analysis of rs6874700 and all other genome-wide significant markers in Linkage Disequilibrium (LD) with rs6874700 (D = 1.0; R2 \u003e 0.90) revealed marker rs2303751 (p value 8.12 x 10-20) as the marker with the highest regulatory effect predicted. Here, we describe a novel 1.2 kb intragenic promoter residing between 6.2 and 7.4 kb downstream of the ISL1 transcription starting site, which is located in the reverse DNA strand and harbors a binding side for EZH2 at the exact region of marker rs2303751. We show, that EZH2 silencing in HEK cells reduces ISL1 expression. We show that ezh2-/- ko zebrafish larvae display tissues specificity of ISL1 regulation with reduced expression of Isl1 in the pronephric region of zebrafish larvae. In addition, a shorter and malformed nephric duct is observed in ezh2-/- ko zebrafish Tg(wt1{beta}:eGFP) reporter lines. Our study shows, that Ezh2 is a key regulator of Isl1 during urinary tract formation and suggests tissue specific ISL1 dysregulation as an underlying mechanism for CBE formation.\nOverexpression and nonsynonymous mutations of UDP-glycosyltransferases potentially associated with pyrethroid resistance in Anopheles funestus\nAuthors: Al-Yazeedi, T.; Muhammad, A.; Irving, H.; Ahn, S.-J.; Hearn, J.; S. Wondji, C.\nScore: 1.5, Published: 2023-08-25 DOI: 10.1101/2023.08.25.554687\nUDP-glycosyltransferases (UGTs) constitute a superfamily of enzymes that play a vital role in the biotransformation of diverse hydrophobic substrates into more hydrophilic products, thereby facilitating their excretion from the cell through transporters. The significance of UGTs in conferring insecticide resistance has been emphasized in various insect species. In this study, we characterised Anopheles funestus UGT genes genome-wide and explored their evolution and association with pyrethroid resistance. We combined genome-wide association of pooled-template sequencing (GWAS-PoolSeq) with the transcriptomic profile of pyrethroid-resistant An. funestus populations, and deep targeted sequencing of UGTs from 80 individual mosquitoes collected in Malawi, Uganda, Cameroon and the two laboratory colonies (FANG and FUMOZ) to investigate the role of UGTs in pyrethroid resistance. We identified common overexpression of UGT310B2 (AFUN000679) in the resistant laboratory colony (FUMOZ) and resistant field populations from Malawi, Cameroon and Uganda. Significant gene-wise Fst differentiation between the resistant and putatively susceptible populations was observed for UGT301C2 and UGT302A3 in Malawi, as well as UGT306C2 in Uganda. Furthermore, the gene-wise Tajimas D density curves of the sequenced regions provided insights into genome-wide processes elucidating population structures within An. funestus populations from these three countries, supporting previous observations. Additionally, we identified significantly differentiated nonsynonymous mutations within UGT genes, which may potentially contribute to pyrethroid resistance. The identified role of An. funestus UGT genes in pyrethroid resistance has direct implications for current vector control strategies, management approaches, and the prediction of potential cross-resistance to other insecticides that can be directly detoxified by UGTs.\n",
  "wordCount" : "2675",
  "inLanguage": "en",
  "datePublished": "2023-08-27T10:37:02Z",
  "dateModified": "2023-08-27T10:37:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on August 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.552929">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.552929" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.552929">
        <p class="paperTitle">Enhancing In Vitro SMN Protein Expression and Cell Viability through Xeno-Nucleic Acid-Based ASOs in Spinal Muscular Atrophy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.552929" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.552929" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kilic, O.; Babar, H. A.; Inci, C. C.; Odabas, S. P.; Yelgen, G.; Oltan, S.; Kulac, S.; Tastan, C.</p>
        <p class="info">Score: 6.9, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.552929' target='https://doi.org/10.1101/2023.08.21.552929'> 10.1101/2023.08.21.552929</a></p>
        <p class="abstract">Spinal Muscular Atrophy (SMA) stands as a devastating ailment arising from the dearth of functional SMN (Survival Motor Neuron) protein due to genetic anomalies within the SMN1 gene. This condition is marked by the consequential attrition of motor neurons, precipitating a progressive decline in muscular strength and culminating in the disruption of neuromuscular junctions. Existing therapeutic approaches encompassing Zolgensma, Nursinersen, and Evrysdi employ innovative genetic therapeutic strategies involving transgene delivery, Antisense Oligonucleotide (ASO) technology, and modulation of pre-mRNA processing to enhance functional SMN protein expression. However, the ASO therapeutics remain suboptimal in establishing a sustained panacea for SMA, as they inadequately maintain consistent levels of functional SMN protein expression. In this study, we present a discerning inquiry into focusing on XNA-DNA-ASO products that exhibit enhanced safety and stability compared to conventional DNA/RNA-ASO sequences. Through precise targeting of the ISSN-1 region within SMN2 genes intron 7, our approach seeks to amplify SMN protein expression. Employing Xeno Nucleic Acid (XNA) bases, known for their augmented hydrophobicity and stability, our strategy surmounts previous limitations associated with chemical modifications, showcasing heightened endonuclease resistance. Comparative analyses with conventional DNA/RNA-ASO products substantiate the superiority of XNA-DNA-ASO sequences, underscoring elevated SMN protein expression and reduced toxicity. In a comprehensive evaluation, our gene therapy paradigm is scrutinized within a type 1 SMA fibroblast cell line. Utilizing diverse analytical methodologies, encompassing Annexin V-PI analysis for cytotoxicity, MTT assays for mitochondrial activity, and flow cytometry for SMN protein expression profile, we gauge therapeutic impact and potential toxicity. In conclusion, our investigation not only spotlights the promise of XNA-DNA-ASO sequences but also holds implications for refining SMA treatment strategies, converging on minimized dosages, lowered toxicity, and heightened therapeutic efficacy, thus shaping the landscape of gene therapy for SMA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.553157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.553157" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.553157">
        <p class="paperTitle">Variant-to-function analysis of the childhood obesity chr12q13 locus implicates rs7132908 as a causal variant within the 3&#39; UTR of FAIM2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.553157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.553157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Littleton, S. H.; Trang, K. B.; Volpe, C. M.; Cook, K.; DeBruyne, N.; Maguire, J. A.; Weidekamp, M. A.; Boehm, K.; Chesi, A.; Pippin, J. A.; Anderson, S. A.; Wells, A. D.; Pahl, M. C.; Grant, S. F.</p>
        <p class="info">Score: 6.8, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.553157' target='https://doi.org/10.1101/2023.08.21.553157'> 10.1101/2023.08.21.553157</a></p>
        <p class="abstract">The ch12q13 obesity locus is among the most significant childhood obesity loci identified in genome-wide association studies. This locus resides in a non-coding region within FAIM2; thus, the underlying causal variant(s) presumably influence disease susceptibility via an influence on cis-regulation within the genomic region. We implicated rs7132908 as a putative causal variant at this locus leveraging a combination of our inhouse 3D genomic data, public domain datasets, and several computational approaches. Using a luciferase reporter assay in human primary astrocytes, we observed allele-specific cis-regulatory activity of the immediate region harboring rs7132908. Motivated by this finding, we went on to generate isogenic human embryonic stem cell lines homozygous for either rs7132908 allele with CRISPR-Cas9 homology-directed repair to assess changes in gene expression due to genotype and chromatin accessibility throughout a differentiation to hypothalamic neurons, a key cell type known to regulate feeding behavior. We observed that the rs7132908 obesity risk allele influenced the expression of FAIM2 along with other genes, decreased the proportion of neurons produced during differentiation, up-regulated cell death gene sets, and conversely down-regulated neuron differentiation gene sets. We have therefore functionally validated rs7132908 as a causal obesity variant which temporally regulates nearby effector genes at the ch12q13 locus and influences neurodevelopment and survival.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553367">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553367" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553367">
        <p class="paperTitle">Hemin decreases cellular aging and enhances healthspan and lifespan through the AMPK pathway</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553367" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553367" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Y.; Naaz, A.; Faidzinn, N. A.; Yogasundaram, S.; Cheng, T. Y. N.; Jing, J. L.; Morel Gan, I. W.-H. J.; Junqi, C.; Alfatah, M.</p>
        <p class="info">Score: 7.3, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553367' target='https://doi.org/10.1101/2023.08.15.553367'> 10.1101/2023.08.15.553367</a></p>
        <p class="abstract">The quest to understand and manipulate the mechanisms of cellular aging has far-reaching implications for improving human health and longevity. Our comprehensive effort has led to the discovery of the intriguing anti-aging potential of hemin, an FDA-approved drug primarily used for the treatment of acute intermittent porphyria. Leveraging both yeast and human cell models, we investigate the multifaceted effects of hemin on extending cellular lifespan. Intriguingly, the involvement of the AMPK pathway emerges as a pivotal mechanism underlying hemins anti-aging effects. The exploration of hemins impact on cellular functionality further uncovers its influence on mitochondrial processes. Notably, both mitochondrial-dependent and -independent mechanisms are implicated in hemins ability to extend cellular lifespan, with autophagy playing a significant role in the latter. Additionally, a striking synergy between hemin and the TORC1 inhibitor rapamycin is unveiled, underlining the complexity of cellular signaling networks involved in lifespan extension. Translating these findings to human cells, hemin demonstrates an analogous ability to induce mitochondrial biogenesis, reduce proinflammatory cytokine expression, and enhance antioxidant response. The conservation of hemins anti-aging effects across species holds promise for therapeutic applications in addressing age-related diseases and promoting healthier aging.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554659">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554659" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554659">
        <p class="paperTitle">Genetic variation at 11q23.1 confers colorectal cancer risk by dysregulation of colonic tuft cell transcriptional activator POU2AF2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554659" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554659" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rajasekaran, V.; Harris, B. T.; Osborn, R. T.; Smillie, C.; Donnelly, K.; Bacou, M.; Esiri-Bloom, E.; Ooi, L. Y.; Allan, M.; Walker, M.; Reid, S.; Meynert, A.; Grimes, G.; Blackmur, J. P.; Vaughan-Shaw, P. G.; Law, P. J.; Fernandez-Rozadilla, C.; Tomlinson, I. P.; Houlston, R.; Myant, K. B.; Din, F. V. N.; Dunlop, M. G.; Farrington, S. M.</p>
        <p class="info">Score: 6.3, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554659' target='https://doi.org/10.1101/2023.08.24.554659'> 10.1101/2023.08.24.554659</a></p>
        <p class="abstract">Common genetic variation at 11q23.1 is associated with colorectal cancer (CRC) risk, and exerts local (cis) expression quantitative trait locus (cis-eQTL) effects on POU2AF2, COLCA1 and POU2AF3 genes. However, complex linkage disequilibrium and correlated expression at the 11q23.1 locus has thus far hindered elucidation of the mechanisms by which genetic variants impart CRC risk. Here, we establish that rs3087967 is the likely causal eQTL at this locus, co-localising with expression of POU2AF2 and CRC risk. Furthermore, we show trans-eQTL effects on 21 distant target genes, which are highly enriched for Tuft cell markers. Analysis of available scRNAseq, ChIPseq and scATACseq data implicates POU2AF2 as the primary controller of the tuft cell specific trans-genes through POU2F3-correlated genetic regulation. Immunofluorescence demonstrates that the rs3087967 risk genotype (T) is associated with lower tuft cell abundance in human colonic epithelium. CRISPR-mediated deletion of the 11q23.1 risk locus in the mouse germline exacerbated the ApcMin/&#43; mouse phenotype upon abrogation of Pou2af2 expression specifically. Taken together, we implicate a key protective role of tuft cells in the large bowel and the importance of mis-regulation of POU2AF2 as the prime tuft cell transcriptional activator at this locus.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553567">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553567" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553567">
        <p class="paperTitle">CryptoCEN: A Co-Expression Network for Cryptococcus neoformans reveals novel proteins involved in DNA damage repair</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553567" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553567" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: O&#39;Meara, M. J.; Rapala, J. R.; Nichols, C. B.; Alexandre, C.; Billmyre, R. B.; Steenwyk, J. L.; Alspaugh, J. A.; O&#39;Meara, T. R.</p>
        <p class="info">Score: 9.0, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553567' target='https://doi.org/10.1101/2023.08.17.553567'> 10.1101/2023.08.17.553567</a></p>
        <p class="abstract">Elucidating gene function is a major goal in biology, especially among non-model organisms. However, doing so is complicated by the fact that molecular conservation does not always mirror functional conservation, and that complex relationships among genes are responsible for encoding pathways and higher-order biological processes. Co-expression, a promising approach for predicting gene function, relies on the general principal that genes with similar expression patterns across multiple conditions will likely be involved in the same biological process. For Cryptococcus neoformans, a prevalent human fungal pathogen greatly diverged from model yeasts, approximately 60% of the predicted genes in the genome lack functional annotations. Here, we leveraged a large amount of publicly available transcriptomic data to generate a C. neoformans Co-Expression Network (CryptoCEN), successfully recapitulating known protein networks, predicting gene function, and enabling insights into the principles influencing co-expression. With 100% predictive accuracy, we used CryptoCEN to identify 13 new DNA damage response genes, underscoring the utility of guilt-by-association for determining gene function. Overall, co-expression is a powerful tool for uncovering gene function, and decreases the experimental tests needed to identify functions for currently under-annotated genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549924">
        <p class="paperTitle">Glial-derived mitochondrial signals impact neuronal proteostasis and aging.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bar-Ziv, R.; Dutta, N.; Hruby, A.; Sukarto, E.; Henderson, H. R.; Durieux, J.; Tronnes, S. U.; Averbukh, M.; Alcala, A.; Ahmad, Q.; Bolas, T.; Perez, J.; Dishart, J. G.; Vega, M.; Garcia, G.; Higuchi-Sanabria, R.; Dillin, A.</p>
        <p class="info">Score: 7.1, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549924' target='https://doi.org/10.1101/2023.07.20.549924'> 10.1101/2023.07.20.549924</a></p>
        <p class="abstract">The nervous system plays a critical role in maintaining whole-organism homeostasis; neurons experiencing mitochondrial stress can coordinate the induction of protective cellular pathways, such as the mitochondrial unfolded protein response (UPRMT), between tissues. However, these studies largely ignored non-neuronal cells of the nervous system. Here, we found that UPRMT activation in four, astrocyte-like glial cells in the nematode, C. elegans, can promote protein homeostasis by alleviating protein aggregation in neurons. Surprisingly, we find that glial cells utilize small clear vesicles (SCVs) to signal to neurons, which then relay the signal to the periphery using dense-core vesicles (DCVs). This work underlines the importance of glia in establishing and regulating protein homeostasis within the nervous system, which can then impact neuron-mediated effects in organismal homeostasis and longevity.

One-Sentence SummaryGlial cells sense mitochondrial stress and signal a beneficial stress signal to promote neuronal health and longevity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.553861">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.553861" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.553861">
        <p class="paperTitle">Identification and functional characterisation of a locus for target site integration in Fusarium graminearum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.553861" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.553861" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Darino, M.; Urban, M.; Kaur, N.; Machado-Wood, A.; Grimwade-Mann, M.; Smith, D.; Beacham, A.; Hammond-kosack, K.</p>
        <p class="info">Score: 2.9, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.553861' target='https://doi.org/10.1101/2023.08.20.553861'> 10.1101/2023.08.20.553861</a></p>
        <p class="abstract">BackgroundFusarium Head Blight is a destructive floral disease of different cereal crops. The Ascomycete fungus Fusarium graminearum (Fg) is one of the main causal agents of FHB in wheat and barley. The role(s) in virulence of Fg genes include genetic studies that involve the transformation of the fungus with different expression cassettes. We have observed in several studies where Fg genes functions were characterised that integration of expression cassettes occurred randomly. Random insertion of a cassette may disrupt gene expression and/or protein functions and hence the overall conclusion of the study. Target site integration (TSI) is an approach that consists in identifying a chromosomal region where the cassette can be inserted. The identification of a suitable locus for TSI in Fg would avert the potential risks of ectopic integration.

ResultsHere, we identified a highly conserved intergenic region on chromosome 1 suitable for TSI. We named this intergenic region the TSI locus 1. We developed an efficient cloning vector system based on the Golden Gate method to clone different expression cassettes for use in combination with TSI locus 1. We present evidence that integrations in the TSI locus 1 affects neither fungal virulence nor fungal growth under different stress conditions. Integrations at the TSI locus 1 resulted in the expression of different gene fusions. In addition, the activities of Fg native promoters were not altered by integration into the TSI locus 1. We have developed a bespoke bioinformatic pipeline to analyse the existence of ectopic integrations and tandem insertions of the cassette that may occurred during the transformation process. Finally, we established a protocol to study protein secretion in wheat coleoptiles using confocal microscopy and the TSI locus 1.

ConclusionThe TSI locus 1 can be used in Fg and potentially other cereal infecting Fusarium species for diverse studies including promoter activity analysis, secretion, protein localisation studies and gene complementation. The bespoke bioinformatic pipeline developed in this work can be an alternative to southern blotting, the gold standard technique to identify ectopic integration and tandem insertions in fungal transformation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554285">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554285" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554285">
        <p class="paperTitle">Admixture as a source for HLA variation in Neolithic European farming communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554285" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554285" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: da Silva, N. A.; Ozer, O.; Haller, M.; Chen, Y.-R.; Kolbe, D.; Schade-Linding, S.; Wahl, J.; Berszin, C.; Francken, M.; Gorner, I.; Schierhold, K.; Pechtl, J.; Grupe, G.; Rinne, C.; Muller, J.; Lenz, T. L.; Nebel, A.; Krause-Kyora, B.</p>
        <p class="info">Score: 2.4, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554285' target='https://doi.org/10.1101/2023.08.23.554285'> 10.1101/2023.08.23.554285</a></p>
        <p class="abstract">The northern European Neolithic is characterized by two major demographic events: immigration of early farmers (EF) from Anatolia (5500 BCE) and their admixture (from ~4200 BCE) with western hunter-gatherers (WHG) forming late farmers (LF). The influence of this admixture event on variation in the immune-relevant human leukocyte antigen (HLA) region is understudied. Here, we conducted population and immunogenetic analyses on 83 individuals from six EF and LF sites located in present-day Germany. We observed significant shifts in HLA allele frequencies from EF to LF. The HLA diversity increased from EF to LF, likely due to admixture with WHG. However, it was considerably lower than in modern populations. Both EF and LF exhibited a relatively narrow HLA allele spectrum compared to today. This coincides with sparse traces of pathogen DNA, potentially indicating a lower pathogen pressure at the time. We additionally noted that LF resulted from sex-biased admixture from male WHG.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554100">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554100" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554100">
        <p class="paperTitle">EZH2 specifically regulates ISL1 during embryonic urinary tract formation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554100" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554100" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mingrado, E.; Kalanithy, J.; Dworschak, G.; Ishorst, N.; Yilmaz, O.; Lindenberg, T.; Hollstein, R.; Felger, T.; Angrand, P.-O.; Reutter, H.; Odermatt, B.</p>
        <p class="info">Score: 1.8, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554100' target='https://doi.org/10.1101/2023.08.21.554100'> 10.1101/2023.08.21.554100</a></p>
        <p class="abstract">Isl1 has been described as an embryonic master control gene expressed in the pericloacal mesenchyme. Deletion of Isl1 from the genital mesenchyme in mice leads to an ectopic urethral opening and epispadias-like phenotype. Using genome wide association methods, we identified ISL1 as the key susceptibility gene for classic bladder exstrophy (CBE), comprising epispadias and exstrophy of the urinary bladder. The most significant marker (rs6874700) identified in our recent GWAS meta-analysis achieved a p value of 1.48 x 10-24 within the ISL1 region. In silico analysis of rs6874700 and all other genome-wide significant markers in Linkage Disequilibrium (LD) with rs6874700 (D = 1.0; R2 &gt; 0.90) revealed marker rs2303751 (p value 8.12 x 10-20) as the marker with the highest regulatory effect predicted. Here, we describe a novel 1.2 kb intragenic promoter residing between 6.2 and 7.4 kb downstream of the ISL1 transcription starting site, which is located in the reverse DNA strand and harbors a binding side for EZH2 at the exact region of marker rs2303751. We show, that EZH2 silencing in HEK cells reduces ISL1 expression. We show that ezh2-/- ko zebrafish larvae display tissues specificity of ISL1 regulation with reduced expression of Isl1 in the pronephric region of zebrafish larvae. In addition, a shorter and malformed nephric duct is observed in ezh2-/- ko zebrafish Tg(wt1{beta}:eGFP) reporter lines. Our study shows, that Ezh2 is a key regulator of Isl1 during urinary tract formation and suggests tissue specific ISL1 dysregulation as an underlying mechanism for CBE formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554687">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554687" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554687">
        <p class="paperTitle">Overexpression and nonsynonymous mutations of UDP-glycosyltransferases potentially associated with pyrethroid resistance in Anopheles funestus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554687" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554687" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Al-Yazeedi, T.; Muhammad, A.; Irving, H.; Ahn, S.-J.; Hearn, J.; S. Wondji, C.</p>
        <p class="info">Score: 1.5, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554687' target='https://doi.org/10.1101/2023.08.25.554687'> 10.1101/2023.08.25.554687</a></p>
        <p class="abstract">UDP-glycosyltransferases (UGTs) constitute a superfamily of enzymes that play a vital role in the biotransformation of diverse hydrophobic substrates into more hydrophilic products, thereby facilitating their excretion from the cell through transporters. The significance of UGTs in conferring insecticide resistance has been emphasized in various insect species. In this study, we characterised Anopheles funestus UGT genes genome-wide and explored their evolution and association with pyrethroid resistance. We combined genome-wide association of pooled-template sequencing (GWAS-PoolSeq) with the transcriptomic profile of pyrethroid-resistant An. funestus populations, and deep targeted sequencing of UGTs from 80 individual mosquitoes collected in Malawi, Uganda, Cameroon and the two laboratory colonies (FANG and FUMOZ) to investigate the role of UGTs in pyrethroid resistance. We identified common overexpression of UGT310B2 (AFUN000679) in the resistant laboratory colony (FUMOZ) and resistant field populations from Malawi, Cameroon and Uganda. Significant gene-wise Fst differentiation between the resistant and putatively susceptible populations was observed for UGT301C2 and UGT302A3 in Malawi, as well as UGT306C2 in Uganda. Furthermore, the gene-wise Tajimas D density curves of the sequenced regions provided insights into genome-wide processes elucidating population structures within An. funestus populations from these three countries, supporting previous observations. Additionally, we identified significantly differentiated nonsynonymous mutations within UGT genes, which may potentially contribute to pyrethroid resistance. The identified role of An. funestus UGT genes in pyrethroid resistance has direct implications for current vector control strategies, management approaches, and the prediction of potential cross-resistance to other insecticides that can be directly detoxified by UGTs.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
