<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Clark (2023) and the Persistence of Hereditarian Fallacies
Authors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.
Score: 88.2, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565061
Clark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people &#34;remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England&#34;; and (3) social mobility has remained static over this time period due to strong assortative mating on a &#34;social genotype.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Clark (2023) and the Persistence of Hereditarian Fallacies
Authors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.
Score: 88.2, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565061
Clark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people &#34;remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England&#34;; and (3) social mobility has remained static over this time period due to strong assortative mating on a &#34;social genotype." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Clark (2023) and the Persistence of Hereditarian Fallacies
Authors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.
Score: 88.2, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565061
Clark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people &#34;remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England&#34;; and (3) social mobility has remained static over this time period due to strong assortative mating on a &#34;social genotype."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Clark (2023) and the Persistence of Hereditarian Fallacies\nAuthors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.\nScore: 88.2, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565061\nClark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people \u0026#34;remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England\u0026#34;; and (3) social mobility has remained static over this time period due to strong assortative mating on a \u0026#34;social genotype.",
  "keywords": [
    
  ],
  "articleBody": " Clark (2023) and the Persistence of Hereditarian Fallacies\nAuthors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.\nScore: 88.2, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565061\nClark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people \"remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England\"; and (3) social mobility has remained static over this time period due to strong assortative mating on a \"social genotype.\" These conclusions are based on a misconstrual of model parameters, which conflates genetic and non-genetic transmission (e.g. of wealth) within families. As we show, there is strong confounding of genetic and non-genetic sources of similarity in these data. Inconsistent with claims (2) and (3), we show that familial correlations in status are variable--generally decreasing--through the time period analyzed. Lastly, we find that statistical artifacts substantially bias estimates of familial correlations in the paper. Overall, Clark (2023) provides no information about the relative contribution of genetic and non-genetic factors to social status.\nAssessing the impact of 20th century internal migrations on the genetic structure of Estonia\nAuthors: Kuznetsov, I. A.; Estonian Biobank Research Team, ; Metspalu, M.; Vainik, U.; Pagani, L.; Montinaro, F.; Pankratov, V.\nScore: 22.3, Published: 2023-10-29 DOI: 10.1101/2023.10.25.564036\nSpatial genetic structure observed in many human populations is in large part attributed to past demographic events and isolation by distance. However, how intensifying migration affects this structure remains understudied. Here we harness a sample of more than 180 thousand individuals to explore the genetic correlates and consequences of contemporary migrations in Estonia. While we show that migration smoothens the genome-wide genetic structure, it intensifies inter-regional differences in polygenic scores (PGS) for certain traits, derived both from population as well as within-sibship studies. The strongest effect is observed for educational attainment which is consistent with previous observations in the UK and suggests this to be a general pattern. We explore those regional differences in PGS in terms of the driving forces behind them and from a temporal perspective, and suggest urbanisation as a major driver for this pattern in Estonia from at least the first half of the 20th century.\nA statistical framework for powerful multi-trait rare variant analysis in large-scale whole-genome sequencing studies\nAuthors: Li, X.; Chen, H.; Selvaraj, M. S.; Van Buren, E.; Zhou, H.; Wang, Y.; Sun, R.; McCaw, Z. R.; Yu, Z.; Arnett, D. K.; Bis, J. C.; Blangero, J.; Boerwinkle, E.; Bowden, D. W.; Brody, J. A.; Cade, B. E.; Carson, A. P.; Carlson, J. C.; Chami, N.; Chen, Y.-D. I.; Curran, J. E.; de Vries, P. S.; Fornage, M.; Franceschini, N.; Freedman, B. I.; Gu, C.; Heard-Costa, N. L.; He, J.; Hou, L.; Hung, Y.-J.; Irvin, M. R.; Kaplan, R. C.; Kardia, S. L. R.; Kelly, T.; Konigsberg, I.; Kooperberg, C.; Kral, B. G.; Li, C.; Loos, R. J. F.; Mahaney, M. C.; Martin, L. W.; Mathias, R. A.; Minster, R. L.; Mitchell, B. D\nScore: 14.6, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564764\nLarge-scale whole-genome sequencing (WGS) studies have improved our understanding of the contributions of coding and noncoding rare variants to complex human traits. Leveraging association effect sizes across multiple traits in WGS rare variant association analysis can improve statistical power over single-trait analysis, and also detect pleiotropic genes and regions. Existing multi-trait methods have limited ability to perform rare variant analysis of large-scale WGS data. We propose MultiSTAAR, a statistical framework and computationally-scalable analytical pipeline for functionally-informed multi-trait rare variant analysis in large-scale WGS studies. MultiSTAAR accounts for relatedness, population structure and correlation among phenotypes by jointly analyzing multiple traits, and further empowers rare variant association analysis by incorporating multiple functional annotations. We applied MultiSTAAR to jointly analyze three lipid traits (low-density lipoprotein cholesterol, high-density lipoprotein cholesterol and triglycerides) in 61,861 multi-ethnic samples from the Trans-Omics for Precision Medicine (TOPMed) Program. We discovered new associations with lipid traits missed by single-trait analysis, including rare variants within an enhancer of NIPSNAP3A and an intergenic region on chromosome 1.\nPitfalls in performing genome-wide association studies on ratio traits\nAuthors: McCaw, Z. R.; Dey, R.; Somineni, H.; Amar, D.; Mukherjee, S.; Sandor, K.; insitro Research Team, ; Karaletsos, T.; Koller, D.; Davey Smith, G.; MacArthur, D.; O'Dushlaine, C.; Soare, T. W.\nScore: 13.6, Published: 2023-11-01 DOI: 10.1101/2023.10.27.564385\nGenome-wide association studies (GWAS) are often performed on ratios composed of a numerator trait divided by a denominator trait. Examples include body mass index (BMI) and the waist-to-hip ratio, among many others. Explicitly or implicitly, the goal of forming the ratio is typically to adjust the numerator for the denominator. While forming ratios may be clinically expedient, there are several important issues with performing GWAS on ratios. Forming a ratio does not \"adjust\" for the denominator in the sense of holding it constant, and it is unclear whether associations with ratios are attributable to the numerator, the denominator, or both. Here we demonstrate that associations arising in ratio GWAS can be entirely denominator-driven, implying that at least some associations uncovered by ratio GWAS may be due solely to a putative adjustment variable. In a survey of 10 exemplar ratios, we find that the ratio model disagrees with the adjusted model (performing GWAS on the numerator while conditioning on the denominator) at around 1/3 of loci. Using BMI as an example, we show that variants detected by only the ratio model are more strongly associated with the denominator (height), while variants detected by only the adjusted model are more strongly associated with the numerator (weight). Although the adjusted model provides effect sizes with a clearer interpretation, it is susceptible to collider bias. We propose and validate a simple method of correcting for the genetic collider bias via leave-one-chromosome-out polygenic scoring.\nA general and efficient representation of ancestral recombination graphs\nAuthors: Wong, Y.; Ignatieva, A.; Koskela, J.; Gorjanc, G.; Wohns, A. W.; Kelleher, J.\nScore: 12.6, Published: 2023-11-04 DOI: 10.1101/2023.11.03.565466\nAs a result of recombination, adjacent nucleotides can have different paths of genetic inheritance and therefore the genealogical trees for a sample of DNA sequences vary along the genome. The structure capturing the details of these intricately interwoven paths of inheritance is referred to as an ancestral recombination graph (ARG). New developments have made it possible to infer ARGs at scale, enabling many new applications in population and statistical genetics. This rapid progress, however, has led to a substantial gap opening between theory and practice. Standard mathematical formalisms, based on exhaustively detailing the \"events\" that occur in the history of a sample, are insufficient to describe the outputs of current methods. Moreover, we argue that the underlying assumption that all events can be known and precisely estimated is fundamentally unsuited to the realities of modern, population-scale datasets. We propose an alternative mathematical formulation that encompasses the outputs of recent methods and can capture the full richness of modern large-scale datasets. By defining this ARG encoding in terms of specific genomes and their intervals of genetic inheritance, we avoid the need to exhaustively list (and estimate) all events. The effects of multiple events can be aggregated in different ways, providing a natural way to express many forms of approximate and partial knowledge about the recombinant ancestry of a sample.\nTranscription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency\nAuthors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.\nScore: 45.6, Published: 2023-10-21 DOI: 10.1101/2023.10.04.560808\nEctopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming. Using neural networks that map DNA sequence to ATAC-seq profiles at base-resolution, we annotated cell-state-specific predictive transcription factor (TF) motif syntax in regulatory elements, inferred affinity- and concentration-dependent dynamics of Tn5-bias corrected TF footprints, linked peaks to putative target genes, and elucidated rewiring of TF- to-gene cis-regulatory networks. Our models reveal that early in reprogramming, OSK, at supraphysiological concentrations, rapidly open transient regulatory elements by occupying non-canonical low-affinity binding sites. As OSK concentration falls, the accessibility of these transient elements decays as a function of motif affinity. We find that these OSK-dependent transient elements sequester the somatic TF AP-1. This redistribution is strongly associated with the silencing of fibroblast-specific genes within individual nuclei. Together, our integrated single-cell resource and models reveal insights into the cis-regulatory code of reprogramming at unprecedented resolution, connect TF stoichiometry and motif syntax to diversification of cell fate trajectories, and provide new perspectives on the dynamics and role of transient regulatory elements in somatic silencing.\nTrait selection strategy in multi-trait GWAS: Boosting SNPs discoverability\nAuthors: Suzuki, Y.; Menager, H.; Brancotte, B.; Vernet, R.; Nerin, C.; Boetto, C.; Auvergne, A.; Linhard, C.; Torchet, R.; Lechat, P.; Troubat, L.; Cho, M. H.; Bouzigon, E.; Aschard, H.; Julienne, H.\nScore: 10.1, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564319\nSince the first Genome-Wide Association Studies (GWAS), thousands of variant-trait associations have been discovered. However, the sample size required to detect additional variants using standard univariate association screening is increasingly prohibitive. Multi-trait GWAS offers a relevant alternative: it can improve statistical power and lead to new insights about gene function and the joint genetic architecture of human phenotypes. Although many methodological hurdles of multi-trait testing have been discussed, the strategy to select trait, among overwhelming possibilities, has been overlooked. In this study, we conducted extensive multi-trait tests using JASS (Joint Analysis of Summary Statistics) and assessed which genetic features of the analysed sets were associated with anincreased detection of variants as compared to univariate screening. Our analyses identified multiple factors associated with the gain in the association detection in multi-trait tests. Together, these factors of the analysed sets are predictive of the gain of the multi-trait test (Pearsons {rho} equal to 0.43 between the observed and predicted gain, P \u003c 1.6 x 10-60). Applying an alternative multi-trait approach (MTAG, multi-trait analysis of GWAS), we found that in most scenarios but particularly those with larger numbers of traits, JASS outperformed MTAG. Finally, we benchmark several strategies to select set of traits including the prevalent strategy of selecting clinically similar traits, which systematically underperformed selecting clinically heterogenous traits or selecting sets that issued from our data-driven models. This work provides a unique picture of the determinant of multi-trait GWAS statistical power and outline practical strategies for multi-trait testing.\nGenotype prediction of 336,463 samples from public expression data\nAuthors: Razi, A.; Lo, C. C.; Wang, S.; Leek, J. T.; Hansen, K. D.\nScore: 17.9, Published: 2023-10-22 DOI: 10.1101/2023.10.21.562237\nTens of thousands of RNA-sequencing experiments comprising hundreds of thousands of individual samples have now been performed. These data represent a broad range of experimental conditions, sequencing technologies, and hypotheses under study. The Recount project has aggregated and uniformly processed hundreds of thousands of publicly available RNA-seq samples. Most of these samples only include RNA expression measurements; genotype data for these same samples would enable a wide range of analyses including variant prioritization, eQTL analysis, and studies of allele specific expression. Here, we developed a statistical model based on the existing reference and alternative read counts from the RNA-seq experiments available through Recount3 to predict genotypes at autosomal biallelic loci in coding regions. We demonstrate the accuracy of our model using large-scale studies that measured both gene expression and genotype genome-wide. We show that our predictive model is highly accurate with 99.5% overall accuracy, 99.6% major allele accuracy, and 90.4% minor allele accuracy. Our model is robust to tissue and study effects, provided the coverage is high enough. We applied this model to genotype all the samples in Recount3 and provide the largest ready-to-use expression repository containing genotype information. We illustrate that the predicted genotype from RNA-seq data is sufficient to unravel the underlying population structure of samples in Recount3 using Principal Component Analysis.\nLarge-scale causal discovery using interventional data sheds light on the regulatory network architecture of blood traits\nAuthors: Brown, B. C.; Morris, J. A.; Lappalainen, T.; Knowles, D. A.\nScore: 8.0, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562293\nInference of directed biological networks is an important but notoriously challenging problem. We introduce inverse sparse regression (inspre), an approach to learning causal networks that leverages large-scale intervention-response data. Applied to 788 genes from the genome-wide perturb-seq dataset, inspre helps elucidate the network architecture of blood traits.\nDrosophila models of PIGA-CDG mirror patient phenotypes\nAuthors: Thorpe, H. J.; Owings, K. G.; Aziz, M. C.; Haller, M.; Coelho, E.; Chow, C. Y.\nScore: 6.4, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564441\nMutations in the phosphatidylinositol glycan biosynthesis class A (PIGA) gene cause a rare, X-linked recessive congenital disorder of glycosylation (CDG). PIGA-CDG is characterized by seizures, intellectual and developmental delay, and congenital malformations. The PIGA gene encodes an enzyme involved in the first step of GPI anchor biosynthesis. There are over 100 GPI anchored proteins that attach to the cell surface and are involved in cell signaling, immunity, and adhesion. Little is known about the pathophysiology of PIGA-CDG. Here we describe the first Drosophila model of PIGA-CDG and demonstrate that loss of PIG-A function in Drosophila accurately models the human disease. As expected, complete loss of PIG-A function is larval lethal. Heterozygous null animals appear healthy, but when challenged, have a seizure phenotype similar to what is observed in patients. To identify the cell-type specific contributions to disease, we generated neuron- and glia-specific knockdown of PIG-A. Neuron-specific knockdown resulted in reduced lifespan and a number of neurological phenotypes, but no seizure phenotype. Glia-knockdown also reduced lifespan and, notably, resulted in a very strong seizure phenotype. RNAseq analyses demonstrated that there are fundamentally different molecular processes that are disrupted when PIG-A function is eliminated in different cell types. In particular, loss of PIG-A in neurons resulted in upregulation of glycolysis, but loss of PIG-A in glia resulted in upregulation of protein translation machinery. Here we demonstrate that Drosophila is a good model of PIGA-CDG and provide new data resources for future study of PIGA-CDG and other GPI anchor disorders. Article SummaryPIGA-CDG is a rare genetic disorder. In order to study this rare disease, we generated and characterized several Drosophila models of PIGA-CDG. These models faithfully recapitulate different patient phenotypes, including movement disorder and seizures. Drosophila is a good model for PIGA-CDG and other GPI anchor disorders.\n",
  "wordCount" : "2433",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565061">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565061" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565061">
        <p class="paperTitle">Clark (2023) and the Persistence of Hereditarian Fallacies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565061" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565061" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benning, J. W.; Carlson, J.; Shaw, R.; Harpak, A. W.</p>
        <p class="info">Score: 88.2, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565061' target='https://doi.org/10.1101/2023.11.01.565061'> 10.1101/2023.11.01.565061</a></p>
        <p class="abstract">Clark (2023) considers the similarity in socioeconomic status between relatives, drawing on records spanning four centuries in England. The paper adapts a classic quantitative genetics model in order to argue the fit of the model to the data suggests that: (1) variation in socioeconomic status is largely determined by additive genetic variation; (2) contemporary English people &#34;remain correlated in outcomes with their lineage relatives in exactly the same way as in preindustrial England&#34;; and (3) social mobility has remained static over this time period due to strong assortative mating on a &#34;social genotype.&#34; These conclusions are based on a misconstrual of model parameters, which conflates genetic and non-genetic transmission (e.g. of wealth) within families. As we show, there is strong confounding of genetic and non-genetic sources of similarity in these data. Inconsistent with claims (2) and (3), we show that familial correlations in status are variable--generally decreasing--through the time period analyzed. Lastly, we find that statistical artifacts substantially bias estimates of familial correlations in the paper. Overall, Clark (2023) provides no information about the relative contribution of genetic and non-genetic factors to social status.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564036">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564036" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564036">
        <p class="paperTitle">Assessing the impact of 20th century internal migrations on the genetic structure of Estonia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564036" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564036" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kuznetsov, I. A.; Estonian Biobank Research Team,  ; Metspalu, M.; Vainik, U.; Pagani, L.; Montinaro, F.; Pankratov, V.</p>
        <p class="info">Score: 22.3, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564036' target='https://doi.org/10.1101/2023.10.25.564036'> 10.1101/2023.10.25.564036</a></p>
        <p class="abstract">Spatial genetic structure observed in many human populations is in large part attributed to past demographic events and isolation by distance. However, how intensifying migration affects this structure remains understudied. Here we harness a sample of more than 180 thousand individuals to explore the genetic correlates and consequences of contemporary migrations in Estonia. While we show that migration smoothens the genome-wide genetic structure, it intensifies inter-regional differences in polygenic scores (PGS) for certain traits, derived both from population as well as within-sibship studies. The strongest effect is observed for educational attainment which is consistent with previous observations in the UK and suggests this to be a general pattern. We explore those regional differences in PGS in terms of the driving forces behind them and from a temporal perspective, and suggest urbanisation as a major driver for this pattern in Estonia from at least the first half of the 20th century.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564764" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564764">
        <p class="paperTitle">A statistical framework for powerful multi-trait rare variant analysis in large-scale whole-genome sequencing studies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, X.; Chen, H.; Selvaraj, M. S.; Van Buren, E.; Zhou, H.; Wang, Y.; Sun, R.; McCaw, Z. R.; Yu, Z.; Arnett, D. K.; Bis, J. C.; Blangero, J.; Boerwinkle, E.; Bowden, D. W.; Brody, J. A.; Cade, B. E.; Carson, A. P.; Carlson, J. C.; Chami, N.; Chen, Y.-D. I.; Curran, J. E.; de Vries, P. S.; Fornage, M.; Franceschini, N.; Freedman, B. I.; Gu, C.; Heard-Costa, N. L.; He, J.; Hou, L.; Hung, Y.-J.; Irvin, M. R.; Kaplan, R. C.; Kardia, S. L. R.; Kelly, T.; Konigsberg, I.; Kooperberg, C.; Kral, B. G.; Li, C.; Loos, R. J. F.; Mahaney, M. C.; Martin, L. W.; Mathias, R. A.; Minster, R. L.; Mitchell, B. D</p>
        <p class="info">Score: 14.6, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564764' target='https://doi.org/10.1101/2023.10.30.564764'> 10.1101/2023.10.30.564764</a></p>
        <p class="abstract">Large-scale whole-genome sequencing (WGS) studies have improved our understanding of the contributions of coding and noncoding rare variants to complex human traits. Leveraging association effect sizes across multiple traits in WGS rare variant association analysis can improve statistical power over single-trait analysis, and also detect pleiotropic genes and regions. Existing multi-trait methods have limited ability to perform rare variant analysis of large-scale WGS data. We propose MultiSTAAR, a statistical framework and computationally-scalable analytical pipeline for functionally-informed multi-trait rare variant analysis in large-scale WGS studies. MultiSTAAR accounts for relatedness, population structure and correlation among phenotypes by jointly analyzing multiple traits, and further empowers rare variant association analysis by incorporating multiple functional annotations. We applied MultiSTAAR to jointly analyze three lipid traits (low-density lipoprotein cholesterol, high-density lipoprotein cholesterol and triglycerides) in 61,861 multi-ethnic samples from the Trans-Omics for Precision Medicine (TOPMed) Program. We discovered new associations with lipid traits missed by single-trait analysis, including rare variants within an enhancer of NIPSNAP3A and an intergenic region on chromosome 1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564385">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564385" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564385">
        <p class="paperTitle">Pitfalls in performing genome-wide association studies on ratio traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564385" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564385" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McCaw, Z. R.; Dey, R.; Somineni, H.; Amar, D.; Mukherjee, S.; Sandor, K.; insitro Research Team,  ; Karaletsos, T.; Koller, D.; Davey Smith, G.; MacArthur, D.; O&#39;Dushlaine, C.; Soare, T. W.</p>
        <p class="info">Score: 13.6, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564385' target='https://doi.org/10.1101/2023.10.27.564385'> 10.1101/2023.10.27.564385</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) are often performed on ratios composed of a numerator trait divided by a denominator trait. Examples include body mass index (BMI) and the waist-to-hip ratio, among many others. Explicitly or implicitly, the goal of forming the ratio is typically to adjust the numerator for the denominator. While forming ratios may be clinically expedient, there are several important issues with performing GWAS on ratios. Forming a ratio does not &#34;adjust&#34; for the denominator in the sense of holding it constant, and it is unclear whether associations with ratios are attributable to the numerator, the denominator, or both. Here we demonstrate that associations arising in ratio GWAS can be entirely denominator-driven, implying that at least some associations uncovered by ratio GWAS may be due solely to a putative adjustment variable. In a survey of 10 exemplar ratios, we find that the ratio model disagrees with the adjusted model (performing GWAS on the numerator while conditioning on the denominator) at around 1/3 of loci. Using BMI as an example, we show that variants detected by only the ratio model are more strongly associated with the denominator (height), while variants detected by only the adjusted model are more strongly associated with the numerator (weight). Although the adjusted model provides effect sizes with a clearer interpretation, it is susceptible to collider bias. We propose and validate a simple method of correcting for the genetic collider bias via leave-one-chromosome-out polygenic scoring.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565466">
        <p class="paperTitle">A general and efficient representation of ancestral recombination graphs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wong, Y.; Ignatieva, A.; Koskela, J.; Gorjanc, G.; Wohns, A. W.; Kelleher, J.</p>
        <p class="info">Score: 12.6, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565466' target='https://doi.org/10.1101/2023.11.03.565466'> 10.1101/2023.11.03.565466</a></p>
        <p class="abstract">As a result of recombination, adjacent nucleotides can have different paths of genetic inheritance and therefore the genealogical trees for a sample of DNA sequences vary along the genome. The structure capturing the details of these intricately interwoven paths of inheritance is referred to as an ancestral recombination graph (ARG). New developments have made it possible to infer ARGs at scale, enabling many new applications in population and statistical genetics. This rapid progress, however, has led to a substantial gap opening between theory and practice. Standard mathematical formalisms, based on exhaustively detailing the &#34;events&#34; that occur in the history of a sample, are insufficient to describe the outputs of current methods. Moreover, we argue that the underlying assumption that all events can be known and precisely estimated is fundamentally unsuited to the realities of modern, population-scale datasets. We propose an alternative mathematical formulation that encompasses the outputs of recent methods and can capture the full richness of modern large-scale datasets. By defining this ARG encoding in terms of specific genomes and their intervals of genetic inheritance, we avoid the need to exhaustively list (and estimate) all events. The effects of multiple events can be aggregated in different ways, providing a natural way to express many forms of approximate and partial knowledge about the recombinant ancestry of a sample.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560808">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560808" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560808">
        <p class="paperTitle">Transcription factor stoichiometry, motif affinity and syntax regulate single-cell chromatin dynamics during fibroblast reprogramming to pluripotency</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560808" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560808" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nair, S.; Ameen, M.; Sundaram, L.; Pampari, A.; Schreiber, J.; Balsubramani, A.; Wang, Y. X.; Burns, D.; Blau, H. M.; Karakikes, I.; Wang, K. C.; Kundaje, A.</p>
        <p class="info">Score: 45.6, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560808' target='https://doi.org/10.1101/2023.10.04.560808'> 10.1101/2023.10.04.560808</a></p>
        <p class="abstract">Ectopic expression of OCT4, SOX2, KLF4 and MYC (OSKM) transforms differentiated cells into induced pluripotent stem cells. To refine our mechanistic understanding of reprogramming, especially during the early stages, we profiled chromatin accessibility and gene expression at single-cell resolution across a densely sampled time course of human fibroblast reprogramming. Using neural networks that map DNA sequence to ATAC-seq profiles at base-resolution, we annotated cell-state-specific predictive transcription factor (TF) motif syntax in regulatory elements, inferred affinity- and concentration-dependent dynamics of Tn5-bias corrected TF footprints, linked peaks to putative target genes, and elucidated rewiring of TF- to-gene cis-regulatory networks. Our models reveal that early in reprogramming, OSK, at supraphysiological concentrations, rapidly open transient regulatory elements by occupying non-canonical low-affinity binding sites. As OSK concentration falls, the accessibility of these transient elements decays as a function of motif affinity. We find that these OSK-dependent transient elements sequester the somatic TF AP-1. This redistribution is strongly associated with the silencing of fibroblast-specific genes within individual nuclei. Together, our integrated single-cell resource and models reveal insights into the cis-regulatory code of reprogramming at unprecedented resolution, connect TF stoichiometry and motif syntax to diversification of cell fate trajectories, and provide new perspectives on the dynamics and role of transient regulatory elements in somatic silencing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564319">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564319" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564319">
        <p class="paperTitle">Trait selection strategy in multi-trait GWAS: Boosting SNPs discoverability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564319" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564319" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suzuki, Y.; Menager, H.; Brancotte, B.; Vernet, R.; Nerin, C.; Boetto, C.; Auvergne, A.; Linhard, C.; Torchet, R.; Lechat, P.; Troubat, L.; Cho, M. H.; Bouzigon, E.; Aschard, H.; Julienne, H.</p>
        <p class="info">Score: 10.1, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564319' target='https://doi.org/10.1101/2023.10.27.564319'> 10.1101/2023.10.27.564319</a></p>
        <p class="abstract">Since the first Genome-Wide Association Studies (GWAS), thousands of variant-trait associations have been discovered. However, the sample size required to detect additional variants using standard univariate association screening is increasingly prohibitive. Multi-trait GWAS offers a relevant alternative: it can improve statistical power and lead to new insights about gene function and the joint genetic architecture of human phenotypes. Although many methodological hurdles of multi-trait testing have been discussed, the strategy to select trait, among overwhelming possibilities, has been overlooked. In this study, we conducted extensive multi-trait tests using JASS (Joint Analysis of Summary Statistics) and assessed which genetic features of the analysed sets were associated with anincreased detection of variants as compared to univariate screening. Our analyses identified multiple factors associated with the gain in the association detection in multi-trait tests. Together, these factors of the analysed sets are predictive of the gain of the multi-trait test (Pearsons {rho} equal to 0.43 between the observed and predicted gain, P &lt; 1.6 x 10-60). Applying an alternative multi-trait approach (MTAG, multi-trait analysis of GWAS), we found that in most scenarios but particularly those with larger numbers of traits, JASS outperformed MTAG. Finally, we benchmark several strategies to select set of traits including the prevalent strategy of selecting clinically similar traits, which systematically underperformed selecting clinically heterogenous traits or selecting sets that issued from our data-driven models. This work provides a unique picture of the determinant of multi-trait GWAS statistical power and outline practical strategies for multi-trait testing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.21.562237">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.21.562237" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.21.562237">
        <p class="paperTitle">Genotype prediction of 336,463 samples from public expression data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.21.562237" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.21.562237" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Razi, A.; Lo, C. C.; Wang, S.; Leek, J. T.; Hansen, K. D.</p>
        <p class="info">Score: 17.9, Published: 2023-10-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.21.562237' target='https://doi.org/10.1101/2023.10.21.562237'> 10.1101/2023.10.21.562237</a></p>
        <p class="abstract">Tens of thousands of RNA-sequencing experiments comprising hundreds of thousands of individual samples have now been performed. These data represent a broad range of experimental conditions, sequencing technologies, and hypotheses under study. The Recount project has aggregated and uniformly processed hundreds of thousands of publicly available RNA-seq samples. Most of these samples only include RNA expression measurements; genotype data for these same samples would enable a wide range of analyses including variant prioritization, eQTL analysis, and studies of allele specific expression. Here, we developed a statistical model based on the existing reference and alternative read counts from the RNA-seq experiments available through Recount3 to predict genotypes at autosomal biallelic loci in coding regions. We demonstrate the accuracy of our model using large-scale studies that measured both gene expression and genotype genome-wide. We show that our predictive model is highly accurate with 99.5% overall accuracy, 99.6% major allele accuracy, and 90.4% minor allele accuracy. Our model is robust to tissue and study effects, provided the coverage is high enough. We applied this model to genotype all the samples in Recount3 and provide the largest ready-to-use expression repository containing genotype information. We illustrate that the predicted genotype from RNA-seq data is sufficient to unravel the underlying population structure of samples in Recount3 using Principal Component Analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562293" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562293">
        <p class="paperTitle">Large-scale causal discovery using interventional data sheds light on the regulatory network architecture of blood traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brown, B. C.; Morris, J. A.; Lappalainen, T.; Knowles, D. A.</p>
        <p class="info">Score: 8.0, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562293' target='https://doi.org/10.1101/2023.10.13.562293'> 10.1101/2023.10.13.562293</a></p>
        <p class="abstract">Inference of directed biological networks is an important but notoriously challenging problem. We introduce inverse sparse regression (inspre), an approach to learning causal networks that leverages large-scale intervention-response data. Applied to 788 genes from the genome-wide perturb-seq dataset, inspre helps elucidate the network architecture of blood traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564441">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564441" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564441">
        <p class="paperTitle">Drosophila models of PIGA-CDG mirror patient phenotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564441" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564441" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thorpe, H. J.; Owings, K. G.; Aziz, M. C.; Haller, M.; Coelho, E.; Chow, C. Y.</p>
        <p class="info">Score: 6.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564441' target='https://doi.org/10.1101/2023.10.27.564441'> 10.1101/2023.10.27.564441</a></p>
        <p class="abstract">Mutations in the phosphatidylinositol glycan biosynthesis class A (PIGA) gene cause a rare, X-linked recessive congenital disorder of glycosylation (CDG). PIGA-CDG is characterized by seizures, intellectual and developmental delay, and congenital malformations. The PIGA gene encodes an enzyme involved in the first step of GPI anchor biosynthesis. There are over 100 GPI anchored proteins that attach to the cell surface and are involved in cell signaling, immunity, and adhesion. Little is known about the pathophysiology of PIGA-CDG. Here we describe the first Drosophila model of PIGA-CDG and demonstrate that loss of PIG-A function in Drosophila accurately models the human disease. As expected, complete loss of PIG-A function is larval lethal. Heterozygous null animals appear healthy, but when challenged, have a seizure phenotype similar to what is observed in patients. To identify the cell-type specific contributions to disease, we generated neuron- and glia-specific knockdown of PIG-A. Neuron-specific knockdown resulted in reduced lifespan and a number of neurological phenotypes, but no seizure phenotype. Glia-knockdown also reduced lifespan and, notably, resulted in a very strong seizure phenotype. RNAseq analyses demonstrated that there are fundamentally different molecular processes that are disrupted when PIG-A function is eliminated in different cell types. In particular, loss of PIG-A in neurons resulted in upregulation of glycolysis, but loss of PIG-A in glia resulted in upregulation of protein translation machinery. Here we demonstrate that Drosophila is a good model of PIGA-CDG and provide new data resources for future study of PIGA-CDG and other GPI anchor disorders.

Article SummaryPIGA-CDG is a rare genetic disorder. In order to study this rare disease, we generated and characterized several Drosophila models of PIGA-CDG. These models faithfully recapitulate different patient phenotypes, including movement disorder and seizures. Drosophila is a good model for PIGA-CDG and other GPI anchor disorders.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
