<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Developmental disruption of the mitochondrial fissiongene drp-1 extends the longevity of daf-2 insulin/IGF-1 receptor mutant
Authors: Traa, A.; Van Raamsdonk, J.
Score: 21.4, Published: 2024-02-08 DOI: 10.1101/2024.02.07.579360
The dynamic nature of the mitochondrial network is regulated by mitochondrial fission and fusion, allowing for re-organization of mitochondria to adapt to the cells ever-changing needs. As organisms age, mitochondrial fission and fusion become dysregulated and mitochondrial networks become increasingly fragmented. Modulation of mitochondrial dynamics has been shown to affect longevity in fungi, yeast, Drosophila and C.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Developmental disruption of the mitochondrial fissiongene drp-1 extends the longevity of daf-2 insulin/IGF-1 receptor mutant
Authors: Traa, A.; Van Raamsdonk, J.
Score: 21.4, Published: 2024-02-08 DOI: 10.1101/2024.02.07.579360
The dynamic nature of the mitochondrial network is regulated by mitochondrial fission and fusion, allowing for re-organization of mitochondria to adapt to the cells ever-changing needs. As organisms age, mitochondrial fission and fusion become dysregulated and mitochondrial networks become increasingly fragmented. Modulation of mitochondrial dynamics has been shown to affect longevity in fungi, yeast, Drosophila and C." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Developmental disruption of the mitochondrial fissiongene drp-1 extends the longevity of daf-2 insulin/IGF-1 receptor mutant
Authors: Traa, A.; Van Raamsdonk, J.
Score: 21.4, Published: 2024-02-08 DOI: 10.1101/2024.02.07.579360
The dynamic nature of the mitochondrial network is regulated by mitochondrial fission and fusion, allowing for re-organization of mitochondria to adapt to the cells ever-changing needs. As organisms age, mitochondrial fission and fusion become dysregulated and mitochondrial networks become increasingly fragmented. Modulation of mitochondrial dynamics has been shown to affect longevity in fungi, yeast, Drosophila and C."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Developmental disruption of the mitochondrial fissiongene drp-1 extends the longevity of daf-2 insulin/IGF-1 receptor mutant\nAuthors: Traa, A.; Van Raamsdonk, J.\nScore: 21.4, Published: 2024-02-08 DOI: 10.1101/2024.02.07.579360\nThe dynamic nature of the mitochondrial network is regulated by mitochondrial fission and fusion, allowing for re-organization of mitochondria to adapt to the cells ever-changing needs. As organisms age, mitochondrial fission and fusion become dysregulated and mitochondrial networks become increasingly fragmented. Modulation of mitochondrial dynamics has been shown to affect longevity in fungi, yeast, Drosophila and C.",
  "keywords": [
    
  ],
  "articleBody": " Developmental disruption of the mitochondrial fissiongene drp-1 extends the longevity of daf-2 insulin/IGF-1 receptor mutant\nAuthors: Traa, A.; Van Raamsdonk, J.\nScore: 21.4, Published: 2024-02-08 DOI: 10.1101/2024.02.07.579360\nThe dynamic nature of the mitochondrial network is regulated by mitochondrial fission and fusion, allowing for re-organization of mitochondria to adapt to the cells ever-changing needs. As organisms age, mitochondrial fission and fusion become dysregulated and mitochondrial networks become increasingly fragmented. Modulation of mitochondrial dynamics has been shown to affect longevity in fungi, yeast, Drosophila and C. elegans. While disruption of the mitochondrial fission gene drp-1 only mildly increases wild-type lifespan, it drastically increases the already long lifespan of daf-2 insulin/IGF-1 signaling (IIS) mutants. In this work, we determined the conditions required for drp-1 disruption to extend daf-2 longevity and explored the molecular mechanisms involved. We found that knockdown of drp-1 during development is sufficient to extend daf-2 lifespan, while tissue-specific knockdown of drp-1 in neurons, intestine or muscle failed to increase daf-2 longevity. Disruption of other genes involved in mitochondrial fission also increased daf-2 lifespan as did treatment with a number of different RNAi clones that decrease mitochondrial fragmentation. In exploring potential mechanisms involved, we found that deletion of drp-1 increases resistance to chronic stresses and slows physiologic rates in daf-2 worms. In addition, we found that disruption of drp-1 increased mitochondrial and peroxisomal connectedness in daf-2 worms, increased oxidative phosphorylation and ATP levels, and increased mitophagy in daf-2 worms, but did not affect their ROS levels or mitochondrial membrane potential. Overall, this work defined the conditions under which drp-1 disruption increases daf-2 lifespan and has identified multiple changes in daf-2;drp-1 mutants that may contribute to their lifespan extension.\nDNA methylation modulates nucleosome retention in sperm and H3K4 methylation deposition in early mouse embryos.\nAuthors: Fanourgakis, G.; Gaspa-Toneu, L.; Komarov, P. A.; Ozonov, E. A.; Smallwood, S. A.; Peters, A. H. F. M.\nScore: 20.0, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579069\nDNA methylation (DNAme) serves a stable gene regulatory function in somatic cells (1). In the germ line and during early embryogenesis, however, DNAme undergoes global erasure and re-establishment to support germ cell and embryonic development (2). While de novo DNAme acquisition during male germ cell development is essential for setting genomic DNA methylation imprints, other intergenerational roles for paternal DNAme in defining embryonic chromatin after fertilization are unknown. To approach this question, we reduced levels of DNAme in developing male germ cells through conditional gene deletion of the de novo DNA methyltransferases DNMT3A and DNMT3B in undifferentiated spermatogonia. We observed that DNMT3A serves a DNAme maintenance function in undifferentiated spermatogonia while DNMT3B catalyzes de novo DNAme during spermatogonial differentiation. Mutant male germ cells nevertheless completed their differentiation to sperm. Failing de novo DNAme in Dnmt3a/Dnmt3b double deficient spermatogonia is associated with increased nucleosome occupancy in mature sperm, preferentially at sites with higher CpG content, supporting the model that DNAme modulates nucleosome retention in sperm (3). To assess the impact of altered sperm chromatin in the formation of embryonic chromatin, we measured H3K4me3 occupancy at paternal and maternal alleles in 2-cell embryos using a newly developed transposon-based tagging assay for modified chromatin. Our data show that reduced DNAme in sperm renders paternal alleles permissive for H3K4me3 establishment in early embryos, independently of possible paternal inheritance of sperm born H3K4me3. Together, this study provides first evidence that paternally inherited DNAme directs chromatin formation during early embryonic development.\nThe ivory lncRNA regulates seasonal color patterns in buckeye butterflies\nAuthors: Fandino, R. A.; Brady, N. K.; Chatterjee, M.; McDonald, J. M. C.; Livraghi, L.; van der burg, K.; Mazo-Vargas, A.; Reed, R. D.\nScore: 10.9, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579733\nLong non-coding RNAs (lncRNAs) are transcribed elements increasingly recognized for their roles in regulating gene expression. Thus far, however, we have little understanding of how lncRNAs contribute to evolution and adaptation. Here we show that a conserved lncRNA, ivory, is an important color patterning gene in the buckeye butterfly Junonia coenia. ivory overlaps with cortex, a locus linked to multiple cases of crypsis and mimicry in Lepidoptera. Along with the Livraghi et. al companion paper, we argue that ivory, not cortex, is the color pattern gene of interest at this locus. In J. coenia a cluster of cis-regulatory elements (CREs) in the first intron of ivory are genetically associated with natural variation in seasonal color pattern plasticity, and targeted deletions of these CREs phenocopy seasonal phenotypes. Deletions of different ivory CREs produce other distinct phenotypes as well, including loss of melanic eyespot rings, and positive and negative changes in overall wing pigmentation. We show that the color pattern transcription factors Spineless, Bric-a-brac, and Ftz-f1 bind to the ivory promoter during wing pattern development, suggesting that they directly regulate ivory. This case study demonstrates how cis-regulation of a single non-coding RNA can exert diverse and nuanced effects on the evolution and development of color patterns, including modulating seasonally plastic color patterns.\nA micro-RNA drives a 100-million-year adaptive evolution of melanic patterns in butterflies and moths\nAuthors: Tian, S.; Banerjee, T. D.; Wee, J. L. Q.; Wang, Y.; Murugesan, S. N.; Monteiro, A.\nScore: 8.0, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579741\nA long-standing question in biology is how phenotypic variation is generated from genetic variation. In Lepidoptera (butterflies and moths), an enigmatic genomic region containing the protein-coding gene cortex is repeatedly used to generate intraspecific wing color pattern polymorphisms in multiple species. While emerging evidence suggest that cortex itself is not the effector gene of this locus, the identity of the effector remains unknown. Here, we investigated two deeply conserved miRNAs embedded in the cortex locus, and discovered that one of them, mir-193, promotes melanic wing color. This miRNA is likely derived from an intron of a gigantic long non-coding RNA spanning the entire locus, and it elicits its function by directly repressing multiple pigmentation genes including a well-known melanin pathway gene, ebony. The function of mir-193 is conserved in a nymphalid and a pierid butterfly, belonging two families that diverged 90 Mya. We propose that mir-193 is the effector gene of this hotspot locus underlying a 100-million-year evolution of intraspecific wing color pattern polymorphisms in Lepidoptera. Our results suggest that small non-coding RNAs can drive adaptive evolution in animals.\nLimitations of the inference of the distribution of fitness effects of new mutations in partially selfing populations with linkage\nAuthors: Daigle, A.; Johri, P.\nScore: 8.0, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579142\nThe accurate estimation of the distribution of fitness effects (DFE) of new mutations is critical for population genetic inference but remains a challenging task. While various methods have been developed for DFE inference using the site frequency spectrum of putatively neutral and selected sites, their applicability in species with diverse life history traits and complex demographic scenarios is not well understood. Selfing is common among eukaryotic species and can lead to decreased effective recombination rates in such populations, increasing the effects of selection at linked sites, including interference between selected alleles. We employ forward simulations to investigate the limitations of current DFE estimation approaches in the presence of selfing and linked effects of selection. We find that distortions of the site frequency spectrum due to Hill-Robertson interference in highly selfing populations lead to mis-inference of the deleterious DFE of new mutations. While accounting for the decrease in the effective population size due to linked effects of selection largely accounts for the observed bias in populations with moderate levels of selfing, this correction is unable to accurately estimate the DFE in highly selfing populations. In addition, the presence of cryptic population structure and uneven sampling across subpopulations leads to the false inference of a deleterious DFE skewed towards effectively neutral/mildly deleterious mutations. Finally, the proportion of adaptive substitutions estimated at high rates of selfing is substantially overestimated. Our results clarify the parameter space where current DFE methods might be problematic and where they remain robust in the presence of selfing and other model violations, such as departures from semidominance, population structure, and uneven sampling. Our observations apply broadly to species and genomic regions with little/no recombination.\nThe genetic landscape, gene flow and adaptation in Ladakh highlanders of trans-Himalayas\nAuthors: Kumar, L.; Rajpal, R.; Ahlawat, B.; Pasupuleti, N.; Konar, S.; Dwivedi, A.; Kumar, S.; Spalzin, S.; Rabyang, S.; Thangaraj, K.; Rai, N.\nScore: 7.5, Published: 2024-02-08 DOI: 10.1101/2024.02.05.579041\nThe Ladakh in the trans-Himalayas has witnessed complex cultural movements, genetic assimilation and demographic changes since the Neolithic period, which continue despite the harsh, inhospitable and cold climate. Although geographically isolated from mainland South Asia, Ladakh has historic trade routes (the Silk Road) and is well connected and accessible to travelers from Tibet and Central Asia. Despite its rich cultural heritage, a detailed description of the genetic landscape of the Ladakh region is completely lacking, particularly with regard to genome-wide markers and larger sample sizes. Therefore, in our current study, we genotyped 80 individuals from Kargil and Leh districts of the Union Territory of Ladakh, India. Here, we performed a comprehensive genetic analysis based on allele frequency and haplotype sharing. Our analysis revealed the presence of two distinct genetic lineages in the region with quite different genetic composition. The population of the Leh region is more similar to East Asian and Southeast Asian populations with population drift. In contrast, the population of the Kargil region is more similar to the Indo-European population. The former group shares a genetic history with Tibetans and Sherpas, while the latter group from Kargil associates with Kashmiri Muslims, Gujjars and Nepalese Brahmins. For both groups, there was a founder event at about 18th to 22th generations ago and duration coincided with the Mughal invasion. The genome-wide scan for a signal of positive selection revealed genetic signatures of altitude adaptation in the Leh population, while in the Kargil population the key gene signatures were associated with immunity and female fertility.\nUsing machine learning to integrate genetic and environmental data to model genotype-by-environment interactions\nAuthors: Fernandes, I. K.; Vieira, C. C.; Dias, K. O. d. G.; Fernandes, S. B.\nScore: 4.7, Published: 2024-02-12 DOI: 10.1101/2024.02.08.579534\nComplementing phenotypic traits and molecular markers with high-dimensional data such as climate and soil information is becoming a common practice in breeding programs. This study explored new ways to integrate non-genetic information in genomic prediction models using machine learning (ML). Using the multi-environment trial data from the Genomes To Fields initiative, different models to predict maize grain yield were adjusted using various inputs: genetic, environmental, or a combination of both, either in an additive (genetic-and-environmental; G+E) or a multiplicative (genotype-by-environment interaction; GEI) manner. When including environmental data, the mean predictive ability of machine learning genomic prediction models increased from 7-9% over the well-established Factor Analytic Multiplicative Mixed Model (FA) among the three cross-validation scenarios evaluated. Moreover, using the G+E model was more advantageous than the GEI model given the superior, or at least comparable, predictive ability, the lower usage of computational memory and time, and the flexibility of accounting for interactions by construction. Our results illustrate the flexibility provided by the ML framework, particularly with feature engineering. We show that the featured engineering stage offers a viable option for envirotyping and generates valuable information for machine learning-based genomic prediction models. Furthermore, we verified that the genotype-by-environment interactions may be considered using tree-based approaches without explicitly including interactions in the model. These findings support the growing interest in merging high-dimensional genotypic and environmental data into predictive modeling.\nGenetic Studies Highlight the Role of TET2 and INO80 in DNA Damage Response and Kidney Disease Pathogenesis\nAuthors: Liang, X.; Liu, H.; Hu, H.; Zhou, J.; Abedini, A.; Navarro, A. S.; Klotzer, K. A.; Susztak, K.\nScore: 4.6, Published: 2024-02-07 DOI: 10.1101/2024.02.02.578718\nGenome-wide association studies (GWAS) have identified over 800 loci associated with kidney function, yet the specific genes, variants, and pathways involved remain elusive. By integrating kidney function GWAS, human kidney expression and methylation quantitative trait analyses, we identified Ten-Eleven Translocation (TET) DNA demethylase 2: TET2 as a novel kidney disease risk gene. Utilizing single-cell chromatin accessibility and CRISPR-based genome editing, we highlight GWAS variants that influence TET2 expression in kidney proximal tubule cells. Experiments using kidney-tubule-specific Tet2 knockout mice indicated its protective role in cisplatin-induced acute kidney injury, as well as chronic kidney disease and fibrosis, induced by unilateral ureteral obstruction or adenine diet. Single-cell gene profiling of kidneys from Tet2 knockout mice and TET2-knock-down tubule cells revealed the altered expression of DNA damage repair and chromosome segregation genes, notably including INO80, another kidney function GWAS target gene itself. Remarkably both TET2-null and INO80-null cells exhibited an increased accumulation of micronuclei after injury, leading to the activation of cytosolic nucleotide sensor cGAS-STING. Genetic deletion of cGAS or STING in kidney tubules or pharmacological inhibition of STING protected TET2 null mice from disease development. In conclusion, our findings highlight TET2 and INO80 as key genes in the pathogenesis of kidney diseases, indicating the importance of DNA damage repair mechanisms.\nThe simplicity of protein sequence-function relationships\nAuthors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.\nScore: 35.1, Published: 2024-02-07 DOI: 10.1101/2023.09.02.556057\nHow complicated is the genetic architecture of proteins - the set of causal effects by which sequence determines function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most studies, however, used methods that overestimate epistasis, because they analyze genetic architecture relative to a designated reference sequence - causing measurement noise and small local idiosyncrasies to propagate into pervasive high-order interactions - or have not effectively accounted for global nonlinearity in the sequence-function relationship. Here we present a new reference-free method that jointly estimates global nonlinearity and specific epistatic interactions across a proteins entire genotype-phenotype map. This method yields a maximally efficient explanation of a proteins genetic architecture and is more robust than existing methods to measurement noise, partial sampling, and model misspecification. We reanalyze 20 combinatorial mutagenesis experiments from a diverse set of proteins and find that additive and pairwise effects, along with a simple nonlinearity to account for limited dynamic range, explain a median of 96% of total variance in measured phenotypes (and \u003e92% in every case). Only a tiny fraction of genotypes are strongly affected by third- or higher-order epistasis. Genetic architecture is also sparse: the number of terms required to explain the vast majority of variance is smaller than the number of genotypes by many orders of magnitude. The sequence-function relationship in most proteins is therefore far simpler than previously thought, opening the way for new and tractable approaches to characterize it.\nThe RNA-binding activity of the TRIM-NHL protein NHL-2 is essential for miRNA-mediated gene regulation\nAuthors: Saadat, N.; Colson, R. N.; Grimme, A. L.; Seroussi, U.; Anderson, J. W. T.; Claycomb, J. M.; Wilce, M. C.; McJunkin, K.; Wilce, J. A.; Boag, P. R.\nScore: 3.2, Published: 2024-02-13 DOI: 10.1101/2024.02.13.580109\nThe conserved TRIM-NHL protein, NHL-2, plays a key role in small RNA pathways in Caenorhabditis elegans. NHL-2 has been shown to interact with U-rich RNA through its NHL domain, but the importance to its biological function is unknown. We defined the crystal structure of the NHL domain to 1.4 [A] resolution and identified residues that affect affinity for U-rich RNA. Functional analysis of an NHL-2 RNA-binding loss-of-function mutant demonstrated defects in the heterochronic pathway, suggesting that RNA binding is essential for its role in this miRNA pathway. Processing bodies were enlarged in the NHL-2 RNA-binding mutant, suggesting a defect in mRNA decay. We also identified the eIF4E binding protein IFET-1 as a strong synthetic interactor with NHL-2 and the DEAD box RNA helicase CGH-1 (DDX6), linking NHL-2 function to translation repression. We demonstrated that in the absence of NHL-2, there was an enrichment of miRNA transcripts associated with the miRNA pathway Argonaute proteins ALG-2 and ALG-2. We demonstrate that NHL-2 RNA-binding activity is essential for let-7 family miRNA-mediated translational repression. We conclude that the NHL-2, CGH-1, and IFET-1 regulatory axes work with the core miRISC components to form an effector complex that is required for some, but not all, miRNAs.\n",
  "wordCount" : "2658",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579360">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579360" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579360">
        <p class="paperTitle">Developmental disruption of the mitochondrial fissiongene drp-1 extends the longevity of daf-2 insulin/IGF-1 receptor mutant</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579360" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579360" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Traa, A.; Van Raamsdonk, J.</p>
        <p class="info">Score: 21.4, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579360' target='https://doi.org/10.1101/2024.02.07.579360'> 10.1101/2024.02.07.579360</a></p>
        <p class="abstract">The dynamic nature of the mitochondrial network is regulated by mitochondrial fission and fusion, allowing for re-organization of mitochondria to adapt to the cells ever-changing needs. As organisms age, mitochondrial fission and fusion become dysregulated and mitochondrial networks become increasingly fragmented. Modulation of mitochondrial dynamics has been shown to affect longevity in fungi, yeast, Drosophila and C. elegans. While disruption of the mitochondrial fission gene drp-1 only mildly increases wild-type lifespan, it drastically increases the already long lifespan of daf-2 insulin/IGF-1 signaling (IIS) mutants. In this work, we determined the conditions required for drp-1 disruption to extend daf-2 longevity and explored the molecular mechanisms involved. We found that knockdown of drp-1 during development is sufficient to extend daf-2 lifespan, while tissue-specific knockdown of drp-1 in neurons, intestine or muscle failed to increase daf-2 longevity. Disruption of other genes involved in mitochondrial fission also increased daf-2 lifespan as did treatment with a number of different RNAi clones that decrease mitochondrial fragmentation. In exploring potential mechanisms involved, we found that deletion of drp-1 increases resistance to chronic stresses and slows physiologic rates in daf-2 worms. In addition, we found that disruption of drp-1 increased mitochondrial and peroxisomal connectedness in daf-2 worms, increased oxidative phosphorylation and ATP levels, and increased mitophagy in daf-2 worms, but did not affect their ROS levels or mitochondrial membrane potential. Overall, this work defined the conditions under which drp-1 disruption increases daf-2 lifespan and has identified multiple changes in daf-2;drp-1 mutants that may contribute to their lifespan extension.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579069">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579069" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579069">
        <p class="paperTitle">DNA methylation modulates nucleosome retention in sperm and H3K4 methylation deposition in early mouse embryos.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579069" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579069" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fanourgakis, G.; Gaspa-Toneu, L.; Komarov, P. A.; Ozonov, E. A.; Smallwood, S. A.; Peters, A. H. F. M.</p>
        <p class="info">Score: 20.0, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579069' target='https://doi.org/10.1101/2024.02.06.579069'> 10.1101/2024.02.06.579069</a></p>
        <p class="abstract">DNA methylation (DNAme) serves a stable gene regulatory function in somatic cells (1). In the germ line and during early embryogenesis, however, DNAme undergoes global erasure and re-establishment to support germ cell and embryonic development (2). While de novo DNAme acquisition during male germ cell development is essential for setting genomic DNA methylation imprints, other intergenerational roles for paternal DNAme in defining embryonic chromatin after fertilization are unknown. To approach this question, we reduced levels of DNAme in developing male germ cells through conditional gene deletion of the de novo DNA methyltransferases DNMT3A and DNMT3B in undifferentiated spermatogonia. We observed that DNMT3A serves a DNAme maintenance function in undifferentiated spermatogonia while DNMT3B catalyzes de novo DNAme during spermatogonial differentiation. Mutant male germ cells nevertheless completed their differentiation to sperm. Failing de novo DNAme in Dnmt3a/Dnmt3b double deficient spermatogonia is associated with increased nucleosome occupancy in mature sperm, preferentially at sites with higher CpG content, supporting the model that DNAme modulates nucleosome retention in sperm (3). To assess the impact of altered sperm chromatin in the formation of embryonic chromatin, we measured H3K4me3 occupancy at paternal and maternal alleles in 2-cell embryos using a newly developed transposon-based tagging assay for modified chromatin. Our data show that reduced DNAme in sperm renders paternal alleles permissive for H3K4me3 establishment in early embryos, independently of possible paternal inheritance of sperm born H3K4me3. Together, this study provides first evidence that paternally inherited DNAme directs chromatin formation during early embryonic development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579733">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579733" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579733">
        <p class="paperTitle">The ivory lncRNA regulates seasonal color patterns in buckeye butterflies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579733" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579733" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fandino, R. A.; Brady, N. K.; Chatterjee, M.; McDonald, J. M. C.; Livraghi, L.; van der burg, K.; Mazo-Vargas, A.; Reed, R. D.</p>
        <p class="info">Score: 10.9, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579733' target='https://doi.org/10.1101/2024.02.09.579733'> 10.1101/2024.02.09.579733</a></p>
        <p class="abstract">Long non-coding RNAs (lncRNAs) are transcribed elements increasingly recognized for their roles in regulating gene expression. Thus far, however, we have little understanding of how lncRNAs contribute to evolution and adaptation. Here we show that a conserved lncRNA, ivory, is an important color patterning gene in the buckeye butterfly Junonia coenia. ivory overlaps with cortex, a locus linked to multiple cases of crypsis and mimicry in Lepidoptera. Along with the Livraghi et. al companion paper, we argue that ivory, not cortex, is the color pattern gene of interest at this locus. In J. coenia a cluster of cis-regulatory elements (CREs) in the first intron of ivory are genetically associated with natural variation in seasonal color pattern plasticity, and targeted deletions of these CREs phenocopy seasonal phenotypes. Deletions of different ivory CREs produce other distinct phenotypes as well, including loss of melanic eyespot rings, and positive and negative changes in overall wing pigmentation. We show that the color pattern transcription factors Spineless, Bric-a-brac, and Ftz-f1 bind to the ivory promoter during wing pattern development, suggesting that they directly regulate ivory. This case study demonstrates how cis-regulation of a single non-coding RNA can exert diverse and nuanced effects on the evolution and development of color patterns, including modulating seasonally plastic color patterns.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579741">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579741" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579741">
        <p class="paperTitle">A micro-RNA drives a 100-million-year adaptive evolution of melanic patterns in butterflies and moths</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579741" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579741" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tian, S.; Banerjee, T. D.; Wee, J. L. Q.; Wang, Y.; Murugesan, S. N.; Monteiro, A.</p>
        <p class="info">Score: 8.0, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579741' target='https://doi.org/10.1101/2024.02.09.579741'> 10.1101/2024.02.09.579741</a></p>
        <p class="abstract">A long-standing question in biology is how phenotypic variation is generated from genetic variation. In Lepidoptera (butterflies and moths), an enigmatic genomic region containing the protein-coding gene cortex is repeatedly used to generate intraspecific wing color pattern polymorphisms in multiple species. While emerging evidence suggest that cortex itself is not the effector gene of this locus, the identity of the effector remains unknown. Here, we investigated two deeply conserved miRNAs embedded in the cortex locus, and discovered that one of them, mir-193, promotes melanic wing color. This miRNA is likely derived from an intron of a gigantic long non-coding RNA spanning the entire locus, and it elicits its function by directly repressing multiple pigmentation genes including a well-known melanin pathway gene, ebony. The function of mir-193 is conserved in a nymphalid and a pierid butterfly, belonging two families that diverged 90 Mya. We propose that mir-193 is the effector gene of this hotspot locus underlying a 100-million-year evolution of intraspecific wing color pattern polymorphisms in Lepidoptera. Our results suggest that small non-coding RNAs can drive adaptive evolution in animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579142">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579142" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579142">
        <p class="paperTitle">Limitations of the inference of the distribution of fitness effects of new mutations in partially selfing populations with linkage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579142" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579142" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daigle, A.; Johri, P.</p>
        <p class="info">Score: 8.0, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579142' target='https://doi.org/10.1101/2024.02.06.579142'> 10.1101/2024.02.06.579142</a></p>
        <p class="abstract">The accurate estimation of the distribution of fitness effects (DFE) of new mutations is critical for population genetic inference but remains a challenging task. While various methods have been developed for DFE inference using the site frequency spectrum of putatively neutral and selected sites, their applicability in species with diverse life history traits and complex demographic scenarios is not well understood. Selfing is common among eukaryotic species and can lead to decreased effective recombination rates in such populations, increasing the effects of selection at linked sites, including interference between selected alleles. We employ forward simulations to investigate the limitations of current DFE estimation approaches in the presence of selfing and linked effects of selection. We find that distortions of the site frequency spectrum due to Hill-Robertson interference in highly selfing populations lead to mis-inference of the deleterious DFE of new mutations. While accounting for the decrease in the effective population size due to linked effects of selection largely accounts for the observed bias in populations with moderate levels of selfing, this correction is unable to accurately estimate the DFE in highly selfing populations. In addition, the presence of cryptic population structure and uneven sampling across subpopulations leads to the false inference of a deleterious DFE skewed towards effectively neutral/mildly deleterious mutations. Finally, the proportion of adaptive substitutions estimated at high rates of selfing is substantially overestimated. Our results clarify the parameter space where current DFE methods might be problematic and where they remain robust in the presence of selfing and other model violations, such as departures from semidominance, population structure, and uneven sampling. Our observations apply broadly to species and genomic regions with little/no recombination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.579041">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.579041" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.579041">
        <p class="paperTitle">The genetic landscape, gene flow and adaptation in Ladakh highlanders of trans-Himalayas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.579041" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.579041" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumar, L.; Rajpal, R.; Ahlawat, B.; Pasupuleti, N.; Konar, S.; Dwivedi, A.; Kumar, S.; Spalzin, S.; Rabyang, S.; Thangaraj, K.; Rai, N.</p>
        <p class="info">Score: 7.5, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.579041' target='https://doi.org/10.1101/2024.02.05.579041'> 10.1101/2024.02.05.579041</a></p>
        <p class="abstract">The Ladakh in the trans-Himalayas has witnessed complex cultural movements, genetic assimilation and demographic changes since the Neolithic period, which continue despite the harsh, inhospitable and cold climate. Although geographically isolated from mainland South Asia, Ladakh has historic trade routes (the Silk Road) and is well connected and accessible to travelers from Tibet and Central Asia. Despite its rich cultural heritage, a detailed description of the genetic landscape of the Ladakh region is completely lacking, particularly with regard to genome-wide markers and larger sample sizes. Therefore, in our current study, we genotyped 80 individuals from Kargil and Leh districts of the Union Territory of Ladakh, India. Here, we performed a comprehensive genetic analysis based on allele frequency and haplotype sharing. Our analysis revealed the presence of two distinct genetic lineages in the region with quite different genetic composition. The population of the Leh region is more similar to East Asian and Southeast Asian populations with population drift. In contrast, the population of the Kargil region is more similar to the Indo-European population. The former group shares a genetic history with Tibetans and Sherpas, while the latter group from Kargil associates with Kashmiri Muslims, Gujjars and Nepalese Brahmins. For both groups, there was a founder event at about 18th to 22th generations ago and duration coincided with the Mughal invasion. The genome-wide scan for a signal of positive selection revealed genetic signatures of altitude adaptation in the Leh population, while in the Kargil population the key gene signatures were associated with immunity and female fertility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.08.579534">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.08.579534" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.08.579534">
        <p class="paperTitle">Using machine learning to integrate genetic and environmental data to model genotype-by-environment interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.08.579534" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.08.579534" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fernandes, I. K.; Vieira, C. C.; Dias, K. O. d. G.; Fernandes, S. B.</p>
        <p class="info">Score: 4.7, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.08.579534' target='https://doi.org/10.1101/2024.02.08.579534'> 10.1101/2024.02.08.579534</a></p>
        <p class="abstract">Complementing phenotypic traits and molecular markers with high-dimensional data such as climate and soil information is becoming a common practice in breeding programs. This study explored new ways to integrate non-genetic information in genomic prediction models using machine learning (ML). Using the multi-environment trial data from the Genomes To Fields initiative, different models to predict maize grain yield were adjusted using various inputs: genetic, environmental, or a combination of both, either in an additive (genetic-and-environmental; G&#43;E) or a multiplicative (genotype-by-environment interaction; GEI) manner. When including environmental data, the mean predictive ability of machine learning genomic prediction models increased from 7-9% over the well-established Factor Analytic Multiplicative Mixed Model (FA) among the three cross-validation scenarios evaluated. Moreover, using the G&#43;E model was more advantageous than the GEI model given the superior, or at least comparable, predictive ability, the lower usage of computational memory and time, and the flexibility of accounting for interactions by construction. Our results illustrate the flexibility provided by the ML framework, particularly with feature engineering. We show that the featured engineering stage offers a viable option for envirotyping and generates valuable information for machine learning-based genomic prediction models. Furthermore, we verified that the genotype-by-environment interactions may be considered using tree-based approaches without explicitly including interactions in the model. These findings support the growing interest in merging high-dimensional genotypic and environmental data into predictive modeling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578718">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578718" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578718">
        <p class="paperTitle">Genetic Studies Highlight the Role of TET2 and INO80 in DNA Damage Response and Kidney Disease Pathogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578718" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578718" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liang, X.; Liu, H.; Hu, H.; Zhou, J.; Abedini, A.; Navarro, A. S.; Klotzer, K. A.; Susztak, K.</p>
        <p class="info">Score: 4.6, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578718' target='https://doi.org/10.1101/2024.02.02.578718'> 10.1101/2024.02.02.578718</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have identified over 800 loci associated with kidney function, yet the specific genes, variants, and pathways involved remain elusive. By integrating kidney function GWAS, human kidney expression and methylation quantitative trait analyses, we identified Ten-Eleven Translocation (TET) DNA demethylase 2: TET2 as a novel kidney disease risk gene. Utilizing single-cell chromatin accessibility and CRISPR-based genome editing, we highlight GWAS variants that influence TET2 expression in kidney proximal tubule cells.

Experiments using kidney-tubule-specific Tet2 knockout mice indicated its protective role in cisplatin-induced acute kidney injury, as well as chronic kidney disease and fibrosis, induced by unilateral ureteral obstruction or adenine diet. Single-cell gene profiling of kidneys from Tet2 knockout mice and TET2-knock-down tubule cells revealed the altered expression of DNA damage repair and chromosome segregation genes, notably including INO80, another kidney function GWAS target gene itself.

Remarkably both TET2-null and INO80-null cells exhibited an increased accumulation of micronuclei after injury, leading to the activation of cytosolic nucleotide sensor cGAS-STING. Genetic deletion of cGAS or STING in kidney tubules or pharmacological inhibition of STING protected TET2 null mice from disease development. In conclusion, our findings highlight TET2 and INO80 as key genes in the pathogenesis of kidney diseases, indicating the importance of DNA damage repair mechanisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.02.556057">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.02.556057" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.02.556057">
        <p class="paperTitle">The simplicity of protein sequence-function relationships</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.02.556057" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.02.556057" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Park, Y.; Metzger, B. P. H.; Thornton, J. W.</p>
        <p class="info">Score: 35.1, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.02.556057' target='https://doi.org/10.1101/2023.09.02.556057'> 10.1101/2023.09.02.556057</a></p>
        <p class="abstract">How complicated is the genetic architecture of proteins - the set of causal effects by which sequence determines function? High-order epistatic interactions among residues are thought to be pervasive, making a proteins function difficult to predict or understand from its sequence. Most studies, however, used methods that overestimate epistasis, because they analyze genetic architecture relative to a designated reference sequence - causing measurement noise and small local idiosyncrasies to propagate into pervasive high-order interactions - or have not effectively accounted for global nonlinearity in the sequence-function relationship. Here we present a new reference-free method that jointly estimates global nonlinearity and specific epistatic interactions across a proteins entire genotype-phenotype map. This method yields a maximally efficient explanation of a proteins genetic architecture and is more robust than existing methods to measurement noise, partial sampling, and model misspecification. We reanalyze 20 combinatorial mutagenesis experiments from a diverse set of proteins and find that additive and pairwise effects, along with a simple nonlinearity to account for limited dynamic range, explain a median of 96% of total variance in measured phenotypes (and &gt;92% in every case). Only a tiny fraction of genotypes are strongly affected by third- or higher-order epistasis. Genetic architecture is also sparse: the number of terms required to explain the vast majority of variance is smaller than the number of genotypes by many orders of magnitude. The sequence-function relationship in most proteins is therefore far simpler than previously thought, opening the way for new and tractable approaches to characterize it.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.13.580109">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.13.580109" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.13.580109">
        <p class="paperTitle">The RNA-binding activity of the TRIM-NHL protein NHL-2 is essential for miRNA-mediated gene regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.13.580109" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.13.580109" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saadat, N.; Colson, R. N.; Grimme, A. L.; Seroussi, U.; Anderson, J. W. T.; Claycomb, J. M.; Wilce, M. C.; McJunkin, K.; Wilce, J. A.; Boag, P. R.</p>
        <p class="info">Score: 3.2, Published: 2024-02-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.13.580109' target='https://doi.org/10.1101/2024.02.13.580109'> 10.1101/2024.02.13.580109</a></p>
        <p class="abstract">The conserved TRIM-NHL protein, NHL-2, plays a key role in small RNA pathways in Caenorhabditis elegans. NHL-2 has been shown to interact with U-rich RNA through its NHL domain, but the importance to its biological function is unknown. We defined the crystal structure of the NHL domain to 1.4 [A] resolution and identified residues that affect affinity for U-rich RNA. Functional analysis of an NHL-2 RNA-binding loss-of-function mutant demonstrated defects in the heterochronic pathway, suggesting that RNA binding is essential for its role in this miRNA pathway. Processing bodies were enlarged in the NHL-2 RNA-binding mutant, suggesting a defect in mRNA decay. We also identified the eIF4E binding protein IFET-1 as a strong synthetic interactor with NHL-2 and the DEAD box RNA helicase CGH-1 (DDX6), linking NHL-2 function to translation repression. We demonstrated that in the absence of NHL-2, there was an enrichment of miRNA transcripts associated with the miRNA pathway Argonaute proteins ALG-2 and ALG-2. We demonstrate that NHL-2 RNA-binding activity is essential for let-7 family miRNA-mediated translational repression. We conclude that the NHL-2, CGH-1, and IFET-1 regulatory axes work with the core miRISC components to form an effector complex that is required for some, but not all, miRNAs.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
