<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases
Authors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.
Score: 124.6, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569447
Transposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases
Authors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.
Score: 124.6, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569447
Transposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-10T10:37:48+00:00" />
<meta property="article:modified_time" content="2023-12-10T10:37:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases
Authors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.
Score: 124.6, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569447
Transposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases\nAuthors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.\nScore: 124.6, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569447\nTransposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6.",
  "keywords": [
    
  ],
  "articleBody": " Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases\nAuthors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.\nScore: 124.6, Published: 2023-11-30 DOI: 10.1101/2023.11.30.569447\nTransposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6. We set out to test the hypothesis that TnpB nucleases may have also been repurposed for novel, unexpected functions other than CRIS-PR-Cas. Here, using phylogenetics, structural predictions, comparative genomics, and functional assays, we uncover multiple instances of programmable transcription factors that we name TnpB-like nuclease-dead repressors (TldR). These proteins employ naturally occurring guide RNAs to specifically target conserved promoter regions of the genome, leading to potent gene repression in a mechanism akin to CRISPRi technologies invented by humans7. Focusing on a TldR clade found broadly in Enterobacteriaceae, we discover that bacteriophages exploit the combined action of TldR and an adjacently encoded phage gene to alter the expression and composition of the host flagellar assembly, a transformation with the potential to impact motility8, phage susceptibility9, and host immunity10. Col-lectively, this work showcases the diverse molecular innovations that were enabled through repeated exaptation of genes encoded by transposable elements, and reveals that RNA-guided transcription factors emerged long before the development of dCas9-based editors.\npIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish\nAuthors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.\nScore: 31.6, Published: 2023-12-08 DOI: 10.1101/2023.12.08.570868\nStandard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis. However, while the phiC31 system functions in zebrafish, validated loci carrying attP-based landing or safe harbor sites for transgene integration in zebrafish have not been established. Here, using CRISPR-Cas9, we converted two well-validated single insertion Tol2-based zebrafish transgenes with long-standing genetic stability into two attP landing sites, called phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET). Generating fluorescent reporters, loxP-based Switch lines, CreERT2 effector transgenes, and gene-regulatory variant reporters in the pIGLET14a and pIGLET24b landing site alleles, we document their suitability for transgenesis applications across cell types and developmental stages. For both landing sites, we routinely achieve 25-50% germline transmission of targeted transgene integrations, drastically reducing the number of required animals and necessary resources to generate individual transgenic lines. We document that phiC31 integrase-based transgenesis into pIGLET14a and pIGLET24b reproducibly results in representative reporter expression patterns in injected F0 zebrafish embryos for enhancer discovery and enables qualitative and quantitative comparison of gene-regulatory element variants. Taken together, our new phiC31 integrase-based transgene landing sites establish reproducible, targeted zebrafish transgenesis for numerous applications including quantitative experiments while greatly reducing the workload of generating new transgenic zebrafish lines.\nMechanism of homology search expansion during recombinational DNA break repair\nAuthors: Dumont, A.; Mendiboure, N.; Savocco, J.; Anani, L.; Moreau, P.; Thierry, A.; Modolo, L.; Jost, D.; Piazza, A.\nScore: 25.5, Published: 2023-12-01 DOI: 10.1101/2023.12.01.569403\nHomology search catalyzed by a RecA/Rad51 nucleoprotein filament (NPF) is a central step of DNA double-strand break (DSB) repair by homologous recombination. How it operates in cells remains elusive. Here we developed a Hi-C-based methodology to map single-stranded DNA (ssDNA) contacts genome-wide in S. cerevisiae, which revealed two main homology search phases. Initial search conducted by short NPFs is confined in cis by cohesin-mediated chromatin loop folding. Progressive growth of stiff NPFs enables exploration of distant genomic sites. Long-range resection by Exo1 drives this transition from local to genome-wide search by providing ssDNA substrates for assembly of extensive NPFs. DSB end-tethering promotes coordinated homology search by NPFs formed on the two DSB ends. Finally, an autonomous genetic element on chromosome III engages the NPF and stimulates homology search in its vicinity. This work reveals the mechanism of the progressive and uneven expansion of homology search orchestrated by chromatin organizers, long-range resection, end-tethering, specialized genetic elements, and that exploits the stiff NPF structure conferred by Rad51 oligomerization. Highlights- Cohesin-mediated chromatin loops constrain homology search in cis for NPF regions close to the resection front - Stiffening of ssDNA by Rad51 enables genome-wide homology search by DSB-proximal sites - Exo1-mediated long-range resection promotes genome-wide homology search - DSB end-tethering promotes coordinated homology search by NPFs formed on both DSB ends - The recombination enhancer focuses homology search in its vicinity\nSomatic Mosaicism in Amyotrophic Lateral Sclerosis and Frontotemporal Dementia Reveals Widespread Degeneration from Focal Mutations\nAuthors: Zhou, Z.; Kim, J.; Huang, A. Y.; Nolan, M.; Park, J.; Doan, R.; Shin, T.; Miller, M. B.; Chhouk, B.; Morillo, K.; Yeh, R. C.; Kenny, C.; Neil, J. E.; Lee, C.-Z.; Ohkubo, T.; Ravits, J.; Ansorge, O.; Ostrow, L. W.; Lagier-Tourenne, C.; Lee, E. A.; Walsh, C. A.\nScore: 24.1, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569436\nAlthough mutations in dozens of genes have been implicated in familial forms of amyotrophic lateral sclerosis (fALS) and frontotemporal degeneration (fFTD), most cases of these conditions are sporadic (sALS and sFTD), with no family history, and their etiology remains obscure. We tested the hypothesis that somatic mosaic mutations, present in some but not all cells, might contribute in these cases, by performing ultra-deep, targeted sequencing of 88 genes associated with neurodegenerative diseases in postmortem brain and spinal cord samples from 404 individuals with sALS or sFTD and 144 controls. Known pathogenic germline mutations were found in 20.6% of ALS, and 26.5% of FTD cases. Predicted pathogenic somatic mutations in ALS/FTD genes were observed in 2.7% of sALS and sFTD cases that did not carry known pathogenic or novel germline mutations. Somatic mutations showed low variant allele fraction (typically \u003c2%) and were often restricted to the region of initial discovery, preventing detection through genetic screening in peripheral tissues. Damaging somatic mutations were preferentially enriched in primary motor cortex of sALS and prefrontal cortex of sFTD, mirroring regions most severely affected in each disease. Somatic mutation analysis of bulk RNA-seq data from brain and spinal cord from an additional 143 sALS cases and 23 controls confirmed an overall enrichment of somatic mutations in sALS. Two adult sALS cases were identified bearing pathogenic somatic mutations in DYNC1H1 and LMNA, two genes associated with pediatric motor neuron degeneration. Our study suggests that somatic mutations in fALS/fFTD genes, and in genes associated with more severe diseases in the germline state, contribute to sALS and sFTD, and that mosaic mutations in a small fraction of cells in focal regions of the nervous system can ultimately result in widespread degeneration.\nDigital telomere measurement by long-read sequencing distinguishes healthy aging from disease\nAuthors: Sanchez, S. E.; Gu, J.; Golla, A.; Martin, A.; Shomali, W.; Hockemeyer, D.; Savage, S.; Artandi, S.\nScore: 17.2, Published: 2023-12-01 DOI: 10.1101/2023.11.29.569263\nTelomere length is an important biomarker of organismal aging and cellular replicative potential, but existing measurement methods are limited in resolution and accuracy. Here, we deploy digital telomere measurement by nanopore sequencing to understand how distributions of human telomere length change with age and disease. We measure telomere attrition and de novo elongation with unprecedented resolution in genetically defined populations of human cells, in blood cells from healthy donors and in blood cells from patients with genetic defects in telomere maintenance. We find that human aging is accompanied by a progressive loss of long telomeres and an accumulation of shorter telomeres. In patients with defects in telomere maintenance, the accumulation of short telomeres is more pronounced and correlates with phenotypic severity. We apply machine learning to train a binary classification model that distinguishes healthy individuals from those with telomere biology disorders. This sequencing and bioinformatic pipeline will advance our understanding of telomere maintenance mechanisms and the use of telomere length as a clinical biomarker of aging and disease.\nNon-coding rare variant associations with blood traits on 166,740 UK Biobank genomes\nAuthors: Ribeiro, D.; Delaneau, O.\nScore: 16.4, Published: 2023-12-04 DOI: 10.1101/2023.12.01.569422\nLarge biobanks with whole-genome sequencing now enable the association of non-coding rare variants with complex human traits. Given that \u003e98% of the genome is available for exploration, the selection of non-coding variants remains a critical yet unresolved challenge in these analyses. Here, we leverage knowledge of blood gene regulation and deleteriousness scores to select non-coding variants pertinent for association with blood-related traits. We leverage whole genome sequencing and 59 blood cell count and biomarker measurements for 166 740 UK Biobank samples to perform variant collapsing tests. We identified hundreds of gene-trait associations involving non-coding variants across the 59 traits. However, we demonstrate that the majority of these non-coding rare variant associations (i) reproduce associations known from common variant studies and (ii) are driven by linkage disequilibrium between nearby common and rare variants. This study underscores the prevailing challenges in rare variant analysis and the need for caution when interpreting non-coding rare variant association results.\nWhole genome association testing in 333,100 individuals across three biobanks identifies rare non-coding single variant and genomic aggregate associations with height\nAuthors: Hawkes, G.; Beaumont, R. N.; Li, Z.; Mandla, R.; Li, X.; Albert, C. M.; Arnett, D. K.; Ashley-Koch, A. E.; Ashrani, A. A.; Barnes, K. C.; Boerwinkle, E.; Brody, J. A.; Carson, A. P.; Chami, N.; Chen, Y.-D. I.; Chung, M. K.; Curran, J. E.; Darbar, D.; Ellinor, P. T.; Fornage, M.; Gordeuk, V. R.; Guo, X.; He, J.; Hwu, C.-M.; Kalyani, R. R.; Kaplan, R.; Kardia, S. L. R.; Kooperberg, C.; Loos, R. J. F.; Lubitz, S. A.; Minster, R. L.; Mitchell, B. D.; Murabito, J. M.; Palmer, N. D.; Psaty, B. M.; Redline, S.; Shoemaker, M. B.; Silverman, E. K.; Telen, M. J.; Weiss, S. T.; Yanek, L. R.; Zhou, H.; NH\nScore: 46.9, Published: 2023-11-20 DOI: 10.1101/2023.11.19.566520\nThe role of rare non-coding variation in complex human phenotypes is still largely unknown. To elucidate the impact of rare variants in regulatory elements, we performed a whole-genome sequencing association analysis for height using 333,100 individuals from three datasets: UK Biobank (N=200,003), TOPMed (N=87,652) and All of Us (N=45,445). We performed rare (\u003c0.1% minor-allele-frequency) single-variant and aggregate testing of non-coding variants in regulatory regions based on proximal, intergenic and deep-intronic annotation. We observed 29 independent variants associated with height at P \u003c 6 x 10-10 after conditioning on previously reported variants, with effect sizes ranging from -7cm to +4.7cm. We also identified and replicated non-coding aggregate-based associations proximal to HMGA1 containing variants associated with a 5cm taller height and of highly-conserved variants in MIR497HG on chromosome 17. We have developed a novel approach for identifying non-coding rare variants in regulatory regions with large effects from whole-genome sequencing data associated with complex traits.\nAberrant expression prediction across human tissues\nAuthors: Hölzlwimmer, F. R.; Lindner, J.; Wagner, N.; Yepez, V. A.; Casale, F. P.; Gagneur, J.\nScore: 12.0, Published: 2023-12-04 DOI: 10.1101/2023.12.04.569414\nDespite the frequent implication of aberrant gene expression in diseases, algorithms predicting aberrantly expressed genes of an individual are lacking. To address this need, we compiled an aberrant expression prediction benchmark covering 8.2 million rare variants from 633 individuals across 48 tissues. While not geared toward aberrant expression, the deleteriousness score CADD and the loss-of-function predictor LOFTEE showed mild predictive ability (1-1.5% average precision). Leveraging these and further variant annotations, we next trained AbExp, a model that yielded 10% average precision by combining in a tissue-specific fashion expression variability with variant effects on isoforms and on aberrant splicing. Integrating expression measurements from clinically accessible tissues led to another two-fold improvement. Furthermore, we show on UK Biobank blood traits that performing rare variant association testing using the continuous and tissue-specific AbExp variant scores instead of LOFTEE variant burden increases gene discovery sensitivity and enables improved phenotype predictions.\nThe evolution and convergence of mutation spectra across mammals\nAuthors: Talenti, A.; Wilkinson, T.; Morrison, L. J.; Prendergast, J. G.\nScore: 10.0, Published: 2023-12-04 DOI: 10.1101/2023.12.02.569698\nDespite the key role genetic mutations play in shaping phenotypic differences between species, little is currently known about the evolution of germline mutation spectra across mammals. Domesticated species are likely particularly interesting case studies because of their high mutation rates and complex evolutionary histories, which can span multiple founding events and genetic bottlenecks. Here we have developed a new reusable workflow, nSPECTRa, that can undertake the key steps in characterising mutation spectra, from determining ancestral alleles to characterising multiple forms of variation. We apply nSPECTRa to seven species, including several that have undergone domestication, and highlight how nSPECTRa can provide important novel insights into mutation rate evolution. While mutation spectra most often show marked differences between species and even breeds, certain mutation types have risen to a high frequency in subpopulations of different species, indicative of convergent evolution in mutation rates. This includes the previously characterized TCC-\u003eTTC change enriched among European humans, which is also enriched among East Asian cattle. We show Indicine cattle are particularly interesting examples of how different mutation spectra segregate within a population and subsequently spread across the globe. Together, this work has important implications for understanding the control and evolution of mammalian mutation rates.\nSingle-cell eQTL mapping in yeast reveals a tradeoff between growth and reproduction\nAuthors: Boocock, J.; Alexander, N.; Tapia, L. A.; Walter-McNeill, L.; Munugala, C.; Bloom, J. S.; Kruglyak, L.\nScore: 7.2, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570640\nExpression quantitative trait loci (eQTLs) provide a key bridge between noncoding DNA sequence variants and organismal traits. The effects of eQTLs can differ among tissues, cell types, and cellular states, but these differences are obscured by gene expression measurements in bulk populations. We developed a one-pot approach to map eQTLs in Saccharomyces cerevisiae by single-cell RNA sequencing (scRNA-seq) and applied it to over 100,000 single cells from three crosses. We used scRNA-seq data to genotype each cell, measure gene expression, and classify the cells by cell-cycle stage. We mapped thousands of local and distant eQTLs and identified interactions between eQTL effects and cell-cycle stages. We took advantage of single-cell expression information to identify hundreds of genes with allele-specific effects on expression noise. We used cell-cycle stage classification to map 20 loci that influence cell-cycle progression. One of these loci influenced the expression of genes involved in the mating response. We showed that the effects of this locus arise from a common variant (W82R) in the gene GPA1, which encodes a signaling protein that negatively regulates the mating pathway. The 82R allele increases mating efficiency at the cost of slower cell-cycle progression and is associated with a higher rate of outcrossing in nature. Our results provide a more granular picture of the effects of genetic variants on gene expression and downstream traits.\n",
  "wordCount" : "2447",
  "inLanguage": "en",
  "datePublished": "2023-12-10T10:37:48Z",
  "dateModified": "2023-12-10T10:37:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 10, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569447" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569447">
        <p class="paperTitle">Emergence of RNA-guided transcription factors via domestication of transposon-encoded TnpB nucleases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wiegand, T.; Hoffmann, F. T.; Walker, M. W. G.; Tang, S.; Richard, E.; Le, H. C.; Meers, C.; Sternberg, S. H.</p>
        <p class="info">Score: 124.6, Published: 2023-11-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569447' target='https://doi.org/10.1101/2023.11.30.569447'> 10.1101/2023.11.30.569447</a></p>
        <p class="abstract">Transposon-encoded tnpB genes encode RNA-guided DNA nucleases that promote their own selfish spread through targeted DNA cleavage and homologous recombination1-4. This widespread gene family was repeatedly domesticated over evolutionary timescales, leading to the emergence of diverse CRISPR-associated nucleases including Cas9 and Cas125,6. We set out to test the hypothesis that TnpB nucleases may have also been repurposed for novel, unexpected functions other than CRIS-PR-Cas. Here, using phylogenetics, structural predictions, comparative genomics, and functional assays, we uncover multiple instances of programmable transcription factors that we name TnpB-like nuclease-dead repressors (TldR). These proteins employ naturally occurring guide RNAs to specifically target conserved promoter regions of the genome, leading to potent gene repression in a mechanism akin to CRISPRi technologies invented by humans7. Focusing on a TldR clade found broadly in Enterobacteriaceae, we discover that bacteriophages exploit the combined action of TldR and an adjacently encoded phage gene to alter the expression and composition of the host flagellar assembly, a transformation with the potential to impact motility8, phage susceptibility9, and host immunity10. Col-lectively, this work showcases the diverse molecular innovations that were enabled through repeated exaptation of genes encoded by transposable elements, and reveals that RNA-guided transcription factors emerged long before the development of dCas9-based editors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570868">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570868" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570868">
        <p class="paperTitle">pIGLET: Safe harbor landing sites for reproducible and efficient transgenesis in zebrafish</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570868" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570868" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lalonde, R. L.; Wells, H. H.; Kemmler, C. L.; Nieuwenhuize, S.; Lerma, R.; Burger, A.; Mosimann, C.</p>
        <p class="info">Score: 31.6, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570868' target='https://doi.org/10.1101/2023.12.08.570868'> 10.1101/2023.12.08.570868</a></p>
        <p class="abstract">Standard methods for transgenesis in zebrafish depend on random transgene integration into the genome followed by resource-intensive screening and validation. Targeted vector integration into validated genomic loci using phiC31 integrase-based attP/attB recombination has transformed mouse and Drosophila transgenesis. However, while the phiC31 system functions in zebrafish, validated loci carrying attP-based landing or safe harbor sites for transgene integration in zebrafish have not been established. Here, using CRISPR-Cas9, we converted two well-validated single insertion Tol2-based zebrafish transgenes with long-standing genetic stability into two attP landing sites, called phiC31 Integrase Genomic Loci Engineered for Transgenesis (pIGLET). Generating fluorescent reporters, loxP-based Switch lines, CreERT2 effector transgenes, and gene-regulatory variant reporters in the pIGLET14a and pIGLET24b landing site alleles, we document their suitability for transgenesis applications across cell types and developmental stages. For both landing sites, we routinely achieve 25-50% germline transmission of targeted transgene integrations, drastically reducing the number of required animals and necessary resources to generate individual transgenic lines. We document that phiC31 integrase-based transgenesis into pIGLET14a and pIGLET24b reproducibly results in representative reporter expression patterns in injected F0 zebrafish embryos for enhancer discovery and enables qualitative and quantitative comparison of gene-regulatory element variants. Taken together, our new phiC31 integrase-based transgene landing sites establish reproducible, targeted zebrafish transgenesis for numerous applications including quantitative experiments while greatly reducing the workload of generating new transgenic zebrafish lines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569403" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569403">
        <p class="paperTitle">Mechanism of homology search expansion during recombinational DNA break repair</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dumont, A.; Mendiboure, N.; Savocco, J.; Anani, L.; Moreau, P.; Thierry, A.; Modolo, L.; Jost, D.; Piazza, A.</p>
        <p class="info">Score: 25.5, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569403' target='https://doi.org/10.1101/2023.12.01.569403'> 10.1101/2023.12.01.569403</a></p>
        <p class="abstract">Homology search catalyzed by a RecA/Rad51 nucleoprotein filament (NPF) is a central step of DNA double-strand break (DSB) repair by homologous recombination. How it operates in cells remains elusive. Here we developed a Hi-C-based methodology to map single-stranded DNA (ssDNA) contacts genome-wide in S. cerevisiae, which revealed two main homology search phases. Initial search conducted by short NPFs is confined in cis by cohesin-mediated chromatin loop folding. Progressive growth of stiff NPFs enables exploration of distant genomic sites. Long-range resection by Exo1 drives this transition from local to genome-wide search by providing ssDNA substrates for assembly of extensive NPFs. DSB end-tethering promotes coordinated homology search by NPFs formed on the two DSB ends. Finally, an autonomous genetic element on chromosome III engages the NPF and stimulates homology search in its vicinity. This work reveals the mechanism of the progressive and uneven expansion of homology search orchestrated by chromatin organizers, long-range resection, end-tethering, specialized genetic elements, and that exploits the stiff NPF structure conferred by Rad51 oligomerization.

Highlights- Cohesin-mediated chromatin loops constrain homology search in cis for NPF regions close to the resection front
- Stiffening of ssDNA by Rad51 enables genome-wide homology search by DSB-proximal sites
- Exo1-mediated long-range resection promotes genome-wide homology search
- DSB end-tethering promotes coordinated homology search by NPFs formed on both DSB ends
- The recombination enhancer focuses homology search in its vicinity</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569436">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569436" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569436">
        <p class="paperTitle">Somatic Mosaicism in Amyotrophic Lateral Sclerosis and Frontotemporal Dementia Reveals Widespread Degeneration from Focal Mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569436" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569436" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, Z.; Kim, J.; Huang, A. Y.; Nolan, M.; Park, J.; Doan, R.; Shin, T.; Miller, M. B.; Chhouk, B.; Morillo, K.; Yeh, R. C.; Kenny, C.; Neil, J. E.; Lee, C.-Z.; Ohkubo, T.; Ravits, J.; Ansorge, O.; Ostrow, L. W.; Lagier-Tourenne, C.; Lee, E. A.; Walsh, C. A.</p>
        <p class="info">Score: 24.1, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569436' target='https://doi.org/10.1101/2023.11.30.569436'> 10.1101/2023.11.30.569436</a></p>
        <p class="abstract">Although mutations in dozens of genes have been implicated in familial forms of amyotrophic lateral sclerosis (fALS) and frontotemporal degeneration (fFTD), most cases of these conditions are sporadic (sALS and sFTD), with no family history, and their etiology remains obscure. We tested the hypothesis that somatic mosaic mutations, present in some but not all cells, might contribute in these cases, by performing ultra-deep, targeted sequencing of 88 genes associated with neurodegenerative diseases in postmortem brain and spinal cord samples from 404 individuals with sALS or sFTD and 144 controls. Known pathogenic germline mutations were found in 20.6% of ALS, and 26.5% of FTD cases. Predicted pathogenic somatic mutations in ALS/FTD genes were observed in 2.7% of sALS and sFTD cases that did not carry known pathogenic or novel germline mutations. Somatic mutations showed low variant allele fraction (typically &lt;2%) and were often restricted to the region of initial discovery, preventing detection through genetic screening in peripheral tissues. Damaging somatic mutations were preferentially enriched in primary motor cortex of sALS and prefrontal cortex of sFTD, mirroring regions most severely affected in each disease. Somatic mutation analysis of bulk RNA-seq data from brain and spinal cord from an additional 143 sALS cases and 23 controls confirmed an overall enrichment of somatic mutations in sALS. Two adult sALS cases were identified bearing pathogenic somatic mutations in DYNC1H1 and LMNA, two genes associated with pediatric motor neuron degeneration. Our study suggests that somatic mutations in fALS/fFTD genes, and in genes associated with more severe diseases in the germline state, contribute to sALS and sFTD, and that mosaic mutations in a small fraction of cells in focal regions of the nervous system can ultimately result in widespread degeneration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569263">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569263" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569263">
        <p class="paperTitle">Digital telomere measurement by long-read sequencing distinguishes healthy aging from disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569263" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569263" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez, S. E.; Gu, J.; Golla, A.; Martin, A.; Shomali, W.; Hockemeyer, D.; Savage, S.; Artandi, S.</p>
        <p class="info">Score: 17.2, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569263' target='https://doi.org/10.1101/2023.11.29.569263'> 10.1101/2023.11.29.569263</a></p>
        <p class="abstract">Telomere length is an important biomarker of organismal aging and cellular replicative potential, but existing measurement methods are limited in resolution and accuracy. Here, we deploy digital telomere measurement by nanopore sequencing to understand how distributions of human telomere length change with age and disease. We measure telomere attrition and de novo elongation with unprecedented resolution in genetically defined populations of human cells, in blood cells from healthy donors and in blood cells from patients with genetic defects in telomere maintenance. We find that human aging is accompanied by a progressive loss of long telomeres and an accumulation of shorter telomeres. In patients with defects in telomere maintenance, the accumulation of short telomeres is more pronounced and correlates with phenotypic severity. We apply machine learning to train a binary classification model that distinguishes healthy individuals from those with telomere biology disorders. This sequencing and bioinformatic pipeline will advance our understanding of telomere maintenance mechanisms and the use of telomere length as a clinical biomarker of aging and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569422">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569422" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569422">
        <p class="paperTitle">Non-coding rare variant associations with blood traits on 166,740 UK Biobank genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569422" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569422" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ribeiro, D.; Delaneau, O.</p>
        <p class="info">Score: 16.4, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569422' target='https://doi.org/10.1101/2023.12.01.569422'> 10.1101/2023.12.01.569422</a></p>
        <p class="abstract">Large biobanks with whole-genome sequencing now enable the association of non-coding rare variants with complex human traits. Given that &gt;98% of the genome is available for exploration, the selection of non-coding variants remains a critical yet unresolved challenge in these analyses. Here, we leverage knowledge of blood gene regulation and deleteriousness scores to select non-coding variants pertinent for association with blood-related traits. We leverage whole genome sequencing and 59 blood cell count and biomarker measurements for 166 740 UK Biobank samples to perform variant collapsing tests. We identified hundreds of gene-trait associations involving non-coding variants across the 59 traits. However, we demonstrate that the majority of these non-coding rare variant associations (i) reproduce associations known from common variant studies and (ii) are driven by linkage disequilibrium between nearby common and rare variants. This study underscores the prevailing challenges in rare variant analysis and the need for caution when interpreting non-coding rare variant association results.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.566520">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.566520" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.566520">
        <p class="paperTitle">Whole genome association testing in 333,100 individuals across three biobanks identifies rare non-coding single variant and genomic aggregate associations with height</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.566520" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.566520" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hawkes, G.; Beaumont, R. N.; Li, Z.; Mandla, R.; Li, X.; Albert, C. M.; Arnett, D. K.; Ashley-Koch, A. E.; Ashrani, A. A.; Barnes, K. C.; Boerwinkle, E.; Brody, J. A.; Carson, A. P.; Chami, N.; Chen, Y.-D. I.; Chung, M. K.; Curran, J. E.; Darbar, D.; Ellinor, P. T.; Fornage, M.; Gordeuk, V. R.; Guo, X.; He, J.; Hwu, C.-M.; Kalyani, R. R.; Kaplan, R.; Kardia, S. L. R.; Kooperberg, C.; Loos, R. J. F.; Lubitz, S. A.; Minster, R. L.; Mitchell, B. D.; Murabito, J. M.; Palmer, N. D.; Psaty, B. M.; Redline, S.; Shoemaker, M. B.; Silverman, E. K.; Telen, M. J.; Weiss, S. T.; Yanek, L. R.; Zhou, H.; NH</p>
        <p class="info">Score: 46.9, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.566520' target='https://doi.org/10.1101/2023.11.19.566520'> 10.1101/2023.11.19.566520</a></p>
        <p class="abstract">The role of rare non-coding variation in complex human phenotypes is still largely unknown. To elucidate the impact of rare variants in regulatory elements, we performed a whole-genome sequencing association analysis for height using 333,100 individuals from three datasets: UK Biobank (N=200,003), TOPMed (N=87,652) and All of Us (N=45,445). We performed rare (&lt;0.1% minor-allele-frequency) single-variant and aggregate testing of non-coding variants in regulatory regions based on proximal, intergenic and deep-intronic annotation. We observed 29 independent variants associated with height at P &lt; 6 x 10-10 after conditioning on previously reported variants, with effect sizes ranging from -7cm to &#43;4.7cm. We also identified and replicated non-coding aggregate-based associations proximal to HMGA1 containing variants associated with a 5cm taller height and of highly-conserved variants in MIR497HG on chromosome 17. We have developed a novel approach for identifying non-coding rare variants in regulatory regions with large effects from whole-genome sequencing data associated with complex traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569414">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569414" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569414">
        <p class="paperTitle">Aberrant expression prediction across human tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569414" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569414" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hölzlwimmer, F. R.; Lindner, J.; Wagner, N.; Yepez, V. A.; Casale, F. P.; Gagneur, J.</p>
        <p class="info">Score: 12.0, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569414' target='https://doi.org/10.1101/2023.12.04.569414'> 10.1101/2023.12.04.569414</a></p>
        <p class="abstract">Despite the frequent implication of aberrant gene expression in diseases, algorithms predicting aberrantly expressed genes of an individual are lacking. To address this need, we compiled an aberrant expression prediction benchmark covering 8.2 million rare variants from 633 individuals across 48 tissues. While not geared toward aberrant expression, the deleteriousness score CADD and the loss-of-function predictor LOFTEE showed mild predictive ability (1-1.5% average precision). Leveraging these and further variant annotations, we next trained AbExp, a model that yielded 10% average precision by combining in a tissue-specific fashion expression variability with variant effects on isoforms and on aberrant splicing. Integrating expression measurements from clinically accessible tissues led to another two-fold improvement. Furthermore, we show on UK Biobank blood traits that performing rare variant association testing using the continuous and tissue-specific AbExp variant scores instead of LOFTEE variant burden increases gene discovery sensitivity and enables improved phenotype predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.02.569698">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.02.569698" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.02.569698">
        <p class="paperTitle">The evolution and convergence of mutation spectra across mammals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.02.569698" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.02.569698" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Talenti, A.; Wilkinson, T.; Morrison, L. J.; Prendergast, J. G.</p>
        <p class="info">Score: 10.0, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.02.569698' target='https://doi.org/10.1101/2023.12.02.569698'> 10.1101/2023.12.02.569698</a></p>
        <p class="abstract">Despite the key role genetic mutations play in shaping phenotypic differences between species, little is currently known about the evolution of germline mutation spectra across mammals. Domesticated species are likely particularly interesting case studies because of their high mutation rates and complex evolutionary histories, which can span multiple founding events and genetic bottlenecks. Here we have developed a new reusable workflow, nSPECTRa, that can undertake the key steps in characterising mutation spectra, from determining ancestral alleles to characterising multiple forms of variation. We apply nSPECTRa to seven species, including several that have undergone domestication, and highlight how nSPECTRa can provide important novel insights into mutation rate evolution. While mutation spectra most often show marked differences between species and even breeds, certain mutation types have risen to a high frequency in subpopulations of different species, indicative of convergent evolution in mutation rates. This includes the previously characterized TCC-&gt;TTC change enriched among European humans, which is also enriched among East Asian cattle. We show Indicine cattle are particularly interesting examples of how different mutation spectra segregate within a population and subsequently spread across the globe. Together, this work has important implications for understanding the control and evolution of mammalian mutation rates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570640">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570640" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570640">
        <p class="paperTitle">Single-cell eQTL mapping in yeast reveals a tradeoff between growth and reproduction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570640" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570640" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boocock, J.; Alexander, N.; Tapia, L. A.; Walter-McNeill, L.; Munugala, C.; Bloom, J. S.; Kruglyak, L.</p>
        <p class="info">Score: 7.2, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570640' target='https://doi.org/10.1101/2023.12.07.570640'> 10.1101/2023.12.07.570640</a></p>
        <p class="abstract">Expression quantitative trait loci (eQTLs) provide a key bridge between noncoding DNA sequence variants and organismal traits. The effects of eQTLs can differ among tissues, cell types, and cellular states, but these differences are obscured by gene expression measurements in bulk populations. We developed a one-pot approach to map eQTLs in Saccharomyces cerevisiae by single-cell RNA sequencing (scRNA-seq) and applied it to over 100,000 single cells from three crosses. We used scRNA-seq data to genotype each cell, measure gene expression, and classify the cells by cell-cycle stage. We mapped thousands of local and distant eQTLs and identified interactions between eQTL effects and cell-cycle stages. We took advantage of single-cell expression information to identify hundreds of genes with allele-specific effects on expression noise. We used cell-cycle stage classification to map 20 loci that influence cell-cycle progression. One of these loci influenced the expression of genes involved in the mating response. We showed that the effects of this locus arise from a common variant (W82R) in the gene GPA1, which encodes a signaling protein that negatively regulates the mating pathway. The 82R allele increases mating efficiency at the cost of slower cell-cycle progression and is associated with a higher rate of outcrossing in nature. Our results provide a more granular picture of the effects of genetic variants on gene expression and downstream traits.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
