<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization
Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.
Score: 198.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478
Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization
Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.
Score: 198.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478
Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-23T10:36:34+00:00" />
<meta property="article:modified_time" content="2023-07-23T10:36:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization
Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.
Score: 198.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478
Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization\nAuthors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.\nScore: 198.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478\nAging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging.",
  "keywords": [
    
  ],
  "articleBody": " Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization\nAuthors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.\nScore: 198.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549478\nAging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging. Epigenetic clocks have been shown to be valuable tools for measuring biological age in a variety of tissues and samples. As such, these epigenetic clocks can determine accelerated biological aging and time-to-mortality across various tissues. Previous reports have shown accelerated biological aging and telomere attrition acceleration following SARS-CoV-2 infection. However, the effect of accelerated epigenetic aging on outcome (death/recovery) in COVID-19 patients with Acute Respiratory Distress Syndrome (ARDS) has not been well investigated. In this study, we measured DNA methylation age and telomere attrition in 87 severe COVID-19 cases with ARDS under mechanical ventilation. Furthermore, we compared dynamic changes in epigenetic aging across multiples time-points until recovery or death. Epigenetic age was measured using the Horvath, Hannum, DNAm skin and blood, GrimAge, and PhenoAge clocks, whereas telomere length was calculated using the surrogate marker DNAmTL. Our analysis revealed significant accelerated epigenetic aging but no telomere attrition acceleration in severe COVID-19 cases. In addition, we observed epigenetic age deceleration at inclusion vs end of follow-up in recovered but not in deceased COVID-19 cases using certain clocks. When comparing dynamic changes in epigenetic age acceleration (EAA), we detected higher EAA using both the Horvath and PhenoAge clocks in deceased vs recovered patients. The DNAmTL measurements revealed telomere attrition acceleration in deceased COVID19 patients between inclusion and end of follow-up as well as a significant change in dynamic telomere attrition acceleration when comparing patients who recovered vs those who died. In conclusion, EAA and telomere attrition acceleration was associated with treatment outcome in hospitalized COVID-19 Patients with ARDS. A better understanding of the long-term effects of EAA in COVID19 patients and how they might contribute to Long COVID symptoms in recovered individuals is urgently needed.\nCRIMP: A CRISPR/Cas9 Insertional Mutagenesis Protocol and the CRIMP Toolkit\nAuthors: Miles, L. B.; Calcinotto, V.; Sonntag, C.; Lee, C.; Bryson-Richardson, R. J.\nScore: 24.7, Published: 2023-07-14 DOI: 10.1101/2023.07.13.548929\nWe developed a highly efficient targeted insertional mutagenesis system, CRIMP, and an associated plasmid toolkit, CRIMPkit, that disrupts native gene expression by inducing complete transcriptional termination to produce null mutant alleles without inducing genetic compensation. The CRIMPkit contains over 30 ready-to-use plasmid vectors allowing easy and complete mutagenesis of any gene in any reading frame without requiring custom sequences, modification, or subcloning, and also provides a fluorescent readout of successfully mutagenised fish.\nVirus replication in the honey bee parasite, Varroa destructor\nAuthors: Damayo, J. E.; McKee, R. C.; Buchmann, G.; Norton, A. M.; Ashe, A.; Remnant, E. J.\nScore: 13.8, Published: 2023-07-16 DOI: 10.1101/2023.07.16.549232\nArthropod vectors such as mites and ticks introduce an alternative viral transmission route between their hosts. The ectoparasitic mite Varroa destructor is the leading threat to the health of Western honey bees (Apis mellifera) primarily through its action as a vector of viruses. However, it is unclear whether viruses transmitted by V. destructor actively infect and replicate in mites, which could facilitate increased transmission and select for more virulent strains. To better understand the role of V. destructor as a vector, we took advantage of differences between bee and mite antiviral RNA interference pathways to identify the host specificity of replicating viruses. We used small RNA sequencing of individual V. destructor mites to examine viral small interfering RNA (vsiRNA) profiles of Deformed wing virus genotypes (DWV-A and DWV-B), associated with colony declines, as well as nine other viruses present in our samples. We found active replication of six V. destructor-associated viruses, including a novel virus, Varroa destructor virus 9 (VDV-9), and replication of two honey bee associated viruses, including both DWV-A and -B genotypes, suggesting that mites are biological vectors for important bee pathogens. We show that the antiviral RNAi response can be used define the host range of viruses in host-parasite interactions, such as honey bees and their parasites, enabling a better understanding of the role of a vector in the evolution and spread of honey bee pathogens.\nRecombination map tailored to Native Hawaiians reduces false positive findings in genomic scans of positive selection\nAuthors: Dinh, B. L.; Tang, E.; Taparra, K.; Nakatsuka, N.; Chen, F.; Chiang, C. W. K.\nScore: 10.4, Published: 2023-07-14 DOI: 10.1101/2023.07.12.548735\nRecombination events establish the patterns of haplotypic structure in a population and estimates of recombination rates are used in several downstream population and statistical genetic analyses. However, the recombination landscape of populations not represented in the 1000 Genomes Project (1KGP) is poorly understood. Using suboptimal maps from distantly related populations reduces the efficacy of extending genomic analyses to underrepresented populations, including Native Hawaiians. To overcome this challenge, we constructed linkage disequilibrium (LD)- and Identity-by-Descent (IBD)-based recombination maps specifically for Native Hawaiians. We created and characterized two recombination maps from our study sample: one based on a random subset of our sample reflecting the current admixed state of Native Hawaiians (NH map), and one based on a subset of individuals with enriched Polynesian ancestries (PNS map) with potential to be used for less admixed Polynesian populations such as the Samoans. We found the recombination landscape to be less correlated with those from other 1KGP populations (e.g. Spearmans rho = 0.79 between PNS and CEU (Utah residents with Northern and Western European ancestry) compared to 0.92 between YRI (Yoruba in Ibadan, Nigeria) and CEU at 50 kb resolution. PNS shared the fewest recombination hotspots with other populations from 1KGP (e.g. 8% of hotspots shared between PNS and CEU compared to 27% of hotspots shared between YRI and CEU). We found that downstream analysis in the Native Hawaiian population, such as local ancestry inference, imputation, and IBD segment and relatedness detections, would achieve similar efficacy when using the NH map compared to an omnibus map. However, for genome scans of adaptive loci using integrated haplotype scores, we found several loci with apparent genome-wide significant signals (|Z-score| \u003e 4) in Native Hawaiians that would not have been significant when analyzed using NH-specific maps. Therefore, population-specific recombination maps improve the robustness of haplotype-based statistics and help us better characterize the evolutionary history that may underlie Native Hawaiian-specific health conditions that persist today.\nPopulation suppression by release of insects carrying a dominant sterile homing gene drive targeting doublesex in Drosophila\nAuthors: Chen, W.; Guo, J.; Liu, Y.; Champer, J.\nScore: 5.6, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549342\nGene drive alleles, which bias their own inheritance and increase in frequency, show great promise for blocking disease transmission or directly suppressing pest populations. The most common engineered drive system is the CRISPR homing drive, which converts wild-type alleles to drive alleles in the germline of drive heterozygotes by homology-directed repair after CRISPR cleavage. One successful homing drive example targets a female-specific exon in doublesex in Anopheles mosquitos, suppressing the population by inducing recessive sterility in female drive homozygotes. We found that in Drosophila melanogaster, a 3-gRNA drive disrupting the doublesex female exon resulted in a masculine phenotype and dominant female sterility. Resistance alleles formed by end-joining repair were also dominant sterile. This was likely caused by expression of male-specific transcripts in females with drive and resistance alleles, disrupting sex development. Based on this construct, we proposed a new pest suppression system called \"Release of Insects carrying a Dominant-sterile Drive\" (RIDD). This entails continuously releasing drive heterozygous males, with drive and resistance alleles causing sterility in females. The drive remains at high frequency longer than currently used dominant female-lethal alleles (RIDL) due to drive conversion in males, and drive alleles also cause sterility based on resistance, both substantial advantages. With weekly releases of drive males into a cage population with overlapping generations, our RIDD system targeting dsx reached 100% prevalence within 27 weeks, progressively reducing egg production and eventually causing total population collapse. RIDD combines the merits of homing gene drive and RIDL. It is powerful but self-limiting, unlike unconfined standard homing drives, allowing for targeted population suppression.\nValidation of human telomere length trans-ancestry meta-analysis association signals identifies POP5 and KBTBD6 as novel human telomere length regulation genes\nAuthors: Keener, R.; Chhetri, S. B.; Connelly, C. J.; Taub, M. A.; Conomos, M. P.; Weinstock, J. S.; Ni, B.; Strober, B. J.; Aslibekyan, S.; Auer, P. L.; Barwick, L.; Becker, L. C.; Blangero, J.; Bleecker, E. R.; Brody, J. A.; Cade, B. E.; Celedon, J. C.; Chang, Y.-C.; Cupples, L. A.; Custer, B.; Freedman, B. I.; Gladwin, M. T.; Heckbert, S. R.; Hou, L.; Irvin, M. R.; Isasi, C. R.; Johnsen, J. M.; Kenny, E. E.; Kooperberg, C.; Minster, R. L.; Nekhai, S.; Pankratz, N.; Peyser, P. A.; Rotter, J. I.; Taylor, K. D.; Telen, M. J.; Wu, B.; Yanek, L. R.; Yang, I. V.; Albert, C.; Arnett, D. K.; Ashley-Koch, A.\nScore: 9.2, Published: 2023-07-14 DOI: 10.1101/2023.07.12.548702\nTelomere length genome-wide association studies (GWAS) have become well-powered to detect novel genes in telomere length regulation. However, no prior work has validated these putative novel genes to confirm the contribution of GWAS loci to telomere length regulation. We conducted a trans-ancestry meta-analysis of 211,369 individuals. Through enrichment analyses of chromatin state and cell-type heritability we identified blood and immune cells as the most relevant cell type to examine telomere length association signals. We validated specific GWAS associations by overexpressing KBTBD6, a component of an E3 ubiquitin ligase complex, and POP5, a component of the Ribonuclease P/MRP complex, and demonstrating that both lengthened telomeres as predicted by our statistical analyses. CRISPR/Cas9 deletion of the predicted causal regions of these association peaks in K562 immortalized blood cells reduced expression of these genes, demonstrating that these loci are related to transcriptional regulation of KBTBD6 and POP5, respectively. Together our results demonstrate the utility of telomere length GWAS in the identification of novel telomere length regulation mechanisms and highlight the importance of the proteasome-ubiquitin pathway in telomere length regulation.\nSocial and genetic diversity in first farmers of central Europe\nAuthors: Gelabert, P.; Bickle, P.; Hofmann, D.; Teschler-Nicola, M.; Anders, A.; Huang, X.; Olalde, I.; Fournier, R.; Ringbauer, H.; Akbari, A.; Cheronet, O.; Lazaridis, I.; Broomandkhoshbacht, N.; Fernandes, D. M.; Buttinger, K.; Callan, K.; Candilio, F.; Bravo, G.; Curtis, E.; Ferry, M.; Keating, D.; Freilich, S.; Kearns, A.; Harney, E.; Lawson, A. M.; Mandl, K.; Michel, M.; Oberreiter, V.; Oppenheimer, J.; Sawyer, S.; Schattke, C.; Ozdogan, K. T. T.; Qiu, L.; Workman, N.; Zalzala, F.; Mallick, S.; Mah, M.; Micco, A.; Pieler, F.; Pavuk, J.; Lazar, C.; Skrivanko, M. K.; Slaus, M.; Bedic, Z.; Novotny, Score: 32.8, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548126\nThe Linearbandkeramik (LBK) Neolithic communities were the first to spread farming across large parts of central Europe, settling fertile regions from Ukraine to France during the second half of the 6th millennium BCE. The LBK had a high degree of material culture uniformity, albeit with regional differences in settlement patterns, subsistence, and mortuary practices. To date, ancient DNA data from LBK individuals have been generated for a limited number of locations and often in small sample sizes, making it challenging to study variation within and across sites. We report genome-wide data for 178 LBK individuals, from the Alfold Linearbankeramik Culture (ALPC) eastern LBK site of Polgar-Ferenci-hat in Hungary, the western LBK site of Nitra in Slovakia, and the enclosed western LBK settlement and massacre site of Schletz in Austria, as well as 42 LBK individuals from 18 other sites. We also report genome-wide data for 28 Early Neolithic Koros and Star[c]evo individuals from 13 sites, viewed as the predecessors of the LBK. We observe a higher percentage of western hunter-gatherer (WHG) admixture among individuals in the eastern LBK than in the far more widely distributed western LBK, showing that these two archaeologically distinct cultures also had different genetic trajectories. Most WHG-farmer mixture occurred just before the dawn of the LBK culture and there is no evidence that the WHG ancestry came systematically more from males or females. However, we do find strong genetic evidence for patrilocality among the LBK, extending previous findings based on isotopic analysis, with more genetic structure across sites on the male than on the female line, and a higher rate of within-site relatives for males. At Schletz we detect almost no first-degree relatives despite reporting data from almost every skeleton present at the site, showing that this massacre involved people from a large population, not a small community.\nGenetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions\nAuthors: Suhre, K.\nScore: 4.8, Published: 2023-07-20 DOI: 10.1101/2023.07.19.549734\nProtein quantitative trait loci (pQTLs) are an invaluable source of information for drug target development as they provide genetic evidence to support protein function, suggest relationships between cis- and trans-associated proteins, and link proteins to disease where they collocate with genetic risk loci for clinical endpoints. Using the recently released Olink proteomics data for 1,463 proteins measured in over 54,000 samples of the UK Biobank we identified and replicated 4,248 associations with 2,821 ratios between protein levels (rQTLs) where the strengths of association at known pQTL loci increased by up to several hundred orders of magnitude. We attribute this increase in statistical power (p-gain) to accounting for genetic and non-genetic variance shared by the two proteins in the ratio pair. Protein pairs with a significant p-gain were 7.6-fold enriched in known protein-protein interactions, suggesting that their ratios reflect biological links between the implicated proteins. We then conducted a GWAS on the 2,821 ratios and identified 2,527 novel rQTLs, increasing the number of discovered genetic signals compared to the original protein-only GWAS by 24.7%. At examples we demonstrate that this approach can identify novel loci of clinical relevance, support causal gene identification, and reveal complex networks of interacting proteins. Taken together, our study adds significant value to the genetic insights that can be derived from the UKB proteomics data and motivates the wider use of ratios in large scale GWAS.\nT2T-YAO: a Telomere-to-telomere Assembled Diploid Reference Genome for Han Chinese\nAuthors: He, Y.; Chu, Y.; Guo, S.; Hu, J.; Li, R.; Zheng, Y.; Ma, X.; Du, Z.; Zhao, L.; Yu, W.; Xue, J.; Bian, W.; Yang, F.; Chen, X.; Zhang, P.; Wu, R.; Ma, Y.; Shao, C.; Chen, J.; Wang, J.; Li, J.; Wu, J.; Hu, X.; Long, Q.; Jiang, M.; Ye, H.; Song, S.; Li, G.; Wei, Y.; Xu, Y.; Ma, Y.; Chen, Y.; Wang, K.; Bao, J.; Xi, W.; Wang, F.; Ni, W.; Zhang, M.; Yu, Y.; Li, S.; Kang, Y.; Gao, Z.\nScore: 4.8, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549286\nSince its initial release in 2001, the human reference genome has been continuously improved in both continuity and accuracy, and the recently-released telomere-to-telomere version--T2T-CHM13--reaches its top quality after 20 years of effort. However, T2T-CHM13 does not represent an authentic diploid human genome, but rather one derived from a simplified, nearly homozygous genome of a hydatidiform mole cell line. To address this limitation and provide an alternative pertinent to the Chinese population, the largest ethnic group in the world, we have assembled a complete diploid human genome of a male Han Chinese, T2T-YAO, which includes telomere-to-telomere assemblies for all the 22+X+M and 22+Y chromosomes in his two haploids inherited separately from his parents. Both haplotypes contain no artificial sequences or model nucleotides and possess a high quality comparable to CHM13, with fewer than one error per [~]14 Mb. Derived from the individual who lives in the aboriginal region of Han Chinese, T2T-YAO shows clear ancestry and potential genetic continuity from the ancient ancestors of the Han population. Each haplotype of T2T-YAO possesses [~]340 Mb exclusive sequences and [~]3100 unique genes as compared to CHM13, and their genome sequences show greater genetic distance to CHM13 than to each other in terms of nucleotide polymorphism and structural variations. The construction of T2T-YAO would serve as a high-quality diploid reference that enables precise delineation of genomic variations in a haplotype-sensitive manner, which could advance our understandings in human evolution, hereditability of diseases and phenotypes, especially within the context of the unique variations of the Chinese population.\nIdentification of the yeast mannoprotein gene HZY1\tas a key genetic determinant for yeast-derived haze in beer\nAuthors: Lacy, K. M.; Mormando, R.; Smith, J. R.; Gibney, P. A.; Shaner, L. M.; Burns, L. T.\nScore: 6.6, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548400\nWith the sustained popularity of hazy IPAs, brewers have explored multiple approaches to maximizing stable haze that will remain in suspension throughout the shelf life of the beer. Our recent investigations into yeast-dependent haze have uncovered specific brewing yeast strains that promote the formation of haze in heavily dry-hopped beer styles. These brewing strains have been termed \"haze-positive\" and furthermore, the timing of dry hop additions has been found to be another key factor in producing this stable haze. Classical genetics have identified YIL169C (herein referred to as HZY1) as both necessary and sufficient for the haze-positive phenotype in the yeast strain most widely used for Hazy IPAs. HZY1 encodes a candidate glycoprotein and our recent findings suggest it is localized to the cell wall through a GPI anchor. Surprisingly, using long-read sequencing data we uncovered extensive genetic variation in HZY1 across brewing strains. The haze-positive phenotype correlates with an expansion in the N-terminal serine-rich region. We propose that the Hzy1 glycoprotein is a critical component to yeast-dependent colloidal haze and the genetic variation in this locus contributes the range of haze phenotypes observed across industrial brewing strains.\n",
  "wordCount" : "2930",
  "inLanguage": "en",
  "datePublished": "2023-07-23T10:36:34Z",
  "dateModified": "2023-07-23T10:36:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on July 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549478">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549478" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549478">
        <p class="paperTitle">Epigenetic Age Acceleration in Surviving versus Deceased COVID-19 Patients with Acute Respiratory Distress Syndrome following Hospitalization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549478" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549478" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bejaoui, Y.; Humaira Amanullah, F.; Saad, M.; Taleb, S.; Bradic, M.; Megarbane, A.; Ait Hssain, A.; Abi Khalil, C.; El Hajj, N.</p>
        <p class="info">Score: 198.9, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549478' target='https://doi.org/10.1101/2023.07.18.549478'> 10.1101/2023.07.18.549478</a></p>
        <p class="abstract">Aging has been reported as a major risk factor for severe symptoms and higher mortality rates in COVID-19 patients. Molecular hallmarks such as epigenetic alterations and telomere attenuation reflect the biological process of aging. Epigenetic clocks have been shown to be valuable tools for measuring biological age in a variety of tissues and samples. As such, these epigenetic clocks can determine accelerated biological aging and time-to-mortality across various tissues. Previous reports have shown accelerated biological aging and telomere attrition acceleration following SARS-CoV-2 infection. However, the effect of accelerated epigenetic aging on outcome (death/recovery) in COVID-19 patients with Acute Respiratory Distress Syndrome (ARDS) has not been well investigated. In this study, we measured DNA methylation age and telomere attrition in 87 severe COVID-19 cases with ARDS under mechanical ventilation. Furthermore, we compared dynamic changes in epigenetic aging across multiples time-points until recovery or death. Epigenetic age was measured using the Horvath, Hannum, DNAm skin and blood, GrimAge, and PhenoAge clocks, whereas telomere length was calculated using the surrogate marker DNAmTL. Our analysis revealed significant accelerated epigenetic aging but no telomere attrition acceleration in severe COVID-19 cases. In addition, we observed epigenetic age deceleration at inclusion vs end of follow-up in recovered but not in deceased COVID-19 cases using certain clocks. When comparing dynamic changes in epigenetic age acceleration (EAA), we detected higher EAA using both the Horvath and PhenoAge clocks in deceased vs recovered patients. The DNAmTL measurements revealed telomere attrition acceleration in deceased COVID19 patients between inclusion and end of follow-up as well as a significant change in dynamic telomere attrition acceleration when comparing patients who recovered vs those who died. In conclusion, EAA and telomere attrition acceleration was associated with treatment outcome in hospitalized COVID-19 Patients with ARDS. A better understanding of the long-term effects of EAA in COVID19 patients and how they might contribute to Long COVID symptoms in recovered individuals is urgently needed.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548929">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548929" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548929">
        <p class="paperTitle">CRIMP: A CRISPR/Cas9 Insertional Mutagenesis Protocol and the CRIMP Toolkit</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548929" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548929" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miles, L. B.; Calcinotto, V.; Sonntag, C.; Lee, C.; Bryson-Richardson, R. J.</p>
        <p class="info">Score: 24.7, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548929' target='https://doi.org/10.1101/2023.07.13.548929'> 10.1101/2023.07.13.548929</a></p>
        <p class="abstract">We developed a highly efficient targeted insertional mutagenesis system, CRIMP, and an associated plasmid toolkit, CRIMPkit, that disrupts native gene expression by inducing complete transcriptional termination to produce null mutant alleles without inducing genetic compensation. The CRIMPkit contains over 30 ready-to-use plasmid vectors allowing easy and complete mutagenesis of any gene in any reading frame without requiring custom sequences, modification, or subcloning, and also provides a fluorescent readout of successfully mutagenised fish.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549232">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549232" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549232">
        <p class="paperTitle">Virus replication in the honey bee parasite, Varroa destructor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549232" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549232" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Damayo, J. E.; McKee, R. C.; Buchmann, G.; Norton, A. M.; Ashe, A.; Remnant, E. J.</p>
        <p class="info">Score: 13.8, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549232' target='https://doi.org/10.1101/2023.07.16.549232'> 10.1101/2023.07.16.549232</a></p>
        <p class="abstract">Arthropod vectors such as mites and ticks introduce an alternative viral transmission route between their hosts. The ectoparasitic mite Varroa destructor is the leading threat to the health of Western honey bees (Apis mellifera) primarily through its action as a vector of viruses. However, it is unclear whether viruses transmitted by V. destructor actively infect and replicate in mites, which could facilitate increased transmission and select for more virulent strains. To better understand the role of V. destructor as a vector, we took advantage of differences between bee and mite antiviral RNA interference pathways to identify the host specificity of replicating viruses. We used small RNA sequencing of individual V. destructor mites to examine viral small interfering RNA (vsiRNA) profiles of Deformed wing virus genotypes (DWV-A and DWV-B), associated with colony declines, as well as nine other viruses present in our samples. We found active replication of six V. destructor-associated viruses, including a novel virus, Varroa destructor virus 9 (VDV-9), and replication of two honey bee associated viruses, including both DWV-A and -B genotypes, suggesting that mites are biological vectors for important bee pathogens. We show that the antiviral RNAi response can be used define the host range of viruses in host-parasite interactions, such as honey bees and their parasites, enabling a better understanding of the role of a vector in the evolution and spread of honey bee pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548735" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548735">
        <p class="paperTitle">Recombination map tailored to Native Hawaiians reduces false positive findings in genomic scans of positive selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dinh, B. L.; Tang, E.; Taparra, K.; Nakatsuka, N.; Chen, F.; Chiang, C. W. K.</p>
        <p class="info">Score: 10.4, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548735' target='https://doi.org/10.1101/2023.07.12.548735'> 10.1101/2023.07.12.548735</a></p>
        <p class="abstract">Recombination events establish the patterns of haplotypic structure in a population and estimates of recombination rates are used in several downstream population and statistical genetic analyses. However, the recombination landscape of populations not represented in the 1000 Genomes Project (1KGP) is poorly understood. Using suboptimal maps from distantly related populations reduces the efficacy of extending genomic analyses to underrepresented populations, including Native Hawaiians. To overcome this challenge, we constructed linkage disequilibrium (LD)- and Identity-by-Descent (IBD)-based recombination maps specifically for Native Hawaiians. We created and characterized two recombination maps from our study sample: one based on a random subset of our sample reflecting the current admixed state of Native Hawaiians (NH map), and one based on a subset of individuals with enriched Polynesian ancestries (PNS map) with potential to be used for less admixed Polynesian populations such as the Samoans. We found the recombination landscape to be less correlated with those from other 1KGP populations (e.g. Spearmans rho = 0.79 between PNS and CEU (Utah residents with Northern and Western European ancestry) compared to 0.92 between YRI (Yoruba in Ibadan, Nigeria) and CEU at 50 kb resolution. PNS shared the fewest recombination hotspots with other populations from 1KGP (e.g. 8% of hotspots shared between PNS and CEU compared to 27% of hotspots shared between YRI and CEU). We found that downstream analysis in the Native Hawaiian population, such as local ancestry inference, imputation, and IBD segment and relatedness detections, would achieve similar efficacy when using the NH map compared to an omnibus map. However, for genome scans of adaptive loci using integrated haplotype scores, we found several loci with apparent genome-wide significant signals (|Z-score| &gt; 4) in Native Hawaiians that would not have been significant when analyzed using NH-specific maps. Therefore, population-specific recombination maps improve the robustness of haplotype-based statistics and help us better characterize the evolutionary history that may underlie Native Hawaiian-specific health conditions that persist today.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549342">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549342" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549342">
        <p class="paperTitle">Population suppression by release of insects carrying a dominant sterile homing gene drive targeting doublesex in Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549342" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549342" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, W.; Guo, J.; Liu, Y.; Champer, J.</p>
        <p class="info">Score: 5.6, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549342' target='https://doi.org/10.1101/2023.07.17.549342'> 10.1101/2023.07.17.549342</a></p>
        <p class="abstract">Gene drive alleles, which bias their own inheritance and increase in frequency, show great promise for blocking disease transmission or directly suppressing pest populations. The most common engineered drive system is the CRISPR homing drive, which converts wild-type alleles to drive alleles in the germline of drive heterozygotes by homology-directed repair after CRISPR cleavage. One successful homing drive example targets a female-specific exon in doublesex in Anopheles mosquitos, suppressing the population by inducing recessive sterility in female drive homozygotes. We found that in Drosophila melanogaster, a 3-gRNA drive disrupting the doublesex female exon resulted in a masculine phenotype and dominant female sterility. Resistance alleles formed by end-joining repair were also dominant sterile. This was likely caused by expression of male-specific transcripts in females with drive and resistance alleles, disrupting sex development. Based on this construct, we proposed a new pest suppression system called &#34;Release of Insects carrying a Dominant-sterile Drive&#34; (RIDD). This entails continuously releasing drive heterozygous males, with drive and resistance alleles causing sterility in females. The drive remains at high frequency longer than currently used dominant female-lethal alleles (RIDL) due to drive conversion in males, and drive alleles also cause sterility based on resistance, both substantial advantages. With weekly releases of drive males into a cage population with overlapping generations, our RIDD system targeting dsx reached 100% prevalence within 27 weeks, progressively reducing egg production and eventually causing total population collapse. RIDD combines the merits of homing gene drive and RIDL. It is powerful but self-limiting, unlike unconfined standard homing drives, allowing for targeted population suppression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548702">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548702" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548702">
        <p class="paperTitle">Validation of human telomere length trans-ancestry meta-analysis association signals identifies POP5 and KBTBD6 as novel human telomere length regulation genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548702" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548702" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Keener, R.; Chhetri, S. B.; Connelly, C. J.; Taub, M. A.; Conomos, M. P.; Weinstock, J. S.; Ni, B.; Strober, B. J.; Aslibekyan, S.; Auer, P. L.; Barwick, L.; Becker, L. C.; Blangero, J.; Bleecker, E. R.; Brody, J. A.; Cade, B. E.; Celedon, J. C.; Chang, Y.-C.; Cupples, L. A.; Custer, B.; Freedman, B. I.; Gladwin, M. T.; Heckbert, S. R.; Hou, L.; Irvin, M. R.; Isasi, C. R.; Johnsen, J. M.; Kenny, E. E.; Kooperberg, C.; Minster, R. L.; Nekhai, S.; Pankratz, N.; Peyser, P. A.; Rotter, J. I.; Taylor, K. D.; Telen, M. J.; Wu, B.; Yanek, L. R.; Yang, I. V.; Albert, C.; Arnett, D. K.; Ashley-Koch, A.</p>
        <p class="info">Score: 9.2, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548702' target='https://doi.org/10.1101/2023.07.12.548702'> 10.1101/2023.07.12.548702</a></p>
        <p class="abstract">Telomere length genome-wide association studies (GWAS) have become well-powered to detect novel genes in telomere length regulation. However, no prior work has validated these putative novel genes to confirm the contribution of GWAS loci to telomere length regulation. We conducted a trans-ancestry meta-analysis of 211,369 individuals. Through enrichment analyses of chromatin state and cell-type heritability we identified blood and immune cells as the most relevant cell type to examine telomere length association signals. We validated specific GWAS associations by overexpressing KBTBD6, a component of an E3 ubiquitin ligase complex, and POP5, a component of the Ribonuclease P/MRP complex, and demonstrating that both lengthened telomeres as predicted by our statistical analyses. CRISPR/Cas9 deletion of the predicted causal regions of these association peaks in K562 immortalized blood cells reduced expression of these genes, demonstrating that these loci are related to transcriptional regulation of KBTBD6 and POP5, respectively. Together our results demonstrate the utility of telomere length GWAS in the identification of novel telomere length regulation mechanisms and highlight the importance of the proteasome-ubiquitin pathway in telomere length regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548126">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548126" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548126">
        <p class="paperTitle">Social and genetic diversity in first farmers of central Europe</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548126" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548126" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gelabert, P.; Bickle, P.; Hofmann, D.; Teschler-Nicola, M.; Anders, A.; Huang, X.; Olalde, I.; Fournier, R.; Ringbauer, H.; Akbari, A.; Cheronet, O.; Lazaridis, I.; Broomandkhoshbacht, N.; Fernandes, D. M.; Buttinger, K.; Callan, K.; Candilio, F.; Bravo, G.; Curtis, E.; Ferry, M.; Keating, D.; Freilich, S.; Kearns, A.; Harney, E.; Lawson, A. M.; Mandl, K.; Michel, M.; Oberreiter, V.; Oppenheimer, J.; Sawyer, S.; Schattke, C.; Ozdogan, K. T. T.; Qiu, L.; Workman, N.; Zalzala, F.; Mallick, S.; Mah, M.; Micco, A.; Pieler, F.; Pavuk, J.; Lazar, C.; Skrivanko, M. K.; Slaus, M.; Bedic, Z.; Novotny, </p>
        <p class="info">Score: 32.8, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548126' target='https://doi.org/10.1101/2023.07.07.548126'> 10.1101/2023.07.07.548126</a></p>
        <p class="abstract">The Linearbandkeramik (LBK) Neolithic communities were the first to spread farming across large parts of central Europe, settling fertile regions from Ukraine to France during the second half of the 6th millennium BCE. The LBK had a high degree of material culture uniformity, albeit with regional differences in settlement patterns, subsistence, and mortuary practices. To date, ancient DNA data from LBK individuals have been generated for a limited number of locations and often in small sample sizes, making it challenging to study variation within and across sites. We report genome-wide data for 178 LBK individuals, from the Alfold Linearbankeramik Culture (ALPC) eastern LBK site of Polgar-Ferenci-hat in Hungary, the western LBK site of Nitra in Slovakia, and the enclosed western LBK settlement and massacre site of Schletz in Austria, as well as 42 LBK individuals from 18 other sites. We also report genome-wide data for 28 Early Neolithic Koros and Star[c]evo individuals from 13 sites, viewed as the predecessors of the LBK. We observe a higher percentage of western hunter-gatherer (WHG) admixture among individuals in the eastern LBK than in the far more widely distributed western LBK, showing that these two archaeologically distinct cultures also had different genetic trajectories. Most WHG-farmer mixture occurred just before the dawn of the LBK culture and there is no evidence that the WHG ancestry came systematically more from males or females. However, we do find strong genetic evidence for patrilocality among the LBK, extending previous findings based on isotopic analysis, with more genetic structure across sites on the male than on the female line, and a higher rate of within-site relatives for males. At Schletz we detect almost no first-degree relatives despite reporting data from almost every skeleton present at the site, showing that this massacre involved people from a large population, not a small community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549734">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549734" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549734">
        <p class="paperTitle">Genetic associations with ratios between protein levels detect new pQTLs and reveal protein-protein interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549734" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549734" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suhre, K.</p>
        <p class="info">Score: 4.8, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549734' target='https://doi.org/10.1101/2023.07.19.549734'> 10.1101/2023.07.19.549734</a></p>
        <p class="abstract">Protein quantitative trait loci (pQTLs) are an invaluable source of information for drug target development as they provide genetic evidence to support protein function, suggest relationships between cis- and trans-associated proteins, and link proteins to disease where they collocate with genetic risk loci for clinical endpoints. Using the recently released Olink proteomics data for 1,463 proteins measured in over 54,000 samples of the UK Biobank we identified and replicated 4,248 associations with 2,821 ratios between protein levels (rQTLs) where the strengths of association at known pQTL loci increased by up to several hundred orders of magnitude. We attribute this increase in statistical power (p-gain) to accounting for genetic and non-genetic variance shared by the two proteins in the ratio pair. Protein pairs with a significant p-gain were 7.6-fold enriched in known protein-protein interactions, suggesting that their ratios reflect biological links between the implicated proteins. We then conducted a GWAS on the 2,821 ratios and identified 2,527 novel rQTLs, increasing the number of discovered genetic signals compared to the original protein-only GWAS by 24.7%. At examples we demonstrate that this approach can identify novel loci of clinical relevance, support causal gene identification, and reveal complex networks of interacting proteins. Taken together, our study adds significant value to the genetic insights that can be derived from the UKB proteomics data and motivates the wider use of ratios in large scale GWAS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549286">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549286" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549286">
        <p class="paperTitle">T2T-YAO: a Telomere-to-telomere Assembled Diploid Reference Genome for Han Chinese</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549286" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549286" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, Y.; Chu, Y.; Guo, S.; Hu, J.; Li, R.; Zheng, Y.; Ma, X.; Du, Z.; Zhao, L.; Yu, W.; Xue, J.; Bian, W.; Yang, F.; Chen, X.; Zhang, P.; Wu, R.; Ma, Y.; Shao, C.; Chen, J.; Wang, J.; Li, J.; Wu, J.; Hu, X.; Long, Q.; Jiang, M.; Ye, H.; Song, S.; Li, G.; Wei, Y.; Xu, Y.; Ma, Y.; Chen, Y.; Wang, K.; Bao, J.; Xi, W.; Wang, F.; Ni, W.; Zhang, M.; Yu, Y.; Li, S.; Kang, Y.; Gao, Z.</p>
        <p class="info">Score: 4.8, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549286' target='https://doi.org/10.1101/2023.07.17.549286'> 10.1101/2023.07.17.549286</a></p>
        <p class="abstract">Since its initial release in 2001, the human reference genome has been continuously improved in both continuity and accuracy, and the recently-released telomere-to-telomere version--T2T-CHM13--reaches its top quality after 20 years of effort. However, T2T-CHM13 does not represent an authentic diploid human genome, but rather one derived from a simplified, nearly homozygous genome of a hydatidiform mole cell line. To address this limitation and provide an alternative pertinent to the Chinese population, the largest ethnic group in the world, we have assembled a complete diploid human genome of a male Han Chinese, T2T-YAO, which includes telomere-to-telomere assemblies for all the 22&#43;X&#43;M and 22&#43;Y chromosomes in his two haploids inherited separately from his parents. Both haplotypes contain no artificial sequences or model nucleotides and possess a high quality comparable to CHM13, with fewer than one error per [~]14 Mb. Derived from the individual who lives in the aboriginal region of Han Chinese, T2T-YAO shows clear ancestry and potential genetic continuity from the ancient ancestors of the Han population. Each haplotype of T2T-YAO possesses [~]340 Mb exclusive sequences and [~]3100 unique genes as compared to CHM13, and their genome sequences show greater genetic distance to CHM13 than to each other in terms of nucleotide polymorphism and structural variations. The construction of T2T-YAO would serve as a high-quality diploid reference that enables precise delineation of genomic variations in a haplotype-sensitive manner, which could advance our understandings in human evolution, hereditability of diseases and phenotypes, especially within the context of the unique variations of the Chinese population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548400">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548400" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548400">
        <p class="paperTitle">Identification of the yeast mannoprotein gene HZY1	 as a key genetic determinant for yeast-derived haze in beer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548400" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548400" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lacy, K. M.; Mormando, R.; Smith, J. R.; Gibney, P. A.; Shaner, L. M.; Burns, L. T.</p>
        <p class="info">Score: 6.6, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548400' target='https://doi.org/10.1101/2023.07.10.548400'> 10.1101/2023.07.10.548400</a></p>
        <p class="abstract">With the sustained popularity of hazy IPAs, brewers have explored multiple approaches to maximizing stable haze that will remain in suspension throughout the shelf life of the beer. Our recent investigations into yeast-dependent haze have uncovered specific brewing yeast strains that promote the formation of haze in heavily dry-hopped beer styles. These brewing strains have been termed &#34;haze-positive&#34; and furthermore, the timing of dry hop additions has been found to be another key factor in producing this stable haze. Classical genetics have identified YIL169C (herein referred to as HZY1) as both necessary and sufficient for the haze-positive phenotype in the yeast strain most widely used for Hazy IPAs. HZY1 encodes a candidate glycoprotein and our recent findings suggest it is localized to the cell wall through a GPI anchor. Surprisingly, using long-read sequencing data we uncovered extensive genetic variation in HZY1 across brewing strains. The haze-positive phenotype correlates with an expansion in the N-terminal serine-rich region. We propose that the Hzy1 glycoprotein is a critical component to yeast-dependent colloidal haze and the genetic variation in this locus contributes the range of haze phenotypes observed across industrial brewing strains.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
