<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Broad functional profiling of fission yeast proteins using phenomics and machine learning
Authors: Rodriguez-Lopez, M.; Bordin, N.; Lees, J.; Scholes, H.; Hassan, S.; Saintain, Q.; Kamrad, S.; Orengo, C.; Bähler, J.
Score: 68.9, Published: 2023-07-17 DOI: 10.1101/2023.04.05.535764
Many proteins remain poorly characterized even in well-studied organisms, presenting a bottleneck for research. We applied phenomics and machine-learning approaches with Schizosaccharomyces pombe for broad cues on protein functions. We assayed colony-growth phenotypes to measure the fitness of deletion mutants for 3509 non-essential genes in 131 conditions with different nutrients, drugs, and stresses.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Broad functional profiling of fission yeast proteins using phenomics and machine learning
Authors: Rodriguez-Lopez, M.; Bordin, N.; Lees, J.; Scholes, H.; Hassan, S.; Saintain, Q.; Kamrad, S.; Orengo, C.; Bähler, J.
Score: 68.9, Published: 2023-07-17 DOI: 10.1101/2023.04.05.535764
Many proteins remain poorly characterized even in well-studied organisms, presenting a bottleneck for research. We applied phenomics and machine-learning approaches with Schizosaccharomyces pombe for broad cues on protein functions. We assayed colony-growth phenotypes to measure the fitness of deletion mutants for 3509 non-essential genes in 131 conditions with different nutrients, drugs, and stresses." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Broad functional profiling of fission yeast proteins using phenomics and machine learning
Authors: Rodriguez-Lopez, M.; Bordin, N.; Lees, J.; Scholes, H.; Hassan, S.; Saintain, Q.; Kamrad, S.; Orengo, C.; Bähler, J.
Score: 68.9, Published: 2023-07-17 DOI: 10.1101/2023.04.05.535764
Many proteins remain poorly characterized even in well-studied organisms, presenting a bottleneck for research. We applied phenomics and machine-learning approaches with Schizosaccharomyces pombe for broad cues on protein functions. We assayed colony-growth phenotypes to measure the fitness of deletion mutants for 3509 non-essential genes in 131 conditions with different nutrients, drugs, and stresses."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Broad functional profiling of fission yeast proteins using phenomics and machine learning\nAuthors: Rodriguez-Lopez, M.; Bordin, N.; Lees, J.; Scholes, H.; Hassan, S.; Saintain, Q.; Kamrad, S.; Orengo, C.; Bähler, J.\nScore: 68.9, Published: 2023-07-17 DOI: 10.1101/2023.04.05.535764\nMany proteins remain poorly characterized even in well-studied organisms, presenting a bottleneck for research. We applied phenomics and machine-learning approaches with Schizosaccharomyces pombe for broad cues on protein functions. We assayed colony-growth phenotypes to measure the fitness of deletion mutants for 3509 non-essential genes in 131 conditions with different nutrients, drugs, and stresses.",
  "keywords": [
    
  ],
  "articleBody": " Broad functional profiling of fission yeast proteins using phenomics and machine learning\nAuthors: Rodriguez-Lopez, M.; Bordin, N.; Lees, J.; Scholes, H.; Hassan, S.; Saintain, Q.; Kamrad, S.; Orengo, C.; Bähler, J.\nScore: 68.9, Published: 2023-07-17 DOI: 10.1101/2023.04.05.535764\nMany proteins remain poorly characterized even in well-studied organisms, presenting a bottleneck for research. We applied phenomics and machine-learning approaches with Schizosaccharomyces pombe for broad cues on protein functions. We assayed colony-growth phenotypes to measure the fitness of deletion mutants for 3509 non-essential genes in 131 conditions with different nutrients, drugs, and stresses. These analyses exposed phenotypes for 3492 mutants, including 124 mutants of priority unstudied proteins conserved in humans, providing varied functional clues. For example, over 900 proteins were newly implicated in the resistance to oxidative stress. Phenotype-correlation networks suggested roles for poorly characterized proteins through guilt by association with known proteins. For complementary functional insights, we predicted Gene Ontology (GO) terms using machine learning methods exploiting protein-network and protein-homology data (NET-FF). We obtained 56,594 high-scoring GO predictions, of which 22,060 also featured high information content. Our phenotype-correlation data and NET-FF predictions showed a strong concordance with existing PomBase GO annotations and protein networks, with integrated analyses revealing 1,675 novel GO predictions for 783 genes, including 47 predictions for 23 priority unstudied proteins. Experimental validation identified new proteins involved in cellular ageing, showing that these predictions and phenomics data provide a rich resource to uncover new protein functions.\nLeveraging fine-scale population structure reveals conservation in genetic effect sizes between human populations across a range of human phenotypes\nAuthors: Hu, S.; Ferreira, L. A. F.; Shi, S.; Hellenthal, G.; Marchini, J.; Lawson, D. J.; Myers, S. R.\nScore: 25.9, Published: 2023-08-09 DOI: 10.1101/2023.08.08.552281\nAn understanding of genetic differences between populations is essential for avoiding confounding in genome-wide association studies (GWAS) and understanding the evolution of human traits. Polygenic risk scores constructed in one group perform poorly in highly genetically-differentiated populations, for reasons which remain controversial. We developed a statistical ancestry inference pipeline able to decompose ancestry both within and between countries, and applied it to the UK Biobank data. This identifies fine-scale patterns of genetic relatedness not captured by standard and widely used principal components (PCs), and allows fine-scale population stratification correction that removes both false positive and false negative associations for traits with geographic correlations. We also develop and apply ANCHOR, an approach leveraging segments of distinct ancestries within individuals to estimate similarity in underlying causal effect sizes between groups, using an existing PGS. Applying ANCHOR to \u003e8000 people of mixed African and European ancestry, we demonstrate that estimated causal effect sizes are highly similar across these ancestries for 26 of 29 quantitative molecular and non-molecular phenotypes (mean correlation 0.98 +/-0.08), providing evidence that gene-environment and gene-gene interactions do not play major roles in the poor prediction of European-ancestry PRS scores in African populations for these traits, contradicting previous findings. Instead our results provide optimism that shared causal mutations operate similarly in different groups, focussing the challenge of improving GWAS \"portability\" between groups on joint fine-mapping.\nLocal population structure in Cambridgeshire during the Roman occupation\nAuthors: Scheib, C. L.; Hui, R.; Rose, A. K.; Solnik, A.; D'Atanasio, E.; Inskip, S. A.; Cessford, C.; Griffith, S. J.; Wiseman, R.; Neil, B.; Biers, T.; Harknett, S.-J.; Sasso, S.; Biagini, S. A.; Runfeldt, G.; Duhig, C.; Evans, C.; O'Connell, T. C.; Metspalu, M.; Millett, M. J.; Robb, J. E.; Kivisild, T.\nScore: 27.1, Published: 2023-08-02 DOI: 10.1101/2023.07.31.551265\nThe Roman period saw the empire expand across Europe and the Mediterranean, including much of what is today the United Kingdom. While there is written evidence of high mobility into and out of Britain for administrators, traders and the military, the impact of imperialism on local population structure is invisible in the textual record. The extent of genetic change that occurred in Britain before the Early Medieval Period and how closely linked by genetic kinship the local populations were, remains underexplored. Here, using genome-wide data from 52 ancient individuals from Cambridgeshire, we show low levels of genetic ancestry differentiation between Romano-British sites and lower levels of runs of homozygosity over 4 centimorgans (cM than in the Bronze Age and Neolithic. We find fourteen cases of genetic relatedness within and one between sites without evidence of patrilineal dominance and one case of temporary mobility within a family unit during the Late Romano-British period. We also show that the modern patterns of genetic ancestry composition in Modern Britain emerged after the Roman period.\nMultiplexed amplicon sequencing reveals the heterogeneous spatial distribution of pyrethroid resistance mutations in Aedes albopictus mosquito populations in Southern France.\nAuthors: Fontaine, A.; Mignotte, A.; Lacour, G.; Nguyen, A.; Gomez, N.; Chanaud, L.; L'Ambert, G.; Briolant, S.\nScore: 8.6, Published: 2023-08-05 DOI: 10.1101/2023.08.05.552101\nThe risk of mosquito-borne diseases transmission is moving fast toward temperate climates with the colonization and proliferation of the Asian tiger mosquito vector Aedes albopictus and the rapid and mass transport of passengers returning from tropical regions where the viruses are endemic. The prevention of major Aedes-borne viruses heavily relies on the use of insecticides for vector control, mainly pyrethroids In Europe. High-throughput molecular assays can provide a cost-effective surrogate to phenotypic insecticide resistance assays when mutations have been previously linked to a resistance phenotype. Here, we screened for the spatial distribution of kdr mutations at a large scale using a two-step approach based on multiplexed amplicon sequencing and an unprecedented collection of field-derived mosquitoes in South of France. We identified the presence of the V1016G allele in 14 sites. The V1016G allele was predominantly found in South-East France close to the Italian border with two additional isolated sites close to Bordeaux and Marmande. All mosquitoes were heterozygous for this mutation and should not be phenotypically resistant to pyrethroid insecticide. Four other mutations were identified in our targeted genomic sequence: I1532T, M1006L, M1586L, M995L. Sequencing a section of maternally inherited mitochondrial genome confirmed that the spread of Ae. albopictus in France originated from founders with haplogroup A1. These findings contribute to the broader understanding of resistance dynamics in Europe and can inform targeted approaches to mitigate the impact of resistance on vector control.\nVitamin C activates young LINE-1 elements in mouse embryonic stem cells via H3K9me3 demethylation\nAuthors: Cheng, K. C. L.; Frost, J. M.; Sanchez-Luque, F. J.; Garcia-Canadas, M.; Taylor, D.; Yang, W. R.; Irayanar, B.; Sampath, S.; Patani, H.; Agger, K.; Helin, K.; Ficz, G.; Burns, K. H.; Ewing, A. D.; Garcia-Perez, J. L.; Branco, M. R.\nScore: 8.6, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552254\nVitamin C (vitC) enhances the activity of 2-oxoglutarate-dependent dioxygenases, including TET enzymes, which catalyse DNA demethylation, and Jumonji-domain histone demethylases. The epigenetic remodelling promoted by vitC improves the efficiency of induced pluripotent stem cell derivation, and is required to attain a ground-state of pluripotency in embryonic stem cells (ESCs) that closely mimics the inner cell mass of the early blastocyst. However, genome-wide DNA and histone demethylation can lead to upregulation of transposable elements (TEs), and it is not known how vitC addition in culture media affects TE expression in pluripotent stem cells. Here we show that vitC increases the expression of evolutionarily young LINE-1 (L1) elements in mouse ESCs. We find that TET activity is dispensable for these effects, and that instead L1 upregulation occurs largely as a result of H3K9me3 loss mediated by KDM4A/C histone demethylases. Despite increased L1 levels, we did not detect increased somatic insertion rates in vitC-treated cells. Notably, treatment of human ESCs with vitC also increases L1 protein levels, which could impact the genetic and epigenetic stability of human pluripotent stem cells.\nZebrafish models of Mucopolysaccharidosis types IIIA, B, \u0026 C show hyperactivity and changes in oligodendrocyte state\nAuthors: Gerken, E.; Ahmad, S.; Barthelson, K.; Lardelli, M.\nScore: 9.2, Published: 2023-08-03 DOI: 10.1101/2023.08.02.550904\nSanfilippo syndrome childhood dementia, also known as mucopolysaccharidosis type III (MPS III), is a rare inherited lysosomal storage disorder. Subtypes of MPS III are caused by deficiencies in one of four enzymes required for degradation of the glycosaminoglycan heparan sulfate (HS). An inability to degrade HS leads to progressive neurodegeneration and death in the second or third decades of life. Knowledge of MPS III pathogenesis is incomplete, and no effective therapies exist. We generated the hypomorphic mutations sgshS387Lfs, nagluA603Efs and hgsnatG577Sfsin the endogenous zebrafish genes orthologous to human SGSH, NAGLU, and HGSNAT that are loci for mutations causing MPS III subtypes MPS IIIA, B and C respectively. Our models display the primary MPS III disease signature of significant brain accumulation of HS, while behavioural analyses support hyperactivity phenotypes. Brain transcriptome analysis revealed changes related to lysosomal, glycosaminoglycan, immune system and iron homeostasis biology in all three models but also distinct differences in brain transcriptome state between models. The transcriptome analysis also indicated marked disturbance of the oligodendrocyte cell state in the brains of MPS IIIA, B and C zebrafish, supporting that effects on this cell type are an early and consistent characteristic of MPS III. Overall, our zebrafish models recapture key characteristics of the human disease and phenotypes seen in mouse models. Our models will allow exploitation of the zebrafishs extreme fecundity and accessible anatomy to dissect the pathological mechanisms both common and divergent between the MPS IIIA, B, and C subtypes.\nPostzygotic barriers persist despite ongoing introgression in hybridizing Mimulus species\nAuthors: Mantel, S. J.; Sweigart, A. L.\nScore: 4.2, Published: 2023-08-06 DOI: 10.1101/2023.08.05.552095\nThe evolution of postzygotic isolation is thought to be a key step in maintaining species boundaries upon secondary contact. Yet the dynamics and persistence of hybrid incompatibilities in naturally hybridizing species is not well understood. Here, we explore these issues with genetic mapping in three independent populations of recombinant inbred lines between naturally hybridizing monkeyflowers Mimulus guttatus and M. nasutus from the sympatric Catherine Creek population. We discovered that the three M. guttatus founders differ dramatically in admixture history, with nearly a quarter of one founders genome introgressed from M. nasutus. Comparative genetic mapping in the three RIL populations revealed three new putative inversions, each one segregating among the M. guttatus founders, two due to admixture. We also found strong, genome-wide transmission ratio distortion in all RILs, but patterns were highly variable among the three populations. At least some of this distortion appears to be explained by epistatic selection favoring parental genotypes, but tests of inter-chromosomal linkage disequilibrium also revealed multiple candidate Dobzhansky-Muller incompatibilities. We also mapped several genetic loci for hybrid pollen viability, including two interacting pairs that coincide with peaks of distortion. Remarkably, even with this limited sample of three M. guttatus lines, we discovered abundant segregating variation for hybrid incompatibilities with M. nasutus, suggesting this population harbors diverse contributors to postzygotic isolation. Moreover, even with substantial admixture, hybrid incompatibilities between Mimulus species persist, suggesting postzygotic isolation might be a potent force in maintaining species barriers in this system.\nUsing human genetics to develop strategies to increase erythropoietic output from genome-edited hematopoietic stem and progenitor cells\nAuthors: Camarena, J.; Luna, S. E.; Hampton, J. P.; Majeti, K. R.; Charlesworth, C. T.; Soupene, E.; Sheehan, V. A.; Cromer, M. K.; Porteus, M. H.\nScore: 4.6, Published: 2023-08-04 DOI: 10.1101/2023.08.04.552064\nHuman genetic polymorphisms result in a diversity of phenotypes. Some sequences are pathologic and lead to monogenic diseases, while others may confer beneficial traits. Genome editing is a powerful tool to recreate genotypes found in the population, including the ability to correct pathologic mutations. One interesting naturally occurring mutation is a truncation in the erythropoietin receptor (tEPOR) causing congenital erythrocytosis-- characterized by non-pathogenic hyper-production of red blood cells (RBCs). Using the precision of CRISPR/Cas9 genome editing, we have recreated tEPOR and studied the effect of variations of the genotype on RBC development. We then combined tEPOR with a correction strategy developed for {beta}-thalassemia and demonstrated that coupling the two genome editing events gave RBCs a significant selective advantage. This demonstrates the potential of combining human genetics with the precision of genome editing to enable safer and more effective genome editing therapies for patients with serious genetic diseases.\nWithin-host evolution of SARS-CoV-2: how often are mutations transmitted?\nAuthors: Korosec, C. S.; Wahl, L. M.; Heffernan, J. M.\nScore: 3.4, Published: 2023-08-08 DOI: 10.1101/2023.08.08.552503\nDespite a relatively low mutation rate, the large number of SARS-CoV-2 infections has allowed for substantial genetic change, leading to a multitude of emerging variants. Using a recently determined mutation rate (per site replication), as well as parameter estimates for within-host SARS-CoV-2 infection, we apply a stochastic transmission-bottleneck model to describe the survival probability of de novo SARS-CoV-2 mutations. For narrow bottlenecks, we find mutations affecting pertarget-cell attachment rate (with phenotypes associated with fusogenicity and ACE2 binding), have similar transmission probabilities to mutations affecting viral load clearance (with phenotypes associated with humoral evasion). We further find that mutations affecting the eclipse rate (with phenotypes associated with reorganization of cellular metabolic processes and synthesis of viral budding precursor material) are highly favoured relative to all other traits examined. We find mutations leading to reduced removal rates of infected cells (with phenotypes associated with innate immune evasion) have limited transmission advantage relative to mutations leading to humoral evasion. Predicted transmission probabilities, however, for mutations affecting innate immune evasion are more consistent with the range of clinically-estimated household transmission probabilities for de novo mutations. This result suggests that although mutations affecting humoral evasion are more easily transmitted when they occur, mutations affecting innate immune evasion may occur more readily. We examine our predictions in the context of a number of previously characterized mutations in circulating strains of SARS-CoV-2. Our work offers both a null model for SARS-CoV-2 substitution rates and predicts which aspects of viral life history are most likely to successfully evolve, despite low mutation rates and repeated transmission bottlenecks.\nT2T-YAO: a Telomere-to-telomere Assembled Diploid Reference Genome for Han Chinese\nAuthors: He, Y.; Chu, Y.; Guo, S.; Hu, J.; Li, R.; Zheng, Y.; Ma, X.; Du, Z.; Zhao, L.; Yu, W.; Xue, J.; Bian, W.; Yang, F.; Chen, X.; Zhang, P.; Wu, R.; Ma, Y.; Shao, C.; Chen, J.; Wang, J.; Li, J.; Wu, J.; Hu, X.; Long, Q.; Jiang, M.; Ye, H.; Song, S.; Li, G.; Wei, Y.; Xu, Y.; Ma, Y.; Chen, Y.; Wang, K.; Bao, J.; Xi, W.; Wang, F.; Ni, W.; Zhang, M.; Yu, Y.; Li, S.; Kang, Y.; Gao, Z.\nScore: 7.7, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549286\nSince its initial release in 2001, the human reference genome has been continuously improved in both continuity and accuracy, and the recently-released telomere-to-telomere version--T2T-CHM13--reaches its top quality after 20 years of effort. However, T2T-CHM13 does not represent an authentic diploid human genome, but rather one derived from a simplified, nearly homozygous genome of a hydatidiform mole cell line. To address this limitation and provide an alternative pertinent to the Chinese population, the largest ethnic group in the world, we have assembled a complete diploid human genome of a male Han Chinese, T2T-YAO, which includes telomere-to-telomere assemblies for all the 22+X+M and 22+Y chromosomes in his two haploids inherited separately from his parents. Both haplotypes contain no artificial sequences or model nucleotides and possess a high quality comparable to CHM13, with fewer than one error per [~]14 Mb. Derived from the individual who lives in the aboriginal region of Han Chinese, T2T-YAO shows clear ancestry and potential genetic continuity from the ancient ancestors of the Han population. Each haplotype of T2T-YAO possesses [~]340 Mb exclusive sequences and [~]3100 unique genes as compared to CHM13, and their genome sequences show greater genetic distance to CHM13 than to each other in terms of nucleotide polymorphism and structural variations. The construction of T2T-YAO would serve as a high-quality diploid reference that enables precise delineation of genomic variations in a haplotype-sensitive manner, which could advance our understandings in human evolution, hereditability of diseases and phenotypes, especially within the context of the unique variations of the Chinese population.\n",
  "wordCount" : "2627",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.04.05.535764">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.04.05.535764" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.04.05.535764">
        <p class="paperTitle">Broad functional profiling of fission yeast proteins using phenomics and machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.04.05.535764" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.04.05.535764" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rodriguez-Lopez, M.; Bordin, N.; Lees, J.; Scholes, H.; Hassan, S.; Saintain, Q.; Kamrad, S.; Orengo, C.; Bähler, J.</p>
        <p class="info">Score: 68.9, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.04.05.535764' target='https://doi.org/10.1101/2023.04.05.535764'> 10.1101/2023.04.05.535764</a></p>
        <p class="abstract">Many proteins remain poorly characterized even in well-studied organisms, presenting a bottleneck for research. We applied phenomics and machine-learning approaches with Schizosaccharomyces pombe for broad cues on protein functions. We assayed colony-growth phenotypes to measure the fitness of deletion mutants for 3509 non-essential genes in 131 conditions with different nutrients, drugs, and stresses. These analyses exposed phenotypes for 3492 mutants, including 124 mutants of  priority unstudied proteins conserved in humans, providing varied functional clues. For example, over 900 proteins were newly implicated in the resistance to oxidative stress. Phenotype-correlation networks suggested roles for poorly characterized proteins through  guilt by association with known proteins. For complementary functional insights, we predicted Gene Ontology (GO) terms using machine learning methods exploiting protein-network and protein-homology data (NET-FF). We obtained 56,594 high-scoring GO predictions, of which 22,060 also featured high information content. Our phenotype-correlation data and NET-FF predictions showed a strong concordance with existing PomBase GO annotations and protein networks, with integrated analyses revealing 1,675 novel GO predictions for 783 genes, including 47 predictions for 23 priority unstudied proteins. Experimental validation identified new proteins involved in cellular ageing, showing that these predictions and phenomics data provide a rich resource to uncover new protein functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552281">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552281" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552281">
        <p class="paperTitle">Leveraging fine-scale population structure reveals conservation in genetic effect sizes between human populations across a range of human phenotypes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552281" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552281" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hu, S.; Ferreira, L. A. F.; Shi, S.; Hellenthal, G.; Marchini, J.; Lawson, D. J.; Myers, S. R.</p>
        <p class="info">Score: 25.9, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552281' target='https://doi.org/10.1101/2023.08.08.552281'> 10.1101/2023.08.08.552281</a></p>
        <p class="abstract">An understanding of genetic differences between populations is essential for avoiding confounding in genome-wide association studies (GWAS) and understanding the evolution of human traits. Polygenic risk scores constructed in one group perform poorly in highly genetically-differentiated populations, for reasons which remain controversial. We developed a statistical ancestry inference pipeline able to decompose ancestry both within and between countries, and applied it to the UK Biobank data. This identifies fine-scale patterns of genetic relatedness not captured by standard and widely used principal components (PCs), and allows fine-scale population stratification correction that removes both false positive and false negative associations for traits with geographic correlations. We also develop and apply ANCHOR, an approach leveraging segments of distinct ancestries within individuals to estimate similarity in underlying causal effect sizes between groups, using an existing PGS. Applying ANCHOR to &gt;8000 people of mixed African and European ancestry, we demonstrate that estimated causal effect sizes are highly similar across these ancestries for 26 of 29 quantitative molecular and non-molecular phenotypes (mean correlation 0.98 &#43;/-0.08), providing evidence that gene-environment and gene-gene interactions do not play major roles in the poor prediction of European-ancestry PRS scores in African populations for these traits, contradicting previous findings. Instead our results provide optimism that shared causal mutations operate similarly in different groups, focussing the challenge of improving GWAS &#34;portability&#34; between groups on joint fine-mapping.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551265" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551265">
        <p class="paperTitle">Local population structure in Cambridgeshire during the Roman occupation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scheib, C. L.; Hui, R.; Rose, A. K.; Solnik, A.; D&#39;Atanasio, E.; Inskip, S. A.; Cessford, C.; Griffith, S. J.; Wiseman, R.; Neil, B.; Biers, T.; Harknett, S.-J.; Sasso, S.; Biagini, S. A.; Runfeldt, G.; Duhig, C.; Evans, C.; O&#39;Connell, T. C.; Metspalu, M.; Millett, M. J.; Robb, J. E.; Kivisild, T.</p>
        <p class="info">Score: 27.1, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551265' target='https://doi.org/10.1101/2023.07.31.551265'> 10.1101/2023.07.31.551265</a></p>
        <p class="abstract">The Roman period saw the empire expand across Europe and the Mediterranean, including much of what is today the United Kingdom. While there is written evidence of high mobility into and out of Britain for administrators, traders and the military, the impact of imperialism on local population structure is invisible in the textual record. The extent of genetic change that occurred in Britain before the Early Medieval Period and how closely linked by genetic kinship the local populations were, remains underexplored. Here, using genome-wide data from 52 ancient individuals from Cambridgeshire, we show low levels of genetic ancestry differentiation between Romano-British sites and lower levels of runs of homozygosity over 4 centimorgans (cM than in the Bronze Age and Neolithic. We find fourteen cases of genetic relatedness within and one between sites without evidence of patrilineal dominance and one case of temporary mobility within a family unit during the Late Romano-British period. We also show that the modern patterns of genetic ancestry composition in Modern Britain emerged after the Roman period.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552101">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552101" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552101">
        <p class="paperTitle">Multiplexed amplicon sequencing reveals the heterogeneous spatial distribution of pyrethroid resistance mutations in Aedes albopictus mosquito populations in Southern France.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552101" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552101" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fontaine, A.; Mignotte, A.; Lacour, G.; Nguyen, A.; Gomez, N.; Chanaud, L.; L&#39;Ambert, G.; Briolant, S.</p>
        <p class="info">Score: 8.6, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552101' target='https://doi.org/10.1101/2023.08.05.552101'> 10.1101/2023.08.05.552101</a></p>
        <p class="abstract">The risk of mosquito-borne diseases transmission is moving fast toward temperate climates with the colonization and proliferation of the Asian tiger mosquito vector Aedes albopictus and the rapid and mass transport of passengers returning from tropical regions where the viruses are endemic. The prevention of major Aedes-borne viruses heavily relies on the use of insecticides for vector control, mainly pyrethroids In Europe. High-throughput molecular assays can provide a cost-effective surrogate to phenotypic insecticide resistance assays when mutations have been previously linked to a resistance phenotype. Here, we screened for the spatial distribution of kdr mutations at a large scale using a two-step approach based on multiplexed amplicon sequencing and an unprecedented collection of field-derived mosquitoes in South of France. We identified the presence of the V1016G allele in 14 sites. The V1016G allele was predominantly found in South-East France close to the Italian border with two additional isolated sites close to Bordeaux and Marmande. All mosquitoes were heterozygous for this mutation and should not be phenotypically resistant to pyrethroid insecticide. Four other mutations were identified in our targeted genomic sequence: I1532T, M1006L, M1586L, M995L. Sequencing a section of maternally inherited mitochondrial genome confirmed that the spread of Ae. albopictus in France originated from founders with haplogroup A1. These findings contribute to the broader understanding of resistance dynamics in Europe and can inform targeted approaches to mitigate the impact of resistance on vector control.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552254">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552254" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552254">
        <p class="paperTitle">Vitamin C activates young LINE-1 elements in mouse embryonic stem cells via H3K9me3 demethylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552254" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552254" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cheng, K. C. L.; Frost, J. M.; Sanchez-Luque, F. J.; Garcia-Canadas, M.; Taylor, D.; Yang, W. R.; Irayanar, B.; Sampath, S.; Patani, H.; Agger, K.; Helin, K.; Ficz, G.; Burns, K. H.; Ewing, A. D.; Garcia-Perez, J. L.; Branco, M. R.</p>
        <p class="info">Score: 8.6, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552254' target='https://doi.org/10.1101/2023.08.07.552254'> 10.1101/2023.08.07.552254</a></p>
        <p class="abstract">Vitamin C (vitC) enhances the activity of 2-oxoglutarate-dependent dioxygenases, including TET enzymes, which catalyse DNA demethylation, and Jumonji-domain histone demethylases. The epigenetic remodelling promoted by vitC improves the efficiency of induced pluripotent stem cell derivation, and is required to attain a ground-state of pluripotency in embryonic stem cells (ESCs) that closely mimics the inner cell mass of the early blastocyst. However, genome-wide DNA and histone demethylation can lead to upregulation of transposable elements (TEs), and it is not known how vitC addition in culture media affects TE expression in pluripotent stem cells. Here we show that vitC increases the expression of evolutionarily young LINE-1 (L1) elements in mouse ESCs. We find that TET activity is dispensable for these effects, and that instead L1 upregulation occurs largely as a result of H3K9me3 loss mediated by KDM4A/C histone demethylases. Despite increased L1 levels, we did not detect increased somatic insertion rates in vitC-treated cells. Notably, treatment of human ESCs with vitC also increases L1 protein levels, which could impact the genetic and epigenetic stability of human pluripotent stem cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.550904">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.550904" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.550904">
        <p class="paperTitle">Zebrafish models of Mucopolysaccharidosis types IIIA, B, &amp; C show hyperactivity and changes in oligodendrocyte state</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.550904" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.550904" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gerken, E.; Ahmad, S.; Barthelson, K.; Lardelli, M.</p>
        <p class="info">Score: 9.2, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.550904' target='https://doi.org/10.1101/2023.08.02.550904'> 10.1101/2023.08.02.550904</a></p>
        <p class="abstract">Sanfilippo syndrome childhood dementia, also known as mucopolysaccharidosis type III (MPS III), is a rare inherited lysosomal storage disorder. Subtypes of MPS III are caused by deficiencies in one of four enzymes required for degradation of the glycosaminoglycan heparan sulfate (HS). An inability to degrade HS leads to progressive neurodegeneration and death in the second or third decades of life. Knowledge of MPS III pathogenesis is incomplete, and no effective therapies exist. We generated the hypomorphic mutations sgshS387Lfs, nagluA603Efs and hgsnatG577Sfsin the endogenous zebrafish genes orthologous to human SGSH, NAGLU, and HGSNAT that are loci for mutations causing MPS III subtypes MPS IIIA, B and C respectively. Our models display the primary MPS III disease signature of significant brain accumulation of HS, while behavioural analyses support hyperactivity phenotypes. Brain transcriptome analysis revealed changes related to lysosomal, glycosaminoglycan, immune system and iron homeostasis biology in all three models but also distinct differences in brain transcriptome state between models. The transcriptome analysis also indicated marked disturbance of the oligodendrocyte cell state in the brains of MPS IIIA, B and C zebrafish, supporting that effects on this cell type are an early and consistent characteristic of MPS III. Overall, our zebrafish models recapture key characteristics of the human disease and phenotypes seen in mouse models. Our models will allow exploitation of the zebrafishs extreme fecundity and accessible anatomy to dissect the pathological mechanisms both common and divergent between the MPS IIIA, B, and C subtypes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552095">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552095" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552095">
        <p class="paperTitle">Postzygotic barriers persist despite ongoing introgression in hybridizing Mimulus species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552095" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552095" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mantel, S. J.; Sweigart, A. L.</p>
        <p class="info">Score: 4.2, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552095' target='https://doi.org/10.1101/2023.08.05.552095'> 10.1101/2023.08.05.552095</a></p>
        <p class="abstract">The evolution of postzygotic isolation is thought to be a key step in maintaining species boundaries upon secondary contact. Yet the dynamics and persistence of hybrid incompatibilities in naturally hybridizing species is not well understood. Here, we explore these issues with genetic mapping in three independent populations of recombinant inbred lines between naturally hybridizing monkeyflowers Mimulus guttatus and M. nasutus from the sympatric Catherine Creek population. We discovered that the three M. guttatus founders differ dramatically in admixture history, with nearly a quarter of one founders genome introgressed from M. nasutus. Comparative genetic mapping in the three RIL populations revealed three new putative inversions, each one segregating among the M. guttatus founders, two due to admixture. We also found strong, genome-wide transmission ratio distortion in all RILs, but patterns were highly variable among the three populations. At least some of this distortion appears to be explained by epistatic selection favoring parental genotypes, but tests of inter-chromosomal linkage disequilibrium also revealed multiple candidate Dobzhansky-Muller incompatibilities. We also mapped several genetic loci for hybrid pollen viability, including two interacting pairs that coincide with peaks of distortion. Remarkably, even with this limited sample of three M. guttatus lines, we discovered abundant segregating variation for hybrid incompatibilities with M. nasutus, suggesting this population harbors diverse contributors to postzygotic isolation. Moreover, even with substantial admixture, hybrid incompatibilities between Mimulus species persist, suggesting postzygotic isolation might be a potent force in maintaining species barriers in this system.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552064">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552064" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552064">
        <p class="paperTitle">Using human genetics to develop strategies to increase erythropoietic output from genome-edited hematopoietic stem and progenitor cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552064" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552064" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Camarena, J.; Luna, S. E.; Hampton, J. P.; Majeti, K. R.; Charlesworth, C. T.; Soupene, E.; Sheehan, V. A.; Cromer, M. K.; Porteus, M. H.</p>
        <p class="info">Score: 4.6, Published: 2023-08-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552064' target='https://doi.org/10.1101/2023.08.04.552064'> 10.1101/2023.08.04.552064</a></p>
        <p class="abstract">Human genetic polymorphisms result in a diversity of phenotypes. Some sequences are pathologic and lead to monogenic diseases, while others may confer beneficial traits. Genome editing is a powerful tool to recreate genotypes found in the population, including the ability to correct pathologic mutations. One interesting naturally occurring mutation is a truncation in the erythropoietin receptor (tEPOR) causing congenital erythrocytosis-- characterized by non-pathogenic hyper-production of red blood cells (RBCs). Using the precision of CRISPR/Cas9 genome editing, we have recreated tEPOR and studied the effect of variations of the genotype on RBC development. We then combined tEPOR with a correction strategy developed for {beta}-thalassemia and demonstrated that coupling the two genome editing events gave RBCs a significant selective advantage. This demonstrates the potential of combining human genetics with the precision of genome editing to enable safer and more effective genome editing therapies for patients with serious genetic diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552503">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552503" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552503">
        <p class="paperTitle">Within-host evolution of SARS-CoV-2: how often are mutations transmitted?</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552503" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552503" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Korosec, C. S.; Wahl, L. M.; Heffernan, J. M.</p>
        <p class="info">Score: 3.4, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552503' target='https://doi.org/10.1101/2023.08.08.552503'> 10.1101/2023.08.08.552503</a></p>
        <p class="abstract">Despite a relatively low mutation rate, the large number of SARS-CoV-2 infections has allowed for substantial genetic change, leading to a multitude of emerging variants. Using a recently determined mutation rate (per site replication), as well as parameter estimates for within-host SARS-CoV-2 infection, we apply a stochastic transmission-bottleneck model to describe the survival probability of de novo SARS-CoV-2 mutations. For narrow bottlenecks, we find mutations affecting pertarget-cell attachment rate (with phenotypes associated with fusogenicity and ACE2 binding), have similar transmission probabilities to mutations affecting viral load clearance (with phenotypes associated with humoral evasion). We further find that mutations affecting the eclipse rate (with phenotypes associated with reorganization of cellular metabolic processes and synthesis of viral budding precursor material) are highly favoured relative to all other traits examined. We find mutations leading to reduced removal rates of infected cells (with phenotypes associated with innate immune evasion) have limited transmission advantage relative to mutations leading to humoral evasion. Predicted transmission probabilities, however, for mutations affecting innate immune evasion are more consistent with the range of clinically-estimated household transmission probabilities for de novo mutations. This result suggests that although mutations affecting humoral evasion are more easily transmitted when they occur, mutations affecting innate immune evasion may occur more readily. We examine our predictions in the context of a number of previously characterized mutations in circulating strains of SARS-CoV-2. Our work offers both a null model for SARS-CoV-2 substitution rates and predicts which aspects of viral life history are most likely to successfully evolve, despite low mutation rates and repeated transmission bottlenecks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549286">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549286" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549286">
        <p class="paperTitle">T2T-YAO: a Telomere-to-telomere Assembled Diploid Reference Genome for Han Chinese</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549286" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549286" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, Y.; Chu, Y.; Guo, S.; Hu, J.; Li, R.; Zheng, Y.; Ma, X.; Du, Z.; Zhao, L.; Yu, W.; Xue, J.; Bian, W.; Yang, F.; Chen, X.; Zhang, P.; Wu, R.; Ma, Y.; Shao, C.; Chen, J.; Wang, J.; Li, J.; Wu, J.; Hu, X.; Long, Q.; Jiang, M.; Ye, H.; Song, S.; Li, G.; Wei, Y.; Xu, Y.; Ma, Y.; Chen, Y.; Wang, K.; Bao, J.; Xi, W.; Wang, F.; Ni, W.; Zhang, M.; Yu, Y.; Li, S.; Kang, Y.; Gao, Z.</p>
        <p class="info">Score: 7.7, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549286' target='https://doi.org/10.1101/2023.07.17.549286'> 10.1101/2023.07.17.549286</a></p>
        <p class="abstract">Since its initial release in 2001, the human reference genome has been continuously improved in both continuity and accuracy, and the recently-released telomere-to-telomere version--T2T-CHM13--reaches its top quality after 20 years of effort. However, T2T-CHM13 does not represent an authentic diploid human genome, but rather one derived from a simplified, nearly homozygous genome of a hydatidiform mole cell line. To address this limitation and provide an alternative pertinent to the Chinese population, the largest ethnic group in the world, we have assembled a complete diploid human genome of a male Han Chinese, T2T-YAO, which includes telomere-to-telomere assemblies for all the 22&#43;X&#43;M and 22&#43;Y chromosomes in his two haploids inherited separately from his parents. Both haplotypes contain no artificial sequences or model nucleotides and possess a high quality comparable to CHM13, with fewer than one error per [~]14 Mb. Derived from the individual who lives in the aboriginal region of Han Chinese, T2T-YAO shows clear ancestry and potential genetic continuity from the ancient ancestors of the Han population. Each haplotype of T2T-YAO possesses [~]340 Mb exclusive sequences and [~]3100 unique genes as compared to CHM13, and their genome sequences show greater genetic distance to CHM13 than to each other in terms of nucleotide polymorphism and structural variations. The construction of T2T-YAO would serve as a high-quality diploid reference that enables precise delineation of genomic variations in a haplotype-sensitive manner, which could advance our understandings in human evolution, hereditability of diseases and phenotypes, especially within the context of the unique variations of the Chinese population.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
