<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Higher-order combinatorial chromatin perturbations by engineered CRISPR-Cas12a for functional genomics
Authors: Hsiung, C. C.- S.; Wilson, C. M.; Sambold, N. A.; Dai, R.; Chen, Q.; Misiukiewicz, S.; Arab, A.; Teyssier, N.; O&#39;Loughlin, T.; Cofsky, J.; Shi, J.; Gilbert, L. A.
Score: 34.7, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558350
Multiplexed genetic perturbations are critical for testing functional interactions among coding or non-coding genetic elements. Compared to DNA cutting, repressive chromatin formation using CRISPR interference (CRISPRi) avoids genotoxicity and is more effective for perturbing non-coding regulatory elements in pooled assays.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Higher-order combinatorial chromatin perturbations by engineered CRISPR-Cas12a for functional genomics
Authors: Hsiung, C. C.- S.; Wilson, C. M.; Sambold, N. A.; Dai, R.; Chen, Q.; Misiukiewicz, S.; Arab, A.; Teyssier, N.; O&#39;Loughlin, T.; Cofsky, J.; Shi, J.; Gilbert, L. A.
Score: 34.7, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558350
Multiplexed genetic perturbations are critical for testing functional interactions among coding or non-coding genetic elements. Compared to DNA cutting, repressive chromatin formation using CRISPR interference (CRISPRi) avoids genotoxicity and is more effective for perturbing non-coding regulatory elements in pooled assays." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Higher-order combinatorial chromatin perturbations by engineered CRISPR-Cas12a for functional genomics
Authors: Hsiung, C. C.- S.; Wilson, C. M.; Sambold, N. A.; Dai, R.; Chen, Q.; Misiukiewicz, S.; Arab, A.; Teyssier, N.; O&#39;Loughlin, T.; Cofsky, J.; Shi, J.; Gilbert, L. A.
Score: 34.7, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558350
Multiplexed genetic perturbations are critical for testing functional interactions among coding or non-coding genetic elements. Compared to DNA cutting, repressive chromatin formation using CRISPR interference (CRISPRi) avoids genotoxicity and is more effective for perturbing non-coding regulatory elements in pooled assays."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Higher-order combinatorial chromatin perturbations by engineered CRISPR-Cas12a for functional genomics\nAuthors: Hsiung, C. C.- S.; Wilson, C. M.; Sambold, N. A.; Dai, R.; Chen, Q.; Misiukiewicz, S.; Arab, A.; Teyssier, N.; O\u0026#39;Loughlin, T.; Cofsky, J.; Shi, J.; Gilbert, L. A.\nScore: 34.7, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558350\nMultiplexed genetic perturbations are critical for testing functional interactions among coding or non-coding genetic elements. Compared to DNA cutting, repressive chromatin formation using CRISPR interference (CRISPRi) avoids genotoxicity and is more effective for perturbing non-coding regulatory elements in pooled assays.",
  "keywords": [
    
  ],
  "articleBody": " Higher-order combinatorial chromatin perturbations by engineered CRISPR-Cas12a for functional genomics\nAuthors: Hsiung, C. C.- S.; Wilson, C. M.; Sambold, N. A.; Dai, R.; Chen, Q.; Misiukiewicz, S.; Arab, A.; Teyssier, N.; O'Loughlin, T.; Cofsky, J.; Shi, J.; Gilbert, L. A.\nScore: 34.7, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558350\nMultiplexed genetic perturbations are critical for testing functional interactions among coding or non-coding genetic elements. Compared to DNA cutting, repressive chromatin formation using CRISPR interference (CRISPRi) avoids genotoxicity and is more effective for perturbing non-coding regulatory elements in pooled assays. However, current CRISPRi pooled screening approaches are generally limited to targeting 1-3 genomic sites per cell. To develop a tool for higher-order ([\u0026ge;]3) combinatorial targeting of genomic sites with CRISPRi in functional genomics screens, we engineered an Acidaminococcus Cas12a variant -- referred to as multiplexed transcriptional interference AsCas12a (multiAsCas12a). multiAsCas12a significantly outperforms state-of-the-art Cas12a variants in combinatorial CRISPRi targeting using high-order multiplexed arrays of CRISPR RNAs (crRNA) delivered by lentiviral transduction, including in high-throughput pooled screens using 6-plex crRNA array libraries. Using multiAsCas12a CRISPRi, we discover new enhancer elements and dissect the combinatorial function of cis-regulatory elements. These results demonstrate that multiAsCas12a enables group testing strategies to efficiently survey potentially numerous combinations of chromatin perturbations for biological discovery and engineering.\nThe nuclear PIWI-interacting protein Gtsf1 controls the selective degradation of small RNAs in Paramecium\nAuthors: Charmant, O.; Gruchota, J.; Arnaiz, O.; Zangarelli, C.; Betermier, M.; Nowak, K.; Legros, V.; Chevreux, G.; Nowak, J.; Duharcourt, S.\nScore: 22.8, Published: 2023-09-19 DOI: 10.1101/2023.09.19.558372\nCiliates undergo developmentally programmed genome elimination, in which small RNAs direct the removal of target DNA segments, including transposable elements. At each sexual generation, the development of the macronucleus (MAC) requires massive and reproducible elimination of a large proportion of the germline micronuclear (MIC) genome, leading to a highly streamlined somatic MAC genome. 25-nt long scnRNAs are produced from the entire germline MIC genome during meiosis, and this initial complex small RNA population is then transported to the maternal MAC, where selection of scnRNAs corresponding to germline (MIC)-specific sequences is thought to take place. Selected scnRNAs, loaded onto the PIWI protein Ptiwi09, guide the deposition of histone H3 post-translational modifications (H3K9me3 and H3K27me3) onto transposable elements in the developing macronucleus, ultimately triggering their specific elimination. How germline-specific MIC scnRNAs are selected remains to be determined. Here, we provide important mechanistic insights into the scnRNA selection pathway by identifying a Paramecium homolog of Gametocyte specific factor 1 (Gtsf1) as essential for the selective degradation of scnRNAs corresponding to retained somatic MAC sequences. Consistently, we also show that Gtsf1 is exclusively localized in the maternal macronucleus and associates with the scnRNA-binding protein Ptiwi09. Furthermore, Gtsf1 is necessary for DNA elimination and correct H3K9me3 and H3K27me3 localization in the new developing macronucleus, demonstrating that the scnRNA selection process is important for genome elimination. We propose that Gtsf1 is required for the coordinated degradation of Ptiwi09-scnRNA complexes that pair with nascent RNA transcribed from the maternal MAC genome, similarly to the mechanism suggested for microRNA target-directed degradation in metazoans.\nProximity labeling reveals new insights into the relationships between meiotic recombination proteins in S. cerevisiae\nAuthors: Voelkel-Meiman, K.; Poppel, A. J.; Liddle, J. C.; Balsbaugh, J. L.; MacQueen, A. J.\nScore: 9.4, Published: 2023-09-17 DOI: 10.1101/2023.09.17.558147\nSeveral protein ensembles facilitate MutS{gamma} crossover recombination and the associated process of synaptonemal complex (SC) assembly during meiosis, but the physical and functional relationships between the components involved remain obscure. We have employed proximity labeling as a phenotypic tool to discern functional relationships between meiotic recombination and SC proteins in S. cerevisiae, and to gain deeper insight into molecular deficits of crossover-defective mutants. We find that recombination initiation (Spo11) and the Mer3 helicase are dispensable for proximity labeling of the Zip3 E3 ligase by components of the ZZS ensemble (Zip2, Zip4 and Spo16) but are required for proximity labeling of Zip3 by Msh4, consistent with the possibility that MutS{gamma} joins Zip3 only after a specific recombination intermediate has been generated. Proximity labeling analysis of crossover-defective zip1 mutants suggests a key shared defect is a failure to assemble an early recombination ensemble where ZZS can properly engage Zip3. We furthermore discovered that Zip3s abundance within the meiotic cell is uniquely dependent on the presence of Zip1, and that the post-translational modification of Zip3 is promoted by most MutS{gamma} pathway proteins but countered by Zip1. Based on this and additional data, we propose a model whereby Zip1 stabilizes a functional, unmodified form of Zip3 until intermediate steps in recombination are complete. We also find that SC structural protein Ecm11 is proximity labeled by ZZS complex proteins in a Zip4-dependent manner, but by Zip3 and Msh4, at least in part, via a distinct pathway. Finally, streptavidin pulldowns followed by mass spectrometry on eleven different proximity labeling strains uncovers shared proximity targets of MutS{gamma}-associated proteins, some with known meiotic functions and others not yet implicated in a meiotic activity, highlighting the potential power of proximity labeling as a discovery tool.\nDiverse Plant-Parasitic Nematodes are Selectively Killed by Oxadiazole Thioether Pro-Nematicides\nAuthors: Knox, J.; Burns, A. R.; Cooke, B.; Cammalleri, S. R.; Kitner, M.; Ching, J.; Castelli, J. M. P.; Puumala, E.; Snider, J.; Koury, E.; Collins, J. B.; Andersen, E. C.; Stagljar, I.; Cowen, L.; Lautens, M.; Zasada, I.; Roy, P.\nScore: 7.1, Published: 2023-09-17 DOI: 10.1101/2023.09.14.557801\nLeft unchecked, plant-parasitic nematodes have the potential to devastate crops globally. Highly effective but non-selective nematicides are justifiably being phased-out, leaving farmers with limited options for managing nematode infestation. Here, we report our discovery of a 1,3,4-oxadiazole thioether scaffold called Cyprocide that selectively kills diverse plant-parasitic nematodes. Cyprocide is bioactivated into a lethal reactive electrophilic metabolite by specific nematode cytochrome P450 enzymes. Because Cyprocide fails to kill non-target organisms, we infer that the targeted lethality of this pro-nematicide derives from P450 substrate selectivity. Our findings demonstrate that Cyprocide is a selective nematicidal scaffold with broad-spectrum activity that holds the potential to help safeguard our global food supply.\nAn amplicon panel for high-throughput and low-cost genotyping of Pacific oyster\nAuthors: Sutherland, B. J. G.; Thompson, N. F.; Gujjula, K. R.; Carrasco, C. D.; Chadaram, S.; Chan, A. M.; Suttle, C. A.; Green, T. J.\nScore: 6.9, Published: 2023-08-28 DOI: 10.1101/2023.08.26.554329\nMaintaining genetic diversity in cultured shellfish can be challenging due to high variance in individual reproductive output, founder effects, and rapid genetic drift. However, it is important to consider for retaining adaptive potential and avoiding inbreeding depression, particularly in species with high mutational load. To support management of genetic diversity and selective breeding efforts in cultured Pacific oyster (Crassostrea gigas; a.k.a. Magallana gigas), we developed an amplicon panel comprising 592 amplicons positioned throughout the genome with an average of 50 markers per chromosome. Markers were selected based on high heterozygosity or high differentiation in Pacific oyster populations from the eastern Pacific Ocean. The developed panel was tested in a pilot study on 371 unique individuals, including those from hatcheries and farms (n = 72) and from three generations of the Vancouver Island University (VIU) breeding program (n = 181), as well as from naturalized or wild populations in the Northern Hemisphere (n = 118). High repeatability was observed in technical replicate samples; on average, 97.5% of typed markers were concordant between samples (n = 68 replicate sample pairs). In the pilot dataset, following quality filters, 463 markers were retained for analysis (average observed heterozygosity = 0.29 {+/-} 0.19; and FST = 0.040 {+/-} 0.046). Principal components analysis showed separate clustering of the Japan translocation lineage, China, and a commercial aquaculture population. Parentage analysis of the VIU breeding program identified 84% and 66% of the possible parent-offspring assignments in two spawning events involving 98 and 40 offspring each, respectively. Results were strongest in individuals with the highest genotyping rates, highlighting the value of sample and target SNP quality. The analysis identified pedigree errors that were then corrected. All code required to analyze the data is provided, including amplitools, a repository to move the panel target SNP output through parentage analysis. Collectively, these advances will enable rapid, consistent, and cost-effective genotyping of Pacific oyster.\nStable population structure in Europe since the Iron Age, despite high mobility\nAuthors: Antonio, M. L.; Weiss, C. L.; Gao, Z.; Sawyer, S.; Oberreiter, V.; Moots, H. M.; Spence, J. P.; Cheronet, O.; Zagorc, B.; Praxmarer, E.; Ozdogan, K. T.; Demetz, L.; Gelabert, P.; Fernandes, D.; Lucci, M.; Alihodzic, T.; Amrani, S.; Avetisyan, P.; Baillif-Ducros, C.; Bedic, Z.; Bertrand, A.; Bilic, M.; Bondioli, L.; Borowka, P.; Botte, E.; Burmaz, J.; Buzanic, D.; Candilio, F.; Cvetko, M.; De Angelis, D.; Drnic, I.; Elschek, K.; Fantar, M.; Gaspari, A.; Gasperetti, G.; Genchi, F.; Golubovic, S.; Hukelova, Z.; Jankauskas, R.; Vuckovic, K. J.; Jeremic, G.; Kaic, I.; Kazek, K.; Khachatryan, H.; Kh\nScore: 171.2, Published: 2023-09-21 DOI: 10.1101/2022.05.15.491973\nAncient DNA research in the past decade has revealed that European population structure changed dramatically in the prehistoric period (14,000-3,000 years before present, YBP), reflecting the widespread introduction of Neolithic farmer and Bronze Age Steppe ancestries. However, little is known about how population structure changed from the historical period onward (3,000 YBP - present). To address this, we collected whole genomes from 204 individuals from Europe and the Mediterranean, many of which are the first historical period genomes from their region (e.g. Armenia and France). We found that most regions show remarkable inter-individual heterogeneity. At least 7% of historical individuals carry ancestry uncommon in the region where they were sampled, some indicating cross-Mediterranean contacts. Despite this high level of mobility, overall population structure across western Eurasia is relatively stable through the historical period up to the present, mirroring geography. We show that, under standard population genetics models with local panmixia, the observed level of dispersal would lead to a collapse of population structure. Persistent population structure thus suggests a lower effective migration rate than indicated by the observed dispersal. We hypothesize that this phenomenon can be explained by extensive transient dispersal arising from drastically improved transportation networks and the Roman Empire's mobilization of people for trade, labor, and military. This work highlights the utility of ancient DNA in elucidating finer scale human population dynamics in recent history.\nPartitioning heritability using single-cell multi-omics identifies a novel macrophage subpopulation conveying genetic risks of coronary artery disease.\nAuthors: Jiang, J.; Hiron, T. K.; Agbaedeng, T.; Malhotra, Y.; Drydale, E. N.; Bancroft, J.; Ng, E.; Reschen, M.; Davison, L. J.; O'Callaghan, C.\nScore: 3.2, Published: 2023-09-17 DOI: 10.1101/2023.09.14.557845\nBackgroundCoronary artery disease (CAD), the leading cause of death worldwide, is influenced by both environmental and genetic factors. While over 250 genetic risk loci have been identified through genome-wide association studies, the specific causal variants and their regulatory mechanisms are still largely unknown, particularly in disease-relevant cell types like macrophages. MethodsWe utilized single-cell RNA-seq (scRNA-seq) and single-cell multi-omics approaches in primary human monocyte-derived macrophages to explore the transcriptional regulatory network involved in a critical pathogenic event of coronary atherosclerosis--the formation of lipid-laden foam cells. Meta-analysis of scRNA-seq datasets from 26 human plaque samples was undertaken to provide a comprehensive atlas of lesional macrophages and to correlate subpopulations in vivo and ex vivo. The genetic risk levels of CAD were assessed by partitioning disease heritability across different macrophage subpopulations. ResultsWe identified a novel macrophage subpopulation, termed lipid-handling macrophages, both ex vivo and in vivo, and identified associated marker genes, transcription regulators, and functional pathways. 18,782 cis-regulatory elements were identified by jointly profiling the gene expression and chromatin accessibility of \u003e5000 macrophages. Integration with CAD GWAS data prioritized 121 CAD-related genetic variants and 56 candidate causal genes. We showed that CAD heritability was not uniformly distributed and was particularly enriched in the gene programs of lipid-handling macrophages. We investigated the cis-regulatory effect of a risk variant rs10488763 on FDX1, implicating the recruitment of AP-1 and C/EBP-beta in the causal mechanisms at this locus. ConclusionsOur results provide genetic evidence of the divergent roles of macrophage subsets in atherogenesis and highlight lipid-handling macrophages as a key sub-population through which genetic variants actively influence disease. These findings provide an unbiased framework for functional fine-mapping of GWAS results using single-cell multi-omics and offer new insights into the genotype-environment interactions underlying atherosclerotic disease.\nSeptoria tritici blotch resistance gene Stb15 encodes a lectin receptor-like kinase\nAuthors: Hafeez, A. N.; Chartrain, L.; Cong, F.; Cambon, F.; Clarke, M.; Griffiths, S.; Hayta, S.; Jiang, M.; Keller, B.; Kirby, R.; Kolodziej, M. C.; Powell, O. R.; Smedley, M.; Steuernagel, B.; Xian, W.; Wingen, L. U.; Cheng, S.; Saintenac, C.; Wulff, B. B. H.; Brown, J. K. M.\nScore: 11.1, Published: 2023-09-12 DOI: 10.1101/2023.09.11.557217\nSeptoria tritici blotch (STB), caused by the Dothideomycete fungus Zymoseptoria tritici, is of one of the most damaging diseases of bread wheat (Triticum aestivum)1 and the target of costly fungicide applications2. In line with the fungus apoplastic lifestyle, STB resistance genes isolated to date encode receptor-like kinases (RLKs) including a wall-associated kinase (Stb6) and a cysteine-rich kinase (Stb16q)3,4. Here, we used genome-wide association studies (GWAS) on a panel of 300 whole-genome shotgun-sequenced diverse wheat landraces (WatSeq consortium) to identify a 99 kb region containing six candidates for the Stb15 resistance gene. Mutagenesis and transgenesis confirmed a gene encoding an intronless G-type lectin RLK (LecRK) as Stb15. The characterisation of Stb15 exemplifies the unexpected diversity of RLKs conferring Z. tritici resistance in wheat.\nApplication of CRISPR-Cas12a technology in zebrafish to simulate beta-globin gene (HBB) mutations\nAuthors: Shafique, F.; Ali, S.; Balckburn, E.; ul Hassan, M.\nScore: 2.4, Published: 2023-09-23 DOI: 10.1101/2023.09.21.558926\nBackground: The human beta-globin (HBB) gene translated into the hemoglobin subunit beta (beta) of the hemoglobin protein. When mutated, it can lead to the blood disorder beta-thalassemia. Recently, the CRISPR-Cas12 technology has shown promising potential in treating different genetic abnormalities. Objectives: This study aimed to evaluate the feasibility of zebrafish hbbe1.1 gene editing via CRISPR-Cas12a gene-technology. Methods: In current study, thalassemic mutations were replicated in zebrafish to improve gene editing. the embryonic hemoglobin gene hbbe1.1 was preferred over adult hemoglobin hbbeta-a1 due to its early detection at larval stages. The CRISPR-Cas12 technology was utilised in combination with phosphorothioated \"rescue template\" (ssDNA L33P) to introduce base edits to the DNA sequence of zebrafish. Results: These experiments indeed resulted in the alteration of a single amino acid at the protein level. With the help of this genetic editing method, we were able to generate a novel zebrafish strain that carried specific amino acid alterations resembling the pathogenic mutations found in HBB for beta-thalassemia disease. However, in several cases additional indels or base alterations were observed. Conclusion: Our findings suggests that MMEJ double stranded break repair mechanism causes more knock-in events and germline inheritance than HR-mediated events. Modifying the technique could improve results by reducing MMEJ frequency.\nIdentification of Genetic Suppressors for a Berardinelli-Seip Congenital Generalized Lipodystrophy Type 2 (BSCL2) Pathogenic Variant in C. elegans\nAuthors: Bai, X.; Smith, H. E.; Golden, A.\nScore: 2.0, Published: 2023-09-24 DOI: 10.1101/2023.09.22.559059\nMaintaining the metabolic homeostasis of fatty acids is crucial for human health. Excess fatty acids are stored in lipid droplets (LDs), the primary energy reservoir that helps regulate fat and lipid homeostasis in nearly all cell types. Seipin (BSCL2), a conserved endoplasmic reticulum protein, plays a critical role in LD biogenesis and regulating LD morphology. Pathogenic variants of seipin are associated with multiple human genetic diseases, including Berardinelli-Seip Congenital Generalized Lipodystrophy Type 2 (BSCL2). However, the cellular and molecular mechanisms by which dysfunctional seipin leads to these diseases remain unclear. To model BSCL2 disease, we generated an orthologous BSCL2 pathogenic variant seip-1(A185P) using CRISPR/Cas9 genome editing in Caenorhabditis elegans. This variant led to severe developmental and cellular defects, including embryonic lethality, impaired eggshell formation, and abnormally enlarged LDs. We set out to identify genetic determinants that could suppress these defective phenotypes in the seip-1(A185P) mutant background. To this end, we conducted an unbiased chemical mutagenesis screen to identify genetic suppressors that restore embryonic viability in the seip-1(A185P) mutant background. A total of five suppressor lines were isolated and recovered from the screen. The defective phenotypes of seip-1(A185P), including embryonic lethality and impaired eggshell formation, were significantly suppressed in each suppressor line. Two of the five suppressor lines also alleviated the enlarged LDs in the oocytes. We then mapped a suppressor candidate gene, R05D3.2 (renamed as lmbr-1), which is an ortholog of human LMBR1 (limb development membrane protein 1). The CRISPR/Cas9 edited lmbr-1 suppressor alleles, lmbr-1(Ser647Phe) and lmbr-1(Pro314Leu), both significantly suppressed embryonic lethality and defective eggshell formation in the seip-1(A185P) background. The newly identified suppressor lines offer valuable insights into potential genetic interactors and pathways that may regulate seipin in the lipodystrophy model.\n",
  "wordCount" : "2751",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558350">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558350" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558350">
        <p class="paperTitle">Higher-order combinatorial chromatin perturbations by engineered CRISPR-Cas12a for functional genomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558350" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558350" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hsiung, C. C.- S.; Wilson, C. M.; Sambold, N. A.; Dai, R.; Chen, Q.; Misiukiewicz, S.; Arab, A.; Teyssier, N.; O&#39;Loughlin, T.; Cofsky, J.; Shi, J.; Gilbert, L. A.</p>
        <p class="info">Score: 34.7, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558350' target='https://doi.org/10.1101/2023.09.18.558350'> 10.1101/2023.09.18.558350</a></p>
        <p class="abstract">Multiplexed genetic perturbations are critical for testing functional interactions among coding or non-coding genetic elements. Compared to DNA cutting, repressive chromatin formation using CRISPR interference (CRISPRi) avoids genotoxicity and is more effective for perturbing non-coding regulatory elements in pooled assays. However, current CRISPRi pooled screening approaches are generally limited to targeting 1-3 genomic sites per cell. To develop a tool for higher-order ([&amp;ge;]3) combinatorial targeting of genomic sites with CRISPRi in functional genomics screens, we engineered an Acidaminococcus Cas12a variant -- referred to as multiplexed transcriptional interference AsCas12a (multiAsCas12a). multiAsCas12a significantly outperforms state-of-the-art Cas12a variants in combinatorial CRISPRi targeting using high-order multiplexed arrays of CRISPR RNAs (crRNA) delivered by lentiviral transduction, including in high-throughput pooled screens using 6-plex crRNA array libraries. Using multiAsCas12a CRISPRi, we discover new enhancer elements and dissect the combinatorial function of cis-regulatory elements. These results demonstrate that multiAsCas12a enables group testing strategies to efficiently survey potentially numerous combinations of chromatin perturbations for biological discovery and engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558372">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558372" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558372">
        <p class="paperTitle">The nuclear PIWI-interacting protein Gtsf1 controls the selective degradation of small RNAs in Paramecium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558372" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558372" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Charmant, O.; Gruchota, J.; Arnaiz, O.; Zangarelli, C.; Betermier, M.; Nowak, K.; Legros, V.; Chevreux, G.; Nowak, J.; Duharcourt, S.</p>
        <p class="info">Score: 22.8, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558372' target='https://doi.org/10.1101/2023.09.19.558372'> 10.1101/2023.09.19.558372</a></p>
        <p class="abstract">Ciliates undergo developmentally programmed genome elimination, in which small RNAs direct the removal of target DNA segments, including transposable elements. At each sexual generation, the development of the macronucleus (MAC) requires massive and reproducible elimination of a large proportion of the germline micronuclear (MIC) genome, leading to a highly streamlined somatic MAC genome. 25-nt long scnRNAs are produced from the entire germline MIC genome during meiosis, and this initial complex small RNA population is then transported to the maternal MAC, where selection of scnRNAs corresponding to germline (MIC)-specific sequences is thought to take place. Selected scnRNAs, loaded onto the PIWI protein Ptiwi09, guide the deposition of histone H3 post-translational modifications (H3K9me3 and H3K27me3) onto transposable elements in the developing macronucleus, ultimately triggering their specific elimination. How germline-specific MIC scnRNAs are selected remains to be determined. Here, we provide important mechanistic insights into the scnRNA selection pathway by identifying a Paramecium homolog of Gametocyte specific factor 1 (Gtsf1) as essential for the selective degradation of scnRNAs corresponding to retained somatic MAC sequences. Consistently, we also show that Gtsf1 is exclusively localized in the maternal macronucleus and associates with the scnRNA-binding protein Ptiwi09. Furthermore, Gtsf1 is necessary for DNA elimination and correct H3K9me3 and H3K27me3 localization in the new developing macronucleus, demonstrating that the scnRNA selection process is important for genome elimination. We propose that Gtsf1 is required for the coordinated degradation of Ptiwi09-scnRNA complexes that pair with nascent RNA transcribed from the maternal MAC genome, similarly to the mechanism suggested for microRNA target-directed degradation in metazoans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.558147">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.558147" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.558147">
        <p class="paperTitle">Proximity labeling reveals new insights into the relationships between meiotic recombination proteins in S. cerevisiae</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.558147" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.558147" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Voelkel-Meiman, K.; Poppel, A. J.; Liddle, J. C.; Balsbaugh, J. L.; MacQueen, A. J.</p>
        <p class="info">Score: 9.4, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.558147' target='https://doi.org/10.1101/2023.09.17.558147'> 10.1101/2023.09.17.558147</a></p>
        <p class="abstract">Several protein ensembles facilitate MutS{gamma} crossover recombination and the associated process of synaptonemal complex (SC) assembly during meiosis, but the physical and functional relationships between the components involved remain obscure. We have employed proximity labeling as a phenotypic tool to discern functional relationships between meiotic recombination and SC proteins in S. cerevisiae, and to gain deeper insight into molecular deficits of crossover-defective mutants. We find that recombination initiation (Spo11) and the Mer3 helicase are dispensable for proximity labeling of the Zip3 E3 ligase by components of the ZZS ensemble (Zip2, Zip4 and Spo16) but are required for proximity labeling of Zip3 by Msh4, consistent with the possibility that MutS{gamma} joins Zip3 only after a specific recombination intermediate has been generated. Proximity labeling analysis of crossover-defective zip1 mutants suggests a key shared defect is a failure to assemble an early recombination ensemble where ZZS can properly engage Zip3. We furthermore discovered that Zip3s abundance within the meiotic cell is uniquely dependent on the presence of Zip1, and that the post-translational modification of Zip3 is promoted by most MutS{gamma} pathway proteins but countered by Zip1. Based on this and additional data, we propose a model whereby Zip1 stabilizes a functional, unmodified form of Zip3 until intermediate steps in recombination are complete. We also find that SC structural protein Ecm11 is proximity labeled by ZZS complex proteins in a Zip4-dependent manner, but by Zip3 and Msh4, at least in part, via a distinct pathway. Finally, streptavidin pulldowns followed by mass spectrometry on eleven different proximity labeling strains uncovers shared proximity targets of MutS{gamma}-associated proteins, some with known meiotic functions and others not yet implicated in a meiotic activity, highlighting the potential power of proximity labeling as a discovery tool.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557801">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557801" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557801">
        <p class="paperTitle">Diverse Plant-Parasitic Nematodes are Selectively Killed by Oxadiazole Thioether Pro-Nematicides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557801" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557801" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Knox, J.; Burns, A. R.; Cooke, B.; Cammalleri, S. R.; Kitner, M.; Ching, J.; Castelli, J. M. P.; Puumala, E.; Snider, J.; Koury, E.; Collins, J. B.; Andersen, E. C.; Stagljar, I.; Cowen, L.; Lautens, M.; Zasada, I.; Roy, P.</p>
        <p class="info">Score: 7.1, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557801' target='https://doi.org/10.1101/2023.09.14.557801'> 10.1101/2023.09.14.557801</a></p>
        <p class="abstract">Left unchecked, plant-parasitic nematodes have the potential to devastate crops globally. Highly effective but non-selective nematicides are justifiably being phased-out, leaving farmers with limited options for managing nematode infestation. Here, we report our discovery of a 1,3,4-oxadiazole thioether scaffold called Cyprocide that selectively kills diverse plant-parasitic nematodes. Cyprocide is bioactivated into a lethal reactive electrophilic metabolite by specific nematode cytochrome P450 enzymes. Because Cyprocide fails to kill non-target organisms, we infer that the targeted lethality of this pro-nematicide derives from P450 substrate selectivity. Our findings demonstrate that Cyprocide is a selective nematicidal scaffold with broad-spectrum activity that holds the potential to help safeguard our global food supply.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.26.554329">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.26.554329" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.26.554329">
        <p class="paperTitle">An amplicon panel for high-throughput and low-cost genotyping of Pacific oyster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.26.554329" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.26.554329" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sutherland, B. J. G.; Thompson, N. F.; Gujjula, K. R.; Carrasco, C. D.; Chadaram, S.; Chan, A. M.; Suttle, C. A.; Green, T. J.</p>
        <p class="info">Score: 6.9, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.26.554329' target='https://doi.org/10.1101/2023.08.26.554329'> 10.1101/2023.08.26.554329</a></p>
        <p class="abstract">Maintaining genetic diversity in cultured shellfish can be challenging due to high variance in individual reproductive output, founder effects, and rapid genetic drift. However, it is important to consider for retaining adaptive potential and avoiding inbreeding depression, particularly in species with high mutational load. To support management of genetic diversity and selective breeding efforts in cultured Pacific oyster (Crassostrea gigas; a.k.a. Magallana gigas), we developed an amplicon panel comprising 592 amplicons positioned throughout the genome with an average of 50 markers per chromosome. Markers were selected based on high heterozygosity or high differentiation in Pacific oyster populations from the eastern Pacific Ocean. The developed panel was tested in a pilot study on 371 unique individuals, including those from hatcheries and farms (n = 72) and from three generations of the Vancouver Island University (VIU) breeding program (n = 181), as well as from naturalized or wild populations in the Northern Hemisphere (n = 118). High repeatability was observed in technical replicate samples; on average, 97.5% of typed markers were concordant between samples (n = 68 replicate sample pairs). In the pilot dataset, following quality filters, 463 markers were retained for analysis (average observed heterozygosity = 0.29 {&#43;/-} 0.19; and FST = 0.040 {&#43;/-} 0.046). Principal components analysis showed separate clustering of the Japan translocation lineage, China, and a commercial aquaculture population. Parentage analysis of the VIU breeding program identified 84% and 66% of the possible parent-offspring assignments in two spawning events involving 98 and 40 offspring each, respectively. Results were strongest in individuals with the highest genotyping rates, highlighting the value of sample and target SNP quality. The analysis identified pedigree errors that were then corrected. All code required to analyze the data is provided, including amplitools, a repository to move the panel target SNP output through parentage analysis. Collectively, these advances will enable rapid, consistent, and cost-effective genotyping of Pacific oyster.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.05.15.491973">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.05.15.491973" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.05.15.491973">
        <p class="paperTitle">Stable population structure in Europe since the Iron Age, despite high mobility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.05.15.491973" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.05.15.491973" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Antonio, M. L.; Weiss, C. L.; Gao, Z.; Sawyer, S.; Oberreiter, V.; Moots, H. M.; Spence, J. P.; Cheronet, O.; Zagorc, B.; Praxmarer, E.; Ozdogan, K. T.; Demetz, L.; Gelabert, P.; Fernandes, D.; Lucci, M.; Alihodzic, T.; Amrani, S.; Avetisyan, P.; Baillif-Ducros, C.; Bedic, Z.; Bertrand, A.; Bilic, M.; Bondioli, L.; Borowka, P.; Botte, E.; Burmaz, J.; Buzanic, D.; Candilio, F.; Cvetko, M.; De Angelis, D.; Drnic, I.; Elschek, K.; Fantar, M.; Gaspari, A.; Gasperetti, G.; Genchi, F.; Golubovic, S.; Hukelova, Z.; Jankauskas, R.; Vuckovic, K. J.; Jeremic, G.; Kaic, I.; Kazek, K.; Khachatryan, H.; Kh</p>
        <p class="info">Score: 171.2, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.05.15.491973' target='https://doi.org/10.1101/2022.05.15.491973'> 10.1101/2022.05.15.491973</a></p>
        <p class="abstract">Ancient DNA research in the past decade has revealed that European population structure changed dramatically in the prehistoric period (14,000-3,000 years before present, YBP), reflecting the widespread introduction of Neolithic farmer and Bronze Age Steppe ancestries. However, little is known about how population structure changed from the historical period onward (3,000 YBP - present). To address this, we collected whole genomes from 204 individuals from Europe and the Mediterranean, many of which are the first historical period genomes from their region (e.g. Armenia and France). We found that most regions show remarkable inter-individual heterogeneity. At least 7% of historical individuals carry ancestry uncommon in the region where they were sampled, some indicating cross-Mediterranean contacts. Despite this high level of mobility, overall population structure across western Eurasia is relatively stable through the historical period up to the present, mirroring geography. We show that, under standard population genetics models with local panmixia, the observed level of dispersal would lead to a collapse of population structure. Persistent population structure thus suggests a lower effective migration rate than indicated by the observed dispersal. We hypothesize that this phenomenon can be explained by extensive transient dispersal arising from drastically improved transportation networks and the Roman Empire&#39;s mobilization of people for trade, labor, and military. This work highlights the utility of ancient DNA in elucidating finer scale human population dynamics in recent history.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557845">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557845" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557845">
        <p class="paperTitle">Partitioning heritability using single-cell multi-omics identifies a novel macrophage subpopulation conveying genetic risks of coronary artery disease.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557845" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557845" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, J.; Hiron, T. K.; Agbaedeng, T.; Malhotra, Y.; Drydale, E. N.; Bancroft, J.; Ng, E.; Reschen, M.; Davison, L. J.; O&#39;Callaghan, C.</p>
        <p class="info">Score: 3.2, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557845' target='https://doi.org/10.1101/2023.09.14.557845'> 10.1101/2023.09.14.557845</a></p>
        <p class="abstract">BackgroundCoronary artery disease (CAD), the leading cause of death worldwide, is influenced by both environmental and genetic factors. While over 250 genetic risk loci have been identified through genome-wide association studies, the specific causal variants and their regulatory mechanisms are still largely unknown, particularly in disease-relevant cell types like macrophages.

MethodsWe utilized single-cell RNA-seq (scRNA-seq) and single-cell multi-omics approaches in primary human monocyte-derived macrophages to explore the transcriptional regulatory network involved in a critical pathogenic event of coronary atherosclerosis--the formation of lipid-laden foam cells. Meta-analysis of scRNA-seq datasets from 26 human plaque samples was undertaken to provide a comprehensive atlas of lesional macrophages and to correlate subpopulations in vivo and ex vivo. The genetic risk levels of CAD were assessed by partitioning disease heritability across different macrophage subpopulations.

ResultsWe identified a novel macrophage subpopulation, termed lipid-handling macrophages, both ex vivo and in vivo, and identified associated marker genes, transcription regulators, and functional pathways. 18,782 cis-regulatory elements were identified by jointly profiling the gene expression and chromatin accessibility of &gt;5000 macrophages. Integration with CAD GWAS data prioritized 121 CAD-related genetic variants and 56 candidate causal genes. We showed that CAD heritability was not uniformly distributed and was particularly enriched in the gene programs of lipid-handling macrophages. We investigated the cis-regulatory effect of a risk variant rs10488763 on FDX1, implicating the recruitment of AP-1 and C/EBP-beta in the causal mechanisms at this locus.

ConclusionsOur results provide genetic evidence of the divergent roles of macrophage subsets in atherogenesis and highlight lipid-handling macrophages as a key sub-population through which genetic variants actively influence disease. These findings provide an unbiased framework for functional fine-mapping of GWAS results using single-cell multi-omics and offer new insights into the genotype-environment interactions underlying atherosclerotic disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557217">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557217" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557217">
        <p class="paperTitle">Septoria tritici blotch resistance gene Stb15 encodes a lectin receptor-like kinase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557217" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557217" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hafeez, A. N.; Chartrain, L.; Cong, F.; Cambon, F.; Clarke, M.; Griffiths, S.; Hayta, S.; Jiang, M.; Keller, B.; Kirby, R.; Kolodziej, M. C.; Powell, O. R.; Smedley, M.; Steuernagel, B.; Xian, W.; Wingen, L. U.; Cheng, S.; Saintenac, C.; Wulff, B. B. H.; Brown, J. K. M.</p>
        <p class="info">Score: 11.1, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557217' target='https://doi.org/10.1101/2023.09.11.557217'> 10.1101/2023.09.11.557217</a></p>
        <p class="abstract">Septoria tritici blotch (STB), caused by the Dothideomycete fungus Zymoseptoria tritici, is of one of the most damaging diseases of bread wheat (Triticum aestivum)1 and the target of costly fungicide applications2. In line with the fungus apoplastic lifestyle, STB resistance genes isolated to date encode receptor-like kinases (RLKs) including a wall-associated kinase (Stb6) and a cysteine-rich kinase (Stb16q)3,4. Here, we used genome-wide association studies (GWAS) on a panel of 300 whole-genome shotgun-sequenced diverse wheat landraces (WatSeq consortium) to identify a 99 kb region containing six candidates for the Stb15 resistance gene. Mutagenesis and transgenesis confirmed a gene encoding an intronless G-type lectin RLK (LecRK) as Stb15. The characterisation of Stb15 exemplifies the unexpected diversity of RLKs conferring Z. tritici resistance in wheat.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558926">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558926" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558926">
        <p class="paperTitle">Application of CRISPR-Cas12a technology in zebrafish to simulate beta-globin gene (HBB) mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558926" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558926" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shafique, F.; Ali, S.; Balckburn, E.; ul Hassan, M.</p>
        <p class="info">Score: 2.4, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558926' target='https://doi.org/10.1101/2023.09.21.558926'> 10.1101/2023.09.21.558926</a></p>
        <p class="abstract">Background: The human beta-globin (HBB) gene translated into the hemoglobin subunit beta (beta) of the hemoglobin protein. When mutated, it can lead to the blood disorder beta-thalassemia. Recently, the CRISPR-Cas12 technology has shown promising potential in treating different genetic abnormalities. Objectives: This study aimed to evaluate the feasibility of zebrafish hbbe1.1 gene editing via CRISPR-Cas12a gene-technology. Methods: In current study, thalassemic mutations were replicated in zebrafish to improve gene editing. the embryonic hemoglobin gene hbbe1.1 was preferred over adult hemoglobin hbbeta-a1 due to its early detection at larval stages. The CRISPR-Cas12 technology was utilised in combination with phosphorothioated &#34;rescue template&#34; (ssDNA L33P) to introduce base edits to the DNA sequence of zebrafish. Results: These experiments indeed resulted in the alteration of a single amino acid at the protein level. With the help of this genetic editing method, we were able to generate a novel zebrafish strain that carried specific amino acid alterations resembling the pathogenic mutations found in HBB for beta-thalassemia disease. However, in several cases additional indels or base alterations were observed. Conclusion: Our findings suggests that MMEJ double stranded break repair mechanism causes more knock-in events and germline inheritance than HR-mediated events. Modifying the technique could improve results by reducing MMEJ frequency.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559059">
        <p class="paperTitle">Identification of Genetic Suppressors for a Berardinelli-Seip Congenital Generalized Lipodystrophy Type 2 (BSCL2) Pathogenic Variant in  C. elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bai, X.; Smith, H. E.; Golden, A.</p>
        <p class="info">Score: 2.0, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559059' target='https://doi.org/10.1101/2023.09.22.559059'> 10.1101/2023.09.22.559059</a></p>
        <p class="abstract">Maintaining the metabolic homeostasis of fatty acids is crucial for human health. Excess fatty acids are stored in lipid droplets (LDs), the primary energy reservoir that helps regulate fat and lipid homeostasis in nearly all cell types. Seipin (BSCL2), a conserved endoplasmic reticulum protein, plays a critical role in LD biogenesis and regulating LD morphology. Pathogenic variants of seipin are associated with multiple human genetic diseases, including Berardinelli-Seip Congenital Generalized Lipodystrophy Type 2 (BSCL2). However, the cellular and molecular mechanisms by which dysfunctional seipin leads to these diseases remain unclear. To model BSCL2 disease, we generated an orthologous BSCL2 pathogenic variant seip-1(A185P) using CRISPR/Cas9 genome editing in Caenorhabditis elegans. This variant led to severe developmental and cellular defects, including embryonic lethality, impaired eggshell formation, and abnormally enlarged LDs. We set out to identify genetic determinants that could suppress these defective phenotypes in the seip-1(A185P) mutant background. To this end, we conducted an unbiased chemical mutagenesis screen to identify genetic suppressors that restore embryonic viability in the seip-1(A185P) mutant background. A total of five suppressor lines were isolated and recovered from the screen. The defective phenotypes of seip-1(A185P), including embryonic lethality and impaired eggshell formation, were significantly suppressed in each suppressor line. Two of the five suppressor lines also alleviated the enlarged LDs in the oocytes. We then mapped a suppressor candidate gene, R05D3.2 (renamed as lmbr-1), which is an ortholog of human LMBR1 (limb development membrane protein 1). The CRISPR/Cas9 edited lmbr-1 suppressor alleles, lmbr-1(Ser647Phe) and lmbr-1(Pro314Leu), both significantly suppressed embryonic lethality and defective eggshell formation in the seip-1(A185P) background. The newly identified suppressor lines offer valuable insights into potential genetic interactors and pathways that may regulate seipin in the lipodystrophy model.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
