<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley
Authors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.
Score: 26.0, Published: 2023-09-22 DOI: 10.1101/2023.09.22.557807
Direct observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII).">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley
Authors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.
Score: 26.0, Published: 2023-09-22 DOI: 10.1101/2023.09.22.557807
Direct observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-27T10:39:11+00:00" />
<meta property="article:modified_time" content="2023-09-27T10:39:11+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley
Authors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.
Score: 26.0, Published: 2023-09-22 DOI: 10.1101/2023.09.22.557807
Direct observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII)."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley\nAuthors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.\nScore: 26.0, Published: 2023-09-22 DOI: 10.1101/2023.09.22.557807\nDirect observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII).",
  "keywords": [
    
  ],
  "articleBody": " Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley\nAuthors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.\nScore: 26.0, Published: 2023-09-22 DOI: 10.1101/2023.09.22.557807\nDirect observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII). CCII was founded in 1929 with tens of thousands of unique genotypes and has been adapted to local conditions in Davis, CA, USA for 58 generations. We find that natural selection has massively reduced genetic diversity leading to a single clonal lineage constituting most of the population by generation F50. Selection favored alleles originating from similar climates to that of Davis, and targeted genes regulating reproductive development, including some of the most well-characterized barley diversification loci, Vrs1, HvCEN, and Ppd-H1. We chronicle the dynamic evolution of reproductive timing in the population and uncover how parallel molecular pathways are targeted by stabilizing selection to optimize this trait. Our findings point to selection as the predominant force shaping genomic variation in one of the worlds oldest ongoing biological experiments. One-Sentence SummaryWholesale genetic restructuring of an experimental population is a consequence of rapid environmental adaptation.\nThe nuclear PIWI-interacting protein Gtsf1 controls the selective degradation of small RNAs in Paramecium\nAuthors: Charmant, O.; Gruchota, J.; Arnaiz, O.; Zangarelli, C.; Betermier, M.; Nowak, K.; Legros, V.; Chevreux, G.; Nowak, J.; Duharcourt, S.\nScore: 22.8, Published: 2023-09-19 DOI: 10.1101/2023.09.19.558372\nCiliates undergo developmentally programmed genome elimination, in which small RNAs direct the removal of target DNA segments, including transposable elements. At each sexual generation, the development of the macronucleus (MAC) requires massive and reproducible elimination of a large proportion of the germline micronuclear (MIC) genome, leading to a highly streamlined somatic MAC genome. 25-nt long scnRNAs are produced from the entire germline MIC genome during meiosis, and this initial complex small RNA population is then transported to the maternal MAC, where selection of scnRNAs corresponding to germline (MIC)-specific sequences is thought to take place. Selected scnRNAs, loaded onto the PIWI protein Ptiwi09, guide the deposition of histone H3 post-translational modifications (H3K9me3 and H3K27me3) onto transposable elements in the developing macronucleus, ultimately triggering their specific elimination. How germline-specific MIC scnRNAs are selected remains to be determined. Here, we provide important mechanistic insights into the scnRNA selection pathway by identifying a Paramecium homolog of Gametocyte specific factor 1 (Gtsf1) as essential for the selective degradation of scnRNAs corresponding to retained somatic MAC sequences. Consistently, we also show that Gtsf1 is exclusively localized in the maternal macronucleus and associates with the scnRNA-binding protein Ptiwi09. Furthermore, Gtsf1 is necessary for DNA elimination and correct H3K9me3 and H3K27me3 localization in the new developing macronucleus, demonstrating that the scnRNA selection process is important for genome elimination. We propose that Gtsf1 is required for the coordinated degradation of Ptiwi09-scnRNA complexes that pair with nascent RNA transcribed from the maternal MAC genome, similarly to the mechanism suggested for microRNA target-directed degradation in metazoans.\nCyclin N-terminal domain-containing 1 (CNTD1) is critical for crossover designation during meiotic prophase I and for maintenance of ovarian reserve in mammalian oocytes\nAuthors: Wood, A.; Ahmed, R. M.; Bradley, R. A.; Wolff, I. D.; Cohen, P. E.\nScore: 15.0, Published: 2023-09-25 DOI: 10.1101/2023.09.25.559118\nIn meiotic prophase I, hundreds of double-strand breaks (DSB) are induced to initiate homologous recombination, resulting in the formation of non-crossovers (NCO) and crossovers (CO). Of the 250-300 DSBs that form in mice, only ~10% are resolved as COs, with all remaining DSB events resolving as NCO. Thus, both the frequency and distribution of COs must be tightly regulated to ensure their appropriate localization across the genome. Errors in crossover designation and maturation can result in nondisjunction, leading to aneuploidy, germ cell death, birth defects, or infertility. That these errors are more evident in female meiosis compared to males suggests that the events of meiotic prophase I are sexually dimorphic with respect to DSB processing, resolution, and/or surveillance. Therefore, it is essential to investigate how COs are designated and regulated during prophase I in the female germline. Previous studies in our lab showed that Cyclin N-Terminal Domain Containing 1 (CNTD1) protein plays a critical role in CO designation during male meiosis, co-localizing with other essential CO proteins such as MutL{gamma} (MLH1/MLH3) and crossover site-associated cyclin-dependent kinase-2 (CDK2) during pachynema. In the current study, we investigated whether analogous roles for CNTD1 existed in female meiosis. Similar to Cntd1-/- males, Cntd1-/- females are sterile, with no gross morphological ovarian defects in adults albeit with a depletion in ovarian follicle numbers. During prophase I, early DSB induction and repair occur normally, but oocytes from Cntd1-/- females fail to accumulate MutL{gamma} in pachynema. Phenotypically, meiotic prophase I disruption in Cntd1-/- females is similar to that seen in Cntd1-/- males. Importantly, whereas the Caernohabditis elegans ortholog of CNTD1, COSA-1, appears to interact with CDK-2 during pachynema in oocytes, CNTD1 in mouse oocytes lacks the CDK-interacting motif, suggesting that its function in oocytes does not involve a cyclin-like induction of CDK2 phosphorylation activity. In contradiction to the timing of prophase I disruption in Cntd1-/- females at pachynema, we observe a dramatic depletion of primordial follicles, beginning shortly after birth and increasing dramatically through to adult ages. Those primordial follicles that persist, however, are able to develop through dictyate arrest into primary and larger follicular structures in pre-pubertal females, and only by adulthood were significant declines observed in all follicles. The early loss of oocytes in Cntd1-/- females from soon after birth, considerably earlier than that observed for mice lacking MLH1 or MLH3, suggests a novel early role for CNTD1 in CO designation and/or in ensuring progression of oocytes into dictyate arrest at late pachynema.\nStable population structure in Europe since the Iron Age, despite high mobility\nAuthors: Antonio, M. L.; Weiss, C. L.; Gao, Z.; Sawyer, S.; Oberreiter, V.; Moots, H. M.; Spence, J. P.; Cheronet, O.; Zagorc, B.; Praxmarer, E.; Ozdogan, K. T.; Demetz, L.; Gelabert, P.; Fernandes, D.; Lucci, M.; Alihodzic, T.; Amrani, S.; Avetisyan, P.; Baillif-Ducros, C.; Bedic, Z.; Bertrand, A.; Bilic, M.; Bondioli, L.; Borowka, P.; Botte, E.; Burmaz, J.; Buzanic, D.; Candilio, F.; Cvetko, M.; De Angelis, D.; Drnic, I.; Elschek, K.; Fantar, M.; Gaspari, A.; Gasperetti, G.; Genchi, F.; Golubovic, S.; Hukelova, Z.; Jankauskas, R.; Vuckovic, K. J.; Jeremic, G.; Kaic, I.; Kazek, K.; Khachatryan, H.; Kh\nScore: 174.1, Published: 2023-09-21 DOI: 10.1101/2022.05.15.491973\nAncient DNA research in the past decade has revealed that European population structure changed dramatically in the prehistoric period (14,000-3,000 years before present, YBP), reflecting the widespread introduction of Neolithic farmer and Bronze Age Steppe ancestries. However, little is known about how population structure changed from the historical period onward (3,000 YBP - present). To address this, we collected whole genomes from 204 individuals from Europe and the Mediterranean, many of which are the first historical period genomes from their region (e.g. Armenia and France). We found that most regions show remarkable inter-individual heterogeneity. At least 7% of historical individuals carry ancestry uncommon in the region where they were sampled, some indicating cross-Mediterranean contacts. Despite this high level of mobility, overall population structure across western Eurasia is relatively stable through the historical period up to the present, mirroring geography. We show that, under standard population genetics models with local panmixia, the observed level of dispersal would lead to a collapse of population structure. Persistent population structure thus suggests a lower effective migration rate than indicated by the observed dispersal. We hypothesize that this phenomenon can be explained by extensive transient dispersal arising from drastically improved transportation networks and the Roman Empires mobilization of people for trade, labor, and military. This work highlights the utility of ancient DNA in elucidating finer scale human population dynamics in recent history.\nHigher-order combinatorial chromatin perturbations by engineered CRISPR-Cas12a for functional genomics\nAuthors: Hsiung, C. C.- S.; Wilson, C. M.; Sambold, N. A.; Dai, R.; Chen, Q.; Misiukiewicz, S.; Arab, A.; Teyssier, N.; O'Loughlin, T.; Cofsky, J.; Shi, J.; Gilbert, L. A.\nScore: 35.5, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558350\nMultiplexed genetic perturbations are critical for testing functional interactions among coding or non-coding genetic elements. Compared to DNA cutting, repressive chromatin formation using CRISPR interference (CRISPRi) avoids genotoxicity and is more effective for perturbing non-coding regulatory elements in pooled assays. However, current CRISPRi pooled screening approaches are generally limited to targeting 1-3 genomic sites per cell. To develop a tool for higher-order ([\u0026ge;]3) combinatorial targeting of genomic sites with CRISPRi in functional genomics screens, we engineered an Acidaminococcus Cas12a variant -- referred to as multiplexed transcriptional interference AsCas12a (multiAsCas12a). multiAsCas12a significantly outperforms state-of-the-art Cas12a variants in combinatorial CRISPRi targeting using high-order multiplexed arrays of CRISPR RNAs (crRNA) delivered by lentiviral transduction, including in high-throughput pooled screens using 6-plex crRNA array libraries. Using multiAsCas12a CRISPRi, we discover new enhancer elements and dissect the combinatorial function of cis-regulatory elements. These results demonstrate that multiAsCas12a enables group testing strategies to efficiently survey potentially numerous combinations of chromatin perturbations for biological discovery and engineering.\nTranscriptome-based host epistasis and pathogen co-expression in barley-powdery mildew interactions\nAuthors: Velasquez-Zapata, V.; Smith, S. D.; Surana, P.; Chapman, A. V. E.; Wise, R. P.\nScore: 6.7, Published: 2023-09-23 DOI: 10.1101/2023.09.18.558274\nMildew locus a (Mla) from the Triticeae grain crop barley (Hordeum vulgare L.) encodes a multi-allelic series of nucleotide-binding leucine-rich repeat (NLR) immune receptors. These variable NLRs recognize complementary secreted effectors from the powdery mildew fungus, Blumeria hordei (Bh), to block disease progression. We used a dynamic time-course transcriptome of barley infected with Bh to infer gene effects and epistatic relationships governed by the Mla6 NLR, two other host loci critical to the interaction, Blufensin1 (Bln1) and Required for Mla6 resistance3 (Rar3), and genes that interact with them. Bln1 is an R-gene independent regulator of immunity and the resistant bln1 mutant exhibits enhanced basal defense to compatible Bh. Rar3 is required for MLA6-mediated generation of H2O2 and the hypersensitive reaction; the rar3 mutant contains an in-frame Lys-Leu deletion in the SGT1-specific domain that compromises immunity by a subset of Mla alleles. Interactions of Mla6 and Bln1 resulted in symmetric, suppression and masked epistasis on the Bh-induced barley transcriptome. Likewise, dominant or equal effects were caused by Mla6 and Sgt1. Of a total of 468 barley NLRs, 366 were expressed in our dataset and 115 of those were grouped under different gene effect models, which localized to several chromosome hotspots. The corresponding Bh infection transcriptome was classified into nine co-expressed modules, linking differential expression with pathogen development. Expression of most of 517 Bh effectors exhibited dependence on disease phenotype and was associated with appressorial or haustorial structures, suggesting that disease is regulated by a host-pathogen intercommunication network that diversifies the response.\nIdentification of Genetic Suppressors for a Berardinelli-Seip Congenital Generalized Lipodystrophy Type 2 (BSCL2) Pathogenic Variant in C. elegans\nAuthors: Bai, X.; Smith, H. E.; Golden, A.\nScore: 3.6, Published: 2023-09-24 DOI: 10.1101/2023.09.22.559059\nMaintaining the metabolic homeostasis of fatty acids is crucial for human health. Excess fatty acids are stored in lipid droplets (LDs), the primary energy reservoir that helps regulate fat and lipid homeostasis in nearly all cell types. Seipin (BSCL2), a conserved endoplasmic reticulum protein, plays a critical role in LD biogenesis and regulating LD morphology. Pathogenic variants of seipin are associated with multiple human genetic diseases, including Berardinelli-Seip Congenital Generalized Lipodystrophy Type 2 (BSCL2). However, the cellular and molecular mechanisms by which dysfunctional seipin leads to these diseases remain unclear. To model BSCL2 disease, we generated an orthologous BSCL2 pathogenic variant seip-1(A185P) using CRISPR/Cas9 genome editing in Caenorhabditis elegans. This variant led to severe developmental and cellular defects, including embryonic lethality, impaired eggshell formation, and abnormally enlarged LDs. We set out to identify genetic determinants that could suppress these defective phenotypes in the seip-1(A185P) mutant background. To this end, we conducted an unbiased chemical mutagenesis screen to identify genetic suppressors that restore embryonic viability in the seip-1(A185P) mutant background. A total of five suppressor lines were isolated and recovered from the screen. The defective phenotypes of seip-1(A185P), including embryonic lethality and impaired eggshell formation, were significantly suppressed in each suppressor line. Two of the five suppressor lines also alleviated the enlarged LDs in the oocytes. We then mapped a suppressor candidate gene, R05D3.2 (renamed as lmbr-1), which is an ortholog of human LMBR1 (limb development membrane protein 1). The CRISPR/Cas9 edited lmbr-1 suppressor alleles, lmbr-1(Ser647Phe) and lmbr-1(Pro314Leu), both significantly suppressed embryonic lethality and defective eggshell formation in the seip-1(A185P) background. The newly identified suppressor lines offer valuable insights into potential genetic interactors and pathways that may regulate seipin in the lipodystrophy model.\nApplication of CRISPR-Cas12a technology in zebrafish to simulate beta-globin gene (HBB) mutations\nAuthors: Shafique, F.; Ali, S.; Balckburn, E.; ul Hassan, M.\nScore: 2.4, Published: 2023-09-23 DOI: 10.1101/2023.09.21.558926\nBackgroundThe human {beta}-globin (HBB) gene translated into the hemoglobin subunit beta ({beta}) of the hemoglobin protein. When mutated, it can lead to the blood disorder {beta}-thalassemia. Recently, the CRISPR-Cas12 technology has shown promising potential in treating different genetic abnormalities. ObjectivesThis study aimed to evaluate the feasibility of zebrafish hbbe1.1 gene editing via CRISPR-Cas12a gene-technology. MethodsIn current study, thalassemic mutations were replicated in zebrafish to improve gene editing. the embryonic hemoglobin gene hbbe1.1 was preferred over adult hemoglobin hb{beta}-a1 due to its early detection at larval stages. The CRISPR-Cas12 technology was utilised in combination with phosphorothioated \"rescue template\" (ssDNA L33P) to introduce base edits to the DNA sequence of zebrafish. ResultsThese experiments indeed resulted in the alteration of a single amino acid at the protein level. With the help of this genetic editing method, we were able to generate a novel zebrafish strain that carried specific amino acid alterations resembling the pathogenic mutations found in HBB for {beta}-thalassemia disease. However, in several cases additional indels or base alterations were observed. ConclusionOur findings suggests that MMEJ double stranded break repair mechanism causes more knock-in events and germline inheritance than HR-mediated events. Modifying the technique could improve results by reducing MMEJ frequency.\nDiverse Plant-Parasitic Nematodes are Selectively Killed by Oxadiazole Thioether Pro-Nematicides\nAuthors: Knox, J.; Burns, A. R.; Cooke, B.; Cammalleri, S. R.; Kitner, M.; Ching, J.; Castelli, J. M. P.; Puumala, E.; Snider, J.; Koury, E.; Collins, J. B.; Andersen, E. C.; Stagljar, I.; Cowen, L.; Lautens, M.; Zasada, I.; Roy, P.\nScore: 7.1, Published: 2023-09-17 DOI: 10.1101/2023.09.14.557801\nLeft unchecked, plant-parasitic nematodes have the potential to devastate crops globally. Highly effective but non-selective nematicides are justifiably being phased-out, leaving farmers with limited options for managing nematode infestation. Here, we report our discovery of a 1,3,4-oxadiazole thioether scaffold called Cyprocide that selectively kills diverse plant-parasitic nematodes. Cyprocide is bioactivated into a lethal reactive electrophilic metabolite by specific nematode cytochrome P450 enzymes. Because Cyprocide fails to kill non-target organisms, we infer that the targeted lethality of this pro-nematicide derives from P450 substrate selectivity. Our findings demonstrate that Cyprocide is a selective nematicidal scaffold with broad-spectrum activity that holds the potential to help safeguard our global food supply.\nEnsembled best subset selection using summary statistics for polygenic risk prediction\nAuthors: Chen, T.; Zhang, H.; Mazumder, R.; Lin, X.\nScore: 2.0, Published: 2023-09-26 DOI: 10.1101/2023.09.25.559307\nPolygenic risk scores (PRS) enhance population risk stratification and advance personalized medicine, yet existing methods face a tradeoff between predictive power and computational efficiency. We introduce ALL-Sum, an easy-to-implement PRS method that combines an efficient summary statistic-based L0L2 penalized regression algorithm with an ensembling step that aggregates estimates from different tuning parameters for improved prediction performance. In extensive large-scale simulations across a wide range of polygenicity and genome-wide association studies (GWAS) sample sizes, ALL-Sum consistently outperforms popular alternative methods in terms of prediction accuracy, runtime, and memory usage. We analyze 27 published GWAS summary statistics for 11 complex traits from 9 reputable data sources, including the Global Lipids Genetics Consortium, Breast Cancer Association Consortium, and FinnGen, evaluated using individual-level UKBB data. ALL-Sum achieves the highest accuracy for most traits, particularly for GWAS with large sample sizes. We provide ALL-Sum as a user-friendly command-line software with pre-computed reference data for streamlined user-end analysis.\n",
  "wordCount" : "2683",
  "inLanguage": "en",
  "datePublished": "2023-09-27T10:39:11Z",
  "dateModified": "2023-09-27T10:39:11Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta"><span>updated on September 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.557807">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.557807" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.557807">
        <p class="paperTitle">Natural selection drives emergent genetic homogeneity in a century-scale experiment with barley</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.557807" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.557807" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Landis, J. B.; Guercio, A. M.; Brown, K. E.; Fiscus, C. J.; Morrell, P. L.; Koenig, D.</p>
        <p class="info">Score: 26.0, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.557807' target='https://doi.org/10.1101/2023.09.22.557807'> 10.1101/2023.09.22.557807</a></p>
        <p class="abstract">Direct observation is central to our understanding of the process of adaptation, but evolution is rarely documented in a large, multicellular organism for more than a few generations. Here, we observe genetic and phenotypic evolution across a century-scale competition experiment, barley composite cross II (CCII). CCII was founded in 1929 with tens of thousands of unique genotypes and has been adapted to local conditions in Davis, CA, USA for 58 generations. We find that natural selection has massively reduced genetic diversity leading to a single clonal lineage constituting most of the population by generation F50. Selection favored alleles originating from similar climates to that of Davis, and targeted genes regulating reproductive development, including some of the most well-characterized barley diversification loci, Vrs1, HvCEN, and Ppd-H1. We chronicle the dynamic evolution of reproductive timing in the population and uncover how parallel molecular pathways are targeted by stabilizing selection to optimize this trait. Our findings point to selection as the predominant force shaping genomic variation in one of the worlds oldest ongoing biological experiments.

One-Sentence SummaryWholesale genetic restructuring of an experimental population is a consequence of rapid environmental adaptation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558372">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558372" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558372">
        <p class="paperTitle">The nuclear PIWI-interacting protein Gtsf1 controls the selective degradation of small RNAs in Paramecium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558372" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558372" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Charmant, O.; Gruchota, J.; Arnaiz, O.; Zangarelli, C.; Betermier, M.; Nowak, K.; Legros, V.; Chevreux, G.; Nowak, J.; Duharcourt, S.</p>
        <p class="info">Score: 22.8, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558372' target='https://doi.org/10.1101/2023.09.19.558372'> 10.1101/2023.09.19.558372</a></p>
        <p class="abstract">Ciliates undergo developmentally programmed genome elimination, in which small RNAs direct the removal of target DNA segments, including transposable elements. At each sexual generation, the development of the macronucleus (MAC) requires massive and reproducible elimination of a large proportion of the germline micronuclear (MIC) genome, leading to a highly streamlined somatic MAC genome. 25-nt long scnRNAs are produced from the entire germline MIC genome during meiosis, and this initial complex small RNA population is then transported to the maternal MAC, where selection of scnRNAs corresponding to germline (MIC)-specific sequences is thought to take place. Selected scnRNAs, loaded onto the PIWI protein Ptiwi09, guide the deposition of histone H3 post-translational modifications (H3K9me3 and H3K27me3) onto transposable elements in the developing macronucleus, ultimately triggering their specific elimination. How germline-specific MIC scnRNAs are selected remains to be determined. Here, we provide important mechanistic insights into the scnRNA selection pathway by identifying a Paramecium homolog of Gametocyte specific factor 1 (Gtsf1) as essential for the selective degradation of scnRNAs corresponding to retained somatic MAC sequences. Consistently, we also show that Gtsf1 is exclusively localized in the maternal macronucleus and associates with the scnRNA-binding protein Ptiwi09. Furthermore, Gtsf1 is necessary for DNA elimination and correct H3K9me3 and H3K27me3 localization in the new developing macronucleus, demonstrating that the scnRNA selection process is important for genome elimination. We propose that Gtsf1 is required for the coordinated degradation of Ptiwi09-scnRNA complexes that pair with nascent RNA transcribed from the maternal MAC genome, similarly to the mechanism suggested for microRNA target-directed degradation in metazoans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559118">
        <p class="paperTitle">Cyclin N-terminal domain-containing 1 (CNTD1) is critical for crossover designation during meiotic prophase I and for maintenance of ovarian reserve in mammalian oocytes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wood, A.; Ahmed, R. M.; Bradley, R. A.; Wolff, I. D.; Cohen, P. E.</p>
        <p class="info">Score: 15.0, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559118' target='https://doi.org/10.1101/2023.09.25.559118'> 10.1101/2023.09.25.559118</a></p>
        <p class="abstract">In meiotic prophase I, hundreds of double-strand breaks (DSB) are induced to initiate homologous recombination, resulting in the formation of non-crossovers (NCO) and crossovers (CO). Of the 250-300 DSBs that form in mice, only ~10% are resolved as COs, with all remaining DSB events resolving as NCO. Thus, both the frequency and distribution of COs must be tightly regulated to ensure their appropriate localization across the genome. Errors in crossover designation and maturation can result in nondisjunction, leading to aneuploidy, germ cell death, birth defects, or infertility. That these errors are more evident in female meiosis compared to males suggests that the events of meiotic prophase I are sexually dimorphic with respect to DSB processing, resolution, and/or surveillance. Therefore, it is essential to investigate how COs are designated and regulated during prophase I in the female germline. Previous studies in our lab showed that Cyclin N-Terminal Domain Containing 1 (CNTD1) protein plays a critical role in CO designation during male meiosis, co-localizing with other essential CO proteins such as MutL{gamma} (MLH1/MLH3) and crossover site-associated cyclin-dependent kinase-2 (CDK2) during pachynema. In the current study, we investigated whether analogous roles for CNTD1 existed in female meiosis. Similar to Cntd1-/- males, Cntd1-/- females are sterile, with no gross morphological ovarian defects in adults albeit with a depletion in ovarian follicle numbers. During prophase I, early DSB induction and repair occur normally, but oocytes from Cntd1-/- females fail to accumulate MutL{gamma} in pachynema. Phenotypically, meiotic prophase I disruption in Cntd1-/- females is similar to that seen in Cntd1-/- males. Importantly, whereas the Caernohabditis elegans ortholog of CNTD1, COSA-1, appears to interact with CDK-2 during pachynema in oocytes, CNTD1 in mouse oocytes lacks the CDK-interacting motif, suggesting that its function in oocytes does not involve a cyclin-like induction of CDK2 phosphorylation activity. In contradiction to the timing of prophase I disruption in Cntd1-/- females at pachynema, we observe a dramatic depletion of primordial follicles, beginning shortly after birth and increasing dramatically through to adult ages. Those primordial follicles that persist, however, are able to develop through dictyate arrest into primary and larger follicular structures in pre-pubertal females, and only by adulthood were significant declines observed in all follicles. The early loss of oocytes in Cntd1-/- females from soon after birth, considerably earlier than that observed for mice lacking MLH1 or MLH3, suggests a novel early role for CNTD1 in CO designation and/or in ensuring progression of oocytes into dictyate arrest at late pachynema.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.05.15.491973">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.05.15.491973" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.05.15.491973">
        <p class="paperTitle">Stable population structure in Europe since the Iron Age, despite high mobility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.05.15.491973" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.05.15.491973" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Antonio, M. L.; Weiss, C. L.; Gao, Z.; Sawyer, S.; Oberreiter, V.; Moots, H. M.; Spence, J. P.; Cheronet, O.; Zagorc, B.; Praxmarer, E.; Ozdogan, K. T.; Demetz, L.; Gelabert, P.; Fernandes, D.; Lucci, M.; Alihodzic, T.; Amrani, S.; Avetisyan, P.; Baillif-Ducros, C.; Bedic, Z.; Bertrand, A.; Bilic, M.; Bondioli, L.; Borowka, P.; Botte, E.; Burmaz, J.; Buzanic, D.; Candilio, F.; Cvetko, M.; De Angelis, D.; Drnic, I.; Elschek, K.; Fantar, M.; Gaspari, A.; Gasperetti, G.; Genchi, F.; Golubovic, S.; Hukelova, Z.; Jankauskas, R.; Vuckovic, K. J.; Jeremic, G.; Kaic, I.; Kazek, K.; Khachatryan, H.; Kh</p>
        <p class="info">Score: 174.1, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.05.15.491973' target='https://doi.org/10.1101/2022.05.15.491973'> 10.1101/2022.05.15.491973</a></p>
        <p class="abstract">Ancient DNA research in the past decade has revealed that European population structure changed dramatically in the prehistoric period (14,000-3,000 years before present, YBP), reflecting the widespread introduction of Neolithic farmer and Bronze Age Steppe ancestries. However, little is known about how population structure changed from the historical period onward (3,000 YBP - present). To address this, we collected whole genomes from 204 individuals from Europe and the Mediterranean, many of which are the first historical period genomes from their region (e.g. Armenia and France). We found that most regions show remarkable inter-individual heterogeneity. At least 7% of historical individuals carry ancestry uncommon in the region where they were sampled, some indicating cross-Mediterranean contacts. Despite this high level of mobility, overall population structure across western Eurasia is relatively stable through the historical period up to the present, mirroring geography. We show that, under standard population genetics models with local panmixia, the observed level of dispersal would lead to a collapse of population structure. Persistent population structure thus suggests a lower effective migration rate than indicated by the observed dispersal. We hypothesize that this phenomenon can be explained by extensive transient dispersal arising from drastically improved transportation networks and the Roman Empires mobilization of people for trade, labor, and military. This work highlights the utility of ancient DNA in elucidating finer scale human population dynamics in recent history.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558350">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558350" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558350">
        <p class="paperTitle">Higher-order combinatorial chromatin perturbations by engineered CRISPR-Cas12a for functional genomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558350" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558350" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hsiung, C. C.- S.; Wilson, C. M.; Sambold, N. A.; Dai, R.; Chen, Q.; Misiukiewicz, S.; Arab, A.; Teyssier, N.; O&#39;Loughlin, T.; Cofsky, J.; Shi, J.; Gilbert, L. A.</p>
        <p class="info">Score: 35.5, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558350' target='https://doi.org/10.1101/2023.09.18.558350'> 10.1101/2023.09.18.558350</a></p>
        <p class="abstract">Multiplexed genetic perturbations are critical for testing functional interactions among coding or non-coding genetic elements. Compared to DNA cutting, repressive chromatin formation using CRISPR interference (CRISPRi) avoids genotoxicity and is more effective for perturbing non-coding regulatory elements in pooled assays. However, current CRISPRi pooled screening approaches are generally limited to targeting 1-3 genomic sites per cell. To develop a tool for higher-order ([&amp;ge;]3) combinatorial targeting of genomic sites with CRISPRi in functional genomics screens, we engineered an Acidaminococcus Cas12a variant -- referred to as multiplexed transcriptional interference AsCas12a (multiAsCas12a). multiAsCas12a significantly outperforms state-of-the-art Cas12a variants in combinatorial CRISPRi targeting using high-order multiplexed arrays of CRISPR RNAs (crRNA) delivered by lentiviral transduction, including in high-throughput pooled screens using 6-plex crRNA array libraries. Using multiAsCas12a CRISPRi, we discover new enhancer elements and dissect the combinatorial function of cis-regulatory elements. These results demonstrate that multiAsCas12a enables group testing strategies to efficiently survey potentially numerous combinations of chromatin perturbations for biological discovery and engineering.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558274">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558274" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558274">
        <p class="paperTitle">Transcriptome-based host epistasis and pathogen co-expression in barley-powdery mildew interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558274" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558274" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Velasquez-Zapata, V.; Smith, S. D.; Surana, P.; Chapman, A. V. E.; Wise, R. P.</p>
        <p class="info">Score: 6.7, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558274' target='https://doi.org/10.1101/2023.09.18.558274'> 10.1101/2023.09.18.558274</a></p>
        <p class="abstract">Mildew locus a (Mla) from the Triticeae grain crop barley (Hordeum vulgare L.) encodes a multi-allelic series of nucleotide-binding leucine-rich repeat (NLR) immune receptors. These variable NLRs recognize complementary secreted effectors from the powdery mildew fungus, Blumeria hordei (Bh), to block disease progression. We used a dynamic time-course transcriptome of barley infected with Bh to infer gene effects and epistatic relationships governed by the Mla6 NLR, two other host loci critical to the interaction, Blufensin1 (Bln1) and Required for Mla6 resistance3 (Rar3), and genes that interact with them. Bln1 is an R-gene independent regulator of immunity and the resistant bln1 mutant exhibits enhanced basal defense to compatible Bh. Rar3 is required for MLA6-mediated generation of H2O2 and the hypersensitive reaction; the rar3 mutant contains an in-frame Lys-Leu deletion in the SGT1-specific domain that compromises immunity by a subset of Mla alleles. Interactions of Mla6 and Bln1 resulted in symmetric, suppression and masked epistasis on the Bh-induced barley transcriptome. Likewise, dominant or equal effects were caused by Mla6 and Sgt1. Of a total of 468 barley NLRs, 366 were expressed in our dataset and 115 of those were grouped under different gene effect models, which localized to several chromosome hotspots. The corresponding Bh infection transcriptome was classified into nine co-expressed modules, linking differential expression with pathogen development. Expression of most of 517 Bh effectors exhibited dependence on disease phenotype and was associated with appressorial or haustorial structures, suggesting that disease is regulated by a host-pathogen intercommunication network that diversifies the response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559059" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559059">
        <p class="paperTitle">Identification of Genetic Suppressors for a Berardinelli-Seip Congenital Generalized Lipodystrophy Type 2 (BSCL2) Pathogenic Variant in  C. elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bai, X.; Smith, H. E.; Golden, A.</p>
        <p class="info">Score: 3.6, Published: 2023-09-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559059' target='https://doi.org/10.1101/2023.09.22.559059'> 10.1101/2023.09.22.559059</a></p>
        <p class="abstract">Maintaining the metabolic homeostasis of fatty acids is crucial for human health. Excess fatty acids are stored in lipid droplets (LDs), the primary energy reservoir that helps regulate fat and lipid homeostasis in nearly all cell types. Seipin (BSCL2), a conserved endoplasmic reticulum protein, plays a critical role in LD biogenesis and regulating LD morphology. Pathogenic variants of seipin are associated with multiple human genetic diseases, including Berardinelli-Seip Congenital Generalized Lipodystrophy Type 2 (BSCL2). However, the cellular and molecular mechanisms by which dysfunctional seipin leads to these diseases remain unclear. To model BSCL2 disease, we generated an orthologous BSCL2 pathogenic variant seip-1(A185P) using CRISPR/Cas9 genome editing in Caenorhabditis elegans. This variant led to severe developmental and cellular defects, including embryonic lethality, impaired eggshell formation, and abnormally enlarged LDs. We set out to identify genetic determinants that could suppress these defective phenotypes in the seip-1(A185P) mutant background. To this end, we conducted an unbiased chemical mutagenesis screen to identify genetic suppressors that restore embryonic viability in the seip-1(A185P) mutant background. A total of five suppressor lines were isolated and recovered from the screen. The defective phenotypes of seip-1(A185P), including embryonic lethality and impaired eggshell formation, were significantly suppressed in each suppressor line. Two of the five suppressor lines also alleviated the enlarged LDs in the oocytes. We then mapped a suppressor candidate gene, R05D3.2 (renamed as lmbr-1), which is an ortholog of human LMBR1 (limb development membrane protein 1). The CRISPR/Cas9 edited lmbr-1 suppressor alleles, lmbr-1(Ser647Phe) and lmbr-1(Pro314Leu), both significantly suppressed embryonic lethality and defective eggshell formation in the seip-1(A185P) background. The newly identified suppressor lines offer valuable insights into potential genetic interactors and pathways that may regulate seipin in the lipodystrophy model.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558926">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558926" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558926">
        <p class="paperTitle">Application of CRISPR-Cas12a technology in zebrafish to simulate beta-globin gene (HBB) mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558926" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558926" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shafique, F.; Ali, S.; Balckburn, E.; ul Hassan, M.</p>
        <p class="info">Score: 2.4, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558926' target='https://doi.org/10.1101/2023.09.21.558926'> 10.1101/2023.09.21.558926</a></p>
        <p class="abstract">BackgroundThe human {beta}-globin (HBB) gene translated into the hemoglobin subunit beta ({beta}) of the hemoglobin protein. When mutated, it can lead to the blood disorder {beta}-thalassemia. Recently, the CRISPR-Cas12 technology has shown promising potential in treating different genetic abnormalities.

ObjectivesThis study aimed to evaluate the feasibility of zebrafish hbbe1.1 gene editing via CRISPR-Cas12a gene-technology.

MethodsIn current study, thalassemic mutations were replicated in zebrafish to improve gene editing. the embryonic hemoglobin gene hbbe1.1 was preferred over adult hemoglobin hb{beta}-a1 due to its early detection at larval stages. The CRISPR-Cas12 technology was utilised in combination with phosphorothioated &#34;rescue template&#34; (ssDNA L33P) to introduce base edits to the DNA sequence of zebrafish.

ResultsThese experiments indeed resulted in the alteration of a single amino acid at the protein level. With the help of this genetic editing method, we were able to generate a novel zebrafish strain that carried specific amino acid alterations resembling the pathogenic mutations found in HBB for {beta}-thalassemia disease. However, in several cases additional indels or base alterations were observed.

ConclusionOur findings suggests that MMEJ double stranded break repair mechanism causes more knock-in events and germline inheritance than HR-mediated events. Modifying the technique could improve results by reducing MMEJ frequency.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557801">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557801" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557801">
        <p class="paperTitle">Diverse Plant-Parasitic Nematodes are Selectively Killed by Oxadiazole Thioether Pro-Nematicides</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557801" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557801" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Knox, J.; Burns, A. R.; Cooke, B.; Cammalleri, S. R.; Kitner, M.; Ching, J.; Castelli, J. M. P.; Puumala, E.; Snider, J.; Koury, E.; Collins, J. B.; Andersen, E. C.; Stagljar, I.; Cowen, L.; Lautens, M.; Zasada, I.; Roy, P.</p>
        <p class="info">Score: 7.1, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557801' target='https://doi.org/10.1101/2023.09.14.557801'> 10.1101/2023.09.14.557801</a></p>
        <p class="abstract">Left unchecked, plant-parasitic nematodes have the potential to devastate crops globally. Highly effective but non-selective nematicides are justifiably being phased-out, leaving farmers with limited options for managing nematode infestation. Here, we report our discovery of a 1,3,4-oxadiazole thioether scaffold called Cyprocide that selectively kills diverse plant-parasitic nematodes. Cyprocide is bioactivated into a lethal reactive electrophilic metabolite by specific nematode cytochrome P450 enzymes. Because Cyprocide fails to kill non-target organisms, we infer that the targeted lethality of this pro-nematicide derives from P450 substrate selectivity. Our findings demonstrate that Cyprocide is a selective nematicidal scaffold with broad-spectrum activity that holds the potential to help safeguard our global food supply.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559307">
        <p class="paperTitle">Ensembled best subset selection using summary statistics for polygenic risk prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, T.; Zhang, H.; Mazumder, R.; Lin, X.</p>
        <p class="info">Score: 2.0, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559307' target='https://doi.org/10.1101/2023.09.25.559307'> 10.1101/2023.09.25.559307</a></p>
        <p class="abstract">Polygenic risk scores (PRS) enhance population risk stratification and advance personalized medicine, yet existing methods face a tradeoff between predictive power and computational efficiency. We introduce ALL-Sum, an easy-to-implement PRS method that combines an efficient summary statistic-based L0L2 penalized regression algorithm with an ensembling step that aggregates estimates from different tuning parameters for improved prediction performance. In extensive large-scale simulations across a wide range of polygenicity and genome-wide association studies (GWAS) sample sizes, ALL-Sum consistently outperforms popular alternative methods in terms of prediction accuracy, runtime, and memory usage. We analyze 27 published GWAS summary statistics for 11 complex traits from 9 reputable data sources, including the Global Lipids Genetics Consortium, Breast Cancer Association Consortium, and FinnGen, evaluated using individual-level UKBB data. ALL-Sum achieves the highest accuracy for most traits, particularly for GWAS with large sample sizes. We provide ALL-Sum as a user-friendly command-line software with pre-computed reference data for streamlined user-end analysis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
