<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genetics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genetics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Patient derived model of UBA5-associated encephalopathy identifies defects in neurodevelopment and highlights potential therapies
Authors: Chen, H.; Wang, Y.-D.; Blan, A. W.; Almanza-Fuerte, E. P.; Bonkowski, E. S.; Bajpai, R.; Pruett-Miller, S. M.; Mefford, H. C.
Score: 12.6, Published: 2024-01-27 DOI: 10.1101/2024.01.25.577254
UBA5 encodes for the E1 enzyme of the UFMylation cascade, which plays an essential role in ER homeostasis. The clinical phenotypes of UBA5-associated encephalopathy include developmental delays, epilepsy and intellectual disability.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genetics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genetics" />
<meta property="og:description" content="Patient derived model of UBA5-associated encephalopathy identifies defects in neurodevelopment and highlights potential therapies
Authors: Chen, H.; Wang, Y.-D.; Blan, A. W.; Almanza-Fuerte, E. P.; Bonkowski, E. S.; Bajpai, R.; Pruett-Miller, S. M.; Mefford, H. C.
Score: 12.6, Published: 2024-01-27 DOI: 10.1101/2024.01.25.577254
UBA5 encodes for the E1 enzyme of the UFMylation cascade, which plays an essential role in ER homeostasis. The clinical phenotypes of UBA5-associated encephalopathy include developmental delays, epilepsy and intellectual disability." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genetics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genetics"/>
<meta name="twitter:description" content="Patient derived model of UBA5-associated encephalopathy identifies defects in neurodevelopment and highlights potential therapies
Authors: Chen, H.; Wang, Y.-D.; Blan, A. W.; Almanza-Fuerte, E. P.; Bonkowski, E. S.; Bajpai, R.; Pruett-Miller, S. M.; Mefford, H. C.
Score: 12.6, Published: 2024-01-27 DOI: 10.1101/2024.01.25.577254
UBA5 encodes for the E1 enzyme of the UFMylation cascade, which plays an essential role in ER homeostasis. The clinical phenotypes of UBA5-associated encephalopathy include developmental delays, epilepsy and intellectual disability."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genetics",
      "item": "https://trxiv.yorks0n.com/posts/genetics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genetics",
  "name": "genetics",
  "description": "Patient derived model of UBA5-associated encephalopathy identifies defects in neurodevelopment and highlights potential therapies\nAuthors: Chen, H.; Wang, Y.-D.; Blan, A. W.; Almanza-Fuerte, E. P.; Bonkowski, E. S.; Bajpai, R.; Pruett-Miller, S. M.; Mefford, H. C.\nScore: 12.6, Published: 2024-01-27 DOI: 10.1101/2024.01.25.577254\nUBA5 encodes for the E1 enzyme of the UFMylation cascade, which plays an essential role in ER homeostasis. The clinical phenotypes of UBA5-associated encephalopathy include developmental delays, epilepsy and intellectual disability.",
  "keywords": [
    
  ],
  "articleBody": " Patient derived model of UBA5-associated encephalopathy identifies defects in neurodevelopment and highlights potential therapies\nAuthors: Chen, H.; Wang, Y.-D.; Blan, A. W.; Almanza-Fuerte, E. P.; Bonkowski, E. S.; Bajpai, R.; Pruett-Miller, S. M.; Mefford, H. C.\nScore: 12.6, Published: 2024-01-27 DOI: 10.1101/2024.01.25.577254\nUBA5 encodes for the E1 enzyme of the UFMylation cascade, which plays an essential role in ER homeostasis. The clinical phenotypes of UBA5-associated encephalopathy include developmental delays, epilepsy and intellectual disability. To date, there is no humanized neuronal model to study the cellular and molecular consequences of UBA5 pathogenic variants. We developed and characterized patient-derived cortical organoid cultures and identified defects in GABAergic interneuron development. We demonstrated aberrant neuronal firing and microcephaly phenotypes in patient-derived organoids. Mechanistically, we show that ER homeostasis is perturbed along with exacerbated unfolded protein response pathway in cells and organoids expressing UBA5 pathogenic variants. We also assessed two gene expression modalities that augmented UBA5 expression to rescue aberrant molecular and cellular phenotypes. Our study provides a novel humanized model that allows further investigations of UBA5 variants in the brain and highlights novel systemic approaches to alleviate cellular aberrations for this rare, developmental disorder. One-sentence summaryPatient derived model of UBA5-assoicated DEE recapitulated disease phenotype, revealed defects in neurodevelopment, and highlighted potential therapies.\nBridge RNAs direct modular and programmable recombination of target and donor DNA\nAuthors: Durrant, M. G.; Perry, N. T.; Pai, J. J.; Jangid, A. R.; Athukoralage, J. S.; Hiraizumi, M.; McSpedon, J. P.; Pawluk, A.; Nishimasu, H.; Konermann, S.; Hsu, P. D.\nScore: 9.6, Published: 2024-01-26 DOI: 10.1101/2024.01.24.577089\nGenomic rearrangements, encompassing mutational changes in the genome such as insertions, deletions, or inversions, are essential for genetic diversity. These rearrangements are typically orchestrated by enzymes involved in fundamental DNA repair processes such as homologous recombination or in the transposition of foreign genetic material by viruses and mobile genetic elements (MGEs). We report that IS110 insertion sequences, a family of minimal and autonomous MGEs, express a structured non-coding RNA that binds specifically to their encoded recombinase. This bridge RNA contains two internal loops encoding nucleotide stretches that base-pair with the target DNA and donor DNA, which is the IS110 element itself. We demonstrate that the target-binding and donor-binding loops can be independently reprogrammed to direct sequence-specific recombination between two DNA molecules. This modularity enables DNA insertion into genomic target sites as well as programmable DNA excision and inversion. The IS110 bridge system expands the diversity of nucleic acid-guided systems beyond CRISPR and RNA interference, offering a unified mechanism for the three fundamental DNA rearrangements required for genome design.\nHistone variant H2BE enhances chromatin accessibility in neurons to promote synaptic gene expression and long-term memory\nAuthors: Feierman, E. R.; Louzon, S.; Prescott, N. A.; Biaco, T.; Gao, Q.; Qiu, Q.; Choi, K.; Palozola, K. C.; Voss, A. J.; Mehta, S.; Quaye, C.; Lynch, K.; Fuccillo, M. V.; Wu, H.; David, Y.; Korb, E.\nScore: 4.1, Published: 2024-01-30 DOI: 10.1101/2024.01.29.575103\nRegulation of histones occurs through multiple mechanisms including exchange with histone variants. Unlike canonical histones, variants are replication-independent and therefore accumulate in post-mitotic cells such as neurons. While recent findings link variants to neurological and neuropsychiatric disorders, few are well studied in the context of the brain. H2BE is the single H2B variant found outside germline tissues, yet its expression and effects on chromatin remained unclear. We applied new tools including novel antibodies, biochemical assays, and sequencing approaches to reveal broad H2BE expression in the brain and its role in regulating chromatin structure, neuronal transcription, and mouse behavior. H2BE is enriched at promoters and enhances chromatin accessibility. We further identify a single amino acid driving these accessibility changes. Lastly, we show that H2BE is critical for synaptic gene expression and long-term memory. Together, these data reveal a novel mechanism linking histone variants to chromatin regulation and neuronal function underlying memory.\nAssessing the potential of germplasm collections for the management of genetic diversity: the case of the French National Cryobank\nAuthors: Jacques, A.; Duclos, D.; Danchin-Burge, C.; Mercat, M.-J.; Tixier-Boichard, M.; Restoux, G.\nScore: 4.8, Published: 2024-01-30 DOI: 10.1101/2023.07.19.549644\nThrough a combination of selective pressure and genetic drift, there has been a notable erosion of genetic diversity in domesticated animal populations. In response, many countries, including France, have developed gene banks in order to conserve reproductive genetic material. Cryopreserved resources can potentially be used to manage the genetic diversity of populations, but this opportunity is seldom exploited. As an initial step toward this goal, we describe here a methodology for the characterization of the genetic diversity of cryopreserved collections. Using the example of the French National Cryobank, this study employs newly proposed biodiversity metrics to conduct a detailed assessment of the status of collections for six livestock species: cattle, sheep, goat, horse, donkey, and pig. Both the amount of resources available and their characteristics varied among species and/or breeds. Overall, breeds with a large commercial distribution had more donors in the collection than local breeds, while the number of doses available was mainly determined by the physiology of the species. An adapted version of the Gini-Simpson equitability index revealed an unbalanced number of donors between breeds for some species. Similarly, estimates of effective donor numbers (De) highlighted the unequal distribution of donors within a breed. Finally, we developed a new index of diversity impact (IDI) to assess the potential of a collection to reintroduce genetic diversity in contemporary populations. The IDI was calculated on the basis of pedigree data for 17 breeds of three livestock species, pig, sheep, and cattle, which differed in both population size and management program. IDI values are negative when the use of cryoconserved sires would decrease the overall kinship of the current population and positive when it would increase it, enabling the most interesting donors to be chosen for immediate use. Negative (favorable) IDI values 41 were found for both local breeds as well as for commercial populations. In general, older collections exhibited better IDI values but recently collected donors could also be useful for populations undergoing strong selection. Within a breed, IDI can be computed individually and thus be used to select the best sires for a given objective. In the absence of pedigree data, IDI values could also be calculated on the basis of marker genotypes. Overall, this study proposes a framework for the assessment of germplasm collections in the service of various objectives. Compared to FAO indicators motivated by breed reconstitution, the Gini-Simpson and De indices can help to plan sampling more efficiently, whereas IDI can guide donor selection in order to manage the diversity of existing populations. These indicators can be calculated at regular intervals to support the planning and management of collections at national and international levels and help population managers to exploit the resources currently available.\nA potential role of inefficient and non-specific piRNA production from the whole transcriptome\nAuthors: Shoji, K.; Tomari, Y.\nScore: 3.0, Published: 2024-01-24 DOI: 10.1101/2024.01.24.577019\nPIWI-interacting RNAs (piRNAs) are small RNAs that play a critical role in repressing transposons in the germline by guiding PIWI proteins to complementary target sequences. It is well known that the 5 ends of piRNAs are typically defined through cleavage by PIWI proteins or by the endonuclease Zucchini, and piRNAs are amplified via reciprocal cleavage of sense and antisense transcripts by PIWI proteins. Here, we show that low levels of sense-stranded piRNAs are produced from the whole transcriptome in silkworms, flies, and mice, in a manner independent from the known biogenesis pathways. Notably, this mechanism is utilized in silkworm cultured cells for the production of piRNAs derived from the extremely abundant sub-genomic RNAs of Bombyx mori latent virus, which have been suggested to play a role in suppressing viral replication. We propose that this inefficient and non-specific piRNA-production pathway acts as a primitive sensor for aberrantly abundant transcripts in the germline, potentially seeding piRNA sequence repertoire for more efficient and established biogenesis pathways.\nAccurate inference of population history in the presence of background selection\nAuthors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.\nScore: 34.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576291\n1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species. Simulations and theoretical work have shown that methods to infer changes in effective population size over time (Ne(t)) become increasingly inaccurate as the strength of linked selection increases. Here, we introduce an extension to the Pairwise Sequentially Markovian Coalescent (PSMC) algorithm, PSMC+, which explicitly co-models demographic history and natural selection. We benchmark our method using forward-in-time simulations with BGS and find that our approach improves the accuracy of effective population size inference. Leveraging a high resolution map of BGS in humans, we infer considerable changes in the magnitude of inferred effective population size relative to previous reports. Finally, we separately infer Ne(t) on the X chromosome and on the autosomes in diverse great apes without making a correction for selection, and find that the inferred ratio fluctuates substantially through time in a way that differs across species, showing that uncorrected selection may be an important driver of signals of genetic difference on the X chromosome and autosomes.\nSomatic epigenetic drift during shoot branching: a cell lineage-based model\nAuthors: Chen, Y.; Burian, A.; Johannes, F.\nScore: 1.9, Published: 2024-01-29 DOI: 10.1101/2024.01.24.577071\nPlant architecture is shaped by the continuous production of new organs, most of which emerge post-embryonically. This process includes the formation of new lateral branches along existing shoots. Shoot branching is fundamental to plant development, plant-environment interactions, and vegetative propagation. Current empirical evidence supports a 'detached meristem' model as the cellular basis of lateral shoot initiation. In this model, a small number of undifferentiated cells are 'sampled' from the periphery of the shoot apical meristem (SAM) to act as precursors for axillary buds, which eventually develop into new shoots. Repeated branching thus creates a series of cellular bottlenecks (i.e. somatic drift) that affect how de novo genetic and epigenetic mutations propagate through the plant body during development. Somatic drift could be particularly relevant for epigenetic changes in the form of stochastic DNA methylation gains and losses (i.e. spontaneous epimutations), as they have been shown to arise rapidly with each cell division. Here, we formalize a special case of the 'detached meristem' model, where precursor cells are randomly sampled from the SAM periphery in a way that maximizes cell lineage independence. By following a population of SAM cells through repeated branching processes, we show that somatic drift gives rise to a complex mixture of cellular phylogenies, which shape the evolution of cell-to-cell DNA methylation heterogeneity within the SAM over time. This process is dependent on the number of branch points, the strength of somatic drift as well as the epimutation rate. For many realistic cell biological settings, our model predicts that cell-to-cell DNA methylation heterogeneity in the SAM converges to non-zero states during development, suggesting that epigenetic variation is an inherent property of the SAM cell population. Our insights have direct implications for empirical studies of somatic (epi)genomic diversity in long-lived perennial and clonal species using bulk or single-cell sequencing approaches.\nThe neocortical infrastructure for language involves region-specific patterns of laminar gene expression\nAuthors: Wong, M. M. K.; Sha, Z.; Lutje, L.; Kong, X.-Z.; Velthuijs, N.; van Heukelum, S.; van de Berg, W. D. J.; Jonkman, L. E.; Fisher, S. E.; Francks, C.\nScore: 11.4, Published: 2024-01-20 DOI: 10.1101/2024.01.17.575815\nThe language network of the human brain has core components in the inferior frontal cortex and superior/middle temporal cortex, with left-hemisphere dominance in most people. Functional specialization and interconnectivity of these neocortical regions is likely to be reflected in their molecular and cellular profiles. Excitatory connections between cortical regions arise and innervate according to layer-specific patterns. Here we generated a new gene expression dataset from human postmortem cortical tissue samples from core language network regions, using spatial transcriptomics to discriminate gene expression across cortical layers. Integration of these data with existing single-cell expression data identified 56 genes that showed differences in laminar expression profiles between frontal and temporal language cortex together with upregulation in layer II/III and/or layer V/VI excitatory neurons. Based on data from large-scale genome-wide screening in the population, DNA variants within these 56 genes showed set-level associations with inter-individual variation in structural connectivity between left-hemisphere frontal and temporal language cortex, and with predisposition to dyslexia. The axon guidance genes SLIT1 and SLIT2 were consistently implicated. These findings identify region-specific patterns of laminar gene expression as a feature of the brains language network.\nA critical role for Pol II CTD phosphorylation in heterochromatic gene activation\nAuthors: Kainth, A. S.; Zhang, H.; Gross, D. S.\nScore: 1.5, Published: 2024-01-27 DOI: 10.1101/2024.01.26.577526\nHow gene activation works in heterochromatin, and how the mechanism might differ from the one used in euchromatin, has been largely unexplored. Previous work has shown that in SIR-regulated heterochromatin of Saccharomyces cerevisiae, gene activation occurs in the absence of covalent histone modifications and other alterations of chromatin commonly associated with transcription. Here we demonstrate that such activation occurs in a substantial fraction of cells, and this raises the possibility that an alternative activation pathway might be used. We address one such possibility, Pol II CTD phosphorylation, and explore this idea using a natural telomere-linked gene, YFR057w, as a model. Unlike covalent histone modifications, CTD phosphorylation at Ser2, Ser5 and Ser7 is abundant at the drug-induced heterochromatic gene. Particularly enriched relative to the euchromatic state is Ser2 phosphorylation. Consistent with a functional role for Ser2P, YFR057w is negligibly activated in cells deficient in the Ser2 CTD kinases Ctk1 and Bur1 even though the gene is strongly stimulated in ctk1{Delta} bur1 cells when it is placed in a euchromatic context. Collectively, our results are consistent with a critical role for Ser2 CTD phosphorylation in driving Pol II recruitment and transcription of a natural heterochromatic gene - an activity that may help supplant the need for histone epigenetic modifications.\nEvolution, diversity, function, and marker-based elimination of the disease susceptibility gene Snn1 in wheat\nAuthors: Seneviratne, S.; Shi, G.; Szabo-Hever, A.; Zhang, Z.; Peters Haugrud, A.; Running, K. L. D.; Singh, G.; Nandety, R. S.; Fiedler, J. D.; McClean, P. E.; Xu, S.; Friesen, T. L.; Faris, J. D.\nScore: 1.4, Published: 2024-01-26 DOI: 10.1101/2024.01.24.577084\nSeptoria nodorum blotch (SNB), caused by Parastagonospora nodorum, is a disease of durum and common wheat initiated by the recognition of pathogen-produced necrotrophic effectors (NEs) by specific wheat genes. The wheat gene Snn1 encodes a wall-associated kinase that directly interacts with the NE SnTox1 leading to the development of SNB. Here, sequence analysis of Snn1 from 114 accessions including diploid, tetraploid and hexaploid wheat species revealed that some wheat lines possess two copies of Snn1 (designated Snn1-B1 and Snn1-B2) approximately 120 kb apart. Snn1-B2 evolved relatively recently as a paralog of Snn1-B1, and both genes have undergone diversifying selection. Three point mutations associated with the formation of the first SnTox1-sensitive Snn1-B1 allele from a primitive wild wheat were identified. Four subsequent and independent SNPs, three in Snn1-B1 and one in Snn1-B2, converted the sensitive alleles to insensitive forms. Protein modeling indicated these four mutations could abolish Snn1-SnTox1 compatibility either through destabilization of the Snn1 protein or direct disruption of the protein-protein interaction. High-throughput markers were developed for the causal mutations and evaluated on panels of durum and common wheat. The markers were able to correctly identify 96.9 % of SnTox1-sensitive durum wheat accessions, and a marker for the null allele was 100% accurate at predicting SnTox1-insensitive lines in both durum and spring wheat. Results of this study increase our understanding of the evolution, diversity, and function of Snn1-B1 and Snn1-B2 genes and will be useful for marker-assisted elimination of these genes for better host resistance.\n",
  "wordCount" : "2645",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genetics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genetics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577254">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577254" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577254">
        <p class="paperTitle">Patient derived model of UBA5-associated encephalopathy identifies defects in neurodevelopment and highlights potential therapies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577254" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577254" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, H.; Wang, Y.-D.; Blan, A. W.; Almanza-Fuerte, E. P.; Bonkowski, E. S.; Bajpai, R.; Pruett-Miller, S. M.; Mefford, H. C.</p>
        <p class="info">Score: 12.6, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577254' target='https://doi.org/10.1101/2024.01.25.577254'> 10.1101/2024.01.25.577254</a></p>
        <p class="abstract">UBA5 encodes for the E1 enzyme of the UFMylation cascade, which plays an essential role in ER homeostasis. The clinical phenotypes of UBA5-associated encephalopathy include developmental delays, epilepsy and intellectual disability. To date, there is no humanized neuronal model to study the cellular and molecular consequences of UBA5 pathogenic variants. We developed and characterized patient-derived cortical organoid cultures and identified defects in GABAergic interneuron development. We demonstrated aberrant neuronal firing and microcephaly phenotypes in patient-derived organoids. Mechanistically, we show that ER homeostasis is perturbed along with exacerbated unfolded protein response pathway in cells and organoids expressing UBA5 pathogenic variants. We also assessed two gene expression modalities that augmented UBA5 expression to rescue aberrant molecular and cellular phenotypes. Our study provides a novel humanized model that allows further investigations of UBA5 variants in the brain and highlights novel systemic approaches to alleviate cellular aberrations for this rare, developmental disorder.

One-sentence summaryPatient derived model of UBA5-assoicated DEE recapitulated disease phenotype, revealed defects in neurodevelopment, and highlighted potential therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577089">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577089" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577089">
        <p class="paperTitle">Bridge RNAs direct modular and programmable recombination of target and donor DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577089" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577089" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Durrant, M. G.; Perry, N. T.; Pai, J. J.; Jangid, A. R.; Athukoralage, J. S.; Hiraizumi, M.; McSpedon, J. P.; Pawluk, A.; Nishimasu, H.; Konermann, S.; Hsu, P. D.</p>
        <p class="info">Score: 9.6, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577089' target='https://doi.org/10.1101/2024.01.24.577089'> 10.1101/2024.01.24.577089</a></p>
        <p class="abstract">Genomic rearrangements, encompassing mutational changes in the genome such as insertions, deletions, or inversions, are essential for genetic diversity. These rearrangements are typically orchestrated by enzymes involved in fundamental DNA repair processes such as homologous recombination or in the transposition of foreign genetic material by viruses and mobile genetic elements (MGEs). We report that IS110 insertion sequences, a family of minimal and autonomous MGEs, express a structured non-coding RNA that binds specifically to their encoded recombinase. This bridge RNA contains two internal loops encoding nucleotide stretches that base-pair with the target DNA and donor DNA, which is the IS110 element itself. We demonstrate that the target-binding and donor-binding loops can be independently reprogrammed to direct sequence-specific recombination between two DNA molecules. This modularity enables DNA insertion into genomic target sites as well as programmable DNA excision and inversion. The IS110 bridge system expands the diversity of nucleic acid-guided systems beyond CRISPR and RNA interference, offering a unified mechanism for the three fundamental DNA rearrangements required for genome design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.575103">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.575103" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.575103">
        <p class="paperTitle">Histone variant H2BE enhances chromatin accessibility in neurons to promote synaptic gene expression and long-term memory</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.575103" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.575103" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Feierman, E. R.; Louzon, S.; Prescott, N. A.; Biaco, T.; Gao, Q.; Qiu, Q.; Choi, K.; Palozola, K. C.; Voss, A. J.; Mehta, S.; Quaye, C.; Lynch, K.; Fuccillo, M. V.; Wu, H.; David, Y.; Korb, E.</p>
        <p class="info">Score: 4.1, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.575103' target='https://doi.org/10.1101/2024.01.29.575103'> 10.1101/2024.01.29.575103</a></p>
        <p class="abstract">Regulation of histones occurs through multiple mechanisms including exchange with histone variants. Unlike canonical histones, variants are replication-independent and therefore accumulate in post-mitotic cells such as neurons. While recent findings link variants to neurological and neuropsychiatric disorders, few are well studied in the context of the brain. H2BE is the single H2B variant found outside germline tissues, yet its expression and effects on chromatin remained unclear. We applied new tools including novel antibodies, biochemical assays, and sequencing approaches to reveal broad H2BE expression in the brain and its role in regulating chromatin structure, neuronal transcription, and mouse behavior. H2BE is enriched at promoters and enhances chromatin accessibility. We further identify a single amino acid driving these accessibility changes. Lastly, we show that H2BE is critical for synaptic gene expression and long-term memory. Together, these data reveal a novel mechanism linking histone variants to chromatin regulation and neuronal function underlying memory.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549644">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549644" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549644">
        <p class="paperTitle">Assessing the potential of germplasm collections for the management of genetic diversity: the case of the French National Cryobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549644" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549644" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jacques, A.; Duclos, D.; Danchin-Burge, C.; Mercat, M.-J.; Tixier-Boichard, M.; Restoux, G.</p>
        <p class="info">Score: 4.8, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549644' target='https://doi.org/10.1101/2023.07.19.549644'> 10.1101/2023.07.19.549644</a></p>
        <p class="abstract">Through a combination of selective pressure and genetic drift, there has been a notable erosion of genetic diversity in domesticated animal populations. In response, many countries, including France, have developed gene banks in order to conserve reproductive genetic material. Cryopreserved resources can potentially be used to manage the genetic diversity of populations, but this opportunity is seldom exploited. As an initial step toward this goal, we describe here a methodology for the characterization of the genetic diversity of cryopreserved collections. Using the example of the French National Cryobank, this study employs newly proposed biodiversity metrics to conduct a detailed assessment of the status of collections for six livestock species: cattle, sheep, goat, horse, donkey, and pig.

Both the amount of resources available and their characteristics varied among species and/or breeds. Overall, breeds with a large commercial distribution had more donors in the collection than local breeds, while the number of doses available was mainly determined by the physiology of the species. An adapted version of the Gini-Simpson equitability index revealed an unbalanced number of donors between breeds for some species. Similarly, estimates of effective donor numbers (De) highlighted the unequal distribution of donors within a breed. Finally, we developed a new index of diversity impact (IDI) to assess the potential of a collection to reintroduce genetic diversity in contemporary populations. The IDI was calculated on the basis of pedigree data for 17 breeds of three livestock species, pig, sheep, and cattle, which differed in both population size and management program. IDI values are negative when the use of cryoconserved sires would decrease the overall kinship of the current population and positive when it would increase it, enabling the most interesting donors to be chosen for immediate use. Negative (favorable) IDI values 41 were found for both local breeds as well as for commercial populations. In general, older collections exhibited better IDI values but recently collected donors could also be useful for populations undergoing strong selection. Within a breed, IDI can be computed individually and thus be used to select the best sires for a given objective. In the absence of pedigree data, IDI values could also be calculated on the basis of marker genotypes.

Overall, this study proposes a framework for the assessment of germplasm collections in the service of various objectives. Compared to FAO indicators motivated by breed reconstitution, the Gini-Simpson and De indices can help to plan sampling more efficiently, whereas IDI can guide donor selection in order to manage the diversity of existing populations. These indicators can be calculated at regular intervals to support the planning and management of collections at national and international levels and help population managers to exploit the resources currently available.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577019">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577019" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577019">
        <p class="paperTitle">A potential role of inefficient and non-specific piRNA production from the whole transcriptome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577019" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577019" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shoji, K.; Tomari, Y.</p>
        <p class="info">Score: 3.0, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577019' target='https://doi.org/10.1101/2024.01.24.577019'> 10.1101/2024.01.24.577019</a></p>
        <p class="abstract">PIWI-interacting RNAs (piRNAs) are small RNAs that play a critical role in repressing transposons in the germline by guiding PIWI proteins to complementary target sequences. It is well known that the 5 ends of piRNAs are typically defined through cleavage by PIWI proteins or by the endonuclease Zucchini, and piRNAs are amplified via reciprocal cleavage of sense and antisense transcripts by PIWI proteins. Here, we show that low levels of sense-stranded piRNAs are produced from the whole transcriptome in silkworms, flies, and mice, in a manner independent from the known biogenesis pathways. Notably, this mechanism is utilized in silkworm cultured cells for the production of piRNAs derived from the extremely abundant sub-genomic RNAs of Bombyx mori latent virus, which have been suggested to play a role in suppressing viral replication. We propose that this inefficient and non-specific piRNA-production pathway acts as a primitive sensor for aberrantly abundant transcripts in the germline, potentially seeding piRNA sequence repertoire for more efficient and established biogenesis pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576291">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576291" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576291">
        <p class="paperTitle">Accurate inference of population history in the presence of background selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576291" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576291" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cousins, T.; Tabin, D.; Patterson, N.; Reich, D.; Durvasula, A.</p>
        <p class="info">Score: 34.3, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576291' target='https://doi.org/10.1101/2024.01.18.576291'> 10.1101/2024.01.18.576291</a></p>
        <p class="abstract">1All published methods for learning about demographic history make the simplifying assumption that the genome evolves neutrally, and do not seek to account for the effects of natural selection on patterns of variation. This is a major concern, as ample work has demonstrated the pervasive effects of natural selection and in particular background selection (BGS) on patterns of genetic variation in diverse species. Simulations and theoretical work have shown that methods to infer changes in effective population size over time (Ne(t)) become increasingly inaccurate as the strength of linked selection increases. Here, we introduce an extension to the Pairwise Sequentially Markovian Coalescent (PSMC) algorithm, PSMC&#43;, which explicitly co-models demographic history and natural selection. We benchmark our method using forward-in-time simulations with BGS and find that our approach improves the accuracy of effective population size inference. Leveraging a high resolution map of BGS in humans, we infer considerable changes in the magnitude of inferred effective population size relative to previous reports. Finally, we separately infer Ne(t) on the X chromosome and on the autosomes in diverse great apes without making a correction for selection, and find that the inferred ratio fluctuates substantially through time in a way that differs across species, showing that uncorrected selection may be an important driver of signals of genetic difference on the X chromosome and autosomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577071">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577071" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577071">
        <p class="paperTitle">Somatic epigenetic drift during shoot branching: a cell lineage-based model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577071" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577071" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, Y.; Burian, A.; Johannes, F.</p>
        <p class="info">Score: 1.9, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577071' target='https://doi.org/10.1101/2024.01.24.577071'> 10.1101/2024.01.24.577071</a></p>
        <p class="abstract">Plant architecture is shaped by the continuous production of new organs, most of which emerge post-embryonically. This process includes the formation of new lateral branches along existing shoots. Shoot branching is fundamental to plant development, plant-environment interactions, and vegetative propagation. Current empirical evidence supports a &#39;detached meristem&#39; model as the cellular basis of lateral shoot initiation. In this model, a small number of undifferentiated cells are &#39;sampled&#39; from the periphery of the shoot apical meristem (SAM) to act as precursors for axillary buds, which eventually develop into new shoots. Repeated branching thus creates a series of cellular bottlenecks (i.e. somatic drift) that affect how de novo genetic and epigenetic mutations propagate through the plant body during development. Somatic drift could be particularly relevant for epigenetic changes in the form of stochastic DNA methylation gains and losses (i.e. spontaneous epimutations), as they have been shown to arise rapidly with each cell division. Here, we formalize a special case of the &#39;detached meristem&#39; model, where precursor cells are randomly sampled from the SAM periphery in a way that maximizes cell lineage independence. By following a population of SAM cells through repeated branching processes, we show that somatic drift gives rise to a complex mixture of cellular phylogenies, which shape the evolution of cell-to-cell DNA methylation heterogeneity within the SAM over time. This process is dependent on the number of branch points, the strength of somatic drift as well as the epimutation rate. For many realistic cell biological settings, our model predicts that cell-to-cell DNA methylation heterogeneity in the SAM converges to non-zero states during development, suggesting that epigenetic variation is an inherent property of the SAM cell population. Our insights have direct implications for empirical studies of somatic (epi)genomic diversity in long-lived perennial and clonal species using bulk or single-cell sequencing approaches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.575815">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.575815" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.575815">
        <p class="paperTitle">The neocortical infrastructure for language involves region-specific patterns of laminar gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.575815" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.575815" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wong, M. M. K.; Sha, Z.; Lutje, L.; Kong, X.-Z.; Velthuijs, N.; van Heukelum, S.; van de Berg, W. D. J.; Jonkman, L. E.; Fisher, S. E.; Francks, C.</p>
        <p class="info">Score: 11.4, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.575815' target='https://doi.org/10.1101/2024.01.17.575815'> 10.1101/2024.01.17.575815</a></p>
        <p class="abstract">The language network of the human brain has core components in the inferior frontal cortex and superior/middle temporal cortex, with left-hemisphere dominance in most people. Functional specialization and interconnectivity of these neocortical regions is likely to be reflected in their molecular and cellular profiles. Excitatory connections between cortical regions arise and innervate according to layer-specific patterns. Here we generated a new gene expression dataset from human postmortem cortical tissue samples from core language network regions, using spatial transcriptomics to discriminate gene expression across cortical layers. Integration of these data with existing single-cell expression data identified 56 genes that showed differences in laminar expression profiles between frontal and temporal language cortex together with upregulation in layer II/III and/or layer V/VI excitatory neurons. Based on data from large-scale genome-wide screening in the population, DNA variants within these 56 genes showed set-level associations with inter-individual variation in structural connectivity between left-hemisphere frontal and temporal language cortex, and with predisposition to dyslexia. The axon guidance genes SLIT1 and SLIT2 were consistently implicated. These findings identify region-specific patterns of laminar gene expression as a feature of the brains language network.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577526">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577526" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577526">
        <p class="paperTitle">A critical role for Pol II CTD phosphorylation in heterochromatic gene activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577526" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577526" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kainth, A. S.; Zhang, H.; Gross, D. S.</p>
        <p class="info">Score: 1.5, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577526' target='https://doi.org/10.1101/2024.01.26.577526'> 10.1101/2024.01.26.577526</a></p>
        <p class="abstract">How gene activation works in heterochromatin, and how the mechanism might differ from the one used in euchromatin, has been largely unexplored. Previous work has shown that in SIR-regulated heterochromatin of Saccharomyces cerevisiae, gene activation occurs in the absence of covalent histone modifications and other alterations of chromatin commonly associated with transcription. Here we demonstrate that such activation occurs in a substantial fraction of cells, and this raises the possibility that an alternative activation pathway might be used. We address one such possibility, Pol II CTD phosphorylation, and explore this idea using a natural telomere-linked gene, YFR057w, as a model. Unlike covalent histone modifications, CTD phosphorylation at Ser2, Ser5 and Ser7 is abundant at the drug-induced heterochromatic gene. Particularly enriched relative to the euchromatic state is Ser2 phosphorylation. Consistent with a functional role for Ser2P, YFR057w is negligibly activated in cells deficient in the Ser2 CTD kinases Ctk1 and Bur1 even though the gene is strongly stimulated in ctk1{Delta} bur1 cells when it is placed in a euchromatic context. Collectively, our results are consistent with a critical role for Ser2 CTD phosphorylation in driving Pol II recruitment and transcription of a natural heterochromatic gene - an activity that may help supplant the need for histone epigenetic modifications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.577084">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.577084" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.577084">
        <p class="paperTitle">Evolution, diversity, function, and marker-based elimination of the disease susceptibility gene Snn1 in wheat</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.577084" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.577084" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seneviratne, S.; Shi, G.; Szabo-Hever, A.; Zhang, Z.; Peters Haugrud, A.; Running, K. L. D.; Singh, G.; Nandety, R. S.; Fiedler, J. D.; McClean, P. E.; Xu, S.; Friesen, T. L.; Faris, J. D.</p>
        <p class="info">Score: 1.4, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.577084' target='https://doi.org/10.1101/2024.01.24.577084'> 10.1101/2024.01.24.577084</a></p>
        <p class="abstract">Septoria nodorum blotch (SNB), caused by Parastagonospora nodorum, is a disease of durum and common wheat initiated by the recognition of pathogen-produced necrotrophic effectors (NEs) by specific wheat genes. The wheat gene Snn1 encodes a wall-associated kinase that directly interacts with the NE SnTox1 leading to the development of SNB. Here, sequence analysis of Snn1 from 114 accessions including diploid, tetraploid and hexaploid wheat species revealed that some wheat lines possess two copies of Snn1 (designated Snn1-B1 and Snn1-B2) approximately 120 kb apart. Snn1-B2 evolved relatively recently as a paralog of Snn1-B1, and both genes have undergone diversifying selection. Three point mutations associated with the formation of the first SnTox1-sensitive Snn1-B1 allele from a primitive wild wheat were identified. Four subsequent and independent SNPs, three in Snn1-B1 and one in Snn1-B2, converted the sensitive alleles to insensitive forms. Protein modeling indicated these four mutations could abolish Snn1-SnTox1 compatibility either through destabilization of the Snn1 protein or direct disruption of the protein-protein interaction. High-throughput markers were developed for the causal mutations and evaluated on panels of durum and common wheat. The markers were able to correctly identify 96.9 % of SnTox1-sensitive durum wheat accessions, and a marker for the null allele was 100% accurate at predicting SnTox1-insensitive lines in both durum and spring wheat. Results of this study increase our understanding of the evolution, diversity, and function of Snn1-B1 and Snn1-B2 genes and will be useful for marker-assisted elimination of these genes for better host resistance.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
