<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Interplay between environmental yielding and dynamic forcing regulates bacterial growth
Authors: Hancock, A. M.; Datta, S. S.
Score: 17.9, Published: 2023-12-07 DOI: 10.1101/2023.12.05.569991
Many bacterial habitats---ranging from gels and tissues in the body to cell-secreted exopolysaccharides in biofilms---are rheologically complex, undergo dynamic external forcing, and have unevenly-distributed nutrients. How do these features jointly influence how the resident cells grow and proliferate? Here, we address this question by studying the growth of Escherichia coli dispersed in granular hydrogel matrices with defined and highly-tunable structural and rheological properties, under different amounts of external forcing imposed by mechanical shaking, and in both aerobic and anaerobic conditions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Interplay between environmental yielding and dynamic forcing regulates bacterial growth
Authors: Hancock, A. M.; Datta, S. S.
Score: 17.9, Published: 2023-12-07 DOI: 10.1101/2023.12.05.569991
Many bacterial habitats---ranging from gels and tissues in the body to cell-secreted exopolysaccharides in biofilms---are rheologically complex, undergo dynamic external forcing, and have unevenly-distributed nutrients. How do these features jointly influence how the resident cells grow and proliferate? Here, we address this question by studying the growth of Escherichia coli dispersed in granular hydrogel matrices with defined and highly-tunable structural and rheological properties, under different amounts of external forcing imposed by mechanical shaking, and in both aerobic and anaerobic conditions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-10T10:37:48+00:00" />
<meta property="article:modified_time" content="2023-12-10T10:37:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Interplay between environmental yielding and dynamic forcing regulates bacterial growth
Authors: Hancock, A. M.; Datta, S. S.
Score: 17.9, Published: 2023-12-07 DOI: 10.1101/2023.12.05.569991
Many bacterial habitats---ranging from gels and tissues in the body to cell-secreted exopolysaccharides in biofilms---are rheologically complex, undergo dynamic external forcing, and have unevenly-distributed nutrients. How do these features jointly influence how the resident cells grow and proliferate? Here, we address this question by studying the growth of Escherichia coli dispersed in granular hydrogel matrices with defined and highly-tunable structural and rheological properties, under different amounts of external forcing imposed by mechanical shaking, and in both aerobic and anaerobic conditions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Interplay between environmental yielding and dynamic forcing regulates bacterial growth\nAuthors: Hancock, A. M.; Datta, S. S.\nScore: 17.9, Published: 2023-12-07 DOI: 10.1101/2023.12.05.569991\nMany bacterial habitats---ranging from gels and tissues in the body to cell-secreted exopolysaccharides in biofilms---are rheologically complex, undergo dynamic external forcing, and have unevenly-distributed nutrients. How do these features jointly influence how the resident cells grow and proliferate? Here, we address this question by studying the growth of Escherichia coli dispersed in granular hydrogel matrices with defined and highly-tunable structural and rheological properties, under different amounts of external forcing imposed by mechanical shaking, and in both aerobic and anaerobic conditions.",
  "keywords": [
    
  ],
  "articleBody": " Interplay between environmental yielding and dynamic forcing regulates bacterial growth\nAuthors: Hancock, A. M.; Datta, S. S.\nScore: 17.9, Published: 2023-12-07 DOI: 10.1101/2023.12.05.569991\nMany bacterial habitats---ranging from gels and tissues in the body to cell-secreted exopolysaccharides in biofilms---are rheologically complex, undergo dynamic external forcing, and have unevenly-distributed nutrients. How do these features jointly influence how the resident cells grow and proliferate? Here, we address this question by studying the growth of Escherichia coli dispersed in granular hydrogel matrices with defined and highly-tunable structural and rheological properties, under different amounts of external forcing imposed by mechanical shaking, and in both aerobic and anaerobic conditions. Our experiments establish a general principle: that the balance between the yield stress of the environment that the cells inhabit {sigma}y and the external stress imposed on the environment {sigma} regulates bacterial growth by modulating transport of essential nutrients to the cells. In particular, when {sigma}y\u003c{sigma}, the environment is easily fluidized and mixed over large scales, providing nutrients to the cells and sustaining complete cellular growth. By contrast, when {sigma}y\u003e{sigma}, the elasticity of the environment suppresses large-scale fluid mixing, limiting nutrient availability and arresting cellular growth. Our work thus reveals a new mechanism, beyond effects that change cellular behavior via local forcing, by which the rheology of the environment may regulate microbial physiology in diverse natural and industrial settings.\nEstimation of the mass density of biological matter from refractive index measurements\nAuthors: Moeckel, C.; Beck, T.; Kaliman, S.; Abuhattum, S.; Kim, K.; Kolb, J.; Wehner, D.; Zaburdaev, V.; Guck, J.\nScore: 11.8, Published: 2023-12-05 DOI: 10.1101/2023.12.05.569868\nThe quantification of physical properties of biological matter gives rise to novel ways of understanding functional mechanisms by utilizing models that explicitly depend on physical observables. One of the basic biophysical properties is the mass density (MD), which determines the degree of crowdedness. It impacts the dynamics in sub-cellular compartments and further plays a major role in defining the opto-acoustical properties of cells and tissues. As such, the MD can be connected to the refractive index (RI) via the well known Lorentz-Lorenz relation, which takes into account the polarizability of matter. However, computing the MD based on RI measurements poses a challenge as it requires detailed knowledge of the biochemical composition of the sample. Here we propose a methodology on how to account for a priori and a posteriori assumptions about the biochemical composition of the sample as well as respective RI measurements. To that aim, we employ the Biot mixing rule of RIs alongside the assumption of volume additivity to find an approximate relation of MD and RI. We use Monte-Carlo simulations as well as Gaussian propagation of uncertainty to obtain approximate analytical solutions for the respective uncertainties of MD and RI. We validate this approach by applying it to a set of well characterized complex mixtures given by bovine milk and intralipid emulsion. Further, we employ it to estimate the mass density of trunk tissue of living zebrafish (Danio rerio) larvae. Our results enable quantifying changes of mass density estimates based on variations in the a priori assumptions. This illustrates the importance of implementing this methodology not only for MD estimations but for many other related biophysical problems, such as mechanical measurements using Brillouin microscopy and transient optical coherence elastography.\nPower-law growth models explain incidences and sizes of pancreatic cancer precursor lesions and confirm spatial genomic findings\nAuthors: Kiemen, A. L.; Wu, P.; Braxton, A. M.; Cornish, T. C.; Hruban, R. H.; Wood, L. D.; Wirtz, D.; Zwicker, D.\nScore: 10.3, Published: 2023-12-04 DOI: 10.1101/2023.12.01.569633\nPancreatic ductal adenocarcinoma is a rare but lethal cancer. Recent evidence reveals that pancreatic intraepithelial neoplasms (PanINs), the microscopic precursor lesions in the pancreatic ducts that can give rise to invasive pancreatic cancer, are significantly larger and more prevalent than previously believed. Better understanding of the growth law dynamics of PanINs may improve our ability to understand how a miniscule fraction of these lesions makes the transition to invasive cancer. Here, using artificial intelligence (AI)-based three-dimensional (3D) tissue mapping method, we measured the volumes of \u003e1,000 PanIN and found that lesion size is distributed according to a power law with a fitted exponent of -1.7 over \u003e 3 orders of magnitude. Our data also suggest that PanIN growth is not very sensitive to the pancreatic microenvironment or an individuals age, family history, and lifestyle, and is rather shaped by general growth behavior. We analyze several models of PanIN growth and fit the predicted size distributions to the observed data. The best fitting models suggest that both intraductal spread of PanIN lesions and fusing of multiple lesions into large, highly branched structures drive PanIN growth patterns. This work lays the groundwork for future mathematical modeling efforts integrating PanIN incidence, morphology, genomic, and transcriptomic features to understand pancreas tumorigenesis, and demonstrates the utility of combining experimental measurement of human tissues with dynamic modeling for understanding cancer tumorigenesis.\nCell-type-specific 3D-genome organization and transcription regulation in the brain\nAuthors: Liu, S.; Zheng, P.; Wang, C. Y.; Jia, B. B.; Zemke, N. R.; Ren, B.; Zhuang, X.\nScore: 9.4, Published: 2023-12-05 DOI: 10.1101/2023.12.04.570024\n3D organization of the genome plays a critical role in regulating gene expression. However, it remains unclear how chromatin organization differs among different cell types in the brain. Here we used genome-scale DNA and RNA imaging to investigate 3D-genome organization in transcriptionally distinct cell types in the primary motor cortex of the mouse brain. We uncovered a wide spectrum of differences in the nuclear architecture and 3D-genome organization among different cell types, ranging from the physical size of the cell nucleus to the active-inactive chromatin compartmentalization and radial positioning of chromatin loci within the nucleus. These cell-type-dependent variations in nuclear architecture and chromatin organization exhibited strong correlation with both total transcriptional activity of the cell and transcriptional regulation of cell-type-specific marker genes. Moreover, we found that the methylated-DNA-binding protein MeCP2 regulates transcription in a divergent manner, depending on the nuclear radial positions of chromatin loci, through modulating active-inactive chromatin compartmentalization.\nDeep learning assisted single particle tracking for automated correlation between diffusion and function\nAuthors: Kaestel-Hansen, J.; de Sautu, M.; Saminathan, A.; Scanavachi, G.; Correia, R. F. B. D. C.; Nielsen, A. J.; Bleshoy, S. V.; Boomsma, W.; Kirchhausen, T.; Hatzakis, N. S.\nScore: 12.5, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567393\nSub-cellular diffusion in living systems reflects cellular processes and interactions. Recent advances in optical microscopy allow the tracking of this nanoscale diffusion of individual objects with an unprecedented level of precision. However, the agnostic and automated extraction of functional information from the diffusion of molecules and organelles within the sub-cellular environment, is labor-intensive and poses a significant challenge. Here we introduce DeepSPT, a deep learning framework to interpret the diffusional 2D or 3D temporal behavior of objects in a rapid and efficient manner, agnostically. Demonstrating its versatility, we have applied DeepSPT to automated mapping of the early events of viral infections, identifying distinct types of endosomal organelles, and clathrin-coated pits and vesicles with up to 95% accuracy and within seconds instead of weeks. The fact that DeepSPT effectively extracts biological information from diffusion alone indicates that besides structure, motion encodes function at the molecular and subcellular level.\nSingle-Molecule FRET at 10 MHz Count Rates\nAuthors: Grabenhorst, L.; Sturzenegger, F.; Hasler, M.; Schuler, B.; Tinnefeld, P.\nScore: 7.9, Published: 2023-12-08 DOI: 10.1101/2023.12.08.570755\nA bottleneck in many studies utilizing single-molecule Forster Resonance Energy Transfer (smFRET) is the attainable photon count rate as it determines the temporal resolution of the experiment. As many biologically relevant processes occur on timescales that are hardly accessible with currently achievable photon count rates, there has been considerable effort to find strategies to increase the stability and brightness of fluorescent dyes. Here, we use DNA nanoantennas to drastically increase the achievable photon count rates and to observe fast biomolecular dynamics in the small volume between two plasmonic nanoparticles. As a proof of concept, we observe the coupled folding and binding of two intrinsically disordered proteins which form transient encounter complexes with lifetimes on the order of 100 us. To test the limits of our approach, we also investigated the hybridization of a short single-stranded DNA to its complementary counterpart, revealing a transition path time of 17 us at photon count rates of around 10 MHz, which is an order-of-magnitude improvement when compared to the state of the art. Concomitantly, the photostability was increased, enabling many seconds long megahertz fluorescence time traces. Due to the modular nature of the DNA origami method, this platform can be adapted to a broad range of biomolecules, providing a promising approach to study previously unobservable ultrafast biophysical processes.\nA Structured Illumination Microscopy Framework with Spatial-Domain Noise Propagation.\nAuthors: Saurabh, A.; Brown, P. T.; Bryan, J. S.; Fox, Z. R.; Kruithoff, R.; Shepherd, D. P.; Presse, S. P.\nScore: 5.0, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570701\nBiological images captured by a microscope are characterized by heterogeneous signal-to-noise ratios (SNRs) across the field of view due to spatially varying photon emission and camera noise. State-of-the-art unsupervised structured illumination microscopy (SIM) reconstruction algorithms, commonly implemented in the Fourier domain, do not accurately model this noise and suffer from high-frequency artifacts, user-dependent choices of smoothness constraints making assumptions on biological features, and unphysical negative values in the recovered fluorescence intensity map. On the other hand, supervised methods rely on large datasets for training, and often require retraining for new sample structures. Consequently, achieving high contrast near the maximum theoretical resolution in an unsupervised, physically principled manner remains a challenging task. Here, we propose Bayesian-SIM (B-SIM), an unsupervised Bayesian framework to quantitatively reconstruct SIM data, rectifying these shortcomings by accurately incorporating all noise sources in the spatial domain. To accelerate the reconstruction process for computational feasibility, we devise a parallelized Monte Carlo sampling strategy for inference. We benchmark our framework on both simulated and experimental images, and demonstrate improved contrast permitting feature recovery at up to 25% shorter length scales over state-of-the-art methods at both high- and low-SNR. B-SIM enables unsupervised, quantitative, physically accurate reconstruction without the need for labeled training data, democratizing high-quality SIM reconstruction and expands the capabilities of live-cell SIM to lower SNR, potentially revealing biological features in previously inaccessible regimes.\nDeePNAP: A deep learning method to predict protein-nucleic acids binding affinity from sequence\nAuthors: Pandey, U.; Behara, S. M.; Sharma, S.; Patil, R. S.; Nambiar, S.; Koner, D.; Bhukya, H.\nScore: 4.1, Published: 2023-12-05 DOI: 10.1101/2023.12.03.569768\nPredicting the protein-nucleic acid (PNA) binding affinity solely from their sequences is of paramount importance for the experimental design and analysis of PNA interactions (PNAIs). A large number of currently developed models for binding affinity prediction are limited to specific PNAIs, while also relying on both sequence and structural information of the PNA complexes for both train/test and also as inputs. As PNA complex structures available are scarce, this significantly limits the diversity and generalizability due to a small training dataset. Additionally, a majority of the tools predict a single parameter such as binding affinity or free energy changes upon mutations, rendering a model less versatile for usage. Hence, we propose DeePNAP, a machine learning-based model trained on a vast and heterogeneous dataset with 14,401 entries (from both eukaryotes and prokaryotes) of ProNAB database, consisting of wild-type and mutant PNA complex binding parameters. Our model precisely predicts the binding affinity and free energy changes due to the mutation(s) of PNAIs exclusively from the sequences. While other similar tools extract features from both sequence and structure information, DeePNAP employs sequence-based features to yield high correlation coefficients between the predicted and experimental values with low root mean squared errors for PNA complexes in predicting the KD and {Delta}{Delta}G implying the generalizability of DeePNAP. Additionally, we have also developed a web interface hosting DeePNAP that can serve as a powerful tool to rapidly predict binding affinities for a myriad of PNAIs with high precision toward developing a deeper understanding of their implications in various biological systems. Web interface: http://14.139.174.41:8080/\nUnderstanding how genetically encoded tags affect phase separation by Heterochromatin Protein HP1α\nAuthors: Zhou, Z.; Narlikar, G. J.\nScore: 3.8, Published: 2023-12-05 DOI: 10.1101/2023.12.04.569983\nLiquid-liquid phase separation (LLPS) is driven by weak multi-valent interactions. Such interactions can result in the formation of puncta in cells and droplets in vitro. The heterochromatin protein HP1 forms droplets with chromatin in vitro and is found in puncta in cells. A common approach to visualize the dynamics of HP1 in cells is to genetically encode fluorescent tags on the protein. HP1 modified with tags such as GFP has been shown to localize to dynamic puncta in vivo. However, whether tagged HP1 retains its intrinsic phase separation properties has not been systematically studied. Here, using different C-terminal tags (AID-sfGFP, mEGFP, and UnaG), we assessed how tag size and linker length affected the phase separation ability of HP1 with DNA in vitro. We found that the AID-sfGFP tag (52 kDa) promoted HP1 phase-separation, possibly driven by the highly disordered AID degron. The mEGFP tag (27 kDa) inhibited phase-separation by HP1, whereas an UnaG tag (13 kDa) with a 16 amino acid linker showed minimal perturbation. The UnaG tag can thus be used in cellular studies of HP1 to better correlate in vitro and in vivo studies. To test if cellular crowding overcomes the negative effects of large tags in vivo, we used polyethylene glycol (PEG) to mimic crowding in vitro. We found that addition of 10% PEG8000 or PEG4000 enables phase separation by GFP-tagged HP1 at comparable concentrations as untagged HP1. However, these crowding agents also substantially dampened the differences in phase-separation between wild-type and mutant HP1 proteins. PEG further drove phase-separation of Maltose Binding Protein (MBP), a tag often used to solubilize other proteins. These results suggest that phase-separation of biological macromolecules with PEG should be interpreted with caution as PEG-based crowding agents may change the types of interactions made within the phases.\nGotGlycans: Role of N343 Glycosylation on the SARS-CoV-2 S RBD Structure and Co-Receptor Binding Across Variants of Concern\nAuthors: Ives, C. M.; Nguyen, L.; Fogarty, C. A.; Harbison, A. M.; Durocher, Y.; Klassen, J. S.; fadda, e.\nScore: 3.6, Published: 2023-12-05 DOI: 10.1101/2023.12.05.570076\nGlycosylation of the SARS-CoV-2 spike (S) protein represents a key target for viral evolution because it affects both viral evasion and fitness. Successful variations in the glycan shield are difficult to achieve though, as protein glycosylation is also critical to folding and to structural stability. Within this framework, the identification of glycosylation sites that are structurally dispensable can provide insight into the evolutionary mechanisms of the shield and inform immune surveillance. In this work we show through over 45 s of cumulative sampling from conventional and enhanced molecular dynamics (MD) simulations, how the structure of the immunodominant S receptor binding domain (RBD) is regulated by N-glycosylation at N343 and how this glycans structural role changes from WHu-1, alpha (B.1.1.7), and beta (B.1.351), to the delta (B.1.617.2) and omicron (BA.1 and BA.2.86) variants. More specifically, we find that the amphipathic nature of the N-glycan is instrumental to preserve the structural integrity of the RBD hydrophobic core and that loss of glycosylation at N343 triggers a specific and consistent conformational change. We show how this change allosterically regulates the conformation of the receptor binding motif (RBM) in the WHu-1, alpha and beta RBDs, but not in the delta and omicron variants, due to mutations that reinforce the RBD architecture. In support of these findings, we show that the binding of the RBD to monosialylated ganglioside co-receptors is highly dependent on N343 glycosylation in the WHu-1, but not in the delta RBD, and that affinity changes significantly across VoCs. Ultimately, the molecular and functional insight we provide in this work reinforces our understanding of the role of glycosylation in protein structure and function and it also allows us to identify the structural constraints within which the glycosylation site at N343 can become a hotspot for mutations in the SARS-CoV-2 S glycan shield.\n",
  "wordCount" : "2638",
  "inLanguage": "en",
  "datePublished": "2023-12-10T10:37:48Z",
  "dateModified": "2023-12-10T10:37:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 10, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.569991">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.569991" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.569991">
        <p class="paperTitle">Interplay between environmental yielding and dynamic forcing regulates bacterial growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.569991" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.569991" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hancock, A. M.; Datta, S. S.</p>
        <p class="info">Score: 17.9, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.569991' target='https://doi.org/10.1101/2023.12.05.569991'> 10.1101/2023.12.05.569991</a></p>
        <p class="abstract">Many bacterial habitats---ranging from gels and tissues in the body to cell-secreted exopolysaccharides in biofilms---are rheologically complex, undergo dynamic external forcing, and have unevenly-distributed nutrients. How do these features jointly influence how the resident cells grow and proliferate? Here, we address this question by studying the growth of Escherichia coli dispersed in granular hydrogel matrices with defined and highly-tunable structural and rheological properties, under different amounts of external forcing imposed by mechanical shaking, and in both aerobic and anaerobic conditions. Our experiments establish a general principle: that the balance between the yield stress of the environment that the cells inhabit {sigma}y and the external stress imposed on the environment {sigma} regulates bacterial growth by modulating transport of essential nutrients to the cells. In particular, when {sigma}y&lt;{sigma}, the environment is easily fluidized and mixed over large scales, providing nutrients to the cells and sustaining complete cellular growth. By contrast, when {sigma}y&gt;{sigma}, the elasticity of the environment suppresses large-scale fluid mixing, limiting nutrient availability and arresting cellular growth. Our work thus reveals a new mechanism, beyond effects that change cellular behavior via local forcing, by which the rheology of the environment may regulate microbial physiology in diverse natural and industrial settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.569868">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.569868" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.569868">
        <p class="paperTitle">Estimation of the mass density of biological matter from refractive index measurements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.569868" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.569868" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moeckel, C.; Beck, T.; Kaliman, S.; Abuhattum, S.; Kim, K.; Kolb, J.; Wehner, D.; Zaburdaev, V.; Guck, J.</p>
        <p class="info">Score: 11.8, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.569868' target='https://doi.org/10.1101/2023.12.05.569868'> 10.1101/2023.12.05.569868</a></p>
        <p class="abstract">The quantification of physical properties of biological matter gives rise to novel ways of understanding functional mechanisms by utilizing models that explicitly depend on physical observables. One of the basic biophysical properties is the mass density (MD), which determines the degree of crowdedness. It impacts the dynamics in sub-cellular compartments and further plays a major role in defining the opto-acoustical properties of cells and tissues. As such, the MD can be connected to the refractive index (RI) via the well known Lorentz-Lorenz relation, which takes into account the polarizability of matter. However, computing the MD based on RI measurements poses a challenge as it requires detailed knowledge of the biochemical composition of the sample. Here we propose a methodology on how to account for a priori and a posteriori assumptions about the biochemical composition of the sample as well as respective RI measurements. To that aim, we employ the Biot mixing rule of RIs alongside the assumption of volume additivity to find an approximate relation of MD and RI. We use Monte-Carlo simulations as well as Gaussian propagation of uncertainty to obtain approximate analytical solutions for the respective uncertainties of MD and RI. We validate this approach by applying it to a set of well characterized complex mixtures given by bovine milk and intralipid emulsion. Further, we employ it to estimate the mass density of trunk tissue of living zebrafish (Danio rerio) larvae. Our results enable quantifying changes of mass density estimates based on variations in the a priori assumptions. This illustrates the importance of implementing this methodology not only for MD estimations but for many other related biophysical problems, such as mechanical measurements using Brillouin microscopy and transient optical coherence elastography.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569633">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569633" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569633">
        <p class="paperTitle">Power-law growth models explain incidences and sizes of pancreatic cancer precursor lesions and confirm spatial genomic findings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569633" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569633" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kiemen, A. L.; Wu, P.; Braxton, A. M.; Cornish, T. C.; Hruban, R. H.; Wood, L. D.; Wirtz, D.; Zwicker, D.</p>
        <p class="info">Score: 10.3, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569633' target='https://doi.org/10.1101/2023.12.01.569633'> 10.1101/2023.12.01.569633</a></p>
        <p class="abstract">Pancreatic ductal adenocarcinoma is a rare but lethal cancer. Recent evidence reveals that pancreatic intraepithelial neoplasms (PanINs), the microscopic precursor lesions in the pancreatic ducts that can give rise to invasive pancreatic cancer, are significantly larger and more prevalent than previously believed. Better understanding of the growth law dynamics of PanINs may improve our ability to understand how a miniscule fraction of these lesions makes the transition to invasive cancer. Here, using artificial intelligence (AI)-based three-dimensional (3D) tissue mapping method, we measured the volumes of &gt;1,000 PanIN and found that lesion size is distributed according to a power law with a fitted exponent of -1.7 over &gt; 3 orders of magnitude. Our data also suggest that PanIN growth is not very sensitive to the pancreatic microenvironment or an individuals age, family history, and lifestyle, and is rather shaped by general growth behavior. We analyze several models of PanIN growth and fit the predicted size distributions to the observed data. The best fitting models suggest that both intraductal spread of PanIN lesions and fusing of multiple lesions into large, highly branched structures drive PanIN growth patterns. This work lays the groundwork for future mathematical modeling efforts integrating PanIN incidence, morphology, genomic, and transcriptomic features to understand pancreas tumorigenesis, and demonstrates the utility of combining experimental measurement of human tissues with dynamic modeling for understanding cancer tumorigenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.570024">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.570024" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.570024">
        <p class="paperTitle">Cell-type-specific 3D-genome organization and transcription regulation in the brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.570024" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.570024" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, S.; Zheng, P.; Wang, C. Y.; Jia, B. B.; Zemke, N. R.; Ren, B.; Zhuang, X.</p>
        <p class="info">Score: 9.4, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.570024' target='https://doi.org/10.1101/2023.12.04.570024'> 10.1101/2023.12.04.570024</a></p>
        <p class="abstract">3D organization of the genome plays a critical role in regulating gene expression. However, it remains unclear how chromatin organization differs among different cell types in the brain. Here we used genome-scale DNA and RNA imaging to investigate 3D-genome organization in transcriptionally distinct cell types in the primary motor cortex of the mouse brain. We uncovered a wide spectrum of differences in the nuclear architecture and 3D-genome organization among different cell types, ranging from the physical size of the cell nucleus to the active-inactive chromatin compartmentalization and radial positioning of chromatin loci within the nucleus. These cell-type-dependent variations in nuclear architecture and chromatin organization exhibited strong correlation with both total transcriptional activity of the cell and transcriptional regulation of cell-type-specific marker genes. Moreover, we found that the methylated-DNA-binding protein MeCP2 regulates transcription in a divergent manner, depending on the nuclear radial positions of chromatin loci, through modulating active-inactive chromatin compartmentalization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567393">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567393" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567393">
        <p class="paperTitle">Deep learning assisted single particle tracking for automated correlation between diffusion and function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567393" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567393" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaestel-Hansen, J.; de Sautu, M.; Saminathan, A.; Scanavachi, G.; Correia, R. F. B. D. C.; Nielsen, A. J.; Bleshoy, S. V.; Boomsma, W.; Kirchhausen, T.; Hatzakis, N. S.</p>
        <p class="info">Score: 12.5, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567393' target='https://doi.org/10.1101/2023.11.16.567393'> 10.1101/2023.11.16.567393</a></p>
        <p class="abstract">Sub-cellular diffusion in living systems reflects cellular processes and interactions. Recent advances in optical microscopy allow the tracking of this nanoscale diffusion of individual objects with an unprecedented level of precision. However, the agnostic and automated extraction of functional information from the diffusion of molecules and organelles within the sub-cellular environment, is labor-intensive and poses a significant challenge. Here we introduce DeepSPT, a deep learning framework to interpret the diffusional 2D or 3D temporal behavior of objects in a rapid and efficient manner, agnostically. Demonstrating its versatility, we have applied DeepSPT to automated mapping of the early events of viral infections, identifying distinct types of endosomal organelles, and clathrin-coated pits and vesicles with up to 95% accuracy and within seconds instead of weeks. The fact that DeepSPT effectively extracts biological information from diffusion alone indicates that besides structure, motion encodes function at the molecular and subcellular level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570755">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570755" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570755">
        <p class="paperTitle">Single-Molecule FRET at 10 MHz Count Rates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570755" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570755" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grabenhorst, L.; Sturzenegger, F.; Hasler, M.; Schuler, B.; Tinnefeld, P.</p>
        <p class="info">Score: 7.9, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570755' target='https://doi.org/10.1101/2023.12.08.570755'> 10.1101/2023.12.08.570755</a></p>
        <p class="abstract">A bottleneck in many studies utilizing single-molecule Forster Resonance Energy Transfer (smFRET) is the attainable photon count rate as it determines the temporal resolution of the experiment. As many biologically relevant processes occur on timescales that are hardly accessible with currently achievable photon count rates, there has been considerable effort to find strategies to increase the stability and brightness of fluorescent dyes. Here, we use DNA nanoantennas to drastically increase the achievable photon count rates and to observe fast biomolecular dynamics in the small volume between two plasmonic nanoparticles. As a proof of concept, we observe the coupled folding and binding of two intrinsically disordered proteins which form transient encounter complexes with lifetimes on the order of 100 us. To test the limits of our approach, we also investigated the hybridization of a short single-stranded DNA to its complementary counterpart, revealing a transition path time of 17 us at photon count rates of around 10 MHz, which is an order-of-magnitude improvement when compared to the state of the art. Concomitantly, the photostability was increased, enabling many seconds long megahertz fluorescence time traces. Due to the modular nature of the DNA origami method, this platform can be adapted to a broad range of biomolecules, providing a promising approach to study previously unobservable ultrafast biophysical processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570701">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570701" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570701">
        <p class="paperTitle">A Structured Illumination Microscopy Framework with Spatial-Domain Noise Propagation.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570701" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570701" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saurabh, A.; Brown, P. T.; Bryan, J. S.; Fox, Z. R.; Kruithoff, R.; Shepherd, D. P.; Presse, S. P.</p>
        <p class="info">Score: 5.0, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570701' target='https://doi.org/10.1101/2023.12.07.570701'> 10.1101/2023.12.07.570701</a></p>
        <p class="abstract">Biological images captured by a microscope are characterized by heterogeneous signal-to-noise ratios (SNRs) across the field of view due to spatially varying photon emission and camera noise. State-of-the-art unsupervised structured illumination microscopy (SIM) reconstruction algorithms, commonly implemented in the Fourier domain, do not accurately model this noise and suffer from high-frequency artifacts, user-dependent choices of smoothness constraints making assumptions on biological features, and unphysical negative values in the recovered fluorescence intensity map. On the other hand, supervised methods rely on large datasets for training, and often require retraining for new sample structures. Consequently, achieving high contrast near the maximum theoretical resolution in an unsupervised, physically principled manner remains a challenging task. Here, we propose Bayesian-SIM (B-SIM), an unsupervised Bayesian framework to quantitatively reconstruct SIM data, rectifying these shortcomings by accurately incorporating all noise sources in the spatial domain. To accelerate the reconstruction process for computational feasibility, we devise a parallelized Monte Carlo sampling strategy for inference. We benchmark our framework on both simulated and experimental images, and demonstrate improved contrast permitting feature recovery at up to 25% shorter length scales over state-of-the-art methods at both high- and low-SNR. B-SIM enables unsupervised, quantitative, physically accurate reconstruction without the need for labeled training data, democratizing high-quality SIM reconstruction and expands the capabilities of live-cell SIM to lower SNR, potentially revealing biological features in previously inaccessible regimes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569768">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569768" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569768">
        <p class="paperTitle">DeePNAP: A deep learning method to predict protein-nucleic acids binding affinity from sequence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569768" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569768" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pandey, U.; Behara, S. M.; Sharma, S.; Patil, R. S.; Nambiar, S.; Koner, D.; Bhukya, H.</p>
        <p class="info">Score: 4.1, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569768' target='https://doi.org/10.1101/2023.12.03.569768'> 10.1101/2023.12.03.569768</a></p>
        <p class="abstract">Predicting the protein-nucleic acid (PNA) binding affinity solely from their sequences is of paramount importance for the experimental design and analysis of PNA interactions (PNAIs). A large number of currently developed models for binding affinity prediction are limited to specific PNAIs, while also relying on both sequence and structural information of the PNA complexes for both train/test and also as inputs. As PNA complex structures available are scarce, this significantly limits the diversity and generalizability due to a small training dataset. Additionally, a majority of the tools predict a single parameter such as binding affinity or free energy changes upon mutations, rendering a model less versatile for usage. Hence, we propose DeePNAP, a machine learning-based model trained on a vast and heterogeneous dataset with 14,401 entries (from both eukaryotes and prokaryotes) of ProNAB database, consisting of wild-type and mutant PNA complex binding parameters. Our model precisely predicts the binding affinity and free energy changes due to the mutation(s) of PNAIs exclusively from the sequences. While other similar tools extract features from both sequence and structure information, DeePNAP employs sequence-based features to yield high correlation coefficients between the predicted and experimental values with low root mean squared errors for PNA complexes in predicting the KD and {Delta}{Delta}G implying the generalizability of DeePNAP. Additionally, we have also developed a web interface hosting DeePNAP that can serve as a powerful tool to rapidly predict binding affinities for a myriad of PNAIs with high precision toward developing a deeper understanding of their implications in various biological systems. Web interface: http://14.139.174.41:8080/</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569983">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569983" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569983">
        <p class="paperTitle">Understanding how genetically encoded tags affect phase separation by Heterochromatin Protein HP1α</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569983" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569983" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, Z.; Narlikar, G. J.</p>
        <p class="info">Score: 3.8, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569983' target='https://doi.org/10.1101/2023.12.04.569983'> 10.1101/2023.12.04.569983</a></p>
        <p class="abstract">Liquid-liquid phase separation (LLPS) is driven by weak multi-valent interactions. Such interactions can result in the formation of puncta in cells and droplets in vitro. The heterochromatin protein HP1 forms droplets with chromatin in vitro and is found in puncta in cells. A common approach to visualize the dynamics of HP1 in cells is to genetically encode fluorescent tags on the protein. HP1 modified with tags such as GFP has been shown to localize to dynamic puncta in vivo. However, whether tagged HP1 retains its intrinsic phase separation properties has not been systematically studied. Here, using different C-terminal tags (AID-sfGFP, mEGFP, and UnaG), we assessed how tag size and linker length affected the phase separation ability of HP1 with DNA in vitro. We found that the AID-sfGFP tag (52 kDa) promoted HP1 phase-separation, possibly driven by the highly disordered AID degron. The mEGFP tag (27 kDa) inhibited phase-separation by HP1, whereas an UnaG tag (13 kDa) with a 16 amino acid linker showed minimal perturbation. The UnaG tag can thus be used in cellular studies of HP1 to better correlate in vitro and in vivo studies. To test if cellular crowding overcomes the negative effects of large tags in vivo, we used polyethylene glycol (PEG) to mimic crowding in vitro. We found that addition of 10% PEG8000 or PEG4000 enables phase separation by GFP-tagged HP1 at comparable concentrations as untagged HP1. However, these crowding agents also substantially dampened the differences in phase-separation between wild-type and mutant HP1 proteins. PEG further drove phase-separation of Maltose Binding Protein (MBP), a tag often used to solubilize other proteins. These results suggest that phase-separation of biological macromolecules with PEG should be interpreted with caution as PEG-based crowding agents may change the types of interactions made within the phases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.570076">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.570076" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.570076">
        <p class="paperTitle">GotGlycans: Role of N343 Glycosylation on the SARS-CoV-2 S RBD Structure and Co-Receptor Binding Across Variants of Concern</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.570076" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.570076" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ives, C. M.; Nguyen, L.; Fogarty, C. A.; Harbison, A. M.; Durocher, Y.; Klassen, J. S.; fadda, e.</p>
        <p class="info">Score: 3.6, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.570076' target='https://doi.org/10.1101/2023.12.05.570076'> 10.1101/2023.12.05.570076</a></p>
        <p class="abstract">Glycosylation of the SARS-CoV-2 spike (S) protein represents a key target for viral evolution because it affects both viral evasion and fitness. Successful variations in the glycan shield are difficult to achieve though, as protein glycosylation is also critical to folding and to structural stability. Within this framework, the identification of glycosylation sites that are structurally dispensable can provide insight into the evolutionary mechanisms of the shield and inform immune surveillance. In this work we show through over 45 s of cumulative sampling from conventional and enhanced molecular dynamics (MD) simulations, how the structure of the immunodominant S receptor binding domain (RBD) is regulated by N-glycosylation at N343 and how this glycans structural role changes from WHu-1, alpha (B.1.1.7), and beta (B.1.351), to the delta (B.1.617.2) and omicron (BA.1 and BA.2.86) variants. More specifically, we find that the amphipathic nature of the N-glycan is instrumental to preserve the structural integrity of the RBD hydrophobic core and that loss of glycosylation at N343 triggers a specific and consistent conformational change. We show how this change allosterically regulates the conformation of the receptor binding motif (RBM) in the WHu-1, alpha and beta RBDs, but not in the delta and omicron variants, due to mutations that reinforce the RBD architecture. In support of these findings, we show that the binding of the RBD to monosialylated ganglioside co-receptors is highly dependent on N343 glycosylation in the WHu-1, but not in the delta RBD, and that affinity changes significantly across VoCs. Ultimately, the molecular and functional insight we provide in this work reinforces our understanding of the role of glycosylation in protein structure and function and it also allows us to identify the structural constraints within which the glycosylation site at N343 can become a hotspot for mutations in the SARS-CoV-2 S glycan shield.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
