<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity
Authors: Motone, K.; Kontogiorgos-Heintz, D.; Wee, J.; Kurihara, K.; Yang, S.; Roote, G.; Fang, Y.; Cardozo, N.; Nivala, J.
Score: 164.5, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563182
The ability to sequence single protein molecules in their native, full-length form would enable a more comprehensive understanding of proteomic diversity. Current technologies, however, are limited in achieving this goal. Here, we establish a method for long-range, single-molecule reading of intact protein strands on a commercial nanopore sensor array.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity
Authors: Motone, K.; Kontogiorgos-Heintz, D.; Wee, J.; Kurihara, K.; Yang, S.; Roote, G.; Fang, Y.; Cardozo, N.; Nivala, J.
Score: 164.5, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563182
The ability to sequence single protein molecules in their native, full-length form would enable a more comprehensive understanding of proteomic diversity. Current technologies, however, are limited in achieving this goal. Here, we establish a method for long-range, single-molecule reading of intact protein strands on a commercial nanopore sensor array." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-25T10:39:04+00:00" />
<meta property="article:modified_time" content="2023-10-25T10:39:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity
Authors: Motone, K.; Kontogiorgos-Heintz, D.; Wee, J.; Kurihara, K.; Yang, S.; Roote, G.; Fang, Y.; Cardozo, N.; Nivala, J.
Score: 164.5, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563182
The ability to sequence single protein molecules in their native, full-length form would enable a more comprehensive understanding of proteomic diversity. Current technologies, however, are limited in achieving this goal. Here, we establish a method for long-range, single-molecule reading of intact protein strands on a commercial nanopore sensor array."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity\nAuthors: Motone, K.; Kontogiorgos-Heintz, D.; Wee, J.; Kurihara, K.; Yang, S.; Roote, G.; Fang, Y.; Cardozo, N.; Nivala, J.\nScore: 164.5, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563182\nThe ability to sequence single protein molecules in their native, full-length form would enable a more comprehensive understanding of proteomic diversity. Current technologies, however, are limited in achieving this goal. Here, we establish a method for long-range, single-molecule reading of intact protein strands on a commercial nanopore sensor array.",
  "keywords": [
    
  ],
  "articleBody": " Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity\nAuthors: Motone, K.; Kontogiorgos-Heintz, D.; Wee, J.; Kurihara, K.; Yang, S.; Roote, G.; Fang, Y.; Cardozo, N.; Nivala, J.\nScore: 164.5, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563182\nThe ability to sequence single protein molecules in their native, full-length form would enable a more comprehensive understanding of proteomic diversity. Current technologies, however, are limited in achieving this goal. Here, we establish a method for long-range, single-molecule reading of intact protein strands on a commercial nanopore sensor array. By using the ClpX unfoldase to ratchet proteins through a CsgG nanopore, we achieve single-amino acid level sensitivity, enabling sequencing of combinations of amino acid substitutions across long protein strands. For greater sequencing accuracy, we demonstrate the ability to reread individual protein molecules, spanning hundreds of amino acids in length, multiple times, and explore the potential for high accuracy protein barcode sequencing. Further, we develop a biophysical model that can simulate raw nanopore signals a priori, based on amino acid volume and charge, enhancing the interpretation of raw signal data. Finally, we apply these methods to examine intact, folded protein domains for complete end-to-end analysis. These results provide proof-of-concept for a platform that has the potential to identify and characterize full-length proteoforms at single-molecule resolution.\nData-driven regularisation lowers the size barrier of cryo-EM structure determination\nAuthors: Kimanius, D.; Jamali, K.; Wilkinson, M. E.; LoÌˆvestam, S.; Velazhahan, V.; Nakane, T.; Scheres, S. H.\nScore: 78.2, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563586\nMacromolecular structure determination by electron cryo-microscopy (cryo-EM) is limited by the alignment of noisy images of individual particles. Because smaller particles have weaker signals, alignment errors impose size limitations on its applicability. Here, we explore how image alignment is improved by the application of deep-learning to exploit prior knowledge about biological macromolecular structures that would otherwise be difficult to express mathematically. We train a denoising convolutional neural network on pairs of half-set reconstructions from the electron microscopy data bank (EMDB) and use this denoiser as an alternative to a commonly used smoothness prior. We demonstrate that this approach, which we call Blush regularisation, yields better reconstructions than existing algorithms, in particular for data with low signal-to-noise ratios. The reconstruction of a protein-nucleic acid complex with a molecular weight of 40 kDa, which was previously intractable, illustrates that regularisation through denoising will expand the applicability of cryo-EM structure determination for a wide range of biological macromolecules.\nDynaMight: estimating molecular motions with improved reconstruction from cryo-EM images\nAuthors: Schwab, J.; Kimanius, D.; Burt, A.; Dendooven, T.; Scheres, S.\nScore: 46.4, Published: 2023-10-19 DOI: 10.1101/2023.10.18.562877\nHow to deal with continuously flexing molecules is one of the biggest outstanding challenges in single-particle analysis of proteins from cryo-EM images. Here, we present DynaMight, a new software tool that estimates a continuous space of conformations in a cryo-EM data set by learning 3D deformations of a Gaussian pseudo-atomic model of a consensus structure for every particle image. Inversion of the learnt deformations is then used to obtain an improved reconstruction of the consensus structure. We illustrate the performance of DynaMight for several experimental cryo-EM data sets. We also show how error estimates on the deformations may be obtained by independently training two variational autoencoders (VAEs) on half sets of the cryo-EM data, and how regularisation of the 3D deformations through the use of atomic models may lead to important artefacts due to model bias. DynaMight is distributed as free, open-source software, as part of RELION-5.\nStructural and mechanistic insights into Streptococcus pneumoniae NADPH oxidase\nAuthors: Dubach, V. R.; San Segundo-Acosta, P.; Murphy, B. J.\nScore: 27.4, Published: 2023-10-19 DOI: 10.1101/2023.10.17.562464\nNADPH oxidases (NOXs) play a major role in the physiology of eukaryotic cells by mediating the production of reactive oxygen species (ROS). Evolutionarily distant proteins sharing the NOX catalytic core have been recently described in Bacteria. Among them, the Streptococcus pneumoniae NOX (SpNOX) has been proposed as a model for the study of NOXs due to its high activity and stability in detergent micelles. Here, we report high-resolution cryo-EM structures of substrate-free and stably reduced NADH-bound SpNOX, and of the NADPH-bound SpNOX and a Phe397Ala mutant under turnover conditions. In combination with structure-guided mutagenesis and biochemical analyses, we provide the structural basis for constitutive activity, the lack of substrate specificity towards NADPH and the electron transfer pathway. Additionally, we shed light on the catalytic regulation by the C-terminal tail residue Phe397 and the potential in vivo function of this protein.\nInterfacial morphodynamics of proliferating microbial communities\nAuthors: Martinez-Calvo, A.; Trenado-Yuste, C.; Lee, H.; Gore, J.; Wingreen, N. S.; Datta, S. S.\nScore: 22.8, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563665\nIn microbial communities, various cell types often coexist by occupying distinct spatial domains. What determines the shape of the interface between such domains---which in turn influences the interactions between cells and overall community function? Here, we address this question by developing a continuum model of a 2D spatially-structured microbial community with two distinct cell types. We find that, depending on the balance of the different cell proliferation rates and substrate friction coefficients, the interface between domains is either stable and smooth, or unstable and develops finger-like protrusions. We establish quantitative principles describing when these different interfacial behaviors arise, and find good agreement both with the results of previous experimental reports as well as new experiments performed here. Our work thus helps to provide a biophysical basis for understanding the interfacial morphodynamics of proliferating microbial communities, as well as a broader range of proliferating active systems.\nAutomated 3D multi-color single-molecule localization microscopy\nAuthors: Power, R. M.; Tschanz, A.; Zimmermann, T.; Ries, J.\nScore: 17.9, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563122\nSince its inception, single molecule localization microscopy (SMLM) has enabled imaging scientists to visualize biological structures with unprecedented resolution. Particularly powerful implementations capable of 3D, multi-color and high-throughput imaging have yielded key biological insights although widespread access to such technologies has been limited. The purpose of this protocol is to provide a guide for interested researchers to establish high-end SMLM in their laboratories. We detail the initial configuration and subsequent assembly of the SMLM, including instructions for alignment of all optical pathways, software/hardware integration and operation of the instrument. We describe validation steps including the preparation and imaging of test- and biological samples with structures of well-defined geometry and assist the user in troubleshooting and benchmarking performance. Additionally, we provide a walkthrough of the reconstruction of a super-resolved dataset from acquired raw images using the Super-resolution Microscopy Analysis Platform (SMAP). Depending on the instrument configuration, the cost of components is in the range $80,000 - 160,000, a fraction of the cost of a commercial instrument. A builder with some experience of optical systems is expected to require 3 - 6 months from the start of system construction to attain high-quality 3D and multi-color biological images.\nDevelopment of covalent chemogenetic K2P channel activators\nAuthors: Deal, P. E.; Lee, H.; Mondal, A.; Lolicato, M.; Furtado de Mandonca, P. R.; Black, H.; El-Hilali, X.; Bryant, C.; Isacoff, E. Y.; Renslo, A.; Minor, D. L.\nScore: 14.2, Published: 2023-10-18 DOI: 10.1101/2023.10.15.561774\nK2P potassium channels regulate excitability by affecting cellular resting membrane potential in the brain, cardiovascular system, immune cells, and sensory organs. Despite their important roles in anesthesia, arrhythmia, pain, hypertension, sleep, and migraine, the ability to control K2P function remains limited. Here, we describe a chemogenetic strategy termed CATKLAMP (Covalent Activation of TREK family K+ channels to cLAmp Membrane Potential) that leverages the discovery of a site in the K2P modulator pocket that reacts with electrophile-bearing derivatives of a TREK subfamily small molecule activator, ML335, to activate the channel irreversibly. We show that the CATKLAMP strategy can be used to probe fundamental aspects of K2P function, as a switch to silence neuronal firing, and is applicable to all TREK subfamily members. Together, our findings exemplify a new means to alter K2P channel activity that should facilitate studies both molecular and systems level studies of K2P function and enable the search for new K2P modulators.\nNucleo-cytoplasmic environment modulates spatio-temporal p53 phase separation\nAuthors: Datta, D.; Navalkar, A.; Sakunthala, A.; Paul, A.; Patel, K.; Masurkar, S.; Gadhe, L.; Sengupta, S.; Poudyal, M.; Devi, J.; Sawner, A. S.; Kadu, P.; Shaw, R.; Pandey, S.; Mukherjee, S.; Gahlot, N.; Sengupta, K.; Maji, S. K.\nScore: 14.1, Published: 2023-10-18 DOI: 10.1101/2023.10.16.562512\nPhase separation of various transcription factors and nucleic acids into biomolecular condensates is known to play an essential role in the regulation of gene expression. Here, we show that p53, a tumor suppressor and transcription factor, phase separates and forms biomolecular condensates in the nucleus of cancer cells as well as when overexpressed in the various cell lines. Although the nuclear condensates of wild-type (WT) p53 maintain their liquid state and are able to bind DNA, cancer-associated mutations not only promote misfolding but also partially rigidify the p53 condensates, which are unable to bind the DNA. Irrespective of WT or mutant form, the cytoplasmic partitioning of p53 with time also results in biomolecular condensate formation, which eventually undergoes rigidification. In vitro, WT p53 core domain (p53C) forms biomolecular condensates, which rigidify with time and the process is further promoted by cancer-associated mutations. Both RNA and non-specific DNA promote LLPS of p53C, but specific DNA promotes the dissolution of p53C condensates. The result suggests that the cellular microenvironment regulates p53 LLPS, material property and its functions.\nDesign of intrinsically disordered protein variants with diverse structural properties\nAuthors: Pesce, F.; Bremer, A.; Tesei, G.; Hopkins, J. B.; Grace, C. R.; Mittag, T.; Lindorff-Larsen, K.\nScore: 13.8, Published: 2023-10-24 DOI: 10.1101/2023.10.22.563461\nIntrinsically disordered proteins (IDPs) perform a wide range of functions in biology, suggesting that the ability to design IDPs could help expand the repertoire of proteins with novel functions. Designing IDPs with specific structural or functional properties has, however, been difficult, in part because determining accurate conformational ensembles of IDPs generally requires a combination of computational modelling and experiments. Motivated by recent advancements in efficient physics-based models for simulations of IDPs, we have developed a general algorithm for designing IDPs with specific structural properties. We demonstrate the power of the algorithm by generating variants of naturally occurring IDPs with different levels of compaction and that vary more than 100 fold in their propensity to undergo phase separation, even while keeping a fixed amino acid composition. We experimentally tested designs of variants of the low-complexity domain of hnRNPA1 and find high accuracy in our computational predictions, both in terms of single-chain compaction and propensity to undergo phase separation. We analyze the sequence features that determine changes in compaction and propensity to phase separate and find an overall good agreement with previous findings for naturally occurring sequences. Our general, physics-based method enables the design of disordered sequences with specified conformational properties. Our algorithm thus expands the toolbox for protein design to include also the most flexible proteins and will enable the design of proteins whose functions exploit the many properties afforded by protein disorder.\nFast and universal single molecule localization using multi-dimensional point spread functions\nAuthors: Li, M.; Shi, W.; Liu, S.; Fu, S.; Fei, Y.; Zhou, L.; Li, Y.\nScore: 10.7, Published: 2023-10-20 DOI: 10.1101/2023.10.17.562517\nThe recent development of single molecule imaging techniques has enabled not only high accuracy spatial resolution imaging but also information rich functional imaging. Abundant information of the single molecules can be encoded in its diffraction pattern and be extracted precisely (e.g. 3D position, wavelength, dipole orientation). However, sophisticated high dimensional point spread function (PSF) modeling and analyzing methods have greatly impeded the broad accessibility of these techniques. Here, we present a graphics processing unit (GPU)-based B-spline PSF modeling method which could flexibly model high dimensional PSFs with arbitrary shape without greatly increasing the model parameters. Our B-spline fitter achieves 100 times speed improvement and minimal uncertainty for each dimension, enabling efficient high dimensional single molecule analysis. We demonstrated, both in simulations and experiments, the universality and flexibility of our B-spline fitter to accurately extract the abundant information from different types of high dimensional single molecule data including multicolor PSF (3D + color), multi-channel four-dimensional 4Pi-PSF (3D + interference phase) and five-dimensional vortex PSF (3D + dipole orientation).\n",
  "wordCount" : "2014",
  "inLanguage": "en",
  "datePublished": "2023-10-25T10:39:04Z",
  "dateModified": "2023-10-25T10:39:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on October 25, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563182">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563182" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563182">
        <p class="paperTitle">Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563182" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563182" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Motone, K.; Kontogiorgos-Heintz, D.; Wee, J.; Kurihara, K.; Yang, S.; Roote, G.; Fang, Y.; Cardozo, N.; Nivala, J.</p>
        <p class="info">Score: 164.5, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563182' target='https://doi.org/10.1101/2023.10.19.563182'> 10.1101/2023.10.19.563182</a></p>
        <p class="abstract">The ability to sequence single protein molecules in their native, full-length form would enable a more comprehensive understanding of proteomic diversity. Current technologies, however, are limited in achieving this goal. Here, we establish a method for long-range, single-molecule reading of intact protein strands on a commercial nanopore sensor array. By using the ClpX unfoldase to ratchet proteins through a CsgG nanopore, we achieve single-amino acid level sensitivity, enabling sequencing of combinations of amino acid substitutions across long protein strands. For greater sequencing accuracy, we demonstrate the ability to reread individual protein molecules, spanning hundreds of amino acids in length, multiple times, and explore the potential for high accuracy protein barcode sequencing. Further, we develop a biophysical model that can simulate raw nanopore signals a priori, based on amino acid volume and charge, enhancing the interpretation of raw signal data. Finally, we apply these methods to examine intact, folded protein domains for complete end-to-end analysis. These results provide proof-of-concept for a platform that has the potential to identify and characterize full-length proteoforms at single-molecule resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563586">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563586" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563586">
        <p class="paperTitle">Data-driven regularisation lowers the size barrier of cryo-EM structure determination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563586" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563586" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kimanius, D.; Jamali, K.; Wilkinson, M. E.; LoÌˆvestam, S.; Velazhahan, V.; Nakane, T.; Scheres, S. H.</p>
        <p class="info">Score: 78.2, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563586' target='https://doi.org/10.1101/2023.10.23.563586'> 10.1101/2023.10.23.563586</a></p>
        <p class="abstract">Macromolecular structure determination by electron cryo-microscopy (cryo-EM) is limited by the alignment of noisy images of individual particles. Because smaller particles have weaker signals, alignment errors impose size limitations on its applicability. Here, we explore how image alignment is improved by the application of deep-learning to exploit prior knowledge about biological macromolecular structures that would otherwise be difficult to express mathematically. We train a denoising convolutional neural network on pairs of half-set reconstructions from the electron microscopy data bank (EMDB) and use this denoiser as an alternative to a commonly used smoothness prior. We demonstrate that this approach, which we call Blush regularisation, yields better reconstructions than existing algorithms, in particular for data with low signal-to-noise ratios. The reconstruction of a protein-nucleic acid complex with a molecular weight of 40 kDa, which was previously intractable, illustrates that regularisation through denoising will expand the applicability of cryo-EM structure determination for a wide range of biological macromolecules.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562877">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562877" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562877">
        <p class="paperTitle">DynaMight: estimating molecular motions with improved reconstruction from cryo-EM images</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562877" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562877" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schwab, J.; Kimanius, D.; Burt, A.; Dendooven, T.; Scheres, S.</p>
        <p class="info">Score: 46.4, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562877' target='https://doi.org/10.1101/2023.10.18.562877'> 10.1101/2023.10.18.562877</a></p>
        <p class="abstract">How to deal with continuously flexing molecules is one of the biggest outstanding challenges in single-particle analysis of proteins from cryo-EM images. Here, we present DynaMight, a new software tool that estimates a continuous space of conformations in a cryo-EM data set by learning 3D deformations of a Gaussian pseudo-atomic model of a consensus structure for every particle image. Inversion of the learnt deformations is then used to obtain an improved reconstruction of the consensus structure. We illustrate the performance of DynaMight for several experimental cryo-EM data sets. We also show how error estimates on the deformations may be obtained by independently training two variational autoencoders (VAEs) on half sets of the cryo-EM data, and how regularisation of the 3D deformations through the use of atomic models may lead to important artefacts due to model bias. DynaMight is distributed as free, open-source software, as part of RELION-5.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562464">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562464" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562464">
        <p class="paperTitle">Structural and mechanistic insights into Streptococcus pneumoniae NADPH oxidase</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562464" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562464" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dubach, V. R.; San Segundo-Acosta, P.; Murphy, B. J.</p>
        <p class="info">Score: 27.4, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562464' target='https://doi.org/10.1101/2023.10.17.562464'> 10.1101/2023.10.17.562464</a></p>
        <p class="abstract">NADPH oxidases (NOXs) play a major role in the physiology of eukaryotic cells by mediating the production of reactive oxygen species (ROS). Evolutionarily distant proteins sharing the NOX catalytic core have been recently described in Bacteria. Among them, the Streptococcus pneumoniae NOX (SpNOX) has been proposed as a model for the study of NOXs due to its high activity and stability in detergent micelles. Here, we report high-resolution cryo-EM structures of substrate-free and stably reduced NADH-bound SpNOX, and of the NADPH-bound SpNOX and a Phe397Ala mutant under turnover conditions. In combination with structure-guided mutagenesis and biochemical analyses, we provide the structural basis for constitutive activity, the lack of substrate specificity towards NADPH and the electron transfer pathway. Additionally, we shed light on the catalytic regulation by the C-terminal tail residue Phe397 and the potential in vivo function of this protein.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563665">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563665" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563665">
        <p class="paperTitle">Interfacial morphodynamics of proliferating microbial communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563665" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563665" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Calvo, A.; Trenado-Yuste, C.; Lee, H.; Gore, J.; Wingreen, N. S.; Datta, S. S.</p>
        <p class="info">Score: 22.8, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563665' target='https://doi.org/10.1101/2023.10.23.563665'> 10.1101/2023.10.23.563665</a></p>
        <p class="abstract">In microbial communities, various cell types often coexist by occupying distinct spatial domains. What determines the shape of the interface between such domains---which in turn influences the interactions between cells and overall community function? Here, we address this question by developing a continuum model of a 2D spatially-structured microbial community with two distinct cell types. We find that, depending on the balance of the different cell proliferation rates and substrate friction coefficients, the interface between domains is either stable and smooth, or unstable and develops finger-like protrusions. We establish quantitative principles describing when these different interfacial behaviors arise, and find good agreement both with the results of previous experimental reports as well as new experiments performed here. Our work thus helps to provide a biophysical basis for understanding the interfacial morphodynamics of proliferating microbial communities, as well as a broader range of proliferating active systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563122">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563122" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563122">
        <p class="paperTitle">Automated 3D multi-color single-molecule localization microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563122" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563122" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Power, R. M.; Tschanz, A.; Zimmermann, T.; Ries, J.</p>
        <p class="info">Score: 17.9, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563122' target='https://doi.org/10.1101/2023.10.23.563122'> 10.1101/2023.10.23.563122</a></p>
        <p class="abstract">Since its inception, single molecule localization microscopy (SMLM) has enabled imaging scientists to visualize biological structures with unprecedented resolution. Particularly powerful implementations capable of 3D, multi-color and high-throughput imaging have yielded key biological insights although widespread access to such technologies has been limited. The purpose of this protocol is to provide a guide for interested researchers to establish high-end SMLM in their laboratories. We detail the initial configuration and subsequent assembly of the SMLM, including instructions for alignment of all optical pathways, software/hardware integration and operation of the instrument. We describe validation steps including the preparation and imaging of test- and biological samples with structures of well-defined geometry and assist the user in troubleshooting and benchmarking performance. Additionally, we provide a walkthrough of the reconstruction of a super-resolved dataset from acquired raw images using the Super-resolution Microscopy Analysis Platform (SMAP). Depending on the instrument configuration, the cost of components is in the range $80,000 - 160,000, a fraction of the cost of a commercial instrument. A builder with some experience of optical systems is expected to require 3 - 6 months from the start of system construction to attain high-quality 3D and multi-color biological images.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.15.561774">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.15.561774" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.15.561774">
        <p class="paperTitle">Development of covalent chemogenetic K2P channel activators</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.15.561774" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.15.561774" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Deal, P. E.; Lee, H.; Mondal, A.; Lolicato, M.; Furtado de Mandonca, P. R.; Black, H.; El-Hilali, X.; Bryant, C.; Isacoff, E. Y.; Renslo, A.; Minor, D. L.</p>
        <p class="info">Score: 14.2, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.15.561774' target='https://doi.org/10.1101/2023.10.15.561774'> 10.1101/2023.10.15.561774</a></p>
        <p class="abstract">K2P potassium channels regulate excitability by affecting cellular resting membrane potential in the brain, cardiovascular system, immune cells, and sensory organs. Despite their important roles in anesthesia, arrhythmia, pain, hypertension, sleep, and migraine, the ability to control K2P function remains limited. Here, we describe a chemogenetic strategy termed CATKLAMP (Covalent Activation of TREK family K&#43; channels to cLAmp Membrane Potential) that leverages the discovery of a site in the K2P modulator pocket that reacts with electrophile-bearing derivatives of a TREK subfamily small molecule activator, ML335, to activate the channel irreversibly. We show that the CATKLAMP strategy can be used to probe fundamental aspects of K2P function, as a switch to silence neuronal firing, and is applicable to all TREK subfamily members. Together, our findings exemplify a new means to alter K2P channel activity that should facilitate studies both molecular and systems level studies of K2P function and enable the search for new K2P modulators.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562512">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562512" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562512">
        <p class="paperTitle">Nucleo-cytoplasmic environment modulates spatio-temporal p53 phase separation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562512" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562512" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Datta, D.; Navalkar, A.; Sakunthala, A.; Paul, A.; Patel, K.; Masurkar, S.; Gadhe, L.; Sengupta, S.; Poudyal, M.; Devi, J.; Sawner, A. S.; Kadu, P.; Shaw, R.; Pandey, S.; Mukherjee, S.; Gahlot, N.; Sengupta, K.; Maji, S. K.</p>
        <p class="info">Score: 14.1, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562512' target='https://doi.org/10.1101/2023.10.16.562512'> 10.1101/2023.10.16.562512</a></p>
        <p class="abstract">Phase separation of various transcription factors and nucleic acids into biomolecular condensates is known to play an essential role in the regulation of gene expression. Here, we show that p53, a tumor suppressor and transcription factor, phase separates and forms biomolecular condensates in the nucleus of cancer cells as well as when overexpressed in the various cell lines. Although the nuclear condensates of wild-type (WT) p53 maintain their liquid state and are able to bind DNA, cancer-associated mutations not only promote misfolding but also partially rigidify the p53 condensates, which are unable to bind the DNA. Irrespective of WT or mutant form, the cytoplasmic partitioning of p53 with time also results in biomolecular condensate formation, which eventually undergoes rigidification. In vitro, WT p53 core domain (p53C) forms biomolecular condensates, which rigidify with time and the process is further promoted by cancer-associated mutations. Both RNA and non-specific DNA promote LLPS of p53C, but specific DNA promotes the dissolution of p53C condensates. The result suggests that the cellular microenvironment regulates p53 LLPS, material property and its functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.22.563461">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.22.563461" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.22.563461">
        <p class="paperTitle">Design of intrinsically disordered protein variants with diverse structural properties</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.22.563461" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.22.563461" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pesce, F.; Bremer, A.; Tesei, G.; Hopkins, J. B.; Grace, C. R.; Mittag, T.; Lindorff-Larsen, K.</p>
        <p class="info">Score: 13.8, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.22.563461' target='https://doi.org/10.1101/2023.10.22.563461'> 10.1101/2023.10.22.563461</a></p>
        <p class="abstract">Intrinsically disordered proteins (IDPs) perform a wide range of functions in biology, suggesting that the ability to design IDPs could help expand the repertoire of proteins with novel functions. Designing IDPs with specific structural or functional properties has, however, been difficult, in part because determining accurate conformational ensembles of IDPs generally requires a combination of computational modelling and experiments. Motivated by recent advancements in efficient physics-based models for simulations of IDPs, we have developed a general algorithm for designing IDPs with specific structural properties. We demonstrate the power of the algorithm by generating variants of naturally occurring IDPs with different levels of compaction and that vary more than 100 fold in their propensity to undergo phase separation, even while keeping a fixed amino acid composition. We experimentally tested designs of variants of the low-complexity domain of hnRNPA1 and find high accuracy in our computational predictions, both in terms of single-chain compaction and propensity to undergo phase separation. We analyze the sequence features that determine changes in compaction and propensity to phase separate and find an overall good agreement with previous findings for naturally occurring sequences. Our general, physics-based method enables the design of disordered sequences with specified conformational properties. Our algorithm thus expands the toolbox for protein design to include also the most flexible proteins and will enable the design of proteins whose functions exploit the many properties afforded by protein disorder.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562517">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562517" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562517">
        <p class="paperTitle">Fast and universal single molecule localization using multi-dimensional point spread functions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562517" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562517" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, M.; Shi, W.; Liu, S.; Fu, S.; Fei, Y.; Zhou, L.; Li, Y.</p>
        <p class="info">Score: 10.7, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562517' target='https://doi.org/10.1101/2023.10.17.562517'> 10.1101/2023.10.17.562517</a></p>
        <p class="abstract">The recent development of single molecule imaging techniques has enabled not only high accuracy spatial resolution imaging but also information rich functional imaging. Abundant information of the single molecules can be encoded in its diffraction pattern and be extracted precisely (e.g. 3D position, wavelength, dipole orientation). However, sophisticated high dimensional point spread function (PSF) modeling and analyzing methods have greatly impeded the broad accessibility of these techniques. Here, we present a graphics processing unit (GPU)-based B-spline PSF modeling method which could flexibly model high dimensional PSFs with arbitrary shape without greatly increasing the model parameters. Our B-spline fitter achieves 100 times speed improvement and minimal uncertainty for each dimension, enabling efficient high dimensional single molecule analysis. We demonstrated, both in simulations and experiments, the universality and flexibility of our B-spline fitter to accurately extract the abundant information from different types of high dimensional single molecule data including multicolor PSF (3D &#43; color), multi-channel four-dimensional 4Pi-PSF (3D &#43; interference phase) and five-dimensional vortex PSF (3D &#43; dipole orientation).</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
