<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Disease-specific tau filaments assemble via polymorphic intermediates
Authors: Lövestam, S.; Li, D.; Wagstaff, J. L.; Kotecha, A.; Kimanius, D.; McLaughlin, S. H.; Murzin, A. G.; Freund, S. M. V.; Goedert, M.; Scheres, S. H. W.
Score: 29.6, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550295
Intermediate species in the assembly of amyloid filaments are believed to play a central role in neurodegenerative diseases and may constitute important targets for therapeutic intervention. However, structural information about intermediate species has been scarce and the molecular mechanisms by which amyloids assemble remain largely unknown.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Disease-specific tau filaments assemble via polymorphic intermediates
Authors: Lövestam, S.; Li, D.; Wagstaff, J. L.; Kotecha, A.; Kimanius, D.; McLaughlin, S. H.; Murzin, A. G.; Freund, S. M. V.; Goedert, M.; Scheres, S. H. W.
Score: 29.6, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550295
Intermediate species in the assembly of amyloid filaments are believed to play a central role in neurodegenerative diseases and may constitute important targets for therapeutic intervention. However, structural information about intermediate species has been scarce and the molecular mechanisms by which amyloids assemble remain largely unknown." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-30T10:36:57+00:00" />
<meta property="article:modified_time" content="2023-07-30T10:36:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Disease-specific tau filaments assemble via polymorphic intermediates
Authors: Lövestam, S.; Li, D.; Wagstaff, J. L.; Kotecha, A.; Kimanius, D.; McLaughlin, S. H.; Murzin, A. G.; Freund, S. M. V.; Goedert, M.; Scheres, S. H. W.
Score: 29.6, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550295
Intermediate species in the assembly of amyloid filaments are believed to play a central role in neurodegenerative diseases and may constitute important targets for therapeutic intervention. However, structural information about intermediate species has been scarce and the molecular mechanisms by which amyloids assemble remain largely unknown."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Disease-specific tau filaments assemble via polymorphic intermediates\nAuthors: Lövestam, S.; Li, D.; Wagstaff, J. L.; Kotecha, A.; Kimanius, D.; McLaughlin, S. H.; Murzin, A. G.; Freund, S. M. V.; Goedert, M.; Scheres, S. H. W.\nScore: 29.6, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550295\nIntermediate species in the assembly of amyloid filaments are believed to play a central role in neurodegenerative diseases and may constitute important targets for therapeutic intervention. However, structural information about intermediate species has been scarce and the molecular mechanisms by which amyloids assemble remain largely unknown.",
  "keywords": [
    
  ],
  "articleBody": " Disease-specific tau filaments assemble via polymorphic intermediates\nAuthors: Lövestam, S.; Li, D.; Wagstaff, J. L.; Kotecha, A.; Kimanius, D.; McLaughlin, S. H.; Murzin, A. G.; Freund, S. M. V.; Goedert, M.; Scheres, S. H. W.\nScore: 29.6, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550295\nIntermediate species in the assembly of amyloid filaments are believed to play a central role in neurodegenerative diseases and may constitute important targets for therapeutic intervention. However, structural information about intermediate species has been scarce and the molecular mechanisms by which amyloids assemble remain largely unknown. Here, we use time-resolved electron cryo-microscopy (cryo-EM) to study the in vitro assembly of recombinant truncated tau (amino acids 297-391) into paired helical filaments of Alzheimers disease or into filaments of chronic traumatic encephalopathy. We report the formation of a shared first intermediate amyloid (FIA), with an ordered core comprising amino acids 302-316. Nuclear magnetic resonance indicates that the same amino acids adopt rigid, {beta}-strand-like conformations in monomeric tau. At later time points, the FIAs disappear and we observe many different intermediate amyloid filaments, with structures that depend on the reaction conditions. At the end of both reactions, most intermediate amyloids disappear and filaments with the same ordered cores as those from human brains remain. Our results provide structural insights into the processes of primary and secondary nucleation of amyloid assembly, with implications for the design of novel therapies.\nCoevolution-driven method for efficiently simulating conformational changes in proteins reveals molecular details of ligand effects in the beta2AR receptor\nAuthors: Mitrovic, D.; Chen, Y.; Marciniak, A.; Delemotte, L.\nScore: 36.0, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549854\nWith the advent of AI-powered structure prediction, the scientific community is inching ever closer to solving protein folding. An unresolved enigma, however, is to accurately, reliably and deterministically predict alternative conformational states that are crucial for the function of e.g. transporters, receptors or ion channels where conformational cycling is innately coupled to protein function. Accurately discovering and exploring all conformational states of membrane proteins has been challenging due to the need to retain atomistic detail while enhancing the sampling along interesting degrees of freedom. The challenges include but are not limited to finding which degrees of freedom are relevant, how to accelerate the sampling along them, and then quantifying the populations of each micro- and macrostate. In this work, we present a methodology that finds the relevant degrees of freedom by combining evolution and physics through machine learning and apply it to the {beta}2 adrenergic receptor conformational sampling. In addition to predicting new conformations that are beyond the training set, we have computed free energy surfaces associated with the proteins conformational landscape. We then show that the methodology is able to quantitatively predict the effect of an array of ligands on the {beta}2 adrenergic receptor activation, and that the full conformational landscape, including states related to biased signaling, is discovered using this procedure. Lastly, we also stake out the structural determinants of activation and inactivation pathway signaling through different ligands.\nMechano-induced homotypic patterned domain formation by monocytes\nAuthors: Du, W.; Zhu, J.; Wu, Y.; Kiemen, A.; Sun, S.; Wirtz, D.\nScore: 20.8, Published: 2023-07-29 DOI: 10.1101/2023.07.27.550819\nMatrix stiffness and corresponding mechano-signaling play indispensable roles in cellular phenotypes and functions. How tissue stiffness influences the behavior of monocytes, a major circulating leukocyte of the innate system, and how it may promote the emergence of collective cell behavior is less understood. Here, using tunable collagen-coated hydrogels of physiological stiffness, we show that human primary monocytes undergo a dynamic local phase separation to form highly patterned multicellular multi-layered domains on soft matrix. Local activation of the {beta}2 integrin initiates inter-cellular adhesion, while global soluble inhibitory factors maintain the steady-state domain pattern over days. Patterned domain formation generated by monocytes is unique among other key immune cells, including macrophages, B cells, T cells, and NK cells. While inhibiting their phagocytic capability, domain formation promotes monocytes' survival. We develop a computational model based on the Cahn-Hilliard equation, which includes combined local activation and global inhibition mechanisms of intercellular adhesion suggested by our experiments, and provides experimentally validated predictions of the role of seeding density and both chemotactic and random cell migration on pattern formation.\nGraph Attention Site Prediction (GrASP): Identifying Druggable Binding Sites Using Graph Neural Networks with Attention\nAuthors: Smith, Z.; Strobel, M.; Vani, B.; Tiwary, P.\nScore: 18.9, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550565\nIdentifying and discovering druggable protein binding sites is an important early step in computer-aided drug discovery but remains a difficult task where most campaigns rely on a priori knowledge of binding sites from experiments. Here we present a novel binding site prediction method called Graph Attention Site Prediction (GrASP) and re-evaluate assumptions in nearly every step in the site prediction workflow from dataset preparation to model evaluation. GrASP is able to achieve state-of-the-art performance at recovering binding sites in PDB structures while maintaining a high degree of precision which will minimize wasted computation in downstream tasks such as docking and free energy perturbation.\nDirect optical measurement of intra-molecular distances down to the Angstrom scale\nAuthors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.\nScore: 126.6, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133\nOptical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision. Our method is validated by quantifying well-characterized 1-to-10 nm distances in polypeptides and proteins. Moreover, we visualize the orientations of immunoglobulin subunits and reveal specific configurations of a histidine kinase domain dimer. Our results open the door for examining proximities and interactions of macromolecules under physiological conditions.\nFolding-upon-binding pathways of an intrinsically disordered protein from a deep Markov state model\nAuthors: Sisk, T.; Robustelli, P.\nScore: 13.3, Published: 2023-07-25 DOI: 10.1101/2023.07.21.550103\nA central challenge in the study of intrinsically disordered proteins is the characterization of the mechanisms by which they bind their physiological interaction partners. Here, we utilize a deep learning based Markov state modeling approach to characterize the folding-upon-binding pathways observed in a long-time scale molecular dynamics simulation of a disordered region of the measles virus nucleoprotein NTAIL reversibly binding the X domain of the measles virus phosphoprotein complex. We find that folding-upon-binding predominantly occurs via two distinct encounter complexes that are differentiated by the binding orientation, helical content, and conformational heterogeneity of NTAIL. We do not, however, find evidence for the existence of canonical conformational selection or induced fit binding pathways. We observe four kinetically separated native-like bound states that interconvert on time scales of eighty to five hundred nanoseconds. These bound states share a core set of native intermolecular contacts and stable NTAIL helices and are differentiated by a sequential formation of native and non-native contacts and additional helical turns. Our analyses provide an atomic resolution structural description of intermediate states in a folding-upon-binding pathway and elucidate the nature of the kinetic barriers between metastable states in a dynamic and heterogenous, or \"fuzzy\", protein complex.\nLearning high-dimensional reaction coordinates of fast-folding proteins using State Predictive Information Bottleneck and Bias Exchange Metadynamics\nAuthors: Pomarici, N.; Mehdi, S.; Quoika, P.; Lee, S.; Loeffler, J. R.; Liedl, K.; Tiwary, P.; Fernandez-Quintero, M. L.\nScore: 13.3, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550401\nBiological events occurring on long timescales, such as protein folding, remain hard to capture with conventional molecular dynamics (MD) simulation. To overcome these limitations, enhanced sampling techniques can be used to sample regions of the free energy landscape separated by high energy barriers, thereby allowing to observe these rare events. However, many of these techniques require a priori knowledge of the appropriate reaction coordinates (RCs) that describe the process of interest. In recent years, Artificial Intelligence (AI) models have emerged as promising approaches to accelerate rare event sampling. However, integration of these AI methods with MD for automated learning of improved RCs is not trivial, particularly when working with undersampled trajectories and highly complex systems. In this study, we employed the State Predictive Information Bottleneck (SPIB) neural network, coupled with bias exchange metadynamics simulations (BE-metaD), to investigate the unfolding process of two proteins, chignolin and villin. By utilizing the high-dimensional RCs learned from SPIB even with poor training data, BE-metaD simulations dramatically accelerate the sampling of the unfolding process for both proteins. In addition, we compare different RCs and find that the careful selection of RCs is crucial to substantially speed up the sampling of rare events. Thus, this approach, leveraging the power of AI and enhanced sampling techniques, holds great promise for advancing our understanding of complex biological processes occurring on long timescales. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=139 SRC=\"FIGDIR/small/550401v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (29K): org.highwire.dtl.DTLVardef@187ff8corg.highwire.dtl.DTLVardef@19a7fb0org.highwire.dtl.DTLVardef@350afforg.highwire.dtl.DTLVardef@64f095_HPS_FORMAT_FIGEXP M_FIG TABLE OF CONTENT GRAPHIC C_FIG\nPhase-Separated Droplets Swim to Their Dissolution\nAuthors: Jambon-Puillet, E.; Testa, A.; Lorenz, C. M.; Style, R. W.; Rebane, A.; Dufresne, E. R.\nScore: 23.8, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549556\nBiological macromolecules can condense into liquid domains. In cells, these condensates form membraneless organelles that can organize chemical reactions1,2. However, little is known about the physical consequences of chemical activity in and around condensates. Working with model bovine serum albumin (BSA) condensates3, we show that droplets swim along chemical gradients. Active BSA droplets loaded with urease swim toward each other. Passive BSA droplets show diverse responses to externally applied gradients of the enzymes substrate and products. In all these cases, droplets swim toward solvent conditions that favor their dissolution. We call this behavior dialytaxis, and expect it to be generic, as conditions which favor dissolution typically reduce interfacial tension, whose gradients are well-known to drive droplet motion4,5. These results suggest alternative physical mechanisms for active transport in living cells, and may enable the design of fluid micro-robots.\nDeep learning insights into the architecture of the mammalian egg-sperm fusion synapse\nAuthors: Elofsson, A.; Han, L.; Bianchi, E.; Wright, G. J.; Jovine, L.\nScore: 5.6, Published: 2023-07-28 DOI: 10.1101/2023.07.27.550195\nFertilization is a crucial event in sexual reproduction culminating in the fusion of the haploid sperm and egg to form a new diploid organism. In mammals, egg receptor JUNO and sperm IZUMO1 directly interact to mediate adhesion between gamete membranes; however, their role in fusion remains enigmatic. Here, we used AlphaFold to predict the structure of other extracellular proteins that are essential for fertilization and determine if they could form a fusogenic complex. We first identified TMEM81, whose gene is expressed by mouse spermatids, as a protein that has structural homologies with both IZUMO1 and another essential sperm ligand, SPACA6. Using a set of proteins known to be important for fertilization, we systematically searched for predicted direct binary interactions using an unguided approach. We identified a tetrameric complex of two sperm (IZUMO1 and SPACA6) and two egg (JUNO and CD9) proteins, and further modeling using this core complex identified TMEM81 as an additional component. We show that the complex is structurally consistent with both the expected topology on opposing gamete membranes and the location of predicted N-linked glycans that were not considered during prediction with AlphaFold-Multimer. Our model suggests that these proteins could organize to form a synapse-like complex between the sperm and egg membrane at the point of fertilization. Finally, the structural modeling approach described here could be more generally useful to gain structural insights into highly transient protein complexes that are difficult to detect experimentally.\nInfluenza A virus infection alters lipid packing and surface electrostatic potential of the host plasma membrane\nAuthors: Petrich, A.; Chiantia, S.\nScore: 5.5, Published: 2023-07-26 DOI: 10.1101/2023.07.25.550511\nThe pathogenesis of influenza A viruses (IAVs) is influenced by several factors, including IAV strain origin and reassortment, tissue tropism and host type. While such factors were mostly investigated in the context of virus entry, fusion and replication, little is known about the viral-induced changes to the host lipid membranes which might be relevant in the context of virion assembly. In this work, we applied several biophysical fluorescence microscope techniques (i.e., Forster energy resonance transfer, generalized polarization imaging and scanning fluorescence correlation spectroscopy) to quantify the effect of infection by two IAV strains of different origin on the plasma membrane (PM) of avian and human cell lines. We found that IAV infection affects the membrane charge of the inner leaflet of the PM. Moreover, we showed that IAV infection impacts lipid-lipid interactions by decreasing membrane fluidity and increasing lipid packing. Because of such alterations, diffusive dynamics of membrane-associated proteins are hindered. Taken to-gether, our results indicate that the infection of avian and human cell lines with IAV strains of different origins had similar effects on the biophysical properties of the PM.\n",
  "wordCount" : "2108",
  "inLanguage": "en",
  "datePublished": "2023-07-30T10:36:57Z",
  "dateModified": "2023-07-30T10:36:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on July 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550295">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550295" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550295">
        <p class="paperTitle">Disease-specific tau filaments assemble via polymorphic intermediates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550295" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550295" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lövestam, S.; Li, D.; Wagstaff, J. L.; Kotecha, A.; Kimanius, D.; McLaughlin, S. H.; Murzin, A. G.; Freund, S. M. V.; Goedert, M.; Scheres, S. H. W.</p>
        <p class="info">Score: 29.6, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550295' target='https://doi.org/10.1101/2023.07.24.550295'> 10.1101/2023.07.24.550295</a></p>
        <p class="abstract">Intermediate species in the assembly of amyloid filaments are believed to play a central role in neurodegenerative diseases and may constitute important targets for therapeutic intervention. However, structural information about intermediate species has been scarce and the molecular mechanisms by which amyloids assemble remain largely unknown. Here, we use time-resolved electron cryo-microscopy (cryo-EM) to study the in vitro assembly of recombinant truncated tau (amino acids 297-391) into paired helical filaments of Alzheimers disease or into filaments of chronic traumatic encephalopathy. We report the formation of a shared first intermediate amyloid (FIA), with an ordered core comprising amino acids 302-316. Nuclear magnetic resonance indicates that the same amino acids adopt rigid, {beta}-strand-like conformations in monomeric tau. At later time points, the FIAs disappear and we observe many different intermediate amyloid filaments, with structures that depend on the reaction conditions. At the end of both reactions, most intermediate amyloids disappear and filaments with the same ordered cores as those from human brains remain. Our results provide structural insights into the processes of primary and secondary nucleation of amyloid assembly, with implications for the design of novel therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549854">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549854" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549854">
        <p class="paperTitle">Coevolution-driven method for efficiently simulating conformational changes in proteins reveals molecular details of ligand effects in the beta2AR receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549854" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549854" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mitrovic, D.; Chen, Y.; Marciniak, A.; Delemotte, L.</p>
        <p class="info">Score: 36.0, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549854' target='https://doi.org/10.1101/2023.07.20.549854'> 10.1101/2023.07.20.549854</a></p>
        <p class="abstract">With the advent of AI-powered structure prediction, the scientific community is inching ever closer to solving protein folding. An unresolved enigma, however, is to accurately, reliably and deterministically predict alternative conformational states that are crucial for the function of e.g. transporters, receptors or ion channels where conformational cycling is innately coupled to protein function. Accurately discovering and exploring all conformational states of membrane proteins has been challenging due to the need to retain atomistic detail while enhancing the sampling along interesting degrees of freedom. The challenges include but are not limited to finding which degrees of freedom are relevant, how to accelerate the sampling along them, and then quantifying the populations of each micro- and macrostate. In this work, we present a methodology that finds the relevant degrees of freedom by combining evolution and physics through machine learning and apply it to the {beta}2 adrenergic receptor conformational sampling. In addition to predicting new conformations that are beyond the training set, we have computed free energy surfaces associated with the proteins conformational landscape. We then show that the methodology is able to quantitatively predict the effect of an array of ligands on the {beta}2 adrenergic receptor activation, and that the full conformational landscape, including states related to biased signaling, is discovered using this procedure. Lastly, we also stake out the structural determinants of activation and inactivation pathway signaling through different ligands.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550819">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550819" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550819">
        <p class="paperTitle">Mechano-induced homotypic patterned domain formation by monocytes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550819" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550819" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Du, W.; Zhu, J.; Wu, Y.; Kiemen, A.; Sun, S.; Wirtz, D.</p>
        <p class="info">Score: 20.8, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550819' target='https://doi.org/10.1101/2023.07.27.550819'> 10.1101/2023.07.27.550819</a></p>
        <p class="abstract">Matrix stiffness and corresponding mechano-signaling play indispensable roles in cellular phenotypes and functions. How tissue stiffness influences the behavior of monocytes, a major circulating leukocyte of the innate system, and how it may promote the emergence of collective cell behavior is less understood. Here, using tunable collagen-coated hydrogels of physiological stiffness, we show that human primary monocytes undergo a dynamic local phase separation to form highly patterned multicellular multi-layered domains on soft matrix. Local activation of the {beta}2 integrin initiates inter-cellular adhesion, while global soluble inhibitory factors maintain the steady-state domain pattern over days. Patterned domain formation generated by monocytes is unique among other key immune cells, including macrophages, B cells, T cells, and NK cells. While inhibiting their phagocytic capability, domain formation promotes monocytes&#39; survival. We develop a computational model based on the Cahn-Hilliard equation, which includes combined local activation and global inhibition mechanisms of intercellular adhesion suggested by our experiments, and provides experimentally validated predictions of the role of seeding density and both chemotactic and random cell migration on pattern formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550565">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550565" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550565">
        <p class="paperTitle">Graph Attention Site Prediction (GrASP): Identifying Druggable Binding Sites Using Graph Neural Networks with Attention</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550565" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550565" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, Z.; Strobel, M.; Vani, B.; Tiwary, P.</p>
        <p class="info">Score: 18.9, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550565' target='https://doi.org/10.1101/2023.07.25.550565'> 10.1101/2023.07.25.550565</a></p>
        <p class="abstract">Identifying and discovering druggable protein binding sites is an important early step in computer-aided drug discovery but remains a difficult task where most campaigns rely on a priori knowledge of binding sites from experiments. Here we present a novel binding site prediction method called Graph Attention Site Prediction (GrASP) and re-evaluate assumptions in nearly every step in the site prediction workflow from dataset preparation to model evaluation. GrASP is able to achieve state-of-the-art performance at recovering binding sites in PDB structures while maintaining a high degree of precision which will minimize wasted computation in downstream tasks such as docking and free energy perturbation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548133" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548133">
        <p class="paperTitle">Direct optical measurement of intra-molecular distances down to the Angstrom scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.</p>
        <p class="info">Score: 126.6, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548133' target='https://doi.org/10.1101/2023.07.07.548133'> 10.1101/2023.07.07.548133</a></p>
        <p class="abstract">Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision. Our method is validated by quantifying well-characterized 1-to-10 nm distances in polypeptides and proteins. Moreover, we visualize the orientations of immunoglobulin subunits and reveal specific configurations of a histidine kinase domain dimer. Our results open the door for examining proximities and interactions of macromolecules under physiological conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550103">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550103" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550103">
        <p class="paperTitle">Folding-upon-binding pathways of an intrinsically disordered protein from a deep Markov state model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550103" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550103" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sisk, T.; Robustelli, P.</p>
        <p class="info">Score: 13.3, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550103' target='https://doi.org/10.1101/2023.07.21.550103'> 10.1101/2023.07.21.550103</a></p>
        <p class="abstract">A central challenge in the study of intrinsically disordered proteins is the characterization of the mechanisms by which they bind their physiological interaction partners. Here, we utilize a deep learning based Markov state modeling approach to characterize the folding-upon-binding pathways observed in a long-time scale molecular dynamics simulation of a disordered region of the measles virus nucleoprotein NTAIL reversibly binding the X domain of the measles virus phosphoprotein complex. We find that folding-upon-binding predominantly occurs via two distinct encounter complexes that are differentiated by the binding orientation, helical content, and conformational heterogeneity of NTAIL. We do not, however, find evidence for the existence of canonical conformational selection or induced fit binding pathways. We observe four kinetically separated native-like bound states that interconvert on time scales of eighty to five hundred nanoseconds. These bound states share a core set of native intermolecular contacts and stable NTAIL helices and are differentiated by a sequential formation of native and non-native contacts and additional helical turns. Our analyses provide an atomic resolution structural description of intermediate states in a folding-upon-binding pathway and elucidate the nature of the kinetic barriers between metastable states in a dynamic and heterogenous, or &#34;fuzzy&#34;, protein complex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550401">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550401" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550401">
        <p class="paperTitle">Learning high-dimensional reaction coordinates of fast-folding proteins using State Predictive Information Bottleneck and Bias Exchange Metadynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550401" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550401" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pomarici, N.; Mehdi, S.; Quoika, P.; Lee, S.; Loeffler, J. R.; Liedl, K.; Tiwary, P.; Fernandez-Quintero, M. L.</p>
        <p class="info">Score: 13.3, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550401' target='https://doi.org/10.1101/2023.07.24.550401'> 10.1101/2023.07.24.550401</a></p>
        <p class="abstract">Biological events occurring on long timescales, such as protein folding, remain hard to capture with conventional molecular dynamics (MD) simulation. To overcome these limitations, enhanced sampling techniques can be used to sample regions of the free energy landscape separated by high energy barriers, thereby allowing to observe these rare events. However, many of these techniques require a priori knowledge of the appropriate reaction coordinates (RCs) that describe the process of interest. In recent years, Artificial Intelligence (AI) models have emerged as promising approaches to accelerate rare event sampling. However, integration of these AI methods with MD for automated learning of improved RCs is not trivial, particularly when working with undersampled trajectories and highly complex systems. In this study, we employed the State Predictive Information Bottleneck (SPIB) neural network, coupled with bias exchange metadynamics simulations (BE-metaD), to investigate the unfolding process of two proteins, chignolin and villin. By utilizing the high-dimensional RCs learned from SPIB even with poor training data, BE-metaD simulations dramatically accelerate the sampling of the unfolding process for both proteins. In addition, we compare different RCs and find that the careful selection of RCs is crucial to substantially speed up the sampling of rare events. Thus, this approach, leveraging the power of AI and enhanced sampling techniques, holds great promise for advancing our understanding of complex biological processes occurring on long timescales.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=139 SRC=&#34;FIGDIR/small/550401v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (29K):
org.highwire.dtl.DTLVardef@187ff8corg.highwire.dtl.DTLVardef@19a7fb0org.highwire.dtl.DTLVardef@350afforg.highwire.dtl.DTLVardef@64f095_HPS_FORMAT_FIGEXP  M_FIG TABLE OF CONTENT GRAPHIC

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549556">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549556" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549556">
        <p class="paperTitle">Phase-Separated Droplets Swim to Their Dissolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549556" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549556" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jambon-Puillet, E.; Testa, A.; Lorenz, C. M.; Style, R. W.; Rebane, A.; Dufresne, E. R.</p>
        <p class="info">Score: 23.8, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549556' target='https://doi.org/10.1101/2023.07.18.549556'> 10.1101/2023.07.18.549556</a></p>
        <p class="abstract">Biological macromolecules can condense into liquid domains. In cells, these condensates form membraneless organelles that can organize chemical reactions1,2. However, little is known about the physical consequences of chemical activity in and around condensates. Working with model bovine serum albumin (BSA) condensates3, we show that droplets swim along chemical gradients. Active BSA droplets loaded with urease swim toward each other. Passive BSA droplets show diverse responses to externally applied gradients of the enzymes substrate and products. In all these cases, droplets swim toward solvent conditions that favor their dissolution. We call this behavior dialytaxis, and expect it to be generic, as conditions which favor dissolution typically reduce interfacial tension, whose gradients are well-known to drive droplet motion4,5. These results suggest alternative physical mechanisms for active transport in living cells, and may enable the design of fluid micro-robots.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550195">
        <p class="paperTitle">Deep learning insights into the architecture of the mammalian egg-sperm fusion synapse</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Elofsson, A.; Han, L.; Bianchi, E.; Wright, G. J.; Jovine, L.</p>
        <p class="info">Score: 5.6, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550195' target='https://doi.org/10.1101/2023.07.27.550195'> 10.1101/2023.07.27.550195</a></p>
        <p class="abstract">Fertilization is a crucial event in sexual reproduction culminating in the fusion of the haploid sperm and egg to form a new diploid organism. In mammals, egg receptor JUNO and sperm IZUMO1 directly interact to mediate adhesion between gamete membranes; however, their role in fusion remains enigmatic. Here, we used AlphaFold to predict the structure of other extracellular proteins that are essential for fertilization and determine if they could form a fusogenic complex. We first identified TMEM81, whose gene is expressed by mouse spermatids, as a protein that has structural homologies with both IZUMO1 and another essential sperm ligand, SPACA6. Using a set of proteins known to be important for fertilization, we systematically searched for predicted direct binary interactions using an unguided approach. We identified a tetrameric complex of two sperm (IZUMO1 and SPACA6) and two egg (JUNO and CD9) proteins, and further modeling using this core complex identified TMEM81 as an additional component. We show that the complex is structurally consistent with both the expected topology on opposing gamete membranes and the location of predicted N-linked glycans that were not considered during prediction with AlphaFold-Multimer. Our model suggests that these proteins could organize to form a synapse-like complex between the sperm and egg membrane at the point of fertilization. Finally, the structural modeling approach described here could be more generally useful to gain structural insights into highly transient protein complexes that are difficult to detect experimentally.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550511">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550511" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550511">
        <p class="paperTitle">Influenza A virus infection alters lipid packing and surface electrostatic potential of the host plasma membrane</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550511" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550511" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Petrich, A.; Chiantia, S.</p>
        <p class="info">Score: 5.5, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550511' target='https://doi.org/10.1101/2023.07.25.550511'> 10.1101/2023.07.25.550511</a></p>
        <p class="abstract">The pathogenesis of influenza A viruses (IAVs) is influenced by several factors, including IAV strain origin and reassortment, tissue tropism and host type. While such factors were mostly investigated in the context of virus entry, fusion and replication, little is known about the viral-induced changes to the host lipid membranes which might be relevant in the context of virion assembly. In this work, we applied several biophysical fluorescence microscope techniques (i.e., Forster energy resonance transfer, generalized polarization imaging and scanning fluorescence correlation spectroscopy) to quantify the effect of infection by two IAV strains of different origin on the plasma membrane (PM) of avian and human cell lines. We found that IAV infection affects the membrane charge of the inner leaflet of the PM. Moreover, we showed that IAV infection impacts lipid-lipid interactions by decreasing membrane fluidity and increasing lipid packing. Because of such alterations, diffusive dynamics of membrane-associated proteins are hindered. Taken to-gether, our results indicate that the infection of avian and human cell lines with IAV strains of different origins had similar effects on the biophysical properties of the PM.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
