<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="PatternJ: an ImageJ toolset for the automated and quantitative analysis of regular spatial patterns found in sarcomeres, axons, somites, and more
Authors: Baheux Blin, M.; Loreau, V.; Schnorrer, F.; Mangeol, P.
Score: 21.1, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576053
Regular spatial patterns are ubiquitous forms of organization in nature. In animals, regular patterns can be found from the cellular scale to the tissue scale, and from early stages of development to adulthood.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="PatternJ: an ImageJ toolset for the automated and quantitative analysis of regular spatial patterns found in sarcomeres, axons, somites, and more
Authors: Baheux Blin, M.; Loreau, V.; Schnorrer, F.; Mangeol, P.
Score: 21.1, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576053
Regular spatial patterns are ubiquitous forms of organization in nature. In animals, regular patterns can be found from the cellular scale to the tissue scale, and from early stages of development to adulthood." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="PatternJ: an ImageJ toolset for the automated and quantitative analysis of regular spatial patterns found in sarcomeres, axons, somites, and more
Authors: Baheux Blin, M.; Loreau, V.; Schnorrer, F.; Mangeol, P.
Score: 21.1, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576053
Regular spatial patterns are ubiquitous forms of organization in nature. In animals, regular patterns can be found from the cellular scale to the tissue scale, and from early stages of development to adulthood."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "PatternJ: an ImageJ toolset for the automated and quantitative analysis of regular spatial patterns found in sarcomeres, axons, somites, and more\nAuthors: Baheux Blin, M.; Loreau, V.; Schnorrer, F.; Mangeol, P.\nScore: 21.1, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576053\nRegular spatial patterns are ubiquitous forms of organization in nature. In animals, regular patterns can be found from the cellular scale to the tissue scale, and from early stages of development to adulthood.",
  "keywords": [
    
  ],
  "articleBody": " PatternJ: an ImageJ toolset for the automated and quantitative analysis of regular spatial patterns found in sarcomeres, axons, somites, and more\nAuthors: Baheux Blin, M.; Loreau, V.; Schnorrer, F.; Mangeol, P.\nScore: 21.1, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576053\nRegular spatial patterns are ubiquitous forms of organization in nature. In animals, regular patterns can be found from the cellular scale to the tissue scale, and from early stages of development to adulthood. To understand the formation of these patterns, how they form and mature, and how they are affected by perturbations, a precise quantitative description of the patterns is essential. However, accessible tools that offer in-depth analysis without the need for computational skills are lacking for biologists. Here we present PatternJ, a novel toolset to analyze regular pattern organizations precisely and automatically. This toolset, to be used with the popular imaging processing program ImageJ/Fiji, facilitates the extraction of key geometric features within and between pattern repeats. We validated PatternJ on simulated data and tested it on images of sarcomeres in insect muscles and cardiomyocytes, actin rings in neurons, and somites in zebrafish embryos obtained using confocal fluorescence microscopy, STORM, electron microscopy, and bright-field imaging. We show that the toolset delivers subpixel feature extraction reliably even with images of low signal-to-noise ratio. PatternJs straightforward use and functionalities make it valuable for various scientific fields requiring quantitative pattern analysis, including the sarcomere biology of muscles or the patterning of mammalian axons, speeding up discoveries with the bonus of high reproducibility.\nEngineering CRBN for rapid identification of next generation binders\nAuthors: Bailey, H. J.; Eisert, J.; Vollrath, J.; Leibrock, E.-M.; Kondratov, I.; Matviyuk, T.; Tolmachova, N.; Langer, J. D.; Wegener, A. A.; Sorrell, F. J.; Dikic, I.\nScore: 15.5, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576231\nThe majority of clinical degrader candidates utilize an immunomodulatory imide drug (IMiD)-based derivative, that directs their target substrate to the E3 ligase receptor Cereblon (CRBN), however, concomitant neo-substrate degradation by IMiDs often results in severe off-target effects. Further biophysical screening is needed to discover CRBN binders that might overcome these safety concerns, but the previously reported CRBN protein constructs suffer significant limitations that reduce their applicability as tools for biophysical assays making large-scale screening efforts a costly endeavor. This is due to the requirement of CRBN co-expression with DDB1 in insect cells to produce soluble protein that contains all the reported structural features necessary for proper compound interaction. Here, a near full-length human CRBN construct was designed that retains these required features, but for the first time allows the generation of highly homogenous and cost-efficient expression in E.coli, eliminating the need for DDB1 co-expression. We have extensively profiled the new construct against the existing CRBN constructs in binary and ternary degrader complexes, showing it to be the \"best of both worlds\" in terms of binding characteristics and ease of protein production. We then designed the Enamine CRBN focused IMiD library of 4480-compounds and demonstrated applicability of the new construct to high throughput screening by identifying novel CRBN binders with high potency and ligand efficiency. The new methods described here should serve as valuable tools for the development of the next generation IMiD-based degraders.\nSifting Through the Noise: A Computational Pipeline for Accurate Prioritization of Protein- Protein Binding Candidates in High-Throughput Protein Libraries\nAuthors: Mondal, A.; Singh, B.; Felkner, R. H.; De Falco, A.; Swapna, G.; Montelione, G. T.; Roth, M.; Perez, A.\nScore: 13.0, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576374\nIdentifying the interactome for a protein of interest is challenging due to the large number of possible binders. High-throughput experimental approaches narrow down possible binding partners, but often include false positives. Furthermore, they provide no information about what the binding region is (e.g. the binding epitope). We introduce a novel computational pipeline based on an AlphaFold2 (AF) Competition Assay (AF-CBA) to identify proteins that bind a target of interest from a pull-down experiment, along with the binding epitope. Our focus is on proteins that bind the Extraterminal (ET) domain of Bromo and Extraterminal domain (BET) proteins, but we also introduce nine additional systems to show transferability to other peptide-protein systems. We describe a series of limitations to the methodology based on intrinsic deficiencies to AF and AF-CBA, to help users identify scenarios where the approach will be most useful. Given the speed and accuracy of the methodology, we expect it to be generally applicable to facilitate target selection for experimental verification starting from high-throughput protein libraries. Table of Contents O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=173 SRC=\"FIGDIR/small/576374v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (41K): org.highwire.dtl.DTLVardef@1b29a98org.highwire.dtl.DTLVardef@c3aa72org.highwire.dtl.DTLVardef@1eb3e74org.highwire.dtl.DTLVardef@15166f9_HPS_FORMAT_FIGEXP M_FIG C_FIG\nDiffraction minima resolve point scatterers at tiny fractions (1/80) of the wavelength\nAuthors: Hensel, T. A.; Wirth, J. O.; Hell, S. W.\nScore: 11.5, Published: 2024-01-26 DOI: 10.1101/2024.01.24.576982\nDiscerning two or more identical and constantly scattering point sources using freely propagating waves is thought to be limited by diffraction. Here we show both theoretically and experimentally that by employing a diffraction minimum rather than a maximum for resolution, a given number of point scatterers can be discerned at tiny fractions of the employed wavelength. Specifically, we identify an 8 nm distance between two constantly emitting (non-blinking, non-switchable) fluorescent molecules, corresponding to 1/80 of the wavelength. Moreover, we show that contrary to naive expectations, the measurement precision improves with decreasing distance between the scatterers and with increased scatterer density, thus opening up the prospect of resolving clusters of (optical) point scatterers at tiny fractions of the wavelength.\nLocalising elements in single-particle reconstructions by REEL-EM: Reconstructed Electron Energy-Loss - Elemental Mapping\nAuthors: Pfeil-Gardiner, O.; Rosa, H. V. D.; Riedel, D.; Chen, Y. S.; Lorks, D.; Kukelhan, P.; Linck, M.; Muller, H.; Van Petegem, F.; Murphy, B. J.\nScore: 34.5, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575858\nFor structures determined by single particle cryo-EM, no technique currently exists for mapping elements to defined locations, leading to errors in the assignment of metals and other ions, cofactors, substrates, inhibitors, and lipids that play essential roles in activity and regulation. Elemental mapping in the electron microscope is well established for dose-tolerant samples but is challenging for biological samples, especially in a cryo-preserved state. Here, we combine electron energy-loss spectroscopy (EELS) with single-particle image processing to allow elemental mapping in cryo-preserved macromolecular complexes. Proof-of-principle data show that our method, REEL-EM, allows 3D reconstruction of EELS data, such that a high total electron dose is accumulated across many copies of a complex. Working with two test samples, we demonstrate that we can reliably localise abundant elements. We discuss the current limitations of the method and potential future developments.\nStructure, motion, and multiscale search of traveling networks\nAuthors: Cira, N. J.; Paull, M. L.; Sinha, S.; Zanini, F.; Ma, E. Y.; Riedel-Kruse, I. H.\nScore: 10.3, Published: 2024-01-20 DOI: 10.1101/2024.01.16.575883\nNetwork models are widely applied to describe connectivity and flow in diverse systems. In contrast, the fact that many connected systems move through space as the result of dynamic restructuring has received little attention. Therefore, we introduce the concept of traveling networks, and we analyze a tree-based model where the leaves are stochastically manipulated to grow, branch, and retract. We derive how these restructuring rates determine key attributes of network structure and motion, enabling a compact understanding of higher-level network behaviors such as multiscale search. These networks self-organize to the critical point between exponential growth and decay, allowing them to detect and respond to environmental signals with high sensitivity. Finally, we demonstrate how the traveling network concept applies to real-world systems, such as slime molds, the actin cytoskeleton, and human organizations, exemplifying how restructuring rules and rates in general can select for versatile search strategies in real or abstract spaces.\nBurkholderia cenocepacia epigenetic regulator M.BceJIV simultaneously engages two DNA recognition sequences for methylation\nAuthors: Quintana-Feliciano, R.; Kottur, J.; Ni, M.; Ghosh, R.; Salas-Estrada, L.; Rechkoblit, O.; Filizola, M.; Fang, G.; Aggarwal, A.\nScore: 8.0, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576384\nBurkholderia cenocepacia is an opportunistic and infective bacterium containing an orphan DNA methyltransferase (M.BceJIV) with roles in regulating gene expression and motility of the bacterium. M.BceJIV recognizes a GTWWAC motif (where W can be an adenine or a thymine) and methylates the N6 of the adenine at the fifth base position (GTWWAC). Here, we present a high-resolution crystal structure of M.BceJIV/DNA/sinefungin ternary complex and allied biochemical, computational, and thermodynamic analyses. Remarkably, the structure shows not one, but two DNA substrates bound to the M.BceJIV dimer, wherein each monomer contributes to the recognition of two recognition sequences. This unexpected mode of DNA binding and methylation has not been observed previously and sets a new precedent for a DNA methyltransferase. We also show that methylation at two recognition sequences occurs independently, and that GTWWAC motifs are enriched in intergenic regions of a strain of B. cenocepacias genome. We further computationally assess the interactions underlying the affinities of different ligands (SAM, SAH, and sinefungin) for M.BceJIV, as a step towards developing selective inhibitors for limiting B. cenocepacia infection.\nStructural features of heteromeric channels composed of CALHM2 and CALHM4 paralogs\nAuthors: Drozdzyk, K.; Peter, M.; Dutzler, R.\nScore: 6.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576238\nThe CALHM proteins constitute a family of large pore channels that contains six closely related paralogs in human. Two family members, CALHM1 and 3, have been associated with the release of ATP during taste sensation. Both proteins form heteromeric channels that activate at positive potential and decreased extracellular Ca2+ concentration. Although the structures of several family members displayed large oligomeric organizations of different size, their function has in most cases remained elusive. Our previous study has identified the paralogs CALHM2, 4 and 6 to be highly expressed in the placenta and defined their structural properties as membrane proteins exhibiting features of large pore channels with unknown activation properties (Drozdzyk et al., 2020). Here we investigated whether these placental paralogs would form heteromers and characterized heteromeric complexes consisting of CALHM2 and CALHM4 subunits using specific binders as fiducial markers. Both proteins assemble with different stoichiometries with the largest population containing CALHM2 as predominant component. In these oligomers, the subunits segregate and reside in their preferred conformation found in homomeric channels. Our study has thus revealed the properties that govern the formation of CALHM heteromers in a process of potential relevance in a cellular context.\nChaperone quality control in liquid-phase separated organelles\nAuthors: Scheidt, T.; Lemke, E. A.\nScore: 5.8, Published: 2024-01-23 DOI: 10.1101/2024.01.23.576883\nMolecular chaperones, central to the cellular proteostasis network, play an essential role in preventing the formation and proliferation of harmful aggregates associated with neurodegenerative diseases. Notably, for many intrinsically disordered proteins (IDPs), which are prone to form such damaging deposits, the formation of nano-clusters and phase separation into organelles prior to aggregation have been observed. The impact of molecular chaperones on such assemblies, remains unclear. In our study, we concentrated on the family of small heat shock proteins (sHsps), which are typically dynamic and form large oligomeric structures. While sHsps are mainly structured/folded proteins, they can undergo transient multivalent interactions, like many IDPs. Thus, sHsps might be a suitable regulator for vital and ubiquitous formation of membrane-less organelles in eukaryotic cells rich in IDPs and to inhibit aberrant aggregation. Here we show, using microfluidic diffusional sizing, that the formation of nano-clusters of FUS, associated with neurodegenerative diseases can be inhibited by the presence of sHsps. Furthermore, we identify that, depending on their assembly state, sHsps are capable of targeting specifically the interface between the dense droplet phase and the dilute phase not only of FUS but also of TDP-43, likely because the interface is the primary starting point for fibril formation or protein aggregation in general. Our findings emphasise the impact of molecular chaperones on maintaining the homeostasis of IDPs in the dilute and condensed phase. This could help to understand how chaperone dysregulation can influence aberrant protein association.\nSpectrally Resolved Localization Microscopy with Ultrahigh-Throughput\nAuthors: Batey, J. E.; Kim, G. W.; Yang, M.; Heffer, D. C.; Pott, E. D.; Giang, H.; Dong, B.\nScore: 5.2, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576082\nSingle-molecule localization microscopy (SMLM) has become a strong technique in the toolbox of chemists, biologists, physicists, and engineers in recent years for its unique ability to resolve characteristic features quickly and accurately in complex environments at the nanoscopic level. Multicolor super-resolution imaging has seen the greatest advancement among SMLM techniques, drastically improving the differentiation ability of nanostructures beyond the diffraction limit and increasing the resolution with which previously unresolvable structures are studied. However, current multicolor SMLM methodologies present low spatial resolution and throughput and require complex optical systems. Here, we overcome these drawbacks by developing an ultrahigh-throughput SMLM methodology that allows for ultrahigh throughput multicolor imaging at the nanoscopic level using a color glass filter. Our methodology can readily distinguish fluorophores of close spectral emission and achieves sub-10 nm localization and sub-5nm spectral precisions.\n",
  "wordCount" : "2111",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576053">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576053" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576053">
        <p class="paperTitle">PatternJ: an ImageJ toolset for the automated and quantitative analysis of regular spatial patterns found in sarcomeres, axons, somites, and more</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576053" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576053" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baheux Blin, M.; Loreau, V.; Schnorrer, F.; Mangeol, P.</p>
        <p class="info">Score: 21.1, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576053' target='https://doi.org/10.1101/2024.01.17.576053'> 10.1101/2024.01.17.576053</a></p>
        <p class="abstract">Regular spatial patterns are ubiquitous forms of organization in nature. In animals, regular patterns can be found from the cellular scale to the tissue scale, and from early stages of development to adulthood. To understand the formation of these patterns, how they form and mature, and how they are affected by perturbations, a precise quantitative description of the patterns is essential. However, accessible tools that offer in-depth analysis without the need for computational skills are lacking for biologists. Here we present PatternJ, a novel toolset to analyze regular pattern organizations precisely and automatically. This toolset, to be used with the popular imaging processing program ImageJ/Fiji, facilitates the extraction of key geometric features within and between pattern repeats. We validated PatternJ on simulated data and tested it on images of sarcomeres in insect muscles and cardiomyocytes, actin rings in neurons, and somites in zebrafish embryos obtained using confocal fluorescence microscopy, STORM, electron microscopy, and bright-field imaging. We show that the toolset delivers subpixel feature extraction reliably even with images of low signal-to-noise ratio. PatternJs straightforward use and functionalities make it valuable for various scientific fields requiring quantitative pattern analysis, including the sarcomere biology of muscles or the patterning of mammalian axons, speeding up discoveries with the bonus of high reproducibility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576231">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576231" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576231">
        <p class="paperTitle">Engineering CRBN for rapid identification of next generation binders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576231" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576231" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bailey, H. J.; Eisert, J.; Vollrath, J.; Leibrock, E.-M.; Kondratov, I.; Matviyuk, T.; Tolmachova, N.; Langer, J. D.; Wegener, A. A.; Sorrell, F. J.; Dikic, I.</p>
        <p class="info">Score: 15.5, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576231' target='https://doi.org/10.1101/2024.01.18.576231'> 10.1101/2024.01.18.576231</a></p>
        <p class="abstract">The majority of clinical degrader candidates utilize an immunomodulatory imide drug (IMiD)-based derivative, that directs their target substrate to the E3 ligase receptor Cereblon (CRBN), however, concomitant neo-substrate degradation by IMiDs often results in severe off-target effects. Further biophysical screening is needed to discover CRBN binders that might overcome these safety concerns, but the previously reported CRBN protein constructs suffer significant limitations that reduce their applicability as tools for biophysical assays making large-scale screening efforts a costly endeavor. This is due to the requirement of CRBN co-expression with DDB1 in insect cells to produce soluble protein that contains all the reported structural features necessary for proper compound interaction. Here, a near full-length human CRBN construct was designed that retains these required features, but for the first time allows the generation of highly homogenous and cost-efficient expression in E.coli, eliminating the need for DDB1 co-expression. We have extensively profiled the new construct against the existing CRBN constructs in binary and ternary degrader complexes, showing it to be the &#34;best of both worlds&#34; in terms of binding characteristics and ease of protein production. We then designed the Enamine CRBN focused IMiD library of 4480-compounds and demonstrated applicability of the new construct to high throughput screening by identifying novel CRBN binders with high potency and ligand efficiency. The new methods described here should serve as valuable tools for the development of the next generation IMiD-based degraders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576374">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576374" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576374">
        <p class="paperTitle">Sifting Through the Noise: A Computational Pipeline for Accurate Prioritization of Protein- Protein Binding Candidates in High-Throughput Protein Libraries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576374" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576374" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mondal, A.; Singh, B.; Felkner, R. H.; De Falco, A.; Swapna, G.; Montelione, G. T.; Roth, M.; Perez, A.</p>
        <p class="info">Score: 13.0, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576374' target='https://doi.org/10.1101/2024.01.20.576374'> 10.1101/2024.01.20.576374</a></p>
        <p class="abstract">Identifying the interactome for a protein of interest is challenging due to the large number of possible binders. High-throughput experimental approaches narrow down possible binding partners, but often include false positives. Furthermore, they provide no information about what the binding region is (e.g. the binding epitope). We introduce a novel computational pipeline based on an AlphaFold2 (AF) Competition Assay (AF-CBA) to identify proteins that bind a target of interest from a pull-down experiment, along with the binding epitope. Our focus is on proteins that bind the Extraterminal (ET) domain of Bromo and Extraterminal domain (BET) proteins, but we also introduce nine additional systems to show transferability to other peptide-protein systems. We describe a series of limitations to the methodology based on intrinsic deficiencies to AF and AF-CBA, to help users identify scenarios where the approach will be most useful. Given the speed and accuracy of the methodology, we expect it to be generally applicable to facilitate target selection for experimental verification starting from high-throughput protein libraries.

Table of Contents

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=173 SRC=&#34;FIGDIR/small/576374v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (41K):
org.highwire.dtl.DTLVardef@1b29a98org.highwire.dtl.DTLVardef@c3aa72org.highwire.dtl.DTLVardef@1eb3e74org.highwire.dtl.DTLVardef@15166f9_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.576982">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.576982" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.576982">
        <p class="paperTitle">Diffraction minima resolve point scatterers at tiny fractions (1/80) of the wavelength</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.576982" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.576982" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hensel, T. A.; Wirth, J. O.; Hell, S. W.</p>
        <p class="info">Score: 11.5, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.576982' target='https://doi.org/10.1101/2024.01.24.576982'> 10.1101/2024.01.24.576982</a></p>
        <p class="abstract">Discerning two or more identical and constantly scattering point sources using freely propagating waves is thought to be limited by diffraction. Here we show both theoretically and experimentally that by employing a diffraction minimum rather than a maximum for resolution, a given number of point scatterers can be discerned at tiny fractions of the employed wavelength. Specifically, we identify an 8 nm distance between two constantly emitting (non-blinking, non-switchable) fluorescent molecules, corresponding to 1/80 of the wavelength. Moreover, we show that contrary to naive expectations, the measurement precision improves with decreasing distance between the scatterers and with increased scatterer density, thus opening up the prospect of resolving clusters of (optical) point scatterers at tiny fractions of the wavelength.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.575858">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.575858" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.575858">
        <p class="paperTitle">Localising elements in single-particle reconstructions by REEL-EM: Reconstructed Electron Energy-Loss - Elemental Mapping</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.575858" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.575858" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pfeil-Gardiner, O.; Rosa, H. V. D.; Riedel, D.; Chen, Y. S.; Lorks, D.; Kukelhan, P.; Linck, M.; Muller, H.; Van Petegem, F.; Murphy, B. J.</p>
        <p class="info">Score: 34.5, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.575858' target='https://doi.org/10.1101/2024.01.18.575858'> 10.1101/2024.01.18.575858</a></p>
        <p class="abstract">For structures determined by single particle cryo-EM, no technique currently exists for mapping elements to defined locations, leading to errors in the assignment of metals and other ions, cofactors, substrates, inhibitors, and lipids that play essential roles in activity and regulation. Elemental mapping in the electron microscope is well established for dose-tolerant samples but is challenging for biological samples, especially in a cryo-preserved state. Here, we combine electron energy-loss spectroscopy (EELS) with single-particle image processing to allow elemental mapping in cryo-preserved macromolecular complexes. Proof-of-principle data show that our method, REEL-EM, allows 3D reconstruction of EELS data, such that a high total electron dose is accumulated across many copies of a complex. Working with two test samples, we demonstrate that we can reliably localise abundant elements. We discuss the current limitations of the method and potential future developments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575883">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575883" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575883">
        <p class="paperTitle">Structure, motion, and multiscale search of traveling networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575883" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575883" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cira, N. J.; Paull, M. L.; Sinha, S.; Zanini, F.; Ma, E. Y.; Riedel-Kruse, I. H.</p>
        <p class="info">Score: 10.3, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575883' target='https://doi.org/10.1101/2024.01.16.575883'> 10.1101/2024.01.16.575883</a></p>
        <p class="abstract">Network models are widely applied to describe connectivity and flow in diverse systems. In contrast, the fact that many connected systems move through space as the result of dynamic restructuring has received little attention. Therefore, we introduce the concept of  traveling networks, and we analyze a tree-based model where the leaves are stochastically manipulated to grow, branch, and retract. We derive how these restructuring rates determine key attributes of network structure and motion, enabling a compact understanding of higher-level network behaviors such as multiscale search. These networks self-organize to the critical point between exponential growth and decay, allowing them to detect and respond to environmental signals with high sensitivity. Finally, we demonstrate how the traveling network concept applies to real-world systems, such as slime molds, the actin cytoskeleton, and human organizations, exemplifying how restructuring rules and rates in general can select for versatile search strategies in real or abstract spaces.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576384">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576384" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576384">
        <p class="paperTitle">Burkholderia cenocepacia epigenetic regulator M.BceJIV simultaneously engages two DNA recognition sequences for methylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576384" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576384" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Quintana-Feliciano, R.; Kottur, J.; Ni, M.; Ghosh, R.; Salas-Estrada, L.; Rechkoblit, O.; Filizola, M.; Fang, G.; Aggarwal, A.</p>
        <p class="info">Score: 8.0, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576384' target='https://doi.org/10.1101/2024.01.20.576384'> 10.1101/2024.01.20.576384</a></p>
        <p class="abstract">Burkholderia cenocepacia is an opportunistic and infective bacterium containing an orphan DNA methyltransferase (M.BceJIV) with roles in regulating gene expression and motility of the bacterium. M.BceJIV recognizes a GTWWAC motif (where W can be an adenine or a thymine) and methylates the N6 of the adenine at the fifth base position (GTWWAC). Here, we present a high-resolution crystal structure of M.BceJIV/DNA/sinefungin ternary complex and allied biochemical, computational, and thermodynamic analyses. Remarkably, the structure shows not one, but two DNA substrates bound to the M.BceJIV dimer, wherein each monomer contributes to the recognition of two recognition sequences. This unexpected mode of DNA binding and methylation has not been observed previously and sets a new precedent for a DNA methyltransferase. We also show that methylation at two recognition sequences occurs independently, and that GTWWAC motifs are enriched in intergenic regions of a strain of B. cenocepacias genome. We further computationally assess the interactions underlying the affinities of different ligands (SAM, SAH, and sinefungin) for M.BceJIV, as a step towards developing selective inhibitors for limiting B. cenocepacia infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576238">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576238" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576238">
        <p class="paperTitle">Structural features of heteromeric channels composed of CALHM2 and CALHM4 paralogs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576238" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576238" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Drozdzyk, K.; Peter, M.; Dutzler, R.</p>
        <p class="info">Score: 6.3, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576238' target='https://doi.org/10.1101/2024.01.18.576238'> 10.1101/2024.01.18.576238</a></p>
        <p class="abstract">The CALHM proteins constitute a family of large pore channels that contains six closely related paralogs in human. Two family members, CALHM1 and 3, have been associated with the release of ATP during taste sensation. Both proteins form heteromeric channels that activate at positive potential and decreased extracellular Ca2&#43; concentration. Although the structures of several family members displayed large oligomeric organizations of different size, their function has in most cases remained elusive. Our previous study has identified the paralogs CALHM2, 4 and 6 to be highly expressed in the placenta and defined their structural properties as membrane proteins exhibiting features of large pore channels with unknown activation properties (Drozdzyk et al., 2020). Here we investigated whether these placental paralogs would form heteromers and characterized heteromeric complexes consisting of CALHM2 and CALHM4 subunits using specific binders as fiducial markers. Both proteins assemble with different stoichiometries with the largest population containing CALHM2 as predominant component. In these oligomers, the subunits segregate and reside in their preferred conformation found in homomeric channels. Our study has thus revealed the properties that govern the formation of CALHM heteromers in a process of potential relevance in a cellular context.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576883">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576883" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576883">
        <p class="paperTitle">Chaperone quality control in liquid-phase separated organelles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576883" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576883" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scheidt, T.; Lemke, E. A.</p>
        <p class="info">Score: 5.8, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576883' target='https://doi.org/10.1101/2024.01.23.576883'> 10.1101/2024.01.23.576883</a></p>
        <p class="abstract">Molecular chaperones, central to the cellular proteostasis network, play an essential role in preventing the formation and proliferation of harmful aggregates associated with neurodegenerative diseases. Notably, for many intrinsically disordered proteins (IDPs), which are prone to form such damaging deposits, the formation of nano-clusters and phase separation into organelles prior to aggregation have been observed. The impact of molecular chaperones on such assemblies, remains unclear. In our study, we concentrated on the family of small heat shock proteins (sHsps), which are typically dynamic and form large oligomeric structures. While sHsps are mainly structured/folded proteins, they can undergo transient multivalent interactions, like many IDPs. Thus, sHsps might be a suitable regulator for vital and ubiquitous formation of membrane-less organelles in eukaryotic cells rich in IDPs and to inhibit aberrant aggregation. Here we show, using microfluidic diffusional sizing, that the formation of nano-clusters of FUS, associated with neurodegenerative diseases can be inhibited by the presence of sHsps. Furthermore, we identify that, depending on their assembly state, sHsps are capable of targeting specifically the interface between the dense droplet phase and the dilute phase not only of FUS but also of TDP-43, likely because the interface is the primary starting point for fibril formation or protein aggregation in general. Our findings emphasise the impact of molecular chaperones on maintaining the homeostasis of IDPs in the dilute and condensed phase. This could help to understand how chaperone dysregulation can influence aberrant protein association.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576082">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576082" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576082">
        <p class="paperTitle">Spectrally Resolved Localization Microscopy with Ultrahigh-Throughput</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576082" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576082" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Batey, J. E.; Kim, G. W.; Yang, M.; Heffer, D. C.; Pott, E. D.; Giang, H.; Dong, B.</p>
        <p class="info">Score: 5.2, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576082' target='https://doi.org/10.1101/2024.01.17.576082'> 10.1101/2024.01.17.576082</a></p>
        <p class="abstract">Single-molecule localization microscopy (SMLM) has become a strong technique in the toolbox of chemists, biologists, physicists, and engineers in recent years for its unique ability to resolve characteristic features quickly and accurately in complex environments at the nanoscopic level. Multicolor super-resolution imaging has seen the greatest advancement among SMLM techniques, drastically improving the differentiation ability of nanostructures beyond the diffraction limit and increasing the resolution with which previously unresolvable structures are studied. However, current multicolor SMLM methodologies present low spatial resolution and throughput and require complex optical systems. Here, we overcome these drawbacks by developing an ultrahigh-throughput SMLM methodology that allows for ultrahigh throughput multicolor imaging at the nanoscopic level using a color glass filter. Our methodology can readily distinguish fluorophores of close spectral emission and achieves sub-10 nm localization and sub-5nm spectral precisions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
