<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Structural and quantum chemical basis for OCP-mediated quenching of phycobilisomes
Authors: Sauer, P. V.; Cupellini, L.; Sutter, M.; Bondanza, M.; Dominguez Martin, M. A.; Kirst, H.; Bina, D.; Koh, A. F.; Kotecha, A.; Greber, B. J.; Nogales, E.; Polivka, T.; Mennucci, B.; Kerfeld, C. A.
Score: 28.2, Published: 2023-09-30 DOI: 10.1101/2023.09.30.560311
Cyanobacteria employ large antenna complexes called phycobilisomes (PBS) for light harvesting. However, intense light triggers non-photochemical quenching, where the Orange Carotenoid Protein (OCP) binds to PBS, dissipating excess energy as heat.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Structural and quantum chemical basis for OCP-mediated quenching of phycobilisomes
Authors: Sauer, P. V.; Cupellini, L.; Sutter, M.; Bondanza, M.; Dominguez Martin, M. A.; Kirst, H.; Bina, D.; Koh, A. F.; Kotecha, A.; Greber, B. J.; Nogales, E.; Polivka, T.; Mennucci, B.; Kerfeld, C. A.
Score: 28.2, Published: 2023-09-30 DOI: 10.1101/2023.09.30.560311
Cyanobacteria employ large antenna complexes called phycobilisomes (PBS) for light harvesting. However, intense light triggers non-photochemical quenching, where the Orange Carotenoid Protein (OCP) binds to PBS, dissipating excess energy as heat." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Structural and quantum chemical basis for OCP-mediated quenching of phycobilisomes
Authors: Sauer, P. V.; Cupellini, L.; Sutter, M.; Bondanza, M.; Dominguez Martin, M. A.; Kirst, H.; Bina, D.; Koh, A. F.; Kotecha, A.; Greber, B. J.; Nogales, E.; Polivka, T.; Mennucci, B.; Kerfeld, C. A.
Score: 28.2, Published: 2023-09-30 DOI: 10.1101/2023.09.30.560311
Cyanobacteria employ large antenna complexes called phycobilisomes (PBS) for light harvesting. However, intense light triggers non-photochemical quenching, where the Orange Carotenoid Protein (OCP) binds to PBS, dissipating excess energy as heat."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Structural and quantum chemical basis for OCP-mediated quenching of phycobilisomes\nAuthors: Sauer, P. V.; Cupellini, L.; Sutter, M.; Bondanza, M.; Dominguez Martin, M. A.; Kirst, H.; Bina, D.; Koh, A. F.; Kotecha, A.; Greber, B. J.; Nogales, E.; Polivka, T.; Mennucci, B.; Kerfeld, C. A.\nScore: 28.2, Published: 2023-09-30 DOI: 10.1101/2023.09.30.560311\nCyanobacteria employ large antenna complexes called phycobilisomes (PBS) for light harvesting. However, intense light triggers non-photochemical quenching, where the Orange Carotenoid Protein (OCP) binds to PBS, dissipating excess energy as heat.",
  "keywords": [
    
  ],
  "articleBody": " Structural and quantum chemical basis for OCP-mediated quenching of phycobilisomes\nAuthors: Sauer, P. V.; Cupellini, L.; Sutter, M.; Bondanza, M.; Dominguez Martin, M. A.; Kirst, H.; Bina, D.; Koh, A. F.; Kotecha, A.; Greber, B. J.; Nogales, E.; Polivka, T.; Mennucci, B.; Kerfeld, C. A.\nScore: 28.2, Published: 2023-09-30 DOI: 10.1101/2023.09.30.560311\nCyanobacteria employ large antenna complexes called phycobilisomes (PBS) for light harvesting. However, intense light triggers non-photochemical quenching, where the Orange Carotenoid Protein (OCP) binds to PBS, dissipating excess energy as heat. The mechanism of efficiently transferring energy from phycocyanobilins in PBS to canthaxanthin in OCP remains insufficiently understood. Using advanced cryogenic-electron microscopy, we unveiled the OCP-PBS complex structure at 1.6-2.1 [A] resolution, showcasing its inherent flexibility. Employing multiscale quantum chemistry, we disclosed the quenching mechanism. Identifying key protein residues, we clarified how canthaxanthins transition dipole moment in its lowest-energy dark state becomes large enough for efficient energy transfer from phycocyanobilins. Our energy transfer model offers a detailed understanding of the atomic determinants of light harvesting regulation and antenna architecture in cyanobacteria. One sentence summaryHigh-resolution cryo-EM structure of the OCP-PBS complex reveals intrinsic motions and enables the atomic simulation of the quenching mechanism\nShort N-terminal disordered regions and the proline-rich domain are major regulators of phase transitions for full-length UBQLN1, UBQLN2 and UBQLN4\nAuthors: Dao, T. P.; Rajendran, A.; Galagedera, S. K. K.; Haws, W.; Castaneda, C. A.\nScore: 13.0, Published: 2023-09-29 DOI: 10.1101/2023.09.27.559790\nHighly homologous ubiquitin-binding shuttle proteins UBQLN1, UBQLN2 and UBQLN4 differ in both their specific protein quality control functions and their propensities to localize to stress-induced condensates, cellular aggregates and aggresomes. We previously showed that UBQLN2 phase separates in vitro, and that the phase separation propensities of UBQLN2 deletion constructs correlate with their ability to form condensates in cells. Here, we demonstrated that full-length UBQLN1, UBQLN2 and UBQLN4 exhibit distinct phase behaviors in vitro. Strikingly, UBQLN4 phase separates at a much lower saturation concentration than UBQLN1. However, neither UBQLN1 nor UBQLN4 phase separates with a strong temperature dependence, unlike UBQLN2. We determined that the temperature-dependent phase behavior of UBQLN2 stems from its unique proline-rich (Pxx) region, which is absent in the other UBQLNs. We found that the short N-terminal disordered regions of UBQLN1, UBQLN2 and UBQLN4 inhibit UBQLN phase separation via electrostatics interactions. Charge variants of the N-terminal regions exhibit altered phase behaviors. Consistent with the sensitivity of UBQLN phase separation to the composition of the N-terminal regions, epitope tags placed on the N-termini of the UBQLNs tune phase separation. Overall, our in vitro results have important implications for studies of UBQLNs in cells, including the identification of phase separation as a potential mechanism to distinguish the cellular roles of UBQLNs, and the need to apply caution when using epitope tags to prevent experimental artifacts. Highlights1) Full-length UBQLN1, UBQLN2 and UBQLN4 all exhibit LCST phase transitions but to different degrees. 2) The N-terminal region (N-terminal to the UBL) substantially regulates UBQLN phase separation. 3) Removal of the disease-associated proline-rich (Pxx) region in UBQLN2 removes the strong temperature dependence of UBQLN2 phase separation.\nSynthesis, Insertion and Characterization of SARS-CoV-2 Membrane Protein Within Lipid Bilayers\nAuthors: Zhang, Y.; Anbir, S.; Mctiernan, J.; Li, S.; Worcester, M.; Mishra, P.; Colvin, M. E.; Gopinathan, A.; Mohideen, U.; Zandi, R.; Kuhlman, T. E.\nScore: 6.4, Published: 2023-10-02 DOI: 10.1101/2023.09.30.560318\nSUMMARY/ABSTRACTThe membrane protein (M) is the most abundant structural protein in the SARS-CoV-2 virus and functions exclusively as a membrane-embedded homodimer. M protein is required for the formation of the SARS-CoV-2 virus particle and has been shown to interact with the Spike and Envelope proteins, as well as the RNA-packaging Nucleocapsid protein. Our knowledge of M protein is very limited due to its small size and challenges in expressing enough protein for use in structural and biophysical experiments. We report the successful development of a SUMO tag-based expression system to produce and purify significant quantities of M protein, and a method to insert the synthesized dimers into a suspended lipid membrane in a homogeneous orientation. We used AFM and Cryo-EM to image individual membrane-bound M protein dimers and characterize the configurations that they can assume. Our experimental results are in agreement with our molecular dynamics simulations which predict thinning of the membrane around the M protein and a propensity to induce local membrane curvature. Taken together, our results shed new light on M protein properties within the lipid bilayer and suggest mechanisms that could contribute to viral assembly and budding.\nTransition-based reweighting and neural rational inference analysis of effects of new psychoactive substances on cannabinoid receptors\nAuthors: Dutta, S.; Shukla, D.\nScore: 6.1, Published: 2023-10-02 DOI: 10.1101/2023.09.29.560261\nNew psychoactive substances (NPS) targeting cannabinoid receptor 1 (CB1) pose a significant threat to society as recreational abusive drugs that can avoid detection and have higher physiological side effects. These physiological side effects of NPS are shown to be linked to the higher {beta}-arrestin signaling. We hypothesize that the difference in conformational dynamics of the NPxxY motif causes the distinct downstream signaling of NPS contrary to the classical cannabinoids. To compare the dynamic effects of the NPS and classical cannabinoid binding on the NPxxY conformational ensemble, we simulate (un)binding process of NPS MDMB-Fubinaca and classical cannabinoid HU-210 from CB1 using unbiased and biased molecular dynamics simulations. The transition-based reweighing method (TRAM) is used to combine multi-ensemble simulations for the estimation of transition rates and underlying thermodynamics of (un)binding processes of ligands with nanomolar affinities, where it is more expensive to obtain local reversible sampling. Our analyses suggest that the ligands unbind from the receptors using the same pathway but by a different mechanism. Further analyses reveal higher conformational fluctuation in the NPxxY motif for NPS bound CB1, supporting our hypothesis. The observation is further validated using a Variational autoencoder (VAE) based on Neural rational inference, which shows higher dynamic allostery-based interactions between the binding pocket residues and NPxxY for NPS bound CB1. Hence, in this work, MD simulation, data-driven statistical methods, and deep learning point out the significant differences in (un)binding and downstream signaling of NPS and classical cannabinoids.\nA molecular switch in RCK2 triggers sodium-dependent activation of KNa1.1 (KCNT1) potassium channels.\nAuthors: Cole, B. A.; Kalli, A. C.; Pilati, N.; Muench, S. P.; Lippiat, J. D.\nScore: 6.5, Published: 2023-09-29 DOI: 10.1101/2023.09.28.559907\nThe Na+-activated K+ channel KNa1.1, encoded by the KCNT1 gene, is an important regulator of neuronal excitability. How intracellular Na+ ions bind and increase channel activity is not well understood. Analysis of KNa1.1 channel structures indicate that there is a large twisting of the {beta}N-Q loop in the intracellular RCK2 domain between the inactive and Na+-activated conformations, with a lysine (K885, human subunit numbering) close enough to form a salt bridge with aspartate (D839) in the Na+-activated state. Concurrently, an aspartate (D884) adjacent in the same loop adopts a position within 4 [A] of several acidic or polar residues. In carrying out mutagenesis and electrophysiology with human KNa1.1, we found alanine substitution of each of these residues resulted in almost negligible currents in the presence of up to 40 mM intracellular Na+. The exception was D884A, which resulted in constitutively active channels in both the presence and absence of intracellular Na+. Further mutagenesis of this site revealed an amino acid size-dependent effect. Substitutions at this site by an amino acid smaller than aspartate (D884V) also yielded constitutively active KNa1.1, D884I had Na+- dependence similar to wild-type KNa1.1, whilst increasing the side chain size larger than aspartate (D884E or D884F) yielded channels that could not be activated by up to 40 mM intracellular Na+. We conclude that Na+ binding results in a conformational change that accommodates D884 in the acid-rich pocket, which triggers further conformational changes in the RCK domains and channel activation. Statement of SignificanceSodium-activated potassium channels regulate neuronal excitability, and their dysfunction causes severe childhood disorders. Here, we identify a structural determinant in the intracellular domains that is responsible for triggering channel activation in response to sodium ion binding. An increase in the size of a particular amino acid renders the channel sodium-insensitive, whilst a decrease in size enables the channel to activate in the absence of sodium. This enhances our understanding of how this subclass of potassium channels respond to changes in the intracellular ionic environment. Furthermore, this may also further our understanding of the basis of human neurological disorders and their treatment.\nArchitecture and rearrangements of a sperm-specific Na+/H+ exchanger\nAuthors: Pal, K.; Chowdhury, S.\nScore: 5.5, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560940\nThe sperm-specific sodium hydrogen exchanger, SLC9C1, underlies hyperpolarization and cyclic nucleotide stimulated proton fluxes across sperm membranes and regulates their hyperactivated motility. SLC9C1 is the first known instance of an ion transporter that uses a canonical voltage-sensing domain (VSD) and an evolutionarily conserved cyclic nucleotide binding domain (CNBD) to influence the dynamics of its ion-exchange domain (ED). The structural organization of this tripartite transporter and the mechanisms whereby it integrates physical (membrane voltage) and chemical (cyclic nucleotide) cues are unknown. In this study, we use single particle cryo-electron microscopy to determine structures of a metazoan SLC9C1 in different conformational states. We find that the three structural domains are uniquely organized around a distinct ring-shaped scaffold that we call the allosteric ring domainor ARD. The ARD undergoes coupled proton-dependent rearrangements with the ED and acts as a signaling hub enabling allosteric communication between the key functional modules of sp9C1. We demonstrate that binding of cAMP causes large conformational changes in the cytoplasmic domains and disrupts key ARD-linked interfaces. We propose that these structural changes rescue the transmembrane domains from an auto-inhibited state and facilitate their functional dynamics. Our study provides a structural framework to understand and further probe electrochemical linkage in SLC9C1.\nSingle-molecule analysis reveals TDG exhibits multiple modes of linear diffusion to process 5-formylcytosine.\nAuthors: Schnable, B. L.; Schaich, M. A.; Roginskaya, V.; Leary, L. P.; Weaver, T. M.; Freudenthal, B. D.; Drohat, A. C.; Van Houten, B.\nScore: 5.4, Published: 2023-10-04 DOI: 10.1101/2023.10.04.560925\nDNA methylation plays a key role in epigenetics, with 60-80% of CpG sites containing 5-methylcytosine. Base excision repair (BER) is suggested to be the main pathway involved in active DNA demethylation. 5-formylctyosine (5fC), an oxidized moiety of methylated cytosine, is recognized and removed by thymine DNA glycosylase (TDG) to generate an abasic site. TDG binds avidly to abasic sites and is product inhibited. Using single molecule fluorescence experiments, we saw TDG interact with DNA containing 5fC specifically and non-specifically with lifetimes of 72.9 and 7.5 seconds, respectively. These results indicate that TDG cleaves the 5fC and stays bound for an extended time at the generated abasic site. Mean squared displacement analysis and a two color TDG experiment indicate that TDG exhibits multiple modes of linear diffusion, including hopping and sliding, in search of a lesion. The catalytically crippled variants, N140A and R275A/L, have a reduced binding lifetime compared to wild type and Mean Squared Displacement (MSD) analysis indicates that R275L/A moves on the DNA with a faster diffusivity. These results indicate that mutating R275, but not N140 interferes with damage recognition by TDG. Our findings give insight into how TDG searches for its lesions in long stretches of undamaged DNA.\niMAX FRET (Information Maximized FRET) for multipoint single-molecule structural analysis\nAuthors: Joshi, B. S.; de Lannoy, C. V.; Howarth, M.; Kim, S. H.; Joo, C.\nScore: 6.6, Published: 2023-09-29 DOI: 10.1101/2023.09.27.559821\nUnderstanding the structure of biomolecules is vital for deciphering their characteristics and roles in biological systems. While current structural analysis techniques like nuclear magnetic resonance and X-ray crystallography excel in many aspects, they fall short in capturing comprehensive single-molecule information. To address this limitation and to better capture the heterogeneity and dynamic range of biomolecular reactions, there is a need for single-molecule structural analysis tools. To achieve this, we introduce iMAX FRET, a one-pot FRET-based single-molecule method integrated with geometrical 3D reconstruction, offering comprehensive ab initio structural analysis. Through the stochastic exchange of fluorescent weak binders, iMAX FRET allows simultaneous assessment of multiple spatial coordinates on a biomolecule within a few minutes of time to generate distinct FRET fingerprints for 3D structural profiling. We demonstrate a mathematical approach for de novo structural prediction using iMAX data, opening avenues for native biomolecule analysis. Furthermore, this method facilitates the investigation of conformational changes in individual molecules, illuminating single-molecule structural dynamics. Our technique has the potential to emerge as a powerful approach to advance our understanding of biomolecular structures.\nMechanical imbalance between normal and cancer cells drives epithelial defence against cancer\nAuthors: Gupta, P.; Kayal, S.; Pothapragada, S. P.; Senapati, H. K.; Devendran, P.; Bi, D.; Das, T.\nScore: 5.8, Published: 2023-09-29 DOI: 10.1101/2023.09.27.559723\nCell competition enables normal wildtype cells of epithelial tissue to eliminate mutant cells expressing activated oncoproteins such as HRasV12. However, the driving force behind this fundamental form of epithelial defence against cancer remains enigmatic. Here, we employ tissue stress microscopy, theoretical modelling, and a new collective compressibility measurement technique called gel compression microscopy to unveil the mechanism governing competition. Stress microscopy reveals unique compressive stress experienced by the mutant cells, contrasting with predominantly tensile stress experienced by normal cells. A cell-based computer simulation of the process then predicts that this compressive stress arises out of a mechanical imbalance between two competing populations due to a difference in their collective compressibility and rigidity. Gel compression microscopy then empirically confirms that the mutant population is three times more compressible than the normal population. Mechanistically, this difference stems from the lower abundance and coupling of junctional E-cadherin in the mutant cells, which weakens cell-cell adhesions, rendering the mutant population more compressible. Taken together, our study elucidates the physical principle and underlying molecular mechanism driving cell competition in epithelial defence against cancer, opening new directions for mechanomedicine.\nDP/MM: A Hybrid Model for Zinc-Protein Interactions in Molecular Dynamics\nAuthors: Ding, Y.; Huang, J.\nScore: 4.5, Published: 2023-10-02 DOI: 10.1101/2023.09.29.560253\nZinc-containing proteins are essential to a variety of biological processes, yet accurately modeling them using classical force fields is hindered by complicated polarization and charge transfer effects. This study introduces DP/MM, a hybrid force field model that combines ab initio accuracy with MM-level efficiency for modeling zinc-protein interactions. The DP/MM scheme utilizes a deep potential model to correct the atomic forces of zinc ions and their coordinated atoms, elevating them from MM to QM levels of accuracy. The model is trained on the difference in atomic forces between MM and QM calculations across diverse zinc coordination groups. Simulations on a variety of zinccontaining proteins demonstrate that DP/MM faithfully reproduces their coordination geometry and structural characteristics, for example, the tetrahedral coordination structures for the Cys4 and the Cys3His1 groups. Furthermore, DP/MM is capable of handling exchangeable water molecules in the zinc coordination environment. With its unique blend of accuracy, efficiency, flexibility, and transferability, DP/MM not only serves as a valuable tool for studying zinc-containing proteins but also represents a pioneering approach that augments the growing landscape of machine learning potentials in molecular modeling.\n",
  "wordCount" : "2473",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560311">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560311" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560311">
        <p class="paperTitle">Structural and quantum chemical basis for OCP-mediated quenching of phycobilisomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560311" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560311" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sauer, P. V.; Cupellini, L.; Sutter, M.; Bondanza, M.; Dominguez Martin, M. A.; Kirst, H.; Bina, D.; Koh, A. F.; Kotecha, A.; Greber, B. J.; Nogales, E.; Polivka, T.; Mennucci, B.; Kerfeld, C. A.</p>
        <p class="info">Score: 28.2, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560311' target='https://doi.org/10.1101/2023.09.30.560311'> 10.1101/2023.09.30.560311</a></p>
        <p class="abstract">Cyanobacteria employ large antenna complexes called phycobilisomes (PBS) for light harvesting. However, intense light triggers non-photochemical quenching, where the Orange Carotenoid Protein (OCP) binds to PBS, dissipating excess energy as heat. The mechanism of efficiently transferring energy from phycocyanobilins in PBS to canthaxanthin in OCP remains insufficiently understood. Using advanced cryogenic-electron microscopy, we unveiled the OCP-PBS complex structure at 1.6-2.1 [A] resolution, showcasing its inherent flexibility. Employing multiscale quantum chemistry, we disclosed the quenching mechanism. Identifying key protein residues, we clarified how canthaxanthins transition dipole moment in its lowest-energy dark state becomes large enough for efficient energy transfer from phycocyanobilins. Our energy transfer model offers a detailed understanding of the atomic determinants of light harvesting regulation and antenna architecture in cyanobacteria.

One sentence summaryHigh-resolution cryo-EM structure of the OCP-PBS complex reveals intrinsic motions and enables the atomic simulation of the quenching mechanism</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559790">
        <p class="paperTitle">Short N-terminal disordered regions and the proline-rich domain are major regulators of phase transitions for full-length UBQLN1, UBQLN2 and UBQLN4</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dao, T. P.; Rajendran, A.; Galagedera, S. K. K.; Haws, W.; Castaneda, C. A.</p>
        <p class="info">Score: 13.0, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559790' target='https://doi.org/10.1101/2023.09.27.559790'> 10.1101/2023.09.27.559790</a></p>
        <p class="abstract">Highly homologous ubiquitin-binding shuttle proteins UBQLN1, UBQLN2 and UBQLN4 differ in both their specific protein quality control functions and their propensities to localize to stress-induced condensates, cellular aggregates and aggresomes. We previously showed that UBQLN2 phase separates in vitro, and that the phase separation propensities of UBQLN2 deletion constructs correlate with their ability to form condensates in cells. Here, we demonstrated that full-length UBQLN1, UBQLN2 and UBQLN4 exhibit distinct phase behaviors in vitro. Strikingly, UBQLN4 phase separates at a much lower saturation concentration than UBQLN1. However, neither UBQLN1 nor UBQLN4 phase separates with a strong temperature dependence, unlike UBQLN2. We determined that the temperature-dependent phase behavior of UBQLN2 stems from its unique proline-rich (Pxx) region, which is absent in the other UBQLNs. We found that the short N-terminal disordered regions of UBQLN1, UBQLN2 and UBQLN4 inhibit UBQLN phase separation via electrostatics interactions. Charge variants of the N-terminal regions exhibit altered phase behaviors. Consistent with the sensitivity of UBQLN phase separation to the composition of the N-terminal regions, epitope tags placed on the N-termini of the UBQLNs tune phase separation. Overall, our in vitro results have important implications for studies of UBQLNs in cells, including the identification of phase separation as a potential mechanism to distinguish the cellular roles of UBQLNs, and the need to apply caution when using epitope tags to prevent experimental artifacts.

Highlights1) Full-length UBQLN1, UBQLN2 and UBQLN4 all exhibit LCST phase transitions but to different degrees.

2) The N-terminal region (N-terminal to the UBL) substantially regulates UBQLN phase separation.

3) Removal of the disease-associated proline-rich (Pxx) region in UBQLN2 removes the strong temperature dependence of UBQLN2 phase separation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560318">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560318" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560318">
        <p class="paperTitle">Synthesis, Insertion and Characterization of SARS-CoV-2 Membrane Protein Within Lipid Bilayers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560318" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560318" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Y.; Anbir, S.; Mctiernan, J.; Li, S.; Worcester, M.; Mishra, P.; Colvin, M. E.; Gopinathan, A.; Mohideen, U.; Zandi, R.; Kuhlman, T. E.</p>
        <p class="info">Score: 6.4, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560318' target='https://doi.org/10.1101/2023.09.30.560318'> 10.1101/2023.09.30.560318</a></p>
        <p class="abstract">SUMMARY/ABSTRACTThe membrane protein (M) is the most abundant structural protein in the SARS-CoV-2 virus and functions exclusively as a membrane-embedded homodimer. M protein is required for the formation of the SARS-CoV-2 virus particle and has been shown to interact with the Spike and Envelope proteins, as well as the RNA-packaging Nucleocapsid protein. Our knowledge of M protein is very limited due to its small size and challenges in expressing enough protein for use in structural and biophysical experiments. We report the successful development of a SUMO tag-based expression system to produce and purify significant quantities of M protein, and a method to insert the synthesized dimers into a suspended lipid membrane in a homogeneous orientation. We used AFM and Cryo-EM to image individual membrane-bound M protein dimers and characterize the configurations that they can assume. Our experimental results are in agreement with our molecular dynamics simulations which predict thinning of the membrane around the M protein and a propensity to induce local membrane curvature. Taken together, our results shed new light on M protein properties within the lipid bilayer and suggest mechanisms that could contribute to viral assembly and budding.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560261">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560261" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560261">
        <p class="paperTitle">Transition-based reweighting and neural rational inference analysis of effects of new psychoactive substances on cannabinoid receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560261" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560261" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dutta, S.; Shukla, D.</p>
        <p class="info">Score: 6.1, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560261' target='https://doi.org/10.1101/2023.09.29.560261'> 10.1101/2023.09.29.560261</a></p>
        <p class="abstract">New psychoactive substances (NPS) targeting cannabinoid receptor 1 (CB1) pose a significant threat to society as recreational abusive drugs that can avoid detection and have higher physiological side effects. These physiological side effects of NPS are shown to be linked to the higher {beta}-arrestin signaling. We hypothesize that the difference in conformational dynamics of the NPxxY motif causes the distinct downstream signaling of NPS contrary to the classical cannabinoids. To compare the dynamic effects of the NPS and classical cannabinoid binding on the NPxxY conformational ensemble, we simulate (un)binding process of NPS MDMB-Fubinaca and classical cannabinoid HU-210 from CB1 using unbiased and biased molecular dynamics simulations. The transition-based reweighing method (TRAM) is used to combine multi-ensemble simulations for the estimation of transition rates and underlying thermodynamics of (un)binding processes of ligands with nanomolar affinities, where it is more expensive to obtain local reversible sampling. Our analyses suggest that the ligands unbind from the receptors using the same pathway but by a different mechanism. Further analyses reveal higher conformational fluctuation in the NPxxY motif for NPS bound CB1, supporting our hypothesis. The observation is further validated using a Variational autoencoder (VAE) based on Neural rational inference, which shows higher dynamic allostery-based interactions between the binding pocket residues and NPxxY for NPS bound CB1. Hence, in this work, MD simulation, data-driven statistical methods, and deep learning point out the significant differences in (un)binding and downstream signaling of NPS and classical cannabinoids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559907">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559907" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559907">
        <p class="paperTitle">A molecular switch in RCK2 triggers sodium-dependent activation of KNa1.1 (KCNT1) potassium channels.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559907" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559907" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cole, B. A.; Kalli, A. C.; Pilati, N.; Muench, S. P.; Lippiat, J. D.</p>
        <p class="info">Score: 6.5, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559907' target='https://doi.org/10.1101/2023.09.28.559907'> 10.1101/2023.09.28.559907</a></p>
        <p class="abstract">The Na&#43;-activated K&#43; channel KNa1.1, encoded by the KCNT1 gene, is an important regulator of neuronal excitability. How intracellular Na&#43; ions bind and increase channel activity is not well understood. Analysis of KNa1.1 channel structures indicate that there is a large twisting of the {beta}N-Q loop in the intracellular RCK2 domain between the inactive and Na&#43;-activated conformations, with a lysine (K885, human subunit numbering) close enough to form a salt bridge with aspartate (D839) in the Na&#43;-activated state. Concurrently, an aspartate (D884) adjacent in the same loop adopts a position within 4 [A] of several acidic or polar residues. In carrying out mutagenesis and electrophysiology with human KNa1.1, we found alanine substitution of each of these residues resulted in almost negligible currents in the presence of up to 40 mM intracellular Na&#43;. The exception was D884A, which resulted in constitutively active channels in both the presence and absence of intracellular Na&#43;. Further mutagenesis of this site revealed an amino acid size-dependent effect. Substitutions at this site by an amino acid smaller than aspartate (D884V) also yielded constitutively active KNa1.1, D884I had Na&#43;- dependence similar to wild-type KNa1.1, whilst increasing the side chain size larger than aspartate (D884E or D884F) yielded channels that could not be activated by up to 40 mM intracellular Na&#43;. We conclude that Na&#43; binding results in a conformational change that accommodates D884 in the acid-rich pocket, which triggers further conformational changes in the RCK domains and channel activation.

Statement of SignificanceSodium-activated potassium channels regulate neuronal excitability, and their dysfunction causes severe childhood disorders. Here, we identify a structural determinant in the intracellular domains that is responsible for triggering channel activation in response to sodium ion binding. An increase in the size of a particular amino acid renders the channel sodium-insensitive, whilst a decrease in size enables the channel to activate in the absence of sodium. This enhances our understanding of how this subclass of potassium channels respond to changes in the intracellular ionic environment. Furthermore, this may also further our understanding of the basis of human neurological disorders and their treatment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560940">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560940" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560940">
        <p class="paperTitle">Architecture and rearrangements of a sperm-specific Na&#43;/H&#43; exchanger</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560940" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560940" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pal, K.; Chowdhury, S.</p>
        <p class="info">Score: 5.5, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560940' target='https://doi.org/10.1101/2023.10.04.560940'> 10.1101/2023.10.04.560940</a></p>
        <p class="abstract">The sperm-specific sodium hydrogen exchanger, SLC9C1, underlies hyperpolarization and cyclic nucleotide stimulated proton fluxes across sperm membranes and regulates their hyperactivated motility. SLC9C1 is the first known instance of an ion transporter that uses a canonical voltage-sensing domain (VSD) and an evolutionarily conserved cyclic nucleotide binding domain (CNBD) to influence the dynamics of its ion-exchange domain (ED). The structural organization of this tripartite transporter and the mechanisms whereby it integrates physical (membrane voltage) and chemical (cyclic nucleotide) cues are unknown. In this study, we use single particle cryo-electron microscopy to determine structures of a metazoan SLC9C1 in different conformational states. We find that the three structural domains are uniquely organized around a distinct ring-shaped scaffold that we call the allosteric ring domainor ARD. The ARD undergoes coupled proton-dependent rearrangements with the ED and acts as a signaling hub enabling allosteric communication between the key functional modules of sp9C1. We demonstrate that binding of cAMP causes large conformational changes in the cytoplasmic domains and disrupts key ARD-linked interfaces. We propose that these structural changes rescue the transmembrane domains from an auto-inhibited state and facilitate their functional dynamics. Our study provides a structural framework to understand and further probe electrochemical linkage in SLC9C1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560925">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560925" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560925">
        <p class="paperTitle">Single-molecule analysis reveals TDG exhibits multiple modes of linear diffusion to process 5-formylcytosine.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560925" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560925" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schnable, B. L.; Schaich, M. A.; Roginskaya, V.; Leary, L. P.; Weaver, T. M.; Freudenthal, B. D.; Drohat, A. C.; Van Houten, B.</p>
        <p class="info">Score: 5.4, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560925' target='https://doi.org/10.1101/2023.10.04.560925'> 10.1101/2023.10.04.560925</a></p>
        <p class="abstract">DNA methylation plays a key role in epigenetics, with 60-80% of CpG sites containing 5-methylcytosine. Base excision repair (BER) is suggested to be the main pathway involved in active DNA demethylation. 5-formylctyosine (5fC), an oxidized moiety of methylated cytosine, is recognized and removed by thymine DNA glycosylase (TDG) to generate an abasic site. TDG binds avidly to abasic sites and is product inhibited. Using single molecule fluorescence experiments, we saw TDG interact with DNA containing 5fC specifically and non-specifically with lifetimes of 72.9 and 7.5 seconds, respectively. These results indicate that TDG cleaves the 5fC and stays bound for an extended time at the generated abasic site. Mean squared displacement analysis and a two color TDG experiment indicate that TDG exhibits multiple modes of linear diffusion, including hopping and sliding, in search of a lesion. The catalytically crippled variants, N140A and R275A/L, have a reduced binding lifetime compared to wild type and Mean Squared Displacement (MSD) analysis indicates that R275L/A moves on the DNA with a faster diffusivity. These results indicate that mutating R275, but not N140 interferes with damage recognition by TDG. Our findings give insight into how TDG searches for its lesions in long stretches of undamaged DNA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559821">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559821" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559821">
        <p class="paperTitle">iMAX FRET (Information Maximized FRET) for multipoint single-molecule structural analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559821" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559821" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joshi, B. S.; de Lannoy, C. V.; Howarth, M.; Kim, S. H.; Joo, C.</p>
        <p class="info">Score: 6.6, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559821' target='https://doi.org/10.1101/2023.09.27.559821'> 10.1101/2023.09.27.559821</a></p>
        <p class="abstract">Understanding the structure of biomolecules is vital for deciphering their characteristics and roles in biological systems. While current structural analysis techniques like nuclear magnetic resonance and X-ray crystallography excel in many aspects, they fall short in capturing comprehensive single-molecule information. To address this limitation and to better capture the heterogeneity and dynamic range of biomolecular reactions, there is a need for single-molecule structural analysis tools. To achieve this, we introduce iMAX FRET, a one-pot FRET-based single-molecule method integrated with geometrical 3D reconstruction, offering comprehensive ab initio structural analysis. Through the stochastic exchange of fluorescent weak binders, iMAX FRET allows simultaneous assessment of multiple spatial coordinates on a biomolecule within a few minutes of time to generate distinct FRET fingerprints for 3D structural profiling. We demonstrate a mathematical approach for de novo structural prediction using iMAX data, opening avenues for native biomolecule analysis. Furthermore, this method facilitates the investigation of conformational changes in individual molecules, illuminating single-molecule structural dynamics. Our technique has the potential to emerge as a powerful approach to advance our understanding of biomolecular structures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559723">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559723" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559723">
        <p class="paperTitle">Mechanical imbalance between normal and cancer cells drives epithelial defence against cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559723" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559723" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gupta, P.; Kayal, S.; Pothapragada, S. P.; Senapati, H. K.; Devendran, P.; Bi, D.; Das, T.</p>
        <p class="info">Score: 5.8, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559723' target='https://doi.org/10.1101/2023.09.27.559723'> 10.1101/2023.09.27.559723</a></p>
        <p class="abstract">Cell competition enables normal wildtype cells of epithelial tissue to eliminate mutant cells expressing activated oncoproteins such as HRasV12. However, the driving force behind this fundamental form of epithelial defence against cancer remains enigmatic. Here, we employ tissue stress microscopy, theoretical modelling, and a new collective compressibility measurement technique called gel compression microscopy to unveil the mechanism governing competition. Stress microscopy reveals unique compressive stress experienced by the mutant cells, contrasting with predominantly tensile stress experienced by normal cells. A cell-based computer simulation of the process then predicts that this compressive stress arises out of a mechanical imbalance between two competing populations due to a difference in their collective compressibility and rigidity. Gel compression microscopy then empirically confirms that the mutant population is three times more compressible than the normal population. Mechanistically, this difference stems from the lower abundance and coupling of junctional E-cadherin in the mutant cells, which weakens cell-cell adhesions, rendering the mutant population more compressible. Taken together, our study elucidates the physical principle and underlying molecular mechanism driving cell competition in epithelial defence against cancer, opening new directions for mechanomedicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560253">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560253" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560253">
        <p class="paperTitle">DP/MM: A Hybrid Model for Zinc-Protein Interactions in Molecular Dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560253" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560253" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ding, Y.; Huang, J.</p>
        <p class="info">Score: 4.5, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560253' target='https://doi.org/10.1101/2023.09.29.560253'> 10.1101/2023.09.29.560253</a></p>
        <p class="abstract">Zinc-containing proteins are essential to a variety of biological processes, yet accurately modeling them using classical force fields is hindered by complicated polarization and charge transfer effects. This study introduces DP/MM, a hybrid force field model that combines ab initio accuracy with MM-level efficiency for modeling zinc-protein interactions. The DP/MM scheme utilizes a deep potential model to correct the atomic forces of zinc ions and their coordinated atoms, elevating them from MM to QM levels of accuracy. The model is trained on the difference in atomic forces between MM and QM calculations across diverse zinc coordination groups. Simulations on a variety of zinccontaining proteins demonstrate that DP/MM faithfully reproduces their coordination geometry and structural characteristics, for example, the tetrahedral coordination structures for the Cys4 and the Cys3His1 groups. Furthermore, DP/MM is capable of handling exchangeable water molecules in the zinc coordination environment. With its unique blend of accuracy, efficiency, flexibility, and transferability, DP/MM not only serves as a valuable tool for studying zinc-containing proteins but also represents a pioneering approach that augments the growing landscape of machine learning potentials in molecular modeling.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
