<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="High-throughput computational discovery of inhibitory protein fragments with AlphaFold
Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.
Score: 81.6, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389
Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="High-throughput computational discovery of inhibitory protein fragments with AlphaFold
Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.
Score: 81.6, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389
Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="High-throughput computational discovery of inhibitory protein fragments with AlphaFold
Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.
Score: 81.6, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389
Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "High-throughput computational discovery of inhibitory protein fragments with AlphaFold\nAuthors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.\nScore: 81.6, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389\nPeptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells.",
  "keywords": [
    
  ],
  "articleBody": " High-throughput computational discovery of inhibitory protein fragments with AlphaFold\nAuthors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.\nScore: 81.6, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389\nPeptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells. However, it has thus far not been possible to predict de novo which of the many possible protein fragments bind their protein targets, let alone act as inhibitors. We have developed a computational method, FragFold, that employs AlphaFold to predict protein fragment binding to full-length protein targets in a high-throughput manner. Applying FragFold to thousands of fragments tiling across diverse proteins revealed peaks of predicted binding along each protein sequence. These predictions were compared with experimentally measured peaks of inhibitory activity in E. coli. We establish that our approach is a sensitive predictor of protein fragment function: Evaluating inhibitory fragments derived from known protein-protein interaction interfaces, we found 87% were predicted by FragFold to bind in a native-like mode. Across full protein sequences, 68% of FragFold-predicted binding peaks match experimentally measured inhibitory peaks. This is true even when the underlying inhibitory mechanism is unclear from existing structural data, and we find FragFold is able to predict novel binding modes for inhibitory fragments of unknown structure, explaining previous genetic and biochemical data for these fragments. The success rate of FragFold demonstrates that this computational approach should be broadly applicable for discovering inhibitory protein fragments across proteomes.\n1D confinement mimicking microvessel geometry controls pericyte shape and motility\nAuthors: Sagnimorte, A.; Adler, M. R.; de Tournemire, G.; Saez, P. J.; Gonzalez-Rodriguez, D.; Dessalles, C. A.; Babataheri, A.\nScore: 18.7, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572195\nPericytes are mural cells of the microvasculature, characterised by their elongated distinct shape. Pericytes span along the axis of the vessels they adhere to, therefore they experience extreme lateral and longitudinal confinement. Pericyte shape is key for their function during vascular regulation and their spatial distribution is established by cell migration during the embryonic stage and maintained through controlled motility in the adult. However, how pericyte morphology is associated with migration and function remains unknown. We use micropatterns to mimic pericyte adhesion to vessels, and to reproduce in vitro the shapes adopted by pericytes in vivo. We show that lateral confinement controls cell shape and produces in vivo-like phenotype. Modelling the pericyte as an incompressible linear elastic material predicts strain and shape of pericytes as a function of lateral confinement. Pericyte kinetics on both laterally confining lanes, and longitudinally constraining motifs is described by dry friction theory. Pericytes are capable of crossing gaps of different sizes. The percentage of crossings is correctly predicted by the likelihood of a fluctuating system to overcome an energy barrier. Our joint experimental and theoretical approach demonstrates the effect of in vivo-like geometrical confinement on pericyte morphology and migration which is accurately described by dry friction theory.\nHepatitis B virus neutralization with DNA origami nanoshells\nAuthors: Willner, E.-M.; Kolbe, F.; Momburg, F.; Protzer, U.; Dietz, H.\nScore: 17.6, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572526\nWe demonstrate the use of DNA origami to create virus-trapping nanoshells that efficiently neutralize hepatitis B virus (HBV) in cell culture. By modifying the shells with a synthetic monoclonal antibody that binds to the HBV envelope, the effective neutralization potency per antibody is increased by approximately 100 times compared to using free antibodies. The improvements in neutralizing the virus are attributed to two factors: first, the shells act as a physical barrier that block the virus from interacting with host cells; second, the multivalent binding of the antibodies inside the shells lead to stronger attachment to the trapped virus, a phenomenon known as avidity. Pre-incubation of shells with HBV and simultaneous addition of both components separately to cells lead to comparable levels of neutralization, indicating rapid trapping of the virions by the shells. Our study highlights the potential of the DNA shell system to rationally create novel antivirals using components that, when used individually, show little to no antiviral effectiveness.\nActive membrane deformations of a minimal synthetic cell\nAuthors: Sciortino, A.; Faizi, H.; Uplap, S.; Frechette, L.; Peterson, M. S.; Vlahovska, P.; Baskaran, A.; Hagan, M. F.; Bausch, A.\nScore: 41.7, Published: 2023-12-18 DOI: 10.1101/2023.12.18.571643\nBiological cells exhibit the remarkable ability to adapt their shape in response to their environment, a phenomenon that hinges on the intricate interplay between their deformable membrane and the underlying activity of their cytoskeleton. Yet, the precise physical mechanisms of this coupling remain mostly elusive. Here, we introduce a synthetic cell model, comprised of an active cytoskeletal network of microtubules, crosslinkers and molecular motors encapsulated inside giant vesicles. Remarkably, these active vesicles exhibit large shape fluctuations and life-like morphing abilities. Active forces from the encapsulated cytoskeleton give rise to large-scale traveling membrane deformations. Quantitative analysis of membrane and microtubule fluctuations shows how the intricate coupling of confinement, membrane material properties and cytoskeletal forces yields fluctuation spectra whose characteristic scales in space and time are distinctly different from passive vesicles. We demonstrate how activity leads to uneven probability fluxes between fluctuation modes and hence sets the temporal scale of membrane fluctuations. Using simulations and theoretical modelling, we extend the classical approach to membrane fluctuations to active cytoskeleton-driven vesicles, highlighting the effect of correlated activity on the dynamics of membrane deformations and paving the way for quantitative descriptions of the shape-morphing ability typical of living systems.\nAccelerated simulations of RNA phase separation:a systematic study of non-redundant tandem repeats\nAuthors: Aierken, D.; Joseph, J. A.\nScore: 15.8, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573204\nRNA promotes phase separation of proteins to form biomolecular condensates. Notably, pure RNA systems composed of RNA repeat sequences can phase separate and form liquid-like droplets. Significantly, such RNA tandem repeats have been implicated in several neurological and neuromuscular disorders. Thus, investigating the principles governing RNA repeat condensation can provide insights into RNA repeat expansion disorders. However, it is still challenging to characterize the microscopic structures and dynamics of RNA condensates. To address this, we have implemented and optimized a coarse-grained model to study the phase behavior of RNA. Our simulations reproduce key experimental findings on RNA-RNA phase separation. Importantly, we achieve multifold speed up in the simulation time compared to previous work. Leveraging this efficiency, we study the phase separation propensity of all 20 non-redundant trinucleotide repeat sequences. Our study aligns with findings from in vivo experiments, emphasizing that canonical base pairing and G-U wobble pairs play a dominant role in influencing the condensation of RNA repeat sequences and regulating the composition of ensuing condensates. Our implementation of this RNA model should make it feasible to efficiently probe sequence phase behavior relationships for RNAs, and, when combined with coarse-grained protein models, elucidate the rules governing composition and dynamics of normal and aberrant biomolecular condensates.\nDe novo design of drug-binding proteins with predictable binding energy and specificity\nAuthors: Lu, L.; Gou, X.; Tan, S.; Mann, S.; Yang, H.; Zhong, X.; Gazgalis, D.; Valdiviezo, J.; Jo, H.; Wu, Y.; Diolaiti, M.; Ashworth, A.; Polizzi, N.; DeGrado, W.\nScore: 14.2, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573178\nThe de novo design of small-molecule-binding proteins has seen exciting recent progress; however, the ability to achieve exquisite affinity for binding small molecules while tuning specificity has not yet been demonstrated directly from computation. Here, we develop a computational procedure that results in the highest affinity binders to date with predetermined relative affinities, targeting a series of PARP1 inhibitors. Two of four designed proteins bound with affinities ranging from \u003c 5 nM to low M; in a predictable manner. X-ray crystal structures confirmed the accuracy of the designed protein-drug interactions. Molecular dynamics simulations informed the role of water in binding. Binding free-energy calculations performed directly on the designed models are in excellent agreement with the experimentally measured affinities, suggesting that the de novo design of small-molecule-binding proteins with tuned interaction energies is now feasible entirely from computation. We expect these methods to open many opportunities in biomedicine, including rapid sensor development, antidote design, and drug delivery vehicles.\nStructure of the human systemic RNAi defective transmembrane protein 1 (hSIDT1) reveals the conformational flexibility of its lipid binding domain\nAuthors: Navratna, V.; Kumar, A.; Rana, J. K.; Mosalaganti, S.\nScore: 13.6, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572875\nIn C. elegans, the inter-cellular transport of the small non-coding RNA that causes systemic RNA interference (RNAi) is mediated by the transmembrane protein SID1, encoded by the sid1 gene in the systemic RNA interference-defective (sid) loci. SID1 shares remarkable structural and sequence similarity with cholesterol uptake protein 1 (CHUP1), and they are classified together as members of the cholesterol uptake family (ChUP). Although systemic RNAi is not an evolutionarily conserved process, the sid gene products are found across the animal kingdom, suggesting a potential for the existence of other novel gene regulatory mechanisms mediated by small non-coding RNAs. Human homologs of sid gene products, hSIDT1 and hSIDT2, are believed to mediate contact-dependent lipophilic small non-coding dsRNA transport. Here, we report the structure of recombinant full-length human SIDT1. We find that the extra-cytosolic domain (ECD) of hSIDT1 adopts a double jelly roll fold, and the transmembrane domain (TMD) exists as two modules, a flexible lipid binding domain (LBD) and a rigid TMD core. Our structural analyses provide insights into the potential role of the lipid binding domain in the function of hSIDT1 and other cholesterol uptake (ChUP) family members.\nProtein search processes mediated by chromatin topology\nAuthors: Dutta, S.; R., A.; Padinhateeri, R.; Mitra, M. K.\nScore: 12.4, Published: 2023-12-22 DOI: 10.1101/2023.12.12.571394\nWe investigate the role of compaction of chromatin domains in modulating search kinetics of proteins. Collapsed conformations of chromatin, characterised by long loops which bring distant regions of the genome into contact, and manifested structurally as Topologically Associated Domains (TADs) affect search kinetics of DNA associated transcription factors and other proteins. In this study, we investigate the role of the compactness of chromatin on the dynamics of proteins using a minimal model. Using theory and simulations, we show that an optimal compaction exists for which the residence time of proteins on a chromatin-like polymer backbone is minimum. We then show that the fraction of unique sites visited also shows a similar non-monotonic behavior as a function of the polymer compaction. We extend these results to more detailed polymer models - using the Freely Rotating Chain model, a Lennard-Jones bead-spring polymer, and a soft polymer chain model, which approximates chromatin behavior. We show that our results continue to hold for these polymer models, with a minimum residence time at an optimum polymer compaction. Finally, we also analyse the dynamics of proteins on networks generated using experimental chromatin conformation data for several TADs extracted from human chromosomes. Our analysis suggests that TADs exist near this zone of optimality, indicating that chromatin conformations can play a crucial role in modulating protein search strategies.\nGSK3β phosphorylation catalyzes the aggregation of Tau into Alzheimer's disease-like amyloid strain\nAuthors: Chakraborty, P.; Ibanez de Opakua, A.; Purslow, J. A.; Fromm, S. A.; Chatterjee, D.; Zachrdla, M.; Puri, S.; Wolozin, B.; Zweckstetter, M.\nScore: 11.5, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572292\nThe pathological deposition of proteins is a hallmark of several devastating neurodegenerative diseases. These pathological deposits comprise aggregates of proteins that adopt distinct structures named strains. However, the molecular factors responsible for the formation of distinct aggregate strains are unknown. Here we show that the serine/threonine kinase GSK3{beta} catalyzes the aggregation of the protein tau into an Alzheimers disease-like amyloid strain. We demonstrate that phosphorylation by GSK3{beta}, but not by several other kinases, promotes the aggregation of full-length tau through enhanced phase separation into gel-like condensate structures. Cryo-electron microscopy further reveals that the amyloid fibrils formed by GSK3{beta}-phosphorylated tau adopt a fold comparable to that of paired helical filaments isolated from the brains of AD patients. Our results elucidate the intricate relationship between post-translational modification and the formation of tau strains in neurodegenerative diseases.\nRole of diffusion and reaction of the constituents in spreading of histone modification marks\nAuthors: Manivannan, V.; Inamdar, M. M.; Padinhateeri, R.\nScore: 10.8, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571420\nCells switch genes ON or OFF by altering the state of chromatin via histone modifications at specific regulatory locations along the chromatin polymer. These gene regulation processes are carried out by a network of reactions in which the histone marks spread to neighboring regions with the help of enzymes. In the literature, this spreading has been studied as a purely kinetic, non-diffusive process considering the interactions between neighboring nucleosomes. In this work, we go beyond this framework and study the spreading of modifications using a reaction-diffusion (RD) model accounting for the diffusion of the constituents. We quantitatively segregate the modification profiles generated from kinetic and RD models. The diffusion and degradation of enzymes set a natural length scale for limiting the domain size of modification spreading, and the resulting enzyme limitation is inherent in our model. We also demonstrate the emergence of confined modification domains without the explicit requirement of a nucleation site. We explore polymer compaction effects on spreading and show that single-cell domains may differ from averaged profiles. We find that the modification profiles from our model are comparable with existing H3K9me3 data of S. pombe.\n",
  "wordCount" : "2217",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572389" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572389">
        <p class="paperTitle">High-throughput computational discovery of inhibitory protein fragments with AlphaFold</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.</p>
        <p class="info">Score: 81.6, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572389' target='https://doi.org/10.1101/2023.12.19.572389'> 10.1101/2023.12.19.572389</a></p>
        <p class="abstract">Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells. However, it has thus far not been possible to predict de novo which of the many possible protein fragments bind their protein targets, let alone act as inhibitors. We have developed a computational method, FragFold, that employs AlphaFold to predict protein fragment binding to full-length protein targets in a high-throughput manner. Applying FragFold to thousands of fragments tiling across diverse proteins revealed peaks of predicted binding along each protein sequence. These predictions were compared with experimentally measured peaks of inhibitory activity in E. coli. We establish that our approach is a sensitive predictor of protein fragment function: Evaluating inhibitory fragments derived from known protein-protein interaction interfaces, we found 87% were predicted by FragFold to bind in a native-like mode. Across full protein sequences, 68% of FragFold-predicted binding peaks match experimentally measured inhibitory peaks. This is true even when the underlying inhibitory mechanism is unclear from existing structural data, and we find FragFold is able to predict novel binding modes for inhibitory fragments of unknown structure, explaining previous genetic and biochemical data for these fragments. The success rate of FragFold demonstrates that this computational approach should be broadly applicable for discovering inhibitory protein fragments across proteomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572195">
        <p class="paperTitle">1D confinement mimicking microvessel geometry controls pericyte shape and motility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sagnimorte, A.; Adler, M. R.; de Tournemire, G.; Saez, P. J.; Gonzalez-Rodriguez, D.; Dessalles, C. A.; Babataheri, A.</p>
        <p class="info">Score: 18.7, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572195' target='https://doi.org/10.1101/2023.12.20.572195'> 10.1101/2023.12.20.572195</a></p>
        <p class="abstract">Pericytes are mural cells of the microvasculature, characterised by their elongated distinct shape. Pericytes span along the axis of the vessels they adhere to, therefore they experience extreme lateral and longitudinal confinement. Pericyte shape is key for their function during vascular regulation and their spatial distribution is established by cell migration during the embryonic stage and maintained through controlled motility in the adult. However, how pericyte morphology is associated with migration and function remains unknown. We use micropatterns to mimic pericyte adhesion to vessels, and to reproduce in vitro the shapes adopted by pericytes in vivo. We show that lateral confinement controls cell shape and produces in vivo-like phenotype. Modelling the pericyte as an incompressible linear elastic material predicts strain and shape of pericytes as a function of lateral confinement. Pericyte kinetics on both laterally confining lanes, and longitudinally constraining motifs is described by dry friction theory. Pericytes are capable of crossing gaps of different sizes. The percentage of crossings is correctly predicted by the likelihood of a fluctuating system to overcome an energy barrier. Our joint experimental and theoretical approach demonstrates the effect of in vivo-like geometrical confinement on pericyte morphology and migration which is accurately described by dry friction theory.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572526">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572526" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572526">
        <p class="paperTitle">Hepatitis B virus neutralization with DNA origami nanoshells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572526" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572526" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Willner, E.-M.; Kolbe, F.; Momburg, F.; Protzer, U.; Dietz, H.</p>
        <p class="info">Score: 17.6, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572526' target='https://doi.org/10.1101/2023.12.20.572526'> 10.1101/2023.12.20.572526</a></p>
        <p class="abstract">We demonstrate the use of DNA origami to create virus-trapping nanoshells that efficiently neutralize hepatitis B virus (HBV) in cell culture. By modifying the shells with a synthetic monoclonal antibody that binds to the HBV envelope, the effective neutralization potency per antibody is increased by approximately 100 times compared to using free antibodies. The improvements in neutralizing the virus are attributed to two factors: first, the shells act as a physical barrier that block the virus from interacting with host cells; second, the multivalent binding of the antibodies inside the shells lead to stronger attachment to the trapped virus, a phenomenon known as avidity. Pre-incubation of shells with HBV and simultaneous addition of both components separately to cells lead to comparable levels of neutralization, indicating rapid trapping of the virions by the shells. Our study highlights the potential of the DNA shell system to rationally create novel antivirals using components that, when used individually, show little to no antiviral effectiveness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.571643">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.571643" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.571643">
        <p class="paperTitle">Active membrane deformations of a minimal synthetic cell</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.571643" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.571643" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sciortino, A.; Faizi, H.; Uplap, S.; Frechette, L.; Peterson, M. S.; Vlahovska, P.; Baskaran, A.; Hagan, M. F.; Bausch, A.</p>
        <p class="info">Score: 41.7, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.571643' target='https://doi.org/10.1101/2023.12.18.571643'> 10.1101/2023.12.18.571643</a></p>
        <p class="abstract">Biological cells exhibit the remarkable ability to adapt their shape in response to their environment, a phenomenon that hinges on the intricate interplay between their deformable membrane and the underlying activity of their cytoskeleton. Yet, the precise physical mechanisms of this coupling remain mostly elusive. Here, we introduce a synthetic cell model, comprised of an active cytoskeletal network of microtubules, crosslinkers and molecular motors encapsulated inside giant vesicles. Remarkably, these active vesicles exhibit large shape fluctuations and life-like morphing abilities. Active forces from the encapsulated cytoskeleton give rise to large-scale traveling membrane deformations. Quantitative analysis of membrane and microtubule fluctuations shows how the intricate coupling of confinement, membrane material properties and cytoskeletal forces yields fluctuation spectra whose characteristic scales in space and time are distinctly different from passive vesicles. We demonstrate how activity leads to uneven probability fluxes between fluctuation modes and hence sets the temporal scale of membrane fluctuations. Using simulations and theoretical modelling, we extend the classical approach to membrane fluctuations to active cytoskeleton-driven vesicles, highlighting the effect of correlated activity on the dynamics of membrane deformations and paving the way for quantitative descriptions of the shape-morphing ability typical of living systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573204">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573204" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573204">
        <p class="paperTitle">Accelerated simulations of RNA phase separation:a systematic study of non-redundant tandem repeats</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573204" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573204" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aierken, D.; Joseph, J. A.</p>
        <p class="info">Score: 15.8, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573204' target='https://doi.org/10.1101/2023.12.23.573204'> 10.1101/2023.12.23.573204</a></p>
        <p class="abstract">RNA promotes phase separation of proteins to form biomolecular condensates. Notably, pure RNA systems composed of RNA repeat sequences can phase separate and form liquid-like droplets. Significantly, such RNA tandem repeats have been implicated in several neurological and neuromuscular disorders. Thus, investigating the principles governing RNA repeat condensation can provide insights into RNA repeat expansion disorders. However, it is still challenging to characterize the microscopic structures and dynamics of RNA condensates. To address this, we have implemented and optimized a coarse-grained model to study the phase behavior of RNA. Our simulations reproduce key experimental findings on RNA-RNA phase separation. Importantly, we achieve multifold speed up in the simulation time compared to previous work. Leveraging this efficiency, we study the phase separation propensity of all 20 non-redundant trinucleotide repeat sequences. Our study aligns with findings from in vivo experiments, emphasizing that canonical base pairing and G-U wobble pairs play a dominant role in influencing the condensation of RNA repeat sequences and regulating the composition of ensuing condensates. Our implementation of this RNA model should make it feasible to efficiently probe sequence phase behavior relationships for RNAs, and, when combined with coarse-grained protein models, elucidate the rules governing composition and dynamics of normal and aberrant biomolecular condensates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573178">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573178" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573178">
        <p class="paperTitle">De novo design of drug-binding proteins with predictable binding energy and specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573178" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573178" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, L.; Gou, X.; Tan, S.; Mann, S.; Yang, H.; Zhong, X.; Gazgalis, D.; Valdiviezo, J.; Jo, H.; Wu, Y.; Diolaiti, M.; Ashworth, A.; Polizzi, N.; DeGrado, W.</p>
        <p class="info">Score: 14.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573178' target='https://doi.org/10.1101/2023.12.23.573178'> 10.1101/2023.12.23.573178</a></p>
        <p class="abstract">The de novo design of small-molecule-binding proteins has seen exciting recent progress; however, the ability to achieve exquisite affinity for binding small molecules while tuning specificity has not yet been demonstrated directly from computation. Here, we develop a computational procedure that results in the highest affinity binders to date with predetermined relative affinities, targeting a series of PARP1 inhibitors. Two of four designed proteins bound with affinities ranging from &lt; 5 nM to low M; in a predictable manner. X-ray crystal structures confirmed the accuracy of the designed protein-drug interactions. Molecular dynamics simulations informed the role of water in binding. Binding free-energy calculations performed directly on the designed models are in excellent agreement with the experimentally measured affinities, suggesting that the de novo design of small-molecule-binding proteins with tuned interaction energies is now feasible entirely from computation. We expect these methods to open many opportunities in biomedicine, including rapid sensor development, antidote design, and drug delivery vehicles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572875">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572875" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572875">
        <p class="paperTitle">Structure of the human systemic RNAi defective transmembrane protein 1 (hSIDT1) reveals the conformational flexibility of its lipid binding domain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572875" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572875" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Navratna, V.; Kumar, A.; Rana, J. K.; Mosalaganti, S.</p>
        <p class="info">Score: 13.6, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572875' target='https://doi.org/10.1101/2023.12.21.572875'> 10.1101/2023.12.21.572875</a></p>
        <p class="abstract">In C. elegans, the inter-cellular transport of the small non-coding RNA that causes systemic RNA interference (RNAi) is mediated by the transmembrane protein SID1, encoded by the sid1 gene in the systemic RNA interference-defective (sid) loci. SID1 shares remarkable structural and sequence similarity with cholesterol uptake protein 1 (CHUP1), and they are classified together as members of the cholesterol uptake family (ChUP). Although systemic RNAi is not an evolutionarily conserved process, the sid gene products are found across the animal kingdom, suggesting a potential for the existence of other novel gene regulatory mechanisms mediated by small non-coding RNAs. Human homologs of sid gene products, hSIDT1 and hSIDT2, are believed to mediate contact-dependent lipophilic small non-coding dsRNA transport. Here, we report the structure of recombinant full-length human SIDT1. We find that the extra-cytosolic domain (ECD) of hSIDT1 adopts a double jelly roll fold, and the transmembrane domain (TMD) exists as two modules, a flexible lipid binding domain (LBD) and a rigid TMD core. Our structural analyses provide insights into the potential role of the lipid binding domain in the function of hSIDT1 and other cholesterol uptake (ChUP) family members.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571394">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571394" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571394">
        <p class="paperTitle">Protein search processes mediated by chromatin topology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571394" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571394" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dutta, S.; R., A.; Padinhateeri, R.; Mitra, M. K.</p>
        <p class="info">Score: 12.4, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571394' target='https://doi.org/10.1101/2023.12.12.571394'> 10.1101/2023.12.12.571394</a></p>
        <p class="abstract">We investigate the role of compaction of chromatin domains in modulating search kinetics of proteins. Collapsed conformations of chromatin, characterised by long loops which bring distant regions of the genome into contact, and manifested structurally as Topologically Associated Domains (TADs) affect search kinetics of DNA associated transcription factors and other proteins. In this study, we investigate the role of the compactness of chromatin on the dynamics of proteins using a minimal model. Using theory and simulations, we show that an optimal compaction exists for which the residence time of proteins on a chromatin-like polymer backbone is minimum. We then show that the fraction of unique sites visited also shows a similar non-monotonic behavior as a function of the polymer compaction. We extend these results to more detailed polymer models - using the Freely Rotating Chain model, a Lennard-Jones bead-spring polymer, and a soft polymer chain model, which approximates chromatin behavior. We show that our results continue to hold for these polymer models, with a minimum residence time at an optimum polymer compaction. Finally, we also analyse the dynamics of proteins on networks generated using experimental chromatin conformation data for several TADs extracted from human chromosomes. Our analysis suggests that TADs exist near this zone of optimality, indicating that chromatin conformations can play a crucial role in modulating protein search strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572292">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572292" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572292">
        <p class="paperTitle">GSK3β phosphorylation catalyzes the aggregation of Tau into Alzheimer&#39;s disease-like amyloid strain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572292" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572292" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chakraborty, P.; Ibanez de Opakua, A.; Purslow, J. A.; Fromm, S. A.; Chatterjee, D.; Zachrdla, M.; Puri, S.; Wolozin, B.; Zweckstetter, M.</p>
        <p class="info">Score: 11.5, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572292' target='https://doi.org/10.1101/2023.12.19.572292'> 10.1101/2023.12.19.572292</a></p>
        <p class="abstract">The pathological deposition of proteins is a hallmark of several devastating neurodegenerative diseases. These pathological deposits comprise aggregates of proteins that adopt distinct structures named strains. However, the molecular factors responsible for the formation of distinct aggregate strains are unknown. Here we show that the serine/threonine kinase GSK3{beta} catalyzes the aggregation of the protein tau into an Alzheimers disease-like amyloid strain. We demonstrate that phosphorylation by GSK3{beta}, but not by several other kinases, promotes the aggregation of full-length tau through enhanced phase separation into gel-like condensate structures. Cryo-electron microscopy further reveals that the amyloid fibrils formed by GSK3{beta}-phosphorylated tau adopt a fold comparable to that of paired helical filaments isolated from the brains of AD patients. Our results elucidate the intricate relationship between post-translational modification and the formation of tau strains in neurodegenerative diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571420">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571420" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571420">
        <p class="paperTitle">Role of diffusion and reaction of the constituents in spreading of histone modification marks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571420" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571420" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manivannan, V.; Inamdar, M. M.; Padinhateeri, R.</p>
        <p class="info">Score: 10.8, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571420' target='https://doi.org/10.1101/2023.12.13.571420'> 10.1101/2023.12.13.571420</a></p>
        <p class="abstract">Cells switch genes ON or OFF by altering the state of chromatin via histone modifications at specific regulatory locations along the chromatin polymer. These gene regulation processes are carried out by a network of reactions in which the histone marks spread to neighboring regions with the help of enzymes. In the literature, this spreading has been studied as a purely kinetic, non-diffusive process considering the interactions between neighboring nucleosomes. In this work, we go beyond this framework and study the spreading of modifications using a reaction-diffusion (RD) model accounting for the diffusion of the constituents. We quantitatively segregate the modification profiles generated from kinetic and RD models. The diffusion and degradation of enzymes set a natural length scale for limiting the domain size of modification spreading, and the resulting enzyme limitation is inherent in our model. We also demonstrate the emergence of confined modification domains without the explicit requirement of a nucleation site. We explore polymer compaction effects on spreading and show that single-cell domains may differ from averaged profiles. We find that the modification profiles from our model are comparable with existing H3K9me3 data of S. pombe.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
