<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Engineering Modular and Tunable Single Molecule Sensors by Decoupling Sensing from Signal Output
Authors: Grabenhorst, L.; Pfeiffer, M.; Schinkel, T.; Kummerlin, M.; Maglic, J. B.; Bruggenthies, G. A.; Selbach, F.; Murr, A. T.; Tinnefeld, P.; Glembockyte, V.
Score: 14.5, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565795
Biosensors play key roles in medical research and diagnostics, but there currently is a lack of sensing platforms that combine easy adaptation to new targets, strategies to tune the response window to relevant analyte concentration ranges and allow for the incorporation of multiple sensing elements to benefit from multivalency.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Engineering Modular and Tunable Single Molecule Sensors by Decoupling Sensing from Signal Output
Authors: Grabenhorst, L.; Pfeiffer, M.; Schinkel, T.; Kummerlin, M.; Maglic, J. B.; Bruggenthies, G. A.; Selbach, F.; Murr, A. T.; Tinnefeld, P.; Glembockyte, V.
Score: 14.5, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565795
Biosensors play key roles in medical research and diagnostics, but there currently is a lack of sensing platforms that combine easy adaptation to new targets, strategies to tune the response window to relevant analyte concentration ranges and allow for the incorporation of multiple sensing elements to benefit from multivalency." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-15T10:40:26+00:00" />
<meta property="article:modified_time" content="2023-11-15T10:40:26+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Engineering Modular and Tunable Single Molecule Sensors by Decoupling Sensing from Signal Output
Authors: Grabenhorst, L.; Pfeiffer, M.; Schinkel, T.; Kummerlin, M.; Maglic, J. B.; Bruggenthies, G. A.; Selbach, F.; Murr, A. T.; Tinnefeld, P.; Glembockyte, V.
Score: 14.5, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565795
Biosensors play key roles in medical research and diagnostics, but there currently is a lack of sensing platforms that combine easy adaptation to new targets, strategies to tune the response window to relevant analyte concentration ranges and allow for the incorporation of multiple sensing elements to benefit from multivalency."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Engineering Modular and Tunable Single Molecule Sensors by Decoupling Sensing from Signal Output\nAuthors: Grabenhorst, L.; Pfeiffer, M.; Schinkel, T.; Kummerlin, M.; Maglic, J. B.; Bruggenthies, G. A.; Selbach, F.; Murr, A. T.; Tinnefeld, P.; Glembockyte, V.\nScore: 14.5, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565795\nBiosensors play key roles in medical research and diagnostics, but there currently is a lack of sensing platforms that combine easy adaptation to new targets, strategies to tune the response window to relevant analyte concentration ranges and allow for the incorporation of multiple sensing elements to benefit from multivalency.",
  "keywords": [
    
  ],
  "articleBody": " Engineering Modular and Tunable Single Molecule Sensors by Decoupling Sensing from Signal Output\nAuthors: Grabenhorst, L.; Pfeiffer, M.; Schinkel, T.; Kummerlin, M.; Maglic, J. B.; Bruggenthies, G. A.; Selbach, F.; Murr, A. T.; Tinnefeld, P.; Glembockyte, V.\nScore: 14.5, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565795\nBiosensors play key roles in medical research and diagnostics, but there currently is a lack of sensing platforms that combine easy adaptation to new targets, strategies to tune the response window to relevant analyte concentration ranges and allow for the incorporation of multiple sensing elements to benefit from multivalency. Utilizing a DNA origami nanostructure as a scaffold for arranging the different sensor components, we here propose an approach for the development of modular and tunable single-molecule sensors capable of detecting a variety of biomolecular targets such as nucleic acids, antibodies and restriction enzymes while offering mechanisms to tune the dynamic window, the specificity and the cooperativity of the sensor.\nCryoSTAR: Leveraging Structural Prior and Constraints for Cryo-EM Heterogeneous Reconstruction\nAuthors: Li, Y.; Zhou, Y.; Yuan, J.; Ye, F.; Gu, Q.\nScore: 51.0, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564872\nResolving conformational heterogeneity in cryo-electron microscopy (cryo-EM) datasets remains a significant challenge in structural biology. Previous methods have often been restricted to working exclusively on volumetric densities, neglecting the potential of incorporating any pre-existing structural knowledge as prior or constraints. In this paper, we present a novel methodology, cryoSTAR, that harnesses atomic model information as structural regularization to elucidate such heterogeneity. Our method uniquely outputs both coarse-grained models and density maps, showcasing the molecular conformational changes at different levels. Validated against four diverse experimental datasets, spanning large complexes, a membrane protein, and a small single-chain protein, our results consistently demonstrate an efficient and effective solution to conformational heterogeneity with minimal human bias. By integrating atomic model insights with cryo-EM data, cryoSTAR represents a meaningful step forward, paving the way for a deeper understanding of dynamic biological processes.1\nRescuing Off-Equilibrium Simulation Data through Dynamic Experimental Data with dynAMMo\nAuthors: Kolloff, C.; Olsson, S.\nScore: 12.3, Published: 2023-11-08 DOI: 10.1101/2023.05.23.541878\nLong-timescale behavior of proteins is fundamental to many biological processes. Molecular Dynamics (MD) simulations and biophysical experiments are often used to study protein dynamics. However, high computational demands of MD limit what timescales are feasible to study, often missing rare events, which are critical to explain experiments. On the other hand, experiments are limited by low resolution. We present dynamic Augmented Markov models (dynAMMo) to bridge the gap between these data and overcome their respective limitations. For the first time, dynAMMo enables the construction of mechanistic models of slow exchange processes that have been not observed in MD data by integrating dynamic experimental observables. As a consequence, dynAMMo allows us to bypass costly and extensive simulations, yet providing mechanistic insights of the system. Validated with controlled model systems and a well-studied protein, dynAMMo offers a new approach to quantitatively model protein dynamics on long timescales in an unprecedented manner.\nCryo-EM structure of Slo1 with the auxiliary γ1 subunit suggests mechanism of depolarization-independent activation\nAuthors: Redhardt, M.; Raunser, S.; Raisch, T.\nScore: 27.6, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565197\nMammalian Ca2+-dependent Slo K+ channels are expressed with {beta} and {gamma} auxiliary subunits that greatly influence voltage- and Ca2+-induced gating, thereby fundamentally altering the behavior of the channel. The four {gamma} subunits reduce the need for voltage-dependent activation, allowing Slo to open in the absence of an action potential. The mechanism of this activation has, however, remained elusive. Here, we present the cryo-EM structure of Slo1 in complex with {gamma}1/LRRC26, revealing how the transmembrane helix of {gamma}1 binds and presumably stabilizes the active conformation of the voltage-sensor domain. This effect is further enhanced by a polybasic stretch on the intracellular side of the membrane which locally changes the charge gradient across the membrane. Sequence differences explain why the four {gamma} subunits possess different activation efficiencies. Simultaneous binding of {gamma} and the unrelated {beta} subunits is structurally possible, as both binding sites do not overlap and the {gamma}1 LRR domains are partially flexible. Thus, our data provide a possible explanation for Slo1 regulation by {gamma} subunits, and furthermore suggest a novel mechanism of activation of voltage-gated ion channels by auxiliary subunits and add to the growing knowledge of their complex regulation.\n3D single-molecule super-resolution imaging of microfabricated fractal substrates for cell culture and self-referenced imaging\nAuthors: Cabriel, C.; Cordova-Castro, R. M.; Berenschot, E. J. W.; Davila-Lezama, A.; Pondman, K.; Le Gac, S.; Tas, N.; Susarrey-Arce, A.; Izeddin, I.\nScore: 3.1, Published: 2023-11-10 DOI: 10.1101/2023.11.07.566090\nSubstrate-mediated cell growth engineering has shown increasing interest in promoting in vitro alternatives to animal experiments, as it opens new possibilities for cell culture, proliferation, and differentiation. Still, the mechanisms by which physical cues drive the cell phenotype are not fully understood, hence the necessity to interrogate cell behaviour at the highest resolution. However, cell 3D high-resolution optical imaging on such substrates remains challenging due to their complexity, as well as axial calibration issues. This work uses multiscale 3D SiO2 substrates consisting of spatially arrayed octahedra features of a few micrometers to hundreds of nanometers. Through optimizations of the structures and optical imaging conditions, we demonstrate the potential of these 3D multiscale structures as calibration tools for 3D super-resolution microscopy. We use their intrinsic multiscale and self-referenced nature to simultaneously perform lateral and axial calibrations in 3D single-molecule localization microscopy (SMLM) and assess imaging resolutions. With this proof of feasibility, we then use these substrates as a platform for high-resolution bioimaging. As proof of concept, we cultivate human mesenchymal stem cells cells on them, revealing very different growth patterns compared to flat glass. Specifically, the spatial distribution of cytoskeleton proteins is vastly modified, as we demonstrate with 3D SMLM assessment.\nFormation and growth of co-culture tumour spheroids: new compartment-based mathematical models and experiments\nAuthors: Murphy, R. J.; Gunasingh, G.; Haass, N. K.; Simpson, M. J.\nScore: 14.3, Published: 2023-10-23 DOI: 10.1101/2022.12.21.521515\nCo-culture tumour spheroid experiments are routinely performed to investigate cancer progression and test anti-cancer therapies. Therefore, methods to quantitatively characterise and interpret coculture spheroid growth are of great interest. However, co-culture spheroid growth is complex. Multiple biological processes occur on overlapping timescales and different cell types within the spheroid may have different characteristics, such as differing proliferation rates or responses to nutrient availability. At present there is no standard, widely-accepted mathematical model of such complex spatio-temporal growth processes. Typical approaches to analyse these experiments focus on the late-time temporal evolution of spheroid size and overlook early-time spheroid formation, spheroid structure and geometry. Here, using a range of ordinary differential equation-based mathematical models and parameter estimation, we interpret new co-culture experimental data. We provide new biological insights about spheroid formation, growth, and structure. As part of this analysis we connect Greenspans seminal mathematical model to co-culture data for the first time. Furthermore, we generalise a class of compartment-based spheroid mathematical models that have previously been restricted to one population so they can be applied to multiple populations. As special cases of the general model, we explore multiple natural two population extensions to Greenspans seminal model and reveal biological mechanisms that can describe the internal dynamics of growing co-culture spheroids and those that cannot. This mathematical and statistical modelling-based framework is well-suited to analyse spheroids grown with multiple different cell types and the new class of mathematical models provide opportunities for further mathematical and biological insights.\nDilated cardiomyopathy mutation in beta-cardiac myosin enhances actin activation of the power stroke and phosphate release\nAuthors: Bodt, S. M.; Ge, J.; Ma, W.; Rasicci, D. V.; Desetty, R.; McCammon, J. A.; Yengo, C. M.\nScore: 3.0, Published: 2023-11-13 DOI: 10.1101/2023.11.10.566646\nInherited mutations in human beta-cardiac myosin (M2{beta}) can lead to severe forms of heart failure. The E525K mutation in M2{beta} is associated with dilated cardiomyopathy (DCM) and was found to stabilize the interacting heads motif (IHM) and autoinhibited super-relaxed (SRX) state in dimeric heavy meromyosin. However, in monomeric M2{beta} subfragment 1 (S1) we found that E525K enhances (3-fold) the maximum steady-state actin-activated ATPase activity (kcat) and decreases (6-fold) the actin concentration at which ATPase is one-half maximal (KATPase). We also found a 3 to 4-fold increase in the actin-activated power stroke and phosphate release rate constants at 30 M actin, which overall enhanced the duty ratio 3-fold. Loaded motility assays revealed that the enhanced intrinsic motor activity translates to increased ensemble force in M2{beta} S1. Glutamate 525, located near the actin binding region in the so-called activation loop, is highly conserved and predicted to form a salt-bridge with another conserved residue (lysine 484) in the relay helix. Enhanced sampling molecular dynamics simulations predict that the charge reversal mutation disrupts the E525-K484 salt-bridge, inducing conformations with a more flexible relay helix and a wide phosphate release tunnel. Our results highlight a highly conserved allosteric pathway associated with actin activation of the power stroke and phosphate release and suggest an important feature of the autoinhibited IHM is to prevent this region of myosin from interacting with actin. The ability of the E525K mutation to stabilize the IHM likely overrides the enhanced intrinsic motor properties, which may be key to triggering DCM pathogenesis.\nSingle protein molecules separation, tracking and counting in ultra-thin silicon channels\nAuthors: Ohayon, S.; Taib, L.; Verma, N. C.; Iarossi, M.; Bhattacharya, I.; Marom, B.; Huttner, D.; Meller, A.\nScore: 3.0, Published: 2023-11-13 DOI: 10.1101/2023.11.09.566381\nEmerging single-molecule protein sensing techniques are ushering in a transformative era in biomedical research. Nevertheless, challenges persist in realizing ultra-fast full-size protein sensing, including loss of molecular integrity due to protein fragmentation, biases introduced by antibodies affinity, identification of proteoforms and low throughputs. Here, we introduce a single-molecule method for parallel protein separation and tracking, yielding multi-dimensional molecular properties used for their identification. Proteins are tagged by dual amino-acid specific labels and are electrophoretically separated by their mass/charge in custom-designed silicon nano-channel. This approach allows us to analyze thousands of individual proteins within a few minutes by tracking their motion during the migration. We demonstrate the power of the method by quantifying a cytokine panel for host-response discrimination between viral and bacterial infections. Moreover, we show that two clinically-relevant splice isoforms of VEGF can be accurately quantified from human serum samples. Being non-destructive and compatible with full-length intact proteins, this method opens up new ways for antibody-free single protein molecule quantification.\nA family of bacterial actin homologues forms a 3-stranded tubular structure\nAuthors: Bergeron, J.; Kollman, J. M.\nScore: 2.5, Published: 2023-11-09 DOI: 10.1101/2023.11.07.565980\nThe cytoskeleton plays a critical role in the organization and movement of cells. In Eukaryotes, actin filaments polymerize into a highly conserved double-stranded linear filamentous structure in the presence of ATP, and disassemble upon ATP hydrolysis. Bacteria also possess actin-like proteins, that drive fundamental cellular function, including cell division, shape maintenance, and DNA segregation. Like eukaryotic actin, bacterial actins assemble dynamic on ATP binding. In general, longitudinal interactions between bacterial actin protomers along each strand are conserved with eukaryotic actin, but variation in interactions between strands gives rise to striking diversity of filament architectures. Here, we report a family of bacterial actins of unknown function, conserved amongst the Verrucomicrobiota phylum, which assembles into a unique tubular structure in the presence of nucleotides. A cryo-EM structure of the filaments reveals it is composed of three strands, unlike other described bacterial actin structures. This architecture provides new insights into the organization of actin-like filaments, and has implications for understanding the diversity and evolution of the bacterial cytoskeleton.\nCharacterisation of ligand gating, ion conduction and the ion selectivity mechanism in the endo-lysosomal ion channel hTPC2\nAuthors: Sahin, A. T.; Zachariae, U.\nScore: 8.5, Published: 2023-11-02 DOI: 10.1101/2023.11.01.565142\nTwo pore channels (TPCs) are two-fold symmetric endo-lysosomal cation channels forming important drug targets especially for antiviral drugs. They are activated by calcium, ligand binding, and membrane voltage, and to date, are the only ion channels shown to alter their ion selectivity depending on the type of bound ligand. However, despite their importance in the field, ligand activation of TPCs and the molecular mechanisms underlying their ion selectivity are still poorly understood. Here, we set out to elucidate the mechanistic basis for the ion selectivity of human TPC2 (hTPC2) and the molecular mechanism of ligand-induced channel activation by the lipid PI(3,5)P2. We performed all-atom in silico electrophysiology simulations to study Na+ and Ca2+ permeation across hTPC2 in real-time and to investigate the conformational changes induced by the presence or absence of bound PI(3,5)P2. Our findings reveal that hTPC2 adopts distinct structures depending on the presence of PI(3,5)P2 and elucidate the conformational transition pathways between these structures. Additionally, we examined the permeation mechanism, solvation states, and binding sites of ions during ion permeation through the pore. Our simulations reproduce the experimental observation that hTPC2 is more selective for Na+ over Ca2+ ions in the presence of PI(3,5)P2 and explain the mechanism of this ion selectivity. They highlight the importance of specific ion binding sites at the luminal channel entrance, the selectivity filter, and the central channel cavity for ion conduction, enabling a distant knock-on mechanism for efficient permeation of Na+ ions.\n",
  "wordCount" : "2165",
  "inLanguage": "en",
  "datePublished": "2023-11-15T10:40:26Z",
  "dateModified": "2023-11-15T10:40:26Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 15, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565795">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565795" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565795">
        <p class="paperTitle">Engineering Modular and Tunable Single Molecule Sensors by Decoupling Sensing from Signal Output</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565795" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565795" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grabenhorst, L.; Pfeiffer, M.; Schinkel, T.; Kummerlin, M.; Maglic, J. B.; Bruggenthies, G. A.; Selbach, F.; Murr, A. T.; Tinnefeld, P.; Glembockyte, V.</p>
        <p class="info">Score: 14.5, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565795' target='https://doi.org/10.1101/2023.11.06.565795'> 10.1101/2023.11.06.565795</a></p>
        <p class="abstract">Biosensors play key roles in medical research and diagnostics, but there currently is a lack of sensing platforms that combine easy adaptation to new targets, strategies to tune the response window to relevant analyte concentration ranges and allow for the incorporation of multiple sensing elements to benefit from multivalency. Utilizing a DNA origami nanostructure as a scaffold for arranging the different sensor components, we here propose an approach for the development of modular and tunable single-molecule sensors capable of detecting a variety of biomolecular targets such as nucleic acids, antibodies and restriction enzymes while offering mechanisms to tune the dynamic window, the specificity and the cooperativity of the sensor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564872">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564872" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564872">
        <p class="paperTitle">CryoSTAR: Leveraging Structural Prior and Constraints for Cryo-EM Heterogeneous Reconstruction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564872" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564872" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Zhou, Y.; Yuan, J.; Ye, F.; Gu, Q.</p>
        <p class="info">Score: 51.0, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564872' target='https://doi.org/10.1101/2023.10.31.564872'> 10.1101/2023.10.31.564872</a></p>
        <p class="abstract">Resolving conformational heterogeneity in cryo-electron microscopy (cryo-EM) datasets remains a significant challenge in structural biology. Previous methods have often been restricted to working exclusively on volumetric densities, neglecting the potential of incorporating any pre-existing structural knowledge as prior or constraints. In this paper, we present a novel methodology, cryoSTAR, that harnesses atomic model information as structural regularization to elucidate such heterogeneity. Our method uniquely outputs both coarse-grained models and density maps, showcasing the molecular conformational changes at different levels. Validated against four diverse experimental datasets, spanning large complexes, a membrane protein, and a small single-chain protein, our results consistently demonstrate an efficient and effective solution to conformational heterogeneity with minimal human bias. By integrating atomic model insights with cryo-EM data, cryoSTAR represents a meaningful step forward, paving the way for a deeper understanding of dynamic biological processes.1</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.23.541878">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.23.541878" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.23.541878">
        <p class="paperTitle">Rescuing Off-Equilibrium Simulation Data through Dynamic Experimental Data with dynAMMo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.23.541878" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.23.541878" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kolloff, C.; Olsson, S.</p>
        <p class="info">Score: 12.3, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.23.541878' target='https://doi.org/10.1101/2023.05.23.541878'> 10.1101/2023.05.23.541878</a></p>
        <p class="abstract">Long-timescale behavior of proteins is fundamental to many biological processes. Molecular Dynamics (MD) simulations and biophysical experiments are often used to study protein dynamics. However, high computational demands of MD limit what timescales are feasible to study, often missing rare events, which are critical to explain experiments. On the other hand, experiments are limited by low resolution. We present dynamic Augmented Markov models (dynAMMo) to bridge the gap between these data and overcome their respective limitations. For the first time, dynAMMo enables the construction of mechanistic models of slow exchange processes that have been not observed in MD data by integrating dynamic experimental observables. As a consequence, dynAMMo allows us to bypass costly and extensive simulations, yet providing mechanistic insights of the system. Validated with controlled model systems and a well-studied protein, dynAMMo offers a new approach to quantitatively model protein dynamics on long timescales in an unprecedented manner.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565197">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565197" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565197">
        <p class="paperTitle">Cryo-EM structure of Slo1 with the auxiliary γ1 subunit suggests mechanism of depolarization-independent activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565197" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565197" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Redhardt, M.; Raunser, S.; Raisch, T.</p>
        <p class="info">Score: 27.6, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565197' target='https://doi.org/10.1101/2023.11.01.565197'> 10.1101/2023.11.01.565197</a></p>
        <p class="abstract">Mammalian Ca2&#43;-dependent Slo K&#43; channels are expressed with {beta} and {gamma} auxiliary subunits that greatly influence voltage- and Ca2&#43;-induced gating, thereby fundamentally altering the behavior of the channel. The four {gamma} subunits reduce the need for voltage-dependent activation, allowing Slo to open in the absence of an action potential. The mechanism of this activation has, however, remained elusive. Here, we present the cryo-EM structure of Slo1 in complex with {gamma}1/LRRC26, revealing how the transmembrane helix of {gamma}1 binds and presumably stabilizes the active conformation of the voltage-sensor domain. This effect is further enhanced by a polybasic stretch on the intracellular side of the membrane which locally changes the charge gradient across the membrane. Sequence differences explain why the four {gamma} subunits possess different activation efficiencies. Simultaneous binding of {gamma} and the unrelated {beta} subunits is structurally possible, as both binding sites do not overlap and the {gamma}1 LRR domains are partially flexible. Thus, our data provide a possible explanation for Slo1 regulation by {gamma} subunits, and furthermore suggest a novel mechanism of activation of voltage-gated ion channels by auxiliary subunits and add to the growing knowledge of their complex regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566090">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566090" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566090">
        <p class="paperTitle">3D single-molecule super-resolution imaging of microfabricated fractal substrates for cell culture and self-referenced imaging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566090" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566090" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cabriel, C.; Cordova-Castro, R. M.; Berenschot, E. J. W.; Davila-Lezama, A.; Pondman, K.; Le Gac, S.; Tas, N.; Susarrey-Arce, A.; Izeddin, I.</p>
        <p class="info">Score: 3.1, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566090' target='https://doi.org/10.1101/2023.11.07.566090'> 10.1101/2023.11.07.566090</a></p>
        <p class="abstract">Substrate-mediated cell growth engineering has shown increasing interest in promoting in vitro alternatives to animal experiments, as it opens new possibilities for cell culture, proliferation, and differentiation. Still, the mechanisms by which physical cues drive the cell phenotype are not fully understood, hence the necessity to interrogate cell behaviour at the highest resolution. However, cell 3D high-resolution optical imaging on such substrates remains challenging due to their complexity, as well as axial calibration issues. This work uses multiscale 3D SiO2 substrates consisting of spatially arrayed octahedra features of a few micrometers to hundreds of nanometers. Through optimizations of the structures and optical imaging conditions, we demonstrate the potential of these 3D multiscale structures as calibration tools for 3D super-resolution microscopy. We use their intrinsic multiscale and self-referenced nature to simultaneously perform lateral and axial calibrations in 3D single-molecule localization microscopy (SMLM) and assess imaging resolutions. With this proof of feasibility, we then use these substrates as a platform for high-resolution bioimaging. As proof of concept, we cultivate human mesenchymal stem cells cells on them, revealing very different growth patterns compared to flat glass. Specifically, the spatial distribution of cytoskeleton proteins is vastly modified, as we demonstrate with 3D SMLM assessment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.12.21.521515">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.12.21.521515" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.12.21.521515">
        <p class="paperTitle">Formation and growth of co-culture tumour spheroids: new compartment-based mathematical models and experiments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.12.21.521515" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.12.21.521515" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Murphy, R. J.; Gunasingh, G.; Haass, N. K.; Simpson, M. J.</p>
        <p class="info">Score: 14.3, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.12.21.521515' target='https://doi.org/10.1101/2022.12.21.521515'> 10.1101/2022.12.21.521515</a></p>
        <p class="abstract">Co-culture tumour spheroid experiments are routinely performed to investigate cancer progression and test anti-cancer therapies. Therefore, methods to quantitatively characterise and interpret coculture spheroid growth are of great interest. However, co-culture spheroid growth is complex. Multiple biological processes occur on overlapping timescales and different cell types within the spheroid may have different characteristics, such as differing proliferation rates or responses to nutrient availability. At present there is no standard, widely-accepted mathematical model of such complex spatio-temporal growth processes. Typical approaches to analyse these experiments focus on the late-time temporal evolution of spheroid size and overlook early-time spheroid formation, spheroid structure and geometry. Here, using a range of ordinary differential equation-based mathematical models and parameter estimation, we interpret new co-culture experimental data. We provide new biological insights about spheroid formation, growth, and structure. As part of this analysis we connect Greenspans seminal mathematical model to co-culture data for the first time. Furthermore, we generalise a class of compartment-based spheroid mathematical models that have previously been restricted to one population so they can be applied to multiple populations. As special cases of the general model, we explore multiple natural two population extensions to Greenspans seminal model and reveal biological mechanisms that can describe the internal dynamics of growing co-culture spheroids and those that cannot. This mathematical and statistical modelling-based framework is well-suited to analyse spheroids grown with multiple different cell types and the new class of mathematical models provide opportunities for further mathematical and biological insights.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566646">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566646" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566646">
        <p class="paperTitle">Dilated cardiomyopathy mutation in beta-cardiac myosin enhances actin activation of the power stroke and phosphate release</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566646" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566646" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bodt, S. M.; Ge, J.; Ma, W.; Rasicci, D. V.; Desetty, R.; McCammon, J. A.; Yengo, C. M.</p>
        <p class="info">Score: 3.0, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566646' target='https://doi.org/10.1101/2023.11.10.566646'> 10.1101/2023.11.10.566646</a></p>
        <p class="abstract">Inherited mutations in human beta-cardiac myosin (M2{beta}) can lead to severe forms of heart failure. The E525K mutation in M2{beta} is associated with dilated cardiomyopathy (DCM) and was found to stabilize the interacting heads motif (IHM) and autoinhibited super-relaxed (SRX) state in dimeric heavy meromyosin. However, in monomeric M2{beta} subfragment 1 (S1) we found that E525K enhances (3-fold) the maximum steady-state actin-activated ATPase activity (kcat) and decreases (6-fold) the actin concentration at which ATPase is one-half maximal (KATPase). We also found a 3 to 4-fold increase in the actin-activated power stroke and phosphate release rate constants at 30 M actin, which overall enhanced the duty ratio 3-fold. Loaded motility assays revealed that the enhanced intrinsic motor activity translates to increased ensemble force in M2{beta} S1. Glutamate 525, located near the actin binding region in the so-called activation loop, is highly conserved and predicted to form a salt-bridge with another conserved residue (lysine 484) in the relay helix. Enhanced sampling molecular dynamics simulations predict that the charge reversal mutation disrupts the E525-K484 salt-bridge, inducing conformations with a more flexible relay helix and a wide phosphate release tunnel. Our results highlight a highly conserved allosteric pathway associated with actin activation of the power stroke and phosphate release and suggest an important feature of the autoinhibited IHM is to prevent this region of myosin from interacting with actin. The ability of the E525K mutation to stabilize the IHM likely overrides the enhanced intrinsic motor properties, which may be key to triggering DCM pathogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566381">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566381" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566381">
        <p class="paperTitle">Single protein molecules separation, tracking and counting in ultra-thin silicon channels</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566381" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566381" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ohayon, S.; Taib, L.; Verma, N. C.; Iarossi, M.; Bhattacharya, I.; Marom, B.; Huttner, D.; Meller, A.</p>
        <p class="info">Score: 3.0, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566381' target='https://doi.org/10.1101/2023.11.09.566381'> 10.1101/2023.11.09.566381</a></p>
        <p class="abstract">Emerging single-molecule protein sensing techniques are ushering in a transformative era in biomedical research. Nevertheless, challenges persist in realizing ultra-fast full-size protein sensing, including loss of molecular integrity due to protein fragmentation, biases introduced by antibodies affinity, identification of proteoforms and low throughputs. Here, we introduce a single-molecule method for parallel protein separation and tracking, yielding multi-dimensional molecular properties used for their identification. Proteins are tagged by dual amino-acid specific labels and are electrophoretically separated by their mass/charge in custom-designed silicon nano-channel. This approach allows us to analyze thousands of individual proteins within a few minutes by tracking their motion during the migration. We demonstrate the power of the method by quantifying a cytokine panel for host-response discrimination between viral and bacterial infections. Moreover, we show that two clinically-relevant splice isoforms of VEGF can be accurately quantified from human serum samples. Being non-destructive and compatible with full-length intact proteins, this method opens up new ways for antibody-free single protein molecule quantification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565980">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565980" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565980">
        <p class="paperTitle">A family of bacterial actin homologues forms a 3-stranded tubular structure</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565980" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565980" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bergeron, J.; Kollman, J. M.</p>
        <p class="info">Score: 2.5, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565980' target='https://doi.org/10.1101/2023.11.07.565980'> 10.1101/2023.11.07.565980</a></p>
        <p class="abstract">The cytoskeleton plays a critical role in the organization and movement of cells. In Eukaryotes, actin filaments polymerize into a highly conserved double-stranded linear filamentous structure in the presence of ATP, and disassemble upon ATP hydrolysis. Bacteria also possess actin-like proteins, that drive fundamental cellular function, including cell division, shape maintenance, and DNA segregation. Like eukaryotic actin, bacterial actins assemble dynamic on ATP binding. In general, longitudinal interactions between bacterial actin protomers along each strand are conserved with eukaryotic actin, but variation in interactions between strands gives rise to striking diversity of filament architectures. Here, we report a family of bacterial actins of unknown function, conserved amongst the Verrucomicrobiota phylum, which assembles into a unique tubular structure in the presence of nucleotides. A cryo-EM structure of the filaments reveals it is composed of three strands, unlike other described bacterial actin structures. This architecture provides new insights into the organization of actin-like filaments, and has implications for understanding the diversity and evolution of the bacterial cytoskeleton.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565142">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565142" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565142">
        <p class="paperTitle">Characterisation of ligand gating, ion conduction and the ion selectivity mechanism in the endo-lysosomal ion channel hTPC2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565142" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565142" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sahin, A. T.; Zachariae, U.</p>
        <p class="info">Score: 8.5, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565142' target='https://doi.org/10.1101/2023.11.01.565142'> 10.1101/2023.11.01.565142</a></p>
        <p class="abstract">Two pore channels (TPCs) are two-fold symmetric endo-lysosomal cation channels forming important drug targets especially for antiviral drugs. They are activated by calcium, ligand binding, and membrane voltage, and to date, are the only ion channels shown to alter their ion selectivity depending on the type of bound ligand. However, despite their importance in the field, ligand activation of TPCs and the molecular mechanisms underlying their ion selectivity are still poorly understood. Here, we set out to elucidate the mechanistic basis for the ion selectivity of human TPC2 (hTPC2) and the molecular mechanism of ligand-induced channel activation by the lipid PI(3,5)P2. We performed all-atom in silico electrophysiology simulations to study Na&#43; and Ca2&#43; permeation across hTPC2 in real-time and to investigate the conformational changes induced by the presence or absence of bound PI(3,5)P2. Our findings reveal that hTPC2 adopts distinct structures depending on the presence of PI(3,5)P2 and elucidate the conformational transition pathways between these structures. Additionally, we examined the permeation mechanism, solvation states, and binding sites of ions during ion permeation through the pore. Our simulations reproduce the experimental observation that hTPC2 is more selective for Na&#43; over Ca2&#43; ions in the presence of PI(3,5)P2 and explain the mechanism of this ion selectivity. They highlight the importance of specific ion binding sites at the luminal channel entrance, the selectivity filter, and the central channel cavity for ion conduction, enabling a distant knock-on mechanism for efficient permeation of Na&#43; ions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
