<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells
Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.
Score: 107.6, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799
Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells
Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.
Score: 107.6, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799
Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-27T10:37:02+00:00" />
<meta property="article:modified_time" content="2023-08-27T10:37:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells
Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.
Score: 107.6, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799
Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells\nAuthors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.\nScore: 107.6, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799\nAdvances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms.",
  "keywords": [
    
  ],
  "articleBody": " Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells\nAuthors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.\nScore: 107.6, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799\nAdvances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms. This method, cryoDRGN-ET, learns a deep generative model of 3D density maps directly from subtomogram tilt series images and can capture states diverse in both composition and conformation. We use this approach to reconstruct the in situ translation dynamics of prokaryotic ribosomes, and we reveal the distribution of functional states during translation elongation populated by S. cerevisiae ribosomes inside cells.\nTime-Resolved Cryo-EM Specimen Preparation with Single Millisecond Precision\nAuthors: Montano Romero, A.; Bonin, C.; Twomey, E. C.\nScore: 43.4, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554704\nMolecular structures can be determined in vitro and in situ with cryo-electron microscopy (cryo-EM). Specimen preparation is a major obstacle in cryo-EM. Typical sample preparation is orders of magnitude slower than biological processes. Time-resolved cryo-EM (TR-cryo-EM) can capture short-lived states. Here, we present Cryo-EM Sample Preparation with light-Activated Molecules (C-SPAM), an open-source, photochemistry-coupled device for TR-cryo-EM with single millisecond resolution, tunable timescales, and broad biological applications.\nSub-membrane actin rings compartmentalize the plasma membrane\nAuthors: Rentsch, J.; Bandstra, S.; Sezen, B.; Sigrist, P. S.; Bottanelli, F.; Schmerl, B.; Shoichet, S. A.; Noe, F.; Sadeghi, M.; Ewers, H.\nScore: 23.0, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554239\nThe compartmentalization of the plasma membrane is a fundamental feature of cells. The diffusivity of membrane proteins in the plane of the membrane is significantly lower in cells than observed in artificial membranes. This seems due to the sub-membranous actin cortex, but tractable assays to prove a direct dependence of membrane protein motion on actin filaments have been elusive. We recently showed that a periodic array of sub-membrane actin filaments that surround neuronal axons in ring-like structures in the axonal initial segment (AIS) confines membrane protein motion between them, but the local enrichment of ion channels may offer an alternative explanation for compartmentalization. Here we show using computational modeling that in contrast to actin rings, the dense array of ion channels in the AIS cannot mediate membrane protein confinement. We go on to show that indeed the actin rings are closely apposed to the plasma membrane, and that they confine membrane protein motion between them in a [~] 200 nm periodic pattern in progenitor-derived neuronal cells. We find that this compartmentalization is also detectable for inner-leaflet membrane proteins and multi-spanning receptors. Strikingly, several cell types with actin rings in their protrusions, like progenitor-derived astrocytes and oligodendrocytes, also exhibit lateral confinement of membrane proteins in a periodic pattern consistent with actin ring spacing. Actin ring-mediated membrane compartmentalization is thus not unique to neurons. Finally, we show that acute actin disruption in live progenitor-derived neuronal cells leads to a loss of membrane compartmentalization in areas that were compartmentalized before treatment. Taken together, we here develop a much-needed tractable experimental system for the investigation of membrane compartmentalization and show that actin rings compartmentalize the plasma membrane of cells.\nATP Mediates Phase Separation of Disordered Basic Proteins by Bridging Intermolecular Interaction Networks\nAuthors: Kota, D.; Prasad, R.; Zhou, H.-X.\nScore: 16.0, Published: 2023-08-20 DOI: 10.1101/2023.08.20.554035\nATP is an abundant molecule with crucial cellular roles as the energy currency and a building block of nucleic acids and for protein phosphorylation. Here we show that ATP mediates the phase separation of basic intrinsically disordered proteins (bIDPs). In the resulting condensates, ATP is highly concentrated (apparent partition coefficients at 200-5000) and serves as bridges between bIDP chains. These liquid-like droplets have some of the lowest interfacial tension ([~]25 pN/m) but high zero-shear viscosities (1-15 Pa s) due to the bridged protein networks, and yet their fusion has some of the highest speeds ([~]1 m/ms). The rapid fusion manifests extreme shear thinning, where the apparent viscosity is lower than zero-shear viscosity by over 100-fold, made possible by fast reformation of the ATP bridges. At still higher concentrations, ATP does not dissolve bIDP droplets but results in aggregates and fibrils.\nShocker - a molecular dynamics protocol and tool for accelerating and analyzing the effects of osmotic shocks\nAuthors: van Tilburg, M. P. A.; Marrink, S.-J.; Koenig, M.; Gruenewald, F.\nScore: 17.1, Published: 2023-08-18 DOI: 10.1101/2023.08.16.553535\nThe process of osmosis, a fundamental phenomenon in life, drives water through a semi-permeable membrane in response to a solute concentration gradient across this membrane. In vitro, osmotic shocks are often used to drive shape changes in lipid vesicles, for instance, to study fission events in the context of artificial cells. While experimental techniques provide a macroscopic picture of large-scale membrane remodeling processes, molecular dynamics (MD) simulations are a powerful tool to study membrane deformations at the molecular level. However, simulating an osmotic shock is a time-consuming process due to the slow water diffusion across the membrane, making it practically impossible to examine its effects in classic MD simulations. In this paper, we present Shocker, a Python-based MD tool for simulating the effects of an osmotic shock by selecting and relocating water particles across a membrane over the course of several pumping cycles. Although this method is primarily aimed at efficiently simulating volume changes of vesicles it can handle membrane tubes and double bilayer systems as well. Additionally, Shocker is force field independent and compatible with both coarse-grained and all-atom systems. We demonstrate that our tool is applicable to simulate both hypertonic and hypotonic osmotic shocks for a range of vesicular and bilamellar setups, including complex multi-component systems containing membrane proteins or crowded internal solutions.\nNMR measurements of transient low-populated tautomeric and anionic Watson-Crick-like G·T/U in RNA:DNA hybrids: Implications for the fidelity of transcription and CRISPR/Cas9 gene editing\nAuthors: Szekely, O.; Rangadurai, A. K.; Gu, S.; Manghrani, A.; Guseva, S.; Al-Hashimi, H.\nScore: 12.2, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554670\nMany biochemical processes use the Watson-Crick geometry to distinguish correct from incorrect base pairing. However, on rare occasions, mismatches such as G{middle dot}T/U can transiently adopt Watson-Crick-like conformations through tautomerization or ionization of the bases, giving rise to replicative and translational errors. The propensities to form Watson-Crick-like mismatches in RNA:DNA hybrids remain unknown, making it unclear whether they can also contribute to errors during processes such as transcription and CRISPR/Cas editing. Here, using NMR R1{rho} experiments, we show that dG{middle dot}rU and dT{middle dot}rG mismatches in two RNA:DNA hybrids transiently form tautomeric (Genol{middle dot}T/U {rightleftarrows} G{middle dot}Tenol/Uenol) and anionic (G{middle dot}T-/U-) Watson-Crick-like conformations. The tautomerization dynamics were like those measured in A-RNA and B-DNA duplexes. However, anionic dG{middle dot}rU- formed with a ten-fold higher propensity relative to dT-{middle dot}rG and dG{middle dot}dT- and this could be attributed to the lower pKa ({Delta}pKa ~0.4-0.9) of U versus T. Our findings suggest plausible roles for Watson-Crick-like G{middle dot}T/U mismatches in transcriptional errors and CRISPR/Cas9 off-target gene editing, uncover a crucial difference between the chemical dynamics of G{middle dot}U versus G{middle dot}T, and indicate that anionic Watson-Crick-like G{middle dot}U- could play a significant role evading Watson-Crick fidelity checkpoints in RNA:DNA hybrids and RNA duplexes.\nNanopores reveal the stoichiometry of single oligo-adenylates produced by type III CRISPR-Cas\nAuthors: Fuentenebro-Navas, D.; Steens, J. A.; De Lannoy, C.; Noordijk, B.; de Ridder, D.; Staals, R. H. J.; Schmid, S.\nScore: 13.0, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553839\nCyclic oligoadenylates (cOAs) are small second messenger molecules produced by the type III CRISPR-Cas system as part of the prokaryotic immune response. The role of cOAs is to allosterically activate downstream effector proteins that induce dormancy or cell death, and thus abort viral spread through the population. Interestingly, different type III systems have been reported to utilize different cOA stoichiometries (with 3 to 6 adenylate monophosphates). However, so far, their characterization has only been possible in bulk and with sophisticated equipment, while a portable assay with single-molecule resolution has been lacking. Here, we demonstrate the label-free detection of single cOA molecules using a simple protein nanopore assay. It sensitively identifies the stoichiometry of individual cOA molecules and their mixtures from synthetic and enzymatic origin. To achieve this, we trained a convolutional neural network (CNN) and validated it with a series of experiments on mono- and polydisperse cOA samples. Ultimately, we determined the stoichiometric composition of cOAs produced enzymatically by the CRISPR type III-A and III-B variants of Thermus thermophilus. Interestingly, both variants produce cOAs of nearly identical composition, and we discuss the biological implications of this finding. The presented nanopore-CNN workflow with single-cOA resolution can be adapted to many other signaling molecules (including eukaryotic ones), and it may be integrated into portable handheld devices with potential point-of-care applications.\nRole of protonation states in stability of molecular dynamics simulations of high-resolution membrane protein structures\nAuthors: Lasham, J.; Djurabekova, A.; Zickermann, V.; Vonck, J.; Sharma, V.\nScore: 11.2, Published: 2023-08-25 DOI: 10.1101/2023.08.24.554589\nClassical molecular dynamics (MD) simulations provide unmatched spatial and time resolution of protein structure and function. However, accuracy of MD simulations often depends on the quality of force field parameters and the time scale of sampling. Another limitation of conventional MD simulations is that the protonation states of titratable amino acid residues remain fixed during simulations, even though protonation state changes coupled to conformational dynamics are central to protein function. Due to the uncertainty in selecting protonation states, classical MD simulations are sometimes performed with all amino acids modeled in their standard charged states at pH 7. Here we performed and analyzed classical MD simulations on high-resolution cryo-EM structures of two membrane proteins that transfer protons by catalyzing protonation/deprotonation reactions. In simulations performed with amino acids modeled in their standard protonation state the structure diverges far from its starting conformation. In comparison, MD simulations performed with pre-determined protonation states of amino acid residues reproduce the structural conformation, protein hydration, and protein-water and protein-protein interactions of the structure much better. The results suggest it is crucial to perform basic protonation state calculations, especially on structures where protonation changes play an important functional role, prior to launching any MD simulations. Furthermore, the combined approach of protonation state prediction and MD simulations can provide valuable information on the charge states of amino acids in the cryo-EM sample. Even though accurate prediction of protonation states currently remains a challenge, we introduce an approach of combining pKa prediction with cryo-EM density map analysis that helps in improving not only the protonation state predictions, but also the atomic modeling of density data.\nModeling membrane geometries implicitly in Rosetta\nAuthors: Woods, H.; Koehler Leman, J.; Meiler, J.\nScore: 8.3, Published: 2023-08-23 DOI: 10.1101/2023.08.23.554394\nInteractions between membrane proteins (MPs) and lipid bilayers are critical for many cellular functions. In the Rosetta molecular modeling suite, the implicit membrane energy function is based on a 'slab' model, which represent the membrane as a flat bilayer. However, in nature membranes often have a curvature that is important for function and/or stability. Even more prevalent, in structural biology research MPs are reconstituted in model membrane systems such as micelles, bicelles, nanodiscs, or liposomes. Thus, we have modified the existing membrane energy potentials within the RosettaMP framework to allow users to model MPs in different membrane geometries. We show that these modifications can be utilized in core applications within Rosetta such as structure refinement, protein-protein docking, and protein design. For MPs structures found in curved membranes, refining these structures in curved, implicit membranes produces higher quality models with structures closer to experimentally determined structures. For MP systems embedded in multiple membranes, representing both membranes results in more favorable scores compared to only representing one of the membranes. Modeling MPs in geometries mimicking the membrane model system used in structure determination can improve model quality and model discrimination.\nDifferences in Oligomerization of the SARS-CoV-2 Envelope Protein, Poliovirus VP4, and HIV Vpu\nAuthors: Townsend, J. A.; Fapohunda, O.; Wang, Z.; Pham, H.; Taylor, M. T.; Kloss, B.; Park, S. H.; Opella, S. J.; Aspinwall, C. A.; Marty, M. T.\nScore: 7.8, Published: 2023-08-20 DOI: 10.1101/2023.08.18.553902\nViroporins constitute a class of viral membrane proteins with diverse roles in the viral life cycle. They can self-assemble and form pores within the bilayer that transport substrates, such as ions and genetic material, that are critical to the viral infection cycle. However, there is little known about the oligomeric state of most viroporins. Here, we use native mass spectrometry (MS) in detergent micelles to uncover the patterns of oligomerization of the full-length SARS-CoV-2 envelope (E) protein, poliovirus VP4, and HIV Vpu. Our data suggest that the E protein is a specific dimer, VP4 is exclusively monomeric, and Vpu assembles into a polydisperse mixture of oligomers under these conditions. Overall, these results revealed the diversity in the oligomerization of viroporins, which has implications for mechanisms of their biological functions as well as their potential as therapeutic targets.\n",
  "wordCount" : "2142",
  "inLanguage": "en",
  "datePublished": "2023-08-27T10:37:02Z",
  "dateModified": "2023-08-27T10:37:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on August 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553799">
        <p class="paperTitle">Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.</p>
        <p class="info">Score: 107.6, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553799' target='https://doi.org/10.1101/2023.08.18.553799'> 10.1101/2023.08.18.553799</a></p>
        <p class="abstract">Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms. This method, cryoDRGN-ET, learns a deep generative model of 3D density maps directly from subtomogram tilt series images and can capture states diverse in both composition and conformation. We use this approach to reconstruct the in situ translation dynamics of prokaryotic ribosomes, and we reveal the distribution of functional states during translation elongation populated by S. cerevisiae ribosomes inside cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554704">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554704" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554704">
        <p class="paperTitle">Time-Resolved Cryo-EM Specimen Preparation with Single Millisecond Precision</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554704" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554704" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Montano Romero, A.; Bonin, C.; Twomey, E. C.</p>
        <p class="info">Score: 43.4, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554704' target='https://doi.org/10.1101/2023.08.24.554704'> 10.1101/2023.08.24.554704</a></p>
        <p class="abstract">Molecular structures can be determined in vitro and in situ with cryo-electron microscopy (cryo-EM). Specimen preparation is a major obstacle in cryo-EM. Typical sample preparation is orders of magnitude slower than biological processes. Time-resolved cryo-EM (TR-cryo-EM) can capture short-lived states. Here, we present Cryo-EM Sample Preparation with light-Activated Molecules (C-SPAM), an open-source, photochemistry-coupled device for TR-cryo-EM with single millisecond resolution, tunable timescales, and broad biological applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554239">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554239" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554239">
        <p class="paperTitle">Sub-membrane actin rings compartmentalize the plasma membrane</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554239" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554239" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rentsch, J.; Bandstra, S.; Sezen, B.; Sigrist, P. S.; Bottanelli, F.; Schmerl, B.; Shoichet, S. A.; Noe, F.; Sadeghi, M.; Ewers, H.</p>
        <p class="info">Score: 23.0, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554239' target='https://doi.org/10.1101/2023.08.22.554239'> 10.1101/2023.08.22.554239</a></p>
        <p class="abstract">The compartmentalization of the plasma membrane is a fundamental feature of cells. The diffusivity of membrane proteins in the plane of the membrane is significantly lower in cells than observed in artificial membranes. This seems due to the sub-membranous actin cortex, but tractable assays to prove a direct dependence of membrane protein motion on actin filaments have been elusive. We recently showed that a periodic array of sub-membrane actin filaments that surround neuronal axons in ring-like structures in the axonal initial segment (AIS) confines membrane protein motion between them, but the local enrichment of ion channels may offer an alternative explanation for compartmentalization. Here we show using computational modeling that in contrast to actin rings, the dense array of ion channels in the AIS cannot mediate membrane protein confinement. We go on to show that indeed the actin rings are closely apposed to the plasma membrane, and that they confine membrane protein motion between them in a [~] 200 nm periodic pattern in progenitor-derived neuronal cells. We find that this compartmentalization is also detectable for inner-leaflet membrane proteins and multi-spanning receptors. Strikingly, several cell types with actin rings in their protrusions, like progenitor-derived astrocytes and oligodendrocytes, also exhibit lateral confinement of membrane proteins in a periodic pattern consistent with actin ring spacing. Actin ring-mediated membrane compartmentalization is thus not unique to neurons. Finally, we show that acute actin disruption in live progenitor-derived neuronal cells leads to a loss of membrane compartmentalization in areas that were compartmentalized before treatment. Taken together, we here develop a much-needed tractable experimental system for the investigation of membrane compartmentalization and show that actin rings compartmentalize the plasma membrane of cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.554035">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.554035" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.554035">
        <p class="paperTitle">ATP Mediates Phase Separation of Disordered Basic Proteins by Bridging Intermolecular Interaction Networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.554035" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.554035" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kota, D.; Prasad, R.; Zhou, H.-X.</p>
        <p class="info">Score: 16.0, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.554035' target='https://doi.org/10.1101/2023.08.20.554035'> 10.1101/2023.08.20.554035</a></p>
        <p class="abstract">ATP is an abundant molecule with crucial cellular roles as the energy currency and a building block of nucleic acids and for protein phosphorylation. Here we show that ATP mediates the phase separation of basic intrinsically disordered proteins (bIDPs). In the resulting condensates, ATP is highly concentrated (apparent partition coefficients at 200-5000) and serves as bridges between bIDP chains. These liquid-like droplets have some of the lowest interfacial tension ([~]25 pN/m) but high zero-shear viscosities (1-15 Pa s) due to the bridged protein networks, and yet their fusion has some of the highest speeds ([~]1 m/ms). The rapid fusion manifests extreme shear thinning, where the apparent viscosity is lower than zero-shear viscosity by over 100-fold, made possible by fast reformation of the ATP bridges. At still higher concentrations, ATP does not dissolve bIDP droplets but results in aggregates and fibrils.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553535">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553535" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553535">
        <p class="paperTitle">Shocker - a molecular dynamics protocol and tool for accelerating and analyzing the effects of osmotic shocks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553535" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553535" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van Tilburg, M. P. A.; Marrink, S.-J.; Koenig, M.; Gruenewald, F.</p>
        <p class="info">Score: 17.1, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553535' target='https://doi.org/10.1101/2023.08.16.553535'> 10.1101/2023.08.16.553535</a></p>
        <p class="abstract">The process of osmosis, a fundamental phenomenon in life, drives water through a semi-permeable membrane in response to a solute concentration gradient across this membrane. In vitro, osmotic shocks are often used to drive shape changes in lipid vesicles, for instance, to study fission events in the context of artificial cells. While experimental techniques provide a macroscopic picture of large-scale membrane remodeling processes, molecular dynamics (MD) simulations are a powerful tool to study membrane deformations at the molecular level. However, simulating an osmotic shock is a time-consuming process due to the slow water diffusion across the membrane, making it practically impossible to examine its effects in classic MD simulations. In this paper, we present Shocker, a Python-based MD tool for simulating the effects of an osmotic shock by selecting and relocating water particles across a membrane over the course of several pumping cycles. Although this method is primarily aimed at efficiently simulating volume changes of vesicles it can handle membrane tubes and double bilayer systems as well. Additionally, Shocker is force field independent and compatible with both coarse-grained and all-atom systems. We demonstrate that our tool is applicable to simulate both hypertonic and hypotonic osmotic shocks for a range of vesicular and bilamellar setups, including complex multi-component systems containing membrane proteins or crowded internal solutions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554670">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554670" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554670">
        <p class="paperTitle">NMR measurements of transient low-populated tautomeric and anionic Watson-Crick-like G·T/U in RNA:DNA hybrids: Implications for the fidelity of transcription and CRISPR/Cas9 gene editing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554670" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554670" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Szekely, O.; Rangadurai, A. K.; Gu, S.; Manghrani, A.; Guseva, S.; Al-Hashimi, H.</p>
        <p class="info">Score: 12.2, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554670' target='https://doi.org/10.1101/2023.08.24.554670'> 10.1101/2023.08.24.554670</a></p>
        <p class="abstract">Many biochemical processes use the Watson-Crick geometry to distinguish correct from incorrect base pairing. However, on rare occasions, mismatches such as G{middle dot}T/U can transiently adopt Watson-Crick-like conformations through tautomerization or ionization of the bases, giving rise to replicative and translational errors. The propensities to form Watson-Crick-like mismatches in RNA:DNA hybrids remain unknown, making it unclear whether they can also contribute to errors during processes such as transcription and CRISPR/Cas editing. Here, using NMR R1{rho} experiments, we show that dG{middle dot}rU and dT{middle dot}rG mismatches in two RNA:DNA hybrids transiently form tautomeric (Genol{middle dot}T/U {rightleftarrows} G{middle dot}Tenol/Uenol) and anionic (G{middle dot}T-/U-) Watson-Crick-like conformations. The tautomerization dynamics were like those measured in A-RNA and B-DNA duplexes. However, anionic dG{middle dot}rU- formed with a ten-fold higher propensity relative to dT-{middle dot}rG and dG{middle dot}dT- and this could be attributed to the lower pKa ({Delta}pKa ~0.4-0.9) of U versus T. Our findings suggest plausible roles for Watson-Crick-like G{middle dot}T/U mismatches in transcriptional errors and CRISPR/Cas9 off-target gene editing, uncover a crucial difference between the chemical dynamics of G{middle dot}U versus G{middle dot}T, and indicate that anionic Watson-Crick-like G{middle dot}U- could play a significant role evading Watson-Crick fidelity checkpoints in RNA:DNA hybrids and RNA duplexes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553839">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553839" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553839">
        <p class="paperTitle">Nanopores reveal the stoichiometry of single oligo-adenylates produced by type III CRISPR-Cas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553839" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553839" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fuentenebro-Navas, D.; Steens, J. A.; De Lannoy, C.; Noordijk, B.; de Ridder, D.; Staals, R. H. J.; Schmid, S.</p>
        <p class="info">Score: 13.0, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553839' target='https://doi.org/10.1101/2023.08.18.553839'> 10.1101/2023.08.18.553839</a></p>
        <p class="abstract">Cyclic oligoadenylates (cOAs) are small second messenger molecules produced by the type III CRISPR-Cas system as part of the prokaryotic immune response. The role of cOAs is to allosterically activate downstream effector proteins that induce dormancy or cell death, and thus abort viral spread through the population. Interestingly, different type III systems have been reported to utilize different cOA stoichiometries (with 3 to 6 adenylate monophosphates). However, so far, their characterization has only been possible in bulk and with sophisticated equipment, while a portable assay with single-molecule resolution has been lacking. Here, we demonstrate the label-free detection of single cOA molecules using a simple protein nanopore assay. It sensitively identifies the stoichiometry of individual cOA molecules and their mixtures from synthetic and enzymatic origin. To achieve this, we trained a convolutional neural network (CNN) and validated it with a series of experiments on mono- and polydisperse cOA samples. Ultimately, we determined the stoichiometric composition of cOAs produced enzymatically by the CRISPR type III-A and III-B variants of Thermus thermophilus. Interestingly, both variants produce cOAs of nearly identical composition, and we discuss the biological implications of this finding. The presented nanopore-CNN workflow with single-cOA resolution can be adapted to many other signaling molecules (including eukaryotic ones), and it may be integrated into portable handheld devices with potential point-of-care applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554589">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554589" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554589">
        <p class="paperTitle">Role of protonation states in stability of molecular dynamics simulations of high-resolution membrane protein structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554589" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554589" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lasham, J.; Djurabekova, A.; Zickermann, V.; Vonck, J.; Sharma, V.</p>
        <p class="info">Score: 11.2, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554589' target='https://doi.org/10.1101/2023.08.24.554589'> 10.1101/2023.08.24.554589</a></p>
        <p class="abstract">Classical molecular dynamics (MD) simulations provide unmatched spatial and time resolution of protein structure and function. However, accuracy of MD simulations often depends on the quality of force field parameters and the time scale of sampling. Another limitation of conventional MD simulations is that the protonation states of titratable amino acid residues remain fixed during simulations, even though protonation state changes coupled to conformational dynamics are central to protein function. Due to the uncertainty in selecting protonation states, classical MD simulations are sometimes performed with all amino acids modeled in their standard charged states at pH 7. Here we performed and analyzed classical MD simulations on high-resolution cryo-EM structures of two membrane proteins that transfer protons by catalyzing protonation/deprotonation reactions. In simulations performed with amino acids modeled in their standard protonation state the structure diverges far from its starting conformation. In comparison, MD simulations performed with pre-determined protonation states of amino acid residues reproduce the structural conformation, protein hydration, and protein-water and protein-protein interactions of the structure much better. The results suggest it is crucial to perform basic protonation state calculations, especially on structures where protonation changes play an important functional role, prior to launching any MD simulations. Furthermore, the combined approach of protonation state prediction and MD simulations can provide valuable information on the charge states of amino acids in the cryo-EM sample. Even though accurate prediction of protonation states currently remains a challenge, we introduce an approach of combining pKa prediction with cryo-EM density map analysis that helps in improving not only the protonation state predictions, but also the atomic modeling of density data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554394">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554394" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554394">
        <p class="paperTitle">Modeling membrane geometries implicitly in Rosetta</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554394" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554394" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Woods, H.; Koehler Leman, J.; Meiler, J.</p>
        <p class="info">Score: 8.3, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554394' target='https://doi.org/10.1101/2023.08.23.554394'> 10.1101/2023.08.23.554394</a></p>
        <p class="abstract">Interactions between membrane proteins (MPs) and lipid bilayers are critical for many cellular functions. In the Rosetta molecular modeling suite, the implicit membrane energy function is based on a &#39;slab&#39; model, which represent the membrane as a flat bilayer. However, in nature membranes often have a curvature that is important for function and/or stability. Even more prevalent, in structural biology research MPs are reconstituted in model membrane systems such as micelles, bicelles, nanodiscs, or liposomes. Thus, we have modified the existing membrane energy potentials within the RosettaMP framework to allow users to model MPs in different membrane geometries. We show that these modifications can be utilized in core applications within Rosetta such as structure refinement, protein-protein docking, and protein design. For MPs structures found in curved membranes, refining these structures in curved, implicit membranes produces higher quality models with structures closer to experimentally determined structures. For MP systems embedded in multiple membranes, representing both membranes results in more favorable scores compared to only representing one of the membranes. Modeling MPs in geometries mimicking the membrane model system used in structure determination can improve model quality and model discrimination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553902">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553902" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553902">
        <p class="paperTitle">Differences in Oligomerization of the SARS-CoV-2 Envelope Protein, Poliovirus VP4, and HIV Vpu</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553902" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553902" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Townsend, J. A.; Fapohunda, O.; Wang, Z.; Pham, H.; Taylor, M. T.; Kloss, B.; Park, S. H.; Opella, S. J.; Aspinwall, C. A.; Marty, M. T.</p>
        <p class="info">Score: 7.8, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553902' target='https://doi.org/10.1101/2023.08.18.553902'> 10.1101/2023.08.18.553902</a></p>
        <p class="abstract">Viroporins constitute a class of viral membrane proteins with diverse roles in the viral life cycle. They can self-assemble and form pores within the bilayer that transport substrates, such as ions and genetic material, that are critical to the viral infection cycle. However, there is little known about the oligomeric state of most viroporins. Here, we use native mass spectrometry (MS) in detergent micelles to uncover the patterns of oligomerization of the full-length SARS-CoV-2 envelope (E) protein, poliovirus VP4, and HIV Vpu. Our data suggest that the E protein is a specific dimer, VP4 is exclusively monomeric, and Vpu assembles into a polydisperse mixture of oligomers under these conditions. Overall, these results revealed the diversity in the oligomerization of viroporins, which has implications for mechanisms of their biological functions as well as their potential as therapeutic targets.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
