<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="High-throughput computational discovery of inhibitory protein fragments with AlphaFold
Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.
Score: 74.7, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389
Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="High-throughput computational discovery of inhibitory protein fragments with AlphaFold
Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.
Score: 74.7, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389
Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-24T10:38:02+00:00" />
<meta property="article:modified_time" content="2023-12-24T10:38:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="High-throughput computational discovery of inhibitory protein fragments with AlphaFold
Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.
Score: 74.7, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389
Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "High-throughput computational discovery of inhibitory protein fragments with AlphaFold\nAuthors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.\nScore: 74.7, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389\nPeptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells.",
  "keywords": [
    
  ],
  "articleBody": " High-throughput computational discovery of inhibitory protein fragments with AlphaFold\nAuthors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.\nScore: 74.7, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389\nPeptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells. However, it has thus far not been possible to predict de novo which of the many possible protein fragments bind their protein targets, let alone act as inhibitors. We have developed a computational method, FragFold, that employs AlphaFold to predict protein fragment binding to full-length protein targets in a high-throughput manner. Applying FragFold to thousands of fragments tiling across diverse proteins revealed peaks of predicted binding along each protein sequence. These predictions were compared with experimentally measured peaks of inhibitory activity in E. coli. We establish that our approach is a sensitive predictor of protein fragment function: Evaluating inhibitory fragments derived from known protein-protein interaction interfaces, we found 87% were predicted by FragFold to bind in a native-like mode. Across full protein sequences, 68% of FragFold-predicted binding peaks match experimentally measured inhibitory peaks. This is true even when the underlying inhibitory mechanism is unclear from existing structural data, and we find FragFold is able to predict novel binding modes for inhibitory fragments of unknown structure, explaining previous genetic and biochemical data for these fragments. The success rate of FragFold demonstrates that this computational approach should be broadly applicable for discovering inhibitory protein fragments across proteomes.\nActive membrane deformations of a minimal synthetic cell\nAuthors: Sciortino, A.; Faizi, H.; Uplap, S.; Frechette, L.; Peterson, M. S.; Vlahovska, P.; Baskaran, A.; Hagan, M. F.; Bausch, A.\nScore: 39.0, Published: 2023-12-18 DOI: 10.1101/2023.12.18.571643\nBiological cells exhibit the remarkable ability to adapt their shape in response to their environment, a phenomenon that hinges on the intricate interplay between their deformable membrane and the underlying activity of their cytoskeleton. Yet, the precise physical mechanisms of this coupling remain mostly elusive. Here, we introduce a synthetic cell model, comprised of an active cytoskeletal network of microtubules, crosslinkers and molecular motors encapsulated inside giant vesicles. Remarkably, these active vesicles exhibit large shape fluctuations and life-like morphing abilities. Active forces from the encapsulated cytoskeleton give rise to large-scale traveling membrane deformations. Quantitative analysis of membrane and microtubule fluctuations shows how the intricate coupling of confinement, membrane material properties and cytoskeletal forces yields fluctuation spectra whose characteristic scales in space and time are distinctly different from passive vesicles. We demonstrate how activity leads to uneven probability fluxes between fluctuation modes and hence sets the temporal scale of membrane fluctuations. Using simulations and theoretical modelling, we extend the classical approach to membrane fluctuations to active cytoskeleton-driven vesicles, highlighting the effect of correlated activity on the dynamics of membrane deformations and paving the way for quantitative descriptions of the shape-morphing ability typical of living systems.\n1D confinement mimicking microvessel geometry controls pericyte shape and motility\nAuthors: Sagnimorte, A.; Adler, M. R.; de Tournemire, G.; Saez, P. J.; Gonzalez-Rodriguez, D.; Dessalles, C. A.; Babataheri, A.\nScore: 18.7, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572195\nPericytes are mural cells of the microvasculature, characterised by their elongated distinct shape. Pericytes span along the axis of the vessels they adhere to, therefore they experience extreme lateral and longitudinal confinement. Pericyte shape is key for their function during vascular regulation and their spatial distribution is established by cell migration during the embryonic stage and maintained through controlled motility in the adult. However, how pericyte morphology is associated with migration and function remains unknown. We use micropatterns to mimic pericyte adhesion to vessels, and to reproduce in vitro the shapes adopted by pericytes in vivo. We show that lateral confinement controls cell shape and produces in vivo-like phenotype. Modelling the pericyte as an incompressible linear elastic material predicts strain and shape of pericytes as a function of lateral confinement. Pericyte kinetics on both laterally confining lanes, and longitudinally constraining motifs is described by dry friction theory. Pericytes are capable of crossing gaps of different sizes. The percentage of crossings is correctly predicted by the likelihood of a fluctuating system to overcome an energy barrier. Our joint experimental and theoretical approach demonstrates the effect of in vivo-like geometrical confinement on pericyte morphology and migration which is accurately described by dry friction theory.\nPlant infection by the necrotrophic fungus Botrytis requires actin-dependent generation of high invasive turgor pressure\nAuthors: Mueller, T.; Bronkhorst, J.; Mueller, J.; Safari, N.; Hahn, M.; Sprakel, J.; Scheuring, D.\nScore: 20.8, Published: 2023-12-16 DOI: 10.1101/2023.12.15.571920\nThe devastating pathogen Botrytis cinerea infects a broad spectrum of host plants, causing great socio-economic losses. The necrotrophic fungus rapidly kills plant cells, nourishing their walls and cellular contents. To this end, necrotrophs secretes a cocktail of cell wall degrading enzymes, phytotoxic proteins and metabolites. Additionally, many fungi produce specialized invasion organs that generate high invasive pressures to force their way into the plant cell. However, for most necrotrophs, including Botrytis, the biomechanics of penetration and its contribution to virulence are poorly understood. Here we use a combination of quantitative micromechanical imaging and CRISPR-Cas guided mutagenesis to show that Botrytis uses substantial invasive pressure, in combination with strong surface adherence, for penetration. We found that the fungus establishes a unique mechanical geometry of penetration that develops over time during penetration events, and which is actin cytoskeleton dependent. Furthermore, interference of force generation by blocking actin polymerization was found to decrease Botrytis virulence, indicating that also for necrotrophs, mechanical pressure is important in host colonization. Our results demonstrate for the first time mechanistically how a necrotrophic fungus such as Botrytis employs this \"brute force\" approach, in addition to the secretion of lytic proteins and phytotoxic metabolites, to overcome plant host resistance.\nHepatitis B virus neutralization with DNA origami nanoshells\nAuthors: Willner, E.-M.; Kolbe, F.; Momburg, F.; Protzer, U.; Dietz, H.\nScore: 17.6, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572526\nWe demonstrate the use of DNA origami to create virus-trapping nanoshells that efficiently neutralize hepatitis B virus (HBV) in cell culture. By modifying the shells with a synthetic monoclonal antibody that binds to the HBV envelope, the effective neutralization potency per antibody is increased by approximately 100 times compared to using free antibodies. The improvements in neutralizing the virus are attributed to two factors: first, the shells act as a physical barrier that block the virus from interacting with host cells; second, the multivalent binding of the antibodies inside the shells lead to stronger attachment to the trapped virus, a phenomenon known as avidity. Pre-incubation of shells with HBV and simultaneous addition of both components separately to cells lead to comparable levels of neutralization, indicating rapid trapping of the virions by the shells. Our study highlights the potential of the DNA shell system to rationally create novel antivirals using components that, when used individually, show little to no antiviral effectiveness.\n3D Mitochondrial Structure in Aging Human Skeletal Muscle: Insights into MFN-2 Mediated Changes\nAuthors: Scudese, E.; Vue, Z.; Katti, P.; Marshall, A.; Vang, L.; Garza Lopez, E.; Neikirk, K.; Stephens, D.; Hall, D. D.; Rostami, R.; Shao, J.-q.; Mungai, M.; AshShareef, S.; Hicsasmaz, I.; Manus, S.; Whiteside, A.; Williams, C.; Damo, S.; Gaddy, J.; Kirabo, A.; Glancy, B.; Henrique Martin Dantas, E.; Kinder, A.; Scartoni, F.; Baffi, M.; McReynolds, M.; Wanjalla, C. N.; Phillips, M. A.; Exil, V.; Mobley, B.; Hinton, A.\nScore: 14.2, Published: 2023-12-14 DOI: 10.1101/2023.11.13.566502\nSarcopenia is an age-related loss of skeletal muscle, characterized by loss of mass, strength, endurance, and oxidative capacity during aging. Notably, bioenergetics and protein turnover studies have shown that mitochondria mediate this decline in function. Although mitochondrial aging is associated with decreased mitochondrial capacity, the three-dimensional (3D) mitochondrial structure associated with morphological changes in skeletal muscle during aging still requires further elucidation. Although exercise has been the only therapy to mitigate sarcopenia, the mechanisms that govern these changes remain unclear. We hypothesized that aging causes structural remodeling of mitochondrial 3D architecture representative of dysfunction, and this effect is mitigated by exercise. We used serial block-face scanning electron microscopy to image human skeletal tissue samples, followed by manual contour tracing using Amira software for 3D reconstruction and subsequent analysis of mitochondria. We then applied a rigorous in vitro and in vivo exercise regimen during aging. We found that mitochondria became less complex with age. Specifically, mitochondria lost surface area, complexity, and perimeter, indicating age-related declines in ATP synthesis and interaction capacity. Concomitantly, muscle area, exercise capacity, and mitochondrial dynamic proteins showed age-related losses. Exercise stimulation restored mitofusin 2 (MFN2), which we show is required for mitochondrial structure. Furthermore, we show that this pathway is evolutionarily conserved with Marf, the MFN2 ortholog in Drosophila, as Marf knockdown alters mitochondrial morphology and leads to the downregulation of genes regulating mitochondrial processes. Our results define age-related structural changes in mitochondria and further suggest that exercise may mitigate age-related structural decline through modulation of mitofusins.\nGSK3β phosphorylation catalyzes the aggregation of Tau into Alzheimer's disease-like amyloid strain\nAuthors: Chakraborty, P.; Ibanez de Opakua, A.; Purslow, J. A.; Fromm, S. A.; Chatterjee, D.; Zachrdla, M.; Puri, S.; Wolozin, B.; Zweckstetter, M.\nScore: 10.4, Published: 2023-12-19 DOI: 10.1101/2023.12.19.572292\nThe pathological deposition of proteins is a hallmark of several devastating neurodegenerative diseases. These pathological deposits comprise aggregates of proteins that adopt distinct structures named strains. However, the molecular factors responsible for the formation of distinct aggregate strains are unknown. Here we show that the serine/threonine kinase GSK3{beta} catalyzes the aggregation of the protein tau into an Alzheimers disease-like amyloid strain. We demonstrate that phosphorylation by GSK3{beta}, but not by several other kinases, promotes the aggregation of full-length tau through enhanced phase separation into gel-like condensate structures. Cryo-electron microscopy further reveals that the amyloid fibrils formed by GSK3{beta}-phosphorylated tau adopt a fold comparable to that of paired helical filaments isolated from the brains of AD patients. Our results elucidate the intricate relationship between post-translational modification and the formation of tau strains in neurodegenerative diseases.\nStructure of the human systemic RNAi defective transmembrane protein 1 (hSIDT1) reveals the conformational flexibility of its lipid binding domain\nAuthors: Navratna, V.; Kumar, A.; Rana, J. K.; Mosalaganti, S.\nScore: 9.1, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572875\nIn C. elegans, the inter-cellular transport of the small non-coding RNA that causes systemic RNA interference (RNAi) is mediated by the transmembrane protein SID1, encoded by the sid1 gene in the systemic RNA interference-defective (sid) loci. SID1 shares remarkable structural and sequence similarity with cholesterol uptake protein 1 (CHUP1), and they are classified together as members of the cholesterol uptake family (ChUP). Although systemic RNAi is not an evolutionarily conserved process, the sid gene products are found across the animal kingdom, suggesting a potential for the existence of other novel gene regulatory mechanisms mediated by small non-coding RNAs. Human homologs of sid gene products, hSIDT1 and hSIDT2, are believed to mediate contact-dependent lipophilic small non-coding dsRNA transport. Here, we report the structure of recombinant full-length human SIDT1. We find that the extra-cytosolic domain (ECD) of hSIDT1 adopts a double jelly roll fold, and the transmembrane domain (TMD) exists as two modules, a flexible lipid binding domain (LBD) and a rigid TMD core. Our structural analyses provide insights into the potential role of the lipid binding domain in the function of hSIDT1 and other cholesterol uptake (ChUP) family members.\nConformational sampling and interpolation using language-based protein folding neural networks\nAuthors: del Alamo, D.; Jeliazkov, J. R.; Truan, D.; Karpiak, J.\nScore: 8.9, Published: 2023-12-19 DOI: 10.1101/2023.12.16.571997\nProtein language models (PLMs), such ESM2, learn a rich semantic grammar of the protein sequence space. When coupled to protein folding neural networks (e.g., ESMFold), they can facilitate the prediction of tertiary and quaternary protein structures at high accuracy. However, they are limited to modeling protein structures in single states. This manuscript demonstrates that ESMFold can predict alternate conformations of some proteins, including de novo designed proteins. Randomly masking the sequence prior to PLM input returned alternate embeddings that ESMFold sometimes mapped to distinct physiologically relevant conformations. From there, inversion of the ESMFold trunk facilitated the generation of high-confidence interconversion paths between the two states. These paths provide a deeper glimpse of how language-based protein folding neural networks derive structural information from high-dimensional sequence representations, while exposing limitations in their general understanding of protein structure and folding.\nRASP: Optimal single fluorescent puncta detection in complex cellular backgrounds\nAuthors: Fu, B.; Brock, E. E.; Andrews, R.; Breiter, J. C.; Tian, R.; Toomey, C. E.; Lachica, J.; Lashley, T.; Ryten, M.; Wood, N. W.; Vendruscolo, M.; Gandhi, S.; Weiss, L. E.; Beckwith, J. S.; Lee, S. F.\nScore: 8.8, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572148\nSuper-resolution and single-molecule microscopy are increasingly applied to complex biological systems. A major challenge of this approach is that fluorescent puncta must be detected in the low signal, high noise, heterogeneous background environments of cells and tissue. We present RASP, Radiality Analysis of Single Puncta, a bioimaging-segmentation method that solves this problem. RASP removes false positive puncta that other analysis methods detect, and detects features over a broad range of spatial scales: from single proteins to complex cell phenotypes. RASP outperforms the state-of-the-art in precision and speed, using image gradients to separate Gaussian-shaped objects from background. We demonstrate RASPs power by showing it can extract spatial correlations between microglia, neurons, and -synuclein oligomers in the human brain. This sensitive, computationally efficient approach enables fluorescent puncta and cellular features to be distinguished in cellular and tissue environments with a sensitivity down to the level of the single protein.\n",
  "wordCount" : "2264",
  "inLanguage": "en",
  "datePublished": "2023-12-24T10:38:02Z",
  "dateModified": "2023-12-24T10:38:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 24, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572389" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572389">
        <p class="paperTitle">High-throughput computational discovery of inhibitory protein fragments with AlphaFold</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.</p>
        <p class="info">Score: 74.7, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572389' target='https://doi.org/10.1101/2023.12.19.572389'> 10.1101/2023.12.19.572389</a></p>
        <p class="abstract">Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells. However, it has thus far not been possible to predict de novo which of the many possible protein fragments bind their protein targets, let alone act as inhibitors. We have developed a computational method, FragFold, that employs AlphaFold to predict protein fragment binding to full-length protein targets in a high-throughput manner. Applying FragFold to thousands of fragments tiling across diverse proteins revealed peaks of predicted binding along each protein sequence. These predictions were compared with experimentally measured peaks of inhibitory activity in E. coli. We establish that our approach is a sensitive predictor of protein fragment function: Evaluating inhibitory fragments derived from known protein-protein interaction interfaces, we found 87% were predicted by FragFold to bind in a native-like mode. Across full protein sequences, 68% of FragFold-predicted binding peaks match experimentally measured inhibitory peaks. This is true even when the underlying inhibitory mechanism is unclear from existing structural data, and we find FragFold is able to predict novel binding modes for inhibitory fragments of unknown structure, explaining previous genetic and biochemical data for these fragments. The success rate of FragFold demonstrates that this computational approach should be broadly applicable for discovering inhibitory protein fragments across proteomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.571643">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.571643" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.571643">
        <p class="paperTitle">Active membrane deformations of a minimal synthetic cell</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.571643" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.571643" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sciortino, A.; Faizi, H.; Uplap, S.; Frechette, L.; Peterson, M. S.; Vlahovska, P.; Baskaran, A.; Hagan, M. F.; Bausch, A.</p>
        <p class="info">Score: 39.0, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.571643' target='https://doi.org/10.1101/2023.12.18.571643'> 10.1101/2023.12.18.571643</a></p>
        <p class="abstract">Biological cells exhibit the remarkable ability to adapt their shape in response to their environment, a phenomenon that hinges on the intricate interplay between their deformable membrane and the underlying activity of their cytoskeleton. Yet, the precise physical mechanisms of this coupling remain mostly elusive. Here, we introduce a synthetic cell model, comprised of an active cytoskeletal network of microtubules, crosslinkers and molecular motors encapsulated inside giant vesicles. Remarkably, these active vesicles exhibit large shape fluctuations and life-like morphing abilities. Active forces from the encapsulated cytoskeleton give rise to large-scale traveling membrane deformations. Quantitative analysis of membrane and microtubule fluctuations shows how the intricate coupling of confinement, membrane material properties and cytoskeletal forces yields fluctuation spectra whose characteristic scales in space and time are distinctly different from passive vesicles. We demonstrate how activity leads to uneven probability fluxes between fluctuation modes and hence sets the temporal scale of membrane fluctuations. Using simulations and theoretical modelling, we extend the classical approach to membrane fluctuations to active cytoskeleton-driven vesicles, highlighting the effect of correlated activity on the dynamics of membrane deformations and paving the way for quantitative descriptions of the shape-morphing ability typical of living systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572195">
        <p class="paperTitle">1D confinement mimicking microvessel geometry controls pericyte shape and motility</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sagnimorte, A.; Adler, M. R.; de Tournemire, G.; Saez, P. J.; Gonzalez-Rodriguez, D.; Dessalles, C. A.; Babataheri, A.</p>
        <p class="info">Score: 18.7, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572195' target='https://doi.org/10.1101/2023.12.20.572195'> 10.1101/2023.12.20.572195</a></p>
        <p class="abstract">Pericytes are mural cells of the microvasculature, characterised by their elongated distinct shape. Pericytes span along the axis of the vessels they adhere to, therefore they experience extreme lateral and longitudinal confinement. Pericyte shape is key for their function during vascular regulation and their spatial distribution is established by cell migration during the embryonic stage and maintained through controlled motility in the adult. However, how pericyte morphology is associated with migration and function remains unknown. We use micropatterns to mimic pericyte adhesion to vessels, and to reproduce in vitro the shapes adopted by pericytes in vivo. We show that lateral confinement controls cell shape and produces in vivo-like phenotype. Modelling the pericyte as an incompressible linear elastic material predicts strain and shape of pericytes as a function of lateral confinement. Pericyte kinetics on both laterally confining lanes, and longitudinally constraining motifs is described by dry friction theory. Pericytes are capable of crossing gaps of different sizes. The percentage of crossings is correctly predicted by the likelihood of a fluctuating system to overcome an energy barrier. Our joint experimental and theoretical approach demonstrates the effect of in vivo-like geometrical confinement on pericyte morphology and migration which is accurately described by dry friction theory.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571920">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571920" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571920">
        <p class="paperTitle">Plant infection by the necrotrophic fungus Botrytis requires actin-dependent generation of high invasive turgor pressure</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571920" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571920" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mueller, T.; Bronkhorst, J.; Mueller, J.; Safari, N.; Hahn, M.; Sprakel, J.; Scheuring, D.</p>
        <p class="info">Score: 20.8, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571920' target='https://doi.org/10.1101/2023.12.15.571920'> 10.1101/2023.12.15.571920</a></p>
        <p class="abstract">The devastating pathogen Botrytis cinerea infects a broad spectrum of host plants, causing great socio-economic losses. The necrotrophic fungus rapidly kills plant cells, nourishing their walls and cellular contents. To this end, necrotrophs secretes a cocktail of cell wall degrading enzymes, phytotoxic proteins and metabolites. Additionally, many fungi produce specialized invasion organs that generate high invasive pressures to force their way into the plant cell. However, for most necrotrophs, including Botrytis, the biomechanics of penetration and its contribution to virulence are poorly understood. Here we use a combination of quantitative micromechanical imaging and CRISPR-Cas guided mutagenesis to show that Botrytis uses substantial invasive pressure, in combination with strong surface adherence, for penetration. We found that the fungus establishes a unique mechanical geometry of penetration that develops over time during penetration events, and which is actin cytoskeleton dependent. Furthermore, interference of force generation by blocking actin polymerization was found to decrease Botrytis virulence, indicating that also for necrotrophs, mechanical pressure is important in host colonization. Our results demonstrate for the first time mechanistically how a necrotrophic fungus such as Botrytis employs this &#34;brute force&#34; approach, in addition to the secretion of lytic proteins and phytotoxic metabolites, to overcome plant host resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572526">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572526" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572526">
        <p class="paperTitle">Hepatitis B virus neutralization with DNA origami nanoshells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572526" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572526" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Willner, E.-M.; Kolbe, F.; Momburg, F.; Protzer, U.; Dietz, H.</p>
        <p class="info">Score: 17.6, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572526' target='https://doi.org/10.1101/2023.12.20.572526'> 10.1101/2023.12.20.572526</a></p>
        <p class="abstract">We demonstrate the use of DNA origami to create virus-trapping nanoshells that efficiently neutralize hepatitis B virus (HBV) in cell culture. By modifying the shells with a synthetic monoclonal antibody that binds to the HBV envelope, the effective neutralization potency per antibody is increased by approximately 100 times compared to using free antibodies. The improvements in neutralizing the virus are attributed to two factors: first, the shells act as a physical barrier that block the virus from interacting with host cells; second, the multivalent binding of the antibodies inside the shells lead to stronger attachment to the trapped virus, a phenomenon known as avidity. Pre-incubation of shells with HBV and simultaneous addition of both components separately to cells lead to comparable levels of neutralization, indicating rapid trapping of the virions by the shells. Our study highlights the potential of the DNA shell system to rationally create novel antivirals using components that, when used individually, show little to no antiviral effectiveness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566502">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566502" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566502">
        <p class="paperTitle">3D Mitochondrial Structure in Aging Human Skeletal Muscle: Insights into MFN-2 Mediated Changes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566502" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566502" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scudese, E.; Vue, Z.; Katti, P.; Marshall, A.; Vang, L.; Garza Lopez, E.; Neikirk, K.; Stephens, D.; Hall, D. D.; Rostami, R.; Shao, J.-q.; Mungai, M.; AshShareef, S.; Hicsasmaz, I.; Manus, S.; Whiteside, A.; Williams, C.; Damo, S.; Gaddy, J.; Kirabo, A.; Glancy, B.; Henrique Martin Dantas, E.; Kinder, A.; Scartoni, F.; Baffi, M.; McReynolds, M.; Wanjalla, C. N.; Phillips, M. A.; Exil, V.; Mobley, B.; Hinton, A.</p>
        <p class="info">Score: 14.2, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566502' target='https://doi.org/10.1101/2023.11.13.566502'> 10.1101/2023.11.13.566502</a></p>
        <p class="abstract">Sarcopenia is an age-related loss of skeletal muscle, characterized by loss of mass, strength, endurance, and oxidative capacity during aging. Notably, bioenergetics and protein turnover studies have shown that mitochondria mediate this decline in function. Although mitochondrial aging is associated with decreased mitochondrial capacity, the three-dimensional (3D) mitochondrial structure associated with morphological changes in skeletal muscle during aging still requires further elucidation. Although exercise has been the only therapy to mitigate sarcopenia, the mechanisms that govern these changes remain unclear. We hypothesized that aging causes structural remodeling of mitochondrial 3D architecture representative of dysfunction, and this effect is mitigated by exercise. We used serial block-face scanning electron microscopy to image human skeletal tissue samples, followed by manual contour tracing using Amira software for 3D reconstruction and subsequent analysis of mitochondria. We then applied a rigorous in vitro and in vivo exercise regimen during aging. We found that mitochondria became less complex with age. Specifically, mitochondria lost surface area, complexity, and perimeter, indicating age-related declines in ATP synthesis and interaction capacity. Concomitantly, muscle area, exercise capacity, and mitochondrial dynamic proteins showed age-related losses. Exercise stimulation restored mitofusin 2 (MFN2), which we show is required for mitochondrial structure. Furthermore, we show that this pathway is evolutionarily conserved with Marf, the MFN2 ortholog in Drosophila, as Marf knockdown alters mitochondrial morphology and leads to the downregulation of genes regulating mitochondrial processes. Our results define age-related structural changes in mitochondria and further suggest that exercise may mitigate age-related structural decline through modulation of mitofusins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572292">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572292" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572292">
        <p class="paperTitle">GSK3β phosphorylation catalyzes the aggregation of Tau into Alzheimer&#39;s disease-like amyloid strain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572292" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572292" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chakraborty, P.; Ibanez de Opakua, A.; Purslow, J. A.; Fromm, S. A.; Chatterjee, D.; Zachrdla, M.; Puri, S.; Wolozin, B.; Zweckstetter, M.</p>
        <p class="info">Score: 10.4, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572292' target='https://doi.org/10.1101/2023.12.19.572292'> 10.1101/2023.12.19.572292</a></p>
        <p class="abstract">The pathological deposition of proteins is a hallmark of several devastating neurodegenerative diseases. These pathological deposits comprise aggregates of proteins that adopt distinct structures named strains. However, the molecular factors responsible for the formation of distinct aggregate strains are unknown. Here we show that the serine/threonine kinase GSK3{beta} catalyzes the aggregation of the protein tau into an Alzheimers disease-like amyloid strain. We demonstrate that phosphorylation by GSK3{beta}, but not by several other kinases, promotes the aggregation of full-length tau through enhanced phase separation into gel-like condensate structures. Cryo-electron microscopy further reveals that the amyloid fibrils formed by GSK3{beta}-phosphorylated tau adopt a fold comparable to that of paired helical filaments isolated from the brains of AD patients. Our results elucidate the intricate relationship between post-translational modification and the formation of tau strains in neurodegenerative diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572875">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572875" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572875">
        <p class="paperTitle">Structure of the human systemic RNAi defective transmembrane protein 1 (hSIDT1) reveals the conformational flexibility of its lipid binding domain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572875" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572875" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Navratna, V.; Kumar, A.; Rana, J. K.; Mosalaganti, S.</p>
        <p class="info">Score: 9.1, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572875' target='https://doi.org/10.1101/2023.12.21.572875'> 10.1101/2023.12.21.572875</a></p>
        <p class="abstract">In C. elegans, the inter-cellular transport of the small non-coding RNA that causes systemic RNA interference (RNAi) is mediated by the transmembrane protein SID1, encoded by the sid1 gene in the systemic RNA interference-defective (sid) loci. SID1 shares remarkable structural and sequence similarity with cholesterol uptake protein 1 (CHUP1), and they are classified together as members of the cholesterol uptake family (ChUP). Although systemic RNAi is not an evolutionarily conserved process, the sid gene products are found across the animal kingdom, suggesting a potential for the existence of other novel gene regulatory mechanisms mediated by small non-coding RNAs. Human homologs of sid gene products, hSIDT1 and hSIDT2, are believed to mediate contact-dependent lipophilic small non-coding dsRNA transport. Here, we report the structure of recombinant full-length human SIDT1. We find that the extra-cytosolic domain (ECD) of hSIDT1 adopts a double jelly roll fold, and the transmembrane domain (TMD) exists as two modules, a flexible lipid binding domain (LBD) and a rigid TMD core. Our structural analyses provide insights into the potential role of the lipid binding domain in the function of hSIDT1 and other cholesterol uptake (ChUP) family members.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.571997">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.571997" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.571997">
        <p class="paperTitle">Conformational sampling and interpolation using language-based protein folding neural networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.571997" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.571997" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: del Alamo, D.; Jeliazkov, J. R.; Truan, D.; Karpiak, J.</p>
        <p class="info">Score: 8.9, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.571997' target='https://doi.org/10.1101/2023.12.16.571997'> 10.1101/2023.12.16.571997</a></p>
        <p class="abstract">Protein language models (PLMs), such ESM2, learn a rich semantic grammar of the protein sequence space. When coupled to protein folding neural networks (e.g., ESMFold), they can facilitate the prediction of tertiary and quaternary protein structures at high accuracy. However, they are limited to modeling protein structures in single states. This manuscript demonstrates that ESMFold can predict alternate conformations of some proteins, including de novo designed proteins. Randomly masking the sequence prior to PLM input returned alternate embeddings that ESMFold sometimes mapped to distinct physiologically relevant conformations. From there, inversion of the ESMFold trunk facilitated the generation of high-confidence interconversion paths between the two states. These paths provide a deeper glimpse of how language-based protein folding neural networks derive structural information from high-dimensional sequence representations, while exposing limitations in their general understanding of protein structure and folding.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572148">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572148" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572148">
        <p class="paperTitle">RASP: Optimal single fluorescent puncta detection in complex cellular backgrounds</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572148" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572148" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fu, B.; Brock, E. E.; Andrews, R.; Breiter, J. C.; Tian, R.; Toomey, C. E.; Lachica, J.; Lashley, T.; Ryten, M.; Wood, N. W.; Vendruscolo, M.; Gandhi, S.; Weiss, L. E.; Beckwith, J. S.; Lee, S. F.</p>
        <p class="info">Score: 8.8, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572148' target='https://doi.org/10.1101/2023.12.18.572148'> 10.1101/2023.12.18.572148</a></p>
        <p class="abstract">Super-resolution and single-molecule microscopy are increasingly applied to complex biological systems. A major challenge of this approach is that fluorescent puncta must be detected in the low signal, high noise, heterogeneous background environments of cells and tissue. We present RASP, Radiality Analysis of Single Puncta, a bioimaging-segmentation method that solves this problem. RASP removes false positive puncta that other analysis methods detect, and detects features over a broad range of spatial scales: from single proteins to complex cell phenotypes. RASP outperforms the state-of-the-art in precision and speed, using image gradients to separate Gaussian-shaped objects from background. We demonstrate RASPs power by showing it can extract spatial correlations between microglia, neurons, and -synuclein oligomers in the human brain. This sensitive, computationally efficient approach enables fluorescent puncta and cellular features to be distinguished in cellular and tissue environments with a sensitivity down to the level of the single protein.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
