<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The mechanism of mRNA activation
Authors: Gentry, R. C.; Ide, N. A.; Comunale, V. M.; Hartwick, E. W.; Kinz-Thompson, C. D.; Gonzalez, R. L.
Score: 21.9, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567265
During translation initiation, messenger RNA molecules must be identified and activated for loading into a ribosome. In this rate-limiting step, the heterotrimeric protein eukaryotic initiation factor eIF4F must recognize and productively interact with the 7-methylguanosine cap at the 5&#39; end of the messenger RNA and subsequently activate the message.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="The mechanism of mRNA activation
Authors: Gentry, R. C.; Ide, N. A.; Comunale, V. M.; Hartwick, E. W.; Kinz-Thompson, C. D.; Gonzalez, R. L.
Score: 21.9, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567265
During translation initiation, messenger RNA molecules must be identified and activated for loading into a ribosome. In this rate-limiting step, the heterotrimeric protein eukaryotic initiation factor eIF4F must recognize and productively interact with the 7-methylguanosine cap at the 5&#39; end of the messenger RNA and subsequently activate the message." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="The mechanism of mRNA activation
Authors: Gentry, R. C.; Ide, N. A.; Comunale, V. M.; Hartwick, E. W.; Kinz-Thompson, C. D.; Gonzalez, R. L.
Score: 21.9, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567265
During translation initiation, messenger RNA molecules must be identified and activated for loading into a ribosome. In this rate-limiting step, the heterotrimeric protein eukaryotic initiation factor eIF4F must recognize and productively interact with the 7-methylguanosine cap at the 5&#39; end of the messenger RNA and subsequently activate the message."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "The mechanism of mRNA activation\nAuthors: Gentry, R. C.; Ide, N. A.; Comunale, V. M.; Hartwick, E. W.; Kinz-Thompson, C. D.; Gonzalez, R. L.\nScore: 21.9, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567265\nDuring translation initiation, messenger RNA molecules must be identified and activated for loading into a ribosome. In this rate-limiting step, the heterotrimeric protein eukaryotic initiation factor eIF4F must recognize and productively interact with the 7-methylguanosine cap at the 5\u0026#39; end of the messenger RNA and subsequently activate the message.",
  "keywords": [
    
  ],
  "articleBody": " The mechanism of mRNA activation\nAuthors: Gentry, R. C.; Ide, N. A.; Comunale, V. M.; Hartwick, E. W.; Kinz-Thompson, C. D.; Gonzalez, R. L.\nScore: 21.9, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567265\nDuring translation initiation, messenger RNA molecules must be identified and activated for loading into a ribosome. In this rate-limiting step, the heterotrimeric protein eukaryotic initiation factor eIF4F must recognize and productively interact with the 7-methylguanosine cap at the 5' end of the messenger RNA and subsequently activate the message. Despite its fundamental, regulatory role in gene expression, the molecular events underlying cap recognition and messenger RNA activation remain mysterious. Here, we generate a unique, single-molecule fluorescence imaging system to interrogate the dynamics with which eIF4F discriminates productive and non-productive locations on full-length, native messenger RNA molecules. At the single-molecule level, we observe stochastic sampling of eIF4F along the length of the messenger RNA and identify allosteric communication between the eIF4F subunits which ultimately drive cap-recognition and subsequent activation of the message. Our experiments uncover novel functions for each subunit of eIF4F and we conclude by presenting a model for messenger RNA activation which precisely defines the composition of the activated message. This model provides a general framework for understanding how messenger RNA molecules may be discriminated from one another, and how other RNA-binding proteins may control the efficiency of translation initiation.\nSpatially resolved DNP-assisted NMR illuminates the conformational ensemble of Î±-synuclein in intact viable cells.\nAuthors: Kragelj, J.; Ghosh, R.; Xiao, Y.; Dumarieh, R.; Lagasca, D.; Krishna, S.; Frederick, K. K.\nScore: 12.1, Published: 2023-10-25 DOI: 10.1101/2023.10.24.563877\nThe protein -syn adopts a wide variety of conformations including an intrinsically disordered monomeric form and an -helical rich membrane-associated form that is thought to play an important role in cellular membrane processes. However, despite the high affinity of -syn for membranes, evidence that the -helical form of -syn is adopted inside cells has thus far been indirect. In cell DNP-assisted solid state NMR on frozen samples has the potential to report directly on the entire conformational ensemble. Moreover, because the DNP polarization agent can be dispersed both homogenously and inhomogenously throughout the cellular biomass, in cell DNP-assisted solid state NMR experiments can report either quantitatively upon the structural ensemble or can preferentially report upon the structural ensemble with a spatial bias. Using DNP-assisted MAS NMR we establish that the spectra of purified -syn in the membrane-associated and intrinsically disordered forms have distinguishable spectra. When the polarization agent is introduced into cells by electroporation and dispersed homogenously, a minority of the -syn inside HEK293 cells adopts a highly -helical rich conformation. Alteration of the spatial distribution of the polarization agent preferentially enhances the signal from molecules nearer to the cellular periphery, thus the -helical rich population is preferentially adopted toward the cellular periphery. This demonstrates how selectively altering the spatial distribution of the DNP polarization agent can be a powerful tool for preferential reporting on specific structural ensembles, paving the way for more nuanced investigations into the conformations that proteins adopt in different areas of the cell.\nEngineering Modular and Tunable Single Molecule Sensors by Decoupling Sensing from Signal Output\nAuthors: Grabenhorst, L.; Pfeiffer, M.; Schinkel, T.; Kummerlin, M.; Maglic, J. B.; Bruggenthies, G. A.; Selbach, F.; Murr, A. T.; Tinnefeld, P.; Glembockyte, V.\nScore: 14.5, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565795\nBiosensors play key roles in medical research and diagnostics, but there currently is a lack of sensing platforms that combine easy adaptation to new targets, strategies to tune the response window to relevant analyte concentration ranges and allow for the incorporation of multiple sensing elements to benefit from multivalency. Utilizing a DNA origami nanostructure as a scaffold for arranging the different sensor components, we here propose an approach for the development of modular and tunable single-molecule sensors capable of detecting a variety of biomolecular targets such as nucleic acids, antibodies and restriction enzymes while offering mechanisms to tune the dynamic window, the specificity and the cooperativity of the sensor.\nDeep learning assisted single particle tracking for automated correlation between diffusion and function\nAuthors: Kaestel-Hansen, J.; de Sautu, M.; Saminathan, A.; Scanavachi, G.; Correia, R. F. B. D. C.; Nielsen, A. J.; Bleshoy, S. V.; Boomsma, W.; Kirchhausen, T.; Hatzakis, N. S.\nScore: 4.5, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567393\nSub-cellular diffusion in living systems reflects cellular processes and interactions. Recent advances in optical microscopy allow the tracking of this nanoscale diffusion of individual objects with an unprecedented level of precision. However, the agnostic and automated extraction of functional information from the diffusion of molecules and organelles within the sub-cellular environment, is labor-intensive and poses a significant challenge. Here we introduce DeepSPT, a deep learning framework to interpret the diffusional 2D or 3D temporal behavior of objects in a rapid and efficient manner, agnostically. Demonstrating its versatility, we have applied DeepSPT to automated mapping of the early events of viral infections, identifying distinct types of endosomal organelles, and clathrin-coated pits and vesicles with up to 95% accuracy and within seconds instead of weeks. The fact that DeepSPT effectively extracts biological information from diffusion alone indicates that besides structure, motion encodes function at the molecular and subcellular level.\nRaman microscopy of cryofixed biological specimens for high-resolution and high-sensitivity chemical imaging\nAuthors: Mizushima, K.; Kumamoto, Y.; Tamura, S.; Yamanaka, M.; Mochizuki, K.; Li, M.; Egoshi, S.; Dodo, K.; Harada, Y.; Smith, N. I.; Sodeoka, M.; Tanaka, H.; Fujita, K.\nScore: 4.2, Published: 2023-11-16 DOI: 10.1101/2023.11.15.567077\nRaman microscopy is an emerging molecular imaging technology, yet its signal-to-noise-ratio (SNR) in measurements of biological specimens is severely limited due to the small cross-section of Raman scattering. Here, we present Raman imaging techniques of cryofixed specimens to overcome SNR limitations by enabling long exposure of specimens under highly stabilized low-temperature conditions. The observation of frozen specimens in a cryostat at a constant low temperature immediately after rapid freezing enabled the improvement of SNR and significantly enhanced spatial and spectral resolution. We also confirmed that the cryofixation can preserve physicochemical states of specimens by observing alkyne-labeled coenzyme Q in cytosol and hemeproteins in acute ischemic myocardium, which cannot be done by fixation using chemical reagents. Finally, we applied the technique for multiplex Raman imaging of label-free endogenous molecules and alkyne-tagged molecules in cryofixed HeLa cells, demonstrating its capability of high-content imaging of complex biological phenomena while maintaining physiological conditions.\nUse of High Pressure NMR Spectroscopy to Rapidly Identify Proteins with Internal Ligand-Binding Voids\nAuthors: Gagne, D.; Azad, R.; Aramini, J. M.; Xu, X.; Isiorho, E. A.; Edupuganti, U. R.; Williams, J.; Marcelino, L. P.; Akasaka, K.; Gardner, K. H.\nScore: 10.3, Published: 2023-10-29 DOI: 10.1101/2020.08.25.267195\nSmall molecule binding within internal cavities provides a way to control protein function and structure, as exhibited in numerous natural and artificial settings. Unfortunately, most ways to identify suitable cavities require high-resolution structures a priori and may miss potential cryptic sites. Here we address this limitation via high-pressure solution NMR spectroscopy, taking advantage of the distinctive nonlinear pressure-induced chemical shift changes observed in proteins containing internal cavities and voids. We developed a method to rapidly characterize such nonlinearity among backbone 1H and 15N amide signals without needing to have sequence-specific chemical shift assignments, taking advantage of routinely available 15N-labeled samples, instrumentation, and 2D 1H/15N HSQC experiments. From such data, we find a strong correlation in the site-to-site variability in such nonlinearity with the total void volume within proteins, providing insights useful for prioritizing domains for ligand binding and indicating mode-of-action among such protein/ligand systems. We suggest that this approach provides a rapid and useful way to rapidly assess otherwise hidden dynamic architectures of protein that reflect fundamental properties associated with ligand binding and control. Significance StatementMany proteins can be regulated by internally binding small molecule ligands, but it is often not clear a priori which proteins are controllable in such a way. Here we describe a rapid method to address this challenge, using solution NMR spectroscopy to monitor the response of proteins to the application of high pressure. While the locations of NMR signals from most proteins respond to high pressure with linear chemical shift changes, proteins containing internal cavities that can bind small molecule ligands respond with easily identified non-linear changes. We demonstrate this approach on several proteins and protein/ligand complexes, suggesting that it has general utility.\nDilated cardiomyopathy mutation in beta-cardiac myosin enhances actin activation of the power stroke and phosphate release\nAuthors: Bodt, S. M.; Ge, J.; Ma, W.; Rasicci, D. V.; Desetty, R.; McCammon, J. A.; Yengo, C. M.\nScore: 3.0, Published: 2023-11-13 DOI: 10.1101/2023.11.10.566646\nInherited mutations in human beta-cardiac myosin (M2{beta}) can lead to severe forms of heart failure. The E525K mutation in M2{beta} is associated with dilated cardiomyopathy (DCM) and was found to stabilize the interacting heads motif (IHM) and autoinhibited super-relaxed (SRX) state in dimeric heavy meromyosin. However, in monomeric M2{beta} subfragment 1 (S1) we found that E525K enhances (3-fold) the maximum steady-state actin-activated ATPase activity (kcat) and decreases (6-fold) the actin concentration at which ATPase is one-half maximal (KATPase). We also found a 3 to 4-fold increase in the actin-activated power stroke and phosphate release rate constants at 30 M actin, which overall enhanced the duty ratio 3-fold. Loaded motility assays revealed that the enhanced intrinsic motor activity translates to increased ensemble force in M2{beta} S1. Glutamate 525, located near the actin binding region in the so-called activation loop, is highly conserved and predicted to form a salt-bridge with another conserved residue (lysine 484) in the relay helix. Enhanced sampling molecular dynamics simulations predict that the charge reversal mutation disrupts the E525-K484 salt-bridge, inducing conformations with a more flexible relay helix and a wide phosphate release tunnel. Our results highlight a highly conserved allosteric pathway associated with actin activation of the power stroke and phosphate release and suggest an important feature of the autoinhibited IHM is to prevent this region of myosin from interacting with actin. The ability of the E525K mutation to stabilize the IHM likely overrides the enhanced intrinsic motor properties, which may be key to triggering DCM pathogenesis. Significance StatementHeart disease can be caused by inherited mutations in beta-cardiac myosin, the molecular motor that powers systolic contraction in the ventricles of the heart. However, it remains unclear how these mutations lead to contractile dysfunction and pathogenic remodeling of the heart. We investigated a unique dilated cardiomyopathy mutation (E525K) that dramatically stabilizes the autoinhibited state while enhancing intrinsic motor function. Thus, we examined how this mutation impacts transient kinetic steps of the ATPase cycle, motile properties, and structural changes associated with the power stroke and phosphate release. Our results provide a kinetic and structural basis for how beta-cardiac myosin mutations may disrupt molecular-level contractile function in complex ways, which may inform the development of targeted therapeutics.\nSingle protein molecules separation, tracking and counting in ultra-thin silicon channels\nAuthors: Ohayon, S.; Taib, L.; Verma, N. C.; Iarossi, M.; Bhattacharya, I.; Marom, B.; Huttner, D.; Meller, A.\nScore: 3.0, Published: 2023-11-13 DOI: 10.1101/2023.11.09.566381\nEmerging single-molecule protein sensing techniques are ushering in a transformative era in biomedical research. Nevertheless, challenges persist in realizing ultra-fast full-size protein sensing, including loss of molecular integrity due to protein fragmentation, biases introduced by antibodies affinity, identification of proteoforms and low throughputs. Here, we introduce a single-molecule method for parallel protein separation and tracking, yielding multi-dimensional molecular properties used for their identification. Proteins are tagged by dual amino-acid specific labels and are electrophoretically separated by their mass/charge in custom-designed silicon nano-channel. This approach allows us to analyze thousands of individual proteins within a few minutes by tracking their motion during the migration. We demonstrate the power of the method by quantifying a cytokine panel for host-response discrimination between viral and bacterial infections. Moreover, we show that two clinically-relevant splice isoforms of VEGF can be accurately quantified from human serum samples. Being non-destructive and compatible with full-length intact proteins, this method opens up new ways for antibody-free single protein molecule quantification.\nThe Role of Electron Spin-Polarizability and Charge Dynamics in Protein Function\nAuthors: Vyas, P.; Santra, K.; Preeyanka, N.; Gupta, A.; Weil-Ktorza, O.; Zhu, Q.; Longo, L. M.; Fransson, J.; Metanis, N.; Naaman, R.\nScore: 2.9, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567399\nWe investigated the association of two model primordial polypeptides, each bearing an ancient and ubiquitous phosphate-binding motif, with DNA. The association rate and the amount of bound DNA relates to the electron spin-dependent polarizability of the binding protein. Contributions to protein polarizability are also made by residues that are far from the spin injection site. The spin-dependence is associated with the chirality of the proteins and is a consequence of chiral-induced spin selectivity (CISS). Since the polypeptides studied are hypothesized to be among the initial stand-alone 'seed' fragments from which contemporary protein domains evolved, the results suggest the importance of spin-dependent polarizability early in protein evolution. As the spin effect is a direct consequence of biopolymer homochirality, it was likely significant from even the earliest stages of biological evolution. We provide model calculations clarifying the underlying reasons for spin selectivity and estimating its impact on protein function.\n3D Mitochondrial Structure in Aging Human Skeletal Muscle: Insights into MFN-2 Mediated Changes\nAuthors: Scudese, E.; Vue, Z.; Katti, P.; Marshall, A.; Vang, L.; Garza Lopez, E.; Neikirk, K.; Stephens, D.; Hall, D. D.; Rostami, R.; Shao, J.-q.; Mungai, M.; AshShareef, S.; Hicsasmaz, I.; Manus, S.; Whiteside, A.; Williams, C.; Damo, S.; Gaddy, J.; Kirabo, A.; Glancy, B.; Henrique Martin Dantas, E.; Kinder, A.; Scartoni, F.; Baffi, M.; McReynolds, M.; Wanjalla, C. N.; Phillips, M. A.; Exil, V.; Mobley, B.; Hinton, A.\nScore: 2.9, Published: 2023-11-15 DOI: 10.1101/2023.11.13.566502\nSarcopenia is an age-related loss of skeletal muscle, characterized by loss of mass, strength, endurance, and oxidative capacity during aging. Notably, bioenergetics and protein turnover studies have shown that mitochondria mediate this decline in function. Although mitochondrial aging is associated with endoplasmic reticulum stress, fragmented mitochondria, and decreased mitochondrial capacity, the three-dimensional (3D) mitochondrial structure associated with morphological changes in skeletal muscle during aging still requires further elucidation. Moreover, the mechanisms by which 3D mitochondrial networks and mitochondrial specialization are altered during aging are not completely understood. Although exercise has been the only therapy to mitigate sarcopenia, the mechanisms that govern these changes remain unclear. We hypothesized that aging causes structural remodeling of mitochondrial 3D architecture representative of dysfunction, and this effect is mitigated by exercise. We used serial block-face scanning electron microscopy to image human skeletal tissue samples, followed by manual contour tracing using Amira software for 3D reconstruction and subsequent analysis of mitochondria. We then applied a rigorous in vitro and in vivo exercise regimen during aging. We found that mitochondria became less complex with age. Specifically, mitochondria lost surface area, complexity, and perimeter, indicating age-related declines in ATP synthesis and interaction capacity. Concomitantly, muscle area, exercise capacity, and mitochondrial dynamic proteins showed age-related losses. Exercise stimulation restored mitofusin 2 (MFN2), which we show is required for mitochondrial structure. Furthermore, we show that this pathway is evolutionarily conserved with Marf, the MFN2 ortholog in Drosophila, as Marf knockdown alters mitochondrial morphology and leads to the downregulation of genes regulating mitochondrial processes. Our results define age-related structural changes in mitochondria and further suggest that exercise may mitigate age-related structural decline through modulation of mitofusins.\n",
  "wordCount" : "2523",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567265" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567265">
        <p class="paperTitle">The mechanism of mRNA activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gentry, R. C.; Ide, N. A.; Comunale, V. M.; Hartwick, E. W.; Kinz-Thompson, C. D.; Gonzalez, R. L.</p>
        <p class="info">Score: 21.9, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567265' target='https://doi.org/10.1101/2023.11.15.567265'> 10.1101/2023.11.15.567265</a></p>
        <p class="abstract">During translation initiation, messenger RNA molecules must be identified and activated for loading into a ribosome. In this rate-limiting step, the heterotrimeric protein eukaryotic initiation factor eIF4F must recognize and productively interact with the 7-methylguanosine cap at the 5&#39; end of the messenger RNA and subsequently activate the message. Despite its fundamental, regulatory role in gene expression, the molecular events underlying cap recognition and messenger RNA activation remain mysterious. Here, we generate a unique, single-molecule fluorescence imaging system to interrogate the dynamics with which eIF4F discriminates productive and non-productive locations on full-length, native messenger RNA molecules. At the single-molecule level, we observe stochastic sampling of eIF4F along the length of the messenger RNA and identify allosteric communication between the eIF4F subunits which ultimately drive cap-recognition and subsequent activation of the message. Our experiments uncover novel functions for each subunit of eIF4F and we conclude by presenting a model for messenger RNA activation which precisely defines the composition of the activated message. This model provides a general framework for understanding how messenger RNA molecules may be discriminated from one another, and how other RNA-binding proteins may control the efficiency of translation initiation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563877">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563877" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563877">
        <p class="paperTitle">Spatially resolved DNP-assisted NMR illuminates the conformational ensemble of Î±-synuclein in intact viable cells.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563877" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563877" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kragelj, J.; Ghosh, R.; Xiao, Y.; Dumarieh, R.; Lagasca, D.; Krishna, S.; Frederick, K. K.</p>
        <p class="info">Score: 12.1, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563877' target='https://doi.org/10.1101/2023.10.24.563877'> 10.1101/2023.10.24.563877</a></p>
        <p class="abstract">The protein -syn adopts a wide variety of conformations including an intrinsically disordered monomeric form and an -helical rich membrane-associated form that is thought to play an important role in cellular membrane processes. However, despite the high affinity of -syn for membranes, evidence that the -helical form of -syn is adopted inside cells has thus far been indirect. In cell DNP-assisted solid state NMR on frozen samples has the potential to report directly on the entire conformational ensemble. Moreover, because the DNP polarization agent can be dispersed both homogenously and inhomogenously throughout the cellular biomass, in cell DNP-assisted solid state NMR experiments can report either quantitatively upon the structural ensemble or can preferentially report upon the structural ensemble with a spatial bias. Using DNP-assisted MAS NMR we establish that the spectra of purified -syn in the membrane-associated and intrinsically disordered forms have distinguishable spectra. When the polarization agent is introduced into cells by electroporation and dispersed homogenously, a minority of the -syn inside HEK293 cells adopts a highly -helical rich conformation. Alteration of the spatial distribution of the polarization agent preferentially enhances the signal from molecules nearer to the cellular periphery, thus the -helical rich population is preferentially adopted toward the cellular periphery. This demonstrates how selectively altering the spatial distribution of the DNP polarization agent can be a powerful tool for preferential reporting on specific structural ensembles, paving the way for more nuanced investigations into the conformations that proteins adopt in different areas of the cell.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565795">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565795" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565795">
        <p class="paperTitle">Engineering Modular and Tunable Single Molecule Sensors by Decoupling Sensing from Signal Output</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565795" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565795" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grabenhorst, L.; Pfeiffer, M.; Schinkel, T.; Kummerlin, M.; Maglic, J. B.; Bruggenthies, G. A.; Selbach, F.; Murr, A. T.; Tinnefeld, P.; Glembockyte, V.</p>
        <p class="info">Score: 14.5, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565795' target='https://doi.org/10.1101/2023.11.06.565795'> 10.1101/2023.11.06.565795</a></p>
        <p class="abstract">Biosensors play key roles in medical research and diagnostics, but there currently is a lack of sensing platforms that combine easy adaptation to new targets, strategies to tune the response window to relevant analyte concentration ranges and allow for the incorporation of multiple sensing elements to benefit from multivalency. Utilizing a DNA origami nanostructure as a scaffold for arranging the different sensor components, we here propose an approach for the development of modular and tunable single-molecule sensors capable of detecting a variety of biomolecular targets such as nucleic acids, antibodies and restriction enzymes while offering mechanisms to tune the dynamic window, the specificity and the cooperativity of the sensor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567393">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567393" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567393">
        <p class="paperTitle">Deep learning assisted single particle tracking for automated correlation between diffusion and function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567393" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567393" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaestel-Hansen, J.; de Sautu, M.; Saminathan, A.; Scanavachi, G.; Correia, R. F. B. D. C.; Nielsen, A. J.; Bleshoy, S. V.; Boomsma, W.; Kirchhausen, T.; Hatzakis, N. S.</p>
        <p class="info">Score: 4.5, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567393' target='https://doi.org/10.1101/2023.11.16.567393'> 10.1101/2023.11.16.567393</a></p>
        <p class="abstract">Sub-cellular diffusion in living systems reflects cellular processes and interactions. Recent advances in optical microscopy allow the tracking of this nanoscale diffusion of individual objects with an unprecedented level of precision. However, the agnostic and automated extraction of functional information from the diffusion of molecules and organelles within the sub-cellular environment, is labor-intensive and poses a significant challenge. Here we introduce DeepSPT, a deep learning framework to interpret the diffusional 2D or 3D temporal behavior of objects in a rapid and efficient manner, agnostically. Demonstrating its versatility, we have applied DeepSPT to automated mapping of the early events of viral infections, identifying distinct types of endosomal organelles, and clathrin-coated pits and vesicles with up to 95% accuracy and within seconds instead of weeks. The fact that DeepSPT effectively extracts biological information from diffusion alone indicates that besides structure, motion encodes function at the molecular and subcellular level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567077">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567077" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567077">
        <p class="paperTitle">Raman microscopy of cryofixed biological specimens for high-resolution and high-sensitivity chemical imaging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567077" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567077" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mizushima, K.; Kumamoto, Y.; Tamura, S.; Yamanaka, M.; Mochizuki, K.; Li, M.; Egoshi, S.; Dodo, K.; Harada, Y.; Smith, N. I.; Sodeoka, M.; Tanaka, H.; Fujita, K.</p>
        <p class="info">Score: 4.2, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567077' target='https://doi.org/10.1101/2023.11.15.567077'> 10.1101/2023.11.15.567077</a></p>
        <p class="abstract">Raman microscopy is an emerging molecular imaging technology, yet its signal-to-noise-ratio (SNR) in measurements of biological specimens is severely limited due to the small cross-section of Raman scattering. Here, we present Raman imaging techniques of cryofixed specimens to overcome SNR limitations by enabling long exposure of specimens under highly stabilized low-temperature conditions. The observation of frozen specimens in a cryostat at a constant low temperature immediately after rapid freezing enabled the improvement of SNR and significantly enhanced spatial and spectral resolution. We also confirmed that the cryofixation can preserve physicochemical states of specimens by observing alkyne-labeled coenzyme Q in cytosol and hemeproteins in acute ischemic myocardium, which cannot be done by fixation using chemical reagents. Finally, we applied the technique for multiplex Raman imaging of label-free endogenous molecules and alkyne-tagged molecules in cryofixed HeLa cells, demonstrating its capability of high-content imaging of complex biological phenomena while maintaining physiological conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2020.08.25.267195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2020.08.25.267195" aria-expanded="false" aria-controls="flush-collapse10.1101/2020.08.25.267195">
        <p class="paperTitle">Use of High Pressure NMR Spectroscopy to Rapidly Identify Proteins with Internal Ligand-Binding Voids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2020.08.25.267195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2020.08.25.267195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gagne, D.; Azad, R.; Aramini, J. M.; Xu, X.; Isiorho, E. A.; Edupuganti, U. R.; Williams, J.; Marcelino, L. P.; Akasaka, K.; Gardner, K. H.</p>
        <p class="info">Score: 10.3, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2020.08.25.267195' target='https://doi.org/10.1101/2020.08.25.267195'> 10.1101/2020.08.25.267195</a></p>
        <p class="abstract">Small molecule binding within internal cavities provides a way to control protein function and structure, as exhibited in numerous natural and artificial settings. Unfortunately, most ways to identify suitable cavities require high-resolution structures a priori and may miss potential cryptic sites. Here we address this limitation via high-pressure solution NMR spectroscopy, taking advantage of the distinctive nonlinear pressure-induced chemical shift changes observed in proteins containing internal cavities and voids. We developed a method to rapidly characterize such nonlinearity among backbone 1H and 15N amide signals without needing to have sequence-specific chemical shift assignments, taking advantage of routinely available 15N-labeled samples, instrumentation, and 2D 1H/15N HSQC experiments. From such data, we find a strong correlation in the site-to-site variability in such nonlinearity with the total void volume within proteins, providing insights useful for prioritizing domains for ligand binding and indicating mode-of-action among such protein/ligand systems. We suggest that this approach provides a rapid and useful way to rapidly assess otherwise hidden dynamic architectures of protein that reflect fundamental properties associated with ligand binding and control.

Significance StatementMany proteins can be regulated by internally binding small molecule ligands, but it is often not clear a priori which proteins are controllable in such a way. Here we describe a rapid method to address this challenge, using solution NMR spectroscopy to monitor the response of proteins to the application of high pressure. While the locations of NMR signals from most proteins respond to high pressure with linear chemical shift changes, proteins containing internal cavities that can bind small molecule ligands respond with easily identified non-linear changes. We demonstrate this approach on several proteins and protein/ligand complexes, suggesting that it has general utility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566646">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566646" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566646">
        <p class="paperTitle">Dilated cardiomyopathy mutation in beta-cardiac myosin enhances actin activation of the power stroke and phosphate release</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566646" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566646" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bodt, S. M.; Ge, J.; Ma, W.; Rasicci, D. V.; Desetty, R.; McCammon, J. A.; Yengo, C. M.</p>
        <p class="info">Score: 3.0, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566646' target='https://doi.org/10.1101/2023.11.10.566646'> 10.1101/2023.11.10.566646</a></p>
        <p class="abstract">Inherited mutations in human beta-cardiac myosin (M2{beta}) can lead to severe forms of heart failure. The E525K mutation in M2{beta} is associated with dilated cardiomyopathy (DCM) and was found to stabilize the interacting heads motif (IHM) and autoinhibited super-relaxed (SRX) state in dimeric heavy meromyosin. However, in monomeric M2{beta} subfragment 1 (S1) we found that E525K enhances (3-fold) the maximum steady-state actin-activated ATPase activity (kcat) and decreases (6-fold) the actin concentration at which ATPase is one-half maximal (KATPase). We also found a 3 to 4-fold increase in the actin-activated power stroke and phosphate release rate constants at 30 M actin, which overall enhanced the duty ratio 3-fold. Loaded motility assays revealed that the enhanced intrinsic motor activity translates to increased ensemble force in M2{beta} S1. Glutamate 525, located near the actin binding region in the so-called activation loop, is highly conserved and predicted to form a salt-bridge with another conserved residue (lysine 484) in the relay helix. Enhanced sampling molecular dynamics simulations predict that the charge reversal mutation disrupts the E525-K484 salt-bridge, inducing conformations with a more flexible relay helix and a wide phosphate release tunnel. Our results highlight a highly conserved allosteric pathway associated with actin activation of the power stroke and phosphate release and suggest an important feature of the autoinhibited IHM is to prevent this region of myosin from interacting with actin. The ability of the E525K mutation to stabilize the IHM likely overrides the enhanced intrinsic motor properties, which may be key to triggering DCM pathogenesis.

Significance StatementHeart disease can be caused by inherited mutations in beta-cardiac myosin, the molecular motor that powers systolic contraction in the ventricles of the heart. However, it remains unclear how these mutations lead to contractile dysfunction and pathogenic remodeling of the heart. We investigated a unique dilated cardiomyopathy mutation (E525K) that dramatically stabilizes the autoinhibited state while enhancing intrinsic motor function. Thus, we examined how this mutation impacts transient kinetic steps of the ATPase cycle, motile properties, and structural changes associated with the power stroke and phosphate release. Our results provide a kinetic and structural basis for how beta-cardiac myosin mutations may disrupt molecular-level contractile function in complex ways, which may inform the development of targeted therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566381">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566381" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566381">
        <p class="paperTitle">Single protein molecules separation, tracking and counting in ultra-thin silicon channels</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566381" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566381" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ohayon, S.; Taib, L.; Verma, N. C.; Iarossi, M.; Bhattacharya, I.; Marom, B.; Huttner, D.; Meller, A.</p>
        <p class="info">Score: 3.0, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566381' target='https://doi.org/10.1101/2023.11.09.566381'> 10.1101/2023.11.09.566381</a></p>
        <p class="abstract">Emerging single-molecule protein sensing techniques are ushering in a transformative era in biomedical research. Nevertheless, challenges persist in realizing ultra-fast full-size protein sensing, including loss of molecular integrity due to protein fragmentation, biases introduced by antibodies affinity, identification of proteoforms and low throughputs. Here, we introduce a single-molecule method for parallel protein separation and tracking, yielding multi-dimensional molecular properties used for their identification. Proteins are tagged by dual amino-acid specific labels and are electrophoretically separated by their mass/charge in custom-designed silicon nano-channel. This approach allows us to analyze thousands of individual proteins within a few minutes by tracking their motion during the migration. We demonstrate the power of the method by quantifying a cytokine panel for host-response discrimination between viral and bacterial infections. Moreover, we show that two clinically-relevant splice isoforms of VEGF can be accurately quantified from human serum samples. Being non-destructive and compatible with full-length intact proteins, this method opens up new ways for antibody-free single protein molecule quantification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567399">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567399" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567399">
        <p class="paperTitle">The Role of Electron Spin-Polarizability and Charge Dynamics in Protein Function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567399" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567399" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vyas, P.; Santra, K.; Preeyanka, N.; Gupta, A.; Weil-Ktorza, O.; Zhu, Q.; Longo, L. M.; Fransson, J.; Metanis, N.; Naaman, R.</p>
        <p class="info">Score: 2.9, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567399' target='https://doi.org/10.1101/2023.11.16.567399'> 10.1101/2023.11.16.567399</a></p>
        <p class="abstract">We investigated the association of two model primordial polypeptides, each bearing an ancient and ubiquitous phosphate-binding motif, with DNA. The association rate and the amount of bound DNA relates to the electron spin-dependent polarizability of the binding protein. Contributions to protein polarizability are also made by residues that are far from the spin injection site. The spin-dependence is associated with the chirality of the proteins and is a consequence of chiral-induced spin selectivity (CISS). Since the polypeptides studied are hypothesized to be among the initial stand-alone &#39;seed&#39; fragments from which contemporary protein domains evolved, the results suggest the importance of spin-dependent polarizability early in protein evolution. As the spin effect is a direct consequence of biopolymer homochirality, it was likely significant from even the earliest stages of biological evolution. We provide model calculations clarifying the underlying reasons for spin selectivity and estimating its impact on protein function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566502">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566502" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566502">
        <p class="paperTitle">3D Mitochondrial Structure in Aging Human Skeletal Muscle: Insights into MFN-2 Mediated Changes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566502" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566502" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scudese, E.; Vue, Z.; Katti, P.; Marshall, A.; Vang, L.; Garza Lopez, E.; Neikirk, K.; Stephens, D.; Hall, D. D.; Rostami, R.; Shao, J.-q.; Mungai, M.; AshShareef, S.; Hicsasmaz, I.; Manus, S.; Whiteside, A.; Williams, C.; Damo, S.; Gaddy, J.; Kirabo, A.; Glancy, B.; Henrique Martin Dantas, E.; Kinder, A.; Scartoni, F.; Baffi, M.; McReynolds, M.; Wanjalla, C. N.; Phillips, M. A.; Exil, V.; Mobley, B.; Hinton, A.</p>
        <p class="info">Score: 2.9, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566502' target='https://doi.org/10.1101/2023.11.13.566502'> 10.1101/2023.11.13.566502</a></p>
        <p class="abstract">Sarcopenia is an age-related loss of skeletal muscle, characterized by loss of mass, strength, endurance, and oxidative capacity during aging. Notably, bioenergetics and protein turnover studies have shown that mitochondria mediate this decline in function. Although mitochondrial aging is associated with endoplasmic reticulum stress, fragmented mitochondria, and decreased mitochondrial capacity, the three-dimensional (3D) mitochondrial structure associated with morphological changes in skeletal muscle during aging still requires further elucidation. Moreover, the mechanisms by which 3D mitochondrial networks and mitochondrial specialization are altered during aging are not completely understood. Although exercise has been the only therapy to mitigate sarcopenia, the mechanisms that govern these changes remain unclear. We hypothesized that aging causes structural remodeling of mitochondrial 3D architecture representative of dysfunction, and this effect is mitigated by exercise. We used serial block-face scanning electron microscopy to image human skeletal tissue samples, followed by manual contour tracing using Amira software for 3D reconstruction and subsequent analysis of mitochondria. We then applied a rigorous in vitro and in vivo exercise regimen during aging. We found that mitochondria became less complex with age. Specifically, mitochondria lost surface area, complexity, and perimeter, indicating age-related declines in ATP synthesis and interaction capacity. Concomitantly, muscle area, exercise capacity, and mitochondrial dynamic proteins showed age-related losses. Exercise stimulation restored mitofusin 2 (MFN2), which we show is required for mitochondrial structure. Furthermore, we show that this pathway is evolutionarily conserved with Marf, the MFN2 ortholog in Drosophila, as Marf knockdown alters mitochondrial morphology and leads to the downregulation of genes regulating mitochondrial processes. Our results define age-related structural changes in mitochondria and further suggest that exercise may mitigate age-related structural decline through modulation of mitofusins.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
