<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states
Authors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.
Score: 30.9, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575649
Cell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states
Authors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.
Score: 30.9, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575649
Cell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-21T10:39:08+00:00" />
<meta property="article:modified_time" content="2024-01-21T10:39:08+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states
Authors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.
Score: 30.9, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575649
Cell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states\nAuthors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.\nScore: 30.9, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575649\nCell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies.",
  "keywords": [
    
  ],
  "articleBody": " High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states\nAuthors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.\nScore: 30.9, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575649\nCell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies. Here, we describe a cytometry-based method with single-cell resolution which fills this technological gap by combining biophysical profiling with conventional biomarker analysis. This platform allows to reveal notable cell type-dependent remodelling of membrane fluidity during immune stimulations and in diseases. Using immune cells exposed to tumour microenvironment as well as from long COVID and chronic lymphocyte leukaemia patients, we demonstrate that membrane fluidity is orthogonal to surface marker expression. Moreover, this biophysical parameter identifies new functional and pathological states of immune cells previously undetected via surface marker profiling alone. Our findings will contribute to a more precise definition of immune cell states based on their biophysical properties and will pave the way for a better understanding of the functional heterogeneity of immune cells.\nLocalising elements in single-particle reconstructions by REEL-EM: Reconstructed Electron Energy-Loss - Elemental Mapping\nAuthors: Pfeil-Gardiner, O.; Rosa, H. V. D.; Riedel, D.; Chen, Y. S.; Lorks, D.; Kukelhan, P.; Linck, M.; Muller, H.; Van Petegem, F.; Murphy, B. J.\nScore: 25.6, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575858\nFor structures determined by single particle cryo-EM, no technique currently exists for mapping elements to defined locations, leading to errors in the assignment of metals and other ions, cofactors, substrates, inhibitors, and lipids that play essential roles in activity and regulation. Elemental mapping in the electron microscope is well established for dose-tolerant samples but is challenging for biological samples, especially in a cryo-preserved state. Here, we combine electron energy-loss spectroscopy (EELS) with single-particle image processing to allow elemental mapping in cryo-preserved macromolecular complexes. Proof-of-principle data show that our method, REEL-EM, allows 3D reconstruction of EELS data, such that a high total electron dose is accumulated across many copies of a complex. Working with two test samples, we demonstrate that we can reliably localise abundant elements. We discuss the current limitations of the method and potential future developments.\nHigh-Throughput DNA melt measurements enable improved models of DNA folding thermodynamics\nAuthors: Ke, Y.; Sharma, E.; Wayment-Steele, H. K.; Becker, W. R.; Ho, A.; Marklund, E.; Greenleaf, W. J.\nScore: 16.5, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574731\nDNA folding thermodynamics are central to many biological processes and biotechnological applications involving base-pairing. Current methods for predicting stability from DNA sequence use nearest-neighbor (NN) models that struggle to accurately capture the diverse sequence-dependency of elements other than Watson-Crick base pairs, likely due to insufficient experimental data. We introduce a massively parallel method-Array Melt-that uses fluorescence-based quenching signals to measure equilibrium stability of millions of DNA hairpins simultaneously on a repurposed Illumina sequencing flow cell. By leveraging this dataset of 27,732 sequences with two-state melting behavior, we derived a refined NUPACK-compatible NN model, a richer parameterization NN model that exhibits higher accuracy, and a graph neural network (GNN) model that identifies relevant interactions within DNA beyond nearest neighbors. All models provide improved accuracy in predicting DNA folding thermodynamics, providing improvements relevant for in silico design of qPCR primers, oligo hybridization probes, and DNA origami.\nAstigmatism-based focus stabilisation with universal objective lens compatibility, extended operating range and nanometre precision\nAuthors: Rahmani, A.; Cox, T.; Thamaravelil Abhimanue Achary, A.; Ponjavic, A.\nScore: 12.4, Published: 2024-01-15 DOI: 10.1101/2024.01.15.575442\nFocus stabilisation is vital for long-term fluorescence imaging, particularly in the case of high-resolution imaging techniques. Current stabilisation solutions either rely on fiducial markers that can be perturbative, or on beam reflection monitoring that is limited to high-numerical aperture objective lenses, making multimodal and large-scale imaging challenging. We introduce a beam-based method that relies on astigmatism, which offers advantages in terms of precision and the range over which focus stabilisation is effective. This approach is shown to be compatible with a wide range of objective lenses (10x-100x), typically achieving \u003c10 nm precision with \u003e10 m operating range. Notably, our technique is largely unaffected by pointing stability errors, which in combination with implementation through a standalone Raspberry Pi architecture, offers a versatile focus stabilisation unit that can be added onto most existing microscope setups.\nCryoRhodopsins: a comprehensive characterization of a new clade of microbial rhodopsins from cold environments\nAuthors: Lamm, G. H. U.; Marin, E.; Schellbach, A. V.; Stetsenko, A.; Alekseev, A.; Bourenkov, G.; Borshchevskiy, V.; Asido, M.; Agthe, M.; Engilberge, S.; Rose, S. L.; Caramello, N.; Royant, A.; Schneider, T. R.; Bateman, A. I.; Mager, T.; Moser, T.; Wachtveitl, J.; Guskov, A.; Kovalev, K.\nScore: 9.4, Published: 2024-01-17 DOI: 10.1101/2024.01.15.575777\nMicrobial rhodopsins are omnipresent on Earth, however the vast majority of them remain uncharacterized. Here we describe a new rhodopsin clade from cold-adapted organisms and cold environments, such as glaciers, denoted as CryoRhodopsins (CryoRs). Our data suggest that CryoRs have photosensory activity. A distinguishing feature of the clade is the presence of a buried arginine residue close to the cytoplasmic face of its members. Combining single-particle cryo-electron microscopy and X-ray crystallography with the rhodopsin activation by light, we demonstrate that the arginine stabilizes a strongly blue-shifted intermediate of an extremely slow CryoRhodopsin photocycle. Together with extensive spectroscopic characterization, our investigations on CryoR1 and CryoR2 proteins reveal mechanisms of photoswitching in the newly identified clade and demonstrate principles of the adaptation of these rhodopsins to low temperatures.\nOn-chip large-scale-integration and 2D collective modes of genetically programmed artificial cells\nAuthors: Ricouvier, J.; Mostov, P.; Shabtai, O.; Tayar, A.; Karzbrun, E.; Khakimzhan, A.; Noireaux, V.; Shulman Daube, S.; Bar-Ziv, R.\nScore: 9.4, Published: 2024-01-12 DOI: 10.1101/2024.01.10.575012\nThe on-chip large-scale-integration of genetically programmed artificial cells capable of exhibiting collective modes is an important goal for fundamental research and technology. Here, we report assembly of a 2D layout of 1024 monolithic DNA compartments as artificial cells on a 5-millimeter square silicon chip. Homeostatic cell-free protein synthesis reactions driven by genetic circuits occur inside the compartments. We created a reaction-diffusion system with a 30x30 square lattice of artificial cells interconnected by thin capillaries for diffusion of products. Driving the system by a genetic oscillator revealed emergent collective modes of synchrony and propagating phase waves in 2D, with dynamics controlled by geometry. This demonstrates a class of nonequilibrium autonomous systems, where chemical energy consumed to make proteins induces 2D collective patterns of gene expression on multicellular scales, with applications in biological computing, sensing, and materials synthesis.\nCryo-electron tomography reveals how COPII assembles on cargo-containing membranes\nAuthors: Pyle, E.; Zanetti, G.\nScore: 8.8, Published: 2024-01-17 DOI: 10.1101/2024.01.17.576008\nProteins traverse the eukaryotic secretory pathway via membrane trafficking between organelles. The COPII coat mediates the anterograde transport of newly synthesised proteins from the endoplasmic reticulum, engaging cargoes with wide ranges of sizes and biophysical properties. The native architecture of the COPII coat and the cargo-dependent regulation of its assembly remain poorly understood. Here, we have reconstituted COPII-coated membrane carriers using purified S. cerevisiae proteins and cell-derived microsomes as a native membrane source. Using cryo-electron tomography with subtomogram averaging, we demonstrate that the COPII coat binds cargo and forms largely spherical vesicles from native membranes. We reveal the architecture of the inner and outer coat layers and shed light on how spherical carriers are formed. Our results provide novel insights into the architecture and regulation of the COPII coat and challenge our current understanding of how membrane curvature is generated.\nThe Alzheimer's Aβ peptide forms biomolecular condensates that trigger amyloid aggregation\nAuthors: Sneideriene, G.; Gonzalez Diaz, A.; Adhikari, S. D.; Wei, J.; Michaels, T.; Sneideris, T.; Linse, S.; Vendruscolo, M.; Garai, K.; Knowles, T.\nScore: 8.2, Published: 2024-01-15 DOI: 10.1101/2024.01.14.575549\nThe onset and development of Alzheimers disease (AD) involves the accumulation of pathological aggregates formed from the normally monomeric amyloid-{beta} peptide within the central nervous system. While clinical therapies successfully target these aggregates, the fundamental molecular steps that trigger the initial nucleation event leading to the conversion of monomeric A{beta} peptide into pathological aggregates remain unknown. Here we show that the A{beta} peptide can form biomolecular condensates on lipid bilayers both in molecular assays and in living cells. These A{beta} condensates can significantly accelerate the primary nucleation step in the amyloid conversion cascade that leads to the formation of amyloid aggregates and plaque. We show that A{beta} condensates contain phospholipids, are intrinsically heterogenous, and undergo a liquid-to-solid transition leading to the formation of amyloid fibrils. Unveiling the liquid-liquid phase separation of A{beta} peptide identifies a novel, early molecular step in the AD aggregation cascade, offering potential for innovative therapeutic interventions.\nAdsorption-driven deformation and landing-footprints of the RBD proteins in SARS-CoV-2 variants onto biological and inanimate surfaces\nAuthors: Bosch, A.; V. Guzman, H.; Perez, R.\nScore: 6.7, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575706\nRespiratory viruses, carried through airborne microdroplets, frequently adhere to surfaces, including plastics and metals. However, our understanding of the interactions between viruses and materials remains limited, particularly in scenarios involving polarizable surfaces. Here, we investigate the role of receptor-binding domain (RBD) mutations on the adsorption of SARS-CoV-2 to hydrophobic and hydrophilic surfaces employing molecular simulations. To contextualize our findings, we contrast the interactions on inanimate surfaces with those on native-biological interfaces, specifically the ACE2 receptor. Notably, we identify a twofold increase in structural deformations for the proteins receptor binding motif onto the inanimate surfaces, indicative of enhanced shock-absorbing mechanisms. Furthermore, the distribution of amino acids (landing-footprints) on the inanimate surface reveals a distinct regional asymmetry relative to the biological interface. In spite of the H-bonds formed at the hydrophilic substrate, the simulations consistently show a higher number of contacts and interfacial area with the hydrophobic surface, with the WT RBD adsorbed more strongly to than the delta or omicron RBDs. In contrast, the adsorption of delta and omicron to hydrophilic surfaces was characterized by a distinctive hopping-pattern. The novel shock-absorbing mechanisms identified in the virus adsorption on inanimate surfaces could lead current experimental efforts in the design of virucidal surfaces.\nCorrelating fluorescence microscopy, optical and magnetic tweezers to study single chiral biopolymers, tested on DNA plectoneme formation dynamics\nAuthors: Shepherd, J.; Guilbaud, S.; Zhou, Z.; Howard, J.; Burman, M.; Schaefer, C.; Kerrigan, A.; Steele-King, C.; Noy, A.; Leake, M. C.\nScore: 6.4, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576226\nBiopolymer topology is critical for determining interactions inside cell environments, exemplified by DNA where its response to mechanical perturbation is as important as biochemical properties to its cellular roles. The dynamic structures of chiral biopolymers exhibit complex dependence with extension and torsion, however the physical mechanisms underpinning the emergence of structural motifs upon physiological twisting and stretching are poorly understood due to technological limitations in correlating force, torque and spatial localization information. We present COMBI-Tweez (Combined Optical and Magnetic BIomolecule TWEEZers), a transformative tool that overcomes these challenges by integrating optical trapping, time-resolved electromagnetic tweezers, and fluorescence microscopy, demonstrated on single DNA molecules, that can controllably form and visualise higher order structural motifs including plectonemes. This technology combined with cutting-edge MD simulations provides quantitative insight into complex dynamic structures relevant to DNA cellular processes and can be adapted to study a range of filamentous biopolymers.\n",
  "wordCount" : "1874",
  "inLanguage": "en",
  "datePublished": "2024-01-21T10:39:08Z",
  "dateModified": "2024-01-21T10:39:08Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 21, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575649" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575649">
        <p class="paperTitle">High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.</p>
        <p class="info">Score: 30.9, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575649' target='https://doi.org/10.1101/2024.01.15.575649'> 10.1101/2024.01.15.575649</a></p>
        <p class="abstract">Cell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies. Here, we describe a cytometry-based method with single-cell resolution which fills this technological gap by combining biophysical profiling with conventional biomarker analysis. This platform allows to reveal notable cell type-dependent remodelling of membrane fluidity during immune stimulations and in diseases. Using immune cells exposed to tumour microenvironment as well as from long COVID and chronic lymphocyte leukaemia patients, we demonstrate that membrane fluidity is orthogonal to surface marker expression. Moreover, this biophysical parameter identifies new functional and pathological states of immune cells previously undetected via surface marker profiling alone. Our findings will contribute to a more precise definition of immune cell states based on their biophysical properties and will pave the way for a better understanding of the functional heterogeneity of immune cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.575858">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.575858" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.575858">
        <p class="paperTitle">Localising elements in single-particle reconstructions by REEL-EM: Reconstructed Electron Energy-Loss - Elemental Mapping</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.575858" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.575858" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pfeil-Gardiner, O.; Rosa, H. V. D.; Riedel, D.; Chen, Y. S.; Lorks, D.; Kukelhan, P.; Linck, M.; Muller, H.; Van Petegem, F.; Murphy, B. J.</p>
        <p class="info">Score: 25.6, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.575858' target='https://doi.org/10.1101/2024.01.18.575858'> 10.1101/2024.01.18.575858</a></p>
        <p class="abstract">For structures determined by single particle cryo-EM, no technique currently exists for mapping elements to defined locations, leading to errors in the assignment of metals and other ions, cofactors, substrates, inhibitors, and lipids that play essential roles in activity and regulation. Elemental mapping in the electron microscope is well established for dose-tolerant samples but is challenging for biological samples, especially in a cryo-preserved state. Here, we combine electron energy-loss spectroscopy (EELS) with single-particle image processing to allow elemental mapping in cryo-preserved macromolecular complexes. Proof-of-principle data show that our method, REEL-EM, allows 3D reconstruction of EELS data, such that a high total electron dose is accumulated across many copies of a complex. Working with two test samples, we demonstrate that we can reliably localise abundant elements. We discuss the current limitations of the method and potential future developments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574731">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574731" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574731">
        <p class="paperTitle">High-Throughput DNA melt measurements enable improved models of DNA folding thermodynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574731" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574731" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ke, Y.; Sharma, E.; Wayment-Steele, H. K.; Becker, W. R.; Ho, A.; Marklund, E.; Greenleaf, W. J.</p>
        <p class="info">Score: 16.5, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574731' target='https://doi.org/10.1101/2024.01.08.574731'> 10.1101/2024.01.08.574731</a></p>
        <p class="abstract">DNA folding thermodynamics are central to many biological processes and biotechnological applications involving base-pairing. Current methods for predicting stability from DNA sequence use nearest-neighbor (NN) models that struggle to accurately capture the diverse sequence-dependency of elements other than Watson-Crick base pairs, likely due to insufficient experimental data. We introduce a massively parallel method-Array Melt-that uses fluorescence-based quenching signals to measure equilibrium stability of millions of DNA hairpins simultaneously on a repurposed Illumina sequencing flow cell. By leveraging this dataset of 27,732 sequences with two-state melting behavior, we derived a refined NUPACK-compatible NN model, a richer parameterization NN model that exhibits higher accuracy, and a graph neural network (GNN) model that identifies relevant interactions within DNA beyond nearest neighbors. All models provide improved accuracy in predicting DNA folding thermodynamics, providing improvements relevant for in silico design of qPCR primers, oligo hybridization probes, and DNA origami.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575442">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575442" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575442">
        <p class="paperTitle">Astigmatism-based focus stabilisation with universal objective lens compatibility, extended operating range and nanometre precision</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575442" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575442" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rahmani, A.; Cox, T.; Thamaravelil Abhimanue Achary, A.; Ponjavic, A.</p>
        <p class="info">Score: 12.4, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575442' target='https://doi.org/10.1101/2024.01.15.575442'> 10.1101/2024.01.15.575442</a></p>
        <p class="abstract">Focus stabilisation is vital for long-term fluorescence imaging, particularly in the case of high-resolution imaging techniques. Current stabilisation solutions either rely on fiducial markers that can be perturbative, or on beam reflection monitoring that is limited to high-numerical aperture objective lenses, making multimodal and large-scale imaging challenging. We introduce a beam-based method that relies on astigmatism, which offers advantages in terms of precision and the range over which focus stabilisation is effective. This approach is shown to be compatible with a wide range of objective lenses (10x-100x), typically achieving &lt;10 nm precision with &gt;10 m operating range. Notably, our technique is largely unaffected by pointing stability errors, which in combination with implementation through a standalone Raspberry Pi architecture, offers a versatile focus stabilisation unit that can be added onto most existing microscope setups.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575777" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575777">
        <p class="paperTitle">CryoRhodopsins: a comprehensive characterization of a new clade of microbial rhodopsins from cold environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lamm, G. H. U.; Marin, E.; Schellbach, A. V.; Stetsenko, A.; Alekseev, A.; Bourenkov, G.; Borshchevskiy, V.; Asido, M.; Agthe, M.; Engilberge, S.; Rose, S. L.; Caramello, N.; Royant, A.; Schneider, T. R.; Bateman, A. I.; Mager, T.; Moser, T.; Wachtveitl, J.; Guskov, A.; Kovalev, K.</p>
        <p class="info">Score: 9.4, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575777' target='https://doi.org/10.1101/2024.01.15.575777'> 10.1101/2024.01.15.575777</a></p>
        <p class="abstract">Microbial rhodopsins are omnipresent on Earth, however the vast majority of them remain uncharacterized. Here we describe a new rhodopsin clade from cold-adapted organisms and cold environments, such as glaciers, denoted as CryoRhodopsins (CryoRs). Our data suggest that CryoRs have photosensory activity. A distinguishing feature of the clade is the presence of a buried arginine residue close to the cytoplasmic face of its members. Combining single-particle cryo-electron microscopy and X-ray crystallography with the rhodopsin activation by light, we demonstrate that the arginine stabilizes a strongly blue-shifted intermediate of an extremely slow CryoRhodopsin photocycle. Together with extensive spectroscopic characterization, our investigations on CryoR1 and CryoR2 proteins reveal mechanisms of photoswitching in the newly identified clade and demonstrate principles of the adaptation of these rhodopsins to low temperatures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575012">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575012" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575012">
        <p class="paperTitle">On-chip large-scale-integration and 2D collective modes of genetically programmed artificial cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575012" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575012" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ricouvier, J.; Mostov, P.; Shabtai, O.; Tayar, A.; Karzbrun, E.; Khakimzhan, A.; Noireaux, V.; Shulman Daube, S.; Bar-Ziv, R.</p>
        <p class="info">Score: 9.4, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575012' target='https://doi.org/10.1101/2024.01.10.575012'> 10.1101/2024.01.10.575012</a></p>
        <p class="abstract">The on-chip large-scale-integration of genetically programmed artificial cells capable of exhibiting collective modes is an important goal for fundamental research and technology. Here, we report assembly of a 2D layout of 1024 monolithic DNA compartments as artificial cells on a 5-millimeter square silicon chip. Homeostatic cell-free protein synthesis reactions driven by genetic circuits occur inside the compartments. We created a reaction-diffusion system with a 30x30 square lattice of artificial cells interconnected by thin capillaries for diffusion of products. Driving the system by a genetic oscillator revealed emergent collective modes of synchrony and propagating phase waves in 2D, with dynamics controlled by geometry. This demonstrates a class of nonequilibrium autonomous systems, where chemical energy consumed to make proteins induces 2D collective patterns of gene expression on multicellular scales, with applications in biological computing, sensing, and materials synthesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576008" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576008">
        <p class="paperTitle">Cryo-electron tomography reveals how COPII assembles on cargo-containing membranes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pyle, E.; Zanetti, G.</p>
        <p class="info">Score: 8.8, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576008' target='https://doi.org/10.1101/2024.01.17.576008'> 10.1101/2024.01.17.576008</a></p>
        <p class="abstract">Proteins traverse the eukaryotic secretory pathway via membrane trafficking between organelles. The COPII coat mediates the anterograde transport of newly synthesised proteins from the endoplasmic reticulum, engaging cargoes with wide ranges of sizes and biophysical properties. The native architecture of the COPII coat and the cargo-dependent regulation of its assembly remain poorly understood. Here, we have reconstituted COPII-coated membrane carriers using purified S. cerevisiae proteins and cell-derived microsomes as a native membrane source. Using cryo-electron tomography with subtomogram averaging, we demonstrate that the COPII coat binds cargo and forms largely spherical vesicles from native membranes. We reveal the architecture of the inner and outer coat layers and shed light on how spherical carriers are formed. Our results provide novel insights into the architecture and regulation of the COPII coat and challenge our current understanding of how membrane curvature is generated.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575549">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575549" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575549">
        <p class="paperTitle">The Alzheimer&#39;s Aβ peptide forms biomolecular condensates that trigger amyloid aggregation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575549" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575549" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sneideriene, G.; Gonzalez Diaz, A.; Adhikari, S. D.; Wei, J.; Michaels, T.; Sneideris, T.; Linse, S.; Vendruscolo, M.; Garai, K.; Knowles, T.</p>
        <p class="info">Score: 8.2, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575549' target='https://doi.org/10.1101/2024.01.14.575549'> 10.1101/2024.01.14.575549</a></p>
        <p class="abstract">The onset and development of Alzheimers disease (AD) involves the accumulation of pathological aggregates formed from the normally monomeric amyloid-{beta} peptide within the central nervous system. While clinical therapies successfully target these aggregates, the fundamental molecular steps that trigger the initial nucleation event leading to the conversion of monomeric A{beta} peptide into pathological aggregates remain unknown. Here we show that the A{beta} peptide can form biomolecular condensates on lipid bilayers both in molecular assays and in living cells. These A{beta} condensates can significantly accelerate the primary nucleation step in the amyloid conversion cascade that leads to the formation of amyloid aggregates and plaque. We show that A{beta} condensates contain phospholipids, are intrinsically heterogenous, and undergo a liquid-to-solid transition leading to the formation of amyloid fibrils. Unveiling the liquid-liquid phase separation of A{beta} peptide identifies a novel, early molecular step in the AD aggregation cascade, offering potential for innovative therapeutic interventions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575706" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575706">
        <p class="paperTitle">Adsorption-driven deformation and landing-footprints of the RBD proteins in SARS-CoV-2 variants onto biological and inanimate surfaces</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bosch, A.; V. Guzman, H.; Perez, R.</p>
        <p class="info">Score: 6.7, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575706' target='https://doi.org/10.1101/2024.01.15.575706'> 10.1101/2024.01.15.575706</a></p>
        <p class="abstract">Respiratory viruses, carried through airborne microdroplets, frequently adhere to surfaces, including plastics and metals. However, our understanding of the interactions between viruses and materials remains limited, particularly in scenarios involving polarizable surfaces. Here, we investigate the role of receptor-binding domain (RBD) mutations on the adsorption of SARS-CoV-2 to hydrophobic and hydrophilic surfaces employing molecular simulations. To contextualize our findings, we contrast the interactions on inanimate surfaces with those on native-biological interfaces, specifically the ACE2 receptor. Notably, we identify a twofold increase in structural deformations for the proteins receptor binding motif onto the inanimate surfaces, indicative of enhanced shock-absorbing mechanisms. Furthermore, the distribution of amino acids (landing-footprints) on the inanimate surface reveals a distinct regional asymmetry relative to the biological interface. In spite of the H-bonds formed at the hydrophilic substrate, the simulations consistently show a higher number of contacts and interfacial area with the hydrophobic surface, with the WT RBD adsorbed more strongly to than the delta or omicron RBDs. In contrast, the adsorption of delta and omicron to hydrophilic surfaces was characterized by a distinctive hopping-pattern. The novel shock-absorbing mechanisms identified in the virus adsorption on inanimate surfaces could lead current experimental efforts in the design of virucidal surfaces.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576226">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576226" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576226">
        <p class="paperTitle">Correlating fluorescence microscopy, optical and magnetic tweezers to study single chiral biopolymers, tested on DNA plectoneme formation dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576226" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576226" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shepherd, J.; Guilbaud, S.; Zhou, Z.; Howard, J.; Burman, M.; Schaefer, C.; Kerrigan, A.; Steele-King, C.; Noy, A.; Leake, M. C.</p>
        <p class="info">Score: 6.4, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576226' target='https://doi.org/10.1101/2024.01.18.576226'> 10.1101/2024.01.18.576226</a></p>
        <p class="abstract">Biopolymer topology is critical for determining interactions inside cell environments, exemplified by DNA where its response to mechanical perturbation is as important as biochemical properties to its cellular roles. The dynamic structures of chiral biopolymers exhibit complex dependence with extension and torsion, however the physical mechanisms underpinning the emergence of structural motifs upon physiological twisting and stretching are poorly understood due to technological limitations in correlating force, torque and spatial localization information. We present COMBI-Tweez (Combined Optical and Magnetic BIomolecule TWEEZers), a transformative tool that overcomes these challenges by integrating optical trapping, time-resolved electromagnetic tweezers, and fluorescence microscopy, demonstrated on single DNA molecules, that can controllably form and visualise higher order structural motifs including plectonemes. This technology combined with cutting-edge MD simulations provides quantitative insight into complex dynamic structures relevant to DNA cellular processes and can be adapted to study a range of filamentous biopolymers.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
