<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Imaging proteins sensitive to direct fusions using transient peptide-peptide interactions
Authors: Gidden, Z.; Oi, C.; Johnston, E.; Bhaskar, H.; Rosser, S.; Mochrie, S. G. J.; Horrocks, M. H.; Regan, L.
Score: 30.2, Published: 2023-07-02 DOI: 10.1101/2023.07.01.547312
Fluorescence microscopy enables specific visualization of proteins in living cells and has played an important role in our understanding of protein subcellular location and function. Some proteins, however, show altered localization and/or function when labeled using direct fusions to fluorescent proteins, making them difficult to study in live cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Imaging proteins sensitive to direct fusions using transient peptide-peptide interactions
Authors: Gidden, Z.; Oi, C.; Johnston, E.; Bhaskar, H.; Rosser, S.; Mochrie, S. G. J.; Horrocks, M. H.; Regan, L.
Score: 30.2, Published: 2023-07-02 DOI: 10.1101/2023.07.01.547312
Fluorescence microscopy enables specific visualization of proteins in living cells and has played an important role in our understanding of protein subcellular location and function. Some proteins, however, show altered localization and/or function when labeled using direct fusions to fluorescent proteins, making them difficult to study in live cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-09T10:40:17+00:00" />
<meta property="article:modified_time" content="2023-07-09T10:40:17+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Imaging proteins sensitive to direct fusions using transient peptide-peptide interactions
Authors: Gidden, Z.; Oi, C.; Johnston, E.; Bhaskar, H.; Rosser, S.; Mochrie, S. G. J.; Horrocks, M. H.; Regan, L.
Score: 30.2, Published: 2023-07-02 DOI: 10.1101/2023.07.01.547312
Fluorescence microscopy enables specific visualization of proteins in living cells and has played an important role in our understanding of protein subcellular location and function. Some proteins, however, show altered localization and/or function when labeled using direct fusions to fluorescent proteins, making them difficult to study in live cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Imaging proteins sensitive to direct fusions using transient peptide-peptide interactions\nAuthors: Gidden, Z.; Oi, C.; Johnston, E.; Bhaskar, H.; Rosser, S.; Mochrie, S. G. J.; Horrocks, M. H.; Regan, L.\nScore: 30.2, Published: 2023-07-02 DOI: 10.1101/2023.07.01.547312\nFluorescence microscopy enables specific visualization of proteins in living cells and has played an important role in our understanding of protein subcellular location and function. Some proteins, however, show altered localization and/or function when labeled using direct fusions to fluorescent proteins, making them difficult to study in live cells.",
  "keywords": [
    
  ],
  "articleBody": " Imaging proteins sensitive to direct fusions using transient peptide-peptide interactions\nAuthors: Gidden, Z.; Oi, C.; Johnston, E.; Bhaskar, H.; Rosser, S.; Mochrie, S. G. J.; Horrocks, M. H.; Regan, L.\nScore: 30.2, Published: 2023-07-02 DOI: 10.1101/2023.07.01.547312\nFluorescence microscopy enables specific visualization of proteins in living cells and has played an important role in our understanding of protein subcellular location and function. Some proteins, however, show altered localization and/or function when labeled using direct fusions to fluorescent proteins, making them difficult to study in live cells. Additionally, the resolution of fluorescence microscopy is limited to [~]200 nm, which is two orders of magnitude larger than the size of most proteins. To circumvent these challenges, we previously developed LIVE-PAINT, a live-cell super-resolution approach that takes advantage of short interacting peptides to transiently bind a fluorescent protein to the protein-of-interest. Here, we successfully use LIVE-PAINT to image yeast membrane proteins that do not tolerate the direct fusion of a fluorescent protein by using peptide tags as short as 5-residues. We also demonstrate that it is possible to resolve multiple proteins at the nanoscale concurrently using orthogonal peptide interaction pairs. FOR TABLE OF CONTENTS ONLY O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=116 SRC=\"FIGDIR/small/547312v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (29K): org.highwire.dtl.DTLVardef@1298e93org.highwire.dtl.DTLVardef@14f86baorg.highwire.dtl.DTLVardef@16566dorg.highwire.dtl.DTLVardef@843628_HPS_FORMAT_FIGEXP M_FIG C_FIG\nRapid assessment of changes in phage bioactivity using dynamic light scattering\nAuthors: Dharmaraj, T.; Kratochvil, M. J.; Pourtois, J. D.; Chen, Q.; Hajfathalian, M.; Hargil, A.; Lin, Y.-H.; Evans, Z. R.; Oromi-Bosch, A.; Berry, J. D.; McBride, R.; Haddock, N. L.; Holman, D. R.; Van Belleghem, J. D.; Chang, T. H.; Barr, J. J.; Lavigne, R.; Heilshorn, S. C.; Blankenberg, F. G.; Bollyky, P. L.\nScore: 26.2, Published: 2023-07-02 DOI: 10.1101/2023.07.02.547396\nExtensive efforts are underway to develop bacteriophages as therapies against antibiotic-resistant bacteria. However, these efforts are confounded by the instability of phage preparations and a lack of suitable tools to assess active phage concentrations over time. Here, we use Dynamic Light Scattering (DLS) to measure changes in phage physical state in response to environmental factors and time, finding that phages tend to decay and form aggregates and that the degree of aggregation can be used to predict phage bioactivity. We then use DLS to optimize phage storage conditions for phages from human clinical trials, predict bioactivity in 50-year-old archival stocks, and evaluate phage samples for use in a phage therapy/wound infection model. We also provide a web-application (Phage-ELF) to facilitate DLS studies of phages. We conclude that DLS provides a rapid, convenient, and non-destructive tool for quality control of phage preparations in academic and commercial settings. Significance StatementPhages are promising for use in treating antibiotic-resistant infections, but their decay over time in refrigerated storage and higher temperatures has been a difficult barrier to overcome. This is in part because there are no suitable methods to monitor phage activity over time, especially in clinical settings. Here, we show that Dynamic Light Scattering (DLS) can be used to measure the physical state of phage preparations, which provides accurate and precise information on their lytic function - the key parameter underlying clinical efficacy. This study reveals a \"structure-function\" relationship for lytic phages and establishes DLS as a method to optimize the storage, handling, and clinical use of phages.\nEvolution of hierarchy and irreversibility in theoretical cell differentiation model\nAuthors: Nakamura, Y. T.; Himeoka, Y.; Saito, N.; Furusawa, C.\nScore: 22.1, Published: 2023-07-07 DOI: 10.1101/2023.07.04.547517\nThe process of cell differentiation in multicellular organisms is characterized by hierarchy and irreversibility in many cases. However, the conditions and selection pressures that give rise to these characteristics remain poorly understood. By using a mathematical model, here we show that the network of differentiation potency (differentiation diagram) becomes necessarily hierarchical and irreversible by increasing the number of terminally differentiated states under certain conditions. The mechanisms generating these characteristics are clarified using geometry in the cell state space. The results demonstrate that the appearance of these characteristics can be driven without assuming the adaptive significance. The study also provides a new perspective on the structure of gene regulatory networks that produce hierarchical and irreversible differentiation diagrams. These results indicate some constraints on cell differentiation, which are expected to provide a starting point for theoretical discussion of the implicit limits and directions of evolution in multicellular organisms.\nBaited reconstruction with 2D template matching for high-resolution structure determination in vitro and in vivo without template bias\nAuthors: Lucas, B. A.; Himes, B. A.; Grigorieff, N.\nScore: 19.3, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547552\nCryogenic electron microscopy (cryo-EM) has revolutionized structural biology, rapidly increasing the number of available molecular structures. Because of this, as well as advances in structure prediction, the focus of structural biology has begun to shift to studying macromolecular structures in their native cellular environment. A dominant feature of cryo-EM images is shot noise, making the identification of small particles of interest difficult. This is further compounded by structural noise if these particles are imaged against a background of other molecules, such as inside a cell. 2D template matching (2DTM) can be used to localize complexes with high precision, even in the presence of cellular background. Once localized, these particles may be averaged together in 3D reconstructions; however, regions included in the template may suffer from template bias, leading to inflated resolution estimates and making the interpretation of high-resolution features unreliable. We evaluate conditions that minimize template bias and show that molecular features not present in the template can be reconstructed at high resolution from targets found by 2DTM, extending prior work at low-resolution. Moreover, we present a quantitative metric for template bias to aid the interpretation of 3D reconstructions calculated using particles localized by 2DTM.\nThe killing of human gut commensal E. coli ED1a by tetracycline is associated with severe ribosome dysfunction\nAuthors: Khusainov, I.; Romanov, N.; Goemans, C.; Turonova, B.; Zimmerli, C. E.; Welsch, S.; Langer, J.; Typas, A.; Beck, M.\nScore: 18.8, Published: 2023-07-06 DOI: 10.1101/2023.07.06.546847\nRibosomes translate the genetic code into proteins. Recent technical advances have facilitated in situ structural analyses of ribosome functional states inside eukaryotic cells and the minimal bacterium Mycoplasma. However, such analyses of Gram-negative bacteria are lacking, despite their ribosomes being major antimicrobial drug targets. Here we compare two E. coli strains, a lab E. coli K-12 and human gut isolate E. coli ED1a, for which tetracycline exhibits bacteriostatic and bactericidal action, respectively. The in situ ribosome structures upon tetracycline treatment show a virtually identical drug binding-site in both strains, yet the distribution of ribosomal complexes clearly differs. While K-12 retains ribosomes in a translation competent state, tRNAs are lost in the vast majority of ED1a ribosomes. A differential response is also reflected in proteome-wide abundance and thermal stability assessment. Our study underlines the need to include molecular analyses and to consider gut bacteria when addressing antibiotic mode of action.\nFunctional Protein Dynamics in a Crystal\nAuthors: Klyshko, E.; Kim, J. S.-H.; McGough, L.; Valeeva, V.; Lee, E.; Ranganathan, R.; Rauscher, S.\nScore: 16.2, Published: 2023-07-07 DOI: 10.1101/2023.07.06.548023\nProteins are molecular machines and to understand how they work, we need to understand how they move. New \"pump-probe\" time-resolved X-ray diffraction methods open up ways to initiate and observe protein motions with atomistic detail in crystals on biologically relevant timescales. However, practical limitations of these experiments demands parallel development of effective molecular dynamics approaches to accelerate progress and extract meaning. Here, we establish robust and accurate methods for simulating dynamics in protein crystals, a nontrivial process requiring careful attention to equilibration, environmental composition, and choice of force fields. With more than seven milliseconds of sampling of a single chain, we identify critical factors controlling agreement between simulation and experiments and show that simulated motions recapitulate ligand-induced conformational changes. This work enables a virtuous cycle between simulation and experiments for visualizing and understanding the basic functional motions of proteins.\nLoop-extruders alter bacterial chromosome topology to direct entropic forces for segregation\nAuthors: Harju, J.; van Teeseling, M. C. F.; Broedersz, C. P.\nScore: 15.0, Published: 2023-07-01 DOI: 10.1101/2023.06.30.547230\nEntropic forces have been argued to allow bacteria to segregate their chromosomes during replication. In many bacterial species, however, specifically evolved mechanisms, such as loop-extruding SMC complexes and the ParABS origin segregation system, are required for chromosome segregation, suggesting that entropic forces alone may be insufficient. The interplay between and the relative contributions of these segregation mechanisms remain unclear. Here, we develop a biophysical model showing that purely entropic forces actually inhibit bacterial chromosome segregation until late stages of the replication cycle. By contrast, we find that loop-extruders loaded at the origins of replication, as found in many bacterial species, alter the effective topology of the chromosome, thereby redirecting and enhancing entropic forces to enable accurate chromosome segregation concurrent with replication. In addition, our model indicates that locally acting origin separation mechanisms, such as the ParABS system, cannot redirect segregation-inhibiting entropic forces. We confirm our predictions with 3D simulations of a replicating polymer: purely entropic forces do not allow for concurrent replication and segregation; entropic forces steered by loop-extruders loaded at the origin lead to robust, global chromosome segregation; and origin separation alone is effective at early replication stages, but does not fully segregate terminal regions. Together, our results illustrate how entropic forces and loop-extrusion play synergistic roles during bacterial chromosome segregation, whereas locally acting mechanisms serve a complementary function.\nAutomated Purification of DNA Origami with SPRI Beads\nAuthors: Chau, C.; Mohanan, G.; Macaulay, I. C.; Actis, P.; Walti, C.\nScore: 13.8, Published: 2023-07-05 DOI: 10.1101/2023.07.05.544573\nDNA origami synthesis is a well-established technique and has been employed in various applications. The synthesised origami must be purified to eliminate the excess materials such as DNA oligos and other molecules. While several purification techniques are routinely used, they all have limitations, and none can be automated to simultaneously handle large numbers and quantities of samples. Here we introduce the use of solid-phase immobilisation (SPRI) beads as an easy-to-adopt, scalable, high-throughput and automation-compatible method to purify DNA origami. Not only can this method remove excess oligos and biomolecules with comparable yield to existing methods while maintaining high structural integrity of the origami, but it also allows an automated workflow to simultaneously purify large numbers of samples within a limited time. We envision that the SPRI beads purification approach will improve the scalability of DNA nanostructures synthesis both for research and commercial applications.\nTopology-dependent DNA binding\nAuthors: Kolbeck, P. J.; Tisma, M.; Analikwu, B. T.; Vanderlinden, W.; Dekker, C.; Lipfert, J.\nScore: 13.2, Published: 2023-07-01 DOI: 10.1101/2023.06.30.547266\nDNA stores our genetic information and is ubiquitous in biological and biotechnological applications, where it interacts with binding partners ranging from small molecules to large macromolecular complexes. Binding is modulated by mechanical strains in the molecule and, in turn, can change the local DNA structure. Frequently, DNA occurs in closed topological forms where topology and supercoiling add a global constraint to the interplay of binding-induced deformations and strain-modulated binding. Here, we present a quantitative model of how the global constraints introduced by DNA topology modulate binding and create a complex interplay between topology and affinity. We focus on fluorescent intercalators, which unwind DNA and enable direct quantification via fluorescence detection. Using bulk measurements, we show that DNA supercoiling can increase or decrease intercalation relative to an open topology depending on ligand concentration and the initial topology. Our model quantitatively accounts for observations obtained using psoralen for UV-induced DNA crosslinking, which is frequently used to quantify supercoiling in vivo. Finally, we observe topology-dependent binding in a single-molecule assay, which provides direct access to binding kinetics and DNA supercoil dynamics. Our results have broad implications for the detection and quantification of DNA and for the modulation of DNA binding in cellular contexts.\nDiffusiophoresis promotes phase separation and transport of biomolecular condensates\nAuthors: Doan, V. S.; Alshareedah, I.; Singh, A.; Banerjee, P. R.; Shin, S.\nScore: 10.9, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547532\nThe internal microenvironment of a living cell is heterogeneous and comprises a multitude of organelles with distinct biochemistry. Amongst them are biomolecular condensates, which are membrane-less, phase-separated compartments enriched in system-specific proteins and nucleic acids. The heterogeneity of the cell engenders the presence of multiple spatiotemporal gradients in chemistry, charge, concentration, temperature, and pressure. Such thermodynamic gradients can lead to non-equilibrium driving forces for the formation and transport of biomolecular condensates. Here, we report how ion gradients impact the transport processes of biomolecular condensates on the mesoscale and biomolecules on the microscale. Utilizing a microfluidic platform, we demonstrate that the presence of ion concentration gradients can accelerate the transport of biomolecules, including nucleic acids and proteins, via diffusiophoresis. This hydrodynamic transport process allows localized enrichment of biomolecules, thereby promoting the location-specific formation of biomolecular condensates via phase separation. The ion gradients further impart active motility of condensates, allowing them to exhibit enhanced diffusion along the gradient. Coupled with reentrant phase behavior, the gradient-induced active motility leads to a dynamical redistribution of condensates that ultimately extends their lifetime. Together, our results demonstrate diffusiophoresis as a non-equilibrium thermodynamic force that governs the formation and active transport of biomolecular condensates.\n",
  "wordCount" : "2156",
  "inLanguage": "en",
  "datePublished": "2023-07-09T10:40:17Z",
  "dateModified": "2023-07-09T10:40:17Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on July 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.01.547312">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.01.547312" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.01.547312">
        <p class="paperTitle">Imaging proteins sensitive to direct fusions using transient peptide-peptide interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.01.547312" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.01.547312" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gidden, Z.; Oi, C.; Johnston, E.; Bhaskar, H.; Rosser, S.; Mochrie, S. G. J.; Horrocks, M. H.; Regan, L.</p>
        <p class="info">Score: 30.2, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.01.547312' target='https://doi.org/10.1101/2023.07.01.547312'> 10.1101/2023.07.01.547312</a></p>
        <p class="abstract">Fluorescence microscopy enables specific visualization of proteins in living cells and has played an important role in our understanding of protein subcellular location and function. Some proteins, however, show altered localization and/or function when labeled using direct fusions to fluorescent proteins, making them difficult to study in live cells. Additionally, the resolution of fluorescence microscopy is limited to [~]200 nm, which is two orders of magnitude larger than the size of most proteins. To circumvent these challenges, we previously developed LIVE-PAINT, a live-cell super-resolution approach that takes advantage of short interacting peptides to transiently bind a fluorescent protein to the protein-of-interest. Here, we successfully use LIVE-PAINT to image yeast membrane proteins that do not tolerate the direct fusion of a fluorescent protein by using peptide tags as short as 5-residues. We also demonstrate that it is possible to resolve multiple proteins at the nanoscale concurrently using orthogonal peptide interaction pairs.

FOR TABLE OF CONTENTS ONLY

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=116 SRC=&#34;FIGDIR/small/547312v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (29K):
org.highwire.dtl.DTLVardef@1298e93org.highwire.dtl.DTLVardef@14f86baorg.highwire.dtl.DTLVardef@16566dorg.highwire.dtl.DTLVardef@843628_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.02.547396">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.02.547396" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.02.547396">
        <p class="paperTitle">Rapid assessment of changes in phage bioactivity using dynamic light scattering</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.02.547396" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.02.547396" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dharmaraj, T.; Kratochvil, M. J.; Pourtois, J. D.; Chen, Q.; Hajfathalian, M.; Hargil, A.; Lin, Y.-H.; Evans, Z. R.; Oromi-Bosch, A.; Berry, J. D.; McBride, R.; Haddock, N. L.; Holman, D. R.; Van Belleghem, J. D.; Chang, T. H.; Barr, J. J.; Lavigne, R.; Heilshorn, S. C.; Blankenberg, F. G.; Bollyky, P. L.</p>
        <p class="info">Score: 26.2, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.02.547396' target='https://doi.org/10.1101/2023.07.02.547396'> 10.1101/2023.07.02.547396</a></p>
        <p class="abstract">Extensive efforts are underway to develop bacteriophages as therapies against antibiotic-resistant bacteria. However, these efforts are confounded by the instability of phage preparations and a lack of suitable tools to assess active phage concentrations over time. Here, we use Dynamic Light Scattering (DLS) to measure changes in phage physical state in response to environmental factors and time, finding that phages tend to decay and form aggregates and that the degree of aggregation can be used to predict phage bioactivity. We then use DLS to optimize phage storage conditions for phages from human clinical trials, predict bioactivity in 50-year-old archival stocks, and evaluate phage samples for use in a phage therapy/wound infection model. We also provide a web-application (Phage-ELF) to facilitate DLS studies of phages. We conclude that DLS provides a rapid, convenient, and non-destructive tool for quality control of phage preparations in academic and commercial settings.

Significance StatementPhages are promising for use in treating antibiotic-resistant infections, but their decay over time in refrigerated storage and higher temperatures has been a difficult barrier to overcome. This is in part because there are no suitable methods to monitor phage activity over time, especially in clinical settings. Here, we show that Dynamic Light Scattering (DLS) can be used to measure the physical state of phage preparations, which provides accurate and precise information on their lytic function - the key parameter underlying clinical efficacy. This study reveals a &#34;structure-function&#34; relationship for lytic phages and establishes DLS as a method to optimize the storage, handling, and clinical use of phages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547517">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547517" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547517">
        <p class="paperTitle">Evolution of hierarchy and irreversibility in theoretical cell differentiation model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547517" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547517" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nakamura, Y. T.; Himeoka, Y.; Saito, N.; Furusawa, C.</p>
        <p class="info">Score: 22.1, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547517' target='https://doi.org/10.1101/2023.07.04.547517'> 10.1101/2023.07.04.547517</a></p>
        <p class="abstract">The process of cell differentiation in multicellular organisms is characterized by hierarchy and irreversibility in many cases. However, the conditions and selection pressures that give rise to these characteristics remain poorly understood. By using a mathematical model, here we show that the network of differentiation potency (differentiation diagram) becomes necessarily hierarchical and irreversible by increasing the number of terminally differentiated states under certain conditions. The mechanisms generating these characteristics are clarified using geometry in the cell state space. The results demonstrate that the appearance of these characteristics can be driven without assuming the adaptive significance. The study also provides a new perspective on the structure of gene regulatory networks that produce hierarchical and irreversible differentiation diagrams. These results indicate some constraints on cell differentiation, which are expected to provide a starting point for theoretical discussion of the implicit limits and directions of evolution in multicellular organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547552">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547552" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547552">
        <p class="paperTitle">Baited reconstruction with 2D template matching for high-resolution structure determination in vitro and in vivo without template bias</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547552" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547552" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lucas, B. A.; Himes, B. A.; Grigorieff, N.</p>
        <p class="info">Score: 19.3, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547552' target='https://doi.org/10.1101/2023.07.03.547552'> 10.1101/2023.07.03.547552</a></p>
        <p class="abstract">Cryogenic electron microscopy (cryo-EM) has revolutionized structural biology, rapidly increasing the number of available molecular structures. Because of this, as well as advances in structure prediction, the focus of structural biology has begun to shift to studying macromolecular structures in their native cellular environment. A dominant feature of cryo-EM images is shot noise, making the identification of small particles of interest difficult. This is further compounded by structural noise if these particles are imaged against a background of other molecules, such as inside a cell. 2D template matching (2DTM) can be used to localize complexes with high precision, even in the presence of cellular background. Once localized, these particles may be averaged together in 3D reconstructions; however, regions included in the template may suffer from template bias, leading to inflated resolution estimates and making the interpretation of high-resolution features unreliable. We evaluate conditions that minimize template bias and show that molecular features not present in the template can be reconstructed at high resolution from targets found by 2DTM, extending prior work at low-resolution. Moreover, we present a quantitative metric for template bias to aid the interpretation of 3D reconstructions calculated using particles localized by 2DTM.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.546847">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.546847" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.546847">
        <p class="paperTitle">The killing of human gut commensal E. coli ED1a by tetracycline is associated with severe ribosome dysfunction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.546847" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.546847" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khusainov, I.; Romanov, N.; Goemans, C.; Turonova, B.; Zimmerli, C. E.; Welsch, S.; Langer, J.; Typas, A.; Beck, M.</p>
        <p class="info">Score: 18.8, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.546847' target='https://doi.org/10.1101/2023.07.06.546847'> 10.1101/2023.07.06.546847</a></p>
        <p class="abstract">Ribosomes translate the genetic code into proteins. Recent technical advances have facilitated in situ structural analyses of ribosome functional states inside eukaryotic cells and the minimal bacterium Mycoplasma. However, such analyses of Gram-negative bacteria are lacking, despite their ribosomes being major antimicrobial drug targets. Here we compare two E. coli strains, a lab E. coli K-12 and human gut isolate E. coli ED1a, for which tetracycline exhibits bacteriostatic and bactericidal action, respectively. The in situ ribosome structures upon tetracycline treatment show a virtually identical drug binding-site in both strains, yet the distribution of ribosomal complexes clearly differs. While K-12 retains ribosomes in a translation competent state, tRNAs are lost in the vast majority of ED1a ribosomes. A differential response is also reflected in proteome-wide abundance and thermal stability assessment. Our study underlines the need to include molecular analyses and to consider gut bacteria when addressing antibiotic mode of action.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.548023">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.548023" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.548023">
        <p class="paperTitle">Functional Protein Dynamics in a Crystal</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.548023" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.548023" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Klyshko, E.; Kim, J. S.-H.; McGough, L.; Valeeva, V.; Lee, E.; Ranganathan, R.; Rauscher, S.</p>
        <p class="info">Score: 16.2, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.548023' target='https://doi.org/10.1101/2023.07.06.548023'> 10.1101/2023.07.06.548023</a></p>
        <p class="abstract">Proteins are molecular machines and to understand how they work, we need to understand how they move. New &#34;pump-probe&#34; time-resolved X-ray diffraction methods open up ways to initiate and observe protein motions with atomistic detail in crystals on biologically relevant timescales. However, practical limitations of these experiments demands parallel development of effective molecular dynamics approaches to accelerate progress and extract meaning. Here, we establish robust and accurate methods for simulating dynamics in protein crystals, a nontrivial process requiring careful attention to equilibration, environmental composition, and choice of force fields. With more than seven milliseconds of sampling of a single chain, we identify critical factors controlling agreement between simulation and experiments and show that simulated motions recapitulate ligand-induced conformational changes. This work enables a virtuous cycle between simulation and experiments for visualizing and understanding the basic functional motions of proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547230">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547230" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547230">
        <p class="paperTitle">Loop-extruders alter bacterial chromosome topology to direct entropic forces for segregation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547230" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547230" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harju, J.; van Teeseling, M. C. F.; Broedersz, C. P.</p>
        <p class="info">Score: 15.0, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547230' target='https://doi.org/10.1101/2023.06.30.547230'> 10.1101/2023.06.30.547230</a></p>
        <p class="abstract">Entropic forces have been argued to allow bacteria to segregate their chromosomes during replication. In many bacterial species, however, specifically evolved mechanisms, such as loop-extruding SMC complexes and the ParABS origin segregation system, are required for chromosome segregation, suggesting that entropic forces alone may be insufficient. The interplay between and the relative contributions of these segregation mechanisms remain unclear. Here, we develop a biophysical model showing that purely entropic forces actually inhibit bacterial chromosome segregation until late stages of the replication cycle. By contrast, we find that loop-extruders loaded at the origins of replication, as found in many bacterial species, alter the effective topology of the chromosome, thereby redirecting and enhancing entropic forces to enable accurate chromosome segregation concurrent with replication. In addition, our model indicates that locally acting origin separation mechanisms, such as the ParABS system, cannot redirect segregation-inhibiting entropic forces. We confirm our predictions with 3D simulations of a replicating polymer: purely entropic forces do not allow for concurrent replication and segregation; entropic forces steered by loop-extruders loaded at the origin lead to robust, global chromosome segregation; and origin separation alone is effective at early replication stages, but does not fully segregate terminal regions. Together, our results illustrate how entropic forces and loop-extrusion play synergistic roles during bacterial chromosome segregation, whereas locally acting mechanisms serve a complementary function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.544573">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.544573" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.544573">
        <p class="paperTitle">Automated Purification of DNA Origami with SPRI Beads</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.544573" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.544573" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chau, C.; Mohanan, G.; Macaulay, I. C.; Actis, P.; Walti, C.</p>
        <p class="info">Score: 13.8, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.544573' target='https://doi.org/10.1101/2023.07.05.544573'> 10.1101/2023.07.05.544573</a></p>
        <p class="abstract">DNA origami synthesis is a well-established technique and has been employed in various applications. The synthesised origami must be purified to eliminate the excess materials such as DNA oligos and other molecules. While several purification techniques are routinely used, they all have limitations, and none can be automated to simultaneously handle large numbers and quantities of samples. Here we introduce the use of solid-phase immobilisation (SPRI) beads as an easy-to-adopt, scalable, high-throughput and automation-compatible method to purify DNA origami. Not only can this method remove excess oligos and biomolecules with comparable yield to existing methods while maintaining high structural integrity of the origami, but it also allows an automated workflow to simultaneously purify large numbers of samples within a limited time. We envision that the SPRI beads purification approach will improve the scalability of DNA nanostructures synthesis both for research and commercial applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547266">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547266" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547266">
        <p class="paperTitle">Topology-dependent DNA binding</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547266" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547266" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kolbeck, P. J.; Tisma, M.; Analikwu, B. T.; Vanderlinden, W.; Dekker, C.; Lipfert, J.</p>
        <p class="info">Score: 13.2, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547266' target='https://doi.org/10.1101/2023.06.30.547266'> 10.1101/2023.06.30.547266</a></p>
        <p class="abstract">DNA stores our genetic information and is ubiquitous in biological and biotechnological applications, where it interacts with binding partners ranging from small molecules to large macromolecular complexes. Binding is modulated by mechanical strains in the molecule and, in turn, can change the local DNA structure. Frequently, DNA occurs in closed topological forms where topology and supercoiling add a global constraint to the interplay of binding-induced deformations and strain-modulated binding. Here, we present a quantitative model of how the global constraints introduced by DNA topology modulate binding and create a complex interplay between topology and affinity. We focus on fluorescent intercalators, which unwind DNA and enable direct quantification via fluorescence detection. Using bulk measurements, we show that DNA supercoiling can increase or decrease intercalation relative to an open topology depending on ligand concentration and the initial topology. Our model quantitatively accounts for observations obtained using psoralen for UV-induced DNA crosslinking, which is frequently used to quantify supercoiling in vivo. Finally, we observe topology-dependent binding in a single-molecule assay, which provides direct access to binding kinetics and DNA supercoil dynamics. Our results have broad implications for the detection and quantification of DNA and for the modulation of DNA binding in cellular contexts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547532">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547532" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547532">
        <p class="paperTitle">Diffusiophoresis promotes phase separation and transport of biomolecular condensates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547532" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547532" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Doan, V. S.; Alshareedah, I.; Singh, A.; Banerjee, P. R.; Shin, S.</p>
        <p class="info">Score: 10.9, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547532' target='https://doi.org/10.1101/2023.07.03.547532'> 10.1101/2023.07.03.547532</a></p>
        <p class="abstract">The internal microenvironment of a living cell is heterogeneous and comprises a multitude of organelles with distinct biochemistry. Amongst them are biomolecular condensates, which are membrane-less, phase-separated compartments enriched in system-specific proteins and nucleic acids. The heterogeneity of the cell engenders the presence of multiple spatiotemporal gradients in chemistry, charge, concentration, temperature, and pressure. Such thermodynamic gradients can lead to non-equilibrium driving forces for the formation and transport of biomolecular condensates. Here, we report how ion gradients impact the transport processes of biomolecular condensates on the mesoscale and biomolecules on the microscale. Utilizing a microfluidic platform, we demonstrate that the presence of ion concentration gradients can accelerate the transport of biomolecules, including nucleic acids and proteins, via diffusiophoresis. This hydrodynamic transport process allows localized enrichment of biomolecules, thereby promoting the location-specific formation of biomolecular condensates via phase separation. The ion gradients further impart active motility of condensates, allowing them to exhibit enhanced diffusion along the gradient. Coupled with reentrant phase behavior, the gradient-induced active motility leads to a dynamical redistribution of condensates that ultimately extends their lifetime. Together, our results demonstrate diffusiophoresis as a non-equilibrium thermodynamic force that governs the formation and active transport of biomolecular condensates.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
