<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="High-throughput thermodynamic and kinetic measurements of transcription factor/DNA mutations reveal how conformational heterogeneity can shape motif selectivity
Authors: Hastings, R.; Aditham, A.; DelRosso, N.; Suzuki, P.; Fordyce, P. M.
Score: 21.3, Published: 2023-11-19 DOI: 10.1101/2023.11.13.566946
Transcription factors (TFs) bind DNA sequences with a range of affinities, yet the mechanisms determining energetic differences between high- and low-affinity sequences ( selectivity) remain poorly understood. Here, we investigated two basic helix-loop-helix TFs, MAX (H.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="High-throughput thermodynamic and kinetic measurements of transcription factor/DNA mutations reveal how conformational heterogeneity can shape motif selectivity
Authors: Hastings, R.; Aditham, A.; DelRosso, N.; Suzuki, P.; Fordyce, P. M.
Score: 21.3, Published: 2023-11-19 DOI: 10.1101/2023.11.13.566946
Transcription factors (TFs) bind DNA sequences with a range of affinities, yet the mechanisms determining energetic differences between high- and low-affinity sequences ( selectivity) remain poorly understood. Here, we investigated two basic helix-loop-helix TFs, MAX (H." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-03T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-12-03T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="High-throughput thermodynamic and kinetic measurements of transcription factor/DNA mutations reveal how conformational heterogeneity can shape motif selectivity
Authors: Hastings, R.; Aditham, A.; DelRosso, N.; Suzuki, P.; Fordyce, P. M.
Score: 21.3, Published: 2023-11-19 DOI: 10.1101/2023.11.13.566946
Transcription factors (TFs) bind DNA sequences with a range of affinities, yet the mechanisms determining energetic differences between high- and low-affinity sequences ( selectivity) remain poorly understood. Here, we investigated two basic helix-loop-helix TFs, MAX (H."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "High-throughput thermodynamic and kinetic measurements of transcription factor/DNA mutations reveal how conformational heterogeneity can shape motif selectivity\nAuthors: Hastings, R.; Aditham, A.; DelRosso, N.; Suzuki, P.; Fordyce, P. M.\nScore: 21.3, Published: 2023-11-19 DOI: 10.1101/2023.11.13.566946\nTranscription factors (TFs) bind DNA sequences with a range of affinities, yet the mechanisms determining energetic differences between high- and low-affinity sequences ( selectivity) remain poorly understood. Here, we investigated two basic helix-loop-helix TFs, MAX (H.",
  "keywords": [
    
  ],
  "articleBody": " High-throughput thermodynamic and kinetic measurements of transcription factor/DNA mutations reveal how conformational heterogeneity can shape motif selectivity\nAuthors: Hastings, R.; Aditham, A.; DelRosso, N.; Suzuki, P.; Fordyce, P. M.\nScore: 21.3, Published: 2023-11-19 DOI: 10.1101/2023.11.13.566946\nTranscription factors (TFs) bind DNA sequences with a range of affinities, yet the mechanisms determining energetic differences between high- and low-affinity sequences ( selectivity) remain poorly understood. Here, we investigated two basic helix-loop-helix TFs, MAX (H. sapiens) and Pho4 (S. cerevisiae), that bind the same high-affinity sequence with highly similar nucleotide-contacting residues and bound structures but are differentially selective for non-cognate sequences. By measuring \u003e1700 Kds and \u003e500 rate constants for Pho4 and MAX mutant libraries binding multiple DNA sequences and comparing these measurements with thermodynamic and kinetic models, we identify the biophysical mechanisms by which changes to TF sequence alter both bound and unbound conformational ensembles to shape specificity landscapes. These results highlight the importance of conformational heterogeneity in determining sequence specificity and selectivity and can guide future efforts to engineer nucleic acid-binding proteins with enhanced selectivity.\nAllosteric Competition and Inhibition in AMPA Receptors\nAuthors: Hale, W. D.; Romero, A. M.; Gonzalez, C. U.; Jayaraman, V.; Lau, A. Y.; Huganir, R. L.; Twomey, E. C.\nScore: 17.8, Published: 2023-11-28 DOI: 10.1101/2023.11.28.569057\nExcitatory neurotransmission is principally mediated by AMPA-subtype ionotropic glutamate receptors (AMPARs). Dysregulation of AMPARs is the cause of many neurological disorders and how therapeutic candidates such as negative allosteric modulators inhibit AMPARs is unclear. Here, we show that non-competitive inhibition desensitizes AMPARs to activation and prevents positive allosteric modulation. We dissected the noncompetitive inhibition mechanism of action by capturing AMPARs bound to glutamate and the prototypical negative allosteric modulator, GYKI-52466, with cryo-electron microscopy. Noncompetitive inhibition by GYKI-52466, which binds in the transmembrane collar region surrounding the ion channel, negatively modulates AMPARs by decoupling glutamate binding in the ligand binding domain from the ion channel. Furthermore, during allosteric competition between negative and positive modulators, negative allosteric modulation by GKYI-52466 outcompetes positive allosteric modulators to control AMPAR function. Our data provide a new framework for understanding allostery of AMPARs and foundations for rational design of therapeutics targeting AMPARs in neurological diseases.\nStructure of Transmembrane AMPA Receptor Regulatory Protein Subunit Î³2\nAuthors: Hale, W. D.; Romero, A. M.; Huganir, R. L.; Twomey, E. C.\nScore: 15.8, Published: 2023-11-28 DOI: 10.1101/2023.11.28.569079\nTransmembrane AMPA receptor regulatory proteins (TARPs) are claudin-like proteins that tightly regulate AMPA receptors (AMPARs) and are fundamental for excitatory neurotransmission. We used cryo-electron microscopy (cryo-EM) to reconstruct the 36 kDa TARP subunit {gamma}2 to 2.3 [A] and reveal the structural diversity of TARPs. Our data reveals critical motifs that distinguish TARPs from claudins and define how sequence variations within TARPs differentiate subfamilies and their regulation of AMPARs.\nUnbiased MD simulations characterize lipid binding pockets in lipid transfer proteins\nAuthors: Vanni, S.; Srinivasan, S.; Alvarez Lorenzo, D.\nScore: 10.8, Published: 2023-11-24 DOI: 10.1101/2023.11.24.568550\nThe molecular characterization of lipid binding to lipid transfer proteins (LTPs) is fundamental to the understanding of several aspects of their mechanistic mode of action. However, obtaining lipid-bound structures of LTPs is not straightforward owing to caveats in current experimental structural biology approaches. As a result, several structures of LTPs, and most notably almost all of those that have been proposed to act as bridges between membrane organelles, do not provide the precise location of their endogenous lipid ligands. To address this limitation, computational approaches are a powerful alternative methodology, but they are often limited by the high flexibility of lipid substrates. In this work, we develop an in silico protocol based on unbiased coarse grain molecular simulations in which lipids placed in bulk solvent away from the protein can spontaneously bind to LTPs. This approach accurately determines binding pockets in LTPs and provides a working hypothesis for the pathway via which lipids enter LTPs. We apply this approach to characterize lipid binding to bridge-like LTPs belonging to the Vps13-Atg2 family, for which the lipid localization inside the protein is currently unknown. Overall, our work paves the way to determine binding pockets and entry pathways for several LTPs in an inexpensive, fast, and accurate manner.\nTertiary folds of the SL5 RNA from the 5' proximal region of SARS-CoV-2 and related coronaviruses\nAuthors: Kretsch, R. C.; Xu, L.; Zheludev, I. N.; Zhou, X.; Huang, R.; Nye, G.; Li, S.; Zhang, K.; Chiu, W.; Das, R.\nScore: 8.7, Published: 2023-11-27 DOI: 10.1101/2023.11.22.567964\nCoronavirus genomes sequester their start codons within stem-loop 5 (SL5), a structured, 5' genomic RNA element. In most alpha- and betacoronaviruses, the secondary structure of SL5 is predicted to contain a four-way junction of helical stems, some of which are capped with UUYYGU hexaloops. Here, using cryogenic electron microscopy (cryo-EM) and computational modeling with biochemically-determined secondary structures, we present three-dimensional structures of SL5 from six coronaviruses. The SL5 domain of betacoronavirus SARS-CoV-2, resolved at 4.7 [A] resolution, exhibits a T-shaped structure, with its UUYYGU hexaloops at opposing ends of a coaxial stack, the Ts \"arms.\" Further analysis of SL5 domains from SARS-CoV-1 and MERS (7.1 and 6.4-6.9 [A] resolution, respectively) indicate that the junction geometry and inter-hexaloop distances are conserved features across the studied human-infecting betacoronaviruses. The MERS SL5 domain displays an additional tertiary interaction, which is also observed in the non-human-infecting betacoronavirus BtCoV-HKU5 (5.9-8.0 [A] resolution). SL5s from human-infecting alphacoronaviruses, HCoV-229E and HCoV-NL63 (6.5 and 8.4-9.0 [A] resolution, respectively), exhibit the same coaxial stacks, including the UUYYGU-capped arms, but with a phylogenetically distinct crossing angle, an X-shape. As such, all SL5 domains studied herein fold into stable tertiary structures with cross-genus similarities, with implications for potential protein-binding modes and therapeutic targets. SignificanceThe three-dimensional structures of viral RNAs are of interest to the study of viral pathogenesis and therapeutic design, but the three-dimensional structures of viral RNAs remain poorly characterized. Here, we provide the first 3D structures of the SL5 domain (124-160 nt, 40.0-51.4 kDa) from the majority of human-infecting coronaviruses. All studied SL5s exhibit a similar 4-way junction, with their crossing angles grouped along phylogenetic boundaries. Further, across all species studied, conserved UUYYGU hexaloop pairs are located at opposing ends of a coaxial stack, suggesting that their three-dimensional arrangement is important for their as-of-yet defined function. These conserved tertiary features support the relevance of SL5 for pan-coronavirus fitness and highlight new routes in understanding its molecular and virological roles and in developing SL5-based antivirals. Classification: Biological Sciences, Biophysics and Computational Biology\nNanoLocz: Image analysis platform for AFM, high-speed AFM and localization AFM\nAuthors: Heath, G. R.; Micklethwaite, E.; Storer, T. M.\nScore: 17.9, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568405\nNanoLocz is an open-source computer program designed for high-throughput automatic processing and single-particle analysis of Atomic Force Microscopy (AFM) image data. High-Speed AFM and Localization AFM (LAFM) enable the study of single molecules with increasingly higher spatiotemporal resolution. However, efficient and rapid analysis of the images and movies produced by these techniques can be challenging, often requiring the use of multiple image processing software applications and scripts. Here, we introduce NanoLocz, an AFM and high-speed AFM analysis program that facilitates various single-particle analysis workflows through a simple, interactive interface. Workflows include but are not limited to: single-particle tracking, single-particle topographic feature analysis, single-molecule LAFM, time-resolved LAFM, and simulation LAFM. The source code and installation instructions for NanoLocz are available at https://github.com/George-R-Heath/NanoLocz.\nPreserving condensate structure and composition by lowering sequence complexity\nAuthors: Sood, A.; Zhang, B.\nScore: 5.7, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569249\nBiological condensates play a vital role in organizing cellular chemistry. They selectively partition biomolecules, preventing unwanted cross-talk and buffering against chemical noise. Intrinsically disordered proteins (IDPs) serve as primary components of these condensates due to their flexibility and ability to engage in multivalent, nonspecific interactions, leading to spontaneous aggregation. Theoretical advancements are critical at connecting IDP sequences with condensate emergent properties to establish the so-called molecular grammar. We proposed an extension to the stickers and spacers model, incorporating non-specific pairwise interactions between spacers alongside specific interactions among stickers. Our investigation revealed that while spacer interactions contribute to phase separation and co-condensation, their non-specific nature leads to disorganized condensates. Specific sticker-sticker interactions drive the formation of condensates with well-defined structures and molecular composition. We discussed how evolutionary pressures might emerge to affect these interactions, leading to the prevalence of low complexity domains in IDP sequences. These domains suppress spurious interactions and facilitate the formation of biologically meaningful condensates. Significance StatementBiomolecular condensates serve as pivotal mechanisms in cellular organization, often characterized by an abundance of intrinsically disordered proteins (IDPs) that undergo frequent mutations in their sequences. Despite this, IDP sequences exhibit non-random patterns, yet the precise relationship between these sequences and the emergent properties of condensates remains unclear. To address this gap, we propose a molecular theory that delineates how various sequence features of IDPs contribute to the organization and composition of condensates. This theory not only sheds light on the evolution of IDPs but also elucidates the emergence of non-random sequence patterns as essential elements for the formation of functional condensates. Correspondingly, we posit that the prevalence of low-complexity regions within IDPs is a result of evolutionary selection.\nThe Sec61/TRAP Translocon Scrambles Lipids\nAuthors: Javanainen, M.; Karki, S.; Tranter, D.; Biriukov, D.; Paavilainen, V. O.\nScore: 6.6, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568215\nCell growth relies on the rapid flip-flop of newly synthesized lipids across the ER membrane. This process is facilitated without the need for ATP by specific membrane proteins--scramblases--a few of which have been very recently identified in the ER. We have previously resolved the structure of the translocon-associated protein (TRAP) bound to the Sec61 translocon in the ER membrane, and found this complex to render the membrane locally thinner. Moreover, Sec61 and TRAP each contain a crevice rich in polar residues that can shield a lipid head group as it traverses the hydrophobic membrane environment. We thus hypothesized that both Sec61 and TRAP act as ER scramblases. Here, we characterized the scrambling activity of Sec61 and TRAP using extensive molecular dynamics simulations. We observed that both Sec61 and TRAP efficiently scramble lipids via a credit card mechanism. We analyzed the kinetics and thermodynamics of lipid scrambling and demonstrated that local membrane thinning provides a key contribution to scrambling efficiency. Both proteins appear seemingly selective towards phosphatidylcholine lipids over phosphatidylethanolamine and phosphatidylserine, yet this behavior rather reflects the trends observed for these lipids in a protein-free membrane. The identified scrambling pathway in Sec61 structure is physiologically rarely unoccupied due to its role in protein translocation. Furthermore, we found that the scrambling activity of this pathway might be impeded by the presence of ions at a physiological concentration. However, the trimeric bundle of TRAP{beta}, TRAP{gamma}, and TRAP{delta} might provide scrambling activity insensitive to the functional state of the translocon and the solvent conditions.\nLow-cost 3D printed lenses for brightfield and fluorescence microscopy\nAuthors: Christopher, J.; Rooney, L. M.; Donnachie, M.; Uttamchandani, D.; McConnell, G.; Bauer, R.\nScore: 14.0, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568227\nWe present the fabrication and implementation of low-cost optical quality 3D printed lenses, and their application as microscope objectives with different prescriptions. The imaging performance of the 3D printed lenses was benchmarked against commercially available optics including a 20 mm focal length 12.7 mm diameter NBK-7 plano-convex lens used as a low magnification objective, and a separate high magnification objective featuring three 6 mm diameter NBK-7 lenses with different positive and negative focal lengths. We describe the design and manufacturing processes to produce high-quality 3D printed lenses. We tested their transmissivity and surface quality using a stylus profilometer, showing that they conform to that of commercial glass counterpart lenses. The 3D printed lenses were used as microscope objectives in both brightfield and epi-fluorescence imaging of specimens including onion, cyanobacteria, and variegated Hosta leaves, demonstrating a sub-cellular resolution performance obtained with low-cost 3D printed optical elements within brightfield and fluorescence microscopy.\nTau P301L mutation promotes core 4R tauopathy fibril fold through near-surface water structuring and conformational rearrangement\nAuthors: Vigers, M.; Lobo, S.; Najafi, S.; Dubose, A.; Tsay, K.; Ganguly, P.; Longhini, A. P.; Jin, Y.; Buratto, S. K.; Kosik, K. S.; Shell, M. S.; Shea, J.-E.; Han, S.\nScore: 4.5, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568818\nTau forms toxic fibrillar aggregates in a family of neurodegenerative diseases known as tauopathies. The faithful replication of tauopathy-specific fibril structures is a critical gap for developing diagnostic and therapeutic tools. This study debuts a strategy of identifying a critical segment of tau that forms a folding motif that is characteristic of a family of tauopathies and isolating it as a standalone peptide that form seeding-competent fibrils. The 19-residue jR2R3 peptide (295-313) spanning the R2/R3 splice junction of tau, in the presence of P301L, forms seeding-competent amyloid fibrils. This tau fragment contains the hydrophobic VQIVYK hexapeptide that is part of the core of every pathological tau fibril structure solved to-date and an intramolecular counter-strand that stabilizes the strand-loop-strand (SLS) motif observed in 4R tauopathy fibrils. This study shows that P301L exhibits a duality of effects: it lowers the barrier for the peptide to adopt aggregation-prone conformations and enhances the local structuring of water around the mutation site that facilitates site-specific dewetting and in-register stacking of tau to form cross {beta}-sheets. We solve a 3 [A] cryo-EM structure of jR2R3-P301L fibrils with a pseudo 21 screw symmetry in which each half of the fibrils cross-section contains two jR2R3-P301L peptides. One chain adopts a SLS fold found in 4R tauopathies that is stabilized by a second chain wrapping around the SLS fold, reminiscent of the 3-fold and 4-fold structures observed in 4R tauopathies. These jR2R3-P301L fibrils are able to template full length tau in a prion-like fashion. Significance StatementThis study presents a first step towards designing a tauopathy specific aggregation pathway by engineering a minimal tau prion building block, jR2R3, that can template and propagate distinct disease folds. We present the discovery that P301L--among the widest used mutations in cell and animal models of Alzheimers Disease--destabilizes an aggregation-prohibiting internal hairpin and enhances the local surface water structure that serves as an entropic hotspot to exert a hyper-localized effect in jR2R3. Our study suggests that P301L may be a more suitable mutation to include in modeling 4R tauopathies than for modelling Alzheimers Disease, and that mutations are powerful tools for the purpose of designing of tau prion models as therapeutic tools.\n",
  "wordCount" : "2367",
  "inLanguage": "en",
  "datePublished": "2023-12-03T10:38:01Z",
  "dateModified": "2023-12-03T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 3, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566946">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566946" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566946">
        <p class="paperTitle">High-throughput thermodynamic and kinetic measurements of transcription factor/DNA mutations reveal how conformational heterogeneity can shape motif selectivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566946" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566946" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hastings, R.; Aditham, A.; DelRosso, N.; Suzuki, P.; Fordyce, P. M.</p>
        <p class="info">Score: 21.3, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566946' target='https://doi.org/10.1101/2023.11.13.566946'> 10.1101/2023.11.13.566946</a></p>
        <p class="abstract">Transcription factors (TFs) bind DNA sequences with a range of affinities, yet the mechanisms determining energetic differences between high- and low-affinity sequences ( selectivity) remain poorly understood. Here, we investigated two basic helix-loop-helix TFs, MAX (H. sapiens) and Pho4 (S. cerevisiae), that bind the same high-affinity sequence with highly similar nucleotide-contacting residues and bound structures but are differentially selective for non-cognate sequences. By measuring &gt;1700 Kds and &gt;500 rate constants for Pho4 and MAX mutant libraries binding multiple DNA sequences and comparing these measurements with thermodynamic and kinetic models, we identify the biophysical mechanisms by which changes to TF sequence alter both bound and unbound conformational ensembles to shape specificity landscapes. These results highlight the importance of conformational heterogeneity in determining sequence specificity and selectivity and can guide future efforts to engineer nucleic acid-binding proteins with enhanced selectivity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569057">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569057" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569057">
        <p class="paperTitle">Allosteric Competition and Inhibition in AMPA Receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569057" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569057" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hale, W. D.; Romero, A. M.; Gonzalez, C. U.; Jayaraman, V.; Lau, A. Y.; Huganir, R. L.; Twomey, E. C.</p>
        <p class="info">Score: 17.8, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569057' target='https://doi.org/10.1101/2023.11.28.569057'> 10.1101/2023.11.28.569057</a></p>
        <p class="abstract">Excitatory neurotransmission is principally mediated by AMPA-subtype ionotropic glutamate receptors (AMPARs). Dysregulation of AMPARs is the cause of many neurological disorders and how therapeutic candidates such as negative allosteric modulators inhibit AMPARs is unclear. Here, we show that non-competitive inhibition desensitizes AMPARs to activation and prevents positive allosteric modulation. We dissected the noncompetitive inhibition mechanism of action by capturing AMPARs bound to glutamate and the prototypical negative allosteric modulator, GYKI-52466, with cryo-electron microscopy. Noncompetitive inhibition by GYKI-52466, which binds in the transmembrane collar region surrounding the ion channel, negatively modulates AMPARs by decoupling glutamate binding in the ligand binding domain from the ion channel. Furthermore, during allosteric competition between negative and positive modulators, negative allosteric modulation by GKYI-52466 outcompetes positive allosteric modulators to control AMPAR function. Our data provide a new framework for understanding allostery of AMPARs and foundations for rational design of therapeutics targeting AMPARs in neurological diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569079">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569079" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569079">
        <p class="paperTitle">Structure of Transmembrane AMPA Receptor Regulatory Protein Subunit Î³2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569079" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569079" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hale, W. D.; Romero, A. M.; Huganir, R. L.; Twomey, E. C.</p>
        <p class="info">Score: 15.8, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569079' target='https://doi.org/10.1101/2023.11.28.569079'> 10.1101/2023.11.28.569079</a></p>
        <p class="abstract">Transmembrane AMPA receptor regulatory proteins (TARPs) are claudin-like proteins that tightly regulate AMPA receptors (AMPARs) and are fundamental for excitatory neurotransmission. We used cryo-electron microscopy (cryo-EM) to reconstruct the 36 kDa TARP subunit {gamma}2 to 2.3 [A] and reveal the structural diversity of TARPs. Our data reveals critical motifs that distinguish TARPs from claudins and define how sequence variations within TARPs differentiate subfamilies and their regulation of AMPARs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568550">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568550" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568550">
        <p class="paperTitle">Unbiased MD simulations characterize lipid binding pockets in lipid transfer proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568550" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568550" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vanni, S.; Srinivasan, S.; Alvarez Lorenzo, D.</p>
        <p class="info">Score: 10.8, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568550' target='https://doi.org/10.1101/2023.11.24.568550'> 10.1101/2023.11.24.568550</a></p>
        <p class="abstract">The molecular characterization of lipid binding to lipid transfer proteins (LTPs) is fundamental to the understanding of several aspects of their mechanistic mode of action. However, obtaining lipid-bound structures of LTPs is not straightforward owing to caveats in current experimental structural biology approaches. As a result, several structures of LTPs, and most notably almost all of those that have been proposed to act as bridges between membrane organelles, do not provide the precise location of their endogenous lipid ligands. To address this limitation, computational approaches are a powerful alternative methodology, but they are often limited by the high flexibility of lipid substrates. In this work, we develop an in silico protocol based on unbiased coarse grain molecular simulations in which lipids placed in bulk solvent away from the protein can spontaneously bind to LTPs. This approach accurately determines binding pockets in LTPs and provides a working hypothesis for the pathway via which lipids enter LTPs. We apply this approach to characterize lipid binding to bridge-like LTPs belonging to the Vps13-Atg2 family, for which the lipid localization inside the protein is currently unknown. Overall, our work paves the way to determine binding pockets and entry pathways for several LTPs in an inexpensive, fast, and accurate manner.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.567964">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.567964" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.567964">
        <p class="paperTitle">Tertiary folds of the SL5 RNA from the 5&#39; proximal region of SARS-CoV-2 and related coronaviruses</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.567964" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.567964" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kretsch, R. C.; Xu, L.; Zheludev, I. N.; Zhou, X.; Huang, R.; Nye, G.; Li, S.; Zhang, K.; Chiu, W.; Das, R.</p>
        <p class="info">Score: 8.7, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.567964' target='https://doi.org/10.1101/2023.11.22.567964'> 10.1101/2023.11.22.567964</a></p>
        <p class="abstract">Coronavirus genomes sequester their start codons within stem-loop 5 (SL5), a structured, 5&#39; genomic RNA element. In most alpha- and betacoronaviruses, the secondary structure of SL5 is predicted to contain a four-way junction of helical stems, some of which are capped with UUYYGU hexaloops. Here, using cryogenic electron microscopy (cryo-EM) and computational modeling with biochemically-determined secondary structures, we present three-dimensional structures of SL5 from six coronaviruses. The SL5 domain of betacoronavirus SARS-CoV-2, resolved at 4.7 [A] resolution, exhibits a T-shaped structure, with its UUYYGU hexaloops at opposing ends of a coaxial stack, the Ts &#34;arms.&#34; Further analysis of SL5 domains from SARS-CoV-1 and MERS (7.1 and 6.4-6.9 [A] resolution, respectively) indicate that the junction geometry and inter-hexaloop distances are conserved features across the studied human-infecting betacoronaviruses. The MERS SL5 domain displays an additional tertiary interaction, which is also observed in the non-human-infecting betacoronavirus BtCoV-HKU5 (5.9-8.0 [A] resolution). SL5s from human-infecting alphacoronaviruses, HCoV-229E and HCoV-NL63 (6.5 and 8.4-9.0 [A] resolution, respectively), exhibit the same coaxial stacks, including the UUYYGU-capped arms, but with a phylogenetically distinct crossing angle, an X-shape. As such, all SL5 domains studied herein fold into stable tertiary structures with cross-genus similarities, with implications for potential protein-binding modes and therapeutic targets.

SignificanceThe three-dimensional structures of viral RNAs are of interest to the study of viral pathogenesis and therapeutic design, but the three-dimensional structures of viral RNAs remain poorly characterized. Here, we provide the first 3D structures of the SL5 domain (124-160 nt, 40.0-51.4 kDa) from the majority of human-infecting coronaviruses. All studied SL5s exhibit a similar 4-way junction, with their crossing angles grouped along phylogenetic boundaries. Further, across all species studied, conserved UUYYGU hexaloop pairs are located at opposing ends of a coaxial stack, suggesting that their three-dimensional arrangement is important for their as-of-yet defined function. These conserved tertiary features support the relevance of SL5 for pan-coronavirus fitness and highlight new routes in understanding its molecular and virological roles and in developing SL5-based antivirals.

Classification: Biological Sciences, Biophysics and Computational Biology</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568405">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568405" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568405">
        <p class="paperTitle">NanoLocz: Image analysis platform for AFM, high-speed AFM and localization AFM</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568405" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568405" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heath, G. R.; Micklethwaite, E.; Storer, T. M.</p>
        <p class="info">Score: 17.9, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568405' target='https://doi.org/10.1101/2023.11.23.568405'> 10.1101/2023.11.23.568405</a></p>
        <p class="abstract">NanoLocz is an open-source computer program designed for high-throughput automatic processing and single-particle analysis of Atomic Force Microscopy (AFM) image data. High-Speed AFM and Localization AFM (LAFM) enable the study of single molecules with increasingly higher spatiotemporal resolution. However, efficient and rapid analysis of the images and movies produced by these techniques can be challenging, often requiring the use of multiple image processing software applications and scripts. Here, we introduce NanoLocz, an AFM and high-speed AFM analysis program that facilitates various single-particle analysis workflows through a simple, interactive interface. Workflows include but are not limited to: single-particle tracking, single-particle topographic feature analysis, single-molecule LAFM, time-resolved LAFM, and simulation LAFM. The source code and installation instructions for NanoLocz are available at https://github.com/George-R-Heath/NanoLocz.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569249">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569249" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569249">
        <p class="paperTitle">Preserving condensate structure and composition by lowering sequence complexity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569249" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569249" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sood, A.; Zhang, B.</p>
        <p class="info">Score: 5.7, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569249' target='https://doi.org/10.1101/2023.11.29.569249'> 10.1101/2023.11.29.569249</a></p>
        <p class="abstract">Biological condensates play a vital role in organizing cellular chemistry. They selectively partition biomolecules, preventing unwanted cross-talk and buffering against chemical noise. Intrinsically disordered proteins (IDPs) serve as primary components of these condensates due to their flexibility and ability to engage in multivalent, nonspecific interactions, leading to spontaneous aggregation. Theoretical advancements are critical at connecting IDP sequences with condensate emergent properties to establish the so-called molecular grammar. We proposed an extension to the stickers and spacers model, incorporating non-specific pairwise interactions between spacers alongside specific interactions among stickers. Our investigation revealed that while spacer interactions contribute to phase separation and co-condensation, their non-specific nature leads to disorganized condensates. Specific sticker-sticker interactions drive the formation of condensates with well-defined structures and molecular composition. We discussed how evolutionary pressures might emerge to affect these interactions, leading to the prevalence of low complexity domains in IDP sequences. These domains suppress spurious interactions and facilitate the formation of biologically meaningful condensates.

Significance StatementBiomolecular condensates serve as pivotal mechanisms in cellular organization, often characterized by an abundance of intrinsically disordered proteins (IDPs) that undergo frequent mutations in their sequences. Despite this, IDP sequences exhibit non-random patterns, yet the precise relationship between these sequences and the emergent properties of condensates remains unclear. To address this gap, we propose a molecular theory that delineates how various sequence features of IDPs contribute to the organization and composition of condensates. This theory not only sheds light on the evolution of IDPs but also elucidates the emergence of non-random sequence patterns as essential elements for the formation of functional condensates. Correspondingly, we posit that the prevalence of low-complexity regions within IDPs is a result of evolutionary selection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568215">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568215" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568215">
        <p class="paperTitle">The Sec61/TRAP Translocon Scrambles Lipids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568215" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568215" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Javanainen, M.; Karki, S.; Tranter, D.; Biriukov, D.; Paavilainen, V. O.</p>
        <p class="info">Score: 6.6, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568215' target='https://doi.org/10.1101/2023.11.23.568215'> 10.1101/2023.11.23.568215</a></p>
        <p class="abstract">Cell growth relies on the rapid flip-flop of newly synthesized lipids across the ER membrane. This process is facilitated without the need for ATP by specific membrane proteins--scramblases--a few of which have been very recently identified in the ER. We have previously resolved the structure of the translocon-associated protein (TRAP) bound to the Sec61 translocon in the ER membrane, and found this complex to render the membrane locally thinner. Moreover, Sec61 and TRAP each contain a crevice rich in polar residues that can shield a lipid head group as it traverses the hydrophobic membrane environment. We thus hypothesized that both Sec61 and TRAP act as ER scramblases. Here, we characterized the scrambling activity of Sec61 and TRAP using extensive molecular dynamics simulations. We observed that both Sec61 and TRAP efficiently scramble lipids via a credit card mechanism. We analyzed the kinetics and thermodynamics of lipid scrambling and demonstrated that local membrane thinning provides a key contribution to scrambling efficiency. Both proteins appear seemingly selective towards phosphatidylcholine lipids over phosphatidylethanolamine and phosphatidylserine, yet this behavior rather reflects the trends observed for these lipids in a protein-free membrane. The identified scrambling pathway in Sec61 structure is physiologically rarely unoccupied due to its role in protein translocation. Furthermore, we found that the scrambling activity of this pathway might be impeded by the presence of ions at a physiological concentration. However, the trimeric bundle of TRAP{beta}, TRAP{gamma}, and TRAP{delta} might provide scrambling activity insensitive to the functional state of the translocon and the solvent conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568227">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568227" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568227">
        <p class="paperTitle">Low-cost 3D printed lenses for brightfield and fluorescence microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568227" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568227" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Christopher, J.; Rooney, L. M.; Donnachie, M.; Uttamchandani, D.; McConnell, G.; Bauer, R.</p>
        <p class="info">Score: 14.0, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568227' target='https://doi.org/10.1101/2023.11.22.568227'> 10.1101/2023.11.22.568227</a></p>
        <p class="abstract">We present the fabrication and implementation of low-cost optical quality 3D printed lenses, and their application as microscope objectives with different prescriptions. The imaging performance of the 3D printed lenses was benchmarked against commercially available optics including a 20 mm focal length 12.7 mm diameter NBK-7 plano-convex lens used as a low magnification objective, and a separate high magnification objective featuring three 6 mm diameter NBK-7 lenses with different positive and negative focal lengths. We describe the design and manufacturing processes to produce high-quality 3D printed lenses. We tested their transmissivity and surface quality using a stylus profilometer, showing that they conform to that of commercial glass counterpart lenses. The 3D printed lenses were used as microscope objectives in both brightfield and epi-fluorescence imaging of specimens including onion, cyanobacteria, and variegated Hosta leaves, demonstrating a sub-cellular resolution performance obtained with low-cost 3D printed optical elements within brightfield and fluorescence microscopy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568818">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568818" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568818">
        <p class="paperTitle">Tau P301L mutation promotes core 4R tauopathy fibril fold through near-surface water structuring and conformational rearrangement</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568818" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568818" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vigers, M.; Lobo, S.; Najafi, S.; Dubose, A.; Tsay, K.; Ganguly, P.; Longhini, A. P.; Jin, Y.; Buratto, S. K.; Kosik, K. S.; Shell, M. S.; Shea, J.-E.; Han, S.</p>
        <p class="info">Score: 4.5, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568818' target='https://doi.org/10.1101/2023.11.28.568818'> 10.1101/2023.11.28.568818</a></p>
        <p class="abstract">Tau forms toxic fibrillar aggregates in a family of neurodegenerative diseases known as tauopathies. The faithful replication of tauopathy-specific fibril structures is a critical gap for developing diagnostic and therapeutic tools. This study debuts a strategy of identifying a critical segment of tau that forms a folding motif that is characteristic of a family of tauopathies and isolating it as a standalone peptide that form seeding-competent fibrils. The 19-residue jR2R3 peptide (295-313) spanning the R2/R3 splice junction of tau, in the presence of P301L, forms seeding-competent amyloid fibrils. This tau fragment contains the hydrophobic VQIVYK hexapeptide that is part of the core of every pathological tau fibril structure solved to-date and an intramolecular counter-strand that stabilizes the strand-loop-strand (SLS) motif observed in 4R tauopathy fibrils. This study shows that P301L exhibits a duality of effects: it lowers the barrier for the peptide to adopt aggregation-prone conformations and enhances the local structuring of water around the mutation site that facilitates site-specific dewetting and in-register stacking of tau to form cross {beta}-sheets. We solve a 3 [A] cryo-EM structure of jR2R3-P301L fibrils with a pseudo 21 screw symmetry in which each half of the fibrils cross-section contains two jR2R3-P301L peptides. One chain adopts a SLS fold found in 4R tauopathies that is stabilized by a second chain wrapping around the SLS fold, reminiscent of the 3-fold and 4-fold structures observed in 4R tauopathies. These jR2R3-P301L fibrils are able to template full length tau in a prion-like fashion.

Significance StatementThis study presents a first step towards designing a tauopathy specific aggregation pathway by engineering a minimal tau prion building block, jR2R3, that can template and propagate distinct disease folds. We present the discovery that P301L--among the widest used mutations in cell and animal models of Alzheimers Disease--destabilizes an aggregation-prohibiting internal hairpin and enhances the local surface water structure that serves as an entropic hotspot to exert a hyper-localized effect in jR2R3. Our study suggests that P301L may be a more suitable mutation to include in modeling 4R tauopathies than for modelling Alzheimers Disease, and that mutations are powerful tools for the purpose of designing of tau prion models as therapeutic tools.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
