<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Bacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems
Authors: Chambers, L. R.; Ye, Q.; Cai, J.; Gong, M.; Ledvina, H. E.; Zhou, H.; Whiteley, A. T.; Suhandynata, R. T.; Corbett, K. D.
Score: 47.7, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559546
Ubiquitination and related pathways play crucial roles in protein homeostasis, signaling, and innate immunity1-3. In these pathways, an enzymatic cascade of E1, E2, and E3 proteins conjugates ubiquitin or a ubiquitin-like protein (Ubl) to targetprotein lysine residues4.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Bacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems
Authors: Chambers, L. R.; Ye, Q.; Cai, J.; Gong, M.; Ledvina, H. E.; Zhou, H.; Whiteley, A. T.; Suhandynata, R. T.; Corbett, K. D.
Score: 47.7, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559546
Ubiquitination and related pathways play crucial roles in protein homeostasis, signaling, and innate immunity1-3. In these pathways, an enzymatic cascade of E1, E2, and E3 proteins conjugates ubiquitin or a ubiquitin-like protein (Ubl) to targetprotein lysine residues4." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-01T10:37:47+00:00" />
<meta property="article:modified_time" content="2023-10-01T10:37:47+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Bacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems
Authors: Chambers, L. R.; Ye, Q.; Cai, J.; Gong, M.; Ledvina, H. E.; Zhou, H.; Whiteley, A. T.; Suhandynata, R. T.; Corbett, K. D.
Score: 47.7, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559546
Ubiquitination and related pathways play crucial roles in protein homeostasis, signaling, and innate immunity1-3. In these pathways, an enzymatic cascade of E1, E2, and E3 proteins conjugates ubiquitin or a ubiquitin-like protein (Ubl) to targetprotein lysine residues4."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Bacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems\nAuthors: Chambers, L. R.; Ye, Q.; Cai, J.; Gong, M.; Ledvina, H. E.; Zhou, H.; Whiteley, A. T.; Suhandynata, R. T.; Corbett, K. D.\nScore: 47.7, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559546\nUbiquitination and related pathways play crucial roles in protein homeostasis, signaling, and innate immunity1-3. In these pathways, an enzymatic cascade of E1, E2, and E3 proteins conjugates ubiquitin or a ubiquitin-like protein (Ubl) to targetprotein lysine residues4.",
  "keywords": [
    
  ],
  "articleBody": " Bacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems\nAuthors: Chambers, L. R.; Ye, Q.; Cai, J.; Gong, M.; Ledvina, H. E.; Zhou, H.; Whiteley, A. T.; Suhandynata, R. T.; Corbett, K. D.\nScore: 47.7, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559546\nUbiquitination and related pathways play crucial roles in protein homeostasis, signaling, and innate immunity1-3. In these pathways, an enzymatic cascade of E1, E2, and E3 proteins conjugates ubiquitin or a ubiquitin-like protein (Ubl) to targetprotein lysine residues4. Bacteria encode ancient relatives of E1 and Ubl proteins involved in sulfur metabolism5,6 but these proteins do not mediate Ubl-target conjugation, leaving open the question of whether bacteria can perform ubiquitination-like protein conjugation. Here, we demonstrate that a bacterial antiviral immune system encodes a complete ubiquitination pathway. Two structures of a bacterial E1:E2:Ubl complex reveal striking architectural parallels with canonical eukaryotic ubiquitination machinery. The bacterial E1 encodes an N-terminal inactive adenylation domain (IAD) and a C-terminal active adenylation domain (AAD) with a mobile -helical insertion containing the catalytic cysteine (CYS domain). One structure reveals a pre-reaction state with the bacterial Ubl C-terminus positioned for adenylation, and the E1 CYS domain poised nearby for thioester formation. A second structure mimics an E1-to-E2 transthioesterification state, with the E1 CYS domain rotated outward and its catalytic cysteine adjacent to the bound E2. We show that a deubiquitinase (DUB) in the same pathway pre-processes the bacterial Ubl, exposing its C-terminal glycine for adenylation. Finally, we show that the bacterial E1 and E2 collaborate to conjugate Ubl to target-protein lysine residues. Together, these data reveal that bacteria possess bona fide ubiquitination systems with strong mechanistic and architectural parallels to canonical eukaryotic ubiquitination pathways, suggesting that these pathways arose first in bacteria.\nPattern formation by bacteria-phage interactions\nAuthors: Martinez-Calvo, A.; Wingreen, N. S.; Datta, S. S.\nScore: 38.1, Published: 2023-09-20 DOI: 10.1101/2023.09.19.558479\nThe interactions between bacteria and phages--viruses that infect bacteria--play critical roles in agriculture, ecology, and medicine; however, how these interactions influence the spatial organization of both bacteria and phages remain largely unexplored. Here, we address this gap in knowledge by developing a theoretical model of motile, proliferating bacteria that aggregate via motility-induced phase separation (MIPS) and encounter phage that infect and lyse the cells. We find that the non-reciprocal predator-prey interactions between phage and bacteria strongly alter spatial organization, in some cases giving rise to a rich array of finite-scale stationary and dynamic patterns in which bacteria and phage coexist. We establish principles describing the onset and characteristics of these diverse behaviors, thereby helping to provide a biophysical basis for understanding pattern formation in bacteria-phage systems, as well as in a broader range of active and living systems with similar predator-prey or other non-reciprocal interactions.\nMarangoni-like tissue flows enhance symmetry breaking of embryonic organoids\nAuthors: Gsell, S.; Tlili, S. L.; Merkel, M.; Lenne, P.-F.\nScore: 19.2, Published: 2023-09-23 DOI: 10.1101/2023.09.22.559003\nDuring early development of multi-cellular animals, cells self-organize to set up the body axes, such as the primary head-to-tail axis, based on which the later body plan is defined. Several signaling pathways are known to control body axis formation. Here, we show, however, that tissue mechanics plays an important role during this process. We focus on the emergence of a primary axis in initially spherical aggregates of mouse embryonic stem cells, which mirrors events in the early mouse embryo. These aggregates break rotational symmetry to establish an axial organization with domains of different expression profiles, e.g. of the transcription factor T/Bra and the adhesion molecule E-cadherin. Combining quantitative microscopy and physical modeling, we identify large-scale tissue flows with a recirculation component and demonstrate that they significantly contribute to symmetry breaking. We show that the recirculating flows are explained by a difference in tissue surface tension across domains, akin to Marangoni flows, which we further confirm by aggregate fusion experiments. Our work highlights that body axis formation is not only driven by biochemical processes, but that it can also be amplified by tissue flows. We expect this type of amplification to operate in many other organoid and in-vivo systems.\nRegeneration of actin filament branches from the same Arp2/3 complex\nAuthors: Ghasemi, F.; Cao, L.; Mladenov, M.; Guichard, B.; Way, M.; Jegou, A.; Romet-Lemonne, G.\nScore: 19.5, Published: 2023-09-22 DOI: 10.1101/2023.09.22.558980\nBranched actin filaments are found in many key cellular structures. Branches are nucleated by the Arp2/3 complex activated by nucleation-promoting factor (NPF) proteins and bound to the side of pre-existing mother filaments. Over time, branches dissociate from their mother filament, leading to network reorganization and turnover, but this mechanism is less understood. Here, using microfluidics and purified proteins, we examined the dissociation of individual branches under controlled biochemical and mechanical conditions. We observe that Arp2/3 remains bound to the mother filament after most debranching events, even when accelerated by force. Unexpectedly, this mother-remaining Arp2/3 readily nucleates a new actin filament branch, without being activated anew by an NPF: it simply needs to exchange its nucleotide and bind an actin monomer. The protein GMF, which accelerates debranching, prevents branch re-nucleation. Our results suggest that actin filament re-nucleation can provide a self-repair mechanism, helping branched networks to sustain mechanical stress in cells over extended periods of time.\nA model for collagen secretion by intercompartmental continuities\nAuthors: Bunel, L.; Pincet, L.; Malhotra, V.; Raote, I.; Pincet, F.\nScore: 16.0, Published: 2023-09-23 DOI: 10.1101/2023.09.21.558805\nNewly synthesized secretory proteins are exported from endoplasmic reticulum (ER) at specialized subcompartments called exit sites (ERES). Cargoes like procollagen are too large for export by the standard COPII-coated vesicle of 60 nm average diameter. We have previously suggested that procollagen is transported from the ER to the next secretory organelle, the ERGIC, in TANGO1-dependent inter-organelle tunnels. Here, we show that intrinsically disordered domains of TANGO1 in the ER lumen generate an entropic contraction that pulls procollagen towards the ERES. Molecular gradients of pH and HSP47 between the ER and ERGIC generate a force in the range of tens of femtoNewtons (fN), which is sufficient to propel procollagen from the ER at a speed of [~]1 nm.s-1. This calculated speed and the quantities of collagen secreted are similar to its observed physiological secretion rate in fibroblasts, consistent with the proposal that ER export is the rate limiting step for procollagen secretion. Our theoretical model explains how cells can utilize molecular gradients to export procollagens at a rate commensurate with physiological needs. Significance StatementProcollagen cannot be exported from the endoplasmic reticulum (ER) by standard COPII-coated vesicle of 60 nm average diameter. We have previously suggested that collagen is transported from the ER to the next secretory organelle, the ERGIC, in TANGO1-dependent inter-organelle tunnels. ER and ERGIC differ in molecular composition including their pH and protein composition. We propose a mechanical/entropic ratchet model whereby molecular gradients of pH and the collagen chaperone HSP47, provide the energy to propel procollagen from the ER at a speed that matches the physiological rate of collagen secretion.\nLipid peroxidation drives liquid-liquid phase separation and disrupts raft protein partitioning in biological membranes\nAuthors: Balakrishnan, M.; Kenworthy, A. K.\nScore: 16.4, Published: 2023-09-14 DOI: 10.1101/2023.09.12.557355\nThe peroxidation of membrane lipids by free radicals contributes to aging, numerous diseases, and ferroptosis, an iron-dependent form of cell death. Peroxidation changes the structure, conformation and physicochemical properties of lipids, leading to major membrane alterations including bilayer thinning, altered fluidity, and increased permeability. Whether and how lipid peroxidation impacts the lateral organization of proteins and lipids in biological membranes, however, remains poorly understood. Here, we employ cell-derived giant plasma membrane vesicles (GPMVs) as a model to investigate the impact of lipid peroxidation on ordered membrane domains, often termed membrane rafts. We show that lipid peroxidation induced by the Fenton reaction dramatically enhances phase separation propensity of GPMVs into co-existing liquid ordered (raft) and liquid disordered (non-raft) domains and increases the relative abundance of the disordered, non-raft phase. Peroxidation also leads to preferential accumulation of peroxidized lipids and 4-hydroxynonenal (4-HNE) adducts in the disordered phase, decreased lipid packing in both raft and non-raft domains, and translocation of multiple classes of proteins out of rafts. These findings indicate that peroxidation of plasma membrane lipids disturbs many aspects of membrane rafts, including their stability, abundance, packing, and protein and lipid composition. We propose that these disruptions contribute to the pathological consequences of lipid peroxidation during aging and disease, and thus serve as potential targets for therapeutic intervention.\nFull-Length Single-Molecule Protein Fingerprinting\nAuthors: Filius, M.; van Wee, R.; de Lannoy, C.; Westerlaken, I.; Li, Z.; Kim, S. H.; de Agrela Pinto, C.; Wu, Y.; Boons, G.-J.; Pabst, M.; de Ridder, D.; Joo, C.\nScore: 10.2, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559471\nProteins are the primary functional actors of the cell. Hence, their identification is pivotal to advance our understanding of cell biology and disease. Current protein analysis methods are of limited use for distinguishing proteoforms. In particular, mass spectrometric methods often provide only ambiguous information on post-translational modification sites, and sequences of co-existing modifications may not be resolved. Here we demonstrate FRET-based single-molecule protein fingerprinting to map the location of individual amino acids and a post-translational modification within single full-length protein molecules. Using an approach that relies on transient binding of fluorescently labeled DNA strands to probe the amino acids on a protein one by one we show that we can fingerprint intrinsically disordered proteins as well as folded globular proteins with sub-nanometer resolution. We anticipate that this technology will be used for proteoform identification in biological and translational research with ultimate sensitivity.\nIncorporating prior knowledge to seeds of adaptive sampling molecular dynamics simulations of ligand transport in enzymes with buried active sites\nAuthors: Sarkar, D. K.; Surpeta, B.; Brezovsky, J.\nScore: 8.1, Published: 2023-09-23 DOI: 10.1101/2023.09.21.558608\nGiven that most proteins have buried active sites, protein tunnels or channels play a crucial role in mitigating the transport of small molecules to the buried cavity for enzymatic catalysis. Tunnels can critically modulate the biological process of protein-ligand recognition. Various molecular dynamics methods have been developed for exploring and exploiting the protein-ligand conformational space to extract high-resolution details of the binding processes, one of the most recent represented by energetically unbiased high-throughput adaptive sampling simulations. The current study systematically contrasts the role of integrating prior knowledge while generating useful initial protein-ligand configurations, called seeds, for these simulations. Using a non-trivial system of haloalkane dehalogenase mutant with multiple transport tunnels leading to a deeply buried active site, these simulations were employed to derive kinetic models describing the process of association and dissociation of the substrate molecule. The more knowledge-based seed generation enabled high-throughput simulations that could more consistently capture the entire transport process, effectively explore the complex network of transport tunnels, and predict equilibrium dissociation constants, koff/kon, on the same order of magnitude as experimental measurements. Overall, the infusion of more knowledge into the initial seeds of adaptive sampling simulations could render analyses of transport mechanisms in enzymes more consistent even for very complex biomolecular systems, thereby promoting the rational design of enzymes with buried active sites and drug development efforts.\nAn optogenetic method for the controlled release of single molecules\nAuthors: Kashyap, P.; Bertelli, S.; Cao, F.; Kostritskaia, Y.; Blank, F.; Srikanth, N.; Saleppico, R.; Bierhuizen, D.; Lu, X.; Nickel, W.; Campbell, R. E.; Plested, A.; Stauber, T.; Taylor, M. J.; Ewers, H.\nScore: 25.8, Published: 2023-09-17 DOI: 10.1101/2023.09.16.557871\nWe developed a system for optogenetic release of single molecules in live cells. We confined soluble and transmembrane proteins to the Golgi apparatus via a photocleavable protein and released them by short pulses of light. Our method allows for the controlled delivery of functional proteins to cytosol and plasma membrane in amounts compatible with single molecule imaging, greatly simplifying access to single molecule microscopy of any protein in live cells. Furthermore, we could reconstitute cellular functions such as ion conductance by delivering BK and VRAC ion channels to the plasma membrane. Finally, we could induce NF-kB signaling in T-Lymphoblasts stimulated by IL-1 by controlled release of a signaling protein that had been knocked-out in the same cells. We observed light induced formation of functional inflammatory signaling complexes that could trigger IKK phosphorylation in single cells. We thus developed an optogenetic method for the reconstitution and investigation of cellular function at the single molecule level.\n4D Mesoscale liquid model of nucleus resolves chromatin's radial organization\nAuthors: Laghmach, R.; Di Pierro, M.; Potoyan, D.\nScore: 8.3, Published: 2023-09-23 DOI: 10.1101/2023.09.21.558819\nRecent chromatin capture, imaging techniques, and polymer modeling advancements have dramatically enhanced our quantitative understanding of chromosomal folding. However, the dynamism inherent in genome architectures due to physical and biochemical forces and their impact on nuclear architecture and cellular functions remains elusive. While imaging techniques capable of probing the physical properties of chromatin in 4D are growing, there is a conspicuous lack of physics-based computational tools appropriate for revealing the underlying forces that shape nuclear architecture and dynamics. To this end, we have developed a multi-phase liquid model of the nucleus, which can resolve chromosomal territories, compartments, and nuclear lamina using a physics-based and data-informed free energy function. The model enables rapid hypothesis-driven prototyping of nuclear dynamics in 4D, thereby facilitating comparison with whole nucleus imaging experiments. As an application, we model the Drosophila nucleus spanning the interphase and map phase diagram of nuclear morphologies. We shed light on the interplay of adhesive and cohesive interactions within the nucleus, giving rise to distinct radial organization seen in conventional, inverted, and senescent nuclear architectures. The results also show the highly dynamic nature of the radial organization, the disruption of which leads to significant variability in domain coarsening dynamics and, consequently, variability of chromatin architecture. The model also highlights the impact of oblate nuclear geometry and heterochromatin sub-type interactions on the global chromatin architecture and local asymmetry of chromatin compartments.\n",
  "wordCount" : "2275",
  "inLanguage": "en",
  "datePublished": "2023-10-01T10:37:47Z",
  "dateModified": "2023-10-01T10:37:47Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on October 1, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559546">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559546" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559546">
        <p class="paperTitle">Bacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559546" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559546" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chambers, L. R.; Ye, Q.; Cai, J.; Gong, M.; Ledvina, H. E.; Zhou, H.; Whiteley, A. T.; Suhandynata, R. T.; Corbett, K. D.</p>
        <p class="info">Score: 47.7, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559546' target='https://doi.org/10.1101/2023.09.26.559546'> 10.1101/2023.09.26.559546</a></p>
        <p class="abstract">Ubiquitination and related pathways play crucial roles in protein homeostasis, signaling, and innate immunity1-3. In these pathways, an enzymatic cascade of E1, E2, and E3 proteins conjugates ubiquitin or a ubiquitin-like protein (Ubl) to targetprotein lysine residues4. Bacteria encode ancient relatives of E1 and Ubl proteins involved in sulfur metabolism5,6 but these proteins do not mediate Ubl-target conjugation, leaving open the question of whether bacteria can perform ubiquitination-like protein conjugation. Here, we demonstrate that a bacterial antiviral immune system encodes a complete ubiquitination pathway. Two structures of a bacterial E1:E2:Ubl complex reveal striking architectural parallels with canonical eukaryotic ubiquitination machinery. The bacterial E1 encodes an N-terminal inactive adenylation domain (IAD) and a C-terminal active adenylation domain (AAD) with a mobile -helical insertion containing the catalytic cysteine (CYS domain). One structure reveals a pre-reaction state with the bacterial Ubl C-terminus positioned for adenylation, and the E1 CYS domain poised nearby for thioester formation. A second structure mimics an E1-to-E2 transthioesterification state, with the E1 CYS domain rotated outward and its catalytic cysteine adjacent to the bound E2. We show that a deubiquitinase (DUB) in the same pathway pre-processes the bacterial Ubl, exposing its C-terminal glycine for adenylation. Finally, we show that the bacterial E1 and E2 collaborate to conjugate Ubl to target-protein lysine residues. Together, these data reveal that bacteria possess bona fide ubiquitination systems with strong mechanistic and architectural parallels to canonical eukaryotic ubiquitination pathways, suggesting that these pathways arose first in bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558479">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558479" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558479">
        <p class="paperTitle">Pattern formation by bacteria-phage interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558479" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558479" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Calvo, A.; Wingreen, N. S.; Datta, S. S.</p>
        <p class="info">Score: 38.1, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558479' target='https://doi.org/10.1101/2023.09.19.558479'> 10.1101/2023.09.19.558479</a></p>
        <p class="abstract">The interactions between bacteria and phages--viruses that infect bacteria--play critical roles in agriculture, ecology, and medicine; however, how these interactions influence the spatial organization of both bacteria and phages remain largely unexplored. Here, we address this gap in knowledge by developing a theoretical model of motile, proliferating bacteria that aggregate via motility-induced phase separation (MIPS) and encounter phage that infect and lyse the cells. We find that the non-reciprocal predator-prey interactions between phage and bacteria strongly alter spatial organization, in some cases giving rise to a rich array of finite-scale stationary and dynamic patterns in which bacteria and phage coexist. We establish principles describing the onset and characteristics of these diverse behaviors, thereby helping to provide a biophysical basis for understanding pattern formation in bacteria-phage systems, as well as in a broader range of active and living systems with similar predator-prey or other non-reciprocal interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559003">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559003" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559003">
        <p class="paperTitle">Marangoni-like tissue flows enhance symmetry breaking of embryonic organoids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559003" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559003" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gsell, S.; Tlili, S. L.; Merkel, M.; Lenne, P.-F.</p>
        <p class="info">Score: 19.2, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559003' target='https://doi.org/10.1101/2023.09.22.559003'> 10.1101/2023.09.22.559003</a></p>
        <p class="abstract">During early development of multi-cellular animals, cells self-organize to set up the body axes, such as the primary head-to-tail axis, based on which the later body plan is defined. Several signaling pathways are known to control body axis formation. Here, we show, however, that tissue mechanics plays an important role during this process. We focus on the emergence of a primary axis in initially spherical aggregates of mouse embryonic stem cells, which mirrors events in the early mouse embryo. These aggregates break rotational symmetry to establish an axial organization with domains of different expression profiles, e.g. of the transcription factor T/Bra and the adhesion molecule E-cadherin. Combining quantitative microscopy and physical modeling, we identify large-scale tissue flows with a recirculation component and demonstrate that they significantly contribute to symmetry breaking. We show that the recirculating flows are explained by a difference in tissue surface tension across domains, akin to Marangoni flows, which we further confirm by aggregate fusion experiments. Our work highlights that body axis formation is not only driven by biochemical processes, but that it can also be amplified by tissue flows. We expect this type of amplification to operate in many other organoid and in-vivo systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558980">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558980" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558980">
        <p class="paperTitle">Regeneration of actin filament branches from the same Arp2/3 complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558980" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558980" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghasemi, F.; Cao, L.; Mladenov, M.; Guichard, B.; Way, M.; Jegou, A.; Romet-Lemonne, G.</p>
        <p class="info">Score: 19.5, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558980' target='https://doi.org/10.1101/2023.09.22.558980'> 10.1101/2023.09.22.558980</a></p>
        <p class="abstract">Branched actin filaments are found in many key cellular structures. Branches are nucleated by the Arp2/3 complex activated by nucleation-promoting factor (NPF) proteins and bound to the side of pre-existing  mother filaments. Over time, branches dissociate from their mother filament, leading to network reorganization and turnover, but this mechanism is less understood. Here, using microfluidics and purified proteins, we examined the dissociation of individual branches under controlled biochemical and mechanical conditions. We observe that Arp2/3 remains bound to the mother filament after most debranching events, even when accelerated by force. Unexpectedly, this mother-remaining Arp2/3 readily nucleates a new actin filament branch, without being activated anew by an NPF: it simply needs to exchange its nucleotide and bind an actin monomer. The protein GMF, which accelerates debranching, prevents branch re-nucleation. Our results suggest that actin filament re-nucleation can provide a self-repair mechanism, helping branched networks to sustain mechanical stress in cells over extended periods of time.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558805">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558805" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558805">
        <p class="paperTitle">A model for collagen secretion by intercompartmental continuities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558805" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558805" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bunel, L.; Pincet, L.; Malhotra, V.; Raote, I.; Pincet, F.</p>
        <p class="info">Score: 16.0, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558805' target='https://doi.org/10.1101/2023.09.21.558805'> 10.1101/2023.09.21.558805</a></p>
        <p class="abstract">Newly synthesized secretory proteins are exported from endoplasmic reticulum (ER) at specialized subcompartments called exit sites (ERES). Cargoes like procollagen are too large for export by the standard COPII-coated vesicle of 60 nm average diameter. We have previously suggested that procollagen is transported from the ER to the next secretory organelle, the ERGIC, in TANGO1-dependent inter-organelle tunnels. Here, we show that intrinsically disordered domains of TANGO1 in the ER lumen generate an entropic contraction that pulls procollagen towards the ERES. Molecular gradients of pH and HSP47 between the ER and ERGIC generate a force in the range of tens of femtoNewtons (fN), which is sufficient to propel procollagen from the ER at a speed of [~]1 nm.s-1. This calculated speed and the quantities of collagen secreted are similar to its observed physiological secretion rate in fibroblasts, consistent with the proposal that ER export is the rate limiting step for procollagen secretion. Our theoretical model explains how cells can utilize molecular gradients to export procollagens at a rate commensurate with physiological needs.

Significance StatementProcollagen cannot be exported from the endoplasmic reticulum (ER) by standard COPII-coated vesicle of 60 nm average diameter. We have previously suggested that collagen is transported from the ER to the next secretory organelle, the ERGIC, in TANGO1-dependent inter-organelle tunnels. ER and ERGIC differ in molecular composition including their pH and protein composition. We propose a mechanical/entropic ratchet model whereby molecular gradients of pH and the collagen chaperone HSP47, provide the energy to propel procollagen from the ER at a speed that matches the physiological rate of collagen secretion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557355">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557355" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557355">
        <p class="paperTitle">Lipid peroxidation drives liquid-liquid phase separation and disrupts raft protein partitioning in biological membranes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557355" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557355" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Balakrishnan, M.; Kenworthy, A. K.</p>
        <p class="info">Score: 16.4, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557355' target='https://doi.org/10.1101/2023.09.12.557355'> 10.1101/2023.09.12.557355</a></p>
        <p class="abstract">The peroxidation of membrane lipids by free radicals contributes to aging, numerous diseases, and ferroptosis, an iron-dependent form of cell death. Peroxidation changes the structure, conformation and physicochemical properties of lipids, leading to major membrane alterations including bilayer thinning, altered fluidity, and increased permeability. Whether and how lipid peroxidation impacts the lateral organization of proteins and lipids in biological membranes, however, remains poorly understood. Here, we employ cell-derived giant plasma membrane vesicles (GPMVs) as a model to investigate the impact of lipid peroxidation on ordered membrane domains, often termed membrane rafts. We show that lipid peroxidation induced by the Fenton reaction dramatically enhances phase separation propensity of GPMVs into co-existing liquid ordered (raft) and liquid disordered (non-raft) domains and increases the relative abundance of the disordered, non-raft phase. Peroxidation also leads to preferential accumulation of peroxidized lipids and 4-hydroxynonenal (4-HNE) adducts in the disordered phase, decreased lipid packing in both raft and non-raft domains, and translocation of multiple classes of proteins out of rafts. These findings indicate that peroxidation of plasma membrane lipids disturbs many aspects of membrane rafts, including their stability, abundance, packing, and protein and lipid composition. We propose that these disruptions contribute to the pathological consequences of lipid peroxidation during aging and disease, and thus serve as potential targets for therapeutic intervention.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559471">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559471" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559471">
        <p class="paperTitle">Full-Length Single-Molecule Protein Fingerprinting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559471" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559471" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Filius, M.; van Wee, R.; de Lannoy, C.; Westerlaken, I.; Li, Z.; Kim, S. H.; de Agrela Pinto, C.; Wu, Y.; Boons, G.-J.; Pabst, M.; de Ridder, D.; Joo, C.</p>
        <p class="info">Score: 10.2, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559471' target='https://doi.org/10.1101/2023.09.26.559471'> 10.1101/2023.09.26.559471</a></p>
        <p class="abstract">Proteins are the primary functional actors of the cell. Hence, their identification is pivotal to advance our understanding of cell biology and disease. Current protein analysis methods are of limited use for distinguishing proteoforms. In particular, mass spectrometric methods often provide only ambiguous information on post-translational modification sites, and sequences of co-existing modifications may not be resolved. Here we demonstrate FRET-based single-molecule protein fingerprinting to map the location of individual amino acids and a post-translational modification within single full-length protein molecules. Using an approach that relies on transient binding of fluorescently labeled DNA strands to probe the amino acids on a protein one by one we show that we can fingerprint intrinsically disordered proteins as well as folded globular proteins with sub-nanometer resolution. We anticipate that this technology will be used for proteoform identification in biological and translational research with ultimate sensitivity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558608">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558608" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558608">
        <p class="paperTitle">Incorporating prior knowledge to seeds of adaptive sampling molecular dynamics simulations of ligand transport in enzymes with buried active sites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558608" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558608" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sarkar, D. K.; Surpeta, B.; Brezovsky, J.</p>
        <p class="info">Score: 8.1, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558608' target='https://doi.org/10.1101/2023.09.21.558608'> 10.1101/2023.09.21.558608</a></p>
        <p class="abstract">Given that most proteins have buried active sites, protein tunnels or channels play a crucial role in mitigating the transport of small molecules to the buried cavity for enzymatic catalysis. Tunnels can critically modulate the biological process of protein-ligand recognition. Various molecular dynamics methods have been developed for exploring and exploiting the protein-ligand conformational space to extract high-resolution details of the binding processes, one of the most recent represented by energetically unbiased high-throughput adaptive sampling simulations. The current study systematically contrasts the role of integrating prior knowledge while generating useful initial protein-ligand configurations, called seeds, for these simulations. Using a non-trivial system of haloalkane dehalogenase mutant with multiple transport tunnels leading to a deeply buried active site, these simulations were employed to derive kinetic models describing the process of association and dissociation of the substrate molecule. The more knowledge-based seed generation enabled high-throughput simulations that could more consistently capture the entire transport process, effectively explore the complex network of transport tunnels, and predict equilibrium dissociation constants, koff/kon, on the same order of magnitude as experimental measurements. Overall, the infusion of more knowledge into the initial seeds of adaptive sampling simulations could render analyses of transport mechanisms in enzymes more consistent even for very complex biomolecular systems, thereby promoting the rational design of enzymes with buried active sites and drug development efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.16.557871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.16.557871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.16.557871">
        <p class="paperTitle">An optogenetic method for the controlled release of single molecules</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.16.557871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.16.557871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kashyap, P.; Bertelli, S.; Cao, F.; Kostritskaia, Y.; Blank, F.; Srikanth, N.; Saleppico, R.; Bierhuizen, D.; Lu, X.; Nickel, W.; Campbell, R. E.; Plested, A.; Stauber, T.; Taylor, M. J.; Ewers, H.</p>
        <p class="info">Score: 25.8, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.16.557871' target='https://doi.org/10.1101/2023.09.16.557871'> 10.1101/2023.09.16.557871</a></p>
        <p class="abstract">We developed a system for optogenetic release of single molecules in live cells. We confined soluble and transmembrane proteins to the Golgi apparatus via a photocleavable protein and released them by short pulses of light. Our method allows for the controlled delivery of functional proteins to cytosol and plasma membrane in amounts compatible with single molecule imaging, greatly simplifying access to single molecule microscopy of any protein in live cells. Furthermore, we could reconstitute cellular functions such as ion conductance by delivering BK and VRAC ion channels to the plasma membrane. Finally, we could induce NF-kB signaling in T-Lymphoblasts stimulated by IL-1 by controlled release of a signaling protein that had been knocked-out in the same cells. We observed light induced formation of functional inflammatory signaling complexes that could trigger IKK phosphorylation in single cells. We thus developed an optogenetic method for the reconstitution and investigation of cellular function at the single molecule level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.21.558819">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.21.558819" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.21.558819">
        <p class="paperTitle">4D Mesoscale liquid model of nucleus resolves chromatin&#39;s radial organization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.21.558819" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.21.558819" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Laghmach, R.; Di Pierro, M.; Potoyan, D.</p>
        <p class="info">Score: 8.3, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.21.558819' target='https://doi.org/10.1101/2023.09.21.558819'> 10.1101/2023.09.21.558819</a></p>
        <p class="abstract">Recent chromatin capture, imaging techniques, and polymer modeling advancements have dramatically enhanced our quantitative understanding of chromosomal folding. However, the dynamism inherent in genome architectures due to physical and biochemical forces and their impact on nuclear architecture and cellular functions remains elusive. While imaging techniques capable of probing the physical properties of chromatin in 4D are growing, there is a conspicuous lack of physics-based computational tools appropriate for revealing the underlying forces that shape nuclear architecture and dynamics. To this end, we have developed a multi-phase liquid model of the nucleus, which can resolve chromosomal territories, compartments, and nuclear lamina using a physics-based and data-informed free energy function. The model enables rapid hypothesis-driven prototyping of nuclear dynamics in 4D, thereby facilitating comparison with whole nucleus imaging experiments. As an application, we model the Drosophila nucleus spanning the interphase and map phase diagram of nuclear morphologies. We shed light on the interplay of adhesive and cohesive interactions within the nucleus, giving rise to distinct radial organization seen in conventional, inverted, and senescent nuclear architectures. The results also show the highly dynamic nature of the radial organization, the disruption of which leads to significant variability in domain coarsening dynamics and, consequently, variability of chromatin architecture. The model also highlights the impact of oblate nuclear geometry and heterochromatin sub-type interactions on the global chromatin architecture and local asymmetry of chromatin compartments.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
