<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Approximating conformational Boltzmann distributions with AlphaFold2 predictions
Authors: Brown, B. P.; Stein, R. A.; Meiler, J.; Mchaourab, H.
Score: 24.6, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552168
Protein dynamics are intimately tied to biological function and can enable processes such as signal transduction, enzyme catalysis, and molecular recognition. The relative free energies of conformations that contribute to these functional equilibria are evolved for the physiology of the organism. Despite the importance of these equilibria for understanding biological function and developing treatments for disease, the computational and experimental methods capable of quantifying them are limited to systems of modest size.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Approximating conformational Boltzmann distributions with AlphaFold2 predictions
Authors: Brown, B. P.; Stein, R. A.; Meiler, J.; Mchaourab, H.
Score: 24.6, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552168
Protein dynamics are intimately tied to biological function and can enable processes such as signal transduction, enzyme catalysis, and molecular recognition. The relative free energies of conformations that contribute to these functional equilibria are evolved for the physiology of the organism. Despite the importance of these equilibria for understanding biological function and developing treatments for disease, the computational and experimental methods capable of quantifying them are limited to systems of modest size." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Approximating conformational Boltzmann distributions with AlphaFold2 predictions
Authors: Brown, B. P.; Stein, R. A.; Meiler, J.; Mchaourab, H.
Score: 24.6, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552168
Protein dynamics are intimately tied to biological function and can enable processes such as signal transduction, enzyme catalysis, and molecular recognition. The relative free energies of conformations that contribute to these functional equilibria are evolved for the physiology of the organism. Despite the importance of these equilibria for understanding biological function and developing treatments for disease, the computational and experimental methods capable of quantifying them are limited to systems of modest size."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Approximating conformational Boltzmann distributions with AlphaFold2 predictions\nAuthors: Brown, B. P.; Stein, R. A.; Meiler, J.; Mchaourab, H.\nScore: 24.6, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552168\nProtein dynamics are intimately tied to biological function and can enable processes such as signal transduction, enzyme catalysis, and molecular recognition. The relative free energies of conformations that contribute to these functional equilibria are evolved for the physiology of the organism. Despite the importance of these equilibria for understanding biological function and developing treatments for disease, the computational and experimental methods capable of quantifying them are limited to systems of modest size.",
  "keywords": [
    
  ],
  "articleBody": " Approximating conformational Boltzmann distributions with AlphaFold2 predictions\nAuthors: Brown, B. P.; Stein, R. A.; Meiler, J.; Mchaourab, H.\nScore: 24.6, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552168\nProtein dynamics are intimately tied to biological function and can enable processes such as signal transduction, enzyme catalysis, and molecular recognition. The relative free energies of conformations that contribute to these functional equilibria are evolved for the physiology of the organism. Despite the importance of these equilibria for understanding biological function and developing treatments for disease, the computational and experimental methods capable of quantifying them are limited to systems of modest size. Here, we demonstrate that AlphaFold2 contact distance distributions can approximate conformational Boltzmann distributions, which we evaluate through examination of the joint probability distributions of inter-residue contact distances along functionally relevant collective variables of several protein systems. Further, we show that contact distance probability distributions generated by AlphaFold2 are sensitive to points mutations thus AF2 can predict the structural effects of mutations in some systems. We anticipate that our approach will be a valuable tool to model the thermodynamics of conformational changes in large biomolecular systems.\nConserved molecular recognition by an intrinsically disordered region in the absence of sequence conservation\nAuthors: Alston, J.; Soranno, A.; Holehouse, A. S.\nScore: 17.8, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552128\nIntrinsically disordered regions (IDRs) are critical for cellular function, yet often appear to lack sequence conservation when assessed by multiple sequence alignments. This raises the question of if and how function can be encoded and preserved in these regions despite massive sequence variation. To address this question, we have applied coarse-grained molecular dynamics simulations to investigate non-specific RNA binding of coronavirus nucleocapsid proteins. Coronavirus nucleocapsid proteins consist of multiple interspersed disordered and folded domains that bind RNA. We focussed here on the first two domains of coronavirus nucleocapsid proteins, the disordered N-terminal domain (NTD) followed by the folded RNA binding domain (RBD). While the NTD is highly variable across evolution, the RBD is structurally conserved. This combination makes the NTD-RBD a convenient model system to explore the interplay between an IDR adjacent to a folded domain, and how changes in IDR sequence can influence molecular recognition of a partner. Our results reveal a surprising degree of sequence-specificity encoded by both the composition and the precise order of the amino acids in the NTD. The presence of an NTD can - depending on the sequence - either suppress or enhance RNA binding. Despite this sensitivity, large-scale variation in NTD sequences is possible while certain sequence features are retained. Consequently, a conformationally-conserved fuzzy RNA:protein complex is found across nucleocapsid protein orthologs, despite large-scale changes in both NTD sequence and RBD surface chemistry. Taken together, these insights shed light on the ability of disordered regions to preserve functional characteristics despite their sequence variability.\nConstitutive activity of ionotropic glutamate receptors via a hydrophobic plug in the ligand-binding domain\nAuthors: Seljeset, S.; Sintsova, O.; Wang, Y.; Harb, H. Y.; Lynagh, T.\nScore: 17.8, Published: 2023-08-06 DOI: 10.1101/2023.08.03.551817\nNeurotransmitter ligands electrically excite neurons by activating ionotropic glutamate receptor (iGluR) ion channels. Knowledge of the iGluR amino acid residues that dominate ligand-induced activation would enable the prediction of function from sequence. We therefore explored determinants of activity in rat NMDA-type iGluRs (NMDA receptors), complex heteromeric iGluRs comprising two glycine-binding GluN1 and two glutamate-binding GluN2 subunits, using amino acid sequence analysis, mutagenesis, and electrophysiology. We find that a broadly conserved aspartate residue controls both ligand potency and channel activity, to the extent that certain substitutions at this position bypass the need for ligand-binding in GluN1 subunits, generating NMDA receptors activated solely by glutamate. Furthermore, we identify a homomeric iGluR from the placozoan Trichoplax adhaerens that has utilized this crucial residue to evolve into a leak channel that is inhibited by neurotransmitter binding, pointing to a dominant role of this residue throughout the iGluR superfamily.\nMesoscopic Oblique Plane Microscopy via Light-sheet Mirroring\nAuthors: Daetwyler, S.; Chang, B.-J.; Chen, B.; Zhou, F. Y.; Fiolka, R. P.\nScore: 14.1, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552834\nUnderstanding the intricate interplay and inter-connectivity of biological processes across an entire organism is important in various fields of biology, including cardiovascular research, neuroscience, and developmental biology. Here, we present a mesoscopic oblique plane microscope (OPM) that enables whole organism imaging with high speed and subcellular resolution. A microprism underneath the sample enhances the axial resolution and optical sectioning through total internal reflection of the light-sheet. Through rapid refocusing of the light-sheet, the imaging depth is extended up to threefold while keeping the axial resolution constant. Using low magnification objectives with a large field of view, we realize mesoscopic imaging over a volume of 3.7x1.5x1 mm3 with ~2.3 microns lateral and ~9.2 microns axial resolution. Applying the mesoscopic OPM, we demonstrate in vivo and in toto whole organism imaging of the zebrafish vasculature and its endothelial nuclei, and blood flow dynamics at 12 Hz acquisition rate, resulting in a quantitative map of blood flow across the entire organism.\nGraph Attention Site Prediction (GrASP): Identifying Druggable Binding Sites Using Graph Neural Networks with Attention\nAuthors: Smith, Z.; Strobel, M.; Vani, B.; Tiwary, P.\nScore: 44.6, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550565\nIdentifying and discovering druggable protein binding sites is an important early step in computer-aided drug discovery but remains a difficult task where most campaigns rely on a priori knowledge of binding sites from experiments. Here we present a novel binding site prediction method called Graph Attention Site Prediction (GrASP) and re-evaluate assumptions in nearly every step in the site prediction workflow from dataset preparation to model evaluation. GrASP is able to achieve state-of-the-art performance at recovering binding sites in PDB structures while maintaining a high degree of precision which will minimize wasted computation in downstream tasks such as docking and free energy perturbation.\nReliable protein-protein docking with AlphaFold, Rosetta and replica-exchange\nAuthors: Harmalkar, A.; Lyskov, S.; Gray, J. J.\nScore: 71.8, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551063\nDespite the recent breakthrough of AlphaFold (AF) in the field of protein sequence-to-structure prediction, modeling protein interfaces and predicting protein complex structures remains challenging, especially when there is a significant conformational change in one or both binding partners. Prior studies have demonstrated that AF-multimer (AFm) can predict accurate protein complexes in only up to 43% of cases. 1 In this work, we combine AlphaFold as a structural template generator with a physics-based replica exchange docking algorithm. Using a curated collection of 254 available protein targets with both unbound and bound structures, we first demonstrate that AlphaFold confidence measures can be repurposed for estimating protein flexibility and docking accuracy for multimers. We incorporate these metrics within our ReplicaDock 2.0 protocol 2 to complete a robust in-silico pipeline for accurate protein complex structure prediction. AlphaRED (AlphaFold-initiated Replica Exchange Docking) successfully docks failed AF predictions including 97 failure cases in Docking Benchmark Set 5.5. AlphaRED generates CAPRI acceptable-quality or better predictions for 66% of benchmark targets. Further, on a subset of antigen-antibody targets, which is challenging for AFm (19% success rate), AlphaRED demonstrates a success rate of 51%. This new strategy demonstrates the success possible by integrating deep-learning based architectures trained on evolutionary information with physics-based enhanced sampling. The pipeline is available at github.com/Graylab/AlphaRED.\nCell Membranes Sustain Phospholipid Imbalance Via Cholesterol Asymmetry\nAuthors: Doktorova, M.; Symons, J. L.; Zhang, X.; Wang, H.-Y.; Schlegel, J.; Lorent, J. H.; Heberle, F. A.; Sezgin, E.; Lyman, E.; Levental, K. R.; Levental, I.\nScore: 115.8, Published: 2023-07-31 DOI: 10.1101/2023.07.30.551157\nMembranes are molecular interfaces that insulate cells from external stresses, compartmentalize the cytoplasm, and control the flow of nutrients and information1. These functions are facilitated by diverse collections of lipids, nearly all of which are distributed asymmetrically between the two leaflets of living bilayers2,3. Previous models of biomembrane structure and function have rested upon the implicit assumption that the two membrane leaflets have similar abundances of phospholipids. Here, we show that this assumption is generally invalid and investigate the consequences of lipid abundance imbalances in mammalian plasma membranes (PM). Using quantitative lipidomics, we discovered that cytoplasmic leaflets of human erythrocyte PMs have \u003e50% overabundance of phospholipids compared to exoplasmic leaflets. We show that this phospholipid imbalance is enabled by an asymmetric interleaflet distribution of cholesterol4,5, which rapidly redistributes to buffer leaflet stresses. Asymmetric phospholipid abundance and composition combine to enrich cholesterol in the exoplasmic PM leaflet. Through a combination of experimental and computational approaches we demonstrate how these lipid distributions impart unique functional characteristics to PMs, including low permeability, surprisingly fast cholesterol diffusion6, and resting tension in the cytoplasmic monolayer that regulates protein localization. Our observations of these previously overlooked aspects of membrane asymmetry represent an evolution of classic paradigms1,7 of biomembrane structure and physiology.\nDifferential conformational dynamics in two type-A RNA-binding domains drive the double-stranded RNA recognition and binding\nAuthors: Parvez, F.; Sangpal, D.; Paithankar, H.; Amin, Z.; Chugh, J.\nScore: 5.2, Published: 2023-08-06 DOI: 10.1101/2023.08.06.552166\nTAR RNA binding protein (TRBP) has emerged as a key player in the RNA interference (RNAi) pathway. While facilitating Dicer-mediated pre-miRNA cleavage and recruitment of the RISC complex, TRBP binds to different pre-miRNAs \u0026 siRNAs, each varying in sequence, structure, or both. Our hypothesis posits that TRBP displays dynamic adaptability to accommodate this heterogeneity in target RNA structures. Thus, it is crucial to ascertain the role of intrinsic and RNA-induced protein dynamics in RNA recognition and binding. Prior investigations from the lab have elucidated the role of intrinsic and RNA- induced conformational exchange in the double-stranded RNA-binding domain 1 (dsRBD1) of TRBP in shape-dependent RNA recognition. Building further on this, the current study delves into the intrinsic and RNA-induced conformational dynamics of the dsRBD2 of TRBP, juxtaposed with dsRBD1. Remarkably, dsRBD2 exhibits a higher binding affinity to a short (12bp) dsRNA when compared with dsRBD1, which can be attributed to the presence of residues critical to RNA binding and structural plasticity in dsRBD2. Utilizing state-of- the-art NMR spin relaxation, relaxation dispersion experiments, and MD simulations, we report that dsRBD2 depicts constrained conformational plasticity when compared to dsRBD1, as evidenced by the fast (ps-ns) and slower (s-ms) conformational dynamics measurements. Although, in the presence of RNA, dsRBD2 undergoes induced conformational exchange within the designated RNA-binding regions and the other residues, the amplitude of the motions remains modest when compared to those observed in dsRBD1. The culmination of our findings leads us to propose a dynamics-driven model of the two tandem domains of TRBP, substantiating their contributions to the versatility of dsRNA recognition and binding.\nFRET-guided modeling of nucleic acids\nAuthors: Steffen, F. D.; Cunha, R. A.; Sigel, R. K. O.; Borner, R.\nScore: 4.5, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552238\nThe functional diversity of RNA is encoded in their innate conformational heterogeneity. The combination of single-molecule spectroscopy and computational modeling offers new, attractive opportunities to map structural transitions within nucleic acid ensembles. Here, we describe a framework to harmonize single-molecule FRET measurements with molecular dynamics simulations and de novo structure prediction. Using either all-atom or implicit fluorophore modeling we recreate FRET experiments in silico, visualize the underlying structural dynamics and quantify the simulated reaction coordinates. Using multiple accessible-contact volumes (multi-ACV) as a post-hoc scoring method for fragment-assembly in Rosetta, we demonstrate that FRET effectively refines de novo RNA structure prediction without the need of explicit dye labeling. We benchmark our FRET-assisted modeling approach on double-labeled DNA strands and validate it against an intrinsically dynamic manganese(II)-binding riboswitch. We show that a FRET coordinate describing the assembly of a four-way junction allows our pipeline to recapitulate the global fold of the riboswitch with sub-helical accuracy to the crystal structure. We conclude that computational fluorescence spectroscopy facilitates the interpretability of dynamic structural ensembles and improves the mechanistic understanding of nucleic acid interactions. Graphical abstractSchematic workflow of integrative FRET modeling using all-atom fluorophores or an accessible-contact volume dye model. All-atom molecular dynamics track the dye coordinate explicitly as part of the simulation while multi-ACV infer mean dye positions post hoc. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=150 SRC=\"FIGDIR/small/552238v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (81K): org.highwire.dtl.DTLVardef@57c9org.highwire.dtl.DTLVardef@837f63org.highwire.dtl.DTLVardef@1203ee5org.highwire.dtl.DTLVardef@1580c_HPS_FORMAT_FIGEXP M_FIG C_FIG\nMechanism of Phosphate Release from Actin Filaments\nAuthors: Wang, Y.; Zsolnay, V.; Pollard, T. D.; Voth, G. A.\nScore: 4.3, Published: 2023-08-05 DOI: 10.1101/2023.08.03.551904\nAfter ATP-actin monomers assemble filaments, the {gamma}-phosphate is hydrolyzed from ATP within seconds and dissociates from the filament over several minutes. We used all-atom 2-dimensional well-tempered metadynamics computer simulations to sample the release of phosphate from filaments with unbiased molecular dynamics simulations to study residues that gate release. Dissociation of phosphate from Mg2+ is rate limiting and associated with an energy barrier of 20 kcal/mol, consistent with experimental rates of phosphate release. Phosphate then diffuses in an internal cavity toward a gate formed by R177 recognized in prior computational studies and cryo-EM structures. The gate is closed when R177 hydrogen bonds with N111 and is open when R177 forms a salt bridge with D179. Most of the time interactions of R177 with other residues occludes the phosphate release pathway. Machine learning analysis reveals that the occluding interactions fluctuate rapidly. These occluded states have not yet been documented in cryo-EM reconstructions. (150 words) Significance StatementThe protein actin assembles into filaments that participate in muscle contraction and other cellular movements. An ATP bound to the actin monomer is hydrolyzed rapidly during filament assembly, but the {gamma}-phosphate dissociates slowly from the filament. We used molecular dynamics simulations to characterize the slow release process. Separation of phosphate from Mg2+ in the active site is rate limiting owing to a large free energy barrier 20 kcal/mol. Release of phosphate from the protein requires opening a gate in the actin molecule formed by interaction of the sidechains of arginine 177 and asparagine 111. Surprisingly, the simulations revealed other interactions of the sidechain of arginine 177 that occlude the release pathway most of the time but have not been observed in cryo-EM structures. (124 words)\n",
  "wordCount" : "2282",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552168">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552168" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552168">
        <p class="paperTitle">Approximating conformational Boltzmann distributions with AlphaFold2 predictions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552168" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552168" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brown, B. P.; Stein, R. A.; Meiler, J.; Mchaourab, H.</p>
        <p class="info">Score: 24.6, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552168' target='https://doi.org/10.1101/2023.08.06.552168'> 10.1101/2023.08.06.552168</a></p>
        <p class="abstract">Protein dynamics are intimately tied to biological function and can enable processes such as signal transduction, enzyme catalysis, and molecular recognition. The relative free energies of conformations that contribute to these functional equilibria are evolved for the physiology of the organism. Despite the importance of these equilibria for understanding biological function and developing treatments for disease, the computational and experimental methods capable of quantifying them are limited to systems of modest size. Here, we demonstrate that AlphaFold2 contact distance distributions can approximate conformational Boltzmann distributions, which we evaluate through examination of the joint probability distributions of inter-residue contact distances along functionally relevant collective variables of several protein systems. Further, we show that contact distance probability distributions generated by AlphaFold2 are sensitive to points mutations thus AF2 can predict the structural effects of mutations in some systems. We anticipate that our approach will be a valuable tool to model the thermodynamics of conformational changes in large biomolecular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552128">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552128" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552128">
        <p class="paperTitle">Conserved molecular recognition by an intrinsically disordered region in the absence of sequence conservation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552128" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552128" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alston, J.; Soranno, A.; Holehouse, A. S.</p>
        <p class="info">Score: 17.8, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552128' target='https://doi.org/10.1101/2023.08.06.552128'> 10.1101/2023.08.06.552128</a></p>
        <p class="abstract">Intrinsically disordered regions (IDRs) are critical for cellular function, yet often appear to lack sequence conservation when assessed by multiple sequence alignments. This raises the question of if and how function can be encoded and preserved in these regions despite massive sequence variation. To address this question, we have applied coarse-grained molecular dynamics simulations to investigate non-specific RNA binding of coronavirus nucleocapsid proteins. Coronavirus nucleocapsid proteins consist of multiple interspersed disordered and folded domains that bind RNA. We focussed here on the first two domains of coronavirus nucleocapsid proteins, the disordered N-terminal domain (NTD) followed by the folded RNA binding domain (RBD). While the NTD is highly variable across evolution, the RBD is structurally conserved. This combination makes the NTD-RBD a convenient model system to explore the interplay between an IDR adjacent to a folded domain, and how changes in IDR sequence can influence molecular recognition of a partner. Our results reveal a surprising degree of sequence-specificity encoded by both the composition and the precise order of the amino acids in the NTD. The presence of an NTD can - depending on the sequence - either suppress or enhance RNA binding. Despite this sensitivity, large-scale variation in NTD sequences is possible while certain sequence features are retained. Consequently, a conformationally-conserved fuzzy RNA:protein complex is found across nucleocapsid protein orthologs, despite large-scale changes in both NTD sequence and RBD surface chemistry. Taken together, these insights shed light on the ability of disordered regions to preserve functional characteristics despite their sequence variability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551817">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551817" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551817">
        <p class="paperTitle">Constitutive activity of ionotropic glutamate receptors via a hydrophobic plug in the ligand-binding domain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551817" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551817" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seljeset, S.; Sintsova, O.; Wang, Y.; Harb, H. Y.; Lynagh, T.</p>
        <p class="info">Score: 17.8, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551817' target='https://doi.org/10.1101/2023.08.03.551817'> 10.1101/2023.08.03.551817</a></p>
        <p class="abstract">Neurotransmitter ligands electrically excite neurons by activating ionotropic glutamate receptor (iGluR) ion channels. Knowledge of the iGluR amino acid residues that dominate ligand-induced activation would enable the prediction of function from sequence. We therefore explored determinants of activity in rat NMDA-type iGluRs (NMDA receptors), complex heteromeric iGluRs comprising two glycine-binding GluN1 and two glutamate-binding GluN2 subunits, using amino acid sequence analysis, mutagenesis, and electrophysiology. We find that a broadly conserved aspartate residue controls both ligand potency and channel activity, to the extent that certain substitutions at this position bypass the need for ligand-binding in GluN1 subunits, generating NMDA receptors activated solely by glutamate. Furthermore, we identify a homomeric iGluR from the placozoan Trichoplax adhaerens that has utilized this crucial residue to evolve into a leak channel that is inhibited by neurotransmitter binding, pointing to a dominant role of this residue throughout the iGluR superfamily.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552834">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552834" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552834">
        <p class="paperTitle">Mesoscopic Oblique Plane Microscopy via Light-sheet Mirroring</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552834" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552834" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daetwyler, S.; Chang, B.-J.; Chen, B.; Zhou, F. Y.; Fiolka, R. P.</p>
        <p class="info">Score: 14.1, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552834' target='https://doi.org/10.1101/2023.08.10.552834'> 10.1101/2023.08.10.552834</a></p>
        <p class="abstract">Understanding the intricate interplay and inter-connectivity of biological processes across an entire organism is important in various fields of biology, including cardiovascular research, neuroscience, and developmental biology. Here, we present a mesoscopic oblique plane microscope (OPM) that enables whole organism imaging with high speed and subcellular resolution. A microprism underneath the sample enhances the axial resolution and optical sectioning through total internal reflection of the light-sheet. Through rapid refocusing of the light-sheet, the imaging depth is extended up to threefold while keeping the axial resolution constant. Using low magnification objectives with a large field of view, we realize mesoscopic imaging over a volume of 3.7x1.5x1 mm3 with ~2.3 microns lateral and ~9.2 microns axial resolution. Applying the mesoscopic OPM, we demonstrate in vivo and in toto whole organism imaging of the zebrafish vasculature and its endothelial nuclei, and blood flow dynamics at 12 Hz acquisition rate, resulting in a quantitative map of blood flow across the entire organism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550565">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550565" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550565">
        <p class="paperTitle">Graph Attention Site Prediction (GrASP): Identifying Druggable Binding Sites Using Graph Neural Networks with Attention</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550565" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550565" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, Z.; Strobel, M.; Vani, B.; Tiwary, P.</p>
        <p class="info">Score: 44.6, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550565' target='https://doi.org/10.1101/2023.07.25.550565'> 10.1101/2023.07.25.550565</a></p>
        <p class="abstract">Identifying and discovering druggable protein binding sites is an important early step in computer-aided drug discovery but remains a difficult task where most campaigns rely on a priori knowledge of binding sites from experiments. Here we present a novel binding site prediction method called Graph Attention Site Prediction (GrASP) and re-evaluate assumptions in nearly every step in the site prediction workflow from dataset preparation to model evaluation. GrASP is able to achieve state-of-the-art performance at recovering binding sites in PDB structures while maintaining a high degree of precision which will minimize wasted computation in downstream tasks such as docking and free energy perturbation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.551063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.551063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.551063">
        <p class="paperTitle">Reliable protein-protein docking with AlphaFold, Rosetta and replica-exchange</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.551063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.551063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harmalkar, A.; Lyskov, S.; Gray, J. J.</p>
        <p class="info">Score: 71.8, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.551063' target='https://doi.org/10.1101/2023.07.28.551063'> 10.1101/2023.07.28.551063</a></p>
        <p class="abstract">Despite the recent breakthrough of AlphaFold (AF) in the field of protein sequence-to-structure prediction, modeling protein interfaces and predicting protein complex structures remains challenging, especially when there is a significant conformational change in one or both binding partners. Prior studies have demonstrated that AF-multimer (AFm) can predict accurate protein complexes in only up to 43% of cases. 1 In this work, we combine AlphaFold as a structural template generator with a physics-based replica exchange docking algorithm. Using a curated collection of 254 available protein targets with both unbound and bound structures, we first demonstrate that AlphaFold confidence measures can be repurposed for estimating protein flexibility and docking accuracy for multimers. We incorporate these metrics within our ReplicaDock 2.0 protocol 2 to complete a robust in-silico pipeline for accurate protein complex structure prediction. AlphaRED (AlphaFold-initiated Replica Exchange Docking) successfully docks failed AF predictions including 97 failure cases in Docking Benchmark Set 5.5. AlphaRED generates CAPRI acceptable-quality or better predictions for 66% of benchmark targets. Further, on a subset of antigen-antibody targets, which is challenging for AFm (19% success rate), AlphaRED demonstrates a success rate of 51%. This new strategy demonstrates the success possible by integrating deep-learning based architectures trained on evolutionary information with physics-based enhanced sampling. The pipeline is available at github.com/Graylab/AlphaRED.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.30.551157">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.30.551157" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.30.551157">
        <p class="paperTitle">Cell Membranes Sustain Phospholipid Imbalance Via Cholesterol Asymmetry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.30.551157" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.30.551157" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Doktorova, M.; Symons, J. L.; Zhang, X.; Wang, H.-Y.; Schlegel, J.; Lorent, J. H.; Heberle, F. A.; Sezgin, E.; Lyman, E.; Levental, K. R.; Levental, I.</p>
        <p class="info">Score: 115.8, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.30.551157' target='https://doi.org/10.1101/2023.07.30.551157'> 10.1101/2023.07.30.551157</a></p>
        <p class="abstract">Membranes are molecular interfaces that insulate cells from external stresses, compartmentalize the cytoplasm, and control the flow of nutrients and information1. These functions are facilitated by diverse collections of lipids, nearly all of which are distributed asymmetrically between the two leaflets of living bilayers2,3. Previous models of biomembrane structure and function have rested upon the implicit assumption that the two membrane leaflets have similar abundances of phospholipids. Here, we show that this assumption is generally invalid and investigate the consequences of lipid abundance imbalances in mammalian plasma membranes (PM). Using quantitative lipidomics, we discovered that cytoplasmic leaflets of human erythrocyte PMs have &gt;50% overabundance of phospholipids compared to exoplasmic leaflets. We show that this phospholipid imbalance is enabled by an asymmetric interleaflet distribution of cholesterol4,5, which rapidly redistributes to buffer leaflet stresses. Asymmetric phospholipid abundance and composition combine to enrich cholesterol in the exoplasmic PM leaflet. Through a combination of experimental and computational approaches we demonstrate how these lipid distributions impart unique functional characteristics to PMs, including low permeability, surprisingly fast cholesterol diffusion6, and resting tension in the cytoplasmic monolayer that regulates protein localization. Our observations of these previously overlooked aspects of membrane asymmetry represent an evolution of classic paradigms1,7 of biomembrane structure and physiology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552166">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552166" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552166">
        <p class="paperTitle">Differential conformational dynamics in two type-A RNA-binding domains drive the double-stranded RNA recognition and binding</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552166" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552166" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parvez, F.; Sangpal, D.; Paithankar, H.; Amin, Z.; Chugh, J.</p>
        <p class="info">Score: 5.2, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552166' target='https://doi.org/10.1101/2023.08.06.552166'> 10.1101/2023.08.06.552166</a></p>
        <p class="abstract">TAR RNA binding protein (TRBP) has emerged as a key player in the RNA interference (RNAi) pathway. While facilitating Dicer-mediated pre-miRNA cleavage and recruitment of the RISC complex, TRBP binds to different pre-miRNAs &amp; siRNAs, each varying in sequence, structure, or both. Our hypothesis posits that TRBP displays dynamic adaptability to accommodate this heterogeneity in target RNA structures. Thus, it is crucial to ascertain the role of intrinsic and RNA-induced protein dynamics in RNA recognition and binding. Prior investigations from the lab have elucidated the role of intrinsic and RNA- induced conformational exchange in the double-stranded RNA-binding domain 1 (dsRBD1) of TRBP in shape-dependent RNA recognition. Building further on this, the current study delves into the intrinsic and RNA-induced conformational dynamics of the dsRBD2 of TRBP, juxtaposed with dsRBD1. Remarkably, dsRBD2 exhibits a higher binding affinity to a short (12bp) dsRNA when compared with dsRBD1, which can be attributed to the presence of residues critical to RNA binding and structural plasticity in dsRBD2. Utilizing state-of- the-art NMR spin relaxation, relaxation dispersion experiments, and MD simulations, we report that dsRBD2 depicts constrained conformational plasticity when compared to dsRBD1, as evidenced by the fast (ps-ns) and slower (s-ms) conformational dynamics measurements. Although, in the presence of RNA, dsRBD2 undergoes induced conformational exchange within the designated RNA-binding regions and the other residues, the amplitude of the motions remains modest when compared to those observed in dsRBD1. The culmination of our findings leads us to propose a dynamics-driven model of the two tandem domains of TRBP, substantiating their contributions to the versatility of dsRNA recognition and binding.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552238">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552238" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552238">
        <p class="paperTitle">FRET-guided modeling of nucleic acids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552238" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552238" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Steffen, F. D.; Cunha, R. A.; Sigel, R. K. O.; Borner, R.</p>
        <p class="info">Score: 4.5, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552238' target='https://doi.org/10.1101/2023.08.07.552238'> 10.1101/2023.08.07.552238</a></p>
        <p class="abstract">The functional diversity of RNA is encoded in their innate conformational heterogeneity. The combination of single-molecule spectroscopy and computational modeling offers new, attractive opportunities to map structural transitions within nucleic acid ensembles. Here, we describe a framework to harmonize single-molecule FRET measurements with molecular dynamics simulations and de novo structure prediction. Using either all-atom or implicit fluorophore modeling we recreate FRET experiments in silico, visualize the underlying structural dynamics and quantify the simulated reaction coordinates. Using multiple accessible-contact volumes (multi-ACV) as a post-hoc scoring method for fragment-assembly in Rosetta, we demonstrate that FRET effectively refines de novo RNA structure prediction without the need of explicit dye labeling. We benchmark our FRET-assisted modeling approach on double-labeled DNA strands and validate it against an intrinsically dynamic manganese(II)-binding riboswitch. We show that a FRET coordinate describing the assembly of a four-way junction allows our pipeline to recapitulate the global fold of the riboswitch with sub-helical accuracy to the crystal structure. We conclude that computational fluorescence spectroscopy facilitates the interpretability of dynamic structural ensembles and improves the mechanistic understanding of nucleic acid interactions.

Graphical abstractSchematic workflow of integrative FRET modeling using all-atom fluorophores or an accessible-contact volume dye model. All-atom molecular dynamics track the dye coordinate explicitly as part of the simulation while multi-ACV infer mean dye positions post hoc.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=150 SRC=&#34;FIGDIR/small/552238v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (81K):
org.highwire.dtl.DTLVardef@57c9org.highwire.dtl.DTLVardef@837f63org.highwire.dtl.DTLVardef@1203ee5org.highwire.dtl.DTLVardef@1580c_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551904">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551904" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551904">
        <p class="paperTitle">Mechanism of Phosphate Release from Actin Filaments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551904" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551904" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Zsolnay, V.; Pollard, T. D.; Voth, G. A.</p>
        <p class="info">Score: 4.3, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551904' target='https://doi.org/10.1101/2023.08.03.551904'> 10.1101/2023.08.03.551904</a></p>
        <p class="abstract">After ATP-actin monomers assemble filaments, the {gamma}-phosphate is hydrolyzed from ATP within seconds and dissociates from the filament over several minutes. We used all-atom 2-dimensional well-tempered metadynamics computer simulations to sample the release of phosphate from filaments with unbiased molecular dynamics simulations to study residues that gate release. Dissociation of phosphate from Mg2&#43; is rate limiting and associated with an energy barrier of 20 kcal/mol, consistent with experimental rates of phosphate release. Phosphate then diffuses in an internal cavity toward a gate formed by R177 recognized in prior computational studies and cryo-EM structures. The gate is closed when R177 hydrogen bonds with N111 and is open when R177 forms a salt bridge with D179. Most of the time interactions of R177 with other residues occludes the phosphate release pathway. Machine learning analysis reveals that the occluding interactions fluctuate rapidly. These occluded states have not yet been documented in cryo-EM reconstructions. (150 words)

Significance StatementThe protein actin assembles into filaments that participate in muscle contraction and other cellular movements. An ATP bound to the actin monomer is hydrolyzed rapidly during filament assembly, but the {gamma}-phosphate dissociates slowly from the filament. We used molecular dynamics simulations to characterize the slow release process. Separation of phosphate from Mg2&#43; in the active site is rate limiting owing to a large free energy barrier 20 kcal/mol. Release of phosphate from the protein requires opening a gate in the actin molecule formed by interaction of the sidechains of arginine 177 and asparagine 111. Surprisingly, the simulations revealed other interactions of the sidechain of arginine 177 that occlude the release pathway most of the time but have not been observed in cryo-EM structures. (124 words)</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
