<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The genetic architecture of protein stability
Authors: Faure, A. J.; Marti-Aranda, A.; Hidalgo, C.; Schmiedel, J. M.; Lehner, B.
Score: 50.8, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564339
There are more ways to synthesize a 100 amino acid protein (20100) than atoms in the universe. Only a miniscule fraction of such a vast sequence space can ever be experimentally or computationally surveyed. Deep neural networks are increasingly being used to navigate high-dimensional sequence spaces.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="The genetic architecture of protein stability
Authors: Faure, A. J.; Marti-Aranda, A.; Hidalgo, C.; Schmiedel, J. M.; Lehner, B.
Score: 50.8, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564339
There are more ways to synthesize a 100 amino acid protein (20100) than atoms in the universe. Only a miniscule fraction of such a vast sequence space can ever be experimentally or computationally surveyed. Deep neural networks are increasingly being used to navigate high-dimensional sequence spaces." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="The genetic architecture of protein stability
Authors: Faure, A. J.; Marti-Aranda, A.; Hidalgo, C.; Schmiedel, J. M.; Lehner, B.
Score: 50.8, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564339
There are more ways to synthesize a 100 amino acid protein (20100) than atoms in the universe. Only a miniscule fraction of such a vast sequence space can ever be experimentally or computationally surveyed. Deep neural networks are increasingly being used to navigate high-dimensional sequence spaces."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "The genetic architecture of protein stability\nAuthors: Faure, A. J.; Marti-Aranda, A.; Hidalgo, C.; Schmiedel, J. M.; Lehner, B.\nScore: 50.8, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564339\nThere are more ways to synthesize a 100 amino acid protein (20100) than atoms in the universe. Only a miniscule fraction of such a vast sequence space can ever be experimentally or computationally surveyed. Deep neural networks are increasingly being used to navigate high-dimensional sequence spaces.",
  "keywords": [
    
  ],
  "articleBody": " The genetic architecture of protein stability\nAuthors: Faure, A. J.; Marti-Aranda, A.; Hidalgo, C.; Schmiedel, J. M.; Lehner, B.\nScore: 50.8, Published: 2023-10-27 DOI: 10.1101/2023.10.27.564339\nThere are more ways to synthesize a 100 amino acid protein (20100) than atoms in the universe. Only a miniscule fraction of such a vast sequence space can ever be experimentally or computationally surveyed. Deep neural networks are increasingly being used to navigate high-dimensional sequence spaces. However, these models are extremely complicated and provide little insight into the fundamental genetic architecture of proteins. Here, by experimentally exploring sequence spaces \u003e1010, we show that the genetic architecture of at least some proteins is remarkably simple, allowing accurate genetic prediction in high-dimensional sequence spaces with fully interpretable biophysical models. These models capture the non-linear relationships between free energies and phenotypes but otherwise consist of additive free energy changes with a small contribution from pairwise energetic couplings. These energetic couplings are sparse and caused by structural contacts and backbone propagations. Our results suggest that artificial intelligence models may be vastly more complicated than the proteins that they are modeling and that protein genetics is actually both simple and intelligible.\nSequence-ensemble-function relationships for disordered proteins in live cells\nAuthors: Emenecker, R. J.; Guadalupe, K.; Shamoon, N. M.; Sukenik, S.; Holehouse, A. S.\nScore: 26.7, Published: 2023-11-01 DOI: 10.1101/2023.10.29.564547\nIntrinsically disordered protein regions (IDRs) are ubiquitous across all kingdoms of life and play a variety of essential cellular roles. IDRs exist in a collection of structurally distinct conformers known as an ensemble. IDR amino acid sequence determines its ensemble, which in turn can play an important role in dictating molecular function. Yet a clear link connecting IDR sequence, its ensemble properties, and its molecular function in living cells has not been systematically established. Here, we set out to test this sequence-ensemble-function paradigm using a novel computational method (GOOSE) that enables the rational design of libraries of IDRs by systematically varying specific sequence properties. Using ensemble FRET, we measured the ensemble dimensions of a library of rationally designed IDRs in human-derived cell lines, revealing how IDR sequence influences ensemble dimensions in situ. Furthermore, we show that the interplay between sequence and ensemble can tune an IDRs ability to sense changes in cell volume - a de novo molecular function for these synthetic sequences. Our results establish biophysical rules for intracellular sequence-ensemble relationships, enable a new route for understanding how IDR sequences map to function in live cells, and set the ground for the design of synthetic IDRs with de novo function.\nFast and robust 3D MINFLUX excitation with a variable phase plate\nAuthors: Deguchi, T.; Ries, J.\nScore: 19.3, Published: 2023-11-01 DOI: 10.1101/2023.10.31.564945\nMINFLUX has achieved record resolution in superresolution imaging and single fluorophore tracking. It is based on localizing single fluorophores by rapid probing with a patterned beam that features a local intensity minimum. Current implementations, however, are complex and expensive and are limited in speed and robustness. Here, we show that a combination of an electro-optical modulator with a segmented birefringent element such as a spatial light modulator produces a variable phase plate for which the phase can be scanned on the MHz timescale. Bisected or top-hat phase patterns generate high-contrast compact excitation point-spread functions for MINFLUX localization in the x,y, and z-direction, respectively, which can be scanned around a fluorophore within a microsecond and alternated among different excitation wavelengths. We discuss how to compensate for non-optimal performance of the components and present a robust 3D and multi-color MINFLUX excitation module with record speed, which we envision as an integral component of a high-performance and cost-effective open-source MINFLUX.\nCryo-EM structure of Slo1 with the auxiliary γ1 subunit suggests mechanism of depolarization-independent activation\nAuthors: Redhardt, M.; Raunser, S.; Raisch, T.\nScore: 17.9, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565197\nMammalian Ca2+-dependent Slo K+ channels are expressed with {beta} and {gamma} auxiliary subunits that greatly influence voltage- and Ca2+-induced gating, thereby fundamentally altering the behavior of the channel. The four {gamma} subunits reduce the need for voltage-dependent activation, allowing Slo to open in the absence of an action potential. The mechanism of this activation has, however, remained elusive. Here, we present the cryo-EM structure of Slo1 in complex with {gamma}1/LRRC26, revealing how the transmembrane helix of {gamma}1 binds and presumably stabilizes the active conformation of the voltage-sensor domain. This effect is further enhanced by a polybasic stretch on the intracellular side of the membrane which locally changes the charge gradient across the membrane. Sequence differences explain why the four {gamma} subunits possess different activation efficiencies. Simultaneous binding of {gamma} and the unrelated {beta} subunits is structurally possible, as both binding sites do not overlap and the {gamma}1 LRR domains are partially flexible. Thus, our data provide a possible explanation for Slo1 regulation by {gamma} subunits, and furthermore suggest a novel mechanism of activation of voltage-gated ion channels by auxiliary subunits and add to the growing knowledge of their complex regulation.\nTransport and inhibition mechanism for VMAT2-mediated synaptic loading of monoamines\nAuthors: Wang, Y.; Zhang, P.; Chao, Y.; Zhu, Z.; Yang, C.; Zhou, Z.; Li, Y.; Long, Y.; Liu, Y.; Li, D.; Wang, S.; Qu, Q.\nScore: 11.9, Published: 2023-10-29 DOI: 10.1101/2023.10.27.564374\nMonoamine neurotransmitters such as serotonin and dopamine are loaded by vesicular monoamine transporter 2 (VMAT2) into synaptic vesicles for storage and subsequent release in neurons. Impaired VMAT2 function underlies various neuropsychiatric diseases. VMAT2 inhibitors reserpine and tetrabenazine are used to treat hypertension, movement disorders associated with Huntingtons Disease and Tardive Dyskinesia. Despite its physiological and pharmacological significance, the structural basis underlying substrate recognition, and inhibition by varying mechanisms remains unknown. Here we present cryo-EM structures of human apo VMAT2 in addition to states bound to serotonin, tetrabenazine, and reserpine. These structures collectively capture three states, namely the lumen-facing, occluded, and cytosol-facing conformations. Notably, tetrabenazine induces a substantial rearrangement of TM2 and TM7, extending beyond the typical rocker-switch movement. These functionally dynamic snapshots, complemented by biochemical analysis, unveil the essential components responsible for ligand recognition, elucidate the proton-driven exchange cycle, and provide a framework to design improved pharmaceutics targeting VMAT2.\n4D Crystallography Captures Transient IF1-Ribosome Dynamics in Translation Initiation\nAuthors: Yapici, I.; Dao, E. H.; Yokoi, S.; Destan, E.; Ayan, E.; Shafiei Kamel, A.; Ertem, F. B.; Kulakman, C.; Yilmaz, M.; Tosun, B.; Ciftci, H.; Kepceoglu, A.; Johnson, J. A.; Guven, O.; Ergul, A.; Hayes, B.; Rao, Y.; Kupitz, C. J.; Poitevin, F.; Liang, M.; Hunter, M. S.; Milon, P.; Mitsutake, A.; Sierra, R. G.; Wakatsuki, S.; DeMirci, H.\nScore: 9.5, Published: 2023-10-29 DOI: 10.1101/2023.10.27.564398\nInitiation factor 1 (IF1) is one of multiple key ligands involved in the initiation of mRNA translation, a highly dynamic and carefully-orchestrated process. However, details surrounding IF1 transient interactions with the small 30S ribosomal subunit remain incompletely understood despite characterization of unbound and fully-bound 30S states. Improvements in X-ray light sources and crystallographic techniques are now enabling time-resolved structural studies at near-physiological temperature and near-atomic resolution and thus the structural investigation of such dynamic processes. Here, we employed time-resolved serial femtosecond X-ray crystallography (TR-SFX) to probe the binding of IF1 to the small 30S ribosomal subunit in real time. Our time-resolved structural data demonstrates transient cryptic short-, mid-, and long-range allostery among different regions of the small 30 ribosomal subunit during IF1 binding, revealing small- and large-scale protein-target interactions and dynamics within intermediate macromolecular states at unprecedented temporal and spatial resolution. These data represent one of the first such 4D crystallographic studies assessing protein-protein and protein-RNA interactions and could serve as the basis for subsequent studies of the ribosome and of the multitudinous dynamic processes which underpin biology, and therefore, of life.\nInsights into Molecular Diversity within the FET Family: Unraveling Phase Separation of the N-Terminal Low Complexity Domain from RNA-Binding Protein EWS\nAuthors: Johnson, C. N.; Sojitra, K. A.; Sohn, E. J.; Moreno-Romero, A. K.; Baudin, A.; Xu, X.; Mittal, J.; Libich, D. S.\nScore: 9.0, Published: 2023-11-01 DOI: 10.1101/2023.10.27.564484\nThe FET family proteins, which includes FUS, EWS, and TAF15, are RNA chaperones instrumental in processes such as mRNA maturation, transcriptional regulation, and the DNA damage response. These proteins have clinical significance: chromosomal rearrangements in FET proteins are implicated in Ewing family tumors and related sarcomas. Furthermore, point mutations in FUS and TAF15 are associated with neurodegenerative conditions like amyotrophic lateral sclerosis and frontotemporal lobar dementia. The fusion protein EWS::FLI1, the causative mutation of Ewing sarcoma, arises from a genomic translocation that fuses the low-complexity domain (LCD) of EWS (EWSLCD) with the DNA binding domain of the ETS transcription factor FLI1. This fusion not only alters transcriptional programs but also hinders native EWS functions like splicing. However, the precise function of the intrinsically disordered EWSLCD is still a topic of active investigation. Due to its flexible nature, EWSLCD can form transient interactions with itself and other biomolecules, leading to the formation of biomolecular condensates through phase separation - a mechanism thought to be central to the oncogenicity of EWS::FLI1. In our study, we used paramagnetic relaxation enhancement NMR, analytical ultracentrifugation, light microscopy, and all-atom molecular dynamics (MD) simulations to better understand the self-association and phase separation tendencies of EWSLCD. Our aim was to elucidate the molecular events that underpin EWSLCD-mediated biomolecular condensation. Our NMR data suggest tyrosine residues primarily drive the interactions vital for EWSLCD phase separation. Moreover, a higher density and proximity of tyrosine residues amplify the likelihood of condensate formation. Atomistic MD simulations and hydrodynamic experiments revealed that the tyrosine-rich N and C-termini tend to populate compact conformations, establishing unique contact networks, that are connected by a predominantly extended, tyrosine-depleted, linker region. MD simulations provide critical input on the relationship between contacts formed within a single molecule (intramolecular) and inside the condensed phase (intermolecular), and changes in protein conformations upon condensation. These results offer deeper insights into the condensate-forming abilities of the FET proteins and highlights unique structural and functional nuances between EWS and its counterparts, FUS and TAF15.\nG protein activation occurs via a largely universal mechanism\nAuthors: Vithani, N.; Todd, T. D.; Singh, S.; Blumer, K. J.; Bowman, G. R.\nScore: 12.8, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563885\nUnderstanding how signaling proteins like G proteins are allosterically activated is a long-standing challenge with significant biological and medical implications. Because it is difficult to directly observe such dynamic processes, much of our understanding is based on inferences from a limited number of static snapshots of relevant protein structures, mutagenesis data, and patterns of sequence conservation. Here, we use computer simulations to directly interrogate allosteric coupling in six G protein -subunit isoforms covering all four G protein families. To analyze this data, we introduce automated methods for inferring allosteric networks from simulation data and assessing how allostery is conserved or diverged among related protein isoforms. We find that the allosteric networks in these six G protein -subunits are largely conserved and consist of two pathways, which we call pathway-I and pathway-II. This analysis predicts that pathway-I is generally dominant over the second pathway, which we experimentally confirmed by showing that mutations to pathway-I perturb nucleotide exchange more than mutations to pathway-II. In the future, insights into unique elements of each G protein family could inform the design of isoform-specific drugs. More broadly, our tools should also be useful for studying allostery in other proteins and assessing the extent to which this allostery is conserved in related proteins.\nCryo-electron tomography reveals the structural diversity of cardiac proteins in their cellular context\nAuthors: Woldeyes, R. A.; Nishiga, M.; Vander Roest, A. S.; Engel, L.; Giri, P.; Montenegro, G. C.; Wu, A. C.; Dunn, A. R.; Spudich, J. A.; Bernstein, D.; Schmid, M. F.; Wu, J. C.; Chiu, W.\nScore: 17.4, Published: 2023-10-26 DOI: 10.1101/2023.10.26.564098\nCardiovascular diseases are a leading cause of death worldwide, but our understanding of the underlying mechanisms is limited, in part because of the complexity of the cellular machinery that controls the heart muscle contraction cycle. Cryogenic electron tomography (cryo-ET) provides a way to visualize diverse cellular machinery while preserving contextual information like subcellular localization and transient complex formation, but this approach has not been widely applied to the study of heart muscle cells (cardiomyocytes). Here, we deploy a platform for studying cardiovascular disease by combining cryo-ET with human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs). After developing a cryo-ET workflow for visualizing macromolecules in hiPSC-CMs, we reconstructed sub-nanometer resolution structures of the human thin filament, a central component of the contractile machinery. We also visualized a previously unobserved organization of a regulatory complex that connects muscle contraction to calcium signaling (the troponin complex), highlighting the value of our approach for interrogating the structures of cardiac proteins in their cellular context.\nInterfacial morphodynamics of proliferating microbial communities\nAuthors: Martinez-Calvo, A.; Trenado-Yuste, C.; Lee, H.; Gore, J.; Wingreen, N. S.; Datta, S. S.\nScore: 35.7, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563665\nIn microbial communities, various cell types often coexist by occupying distinct spatial domains. What determines the shape of the interface between such domains--which in turn influences the interactions between cells and overall community function? Here, we address this question by developing a continuum model of a 2D spatially-structured microbial community with two distinct cell types. We find that, depending on the balance of the different cell proliferation rates and substrate friction coefficients, the interface between domains is either stable and smooth, or unstable and develops finger-like protrusions. We establish quantitative principles describing when these different interfacial behaviors arise, and find good agreement both with the results of previous experimental reports as well as new experiments performed here. Our work thus helps to provide a biophysical basis for understanding the interfacial morphodynamics of proliferating microbial communities, as well as a broader range of proliferating active systems.\n",
  "wordCount" : "2265",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564339">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564339" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564339">
        <p class="paperTitle">The genetic architecture of protein stability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564339" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564339" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faure, A. J.; Marti-Aranda, A.; Hidalgo, C.; Schmiedel, J. M.; Lehner, B.</p>
        <p class="info">Score: 50.8, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564339' target='https://doi.org/10.1101/2023.10.27.564339'> 10.1101/2023.10.27.564339</a></p>
        <p class="abstract">There are more ways to synthesize a 100 amino acid protein (20100) than atoms in the universe. Only a miniscule fraction of such a vast sequence space can ever be experimentally or computationally surveyed. Deep neural networks are increasingly being used to navigate high-dimensional sequence spaces. However, these models are extremely complicated and provide little insight into the fundamental genetic architecture of proteins. Here, by experimentally exploring sequence spaces &gt;1010, we show that the genetic architecture of at least some proteins is remarkably simple, allowing accurate genetic prediction in high-dimensional sequence spaces with fully interpretable biophysical models. These models capture the non-linear relationships between free energies and phenotypes but otherwise consist of additive free energy changes with a small contribution from pairwise energetic couplings. These energetic couplings are sparse and caused by structural contacts and backbone propagations. Our results suggest that artificial intelligence models may be vastly more complicated than the proteins that they are modeling and that protein genetics is actually both simple and intelligible.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.29.564547">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.29.564547" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.29.564547">
        <p class="paperTitle">Sequence-ensemble-function relationships for disordered proteins in live cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.29.564547" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.29.564547" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Emenecker, R. J.; Guadalupe, K.; Shamoon, N. M.; Sukenik, S.; Holehouse, A. S.</p>
        <p class="info">Score: 26.7, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.29.564547' target='https://doi.org/10.1101/2023.10.29.564547'> 10.1101/2023.10.29.564547</a></p>
        <p class="abstract">Intrinsically disordered protein regions (IDRs) are ubiquitous across all kingdoms of life and play a variety of essential cellular roles. IDRs exist in a collection of structurally distinct conformers known as an ensemble. IDR amino acid sequence determines its ensemble, which in turn can play an important role in dictating molecular function. Yet a clear link connecting IDR sequence, its ensemble properties, and its molecular function in living cells has not been systematically established. Here, we set out to test this sequence-ensemble-function paradigm using a novel computational method (GOOSE) that enables the rational design of libraries of IDRs by systematically varying specific sequence properties. Using ensemble FRET, we measured the ensemble dimensions of a library of rationally designed IDRs in human-derived cell lines, revealing how IDR sequence influences ensemble dimensions in situ. Furthermore, we show that the interplay between sequence and ensemble can tune an IDRs ability to sense changes in cell volume - a de novo molecular function for these synthetic sequences. Our results establish biophysical rules for intracellular sequence-ensemble relationships, enable a new route for understanding how IDR sequences map to function in live cells, and set the ground for the design of synthetic IDRs with de novo function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564945">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564945" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564945">
        <p class="paperTitle">Fast and robust 3D MINFLUX excitation with a variable phase plate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564945" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564945" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Deguchi, T.; Ries, J.</p>
        <p class="info">Score: 19.3, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564945' target='https://doi.org/10.1101/2023.10.31.564945'> 10.1101/2023.10.31.564945</a></p>
        <p class="abstract">MINFLUX has achieved record resolution in superresolution imaging and single fluorophore tracking. It is based on localizing single fluorophores by rapid probing with a patterned beam that features a local intensity minimum. Current implementations, however, are complex and expensive and are limited in speed and robustness. Here, we show that a combination of an electro-optical modulator with a segmented birefringent element such as a spatial light modulator produces a variable phase plate for which the phase can be scanned on the MHz timescale. Bisected or top-hat phase patterns generate high-contrast compact excitation point-spread functions for MINFLUX localization in the x,y, and z-direction, respectively, which can be scanned around a fluorophore within a microsecond and alternated among different excitation wavelengths. We discuss how to compensate for non-optimal performance of the components and present a robust 3D and multi-color MINFLUX excitation module with record speed, which we envision as an integral component of a high-performance and cost-effective open-source MINFLUX.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565197">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565197" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565197">
        <p class="paperTitle">Cryo-EM structure of Slo1 with the auxiliary γ1 subunit suggests mechanism of depolarization-independent activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565197" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565197" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Redhardt, M.; Raunser, S.; Raisch, T.</p>
        <p class="info">Score: 17.9, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565197' target='https://doi.org/10.1101/2023.11.01.565197'> 10.1101/2023.11.01.565197</a></p>
        <p class="abstract">Mammalian Ca2&#43;-dependent Slo K&#43; channels are expressed with {beta} and {gamma} auxiliary subunits that greatly influence voltage- and Ca2&#43;-induced gating, thereby fundamentally altering the behavior of the channel. The four {gamma} subunits reduce the need for voltage-dependent activation, allowing Slo to open in the absence of an action potential. The mechanism of this activation has, however, remained elusive. Here, we present the cryo-EM structure of Slo1 in complex with {gamma}1/LRRC26, revealing how the transmembrane helix of {gamma}1 binds and presumably stabilizes the active conformation of the voltage-sensor domain. This effect is further enhanced by a polybasic stretch on the intracellular side of the membrane which locally changes the charge gradient across the membrane. Sequence differences explain why the four {gamma} subunits possess different activation efficiencies. Simultaneous binding of {gamma} and the unrelated {beta} subunits is structurally possible, as both binding sites do not overlap and the {gamma}1 LRR domains are partially flexible. Thus, our data provide a possible explanation for Slo1 regulation by {gamma} subunits, and furthermore suggest a novel mechanism of activation of voltage-gated ion channels by auxiliary subunits and add to the growing knowledge of their complex regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564374">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564374" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564374">
        <p class="paperTitle">Transport and inhibition mechanism for VMAT2-mediated synaptic loading of monoamines</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564374" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564374" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Zhang, P.; Chao, Y.; Zhu, Z.; Yang, C.; Zhou, Z.; Li, Y.; Long, Y.; Liu, Y.; Li, D.; Wang, S.; Qu, Q.</p>
        <p class="info">Score: 11.9, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564374' target='https://doi.org/10.1101/2023.10.27.564374'> 10.1101/2023.10.27.564374</a></p>
        <p class="abstract">Monoamine neurotransmitters such as serotonin and dopamine are loaded by vesicular monoamine transporter 2 (VMAT2) into synaptic vesicles for storage and subsequent release in neurons. Impaired VMAT2 function underlies various neuropsychiatric diseases. VMAT2 inhibitors reserpine and tetrabenazine are used to treat hypertension, movement disorders associated with Huntingtons Disease and Tardive Dyskinesia. Despite its physiological and pharmacological significance, the structural basis underlying substrate recognition, and inhibition by varying mechanisms remains unknown. Here we present cryo-EM structures of human apo VMAT2 in addition to states bound to serotonin, tetrabenazine, and reserpine. These structures collectively capture three states, namely the lumen-facing, occluded, and cytosol-facing conformations. Notably, tetrabenazine induces a substantial rearrangement of TM2 and TM7, extending beyond the typical rocker-switch movement. These functionally dynamic snapshots, complemented by biochemical analysis, unveil the essential components responsible for ligand recognition, elucidate the proton-driven exchange cycle, and provide a framework to design improved pharmaceutics targeting VMAT2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564398">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564398" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564398">
        <p class="paperTitle">4D Crystallography Captures Transient IF1-Ribosome Dynamics in Translation Initiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564398" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564398" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yapici, I.; Dao, E. H.; Yokoi, S.; Destan, E.; Ayan, E.; Shafiei Kamel, A.; Ertem, F. B.; Kulakman, C.; Yilmaz, M.; Tosun, B.; Ciftci, H.; Kepceoglu, A.; Johnson, J. A.; Guven, O.; Ergul, A.; Hayes, B.; Rao, Y.; Kupitz, C. J.; Poitevin, F.; Liang, M.; Hunter, M. S.; Milon, P.; Mitsutake, A.; Sierra, R. G.; Wakatsuki, S.; DeMirci, H.</p>
        <p class="info">Score: 9.5, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564398' target='https://doi.org/10.1101/2023.10.27.564398'> 10.1101/2023.10.27.564398</a></p>
        <p class="abstract">Initiation factor 1 (IF1) is one of multiple key ligands involved in the initiation of mRNA translation, a highly dynamic and carefully-orchestrated process. However, details surrounding IF1 transient interactions with the small 30S ribosomal subunit remain incompletely understood despite characterization of unbound and fully-bound 30S states. Improvements in X-ray light sources and crystallographic techniques are now enabling time-resolved structural studies at near-physiological temperature and near-atomic resolution and thus the structural investigation of such dynamic processes. Here, we employed time-resolved serial femtosecond X-ray crystallography (TR-SFX) to probe the binding of IF1 to the small 30S ribosomal subunit in real time. Our time-resolved structural data demonstrates transient cryptic short-, mid-, and long-range allostery among different regions of the small 30 ribosomal subunit during IF1 binding, revealing small- and large-scale protein-target interactions and dynamics within intermediate macromolecular states at unprecedented temporal and spatial resolution. These data represent one of the first such 4D crystallographic studies assessing protein-protein and protein-RNA interactions and could serve as the basis for subsequent studies of the ribosome and of the multitudinous dynamic processes which underpin biology, and therefore, of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.564484">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.564484" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.564484">
        <p class="paperTitle">Insights into Molecular Diversity within the FET Family: Unraveling Phase Separation of the N-Terminal Low Complexity Domain from RNA-Binding Protein EWS</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.564484" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.564484" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, C. N.; Sojitra, K. A.; Sohn, E. J.; Moreno-Romero, A. K.; Baudin, A.; Xu, X.; Mittal, J.; Libich, D. S.</p>
        <p class="info">Score: 9.0, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.564484' target='https://doi.org/10.1101/2023.10.27.564484'> 10.1101/2023.10.27.564484</a></p>
        <p class="abstract">The FET family proteins, which includes FUS, EWS, and TAF15, are RNA chaperones instrumental in processes such as mRNA maturation, transcriptional regulation, and the DNA damage response. These proteins have clinical significance: chromosomal rearrangements in FET proteins are implicated in Ewing family tumors and related sarcomas. Furthermore, point mutations in FUS and TAF15 are associated with neurodegenerative conditions like amyotrophic lateral sclerosis and frontotemporal lobar dementia. The fusion protein EWS::FLI1, the causative mutation of Ewing sarcoma, arises from a genomic translocation that fuses the low-complexity domain (LCD) of EWS (EWSLCD) with the DNA binding domain of the ETS transcription factor FLI1. This fusion not only alters transcriptional programs but also hinders native EWS functions like splicing. However, the precise function of the intrinsically disordered EWSLCD is still a topic of active investigation. Due to its flexible nature, EWSLCD can form transient interactions with itself and other biomolecules, leading to the formation of biomolecular condensates through phase separation - a mechanism thought to be central to the oncogenicity of EWS::FLI1. In our study, we used paramagnetic relaxation enhancement NMR, analytical ultracentrifugation, light microscopy, and all-atom molecular dynamics (MD) simulations to better understand the self-association and phase separation tendencies of EWSLCD. Our aim was to elucidate the molecular events that underpin EWSLCD-mediated biomolecular condensation. Our NMR data suggest tyrosine residues primarily drive the interactions vital for EWSLCD phase separation. Moreover, a higher density and proximity of tyrosine residues amplify the likelihood of condensate formation. Atomistic MD simulations and hydrodynamic experiments revealed that the tyrosine-rich N and C-termini tend to populate compact conformations, establishing unique contact networks, that are connected by a predominantly extended, tyrosine-depleted, linker region. MD simulations provide critical input on the relationship between contacts formed within a single molecule (intramolecular) and inside the condensed phase (intermolecular), and changes in protein conformations upon condensation. These results offer deeper insights into the condensate-forming abilities of the FET proteins and highlights unique structural and functional nuances between EWS and its counterparts, FUS and TAF15.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563885">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563885" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563885">
        <p class="paperTitle">G protein activation occurs via a largely universal mechanism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563885" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563885" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vithani, N.; Todd, T. D.; Singh, S.; Blumer, K. J.; Bowman, G. R.</p>
        <p class="info">Score: 12.8, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563885' target='https://doi.org/10.1101/2023.10.24.563885'> 10.1101/2023.10.24.563885</a></p>
        <p class="abstract">Understanding how signaling proteins like G proteins are allosterically activated is a long-standing challenge with significant biological and medical implications. Because it is difficult to directly observe such dynamic processes, much of our understanding is based on inferences from a limited number of static snapshots of relevant protein structures, mutagenesis data, and patterns of sequence conservation. Here, we use computer simulations to directly interrogate allosteric coupling in six G protein -subunit isoforms covering all four G protein families. To analyze this data, we introduce automated methods for inferring allosteric networks from simulation data and assessing how allostery is conserved or diverged among related protein isoforms. We find that the allosteric networks in these six G protein -subunits are largely conserved and consist of two pathways, which we call pathway-I and pathway-II. This analysis predicts that pathway-I is generally dominant over the second pathway, which we experimentally confirmed by showing that mutations to pathway-I perturb nucleotide exchange more than mutations to pathway-II. In the future, insights into unique elements of each G protein family could inform the design of isoform-specific drugs. More broadly, our tools should also be useful for studying allostery in other proteins and assessing the extent to which this allostery is conserved in related proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564098">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564098" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564098">
        <p class="paperTitle">Cryo-electron tomography reveals the structural diversity of cardiac proteins in their cellular context</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564098" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564098" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Woldeyes, R. A.; Nishiga, M.; Vander Roest, A. S.; Engel, L.; Giri, P.; Montenegro, G. C.; Wu, A. C.; Dunn, A. R.; Spudich, J. A.; Bernstein, D.; Schmid, M. F.; Wu, J. C.; Chiu, W.</p>
        <p class="info">Score: 17.4, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564098' target='https://doi.org/10.1101/2023.10.26.564098'> 10.1101/2023.10.26.564098</a></p>
        <p class="abstract">Cardiovascular diseases are a leading cause of death worldwide, but our understanding of the underlying mechanisms is limited, in part because of the complexity of the cellular machinery that controls the heart muscle contraction cycle. Cryogenic electron tomography (cryo-ET) provides a way to visualize diverse cellular machinery while preserving contextual information like subcellular localization and transient complex formation, but this approach has not been widely applied to the study of heart muscle cells (cardiomyocytes). Here, we deploy a platform for studying cardiovascular disease by combining cryo-ET with human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs). After developing a cryo-ET workflow for visualizing macromolecules in hiPSC-CMs, we reconstructed sub-nanometer resolution structures of the human thin filament, a central component of the contractile machinery. We also visualized a previously unobserved organization of a regulatory complex that connects muscle contraction to calcium signaling (the troponin complex), highlighting the value of our approach for interrogating the structures of cardiac proteins in their cellular context.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563665">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563665" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563665">
        <p class="paperTitle">Interfacial morphodynamics of proliferating microbial communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563665" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563665" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Calvo, A.; Trenado-Yuste, C.; Lee, H.; Gore, J.; Wingreen, N. S.; Datta, S. S.</p>
        <p class="info">Score: 35.7, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563665' target='https://doi.org/10.1101/2023.10.23.563665'> 10.1101/2023.10.23.563665</a></p>
        <p class="abstract">In microbial communities, various cell types often coexist by occupying distinct spatial domains. What determines the shape of the interface between such domains--which in turn influences the interactions between cells and overall community function? Here, we address this question by developing a continuum model of a 2D spatially-structured microbial community with two distinct cell types. We find that, depending on the balance of the different cell proliferation rates and substrate friction coefficients, the interface between domains is either stable and smooth, or unstable and develops finger-like protrusions. We establish quantitative principles describing when these different interfacial behaviors arise, and find good agreement both with the results of previous experimental reports as well as new experiments performed here. Our work thus helps to provide a biophysical basis for understanding the interfacial morphodynamics of proliferating microbial communities, as well as a broader range of proliferating active systems.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
