<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="An ultrasensitive genetically encoded voltage indicator uncovers the electrical activity of non-excitable cells
Authors: Ruehl, P.; Nair, A. G.; Gawande, N.; Dehiwalage, S. N. C. W.; Muenster, L.; Schoenherr, R.; Heinemann, S. H.
Score: 19.4, Published: 2023-10-06 DOI: 10.1101/2023.10.05.560122
Genetically encoded voltage indicators (GEVIs) are powerful, non-invasive tools for recording action potentials in excitable cells. However, most animal cell types are non-excitable, and yet variations in the membrane potential are biologically relevant in these cells as well.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="An ultrasensitive genetically encoded voltage indicator uncovers the electrical activity of non-excitable cells
Authors: Ruehl, P.; Nair, A. G.; Gawande, N.; Dehiwalage, S. N. C. W.; Muenster, L.; Schoenherr, R.; Heinemann, S. H.
Score: 19.4, Published: 2023-10-06 DOI: 10.1101/2023.10.05.560122
Genetically encoded voltage indicators (GEVIs) are powerful, non-invasive tools for recording action potentials in excitable cells. However, most animal cell types are non-excitable, and yet variations in the membrane potential are biologically relevant in these cells as well." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-15T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-10-15T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="An ultrasensitive genetically encoded voltage indicator uncovers the electrical activity of non-excitable cells
Authors: Ruehl, P.; Nair, A. G.; Gawande, N.; Dehiwalage, S. N. C. W.; Muenster, L.; Schoenherr, R.; Heinemann, S. H.
Score: 19.4, Published: 2023-10-06 DOI: 10.1101/2023.10.05.560122
Genetically encoded voltage indicators (GEVIs) are powerful, non-invasive tools for recording action potentials in excitable cells. However, most animal cell types are non-excitable, and yet variations in the membrane potential are biologically relevant in these cells as well."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "An ultrasensitive genetically encoded voltage indicator uncovers the electrical activity of non-excitable cells\nAuthors: Ruehl, P.; Nair, A. G.; Gawande, N.; Dehiwalage, S. N. C. W.; Muenster, L.; Schoenherr, R.; Heinemann, S. H.\nScore: 19.4, Published: 2023-10-06 DOI: 10.1101/2023.10.05.560122\nGenetically encoded voltage indicators (GEVIs) are powerful, non-invasive tools for recording action potentials in excitable cells. However, most animal cell types are non-excitable, and yet variations in the membrane potential are biologically relevant in these cells as well.",
  "keywords": [
    
  ],
  "articleBody": " An ultrasensitive genetically encoded voltage indicator uncovers the electrical activity of non-excitable cells\nAuthors: Ruehl, P.; Nair, A. G.; Gawande, N.; Dehiwalage, S. N. C. W.; Muenster, L.; Schoenherr, R.; Heinemann, S. H.\nScore: 19.4, Published: 2023-10-06 DOI: 10.1101/2023.10.05.560122\nGenetically encoded voltage indicators (GEVIs) are powerful, non-invasive tools for recording action potentials in excitable cells. However, most animal cell types are non-excitable, and yet variations in the membrane potential are biologically relevant in these cells as well. Resolving such small voltage signals demands GEVIs with exceptionally high sensitivity. In this study, we applied structure-guided engineering to the GEVI ASAP3 to generate rEstus, a sensor with optimized brightness, voltage sensitivity, and voltage range. rEstus is most sensitive in the resting voltage range of non-excitable cells, exhibits a 3.6-fold improvement in fast voltage spike detection, and allows for absolute voltage calibration at the single-cell level. Using rEstus, we resolved endogenous voltage fluctuations in several non-excitable cell types and demonstrate that correlation analysis of these optically recorded fluctuations provides an easy, non-invasive, real-time readout of electrical gap-junction coupling. Our work provides greatly enhanced tools and methods for the non-invasive study of electrical signaling in excitable and non-excitable cells.\nFast, whole-cell DNA-PAINT imaging with dual-labeled self-quenched imager probes\nAuthors: Sauer, M.; Ghosh, A.; Meub, M.; Helmerich, D. A.; Doose, S.\nScore: 16.6, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561472\nIn DNA-PAINT, background fluorescence emerging from unbound imager probes requires the use of low imager concentrations, reducing the imaging speed substantially. Here, we present a novel class of dual-labeled probes that show strong fluorescence quenching when unbound, enabling imaging at higher probe concentrations with an up to [~]15-fold increase in imaging speed and improved resolution. We demonstrate two-dye imager DNA-PAINT (TDI-DNA-PAINT) imaging in cells and on DNA origami. Finally, we demonstrate lattice light-sheet (LLS)-TDI-DNA-PAINT for fast whole-cell three-dimensional (3D) imaging with superior spatial resolution.\nA novel subgroup of light-driven sodium pumps with an additional Schiff base counterion\nAuthors: Podoliak, E.; Lamm, G. H. U.; Alekseev, A.; Marin, E.; Schellbach, A. V.; Fedotov, D. A.; Stetsenko, A.; Maliar, N.; Bourenkov, G.; Balandin, T.; Baeken, C.; Astashkin, R.; Schneider, T. R.; Bateman, A.; Wachtveitl, J.; Schapiro, I.; Busskamp, V.; Guskov, A.; Gordeliy, V.; Kovalev, K.\nScore: 10.3, Published: 2023-10-12 DOI: 10.1101/2023.10.11.561842\nLight-driven sodium pumps (NaRs) are unique ion-transporting microbial rhodopsins. The major group of NaRs is characterized by an NDQ motif and has two aspartic acid residues in the central region essential for sodium transport. Here we identified a new subgroup of the NDQ rhodopsins bearing an additional glutamic acid residue in the close vicinity to the retinal Schiff base. We thoroughly characterized a member of this subgroup, namely the protein ErNaR from Erythrobacter sp. HL-111 and showed that the additional glutamic acid results in almost complete loss of pH sensitivity for sodium-pumping activity, which is in contrast to previously studied NaRs. ErNaR is capable of transporting sodium efficiently even at acidic pH levels. X-ray crystallography and single particle cryo-electron microscopy reveal that the additional glutamic acid residue mediates the connection between the other two Schiff base counterions and strongly interacts with the aspartic acid of the characteristic NDQ motif. Hence, it reduces its pKa. Our findings shed light on a new subgroup of NaRs and might serve as a basis for their rational optimization for optogenetics.\nArchitecture and rearrangements of a sperm-specific Na+/H+ exchanger\nAuthors: Pal, K.; Chowdhury, S.\nScore: 14.4, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560940\nThe sperm-specific sodium hydrogen exchanger, SLC9C1, underlies hyperpolarization and cyclic nucleotide stimulated proton fluxes across sperm membranes and regulates their hyperactivated motility. SLC9C1 is the first known instance of an ion transporter that uses a canonical voltage-sensing domain (VSD) and an evolutionarily conserved cyclic nucleotide binding domain (CNBD) to influence the dynamics of its ion-exchange domain (ED). The structural organization of this tripartite transporter and the mechanisms whereby it integrates physical (membrane voltage) and chemical (cyclic nucleotide) cues are unknown. In this study, we use single particle cryo-electron microscopy to determine structures of a metazoan SLC9C1 in different conformational states. We find that the three structural domains are uniquely organized around a distinct ring-shaped scaffold that we call the allosteric ring domain or ARD. The ARD undergoes coupled proton-dependent rearrangements with the ED and acts as a signaling hub enabling allosteric communication between the key functional modules of sp9C1. We demonstrate that binding of cAMP causes large conformational changes in the cytoplasmic domains and disrupts key ARD-linked interfaces. We propose that these structural changes rescue the transmembrane domains from an auto-inhibited state and facilitate their functional dynamics. Our study provides a structural framework to understand and further probe electrochemical linkage in SLC9C1.\nBiomolecular condensates form spatially inhomogeneous network fluids\nAuthors: Dar, F.; Cohen, S. R.; Mitrea, D. M.; Phillips, A. H.; Nagy, G.; Leite, W. C.; Stanley, C. B.; Choi, J.-M.; Kriwacki, R. W.; Pappu, R. V.\nScore: 8.5, Published: 2023-10-09 DOI: 10.1101/2023.10.07.561338\nThe functions of biomolecular condensates are thought to be influenced by their material properties, and these are in turn determined by the multiscale structural features within condensates. However, structural characterizations of condensates are challenging, and hence rarely reported. Here, we deploy a combination of small angle neutron scattering, fluorescence recovery after photobleaching, and bespoke coarse-grained molecular dynamics simulations to provide structural descriptions of model condensates that mimic nucleolar granular components (GCs). We show that facsimiles of GCs are network fluids featuring spatial inhomogeneities across hierarchies of length scales that reflect the contributions of distinct protein and peptide domains. The network-like inhomogeneous organization is characterized by a coexistence of liquid- and gas-like macromolecular densities that engenders bimodality of internal molecular dynamics. These insights, extracted from a combination of approaches, suggest that condensates formed by multivalent proteins share features with network fluids formed by associative systems such as patchy or hairy colloids.\nStructural basis of transcription: RNA Polymerase II substrate binding and metal coordination at 3.0 A using a free-electron laser\nAuthors: Guowu, L.; Barnes, C. O.; Weiss, S.; Dutagaci, B.; Qiu, C.; Feig, M.; Lyubimov, A.; Song, J.; Cohen, A. E.; Kaplan, C. D.; Calero, G.\nScore: 7.0, Published: 2023-09-22 DOI: 10.1101/2023.09.22.559052\nCatalysis and translocation of multi-subunit DNA-directed RNA polymerases underlie all cellular mRNA synthesis. RNA polymerase II (Pol II) synthesizes eukaryotic pre-mRNAs from a DNA template strand buried in its active site. Structural details of catalysis at near atomic resolution and precise arrangement of key active site components have been elusive. Here we present the free electron laser (FEL) structure of a matched ATP-bound Pol II, revealing the full active site interaction network at the highest resolution to date, including the trigger loop (TL) in the closed conformation, bonafide occupancy of both site A and B Mg2+, and a putative third (site C) Mg2+ analogous to that described for some DNA polymerases but not observed previously for cellular RNA polymerases. Molecular dynamics (MD) simulations of the structure indicate that the third Mg2+ is coordinated and stabilized at its observed position. TL residues provide half of the substrate binding pocket while multiple TL/bridge helix (BH) interactions induce conformational changes that could propel translocation upon substrate hydrolysis. Consistent with TL/BH communication, a FEL structure and MD simulations of the hyperactive Rpb1 T834P bridge helix mutant reveals rearrangement of some active site interactions supporting potential plasticity in active site function and long-distance effects on both the width of the central channel and TL conformation, likely underlying its increased elongation rate at the expense of fidelity.\nMapping energy landscapes of homopolymeric RNAs via simulated tempering and deep unsupervised learning\nAuthors: Ramachandran, V.; Potoyan, D. A.\nScore: 4.8, Published: 2023-10-10 DOI: 10.1101/2023.10.10.561750\nConformational dynamics plays crucial roles in RNA functions about sensing and responding to environmental signals. The liquid-liquid phase separation of RNAs and the formation of stress granules partly relies on RNAs conformational plasticity and its ability to engage in multivalent interactions. Recent experiments with homopolymeric and low-complexity RNAs have revealed significant differences in phase separations due to differences in base chemistry of RNA units. We hypothesize that differences in RNA phase-transition dynamics can be traced back to the differences in conformational dynamics of single RNA chains. In the present contribution, we utilize atomistic simulations with numerous unsupervised learning to map temperature dependence conformational free energy landscapes for homopolymeric RNA chains. These landscapes reveal a variety of metastable excited states influenced by the nature of base chemistry. We shed light on the distinct contributions of the polyphosphate backbone versus base chemistry in shaping conformational ensembles of different RNAs. We demonstrate that the experimentally observed temperature-driven shifts in metastable state populations align with experimental phase diagrams for homopolymeric RNAs. The work establishes a microscopic framework to reason about base-specific RNA propensity for phase separation. We believe our work will be valuable for designing novel RNA sensors for biological and synthetic applications.\nMolecular mechanisms of liposome interactions with bacterial envelopes\nAuthors: Scheeder, A.; Brockhoff, M.; Ward, E.; Kaminski Schierle, G. S.; Mela, I.; Kaminski, C. F.\nScore: 4.5, Published: 2023-10-10 DOI: 10.1101/2023.10.07.561336\nAlthough fusogenic liposomes offer a promising approach for the delivery of antibiotic payloads across the cell envelope of Gram-negative bacteria, there is still limited understanding of the individual nanocarrier interactions with the bacterial target. Using super-resolution microscopy, we characterize the interaction dynamics of positively charged fusogenic liposomes with Gram-negative (Escherichia coli) and Gram-positive (Bacillus subtilis) bacteria. The liposomes merge with the outer membrane (OM) of Gram-negative bacteria, while attachment or lipid internalization is observed in Gram-positive cells. Employing total internal reflection fluorescence microscopy, we demonstrated liposome fusion with model supported lipid bilayers. For whole E. coli cells, however, we observed heterogeneous membrane integrations, primarily involving liposome attachment and hemifusion events. With increasing lipopolysaccharide length the likelihood of full-fusion events was reduced. The integration of artificial lipids into the OM of Gram-negative cells led to membrane destabilization, resulting in decreased bacterial vitality, membrane detachment, and improved co-delivery of Vancomycin--an effective antibiotic against Gram-positive cells. These findings provide significant insights into the interactions of individual nanocarriers with bacterial envelopes at the single-cell level, uncovering effects that would be missed in bulk measurements. This highlights the importance of conducting single-particle and single-cell investigations to assess the performance of next-generation drug delivery platforms.\nMotion Blur Microscopy\nAuthors: Goreke, U.; Gonzales, A.; Shipley, B.; Man, Y.; Wulftange, W.; An, R.; Hinczewski, M.; Gurkan, U. A.\nScore: 4.3, Published: 2023-10-10 DOI: 10.1101/2023.10.08.561435\nImaging and characterizing the dynamics of cellular adhesion in blood samples is of fundamental importance in understanding biological function. In vitro microscopy methods are widely used for this task, but typically require diluting the blood with a buffer to allow for transmission of light. However whole blood provides crucial mechanical and chemical signaling cues that influence adhesion dynamics, which means that conventional approaches lack the full physiological complexity of living microvasculature. We propose to overcome this challenge by a new in vitro imaging method which we call motion blur microscopy (MBM). By decreasing the source light intensity and increasing the integration time during imaging, flowing cells are blurred, allowing us to identify adhered cells. Combined with an automated analysis using machine learning, we can for the first time reliably image cell interactions in microfluidic channels during whole blood flow. MBM provides a low cost, easy to implement alternative to intravital microscopy, the in vivo approach for studying how the whole blood environment shapes adhesion dynamics. We demonstrate the methods reproducibility and accuracy in two example systems where understanding cell interactions, adhesion, and motility is crucial--sickle red blood cells adhering to laminin, and CAR-T cells adhering to E-selectin. We illustrate the wide range of data types that can be extracted from this approach, including distributions of cell size and eccentricity, adhesion times, trajectories and velocities of adhered cells moving on a functionalized surface, as well as correlations among these different features at the single cell level. In all cases MBM allows for rapid collection and processing of large data sets, ranging from thousands to hundreds of thousands of individual adhesion events. The method is generalizable to study adhesion mechanisms in a variety of diseases, including cancer, blood disorders, thrombosis, inflammatory and autoimmune diseases, as well as providing rich datasets for theoretical modeling of adhesion dynamics.\nStructural basis for CFTR inhibition by CFTRinh-172\nAuthors: Young, P.; Levring, J.; Fiedorczuk, K.; Blanchard, S. C.; Chen, J.\nScore: 2.6, Published: 2023-10-12 DOI: 10.1101/2023.10.11.561899\nThe cystic fibrosis transmembrane conductance regulator (CFTR) is an anion channel that regulates electrolyte and fluid balance in epithelial tissues. Whereas activation of CFTR is vital to treating cystic fibrosis, selective inhibition of CFTR is a potential therapeutic strategy for secretory diarrhea and autosomal dominant polycystic kidney disease (ADPKD). Although several CFTR inhibitors have been developed by high-throughput screening, their modes of action remain elusive. In this study, we determined the structure of CFTR in complex with the inhibitor CFTRinh-172 to 2.7 [A] resolution by cryogenic electron microscopy (cryo-EM). We observe that CFTRinh-172 binds inside the pore near transmembrane helix 8 (TM8), a critical structural element that links ATP hydrolysis with channel gating. Binding of CFTRinh-172 stabilizes a conformation in which the chloride selectivity filter is collapsed and the pore is blocked from the extracellular side of the membrane. Single molecule fluorescence resonance energy transfer (smFRET) experiments indicate that CFTRinh-172 inhibits channel gating without compromising nucleotide-binding domain (NBD) dimerization. Together, these data show that CFTRinh-172 acts as both a pore blocker and a gating modulator, setting it apart from typical ion channel inhibitors. The dual functionality of CFTRinh-172 reconciles previous biophysical observations and provides a molecular basis for its activity.\n",
  "wordCount" : "2240",
  "inLanguage": "en",
  "datePublished": "2023-10-15T10:38:01Z",
  "dateModified": "2023-10-15T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on October 15, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.05.560122">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.05.560122" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.05.560122">
        <p class="paperTitle">An ultrasensitive genetically encoded voltage indicator uncovers the electrical activity of non-excitable cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.05.560122" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.05.560122" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ruehl, P.; Nair, A. G.; Gawande, N.; Dehiwalage, S. N. C. W.; Muenster, L.; Schoenherr, R.; Heinemann, S. H.</p>
        <p class="info">Score: 19.4, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.05.560122' target='https://doi.org/10.1101/2023.10.05.560122'> 10.1101/2023.10.05.560122</a></p>
        <p class="abstract">Genetically encoded voltage indicators (GEVIs) are powerful, non-invasive tools for recording action potentials in excitable cells. However, most animal cell types are non-excitable, and yet variations in the membrane potential are biologically relevant in these cells as well. Resolving such small voltage signals demands GEVIs with exceptionally high sensitivity. In this study, we applied structure-guided engineering to the GEVI ASAP3 to generate rEstus, a sensor with optimized brightness, voltage sensitivity, and voltage range. rEstus is most sensitive in the resting voltage range of non-excitable cells, exhibits a 3.6-fold improvement in fast voltage spike detection, and allows for absolute voltage calibration at the single-cell level. Using rEstus, we resolved endogenous voltage fluctuations in several non-excitable cell types and demonstrate that correlation analysis of these optically recorded fluctuations provides an easy, non-invasive, real-time readout of electrical gap-junction coupling. Our work provides greatly enhanced tools and methods for the non-invasive study of electrical signaling in excitable and non-excitable cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561472">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561472" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561472">
        <p class="paperTitle">Fast, whole-cell DNA-PAINT imaging with dual-labeled self-quenched imager probes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561472" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561472" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sauer, M.; Ghosh, A.; Meub, M.; Helmerich, D. A.; Doose, S.</p>
        <p class="info">Score: 16.6, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561472' target='https://doi.org/10.1101/2023.10.09.561472'> 10.1101/2023.10.09.561472</a></p>
        <p class="abstract">In DNA-PAINT, background fluorescence emerging from unbound imager probes requires the use of low imager concentrations, reducing the imaging speed substantially. Here, we present a novel class of dual-labeled probes that show strong fluorescence quenching when unbound, enabling imaging at higher probe concentrations with an up to [~]15-fold increase in imaging speed and improved resolution. We demonstrate two-dye imager DNA-PAINT (TDI-DNA-PAINT) imaging in cells and on DNA origami. Finally, we demonstrate lattice light-sheet (LLS)-TDI-DNA-PAINT for fast whole-cell three-dimensional (3D) imaging with superior spatial resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561842">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561842" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561842">
        <p class="paperTitle">A novel subgroup of light-driven sodium pumps with an additional Schiff base counterion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561842" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561842" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Podoliak, E.; Lamm, G. H. U.; Alekseev, A.; Marin, E.; Schellbach, A. V.; Fedotov, D. A.; Stetsenko, A.; Maliar, N.; Bourenkov, G.; Balandin, T.; Baeken, C.; Astashkin, R.; Schneider, T. R.; Bateman, A.; Wachtveitl, J.; Schapiro, I.; Busskamp, V.; Guskov, A.; Gordeliy, V.; Kovalev, K.</p>
        <p class="info">Score: 10.3, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561842' target='https://doi.org/10.1101/2023.10.11.561842'> 10.1101/2023.10.11.561842</a></p>
        <p class="abstract">Light-driven sodium pumps (NaRs) are unique ion-transporting microbial rhodopsins. The major group of NaRs is characterized by an NDQ motif and has two aspartic acid residues in the central region essential for sodium transport. Here we identified a new subgroup of the NDQ rhodopsins bearing an additional glutamic acid residue in the close vicinity to the retinal Schiff base. We thoroughly characterized a member of this subgroup, namely the protein ErNaR from Erythrobacter sp. HL-111 and showed that the additional glutamic acid results in almost complete loss of pH sensitivity for sodium-pumping activity, which is in contrast to previously studied NaRs. ErNaR is capable of transporting sodium efficiently even at acidic pH levels. X-ray crystallography and single particle cryo-electron microscopy reveal that the additional glutamic acid residue mediates the connection between the other two Schiff base counterions and strongly interacts with the aspartic acid of the characteristic NDQ motif. Hence, it reduces its pKa. Our findings shed light on a new subgroup of NaRs and might serve as a basis for their rational optimization for optogenetics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560940">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560940" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560940">
        <p class="paperTitle">Architecture and rearrangements of a sperm-specific Na&#43;/H&#43; exchanger</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560940" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560940" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pal, K.; Chowdhury, S.</p>
        <p class="info">Score: 14.4, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560940' target='https://doi.org/10.1101/2023.10.04.560940'> 10.1101/2023.10.04.560940</a></p>
        <p class="abstract">The sperm-specific sodium hydrogen exchanger, SLC9C1, underlies hyperpolarization and cyclic nucleotide stimulated proton fluxes across sperm membranes and regulates their hyperactivated motility. SLC9C1 is the first known instance of an ion transporter that uses a canonical voltage-sensing domain (VSD) and an evolutionarily conserved cyclic nucleotide binding domain (CNBD) to influence the dynamics of its ion-exchange domain (ED). The structural organization of this  tripartite transporter and the mechanisms whereby it integrates physical (membrane voltage) and chemical (cyclic nucleotide) cues are unknown. In this study, we use single particle cryo-electron microscopy to determine structures of a metazoan SLC9C1 in different conformational states. We find that the three structural domains are uniquely organized around a distinct ring-shaped scaffold that we call the  allosteric ring domain or ARD. The ARD undergoes coupled proton-dependent rearrangements with the ED and acts as a  signaling hub enabling allosteric communication between the key functional modules of sp9C1. We demonstrate that binding of cAMP causes large conformational changes in the cytoplasmic domains and disrupts key ARD-linked interfaces. We propose that these structural changes rescue the transmembrane domains from an auto-inhibited state and facilitate their functional dynamics. Our study provides a structural framework to understand and further probe electrochemical linkage in SLC9C1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.07.561338">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.07.561338" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.07.561338">
        <p class="paperTitle">Biomolecular condensates form spatially inhomogeneous network fluids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.07.561338" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.07.561338" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dar, F.; Cohen, S. R.; Mitrea, D. M.; Phillips, A. H.; Nagy, G.; Leite, W. C.; Stanley, C. B.; Choi, J.-M.; Kriwacki, R. W.; Pappu, R. V.</p>
        <p class="info">Score: 8.5, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.07.561338' target='https://doi.org/10.1101/2023.10.07.561338'> 10.1101/2023.10.07.561338</a></p>
        <p class="abstract">The functions of biomolecular condensates are thought to be influenced by their material properties, and these are in turn determined by the multiscale structural features within condensates. However, structural characterizations of condensates are challenging, and hence rarely reported. Here, we deploy a combination of small angle neutron scattering, fluorescence recovery after photobleaching, and bespoke coarse-grained molecular dynamics simulations to provide structural descriptions of model condensates that mimic nucleolar granular components (GCs). We show that facsimiles of GCs are network fluids featuring spatial inhomogeneities across hierarchies of length scales that reflect the contributions of distinct protein and peptide domains. The network-like inhomogeneous organization is characterized by a coexistence of liquid- and gas-like macromolecular densities that engenders bimodality of internal molecular dynamics. These insights, extracted from a combination of approaches, suggest that condensates formed by multivalent proteins share features with network fluids formed by associative systems such as patchy or hairy colloids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.559052">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.559052" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.559052">
        <p class="paperTitle">Structural basis of transcription: RNA Polymerase II substrate binding and metal coordination at 3.0 A using a free-electron laser</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.559052" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.559052" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guowu, L.; Barnes, C. O.; Weiss, S.; Dutagaci, B.; Qiu, C.; Feig, M.; Lyubimov, A.; Song, J.; Cohen, A. E.; Kaplan, C. D.; Calero, G.</p>
        <p class="info">Score: 7.0, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.559052' target='https://doi.org/10.1101/2023.09.22.559052'> 10.1101/2023.09.22.559052</a></p>
        <p class="abstract">Catalysis and translocation of multi-subunit DNA-directed RNA polymerases underlie all cellular mRNA synthesis. RNA polymerase II (Pol II) synthesizes eukaryotic pre-mRNAs from a DNA template strand buried in its active site. Structural details of catalysis at near atomic resolution and precise arrangement of key active site components have been elusive. Here we present the free electron laser (FEL) structure of a matched ATP-bound Pol II, revealing the full active site interaction network at the highest resolution to date, including the trigger loop (TL) in the closed conformation, bonafide occupancy of both site A and B Mg2&#43;, and a putative third (site C) Mg2&#43; analogous to that described for some DNA polymerases but not observed previously for cellular RNA polymerases. Molecular dynamics (MD) simulations of the structure indicate that the third Mg2&#43; is coordinated and stabilized at its observed position. TL residues provide half of the substrate binding pocket while multiple TL/bridge helix (BH) interactions induce conformational changes that could propel translocation upon substrate hydrolysis. Consistent with TL/BH communication, a FEL structure and MD simulations of the hyperactive Rpb1 T834P bridge helix mutant reveals rearrangement of some active site interactions supporting potential plasticity in active site function and long-distance effects on both the width of the central channel and TL conformation, likely underlying its increased elongation rate at the expense of fidelity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561750">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561750" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561750">
        <p class="paperTitle">Mapping energy landscapes of homopolymeric RNAs via simulated tempering and deep unsupervised learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561750" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561750" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramachandran, V.; Potoyan, D. A.</p>
        <p class="info">Score: 4.8, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561750' target='https://doi.org/10.1101/2023.10.10.561750'> 10.1101/2023.10.10.561750</a></p>
        <p class="abstract">Conformational dynamics plays crucial roles in RNA functions about sensing and responding to environmental signals. The liquid-liquid phase separation of RNAs and the formation of stress granules partly relies on RNAs conformational plasticity and its ability to engage in multivalent interactions. Recent experiments with homopolymeric and low-complexity RNAs have revealed significant differences in phase separations due to differences in base chemistry of RNA units. We hypothesize that differences in RNA phase-transition dynamics can be traced back to the differences in conformational dynamics of single RNA chains. In the present contribution, we utilize atomistic simulations with numerous unsupervised learning to map temperature dependence conformational free energy landscapes for homopolymeric RNA chains. These landscapes reveal a variety of metastable excited states influenced by the nature of base chemistry. We shed light on the distinct contributions of the polyphosphate backbone versus base chemistry in shaping conformational ensembles of different RNAs. We demonstrate that the experimentally observed temperature-driven shifts in metastable state populations align with experimental phase diagrams for homopolymeric RNAs. The work establishes a microscopic framework to reason about base-specific RNA propensity for phase separation. We believe our work will be valuable for designing novel RNA sensors for biological and synthetic applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.07.561336">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.07.561336" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.07.561336">
        <p class="paperTitle">Molecular mechanisms of liposome interactions with bacterial envelopes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.07.561336" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.07.561336" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scheeder, A.; Brockhoff, M.; Ward, E.; Kaminski Schierle, G. S.; Mela, I.; Kaminski, C. F.</p>
        <p class="info">Score: 4.5, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.07.561336' target='https://doi.org/10.1101/2023.10.07.561336'> 10.1101/2023.10.07.561336</a></p>
        <p class="abstract">Although fusogenic liposomes offer a promising approach for the delivery of antibiotic payloads across the cell envelope of Gram-negative bacteria, there is still limited understanding of the individual nanocarrier interactions with the bacterial target. Using super-resolution microscopy, we characterize the interaction dynamics of positively charged fusogenic liposomes with Gram-negative (Escherichia coli) and Gram-positive (Bacillus subtilis) bacteria. The liposomes merge with the outer membrane (OM) of Gram-negative bacteria, while attachment or lipid internalization is observed in Gram-positive cells. Employing total internal reflection fluorescence microscopy, we demonstrated liposome fusion with model supported lipid bilayers. For whole E. coli cells, however, we observed heterogeneous membrane integrations, primarily involving liposome attachment and hemifusion events. With increasing lipopolysaccharide length the likelihood of full-fusion events was reduced. The integration of artificial lipids into the OM of Gram-negative cells led to membrane destabilization, resulting in decreased bacterial vitality, membrane detachment, and improved co-delivery of Vancomycin--an effective antibiotic against Gram-positive cells. These findings provide significant insights into the interactions of individual nanocarriers with bacterial envelopes at the single-cell level, uncovering effects that would be missed in bulk measurements. This highlights the importance of conducting single-particle and single-cell investigations to assess the performance of next-generation drug delivery platforms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.08.561435">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.08.561435" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.08.561435">
        <p class="paperTitle">Motion Blur Microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.08.561435" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.08.561435" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goreke, U.; Gonzales, A.; Shipley, B.; Man, Y.; Wulftange, W.; An, R.; Hinczewski, M.; Gurkan, U. A.</p>
        <p class="info">Score: 4.3, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.08.561435' target='https://doi.org/10.1101/2023.10.08.561435'> 10.1101/2023.10.08.561435</a></p>
        <p class="abstract">Imaging and characterizing the dynamics of cellular adhesion in blood samples is of fundamental importance in understanding biological function. In vitro microscopy methods are widely used for this task, but typically require diluting the blood with a buffer to allow for transmission of light. However whole blood provides crucial mechanical and chemical signaling cues that influence adhesion dynamics, which means that conventional approaches lack the full physiological complexity of living microvasculature. We propose to overcome this challenge by a new in vitro imaging method which we call motion blur microscopy (MBM). By decreasing the source light intensity and increasing the integration time during imaging, flowing cells are blurred, allowing us to identify adhered cells. Combined with an automated analysis using machine learning, we can for the first time reliably image cell interactions in microfluidic channels during whole blood flow. MBM provides a low cost, easy to implement alternative to intravital microscopy, the in vivo approach for studying how the whole blood environment shapes adhesion dynamics. We demonstrate the methods reproducibility and accuracy in two example systems where understanding cell interactions, adhesion, and motility is crucial--sickle red blood cells adhering to laminin, and CAR-T cells adhering to E-selectin. We illustrate the wide range of data types that can be extracted from this approach, including distributions of cell size and eccentricity, adhesion times, trajectories and velocities of adhered cells moving on a functionalized surface, as well as correlations among these different features at the single cell level. In all cases MBM allows for rapid collection and processing of large data sets, ranging from thousands to hundreds of thousands of individual adhesion events. The method is generalizable to study adhesion mechanisms in a variety of diseases, including cancer, blood disorders, thrombosis, inflammatory and autoimmune diseases, as well as providing rich datasets for theoretical modeling of adhesion dynamics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561899">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561899" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561899">
        <p class="paperTitle">Structural basis for CFTR inhibition by CFTRinh-172</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561899" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561899" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Young, P.; Levring, J.; Fiedorczuk, K.; Blanchard, S. C.; Chen, J.</p>
        <p class="info">Score: 2.6, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561899' target='https://doi.org/10.1101/2023.10.11.561899'> 10.1101/2023.10.11.561899</a></p>
        <p class="abstract">The cystic fibrosis transmembrane conductance regulator (CFTR) is an anion channel that regulates electrolyte and fluid balance in epithelial tissues. Whereas activation of CFTR is vital to treating cystic fibrosis, selective inhibition of CFTR is a potential therapeutic strategy for secretory diarrhea and autosomal dominant polycystic kidney disease (ADPKD). Although several CFTR inhibitors have been developed by high-throughput screening, their modes of action remain elusive. In this study, we determined the structure of CFTR in complex with the inhibitor CFTRinh-172 to 2.7 [A] resolution by cryogenic electron microscopy (cryo-EM). We observe that CFTRinh-172 binds inside the pore near transmembrane helix 8 (TM8), a critical structural element that links ATP hydrolysis with channel gating. Binding of CFTRinh-172 stabilizes a conformation in which the chloride selectivity filter is collapsed and the pore is blocked from the extracellular side of the membrane. Single molecule fluorescence resonance energy transfer (smFRET) experiments indicate that CFTRinh-172 inhibits channel gating without compromising nucleotide-binding domain (NBD) dimerization. Together, these data show that CFTRinh-172 acts as both a pore blocker and a gating modulator, setting it apart from typical ion channel inhibitors. The dual functionality of CFTRinh-172 reconciles previous biophysical observations and provides a molecular basis for its activity.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
