<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Single-molecule detection and super-resolution imaging with a portable and adaptable 3D-printed microscopy platform (Brick-MIC)
Authors: Moya Munoz, G. G.; Brix, O.; Klocke, P.; Wendler, N. D.; Lerner, E.; Zijlstra, N.; Cordes, T.
Score: 15.4, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573596
Over the past decades, single-molecule spectroscopy and super-resolution microscopy have advanced significantly and by now represent important tools for life science research. Despite rapid progress and ongoing development, there is a growing gap between the state-of-the-art and what is accessible to non-optics specialists, e.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Single-molecule detection and super-resolution imaging with a portable and adaptable 3D-printed microscopy platform (Brick-MIC)
Authors: Moya Munoz, G. G.; Brix, O.; Klocke, P.; Wendler, N. D.; Lerner, E.; Zijlstra, N.; Cordes, T.
Score: 15.4, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573596
Over the past decades, single-molecule spectroscopy and super-resolution microscopy have advanced significantly and by now represent important tools for life science research. Despite rapid progress and ongoing development, there is a growing gap between the state-of-the-art and what is accessible to non-optics specialists, e." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Single-molecule detection and super-resolution imaging with a portable and adaptable 3D-printed microscopy platform (Brick-MIC)
Authors: Moya Munoz, G. G.; Brix, O.; Klocke, P.; Wendler, N. D.; Lerner, E.; Zijlstra, N.; Cordes, T.
Score: 15.4, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573596
Over the past decades, single-molecule spectroscopy and super-resolution microscopy have advanced significantly and by now represent important tools for life science research. Despite rapid progress and ongoing development, there is a growing gap between the state-of-the-art and what is accessible to non-optics specialists, e."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Single-molecule detection and super-resolution imaging with a portable and adaptable 3D-printed microscopy platform (Brick-MIC)\nAuthors: Moya Munoz, G. G.; Brix, O.; Klocke, P.; Wendler, N. D.; Lerner, E.; Zijlstra, N.; Cordes, T.\nScore: 15.4, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573596\nOver the past decades, single-molecule spectroscopy and super-resolution microscopy have advanced significantly and by now represent important tools for life science research. Despite rapid progress and ongoing development, there is a growing gap between the state-of-the-art and what is accessible to non-optics specialists, e.",
  "keywords": [
    
  ],
  "articleBody": " Single-molecule detection and super-resolution imaging with a portable and adaptable 3D-printed microscopy platform (Brick-MIC)\nAuthors: Moya Munoz, G. G.; Brix, O.; Klocke, P.; Wendler, N. D.; Lerner, E.; Zijlstra, N.; Cordes, T.\nScore: 15.4, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573596\nOver the past decades, single-molecule spectroscopy and super-resolution microscopy have advanced significantly and by now represent important tools for life science research. Despite rapid progress and ongoing development, there is a growing gap between the state-of-the-art and what is accessible to non-optics specialists, e.g., biologists, biochemists, medical researchers, and labs with financial constraints. To bridge this gap, we introduce Brick-MIC, a versatile and affordable open-source 3D-printed micro-spectroscopy and imaging platform. Brick-MIC enables the integration of various fluorescence imaging techniques with single-molecule resolution within a single platform and enables exchange between different modalities within minutes. In this work, we present three variants of Brick-MIC that facilitate single-molecule fluorescence detection, fluorescence correlation spectroscopy and super-resolution imaging. With the three variants, we were able to observe conformational changes and absolute inter-dye distances in single macromolecules and perform single-molecule localization microscopy (STORM and PAINT) of DNA origami nanostructures. Detailed descriptions of the hardware and software components, as well as data analysis routines are provided, to allow non-optics specialist to operate their own Brick-MIC with minimal effort and investments. We foresee that our affordable, flexible, and open-source Brick-MIC platform will be a valuable tool for many laboratories worldwide.\nOblique Line Scan Illumination Enables Expansive, Accurate and Sensitive Single Protein Measurements in Solution and in Living Cells\nAuthors: Driouchi, A.; Bretan, M.; Davis, B. J.; Heckert, A. B.; Seeger, M.; Bradley Silva, M.; Forrest, W. S.; Hsiung, J.; Tan, J.; Yang, H.; McSwiggen, D. T.; Song, L.; Sule, A.; Abaie, B.; Conroy, B.; Elliott, L. A.; Gonzalez, E.; Ilkov, F.; Isaacs, J.; Labaria, G.; Lagana, M.; Larsen, D. D.; Margolin, B.; Nguyen, M. K.; Rine, J.; Tang, Y.; Vana, M.; Wilkey, A.; Zhang, Z.; Basham, S.; Ho, J. J.; Johnson, S.; Klammer, A. A.; Lin, K.; Darzacq, X.; Betzig, E.; Berman, R. T.; Anderson, D. J.\nScore: 8.1, Published: 2023-12-23 DOI: 10.1101/2023.12.21.571765\nSingle-molecule localization microscopy (SMLM) techniques, such as single-molecule tracking (SMT), enable in situ measurements in cells from which data-rich metrics can be extracted. SMT has been successfully applied to a variety of biological questions and model systems, aiming to unravel the spatiotemporal regulation of molecular mechanisms that govern protein function, downstream pathway effects, and cellular function. While powerful, SMLM often suffers from low throughput and illumination inhomogeneity, along with microscope and user-induced technical biases. Due to technical limitations in scaling SMLM techniques, a tradeoff between spatiotemporal resolution and throughput has been made historically, restricting broad application of these technologies. Here we address these limitations using Oblique Line Scan (OLS), a robust single-objective light-sheet based illumination and detection modality that achieves nanoscale spatial resolution and sub-millisecond temporal resolution across a 250 x 190 m field of view. We demonstrate OLS-enabled SMT on Halo-Tagged proteins in living cells capturing protein motion up to 14 m2 /s. By exploiting the adaptability of the acquisition frame rate and the improved rejection of out of focus light, we extend the utility of OLS beyond cellular compartments with in-solution SMT (isSMT) for single-molecule measurement of ligand-protein interactions and disruption of protein-protein interactions (PPI). We illustrate the versatility of OLS by showcasing two-color SMT, STORM, and single molecule fluorescence recovery after photobleaching (FRAP). OLS expands the range of SMLM applications and paves the way for robust, high-throughput single-molecule investigations of protein dynamics required for drug screening and systems biology studies, both in cells and in solution.\nEmergence of synchronised growth oscillations in filamentous fungi\nAuthors: Prakash, P.; Jiang, X.; Richards, L.; Schofield, Z.; Schafer, P.; Polin, M.; Soyer, O. s.; Asally, M. s.\nScore: 5.5, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573137\nSoil fungi are important decomposers of organic matter and play crucial roles in the biogeochemical cycles in the soil. Many species of fungi grow in the form of branched networks. While there have been investigations on the growth and architecture of the fungal networks, their growth dynamics in space and time is still not fully understood. In this study, we monitor the growth dynamics of the plant-promoting filamentous fungus Serendipita indica for several days in a controlled environment within a microfluidic channel. We find that this species displays synchronized growth oscillations with the onset of sporulation and at a period of 3 hours. Quantifying this experimental synchronisation of oscillatory dynamics, we show that the synchronisation can be captured by the nearest neighbour Kuramoto model. Our analysis suggested the existence of millimetre-scale cell-cell communication across the fungi network. The microfluidic setup presented in this work may aid the future characterization of the molecular mechanisms of the cell-cell communication, which could in turn be exploited in order to control fungi growth and reproductive sporulation in soil and plant health management.\nLive-cell imaging under centrifugation characterized the cellular force for nuclear centration in the Caenorhabditis elegans embryo\nAuthors: Goda, M.; Shribak, M.; Ikeda, Z.; Okada, N.; Tani, T.; Goshima, G.; Oldenbourg, R.; Kimura, A.\nScore: 4.0, Published: 2024-01-05 DOI: 10.1101/2024.01.03.574024\nOrganelles in cells are appropriately positioned, despite crowding in the cytoplasm. However, our understanding of the force required to move large organelles, such as the nucleus, inside the cytoplasm is limited, in part owing to a lack of accurate methods for measurement. We devised a novel method to apply forces to the nucleus of living, wild-type Caenorhabditis elegans embryos to measure the force generated inside the cell. We utilized a centrifuge polarizing microscope (CPM) to apply centrifugal force and orientation-independent differential interference contrast (OI-DIC) microscopy to characterize the mass density of the nucleus and cytoplasm. The cellular forces moving the nucleus toward the cell center increased linearly at [~]14 pN/m depending on the distance from the center. The frictional coefficient was [~]1,100 pN s/m. The measured values were smaller than previously reported estimates for sea urchin embryos. The forces were consistent with the centrosome-organelle mutual pulling model for nuclear centration. frictional coefficient was reduced when microtubules were shorter or detached from nuclei in mutant embryos, demonstrating the contribution of astral microtubules. Finally, the frictional coefficient was higher than a theoretical estimate, indicating the contribution of uncharacterized properties of the cytoplasm.\nStructural dynamics of the methyl-coenzyme M reductase active site are influenced by coenzyme F430 modifications\nAuthors: Poleto, M. D.; Allen, K. D.; Lemkul, J. A.\nScore: 3.5, Published: 2024-01-08 DOI: 10.1101/2024.01.07.574536\nMethyl-coenzyme M reductase (MCR) is a central player in methane biogeochemistry, governing methanogenesis and the anaerobic oxidation of methane (AOM) in methanogens and anaerobic methanotrophs (ANME), respectively. The prosthetic group of MCR is coenzyme F430, a nickel-containing tetrapyrrole derivative. Additionally, a few modified versions of F430 have been discovered, including the 172-methylthio-F430 (mt-F430) that functions with ANME-1 MCR. This study employs molecular dynamics (MD) simulations to unravel the intricacies of the active-site dynamics of MCR from Methanosarcina acetivorans and ANME-1 when bound to the canonical F430 compared to 172-thioether coenzyme F430 variants and substrates for methane formation. Overall, our simulations indicate that each MCR active site is optimized for a given version of F430 and support the importance of the Gln to Val substitution in accommodating the 172 methylthio modification. Notably, modifications in the 172 position disrupt the canonical coordination among cofactors in M. acetivorans MCR, implicating structural perturbations, but evidence of active site reorganization to maintain substrate positions suggest that the modified F430s could be accommodated in a methanogenic MCR. We additionally report the first quantitative estimate of MCR intrinsic electric fields pivotal in driving methane formation. Our results suggest that the electric field aligned along the CH3-S-CoM thioether bond facilitates homolytic bond cleavage, coinciding with the proposed catalytic mechanism. Structural perturbations, however, weaken and misalign these electric fields, emphasizing the significance of the active site structure in maintaining their integrity. In conclusion, our results deepen the understanding of MCR active-site dynamics, the enzyme's organizational role in intrinsic electric fields for catalysis, and the interplay between active site structure and electrostatics. This work not only advances our comprehension of MCR functionality but also provides a foundation for future investigations employing sophisticated models to capture the complex electronic properties of MCR active sites quantitatively.\nUnderstanding the Energy Landscape of Intrinsically Disordered Protein Ensembles\nAuthors: Viegas, R. G.; Martins, I. B. S.; LEITE, V. B. P.\nScore: 3.4, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574178\nA substantial portion of various organisms proteomes comprises intrinsically dis-ordered proteins (IDPs) that lack a defined three-dimensional structure. These IDPs exhibit a diverse array of conformations, displaying remarkable spatio-temporal het-erogeneity and exceptional conformational flexibility. Characterizing the structure or structural ensemble of IDPs presents significant conceptual and methodological challenges owing to the absence of a well-defined native structure. While databases such as the Protein Ensemble Database (PED) provide IDP ensembles obtained through a combination of experimental data and molecular modeling, the absence of reaction coordinates poses challenges in comprehensively understanding pertinent aspects of the system. In this study, we leverage the Energy Landscape Visualization Method (JCTC, 6482, 2019) to scrutinize four IDP ensembles sourced from PED. ELViM, a methodology that circumvents the need for a priori reaction coordinates, aids in analyzing the ensembles. The specific IDP ensembles investigated are as follows: two fragments of Nucleoporin (NUL: 884-993 and NUS: 1313-1390), Yeast Sic 1 N-terminal (1-90), and the N-terminal SH3 domain of Drk (1-59). Utilizing ELViM enables comprehensive validation of ensembles, facilitating the detection of potential inconsistencies in the sampling process. Additionally, it allows for identifying and characterizing the most prevalent conformations within an ensemble. Moreover, ELViM facilitates the comparative analysis of ensembles obtained under diverse conditions, thereby providing a powerful tool for investigating the functional mechanisms of IDPs.\nBalancing stability, dynamics and kinetics in phase separation of intrinsically disordered proteins\nAuthors: Zhang, G.; Chu, X.\nScore: 2.8, Published: 2024-01-07 DOI: 10.1101/2024.01.05.574441\nLiquid-liquid phase separation is a ubiquitous molecular phe-nomenon that plays crucial roles in a multitude of essential cellular activities. Intrinsically disordered proteins (IDPs), which lack well-defined three-dimensional structures, are prevalent participants in phase separation due to their inherent potential for promoting multivalent binding-the major driving force for this process. Understanding the underlying mechanisms of phase separation is challenging, as phase separation is a complex process, involving numerous molecules and various types of interactions. Here, we used a simplified coarse-grained model of IDPs to investigate the thermodynamic stability of the dense phase, conformational properties of IDPs, chain dynamics and kinetic rates of forming condensates. We focused on the IDP system, in which the oppositely charged IDPs are maximally segregated, inherently possessing a high propensity for phase separation. By varying interaction strengths, salt concentrations and temperatures, we observed that IDPs in the dense phase exhibited highly conserved conformational characteristics, which are more extended than those in the dilute phase. This implies that condensate formation acts as a protective shield, enabling IDPs to maintain conformational ensemble with high resistance to the changes in interactions and environmental conditions. Although the chain motions and global conformational dynamics of IDPs in the condensates are slow due to the high viscosity, local chain flexibility at the short timescales is largely preserved with respect to that at the free state. Strikingly, we observed a non-monotonic relationship between interaction strengths and kinetic rates for forming condensates. As strong interactions of IDPs result in high stable condensates, our results suggest that the thermodynamics and kinetics of phase separation are decoupled and optimized by the speed-stability balance through underlying molecular interactions. Our findings contribute to the molecular-level understanding of phase separation and offer valuable insights into the developments of engineering strategies for precise regulation of biomolecular condensates.\nThe everting Drosophila wing disc is a shape-programmed material\nAuthors: Fuhrmann, J. F.; Krishna, A.; Paijmans, J.; Duclut, C. A.; Eaton, S.; Popovic, M.; Julicher, F.; Modes, C. D.; Dye, N. A.\nScore: 9.6, Published: 2023-12-24 DOI: 10.1101/2023.12.23.573034\nHow complex 3D tissue shape emerges during animal development remains an important open question in biology and biophysics. Although it is well-established that tissue-scale patterns of in-plane cell behaviors can reshape a tissue in 2D, whether and how such patterns could also deform a tissue in 3D remains unclear. Meanwhile, recent advances in materials science, physics, and engineering have enhanced our ability to craft and understand inanimate \"shape-programmable\" materials capable of undergoing blueprinted 3D shape transformations arising from in-plane gradients of spontaneous strains. Here, we use this conceptual framework to understand eversion of the Drosophila wing disc, a complex 3D tissue reshaping event in wing development. Using quantification of spatial patterns of cell dynamics on an evolving 3D tissue surface, combined with physical modeling and genetic perturbation, we demonstrate that the wing can deform itself in 3D through large-scale patterning of in-plane cellular behaviors, primarily rearrangements. This work shows that concepts from shape-programming can be employed to understand animal tissue morphogenesis and suggests that there exist intricate patterns in nature that could present novel designs for shape-programmable materials.\nAn expanded view of ligandability in the allosteric enzyme PTP1B from computational reanalysis of large-scale crystallographic data\nAuthors: Mehlman, T.; Ginn, H. M.; Keedy, D. A.\nScore: 2.6, Published: 2024-01-07 DOI: 10.1101/2024.01.05.574428\nThe recent advent of crystallographic small-molecule fragment screening presents the opportunity to obtain unprecedented numbers of ligand-bound protein crystal structures from a single high-throughput experiment, mapping ligandability across protein surfaces and identifying useful chemical footholds for structure-based drug design. However, due to the low binding affinities of most fragments, detecting bound fragments from crystallographic datasets has been a challenge. Here we report a trove of 65 new fragment hits across 59 new liganded crystal structures for PTP1B, an \"undruggable\" therapeutic target enzyme for diabetes and cancer. These structures were obtained from computational analysis of data from a large crystallographic screen, demonstrating the power of this approach to elucidate many ([~]50% more) \"hidden\" ligand-bound states of proteins. Our new structures include a fragment hit found in a novel binding site in PTP1B with a unique location relative to the active site, one that validates another new binding site recently identified by simulations, one that links adjacent allosteric sites, and, perhaps most strikingly, a fragment that induces long-range allosteric protein conformational responses via a previously unreported intramolecular conduit. Altogether, our research highlights the utility of computational analysis of crystallographic data, makes publicly available dozens of new ligand-bound structures of a high-value drug target, and identifies novel aspects of ligandability and allostery in PTP1B.\nRNA regulates cohesiveness and porosity of a biological condensate\nAuthors: Chou, H.-Y.; Aksimentiev, A.\nScore: 2.5, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574811\nBiological condensates have emerged as key elements of a biological cell function, concentrating disparate biomolecules to accomplish specific biological tasks. RNA was identified as a key ingredient of such condensates, however, its effect on the physical properties of the condensate was found to depend on the condensate's composition while its effect on the microstructure has remained elusive. Here, we characterize the physical properties and the microstructure of a protein--RNA condensate by means of large-scale coarse-grained (CG) molecular dynamics simulations. By developing a custom CG model of RNA compatible with a popular CG model of proteins, we systematically investigate the structural, thermodynamic, and kinetic properties of condensate droplets containing thousands of individual protein and RNA molecules over a range of temperatures. While we find RNA to increase the condensate's cohesiveness, its effect on the condensate's fluidity is more nuanced with longer molecules compacting the condensate and making it less fluid. We show that a biological condensate has a sponge-like morphology of interconnected channels of size that increases with temperature and decreases in the presence of RNA. Our results suggest that longer RNA form a dynamic scaffold within a condensate, regulating not only its fluidity but also permeability to intruder molecules.\n",
  "wordCount" : "2599",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573596">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573596" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573596">
        <p class="paperTitle">Single-molecule detection and super-resolution imaging with a portable and adaptable 3D-printed microscopy platform (Brick-MIC)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573596" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573596" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moya Munoz, G. G.; Brix, O.; Klocke, P.; Wendler, N. D.; Lerner, E.; Zijlstra, N.; Cordes, T.</p>
        <p class="info">Score: 15.4, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573596' target='https://doi.org/10.1101/2023.12.29.573596'> 10.1101/2023.12.29.573596</a></p>
        <p class="abstract">Over the past decades, single-molecule spectroscopy and super-resolution microscopy have advanced significantly and by now represent important tools for life science research. Despite rapid progress and ongoing development, there is a growing gap between the state-of-the-art and what is accessible to non-optics specialists, e.g., biologists, biochemists, medical researchers, and labs with financial constraints. To bridge this gap, we introduce Brick-MIC, a versatile and affordable open-source 3D-printed micro-spectroscopy and imaging platform. Brick-MIC enables the integration of various fluorescence imaging techniques with single-molecule resolution within a single platform and enables exchange between different modalities within minutes. In this work, we present three variants of Brick-MIC that facilitate single-molecule fluorescence detection, fluorescence correlation spectroscopy and super-resolution imaging. With the three variants, we were able to observe conformational changes and absolute inter-dye distances in single macromolecules and perform single-molecule localization microscopy (STORM and PAINT) of DNA origami nanostructures. Detailed descriptions of the hardware and software components, as well as data analysis routines are provided, to allow non-optics specialist to operate their own Brick-MIC with minimal effort and investments. We foresee that our affordable, flexible, and open-source Brick-MIC platform will be a valuable tool for many laboratories worldwide.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.571765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.571765" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.571765">
        <p class="paperTitle">Oblique Line Scan Illumination Enables Expansive, Accurate and Sensitive Single Protein Measurements in Solution and in Living Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.571765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.571765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Driouchi, A.; Bretan, M.; Davis, B. J.; Heckert, A. B.; Seeger, M.; Bradley Silva, M.; Forrest, W. S.; Hsiung, J.; Tan, J.; Yang, H.; McSwiggen, D. T.; Song, L.; Sule, A.; Abaie, B.; Conroy, B.; Elliott, L. A.; Gonzalez, E.; Ilkov, F.; Isaacs, J.; Labaria, G.; Lagana, M.; Larsen, D. D.; Margolin, B.; Nguyen, M. K.; Rine, J.; Tang, Y.; Vana, M.; Wilkey, A.; Zhang, Z.; Basham, S.; Ho, J. J.; Johnson, S.; Klammer, A. A.; Lin, K.; Darzacq, X.; Betzig, E.; Berman, R. T.; Anderson, D. J.</p>
        <p class="info">Score: 8.1, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.571765' target='https://doi.org/10.1101/2023.12.21.571765'> 10.1101/2023.12.21.571765</a></p>
        <p class="abstract">Single-molecule localization microscopy (SMLM) techniques, such as single-molecule tracking (SMT), enable in situ measurements in cells from which data-rich metrics can be extracted. SMT has been successfully applied to a variety of biological questions and model systems, aiming to unravel the spatiotemporal regulation of molecular mechanisms that govern protein function, downstream pathway effects, and cellular function. While powerful, SMLM often suffers from low throughput and illumination inhomogeneity, along with microscope and user-induced technical biases. Due to technical limitations in scaling SMLM techniques, a tradeoff between spatiotemporal resolution and throughput has been made historically, restricting broad application of these technologies. Here we address these limitations using Oblique Line Scan (OLS), a robust single-objective light-sheet based illumination and detection modality that achieves nanoscale spatial resolution and sub-millisecond temporal resolution across a 250 x 190 m field of view. We demonstrate OLS-enabled SMT on Halo-Tagged proteins in living cells capturing protein motion up to 14 m2 /s. By exploiting the adaptability of the acquisition frame rate and the improved rejection of out of focus light, we extend the utility of OLS beyond cellular compartments with in-solution SMT (isSMT) for single-molecule measurement of ligand-protein interactions and disruption of protein-protein interactions (PPI). We illustrate the versatility of OLS by showcasing two-color SMT, STORM, and single molecule fluorescence recovery after photobleaching (FRAP). OLS expands the range of SMLM applications and paves the way for robust, high-throughput single-molecule investigations of protein dynamics required for drug screening and systems biology studies, both in cells and in solution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573137">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573137" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573137">
        <p class="paperTitle">Emergence of synchronised growth oscillations in filamentous fungi</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573137" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573137" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Prakash, P.; Jiang, X.; Richards, L.; Schofield, Z.; Schafer, P.; Polin, M.; Soyer, O. s.; Asally, M. s.</p>
        <p class="info">Score: 5.5, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573137' target='https://doi.org/10.1101/2023.12.22.573137'> 10.1101/2023.12.22.573137</a></p>
        <p class="abstract">Soil fungi are important decomposers of organic matter and play crucial roles in the biogeochemical cycles in the soil. Many species of fungi grow in the form of branched networks. While there have been investigations on the growth and architecture of the fungal networks, their growth dynamics in space and time is still not fully understood. In this study, we monitor the growth dynamics of the plant-promoting filamentous fungus Serendipita indica for several days in a controlled environment within a microfluidic channel. We find that this species displays synchronized growth oscillations with the onset of sporulation and at a period of 3 hours. Quantifying this experimental synchronisation of oscillatory dynamics, we show that the synchronisation can be captured by the nearest neighbour Kuramoto model. Our analysis suggested the existence of millimetre-scale cell-cell communication across the fungi network. The microfluidic setup presented in this work may aid the future characterization of the molecular mechanisms of the cell-cell communication, which could in turn be exploited in order to control fungi growth and reproductive sporulation in soil and plant health management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574024">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574024" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574024">
        <p class="paperTitle">Live-cell imaging under centrifugation characterized the cellular force for nuclear centration in the Caenorhabditis elegans embryo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574024" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574024" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goda, M.; Shribak, M.; Ikeda, Z.; Okada, N.; Tani, T.; Goshima, G.; Oldenbourg, R.; Kimura, A.</p>
        <p class="info">Score: 4.0, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574024' target='https://doi.org/10.1101/2024.01.03.574024'> 10.1101/2024.01.03.574024</a></p>
        <p class="abstract">Organelles in cells are appropriately positioned, despite crowding in the cytoplasm. However, our understanding of the force required to move large organelles, such as the nucleus, inside the cytoplasm is limited, in part owing to a lack of accurate methods for measurement. We devised a novel method to apply forces to the nucleus of living, wild-type Caenorhabditis elegans embryos to measure the force generated inside the cell. We utilized a centrifuge polarizing microscope (CPM) to apply centrifugal force and orientation-independent differential interference contrast (OI-DIC) microscopy to characterize the mass density of the nucleus and cytoplasm. The cellular forces moving the nucleus toward the cell center increased linearly at [~]14 pN/m depending on the distance from the center. The frictional coefficient was [~]1,100 pN s/m. The measured values were smaller than previously reported estimates for sea urchin embryos. The forces were consistent with the centrosome-organelle mutual pulling model for nuclear centration. frictional coefficient was reduced when microtubules were shorter or detached from nuclei in mutant embryos, demonstrating the contribution of astral microtubules. Finally, the frictional coefficient was higher than a theoretical estimate, indicating the contribution of uncharacterized properties of the cytoplasm.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.574536">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.574536" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.574536">
        <p class="paperTitle">Structural dynamics of the methyl-coenzyme M reductase active site are influenced by coenzyme F430 modifications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.574536" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.574536" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Poleto, M. D.; Allen, K. D.; Lemkul, J. A.</p>
        <p class="info">Score: 3.5, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.574536' target='https://doi.org/10.1101/2024.01.07.574536'> 10.1101/2024.01.07.574536</a></p>
        <p class="abstract">Methyl-coenzyme M reductase (MCR) is a central player in methane biogeochemistry, governing methanogenesis and the anaerobic oxidation of methane (AOM) in methanogens and anaerobic methanotrophs (ANME), respectively. The prosthetic group of MCR is coenzyme F430, a nickel-containing tetrapyrrole derivative. Additionally, a few modified versions of F430 have been discovered, including the 172-methylthio-F430 (mt-F430) that functions with ANME-1 MCR. This study employs molecular dynamics (MD) simulations to unravel the intricacies of the active-site dynamics of MCR from Methanosarcina acetivorans and ANME-1 when bound to the canonical F430 compared to 172-thioether coenzyme F430 variants and substrates for methane formation. Overall, our simulations indicate that each MCR active site is optimized for a given version of F430 and support the importance of the Gln to Val substitution in accommodating the 172 methylthio modification. Notably, modifications in the 172 position disrupt the canonical coordination among cofactors in M. acetivorans MCR, implicating structural perturbations, but evidence of active site reorganization to maintain substrate positions suggest that the modified F430s could be accommodated in a methanogenic MCR. We additionally report the first quantitative estimate of MCR intrinsic electric fields pivotal in driving methane formation. Our results suggest that the electric field aligned along the CH3-S-CoM thioether bond facilitates homolytic bond cleavage, coinciding with the proposed catalytic mechanism. Structural perturbations, however, weaken and misalign these electric fields, emphasizing the significance of the active site structure in maintaining their integrity. In conclusion, our results deepen the understanding of MCR active-site dynamics, the enzyme&#39;s organizational role in intrinsic electric fields for catalysis, and the interplay between active site structure and electrostatics. This work not only advances our comprehension of MCR functionality but also provides a foundation for future investigations employing sophisticated models to capture the complex electronic properties of MCR active sites quantitatively.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574178">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574178" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574178">
        <p class="paperTitle">Understanding the Energy Landscape of Intrinsically Disordered Protein Ensembles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574178" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574178" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Viegas, R. G.; Martins, I. B. S.; LEITE, V. B. P.</p>
        <p class="info">Score: 3.4, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574178' target='https://doi.org/10.1101/2024.01.04.574178'> 10.1101/2024.01.04.574178</a></p>
        <p class="abstract">A substantial portion of various organisms proteomes comprises intrinsically dis-ordered proteins (IDPs) that lack a defined three-dimensional structure. These IDPs exhibit a diverse array of conformations, displaying remarkable spatio-temporal het-erogeneity and exceptional conformational flexibility. Characterizing the structure or structural ensemble of IDPs presents significant conceptual and methodological challenges owing to the absence of a well-defined native structure. While databases such as the Protein Ensemble Database (PED) provide IDP ensembles obtained through a combination of experimental data and molecular modeling, the absence of reaction coordinates poses challenges in comprehensively understanding pertinent aspects of the system. In this study, we leverage the Energy Landscape Visualization Method (JCTC, 6482, 2019) to scrutinize four IDP ensembles sourced from PED. ELViM, a methodology that circumvents the need for a priori reaction coordinates, aids in analyzing the ensembles. The specific IDP ensembles investigated are as follows: two fragments of Nucleoporin (NUL: 884-993 and NUS: 1313-1390), Yeast Sic 1 N-terminal (1-90), and the N-terminal SH3 domain of Drk (1-59). Utilizing ELViM enables comprehensive validation of ensembles, facilitating the detection of potential inconsistencies in the sampling process. Additionally, it allows for identifying and characterizing the most prevalent conformations within an ensemble. Moreover, ELViM facilitates the comparative analysis of ensembles obtained under diverse conditions, thereby providing a powerful tool for investigating the functional mechanisms of IDPs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574441">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574441" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574441">
        <p class="paperTitle">Balancing stability, dynamics and kinetics in phase separation of intrinsically disordered proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574441" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574441" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, G.; Chu, X.</p>
        <p class="info">Score: 2.8, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574441' target='https://doi.org/10.1101/2024.01.05.574441'> 10.1101/2024.01.05.574441</a></p>
        <p class="abstract">Liquid-liquid phase separation is a ubiquitous molecular phe-nomenon that plays crucial roles in a multitude of essential cellular activities. Intrinsically disordered proteins (IDPs), which lack well-defined three-dimensional structures, are prevalent participants in phase separation due to their inherent potential for promoting multivalent binding-the major driving force for this process. Understanding the underlying mechanisms of phase separation is challenging, as phase separation is a complex process, involving numerous molecules and various types of interactions. Here, we used a simplified coarse-grained model of IDPs to investigate the thermodynamic stability of the dense phase, conformational properties of IDPs, chain dynamics and kinetic rates of forming condensates. We focused on the IDP system, in which the oppositely charged IDPs are maximally segregated, inherently possessing a high propensity for phase separation. By varying interaction strengths, salt concentrations and temperatures, we observed that IDPs in the dense phase exhibited highly conserved conformational characteristics, which are more extended than those in the dilute phase. This implies that condensate formation acts as a protective shield, enabling IDPs to maintain conformational ensemble with high resistance to the changes in interactions and environmental conditions. Although the chain motions and global conformational dynamics of IDPs in the condensates are slow due to the high viscosity, local chain flexibility at the short timescales is largely preserved with respect to that at the free state. Strikingly, we observed a non-monotonic relationship between interaction strengths and kinetic rates for forming condensates. As strong interactions of IDPs result in high stable condensates, our results suggest that the thermodynamics and kinetics of phase separation are decoupled and optimized by the speed-stability balance through underlying molecular interactions. Our findings contribute to the molecular-level understanding of phase separation and offer valuable insights into the developments of engineering strategies for precise regulation of biomolecular condensates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573034">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573034" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573034">
        <p class="paperTitle">The everting Drosophila wing disc is a shape-programmed material</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573034" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573034" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fuhrmann, J. F.; Krishna, A.; Paijmans, J.; Duclut, C. A.; Eaton, S.; Popovic, M.; Julicher, F.; Modes, C. D.; Dye, N. A.</p>
        <p class="info">Score: 9.6, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573034' target='https://doi.org/10.1101/2023.12.23.573034'> 10.1101/2023.12.23.573034</a></p>
        <p class="abstract">How complex 3D tissue shape emerges during animal development remains an important open question in biology and biophysics. Although it is well-established that tissue-scale patterns of in-plane cell behaviors can reshape a tissue in 2D, whether and how such patterns could also deform a tissue in 3D remains unclear. Meanwhile, recent advances in materials science, physics, and engineering have enhanced our ability to craft and understand inanimate &#34;shape-programmable&#34; materials capable of undergoing blueprinted 3D shape transformations arising from in-plane gradients of spontaneous strains. Here, we use this conceptual framework to understand eversion of the Drosophila wing disc, a complex 3D tissue reshaping event in wing development. Using quantification of spatial patterns of cell dynamics on an evolving 3D tissue surface, combined with physical modeling and genetic perturbation, we demonstrate that the wing can deform itself in 3D through large-scale patterning of in-plane cellular behaviors, primarily rearrangements. This work shows that concepts from shape-programming can be employed to understand animal tissue morphogenesis and suggests that there exist intricate patterns in nature that could present novel designs for shape-programmable materials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574428">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574428" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574428">
        <p class="paperTitle">An expanded view of ligandability in the allosteric enzyme PTP1B from computational reanalysis of large-scale crystallographic data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574428" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574428" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mehlman, T.; Ginn, H. M.; Keedy, D. A.</p>
        <p class="info">Score: 2.6, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574428' target='https://doi.org/10.1101/2024.01.05.574428'> 10.1101/2024.01.05.574428</a></p>
        <p class="abstract">The recent advent of crystallographic small-molecule fragment screening presents the opportunity to obtain unprecedented numbers of ligand-bound protein crystal structures from a single high-throughput experiment, mapping ligandability across protein surfaces and identifying useful chemical footholds for structure-based drug design. However, due to the low binding affinities of most fragments, detecting bound fragments from crystallographic datasets has been a challenge. Here we report a trove of 65 new fragment hits across 59 new liganded crystal structures for PTP1B, an &#34;undruggable&#34; therapeutic target enzyme for diabetes and cancer. These structures were obtained from computational analysis of data from a large crystallographic screen, demonstrating the power of this approach to elucidate many ([~]50% more) &#34;hidden&#34; ligand-bound states of proteins. Our new structures include a fragment hit found in a novel binding site in PTP1B with a unique location relative to the active site, one that validates another new binding site recently identified by simulations, one that links adjacent allosteric sites, and, perhaps most strikingly, a fragment that induces long-range allosteric protein conformational responses via a previously unreported intramolecular conduit. Altogether, our research highlights the utility of computational analysis of crystallographic data, makes publicly available dozens of new ligand-bound structures of a high-value drug target, and identifies novel aspects of ligandability and allostery in PTP1B.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574811">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574811" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574811">
        <p class="paperTitle">RNA regulates cohesiveness and porosity of a biological condensate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574811" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574811" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chou, H.-Y.; Aksimentiev, A.</p>
        <p class="info">Score: 2.5, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574811' target='https://doi.org/10.1101/2024.01.09.574811'> 10.1101/2024.01.09.574811</a></p>
        <p class="abstract">Biological condensates have emerged as key elements of a biological cell function, concentrating disparate biomolecules to accomplish specific biological tasks. RNA was identified as a key ingredient of such condensates, however, its effect on the physical properties of the condensate was found to depend on the condensate&#39;s composition while its effect on the microstructure has remained elusive. Here, we characterize the physical properties and the microstructure of a protein--RNA condensate by means of large-scale coarse-grained (CG) molecular dynamics simulations. By developing a custom CG model of RNA compatible with a popular CG model of proteins, we systematically investigate the structural, thermodynamic, and kinetic properties of condensate droplets containing thousands of individual protein and RNA molecules over a range of temperatures. While we find RNA to increase the condensate&#39;s cohesiveness, its effect on the condensate&#39;s fluidity is more nuanced with longer molecules compacting the condensate and making it less fluid. We show that a biological condensate has a sponge-like morphology of interconnected channels of size that increases with temperature and decreases in the presence of RNA. Our results suggest that longer RNA form a dynamic scaffold within a condensate, regulating not only its fluidity but also permeability to intruder molecules.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
