<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Approximating conformational Boltzmann distributions with AlphaFold2 predictions
Authors: Brown, B. P.; Stein, R. A.; Meiler, J.; Mchaourab, H.
Score: 26.2, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552168
Protein dynamics are intimately tied to biological function and can enable processes such as signal transduction, enzyme catalysis, and molecular recognition. The relative free energies of conformations that contribute to these functional equilibria are evolved for the physiology of the organism. Despite the importance of these equilibria for understanding biological function and developing treatments for disease, the computational and experimental methods capable of quantifying them are limited to systems of modest size.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Approximating conformational Boltzmann distributions with AlphaFold2 predictions
Authors: Brown, B. P.; Stein, R. A.; Meiler, J.; Mchaourab, H.
Score: 26.2, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552168
Protein dynamics are intimately tied to biological function and can enable processes such as signal transduction, enzyme catalysis, and molecular recognition. The relative free energies of conformations that contribute to these functional equilibria are evolved for the physiology of the organism. Despite the importance of these equilibria for understanding biological function and developing treatments for disease, the computational and experimental methods capable of quantifying them are limited to systems of modest size." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-16T10:37:20+00:00" />
<meta property="article:modified_time" content="2023-08-16T10:37:20+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Approximating conformational Boltzmann distributions with AlphaFold2 predictions
Authors: Brown, B. P.; Stein, R. A.; Meiler, J.; Mchaourab, H.
Score: 26.2, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552168
Protein dynamics are intimately tied to biological function and can enable processes such as signal transduction, enzyme catalysis, and molecular recognition. The relative free energies of conformations that contribute to these functional equilibria are evolved for the physiology of the organism. Despite the importance of these equilibria for understanding biological function and developing treatments for disease, the computational and experimental methods capable of quantifying them are limited to systems of modest size."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Approximating conformational Boltzmann distributions with AlphaFold2 predictions\nAuthors: Brown, B. P.; Stein, R. A.; Meiler, J.; Mchaourab, H.\nScore: 26.2, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552168\nProtein dynamics are intimately tied to biological function and can enable processes such as signal transduction, enzyme catalysis, and molecular recognition. The relative free energies of conformations that contribute to these functional equilibria are evolved for the physiology of the organism. Despite the importance of these equilibria for understanding biological function and developing treatments for disease, the computational and experimental methods capable of quantifying them are limited to systems of modest size.",
  "keywords": [
    
  ],
  "articleBody": " Approximating conformational Boltzmann distributions with AlphaFold2 predictions\nAuthors: Brown, B. P.; Stein, R. A.; Meiler, J.; Mchaourab, H.\nScore: 26.2, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552168\nProtein dynamics are intimately tied to biological function and can enable processes such as signal transduction, enzyme catalysis, and molecular recognition. The relative free energies of conformations that contribute to these functional equilibria are evolved for the physiology of the organism. Despite the importance of these equilibria for understanding biological function and developing treatments for disease, the computational and experimental methods capable of quantifying them are limited to systems of modest size. Here, we demonstrate that AlphaFold2 contact distance distributions can approximate conformational Boltzmann distributions, which we evaluate through examination of the joint probability distributions of inter-residue contact distances along functionally relevant collective variables of several protein systems. Further, we show that contact distance probability distributions generated by AlphaFold2 are sensitive to points mutations thus AF2 can predict the structural effects of mutations in some systems. We anticipate that our approach will be a valuable tool to model the thermodynamics of conformational changes in large biomolecular systems.\nGlutamate helps unmask the differences in driving forces for phase separation versus clustering of FET family proteins in sub-saturated solutions\nAuthors: Kar, M.; Vogel, L. T.; Chauhan, G.; Ausserwoger, H.; Welsh, T. J.; Kamath, A. R.; Knowles, T. P. J.; Hyman, A. A.; Seidel, C. A. M.; Pappu, R. V.\nScore: 17.6, Published: 2023-08-13 DOI: 10.1101/2023.08.11.552963\nMultivalent proteins undergo coupled segregative and associative phase transitions. Phase separation, a segregative transition, is driven by macromolecular solubility, and this leads to coexisting phases above system-specific saturation concentrations. Percolation is a continuous transition that is driven by multivalent associations among cohesive motifs. Contributions from percolation are highlighted by the formation of heterogeneous distributions of clusters in sub-saturated solutions, as was recently reported for Fused in sarcoma (FUS) and FET family proteins. Here, we show that clustering and phase separation are defined by a separation of length- and energy-scales. This is unmasked when glutamate is the primary solution anion. Glutamate is preferentially excluded from protein sites, and this enhances molecular associations. Differences between glutamate and chloride are manifest at ultra-low protein concentrations. These differences are amplified as concentrations increase, and they saturate as the micron-scale is approached. Therefore, condensate formation in supersaturated solutions and clustering in sub-saturated are governed by distinct energy and length scales. Glutamate, unlike chloride, is the dominant intracellular anion, and the separation of scales, which is masked in chloride, is unmasked in glutamate. Our work highlights how components of cellular milieus and sequence-encoded interactions contribute to amplifying distinct contributions from associative versus segregative phase transitions.\nMesoscopic Oblique Plane Microscopy via Light-sheet Mirroring\nAuthors: Daetwyler, S.; Chang, B.-J.; Chen, B.; Zhou, F. Y.; Fiolka, R. P.\nScore: 16.5, Published: 2023-08-12 DOI: 10.1101/2023.08.10.552834\nUnderstanding the intricate interplay and inter-connectivity of biological processes across an entire organism is important in various fields of biology, including cardiovascular research, neuroscience, and developmental biology. Here, we present a mesoscopic oblique plane microscope (OPM) that enables whole organism imaging with high speed and subcellular resolution. A microprism underneath the sample enhances the axial resolution and optical sectioning through total internal reflection of the light-sheet. Through rapid refocusing of the light-sheet, the imaging depth is extended up to threefold while keeping the axial resolution constant. Using low magnification objectives with a large field of view, we realize mesoscopic imaging over a volume of 3.7x1.5x1 mm3 with [~]2.3 microns lateral and [~]9.2 microns axial resolution. Applying the mesoscopic OPM, we demonstrate in vivo and in toto whole organism imaging of the zebrafish vasculature and its endothelial nuclei, and blood flow dynamics at 12 Hz acquisition rate, resulting in a quantitative map of blood flow across the entire organism.\nDifferences in Conformational Sampling and Intrinsic Electric Fields Drive Ion Binding in Telomeric and TERRA G-Quadruplexes\nAuthors: Poleto, M. D.; Lemkul, J. A.\nScore: 8.8, Published: 2023-08-14 DOI: 10.1101/2023.08.10.552810\nThe formation of G-quadruplexes (GQs) occurs in guanine rich sequences of DNA and RNA, producing highly stable and structurally diverse noncanonical nucleic acid structures. GQs play crucial roles in regulating transcription, translation, and replication; and maintaining the genome, among others, thus changes to their structures can lead to diseases such as cancer. Previous studies using polarizable molecular dynamics simulations have shown differences in ion binding properties between telomeric and TERRA GQs despite architectural similarities. Here, we used volume-based metadynamics and repulsive potential simulations in conjunction with polarizable force fields to quantify the impact of ion binding on GQ dynamics and ion binding free energies. Furthermore, we describe how GQs exert electric fields on their surroundings to link dynamics with variations in electronic structure. Our findings provide new insights into the energetic, physical, and conformational properties of GQs and expose subtle, but important, differences between DNA and RNA GQs with the same fold.\nGraph Attention Site Prediction (GrASP): Identifying Druggable Binding Sites Using Graph Neural Networks with Attention\nAuthors: Smith, Z.; Strobel, M.; Vani, B.; Tiwary, P.\nScore: 44.3, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550565\nIdentifying and discovering druggable protein binding sites is an important early step in computer-aided drug discovery but remains a difficult task where most campaigns rely on a priori knowledge of binding sites from experiments. Here we present a novel binding site prediction method called Graph Attention Site Prediction (GrASP) and re-evaluate assumptions in nearly every step in the site prediction workflow from dataset preparation to model evaluation. GrASP is able to achieve state-of-the-art performance at recovering binding sites in PDB structures while maintaining a high degree of precision which will minimize wasted computation in downstream tasks such as docking and free energy perturbation.\nFitting a square beam in a square camera: novel condensor apertures for low-dose transmission electron microscopy\nAuthors: Brown, H. G.; Smith, D.; Wardle, B. C.; Hanssen, E.\nScore: 6.9, Published: 2023-08-16 DOI: 10.1101/2023.08.13.553155\nIn transmission electron microscopy (TEM) cameras are square or rectangular but beams are round. With a beam size chosen to fill the camera at a given image magnification, the circular lobes of the beam will extend beyond the camera's field of view and irradiate areas that are not acquired on the camera, damaging and precluding them from future acquisitions if the sample is beam sensitive. In this paper we present development of condenser aperture plates for TEM that have square and rectangular apertures which improve the efficiency of sample area usage by 44% or greater in low dose TEM applications. We demonstrate that the use of these apertures is compatible with high-resolution cryogenic (cryo) TEM by reconstructing sub 2 [A] apo-ferritin models from a datasets recorded with both square and rectangular apertures. Moreover the design of our aperture plates should improve the flexibility of 2 condenser systems for cryo-TEM acquisitions with multiple shots per hole by tailored matching of beam sizes to camera sizes at each magnification.\nConserved molecular recognition by an intrinsically disordered region in the absence of sequence conservation\nAuthors: Alston, J.; Soranno, A.; Holehouse, A. S.\nScore: 17.8, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552128\nIntrinsically disordered regions (IDRs) are critical for cellular function, yet often appear to lack sequence conservation when assessed by multiple sequence alignments. This raises the question of if and how function can be encoded and preserved in these regions despite massive sequence variation. To address this question, we have applied coarse-grained molecular dynamics simulations to investigate non-specific RNA binding of coronavirus nucleocapsid proteins. Coronavirus nucleocapsid proteins consist of multiple interspersed disordered and folded domains that bind RNA. We focussed here on the first two domains of coronavirus nucleocapsid proteins, the disordered N-terminal domain (NTD) followed by the folded RNA binding domain (RBD). While the NTD is highly variable across evolution, the RBD is structurally conserved. This combination makes the NTD-RBD a convenient model system to explore the interplay between an IDR adjacent to a folded domain, and how changes in IDR sequence can influence molecular recognition of a partner. Our results reveal a surprising degree of sequence-specificity encoded by both the composition and the precise order of the amino acids in the NTD. The presence of an NTD can - depending on the sequence - either suppress or enhance RNA binding. Despite this sensitivity, large-scale variation in NTD sequences is possible while certain sequence features are retained. Consequently, a conformationally-conserved fuzzy RNA:protein complex is found across nucleocapsid protein orthologs, despite large-scale changes in both NTD sequence and RBD surface chemistry. Taken together, these insights shed light on the ability of disordered regions to preserve functional characteristics despite their sequence variability.\nVisualizing a two-state conformational ensemble in stem-loop 3 of the transcriptional regulator 7SK RNA\nAuthors: Camara, M. B.; Lange, B.; Yesselman, J. D.; Eichhorn, C. D.\nScore: 5.7, Published: 2023-08-12 DOI: 10.1101/2023.08.09.552709\nStructural plasticity is integral to RNA function; however, there are currently few methods to quantitatively resolve RNAs that have multiple structural states. NMR spectroscopy is a powerful approach for resolving conformational ensembles but is size-limited. Chemical probing is well-suited for large RNAs but provides limited structural and no kinetics information. Here, we integrate the two approaches to visualize a two-state conformational ensemble for the central stem-loop 3 (SL3) of 7SK RNA, a critical element for 7SK RNA function in transcription regulation. We find that the SL3 distal end exchanges between two equally populated yet structurally distinct states in both isolated SL3 constructs and full-length 7SK RNA. We rationally designed constructs that lock SL3 into a single state and demonstrate that both chemical probing and NMR data fit to a linear combination of the two states. Comparison of vertebrate 7SK RNA sequences shows conservation of both states, suggesting functional importance. These results provide new insights into 7SK RNA structural dynamics and demonstrate the utility of integrating chemical probing with NMR spectroscopy to gain quantitative insights into RNA conformational ensembles.\nPhotolipid excitation triggers depolarizing optocapacitive currents and action potentials\nAuthors: Bassetto, C. A. Z.; Pfeffermann, J.; Yadav, R.; Strassgschwandtner, S.; Glasnov, T.; Bezanilla, F.; Pohl, P.\nScore: 5.1, Published: 2023-08-14 DOI: 10.1101/2023.08.11.552849\nOptically-induced changes in membrane capacitance may regulate neuronal activity without requiring genetic modifications. Previously, they mainly relied on sudden temperature jumps due to light absorption by membrane-associated nanomaterials or water. Yet, nanomaterial targeting or the required high infrared light intensities obstruct broad applicability. Now, we propose a very versatile approach: photolipids (azobenzene-containing diacylglycerols) mediate light-triggered cellular de- or hyperpolarization. As planar bilayer experiments show, the respective currents emerge from millisecond-timescale changes in bilayer capacitance. UV light changes photolipid conformation, which awards embedding plasma membranes with increased capacitance and evokes depolarizing currents. They open voltage-gated sodium channels in cells, generating action potentials. Blue light reduces the area per photolipid, decreasing membrane capacitance and eliciting hyperpolarization. If present, mechanosensitive channels respond to the increased mechanical membrane tension, generating large depolarizing currents that elicit action potentials. Membrane self-insertion of administered photolipids and focused illumination allows cell excitation with high spatiotemporal control.\nFRET-guided modeling of nucleic acids\nAuthors: Steffen, F. D.; Cunha, R. A.; Sigel, R. K. O.; Borner, R.\nScore: 4.5, Published: 2023-08-07 DOI: 10.1101/2023.08.07.552238\nThe functional diversity of RNA is encoded in their innate conformational heterogeneity. The combination of single-molecule spectroscopy and computational modeling offers new, attractive opportunities to map structural transitions within nucleic acid ensembles. Here, we describe a framework to harmonize single-molecule FRET measurements with molecular dynamics simulations and de novo structure prediction. Using either all-atom or implicit fluorophore modeling we recreate FRET experiments in silico, visualize the underlying structural dynamics and quantify the simulated reaction coordinates. Using multiple accessible-contact volumes (multi-ACV) as a post-hoc scoring method for fragment-assembly in Rosetta, we demonstrate that FRET effectively refines de novo RNA structure prediction without the need of explicit dye labeling. We benchmark our FRET-assisted modeling approach on double-labeled DNA strands and validate it against an intrinsically dynamic manganese(II)-binding riboswitch. We show that a FRET coordinate describing the assembly of a four-way junction allows our pipeline to recapitulate the global fold of the riboswitch with sub-helical accuracy to the crystal structure. We conclude that computational fluorescence spectroscopy facilitates the interpretability of dynamic structural ensembles and improves the mechanistic understanding of nucleic acid interactions. Graphical abstractSchematic workflow of integrative FRET modeling using all-atom fluorophores or an accessible-contact volume dye model. All-atom molecular dynamics track the dye coordinate explicitly as part of the simulation while multi-ACV infer mean dye positions post hoc. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=150 SRC=\"FIGDIR/small/552238v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (81K): org.highwire.dtl.DTLVardef@d788d2org.highwire.dtl.DTLVardef@57936corg.highwire.dtl.DTLVardef@18fbb88org.highwire.dtl.DTLVardef@160ede8_HPS_FORMAT_FIGEXP M_FIG C_FIG\n",
  "wordCount" : "2056",
  "inLanguage": "en",
  "datePublished": "2023-08-16T10:37:20Z",
  "dateModified": "2023-08-16T10:37:20Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on August 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552168">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552168" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552168">
        <p class="paperTitle">Approximating conformational Boltzmann distributions with AlphaFold2 predictions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552168" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552168" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brown, B. P.; Stein, R. A.; Meiler, J.; Mchaourab, H.</p>
        <p class="info">Score: 26.2, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552168' target='https://doi.org/10.1101/2023.08.06.552168'> 10.1101/2023.08.06.552168</a></p>
        <p class="abstract">Protein dynamics are intimately tied to biological function and can enable processes such as signal transduction, enzyme catalysis, and molecular recognition. The relative free energies of conformations that contribute to these functional equilibria are evolved for the physiology of the organism. Despite the importance of these equilibria for understanding biological function and developing treatments for disease, the computational and experimental methods capable of quantifying them are limited to systems of modest size. Here, we demonstrate that AlphaFold2 contact distance distributions can approximate conformational Boltzmann distributions, which we evaluate through examination of the joint probability distributions of inter-residue contact distances along functionally relevant collective variables of several protein systems. Further, we show that contact distance probability distributions generated by AlphaFold2 are sensitive to points mutations thus AF2 can predict the structural effects of mutations in some systems. We anticipate that our approach will be a valuable tool to model the thermodynamics of conformational changes in large biomolecular systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552963" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552963">
        <p class="paperTitle">Glutamate helps unmask the differences in driving forces for phase separation versus clustering of FET family proteins in sub-saturated solutions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kar, M.; Vogel, L. T.; Chauhan, G.; Ausserwoger, H.; Welsh, T. J.; Kamath, A. R.; Knowles, T. P. J.; Hyman, A. A.; Seidel, C. A. M.; Pappu, R. V.</p>
        <p class="info">Score: 17.6, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552963' target='https://doi.org/10.1101/2023.08.11.552963'> 10.1101/2023.08.11.552963</a></p>
        <p class="abstract">Multivalent proteins undergo coupled segregative and associative phase transitions. Phase separation, a segregative transition, is driven by macromolecular solubility, and this leads to coexisting phases above system-specific saturation concentrations. Percolation is a continuous transition that is driven by multivalent associations among cohesive motifs. Contributions from percolation are highlighted by the formation of heterogeneous distributions of clusters in sub-saturated solutions, as was recently reported for Fused in sarcoma (FUS) and FET family proteins. Here, we show that clustering and phase separation are defined by a separation of length- and energy-scales. This is unmasked when glutamate is the primary solution anion. Glutamate is preferentially excluded from protein sites, and this enhances molecular associations. Differences between glutamate and chloride are manifest at ultra-low protein concentrations. These differences are amplified as concentrations increase, and they saturate as the micron-scale is approached. Therefore, condensate formation in supersaturated solutions and clustering in sub-saturated are governed by distinct energy and length scales. Glutamate, unlike chloride, is the dominant intracellular anion, and the separation of scales, which is masked in chloride, is unmasked in glutamate. Our work highlights how components of cellular milieus and sequence-encoded interactions contribute to amplifying distinct contributions from associative versus segregative phase transitions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552834">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552834" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552834">
        <p class="paperTitle">Mesoscopic Oblique Plane Microscopy via Light-sheet Mirroring</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552834" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552834" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Daetwyler, S.; Chang, B.-J.; Chen, B.; Zhou, F. Y.; Fiolka, R. P.</p>
        <p class="info">Score: 16.5, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552834' target='https://doi.org/10.1101/2023.08.10.552834'> 10.1101/2023.08.10.552834</a></p>
        <p class="abstract">Understanding the intricate interplay and inter-connectivity of biological processes across an entire organism is important in various fields of biology, including cardiovascular research, neuroscience, and developmental biology. Here, we present a mesoscopic oblique plane microscope (OPM) that enables whole organism imaging with high speed and subcellular resolution. A microprism underneath the sample enhances the axial resolution and optical sectioning through total internal reflection of the light-sheet. Through rapid refocusing of the light-sheet, the imaging depth is extended up to threefold while keeping the axial resolution constant. Using low magnification objectives with a large field of view, we realize mesoscopic imaging over a volume of 3.7x1.5x1 mm3 with [~]2.3 microns lateral and [~]9.2 microns axial resolution. Applying the mesoscopic OPM, we demonstrate in vivo and in toto whole organism imaging of the zebrafish vasculature and its endothelial nuclei, and blood flow dynamics at 12 Hz acquisition rate, resulting in a quantitative map of blood flow across the entire organism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552810">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552810" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552810">
        <p class="paperTitle">Differences in Conformational Sampling and Intrinsic Electric Fields Drive Ion Binding in Telomeric and TERRA G-Quadruplexes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552810" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552810" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Poleto, M. D.; Lemkul, J. A.</p>
        <p class="info">Score: 8.8, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552810' target='https://doi.org/10.1101/2023.08.10.552810'> 10.1101/2023.08.10.552810</a></p>
        <p class="abstract">The formation of G-quadruplexes (GQs) occurs in guanine rich sequences of DNA and RNA, producing highly stable and structurally diverse noncanonical nucleic acid structures. GQs play crucial roles in regulating transcription, translation, and replication; and maintaining the genome, among others, thus changes to their structures can lead to diseases such as cancer. Previous studies using polarizable molecular dynamics simulations have shown differences in ion binding properties between telomeric and TERRA GQs despite architectural similarities. Here, we used volume-based metadynamics and repulsive potential simulations in conjunction with polarizable force fields to quantify the impact of ion binding on GQ dynamics and ion binding free energies. Furthermore, we describe how GQs exert electric fields on their surroundings to link dynamics with variations in electronic structure. Our findings provide new insights into the energetic, physical, and conformational properties of GQs and expose subtle, but important, differences between DNA and RNA GQs with the same fold.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550565">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550565" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550565">
        <p class="paperTitle">Graph Attention Site Prediction (GrASP): Identifying Druggable Binding Sites Using Graph Neural Networks with Attention</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550565" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550565" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, Z.; Strobel, M.; Vani, B.; Tiwary, P.</p>
        <p class="info">Score: 44.3, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550565' target='https://doi.org/10.1101/2023.07.25.550565'> 10.1101/2023.07.25.550565</a></p>
        <p class="abstract">Identifying and discovering druggable protein binding sites is an important early step in computer-aided drug discovery but remains a difficult task where most campaigns rely on a priori knowledge of binding sites from experiments. Here we present a novel binding site prediction method called Graph Attention Site Prediction (GrASP) and re-evaluate assumptions in nearly every step in the site prediction workflow from dataset preparation to model evaluation. GrASP is able to achieve state-of-the-art performance at recovering binding sites in PDB structures while maintaining a high degree of precision which will minimize wasted computation in downstream tasks such as docking and free energy perturbation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553155">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553155" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553155">
        <p class="paperTitle">Fitting a square beam in a square camera: novel condensor apertures for low-dose transmission electron microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553155" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553155" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brown, H. G.; Smith, D.; Wardle, B. C.; Hanssen, E.</p>
        <p class="info">Score: 6.9, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553155' target='https://doi.org/10.1101/2023.08.13.553155'> 10.1101/2023.08.13.553155</a></p>
        <p class="abstract">In transmission electron microscopy (TEM) cameras are square or rectangular but beams are round. With a beam size chosen to fill the camera at a given image magnification, the circular lobes of the beam will extend beyond the camera&#39;s field of view and irradiate areas that are not acquired on the camera, damaging and precluding them from future acquisitions if the sample is beam sensitive. In this paper we present development of condenser aperture plates for TEM that have square and rectangular apertures which improve the efficiency of sample area usage by 44% or greater in low dose TEM applications. We demonstrate that the use of these apertures is compatible with high-resolution cryogenic (cryo) TEM by reconstructing sub 2 [A] apo-ferritin models from a datasets recorded with both square and rectangular apertures. Moreover the design of our aperture plates should improve the flexibility of 2 condenser systems for cryo-TEM acquisitions with multiple shots per hole by tailored matching of beam sizes to camera sizes at each magnification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552128">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552128" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552128">
        <p class="paperTitle">Conserved molecular recognition by an intrinsically disordered region in the absence of sequence conservation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552128" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552128" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alston, J.; Soranno, A.; Holehouse, A. S.</p>
        <p class="info">Score: 17.8, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552128' target='https://doi.org/10.1101/2023.08.06.552128'> 10.1101/2023.08.06.552128</a></p>
        <p class="abstract">Intrinsically disordered regions (IDRs) are critical for cellular function, yet often appear to lack sequence conservation when assessed by multiple sequence alignments. This raises the question of if and how function can be encoded and preserved in these regions despite massive sequence variation. To address this question, we have applied coarse-grained molecular dynamics simulations to investigate non-specific RNA binding of coronavirus nucleocapsid proteins. Coronavirus nucleocapsid proteins consist of multiple interspersed disordered and folded domains that bind RNA. We focussed here on the first two domains of coronavirus nucleocapsid proteins, the disordered N-terminal domain (NTD) followed by the folded RNA binding domain (RBD). While the NTD is highly variable across evolution, the RBD is structurally conserved. This combination makes the NTD-RBD a convenient model system to explore the interplay between an IDR adjacent to a folded domain, and how changes in IDR sequence can influence molecular recognition of a partner. Our results reveal a surprising degree of sequence-specificity encoded by both the composition and the precise order of the amino acids in the NTD. The presence of an NTD can - depending on the sequence - either suppress or enhance RNA binding. Despite this sensitivity, large-scale variation in NTD sequences is possible while certain sequence features are retained. Consequently, a conformationally-conserved fuzzy RNA:protein complex is found across nucleocapsid protein orthologs, despite large-scale changes in both NTD sequence and RBD surface chemistry. Taken together, these insights shed light on the ability of disordered regions to preserve functional characteristics despite their sequence variability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552709">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552709" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552709">
        <p class="paperTitle">Visualizing a two-state conformational ensemble in stem-loop 3 of the transcriptional regulator 7SK RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552709" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552709" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Camara, M. B.; Lange, B.; Yesselman, J. D.; Eichhorn, C. D.</p>
        <p class="info">Score: 5.7, Published: 2023-08-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552709' target='https://doi.org/10.1101/2023.08.09.552709'> 10.1101/2023.08.09.552709</a></p>
        <p class="abstract">Structural plasticity is integral to RNA function; however, there are currently few methods to quantitatively resolve RNAs that have multiple structural states. NMR spectroscopy is a powerful approach for resolving conformational ensembles but is size-limited. Chemical probing is well-suited for large RNAs but provides limited structural and no kinetics information. Here, we integrate the two approaches to visualize a two-state conformational ensemble for the central stem-loop 3 (SL3) of 7SK RNA, a critical element for 7SK RNA function in transcription regulation. We find that the SL3 distal end exchanges between two equally populated yet structurally distinct states in both isolated SL3 constructs and full-length 7SK RNA. We rationally designed constructs that lock SL3 into a single state and demonstrate that both chemical probing and NMR data fit to a linear combination of the two states. Comparison of vertebrate 7SK RNA sequences shows conservation of both states, suggesting functional importance. These results provide new insights into 7SK RNA structural dynamics and demonstrate the utility of integrating chemical probing with NMR spectroscopy to gain quantitative insights into RNA conformational ensembles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552849">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552849" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552849">
        <p class="paperTitle">Photolipid excitation triggers depolarizing optocapacitive currents and action potentials</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552849" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552849" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bassetto, C. A. Z.; Pfeffermann, J.; Yadav, R.; Strassgschwandtner, S.; Glasnov, T.; Bezanilla, F.; Pohl, P.</p>
        <p class="info">Score: 5.1, Published: 2023-08-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552849' target='https://doi.org/10.1101/2023.08.11.552849'> 10.1101/2023.08.11.552849</a></p>
        <p class="abstract">Optically-induced changes in membrane capacitance may regulate neuronal activity without requiring genetic modifications. Previously, they mainly relied on sudden temperature jumps due to light absorption by membrane-associated nanomaterials or water. Yet, nanomaterial targeting or the required high infrared light intensities obstruct broad applicability. Now, we propose a very versatile approach: photolipids (azobenzene-containing diacylglycerols) mediate light-triggered cellular de- or hyperpolarization. As planar bilayer experiments show, the respective currents emerge from millisecond-timescale changes in bilayer capacitance. UV light changes photolipid conformation, which awards embedding plasma membranes with increased capacitance and evokes depolarizing currents. They open voltage-gated sodium channels in cells, generating action potentials. Blue light reduces the area per photolipid, decreasing membrane capacitance and eliciting hyperpolarization. If present, mechanosensitive channels respond to the increased mechanical membrane tension, generating large depolarizing currents that elicit action potentials. Membrane self-insertion of administered photolipids and focused illumination allows cell excitation with high spatiotemporal control.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552238">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552238" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552238">
        <p class="paperTitle">FRET-guided modeling of nucleic acids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552238" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552238" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Steffen, F. D.; Cunha, R. A.; Sigel, R. K. O.; Borner, R.</p>
        <p class="info">Score: 4.5, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552238' target='https://doi.org/10.1101/2023.08.07.552238'> 10.1101/2023.08.07.552238</a></p>
        <p class="abstract">The functional diversity of RNA is encoded in their innate conformational heterogeneity. The combination of single-molecule spectroscopy and computational modeling offers new, attractive opportunities to map structural transitions within nucleic acid ensembles. Here, we describe a framework to harmonize single-molecule FRET measurements with molecular dynamics simulations and de novo structure prediction. Using either all-atom or implicit fluorophore modeling we recreate FRET experiments in silico, visualize the underlying structural dynamics and quantify the simulated reaction coordinates. Using multiple accessible-contact volumes (multi-ACV) as a post-hoc scoring method for fragment-assembly in Rosetta, we demonstrate that FRET effectively refines de novo RNA structure prediction without the need of explicit dye labeling. We benchmark our FRET-assisted modeling approach on double-labeled DNA strands and validate it against an intrinsically dynamic manganese(II)-binding riboswitch. We show that a FRET coordinate describing the assembly of a four-way junction allows our pipeline to recapitulate the global fold of the riboswitch with sub-helical accuracy to the crystal structure. We conclude that computational fluorescence spectroscopy facilitates the interpretability of dynamic structural ensembles and improves the mechanistic understanding of nucleic acid interactions.

Graphical abstractSchematic workflow of integrative FRET modeling using all-atom fluorophores or an accessible-contact volume dye model. All-atom molecular dynamics track the dye coordinate explicitly as part of the simulation while multi-ACV infer mean dye positions post hoc.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=150 SRC=&#34;FIGDIR/small/552238v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (81K):
org.highwire.dtl.DTLVardef@d788d2org.highwire.dtl.DTLVardef@57936corg.highwire.dtl.DTLVardef@18fbb88org.highwire.dtl.DTLVardef@160ede8_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
