<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Secreted footprints let cells switch between confined, oscillatory, and exploratory migration
Authors: Ipina, E. P.; D&#39;Alessandro, J.; Ladoux, B.; Camley, B. A.
Score: 22.6, Published: 2023-09-16 DOI: 10.1101/2023.09.14.557437
For eukaryotic cells to heal wounds, respond to immune signals, or metastasize, they must migrate, often by adhering to extracellular matrix. Cells may also secrete matrix factors, leaving behind a footprint that influences their crawling. Recent experiments showed that epithelial cells on micropatterned adhesive stripes move persistently in regions they have previously crawled over, where footprints have been formed, but barely advance into unexplored regions, creating an oscillatory migration of increasing amplitude.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Secreted footprints let cells switch between confined, oscillatory, and exploratory migration
Authors: Ipina, E. P.; D&#39;Alessandro, J.; Ladoux, B.; Camley, B. A.
Score: 22.6, Published: 2023-09-16 DOI: 10.1101/2023.09.14.557437
For eukaryotic cells to heal wounds, respond to immune signals, or metastasize, they must migrate, often by adhering to extracellular matrix. Cells may also secrete matrix factors, leaving behind a footprint that influences their crawling. Recent experiments showed that epithelial cells on micropatterned adhesive stripes move persistently in regions they have previously crawled over, where footprints have been formed, but barely advance into unexplored regions, creating an oscillatory migration of increasing amplitude." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Secreted footprints let cells switch between confined, oscillatory, and exploratory migration
Authors: Ipina, E. P.; D&#39;Alessandro, J.; Ladoux, B.; Camley, B. A.
Score: 22.6, Published: 2023-09-16 DOI: 10.1101/2023.09.14.557437
For eukaryotic cells to heal wounds, respond to immune signals, or metastasize, they must migrate, often by adhering to extracellular matrix. Cells may also secrete matrix factors, leaving behind a footprint that influences their crawling. Recent experiments showed that epithelial cells on micropatterned adhesive stripes move persistently in regions they have previously crawled over, where footprints have been formed, but barely advance into unexplored regions, creating an oscillatory migration of increasing amplitude."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Secreted footprints let cells switch between confined, oscillatory, and exploratory migration\nAuthors: Ipina, E. P.; D\u0026#39;Alessandro, J.; Ladoux, B.; Camley, B. A.\nScore: 22.6, Published: 2023-09-16 DOI: 10.1101/2023.09.14.557437\nFor eukaryotic cells to heal wounds, respond to immune signals, or metastasize, they must migrate, often by adhering to extracellular matrix. Cells may also secrete matrix factors, leaving behind a footprint that influences their crawling. Recent experiments showed that epithelial cells on micropatterned adhesive stripes move persistently in regions they have previously crawled over, where footprints have been formed, but barely advance into unexplored regions, creating an oscillatory migration of increasing amplitude.",
  "keywords": [
    
  ],
  "articleBody": " Secreted footprints let cells switch between confined, oscillatory, and exploratory migration\nAuthors: Ipina, E. P.; D'Alessandro, J.; Ladoux, B.; Camley, B. A.\nScore: 22.6, Published: 2023-09-16 DOI: 10.1101/2023.09.14.557437\nFor eukaryotic cells to heal wounds, respond to immune signals, or metastasize, they must migrate, often by adhering to extracellular matrix. Cells may also secrete matrix factors, leaving behind a footprint that influences their crawling. Recent experiments showed that epithelial cells on micropatterned adhesive stripes move persistently in regions they have previously crawled over, where footprints have been formed, but barely advance into unexplored regions, creating an oscillatory migration of increasing amplitude. Here, we explore through mathematical modeling how footprint secretion and cell responses to footprint combine to allow cells to develop oscillation and other complex migratory motions. We simulate cell crawling with a phase field model coupled to a biochemical model of cell polarity, assuming local contact with the secreted footprint activates Rac1, a polarity protein at the front of the cell. Depending on the footprint secretion rate and the response to the footprint, cells on micropatterned lines can display a variety of types of motility, including confined, oscillatory, and persistent motion. On 2D substrates, we predict a transition between cells undergoing circular motion and cells developing a more exploratory phenotype. Our model shows how minor changes in a cells interaction with its footprint can completely alter exploration, allowing cells to tightly regulate their motion, as well as leading to a wide spectrum of behaviors when secretion or sensing is variable from cell to cell. Consistent with our computational predictions, we find in earlier experimental data evidence of cells undergoing both circular and exploratory motion. Our work proposes a new paradigm for how cells regulate their own motility.\nMesoscale molecular assembly is favored by the active, crowded cytoplasm\nAuthors: Shu, T.; Mitra, G.; Alberts, J. B.; Viana, M.; Levy, E. D.; Hocky, G. M.; Holt, L. J.\nScore: 22.5, Published: 2023-09-21 DOI: 10.1101/2023.09.19.558334\nThe mesoscale organization of molecules into membraneless biomolecular condensates is emerging as a key mechanism of rapid spatiotemporal control in cells. Principles of biomolecular condensation have been revealed through in vitro reconstitution. However, intracellular environments are much more complex than test-tube environments: They are viscoelastic, highly crowded at the mesoscale, and are far from thermodynamic equilibrium due to the constant action of energy-consuming processes. We developed synDrops, a synthetic phase separation system, to study how the cellular environment affects condensate formation. Three key features enable physical analysis: synDrops are inducible, bioorthogonal, and have well-defined geometry. This design allows kinetic analysis of synDrop assembly and facilitates computational simulation of the process. We compared experiments and simulations to determine that macromolecular crowding promotes condensate nucleation but inhibits droplet growth through coalescence. ATP-dependent cellular activities help overcome the frustration of growth. In particular, actomyosin dynamics potentiate droplet growth by reducing confinement and elasticity in the mammalian cytoplasm, thereby enabling synDrop coarsening. Our results demonstrate that mesoscale molecular assembly is favored by the combined effects of crowding and active matter in the cytoplasm. These results move toward a better predictive understanding of condensate formation in vivo.\nPattern formation by bacteria-phage interactions\nAuthors: Martinez-Calvo, A.; Wingreen, N. S.; Datta, S. S.\nScore: 21.3, Published: 2023-09-20 DOI: 10.1101/2023.09.19.558479\nThe interactions between bacteria and phages--viruses that infect bacteria--play critical roles in agriculture, ecology, and medicine; however, how these interactions influence the spatial organization of both bacteria and phages remain largely unexplored. Here, we address this gap in knowledge by developing a theoretical model of motile, proliferating bacteria that aggregate via motility-induced phase separation (MIPS) and encounter phage that infect and lyse the cells. We find that the non-reciprocal predator-prey interactions between phage and bacteria strongly alter spatial organization, in some cases giving rise to a rich array of finite-scale stationary and dynamic patterns in which bacteria and phage coexist. We establish principles describing the onset and characteristics of these diverse behaviors, thereby helping to provide a biophysical basis for understanding pattern formation in bacteria-phage systems, as well as in a broader range of active and living systems with similar predator-prey or other non-reciprocal interactions.\nSolution State Methyl NMR Spectroscopy of Large Non-Deuterated Proteins Enabled by Deep Neural Networks\nAuthors: Karunanithy, G.; Shukla, V. K.; Hansen, D. F.\nScore: 20.6, Published: 2023-09-17 DOI: 10.1101/2023.09.15.557823\nMethyl-TROSY nuclear magnetic resonance (NMR) spectroscopy is a powerful technique for characterising large biomolecules in solution. However, preparing samples for these experiments is arduous and entails deuteration, limiting its use. Here we demonstrate that NMR spectra recorded on protonated, uniformly 13C labelled, samples can be processed using deep neural networks to yield spectra that are of similar quality to typical deuterated methyl-TROSY spectra, potentially providing more information at a fraction of the cost. We validated the new methodology experimentally on three proteins with molecular weights in the range 42-360 kDa and further by analysing deep learning-processed NOESY spectra of Escherichia coli Malate Synthase G (81 kDa), where observed NOE cross-peaks were in good agreement with the available structure. The new method represents a substantial advance in the field of using deep learning to analyse complex magnetic resonance data and could have a major impact on the study of large biomolecules in the years to come.\nAn optogenetic method for the controlled release of single molecules\nAuthors: Kashyap, P.; Bertelli, S.; Cao, F.; Kostritskaia, Y.; Blank, F.; Srikanth, N.; Saleppico, R.; Bierhuizen, D.; Lu, X.; Nickel, W.; Campbell, R. E.; Plested, A.; Stauber, T.; Taylor, M. J.; Ewers, H.\nScore: 15.7, Published: 2023-09-17 DOI: 10.1101/2023.09.16.557871\nWe developed a system for optogenetic release of single molecules in live cells. We confined soluble and transmembrane proteins to the Golgi apparatus via a photocleavable protein and released them by short pulses of light. Our method allows for the controlled delivery of functional proteins to cytosol and plasma membrane in amounts compatible with single molecule imaging, greatly simplifying access to single molecule microscopy of any protein in live cells. Furthermore, we could reconstitute cellular functions such as ion conductance by delivering BK and VRAC ion channels to the plasma membrane. Finally, we could induce NF-kB signaling in T-Lymphoblasts stimulated by IL-1 by controlled release of a signaling protein that had been knocked-out in the same cells. We observed light induced formation of functional inflammatory signaling complexes that could trigger IKK phosphorylation in single cells. We thus developed an optogenetic method for the reconstitution and investigation of cellular function at the single molecule level.\nReentrant DNA shells tune polyphosphate condensate size\nAuthors: Chawla, R.; Tom, J. K. A.; Boyd, T.; Grotjahn, D. A.; Park, D.; Deniz, A. A.; Racki, L. R.\nScore: 13.3, Published: 2023-09-15 DOI: 10.1101/2023.09.13.557044\nThe ancient, inorganic biopolymer polyphosphate (polyP) occurs in all three domains of life and affects myriad cellular processes. An intriguing feature of polyP is its frequent proximity to chromatin, and in the case of many bacteria, its occurrence in the form of magnesium-enriched condensates embedded in the nucleoid, particularly in response to stress. The physical basis of the interaction between polyP and DNA, two fundamental anionic biopolymers, and the resulting effects on the organization of both the nucleoid and polyP condensates remain poorly understood. Given the essential role of magnesium ions in the coordination of polymeric phosphate species, we hypothesized that a minimal system of polyP, magnesium ions, and DNA (polyP-Mg2+-DNA) would capture key features of the interplay between the condensates and bacterial chromatin. We find that DNA can profoundly affect polyP-Mg2+ coacervation even at concentrations several orders of magnitude lower than found in the cell. The DNA forms shells around polyP-Mg2+ condensates and these shells show reentrant behavior, primarily forming in the concentration range close to polyP-Mg2+ charge neutralization. This surface association tunes both condensate size and DNA morphology in a manner dependent on DNA properties, including length and concentration. Our work identifies three components that could form the basis of a central and tunable interaction hub that interfaces with cellular interactors. These studies will inform future efforts to understand the basis of polyP granule composition and consolidation, as well as the potential capacity of these mesoscale assemblies to remodel chromatin in response to diverse stressors at different length and time scales.\nSurprising Features of Nuclear Receptor Interaction Networks Revealed by Live Cell Single Molecule Imaging\nAuthors: Dahal, L.; Graham, T. G. W.; Dailey, G. M.; Heckert, A. B.; Tjian, R.; Darzacq, X.\nScore: 9.6, Published: 2023-09-17 DOI: 10.1101/2023.09.16.558083\nType 2 Nuclear Receptors (T2NRs) require heterodimerization with a common partner, the Retinoid X Receptor (RXR), to bind cognate DNA recognition sites in chromatin. Based on previous biochemical and over-expression studies, binding of T2NRs to chromatin is proposed to be regulated by competition for a limiting pool of the core RXR subunit. However, this mechanism has not yet been tested for endogenous proteins in live cells. Using single molecule tracking (SMT) and proximity-assisted photoactivation (PAPA), we monitored interactions between endogenously tagged retinoid X receptor (RXR) and retinoic acid receptor (RAR) in live cells. Unexpectedly, we find that higher expression of RAR, but not RXR increases heterodimerization and chromatin binding in U2OS cells. This surprising finding indicates the limiting factor is not RXR but likely its cadre of obligate dimer binding partners. SMT and PAPA thus provide a direct way to probe which components are functionally limiting within a complex TF interaction network providing new insights into mechanisms of gene regulation in vivo with implications for drug development targeting nuclear receptors.\nConserved nucleocytoplasmic density homeostasis drives cellular organization across eukaryotes\nAuthors: Biswas, A.; Munoz, O.; Kim, K.; Hoege, C.; Lorton, B. M.; Shechter, D.; Guck, J.; Zaburdaev, V.; Reber, S.\nScore: 83.1, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556409\nThe packing and confinement of macromolecules in the cytoplasm and nucleoplasm has profound implications for cellular biochemistry. How intracellular density distributions vary and affect cellular physiology remains largely unknown. Here, we show that the nucleus is less dense than the cytoplasm and that living systems establish and maintain a constant density ratio between these compartments. Using label-free biophotonics and theory, we show that nuclear density is set by a pressure balance across the nuclear envelope in vitro, in vivo and during early development. Nuclear transport establishes a specific nuclear proteome that exerts a colloid osmotic pressure, which, assisted by entropic chromatin pressure, draws water into the nucleus. Using C. elegans, we show that while nuclear-to-cytoplasmic (N/C) volume ratios change during early development, the N/C density ratio is robustly maintained. We propose that the maintenance of a constant N/C density ratio is the biophysical driver of one of the oldest tenets of cell biology: the N/C volume ratio. In summary, this study reveals a previously unidentified homeostatic coupling of macromolecular densities that drives cellular organization with implications for pathophysiologies such as senescence and cancer.\nStructural insights into the orthosteric inhibition of P2X receptors by non-ATP-analog antagonists\nAuthors: Sheng, D.; Yue, C.; Jin, F.; Wang, Y.; Ichikawa, M.; Yu, Y.; Guo, C.-R.; Hattori, M.\nScore: 8.6, Published: 2023-09-17 DOI: 10.1101/2023.09.15.558037\nP2X receptors are extracellular ATP-gated ion channels that form homo-or heterotrimers and consist of seven subtypes. They are expressed in various tissues, including neuronal and nonneuronal cells, and play critical roles in physiological processes such as neurotransmission, inflammation, pain, and cancer. As a result, P2X receptors have attracted considerable interest as drug targets, and various competitive inhibitors have been developed. However, although several P2X receptor structures from different subtypes have been reported, the limited structural information of P2X receptors in complex with competitive antagonists hampers the understanding of orthosteric inhibition, hindering the further design and optimization of those antagonists for drug discovery. Here, we determined the cryo-EM structures of the mammalian P2X7 receptor in complex with two classical competitive antagonists of pyridoxal-5-phosphate derivatives, PPNDS and PPADS, at 3.3 and 3.6 [A] resolution, respectively, and performed structure-based mutational analysis by patch-clamp recording as well as MD simulations. Our structures revealed the orthosteric site for PPADS/PPNDS, and structural comparison with the previously reported apo-and ATP-bound structures showed how PPADS/PPNDS binding inhibits the conformational changes associated with channel activation. In addition, structure-based mutational analysis identified key residues involved in the PPNDS sensitivity of P2X1 and P2X3, which are known to have higher affinity for PPADS/PPNDS than other P2X subtypes. Overall, our work provides structural insights into the orthosteric inhibition and subtype specificity of P2X receptors by the classical P2X antagonists, pyridoxal-5-phosphate derivatives, thereby facilitating the rational design of novel competitive antagonists for P2X receptors.\nSpatio-temporal Coordination of Active Deformation Forces and Wnt / Hippo-Yap Signaling in Hydra Regeneration\nAuthors: Suzuki, R.; Hiraiwa, T.; Tursch, A.; Hoger, S.; Hayashi, K.; Ozbek, S.; Holstein, T. W.; Tanaka, M.\nScore: 7.8, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558226\nAmple evidence suggests that Wnt signaling and tissue deformation are key determinants for pattern formation in animals. The coordination of these biochemical and biomechanical spatio-temporal asymmetries is often unknown or controversial. We investigated this relationship by studying regeneration in the freshwater polyp Hydra. In both reaggregates of dissociated cells and tissue regenerates, we found significant tissue contraction waves and upregulation of Wnt signaling. Applying a simple mechanical model to the mode analysis of the active deformations, we quantitatively defined the phase reversal of size change and axial deformation in those oscillations as the time point of \"biomechanical\" symmetry breaking. Moreover, overexpression and inhibition of canonical Wnt signaling modulated the timing of this biomechanical symmetry breaking. A direct comparison with the RNAseq data indicates that the biomechanical symmetry breaking occurs only after the upregulation of canonical Wnt signaling. Further data suggest that biochemical signaling and biomechanical active deformation synergistically stabilize the body axis and hence the following head structure formation by Hippo-Yap signaling. The symmetry breaking mechanism identified here in Hydra most likely represents a patterning module that is evolutionary conserved from early metazoan to bilaterian animals.\n",
  "wordCount" : "2277",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557437">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557437" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557437">
        <p class="paperTitle">Secreted footprints let cells switch between confined, oscillatory, and exploratory migration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557437" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557437" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ipina, E. P.; D&#39;Alessandro, J.; Ladoux, B.; Camley, B. A.</p>
        <p class="info">Score: 22.6, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557437' target='https://doi.org/10.1101/2023.09.14.557437'> 10.1101/2023.09.14.557437</a></p>
        <p class="abstract">For eukaryotic cells to heal wounds, respond to immune signals, or metastasize, they must migrate, often by adhering to extracellular matrix. Cells may also secrete matrix factors, leaving behind a footprint that influences their crawling. Recent experiments showed that epithelial cells on micropatterned adhesive stripes move persistently in regions they have previously crawled over, where footprints have been formed, but barely advance into unexplored regions, creating an oscillatory migration of increasing amplitude. Here, we explore through mathematical modeling how footprint secretion and cell responses to footprint combine to allow cells to develop oscillation and other complex migratory motions. We simulate cell crawling with a phase field model coupled to a biochemical model of cell polarity, assuming local contact with the secreted footprint activates Rac1, a polarity protein at the front of the cell. Depending on the footprint secretion rate and the response to the footprint, cells on micropatterned lines can display a variety of types of motility, including confined, oscillatory, and persistent motion. On 2D substrates, we predict a transition between cells undergoing circular motion and cells developing a more exploratory phenotype. Our model shows how minor changes in a cells interaction with its footprint can completely alter exploration, allowing cells to tightly regulate their motion, as well as leading to a wide spectrum of behaviors when secretion or sensing is variable from cell to cell. Consistent with our computational predictions, we find in earlier experimental data evidence of cells undergoing both circular and exploratory motion. Our work proposes a new paradigm for how cells regulate their own motility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558334">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558334" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558334">
        <p class="paperTitle">Mesoscale molecular assembly is favored by the active, crowded cytoplasm</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558334" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558334" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shu, T.; Mitra, G.; Alberts, J. B.; Viana, M.; Levy, E. D.; Hocky, G. M.; Holt, L. J.</p>
        <p class="info">Score: 22.5, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558334' target='https://doi.org/10.1101/2023.09.19.558334'> 10.1101/2023.09.19.558334</a></p>
        <p class="abstract">The mesoscale organization of molecules into membraneless biomolecular condensates is emerging as a key mechanism of rapid spatiotemporal control in cells. Principles of biomolecular condensation have been revealed through in vitro reconstitution. However, intracellular environments are much more complex than test-tube environments: They are viscoelastic, highly crowded at the mesoscale, and are far from thermodynamic equilibrium due to the constant action of energy-consuming processes. We developed synDrops, a synthetic phase separation system, to study how the cellular environment affects condensate formation. Three key features enable physical analysis: synDrops are inducible, bioorthogonal, and have well-defined geometry. This design allows kinetic analysis of synDrop assembly and facilitates computational simulation of the process. We compared experiments and simulations to determine that macromolecular crowding promotes condensate nucleation but inhibits droplet growth through coalescence. ATP-dependent cellular activities help overcome the frustration of growth. In particular, actomyosin dynamics potentiate droplet growth by reducing confinement and elasticity in the mammalian cytoplasm, thereby enabling synDrop coarsening. Our results demonstrate that mesoscale molecular assembly is favored by the combined effects of crowding and active matter in the cytoplasm. These results move toward a better predictive understanding of condensate formation in vivo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558479">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558479" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558479">
        <p class="paperTitle">Pattern formation by bacteria-phage interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558479" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558479" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Calvo, A.; Wingreen, N. S.; Datta, S. S.</p>
        <p class="info">Score: 21.3, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558479' target='https://doi.org/10.1101/2023.09.19.558479'> 10.1101/2023.09.19.558479</a></p>
        <p class="abstract">The interactions between bacteria and phages--viruses that infect bacteria--play critical roles in agriculture, ecology, and medicine; however, how these interactions influence the spatial organization of both bacteria and phages remain largely unexplored. Here, we address this gap in knowledge by developing a theoretical model of motile, proliferating bacteria that aggregate via motility-induced phase separation (MIPS) and encounter phage that infect and lyse the cells. We find that the non-reciprocal predator-prey interactions between phage and bacteria strongly alter spatial organization, in some cases giving rise to a rich array of finite-scale stationary and dynamic patterns in which bacteria and phage coexist. We establish principles describing the onset and characteristics of these diverse behaviors, thereby helping to provide a biophysical basis for understanding pattern formation in bacteria-phage systems, as well as in a broader range of active and living systems with similar predator-prey or other non-reciprocal interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557823">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557823" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557823">
        <p class="paperTitle">Solution State Methyl NMR Spectroscopy of Large Non-Deuterated Proteins Enabled by Deep Neural Networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557823" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557823" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Karunanithy, G.; Shukla, V. K.; Hansen, D. F.</p>
        <p class="info">Score: 20.6, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557823' target='https://doi.org/10.1101/2023.09.15.557823'> 10.1101/2023.09.15.557823</a></p>
        <p class="abstract">Methyl-TROSY nuclear magnetic resonance (NMR) spectroscopy is a powerful technique for characterising large biomolecules in solution. However, preparing samples for these experiments is arduous and entails deuteration, limiting its use. Here we demonstrate that NMR spectra recorded on protonated, uniformly 13C labelled, samples can be processed using deep neural networks to yield spectra that are of similar quality to typical deuterated methyl-TROSY spectra, potentially providing more information at a fraction of the cost. We validated the new methodology experimentally on three proteins with molecular weights in the range 42-360 kDa and further by analysing deep learning-processed NOESY spectra of Escherichia coli Malate Synthase G (81 kDa), where observed NOE cross-peaks were in good agreement with the available structure. The new method represents a substantial advance in the field of using deep learning to analyse complex magnetic resonance data and could have a major impact on the study of large biomolecules in the years to come.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.16.557871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.16.557871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.16.557871">
        <p class="paperTitle">An optogenetic method for the controlled release of single molecules</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.16.557871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.16.557871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kashyap, P.; Bertelli, S.; Cao, F.; Kostritskaia, Y.; Blank, F.; Srikanth, N.; Saleppico, R.; Bierhuizen, D.; Lu, X.; Nickel, W.; Campbell, R. E.; Plested, A.; Stauber, T.; Taylor, M. J.; Ewers, H.</p>
        <p class="info">Score: 15.7, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.16.557871' target='https://doi.org/10.1101/2023.09.16.557871'> 10.1101/2023.09.16.557871</a></p>
        <p class="abstract">We developed a system for optogenetic release of single molecules in live cells. We confined soluble and transmembrane proteins to the Golgi apparatus via a photocleavable protein and released them by short pulses of light. Our method allows for the controlled delivery of functional proteins to cytosol and plasma membrane in amounts compatible with single molecule imaging, greatly simplifying access to single molecule microscopy of any protein in live cells. Furthermore, we could reconstitute cellular functions such as ion conductance by delivering BK and VRAC ion channels to the plasma membrane. Finally, we could induce NF-kB signaling in T-Lymphoblasts stimulated by IL-1 by controlled release of a signaling protein that had been knocked-out in the same cells. We observed light induced formation of functional inflammatory signaling complexes that could trigger IKK phosphorylation in single cells. We thus developed an optogenetic method for the reconstitution and investigation of cellular function at the single molecule level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557044">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557044" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557044">
        <p class="paperTitle">Reentrant DNA shells tune polyphosphate condensate size</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557044" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557044" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chawla, R.; Tom, J. K. A.; Boyd, T.; Grotjahn, D. A.; Park, D.; Deniz, A. A.; Racki, L. R.</p>
        <p class="info">Score: 13.3, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557044' target='https://doi.org/10.1101/2023.09.13.557044'> 10.1101/2023.09.13.557044</a></p>
        <p class="abstract">The ancient, inorganic biopolymer polyphosphate (polyP) occurs in all three domains of life and affects myriad cellular processes. An intriguing feature of polyP is its frequent proximity to chromatin, and in the case of many bacteria, its occurrence in the form of magnesium-enriched condensates embedded in the nucleoid, particularly in response to stress. The physical basis of the interaction between polyP and DNA, two fundamental anionic biopolymers, and the resulting effects on the organization of both the nucleoid and polyP condensates remain poorly understood. Given the essential role of magnesium ions in the coordination of polymeric phosphate species, we hypothesized that a minimal system of polyP, magnesium ions, and DNA (polyP-Mg2&#43;-DNA) would capture key features of the interplay between the condensates and bacterial chromatin. We find that DNA can profoundly affect polyP-Mg2&#43; coacervation even at concentrations several orders of magnitude lower than found in the cell. The DNA forms shells around polyP-Mg2&#43; condensates and these shells show reentrant behavior, primarily forming in the concentration range close to polyP-Mg2&#43; charge neutralization. This surface association tunes both condensate size and DNA morphology in a manner dependent on DNA properties, including length and concentration. Our work identifies three components that could form the basis of a central and tunable interaction hub that interfaces with cellular interactors. These studies will inform future efforts to understand the basis of polyP granule composition and consolidation, as well as the potential capacity of these mesoscale assemblies to remodel chromatin in response to diverse stressors at different length and time scales.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.16.558083">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.16.558083" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.16.558083">
        <p class="paperTitle">Surprising Features of Nuclear Receptor Interaction Networks Revealed by Live Cell Single Molecule Imaging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.16.558083" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.16.558083" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dahal, L.; Graham, T. G. W.; Dailey, G. M.; Heckert, A. B.; Tjian, R.; Darzacq, X.</p>
        <p class="info">Score: 9.6, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.16.558083' target='https://doi.org/10.1101/2023.09.16.558083'> 10.1101/2023.09.16.558083</a></p>
        <p class="abstract">Type 2 Nuclear Receptors (T2NRs) require heterodimerization with a common partner, the Retinoid X Receptor (RXR), to bind cognate DNA recognition sites in chromatin. Based on previous biochemical and over-expression studies, binding of T2NRs to chromatin is proposed to be regulated by competition for a limiting pool of the core RXR subunit. However, this mechanism has not yet been tested for endogenous proteins in live cells. Using single molecule tracking (SMT) and proximity-assisted photoactivation (PAPA), we monitored interactions between endogenously tagged retinoid X receptor (RXR) and retinoic acid receptor (RAR) in live cells. Unexpectedly, we find that higher expression of RAR, but not RXR increases heterodimerization and chromatin binding in U2OS cells. This surprising finding indicates the limiting factor is not RXR but likely its cadre of obligate dimer binding partners. SMT and PAPA thus provide a direct way to probe which components are functionally limiting within a complex TF interaction network providing new insights into mechanisms of gene regulation in vivo with implications for drug development targeting nuclear receptors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556409">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556409" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556409">
        <p class="paperTitle">Conserved nucleocytoplasmic density homeostasis drives cellular organization across eukaryotes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556409" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556409" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Biswas, A.; Munoz, O.; Kim, K.; Hoege, C.; Lorton, B. M.; Shechter, D.; Guck, J.; Zaburdaev, V.; Reber, S.</p>
        <p class="info">Score: 83.1, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556409' target='https://doi.org/10.1101/2023.09.05.556409'> 10.1101/2023.09.05.556409</a></p>
        <p class="abstract">The packing and confinement of macromolecules in the cytoplasm and nucleoplasm has profound implications for cellular biochemistry. How intracellular density distributions vary and affect cellular physiology remains largely unknown. Here, we show that the nucleus is less dense than the cytoplasm and that living systems establish and maintain a constant density ratio between these compartments. Using label-free biophotonics and theory, we show that nuclear density is set by a pressure balance across the nuclear envelope in vitro, in vivo and during early development. Nuclear transport establishes a specific nuclear proteome that exerts a colloid osmotic pressure, which, assisted by entropic chromatin pressure, draws water into the nucleus. Using C. elegans, we show that while nuclear-to-cytoplasmic (N/C) volume ratios change during early development, the N/C density ratio is robustly maintained. We propose that the maintenance of a constant N/C density ratio is the biophysical driver of one of the oldest tenets of cell biology: the N/C volume ratio. In summary, this study reveals a previously unidentified homeostatic coupling of macromolecular densities that drives cellular organization with implications for pathophysiologies such as senescence and cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.558037">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.558037" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.558037">
        <p class="paperTitle">Structural insights into the orthosteric inhibition of P2X receptors by non-ATP-analog antagonists</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.558037" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.558037" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sheng, D.; Yue, C.; Jin, F.; Wang, Y.; Ichikawa, M.; Yu, Y.; Guo, C.-R.; Hattori, M.</p>
        <p class="info">Score: 8.6, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.558037' target='https://doi.org/10.1101/2023.09.15.558037'> 10.1101/2023.09.15.558037</a></p>
        <p class="abstract">P2X receptors are extracellular ATP-gated ion channels that form homo-or heterotrimers and consist of seven subtypes. They are expressed in various tissues, including neuronal and nonneuronal cells, and play critical roles in physiological processes such as neurotransmission, inflammation, pain, and cancer. As a result, P2X receptors have attracted considerable interest as drug targets, and various competitive inhibitors have been developed. However, although several P2X receptor structures from different subtypes have been reported, the limited structural information of P2X receptors in complex with competitive antagonists hampers the understanding of orthosteric inhibition, hindering the further design and optimization of those antagonists for drug discovery.

Here, we determined the cryo-EM structures of the mammalian P2X7 receptor in complex with two classical competitive antagonists of pyridoxal-5-phosphate derivatives, PPNDS and PPADS, at 3.3 and 3.6 [A] resolution, respectively, and performed structure-based mutational analysis by patch-clamp recording as well as MD simulations. Our structures revealed the orthosteric site for PPADS/PPNDS, and structural comparison with the previously reported apo-and ATP-bound structures showed how PPADS/PPNDS binding inhibits the conformational changes associated with channel activation. In addition, structure-based mutational analysis identified key residues involved in the PPNDS sensitivity of P2X1 and P2X3, which are known to have higher affinity for PPADS/PPNDS than other P2X subtypes. Overall, our work provides structural insights into the orthosteric inhibition and subtype specificity of P2X receptors by the classical P2X antagonists, pyridoxal-5-phosphate derivatives, thereby facilitating the rational design of novel competitive antagonists for P2X receptors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558226">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558226" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558226">
        <p class="paperTitle">Spatio-temporal Coordination of Active Deformation Forces and Wnt / Hippo-Yap Signaling in Hydra Regeneration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558226" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558226" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Suzuki, R.; Hiraiwa, T.; Tursch, A.; Hoger, S.; Hayashi, K.; Ozbek, S.; Holstein, T. W.; Tanaka, M.</p>
        <p class="info">Score: 7.8, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558226' target='https://doi.org/10.1101/2023.09.18.558226'> 10.1101/2023.09.18.558226</a></p>
        <p class="abstract">Ample evidence suggests that Wnt signaling and tissue deformation are key determinants for pattern formation in animals. The coordination of these biochemical and biomechanical spatio-temporal asymmetries is often unknown or controversial. We investigated this relationship by studying regeneration in the freshwater polyp Hydra. In both reaggregates of dissociated cells and tissue regenerates, we found significant tissue contraction waves and upregulation of Wnt signaling. Applying a simple mechanical model to the mode analysis of the active deformations, we quantitatively defined the phase reversal of size change and axial deformation in those oscillations as the time point of &#34;biomechanical&#34; symmetry breaking. Moreover, overexpression and inhibition of canonical Wnt signaling modulated the timing of this biomechanical symmetry breaking. A direct comparison with the RNAseq data indicates that the biomechanical symmetry breaking occurs only after the upregulation of canonical Wnt signaling. Further data suggest that biochemical signaling and biomechanical active deformation synergistically stabilize the body axis and hence the following head structure formation by Hippo-Yap signaling. The symmetry breaking mechanism identified here in Hydra most likely represents a patterning module that is evolutionary conserved from early metazoan to bilaterian animals.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
