<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells
Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.
Score: 101.5, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799
Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells
Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.
Score: 101.5, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799
Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells
Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.
Score: 101.5, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799
Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells\nAuthors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.\nScore: 101.5, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799\nAdvances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms.",
  "keywords": [
    
  ],
  "articleBody": " Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells\nAuthors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.\nScore: 101.5, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799\nAdvances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms. This method, cryoDRGN-ET, learns a deep generative model of 3D density maps directly from subtomogram tilt series images and can capture states diverse in both composition and conformation. We use this approach to reconstruct the in situ translation dynamics of prokaryotic ribosomes, and we reveal the distribution of functional states during translation elongation populated by S. cerevisiae ribosomes inside cells.\nCortactin stabilizes actin branches by bridging activated Arp2/3 to its nucleated actin filament\nAuthors: Liu, T.; Cao, L.; Mladenov, M.; JEGOU, A.; Way, M.; Moores, C.\nScore: 25.1, Published: 2023-08-16 DOI: 10.1101/2023.08.16.553490\nRegulation of the assembly and turnover of branched actin filament networks nucleated by the Arp2/3 complex is essential during many cellular processes including cell migration and membrane trafficking. Cortactin plays a key role in stabilizing actin filament branches by interacting with the Arp2/3 complex and actin filaments via its N-terminal Acidic domain (NtA) and 6.5 central unstructured 37 amino acid repeats, respectively 1, but the mechanism of this is unclear. We determined the structure of cortactin-stabilized Arp2/3 actin branches using cryo-electron microscopy. We find that cortactin interacts with the new daughter filament nucleated by the Arp2/3 complex at the branch site rather than the initial mother actin filament. Cortactin preferentially binds activated Arp3 in contrast to other nucleation promoting factors (NPFs) 2,3. Cortactin also stabilizes the F-actin-like interface of activated Arp3 with the first actin subunit of the new filament, and its central repeats extend along successive daughter filament subunits. Cortactin binding to Arp3 is incompatible with NPF interaction and its preference for activated Arp3 explains why it is retained at the actin branch. Our data have uncovered why cortactin displaces NPFs, while at the same time promoting synergy to regulate branched actin network dynamics.\nIntracellular mechanical fingerprint reveals cell type specific mechanical tuning\nAuthors: Muenker, T. M.; Vos, B. E.; Betz, T.\nScore: 23.4, Published: 2023-08-13 DOI: 10.1101/2023.08.10.552769\nLiving cells are complex entities that perform many different complex tasks with astonishing robustness. While the direct dependence of biological processes on controlled protein expression is well established, we only begin to understand how intracellular mechanical characteristics guide and support biological function. This is in stark contrast to the expected functional role that intracellular mechanical properties should have for many core cellular functions such as organization, homeostasis and transport. From a mechanical point of view, cells are complex viscoelastic materials that are continuously driven out of thermodynamic equilibrium, which makes both a physical measurement and mathematical modeling of its properties difficult. Here, we define a \"mechanical fingerprint\" that can not only characterize the intracellular mechanical state, but also carve out the mechanical differences between cell types with the potential to relate these to proper cell function. By analyzing the frequency-dependent viscoelastic properties and intracellular activity of cells using microrheology, we distilled the complex active mechanical state into just 6 parameters that comprise the mechanical fingerprint. The systematic investigation of the fingerprint illustrates a parameter tuning that can be explained by the functional cellular requirements. However, the full potential of the mechanical fingerprint is given by a statistical analysis of its parameters across all investigated cell types, which suggests that cells adjust mechanical parameters in a correlated way to position their intracellular mechanical properties within a well defined phase-space that is spanned between activity, mechanical resistance and fluidity. This paves the way for a systematic study of the interdependence of biological function and intracellular active mechanics.\nGlutamate helps unmask the differences in driving forces for phase separation versus clustering of FET family proteins in sub-saturated solutions\nAuthors: Kar, M.; Vogel, L. T.; Chauhan, G.; Ausserwoger, H.; Welsh, T. J.; Kamath, A. R.; Knowles, T. P. J.; Hyman, A. A.; Seidel, C. A. M.; Pappu, R. V.\nScore: 26.6, Published: 2023-08-13 DOI: 10.1101/2023.08.11.552963\nMultivalent proteins undergo coupled segregative and associative phase transitions. Phase separation, a segregative transition, is driven by macromolecular solubility, and this leads to coexisting phases above system-specific saturation concentrations. Percolation is a continuous transition that is driven by multivalent associations among cohesive motifs. Contributions from percolation are highlighted by the formation of heterogeneous distributions of clusters in sub-saturated solutions, as was recently reported for Fused in sarcoma (FUS) and FET family proteins. Here, we show that clustering and phase separation are defined by a separation of length- and energy-scales. This is unmasked when glutamate is the primary solution anion. Glutamate is preferentially excluded from protein sites, and this enhances molecular associations. Differences between glutamate and chloride are manifest at ultra-low protein concentrations. These differences are amplified as concentrations increase, and they saturate as the micron-scale is approached. Therefore, condensate formation in supersaturated solutions and clustering in sub-saturated are governed by distinct energy and length scales. Glutamate, unlike chloride, is the dominant intracellular anion, and the separation of scales, which is masked in chloride, is unmasked in glutamate. Our work highlights how components of cellular milieus and sequence-encoded interactions contribute to amplifying distinct contributions from associative versus segregative phase transitions.\nShocker - a molecular dynamics protocol and tool for accelerating and analyzing the effects of osmotic shocks\nAuthors: van Tilburg, M. P. A.; Marrink, S.-J.; Koenig, M.; Gruenewald, F.\nScore: 17.1, Published: 2023-08-18 DOI: 10.1101/2023.08.16.553535\nThe process of osmosis, a fundamental phenomenon in life, drives water through a semi-permeable membrane in response to a solute concentration gradient across this membrane. In vitro, osmotic shocks are often used to drive shape changes in lipid vesicles, for instance, to study fission events in the context of artificial cells. While experimental techniques provide a macroscopic picture of large-scale membrane remodeling processes, molecular dynamics (MD) simulations are a powerful tool to study membrane deformations at the molecular level. However, simulating an osmotic shock is a time-consuming process due to the slow water diffusion across the membrane, making it practically impossible to examine its effects in classic MD simulations. In this paper, we present Shocker, a Python-based MD tool for simulating the effects of an osmotic shock by selecting and relocating water particles across a membrane over the course of several pumping cycles. Although this method is primarily aimed at efficiently simulating volume changes of vesicles it can handle membrane tubes and double bilayer systems as well. Additionally, Shocker is force field independent and compatible with both coarse-grained and all-atom systems. We demonstrate that our tool is applicable to simulate both hypertonic and hypotonic osmotic shocks for a range of vesicular and bilamellar setups, including complex multi-component systems containing membrane proteins or crowded internal solutions.\nTranslation as a Biosignature\nAuthors: McKaig, J. M.; Kim, M.; Carr, C. E.\nScore: 18.2, Published: 2023-08-16 DOI: 10.1101/2023.08.10.552839\nLife on Earth relies on mechanisms to store heritable information and translate this information into cellular machinery required for biological activity. In all known life, storage, regulation, and translation are provided by DNA, RNA, and ribosomes. Life beyond Earth, even if ancestrally or chemically distinct from life as we know it may utilize similar structures: it has been proposed that charged linear polymers analogous to nucleic acids may be responsible for storage and regulation of genetic information in non-terran biochemical systems. We further propose that a ribosome-like structure may also exist in such a system, due to the evolutionary advantages of separating heritability from cellular machinery. Here, we use a solid-state nanopore to detect DNA, RNA, and ribosomes, and demonstrate that machine learning can distinguish between biomolecule samples and accurately classify new data. This work is intended to serve as a proof of principal that such biosignatures (i.e., informational polymers or translation apparatuses) could be detected, for example, as part of future missions targeting extant life on Ocean Worlds. A negative detection does not imply the absence of life; however, detection of ribosome-like structures could provide a robust and sensitive method to seek extant life in combination with other methods. One Sentence SummaryLife, defined as a chemical system capable of Darwinian evolution, likely requires an apparatus to translate heritable instructions into cellular machinery, and we propose to detect this as a biosignature of extant life beyond Earth.\nRobust and resource-optimal dynamic pattern formation of Min proteins in vivo\nAuthors: ren, z.; Weyer, H.; Wurthner, L.; Li, D.; Sou, C.; Villarreal, D.; Frey, E.; Jun, S.\nScore: 15.9, Published: 2023-08-16 DOI: 10.1101/2023.08.15.553443\nThe Min system in Escherichia coli plays a crucial role in cellular reproduction by preventing minicell formation through pole-to-pole oscillations. Despite extensive research, predicting the onset of Min protein concentrations for oscillation and understanding the systems robustness under physiological perturbations remains challenging. Our study aims to address these gaps. We show that the Min systems dynamic pattern formation is robust across a wide range of Min protein levels and varying growth physiology. Using genetically engineered E. coli strains, we independently modulated the expression of minCD and minE in E. coli under both fast and slow growth conditions. This led to the construction of a MinD-MinE phase diagram, which revealed not just a large oscillation regime but also complex dynamic patterns such as traveling and standing waves. Interestingly, we found that the natural expression level of Min proteins is nearly optimal. Our work combines experimental findings with biophysical theory based on reaction-diffusion models, reproducing the experimental phase diagram and other key properties quantitatively. This includes the observation of an invariant wavelength of dynamic Min patterns across our phase diagram. Crucially, the success of our model depends on the switching of MinE between its latent and active states, indicating its essential role as a robustness module for Min oscillation in vivo. Our results underline the potential of integrating quantitative cell physiology and biophysical modeling in understanding the fundamental mechanisms controlling cell division machinery, offering insights applicable to other biological processes.\nATP Mediates Phase Separation of Disordered Basic Proteins by Bridging Intermolecular Interaction Networks\nAuthors: Kota, D.; Prasad, R.; Zhou, H.-X.\nScore: 15.2, Published: 2023-08-20 DOI: 10.1101/2023.08.20.554035\nATP is an abundant molecule with crucial cellular roles as the energy currency and a building block of nucleic acids and for protein phosphorylation. Here we show that ATP mediates the phase separation of basic intrinsically disordered proteins (bIDPs). In the resulting condensates, ATP is highly concentrated (apparent partition coefficients at 200-5000) and serves as bridges between bIDP chains. These liquid-like droplets have some of the lowest interfacial tension (~25 pN/m) but high zero-shear viscosities (1-15 Pa s) due to the bridged protein networks, and yet their fusion has some of the highest speeds (~1 m/ms). The rapid fusion manifests extreme shear thinning, where the apparent viscosity is lower than zero-shear viscosity by over 100-fold, made possible by fast reformation of the ATP bridges. At still higher concentrations, ATP does not dissolve bIDP droplets but results in aggregates and fibrils.\nIncreasing the accuracy of single-molecule data analysis using tMAVEN\nAuthors: Verma, A. R.; Ray, K. K.; Bodick, M. E.; Kinz-Thompson, C. D.; Gonzalez, R. L.\nScore: 13.2, Published: 2023-08-17 DOI: 10.1101/2023.08.15.553409\nTime-dependent single-molecule experiments contain rich kinetic information about the functional dynamics of biomolecules. A key step in extracting this information is the application of kinetic models, such as hidden Markov models (HMMs), which characterize the molecular mechanism governing the experimental system. Unfortunately, researchers rarely know the physico-chemical details of this molecular mechanism a priori, which raises questions about how to select the most appropriate kinetic model for a given single-molecule dataset and what consequences arise if the wrong model is chosen. To address these questions, we have developed and used time-series Modeling, Analysis, and Visualization ENvironment (tMAVEN), a comprehensive, open-source, and extensible software platform. tMAVEN can perform each step of the single-molecule analysis pipeline, from pre-processing to kinetic modeling to plotting, and has been designed to enable the analysis of a single-molecule dataset with multiple types of kinetic models. Using tMAVEN, we have systematically investigated mismatches between kinetic models and molecular mechanisms by analyzing simulated examples of prototypical single-molecule datasets exhibiting common experimental complications, such as molecular heterogeneity, with a series of different types of HMMs. Our results show that no single kinetic modeling strategy is mathematically appropriate for all experimental contexts. Indeed, HMMs only correctly capture the underlying molecular mechanism in the simplest of cases. As such, researchers must modify HMMs using physico-chemical principles to avoid the risk of missing the significant biological and biophysical insights into molecular heterogeneity that their experiments provide. By enabling the facile, side-by-side application of multiple types of kinetic models to individual single-molecule datasets, tMAVEN allows researchers to carefully tailor their modeling approach to match the complexity of the underlying biomolecular dynamics and increase the accuracy of their single-molecule data analyses. Statement of SignificanceThe power of time-dependent single-molecule biophysical experiments lies in their ability to uncover the molecular mechanisms governing experimental systems by computationally applying kinetic models to the data. While many software solutions have been developed to estimate the optimal parameters of such models, the results reported here show that the models themselves are often inherently mismatched with the molecular mechanisms they are being used to analyze. To investigate these mismatches and demonstrate how to best model the kinetics of a molecular mechanism, we have used time-series Modeling, Analysis, and Visualization ENvironment (tMAVEN), an open-source software platform we have developed that, among other features, enables the analysis of single-molecule datasets using different kinetic models within a single, extensible, and customizable pipeline.\nNanopores reveal the stoichiometry of single oligo-adenylates produced by type III CRISPR-Cas\nAuthors: Fuentenebro-Navas, D.; Steens, J. A.; De Lannoy, C.; Noordijk, B.; de Ridder, D.; Staals, R. H. J.; Schmid, S.\nScore: 12.5, Published: 2023-08-19 DOI: 10.1101/2023.08.18.553839\nCyclic oligoadenylates (cOAs) are small second messenger molecules produced by the type III CRISPR-Cas system as part of the prokaryotic immune response. The role of cOAs is to allosterically activate downstream effector proteins that induce dormancy or cell death, and thus abort viral spread through the population. Interestingly, different type III systems have been reported to utilize different cOA stoichiometries (with 3 to 6 adenylate monophosphates). However, so far, their characterization has only been possible in bulk and with sophisticated equipment, while a portable assay with single-molecule resolution has been lacking. Here, we demonstrate the label-free detection of single cOA molecules using a simple protein nanopore assay. It sensitively identifies the stoichiometry of individual cOA molecules and their mixtures from synthetic and enzymatic origin. To achieve this, we trained a convolutional neural network (CNN) and validated it with a series of experiments on mono- and polydisperse cOA samples. Ultimately, we determined the stoichiometric composition of cOAs produced enzymatically by the CRISPR type III-A and III-B variants of Thermus thermophilus. Interestingly, both variants produce cOAs of nearly identical composition, and we discuss the biological implications of this finding. The presented nanopore-CNN workflow with single-cOA resolution can be adapted to many other signaling molecules (including eukaryotic ones), and it may be integrated into portable handheld devices with potential point-of-care applications.\n",
  "wordCount" : "2527",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553799">
        <p class="paperTitle">Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.</p>
        <p class="info">Score: 101.5, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553799' target='https://doi.org/10.1101/2023.08.18.553799'> 10.1101/2023.08.18.553799</a></p>
        <p class="abstract">Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms. This method, cryoDRGN-ET, learns a deep generative model of 3D density maps directly from subtomogram tilt series images and can capture states diverse in both composition and conformation. We use this approach to reconstruct the in situ translation dynamics of prokaryotic ribosomes, and we reveal the distribution of functional states during translation elongation populated by S. cerevisiae ribosomes inside cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553490">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553490" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553490">
        <p class="paperTitle">Cortactin stabilizes actin branches by bridging activated Arp2/3 to its nucleated actin filament</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553490" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553490" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, T.; Cao, L.; Mladenov, M.; JEGOU, A.; Way, M.; Moores, C.</p>
        <p class="info">Score: 25.1, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553490' target='https://doi.org/10.1101/2023.08.16.553490'> 10.1101/2023.08.16.553490</a></p>
        <p class="abstract">Regulation of the assembly and turnover of branched actin filament networks nucleated by the Arp2/3 complex is essential during many cellular processes including cell migration and membrane trafficking. Cortactin plays a key role in stabilizing actin filament branches by interacting with the Arp2/3 complex and actin filaments via its N-terminal Acidic domain (NtA) and 6.5 central unstructured 37 amino acid repeats, respectively 1, but the mechanism of this is unclear. We determined the structure of cortactin-stabilized Arp2/3 actin branches using cryo-electron microscopy. We find that cortactin interacts with the new daughter filament nucleated by the Arp2/3 complex at the branch site rather than the initial mother actin filament. Cortactin preferentially binds activated Arp3 in contrast to other nucleation promoting factors (NPFs) 2,3. Cortactin also stabilizes the F-actin-like interface of activated Arp3 with the first actin subunit of the new filament, and its central repeats extend along successive daughter filament subunits. Cortactin binding to Arp3 is incompatible with NPF interaction and its preference for activated Arp3 explains why it is retained at the actin branch. Our data have uncovered why cortactin displaces NPFs, while at the same time promoting synergy to regulate branched actin network dynamics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552769">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552769" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552769">
        <p class="paperTitle">Intracellular mechanical fingerprint reveals cell type specific mechanical tuning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552769" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552769" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muenker, T. M.; Vos, B. E.; Betz, T.</p>
        <p class="info">Score: 23.4, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552769' target='https://doi.org/10.1101/2023.08.10.552769'> 10.1101/2023.08.10.552769</a></p>
        <p class="abstract">Living cells are complex entities that perform many different complex tasks with astonishing robustness. While the direct dependence of biological processes on controlled protein expression is well established, we only begin to understand how intracellular mechanical characteristics guide and support biological function. This is in stark contrast to the expected functional role that intracellular mechanical properties should have for many core cellular functions such as organization, homeostasis and transport. From a mechanical point of view, cells are complex viscoelastic materials that are continuously driven out of thermodynamic equilibrium, which makes both a physical measurement and mathematical modeling of its properties difficult. Here, we define a &#34;mechanical fingerprint&#34; that can not only characterize the intracellular mechanical state, but also carve out the mechanical differences between cell types with the potential to relate these to proper cell function. By analyzing the frequency-dependent viscoelastic properties and intracellular activity of cells using microrheology, we distilled the complex active mechanical state into just 6 parameters that comprise the mechanical fingerprint. The systematic investigation of the fingerprint illustrates a parameter tuning that can be explained by the functional cellular requirements. However, the full potential of the mechanical fingerprint is given by a statistical analysis of its parameters across all investigated cell types, which suggests that cells adjust mechanical parameters in a correlated way to position their intracellular mechanical properties within a well defined phase-space that is spanned between activity, mechanical resistance and fluidity. This paves the way for a systematic study of the interdependence of biological function and intracellular active mechanics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552963">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552963" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552963">
        <p class="paperTitle">Glutamate helps unmask the differences in driving forces for phase separation versus clustering of FET family proteins in sub-saturated solutions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552963" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552963" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kar, M.; Vogel, L. T.; Chauhan, G.; Ausserwoger, H.; Welsh, T. J.; Kamath, A. R.; Knowles, T. P. J.; Hyman, A. A.; Seidel, C. A. M.; Pappu, R. V.</p>
        <p class="info">Score: 26.6, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552963' target='https://doi.org/10.1101/2023.08.11.552963'> 10.1101/2023.08.11.552963</a></p>
        <p class="abstract">Multivalent proteins undergo coupled segregative and associative phase transitions. Phase separation, a segregative transition, is driven by macromolecular solubility, and this leads to coexisting phases above system-specific saturation concentrations. Percolation is a continuous transition that is driven by multivalent associations among cohesive motifs. Contributions from percolation are highlighted by the formation of heterogeneous distributions of clusters in sub-saturated solutions, as was recently reported for Fused in sarcoma (FUS) and FET family proteins. Here, we show that clustering and phase separation are defined by a separation of length- and energy-scales. This is unmasked when glutamate is the primary solution anion. Glutamate is preferentially excluded from protein sites, and this enhances molecular associations. Differences between glutamate and chloride are manifest at ultra-low protein concentrations. These differences are amplified as concentrations increase, and they saturate as the micron-scale is approached. Therefore, condensate formation in supersaturated solutions and clustering in sub-saturated are governed by distinct energy and length scales. Glutamate, unlike chloride, is the dominant intracellular anion, and the separation of scales, which is masked in chloride, is unmasked in glutamate. Our work highlights how components of cellular milieus and sequence-encoded interactions contribute to amplifying distinct contributions from associative versus segregative phase transitions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553535">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553535" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553535">
        <p class="paperTitle">Shocker - a molecular dynamics protocol and tool for accelerating and analyzing the effects of osmotic shocks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553535" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553535" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van Tilburg, M. P. A.; Marrink, S.-J.; Koenig, M.; Gruenewald, F.</p>
        <p class="info">Score: 17.1, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553535' target='https://doi.org/10.1101/2023.08.16.553535'> 10.1101/2023.08.16.553535</a></p>
        <p class="abstract">The process of osmosis, a fundamental phenomenon in life, drives water through a semi-permeable membrane in response to a solute concentration gradient across this membrane. In vitro, osmotic shocks are often used to drive shape changes in lipid vesicles, for instance, to study fission events in the context of artificial cells. While experimental techniques provide a macroscopic picture of large-scale membrane remodeling processes, molecular dynamics (MD) simulations are a powerful tool to study membrane deformations at the molecular level. However, simulating an osmotic shock is a time-consuming process due to the slow water diffusion across the membrane, making it practically impossible to examine its effects in classic MD simulations. In this paper, we present Shocker, a Python-based MD tool for simulating the effects of an osmotic shock by selecting and relocating water particles across a membrane over the course of several pumping cycles. Although this method is primarily aimed at efficiently simulating volume changes of vesicles it can handle membrane tubes and double bilayer systems as well. Additionally, Shocker is force field independent and compatible with both coarse-grained and all-atom systems. We demonstrate that our tool is applicable to simulate both hypertonic and hypotonic osmotic shocks for a range of vesicular and bilamellar setups, including complex multi-component systems containing membrane proteins or crowded internal solutions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552839">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552839" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552839">
        <p class="paperTitle">Translation as a Biosignature</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552839" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552839" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McKaig, J. M.; Kim, M.; Carr, C. E.</p>
        <p class="info">Score: 18.2, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552839' target='https://doi.org/10.1101/2023.08.10.552839'> 10.1101/2023.08.10.552839</a></p>
        <p class="abstract">Life on Earth relies on mechanisms to store heritable information and translate this information into cellular machinery required for biological activity. In all known life, storage, regulation, and translation are provided by DNA, RNA, and ribosomes. Life beyond Earth, even if ancestrally or chemically distinct from life as we know it may utilize similar structures: it has been proposed that charged linear polymers analogous to nucleic acids may be responsible for storage and regulation of genetic information in non-terran biochemical systems. We further propose that a ribosome-like structure may also exist in such a system, due to the evolutionary advantages of separating heritability from cellular machinery. Here, we use a solid-state nanopore to detect DNA, RNA, and ribosomes, and demonstrate that machine learning can distinguish between biomolecule samples and accurately classify new data. This work is intended to serve as a proof of principal that such biosignatures (i.e., informational polymers or translation apparatuses) could be detected, for example, as part of future missions targeting extant life on Ocean Worlds. A negative detection does not imply the absence of life; however, detection of ribosome-like structures could provide a robust and sensitive method to seek extant life in combination with other methods.

One Sentence SummaryLife, defined as a chemical system capable of Darwinian evolution, likely requires an apparatus to translate heritable instructions into cellular machinery, and we propose to detect this as a biosignature of extant life beyond Earth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553443">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553443" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553443">
        <p class="paperTitle">Robust and resource-optimal dynamic pattern formation of Min proteins in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553443" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553443" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: ren, z.; Weyer, H.; Wurthner, L.; Li, D.; Sou, C.; Villarreal, D.; Frey, E.; Jun, S.</p>
        <p class="info">Score: 15.9, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553443' target='https://doi.org/10.1101/2023.08.15.553443'> 10.1101/2023.08.15.553443</a></p>
        <p class="abstract">The Min system in Escherichia coli plays a crucial role in cellular reproduction by preventing minicell formation through pole-to-pole oscillations. Despite extensive research, predicting the onset of Min protein concentrations for oscillation and understanding the systems robustness under physiological perturbations remains challenging. Our study aims to address these gaps. We show that the Min systems dynamic pattern formation is robust across a wide range of Min protein levels and varying growth physiology. Using genetically engineered E. coli strains, we independently modulated the expression of minCD and minE in E. coli under both fast and slow growth conditions. This led to the construction of a MinD-MinE phase diagram, which revealed not just a large oscillation regime but also complex dynamic patterns such as traveling and standing waves. Interestingly, we found that the natural expression level of Min proteins is nearly optimal. Our work combines experimental findings with biophysical theory based on reaction-diffusion models, reproducing the experimental phase diagram and other key properties quantitatively. This includes the observation of an invariant wavelength of dynamic Min patterns across our phase diagram. Crucially, the success of our model depends on the switching of MinE between its latent and active states, indicating its essential role as a robustness module for Min oscillation in vivo. Our results underline the potential of integrating quantitative cell physiology and biophysical modeling in understanding the fundamental mechanisms controlling cell division machinery, offering insights applicable to other biological processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.20.554035">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.20.554035" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.20.554035">
        <p class="paperTitle">ATP Mediates Phase Separation of Disordered Basic Proteins by Bridging Intermolecular Interaction Networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.20.554035" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.20.554035" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kota, D.; Prasad, R.; Zhou, H.-X.</p>
        <p class="info">Score: 15.2, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.20.554035' target='https://doi.org/10.1101/2023.08.20.554035'> 10.1101/2023.08.20.554035</a></p>
        <p class="abstract">ATP is an abundant molecule with crucial cellular roles as the energy currency and a building block of nucleic acids and for protein phosphorylation. Here we show that ATP mediates the phase separation of basic intrinsically disordered proteins (bIDPs). In the resulting condensates, ATP is highly concentrated (apparent partition coefficients at 200-5000) and serves as bridges between bIDP chains. These liquid-like droplets have some of the lowest interfacial tension (~25 pN/m) but high zero-shear viscosities (1-15 Pa s) due to the bridged protein networks, and yet their fusion has some of the highest speeds (~1 m/ms). The rapid fusion manifests extreme shear thinning, where the apparent viscosity is lower than zero-shear viscosity by over 100-fold, made possible by fast reformation of the ATP bridges. At still higher concentrations, ATP does not dissolve bIDP droplets but results in aggregates and fibrils.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553409">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553409" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553409">
        <p class="paperTitle">Increasing the accuracy of single-molecule data analysis using tMAVEN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553409" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553409" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Verma, A. R.; Ray, K. K.; Bodick, M. E.; Kinz-Thompson, C. D.; Gonzalez, R. L.</p>
        <p class="info">Score: 13.2, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553409' target='https://doi.org/10.1101/2023.08.15.553409'> 10.1101/2023.08.15.553409</a></p>
        <p class="abstract">Time-dependent single-molecule experiments contain rich kinetic information about the functional dynamics of biomolecules. A key step in extracting this information is the application of kinetic models, such as hidden Markov models (HMMs), which characterize the molecular mechanism governing the experimental system. Unfortunately, researchers rarely know the physico-chemical details of this molecular mechanism a priori, which raises questions about how to select the most appropriate kinetic model for a given single-molecule dataset and what consequences arise if the wrong model is chosen. To address these questions, we have developed and used time-series Modeling, Analysis, and Visualization ENvironment (tMAVEN), a comprehensive, open-source, and extensible software platform. tMAVEN can perform each step of the single-molecule analysis pipeline, from pre-processing to kinetic modeling to plotting, and has been designed to enable the analysis of a single-molecule dataset with multiple types of kinetic models. Using tMAVEN, we have systematically investigated mismatches between kinetic models and molecular mechanisms by analyzing simulated examples of prototypical single-molecule datasets exhibiting common experimental complications, such as molecular heterogeneity, with a series of different types of HMMs. Our results show that no single kinetic modeling strategy is mathematically appropriate for all experimental contexts. Indeed, HMMs only correctly capture the underlying molecular mechanism in the simplest of cases. As such, researchers must modify HMMs using physico-chemical principles to avoid the risk of missing the significant biological and biophysical insights into molecular heterogeneity that their experiments provide. By enabling the facile, side-by-side application of multiple types of kinetic models to individual single-molecule datasets, tMAVEN allows researchers to carefully tailor their modeling approach to match the complexity of the underlying biomolecular dynamics and increase the accuracy of their single-molecule data analyses.

Statement of SignificanceThe power of time-dependent single-molecule biophysical experiments lies in their ability to uncover the molecular mechanisms governing experimental systems by computationally applying kinetic models to the data. While many software solutions have been developed to estimate the optimal parameters of such models, the results reported here show that the models themselves are often inherently mismatched with the molecular mechanisms they are being used to analyze. To investigate these mismatches and demonstrate how to best model the kinetics of a molecular mechanism, we have used time-series Modeling, Analysis, and Visualization ENvironment (tMAVEN), an open-source software platform we have developed that, among other features, enables the analysis of single-molecule datasets using different kinetic models within a single, extensible, and customizable pipeline.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553839">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553839" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553839">
        <p class="paperTitle">Nanopores reveal the stoichiometry of single oligo-adenylates produced by type III CRISPR-Cas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553839" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553839" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fuentenebro-Navas, D.; Steens, J. A.; De Lannoy, C.; Noordijk, B.; de Ridder, D.; Staals, R. H. J.; Schmid, S.</p>
        <p class="info">Score: 12.5, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553839' target='https://doi.org/10.1101/2023.08.18.553839'> 10.1101/2023.08.18.553839</a></p>
        <p class="abstract">Cyclic oligoadenylates (cOAs) are small second messenger molecules produced by the type III CRISPR-Cas system as part of the prokaryotic immune response. The role of cOAs is to allosterically activate downstream effector proteins that induce dormancy or cell death, and thus abort viral spread through the population. Interestingly, different type III systems have been reported to utilize different cOA stoichiometries (with 3 to 6 adenylate monophosphates). However, so far, their characterization has only been possible in bulk and with sophisticated equipment, while a portable assay with single-molecule resolution has been lacking. Here, we demonstrate the label-free detection of single cOA molecules using a simple protein nanopore assay. It sensitively identifies the stoichiometry of individual cOA molecules and their mixtures from synthetic and enzymatic origin. To achieve this, we trained a convolutional neural network (CNN) and validated it with a series of experiments on mono- and polydisperse cOA samples. Ultimately, we determined the stoichiometric composition of cOAs produced enzymatically by the CRISPR type III-A and III-B variants of Thermus thermophilus. Interestingly, both variants produce cOAs of nearly identical composition, and we discuss the biological implications of this finding. The presented nanopore-CNN workflow with single-cOA resolution can be adapted to many other signaling molecules (including eukaryotic ones), and it may be integrated into portable handheld devices with potential point-of-care applications.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
