<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Connecting Chromatin Structures to Gene Regulation Using Dynamic Polymer Simulations
Authors: Fu, Y.; Clark, F. T.; Nomikou, S.; Tsirigos, A.; Lionnet, T.
Score: 32.3, Published: 2023-11-10 DOI: 10.1101/2023.11.07.566032
The transfer of regulatory information between distal loci on chromatin is thought to involve physical proximity, but key biophysical features of these contacts remain unclear. For instance, it is unknown how close and for how long two loci need to be in order to productively interact.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Connecting Chromatin Structures to Gene Regulation Using Dynamic Polymer Simulations
Authors: Fu, Y.; Clark, F. T.; Nomikou, S.; Tsirigos, A.; Lionnet, T.
Score: 32.3, Published: 2023-11-10 DOI: 10.1101/2023.11.07.566032
The transfer of regulatory information between distal loci on chromatin is thought to involve physical proximity, but key biophysical features of these contacts remain unclear. For instance, it is unknown how close and for how long two loci need to be in order to productively interact." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-22T10:41:31+00:00" />
<meta property="article:modified_time" content="2023-11-22T10:41:31+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Connecting Chromatin Structures to Gene Regulation Using Dynamic Polymer Simulations
Authors: Fu, Y.; Clark, F. T.; Nomikou, S.; Tsirigos, A.; Lionnet, T.
Score: 32.3, Published: 2023-11-10 DOI: 10.1101/2023.11.07.566032
The transfer of regulatory information between distal loci on chromatin is thought to involve physical proximity, but key biophysical features of these contacts remain unclear. For instance, it is unknown how close and for how long two loci need to be in order to productively interact."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Connecting Chromatin Structures to Gene Regulation Using Dynamic Polymer Simulations\nAuthors: Fu, Y.; Clark, F. T.; Nomikou, S.; Tsirigos, A.; Lionnet, T.\nScore: 32.3, Published: 2023-11-10 DOI: 10.1101/2023.11.07.566032\nThe transfer of regulatory information between distal loci on chromatin is thought to involve physical proximity, but key biophysical features of these contacts remain unclear. For instance, it is unknown how close and for how long two loci need to be in order to productively interact.",
  "keywords": [
    
  ],
  "articleBody": " Connecting Chromatin Structures to Gene Regulation Using Dynamic Polymer Simulations\nAuthors: Fu, Y.; Clark, F. T.; Nomikou, S.; Tsirigos, A.; Lionnet, T.\nScore: 32.3, Published: 2023-11-10 DOI: 10.1101/2023.11.07.566032\nThe transfer of regulatory information between distal loci on chromatin is thought to involve physical proximity, but key biophysical features of these contacts remain unclear. For instance, it is unknown how close and for how long two loci need to be in order to productively interact. The main challenge is that it is currently impossible to measure chromatin dynamics with high spatiotemporal resolution at scale. Polymer simulations provide an accessible and rigorous way to test biophysical models of chromatin regulation, yet there is a lack of simple and general methods for extracting the values of model parameters. Here we adapt the Nelder-Mead simplex optimization algorithm to select the best polymer model matching a given Hi-C dataset, using the MYC locus as an example. The models biophysical parameters predict a compartmental rearrangement of the MYC locus in leukemia, which we validate with single-cell measurements. Leveraging trajectories predicted by the model, we find that loci with similar Hi-C contact frequencies can exhibit widely different contact dynamics. Interestingly, the frequency of productive interactions between loci exhibits a non-linear relationship with their Hi-C contact frequency when we enforce a specific capture radius and contact duration. These observations are consistent with recent experimental observations and suggest that the dynamic ensemble of chromatin configurations, rather than average contact matrices, is required to fully predict long-range chromatin interactions.\nThe mechanism of mRNA activation\nAuthors: Gentry, R. C.; Ide, N. A.; Comunale, V. M.; Hartwick, E. W.; Kinz-Thompson, C. D.; Gonzalez, R. L.\nScore: 25.3, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567265\nDuring translation initiation, messenger RNA molecules must be identified and activated for loading into a ribosome. In this rate-limiting step, the heterotrimeric protein eukaryotic initiation factor eIF4F must recognize and productively interact with the 7-methylguanosine cap at the 5 end of the messenger RNA and subsequently activate the message. Despite its fundamental, regulatory role in gene expression, the molecular events underlying cap recognition and messenger RNA activation remain mysterious. Here, we generate a unique, single-molecule fluorescence imaging system to interrogate the dynamics with which eIF4F discriminates productive and non-productive locations on full-length, native messenger RNA molecules. At the single-molecule level, we observe stochastic sampling of eIF4F along the length of the messenger RNA and identify allosteric communication between the eIF4F subunits which ultimately drive cap-recognition and subsequent activation of the message. Our experiments uncover novel functions for each subunit of eIF4F and we conclude by presenting a model for messenger RNA activation which precisely defines the composition of the activated message. This model provides a general framework for understanding how messenger RNA molecules may be discriminated from one another, and how other RNA-binding proteins may control the efficiency of translation initiation.\nReal-Time Structural Biology of DNA and DNA-Protein Complexes on an Optical Microscope\nAuthors: Szalai, A. M.; Ferrari, G.; Richter, L.; Hartmann, J.; Kesici, M.-Z.; Ji, B.; Koshic, K.; Jaeger, A.; Aksimentiev, A.; Tessmer, I.; Kaminska, I.; Vera, A. M.; Tinnefeld, P.\nScore: 19.2, Published: 2023-11-21 DOI: 10.1101/2023.11.21.567962\nThe intricate interplay between DNA and proteins is key for biological functions such as DNA replication, transcription, and repair. To better understand these interactions, it is crucial to develop tools to study DNA-protein complexes with high spatial and temporal resolution. Here, we use the vertical orientation that DNA adopts on graphene and investigate its interactions with proteins via energy transfer from a probe dye to graphene, achieving spatial resolution down to the Ansgtrom scale. We measured the bending angle of DNA induced by adenine tracts, bulges, abasic sites and the binding of Escherichia coli endonuclease IV with unprecedented precision and millisecond time resolution. Additionally, we observed the translocation of the O6-alkylguanine DNA alkyltransferase along double-stranded DNA, reaching single-base pair resolution and detecting an affinity for adenine tracts. Overall, we foresee that this method will become a widespread tool for the dynamical study of nucleic acid and nucleic acid-protein interactions.\nSubstrate accessibility regulation of human TopIIa decatenation by cohesin\nAuthors: Cutts, E.; Saravanan, S.; Fisher, G.; Rueda, D. S.; Aragon, L.\nScore: 14.4, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567865\nHuman topoisomerase II alpha (TOP2) is the main mitotic decatenase 1-3 resolving intertwines between sister chromatids that form during DNA replication 4. Here, we employ quadruple-trap optical tweezers to generate braids between a pair of l-DNA molecules to study, at the single-molecule level and in real-time, how TOP2 untangles these DNAs. We find that TOP2 rapidly resolves single and multiple braids and is inhibited by chemotherapy agent, etoposide. TOP2 is sensitive to DNA conformation, exhibiting a chiral preference for the removal of braids with right-handed crossings and is inhibited when DNAs are held at forces over 20pN. We show that TOP2 must load at the cross between the two DNAs to resolve braids efficiently. TOP2 pre-loaded onto individual DNAs is unable to resolve newly formed braids in the presence of ATP, suggesting that the geometry of pre-bound enzyme is incompatible with the capture of the second DNA. Finally, we show that cohesin binds to braids, preventing TOP2 from resolving them. Our study unveils novel insights into the regulation of TOP2's decatenation, underscoring the importance of substrate accessibility and the role of cohesin in modulating TOP2 activity.\nProtein Stability Prediction by Fine-tuning a Protein Language Model on a Mega-scale Dataset\nAuthors: Chu, K. S.; Siegel, J. B.\nScore: 11.1, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567747\nThe stability of a protein is crucial to its utility in industrial applications. While engineering campaigns can now be routinely used to enhance protein thermal stability to the level needed in an industrial setting, there is a significant desire to fast-track these efforts through predictive tools allowing one to jump in a minimal number of design iterations to a highly stabilized protein. In this work, we explore utilizing a mega-scale dataset for development of a protein language model tuned for stability. This model is trained on the folding stability of 528k sequences derived from 461 small protein domains and designs, and can accommodate deletions, insertions, and multiple-point mutations. We show that a protein language model can be fine-tuned to predict folding stability. The fine-tuned protein language model, named ESMtherm, performs reasonably on small protein domains and generalizes to sequences distal from the training set. Lastly, we discuss its limitations when compared to other state-of-the-art methods in generalizing to larger protein scaffolds and highlight the need of large-scale stability measurement on a diverse dataset that represents the distribution of sequence lengths commonly observed in nature.\nDeep learning assisted single particle tracking for automated correlation between diffusion and function\nAuthors: Kaestel-Hansen, J.; de Sautu, M.; Saminathan, A.; Scanavachi, G.; Correia, R. F. B. D. C.; Nielsen, A. J.; Bleshoy, S. V.; Boomsma, W.; Kirchhausen, T.; Hatzakis, N. S.\nScore: 10.2, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567393\nSub-cellular diffusion in living systems reflects cellular processes and interactions. Recent advances in optical microscopy allow the tracking of this nanoscale diffusion of individual objects with an unprecedented level of precision. However, the agnostic and automated extraction of functional information from the diffusion of molecules and organelles within the sub-cellular environment, is labor-intensive and poses a significant challenge. Here we introduce DeepSPT, a deep learning framework to interpret the diffusional 2D or 3D temporal behavior of objects in a rapid and efficient manner, agnostically. Demonstrating its versatility, we have applied DeepSPT to automated mapping of the early events of viral infections, identifying distinct types of endosomal organelles, and clathrin-coated pits and vesicles with up to 95% accuracy and within seconds instead of weeks. The fact that DeepSPT effectively extracts biological information from diffusion alone indicates that besides structure, motion encodes function at the molecular and subcellular level.\nSpatially resolved DNP-assisted NMR illuminates the conformational ensemble of α-synuclein in intact viable cells.\nAuthors: Kragelj, J.; Ghosh, R.; Xiao, Y.; Dumarieh, R.; Lagasca, D.; Krishna, S.; Frederick, K. K.\nScore: 12.1, Published: 2023-10-25 DOI: 10.1101/2023.10.24.563877\nThe protein -syn adopts a wide variety of conformations including an intrinsically disordered monomeric form and an -helical rich membrane-associated form that is thought to play an important role in cellular membrane processes. However, despite the high affinity of -syn for membranes, evidence that the -helical form of -syn is adopted inside cells has thus far been indirect. In cell DNP-assisted solid state NMR on frozen samples has the potential to report directly on the entire conformational ensemble. Moreover, because the DNP polarization agent can be dispersed both homogenously and inhomogenously throughout the cellular biomass, in cell DNP-assisted solid state NMR experiments can report either quantitatively upon the structural ensemble or can preferentially report upon the structural ensemble with a spatial bias. Using DNP-assisted MAS NMR we establish that the spectra of purified -syn in the membrane-associated and intrinsically disordered forms have distinguishable spectra. When the polarization agent is introduced into cells by electroporation and dispersed homogenously, a minority of the -syn inside HEK293 cells adopts a highly -helical rich conformation. Alteration of the spatial distribution of the polarization agent preferentially enhances the signal from molecules nearer to the cellular periphery, thus the -helical rich population is preferentially adopted toward the cellular periphery. This demonstrates how selectively altering the spatial distribution of the DNP polarization agent can be a powerful tool for preferential reporting on specific structural ensembles, paving the way for more nuanced investigations into the conformations that proteins adopt in different areas of the cell.\nRaman microscopy of cryofixed biological specimens for high-resolution and high-sensitivity chemical imaging\nAuthors: Mizushima, K.; Kumamoto, Y.; Tamura, S.; Yamanaka, M.; Mochizuki, K.; Li, M.; Egoshi, S.; Dodo, K.; Harada, Y.; Smith, N. I.; Sodeoka, M.; Tanaka, H.; Fujita, K.\nScore: 4.2, Published: 2023-11-16 DOI: 10.1101/2023.11.15.567077\nRaman microscopy is an emerging molecular imaging technology, yet its signal-to-noise-ratio (SNR) in measurements of biological specimens is severely limited due to the small cross-section of Raman scattering. Here, we present Raman imaging techniques of cryofixed specimens to overcome SNR limitations by enabling long exposure of specimens under highly stabilized low-temperature conditions. The observation of frozen specimens in a cryostat at a constant low temperature immediately after rapid freezing enabled the improvement of SNR and significantly enhanced spatial and spectral resolution. We also confirmed that the cryofixation can preserve physicochemical states of specimens by observing alkyne-labeled coenzyme Q in cytosol and hemeproteins in acute ischemic myocardium, which cannot be done by fixation using chemical reagents. Finally, we applied the technique for multiplex Raman imaging of label-free endogenous molecules and alkyne-tagged molecules in cryofixed HeLa cells, demonstrating its capability of high-content imaging of complex biological phenomena while maintaining physiological conditions.\nLive-cell imaging reveals the trade-off between target search flexibility and efficiency for Cas9 and Cas12a\nAuthors: Olivi, L.; Bagchus, C.; Pool, V.; Bekkering, E.; Speckner, K.; Wu, W.; Martens, K.; van der Oost, J.; Staals, R.; Hohlbein, J.\nScore: 3.9, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567366\nCRISPR-Cas systems have widely been adopted as genome editing tools, with two frequently employed Cas nucleases being SpyCas9 and LbCas12a. Although both nucleases use RNA guides to find and cleave target DNA sites, the two enzymes differ in terms of protospacer-adjacent motif (PAM) requirements, guide architecture and cleavage mechanism. In the last years, rational engineering led to the creation of PAM-relaxed variants SpRYCas9 and impLbCas12a to broaden the targetable DNA space. By employing their catalytically inactive variants (dCas9/dCas12a), we quantified how the protein-specific characteristics impact the target search process. To allow quantification, we fused these nucleases to the photoactivatable fluorescent protein PAmCherry2.1 and performed single-particle tracking in cells of Escherichia coli. From our tracking analysis, we derived kinetic parameters for each nuclease with a non-targeting RNA guide, strongly suggesting that interrogation of DNA by LbdCas12a variants proceeds faster than that of SpydCas9. In the presence of a targeting RNA guide, both simulations and imaging of cells confirmed that LbdCas12a variants are faster and more efficient in finding a specific target site. Our work demonstrates the trade-off of relaxing PAM requirements in SpydCas9 and LbdCas12a using a powerful framework, which can be applied to other nucleases to quantify their DNA target search.\nAlphaFold-SFA: accelerated sampling of cryptic pocket opening by fusing AlphaFold, slow feature analysis and metadynamics\nAuthors: Vats, S.; Söderhjelm, P.; Bhakat, S.\nScore: 3.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568098\nSampling rare events in proteins is crucial for comprehending complex phenomena like cryptic pocket opening, where transient structural changes expose new binding sites. Understanding these rare events also sheds light on protein-ligand binding and allosteric communications, where distant site interactions influence protein function. Traditional unbiased molecular dynamics simulations often fail to sample such rare events, as the free energy barrier between metastable states is large relative to the thermal energy. This renders these events inaccessible on the timescales typically simulated by standard molecular dynamics, limiting our understanding of these critical processes. In this working paper, we proposed a novel unsupervised learning approach termed as slow feature analysis (SFA) which aims to extract slowly varying features from high-dimensional temporal data. SFA trained on small unbiased molecular dynamics simulations launched from AlphaFold generated conformational ensembles manages to capture rare events governing cryptic pocket opening in plasmepsin II. Metadynamics simulations using SFA as collective variables manage to sample 'deep' cryptic pocket opening within a few hundreds of nanoseconds which was beyond the reach of microsecond long unbiased molecular dynamics simulations. Taken together, our results show how SFA acts as a dimensionality reduction tool which bridges the gap between AlphaFold, molecular dynamics simulation and metadynamics in context of capturing rare events in biomolecules, extending the scope of structure-based drug discovery in the era of AlphaFold.\n",
  "wordCount" : "2256",
  "inLanguage": "en",
  "datePublished": "2023-11-22T10:41:31Z",
  "dateModified": "2023-11-22T10:41:31Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 22, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566032">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566032" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566032">
        <p class="paperTitle">Connecting Chromatin Structures to Gene Regulation Using Dynamic Polymer Simulations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566032" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566032" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fu, Y.; Clark, F. T.; Nomikou, S.; Tsirigos, A.; Lionnet, T.</p>
        <p class="info">Score: 32.3, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566032' target='https://doi.org/10.1101/2023.11.07.566032'> 10.1101/2023.11.07.566032</a></p>
        <p class="abstract">The transfer of regulatory information between distal loci on chromatin is thought to involve physical proximity, but key biophysical features of these contacts remain unclear. For instance, it is unknown how close and for how long two loci need to be in order to productively interact. The main challenge is that it is currently impossible to measure chromatin dynamics with high spatiotemporal resolution at scale. Polymer simulations provide an accessible and rigorous way to test biophysical models of chromatin regulation, yet there is a lack of simple and general methods for extracting the values of model parameters. Here we adapt the Nelder-Mead simplex optimization algorithm to select the best polymer model matching a given Hi-C dataset, using the MYC locus as an example. The models biophysical parameters predict a compartmental rearrangement of the MYC locus in leukemia, which we validate with single-cell measurements. Leveraging trajectories predicted by the model, we find that loci with similar Hi-C contact frequencies can exhibit widely different contact dynamics. Interestingly, the frequency of productive interactions between loci exhibits a non-linear relationship with their Hi-C contact frequency when we enforce a specific capture radius and contact duration. These observations are consistent with recent experimental observations and suggest that the dynamic ensemble of chromatin configurations, rather than average contact matrices, is required to fully predict long-range chromatin interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567265" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567265">
        <p class="paperTitle">The mechanism of mRNA activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gentry, R. C.; Ide, N. A.; Comunale, V. M.; Hartwick, E. W.; Kinz-Thompson, C. D.; Gonzalez, R. L.</p>
        <p class="info">Score: 25.3, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567265' target='https://doi.org/10.1101/2023.11.15.567265'> 10.1101/2023.11.15.567265</a></p>
        <p class="abstract">During translation initiation, messenger RNA molecules must be identified and activated for loading into a ribosome. In this rate-limiting step, the heterotrimeric protein eukaryotic initiation factor eIF4F must recognize and productively interact with the 7-methylguanosine cap at the 5 end of the messenger RNA and subsequently activate the message. Despite its fundamental, regulatory role in gene expression, the molecular events underlying cap recognition and messenger RNA activation remain mysterious. Here, we generate a unique, single-molecule fluorescence imaging system to interrogate the dynamics with which eIF4F discriminates productive and non-productive locations on full-length, native messenger RNA molecules. At the single-molecule level, we observe stochastic sampling of eIF4F along the length of the messenger RNA and identify allosteric communication between the eIF4F subunits which ultimately drive cap-recognition and subsequent activation of the message. Our experiments uncover novel functions for each subunit of eIF4F and we conclude by presenting a model for messenger RNA activation which precisely defines the composition of the activated message. This model provides a general framework for understanding how messenger RNA molecules may be discriminated from one another, and how other RNA-binding proteins may control the efficiency of translation initiation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.567962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.567962" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.567962">
        <p class="paperTitle">Real-Time Structural Biology of DNA and DNA-Protein Complexes on an Optical Microscope</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.567962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.567962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Szalai, A. M.; Ferrari, G.; Richter, L.; Hartmann, J.; Kesici, M.-Z.; Ji, B.; Koshic, K.; Jaeger, A.; Aksimentiev, A.; Tessmer, I.; Kaminska, I.; Vera, A. M.; Tinnefeld, P.</p>
        <p class="info">Score: 19.2, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.567962' target='https://doi.org/10.1101/2023.11.21.567962'> 10.1101/2023.11.21.567962</a></p>
        <p class="abstract">The intricate interplay between DNA and proteins is key for biological functions such as DNA replication, transcription, and repair. To better understand these interactions, it is crucial to develop tools to study DNA-protein complexes with high spatial and temporal resolution. Here, we use the vertical orientation that DNA adopts on graphene and investigate its interactions with proteins via energy transfer from a probe dye to graphene, achieving spatial resolution down to the Ansgtrom scale. We measured the bending angle of DNA induced by adenine tracts, bulges, abasic sites and the binding of Escherichia coli endonuclease IV with unprecedented precision and millisecond time resolution. Additionally, we observed the translocation of the O6-alkylguanine DNA alkyltransferase along double-stranded DNA, reaching single-base pair resolution and detecting an affinity for adenine tracts. Overall, we foresee that this method will become a widespread tool for the dynamical study of nucleic acid and nucleic acid-protein interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567865">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567865" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567865">
        <p class="paperTitle">Substrate accessibility regulation of human TopIIa decatenation by cohesin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567865" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567865" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cutts, E.; Saravanan, S.; Fisher, G.; Rueda, D. S.; Aragon, L.</p>
        <p class="info">Score: 14.4, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567865' target='https://doi.org/10.1101/2023.11.20.567865'> 10.1101/2023.11.20.567865</a></p>
        <p class="abstract">Human topoisomerase II alpha (TOP2) is the main mitotic decatenase 1-3 resolving intertwines between sister chromatids that form during DNA replication 4. Here, we employ quadruple-trap optical tweezers to generate braids between a pair of l-DNA molecules to study, at the single-molecule level and in real-time, how TOP2 untangles these DNAs. We find that TOP2 rapidly resolves single and multiple braids and is inhibited by chemotherapy agent, etoposide. TOP2 is sensitive to DNA conformation, exhibiting a chiral preference for the removal of braids with right-handed crossings and is inhibited when DNAs are held at forces over 20pN. We show that TOP2 must load at the cross between the two DNAs to resolve braids efficiently. TOP2 pre-loaded onto individual DNAs is unable to resolve newly formed braids in the presence of ATP, suggesting that the geometry of pre-bound enzyme is incompatible with the capture of the second DNA. Finally, we show that cohesin binds to braids, preventing TOP2 from resolving them. Our study unveils novel insights into the regulation of TOP2&#39;s decatenation, underscoring the importance of substrate accessibility and the role of cohesin in modulating TOP2 activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567747">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567747" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567747">
        <p class="paperTitle">Protein Stability Prediction by Fine-tuning a Protein Language Model on a Mega-scale Dataset</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567747" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567747" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chu, K. S.; Siegel, J. B.</p>
        <p class="info">Score: 11.1, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567747' target='https://doi.org/10.1101/2023.11.19.567747'> 10.1101/2023.11.19.567747</a></p>
        <p class="abstract">The stability of a protein is crucial to its utility in industrial applications. While engineering campaigns can now be routinely used to enhance protein thermal stability to the level needed in an industrial setting, there is a significant desire to fast-track these efforts through predictive tools allowing one to jump in a minimal number of design iterations to a highly stabilized protein. In this work, we explore utilizing a mega-scale dataset for development of a protein language model tuned for stability. This model is trained on the folding stability of 528k sequences derived from 461 small protein domains and designs, and can accommodate deletions, insertions, and multiple-point mutations. We show that a protein language model can be fine-tuned to predict folding stability. The fine-tuned protein language model, named ESMtherm, performs reasonably on small protein domains and generalizes to sequences distal from the training set. Lastly, we discuss its limitations when compared to other state-of-the-art methods in generalizing to larger protein scaffolds and highlight the need of large-scale stability measurement on a diverse dataset that represents the distribution of sequence lengths commonly observed in nature.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567393">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567393" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567393">
        <p class="paperTitle">Deep learning assisted single particle tracking for automated correlation between diffusion and function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567393" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567393" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaestel-Hansen, J.; de Sautu, M.; Saminathan, A.; Scanavachi, G.; Correia, R. F. B. D. C.; Nielsen, A. J.; Bleshoy, S. V.; Boomsma, W.; Kirchhausen, T.; Hatzakis, N. S.</p>
        <p class="info">Score: 10.2, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567393' target='https://doi.org/10.1101/2023.11.16.567393'> 10.1101/2023.11.16.567393</a></p>
        <p class="abstract">Sub-cellular diffusion in living systems reflects cellular processes and interactions. Recent advances in optical microscopy allow the tracking of this nanoscale diffusion of individual objects with an unprecedented level of precision. However, the agnostic and automated extraction of functional information from the diffusion of molecules and organelles within the sub-cellular environment, is labor-intensive and poses a significant challenge. Here we introduce DeepSPT, a deep learning framework to interpret the diffusional 2D or 3D temporal behavior of objects in a rapid and efficient manner, agnostically. Demonstrating its versatility, we have applied DeepSPT to automated mapping of the early events of viral infections, identifying distinct types of endosomal organelles, and clathrin-coated pits and vesicles with up to 95% accuracy and within seconds instead of weeks. The fact that DeepSPT effectively extracts biological information from diffusion alone indicates that besides structure, motion encodes function at the molecular and subcellular level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563877">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563877" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563877">
        <p class="paperTitle">Spatially resolved DNP-assisted NMR illuminates the conformational ensemble of α-synuclein in intact viable cells.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563877" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563877" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kragelj, J.; Ghosh, R.; Xiao, Y.; Dumarieh, R.; Lagasca, D.; Krishna, S.; Frederick, K. K.</p>
        <p class="info">Score: 12.1, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563877' target='https://doi.org/10.1101/2023.10.24.563877'> 10.1101/2023.10.24.563877</a></p>
        <p class="abstract">The protein -syn adopts a wide variety of conformations including an intrinsically disordered monomeric form and an -helical rich membrane-associated form that is thought to play an important role in cellular membrane processes. However, despite the high affinity of -syn for membranes, evidence that the -helical form of -syn is adopted inside cells has thus far been indirect. In cell DNP-assisted solid state NMR on frozen samples has the potential to report directly on the entire conformational ensemble. Moreover, because the DNP polarization agent can be dispersed both homogenously and inhomogenously throughout the cellular biomass, in cell DNP-assisted solid state NMR experiments can report either quantitatively upon the structural ensemble or can preferentially report upon the structural ensemble with a spatial bias. Using DNP-assisted MAS NMR we establish that the spectra of purified -syn in the membrane-associated and intrinsically disordered forms have distinguishable spectra. When the polarization agent is introduced into cells by electroporation and dispersed homogenously, a minority of the -syn inside HEK293 cells adopts a highly -helical rich conformation. Alteration of the spatial distribution of the polarization agent preferentially enhances the signal from molecules nearer to the cellular periphery, thus the -helical rich population is preferentially adopted toward the cellular periphery. This demonstrates how selectively altering the spatial distribution of the DNP polarization agent can be a powerful tool for preferential reporting on specific structural ensembles, paving the way for more nuanced investigations into the conformations that proteins adopt in different areas of the cell.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567077">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567077" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567077">
        <p class="paperTitle">Raman microscopy of cryofixed biological specimens for high-resolution and high-sensitivity chemical imaging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567077" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567077" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mizushima, K.; Kumamoto, Y.; Tamura, S.; Yamanaka, M.; Mochizuki, K.; Li, M.; Egoshi, S.; Dodo, K.; Harada, Y.; Smith, N. I.; Sodeoka, M.; Tanaka, H.; Fujita, K.</p>
        <p class="info">Score: 4.2, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567077' target='https://doi.org/10.1101/2023.11.15.567077'> 10.1101/2023.11.15.567077</a></p>
        <p class="abstract">Raman microscopy is an emerging molecular imaging technology, yet its signal-to-noise-ratio (SNR) in measurements of biological specimens is severely limited due to the small cross-section of Raman scattering. Here, we present Raman imaging techniques of cryofixed specimens to overcome SNR limitations by enabling long exposure of specimens under highly stabilized low-temperature conditions. The observation of frozen specimens in a cryostat at a constant low temperature immediately after rapid freezing enabled the improvement of SNR and significantly enhanced spatial and spectral resolution. We also confirmed that the cryofixation can preserve physicochemical states of specimens by observing alkyne-labeled coenzyme Q in cytosol and hemeproteins in acute ischemic myocardium, which cannot be done by fixation using chemical reagents. Finally, we applied the technique for multiplex Raman imaging of label-free endogenous molecules and alkyne-tagged molecules in cryofixed HeLa cells, demonstrating its capability of high-content imaging of complex biological phenomena while maintaining physiological conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567366">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567366" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567366">
        <p class="paperTitle">Live-cell imaging reveals the trade-off between target search flexibility and efficiency for Cas9 and Cas12a</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567366" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567366" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Olivi, L.; Bagchus, C.; Pool, V.; Bekkering, E.; Speckner, K.; Wu, W.; Martens, K.; van der Oost, J.; Staals, R.; Hohlbein, J.</p>
        <p class="info">Score: 3.9, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567366' target='https://doi.org/10.1101/2023.11.16.567366'> 10.1101/2023.11.16.567366</a></p>
        <p class="abstract">CRISPR-Cas systems have widely been adopted as genome editing tools, with two frequently employed Cas nucleases being SpyCas9 and LbCas12a. Although both nucleases use RNA guides to find and cleave target DNA sites, the two enzymes differ in terms of protospacer-adjacent motif (PAM) requirements, guide architecture and cleavage mechanism. In the last years, rational engineering led to the creation of PAM-relaxed variants SpRYCas9 and impLbCas12a to broaden the targetable DNA space. By employing their catalytically inactive variants (dCas9/dCas12a), we quantified how the protein-specific characteristics impact the target search process. To allow quantification, we fused these nucleases to the photoactivatable fluorescent protein PAmCherry2.1 and performed single-particle tracking in cells of Escherichia coli. From our tracking analysis, we derived kinetic parameters for each nuclease with a non-targeting RNA guide, strongly suggesting that interrogation of DNA by LbdCas12a variants proceeds faster than that of SpydCas9. In the presence of a targeting RNA guide, both simulations and imaging of cells confirmed that LbdCas12a variants are faster and more efficient in finding a specific target site. Our work demonstrates the trade-off of relaxing PAM requirements in SpydCas9 and LbdCas12a using a powerful framework, which can be applied to other nucleases to quantify their DNA target search.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568098">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568098" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568098">
        <p class="paperTitle">AlphaFold-SFA: accelerated sampling of cryptic pocket opening by fusing AlphaFold, slow feature analysis and metadynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568098" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568098" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vats, S.; Söderhjelm, P.; Bhakat, S.</p>
        <p class="info">Score: 3.8, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568098' target='https://doi.org/10.1101/2023.11.21.568098'> 10.1101/2023.11.21.568098</a></p>
        <p class="abstract">Sampling rare events in proteins is crucial for comprehending complex phenomena like cryptic pocket opening, where transient structural changes expose new binding sites. Understanding these rare events also sheds light on protein-ligand binding and allosteric communications, where distant site interactions influence protein function. Traditional unbiased molecular dynamics simulations often fail to sample such rare events, as the free energy barrier between metastable states is large relative to the thermal energy. This renders these events inaccessible on the timescales typically simulated by standard molecular dynamics, limiting our understanding of these critical processes. In this working paper, we proposed a novel unsupervised learning approach termed as slow feature analysis (SFA) which aims to extract slowly varying features from high-dimensional temporal data. SFA trained on small unbiased molecular dynamics simulations launched from AlphaFold generated conformational ensembles manages to capture rare events governing cryptic pocket opening in plasmepsin II. Metadynamics simulations using SFA as collective variables manage to sample &#39;deep&#39; cryptic pocket opening within a few hundreds of nanoseconds which was beyond the reach of microsecond long unbiased molecular dynamics simulations. Taken together, our results show how SFA acts as a dimensionality reduction tool which bridges the gap between AlphaFold, molecular dynamics simulation and metadynamics in context of capturing rare events in biomolecules, extending the scope of structure-based drug discovery in the era of AlphaFold.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
