<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Quantification of cell phenotype transition manifolds with information geometry
Authors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.
Score: 18.5, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573500
Cell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Quantification of cell phenotype transition manifolds with information geometry
Authors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.
Score: 18.5, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573500
Cell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-07T10:38:59+00:00" />
<meta property="article:modified_time" content="2024-01-07T10:38:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Quantification of cell phenotype transition manifolds with information geometry
Authors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.
Score: 18.5, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573500
Cell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Quantification of cell phenotype transition manifolds with information geometry\nAuthors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.\nScore: 18.5, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573500\nCell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds.",
  "keywords": [
    
  ],
  "articleBody": " Quantification of cell phenotype transition manifolds with information geometry\nAuthors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.\nScore: 18.5, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573500\nCell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds. However, no available method currently exists for quantifying the manifolds from experimental data. Current methods, such as manifold learning, can only preserve topology information and can only be used for visualization. In this work, we present a novel method to quantify the cell phenotype transition manifolds using information geometry. Specifically, we transform single cells high-dimensional expression vectors into probability distributions with Gaussian embedding. The Fisher-Rao metric is then naturally defined in this embedding space. With Gaussian embedding, we calculate the Ollivier-Ricci curvature of each single cell. Our analyses revealed that cells with low curvature are related to critical transitions.To further analyze the invariant characteristics of the manifold in CPT, we calculated the information velocity of each single cell based on RNA velocity. The high information velocity regions correspond to the low curvature regions, indicating that the geometry can guide the dynamics of single cells on the manifolds. This method focuses on analyzing the invariant characteristics of the manifold in CPT, and the results illustrate that it can be a general approach for quantifying the CPT manifold.\nSingle-molecule detection and super-resolution imaging with a portable and adaptable 3D-printed microscopy platform (Brick-MIC)\nAuthors: Moya Munoz, G. G.; Brix, O.; Klocke, P.; Wendler, N. D.; Lerner, E.; Zijlstra, N.; Cordes, T.\nScore: 15.4, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573596\nOver the past decades, single-molecule spectroscopy and super-resolution microscopy have advanced significantly and by now represent important tools for life science research. Despite rapid progress and ongoing development, there is a growing gap between the state-of-the-art and what is accessible to non-optics specialists, e.g., biologists, biochemists, medical researchers, and labs with financial constraints. To bridge this gap, we introduce Brick-MIC, a versatile and affordable open-source 3D-printed micro-spectroscopy and imaging platform. Brick-MIC enables the integration of various fluorescence imaging techniques with single-molecule resolution within a single platform and enables exchange between different modalities within minutes. In this work, we present three variants of Brick-MIC that facilitate single-molecule fluorescence detection, fluorescence correlation spectroscopy and super-resolution imaging. With the three variants, we were able to observe conformational changes and absolute inter-dye distances in single macromolecules and perform single-molecule localization microscopy (STORM and PAINT) of DNA origami nanostructures. Detailed descriptions of the hardware and software components, as well as data analysis routines are provided, to allow non-optics specialist to operate their own Brick-MIC with minimal effort and investments. We foresee that our affordable, flexible, and open-source Brick-MIC platform will be a valuable tool for many laboratories worldwide.\nThe everting Drosophila wing disc is a shape-programmed material\nAuthors: Fuhrmann, J. F.; Krishna, A.; Paijmans, J.; Duclut, C. A.; Eaton, S.; Popovic, M.; Julicher, F.; Modes, C. D.; Dye, N. A.\nScore: 9.9, Published: 2023-12-24 DOI: 10.1101/2023.12.23.573034\nHow complex 3D tissue shape emerges during animal development remains an important open question in biology and biophysics. Although it is well-established that tissue-scale patterns of in-plane cell behaviors can reshape a tissue in 2D, whether and how such patterns could also deform a tissue in 3D remains unclear. Meanwhile, recent advances in materials science, physics, and engineering have enhanced our ability to craft and understand inanimate \"shape-programmable\" materials capable of undergoing blueprinted 3D shape transformations arising from in-plane gradients of spontaneous strains. Here, we use this conceptual framework to understand eversion of the Drosophila wing disc, a complex 3D tissue reshaping event in wing development. Using quantification of spatial patterns of cell dynamics on an evolving 3D tissue surface, combined with physical modeling and genetic perturbation, we demonstrate that the wing can deform itself in 3D through large-scale patterning of in-plane cellular behaviors, primarily rearrangements. This work shows that concepts from shape-programming can be employed to understand animal tissue morphogenesis and suggests that there exist intricate patterns in nature that could present novel designs for shape-programmable materials.\nOblique Line Scan Illumination Enables Expansive, Accurate and Sensitive Single Protein Measurements in Solution and in Living Cells\nAuthors: Driouchi, A.; Bretan, M.; Davis, B. J.; Heckert, A. B.; Seeger, M.; Bradley Silva, M.; Forrest, W. S.; Hsiung, J.; Tan, J.; Yang, H.; McSwiggen, D. T.; Song, L.; Sule, A.; Abaie, B.; Conroy, B.; Elliott, L. A.; Gonzalez, E.; Ilkov, F.; Isaacs, J.; Labaria, G.; Lagana, M.; Larsen, D. D.; Margolin, B.; Nguyen, M. K.; Rine, J.; Tang, Y.; Vana, M.; Wilkey, A.; Zhang, Z.; Basham, S.; Ho, J. J.; Johnson, S.; Klammer, A. A.; Lin, K.; Darzacq, X.; Betzig, E.; Berman, R. T.; Anderson, D. J.\nScore: 8.1, Published: 2023-12-23 DOI: 10.1101/2023.12.21.571765\nSingle-molecule localization microscopy (SMLM) techniques, such as single-molecule tracking (SMT), enable in situ measurements in cells from which data-rich metrics can be extracted. SMT has been successfully applied to a variety of biological questions and model systems, aiming to unravel the spatiotemporal regulation of molecular mechanisms that govern protein function, downstream pathway effects, and cellular function. While powerful, SMLM often suffers from low throughput and illumination inhomogeneity, along with microscope and user-induced technical biases. Due to technical limitations in scaling SMLM techniques, a tradeoff between spatiotemporal resolution and throughput has been made historically, restricting broad application of these technologies. Here we address these limitations using Oblique Line Scan (OLS), a robust single-objective light-sheet based illumination and detection modality that achieves nanoscale spatial resolution and sub-millisecond temporal resolution across a 250 x 190 m field of view. We demonstrate OLS-enabled SMT on Halo-Tagged proteins in living cells capturing protein motion up to 14 m2 /s. By exploiting the adaptability of the acquisition frame rate and the improved rejection of out of focus light, we extend the utility of OLS beyond cellular compartments with in-solution SMT (isSMT) for single-molecule measurement of ligand-protein interactions and disruption of protein-protein interactions (PPI). We illustrate the versatility of OLS by showcasing two-color SMT, STORM, and single molecule fluorescence recovery after photobleaching (FRAP). OLS expands the range of SMLM applications and paves the way for robust, high-throughput single-molecule investigations of protein dynamics required for drug screening and systems biology studies, both in cells and in solution.\nDe novo design of drug-binding proteins with predictable binding energy and specificity\nAuthors: Lu, L.; Gou, X.; Tan, S.; Mann, S.; Yang, H.; Zhong, X.; Gazgalis, D.; Valdiviezo, J.; Jo, H.; Wu, Y.; Diolaiti, M.; Ashworth, A.; Polizzi, N.; DeGrado, W.\nScore: 18.4, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573178\nThe de novo design of small-molecule-binding proteins has seen exciting recent progress; however, the ability to achieve exquisite affinity for binding small molecules while tuning specificity has not yet been demonstrated directly from computation. Here, we develop a computational procedure that results in the highest affinity binders to date with predetermined relative affinities, targeting a series of PARP1 inhibitors. Two of four designed proteins bound with affinities ranging from \u003c 5 nM to low M, in a predictable manner. X-ray crystal structures confirmed the accuracy of the designed protein-drug interactions. Molecular dynamics simulations informed the role of water in binding. Binding free-energy calculations performed directly on the designed models are in excellent agreement with the experimentally measured affinities, suggesting that the de novo design of small-molecule-binding proteins with tuned interaction energies is now feasible entirely from computation. We expect these methods to open many opportunities in biomedicine, including rapid sensor development, antidote design, and drug delivery vehicles. One Sentence SummaryWe use informatic sampling to design low nM drug-binding proteins, and physics-based calculations to accurately predict affinities.\nLive-cell imaging under centrifugation characterized the cellular force for nuclear centration in the Caenorhabditis elegans embryo\nAuthors: Goda, M.; Shribak, M.; Ikeda, Z.; Okada, N.; Tani, T.; Goshima, G.; Oldenbourg, R.; Kimura, A.\nScore: 3.2, Published: 2024-01-05 DOI: 10.1101/2024.01.03.574024\nOrganelles in cells are appropriately positioned, despite crowding in the cytoplasm. However, our understanding of the force required to move large organelles, such as the nucleus, inside the cytoplasm is limited, in part owing to a lack of accurate methods for measurement. We devised a novel method to apply forces to the nucleus of living, wild-type Caenorhabditis elegans embryos to measure the force generated inside the cell. We utilized a centrifuge polarizing microscope (CPM) to apply centrifugal force and orientation-independent differential interference contrast (OI-DIC) microscopy to characterize the mass density of the nucleus and cytoplasm. The cellular forces moving the nucleus toward the cell center increased linearly at ~14 pN/m depending on the distance from the center. The frictional coefficient was ~1,100 pN s/m. The measured values were smaller than previously reported estimates for sea urchin embryos. The forces were consistent with the centrosome-organelle mutual pulling model for nuclear centration. Frictional coefficient was reduced when microtubules were shorter or detached from nuclei in mutant embryos, demonstrating the contribution of astral microtubules. Finally, the frictional coefficient was higher than a theoretical estimate, indicating the contribution of uncharacterized properties of the cytoplasm.\nUnderstanding the Energy Landscape of Intrinsically Disordered Protein Ensembles\nAuthors: Viegas, R. G.; Martins, I. B. S.; LEITE, V. B. P.\nScore: 3.1, Published: 2024-01-05 DOI: 10.1101/2024.01.04.574178\nA substantial portion of various organisms' proteomes comprises intrinsically disordered proteins (IDPs) that lack a defined three-dimensional structure. These IDPs exhibit a diverse array of conformations, displaying remarkable spatio-temporal heterogeneity and exceptional conformational flexibility. Characterizing the structure or structural ensemble of IDPs presents significant conceptual and methodological challenges owing to the absence of a well-defined native structure. While databases such as the Protein Ensemble Database (PED) provide IDP ensembles obtained through a combination of experimental data and molecular modeling, the absence of reaction coordinates poses challenges in comprehensively understanding pertinent aspects of the system. In this study, we leverage the Energy Landscape Visualization Method (JCTC,6482,2019) to scrutinize four IDP ensembles sourced from PED. ELViM, a methodology that circumvents the need for a priori reaction coordinates, aids in analyzing the ensembles. The specific IDP ensembles investigated are as follows: two fragments of Nucleoporin (NUL: 884-993 and NUS: 1313-1390), Yeast Sic 1 N-terminal (1-90), and the N-terminal SH3 domain of Drk (1-59). Utilizing ELViM enables comprehensive validation of ensembles, facilitating the detection of potential inconsistencies in the sampling process. Additionally, it allows for identifying and characterizing the most prevalent conformations within an ensemble. Moreover, ELViM facilitates the comparative analysis of ensembles obtained under diverse conditions, thereby providing a powerful tool for investigating the functional mechanisms of IDPs.\nMultiparameter-based photosynthetic state transitions of single phytoplankton cells\nAuthors: Harris, P. D.; Ben Eliezer, N.; Keren, N.; Lerner, E.\nScore: 2.6, Published: 2024-01-01 DOI: 10.1101/2023.12.31.573751\nFluorescence emitted by light-harvesting pigments responds to the physiological state of phytoplankton. We developed a time resolved system to monitor fluorescence from single cells. It captures multiple spectral channels and fluorescence lifetimes, eliminating ensemble averaging of bulk experiments. Tracking the diurnal cycles of three phytoplankton species, we uncover each species segregation into multiple distinct cell states, a feature previously hidden in bulk measurements. We also tested their response to light-intensity perturbations, and determined each species unique acclimation strategy involving transition between distinct cell subpopulations. This approach will be useful for characterizing natural phytoplankton population responses to environmental conditions, aiming for a better understanding of their acclimation strategies and the effects of global climate changes. One sentence summaryPhotophysiological phytoplankton cell states are uncovered at the multiparameter single cell level.\nNode Features of Chromosome Structure Network and Their Connections to Genome Annotation\nAuthors: Xu, Y.; Das, P.; McCord, R. P.; Shen, T.\nScore: 2.5, Published: 2023-12-30 DOI: 10.1101/2023.12.29.573476\nThe 3D conformations of chromosomes can encode biological significance, and its implication is being increasingly appreciated recently. Certain chromosome structural features, such as A/B compartmentalization, are frequently extracted from pairwise contact information (physical association between different regions of the genome) and compared with linear annotations of the genome, such as histone modifications and lamina association. Here, we investigate how additional properties of chromosome structure can be deduced using the abstract graph representation of the contact heatmap, and how network properties can have a better connection with some of these biological annotations. We constructed chromosome structure networks (CSNs) from bulk Hi-C data and calculated a set of site-resolved (node-based) network properties of these CSNs. We found these network properties are useful for characterizing chromosome structure features. We examined the ability of network properties in differentiating several scenarios, such as haploid vs diploid cells, partially inverted nuclei vs conventional architecture, and structural changes during cell development. We also examined the connection between network properties and a series of other linear annotations, such as histone modifications and chromatin states including poised promoter and enhancer labels. We found that semi-local network properties are more capable of characterizing genome annotations than diffusive or ultra-local node features. For example, local square clustering coefficient can be a strong classifier of lamina-associated domains (LADs), whereas a path-based network property, closeness centrality, does not vary concordantly with LAD status. We demonstrated that network properties can be useful for discerning large-scale chromosome structures that emerge in different biological situations. TOC Figure O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=135 SRC=\"FIGDIR/small/573476v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (51K): org.highwire.dtl.DTLVardef@1be608forg.highwire.dtl.DTLVardef@1b56f08org.highwire.dtl.DTLVardef@8c7a2corg.highwire.dtl.DTLVardef@a8eb4b_HPS_FORMAT_FIGEXP M_FIG C_FIG\nSmart Parallel Automated Cryo Electron tomography\nAuthors: Eisenstein, F.; Fukuda, Y.; Danev, R.\nScore: 40.3, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571776\nIn situ cryo electron tomography enables investigation of macromolecules in their native cellular environment. Data collection, however, requires an experienced operator and valuable microscope time to carefully select targets for tilt series acquisition. Here, we developed a workflow using machine learning approaches to fully automate the entire process, including lamella detection, biological feature segmentation, target selection, and tilt series acquisition, all without the need for human intervention.\n",
  "wordCount" : "2284",
  "inLanguage": "en",
  "datePublished": "2024-01-07T10:38:59Z",
  "dateModified": "2024-01-07T10:38:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573500" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573500">
        <p class="paperTitle">Quantification of cell phenotype transition manifolds with information geometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.</p>
        <p class="info">Score: 18.5, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573500' target='https://doi.org/10.1101/2023.12.28.573500'> 10.1101/2023.12.28.573500</a></p>
        <p class="abstract">Cell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds. However, no available method currently exists for quantifying the manifolds from experimental data. Current methods, such as manifold learning, can only preserve topology information and can only be used for visualization.

In this work, we present a novel method to quantify the cell phenotype transition manifolds using information geometry. Specifically, we transform single cells high-dimensional expression vectors into probability distributions with Gaussian embedding. The Fisher-Rao metric is then naturally defined in this embedding space. With Gaussian embedding, we calculate the Ollivier-Ricci curvature of each single cell. Our analyses revealed that cells with low curvature are related to critical transitions.To further analyze the invariant characteristics of the manifold in CPT, we calculated the information velocity of each single cell based on RNA velocity. The high information velocity regions correspond to the low curvature regions, indicating that the geometry can guide the dynamics of single cells on the manifolds. This method focuses on analyzing the invariant characteristics of the manifold in CPT, and the results illustrate that it can be a general approach for quantifying the CPT manifold.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573596">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573596" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573596">
        <p class="paperTitle">Single-molecule detection and super-resolution imaging with a portable and adaptable 3D-printed microscopy platform (Brick-MIC)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573596" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573596" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moya Munoz, G. G.; Brix, O.; Klocke, P.; Wendler, N. D.; Lerner, E.; Zijlstra, N.; Cordes, T.</p>
        <p class="info">Score: 15.4, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573596' target='https://doi.org/10.1101/2023.12.29.573596'> 10.1101/2023.12.29.573596</a></p>
        <p class="abstract">Over the past decades, single-molecule spectroscopy and super-resolution microscopy have advanced significantly and by now represent important tools for life science research. Despite rapid progress and ongoing development, there is a growing gap between the state-of-the-art and what is accessible to non-optics specialists, e.g., biologists, biochemists, medical researchers, and labs with financial constraints. To bridge this gap, we introduce Brick-MIC, a versatile and affordable open-source 3D-printed micro-spectroscopy and imaging platform. Brick-MIC enables the integration of various fluorescence imaging techniques with single-molecule resolution within a single platform and enables exchange between different modalities within minutes. In this work, we present three variants of Brick-MIC that facilitate single-molecule fluorescence detection, fluorescence correlation spectroscopy and super-resolution imaging. With the three variants, we were able to observe conformational changes and absolute inter-dye distances in single macromolecules and perform single-molecule localization microscopy (STORM and PAINT) of DNA origami nanostructures. Detailed descriptions of the hardware and software components, as well as data analysis routines are provided, to allow non-optics specialist to operate their own Brick-MIC with minimal effort and investments. We foresee that our affordable, flexible, and open-source Brick-MIC platform will be a valuable tool for many laboratories worldwide.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573034">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573034" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573034">
        <p class="paperTitle">The everting Drosophila wing disc is a shape-programmed material</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573034" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573034" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fuhrmann, J. F.; Krishna, A.; Paijmans, J.; Duclut, C. A.; Eaton, S.; Popovic, M.; Julicher, F.; Modes, C. D.; Dye, N. A.</p>
        <p class="info">Score: 9.9, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573034' target='https://doi.org/10.1101/2023.12.23.573034'> 10.1101/2023.12.23.573034</a></p>
        <p class="abstract">How complex 3D tissue shape emerges during animal development remains an important open question in biology and biophysics. Although it is well-established that tissue-scale patterns of in-plane cell behaviors can reshape a tissue in 2D, whether and how such patterns could also deform a tissue in 3D remains unclear. Meanwhile, recent advances in materials science, physics, and engineering have enhanced our ability to craft and understand inanimate &#34;shape-programmable&#34; materials capable of undergoing blueprinted 3D shape transformations arising from in-plane gradients of spontaneous strains. Here, we use this conceptual framework to understand eversion of the Drosophila wing disc, a complex 3D tissue reshaping event in wing development. Using quantification of spatial patterns of cell dynamics on an evolving 3D tissue surface, combined with physical modeling and genetic perturbation, we demonstrate that the wing can deform itself in 3D through large-scale patterning of in-plane cellular behaviors, primarily rearrangements. This work shows that concepts from shape-programming can be employed to understand animal tissue morphogenesis and suggests that there exist intricate patterns in nature that could present novel designs for shape-programmable materials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.571765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.571765" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.571765">
        <p class="paperTitle">Oblique Line Scan Illumination Enables Expansive, Accurate and Sensitive Single Protein Measurements in Solution and in Living Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.571765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.571765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Driouchi, A.; Bretan, M.; Davis, B. J.; Heckert, A. B.; Seeger, M.; Bradley Silva, M.; Forrest, W. S.; Hsiung, J.; Tan, J.; Yang, H.; McSwiggen, D. T.; Song, L.; Sule, A.; Abaie, B.; Conroy, B.; Elliott, L. A.; Gonzalez, E.; Ilkov, F.; Isaacs, J.; Labaria, G.; Lagana, M.; Larsen, D. D.; Margolin, B.; Nguyen, M. K.; Rine, J.; Tang, Y.; Vana, M.; Wilkey, A.; Zhang, Z.; Basham, S.; Ho, J. J.; Johnson, S.; Klammer, A. A.; Lin, K.; Darzacq, X.; Betzig, E.; Berman, R. T.; Anderson, D. J.</p>
        <p class="info">Score: 8.1, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.571765' target='https://doi.org/10.1101/2023.12.21.571765'> 10.1101/2023.12.21.571765</a></p>
        <p class="abstract">Single-molecule localization microscopy (SMLM) techniques, such as single-molecule tracking (SMT), enable in situ measurements in cells from which data-rich metrics can be extracted. SMT has been successfully applied to a variety of biological questions and model systems, aiming to unravel the spatiotemporal regulation of molecular mechanisms that govern protein function, downstream pathway effects, and cellular function. While powerful, SMLM often suffers from low throughput and illumination inhomogeneity, along with microscope and user-induced technical biases. Due to technical limitations in scaling SMLM techniques, a tradeoff between spatiotemporal resolution and throughput has been made historically, restricting broad application of these technologies. Here we address these limitations using Oblique Line Scan (OLS), a robust single-objective light-sheet based illumination and detection modality that achieves nanoscale spatial resolution and sub-millisecond temporal resolution across a 250 x 190 m field of view. We demonstrate OLS-enabled SMT on Halo-Tagged proteins in living cells capturing protein motion up to 14 m2 /s. By exploiting the adaptability of the acquisition frame rate and the improved rejection of out of focus light, we extend the utility of OLS beyond cellular compartments with in-solution SMT (isSMT) for single-molecule measurement of ligand-protein interactions and disruption of protein-protein interactions (PPI). We illustrate the versatility of OLS by showcasing two-color SMT, STORM, and single molecule fluorescence recovery after photobleaching (FRAP). OLS expands the range of SMLM applications and paves the way for robust, high-throughput single-molecule investigations of protein dynamics required for drug screening and systems biology studies, both in cells and in solution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573178">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573178" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573178">
        <p class="paperTitle">De novo design of drug-binding proteins with predictable binding energy and specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573178" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573178" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, L.; Gou, X.; Tan, S.; Mann, S.; Yang, H.; Zhong, X.; Gazgalis, D.; Valdiviezo, J.; Jo, H.; Wu, Y.; Diolaiti, M.; Ashworth, A.; Polizzi, N.; DeGrado, W.</p>
        <p class="info">Score: 18.4, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573178' target='https://doi.org/10.1101/2023.12.23.573178'> 10.1101/2023.12.23.573178</a></p>
        <p class="abstract">The de novo design of small-molecule-binding proteins has seen exciting recent progress; however, the ability to achieve exquisite affinity for binding small molecules while tuning specificity has not yet been demonstrated directly from computation. Here, we develop a computational procedure that results in the highest affinity binders to date with predetermined relative affinities, targeting a series of PARP1 inhibitors. Two of four designed proteins bound with affinities ranging from &lt; 5 nM to low M, in a predictable manner. X-ray crystal structures confirmed the accuracy of the designed protein-drug interactions. Molecular dynamics simulations informed the role of water in binding. Binding free-energy calculations performed directly on the designed models are in excellent agreement with the experimentally measured affinities, suggesting that the de novo design of small-molecule-binding proteins with tuned interaction energies is now feasible entirely from computation. We expect these methods to open many opportunities in biomedicine, including rapid sensor development, antidote design, and drug delivery vehicles.

One Sentence SummaryWe use informatic sampling to design low nM drug-binding proteins, and physics-based calculations to accurately predict affinities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574024">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574024" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574024">
        <p class="paperTitle">Live-cell imaging under centrifugation characterized the cellular force for nuclear centration in the Caenorhabditis elegans embryo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574024" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574024" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goda, M.; Shribak, M.; Ikeda, Z.; Okada, N.; Tani, T.; Goshima, G.; Oldenbourg, R.; Kimura, A.</p>
        <p class="info">Score: 3.2, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574024' target='https://doi.org/10.1101/2024.01.03.574024'> 10.1101/2024.01.03.574024</a></p>
        <p class="abstract">Organelles in cells are appropriately positioned, despite crowding in the cytoplasm. However, our understanding of the force required to move large organelles, such as the nucleus, inside the cytoplasm is limited, in part owing to a lack of accurate methods for measurement. We devised a novel method to apply forces to the nucleus of living, wild-type Caenorhabditis elegans embryos to measure the force generated inside the cell. We utilized a centrifuge polarizing microscope (CPM) to apply centrifugal force and orientation-independent differential interference contrast (OI-DIC) microscopy to characterize the mass density of the nucleus and cytoplasm. The cellular forces moving the nucleus toward the cell center increased linearly at ~14 pN/m depending on the distance from the center. The frictional coefficient was ~1,100 pN s/m. The measured values were smaller than previously reported estimates for sea urchin embryos. The forces were consistent with the centrosome-organelle mutual pulling model for nuclear centration. Frictional coefficient was reduced when microtubules were shorter or detached from nuclei in mutant embryos, demonstrating the contribution of astral microtubules. Finally, the frictional coefficient was higher than a theoretical estimate, indicating the contribution of uncharacterized properties of the cytoplasm.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.574178">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.574178" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.574178">
        <p class="paperTitle">Understanding the Energy Landscape of Intrinsically Disordered Protein Ensembles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.574178" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.574178" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Viegas, R. G.; Martins, I. B. S.; LEITE, V. B. P.</p>
        <p class="info">Score: 3.1, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.574178' target='https://doi.org/10.1101/2024.01.04.574178'> 10.1101/2024.01.04.574178</a></p>
        <p class="abstract">A substantial portion of various organisms&#39; proteomes comprises intrinsically disordered proteins (IDPs) that lack a defined three-dimensional structure. These IDPs exhibit a diverse array of conformations, displaying remarkable spatio-temporal heterogeneity and exceptional conformational flexibility. Characterizing the structure or structural ensemble of IDPs presents significant conceptual and methodological challenges owing to the absence of a well-defined native structure. While databases such as the Protein Ensemble Database (PED) provide IDP ensembles obtained through a combination of experimental data and molecular modeling, the absence of reaction coordinates poses challenges in comprehensively understanding pertinent aspects of the system. In this study, we leverage the Energy Landscape Visualization Method (JCTC,6482,2019) to scrutinize four IDP ensembles sourced from PED. ELViM, a methodology that circumvents the need for a priori reaction coordinates, aids in analyzing the ensembles. The specific IDP ensembles investigated are as follows: two fragments of Nucleoporin (NUL: 884-993 and NUS: 1313-1390), Yeast Sic 1 N-terminal (1-90), and the N-terminal SH3 domain of Drk (1-59). Utilizing ELViM enables comprehensive validation of ensembles, facilitating the detection of potential inconsistencies in the sampling process. Additionally, it allows for identifying and characterizing the most prevalent conformations within an ensemble. Moreover, ELViM facilitates the comparative analysis of ensembles obtained under diverse conditions, thereby providing a powerful tool for investigating the functional mechanisms of IDPs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.31.573751">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.31.573751" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.31.573751">
        <p class="paperTitle">Multiparameter-based photosynthetic state transitions of single phytoplankton cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.31.573751" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.31.573751" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harris, P. D.; Ben Eliezer, N.; Keren, N.; Lerner, E.</p>
        <p class="info">Score: 2.6, Published: 2024-01-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.31.573751' target='https://doi.org/10.1101/2023.12.31.573751'> 10.1101/2023.12.31.573751</a></p>
        <p class="abstract">Fluorescence emitted by light-harvesting pigments responds to the physiological state of phytoplankton. We developed a time resolved system to monitor fluorescence from single cells. It captures multiple spectral channels and fluorescence lifetimes, eliminating ensemble averaging of bulk experiments. Tracking the diurnal cycles of three phytoplankton species, we uncover each species segregation into multiple distinct cell states, a feature previously hidden in bulk measurements. We also tested their response to light-intensity perturbations, and determined each species unique acclimation strategy involving transition between distinct cell subpopulations. This approach will be useful for characterizing natural phytoplankton population responses to environmental conditions, aiming for a better understanding of their acclimation strategies and the effects of global climate changes.

One sentence summaryPhotophysiological phytoplankton cell states are uncovered at the multiparameter single cell level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573476">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573476" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573476">
        <p class="paperTitle">Node Features of Chromosome Structure Network and Their Connections to Genome Annotation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573476" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573476" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, Y.; Das, P.; McCord, R. P.; Shen, T.</p>
        <p class="info">Score: 2.5, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573476' target='https://doi.org/10.1101/2023.12.29.573476'> 10.1101/2023.12.29.573476</a></p>
        <p class="abstract">The 3D conformations of chromosomes can encode biological significance, and its implication is being increasingly appreciated recently. Certain chromosome structural features, such as A/B compartmentalization, are frequently extracted from pairwise contact information (physical association between different regions of the genome) and compared with linear annotations of the genome, such as histone modifications and lamina association. Here, we investigate how additional properties of chromosome structure can be deduced using the abstract graph representation of the contact heatmap, and how network properties can have a better connection with some of these biological annotations. We constructed chromosome structure networks (CSNs) from bulk Hi-C data and calculated a set of site-resolved (node-based) network properties of these CSNs. We found these network properties are useful for characterizing chromosome structure features. We examined the ability of network properties in differentiating several scenarios, such as haploid vs diploid cells, partially inverted nuclei vs conventional architecture, and structural changes during cell development. We also examined the connection between network properties and a series of other linear annotations, such as histone modifications and chromatin states including poised promoter and enhancer labels. We found that semi-local network properties are more capable of characterizing genome annotations than diffusive or ultra-local node features. For example, local square clustering coefficient can be a strong classifier of lamina-associated domains (LADs), whereas a path-based network property, closeness centrality, does not vary concordantly with LAD status. We demonstrated that network properties can be useful for discerning large-scale chromosome structures that emerge in different biological situations.

TOC Figure

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=135 SRC=&#34;FIGDIR/small/573476v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (51K):
org.highwire.dtl.DTLVardef@1be608forg.highwire.dtl.DTLVardef@1b56f08org.highwire.dtl.DTLVardef@8c7a2corg.highwire.dtl.DTLVardef@a8eb4b_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571776">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571776" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571776">
        <p class="paperTitle">Smart Parallel Automated Cryo Electron tomography</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571776" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571776" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eisenstein, F.; Fukuda, Y.; Danev, R.</p>
        <p class="info">Score: 40.3, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571776' target='https://doi.org/10.1101/2023.12.14.571776'> 10.1101/2023.12.14.571776</a></p>
        <p class="abstract">In situ cryo electron tomography enables investigation of macromolecules in their native cellular environment. Data collection, however, requires an experienced operator and valuable microscope time to carefully select targets for tilt series acquisition. Here, we developed a workflow using machine learning approaches to fully automate the entire process, including lamella detection, biological feature segmentation, target selection, and tilt series acquisition, all without the need for human intervention.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
