<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="High-throughput computational discovery of inhibitory protein fragments with AlphaFold
Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.
Score: 88.1, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389
Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="High-throughput computational discovery of inhibitory protein fragments with AlphaFold
Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.
Score: 88.1, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389
Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-31T10:38:19+00:00" />
<meta property="article:modified_time" content="2023-12-31T10:38:19+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="High-throughput computational discovery of inhibitory protein fragments with AlphaFold
Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.
Score: 88.1, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389
Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "High-throughput computational discovery of inhibitory protein fragments with AlphaFold\nAuthors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.\nScore: 88.1, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389\nPeptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells.",
  "keywords": [
    
  ],
  "articleBody": " High-throughput computational discovery of inhibitory protein fragments with AlphaFold\nAuthors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.\nScore: 88.1, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389\nPeptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells. However, it has thus far not been possible to predict de novo which of the many possible protein fragments bind their protein targets, let alone act as inhibitors. We have developed a computational method, FragFold, that employs AlphaFold to predict protein fragment binding to full-length protein targets in a high-throughput manner. Applying FragFold to thousands of fragments tiling across diverse proteins revealed peaks of predicted binding along each protein sequence. These predictions were compared with experimentally measured peaks of inhibitory activity in E. coli. We establish that our approach is a sensitive predictor of protein fragment function: Evaluating inhibitory fragments derived from known protein-protein interaction interfaces, we found 87% were predicted by FragFold to bind in a native-like mode. Across full protein sequences, 68% of FragFold-predicted binding peaks match experimentally measured inhibitory peaks. This is true even when the underlying inhibitory mechanism is unclear from existing structural data, and we find FragFold is able to predict novel binding modes for inhibitory fragments of unknown structure, explaining previous genetic and biochemical data for these fragments. The success rate of FragFold demonstrates that this computational approach should be broadly applicable for discovering inhibitory protein fragments across proteomes.\nAccelerated simulations of RNA phase separation:a systematic study of non-redundant tandem repeats\nAuthors: Aierken, D.; Joseph, J. A.\nScore: 16.0, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573204\nRNA promotes phase separation of proteins to form biomolecular condensates. Notably, pure RNA systems composed of RNA repeat sequences can phase separate and form liquid-like droplets. Significantly, such RNA tandem repeats have been implicated in several neurological and neuromuscular disorders. Thus, investigating the principles governing RNA repeat condensation can provide insights into RNA repeat expansion disorders. However, it is still challenging to characterize the microscopic structures and dynamics of RNA condensates. To address this, we have implemented and optimized a coarse-grained model to study the phase behavior of RNA. Our simulations reproduce key experimental findings on RNA-RNA phase separation. Importantly, we achieve multifold speed up in the simulation time compared to previous work. Leveraging this efficiency, we study the phase separation propensity of all 20 non-redundant trinucleotide repeat sequences. Our study aligns with findings from in vivo experiments, emphasizing that canonical base pairing and G-U wobble pairs play a dominant role in influencing the condensation of RNA repeat sequences and regulating the composition of ensuing condensates. Our implementation of this RNA model should make it feasible to efficiently probe sequence-phase behavior relationships for RNAs, and, when combined with coarse-grained protein models, elucidate the rules governing composition and dynamics of normal and aberrant biomolecular condensates.\nDe novo design of drug-binding proteins with predictable binding energy and specificity\nAuthors: Lu, L.; Gou, X.; Tan, S.; Mann, S.; Yang, H.; Zhong, X.; Gazgalis, D.; Valdiviezo, J.; Jo, H.; Wu, Y.; Diolaiti, M.; Ashworth, A.; Polizzi, N.; DeGrado, W.\nScore: 14.2, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573178\nThe de novo design of small-molecule-binding proteins has seen exciting recent progress; however, the ability to achieve exquisite affinity for binding small molecules while tuning specificity has not yet been demonstrated directly from computation. Here, we develop a computational procedure that results in the highest affinity binders to date with predetermined relative affinities, targeting a series of PARP1 inhibitors. Two of four designed proteins bound with affinities ranging from \u003c 5 nM to low M, in a predictable manner. X-ray crystal structures confirmed the accuracy of the designed protein-drug interactions. Molecular dynamics simulations informed the role of water in binding. Binding free-energy calculations performed directly on the designed models are in excellent agreement with the experimentally measured affinities, suggesting that the de novo design of small-molecule-binding proteins with tuned interaction energies is now feasible entirely from computation. We expect these methods to open many opportunities in biomedicine, including rapid sensor development, antidote design, and drug delivery vehicles. One Sentence SummaryWe use informatic sampling to design low nM drug-binding proteins, and physics-based calculations to accurately predict affinities.\nProtein search processes mediated by chromatin topology\nAuthors: Dutta, S.; R., A.; Padinhateeri, R.; Mitra, M. K.\nScore: 12.7, Published: 2023-12-22 DOI: 10.1101/2023.12.12.571394\nWe investigate the role of compaction of chromatin domains in modulating search kinetics of proteins. Collapsed conformations of chromatin, characterised by long loops which bring distant regions of the genome into contact, and manifested structurally as Topologically Associated Domains (TADs) affect search kinetics of DNA associated transcription factors and other proteins. In this study, we investigate the role of the compactness of chromatin on the dynamics of proteins using a minimal model. Using theory and simulations, we show that an optimal compaction exists for which the residence time of proteins on a chromatin-like polymer backbone is minimum. We then show that the fraction of unique sites visited also shows a similar non-monotonic behavior as a function of the polymer compaction. We extend these results to more detailed polymer models - using the Freely Rotating Chain model, a Lennard-Jones bead-spring polymer, and a soft polymer chain model, which approximates chromatin behavior. We show that our results continue to hold for these polymer models, with a minimum residence time at an optimum polymer compaction. Finally, we also analyse the dynamics of proteins on networks generated using experimental chromatin conformation data for several TADs extracted from human chromosomes. Our analysis suggests that TADs exist near this zone of optimality, indicating that chromatin conformations can play a crucial role in modulating protein search strategies.\nRole of diffusion and reaction of the constituents in spreading of histone modification marks\nAuthors: Manivannan, V.; Inamdar, M. M.; Padinhateeri, R.\nScore: 11.7, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571420\nCells switch genes ON or OFF by altering the state of chromatin via histone modifications at specific regulatory locations along the chromatin polymer. These gene regulation processes are carried out by a network of reactions in which the histone marks spread to neighboring regions with the help of enzymes. In the literature, this spreading has been studied as a purely kinetic, non-diffusive process considering the interactions between neighboring nucleosomes. In this work, we go beyond this framework and study the spreading of modifications using a reaction-diffusion (RD) model accounting for the diffusion of the constituents. We quantitatively segregate the modification profiles generated from kinetic and RD models. The diffusion and degradation of enzymes set a natural length scale for limiting the domain size of modification spreading, and the resulting enzyme limitation is inherent in our model. We also demonstrate the emergence of confined modification domains without the explicit requirement of a nucleation site. We explore polymer compaction effects on spreading and show that single-cell domains may differ from averaged profiles. We find that the modification profiles from our model are comparable with existing H3K9me3 data of S. pombe.\nSmart Parallel Automated Cryo Electron tomography\nAuthors: Eisenstein, F.; Fukuda, Y.; Danev, R.\nScore: 38.0, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571776\nIn situ cryo electron tomography enables investigation of macromolecules in their native cellular environment. Data collection, however, requires an experienced operator and valuable microscope time to carefully select targets for tilt series acquisition. Here, we developed a workflow using machine learning approaches to fully automate the entire process, including lamella detection, biological feature segmentation, target selection, and tilt series acquisition, all without the need for human intervention.\nQuantification of cell phenotype transition manifolds with information geometry\nAuthors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.\nScore: 7.8, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573500\nCell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds. However, no available method currently exists for quantifying the manifolds from experimental data. Current methods, such as manifold learning, can only preserve topology information and can only be used for visualization. In this work, we present a novel method to quantify the cell phenotype transition manifolds using information geometry. Specifically, we transform single cells' high-dimensional expression vectors into probability distributions with Gaussian embedding. The Fisher-Rao metric is then naturally defined in this embedding space. With Gaussian embedding, we calculate the Ollivier-Ricci curvature of each single cell. Our analyses revealed that cells with low curvature are related to critical transitions.To further analyze the invariant characteristics of the manifold in CPT, we calculated the information velocity of each single cell based on RNA velocity. The high information velocity regions correspond to the low curvature regions, indicating that the geometry can guide the dynamics of single cells on the manifolds. This method focuses on analyzing the invariant characteristics of the manifold in CPT, and the results illustrate that it can be a general approach for quantifying the CPT manifold.\nStructure of the human systemic RNAi defective transmembrane protein 1 (hSIDT1) reveals the conformational flexibility of its lipid binding domain\nAuthors: Navratna, V.; Kumar, A.; Rana, J. K.; Mosalaganti, S.\nScore: 13.6, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572875\nIn C. elegans, the inter-cellular transport of the small non-coding RNA that causes systemic RNA interference (RNAi) is mediated by the transmembrane protein SID1, encoded by the sid1 gene in the systemic RNA interference-defective (sid) loci. SID1 shares remarkable structural and sequence similarity with cholesterol uptake protein 1 (CHUP1), and they are classified together as members of the cholesterol uptake family (ChUP). Although systemic RNAi is not an evolutionarily conserved process, the sid gene products are found across the animal kingdom, suggesting a potential for the existence of other novel gene regulatory mechanisms mediated by small non-coding RNAs. Human homologs of sid gene products - hSIDT1 and hSIDT2 - are believed to mediate contact-dependent lipophilic small non-coding dsRNA transport. Here, we report the structure of recombinant full-length human SIDT1. We find that the extra-cytosolic domain (ECD) of hSIDT1 adopts a double jelly roll fold, and the transmembrane domain (TMD) exists as two modules - a flexible lipid binding domain (LBD) and a rigid TMD core. Our structural analyses provide insights into the potential role of the lipid binding domain in the function of hSIDT1 and other cholesterol uptake (ChUP) family members.\np23-FKBP51 chaperone complex regulates tau aggregation\nAuthors: Chakraborty, P.; Zweckstetter, M.\nScore: 5.9, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573415\nThe pathological deposition of tau into insoluble amyloid fibrils is a pathological hallmark of tauopathies. Molecular chaperones are important cellular factors contributing to the regulation of tau misfolding and aggregation. Here we reveal an Hsp90-independent modulation of tau aggregation by a molecular complex of the two Hsp90 co-chaperones p23 and FKBP51. Integrating NMR spectroscopy, SAXS, molecular docking, and site-directed mutagenesis we reveal the structural basis of the p23-FKBP51 complex. We show that p23 specifically recognizes the TPR domain of FKBP51 and interacts with tau through its C-terminal disordered tail. We further show that the p23-FKBP51 complex binds tau to form a dynamic p23-FKBP51-tau trimeric complex that inhibits tau aggregation and thus may counteract Hsp90-FKBP51 mediated toxicity. Taken together, our findings reveal a co-chaperone mediated Hsp90-independent chaperoning of tau protein.\nStructure-based discovery of positive allosteric modulators for the calcium sensing receptor\nAuthors: Liu, F.; Wu, C.-G.; Tu, C.-L.; Glenn, I.; Meyerowitz, J.; Kaplan, A. L.; Lyu, J.; Cheng, Z.; Tarkhanova, O. O.; Moroz, Y. S.; Irwin, J. J.; Chang, W.; Shoichet, B. K.; Skiniotis, G.\nScore: 5.5, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573448\nDrugs acting as positive allosteric modulators (PAMs) to enhance the activation of the calcium sensing receptor (CaSR) and to suppress parathyroid hormone (PTH) secretion can treat hyperparathyroidism but suffer from side effects including hypocalcemia and arrhythmias. Seeking new CaSR modulators, we docked libraries of 2.7 million and 1.2 billion molecules against transforming pockets in the active-state receptor dimer structure. Consistent with the idea that docking improves with library size, billion-molecule docking found new PAMs with a hit rate that was 2.7-fold higher than the million-molecule library and with hits up to 37-fold more potent. Structure-based optimization of ligands from both campaigns led to nanomolar leads, one of which was advanced to animal testing. This PAM displays 100-fold the potency of the standard of care, cinacalcet, in ex vivo organ assays, and reduces serum PTH levels in mice by up to 80% without the hypocalcemia typical of CaSR drugs. Cryo-EM structures with the new PAMs show that they promote CaSR dimer conformations that are closer to the G-protein coupled state compared to established drugs. These findings highlight the promise of large library docking for therapeutic leads, especially when combined with experimental structure determination and mechanism.\n",
  "wordCount" : "2138",
  "inLanguage": "en",
  "datePublished": "2023-12-31T10:38:19Z",
  "dateModified": "2023-12-31T10:38:19Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 31, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572389" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572389">
        <p class="paperTitle">High-throughput computational discovery of inhibitory protein fragments with AlphaFold</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.</p>
        <p class="info">Score: 88.1, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572389' target='https://doi.org/10.1101/2023.12.19.572389'> 10.1101/2023.12.19.572389</a></p>
        <p class="abstract">Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells. However, it has thus far not been possible to predict de novo which of the many possible protein fragments bind their protein targets, let alone act as inhibitors. We have developed a computational method, FragFold, that employs AlphaFold to predict protein fragment binding to full-length protein targets in a high-throughput manner. Applying FragFold to thousands of fragments tiling across diverse proteins revealed peaks of predicted binding along each protein sequence. These predictions were compared with experimentally measured peaks of inhibitory activity in E. coli. We establish that our approach is a sensitive predictor of protein fragment function: Evaluating inhibitory fragments derived from known protein-protein interaction interfaces, we found 87% were predicted by FragFold to bind in a native-like mode. Across full protein sequences, 68% of FragFold-predicted binding peaks match experimentally measured inhibitory peaks. This is true even when the underlying inhibitory mechanism is unclear from existing structural data, and we find FragFold is able to predict novel binding modes for inhibitory fragments of unknown structure, explaining previous genetic and biochemical data for these fragments. The success rate of FragFold demonstrates that this computational approach should be broadly applicable for discovering inhibitory protein fragments across proteomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573204">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573204" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573204">
        <p class="paperTitle">Accelerated simulations of RNA phase separation:a systematic study of non-redundant tandem repeats</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573204" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573204" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aierken, D.; Joseph, J. A.</p>
        <p class="info">Score: 16.0, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573204' target='https://doi.org/10.1101/2023.12.23.573204'> 10.1101/2023.12.23.573204</a></p>
        <p class="abstract">RNA promotes phase separation of proteins to form biomolecular condensates. Notably, pure RNA systems composed of RNA repeat sequences can phase separate and form liquid-like droplets. Significantly, such RNA tandem repeats have been implicated in several neurological and neuromuscular disorders. Thus, investigating the principles governing RNA repeat condensation can provide insights into RNA repeat expansion disorders. However, it is still challenging to characterize the microscopic structures and dynamics of RNA condensates. To address this, we have implemented and optimized a coarse-grained model to study the phase behavior of RNA. Our simulations reproduce key experimental findings on RNA-RNA phase separation. Importantly, we achieve multifold speed up in the simulation time compared to previous work. Leveraging this efficiency, we study the phase separation propensity of all 20 non-redundant trinucleotide repeat sequences. Our study aligns with findings from in vivo experiments, emphasizing that canonical base pairing and G-U wobble pairs play a dominant role in influencing the condensation of RNA repeat sequences and regulating the composition of ensuing condensates. Our implementation of this RNA model should make it feasible to efficiently probe sequence-phase behavior relationships for RNAs, and, when combined with coarse-grained protein models, elucidate the rules governing composition and dynamics of normal and aberrant biomolecular condensates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573178">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573178" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573178">
        <p class="paperTitle">De novo design of drug-binding proteins with predictable binding energy and specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573178" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573178" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, L.; Gou, X.; Tan, S.; Mann, S.; Yang, H.; Zhong, X.; Gazgalis, D.; Valdiviezo, J.; Jo, H.; Wu, Y.; Diolaiti, M.; Ashworth, A.; Polizzi, N.; DeGrado, W.</p>
        <p class="info">Score: 14.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573178' target='https://doi.org/10.1101/2023.12.23.573178'> 10.1101/2023.12.23.573178</a></p>
        <p class="abstract">The de novo design of small-molecule-binding proteins has seen exciting recent progress; however, the ability to achieve exquisite affinity for binding small molecules while tuning specificity has not yet been demonstrated directly from computation. Here, we develop a computational procedure that results in the highest affinity binders to date with predetermined relative affinities, targeting a series of PARP1 inhibitors. Two of four designed proteins bound with affinities ranging from &lt; 5 nM to low M, in a predictable manner. X-ray crystal structures confirmed the accuracy of the designed protein-drug interactions. Molecular dynamics simulations informed the role of water in binding. Binding free-energy calculations performed directly on the designed models are in excellent agreement with the experimentally measured affinities, suggesting that the de novo design of small-molecule-binding proteins with tuned interaction energies is now feasible entirely from computation. We expect these methods to open many opportunities in biomedicine, including rapid sensor development, antidote design, and drug delivery vehicles.

One Sentence SummaryWe use informatic sampling to design low nM drug-binding proteins, and physics-based calculations to accurately predict affinities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571394">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571394" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571394">
        <p class="paperTitle">Protein search processes mediated by chromatin topology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571394" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571394" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dutta, S.; R., A.; Padinhateeri, R.; Mitra, M. K.</p>
        <p class="info">Score: 12.7, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571394' target='https://doi.org/10.1101/2023.12.12.571394'> 10.1101/2023.12.12.571394</a></p>
        <p class="abstract">We investigate the role of compaction of chromatin domains in modulating search kinetics of proteins. Collapsed conformations of chromatin, characterised by long loops which bring distant regions of the genome into contact, and manifested structurally as Topologically Associated Domains (TADs) affect search kinetics of DNA associated transcription factors and other proteins. In this study, we investigate the role of the compactness of chromatin on the dynamics of proteins using a minimal model. Using theory and simulations, we show that an optimal compaction exists for which the residence time of proteins on a chromatin-like polymer backbone is minimum. We then show that the fraction of unique sites visited also shows a similar non-monotonic behavior as a function of the polymer compaction. We extend these results to more detailed polymer models - using the Freely Rotating Chain model, a Lennard-Jones bead-spring polymer, and a soft polymer chain model, which approximates chromatin behavior. We show that our results continue to hold for these polymer models, with a minimum residence time at an optimum polymer compaction. Finally, we also analyse the dynamics of proteins on networks generated using experimental chromatin conformation data for several TADs extracted from human chromosomes. Our analysis suggests that TADs exist near this zone of optimality, indicating that chromatin conformations can play a crucial role in modulating protein search strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571420">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571420" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571420">
        <p class="paperTitle">Role of diffusion and reaction of the constituents in spreading of histone modification marks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571420" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571420" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manivannan, V.; Inamdar, M. M.; Padinhateeri, R.</p>
        <p class="info">Score: 11.7, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571420' target='https://doi.org/10.1101/2023.12.13.571420'> 10.1101/2023.12.13.571420</a></p>
        <p class="abstract">Cells switch genes ON or OFF by altering the state of chromatin via histone modifications at specific regulatory locations along the chromatin polymer. These gene regulation processes are carried out by a network of reactions in which the histone marks spread to neighboring regions with the help of enzymes. In the literature, this spreading has been studied as a purely kinetic, non-diffusive process considering the interactions between neighboring nucleosomes. In this work, we go beyond this framework and study the spreading of modifications using a reaction-diffusion (RD) model accounting for the diffusion of the constituents. We quantitatively segregate the modification profiles generated from kinetic and RD models. The diffusion and degradation of enzymes set a natural length scale for limiting the domain size of modification spreading, and the resulting enzyme limitation is inherent in our model. We also demonstrate the emergence of confined modification domains without the explicit requirement of a nucleation site. We explore polymer compaction effects on spreading and show that single-cell domains may differ from averaged profiles. We find that the modification profiles from our model are comparable with existing H3K9me3 data of S. pombe.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571776">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571776" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571776">
        <p class="paperTitle">Smart Parallel Automated Cryo Electron tomography</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571776" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571776" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eisenstein, F.; Fukuda, Y.; Danev, R.</p>
        <p class="info">Score: 38.0, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571776' target='https://doi.org/10.1101/2023.12.14.571776'> 10.1101/2023.12.14.571776</a></p>
        <p class="abstract">In situ cryo electron tomography enables investigation of macromolecules in their native cellular environment. Data collection, however, requires an experienced operator and valuable microscope time to carefully select targets for tilt series acquisition. Here, we developed a workflow using machine learning approaches to fully automate the entire process, including lamella detection, biological feature segmentation, target selection, and tilt series acquisition, all without the need for human intervention.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573500" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573500">
        <p class="paperTitle">Quantification of cell phenotype transition manifolds with information geometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.</p>
        <p class="info">Score: 7.8, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573500' target='https://doi.org/10.1101/2023.12.28.573500'> 10.1101/2023.12.28.573500</a></p>
        <p class="abstract">Cell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds. However, no available method currently exists for quantifying the manifolds from experimental data. Current methods, such as manifold learning, can only preserve topology information and can only be used for visualization. In this work, we present a novel method to quantify the cell phenotype transition manifolds using information geometry. Specifically, we transform single cells&#39; high-dimensional expression vectors into probability distributions with Gaussian embedding. The Fisher-Rao metric is then naturally defined in this embedding space. With Gaussian embedding, we calculate the Ollivier-Ricci curvature of each single cell. Our analyses revealed that cells with low curvature are related to critical transitions.To further analyze the invariant characteristics of the manifold in CPT, we calculated the information velocity of each single cell based on RNA velocity. The high information velocity regions correspond to the low curvature regions, indicating that the geometry can guide the dynamics of single cells on the manifolds. This method focuses on analyzing the invariant characteristics of the manifold in CPT, and the results illustrate that it can be a general approach for quantifying the CPT manifold.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572875">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572875" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572875">
        <p class="paperTitle">Structure of the human systemic RNAi defective transmembrane protein 1 (hSIDT1) reveals the conformational flexibility of its lipid binding domain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572875" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572875" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Navratna, V.; Kumar, A.; Rana, J. K.; Mosalaganti, S.</p>
        <p class="info">Score: 13.6, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572875' target='https://doi.org/10.1101/2023.12.21.572875'> 10.1101/2023.12.21.572875</a></p>
        <p class="abstract">In C. elegans, the inter-cellular transport of the small non-coding RNA that causes systemic RNA interference (RNAi) is mediated by the transmembrane protein SID1, encoded by the sid1 gene in the systemic RNA interference-defective (sid) loci. SID1 shares remarkable structural and sequence similarity with cholesterol uptake protein 1 (CHUP1), and they are classified together as members of the cholesterol uptake family (ChUP). Although systemic RNAi is not an evolutionarily conserved process, the sid gene products are found across the animal kingdom, suggesting a potential for the existence of other novel gene regulatory mechanisms mediated by small non-coding RNAs. Human homologs of sid gene products - hSIDT1 and hSIDT2 - are believed to mediate contact-dependent lipophilic small non-coding dsRNA transport. Here, we report the structure of recombinant full-length human SIDT1. We find that the extra-cytosolic domain (ECD) of hSIDT1 adopts a double jelly roll fold, and the transmembrane domain (TMD) exists as two modules - a flexible lipid binding domain (LBD) and a rigid TMD core. Our structural analyses provide insights into the potential role of the lipid binding domain in the function of hSIDT1 and other cholesterol uptake (ChUP) family members.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573415">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573415" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573415">
        <p class="paperTitle">p23-FKBP51 chaperone complex regulates tau aggregation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573415" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573415" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chakraborty, P.; Zweckstetter, M.</p>
        <p class="info">Score: 5.9, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573415' target='https://doi.org/10.1101/2023.12.27.573415'> 10.1101/2023.12.27.573415</a></p>
        <p class="abstract">The pathological deposition of tau into insoluble amyloid fibrils is a pathological hallmark of tauopathies. Molecular chaperones are important cellular factors contributing to the regulation of tau misfolding and aggregation. Here we reveal an Hsp90-independent modulation of tau aggregation by a molecular complex of the two Hsp90 co-chaperones p23 and FKBP51. Integrating NMR spectroscopy, SAXS, molecular docking, and site-directed mutagenesis we reveal the structural basis of the p23-FKBP51 complex. We show that p23 specifically recognizes the TPR domain of FKBP51 and interacts with tau through its C-terminal disordered tail. We further show that the p23-FKBP51 complex binds tau to form a dynamic p23-FKBP51-tau trimeric complex that inhibits tau aggregation and thus may counteract Hsp90-FKBP51 mediated toxicity. Taken together, our findings reveal a co-chaperone mediated Hsp90-independent chaperoning of tau protein.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573448">
        <p class="paperTitle">Structure-based discovery of positive allosteric modulators for the calcium sensing receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, F.; Wu, C.-G.; Tu, C.-L.; Glenn, I.; Meyerowitz, J.; Kaplan, A. L.; Lyu, J.; Cheng, Z.; Tarkhanova, O. O.; Moroz, Y. S.; Irwin, J. J.; Chang, W.; Shoichet, B. K.; Skiniotis, G.</p>
        <p class="info">Score: 5.5, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573448' target='https://doi.org/10.1101/2023.12.27.573448'> 10.1101/2023.12.27.573448</a></p>
        <p class="abstract">Drugs acting as positive allosteric modulators (PAMs) to enhance the activation of the calcium sensing receptor (CaSR) and to suppress parathyroid hormone (PTH) secretion can treat hyperparathyroidism but suffer from side effects including hypocalcemia and arrhythmias. Seeking new CaSR modulators, we docked libraries of 2.7 million and 1.2 billion molecules against transforming pockets in the active-state receptor dimer structure. Consistent with the idea that docking improves with library size, billion-molecule docking found new PAMs with a hit rate that was 2.7-fold higher than the million-molecule library and with hits up to 37-fold more potent. Structure-based optimization of ligands from both campaigns led to nanomolar leads, one of which was advanced to animal testing. This PAM displays 100-fold the potency of the standard of care, cinacalcet, in ex vivo organ assays, and reduces serum PTH levels in mice by up to 80% without the hypocalcemia typical of CaSR drugs. Cryo-EM structures with the new PAMs show that they promote CaSR dimer conformations that are closer to the G-protein coupled state compared to established drugs. These findings highlight the promise of large library docking for therapeutic leads, especially when combined with experimental structure determination and mechanism.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
