<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Sifting Through the Noise: A Computational Pipeline for Accurate Prioritization of Protein- Protein Binding Candidates in High-Throughput Protein Libraries
Authors: Mondal, A.; Singh, B.; Felkner, R. H.; De Falco, A.; Swapna, G.; Montelione, G. T.; Roth, M.; Perez, A.
Score: 13.3, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576374
Identifying the interactome for a protein of interest is challenging due to the large number of possible binders. High-throughput experimental approaches narrow down possible binding partners, but often include false positives.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Sifting Through the Noise: A Computational Pipeline for Accurate Prioritization of Protein- Protein Binding Candidates in High-Throughput Protein Libraries
Authors: Mondal, A.; Singh, B.; Felkner, R. H.; De Falco, A.; Swapna, G.; Montelione, G. T.; Roth, M.; Perez, A.
Score: 13.3, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576374
Identifying the interactome for a protein of interest is challenging due to the large number of possible binders. High-throughput experimental approaches narrow down possible binding partners, but often include false positives." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Sifting Through the Noise: A Computational Pipeline for Accurate Prioritization of Protein- Protein Binding Candidates in High-Throughput Protein Libraries
Authors: Mondal, A.; Singh, B.; Felkner, R. H.; De Falco, A.; Swapna, G.; Montelione, G. T.; Roth, M.; Perez, A.
Score: 13.3, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576374
Identifying the interactome for a protein of interest is challenging due to the large number of possible binders. High-throughput experimental approaches narrow down possible binding partners, but often include false positives."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Sifting Through the Noise: A Computational Pipeline for Accurate Prioritization of Protein- Protein Binding Candidates in High-Throughput Protein Libraries\nAuthors: Mondal, A.; Singh, B.; Felkner, R. H.; De Falco, A.; Swapna, G.; Montelione, G. T.; Roth, M.; Perez, A.\nScore: 13.3, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576374\nIdentifying the interactome for a protein of interest is challenging due to the large number of possible binders. High-throughput experimental approaches narrow down possible binding partners, but often include false positives.",
  "keywords": [
    
  ],
  "articleBody": " Sifting Through the Noise: A Computational Pipeline for Accurate Prioritization of Protein- Protein Binding Candidates in High-Throughput Protein Libraries\nAuthors: Mondal, A.; Singh, B.; Felkner, R. H.; De Falco, A.; Swapna, G.; Montelione, G. T.; Roth, M.; Perez, A.\nScore: 13.3, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576374\nIdentifying the interactome for a protein of interest is challenging due to the large number of possible binders. High-throughput experimental approaches narrow down possible binding partners, but often include false positives. Furthermore, they provide no information about what the binding region is (e.g. the binding epitope). We introduce a novel computational pipeline based on an AlphaFold2 (AF) Competition Assay (AF-CBA) to identify proteins that bind a target of interest from a pull-down experiment, along with the binding epitope. Our focus is on proteins that bind the Extraterminal (ET) domain of Bromo and Extraterminal domain (BET) proteins, but we also introduce nine additional systems to show transferability to other peptide-protein systems. We describe a series of limitations to the methodology based on intrinsic deficiencies to AF and AF-CBA, to help users identify scenarios where the approach will be most useful. Given the speed and accuracy of the methodology, we expect it to be generally applicable to facilitate target selection for experimental verification starting from high-throughput protein libraries. Table of Contents O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=173 SRC=\"FIGDIR/small/576374v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (41K): org.highwire.dtl.DTLVardef@b90071org.highwire.dtl.DTLVardef@1cc2f66org.highwire.dtl.DTLVardef@3c1109org.highwire.dtl.DTLVardef@1827353_HPS_FORMAT_FIGEXP M_FIG C_FIG\nDiffraction minima resolve point scatterers at tiny fractions (1/80) of the wavelength\nAuthors: Hensel, T. A.; Wirth, J. O.; Hell, S. W.\nScore: 11.4, Published: 2024-01-26 DOI: 10.1101/2024.01.24.576982\nDiscerning two or more identical and constantly scattering point sources using freely propagating waves is thought to be limited by diffraction. Here we show both theoretically and experimentally that by employing a diffraction minimum rather than a maximum for resolution, a given number of point scatterers can be discerned at tiny fractions of the employed wavelength. Specifically, we identify an 8 nm distance between two constantly emitting (non-blinking, non-switchable) fluorescent molecules, corresponding to 1/80 of the wavelength. Moreover, we show that contrary to naive expectations, the measurement precision improves with decreasing distance between the scatterers and with increased scatterer density, thus opening up the prospect of resolving clusters of (optical) point scatterers at tiny fractions of the wavelength.\nBurkholderia cenocepacia epigenetic regulator M.BceJIV simultaneously engages two DNA recognition sequences for methylation\nAuthors: Quintana-Feliciano, R.; Kottur, J.; Ni, M.; Ghosh, R.; Salas-Estrada, L.; Rechkoblit, O.; Filizola, M.; Fang, G.; Aggarwal, A.\nScore: 8.0, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576384\nBurkholderia cenocepacia is an opportunistic and infective bacterium containing an orphan DNA methyltransferase (M.BceJIV) with roles in regulating gene expression and motility of the bacterium. M.BceJIV recognizes a GTWWAC motif (where W can be an adenine or a thymine) and methylates the N6 of the adenine at the fifth base position (GTWWAC). Here, we present a high-resolution crystal structure of M.BceJIV/DNA/sinefungin ternary complex and allied biochemical, computational, and thermodynamic analyses. Remarkably, the structure shows not one, but two DNA substrates bound to the M.BceJIV dimer, wherein each monomer contributes to the recognition of two recognition sequences. This unexpected mode of DNA binding and methylation has not been observed previously and sets a new precedent for a DNA methyltransferase. We also show that methylation at two recognition sequences occurs independently, and that GTWWAC motifs are enriched in intergenic regions of a strain of B. cenocepacias genome. We further computationally assess the interactions underlying the affinities of different ligands (SAM, SAH, and sinefungin) for M.BceJIV, as a step towards developing selective inhibitors for limiting B. cenocepacia infection.\nSolid-state nanopore detection of partially denatured dsDNA with single-strand binding protein: a preliminary analysis\nAuthors: Howald, N.; Klotz, A.\nScore: 4.3, Published: 2024-01-28 DOI: 10.1101/2024.01.28.577651\nIn this work we investigate the use of a nanopore sensor to detect single-strand binding protein (SSB) attached to AT-rich denaturation bubbles in genomic double-stranded (ds) DNA. DNA from the {lambda} bacteriophage was heated in the presence of E. coli SSB at temperatures predicted to open denaturation bubbles near the center of the molecule. A solid state nanopore sensor measured the ionic current as the DNA-SSB solution flowed through the pore, detecting blockades due to the translocation of biomolecules. Large current spikes were observed in the translocating DNA molecules, consistent with SSB binding. However, spikes were largely localized at either end of the DNA molecule, rather than at the predicted sites. We discuss the physico-chemical effects behind this disagreement and prospects for the future use of this technique for genomic mapping.\nMeasurement of solubility product in a model condensate reveals the interplay of small oligomerization and self-association\nAuthors: Chattaraj, A.; Baltaci, Z.; Mayer, B. J.; Loew, L. M.; Ditlev, J. A.\nScore: 3.9, Published: 2024-01-26 DOI: 10.1101/2024.01.23.576869\nCellular condensates often consist of 10s to 100s of distinct interacting molecular species. Because of the complexity of these interactions, predicting the point at which they will undergo phase separation into discrete compartments is daunting. Using experiments and computation, we therefore studied a simple model system consisting of 2 proteins, polySH3 and polyPRM, designed for pentavalent heterotypic binding. We tested whether the peak solubility product, the product of dilute phase monomer concentrations, is a predictive parameter for the onset of phase separation. Titrating up equal total concentrations of each component showed that the maximum solubility product does approximately coincide with the threshold for phase separation in both the experiments and models. However, we found that measurements of dilute phase concentration include contributions from small oligomers, not just monomers; therefore, a quantitative comparison of the experiments and models required inclusion of small oligomers in the model analysis. We also examined full phase diagrams where the model results were almost symmetric along the diagonal, but the experimental results were highly asymmetric. This led us to perform dynamic light scattering experiments, where we discovered a weak homotypic interaction for polyPRM; when this was added to the computational model, it was able to recapitulate the experimentally observed asymmetry. Thus, comparing experiments to simulation reveals that the solubility product can be predictive of phase separation, even if small oligomers and low affinity homotypic interactions preclude experimental measurement of monomer concentration.\nScorpion α-toxin LqhαIT specifically interacts with a glycan at the pore domain of voltage-gated sodium channels\nAuthors: Phulera, S.; Dickson, C. J.; Schwalen, C. J.; Khoshouei, M.; Cassell, S. J.; Sun, Y.; Condos, T.; Whicher, J.; Weihofen, W. A.\nScore: 3.8, Published: 2024-01-30 DOI: 10.1101/2024.01.26.577479\nVoltage-gated sodium (Nav) channels sense membrane potential and drive cellular electrical activity. Numerous protein toxins have been identified that modulate Nav gating, and structures of Nav channels in complex with these toxins helped elucidate the molecular mechanisms of voltage-dependent channel gating. The deathstalker scorpion -toxin LqhIT exerts a strong action potential prolonging effect on Nav channels. Biochemical studies show that LqhIT features a functionally essential epitope at its C-terminus that is not shared with related scorpion -toxins. To elucidate the mechanism of action of LqhIT, we determined a 3.9\u0026Aring cryo-electron microscopy (cryo-EM) structure of LqhIT in complex with the Nav channel from Periplaneta americana (NavPas). We found that LqhIT binds to voltage sensor domain 4 and traps it in a S4 down conformation to stabilize the open state. To promote binding, the functionally essential C-terminal epitope of LqhIT forms an extensive interface with the glycan scaffold linked to Asn330 of NavPas that augments a small protein-protein interface between NavPas and LqhIT. A combination of molecular dynamics simulations, structural comparisons, and prior mutagenesis experiments demonstrate the functional importance of this toxin-glycan interaction. These findings help establish a structural basis for the specificity achieved by scorpion -toxins and provide crucial insights for the development and optimization of new Nav channel modulators.\nChaperone quality control in liquid-phase separated organelles\nAuthors: Scheidt, T.; Lemke, E. A.\nScore: 5.8, Published: 2024-01-23 DOI: 10.1101/2024.01.23.576883\nMolecular chaperones, central to the cellular proteostasis network, play an essential role in preventing the formation and proliferation of harmful aggregates associated with neurodegenerative diseases. Notably, for many intrinsically disordered proteins (IDPs), which are prone to form such damaging deposits, the formation of nano-clusters and phase separation into organelles prior to aggregation have been observed. The impact of molecular chaperones on such assemblies, remains unclear. In our study, we concentrated on the family of small heat shock proteins (sHsps), which are typically dynamic and form large oligomeric structures. While sHsps are mainly structured/folded proteins, they can undergo transient multivalent interactions, like many IDPs. Thus, sHsps might be a suitable regulator for vital and ubiquitous formation of membrane-less organelles in eukaryotic cells rich in IDPs and to inhibit aberrant aggregation. Here we show, using microfluidic diffusional sizing, that the formation of nano-clusters of FUS, associated with neurodegenerative diseases can be inhibited by the presence of sHsps. Furthermore, we identify that, depending on their assembly state, sHsps are capable of targeting specifically the interface between the dense droplet phase and the dilute phase not only of FUS but also of TDP-43, likely because the interface is the primary starting point for fibril formation or protein aggregation in general. Our findings emphasise the impact of molecular chaperones on maintaining the homeostasis of IDPs in the dilute and condensed phase. This could help to understand how chaperone dysregulation can influence aberrant protein association.\nSpectrally Resolved Localization Microscopy with Ultrahigh-Throughput\nAuthors: Batey, J. E.; Kim, G. W.; Yang, M.; Heffer, D. C.; Pott, E. D.; Giang, H.; Dong, B.\nScore: 5.2, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576082\nSingle-molecule localization microscopy (SMLM) has become a strong technique in the toolbox of chemists, biologists, physicists, and engineers in recent years for its unique ability to resolve characteristic features quickly and accurately in complex environments at the nanoscopic level. Multicolor super-resolution imaging has seen the greatest advancement among SMLM techniques, drastically improving the differentiation ability of nanostructures beyond the diffraction limit and increasing the resolution with which previously unresolvable structures are studied. However, current multicolor SMLM methodologies present low spatial resolution and throughput and require complex optical systems. Here, we overcome these drawbacks by developing an ultrahigh-throughput SMLM methodology that allows for ultrahigh throughput multicolor imaging at the nanoscopic level using a color glass filter. Our methodology can readily distinguish fluorophores of close spectral emission and achieves sub-10 nm localization and sub-5nm spectral precisions.\nSelf-organization of PIP3 signaling is controlled by the confinement, topology, and curvature of the cell membrane.\nAuthors: Erisis, S.; Hoerning, M.\nScore: 3.1, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576405\nPIP3 is a signaling lipid on the plasma membrane that plays a fundamental role in cell signaling with a strong impact on cell physiology and diseases. It is responsible for the protruding edge formation, cell polarization, macropinocytosis and other membrane remodeling dynamics in cells. It has been shown that the membrane confinement and curvature affects the wave formation of PIP3 and Factin. But even in the absence of F-actin, a complex self-organization of the spatiotemporal PIP3 waves is observed. In recent findings, we have shown that these waves can be guided and pinned on strongly bended Dictyostelium membranes caused by molecular crowding and curvature limited diffusion. Based on these experimental findings, we investigate the spatiotemporal PIP3 wave dynamics on realist 3D cell-like membranes to explore the effect of curvature limited diffusion, as observed experimentally. We use an established stochastic reaction-diffusion model with enzymatic MichaelisMenten type reactions that mimics the dynamics of Dictyostelium cells. As these cells mimic the 3D shape and size observed experimentally, we found that the PIP3 wave directionality can be explained by a Hopf-like and a reverse periodic-doubling bifurcation for uniform diffusion and curvature limited diffusion properties. Finally, we compare the results with recent experimental findings and discuss the discrepancy between the biological and numerical results.\nMulti-Channel smFRET study reveals a Compact conformation of EF-G on the Ribosome\nAuthors: Johnson, J.; Steele, J.; Lin, R.; Stepanov, V.; Gavriliuc, M.; wang, y.\nScore: 2.5, Published: 2024-01-28 DOI: 10.1101/2024.01.27.577133\nWhile elongation factor G (EF-G) is crucial for ribosome translocation, the role of its GTP hydrolysis remains ambiguous. The indispensability of EF-G is further exemplified by the phosphorylation of human eukaryotic elongation factor 2 (eEF2) at Thr56, which inhibits protein synthesis globally, but its exact mechanism is not clear. In this study, we developed a multi-channel single-molecule FRET (smFRET) microscopy methodology to examine the conformational changes of E. coli EF-G induced by mutations that closely aligned with Thr56 residue of eEF2. We utilized Alexa 488/594 double-labeled EF-G to catalyze the translocation of fMet-Phe-tRNAPhe-Cy3 inside Cy5-L27 labeled ribosomes, allowing us to probe both processes within the same complex. Our findings indicate that in the presence of either GTP or GDPCP, wild-type EF-G undergoes a conformational extension upon binding to the ribosome to promote normal translocation. On the other hand, T48E and T48V mutations did not affect GTP/GDP binding or GTP hydrolysis, but impeded Poly(Phe) synthesis and caused EF-G to adopt a unique compact conformation, which was not observed when the mutants interact solely with the sarcin/ricin loop. This study provides new insights into EF-G's adaptability and sheds light on the modification mechanism of human eEF2.\n",
  "wordCount" : "2152",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576374">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576374" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576374">
        <p class="paperTitle">Sifting Through the Noise: A Computational Pipeline for Accurate Prioritization of Protein- Protein Binding Candidates in High-Throughput Protein Libraries</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576374" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576374" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mondal, A.; Singh, B.; Felkner, R. H.; De Falco, A.; Swapna, G.; Montelione, G. T.; Roth, M.; Perez, A.</p>
        <p class="info">Score: 13.3, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576374' target='https://doi.org/10.1101/2024.01.20.576374'> 10.1101/2024.01.20.576374</a></p>
        <p class="abstract">Identifying the interactome for a protein of interest is challenging due to the large number of possible binders. High-throughput experimental approaches narrow down possible binding partners, but often include false positives. Furthermore, they provide no information about what the binding region is (e.g. the binding epitope). We introduce a novel computational pipeline based on an AlphaFold2 (AF) Competition Assay (AF-CBA) to identify proteins that bind a target of interest from a pull-down experiment, along with the binding epitope. Our focus is on proteins that bind the Extraterminal (ET) domain of Bromo and Extraterminal domain (BET) proteins, but we also introduce nine additional systems to show transferability to other peptide-protein systems. We describe a series of limitations to the methodology based on intrinsic deficiencies to AF and AF-CBA, to help users identify scenarios where the approach will be most useful. Given the speed and accuracy of the methodology, we expect it to be generally applicable to facilitate target selection for experimental verification starting from high-throughput protein libraries.

Table of Contents

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=173 SRC=&#34;FIGDIR/small/576374v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (41K):
org.highwire.dtl.DTLVardef@b90071org.highwire.dtl.DTLVardef@1cc2f66org.highwire.dtl.DTLVardef@3c1109org.highwire.dtl.DTLVardef@1827353_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.576982">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.576982" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.576982">
        <p class="paperTitle">Diffraction minima resolve point scatterers at tiny fractions (1/80) of the wavelength</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.576982" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.576982" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hensel, T. A.; Wirth, J. O.; Hell, S. W.</p>
        <p class="info">Score: 11.4, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.576982' target='https://doi.org/10.1101/2024.01.24.576982'> 10.1101/2024.01.24.576982</a></p>
        <p class="abstract">Discerning two or more identical and constantly scattering point sources using freely propagating waves is thought to be limited by diffraction. Here we show both theoretically and experimentally that by employing a diffraction minimum rather than a maximum for resolution, a given number of point scatterers can be discerned at tiny fractions of the employed wavelength. Specifically, we identify an 8 nm distance between two constantly emitting (non-blinking, non-switchable) fluorescent molecules, corresponding to 1/80 of the wavelength. Moreover, we show that contrary to naive expectations, the measurement precision improves with decreasing distance between the scatterers and with increased scatterer density, thus opening up the prospect of resolving clusters of (optical) point scatterers at tiny fractions of the wavelength.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576384">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576384" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576384">
        <p class="paperTitle">Burkholderia cenocepacia epigenetic regulator M.BceJIV simultaneously engages two DNA recognition sequences for methylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576384" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576384" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Quintana-Feliciano, R.; Kottur, J.; Ni, M.; Ghosh, R.; Salas-Estrada, L.; Rechkoblit, O.; Filizola, M.; Fang, G.; Aggarwal, A.</p>
        <p class="info">Score: 8.0, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576384' target='https://doi.org/10.1101/2024.01.20.576384'> 10.1101/2024.01.20.576384</a></p>
        <p class="abstract">Burkholderia cenocepacia is an opportunistic and infective bacterium containing an orphan DNA methyltransferase (M.BceJIV) with roles in regulating gene expression and motility of the bacterium. M.BceJIV recognizes a GTWWAC motif (where W can be an adenine or a thymine) and methylates the N6 of the adenine at the fifth base position (GTWWAC). Here, we present a high-resolution crystal structure of M.BceJIV/DNA/sinefungin ternary complex and allied biochemical, computational, and thermodynamic analyses. Remarkably, the structure shows not one, but two DNA substrates bound to the M.BceJIV dimer, wherein each monomer contributes to the recognition of two recognition sequences. This unexpected mode of DNA binding and methylation has not been observed previously and sets a new precedent for a DNA methyltransferase. We also show that methylation at two recognition sequences occurs independently, and that GTWWAC motifs are enriched in intergenic regions of a strain of B. cenocepacias genome. We further computationally assess the interactions underlying the affinities of different ligands (SAM, SAH, and sinefungin) for M.BceJIV, as a step towards developing selective inhibitors for limiting B. cenocepacia infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577651">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577651" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577651">
        <p class="paperTitle">Solid-state nanopore detection of partially denatured dsDNA with single-strand binding protein: a preliminary analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577651" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577651" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Howald, N.; Klotz, A.</p>
        <p class="info">Score: 4.3, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577651' target='https://doi.org/10.1101/2024.01.28.577651'> 10.1101/2024.01.28.577651</a></p>
        <p class="abstract">In this work we investigate the use of a nanopore sensor to detect single-strand binding protein (SSB) attached to AT-rich denaturation bubbles in genomic double-stranded (ds) DNA. DNA from the {lambda} bacteriophage was heated in the presence of E. coli SSB at temperatures predicted to open denaturation bubbles near the center of the molecule. A solid state nanopore sensor measured the ionic current as the DNA-SSB solution flowed through the pore, detecting blockades due to the translocation of biomolecules. Large current spikes were observed in the translocating DNA molecules, consistent with SSB binding. However, spikes were largely localized at either end of the DNA molecule, rather than at the predicted sites. We discuss the physico-chemical effects behind this disagreement and prospects for the future use of this technique for genomic mapping.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576869">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576869" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576869">
        <p class="paperTitle">Measurement of solubility product in a model condensate reveals the interplay of small oligomerization and self-association</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576869" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576869" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chattaraj, A.; Baltaci, Z.; Mayer, B. J.; Loew, L. M.; Ditlev, J. A.</p>
        <p class="info">Score: 3.9, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576869' target='https://doi.org/10.1101/2024.01.23.576869'> 10.1101/2024.01.23.576869</a></p>
        <p class="abstract">Cellular condensates often consist of 10s to 100s of distinct interacting molecular species. Because of the complexity of these interactions, predicting the point at which they will undergo phase separation into discrete compartments is daunting. Using experiments and computation, we therefore studied a simple model system consisting of 2 proteins, polySH3 and polyPRM, designed for pentavalent heterotypic binding. We tested whether the peak solubility product, the product of dilute phase monomer concentrations, is a predictive parameter for the onset of phase separation. Titrating up equal total concentrations of each component showed that the maximum solubility product does approximately coincide with the threshold for phase separation in both the experiments and models. However, we found that measurements of dilute phase concentration include contributions from small oligomers, not just monomers; therefore, a quantitative comparison of the experiments and models required inclusion of small oligomers in the model analysis. We also examined full phase diagrams where the model results were almost symmetric along the diagonal, but the experimental results were highly asymmetric. This led us to perform dynamic light scattering experiments, where we discovered a weak homotypic interaction for polyPRM; when this was added to the computational model, it was able to recapitulate the experimentally observed asymmetry. Thus, comparing experiments to simulation reveals that the solubility product can be predictive of phase separation, even if small oligomers and low affinity homotypic interactions preclude experimental measurement of monomer concentration.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577479">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577479" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577479">
        <p class="paperTitle">Scorpion α-toxin LqhαIT specifically interacts with a glycan at the pore domain of voltage-gated sodium channels</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577479" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577479" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Phulera, S.; Dickson, C. J.; Schwalen, C. J.; Khoshouei, M.; Cassell, S. J.; Sun, Y.; Condos, T.; Whicher, J.; Weihofen, W. A.</p>
        <p class="info">Score: 3.8, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577479' target='https://doi.org/10.1101/2024.01.26.577479'> 10.1101/2024.01.26.577479</a></p>
        <p class="abstract">Voltage-gated sodium (Nav) channels sense membrane potential and drive cellular electrical activity. Numerous protein toxins have been identified that modulate Nav gating, and structures of Nav channels in complex with these toxins helped elucidate the molecular mechanisms of voltage-dependent channel gating. The deathstalker scorpion -toxin LqhIT exerts a strong action potential prolonging effect on Nav channels. Biochemical studies show that LqhIT features a functionally essential epitope at its C-terminus that is not shared with related scorpion -toxins. To elucidate the mechanism of action of LqhIT, we determined a 3.9&amp;Aring cryo-electron microscopy (cryo-EM) structure of LqhIT in complex with the Nav channel from Periplaneta americana (NavPas). We found that LqhIT binds to voltage sensor domain 4 and traps it in a S4 down conformation to stabilize the open state. To promote binding, the functionally essential C-terminal epitope of LqhIT forms an extensive interface with the glycan scaffold linked to Asn330 of NavPas that augments a small protein-protein interface between NavPas and LqhIT. A combination of molecular dynamics simulations, structural comparisons, and prior mutagenesis experiments demonstrate the functional importance of this toxin-glycan interaction. These findings help establish a structural basis for the specificity achieved by scorpion -toxins and provide crucial insights for the development and optimization of new Nav channel modulators.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576883">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576883" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576883">
        <p class="paperTitle">Chaperone quality control in liquid-phase separated organelles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576883" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576883" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scheidt, T.; Lemke, E. A.</p>
        <p class="info">Score: 5.8, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576883' target='https://doi.org/10.1101/2024.01.23.576883'> 10.1101/2024.01.23.576883</a></p>
        <p class="abstract">Molecular chaperones, central to the cellular proteostasis network, play an essential role in preventing the formation and proliferation of harmful aggregates associated with neurodegenerative diseases. Notably, for many intrinsically disordered proteins (IDPs), which are prone to form such damaging deposits, the formation of nano-clusters and phase separation into organelles prior to aggregation have been observed. The impact of molecular chaperones on such assemblies, remains unclear. In our study, we concentrated on the family of small heat shock proteins (sHsps), which are typically dynamic and form large oligomeric structures. While sHsps are mainly structured/folded proteins, they can undergo transient multivalent interactions, like many IDPs. Thus, sHsps might be a suitable regulator for vital and ubiquitous formation of membrane-less organelles in eukaryotic cells rich in IDPs and to inhibit aberrant aggregation. Here we show, using microfluidic diffusional sizing, that the formation of nano-clusters of FUS, associated with neurodegenerative diseases can be inhibited by the presence of sHsps. Furthermore, we identify that, depending on their assembly state, sHsps are capable of targeting specifically the interface between the dense droplet phase and the dilute phase not only of FUS but also of TDP-43, likely because the interface is the primary starting point for fibril formation or protein aggregation in general. Our findings emphasise the impact of molecular chaperones on maintaining the homeostasis of IDPs in the dilute and condensed phase. This could help to understand how chaperone dysregulation can influence aberrant protein association.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576082">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576082" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576082">
        <p class="paperTitle">Spectrally Resolved Localization Microscopy with Ultrahigh-Throughput</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576082" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576082" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Batey, J. E.; Kim, G. W.; Yang, M.; Heffer, D. C.; Pott, E. D.; Giang, H.; Dong, B.</p>
        <p class="info">Score: 5.2, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576082' target='https://doi.org/10.1101/2024.01.17.576082'> 10.1101/2024.01.17.576082</a></p>
        <p class="abstract">Single-molecule localization microscopy (SMLM) has become a strong technique in the toolbox of chemists, biologists, physicists, and engineers in recent years for its unique ability to resolve characteristic features quickly and accurately in complex environments at the nanoscopic level. Multicolor super-resolution imaging has seen the greatest advancement among SMLM techniques, drastically improving the differentiation ability of nanostructures beyond the diffraction limit and increasing the resolution with which previously unresolvable structures are studied. However, current multicolor SMLM methodologies present low spatial resolution and throughput and require complex optical systems. Here, we overcome these drawbacks by developing an ultrahigh-throughput SMLM methodology that allows for ultrahigh throughput multicolor imaging at the nanoscopic level using a color glass filter. Our methodology can readily distinguish fluorophores of close spectral emission and achieves sub-10 nm localization and sub-5nm spectral precisions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576405">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576405" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576405">
        <p class="paperTitle">Self-organization of PIP3 signaling is controlled by the confinement, topology, and curvature of the cell membrane.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576405" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576405" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Erisis, S.; Hoerning, M.</p>
        <p class="info">Score: 3.1, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576405' target='https://doi.org/10.1101/2024.01.20.576405'> 10.1101/2024.01.20.576405</a></p>
        <p class="abstract">PIP3 is a signaling lipid on the plasma membrane that plays a fundamental role in cell signaling with a strong impact on cell physiology and diseases. It is responsible for the protruding edge formation, cell polarization, macropinocytosis and other membrane remodeling dynamics in cells. It has been shown that the membrane confinement and curvature affects the wave formation of PIP3 and Factin. But even in the absence of F-actin, a complex self-organization of the spatiotemporal PIP3 waves is observed. In recent findings, we have shown that these waves can be guided and pinned on strongly bended Dictyostelium membranes caused by molecular crowding and curvature limited diffusion. Based on these experimental findings, we investigate the spatiotemporal PIP3 wave dynamics on realist 3D cell-like membranes to explore the effect of curvature limited diffusion, as observed experimentally. We use an established stochastic reaction-diffusion model with enzymatic MichaelisMenten type reactions that mimics the dynamics of Dictyostelium cells. As these cells mimic the 3D shape and size observed experimentally, we found that the PIP3 wave directionality can be explained by a Hopf-like and a reverse periodic-doubling bifurcation for uniform diffusion and curvature limited diffusion properties. Finally, we compare the results with recent experimental findings and discuss the discrepancy between the biological and numerical results.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.27.577133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.27.577133" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.27.577133">
        <p class="paperTitle">Multi-Channel smFRET study reveals a Compact conformation of EF-G on the Ribosome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.27.577133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.27.577133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, J.; Steele, J.; Lin, R.; Stepanov, V.; Gavriliuc, M.; wang, y.</p>
        <p class="info">Score: 2.5, Published: 2024-01-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.27.577133' target='https://doi.org/10.1101/2024.01.27.577133'> 10.1101/2024.01.27.577133</a></p>
        <p class="abstract">While elongation factor G (EF-G) is crucial for ribosome translocation, the role of its GTP hydrolysis remains ambiguous. The indispensability of EF-G is further exemplified by the phosphorylation of human eukaryotic elongation factor 2 (eEF2) at Thr56, which inhibits protein synthesis globally, but its exact mechanism is not clear. In this study, we developed a multi-channel single-molecule FRET (smFRET) microscopy methodology to examine the conformational changes of E. coli EF-G induced by mutations that closely aligned with Thr56 residue of eEF2. We utilized Alexa 488/594 double-labeled EF-G to catalyze the translocation of fMet-Phe-tRNAPhe-Cy3 inside Cy5-L27 labeled ribosomes, allowing us to probe both processes within the same complex. Our findings indicate that in the presence of either GTP or GDPCP, wild-type EF-G undergoes a conformational extension upon binding to the ribosome to promote normal translocation. On the other hand, T48E and T48V mutations did not affect GTP/GDP binding or GTP hydrolysis, but impeded Poly(Phe) synthesis and caused EF-G to adopt a unique compact conformation, which was not observed when the mutants interact solely with the sarcin/ricin loop. This study provides new insights into EF-G&#39;s adaptability and sheds light on the modification mechanism of human eEF2.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
