<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Bacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems
Authors: Chambers, L. R.; Ye, Q.; Cai, J.; Gong, M.; Ledvina, H. E.; Zhou, H.; Whiteley, A. T.; Suhandynata, R. T.; Corbett, K. D.
Score: 50.8, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559546
Ubiquitination and related pathways play crucial roles in protein homeostasis, signaling, and innate immunity1-3. In these pathways, an enzymatic cascade of E1, E2, and E3 proteins conjugates ubiquitin or a ubiquitin-like protein (Ubl) to targetprotein lysine residues4.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Bacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems
Authors: Chambers, L. R.; Ye, Q.; Cai, J.; Gong, M.; Ledvina, H. E.; Zhou, H.; Whiteley, A. T.; Suhandynata, R. T.; Corbett, K. D.
Score: 50.8, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559546
Ubiquitination and related pathways play crucial roles in protein homeostasis, signaling, and innate immunity1-3. In these pathways, an enzymatic cascade of E1, E2, and E3 proteins conjugates ubiquitin or a ubiquitin-like protein (Ubl) to targetprotein lysine residues4." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-04T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-10-04T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Bacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems
Authors: Chambers, L. R.; Ye, Q.; Cai, J.; Gong, M.; Ledvina, H. E.; Zhou, H.; Whiteley, A. T.; Suhandynata, R. T.; Corbett, K. D.
Score: 50.8, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559546
Ubiquitination and related pathways play crucial roles in protein homeostasis, signaling, and innate immunity1-3. In these pathways, an enzymatic cascade of E1, E2, and E3 proteins conjugates ubiquitin or a ubiquitin-like protein (Ubl) to targetprotein lysine residues4."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Bacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems\nAuthors: Chambers, L. R.; Ye, Q.; Cai, J.; Gong, M.; Ledvina, H. E.; Zhou, H.; Whiteley, A. T.; Suhandynata, R. T.; Corbett, K. D.\nScore: 50.8, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559546\nUbiquitination and related pathways play crucial roles in protein homeostasis, signaling, and innate immunity1-3. In these pathways, an enzymatic cascade of E1, E2, and E3 proteins conjugates ubiquitin or a ubiquitin-like protein (Ubl) to targetprotein lysine residues4.",
  "keywords": [
    
  ],
  "articleBody": " Bacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems\nAuthors: Chambers, L. R.; Ye, Q.; Cai, J.; Gong, M.; Ledvina, H. E.; Zhou, H.; Whiteley, A. T.; Suhandynata, R. T.; Corbett, K. D.\nScore: 50.8, Published: 2023-09-26 DOI: 10.1101/2023.09.26.559546\nUbiquitination and related pathways play crucial roles in protein homeostasis, signaling, and innate immunity1-3. In these pathways, an enzymatic cascade of E1, E2, and E3 proteins conjugates ubiquitin or a ubiquitin-like protein (Ubl) to targetprotein lysine residues4. Bacteria encode ancient relatives of E1 and Ubl proteins involved in sulfur metabolism5,6 but these proteins do not mediate Ubl-target conjugation, leaving open the question of whether bacteria can perform ubiquitination-like protein conjugation. Here, we demonstrate that a bacterial antiviral immune system encodes a complete ubiquitination pathway. Two structures of a bacterial E1:E2:Ubl complex reveal striking architectural parallels with canonical eukaryotic ubiquitination machinery. The bacterial E1 encodes an N-terminal inactive adenylation domain (IAD) and a C-terminal active adenylation domain (AAD) with a mobile -helical insertion containing the catalytic cysteine (CYS domain). One structure reveals a pre-reaction state with the bacterial Ubl C-terminus positioned for adenylation, and the E1 CYS domain poised nearby for thioester formation. A second structure mimics an E1-to-E2 transthioesterification state, with the E1 CYS domain rotated outward and its catalytic cysteine adjacent to the bound E2. We show that a deubiquitinase (DUB) in the same pathway pre-processes the bacterial Ubl, exposing its C-terminal glycine for adenylation. Finally, we show that the bacterial E1 and E2 collaborate to conjugate Ubl to target-protein lysine residues. Together, these data reveal that bacteria possess bona fide ubiquitination systems with strong mechanistic and architectural parallels to canonical eukaryotic ubiquitination pathways, suggesting that these pathways arose first in bacteria.\nStructural and quantum chemical basis for OCP-mediated quenching of phycobilisomes\nAuthors: Sauer, P. V.; Cupellini, L.; Sutter, M.; Bondanza, M.; Dominguez Martin, M. A.; Kirst, H.; Bina, D.; Koh, A. F.; Kotecha, A.; Greber, B. J.; Nogales, E.; Polivka, T.; Mennucci, B.; Kerfeld, C. A.\nScore: 26.8, Published: 2023-09-30 DOI: 10.1101/2023.09.30.560311\nCyanobacteria employ large antenna complexes called phycobilisomes (PBS) for light harvesting. However, intense light triggers non-photochemical quenching, where the Orange Carotenoid Protein (OCP) binds to PBS, dissipating excess energy as heat. The mechanism of efficiently transferring energy from phycocyanobilins in PBS to canthaxanthin in OCP remains insufficiently understood. Using advanced cryogenic-electron microscopy, we unveiled the OCP-PBS complex structure at 1.6-2.1 [A] resolution, showcasing its inherent flexibility. Employing multiscale quantum chemistry, we disclosed the quenching mechanism. Identifying key protein residues, we clarified how canthaxanthin's transition dipole moment in its lowest-energy dark state becomes large enough for efficient energy transfer from phycocyanobilins. Our energy transfer model offers a detailed understanding of the atomic determinants of light harvesting regulation and antenna architecture in cyanobacteria.\nShort N-terminal disordered regions and the proline-rich domain are major regulators of phase transitions for full-length UBQLN1, UBQLN2 and UBQLN4\nAuthors: Dao, T. P.; Rajendran, A.; Galagedera, S. K. K.; Haws, W.; Castaneda, C. A.\nScore: 12.3, Published: 2023-09-29 DOI: 10.1101/2023.09.27.559790\nHighly homologous ubiquitin-binding shuttle proteins UBQLN1, UBQLN2 and UBQLN4 differ in both their specific protein quality control functions and their propensities to localize to stress-induced condensates, cellular aggregates and aggresomes. We previously showed that UBQLN2 phase separates in vitro, and that the phase separation propensities of UBQLN2 deletion constructs correlate with their ability to form condensates in cells. Here, we demonstrated that full-length UBQLN1, UBQLN2 and UBQLN4 exhibit distinct phase behaviors in vitro. Strikingly, UBQLN4 phase separates at a much lower saturation concentration than UBQLN1. However, neither UBQLN1 nor UBQLN4 phase separates with a strong temperature dependence, unlike UBQLN2. We determined that the temperature-dependent phase behavior of UBQLN2 stems from its unique proline-rich (Pxx) region, which is absent in the other UBQLNs. We found that the short N-terminal disordered regions of UBQLN1, UBQLN2 and UBQLN4 inhibit UBQLN phase separation via electrostatics interactions. Charge variants of the N-terminal regions exhibit altered phase behaviors. Consistent with the sensitivity of UBQLN phase separation to the composition of the N-terminal regions, epitope tags placed on the N-termini of the UBQLNs tune phase separation. Overall, our in vitro results have important implications for studies of UBQLNs in cells, including the identification of phase separation as a potential mechanism to distinguish the cellular roles of UBQLNs, and the need to apply caution when using epitope tags to prevent experimental artifacts.\nFull-Length Single-Molecule Protein Fingerprinting\nAuthors: Filius, M.; van Wee, R.; de Lannoy, C.; Westerlaken, I.; Li, Z.; Kim, S. H.; de Agrela Pinto, C.; Wu, Y.; Boons, G.-J.; Pabst, M.; de Ridder, D.; Joo, C.\nScore: 12.1, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559471\nProteins are the primary functional actors of the cell. Hence, their identification is pivotal to advance our understanding of cell biology and disease. Current protein analysis methods are of limited use for distinguishing proteoforms. In particular, mass spectrometric methods often provide only ambiguous information on post-translational modification sites, and sequences of co-existing modifications may not be resolved. Here we demonstrate FRET-based single-molecule protein fingerprinting to map the location of individual amino acids and a post-translational modification within single full-length protein molecules. Using an approach that relies on transient binding of fluorescently labeled DNA strands to probe the amino acids on a protein one by one we show that we can fingerprint intrinsically disordered proteins as well as folded globular proteins with sub-nanometer resolution. We anticipate that this technology will be used for proteoform identification in biological and translational research with ultimate sensitivity.\nAn optogenetic method for the controlled release of single molecules\nAuthors: Kashyap, P.; Bertelli, S.; Cao, F.; Kostritskaia, Y.; Blank, F.; Srikanth, N.; Saleppico, R.; Bierhuizen, D.; Lu, X.; Nickel, W.; Campbell, R. E.; Plested, A.; Stauber, T.; Taylor, M. J.; Ewers, H.\nScore: 25.8, Published: 2023-09-17 DOI: 10.1101/2023.09.16.557871\nWe developed a system for optogenetic release of single molecules in live cells. We confined soluble and transmembrane proteins to the Golgi apparatus via a photocleavable protein and released them by short pulses of light. Our method allows for the controlled delivery of functional proteins to cytosol and plasma membrane in amounts compatible with single molecule imaging, greatly simplifying access to single molecule microscopy of any protein in live cells. Furthermore, we could reconstitute cellular functions such as ion conductance by delivering BK and VRAC ion channels to the plasma membrane. Finally, we could induce NF-kB signaling in T-Lymphoblasts stimulated by IL-1 by controlled release of a signaling protein that had been knocked-out in the same cells. We observed light induced formation of functional inflammatory signaling complexes that could trigger IKK phosphorylation in single cells. We thus developed an optogenetic method for the reconstitution and investigation of cellular function at the single molecule level.\nComputing absolute binding affinities by Streamlined Alchemical Free Energy Perturbation (SAFEP)\nAuthors: Santiago-McRae, E.; Ebrahimi, M.; Sandberg, J. W.; Brannigan, G.; Henin, J.\nScore: 20.0, Published: 2023-09-28 DOI: 10.1101/2022.12.09.519809\nFree Energy Perturbation (FEP) is a powerful but challenging computational technique for estimating differences in free energy between two or more states. This document is intended both as a tutorial and as an adaptable protocol for computing free energies of binding using free energy perturbations in NAMD. We present the Streamlined Alchemical Free Energy Perturbation (SAFEP) framework. SAFEP shifts the computational frame of reference from the ligand to the binding site itself. This both simplifies the thermodynamic cycle and makes the approach more broadly applicable to superficial sites and other less common geometries. As a practical example, we give instructions for calculating the absolute binding free energy of phenol to lysozyme. We assume familiarity with standard procedures for setting up, running, and analyzing molecular dynamics simulations using NAMD and VMD. While simulation times will vary, the human tasks should take no more than 3 to 4 hours for a reader without previous training in free energy calculations or experience with the Colvars Dashboard. Sample data are provided for all key calculations both for comparison and readers' convenience.\nPattern formation by bacteria-phage interactions\nAuthors: Martinez-Calvo, A.; Wingreen, N. S.; Datta, S. S.\nScore: 38.1, Published: 2023-09-20 DOI: 10.1101/2023.09.19.558479\nThe interactions between bacteria and phages--viruses that infect bacteria--play critical roles in agriculture, ecology, and medicine; however, how these interactions influence the spatial organization of both bacteria and phages remain largely unexplored. Here, we address this gap in knowledge by developing a theoretical model of motile, proliferating bacteria that aggregate via motility-induced phase separation (MIPS) and encounter phage that infect and lyse the cells. We find that the non-reciprocal predator-prey interactions between phage and bacteria strongly alter spatial organization, in some cases giving rise to a rich array of finite-scale stationary and dynamic patterns in which bacteria and phage coexist. We establish principles describing the onset and characteristics of these diverse behaviors, thereby helping to provide a biophysical basis for understanding pattern formation in bacteria-phage systems, as well as in a broader range of active and living systems with similar predator-prey or other non-reciprocal interactions.\nSynthesis, Insertion and Characterization of SARS-CoV-2 Membrane Protein Within Lipid Bilayers\nAuthors: Zhang, Y.; Anbir, S.; Mctiernan, J.; Li, S.; Worcester, M.; Mishra, P.; Colvin, M. E.; Gopinathan, A.; Mohideen, U.; Zandi, R.; Kuhlman, T. E.\nScore: 6.9, Published: 2023-10-02 DOI: 10.1101/2023.09.30.560318\nThe membrane protein (M) is the most abundant structural protein in the SARS-CoV-2 virus and functions exclusively as a membrane-embedded homodimer. M protein is required for the formation of the SARS-CoV-2 virus particle and has been shown to interact with the Spike and Envelope proteins, as well as the RNA-packaging Nucleocapsid protein. Our knowledge of M protein is very limited due to its small size and challenges in expressing enough protein for use in structural and biophysical experiments. We report the successful development of a SUMO tag-based expression system to produce and purify significant quantities of M protein, and a method to insert the synthesized dimers into a suspended lipid membrane in a homogeneous orientation. We used AFM and Cryo-EM to image individual membrane-bound M protein dimers and characterize the configurations that they can assume. Our experimental results are in agreement with our molecular dynamics simulations which predict thinning of the membrane around the M protein and a propensity to induce local membrane curvature. Taken together, our results shed new light on M protein properties within the lipid bilayer and suggest mechanisms that could contribute to viral assembly and budding.\niMAX FRET (Information Maximized FRET) for multipoint single-molecule structural analysis\nAuthors: Joshi, B. S.; de Lannoy, C. V.; Howarth, M.; Kim, S. H.; Joo, C.\nScore: 6.3, Published: 2023-09-29 DOI: 10.1101/2023.09.27.559821\nUnderstanding the structure of biomolecules is vital for deciphering their characteristics and roles in biological systems. While current structural analysis techniques like nuclear magnetic resonance and X-ray crystallography excel in many aspects, they fall short in capturing comprehensive single-molecule information. To address this limitation and to better capture the heterogeneity and dynamic range of biomolecular reactions, there is a need for single-molecule structural analysis tools. To achieve this, we introduce iMAX FRET, a one-pot FRET-based single-molecule method integrated with geometrical 3D reconstruction, offering comprehensive ab initio structural analysis. Through the stochastic exchange of fluorescent weak binders, iMAX FRET allows simultaneous assessment of multiple spatial coordinates on a biomolecule within a few minutes of time to generate distinct FRET fingerprints for 3D structural profiling. We demonstrate a mathematical approach for de novo structural prediction using iMAX data, opening avenues for native biomolecule analysis. Furthermore, this method facilitates the investigation of conformational changes in individual molecules, illuminating single-molecule structural dynamics. Our technique has the potential to emerge as a powerful approach to advance our understanding of biomolecular structures.\nTransition-based reweighting and neural rational inference analysis of effects of new psychoactive substances on cannabinoid receptors\nAuthors: Dutta, S.; Shukla, D.\nScore: 6.1, Published: 2023-10-02 DOI: 10.1101/2023.09.29.560261\nNew psychoactive substances (NPS) targeting cannabinoid receptor 1 (CB1) pose a significant threat to society as recreational abusive drugs that can avoid detection and have higher physiological side effects. These physiological side effects of NPS are shown to be linked to the higher {beta}-arrestin signaling. We hypothesize that the difference in conformational dynamics of the NPxxY motif causes the distinct downstream signaling of NPS contrary to the classical cannabinoids. To compare the dynamic effects of the NPS and classical cannabinoid binding on the NPxxY conformational ensemble, we simulate (un)binding process of NPS MDMB-Fubinaca and classical cannabinoid HU-210 from CB1 using unbiased and biased molecular dynamics simulations. The transition-based reweighing method (TRAM) is used to combine multi-ensemble simulations for the estimation of transition rates and underlying thermodynamics of (un)binding processes of ligands with nanomolar affinities, where it is more expensive to obtain local reversible sampling. Our analyses suggest that the ligands unbind from the receptors using the same pathway but by a different mechanism. Further analyses reveal higher conformational fluctuation in the NPxxY motif for NPS bound CB1, supporting our hypothesis. The observation is further validated using a Variational autoencoder (VAE) based on Neural rational inference, which shows higher dynamic allostery-based interactions between the binding pocket residues and NPxxY for NPS bound CB1. Hence, in this work, MD simulation, data-driven statistical methods, and deep learning point out the significant differences in (un)binding and downstream signaling of NPS and classical cannabinoids.\n",
  "wordCount" : "2180",
  "inLanguage": "en",
  "datePublished": "2023-10-04T10:39:33Z",
  "dateModified": "2023-10-04T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on October 4, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559546">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559546" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559546">
        <p class="paperTitle">Bacterial antiviral defense pathways encode eukaryotic-like ubiquitination systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559546" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559546" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chambers, L. R.; Ye, Q.; Cai, J.; Gong, M.; Ledvina, H. E.; Zhou, H.; Whiteley, A. T.; Suhandynata, R. T.; Corbett, K. D.</p>
        <p class="info">Score: 50.8, Published: 2023-09-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559546' target='https://doi.org/10.1101/2023.09.26.559546'> 10.1101/2023.09.26.559546</a></p>
        <p class="abstract">Ubiquitination and related pathways play crucial roles in protein homeostasis, signaling, and innate immunity1-3. In these pathways, an enzymatic cascade of E1, E2, and E3 proteins conjugates ubiquitin or a ubiquitin-like protein (Ubl) to targetprotein lysine residues4. Bacteria encode ancient relatives of E1 and Ubl proteins involved in sulfur metabolism5,6 but these proteins do not mediate Ubl-target conjugation, leaving open the question of whether bacteria can perform ubiquitination-like protein conjugation. Here, we demonstrate that a bacterial antiviral immune system encodes a complete ubiquitination pathway. Two structures of a bacterial E1:E2:Ubl complex reveal striking architectural parallels with canonical eukaryotic ubiquitination machinery. The bacterial E1 encodes an N-terminal inactive adenylation domain (IAD) and a C-terminal active adenylation domain (AAD) with a mobile -helical insertion containing the catalytic cysteine (CYS domain). One structure reveals a pre-reaction state with the bacterial Ubl C-terminus positioned for adenylation, and the E1 CYS domain poised nearby for thioester formation. A second structure mimics an E1-to-E2 transthioesterification state, with the E1 CYS domain rotated outward and its catalytic cysteine adjacent to the bound E2. We show that a deubiquitinase (DUB) in the same pathway pre-processes the bacterial Ubl, exposing its C-terminal glycine for adenylation. Finally, we show that the bacterial E1 and E2 collaborate to conjugate Ubl to target-protein lysine residues. Together, these data reveal that bacteria possess bona fide ubiquitination systems with strong mechanistic and architectural parallels to canonical eukaryotic ubiquitination pathways, suggesting that these pathways arose first in bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560311">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560311" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560311">
        <p class="paperTitle">Structural and quantum chemical basis for OCP-mediated quenching of phycobilisomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560311" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560311" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sauer, P. V.; Cupellini, L.; Sutter, M.; Bondanza, M.; Dominguez Martin, M. A.; Kirst, H.; Bina, D.; Koh, A. F.; Kotecha, A.; Greber, B. J.; Nogales, E.; Polivka, T.; Mennucci, B.; Kerfeld, C. A.</p>
        <p class="info">Score: 26.8, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560311' target='https://doi.org/10.1101/2023.09.30.560311'> 10.1101/2023.09.30.560311</a></p>
        <p class="abstract">Cyanobacteria employ large antenna complexes called phycobilisomes (PBS) for light harvesting. However, intense light triggers non-photochemical quenching, where the Orange Carotenoid Protein (OCP) binds to PBS, dissipating excess energy as heat. The mechanism of efficiently transferring energy from phycocyanobilins in PBS to canthaxanthin in OCP remains insufficiently understood. Using advanced cryogenic-electron microscopy, we unveiled the OCP-PBS complex structure at 1.6-2.1 [A] resolution, showcasing its inherent flexibility. Employing multiscale quantum chemistry, we disclosed the quenching mechanism. Identifying key protein residues, we clarified how canthaxanthin&#39;s transition dipole moment in its lowest-energy dark state becomes large enough for efficient energy transfer from phycocyanobilins. Our energy transfer model offers a detailed understanding of the atomic determinants of light harvesting regulation and antenna architecture in cyanobacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559790">
        <p class="paperTitle">Short N-terminal disordered regions and the proline-rich domain are major regulators of phase transitions for full-length UBQLN1, UBQLN2 and UBQLN4</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dao, T. P.; Rajendran, A.; Galagedera, S. K. K.; Haws, W.; Castaneda, C. A.</p>
        <p class="info">Score: 12.3, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559790' target='https://doi.org/10.1101/2023.09.27.559790'> 10.1101/2023.09.27.559790</a></p>
        <p class="abstract">Highly homologous ubiquitin-binding shuttle proteins UBQLN1, UBQLN2 and UBQLN4 differ in both their specific protein quality control functions and their propensities to localize to stress-induced condensates, cellular aggregates and aggresomes. We previously showed that UBQLN2 phase separates in vitro, and that the phase separation propensities of UBQLN2 deletion constructs correlate with their ability to form condensates in cells. Here, we demonstrated that full-length UBQLN1, UBQLN2 and UBQLN4 exhibit distinct phase behaviors in vitro. Strikingly, UBQLN4 phase separates at a much lower saturation concentration than UBQLN1. However, neither UBQLN1 nor UBQLN4 phase separates with a strong temperature dependence, unlike UBQLN2. We determined that the temperature-dependent phase behavior of UBQLN2 stems from its unique proline-rich (Pxx) region, which is absent in the other UBQLNs. We found that the short N-terminal disordered regions of UBQLN1, UBQLN2 and UBQLN4 inhibit UBQLN phase separation via electrostatics interactions. Charge variants of the N-terminal regions exhibit altered phase behaviors. Consistent with the sensitivity of UBQLN phase separation to the composition of the N-terminal regions, epitope tags placed on the N-termini of the UBQLNs tune phase separation. Overall, our in vitro results have important implications for studies of UBQLNs in cells, including the identification of phase separation as a potential mechanism to distinguish the cellular roles of UBQLNs, and the need to apply caution when using epitope tags to prevent experimental artifacts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559471">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559471" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559471">
        <p class="paperTitle">Full-Length Single-Molecule Protein Fingerprinting</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559471" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559471" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Filius, M.; van Wee, R.; de Lannoy, C.; Westerlaken, I.; Li, Z.; Kim, S. H.; de Agrela Pinto, C.; Wu, Y.; Boons, G.-J.; Pabst, M.; de Ridder, D.; Joo, C.</p>
        <p class="info">Score: 12.1, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559471' target='https://doi.org/10.1101/2023.09.26.559471'> 10.1101/2023.09.26.559471</a></p>
        <p class="abstract">Proteins are the primary functional actors of the cell. Hence, their identification is pivotal to advance our understanding of cell biology and disease. Current protein analysis methods are of limited use for distinguishing proteoforms. In particular, mass spectrometric methods often provide only ambiguous information on post-translational modification sites, and sequences of co-existing modifications may not be resolved. Here we demonstrate FRET-based single-molecule protein fingerprinting to map the location of individual amino acids and a post-translational modification within single full-length protein molecules. Using an approach that relies on transient binding of fluorescently labeled DNA strands to probe the amino acids on a protein one by one we show that we can fingerprint intrinsically disordered proteins as well as folded globular proteins with sub-nanometer resolution. We anticipate that this technology will be used for proteoform identification in biological and translational research with ultimate sensitivity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.16.557871">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.16.557871" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.16.557871">
        <p class="paperTitle">An optogenetic method for the controlled release of single molecules</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.16.557871" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.16.557871" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kashyap, P.; Bertelli, S.; Cao, F.; Kostritskaia, Y.; Blank, F.; Srikanth, N.; Saleppico, R.; Bierhuizen, D.; Lu, X.; Nickel, W.; Campbell, R. E.; Plested, A.; Stauber, T.; Taylor, M. J.; Ewers, H.</p>
        <p class="info">Score: 25.8, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.16.557871' target='https://doi.org/10.1101/2023.09.16.557871'> 10.1101/2023.09.16.557871</a></p>
        <p class="abstract">We developed a system for optogenetic release of single molecules in live cells. We confined soluble and transmembrane proteins to the Golgi apparatus via a photocleavable protein and released them by short pulses of light. Our method allows for the controlled delivery of functional proteins to cytosol and plasma membrane in amounts compatible with single molecule imaging, greatly simplifying access to single molecule microscopy of any protein in live cells. Furthermore, we could reconstitute cellular functions such as ion conductance by delivering BK and VRAC ion channels to the plasma membrane. Finally, we could induce NF-kB signaling in T-Lymphoblasts stimulated by IL-1 by controlled release of a signaling protein that had been knocked-out in the same cells. We observed light induced formation of functional inflammatory signaling complexes that could trigger IKK phosphorylation in single cells. We thus developed an optogenetic method for the reconstitution and investigation of cellular function at the single molecule level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.12.09.519809">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.12.09.519809" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.12.09.519809">
        <p class="paperTitle">Computing absolute binding affinities by Streamlined Alchemical Free Energy Perturbation (SAFEP)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.12.09.519809" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.12.09.519809" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Santiago-McRae, E.; Ebrahimi, M.; Sandberg, J. W.; Brannigan, G.; Henin, J.</p>
        <p class="info">Score: 20.0, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.12.09.519809' target='https://doi.org/10.1101/2022.12.09.519809'> 10.1101/2022.12.09.519809</a></p>
        <p class="abstract">Free Energy Perturbation (FEP) is a powerful but challenging computational technique for estimating differences in free energy between two or more states. This document is intended both as a tutorial and as an adaptable protocol for computing free energies of binding using free energy perturbations in NAMD. We present the Streamlined Alchemical Free Energy Perturbation (SAFEP) framework. SAFEP shifts the computational frame of reference from the ligand to the binding site itself. This both simplifies the thermodynamic cycle and makes the approach more broadly applicable to superficial sites and other less common geometries. As a practical example, we give instructions for calculating the absolute binding free energy of phenol to lysozyme. We assume familiarity with standard procedures for setting up, running, and analyzing molecular dynamics simulations using NAMD and VMD. While simulation times will vary, the human tasks should take no more than 3 to 4 hours for a reader without previous training in free energy calculations or experience with the Colvars Dashboard. Sample data are provided for all key calculations both for comparison and readers&#39; convenience.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.558479">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.558479" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.558479">
        <p class="paperTitle">Pattern formation by bacteria-phage interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.558479" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.558479" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Calvo, A.; Wingreen, N. S.; Datta, S. S.</p>
        <p class="info">Score: 38.1, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.558479' target='https://doi.org/10.1101/2023.09.19.558479'> 10.1101/2023.09.19.558479</a></p>
        <p class="abstract">The interactions between bacteria and phages--viruses that infect bacteria--play critical roles in agriculture, ecology, and medicine; however, how these interactions influence the spatial organization of both bacteria and phages remain largely unexplored. Here, we address this gap in knowledge by developing a theoretical model of motile, proliferating bacteria that aggregate via motility-induced phase separation (MIPS) and encounter phage that infect and lyse the cells. We find that the non-reciprocal predator-prey interactions between phage and bacteria strongly alter spatial organization, in some cases giving rise to a rich array of finite-scale stationary and dynamic patterns in which bacteria and phage coexist. We establish principles describing the onset and characteristics of these diverse behaviors, thereby helping to provide a biophysical basis for understanding pattern formation in bacteria-phage systems, as well as in a broader range of active and living systems with similar predator-prey or other non-reciprocal interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560318">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560318" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560318">
        <p class="paperTitle">Synthesis, Insertion and Characterization of SARS-CoV-2 Membrane Protein Within Lipid Bilayers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560318" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560318" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Y.; Anbir, S.; Mctiernan, J.; Li, S.; Worcester, M.; Mishra, P.; Colvin, M. E.; Gopinathan, A.; Mohideen, U.; Zandi, R.; Kuhlman, T. E.</p>
        <p class="info">Score: 6.9, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560318' target='https://doi.org/10.1101/2023.09.30.560318'> 10.1101/2023.09.30.560318</a></p>
        <p class="abstract">The membrane protein (M) is the most abundant structural protein in the SARS-CoV-2 virus and functions exclusively as a membrane-embedded homodimer. M protein is required for the formation of the SARS-CoV-2 virus particle and has been shown to interact with the Spike and Envelope proteins, as well as the RNA-packaging Nucleocapsid protein. Our knowledge of M protein is very limited due to its small size and challenges in expressing enough protein for use in structural and biophysical experiments. We report the successful development of a SUMO tag-based expression system to produce and purify significant quantities of M protein, and a method to insert the synthesized dimers into a suspended lipid membrane in a homogeneous orientation. We used AFM and Cryo-EM to image individual membrane-bound M protein dimers and characterize the configurations that they can assume. Our experimental results are in agreement with our molecular dynamics simulations which predict thinning of the membrane around the M protein and a propensity to induce local membrane curvature. Taken together, our results shed new light on M protein properties within the lipid bilayer and suggest mechanisms that could contribute to viral assembly and budding.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559821">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559821" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559821">
        <p class="paperTitle">iMAX FRET (Information Maximized FRET) for multipoint single-molecule structural analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559821" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559821" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joshi, B. S.; de Lannoy, C. V.; Howarth, M.; Kim, S. H.; Joo, C.</p>
        <p class="info">Score: 6.3, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559821' target='https://doi.org/10.1101/2023.09.27.559821'> 10.1101/2023.09.27.559821</a></p>
        <p class="abstract">Understanding the structure of biomolecules is vital for deciphering their characteristics and roles in biological systems. While current structural analysis techniques like nuclear magnetic resonance and X-ray crystallography excel in many aspects, they fall short in capturing comprehensive single-molecule information. To address this limitation and to better capture the heterogeneity and dynamic range of biomolecular reactions, there is a need for single-molecule structural analysis tools. To achieve this, we introduce iMAX FRET, a one-pot FRET-based single-molecule method integrated with geometrical 3D reconstruction, offering comprehensive ab initio structural analysis. Through the stochastic exchange of fluorescent weak binders, iMAX FRET allows simultaneous assessment of multiple spatial coordinates on a biomolecule within a few minutes of time to generate distinct FRET fingerprints for 3D structural profiling. We demonstrate a mathematical approach for de novo structural prediction using iMAX data, opening avenues for native biomolecule analysis. Furthermore, this method facilitates the investigation of conformational changes in individual molecules, illuminating single-molecule structural dynamics. Our technique has the potential to emerge as a powerful approach to advance our understanding of biomolecular structures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560261">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560261" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560261">
        <p class="paperTitle">Transition-based reweighting and neural rational inference analysis of effects of new psychoactive substances on cannabinoid receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560261" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560261" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dutta, S.; Shukla, D.</p>
        <p class="info">Score: 6.1, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560261' target='https://doi.org/10.1101/2023.09.29.560261'> 10.1101/2023.09.29.560261</a></p>
        <p class="abstract">New psychoactive substances (NPS) targeting cannabinoid receptor 1 (CB1) pose a significant threat to society as recreational abusive drugs that can avoid detection and have higher physiological side effects. These physiological side effects of NPS are shown to be linked to the higher {beta}-arrestin signaling. We hypothesize that the difference in conformational dynamics of the NPxxY motif causes the distinct downstream signaling of NPS contrary to the classical cannabinoids. To compare the dynamic effects of the NPS and classical cannabinoid binding on the NPxxY conformational ensemble, we simulate (un)binding process of NPS MDMB-Fubinaca and classical cannabinoid HU-210 from CB1 using unbiased and biased molecular dynamics simulations. The transition-based reweighing method (TRAM) is used to combine multi-ensemble simulations for the estimation of transition rates and underlying thermodynamics of (un)binding processes of ligands with nanomolar affinities, where it is more expensive to obtain local reversible sampling. Our analyses suggest that the ligands unbind from the receptors using the same pathway but by a different mechanism. Further analyses reveal higher conformational fluctuation in the NPxxY motif for NPS bound CB1, supporting our hypothesis. The observation is further validated using a Variational autoencoder (VAE) based on Neural rational inference, which shows higher dynamic allostery-based interactions between the binding pocket residues and NPxxY for NPS bound CB1. Hence, in this work, MD simulation, data-driven statistical methods, and deep learning point out the significant differences in (un)binding and downstream signaling of NPS and classical cannabinoids.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
