<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://yorks0n.github.io/TRxiv2" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/TRxiv2/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Adhesion energy controls lipid binding-mediated endocytosis
Authors: Groza, R.; Schmidt, K. V.; Muller, P. M.; Ronchi, P.; Schlack-Leigers, C.; Neu, U.; Puchkov, D.; Dimova, R.; Matthaus, C.; Taraska, J. W.; Weikl, T.; Ewers, H.
Score: 19.9, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546235
Several bacterial toxins and viruses can deform membranes through multivalent binding to lipids for clathrin-independent endocytosis. However, it remains unclear, how membrane deformation and endocytic internalization are mechanistically linked. Here we show that many lipid-binding virions induce membrane deformation and clathrin-independent endocytosis, suggesting a common mechanism based on multivalent lipid binding by globular particles.">
<meta name="author" content="">
<link rel="canonical" href="https://yorks0n.github.io/TRxiv2/posts/biophysics/">
<link crossorigin="anonymous" href="/TRxiv2/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/TRxiv2/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://yorks0n.github.io/TRxiv2/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://yorks0n.github.io/TRxiv2/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://yorks0n.github.io/TRxiv2/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://yorks0n.github.io/TRxiv2/apple-touch-icon.png">
<link rel="mask-icon" href="https://yorks0n.github.io/TRxiv2/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Adhesion energy controls lipid binding-mediated endocytosis
Authors: Groza, R.; Schmidt, K. V.; Muller, P. M.; Ronchi, P.; Schlack-Leigers, C.; Neu, U.; Puchkov, D.; Dimova, R.; Matthaus, C.; Taraska, J. W.; Weikl, T.; Ewers, H.
Score: 19.9, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546235
Several bacterial toxins and viruses can deform membranes through multivalent binding to lipids for clathrin-independent endocytosis. However, it remains unclear, how membrane deformation and endocytic internalization are mechanistically linked. Here we show that many lipid-binding virions induce membrane deformation and clathrin-independent endocytosis, suggesting a common mechanism based on multivalent lipid binding by globular particles." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://yorks0n.github.io/TRxiv2/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T08:34:30+00:00" />
<meta property="article:modified_time" content="2023-06-28T08:34:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Adhesion energy controls lipid binding-mediated endocytosis
Authors: Groza, R.; Schmidt, K. V.; Muller, P. M.; Ronchi, P.; Schlack-Leigers, C.; Neu, U.; Puchkov, D.; Dimova, R.; Matthaus, C.; Taraska, J. W.; Weikl, T.; Ewers, H.
Score: 19.9, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546235
Several bacterial toxins and viruses can deform membranes through multivalent binding to lipids for clathrin-independent endocytosis. However, it remains unclear, how membrane deformation and endocytic internalization are mechanistically linked. Here we show that many lipid-binding virions induce membrane deformation and clathrin-independent endocytosis, suggesting a common mechanism based on multivalent lipid binding by globular particles."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://yorks0n.github.io/TRxiv2/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://yorks0n.github.io/TRxiv2/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Adhesion energy controls lipid binding-mediated endocytosis\nAuthors: Groza, R.; Schmidt, K. V.; Muller, P. M.; Ronchi, P.; Schlack-Leigers, C.; Neu, U.; Puchkov, D.; Dimova, R.; Matthaus, C.; Taraska, J. W.; Weikl, T.; Ewers, H.\nScore: 19.9, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546235\nSeveral bacterial toxins and viruses can deform membranes through multivalent binding to lipids for clathrin-independent endocytosis. However, it remains unclear, how membrane deformation and endocytic internalization are mechanistically linked. Here we show that many lipid-binding virions induce membrane deformation and clathrin-independent endocytosis, suggesting a common mechanism based on multivalent lipid binding by globular particles.",
  "keywords": [
    
  ],
  "articleBody": " Adhesion energy controls lipid binding-mediated endocytosis\nAuthors: Groza, R.; Schmidt, K. V.; Muller, P. M.; Ronchi, P.; Schlack-Leigers, C.; Neu, U.; Puchkov, D.; Dimova, R.; Matthaus, C.; Taraska, J. W.; Weikl, T.; Ewers, H.\nScore: 19.9, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546235\nSeveral bacterial toxins and viruses can deform membranes through multivalent binding to lipids for clathrin-independent endocytosis. However, it remains unclear, how membrane deformation and endocytic internalization are mechanistically linked. Here we show that many lipid-binding virions induce membrane deformation and clathrin-independent endocytosis, suggesting a common mechanism based on multivalent lipid binding by globular particles. We create a synthetic cellular system consisting of a lipid-anchored receptor in the form of GPI-anchored anti-GFP nanobodies and a multivalent globular binder exposing 180 regularly-spaced GFP molecules on its surface. We show that these globular, 40 nm diameter, particles bind to cells expressing the receptor, deform the plasma membrane upon adhesion and become endocytosed in a clathrin-independent manner. We explore the role of the membrane adhesion energy in endocytosis by using receptors with affinities varying over 7 orders of magnitude. Using this system, we find that once a threshold in adhesion energy is overcome to allow for membrane deformation, endocytosis occurs reliably. Multivalent, binding-induced membrane deformation by globular binders is thus sufficient for internalization to occur and we suggest it is the common, purely biophysical mechanism for lipid-binding mediated endocytosis of toxins and pathogens.\nDynamic ParB-DNA interactions initiate and maintain a partition condensate for bacterial chromosome segregation\nAuthors: Tisma, M.; Janissen, R.; Antar, H.; Martin Gonzalez, A.; Barth, R.; Beekman, T.; van der Torre, J.; Michieletto, D.; Gruber, S.; Dekker, C.\nScore: 19.1, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546419\nIn most bacteria, chromosome segregation is driven by the ParABS system where the CTPase protein ParB loads at the parS site to trigger the formation of a large partition complex. Here, we present in vitro studies of the partition complex for Bacillus subtilis ParB, using single-molecule fluorescence microscopy and AFM imaging to show that transient ParB-ParB bridges are essential for forming DNA condensates. Molecular Dynamics simulations confirm that condensation occurs abruptly at a critical concentration of ParB and show that multimerization is a prerequisite for forming the partition complex. Magnetic tweezer force spectroscopy on mutant ParB proteins demonstrates that CTP hydrolysis at the N-terminal domain is essential for DNA condensation. Finally, we show that transcribing RNA polymerases can steadily traverse the ParB-DNA partition complex. These findings uncover how ParB forms a stable yet dynamic partition complex for chromosome segregation that induces DNA condensation and segregation while enabling replication and transcription.\nStudying stochastic systems biology of the cell with single-cell genomics data\nAuthors: Gorin, G.; Vastola, J. J.; Pachter, L.\nScore: 24.7, Published: 2023-05-29 DOI: 10.1101/2023.05.17.541250\nRecent experimental developments in genome-wide RNA quantification hold considerable promise for systems biology. However, rigorously probing the biology of living cells requires a unified mathematical framework that accounts for single-molecule biological stochasticity in the context of technical variation associated with genomics assays. We review models for a variety of RNA transcription processes, as well as the encapsulation and library construction steps of microfluidics-based single-cell RNA sequencing, and present a framework to integrate these phenomena by the manipulation of generating functions. Finally, we use simulated scenarios and biological data to illustrate the implications and applications of the approach.\n",
  "wordCount" : "544",
  "inLanguage": "en",
  "datePublished": "2023-06-28T08:34:30Z",
  "dateModified": "2023-06-28T08:34:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://yorks0n.github.io/TRxiv2/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://yorks0n.github.io/TRxiv2/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://yorks0n.github.io/TRxiv2" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546235">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546235" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546235">
        <p class="paperTitle">Adhesion energy controls lipid binding-mediated endocytosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546235" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546235" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Groza, R.; Schmidt, K. V.; Muller, P. M.; Ronchi, P.; Schlack-Leigers, C.; Neu, U.; Puchkov, D.; Dimova, R.; Matthaus, C.; Taraska, J. W.; Weikl, T.; Ewers, H.</p>
        <p class="info">Score: 19.9, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546235' target='https://doi.org/10.1101/2023.06.23.546235'> 10.1101/2023.06.23.546235</a></p>
        <p class="abstract">Several bacterial toxins and viruses can deform membranes through multivalent binding to lipids for clathrin-independent endocytosis. However, it remains unclear, how membrane deformation and endocytic internalization are mechanistically linked. Here we show that many lipid-binding virions induce membrane deformation and clathrin-independent endocytosis, suggesting a common mechanism based on multivalent lipid binding by globular particles. We create a synthetic cellular system consisting of a lipid-anchored receptor in the form of GPI-anchored anti-GFP nanobodies and a multivalent globular binder exposing 180 regularly-spaced GFP molecules on its surface. We show that these globular, 40 nm diameter, particles bind to cells expressing the receptor, deform the plasma membrane upon adhesion and become endocytosed in a clathrin-independent manner. We explore the role of the membrane adhesion energy in endocytosis by using receptors with affinities varying over 7 orders of magnitude. Using this system, we find that once a threshold in adhesion energy is overcome to allow for membrane deformation, endocytosis occurs reliably. Multivalent, binding-induced membrane deformation by globular binders is thus sufficient for internalization to occur and we suggest it is the common, purely biophysical mechanism for lipid-binding mediated endocytosis of toxins and pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.25.546419">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.25.546419" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.25.546419">
        <p class="paperTitle">Dynamic ParB-DNA interactions initiate and maintain a partition condensate for bacterial chromosome segregation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.25.546419" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.25.546419" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tisma, M.; Janissen, R.; Antar, H.; Martin Gonzalez, A.; Barth, R.; Beekman, T.; van der Torre, J.; Michieletto, D.; Gruber, S.; Dekker, C.</p>
        <p class="info">Score: 19.1, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.25.546419' target='https://doi.org/10.1101/2023.06.25.546419'> 10.1101/2023.06.25.546419</a></p>
        <p class="abstract">In most bacteria, chromosome segregation is driven by the ParABS system where the CTPase protein ParB loads at the parS site to trigger the formation of a large partition complex. Here, we present in vitro studies of the partition complex for Bacillus subtilis ParB, using single-molecule fluorescence microscopy and AFM imaging to show that transient ParB-ParB bridges are essential for forming DNA condensates. Molecular Dynamics simulations confirm that condensation occurs abruptly at a critical concentration of ParB and show that multimerization is a prerequisite for forming the partition complex. Magnetic tweezer force spectroscopy on mutant ParB proteins demonstrates that CTP hydrolysis at the N-terminal domain is essential for DNA condensation. Finally, we show that transcribing RNA polymerases can steadily traverse the ParB-DNA partition complex. These findings uncover how ParB forms a stable yet dynamic partition complex for chromosome segregation that induces DNA condensation and segregation while enabling replication and transcription.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.17.541250">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.17.541250" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.17.541250">
        <p class="paperTitle">Studying stochastic systems biology of the cell with single-cell genomics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.17.541250" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.17.541250" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gorin, G.; Vastola, J. J.; Pachter, L.</p>
        <p class="info">Score: 24.7, Published: 2023-05-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.17.541250' target='https://doi.org/10.1101/2023.05.17.541250'> 10.1101/2023.05.17.541250</a></p>
        <p class="abstract">Recent experimental developments in genome-wide RNA quantification hold considerable promise for systems biology. However, rigorously probing the biology of living cells requires a unified mathematical framework that accounts for single-molecule biological stochasticity in the context of technical variation associated with genomics assays. We review models for a variety of RNA transcription processes, as well as the encapsulation and library construction steps of microfluidics-based single-cell RNA sequencing, and present a framework to integrate these phenomena by the manipulation of generating functions. Finally, we use simulated scenarios and biological data to illustrate the implications and applications of the approach.</p>
      </div>
    </div>
  </div>
</div>









<script src="/TRxiv2/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://yorks0n.github.io/TRxiv2">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
