<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Direct optical measurement of intra-molecular distances down to the Angstrom scale
Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.
Score: 71.5, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133
Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Direct optical measurement of intra-molecular distances down to the Angstrom scale
Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.
Score: 71.5, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133
Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-12T10:41:10+00:00" />
<meta property="article:modified_time" content="2023-07-12T10:41:10+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Direct optical measurement of intra-molecular distances down to the Angstrom scale
Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.
Score: 71.5, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133
Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Direct optical measurement of intra-molecular distances down to the Angstrom scale\nAuthors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.\nScore: 71.5, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133\nOptical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision.",
  "keywords": [
    
  ],
  "articleBody": " Direct optical measurement of intra-molecular distances down to the Angstrom scale\nAuthors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.\nScore: 71.5, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133\nOptical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision. Our method is validated by quantifying well-characterized 1-to-10 nm distances in polypeptides and proteins. Moreover, we visualize the orientations of immunoglobulin subunits and reveal specific configurations of a histidine kinase domain dimer. Our results open the door for examining proximities and interactions of macromolecules under physiological conditions.\nEvolution of hierarchy and irreversibility in theoretical cell differentiation model\nAuthors: Nakamura, Y. T.; Himeoka, Y.; Saito, N.; Furusawa, C.\nScore: 20.8, Published: 2023-07-07 DOI: 10.1101/2023.07.04.547517\nThe process of cell differentiation in multicellular organisms is characterized by hierarchy and irreversibility in many cases. However, the conditions and selection pressures that give rise to these characteristics remain poorly understood. By using a mathematical model, here we show that the network of differentiation potency (differentiation diagram) becomes necessarily hierarchical and irreversible by increasing the number of terminally differentiated states under certain conditions. The mechanisms generating these characteristics are clarified using geometry in the cell state space. The results demonstrate that the appearance of these characteristics can be driven without assuming the adaptive significance. The study also provides a new perspective on the structure of gene regulatory networks that produce hierarchical and irreversible differentiation diagrams. These results indicate some constraints on cell differentiation, which are expected to provide a starting point for theoretical discussion of the implicit limits and directions of evolution in multicellular organisms.\nThe killing of human gut commensal E. coli ED1a by tetracycline is associated with severe ribosome dysfunction\nAuthors: Khusainov, I.; Romanov, N.; Goemans, C.; Turonova, B.; Zimmerli, C. E.; Welsch, S.; Langer, J.; Typas, A.; Beck, M.\nScore: 19.8, Published: 2023-07-06 DOI: 10.1101/2023.07.06.546847\nRibosomes translate the genetic code into proteins. Recent technical advances have facilitated in situ structural analyses of ribosome functional states inside eukaryotic cells and the minimal bacterium Mycoplasma. However, such analyses of Gram-negative bacteria are lacking, despite their ribosomes being major antimicrobial drug targets. Here we compare two E. coli strains, a lab E. coli K-12 and human gut isolate E. coli ED1a, for which tetracycline exhibits bacteriostatic and bactericidal action, respectively. The in situ ribosome structures upon tetracycline treatment show a virtually identical drug binding-site in both strains, yet the distribution of ribosomal complexes clearly differs. While K-12 retains ribosomes in a translation competent state, tRNAs are lost in the vast majority of ED1a ribosomes. A differential response is also reflected in proteome-wide abundance and thermal stability assessment. Our study underlines the need to include molecular analyses and to consider gut bacteria when addressing antibiotic mode of action. O_LSTHIGHLIGHTSC_LST* Ribosome structures of gram-negative bacteria are analyzed in situ * Tetracyline is bactericidal to gut isolate despite identical ribosome structures * When antibiotic is bacteriostatic, ribosomal translation competent states are retained * When antibiotic is bactericidal, cells rapidly accumulate P-tRNAs-deficient ribosomes GRAPHICAL ABSTRACT\nFunctional Protein Dynamics in a Crystal\nAuthors: Klyshko, E.; Kim, J. S.-H.; McGough, L.; Valeeva, V.; Lee, E.; Ranganathan, R.; Rauscher, S.\nScore: 19.2, Published: 2023-07-07 DOI: 10.1101/2023.07.06.548023\nProteins are molecular machines and to understand how they work, we need to understand how they move. New pump-probe time-resolved X-ray diffraction methods open up ways to initiate and observe protein motions with atomistic detail in crystals on biologically relevant timescales. However, practical limitations of these experiments demands parallel development of effective molecular dynamics approaches to accelerate progress and extract meaning. Here, we establish robust and accurate methods for simulating dynamics in protein crystals, a nontrivial process requiring careful attention to equilibration, environmental composition, and choice of force fields. With more than seven milliseconds of sampling of a single chain, we identify critical factors controlling agreement between simulation and experiments and show that simulated motions recapitulate ligand-induced conformational changes. This work enables a virtuous cycle between simulation and experiments for visualizing and understanding the basic functional motions of proteins.\nAutomated Purification of DNA Origami with SPRI Beads\nAuthors: Chau, C.; Mohanan, G.; Macaulay, I. C.; Actis, P.; Walti, C.\nScore: 13.8, Published: 2023-07-05 DOI: 10.1101/2023.07.05.544573\nDNA origami synthesis is a well-established technique and has been employed in various applications. The synthesised origami must be purified to eliminate the excess materials such as DNA oligos and other molecules. While several purification techniques are routinely used, they all have limitations, and none can be automated to simultaneously handle large numbers and quantities of samples. Here we introduce the use of solid-phase immobilisation (SPRI) beads as an easy-to-adopt, scalable, high-throughput and automation-compatible method to purify DNA origami. Not only can this method remove excess oligos and biomolecules with comparable yield to existing methods while maintaining high structural integrity of the origami, but it also allows an automated workflow to simultaneously purify large numbers of samples within a limited time. We envision that the SPRI beads purification approach will improve the scalability of DNA nanostructures synthesis both for research and commercial applications.\nRapid assessment of changes in phage bioactivity using dynamic light scattering\nAuthors: Dharmaraj, T.; Kratochvil, M. J.; Pourtois, J. D.; Chen, Q.; Hajfathalian, M.; Hargil, A.; Lin, Y.-H.; Evans, Z. R.; Oromi-Bosch, A.; Berry, J. D.; McBride, R.; Haddock, N. L.; Holman, D. R.; Van Belleghem, J. D.; Chang, T. H.; Barr, J. J.; Lavigne, R.; Heilshorn, S. C.; Blankenberg, F. G.; Bollyky, P. L.\nScore: 26.7, Published: 2023-07-02 DOI: 10.1101/2023.07.02.547396\nExtensive efforts are underway to develop bacteriophages as therapies against antibiotic-resistant bacteria. However, these efforts are confounded by the instability of phage preparations and a lack of suitable tools to assess active phage concentrations over time. Here, we use Dynamic Light Scattering (DLS) to measure changes in phage physical state in response to environmental factors and time, finding that phages tend to decay and form aggregates and that the degree of aggregation can be used to predict phage bioactivity. We then use DLS to optimize phage storage conditions for phages from human clinical trials, predict bioactivity in 50-year-old archival stocks, and evaluate phage samples for use in a phage therapy/wound infection model. We also provide a web-application (Phage-ELF) to facilitate DLS studies of phages. We conclude that DLS provides a rapid, convenient, and non-destructive tool for quality control of phage preparations in academic and commercial settings. Significance StatementPhages are promising for use in treating antibiotic-resistant infections, but their decay over time in refrigerated storage and higher temperatures has been a difficult barrier to overcome. This is in part because there are no suitable methods to monitor phage activity over time, especially in clinical settings. Here, we show that Dynamic Light Scattering (DLS) can be used to measure the physical state of phage preparations, which provides accurate and precise information on their lytic function - the key parameter underlying clinical efficacy. This study reveals a \"structure-function\" relationship for lytic phages and establishes DLS as a method to optimize the storage, handling, and clinical use of phages.\nThe importance of input sequence set to consensus-derived proteins and their relationship to reconstructed ancestral proteins\nAuthors: Nixon, C.; Lim, S. A.; Sternke, M.; Barrick, D.; Harms, M. J.; Marqusee, S.\nScore: 7.4, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547063\nA protein sequence encodes its energy landscape - all the accessible conformations, energetics, and dynamics. The evolutionary relationship between sequence and landscape can be probed phylogenetically by compiling a multiple sequence alignment of homologous sequences and generating common ancestors via Ancestral Sequence Reconstruction or a consensus protein containing the most common amino acid at each position. Both ancestral and consensus proteins are often more stable than their extant homologs - questioning the differences and suggesting that both approaches serve as general methods to engineer thermostability. We used the Ribonuclease H family to compare these approaches and evaluate how the evolutionary relationship of the input sequences affects the properties of the resulting consensus protein. While the overall consensus protein is structured and active, it neither shows properties of a well-folded protein nor has enhanced stability. In contrast, the consensus protein derived from a phylogenetically-restricted region is significantly more stable and cooperatively folded, suggesting that cooperativity may be encoded by different mechanisms in separate clades and lost when too many diverse clades are combined to generate a consensus protein. To explore this, we compared pairwise covariance scores using a Potts formalism as well as higher-order couplings using singular value decomposition (SVD). We find the SVD coordinates of a stable consensus sequence are close to coordinates of the analogous ancestor sequence and its descendants, whereas the unstable consensus sequences are outliers in SVD space.\nAtomic description of the reciprocal action between supercoils and melting bubbles on linear DNA\nAuthors: Burman, M.; Noy, A.\nScore: 6.2, Published: 2023-07-05 DOI: 10.1101/2023.06.21.545919\nAlthough the mechanical response of DNA to physiological torsion and tension is well characterized, the detailed structures are not yet known. By using molecular dynamics simulations on linear DNA with 300 bp, we provide, for the first time, the conformational phase diagram at atomic resolution. Our simulations also reveal the dynamics and diffusion of supercoils. We observe a new state in negative supercoiling, where denaturation bubbles form in AT-rich regions independently of the underlying DNA topology. We thus propose sequence-dependent bubbles could position plectonemes in longer DNA.\nAtomic-level architecture of Caulobacter crescentus flagellar filaments provide evidence for multi-flagellin filament stabilization\nAuthors: Sanchez, J. C.; Montemayor, E. J.; Ploscariu, N. T.; Parrell, D.; Baumgardt, J. K.; Yang, J. E.; Sibert, B.; Cai, K.; Wright, E. R.\nScore: 4.5, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548443\nFlagella are dynamic, ion-powered machines with assembly pathways that are optimized for efficient flagella production. In bacteria, dozens of genes are coordinated at specific times in the cell lifecycle to generate each component of the flagellum. This is the case for Caulobacter crescentus, but little is known about why this species encodes six different flagellin genes. Furthermore, little is known about the benefits multi-flagellin species possess over single flagellin species, if any, or what molecular properties allow for multi-flagellin filaments to assemble. Here we present an in-depth analysis of several single flagellin filaments from C. crescentus, including an extremely well-resolved structure of a bacterial flagellar filament. We highlight key molecular interactions that differ between each bacterial strain and speculate how these interactions may alleviate or impose helical strain on the overall architecture of the filament. We detail conserved residues within the flagellin subunit that allow for the synthesis of multi-flagellin filaments. We further comment on how these molecular differences impact bacterial motility and highlight how no single flagellin filament achieves wild-type levels of motility, suggesting C. crescentus has evolved to produce a filament optimized for motility comprised of six flagellins. Finally, we highlight an ordered arrangement of glycosylation sites on the surface of the filaments and speculate how these sites may protect the beta-hairpin located on the surface exposed domain of the flagellin subunit.\nHandling difficult cryo-ET samples: A study with primary neurons from Drosophila melanogaster\nAuthors: Kim, J. Y.; Yang, J. E.; Mitchell, J. W.; English, L. A.; Yang, S. Z.; Tenpas, T.; Dent, E. W.; Wildonger, J.; Wright, E. R.\nScore: 4.3, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548468\nCellular neurobiology has benefited from recent advances in the field of cryo-electron tomography (cryo-ET). Numerous structural and ultrastructural insights have been obtained from plunge-frozen primary neurons cultured on electron microscopy grids. With most primary neurons been derived from rodent sources, we sought to expand the breadth of sample availability by using primary neurons derived from 3rd instar Drosophila melanogaster larval brains. Ultrastructural abnormalities were encountered while establishing this model system for cryo-ET, which were exemplified by excessive membrane blebbing and cellular fragmentation. To optimize neuronal samples, we integrated substrate selection, micropatterning, montage data collection, and chemical fixation. Efforts to address difficulties in establishing Drosophila neurons for future cryo-ET studies in cellular neurobiology also provided insights that future practitioners can use when attempting to establish other cell-based model systems.\n",
  "wordCount" : "2032",
  "inLanguage": "en",
  "datePublished": "2023-07-12T10:41:10Z",
  "dateModified": "2023-07-12T10:41:10Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on July 12, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548133" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548133">
        <p class="paperTitle">Direct optical measurement of intra-molecular distances down to the Angstrom scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.</p>
        <p class="info">Score: 71.5, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548133' target='https://doi.org/10.1101/2023.07.07.548133'> 10.1101/2023.07.07.548133</a></p>
        <p class="abstract">Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision. Our method is validated by quantifying well-characterized 1-to-10 nm distances in polypeptides and proteins. Moreover, we visualize the orientations of immunoglobulin subunits and reveal specific configurations of a histidine kinase domain dimer. Our results open the door for examining proximities and interactions of macromolecules under physiological conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547517">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547517" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547517">
        <p class="paperTitle">Evolution of hierarchy and irreversibility in theoretical cell differentiation model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547517" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547517" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nakamura, Y. T.; Himeoka, Y.; Saito, N.; Furusawa, C.</p>
        <p class="info">Score: 20.8, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547517' target='https://doi.org/10.1101/2023.07.04.547517'> 10.1101/2023.07.04.547517</a></p>
        <p class="abstract">The process of cell differentiation in multicellular organisms is characterized by hierarchy and irreversibility in many cases. However, the conditions and selection pressures that give rise to these characteristics remain poorly understood. By using a mathematical model, here we show that the network of differentiation potency (differentiation diagram) becomes necessarily hierarchical and irreversible by increasing the number of terminally differentiated states under certain conditions. The mechanisms generating these characteristics are clarified using geometry in the cell state space. The results demonstrate that the appearance of these characteristics can be driven without assuming the adaptive significance. The study also provides a new perspective on the structure of gene regulatory networks that produce hierarchical and irreversible differentiation diagrams. These results indicate some constraints on cell differentiation, which are expected to provide a starting point for theoretical discussion of the implicit limits and directions of evolution in multicellular organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.546847">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.546847" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.546847">
        <p class="paperTitle">The killing of human gut commensal E. coli ED1a by tetracycline is associated with severe ribosome dysfunction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.546847" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.546847" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khusainov, I.; Romanov, N.; Goemans, C.; Turonova, B.; Zimmerli, C. E.; Welsch, S.; Langer, J.; Typas, A.; Beck, M.</p>
        <p class="info">Score: 19.8, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.546847' target='https://doi.org/10.1101/2023.07.06.546847'> 10.1101/2023.07.06.546847</a></p>
        <p class="abstract">Ribosomes translate the genetic code into proteins. Recent technical advances have facilitated in situ structural analyses of ribosome functional states inside eukaryotic cells and the minimal bacterium Mycoplasma. However, such analyses of Gram-negative bacteria are lacking, despite their ribosomes being major antimicrobial drug targets. Here we compare two E. coli strains, a lab E. coli K-12 and human gut isolate E. coli ED1a, for which tetracycline exhibits bacteriostatic and bactericidal action, respectively. The in situ ribosome structures upon tetracycline treatment show a virtually identical drug binding-site in both strains, yet the distribution of ribosomal complexes clearly differs. While K-12 retains ribosomes in a translation competent state, tRNAs are lost in the vast majority of ED1a ribosomes. A differential response is also reflected in proteome-wide abundance and thermal stability assessment. Our study underlines the need to include molecular analyses and to consider gut bacteria when addressing antibiotic mode of action.

O_LSTHIGHLIGHTSC_LST* Ribosome structures of gram-negative bacteria are analyzed in situ
* Tetracyline is bactericidal to gut isolate despite identical ribosome structures
* When antibiotic is bacteriostatic, ribosomal translation competent states are retained
* When antibiotic is bactericidal, cells rapidly accumulate P-tRNAs-deficient ribosomes


GRAPHICAL ABSTRACT</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.548023">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.548023" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.548023">
        <p class="paperTitle">Functional Protein Dynamics in a Crystal</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.548023" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.548023" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Klyshko, E.; Kim, J. S.-H.; McGough, L.; Valeeva, V.; Lee, E.; Ranganathan, R.; Rauscher, S.</p>
        <p class="info">Score: 19.2, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.548023' target='https://doi.org/10.1101/2023.07.06.548023'> 10.1101/2023.07.06.548023</a></p>
        <p class="abstract">Proteins are molecular machines and to understand how they work, we need to understand how they move. New pump-probe time-resolved X-ray diffraction methods open up ways to initiate and observe protein motions with atomistic detail in crystals on biologically relevant timescales. However, practical limitations of these experiments demands parallel development of effective molecular dynamics approaches to accelerate progress and extract meaning. Here, we establish robust and accurate methods for simulating dynamics in protein crystals, a nontrivial process requiring careful attention to equilibration, environmental composition, and choice of force fields. With more than seven milliseconds of sampling of a single chain, we identify critical factors controlling agreement between simulation and experiments and show that simulated motions recapitulate ligand-induced conformational changes. This work enables a virtuous cycle between simulation and experiments for visualizing and understanding the basic functional motions of proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.544573">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.544573" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.544573">
        <p class="paperTitle">Automated Purification of DNA Origami with SPRI Beads</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.544573" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.544573" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chau, C.; Mohanan, G.; Macaulay, I. C.; Actis, P.; Walti, C.</p>
        <p class="info">Score: 13.8, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.544573' target='https://doi.org/10.1101/2023.07.05.544573'> 10.1101/2023.07.05.544573</a></p>
        <p class="abstract">DNA origami synthesis is a well-established technique and has been employed in various applications. The synthesised origami must be purified to eliminate the excess materials such as DNA oligos and other molecules. While several purification techniques are routinely used, they all have limitations, and none can be automated to simultaneously handle large numbers and quantities of samples. Here we introduce the use of solid-phase immobilisation (SPRI) beads as an easy-to-adopt, scalable, high-throughput and automation-compatible method to purify DNA origami. Not only can this method remove excess oligos and biomolecules with comparable yield to existing methods while maintaining high structural integrity of the origami, but it also allows an automated workflow to simultaneously purify large numbers of samples within a limited time. We envision that the SPRI beads purification approach will improve the scalability of DNA nanostructures synthesis both for research and commercial applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.02.547396">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.02.547396" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.02.547396">
        <p class="paperTitle">Rapid assessment of changes in phage bioactivity using dynamic light scattering</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.02.547396" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.02.547396" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dharmaraj, T.; Kratochvil, M. J.; Pourtois, J. D.; Chen, Q.; Hajfathalian, M.; Hargil, A.; Lin, Y.-H.; Evans, Z. R.; Oromi-Bosch, A.; Berry, J. D.; McBride, R.; Haddock, N. L.; Holman, D. R.; Van Belleghem, J. D.; Chang, T. H.; Barr, J. J.; Lavigne, R.; Heilshorn, S. C.; Blankenberg, F. G.; Bollyky, P. L.</p>
        <p class="info">Score: 26.7, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.02.547396' target='https://doi.org/10.1101/2023.07.02.547396'> 10.1101/2023.07.02.547396</a></p>
        <p class="abstract">Extensive efforts are underway to develop bacteriophages as therapies against antibiotic-resistant bacteria. However, these efforts are confounded by the instability of phage preparations and a lack of suitable tools to assess active phage concentrations over time. Here, we use Dynamic Light Scattering (DLS) to measure changes in phage physical state in response to environmental factors and time, finding that phages tend to decay and form aggregates and that the degree of aggregation can be used to predict phage bioactivity. We then use DLS to optimize phage storage conditions for phages from human clinical trials, predict bioactivity in 50-year-old archival stocks, and evaluate phage samples for use in a phage therapy/wound infection model. We also provide a web-application (Phage-ELF) to facilitate DLS studies of phages. We conclude that DLS provides a rapid, convenient, and non-destructive tool for quality control of phage preparations in academic and commercial settings.

Significance StatementPhages are promising for use in treating antibiotic-resistant infections, but their decay over time in refrigerated storage and higher temperatures has been a difficult barrier to overcome. This is in part because there are no suitable methods to monitor phage activity over time, especially in clinical settings. Here, we show that Dynamic Light Scattering (DLS) can be used to measure the physical state of phage preparations, which provides accurate and precise information on their lytic function - the key parameter underlying clinical efficacy. This study reveals a &#34;structure-function&#34; relationship for lytic phages and establishes DLS as a method to optimize the storage, handling, and clinical use of phages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547063">
        <p class="paperTitle">The importance of input sequence set to consensus-derived proteins and their relationship to reconstructed ancestral proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nixon, C.; Lim, S. A.; Sternke, M.; Barrick, D.; Harms, M. J.; Marqusee, S.</p>
        <p class="info">Score: 7.4, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547063' target='https://doi.org/10.1101/2023.06.29.547063'> 10.1101/2023.06.29.547063</a></p>
        <p class="abstract">A protein sequence encodes its energy landscape - all the accessible conformations, energetics, and dynamics. The evolutionary relationship between sequence and landscape can be probed phylogenetically by compiling a multiple sequence alignment of homologous sequences and generating common ancestors via Ancestral Sequence Reconstruction or a consensus protein containing the most common amino acid at each position. Both ancestral and consensus proteins are often more stable than their extant homologs - questioning the differences and suggesting that both approaches serve as general methods to engineer thermostability. We used the Ribonuclease H family to compare these approaches and evaluate how the evolutionary relationship of the input sequences affects the properties of the resulting consensus protein. While the overall consensus protein is structured and active, it neither shows properties of a well-folded protein nor has enhanced stability. In contrast, the consensus protein derived from a phylogenetically-restricted region is significantly more stable and cooperatively folded, suggesting that cooperativity may be encoded by different mechanisms in separate clades and lost when too many diverse clades are combined to generate a consensus protein. To explore this, we compared pairwise covariance scores using a Potts formalism as well as higher-order couplings using singular value decomposition (SVD). We find the SVD coordinates of a stable consensus sequence are close to coordinates of the analogous ancestor sequence and its descendants, whereas the unstable consensus sequences are outliers in SVD space.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545919">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545919" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545919">
        <p class="paperTitle">Atomic description of the reciprocal action between supercoils and melting bubbles on linear DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545919" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545919" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burman, M.; Noy, A.</p>
        <p class="info">Score: 6.2, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545919' target='https://doi.org/10.1101/2023.06.21.545919'> 10.1101/2023.06.21.545919</a></p>
        <p class="abstract">Although the mechanical response of DNA to physiological torsion and tension is well characterized, the detailed structures are not yet known. By using molecular dynamics simulations on linear DNA with 300 bp, we provide, for the first time, the conformational phase diagram at atomic resolution. Our simulations also reveal the dynamics and diffusion of supercoils. We observe a new state in negative supercoiling, where denaturation bubbles form in AT-rich regions independently of the underlying DNA topology. We thus propose sequence-dependent bubbles could position plectonemes in longer DNA.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548443">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548443" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548443">
        <p class="paperTitle">Atomic-level architecture of Caulobacter crescentus flagellar filaments provide evidence for multi-flagellin filament stabilization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548443" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548443" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez, J. C.; Montemayor, E. J.; Ploscariu, N. T.; Parrell, D.; Baumgardt, J. K.; Yang, J. E.; Sibert, B.; Cai, K.; Wright, E. R.</p>
        <p class="info">Score: 4.5, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548443' target='https://doi.org/10.1101/2023.07.10.548443'> 10.1101/2023.07.10.548443</a></p>
        <p class="abstract">Flagella are dynamic, ion-powered machines with assembly pathways that are optimized for efficient flagella production. In bacteria, dozens of genes are coordinated at specific times in the cell lifecycle to generate each component of the flagellum. This is the case for Caulobacter crescentus, but little is known about why this species encodes six different flagellin genes. Furthermore, little is known about the benefits multi-flagellin species possess over single flagellin species, if any, or what molecular properties allow for multi-flagellin filaments to assemble. Here we present an in-depth analysis of several single flagellin filaments from C. crescentus, including an extremely well-resolved structure of a bacterial flagellar filament. We highlight key molecular interactions that differ between each bacterial strain and speculate how these interactions may alleviate or impose helical strain on the overall architecture of the filament. We detail conserved residues within the flagellin subunit that allow for the synthesis of multi-flagellin filaments. We further comment on how these molecular differences impact bacterial motility and highlight how no single flagellin filament achieves wild-type levels of motility, suggesting C. crescentus has evolved to produce a filament optimized for motility comprised of six flagellins. Finally, we highlight an ordered arrangement of glycosylation sites on the surface of the filaments and speculate how these sites may protect the beta-hairpin located on the surface exposed domain of the flagellin subunit.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548468">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548468" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548468">
        <p class="paperTitle">Handling difficult cryo-ET samples: A study with primary neurons from Drosophila melanogaster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548468" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548468" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, J. Y.; Yang, J. E.; Mitchell, J. W.; English, L. A.; Yang, S. Z.; Tenpas, T.; Dent, E. W.; Wildonger, J.; Wright, E. R.</p>
        <p class="info">Score: 4.3, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548468' target='https://doi.org/10.1101/2023.07.10.548468'> 10.1101/2023.07.10.548468</a></p>
        <p class="abstract">Cellular neurobiology has benefited from recent advances in the field of cryo-electron tomography (cryo-ET). Numerous structural and ultrastructural insights have been obtained from plunge-frozen primary neurons cultured on electron microscopy grids. With most primary neurons been derived from rodent sources, we sought to expand the breadth of sample availability by using primary neurons derived from 3rd instar Drosophila melanogaster larval brains. Ultrastructural abnormalities were encountered while establishing this model system for cryo-ET, which were exemplified by excessive membrane blebbing and cellular fragmentation. To optimize neuronal samples, we integrated substrate selection, micropatterning, montage data collection, and chemical fixation. Efforts to address difficulties in establishing Drosophila neurons for future cryo-ET studies in cellular neurobiology also provided insights that future practitioners can use when attempting to establish other cell-based model systems.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
