<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Allosteric Competition and Inhibition in AMPA Receptors
Authors: Hale, W. D.; Romero, A. M.; Gonzalez, C. U.; Jayaraman, V.; Lau, A. Y.; Huganir, R. L.; Twomey, E. C.
Score: 17.8, Published: 2023-11-28 DOI: 10.1101/2023.11.28.569057
Excitatory neurotransmission is principally mediated by AMPA-subtype ionotropic glutamate receptors (AMPARs). Dysregulation of AMPARs is the cause of many neurological disorders and how therapeutic candidates such as negative allosteric modulators inhibit AMPARs is unclear. Here, we show that non-competitive inhibition desensitizes AMPARs to activation and prevents positive allosteric modulation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Allosteric Competition and Inhibition in AMPA Receptors
Authors: Hale, W. D.; Romero, A. M.; Gonzalez, C. U.; Jayaraman, V.; Lau, A. Y.; Huganir, R. L.; Twomey, E. C.
Score: 17.8, Published: 2023-11-28 DOI: 10.1101/2023.11.28.569057
Excitatory neurotransmission is principally mediated by AMPA-subtype ionotropic glutamate receptors (AMPARs). Dysregulation of AMPARs is the cause of many neurological disorders and how therapeutic candidates such as negative allosteric modulators inhibit AMPARs is unclear. Here, we show that non-competitive inhibition desensitizes AMPARs to activation and prevents positive allosteric modulation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-06T10:40:04+00:00" />
<meta property="article:modified_time" content="2023-12-06T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Allosteric Competition and Inhibition in AMPA Receptors
Authors: Hale, W. D.; Romero, A. M.; Gonzalez, C. U.; Jayaraman, V.; Lau, A. Y.; Huganir, R. L.; Twomey, E. C.
Score: 17.8, Published: 2023-11-28 DOI: 10.1101/2023.11.28.569057
Excitatory neurotransmission is principally mediated by AMPA-subtype ionotropic glutamate receptors (AMPARs). Dysregulation of AMPARs is the cause of many neurological disorders and how therapeutic candidates such as negative allosteric modulators inhibit AMPARs is unclear. Here, we show that non-competitive inhibition desensitizes AMPARs to activation and prevents positive allosteric modulation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Allosteric Competition and Inhibition in AMPA Receptors\nAuthors: Hale, W. D.; Romero, A. M.; Gonzalez, C. U.; Jayaraman, V.; Lau, A. Y.; Huganir, R. L.; Twomey, E. C.\nScore: 17.8, Published: 2023-11-28 DOI: 10.1101/2023.11.28.569057\nExcitatory neurotransmission is principally mediated by AMPA-subtype ionotropic glutamate receptors (AMPARs). Dysregulation of AMPARs is the cause of many neurological disorders and how therapeutic candidates such as negative allosteric modulators inhibit AMPARs is unclear. Here, we show that non-competitive inhibition desensitizes AMPARs to activation and prevents positive allosteric modulation.",
  "keywords": [
    
  ],
  "articleBody": " Allosteric Competition and Inhibition in AMPA Receptors\nAuthors: Hale, W. D.; Romero, A. M.; Gonzalez, C. U.; Jayaraman, V.; Lau, A. Y.; Huganir, R. L.; Twomey, E. C.\nScore: 17.8, Published: 2023-11-28 DOI: 10.1101/2023.11.28.569057\nExcitatory neurotransmission is principally mediated by AMPA-subtype ionotropic glutamate receptors (AMPARs). Dysregulation of AMPARs is the cause of many neurological disorders and how therapeutic candidates such as negative allosteric modulators inhibit AMPARs is unclear. Here, we show that non-competitive inhibition desensitizes AMPARs to activation and prevents positive allosteric modulation. We dissected the noncompetitive inhibition mechanism of action by capturing AMPARs bound to glutamate and the prototypical negative allosteric modulator, GYKI-52466, with cryo-electron microscopy. Noncompetitive inhibition by GYKI-52466, which binds in the transmembrane collar region surrounding the ion channel, negatively modulates AMPARs by decoupling glutamate binding in the ligand binding domain from the ion channel. Furthermore, during allosteric competition between negative and positive modulators, negative allosteric modulation by GKYI-52466 outcompetes positive allosteric modulators to control AMPAR function. Our data provide a new framework for understanding allostery of AMPARs and foundations for rational design of therapeutics targeting AMPARs in neurological diseases.\nStructure of Transmembrane AMPA Receptor Regulatory Protein Subunit Î³2\nAuthors: Hale, W. D.; Romero, A. M.; Huganir, R. L.; Twomey, E. C.\nScore: 15.8, Published: 2023-11-28 DOI: 10.1101/2023.11.28.569079\nTransmembrane AMPA receptor regulatory proteins (TARPs) are claudin-like proteins that tightly regulate AMPA receptors (AMPARs) and are fundamental for excitatory neurotransmission. We used cryo-electron microscopy (cryo-EM) to reconstruct the 36 kDa TARP subunit {gamma}2 to 2.3 [A] and reveal the structural diversity of TARPs. Our data reveals critical motifs that distinguish TARPs from claudins and define how sequence variations within TARPs differentiate subfamilies and their regulation of AMPARs.\nCell-type-specific 3D-genome organization and transcription regulation in the brain\nAuthors: Liu, S.; Zheng, P.; Wang, C. Y.; Jia, B. B.; Zemke, N. R.; Ren, B.; Zhuang, X.\nScore: 9.2, Published: 2023-12-05 DOI: 10.1101/2023.12.04.570024\n3D organization of the genome plays a critical role in regulating gene expression. However, it remains unclear how chromatin organization differs among different cell types in the brain. Here we used genome-scale DNA and RNA imaging to investigate 3D-genome organization in transcriptionally distinct cell types in the primary motor cortex of the mouse brain. We uncovered a wide spectrum of differences in the nuclear architecture and 3D-genome organization among different cell types, ranging from the physical size of the cell nucleus to the active-inactive chromatin compartmentalization and radial positioning of chromatin loci within the nucleus. These cell-type-dependent variations in nuclear architecture and chromatin organization exhibited strong correlation with both total transcriptional activity of the cell and transcriptional regulation of cell-type-specific marker genes. Moreover, we found that the methylated-DNA-binding protein MeCP2 regulates transcription in a divergent manner, depending on the nuclear radial positions of chromatin loci, through modulating active-inactive chromatin compartmentalization.\nUnbiased MD simulations characterize lipid binding pockets in lipid transfer proteins\nAuthors: Vanni, S.; Srinivasan, S.; Alvarez Lorenzo, D.\nScore: 10.8, Published: 2023-11-24 DOI: 10.1101/2023.11.24.568550\nThe molecular characterization of lipid binding to lipid transfer proteins (LTPs) is fundamental to the understanding of several aspects of their mechanistic mode of action. However, obtaining lipid-bound structures of LTPs is not straightforward owing to caveats in current experimental structural biology approaches. As a result, several structures of LTPs, and most notably almost all of those that have been proposed to act as bridges between membrane organelles, do not provide the precise location of their endogenous lipid ligands. To address this limitation, computational approaches are a powerful alternative methodology, but they are often limited by the high flexibility of lipid substrates. In this work, we develop an in silico protocol based on unbiased coarse grain molecular simulations in which lipids placed in bulk solvent away from the protein can spontaneously bind to LTPs. This approach accurately determines binding pockets in LTPs and provides a working hypothesis for the pathway via which lipids enter LTPs. We apply this approach to characterize lipid binding to bridge-like LTPs belonging to the Vps13-Atg2 family, for which the lipid localization inside the protein is currently unknown. Overall, our work paves the way to determine binding pockets and entry pathways for several LTPs in an inexpensive, fast, and accurate manner.\nPower-law growth models explain incidences and sizes of pancreatic cancer precursor lesions and confirm spatial genomic findings\nAuthors: Kiemen, A. L.; Wu, P.; Braxton, A. M.; Cornish, T. C.; Hruban, R. H.; Wood, L. D.; Wirtz, D.; Zwicker, D.\nScore: 5.8, Published: 2023-12-04 DOI: 10.1101/2023.12.01.569633\nPancreatic ductal adenocarcinoma is a rare but lethal cancer. Recent evidence reveals that pancreatic intraepithelial neoplasms (PanINs), the microscopic precursor lesions in the pancreatic ducts that can give rise to invasive pancreatic cancer, are significantly larger and more prevalent than previously believed. Better understanding of the growth law dynamics of PanINs may improve our ability to understand how a miniscule fraction of these lesions makes the transition to invasive cancer. Here, using artificial intelligence (AI)-based three-dimensional (3D) tissue mapping method, we measured the volumes of \u003e1,000 PanIN and found that lesion size is distributed according to a power law with a fitted exponent of -1.7 over \u003e 3 orders of magnitude. Our data also suggest that PanIN growth is not very sensitive to the pancreatic microenvironment or an individual's age, family history, and lifestyle, and is rather shaped by general growth behavior. We analyze several models of PanIN growth and fit the predicted size distributions to the observed data. The best fitting models suggest that both intraductal spread of PanIN lesions and fusing of multiple lesions into large, highly branched structures drive PanIN growth patterns. This work lays the groundwork for future mathematical modeling efforts integrating PanIN incidence, morphology, genomic, and transcriptomic features to understand pancreas tumorigenesis, and demonstrates the utility of combining experimental measurement of human tissues with dynamic modeling for understanding cancer tumorigenesis.\nPreserving condensate structure and composition by lowering sequence complexity\nAuthors: Sood, A.; Zhang, B.\nScore: 5.7, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569249\nBiological condensates play a vital role in organizing cellular chemistry. They selectively partition biomolecules, preventing unwanted cross-talk and buffering against chemical noise. Intrinsically disordered proteins (IDPs) serve as primary components of these condensates due to their flexibility and ability to engage in multivalent, nonspecific interactions, leading to spontaneous aggregation. Theoretical advancements are critical at connecting IDP sequences with condensate emergent properties to establish the so-called molecular grammar. We proposed an extension to the stickers and spacers model, incorporating non-specific pairwise interactions between spacers alongside specific interactions among stickers. Our investigation revealed that while spacer interactions contribute to phase separation and co-condensation, their non-specific nature leads to disorganized condensates. Specific sticker-sticker interactions drive the formation of condensates with well-defined structures and molecular composition. We discussed how evolutionary pressures might emerge to affect these interactions, leading to the prevalence of low complexity domains in IDP sequences. These domains suppress spurious interactions and facilitate the formation of biologically meaningful condensates. Significance StatementBiomolecular condensates serve as pivotal mechanisms in cellular organization, often characterized by an abundance of intrinsically disordered proteins (IDPs) that undergo frequent mutations in their sequences. Despite this, IDP sequences exhibit non-random patterns, yet the precise relationship between these sequences and the emergent properties of condensates remains unclear. To address this gap, we propose a molecular theory that delineates how various sequence features of IDPs contribute to the organization and composition of condensates. This theory not only sheds light on the evolution of IDPs but also elucidates the emergence of non-random sequence patterns as essential elements for the formation of functional condensates. Correspondingly, we posit that the prevalence of low-complexity regions within IDPs is a result of evolutionary selection.\nMechanoProDB: A Web Based Database for Exploring the Mechanical Properties of Proteins\nAuthors: Mesbah, I.; Habermann, B. H.; Rico, F.\nScore: 5.0, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569409\nThe mechanical stability of proteins is crucial for biological processes. To understand the mechanical functions of proteins, it is important to know the protein structure and mechanical properties. Protein mechanics is usually investigated through force spectroscopy experiments and simulations that probe the forces required to unfold the protein of interest. While there is a wealth of data in the literature on force spectroscopy experiments and steered molecular dynamics simulations of forced protein unfolding, this information is spread and difficult to access by non-experts. Here we introduce MechanoProDB, a novel web-based database resource for collecting and mining data obtained from experimental and computational works. MechanoProDB provides a curated repository for a wide range of proteins, including muscle proteins, adhesion molecules and membrane proteins. The database incorporates relevant parameters that provide insights into the mechanical stability of proteins and their conformational stability such as the unfolding forces, energy landscape parameters and contour lengths of unfolding steps. Additionally, it provides intuitive annotations of the unfolding pathways of each protein, allowing users to explore the individual steps during mechanical unfolding. The user-friendly interface of MechanoProBD allows researchers to efficiently navigate, search and download data pertaining to specific protein folds or experimental conditions. Users can visualize protein structures using interactive tools integrated within the database, such as Mol*, and plot available data through integrated plotting tools. To ensure data quality and reliability, we have carefully manually verified and curated the data currently available on MechanoProDB. Furthermore, the database also features an interface that enables users to contribute new data and annotations, promoting community-driven comprehensiveness. The freely available MechanoProDB aims to streamline and accelerate research in the field of mechanobiology and biophysics by offering a unique platform for data sharing and analysis. MechanoProDB is freely available at https://mechanoprodb.ibdm.univ-amu.fr.\nThe Sec61/TRAP Translocon Scrambles Lipids\nAuthors: Javanainen, M.; Karki, S.; Tranter, D.; Biriukov, D.; Paavilainen, V. O.\nScore: 6.6, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568215\nCell growth relies on the rapid flip-flop of newly synthesized lipids across the ER membrane. This process is facilitated without the need for ATP by specific membrane proteins--scramblases--a few of which have been very recently identified in the ER. We have previously resolved the structure of the translocon-associated protein (TRAP) bound to the Sec61 translocon in the ER membrane, and found this complex to render the membrane locally thinner. Moreover, Sec61 and TRAP each contain a crevice rich in polar residues that can shield a lipid head group as it traverses the hydrophobic membrane environment. We thus hypothesized that both Sec61 and TRAP act as ER scramblases. Here, we characterized the scrambling activity of Sec61 and TRAP using extensive molecular dynamics simulations. We observed that both Sec61 and TRAP efficiently scramble lipids via a credit card mechanism. We analyzed the kinetics and thermodynamics of lipid scrambling and demonstrated that local membrane thinning provides a key contribution to scrambling efficiency. Both proteins appear seemingly selective towards phosphatidylcholine lipids over phosphatidylethanolamine and phosphatidylserine, yet this behavior rather reflects the trends observed for these lipids in a protein-free membrane. The identified scrambling pathway in Sec61 structure is physiologically rarely unoccupied due to its role in protein translocation. Furthermore, we found that the scrambling activity of this pathway might be impeded by the presence of ions at a physiological concentration. However, the trimeric bundle of TRAP{beta}, TRAP{gamma}, and TRAP{delta} might provide scrambling activity insensitive to the functional state of the translocon and the solvent conditions.\nTau P301L mutation promotes core 4R tauopathy fibril fold through near-surface water structuring and conformational rearrangement\nAuthors: Vigers, M.; Lobo, S.; Najafi, S.; Dubose, A.; Tsay, K.; Ganguly, P.; Longhini, A. P.; Jin, Y.; Buratto, S. K.; Kosik, K. S.; Shell, M. S.; Shea, J.-E.; Han, S.\nScore: 5.0, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568818\nTau forms toxic fibrillar aggregates in a family of neurodegenerative diseases known as tauopathies. The faithful replication of tauopathy-specific fibril structures is a critical gap for developing diagnostic and therapeutic tools. This study debuts a strategy of identifying a critical segment of tau that forms a folding motif that is characteristic of a family of tauopathies and isolating it as a standalone peptide that form seeding-competent fibrils. The 19-residue jR2R3 peptide (295-313) spanning the R2/R3 splice junction of tau, in the presence of P301L, forms seeding-competent amyloid fibrils. This tau fragment contains the hydrophobic VQIVYK hexapeptide that is part of the core of every pathological tau fibril structure solved to-date and an intramolecular counter-strand that stabilizes the strand-loop-strand (SLS) motif observed in 4R tauopathy fibrils. This study shows that P301L exhibits a duality of effects: it lowers the barrier for the peptide to adopt aggregation-prone conformations and enhances the local structuring of water around the mutation site that facilitates site-specific dewetting and in-register stacking of tau to form cross {beta}-sheets. We solve a 3 [A] cryo-EM structure of jR2R3-P301L fibrils with a pseudo 21 screw symmetry in which each half of the fibrils cross-section contains two jR2R3-P301L peptides. One chain adopts a SLS fold found in 4R tauopathies that is stabilized by a second chain wrapping around the SLS fold, reminiscent of the 3-fold and 4-fold structures observed in 4R tauopathies. These jR2R3-P301L fibrils are able to template full length tau in a prion-like fashion. Significance StatementThis study presents a first step towards designing a tauopathy specific aggregation pathway by engineering a minimal tau prion building block, jR2R3, that can template and propagate distinct disease folds. We present the discovery that P301L--among the widest used mutations in cell and animal models of Alzheimers Disease--destabilizes an aggregation-prohibiting internal hairpin and enhances the local surface water structure that serves as an entropic hotspot to exert a hyper-localized effect in jR2R3. Our study suggests that P301L may be a more suitable mutation to include in modeling 4R tauopathies than for modelling Alzheimers Disease, and that mutations are powerful tools for the purpose of designing of tau prion models as therapeutic tools.\nEstimation of the mass density of biological matter from refractive index measurements\nAuthors: Moeckel, C.; Beck, T.; Kaliman, S.; Abuhattum, S.; Kim, K.; Kolb, J.; Wehner, D.; Zaburdaev, V.; Guck, J.\nScore: 4.3, Published: 2023-12-05 DOI: 10.1101/2023.12.05.569868\nThe quantification of physical properties of biological matter gives rise to novel ways of understanding functional mechanisms by utilizing models that explicitly depend on physical observables. One of the basic biophysical properties is the mass density (MD), which determines the degree of crowdedness. It impacts the dynamics in sub-cellular compartments and further plays a major role in defining the opto-acoustical properties of cells and tissues. As such, the MD can be connected to the refractive index (RI) via the well known Lorentz-Lorenz relation, which takes into account the polarizability of matter. However, computing the MD based on RI measurements poses a challenge as it requires detailed knowledge of the biochemical composition of the sample. Here we propose a methodology on how to account for a priori and a posteriori assumptions about the biochemical composition of the sample as well as respective RI measurements. To that aim, we employ the Biot mixing rule of RIs alongside the assumption of volume additivity to find an approximate relation of MD and RI. We use Monte-Carlo simulations as well as Gaussian propagation of uncertainty to obtain approximate analytical solutions for the respective uncertainties of MD and RI. We validate this approach by applying it to a set of well characterized complex mixtures given by bovine milk and intralipid emulsion. Further, we employ it to estimate the mass density of trunk tissue of living zebrafish Danio rerio larvae. Our results enable quantifying changes of mass density estimates based on variations in the a priori assumptions. This illustrates the importance of implementing this methodology not only for MD estimations but for many other related biophysical problems, such as mechanical measurements using Brillouin microscopy and transient optical coherence elastography.\n",
  "wordCount" : "2581",
  "inLanguage": "en",
  "datePublished": "2023-12-06T10:40:04Z",
  "dateModified": "2023-12-06T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 6, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569057">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569057" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569057">
        <p class="paperTitle">Allosteric Competition and Inhibition in AMPA Receptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569057" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569057" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hale, W. D.; Romero, A. M.; Gonzalez, C. U.; Jayaraman, V.; Lau, A. Y.; Huganir, R. L.; Twomey, E. C.</p>
        <p class="info">Score: 17.8, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569057' target='https://doi.org/10.1101/2023.11.28.569057'> 10.1101/2023.11.28.569057</a></p>
        <p class="abstract">Excitatory neurotransmission is principally mediated by AMPA-subtype ionotropic glutamate receptors (AMPARs). Dysregulation of AMPARs is the cause of many neurological disorders and how therapeutic candidates such as negative allosteric modulators inhibit AMPARs is unclear. Here, we show that non-competitive inhibition desensitizes AMPARs to activation and prevents positive allosteric modulation. We dissected the noncompetitive inhibition mechanism of action by capturing AMPARs bound to glutamate and the prototypical negative allosteric modulator, GYKI-52466, with cryo-electron microscopy. Noncompetitive inhibition by GYKI-52466, which binds in the transmembrane collar region surrounding the ion channel, negatively modulates AMPARs by decoupling glutamate binding in the ligand binding domain from the ion channel. Furthermore, during allosteric competition between negative and positive modulators, negative allosteric modulation by GKYI-52466 outcompetes positive allosteric modulators to control AMPAR function. Our data provide a new framework for understanding allostery of AMPARs and foundations for rational design of therapeutics targeting AMPARs in neurological diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569079">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569079" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569079">
        <p class="paperTitle">Structure of Transmembrane AMPA Receptor Regulatory Protein Subunit Î³2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569079" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569079" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hale, W. D.; Romero, A. M.; Huganir, R. L.; Twomey, E. C.</p>
        <p class="info">Score: 15.8, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569079' target='https://doi.org/10.1101/2023.11.28.569079'> 10.1101/2023.11.28.569079</a></p>
        <p class="abstract">Transmembrane AMPA receptor regulatory proteins (TARPs) are claudin-like proteins that tightly regulate AMPA receptors (AMPARs) and are fundamental for excitatory neurotransmission. We used cryo-electron microscopy (cryo-EM) to reconstruct the 36 kDa TARP subunit {gamma}2 to 2.3 [A] and reveal the structural diversity of TARPs. Our data reveals critical motifs that distinguish TARPs from claudins and define how sequence variations within TARPs differentiate subfamilies and their regulation of AMPARs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.570024">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.570024" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.570024">
        <p class="paperTitle">Cell-type-specific 3D-genome organization and transcription regulation in the brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.570024" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.570024" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, S.; Zheng, P.; Wang, C. Y.; Jia, B. B.; Zemke, N. R.; Ren, B.; Zhuang, X.</p>
        <p class="info">Score: 9.2, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.570024' target='https://doi.org/10.1101/2023.12.04.570024'> 10.1101/2023.12.04.570024</a></p>
        <p class="abstract">3D organization of the genome plays a critical role in regulating gene expression. However, it remains unclear how chromatin organization differs among different cell types in the brain. Here we used genome-scale DNA and RNA imaging to investigate 3D-genome organization in transcriptionally distinct cell types in the primary motor cortex of the mouse brain. We uncovered a wide spectrum of differences in the nuclear architecture and 3D-genome organization among different cell types, ranging from the physical size of the cell nucleus to the active-inactive chromatin compartmentalization and radial positioning of chromatin loci within the nucleus. These cell-type-dependent variations in nuclear architecture and chromatin organization exhibited strong correlation with both total transcriptional activity of the cell and transcriptional regulation of cell-type-specific marker genes. Moreover, we found that the methylated-DNA-binding protein MeCP2 regulates transcription in a divergent manner, depending on the nuclear radial positions of chromatin loci, through modulating active-inactive chromatin compartmentalization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568550">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568550" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568550">
        <p class="paperTitle">Unbiased MD simulations characterize lipid binding pockets in lipid transfer proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568550" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568550" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vanni, S.; Srinivasan, S.; Alvarez Lorenzo, D.</p>
        <p class="info">Score: 10.8, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568550' target='https://doi.org/10.1101/2023.11.24.568550'> 10.1101/2023.11.24.568550</a></p>
        <p class="abstract">The molecular characterization of lipid binding to lipid transfer proteins (LTPs) is fundamental to the understanding of several aspects of their mechanistic mode of action. However, obtaining lipid-bound structures of LTPs is not straightforward owing to caveats in current experimental structural biology approaches. As a result, several structures of LTPs, and most notably almost all of those that have been proposed to act as bridges between membrane organelles, do not provide the precise location of their endogenous lipid ligands. To address this limitation, computational approaches are a powerful alternative methodology, but they are often limited by the high flexibility of lipid substrates. In this work, we develop an in silico protocol based on unbiased coarse grain molecular simulations in which lipids placed in bulk solvent away from the protein can spontaneously bind to LTPs. This approach accurately determines binding pockets in LTPs and provides a working hypothesis for the pathway via which lipids enter LTPs. We apply this approach to characterize lipid binding to bridge-like LTPs belonging to the Vps13-Atg2 family, for which the lipid localization inside the protein is currently unknown. Overall, our work paves the way to determine binding pockets and entry pathways for several LTPs in an inexpensive, fast, and accurate manner.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569633">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569633" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569633">
        <p class="paperTitle">Power-law growth models explain incidences and sizes of pancreatic cancer precursor lesions and confirm spatial genomic findings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569633" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569633" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kiemen, A. L.; Wu, P.; Braxton, A. M.; Cornish, T. C.; Hruban, R. H.; Wood, L. D.; Wirtz, D.; Zwicker, D.</p>
        <p class="info">Score: 5.8, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569633' target='https://doi.org/10.1101/2023.12.01.569633'> 10.1101/2023.12.01.569633</a></p>
        <p class="abstract">Pancreatic ductal adenocarcinoma is a rare but lethal cancer. Recent evidence reveals that pancreatic intraepithelial neoplasms (PanINs), the microscopic precursor lesions in the pancreatic ducts that can give rise to invasive pancreatic cancer, are significantly larger and more prevalent than previously believed. Better understanding of the growth law dynamics of PanINs may improve our ability to understand how a miniscule fraction of these lesions makes the transition to invasive cancer. Here, using artificial intelligence (AI)-based three-dimensional (3D) tissue mapping method, we measured the volumes of &gt;1,000 PanIN and found that lesion size is distributed according to a power law with a fitted exponent of -1.7 over &gt; 3 orders of magnitude. Our data also suggest that PanIN growth is not very sensitive to the pancreatic microenvironment or an individual&#39;s age, family history, and lifestyle, and is rather shaped by general growth behavior. We analyze several models of PanIN growth and fit the predicted size distributions to the observed data. The best fitting models suggest that both intraductal spread of PanIN lesions and fusing of multiple lesions into large, highly branched structures drive PanIN growth patterns. This work lays the groundwork for future mathematical modeling efforts integrating PanIN incidence, morphology, genomic, and transcriptomic features to understand pancreas tumorigenesis, and demonstrates the utility of combining experimental measurement of human tissues with dynamic modeling for understanding cancer tumorigenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569249">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569249" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569249">
        <p class="paperTitle">Preserving condensate structure and composition by lowering sequence complexity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569249" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569249" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sood, A.; Zhang, B.</p>
        <p class="info">Score: 5.7, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569249' target='https://doi.org/10.1101/2023.11.29.569249'> 10.1101/2023.11.29.569249</a></p>
        <p class="abstract">Biological condensates play a vital role in organizing cellular chemistry. They selectively partition biomolecules, preventing unwanted cross-talk and buffering against chemical noise. Intrinsically disordered proteins (IDPs) serve as primary components of these condensates due to their flexibility and ability to engage in multivalent, nonspecific interactions, leading to spontaneous aggregation. Theoretical advancements are critical at connecting IDP sequences with condensate emergent properties to establish the so-called molecular grammar. We proposed an extension to the stickers and spacers model, incorporating non-specific pairwise interactions between spacers alongside specific interactions among stickers. Our investigation revealed that while spacer interactions contribute to phase separation and co-condensation, their non-specific nature leads to disorganized condensates. Specific sticker-sticker interactions drive the formation of condensates with well-defined structures and molecular composition. We discussed how evolutionary pressures might emerge to affect these interactions, leading to the prevalence of low complexity domains in IDP sequences. These domains suppress spurious interactions and facilitate the formation of biologically meaningful condensates.

Significance StatementBiomolecular condensates serve as pivotal mechanisms in cellular organization, often characterized by an abundance of intrinsically disordered proteins (IDPs) that undergo frequent mutations in their sequences. Despite this, IDP sequences exhibit non-random patterns, yet the precise relationship between these sequences and the emergent properties of condensates remains unclear. To address this gap, we propose a molecular theory that delineates how various sequence features of IDPs contribute to the organization and composition of condensates. This theory not only sheds light on the evolution of IDPs but also elucidates the emergence of non-random sequence patterns as essential elements for the formation of functional condensates. Correspondingly, we posit that the prevalence of low-complexity regions within IDPs is a result of evolutionary selection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569409">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569409" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569409">
        <p class="paperTitle">MechanoProDB: A Web Based Database for Exploring the Mechanical Properties of Proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569409" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569409" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mesbah, I.; Habermann, B. H.; Rico, F.</p>
        <p class="info">Score: 5.0, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569409' target='https://doi.org/10.1101/2023.11.30.569409'> 10.1101/2023.11.30.569409</a></p>
        <p class="abstract">The mechanical stability of proteins is crucial for biological processes. To understand the mechanical functions of proteins, it is important to know the protein structure and mechanical properties. Protein mechanics is usually investigated through force spectroscopy experiments and simulations that probe the forces required to unfold the protein of interest. While there is a wealth of data in the literature on force spectroscopy experiments and steered molecular dynamics simulations of forced protein unfolding, this information is spread and difficult to access by non-experts. Here we introduce MechanoProDB, a novel web-based database resource for collecting and mining data obtained from experimental and computational works. MechanoProDB provides a curated repository for a wide range of proteins, including muscle proteins, adhesion molecules and membrane proteins. The database incorporates relevant parameters that provide insights into the mechanical stability of proteins and their conformational stability such as the unfolding forces, energy landscape parameters and contour lengths of unfolding steps. Additionally, it provides intuitive annotations of the unfolding pathways of each protein, allowing users to explore the individual steps during mechanical unfolding. The user-friendly interface of MechanoProBD allows researchers to efficiently navigate, search and download data pertaining to specific protein folds or experimental conditions. Users can visualize protein structures using interactive tools integrated within the database, such as Mol*, and plot available data through integrated plotting tools. To ensure data quality and reliability, we have carefully manually verified and curated the data currently available on MechanoProDB. Furthermore, the database also features an interface that enables users to contribute new data and annotations, promoting community-driven comprehensiveness. The freely available MechanoProDB aims to streamline and accelerate research in the field of mechanobiology and biophysics by offering a unique platform for data sharing and analysis. MechanoProDB is freely available at https://mechanoprodb.ibdm.univ-amu.fr.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568215">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568215" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568215">
        <p class="paperTitle">The Sec61/TRAP Translocon Scrambles Lipids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568215" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568215" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Javanainen, M.; Karki, S.; Tranter, D.; Biriukov, D.; Paavilainen, V. O.</p>
        <p class="info">Score: 6.6, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568215' target='https://doi.org/10.1101/2023.11.23.568215'> 10.1101/2023.11.23.568215</a></p>
        <p class="abstract">Cell growth relies on the rapid flip-flop of newly synthesized lipids across the ER membrane. This process is facilitated without the need for ATP by specific membrane proteins--scramblases--a few of which have been very recently identified in the ER. We have previously resolved the structure of the translocon-associated protein (TRAP) bound to the Sec61 translocon in the ER membrane, and found this complex to render the membrane locally thinner. Moreover, Sec61 and TRAP each contain a crevice rich in polar residues that can shield a lipid head group as it traverses the hydrophobic membrane environment. We thus hypothesized that both Sec61 and TRAP act as ER scramblases. Here, we characterized the scrambling activity of Sec61 and TRAP using extensive molecular dynamics simulations. We observed that both Sec61 and TRAP efficiently scramble lipids via a credit card mechanism. We analyzed the kinetics and thermodynamics of lipid scrambling and demonstrated that local membrane thinning provides a key contribution to scrambling efficiency. Both proteins appear seemingly selective towards phosphatidylcholine lipids over phosphatidylethanolamine and phosphatidylserine, yet this behavior rather reflects the trends observed for these lipids in a protein-free membrane. The identified scrambling pathway in Sec61 structure is physiologically rarely unoccupied due to its role in protein translocation. Furthermore, we found that the scrambling activity of this pathway might be impeded by the presence of ions at a physiological concentration. However, the trimeric bundle of TRAP{beta}, TRAP{gamma}, and TRAP{delta} might provide scrambling activity insensitive to the functional state of the translocon and the solvent conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568818">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568818" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568818">
        <p class="paperTitle">Tau P301L mutation promotes core 4R tauopathy fibril fold through near-surface water structuring and conformational rearrangement</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568818" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568818" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vigers, M.; Lobo, S.; Najafi, S.; Dubose, A.; Tsay, K.; Ganguly, P.; Longhini, A. P.; Jin, Y.; Buratto, S. K.; Kosik, K. S.; Shell, M. S.; Shea, J.-E.; Han, S.</p>
        <p class="info">Score: 5.0, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568818' target='https://doi.org/10.1101/2023.11.28.568818'> 10.1101/2023.11.28.568818</a></p>
        <p class="abstract">Tau forms toxic fibrillar aggregates in a family of neurodegenerative diseases known as tauopathies. The faithful replication of tauopathy-specific fibril structures is a critical gap for developing diagnostic and therapeutic tools. This study debuts a strategy of identifying a critical segment of tau that forms a folding motif that is characteristic of a family of tauopathies and isolating it as a standalone peptide that form seeding-competent fibrils. The 19-residue jR2R3 peptide (295-313) spanning the R2/R3 splice junction of tau, in the presence of P301L, forms seeding-competent amyloid fibrils. This tau fragment contains the hydrophobic VQIVYK hexapeptide that is part of the core of every pathological tau fibril structure solved to-date and an intramolecular counter-strand that stabilizes the strand-loop-strand (SLS) motif observed in 4R tauopathy fibrils. This study shows that P301L exhibits a duality of effects: it lowers the barrier for the peptide to adopt aggregation-prone conformations and enhances the local structuring of water around the mutation site that facilitates site-specific dewetting and in-register stacking of tau to form cross {beta}-sheets. We solve a 3 [A] cryo-EM structure of jR2R3-P301L fibrils with a pseudo 21 screw symmetry in which each half of the fibrils cross-section contains two jR2R3-P301L peptides. One chain adopts a SLS fold found in 4R tauopathies that is stabilized by a second chain wrapping around the SLS fold, reminiscent of the 3-fold and 4-fold structures observed in 4R tauopathies. These jR2R3-P301L fibrils are able to template full length tau in a prion-like fashion.

Significance StatementThis study presents a first step towards designing a tauopathy specific aggregation pathway by engineering a minimal tau prion building block, jR2R3, that can template and propagate distinct disease folds. We present the discovery that P301L--among the widest used mutations in cell and animal models of Alzheimers Disease--destabilizes an aggregation-prohibiting internal hairpin and enhances the local surface water structure that serves as an entropic hotspot to exert a hyper-localized effect in jR2R3. Our study suggests that P301L may be a more suitable mutation to include in modeling 4R tauopathies than for modelling Alzheimers Disease, and that mutations are powerful tools for the purpose of designing of tau prion models as therapeutic tools.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.569868">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.569868" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.569868">
        <p class="paperTitle">Estimation of the mass density of biological matter from refractive index measurements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.569868" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.569868" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moeckel, C.; Beck, T.; Kaliman, S.; Abuhattum, S.; Kim, K.; Kolb, J.; Wehner, D.; Zaburdaev, V.; Guck, J.</p>
        <p class="info">Score: 4.3, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.569868' target='https://doi.org/10.1101/2023.12.05.569868'> 10.1101/2023.12.05.569868</a></p>
        <p class="abstract">The quantification of physical properties of biological matter gives rise to novel ways of understanding functional mechanisms by utilizing models that explicitly depend on physical observables. One of the basic biophysical properties is the mass density (MD), which determines the degree of crowdedness. It impacts the dynamics in sub-cellular compartments and further plays a major role in defining the opto-acoustical properties of cells and tissues. As such, the MD can be connected to the refractive index (RI) via the well known Lorentz-Lorenz relation, which takes into account the polarizability of matter. However, computing the MD based on RI measurements poses a challenge as it requires detailed knowledge of the biochemical composition of the sample. Here we propose a methodology on how to account for a priori and a posteriori assumptions about the biochemical composition of the sample as well as respective RI measurements. To that aim, we employ the Biot mixing rule of RIs alongside the assumption of volume additivity to find an approximate relation of MD and RI. We use Monte-Carlo simulations as well as Gaussian propagation of uncertainty to obtain approximate analytical solutions for the respective uncertainties of MD and RI. We validate this approach by applying it to a set of well characterized complex mixtures given by bovine milk and intralipid emulsion. Further, we employ it to estimate the mass density of trunk tissue of living zebrafish Danio rerio larvae. Our results enable quantifying changes of mass density estimates based on variations in the a priori assumptions. This illustrates the importance of implementing this methodology not only for MD estimations but for many other related biophysical problems, such as mechanical measurements using Brillouin microscopy and transient optical coherence elastography.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
