<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states
Authors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.
Score: 27.1, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575649
Cell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states
Authors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.
Score: 27.1, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575649
Cell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states
Authors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.
Score: 27.1, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575649
Cell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states\nAuthors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.\nScore: 27.1, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575649\nCell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies.",
  "keywords": [
    
  ],
  "articleBody": " High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states\nAuthors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.\nScore: 27.1, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575649\nCell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies. Here, we describe a cytometry-based method with single-cell resolution which fills this technological gap by combining biophysical profiling with conventional biomarker analysis. This platform allows to reveal notable cell type-dependent remodelling of membrane fluidity during immune stimulations and in diseases. Using immune cells exposed to tumour microenvironment as well as from long COVID and chronic lymphocyte leukaemia patients, we demonstrate that membrane fluidity is orthogonal to surface marker expression. Moreover, this biophysical parameter identifies new functional and pathological states of immune cells previously undetected via surface marker profiling alone. Our findings will contribute to a more precise definition of immune cell states based on their biophysical properties and will pave the way for a better understanding of the functional heterogeneity of immune cells.\nHigh-Throughput DNA melt measurements enable improved models of DNA folding thermodynamics\nAuthors: Ke, Y.; Sharma, E.; Wayment-Steele, H. K.; Becker, W. R.; Ho, A.; Marklund, E.; Greenleaf, W. J.\nScore: 16.5, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574731\nDNA folding thermodynamics are central to many biological processes and biotechnological applications involving base-pairing. Current methods for predicting stability from DNA sequence use nearest-neighbor (NN) models that struggle to accurately capture the diverse sequence-dependency of elements other than Watson-Crick base pairs, likely due to insufficient experimental data. We introduce a massively parallel method-Array Melt-that uses fluorescence-based quenching signals to measure equilibrium stability of millions of DNA hairpins simultaneously on a repurposed Illumina sequencing flow cell. By leveraging this dataset of 27,732 sequences with two-state melting behavior, we derived a refined NUPACK-compatible NN model, a richer parameterization NN model that exhibits higher accuracy, and a graph neural network (GNN) model that identifies relevant interactions within DNA beyond nearest neighbors. All models provide improved accuracy in predicting DNA folding thermodynamics, providing improvements relevant for in silico design of qPCR primers, oligo hybridization probes, and DNA origami.\nUser-friendly Oblique Plane Microscopy on a fully functional commercially available microscope base\nAuthors: Sirinakis, G.; Allgeyer, E. S.; Nashchekin, D.; St Johnston, D.\nScore: 12.9, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574832\nIn this work we present an Oblique Plane Microscope designed to work seamlessly with a commercially available microscope base. To support all the functionality offered by the microscope base, where the position of the objective lens is not fixed, we adopted a two-mirror scanning geometry that can compensate for changes to the position of the objective lens during routine microscope operation. We showed that within the expected displacement range of the 100X, 1.35 NA objective lens away from its designed position, and for most practical applications, there is no significant effect on the resolving power, or the fidelity of the 3D data produced by the microscope. Compared to the more traditional scan-lens/galvo-mirror combination, the two-mirror scanning geometry offers higher light-efficiency and a more compact footprint, which could be beneficial to all OPM designs regardless of the use of a commercial base or not.\nSingle molecule delivery into living cells\nAuthors: Chau, C.; Maffeo, C.; Aksimentiev, A.; Radford, S.; Hewitt, E.; Actis, P.\nScore: 12.6, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574652\nControlled manipulation of cultured cells by delivery of exogenous macromolecules is a cornerstone of experimental biology. Here we describe a platform which uses nanopipettes to deliver defined numbers of macromolecules into cultured cell lines and primary cells at single molecule resolution. In the nanoinjection platform the nanopipette is used as both a scanning ion conductance microscope (SICM) scanning probe and as an injection probe. The SICM is used to position the nanopipette above the cell surface, before the nanopipette is inserted into the cell into a defined location and to a predefined depth. We demonstrate that the nanoinjection platform enables the quantitative delivery of DNA, globular proteins and protein fibrils into cells with single molecule resolution and that delivery results in a phenotypic change in the cell that depends on the identity of the molecules introduced. Using experiments and computational modelling, we also show that macromolecular crowding in the cell increases the signal to noise ratio for the detection of translocation events, thus the cell itself enhances the detection of the molecules delivered.\nMeasuring concentration of nanoparticles in polydisperse mixtures using interferometric nanoparticle tracking analysis (iNTA)\nAuthors: Kashkanova, A. D.; Albrecht, D.; Kueppers, M.; Blessing, M.; Sandoghdar, V.\nScore: 10.6, Published: 2024-01-10 DOI: 10.1101/2024.01.09.574819\nQuantitative measurements of nanoparticle concentration in liquid suspensions are in high demand, for example, in the medical and food industries. Conventional methods remain unsatisfactory, especially for polydisperse samples with overlapping size ranges. Recently, we introduced interferometric nanoparticle tracking analysis (iNTA) as a new method for high-precision measurement of nanoparticle size and refractive index. Here, we show that by counting the number of trajectories that cross the focal plane, iNTA can measure concentrations of subpopulations in a polydisperse mixture in a quantitative manner and without the need for a calibration sample. We evaluate our method on both monodisperse samples and mixtures of known concentrations. Furthermore, we assess the concentration of SARS-CoV-2 in supernatant samples obtained from infected cells.\nAstigmatism-based focus stabilisation with universal objective lens compatibility, extended operating range and nanometre precision\nAuthors: Rahmani, A.; Cox, T.; Thamaravelil Abhimanue Achary, A.; Ponjavic, A.\nScore: 9.6, Published: 2024-01-15 DOI: 10.1101/2024.01.15.575442\nFocus stabilisation is vital for long-term fluorescence imaging, particularly in the case of high-resolution imaging techniques. Current stabilisation solutions either rely on fiducial markers that can be perturbative, or on beam reflection monitoring that is limited to high-numerical aperture objective lenses, making multimodal and large-scale imaging challenging. We introduce a beam-based method that relies on astigmatism, which offers advantages in terms of precision and the range over which focus stabilisation is effective. This approach is shown to be compatible with a wide range of objective lenses (10x-100x), typically achieving \u003c10 nm precision with \u003e10 um operating range. Notably, our technique is largely unaffected by pointing stability errors, which in combination with implementation through a standalone Raspberry Pi architecture, offers a versatile focus stabilisation unit that can be added onto most existing microscope setups.\nOn-chip large-scale-integration and 2D collective modes of genetically programmed artificial cells\nAuthors: Ricouvier, J.; Mostov, P.; Shabtai, O.; Tayar, A.; Karzbrun, E.; Khakimzhan, A.; Noireaux, V.; Shulman Daube, S.; Bar-Ziv, R.\nScore: 8.4, Published: 2024-01-12 DOI: 10.1101/2024.01.10.575012\nThe on-chip large-scale-integration of genetically programmed artificial cells capable of exhibiting collective modes is an important goal for fundamental research and technology. Here, we report assembly of a 2D layout of 1024 monolithic DNA compartments as artificial cells on a 5-millimeter square silicon chip. Homeostatic cell-free protein synthesis reactions driven by genetic circuits occur inside the compartments. We created a reaction-diffusion system with a 30x30 square lattice of artificial cells interconnected by thin capillaries for diffusion of products. Driving the system by a genetic oscillator revealed emergent collective modes of synchrony and propagating phase waves in 2D, with dynamics controlled by geometry. This demonstrates a class of nonequilibrium autonomous systems, where chemical energy consumed to make proteins induces 2D collective patterns of gene expression on multicellular scales, with applications in biological computing, sensing, and materials synthesis.\nLipid osmosis, membrane tension, and other mechanochemical driving forces of lipid flow\nAuthors: Zhang, Y.; Lin, C.\nScore: 9.3, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574656\nNonvesicular lipid transport among different membranes or membrane domains plays crucial roles in processes such as lipid homeostasis and organelle biogenesis. However, the forces that drive such lipid transport are not well understood. We propose that lipids tend to flow towards the membrane area with a higher membrane protein density in a process termed lipid osmosis. This process lowers the membrane tension in the area when the lipid flow reaches equilibrium. We examine the thermodynamic basis and experimental evidence of lipid osmosis and the corresponding osmotic membrane tension. We predict that lipid osmosis can drive lipid flows between different membrane regions through lipid transfer proteins, scramblases, or other similar barriers that selectively pass lipids but not membrane proteins. We also speculate on the biological functions of lipid osmosis. Finally, we explore other driving forces for lipid transfer and describe potential methods and systems to further test our theory.\nOblique Line Scan Illumination Enables Expansive, Accurate and Sensitive Single Protein Measurements in Solution and in Living Cells\nAuthors: Driouchi, A.; Bretan, M.; Davis, B. J.; Heckert, A. B.; Seeger, M.; Bradley Silva, M.; Forrest, W. S.; Hsiung, J.; Tan, J.; Yang, H.; McSwiggen, D. T.; Song, L.; Sule, A.; Abaie, B.; Conroy, B.; Elliott, L. A.; Gonzalez, E.; Ilkov, F.; Isaacs, J.; Labaria, G.; Lagana, M.; Larsen, D. D.; Margolin, B.; Nguyen, M. K.; Rine, J.; Tang, Y.; Vana, M.; Wilkey, A.; Zhang, Z.; Basham, S.; Ho, J. J.; Johnson, S.; Klammer, A. A.; Lin, K.; Darzacq, X.; Betzig, E.; Berman, R. T.; Anderson, D. J.\nScore: 14.1, Published: 2023-12-23 DOI: 10.1101/2023.12.21.571765\nSingle-molecule localization microscopy (SMLM) techniques, such as single-molecule tracking (SMT), enable in situ measurements in cells from which data-rich metrics can be extracted. SMT has been successfully applied to a variety of biological questions and model systems, aiming to unravel the spatiotemporal regulation of molecular mechanisms that govern protein function, downstream pathway effects, and cellular function. While powerful, SMLM often suffers from low throughput and illumination inhomogeneity, along with microscope and user-induced technical biases. Due to technical limitations in scaling SMLM techniques, a tradeoff between spatiotemporal resolution and throughput has been made historically, restricting broad application of these technologies. Here we address these limitations using Oblique Line Scan (OLS), a robust single-objective light-sheet based illumination and detection modality that achieves nanoscale spatial resolution and sub-millisecond temporal resolution across a 250 x 190 m field of view. We demonstrate OLS-enabled SMT on Halo-Tagged proteins in living cells capturing protein motion up to 14 m2 /s. By exploiting the adaptability of the acquisition frame rate and the improved rejection of out of focus light, we extend the utility of OLS beyond cellular compartments with in-solution SMT (isSMT) for single-molecule measurement of ligand-protein interactions and disruption of protein-protein interactions (PPI). We illustrate the versatility of OLS by showcasing two-color SMT, STORM, and single molecule fluorescence recovery after photobleaching (FRAP). OLS expands the range of SMLM applications and paves the way for robust, high-throughput single-molecule investigations of protein dynamics required for drug screening and systems biology studies, both in cells and in solution.\nMechanistic Insights into G-protein Activation via Phosphorylation Mediated Non-Canonical Pathway\nAuthors: Shewani, K.; Madhu, M. K.; Murarka, R. K.\nScore: 4.9, Published: 2024-01-15 DOI: 10.1101/2024.01.15.575647\nActivation of heterotrimeric G-proteins (G{beta}{gamma}) downstream to receptor tyrosine kinases (RTKs) is a well-established crosstalk between the signaling pathways mediated by G-protein coupled receptors (GPCRs) and RTKs. While GPCR serves as a guanine exchange factor (GEF) in the canonical activation of G that facilitates the exchange of GDP for GTP, the mechanism through which RTK phosphorylations induce G activation remains unclear. Recent experimental studies revealed that the epidermal growth factor receptor (EGFR), a well-known RTK, phosphorylates the helical domain tyrosine residues Y154 and Y155 and accelerates the GDP release from the Gi3, a subtype of G-protein. Using well-tempered metadynamics and extensive unbiased molecular dynamics simulations, we captured the GDP release event and identified the intermediates between bound and unbound states through Markov state models. The additional negative charges introduced by phosphorylations rewired the inter-residue interactions and significantly weakened the salt bridges at the domain interface, contributing to the increased separation of the Ras-like and helical domains of G-protein. Furthermore, the unfolding of helix F resulted in greater flexibility near the hinge region, facilitating a greater distance between domains in the phosphorylated Gi3. The release of GDP in the phosphorylated G-protein occurred at a faster rate compared to the unphosphorylated state, caused by increased fluctuations in conserved regions of P-loop, switch 1, and switch 2. Overall, this study provides atomistic insights into the activation of G-proteins induced by RTK phosphorylations and identifies the specific structural motifs involved in the process. The knowledge gained from the study could establish a foundation for targeting non-canonical signaling pathways and developing therapeutic strategies against the ailments associated with dysregulated G-protein signaling.\n",
  "wordCount" : "2070",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575649" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575649">
        <p class="paperTitle">High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.</p>
        <p class="info">Score: 27.1, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575649' target='https://doi.org/10.1101/2024.01.15.575649'> 10.1101/2024.01.15.575649</a></p>
        <p class="abstract">Cell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies. Here, we describe a cytometry-based method with single-cell resolution which fills this technological gap by combining biophysical profiling with conventional biomarker analysis. This platform allows to reveal notable cell type-dependent remodelling of membrane fluidity during immune stimulations and in diseases. Using immune cells exposed to tumour microenvironment as well as from long COVID and chronic lymphocyte leukaemia patients, we demonstrate that membrane fluidity is orthogonal to surface marker expression. Moreover, this biophysical parameter identifies new functional and pathological states of immune cells previously undetected via surface marker profiling alone. Our findings will contribute to a more precise definition of immune cell states based on their biophysical properties and will pave the way for a better understanding of the functional heterogeneity of immune cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574731">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574731" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574731">
        <p class="paperTitle">High-Throughput DNA melt measurements enable improved models of DNA folding thermodynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574731" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574731" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ke, Y.; Sharma, E.; Wayment-Steele, H. K.; Becker, W. R.; Ho, A.; Marklund, E.; Greenleaf, W. J.</p>
        <p class="info">Score: 16.5, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574731' target='https://doi.org/10.1101/2024.01.08.574731'> 10.1101/2024.01.08.574731</a></p>
        <p class="abstract">DNA folding thermodynamics are central to many biological processes and biotechnological applications involving base-pairing. Current methods for predicting stability from DNA sequence use nearest-neighbor (NN) models that struggle to accurately capture the diverse sequence-dependency of elements other than Watson-Crick base pairs, likely due to insufficient experimental data. We introduce a massively parallel method-Array Melt-that uses fluorescence-based quenching signals to measure equilibrium stability of millions of DNA hairpins simultaneously on a repurposed Illumina sequencing flow cell. By leveraging this dataset of 27,732 sequences with two-state melting behavior, we derived a refined NUPACK-compatible NN model, a richer parameterization NN model that exhibits higher accuracy, and a graph neural network (GNN) model that identifies relevant interactions within DNA beyond nearest neighbors. All models provide improved accuracy in predicting DNA folding thermodynamics, providing improvements relevant for in silico design of qPCR primers, oligo hybridization probes, and DNA origami.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574832">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574832" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574832">
        <p class="paperTitle">User-friendly Oblique Plane Microscopy on a fully functional commercially available microscope base</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574832" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574832" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sirinakis, G.; Allgeyer, E. S.; Nashchekin, D.; St Johnston, D.</p>
        <p class="info">Score: 12.9, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574832' target='https://doi.org/10.1101/2024.01.09.574832'> 10.1101/2024.01.09.574832</a></p>
        <p class="abstract">In this work we present an Oblique Plane Microscope designed to work seamlessly with a commercially available microscope base. To support all the functionality offered by the microscope base, where the position of the objective lens is not fixed, we adopted a two-mirror scanning geometry that can compensate for changes to the position of the objective lens during routine microscope operation. We showed that within the expected displacement range of the 100X, 1.35 NA objective lens away from its designed position, and for most practical applications, there is no significant effect on the resolving power, or the fidelity of the 3D data produced by the microscope. Compared to the more traditional scan-lens/galvo-mirror combination, the two-mirror scanning geometry offers higher light-efficiency and a more compact footprint, which could be beneficial to all OPM designs regardless of the use of a commercial base or not.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574652">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574652" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574652">
        <p class="paperTitle">Single molecule delivery into living cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574652" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574652" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chau, C.; Maffeo, C.; Aksimentiev, A.; Radford, S.; Hewitt, E.; Actis, P.</p>
        <p class="info">Score: 12.6, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574652' target='https://doi.org/10.1101/2024.01.09.574652'> 10.1101/2024.01.09.574652</a></p>
        <p class="abstract">Controlled manipulation of cultured cells by delivery of exogenous macromolecules is a cornerstone of experimental biology. Here we describe a platform which uses nanopipettes to deliver defined numbers of macromolecules into cultured cell lines and primary cells at single molecule resolution. In the nanoinjection platform the nanopipette is used as both a scanning ion conductance microscope (SICM) scanning probe and as an injection probe. The SICM is used to position the nanopipette above the cell surface, before the nanopipette is inserted into the cell into a defined location and to a predefined depth. We demonstrate that the nanoinjection platform enables the quantitative delivery of DNA, globular proteins and protein fibrils into cells with single molecule resolution and that delivery results in a phenotypic change in the cell that depends on the identity of the molecules introduced. Using experiments and computational modelling, we also show that macromolecular crowding in the cell increases the signal to noise ratio for the detection of translocation events, thus the cell itself enhances the detection of the molecules delivered.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574819">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574819" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574819">
        <p class="paperTitle">Measuring concentration of nanoparticles in polydisperse mixtures using interferometric nanoparticle tracking analysis (iNTA)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574819" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574819" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kashkanova, A. D.; Albrecht, D.; Kueppers, M.; Blessing, M.; Sandoghdar, V.</p>
        <p class="info">Score: 10.6, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574819' target='https://doi.org/10.1101/2024.01.09.574819'> 10.1101/2024.01.09.574819</a></p>
        <p class="abstract">Quantitative measurements of nanoparticle concentration in liquid suspensions are in high demand, for example, in the medical and food industries. Conventional methods remain unsatisfactory, especially for polydisperse samples with overlapping size ranges. Recently, we introduced interferometric nanoparticle tracking analysis (iNTA) as a new method for high-precision measurement of nanoparticle size and refractive index. Here, we show that by counting the number of trajectories that cross the focal plane, iNTA can measure concentrations of subpopulations in a polydisperse mixture in a quantitative manner and without the need for a calibration sample. We evaluate our method on both monodisperse samples and mixtures of known concentrations. Furthermore, we assess the concentration of SARS-CoV-2 in supernatant samples obtained from infected cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575442">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575442" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575442">
        <p class="paperTitle">Astigmatism-based focus stabilisation with universal objective lens compatibility, extended operating range and nanometre precision</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575442" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575442" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rahmani, A.; Cox, T.; Thamaravelil Abhimanue Achary, A.; Ponjavic, A.</p>
        <p class="info">Score: 9.6, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575442' target='https://doi.org/10.1101/2024.01.15.575442'> 10.1101/2024.01.15.575442</a></p>
        <p class="abstract">Focus stabilisation is vital for long-term fluorescence imaging, particularly in the case of high-resolution imaging techniques. Current stabilisation solutions either rely on fiducial markers that can be perturbative, or on beam reflection monitoring that is limited to high-numerical aperture objective lenses, making multimodal and large-scale imaging challenging. We introduce a beam-based method that relies on astigmatism, which offers advantages in terms of precision and the range over which focus stabilisation is effective. This approach is shown to be compatible with a wide range of objective lenses (10x-100x), typically achieving &lt;10 nm precision with &gt;10 um operating range. Notably, our technique is largely unaffected by pointing stability errors, which in combination with implementation through a standalone Raspberry Pi architecture, offers a versatile focus stabilisation unit that can be added onto most existing microscope setups.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575012">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575012" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575012">
        <p class="paperTitle">On-chip large-scale-integration and 2D collective modes of genetically programmed artificial cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575012" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575012" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ricouvier, J.; Mostov, P.; Shabtai, O.; Tayar, A.; Karzbrun, E.; Khakimzhan, A.; Noireaux, V.; Shulman Daube, S.; Bar-Ziv, R.</p>
        <p class="info">Score: 8.4, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575012' target='https://doi.org/10.1101/2024.01.10.575012'> 10.1101/2024.01.10.575012</a></p>
        <p class="abstract">The on-chip large-scale-integration of genetically programmed artificial cells capable of exhibiting collective modes is an important goal for fundamental research and technology. Here, we report assembly of a 2D layout of 1024 monolithic DNA compartments as artificial cells on a 5-millimeter square silicon chip. Homeostatic cell-free protein synthesis reactions driven by genetic circuits occur inside the compartments. We created a reaction-diffusion system with a 30x30 square lattice of artificial cells interconnected by thin capillaries for diffusion of products. Driving the system by a genetic oscillator revealed emergent collective modes of synchrony and propagating phase waves in 2D, with dynamics controlled by geometry. This demonstrates a class of nonequilibrium autonomous systems, where chemical energy consumed to make proteins induces 2D collective patterns of gene expression on multicellular scales, with applications in biological computing, sensing, and materials synthesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574656">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574656" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574656">
        <p class="paperTitle">Lipid osmosis, membrane tension, and other mechanochemical driving forces of lipid flow</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574656" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574656" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Y.; Lin, C.</p>
        <p class="info">Score: 9.3, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574656' target='https://doi.org/10.1101/2024.01.08.574656'> 10.1101/2024.01.08.574656</a></p>
        <p class="abstract">Nonvesicular lipid transport among different membranes or membrane domains plays crucial roles in processes such as lipid homeostasis and organelle biogenesis. However, the forces that drive such lipid transport are not well understood. We propose that lipids tend to flow towards the membrane area with a higher membrane protein density in a process termed lipid osmosis. This process lowers the membrane tension in the area when the lipid flow reaches equilibrium. We examine the thermodynamic basis and experimental evidence of lipid osmosis and the corresponding osmotic membrane tension. We predict that lipid osmosis can drive lipid flows between different membrane regions through lipid transfer proteins, scramblases, or other similar barriers that selectively pass lipids but not membrane proteins. We also speculate on the biological functions of lipid osmosis. Finally, we explore other driving forces for lipid transfer and describe potential methods and systems to further test our theory.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.571765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.571765" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.571765">
        <p class="paperTitle">Oblique Line Scan Illumination Enables Expansive, Accurate and Sensitive Single Protein Measurements in Solution and in Living Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.571765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.571765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Driouchi, A.; Bretan, M.; Davis, B. J.; Heckert, A. B.; Seeger, M.; Bradley Silva, M.; Forrest, W. S.; Hsiung, J.; Tan, J.; Yang, H.; McSwiggen, D. T.; Song, L.; Sule, A.; Abaie, B.; Conroy, B.; Elliott, L. A.; Gonzalez, E.; Ilkov, F.; Isaacs, J.; Labaria, G.; Lagana, M.; Larsen, D. D.; Margolin, B.; Nguyen, M. K.; Rine, J.; Tang, Y.; Vana, M.; Wilkey, A.; Zhang, Z.; Basham, S.; Ho, J. J.; Johnson, S.; Klammer, A. A.; Lin, K.; Darzacq, X.; Betzig, E.; Berman, R. T.; Anderson, D. J.</p>
        <p class="info">Score: 14.1, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.571765' target='https://doi.org/10.1101/2023.12.21.571765'> 10.1101/2023.12.21.571765</a></p>
        <p class="abstract">Single-molecule localization microscopy (SMLM) techniques, such as single-molecule tracking (SMT), enable in situ measurements in cells from which data-rich metrics can be extracted. SMT has been successfully applied to a variety of biological questions and model systems, aiming to unravel the spatiotemporal regulation of molecular mechanisms that govern protein function, downstream pathway effects, and cellular function. While powerful, SMLM often suffers from low throughput and illumination inhomogeneity, along with microscope and user-induced technical biases. Due to technical limitations in scaling SMLM techniques, a tradeoff between spatiotemporal resolution and throughput has been made historically, restricting broad application of these technologies. Here we address these limitations using Oblique Line Scan (OLS), a robust single-objective light-sheet based illumination and detection modality that achieves nanoscale spatial resolution and sub-millisecond temporal resolution across a 250 x 190 m field of view. We demonstrate OLS-enabled SMT on Halo-Tagged proteins in living cells capturing protein motion up to 14 m2 /s. By exploiting the adaptability of the acquisition frame rate and the improved rejection of out of focus light, we extend the utility of OLS beyond cellular compartments with in-solution SMT (isSMT) for single-molecule measurement of ligand-protein interactions and disruption of protein-protein interactions (PPI). We illustrate the versatility of OLS by showcasing two-color SMT, STORM, and single molecule fluorescence recovery after photobleaching (FRAP). OLS expands the range of SMLM applications and paves the way for robust, high-throughput single-molecule investigations of protein dynamics required for drug screening and systems biology studies, both in cells and in solution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575647">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575647" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575647">
        <p class="paperTitle">Mechanistic Insights into G-protein Activation via Phosphorylation Mediated Non-Canonical Pathway</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575647" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575647" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shewani, K.; Madhu, M. K.; Murarka, R. K.</p>
        <p class="info">Score: 4.9, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575647' target='https://doi.org/10.1101/2024.01.15.575647'> 10.1101/2024.01.15.575647</a></p>
        <p class="abstract">Activation of heterotrimeric G-proteins (G{beta}{gamma}) downstream to receptor tyrosine kinases (RTKs) is a well-established crosstalk between the signaling pathways mediated by G-protein coupled receptors (GPCRs) and RTKs. While GPCR serves as a guanine exchange factor (GEF) in the canonical activation of G that facilitates the exchange of GDP for GTP, the mechanism through which RTK phosphorylations induce G activation remains unclear. Recent experimental studies revealed that the epidermal growth factor receptor (EGFR), a well-known RTK, phosphorylates the helical domain tyrosine residues Y154 and Y155 and accelerates the GDP release from the Gi3, a subtype of G-protein. Using well-tempered metadynamics and extensive unbiased molecular dynamics simulations, we captured the GDP release event and identified the intermediates between bound and unbound states through Markov state models. The additional negative charges introduced by phosphorylations rewired the inter-residue interactions and significantly weakened the salt bridges at the domain interface, contributing to the increased separation of the Ras-like and helical domains of G-protein. Furthermore, the unfolding of helix F resulted in greater flexibility near the hinge region, facilitating a greater distance between domains in the phosphorylated Gi3. The release of GDP in the phosphorylated G-protein occurred at a faster rate compared to the unphosphorylated state, caused by increased fluctuations in conserved regions of P-loop, switch 1, and switch 2. Overall, this study provides atomistic insights into the activation of G-proteins induced by RTK phosphorylations and identifies the specific structural motifs involved in the process. The knowledge gained from the study could establish a foundation for targeting non-canonical signaling pathways and developing therapeutic strategies against the ailments associated with dysregulated G-protein signaling.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
