<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A Fast Interferometric Beam Shaper for Multi-Emitter 3D MINFLUX
Authors: Geismann, M. K.; Gomez-Segalas, A.; Passera, A.; Shirzadian, M.; Balzarotti, F.
Score: 20.3, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570565
Displacing beams of light in a controlled manner is integral to a wide range of optical methods such as scanning microscopes and optical tweezers. Optical super-resolution techniques like STED and RESOLFT, as well as localization techniques like MINFLUX and MINSTED rely on accurate and fast beam displacements.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="A Fast Interferometric Beam Shaper for Multi-Emitter 3D MINFLUX
Authors: Geismann, M. K.; Gomez-Segalas, A.; Passera, A.; Shirzadian, M.; Balzarotti, F.
Score: 20.3, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570565
Displacing beams of light in a controlled manner is integral to a wide range of optical methods such as scanning microscopes and optical tweezers. Optical super-resolution techniques like STED and RESOLFT, as well as localization techniques like MINFLUX and MINSTED rely on accurate and fast beam displacements." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="A Fast Interferometric Beam Shaper for Multi-Emitter 3D MINFLUX
Authors: Geismann, M. K.; Gomez-Segalas, A.; Passera, A.; Shirzadian, M.; Balzarotti, F.
Score: 20.3, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570565
Displacing beams of light in a controlled manner is integral to a wide range of optical methods such as scanning microscopes and optical tweezers. Optical super-resolution techniques like STED and RESOLFT, as well as localization techniques like MINFLUX and MINSTED rely on accurate and fast beam displacements."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "A Fast Interferometric Beam Shaper for Multi-Emitter 3D MINFLUX\nAuthors: Geismann, M. K.; Gomez-Segalas, A.; Passera, A.; Shirzadian, M.; Balzarotti, F.\nScore: 20.3, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570565\nDisplacing beams of light in a controlled manner is integral to a wide range of optical methods such as scanning microscopes and optical tweezers. Optical super-resolution techniques like STED and RESOLFT, as well as localization techniques like MINFLUX and MINSTED rely on accurate and fast beam displacements.",
  "keywords": [
    
  ],
  "articleBody": " A Fast Interferometric Beam Shaper for Multi-Emitter 3D MINFLUX\nAuthors: Geismann, M. K.; Gomez-Segalas, A.; Passera, A.; Shirzadian, M.; Balzarotti, F.\nScore: 20.3, Published: 2023-12-10 DOI: 10.1101/2023.12.09.570565\nDisplacing beams of light in a controlled manner is integral to a wide range of optical methods such as scanning microscopes and optical tweezers. Optical super-resolution techniques like STED and RESOLFT, as well as localization techniques like MINFLUX and MINSTED rely on accurate and fast beam displacements. Extending these methods to the third dimension requires axial deflection, which, in contrast to lateral deflection, remains technologically challenging on the microsecond scale. Here, we present a general-purpose beam-shaping polarization interferometer that enables such axial deflections and, based on it, showcase a four-channel dual-color excitation system for three-dimensional MINFLUX imaging and tracking. We include first demonstrations of improved MINFLUX localization schemes that utilize the combination of distinct beam shapes and three-dimensional multi-emitter tracking. We believe that the presented approach will facilitate the broader adoption of three dimensional MINFLUX and provides a versatile basis for future implementations of advanced single-molecule localization methods.\nInterplay between environmental yielding and dynamic forcing regulates bacterial growth\nAuthors: Hancock, A. M.; Datta, S. S.\nScore: 17.9, Published: 2023-12-07 DOI: 10.1101/2023.12.05.569991\nMany bacterial habitats--ranging from gels and tissues in the body to cell-secreted exopolysaccharides in biofilms--are rheologically complex, undergo dynamic external forcing, and have unevenly-distributed nutrients. How do these features jointly influence how the resident cells grow and proliferate? Here, we address this question by studying the growth of Escherichia coli dispersed in granular hydrogel matrices with defined and highly-tunable structural and rheological properties, under different amounts of external forcing imposed by mechanical shaking, and in both aerobic and anaerobic conditions. Our experiments establish a general principle: that the balance between the yield stress of the environment that the cells inhabit{sigma} y and the external stress imposed on the environment{sigma} regulates bacterial growth by modulating transport of essential nutrients to the cells. In particular, when{sigma} y \u003c {sigma}, the environment is easily fluidized and mixed over large scales, providing nutrients to the cells and sustaining complete cellular growth. By contrast, when{sigma} y \u003e {sigma}, the elasticity of the environment suppresses large-scale fluid mixing, limiting nutrient availability and arresting cellular growth. Our work thus reveals a new mechanism, beyond effects that change cellular behavior via local forcing, by which the rheology of the environment may regulate microbial physiology in diverse natural and industrial settings.\nTracking transcription-translation coupling in real-time\nAuthors: Qureshi, N. S.; Duss, O.\nScore: 11.2, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570708\nA central question in biology is how macromolecular machines function cooperatively. In bacteria, transcription and translation occur in the same cellular compartment and can be physically and functionally coupled. While several recently published high-resolution structures of the ribosome-RNA polymerase (RNAP) complex provided first mechanistic insight into the coupling process, we do not know how these structural snapshots are placed along a dynamic reaction trajectory. Here, we reconstitute a complete active transcription-translation system and develop multi-color single-molecule fluorescence microscopy experiments to directly and simultaneously track transcription elongation, translation elongation and the physical and functional coupling between the ribosome and the RNAP in real-time. Our data show that the ribosome slows down while colliding with the RNAP and that coupling following a collision becomes less efficient. Unexpectedly, physical coupling can occur with hundreds of nucleotides of intervening mRNA between both machineries by mRNA looping, and increases in efficiency in presence of NusG. We detect active transcription elongation during mRNA looping and show that NusA-paused RNAPs can be activated by the ribosome by long-range physical coupling. We provide an alternative explanation on how the ribosome can rescue RNAP from frequent pausing without requiring collisions by a closely trailing ribosome. Our data mechanistically highlight an example of how macromolecular machines central to gene expression physically and functionally cooperate.\nStructural basis of connexin-36 gap junction channel inhibition\nAuthors: Ding, X.; Aureli, S.; Vaithia, A.; Lavriha, P.; Schuster, D.; Khanppnavar, B.; Li, X.; Blum, T. B.; Picotti, P.; Gervasio, F. L.; Korkhov, V. M.\nScore: 11.1, Published: 2023-12-09 DOI: 10.1101/2023.12.09.570920\nConnexin gap junction channels and hemichannels play important roles in intercellular communication and signaling. Some of connexin isoforms are associated with diseases, including hereditary neuropathies, heart disease and cancer. Although small molecule inhibitors of connexins show promise as therapeutic agents, the molecular mechanisms of connexin channel inhibition are unknown. Here, we report the cryo-EM structure of connexin-36 (Cx36) bound to an anti-malarial drug mefloquine at 2.1 [A] resolution. Six drug binding sites partially occlude the pore of each connexon forming the channel. Each drug molecule in the ring makes contacts with residues in the pore-lining pocket and with the neighbouring mefloquine molecules, partially occluding the pore and modifying the pore electrostatics, ultimately reducing solute translocation through the channel. Structures of Cx36 in the presence of quinine and quinidine show a similar mode of drug binding. Molecular dynamics simulations of Cx36 bound to mefloquine show that drug binding affects the kinetics of ion passage through the pore. This previously undescribed mode of connexin channel inhibition presents an opportunity for designing subtype-specific connexin inhibitors.\nEstimation of the mass density of biological matter from refractive index measurements\nAuthors: Moeckel, C.; Beck, T.; Kaliman, S.; Abuhattum, S.; Kim, K.; Kolb, J.; Wehner, D.; Zaburdaev, V.; Guck, J.\nScore: 12.3, Published: 2023-12-05 DOI: 10.1101/2023.12.05.569868\nThe quantification of physical properties of biological matter gives rise to novel ways of understanding functional mechanisms by utilizing models that explicitly depend on physical observables. One of the basic biophysical properties is the mass density (MD), which determines the degree of crowdedness. It impacts the dynamics in sub-cellular compartments and further plays a major role in defining the opto-acoustical properties of cells and tissues. As such, the MD can be connected to the refractive index (RI) via the well known Lorentz-Lorenz relation, which takes into account the polarizability of matter. However, computing the MD based on RI measurements poses a challenge as it requires detailed knowledge of the biochemical composition of the sample. Here we propose a methodology on how to account for a priori and a posteriori assumptions about the biochemical composition of the sample as well as respective RI measurements. To that aim, we employ the Biot mixing rule of RIs alongside the assumption of volume additivity to find an approximate relation of MD and RI. We use Monte-Carlo simulations as well as Gaussian propagation of uncertainty to obtain approximate analytical solutions for the respective uncertainties of MD and RI. We validate this approach by applying it to a set of well characterized complex mixtures given by bovine milk and intralipid emulsion. Further, we employ it to estimate the mass density of trunk tissue of living zebrafish (Danio rerio) larvae. Our results enable quantifying changes of mass density estimates based on variations in the a priori assumptions. This illustrates the importance of implementing this methodology not only for MD estimations but for many other related biophysical problems, such as mechanical measurements using Brillouin microscopy and transient optical coherence elastography.\nHigh-throughput viscoelastic characterization of cells in hyperbolic microchannels\nAuthors: Reichel, F.; Goswami, R.; Girardo, S.; Guck, J.\nScore: 9.8, Published: 2023-12-10 DOI: 10.1101/2023.12.10.571005\nExtensive research has demonstrated the potential of cell viscoelastic properties as intrinsic indicators of cell state, functionality, and disease. For this, several microfluidic techniques have been developed to measure cell viscoelasticity with high-throughput. However, current microchannel designs introduce complex stress distributions on cells, leading to inaccuracies in determining the stress-strain relationship and, consequently, the viscoelastic properties. Here, we introduce a novel approach using hyperbolic microchannels that enable precise measurements under a constant extensional stress and offer a straightforward stress-strain relationship, while operating at a measurement rate of up to 100 cells per second. We quantified the stresses acting in the channels using mechanical calibration particles made from polyacrylamide (PAAm) and found that the measurement buffer, a solution of methyl cellulose and phosphate buffered saline, has a constant extensional viscosity of 0.5 Pa s up to 200 s-1. By measuring oil droplets with varying viscosities, we successfully detected changes in the relaxation time of the droplets and our approach could be used to get the interfacial tension and viscosity of liquid-liquid droplet systems from the same measurement. We further applied this methodology to PAAm microgel beads, demonstrating the accurate recovery of Young's moduli and the near-ideal elastic behavior of the beads. To explore the influence of altered cell viscoelasticity, we treated HL60 human leukemia cells with Latrunculin B and Nocodazole, resulting in clear changes in cell stiffness while relaxation times were only minimally affected. In conclusion, our approach offers a streamlined and time-efficient solution for assessing the viscoelastic properties of large cell populations and other microscale soft particles.\nSingle-Molecule FRET at 10 MHz Count Rates\nAuthors: Grabenhorst, L.; Sturzenegger, F.; Hasler, M.; Schuler, B.; Tinnefeld, P.\nScore: 9.6, Published: 2023-12-08 DOI: 10.1101/2023.12.08.570755\nA bottleneck in many studies utilizing single-molecule Forster Resonance Energy Transfer (smFRET) is the attainable photon count rate as it determines the temporal resolution of the experiment. As many biologically relevant processes occur on timescales that are hardly accessible with currently achievable photon count rates, there has been considerable effort to find strategies to increase the stability and brightness of fluorescent dyes. Here, we use DNA nanoantennas to drastically increase the achievable photon count rates and to observe fast biomolecular dynamics in the small volume between two plasmonic nanoparticles. As a proof of concept, we observe the coupled folding and binding of two intrinsically disordered proteins which form transient encounter complexes with lifetimes on the order of 100 s. To test the limits of our approach, we also investigated the hybridization of a short single-stranded DNA to its complementary counterpart, revealing a transition path time of 17 s at photon count rates of around 10 MHz, which is an order-of-magnitude improvement when compared to the state of the art. Concomitantly, the photostability was increased, enabling many seconds long megahertz fluorescence time traces. Due to the modular nature of the DNA origami method, this platform can be adapted to a broad range of biomolecules, providing a promising approach to study previously unobservable ultrafast biophysical processes.\nDeep learning assisted single particle tracking for automated correlation between diffusion and function\nAuthors: Kaestel-Hansen, J.; de Sautu, M.; Saminathan, A.; Scanavachi, G.; Correia, R. F. B. D. C.; Nielsen, A. J.; Bleshoy, S. V.; Boomsma, W.; Kirchhausen, T.; Hatzakis, N. S.\nScore: 12.5, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567393\nSub-cellular diffusion in living systems reflects cellular processes and interactions. Recent advances in optical microscopy allow the tracking of this nanoscale diffusion of individual objects with an unprecedented level of precision. However, the agnostic and automated extraction of functional information from the diffusion of molecules and organelles within the sub-cellular environment, is labor-intensive and poses a significant challenge. Here we introduce DeepSPT, a deep learning framework to interpret the diffusional 2D or 3D temporal behavior of objects in a rapid and efficient manner, agnostically. Demonstrating its versatility, we have applied DeepSPT to automated mapping of the early events of viral infections, identifying distinct types of endosomal organelles, and clathrin-coated pits and vesicles with up to 95% accuracy and within seconds instead of weeks. The fact that DeepSPT effectively extracts biological information from diffusion alone indicates that besides structure, motion encodes function at the molecular and subcellular level.\nLanthanide Cathodophores for Multicolor Electron Microscopy\nAuthors: Abdul Rehman, S.; Conway, J. B.; Nichols, A.; Soucy, E. R.; Dee, A.; Stevens, K.; Merminod, S.; MacNaughton, I.; Curtis, A.; Prigozhin, M. B.\nScore: 8.0, Published: 2023-12-12 DOI: 10.1101/2023.12.11.570835\nElectron microscopy (EM) and fluorescence imaging are indispensable techniques that provide complementary information on cellular organization. Combining these two modalities is a long-standing challenge in bioimaging. In principle, it should be possible to use the electron beam both for ultrastructural imaging and for molecular localization. The latter could be accomplished by directly exciting suitable biomolecular labels and detecting their luminescence - a process termed cathodoluminescence (CL). Here, we achieve multicolor, single-particle CL imaging of sub-20-nm lanthanide nanocrystals (cathodophores) in the same field of view on the surface of a mammalian cell while simultaneously imaging cellular ultrastructure. In pursuit of this goal, we have developed a comprehensive framework for single-particle CL imaging of lanthanide nanocrystals. By mitigating nonlocal excitation due to secondary electrons, we achieved single-particle detection of multiple spectrally distinct types of sub-20-nm cathodophores. The smallest detectable cathodophores were ~15 nm in diameter. We found that the CL emission rate of single nanocrystals increased monotonically with lanthanide doping level and scaled linearly with nanocrystal diameter. Furthermore, even in the absence of inert shells, cathodophores were not quenched in the context of mammalian cells processed for EM imaging using heavy-metal staining and sputter-coating. These findings establish cathodophores as promising biomolecular tags for multicolor EM. Moreover, our results inform general design rules for precise control and rational engineering of future generations of single-particle cathodoluminescent nanoprobes.\nA Structured Illumination Microscopy Framework with Spatial-Domain Noise Propagation.\nAuthors: Saurabh, A.; Brown, P. T.; Bryan, J. S.; Fox, Z. R.; Kruithoff, R.; Shepherd, D. P.; Presse, S. P.\nScore: 7.3, Published: 2023-12-11 DOI: 10.1101/2023.12.07.570701\nBiological images captured by a microscope are characterized by heterogeneous signal-to-noise ratios (SNRs) across the field of view due to spatially varying photon emission and camera noise. State-of-the-art unsupervised structured illumination microscopy (SIM) reconstruction algorithms, commonly implemented in the Fourier domain, do not accurately model this noise and suffer from high-frequency artifacts, user-dependent choices of smoothness constraints making assumptions on biological features, and unphysical negative values in the recovered fluorescence intensity map. On the other hand, supervised methods rely on large datasets for training, and often require retraining for new sample structures. Consequently, achieving high contrast near the maximum theoretical resolution in an unsupervised, physically principled manner remains a challenging task. Here, we propose Bayesian-SIM (B-SIM), an unsupervised Bayesian framework to quantitatively reconstruct SIM data, rectifying these shortcomings by accurately incorporating all noise sources in the spatial domain. To accelerate the reconstruction process for computational feasibility, we devise a parallelized Monte Carlo sampling strategy for inference. We benchmark our framework on both simulated and experimental images, and demonstrate improved contrast permitting feature recovery at up to 25% shorter length scales over state-of-the-art methods at both high- and low-SNR. B-SIM enables unsupervised, quantitative, physically accurate reconstruction without the need for labeled training data, democratizing high-quality SIM reconstruction and expands the capabilities of live-cell SIM to lower SNR, potentially revealing biological features in previously inaccessible regimes.\n",
  "wordCount" : "2394",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.09.570565">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.09.570565" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.09.570565">
        <p class="paperTitle">A Fast Interferometric Beam Shaper for Multi-Emitter 3D MINFLUX</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.09.570565" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.09.570565" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Geismann, M. K.; Gomez-Segalas, A.; Passera, A.; Shirzadian, M.; Balzarotti, F.</p>
        <p class="info">Score: 20.3, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.09.570565' target='https://doi.org/10.1101/2023.12.09.570565'> 10.1101/2023.12.09.570565</a></p>
        <p class="abstract">Displacing beams of light in a controlled manner is integral to a wide range of optical methods such as scanning microscopes and optical tweezers. Optical super-resolution techniques like STED and RESOLFT, as well as localization techniques like MINFLUX and MINSTED rely on accurate and fast beam displacements. Extending these methods to the third dimension requires axial deflection, which, in contrast to lateral deflection, remains technologically challenging on the microsecond scale. Here, we present a general-purpose beam-shaping polarization interferometer that enables such axial deflections and, based on it, showcase a four-channel dual-color excitation system for three-dimensional MINFLUX imaging and tracking. We include first demonstrations of improved MINFLUX localization schemes that utilize the combination of distinct beam shapes and three-dimensional multi-emitter tracking. We believe that the presented approach will facilitate the broader adoption of three dimensional MINFLUX and provides a versatile basis for future implementations of advanced single-molecule localization methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.569991">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.569991" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.569991">
        <p class="paperTitle">Interplay between environmental yielding and dynamic forcing regulates bacterial growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.569991" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.569991" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hancock, A. M.; Datta, S. S.</p>
        <p class="info">Score: 17.9, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.569991' target='https://doi.org/10.1101/2023.12.05.569991'> 10.1101/2023.12.05.569991</a></p>
        <p class="abstract">Many bacterial habitats--ranging from gels and tissues in the body to cell-secreted exopolysaccharides in biofilms--are rheologically complex, undergo dynamic external forcing, and have unevenly-distributed nutrients. How do these features jointly influence how the resident cells grow and proliferate? Here, we address this question by studying the growth of Escherichia coli dispersed in granular hydrogel matrices with defined and highly-tunable structural and rheological properties, under different amounts of external forcing imposed by mechanical shaking, and in both aerobic and anaerobic conditions. Our experiments establish a general principle: that the balance between the yield stress of the environment that the cells inhabit{sigma} y and the external stress imposed on the environment{sigma} regulates bacterial growth by modulating transport of essential nutrients to the cells. In particular, when{sigma} y &lt; {sigma}, the environment is easily fluidized and mixed over large scales, providing nutrients to the cells and sustaining complete cellular growth. By contrast, when{sigma} y &gt; {sigma}, the elasticity of the environment suppresses large-scale fluid mixing, limiting nutrient availability and arresting cellular growth. Our work thus reveals a new mechanism, beyond effects that change cellular behavior via local forcing, by which the rheology of the environment may regulate microbial physiology in diverse natural and industrial settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570708">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570708" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570708">
        <p class="paperTitle">Tracking transcription-translation coupling in real-time</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570708" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570708" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Qureshi, N. S.; Duss, O.</p>
        <p class="info">Score: 11.2, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570708' target='https://doi.org/10.1101/2023.12.07.570708'> 10.1101/2023.12.07.570708</a></p>
        <p class="abstract">A central question in biology is how macromolecular machines function cooperatively. In bacteria, transcription and translation occur in the same cellular compartment and can be physically and functionally coupled. While several recently published high-resolution structures of the ribosome-RNA polymerase (RNAP) complex provided first mechanistic insight into the coupling process, we do not know how these structural snapshots are placed along a dynamic reaction trajectory. Here, we reconstitute a complete active transcription-translation system and develop multi-color single-molecule fluorescence microscopy experiments to directly and simultaneously track transcription elongation, translation elongation and the physical and functional coupling between the ribosome and the RNAP in real-time. Our data show that the ribosome slows down while colliding with the RNAP and that coupling following a collision becomes less efficient. Unexpectedly, physical coupling can occur with hundreds of nucleotides of intervening mRNA between both machineries by mRNA looping, and increases in efficiency in presence of NusG. We detect active transcription elongation during mRNA looping and show that NusA-paused RNAPs can be activated by the ribosome by long-range physical coupling. We provide an alternative explanation on how the ribosome can rescue RNAP from frequent pausing without requiring collisions by a closely trailing ribosome. Our data mechanistically highlight an example of how macromolecular machines central to gene expression physically and functionally cooperate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.09.570920">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.09.570920" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.09.570920">
        <p class="paperTitle">Structural basis of connexin-36 gap junction channel inhibition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.09.570920" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.09.570920" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ding, X.; Aureli, S.; Vaithia, A.; Lavriha, P.; Schuster, D.; Khanppnavar, B.; Li, X.; Blum, T. B.; Picotti, P.; Gervasio, F. L.; Korkhov, V. M.</p>
        <p class="info">Score: 11.1, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.09.570920' target='https://doi.org/10.1101/2023.12.09.570920'> 10.1101/2023.12.09.570920</a></p>
        <p class="abstract">Connexin gap junction channels and hemichannels play important roles in intercellular communication and signaling. Some of connexin isoforms are associated with diseases, including hereditary neuropathies, heart disease and cancer. Although small molecule inhibitors of connexins show promise as therapeutic agents, the molecular mechanisms of connexin channel inhibition are unknown. Here, we report the cryo-EM structure of connexin-36 (Cx36) bound to an anti-malarial drug mefloquine at 2.1 [A] resolution. Six drug binding sites partially occlude the pore of each connexon forming the channel. Each drug molecule in the ring makes contacts with residues in the pore-lining pocket and with the neighbouring mefloquine molecules, partially occluding the pore and modifying the pore electrostatics, ultimately reducing solute translocation through the channel. Structures of Cx36 in the presence of quinine and quinidine show a similar mode of drug binding. Molecular dynamics simulations of Cx36 bound to mefloquine show that drug binding affects the kinetics of ion passage through the pore. This previously undescribed mode of connexin channel inhibition presents an opportunity for designing subtype-specific connexin inhibitors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.05.569868">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.05.569868" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.05.569868">
        <p class="paperTitle">Estimation of the mass density of biological matter from refractive index measurements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.05.569868" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.05.569868" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moeckel, C.; Beck, T.; Kaliman, S.; Abuhattum, S.; Kim, K.; Kolb, J.; Wehner, D.; Zaburdaev, V.; Guck, J.</p>
        <p class="info">Score: 12.3, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.05.569868' target='https://doi.org/10.1101/2023.12.05.569868'> 10.1101/2023.12.05.569868</a></p>
        <p class="abstract">The quantification of physical properties of biological matter gives rise to novel ways of understanding functional mechanisms by utilizing models that explicitly depend on physical observables. One of the basic biophysical properties is the mass density (MD), which determines the degree of crowdedness. It impacts the dynamics in sub-cellular compartments and further plays a major role in defining the opto-acoustical properties of cells and tissues. As such, the MD can be connected to the refractive index (RI) via the well known Lorentz-Lorenz relation, which takes into account the polarizability of matter. However, computing the MD based on RI measurements poses a challenge as it requires detailed knowledge of the biochemical composition of the sample. Here we propose a methodology on how to account for a priori and a posteriori assumptions about the biochemical composition of the sample as well as respective RI measurements. To that aim, we employ the Biot mixing rule of RIs alongside the assumption of volume additivity to find an approximate relation of MD and RI. We use Monte-Carlo simulations as well as Gaussian propagation of uncertainty to obtain approximate analytical solutions for the respective uncertainties of MD and RI. We validate this approach by applying it to a set of well characterized complex mixtures given by bovine milk and intralipid emulsion. Further, we employ it to estimate the mass density of trunk tissue of living zebrafish (Danio rerio) larvae. Our results enable quantifying changes of mass density estimates based on variations in the a priori assumptions. This illustrates the importance of implementing this methodology not only for MD estimations but for many other related biophysical problems, such as mechanical measurements using Brillouin microscopy and transient optical coherence elastography.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.10.571005">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.10.571005" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.10.571005">
        <p class="paperTitle">High-throughput viscoelastic characterization of cells in hyperbolic microchannels</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.10.571005" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.10.571005" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reichel, F.; Goswami, R.; Girardo, S.; Guck, J.</p>
        <p class="info">Score: 9.8, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.10.571005' target='https://doi.org/10.1101/2023.12.10.571005'> 10.1101/2023.12.10.571005</a></p>
        <p class="abstract">Extensive research has demonstrated the potential of cell viscoelastic properties as intrinsic indicators of cell state, functionality, and disease. For this, several microfluidic techniques have been developed to measure cell viscoelasticity with high-throughput. However, current microchannel designs introduce complex stress distributions on cells, leading to inaccuracies in determining the stress-strain relationship and, consequently, the viscoelastic properties. Here, we introduce a novel approach using hyperbolic microchannels that enable precise measurements under a constant extensional stress and offer a straightforward stress-strain relationship, while operating at a measurement rate of up to 100 cells per second. We quantified the stresses acting in the channels using mechanical calibration particles made from polyacrylamide (PAAm) and found that the measurement buffer, a solution of methyl cellulose and phosphate buffered saline, has a constant extensional viscosity of 0.5 Pa s up to 200 s-1. By measuring oil droplets with varying viscosities, we successfully detected changes in the relaxation time of the droplets and our approach could be used to get the interfacial tension and viscosity of liquid-liquid droplet systems from the same measurement. We further applied this methodology to PAAm microgel beads, demonstrating the accurate recovery of Young&#39;s moduli and the near-ideal elastic behavior of the beads. To explore the influence of altered cell viscoelasticity, we treated HL60 human leukemia cells with Latrunculin B and Nocodazole, resulting in clear changes in cell stiffness while relaxation times were only minimally affected. In conclusion, our approach offers a streamlined and time-efficient solution for assessing the viscoelastic properties of large cell populations and other microscale soft particles.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570755">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570755" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570755">
        <p class="paperTitle">Single-Molecule FRET at 10 MHz Count Rates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570755" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570755" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grabenhorst, L.; Sturzenegger, F.; Hasler, M.; Schuler, B.; Tinnefeld, P.</p>
        <p class="info">Score: 9.6, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570755' target='https://doi.org/10.1101/2023.12.08.570755'> 10.1101/2023.12.08.570755</a></p>
        <p class="abstract">A bottleneck in many studies utilizing single-molecule Forster Resonance Energy Transfer (smFRET) is the attainable photon count rate as it determines the temporal resolution of the experiment. As many biologically relevant processes occur on timescales that are hardly accessible with currently achievable photon count rates, there has been considerable effort to find strategies to increase the stability and brightness of fluorescent dyes. Here, we use DNA nanoantennas to drastically increase the achievable photon count rates and to observe fast biomolecular dynamics in the small volume between two plasmonic nanoparticles. As a proof of concept, we observe the coupled folding and binding of two intrinsically disordered proteins which form transient encounter complexes with lifetimes on the order of 100 s. To test the limits of our approach, we also investigated the hybridization of a short single-stranded DNA to its complementary counterpart, revealing a transition path time of 17 s at photon count rates of around 10 MHz, which is an order-of-magnitude improvement when compared to the state of the art. Concomitantly, the photostability was increased, enabling many seconds long megahertz fluorescence time traces. Due to the modular nature of the DNA origami method, this platform can be adapted to a broad range of biomolecules, providing a promising approach to study previously unobservable ultrafast biophysical processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567393">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567393" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567393">
        <p class="paperTitle">Deep learning assisted single particle tracking for automated correlation between diffusion and function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567393" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567393" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kaestel-Hansen, J.; de Sautu, M.; Saminathan, A.; Scanavachi, G.; Correia, R. F. B. D. C.; Nielsen, A. J.; Bleshoy, S. V.; Boomsma, W.; Kirchhausen, T.; Hatzakis, N. S.</p>
        <p class="info">Score: 12.5, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567393' target='https://doi.org/10.1101/2023.11.16.567393'> 10.1101/2023.11.16.567393</a></p>
        <p class="abstract">Sub-cellular diffusion in living systems reflects cellular processes and interactions. Recent advances in optical microscopy allow the tracking of this nanoscale diffusion of individual objects with an unprecedented level of precision. However, the agnostic and automated extraction of functional information from the diffusion of molecules and organelles within the sub-cellular environment, is labor-intensive and poses a significant challenge. Here we introduce DeepSPT, a deep learning framework to interpret the diffusional 2D or 3D temporal behavior of objects in a rapid and efficient manner, agnostically. Demonstrating its versatility, we have applied DeepSPT to automated mapping of the early events of viral infections, identifying distinct types of endosomal organelles, and clathrin-coated pits and vesicles with up to 95% accuracy and within seconds instead of weeks. The fact that DeepSPT effectively extracts biological information from diffusion alone indicates that besides structure, motion encodes function at the molecular and subcellular level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.570835">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.570835" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.570835">
        <p class="paperTitle">Lanthanide Cathodophores for Multicolor Electron Microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.570835" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.570835" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abdul Rehman, S.; Conway, J. B.; Nichols, A.; Soucy, E. R.; Dee, A.; Stevens, K.; Merminod, S.; MacNaughton, I.; Curtis, A.; Prigozhin, M. B.</p>
        <p class="info">Score: 8.0, Published: 2023-12-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.570835' target='https://doi.org/10.1101/2023.12.11.570835'> 10.1101/2023.12.11.570835</a></p>
        <p class="abstract">Electron microscopy (EM) and fluorescence imaging are indispensable techniques that provide complementary information on cellular organization. Combining these two modalities is a long-standing challenge in bioimaging. In principle, it should be possible to use the electron beam both for ultrastructural imaging and for molecular localization. The latter could be accomplished by directly exciting suitable biomolecular labels and detecting their luminescence - a process termed cathodoluminescence (CL). Here, we achieve multicolor, single-particle CL imaging of sub-20-nm lanthanide nanocrystals (cathodophores) in the same field of view on the surface of a mammalian cell while simultaneously imaging cellular ultrastructure. In pursuit of this goal, we have developed a comprehensive framework for single-particle CL imaging of lanthanide nanocrystals. By mitigating nonlocal excitation due to secondary electrons, we achieved single-particle detection of multiple spectrally distinct types of sub-20-nm cathodophores. The smallest detectable cathodophores were ~15 nm in diameter. We found that the CL emission rate of single nanocrystals increased monotonically with lanthanide doping level and scaled linearly with nanocrystal diameter. Furthermore, even in the absence of inert shells, cathodophores were not quenched in the context of mammalian cells processed for EM imaging using heavy-metal staining and sputter-coating. These findings establish cathodophores as promising biomolecular tags for multicolor EM. Moreover, our results inform general design rules for precise control and rational engineering of future generations of single-particle cathodoluminescent nanoprobes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570701">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570701" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570701">
        <p class="paperTitle">A Structured Illumination Microscopy Framework with Spatial-Domain Noise Propagation.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570701" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570701" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Saurabh, A.; Brown, P. T.; Bryan, J. S.; Fox, Z. R.; Kruithoff, R.; Shepherd, D. P.; Presse, S. P.</p>
        <p class="info">Score: 7.3, Published: 2023-12-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570701' target='https://doi.org/10.1101/2023.12.07.570701'> 10.1101/2023.12.07.570701</a></p>
        <p class="abstract">Biological images captured by a microscope are characterized by heterogeneous signal-to-noise ratios (SNRs) across the field of view due to spatially varying photon emission and camera noise. State-of-the-art unsupervised structured illumination microscopy (SIM) reconstruction algorithms, commonly implemented in the Fourier domain, do not accurately model this noise and suffer from high-frequency artifacts, user-dependent choices of smoothness constraints making assumptions on biological features, and unphysical negative values in the recovered fluorescence intensity map. On the other hand, supervised methods rely on large datasets for training, and often require retraining for new sample structures. Consequently, achieving high contrast near the maximum theoretical resolution in an unsupervised, physically principled manner remains a challenging task. Here, we propose Bayesian-SIM (B-SIM), an unsupervised Bayesian framework to quantitatively reconstruct SIM data, rectifying these shortcomings by accurately incorporating all noise sources in the spatial domain. To accelerate the reconstruction process for computational feasibility, we devise a parallelized Monte Carlo sampling strategy for inference. We benchmark our framework on both simulated and experimental images, and demonstrate improved contrast permitting feature recovery at up to 25% shorter length scales over state-of-the-art methods at both high- and low-SNR. B-SIM enables unsupervised, quantitative, physically accurate reconstruction without the need for labeled training data, democratizing high-quality SIM reconstruction and expands the capabilities of live-cell SIM to lower SNR, potentially revealing biological features in previously inaccessible regimes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
