<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells
Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.
Score: 143.8, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799
Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells
Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.
Score: 143.8, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799
Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-06T10:39:07+00:00" />
<meta property="article:modified_time" content="2023-09-06T10:39:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells
Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.
Score: 143.8, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799
Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells\nAuthors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.\nScore: 143.8, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799\nAdvances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms.",
  "keywords": [
    
  ],
  "articleBody": " Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells\nAuthors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.\nScore: 143.8, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799\nAdvances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms. This method, cryoDRGN-ET, learns a deep generative model of 3D density maps directly from subtomogram tilt series images and can capture states diverse in both composition and conformation. We use this approach to reconstruct the in situ translation dynamics of prokaryotic ribosomes, and we reveal the distribution of functional states during translation elongation populated by S. cerevisiae ribosomes inside cells.\nqMaLioffG: A single green fluorescent protein FLIM indicator enabling quantitative imaging of endogenous ATP\nAuthors: Arai, S.; Itoh, H.; Vu, C. Q.; Nakayama, M.; Oshima, M.; Morita, A.; Okamoto, K.; Okuda, S.; Teranishi, A.; Osawa, M.; Tamura, Y.; Nonoyama, S.; Takuma, M.; Fujie, T.; Sarker, S. R.; Sudhaharan, T.; Kiya, T.; Lane, E. B.; Kitaguchi, T.\nScore: 40.6, Published: 2023-08-31 DOI: 10.1101/2023.08.29.555275\nThe widespread use of fluorescence lifetime imaging microscopy (FLIM) for quantitative imaging is hindered by the limited availability of a FLIM-based genetically encoded indicator using a conventional 488 nm laser. Here, we present qMaLioffG, a single green fluorescent protein FLIM indicator showing a fluorescence lifetime change in ATP concentration within the physiological range. This allows quantitative imaging of endogenous ATP to investigate cellular energy status of different cell types.\nDifferentiable simulation to develop molecular dynamics force fields for disordered proteins\nAuthors: Greener, J. G.\nScore: 25.1, Published: 2023-08-31 DOI: 10.1101/2023.08.29.555352\nImplicit solvent force fields are computationally efficient but can be unsuitable for running molecular dynamics on disordered proteins. Here I improve the a99SB-disp force field and the GBNeck2 implicit solvent model to better describe disordered proteins. Differentiable molecular simulations with 5 ns trajectories are used to jointly optimise 108 parameters to better match explicit solvent trajectories. Simulations with the improved force field better reproduce the radius of gyration and secondary structure content seen in experiments, whilst showing slightly degraded performance on folded proteins and protein complexes. The force field, called GB99dms, reproduces the results of a small molecule binding study and improves agreement to experiment for the aggregation of amyloid peptides. GB99dms, which can be used in OpenMM and is available at https://github.com/greener-group/GB99dms, is useful for simulating a range of protein systems. This work is the first to show that gradients can be obtained directly from nanosecond-length differentiable simulations of biomolecules and highlights the effectiveness of this approach to training whole force fields to match desired properties.\nStep-wise activation of a Family C GPCR\nAuthors: Kumar, K. K.; Wang, H.; Habrian, C.; Latorraca, N. R.; Xu, J.; O'Brien, E. S.; Zhang, C.; Montabana, E.; Koehl, A.; Marqusee, S.; Isacoff, E. Y.; Kobilka, B. K.\nScore: 18.3, Published: 2023-08-30 DOI: 10.1101/2023.08.29.555158\nMetabotropic glutamate receptors belong to a family of G protein-coupled receptors that are obligate dimers and possess a large extracellular ligand-binding domain (ECD) that is linked via a cysteine-rich domain (CRDs) to their 7-transmembrane (TM) domain. Upon activation, these receptors undergo a large conformational change to transmit the ligand binding signal from the ECD to the G protein-coupling TM. In this manuscript, we propose a model for a sequential, multistep activation mechanism of metabotropic glutamate receptor subtype 5. We present a series of structures in lipid nanodiscs, from inactive to fully active, including agonist-bound intermediate states. Further, using bulk and single-molecule fluorescence imaging we reveal distinct receptor conformations upon allosteric modulator and G protein binding.\nCFM: Confinement Force Microscopy-a dynamic, precise and stable microconfiner for traction force microscopy in spatial confinement\nAuthors: Betz, T.; Abbasi, F.; Rieck, K.; Brandt, M.; Matis, M.\nScore: 17.5, Published: 2023-08-22 DOI: 10.1101/2023.08.22.554088\nIn vivo, cells experience complex tissue environments with various chemical and physical features. They sense and respond to tissue morphology and mechanical properties and adjust their behavior and function based on the surrounding. In contrast to the free environment experienced on 2D substrates commonly used in research, the 3D natural environment represents a major physical obstacle for cells. Here, cells are usually confined either by the extracellular matrix (ECM) or neighboring cells. The importance of such confinements has been demon-strated in the past decades by showing its influence on cell decision-making in many vital biological processes such as migration, division and cytoskeletal reorganization. Despite these insights, the sheer level of complexity faced when studying cell biological questions in biomimetic confined situations, led to an indispensable need for a 3D system which can simulate the in vivo confined condition, while being capable of providing microenvironments with different chemical and physical properties for the cells and capturing the mechanical forces and properties of the studied biological sample. Here we introduce a microconfiner that finally provides a new imaging capacity, namely the confine-ment force microscopy (CFM). We are able to adjust the confinement level in real time during microscopy while measuring not only the the cellular traction but also the cellular compression forces. Furthermore, the chemical and physical properties of the microenvironment can be optimized for the respective questions. We demonstrate the power of this confinement system by the mechanical response of cells, migration analysis of immune cells, the timed force generation during durotaxis driven adhesion switching and the viscoelastic properties of cancer tissue.\nToxoplasma gondii actin filaments are tuned for rapid disassembly and turnover\nAuthors: Hvorecny, K. L.; Sladewski, T. E.; De La Cruz, E. M.; Kollman, J. M.; Heaslip, A. T.\nScore: 15.6, Published: 2023-08-30 DOI: 10.1101/2023.08.29.555340\nThe cytoskeletal protein actin plays a critical role in the pathogenicity of Toxoplasma gondii, mediating invasion and egress, cargo transport, and organelle inheritance. Advances in live cell imaging have revealed extensive filamentous actin networks in the Apicomplexan parasite, but there is conflicting data regarding the biochemical and biophysical properties of Toxoplasma actin. Here, we imaged the in vitro assembly of individual Toxoplasma actin filaments in real time, showing that native, unstabilized filaments grow tens of microns in length. Unlike skeletal muscle actin, Toxoplasma filaments intrinsically undergo rapid treadmilling due to a high critical concentration, fast monomer dissociation, and rapid nucleotide exchange. Cryo-EM structures of stabilized and unstabilized filaments show an architecture like skeletal actin, with differences in assembly contacts in the D-loop that explain the dynamic nature of the filament, likely a conserved feature of Apicomplexan actin. This work demonstrates that evolutionary changes at assembly interfaces can tune dynamic properties of actin filaments without disrupting their conserved structure.\nLongitudinal Awake Imaging of Deep Mouse Brain Microvasculature with Super-resolution Ultrasound Localization Microscopy\nAuthors: Wang, Y.; Lowerison, M. R.; You, Q.; Lin, B.-Z.; Llano, D. A.; Song, P.\nScore: 12.4, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555789\nSuper-resolution ultrasound localization microscopy (ULM) is an emerging imaging modality that resolves capillary-scale microvasculature in deep tissues. However, existing preclinical ULM applications are largely constrained to anesthetized animals, introducing confounding vascular effects such as vasodilation and altered hemodynamics. As such, ULM quantifications (e.g., vessel diameter, density, and flow velocity) may be confounded by the use of anesthesia, undermining the usefulness of ULM in practice. Here we introduce a method to address this limitation and achieve ULM imaging in awake mouse brain. Pupillary monitoring was used to confirm the awake state during ULM imaging. ULM revealed that veins showed a greater degree of vascularity reduction from anesthesia to awake states than did arteries. The reduction was most significant in the midbrain and least significant in the cortex. ULM also revealed a significant reduction in venous blood flow velocity across different brain regions under awake conditions. Serial in vivo imaging of the same animal brain at weekly intervals demonstrated the highly robust longitudinal imaging capability of the proposed technique. This is the first study demonstrating longitudinal ULM imaging in the awake mouse brain, which is essential for many ULM brain applications that require awake and behaving animals.\nCHARMM-GUI Multicomponent Assembler for Modeling and Simulation of Complex Multicomponent Systems\nAuthors: Kern, N. R.; Lee, J.; Choi, Y. K.; Im, W.\nScore: 5.1, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555590\nAtomic-scale molecular modeling and simulation are powerful tools for computational biology. However, constructing models with large, densely packed molecules, non-water solvents, or with combinations of multiple biomembranes, polymers, and nanomaterials remains challenging and requires significant time and expertise. Furthermore, existing tools do not support such assemblies under the periodic boundary conditions (PBC) necessary for molecular simulation. Here, we describe Multicomponent Assembler in CHARMM-GUI that automates complex molecular assembly and simulation input preparation under the PBC. We demonstrate its versatility by preparing 6 challenging systems with varying density of large components: (1) solvated proteins, (2) solvated proteins with a pre-equilibrated membrane, (3) solvated proteins with a sheet-like nanomaterial, (4) solvated proteins with a sheet-like polymer, (5) a mixed membrane-nanomaterial system, and (6) a sheet-like polymer with gaseous solvent. Multicomponent Assembler is expected to be a unique cyberinfrastructure to facilitate innovative studies of complex interactions between small (organic and inorganic) molecules, biomacromolecules, polymers, and nanomaterials.\nExact solution of a three-stage model of stochastic gene expression including cell-cycle dynamics\nAuthors: Wang, Y.; Yu, Z.; Grima, R.; Cao, Z.\nScore: 3.7, Published: 2023-09-04 DOI: 10.1101/2023.08.29.555255\nThe classical three-stage model of stochastic gene expression predicts the statistics of single cell mRNA and protein number fluctuations as a function of the rates of promoter switching, transcription, translation, degradation and dilution. While this model is easily simulated, its analytical solution remains an unsolved problem. Here we modify this model to explicitly include cell-cycle dynamics and then derive an exact solution for the time-dependent joint distribution of mRNA and protein numbers. We show large differences between this model and the classical model which captures cell-cycle effects implicitly via effective first-order dilution reactions. In particular we find that the Fano factor of protein numbers calculated from a population snapshot measurement are underestimated by the classical model whereas the correlation between mRNA and protein can be either over-or underestimated, depending on the timescales of mRNA degradation and promoter switching relative to the mean cell-cycle duration time.\nDiscovering optimal kinetic pathways for self-assembly using automatic differentiation\nAuthors: Jhaveri, A.; Loggia, S.; Qian, Y.; Johnson, M. E.\nScore: 3.4, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555551\nDuring self-assembly of macromolecules ranging from ribosomes to viral capsids, the formation of long-lived intermediates or kinetic traps can dramatically reduce yield of the functional products. Understanding biological mechanisms for avoiding traps and efficiently assembling is essential for designing synthetic assembly systems, but learning optimal solutions requires numerical searches in high-dimensional parameter spaces. Here, we exploit powerful automatic differentiation algorithms commonly employed by deep learning frameworks to optimize physical models of reversible self-assembly, discovering diverse solutions in the space of rate constants for 3-7 subunit complexes. We define two biologically-inspired protocols that prevent kinetic trapping through either internal design of subunit binding kinetics or external design of subunit titration in time. Our third protocol acts to recycle intermediates, mimicking energy-consuming enzymes. Preventative solutions via interface design are the most efficient and scale better with more subunits, but external control via titration or recycling are effective even for poorly evolved binding kinetics. Whilst all protocols can produce good solutions, diverse subunits always helps; these complexes access more efficient solutions when following external control protocols, and are simpler to design for internal control, as molecular interfaces do not need modification during assembly given sufficient variation in dimerization rates. Our results identify universal scaling in the cost of kinetic trapping, and provide multiple strategies for eliminating trapping and maximizing assembly yield across large parameter spaces. SIGNIFICANCEMacromolecular complexes are frequently composed of diverse subunits. While evolution may favor repeated subunits and symmetry, we show how diversity in subunits generates an expansive parameter space that naturally improves the expressivity of self-assembly, much like a deeper neural network. By using automatic differentiation algorithms commonly used in deep learning, we searched these parameter spaces to identify classes of kinetic protocols that mimic biological solutions for productive self-assembly. Our results reveal how high-yield complexes that easily become kinetically trapped in incomplete intermediates can instead be steered by internal design of rate constants or external and active control of subunits to efficiently assemble, exploiting nonequilibrium control of these ubiquitous dynamical systems.\n",
  "wordCount" : "2041",
  "inLanguage": "en",
  "datePublished": "2023-09-06T10:39:07Z",
  "dateModified": "2023-09-06T10:39:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on September 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553799">
        <p class="paperTitle">Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.</p>
        <p class="info">Score: 143.8, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553799' target='https://doi.org/10.1101/2023.08.18.553799'> 10.1101/2023.08.18.553799</a></p>
        <p class="abstract">Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms. This method, cryoDRGN-ET, learns a deep generative model of 3D density maps directly from subtomogram tilt series images and can capture states diverse in both composition and conformation. We use this approach to reconstruct the in situ translation dynamics of prokaryotic ribosomes, and we reveal the distribution of functional states during translation elongation populated by S. cerevisiae ribosomes inside cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555275">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555275" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555275">
        <p class="paperTitle">qMaLioffG: A single green fluorescent protein FLIM indicator enabling quantitative imaging of endogenous ATP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555275" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555275" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arai, S.; Itoh, H.; Vu, C. Q.; Nakayama, M.; Oshima, M.; Morita, A.; Okamoto, K.; Okuda, S.; Teranishi, A.; Osawa, M.; Tamura, Y.; Nonoyama, S.; Takuma, M.; Fujie, T.; Sarker, S. R.; Sudhaharan, T.; Kiya, T.; Lane, E. B.; Kitaguchi, T.</p>
        <p class="info">Score: 40.6, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555275' target='https://doi.org/10.1101/2023.08.29.555275'> 10.1101/2023.08.29.555275</a></p>
        <p class="abstract">The widespread use of fluorescence lifetime imaging microscopy (FLIM) for quantitative imaging is hindered by the limited availability of a FLIM-based genetically encoded indicator using a conventional 488 nm laser. Here, we present qMaLioffG, a single green fluorescent protein FLIM indicator showing a fluorescence lifetime change in ATP concentration within the physiological range. This allows quantitative imaging of endogenous ATP to investigate cellular energy status of different cell types.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555352">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555352" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555352">
        <p class="paperTitle">Differentiable simulation to develop molecular dynamics force fields for disordered proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555352" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555352" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Greener, J. G.</p>
        <p class="info">Score: 25.1, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555352' target='https://doi.org/10.1101/2023.08.29.555352'> 10.1101/2023.08.29.555352</a></p>
        <p class="abstract">Implicit solvent force fields are computationally efficient but can be unsuitable for running molecular dynamics on disordered proteins. Here I improve the a99SB-disp force field and the GBNeck2 implicit solvent model to better describe disordered proteins. Differentiable molecular simulations with 5 ns trajectories are used to jointly optimise 108 parameters to better match explicit solvent trajectories. Simulations with the improved force field better reproduce the radius of gyration and secondary structure content seen in experiments, whilst showing slightly degraded performance on folded proteins and protein complexes. The force field, called GB99dms, reproduces the results of a small molecule binding study and improves agreement to experiment for the aggregation of amyloid peptides. GB99dms, which can be used in OpenMM and is available at https://github.com/greener-group/GB99dms, is useful for simulating a range of protein systems. This work is the first to show that gradients can be obtained directly from nanosecond-length differentiable simulations of biomolecules and highlights the effectiveness of this approach to training whole force fields to match desired properties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555158">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555158" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555158">
        <p class="paperTitle">Step-wise activation of a Family C GPCR</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555158" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555158" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kumar, K. K.; Wang, H.; Habrian, C.; Latorraca, N. R.; Xu, J.; O&#39;Brien, E. S.; Zhang, C.; Montabana, E.; Koehl, A.; Marqusee, S.; Isacoff, E. Y.; Kobilka, B. K.</p>
        <p class="info">Score: 18.3, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555158' target='https://doi.org/10.1101/2023.08.29.555158'> 10.1101/2023.08.29.555158</a></p>
        <p class="abstract">Metabotropic glutamate receptors belong to a family of G protein-coupled receptors that are obligate dimers and possess a large extracellular ligand-binding domain (ECD) that is linked via a cysteine-rich domain (CRDs) to their 7-transmembrane (TM) domain. Upon activation, these receptors undergo a large conformational change to transmit the ligand binding signal from the ECD to the G protein-coupling TM. In this manuscript, we propose a model for a sequential, multistep activation mechanism of metabotropic glutamate receptor subtype 5. We present a series of structures in lipid nanodiscs, from inactive to fully active, including agonist-bound intermediate states. Further, using bulk and single-molecule fluorescence imaging we reveal distinct receptor conformations upon allosteric modulator and G protein binding.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554088">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554088" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554088">
        <p class="paperTitle">CFM: Confinement Force Microscopy-a dynamic, precise and stable microconfiner for traction force microscopy in spatial confinement</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554088" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554088" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Betz, T.; Abbasi, F.; Rieck, K.; Brandt, M.; Matis, M.</p>
        <p class="info">Score: 17.5, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554088' target='https://doi.org/10.1101/2023.08.22.554088'> 10.1101/2023.08.22.554088</a></p>
        <p class="abstract">In vivo, cells experience complex tissue environments with various chemical and physical features. They sense and respond to tissue morphology and mechanical properties and adjust their behavior and function based on the surrounding. In contrast to the free environment experienced on 2D substrates commonly used in research, the 3D natural environment represents a major physical obstacle for cells. Here, cells are usually confined either by the extracellular matrix (ECM) or neighboring cells. The importance of such confinements has been demon-strated in the past decades by showing its influence on cell decision-making in many vital biological processes such as migration, division and cytoskeletal reorganization. Despite these insights, the sheer level of complexity faced when studying cell biological questions in biomimetic confined situations, led to an indispensable need for a 3D system which can simulate the in vivo confined condition, while being capable of providing microenvironments with different chemical and physical properties for the cells and capturing the mechanical forces and properties of the studied biological sample. Here we introduce a microconfiner that finally provides a new imaging capacity, namely the confine-ment force microscopy (CFM). We are able to adjust the confinement level in real time during microscopy while measuring not only the the cellular traction but also the cellular compression forces. Furthermore, the chemical and physical properties of the microenvironment can be optimized for the respective questions. We demonstrate the power of this confinement system by the mechanical response of cells, migration analysis of immune cells, the timed force generation during durotaxis driven adhesion switching and the viscoelastic properties of cancer tissue.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555340">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555340" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555340">
        <p class="paperTitle">Toxoplasma gondii actin filaments are tuned for rapid disassembly and turnover</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555340" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555340" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hvorecny, K. L.; Sladewski, T. E.; De La Cruz, E. M.; Kollman, J. M.; Heaslip, A. T.</p>
        <p class="info">Score: 15.6, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555340' target='https://doi.org/10.1101/2023.08.29.555340'> 10.1101/2023.08.29.555340</a></p>
        <p class="abstract">The cytoskeletal protein actin plays a critical role in the pathogenicity of Toxoplasma gondii, mediating invasion and egress, cargo transport, and organelle inheritance. Advances in live cell imaging have revealed extensive filamentous actin networks in the Apicomplexan parasite, but there is conflicting data regarding the biochemical and biophysical properties of Toxoplasma actin. Here, we imaged the in vitro assembly of individual Toxoplasma actin filaments in real time, showing that native, unstabilized filaments grow tens of microns in length. Unlike skeletal muscle actin, Toxoplasma filaments intrinsically undergo rapid treadmilling due to a high critical concentration, fast monomer dissociation, and rapid nucleotide exchange. Cryo-EM structures of stabilized and unstabilized filaments show an architecture like skeletal actin, with differences in assembly contacts in the D-loop that explain the dynamic nature of the filament, likely a conserved feature of Apicomplexan actin. This work demonstrates that evolutionary changes at assembly interfaces can tune dynamic properties of actin filaments without disrupting their conserved structure.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555789">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555789" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555789">
        <p class="paperTitle">Longitudinal Awake Imaging of Deep Mouse Brain Microvasculature with Super-resolution Ultrasound Localization Microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555789" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555789" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Lowerison, M. R.; You, Q.; Lin, B.-Z.; Llano, D. A.; Song, P.</p>
        <p class="info">Score: 12.4, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555789' target='https://doi.org/10.1101/2023.09.01.555789'> 10.1101/2023.09.01.555789</a></p>
        <p class="abstract">Super-resolution ultrasound localization microscopy (ULM) is an emerging imaging modality that resolves capillary-scale microvasculature in deep tissues. However, existing preclinical ULM applications are largely constrained to anesthetized animals, introducing confounding vascular effects such as vasodilation and altered hemodynamics. As such, ULM quantifications (e.g., vessel diameter, density, and flow velocity) may be confounded by the use of anesthesia, undermining the usefulness of ULM in practice. Here we introduce a method to address this limitation and achieve ULM imaging in awake mouse brain. Pupillary monitoring was used to confirm the awake state during ULM imaging. ULM revealed that veins showed a greater degree of vascularity reduction from anesthesia to awake states than did arteries. The reduction was most significant in the midbrain and least significant in the cortex. ULM also revealed a significant reduction in venous blood flow velocity across different brain regions under awake conditions. Serial in vivo imaging of the same animal brain at weekly intervals demonstrated the highly robust longitudinal imaging capability of the proposed technique. This is the first study demonstrating longitudinal ULM imaging in the awake mouse brain, which is essential for many ULM brain applications that require awake and behaving animals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555590">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555590" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555590">
        <p class="paperTitle">CHARMM-GUI Multicomponent Assembler for Modeling and Simulation of Complex Multicomponent Systems</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555590" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555590" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kern, N. R.; Lee, J.; Choi, Y. K.; Im, W.</p>
        <p class="info">Score: 5.1, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555590' target='https://doi.org/10.1101/2023.08.30.555590'> 10.1101/2023.08.30.555590</a></p>
        <p class="abstract">Atomic-scale molecular modeling and simulation are powerful tools for computational biology. However, constructing models with large, densely packed molecules, non-water solvents, or with combinations of multiple biomembranes, polymers, and nanomaterials remains challenging and requires significant time and expertise. Furthermore, existing tools do not support such assemblies under the periodic boundary conditions (PBC) necessary for molecular simulation. Here, we describe Multicomponent Assembler in CHARMM-GUI that automates complex molecular assembly and simulation input preparation under the PBC. We demonstrate its versatility by preparing 6 challenging systems with varying density of large components: (1) solvated proteins, (2) solvated proteins with a pre-equilibrated membrane, (3) solvated proteins with a sheet-like nanomaterial, (4) solvated proteins with a sheet-like polymer, (5) a mixed membrane-nanomaterial system, and (6) a sheet-like polymer with gaseous solvent. Multicomponent Assembler is expected to be a unique cyberinfrastructure to facilitate innovative studies of complex interactions between small (organic and inorganic) molecules, biomacromolecules, polymers, and nanomaterials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555255">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555255" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555255">
        <p class="paperTitle">Exact solution of a three-stage model of stochastic gene expression including cell-cycle dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555255" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555255" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Y.; Yu, Z.; Grima, R.; Cao, Z.</p>
        <p class="info">Score: 3.7, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555255' target='https://doi.org/10.1101/2023.08.29.555255'> 10.1101/2023.08.29.555255</a></p>
        <p class="abstract">The classical three-stage model of stochastic gene expression predicts the statistics of single cell mRNA and protein number fluctuations as a function of the rates of promoter switching, transcription, translation, degradation and dilution. While this model is easily simulated, its analytical solution remains an unsolved problem. Here we modify this model to explicitly include cell-cycle dynamics and then derive an exact solution for the time-dependent joint distribution of mRNA and protein numbers. We show large differences between this model and the classical model which captures cell-cycle effects implicitly via effective first-order dilution reactions. In particular we find that the Fano factor of protein numbers calculated from a population snapshot measurement are underestimated by the classical model whereas the correlation between mRNA and protein can be either over-or underestimated, depending on the timescales of mRNA degradation and promoter switching relative to the mean cell-cycle duration time.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555551">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555551" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555551">
        <p class="paperTitle">Discovering optimal kinetic pathways for self-assembly using automatic differentiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555551" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555551" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jhaveri, A.; Loggia, S.; Qian, Y.; Johnson, M. E.</p>
        <p class="info">Score: 3.4, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555551' target='https://doi.org/10.1101/2023.08.30.555551'> 10.1101/2023.08.30.555551</a></p>
        <p class="abstract">During self-assembly of macromolecules ranging from ribosomes to viral capsids, the formation of long-lived intermediates or kinetic traps can dramatically reduce yield of the functional products. Understanding biological mechanisms for avoiding traps and efficiently assembling is essential for designing synthetic assembly systems, but learning optimal solutions requires numerical searches in high-dimensional parameter spaces. Here, we exploit powerful automatic differentiation algorithms commonly employed by deep learning frameworks to optimize physical models of reversible self-assembly, discovering diverse solutions in the space of rate constants for 3-7 subunit complexes. We define two biologically-inspired protocols that prevent kinetic trapping through either internal design of subunit binding kinetics or external design of subunit titration in time. Our third protocol acts to recycle intermediates, mimicking energy-consuming enzymes. Preventative solutions via interface design are the most efficient and scale better with more subunits, but external control via titration or recycling are effective even for poorly evolved binding kinetics. Whilst all protocols can produce good solutions, diverse subunits always helps; these complexes access more efficient solutions when following external control protocols, and are simpler to design for internal control, as molecular interfaces do not need modification during assembly given sufficient variation in dimerization rates. Our results identify universal scaling in the cost of kinetic trapping, and provide multiple strategies for eliminating trapping and maximizing assembly yield across large parameter spaces.

SIGNIFICANCEMacromolecular complexes are frequently composed of diverse subunits. While evolution may favor repeated subunits and symmetry, we show how diversity in subunits generates an expansive parameter space that naturally improves the  expressivity of self-assembly, much like a deeper neural network. By using automatic differentiation algorithms commonly used in deep learning, we searched these parameter spaces to identify classes of kinetic protocols that mimic biological solutions for productive self-assembly. Our results reveal how high-yield complexes that easily become kinetically trapped in incomplete intermediates can instead be steered by internal design of rate constants or external and active control of subunits to efficiently assemble, exploiting nonequilibrium control of these ubiquitous dynamical systems.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
