<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Coevolution-driven method for efficiently simulating conformational changes in proteins reveals molecular details of ligand effects in the beta2AR receptor
Authors: Mitrovic, D.; Chen, Y.; Marciniak, A.; Delemotte, L.
Score: 33.8, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549854
With the advent of AI-powered structure prediction, the scientific community is inching ever closer to solving protein folding. An unresolved enigma, however, is to accurately, reliably and deterministically predict alternative conformational states that are crucial for the function of e.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Coevolution-driven method for efficiently simulating conformational changes in proteins reveals molecular details of ligand effects in the beta2AR receptor
Authors: Mitrovic, D.; Chen, Y.; Marciniak, A.; Delemotte, L.
Score: 33.8, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549854
With the advent of AI-powered structure prediction, the scientific community is inching ever closer to solving protein folding. An unresolved enigma, however, is to accurately, reliably and deterministically predict alternative conformational states that are crucial for the function of e." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Coevolution-driven method for efficiently simulating conformational changes in proteins reveals molecular details of ligand effects in the beta2AR receptor
Authors: Mitrovic, D.; Chen, Y.; Marciniak, A.; Delemotte, L.
Score: 33.8, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549854
With the advent of AI-powered structure prediction, the scientific community is inching ever closer to solving protein folding. An unresolved enigma, however, is to accurately, reliably and deterministically predict alternative conformational states that are crucial for the function of e."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Coevolution-driven method for efficiently simulating conformational changes in proteins reveals molecular details of ligand effects in the beta2AR receptor\nAuthors: Mitrovic, D.; Chen, Y.; Marciniak, A.; Delemotte, L.\nScore: 33.8, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549854\nWith the advent of AI-powered structure prediction, the scientific community is inching ever closer to solving protein folding. An unresolved enigma, however, is to accurately, reliably and deterministically predict alternative conformational states that are crucial for the function of e.",
  "keywords": [
    
  ],
  "articleBody": " Coevolution-driven method for efficiently simulating conformational changes in proteins reveals molecular details of ligand effects in the beta2AR receptor\nAuthors: Mitrovic, D.; Chen, Y.; Marciniak, A.; Delemotte, L.\nScore: 33.8, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549854\nWith the advent of AI-powered structure prediction, the scientific community is inching ever closer to solving protein folding. An unresolved enigma, however, is to accurately, reliably and deterministically predict alternative conformational states that are crucial for the function of e.g. transporters, receptors or ion channels where conformational cycling is innately coupled to protein function. Accurately discovering and exploring all conformational states of membrane proteins has been challenging due to the need to retain atomistic detail while enhancing the sampling along interesting degrees of freedom. The challenges include but are not limited to finding which degrees of freedom are relevant, how to accelerate the sampling along them, and then quantifying the populations of each micro- and macrostate. In this work, we present a methodology that finds the relevant degrees of freedom by combining evolution and physics through machine learning and apply it to the {beta}2 adrenergic receptor conformational sampling. In addition to predicting new conformations that are beyond the training set, we have computed free energy surfaces associated with the proteins conformational landscape. We then show that the methodology is able to quantitatively predict the effect of an array of ligands on the {beta}2 adrenergic receptor activation, and that the full conformational landscape, including states related to biased signaling, is discovered using this procedure. Lastly, we also stake out the structural determinants of activation and inactivation pathway signaling through different ligands.\nDirect optical measurement of intra-molecular distances down to the Angstrom scale\nAuthors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.\nScore: 131.2, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133\nOptical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision. Our method is validated by quantifying well-characterized 1-to-10 nm distances in polypeptides and proteins. Moreover, we visualize the orientations of immunoglobulin subunits and reveal specific configurations of a histidine kinase domain dimer. Our results open the door for examining proximities and interactions of macromolecules under physiological conditions.\nPhase-Separated Droplets Swim to Their Dissolution\nAuthors: Jambon-Puillet, E.; Testa, A.; Lorenz, C. M.; Style, R. W.; Rebane, A.; Dufresne, E. R.\nScore: 22.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549556\nBiological macromolecules can condense into liquid domains. In cells, these condensates form membraneless organelles that can organize chemical reactions1,2. However, little is known about the physical consequences of chemical activity in and around condensates. Working with model bovine serum albumin (BSA) condensates3, we show that droplets swim along chemical gradients. Active BSA droplets loaded with urease swim toward each other. Passive BSA droplets show diverse responses to externally applied gradients of the enzymes substrate and products. In all these cases, droplets swim toward solvent conditions that favor their dissolution. We call this behavior dialytaxis, and expect it to be generic, as conditions which favor dissolution typically reduce interfacial tension, whose gradients are well-known to drive droplet motion4,5. These results suggest alternative physical mechanisms for active transport in living cells, and may enable the design of fluid micro-robots.\nTranscription regulates the spatio-temporal dynamics of genes through micro-compartmentalization\nAuthors: Salari, H.; Fourel, G.; Jost, D.\nScore: 10.3, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549489\nAlthough our understanding of the involvement of heterochromatin architectural factors in shaping nuclear organization is improving, there is still ongoing debate regarding the role of active genes in this process. In this study, we utilize publicly-available Micro-C data from mouse embryonic stem cells to investigate the relationship between gene transcription and 3D gene folding. Our analysis uncovers a nonmonotonic - globally positive - correlation between intragenic contact density and Pol II occupancy, independent of cohesin-based loop extrusion. Through the development of a biophysical model integrating the role of transcription dynamics within a polymer model of chromosome organization, we demonstrate that Pol II-mediated attractive interactions with limited valency between transcribed regions yield quantitative predictions consistent with Hi-C and live-imaging experiments. Our work provides compelling evidence that transcriptional activity shapes the 4D genome through Pol II-mediated micro-compartmentalization.\nProtein semisynthesis underscores role of a conserved lysine in activation and desensitization of acid-sensing ion channels\nAuthors: Sarkar, D.; Galleano, I.; Heusser, S. A.; Ou, S. Y.; Uzun, G. R.; Khoo, K. K.; van Noort, G. J. v. d. H.; Harrison, J. S.; Pless, S. A.\nScore: 10.7, Published: 2023-07-21 DOI: 10.1101/2023.01.23.525296\nAcid-sensing ion channels (ASICs) are trimeric ion channels that open a cation-conducting pore in response to proton binding. Excessive ASIC activation during prolonged acidosis in conditions such as inflammation and ischemia is linked to pain and stroke. A conserved lysine in the extracellular domain (Lys211 in mASIC1a) is suggested to play a key role in ASIC function. However, the precise contributions are difficult to dissect with conventional mutagenesis, as replacement of Lys211 with naturally occurring amino acids invariably changes multiple physico-chemical parameters. Here, we study the contribution of Lys211 to mASIC1a function using tandem protein-trans splicing (tPTS) to incorporate non-canonical lysine analogs. We conduct optimization efforts to improve splicing and functionally interrogate semisynthetic mASIC1a. In combination with molecular modeling, we show that Lys211 charge and side chain length are crucial to activation and desensitization, thus emphasizing that tPTS can enable atomic-scale interrogations of membrane proteins in live cells.\nFolding-upon-binding pathways of an intrinsically disordered protein from a deep Markov state model\nAuthors: Sisk, T.; Robustelli, P.\nScore: 5.3, Published: 2023-07-25 DOI: 10.1101/2023.07.21.550103\nA central challenge in the study of intrinsically disordered proteins is the characterization of the mechanisms by which they bind their physiological interaction partners. Here, we utilize a deep learning based Markov state modeling approach to characterize the folding-upon-binding pathways observed in a long-time scale molecular dynamics simulation of a disordered region of the measles virus nucleoprotein NTAIL reversibly binding the X domain of the measles virus phosphoprotein complex. We find that folding-upon-binding predominantly occurs via two distinct encounter complexes that are differentiated by the binding orientation, helical content, and conformational heterogeneity of NTAIL. We do not, however, find evidence for the existence of canonical conformational selection or induced fit binding pathways. We observe four kinetically separated native-like bound states that interconvert on time scales of eighty to five hundred nanoseconds. These bound states share a core set of native intermolecular contacts and stable NTAIL helices and are differentiated by a sequential formation of native and non-native contacts and additional helical turns. Our analyses provide an atomic resolution structural description of intermediate states in a folding-upon-binding pathway and elucidate the nature of the kinetic barriers between metastable states in a dynamic and heterogenous, or 'fuzzy', protein complex.\nA Generalizable Scaffold-Based Approach for Structure Determination of RNAs by Cryo-EM\nAuthors: Kieft, J. S.; Langeberg, C.\nScore: 16.2, Published: 2023-07-06 DOI: 10.1101/2023.07.06.547879\nSingle-particle cryo-electron microscopy (cryo-EM) can reveal the structures of large and often dynamic molecules, but smaller biomolecules remain challenging targets due to their intrinsic low signal to noise ratio. Methods to resolve small proteins have been applied but development of similar approaches for small structured RNA elements have lagged. Here, we present a scaffold-based approach that we used to recover maps of sub-25 kDa RNA domains to 4.5 - 5.0 [A]. While lacking the detail of true high-resolution maps, these are suitable for model building and preliminary structure determination. We demonstrate this method faithfully recovers the structure of several RNA elements of known structure, and it promises to be generalized to other RNAs without disturbing their native fold. This approach may streamline the sample preparation process and reduce the optimization required for data collection. This first-generation scaffold approach provides a system for RNA structure determination by cryo-EM and lays the groundwork for further scaffold optimization to achieve higher resolution.\nUnraveling membrane protein localization and stabilization in nanodiscs\nAuthors: Kim, S.-J.; Koh, Y. H.; Roh, S.-H.\nScore: 4.6, Published: 2023-07-22 DOI: 10.1101/2023.07.20.549795\nNanodiscs are nanoscale structures consisting of a lipid bilayer surrounded by membrane scaffold proteins (MSPs). They are widely used in the study of membrane proteins (MPs) because they provide a stable lipid environment. However, the precise mechanism governing MP behavior within the nanodisc remains elusive. Here, we examined the cryo-EM structures of various MPs reconstituted in nanodiscs from an electron microscopy database (EMPIAR). By analyzing the heterogeneity and interactions in the nanodiscs, we found that MPs within nanodiscs display a distinct spatial preference toward the edges of the nanodisc shells. Furthermore, we observed that MPs can establish direct, amphipathic interactions with the MSPs, promoting protein stability. These interactions may induce a rearrangement of the MSP-MSP interactions, leading to the formation of MP-MSP interactions Collectively, our study provides structural and biophysical insights into how nanodiscs contribute to MP structural behavior and stability. SIGNIFICANCEBy thoroughly examining multiple deposited datasets of membrane proteins (MPs) reconstituted in nanodiscs, we have gathered compelling evidence that MPs exhibit a clear spatial inclination toward the periphery of the nanodisc shells. Moreover, we have observed that MPs establish direct and amphipathic interactions with membrane scaffold proteins (MSPs). These interactions have the potential to induce a rearrangement of the MSP-MSP interactions, consequently forming MP-MSP interactions. Through quantitative analysis, we have successfully characterized the significant role played by these interactions in ensuring the overall stability of the proteins.\nA binding site for phosphoinositide modulation of voltage-gated sodium channels described by multiscale simulations\nAuthors: Lin, Y.; Tao, E.; Champion, J. P.; Corry, B.\nScore: 6.9, Published: 2023-07-17 DOI: 10.1101/2023.07.16.547149\nVoltage gated sodium channels (Nav) are membrane proteins which open to facilitate the inward flux of sodium ions into excitable cells. In response to stimuli, Nav channels undergo a transition from the resting, closed state to an open state which allows ion influx, before rapidly inactivating. Dysregulation of this functional cycle due to mutations leads to diseases including epilepsy, pain conditions and cardiac disorders, making Nav channels a significant pharmacological target. Phosphoinositides are important lipid cofactors for ion channel function. The phosphoinositide PI(4,5)P2 decreases Nav1.4 activity by increasing the difficulty of channel opening, accelerating fast activation and slowing recovery from fast inactivation. Using multiscale molecular dynamics simulations, we show that PI(4,5)P2 binds stably to inactivated Nav at a conserved site within the DIV S4-S5 linker, which couples the voltage sensing domain (VSD) to the pore. As the Nav C-terminal domain is proposed to also bind here during recovery from inactivation, we hypothesise that PI(4,5)P2 prolongs inactivation by competing to bind to this site. In atomistic simulations, PI(4,5)P2 reduces the mobility of both the DIV S4-S5 linker and the DIII-IV linker, responsible for fast inactivation, slowing the conformational changes required for the channel to recover to the resting state. We further show that in a resting state Nav model, phosphoinositides bind to VSD gating charges, which may anchor them and impede VSD activation. Our results provide a mechanism by which phosphoinositides alter the voltage dependence of activation and the rate of recovery from inactivation, an important step for the development of novel therapies to treat Nav-related diseases.\nEnzymatic metabolon improves kinetic efficiency of reaction-limited enzyme pathways\nAuthors: Ranganathan, S.; Liu, J.; Shakhnovich, E.\nScore: 5.0, Published: 2023-07-17 DOI: 10.1101/2023.07.17.549414\nIn this work we investigate how spatial proximity of enzymes belonging to the same pathway (metabolon) affects metabolic flux. Using off-lattice Langevin Dynamics (LD) simulations in tandem with a stochastic reaction-diffusion protocol and a semi-analytical reaction-diffusion model, we systematically explored how strength of protein-protein interactions, catalytic efficiency and protein-ligand interactions affect metabolic flux through the metabolon. Formation of a metabolon leads to a greater speed up for longer pathways and especially for reaction-limited enzymes while for fully optimized diffusion-limited enzymes the effect is negligible. Notably, specific cluster architectures are not a prerequisite for enhancing reaction flux. Simulations uncover the crucial role of optimal non-specific protein-ligand interactions in enhancing catalytic efficiency of a metabolon. Our theory implies and bioinformatics analysis confirms that longer catalytic pathways are enriched in less optimal enzymes while most diffusion-limited enzymes populate shorter pathways. Our findings point towards a plausible evolutionary strategy where enzymes compensate for less-than-optimal efficiency by increasing their local concentration in the clustered state.\n",
  "wordCount" : "2049",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549854">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549854" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549854">
        <p class="paperTitle">Coevolution-driven method for efficiently simulating conformational changes in proteins reveals molecular details of ligand effects in the beta2AR receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549854" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549854" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mitrovic, D.; Chen, Y.; Marciniak, A.; Delemotte, L.</p>
        <p class="info">Score: 33.8, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549854' target='https://doi.org/10.1101/2023.07.20.549854'> 10.1101/2023.07.20.549854</a></p>
        <p class="abstract">With the advent of AI-powered structure prediction, the scientific community is inching ever closer to solving protein folding. An unresolved enigma, however, is to accurately, reliably and deterministically predict alternative conformational states that are crucial for the function of e.g. transporters, receptors or ion channels where conformational cycling is innately coupled to protein function. Accurately discovering and exploring all conformational states of membrane proteins has been challenging due to the need to retain atomistic detail while enhancing the sampling along interesting degrees of freedom. The challenges include but are not limited to finding which degrees of freedom are relevant, how to accelerate the sampling along them, and then quantifying the populations of each micro- and macrostate. In this work, we present a methodology that finds the relevant degrees of freedom by combining evolution and physics through machine learning and apply it to the {beta}2 adrenergic receptor conformational sampling. In addition to predicting new conformations that are beyond the training set, we have computed free energy surfaces associated with the proteins conformational landscape. We then show that the methodology is able to quantitatively predict the effect of an array of ligands on the {beta}2 adrenergic receptor activation, and that the full conformational landscape, including states related to biased signaling, is discovered using this procedure. Lastly, we also stake out the structural determinants of activation and inactivation pathway signaling through different ligands.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548133" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548133">
        <p class="paperTitle">Direct optical measurement of intra-molecular distances down to the Angstrom scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.</p>
        <p class="info">Score: 131.2, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548133' target='https://doi.org/10.1101/2023.07.07.548133'> 10.1101/2023.07.07.548133</a></p>
        <p class="abstract">Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision. Our method is validated by quantifying well-characterized 1-to-10 nm distances in polypeptides and proteins. Moreover, we visualize the orientations of immunoglobulin subunits and reveal specific configurations of a histidine kinase domain dimer. Our results open the door for examining proximities and interactions of macromolecules under physiological conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549556">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549556" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549556">
        <p class="paperTitle">Phase-Separated Droplets Swim to Their Dissolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549556" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549556" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jambon-Puillet, E.; Testa, A.; Lorenz, C. M.; Style, R. W.; Rebane, A.; Dufresne, E. R.</p>
        <p class="info">Score: 22.9, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549556' target='https://doi.org/10.1101/2023.07.18.549556'> 10.1101/2023.07.18.549556</a></p>
        <p class="abstract">Biological macromolecules can condense into liquid domains. In cells, these condensates form membraneless organelles that can organize chemical reactions1,2. However, little is known about the physical consequences of chemical activity in and around condensates. Working with model bovine serum albumin (BSA) condensates3, we show that droplets swim along chemical gradients. Active BSA droplets loaded with urease swim toward each other. Passive BSA droplets show diverse responses to externally applied gradients of the enzymes substrate and products. In all these cases, droplets swim toward solvent conditions that favor their dissolution. We call this behavior dialytaxis, and expect it to be generic, as conditions which favor dissolution typically reduce interfacial tension, whose gradients are well-known to drive droplet motion4,5. These results suggest alternative physical mechanisms for active transport in living cells, and may enable the design of fluid micro-robots.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549489">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549489" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549489">
        <p class="paperTitle">Transcription regulates the spatio-temporal dynamics of genes through micro-compartmentalization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549489" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549489" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Salari, H.; Fourel, G.; Jost, D.</p>
        <p class="info">Score: 10.3, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549489' target='https://doi.org/10.1101/2023.07.18.549489'> 10.1101/2023.07.18.549489</a></p>
        <p class="abstract">Although our understanding of the involvement of heterochromatin architectural factors in shaping nuclear organization is improving, there is still ongoing debate regarding the role of active genes in this process. In this study, we utilize publicly-available Micro-C data from mouse embryonic stem cells to investigate the relationship between gene transcription and 3D gene folding. Our analysis uncovers a nonmonotonic - globally positive - correlation between intragenic contact density and Pol II occupancy, independent of cohesin-based loop extrusion. Through the development of a biophysical model integrating the role of transcription dynamics within a polymer model of chromosome organization, we demonstrate that Pol II-mediated attractive interactions with limited valency between transcribed regions yield quantitative predictions consistent with Hi-C and live-imaging experiments. Our work provides compelling evidence that transcriptional activity shapes the 4D genome through Pol II-mediated micro-compartmentalization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.23.525296">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.23.525296" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.23.525296">
        <p class="paperTitle">Protein semisynthesis underscores role of a conserved lysine in activation and desensitization of acid-sensing ion channels</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.23.525296" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.23.525296" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sarkar, D.; Galleano, I.; Heusser, S. A.; Ou, S. Y.; Uzun, G. R.; Khoo, K. K.; van Noort, G. J. v. d. H.; Harrison, J. S.; Pless, S. A.</p>
        <p class="info">Score: 10.7, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.23.525296' target='https://doi.org/10.1101/2023.01.23.525296'> 10.1101/2023.01.23.525296</a></p>
        <p class="abstract">Acid-sensing ion channels (ASICs) are trimeric ion channels that open a cation-conducting pore in response to proton binding. Excessive ASIC activation during prolonged acidosis in conditions such as inflammation and ischemia is linked to pain and stroke. A conserved lysine in the extracellular domain (Lys211 in mASIC1a) is suggested to play a key role in ASIC function. However, the precise contributions are difficult to dissect with conventional mutagenesis, as replacement of Lys211 with naturally occurring amino acids invariably changes multiple physico-chemical parameters. Here, we study the contribution of Lys211 to mASIC1a function using tandem protein-trans splicing (tPTS) to incorporate non-canonical lysine analogs. We conduct optimization efforts to improve splicing and functionally interrogate semisynthetic mASIC1a. In combination with molecular modeling, we show that Lys211 charge and side chain length are crucial to activation and desensitization, thus emphasizing that tPTS can enable atomic-scale interrogations of membrane proteins in live cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550103">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550103" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550103">
        <p class="paperTitle">Folding-upon-binding pathways of an intrinsically disordered protein from a deep Markov state model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550103" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550103" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sisk, T.; Robustelli, P.</p>
        <p class="info">Score: 5.3, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550103' target='https://doi.org/10.1101/2023.07.21.550103'> 10.1101/2023.07.21.550103</a></p>
        <p class="abstract">A central challenge in the study of intrinsically disordered proteins is the characterization of the mechanisms by which they bind their physiological interaction partners. Here, we utilize a deep learning based Markov state modeling approach to characterize the folding-upon-binding pathways observed in a long-time scale molecular dynamics simulation of a disordered region of the measles virus nucleoprotein NTAIL reversibly binding the X domain of the measles virus phosphoprotein complex. We find that folding-upon-binding predominantly occurs via two distinct encounter complexes that are differentiated by the binding orientation, helical content, and conformational heterogeneity of NTAIL. We do not, however, find evidence for the existence of canonical conformational selection or induced fit binding pathways. We observe four kinetically separated native-like bound states that interconvert on time scales of eighty to five hundred nanoseconds. These bound states share a core set of native intermolecular contacts and stable NTAIL helices and are differentiated by a sequential formation of native and non-native contacts and additional helical turns. Our analyses provide an atomic resolution structural description of intermediate states in a folding-upon-binding pathway and elucidate the nature of the kinetic barriers between metastable states in a dynamic and heterogenous, or &#39;fuzzy&#39;, protein complex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547879">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547879" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547879">
        <p class="paperTitle">A Generalizable Scaffold-Based Approach for Structure Determination of RNAs by Cryo-EM</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547879" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547879" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kieft, J. S.; Langeberg, C.</p>
        <p class="info">Score: 16.2, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547879' target='https://doi.org/10.1101/2023.07.06.547879'> 10.1101/2023.07.06.547879</a></p>
        <p class="abstract">Single-particle cryo-electron microscopy (cryo-EM) can reveal the structures of large and often dynamic molecules, but smaller biomolecules remain challenging targets due to their intrinsic low signal to noise ratio. Methods to resolve small proteins have been applied but development of similar approaches for small structured RNA elements have lagged. Here, we present a scaffold-based approach that we used to recover maps of sub-25 kDa RNA domains to 4.5 - 5.0 [A]. While lacking the detail of true high-resolution maps, these are suitable for model building and preliminary structure determination. We demonstrate this method faithfully recovers the structure of several RNA elements of known structure, and it promises to be generalized to other RNAs without disturbing their native fold. This approach may streamline the sample preparation process and reduce the optimization required for data collection. This first-generation scaffold approach provides a system for RNA structure determination by cryo-EM and lays the groundwork for further scaffold optimization to achieve higher resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549795">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549795" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549795">
        <p class="paperTitle">Unraveling membrane protein localization and stabilization in nanodiscs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549795" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549795" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, S.-J.; Koh, Y. H.; Roh, S.-H.</p>
        <p class="info">Score: 4.6, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549795' target='https://doi.org/10.1101/2023.07.20.549795'> 10.1101/2023.07.20.549795</a></p>
        <p class="abstract">Nanodiscs are nanoscale structures consisting of a lipid bilayer surrounded by membrane scaffold proteins (MSPs). They are widely used in the study of membrane proteins (MPs) because they provide a stable lipid environment. However, the precise mechanism governing MP behavior within the nanodisc remains elusive. Here, we examined the cryo-EM structures of various MPs reconstituted in nanodiscs from an electron microscopy database (EMPIAR). By analyzing the heterogeneity and interactions in the nanodiscs, we found that MPs within nanodiscs display a distinct spatial preference toward the edges of the nanodisc shells. Furthermore, we observed that MPs can establish direct, amphipathic interactions with the MSPs, promoting protein stability. These interactions may induce a rearrangement of the MSP-MSP interactions, leading to the formation of MP-MSP interactions Collectively, our study provides structural and biophysical insights into how nanodiscs contribute to MP structural behavior and stability.

SIGNIFICANCEBy thoroughly examining multiple deposited datasets of membrane proteins (MPs) reconstituted in nanodiscs, we have gathered compelling evidence that MPs exhibit a clear spatial inclination toward the periphery of the nanodisc shells. Moreover, we have observed that MPs establish direct and amphipathic interactions with membrane scaffold proteins (MSPs). These interactions have the potential to induce a rearrangement of the MSP-MSP interactions, consequently forming MP-MSP interactions. Through quantitative analysis, we have successfully characterized the significant role played by these interactions in ensuring the overall stability of the proteins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.547149">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.547149" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.547149">
        <p class="paperTitle">A binding site for phosphoinositide modulation of voltage-gated sodium channels described by multiscale simulations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.547149" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.547149" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, Y.; Tao, E.; Champion, J. P.; Corry, B.</p>
        <p class="info">Score: 6.9, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.547149' target='https://doi.org/10.1101/2023.07.16.547149'> 10.1101/2023.07.16.547149</a></p>
        <p class="abstract">Voltage gated sodium channels (Nav) are membrane proteins which open to facilitate the inward flux of sodium ions into excitable cells. In response to stimuli, Nav channels undergo a transition from the resting, closed state to an open state which allows ion influx, before rapidly inactivating. Dysregulation of this functional cycle due to mutations leads to diseases including epilepsy, pain conditions and cardiac disorders, making Nav channels a significant pharmacological target. Phosphoinositides are important lipid cofactors for ion channel function. The phosphoinositide PI(4,5)P2 decreases Nav1.4 activity by increasing the difficulty of channel opening, accelerating fast activation and slowing recovery from fast inactivation. Using multiscale molecular dynamics simulations, we show that PI(4,5)P2 binds stably to inactivated Nav at a conserved site within the DIV S4-S5 linker, which couples the voltage sensing domain (VSD) to the pore. As the Nav C-terminal domain is proposed to also bind here during recovery from inactivation, we hypothesise that PI(4,5)P2 prolongs inactivation by competing to bind to this site. In atomistic simulations, PI(4,5)P2 reduces the mobility of both the DIV S4-S5 linker and the DIII-IV linker, responsible for fast inactivation, slowing the conformational changes required for the channel to recover to the resting state. We further show that in a resting state Nav model, phosphoinositides bind to VSD gating charges, which may anchor them and impede VSD activation. Our results provide a mechanism by which phosphoinositides alter the voltage dependence of activation and the rate of recovery from inactivation, an important step for the development of novel therapies to treat Nav-related diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549414">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549414" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549414">
        <p class="paperTitle">Enzymatic metabolon improves kinetic efficiency of reaction-limited enzyme pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549414" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549414" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ranganathan, S.; Liu, J.; Shakhnovich, E.</p>
        <p class="info">Score: 5.0, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549414' target='https://doi.org/10.1101/2023.07.17.549414'> 10.1101/2023.07.17.549414</a></p>
        <p class="abstract">In this work we investigate how spatial proximity of enzymes belonging to the same pathway (metabolon) affects metabolic flux. Using off-lattice Langevin Dynamics (LD) simulations in tandem with a stochastic reaction-diffusion protocol and a semi-analytical reaction-diffusion model, we systematically explored how strength of protein-protein interactions, catalytic efficiency and protein-ligand interactions affect metabolic flux through the metabolon. Formation of a metabolon leads to a greater speed up for longer pathways and especially for reaction-limited enzymes while for fully optimized diffusion-limited enzymes the effect is negligible. Notably, specific cluster architectures are not a prerequisite for enhancing reaction flux. Simulations uncover the crucial role of optimal non-specific protein-ligand interactions in enhancing catalytic efficiency of a metabolon. Our theory implies and bioinformatics analysis confirms that longer catalytic pathways are enriched in less optimal enzymes while most diffusion-limited enzymes populate shorter pathways. Our findings point towards a plausible evolutionary strategy where enzymes compensate for less-than-optimal efficiency by increasing their local concentration in the clustered state.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
