<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Small LEA proteins as an effective air-water interface protectant for fragile samples during cryo-EM grid plunge freezing
Authors: Abe, K. M.; Lim, C. J.
Score: 48.0, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579238
Sample loss due to air-water interface (AWI) interactions is a significant challenge during cryo-electron microscopy (cryo-EM) sample grid plunge freezing. We report that small Late Embryogenesis Abundant (LEA) proteins, which naturally bind to AWI, can protect samples from AWI damage during plunge freezing.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Small LEA proteins as an effective air-water interface protectant for fragile samples during cryo-EM grid plunge freezing
Authors: Abe, K. M.; Lim, C. J.
Score: 48.0, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579238
Sample loss due to air-water interface (AWI) interactions is a significant challenge during cryo-electron microscopy (cryo-EM) sample grid plunge freezing. We report that small Late Embryogenesis Abundant (LEA) proteins, which naturally bind to AWI, can protect samples from AWI damage during plunge freezing." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-11T10:38:55+00:00" />
<meta property="article:modified_time" content="2024-02-11T10:38:55+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Small LEA proteins as an effective air-water interface protectant for fragile samples during cryo-EM grid plunge freezing
Authors: Abe, K. M.; Lim, C. J.
Score: 48.0, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579238
Sample loss due to air-water interface (AWI) interactions is a significant challenge during cryo-electron microscopy (cryo-EM) sample grid plunge freezing. We report that small Late Embryogenesis Abundant (LEA) proteins, which naturally bind to AWI, can protect samples from AWI damage during plunge freezing."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Small LEA proteins as an effective air-water interface protectant for fragile samples during cryo-EM grid plunge freezing\nAuthors: Abe, K. M.; Lim, C. J.\nScore: 48.0, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579238\nSample loss due to air-water interface (AWI) interactions is a significant challenge during cryo-electron microscopy (cryo-EM) sample grid plunge freezing. We report that small Late Embryogenesis Abundant (LEA) proteins, which naturally bind to AWI, can protect samples from AWI damage during plunge freezing.",
  "keywords": [
    
  ],
  "articleBody": " Small LEA proteins as an effective air-water interface protectant for fragile samples during cryo-EM grid plunge freezing\nAuthors: Abe, K. M.; Lim, C. J.\nScore: 48.0, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579238\nSample loss due to air-water interface (AWI) interactions is a significant challenge during cryo-electron microscopy (cryo-EM) sample grid plunge freezing. We report that small Late Embryogenesis Abundant (LEA) proteins, which naturally bind to AWI, can protect samples from AWI damage during plunge freezing. This protection is demonstrated with two LEA proteins from nematodes and tardigrades, which rescued the cryo-EM structural determination outcome of two fragile multisubunit protein complexes.\nA coarse-grained model for disordered and multi-domain proteins\nAuthors: Cao, F.; von Bülow, S.; Tesei, G.; Lindorff-Larsen, K.\nScore: 14.9, Published: 2024-02-07 DOI: 10.1101/2024.02.03.578735\nMany proteins contain more than one folded domain, and such modular multi-domain proteins help expand the functional repertoire of proteins. Because of their larger size and often substantial dynamics, it may be difficult to characterize the conformational ensembles of multi-domain proteins by simulations. Here, we present a coarse-grained model for multi-domain proteins that is both fast and provides an accurate description of the global conformational properties in solution. We show that the accuracy of a one-bead-per-residue coarse-grained model depends on how the interaction sites in the folded domains are represented. Specifically, we find excessive domain-domain interactions if the interaction sites are located at the position of the C atoms. We also show that if the interaction sites are located at the centre of mass of the residue, we obtain good agreement between simulations and experiments across a wide range of proteins. We then optimize our previously described CALVADOS model using this centre-of-mass representation, and validate the resulting model using independent data. Finally, we use our revised model to simulate phase separation of both disordered and multi-domain proteins. Our results provide a starting point for understanding interactions between folded and disordered regions in proteins, and how these regions affect the propensity of proteins to self-associate and undergo phase separation.\nThe accomplices: Heparan sulfates and N-glycans foster SARS-CoV-2 spike:ACE2 receptor binding and virus priming\nAuthors: Paiardi, G.; Ferraz, M.; Rusnati, M.; Wade, R. C.\nScore: 12.2, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578888\nThe SARS-CoV-2 spike glycoprotein mediates virus attachment to human host cells by binding angiotensin-converting enzyme 2 (ACE2) and heparan sulfate (HS) proteoglycans. To elucidate the structure, dynamics, and functional consequences of these interactions, we carried out microsecond-long all-atom molecular dynamics simulations, followed by random acceleration molecular dynamics simulations, of the fully glycosylated spike:ACE2 complex with and without heparin chains bound. We find that heparin, a model for HS, promotes structural and energetic stabilization of the active conformation of the spike receptor binding domain (RBD) and reorientation of ACE2 toward the N-terminal domain in the same spike subunit as the RBD. Spike and ACE2 N-glycans exert synergistic effects, promoting better packing, strengthening the protein:protein interaction, and prolonging the residence time of the complex. ACE2 and heparin binding trigger rearrangement of the S2 functional site through allosteric interdomain communication. HS thus has a multifaceted role in facilitating SARS-CoV-2 infection.\nEnergetics of the Transmembrane Peptide Sorting by Hydrophobic Mismatch\nAuthors: Fabian, B.; Javanainen, M.\nScore: 9.8, Published: 2024-02-07 DOI: 10.1101/2024.02.02.578561\nHydrophobic mismatch between a lipid membrane and embedded transmembrane peptides or proteins plays a role in their lateral localization and function. Earlier studies have resolved numerous mechanisms through which the peptides and membrane proteins adapt to mismatch, yet the energetics of lateral sorting due to hydrophobic mismatch has remained elusive due to the lack of suitable computational or experimental protocols. Here, we pioneer a molecular dynamics simulation approach to study the sorting of peptides along a membrane thickness gradient. Peptides of different lengths tilt and diffuse along the membrane to eliminate mismatch with a rate directly proportional to the magnitude of mismatch. We extract the 2-dimensional free energy profiles as a function of local thickness and peptide orientation, revealing the relative contributions of sorting and tilting, and suggesting their thermally accessible regimes. Our approach can readily be applied to study other membrane systems of biological interest where hydrophobic mismatch, or membrane thickness in general, plays a role.\nPolymer-extracted structure of the mechanosensitive channel MscS reveals the role of protein-lipid interactions in the gating cycle\nAuthors: Moller, E.; Britt, M.; Zhou, F.; Yang, H.; Anishkin, A.; Ernst, R. K.; Sukharev, S. I.; Matthies, D.\nScore: 8.5, Published: 2024-02-05 DOI: 10.1101/2024.01.22.576751\nMembrane protein structure determination is not only technically challenging but is further complicated by the removal or displacement of lipids, which can result in non-native conformations or a strong preference for certain states at the exclusion of others. This is especially applicable to mechanosensitive channels (MSCs) that evolved to gate in response to subtle changes in membrane tension transmitted through the lipid bilayer. E. coli MscS, a model bacterial system, is an ancestral member of the large family of MSCs found across all phyla of walled organisms. As a tension sensor, MscS is very sensitive and highly adaptive; it readily opens under super-threshold tension and closes under no tension, but under lower tensions, it slowly inactivates and can only recover when tension is released. However, existing cryo-EM structures do not explain the entire functional gating cycle of open, closed, and inactivated states. A central question in the field has been the assignment of the frequently observed non-conductive conformation to either a closed or inactivated state. Here, we present a 3 [A] MscS structure in native nanodiscs obtained with Glyco-DIBMA polymer extraction, eliminating the lipid removal step that is common to all previous structures. Besides the protein in the non-conductive conformation, we observe well-resolved densities of four endogenous phospholipid molecules intercalating between the lipid-facing and pore-lining helices in preferred orientations. Mutations of positively charged residues coordinating these lipids inhibit MscS inactivation, whereas removal of a negative charge near the lipid-filled crevice increases inactivation. The functional data allows us to assign this class of structures to the inactivated state. This structure reveals preserved lipids in their native locations, and the functional effects of their destabilization illustrate a novel inactivation mechanism based on an uncoupling of the peripheral tension-sensing helices from the gate.\nThe mechanism of mammalian proton-coupled peptide transporters\nAuthors: Lichtinger, S. M.; Parker, J. L.; Newstead, S.; Biggin, P.\nScore: 8.2, Published: 2024-02-06 DOI: 10.1101/2024.02.04.578827\nProton-coupled oligopeptide transporters (POTs) are of great pharmaceutical interest owing to their promiscuous substrate binding site that has been linked to improved oral bioavailability of several classes of drugs. Members of the POT family are conserved across all phylogenetic kingdoms and function by coupling peptide uptake to the proton electrochemical gradient. Cryo-EM structures and alphafold models have recently provided new insights into different conformational states of two mammalian POTs, SLC15A1 and SLC15A2. Nevertheless, these studies leave open important questions regarding the mechanism of proton and substrate coupling, while simultaneously providing a unique opportunity to investigate these processes using molecular dynamics (MD) simulations. Here, we employ extensive unbiased and enhanced-sampling MD to map out the full SLC15A2 conformational cycle and its thermodynamic driving forces. By computing conformational free energy landscapes in different protonation states and in the absence or presence of peptide substrate, we identify a likely sequence of intermediate protonation steps that drive inward-directed alternating access. These simulations identify key differences in the extracellular gate between mammalian and bacterial POTs, which we validate experimentally in cell-based transport assays. Our results from constant-PH MD and absolute binding free energy (ABFE) calculations also establish a mechanistic link between proton binding and peptide recognition, revealing key details underpining secondary active transport in POTs. This study provides a vital step forward in understanding proton-coupled peptide and drug transport in mammals and pave the way to integrate knowledge of solute carrier structural biology with enhanced drug design to target tissue and organ bioavailability.\nLigand-induced conformational changes in the β1-Adrenergic Receptor Revealed by Hydrogen-Deuterium Exchange Mass Spectrometry\nAuthors: Toporowska, J.; Kapoor, P.; Musgaard, M.; Gherbi, K.; Sengmany, K.; Qu, F.; Soave, M.; Yen, H.-Y.; Hansen, K.; Jazayeri, A.; Hopper, J. T. S.; Politis, A.\nScore: 6.1, Published: 2024-02-09 DOI: 10.1101/2024.02.07.579309\nG-Protein Coupled Receptors (GPCRs) constitute the largest family of signalling proteins responsible for translating extracellular stimuli into intracellular functions. When dysregulated, GPCRs drive numerous diseases and are the most targeted proteins in drug discovery. GPCR structural dynamics and activity can be modulated by a wide range of drugs, including full/partial agonists and antagonists. While crucial for developing novel therapeutics targeting GPCRs, the structural dynamics of the receptors associated with their activity upon drug interactions are not yet fully understood. Here, we employ Hydrogen Deuterium Exchange Mass Spectrometry (HDX-MS), to characterise the structural dynamics of turkey {beta}1-adrenergic receptor (t{beta}1AR) in complex with nine ligands, including agonists, partial agonists and antagonists. We show that dynamic signatures across the GPCR structure can be grouped by compound modality. Surprisingly, we discovered repeated destabilisation of the intracellular loop 1 (ICL1) upon full agonist binding and stabilisation upon antagonist binding, suggesting that increased dynamics in this region are an essential component for G-protein recruitment. Multiple sequence alignments and molecular dynamics simulations indicate that L72 in ICL1 plays important structural role. Differential HDX-MS experiment of t{beta}1AR and t{beta}1AR L72A construct in complex with miniGs, in response to various ligands, suggests involvement of ICL1 in stabilising the GDP bound state by influencing the stability of HG helix of miniGs. Overall, our results provide a platform for determining drug modality and highlight how HDX-MS can be used to dissect receptor ligand interaction properties and GPCR mechanism.\nSterol-lipids enable large-scale, liquid-liquid phase separation in membranes of only 2 components\nAuthors: Wilson, K. J.; Nguyen, H. Q.; Gervay-Hague, J.; Keller, S. L.\nScore: 4.2, Published: 2024-02-08 DOI: 10.1101/2024.02.02.578692\nDespite longstanding excitement and progress toward understanding liquid-liquid phase separation in natural and artificial membranes, fundamental questions have persisted about which molecules are required for this phenomenon. Except in extraordinary circumstances, the smallest number of components that has produced large-scale, liquid-liquid phase separation in bilayers has stubbornly remained at three: a sterol, a phospholipid with ordered chains, and a phospholipid with disordered chains. This requirement of three components is puzzling for two reasons: (1) the Gibbs Phase Rule states that only two components are necessary, and (2) only two components are required for liquid-liquid phase separation in lipid monolayers, which resemble half of a bilayer. Inspired by reports that sterols interact closely with lipids with ordered chains, we tested whether phase separation would occur in bilayers in which a sterol and lipid were replaced by a single, joined sterol-lipid. By evaluating a panel of sterol-lipids, we discovered a minimal bilayer of only two components (PChemsPC and diPhyPC) that demixes into micron-scale, liquid phases. In this system, the sterol-lipid behaves as a 3:1 ratio of cholesterol to phospholipid. Our system gives the computation and theory community a two-component membrane that maps directly onto simplified theories and that can be used to validate simulation force fields. It suggests a new role for sterol-lipids in nature, and it gives experimental communities a membrane in which tie-lines (and, therefore, the lipid composition of each phase) are easily determined and will be consistent across multiple laboratories.\nEpithelial folding through local degradation of an elastic basement membrane plate\nAuthors: Guerra Santillan, K. Y.; Jantzen, C.; Dahmann, C.; Fischer-Friedrich, E.\nScore: 4.2, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579060\nEpithelia are polarised layers of cells that line the outer and inner surfaces of organs. At the basal side, the epithelial cell layer is supported by a basement membrane, which is a thin polymeric layer of self-assembled extracellular matrix (ECM) that tightly adheres to the basal cell surface. Proper shaping of epithelial layers is an important prerequisite for the development of healthy organs during the morphogenesis of an organism. Experimental evidence indicates that local degradation of the basement membrane drives epithelial folding. Here, we present a coarse-grained plate theory model of the basement membrane that assumes force balance between i) cell-transduced active forces and ii) deformation-induced elastic forces. We verify key assumptions of this model through experiments in the Drosophila wing disc epithelium and demonstrate that the model can explain the emergence of outward epithelial folds upon local plate degradation. Our model accounts for local degradation of the basement membrane as a mechanism for the generation of epithelial folds in the absence of epithelial growth.\nEmergent collective alignment gives competitive advantage to longer cells during range expansion\nAuthors: van den Berg, N.; Thijssen, K.; Nguyen, T. T.; Sarlet, A.; Cordero, M.; Vazquez, A. G.; Mitarai, N.; Doostmohammadi, A.; Jauffred, L.\nScore: 6.0, Published: 2024-02-09 DOI: 10.1101/2024.01.26.577059\nBacteria's competition for nutrients and territory drives biofilm evolution (1-4). The factors determining the outcome of competition among diverse bacterial species have a broad impact on a wide range of pathological (5), environmental (6), and microbiome interactions (7). While motility-related traits (8-11) and specific molecular mechanisms (12, 13) have been identified as potential winning attributes in bacteria, a shared and universally conserved feature determining competition remains elusive. Here, we demonstrate that a simple morphological feature of individual bacteria, cell aspect ratio, provides a winning trait for the population. Using range expansion experiments (14), we show that relatively longer bacteria robustly conquer the expanding front, even when initially in minority. Using an agent-based model of dividing bacteria, we reveal that the takeover mechanism is their emergent collective alignment: groups of locally aligned bacteria form 'nematic arms' bridging the central region of the colony to the expanding front. Once at the front, bacteria align parallel to it and block the access of shorter bacteria to nutrients and space. We confirm this observation with single-cell experiments and further generalise our findings by introducing a generic continuum model of alignment dominated competition, explaining both experimental and cell based model observations. Moreover, we extend our predictions to spherical range expansions (15) and confirm the competitive advantage of being longer, even though the effect is less pronounced than in surface attached colonies. Our results uncover a simple, yet hitherto overlooked, mechanical mechanism determining the outcome of bacterial competition, which is potentially ubiquitous among various bacteria. With the current advances in genetic engineering, varying aspect ratios can work as a simple tunable mechanism for the on demand setting of the outcome of bacterial competitions with widespread implications for biofilm control.\n",
  "wordCount" : "2359",
  "inLanguage": "en",
  "datePublished": "2024-02-11T10:38:55Z",
  "dateModified": "2024-02-11T10:38:55Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 11, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579238">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579238" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579238">
        <p class="paperTitle">Small LEA proteins as an effective air-water interface protectant for fragile samples during cryo-EM grid plunge freezing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579238" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579238" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abe, K. M.; Lim, C. J.</p>
        <p class="info">Score: 48.0, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579238' target='https://doi.org/10.1101/2024.02.06.579238'> 10.1101/2024.02.06.579238</a></p>
        <p class="abstract">Sample loss due to air-water interface (AWI) interactions is a significant challenge during cryo-electron microscopy (cryo-EM) sample grid plunge freezing. We report that small Late Embryogenesis Abundant (LEA) proteins, which naturally bind to AWI, can protect samples from AWI damage during plunge freezing. This protection is demonstrated with two LEA proteins from nematodes and tardigrades, which rescued the cryo-EM structural determination outcome of two fragile multisubunit protein complexes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.03.578735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.03.578735" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.03.578735">
        <p class="paperTitle">A coarse-grained model for disordered and multi-domain proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.03.578735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.03.578735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cao, F.; von Bülow, S.; Tesei, G.; Lindorff-Larsen, K.</p>
        <p class="info">Score: 14.9, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.03.578735' target='https://doi.org/10.1101/2024.02.03.578735'> 10.1101/2024.02.03.578735</a></p>
        <p class="abstract">Many proteins contain more than one folded domain, and such modular multi-domain proteins help expand the functional repertoire of proteins. Because of their larger size and often substantial dynamics, it may be difficult to characterize the conformational ensembles of multi-domain proteins by simulations. Here, we present a coarse-grained model for multi-domain proteins that is both fast and provides an accurate description of the global conformational properties in solution. We show that the accuracy of a one-bead-per-residue coarse-grained model depends on how the interaction sites in the folded domains are represented. Specifically, we find excessive domain-domain interactions if the interaction sites are located at the position of the C atoms. We also show that if the interaction sites are located at the centre of mass of the residue, we obtain good agreement between simulations and experiments across a wide range of proteins. We then optimize our previously described CALVADOS model using this centre-of-mass representation, and validate the resulting model using independent data. Finally, we use our revised model to simulate phase separation of both disordered and multi-domain proteins. Our results provide a starting point for understanding interactions between folded and disordered regions in proteins, and how these regions affect the propensity of proteins to self-associate and undergo phase separation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578888">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578888" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578888">
        <p class="paperTitle">The accomplices: Heparan sulfates and N-glycans foster SARS-CoV-2 spike:ACE2 receptor binding and virus priming</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578888" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578888" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paiardi, G.; Ferraz, M.; Rusnati, M.; Wade, R. C.</p>
        <p class="info">Score: 12.2, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578888' target='https://doi.org/10.1101/2024.02.05.578888'> 10.1101/2024.02.05.578888</a></p>
        <p class="abstract">The SARS-CoV-2 spike glycoprotein mediates virus attachment to human host cells by binding angiotensin-converting enzyme 2 (ACE2) and heparan sulfate (HS) proteoglycans. To elucidate the structure, dynamics, and functional consequences of these interactions, we carried out microsecond-long all-atom molecular dynamics simulations, followed by random acceleration molecular dynamics simulations, of the fully glycosylated spike:ACE2 complex with and without heparin chains bound. We find that heparin, a model for HS, promotes structural and energetic stabilization of the active conformation of the spike receptor binding domain (RBD) and reorientation of ACE2 toward the N-terminal domain in the same spike subunit as the RBD. Spike and ACE2 N-glycans exert synergistic effects, promoting better packing, strengthening the protein:protein interaction, and prolonging the residence time of the complex. ACE2 and heparin binding trigger rearrangement of the S2 functional site through allosteric interdomain communication. HS thus has a multifaceted role in facilitating SARS-CoV-2 infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578561">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578561" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578561">
        <p class="paperTitle">Energetics of the Transmembrane Peptide Sorting by Hydrophobic Mismatch</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578561" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578561" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fabian, B.; Javanainen, M.</p>
        <p class="info">Score: 9.8, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578561' target='https://doi.org/10.1101/2024.02.02.578561'> 10.1101/2024.02.02.578561</a></p>
        <p class="abstract">Hydrophobic mismatch between a lipid membrane and embedded transmembrane peptides or proteins plays a role in their lateral localization and function. Earlier studies have resolved numerous mechanisms through which the peptides and membrane proteins adapt to mismatch, yet the energetics of lateral sorting due to hydrophobic mismatch has remained elusive due to the lack of suitable computational or experimental protocols. Here, we pioneer a molecular dynamics simulation approach to study the sorting of peptides along a membrane thickness gradient. Peptides of different lengths tilt and diffuse along the membrane to eliminate mismatch with a rate directly proportional to the magnitude of mismatch. We extract the 2-dimensional free energy profiles as a function of local thickness and peptide orientation, revealing the relative contributions of sorting and tilting, and suggesting their thermally accessible regimes. Our approach can readily be applied to study other membrane systems of biological interest where hydrophobic mismatch, or membrane thickness in general, plays a role.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576751">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576751" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576751">
        <p class="paperTitle">Polymer-extracted structure of the mechanosensitive channel MscS reveals the role of protein-lipid interactions in the gating cycle</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576751" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576751" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moller, E.; Britt, M.; Zhou, F.; Yang, H.; Anishkin, A.; Ernst, R. K.; Sukharev, S. I.; Matthies, D.</p>
        <p class="info">Score: 8.5, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576751' target='https://doi.org/10.1101/2024.01.22.576751'> 10.1101/2024.01.22.576751</a></p>
        <p class="abstract">Membrane protein structure determination is not only technically challenging but is further complicated by the removal or displacement of lipids, which can result in non-native conformations or a strong preference for certain states at the exclusion of others. This is especially applicable to mechanosensitive channels (MSCs) that evolved to gate in response to subtle changes in membrane tension transmitted through the lipid bilayer. E. coli MscS, a model bacterial system, is an ancestral member of the large family of MSCs found across all phyla of walled organisms. As a tension sensor, MscS is very sensitive and highly adaptive; it readily opens under super-threshold tension and closes under no tension, but under lower tensions, it slowly inactivates and can only recover when tension is released. However, existing cryo-EM structures do not explain the entire functional gating cycle of open, closed, and inactivated states. A central question in the field has been the assignment of the frequently observed non-conductive conformation to either a closed or inactivated state. Here, we present a 3 [A] MscS structure in native nanodiscs obtained with Glyco-DIBMA polymer extraction, eliminating the lipid removal step that is common to all previous structures. Besides the protein in the non-conductive conformation, we observe well-resolved densities of four endogenous phospholipid molecules intercalating between the lipid-facing and pore-lining helices in preferred orientations. Mutations of positively charged residues coordinating these lipids inhibit MscS inactivation, whereas removal of a negative charge near the lipid-filled crevice increases inactivation. The functional data allows us to assign this class of structures to the inactivated state. This structure reveals preserved lipids in their native locations, and the functional effects of their destabilization illustrate a novel inactivation mechanism based on an uncoupling of the peripheral tension-sensing helices from the gate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.04.578827">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.04.578827" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.04.578827">
        <p class="paperTitle">The mechanism of mammalian proton-coupled peptide transporters</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.04.578827" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.04.578827" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lichtinger, S. M.; Parker, J. L.; Newstead, S.; Biggin, P.</p>
        <p class="info">Score: 8.2, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.04.578827' target='https://doi.org/10.1101/2024.02.04.578827'> 10.1101/2024.02.04.578827</a></p>
        <p class="abstract">Proton-coupled oligopeptide transporters (POTs) are of great pharmaceutical interest owing to their promiscuous substrate binding site that has been linked to improved oral bioavailability of several classes of drugs. Members of the POT family are conserved across all phylogenetic kingdoms and function by coupling peptide uptake to the proton electrochemical gradient. Cryo-EM structures and alphafold models have recently provided new insights into different conformational states of two mammalian POTs, SLC15A1 and SLC15A2. Nevertheless, these studies leave open important questions regarding the mechanism of proton and substrate coupling, while simultaneously providing a unique opportunity to investigate these processes using molecular dynamics (MD) simulations. Here, we employ extensive unbiased and enhanced-sampling MD to map out the full SLC15A2 conformational cycle and its thermodynamic driving forces. By computing conformational free energy landscapes in different protonation states and in the absence or presence of peptide substrate, we identify a likely sequence of intermediate protonation steps that drive inward-directed alternating access. These simulations identify key differences in the extracellular gate between mammalian and bacterial POTs, which we validate experimentally in cell-based transport assays. Our results from constant-PH MD and absolute binding free energy (ABFE) calculations also establish a mechanistic link between proton binding and peptide recognition, revealing key details underpining secondary active transport in POTs. This study provides a vital step forward in understanding proton-coupled peptide and drug transport in mammals and pave the way to integrate knowledge of solute carrier structural biology with enhanced drug design to target tissue and organ bioavailability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579309">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579309" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579309">
        <p class="paperTitle">Ligand-induced conformational changes in the β1-Adrenergic Receptor Revealed by Hydrogen-Deuterium Exchange Mass Spectrometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579309" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579309" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Toporowska, J.; Kapoor, P.; Musgaard, M.; Gherbi, K.; Sengmany, K.; Qu, F.; Soave, M.; Yen, H.-Y.; Hansen, K.; Jazayeri, A.; Hopper, J. T. S.; Politis, A.</p>
        <p class="info">Score: 6.1, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579309' target='https://doi.org/10.1101/2024.02.07.579309'> 10.1101/2024.02.07.579309</a></p>
        <p class="abstract">G-Protein Coupled Receptors (GPCRs) constitute the largest family of signalling proteins responsible for translating extracellular stimuli into intracellular functions. When dysregulated, GPCRs drive numerous diseases and are the most targeted proteins in drug discovery. GPCR structural dynamics and activity can be modulated by a wide range of drugs, including full/partial agonists and antagonists. While crucial for developing novel therapeutics targeting GPCRs, the structural dynamics of the receptors associated with their activity upon drug interactions are not yet fully understood. Here, we employ Hydrogen Deuterium Exchange Mass Spectrometry (HDX-MS), to characterise the structural dynamics of turkey {beta}1-adrenergic receptor (t{beta}1AR) in complex with nine ligands, including agonists, partial agonists and antagonists. We show that dynamic signatures across the GPCR structure can be grouped by compound modality. Surprisingly, we discovered repeated destabilisation of the intracellular loop 1 (ICL1) upon full agonist binding and stabilisation upon antagonist binding, suggesting that increased dynamics in this region are an essential component for G-protein recruitment. Multiple sequence alignments and molecular dynamics simulations indicate that L72 in ICL1 plays important structural role. Differential HDX-MS experiment of t{beta}1AR and t{beta}1AR L72A construct in complex with miniGs, in response to various ligands, suggests involvement of ICL1 in stabilising the GDP bound state by influencing the stability of HG helix of miniGs. Overall, our results provide a platform for determining drug modality and highlight how HDX-MS can be used to dissect receptor ligand interaction properties and GPCR mechanism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578692">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578692" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578692">
        <p class="paperTitle">Sterol-lipids enable large-scale, liquid-liquid phase separation in membranes of only 2 components</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578692" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578692" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wilson, K. J.; Nguyen, H. Q.; Gervay-Hague, J.; Keller, S. L.</p>
        <p class="info">Score: 4.2, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578692' target='https://doi.org/10.1101/2024.02.02.578692'> 10.1101/2024.02.02.578692</a></p>
        <p class="abstract">Despite longstanding excitement and progress toward understanding liquid-liquid phase separation in natural and artificial membranes, fundamental questions have persisted about which molecules are required for this phenomenon. Except in extraordinary circumstances, the smallest number of components that has produced large-scale, liquid-liquid phase separation in bilayers has stubbornly remained at three: a sterol, a phospholipid with ordered chains, and a phospholipid with disordered chains. This requirement of three components is puzzling for two reasons: (1) the Gibbs Phase Rule states that only two components are necessary, and (2) only two components are required for liquid-liquid phase separation in lipid monolayers, which resemble half of a bilayer. Inspired by reports that sterols interact closely with lipids with ordered chains, we tested whether phase separation would occur in bilayers in which a sterol and lipid were replaced by a single, joined sterol-lipid. By evaluating a panel of sterol-lipids, we discovered a minimal bilayer of only two components (PChemsPC and diPhyPC) that demixes into micron-scale, liquid phases. In this system, the sterol-lipid behaves as a 3:1 ratio of cholesterol to phospholipid. Our system gives the computation and theory community a two-component membrane that maps directly onto simplified theories and that can be used to validate simulation force fields. It suggests a new role for sterol-lipids in nature, and it gives experimental communities a membrane in which tie-lines (and, therefore, the lipid composition of each phase) are easily determined and will be consistent across multiple laboratories.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579060">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579060" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579060">
        <p class="paperTitle">Epithelial folding through local degradation of an elastic basement membrane plate</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579060" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579060" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guerra Santillan, K. Y.; Jantzen, C.; Dahmann, C.; Fischer-Friedrich, E.</p>
        <p class="info">Score: 4.2, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579060' target='https://doi.org/10.1101/2024.02.06.579060'> 10.1101/2024.02.06.579060</a></p>
        <p class="abstract">Epithelia are polarised layers of cells that line the outer and inner surfaces of organs. At the basal side, the epithelial cell layer is supported by a basement membrane, which is a thin polymeric layer of self-assembled extracellular matrix (ECM) that tightly adheres to the basal cell surface. Proper shaping of epithelial layers is an important prerequisite for the development of healthy organs during the morphogenesis of an organism. Experimental evidence indicates that local degradation of the basement membrane drives epithelial folding. Here, we present a coarse-grained plate theory model of the basement membrane that assumes force balance between i) cell-transduced active forces and ii) deformation-induced elastic forces. We verify key assumptions of this model through experiments in the Drosophila wing disc epithelium and demonstrate that the model can explain the emergence of outward epithelial folds upon local plate degradation. Our model accounts for local degradation of the basement membrane as a mechanism for the generation of epithelial folds in the absence of epithelial growth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577059">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577059" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577059">
        <p class="paperTitle">Emergent collective alignment gives competitive advantage to longer cells during range expansion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577059" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577059" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: van den Berg, N.; Thijssen, K.; Nguyen, T. T.; Sarlet, A.; Cordero, M.; Vazquez, A. G.; Mitarai, N.; Doostmohammadi, A.; Jauffred, L.</p>
        <p class="info">Score: 6.0, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577059' target='https://doi.org/10.1101/2024.01.26.577059'> 10.1101/2024.01.26.577059</a></p>
        <p class="abstract">Bacteria&#39;s competition for nutrients and territory drives biofilm evolution (1-4). The factors determining the outcome of competition among diverse bacterial species have a broad impact on a wide range of pathological (5), environmental (6), and microbiome interactions (7). While motility-related traits (8-11) and specific molecular mechanisms (12, 13) have been identified as potential winning attributes in bacteria, a shared and universally conserved feature determining competition remains elusive. Here, we demonstrate that a simple morphological feature of individual bacteria, cell aspect ratio, provides a winning trait for the population. Using range expansion experiments (14), we show that relatively longer bacteria robustly conquer the expanding front, even when initially in minority. Using an agent-based model of dividing bacteria, we reveal that the takeover mechanism is their emergent collective alignment: groups of locally aligned bacteria form &#39;nematic arms&#39; bridging the central region of the colony to the expanding front. Once at the front, bacteria align parallel to it and block the access of shorter bacteria to nutrients and space. We confirm this observation with single-cell experiments and further generalise our findings by introducing a generic continuum model of alignment dominated competition, explaining both experimental and cell based model observations. Moreover, we extend our predictions to spherical range expansions (15) and confirm the competitive advantage of being longer, even though the effect is less pronounced than in surface attached colonies. Our results uncover a simple, yet hitherto overlooked, mechanical mechanism determining the outcome of bacterial competition, which is potentially ubiquitous among various bacteria. With the current advances in genetic engineering, varying aspect ratios can work as a simple tunable mechanism for the on demand setting of the outcome of bacterial competitions with widespread implications for biofilm control.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
