<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Adhesion energy controls lipid binding-mediated endocytosis
Authors: Groza, R.; Schmidt, K. V.; Muller, P. M.; Ronchi, P.; Schlack-Leigers, C.; Neu, U.; Puchkov, D.; Dimova, R.; Matthaus, C.; Taraska, J. W.; Weikl, T.; Ewers, H.
Score: 22.5, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546235
Several bacterial toxins and viruses can deform membranes through multivalent binding to lipids for clathrin-independent endocytosis. However, it remains unclear, how membrane deformation and endocytic internalization are mechanistically linked. Here we show that many lipid-binding virions induce membrane deformation and clathrin-independent endocytosis, suggesting a common mechanism based on multivalent lipid binding by globular particles.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Adhesion energy controls lipid binding-mediated endocytosis
Authors: Groza, R.; Schmidt, K. V.; Muller, P. M.; Ronchi, P.; Schlack-Leigers, C.; Neu, U.; Puchkov, D.; Dimova, R.; Matthaus, C.; Taraska, J. W.; Weikl, T.; Ewers, H.
Score: 22.5, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546235
Several bacterial toxins and viruses can deform membranes through multivalent binding to lipids for clathrin-independent endocytosis. However, it remains unclear, how membrane deformation and endocytic internalization are mechanistically linked. Here we show that many lipid-binding virions induce membrane deformation and clathrin-independent endocytosis, suggesting a common mechanism based on multivalent lipid binding by globular particles." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T11:45:43+00:00" />
<meta property="article:modified_time" content="2023-06-28T11:45:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Adhesion energy controls lipid binding-mediated endocytosis
Authors: Groza, R.; Schmidt, K. V.; Muller, P. M.; Ronchi, P.; Schlack-Leigers, C.; Neu, U.; Puchkov, D.; Dimova, R.; Matthaus, C.; Taraska, J. W.; Weikl, T.; Ewers, H.
Score: 22.5, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546235
Several bacterial toxins and viruses can deform membranes through multivalent binding to lipids for clathrin-independent endocytosis. However, it remains unclear, how membrane deformation and endocytic internalization are mechanistically linked. Here we show that many lipid-binding virions induce membrane deformation and clathrin-independent endocytosis, suggesting a common mechanism based on multivalent lipid binding by globular particles."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Adhesion energy controls lipid binding-mediated endocytosis\nAuthors: Groza, R.; Schmidt, K. V.; Muller, P. M.; Ronchi, P.; Schlack-Leigers, C.; Neu, U.; Puchkov, D.; Dimova, R.; Matthaus, C.; Taraska, J. W.; Weikl, T.; Ewers, H.\nScore: 22.5, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546235\nSeveral bacterial toxins and viruses can deform membranes through multivalent binding to lipids for clathrin-independent endocytosis. However, it remains unclear, how membrane deformation and endocytic internalization are mechanistically linked. Here we show that many lipid-binding virions induce membrane deformation and clathrin-independent endocytosis, suggesting a common mechanism based on multivalent lipid binding by globular particles.",
  "keywords": [
    
  ],
  "articleBody": " Adhesion energy controls lipid binding-mediated endocytosis\nAuthors: Groza, R.; Schmidt, K. V.; Muller, P. M.; Ronchi, P.; Schlack-Leigers, C.; Neu, U.; Puchkov, D.; Dimova, R.; Matthaus, C.; Taraska, J. W.; Weikl, T.; Ewers, H.\nScore: 22.5, Published: 2023-06-25 DOI: 10.1101/2023.06.23.546235\nSeveral bacterial toxins and viruses can deform membranes through multivalent binding to lipids for clathrin-independent endocytosis. However, it remains unclear, how membrane deformation and endocytic internalization are mechanistically linked. Here we show that many lipid-binding virions induce membrane deformation and clathrin-independent endocytosis, suggesting a common mechanism based on multivalent lipid binding by globular particles. We create a synthetic cellular system consisting of a lipid-anchored receptor in the form of GPI-anchored anti-GFP nanobodies and a multivalent globular binder exposing 180 regularly-spaced GFP molecules on its surface. We show that these globular, 40 nm diameter, particles bind to cells expressing the receptor, deform the plasma membrane upon adhesion and become endocytosed in a clathrin-independent manner. We explore the role of the membrane adhesion energy in endocytosis by using receptors with affinities varying over 7 orders of magnitude. Using this system, we find that once a threshold in adhesion energy is overcome to allow for membrane deformation, endocytosis occurs reliably. Multivalent, binding-induced membrane deformation by globular binders is thus sufficient for internalization to occur and we suggest it is the common, purely biophysical mechanism for lipid-binding mediated endocytosis of toxins and pathogens.\nDynamic ParB-DNA interactions initiate and maintain a partition condensate for bacterial chromosome segregation\nAuthors: Tisma, M.; Janissen, R.; Antar, H.; Martin Gonzalez, A.; Barth, R.; Beekman, T.; van der Torre, J.; Michieletto, D.; Gruber, S.; Dekker, C.\nScore: 19.1, Published: 2023-06-25 DOI: 10.1101/2023.06.25.546419\nIn most bacteria, chromosome segregation is driven by the ParABS system where the CTPase protein ParB loads at the parS site to trigger the formation of a large partition complex. Here, we present in vitro studies of the partition complex for Bacillus subtilis ParB, using single-molecule fluorescence microscopy and AFM imaging to show that transient ParB-ParB bridges are essential for forming DNA condensates. Molecular Dynamics simulations confirm that condensation occurs abruptly at a critical concentration of ParB and show that multimerization is a prerequisite for forming the partition complex. Magnetic tweezer force spectroscopy on mutant ParB proteins demonstrates that CTP hydrolysis at the N-terminal domain is essential for DNA condensation. Finally, we show that transcribing RNA polymerases can steadily traverse the ParB-DNA partition complex. These findings uncover how ParB forms a stable yet dynamic partition complex for chromosome segregation that induces DNA condensation and segregation while enabling replication and transcription.\nStudying stochastic systems biology of the cell with single-cell genomics data\nAuthors: Gorin, G.; Vastola, J. J.; Pachter, L.\nScore: 24.7, Published: 2023-05-29 DOI: 10.1101/2023.05.17.541250\nRecent experimental developments in genome-wide RNA quantification hold considerable promise for systems biology. However, rigorously probing the biology of living cells requires a unified mathematical framework that accounts for single-molecule biological stochasticity in the context of technical variation associated with genomics assays. We review models for a variety of RNA transcription processes, as well as the encapsulation and library construction steps of microfluidics-based single-cell RNA sequencing, and present a framework to integrate these phenomena by the manipulation of generating functions. Finally, we use simulated scenarios and biological data to illustrate the implications and applications of the approach.\nDirect observation of autoubiquitination for an integral membrane ubiquitin ligase in ERAD\nAuthors: Moochickal Assainar, B.; Ragunathan, K.; Baldridge, R.\nScore: 11.5, Published: 2023-06-22 DOI: 10.1101/2023.06.20.545802\nThe endoplasmic reticulum associated degradation (ERAD) pathway regulates protein quality control at the endoplasmic reticulum. ERAD of lumenal and membrane proteins requires a conserved E3 ubiquitin ligase, called Hrd1. We do not understand the molecular configurations of Hrd1 that enable autoubiquitination and the subsequent retrotranslocation of misfolded protein substrates from the ER to the cytosol. Here, we have established a generalizable, single-molecule platform that enables high-efficiency labeling, stoichiometry determination, and functional assays for any integral membrane protein. Using this approach, we directly count Hrd1 proteins reconstituted into individual proteoliposomes. We found that Hrd1 assembles in different oligomeric configurations with mostly monomers and dimers detected at limiting dilution. By correlating oligomeric states with ubiquitination in vitro, we determined that Hrd1 monomers were incapable of autoubiquitination while dimers efficiently assembled polyubiquitin chains. Therefore, our results reveal the minimal composition of a Hrd1 oligomer that is capable of autoubiquitination. Our methods are broadly applicable to studying other complex membrane protein functions using reconstituted bilayer systems.\nAssembly of tight junction belts by surface condensation and actin elongation\nAuthors: Sun, D.; Zhao, X.; Wiegand, T.; Bartolucci, G.; Martin-Lemaitre, C.; Grill, S. W.; Hyman, A. A.; Weber, C.; Honigmann, A.\nScore: 11.1, Published: 2023-06-26 DOI: 10.1101/2023.06.24.546380\nFormation of biomolecular condensates via phase separation enables compartmentation of many cellular processes. However, how cells can control condensation at specific locations to create complex cellular structures remains poorly understood. Here, we investigated the mechanism of tight junction formation, which involves condensation of scaffold proteins at cell-cell contacts and elongation of the condensates into a belt around the cellular perimeter. Using cell biology, reconstitution, and thermodynamic theory, we discovered that cells use surface phase transitions to control local condensation at the membrane far below bulk saturation. Surface condensation of junctional ZO-scaffold proteins is mediated by receptor binding and regulated by the receptor's oligomerization state. Functionally, ZO surface condensation is directly coupled to actin polymerization and bundling, which drives elongation of receptor-ZO-actin condensates similar to tight junction belt formation in cells. We conclude that surface phase transitions provide a robust mechanism to control the position and shape of protein condensates.\nMechanism of DNA origami folding elucidated by mesoscopic simulations\nAuthors: DeLuca, M.; Ye, T.; Poirier, M. G.; Ke, Y.; Castro, C.; Arya, G.\nScore: 8.5, Published: 2023-06-22 DOI: 10.1101/2023.06.20.545758\nDNA nanotechnology leverages the canonical base-pairing rules and geometry of DNA to create highly precise nanoscale structures with many potential applications. While the design and fabrication of DNA nanostructures is well-established, the self-assembly process that produces these structures is still poorly understood, especially for DNA origami that involve the assembly of hundreds of strands. Many experimental and computational efforts have sought to better understand DNA origami folding, but the small length and time scales of individual binding events and the long timescale over which folding occurs have posed significant challenges. Here, we present a new mesoscopic model that uses a switchable force field to capture the mechanical behavior of single- and double-stranded DNA motifs and transition between them at a coarseness level of up to 8 nucleotides per particle, allowing access to the long assembly timescales of DNA origami up to several kilobases in size. Brownian dynamics simulations of 4-helix bundle (4HB) structures using this model reveal a hierarchical folding process involving the zipping of structural domains into a partially folded precursor structure followed by gradual crystallization into the final structure. We elucidate the role of hybridization strength, scaffold routing, and staple design in the folding order and kinetics. Simulation of larger 32HB structures reveals heterogeneous staple incorporation kinetics and frequent trapping in metastable states, as opposed to smaller, more accessible structures like the 4HB, which exhibit first-order kinetics and virtually defect-free folding. The development of this model opens an avenue to better understand and design DNA nanostructures for improved yield and folding performance.\nLoops are Geometric Catalysts for DNA Integration\nAuthors: Battaglia, C.; Michieletto, D.\nScore: 8.4, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545982\nThe insertion of HIV and other DNA elements within genomes underpins both genetic diversity and disease when unregulated. Most of these insertions are not random and occupy specific positions within the genome but the physical mechanisms underlying the integration site selection are poorly understood. Here we perform Molecular Dynamics simulations to study the insertion of DNA elements, such as HIV viral DNA or transposons, into naked DNA or chromatin substrate. More specifically, we explore the role of loops in the DNA substrate and discover that they act as \"geometric catalysts\" for DNA integration. Additionally, we discover that the 1D and 3D clustering of loops affects the distribution of integration sites. Finally, we show that loops may compete with nucleosomes at attracting DNA integrations. These results may be tested in vitro and they may help to understand patterns of DNA insertions with implications in genome evolution and gene therapy.\nLabelizer: systematic selection of protein residues for covalent fluorophore labeling\nAuthors: Gebhardt, C.; Bawidamann, P.; Schuetze, K.; Moya Munoz, G. G.; Spring, A.-K.; Griffith, D.; Lipfert, J.; Cordes, T.\nScore: 56.9, Published: 2023-06-12 DOI: 10.1101/2023.06.12.544586\nAn essential requirement for the use of fluorescent dyes in biomedicine, molecular biology, biochemistry, biophysics and optical imaging is their (covalent) attachment to biomolecules. There is, however, no systematic and automated approach for the selection of suitable labeling sites in macromolecules, which is particular problematic for proteins. Here, we present a general and quantitative strategy to identify optimal residues for protein labeling using a naive Bayes classifier. Based on a literature search and bioinformatics analysis of \u003e100 proteins with [~]400 successfully labeled residues, we identified four parameters, which we combined into a labeling score to rank residues for their suitability as a label-site. The utility of our approach for the systematic selection of single residues and residue pairs for FRET experiments is supported by data from the literature and by new experiments on different proteins. To make the method available to a large community of researchers, we developed a python package called \"labelizer\", that performs an analysis of a pdb-structure (or structural models), label score calculation, and FRET assay scoring. We further provide a webserver (https://labelizer.bio.lmu.de/) to conveniently apply our approach and to build up a central open-access database of (non-)successfully labeled protein residues to continuously improve and refine the labelizer approach.\nLiquid-like condensates mediate competition between actin branching and bundling\nAuthors: Graham, K.; Chandrasekaran, A.; Wang, L.; Yang, N.; Lafer, E. M.; Rangamani, P.; Stachowiak, J. C.\nScore: 8.0, Published: 2023-06-26 DOI: 10.1101/2023.06.23.546267\nCellular remodeling of actin networks underlies cell motility during key morphological events, from embryogenesis to metastasis. In these transformations there is an inherent competition between actin branching and bundling, because steric clashes among branches create a mechanical barrier to bundling. Recently, liquid-like condensates consisting purely of proteins involved in either branching or bundling of the cytoskeleton have been found to catalyze their respective functions. Yet in the cell, proteins that drive branching and bundling are present simultaneously. In this complex environment, which factors determine whether a condensate drives filaments to branch versus becoming bundled? To answer this question, we added the branched actin nucleator, Arp2/3, to condensates composed of VASP, an actin bundling protein. At low actin to VASP ratios, branching activity, mediated by Arp2/3, robustly inhibited VASP-mediated bundling of filaments, in agreement with agent-based simulations. In contrast, as the actin to VASP ratio increased, addition of Arp2/3 led to formation of aster-shaped structures, in which bundled filaments emerged from a branched actin core, analogous to filopodia emerging from a branched lamellipodial network. These results demonstrate that multi-component, liquid-like condensates can modulate the inherent competition between bundled and branched actin morphologies, leading to organized, higher-order structures, similar to those found in motile cells.\nArchitecture of the Vertebrate Egg Coat and Structural Basis of the ZP2 Block to Polyspermy\nAuthors: Nishio, S.; Emori, C.; Wiseman, B.; Fahrenkamp, D.; Dioguardi, E.; Zamora-Caballero, S.; Bokhove, M.; Han, L.; Stsiapanava, A.; Lu, Y.; Kodani, M.; Bainbridge, R. E.; Komondor, K. M.; Carlson, A. E.; Landreh, M.; de Sanctis, D.; Yasumasu, S.; Ikawa, M.; Jovine, L.\nScore: 4.5, Published: 2023-06-22 DOI: 10.1101/2023.06.21.544075\nPost-fertilization cleavage of glycoprotein ZP2, a major subunit of egg zona pellucida (ZP) filaments, is crucial for mammalian reproduction by irreversibly blocking polyspermy. ZP2 processing is thought to inactivate a sperm-binding activity located upstream of the proteins cleavage site; however, its molecular consequences and connection with ZP hardening are unknown. Here we report X-ray crystallographic, cryo-EM and biochemical studies showing that cleavage of ZP2 triggers its oligomerization. Deletion of the ZP-N1 domain that precedes the cleavage site of mouse ZP2 allows it to homodimerize even without processing, and animals homozygous for this variant are subfertile by having a semi-hardened ZP that allows sperm attachment but hinders penetration. Combined with the structure of a native egg coat filament, which reveals the molecular basis of heteromeric ZP subunit interaction, this suggests that oligomerization of cleaved ZP2 cross-links the ZP, rigidifying it and making it physically impenetrable to sperm.\n",
  "wordCount" : "2050",
  "inLanguage": "en",
  "datePublished": "2023-06-28T11:45:43Z",
  "dateModified": "2023-06-28T11:45:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546235">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546235" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546235">
        <p class="paperTitle">Adhesion energy controls lipid binding-mediated endocytosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546235" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546235" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Groza, R.; Schmidt, K. V.; Muller, P. M.; Ronchi, P.; Schlack-Leigers, C.; Neu, U.; Puchkov, D.; Dimova, R.; Matthaus, C.; Taraska, J. W.; Weikl, T.; Ewers, H.</p>
        <p class="info">Score: 22.5, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546235' target='https://doi.org/10.1101/2023.06.23.546235'> 10.1101/2023.06.23.546235</a></p>
        <p class="abstract">Several bacterial toxins and viruses can deform membranes through multivalent binding to lipids for clathrin-independent endocytosis. However, it remains unclear, how membrane deformation and endocytic internalization are mechanistically linked. Here we show that many lipid-binding virions induce membrane deformation and clathrin-independent endocytosis, suggesting a common mechanism based on multivalent lipid binding by globular particles. We create a synthetic cellular system consisting of a lipid-anchored receptor in the form of GPI-anchored anti-GFP nanobodies and a multivalent globular binder exposing 180 regularly-spaced GFP molecules on its surface. We show that these globular, 40 nm diameter, particles bind to cells expressing the receptor, deform the plasma membrane upon adhesion and become endocytosed in a clathrin-independent manner. We explore the role of the membrane adhesion energy in endocytosis by using receptors with affinities varying over 7 orders of magnitude. Using this system, we find that once a threshold in adhesion energy is overcome to allow for membrane deformation, endocytosis occurs reliably. Multivalent, binding-induced membrane deformation by globular binders is thus sufficient for internalization to occur and we suggest it is the common, purely biophysical mechanism for lipid-binding mediated endocytosis of toxins and pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.25.546419">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.25.546419" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.25.546419">
        <p class="paperTitle">Dynamic ParB-DNA interactions initiate and maintain a partition condensate for bacterial chromosome segregation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.25.546419" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.25.546419" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tisma, M.; Janissen, R.; Antar, H.; Martin Gonzalez, A.; Barth, R.; Beekman, T.; van der Torre, J.; Michieletto, D.; Gruber, S.; Dekker, C.</p>
        <p class="info">Score: 19.1, Published: 2023-06-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.25.546419' target='https://doi.org/10.1101/2023.06.25.546419'> 10.1101/2023.06.25.546419</a></p>
        <p class="abstract">In most bacteria, chromosome segregation is driven by the ParABS system where the CTPase protein ParB loads at the parS site to trigger the formation of a large partition complex. Here, we present in vitro studies of the partition complex for Bacillus subtilis ParB, using single-molecule fluorescence microscopy and AFM imaging to show that transient ParB-ParB bridges are essential for forming DNA condensates. Molecular Dynamics simulations confirm that condensation occurs abruptly at a critical concentration of ParB and show that multimerization is a prerequisite for forming the partition complex. Magnetic tweezer force spectroscopy on mutant ParB proteins demonstrates that CTP hydrolysis at the N-terminal domain is essential for DNA condensation. Finally, we show that transcribing RNA polymerases can steadily traverse the ParB-DNA partition complex. These findings uncover how ParB forms a stable yet dynamic partition complex for chromosome segregation that induces DNA condensation and segregation while enabling replication and transcription.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.17.541250">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.17.541250" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.17.541250">
        <p class="paperTitle">Studying stochastic systems biology of the cell with single-cell genomics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.17.541250" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.17.541250" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gorin, G.; Vastola, J. J.; Pachter, L.</p>
        <p class="info">Score: 24.7, Published: 2023-05-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.17.541250' target='https://doi.org/10.1101/2023.05.17.541250'> 10.1101/2023.05.17.541250</a></p>
        <p class="abstract">Recent experimental developments in genome-wide RNA quantification hold considerable promise for systems biology. However, rigorously probing the biology of living cells requires a unified mathematical framework that accounts for single-molecule biological stochasticity in the context of technical variation associated with genomics assays. We review models for a variety of RNA transcription processes, as well as the encapsulation and library construction steps of microfluidics-based single-cell RNA sequencing, and present a framework to integrate these phenomena by the manipulation of generating functions. Finally, we use simulated scenarios and biological data to illustrate the implications and applications of the approach.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545802">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545802" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545802">
        <p class="paperTitle">Direct observation of autoubiquitination for an integral membrane ubiquitin ligase in ERAD</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545802" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545802" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moochickal Assainar, B.; Ragunathan, K.; Baldridge, R.</p>
        <p class="info">Score: 11.5, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545802' target='https://doi.org/10.1101/2023.06.20.545802'> 10.1101/2023.06.20.545802</a></p>
        <p class="abstract">The endoplasmic reticulum associated degradation (ERAD) pathway regulates protein quality control at the endoplasmic reticulum. ERAD of lumenal and membrane proteins requires a conserved E3 ubiquitin ligase, called Hrd1. We do not understand the molecular configurations of Hrd1 that enable autoubiquitination and the subsequent retrotranslocation of misfolded protein substrates from the ER to the cytosol. Here, we have established a generalizable, single-molecule platform that enables high-efficiency labeling, stoichiometry determination, and functional assays for any integral membrane protein. Using this approach, we directly count Hrd1 proteins reconstituted into individual proteoliposomes. We found that Hrd1 assembles in different oligomeric configurations with mostly monomers and dimers detected at limiting dilution. By correlating oligomeric states with ubiquitination in vitro, we determined that Hrd1 monomers were incapable of autoubiquitination while dimers efficiently assembled polyubiquitin chains. Therefore, our results reveal the minimal composition of a Hrd1 oligomer that is capable of autoubiquitination. Our methods are broadly applicable to studying other complex membrane protein functions using reconstituted bilayer systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.24.546380">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.24.546380" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.24.546380">
        <p class="paperTitle">Assembly of tight junction belts by surface condensation and actin elongation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.24.546380" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.24.546380" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sun, D.; Zhao, X.; Wiegand, T.; Bartolucci, G.; Martin-Lemaitre, C.; Grill, S. W.; Hyman, A. A.; Weber, C.; Honigmann, A.</p>
        <p class="info">Score: 11.1, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.24.546380' target='https://doi.org/10.1101/2023.06.24.546380'> 10.1101/2023.06.24.546380</a></p>
        <p class="abstract">Formation of biomolecular condensates via phase separation enables compartmentation of many cellular processes. However, how cells can control condensation at specific locations to create complex cellular structures remains poorly understood. Here, we investigated the mechanism of tight junction formation, which involves condensation of scaffold proteins at cell-cell contacts and elongation of the condensates into a belt around the cellular perimeter. Using cell biology, reconstitution, and thermodynamic theory, we discovered that cells use surface phase transitions to control local condensation at the membrane far below bulk saturation. Surface condensation of junctional ZO-scaffold proteins is mediated by receptor binding and regulated by the receptor&#39;s oligomerization state. Functionally, ZO surface condensation is directly coupled to actin polymerization and bundling, which drives elongation of receptor-ZO-actin condensates similar to tight junction belt formation in cells. We conclude that surface phase transitions provide a robust mechanism to control the position and shape of protein condensates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545758">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545758" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545758">
        <p class="paperTitle">Mechanism of DNA origami folding elucidated by mesoscopic simulations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545758" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545758" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: DeLuca, M.; Ye, T.; Poirier, M. G.; Ke, Y.; Castro, C.; Arya, G.</p>
        <p class="info">Score: 8.5, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545758' target='https://doi.org/10.1101/2023.06.20.545758'> 10.1101/2023.06.20.545758</a></p>
        <p class="abstract">DNA nanotechnology leverages the canonical base-pairing rules and geometry of DNA to create highly precise nanoscale structures with many potential applications. While the design and fabrication of DNA nanostructures is well-established, the self-assembly process that produces these structures is still poorly understood, especially for DNA origami that involve the assembly of hundreds of strands. Many experimental and computational efforts have sought to better understand DNA origami folding, but the small length and time scales of individual binding events and the long timescale over which folding occurs have posed significant challenges. Here, we present a new mesoscopic model that uses a switchable force field to capture the mechanical behavior of single- and double-stranded DNA motifs and transition between them at a coarseness level of up to 8 nucleotides per particle, allowing access to the long assembly timescales of DNA origami up to several kilobases in size. Brownian dynamics simulations of 4-helix bundle (4HB) structures using this model reveal a hierarchical folding process involving the zipping of structural domains into a partially folded precursor structure followed by gradual crystallization into the final structure. We elucidate the role of hybridization strength, scaffold routing, and staple design in the folding order and kinetics. Simulation of larger 32HB structures reveals heterogeneous staple incorporation kinetics and frequent trapping in metastable states, as opposed to smaller, more accessible structures like the 4HB, which exhibit first-order kinetics and virtually defect-free folding. The development of this model opens an avenue to better understand and design DNA nanostructures for improved yield and folding performance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545982">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545982" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545982">
        <p class="paperTitle">Loops are Geometric Catalysts for DNA Integration</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545982" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545982" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Battaglia, C.; Michieletto, D.</p>
        <p class="info">Score: 8.4, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545982' target='https://doi.org/10.1101/2023.06.21.545982'> 10.1101/2023.06.21.545982</a></p>
        <p class="abstract">The insertion of HIV and other DNA elements within genomes underpins both genetic diversity and disease when unregulated. Most of these insertions are not random and occupy specific positions within the genome but the physical mechanisms underlying the integration site selection are poorly understood. Here we perform Molecular Dynamics simulations to study the insertion of DNA elements, such as HIV viral DNA or transposons, into naked DNA or chromatin substrate. More specifically, we explore the role of loops in the DNA substrate and discover that they act as &#34;geometric catalysts&#34; for DNA integration. Additionally, we discover that the 1D and 3D clustering of loops affects the distribution of integration sites. Finally, we show that loops may compete with nucleosomes at attracting DNA integrations. These results may be tested in vitro and they may help to understand patterns of DNA insertions with implications in genome evolution and gene therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.12.544586">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.12.544586" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.12.544586">
        <p class="paperTitle">Labelizer: systematic selection of protein residues for covalent fluorophore labeling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.12.544586" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.12.544586" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gebhardt, C.; Bawidamann, P.; Schuetze, K.; Moya Munoz, G. G.; Spring, A.-K.; Griffith, D.; Lipfert, J.; Cordes, T.</p>
        <p class="info">Score: 56.9, Published: 2023-06-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.12.544586' target='https://doi.org/10.1101/2023.06.12.544586'> 10.1101/2023.06.12.544586</a></p>
        <p class="abstract">An essential requirement for the use of fluorescent dyes in biomedicine, molecular biology, biochemistry, biophysics and optical imaging is their (covalent) attachment to biomolecules. There is, however, no systematic and automated approach for the selection of suitable labeling sites in macromolecules, which is particular problematic for proteins. Here, we present a general and quantitative strategy to identify optimal residues for protein labeling using a naive Bayes classifier. Based on a literature search and bioinformatics analysis of &gt;100 proteins with [~]400 successfully labeled residues, we identified four parameters, which we combined into a labeling score to rank residues for their suitability as a label-site. The utility of our approach for the systematic selection of single residues and residue pairs for FRET experiments is supported by data from the literature and by new experiments on different proteins. To make the method available to a large community of researchers, we developed a python package called &#34;labelizer&#34;, that performs an analysis of a pdb-structure (or structural models), label score calculation, and FRET assay scoring. We further provide a webserver (https://labelizer.bio.lmu.de/) to conveniently apply our approach and to build up a central open-access database of (non-)successfully labeled protein residues to continuously improve and refine the labelizer approach.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.546267">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.546267" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.546267">
        <p class="paperTitle">Liquid-like condensates mediate competition between actin branching and bundling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.546267" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.546267" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Graham, K.; Chandrasekaran, A.; Wang, L.; Yang, N.; Lafer, E. M.; Rangamani, P.; Stachowiak, J. C.</p>
        <p class="info">Score: 8.0, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.546267' target='https://doi.org/10.1101/2023.06.23.546267'> 10.1101/2023.06.23.546267</a></p>
        <p class="abstract">Cellular remodeling of actin networks underlies cell motility during key morphological events, from embryogenesis to metastasis. In these transformations there is an inherent competition between actin branching and bundling, because steric clashes among branches create a mechanical barrier to bundling. Recently, liquid-like condensates consisting purely of proteins involved in either branching or bundling of the cytoskeleton have been found to catalyze their respective functions. Yet in the cell, proteins that drive branching and bundling are present simultaneously. In this complex environment, which factors determine whether a condensate drives filaments to branch versus becoming bundled? To answer this question, we added the branched actin nucleator, Arp2/3, to condensates composed of VASP, an actin bundling protein. At low actin to VASP ratios, branching activity, mediated by Arp2/3, robustly inhibited VASP-mediated bundling of filaments, in agreement with agent-based simulations. In contrast, as the actin to VASP ratio increased, addition of Arp2/3 led to formation of aster-shaped structures, in which bundled filaments emerged from a branched actin core, analogous to filopodia emerging from a branched lamellipodial network. These results demonstrate that multi-component, liquid-like condensates can modulate the inherent competition between bundled and branched actin morphologies, leading to organized, higher-order structures, similar to those found in motile cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.544075">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.544075" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.544075">
        <p class="paperTitle">Architecture of the Vertebrate Egg Coat and Structural Basis of the ZP2 Block to Polyspermy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.544075" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.544075" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nishio, S.; Emori, C.; Wiseman, B.; Fahrenkamp, D.; Dioguardi, E.; Zamora-Caballero, S.; Bokhove, M.; Han, L.; Stsiapanava, A.; Lu, Y.; Kodani, M.; Bainbridge, R. E.; Komondor, K. M.; Carlson, A. E.; Landreh, M.; de Sanctis, D.; Yasumasu, S.; Ikawa, M.; Jovine, L.</p>
        <p class="info">Score: 4.5, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.544075' target='https://doi.org/10.1101/2023.06.21.544075'> 10.1101/2023.06.21.544075</a></p>
        <p class="abstract">Post-fertilization cleavage of glycoprotein ZP2, a major subunit of egg zona pellucida (ZP) filaments, is crucial for mammalian reproduction by irreversibly blocking polyspermy. ZP2 processing is thought to inactivate a sperm-binding activity located upstream of the proteins cleavage site; however, its molecular consequences and connection with ZP hardening are unknown. Here we report X-ray crystallographic, cryo-EM and biochemical studies showing that cleavage of ZP2 triggers its oligomerization. Deletion of the ZP-N1 domain that precedes the cleavage site of mouse ZP2 allows it to homodimerize even without processing, and animals homozygous for this variant are subfertile by having a semi-hardened ZP that allows sperm attachment but hinders penetration. Combined with the structure of a native egg coat filament, which reveals the molecular basis of heteromeric ZP subunit interaction, this suggests that oligomerization of cleaved ZP2 cross-links the ZP, rigidifying it and making it physically impenetrable to sperm.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
