<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Small LEA proteins as an effective air-water interface protectant for fragile samples during cryo-EM grid plunge freezing
Authors: Abe, K. M.; Lim, C. J.
Score: 54.1, Published: 2024-02-11 DOI: 10.1101/2024.02.06.579238
Sample loss due to air-water interface (AWI) interactions is a significant challenge during cryo-electron microscopy (cryo-EM) sample grid plunge freezing. We report that small Late Embryogenesis Abundant (LEA) proteins, which naturally bind to AWI, can protect samples from AWI damage during plunge freezing.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Small LEA proteins as an effective air-water interface protectant for fragile samples during cryo-EM grid plunge freezing
Authors: Abe, K. M.; Lim, C. J.
Score: 54.1, Published: 2024-02-11 DOI: 10.1101/2024.02.06.579238
Sample loss due to air-water interface (AWI) interactions is a significant challenge during cryo-electron microscopy (cryo-EM) sample grid plunge freezing. We report that small Late Embryogenesis Abundant (LEA) proteins, which naturally bind to AWI, can protect samples from AWI damage during plunge freezing." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Small LEA proteins as an effective air-water interface protectant for fragile samples during cryo-EM grid plunge freezing
Authors: Abe, K. M.; Lim, C. J.
Score: 54.1, Published: 2024-02-11 DOI: 10.1101/2024.02.06.579238
Sample loss due to air-water interface (AWI) interactions is a significant challenge during cryo-electron microscopy (cryo-EM) sample grid plunge freezing. We report that small Late Embryogenesis Abundant (LEA) proteins, which naturally bind to AWI, can protect samples from AWI damage during plunge freezing."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Small LEA proteins as an effective air-water interface protectant for fragile samples during cryo-EM grid plunge freezing\nAuthors: Abe, K. M.; Lim, C. J.\nScore: 54.1, Published: 2024-02-11 DOI: 10.1101/2024.02.06.579238\nSample loss due to air-water interface (AWI) interactions is a significant challenge during cryo-electron microscopy (cryo-EM) sample grid plunge freezing. We report that small Late Embryogenesis Abundant (LEA) proteins, which naturally bind to AWI, can protect samples from AWI damage during plunge freezing.",
  "keywords": [
    
  ],
  "articleBody": " Small LEA proteins as an effective air-water interface protectant for fragile samples during cryo-EM grid plunge freezing\nAuthors: Abe, K. M.; Lim, C. J.\nScore: 54.1, Published: 2024-02-11 DOI: 10.1101/2024.02.06.579238\nSample loss due to air-water interface (AWI) interactions is a significant challenge during cryo-electron microscopy (cryo-EM) sample grid plunge freezing. We report that small Late Embryogenesis Abundant (LEA) proteins, which naturally bind to AWI, can protect samples from AWI damage during plunge freezing. This protection is demonstrated with two LEA proteins from nematodes and tardigrades, which rescued the cryo-EM structural determination outcome of two fragile multisubunit protein complexes.\nLow-dose cryo-electron ptychography of proteins at sub-nanometer resolution\nAuthors: Kucukoglu, B.; Mohammed, I.; Guerrero-Ferreira, R. C.; Ribet, S.; Varnavides, G.; Leidl, M. L.; Lau, K.; Nazarov, S.; Myasnikov, A.; Sachse, C.; Mueller-Caspary, K.; Ophus, C.; Stahlberg, H.\nScore: 26.8, Published: 2024-02-12 DOI: 10.1101/2024.02.12.579607\nCryo-transmission electron microscopy (cryo-EM) of frozen hydrated specimens is an efficient method for the structural analysis of purified biological molecules. However, for particles smaller than 50kDa, single particle cryo-EM often fails due to the limited imaging contrast of individual micrographs. For dose-resilient samples often studied in the physical sciences, electron ptychography, a coherent diffractive imaging technique using 4D scanning transmission electron microscopy (4D-STEM), has recently demonstrated resolution down to tens of picometers for thin specimens imaged at room temperature. Here we applied ptychographic data analysis to frozen hydrated single protein particles, reaching sub-nanometer resolution 3D reconstructions. We employed low-dose cryo-EM with an aberration-corrected, convergent electron beam to collect 4D-STEM data for our reconstructions. The high speed of the electron detector allowed us to record large datasets of electron diffraction patterns with substantial overlaps between the interaction volumes of adjacent scan position, from which the scattering potentials of the samples were iteratively reconstructed. The reconstructed micrographs show strong contrast enabling the reconstruction of the structure of apoferritin protein at up to 5.8 Angstrom resolution. Ptychography is a promising tool to study the structure of smaller frozen hydrated protein particles, and, once combined with electron tomography tilt series, bears the potential to provide unique insight into the ultrastructure of vitrified biological tissue.\nnavigate: an open-source platform for smart light-sheet microscopy.\nAuthors: Marin, Z.; Wang, X.; Collison, D. W.; McFadden, C.; Lin, J.; Borges, H.; Chen, B.; Mehra, D.; Shen, Q.; Galecki, S.; Daetwyler, S.; Fiolka, R. P.; Dean, K. M.\nScore: 21.9, Published: 2024-02-11 DOI: 10.1101/2024.02.09.579083\nnavigate is a turnkey, open-source software solution designed to enhance light-sheet fluorescence microscopy (LSFM) by integrating smart microscopy techniques into a user-friendly framework. It enables automated, intelligent imaging with a Python-based control system that supports GUI-reconfigurable acquisition routines and the integration of diverse hardware sets. As a comprehensive package, navigate democratizes access to advanced LSFM capabilities, facilitating the development and implementation of smart microscopy workflows without requiring deep programming knowledge or specialized expertise in light-sheet microscopy.\nA coarse-grained model for disordered and multi-domain proteins\nAuthors: Cao, F.; von BuÌˆlow, S.; Tesei, G.; Lindorff-Larsen, K.\nScore: 14.9, Published: 2024-02-07 DOI: 10.1101/2024.02.03.578735\nMany proteins contain more than one folded domain, and such modular multi-domain proteins help expand the functional repertoire of proteins. Because of their larger size and often substantial dynamics, it may be difficult to characterize the conformational ensembles of multi-domain proteins by simulations. Here, we present a coarse-grained model for multi-domain proteins that is both fast and provides an accurate description of the global conformational properties in solution. We show that the accuracy of a one-bead-per-residue coarse-grained model depends on how the interaction sites in the folded domains are represented. Specifically, we find excessive domain-domain interactions if the interaction sites are located at the position of the C atoms. We also show that if the interaction sites are located at the centre of mass of the residue, we obtain good agreement between simulations and experiments across a wide range of proteins. We then optimize our previously described CALVADOS model using this centre-of-mass representation, and validate the resulting model using independent data. Finally, we use our revised model to simulate phase separation of both disordered and multi-domain proteins. Our results provide a starting point for understanding interactions between folded and disordered regions in proteins, and how these regions affect the propensity of proteins to self-associate and undergo phase separation.\nIntermolecular Energy Migration via HomoFRET Captures the Modulation in the Material Property of Phase-Separated Biomolecular Condensates\nAuthors: Joshi, A.; Walimbe, A.; Sarkar, S.; Arora, L.; Kaur, G.; Jhandai, P.; Chatterjee, D.; Banerjee, I.; Mukhopadhyay, S.\nScore: 12.9, Published: 2024-02-12 DOI: 10.1101/2024.02.12.579899\nBiomolecular condensation via phase separation of proteins and nucleic acids has emerged as a crucial mechanism underlying the spatiotemporal organization of cellular components into functional membraneless organelles. However, aberrant maturation of these dynamic, liquid-like assemblies into irreversible gel-like or solid-like aggregates is associated with a wide range of fatal neurodegenerative diseases. New tools are essential to dissect the changes in the internal material properties of these biomolecular condensates that are often modulated by a wide range of factors involving the sequence composition, truncations, mutations, post-translational modifications, and the stoichiometry of nucleic acids and other biomolecules. Here, we employ homo-Forster Resonance Energy Transfer (homoFRET) as a proximity ruler to study intermolecular energy migration that illuminates the molecular packing in the nanometric length scale within biomolecular condensates. We used the homoFRET efficiency, measured by a loss in the fluorescence anisotropy due to rapid depolarization, as a readout of the molecular packing giving rise to material properties of biomolecular condensates. Using single-droplet anisotropy imaging, we recorded spatially-resolved homoFRET efficiencies of condensates formed by fluorescent protein-tagged Fused in Sarcoma (FUS). By performing single-droplet picosecond time-resolved anisotropy measurements, we were able to discern various energy migration events within the dense network of polypeptide chains in FUS condensates. Our homoFRET studies also captured the modulation of material properties by RNA, ATP, and post-translational modification. Additionally, we utilized mammalian cell lines stably expressing FUS to study nuclear FUS and oxidative stress-induced stress granule formation in the cytoplasm. Our studies demonstrate that spatially-resolved homoFRET methodology offers a potent tool for studying intracellular phase transitions in cell physiology and disease.\nEnergetics of the Transmembrane Peptide Sorting by Hydrophobic Mismatch\nAuthors: Fabian, B.; Javanainen, M.\nScore: 9.8, Published: 2024-02-07 DOI: 10.1101/2024.02.02.578561\nHydrophobic mismatch between a lipid membrane and embedded transmembrane peptides or proteins plays a role in their lateral localization and function. Earlier studies have resolved numerous mechanisms through which the peptides and membrane proteins adapt to mismatch, yet the energetics of lateral sorting due to hydrophobic mismatch has remained elusive due to the lack of suitable computational or experimental protocols. Here, we pioneer a molecular dynamics simulation approach to study the sorting of peptides along a membrane thickness gradient. Peptides of different lengths tilt and diffuse along the membrane to eliminate mismatch with a rate directly proportional to the magnitude of mismatch. We extract the 2-dimensional free energy profiles as a function of local thickness and peptide orientation, revealing the relative contributions of sorting and tilting, and suggesting their thermally accessible regimes. Our approach can readily be applied to study other membrane systems of biological interest where hydrophobic mismatch, or membrane thickness in general, plays a role.\nLigand-induced conformational changes in the Î²1-Adrenergic Receptor Revealed by Hydrogen-Deuterium Exchange Mass Spectrometry\nAuthors: Toporowska, J.; Kapoor, P.; Musgaard, M.; Gherbi, K.; Sengmany, K.; Qu, F.; Soave, M.; Yen, H.-Y.; Hansen, K.; Jazayeri, A.; Hopper, J. T. S.; Politis, A.\nScore: 8.8, Published: 2024-02-09 DOI: 10.1101/2024.02.07.579309\nG-Protein Coupled Receptors (GPCRs) constitute the largest family of signalling proteins responsible for translating extracellular stimuli into intracellular functions. When dysregulated, GPCRs drive numerous diseases and are the most targeted proteins in drug discovery. GPCR structural dynamics and activity can be modulated by a wide range of drugs, including full/partial agonists and antagonists. While crucial for developing novel therapeutics targeting GPCRs, the structural dynamics of the receptors associated with their activity upon drug interactions are not yet fully understood. Here, we employ Hydrogen Deuterium Exchange Mass Spectrometry (HDX-MS), to characterise the structural dynamics of turkey {beta}1-adrenergic receptor (t{beta}1AR) in complex with nine ligands, including agonists, partial agonists and antagonists. We show that dynamic signatures across the GPCR structure can be grouped by compound modality. Surprisingly, we discovered repeated destabilisation of the intracellular loop 1 (ICL1) upon full agonist binding and stabilisation upon antagonist binding, suggesting that increased dynamics in this region are an essential component for G-protein recruitment. Multiple sequence alignments and molecular dynamics simulations indicate that L72 in ICL1 plays important structural role. Differential HDX-MS experiment of t{beta}1AR and t{beta}1AR L72A construct in complex with miniGs, in response to various ligands, suggests involvement of ICL1 in stabilising the GDP bound state by influencing the stability of HG helix of miniGs. Overall, our results provide a platform for determining drug modality and highlight how HDX-MS can be used to dissect receptor ligand interaction properties and GPCR mechanism. Significance statementRecent advances in hydrogen-deuterium exchange mass spectrometry have allowed probing conformational signatures of challenging membrane protein assemblies. We studied the structural dynamics of a class A GPCR, namely t{beta}1AR, in response to diverse ligands including agonists, antagonists and partial agonists. We demonstrate that the functional effect of compounds can be discerned by simply profiling the dynamics induced across the receptor, without the need for downstream interaction partners. We showed that ICL1 undergoes a significant change in dynamics between activated and inhibited states consistent with a role in downstream signaling pathways in class A GPCRs.\nThe accomplices: Heparan sulfates and N-glycans foster SARS-CoV-2 spike:ACE2 receptor binding and virus priming\nAuthors: Paiardi, G.; Ferraz, M.; Rusnati, M.; Wade, R. C.\nScore: 13.9, Published: 2024-02-05 DOI: 10.1101/2024.02.05.578888\nThe SARS-CoV-2 spike glycoprotein mediates virus attachment to human host cells by binding angiotensin-converting enzyme 2 (ACE2) and heparan sulfate (HS) proteoglycans. To elucidate the structure, dynamics, and functional consequences of these interactions, we carried out microsecond-long all-atom molecular dynamics simulations, followed by random acceleration molecular dynamics simulations, of the fully glycosylated spike:ACE2 complex with and without heparin chains bound. We find that heparin, a model for HS, promotes structural and energetic stabilization of the active conformation of the spike receptor binding domain (RBD) and reorientation of ACE2 toward the N-terminal domain in the same spike subunit as the RBD. Spike and ACE2 N-glycans exert synergistic effects, promoting better packing, strengthening the protein:protein interaction, and prolonging the residence time of the complex. ACE2 and heparin binding trigger rearrangement of the S2 functional site through allosteric interdomain communication. HS thus has a multifaceted role in facilitating SARS-CoV-2 infection.\nThe mechanism of mammalian proton-coupled peptide transporters\nAuthors: Lichtinger, S. M.; Parker, J. L.; Newstead, S.; Biggin, P.\nScore: 8.2, Published: 2024-02-06 DOI: 10.1101/2024.02.04.578827\nProton-coupled oligopeptide transporters (POTs) are of great pharmaceutical interest owing to their promiscuous substrate binding site that has been linked to improved oral bioavailability of several classes of drugs. Members of the POT family are conserved across all phylogenetic kingdoms and function by coupling peptide uptake to the proton electrochemical gradient. Cryo-EM structures and alphafold models have recently provided new insights into different conformational states of two mammalian POTs, SLC15A1 and SLC15A2. Nevertheless, these studies leave open important questions regarding the mechanism of proton and substrate coupling, while simultaneously providing a unique opportunity to investigate these processes using molecular dynamics (MD) simulations. Here, we employ extensive unbiased and enhanced-sampling MD to map out the full SLC15A2 conformational cycle and its thermodynamic driving forces. By computing conformational free energy landscapes in different protonation states and in the absence or presence of peptide substrate, we identify a likely sequence of intermediate protonation steps that drive inward-directed alternating access. These simulations identify key differences in the extracellular gate between mammalian and bacterial POTs, which we validate experimentally in cell-based transport assays. Our results from constant-PH MD and absolute binding free energy (ABFE) calculations also establish a mechanistic link between proton binding and peptide recognition, revealing key details underpining secondary active transport in POTs. This study provides a vital step forward in understanding proton-coupled peptide and drug transport in mammals and pave the way to integrate knowledge of solute carrier structural biology with enhanced drug design to target tissue and organ bioavailability.\nModified histone peptides uniquely tune the material properties of HP1Î± condensates.\nAuthors: Deshpande, P.; Prentice, E.; Vidal Ceballos, A.; Casaccia, P.; Elbaum-Garfinkle, S.\nScore: 6.0, Published: 2024-02-09 DOI: 10.1101/2024.02.07.579285\nBiomolecular condensates have emerged as a powerful new paradigm in cell biology with broad implications to human health and disease, particularly in the nucleus where phase separation is thought to underly elements of chromatin organization and regulation. Specifically, it has been recently reported that phase separation of heterochromatin protein 1alpha (HP1) with DNA contributes to the formation of condensed chromatin states. HP1 localization to heterochromatic regions is mediated by its binding to specific repressive marks on the tail of histone H3, such as trimethylated lysine 9 on histone H3 (H3K9me3). However, whether epigenetic marks play an active role in modulating the material properties of HP1 and dictating emergent functions of its condensates, remains only partially understood. Here, we leverage a reductionist system, comprised of modified and unmodified histone H3 peptides, HP1 and DNA to examine the contribution of specific epigenetic marks to phase behavior of HP1. We show that the presence of histone peptides bearing the repressive H3K9me3 is compatible with HP1 condensates, while peptides containing unmodified residues or bearing the transcriptional activation mark H3K4me3 are incompatible with HP1 phase separation. In addition, inspired by the decreased ratio of nuclear H3K9me3 to HP1 detected in cells exposed to uniaxial strain, using fluorescence microscopy and rheological approaches we demonstrate that H3K9me3 histone peptides modulate the dynamics and network properties of HP1 condensates in a concentration dependent manner. These data suggest that HP1-DNA condensates are viscoelastic materials, whose properties may provide an explanation for the dynamic behavior of heterochromatin in cells in response to mechanostimulation. Statement of significanceThe organization of genomic information in eukaryotic cells necessitates compartmentalization into functional domains allowing for the expression of cell identity-specific genes, while repressing genes related to alternative fates. Heterochromatin hosts these transcriptionally silent regions of the genome - which ensure the stability of cell identity -and is characterized by repressive histone marks (H3K9m3) and other specialized proteins (HP1a), recently shown to phase separate with DNA. We show that HP1a forms condensates with DNA which persist in the presence of H3K9me3 peptides. The viscoelastic nature of these condensates depend on H3K9me3:HP1 ratios, which are modulated by mechanical strain in cells. Thus, phase separation may explain the dynamic behavior of heterochromatin in cells, in response to mechanostimulation.\n",
  "wordCount" : "2408",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579238">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579238" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579238">
        <p class="paperTitle">Small LEA proteins as an effective air-water interface protectant for fragile samples during cryo-EM grid plunge freezing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579238" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579238" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abe, K. M.; Lim, C. J.</p>
        <p class="info">Score: 54.1, Published: 2024-02-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579238' target='https://doi.org/10.1101/2024.02.06.579238'> 10.1101/2024.02.06.579238</a></p>
        <p class="abstract">Sample loss due to air-water interface (AWI) interactions is a significant challenge during cryo-electron microscopy (cryo-EM) sample grid plunge freezing. We report that small Late Embryogenesis Abundant (LEA) proteins, which naturally bind to AWI, can protect samples from AWI damage during plunge freezing. This protection is demonstrated with two LEA proteins from nematodes and tardigrades, which rescued the cryo-EM structural determination outcome of two fragile multisubunit protein complexes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.579607">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.579607" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.579607">
        <p class="paperTitle">Low-dose cryo-electron ptychography of proteins at sub-nanometer resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.579607" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.579607" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kucukoglu, B.; Mohammed, I.; Guerrero-Ferreira, R. C.; Ribet, S.; Varnavides, G.; Leidl, M. L.; Lau, K.; Nazarov, S.; Myasnikov, A.; Sachse, C.; Mueller-Caspary, K.; Ophus, C.; Stahlberg, H.</p>
        <p class="info">Score: 26.8, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.579607' target='https://doi.org/10.1101/2024.02.12.579607'> 10.1101/2024.02.12.579607</a></p>
        <p class="abstract">Cryo-transmission electron microscopy (cryo-EM) of frozen hydrated specimens is an efficient method for the structural analysis of purified biological molecules. However, for particles smaller than 50kDa, single particle cryo-EM often fails due to the limited imaging contrast of individual micrographs. For dose-resilient samples often studied in the physical sciences, electron ptychography, a coherent diffractive imaging technique using 4D scanning transmission electron microscopy (4D-STEM), has recently demonstrated resolution down to tens of picometers for thin specimens imaged at room temperature. Here we applied ptychographic data analysis to frozen hydrated single protein particles, reaching sub-nanometer resolution 3D reconstructions. We employed low-dose cryo-EM with an aberration-corrected, convergent electron beam to collect 4D-STEM data for our reconstructions. The high speed of the electron detector allowed us to record large datasets of electron diffraction patterns with substantial overlaps between the interaction volumes of adjacent scan position, from which the scattering potentials of the samples were iteratively reconstructed. The reconstructed micrographs show strong contrast enabling the reconstruction of the structure of apoferritin protein at up to 5.8 Angstrom resolution. Ptychography is a promising tool to study the structure of smaller frozen hydrated protein particles, and, once combined with electron tomography tilt series, bears the potential to provide unique insight into the ultrastructure of vitrified biological tissue.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579083">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579083" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579083">
        <p class="paperTitle">navigate: an open-source platform for smart light-sheet microscopy.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579083" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579083" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Marin, Z.; Wang, X.; Collison, D. W.; McFadden, C.; Lin, J.; Borges, H.; Chen, B.; Mehra, D.; Shen, Q.; Galecki, S.; Daetwyler, S.; Fiolka, R. P.; Dean, K. M.</p>
        <p class="info">Score: 21.9, Published: 2024-02-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579083' target='https://doi.org/10.1101/2024.02.09.579083'> 10.1101/2024.02.09.579083</a></p>
        <p class="abstract">navigate is a turnkey, open-source software solution designed to enhance light-sheet fluorescence microscopy (LSFM) by integrating smart microscopy techniques into a user-friendly framework. It enables automated, intelligent imaging with a Python-based control system that supports GUI-reconfigurable acquisition routines and the integration of diverse hardware sets. As a comprehensive package, navigate democratizes access to advanced LSFM capabilities, facilitating the development and implementation of smart microscopy workflows without requiring deep programming knowledge or specialized expertise in light-sheet microscopy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.03.578735">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.03.578735" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.03.578735">
        <p class="paperTitle">A coarse-grained model for disordered and multi-domain proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.03.578735" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.03.578735" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cao, F.; von BuÌˆlow, S.; Tesei, G.; Lindorff-Larsen, K.</p>
        <p class="info">Score: 14.9, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.03.578735' target='https://doi.org/10.1101/2024.02.03.578735'> 10.1101/2024.02.03.578735</a></p>
        <p class="abstract">Many proteins contain more than one folded domain, and such modular multi-domain proteins help expand the functional repertoire of proteins. Because of their larger size and often substantial dynamics, it may be difficult to characterize the conformational ensembles of multi-domain proteins by simulations. Here, we present a coarse-grained model for multi-domain proteins that is both fast and provides an accurate description of the global conformational properties in solution. We show that the accuracy of a one-bead-per-residue coarse-grained model depends on how the interaction sites in the folded domains are represented. Specifically, we find excessive domain-domain interactions if the interaction sites are located at the position of the C atoms. We also show that if the interaction sites are located at the centre of mass of the residue, we obtain good agreement between simulations and experiments across a wide range of proteins. We then optimize our previously described CALVADOS model using this centre-of-mass representation, and validate the resulting model using independent data. Finally, we use our revised model to simulate phase separation of both disordered and multi-domain proteins. Our results provide a starting point for understanding interactions between folded and disordered regions in proteins, and how these regions affect the propensity of proteins to self-associate and undergo phase separation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.579899">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.579899" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.579899">
        <p class="paperTitle">Intermolecular Energy Migration via HomoFRET Captures the Modulation in the Material Property of Phase-Separated Biomolecular Condensates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.579899" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.579899" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joshi, A.; Walimbe, A.; Sarkar, S.; Arora, L.; Kaur, G.; Jhandai, P.; Chatterjee, D.; Banerjee, I.; Mukhopadhyay, S.</p>
        <p class="info">Score: 12.9, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.579899' target='https://doi.org/10.1101/2024.02.12.579899'> 10.1101/2024.02.12.579899</a></p>
        <p class="abstract">Biomolecular condensation via phase separation of proteins and nucleic acids has emerged as a crucial mechanism underlying the spatiotemporal organization of cellular components into functional membraneless organelles. However, aberrant maturation of these dynamic, liquid-like assemblies into irreversible gel-like or solid-like aggregates is associated with a wide range of fatal neurodegenerative diseases. New tools are essential to dissect the changes in the internal material properties of these biomolecular condensates that are often modulated by a wide range of factors involving the sequence composition, truncations, mutations, post-translational modifications, and the stoichiometry of nucleic acids and other biomolecules. Here, we employ homo-Forster Resonance Energy Transfer (homoFRET) as a proximity ruler to study intermolecular energy migration that illuminates the molecular packing in the nanometric length scale within biomolecular condensates. We used the homoFRET efficiency, measured by a loss in the fluorescence anisotropy due to rapid depolarization, as a readout of the molecular packing giving rise to material properties of biomolecular condensates. Using single-droplet anisotropy imaging, we recorded spatially-resolved homoFRET efficiencies of condensates formed by fluorescent protein-tagged Fused in Sarcoma (FUS). By performing single-droplet picosecond time-resolved anisotropy measurements, we were able to discern various energy migration events within the dense network of polypeptide chains in FUS condensates. Our homoFRET studies also captured the modulation of material properties by RNA, ATP, and post-translational modification. Additionally, we utilized mammalian cell lines stably expressing FUS to study nuclear FUS and oxidative stress-induced stress granule formation in the cytoplasm. Our studies demonstrate that spatially-resolved homoFRET methodology offers a potent tool for studying intracellular phase transitions in cell physiology and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578561">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578561" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578561">
        <p class="paperTitle">Energetics of the Transmembrane Peptide Sorting by Hydrophobic Mismatch</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578561" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578561" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fabian, B.; Javanainen, M.</p>
        <p class="info">Score: 9.8, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578561' target='https://doi.org/10.1101/2024.02.02.578561'> 10.1101/2024.02.02.578561</a></p>
        <p class="abstract">Hydrophobic mismatch between a lipid membrane and embedded transmembrane peptides or proteins plays a role in their lateral localization and function. Earlier studies have resolved numerous mechanisms through which the peptides and membrane proteins adapt to mismatch, yet the energetics of lateral sorting due to hydrophobic mismatch has remained elusive due to the lack of suitable computational or experimental protocols. Here, we pioneer a molecular dynamics simulation approach to study the sorting of peptides along a membrane thickness gradient. Peptides of different lengths tilt and diffuse along the membrane to eliminate mismatch with a rate directly proportional to the magnitude of mismatch. We extract the 2-dimensional free energy profiles as a function of local thickness and peptide orientation, revealing the relative contributions of sorting and tilting, and suggesting their thermally accessible regimes. Our approach can readily be applied to study other membrane systems of biological interest where hydrophobic mismatch, or membrane thickness in general, plays a role.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579309">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579309" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579309">
        <p class="paperTitle">Ligand-induced conformational changes in the Î²1-Adrenergic Receptor Revealed by Hydrogen-Deuterium Exchange Mass Spectrometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579309" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579309" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Toporowska, J.; Kapoor, P.; Musgaard, M.; Gherbi, K.; Sengmany, K.; Qu, F.; Soave, M.; Yen, H.-Y.; Hansen, K.; Jazayeri, A.; Hopper, J. T. S.; Politis, A.</p>
        <p class="info">Score: 8.8, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579309' target='https://doi.org/10.1101/2024.02.07.579309'> 10.1101/2024.02.07.579309</a></p>
        <p class="abstract">G-Protein Coupled Receptors (GPCRs) constitute the largest family of signalling proteins responsible for translating extracellular stimuli into intracellular functions. When dysregulated, GPCRs drive numerous diseases and are the most targeted proteins in drug discovery. GPCR structural dynamics and activity can be modulated by a wide range of drugs, including full/partial agonists and antagonists. While crucial for developing novel therapeutics targeting GPCRs, the structural dynamics of the receptors associated with their activity upon drug interactions are not yet fully understood. Here, we employ Hydrogen Deuterium Exchange Mass Spectrometry (HDX-MS), to characterise the structural dynamics of turkey {beta}1-adrenergic receptor (t{beta}1AR) in complex with nine ligands, including agonists, partial agonists and antagonists. We show that dynamic signatures across the GPCR structure can be grouped by compound modality. Surprisingly, we discovered repeated destabilisation of the intracellular loop 1 (ICL1) upon full agonist binding and stabilisation upon antagonist binding, suggesting that increased dynamics in this region are an essential component for G-protein recruitment. Multiple sequence alignments and molecular dynamics simulations indicate that L72 in ICL1 plays important structural role. Differential HDX-MS experiment of t{beta}1AR and t{beta}1AR L72A construct in complex with miniGs, in response to various ligands, suggests involvement of ICL1 in stabilising the GDP bound state by influencing the stability of HG helix of miniGs. Overall, our results provide a platform for determining drug modality and highlight how HDX-MS can be used to dissect receptor ligand interaction properties and GPCR mechanism.

Significance statementRecent advances in hydrogen-deuterium exchange mass spectrometry have allowed probing conformational signatures of challenging membrane protein assemblies. We studied the structural dynamics of a class A GPCR, namely t{beta}1AR, in response to diverse ligands including agonists, antagonists and partial agonists. We demonstrate that the functional effect of compounds can be discerned by simply profiling the dynamics induced across the receptor, without the need for downstream interaction partners. We showed that ICL1 undergoes a significant change in dynamics between activated and inhibited states consistent with a role in downstream signaling pathways in class A GPCRs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578888">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578888" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578888">
        <p class="paperTitle">The accomplices: Heparan sulfates and N-glycans foster SARS-CoV-2 spike:ACE2 receptor binding and virus priming</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578888" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578888" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paiardi, G.; Ferraz, M.; Rusnati, M.; Wade, R. C.</p>
        <p class="info">Score: 13.9, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578888' target='https://doi.org/10.1101/2024.02.05.578888'> 10.1101/2024.02.05.578888</a></p>
        <p class="abstract">The SARS-CoV-2 spike glycoprotein mediates virus attachment to human host cells by binding angiotensin-converting enzyme 2 (ACE2) and heparan sulfate (HS) proteoglycans. To elucidate the structure, dynamics, and functional consequences of these interactions, we carried out microsecond-long all-atom molecular dynamics simulations, followed by random acceleration molecular dynamics simulations, of the fully glycosylated spike:ACE2 complex with and without heparin chains bound. We find that heparin, a model for HS, promotes structural and energetic stabilization of the active conformation of the spike receptor binding domain (RBD) and reorientation of ACE2 toward the N-terminal domain in the same spike subunit as the RBD. Spike and ACE2 N-glycans exert synergistic effects, promoting better packing, strengthening the protein:protein interaction, and prolonging the residence time of the complex. ACE2 and heparin binding trigger rearrangement of the S2 functional site through allosteric interdomain communication. HS thus has a multifaceted role in facilitating SARS-CoV-2 infection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.04.578827">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.04.578827" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.04.578827">
        <p class="paperTitle">The mechanism of mammalian proton-coupled peptide transporters</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.04.578827" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.04.578827" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lichtinger, S. M.; Parker, J. L.; Newstead, S.; Biggin, P.</p>
        <p class="info">Score: 8.2, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.04.578827' target='https://doi.org/10.1101/2024.02.04.578827'> 10.1101/2024.02.04.578827</a></p>
        <p class="abstract">Proton-coupled oligopeptide transporters (POTs) are of great pharmaceutical interest owing to their promiscuous substrate binding site that has been linked to improved oral bioavailability of several classes of drugs. Members of the POT family are conserved across all phylogenetic kingdoms and function by coupling peptide uptake to the proton electrochemical gradient. Cryo-EM structures and alphafold models have recently provided new insights into different conformational states of two mammalian POTs, SLC15A1 and SLC15A2. Nevertheless, these studies leave open important questions regarding the mechanism of proton and substrate coupling, while simultaneously providing a unique opportunity to investigate these processes using molecular dynamics (MD) simulations. Here, we employ extensive unbiased and enhanced-sampling MD to map out the full SLC15A2 conformational cycle and its thermodynamic driving forces. By computing conformational free energy landscapes in different protonation states and in the absence or presence of peptide substrate, we identify a likely sequence of intermediate protonation steps that drive inward-directed alternating access. These simulations identify key differences in the extracellular gate between mammalian and bacterial POTs, which we validate experimentally in cell-based transport assays. Our results from constant-PH MD and absolute binding free energy (ABFE) calculations also establish a mechanistic link between proton binding and peptide recognition, revealing key details underpining secondary active transport in POTs. This study provides a vital step forward in understanding proton-coupled peptide and drug transport in mammals and pave the way to integrate knowledge of solute carrier structural biology with enhanced drug design to target tissue and organ bioavailability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.07.579285">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.07.579285" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.07.579285">
        <p class="paperTitle">Modified histone peptides uniquely tune the material properties of HP1Î± condensates.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.07.579285" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.07.579285" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Deshpande, P.; Prentice, E.; Vidal Ceballos, A.; Casaccia, P.; Elbaum-Garfinkle, S.</p>
        <p class="info">Score: 6.0, Published: 2024-02-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.07.579285' target='https://doi.org/10.1101/2024.02.07.579285'> 10.1101/2024.02.07.579285</a></p>
        <p class="abstract">Biomolecular condensates have emerged as a powerful new paradigm in cell biology with broad implications to human health and disease, particularly in the nucleus where phase separation is thought to underly elements of chromatin organization and regulation. Specifically, it has been recently reported that phase separation of heterochromatin protein 1alpha (HP1) with DNA contributes to the formation of condensed chromatin states. HP1 localization to heterochromatic regions is mediated by its binding to specific repressive marks on the tail of histone H3, such as trimethylated lysine 9 on histone H3 (H3K9me3). However, whether epigenetic marks play an active role in modulating the material properties of HP1 and dictating emergent functions of its condensates, remains only partially understood. Here, we leverage a reductionist system, comprised of modified and unmodified histone H3 peptides, HP1 and DNA to examine the contribution of specific epigenetic marks to phase behavior of HP1. We show that the presence of histone peptides bearing the repressive H3K9me3 is compatible with HP1 condensates, while peptides containing unmodified residues or bearing the transcriptional activation mark H3K4me3 are incompatible with HP1 phase separation. In addition, inspired by the decreased ratio of nuclear H3K9me3 to HP1 detected in cells exposed to uniaxial strain, using fluorescence microscopy and rheological approaches we demonstrate that H3K9me3 histone peptides modulate the dynamics and network properties of HP1 condensates in a concentration dependent manner. These data suggest that HP1-DNA condensates are viscoelastic materials, whose properties may provide an explanation for the dynamic behavior of heterochromatin in cells in response to mechanostimulation.

Statement of significanceThe organization of genomic information in eukaryotic cells necessitates compartmentalization into functional domains allowing for the expression of cell identity-specific genes, while repressing genes related to alternative fates. Heterochromatin hosts these transcriptionally silent regions of the genome - which ensure the stability of cell identity -and is characterized by repressive histone marks (H3K9m3) and other specialized proteins (HP1a), recently shown to phase separate with DNA. We show that HP1a forms condensates with DNA which persist in the presence of H3K9me3 peptides. The viscoelastic nature of these condensates depend on H3K9me3:HP1 ratios, which are modulated by mechanical strain in cells. Thus, phase separation may explain the dynamic behavior of heterochromatin in cells, in response to mechanostimulation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
