<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity
Authors: Motone, K.; Kontogiorgos-Heintz, D.; Wee, J.; Kurihara, K.; Yang, S.; Roote, G.; Fang, Y.; Cardozo, N.; Nivala, J.
Score: 168.6, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563182
The ability to sequence single protein molecules in their native, full-length form would enable a more comprehensive understanding of proteomic diversity. Current technologies, however, are limited in achieving this goal. Here, we establish a method for long-range, single-molecule reading of intact protein strands on a commercial nanopore sensor array.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity
Authors: Motone, K.; Kontogiorgos-Heintz, D.; Wee, J.; Kurihara, K.; Yang, S.; Roote, G.; Fang, Y.; Cardozo, N.; Nivala, J.
Score: 168.6, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563182
The ability to sequence single protein molecules in their native, full-length form would enable a more comprehensive understanding of proteomic diversity. Current technologies, however, are limited in achieving this goal. Here, we establish a method for long-range, single-molecule reading of intact protein strands on a commercial nanopore sensor array." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-29T10:37:40+00:00" />
<meta property="article:modified_time" content="2023-10-29T10:37:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity
Authors: Motone, K.; Kontogiorgos-Heintz, D.; Wee, J.; Kurihara, K.; Yang, S.; Roote, G.; Fang, Y.; Cardozo, N.; Nivala, J.
Score: 168.6, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563182
The ability to sequence single protein molecules in their native, full-length form would enable a more comprehensive understanding of proteomic diversity. Current technologies, however, are limited in achieving this goal. Here, we establish a method for long-range, single-molecule reading of intact protein strands on a commercial nanopore sensor array."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity\nAuthors: Motone, K.; Kontogiorgos-Heintz, D.; Wee, J.; Kurihara, K.; Yang, S.; Roote, G.; Fang, Y.; Cardozo, N.; Nivala, J.\nScore: 168.6, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563182\nThe ability to sequence single protein molecules in their native, full-length form would enable a more comprehensive understanding of proteomic diversity. Current technologies, however, are limited in achieving this goal. Here, we establish a method for long-range, single-molecule reading of intact protein strands on a commercial nanopore sensor array.",
  "keywords": [
    
  ],
  "articleBody": " Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity\nAuthors: Motone, K.; Kontogiorgos-Heintz, D.; Wee, J.; Kurihara, K.; Yang, S.; Roote, G.; Fang, Y.; Cardozo, N.; Nivala, J.\nScore: 168.6, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563182\nThe ability to sequence single protein molecules in their native, full-length form would enable a more comprehensive understanding of proteomic diversity. Current technologies, however, are limited in achieving this goal. Here, we establish a method for long-range, single-molecule reading of intact protein strands on a commercial nanopore sensor array. By using the ClpX unfoldase to ratchet proteins through a CsgG nanopore, we achieve single-amino acid level sensitivity, enabling sequencing of combinations of amino acid substitutions across long protein strands. For greater sequencing accuracy, we demonstrate the ability to reread individual protein molecules, spanning hundreds of amino acids in length, multiple times, and explore the potential for high accuracy protein barcode sequencing. Further, we develop a biophysical model that can simulate raw nanopore signals a priori, based on amino acid volume and charge, enhancing the interpretation of raw signal data. Finally, we apply these methods to examine intact, folded protein domains for complete end-to-end analysis. These results provide proof-of-concept for a platform that has the potential to identify and characterize full-length proteoforms at single-molecule resolution.\nData-driven regularisation lowers the size barrier of cryo-EM structure determination\nAuthors: Kimanius, D.; Jamali, K.; Wilkinson, M. E.; Lövestam, S.; Velazhahan, V.; Nakane, T.; Scheres, S. H.\nScore: 106.2, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563586\nMacromolecular structure determination by electron cryo-microscopy (cryo-EM) is limited by the alignment of noisy images of individual particles. Because smaller particles have weaker signals, alignment errors impose size limitations on its applicability. Here, we explore how image alignment is improved by the application of deep-learning to exploit prior knowledge about biological macromolecular structures that would otherwise be difficult to express mathematically. We train a denoising convolutional neural network on pairs of half-set reconstructions from the electron microscopy data bank (EMDB) and use this denoiser as an alternative to a commonly used smoothness prior. We demonstrate that this approach, which we call Blush regularisation, yields better reconstructions than existing algorithms, in particular for data with low signal-to-noise ratios. The reconstruction of a protein-nucleic acid complex with a molecular weight of 40 kDa, which was previously intractable, illustrates that regularisation through denoising will expand the applicability of cryo-EM structure determination for a wide range of biological macromolecules.\nAutomated 3D multi-color single-molecule localization microscopy\nAuthors: Power, R. M.; Tschanz, A.; Zimmermann, T.; Ries, J.\nScore: 51.1, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563122\nSince its inception, single molecule localization microscopy (SMLM) has enabled imaging scientists to visualize biological structures with unprecedented resolution. Particularly powerful implementations capable of 3D, multi-color and high-throughput imaging have yielded key biological insights although widespread access to such technologies has been limited. The purpose of this protocol is to provide a guide for interested researchers to establish high-end SMLM in their laboratories. We detail the initial configuration and subsequent assembly of the SMLM, including instructions for alignment of all optical pathways, software/hardware integration and operation of the instrument. We describe validation steps including the preparation and imaging of test- and biological samples with structures of well-defined geometry and assist the user in troubleshooting and benchmarking performance. Additionally, we provide a walkthrough of the reconstruction of a super-resolved dataset from acquired raw images using the Super-resolution Microscopy Analysis Platform (SMAP). Depending on the instrument configuration, the cost of components is in the range $80,000 - 160,000, a fraction of the cost of a commercial instrument. A builder with some experience of optical systems is expected to require 3 - 6 months from the start of system construction to attain high-quality 3D and multi-color biological images.\nDesign of intrinsically disordered protein variants with diverse structural properties\nAuthors: Pesce, F.; Bremer, A.; Tesei, G.; Hopkins, J. B.; Grace, C. R.; Mittag, T.; Lindorff-Larsen, K.\nScore: 28.6, Published: 2023-10-24 DOI: 10.1101/2023.10.22.563461\nIntrinsically disordered proteins (IDPs) perform a wide range of functions in biology, suggesting that the ability to design IDPs could help expand the repertoire of proteins with novel functions. Designing IDPs with specific structural or functional properties has, however, been diffcult, in part because determining accurate conformational ensembles of IDPs generally requires a combination of computational modelling and experiments. Motivated by recent advancements in effcient physics-based models for simulations of IDPs, we have developed a general algorithm for designing IDPs with specific structural properties. We demonstrate the power of the algorithm by generating variants of naturally occurring IDPs with different levels of compaction and that vary more than 100 fold in their propensity to undergo phase separation, even while keeping a fixed amino acid composition. We experimentally tested designs of variants of the low-complexity domain of hnRNPA1 and find high accuracy in our computational predictions, both in terms of single-chain compaction and propensity to undergo phase separation. We analyze the sequence features that determine changes in compaction and propensity to phase separate and find an overall good agreement with previous findings for naturally occurring sequences. Our general, physics-based method enables the design of disordered sequences with specified conformational properties. Our algorithm thus expands the toolbox for protein design to include also the most flexible proteins and will enable the design of proteins whose functions exploit the many properties afforded by protein disorder.\nInterfacial morphodynamics of proliferating microbial communities\nAuthors: Martinez-Calvo, A.; Trenado-Yuste, C.; Lee, H.; Gore, J.; Wingreen, N. S.; Datta, S. S.\nScore: 27.8, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563665\nIn microbial communities, various cell types often coexist by occupying distinct spatial domains. What determines the shape of the interface between such domains--which in turn influences the interactions between cells and overall community function? Here, we address this question by developing a continuum model of a 2D spatially-structured microbial community with two distinct cell types. We find that, depending on the balance of the different cell proliferation rates and substrate friction coefficients, the interface between domains is either stable and smooth, or unstable and develops finger-like protrusions. We establish quantitative principles describing when these different interfacial behaviors arise, and find good agreement both with the results of previous experimental reports as well as new experiments performed here. Our work thus helps to provide a biophysical basis for understanding the interfacial morphodynamics of proliferating microbial communities, as well as a broader range of proliferating active systems.\nStructure of the human heparan-α-glucosaminide N-acetyltransferase (HGSNAT)\nAuthors: Navratna, V.; Kumar, A.; Mosalaganti, S.\nScore: 21.1, Published: 2023-10-26 DOI: 10.1101/2023.10.23.563672\nDegradation of heparan sulfate (HS), a glycosaminoglycan (GAG) comprised of repeating units of N-acetylglucosamine and glucuronic acid, begins in the cytosol and is completed in the lysosomes. Acetylation of the terminal non-reducing amino group of -D-glucosamine of HS is essential for its complete breakdown into monosaccharides and free sulfate. Heparan--glucosaminide N-acetyltransferase (HGSNAT), a resident of the lysosomal membrane, catalyzes this essential acetylation reaction by accepting and transferring the acetyl group from cytosolic acetyl-CoA to terminal -D-glucosamine of HS in the lysosomal lumen. Mutation-induced dysfunction in HGSNAT causes abnormal accumulation of HS within the lysosomes and leads to an autosomal recessive neurodegenerative lysosomal storage disorder called mucopolysaccharidosis IIIC (MPS IIIC). There are no approved drugs or treatment strategies to cure or manage the symptoms of, MPS IIIC. Here, we use cryo-electron microscopy (cryo-EM) to determine a high-resolution structure of the HGSNAT-acetyl-CoA complex in an open-to-lumen conformation, the first step in HGSNAT catalyzed acetyltransferase reaction. In addition, we map the known MPS IIIC mutations onto the structure and elucidate the molecular basis for mutation-induced HGSNAT dysfunction.\nStructure and regulation of GSDMD pores at the plasma membrane of pyroptotic cells\nAuthors: Kappelhoff, S.; Holtmannspoetter, M.; Schaefer, S. L.; Margheritis, E. G.; Veit, H.; Danial, J. S. H.; Strauss, S.; Franzkoch, R.; Psathaki, O. E.; Jungmann, R.; Kurre, R.; Hummer, G.; Piehler, J.; Cosentino, K.\nScore: 14.8, Published: 2023-10-24 DOI: 10.1101/2023.10.24.563742\nGasdermin D (GSDMD) executes inflammatory cell death pyroptosis by permeabilizing the plasma membrane (PM). We introduce polymer-supported PM (PSPM) to gain access to the cytoplasmic side of the PM with imaging techniques while preserving the native PM complexity and lipid microenvironment. By combining PSPM with DNA-PAINT super-resolution microscopy we visualized, for the first time, GSDMD nanostructures directly at the PM of pyroptotic cells. We resolved diverse macromolecular architectures with ring-and arc-shaped GSDMD oligomers that enable PM permeabilization. The pyroptotically-inactive mutant GSDMD-C192A (human C191A) still interacts with the PM however fails to form pores. GSDMD expression levels affect pore density but not permeabilization ability. Finally, we identified the local PI(3,4,5)P3 concentration as a key regulatory element of PM permeabilization. Increase in PI(3,4,5)P3 levels in the PM during pyroptosis facilitates growth into large ring-shaped pores. Using molecular dynamics (MD) simulations, we identified the mechanism by which PI(3,4,5)P3 stabilizes the GSDMD assembly.\nQuantifying fluorescence lifetime responsiveness of environment sensitive probes for membrane fluidity measurements\nAuthors: Ragaller, F.; Sjule, E.; Urem, Y. B.; Schlegel, J.; El, R.; Urbancic, D.; Urbancic, I.; Blom, H.; Sezgin, E.\nScore: 14.5, Published: 2023-10-25 DOI: 10.1101/2023.10.23.563572\nThe structural diversity of different lipid species within the membrane defines its biophysical properties such as membrane fluidity, phase transition, curvature, charge distribution and tension. Environment-sensitive probes, which change their spectral properties in response to their surrounding milieu, have greatly contributed to our understanding of such biophysical properties. To realize the full potential of these probes and to avoid misinterpretation of their spectral responses, a detailed investigation of their fluorescence characteristics in different environments is necessary. Here, we examined fluorescence lifetime of two newly developed membrane order probes, NR12S and NR12A, in response to alterations in their environments such as degree of lipid saturation, cholesterol content, double bond position and configuration and phospholipid headgroup. As comparison, we investigated lifetime sensitivity of the membrane tension probe Flipper in these environments. Applying fluorescence lifetime imaging microscopy (FLIM) in both model membranes and biological membranes, all probes distinguished membrane phases by lifetime, but exhibited different lifetime sensitivities to varying membrane biophysical properties (e.g. cholesterol). While the lifetime of Flipper is particularly sensitive to membrane cholesterol content, NR12S and NR12A lifetime is moderately sensitive to both cholesterol content and lipid acyl chains. Moreover, all probes exhibit longer lifetimes at longer emission wavelengths in membranes of any complexity. This emission-wavelength dependency results in varying lifetime resolution at different spectral regions, highly relevant for FLIM data acquisition. Our data provides valuable insights on how to perform FLIM with these probes and highlights both their potential and limitations.\nChemotactic agents combining spatial and temporal gradient-sensing boost spatial comparison if they are large, slow, and less persistent\nAuthors: Rode, J.; Novak, M.; Friedrich, B. M.\nScore: 13.5, Published: 2023-10-17 DOI: 10.1101/2023.10.14.562229\nBiological cells and small organisms navigate in concentration fields of signaling molecules using two fundamental gradient-sensing strategies: spatial comparison of concentrations measured at different positions on their surface, or temporal comparison of concentrations measured at different locations visited along their motion path. It is believed that size and speed dictate which gradient-sensing strategy cells choose, yet this has never been formally proven. Using information theory, we investigate the optimal gradient-sensing mechanism from the perspective of an ideal chemotactic agent that combines spatial and temporal comparison. We account for physical limits of chemo-sensation: molecule counting noise at physiological concentrations, and motility noise inevitable at the micro-scale. Our simulation data collapses onto an empirical power-law that predicts an optimal weighting of information as function of motility and sensing noise, demonstrating how spatial comparison becomes more beneficial for agents that are large, slow and less persistent. This refines and quantifies the previous heuristic notion. Our idealized model provides a base-line for biological chemotaxis and may inspire the design of robots guided by scalar fields.\nUniversal inverse modelling of point spread functions for SMLM localization and microscope characterization\nAuthors: Liu, S.; Chen, J.; Hellgoth, J.; Mueller, L.-R.; Ferdman, B.; Karras, C.; Xiao, D.; Lidke, K. A.; Heintzmann, R.; Shechtman, Y.; Li, Y.; Ries, J.\nScore: 11.9, Published: 2023-10-26 DOI: 10.1101/2023.10.26.564064\nThe point spread function (PSF) of a microscope describes the image of a point emitter. Knowing the accurate PSF model is essential for various imaging tasks, including single molecule localization, aberration correction and deconvolution. Here we present uiPSF (universal inverse modelling of Point Spread Functions), a toolbox to infer accurate PSF models from microscopy data, using either image stacks of fluorescent beads or directly images of blinking fluorophores, the raw data in single molecule localization microscopy (SMLM). The resulting PSF model enables accurate 3D super-resolution imaging using SMLM. Additionally, uiPSF can be used to characterize and optimize a microscope system by quantifying the aberrations, including field-dependent aberrations, and resolutions. Our modular framework is applicable to a variety of microscope modalities and the PSF model incorporates system or sample specific characteristics, e.g., the bead size, depth dependent aberrations and transformations among channels. We demonstrate its application in single or multiple channels or large field-of-view SMLM systems, 4Pi-SMLM, and lattice light-sheet microscopes using either bead data or single molecule blinking data.\n",
  "wordCount" : "2145",
  "inLanguage": "en",
  "datePublished": "2023-10-29T10:37:40Z",
  "dateModified": "2023-10-29T10:37:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on October 29, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563182">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563182" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563182">
        <p class="paperTitle">Multi-pass, single-molecule nanopore reading of long protein strands with single-amino acid sensitivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563182" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563182" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Motone, K.; Kontogiorgos-Heintz, D.; Wee, J.; Kurihara, K.; Yang, S.; Roote, G.; Fang, Y.; Cardozo, N.; Nivala, J.</p>
        <p class="info">Score: 168.6, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563182' target='https://doi.org/10.1101/2023.10.19.563182'> 10.1101/2023.10.19.563182</a></p>
        <p class="abstract">The ability to sequence single protein molecules in their native, full-length form would enable a more comprehensive understanding of proteomic diversity. Current technologies, however, are limited in achieving this goal. Here, we establish a method for long-range, single-molecule reading of intact protein strands on a commercial nanopore sensor array. By using the ClpX unfoldase to ratchet proteins through a CsgG nanopore, we achieve single-amino acid level sensitivity, enabling sequencing of combinations of amino acid substitutions across long protein strands. For greater sequencing accuracy, we demonstrate the ability to reread individual protein molecules, spanning hundreds of amino acids in length, multiple times, and explore the potential for high accuracy protein barcode sequencing. Further, we develop a biophysical model that can simulate raw nanopore signals a priori, based on amino acid volume and charge, enhancing the interpretation of raw signal data. Finally, we apply these methods to examine intact, folded protein domains for complete end-to-end analysis. These results provide proof-of-concept for a platform that has the potential to identify and characterize full-length proteoforms at single-molecule resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563586">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563586" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563586">
        <p class="paperTitle">Data-driven regularisation lowers the size barrier of cryo-EM structure determination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563586" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563586" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kimanius, D.; Jamali, K.; Wilkinson, M. E.; Lövestam, S.; Velazhahan, V.; Nakane, T.; Scheres, S. H.</p>
        <p class="info">Score: 106.2, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563586' target='https://doi.org/10.1101/2023.10.23.563586'> 10.1101/2023.10.23.563586</a></p>
        <p class="abstract">Macromolecular structure determination by electron cryo-microscopy (cryo-EM) is limited by the alignment of noisy images of individual particles. Because smaller particles have weaker signals, alignment errors impose size limitations on its applicability. Here, we explore how image alignment is improved by the application of deep-learning to exploit prior knowledge about biological macromolecular structures that would otherwise be difficult to express mathematically. We train a denoising convolutional neural network on pairs of half-set reconstructions from the electron microscopy data bank (EMDB) and use this denoiser as an alternative to a commonly used smoothness prior. We demonstrate that this approach, which we call Blush regularisation, yields better reconstructions than existing algorithms, in particular for data with low signal-to-noise ratios. The reconstruction of a protein-nucleic acid complex with a molecular weight of 40 kDa, which was previously intractable, illustrates that regularisation through denoising will expand the applicability of cryo-EM structure determination for a wide range of biological macromolecules.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563122">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563122" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563122">
        <p class="paperTitle">Automated 3D multi-color single-molecule localization microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563122" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563122" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Power, R. M.; Tschanz, A.; Zimmermann, T.; Ries, J.</p>
        <p class="info">Score: 51.1, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563122' target='https://doi.org/10.1101/2023.10.23.563122'> 10.1101/2023.10.23.563122</a></p>
        <p class="abstract">Since its inception, single molecule localization microscopy (SMLM) has enabled imaging scientists to visualize biological structures with unprecedented resolution. Particularly powerful implementations capable of 3D, multi-color and high-throughput imaging have yielded key biological insights although widespread access to such technologies has been limited. The purpose of this protocol is to provide a guide for interested researchers to establish high-end SMLM in their laboratories. We detail the initial configuration and subsequent assembly of the SMLM, including instructions for alignment of all optical pathways, software/hardware integration and operation of the instrument. We describe validation steps including the preparation and imaging of test- and biological samples with structures of well-defined geometry and assist the user in troubleshooting and benchmarking performance. Additionally, we provide a walkthrough of the reconstruction of a super-resolved dataset from acquired raw images using the Super-resolution Microscopy Analysis Platform (SMAP). Depending on the instrument configuration, the cost of components is in the range $80,000 - 160,000, a fraction of the cost of a commercial instrument. A builder with some experience of optical systems is expected to require 3 - 6 months from the start of system construction to attain high-quality 3D and multi-color biological images.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.22.563461">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.22.563461" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.22.563461">
        <p class="paperTitle">Design of intrinsically disordered protein variants with diverse structural properties</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.22.563461" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.22.563461" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pesce, F.; Bremer, A.; Tesei, G.; Hopkins, J. B.; Grace, C. R.; Mittag, T.; Lindorff-Larsen, K.</p>
        <p class="info">Score: 28.6, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.22.563461' target='https://doi.org/10.1101/2023.10.22.563461'> 10.1101/2023.10.22.563461</a></p>
        <p class="abstract">Intrinsically disordered proteins (IDPs) perform a wide range of functions in biology, suggesting that the ability to design IDPs could help expand the repertoire of proteins with novel functions. Designing IDPs with specific structural or functional properties has, however, been diffcult, in part because determining accurate conformational ensembles of IDPs generally requires a combination of computational modelling and experiments. Motivated by recent advancements in effcient physics-based models for simulations of IDPs, we have developed a general algorithm for designing IDPs with specific structural properties. We demonstrate the power of the algorithm by generating variants of naturally occurring IDPs with different levels of compaction and that vary more than 100 fold in their propensity to undergo phase separation, even while keeping a fixed amino acid composition. We experimentally tested designs of variants of the low-complexity domain of hnRNPA1 and find high accuracy in our computational predictions, both in terms of single-chain compaction and propensity to undergo phase separation. We analyze the sequence features that determine changes in compaction and propensity to phase separate and find an overall good agreement with previous findings for naturally occurring sequences. Our general, physics-based method enables the design of disordered sequences with specified conformational properties. Our algorithm thus expands the toolbox for protein design to include also the most flexible proteins and will enable the design of proteins whose functions exploit the many properties afforded by protein disorder.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563665">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563665" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563665">
        <p class="paperTitle">Interfacial morphodynamics of proliferating microbial communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563665" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563665" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Calvo, A.; Trenado-Yuste, C.; Lee, H.; Gore, J.; Wingreen, N. S.; Datta, S. S.</p>
        <p class="info">Score: 27.8, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563665' target='https://doi.org/10.1101/2023.10.23.563665'> 10.1101/2023.10.23.563665</a></p>
        <p class="abstract">In microbial communities, various cell types often coexist by occupying distinct spatial domains. What determines the shape of the interface between such domains--which in turn influences the interactions between cells and overall community function? Here, we address this question by developing a continuum model of a 2D spatially-structured microbial community with two distinct cell types. We find that, depending on the balance of the different cell proliferation rates and substrate friction coefficients, the interface between domains is either stable and smooth, or unstable and develops finger-like protrusions. We establish quantitative principles describing when these different interfacial behaviors arise, and find good agreement both with the results of previous experimental reports as well as new experiments performed here. Our work thus helps to provide a biophysical basis for understanding the interfacial morphodynamics of proliferating microbial communities, as well as a broader range of proliferating active systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563672">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563672" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563672">
        <p class="paperTitle">Structure of the human heparan-α-glucosaminide N-acetyltransferase (HGSNAT)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563672" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563672" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Navratna, V.; Kumar, A.; Mosalaganti, S.</p>
        <p class="info">Score: 21.1, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563672' target='https://doi.org/10.1101/2023.10.23.563672'> 10.1101/2023.10.23.563672</a></p>
        <p class="abstract">Degradation of heparan sulfate (HS), a glycosaminoglycan (GAG) comprised of repeating units of N-acetylglucosamine and glucuronic acid, begins in the cytosol and is completed in the lysosomes. Acetylation of the terminal non-reducing amino group of -D-glucosamine of HS is essential for its complete breakdown into monosaccharides and free sulfate. Heparan--glucosaminide N-acetyltransferase (HGSNAT), a resident of the lysosomal membrane, catalyzes this essential acetylation reaction by accepting and transferring the acetyl group from cytosolic acetyl-CoA to terminal -D-glucosamine of HS in the lysosomal lumen. Mutation-induced dysfunction in HGSNAT causes abnormal accumulation of HS within the lysosomes and leads to an autosomal recessive neurodegenerative lysosomal storage disorder called mucopolysaccharidosis IIIC (MPS IIIC). There are no approved drugs or treatment strategies to cure or manage the symptoms of, MPS IIIC. Here, we use cryo-electron microscopy (cryo-EM) to determine a high-resolution structure of the HGSNAT-acetyl-CoA complex in an open-to-lumen conformation, the first step in HGSNAT catalyzed acetyltransferase reaction. In addition, we map the known MPS IIIC mutations onto the structure and elucidate the molecular basis for mutation-induced HGSNAT dysfunction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563742" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563742">
        <p class="paperTitle">Structure and regulation of GSDMD pores at the plasma membrane of pyroptotic cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kappelhoff, S.; Holtmannspoetter, M.; Schaefer, S. L.; Margheritis, E. G.; Veit, H.; Danial, J. S. H.; Strauss, S.; Franzkoch, R.; Psathaki, O. E.; Jungmann, R.; Kurre, R.; Hummer, G.; Piehler, J.; Cosentino, K.</p>
        <p class="info">Score: 14.8, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563742' target='https://doi.org/10.1101/2023.10.24.563742'> 10.1101/2023.10.24.563742</a></p>
        <p class="abstract">Gasdermin D (GSDMD) executes inflammatory cell death pyroptosis by permeabilizing the plasma membrane (PM). We introduce polymer-supported PM (PSPM) to gain access to the cytoplasmic side of the PM with imaging techniques while preserving the native PM complexity and lipid microenvironment. By combining PSPM with DNA-PAINT super-resolution microscopy we visualized, for the first time, GSDMD nanostructures directly at the PM of pyroptotic cells. We resolved diverse macromolecular architectures with ring-and arc-shaped GSDMD oligomers that enable PM permeabilization. The pyroptotically-inactive mutant GSDMD-C192A (human C191A) still interacts with the PM however fails to form pores. GSDMD expression levels affect pore density but not permeabilization ability. Finally, we identified the local PI(3,4,5)P3 concentration as a key regulatory element of PM permeabilization. Increase in PI(3,4,5)P3 levels in the PM during pyroptosis facilitates growth into large ring-shaped pores. Using molecular dynamics (MD) simulations, we identified the mechanism by which PI(3,4,5)P3 stabilizes the GSDMD assembly.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563572">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563572" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563572">
        <p class="paperTitle">Quantifying fluorescence lifetime responsiveness of environment sensitive probes for membrane fluidity measurements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563572" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563572" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ragaller, F.; Sjule, E.; Urem, Y. B.; Schlegel, J.; El, R.; Urbancic, D.; Urbancic, I.; Blom, H.; Sezgin, E.</p>
        <p class="info">Score: 14.5, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563572' target='https://doi.org/10.1101/2023.10.23.563572'> 10.1101/2023.10.23.563572</a></p>
        <p class="abstract">The structural diversity of different lipid species within the membrane defines its biophysical properties such as membrane fluidity, phase transition, curvature, charge distribution and tension. Environment-sensitive probes, which change their spectral properties in response to their surrounding milieu, have greatly contributed to our understanding of such biophysical properties. To realize the full potential of these probes and to avoid misinterpretation of their spectral responses, a detailed investigation of their fluorescence characteristics in different environments is necessary. Here, we examined fluorescence lifetime of two newly developed membrane order probes, NR12S and NR12A, in response to alterations in their environments such as degree of lipid saturation, cholesterol content, double bond position and configuration and phospholipid headgroup. As comparison, we investigated lifetime sensitivity of the membrane tension probe Flipper in these environments. Applying fluorescence lifetime imaging microscopy (FLIM) in both model membranes and biological membranes, all probes distinguished membrane phases by lifetime, but exhibited different lifetime sensitivities to varying membrane biophysical properties (e.g. cholesterol). While the lifetime of Flipper is particularly sensitive to membrane cholesterol content, NR12S and NR12A lifetime is moderately sensitive to both cholesterol content and lipid acyl chains. Moreover, all probes exhibit longer lifetimes at longer emission wavelengths in membranes of any complexity. This emission-wavelength dependency results in varying lifetime resolution at different spectral regions, highly relevant for FLIM data acquisition. Our data provides valuable insights on how to perform FLIM with these probes and highlights both their potential and limitations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.562229">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.562229" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.562229">
        <p class="paperTitle">Chemotactic agents combining spatial and temporal gradient-sensing boost spatial comparison if they are large, slow, and less persistent</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.562229" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.562229" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rode, J.; Novak, M.; Friedrich, B. M.</p>
        <p class="info">Score: 13.5, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.562229' target='https://doi.org/10.1101/2023.10.14.562229'> 10.1101/2023.10.14.562229</a></p>
        <p class="abstract">Biological cells and small organisms navigate in concentration fields of signaling molecules using two fundamental gradient-sensing strategies: spatial comparison of concentrations measured at different positions on their surface, or temporal comparison of concentrations measured at different locations visited along their motion path. It is believed that size and speed dictate which gradient-sensing strategy cells choose, yet this has never been formally proven. Using information theory, we investigate the optimal gradient-sensing mechanism from the perspective of an ideal chemotactic agent that combines spatial and temporal comparison. We account for physical limits of chemo-sensation: molecule counting noise at physiological concentrations, and motility noise inevitable at the micro-scale. Our simulation data collapses onto an empirical power-law that predicts an optimal weighting of information as function of motility and sensing noise, demonstrating how spatial comparison becomes more beneficial for agents that are large, slow and less persistent. This refines and quantifies the previous heuristic notion. Our idealized model provides a base-line for biological chemotaxis and may inspire the design of robots guided by scalar fields.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564064">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564064" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564064">
        <p class="paperTitle">Universal inverse modelling of point spread functions for SMLM localization and microscope characterization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564064" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564064" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, S.; Chen, J.; Hellgoth, J.; Mueller, L.-R.; Ferdman, B.; Karras, C.; Xiao, D.; Lidke, K. A.; Heintzmann, R.; Shechtman, Y.; Li, Y.; Ries, J.</p>
        <p class="info">Score: 11.9, Published: 2023-10-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564064' target='https://doi.org/10.1101/2023.10.26.564064'> 10.1101/2023.10.26.564064</a></p>
        <p class="abstract">The point spread function (PSF) of a microscope describes the image of a point emitter. Knowing the accurate PSF model is essential for various imaging tasks, including single molecule localization, aberration correction and deconvolution. Here we present uiPSF (universal inverse modelling of Point Spread Functions), a toolbox to infer accurate PSF models from microscopy data, using either image stacks of fluorescent beads or directly images of blinking fluorophores, the raw data in single molecule localization microscopy (SMLM). The resulting PSF model enables accurate 3D super-resolution imaging using SMLM. Additionally, uiPSF can be used to characterize and optimize a microscope system by quantifying the aberrations, including field-dependent aberrations, and resolutions. Our modular framework is applicable to a variety of microscope modalities and the PSF model incorporates system or sample specific characteristics, e.g., the bead size, depth dependent aberrations and transformations among channels. We demonstrate its application in single or multiple channels or large field-of-view SMLM systems, 4Pi-SMLM, and lattice light-sheet microscopes using either bead data or single molecule blinking data.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
