<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Direct optical measurement of intra-molecular distances down to the Angstrom scale
Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.
Score: 123.9, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133
Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Direct optical measurement of intra-molecular distances down to the Angstrom scale
Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.
Score: 123.9, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133
Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-23T10:36:34+00:00" />
<meta property="article:modified_time" content="2023-07-23T10:36:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Direct optical measurement of intra-molecular distances down to the Angstrom scale
Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.
Score: 123.9, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133
Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Direct optical measurement of intra-molecular distances down to the Angstrom scale\nAuthors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.\nScore: 123.9, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133\nOptical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision.",
  "keywords": [
    
  ],
  "articleBody": " Direct optical measurement of intra-molecular distances down to the Angstrom scale\nAuthors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.\nScore: 123.9, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133\nOptical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision. Our method is validated by quantifying well-characterized 1-to-10 nm distances in polypeptides and proteins. Moreover, we visualize the orientations of immunoglobulin subunits and reveal specific configurations of a histidine kinase domain dimer. Our results open the door for examining proximities and interactions of macromolecules under physiological conditions.\nPEP-Patch: Electrostatics in Protein-Protein Recognition, Specificity and Antibody Developability\nAuthors: Waibl, F.; Pomarici, N. D.; Hoerschinger, V. J.; Loeffler, J. R.; Deane, C.; Georges, G.; Kettenberger, H.; Fernandez-Quintero, M. L.; Liedl, K. R.\nScore: 23.0, Published: 2023-07-15 DOI: 10.1101/2023.07.14.547811\nThe electrostatic properties of proteins arise from the number and distribution of polar and charged residues. Due to their long-ranged nature, electrostatic interactions in proteins play a critical role in numerous processes, such as molecular recognition, protein solubility, viscosity, and antibody developability. Thus, characterizing and quantifying electrostatic properties of a protein is a pre-requisite for understanding these processes. Here, we present PEP-Patch, a tool to visualize and quantify the electrostatic potential on the protein surface and showcase its applicability to elucidate protease substrate specificity, antibody-antigen recognition and predict heparin column retention times of antibodies as an indicator of pharmacokinetics.\nPhase-Separated Droplets Swim to Their Dissolution\nAuthors: Jambon-Puillet, E.; Testa, A.; Lorenz, C. M.; Style, R. W.; Rebane, A.; Dufresne, E. R.\nScore: 20.9, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549556\nBiological macromolecules can condense into liquid domains. In cells, these condensates form membraneless organelles that can organize chemical reactions1,2. However, little is known about the physical consequences of chemical activity in and around condensates. Working with model bovine serum albumin (BSA) condensates3, we show that droplets swim along chemical gradients. Active BSA droplets loaded with urease swim toward each other. Passive BSA droplets show diverse responses to externally applied gradients of the enzymes substrate and products. In all these cases, droplets swim toward solvent conditions that favor their dissolution. We call this behavior dialytaxis, and expect it to be generic, as conditions which favor dissolution typically reduce interfacial tension, whose gradients are well-known to drive droplet motion4,5. These results suggest alternative physical mechanisms for active transport in living cells, and may enable the design of fluid micro-robots.\nA Generalizable Scaffold-Based Approach for Structure Determination of RNAs by Cryo-EM\nAuthors: Kieft, J. S.; Langeberg, C.\nScore: 17.1, Published: 2023-07-06 DOI: 10.1101/2023.07.06.547879\nSingle-particle cryo-electron microscopy (cryo-EM) can reveal the structures of large and often dynamic molecules, but smaller biomolecules remain challenging targets due to their intrinsic low signal to noise ratio. Methods to resolve small proteins have been applied but development of similar approaches for small structured RNA elements have lagged. Here, we present a scaffold-based approach that we used to recover maps of sub-25 kDa RNA domains to 4.5 - 5.0 [A]. While lacking the detail of true high-resolution maps, these are suitable for model building and preliminary structure determination. We demonstrate this method faithfully recovers the structure of several RNA elements of known structure, and it promises to be generalized to other RNAs without disturbing their native fold. This approach may streamline the sample preparation process and reduce the optimization required for data collection. This first-generation scaffold approach provides a system for RNA structure determination by cryo-EM and lays the groundwork for further scaffold optimization to achieve higher resolution.\nSelf-propagating wave drives noncanonical antidurotaxis of skull bones in vivo\nAuthors: Dang, Y.; Lattner, J.; Lahola-Chomiak, A.; Alves-Afonso, D.; Ulbricht, E.; Taubenberger, A. V.; Rulands, S.; Tabler, J.\nScore: 16.3, Published: 2023-07-10 DOI: 10.1101/2023.07.10.547677\nCell motility is a key feature of tissue morphogenesis, and it is thought to be driven primarily by the active migration of individual cells or collectives. However, this model is unlikely to apply to cells lacking overt cytoskeletal, stable cell-cell or cell-cell adhesions, and molecular polarity, such as mesenchymal cells. Here, by combining a novel imaging pipeline with biophysical modeling, we discover that during skull morphogenesis, a self-generated collagen gradient expands a population of osteoblasts towards a softer matrix. Biomechanical measurements revealed a gradient of stiffness and collagen along which cells move and divide. The moving cells generate an osteogenic front that travels faster than individual tracked cells, indicating that expansion is also driven by cell differentiation. Through biophysical modeling and perturbation experiments, we found that mechanical feedback between stiffness and cell fate drives bone expansion and controls bone size. Our work provides a mechanism for coordinated motion that does not rely upon the cytoskeletal dynamics of cell migration. We term this self-propagating motion down a stiffness gradient, noncanonical antidurotaxis. Identification of alternative mechanisms of cellular motion will help in understanding how directed cellular motility arises in complex environments with inhomogeneous material properties.\nGuiding Discovery of Protein Sequence-Structure-Function Modeling\nAuthors: Hussain, A. L.; Brooks, C. L.\nScore: 12.2, Published: 2023-07-16 DOI: 10.1101/2023.07.14.548822\nProtein engineering techniques are key in designing novel catalysts for a wide range of reactions. Although approaches vary in their exploration of the sequence-structure-function paradigm, they are often hampered by the labor-intensive steps of protein expression and screening. In this work, we describe the development and testing of a high throughput in silico sequence-structure-function pipeline using AlphaFold2 and Fast Fourier Transform docking that is benchmarked with enantioselectivity and reactivity predictions for an ancestral sequence library of fungal flavin-dependent monooxygenases. The predicted enantioselectivities and reactivities correlate well with previously described screens of an experimentally available subset of these proteins and capture known changes in enantioselectivity across the phylogenetic tree representing ancestorial proteins from this family. With this pipeline established as our functional screen, we apply ensemble decision tree models and explainable AI techniques to build sequence function models and extract critical residues within the binding site and the second sphere residues around this site. We demonstrate that the top-identified key residues in the control of enantioselectivity and reactivity correspond to experimentally verified residues. The in silico sequence-to-function pipeline serves as an accelerated framework to inform protein engineering efforts.\nCoevolution-driven method for efficiently simulating conformational changes in proteins reveals molecular details of ligand effects in the beta2AR receptor\nAuthors: Mitrovic, D.; Chen, Y.; Marciniak, A.; Delemotte, L.\nScore: 10.8, Published: 2023-07-21 DOI: 10.1101/2023.07.20.549854\nWith the advent of AI-powered structure prediction, the scientific community is inching ever closer to solving protein folding. An unresolved enigma, however, is to accurately, reliably and deterministically predict alternative conformational states that are crucial for the function of e.g. transporters, receptors or ion channels where conformational cycling is innately coupled to protein function. Accurately discovering and exploring all conformational states of membrane proteins has been challenging due to the need to retain atomistic detail while enhancing the sampling along interesting degrees of freedom. The challenges include but are not limited to finding which degrees of freedom are relevant, how to accelerate the sampling along them, and then quantifying the populations of each micro- and macrostate. In this work, we present a methodology that finds the relevant degrees of freedom by combining evolution and physics through machine learning and apply it to the beta 2 adrenergic receptor conformational sampling. In addition to predicting new conformations that are beyond the training set, we have computed free energy surfaces associated with the conformational landscape of the protein. We then show that the methodology is able to quantitatively predict the effect of an array of ligands on the beta 2 adrenergic receptor activation, and that the full conformational landscape, including states related to biased signaling, is discovered using this procedure. Lastly, we also stake out the structural determinants of activation and inactivation pathway signaling through different ligands.\nTranscription regulates the spatio-temporal dynamics of genes through micro-compartmentalization\nAuthors: Salari, H.; Fourel, G.; Jost, D.\nScore: 9.8, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549489\nAlthough our understanding of the involvement of heterochromatin architectural factors in shaping nuclear organization is improving, there is still ongoing debate regarding the role of active genes in this process. In this study, we utilize publicly-available Micro-C data from mouse embryonic stem cells to investigate the relationship between gene transcription and 3D gene folding. Our analysis uncovers a nonmonotonic - globally positive - correlation between intragenic contact density and Pol II occupancy, independent of cohesin-based loop extrusion. Through the development of a biophysical model integrating the role of transcription dynamics within a polymer model of chromosome organization, we demonstrate that Pol II-mediated attractive interactions with limited valency between transcribed regions yield quantitative predictions consistent with Hi-C and live-imaging experiments. Our work provides compelling evidence that transcriptional activity shapes the 4D genome through Pol II-mediated micro-compartmentalization.\nLong-Distance Electron Transport in Multicellular Freshwater Cable Bacteria\nAuthors: Yang, T.; Chavez, M. S.; Niman, C. M.; Xu, S.; El-Naggar, M. Y.\nScore: 8.2, Published: 2023-07-16 DOI: 10.1101/2023.07.15.548776\nFilamentous multicellular cable bacteria perform centimeter-scale electron transport in a process that couples oxidation of an electron donor (sulfide) in deeper sediment to the reduction of an electron acceptor (oxygen or nitrate) near the surface. While this electric metabolism is prevalent in both marine and freshwater sediments, detailed electronic measurements of the conductivity previously focused on the marine cable bacteria (Candidatus Electrothrix), rather than freshwater cable bacteria, which form a separate genus (Candidatus Electronema) and contribute essential geochemical roles in freshwater sediments. Here, we characterize the electron transport characteristics of Ca. Electronema cable bacteria from Southern California freshwater sediments. Current-voltage measurements of intact cable filaments bridging interdigitated electrodes confirmed their persistent conductivity under a controlled atmosphere and the variable sensitivity of this conduction to air exposure. Electrostatic and conductive atomic force microscopies mapped out the characteristics of the cell envelopes nanofiber network, implicating it as the conductive pathway in a manner consistent with previous findings in marine cable bacteria. Four-probe measurements of microelectrodes addressing intact cables demonstrated nanoampere currents up to 200 m lengths at modest driving voltages, allowing us to quantify the nanofiber conductivity at 0.1 S/cm for freshwater cable bacteria filaments under our measurement conditions. Such a high conductivity can support the remarkable sulfide-to-oxygen electrical currents mediated by cable bacteria in sediments. These measurements expand the knowledgebase of long-distance electron transport to the freshwater niche while shedding light on underlying conductive network of cable bacteria. SignificanceCable bacteria are multicellular filaments composed of up-to-thousands of end-to-end cells and are found worldwide in both marine and freshwater sediments. Remarkably, these cells gain energy from a long-distance electron transport process that carries electrons generated by sulfide oxidation in deeper sediment layers to drive oxygen reduction near the sediment-water interface. This electric metabolism requires an unusually high electronic conductivity, previously thought impossible in natural biological materials. However, the underlying mechanism(s) remain poorly understood, and previous characterization of the conductivity largely focused on marine cable bacteria. Here, we characterize and quantify the electronic conductivity of freshwater cable bacteria from sediments in Southern California, with emphasis on the role of the conductive periplasmic fiber network in routing electron transport along cables.\nVirtual perturbations to assess explainability of deep-learning based cell fate predictors\nAuthors: Soelistyo, C. J.; Charras, G. R.; Lowe, A. R.\nScore: 7.3, Published: 2023-07-17 DOI: 10.1101/2023.07.17.548859\nExplainable deep learning holds significant promise in extracting scientific insights from experimental observations. This is especially so in the field of bio-imaging, where the raw data is often voluminous, yet extremely variable and difficult to study. However, one persistent challenge in deep learning assisted scientific discovery is that the workings of artificial neural networks are often difficult to interpret. Here we present a simple technique for investigating the behaviour of trained neural networks: virtual perturbation. By making precise and systematic alterations to input data or internal representations thereof, we are able to discover causal relationships in the outputs of a deep learning model, and by extension, in the underlying phenomenon itself. As an exemplar, we use our recently described deep-learning based cell fate prediction model. We trained the network to predict the fate of less fit cells in an experimental model of mechanical cell competition. By applying virtual perturbation to the trained network, we discover causal relationships between a cells environment and eventual fate. We compare these with known properties of the biological system under investigation to demonstrate that the model faithfully captures insights previously established by experimental research.\n",
  "wordCount" : "2086",
  "inLanguage": "en",
  "datePublished": "2023-07-23T10:36:34Z",
  "dateModified": "2023-07-23T10:36:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on July 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548133" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548133">
        <p class="paperTitle">Direct optical measurement of intra-molecular distances down to the Angstrom scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.</p>
        <p class="info">Score: 123.9, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548133' target='https://doi.org/10.1101/2023.07.07.548133'> 10.1101/2023.07.07.548133</a></p>
        <p class="abstract">Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision. Our method is validated by quantifying well-characterized 1-to-10 nm distances in polypeptides and proteins. Moreover, we visualize the orientations of immunoglobulin subunits and reveal specific configurations of a histidine kinase domain dimer. Our results open the door for examining proximities and interactions of macromolecules under physiological conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.547811">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.547811" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.547811">
        <p class="paperTitle">PEP-Patch: Electrostatics in Protein-Protein Recognition, Specificity and Antibody Developability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.547811" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.547811" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waibl, F.; Pomarici, N. D.; Hoerschinger, V. J.; Loeffler, J. R.; Deane, C.; Georges, G.; Kettenberger, H.; Fernandez-Quintero, M. L.; Liedl, K. R.</p>
        <p class="info">Score: 23.0, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.547811' target='https://doi.org/10.1101/2023.07.14.547811'> 10.1101/2023.07.14.547811</a></p>
        <p class="abstract">The electrostatic properties of proteins arise from the number and distribution of polar and charged residues. Due to their long-ranged nature, electrostatic interactions in proteins play a critical role in numerous processes, such as molecular recognition, protein solubility, viscosity, and antibody developability. Thus, characterizing and quantifying electrostatic properties of a protein is a pre-requisite for understanding these processes. Here, we present PEP-Patch, a tool to visualize and quantify the electrostatic potential on the protein surface and showcase its applicability to elucidate protease substrate specificity, antibody-antigen recognition and predict heparin column retention times of antibodies as an indicator of pharmacokinetics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549556">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549556" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549556">
        <p class="paperTitle">Phase-Separated Droplets Swim to Their Dissolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549556" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549556" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jambon-Puillet, E.; Testa, A.; Lorenz, C. M.; Style, R. W.; Rebane, A.; Dufresne, E. R.</p>
        <p class="info">Score: 20.9, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549556' target='https://doi.org/10.1101/2023.07.18.549556'> 10.1101/2023.07.18.549556</a></p>
        <p class="abstract">Biological macromolecules can condense into liquid domains. In cells, these condensates form membraneless organelles that can organize chemical reactions1,2. However, little is known about the physical consequences of chemical activity in and around condensates. Working with model bovine serum albumin (BSA) condensates3, we show that droplets swim along chemical gradients. Active BSA droplets loaded with urease swim toward each other. Passive BSA droplets show diverse responses to externally applied gradients of the enzymes substrate and products. In all these cases, droplets swim toward solvent conditions that favor their dissolution. We call this behavior dialytaxis, and expect it to be generic, as conditions which favor dissolution typically reduce interfacial tension, whose gradients are well-known to drive droplet motion4,5. These results suggest alternative physical mechanisms for active transport in living cells, and may enable the design of fluid micro-robots.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547879">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547879" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547879">
        <p class="paperTitle">A Generalizable Scaffold-Based Approach for Structure Determination of RNAs by Cryo-EM</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547879" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547879" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kieft, J. S.; Langeberg, C.</p>
        <p class="info">Score: 17.1, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547879' target='https://doi.org/10.1101/2023.07.06.547879'> 10.1101/2023.07.06.547879</a></p>
        <p class="abstract">Single-particle cryo-electron microscopy (cryo-EM) can reveal the structures of large and often dynamic molecules, but smaller biomolecules remain challenging targets due to their intrinsic low signal to noise ratio. Methods to resolve small proteins have been applied but development of similar approaches for small structured RNA elements have lagged. Here, we present a scaffold-based approach that we used to recover maps of sub-25 kDa RNA domains to 4.5 - 5.0 [A]. While lacking the detail of true high-resolution maps, these are suitable for model building and preliminary structure determination. We demonstrate this method faithfully recovers the structure of several RNA elements of known structure, and it promises to be generalized to other RNAs without disturbing their native fold. This approach may streamline the sample preparation process and reduce the optimization required for data collection. This first-generation scaffold approach provides a system for RNA structure determination by cryo-EM and lays the groundwork for further scaffold optimization to achieve higher resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.547677">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.547677" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.547677">
        <p class="paperTitle">Self-propagating wave drives noncanonical antidurotaxis of skull bones in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.547677" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.547677" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dang, Y.; Lattner, J.; Lahola-Chomiak, A.; Alves-Afonso, D.; Ulbricht, E.; Taubenberger, A. V.; Rulands, S.; Tabler, J.</p>
        <p class="info">Score: 16.3, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.547677' target='https://doi.org/10.1101/2023.07.10.547677'> 10.1101/2023.07.10.547677</a></p>
        <p class="abstract">Cell motility is a key feature of tissue morphogenesis, and it is thought to be driven primarily by the active migration of individual cells or collectives. However, this model is unlikely to apply to cells lacking overt cytoskeletal, stable cell-cell or cell-cell adhesions, and molecular polarity, such as mesenchymal cells. Here, by combining a novel imaging pipeline with biophysical modeling, we discover that during skull morphogenesis, a self-generated collagen gradient expands a population of osteoblasts towards a softer matrix. Biomechanical measurements revealed a gradient of stiffness and collagen along which cells move and divide. The moving cells generate an osteogenic front that travels faster than individual tracked cells, indicating that expansion is also driven by cell differentiation. Through biophysical modeling and perturbation experiments, we found that mechanical feedback between stiffness and cell fate drives bone expansion and controls bone size. Our work provides a mechanism for coordinated motion that does not rely upon the cytoskeletal dynamics of cell migration. We term this self-propagating motion down a stiffness gradient, noncanonical antidurotaxis. Identification of alternative mechanisms of cellular motion will help in understanding how directed cellular motility arises in complex environments with inhomogeneous material properties.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.548822">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.548822" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.548822">
        <p class="paperTitle">Guiding Discovery of Protein Sequence-Structure-Function Modeling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.548822" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.548822" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hussain, A. L.; Brooks, C. L.</p>
        <p class="info">Score: 12.2, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.548822' target='https://doi.org/10.1101/2023.07.14.548822'> 10.1101/2023.07.14.548822</a></p>
        <p class="abstract">Protein engineering techniques are key in designing novel catalysts for a wide range of reactions. Although approaches vary in their exploration of the sequence-structure-function paradigm, they are often hampered by the labor-intensive steps of protein expression and screening. In this work, we describe the development and testing of a high throughput in silico sequence-structure-function pipeline using AlphaFold2 and Fast Fourier Transform docking that is benchmarked with enantioselectivity and reactivity predictions for an ancestral sequence library of fungal flavin-dependent monooxygenases. The predicted enantioselectivities and reactivities correlate well with previously described screens of an experimentally available subset of these proteins and capture known changes in enantioselectivity across the phylogenetic tree representing ancestorial proteins from this family. With this pipeline established as our functional screen, we apply ensemble decision tree models and explainable AI techniques to build sequence function models and extract critical residues within the binding site and the second sphere residues around this site. We demonstrate that the top-identified key residues in the control of enantioselectivity and reactivity correspond to experimentally verified residues. The in silico sequence-to-function pipeline serves as an accelerated framework to inform protein engineering efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549854">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549854" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549854">
        <p class="paperTitle">Coevolution-driven method for efficiently simulating conformational changes in proteins reveals molecular details of ligand effects in the beta2AR receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549854" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549854" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mitrovic, D.; Chen, Y.; Marciniak, A.; Delemotte, L.</p>
        <p class="info">Score: 10.8, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549854' target='https://doi.org/10.1101/2023.07.20.549854'> 10.1101/2023.07.20.549854</a></p>
        <p class="abstract">With the advent of AI-powered structure prediction, the scientific community is inching ever closer to solving protein folding. An unresolved enigma, however, is to accurately, reliably and deterministically predict alternative conformational states that are crucial for the function of e.g. transporters, receptors or ion channels where conformational cycling is innately coupled to protein function. Accurately discovering and exploring all conformational states of membrane proteins has been challenging due to the need to retain atomistic detail while enhancing the sampling along interesting degrees of freedom. The challenges include but are not limited to finding which degrees of freedom are relevant, how to accelerate the sampling along them, and then quantifying the populations of each micro- and macrostate. In this work, we present a methodology that finds the relevant degrees of freedom by combining evolution and physics through machine learning and apply it to the beta 2 adrenergic receptor conformational sampling. In addition to predicting new conformations that are beyond the training set, we have computed free energy surfaces associated with the conformational landscape of the protein. We then show that the methodology is able to quantitatively predict the effect of an array of ligands on the beta 2 adrenergic receptor activation, and that the full conformational landscape, including states related to biased signaling, is discovered using this procedure. Lastly, we also stake out the structural determinants of activation and inactivation pathway signaling through different ligands.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549489">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549489" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549489">
        <p class="paperTitle">Transcription regulates the spatio-temporal dynamics of genes through micro-compartmentalization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549489" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549489" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Salari, H.; Fourel, G.; Jost, D.</p>
        <p class="info">Score: 9.8, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549489' target='https://doi.org/10.1101/2023.07.18.549489'> 10.1101/2023.07.18.549489</a></p>
        <p class="abstract">Although our understanding of the involvement of heterochromatin architectural factors in shaping nuclear organization is improving, there is still ongoing debate regarding the role of active genes in this process. In this study, we utilize publicly-available Micro-C data from mouse embryonic stem cells to investigate the relationship between gene transcription and 3D gene folding. Our analysis uncovers a nonmonotonic - globally positive - correlation between intragenic contact density and Pol II occupancy, independent of cohesin-based loop extrusion. Through the development of a biophysical model integrating the role of transcription dynamics within a polymer model of chromosome organization, we demonstrate that Pol II-mediated attractive interactions with limited valency between transcribed regions yield quantitative predictions consistent with Hi-C and live-imaging experiments. Our work provides compelling evidence that transcriptional activity shapes the 4D genome through Pol II-mediated micro-compartmentalization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.15.548776">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.15.548776" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.15.548776">
        <p class="paperTitle">Long-Distance Electron Transport in Multicellular Freshwater Cable Bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.15.548776" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.15.548776" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, T.; Chavez, M. S.; Niman, C. M.; Xu, S.; El-Naggar, M. Y.</p>
        <p class="info">Score: 8.2, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.15.548776' target='https://doi.org/10.1101/2023.07.15.548776'> 10.1101/2023.07.15.548776</a></p>
        <p class="abstract">Filamentous multicellular cable bacteria perform centimeter-scale electron transport in a process that couples oxidation of an electron donor (sulfide) in deeper sediment to the reduction of an electron acceptor (oxygen or nitrate) near the surface. While this electric metabolism is prevalent in both marine and freshwater sediments, detailed electronic measurements of the conductivity previously focused on the marine cable bacteria (Candidatus Electrothrix), rather than freshwater cable bacteria, which form a separate genus (Candidatus Electronema) and contribute essential geochemical roles in freshwater sediments. Here, we characterize the electron transport characteristics of Ca. Electronema cable bacteria from Southern California freshwater sediments. Current-voltage measurements of intact cable filaments bridging interdigitated electrodes confirmed their persistent conductivity under a controlled atmosphere and the variable sensitivity of this conduction to air exposure. Electrostatic and conductive atomic force microscopies mapped out the characteristics of the cell envelopes nanofiber network, implicating it as the conductive pathway in a manner consistent with previous findings in marine cable bacteria. Four-probe measurements of microelectrodes addressing intact cables demonstrated nanoampere currents up to 200 m lengths at modest driving voltages, allowing us to quantify the nanofiber conductivity at 0.1 S/cm for freshwater cable bacteria filaments under our measurement conditions. Such a high conductivity can support the remarkable sulfide-to-oxygen electrical currents mediated by cable bacteria in sediments. These measurements expand the knowledgebase of long-distance electron transport to the freshwater niche while shedding light on underlying conductive network of cable bacteria.

SignificanceCable bacteria are multicellular filaments composed of up-to-thousands of end-to-end cells and are found worldwide in both marine and freshwater sediments. Remarkably, these cells gain energy from a long-distance electron transport process that carries electrons generated by sulfide oxidation in deeper sediment layers to drive oxygen reduction near the sediment-water interface. This electric metabolism requires an unusually high electronic conductivity, previously thought impossible in natural biological materials. However, the underlying mechanism(s) remain poorly understood, and previous characterization of the conductivity largely focused on marine cable bacteria. Here, we characterize and quantify the electronic conductivity of freshwater cable bacteria from sediments in Southern California, with emphasis on the role of the conductive periplasmic fiber network in routing electron transport along cables.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.548859">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.548859" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.548859">
        <p class="paperTitle">Virtual perturbations to assess explainability of deep-learning based cell fate predictors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.548859" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.548859" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Soelistyo, C. J.; Charras, G. R.; Lowe, A. R.</p>
        <p class="info">Score: 7.3, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.548859' target='https://doi.org/10.1101/2023.07.17.548859'> 10.1101/2023.07.17.548859</a></p>
        <p class="abstract">Explainable deep learning holds significant promise in extracting scientific insights from experimental observations. This is especially so in the field of bio-imaging, where the raw data is often voluminous, yet extremely variable and difficult to study. However, one persistent challenge in deep learning assisted scientific discovery is that the workings of artificial neural networks are often difficult to interpret. Here we present a simple technique for investigating the behaviour of trained neural networks: virtual perturbation. By making precise and systematic alterations to input data or internal representations thereof, we are able to discover causal relationships in the outputs of a deep learning model, and by extension, in the underlying phenomenon itself. As an exemplar, we use our recently described deep-learning based cell fate prediction model. We trained the network to predict the fate of less fit cells in an experimental model of mechanical cell competition. By applying virtual perturbation to the trained network, we discover causal relationships between a cells environment and eventual fate. We compare these with known properties of the biological system under investigation to demonstrate that the model faithfully captures insights previously established by experimental research.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
