<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Quantification of cell phenotype transition manifolds with information geometry
Authors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.
Score: 18.5, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573500
Cell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Quantification of cell phenotype transition manifolds with information geometry
Authors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.
Score: 18.5, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573500
Cell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-03T10:39:43+00:00" />
<meta property="article:modified_time" content="2024-01-03T10:39:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Quantification of cell phenotype transition manifolds with information geometry
Authors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.
Score: 18.5, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573500
Cell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Quantification of cell phenotype transition manifolds with information geometry\nAuthors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.\nScore: 18.5, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573500\nCell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds.",
  "keywords": [
    
  ],
  "articleBody": " Quantification of cell phenotype transition manifolds with information geometry\nAuthors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.\nScore: 18.5, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573500\nCell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds. However, no available method currently exists for quantifying the manifolds from experimental data. Current methods, such as manifold learning, can only preserve topology information and can only be used for visualization. In this work, we present a novel method to quantify the cell phenotype transition manifolds using information geometry. Specifically, we transform single cells high-dimensional expression vectors into probability distributions with Gaussian embedding. The Fisher-Rao metric is then naturally defined in this embedding space. With Gaussian embedding, we calculate the Ollivier-Ricci curvature of each single cell. Our analyses revealed that cells with low curvature are related to critical transitions.To further analyze the invariant characteristics of the manifold in CPT, we calculated the information velocity of each single cell based on RNA velocity. The high information velocity regions correspond to the low curvature regions, indicating that the geometry can guide the dynamics of single cells on the manifolds. This method focuses on analyzing the invariant characteristics of the manifold in CPT, and the results illustrate that it can be a general approach for quantifying the CPT manifold.\nSingle-molecule detection and super-resolution imaging with a portable and adaptable 3D-printed microscopy platform (Brick-MIC)\nAuthors: Moya Munoz, G. G.; Brix, O.; Klocke, P.; Wendler, N. D.; Lerner, E.; Zijlstra, N.; Cordes, T.\nScore: 12.1, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573596\nOver the past decades, single-molecule spectroscopy and super-resolution microscopy have advanced significantly and by now represent important tools for life science research. Despite rapid progress and ongoing development, there is a growing gap between the state-of-the-art and what is accessible to non-optics specialists, e.g., biologists, biochemists, medical researchers, and labs with financial constraints. To bridge this gap, we introduce Brick-MIC, a versatile and affordable open-source 3D-printed micro-spectroscopy and imaging platform. Brick-MIC enables the integration of various fluorescence imaging techniques with single-molecule resolution within a single platform and enables exchange between different modalities within minutes. In this work, we present three variants of Brick-MIC that facilitate single-molecule fluorescence detection, fluorescence correlation spectroscopy and super-resolution imaging. With the three variants, we were able to observe conformational changes and absolute inter-dye distances in single macromolecules and perform single-molecule localization microscopy (STORM and PAINT) of DNA origami nanostructures. Detailed descriptions of the hardware and software components, as well as data analysis routines are provided, to allow non-optics specialist to operate their own Brick-MIC with minimal effort and investments. We foresee that our affordable, flexible, and open-source Brick-MIC platform will be a valuable tool for many laboratories worldwide.\nSmart Parallel Automated Cryo Electron tomography\nAuthors: Eisenstein, F.; Fukuda, Y.; Danev, R.\nScore: 39.0, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571776\nIn situ cryo electron tomography enables investigation of macromolecules in their native cellular environment. Data collection, however, requires an experienced operator and valuable microscope time to carefully select targets for tilt series acquisition. Here, we developed a workflow using machine learning approaches to fully automate the entire process, including lamella detection, biological feature segmentation, target selection, and tilt series acquisition, all without the need for human intervention.\nHigh-throughput computational discovery of inhibitory protein fragments with AlphaFold\nAuthors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.\nScore: 89.2, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572389\nPeptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells. However, it has thus far not been possible to predict de novo which of the many possible protein fragments bind their protein targets, let alone act as inhibitors. We have developed a computational method, FragFold, that employs AlphaFold to predict protein fragment binding to full-length protein targets in a high-throughput manner. Applying FragFold to thousands of fragments tiling across diverse proteins revealed peaks of predicted binding along each protein sequence. These predictions were compared with experimentally measured peaks of inhibitory activity in E. coli. We establish that our approach is a sensitive predictor of protein fragment function: Evaluating inhibitory fragments derived from known protein-protein interaction interfaces, we found 87% were predicted by FragFold to bind in a native-like mode. Across full protein sequences, 68% of FragFold-predicted binding peaks match experimentally measured inhibitory peaks. This is true even when the underlying inhibitory mechanism is unclear from existing structural data, and we find FragFold is able to predict novel binding modes for inhibitory fragments of unknown structure, explaining previous genetic and biochemical data for these fragments. The success rate of FragFold demonstrates that this computational approach should be broadly applicable for discovering inhibitory protein fragments across proteomes.\np23-FKBP51 chaperone complex regulates tau aggregation\nAuthors: Chakraborty, P.; Zweckstetter, M.\nScore: 5.9, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573415\nThe pathological deposition of tau into insoluble amyloid fibrils is a pathological hallmark of tauopathies. Molecular chaperones are important cellular factors contributing to the regulation of tau misfolding and aggregation. Here we reveal an Hsp90-independent modulation of tau aggregation by a molecular complex of the two Hsp90 co-chaperones p23 and FKBP51. Integrating NMR spectroscopy, SAXS, molecular docking, and site-directed mutagenesis we reveal the structural basis of the p23-FKBP51 complex. We show that p23 specifically recognizes the TPR domain of FKBP51 and interacts with tau through its C-terminal disordered tail. We further show that the p23-FKBP51 complex binds tau to form a dynamic p23-FKBP51-tau trimeric complex that inhibits tau aggregation and thus may counteract Hsp90-FKBP51 mediated toxicity. Taken together, our findings reveal a co-chaperone mediated Hsp90-independent chaperoning of tau protein.\nThe everting Drosophila wing disc is a shape-programmed material\nAuthors: Fuhrmann, J. F.; Krishna, A.; Paijmans, J.; Duclut, C. A.; Eaton, S.; Popovic, M.; Julicher, F.; Modes, C. D.; Dye, N. A.\nScore: 7.7, Published: 2023-12-24 DOI: 10.1101/2023.12.23.573034\nHow complex 3D tissue shape emerges during animal development remains an important open question in biology and biophysics. Although it is well-established that tissue-scale patterns of in-plane cell behaviors can reshape a tissue in 2D, whether and how such patterns could also deform a tissue in 3D remains unclear. Meanwhile, recent advances in materials science, physics, and engineering have enhanced our ability to craft and understand inanimate \"shape-programmable\" materials capable of undergoing blueprinted 3D shape transformations arising from in-plane gradients of spontaneous strains. Here, we use this conceptual framework to understand eversion of the Drosophila wing disc, a complex 3D tissue reshaping event in wing development. Using quantification of spatial patterns of cell dynamics on an evolving 3D tissue surface, combined with physical modeling and genetic perturbation, we demonstrate that the wing can deform itself in 3D through large-scale patterning of in-plane cellular behaviors, primarily rearrangements. This work shows that concepts from shape-programming can be employed to understand animal tissue morphogenesis and suggests that there exist intricate patterns in nature that could present novel designs for shape-programmable materials.\nStructure-based discovery of positive allosteric modulators for the calcium sensing receptor\nAuthors: Liu, F.; Wu, C.-G.; Tu, C.-L.; Glenn, I.; Meyerowitz, J.; Kaplan, A. L.; Lyu, J.; Cheng, Z.; Tarkhanova, O. O.; Moroz, Y. S.; Irwin, J. J.; Chang, W.; Shoichet, B. K.; Skiniotis, G.\nScore: 5.5, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573448\nDrugs acting as positive allosteric modulators (PAMs) to enhance the activation of the calcium sensing receptor (CaSR) and to suppress parathyroid hormone (PTH) secretion can treat hyperparathyroidism but suffer from side effects including hypocalcemia and arrhythmias. Seeking new CaSR modulators, we docked libraries of 2.7 million and 1.2 billion molecules against transforming pockets in the active-state receptor dimer structure. Consistent with the idea that docking improves with library size, billion-molecule docking found new PAMs with a hit rate that was 2.7-fold higher than the million-molecule library and with hits up to 37-fold more potent. Structure-based optimization of ligands from both campaigns led to nanomolar leads, one of which was advanced to animal testing. This PAM displays 100-fold the potency of the standard of care, cinacalcet, in ex vivo organ assays, and reduces serum PTH levels in mice by up to 80% without the hypocalcemia typical of CaSR drugs. Cryo-EM structures with the new PAMs show that they promote CaSR dimer conformations that are closer to the G-protein coupled state compared to established drugs. These findings highlight the promise of large library docking for therapeutic leads, especially when combined with experimental structure determination and mechanism.\nPhosphorylation regulates tau's phase separation behavior and interactions with chromatin\nAuthors: Abasi, L. S.; Elathram, N.; Movva, M.; Deep, A.; Corbett, K. D.; Debelouchina, G. T.\nScore: 5.4, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572911\nTau is a microtubule-associated protein often found in neurofibrillary tangles (NFTs) in the brains of patients with Alzheimers disease (AD). Beyond this context, mounting evidence suggests that tau localizes into the nucleus, where it may play a role in DNA protection and heterochromatin regulation. Models of tau depletion or pathology show loss of genetically silent heterochromatin, aberrant expression of heterochromatic genes, and transposable element activation. The molecular mechanisms behind these observations are currently unclear. Using in vitro biophysical experiments, here we demonstrate that tau can undergo liquid-liquid phase separation (LLPS) with DNA, mononucleosomes, and reconstituted nucleosome arrays under low salt conditions. Low concentrations of tau promote chromatin compaction and protect DNA from digestion. While the material state of samples at physiological salt is dominated by chromatin oligomerization, tau can still associate strongly and reversibly with nucleosome arrays. These properties are driven by taus strong interactions with linker and nucleosomal DNA, while magic angle spinning (MAS) solid-state NMR experiments show that tau binding does not drastically alter nucleosome structure and dynamics. In addition, tau co-localizes into droplets formed by nucleosome arrays and phosphorylated HP1, a key heterochromatin constituent thought to function through an LLPS mechanism. Importantly, LLPS and chromatin interactions are disrupted by aberrant tau hyperphosphorylation. These biophysical properties suggest that tau may directly impact DNA and chromatin accessibility and that loss of these interactions could contribute to the aberrant nuclear effects seen in tau pathology.\nDe novo design of drug-binding proteins with predictable binding energy and specificity\nAuthors: Lu, L.; Gou, X.; Tan, S.; Mann, S.; Yang, H.; Zhong, X.; Gazgalis, D.; Valdiviezo, J.; Jo, H.; Wu, Y.; Diolaiti, M.; Ashworth, A.; Polizzi, N.; DeGrado, W.\nScore: 17.4, Published: 2023-12-23 DOI: 10.1101/2023.12.23.573178\nThe de novo design of small-molecule-binding proteins has seen exciting recent progress; however, the ability to achieve exquisite affinity for binding small molecules while tuning specificity has not yet been demonstrated directly from computation. Here, we develop a computational procedure that results in the highest affinity binders to date with predetermined relative affinities, targeting a series of PARP1 inhibitors. Two of four designed proteins bound with affinities ranging from \u003c 5 nM to low M, in a predictable manner. X-ray crystal structures confirmed the accuracy of the designed protein-drug interactions. Molecular dynamics simulations informed the role of water in binding. Binding free-energy calculations performed directly on the designed models are in excellent agreement with the experimentally measured affinities, suggesting that the de novo design of small-molecule-binding proteins with tuned interaction energies is now feasible entirely from computation. We expect these methods to open many opportunities in biomedicine, including rapid sensor development, antidote design, and drug delivery vehicles. One Sentence SummaryWe use informatic sampling to design low nM drug-binding proteins, and physics-based calculations to accurately predict affinities.\nRole of diffusion and reaction of the constituents in spreading of histone modification marks\nAuthors: Manivannan, V.; Inamdar, M. M.; Padinhateeri, R.\nScore: 11.7, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571420\nCells switch genes ON or OFF by altering the state of chromatin via histone modifications at specific regulatory locations along the chromatin polymer. These gene regulation processes are carried out by a network of reactions in which the histone marks spread to neighboring regions with the help of enzymes. In the literature, this spreading has been studied as a purely kinetic, non-diffusive process considering the interactions between neighboring nucleosomes. In this work, we go beyond this framework and study the spreading of modifications using a reaction-diffusion (RD) model accounting for the diffusion of the constituents. We quantitatively segregate the modification profiles generated from kinetic and RD models. The diffusion and degradation of enzymes set a natural length scale for limiting the domain size of modification spreading, and the resulting enzyme limitation is inherent in our model. We also demonstrate the emergence of confined modification domains without the explicit requirement of a nucleation site. We explore polymer compaction effects on spreading and show that single-cell domains may differ from averaged profiles. We find that the modification profiles from our model are comparable with existing H3K9me3 data of S. pombe.\n",
  "wordCount" : "2154",
  "inLanguage": "en",
  "datePublished": "2024-01-03T10:39:43Z",
  "dateModified": "2024-01-03T10:39:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 3, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573500" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573500">
        <p class="paperTitle">Quantification of cell phenotype transition manifolds with information geometry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, M.; Ying, J.; Wang, Y.; Zhou, H.; Zhang, L.; Wang, W.</p>
        <p class="info">Score: 18.5, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573500' target='https://doi.org/10.1101/2023.12.28.573500'> 10.1101/2023.12.28.573500</a></p>
        <p class="abstract">Cell phenotype transition (CPT) is a common occurrence in many biological processes, such as development, and the precise control of CPT is critical in these processes. Recent advancements in single-cell sequencing techniques have revealed that cell transition dynamics in development are confined to low-dimensional manifolds. However, no available method currently exists for quantifying the manifolds from experimental data. Current methods, such as manifold learning, can only preserve topology information and can only be used for visualization.

In this work, we present a novel method to quantify the cell phenotype transition manifolds using information geometry. Specifically, we transform single cells high-dimensional expression vectors into probability distributions with Gaussian embedding. The Fisher-Rao metric is then naturally defined in this embedding space. With Gaussian embedding, we calculate the Ollivier-Ricci curvature of each single cell. Our analyses revealed that cells with low curvature are related to critical transitions.To further analyze the invariant characteristics of the manifold in CPT, we calculated the information velocity of each single cell based on RNA velocity. The high information velocity regions correspond to the low curvature regions, indicating that the geometry can guide the dynamics of single cells on the manifolds. This method focuses on analyzing the invariant characteristics of the manifold in CPT, and the results illustrate that it can be a general approach for quantifying the CPT manifold.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573596">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573596" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573596">
        <p class="paperTitle">Single-molecule detection and super-resolution imaging with a portable and adaptable 3D-printed microscopy platform (Brick-MIC)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573596" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573596" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moya Munoz, G. G.; Brix, O.; Klocke, P.; Wendler, N. D.; Lerner, E.; Zijlstra, N.; Cordes, T.</p>
        <p class="info">Score: 12.1, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573596' target='https://doi.org/10.1101/2023.12.29.573596'> 10.1101/2023.12.29.573596</a></p>
        <p class="abstract">Over the past decades, single-molecule spectroscopy and super-resolution microscopy have advanced significantly and by now represent important tools for life science research. Despite rapid progress and ongoing development, there is a growing gap between the state-of-the-art and what is accessible to non-optics specialists, e.g., biologists, biochemists, medical researchers, and labs with financial constraints. To bridge this gap, we introduce Brick-MIC, a versatile and affordable open-source 3D-printed micro-spectroscopy and imaging platform. Brick-MIC enables the integration of various fluorescence imaging techniques with single-molecule resolution within a single platform and enables exchange between different modalities within minutes. In this work, we present three variants of Brick-MIC that facilitate single-molecule fluorescence detection, fluorescence correlation spectroscopy and super-resolution imaging. With the three variants, we were able to observe conformational changes and absolute inter-dye distances in single macromolecules and perform single-molecule localization microscopy (STORM and PAINT) of DNA origami nanostructures. Detailed descriptions of the hardware and software components, as well as data analysis routines are provided, to allow non-optics specialist to operate their own Brick-MIC with minimal effort and investments. We foresee that our affordable, flexible, and open-source Brick-MIC platform will be a valuable tool for many laboratories worldwide.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571776">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571776" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571776">
        <p class="paperTitle">Smart Parallel Automated Cryo Electron tomography</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571776" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571776" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eisenstein, F.; Fukuda, Y.; Danev, R.</p>
        <p class="info">Score: 39.0, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571776' target='https://doi.org/10.1101/2023.12.14.571776'> 10.1101/2023.12.14.571776</a></p>
        <p class="abstract">In situ cryo electron tomography enables investigation of macromolecules in their native cellular environment. Data collection, however, requires an experienced operator and valuable microscope time to carefully select targets for tilt series acquisition. Here, we developed a workflow using machine learning approaches to fully automate the entire process, including lamella detection, biological feature segmentation, target selection, and tilt series acquisition, all without the need for human intervention.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572389" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572389">
        <p class="paperTitle">High-throughput computational discovery of inhibitory protein fragments with AlphaFold</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Savinov, A.; Swanson, S.; Keating, A. E.; Li, G.-W.</p>
        <p class="info">Score: 89.2, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572389' target='https://doi.org/10.1101/2023.12.19.572389'> 10.1101/2023.12.19.572389</a></p>
        <p class="abstract">Peptides can bind to specific sites on larger proteins and thereby function as inhibitors and regulatory elements. Peptide fragments of larger proteins are particularly attractive for achieving these functions due to their inherent potential to form native-like binding interactions. Recently developed experimental approaches allow for high-throughput measurement of protein fragment inhibitory activity in living cells. However, it has thus far not been possible to predict de novo which of the many possible protein fragments bind their protein targets, let alone act as inhibitors. We have developed a computational method, FragFold, that employs AlphaFold to predict protein fragment binding to full-length protein targets in a high-throughput manner. Applying FragFold to thousands of fragments tiling across diverse proteins revealed peaks of predicted binding along each protein sequence. These predictions were compared with experimentally measured peaks of inhibitory activity in E. coli. We establish that our approach is a sensitive predictor of protein fragment function: Evaluating inhibitory fragments derived from known protein-protein interaction interfaces, we found 87% were predicted by FragFold to bind in a native-like mode. Across full protein sequences, 68% of FragFold-predicted binding peaks match experimentally measured inhibitory peaks. This is true even when the underlying inhibitory mechanism is unclear from existing structural data, and we find FragFold is able to predict novel binding modes for inhibitory fragments of unknown structure, explaining previous genetic and biochemical data for these fragments. The success rate of FragFold demonstrates that this computational approach should be broadly applicable for discovering inhibitory protein fragments across proteomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573415">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573415" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573415">
        <p class="paperTitle">p23-FKBP51 chaperone complex regulates tau aggregation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573415" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573415" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chakraborty, P.; Zweckstetter, M.</p>
        <p class="info">Score: 5.9, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573415' target='https://doi.org/10.1101/2023.12.27.573415'> 10.1101/2023.12.27.573415</a></p>
        <p class="abstract">The pathological deposition of tau into insoluble amyloid fibrils is a pathological hallmark of tauopathies. Molecular chaperones are important cellular factors contributing to the regulation of tau misfolding and aggregation. Here we reveal an Hsp90-independent modulation of tau aggregation by a molecular complex of the two Hsp90 co-chaperones p23 and FKBP51. Integrating NMR spectroscopy, SAXS, molecular docking, and site-directed mutagenesis we reveal the structural basis of the p23-FKBP51 complex. We show that p23 specifically recognizes the TPR domain of FKBP51 and interacts with tau through its C-terminal disordered tail. We further show that the p23-FKBP51 complex binds tau to form a dynamic p23-FKBP51-tau trimeric complex that inhibits tau aggregation and thus may counteract Hsp90-FKBP51 mediated toxicity. Taken together, our findings reveal a co-chaperone mediated Hsp90-independent chaperoning of tau protein.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573034">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573034" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573034">
        <p class="paperTitle">The everting Drosophila wing disc is a shape-programmed material</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573034" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573034" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fuhrmann, J. F.; Krishna, A.; Paijmans, J.; Duclut, C. A.; Eaton, S.; Popovic, M.; Julicher, F.; Modes, C. D.; Dye, N. A.</p>
        <p class="info">Score: 7.7, Published: 2023-12-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573034' target='https://doi.org/10.1101/2023.12.23.573034'> 10.1101/2023.12.23.573034</a></p>
        <p class="abstract">How complex 3D tissue shape emerges during animal development remains an important open question in biology and biophysics. Although it is well-established that tissue-scale patterns of in-plane cell behaviors can reshape a tissue in 2D, whether and how such patterns could also deform a tissue in 3D remains unclear. Meanwhile, recent advances in materials science, physics, and engineering have enhanced our ability to craft and understand inanimate &#34;shape-programmable&#34; materials capable of undergoing blueprinted 3D shape transformations arising from in-plane gradients of spontaneous strains. Here, we use this conceptual framework to understand eversion of the Drosophila wing disc, a complex 3D tissue reshaping event in wing development. Using quantification of spatial patterns of cell dynamics on an evolving 3D tissue surface, combined with physical modeling and genetic perturbation, we demonstrate that the wing can deform itself in 3D through large-scale patterning of in-plane cellular behaviors, primarily rearrangements. This work shows that concepts from shape-programming can be employed to understand animal tissue morphogenesis and suggests that there exist intricate patterns in nature that could present novel designs for shape-programmable materials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573448">
        <p class="paperTitle">Structure-based discovery of positive allosteric modulators for the calcium sensing receptor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, F.; Wu, C.-G.; Tu, C.-L.; Glenn, I.; Meyerowitz, J.; Kaplan, A. L.; Lyu, J.; Cheng, Z.; Tarkhanova, O. O.; Moroz, Y. S.; Irwin, J. J.; Chang, W.; Shoichet, B. K.; Skiniotis, G.</p>
        <p class="info">Score: 5.5, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573448' target='https://doi.org/10.1101/2023.12.27.573448'> 10.1101/2023.12.27.573448</a></p>
        <p class="abstract">Drugs acting as positive allosteric modulators (PAMs) to enhance the activation of the calcium sensing receptor (CaSR) and to suppress parathyroid hormone (PTH) secretion can treat hyperparathyroidism but suffer from side effects including hypocalcemia and arrhythmias. Seeking new CaSR modulators, we docked libraries of 2.7 million and 1.2 billion molecules against transforming pockets in the active-state receptor dimer structure. Consistent with the idea that docking improves with library size, billion-molecule docking found new PAMs with a hit rate that was 2.7-fold higher than the million-molecule library and with hits up to 37-fold more potent. Structure-based optimization of ligands from both campaigns led to nanomolar leads, one of which was advanced to animal testing. This PAM displays 100-fold the potency of the standard of care, cinacalcet, in ex vivo organ assays, and reduces serum PTH levels in mice by up to 80% without the hypocalcemia typical of CaSR drugs. Cryo-EM structures with the new PAMs show that they promote CaSR dimer conformations that are closer to the G-protein coupled state compared to established drugs. These findings highlight the promise of large library docking for therapeutic leads, especially when combined with experimental structure determination and mechanism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572911">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572911" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572911">
        <p class="paperTitle">Phosphorylation regulates tau&#39;s phase separation behavior and interactions with chromatin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572911" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572911" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abasi, L. S.; Elathram, N.; Movva, M.; Deep, A.; Corbett, K. D.; Debelouchina, G. T.</p>
        <p class="info">Score: 5.4, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572911' target='https://doi.org/10.1101/2023.12.21.572911'> 10.1101/2023.12.21.572911</a></p>
        <p class="abstract">Tau is a microtubule-associated protein often found in neurofibrillary tangles (NFTs) in the brains of patients with Alzheimers disease (AD). Beyond this context, mounting evidence suggests that tau localizes into the nucleus, where it may play a role in DNA protection and heterochromatin regulation. Models of tau depletion or pathology show loss of genetically silent heterochromatin, aberrant expression of heterochromatic genes, and transposable element activation. The molecular mechanisms behind these observations are currently unclear. Using in vitro biophysical experiments, here we demonstrate that tau can undergo liquid-liquid phase separation (LLPS) with DNA, mononucleosomes, and reconstituted nucleosome arrays under low salt conditions. Low concentrations of tau promote chromatin compaction and protect DNA from digestion. While the material state of samples at physiological salt is dominated by chromatin oligomerization, tau can still associate strongly and reversibly with nucleosome arrays. These properties are driven by taus strong interactions with linker and nucleosomal DNA, while magic angle spinning (MAS) solid-state NMR experiments show that tau binding does not drastically alter nucleosome structure and dynamics. In addition, tau co-localizes into droplets formed by nucleosome arrays and phosphorylated HP1, a key heterochromatin constituent thought to function through an LLPS mechanism. Importantly, LLPS and chromatin interactions are disrupted by aberrant tau hyperphosphorylation. These biophysical properties suggest that tau may directly impact DNA and chromatin accessibility and that loss of these interactions could contribute to the aberrant nuclear effects seen in tau pathology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.23.573178">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.23.573178" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.23.573178">
        <p class="paperTitle">De novo design of drug-binding proteins with predictable binding energy and specificity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.23.573178" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.23.573178" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, L.; Gou, X.; Tan, S.; Mann, S.; Yang, H.; Zhong, X.; Gazgalis, D.; Valdiviezo, J.; Jo, H.; Wu, Y.; Diolaiti, M.; Ashworth, A.; Polizzi, N.; DeGrado, W.</p>
        <p class="info">Score: 17.4, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.23.573178' target='https://doi.org/10.1101/2023.12.23.573178'> 10.1101/2023.12.23.573178</a></p>
        <p class="abstract">The de novo design of small-molecule-binding proteins has seen exciting recent progress; however, the ability to achieve exquisite affinity for binding small molecules while tuning specificity has not yet been demonstrated directly from computation. Here, we develop a computational procedure that results in the highest affinity binders to date with predetermined relative affinities, targeting a series of PARP1 inhibitors. Two of four designed proteins bound with affinities ranging from &lt; 5 nM to low M, in a predictable manner. X-ray crystal structures confirmed the accuracy of the designed protein-drug interactions. Molecular dynamics simulations informed the role of water in binding. Binding free-energy calculations performed directly on the designed models are in excellent agreement with the experimentally measured affinities, suggesting that the de novo design of small-molecule-binding proteins with tuned interaction energies is now feasible entirely from computation. We expect these methods to open many opportunities in biomedicine, including rapid sensor development, antidote design, and drug delivery vehicles.

One Sentence SummaryWe use informatic sampling to design low nM drug-binding proteins, and physics-based calculations to accurately predict affinities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571420">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571420" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571420">
        <p class="paperTitle">Role of diffusion and reaction of the constituents in spreading of histone modification marks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571420" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571420" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manivannan, V.; Inamdar, M. M.; Padinhateeri, R.</p>
        <p class="info">Score: 11.7, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571420' target='https://doi.org/10.1101/2023.12.13.571420'> 10.1101/2023.12.13.571420</a></p>
        <p class="abstract">Cells switch genes ON or OFF by altering the state of chromatin via histone modifications at specific regulatory locations along the chromatin polymer. These gene regulation processes are carried out by a network of reactions in which the histone marks spread to neighboring regions with the help of enzymes. In the literature, this spreading has been studied as a purely kinetic, non-diffusive process considering the interactions between neighboring nucleosomes. In this work, we go beyond this framework and study the spreading of modifications using a reaction-diffusion (RD) model accounting for the diffusion of the constituents. We quantitatively segregate the modification profiles generated from kinetic and RD models. The diffusion and degradation of enzymes set a natural length scale for limiting the domain size of modification spreading, and the resulting enzyme limitation is inherent in our model. We also demonstrate the emergence of confined modification domains without the explicit requirement of a nucleation site. We explore polymer compaction effects on spreading and show that single-cell domains may differ from averaged profiles. We find that the modification profiles from our model are comparable with existing H3K9me3 data of S. pombe.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
