<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Localising elements in single-particle reconstructions by REEL-EM: Reconstructed Electron Energy-Loss - Elemental Mapping
Authors: Pfeil-Gardiner, O.; Rosa, H. V. D.; Riedel, D.; Chen, Y. S.; Lorks, D.; Kukelhan, P.; Linck, M.; Muller, H.; Van Petegem, F.; Murphy, B. J.
Score: 34.5, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575858
For structures determined by single particle cryo-EM, no technique currently exists for mapping elements to defined locations, leading to errors in the assignment of metals and other ions, cofactors, substrates, inhibitors, and lipids that play essential roles in activity and regulation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Localising elements in single-particle reconstructions by REEL-EM: Reconstructed Electron Energy-Loss - Elemental Mapping
Authors: Pfeil-Gardiner, O.; Rosa, H. V. D.; Riedel, D.; Chen, Y. S.; Lorks, D.; Kukelhan, P.; Linck, M.; Muller, H.; Van Petegem, F.; Murphy, B. J.
Score: 34.5, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575858
For structures determined by single particle cryo-EM, no technique currently exists for mapping elements to defined locations, leading to errors in the assignment of metals and other ions, cofactors, substrates, inhibitors, and lipids that play essential roles in activity and regulation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-24T10:40:04+00:00" />
<meta property="article:modified_time" content="2024-01-24T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Localising elements in single-particle reconstructions by REEL-EM: Reconstructed Electron Energy-Loss - Elemental Mapping
Authors: Pfeil-Gardiner, O.; Rosa, H. V. D.; Riedel, D.; Chen, Y. S.; Lorks, D.; Kukelhan, P.; Linck, M.; Muller, H.; Van Petegem, F.; Murphy, B. J.
Score: 34.5, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575858
For structures determined by single particle cryo-EM, no technique currently exists for mapping elements to defined locations, leading to errors in the assignment of metals and other ions, cofactors, substrates, inhibitors, and lipids that play essential roles in activity and regulation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Localising elements in single-particle reconstructions by REEL-EM: Reconstructed Electron Energy-Loss - Elemental Mapping\nAuthors: Pfeil-Gardiner, O.; Rosa, H. V. D.; Riedel, D.; Chen, Y. S.; Lorks, D.; Kukelhan, P.; Linck, M.; Muller, H.; Van Petegem, F.; Murphy, B. J.\nScore: 34.5, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575858\nFor structures determined by single particle cryo-EM, no technique currently exists for mapping elements to defined locations, leading to errors in the assignment of metals and other ions, cofactors, substrates, inhibitors, and lipids that play essential roles in activity and regulation.",
  "keywords": [
    
  ],
  "articleBody": " Localising elements in single-particle reconstructions by REEL-EM: Reconstructed Electron Energy-Loss - Elemental Mapping\nAuthors: Pfeil-Gardiner, O.; Rosa, H. V. D.; Riedel, D.; Chen, Y. S.; Lorks, D.; Kukelhan, P.; Linck, M.; Muller, H.; Van Petegem, F.; Murphy, B. J.\nScore: 34.5, Published: 2024-01-18 DOI: 10.1101/2024.01.18.575858\nFor structures determined by single particle cryo-EM, no technique currently exists for mapping elements to defined locations, leading to errors in the assignment of metals and other ions, cofactors, substrates, inhibitors, and lipids that play essential roles in activity and regulation. Elemental mapping in the electron microscope is well established for dose-tolerant samples but is challenging for biological samples, especially in a cryo-preserved state. Here, we combine electron energy-loss spectroscopy (EELS) with single-particle image processing to allow elemental mapping in cryo-preserved macromolecular complexes. Proof-of-principle data show that our method, REEL-EM, allows 3D reconstruction of EELS data, such that a high total electron dose is accumulated across many copies of a complex. Working with two test samples, we demonstrate that we can reliably localise abundant elements. We discuss the current limitations of the method and potential future developments.\nPatternJ: an ImageJ toolset for the automated and quantitative analysis of regular spatial patterns found in sarcomeres, axons, somites, and more\nAuthors: Baheux Blin, M.; Loreau, V.; Schnorrer, F.; Mangeol, P.\nScore: 20.6, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576053\nRegular spatial patterns are ubiquitous forms of organization in nature. In animals, regular patterns can be found from the cellular scale to the tissue scale, and from early stages of development to adulthood. To understand the formation of these patterns, how they form and mature, and how they are affected by perturbations, a precise quantitative description of the patterns is essential. However, accessible tools that offer in-depth analysis without the need for computational skills are lacking for biologists. Here we present PatternJ, a novel toolset to analyze regular pattern organizations precisely and automatically. This toolset, to be used with the popular imaging processing program ImageJ/Fiji, facilitates the extraction of key geometric features within and between pattern repeats. We validated PatternJ on simulated data and tested it on images of sarcomeres in insect muscles and cardiomyocytes, actin rings in neurons, and somites in zebrafish embryos obtained using confocal fluorescence microscopy, STORM, electron microscopy, and bright-field imaging. We show that the toolset delivers subpixel feature extraction reliably even with images of low signal-to-noise ratio. PatternJs straightforward use and functionalities make it valuable for various scientific fields requiring quantitative pattern analysis, including the sarcomere biology of muscles or the patterning of mammalian axons, speeding up discoveries with the bonus of high reproducibility.\nEngineering CRBN for rapid identification of next generation binders\nAuthors: Bailey, H. J.; Eisert, J.; Vollrath, J.; Leibrock, E.-M.; Kondratov, I.; Matviyuk, T.; Tolmachova, N.; Langer, J. D.; Wegener, A. A.; Sorrell, F. J.; Dikic, I.\nScore: 14.5, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576231\nThe majority of clinical degrader candidates utilize an immunomodulatory imide drug (IMiD)-based derivative, that directs their target substrate to the E3 ligase receptor Cereblon (CRBN), however, concomitant neo-substrate degradation by IMiDs often results in severe off-target effects. Further biophysical screening is needed to discover CRBN binders that might overcome these safety concerns, but the previously reported CRBN protein constructs suffer significant limitations that reduce their applicability as tools for biophysical assays making large-scale screening efforts a costly endeavor. This is due to the requirement of CRBN co-expression with DDB1 in insect cells to produce soluble protein that contains all the reported structural features necessary for proper compound interaction. Here, a near full-length human CRBN construct was designed that retains these required features, but for the first time allows the generation of highly homogenous and cost-efficient expression in E.coli, eliminating the need for DDB1 co-expression. We have extensively profiled the new construct against the existing CRBN constructs in binary and ternary degrader complexes, showing it to be the \"best of both worlds\" in terms of binding characteristics and ease of protein production. We then designed the Enamine CRBN focused IMiD library of 4480-compounds and demonstrated applicability of the new construct to high throughput screening by identifying novel CRBN binders with high potency and ligand efficiency. The new methods described here should serve as valuable tools for the development of the next generation IMiD-based degraders.\nStructure, motion, and multiscale search of traveling networks\nAuthors: Cira, N. J.; Paull, M. L.; Sinha, S.; Zanini, F.; Ma, E. Y.; Riedel-Kruse, I. H.\nScore: 10.3, Published: 2024-01-20 DOI: 10.1101/2024.01.16.575883\nNetwork models are widely applied to describe connectivity and flow in diverse systems. In contrast, the fact that many connected systems move through space as the result of dynamic restructuring has received little attention. Therefore, we introduce the concept of traveling networks, and we analyze a tree-based model where the leaves are stochastically manipulated to grow, branch, and retract. We derive how these restructuring rates determine key attributes of network structure and motion, enabling a compact understanding of higher-level network behaviors such as multiscale search. These networks self-organize to the critical point between exponential growth and decay, allowing them to detect and respond to environmental signals with high sensitivity. Finally, we demonstrate how the traveling network concept applies to real-world systems, such as slime molds, the actin cytoskeleton, and human organizations, exemplifying how restructuring rules and rates in general can select for versatile search strategies in real or abstract spaces.\nCryoRhodopsins: a comprehensive characterization of a new clade of microbial rhodopsins from cold environments\nAuthors: Lamm, G. H. U.; Marin, E.; Schellbach, A. V.; Stetsenko, A.; Alekseev, A.; Bourenkov, G.; Borshchevskiy, V.; Asido, M.; Agthe, M.; Engilberge, S.; Rose, S. L.; Caramello, N.; Royant, A.; Schneider, T. R.; Bateman, A. I.; Mager, T.; Moser, T.; Wachtveitl, J.; Guskov, A.; Kovalev, K.\nScore: 9.7, Published: 2024-01-17 DOI: 10.1101/2024.01.15.575777\nMicrobial rhodopsins are omnipresent on Earth, however the vast majority of them remain uncharacterized. Here we describe a new rhodopsin clade from cold-adapted organisms and cold environments, such as glaciers, denoted as CryoRhodopsins (CryoRs). Our data suggest that CryoRs have photosensory activity. A distinguishing feature of the clade is the presence of a buried arginine residue close to the cytoplasmic face of its members. Combining single-particle cryo-electron microscopy and X-ray crystallography with the rhodopsin activation by light, we demonstrate that the arginine stabilizes a strongly blue-shifted intermediate of an extremely slow CryoRhodopsin photocycle. Together with extensive spectroscopic characterization, our investigations on CryoR1 and CryoR2 proteins reveal mechanisms of photoswitching in the newly identified clade and demonstrate principles of the adaptation of these rhodopsins to low temperatures.\nCryo-electron tomography reveals how COPII assembles on cargo-containing membranes\nAuthors: Pyle, E.; Zanetti, G.\nScore: 9.1, Published: 2024-01-17 DOI: 10.1101/2024.01.17.576008\nProteins traverse the eukaryotic secretory pathway via membrane trafficking between organelles. The COPII coat mediates the anterograde transport of newly synthesised proteins from the endoplasmic reticulum, engaging cargoes with wide ranges of sizes and biophysical properties. The native architecture of the COPII coat and the cargo-dependent regulation of its assembly remain poorly understood. Here, we have reconstituted COPII-coated membrane carriers using purified S. cerevisiae proteins and cell-derived microsomes as a native membrane source. Using cryo-electron tomography with subtomogram averaging, we demonstrate that the COPII coat binds cargo and forms largely spherical vesicles from native membranes. We reveal the architecture of the inner and outer coat layers and shed light on how spherical carriers are formed. Our results provide novel insights into the architecture and regulation of the COPII coat and challenge our current understanding of how membrane curvature is generated.\nCorrelating fluorescence microscopy, optical and magnetic tweezers to study single chiral biopolymers, tested on DNA plectoneme formation dynamics\nAuthors: Shepherd, J.; Guilbaud, S.; Zhou, Z.; Howard, J.; Burman, M.; Schaefer, C.; Kerrigan, A.; Steele-King, C.; Noy, A.; Leake, M. C.\nScore: 8.9, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576226\nBiopolymer topology is critical for determining interactions inside cell environments, exemplified by DNA where its response to mechanical perturbation is as important as biochemical properties to its cellular roles. The dynamic structures of chiral biopolymers exhibit complex dependence with extension and torsion, however the physical mechanisms underpinning the emergence of structural motifs upon physiological twisting and stretching are poorly understood due to technological limitations in correlating force, torque and spatial localization information. We present COMBI-Tweez (Combined Optical and Magnetic BIomolecule TWEEZers), a transformative tool that overcomes these challenges by integrating optical trapping, time-resolved electromagnetic tweezers, and fluorescence microscopy, demonstrated on single DNA molecules, that can controllably form and visualise higher order structural motifs including plectonemes. This technology combined with cutting-edge MD simulations provides quantitative insight into complex dynamic structures relevant to DNA cellular processes and can be adapted to study a range of filamentous biopolymers.\nHigh-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states\nAuthors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.\nScore: 30.9, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575649\nCell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies. Here, we describe a cytometry-based method with single-cell resolution which fills this technological gap by combining biophysical profiling with conventional biomarker analysis. This platform allows to reveal notable cell type-dependent remodelling of membrane fluidity during immune stimulations and in diseases. Using immune cells exposed to tumour microenvironment as well as from long COVID and chronic lymphocyte leukaemia patients, we demonstrate that membrane fluidity is orthogonal to surface marker expression. Moreover, this biophysical parameter identifies new functional and pathological states of immune cells previously undetected via surface marker profiling alone. Our findings will contribute to a more precise definition of immune cell states based on their biophysical properties and will pave the way for a better understanding of the functional heterogeneity of immune cells.\nAdsorption-driven deformation and landing-footprints of the RBD proteins in SARS-CoV-2 variants onto biological and inanimate surfaces\nAuthors: Bosch, A.; V. Guzman, H.; Perez, R.\nScore: 6.7, Published: 2024-01-22 DOI: 10.1101/2024.01.15.575706\nRespiratory viruses, carried through airborne microdroplets, frequently adhere to surfaces, including plastics and metals. However, our understanding of the interactions between viruses and materials remains limited, particularly in scenarios involving polarizable surfaces. Here, we investigate the role of receptor-binding domain (RBD) mutations on the adsorption of SARS-CoV-2 to hydrophobic and hydrophilic surfaces employing molecular simulations. To contextualize our findings, we contrast the interactions on inanimate surfaces with those on native-biological interfaces, specifically the ACE2 receptor. Notably, we identify a twofold increase in structural deformations for the proteins receptor binding motif onto the inanimate surfaces, indicative of enhanced shock-absorbing mechanisms. Furthermore, the distribution of amino acids (landing-footprints) on the inanimate surface reveals a distinct regional asymmetry relative to the biological interface. In spite of the H-bonds formed at the hydrophilic substrate, the simulations consistently show a higher number of contacts and interfacial area with the hydrophobic surface, with the WT RBD adsorbed more strongly to than the delta or omicron RBDs. In contrast, the adsorption of delta and omicron to hydrophilic surfaces was characterized by a distinctive hopping-pattern. The novel shock-absorbing mechanisms identified in the virus adsorption on inanimate surfaces could lead current experimental efforts in the design of virucidal surfaces.\nStructural features of heteromeric channels composed of CALHM2 and CALHM4 paralogs\nAuthors: Drozdzyk, K.; Peter, M.; Dutzler, R.\nScore: 6.3, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576238\nThe CALHM proteins constitute a family of large pore channels that contains six closely related paralogs in human. Two family members, CALHM1 and 3, have been associated with the release of ATP during taste sensation. Both proteins form heteromeric channels that activate at positive potential and decreased extracellular Ca2+ concentration. Although the structures of several family members displayed large oligomeric organizations of different size, their function has in most cases remained elusive. Our previous study has identified the paralogs CALHM2, 4 and 6 to be highly expressed in the placenta and defined their structural properties as membrane proteins exhibiting features of large pore channels with unknown activation properties (Drozdzyk et al., 2020). Here we investigated whether these placental paralogs would form heteromers and characterized heteromeric complexes consisting of CALHM2 and CALHM4 subunits using specific binders as fiducial markers. Both proteins assemble with different stoichiometries with the largest population containing CALHM2 as predominant component. In these oligomers, the subunits segregate and reside in their preferred conformation found in homomeric channels. Our study has thus revealed the properties that govern the formation of CALHM heteromers in a process of potential relevance in a cellular context.\n",
  "wordCount" : "2087",
  "inLanguage": "en",
  "datePublished": "2024-01-24T10:40:04Z",
  "dateModified": "2024-01-24T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 24, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.575858">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.575858" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.575858">
        <p class="paperTitle">Localising elements in single-particle reconstructions by REEL-EM: Reconstructed Electron Energy-Loss - Elemental Mapping</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.575858" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.575858" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pfeil-Gardiner, O.; Rosa, H. V. D.; Riedel, D.; Chen, Y. S.; Lorks, D.; Kukelhan, P.; Linck, M.; Muller, H.; Van Petegem, F.; Murphy, B. J.</p>
        <p class="info">Score: 34.5, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.575858' target='https://doi.org/10.1101/2024.01.18.575858'> 10.1101/2024.01.18.575858</a></p>
        <p class="abstract">For structures determined by single particle cryo-EM, no technique currently exists for mapping elements to defined locations, leading to errors in the assignment of metals and other ions, cofactors, substrates, inhibitors, and lipids that play essential roles in activity and regulation. Elemental mapping in the electron microscope is well established for dose-tolerant samples but is challenging for biological samples, especially in a cryo-preserved state. Here, we combine electron energy-loss spectroscopy (EELS) with single-particle image processing to allow elemental mapping in cryo-preserved macromolecular complexes. Proof-of-principle data show that our method, REEL-EM, allows 3D reconstruction of EELS data, such that a high total electron dose is accumulated across many copies of a complex. Working with two test samples, we demonstrate that we can reliably localise abundant elements. We discuss the current limitations of the method and potential future developments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576053">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576053" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576053">
        <p class="paperTitle">PatternJ: an ImageJ toolset for the automated and quantitative analysis of regular spatial patterns found in sarcomeres, axons, somites, and more</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576053" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576053" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baheux Blin, M.; Loreau, V.; Schnorrer, F.; Mangeol, P.</p>
        <p class="info">Score: 20.6, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576053' target='https://doi.org/10.1101/2024.01.17.576053'> 10.1101/2024.01.17.576053</a></p>
        <p class="abstract">Regular spatial patterns are ubiquitous forms of organization in nature. In animals, regular patterns can be found from the cellular scale to the tissue scale, and from early stages of development to adulthood. To understand the formation of these patterns, how they form and mature, and how they are affected by perturbations, a precise quantitative description of the patterns is essential. However, accessible tools that offer in-depth analysis without the need for computational skills are lacking for biologists. Here we present PatternJ, a novel toolset to analyze regular pattern organizations precisely and automatically. This toolset, to be used with the popular imaging processing program ImageJ/Fiji, facilitates the extraction of key geometric features within and between pattern repeats. We validated PatternJ on simulated data and tested it on images of sarcomeres in insect muscles and cardiomyocytes, actin rings in neurons, and somites in zebrafish embryos obtained using confocal fluorescence microscopy, STORM, electron microscopy, and bright-field imaging. We show that the toolset delivers subpixel feature extraction reliably even with images of low signal-to-noise ratio. PatternJs straightforward use and functionalities make it valuable for various scientific fields requiring quantitative pattern analysis, including the sarcomere biology of muscles or the patterning of mammalian axons, speeding up discoveries with the bonus of high reproducibility.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576231">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576231" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576231">
        <p class="paperTitle">Engineering CRBN for rapid identification of next generation binders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576231" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576231" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bailey, H. J.; Eisert, J.; Vollrath, J.; Leibrock, E.-M.; Kondratov, I.; Matviyuk, T.; Tolmachova, N.; Langer, J. D.; Wegener, A. A.; Sorrell, F. J.; Dikic, I.</p>
        <p class="info">Score: 14.5, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576231' target='https://doi.org/10.1101/2024.01.18.576231'> 10.1101/2024.01.18.576231</a></p>
        <p class="abstract">The majority of clinical degrader candidates utilize an immunomodulatory imide drug (IMiD)-based derivative, that directs their target substrate to the E3 ligase receptor Cereblon (CRBN), however, concomitant neo-substrate degradation by IMiDs often results in severe off-target effects. Further biophysical screening is needed to discover CRBN binders that might overcome these safety concerns, but the previously reported CRBN protein constructs suffer significant limitations that reduce their applicability as tools for biophysical assays making large-scale screening efforts a costly endeavor. This is due to the requirement of CRBN co-expression with DDB1 in insect cells to produce soluble protein that contains all the reported structural features necessary for proper compound interaction. Here, a near full-length human CRBN construct was designed that retains these required features, but for the first time allows the generation of highly homogenous and cost-efficient expression in E.coli, eliminating the need for DDB1 co-expression. We have extensively profiled the new construct against the existing CRBN constructs in binary and ternary degrader complexes, showing it to be the &#34;best of both worlds&#34; in terms of binding characteristics and ease of protein production. We then designed the Enamine CRBN focused IMiD library of 4480-compounds and demonstrated applicability of the new construct to high throughput screening by identifying novel CRBN binders with high potency and ligand efficiency. The new methods described here should serve as valuable tools for the development of the next generation IMiD-based degraders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575883">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575883" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575883">
        <p class="paperTitle">Structure, motion, and multiscale search of traveling networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575883" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575883" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cira, N. J.; Paull, M. L.; Sinha, S.; Zanini, F.; Ma, E. Y.; Riedel-Kruse, I. H.</p>
        <p class="info">Score: 10.3, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575883' target='https://doi.org/10.1101/2024.01.16.575883'> 10.1101/2024.01.16.575883</a></p>
        <p class="abstract">Network models are widely applied to describe connectivity and flow in diverse systems. In contrast, the fact that many connected systems move through space as the result of dynamic restructuring has received little attention. Therefore, we introduce the concept of  traveling networks, and we analyze a tree-based model where the leaves are stochastically manipulated to grow, branch, and retract. We derive how these restructuring rates determine key attributes of network structure and motion, enabling a compact understanding of higher-level network behaviors such as multiscale search. These networks self-organize to the critical point between exponential growth and decay, allowing them to detect and respond to environmental signals with high sensitivity. Finally, we demonstrate how the traveling network concept applies to real-world systems, such as slime molds, the actin cytoskeleton, and human organizations, exemplifying how restructuring rules and rates in general can select for versatile search strategies in real or abstract spaces.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575777">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575777" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575777">
        <p class="paperTitle">CryoRhodopsins: a comprehensive characterization of a new clade of microbial rhodopsins from cold environments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575777" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575777" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lamm, G. H. U.; Marin, E.; Schellbach, A. V.; Stetsenko, A.; Alekseev, A.; Bourenkov, G.; Borshchevskiy, V.; Asido, M.; Agthe, M.; Engilberge, S.; Rose, S. L.; Caramello, N.; Royant, A.; Schneider, T. R.; Bateman, A. I.; Mager, T.; Moser, T.; Wachtveitl, J.; Guskov, A.; Kovalev, K.</p>
        <p class="info">Score: 9.7, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575777' target='https://doi.org/10.1101/2024.01.15.575777'> 10.1101/2024.01.15.575777</a></p>
        <p class="abstract">Microbial rhodopsins are omnipresent on Earth, however the vast majority of them remain uncharacterized. Here we describe a new rhodopsin clade from cold-adapted organisms and cold environments, such as glaciers, denoted as CryoRhodopsins (CryoRs). Our data suggest that CryoRs have photosensory activity. A distinguishing feature of the clade is the presence of a buried arginine residue close to the cytoplasmic face of its members. Combining single-particle cryo-electron microscopy and X-ray crystallography with the rhodopsin activation by light, we demonstrate that the arginine stabilizes a strongly blue-shifted intermediate of an extremely slow CryoRhodopsin photocycle. Together with extensive spectroscopic characterization, our investigations on CryoR1 and CryoR2 proteins reveal mechanisms of photoswitching in the newly identified clade and demonstrate principles of the adaptation of these rhodopsins to low temperatures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576008" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576008">
        <p class="paperTitle">Cryo-electron tomography reveals how COPII assembles on cargo-containing membranes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pyle, E.; Zanetti, G.</p>
        <p class="info">Score: 9.1, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576008' target='https://doi.org/10.1101/2024.01.17.576008'> 10.1101/2024.01.17.576008</a></p>
        <p class="abstract">Proteins traverse the eukaryotic secretory pathway via membrane trafficking between organelles. The COPII coat mediates the anterograde transport of newly synthesised proteins from the endoplasmic reticulum, engaging cargoes with wide ranges of sizes and biophysical properties. The native architecture of the COPII coat and the cargo-dependent regulation of its assembly remain poorly understood. Here, we have reconstituted COPII-coated membrane carriers using purified S. cerevisiae proteins and cell-derived microsomes as a native membrane source. Using cryo-electron tomography with subtomogram averaging, we demonstrate that the COPII coat binds cargo and forms largely spherical vesicles from native membranes. We reveal the architecture of the inner and outer coat layers and shed light on how spherical carriers are formed. Our results provide novel insights into the architecture and regulation of the COPII coat and challenge our current understanding of how membrane curvature is generated.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576226">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576226" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576226">
        <p class="paperTitle">Correlating fluorescence microscopy, optical and magnetic tweezers to study single chiral biopolymers, tested on DNA plectoneme formation dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576226" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576226" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shepherd, J.; Guilbaud, S.; Zhou, Z.; Howard, J.; Burman, M.; Schaefer, C.; Kerrigan, A.; Steele-King, C.; Noy, A.; Leake, M. C.</p>
        <p class="info">Score: 8.9, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576226' target='https://doi.org/10.1101/2024.01.18.576226'> 10.1101/2024.01.18.576226</a></p>
        <p class="abstract">Biopolymer topology is critical for determining interactions inside cell environments, exemplified by DNA where its response to mechanical perturbation is as important as biochemical properties to its cellular roles. The dynamic structures of chiral biopolymers exhibit complex dependence with extension and torsion, however the physical mechanisms underpinning the emergence of structural motifs upon physiological twisting and stretching are poorly understood due to technological limitations in correlating force, torque and spatial localization information. We present COMBI-Tweez (Combined Optical and Magnetic BIomolecule TWEEZers), a transformative tool that overcomes these challenges by integrating optical trapping, time-resolved electromagnetic tweezers, and fluorescence microscopy, demonstrated on single DNA molecules, that can controllably form and visualise higher order structural motifs including plectonemes. This technology combined with cutting-edge MD simulations provides quantitative insight into complex dynamic structures relevant to DNA cellular processes and can be adapted to study a range of filamentous biopolymers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575649">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575649" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575649">
        <p class="paperTitle">High-throughput analysis of membrane fluidity unveils a hidden dimension in immune cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575649" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575649" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Andronico, L. A.; Jiang, Y.; Carannante, V.; Iskrak, S.; Sandoz, P. A.; Mikes, J.; Klymchenko, A. S.; Buggert, M.; Osterborg, A.; Onfelt, B.; Brodin, P.; Sezgin, E.</p>
        <p class="info">Score: 30.9, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575649' target='https://doi.org/10.1101/2024.01.15.575649'> 10.1101/2024.01.15.575649</a></p>
        <p class="abstract">Cell membranes undergo biophysical remodelling as an adaptation to the surroundings and to perform specific biological functions. However, the extent and relevance of such changes in human immune systems remain unknown, largely due to the lack of high throughput and multidimensional methodologies. Here, we describe a cytometry-based method with single-cell resolution which fills this technological gap by combining biophysical profiling with conventional biomarker analysis. This platform allows to reveal notable cell type-dependent remodelling of membrane fluidity during immune stimulations and in diseases. Using immune cells exposed to tumour microenvironment as well as from long COVID and chronic lymphocyte leukaemia patients, we demonstrate that membrane fluidity is orthogonal to surface marker expression. Moreover, this biophysical parameter identifies new functional and pathological states of immune cells previously undetected via surface marker profiling alone. Our findings will contribute to a more precise definition of immune cell states based on their biophysical properties and will pave the way for a better understanding of the functional heterogeneity of immune cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575706">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575706" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575706">
        <p class="paperTitle">Adsorption-driven deformation and landing-footprints of the RBD proteins in SARS-CoV-2 variants onto biological and inanimate surfaces</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575706" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575706" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bosch, A.; V. Guzman, H.; Perez, R.</p>
        <p class="info">Score: 6.7, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575706' target='https://doi.org/10.1101/2024.01.15.575706'> 10.1101/2024.01.15.575706</a></p>
        <p class="abstract">Respiratory viruses, carried through airborne microdroplets, frequently adhere to surfaces, including plastics and metals. However, our understanding of the interactions between viruses and materials remains limited, particularly in scenarios involving polarizable surfaces. Here, we investigate the role of receptor-binding domain (RBD) mutations on the adsorption of SARS-CoV-2 to hydrophobic and hydrophilic surfaces employing molecular simulations. To contextualize our findings, we contrast the interactions on inanimate surfaces with those on native-biological interfaces, specifically the ACE2 receptor. Notably, we identify a twofold increase in structural deformations for the proteins receptor binding motif onto the inanimate surfaces, indicative of enhanced shock-absorbing mechanisms. Furthermore, the distribution of amino acids (landing-footprints) on the inanimate surface reveals a distinct regional asymmetry relative to the biological interface. In spite of the H-bonds formed at the hydrophilic substrate, the simulations consistently show a higher number of contacts and interfacial area with the hydrophobic surface, with the WT RBD adsorbed more strongly to than the delta or omicron RBDs. In contrast, the adsorption of delta and omicron to hydrophilic surfaces was characterized by a distinctive hopping-pattern. The novel shock-absorbing mechanisms identified in the virus adsorption on inanimate surfaces could lead current experimental efforts in the design of virucidal surfaces.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576238">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576238" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576238">
        <p class="paperTitle">Structural features of heteromeric channels composed of CALHM2 and CALHM4 paralogs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576238" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576238" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Drozdzyk, K.; Peter, M.; Dutzler, R.</p>
        <p class="info">Score: 6.3, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576238' target='https://doi.org/10.1101/2024.01.18.576238'> 10.1101/2024.01.18.576238</a></p>
        <p class="abstract">The CALHM proteins constitute a family of large pore channels that contains six closely related paralogs in human. Two family members, CALHM1 and 3, have been associated with the release of ATP during taste sensation. Both proteins form heteromeric channels that activate at positive potential and decreased extracellular Ca2&#43; concentration. Although the structures of several family members displayed large oligomeric organizations of different size, their function has in most cases remained elusive. Our previous study has identified the paralogs CALHM2, 4 and 6 to be highly expressed in the placenta and defined their structural properties as membrane proteins exhibiting features of large pore channels with unknown activation properties (Drozdzyk et al., 2020). Here we investigated whether these placental paralogs would form heteromers and characterized heteromeric complexes consisting of CALHM2 and CALHM4 subunits using specific binders as fiducial markers. Both proteins assemble with different stoichiometries with the largest population containing CALHM2 as predominant component. In these oligomers, the subunits segregate and reside in their preferred conformation found in homomeric channels. Our study has thus revealed the properties that govern the formation of CALHM heteromers in a process of potential relevance in a cellular context.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
