<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Time-Resolved Cryo-EM Specimen Preparation with Single Millisecond Precision
Authors: Montano Romero, A.; Bonin, C.; Twomey, E. C.
Score: 47.1, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554704
Molecular structures can be determined in vitro and in situ with cryo-electron microscopy (cryo-EM). Specimen preparation is a major obstacle in cryo-EM. Typical sample preparation is orders of magnitude slower than biological processes. Time-resolved cryo-EM (TR-cryo-EM) can capture short-lived states. Here, we present Cryo-EM Sample Preparation with light-Activated Molecules (C-SPAM), an open-source, photochemistry-coupled device for TR-cryo-EM with single millisecond resolution, tunable timescales, and broad biological applications.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Time-Resolved Cryo-EM Specimen Preparation with Single Millisecond Precision
Authors: Montano Romero, A.; Bonin, C.; Twomey, E. C.
Score: 47.1, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554704
Molecular structures can be determined in vitro and in situ with cryo-electron microscopy (cryo-EM). Specimen preparation is a major obstacle in cryo-EM. Typical sample preparation is orders of magnitude slower than biological processes. Time-resolved cryo-EM (TR-cryo-EM) can capture short-lived states. Here, we present Cryo-EM Sample Preparation with light-Activated Molecules (C-SPAM), an open-source, photochemistry-coupled device for TR-cryo-EM with single millisecond resolution, tunable timescales, and broad biological applications." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-30T10:38:23+00:00" />
<meta property="article:modified_time" content="2023-08-30T10:38:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Time-Resolved Cryo-EM Specimen Preparation with Single Millisecond Precision
Authors: Montano Romero, A.; Bonin, C.; Twomey, E. C.
Score: 47.1, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554704
Molecular structures can be determined in vitro and in situ with cryo-electron microscopy (cryo-EM). Specimen preparation is a major obstacle in cryo-EM. Typical sample preparation is orders of magnitude slower than biological processes. Time-resolved cryo-EM (TR-cryo-EM) can capture short-lived states. Here, we present Cryo-EM Sample Preparation with light-Activated Molecules (C-SPAM), an open-source, photochemistry-coupled device for TR-cryo-EM with single millisecond resolution, tunable timescales, and broad biological applications."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Time-Resolved Cryo-EM Specimen Preparation with Single Millisecond Precision\nAuthors: Montano Romero, A.; Bonin, C.; Twomey, E. C.\nScore: 47.1, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554704\nMolecular structures can be determined in vitro and in situ with cryo-electron microscopy (cryo-EM). Specimen preparation is a major obstacle in cryo-EM. Typical sample preparation is orders of magnitude slower than biological processes. Time-resolved cryo-EM (TR-cryo-EM) can capture short-lived states. Here, we present Cryo-EM Sample Preparation with light-Activated Molecules (C-SPAM), an open-source, photochemistry-coupled device for TR-cryo-EM with single millisecond resolution, tunable timescales, and broad biological applications.",
  "keywords": [
    
  ],
  "articleBody": " Time-Resolved Cryo-EM Specimen Preparation with Single Millisecond Precision\nAuthors: Montano Romero, A.; Bonin, C.; Twomey, E. C.\nScore: 47.1, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554704\nMolecular structures can be determined in vitro and in situ with cryo-electron microscopy (cryo-EM). Specimen preparation is a major obstacle in cryo-EM. Typical sample preparation is orders of magnitude slower than biological processes. Time-resolved cryo-EM (TR-cryo-EM) can capture short-lived states. Here, we present Cryo-EM Sample Preparation with light-Activated Molecules (C-SPAM), an open-source, photochemistry-coupled device for TR-cryo-EM with single millisecond resolution, tunable timescales, and broad biological applications.\nSub-membrane actin rings compartmentalize the plasma membrane\nAuthors: Rentsch, J.; Bandstra, S.; Sezen, B.; Sigrist, P. S.; Bottanelli, F.; Schmerl, B.; Shoichet, S. A.; Noe, F.; Sadeghi, M.; Ewers, H.\nScore: 28.7, Published: 2023-08-23 DOI: 10.1101/2023.08.22.554239\nThe compartmentalization of the plasma membrane is a fundamental feature of cells. The diffusivity of membrane proteins in the plane of the membrane is significantly lower in cells than observed in artificial membranes. This seems due to the sub-membranous actin cortex, but tractable assays to prove a direct dependence of membrane protein motion on actin filaments have been elusive. We recently showed that a periodic array of sub-membrane actin filaments that surround neuronal axons in ring-like structures in the axonal initial segment (AIS) confines membrane protein motion between them, but the local enrichment of ion channels may offer an alternative explanation for compartmentalization. Here we show using computational modeling that in contrast to actin rings, the dense array of ion channels in the AIS cannot mediate membrane protein confinement. We go on to show that indeed the actin rings are closely apposed to the plasma membrane, and that they confine membrane protein motion between them in a [~] 200 nm periodic pattern in progenitor-derived neuronal cells. We find that this compartmentalization is also detectable for inner-leaflet membrane proteins and multi-spanning receptors. Strikingly, several cell types with actin rings in their protrusions, like progenitor-derived astrocytes and oligodendrocytes, also exhibit lateral confinement of membrane proteins in a periodic pattern consistent with actin ring spacing. Actin ring-mediated membrane compartmentalization is thus not unique to neurons. Finally, we show that acute actin disruption in live progenitor-derived neuronal cells leads to a loss of membrane compartmentalization in areas that were compartmentalized before treatment. Taken together, we here develop a much-needed tractable experimental system for the investigation of membrane compartmentalization and show that actin rings compartmentalize the plasma membrane of cells.\nCryoEM architecture of a native stretch-sensitive membrane microdomain\nAuthors: Kefauver, J. M.; Hakala, M.; Zou, L.; Alba, J.; Espadas Moreno, J.; Tettamanti, M. G.; Estrozi, L. F.; Vanni, S.; Roux, A.; Desfosses, A.; Loewith, R.\nScore: 19.9, Published: 2023-08-26 DOI: 10.1101/2023.08.25.554800\nBiological membranes are partitioned into functional zones containing specific lipids and proteins, termed membrane microdomains. Their composition and organization remain controversial owing to a paucity of techniques that can visualize lipids in situ without disrupting their native behavior. The yeast eisosome, a membrane compartment scaffolded by the BAR-domain proteins Pil1 and Lsp1, senses and responds to mechanical stress by flattening and releasing sequestered factors. Here, we isolated native eisosomes as helical filaments of Pil1/Lsp1 lattice bound to plasma membrane lipids and solved their structures by helical reconstruction. We observe remarkable organization within the lipid bilayer density from which we could assign headgroups of PI(4,5)P2 and phosphatidylserine bound to Pil1/Lsp1 and a pattern of membrane voids, signatures of sterols, beneath an amphipathic helix. We verified these assignments using in vitro reconstitutions and molecular dynamics simulations. 3D variability analysis of the native eisosomes revealed a dynamic stretching of the Pil1/Lsp1 lattice that affects functionally important lipid sequestration, supporting a mechanism in which membrane stretching liberates lipids otherwise anchored by the Pil1/Lsp1 coat. Our results provide mechanistic insight into how eisosome BAR-domain proteins create a mechanosensitive membrane microdomain and, more globally, resolve long-standing controversies about the architecture and nature of lipid microdomains.\nRole of protonation states in stability of molecular dynamics simulations of high-resolution membrane protein structures\nAuthors: Lasham, J.; Djurabekova, A.; Zickermann, V.; Vonck, J.; Sharma, V.\nScore: 13.5, Published: 2023-08-25 DOI: 10.1101/2023.08.24.554589\nClassical molecular dynamics (MD) simulations provide unmatched spatial and time resolution of protein structure and function. However, accuracy of MD simulations often depends on the quality of force field parameters and the time scale of sampling. Another limitation of conventional MD simulations is that the protonation states of titratable amino acid residues remain fixed during simulations, even though protonation state changes coupled to conformational dynamics are central to protein function. Due to the uncertainty in selecting protonation states, classical MD simulations are sometimes performed with all amino acids modeled in their standard charged states at pH 7. Here we performed and analyzed classical MD simulations on high-resolution cryo-EM structures of two membrane proteins that transfer protons by catalyzing protonation/deprotonation reactions. In simulations performed with amino acids modeled in their standard protonation state the structure diverges far from its starting conformation. In comparison, MD simulations performed with pre-determined protonation states of amino acid residues reproduce the structural conformation, protein hydration, and protein-water and protein-protein interactions of the structure much better. The results suggest it is crucial to perform basic protonation state calculations, especially on structures where protonation changes play an important functional role, prior to launching any MD simulations. Furthermore, the combined approach of protonation state prediction and MD simulations can provide valuable information on the charge states of amino acids in the cryo-EM sample. Even though accurate prediction of protonation states currently remains a challenge, we introduce an approach of combining pKa prediction with cryo-EM density map analysis that helps in improving not only the protonation state predictions, but also the atomic modeling of density data.\nDeep reconstructing generative networks for visualizing dynamic biomolecules inside cells\nAuthors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.\nScore: 109.7, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553799\nAdvances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms. This method, cryoDRGN-ET, learns a deep generative model of 3D density maps directly from subtomogram tilt series images and can capture states diverse in both composition and conformation. We use this approach to reconstruct the in situ translation dynamics of prokaryotic ribosomes, and we reveal the distribution of functional states during translation elongation populated by S. cerevisiae ribosomes inside cells.\nNMR measurements of transient low-populated tautomeric and anionic Watson-Crick-like G·T/U in RNA:DNA hybrids: Implications for the fidelity of transcription and CRISPR/Cas9 gene editing\nAuthors: Szekely, O.; Rangadurai, A. K.; Gu, S.; Manghrani, A.; Guseva, S.; Al-Hashimi, H.\nScore: 12.2, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554670\nMany biochemical processes use the Watson-Crick geometry to distinguish correct from incorrect base pairing. However, on rare occasions, mismatches such as G*T/U can transiently adopt Watson-Crick-like conformations through tautomerization or ionization of the bases, giving rise to replicative and translational errors. The propensities to form Watson-Crick-like mismatches in RNA:DNA hybrids remain unknown, making it unclear whether they can also contribute to errors during processes such as transcription and CRISPR/Cas editing. Here, using NMR R1{rho} experiments, we show that dG*rU and dT*rG mismatches in two RNA:DNA hybrids transiently form tautomeric (Genol*T/U {rightleftarrows}G*Tenol/Uenol) and anionic (G*T-/U-) Watson-Crick-like conformations. The tautomerization dynamics were like those measured in A-RNA and B-DNA duplexes. However, anionic dG*rU- formed with a ten-fold higher propensity relative to dT-*rG and dG*dT- and this could be attributed to the lower pKa ({Delta}pKa [~]0.4-0.9) of U versus T. Our findings suggest plausible roles for Watson-Crick-like G*T/U mismatches in transcriptional errors and CRISPR/Cas9 off-target gene editing, uncover a crucial difference between the chemical dynamics of G*U versus G*T, and indicate that anionic Watson-Crick-like G*U- could play a significant role evading Watson-Crick fidelity checkpoints in RNA:DNA hybrids and RNA duplexes.\nCryo-EM structures of human magnesium channel MRS2 reveal gating and regulatory mechanisms\nAuthors: Lai, L. T. F.; Balaraman, J.; Zhou, F.; Matthies, D.\nScore: 11.9, Published: 2023-08-23 DOI: 10.1101/2023.08.22.553867\nMagnesium ions (Mg2+) play an essential role in cellular physiology. In mitochondria, protein and ATP synthesis and various metabolic pathways are directly regulated by Mg2+. MRS2, a magnesium channel located in the inner mitochondrial membrane, mediates the influx of Mg2+ into the mitochondrial matrix and regulates Mg2+ homeostasis. Knockdown of MRS2 in human cells leads to reduced uptake of Mg2+ into mitochondria and disruption of the mitochondrial metabolism. Despite the importance of MRS2, the Mg2+ translocation and regulation mechanisms of MRS2 are still unclear. Here, using cryo-EM we determined the structure of human MRS2 in the presence and absence of Mg2+ at 2.8 [A] and 3.3 [A], respectively. From the homo-pentameric structures, we identified R332 and M336 as major gating residues, which were then tested using mutagenesis and two cellular divalent ion uptake assays. A network of hydrogen bonds was found connecting the gating residue R332 to the soluble domain, potentially regulating the gate. Two Mg2+-binding sites were identified in the MRS2 soluble domain, distinct from the two sites previously reported in CorA, a homolog of MRS2 in prokaryotes. Altogether, this study provides the molecular basis for understanding the Mg2+ translocation and regulatory mechanisms of MRS2.\nSingle-molecule visualization of twin-supercoiled domains generated during transcription\nAuthors: Janissen, R.; Barth, R.; Polinder, M.; van der Torre, J.; Dekker, C.\nScore: 11.5, Published: 2023-08-26 DOI: 10.1101/2023.08.25.554779\nTranscription-coupled supercoiling of DNA is a key factor in chromosome compaction and the regulation of genetic processes in all domains of life. It has become common knowledge that, during transcription, the DNA-dependent RNA polymerase (RNAP) induces positive supercoiling ahead of it (downstream) and negative supercoils in its wake (upstream), as rotation of RNAP around the DNA axis upon tracking its helical groove gets constrained due to drag on its RNA transcript. Here, we experimentally validate this so-called twin-supercoiled-domain model with in vitro real-time visualization at the single-molecule scale. Upon binding to the promoter site on a supercoiled DNA molecule, RNAP merges all DNA supercoils into one large pinned plectoneme with RNAP residing at its apex. Transcription by RNAP in real time demonstrates that up- and downstream supercoils are generated simultaneously and in equal portions, in agreement with the twin-supercoiled-domain model. Experiments carried out in the presence of RNases A and H, revealed that an additional viscous drag of the RNA transcript is not necessary for the RNAP to induce supercoils. The latter results contrast the current consensus and simulations on the origin of the twin-supercoiled domains, pointing at an additional mechanistic cause underlying supercoil generation by RNAP in transcription.\nModeling membrane geometries implicitly in Rosetta\nAuthors: Woods, H.; Koehler Leman, J.; Meiler, J.\nScore: 9.3, Published: 2023-08-23 DOI: 10.1101/2023.08.23.554394\nInteractions between membrane proteins (MPs) and lipid bilayers are critical for many cellular functions. In the Rosetta molecular modeling suite, the implicit membrane energy function is based on a \"slab\" model, which represent the membrane as a flat bilayer. However, in nature membranes often have a curvature that is important for function and/or stability. Even more prevalent, in structural biology research MPs are reconstituted in model membrane systems such as micelles, bicelles, nanodiscs, or liposomes. Thus, we have modified the existing membrane energy potentials within the RosettaMP framework to allow users to model MPs in different membrane geometries. We show that these modifications can be utilized in core applications within Rosetta such as structure refinement, protein-protein docking, and protein design. For MPs structures found in curved membranes, refining these structures in curved, implicit membranes produces higher quality models with structures closer to experimentally determined structures. For MP systems embedded in multiple membranes, representing both membranes results in more favorable scores compared to only representing one of the membranes. Modeling MPs in geometries mimicking the membrane model system used in structure determination can improve model quality and model discrimination.\nStructural elucidation of recombinant Trichomonas vaginalis 20S proteasome bound to covalent inhibitors\nAuthors: Silhan, J.; Fajtova, P.; Bartosova, J.; Hurysz, B.; Almaliti, J.; Miyamoto, Y.; Eckmann, L.; Gerwick, W.; O Donoghue, A.; Boura, E.\nScore: 8.9, Published: 2023-08-17 DOI: 10.1101/2023.08.17.553660\nProteasomes are essential for protein homeostasis in mammalian cells1-4 and in protozoan parasites such as Trichomonas vaginalis (Tv).5 Tv and other protozoan 20S proteasomes have been validated as druggable targets.6-8 However, in the case of Tv 20S proteasome (Tv20S), biochemical and structural studies were impeded by low yields and purity of the native proteasome. We successfully made recombinant Tv20S by expressing all seven and seven {beta} subunits together with the Ump-1 chaperone in insect cells. We isolated recombinant proteasome and showed that it was biochemically indistinguishable from the native enzyme. We confirmed that the recombinant Tv20S is inhibited by the natural product marizomib (MZB)9 and the recently developed peptide inhibitor carmaphycin-17 (CP-17)8,10. Specifically, MZB binds to the {beta}1, {beta}2 and {beta}5 subunits, while CP-17 binds the {beta}2 and {beta}5 subunits. Next, we obtained cryo-EM structures of Tv20S in complex with these covalent inhibitors at 2.8[A] resolution. The structures revealed the overall fold of the Tv20S and the binding mode of MZB and CP-17. Our work explains the low specificity of MZB and higher specificity of CP-17 towards Tv20S as compared to human proteasome and provides the platform for the development of Tv20S inhibitors for treatment of trichomoniasis.\n",
  "wordCount" : "2201",
  "inLanguage": "en",
  "datePublished": "2023-08-30T10:38:23Z",
  "dateModified": "2023-08-30T10:38:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on August 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554704">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554704" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554704">
        <p class="paperTitle">Time-Resolved Cryo-EM Specimen Preparation with Single Millisecond Precision</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554704" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554704" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Montano Romero, A.; Bonin, C.; Twomey, E. C.</p>
        <p class="info">Score: 47.1, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554704' target='https://doi.org/10.1101/2023.08.24.554704'> 10.1101/2023.08.24.554704</a></p>
        <p class="abstract">Molecular structures can be determined in vitro and in situ with cryo-electron microscopy (cryo-EM). Specimen preparation is a major obstacle in cryo-EM. Typical sample preparation is orders of magnitude slower than biological processes. Time-resolved cryo-EM (TR-cryo-EM) can capture short-lived states. Here, we present Cryo-EM Sample Preparation with light-Activated Molecules (C-SPAM), an open-source, photochemistry-coupled device for TR-cryo-EM with single millisecond resolution, tunable timescales, and broad biological applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.554239">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.554239" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.554239">
        <p class="paperTitle">Sub-membrane actin rings compartmentalize the plasma membrane</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.554239" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.554239" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rentsch, J.; Bandstra, S.; Sezen, B.; Sigrist, P. S.; Bottanelli, F.; Schmerl, B.; Shoichet, S. A.; Noe, F.; Sadeghi, M.; Ewers, H.</p>
        <p class="info">Score: 28.7, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.554239' target='https://doi.org/10.1101/2023.08.22.554239'> 10.1101/2023.08.22.554239</a></p>
        <p class="abstract">The compartmentalization of the plasma membrane is a fundamental feature of cells. The diffusivity of membrane proteins in the plane of the membrane is significantly lower in cells than observed in artificial membranes. This seems due to the sub-membranous actin cortex, but tractable assays to prove a direct dependence of membrane protein motion on actin filaments have been elusive. We recently showed that a periodic array of sub-membrane actin filaments that surround neuronal axons in ring-like structures in the axonal initial segment (AIS) confines membrane protein motion between them, but the local enrichment of ion channels may offer an alternative explanation for compartmentalization. Here we show using computational modeling that in contrast to actin rings, the dense array of ion channels in the AIS cannot mediate membrane protein confinement. We go on to show that indeed the actin rings are closely apposed to the plasma membrane, and that they confine membrane protein motion between them in a [~] 200 nm periodic pattern in progenitor-derived neuronal cells. We find that this compartmentalization is also detectable for inner-leaflet membrane proteins and multi-spanning receptors. Strikingly, several cell types with actin rings in their protrusions, like progenitor-derived astrocytes and oligodendrocytes, also exhibit lateral confinement of membrane proteins in a periodic pattern consistent with actin ring spacing. Actin ring-mediated membrane compartmentalization is thus not unique to neurons. Finally, we show that acute actin disruption in live progenitor-derived neuronal cells leads to a loss of membrane compartmentalization in areas that were compartmentalized before treatment. Taken together, we here develop a much-needed tractable experimental system for the investigation of membrane compartmentalization and show that actin rings compartmentalize the plasma membrane of cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554800">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554800" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554800">
        <p class="paperTitle">CryoEM architecture of a native stretch-sensitive membrane microdomain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554800" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554800" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kefauver, J. M.; Hakala, M.; Zou, L.; Alba, J.; Espadas Moreno, J.; Tettamanti, M. G.; Estrozi, L. F.; Vanni, S.; Roux, A.; Desfosses, A.; Loewith, R.</p>
        <p class="info">Score: 19.9, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554800' target='https://doi.org/10.1101/2023.08.25.554800'> 10.1101/2023.08.25.554800</a></p>
        <p class="abstract">Biological membranes are partitioned into functional zones containing specific lipids and proteins, termed membrane microdomains. Their composition and organization remain controversial owing to a paucity of techniques that can visualize lipids in situ without disrupting their native behavior. The yeast eisosome, a membrane compartment scaffolded by the BAR-domain proteins Pil1 and Lsp1, senses and responds to mechanical stress by flattening and releasing sequestered factors. Here, we isolated native eisosomes as helical filaments of Pil1/Lsp1 lattice bound to plasma membrane lipids and solved their structures by helical reconstruction. We observe remarkable organization within the lipid bilayer density from which we could assign headgroups of PI(4,5)P2 and phosphatidylserine bound to Pil1/Lsp1 and a pattern of membrane voids, signatures of sterols, beneath an amphipathic helix. We verified these assignments using in vitro reconstitutions and molecular dynamics simulations. 3D variability analysis of the native eisosomes revealed a dynamic stretching of the Pil1/Lsp1 lattice that affects functionally important lipid sequestration, supporting a mechanism in which membrane stretching liberates lipids otherwise anchored by the Pil1/Lsp1 coat. Our results provide mechanistic insight into how eisosome BAR-domain proteins create a mechanosensitive membrane microdomain and, more globally, resolve long-standing controversies about the architecture and nature of lipid microdomains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554589">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554589" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554589">
        <p class="paperTitle">Role of protonation states in stability of molecular dynamics simulations of high-resolution membrane protein structures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554589" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554589" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lasham, J.; Djurabekova, A.; Zickermann, V.; Vonck, J.; Sharma, V.</p>
        <p class="info">Score: 13.5, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554589' target='https://doi.org/10.1101/2023.08.24.554589'> 10.1101/2023.08.24.554589</a></p>
        <p class="abstract">Classical molecular dynamics (MD) simulations provide unmatched spatial and time resolution of protein structure and function. However, accuracy of MD simulations often depends on the quality of force field parameters and the time scale of sampling. Another limitation of conventional MD simulations is that the protonation states of titratable amino acid residues remain fixed during simulations, even though protonation state changes coupled to conformational dynamics are central to protein function. Due to the uncertainty in selecting protonation states, classical MD simulations are sometimes performed with all amino acids modeled in their standard charged states at pH 7. Here we performed and analyzed classical MD simulations on high-resolution cryo-EM structures of two membrane proteins that transfer protons by catalyzing protonation/deprotonation reactions. In simulations performed with amino acids modeled in their standard protonation state the structure diverges far from its starting conformation. In comparison, MD simulations performed with pre-determined protonation states of amino acid residues reproduce the structural conformation, protein hydration, and protein-water and protein-protein interactions of the structure much better. The results suggest it is crucial to perform basic protonation state calculations, especially on structures where protonation changes play an important functional role, prior to launching any MD simulations. Furthermore, the combined approach of protonation state prediction and MD simulations can provide valuable information on the charge states of amino acids in the cryo-EM sample. Even though accurate prediction of protonation states currently remains a challenge, we introduce an approach of combining pKa prediction with cryo-EM density map analysis that helps in improving not only the protonation state predictions, but also the atomic modeling of density data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553799">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553799" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553799">
        <p class="paperTitle">Deep reconstructing generative networks for visualizing dynamic biomolecules inside cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553799" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553799" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rangan, R.; Khavnekar, S.; Lerer, A.; Johnston, J.; Kelley, R.; Obr, M.; Kotecha, A.; Zhong, E. D.</p>
        <p class="info">Score: 109.7, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553799' target='https://doi.org/10.1101/2023.08.18.553799'> 10.1101/2023.08.18.553799</a></p>
        <p class="abstract">Advances in cryo-electron tomography (cryo-ET) have produced new opportunities to visualize the structures of dynamic macromolecular machinery in native cellular environments. Here, we describe a machine learning approach that can reconstruct the structural landscape and dynamics of biomolecular complexes present in cryo-ET subtomograms. This method, cryoDRGN-ET, learns a deep generative model of 3D density maps directly from subtomogram tilt series images and can capture states diverse in both composition and conformation. We use this approach to reconstruct the in situ translation dynamics of prokaryotic ribosomes, and we reveal the distribution of functional states during translation elongation populated by S. cerevisiae ribosomes inside cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554670">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554670" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554670">
        <p class="paperTitle">NMR measurements of transient low-populated tautomeric and anionic Watson-Crick-like G·T/U in RNA:DNA hybrids: Implications for the fidelity of transcription and CRISPR/Cas9 gene editing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554670" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554670" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Szekely, O.; Rangadurai, A. K.; Gu, S.; Manghrani, A.; Guseva, S.; Al-Hashimi, H.</p>
        <p class="info">Score: 12.2, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554670' target='https://doi.org/10.1101/2023.08.24.554670'> 10.1101/2023.08.24.554670</a></p>
        <p class="abstract">Many biochemical processes use the Watson-Crick geometry to distinguish correct from incorrect base pairing. However, on rare occasions, mismatches such as G*T/U can transiently adopt Watson-Crick-like conformations through tautomerization or ionization of the bases, giving rise to replicative and translational errors. The propensities to form Watson-Crick-like mismatches in RNA:DNA hybrids remain unknown, making it unclear whether they can also contribute to errors during processes such as transcription and CRISPR/Cas editing. Here, using NMR R1{rho} experiments, we show that dG*rU and dT*rG mismatches in two RNA:DNA hybrids transiently form tautomeric (Genol*T/U {rightleftarrows}G*Tenol/Uenol) and anionic (G*T-/U-) Watson-Crick-like conformations. The tautomerization dynamics were like those measured in A-RNA and B-DNA duplexes. However, anionic dG*rU- formed with a ten-fold higher propensity relative to dT-*rG and dG*dT- and this could be attributed to the lower pKa ({Delta}pKa [~]0.4-0.9) of U versus T. Our findings suggest plausible roles for Watson-Crick-like G*T/U mismatches in transcriptional errors and CRISPR/Cas9 off-target gene editing, uncover a crucial difference between the chemical dynamics of G*U versus G*T, and indicate that anionic Watson-Crick-like G*U- could play a significant role evading Watson-Crick fidelity checkpoints in RNA:DNA hybrids and RNA duplexes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.553867">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.553867" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.553867">
        <p class="paperTitle">Cryo-EM structures of human magnesium channel MRS2 reveal gating and regulatory mechanisms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.553867" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.553867" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lai, L. T. F.; Balaraman, J.; Zhou, F.; Matthies, D.</p>
        <p class="info">Score: 11.9, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.553867' target='https://doi.org/10.1101/2023.08.22.553867'> 10.1101/2023.08.22.553867</a></p>
        <p class="abstract">Magnesium ions (Mg2&#43;) play an essential role in cellular physiology. In mitochondria, protein and ATP synthesis and various metabolic pathways are directly regulated by Mg2&#43;. MRS2, a magnesium channel located in the inner mitochondrial membrane, mediates the influx of Mg2&#43; into the mitochondrial matrix and regulates Mg2&#43; homeostasis. Knockdown of MRS2 in human cells leads to reduced uptake of Mg2&#43; into mitochondria and disruption of the mitochondrial metabolism. Despite the importance of MRS2, the Mg2&#43; translocation and regulation mechanisms of MRS2 are still unclear. Here, using cryo-EM we determined the structure of human MRS2 in the presence and absence of Mg2&#43; at 2.8 [A] and 3.3 [A], respectively. From the homo-pentameric structures, we identified R332 and M336 as major gating residues, which were then tested using mutagenesis and two cellular divalent ion uptake assays. A network of hydrogen bonds was found connecting the gating residue R332 to the soluble domain, potentially regulating the gate. Two Mg2&#43;-binding sites were identified in the MRS2 soluble domain, distinct from the two sites previously reported in CorA, a homolog of MRS2 in prokaryotes. Altogether, this study provides the molecular basis for understanding the Mg2&#43; translocation and regulatory mechanisms of MRS2.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554779">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554779" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554779">
        <p class="paperTitle">Single-molecule visualization of twin-supercoiled domains generated during transcription</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554779" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554779" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janissen, R.; Barth, R.; Polinder, M.; van der Torre, J.; Dekker, C.</p>
        <p class="info">Score: 11.5, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554779' target='https://doi.org/10.1101/2023.08.25.554779'> 10.1101/2023.08.25.554779</a></p>
        <p class="abstract">Transcription-coupled supercoiling of DNA is a key factor in chromosome compaction and the regulation of genetic processes in all domains of life. It has become common knowledge that, during transcription, the DNA-dependent RNA polymerase (RNAP) induces positive supercoiling ahead of it (downstream) and negative supercoils in its wake (upstream), as rotation of RNAP around the DNA axis upon tracking its helical groove gets constrained due to drag on its RNA transcript. Here, we experimentally validate this so-called twin-supercoiled-domain model with in vitro real-time visualization at the single-molecule scale. Upon binding to the promoter site on a supercoiled DNA molecule, RNAP merges all DNA supercoils into one large pinned plectoneme with RNAP residing at its apex. Transcription by RNAP in real time demonstrates that up- and downstream supercoils are generated simultaneously and in equal portions, in agreement with the twin-supercoiled-domain model. Experiments carried out in the presence of RNases A and H, revealed that an additional viscous drag of the RNA transcript is not necessary for the RNAP to induce supercoils. The latter results contrast the current consensus and simulations on the origin of the twin-supercoiled domains, pointing at an additional mechanistic cause underlying supercoil generation by RNAP in transcription.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.23.554394">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.23.554394" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.23.554394">
        <p class="paperTitle">Modeling membrane geometries implicitly in Rosetta</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.23.554394" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.23.554394" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Woods, H.; Koehler Leman, J.; Meiler, J.</p>
        <p class="info">Score: 9.3, Published: 2023-08-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.23.554394' target='https://doi.org/10.1101/2023.08.23.554394'> 10.1101/2023.08.23.554394</a></p>
        <p class="abstract">Interactions between membrane proteins (MPs) and lipid bilayers are critical for many cellular functions. In the Rosetta molecular modeling suite, the implicit membrane energy function is based on a &#34;slab&#34; model, which represent the membrane as a flat bilayer. However, in nature membranes often have a curvature that is important for function and/or stability. Even more prevalent, in structural biology research MPs are reconstituted in model membrane systems such as micelles, bicelles, nanodiscs, or liposomes. Thus, we have modified the existing membrane energy potentials within the RosettaMP framework to allow users to model MPs in different membrane geometries. We show that these modifications can be utilized in core applications within Rosetta such as structure refinement, protein-protein docking, and protein design. For MPs structures found in curved membranes, refining these structures in curved, implicit membranes produces higher quality models with structures closer to experimentally determined structures. For MP systems embedded in multiple membranes, representing both membranes results in more favorable scores compared to only representing one of the membranes. Modeling MPs in geometries mimicking the membrane model system used in structure determination can improve model quality and model discrimination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553660">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553660" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553660">
        <p class="paperTitle">Structural elucidation of recombinant Trichomonas vaginalis 20S proteasome bound to covalent inhibitors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553660" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553660" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Silhan, J.; Fajtova, P.; Bartosova, J.; Hurysz, B.; Almaliti, J.; Miyamoto, Y.; Eckmann, L.; Gerwick, W.; O Donoghue, A.; Boura, E.</p>
        <p class="info">Score: 8.9, Published: 2023-08-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553660' target='https://doi.org/10.1101/2023.08.17.553660'> 10.1101/2023.08.17.553660</a></p>
        <p class="abstract">Proteasomes are essential for protein homeostasis in mammalian cells1-4 and in protozoan parasites such as Trichomonas vaginalis (Tv).5 Tv and other protozoan 20S proteasomes have been validated as druggable targets.6-8 However, in the case of Tv 20S proteasome (Tv20S), biochemical and structural studies were impeded by low yields and purity of the native proteasome. We successfully made recombinant Tv20S by expressing all seven  and seven {beta} subunits together with the Ump-1 chaperone in insect cells. We isolated recombinant proteasome and showed that it was biochemically indistinguishable from the native enzyme. We confirmed that the recombinant Tv20S is inhibited by the natural product marizomib (MZB)9 and the recently developed peptide inhibitor carmaphycin-17 (CP-17)8,10. Specifically, MZB binds to the {beta}1, {beta}2 and {beta}5 subunits, while CP-17 binds the {beta}2 and {beta}5 subunits. Next, we obtained cryo-EM structures of Tv20S in complex with these covalent inhibitors at 2.8[A] resolution. The structures revealed the overall fold of the Tv20S and the binding mode of MZB and CP-17. Our work explains the low specificity of MZB and higher specificity of CP-17 towards Tv20S as compared to human proteasome and provides the platform for the development of Tv20S inhibitors for treatment of trichomoniasis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
