<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Direct optical measurement of intra-molecular distances down to the Angstrom scale
Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.
Score: 81.1, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133
Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Direct optical measurement of intra-molecular distances down to the Angstrom scale
Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.
Score: 81.1, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133
Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-16T10:39:40+00:00" />
<meta property="article:modified_time" content="2023-07-16T10:39:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Direct optical measurement of intra-molecular distances down to the Angstrom scale
Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.
Score: 81.1, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133
Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Direct optical measurement of intra-molecular distances down to the Angstrom scale\nAuthors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.\nScore: 81.1, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133\nOptical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision.",
  "keywords": [
    
  ],
  "articleBody": " Direct optical measurement of intra-molecular distances down to the Angstrom scale\nAuthors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.\nScore: 81.1, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548133\nOptical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision. Our method is validated by quantifying well-characterized 1-to-10 nm distances in polypeptides and proteins. Moreover, we visualize the orientations of immunoglobulin subunits and reveal specific configurations of a histidine kinase domain dimer. Our results open the door for examining proximities and interactions of macromolecules under physiological conditions.\nMapping Single-molecule Protein Complexes in 3D with DNA Nanoswitch Calipers\nAuthors: Shrestha, P.; Yang, D.; Shih, W. M.; Wong, W. P.\nScore: 11.7, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548386\nThe ability to accurately map the 3D geometry of single-molecule complexes in trace samples would lead to new insights into molecular mechanics and provide an approach for single-molecule structural proteomics. To enable this, we have developed a high-resolution force-spectroscopy method capable of measuring multiple distances between labeled sites in natively folded protein complexes. Our approach combines reconfigurable nanoscale devices we call DNA Nanoswitch Calipers, which we have previously introduced, with a force-based barcoding system to distinguish each measurement location. We demonstrate our approach by reconstructing the tetrahedral geometry of biotin-binding sites in natively folded streptavidin, with 1.5-2.5[A] agreement to previously reported structures.\nIntegrated Fluorescence Microscopy (iFLM) for Cryo-FIB-milling and In-situ Cryo-ET\nAuthors: Yang, J. E.; Vrbovska, V.; Franke, T.; Sibert, B.; Larson, M. R.; Hall, A.; Rigort, A.; Mitchels, J.; Wright, E. R.\nScore: 10.7, Published: 2023-07-12 DOI: 10.1101/2023.07.11.548578\nCorrelative cryo-FLM-FIB milling is a powerful sample preparation technique for in situ cryo-ET. However, correlative workflows that incorporate precise targeting remain challenging. Here, we demonstrate the development and use of an integrated Fluorescence Light Microscope (iFLM) module within a cryo-FIB-SEM to enable a coordinate-based two-point 3D correlative workflow. The iFLM guided targeting of regions of interest coupled with an automated milling process of the cryo-FIB-SEM instrument allows for the efficient preparation of 9-12 ~200 nm thick lamellae within 24 hours. Using regular and montage-cryo-ET data collection schemes, we acquired data from FIB-milled lamellae of HeLa cells to examine cellular ultrastructure. Overall, this workflow facilitates on-the-fly targeting and automated FIB-milling of cryo-preserved cells, bacteria, and possibly high pressure frozen tissue, to produce lamellae for downstream cryo-ET data collection.\nMultivalent interactions of the disordered regions of XLF and XRCC4 foster robust cellular NHEJ and drive the formation of ligation-boosting condensates in vitro\nAuthors: Vu, D.-D.; Bonucci, A.; Breniere, M.; Cisneros-Aguirre, M.; Pelupessy, P.; Wang, Z.; Carlier, L.; Bouvignies, G.; Cortes, P.; Aggarwal, A. K.; Blackledge, M.; Gueroui, Z.; Belle, V.; STARK, J. M.; Modesti, M. M.; Ferrage, F.\nScore: 10.2, Published: 2023-07-13 DOI: 10.1101/2023.07.12.548668\nIn mammalian cells, DNA double-strand breaks are predominantly repaired by non-homologous end joining (NHEJ). During repair, the Ku70/80 heterodimer (Ku), XRCC4 in complex with DNA Ligase 4 (X4L4), and XLF form a flexible scaffold that holds the broken DNA ends together. Insights into the architectural organization of the NHEJ scaffold and its regulation by the DNA-dependent protein kinase catalytic subunit (DNA-PKcs) have recently been obtained by single-particle cryo-electron microscopy analysis. However, several regions, especially the C-terminal regions (CTRs) of the XRCC4 and XLF scaffolding proteins, have largely remained unresolved in experimental structures, which hampers the understanding of their functions. Here, we used magnetic resonance techniques and biochemical assays to comprehensively characterize the interactions and dynamics of the XRCC4 and XLF CTRs at atomic resolution. We show that the CTRs of XRCC4 and XLF are intrinsically disordered and form a network of multivalent heterotypic and homotypic interactions that promotes robust cellular NHEJ activity. Importantly, we demonstrate that the multivalent interactions of these CTRs led to the formation of XLF and X4L4 condensates in vitro which can recruit relevant effectors and critically stimulate DNA end ligation. Our work highlights the role of disordered regions in the mechanism and dynamics of NHEJ and lays the groundwork for the investigation of NHEJ protein disorder and its associated condensates inside cells with implications in cancer biology, immunology and the development of genome editing strategies.\nHandling difficult cryo-ET samples: A study with primary neurons from Drosophila melanogaster\nAuthors: Kim, J. Y.; Yang, J. E.; Mitchell, J. W.; English, L. A.; Yang, S. Z.; Tenpas, T.; Dent, E. W.; Wildonger, J.; Wright, E. R.\nScore: 9.4, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548468\nCellular neurobiology has benefited from recent advances in the field of cryo-electron tomography (cryo-ET). Numerous structural and ultrastructural insights have been obtained from plunge-frozen primary neurons cultured on electron microscopy grids. With most primary neurons been derived from rodent sources, we sought to expand the breadth of sample availability by using primary neurons derived from 3rd instar Drosophila melanogaster larval brains. Ultrastructural abnormalities were encountered while establishing this model system for cryo-ET, which were exemplified by excessive membrane blebbing and cellular fragmentation. To optimize neuronal samples, we integrated substrate selection, micropatterning, montage data collection, and chemical fixation. Efforts to address difficulties in establishing Drosophila neurons for future cryo-ET studies in cellular neurobiology also provided insights that future practitioners can use when attempting to establish other cell-based model systems.\nSingle-molecule digital sizing of proteins in solution\nAuthors: Krainer, G.; Jacquat, R. P. B.; Schneider, M. M.; Welsh, T. J.; Fan, J.; Peter, Q. A. E.; Andrzejewska, E. A.; Sneideriene, G.; Czekalska, M. A.; Ausserwoeger, H.; Chai, L.; Arter, W. E.; Saar, K. L.; Herling, T. W.; Franzmann, T. M.; Kosmoliaptsis, V.; Alberti, S.; Hartl, F.-U.; Lee, S. F.; Knowles, T. P. J.\nScore: 7.8, Published: 2023-07-14 DOI: 10.1101/2023.07.12.548675\nProteins constitute the molecular machinery of life and exert their biological function by interacting with other proteins, as well as by assembling into biomolecular complexes and higher order structures. Characterizing the sizes, interactions, and assembly states of proteins is thus key for understanding the normal functional behavior of proteins and for elucidating aberrant processes and interactions that can lead to dysfunction and disease. However, the physical characterization of proteins has remained a challenging problem due to the inherent compositional heterogeneity of protein mixtures as well as the polydisperse nature of protein complexes. Here, we address this challenge by demonstrating measurements of molecular diffusivity of single proteins and protein assemblies in microchannels using single-molecule fluorescence detection. The approach, termed single-molecule microfluidic diffusional sizing (smMDS), allows individual molecules to be counted directly, that is, in a digital manner, to enable single-molecule diffusional-sizing-based monitoring of protein hydrodynamic radii even within heterogenous multicomponent mixtures. Applying smMDS to a variety of protein systems, we show that the high sensitivity provided by smMDS enables ultrasensitive sizing of proteins in the femtomolar concentration range. We further demonstrate the applicability of the approach towards affinity profiling of protein interactions at the single-molecule level and illustrate the potential of smMDS in resolving different assembly states of high- and low-molecular weight protein oligomers. Furthermore, we highlight the digital nature of the detection process by sizing multiple protein species within complex aggregation mixtures. Finally, we apply the approach to characterize nanoscale clusters of a phase separating protein system. Taken together, smMDS constitutes a versatile approach for digital, in-solution characterization of the sizes, interactions, and assembly states of proteins. We anticipate that smMDS will facilitate the discovery of new biomolecular mechanisms of proteins and will find broad applicability in the analysis of protein complexes in the biological, biophysical, and biomedical sciences, and beyond.\nPEP-Patch: Electrostatics in Protein-Protein Recognition, Specificity and Antibody Developability\nAuthors: Waibl, F.; Pomarici, N. D.; Hoerschinger, V. J.; Loeffler, J. R.; Deane, C.; Georges, G.; Kettenberger, H.; Fernandez-Quintero, M. L.; Liedl, K. R.\nScore: 6.7, Published: 2023-07-15 DOI: 10.1101/2023.07.14.547811\nThe electrostatic properties of proteins arise from the number and distribution of polar and charged residues. Due to their long-ranged nature, electrostatic interactions in proteins play a critical role in numerous processes, such as molecular recognition, protein solubility, viscosity, and antibody developability. Thus, characterizing and quantifying electrostatic properties of a protein is a pre-requisite for understanding these processes. Here, we present PEP-Patch, a tool to visualize and quantify the electrostatic potential on the protein surface and showcase its applicability to elucidate protease substrate specificity, antibody-antigen recognition and predict heparin column retention times of antibodies as an indicator of pharmacokinetics.\nThe importance of input sequence set to consensus-derived proteins and their relationship to reconstructed ancestral proteins\nAuthors: Nixon, C.; Lim, S. A.; Sternke, M.; Barrick, D.; Harms, M. J.; Marqusee, S.\nScore: 8.3, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547063\nA protein sequence encodes its energy landscape - all the accessible conformations, energetics, and dynamics. The evolutionary relationship between sequence and landscape can be probed phylogenetically by compiling a multiple sequence alignment of homologous sequences and generating common ancestors via Ancestral Sequence Reconstruction or a consensus protein containing the most common amino acid at each position. Both ancestral and consensus proteins are often more stable than their extant homologs - questioning the differences and suggesting that both approaches serve as general methods to engineer thermostability. We used the Ribonuclease H family to compare these approaches and evaluate how the evolutionary relationship of the input sequences affects the properties of the resulting consensus protein. While the overall consensus protein is structured and active, it neither shows properties of a well-folded protein nor has enhanced stability. In contrast, the consensus protein derived from a phylogenetically-restricted region is significantly more stable and cooperatively folded, suggesting that cooperativity may be encoded by different mechanisms in separate clades and lost when too many diverse clades are combined to generate a consensus protein. To explore this, we compared pairwise covariance scores using a Potts formalism as well as higher-order couplings using singular value decomposition (SVD). We find the SVD coordinates of a stable consensus sequence are close to coordinates of the analogous ancestor sequence and its descendants, whereas the unstable consensus sequences are outliers in SVD space.\nAtomic-level architecture of Caulobacter crescentus flagellar filaments provide evidence for multi-flagellin filament stabilization\nAuthors: Sanchez, J. C.; Montemayor, E. J.; Ploscariu, N. T.; Parrell, D.; Baumgardt, J. K.; Yang, J. E.; Sibert, B.; Cai, K.; Wright, E. R.\nScore: 5.0, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548443\nFlagella are dynamic, ion-powered machines with assembly pathways that are optimized for efficient flagella production. In bacteria, dozens of genes are coordinated at specific times in the cell lifecycle to generate each component of the flagellum. This is the case for Caulobacter crescentus, but little is known about why this species encodes six different flagellin genes. Furthermore, little is known about the benefits multi-flagellin species possess over single flagellin species, if any, or what molecular properties allow for multi-flagellin filaments to assemble. Here we present an in-depth analysis of several single flagellin filaments from C. crescentus, including an extremely well-resolved structure of a bacterial flagellar filament. We highlight key molecular interactions that differ between each bacterial strain and speculate how these interactions may alleviate or impose helical strain on the overall architecture of the filament. We detail conserved residues within the flagellin subunit that allow for the synthesis of multi-flagellin filaments. We further comment on how these molecular differences impact bacterial motility and highlight how no single flagellin filament achieves wild-type levels of motility, suggesting C. crescentus has evolved to produce a filament optimized for motility comprised of six flagellins. Finally, we highlight an ordered arrangement of glycosylation sites on the surface of the filaments and speculate how these sites may protect the {beta}-hairpin located on the surface exposed domain of the flagellin subunit.\nQuantitative Models of Molecular Dynamics from Sparse Simulation and Experimental Data\nAuthors: Kolloff, C.; Olsson, S.\nScore: 9.5, Published: 2023-07-11 DOI: 10.1101/2023.05.23.541878\nLong-timescale behavior of proteins is fundamental to many biological processes. Molecular Dynamics (MD) simulations and biophysical experiments are often used to study protein dynamics. However, high computational demands of MD limit what timescales are feasible to study, often missing rare events, which are critical to explain experiments. On the other hand, experiments are limited by low resolution. We present dynamic Augmented Markov models (dynAMMo) to bridge the gap between these data and overcome their respective limitations. For the first time, dynAMMo enables the construction of mechanistic models of slow exchange processes that have been not observed in MD data by integrating dynamic experimental observables. As a consequence, dynAMMo allows us to bypass costly and extensive simulations, yet providing mechanistic insights of the system. Validated with controlled model systems and a well-studied protein, dynAMMo offers a new approach to quantitatively model protein dynamics on long timescales in an unprecedented manner.\n",
  "wordCount" : "2116",
  "inLanguage": "en",
  "datePublished": "2023-07-16T10:39:40Z",
  "dateModified": "2023-07-16T10:39:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on July 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548133" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548133">
        <p class="paperTitle">Direct optical measurement of intra-molecular distances down to the Angstrom scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sahl, S. J.; Matthias, J.; Inamdar, K.; Khan, T. A.; Weber, M.; Becker, S.; Griesinger, C.; Broichhagen, J.; Hell, S. W.</p>
        <p class="info">Score: 81.1, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548133' target='https://doi.org/10.1101/2023.07.07.548133'> 10.1101/2023.07.07.548133</a></p>
        <p class="abstract">Optical investigations of nanometer distances between proteins, their subunits, or other biomolecules have been the exclusive prerogative of Forster Resonance Energy Transfer (FRET) microscopy for decades. Here we show that MINFLUX fluorescence nanoscopy measures intra-molecular distances down to 1 nm - and in planar projections down to 1 Angstrom - directly, linearly, and with Angstrom precision. Our method is validated by quantifying well-characterized 1-to-10 nm distances in polypeptides and proteins. Moreover, we visualize the orientations of immunoglobulin subunits and reveal specific configurations of a histidine kinase domain dimer. Our results open the door for examining proximities and interactions of macromolecules under physiological conditions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548386">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548386" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548386">
        <p class="paperTitle">Mapping Single-molecule Protein Complexes in 3D with DNA Nanoswitch Calipers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548386" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548386" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shrestha, P.; Yang, D.; Shih, W. M.; Wong, W. P.</p>
        <p class="info">Score: 11.7, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548386' target='https://doi.org/10.1101/2023.07.10.548386'> 10.1101/2023.07.10.548386</a></p>
        <p class="abstract">The ability to accurately map the 3D geometry of single-molecule complexes in trace samples would lead to new insights into molecular mechanics and provide an approach for single-molecule structural proteomics. To enable this, we have developed a high-resolution force-spectroscopy method capable of measuring multiple distances between labeled sites in natively folded protein complexes. Our approach combines reconfigurable nanoscale devices we call DNA Nanoswitch Calipers, which we have previously introduced, with a force-based barcoding system to distinguish each measurement location. We demonstrate our approach by reconstructing the tetrahedral geometry of biotin-binding sites in natively folded streptavidin, with 1.5-2.5[A] agreement to previously reported structures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548578">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548578" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548578">
        <p class="paperTitle">Integrated Fluorescence Microscopy (iFLM) for Cryo-FIB-milling and In-situ Cryo-ET</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548578" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548578" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yang, J. E.; Vrbovska, V.; Franke, T.; Sibert, B.; Larson, M. R.; Hall, A.; Rigort, A.; Mitchels, J.; Wright, E. R.</p>
        <p class="info">Score: 10.7, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548578' target='https://doi.org/10.1101/2023.07.11.548578'> 10.1101/2023.07.11.548578</a></p>
        <p class="abstract">Correlative cryo-FLM-FIB milling is a powerful sample preparation technique for in situ cryo-ET. However, correlative workflows that incorporate precise targeting remain challenging. Here, we demonstrate the development and use of an integrated Fluorescence Light Microscope (iFLM) module within a cryo-FIB-SEM to enable a coordinate-based two-point 3D correlative workflow. The iFLM guided targeting of regions of interest coupled with an automated milling process of the cryo-FIB-SEM instrument allows for the efficient preparation of 9-12 ~200 nm thick lamellae within 24 hours. Using regular and montage-cryo-ET data collection schemes, we acquired data from FIB-milled lamellae of HeLa cells to examine cellular ultrastructure. Overall, this workflow facilitates on-the-fly targeting and automated FIB-milling of cryo-preserved cells, bacteria, and possibly high pressure frozen tissue, to produce lamellae for downstream cryo-ET data collection.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548668">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548668" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548668">
        <p class="paperTitle">Multivalent interactions of the disordered regions of XLF and XRCC4 foster robust cellular NHEJ and drive the formation of ligation-boosting condensates in vitro</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548668" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548668" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vu, D.-D.; Bonucci, A.; Breniere, M.; Cisneros-Aguirre, M.; Pelupessy, P.; Wang, Z.; Carlier, L.; Bouvignies, G.; Cortes, P.; Aggarwal, A. K.; Blackledge, M.; Gueroui, Z.; Belle, V.; STARK, J. M.; Modesti, M. M.; Ferrage, F.</p>
        <p class="info">Score: 10.2, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548668' target='https://doi.org/10.1101/2023.07.12.548668'> 10.1101/2023.07.12.548668</a></p>
        <p class="abstract">In mammalian cells, DNA double-strand breaks are predominantly repaired by non-homologous end joining (NHEJ). During repair, the Ku70/80 heterodimer (Ku), XRCC4 in complex with DNA Ligase 4 (X4L4), and XLF form a flexible scaffold that holds the broken DNA ends together. Insights into the architectural organization of the NHEJ scaffold and its regulation by the DNA-dependent protein kinase catalytic subunit (DNA-PKcs) have recently been obtained by single-particle cryo-electron microscopy analysis. However, several regions, especially the C-terminal regions (CTRs) of the XRCC4 and XLF scaffolding proteins, have largely remained unresolved in experimental structures, which hampers the understanding of their functions. Here, we used magnetic resonance techniques and biochemical assays to comprehensively characterize the interactions and dynamics of the XRCC4 and XLF CTRs at atomic resolution. We show that the CTRs of XRCC4 and XLF are intrinsically disordered and form a network of multivalent heterotypic and homotypic interactions that promotes robust cellular NHEJ activity. Importantly, we demonstrate that the multivalent interactions of these CTRs led to the formation of XLF and X4L4 condensates in vitro which can recruit relevant effectors and critically stimulate DNA end ligation. Our work highlights the role of disordered regions in the mechanism and dynamics of NHEJ and lays the groundwork for the investigation of NHEJ protein disorder and its associated condensates inside cells with implications in cancer biology, immunology and the development of genome editing strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548468">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548468" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548468">
        <p class="paperTitle">Handling difficult cryo-ET samples: A study with primary neurons from Drosophila melanogaster</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548468" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548468" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, J. Y.; Yang, J. E.; Mitchell, J. W.; English, L. A.; Yang, S. Z.; Tenpas, T.; Dent, E. W.; Wildonger, J.; Wright, E. R.</p>
        <p class="info">Score: 9.4, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548468' target='https://doi.org/10.1101/2023.07.10.548468'> 10.1101/2023.07.10.548468</a></p>
        <p class="abstract">Cellular neurobiology has benefited from recent advances in the field of cryo-electron tomography (cryo-ET). Numerous structural and ultrastructural insights have been obtained from plunge-frozen primary neurons cultured on electron microscopy grids. With most primary neurons been derived from rodent sources, we sought to expand the breadth of sample availability by using primary neurons derived from 3rd instar Drosophila melanogaster larval brains. Ultrastructural abnormalities were encountered while establishing this model system for cryo-ET, which were exemplified by excessive membrane blebbing and cellular fragmentation. To optimize neuronal samples, we integrated substrate selection, micropatterning, montage data collection, and chemical fixation. Efforts to address difficulties in establishing Drosophila neurons for future cryo-ET studies in cellular neurobiology also provided insights that future practitioners can use when attempting to establish other cell-based model systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548675">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548675" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548675">
        <p class="paperTitle">Single-molecule digital sizing of proteins in solution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548675" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548675" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krainer, G.; Jacquat, R. P. B.; Schneider, M. M.; Welsh, T. J.; Fan, J.; Peter, Q. A. E.; Andrzejewska, E. A.; Sneideriene, G.; Czekalska, M. A.; Ausserwoeger, H.; Chai, L.; Arter, W. E.; Saar, K. L.; Herling, T. W.; Franzmann, T. M.; Kosmoliaptsis, V.; Alberti, S.; Hartl, F.-U.; Lee, S. F.; Knowles, T. P. J.</p>
        <p class="info">Score: 7.8, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548675' target='https://doi.org/10.1101/2023.07.12.548675'> 10.1101/2023.07.12.548675</a></p>
        <p class="abstract">Proteins constitute the molecular machinery of life and exert their biological function by interacting with other proteins, as well as by assembling into biomolecular complexes and higher order structures. Characterizing the sizes, interactions, and assembly states of proteins is thus key for understanding the normal functional behavior of proteins and for elucidating aberrant processes and interactions that can lead to dysfunction and disease. However, the physical characterization of proteins has remained a challenging problem due to the inherent compositional heterogeneity of protein mixtures as well as the polydisperse nature of protein complexes. Here, we address this challenge by demonstrating measurements of molecular diffusivity of single proteins and protein assemblies in microchannels using single-molecule fluorescence detection. The approach, termed single-molecule microfluidic diffusional sizing (smMDS), allows individual molecules to be counted directly, that is, in a digital manner, to enable single-molecule diffusional-sizing-based monitoring of protein hydrodynamic radii even within heterogenous multicomponent mixtures. Applying smMDS to a variety of protein systems, we show that the high sensitivity provided by smMDS enables ultrasensitive sizing of proteins in the femtomolar concentration range. We further demonstrate the applicability of the approach towards affinity profiling of protein interactions at the single-molecule level and illustrate the potential of smMDS in resolving different assembly states of high- and low-molecular weight protein oligomers. Furthermore, we highlight the digital nature of the detection process by sizing multiple protein species within complex aggregation mixtures. Finally, we apply the approach to characterize nanoscale clusters of a phase separating protein system. Taken together, smMDS constitutes a versatile approach for digital, in-solution characterization of the sizes, interactions, and assembly states of proteins. We anticipate that smMDS will facilitate the discovery of new biomolecular mechanisms of proteins and will find broad applicability in the analysis of protein complexes in the biological, biophysical, and biomedical sciences, and beyond.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.547811">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.547811" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.547811">
        <p class="paperTitle">PEP-Patch: Electrostatics in Protein-Protein Recognition, Specificity and Antibody Developability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.547811" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.547811" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waibl, F.; Pomarici, N. D.; Hoerschinger, V. J.; Loeffler, J. R.; Deane, C.; Georges, G.; Kettenberger, H.; Fernandez-Quintero, M. L.; Liedl, K. R.</p>
        <p class="info">Score: 6.7, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.547811' target='https://doi.org/10.1101/2023.07.14.547811'> 10.1101/2023.07.14.547811</a></p>
        <p class="abstract">The electrostatic properties of proteins arise from the number and distribution of polar and charged residues. Due to their long-ranged nature, electrostatic interactions in proteins play a critical role in numerous processes, such as molecular recognition, protein solubility, viscosity, and antibody developability. Thus, characterizing and quantifying electrostatic properties of a protein is a pre-requisite for understanding these processes. Here, we present PEP-Patch, a tool to visualize and quantify the electrostatic potential on the protein surface and showcase its applicability to elucidate protease substrate specificity, antibody-antigen recognition and predict heparin column retention times of antibodies as an indicator of pharmacokinetics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547063">
        <p class="paperTitle">The importance of input sequence set to consensus-derived proteins and their relationship to reconstructed ancestral proteins</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nixon, C.; Lim, S. A.; Sternke, M.; Barrick, D.; Harms, M. J.; Marqusee, S.</p>
        <p class="info">Score: 8.3, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547063' target='https://doi.org/10.1101/2023.06.29.547063'> 10.1101/2023.06.29.547063</a></p>
        <p class="abstract">A protein sequence encodes its energy landscape - all the accessible conformations, energetics, and dynamics. The evolutionary relationship between sequence and landscape can be probed phylogenetically by compiling a multiple sequence alignment of homologous sequences and generating common ancestors via Ancestral Sequence Reconstruction or a consensus protein containing the most common amino acid at each position. Both ancestral and consensus proteins are often more stable than their extant homologs - questioning the differences and suggesting that both approaches serve as general methods to engineer thermostability. We used the Ribonuclease H family to compare these approaches and evaluate how the evolutionary relationship of the input sequences affects the properties of the resulting consensus protein. While the overall consensus protein is structured and active, it neither shows properties of a well-folded protein nor has enhanced stability. In contrast, the consensus protein derived from a phylogenetically-restricted region is significantly more stable and cooperatively folded, suggesting that cooperativity may be encoded by different mechanisms in separate clades and lost when too many diverse clades are combined to generate a consensus protein. To explore this, we compared pairwise covariance scores using a Potts formalism as well as higher-order couplings using singular value decomposition (SVD). We find the SVD coordinates of a stable consensus sequence are close to coordinates of the analogous ancestor sequence and its descendants, whereas the unstable consensus sequences are outliers in SVD space.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548443">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548443" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548443">
        <p class="paperTitle">Atomic-level architecture of Caulobacter crescentus flagellar filaments provide evidence for multi-flagellin filament stabilization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548443" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548443" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez, J. C.; Montemayor, E. J.; Ploscariu, N. T.; Parrell, D.; Baumgardt, J. K.; Yang, J. E.; Sibert, B.; Cai, K.; Wright, E. R.</p>
        <p class="info">Score: 5.0, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548443' target='https://doi.org/10.1101/2023.07.10.548443'> 10.1101/2023.07.10.548443</a></p>
        <p class="abstract">Flagella are dynamic, ion-powered machines with assembly pathways that are optimized for efficient flagella production. In bacteria, dozens of genes are coordinated at specific times in the cell lifecycle to generate each component of the flagellum. This is the case for Caulobacter crescentus, but little is known about why this species encodes six different flagellin genes. Furthermore, little is known about the benefits multi-flagellin species possess over single flagellin species, if any, or what molecular properties allow for multi-flagellin filaments to assemble. Here we present an in-depth analysis of several single flagellin filaments from C. crescentus, including an extremely well-resolved structure of a bacterial flagellar filament. We highlight key molecular interactions that differ between each bacterial strain and speculate how these interactions may alleviate or impose helical strain on the overall architecture of the filament. We detail conserved residues within the flagellin subunit that allow for the synthesis of multi-flagellin filaments. We further comment on how these molecular differences impact bacterial motility and highlight how no single flagellin filament achieves wild-type levels of motility, suggesting C. crescentus has evolved to produce a filament optimized for motility comprised of six flagellins. Finally, we highlight an ordered arrangement of glycosylation sites on the surface of the filaments and speculate how these sites may protect the {beta}-hairpin located on the surface exposed domain of the flagellin subunit.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.23.541878">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.23.541878" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.23.541878">
        <p class="paperTitle">Quantitative Models of Molecular Dynamics from Sparse Simulation and Experimental Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.23.541878" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.23.541878" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kolloff, C.; Olsson, S.</p>
        <p class="info">Score: 9.5, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.23.541878' target='https://doi.org/10.1101/2023.05.23.541878'> 10.1101/2023.05.23.541878</a></p>
        <p class="abstract">Long-timescale behavior of proteins is fundamental to many biological processes. Molecular Dynamics (MD) simulations and biophysical experiments are often used to study protein dynamics. However, high computational demands of MD limit what timescales are feasible to study, often missing rare events, which are critical to explain experiments. On the other hand, experiments are limited by low resolution. We present dynamic Augmented Markov models (dynAMMo) to bridge the gap between these data and overcome their respective limitations. For the first time, dynAMMo enables the construction of mechanistic models of slow exchange processes that have been not observed in MD data by integrating dynamic experimental observables. As a consequence, dynAMMo allows us to bypass costly and extensive simulations, yet providing mechanistic insights of the system. Validated with controlled model systems and a well-studied protein, dynAMMo offers a new approach to quantitatively model protein dynamics on long timescales in an unprecedented manner.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
