<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Real-Time Structural Biology of DNA and DNA-Protein Complexes on an Optical Microscope
Authors: Szalai, A. M.; Ferrari, G.; Richter, L.; Hartmann, J.; Kesici, M.-Z.; Ji, B.; Koshic, K.; Jaeger, A.; Aksimentiev, A.; Tessmer, I.; Kaminska, I.; Vera, A. M.; Tinnefeld, P.
Score: 40.7, Published: 2023-11-21 DOI: 10.1101/2023.11.21.567962
The intricate interplay between DNA and proteins is key for biological functions such as DNA replication, transcription, and repair. To better understand these interactions, it is crucial to develop tools to study DNA-protein complexes with high spatial and temporal resolution.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Real-Time Structural Biology of DNA and DNA-Protein Complexes on an Optical Microscope
Authors: Szalai, A. M.; Ferrari, G.; Richter, L.; Hartmann, J.; Kesici, M.-Z.; Ji, B.; Koshic, K.; Jaeger, A.; Aksimentiev, A.; Tessmer, I.; Kaminska, I.; Vera, A. M.; Tinnefeld, P.
Score: 40.7, Published: 2023-11-21 DOI: 10.1101/2023.11.21.567962
The intricate interplay between DNA and proteins is key for biological functions such as DNA replication, transcription, and repair. To better understand these interactions, it is crucial to develop tools to study DNA-protein complexes with high spatial and temporal resolution." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-26T10:37:33+00:00" />
<meta property="article:modified_time" content="2023-11-26T10:37:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Real-Time Structural Biology of DNA and DNA-Protein Complexes on an Optical Microscope
Authors: Szalai, A. M.; Ferrari, G.; Richter, L.; Hartmann, J.; Kesici, M.-Z.; Ji, B.; Koshic, K.; Jaeger, A.; Aksimentiev, A.; Tessmer, I.; Kaminska, I.; Vera, A. M.; Tinnefeld, P.
Score: 40.7, Published: 2023-11-21 DOI: 10.1101/2023.11.21.567962
The intricate interplay between DNA and proteins is key for biological functions such as DNA replication, transcription, and repair. To better understand these interactions, it is crucial to develop tools to study DNA-protein complexes with high spatial and temporal resolution."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Real-Time Structural Biology of DNA and DNA-Protein Complexes on an Optical Microscope\nAuthors: Szalai, A. M.; Ferrari, G.; Richter, L.; Hartmann, J.; Kesici, M.-Z.; Ji, B.; Koshic, K.; Jaeger, A.; Aksimentiev, A.; Tessmer, I.; Kaminska, I.; Vera, A. M.; Tinnefeld, P.\nScore: 40.7, Published: 2023-11-21 DOI: 10.1101/2023.11.21.567962\nThe intricate interplay between DNA and proteins is key for biological functions such as DNA replication, transcription, and repair. To better understand these interactions, it is crucial to develop tools to study DNA-protein complexes with high spatial and temporal resolution.",
  "keywords": [
    
  ],
  "articleBody": " Real-Time Structural Biology of DNA and DNA-Protein Complexes on an Optical Microscope\nAuthors: Szalai, A. M.; Ferrari, G.; Richter, L.; Hartmann, J.; Kesici, M.-Z.; Ji, B.; Koshic, K.; Jaeger, A.; Aksimentiev, A.; Tessmer, I.; Kaminska, I.; Vera, A. M.; Tinnefeld, P.\nScore: 40.7, Published: 2023-11-21 DOI: 10.1101/2023.11.21.567962\nThe intricate interplay between DNA and proteins is key for biological functions such as DNA replication, transcription, and repair. To better understand these interactions, it is crucial to develop tools to study DNA-protein complexes with high spatial and temporal resolution. Here, we use the vertical orientation that DNA adopts on graphene and investigate its interactions with proteins via energy transfer from a probe dye to graphene, achieving spatial resolution down to the [A]ngstrom scale. We measured the bending angle of DNA induced by adenine tracts, bulges, abasic sites and the binding of Escherichia coli endonuclease IV with unprecedented precision and millisecond time resolution. Additionally, we observed the translocation of the O6-alkylguanine DNA alkyltransferase along double-stranded DNA, reaching single-base pair resolution and detecting an affinity for adenine tracts. Overall, we foresee that this method will become a widespread tool for the dynamical study of nucleic acid and nucleic acid-protein interactions.\nConnecting Chromatin Structures to Gene Regulation Using Dynamic Polymer Simulations\nAuthors: Fu, Y.; Clark, F. T.; Nomikou, S.; Tsirigos, A.; Lionnet, T.\nScore: 38.5, Published: 2023-11-10 DOI: 10.1101/2023.11.07.566032\nThe transfer of regulatory information between distal loci on chromatin is thought to involve physical proximity, but key biophysical features of these contacts remain unclear. For instance, it is unknown how close and for how long two loci need to be in order to productively interact. The main challenge is that it is currently impossible to measure chromatin dynamics with high spatiotemporal resolution at scale. Polymer simulations provide an accessible and rigorous way to test biophysical models of chromatin regulation, yet there is a lack of simple and general methods for extracting the values of model parameters. Here we adapt the Nelder-Mead simplex optimization algorithm to select the best polymer model matching a given Hi-C dataset, using the MYC locus as an example. The models biophysical parameters predict a compartmental rearrangement of the MYC locus in leukemia, which we validate with single-cell measurements. Leveraging trajectories predicted by the model, we find that loci with similar Hi-C contact frequencies can exhibit widely different contact dynamics. Interestingly, the frequency of productive interactions between loci exhibits a non-linear relationship with their Hi-C contact frequency when we enforce a specific capture radius and contact duration. These observations are consistent with recent experimental observations and suggest that the dynamic ensemble of chromatin configurations, rather than average contact matrices, is required to fully predict long-range chromatin interactions.\nPrinting, Characterising, and Assessing Transparent 3D Printed Lenses for Optical Imaging\nAuthors: Rooney, L. M.; Christopher, J.; Watson, B.; Susir Kumar, Y.; Copeland, L.; Walker, L. D.; Foylan, S.; Amos, W. B.; Bauer, R.; McConnell, G.\nScore: 35.4, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568002\nThe production of high-quality lenses for optical instrumentation has involved subtractive manufacturing methods for centuries. These methods have demanded specialist equipment and expertise that often render custom high-grade bulk glass optics inaccessible. We aimed to develop a low-cost, accessible, and reproducible method using consumer-grade technology to manufacture high-quality three-dimensional (3D) printed lenses with comparable performance to glass lenses. Various 3D printed planoconvex lenses were produced using a consumer grade 3D printer and low-cost spin coating setup, and printed lenses were compared to their commercial glass counterparts. A range of mechanical and optical methods are introduced to determine the surface curvature of 3D printed lenses, providing a wide range of quality control methods. Amongst others, high-resolution interference reflection microscopy methods were used to reconstruct the convex surface of printed lenses and quantify their radius of curvature. Moreover, the optical throughput and focusing performance of 3D printed lenses were assessed using optical transmissivity measurements and classical beam characterisation methods. We determined that all 3D printed lens prescriptions tested had comparable surface curvature and optical performance to commercial glass lenses. Finally, we demonstrated the application of 3D printed lenses for brightfield transmission microscopy, resolving sub-cellular structures over a 2.3 mm field of view. The high reproducibility and comparable performance of 3D printed lenses present a great opportunity for additive manufacturing in the production of bespoke optical elements for low-cost rapid prototyping and improved accessibility to high-quality optics in low-resource settings.\nSubstrate accessibility regulation of human TopIIa decatenation by cohesin\nAuthors: Cutts, E.; Saravanan, S.; Fisher, G.; Rueda, D. S.; Aragon, L.\nScore: 16.8, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567865\nHuman topoisomerase II alpha (TOP2) is the main mitotic decatenase 1-3 resolving intertwines between sister chromatids that form during DNA replication 4. Here, we employ quadruple-trap optical tweezers to generate braids between a pair of {lambda}-DNA molecules to study, at the single-molecule level and in real-time, how TOP2 untangles these DNAs. We find that TOP2 rapidly resolves single and multiple braids and is inhibited by chemotherapy agent, etoposide. TOP2 is sensitive to DNA conformation, exhibiting a chiral preference for the removal of braids with right-handed crossings and is inhibited when DNAs are held at forces over 20pN. We show that TOP2 must load at the cross between the two DNAs to resolve braids efficiently. TOP2 pre-loaded onto individual DNAs is unable to resolve newly formed braids in the presence of ATP, suggesting that the geometry of pre-bound enzyme is incompatible with the capture of the second DNA. Finally, we show that cohesin binds to braids, preventing TOP2 from resolving them. Our study unveils novel insights into the regulation of TOP2s decatenation, underscoring the importance of substrate accessibility and the role of cohesin in modulating TOP2 activity.\nNanoLocz: Image analysis platform for AFM, high-speed AFM and localization AFM\nAuthors: Heath, G. R.; Micklethwaite, E.; Storer, T. M.\nScore: 13.7, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568405\nNanoLocz is an open-source computer program designed for high-throughput automatic processing and single-particle analysis of Atomic Force Microscopy (AFM) image data. High-Speed AFM and Localization AFM (LAFM) enable the study of single molecules with increasingly higher spatiotemporal resolution. However, efficient and rapid analysis of the images and movies produced by these techniques can be challenging, often requiring the use of multiple image processing software applications and scripts. Here, we introduce NanoLocz, an AFM and high-speed AFM analysis program that facilitates various single-particle analysis workflows through a simple, interactive interface. Workflows include but are not limited to: single-particle tracking, single-particle topographic feature analysis, single-molecule LAFM, time-resolved LAFM, and simulation LAFM. The source code and installation instructions for NanoLocz are available at https://github.com/George-R-Heath/NanoLocz.\nAccurate Characterization of Conformational Ensembles and Binding Mechanisms of the SARS-CoV-2 Omicron BA.2 and BA.2.86 Spike Protein with the Host Receptor and Distinct Classes of Antibodies Using AlphaFold2-Augmented Integrative Computational Modeling\nAuthors: Raisinghani, N.; Alshahrani, M.; Gupta, G.; Xiao, S.; Tao, P.; Verkhivker, G.\nScore: 13.3, Published: 2023-11-21 DOI: 10.1101/2023.11.18.567697\nThe latest wave SARS-CoV-2 Omicron variants displayed a growth advantage and the increased viral fitness through convergent evolution of functional hotspots that work synchronously to balance fitness requirements for productive receptor binding and efficient immune evasion. In this study, we combined AlphaFold2-based structural modeling approaches with all-atom MD simulations and mutational profiling of binding energetics and stability for prediction and comprehensive analysis of the structure, dynamics, and binding of the SARS-CoV-2 Omicron BA.2.86 spike variant with ACE2 host receptor and distinct classes of antibodies. We adapted several AlphaFold2 approaches to predict both structure and conformational ensembles of the Omicron BA.2.86 spike protein in the complex with the host receptor. The results showed that AlphaFold2-predicted conformational ensemble of the BA.2.86 spike protein complex can accurately capture the main dynamics signatures obtained from microscond molecular dynamics simulations. The ensemble-based dynamic mutational scanning of the receptor binding domain residues in the BA.2 and BA.2.86 spike complexes with ACE2 dissected the role of the BA.2 and BA.2.86 backgrounds in modulating binding free energy changes revealing a group of conserved hydrophobic hotspots and critical variant-specific contributions of the BA.2.86 mutational sites R403K, F486P and R493Q. To examine immune evasion properties of BA.2.86 in atomistic detail, we performed large scale structure-based mutational profiling of the S protein binding interfaces with distinct classes of antibodies that displayed significantly reduced neutralization against BA.2.86 variant. The results quantified specific function of the BA.2.86 mutations to ensure broad resistance against different classes of RBD antibodies. This study revealed the molecular basis of compensatory functional effects of the binding hotspots, showing that BA.2.86 lineage may have primarily evolved to improve immune escape while modulating binding affinity with ACE2 through cooperative effect of R403K, F486P and R493Q mutations. The study supports a hypothesis that the impact of the increased ACE2 binding affinity on viral fitness is more universal and is mediated through cross-talk between convergent mutational hotspots, while the effect of immune evasion could be more variant-dependent.\nProtein Stability Prediction by Fine-tuning a Protein Language Model on a Mega-scale Dataset\nAuthors: Chu, K. S.; Siegel, J. B.\nScore: 12.2, Published: 2023-11-19 DOI: 10.1101/2023.11.19.567747\nThe stability of a protein is crucial to its utility in industrial applications. While engineering campaigns can now be routinely used to enhance protein thermal stability to the level needed in an industrial setting, there is a significant desire to fast-track these efforts through predictive tools allowing one to jump in a minimal number of design iterations to a highly stabilized protein. In this work, we explore utilizing a mega-scale dataset for development of a protein language model tuned for stability. This model is trained on the folding stability of 528k sequences derived from 461 small protein domains and designs, and can accommodate deletions, insertions, and multiple-point mutations. We show that a protein language model can be fine-tuned to predict folding stability. The fine-tuned protein language model, named ESMtherm, performs reasonably on small protein domains and generalizes to sequences distal from the training set. Lastly, we discuss its limitations when compared to other state-of-the-art methods in generalizing to larger protein scaffolds and highlight the need of large-scale stability measurement on a diverse dataset that represents the distribution of sequence lengths commonly observed in nature.\nThe mechanism of mRNA activation\nAuthors: Gentry, R. C.; Ide, N. A.; Comunale, V. M.; Hartwick, E. W.; Kinz-Thompson, C. D.; Gonzalez, R. L.\nScore: 32.9, Published: 2023-11-15 DOI: 10.1101/2023.11.15.567265\nDuring translation initiation, messenger RNA molecules must be identified and activated for loading into a ribosome. In this rate-limiting step, the heterotrimeric protein eukaryotic initiation factor eIF4F must recognize and productively interact with the 7-methylguanosine cap at the 5 end of the messenger RNA and subsequently activate the message. Despite its fundamental, regulatory role in gene expression, the molecular events underlying cap recognition and messenger RNA activation remain mysterious. Here, we generate a unique, single-molecule fluorescence imaging system to interrogate the dynamics with which eIF4F discriminates productive and non-productive locations on full-length, native messenger RNA molecules. At the single-molecule level, we observe stochastic sampling of eIF4F along the length of the messenger RNA and identify allosteric communication between the eIF4F subunits which ultimately drive cap-recognition and subsequent activation of the message. Our experiments uncover novel functions for each subunit of eIF4F and we conclude by presenting a model for messenger RNA activation which precisely defines the composition of the activated message. This model provides a general framework for understanding how messenger RNA molecules may be discriminated from one another, and how other RNA-binding proteins may control the efficiency of translation initiation.\nLow-cost 3D printed lenses for brightfield and fluorescence microscopy\nAuthors: Christopher, J.; Rooney, L. M.; Donnachie, M.; Uttamchandani, D.; McConnell, G.; Bauer, R.\nScore: 10.2, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568227\nWe present the fabrication and implementation of low-cost optical quality 3D printed lenses, and their application as microscope objectives with different prescriptions. The imaging performance of the 3D printed lenses was benchmarked against commercially available optics including a 20 mm focal length 12.7 mm diameter NBK-7 plano-convex lens used as a low magnification objective, and a separate high magnification objective featuring three 6 mm diameter NBK-7 lenses with different positive and negative focal lengths. We describe the design and manufacturing processes to produce high-quality 3D printed lenses. We tested their transmissivity and surface quality using a stylus profilometer, showing that they conform to that of commercial glass counterpart lenses. The 3D printed lenses were used as microscope objectives in both brightfield and epi-fluorescence imaging of specimens including onion, cyanobacteria, and variegated Hosta leaves, demonstrating a sub-cellular resolution performance obtained with low-cost 3D printed optical elements within brightfield and fluorescence microscopy.\nA DNA condensation code for linker histones\nAuthors: Watson, M.; Sabirova, D.; Hardy, M. C.; Pan, Y.; Yates, H.; Wright, C. J.; Chan, W. H.; Destan, E.; Stott, K.\nScore: 9.6, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567813\nLinker histones play an essential role in chromatin packaging by facilitating compaction of the 11-nm fibre of nucleosomal \"beads on a string\". The result is a heterogeneous condensed state with local properties that range from dynamic, irregular and liquid-like, to stable and regular structures (the 30-nm fibre), which in turn impact chromatin-dependent activities at a fundamental level. The properties of the condensed state depend on the type of linker histone, particularly on the highly disordered C-terminal tail, which is the most variable region of the protein, both between species, and within the various subtypes and cell-type specific variants of a given organism. We have developed an in-vitro model system comprising linker histone tail and linker DNA, which although very minimal, displays surprisingly complex behaviour, and is sufficient to model the known states of linker-histone-condensed chromatin: disordered \"fuzzy\" complexes (\"open\" chromatin), dense liquid-like assemblies (dynamic condensates) and higher-order structures (organised 30-nm fibres). A crucial advantage of such a simple model is that it allows the study of the various condensed states by NMR, CD and scattering methods. Moreover, it allows capture of the thermodynamics underpinning the transitions between states through calorimetry. We have leveraged this to rationalise the distinct condensing properties of linker histone subtypes and variants across species that are encoded by the amino acid content of their C-terminal tails. Three properties emerge as key to defining the condensed state: charge density, lysine/arginine ratio, and proline-free regions, and we evaluate each separately using a strategic mutagenesis approach.\n",
  "wordCount" : "2351",
  "inLanguage": "en",
  "datePublished": "2023-11-26T10:37:33Z",
  "dateModified": "2023-11-26T10:37:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 26, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.567962">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.567962" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.567962">
        <p class="paperTitle">Real-Time Structural Biology of DNA and DNA-Protein Complexes on an Optical Microscope</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.567962" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.567962" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Szalai, A. M.; Ferrari, G.; Richter, L.; Hartmann, J.; Kesici, M.-Z.; Ji, B.; Koshic, K.; Jaeger, A.; Aksimentiev, A.; Tessmer, I.; Kaminska, I.; Vera, A. M.; Tinnefeld, P.</p>
        <p class="info">Score: 40.7, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.567962' target='https://doi.org/10.1101/2023.11.21.567962'> 10.1101/2023.11.21.567962</a></p>
        <p class="abstract">The intricate interplay between DNA and proteins is key for biological functions such as DNA replication, transcription, and repair. To better understand these interactions, it is crucial to develop tools to study DNA-protein complexes with high spatial and temporal resolution. Here, we use the vertical orientation that DNA adopts on graphene and investigate its interactions with proteins via energy transfer from a probe dye to graphene, achieving spatial resolution down to the [A]ngstrom scale. We measured the bending angle of DNA induced by adenine tracts, bulges, abasic sites and the binding of Escherichia coli endonuclease IV with unprecedented precision and millisecond time resolution. Additionally, we observed the translocation of the O6-alkylguanine DNA alkyltransferase along double-stranded DNA, reaching single-base pair resolution and detecting an affinity for adenine tracts. Overall, we foresee that this method will become a widespread tool for the dynamical study of nucleic acid and nucleic acid-protein interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566032">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566032" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566032">
        <p class="paperTitle">Connecting Chromatin Structures to Gene Regulation Using Dynamic Polymer Simulations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566032" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566032" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fu, Y.; Clark, F. T.; Nomikou, S.; Tsirigos, A.; Lionnet, T.</p>
        <p class="info">Score: 38.5, Published: 2023-11-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566032' target='https://doi.org/10.1101/2023.11.07.566032'> 10.1101/2023.11.07.566032</a></p>
        <p class="abstract">The transfer of regulatory information between distal loci on chromatin is thought to involve physical proximity, but key biophysical features of these contacts remain unclear. For instance, it is unknown how close and for how long two loci need to be in order to productively interact. The main challenge is that it is currently impossible to measure chromatin dynamics with high spatiotemporal resolution at scale. Polymer simulations provide an accessible and rigorous way to test biophysical models of chromatin regulation, yet there is a lack of simple and general methods for extracting the values of model parameters. Here we adapt the Nelder-Mead simplex optimization algorithm to select the best polymer model matching a given Hi-C dataset, using the MYC locus as an example. The models biophysical parameters predict a compartmental rearrangement of the MYC locus in leukemia, which we validate with single-cell measurements. Leveraging trajectories predicted by the model, we find that loci with similar Hi-C contact frequencies can exhibit widely different contact dynamics. Interestingly, the frequency of productive interactions between loci exhibits a non-linear relationship with their Hi-C contact frequency when we enforce a specific capture radius and contact duration. These observations are consistent with recent experimental observations and suggest that the dynamic ensemble of chromatin configurations, rather than average contact matrices, is required to fully predict long-range chromatin interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568002">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568002" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568002">
        <p class="paperTitle">Printing, Characterising, and Assessing Transparent 3D Printed Lenses for Optical Imaging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568002" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568002" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rooney, L. M.; Christopher, J.; Watson, B.; Susir Kumar, Y.; Copeland, L.; Walker, L. D.; Foylan, S.; Amos, W. B.; Bauer, R.; McConnell, G.</p>
        <p class="info">Score: 35.4, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568002' target='https://doi.org/10.1101/2023.11.22.568002'> 10.1101/2023.11.22.568002</a></p>
        <p class="abstract">The production of high-quality lenses for optical instrumentation has involved subtractive manufacturing methods for centuries. These methods have demanded specialist equipment and expertise that often render custom high-grade bulk glass optics inaccessible. We aimed to develop a low-cost, accessible, and reproducible method using consumer-grade technology to manufacture high-quality three-dimensional (3D) printed lenses with comparable performance to glass lenses. Various 3D printed planoconvex lenses were produced using a consumer grade 3D printer and low-cost spin coating setup, and printed lenses were compared to their commercial glass counterparts. A range of mechanical and optical methods are introduced to determine the surface curvature of 3D printed lenses, providing a wide range of quality control methods. Amongst others, high-resolution interference reflection microscopy methods were used to reconstruct the convex surface of printed lenses and quantify their radius of curvature. Moreover, the optical throughput and focusing performance of 3D printed lenses were assessed using optical transmissivity measurements and classical beam characterisation methods. We determined that all 3D printed lens prescriptions tested had comparable surface curvature and optical performance to commercial glass lenses. Finally, we demonstrated the application of 3D printed lenses for brightfield transmission microscopy, resolving sub-cellular structures over a 2.3 mm field of view. The high reproducibility and comparable performance of 3D printed lenses present a great opportunity for additive manufacturing in the production of bespoke optical elements for low-cost rapid prototyping and improved accessibility to high-quality optics in low-resource settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567865">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567865" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567865">
        <p class="paperTitle">Substrate accessibility regulation of human TopIIa decatenation by cohesin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567865" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567865" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cutts, E.; Saravanan, S.; Fisher, G.; Rueda, D. S.; Aragon, L.</p>
        <p class="info">Score: 16.8, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567865' target='https://doi.org/10.1101/2023.11.20.567865'> 10.1101/2023.11.20.567865</a></p>
        <p class="abstract">Human topoisomerase II alpha (TOP2) is the main mitotic decatenase 1-3 resolving intertwines between sister chromatids that form during DNA replication 4. Here, we employ quadruple-trap optical tweezers to generate braids between a pair of {lambda}-DNA molecules to study, at the single-molecule level and in real-time, how TOP2 untangles these DNAs. We find that TOP2 rapidly resolves single and multiple braids and is inhibited by chemotherapy agent, etoposide. TOP2 is sensitive to DNA conformation, exhibiting a chiral preference for the removal of braids with right-handed crossings and is inhibited when DNAs are held at forces over 20pN. We show that TOP2 must load at the cross between the two DNAs to resolve braids efficiently. TOP2 pre-loaded onto individual DNAs is unable to resolve newly formed braids in the presence of ATP, suggesting that the geometry of pre-bound enzyme is incompatible with the capture of the second DNA. Finally, we show that cohesin binds to braids, preventing TOP2 from resolving them. Our study unveils novel insights into the regulation of TOP2s decatenation, underscoring the importance of substrate accessibility and the role of cohesin in modulating TOP2 activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568405">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568405" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568405">
        <p class="paperTitle">NanoLocz: Image analysis platform for AFM, high-speed AFM and localization AFM</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568405" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568405" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Heath, G. R.; Micklethwaite, E.; Storer, T. M.</p>
        <p class="info">Score: 13.7, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568405' target='https://doi.org/10.1101/2023.11.23.568405'> 10.1101/2023.11.23.568405</a></p>
        <p class="abstract">NanoLocz is an open-source computer program designed for high-throughput automatic processing and single-particle analysis of Atomic Force Microscopy (AFM) image data. High-Speed AFM and Localization AFM (LAFM) enable the study of single molecules with increasingly higher spatiotemporal resolution. However, efficient and rapid analysis of the images and movies produced by these techniques can be challenging, often requiring the use of multiple image processing software applications and scripts. Here, we introduce NanoLocz, an AFM and high-speed AFM analysis program that facilitates various single-particle analysis workflows through a simple, interactive interface. Workflows include but are not limited to: single-particle tracking, single-particle topographic feature analysis, single-molecule LAFM, time-resolved LAFM, and simulation LAFM. The source code and installation instructions for NanoLocz are available at https://github.com/George-R-Heath/NanoLocz.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.18.567697">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.18.567697" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.18.567697">
        <p class="paperTitle">Accurate Characterization of Conformational Ensembles and Binding Mechanisms of the SARS-CoV-2 Omicron BA.2 and BA.2.86 Spike Protein with the Host Receptor and Distinct Classes of Antibodies Using AlphaFold2-Augmented Integrative Computational Modeling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.18.567697" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.18.567697" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Raisinghani, N.; Alshahrani, M.; Gupta, G.; Xiao, S.; Tao, P.; Verkhivker, G.</p>
        <p class="info">Score: 13.3, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.18.567697' target='https://doi.org/10.1101/2023.11.18.567697'> 10.1101/2023.11.18.567697</a></p>
        <p class="abstract">The latest wave SARS-CoV-2 Omicron variants displayed a growth advantage and the increased viral fitness through convergent evolution of functional hotspots that work synchronously to balance fitness requirements for productive receptor binding and efficient immune evasion. In this study, we combined AlphaFold2-based structural modeling approaches with all-atom MD simulations and mutational profiling of binding energetics and stability for prediction and comprehensive analysis of the structure, dynamics, and binding of the SARS-CoV-2 Omicron BA.2.86 spike variant with ACE2 host receptor and distinct classes of antibodies. We adapted several AlphaFold2 approaches to predict both structure and conformational ensembles of the Omicron BA.2.86 spike protein in the complex with the host receptor. The results showed that AlphaFold2-predicted conformational ensemble of the BA.2.86 spike protein complex can accurately capture the main dynamics signatures obtained from microscond molecular dynamics simulations. The ensemble-based dynamic mutational scanning of the receptor binding domain residues in the BA.2 and BA.2.86 spike complexes with ACE2 dissected the role of the BA.2 and BA.2.86 backgrounds in modulating binding free energy changes revealing a group of conserved hydrophobic hotspots and critical variant-specific contributions of the BA.2.86 mutational sites R403K, F486P and R493Q. To examine immune evasion properties of BA.2.86 in atomistic detail, we performed large scale structure-based mutational profiling of the S protein binding interfaces with distinct classes of antibodies that displayed significantly reduced neutralization against BA.2.86 variant. The results quantified specific function of the BA.2.86 mutations to ensure broad resistance against different classes of RBD antibodies. This study revealed the molecular basis of compensatory functional effects of the binding hotspots, showing that BA.2.86 lineage may have primarily evolved to improve immune escape while modulating binding affinity with ACE2 through cooperative effect of R403K, F486P and R493Q mutations. The study supports a hypothesis that the impact of the increased ACE2 binding affinity on viral fitness is more universal and is mediated through cross-talk between convergent mutational hotspots, while the effect of immune evasion could be more variant-dependent.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567747">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567747" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567747">
        <p class="paperTitle">Protein Stability Prediction by Fine-tuning a Protein Language Model on a Mega-scale Dataset</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567747" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567747" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chu, K. S.; Siegel, J. B.</p>
        <p class="info">Score: 12.2, Published: 2023-11-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567747' target='https://doi.org/10.1101/2023.11.19.567747'> 10.1101/2023.11.19.567747</a></p>
        <p class="abstract">The stability of a protein is crucial to its utility in industrial applications. While engineering campaigns can now be routinely used to enhance protein thermal stability to the level needed in an industrial setting, there is a significant desire to fast-track these efforts through predictive tools allowing one to jump in a minimal number of design iterations to a highly stabilized protein. In this work, we explore utilizing a mega-scale dataset for development of a protein language model tuned for stability. This model is trained on the folding stability of 528k sequences derived from 461 small protein domains and designs, and can accommodate deletions, insertions, and multiple-point mutations. We show that a protein language model can be fine-tuned to predict folding stability. The fine-tuned protein language model, named ESMtherm, performs reasonably on small protein domains and generalizes to sequences distal from the training set. Lastly, we discuss its limitations when compared to other state-of-the-art methods in generalizing to larger protein scaffolds and highlight the need of large-scale stability measurement on a diverse dataset that represents the distribution of sequence lengths commonly observed in nature.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567265" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567265">
        <p class="paperTitle">The mechanism of mRNA activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gentry, R. C.; Ide, N. A.; Comunale, V. M.; Hartwick, E. W.; Kinz-Thompson, C. D.; Gonzalez, R. L.</p>
        <p class="info">Score: 32.9, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567265' target='https://doi.org/10.1101/2023.11.15.567265'> 10.1101/2023.11.15.567265</a></p>
        <p class="abstract">During translation initiation, messenger RNA molecules must be identified and activated for loading into a ribosome. In this rate-limiting step, the heterotrimeric protein eukaryotic initiation factor eIF4F must recognize and productively interact with the 7-methylguanosine cap at the 5 end of the messenger RNA and subsequently activate the message. Despite its fundamental, regulatory role in gene expression, the molecular events underlying cap recognition and messenger RNA activation remain mysterious. Here, we generate a unique, single-molecule fluorescence imaging system to interrogate the dynamics with which eIF4F discriminates productive and non-productive locations on full-length, native messenger RNA molecules. At the single-molecule level, we observe stochastic sampling of eIF4F along the length of the messenger RNA and identify allosteric communication between the eIF4F subunits which ultimately drive cap-recognition and subsequent activation of the message. Our experiments uncover novel functions for each subunit of eIF4F and we conclude by presenting a model for messenger RNA activation which precisely defines the composition of the activated message. This model provides a general framework for understanding how messenger RNA molecules may be discriminated from one another, and how other RNA-binding proteins may control the efficiency of translation initiation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568227">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568227" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568227">
        <p class="paperTitle">Low-cost 3D printed lenses for brightfield and fluorescence microscopy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568227" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568227" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Christopher, J.; Rooney, L. M.; Donnachie, M.; Uttamchandani, D.; McConnell, G.; Bauer, R.</p>
        <p class="info">Score: 10.2, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568227' target='https://doi.org/10.1101/2023.11.22.568227'> 10.1101/2023.11.22.568227</a></p>
        <p class="abstract">We present the fabrication and implementation of low-cost optical quality 3D printed lenses, and their application as microscope objectives with different prescriptions. The imaging performance of the 3D printed lenses was benchmarked against commercially available optics including a 20 mm focal length 12.7 mm diameter NBK-7 plano-convex lens used as a low magnification objective, and a separate high magnification objective featuring three 6 mm diameter NBK-7 lenses with different positive and negative focal lengths. We describe the design and manufacturing processes to produce high-quality 3D printed lenses. We tested their transmissivity and surface quality using a stylus profilometer, showing that they conform to that of commercial glass counterpart lenses. The 3D printed lenses were used as microscope objectives in both brightfield and epi-fluorescence imaging of specimens including onion, cyanobacteria, and variegated Hosta leaves, demonstrating a sub-cellular resolution performance obtained with low-cost 3D printed optical elements within brightfield and fluorescence microscopy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567813">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567813" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567813">
        <p class="paperTitle">A DNA condensation code for linker histones</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567813" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567813" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Watson, M.; Sabirova, D.; Hardy, M. C.; Pan, Y.; Yates, H.; Wright, C. J.; Chan, W. H.; Destan, E.; Stott, K.</p>
        <p class="info">Score: 9.6, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567813' target='https://doi.org/10.1101/2023.11.20.567813'> 10.1101/2023.11.20.567813</a></p>
        <p class="abstract">Linker histones play an essential role in chromatin packaging by facilitating compaction of the 11-nm fibre of nucleosomal &#34;beads on a string&#34;. The result is a heterogeneous condensed state with local properties that range from dynamic, irregular and liquid-like, to stable and regular structures (the 30-nm fibre), which in turn impact chromatin-dependent activities at a fundamental level. The properties of the condensed state depend on the type of linker histone, particularly on the highly disordered C-terminal tail, which is the most variable region of the protein, both between species, and within the various subtypes and cell-type specific variants of a given organism. We have developed an in-vitro model system comprising linker histone tail and linker DNA, which although very minimal, displays surprisingly complex behaviour, and is sufficient to model the known states of linker-histone-condensed chromatin: disordered &#34;fuzzy&#34; complexes (&#34;open&#34; chromatin), dense liquid-like assemblies (dynamic condensates) and higher-order structures (organised 30-nm fibres). A crucial advantage of such a simple model is that it allows the study of the various condensed states by NMR, CD and scattering methods. Moreover, it allows capture of the thermodynamics underpinning the transitions between states through calorimetry. We have leveraged this to rationalise the distinct condensing properties of linker histone subtypes and variants across species that are encoded by the amino acid content of their C-terminal tails. Three properties emerge as key to defining the condensed state: charge density, lysine/arginine ratio, and proline-free regions, and we evaluate each separately using a strategic mutagenesis approach.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
