<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="CryoSTAR: Leveraging Structural Prior and Constraints for Cryo-EM Heterogeneous Reconstruction
Authors: Li, Y.; Zhou, Y.; Yuan, J.; Ye, F.; Gu, Q.
Score: 51.3, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564872
Resolving conformational heterogeneity in cryo-electron microscopy (cryo-EM) datasets remains a significant challenge in structural biology. Previous methods have often been restricted to working exclusively on volumetric densities, neglecting the potential of incorporating any pre-existing structural knowledge as prior or constraints. In this paper, we present a novel methodology, cryoSTAR, that harnesses atomic model information as structural regularization to elucidate such heterogeneity.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="CryoSTAR: Leveraging Structural Prior and Constraints for Cryo-EM Heterogeneous Reconstruction
Authors: Li, Y.; Zhou, Y.; Yuan, J.; Ye, F.; Gu, Q.
Score: 51.3, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564872
Resolving conformational heterogeneity in cryo-electron microscopy (cryo-EM) datasets remains a significant challenge in structural biology. Previous methods have often been restricted to working exclusively on volumetric densities, neglecting the potential of incorporating any pre-existing structural knowledge as prior or constraints. In this paper, we present a novel methodology, cryoSTAR, that harnesses atomic model information as structural regularization to elucidate such heterogeneity." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-12T10:37:43+00:00" />
<meta property="article:modified_time" content="2023-11-12T10:37:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="CryoSTAR: Leveraging Structural Prior and Constraints for Cryo-EM Heterogeneous Reconstruction
Authors: Li, Y.; Zhou, Y.; Yuan, J.; Ye, F.; Gu, Q.
Score: 51.3, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564872
Resolving conformational heterogeneity in cryo-electron microscopy (cryo-EM) datasets remains a significant challenge in structural biology. Previous methods have often been restricted to working exclusively on volumetric densities, neglecting the potential of incorporating any pre-existing structural knowledge as prior or constraints. In this paper, we present a novel methodology, cryoSTAR, that harnesses atomic model information as structural regularization to elucidate such heterogeneity."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "CryoSTAR: Leveraging Structural Prior and Constraints for Cryo-EM Heterogeneous Reconstruction\nAuthors: Li, Y.; Zhou, Y.; Yuan, J.; Ye, F.; Gu, Q.\nScore: 51.3, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564872\nResolving conformational heterogeneity in cryo-electron microscopy (cryo-EM) datasets remains a significant challenge in structural biology. Previous methods have often been restricted to working exclusively on volumetric densities, neglecting the potential of incorporating any pre-existing structural knowledge as prior or constraints. In this paper, we present a novel methodology, cryoSTAR, that harnesses atomic model information as structural regularization to elucidate such heterogeneity.",
  "keywords": [
    
  ],
  "articleBody": " CryoSTAR: Leveraging Structural Prior and Constraints for Cryo-EM Heterogeneous Reconstruction\nAuthors: Li, Y.; Zhou, Y.; Yuan, J.; Ye, F.; Gu, Q.\nScore: 51.3, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564872\nResolving conformational heterogeneity in cryo-electron microscopy (cryo-EM) datasets remains a significant challenge in structural biology. Previous methods have often been restricted to working exclusively on volumetric densities, neglecting the potential of incorporating any pre-existing structural knowledge as prior or constraints. In this paper, we present a novel methodology, cryoSTAR, that harnesses atomic model information as structural regularization to elucidate such heterogeneity. Our method uniquely outputs both coarse-grained models and density maps, showcasing the molecular conformational changes at different levels. Validated against four diverse experimental datasets, spanning large complexes, a membrane protein, and a small single-chain protein, our results consistently demonstrate an efficient and effective solution to conformational heterogeneity with minimal human bias. By integrating atomic model insights with cryo-EM data, cryoSTAR represents a meaningful step forward, paving the way for a deeper understanding of dynamic biological processes.1\nCryo-EM structure of Slo1 with the auxiliary γ1 subunit suggests mechanism of depolarization-independent activation\nAuthors: Redhardt, M.; Raunser, S.; Raisch, T.\nScore: 28.0, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565197\nMammalian Ca2+-dependent Slo K+ channels are expressed with {beta} and {gamma} auxiliary subunits that greatly influence voltage- and Ca2+-induced gating, thereby fundamentally altering the behavior of the channel. The four {gamma} subunits reduce the need for voltage-dependent activation, allowing Slo to open in the absence of an action potential. The mechanism of this activation has, however, remained elusive. Here, we present the cryo-EM structure of Slo1 in complex with {gamma}1/LRRC26, revealing how the transmembrane helix of {gamma}1 binds and presumably stabilizes the active conformation of the voltage-sensor domain. This effect is further enhanced by a polybasic stretch on the intracellular side of the membrane which locally changes the charge gradient across the membrane. Sequence differences explain why the four {gamma} subunits possess different activation efficiencies. Simultaneous binding of {gamma} and the unrelated {beta} subunits is structurally possible, as both binding sites do not overlap and the {gamma}1 LRR domains are partially flexible. Thus, our data provide a possible explanation for Slo1 regulation by {gamma} subunits, and furthermore suggest a novel mechanism of activation of voltage-gated ion channels by auxiliary subunits and add to the growing knowledge of their complex regulation.\n4Pi MINFLUX arrangement maximizes spatio-temporal localization precision of fluorescence emitter\nAuthors: Rickert, J.; Held, M. O.; Engelhardt, J.; Hell, S. W.\nScore: 12.4, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564755\nWe introduce MINFLUX localization with interferometric illumination through opposing objective lenses for maximizing the attainable precision in 3D-localization of single inelastic scatterers, such as fluorophores. Our 4Pi optical configuration employs three sequentially tilted counter-propagating beam pairs for illumination, each providing a narrow interference minimum of illumination intensity at the focal point. The localization precision is additionally improved by adding the inelastically scattered (fluorescence) photons collected through both objective lenses. Our 4Pi configuration yields the currently highest precision per detected photon among all localization schemes. Tracking gold nanoparticles as non-blinking inelastic scatterers rendered a position uncertainty \u003c 0.4 nm3 in volume at a localization frequency of 2.9 kHz. We harnessed the record spatio-temporal precision of our 4Pi MINFLUX approach to examine the diffusion of single fluorophores and fluorescent nanobeads in solutions of sucrose in water, revealing local heterogeneities at the nanoscale. Our results show the applicability of 4Pi MINFLUX to study molecular nano-environments of diffusion and its potential for quantifying rapid movements of molecules in cells and other material composites.\nEngineering Modular and Tunable Single Molecule Sensors by Decoupling Sensing from Signal Output\nAuthors: Grabenhorst, L.; Pfeiffer, M.; Schinkel, T.; Kummerlin, M.; Maglic, J. B.; Bruggenthies, G. A.; Selbach, F.; Murr, A. T.; Tinnefeld, P.; Glembockyte, V.\nScore: 9.6, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565795\nBiosensors play key roles in medical research and diagnostics, but there currently is a lack of sensing platforms that combine easy adaptation to new targets, strategies to tune the response window to relevant analyte concentration ranges and allow for the incorporation of multiple sensing elements to benefit from multivalency. Utilizing a DNA origami nanostructure as a scaffold for arranging the different sensor components, we here propose an approach for the development of modular and tunable single-molecule sensors capable of detecting a variety of biomolecular targets such as nucleic acids, antibodies and restriction enzymes while offering mechanisms to tune the dynamic window, the specificity and the cooperativity of the sensor.\nRegulation of chromatin transcription dynamics by DNA supercoiling\nAuthors: Brahmachari, S.; Tripathi, S.; Onuchic, J. N.; Levine, H.\nScore: 4.3, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565891\nTranscription has a mechanical component, as the translocation of the transcription machinery or RNA polymerase (RNAP) on DNA or chromatin is dynamically coupled to the chromatin torsion. This posits chromatin mechanics as a possible regulator of eukaryotic transcription, however, the modes and mechanisms of this regulation are elusive. Here, we first take a statistical mechanics approach to model the torsional response of topology-constrained chromatin. Our model recapitulates the experimentally observed weaker torsional rigidity of chromatin compared to bare DNA, and proposes structural transitions of nucleosomes into chirally distinct states as the driver of the contrasting torsional mechanics. Coupling chromatin mechanics with RNAP translocation in stochastic simulations, we reveal a complex interplay of DNA supercoiling and nucleosome dynamics in governing RNAP velocity. Nucleosomes play a dual role in controlling the transcription dynamics. The steric barrier aspect of nucleosomes in the gene body counteracts transcription via hindering RNAP motion, whereas the chiral transitions facilitate RNAP motion via driving a low restoring torque upon twisting the DNA. While nucleosomes with low dissociation rates are typically transcriptionally repressive, highly dynamic nucleosomes offer less of a steric barrier and enhance the transcription elongation dynamics of weakly transcribed genes via buffering DNA twist. We use the model to predict transcription-dependent levels of DNA supercoiling in segments of the budding yeast genome that are in accord with available experimental data. The model unveils a paradigm of DNA supercoiling-mediated interaction between genes and makes testable predictions that will guide experimental design.\nRescuing Off-Equilibrium Simulation Data through Dynamic Experimental Data with dynAMMo\nAuthors: Kolloff, C.; Olsson, S.\nScore: 12.3, Published: 2023-11-08 DOI: 10.1101/2023.05.23.541878\nLong-timescale behavior of proteins is fundamental to many biological processes. Molecular Dynamics (MD) simulations and biophysical experiments are often used to study protein dynamics. However, high computational demands of MD limit what timescales are feasible to study, often missing rare events, which are critical to explain experiments. On the other hand, experiments are limited by low resolution. We present dynamic Augmented Markov models (dynAMMo) to bridge the gap between these data and overcome their respective limitations. For the first time, dynAMMo enables the construction of mechanistic models of slow exchange processes that have been not observed in MD data by integrating dynamic experimental observables. As a consequence, dynAMMo allows us to bypass costly and extensive simulations, yet providing mechanistic insights of the system. Validated with controlled model systems and a well-studied protein, dynAMMo offers a new approach to quantitatively model protein dynamics on long timescales in an unprecedented manner.\nFormation and growth of co-culture tumour spheroids: new compartment-based mathematical models and experiments\nAuthors: Murphy, R. J.; Gunasingh, G.; Haass, N. K.; Simpson, M. J.\nScore: 14.3, Published: 2023-10-23 DOI: 10.1101/2022.12.21.521515\nCo-culture tumour spheroid experiments are routinely performed to investigate cancer progression and test anti-cancer therapies. Therefore, methods to quantitatively characterise and interpret coculture spheroid growth are of great interest. However, co-culture spheroid growth is complex. Multiple biological processes occur on overlapping timescales and different cell types within the spheroid may have different characteristics, such as differing proliferation rates or responses to nutrient availability. At present there is no standard, widely-accepted mathematical model of such complex spatio-temporal growth processes. Typical approaches to analyse these experiments focus on the late-time temporal evolution of spheroid size and overlook early-time spheroid formation, spheroid structure and geometry. Here, using a range of ordinary differential equation-based mathematical models and parameter estimation, we interpret new co-culture experimental data. We provide new biological insights about spheroid formation, growth, and structure. As part of this analysis we connect Greenspans seminal mathematical model to co-culture data for the first time. Furthermore, we generalise a class of compartment-based spheroid mathematical models that have previously been restricted to one population so they can be applied to multiple populations. As special cases of the general model, we explore multiple natural two population extensions to Greenspans seminal model and reveal biological mechanisms that can describe the internal dynamics of growing co-culture spheroids and those that cannot. This mathematical and statistical modelling-based framework is well-suited to analyse spheroids grown with multiple different cell types and the new class of mathematical models provide opportunities for further mathematical and biological insights.\nInterfacial morphodynamics of proliferating microbial communities\nAuthors: Martinez-Calvo, A.; Trenado-Yuste, C.; Lee, H.; Gore, J.; Wingreen, N. S.; Datta, S. S.\nScore: 37.2, Published: 2023-10-24 DOI: 10.1101/2023.10.23.563665\nIn microbial communities, various cell types often coexist by occupying distinct spatial domains. What determines the shape of the interface between such domains--which in turn influences the interactions between cells and overall community function? Here, we address this question by developing a continuum model of a 2D spatially-structured microbial community with two distinct cell types. We find that, depending on the balance of the different cell proliferation rates and substrate friction coefficients, the interface between domains is either stable and smooth, or unstable and develops finger-like protrusions. We establish quantitative principles describing when these different interfacial behaviors arise, and find good agreement both with the results of previous experimental reports as well as new experiments performed here. Our work thus helps to provide a biophysical basis for understanding the interfacial morphodynamics of proliferating microbial communities, as well as a broader range of proliferating active systems.\nNuclear deformation and dynamics of migrating cells in 3D confinement reveal adaptation of pulling and pushing forces\nAuthors: Stoeberl, S.; Flommersfeld, J.; Kreft, M.; Benoit, M.; Broedersz, C.; Raedler, J. O.\nScore: 7.8, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564765\nEukaryotic cells show an astounding ability to migrate through pores and constrictions smaller than their nuclear diameter. However, the forces engaged in nuclear deformation and their effect on confined cell dynamics remain unclear. Here, we study the mechanics and dynamics of nuclei of mesenchymal cancer cells as they spontaneously and repeatedly transition through 3D compliant hydrogel channels. We find a biphasic dependence of migration speed and transition frequency on channel width, revealing maximal transition rates at widths comparable to the nuclear diameter. Using confocal imaging and hydrogel bead displacement, we determine the nuclear deformation and corresponding forces during spontaneous confined migration. We find the nucleus to reversibly deform with an elastic modulus not adapting to the confinement. Instead, with decreasing channel width, the nuclear shape during transmigration changes biphasically concomitant with the transitioning dynamics. The nucleus exhibits a prolate form along the migration direction in wide channels and a more compressed oblate shape in narrow channels. We propose a physical model for confined cell migration that explains the observed nuclear shapes and slowing down in terms of the cytoskeletal force-generation adapting from a pulling-to a pushing-dominated mechanism with increasing nuclear confinement.\nA family of bacterial actin homologues forms a 3-stranded tubular structure\nAuthors: Bergeron, J.; Kollman, J. M.\nScore: 2.5, Published: 2023-11-09 DOI: 10.1101/2023.11.07.565980\nThe cytoskeleton plays a critical role in the organization and movement of cells. In Eukaryotes, actin filaments polymerize into a highly conserved double-stranded linear filamentous structure in the presence of ATP, and disassemble upon ATP hydrolysis. Bacteria also possess actin-like proteins, that drive fundamental cellular function, including cell division, shape maintenance, and DNA segregation. Like eukaryotic actin, bacterial actins assemble dynamic on ATP binding. In general, longitudinal interactions between bacterial actin protomers along each strand are conserved with eukaryotic actin, but variation in interactions between strands gives rise to striking diversity of filament architectures. Here, we report a family of bacterial actins of unknown function, conserved amongst the Verrucomicrobiota phylum, which assembles into a unique tubular structure in the presence of nucleotides. A cryo-EM structure of the filaments reveals it is composed of three strands, unlike other described bacterial actin structures. This architecture provides new insights into the organization of actin-like filaments, and has implications for understanding the diversity and evolution of the bacterial cytoskeleton.\n",
  "wordCount" : "2029",
  "inLanguage": "en",
  "datePublished": "2023-11-12T10:37:43Z",
  "dateModified": "2023-11-12T10:37:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 12, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564872">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564872" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564872">
        <p class="paperTitle">CryoSTAR: Leveraging Structural Prior and Constraints for Cryo-EM Heterogeneous Reconstruction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564872" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564872" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Y.; Zhou, Y.; Yuan, J.; Ye, F.; Gu, Q.</p>
        <p class="info">Score: 51.3, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564872' target='https://doi.org/10.1101/2023.10.31.564872'> 10.1101/2023.10.31.564872</a></p>
        <p class="abstract">Resolving conformational heterogeneity in cryo-electron microscopy (cryo-EM) datasets remains a significant challenge in structural biology. Previous methods have often been restricted to working exclusively on volumetric densities, neglecting the potential of incorporating any pre-existing structural knowledge as prior or constraints. In this paper, we present a novel methodology, cryoSTAR, that harnesses atomic model information as structural regularization to elucidate such heterogeneity. Our method uniquely outputs both coarse-grained models and density maps, showcasing the molecular conformational changes at different levels. Validated against four diverse experimental datasets, spanning large complexes, a membrane protein, and a small single-chain protein, our results consistently demonstrate an efficient and effective solution to conformational heterogeneity with minimal human bias. By integrating atomic model insights with cryo-EM data, cryoSTAR represents a meaningful step forward, paving the way for a deeper understanding of dynamic biological processes.1</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565197">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565197" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565197">
        <p class="paperTitle">Cryo-EM structure of Slo1 with the auxiliary γ1 subunit suggests mechanism of depolarization-independent activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565197" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565197" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Redhardt, M.; Raunser, S.; Raisch, T.</p>
        <p class="info">Score: 28.0, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565197' target='https://doi.org/10.1101/2023.11.01.565197'> 10.1101/2023.11.01.565197</a></p>
        <p class="abstract">Mammalian Ca2&#43;-dependent Slo K&#43; channels are expressed with {beta} and {gamma} auxiliary subunits that greatly influence voltage- and Ca2&#43;-induced gating, thereby fundamentally altering the behavior of the channel. The four {gamma} subunits reduce the need for voltage-dependent activation, allowing Slo to open in the absence of an action potential. The mechanism of this activation has, however, remained elusive. Here, we present the cryo-EM structure of Slo1 in complex with {gamma}1/LRRC26, revealing how the transmembrane helix of {gamma}1 binds and presumably stabilizes the active conformation of the voltage-sensor domain. This effect is further enhanced by a polybasic stretch on the intracellular side of the membrane which locally changes the charge gradient across the membrane. Sequence differences explain why the four {gamma} subunits possess different activation efficiencies. Simultaneous binding of {gamma} and the unrelated {beta} subunits is structurally possible, as both binding sites do not overlap and the {gamma}1 LRR domains are partially flexible. Thus, our data provide a possible explanation for Slo1 regulation by {gamma} subunits, and furthermore suggest a novel mechanism of activation of voltage-gated ion channels by auxiliary subunits and add to the growing knowledge of their complex regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564755">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564755" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564755">
        <p class="paperTitle">4Pi MINFLUX arrangement maximizes spatio-temporal localization precision of fluorescence emitter</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564755" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564755" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rickert, J.; Held, M. O.; Engelhardt, J.; Hell, S. W.</p>
        <p class="info">Score: 12.4, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564755' target='https://doi.org/10.1101/2023.10.30.564755'> 10.1101/2023.10.30.564755</a></p>
        <p class="abstract">We introduce MINFLUX localization with interferometric illumination through opposing objective lenses for maximizing the attainable precision in 3D-localization of single inelastic scatterers, such as fluorophores. Our 4Pi optical configuration employs three sequentially tilted counter-propagating beam pairs for illumination, each providing a narrow interference minimum of illumination intensity at the focal point. The localization precision is additionally improved by adding the inelastically scattered (fluorescence) photons collected through both objective lenses. Our 4Pi configuration yields the currently highest precision per detected photon among all localization schemes. Tracking gold nanoparticles as non-blinking inelastic scatterers rendered a position uncertainty &lt; 0.4 nm3 in volume at a localization frequency of 2.9 kHz. We harnessed the record spatio-temporal precision of our 4Pi MINFLUX approach to examine the diffusion of single fluorophores and fluorescent nanobeads in solutions of sucrose in water, revealing local heterogeneities at the nanoscale. Our results show the applicability of 4Pi MINFLUX to study molecular nano-environments of diffusion and its potential for quantifying rapid movements of molecules in cells and other material composites.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565795">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565795" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565795">
        <p class="paperTitle">Engineering Modular and Tunable Single Molecule Sensors by Decoupling Sensing from Signal Output</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565795" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565795" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grabenhorst, L.; Pfeiffer, M.; Schinkel, T.; Kummerlin, M.; Maglic, J. B.; Bruggenthies, G. A.; Selbach, F.; Murr, A. T.; Tinnefeld, P.; Glembockyte, V.</p>
        <p class="info">Score: 9.6, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565795' target='https://doi.org/10.1101/2023.11.06.565795'> 10.1101/2023.11.06.565795</a></p>
        <p class="abstract">Biosensors play key roles in medical research and diagnostics, but there currently is a lack of sensing platforms that combine easy adaptation to new targets, strategies to tune the response window to relevant analyte concentration ranges and allow for the incorporation of multiple sensing elements to benefit from multivalency. Utilizing a DNA origami nanostructure as a scaffold for arranging the different sensor components, we here propose an approach for the development of modular and tunable single-molecule sensors capable of detecting a variety of biomolecular targets such as nucleic acids, antibodies and restriction enzymes while offering mechanisms to tune the dynamic window, the specificity and the cooperativity of the sensor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565891">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565891" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565891">
        <p class="paperTitle">Regulation of chromatin transcription dynamics by DNA supercoiling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565891" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565891" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brahmachari, S.; Tripathi, S.; Onuchic, J. N.; Levine, H.</p>
        <p class="info">Score: 4.3, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565891' target='https://doi.org/10.1101/2023.11.06.565891'> 10.1101/2023.11.06.565891</a></p>
        <p class="abstract">Transcription has a mechanical component, as the translocation of the transcription machinery or RNA polymerase (RNAP) on DNA or chromatin is dynamically coupled to the chromatin torsion. This posits chromatin mechanics as a possible regulator of eukaryotic transcription, however, the modes and mechanisms of this regulation are elusive. Here, we first take a statistical mechanics approach to model the torsional response of topology-constrained chromatin. Our model recapitulates the experimentally observed weaker torsional rigidity of chromatin compared to bare DNA, and proposes structural transitions of nucleosomes into chirally distinct states as the driver of the contrasting torsional mechanics. Coupling chromatin mechanics with RNAP translocation in stochastic simulations, we reveal a complex interplay of DNA supercoiling and nucleosome dynamics in governing RNAP velocity. Nucleosomes play a dual role in controlling the transcription dynamics. The steric barrier aspect of nucleosomes in the gene body counteracts transcription via hindering RNAP motion, whereas the chiral transitions facilitate RNAP motion via driving a low restoring torque upon twisting the DNA. While nucleosomes with low dissociation rates are typically transcriptionally repressive, highly dynamic nucleosomes offer less of a steric barrier and enhance the transcription elongation dynamics of weakly transcribed genes via buffering DNA twist. We use the model to predict transcription-dependent levels of DNA supercoiling in segments of the budding yeast genome that are in accord with available experimental data. The model unveils a paradigm of DNA supercoiling-mediated interaction between genes and makes testable predictions that will guide experimental design.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.23.541878">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.23.541878" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.23.541878">
        <p class="paperTitle">Rescuing Off-Equilibrium Simulation Data through Dynamic Experimental Data with dynAMMo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.23.541878" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.23.541878" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kolloff, C.; Olsson, S.</p>
        <p class="info">Score: 12.3, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.23.541878' target='https://doi.org/10.1101/2023.05.23.541878'> 10.1101/2023.05.23.541878</a></p>
        <p class="abstract">Long-timescale behavior of proteins is fundamental to many biological processes. Molecular Dynamics (MD) simulations and biophysical experiments are often used to study protein dynamics. However, high computational demands of MD limit what timescales are feasible to study, often missing rare events, which are critical to explain experiments. On the other hand, experiments are limited by low resolution. We present dynamic Augmented Markov models (dynAMMo) to bridge the gap between these data and overcome their respective limitations. For the first time, dynAMMo enables the construction of mechanistic models of slow exchange processes that have been not observed in MD data by integrating dynamic experimental observables. As a consequence, dynAMMo allows us to bypass costly and extensive simulations, yet providing mechanistic insights of the system. Validated with controlled model systems and a well-studied protein, dynAMMo offers a new approach to quantitatively model protein dynamics on long timescales in an unprecedented manner.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.12.21.521515">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.12.21.521515" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.12.21.521515">
        <p class="paperTitle">Formation and growth of co-culture tumour spheroids: new compartment-based mathematical models and experiments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.12.21.521515" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.12.21.521515" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Murphy, R. J.; Gunasingh, G.; Haass, N. K.; Simpson, M. J.</p>
        <p class="info">Score: 14.3, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.12.21.521515' target='https://doi.org/10.1101/2022.12.21.521515'> 10.1101/2022.12.21.521515</a></p>
        <p class="abstract">Co-culture tumour spheroid experiments are routinely performed to investigate cancer progression and test anti-cancer therapies. Therefore, methods to quantitatively characterise and interpret coculture spheroid growth are of great interest. However, co-culture spheroid growth is complex. Multiple biological processes occur on overlapping timescales and different cell types within the spheroid may have different characteristics, such as differing proliferation rates or responses to nutrient availability. At present there is no standard, widely-accepted mathematical model of such complex spatio-temporal growth processes. Typical approaches to analyse these experiments focus on the late-time temporal evolution of spheroid size and overlook early-time spheroid formation, spheroid structure and geometry. Here, using a range of ordinary differential equation-based mathematical models and parameter estimation, we interpret new co-culture experimental data. We provide new biological insights about spheroid formation, growth, and structure. As part of this analysis we connect Greenspans seminal mathematical model to co-culture data for the first time. Furthermore, we generalise a class of compartment-based spheroid mathematical models that have previously been restricted to one population so they can be applied to multiple populations. As special cases of the general model, we explore multiple natural two population extensions to Greenspans seminal model and reveal biological mechanisms that can describe the internal dynamics of growing co-culture spheroids and those that cannot. This mathematical and statistical modelling-based framework is well-suited to analyse spheroids grown with multiple different cell types and the new class of mathematical models provide opportunities for further mathematical and biological insights.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563665">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563665" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563665">
        <p class="paperTitle">Interfacial morphodynamics of proliferating microbial communities</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563665" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563665" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez-Calvo, A.; Trenado-Yuste, C.; Lee, H.; Gore, J.; Wingreen, N. S.; Datta, S. S.</p>
        <p class="info">Score: 37.2, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563665' target='https://doi.org/10.1101/2023.10.23.563665'> 10.1101/2023.10.23.563665</a></p>
        <p class="abstract">In microbial communities, various cell types often coexist by occupying distinct spatial domains. What determines the shape of the interface between such domains--which in turn influences the interactions between cells and overall community function? Here, we address this question by developing a continuum model of a 2D spatially-structured microbial community with two distinct cell types. We find that, depending on the balance of the different cell proliferation rates and substrate friction coefficients, the interface between domains is either stable and smooth, or unstable and develops finger-like protrusions. We establish quantitative principles describing when these different interfacial behaviors arise, and find good agreement both with the results of previous experimental reports as well as new experiments performed here. Our work thus helps to provide a biophysical basis for understanding the interfacial morphodynamics of proliferating microbial communities, as well as a broader range of proliferating active systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564765" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564765">
        <p class="paperTitle">Nuclear deformation and dynamics of migrating cells in 3D confinement reveal adaptation of pulling and pushing forces</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stoeberl, S.; Flommersfeld, J.; Kreft, M.; Benoit, M.; Broedersz, C.; Raedler, J. O.</p>
        <p class="info">Score: 7.8, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564765' target='https://doi.org/10.1101/2023.10.30.564765'> 10.1101/2023.10.30.564765</a></p>
        <p class="abstract">Eukaryotic cells show an astounding ability to migrate through pores and constrictions smaller than their nuclear diameter. However, the forces engaged in nuclear deformation and their effect on confined cell dynamics remain unclear. Here, we study the mechanics and dynamics of nuclei of mesenchymal cancer cells as they spontaneously and repeatedly transition through 3D compliant hydrogel channels. We find a biphasic dependence of migration speed and transition frequency on channel width, revealing maximal transition rates at widths comparable to the nuclear diameter. Using confocal imaging and hydrogel bead displacement, we determine the nuclear deformation and corresponding forces during spontaneous confined migration. We find the nucleus to reversibly deform with an elastic modulus not adapting to the confinement. Instead, with decreasing channel width, the nuclear shape during transmigration changes biphasically concomitant with the transitioning dynamics. The nucleus exhibits a prolate form along the migration direction in wide channels and a more compressed oblate shape in narrow channels. We propose a physical model for confined cell migration that explains the observed nuclear shapes and slowing down in terms of the cytoskeletal force-generation adapting from a pulling-to a pushing-dominated mechanism with increasing nuclear confinement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565980">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565980" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565980">
        <p class="paperTitle">A family of bacterial actin homologues forms a 3-stranded tubular structure</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565980" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565980" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bergeron, J.; Kollman, J. M.</p>
        <p class="info">Score: 2.5, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565980' target='https://doi.org/10.1101/2023.11.07.565980'> 10.1101/2023.11.07.565980</a></p>
        <p class="abstract">The cytoskeleton plays a critical role in the organization and movement of cells. In Eukaryotes, actin filaments polymerize into a highly conserved double-stranded linear filamentous structure in the presence of ATP, and disassemble upon ATP hydrolysis. Bacteria also possess actin-like proteins, that drive fundamental cellular function, including cell division, shape maintenance, and DNA segregation. Like eukaryotic actin, bacterial actins assemble dynamic on ATP binding. In general, longitudinal interactions between bacterial actin protomers along each strand are conserved with eukaryotic actin, but variation in interactions between strands gives rise to striking diversity of filament architectures. Here, we report a family of bacterial actins of unknown function, conserved amongst the Verrucomicrobiota phylum, which assembles into a unique tubular structure in the presence of nucleotides. A cryo-EM structure of the filaments reveals it is composed of three strands, unlike other described bacterial actin structures. This architecture provides new insights into the organization of actin-like filaments, and has implications for understanding the diversity and evolution of the bacterial cytoskeleton.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
