<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>biophysics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>biophysics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Reliable protein-protein docking with AlphaFold, Rosetta and replica-exchange
Authors: Harmalkar, A.; Lyskov, S.; Gray, J. J.
Score: 67.6, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551063
Despite the recent breakthrough of AlphaFold (AF) in the field of protein sequence-to-structure prediction, modeling protein interfaces and predicting protein complex structures remains challenging, especially when there is a significant conformational change in one or both binding partners. Prior studies have demonstrated that AF-multimer (AFm) can predict accurate protein complexes in only up to 43% of cases.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/biophysics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="biophysics" />
<meta property="og:description" content="Reliable protein-protein docking with AlphaFold, Rosetta and replica-exchange
Authors: Harmalkar, A.; Lyskov, S.; Gray, J. J.
Score: 67.6, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551063
Despite the recent breakthrough of AlphaFold (AF) in the field of protein sequence-to-structure prediction, modeling protein interfaces and predicting protein complex structures remains challenging, especially when there is a significant conformational change in one or both binding partners. Prior studies have demonstrated that AF-multimer (AFm) can predict accurate protein complexes in only up to 43% of cases." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/biophysics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-02T10:38:09+00:00" />
<meta property="article:modified_time" content="2023-08-02T10:38:09+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="biophysics"/>
<meta name="twitter:description" content="Reliable protein-protein docking with AlphaFold, Rosetta and replica-exchange
Authors: Harmalkar, A.; Lyskov, S.; Gray, J. J.
Score: 67.6, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551063
Despite the recent breakthrough of AlphaFold (AF) in the field of protein sequence-to-structure prediction, modeling protein interfaces and predicting protein complex structures remains challenging, especially when there is a significant conformational change in one or both binding partners. Prior studies have demonstrated that AF-multimer (AFm) can predict accurate protein complexes in only up to 43% of cases."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "biophysics",
      "item": "https://trxiv.yorks0n.com/posts/biophysics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "biophysics",
  "name": "biophysics",
  "description": "Reliable protein-protein docking with AlphaFold, Rosetta and replica-exchange\nAuthors: Harmalkar, A.; Lyskov, S.; Gray, J. J.\nScore: 67.6, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551063\nDespite the recent breakthrough of AlphaFold (AF) in the field of protein sequence-to-structure prediction, modeling protein interfaces and predicting protein complex structures remains challenging, especially when there is a significant conformational change in one or both binding partners. Prior studies have demonstrated that AF-multimer (AFm) can predict accurate protein complexes in only up to 43% of cases.",
  "keywords": [
    
  ],
  "articleBody": " Reliable protein-protein docking with AlphaFold, Rosetta and replica-exchange\nAuthors: Harmalkar, A.; Lyskov, S.; Gray, J. J.\nScore: 67.6, Published: 2023-07-29 DOI: 10.1101/2023.07.28.551063\nDespite the recent breakthrough of AlphaFold (AF) in the field of protein sequence-to-structure prediction, modeling protein interfaces and predicting protein complex structures remains challenging, especially when there is a significant conformational change in one or both binding partners. Prior studies have demonstrated that AF-multimer (AFm) can predict accurate protein complexes in only up to 43% of cases. 1 In this work, we combine AlphaFold as a structural template generator with a physics-based replica exchange docking algorithm. Using a curated collection of 254 available protein targets with both unbound and bound structures, we first demonstrate that AlphaFold confidence measures can be repurposed for estimating protein flexibility and docking accuracy for multimers. We incorporate these metrics within our ReplicaDock 2.0 protocol 2 to complete a robust in-silico pipeline for accurate protein complex structure prediction. AlphaRED (AlphaFold-initiated Replica Exchange Docking) successfully docks failed AF predictions including 97 failure cases in Docking Benchmark Set 5.5. AlphaRED generates CAPRI acceptable-quality or better predictions for 66% of benchmark targets. Further, on a subset of antigen-antibody targets, which is challenging for AFm (19% success rate), AlphaRED demonstrates a success rate of 51%. This new strategy demonstrates the success possible by integrating deep-learning based architectures trained on evolutionary information with physics-based enhanced sampling. The pipeline is available at github.com/Graylab/AlphaRED.\nSquare beams for optimal tiling in TEM\nAuthors: Chua, E. Y. D.; Alink, L. M.; Kopylov, M.; de Marco, A.\nScore: 43.9, Published: 2023-07-30 DOI: 10.1101/2023.07.29.551095\nImaging large fields-of-view at a high magnification requires tiling. Transmission electron microscopes typically have round beam profiles; therefore, tiling across a large field-of-view is either imperfect or results in uneven exposures, which is a problem on dose-sensitive samples. Here we introduce a square electron beam that can be easily retrofitted in existing microscopes and demonstrate its application showing it can tile nearly perfectly and deliver cryo-EM imaging with resolution comparable to conventional setups.\nDisease-specific tau filaments assemble via polymorphic intermediates\nAuthors: LoÌˆvestam, S.; Li, D.; Wagstaff, J. L.; Kotecha, A.; Kimanius, D.; McLaughlin, S. H.; Murzin, A. G.; Freund, S. M. V.; Goedert, M.; Scheres, S. H. W.\nScore: 31.5, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550295\nIntermediate species in the assembly of amyloid filaments are believed to play a central role in neurodegenerative diseases and may constitute important targets for therapeutic intervention. However, structural information about intermediate species has been scarce and the molecular mechanisms by which amyloids assemble remain largely unknown. Here, we use time-resolved electron cryo-microscopy (cryo-EM) to study the in vitro assembly of recombinant truncated tau (amino acids 297-391) into paired helical filaments of Alzheimers disease or into filaments of chronic traumatic encephalopathy. We report the formation of a shared first intermediate amyloid (FIA), with an ordered core comprising amino acids 302-316. Nuclear magnetic resonance indicates that the same amino acids adopt rigid, {beta}-strand-like conformations in monomeric tau. At later time points, the FIAs disappear and we observe many different intermediate amyloid filaments, with structures that depend on the reaction conditions. At the end of both reactions, most intermediate amyloids disappear and filaments with the same ordered cores as those from human brains remain. Our results provide structural insights into the processes of primary and secondary nucleation of amyloid assembly, with implications for the design of novel therapies.\nMechanical control of cell proliferation patterns in growing tissues\nAuthors: Carpenter, L. C.; Perez-Verdugo, F.; Banerjee, S.\nScore: 28.9, Published: 2023-07-27 DOI: 10.1101/2023.07.25.550581\nCell proliferation plays a crucial role in regulating tissue homeostasis and development. However, our understanding of how cell proliferation is controlled in densely packed tissues is limited. Here we develop a computational framework to predict the patterns of cell proliferation in growing tissues, connecting single-cell behaviors and cell-cell interactions to tissue-level growth. Our model incorporates probabilistic rules governing cell growth, division, and elimination, while also taking into account their feedback with tissue mechanics. In particular, cell growth is suppressed and apoptosis is enhanced in regions of high cell density. With these rules and model parameters calibrated using experimental data, we predict how tissue confinement influences cell size and proliferation dynamics, and how single-cell physical properties influence the spatiotemporal patterns of tissue growth. Our findings indicate that mechanical feedback between tissue confinement and cell growth leads to enhanced cell proliferation at tissue boundaries, whereas cell growth in the bulk is arrested. By tuning cellular elasticity and contact inhibition of proliferation we can regulate the emergent patterns of cell proliferation, ranging from uniform growth at low contact inhibition to localized growth at higher contact inhibition. Furthermore, mechanical state of the tissue governs the dynamics of tissue growth, with cellular parameters affecting tissue pressure playing a significant role in determining the overall growth rate. Our computational study thus underscores the impact of cell mechanical properties on the spatiotemporal patterns of cell proliferation in growing tissues.\nMechano-induced homotypic patterned domain formation by monocytes\nAuthors: Du, W.; Zhu, J.; Wu, Y.; Kiemen, A.; Sun, S.; Wirtz, D.\nScore: 26.5, Published: 2023-07-29 DOI: 10.1101/2023.07.27.550819\nMatrix stiffness and corresponding mechano-signaling play indispensable roles in cellular phenotypes and functions. How tissue stiffness influences the behavior of monocytes, a major circulating leukocyte of the innate system, and how it may promote the emergence of collective cell behavior is less understood. Here, using tunable collagen-coated hydrogels of physiological stiffness, we show that human primary monocytes undergo a dynamic local phase separation to form highly patterned multicellular multi-layered domains on soft matrix. Local activation of the {beta}2 integrin initiates inter-cellular adhesion, while global soluble inhibitory factors maintain the steady-state domain pattern over days. Patterned domain formation generated by monocytes is unique among other key immune cells, including macrophages, B cells, T cells, and NK cells. While inhibiting their phagocytic capability, domain formation promotes monocytes survival. We develop a computational model based on the Cahn-Hilliard equation, which includes combined local activation and global inhibition mechanisms of intercellular adhesion suggested by our experiments, and provides experimentally validated predictions of the role of seeding density and both chemotactic and random cell migration on pattern formation.\nGraph Attention Site Prediction (GrASP): Identifying Druggable Binding Sites Using Graph Neural Networks with Attention\nAuthors: Smith, Z.; Strobel, M.; Vani, B.; Tiwary, P.\nScore: 19.5, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550565\nIdentifying and discovering druggable protein binding sites is an important early step in computer-aided drug discovery but remains a difficult task where most campaigns rely on a priori knowledge of binding sites from experiments. Here we present a novel binding site prediction method called Graph Attention Site Prediction (GrASP) and re-evaluate assumptions in nearly every step in the site prediction workflow from dataset preparation to model evaluation. GrASP is able to achieve state-of-the-art performance at recovering binding sites in PDB structures while maintaining a high degree of precision which will minimize wasted computation in downstream tasks such as docking and free energy perturbation.\nLearning high-dimensional reaction coordinates of fast-folding proteins using State Predictive Information Bottleneck and Bias Exchange Metadynamics\nAuthors: Pomarici, N.; Mehdi, S.; Quoika, P.; Lee, S.; Loeffler, J. R.; Liedl, K.; Tiwary, P.; Fernandez-Quintero, M. L.\nScore: 13.3, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550401\nBiological events occurring on long timescales, such as protein folding, remain hard to capture with conventional molecular dynamics (MD) simulation. To overcome these limitations, enhanced sampling techniques can be used to sample regions of the free energy landscape separated by high energy barriers, thereby allowing to observe these rare events. However, many of these techniques require a priori knowledge of the appropriate reaction coordinates (RCs) that describe the process of interest. In recent years, Artificial Intelligence (AI) models have emerged as promising approaches to accelerate rare event sampling. However, integration of these AI methods with MD for automated learning of improved RCs is not trivial, particularly when working with undersampled trajectories and highly complex systems. In this study, we employed the State Predictive Information Bottleneck (SPIB) neural network, coupled with bias exchange metadynamics simulations (BE-metaD), to investigate the unfolding process of two proteins, chignolin and villin. By utilizing the high-dimensional RCs learned from SPIB even with poor training data, BE-metaD simulations dramatically accelerate the sampling of the unfolding process for both proteins. In addition, we compare different RCs and find that the careful selection of RCs is crucial to substantially speed up the sampling of rare events. Thus, this approach, leveraging the power of AI and enhanced sampling techniques, holds great promise for advancing our understanding of complex biological processes occurring on long timescales. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=139 SRC=\"FIGDIR/small/550401v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (29K): org.highwire.dtl.DTLVardef@73cc3dorg.highwire.dtl.DTLVardef@65e836org.highwire.dtl.DTLVardef@704d20org.highwire.dtl.DTLVardef@d79f49_HPS_FORMAT_FIGEXP M_FIG TABLE OF CONTENT GRAPHIC C_FIG\nFolding-upon-binding pathways of an intrinsically disordered protein from a deep Markov state model\nAuthors: Sisk, T.; Robustelli, P.\nScore: 13.3, Published: 2023-07-25 DOI: 10.1101/2023.07.21.550103\nA central challenge in the study of intrinsically disordered proteins is the characterization of the mechanisms by which they bind their physiological interaction partners. Here, we utilize a deep learning based Markov state modeling approach to characterize the folding-upon-binding pathways observed in a long-time scale molecular dynamics simulation of a disordered region of the measles virus nucleoprotein NTAIL reversibly binding the X domain of the measles virus phosphoprotein complex. We find that folding-upon-binding predominantly occurs via two distinct encounter complexes that are differentiated by the binding orientation, helical content, and conformational heterogeneity of NTAIL. We do not, however, find evidence for the existence of canonical conformational selection or induced fit binding pathways. We observe four kinetically separated native-like bound states that interconvert on time scales of eighty to five hundred nanoseconds. These bound states share a core set of native intermolecular contacts and stable NTAIL helices and are differentiated by a sequential formation of native and non-native contacts and additional helical turns. Our analyses provide an atomic resolution structural description of intermediate states in a folding-upon-binding pathway and elucidate the nature of the kinetic barriers between metastable states in a dynamic and heterogenous, or \"fuzzy\", protein complex.\nAlphaFold2 modeling and molecular dynamics simulations of an intrinsically disordered protein\nAuthors: Guo, H.-B.; Huntington, B.; Perminov, A.; Smith, K.; Hastings, N.; Dennis, P.; Kelley-Loughnane, N.; Berry, R. J.\nScore: 9.7, Published: 2023-07-30 DOI: 10.1101/2023.07.29.551023\nWe use AlphaFold2 (AF2) to model the monomer and dimer structures of an intrinsically disordered protein (IDP), Nvjp-1, assisted by molecular dynamics (MD) simulations. We observe relatively rigid dimeric structures of Nvjp-1 when compared with the monomer structures. We suggest that protein conformations from multiple AF2 models and those from MD trajectories exhibit a coherent trend: the conformations of an IDP are deviated from each other and the conformations of a well-folded protein are consistent with each other. We use a residue-residue interaction network (RIN) derived from the contact map which show that the residue-residue interactions in Nvjp-1 are mainly transient; however, those in a well-folded protein are mainly persistent. Despite the variation in 3D shapes, we show that the AF2 models of both disordered and ordered proteins exhibit highly consistent profiles of the pLDDT (predicted local distance difference test) scores. These results indicate a potential protocol to justify the IDPs based on multiple AF2 models and MD simulations. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=106 SRC=\"FIGDIR/small/551023v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (22K): org.highwire.dtl.DTLVardef@1adcba4org.highwire.dtl.DTLVardef@1149056org.highwire.dtl.DTLVardef@10fffaeorg.highwire.dtl.DTLVardef@15b16cd_HPS_FORMAT_FIGEXP M_FIG The TOC figure: C_FIG\nDeep learning insights into the architecture of the mammalian egg-sperm fusion synapse\nAuthors: Elofsson, A.; Han, L.; Bianchi, E.; Wright, G. J.; Jovine, L.\nScore: 5.6, Published: 2023-07-28 DOI: 10.1101/2023.07.27.550195\nFertilization is a crucial event in sexual reproduction culminating in the fusion of the haploid sperm and egg to form a new diploid organism. In mammals, egg receptor JUNO and sperm IZUMO1 directly interact to mediate adhesion between gamete membranes; however, their role in fusion remains enigmatic. Here, we used AlphaFold to predict the structure of other extracellular proteins that are essential for fertilization and determine if they could form a fusogenic complex. We first identified TMEM81, whose gene is expressed by mouse spermatids, as a protein that has structural homologies with both IZUMO1 and another essential sperm molecule, SPACA6. Using a set of proteins known to be important for fertilization, we systematically searched for predicted direct binary interactions using an unguided approach. We identified a tetrameric complex of two sperm (IZUMO1 and SPACA6) and two egg (JUNO and CD9) proteins, and further modeling using this core complex identified TMEM81 as an additional component. We show that the complex is structurally consistent with both the expected topology on opposing gamete membranes and the location of predicted N-linked glycans that were not considered during prediction with AlphaFold-Multimer. Our model suggests that these proteins could organize to form a synapse-like complex between the sperm and egg membrane at the point of fertilization. Finally, the structural modeling approach described here could be more generally useful to gain structural insights into highly transient protein complexes that are difficult to detect experimentally.\n",
  "wordCount" : "2127",
  "inLanguage": "en",
  "datePublished": "2023-08-02T10:38:09Z",
  "dateModified": "2023-08-02T10:38:09Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/biophysics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      biophysics
    </h1>
    <div class="post-meta"><span>updated on August 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.551063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.551063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.551063">
        <p class="paperTitle">Reliable protein-protein docking with AlphaFold, Rosetta and replica-exchange</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.551063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.551063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harmalkar, A.; Lyskov, S.; Gray, J. J.</p>
        <p class="info">Score: 67.6, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.551063' target='https://doi.org/10.1101/2023.07.28.551063'> 10.1101/2023.07.28.551063</a></p>
        <p class="abstract">Despite the recent breakthrough of AlphaFold (AF) in the field of protein sequence-to-structure prediction, modeling protein interfaces and predicting protein complex structures remains challenging, especially when there is a significant conformational change in one or both binding partners. Prior studies have demonstrated that AF-multimer (AFm) can predict accurate protein complexes in only up to 43% of cases. 1 In this work, we combine AlphaFold as a structural template generator with a physics-based replica exchange docking algorithm. Using a curated collection of 254 available protein targets with both unbound and bound structures, we first demonstrate that AlphaFold confidence measures can be repurposed for estimating protein flexibility and docking accuracy for multimers. We incorporate these metrics within our ReplicaDock 2.0 protocol 2 to complete a robust in-silico pipeline for accurate protein complex structure prediction. AlphaRED (AlphaFold-initiated Replica Exchange Docking) successfully docks failed AF predictions including 97 failure cases in Docking Benchmark Set 5.5. AlphaRED generates CAPRI acceptable-quality or better predictions for 66% of benchmark targets. Further, on a subset of antigen-antibody targets, which is challenging for AFm (19% success rate), AlphaRED demonstrates a success rate of 51%. This new strategy demonstrates the success possible by integrating deep-learning based architectures trained on evolutionary information with physics-based enhanced sampling. The pipeline is available at github.com/Graylab/AlphaRED.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.551095">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.551095" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.551095">
        <p class="paperTitle">Square beams for optimal tiling in TEM</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.551095" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.551095" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chua, E. Y. D.; Alink, L. M.; Kopylov, M.; de Marco, A.</p>
        <p class="info">Score: 43.9, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.551095' target='https://doi.org/10.1101/2023.07.29.551095'> 10.1101/2023.07.29.551095</a></p>
        <p class="abstract">Imaging large fields-of-view at a high magnification requires tiling. Transmission electron microscopes typically have round beam profiles; therefore, tiling across a large field-of-view is either imperfect or results in uneven exposures, which is a problem on dose-sensitive samples. Here we introduce a square electron beam that can be easily retrofitted in existing microscopes and demonstrate its application showing it can tile nearly perfectly and deliver cryo-EM imaging with resolution comparable to conventional setups.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550295">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550295" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550295">
        <p class="paperTitle">Disease-specific tau filaments assemble via polymorphic intermediates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550295" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550295" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: LoÌˆvestam, S.; Li, D.; Wagstaff, J. L.; Kotecha, A.; Kimanius, D.; McLaughlin, S. H.; Murzin, A. G.; Freund, S. M. V.; Goedert, M.; Scheres, S. H. W.</p>
        <p class="info">Score: 31.5, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550295' target='https://doi.org/10.1101/2023.07.24.550295'> 10.1101/2023.07.24.550295</a></p>
        <p class="abstract">Intermediate species in the assembly of amyloid filaments are believed to play a central role in neurodegenerative diseases and may constitute important targets for therapeutic intervention. However, structural information about intermediate species has been scarce and the molecular mechanisms by which amyloids assemble remain largely unknown. Here, we use time-resolved electron cryo-microscopy (cryo-EM) to study the in vitro assembly of recombinant truncated tau (amino acids 297-391) into paired helical filaments of Alzheimers disease or into filaments of chronic traumatic encephalopathy. We report the formation of a shared first intermediate amyloid (FIA), with an ordered core comprising amino acids 302-316. Nuclear magnetic resonance indicates that the same amino acids adopt rigid, {beta}-strand-like conformations in monomeric tau. At later time points, the FIAs disappear and we observe many different intermediate amyloid filaments, with structures that depend on the reaction conditions. At the end of both reactions, most intermediate amyloids disappear and filaments with the same ordered cores as those from human brains remain. Our results provide structural insights into the processes of primary and secondary nucleation of amyloid assembly, with implications for the design of novel therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550581">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550581" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550581">
        <p class="paperTitle">Mechanical control of cell proliferation patterns in growing tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550581" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550581" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carpenter, L. C.; Perez-Verdugo, F.; Banerjee, S.</p>
        <p class="info">Score: 28.9, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550581' target='https://doi.org/10.1101/2023.07.25.550581'> 10.1101/2023.07.25.550581</a></p>
        <p class="abstract">Cell proliferation plays a crucial role in regulating tissue homeostasis and development. However, our understanding of how cell proliferation is controlled in densely packed tissues is limited. Here we develop a computational framework to predict the patterns of cell proliferation in growing tissues, connecting single-cell behaviors and cell-cell interactions to tissue-level growth. Our model incorporates probabilistic rules governing cell growth, division, and elimination, while also taking into account their feedback with tissue mechanics. In particular, cell growth is suppressed and apoptosis is enhanced in regions of high cell density. With these rules and model parameters calibrated using experimental data, we predict how tissue confinement influences cell size and proliferation dynamics, and how single-cell physical properties influence the spatiotemporal patterns of tissue growth. Our findings indicate that mechanical feedback between tissue confinement and cell growth leads to enhanced cell proliferation at tissue boundaries, whereas cell growth in the bulk is arrested. By tuning cellular elasticity and contact inhibition of proliferation we can regulate the emergent patterns of cell proliferation, ranging from uniform growth at low contact inhibition to localized growth at higher contact inhibition. Furthermore, mechanical state of the tissue governs the dynamics of tissue growth, with cellular parameters affecting tissue pressure playing a significant role in determining the overall growth rate. Our computational study thus underscores the impact of cell mechanical properties on the spatiotemporal patterns of cell proliferation in growing tissues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550819">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550819" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550819">
        <p class="paperTitle">Mechano-induced homotypic patterned domain formation by monocytes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550819" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550819" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Du, W.; Zhu, J.; Wu, Y.; Kiemen, A.; Sun, S.; Wirtz, D.</p>
        <p class="info">Score: 26.5, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550819' target='https://doi.org/10.1101/2023.07.27.550819'> 10.1101/2023.07.27.550819</a></p>
        <p class="abstract">Matrix stiffness and corresponding mechano-signaling play indispensable roles in cellular phenotypes and functions. How tissue stiffness influences the behavior of monocytes, a major circulating leukocyte of the innate system, and how it may promote the emergence of collective cell behavior is less understood. Here, using tunable collagen-coated hydrogels of physiological stiffness, we show that human primary monocytes undergo a dynamic local phase separation to form highly patterned multicellular multi-layered domains on soft matrix. Local activation of the {beta}2 integrin initiates inter-cellular adhesion, while global soluble inhibitory factors maintain the steady-state domain pattern over days. Patterned domain formation generated by monocytes is unique among other key immune cells, including macrophages, B cells, T cells, and NK cells. While inhibiting their phagocytic capability, domain formation promotes monocytes survival. We develop a computational model based on the Cahn-Hilliard equation, which includes combined local activation and global inhibition mechanisms of intercellular adhesion suggested by our experiments, and provides experimentally validated predictions of the role of seeding density and both chemotactic and random cell migration on pattern formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550565">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550565" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550565">
        <p class="paperTitle">Graph Attention Site Prediction (GrASP): Identifying Druggable Binding Sites Using Graph Neural Networks with Attention</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550565" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550565" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, Z.; Strobel, M.; Vani, B.; Tiwary, P.</p>
        <p class="info">Score: 19.5, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550565' target='https://doi.org/10.1101/2023.07.25.550565'> 10.1101/2023.07.25.550565</a></p>
        <p class="abstract">Identifying and discovering druggable protein binding sites is an important early step in computer-aided drug discovery but remains a difficult task where most campaigns rely on a priori knowledge of binding sites from experiments. Here we present a novel binding site prediction method called Graph Attention Site Prediction (GrASP) and re-evaluate assumptions in nearly every step in the site prediction workflow from dataset preparation to model evaluation. GrASP is able to achieve state-of-the-art performance at recovering binding sites in PDB structures while maintaining a high degree of precision which will minimize wasted computation in downstream tasks such as docking and free energy perturbation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550401">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550401" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550401">
        <p class="paperTitle">Learning high-dimensional reaction coordinates of fast-folding proteins using State Predictive Information Bottleneck and Bias Exchange Metadynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550401" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550401" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pomarici, N.; Mehdi, S.; Quoika, P.; Lee, S.; Loeffler, J. R.; Liedl, K.; Tiwary, P.; Fernandez-Quintero, M. L.</p>
        <p class="info">Score: 13.3, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550401' target='https://doi.org/10.1101/2023.07.24.550401'> 10.1101/2023.07.24.550401</a></p>
        <p class="abstract">Biological events occurring on long timescales, such as protein folding, remain hard to capture with conventional molecular dynamics (MD) simulation. To overcome these limitations, enhanced sampling techniques can be used to sample regions of the free energy landscape separated by high energy barriers, thereby allowing to observe these rare events. However, many of these techniques require a priori knowledge of the appropriate reaction coordinates (RCs) that describe the process of interest. In recent years, Artificial Intelligence (AI) models have emerged as promising approaches to accelerate rare event sampling. However, integration of these AI methods with MD for automated learning of improved RCs is not trivial, particularly when working with undersampled trajectories and highly complex systems. In this study, we employed the State Predictive Information Bottleneck (SPIB) neural network, coupled with bias exchange metadynamics simulations (BE-metaD), to investigate the unfolding process of two proteins, chignolin and villin. By utilizing the high-dimensional RCs learned from SPIB even with poor training data, BE-metaD simulations dramatically accelerate the sampling of the unfolding process for both proteins. In addition, we compare different RCs and find that the careful selection of RCs is crucial to substantially speed up the sampling of rare events. Thus, this approach, leveraging the power of AI and enhanced sampling techniques, holds great promise for advancing our understanding of complex biological processes occurring on long timescales.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=139 SRC=&#34;FIGDIR/small/550401v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (29K):
org.highwire.dtl.DTLVardef@73cc3dorg.highwire.dtl.DTLVardef@65e836org.highwire.dtl.DTLVardef@704d20org.highwire.dtl.DTLVardef@d79f49_HPS_FORMAT_FIGEXP  M_FIG TABLE OF CONTENT GRAPHIC

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550103">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550103" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550103">
        <p class="paperTitle">Folding-upon-binding pathways of an intrinsically disordered protein from a deep Markov state model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550103" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550103" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sisk, T.; Robustelli, P.</p>
        <p class="info">Score: 13.3, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550103' target='https://doi.org/10.1101/2023.07.21.550103'> 10.1101/2023.07.21.550103</a></p>
        <p class="abstract">A central challenge in the study of intrinsically disordered proteins is the characterization of the mechanisms by which they bind their physiological interaction partners. Here, we utilize a deep learning based Markov state modeling approach to characterize the folding-upon-binding pathways observed in a long-time scale molecular dynamics simulation of a disordered region of the measles virus nucleoprotein NTAIL reversibly binding the X domain of the measles virus phosphoprotein complex. We find that folding-upon-binding predominantly occurs via two distinct encounter complexes that are differentiated by the binding orientation, helical content, and conformational heterogeneity of NTAIL. We do not, however, find evidence for the existence of canonical conformational selection or induced fit binding pathways. We observe four kinetically separated native-like bound states that interconvert on time scales of eighty to five hundred nanoseconds. These bound states share a core set of native intermolecular contacts and stable NTAIL helices and are differentiated by a sequential formation of native and non-native contacts and additional helical turns. Our analyses provide an atomic resolution structural description of intermediate states in a folding-upon-binding pathway and elucidate the nature of the kinetic barriers between metastable states in a dynamic and heterogenous, or &#34;fuzzy&#34;, protein complex.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.29.551023">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.29.551023" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.29.551023">
        <p class="paperTitle">AlphaFold2 modeling and molecular dynamics simulations of an intrinsically disordered protein</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.29.551023" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.29.551023" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, H.-B.; Huntington, B.; Perminov, A.; Smith, K.; Hastings, N.; Dennis, P.; Kelley-Loughnane, N.; Berry, R. J.</p>
        <p class="info">Score: 9.7, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.29.551023' target='https://doi.org/10.1101/2023.07.29.551023'> 10.1101/2023.07.29.551023</a></p>
        <p class="abstract">We use AlphaFold2 (AF2) to model the monomer and dimer structures of an intrinsically disordered protein (IDP), Nvjp-1, assisted by molecular dynamics (MD) simulations. We observe relatively rigid dimeric structures of Nvjp-1 when compared with the monomer structures. We suggest that protein conformations from multiple AF2 models and those from MD trajectories exhibit a coherent trend: the conformations of an IDP are deviated from each other and the conformations of a well-folded protein are consistent with each other. We use a residue-residue interaction network (RIN) derived from the contact map which show that the residue-residue interactions in Nvjp-1 are mainly transient; however, those in a well-folded protein are mainly persistent. Despite the variation in 3D shapes, we show that the AF2 models of both disordered and ordered proteins exhibit highly consistent profiles of the pLDDT (predicted local distance difference test) scores. These results indicate a potential protocol to justify the IDPs based on multiple AF2 models and MD simulations.

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=106 SRC=&#34;FIGDIR/small/551023v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (22K):
org.highwire.dtl.DTLVardef@1adcba4org.highwire.dtl.DTLVardef@1149056org.highwire.dtl.DTLVardef@10fffaeorg.highwire.dtl.DTLVardef@15b16cd_HPS_FORMAT_FIGEXP  M_FIG The TOC figure:

C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550195">
        <p class="paperTitle">Deep learning insights into the architecture of the mammalian egg-sperm fusion synapse</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Elofsson, A.; Han, L.; Bianchi, E.; Wright, G. J.; Jovine, L.</p>
        <p class="info">Score: 5.6, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550195' target='https://doi.org/10.1101/2023.07.27.550195'> 10.1101/2023.07.27.550195</a></p>
        <p class="abstract">Fertilization is a crucial event in sexual reproduction culminating in the fusion of the haploid sperm and egg to form a new diploid organism. In mammals, egg receptor JUNO and sperm IZUMO1 directly interact to mediate adhesion between gamete membranes; however, their role in fusion remains enigmatic. Here, we used AlphaFold to predict the structure of other extracellular proteins that are essential for fertilization and determine if they could form a fusogenic complex. We first identified TMEM81, whose gene is expressed by mouse spermatids, as a protein that has structural homologies with both IZUMO1 and another essential sperm molecule, SPACA6. Using a set of proteins known to be important for fertilization, we systematically searched for predicted direct binary interactions using an unguided approach. We identified a tetrameric complex of two sperm (IZUMO1 and SPACA6) and two egg (JUNO and CD9) proteins, and further modeling using this core complex identified TMEM81 as an additional component. We show that the complex is structurally consistent with both the expected topology on opposing gamete membranes and the location of predicted N-linked glycans that were not considered during prediction with AlphaFold-Multimer. Our model suggests that these proteins could organize to form a synapse-like complex between the sperm and egg membrane at the point of fertilization. Finally, the structural modeling approach described here could be more generally useful to gain structural insights into highly transient protein complexes that are difficult to detect experimentally.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
