<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cardiovascular medicine</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cardiovascular medicine | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="The Cardiovascular Impact and Genetics of Pericardial Adiposity
Authors: Ramo, J. T.; Kany, S.; Hou, C. R.; Friedman, S. F.; Roselli, C.; Nauffal, V.; Koyama, S.; Karjalainen, J.; FinnGen, ; Maddah, M.; Palotie, A.; Ellinor, P. T.; Pirruccello, J. P.
Score: 15.0, Published: 2023-07-18 DOI: 10.1101/2023.07.16.23292729
BackgroundWhile previous studies have reported associations of pericardial adipose tissue (PAT) with cardiovascular diseases such as atrial fibrillation and coronary artery disease, they have been limited in sample size or drawn from selected populations.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cardiovascular-medicine/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cardiovascular medicine" />
<meta property="og:description" content="The Cardiovascular Impact and Genetics of Pericardial Adiposity
Authors: Ramo, J. T.; Kany, S.; Hou, C. R.; Friedman, S. F.; Roselli, C.; Nauffal, V.; Koyama, S.; Karjalainen, J.; FinnGen, ; Maddah, M.; Palotie, A.; Ellinor, P. T.; Pirruccello, J. P.
Score: 15.0, Published: 2023-07-18 DOI: 10.1101/2023.07.16.23292729
BackgroundWhile previous studies have reported associations of pericardial adipose tissue (PAT) with cardiovascular diseases such as atrial fibrillation and coronary artery disease, they have been limited in sample size or drawn from selected populations." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cardiovascular-medicine/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-23T10:36:34+00:00" />
<meta property="article:modified_time" content="2023-07-23T10:36:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cardiovascular medicine"/>
<meta name="twitter:description" content="The Cardiovascular Impact and Genetics of Pericardial Adiposity
Authors: Ramo, J. T.; Kany, S.; Hou, C. R.; Friedman, S. F.; Roselli, C.; Nauffal, V.; Koyama, S.; Karjalainen, J.; FinnGen, ; Maddah, M.; Palotie, A.; Ellinor, P. T.; Pirruccello, J. P.
Score: 15.0, Published: 2023-07-18 DOI: 10.1101/2023.07.16.23292729
BackgroundWhile previous studies have reported associations of pericardial adipose tissue (PAT) with cardiovascular diseases such as atrial fibrillation and coronary artery disease, they have been limited in sample size or drawn from selected populations."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cardiovascular medicine",
      "item": "https://trxiv.yorks0n.com/posts/cardiovascular-medicine/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cardiovascular medicine",
  "name": "cardiovascular medicine",
  "description": "The Cardiovascular Impact and Genetics of Pericardial Adiposity\nAuthors: Ramo, J. T.; Kany, S.; Hou, C. R.; Friedman, S. F.; Roselli, C.; Nauffal, V.; Koyama, S.; Karjalainen, J.; FinnGen, ; Maddah, M.; Palotie, A.; Ellinor, P. T.; Pirruccello, J. P.\nScore: 15.0, Published: 2023-07-18 DOI: 10.1101/2023.07.16.23292729\nBackgroundWhile previous studies have reported associations of pericardial adipose tissue (PAT) with cardiovascular diseases such as atrial fibrillation and coronary artery disease, they have been limited in sample size or drawn from selected populations.",
  "keywords": [
    
  ],
  "articleBody": " The Cardiovascular Impact and Genetics of Pericardial Adiposity\nAuthors: Ramo, J. T.; Kany, S.; Hou, C. R.; Friedman, S. F.; Roselli, C.; Nauffal, V.; Koyama, S.; Karjalainen, J.; FinnGen, ; Maddah, M.; Palotie, A.; Ellinor, P. T.; Pirruccello, J. P.\nScore: 15.0, Published: 2023-07-18 DOI: 10.1101/2023.07.16.23292729\nBackgroundWhile previous studies have reported associations of pericardial adipose tissue (PAT) with cardiovascular diseases such as atrial fibrillation and coronary artery disease, they have been limited in sample size or drawn from selected populations. Additionally, the genetic determinants of PAT remain largely unknown. We aimed to evaluate the association of PAT with prevalent and incident cardiovascular disease and to elucidate the genetic basis of PAT in a large population cohort. MethodsA deep learning model was trained to quantify PAT area from four-chamber magnetic resonance images in the UK Biobank using semantic segmentation. Cross-sectional and prospective cardiovascular disease associations were evaluated, controlling for sex and age. A genome-wide association study was performed, and a polygenic score (PGS) for PAT was examined in 453,733 independent FinnGen study participants. ResultsA total of 44,725 UK Biobank participants (51.7% female, mean [SD] age 64.1 [7.7] years) were included. PAT was positively associated with male sex ({beta} = +0.76 SD in PAT), age (r = 0.15), body mass index (BMI; r = 0.47) and waist-to-hip ratio (r = 0.55) (P \u003c 1x10-230). PAT was more elevated in prevalent heart failure ({beta} = +0.46 SD units) and type 2 diabetes ({beta} = +0.56) than in coronary artery disease ({beta} = +0.22) or AF ({beta} = +0.18). PAT was associated with incident heart failure (HR = 1.29 per +1 SD in PAT [95% CI 1.17-1.43]) and type 2 diabetes (HR = 1.63 [1.51-1.76]) during a mean 3.2 ({+/-}1.5) years of follow-up; the associations remained significant when controlling for BMI. We identified 5 novel genetic loci for PAT and implicated transcriptional regulators of adipocyte morphology and brown adipogenesis (EBF1, EBF2 and CEBPA) and regulators of visceral adiposity (WARS2 and TRIB2). The PAT PGS was associated with T2D, heart failure, coronary artery disease and atrial fibrillation in FinnGen (ORs 1.03-1.06 per +1 SD in PGS, P \u003c 2x10-10). ConclusionsPAT shares genetic determinants with abdominal adiposity and is an independent predictor of incident type 2 diabetes and heart failure. Clinical PerspectiveWhat is new? O_LIIn a large, prospective and uniformly phenotyped cohort, pericardial adipose tissue was independently predictive of incident heart failure and type 2 diabetes when adjusted for body mass index. C_LIO_LIIn contrast, pericardial adipose tissue was not independently predictive of atrial fibrillation. C_LIO_LIA genome-wide association study of pericardial adipose tissue identified five novel loci, implicating genes influencing adipocyte morphology, brown-like adipose tissue differentiation and abdominal adiposity. C_LI What are the clinical implications? O_LIPericardial adipose tissue accumulation may reflect a metabolically unhealthy adiposity phenotype similarly to abdominal visceral adiposity. C_LI\nCommon- and rare-variant genetic architecture of heart failure across the allele frequency spectrum\nAuthors: Lee, D. S. M.; DePaolo, J. S.; Aragam, K. G.; Biddinger, K.; Conery, M.; Dilitikas, O.; Hoffman-Andrews, L.; Judy, R. L.; Khan, A.; Kulo, I.; Puckelwartz, M. J.; Reza, N.; Satterfield, B. A.; Singhal, P.; Regeneron Genetics Center, ; Arany, Z. P.; Cappola, T. P.; Carruth, E.; Day, S. M.; Do, R.; Haggarty, C. M.; Joseph, J.; McNally, E.; Nadkarni, G.; Owens, A. T.; Rader, D. J.; Ritchie, M. D.; Sun, Y.; Voight, B. F.; Levin, M. G.; Damrauer, S. M.\nScore: 3.1, Published: 2023-07-19 DOI: 10.1101/2023.07.16.23292724\nHeart failure (HF) is a complex trait, influenced by environmental and genetic factors, that affects over 30 million individuals worldwide. Historically, the genetics of HF have been studied in Mendelian forms of disease, linking rare genetic variants to familial cardiomyopathies. More recently, genome-wide association studies (GWAS) have successfully identified common genetic variants associated with risk of HF. However, the relative importance of genetic variants across the allele-frequency spectrum remains incompletely characterized. Here, we report the results of common- and rare-variant association studies of all-cause heart failure, applying recently developed methods to quantify the heritability of HF attributable to different classes of genetic variation. We combine GWAS data across multi-ancestry populations including 207,346 individuals from HF and 2,151,210 without, identifying 176 risk loci at genome-wide significance (p \u003c 5x10-8). Signals at newly identified common-variant loci include coding variants in Mendelian cardiomyopathy genes (MYBPC3, BAG3), as well as regulators of lipoprotein (LPL) and glucose metabolism (GIPR, GLP1R), and are enriched in cardiac, muscle, nerve, and vascular tissues, as well as myocyte and adipocyte cell types. Gene burden studies in the Penn Medicine BioBank uncover exome-wide significant (p \u003c 3.03x10-6) associations for HF and rare predicted loss-of-function variants in TTN and MYBPC3. Although the total burden heritability of rare coding variants in HF (3.5%, 95% CI 0.94-6.0%) is comparable to common variant heritability (4.3%, 95% CI 3.9-4.7%), burden heritability is highly concentrated in a small set of Mendelian cardiomyopathy genes (30.4% explained by TTN, LMNA, FLNC, and MYBPC3), while common-variant heritability is more diffusely spread throughout the genome. Finally, we demonstrate that common-variant background, in the form of a polygenic risk score (PRS), significantly modifies the risk of HF among carriers of pathogenic truncating variants in the Mendelian cardiomyopathy gene TTN. These findings suggest a significant polygenic component to HF exists that is not captured by current clinical genetic testing.\nAlternations in gut microbiota and host transcriptome of patients with coronary artery disease\nAuthors: Chen, L.; Mou, X.; Li, J.; Li, M.; Ye, C.; Gao, X.; Liu, X.; Ma, Y.; Xu, Y.; Zhong, Y.\nScore: 1.2, Published: 2023-07-16 DOI: 10.1101/2023.07.14.23292642\nBackgroundCoronary artery disease (CAD) is a widespread heart condition caused by atherosclerosis and influences millions of people worldwide. Early detection of CAD is challenging due to the lack of specific biomarkers. The gut microbiota and host-microbiota interactions have been well documented to affect human health. However, investigation that reveals the role of gut microbes in CAD is still limited. This study aims to uncover the synergistic effects of host genes and gut microbes associated with CAD through integrative genomic analyses. ResultsHerein, we collected 54 fecal and 54 blood samples from CAD patients and matched controls, and performed amplicon and transcriptomic sequencing on these samples, respectively. By comparing CAD patients with health controls, we found that dysregulated gut microbes were significantly associated with CAD. By leveraging the Random Forest method, we found that 10 bacteria biomarkers can distinguish CAD patients from health controls with a high performance (AUC = 0.939). We observed that there existed prominent associations of gut microbes with several clinical indices relevant to heart functions. Integration analysis revealed that CAD-relevant gut microbe genus Fusicatenibacter was associated with expression of CAD-risk genes, such as GBP2, MLKL, and CPR65. In addition, the upregulation of immune-related pathways in CAD patients were identified to be primarily associated with higher abundance of genus Blautia, Eubacterium, Fusicatenibacter, and Monoglobus. ConclusionsOur results highlight that dysregulated gut microbes contribute risk to CAD by interacting with host genes. These identified microbes and interacted risk genes may have high potentials as biomarkers for CAD.\n",
  "wordCount" : "1159",
  "inLanguage": "en",
  "datePublished": "2023-07-23T10:36:34Z",
  "dateModified": "2023-07-23T10:36:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cardiovascular-medicine/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cardiovascular medicine
    </h1>
    <div class="post-meta"><span>updated on July 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.23292729">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.23292729" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.23292729">
        <p class="paperTitle">The Cardiovascular Impact and Genetics of Pericardial Adiposity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.23292729" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.23292729" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramo, J. T.; Kany, S.; Hou, C. R.; Friedman, S. F.; Roselli, C.; Nauffal, V.; Koyama, S.; Karjalainen, J.; FinnGen,  ; Maddah, M.; Palotie, A.; Ellinor, P. T.; Pirruccello, J. P.</p>
        <p class="info">Score: 15.0, Published: 2023-07-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.23292729' target='https://doi.org/10.1101/2023.07.16.23292729'> 10.1101/2023.07.16.23292729</a></p>
        <p class="abstract">BackgroundWhile previous studies have reported associations of pericardial adipose tissue (PAT) with cardiovascular diseases such as atrial fibrillation and coronary artery disease, they have been limited in sample size or drawn from selected populations. Additionally, the genetic determinants of PAT remain largely unknown. We aimed to evaluate the association of PAT with prevalent and incident cardiovascular disease and to elucidate the genetic basis of PAT in a large population cohort.

MethodsA deep learning model was trained to quantify PAT area from four-chamber magnetic resonance images in the UK Biobank using semantic segmentation. Cross-sectional and prospective cardiovascular disease associations were evaluated, controlling for sex and age. A genome-wide association study was performed, and a polygenic score (PGS) for PAT was examined in 453,733 independent FinnGen study participants.

ResultsA total of 44,725 UK Biobank participants (51.7% female, mean [SD] age 64.1 [7.7] years) were included. PAT was positively associated with male sex ({beta} = &#43;0.76 SD in PAT), age (r = 0.15), body mass index (BMI; r = 0.47) and waist-to-hip ratio (r = 0.55) (P &lt; 1x10-230). PAT was more elevated in prevalent heart failure ({beta} = &#43;0.46 SD units) and type 2 diabetes ({beta} = &#43;0.56) than in coronary artery disease ({beta} = &#43;0.22) or AF ({beta} = &#43;0.18). PAT was associated with incident heart failure (HR = 1.29 per &#43;1 SD in PAT [95% CI 1.17-1.43]) and type 2 diabetes (HR = 1.63 [1.51-1.76]) during a mean 3.2 ({&#43;/-}1.5) years of follow-up; the associations remained significant when controlling for BMI. We identified 5 novel genetic loci for PAT and implicated transcriptional regulators of adipocyte morphology and brown adipogenesis (EBF1, EBF2 and CEBPA) and regulators of visceral adiposity (WARS2 and TRIB2). The PAT PGS was associated with T2D, heart failure, coronary artery disease and atrial fibrillation in FinnGen (ORs 1.03-1.06 per &#43;1 SD in PGS, P &lt; 2x10-10).

ConclusionsPAT shares genetic determinants with abdominal adiposity and is an independent predictor of incident type 2 diabetes and heart failure.

Clinical PerspectiveWhat is new? O_LIIn a large, prospective and uniformly phenotyped cohort, pericardial adipose tissue was independently predictive of incident heart failure and type 2 diabetes when adjusted for body mass index.
C_LIO_LIIn contrast, pericardial adipose tissue was not independently predictive of atrial fibrillation.
C_LIO_LIA genome-wide association study of pericardial adipose tissue identified five novel loci, implicating genes influencing adipocyte morphology, brown-like adipose tissue differentiation and abdominal adiposity.
C_LI

What are the clinical implications? O_LIPericardial adipose tissue accumulation may reflect a metabolically unhealthy adiposity phenotype similarly to abdominal visceral adiposity.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.23292724">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.23292724" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.23292724">
        <p class="paperTitle">Common- and rare-variant genetic architecture of heart failure across the allele frequency spectrum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.23292724" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.23292724" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, D. S. M.; DePaolo, J. S.; Aragam, K. G.; Biddinger, K.; Conery, M.; Dilitikas, O.; Hoffman-Andrews, L.; Judy, R. L.; Khan, A.; Kulo, I.; Puckelwartz, M. J.; Reza, N.; Satterfield, B. A.; Singhal, P.; Regeneron Genetics Center,  ; Arany, Z. P.; Cappola, T. P.; Carruth, E.; Day, S. M.; Do, R.; Haggarty, C. M.; Joseph, J.; McNally, E.; Nadkarni, G.; Owens, A. T.; Rader, D. J.; Ritchie, M. D.; Sun, Y.; Voight, B. F.; Levin, M. G.; Damrauer, S. M.</p>
        <p class="info">Score: 3.1, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.23292724' target='https://doi.org/10.1101/2023.07.16.23292724'> 10.1101/2023.07.16.23292724</a></p>
        <p class="abstract">Heart failure (HF) is a complex trait, influenced by environmental and genetic factors, that affects over 30 million individuals worldwide. Historically, the genetics of HF have been studied in Mendelian forms of disease, linking rare genetic variants to familial cardiomyopathies. More recently, genome-wide association studies (GWAS) have successfully identified common genetic variants associated with risk of HF. However, the relative importance of genetic variants across the allele-frequency spectrum remains incompletely characterized. Here, we report the results of common- and rare-variant association studies of all-cause heart failure, applying recently developed methods to quantify the heritability of HF attributable to different classes of genetic variation. We combine GWAS data across multi-ancestry populations including 207,346 individuals from HF and 2,151,210 without, identifying 176 risk loci at genome-wide significance (p &lt; 5x10-8). Signals at newly identified common-variant loci include coding variants in Mendelian cardiomyopathy genes (MYBPC3, BAG3), as well as regulators of lipoprotein (LPL) and glucose metabolism (GIPR, GLP1R), and are enriched in cardiac, muscle, nerve, and vascular tissues, as well as myocyte and adipocyte cell types. Gene burden studies in the Penn Medicine BioBank uncover exome-wide significant (p &lt; 3.03x10-6) associations for HF and rare predicted loss-of-function variants in TTN and MYBPC3. Although the total burden heritability of rare coding variants in HF (3.5%, 95% CI 0.94-6.0%) is comparable to common variant heritability (4.3%, 95% CI 3.9-4.7%), burden heritability is highly concentrated in a small set of Mendelian cardiomyopathy genes (30.4% explained by TTN, LMNA, FLNC, and MYBPC3), while common-variant heritability is more diffusely spread throughout the genome. Finally, we demonstrate that common-variant background, in the form of a polygenic risk score (PRS), significantly modifies the risk of HF among carriers of pathogenic truncating variants in the Mendelian cardiomyopathy gene TTN. These findings suggest a significant polygenic component to HF exists that is not captured by current clinical genetic testing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.23292642">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.23292642" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.23292642">
        <p class="paperTitle">Alternations in gut microbiota and host transcriptome of patients with coronary artery disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.23292642" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.23292642" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, L.; Mou, X.; Li, J.; Li, M.; Ye, C.; Gao, X.; Liu, X.; Ma, Y.; Xu, Y.; Zhong, Y.</p>
        <p class="info">Score: 1.2, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.23292642' target='https://doi.org/10.1101/2023.07.14.23292642'> 10.1101/2023.07.14.23292642</a></p>
        <p class="abstract">BackgroundCoronary artery disease (CAD) is a widespread heart condition caused by atherosclerosis and influences millions of people worldwide. Early detection of CAD is challenging due to the lack of specific biomarkers. The gut microbiota and host-microbiota interactions have been well documented to affect human health. However, investigation that reveals the role of gut microbes in CAD is still limited. This study aims to uncover the synergistic effects of host genes and gut microbes associated with CAD through integrative genomic analyses.

ResultsHerein, we collected 54 fecal and 54 blood samples from CAD patients and matched controls, and performed amplicon and transcriptomic sequencing on these samples, respectively. By comparing CAD patients with health controls, we found that dysregulated gut microbes were significantly associated with CAD. By leveraging the Random Forest method, we found that 10 bacteria biomarkers can distinguish CAD patients from health controls with a high performance (AUC = 0.939). We observed that there existed prominent associations of gut microbes with several clinical indices relevant to heart functions. Integration analysis revealed that CAD-relevant gut microbe genus Fusicatenibacter was associated with expression of CAD-risk genes, such as GBP2, MLKL, and CPR65. In addition, the upregulation of immune-related pathways in CAD patients were identified to be primarily associated with higher abundance of genus Blautia, Eubacterium, Fusicatenibacter, and Monoglobus.

ConclusionsOur results highlight that dysregulated gut microbes contribute risk to CAD by interacting with host genes. These identified microbes and interacted risk genes may have high potentials as biomarkers for CAD.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
