<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample
Authors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O&#39;Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample
Authors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O&#39;Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-12T10:41:10+00:00" />
<meta property="article:modified_time" content="2023-07-12T10:41:10+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample
Authors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O&#39;Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample\nAuthors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O\u0026#39;Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J.",
  "keywords": [
    
  ],
  "articleBody": " Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample\nAuthors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O'Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J. P.; Pletch, K.; Ruiz-Herrera, A.; Lander, E. S.; Rowley, M. J.; Onuchic, J. N.; Dalen, L.; Marti-Renom, M. A.; Gilbert, M. T. P.; Aiden, E. L.\nScore: 61.7, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547175\nAncient DNA (aDNA) sequencing analysis typically involves alignment to a modern reference genome assembly from a related species. Since aDNA molecules are fragmentary, these alignments yield information about small-scale differences, but provide no information about larger features such as the chromosome structure of ancient species. We report the genome assembly of a female Late Pleistocene woolly mammoth (Mammuthus primigenius) with twenty-eight chromosome-length scaffolds, generated using mammoth skin preserved in permafrost for roughly 52,000 years. We began by creating a modified Hi-C protocol, dubbed PaleoHi-C, optimized for ancient samples, and using it to map chromatin contacts in a woolly mammoth. Next, we developed \"reference-assisted 3D genome assembly,\" which begins with a reference genome assembly from a related species, and uses Hi-C and DNA-Seq data from a target species to split, order, orient, and correct sequences on the basis of their 3D proximity, yielding accurate chromosome-length scaffolds for the target species. By means of this reference-assisted 3D genome assembly, PaleoHi-C data reveals the 3D architecture of a woolly mammoth genome, including chromosome territories, compartments, domains, and loops. The active (A) and inactive (B) genome compartments in mammoth skin more closely resemble those observed in Asian elephant skin than the compartmentalization patterns seen in other Asian elephant tissues. Differences in compartmentalization between these skin samples reveal sequences whose transcription was potentially altered in mammoth. We observe a tetradic structure for the inactive X chromosome in mammoth, distinct from the bipartite architecture seen in human and mouse. Generating chromosome-length genome assemblies for two other elephantids (Asian and African elephant), we find that the overall karyotype, and this tetradic Xi structure, are conserved throughout the clade. These results illustrate that cell-type specific epigenetic information can be preserved in ancient samples, in the form of DNA geometry, and that it may be feasible to perform de novo genome assembly of some extinct species.\nTopological stratification of continuous genetic variation in large biobanks\nAuthors: Diaz-Papkovich, A.; Zabad, S.; Ben-Eghan, C.; Anderson-Trocme, L.; Femerling, G.; Nathan, V.; Patel, J.; Gravel, S.\nScore: 23.6, Published: 2023-07-07 DOI: 10.1101/2023.07.06.548007\nBiobanks now contain genetic data from millions of individuals. Dimension-ality reduction, visualization and stratification are standard when exploring data at these scales; while efficient and tractable methods exist for the first two, stratification remains challenging because of uncertainty about sources of population structure. In practice, stratification is commonly performed by drawing shapes around dimensionally reduced data or assuming populations have a \"type\" genome. We propose a method of stratifying data with topo-logical analysis that is fast, easy to implement, and integrates with existing pipelines. The approach is robust to the presence of sub-populations of vary-ing sizes and wide ranges of population structure patterns. We demonstrate its effectiveness on genotypes from three biobanks and illustrate how topolog-ical genetic strata can help us understand structure within biobanks, evaluate distributions of genotypic and phenotypic data, examine polygenic score trans-ferability, identify potential influential alleles, and perform quality control.\nA method to correct for local alterations in DNA copy number that bias functional genomics assays applied to antibiotic-treated bacteria\nAuthors: Sullivan, G. J.; Barquist, L.; Cain, A. K.\nScore: 22.7, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548391\nFunctional genomics techniques, such as transposon insertion sequencing and RNA-sequencing, are key to studying relative differences in mutant fitness or gene expression under selective conditions. However, certain stress conditions, mutations, or antibiotics can directly interfere with DNA synthesis, resulting in systematic changes in local DNA copy number along the chromosome. This can lead to artefacts in sequencing-based functional genomics data when comparing antibiotic treatment to an unstressed control, with relative differences in gene-wise read counts being the result of alterations in chromosomal replication dynamics rather than selection or direct gene regulation. We term this artefact 'chromosomal location bias' and implement a principled statistical approach to correct for it by calculating local normalization factors along the chromosome. These normalization factors are then directly incorporated in statistical analyses using standard RNA-sequencing analysis methods without modifying the read counts themselves, preserving important information about the mean-variance relationship in the data. We illustrate the utility of this approach by generating and analysing a ciprofloxacin-treated transposon insertion sequencing dataset in Escherichia coli as a case study. We show that ciprofloxacin treatment generates chromosomal location bias in the resulting data, and we further demonstrate that failing to correct for this bias leads to false predictions of mutant drug sensitivity as measured by minimum inhibitory concentrations. We have developed an R package and user-friendly graphical Shiny application, ChromoCorrect, that detects and corrects for chromosomal bias in read count data, enabling the application of functional genomics technologies to the study of antibiotic stress.\nAllelic bias when performing in-solution enrichment of ancient human DNA\nAuthors: Davidson, R.; Williams, M. P.; Roca-Rada, X.; Kassadjikova, K.; Tobler, R.; Fehren-Schmitz, L.; Llamas, B.\nScore: 20.4, Published: 2023-07-05 DOI: 10.1101/2023.07.04.547445\nIn-solution hybridisation enrichment of genetic variation is a valuable methodology in human paleogenomics. It allows enrichment of endogenous DNA by targeting genetic markers that are comparable between sequencing libraries. Many studies have used the 1240k reagent--which enriches 1,237,207 genome-wide SNPs--since 2015, though access was restricted. In 2021, Twist Biosciences and Daicel Arbor Biosciences independently released commercial kits that enabled all researchers to perform enrichments for the same 1240k SNPs. We used the Daicel Arbor Biosciences Prime Plus kit to enrich 268 ancient samples from four continents. We identified a systematic assay bias that increases genetic similarity between enriched samples and that cannot be explained by batch effects. We present the impact of the bias on population genetics inferences (e.g., Principal Components Analysis, [f]-statistics) and genetic relatedness (READ). We compare the Prime Plus bias to that previously reported of the legacy 1240k enrichment assay. In [f]-statistics, we find that all Prime-Plus-generated data exhibit artefactual excess shared drift, such that within continent relationships cannot be correctly determined. In PCA, the first PCs may capture the assay bias rather than the underlying genetic structure. The bias is more subtle in READ, though interpretation of the results can still be misleading in specific contexts. We expect the bias may affect analyses we have not yet tested. Our observations support previously reported concerns for the integration of different data types in paleogenomics. We also caution that technological solutions to generate 1240k data necessitate a thorough validation process before their adoption in the paleogenomic community.\nComprehensive Assessment of Eleven de novo HiFi Assemblers on Complex Eukaryotic Genomes and Metagenomes\nAuthors: Yu, W.; Luo, H.; Yang, J.; Zhang, S.; Jiang, H.; Zhao, X.; Hui, X.; Sun, D.; Li, L.; Wei, X.-q.; Lonardi, S.; Pan, W.\nScore: 17.1, Published: 2023-06-29 DOI: 10.1101/2023.06.29.546998\nBackgroundPacific Bioscience HiFi sequencing technology generates long reads (\u003e10 kbp) with very high accuracy (less than 0.01% sequencing error). While several de novo assembly tools are available for HiFi reads, there are no comprehensive studies on the evaluation of these assemblers. ResultsWe evaluated the performance of eleven de novo HiFi assemblers on (i) real data for three eukaryotic genomes, (ii) 34 synthetic datasets with different ploidy, sequencing coverage levels, heterozygosity rates and sequencing error rates, (iii) one real metagenomic dataset, and (iv) five synthetic metagenomic datasets with different composition abundance and heterozygosity rates. The nine assemblers were evaluated using QUAST (Quality Assessment Tool) and BUSCO (Benchmarking Universal Single-Copy Ortholog). We also used several additional criteria, namely, completion rate, single-copy completion rate, duplicated completion rate, average proportion of largest category, average distance difference, quality value, run-time and memory utilization. On complex eukaryotic genomes, Hifiasm had a clear advantage over the other assemblers in all tested experiments. On synthetic datasets, Hifiasm, HiCanu, and HiFlye performed equally well. Shasta and Peregrine had good performance across varying ploidy, but required high computational resources. On metagenomic datasets, Hifiasm-meta demonstrated a clear advantage over other assemblers. ConclusionWe carried out a comprehensive benchmarking study of commonly used assemblers on complex eukaryotic genomes and metagenomes. Our study will help the research community to choose the most appropriate assembler for their data and identify possible improvements in assembly algorithms.\nNon-catalytic function of PRC2 in the control of small RNA dynamics during programmed genome elimination in Paramecium\nAuthors: Miro-Pina, C.; Arnaiz, O.; de Vanssay, A.; Frapporti, A.; Charmant, O.; Humbert, A.; Lhuillier-Akakpo, M.; Duharcourt, S.\nScore: 13.4, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547679\nTo limit transposable element (TE) mobilization, most eukaryotes have evolved small RNAs to silence TE activity via homology-dependent mechanisms. Small RNAs, 20-30 nucleotides in length, bind to PIWI proteins and guide them to nascent transcripts by sequence complementarity, triggering the recruitment of histone methyltransferase enzymes on chromatin to repress the transcriptional activity of TEs and other repeats. In the ciliate Paramecium tetraurelia, 25-nt scnRNAs corresponding to TEs recruit Polycomb Repressive Complex 2 (PRC2), and trigger their elimination during the formation of the somatic nucleus. Here, we sequenced sRNAs during the entire sexual cycle with unprecedented precision. Our data confirmed that scnRNAs are produced from the entire germline genome, from TEs and non-TE sequences, during meiosis. Non-TE scnRNAs are selectively degraded, which results in the specific selection of TE-scnRNAs. We provide important mechanistic insight into the scnRNA selection pathway by identifying PRC2 and its cofactors as essential for the selective degradation of non-TE-scnRNAs. Our findings reveal a new mechanism for PRC2 that involves a non-methyltransferase function for regulating small RNA dynamics during development.\nMultimodal hierarchical classification of CITE-seq data delineates immune cell states across lineages and tissues\nAuthors: Caron, D. P.; Specht, W. L.; Chen, D.; Wells, S. B.; Szabo, P. A.; Jensen, I. J.; Farber, D. L.; Sims, P. A.\nScore: 9.4, Published: 2023-07-07 DOI: 10.1101/2023.07.06.547944\nSingle-cell RNA sequencing (scRNA-seq) is invaluable for profiling cellular heterogeneity and dissecting transcriptional states, but transcriptomic profiles do not always delineate subsets defined by surface proteins, as in cells of the immune system. Cellular Indexing of Transcriptomes and Epitopes (CITE-seq) enables simultaneous profiling of single-cell transcriptomes and surface proteomes; however, accurate cell type annotation requires a classifier that integrates this multimodal data. Here, we describe MultiModal Classifier Hierarchy (MMoCHi), a marker-based approach for classification, reconciling gene and protein expression without reliance on reference atlases. We benchmark MMoCHi using sorted T lymphocyte subsets and annotate a cross-tissue human immune cell dataset. MMoCHi outperforms leading transcriptome-based classifiers and multimodal unsupervised clustering in its ability to identify immune cell subsets that are not readily resolved and to reveal novel subset markers. MMoCHi is designed for adaptability and can integrate CITE-seq annotation of cell types and developmental states across diverse lineages, tissues, or individuals.\nHuman Immune Cell Epigenomic Signatures in Response to Infectious Diseases and Chemical Exposures\nAuthors: Wang, W.; Hariharan, M.; Bartlett, A.; Barragan, C.; Castanon, R.; Rothenberg, V.; Song, H.; Nery, J.; Aldridge, A.; Altshul, J.; Kenworthy, M.; Ding, W.; Liu, H.; Tian, W.; Zhou, J.; Chen, H.; Wei, B.; Gunduz, I. B.; Norell, T.; Broderick, T. J.; McClain, M.; Satterwhite, L.; Burke, T.; Petzold, E.; Shen, X.; Woods, C.; Fowler, V. G.; Ruffin, F.; Panuwet, P.; Barr, D. B.; Beare, J. L.; Smith, A. K.; Spurbeck, R. R.; Vangeti, S.; Ramos, I.; Nudelman, G.; Sealfon, S. C.; Castellino, F.; Walley, A. M.; Evans, T.; Muller, F.; Greenleaf, W. J.; Ecker, J. R.\nScore: 12.7, Published: 2023-06-30 DOI: 10.1101/2023.06.29.546792\nVariations in DNA methylation patterns in human tissues have been linked to various environmental exposures and infections. Here, we identified the DNA methylation signatures associated with multiple exposures in nine major immune cell types derived from peripheral blood mononuclear cells (PBMCs) at single-cell resolution. We performed methylome sequencing on 111,180 immune cells obtained from 112 individuals who were exposed to different viruses, bacteria, or chemicals. Our analysis revealed 790,662 differentially methylated regions (DMRs) associated with these exposures, which are mostly individual CpG sites. Additionally, we integrated methylation and ATAC-seq data from same samples and found strong correlations between the two modalities. However, the epigenomic remodeling in these two modalities are complementary. Finally, we identified the minimum set of DMRs that can predict exposures. Overall, our study provides the first comprehensive dataset of single immune cell methylation profiles, along with unique methylation biomarkers for various biological and chemical exposures.\nDeep immunophenotyping reveals circulating activated lymphocytes in individuals at risk for rheumatoid arthritis\nAuthors: Inamo, J.; Keegan, J.; Griffith, A.; Ghosh, T.; Horisberger, A.; Howard, K.; Pulford, J.; Murzin, E.; Hancock, B.; Jonsson, A. H.; Seifert, J.; Feser, M. L.; Norris, J. M.; Cao, Y.; Apruzzese, W.; Bridges, S. L.; Bykerk, V.; Goodman, S.; Donlin, L.; Firestein, G. S.; Perlman, H.; Bathon, J. M.; Hughes, L. B.; Tabechian, D.; Filer, A.; Pitzalis, C.; H. Anolik, J.; Moreland, L.; Guthridge, J. M.; James, J. A.; Brenner, M. B.; Raychaudhuri, S.; Sparks, J. A.; The Accelerating Medicines Partnership RA/SLE Network, ; Holers, V. M.; Deane, K. D.; Lederer, J. A.; Rao, D. A.; Zhang, F.\nScore: 9.1, Published: 2023-07-04 DOI: 10.1101/2023.07.03.547507\nRheumatoid arthritis (RA) is a systemic autoimmune disease with currently no universally highly effective prevention strategies. Identifying pathogenic immune phenotypes in At-Risk populations prior to clinical disease onset is crucial to establishing effective prevention strategies. Here, we applied mass cytometry to deeply characterize the immunophenotypes in blood from At-Risk individuals identified through the presence of serum antibodies to citrullinated protein antigens (ACPA) and/or first-degree relative (FDR) status (n=52), as compared to established RA (n=67), and healthy controls (n=48). We identified significant cell expansions in At-Risk individuals compared with controls, including CCR2+CD4+ T cells, T peripheral helper (Tph) cells, type 1 T helper cells, and CXCR5+CD8+ T cells. We also found that CD15+ classical monocytes were specifically expanded in ACPA-negative FDRs, and an activated PAX5low naive B cell population was expanded in ACPA-positive FDRs. Further, we developed an \"RA immunophenotype score\" classification method based on the degree of enrichment of cell states relevant to established RA patients. This score significantly distinguished At-Risk individuals from controls. In all, we systematically identified activated lymphocyte phenotypes in At-Risk individuals, along with immunophenotypic differences among both ACPA+ and ACPA-FDR At-Risk subpopulations. Our classification model provides a promising approach for understanding RA pathogenesis with the goal to further improve prevention strategies and identify novel therapeutic targets.\nExhaustive identification of genome-wide binding events of transcriptional regulators with ICEBERG\nAuthors: Nordin, A.; Pagella, P.; Zambanini, G.; Cantu, C.\nScore: 42.4, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547050\nGenome-wide protein interaction assays aspire to map the complete binding pattern of gene regulators. How-ever, common practice relies on replication and high stringency statistics which favor false negatives over false positives, thereby excluding portions of signal which may represent biologically relevant events. Here, we present ICEBERG (Increased Capture of Enrichment By Exhaustive Replicate aGgregation), an experimental and analytical pipeline that harnesses large numbers of CUT\u0026RUN replicates to discover the full set of binding events and chart the line between false positives and false negatives. We employed ICEBERG to map the full set of H3K4me3-marked regulatory regions and {beta}-catenin targets in human colorectal cancer cells. The ICE-BERG datasets allow benchmarking of individual replicates, comparison of the performance of peak calling and replication approaches and expose the arbitrary nature of other strategies to identify reproducible peaks. Instead of a static view of genomic targets, ICEBERG established a spectrum of detection probabilities across the genome for a given factor, underlying the intrinsic dynamicity of its mechanism of action, and permitting to distinguish frequent from rare regulation events. Finally, ICEBERG discovered instances, undetectable with other approaches, that might underlie novel mechanisms of colorectal cancer progression.\n",
  "wordCount" : "2639",
  "inLanguage": "en",
  "datePublished": "2023-07-12T10:41:10Z",
  "dateModified": "2023-07-12T10:41:10Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on July 12, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547175">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547175" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547175">
        <p class="paperTitle">Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547175" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547175" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O&#39;Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J. P.; Pletch, K.; Ruiz-Herrera, A.; Lander, E. S.; Rowley, M. J.; Onuchic, J. N.; Dalen, L.; Marti-Renom, M. A.; Gilbert, M. T. P.; Aiden, E. L.</p>
        <p class="info">Score: 61.7, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547175' target='https://doi.org/10.1101/2023.06.30.547175'> 10.1101/2023.06.30.547175</a></p>
        <p class="abstract">Ancient DNA (aDNA) sequencing analysis typically involves alignment to a modern reference genome assembly from a related species. Since aDNA molecules are fragmentary, these alignments yield information about small-scale differences, but provide no information about larger features such as the chromosome structure of ancient species. We report the genome assembly of a female Late Pleistocene woolly mammoth (Mammuthus primigenius) with twenty-eight chromosome-length scaffolds, generated using mammoth skin preserved in permafrost for roughly 52,000 years. We began by creating a modified Hi-C protocol, dubbed PaleoHi-C, optimized for ancient samples, and using it to map chromatin contacts in a woolly mammoth. Next, we developed &#34;reference-assisted 3D genome assembly,&#34; which begins with a reference genome assembly from a related species, and uses Hi-C and DNA-Seq data from a target species to split, order, orient, and correct sequences on the basis of their 3D proximity, yielding accurate chromosome-length scaffolds for the target species. By means of this reference-assisted 3D genome assembly, PaleoHi-C data reveals the 3D architecture of a woolly mammoth genome, including chromosome territories, compartments, domains, and loops. The active (A) and inactive (B) genome compartments in mammoth skin more closely resemble those observed in Asian elephant skin than the compartmentalization patterns seen in other Asian elephant tissues. Differences in compartmentalization between these skin samples reveal sequences whose transcription was potentially altered in mammoth. We observe a tetradic structure for the inactive X chromosome in mammoth, distinct from the bipartite architecture seen in human and mouse. Generating chromosome-length genome assemblies for two other elephantids (Asian and African elephant), we find that the overall karyotype, and this tetradic Xi structure, are conserved throughout the clade. These results illustrate that cell-type specific epigenetic information can be preserved in ancient samples, in the form of DNA geometry, and that it may be feasible to perform de novo genome assembly of some extinct species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.548007">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.548007" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.548007">
        <p class="paperTitle">Topological stratification of continuous genetic variation in large biobanks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.548007" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.548007" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Diaz-Papkovich, A.; Zabad, S.; Ben-Eghan, C.; Anderson-Trocme, L.; Femerling, G.; Nathan, V.; Patel, J.; Gravel, S.</p>
        <p class="info">Score: 23.6, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.548007' target='https://doi.org/10.1101/2023.07.06.548007'> 10.1101/2023.07.06.548007</a></p>
        <p class="abstract">Biobanks now contain genetic data from millions of individuals. Dimension-ality reduction, visualization and stratification are standard when exploring data at these scales; while efficient and tractable methods exist for the first two, stratification remains challenging because of uncertainty about sources of population structure. In practice, stratification is commonly performed by drawing shapes around dimensionally reduced data or assuming populations have a &#34;type&#34; genome. We propose a method of stratifying data with topo-logical analysis that is fast, easy to implement, and integrates with existing pipelines. The approach is robust to the presence of sub-populations of vary-ing sizes and wide ranges of population structure patterns. We demonstrate its effectiveness on genotypes from three biobanks and illustrate how topolog-ical genetic strata can help us understand structure within biobanks, evaluate distributions of genotypic and phenotypic data, examine polygenic score trans-ferability, identify potential influential alleles, and perform quality control.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548391">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548391" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548391">
        <p class="paperTitle">A method to correct for local alterations in DNA copy number that bias functional genomics assays applied to antibiotic-treated bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548391" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548391" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sullivan, G. J.; Barquist, L.; Cain, A. K.</p>
        <p class="info">Score: 22.7, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548391' target='https://doi.org/10.1101/2023.07.10.548391'> 10.1101/2023.07.10.548391</a></p>
        <p class="abstract">Functional genomics techniques, such as transposon insertion sequencing and RNA-sequencing, are key to studying relative differences in mutant fitness or gene expression under selective conditions. However, certain stress conditions, mutations, or antibiotics can directly interfere with DNA synthesis, resulting in systematic changes in local DNA copy number along the chromosome. This can lead to artefacts in sequencing-based functional genomics data when comparing antibiotic treatment to an unstressed control, with relative differences in gene-wise read counts being the result of alterations in chromosomal replication dynamics rather than selection or direct gene regulation. We term this artefact &#39;chromosomal location bias&#39; and implement a principled statistical approach to correct for it by calculating local normalization factors along the chromosome. These normalization factors are then directly incorporated in statistical analyses using standard RNA-sequencing analysis methods without modifying the read counts themselves, preserving important information about the mean-variance relationship in the data. We illustrate the utility of this approach by generating and analysing a ciprofloxacin-treated transposon insertion sequencing dataset in Escherichia coli as a case study. We show that ciprofloxacin treatment generates chromosomal location bias in the resulting data, and we further demonstrate that failing to correct for this bias leads to false predictions of mutant drug sensitivity as measured by minimum inhibitory concentrations. We have developed an R package and user-friendly graphical Shiny application, ChromoCorrect, that detects and corrects for chromosomal bias in read count data, enabling the application of functional genomics technologies to the study of antibiotic stress.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547445">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547445" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547445">
        <p class="paperTitle">Allelic bias when performing in-solution enrichment of ancient human DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547445" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547445" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davidson, R.; Williams, M. P.; Roca-Rada, X.; Kassadjikova, K.; Tobler, R.; Fehren-Schmitz, L.; Llamas, B.</p>
        <p class="info">Score: 20.4, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547445' target='https://doi.org/10.1101/2023.07.04.547445'> 10.1101/2023.07.04.547445</a></p>
        <p class="abstract">In-solution hybridisation enrichment of genetic variation is a valuable methodology in human paleogenomics. It allows enrichment of endogenous DNA by targeting genetic markers that are comparable between sequencing libraries. Many studies have used the 1240k reagent--which enriches 1,237,207 genome-wide SNPs--since 2015, though access was restricted. In 2021, Twist Biosciences and Daicel Arbor Biosciences independently released commercial kits that enabled all researchers to perform enrichments for the same 1240k SNPs. We used the Daicel Arbor Biosciences Prime Plus kit to enrich 268 ancient samples from four continents. We identified a systematic assay bias that increases genetic similarity between enriched samples and that cannot be explained by batch effects. We present the impact of the bias on population genetics inferences (e.g., Principal Components Analysis, [f]-statistics) and genetic relatedness (READ). We compare the Prime Plus bias to that previously reported of the legacy 1240k enrichment assay. In [f]-statistics, we find that all Prime-Plus-generated data exhibit artefactual excess shared drift, such that within continent relationships cannot be correctly determined. In PCA, the first PCs may capture the assay bias rather than the underlying genetic structure. The bias is more subtle in READ, though interpretation of the results can still be misleading in specific contexts. We expect the bias may affect analyses we have not yet tested. Our observations support previously reported concerns for the integration of different data types in paleogenomics. We also caution that technological solutions to generate 1240k data necessitate a thorough validation process before their adoption in the paleogenomic community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546998">
        <p class="paperTitle">Comprehensive Assessment of Eleven de novo HiFi Assemblers on Complex Eukaryotic Genomes and Metagenomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, W.; Luo, H.; Yang, J.; Zhang, S.; Jiang, H.; Zhao, X.; Hui, X.; Sun, D.; Li, L.; Wei, X.-q.; Lonardi, S.; Pan, W.</p>
        <p class="info">Score: 17.1, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546998' target='https://doi.org/10.1101/2023.06.29.546998'> 10.1101/2023.06.29.546998</a></p>
        <p class="abstract">BackgroundPacific Bioscience HiFi sequencing technology generates long reads (&gt;10 kbp) with very high accuracy (less than 0.01% sequencing error). While several de novo assembly tools are available for HiFi reads, there are no comprehensive studies on the evaluation of these assemblers.

ResultsWe evaluated the performance of eleven de novo HiFi assemblers on (i) real data for three eukaryotic genomes, (ii) 34 synthetic datasets with different ploidy, sequencing coverage levels, heterozygosity rates and sequencing error rates, (iii) one real metagenomic dataset, and (iv) five synthetic metagenomic datasets with different composition abundance and heterozygosity rates. The nine assemblers were evaluated using QUAST (Quality Assessment Tool) and BUSCO (Benchmarking Universal Single-Copy Ortholog). We also used several additional criteria, namely, completion rate, single-copy completion rate, duplicated completion rate, average proportion of largest category, average distance difference, quality value, run-time and memory utilization. On complex eukaryotic genomes, Hifiasm had a clear advantage over the other assemblers in all tested experiments. On synthetic datasets, Hifiasm, HiCanu, and HiFlye performed equally well. Shasta and Peregrine had good performance across varying ploidy, but required high computational resources. On metagenomic datasets, Hifiasm-meta demonstrated a clear advantage over other assemblers.

ConclusionWe carried out a comprehensive benchmarking study of commonly used assemblers on complex eukaryotic genomes and metagenomes. Our study will help the research community to choose the most appropriate assembler for their data and identify possible improvements in assembly algorithms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547679">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547679" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547679">
        <p class="paperTitle">Non-catalytic function of PRC2 in the control of small RNA dynamics during programmed genome elimination in Paramecium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547679" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547679" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miro-Pina, C.; Arnaiz, O.; de Vanssay, A.; Frapporti, A.; Charmant, O.; Humbert, A.; Lhuillier-Akakpo, M.; Duharcourt, S.</p>
        <p class="info">Score: 13.4, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547679' target='https://doi.org/10.1101/2023.07.04.547679'> 10.1101/2023.07.04.547679</a></p>
        <p class="abstract">To limit transposable element (TE) mobilization, most eukaryotes have evolved small RNAs to silence TE activity via homology-dependent mechanisms. Small RNAs, 20-30 nucleotides in length, bind to PIWI proteins and guide them to nascent transcripts by sequence complementarity, triggering the recruitment of histone methyltransferase enzymes on chromatin to repress the transcriptional activity of TEs and other repeats. In the ciliate Paramecium tetraurelia, 25-nt scnRNAs corresponding to TEs recruit Polycomb Repressive Complex 2 (PRC2), and trigger their elimination during the formation of the somatic nucleus. Here, we sequenced sRNAs during the entire sexual cycle with unprecedented precision. Our data confirmed that scnRNAs are produced from the entire germline genome, from TEs and non-TE sequences, during meiosis. Non-TE scnRNAs are selectively degraded, which results in the specific selection of TE-scnRNAs. We provide important mechanistic insight into the scnRNA selection pathway by identifying PRC2 and its cofactors as essential for the selective degradation of non-TE-scnRNAs. Our findings reveal a new mechanism for PRC2 that involves a non-methyltransferase function for regulating small RNA dynamics during development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547944">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547944" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547944">
        <p class="paperTitle">Multimodal hierarchical classification of CITE-seq data delineates immune cell states across lineages and tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547944" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547944" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caron, D. P.; Specht, W. L.; Chen, D.; Wells, S. B.; Szabo, P. A.; Jensen, I. J.; Farber, D. L.; Sims, P. A.</p>
        <p class="info">Score: 9.4, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547944' target='https://doi.org/10.1101/2023.07.06.547944'> 10.1101/2023.07.06.547944</a></p>
        <p class="abstract">Single-cell RNA sequencing (scRNA-seq) is invaluable for profiling cellular heterogeneity and dissecting transcriptional states, but transcriptomic profiles do not always delineate subsets defined by surface proteins, as in cells of the immune system. Cellular Indexing of Transcriptomes and Epitopes (CITE-seq) enables simultaneous profiling of single-cell transcriptomes and surface proteomes; however, accurate cell type annotation requires a classifier that integrates this multimodal data. Here, we describe MultiModal Classifier Hierarchy (MMoCHi), a marker-based approach for classification, reconciling gene and protein expression without reliance on reference atlases. We benchmark MMoCHi using sorted T lymphocyte subsets and annotate a cross-tissue human immune cell dataset. MMoCHi outperforms leading transcriptome-based classifiers and multimodal unsupervised clustering in its ability to identify immune cell subsets that are not readily resolved and to reveal novel subset markers. MMoCHi is designed for adaptability and can integrate CITE-seq annotation of cell types and developmental states across diverse lineages, tissues, or individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546792">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546792" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546792">
        <p class="paperTitle">Human Immune Cell Epigenomic Signatures in Response to Infectious Diseases and Chemical Exposures</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546792" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546792" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, W.; Hariharan, M.; Bartlett, A.; Barragan, C.; Castanon, R.; Rothenberg, V.; Song, H.; Nery, J.; Aldridge, A.; Altshul, J.; Kenworthy, M.; Ding, W.; Liu, H.; Tian, W.; Zhou, J.; Chen, H.; Wei, B.; Gunduz, I. B.; Norell, T.; Broderick, T. J.; McClain, M.; Satterwhite, L.; Burke, T.; Petzold, E.; Shen, X.; Woods, C.; Fowler, V. G.; Ruffin, F.; Panuwet, P.; Barr, D. B.; Beare, J. L.; Smith, A. K.; Spurbeck, R. R.; Vangeti, S.; Ramos, I.; Nudelman, G.; Sealfon, S. C.; Castellino, F.; Walley, A. M.; Evans, T.; Muller, F.; Greenleaf, W. J.; Ecker, J. R.</p>
        <p class="info">Score: 12.7, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546792' target='https://doi.org/10.1101/2023.06.29.546792'> 10.1101/2023.06.29.546792</a></p>
        <p class="abstract">Variations in DNA methylation patterns in human tissues have been linked to various environmental exposures and infections. Here, we identified the DNA methylation signatures associated with multiple exposures in nine major immune cell types derived from peripheral blood mononuclear cells (PBMCs) at single-cell resolution. We performed methylome sequencing on 111,180 immune cells obtained from 112 individuals who were exposed to different viruses, bacteria, or chemicals. Our analysis revealed 790,662 differentially methylated regions (DMRs) associated with these exposures, which are mostly individual CpG sites. Additionally, we integrated methylation and ATAC-seq data from same samples and found strong correlations between the two modalities. However, the epigenomic remodeling in these two modalities are complementary. Finally, we identified the minimum set of DMRs that can predict exposures. Overall, our study provides the first comprehensive dataset of single immune cell methylation profiles, along with unique methylation biomarkers for various biological and chemical exposures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547507">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547507" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547507">
        <p class="paperTitle">Deep immunophenotyping reveals circulating activated lymphocytes in individuals at risk for rheumatoid arthritis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547507" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547507" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Inamo, J.; Keegan, J.; Griffith, A.; Ghosh, T.; Horisberger, A.; Howard, K.; Pulford, J.; Murzin, E.; Hancock, B.; Jonsson, A. H.; Seifert, J.; Feser, M. L.; Norris, J. M.; Cao, Y.; Apruzzese, W.; Bridges, S. L.; Bykerk, V.; Goodman, S.; Donlin, L.; Firestein, G. S.; Perlman, H.; Bathon, J. M.; Hughes, L. B.; Tabechian, D.; Filer, A.; Pitzalis, C.; H. Anolik, J.; Moreland, L.; Guthridge, J. M.; James, J. A.; Brenner, M. B.; Raychaudhuri, S.; Sparks, J. A.; The Accelerating Medicines Partnership RA/SLE Network,  ; Holers, V. M.; Deane, K. D.; Lederer, J. A.; Rao, D. A.; Zhang, F.</p>
        <p class="info">Score: 9.1, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547507' target='https://doi.org/10.1101/2023.07.03.547507'> 10.1101/2023.07.03.547507</a></p>
        <p class="abstract">Rheumatoid arthritis (RA) is a systemic autoimmune disease with currently no universally highly effective prevention strategies. Identifying pathogenic immune phenotypes in  At-Risk populations prior to clinical disease onset is crucial to establishing effective prevention strategies. Here, we applied mass cytometry to deeply characterize the immunophenotypes in blood from At-Risk individuals identified through the presence of serum antibodies to citrullinated protein antigens (ACPA) and/or first-degree relative (FDR) status (n=52), as compared to established RA (n=67), and healthy controls (n=48). We identified significant cell expansions in At-Risk individuals compared with controls, including CCR2&#43;CD4&#43; T cells, T peripheral helper (Tph) cells, type 1 T helper cells, and CXCR5&#43;CD8&#43; T cells. We also found that CD15&#43; classical monocytes were specifically expanded in ACPA-negative FDRs, and an activated PAX5low naive B cell population was expanded in ACPA-positive FDRs. Further, we developed an &#34;RA immunophenotype score&#34; classification method based on the degree of enrichment of cell states relevant to established RA patients. This score significantly distinguished At-Risk individuals from controls. In all, we systematically identified activated lymphocyte phenotypes in At-Risk individuals, along with immunophenotypic differences among both ACPA&#43; and ACPA-FDR At-Risk subpopulations. Our classification model provides a promising approach for understanding RA pathogenesis with the goal to further improve prevention strategies and identify novel therapeutic targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547050">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547050" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547050">
        <p class="paperTitle">Exhaustive identification of genome-wide binding events of transcriptional regulators with ICEBERG</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547050" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547050" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nordin, A.; Pagella, P.; Zambanini, G.; Cantu, C.</p>
        <p class="info">Score: 42.4, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547050' target='https://doi.org/10.1101/2023.06.29.547050'> 10.1101/2023.06.29.547050</a></p>
        <p class="abstract">Genome-wide protein interaction assays aspire to map the complete binding pattern of gene regulators. How-ever, common practice relies on replication and high stringency statistics which favor false negatives over false positives, thereby excluding portions of signal which may represent biologically relevant events. Here, we present ICEBERG (Increased Capture of Enrichment By Exhaustive Replicate aGgregation), an experimental and analytical pipeline that harnesses large numbers of CUT&amp;RUN replicates to discover the full set of binding events and chart the line between false positives and false negatives. We employed ICEBERG to map the full set of H3K4me3-marked regulatory regions and {beta}-catenin targets in human colorectal cancer cells. The ICE-BERG datasets allow benchmarking of individual replicates, comparison of the performance of peak calling and replication approaches and expose the arbitrary nature of other strategies to identify reproducible peaks. Instead of a static view of genomic targets, ICEBERG established a spectrum of detection probabilities across the genome for a given factor, underlying the intrinsic dynamicity of its mechanism of action, and permitting to distinguish frequent from rare regulation events. Finally, ICEBERG discovered instances, undetectable with other approaches, that might underlie novel mechanisms of colorectal cancer progression.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
