<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample
Authors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O&#39;Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample
Authors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O&#39;Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-16T10:39:40+00:00" />
<meta property="article:modified_time" content="2023-07-16T10:39:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample
Authors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O&#39;Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample\nAuthors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O\u0026#39;Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J.",
  "keywords": [
    
  ],
  "articleBody": " Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample\nAuthors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O'Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J. P.; Pletch, K.; Ruiz-Herrera, A.; Lander, E. S.; Rowley, M. J.; Onuchic, J. N.; Dalen, L.; Marti-Renom, M. A.; Gilbert, M. T. P.; Aiden, E. L.\nScore: 63.3, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547175\nAncient DNA (aDNA) sequencing analysis typically involves alignment to a modern reference genome assembly from a related species. Since aDNA molecules are fragmentary, these alignments yield information about small-scale differences, but provide no information about larger features such as the chromosome structure of ancient species. We report the genome assembly of a female Late Pleistocene woolly mammoth (Mammuthus primigenius) with twenty-eight chromosome-length scaffolds, generated using mammoth skin preserved in permafrost for roughly 52,000 years. We began by creating a modified Hi-C protocol, dubbed PaleoHi-C, optimized for ancient samples, and using it to map chromatin contacts in a woolly mammoth. Next, we developed \"reference-assisted 3D genome assembly,\" which begins with a reference genome assembly from a related species, and uses Hi-C and DNA-Seq data from a target species to split, order, orient, and correct sequences on the basis of their 3D proximity, yielding accurate chromosome-length scaffolds for the target species. By means of this reference-assisted 3D genome assembly, PaleoHi-C data reveals the 3D architecture of a woolly mammoth genome, including chromosome territories, compartments, domains, and loops. The active (A) and inactive (B) genome compartments in mammoth skin more closely resemble those observed in Asian elephant skin than the compartmentalization patterns seen in other Asian elephant tissues. Differences in compartmentalization between these skin samples reveal sequences whose transcription was potentially altered in mammoth. We observe a tetradic structure for the inactive X chromosome in mammoth, distinct from the bipartite architecture seen in human and mouse. Generating chromosome-length genome assemblies for two other elephantids (Asian and African elephant), we find that the overall karyotype, and this tetradic Xi structure, are conserved throughout the clade. These results illustrate that cell-type specific epigenetic information can be preserved in ancient samples, in the form of DNA geometry, and that it may be feasible to perform de novo genome assembly of some extinct species.\nTopological stratification of continuous genetic variation in large biobanks\nAuthors: Diaz-Papkovich, A.; Zabad, S.; Ben-Eghan, C.; Anderson-Trocme, L.; Femerling, G.; Nathan, V.; Patel, J.; Gravel, S.\nScore: 29.8, Published: 2023-07-07 DOI: 10.1101/2023.07.06.548007\nBiobanks now contain genetic data from millions of individuals. Dimension-ality reduction, visualization and stratification are standard when exploring data at these scales; while efficient and tractable methods exist for the first two, stratification remains challenging because of uncertainty about sources of population structure. In practice, stratification is commonly performed by drawing shapes around dimensionally reduced data or assuming populations have a \"type\" genome. We propose a method of stratifying data with topo-logical analysis that is fast, easy to implement, and integrates with existing pipelines. The approach is robust to the presence of sub-populations of vary-ing sizes and wide ranges of population structure patterns. We demonstrate its effectiveness on genotypes from three biobanks and illustrate how topolog-ical genetic strata can help us understand structure within biobanks, evaluate distributions of genotypic and phenotypic data, examine polygenic score trans-ferability, identify potential influential alleles, and perform quality control.\nA method to correct for local alterations in DNA copy number that bias functional genomics assays applied to antibiotic-treated bacteria\nAuthors: Sullivan, G. J.; Barquist, L.; Cain, A. K.\nScore: 25.2, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548391\nFunctional genomics techniques, such as transposon insertion sequencing and RNA sequencing, are key to studying relative differences in mutant fitness or gene expression under selective conditions. However, certain stress conditions, mutations, or antibiotics can directly interfere with DNA synthesis, resulting in systematic changes in local DNA copy number along the chromosome. This can lead to artefacts in sequencing-based functional genomics data when comparing antibiotic treatment to an unstressed control, with relative differences in gene-wise read counts being the result of alterations in chromosomal replication dynamics rather than selection or direct gene regulation. We term this artefact chromosomal location bias and implement a principled statistical approach to correct for it by calculating local normalization factors along the chromosome. These normalization factors are then directly incorporated in statistical analyses using standard RNA-sequencing analysis methods without modifying the read counts themselves, preserving important information about the mean-variance relationship in the data. We illustrate the utility of this approach by generating and analysing a ciprofloxacin-treated transposon insertion sequencing dataset in Escherichia coli as a case study. We show that ciprofloxacin treatment generates chromosomal location bias in the resulting data, and we further demonstrate that failing to correct for this bias leads to false predictions of mutant drug sensitivity as measured by minimum inhibitory concentrations. We have developed an R package and user-friendly graphical Shiny application, ChromoCorrect, that detects and corrects for chromosomal bias in read count data, enabling the application of functional genomics technologies to the study of antibiotic stress. IMPORTANCEAltered gene dosage due to changes in DNA replication has been observed under a variety of stresses with a variety of experimental techniques. However, the implications of changes in gene dosage for sequencing-based functional genomics assays are rarely considered. We present a statistically principled approach to correcting for the effect of changes in gene dosage, enabling testing for differences in the fitness effects or regulation of individual genes in the presence of confounding differences in DNA copy number. We show that failing to correct for these effects can lead to incorrect predictions of resistance phenotype when applying functional genomics assays to investigate antibiotic stress, and we provide a user-friendly application to detect and correct for changes in DNA copy number.\nHigh throughput PRIME editing screens identify functional DNA variants in the human genome\nAuthors: Ren, X.; Yang, H.; Nierenberg, J. L.; Sun, Y.; Chen, J.; Beaman, C.; Pham, T.; Nobuhara, M.; Takagi, M. A.; Narayan, V.; Li, Y.; Ziv, E.; Shen, Y.\nScore: 13.6, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548736\nDespite tremendous progress in detecting DNA variants associated with human disease, interpreting their functional impact in a high-throughput and base-pair resolution manner remains challenging. Here, we develop a novel pooled prime editing screen method, PRIME, which can be applied to characterize thousands of coding and non-coding variants in a single experiment with high reproducibility. To showcase its applications, we first identified essential nucleotides for a 716 bp MYC enhancer via PRIME-mediated saturation mutagenesis. Next, we applied PRIME to functionally characterize 1,304 non-coding variants associated with breast cancer and 3,699 variants from ClinVar. We discovered that 103 non-coding variants and 156 variants of uncertain significance are functional via affecting cell fitness. Collectively, we demonstrate PRIME capable of characterizing genetic variants at base-pair resolution and scale, advancing accurate genome annotation for disease risk prediction, diagnosis, and therapeutic target identification.\nDetecting differential transcript usage in complex diseases with SPIT\nAuthors: Erdogdu, B.; Varabyou, A.; Hicks, S. C.; Salzberg, S. L.; Pertea, M.\nScore: 10.7, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548289\nDifferential transcript usage (DTU) plays a crucial role in determining how gene expression differs among cells, tissues, and different developmental stages, thereby contributing to the complexity and diversity of biological systems. In abnormal cells, it can also lead to deficiencies in protein function, potentially leading to pathogenesis of diseases. Detecting such events for single-gene genetic traits is relatively uncomplicated; however, the heterogeneity of populations with complex diseases presents an intricate challenge due to the presence of diverse causal events and undetermined subtypes. SPIT is the first statistical tool that quantifies the heterogeneity in transcript usage within a population and identifies predominant subgroups along with their distinctive sets of DTU events. We provide comprehensive assessments of SPITs methodology in both single-gene and complex traits and report the results of applying SPIT to analyze brain samples from individuals with schizophrenia. Our analysis reveals previously unreported DTU events in six candidate genes.\nComprehensive Assessment of Eleven de novo HiFi Assemblers on Complex Eukaryotic Genomes and Metagenomes\nAuthors: Yu, W.; Luo, H.; Yang, J.; Zhang, S.; Jiang, H.; Zhao, X.; Hui, X.; Sun, D.; Li, L.; Wei, X.-q.; Lonardi, S.; Pan, W.\nScore: 26.4, Published: 2023-06-29 DOI: 10.1101/2023.06.29.546998\nBackgroundPacific Bioscience HiFi sequencing technology generates long reads (\u003e10 kbp) with very high accuracy (less than 0.01% sequencing error). While several de novo assembly tools are available for HiFi reads, there are no comprehensive studies on the evaluation of these assemblers. ResultsWe evaluated the performance of eleven de novo HiFi assemblers on (i) real data for three eukaryotic genomes, (ii) 34 synthetic datasets with different ploidy, sequencing coverage levels, heterozygosity rates and sequencing error rates, (iii) one real metagenomic dataset, and (iv) five synthetic metagenomic datasets with different composition abundance and heterozygosity rates. The nine assemblers were evaluated using QUAST (Quality Assessment Tool) and BUSCO (Benchmarking Universal Single-Copy Ortholog). We also used several additional criteria, namely, completion rate, single-copy completion rate, duplicated completion rate, average proportion of largest category, average distance difference, quality value, run-time and memory utilization. On complex eukaryotic genomes, Hifiasm had a clear advantage over the other assemblers in all tested experiments. On synthetic datasets, Hifiasm, HiCanu, and HiFlye performed equally well. Shasta and Peregrine had good performance across varying ploidy, but required high computational resources. On metagenomic datasets, Hifiasm-meta demonstrated a clear advantage over other assemblers. ConclusionWe carried out a comprehensive benchmarking study of commonly used assemblers on complex eukaryotic genomes and metagenomes. Our study will help the research community to choose the most appropriate assembler for their data and identify possible improvements in assembly algorithms.\nHigh quality genome assembly and annotation (v1) of the eukaryotic terrestrial microalga Coccomyxa viridis SAG 216-4\nAuthors: Kraege, A.; Chavarro-Carrero, E.; Guiglielmoni, N.; Schnell, E.; Kirangwa, J.; Heilmann-Heimbach, S.; Becker, K.; Kohrer, K.; WGGC Team, ; DeRGA Community, ; Schiffer, P.; Thomma, B. P.; Rovenich, H.\nScore: 8.7, Published: 2023-07-12 DOI: 10.1101/2023.07.11.548521\nUnicellular green algae of the genus Coccomyxa are recognized for their worldwide distribution and ecological versatility. Most species described to date live in close association with various host species, such as in lichen associations. However, little is known about the molecular mechanisms that drive such symbiotic lifestyles. We generated a high-quality genome assembly for the lichen photobiont Coccomyxa viridis SAG 216-4 (formerly C. mucigena). Using long-read PacBio HiFi and Oxford Nanopore Technologies in combination with chromatin conformation capture (Hi-C) sequencing, we assembled the genome into 21 scaffolds with a total length of 50.9 Mb, an N50 of 2.7 Mb and a BUSCO score of 98.6%. While nineteen scaffolds represent full-length nuclear chromosomes, two additional scaffolds represent the mitochondrial and plastid genomes. Transcriptome-guided gene annotation resulted in the identification of 13,557 protein-coding genes, of which 68% have annotated PFAM domains and 962 are predicted to be secreted.\nCell type and condition specific functional annotation of schizophrenia associated non-coding genetic variants\nAuthors: Rummel, C. K.; Gagliardi, M.; Herholt, A.; Ahmad, R.; Murek, V.; Weigert, L.; Hausruckinger, A.; Maidl, S.; Jimenez-Barron, L.; Trastulla, L.; Hauger, B.; Eder, M.; Rossner, M. J.; Ziller, M. J.\nScore: 8.6, Published: 2023-06-27 DOI: 10.1101/2023.06.27.545266\nSchizophrenia (SCZ) is a highly polygenic disease and genome wide association studies have identified thousands of genetic variants that are statistically associated with this psychiatric disorder. However, our ability to translate these associations into insights on the disease mechanisms has been challenging since the causal genetic variants, their molecular function and their target genes remain largely unknown. In order to address these questions, we established a functional genomics pipeline in combination with induced pluripotent stem cell technology to functionally characterize ~35,000 non-coding genetic variants associated with schizophrenia along with their target genes. This analysis identified a set of 620 (1.7%) single nucleotide polymorphisms as functional on a molecular level in a highly cell type and condition specific fashion. These results provide a high-resolution map of functional variant-gene combinations and offer comprehensive biological insights into the developmental context and stimulation dependent molecular processes modulated by SCZ associated genetic variation.\nHigh diversity of Proteobacteria amid the dominance of a few Aquificae species in the vent-water of a Trans-Himalayan sulfur-borax hot spring\nAuthors: Mondal, N.; Dutta, S.; Chatterjee, S.; Sarkar, J.; Mondal, M.; Roy, C.; Chakraborty, R.; Ghosh, W.\nScore: 5.1, Published: 2023-07-11 DOI: 10.1101/2023.07.10.548480\nLittle is known about the relative abundance and metabolisms of microorganisms present in the vent-waters of Trans-Himalayan hot springs. This study revealed a Bacteria-dominated microbiome in the [~]84{degrees}C vent-water (microbial cell density [~]8.5 x 104 mL-1; live:dead cell ratio 17:10) of a sulfur-borax spring called Lotus Pond, situated at 4436 m, in Puga Valley. Aquificae accounted for 80% of all 16S rRNA-encoding reads within the metagenomic dataset, while only 14% of such reads were attributed to Proteobacteria. Post assembly, only 25% of all protein-coding genes identified were attributable to Aquificae, and [~]41% to Proteobacteria. Annotation of raw metagenomic reads encoding 16S rRNAs, and/or PCR-amplified 16S rRNA genes, identified 163 bacterial genera, out of which 66 had been detected in past investigations of Lotus Ponds vent-water via 16S amplicon sequencing. Among these 66, Fervidobacterium, Halomonas, Hydrogenobacter, Paracoccus, Sulfurihydrogenibium, Tepidimonas, Thermus and Thiofaba (or their close phylogenomic relatives) were presently detected as metagenome-assembled genomes (MAGs). Remarkably, the MAG related to Hydrogenobacter alone accounted for [~]56% of the entire metagenome, even as literature survey showed that only 15 out of the 66 genera consistently present in Lotus Pond have strains growing in the laboratory at \u003e45{degrees}C, reflecting the continued existence of diverse mesophiles in the habitat. Most genes annotated within the MAGs/assembled-metagenome rendered bioenergetic functions (especially oxidative phosphorylation and rTCA/TCA cycles), while many (especially from Aquificae) conferred virulence functions and antibiotics synthesis/resistance. These could help the Aquificae outcompete other native thermophiles, and contain the proliferation of the bacterial mesophiles that infiltrate the ecosystem.\nA reference genome for the long-term kleptoplast-retaining sea slug Elysia crispata morphotype clarki\nAuthors: Eastman, K. E.; Pendleton, A. L.; Shaikh, M. A.; Suttiyut, T.; Ogas, R.; Tomko, P.; Gavelis, G.; Widhalm, J. R.; Wisecaver, J. H.\nScore: 4.3, Published: 2023-07-09 DOI: 10.1101/2023.07.07.548153\nSeveral species of sacoglossan sea slugs possess the incredible ability to sequester chloroplasts from the algae they consume. These photosynthetic animals incorporate stolen chloroplasts, called kleptoplasts, into the epithelial cells of tubules that extend from their digestive tracts throughout their bodies. The mechanism by which these slugs maintain functioning kleptoplasts in the absence of an algal nuclear genome is unknown. Here, we report a draft genome of the saccoglossan slug Elysia crispata morphotype clarki, a morphotype native to the Florida Keys that can retain photosynthetically active kleptoplasts for several months without feeding. We used a combination of Oxford Nanopore Technologies long reads and Illumina short reads to produce a 786 Mbp assembly containing 68,514 predicted protein-coding genes. A phylogenetic analysis found no evidence of horizontal acquisition of genes from algae. We performed gene family and gene expression analyses to identify E. crispata genes unique to kleptoplast-containing slugs that were more highly expressed in fed versus unfed developmental life stages. Consistent with analyses in other kleptoplastic slugs, our investigation suggests that genes encoding lectin carbohydrate-binding proteins and those involved in regulation of reactive oxygen species and immunity may play a role in kleptoplast retention. Lastly, we identified four polyketide synthase genes that could potentially encode proteins producing UV- and oxidation-blocking compounds in slug cell membranes. The genome of E. crispata is a quality resource that provides potential targets for functional analyses and enables further investigation into the evolution and mechanisms of kleptoplasty in animals.\n",
  "wordCount" : "2600",
  "inLanguage": "en",
  "datePublished": "2023-07-16T10:39:40Z",
  "dateModified": "2023-07-16T10:39:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on July 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547175">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547175" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547175">
        <p class="paperTitle">Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547175" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547175" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O&#39;Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J. P.; Pletch, K.; Ruiz-Herrera, A.; Lander, E. S.; Rowley, M. J.; Onuchic, J. N.; Dalen, L.; Marti-Renom, M. A.; Gilbert, M. T. P.; Aiden, E. L.</p>
        <p class="info">Score: 63.3, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547175' target='https://doi.org/10.1101/2023.06.30.547175'> 10.1101/2023.06.30.547175</a></p>
        <p class="abstract">Ancient DNA (aDNA) sequencing analysis typically involves alignment to a modern reference genome assembly from a related species. Since aDNA molecules are fragmentary, these alignments yield information about small-scale differences, but provide no information about larger features such as the chromosome structure of ancient species. We report the genome assembly of a female Late Pleistocene woolly mammoth (Mammuthus primigenius) with twenty-eight chromosome-length scaffolds, generated using mammoth skin preserved in permafrost for roughly 52,000 years. We began by creating a modified Hi-C protocol, dubbed PaleoHi-C, optimized for ancient samples, and using it to map chromatin contacts in a woolly mammoth. Next, we developed &#34;reference-assisted 3D genome assembly,&#34; which begins with a reference genome assembly from a related species, and uses Hi-C and DNA-Seq data from a target species to split, order, orient, and correct sequences on the basis of their 3D proximity, yielding accurate chromosome-length scaffolds for the target species. By means of this reference-assisted 3D genome assembly, PaleoHi-C data reveals the 3D architecture of a woolly mammoth genome, including chromosome territories, compartments, domains, and loops. The active (A) and inactive (B) genome compartments in mammoth skin more closely resemble those observed in Asian elephant skin than the compartmentalization patterns seen in other Asian elephant tissues. Differences in compartmentalization between these skin samples reveal sequences whose transcription was potentially altered in mammoth. We observe a tetradic structure for the inactive X chromosome in mammoth, distinct from the bipartite architecture seen in human and mouse. Generating chromosome-length genome assemblies for two other elephantids (Asian and African elephant), we find that the overall karyotype, and this tetradic Xi structure, are conserved throughout the clade. These results illustrate that cell-type specific epigenetic information can be preserved in ancient samples, in the form of DNA geometry, and that it may be feasible to perform de novo genome assembly of some extinct species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.548007">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.548007" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.548007">
        <p class="paperTitle">Topological stratification of continuous genetic variation in large biobanks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.548007" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.548007" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Diaz-Papkovich, A.; Zabad, S.; Ben-Eghan, C.; Anderson-Trocme, L.; Femerling, G.; Nathan, V.; Patel, J.; Gravel, S.</p>
        <p class="info">Score: 29.8, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.548007' target='https://doi.org/10.1101/2023.07.06.548007'> 10.1101/2023.07.06.548007</a></p>
        <p class="abstract">Biobanks now contain genetic data from millions of individuals. Dimension-ality reduction, visualization and stratification are standard when exploring data at these scales; while efficient and tractable methods exist for the first two, stratification remains challenging because of uncertainty about sources of population structure. In practice, stratification is commonly performed by drawing shapes around dimensionally reduced data or assuming populations have a &#34;type&#34; genome. We propose a method of stratifying data with topo-logical analysis that is fast, easy to implement, and integrates with existing pipelines. The approach is robust to the presence of sub-populations of vary-ing sizes and wide ranges of population structure patterns. We demonstrate its effectiveness on genotypes from three biobanks and illustrate how topolog-ical genetic strata can help us understand structure within biobanks, evaluate distributions of genotypic and phenotypic data, examine polygenic score trans-ferability, identify potential influential alleles, and perform quality control.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548391">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548391" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548391">
        <p class="paperTitle">A method to correct for local alterations in DNA copy number that bias functional genomics assays applied to antibiotic-treated bacteria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548391" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548391" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sullivan, G. J.; Barquist, L.; Cain, A. K.</p>
        <p class="info">Score: 25.2, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548391' target='https://doi.org/10.1101/2023.07.10.548391'> 10.1101/2023.07.10.548391</a></p>
        <p class="abstract">Functional genomics techniques, such as transposon insertion sequencing and RNA sequencing, are key to studying relative differences in mutant fitness or gene expression under selective conditions. However, certain stress conditions, mutations, or antibiotics can directly interfere with DNA synthesis, resulting in systematic changes in local DNA copy number along the chromosome. This can lead to artefacts in sequencing-based functional genomics data when comparing antibiotic treatment to an unstressed control, with relative differences in gene-wise read counts being the result of alterations in chromosomal replication dynamics rather than selection or direct gene regulation. We term this artefact  chromosomal location bias and implement a principled statistical approach to correct for it by calculating local normalization factors along the chromosome. These normalization factors are then directly incorporated in statistical analyses using standard RNA-sequencing analysis methods without modifying the read counts themselves, preserving important information about the mean-variance relationship in the data. We illustrate the utility of this approach by generating and analysing a ciprofloxacin-treated transposon insertion sequencing dataset in Escherichia coli as a case study. We show that ciprofloxacin treatment generates chromosomal location bias in the resulting data, and we further demonstrate that failing to correct for this bias leads to false predictions of mutant drug sensitivity as measured by minimum inhibitory concentrations. We have developed an R package and user-friendly graphical Shiny application, ChromoCorrect, that detects and corrects for chromosomal bias in read count data, enabling the application of functional genomics technologies to the study of antibiotic stress.

IMPORTANCEAltered gene dosage due to changes in DNA replication has been observed under a variety of stresses with a variety of experimental techniques. However, the implications of changes in gene dosage for sequencing-based functional genomics assays are rarely considered. We present a statistically principled approach to correcting for the effect of changes in gene dosage, enabling testing for differences in the fitness effects or regulation of individual genes in the presence of confounding differences in DNA copy number. We show that failing to correct for these effects can lead to incorrect predictions of resistance phenotype when applying functional genomics assays to investigate antibiotic stress, and we provide a user-friendly application to detect and correct for changes in DNA copy number.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548736">
        <p class="paperTitle">High throughput PRIME editing screens identify functional DNA variants in the human genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ren, X.; Yang, H.; Nierenberg, J. L.; Sun, Y.; Chen, J.; Beaman, C.; Pham, T.; Nobuhara, M.; Takagi, M. A.; Narayan, V.; Li, Y.; Ziv, E.; Shen, Y.</p>
        <p class="info">Score: 13.6, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548736' target='https://doi.org/10.1101/2023.07.12.548736'> 10.1101/2023.07.12.548736</a></p>
        <p class="abstract">Despite tremendous progress in detecting DNA variants associated with human disease, interpreting their functional impact in a high-throughput and base-pair resolution manner remains challenging. Here, we develop a novel pooled prime editing screen method, PRIME, which can be applied to characterize thousands of coding and non-coding variants in a single experiment with high reproducibility. To showcase its applications, we first identified essential nucleotides for a 716 bp MYC enhancer via PRIME-mediated saturation mutagenesis. Next, we applied PRIME to functionally characterize 1,304 non-coding variants associated with breast cancer and 3,699 variants from ClinVar. We discovered that 103 non-coding variants and 156 variants of uncertain significance are functional via affecting cell fitness. Collectively, we demonstrate PRIME capable of characterizing genetic variants at base-pair resolution and scale, advancing accurate genome annotation for disease risk prediction, diagnosis, and therapeutic target identification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548289">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548289" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548289">
        <p class="paperTitle">Detecting differential transcript usage in complex diseases with SPIT</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548289" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548289" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Erdogdu, B.; Varabyou, A.; Hicks, S. C.; Salzberg, S. L.; Pertea, M.</p>
        <p class="info">Score: 10.7, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548289' target='https://doi.org/10.1101/2023.07.10.548289'> 10.1101/2023.07.10.548289</a></p>
        <p class="abstract">Differential transcript usage (DTU) plays a crucial role in determining how gene expression differs among cells, tissues, and different developmental stages, thereby contributing to the complexity and diversity of biological systems. In abnormal cells, it can also lead to deficiencies in protein function, potentially leading to pathogenesis of diseases. Detecting such events for single-gene genetic traits is relatively uncomplicated; however, the heterogeneity of populations with complex diseases presents an intricate challenge due to the presence of diverse causal events and undetermined subtypes. SPIT is the first statistical tool that quantifies the heterogeneity in transcript usage within a population and identifies predominant subgroups along with their distinctive sets of DTU events. We provide comprehensive assessments of SPITs methodology in both single-gene and complex traits and report the results of applying SPIT to analyze brain samples from individuals with schizophrenia. Our analysis reveals previously unreported DTU events in six candidate genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546998">
        <p class="paperTitle">Comprehensive Assessment of Eleven de novo HiFi Assemblers on Complex Eukaryotic Genomes and Metagenomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, W.; Luo, H.; Yang, J.; Zhang, S.; Jiang, H.; Zhao, X.; Hui, X.; Sun, D.; Li, L.; Wei, X.-q.; Lonardi, S.; Pan, W.</p>
        <p class="info">Score: 26.4, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546998' target='https://doi.org/10.1101/2023.06.29.546998'> 10.1101/2023.06.29.546998</a></p>
        <p class="abstract">BackgroundPacific Bioscience HiFi sequencing technology generates long reads (&gt;10 kbp) with very high accuracy (less than 0.01% sequencing error). While several de novo assembly tools are available for HiFi reads, there are no comprehensive studies on the evaluation of these assemblers.

ResultsWe evaluated the performance of eleven de novo HiFi assemblers on (i) real data for three eukaryotic genomes, (ii) 34 synthetic datasets with different ploidy, sequencing coverage levels, heterozygosity rates and sequencing error rates, (iii) one real metagenomic dataset, and (iv) five synthetic metagenomic datasets with different composition abundance and heterozygosity rates. The nine assemblers were evaluated using QUAST (Quality Assessment Tool) and BUSCO (Benchmarking Universal Single-Copy Ortholog). We also used several additional criteria, namely, completion rate, single-copy completion rate, duplicated completion rate, average proportion of largest category, average distance difference, quality value, run-time and memory utilization. On complex eukaryotic genomes, Hifiasm had a clear advantage over the other assemblers in all tested experiments. On synthetic datasets, Hifiasm, HiCanu, and HiFlye performed equally well. Shasta and Peregrine had good performance across varying ploidy, but required high computational resources. On metagenomic datasets, Hifiasm-meta demonstrated a clear advantage over other assemblers.

ConclusionWe carried out a comprehensive benchmarking study of commonly used assemblers on complex eukaryotic genomes and metagenomes. Our study will help the research community to choose the most appropriate assembler for their data and identify possible improvements in assembly algorithms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548521">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548521" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548521">
        <p class="paperTitle">High quality genome assembly and annotation (v1) of the eukaryotic terrestrial microalga Coccomyxa viridis SAG 216-4</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548521" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548521" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kraege, A.; Chavarro-Carrero, E.; Guiglielmoni, N.; Schnell, E.; Kirangwa, J.; Heilmann-Heimbach, S.; Becker, K.; Kohrer, K.; WGGC Team,  ; DeRGA Community,  ; Schiffer, P.; Thomma, B. P.; Rovenich, H.</p>
        <p class="info">Score: 8.7, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548521' target='https://doi.org/10.1101/2023.07.11.548521'> 10.1101/2023.07.11.548521</a></p>
        <p class="abstract">Unicellular green algae of the genus Coccomyxa are recognized for their worldwide distribution and ecological versatility. Most species described to date live in close association with various host species, such as in lichen associations. However, little is known about the molecular mechanisms that drive such symbiotic lifestyles. We generated a high-quality genome assembly for the lichen photobiont Coccomyxa viridis SAG 216-4 (formerly C. mucigena). Using long-read PacBio HiFi and Oxford Nanopore Technologies in combination with chromatin conformation capture (Hi-C) sequencing, we assembled the genome into 21 scaffolds with a total length of 50.9 Mb, an N50 of 2.7 Mb and a BUSCO score of 98.6%. While nineteen scaffolds represent full-length nuclear chromosomes, two additional scaffolds represent the mitochondrial and plastid genomes. Transcriptome-guided gene annotation resulted in the identification of 13,557 protein-coding genes, of which 68% have annotated PFAM domains and 962 are predicted to be secreted.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.545266">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.545266" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.545266">
        <p class="paperTitle">Cell type and condition specific functional annotation of schizophrenia associated non-coding genetic variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.545266" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.545266" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rummel, C. K.; Gagliardi, M.; Herholt, A.; Ahmad, R.; Murek, V.; Weigert, L.; Hausruckinger, A.; Maidl, S.; Jimenez-Barron, L.; Trastulla, L.; Hauger, B.; Eder, M.; Rossner, M. J.; Ziller, M. J.</p>
        <p class="info">Score: 8.6, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.545266' target='https://doi.org/10.1101/2023.06.27.545266'> 10.1101/2023.06.27.545266</a></p>
        <p class="abstract">Schizophrenia (SCZ) is a highly polygenic disease and genome wide association studies have identified thousands of genetic variants that are statistically associated with this psychiatric disorder. However, our ability to translate these associations into insights on the disease mechanisms has been challenging since the causal genetic variants, their molecular function and their target genes remain largely unknown. In order to address these questions, we established a functional genomics pipeline in combination with induced pluripotent stem cell technology to functionally characterize ~35,000 non-coding genetic variants associated with schizophrenia along with their target genes. This analysis identified a set of 620 (1.7%) single nucleotide polymorphisms as functional on a molecular level in a highly cell type and condition specific fashion. These results provide a high-resolution map of functional variant-gene combinations and offer comprehensive biological insights into the developmental context and stimulation dependent molecular processes modulated by SCZ associated genetic variation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548480">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548480" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548480">
        <p class="paperTitle">High diversity of Proteobacteria amid the dominance of a few Aquificae species in the vent-water of a Trans-Himalayan sulfur-borax hot spring</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548480" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548480" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mondal, N.; Dutta, S.; Chatterjee, S.; Sarkar, J.; Mondal, M.; Roy, C.; Chakraborty, R.; Ghosh, W.</p>
        <p class="info">Score: 5.1, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548480' target='https://doi.org/10.1101/2023.07.10.548480'> 10.1101/2023.07.10.548480</a></p>
        <p class="abstract">Little is known about the relative abundance and metabolisms of microorganisms present in the vent-waters of Trans-Himalayan hot springs. This study revealed a Bacteria-dominated microbiome in the [~]84{degrees}C vent-water (microbial cell density [~]8.5 x 104 mL-1; live:dead cell ratio 17:10) of a sulfur-borax spring called Lotus Pond, situated at 4436 m, in Puga Valley. Aquificae accounted for 80% of all 16S rRNA-encoding reads within the metagenomic dataset, while only 14% of such reads were attributed to Proteobacteria. Post assembly, only 25% of all protein-coding genes identified were attributable to Aquificae, and [~]41% to Proteobacteria. Annotation of raw metagenomic reads encoding 16S rRNAs, and/or PCR-amplified 16S rRNA genes, identified 163 bacterial genera, out of which 66 had been detected in past investigations of Lotus Ponds vent-water via 16S amplicon sequencing. Among these 66, Fervidobacterium, Halomonas, Hydrogenobacter, Paracoccus, Sulfurihydrogenibium, Tepidimonas, Thermus and Thiofaba (or their close phylogenomic relatives) were presently detected as metagenome-assembled genomes (MAGs). Remarkably, the MAG related to Hydrogenobacter alone accounted for [~]56% of the entire metagenome, even as literature survey showed that only 15 out of the 66 genera consistently present in Lotus Pond have strains growing in the laboratory at &gt;45{degrees}C, reflecting the continued existence of diverse mesophiles in the habitat. Most genes annotated within the MAGs/assembled-metagenome rendered bioenergetic functions (especially oxidative phosphorylation and rTCA/TCA cycles), while many (especially from Aquificae) conferred virulence functions and antibiotics synthesis/resistance. These could help the Aquificae outcompete other native thermophiles, and contain the proliferation of the bacterial mesophiles that infiltrate the ecosystem.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548153">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548153" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548153">
        <p class="paperTitle">A reference genome for the long-term kleptoplast-retaining sea slug Elysia crispata morphotype clarki</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548153" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548153" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eastman, K. E.; Pendleton, A. L.; Shaikh, M. A.; Suttiyut, T.; Ogas, R.; Tomko, P.; Gavelis, G.; Widhalm, J. R.; Wisecaver, J. H.</p>
        <p class="info">Score: 4.3, Published: 2023-07-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548153' target='https://doi.org/10.1101/2023.07.07.548153'> 10.1101/2023.07.07.548153</a></p>
        <p class="abstract">Several species of sacoglossan sea slugs possess the incredible ability to sequester chloroplasts from the algae they consume. These  photosynthetic animals incorporate stolen chloroplasts, called kleptoplasts, into the epithelial cells of tubules that extend from their digestive tracts throughout their bodies. The mechanism by which these slugs maintain functioning kleptoplasts in the absence of an algal nuclear genome is unknown. Here, we report a draft genome of the saccoglossan slug Elysia crispata morphotype clarki, a morphotype native to the Florida Keys that can retain photosynthetically active kleptoplasts for several months without feeding. We used a combination of Oxford Nanopore Technologies long reads and Illumina short reads to produce a 786 Mbp assembly containing 68,514 predicted protein-coding genes. A phylogenetic analysis found no evidence of horizontal acquisition of genes from algae. We performed gene family and gene expression analyses to identify E. crispata genes unique to kleptoplast-containing slugs that were more highly expressed in fed versus unfed developmental life stages. Consistent with analyses in other kleptoplastic slugs, our investigation suggests that genes encoding lectin carbohydrate-binding proteins and those involved in regulation of reactive oxygen species and immunity may play a role in kleptoplast retention. Lastly, we identified four polyketide synthase genes that could potentially encode proteins producing UV- and oxidation-blocking compounds in slug cell membranes. The genome of E. crispata is a quality resource that provides potential targets for functional analyses and enables further investigation into the evolution and mechanisms of kleptoplasty in animals.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
