<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Biophysically Interpretable Inference of Cell Types from Multimodal Sequencing Data
Authors: Chari, T.; Gorin, G.; Pachter, L.
Score: 37.5, Published: 2023-09-19 DOI: 10.1101/2023.09.17.558131
Multimodal, single-cell genomics technologies enable simultaneous capture of multiple facets of DNA and RNA processing in the cell. This creates opportunities for transcriptome-wide, mechanistic studies of cellular processing in heterogeneous cell types, with applications ranging from inferring kinetic differences between cells, to the role of stochasticity in driving heterogeneity.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Biophysically Interpretable Inference of Cell Types from Multimodal Sequencing Data
Authors: Chari, T.; Gorin, G.; Pachter, L.
Score: 37.5, Published: 2023-09-19 DOI: 10.1101/2023.09.17.558131
Multimodal, single-cell genomics technologies enable simultaneous capture of multiple facets of DNA and RNA processing in the cell. This creates opportunities for transcriptome-wide, mechanistic studies of cellular processing in heterogeneous cell types, with applications ranging from inferring kinetic differences between cells, to the role of stochasticity in driving heterogeneity." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-27T10:39:11+00:00" />
<meta property="article:modified_time" content="2023-09-27T10:39:11+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Biophysically Interpretable Inference of Cell Types from Multimodal Sequencing Data
Authors: Chari, T.; Gorin, G.; Pachter, L.
Score: 37.5, Published: 2023-09-19 DOI: 10.1101/2023.09.17.558131
Multimodal, single-cell genomics technologies enable simultaneous capture of multiple facets of DNA and RNA processing in the cell. This creates opportunities for transcriptome-wide, mechanistic studies of cellular processing in heterogeneous cell types, with applications ranging from inferring kinetic differences between cells, to the role of stochasticity in driving heterogeneity."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Biophysically Interpretable Inference of Cell Types from Multimodal Sequencing Data\nAuthors: Chari, T.; Gorin, G.; Pachter, L.\nScore: 37.5, Published: 2023-09-19 DOI: 10.1101/2023.09.17.558131\nMultimodal, single-cell genomics technologies enable simultaneous capture of multiple facets of DNA and RNA processing in the cell. This creates opportunities for transcriptome-wide, mechanistic studies of cellular processing in heterogeneous cell types, with applications ranging from inferring kinetic differences between cells, to the role of stochasticity in driving heterogeneity.",
  "keywords": [
    
  ],
  "articleBody": " Biophysically Interpretable Inference of Cell Types from Multimodal Sequencing Data\nAuthors: Chari, T.; Gorin, G.; Pachter, L.\nScore: 37.5, Published: 2023-09-19 DOI: 10.1101/2023.09.17.558131\nMultimodal, single-cell genomics technologies enable simultaneous capture of multiple facets of DNA and RNA processing in the cell. This creates opportunities for transcriptome-wide, mechanistic studies of cellular processing in heterogeneous cell types, with applications ranging from inferring kinetic differences between cells, to the role of stochasticity in driving heterogeneity. However, current methods for determining cell types or clusters present in multimodal data often rely on ad hoc or independent treatment of modalities, and assumptions ignoring inherent properties of the count data. To enable interpretable and consistent cell cluster determination from multimodal data, we present meK-Means (mechanistic K-Means) which integrates modalities and learns underlying, shared biophysical states through a unifying model of transcription. In particular, we demonstrate how meK-Means can be used to cluster cells from unspliced and spliced mRNA count modalities. By utilizing the causal, physical relationships underlying these modalities, we identify shared transcriptional kinetics across cells, which induce the observed gene expression profiles, and provide an alternative definition for clusters through the governing parameters of cellular processes.\nMassive-scale single-cell chromatin accessibility sequencing using combinatorial fluidic indexing\nAuthors: Zhang, X.; Marand, A. P.; Yan, H.; Schmitz, R. J.\nScore: 28.9, Published: 2023-09-20 DOI: 10.1101/2023.09.17.558155\nSingle-cell ATAC-seq has emerged as a powerful approach for revealing candidate cis-regulatory elements genome-wide at cell-type resolution. However, current single-cell methods suffer from limited throughput and high costs. Here, we present a novel technique called single-cell combinatorial fluidic indexing ATAC-sequencing (\"scifi-ATAC-seq\"), which combines a barcoded Tn5 pre-indexing step with droplet-based single-cell ATAC-seq using a widely commercialized microfluidics platform (10X Genomics). With scifi-ATAC-seq, up to 200,000 nuclei across multiple samples in a single emulsion reaction can be indexed, representing a [~]20-fold increase in throughput compared to the standard 10X Genomics workflow.\nTuning gene expression to music: the compensatory effect of music on age-related cognitive disorders\nAuthors: Gomez-Carballa, A.; Navarro, L.; Pardo-Seco, J.; Bello, X.; Pischedda, S.; Viz-Lasheras, S.; Camino-Mera, A.; Curras, M. J.; Ferreiros, I.; Mallah, N.; Rey-Vazquez, S.; Redondo, L.; Dacosta-Urbieta, A.; Caamanho-Vinha, F.; Rivero-Calle, I.; Rodriguez-Tenrreiro, C.; Martinon-Torres, F.; Salas, A.\nScore: 35.2, Published: 2023-09-15 DOI: 10.1101/2023.09.12.557408\nExtensive literature has explored the beneficial effects of music in age-related cognitive disorders (ACD), but limited knowledge exists regarding its impact on gene expression. We analyzed transcriptomes of ACD patients and healthy controls, pre-post a music session (n=60), and main genes/pathways were compared to those dysregulated in mild cognitive impairment (MCI) and Alzheimers disease (AD) as revealed by a multi-cohort study (n=1269 MCI/AD and controls). Music was associated with 2.3 times more whole-genome gene expression, particularly on neurodegeneration-related genes, in ACD than controls. Co-expressed gene-modules and pathways analysis demonstrated that music impacted autophagy, vesicle and endosome organization, biological processes commonly dysregulated in MCI/AD. Notably, the data indicated a strong negative correlation between musically-modified genes/pathways in ACD and those dysregulated in MCI/AD. These findings highlight the compensatory effect of music on genes/biological processes affected in MCI/AD, providing insights into the molecular mechanisms underlying the benefits of music on these disorders.\nCohesin mediated loop extrusion from active enhancers form chromatin jets in C. elegans\nAuthors: Kim, J.; Ercan, S.\nScore: 22.7, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558239\nIn mammals, cohesin and CTCF organize the 3D genome into topologically associated domains (TADs) to regulate communication between cis-regulatory elements. However, many organisms, including S. cerevisiae, C. elegans, and A. thaliana lack CTCF. Here, we use C. elegans as a model to investigate the function of cohesin in 3D genome organization in an animal without CTCF. We use auxin-inducible degradation to acutely deplete SMC-3 or its negative regulator WAPL-1 from somatic cells. Using Hi-C data, we identify a cohesin-dependent 3D genome feature termed chromatin jets (aka fountains), also observed in zebrafish and mammalian genomes. The jets emerge from NIPBL occupied segments, and the trajectory of the jets coincides with cohesin binding. The spreading of cohesin from jet origins depends on a fully intact cohesin complex and is extended upon WAPL-1 depletion. These results support the idea that cohesin is preferentially loaded at NIPBL occupied sites, from which cohesin loop extrudes in an effectively two-sided manner. The location of putative loading sites coincides with active enhancers and the pattern of chromatin jets correlates with transcription. We propose that in the absence of CTCF, preferential loading of cohesin at enhancers is a conserved mechanism of genome organization that regulates the interaction of gene regulatory elements in 3D.\nIntegrative PTEN Enhancer Discovery Reveals a New Model of Enhancer Organization\nAuthors: Cerda-Smith, C.; Hutchinson, H. M.; Liu, A.; Goel, V. Y.; Sept, C.; Kim, H.; Casani-Galdon, S.; Burkman, K. G.; Bassil, C. F.; Hansen, A. S.; Aryee, M. J.; Johnstone, S. E.; Eyler, C. E.; Wood, K. C.\nScore: 17.0, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558459\nEnhancers possess both structural elements mediating promoter looping and functional elements mediating gene expression. Traditional models of enhancer-mediated gene regulation imply genomic overlap or immediate adjacency of these elements. We test this model by combining densely-tiled CRISPRa screening with nucleosome-resolution Region Capture Micro-C topology analysis. Using this integrated approach, we comprehensively define the cis-regulatory landscape for the tumor suppressor PTEN, identifying and validating 10 distinct enhancers and defining their 3D spatial organization. Unexpectedly, we identify several long-range functional enhancers whose promoter proximity is facilitated by chromatin loop anchors several kilobases away, and demonstrate that accounting for this spatial separation improves the computational prediction of validated enhancers. Thus, we propose a new model of enhancer organization incorporating spatial separation of essential functional and structural components.\nHigh standing diversity masks extreme genetic erosion in a declining snake.\nAuthors: Pozzi, A. V.; Owens, J. B.; Üveges, B.; Major, T.; Morris, E.; Graham, S.; Togridou, A.; Papadopulos, A. S. T.; Wüster, W.; Barlow, A.\nScore: 16.3, Published: 2023-09-21 DOI: 10.1101/2023.09.19.557540\nAverage heterozygosity is frequently used as a proxy for genetic health, and to compare genetic diversity between species and populations. However, this measurement could be misleading if the distribution of heterozygosity across the genome is highly skewed. We investigated this pitfall in methodology using whole-genome sequencing of the adder (Vipera berus), a species experiencing dramatic declines in the UK. We find that mean heterozygosity in adders is notably high, exceeding that of other vertebrates typically regarded as genetically diverse. Their genome-wide distribution of heterozygosity, however, approximates a negative exponential distribution, with most genome regions showing extremely low heterozygosity. Modelling approaches show that this pattern is likely to have resulted from a recent, severe bottleneck and fragmentation most likely caused by anthropogenic activity in a previously large, interconnected adder population. Our results highlight that high standing diversity may mask severe genetic erosion when declines are recent and rapid. In such situations, whole-genome sequencing may provide the best option for genetic risk assessment and targeted conservation actions.\nSingle-cell nascent RNA sequencing using click-chemistry unveils coordinated transcription\nAuthors: Mahat, D. B.; Tippens, N. D.; Martin-Rufino, J. D.; Waterton, S. K.; Fu, J.; Blatt, S. E.; Sharp, P. A.\nScore: 43.6, Published: 2023-09-19 DOI: 10.1101/2023.09.15.558015\nTranscription is the primary regulatory step in gene expression. Divergent transcription initiation from promoters and enhancers produces stable RNAs from genes and unstable RNAs from enhancers1-5. Nascent RNA capture and sequencing assays simultaneously measure gene and enhancer activity in cell populations6-9. However, fundamental questions in the temporal regulation of transcription and enhancer-gene synchrony remain unanswered primarily due to the absence of a single-cell perspective on active transcription. In this study, we present scGRO-seq - a novel single-cell nascent RNA sequencing assay using click-chemistry - and unveil the coordinated transcription throughout the genome. scGRO-seq demonstrates the episodic nature of transcription, and estimates burst size and frequency by directly quantifying transcribing RNA polymerases in individual cells. It reveals the co-transcription of functionally related genes and leverages the replication-dependent non-polyadenylated histone genes transcription to elucidate cell-cycle dynamics. The single-nucleotide spatial and temporal resolution of scGRO-seq identifies networks of enhancers and genes and indicates that the bursting of transcription at super-enhancers precedes the burst from associated genes. By imparting insights into the dynamic nature of transcription and the origin and propagation of transcription signals, scGRO-seq demonstrates its unique ability to investigate the mechanisms of transcription regulation and the role of enhancers in gene expression.\nDiscovering chromatin dysregulation induced by protein-coding perturbations at scale\nAuthors: Frenkel, M.; Hujoel, M. L. A.; Morris, Z.; Raman, S.\nScore: 14.5, Published: 2023-09-21 DOI: 10.1101/2023.09.20.555752\nAlthough population-scale databases have expanded to millions of protein-coding variants, insight into variant mechanisms has not kept pace. We present PROD-ATAC, a high-throughput method for discovering the effects of protein-coding variants on chromatin. A pooled library of variants is expressed in a disease-agnostic cell line, and single-cell ATAC resolves each variants effect on chromatin. Using PROD-ATAC, we characterized the effects of \u003e100 oncofusions (a class of cancer-causing chimeric proteins) and controls and revealed that pioneer activity is a common feature of fusions spanning an enormous range of fusion frequencies. Further, fusion-induced dysregulation can be context-agnostic as observed mechanisms often overlapped with cancer and cell-type specific prior knowledge. We also showed that gain-of-function pioneering is common among oncofusions. This work provides a global view of fusion-induced chromatin. We uncovered convergent mechanisms among disparate oncofusions and shared modes of dysregulation across different cancers. PROD-ATAC is generalizable to any set of protein-coding variants.\nGrain Utilization by the Gut Microbiome as a Human Health Phenotype to Identify Multiple Effect Loci in Genome-Wide Association Studies of Sorghum bicolor\nAuthors: Korth, N.; Yang, Q.; Van Haute, M. J.; Tross, M. C.; Peng, B.; Shrestha, N.; Zwiener, M.; Mural, R. V.; Schnable, J. C.; Benson, A. K.\nScore: 11.3, Published: 2023-09-23 DOI: 10.1101/2023.09.20.558616\nA growing epidemic of complex lifestyle diseases such as obesity and metabolic diseases are explained in part by dysbiosis of the human gut microbiome. The gut microbiome, comprising trillions of microorganisms, contributes to functions ranging from digestion to the immune system. Diet plays a critical role in determining the species composition and functionality of the gut microbiome. Substantial functional metabolic diversity exists within the cultivated grain crops which directly or indirectly provide more than half of all calories consumed by humans around the globe, however much of this diversity is poorly characterized and the effects of such diversity on the human gut microbiome is not well studied. We employed a quantitative genetics approach to identify genetic variants in sorghum that alter the composition and function of human gut microbes. Using an automated high-throughput phenotyping method based on in vitro microbiome fermentation of grain from a diverse population of Sorghum bicolor cultivars, we demonstrate sorghum genetics can explain effects of grain variation on fermentation patterns of bacterial taxa across multiple human microbiomes. In a genome-wide analysis using a sorghum association panel, we identified fifteen multiple-effect loci (MEL) where different alleles in the sorghum genome produced changes in seed that affect the abundance of multiple bacterial taxa across two human microbiomes in automated in vitro fermentations. In a number of cases parallel genome-wide association studies conducted for biochemical and agronomic traits identified seed traits potentially causal for the link between sorghum genetics and human microbiome outcomes. This work demonstrates that genetic factors affecting sorghum seed can drive significant effects on human gut microbes, particularly bacterial taxa considered beneficial. Understanding these relationships will enable targeted crop breeding strategies to improve human health through gut microbiome modulation.\neQTLs identify regulatory networks and drivers of variation in the individual response to sepsis\nAuthors: Burnham, K. L.; Milind, N.; Lee, W.; Kwok, A. J.; Cano-Gamez, E.; Mi, Y.; Geoghegan, C. G.; GAinS Investigators, ; McKechnie, S.; Soranzo, N.; Hinds, C. J.; Knight, J. C.; Davenport, E. E.\nScore: 10.9, Published: 2023-09-22 DOI: 10.1101/2023.09.22.558983\nSepsis is a clinical syndrome of life-threatening organ dysfunction caused by a dysregulated response to infection, for which disease heterogeneity is a major obstacle to developing targeted treatments. We have previously identified gene expression-based patient subgroups (Sepsis Response Signatures: SRS) informative for outcome and underlying pathophysiology. Here we aimed to investigate the role of genetic variation in determining the host transcriptomic response and to delineate regulatory networks underlying SRS. Using genotyping and RNA-seq data on 638 adult sepsis patients, we report 16,049 independent expression (eQTLs) and 32 co-expression module (modQTLs) quantitative trait loci in this disease context. We identified significant interactions between SRS and genotype for 1,578 SNP-gene pairs, and combined transcription factor (TF) binding site information (SNP2TFBS) and predicted regulon activity (DoRothEA) to identify candidate upstream regulators. These included HIF1A and CEBPB, which were associated with progenitor and immature neutrophil subsets respectively, further implicating glycolysis and emergency granulopoiesis in SRS1. Overall, these approaches identified putative mechanistic links between host genetic variation, cell subtypes, and the individual transcriptomic response to infection. Understanding the regulatory networks underlying patient heterogeneity provides additional information for developing immunomodulatory treatments and a personalised medicine approach to treating sepsis.\n",
  "wordCount" : "2127",
  "inLanguage": "en",
  "datePublished": "2023-09-27T10:39:11Z",
  "dateModified": "2023-09-27T10:39:11Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on September 27, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.558131">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.558131" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.558131">
        <p class="paperTitle">Biophysically Interpretable Inference of Cell Types from Multimodal Sequencing Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.558131" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.558131" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chari, T.; Gorin, G.; Pachter, L.</p>
        <p class="info">Score: 37.5, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.558131' target='https://doi.org/10.1101/2023.09.17.558131'> 10.1101/2023.09.17.558131</a></p>
        <p class="abstract">Multimodal, single-cell genomics technologies enable simultaneous capture of multiple facets of DNA and RNA processing in the cell. This creates opportunities for transcriptome-wide, mechanistic studies of cellular processing in heterogeneous cell types, with applications ranging from inferring kinetic differences between cells, to the role of stochasticity in driving heterogeneity. However, current methods for determining cell types or  clusters present in multimodal data often rely on ad hoc or independent treatment of modalities, and assumptions ignoring inherent properties of the count data. To enable interpretable and consistent cell cluster determination from multimodal data, we present meK-Means (mechanistic K-Means) which integrates modalities and learns underlying, shared biophysical states through a unifying model of transcription. In particular, we demonstrate how meK-Means can be used to cluster cells from unspliced and spliced mRNA count modalities. By utilizing the causal, physical relationships underlying these modalities, we identify shared transcriptional kinetics across cells, which induce the observed gene expression profiles, and provide an alternative definition for  clusters through the governing parameters of cellular processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.558155">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.558155" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.558155">
        <p class="paperTitle">Massive-scale single-cell chromatin accessibility sequencing using combinatorial fluidic indexing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.558155" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.558155" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, X.; Marand, A. P.; Yan, H.; Schmitz, R. J.</p>
        <p class="info">Score: 28.9, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.558155' target='https://doi.org/10.1101/2023.09.17.558155'> 10.1101/2023.09.17.558155</a></p>
        <p class="abstract">Single-cell ATAC-seq has emerged as a powerful approach for revealing candidate cis-regulatory elements genome-wide at cell-type resolution. However, current single-cell methods suffer from limited throughput and high costs. Here, we present a novel technique called single-cell combinatorial fluidic indexing ATAC-sequencing (&#34;scifi-ATAC-seq&#34;), which combines a barcoded Tn5 pre-indexing step with droplet-based single-cell ATAC-seq using a widely commercialized microfluidics platform (10X Genomics). With scifi-ATAC-seq, up to 200,000 nuclei across multiple samples in a single emulsion reaction can be indexed, representing a [~]20-fold increase in throughput compared to the standard 10X Genomics workflow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557408">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557408" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557408">
        <p class="paperTitle">Tuning gene expression to music: the compensatory effect of music on age-related cognitive disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557408" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557408" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gomez-Carballa, A.; Navarro, L.; Pardo-Seco, J.; Bello, X.; Pischedda, S.; Viz-Lasheras, S.; Camino-Mera, A.; Curras, M. J.; Ferreiros, I.; Mallah, N.; Rey-Vazquez, S.; Redondo, L.; Dacosta-Urbieta, A.; Caamanho-Vinha, F.; Rivero-Calle, I.; Rodriguez-Tenrreiro, C.; Martinon-Torres, F.; Salas, A.</p>
        <p class="info">Score: 35.2, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557408' target='https://doi.org/10.1101/2023.09.12.557408'> 10.1101/2023.09.12.557408</a></p>
        <p class="abstract">Extensive literature has explored the beneficial effects of music in age-related cognitive disorders (ACD), but limited knowledge exists regarding its impact on gene expression. We analyzed transcriptomes of ACD patients and healthy controls, pre-post a music session (n=60), and main genes/pathways were compared to those dysregulated in mild cognitive impairment (MCI) and Alzheimers disease (AD) as revealed by a multi-cohort study (n=1269 MCI/AD and controls). Music was associated with 2.3 times more whole-genome gene expression, particularly on neurodegeneration-related genes, in ACD than controls. Co-expressed gene-modules and pathways analysis demonstrated that music impacted autophagy, vesicle and endosome organization, biological processes commonly dysregulated in MCI/AD. Notably, the data indicated a strong negative correlation between musically-modified genes/pathways in ACD and those dysregulated in MCI/AD. These findings highlight the compensatory effect of music on genes/biological processes affected in MCI/AD, providing insights into the molecular mechanisms underlying the benefits of music on these disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558239">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558239" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558239">
        <p class="paperTitle">Cohesin mediated loop extrusion from active enhancers form chromatin jets in C. elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558239" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558239" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, J.; Ercan, S.</p>
        <p class="info">Score: 22.7, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558239' target='https://doi.org/10.1101/2023.09.18.558239'> 10.1101/2023.09.18.558239</a></p>
        <p class="abstract">In mammals, cohesin and CTCF organize the 3D genome into topologically associated domains (TADs) to regulate communication between cis-regulatory elements. However, many organisms, including S. cerevisiae, C. elegans, and A. thaliana lack CTCF. Here, we use C. elegans as a model to investigate the function of cohesin in 3D genome organization in an animal without CTCF. We use auxin-inducible degradation to acutely deplete SMC-3 or its negative regulator WAPL-1 from somatic cells. Using Hi-C data, we identify a cohesin-dependent 3D genome feature termed chromatin jets (aka fountains), also observed in zebrafish and mammalian genomes. The jets emerge from NIPBL occupied segments, and the trajectory of the jets coincides with cohesin binding. The spreading of cohesin from jet origins depends on a fully intact cohesin complex and is extended upon WAPL-1 depletion. These results support the idea that cohesin is preferentially loaded at NIPBL occupied sites, from which cohesin loop extrudes in an effectively two-sided manner. The location of putative loading sites coincides with active enhancers and the pattern of chromatin jets correlates with transcription. We propose that in the absence of CTCF, preferential loading of cohesin at enhancers is a conserved mechanism of genome organization that regulates the interaction of gene regulatory elements in 3D.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558459">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558459" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558459">
        <p class="paperTitle">Integrative PTEN Enhancer Discovery Reveals a New Model of Enhancer Organization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558459" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558459" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cerda-Smith, C.; Hutchinson, H. M.; Liu, A.; Goel, V. Y.; Sept, C.; Kim, H.; Casani-Galdon, S.; Burkman, K. G.; Bassil, C. F.; Hansen, A. S.; Aryee, M. J.; Johnstone, S. E.; Eyler, C. E.; Wood, K. C.</p>
        <p class="info">Score: 17.0, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558459' target='https://doi.org/10.1101/2023.09.20.558459'> 10.1101/2023.09.20.558459</a></p>
        <p class="abstract">Enhancers possess both structural elements mediating promoter looping and functional elements mediating gene expression. Traditional models of enhancer-mediated gene regulation imply genomic overlap or immediate adjacency of these elements. We test this model by combining densely-tiled CRISPRa screening with nucleosome-resolution Region Capture Micro-C topology analysis. Using this integrated approach, we comprehensively define the cis-regulatory landscape for the tumor suppressor PTEN, identifying and validating 10 distinct enhancers and defining their 3D spatial organization. Unexpectedly, we identify several long-range functional enhancers whose promoter proximity is facilitated by chromatin loop anchors several kilobases away, and demonstrate that accounting for this spatial separation improves the computational prediction of validated enhancers. Thus, we propose a new model of enhancer organization incorporating spatial separation of essential functional and structural components.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.19.557540">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.19.557540" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.19.557540">
        <p class="paperTitle">High standing diversity masks extreme genetic erosion in a declining snake.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.19.557540" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.19.557540" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pozzi, A. V.; Owens, J. B.; Üveges, B.; Major, T.; Morris, E.; Graham, S.; Togridou, A.; Papadopulos, A. S. T.; Wüster, W.; Barlow, A.</p>
        <p class="info">Score: 16.3, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.19.557540' target='https://doi.org/10.1101/2023.09.19.557540'> 10.1101/2023.09.19.557540</a></p>
        <p class="abstract">Average heterozygosity is frequently used as a proxy for genetic health, and to compare genetic diversity between species and populations. However, this measurement could be misleading if the distribution of heterozygosity across the genome is highly skewed. We investigated this pitfall in methodology using whole-genome sequencing of the adder (Vipera berus), a species experiencing dramatic declines in the UK. We find that mean heterozygosity in adders is notably high, exceeding that of other vertebrates typically regarded as genetically diverse. Their genome-wide distribution of heterozygosity, however, approximates a negative exponential distribution, with most genome regions showing extremely low heterozygosity. Modelling approaches show that this pattern is likely to have resulted from a recent, severe bottleneck and fragmentation most likely caused by anthropogenic activity in a previously large, interconnected adder population. Our results highlight that high standing diversity may mask severe genetic erosion when declines are recent and rapid. In such situations, whole-genome sequencing may provide the best option for genetic risk assessment and targeted conservation actions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.558015">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.558015" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.558015">
        <p class="paperTitle">Single-cell nascent RNA sequencing using click-chemistry unveils coordinated transcription</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.558015" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.558015" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mahat, D. B.; Tippens, N. D.; Martin-Rufino, J. D.; Waterton, S. K.; Fu, J.; Blatt, S. E.; Sharp, P. A.</p>
        <p class="info">Score: 43.6, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.558015' target='https://doi.org/10.1101/2023.09.15.558015'> 10.1101/2023.09.15.558015</a></p>
        <p class="abstract">Transcription is the primary regulatory step in gene expression. Divergent transcription initiation from promoters and enhancers produces stable RNAs from genes and unstable RNAs from enhancers1-5. Nascent RNA capture and sequencing assays simultaneously measure gene and enhancer activity in cell populations6-9. However, fundamental questions in the temporal regulation of transcription and enhancer-gene synchrony remain unanswered primarily due to the absence of a single-cell perspective on active transcription. In this study, we present scGRO-seq - a novel single-cell nascent RNA sequencing assay using click-chemistry - and unveil the coordinated transcription throughout the genome. scGRO-seq demonstrates the episodic nature of transcription, and estimates burst size and frequency by directly quantifying transcribing RNA polymerases in individual cells. It reveals the co-transcription of functionally related genes and leverages the replication-dependent non-polyadenylated histone genes transcription to elucidate cell-cycle dynamics. The single-nucleotide spatial and temporal resolution of scGRO-seq identifies networks of enhancers and genes and indicates that the bursting of transcription at super-enhancers precedes the burst from associated genes. By imparting insights into the dynamic nature of transcription and the origin and propagation of transcription signals, scGRO-seq demonstrates its unique ability to investigate the mechanisms of transcription regulation and the role of enhancers in gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.555752">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.555752" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.555752">
        <p class="paperTitle">Discovering chromatin dysregulation induced by protein-coding perturbations at scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.555752" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.555752" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frenkel, M.; Hujoel, M. L. A.; Morris, Z.; Raman, S.</p>
        <p class="info">Score: 14.5, Published: 2023-09-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.555752' target='https://doi.org/10.1101/2023.09.20.555752'> 10.1101/2023.09.20.555752</a></p>
        <p class="abstract">Although population-scale databases have expanded to millions of protein-coding variants, insight into variant mechanisms has not kept pace. We present PROD-ATAC, a high-throughput method for discovering the effects of protein-coding variants on chromatin. A pooled library of variants is expressed in a disease-agnostic cell line, and single-cell ATAC resolves each variants effect on chromatin. Using PROD-ATAC, we characterized the effects of &gt;100 oncofusions (a class of cancer-causing chimeric proteins) and controls and revealed that pioneer activity is a common feature of fusions spanning an enormous range of fusion frequencies. Further, fusion-induced dysregulation can be context-agnostic as observed mechanisms often overlapped with cancer and cell-type specific prior knowledge. We also showed that gain-of-function pioneering is common among oncofusions. This work provides a global view of fusion-induced chromatin. We uncovered convergent mechanisms among disparate oncofusions and shared modes of dysregulation across different cancers. PROD-ATAC is generalizable to any set of protein-coding variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558616">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558616" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558616">
        <p class="paperTitle">Grain Utilization by the Gut Microbiome as a Human Health Phenotype to Identify Multiple Effect Loci in Genome-Wide Association Studies of Sorghum bicolor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558616" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558616" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Korth, N.; Yang, Q.; Van Haute, M. J.; Tross, M. C.; Peng, B.; Shrestha, N.; Zwiener, M.; Mural, R. V.; Schnable, J. C.; Benson, A. K.</p>
        <p class="info">Score: 11.3, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558616' target='https://doi.org/10.1101/2023.09.20.558616'> 10.1101/2023.09.20.558616</a></p>
        <p class="abstract">A growing epidemic of complex lifestyle diseases such as obesity and metabolic diseases are explained in part by dysbiosis of the human gut microbiome. The gut microbiome, comprising trillions of microorganisms, contributes to functions ranging from digestion to the immune system. Diet plays a critical role in determining the species composition and functionality of the gut microbiome. Substantial functional metabolic diversity exists within the cultivated grain crops which directly or indirectly provide more than half of all calories consumed by humans around the globe, however much of this diversity is poorly characterized and the effects of such diversity on the human gut microbiome is not well studied. We employed a quantitative genetics approach to identify genetic variants in sorghum that alter the composition and function of human gut microbes. Using an automated high-throughput phenotyping method based on in vitro microbiome fermentation of grain from a diverse population of Sorghum bicolor cultivars, we demonstrate sorghum genetics can explain effects of grain variation on fermentation patterns of bacterial taxa across multiple human microbiomes. In a genome-wide analysis using a sorghum association panel, we identified fifteen multiple-effect loci (MEL) where different alleles in the sorghum genome produced changes in seed that affect the abundance of multiple bacterial taxa across two human microbiomes in automated in vitro fermentations. In a number of cases parallel genome-wide association studies conducted for biochemical and agronomic traits identified seed traits potentially causal for the link between sorghum genetics and human microbiome outcomes. This work demonstrates that genetic factors affecting sorghum seed can drive significant effects on human gut microbes, particularly bacterial taxa considered beneficial. Understanding these relationships will enable targeted crop breeding strategies to improve human health through gut microbiome modulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558983">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558983" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558983">
        <p class="paperTitle">eQTLs identify regulatory networks and drivers of variation in the individual response to sepsis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558983" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558983" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burnham, K. L.; Milind, N.; Lee, W.; Kwok, A. J.; Cano-Gamez, E.; Mi, Y.; Geoghegan, C. G.; GAinS Investigators,  ; McKechnie, S.; Soranzo, N.; Hinds, C. J.; Knight, J. C.; Davenport, E. E.</p>
        <p class="info">Score: 10.9, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558983' target='https://doi.org/10.1101/2023.09.22.558983'> 10.1101/2023.09.22.558983</a></p>
        <p class="abstract">Sepsis is a clinical syndrome of life-threatening organ dysfunction caused by a dysregulated response to infection, for which disease heterogeneity is a major obstacle to developing targeted treatments. We have previously identified gene expression-based patient subgroups (Sepsis Response Signatures: SRS) informative for outcome and underlying pathophysiology. Here we aimed to investigate the role of genetic variation in determining the host transcriptomic response and to delineate regulatory networks underlying SRS. Using genotyping and RNA-seq data on 638 adult sepsis patients, we report 16,049 independent expression (eQTLs) and 32 co-expression module (modQTLs) quantitative trait loci in this disease context. We identified significant interactions between SRS and genotype for 1,578 SNP-gene pairs, and combined transcription factor (TF) binding site information (SNP2TFBS) and predicted regulon activity (DoRothEA) to identify candidate upstream regulators. These included HIF1A and CEBPB, which were associated with progenitor and immature neutrophil subsets respectively, further implicating glycolysis and emergency granulopoiesis in SRS1. Overall, these approaches identified putative mechanistic links between host genetic variation, cell subtypes, and the individual transcriptomic response to infection. Understanding the regulatory networks underlying patient heterogeneity provides additional information for developing immunomodulatory treatments and a personalised medicine approach to treating sepsis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
