<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genomic analysis of Staphylococcus aureus isolates from bacteremia reveals genetic features associated with the COVID-19 pandemic
Authors: Sanchez-Osuna, M.; Pedrosa, M.; Bierge, P.; Gomez-Sanchez, I.; Alguacil-Guillen, M.; Espasa, M.; Erill, I.; Gasch, O.; Pich, O. Q.
Score: 261.4, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572975
Genomic analyses of bacterial isolates are necessary to monitor the prevalence of antibiotic resistance genes and virulence determinants. Herein, we provide a comprehensive genomic description of a collection of 339 Staphylococcus aureus strains isolated from patients with bacteremia between 2014 and 2022.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Genomic analysis of Staphylococcus aureus isolates from bacteremia reveals genetic features associated with the COVID-19 pandemic
Authors: Sanchez-Osuna, M.; Pedrosa, M.; Bierge, P.; Gomez-Sanchez, I.; Alguacil-Guillen, M.; Espasa, M.; Erill, I.; Gasch, O.; Pich, O. Q.
Score: 261.4, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572975
Genomic analyses of bacterial isolates are necessary to monitor the prevalence of antibiotic resistance genes and virulence determinants. Herein, we provide a comprehensive genomic description of a collection of 339 Staphylococcus aureus strains isolated from patients with bacteremia between 2014 and 2022." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Genomic analysis of Staphylococcus aureus isolates from bacteremia reveals genetic features associated with the COVID-19 pandemic
Authors: Sanchez-Osuna, M.; Pedrosa, M.; Bierge, P.; Gomez-Sanchez, I.; Alguacil-Guillen, M.; Espasa, M.; Erill, I.; Gasch, O.; Pich, O. Q.
Score: 261.4, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572975
Genomic analyses of bacterial isolates are necessary to monitor the prevalence of antibiotic resistance genes and virulence determinants. Herein, we provide a comprehensive genomic description of a collection of 339 Staphylococcus aureus strains isolated from patients with bacteremia between 2014 and 2022."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Genomic analysis of Staphylococcus aureus isolates from bacteremia reveals genetic features associated with the COVID-19 pandemic\nAuthors: Sanchez-Osuna, M.; Pedrosa, M.; Bierge, P.; Gomez-Sanchez, I.; Alguacil-Guillen, M.; Espasa, M.; Erill, I.; Gasch, O.; Pich, O. Q.\nScore: 261.4, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572975\nGenomic analyses of bacterial isolates are necessary to monitor the prevalence of antibiotic resistance genes and virulence determinants. Herein, we provide a comprehensive genomic description of a collection of 339 Staphylococcus aureus strains isolated from patients with bacteremia between 2014 and 2022.",
  "keywords": [
    
  ],
  "articleBody": " Genomic analysis of Staphylococcus aureus isolates from bacteremia reveals genetic features associated with the COVID-19 pandemic\nAuthors: Sanchez-Osuna, M.; Pedrosa, M.; Bierge, P.; Gomez-Sanchez, I.; Alguacil-Guillen, M.; Espasa, M.; Erill, I.; Gasch, O.; Pich, O. Q.\nScore: 261.4, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572975\nGenomic analyses of bacterial isolates are necessary to monitor the prevalence of antibiotic resistance genes and virulence determinants. Herein, we provide a comprehensive genomic description of a collection of 339 Staphylococcus aureus strains isolated from patients with bacteremia between 2014 and 2022. Nosocomial acquisition accounted for 56.6% of episodes, with vascular catheters being the predominant source of infection (31.8%). Cases of fatality (27.4%), persistent bacteremia (19.5%) and diagnosis of septic emboli (24.2%) were documented. During the COVID-19 pandemic, we observed a 140% increase of the episodes of S. aureus bacteremia per year, with a concomitant increase of the cases from nosocomial origin. This prompted us to investigate the existence of genetic features associated with S. aureus isolates from the COVID-19 pandemic. While genes conferring resistance to {beta}-lactams (blaI-blaR-blaZ), macrolides (ermA, ermC, ermT, mphC, msrA) and aminoglycosides (ant(4)-Ia, ant(9)-Ia, aph(3)-IIIa, aph(2)-Ih) were prevalent in our collection, detection of the msrA and mphC genes increased significantly in pandemic S. aureus isolates. Similarly, we observed a higher prevalence of isolates carrying the genes encoding the Clumping Factors A and B, involved in fibrinogen binding. Of note, macrolides were extensively used as accessory therapy for COVID-19 and fibrinogen levels were usually elevated upon SARS-CoV-2 infection. Therefore, our results reveal a remarkable adaptation of the S. aureus isolates to the COVID-19 pandemic context and demonstrates the potential of whole-genome sequencing to conduct molecular epidemiology studies.\nChIP-DIP: A multiplexed method for mapping hundreds of proteins to DNA uncovers diverse regulatory elements controlling gene expression\nAuthors: Perez, A. A.; Goronzy, I. N.; Blanco, M. R.; Guo, J. K.; Guttman, M.\nScore: 146.0, Published: 2023-12-18 DOI: 10.1101/2023.12.14.571730\nGene expression is controlled by the dynamic localization of thousands of distinct regulatory proteins to precise regions of DNA. Understanding this cell-type specific process has been a goal of molecular biology for decades yet remains challenging because most current DNA-protein mapping methods study one protein at a time. To overcome this, we developed ChIP-DIP (ChIP Done In Parallel), a split-pool based method that enables simultaneous, genome-wide mapping of hundreds of diverse regulatory proteins in a single experiment. We demonstrate that ChIP-DIP generates highly accurate maps for all classes of DNA-associated proteins, including histone modifications, chromatin regulators, transcription factors, and RNA Polymerases. Using these data, we explore quantitative combinations of protein localization on genomic DNA to define distinct classes of regulatory elements and their functional activity. Our data demonstrate that ChIP-DIP enables the generation of consortium level, context-specific protein localization maps within any molecular biology lab.\nAtlas of telomeric repeat diversity in Arabidopsis thaliana\nAuthors: Tao, Y.; Xian, W.; Rabanal, F.; Movilli, A.; Lanz, C.; Shirsekar, G.; Weigel, D.\nScore: 22.1, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572118\nTelomeric repeat arrays at the ends of chromosomes are highly dynamic but their repetitive nature and technological limitations have made it difficult to assess the variation in genome diversity surveys. Here we present a comprehensive characterization of the sequence variation immediately adjacent to the canonical telomeric repeat arrays at the very ends of chromosomes in 49 genetically diverse Arabidopsis thaliana accessions. We reveal several types of distinct telomeric repeat units and identify evolutionary processes such as local homogenization and higher-order repeat formation that shape diversity of chromosome ends. The identification of segmental duplications and at least one recombination event suggests a plausible history of telomerase-independent maintenance generation. By comparing largely isogenic samples, we are able to determine variant telomeric repeat number variation at both the germline and somatic levels. Analysis of haplotype structure uncovers chromosome end-specific as well as genetic group-specific patterns in telomeric repeat diversity and provides evidence for linkage disequilibrium between repeat arrays and their adjacent non-coding regions. Together, our findings illustrate the fine-scale telomeric repeat spectrum in A. thaliana, expanding our knowledge of the evolution of chromosome ends.\nA comparative analysis of planarian genomes reveals regulatory conservation in the face of rapid structural divergence\nAuthors: Ivankovic, M.; Brand, J. N.; Pandolfini, L.; Brown, T.; Pippel, M.; Rozanski, A.; Schubert, T.; Grohme, M. A.; Winkler, S.; Robledillo, L.; Zhang, M.; Codino, A.; Gustincich, S.; Vila-Farre, M.; Zhang, S.; Papantonis, A.; Marques, A.; Rink, J. C.\nScore: 20.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572568\nThe planarian Schmidtea mediterranea can regenerate its entire body from small tissue fragments and is studied as whole-body regeneration model species. While well-annotated genome resources are critical for understanding regeneration, the assembly and functional analysis of planarian genomes has proven challenging due to extreme compositional biases (70% A/T), high repeat contents and limited utility of routine laboratory protocols owing to their divergent biochemistry. Only few and often fragmented genome assemblies are currently available, and open challenges include the provision of well-annotated chromosome-scale reference assemblies of the model species and other planarians for a comparative genome evolution perspective. Here we report a haplotype-phased, chromosome-scale genome assembly and high-quality gene annotations of the sexual strain of S. mediterranea. We also provide putative regulatory region annotations via optimized ATAC-seq and ChIP-seq protocols. To additionally leverage sequence conservation for regulatory element annotations, we generated annotated chromosome-scale genome assemblies and chromatin accessibility data for the three closest relatives of S. mediterranea: S. polychroa, S. nova, and S. lugubris. Although we find broad sequence divergence between the genomes, including in open chromatin regions, ChIP-mark bearing S. mediterranea regulatory elements displayed significant sequence conservation. The resulting high-confidence set of evolutionary conserved enhancers and promoters provides a valuable resource for the analysis of gene regulatory circuits and their evolution within the taxon. In addition, our four chromosome-scale genome assemblies provide a first comparative perspective on planarian genome evolution. Our analyses reveal frequent retrotransposon-associated chromosomal inversions and inter-chromosomal translocations that lead to a degradation of synteny across the genus. Interestingly, we further find independent and near-complete losses of the ancestral metazoan synteny across Schmidtea and two other flatworm groups, indicating that platyhelminth genomes largely evolve without syntenic constraints. We provide valuable genome resources for the planarian research community and set a foundation for comparative genomics of planarians. We find that the rapid structural evolution of planarian genomes contrasts with the conservation of regulatory elements. Therefore, the unusual biology of flatworms appears paired with equally unique patterns of genome evolution, in which the position of genes in the genome may not matter.\nAnnelid methylomes reveal ancestral developmental and ageing-associated epigenetic erosion across Bilateria\nAuthors: Guynes, K.; Sarre, L. A.; Carrillo-Baltodano, A. M.; Davies, B. E.; Xu, L.; Liang, Y.; Martin-Zamora, F. M.; Hurd, P. J.; de Mendoza, A.; Martin-Duran, J. M.\nScore: 9.8, Published: 2023-12-25 DOI: 10.1101/2023.12.21.572802\nBackground: DNA methylation in the form of 5-methylcytosine (5mC) is the most abundant base modification in animals. However, 5mC levels vary widely across taxa. While vertebrate genomes are hypermethylated, in most invertebrates, 5mC concentrates on constantly and highly transcribed genes (gene body methylation; GbM) and, in some species, on transposable elements (TEs), a pattern known as \"mosaic\". Yet, the role and developmental dynamics of 5mC and how these explain interspecific differences in DNA methylation patterns remain poorly understood, especially in Spiralia, a large clade of invertebrates comprising nearly half of the animal phyla. Results: Here, we generate base-resolution methylomes for three species with distinct genomic features and phylogenetic positions in Annelida, a major spiralian phylum. All possible 5mC patterns occur in annelids, from typical invertebrate intermediate levels in a mosaic distribution to hypermethylation and methylation loss. GbM is common to annelids with 5mC, and methylation differences across species are explained by taxon-specific transcriptional dynamics or the presence of intronic TEs. Notably, the link between GbM and transcription decays during development, and there is a gradual and global, age-dependent demethylation in adult stages. Moreover, reducing 5mC levels with cytidine analogues during early development impairs normal embryogenesis and reactivates TEs in the annelid Owenia fusiformis. Conclusions: Our study indicates that global epigenetic erosion during development and ageing is an ancestral feature of bilateral animals. However, the tight link between transcription and gene body methylation is likely important in early embryonic stages, and 5mC-mediated TE silencing probably emerged convergently across animal lineages.\nImage-based spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis\nAuthors: Vannan, A.; Lyu, R.; Williams, A. L.; Negretti, N. M.; Mee, E. D.; Hirsh, J.; Hirsh, S.; Nichols, D. S.; Calvi, C. L.; Talor, C. J.; Polosukhin, V. V.; Serezani, A. P.; McCall, A. S.; Gokey, J. J.; Shim, H.; Ware, L. B.; Bacchetta, M. J.; Shaver, C. M.; Blackwell, T. S.; Walia, R.; Sucre, J. M.; Kropski, J. A.; McCarthy, D. J.; Banovich, N. E.\nScore: 35.1, Published: 2023-12-17 DOI: 10.1101/2023.12.15.571954\nThe human lung is structurally complex, with a diversity of specialized epithelial, stromal and immune cells playing specific functional roles in anatomically distinct locations, and large-scale changes in the structure and cellular makeup of this distal lung is a hallmark of pulmonary fibrosis (PF) and other progressive chronic lung diseases. Single-cell transcriptomic studies have revealed numerous disease-emergent/enriched cell types/states in PF lungs, but the spatial contexts wherein these cells contribute to disease pathogenesis has remained uncertain. Using sub-cellular resolution image-based spatial transcriptomics, we analyzed the gene expression of more than 1 million cells from 19 unique lungs. Through complementary cell-based and innovative cell-agnostic analyses, we characterized the localization of PF-emergent cell-types, established the cellular and molecular basis of classical PF histopathologic disease features, and identified a diversity of distinct molecularly-defined spatial niches in control and PF lungs. Using machine-learning and trajectory analysis methods to segment and rank airspaces on a gradient from normal to most severely remodeled, we identified a sequence of compositional and molecular changes that associate with progressive distal lung pathology, beginning with alveolar epithelial dysregulation and culminating with changes in macrophage polarization. Together, these results provide a unique, spatially-resolved characterization of the cellular and molecular programs of PF and control lungs, provide new insights into the heterogeneous pathobiology of PF, and establish analytical approaches which should be broadly applicable to other imaging-based spatial transcriptomic studies.\nCharacterizing uncertainty in predictions of genomic sequence-to-activity models\nAuthors: Bajwa, A.; Rastogi, R.; Kathail, P.; Shuai, R. W.; Ioannidis, N. M.\nScore: 4.6, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572730\nGenomic sequence-to-activity models are increasingly utilized to understand gene regulatory syntax and probe the functional consequences of regulatory variation. Current models make accurate predictions of relative activity levels across the human reference genome, but their performance is more limited for predicting the effects of genetic variants, such as explaining gene expression variation across individuals. To better understand the causes of these shortcomings, we examine the uncertainty in predictions of genomic sequence-to-activity models using an ensemble of Basenji2 model replicates. We characterize prediction consistency on four types of sequences: reference genome sequences, reference genome sequences perturbed with TF motifs, eQTLs, and personal genome sequences. We observe that models tend to make high-confidence predictions on reference sequences, even when incorrect, and low-confidence predictions on sequences with variants. For eQTLs and personal genome sequences, we find that model replicates make inconsistent predictions in \u003e50% of cases. Our findings suggest strategies to improve performance of these models.\nA validated pangenome-scale metabolic model for the Klebsiella pneumoniae species complex\nAuthors: Cooper, H. B.; Vezina, B.; Hawkey, J.; Passet, V.; Lopez-Fernandez, S.; Monk, J. M.; Brisse, S.; Holt, K. E.; Wyres, K. L.\nScore: 4.5, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572682\nThe Klebsiella pneumoniae Species Complex (KpSC) is a major source of nosocomial infections globally with high rates of resistance to antimicrobials. Consequently, there is growing interest in understanding virulence factors and their association with cellular metabolic processes for developing novel anti-KpSC therapeutics. Phenotypic assays have revealed metabolic diversity within the KpSC, but metabolism research has been neglected due to experiments being difficult and cost-intensive. Genome-scale metabolic models (GSMMs) represent a rapid and scalable in silico approach for exploring metabolic diversity, which compiles genomic and biochemical data to reconstruct the metabolic network of an organism. Here we use a diverse collection of 507 KpSC isolates, including representatives of globally distributed clinically-relevant lineages, to construct the most comprehensive KpSC pan-metabolic model to-date, KpSC pan v2. Candidate metabolic reactions were identified using gene orthology to known metabolic genes, prior to manual curation via extensive literature and database searches. The final model comprised a total of 3,550 reactions, 2,403 genes and can simulate growth on 360 unique substrates. We used KpSC pan v2 as a reference to derive strain-specific GSMMs for all 507 KpSC isolates, and compared these to GSMMs generated using a prior KpSC pan-reference (KpSC pan v1) and two single-strain references. We show that KpSC pan v2 includes a greater proportion of accessory reactions (8.8%) than KpSC pan v1 (2.5%). GSMMs derived from KpSC pan v2 also result in more accuracy growth predictions than those derived from other references in both aerobic (median accuracy = 95.4%) and anaerobic (median accuracy = 78.8%). KpSC pan v2 also generates more accurate growth predictions, with high median accuracies of 95.4% (aerobic, n=37 isolates) and 78.8% (anaerobic, n=36 isolates) for 124 matched carbon substrates. KpSC pan v2 is freely available at https://github.com/kelwyres/KpSC-pan-metabolic-model, representing a valuable resource for the scientific community, both as a source of curated metabolic information and as a reference to derive accurate strain-specific GSMMs. The latter can be used to investigate the relationship between KpSC metabolism and traits of interest, such as reservoirs, epidemiology, drug resistance or virulence, and ultimately to inform novel KpSC control strategies. Significance as a BioResource to the communityKlebsiella pneumoniae and its close relatives in the K. pneumoniae Species Complex (KpSC) are priority antimicrobial resistant pathogens that exhibit extensive genomic diversity. There is growing interest in understanding KpSC metabolism, and genome scale metabolic models (GSMMs) provide a rapid, scalable option for exploration of whole cell metabolism plus phenotype prediction. Here we present a KpSC pan-metabolic model representing the cellular metabolism of 507 diverse KpSC isolates. Our model is the largest and most comprehensive of its kind, comprising \u003e2,400 genes associated with \u003e3,500 metabolic reactions, plus manually curated evidence annotations. These data alone represent a key knowledge resource for the Klebsiella research community; however, our models greatest impact lies in its potential for use as a reference from which highly accurate strain-specific GSMMs can be derived to inform in depth strain-specific and/or large-scale comparative analyses. Data summaryO_LIKlebsiella pneumoniae species complex (KpSC) pan v2 metabolic model available at https://github.com/kelwyres/KpSC-pan-metabolic-model. C_LIO_LIAll KpSC isolate whole genome sequences used in this work were reported previously and are available under Bioprojects PRJEB6891, PRJNA351909, PRJNA493667, PRJNA768294, PRJNA253462, PRJNA292902 and PRJNA391323. Individual accessions listed in Table S1. C_LIO_LIStrain-specific GSMMs used for comparative analyses (deposited in Figshare - 10.6084/m9.figshare.24871914), plus their associated MEMOTE reports (indicates completeness and annotation quality), reaction and gene presence-absence matrices across all isolates. C_LIO_LIGrowth phenotype predictions derived from strain-specific GSMMs are available in Table S4. C_LIO_LIBinarised Biolog growth phenotype data for n=37 isolates (plates PM1 and PM2, aerobic and anaerobic conditions) are available in Tables S6 \u0026 S7. C_LIO_LIAdditional growth assay data for six substrates not included on Biolog plates PM1 and PM2 (deposited in Figshare - 10.6084/m9.figshare.24871914). C_LI\nCRISPRmap: Sequencing-free optical pooled screens mapping multi-omic phenotypes in cells and tissue\nAuthors: Gu, J.; Iyer, A.; Wesley, B.; Taglialatela, A.; Leuzzi, G.; Hangai, S.; Decker, A.; Gu, R.; Klickstein, N.; Shuai, Y.; Jankovic, K.; Parker-Burns, L.; Jin, Y.; Zhang, J. Y.; Hong, J.; Niu, X.; Chou, J.; Landau, D. A.; Azizi, E.; Chan, E. M.; Ciccia, A.; Gaublomme, J. T.\nScore: 4.3, Published: 2023-12-26 DOI: 10.1101/2023.12.26.572587\nPooled genetic screens are powerful tools to study gene function in a high-throughput manner. Typically, sequencing-based screens require cell lysis, which limits the examination of critical phenotypes such as cell morphology, protein subcellular localization, and cell-cell/tissue interactions. In contrast, emerging optical pooled screening methods enable the investigation of these spatial phenotypes in response to targeted CRISPR perturbations. In this study, we report a multi-omic optical pooled CRISPR screening method, which we have named CRISPRmap. Our method combines a novel in situ CRISPR guide identifying barcode readout approach with concurrent multiplexed immunofluorescence and in situ RNA detection. CRISPRmap barcodes are detected and read out through combinatorial hybridization of DNA oligos, enhancing barcode detection efficiency, while reducing both dependency on third party proprietary sequencing reagents and assay cost. Notably, we conducted a multi-omic base-editing screen in a breast cancer cell line on core DNA damage repair genes involved in the homologous recombination and Fanconi anemia pathways investigating how nucleotide variants in those genes influence DNA damage signaling and cell cycle regulation following treatment with ionizing radiation or DNA damaging agents commonly used for cancer therapy. Approximately a million cells were profiled with our multi-omic approach, providing a comprehensive phenotypic assessment of the functional consequences of the studied variants. CRISPRmap enabled us to pinpoint likely-pathogenic patient-derived mutations that were previously classified as variants of unknown clinical significance. Furthermore, our approach effectively distinguished barcodes of a pooled library in tumor tissue, and we coupled it with cell-type and molecular phenotyping by cyclic immunofluorescence. Multi-omic spatial analysis of how CRISPR-perturbed cells respond to various environmental cues in the tissue context offers the potential to significantly expand our understanding of tissue biology in both health and disease.\nFunctional characterization of eQTLs and asthma risk loci with scATAC-seq across immune cell types and contexts\nAuthors: Wei, J.; Resztak, J. A.; Alazizi, A.; Mair-Meijers, H. E.; Slatcher, R. B.; Zilioli, S.; Wen, X.; Luca, F.; Pique-Regi, R.\nScore: 4.2, Published: 2023-12-25 DOI: 10.1101/2023.12.24.573260\nCis-regulatory elements (CREs) control gene transcription dynamics across cell types and in response to the environment. In asthma, multiple immune cell types play an important role in the inflammatory process. Genetic variants in CREs can also affect gene expression response dynamics and contribute to asthma risk. However, the regulatory mechanisms underlying control of transcriptional dynamics across different environmental contexts and cell-types at single cell resolution remains to be elucidated. To resolve this question, we performed scATAC-seq in activated peripheral blood mononuclear cells (PBMC) from children with asthma with phytohemagglutinin (PHA) or lipopolysaccharide (LPS), and treated with dexamethasone (DEX), an anti-inflammatory glucocorticoid. We analyzed changes in chromatin accessibility, measured transcription factor motif activity, and identified treatment and cell-type specific transcription factors that drive changes in both gene expression mean and variability. We observed strong positive linear dependence between motif response and their target gene expression changes, but negative in variability changes. This result suggests that an increase of transcription factor binding tightens the variability of gene expression around the mean. We then annotated genetic variants in chromatin accessibility peaks and response motifs followed by computational fine-mapping of eQTL signals from a pediatric asthma cohort. We found that eQTLs were highly enriched in peaks with response motifs and refined the credible set of several asthma risk genes. In conclusion, scATAC-seq enhances the understanding of molecular mechanisms for asthma risk variants mediated by gene expression.\n",
  "wordCount" : "3139",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572975">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572975" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572975">
        <p class="paperTitle">Genomic analysis of Staphylococcus aureus isolates from bacteremia reveals genetic features associated with the COVID-19 pandemic</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572975" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572975" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez-Osuna, M.; Pedrosa, M.; Bierge, P.; Gomez-Sanchez, I.; Alguacil-Guillen, M.; Espasa, M.; Erill, I.; Gasch, O.; Pich, O. Q.</p>
        <p class="info">Score: 261.4, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572975' target='https://doi.org/10.1101/2023.12.22.572975'> 10.1101/2023.12.22.572975</a></p>
        <p class="abstract">Genomic analyses of bacterial isolates are necessary to monitor the prevalence of antibiotic resistance genes and virulence determinants. Herein, we provide a comprehensive genomic description of a collection of 339 Staphylococcus aureus strains isolated from patients with bacteremia between 2014 and 2022. Nosocomial acquisition accounted for 56.6% of episodes, with vascular catheters being the predominant source of infection (31.8%). Cases of fatality (27.4%), persistent bacteremia (19.5%) and diagnosis of septic emboli (24.2%) were documented. During the COVID-19 pandemic, we observed a 140% increase of the episodes of S. aureus bacteremia per year, with a concomitant increase of the cases from nosocomial origin. This prompted us to investigate the existence of genetic features associated with S. aureus isolates from the COVID-19 pandemic. While genes conferring resistance to {beta}-lactams (blaI-blaR-blaZ), macrolides (ermA, ermC, ermT, mphC, msrA) and aminoglycosides (ant(4)-Ia, ant(9)-Ia, aph(3)-IIIa, aph(2)-Ih) were prevalent in our collection, detection of the msrA and mphC genes increased significantly in pandemic S. aureus isolates. Similarly, we observed a higher prevalence of isolates carrying the genes encoding the Clumping Factors A and B, involved in fibrinogen binding. Of note, macrolides were extensively used as accessory therapy for COVID-19 and fibrinogen levels were usually elevated upon SARS-CoV-2 infection. Therefore, our results reveal a remarkable adaptation of the S. aureus isolates to the COVID-19 pandemic context and demonstrates the potential of whole-genome sequencing to conduct molecular epidemiology studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571730" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571730">
        <p class="paperTitle">ChIP-DIP: A multiplexed method for mapping hundreds of proteins to DNA uncovers diverse regulatory elements controlling gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perez, A. A.; Goronzy, I. N.; Blanco, M. R.; Guo, J. K.; Guttman, M.</p>
        <p class="info">Score: 146.0, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571730' target='https://doi.org/10.1101/2023.12.14.571730'> 10.1101/2023.12.14.571730</a></p>
        <p class="abstract">Gene expression is controlled by the dynamic localization of thousands of distinct regulatory proteins to precise regions of DNA. Understanding this cell-type specific process has been a goal of molecular biology for decades yet remains challenging because most current DNA-protein mapping methods study one protein at a time. To overcome this, we developed ChIP-DIP (ChIP Done In Parallel), a split-pool based method that enables simultaneous, genome-wide mapping of hundreds of diverse regulatory proteins in a single experiment. We demonstrate that ChIP-DIP generates highly accurate maps for all classes of DNA-associated proteins, including histone modifications, chromatin regulators, transcription factors, and RNA Polymerases. Using these data, we explore quantitative combinations of protein localization on genomic DNA to define distinct classes of regulatory elements and their functional activity. Our data demonstrate that ChIP-DIP enables the generation of  consortium level, context-specific protein localization maps within any molecular biology lab.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572118">
        <p class="paperTitle">Atlas of telomeric repeat diversity in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tao, Y.; Xian, W.; Rabanal, F.; Movilli, A.; Lanz, C.; Shirsekar, G.; Weigel, D.</p>
        <p class="info">Score: 22.1, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572118' target='https://doi.org/10.1101/2023.12.18.572118'> 10.1101/2023.12.18.572118</a></p>
        <p class="abstract">Telomeric repeat arrays at the ends of chromosomes are highly dynamic but their repetitive nature and technological limitations have made it difficult to assess the variation in genome diversity surveys. Here we present a comprehensive characterization of the sequence variation immediately adjacent to the canonical telomeric repeat arrays at the very ends of chromosomes in 49 genetically diverse Arabidopsis thaliana accessions. We reveal several types of distinct telomeric repeat units and identify evolutionary processes such as local homogenization and higher-order repeat formation that shape diversity of chromosome ends. The identification of segmental duplications and at least one recombination event suggests a plausible history of telomerase-independent maintenance generation. By comparing largely isogenic samples, we are able to determine variant telomeric repeat number variation at both the germline and somatic levels. Analysis of haplotype structure uncovers chromosome end-specific as well as genetic group-specific patterns in telomeric repeat diversity and provides evidence for linkage disequilibrium between repeat arrays and their adjacent non-coding regions. Together, our findings illustrate the fine-scale telomeric repeat spectrum in A. thaliana, expanding our knowledge of the evolution of chromosome ends.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572568">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572568" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572568">
        <p class="paperTitle">A comparative analysis of planarian genomes reveals regulatory conservation in the face of rapid structural divergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572568" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572568" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ivankovic, M.; Brand, J. N.; Pandolfini, L.; Brown, T.; Pippel, M.; Rozanski, A.; Schubert, T.; Grohme, M. A.; Winkler, S.; Robledillo, L.; Zhang, M.; Codino, A.; Gustincich, S.; Vila-Farre, M.; Zhang, S.; Papantonis, A.; Marques, A.; Rink, J. C.</p>
        <p class="info">Score: 20.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572568' target='https://doi.org/10.1101/2023.12.22.572568'> 10.1101/2023.12.22.572568</a></p>
        <p class="abstract">The planarian Schmidtea mediterranea can regenerate its entire body from small tissue fragments and is studied as whole-body regeneration model species. While well-annotated genome resources are critical for understanding regeneration, the assembly and functional analysis of planarian genomes has proven challenging due to extreme compositional biases (70% A/T), high repeat contents and limited utility of routine laboratory protocols owing to their divergent biochemistry. Only few and often fragmented genome assemblies are currently available, and open challenges include the provision of well-annotated chromosome-scale reference assemblies of the model species and other planarians for a comparative genome evolution perspective. Here we report a haplotype-phased, chromosome-scale genome assembly and high-quality gene annotations of the sexual strain of S. mediterranea. We also provide putative regulatory region annotations via optimized ATAC-seq and ChIP-seq protocols. To additionally leverage sequence conservation for regulatory element annotations, we generated annotated chromosome-scale genome assemblies and chromatin accessibility data for the three closest relatives of S. mediterranea: S. polychroa, S. nova, and S. lugubris. Although we find broad sequence divergence between the genomes, including in open chromatin regions, ChIP-mark bearing S. mediterranea regulatory elements displayed significant sequence conservation. The resulting high-confidence set of evolutionary conserved enhancers and promoters provides a valuable resource for the analysis of gene regulatory circuits and their evolution within the taxon. In addition, our four chromosome-scale genome assemblies provide a first comparative perspective on planarian genome evolution. Our analyses reveal frequent retrotransposon-associated chromosomal inversions and inter-chromosomal translocations that lead to a degradation of synteny across the genus. Interestingly, we further find independent and near-complete losses of the ancestral metazoan synteny across Schmidtea and two other flatworm groups, indicating that platyhelminth genomes largely evolve without syntenic constraints. We provide valuable genome resources for the planarian research community and set a foundation for comparative genomics of planarians. We find that the rapid structural evolution of planarian genomes contrasts with the conservation of regulatory elements. Therefore, the unusual biology of flatworms appears paired with equally unique patterns of genome evolution, in which the position of genes in the genome may not matter.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572802">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572802" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572802">
        <p class="paperTitle">Annelid methylomes reveal ancestral developmental and ageing-associated epigenetic erosion across Bilateria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572802" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572802" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guynes, K.; Sarre, L. A.; Carrillo-Baltodano, A. M.; Davies, B. E.; Xu, L.; Liang, Y.; Martin-Zamora, F. M.; Hurd, P. J.; de Mendoza, A.; Martin-Duran, J. M.</p>
        <p class="info">Score: 9.8, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572802' target='https://doi.org/10.1101/2023.12.21.572802'> 10.1101/2023.12.21.572802</a></p>
        <p class="abstract">Background: DNA methylation in the form of 5-methylcytosine (5mC) is the most abundant base modification in animals. However, 5mC levels vary widely across taxa. While vertebrate genomes are hypermethylated, in most invertebrates, 5mC concentrates on constantly and highly transcribed genes (gene body methylation; GbM) and, in some species, on transposable elements (TEs), a pattern known as &#34;mosaic&#34;. Yet, the role and developmental dynamics of 5mC and how these explain interspecific differences in DNA methylation patterns remain poorly understood, especially in Spiralia, a large clade of invertebrates comprising nearly half of the animal phyla. Results: Here, we generate base-resolution methylomes for three species with distinct genomic features and phylogenetic positions in Annelida, a major spiralian phylum. All possible 5mC patterns occur in annelids, from typical invertebrate intermediate levels in a mosaic distribution to hypermethylation and methylation loss. GbM is common to annelids with 5mC, and methylation differences across species are explained by taxon-specific transcriptional dynamics or the presence of intronic TEs. Notably, the link between GbM and transcription decays during development, and there is a gradual and global, age-dependent demethylation in adult stages. Moreover, reducing 5mC levels with cytidine analogues during early development impairs normal embryogenesis and reactivates TEs in the annelid Owenia fusiformis. Conclusions: Our study indicates that global epigenetic erosion during development and ageing is an ancestral feature of bilateral animals. However, the tight link between transcription and gene body methylation is likely important in early embryonic stages, and 5mC-mediated TE silencing probably emerged convergently across animal lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571954">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571954" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571954">
        <p class="paperTitle">Image-based spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571954" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571954" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vannan, A.; Lyu, R.; Williams, A. L.; Negretti, N. M.; Mee, E. D.; Hirsh, J.; Hirsh, S.; Nichols, D. S.; Calvi, C. L.; Talor, C. J.; Polosukhin, V. V.; Serezani, A. P.; McCall, A. S.; Gokey, J. J.; Shim, H.; Ware, L. B.; Bacchetta, M. J.; Shaver, C. M.; Blackwell, T. S.; Walia, R.; Sucre, J. M.; Kropski, J. A.; McCarthy, D. J.; Banovich, N. E.</p>
        <p class="info">Score: 35.1, Published: 2023-12-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571954' target='https://doi.org/10.1101/2023.12.15.571954'> 10.1101/2023.12.15.571954</a></p>
        <p class="abstract">The human lung is structurally complex, with a diversity of specialized epithelial, stromal and immune cells playing specific functional roles in anatomically distinct locations, and large-scale changes in the structure and cellular makeup of this distal lung is a hallmark of pulmonary fibrosis (PF) and other progressive chronic lung diseases. Single-cell transcriptomic studies have revealed numerous disease-emergent/enriched cell types/states in PF lungs, but the spatial contexts wherein these cells contribute to disease pathogenesis has remained uncertain. Using sub-cellular resolution image-based spatial transcriptomics, we analyzed the gene expression of more than 1 million cells from 19 unique lungs. Through complementary cell-based and innovative cell-agnostic analyses, we characterized the localization of PF-emergent cell-types, established the cellular and molecular basis of classical PF histopathologic disease features, and identified a diversity of distinct molecularly-defined spatial niches in control and PF lungs. Using machine-learning and trajectory analysis methods to segment and rank airspaces on a gradient from normal to most severely remodeled, we identified a sequence of compositional and molecular changes that associate with progressive distal lung pathology, beginning with alveolar epithelial dysregulation and culminating with changes in macrophage polarization. Together, these results provide a unique, spatially-resolved characterization of the cellular and molecular programs of PF and control lungs, provide new insights into the heterogeneous pathobiology of PF, and establish analytical approaches which should be broadly applicable to other imaging-based spatial transcriptomic studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572730" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572730">
        <p class="paperTitle">Characterizing uncertainty in predictions of genomic sequence-to-activity models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bajwa, A.; Rastogi, R.; Kathail, P.; Shuai, R. W.; Ioannidis, N. M.</p>
        <p class="info">Score: 4.6, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572730' target='https://doi.org/10.1101/2023.12.21.572730'> 10.1101/2023.12.21.572730</a></p>
        <p class="abstract">Genomic sequence-to-activity models are increasingly utilized to understand gene regulatory syntax and probe the functional consequences of regulatory variation. Current models make accurate predictions of relative activity levels across the human reference genome, but their performance is more limited for predicting the effects of genetic variants, such as explaining gene expression variation across individuals. To better understand the causes of these shortcomings, we examine the uncertainty in predictions of genomic sequence-to-activity models using an ensemble of Basenji2 model replicates. We characterize prediction consistency on four types of sequences: reference genome sequences, reference genome sequences perturbed with TF motifs, eQTLs, and personal genome sequences. We observe that models tend to make high-confidence predictions on reference sequences, even when incorrect, and low-confidence predictions on sequences with variants. For eQTLs and personal genome sequences, we find that model replicates make inconsistent predictions in &gt;50% of cases. Our findings suggest strategies to improve performance of these models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572682">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572682" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572682">
        <p class="paperTitle">A validated pangenome-scale metabolic model for the Klebsiella pneumoniae species complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572682" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572682" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cooper, H. B.; Vezina, B.; Hawkey, J.; Passet, V.; Lopez-Fernandez, S.; Monk, J. M.; Brisse, S.; Holt, K. E.; Wyres, K. L.</p>
        <p class="info">Score: 4.5, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572682' target='https://doi.org/10.1101/2023.12.20.572682'> 10.1101/2023.12.20.572682</a></p>
        <p class="abstract">The Klebsiella pneumoniae Species Complex (KpSC) is a major source of nosocomial infections globally with high rates of resistance to antimicrobials. Consequently, there is growing interest in understanding virulence factors and their association with cellular metabolic processes for developing novel anti-KpSC therapeutics. Phenotypic assays have revealed metabolic diversity within the KpSC, but metabolism research has been neglected due to experiments being difficult and cost-intensive.

Genome-scale metabolic models (GSMMs) represent a rapid and scalable in silico approach for exploring metabolic diversity, which compiles genomic and biochemical data to reconstruct the metabolic network of an organism. Here we use a diverse collection of 507 KpSC isolates, including representatives of globally distributed clinically-relevant lineages, to construct the most comprehensive KpSC pan-metabolic model to-date, KpSC pan v2. Candidate metabolic reactions were identified using gene orthology to known metabolic genes, prior to manual curation via extensive literature and database searches. The final model comprised a total of 3,550 reactions, 2,403 genes and can simulate growth on 360 unique substrates. We used KpSC pan v2 as a reference to derive strain-specific GSMMs for all 507 KpSC isolates, and compared these to GSMMs generated using a prior KpSC pan-reference (KpSC pan v1) and two single-strain references. We show that KpSC pan v2 includes a greater proportion of accessory reactions (8.8%) than KpSC pan v1 (2.5%). GSMMs derived from KpSC pan v2 also result in more accuracy growth predictions than those derived from other references in both aerobic (median accuracy = 95.4%) and anaerobic (median accuracy = 78.8%). KpSC pan v2 also generates more accurate growth predictions, with high median accuracies of 95.4% (aerobic, n=37 isolates) and 78.8% (anaerobic, n=36 isolates) for 124 matched carbon substrates.

KpSC pan v2 is freely available at https://github.com/kelwyres/KpSC-pan-metabolic-model, representing a valuable resource for the scientific community, both as a source of curated metabolic information and as a reference to derive accurate strain-specific GSMMs. The latter can be used to investigate the relationship between KpSC metabolism and traits of interest, such as reservoirs, epidemiology, drug resistance or virulence, and ultimately to inform novel KpSC control strategies.

Significance as a BioResource to the communityKlebsiella pneumoniae and its close relatives in the K. pneumoniae Species Complex (KpSC) are priority antimicrobial resistant pathogens that exhibit extensive genomic diversity. There is growing interest in understanding KpSC metabolism, and genome scale metabolic models (GSMMs) provide a rapid, scalable option for exploration of whole cell metabolism plus phenotype prediction. Here we present a KpSC pan-metabolic model representing the cellular metabolism of 507 diverse KpSC isolates. Our model is the largest and most comprehensive of its kind, comprising &gt;2,400 genes associated with &gt;3,500 metabolic reactions, plus manually curated evidence annotations. These data alone represent a key knowledge resource for the Klebsiella research community; however, our models greatest impact lies in its potential for use as a reference from which highly accurate strain-specific GSMMs can be derived to inform in depth strain-specific and/or large-scale comparative analyses.

Data summaryO_LIKlebsiella pneumoniae species complex (KpSC) pan v2 metabolic model available at https://github.com/kelwyres/KpSC-pan-metabolic-model.
C_LIO_LIAll KpSC isolate whole genome sequences used in this work were reported previously and are available under Bioprojects PRJEB6891, PRJNA351909, PRJNA493667, PRJNA768294, PRJNA253462, PRJNA292902 and PRJNA391323. Individual accessions listed in Table S1.
C_LIO_LIStrain-specific GSMMs used for comparative analyses (deposited in Figshare - 10.6084/m9.figshare.24871914), plus their associated MEMOTE reports (indicates completeness and annotation quality), reaction and gene presence-absence matrices across all isolates.
C_LIO_LIGrowth phenotype predictions derived from strain-specific GSMMs are available in Table S4.
C_LIO_LIBinarised Biolog growth phenotype data for n=37 isolates (plates PM1 and PM2, aerobic and anaerobic conditions) are available in Tables S6 &amp; S7.
C_LIO_LIAdditional growth assay data for six substrates not included on Biolog plates PM1 and PM2 (deposited in Figshare - 10.6084/m9.figshare.24871914).
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.572587">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.572587" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.572587">
        <p class="paperTitle">CRISPRmap: Sequencing-free optical pooled screens mapping multi-omic phenotypes in cells and tissue</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.572587" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.572587" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gu, J.; Iyer, A.; Wesley, B.; Taglialatela, A.; Leuzzi, G.; Hangai, S.; Decker, A.; Gu, R.; Klickstein, N.; Shuai, Y.; Jankovic, K.; Parker-Burns, L.; Jin, Y.; Zhang, J. Y.; Hong, J.; Niu, X.; Chou, J.; Landau, D. A.; Azizi, E.; Chan, E. M.; Ciccia, A.; Gaublomme, J. T.</p>
        <p class="info">Score: 4.3, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.572587' target='https://doi.org/10.1101/2023.12.26.572587'> 10.1101/2023.12.26.572587</a></p>
        <p class="abstract">Pooled genetic screens are powerful tools to study gene function in a high-throughput manner. Typically, sequencing-based screens require cell lysis, which limits the examination of critical phenotypes such as cell morphology, protein subcellular localization, and cell-cell/tissue interactions. In contrast, emerging optical pooled screening methods enable the investigation of these spatial phenotypes in response to targeted CRISPR perturbations. In this study, we report a multi-omic optical pooled CRISPR screening method, which we have named CRISPRmap. Our method combines a novel in situ CRISPR guide identifying barcode readout approach with concurrent multiplexed immunofluorescence and in situ RNA detection. CRISPRmap barcodes are detected and read out through combinatorial hybridization of DNA oligos, enhancing barcode detection efficiency, while reducing both dependency on third party proprietary sequencing reagents and assay cost. Notably, we conducted a multi-omic base-editing screen in a breast cancer cell line on core DNA damage repair genes involved in the homologous recombination and Fanconi anemia pathways investigating how nucleotide variants in those genes influence DNA damage signaling and cell cycle regulation following treatment with ionizing radiation or DNA damaging agents commonly used for cancer therapy. Approximately a million cells were profiled with our multi-omic approach, providing a comprehensive phenotypic assessment of the functional consequences of the studied variants. CRISPRmap enabled us to pinpoint likely-pathogenic patient-derived mutations that were previously classified as variants of unknown clinical significance. Furthermore, our approach effectively distinguished barcodes of a pooled library in tumor tissue, and we coupled it with cell-type and molecular phenotyping by cyclic immunofluorescence. Multi-omic spatial analysis of how CRISPR-perturbed cells respond to various environmental cues in the tissue context offers the potential to significantly expand our understanding of tissue biology in both health and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.573260">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.573260" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.573260">
        <p class="paperTitle">Functional characterization of eQTLs and asthma risk loci with scATAC-seq across immune cell types and contexts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.573260" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.573260" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wei, J.; Resztak, J. A.; Alazizi, A.; Mair-Meijers, H. E.; Slatcher, R. B.; Zilioli, S.; Wen, X.; Luca, F.; Pique-Regi, R.</p>
        <p class="info">Score: 4.2, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.573260' target='https://doi.org/10.1101/2023.12.24.573260'> 10.1101/2023.12.24.573260</a></p>
        <p class="abstract">Cis-regulatory elements (CREs) control gene transcription dynamics across cell types and in response to the environment. In asthma, multiple immune cell types play an important role in the inflammatory process. Genetic variants in CREs can also affect gene expression response dynamics and contribute to asthma risk. However, the regulatory mechanisms underlying control of transcriptional dynamics across different environmental contexts and cell-types at single cell resolution remains to be elucidated. To resolve this question, we performed scATAC-seq in activated peripheral blood mononuclear cells (PBMC) from children with asthma with phytohemagglutinin (PHA) or lipopolysaccharide (LPS), and treated with dexamethasone (DEX), an anti-inflammatory glucocorticoid. We analyzed changes in chromatin accessibility, measured transcription factor motif activity, and identified treatment and cell-type specific transcription factors that drive changes in both gene expression mean and variability. We observed strong positive linear dependence between motif response and their target gene expression changes, but negative in variability changes. This result suggests that an increase of transcription factor binding tightens the variability of gene expression around the mean. We then annotated genetic variants in chromatin accessibility peaks and response motifs followed by computational fine-mapping of eQTL signals from a pediatric asthma cohort. We found that eQTLs were highly enriched in peaks with response motifs and refined the credible set of several asthma risk genes. In conclusion, scATAC-seq enhances the understanding of molecular mechanisms for asthma risk variants mediated by gene expression.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
