<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Randomizing the human genome by engineering recombination between repeat elements
Authors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.
Score: 97.3, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576745
While protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Randomizing the human genome by engineering recombination between repeat elements
Authors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.
Score: 97.3, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576745
While protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Randomizing the human genome by engineering recombination between repeat elements
Authors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.
Score: 97.3, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576745
While protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Randomizing the human genome by engineering recombination between repeat elements\nAuthors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.\nScore: 97.3, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576745\nWhile protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent.",
  "keywords": [
    
  ],
  "articleBody": " Randomizing the human genome by engineering recombination between repeat elements\nAuthors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.\nScore: 97.3, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576745\nWhile protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent. To bridge this gap, we have developed a toolbox to create deletions, inversions, translocations, and extrachromosomal circular DNA at scale by highly multiplexed insertion of recombinase recognition sites into repetitive sequences with CRISPR prime editing. Using this strategy, we derived stable human cell lines with several thousand clonal insertions, the highest number of novel sequences inserted into single human genomes. Subsequent recombinase induction generated an average of more than one hundred megabase-sized rearrangements per cell, and thousands across the whole population. The ability to detect rearrangements as they are generated and to track their abundance over time allowed us to measure the selection pressures acting on different types of structural changes. We observed a consolidation towards shorter variants that preferentially delete growth-inhibiting genes and a depletion of translocations. We isolated and characterized 21 clones with multiple recombinase-induced rearrangements. These included viable haploid clones with deletions that span hundreds of kilobases as well as triploid HEK293T clones with aneuploidies and fold back chromosomes. We mapped the impact of these genetic changes on gene expression to decipher how structural variants affect gene regulation. The genome scrambling strategy developed here makes it possible to delete megabases of sequence, move sequences between and within chromosomes, and implant regulatory elements into new contexts which will shed light on the genome organization principles of humans and other species.\nMultiplex generation and single cell analysis of structural variants in a mammalian genome\nAuthors: Pinglay, S.; Lalanne, J.-B.; Daza, R. M.; Koeppel, J.; Li, X.; Lee, D. S.; Shendure, J.\nScore: 59.3, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576756\nThe functional consequences of structural variants (SVs) in mammalian genomes are challenging to study. This is due to several factors, including: 1) their numerical paucity relative to other forms of standing genetic variation such as single nucleotide variants (SNVs) and short insertions or deletions (indels); 2) the fact that a single SV can involve and potentially impact the function of more than one gene and/or cis regulatory element; and 3) the relative immaturity of methods to generate SVs, either randomly or in targeted fashion, in in vitro or in vivo model systems. Towards addressing some of these challenges, we developed Genome-Shuffle-seq, a straightforward method that enables the multiplex generation of several major forms of SVs (deletions, inversions, translocations) throughout a mammalian genome. Genome-Shuffle-seq is based on the random integration of \"shuffle cassettes\" to the genome, wherein each shuffle cassette contains components that facilitate its site-specific recombination (SSR) with other integrated shuffle cassettes (via Cre-loxP), its mapping to a specific genomic location (via T7-mediated in vitro transcription or IVT), and its identification in single-cell RNA-seq (scRNA-seq) data (via T7-mediated in situ transcription or IST). In this proof-of-concept, we apply Genome-Shuffle-seq to induce and map thousands of genomic SVs in mouse embryonic stem cells (mESCs) in a single experiment. Induced SVs are rapidly depleted from the cellular population over time, possibly due to some combination of Cre-mediated toxicity and negative selection on the rearrangements themselves. We demonstrate that we can efficiently genotype which SVs are present in association with each of many single cell transcriptomes in scRNA-seq data. Finally, preliminary evidence suggests our method may be a powerful means of generating extrachromosomal circular DNAs (ecDNAs). Looking forward, we anticipate that Genome-Shuffle-seq may be broadly useful for the systematic exploration of the functional consequences of SVs on gene expression, the chromatin landscape, and 3D nuclear architecture. We further anticipate potential uses for in vitro modeling of ecDNAs, as well as in paving the path to a minimal mammalian genome.\nA Comprehensive Resource for Exploring Antiphage Defense: DefenseFinder Webservice, Wiki and Databases.\nAuthors: Tesson, F.; Planel, R.; Egorov, A. A.; Georjon, H.; Vaysset, H.; Brancotte, B.; Neron, B.; Mordret, E.; Atkinson, G. C.; Bernheim, A.; Cury, J.\nScore: 36.0, Published: 2024-01-26 DOI: 10.1101/2024.01.25.577194\nIn recent years, a vast number of novel antiphage defense mechanisms were uncovered. To facilitate the exploration of mechanistic, ecological, and evolutionary aspects related to antiphage defense systems, we released DefenseFinder in 2021 (Tesson et al., 2022). DefenseFinder is a bioinformatic program designed for the systematic identification of all known antiphage defense mechanisms. The initial release of DefenseFinder v1.0.0 included 60 systems. Over the past three years, the number of antiphage systems incorporated into DefenseFinder has grown to 152. The increasing number of known systems makes it a challenge to enter the field and makes the interpretation of detections of antiphage systems difficult. Moreover, the rapid development of sequence-based predictions of structures offers novel possibilities of analysis and should be easily available. To overcome these challenges, we present a hub of resources on defense systems including 1) a search tool (DefenseFinder as a web service) 2) a knowledge base, and 3) precomputed databases (results of DefenseFinder ran on RefSeq database, predicted structure database computed with AlphaFold). These pages can be freely accessed for users as a starting point on their journey to better understand a given system. We anticipate that these resources will foster the use of bioinformatics in the study of antiphage systems and will serve the community of researchers who study antiphage systems. This resource is available at: https://defensefinder.mdmlab.fr.\nAggregation of recount3 RNA-seq data improves the inference of consensus and tissue-specific gene co-expression networks.\nAuthors: Ravichandran, P.; Parsana, P.; Keener, R.; Hansen, K.; Battle, A.\nScore: 14.9, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576447\nBackgroundGene co-expression networks (GCNs) describe relationships among expressed genes key to maintaining cellular identity and homeostasis. However, the small sample size of typical RNA-seq experiments which is several orders of magnitude fewer than the number of genes is too low to infer GCNs reliably. recount3, a publicly available dataset comprised of 316,443 uniformly processed human RNA-seq samples, provides an opportunity to improve power for accurate network reconstruction and obtain biological insight from the resulting networks. ResultsWe compared alternate aggregation strategies to identify an optimal workflow for GCN inference by data aggregation and inferred three consensus networks: a universal network, a non-cancer network, and a cancer network in addition to 27 tissue context-specific networks. Central network genes from our consensus networks were enriched for evolutionarily constrained genes and ubiquitous biological pathways, whereas central context-specific network genes included tissue-specific transcription factors and factorization based on the hubs led to clustering of related tissue contexts. We discovered that annotations corresponding to context-specific networks inferred from aggregated data were enriched for trait heritability beyond known functional genomic annotations and were significantly more enriched when we aggregated over a larger number of samples. ConclusionThis study outlines best practices for network GCN inference and evaluation by data aggregation. We recommend estimating and regressing confounders in each data set before aggregation and prioritizing large sample size studies for GCN reconstruction. Increased statistical power in inferring context-specific networks enabled the derivation of variant annotations that were enriched for concordant trait heritability independent of functional genomic annotations that are context-agnostic. While we observed strictly increasing held-out log-likelihood with data aggregation, we noted diminishing marginal improvements. Future directions aimed at alternate methods for estimating confounders and integrating orthogonal information from modalities such as Hi-C and ChIP-seq can further improve GCN inference.\nWhole-genome phylogenomics of the tinamous (Aves: Tinamidae): comparing gene tree estimation error between BUSCOs and UCEs illuminates rapid divergence with introgression\nAuthors: Musher, L. J.; Catanach, T. A.; Valqui, T.; Brumfield, R. T.; Aleixo, A.; Johnson, K. P.; Weckstein, J. D.\nScore: 14.7, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576737\nIncomplete lineage sorting (ILS) and introgression increase genealogical discordance across the genome, which complicates phylogenetic inference. In such cases, identifying orthologs that result in gene trees with low estimation error is crucial because phylogenomic methods rely on accurate gene histories. We sequenced whole genomes for the tinamous (Aves: Tinamidae) to dissect the sources of gene and species-tree discordance and reconstruct their interrelationships. We compared results based on four ortholog sets: (1) coding genes (BUSCOs), (2) ultraconserved elements (UCEs) with short flanking regions, (3) UCEs with intermediate flanks, and (4) UCEs with long flanks. We hypothesized that orthologs with more phylogenetically informative sites would result in more accurate species trees because the resulting gene trees contain lower error. Consistent with our hypothesis, we found that long UCEs had the most informative sites and lowest rates of error. However, despite having many informative sites, BUSCO gene trees contained high error compared to long UCEs. Unlike UCEs, BUSCO gene sequences showed a positive association between the proportion of parsimony informative sites and gene tree error. Thus, BUSCO and UCE datasets have different underlying properties of molecular evolution, and these differences should be considered when selecting loci for phylogenomic analysis. Still, species trees from different datasets were mostly congruent. Only one clade, with a history of ILS and introgression, exhibited substantial species-tree discordance across the different data sets. Overall, we present the most complete phylogeny for tinamous to date, identify a new species, and provide a case study for species-level phylogenomic analysis using whole-genomes.\nA high coverage Mesolithic aurochs genome and effective leveraging of ancient cattle genomes using whole genome imputation.\nAuthors: Erven, J. A. M.; Scheu, A.; Verdugo, M. P.; Cassidy, L.; Chen, N.; Gehlen, B.; Street, M.; Madsen, O.; Mullin, V. E.\nScore: 14.1, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576850\nAncient genomic analyses are often restricted to utilising pseudo-haploid data due to low genome coverage. Leveraging low coverage data by imputation to calculate phased diploid genotypes that enable haplotype-based interrogation and SNP calling at unsequenced positions is highly desirable. This has not been investigated for ancient cattle genomes despite these being compelling subjects for archaeological, evolutionary and economic reasons. Here we test this approach by sequencing a Mesolithic European aurochs (18.49x; 9852-9376 calBC), an Early Medieval European cow (18.69x; 427-580 calAD), and combine these with published individuals; two ancient and three modern. We downsample these genomes (0.25x, 0.5x, 1.0x, 2.0x) and impute diploid genotypes, utilising a reference panel of 171 published modern cattle genomes that we curated for 21.7 million (Mn) phased single-nucleotide polymorphisms (SNPs). We recover high densities of correct calls with an accuracy of \u003e99.1% at variant sites for the lowest downsample depth of 0.25x, increasing to \u003e99.5% for 2.0x (transversions only, minor allele frequency (MAF) [\u0026ge;]2.5%). The recovery of SNPs correlates with coverage, on average 58% of sites are recovered for 0.25x increasing to 87% for 2.0x, utilising an average of 3.5 million (Mn) transversions (MAF [\u0026ge;]2.5%), even in the aurochs which is temporally and morphologically distinct from the reference panel. Our imputed genomes behave similarly to directly called data in allele-frequency-based analyses; for example consistently identifying runs of homozygosity \u003e2mb, including a long homozygous region in the Mesolithic European aurochs.\nEpigenetic reprogramming driving successful and failed repair in acute kidney injury\nAuthors: Muto, Y.; Dixon, E. E.; Yoshimura, Y.; Ledru, N.; Kirita, Y.; Wu, H.; Humphreys, B. D.\nScore: 13.0, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576421\nAcute kidney injury (AKI) causes epithelial damage followed by subsequent repair. While successful repair restores kidney function, this process is often incomplete and can lead to chronic kidney disease (CKD) in a process called failed repair. To better understand the epigenetic reprogramming driving this AKI-to-CKD transition we generated a single nucleus multiomic atlas for the full mouse AKI time course, consisting of [~]280,000 single nucleus transcriptomes and epigenomes. We reveal cell-specific dynamic alterations in gene regulatory landscapes reflecting especially activation of proinflammatory pathways. We further generated single nucleus multiomic data from four human AKI samples including validation by genome-wide identification of NF-kB binding sites. A regularized regression analysis identifies key regulators involved in both successful and failed repair cell fate, identifying the transcription factor CREB5 as a regulator of both successful and failed tubular repair that also drives proximal tubule cell proliferation after injury. Our interspecies multiomic approach provides a foundation to comprehensively understand cell states in AKI.\nDORA: an interactive map for the visualization and analysis of ancient human DNA and associated data\nAuthors: Harris, K. D.; Greenbaum, G.\nScore: 20.2, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575663\nThe ability to sequence ancient genomes has revolutionized the way in which we study evolutionary history by providing access to the most important aspect of evolution -- time. Until recently, studying human demography, ecology, biology, and history using population genomic inference relied on contemporary genomic datasets. Over the past decade, the availability of human ancient DNA (aDNA) has increased rapidly, almost doubling every year, opening the way for spatiotemporal studies of ancient human populations. However, the multidimensionality of aDNA, with genotypes having temporal, spatial and genomic coordinates, and the need to integrate multiple sources of data, poses a challenge for developing meta-analyses pipelines. To address this challenge, we developed a publicly-available interactive tool, DORA, which integrates multiple data types, genomic and non-genomic, in a unified interface. This web-based tool allows users to browse sample metadata along with additional layers of information, such as population structure, climatic data, and unpublished samples. Users can then perform analyses on genotypes of these samples, or export sample subsets for external analyses. DORA integrates analyses and visualizations in a single intuitive interface, resolving the technical issues of combining datasets from different sources and formats, and allowing researchers to focus on analysis and the scientific questions that can be addressed through analysis of aDNA datasets.\nGenomic diversity and antimicrobial resistance in clinical Klebsiella pneumoniae isolates from tertiary hospitals in Southern Ghana\nAuthors: Mills, R. O.; Dadzie, I.; Le-Viet, T.; Baker, D. J.; Addy, H. P. K.; Akwetey, S. A.; Donkoh, I. E.; Quansah, E.; Semanshia, P. S.; Morgan, J.; Mensah, A.; Adade, N. E.; Ampah, E. O.; Owusu, E.; Mwintige, P.; Amoako, E. O.; Spadar, A.; Holt, K. E.; Foster-Nyarko, E.\nScore: 12.7, Published: 2024-01-22 DOI: 10.1101/2024.01.20.576413\nComprehensive data on the genomic epidemiology of hospital-associated Klebsiella pneumoniae in Ghana is scarce. This study sequenced 103 clinical K. pneumoniae isolates from five tertiary hospitals in Southern Ghana, predominantly from paediatric patients under five years (67/103, 65%), with the majority collected from urine (32/103, 31%) and blood (25/103, 24%) cultures. We employed Pathogenwatch for genotyping via Kaptive (K/O antigens) and Kleborate (antimicrobial resistance and hypervirulence) and determined clonal relationships using core-genome multilocus sequence typing (cgMLST). Among the 44 distinct sequence types (STs) detected, ST133 was the most common, comprising 23% of isolates (n=23/103). We discovered 27 different capsular (K) locus antigens and seven lipopolysaccharide (O) types; KL116 (28/103, 27%) and O1 (66/103, 64%) were the most prevalent. Single-linkage clustering highlighted the global spread of multidrug-resistant clones such as ST15, ST307, ST17, ST11, ST101, and ST48, with minimal allele differences (1-5) from publicly available genomes worldwide. Conversely, several isolates (n=17) constituted novel clonal groups and lacked close relatives among publicly available genomes, displaying unique genetic diversity within our study population. A significant proportion of isolates (88/103, 85%) carried resistance genes for three or more antibiotic classes, with the blaCTXM-15 gene present in 78% (n=80/103). Carbapenem resistance, predominantly due to blaOXA-181 and blaNDM-1 genes, was found in 10% (n=10/103) of the isolates. Yersiniabactin was the predominant acquired virulence trait, identified in 70% (n=72/103) of the isolates. Our findings reveal a complex genomic landscape of K. pneumoniae in Southern Ghana, underscoring the critical need for ongoing genomic surveillance to manage the substantial burden of antimicrobial resistance.\nA complex Plasmodium falciparum cryptotype circulating at low frequency across the African continent\nAuthors: Miotto, O.; Amambua-Ngwa, A.; Amenga-Etego, L. N.; Abdel Hamid, M. M.; Adam, I.; Aninagyei, E.; Apinjoh, T.; Awandare, G. A.; Bejon, P.; Bertin, G.; Bouyou-Akotet, M. K.; Claessens, A.; Conway, D.; D'Alessandro, U.; Diakite, M.; Djimde, A.; Dondorp, A. M.; Duffy, P.; Fairhurst, R. M.; Fanello, C. I.; Ghansah, A.; Ishengoma, D.; Lawniczak, M.; Maiga-Ascofare, O.; Auburn, S.; Rosanas-Urgell, A.; Wasakul, V.; White, N.; Almagro-Garcia, J.; Pearson, R.; Goncalves, S.; Ariani, C.; Bozdech, Z.; Hamilton, W. L.; Simpson, V.; Kwiatkowski, D. P.\nScore: 11.6, Published: 2024-01-22 DOI: 10.1101/2024.01.20.576496\nThe population structure of the malaria parasite Plasmodium falciparum can reveal underlying demographic and adaptive evolutionary processes. Here, we analyse population structure in 4,376 P. falciparum genomes from 21 countries across Africa. We identified a strongly differentiated cluster of parasites, comprising [~]1.2% of samples analysed, geographically distributed over 13 countries across the continent. Members of this cluster, named AF1, carry a genetic background consisting of a large number of highly differentiated variants, rarely observed outside this cluster, at a multitude of genomic loci distributed across most chromosomes. At these loci, the AF1 haplotypes appear to have common ancestry, irrespective of the sampling location; outside the shared loci, however, AF1 members are genetically similar to their sympatric parasites. AF1 parasites sharing up to 23 genomic co-inherited regions were found in all major regions of Africa, at locations over 7,000 km apart. We coined the term cryptotype to describe a complex common background which is geographically widespread, but concealed by genomic regions of local origin. Most AF1 differentiated variants are functionally related, comprising structural variations and single nucleotide polymorphisms in components of the MSP1 complex and several other genes involved in interactions with red blood cells, including invasion and erythrocyte antigen export. We propose that AF1 parasites have adapted to some as yet unidentified evolutionary niche, by acquiring a complex compendium of interacting variants that rarely circulate separately in Africa. As the cryptotype spread across the continent, it appears to have been maintained mostly intact in spite of recombination events, suggesting a selective advantage. It is possible that other cryptotypes circulate in Africa, and new analysis methods may be needed to identify them.\n",
  "wordCount" : "2955",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576745">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576745" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576745">
        <p class="paperTitle">Randomizing the human genome by engineering recombination between repeat elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576745" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576745" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.</p>
        <p class="info">Score: 97.3, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576745' target='https://doi.org/10.1101/2024.01.22.576745'> 10.1101/2024.01.22.576745</a></p>
        <p class="abstract">While protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent. To bridge this gap, we have developed a toolbox to create deletions, inversions, translocations, and extrachromosomal circular DNA at scale by highly multiplexed insertion of recombinase recognition sites into repetitive sequences with CRISPR prime editing. Using this strategy, we derived stable human cell lines with several thousand clonal insertions, the highest number of novel sequences inserted into single human genomes. Subsequent recombinase induction generated an average of more than one hundred megabase-sized rearrangements per cell, and thousands across the whole population. The ability to detect rearrangements as they are generated and to track their abundance over time allowed us to measure the selection pressures acting on different types of structural changes. We observed a consolidation towards shorter variants that preferentially delete growth-inhibiting genes and a depletion of translocations. We isolated and characterized 21 clones with multiple recombinase-induced rearrangements. These included viable haploid clones with deletions that span hundreds of kilobases as well as triploid HEK293T clones with aneuploidies and fold back chromosomes. We mapped the impact of these genetic changes on gene expression to decipher how structural variants affect gene regulation. The genome scrambling strategy developed here makes it possible to delete megabases of sequence, move sequences between and within chromosomes, and implant regulatory elements into new contexts which will shed light on the genome organization principles of humans and other species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576756" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576756">
        <p class="paperTitle">Multiplex generation and single cell analysis of structural variants in a mammalian genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pinglay, S.; Lalanne, J.-B.; Daza, R. M.; Koeppel, J.; Li, X.; Lee, D. S.; Shendure, J.</p>
        <p class="info">Score: 59.3, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576756' target='https://doi.org/10.1101/2024.01.22.576756'> 10.1101/2024.01.22.576756</a></p>
        <p class="abstract">The functional consequences of structural variants (SVs) in mammalian genomes are challenging to study. This is due to several factors, including: 1) their numerical paucity relative to other forms of standing genetic variation such as single nucleotide variants (SNVs) and short insertions or deletions (indels); 2) the fact that a single SV can involve and potentially impact the function of more than one gene and/or cis regulatory element; and 3) the relative immaturity of methods to generate SVs, either randomly or in targeted fashion, in in vitro or in vivo model systems. Towards addressing some of these challenges, we developed Genome-Shuffle-seq, a straightforward method that enables the multiplex generation of several major forms of SVs (deletions, inversions, translocations) throughout a mammalian genome. Genome-Shuffle-seq is based on the random integration of &#34;shuffle cassettes&#34; to the genome, wherein each shuffle cassette contains components that facilitate its site-specific recombination (SSR) with other integrated shuffle cassettes (via Cre-loxP), its mapping to a specific genomic location (via T7-mediated in vitro transcription or IVT), and its identification in single-cell RNA-seq (scRNA-seq) data (via T7-mediated in situ transcription or IST). In this proof-of-concept, we apply Genome-Shuffle-seq to induce and map thousands of genomic SVs in mouse embryonic stem cells (mESCs) in a single experiment. Induced SVs are rapidly depleted from the cellular population over time, possibly due to some combination of Cre-mediated toxicity and negative selection on the rearrangements themselves. We demonstrate that we can efficiently genotype which SVs are present in association with each of many single cell transcriptomes in scRNA-seq data. Finally, preliminary evidence suggests our method may be a powerful means of generating extrachromosomal circular DNAs (ecDNAs). Looking forward, we anticipate that Genome-Shuffle-seq may be broadly useful for the systematic exploration of the functional consequences of SVs on gene expression, the chromatin landscape, and 3D nuclear architecture. We further anticipate potential uses for in vitro modeling of ecDNAs, as well as in paving the path to a minimal mammalian genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577194">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577194" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577194">
        <p class="paperTitle">A Comprehensive Resource for Exploring Antiphage Defense: DefenseFinder Webservice, Wiki and Databases.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577194" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577194" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tesson, F.; Planel, R.; Egorov, A. A.; Georjon, H.; Vaysset, H.; Brancotte, B.; Neron, B.; Mordret, E.; Atkinson, G. C.; Bernheim, A.; Cury, J.</p>
        <p class="info">Score: 36.0, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577194' target='https://doi.org/10.1101/2024.01.25.577194'> 10.1101/2024.01.25.577194</a></p>
        <p class="abstract">In recent years, a vast number of novel antiphage defense mechanisms were uncovered. To facilitate the exploration of mechanistic, ecological, and evolutionary aspects related to antiphage defense systems, we released DefenseFinder in 2021 (Tesson et al., 2022). DefenseFinder is a bioinformatic program designed for the systematic identification of all known antiphage defense mechanisms. The initial release of DefenseFinder v1.0.0 included 60 systems. Over the past three years, the number of antiphage systems incorporated into DefenseFinder has grown to 152. The increasing number of known systems makes it a challenge to enter the field and makes the interpretation of detections of antiphage systems difficult. Moreover, the rapid development of sequence-based predictions of structures offers novel possibilities of analysis and should be easily available. To overcome these challenges, we present a hub of resources on defense systems including 1) a search tool (DefenseFinder as a web service) 2) a knowledge base, and 3) precomputed databases (results of DefenseFinder ran on RefSeq database, predicted structure database computed with AlphaFold). These pages can be freely accessed for users as a starting point on their journey to better understand a given system. We anticipate that these resources will foster the use of bioinformatics in the study of antiphage systems and will serve the community of researchers who study antiphage systems. This resource is available at: https://defensefinder.mdmlab.fr.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576447" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576447">
        <p class="paperTitle">Aggregation of recount3 RNA-seq data improves the inference of consensus and tissue-specific gene co-expression networks.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ravichandran, P.; Parsana, P.; Keener, R.; Hansen, K.; Battle, A.</p>
        <p class="info">Score: 14.9, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576447' target='https://doi.org/10.1101/2024.01.20.576447'> 10.1101/2024.01.20.576447</a></p>
        <p class="abstract">BackgroundGene co-expression networks (GCNs) describe relationships among expressed genes key to maintaining cellular identity and homeostasis. However, the small sample size of typical RNA-seq experiments which is several orders of magnitude fewer than the number of genes is too low to infer GCNs reliably. recount3, a publicly available dataset comprised of 316,443 uniformly processed human RNA-seq samples, provides an opportunity to improve power for accurate network reconstruction and obtain biological insight from the resulting networks.

ResultsWe compared alternate aggregation strategies to identify an optimal workflow for GCN inference by data aggregation and inferred three consensus networks: a universal network, a non-cancer network, and a cancer network in addition to 27 tissue context-specific networks. Central network genes from our consensus networks were enriched for evolutionarily constrained genes and ubiquitous biological pathways, whereas central context-specific network genes included tissue-specific transcription factors and factorization based on the hubs led to clustering of related tissue contexts. We discovered that annotations corresponding to context-specific networks inferred from aggregated data were enriched for trait heritability beyond known functional genomic annotations and were significantly more enriched when we aggregated over a larger number of samples.

ConclusionThis study outlines best practices for network GCN inference and evaluation by data aggregation. We recommend estimating and regressing confounders in each data set before aggregation and prioritizing large sample size studies for GCN reconstruction. Increased statistical power in inferring context-specific networks enabled the derivation of variant annotations that were enriched for concordant trait heritability independent of functional genomic annotations that are context-agnostic. While we observed strictly increasing held-out log-likelihood with data aggregation, we noted diminishing marginal improvements. Future directions aimed at alternate methods for estimating confounders and integrating orthogonal information from modalities such as Hi-C and ChIP-seq can further improve GCN inference.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576737">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576737" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576737">
        <p class="paperTitle">Whole-genome phylogenomics of the tinamous (Aves: Tinamidae): comparing gene tree estimation error between BUSCOs and UCEs illuminates rapid divergence with introgression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576737" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576737" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Musher, L. J.; Catanach, T. A.; Valqui, T.; Brumfield, R. T.; Aleixo, A.; Johnson, K. P.; Weckstein, J. D.</p>
        <p class="info">Score: 14.7, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576737' target='https://doi.org/10.1101/2024.01.22.576737'> 10.1101/2024.01.22.576737</a></p>
        <p class="abstract">Incomplete lineage sorting (ILS) and introgression increase genealogical discordance across the genome, which complicates phylogenetic inference. In such cases, identifying orthologs that result in gene trees with low estimation error is crucial because phylogenomic methods rely on accurate gene histories. We sequenced whole genomes for the tinamous (Aves: Tinamidae) to dissect the sources of gene and species-tree discordance and reconstruct their interrelationships. We compared results based on four ortholog sets: (1) coding genes (BUSCOs), (2) ultraconserved elements (UCEs) with short flanking regions, (3) UCEs with intermediate flanks, and (4) UCEs with long flanks. We hypothesized that orthologs with more phylogenetically informative sites would result in more accurate species trees because the resulting gene trees contain lower error. Consistent with our hypothesis, we found that long UCEs had the most informative sites and lowest rates of error. However, despite having many informative sites, BUSCO gene trees contained high error compared to long UCEs. Unlike UCEs, BUSCO gene sequences showed a positive association between the proportion of parsimony informative sites and gene tree error. Thus, BUSCO and UCE datasets have different underlying properties of molecular evolution, and these differences should be considered when selecting loci for phylogenomic analysis. Still, species trees from different datasets were mostly congruent. Only one clade, with a history of ILS and introgression, exhibited substantial species-tree discordance across the different data sets. Overall, we present the most complete phylogeny for tinamous to date, identify a new species, and provide a case study for species-level phylogenomic analysis using whole-genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576850">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576850" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576850">
        <p class="paperTitle">A high coverage Mesolithic aurochs genome and effective leveraging of ancient cattle genomes using whole genome imputation.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576850" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576850" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Erven, J. A. M.; Scheu, A.; Verdugo, M. P.; Cassidy, L.; Chen, N.; Gehlen, B.; Street, M.; Madsen, O.; Mullin, V. E.</p>
        <p class="info">Score: 14.1, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576850' target='https://doi.org/10.1101/2024.01.23.576850'> 10.1101/2024.01.23.576850</a></p>
        <p class="abstract">Ancient genomic analyses are often restricted to utilising pseudo-haploid data due to low genome coverage. Leveraging low coverage data by imputation to calculate phased diploid genotypes that enable haplotype-based interrogation and SNP calling at unsequenced positions is highly desirable. This has not been investigated for ancient cattle genomes despite these being compelling subjects for archaeological, evolutionary and economic reasons. Here we test this approach by sequencing a Mesolithic European aurochs (18.49x; 9852-9376 calBC), an Early Medieval European cow (18.69x; 427-580 calAD), and combine these with published individuals; two ancient and three modern. We downsample these genomes (0.25x, 0.5x, 1.0x, 2.0x) and impute diploid genotypes, utilising a reference panel of 171 published modern cattle genomes that we curated for 21.7 million (Mn) phased single-nucleotide polymorphisms (SNPs). We recover high densities of correct calls with an accuracy of &gt;99.1% at variant sites for the lowest downsample depth of 0.25x, increasing to &gt;99.5% for 2.0x (transversions only, minor allele frequency (MAF) [&amp;ge;]2.5%). The recovery of SNPs correlates with coverage, on average 58% of sites are recovered for 0.25x increasing to 87% for 2.0x, utilising an average of 3.5 million (Mn) transversions (MAF [&amp;ge;]2.5%), even in the aurochs which is temporally and morphologically distinct from the reference panel. Our imputed genomes behave similarly to directly called data in allele-frequency-based analyses; for example consistently identifying runs of homozygosity &gt;2mb, including a long homozygous region in the Mesolithic European aurochs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576421">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576421" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576421">
        <p class="paperTitle">Epigenetic reprogramming driving successful and failed repair in acute kidney injury</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576421" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576421" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muto, Y.; Dixon, E. E.; Yoshimura, Y.; Ledru, N.; Kirita, Y.; Wu, H.; Humphreys, B. D.</p>
        <p class="info">Score: 13.0, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576421' target='https://doi.org/10.1101/2024.01.20.576421'> 10.1101/2024.01.20.576421</a></p>
        <p class="abstract">Acute kidney injury (AKI) causes epithelial damage followed by subsequent repair. While successful repair restores kidney function, this process is often incomplete and can lead to chronic kidney disease (CKD) in a process called failed repair. To better understand the epigenetic reprogramming driving this AKI-to-CKD transition we generated a single nucleus multiomic atlas for the full mouse AKI time course, consisting of [~]280,000 single nucleus transcriptomes and epigenomes. We reveal cell-specific dynamic alterations in gene regulatory landscapes reflecting especially activation of proinflammatory pathways. We further generated single nucleus multiomic data from four human AKI samples including validation by genome-wide identification of NF-kB binding sites. A regularized regression analysis identifies key regulators involved in both successful and failed repair cell fate, identifying the transcription factor CREB5 as a regulator of both successful and failed tubular repair that also drives proximal tubule cell proliferation after injury. Our interspecies multiomic approach provides a foundation to comprehensively understand cell states in AKI.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575663" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575663">
        <p class="paperTitle">DORA: an interactive map for the visualization and analysis of ancient human DNA and associated data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harris, K. D.; Greenbaum, G.</p>
        <p class="info">Score: 20.2, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575663' target='https://doi.org/10.1101/2024.01.15.575663'> 10.1101/2024.01.15.575663</a></p>
        <p class="abstract">The ability to sequence ancient genomes has revolutionized the way in which we study evolutionary history by providing access to the most important aspect of evolution -- time. Until recently, studying human demography, ecology, biology, and history using population genomic inference relied on contemporary genomic datasets. Over the past decade, the availability of human ancient DNA (aDNA) has increased rapidly, almost doubling every year, opening the way for spatiotemporal studies of ancient human populations. However, the multidimensionality of aDNA, with genotypes having temporal, spatial and genomic coordinates, and the need to integrate multiple sources of data, poses a challenge for developing meta-analyses pipelines. To address this challenge, we developed a publicly-available interactive tool, DORA, which integrates multiple data types, genomic and non-genomic, in a unified interface. This web-based tool allows users to browse sample metadata along with additional layers of information, such as population structure, climatic data, and unpublished samples. Users can then perform analyses on genotypes of these samples, or export sample subsets for external analyses. DORA integrates analyses and visualizations in a single intuitive interface, resolving the technical issues of combining datasets from different sources and formats, and allowing researchers to focus on analysis and the scientific questions that can be addressed through analysis of aDNA datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576413">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576413" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576413">
        <p class="paperTitle">Genomic diversity and antimicrobial resistance in clinical Klebsiella pneumoniae isolates from tertiary hospitals in Southern Ghana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576413" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576413" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mills, R. O.; Dadzie, I.; Le-Viet, T.; Baker, D. J.; Addy, H. P. K.; Akwetey, S. A.; Donkoh, I. E.; Quansah, E.; Semanshia, P. S.; Morgan, J.; Mensah, A.; Adade, N. E.; Ampah, E. O.; Owusu, E.; Mwintige, P.; Amoako, E. O.; Spadar, A.; Holt, K. E.; Foster-Nyarko, E.</p>
        <p class="info">Score: 12.7, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576413' target='https://doi.org/10.1101/2024.01.20.576413'> 10.1101/2024.01.20.576413</a></p>
        <p class="abstract">Comprehensive data on the genomic epidemiology of hospital-associated Klebsiella pneumoniae in Ghana is scarce. This study sequenced 103 clinical K. pneumoniae isolates from five tertiary hospitals in Southern Ghana, predominantly from paediatric patients under five years (67/103, 65%), with the majority collected from urine (32/103, 31%) and blood (25/103, 24%) cultures. We employed Pathogenwatch for genotyping via Kaptive (K/O antigens) and Kleborate (antimicrobial resistance and hypervirulence) and determined clonal relationships using core-genome multilocus sequence typing (cgMLST). Among the 44 distinct sequence types (STs) detected, ST133 was the most common, comprising 23% of isolates (n=23/103). We discovered 27 different capsular (K) locus antigens and seven lipopolysaccharide (O) types; KL116 (28/103, 27%) and O1 (66/103, 64%) were the most prevalent. Single-linkage clustering highlighted the global spread of multidrug-resistant clones such as ST15, ST307, ST17, ST11, ST101, and ST48, with minimal allele differences (1-5) from publicly available genomes worldwide. Conversely, several isolates (n=17) constituted novel clonal groups and lacked close relatives among publicly available genomes, displaying unique genetic diversity within our study population. A significant proportion of isolates (88/103, 85%) carried resistance genes for three or more antibiotic classes, with the blaCTXM-15 gene present in 78% (n=80/103). Carbapenem resistance, predominantly due to blaOXA-181 and blaNDM-1 genes, was found in 10% (n=10/103) of the isolates. Yersiniabactin was the predominant acquired virulence trait, identified in 70% (n=72/103) of the isolates. Our findings reveal a complex genomic landscape of K. pneumoniae in Southern Ghana, underscoring the critical need for ongoing genomic surveillance to manage the substantial burden of antimicrobial resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576496" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576496">
        <p class="paperTitle">A complex Plasmodium falciparum cryptotype circulating at low frequency across the African continent</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miotto, O.; Amambua-Ngwa, A.; Amenga-Etego, L. N.; Abdel Hamid, M. M.; Adam, I.; Aninagyei, E.; Apinjoh, T.; Awandare, G. A.; Bejon, P.; Bertin, G.; Bouyou-Akotet, M. K.; Claessens, A.; Conway, D.; D&#39;Alessandro, U.; Diakite, M.; Djimde, A.; Dondorp, A. M.; Duffy, P.; Fairhurst, R. M.; Fanello, C. I.; Ghansah, A.; Ishengoma, D.; Lawniczak, M.; Maiga-Ascofare, O.; Auburn, S.; Rosanas-Urgell, A.; Wasakul, V.; White, N.; Almagro-Garcia, J.; Pearson, R.; Goncalves, S.; Ariani, C.; Bozdech, Z.; Hamilton, W. L.; Simpson, V.; Kwiatkowski, D. P.</p>
        <p class="info">Score: 11.6, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576496' target='https://doi.org/10.1101/2024.01.20.576496'> 10.1101/2024.01.20.576496</a></p>
        <p class="abstract">The population structure of the malaria parasite Plasmodium falciparum can reveal underlying demographic and adaptive evolutionary processes. Here, we analyse population structure in 4,376 P. falciparum genomes from 21 countries across Africa. We identified a strongly differentiated cluster of parasites, comprising [~]1.2% of samples analysed, geographically distributed over 13 countries across the continent. Members of this cluster, named AF1, carry a genetic background consisting of a large number of highly differentiated variants, rarely observed outside this cluster, at a multitude of genomic loci distributed across most chromosomes. At these loci, the AF1 haplotypes appear to have common ancestry, irrespective of the sampling location; outside the shared loci, however, AF1 members are genetically similar to their sympatric parasites. AF1 parasites sharing up to 23 genomic co-inherited regions were found in all major regions of Africa, at locations over 7,000 km apart. We coined the term cryptotype to describe a complex common background which is geographically widespread, but concealed by genomic regions of local origin. Most AF1 differentiated variants are functionally related, comprising structural variations and single nucleotide polymorphisms in components of the MSP1 complex and several other genes involved in interactions with red blood cells, including invasion and erythrocyte antigen export. We propose that AF1 parasites have adapted to some as yet unidentified evolutionary niche, by acquiring a complex compendium of interacting variants that rarely circulate separately in Africa. As the cryptotype spread across the continent, it appears to have been maintained mostly intact in spite of recombination events, suggesting a selective advantage. It is possible that other cryptotypes circulate in Africa, and new analysis methods may be needed to identify them.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
