<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="CRISPRmap: Sequencing-free optical pooled screens mapping multi-omic phenotypes in cells and tissue
Authors: Gu, J.; Iyer, A.; Wesley, B.; Taglialatela, A.; Leuzzi, G.; Hangai, S.; Decker, A.; Gu, R.; Klickstein, N.; Shuai, Y.; Jankovic, K.; Parker-Burns, L.; Jin, Y.; Zhang, J. Y.; Hong, J.; Niu, X.; Chou, J.; Landau, D. A.; Azizi, E.; Chan, E. M.; Ciccia, A.; Gaublomme, J. T.
Score: 49.3, Published: 2023-12-26 DOI: 10.1101/2023.12.26.572587
Pooled genetic screens are powerful tools to study gene function in a high-throughput manner.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="CRISPRmap: Sequencing-free optical pooled screens mapping multi-omic phenotypes in cells and tissue
Authors: Gu, J.; Iyer, A.; Wesley, B.; Taglialatela, A.; Leuzzi, G.; Hangai, S.; Decker, A.; Gu, R.; Klickstein, N.; Shuai, Y.; Jankovic, K.; Parker-Burns, L.; Jin, Y.; Zhang, J. Y.; Hong, J.; Niu, X.; Chou, J.; Landau, D. A.; Azizi, E.; Chan, E. M.; Ciccia, A.; Gaublomme, J. T.
Score: 49.3, Published: 2023-12-26 DOI: 10.1101/2023.12.26.572587
Pooled genetic screens are powerful tools to study gene function in a high-throughput manner." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-03T10:39:43+00:00" />
<meta property="article:modified_time" content="2024-01-03T10:39:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="CRISPRmap: Sequencing-free optical pooled screens mapping multi-omic phenotypes in cells and tissue
Authors: Gu, J.; Iyer, A.; Wesley, B.; Taglialatela, A.; Leuzzi, G.; Hangai, S.; Decker, A.; Gu, R.; Klickstein, N.; Shuai, Y.; Jankovic, K.; Parker-Burns, L.; Jin, Y.; Zhang, J. Y.; Hong, J.; Niu, X.; Chou, J.; Landau, D. A.; Azizi, E.; Chan, E. M.; Ciccia, A.; Gaublomme, J. T.
Score: 49.3, Published: 2023-12-26 DOI: 10.1101/2023.12.26.572587
Pooled genetic screens are powerful tools to study gene function in a high-throughput manner."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "CRISPRmap: Sequencing-free optical pooled screens mapping multi-omic phenotypes in cells and tissue\nAuthors: Gu, J.; Iyer, A.; Wesley, B.; Taglialatela, A.; Leuzzi, G.; Hangai, S.; Decker, A.; Gu, R.; Klickstein, N.; Shuai, Y.; Jankovic, K.; Parker-Burns, L.; Jin, Y.; Zhang, J. Y.; Hong, J.; Niu, X.; Chou, J.; Landau, D. A.; Azizi, E.; Chan, E. M.; Ciccia, A.; Gaublomme, J. T.\nScore: 49.3, Published: 2023-12-26 DOI: 10.1101/2023.12.26.572587\nPooled genetic screens are powerful tools to study gene function in a high-throughput manner.",
  "keywords": [
    
  ],
  "articleBody": " CRISPRmap: Sequencing-free optical pooled screens mapping multi-omic phenotypes in cells and tissue\nAuthors: Gu, J.; Iyer, A.; Wesley, B.; Taglialatela, A.; Leuzzi, G.; Hangai, S.; Decker, A.; Gu, R.; Klickstein, N.; Shuai, Y.; Jankovic, K.; Parker-Burns, L.; Jin, Y.; Zhang, J. Y.; Hong, J.; Niu, X.; Chou, J.; Landau, D. A.; Azizi, E.; Chan, E. M.; Ciccia, A.; Gaublomme, J. T.\nScore: 49.3, Published: 2023-12-26 DOI: 10.1101/2023.12.26.572587\nPooled genetic screens are powerful tools to study gene function in a high-throughput manner. Typically, sequencing-based screens require cell lysis, which limits the examination of critical phenotypes such as cell morphology, protein subcellular localization, and cell-cell/tissue interactions. In contrast, emerging optical pooled screening methods enable the investigation of these spatial phenotypes in response to targeted CRISPR perturbations. In this study, we report a multi-omic optical pooled CRISPR screening method, which we have named CRISPRmap. Our method combines a novel in situ CRISPR guide identifying barcode readout approach with concurrent multiplexed immunofluorescence and in situ RNA detection. CRISPRmap barcodes are detected and read out through combinatorial hybridization of DNA oligos, enhancing barcode detection efficiency, while reducing both dependency on third party proprietary sequencing reagents and assay cost. Notably, we conducted a multi-omic base-editing screen in a breast cancer cell line on core DNA damage repair genes involved in the homologous recombination and Fanconi anemia pathways investigating how nucleotide variants in those genes influence DNA damage signaling and cell cycle regulation following treatment with ionizing radiation or DNA damaging agents commonly used for cancer therapy. Approximately a million cells were profiled with our multi-omic approach, providing a comprehensive phenotypic assessment of the functional consequences of the studied variants. CRISPRmap enabled us to pinpoint likely-pathogenic patient-derived mutations that were previously classified as variants of unknown clinical significance. Furthermore, our approach effectively distinguished barcodes of a pooled library in tumor tissue, and we coupled it with cell-type and molecular phenotyping by cyclic immunofluorescence. Multi-omic spatial analysis of how CRISPR-perturbed cells respond to various environmental cues in the tissue context offers the potential to significantly expand our understanding of tissue biology in both health and disease.\nScalable ultra-high-throughput single-cell chromatin and RNA sequencing reveals gene regulatory dynamics linking macrophage polarization to autoimmune disease\nAuthors: Lobato-Moreno, S.; Yildiz, U.; Claringbould, A.; Servaas, N. H.; Vlachou, E. P.; Arnold, C.; Bauersachs, H. G.; Campos-Fornes, V.; Prummel, K. D.; Noh, K. M.; Marttinen, M.; Zaugg, J. B.\nScore: 41.2, Published: 2023-12-26 DOI: 10.1101/2023.12.26.573253\nEnhancers and transcription factors (TFs) are crucial in regulating cellular processes, including disease-associated cell states. Current multiomic technologies to study these elements in gene regulatory mechanisms lack multiplexing capability and scalability. Here, we present SUM-seq, a cost-effective, scalable Single-cell Ultra-high-throughput Multiomic sequencing method for co-assaying chromatin accessibility and gene expression in single nuclei. SUM-seq enables profiling hundreds of samples at the million cell scale and outperforms current high-throughput single-cell methods. We applied SUM-seq to dissect the gene regulatory mechanisms governing macrophage polarization and explored their link to traits from genome-wide association studies (GWAS). Our analyses confirmed known TFs orchestrating M1 and M2 macrophage programs, unveiled key regulators, and demonstrated extensive enhancer rewiring. Integration with GWAS data further pinpointed the impact of specific TFs on a set of immune traits. Notably, inferred enhancers regulated by the STAT1/STAT2/IRF9 (ISGF3) complex were enriched for rheumatoid arthritis-associated genetic variants, and their target genes included known drug targets. This highlights the potential of SUM-seq for dissecting molecular disease mechanisms. SUM-seq offers a cost-effective, scalable solution for ultra-high-throughput single-cell multiomic sequencing, excelling in unraveling complex gene regulatory networks in cell differentiation, responses to perturbations, and disease studies.\nA single-cell transposable element atlas of human cell identity\nAuthors: Reyes-Gopar, H.; Marston, J. L.; Singh, B.; Greenig, M.; Lin, J.; Ostrowski, M. A.; Randall, K. N.; Sandoval-Motta, S.; Dopkins, N.; Lawrence, E.; O'Mara, M. M.; Fei, T.; Duarte, R. R. R.; Powell, T. R.; Hernandez-Lemus, E.; Iniguez, L. P.; Nixon, D. F.; Bendall, M. L.\nScore: 20.6, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573568\nSingle cell RNA sequencing (scRNA-seq) is revolutionizing the study of complex biological systems. However, most sequencing studies overlook the contribution of transposable element (TE) expression to the transcriptome. In both scRNA-seq and bulk tissue RNA sequencing (RNA-seq), quantification of TE expression is challenging due to repetitive sequence content and poorly characterized TE gene models. Here, we developed a tool and analysis pipeline for Single cell Transposable Element Locus Level Analysis of scRNA Sequencing (Stellarscope) that reassigns multi-mapped reads to specific genomic loci using an expectation-maximization algorithm. Using Stellarscope, we built an atlas of TE expression in human PBMCs. We found that locus-specific TEs delineate cell types and define new cell subsets not identified by standard mRNA expression profiles. Altogether, this study provides comprehensive insights into the influence of transposable elements in human biology.\nComparative genomics of Cryptococcus and Kwoniella reveals pathogenesis evolution and contrasting modes of karyotype evolution via chromosome fusion or intercentromeric recombination\nAuthors: Coelho, M. A.; David-Palma, M.; Shea, T.; Bowers, K.; McGinley-Smith, S.; Mohammad, A. W.; Gnirke, A.; Yurkov, A. M.; Nowrousian, M.; Sun, S.; Cuomo, C. A.; Heitman, J.\nScore: 19.9, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573464\nA large-scale comparative genomic analysis was conducted for the global human fungal pathogens within the Cryptococcus genus, compared to non-pathogenic Cryptococcus species, and related species from the sister genus Kwoniella. Chromosome-level genome assemblies were generated for multiple species of both genera, resulting in a dataset encompassing virtually all of their known diversity. Although Cryptococcus and Kwoniella have comparable genome sizes (about 19.2 and 22.9 Mb) and similar gene content, hinting at pre-adaptive pathogenic potential, our analysis found evidence in pathogenic Cryptococcus species of specific examples of gene gain (via horizontal gene transfer) and gene loss, which might represent evolutionary signatures of pathogenic development. Genome analysis also revealed a significant variation in chromosome number and structure between the two genera. By combining synteny analysis and experimental centromere validation, we found that most Cryptococcus species have 14 chromosomes, whereas most Kwoniella species have fewer (11, 8, 5 or even as few as 3). Reduced chromosome number in Kwoniella is associated with formation of giant chromosomes (up to 18 Mb) through repeated chromosome fusion events, each marked by a pericentric inversion and centromere loss. While similar chromosome inversion-fusion patterns were observed in all Kwoniella species with fewer than 14 chromosomes, no such pattern was detected in Cryptococcus. Instead, Cryptococcus species with less than 14 chromosomes, underwent chromosome reductions primarily through rearrangements associated with the loss of repeat-rich centromeres. Additionally, Cryptococcus genomes exhibited frequent interchromosomal translocations, including intercentromeric recombination facilitated by transposons shared between centromeres. Taken together, our findings advance our understanding of genomic changes possibly associated with pathogenicity in Cryptococcus and provide a foundation to elucidate mechanisms of centromere loss and chromosome fusion driving distinct karyotypes in closely related fungal species, including prominent global human pathogens.\nCharacterizing uncertainty in predictions of genomic sequence-to-activity models\nAuthors: Bajwa, A.; Rastogi, R.; Kathail, P.; Shuai, R. W.; Ioannidis, N. M.\nScore: 13.7, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572730\nGenomic sequence-to-activity models are increasingly utilized to understand gene regulatory syntax and probe the functional consequences of regulatory variation. Current models make accurate predictions of relative activity levels across the human reference genome, but their performance is more limited for predicting the effects of genetic variants, such as explaining gene expression variation across individuals. To better understand the causes of these shortcomings, we examine the uncertainty in predictions of genomic sequence-to-activity models using an ensemble of Basenji2 model replicates. We characterize prediction consistency on four types of sequences: reference genome sequences, reference genome sequences perturbed with TF motifs, eQTLs, and personal genome sequences. We observe that models tend to make high-confidence predictions on reference sequences, even when incorrect, and low-confidence predictions on sequences with variants. For eQTLs and personal genome sequences, we find that model replicates make inconsistent predictions in \u003e50% of cases. Our findings suggest strategies to improve performance of these models.\nAnnelid methylomes reveal ancestral developmental and ageing-associated epigenetic erosion across Bilateria\nAuthors: Guynes, K.; Sarre, L. A.; Carrillo-Baltodano, A. M.; Davies, B. E.; Xu, L.; Liang, Y.; Martin-Zamora, F. M.; Hurd, P. J.; de Mendoza, A.; Martin-Duran, J. M.\nScore: 18.4, Published: 2023-12-25 DOI: 10.1101/2023.12.21.572802\nBackgroundDNA methylation in the form of 5-methylcytosine (5mC) is the most abundant base modification in animals. However, 5mC levels vary widely across taxa. While vertebrate genomes are hypermethylated, in most invertebrates, 5mC concentrates on constantly and highly transcribed genes (gene body methylation; GbM) and, in some species, on transposable elements (TEs), a pattern known as mosaic. Yet, the role and developmental dynamics of 5mC and how these explain interspecific differences in DNA methylation patterns remain poorly understood, especially in Spiralia, a large clade of invertebrates comprising nearly half of the animal phyla. ResultsHere, we generate base-resolution methylomes for three species with distinct genomic features and phylogenetic positions in Annelida, a major spiralian phylum. All possible 5mC patterns occur in annelids, from typical invertebrate intermediate levels in a mosaic distribution to hypermethylation and methylation loss. GbM is common to annelids with 5mC, and methylation differences across species are explained by taxon-specific transcriptional dynamics or the presence of intronic TEs. Notably, the link between GbM and transcription decays during development, and there is a gradual and global, age-dependent demethylation in adult stages. Moreover, reducing 5mC levels with cytidine analogues during early development impairs normal embryogenesis and reactivates TEs in the annelid Owenia fusiformis. ConclusionsOur study indicates that global epigenetic erosion during development and ageing is an ancestral feature of bilateral animals. However, the tight link between transcription and gene body methylation is likely important in early embryonic stages, and 5mC-mediated TE silencing probably emerged convergently across animal lineages.\nComparison of software packages for detecting unannotated translated small open reading frames by Ribo-seq\nAuthors: Martinez, T. F.; Tong, G.; Hah, N.\nScore: 8.6, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573709\nAccurate and comprehensive annotation of microprotein-coding small open reading frames (smORFs) is critical to our understanding of normal physiology and disease. Empirical identification of translated smORFs is carried out primarily using ribosome profiling (Ribo-seq). While effective, published Ribo-seq datasets can vary drastically in quality and different analysis tools are frequently employed. Here, we examine the impact of these factors on identifying translated smORFs. We compared five commonly used software tools that assess ORF translation from Ribo-seq (RibORFv0.1, RibORFv1.0, RiboCode, ORFquant, and Ribo-TISH), and found surprisingly low agreement across all tools. Only [~]2% of smORFs were called translated by all five tools and [~]15% by three or more tools when assessing the same high- resolution Ribo-seq dataset. For larger annotated genes, the same analysis showed [~]72% agreement across all five tools. We also found that some tools are strongly biased against low- resolution Ribo-seq data, while others are more tolerant. Analyzing Ribo-seq coverage as a proxy for translation levels revealed that highly translated smORFs are more likely to be detected by more than one tool. Together these results support employing multiple tools to identify the most confident microprotein-coding smORFs, and choosing the tools based on the quality of the dataset and planned downstream characterization experiments of predicted smORFs.\nMnM: a machine learning approach to detect replication states and genomic subpopulations for single-cell DNA replication timing disentanglement\nAuthors: Josephides, J. M.; Chen, C.-L.\nScore: 4.8, Published: 2023-12-28 DOI: 10.1101/2023.12.26.573369\nWe introduce MnM, an efficient tool for characterising single-cell DNA replication states and revealing genomic subpopulations in heterogeneous samples, notably cancers. MnM uses sin-gle-cell copy-number data to accurately perform missing-value imputation, classify cell repli-cation states and detect genomic heterogeneity, which allows to separate somatic copy-num-ber alterations from copy-number variations due to DNA replication. By applying our machine learning methods, our research unveils critical insights into chromosomal aberrations and showcases ubiquitous aneuploidy in tumorigenesis. MnM democratises single-cell subpopu-lation detection which, in hand, enables the extraction of single-cell DNA replication timing (scRT) profiles from genomically-heterogenous subpopulations detected by DNA content and issued from single samples. By analysing over 119,000 human single cells from cultured cell lines, patient tumours as well as patient-derived xenograft samples, the copy-number and rep-lication timing profiles issued in this study lead to the first multi-sample subpopulation-disen-tangled scRT atlas and act as data contribution for further cancer research. Our results high-light the necessity of studying in vivo samples to comprehensively grasp the complexities of DNA replication, given that cell lines, while convenient, lack dynamic environmental factors. This tool offers to advance our understanding of cancer initiation and progression, facilitating further research in the interface of genomic instability and replication stress. GRAPHICAL ABSTRACT O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=150 SRC=\"FIGDIR/small/573369v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (30K): org.highwire.dtl.DTLVardef@1a6412org.highwire.dtl.DTLVardef@9fe40dorg.highwire.dtl.DTLVardef@1353ad5org.highwire.dtl.DTLVardef@1a98f30_HPS_FORMAT_FIGEXP M_FIG C_FIG\nA comparative analysis of planarian genomes reveals regulatory conservation in the face of rapid structural divergence\nAuthors: Ivankovic, M.; Brand, J. N.; Pandolfini, L.; Brown, T.; Pippel, M.; Rozanski, A.; Schubert, T.; Grohme, M. A.; Winkler, S.; Robledillo, L.; Zhang, M.; Codino, A.; Gustincich, S.; Vila-Farre, M.; Zhang, S.; Papantonis, A.; Marques, A.; Rink, J. C.\nScore: 22.6, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572568\nThe planarian Schmidtea mediterranea can regenerate its entire body from small tissue fragments and is studied as regeneration model species. The assembly and functional analysis of planarian genomes has proven challenging due its high A/T content (70% A/T), repetitive nature, and limited transferability of routine laboratory protocols due to their divergent biochemistry. Only few and often fragmented genome assemblies are currently available, and open challenges include the provision of well-annotated chromosome-scale reference assemblies of the model species and other planarians for a comparative genome evolution perspective. Here we report a haplotype-phased, chromosome-scale genome assembly and high-quality gene annotations of the sexual S2 strain of S. mediterranea and provide putative regulatory region annotations via optimized ATAC-seq and ChIP-seq protocols. To additionally leverage sequence conservation for regulatory element annotations, we generated chromosome-scale genome assemblies and chromatin accessibility data for the three closest relatives of S. mediterranea: S. polychroa, S. nova, and S. lugubris. We find substantial divergence in protein-coding sequences and regulatory regions, yet reveal remarkable conservation in ChIP-mark bearing open chromatin regions identified as promoters and enhancers in S. mediterranea. The resulting high-confidence set of evolutionary conserved enhancers and promoters provides a valuable resource for the analysis of gene regulatory circuits and their evolution within the taxon. In addition, our four chromosome-scale genome assemblies provide a first comparative perspective on planarian genome evolution. Our analyses reveal frequent retrotransposon-associated chromosomal inversions and inter-chromosomal translocations that lead to a degradation of synteny across the genus. Interestingly, we further find independent and near-complete losses of the ancestral metazoan synteny across Schmidtea and two other flatworm groups, indicating that platyhelminth genomes largely evolve without syntenic constraints. Our work provides valuable genome resources for the planarian research community and sets a foundation for the comparative genomics of planarians. We reveal a contrast between the fast structural evolution of planarian genomes and the conservation of their regulatory elements, suggesting a unique genome evolution in flatworms where gene positioning may not be essential.\nSingle-nucleus RNA-sequencing reveals novel potential mechanisms of ovarian insufficiency in 45,X Turner Syndrome\nAuthors: McGlacken-Byrne, S. M.; Del Valle, I.; Xenakis, T.; Suntharalingham, J. P.; Nel, L.; Liptrot, D.; Crespo, B.; Ogunbiyi, O. K.; Niola, P.; Brooks, T.; Solanky, N.; Conway, G. S.; Achermann, J. C.\nScore: 4.9, Published: 2023-12-23 DOI: 10.1101/2023.12.22.572342\nStudy questionCan single-nuclei and bulk RNA sequencing technologies be used to elucidate novel mechanisms of ovarian insufficiency in Turner Syndrome (TS)? Summary answerUsing single-nucleus and bulk RNA sequencing approaches, we identified novel potential pathogenic mechanisms underlying ovarian insufficiency in TS including and beyond X chromosome haploinsufficiency. What is known alreadyTurner syndrome (TS) is the most common genetic cause of Primary Ovarian Insufficiency (POI) in humans. Morphological analyses of human fetal 45,X ovaries have demonstrated fewer germ cells and marked apoptosis established by 15-20 weeks post conception (wpc); however, we do not understand why POI develops mechanistically in the first instance. Study design, size, durationSingle-nucleus RNA sequencing (snRNA-seq): two 46,XX and two 45,X (TS) human fetal ovaries at 12-13 wpc. Bulk RNA sequencing: 19 human fetal ovary, 20 fetal testis, and 8 fetal control tissue (n=47 total samples; Carnegie Stage 22-16wpc). Participants/materials, setting, methodsTo identify novel potential mechanisms of ovarian insufficiency in TS and to characterise X chromosome gene expression in the 45,X ovary, we performed snRNA-seq of peri-meiotic 46,XX (n=2) and 45,X (n=2) fetal ovaries at 12-13 weeks post conception (wpc); and 2) a bulk RNA sequencing time-series analysis of fetal ovary, testis, and control samples across four developmental timepoints. Main results and the role of chanceGerm and somatic cell subpopulations were mostly shared across 46,XX and 45,X ovaries, aside from a 46XX-specific/45,X-depleted cluster of oogonia (\"synaptic oogonia\") containing genes with functions relating to sex chromosome synapsis; histone modification; intracellular protein regulation and chaperone systems. snRNA-seq enabled accurate cell counting localised to individual cell clusters; the 45,X ovary has fewer germ cells than the 46,XX ovary in every germ cell subpopulation, confirmed by histopathological analysis. The normal sequence of X-chromosome inactivation and reactivation is disrupted in 45,X ovaries; XIST was not expressed in 45,X somatic cells but was present in germ cell clusters, albeit with lower expression than in corresponding 46,XX clusters. The 45,X ovary has a globally abnormal transcriptome, with low expression of genes with proteostasis functions (RSP4X); cell cycle progression (BUB1B); and OXPHOS mitochondrial energy production (COX6C, ATP11C). Genes with higher expression in 45,X cell populations were enriched for apoptotic functions (e.g., NR4A1). Limitations, reasons for cautionLimitations include the relatively small sample size of the snRNA-seq analysis and the focus on a fixed meiotic timepoint which may overlook a dynamic process over time. Wider implications of the findingsWe characterise the human fetal peri-meiotic 45,X ovary at single-cell resolution and offer insights into novel pathogenic mechanisms underlying ovarian insufficiency in TS. Although asynapsis due to X chromosome haploinsufficiency likely plays a significant role, these data suggest meiotic failure and ovarian insufficiency may be a combinatorial process characterised by periods of vulnerability throughout early 45,X germ cell development Study funding/competing interest(s)This research was funded in whole, or in part, by the Wellcome Trust Grants 216362/Z/19/Z to SMcG-B and 209328/Z/17/Z to JCA. Human fetal material was provided by the Joint MRC/Wellcome Trust (Grant MR/R006237/1) Human Developmental Biology Resource (http://www.hdbr.org). Research at UCL Great Ormond Street Institute of Child Health is supported by the National Institute for Health Research, Great Ormond Street Hospital Biomedical Research Centre (grant IS-BRC-1215-20012).\n",
  "wordCount" : "3035",
  "inLanguage": "en",
  "datePublished": "2024-01-03T10:39:43Z",
  "dateModified": "2024-01-03T10:39:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 3, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.572587">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.572587" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.572587">
        <p class="paperTitle">CRISPRmap: Sequencing-free optical pooled screens mapping multi-omic phenotypes in cells and tissue</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.572587" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.572587" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gu, J.; Iyer, A.; Wesley, B.; Taglialatela, A.; Leuzzi, G.; Hangai, S.; Decker, A.; Gu, R.; Klickstein, N.; Shuai, Y.; Jankovic, K.; Parker-Burns, L.; Jin, Y.; Zhang, J. Y.; Hong, J.; Niu, X.; Chou, J.; Landau, D. A.; Azizi, E.; Chan, E. M.; Ciccia, A.; Gaublomme, J. T.</p>
        <p class="info">Score: 49.3, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.572587' target='https://doi.org/10.1101/2023.12.26.572587'> 10.1101/2023.12.26.572587</a></p>
        <p class="abstract">Pooled genetic screens are powerful tools to study gene function in a high-throughput manner. Typically, sequencing-based screens require cell lysis, which limits the examination of critical phenotypes such as cell morphology, protein subcellular localization, and cell-cell/tissue interactions. In contrast, emerging optical pooled screening methods enable the investigation of these spatial phenotypes in response to targeted CRISPR perturbations. In this study, we report a multi-omic optical pooled CRISPR screening method, which we have named CRISPRmap. Our method combines a novel in situ CRISPR guide identifying barcode readout approach with concurrent multiplexed immunofluorescence and in situ RNA detection. CRISPRmap barcodes are detected and read out through combinatorial hybridization of DNA oligos, enhancing barcode detection efficiency, while reducing both dependency on third party proprietary sequencing reagents and assay cost. Notably, we conducted a multi-omic base-editing screen in a breast cancer cell line on core DNA damage repair genes involved in the homologous recombination and Fanconi anemia pathways investigating how nucleotide variants in those genes influence DNA damage signaling and cell cycle regulation following treatment with ionizing radiation or DNA damaging agents commonly used for cancer therapy. Approximately a million cells were profiled with our multi-omic approach, providing a comprehensive phenotypic assessment of the functional consequences of the studied variants. CRISPRmap enabled us to pinpoint likely-pathogenic patient-derived mutations that were previously classified as variants of unknown clinical significance. Furthermore, our approach effectively distinguished barcodes of a pooled library in tumor tissue, and we coupled it with cell-type and molecular phenotyping by cyclic immunofluorescence. Multi-omic spatial analysis of how CRISPR-perturbed cells respond to various environmental cues in the tissue context offers the potential to significantly expand our understanding of tissue biology in both health and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573253">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573253" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573253">
        <p class="paperTitle">Scalable ultra-high-throughput single-cell chromatin and RNA sequencing reveals gene regulatory dynamics linking macrophage polarization to autoimmune disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573253" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573253" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lobato-Moreno, S.; Yildiz, U.; Claringbould, A.; Servaas, N. H.; Vlachou, E. P.; Arnold, C.; Bauersachs, H. G.; Campos-Fornes, V.; Prummel, K. D.; Noh, K. M.; Marttinen, M.; Zaugg, J. B.</p>
        <p class="info">Score: 41.2, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573253' target='https://doi.org/10.1101/2023.12.26.573253'> 10.1101/2023.12.26.573253</a></p>
        <p class="abstract">Enhancers and transcription factors (TFs) are crucial in regulating cellular processes, including disease-associated cell states. Current multiomic technologies to study these elements in gene regulatory mechanisms lack multiplexing capability and scalability. Here, we present SUM-seq, a cost-effective, scalable Single-cell Ultra-high-throughput Multiomic sequencing method for co-assaying chromatin accessibility and gene expression in single nuclei. SUM-seq enables profiling hundreds of samples at the million cell scale and outperforms current high-throughput single-cell methods. We applied SUM-seq to dissect the gene regulatory mechanisms governing macrophage polarization and explored their link to traits from genome-wide association studies (GWAS). Our analyses confirmed known TFs orchestrating M1 and M2 macrophage programs, unveiled key regulators, and demonstrated extensive enhancer rewiring. Integration with GWAS data further pinpointed the impact of specific TFs on a set of immune traits. Notably, inferred enhancers regulated by the STAT1/STAT2/IRF9 (ISGF3) complex were enriched for rheumatoid arthritis-associated genetic variants, and their target genes included known drug targets. This highlights the potential of SUM-seq for dissecting molecular disease mechanisms. SUM-seq offers a cost-effective, scalable solution for ultra-high-throughput single-cell multiomic sequencing, excelling in unraveling complex gene regulatory networks in cell differentiation, responses to perturbations, and disease studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573568">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573568" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573568">
        <p class="paperTitle">A single-cell transposable element atlas of human cell identity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573568" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573568" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reyes-Gopar, H.; Marston, J. L.; Singh, B.; Greenig, M.; Lin, J.; Ostrowski, M. A.; Randall, K. N.; Sandoval-Motta, S.; Dopkins, N.; Lawrence, E.; O&#39;Mara, M. M.; Fei, T.; Duarte, R. R. R.; Powell, T. R.; Hernandez-Lemus, E.; Iniguez, L. P.; Nixon, D. F.; Bendall, M. L.</p>
        <p class="info">Score: 20.6, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573568' target='https://doi.org/10.1101/2023.12.28.573568'> 10.1101/2023.12.28.573568</a></p>
        <p class="abstract">Single cell RNA sequencing (scRNA-seq) is revolutionizing the study of complex biological systems. However, most sequencing studies overlook the contribution of transposable element (TE) expression to the transcriptome. In both scRNA-seq and bulk tissue RNA sequencing (RNA-seq), quantification of TE expression is challenging due to repetitive sequence content and poorly characterized TE gene models. Here, we developed a tool and analysis pipeline for Single cell Transposable Element Locus Level Analysis of scRNA Sequencing (Stellarscope) that reassigns multi-mapped reads to specific genomic loci using an expectation-maximization algorithm. Using Stellarscope, we built an atlas of TE expression in human PBMCs. We found that locus-specific TEs delineate cell types and define new cell subsets not identified by standard mRNA expression profiles. Altogether, this study provides comprehensive insights into the influence of transposable elements in human biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573464">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573464" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573464">
        <p class="paperTitle">Comparative genomics of Cryptococcus and Kwoniella reveals pathogenesis evolution and contrasting modes of karyotype evolution via chromosome fusion or intercentromeric recombination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573464" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573464" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coelho, M. A.; David-Palma, M.; Shea, T.; Bowers, K.; McGinley-Smith, S.; Mohammad, A. W.; Gnirke, A.; Yurkov, A. M.; Nowrousian, M.; Sun, S.; Cuomo, C. A.; Heitman, J.</p>
        <p class="info">Score: 19.9, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573464' target='https://doi.org/10.1101/2023.12.27.573464'> 10.1101/2023.12.27.573464</a></p>
        <p class="abstract">A large-scale comparative genomic analysis was conducted for the global human fungal pathogens within the Cryptococcus genus, compared to non-pathogenic Cryptococcus species, and related species from the sister genus Kwoniella. Chromosome-level genome assemblies were generated for multiple species of both genera, resulting in a dataset encompassing virtually all of their known diversity. Although Cryptococcus and Kwoniella have comparable genome sizes (about 19.2 and 22.9 Mb) and similar gene content, hinting at pre-adaptive pathogenic potential, our analysis found evidence in pathogenic Cryptococcus species of specific examples of gene gain (via horizontal gene transfer) and gene loss, which might represent evolutionary signatures of pathogenic development. Genome analysis also revealed a significant variation in chromosome number and structure between the two genera. By combining synteny analysis and experimental centromere validation, we found that most Cryptococcus species have 14 chromosomes, whereas most Kwoniella species have fewer (11, 8, 5 or even as few as 3). Reduced chromosome number in Kwoniella is associated with formation of giant chromosomes (up to 18 Mb) through repeated chromosome fusion events, each marked by a pericentric inversion and centromere loss. While similar chromosome inversion-fusion patterns were observed in all Kwoniella species with fewer than 14 chromosomes, no such pattern was detected in Cryptococcus. Instead, Cryptococcus species with less than 14 chromosomes, underwent chromosome reductions primarily through rearrangements associated with the loss of repeat-rich centromeres. Additionally, Cryptococcus genomes exhibited frequent interchromosomal translocations, including intercentromeric recombination facilitated by transposons shared between centromeres. Taken together, our findings advance our understanding of genomic changes possibly associated with pathogenicity in Cryptococcus and provide a foundation to elucidate mechanisms of centromere loss and chromosome fusion driving distinct karyotypes in closely related fungal species, including prominent global human pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572730" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572730">
        <p class="paperTitle">Characterizing uncertainty in predictions of genomic sequence-to-activity models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bajwa, A.; Rastogi, R.; Kathail, P.; Shuai, R. W.; Ioannidis, N. M.</p>
        <p class="info">Score: 13.7, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572730' target='https://doi.org/10.1101/2023.12.21.572730'> 10.1101/2023.12.21.572730</a></p>
        <p class="abstract">Genomic sequence-to-activity models are increasingly utilized to understand gene regulatory syntax and probe the functional consequences of regulatory variation. Current models make accurate predictions of relative activity levels across the human reference genome, but their performance is more limited for predicting the effects of genetic variants, such as explaining gene expression variation across individuals. To better understand the causes of these shortcomings, we examine the uncertainty in predictions of genomic sequence-to-activity models using an ensemble of Basenji2 model replicates. We characterize prediction consistency on four types of sequences: reference genome sequences, reference genome sequences perturbed with TF motifs, eQTLs, and personal genome sequences. We observe that models tend to make high-confidence predictions on reference sequences, even when incorrect, and low-confidence predictions on sequences with variants. For eQTLs and personal genome sequences, we find that model replicates make inconsistent predictions in &gt;50% of cases. Our findings suggest strategies to improve performance of these models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572802">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572802" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572802">
        <p class="paperTitle">Annelid methylomes reveal ancestral developmental and ageing-associated epigenetic erosion across Bilateria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572802" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572802" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guynes, K.; Sarre, L. A.; Carrillo-Baltodano, A. M.; Davies, B. E.; Xu, L.; Liang, Y.; Martin-Zamora, F. M.; Hurd, P. J.; de Mendoza, A.; Martin-Duran, J. M.</p>
        <p class="info">Score: 18.4, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572802' target='https://doi.org/10.1101/2023.12.21.572802'> 10.1101/2023.12.21.572802</a></p>
        <p class="abstract">BackgroundDNA methylation in the form of 5-methylcytosine (5mC) is the most abundant base modification in animals. However, 5mC levels vary widely across taxa. While vertebrate genomes are hypermethylated, in most invertebrates, 5mC concentrates on constantly and highly transcribed genes (gene body methylation; GbM) and, in some species, on transposable elements (TEs), a pattern known as  mosaic. Yet, the role and developmental dynamics of 5mC and how these explain interspecific differences in DNA methylation patterns remain poorly understood, especially in Spiralia, a large clade of invertebrates comprising nearly half of the animal phyla.

ResultsHere, we generate base-resolution methylomes for three species with distinct genomic features and phylogenetic positions in Annelida, a major spiralian phylum. All possible 5mC patterns occur in annelids, from typical invertebrate intermediate levels in a mosaic distribution to hypermethylation and methylation loss. GbM is common to annelids with 5mC, and methylation differences across species are explained by taxon-specific transcriptional dynamics or the presence of intronic TEs. Notably, the link between GbM and transcription decays during development, and there is a gradual and global, age-dependent demethylation in adult stages. Moreover, reducing 5mC levels with cytidine analogues during early development impairs normal embryogenesis and reactivates TEs in the annelid Owenia fusiformis.

ConclusionsOur study indicates that global epigenetic erosion during development and ageing is an ancestral feature of bilateral animals. However, the tight link between transcription and gene body methylation is likely important in early embryonic stages, and 5mC-mediated TE silencing probably emerged convergently across animal lineages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573709">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573709" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573709">
        <p class="paperTitle">Comparison of software packages for detecting unannotated translated small open reading frames by Ribo-seq</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573709" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573709" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez, T. F.; Tong, G.; Hah, N.</p>
        <p class="info">Score: 8.6, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573709' target='https://doi.org/10.1101/2023.12.30.573709'> 10.1101/2023.12.30.573709</a></p>
        <p class="abstract">Accurate and comprehensive annotation of microprotein-coding small open reading frames (smORFs) is critical to our understanding of normal physiology and disease. Empirical identification of translated smORFs is carried out primarily using ribosome profiling (Ribo-seq). While effective, published Ribo-seq datasets can vary drastically in quality and different analysis tools are frequently employed. Here, we examine the impact of these factors on identifying translated smORFs. We compared five commonly used software tools that assess ORF translation from Ribo-seq (RibORFv0.1, RibORFv1.0, RiboCode, ORFquant, and Ribo-TISH), and found surprisingly low agreement across all tools. Only [~]2% of smORFs were called translated by all five tools and [~]15% by three or more tools when assessing the same high- resolution Ribo-seq dataset. For larger annotated genes, the same analysis showed [~]72% agreement across all five tools. We also found that some tools are strongly biased against low- resolution Ribo-seq data, while others are more tolerant. Analyzing Ribo-seq coverage as a proxy for translation levels revealed that highly translated smORFs are more likely to be detected by more than one tool. Together these results support employing multiple tools to identify the most confident microprotein-coding smORFs, and choosing the tools based on the quality of the dataset and planned downstream characterization experiments of predicted smORFs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573369">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573369" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573369">
        <p class="paperTitle">MnM: a machine learning approach to detect replication states and genomic subpopulations for single-cell DNA replication timing disentanglement</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573369" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573369" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Josephides, J. M.; Chen, C.-L.</p>
        <p class="info">Score: 4.8, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573369' target='https://doi.org/10.1101/2023.12.26.573369'> 10.1101/2023.12.26.573369</a></p>
        <p class="abstract">We introduce MnM, an efficient tool for characterising single-cell DNA replication states and revealing genomic subpopulations in heterogeneous samples, notably cancers. MnM uses sin-gle-cell copy-number data to accurately perform missing-value imputation, classify cell repli-cation states and detect genomic heterogeneity, which allows to separate somatic copy-num-ber alterations from copy-number variations due to DNA replication. By applying our machine learning methods, our research unveils critical insights into chromosomal aberrations and showcases ubiquitous aneuploidy in tumorigenesis. MnM democratises single-cell subpopu-lation detection which, in hand, enables the extraction of single-cell DNA replication timing (scRT) profiles from genomically-heterogenous subpopulations detected by DNA content and issued from single samples. By analysing over 119,000 human single cells from cultured cell lines, patient tumours as well as patient-derived xenograft samples, the copy-number and rep-lication timing profiles issued in this study lead to the first multi-sample subpopulation-disen-tangled scRT atlas and act as data contribution for further cancer research. Our results high-light the necessity of studying in vivo samples to comprehensively grasp the complexities of DNA replication, given that cell lines, while convenient, lack dynamic environmental factors. This tool offers to advance our understanding of cancer initiation and progression, facilitating further research in the interface of genomic instability and replication stress.

GRAPHICAL ABSTRACT

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=150 SRC=&#34;FIGDIR/small/573369v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (30K):
org.highwire.dtl.DTLVardef@1a6412org.highwire.dtl.DTLVardef@9fe40dorg.highwire.dtl.DTLVardef@1353ad5org.highwire.dtl.DTLVardef@1a98f30_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572568">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572568" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572568">
        <p class="paperTitle">A comparative analysis of planarian genomes reveals regulatory conservation in the face of rapid structural divergence</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572568" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572568" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ivankovic, M.; Brand, J. N.; Pandolfini, L.; Brown, T.; Pippel, M.; Rozanski, A.; Schubert, T.; Grohme, M. A.; Winkler, S.; Robledillo, L.; Zhang, M.; Codino, A.; Gustincich, S.; Vila-Farre, M.; Zhang, S.; Papantonis, A.; Marques, A.; Rink, J. C.</p>
        <p class="info">Score: 22.6, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572568' target='https://doi.org/10.1101/2023.12.22.572568'> 10.1101/2023.12.22.572568</a></p>
        <p class="abstract">The planarian Schmidtea mediterranea can regenerate its entire body from small tissue fragments and is studied as regeneration model species. The assembly and functional analysis of planarian genomes has proven challenging due its high A/T content (70% A/T), repetitive nature, and limited transferability of routine laboratory protocols due to their divergent biochemistry. Only few and often fragmented genome assemblies are currently available, and open challenges include the provision of well-annotated chromosome-scale reference assemblies of the model species and other planarians for a comparative genome evolution perspective. Here we report a haplotype-phased, chromosome-scale genome assembly and high-quality gene annotations of the sexual S2 strain of S. mediterranea and provide putative regulatory region annotations via optimized ATAC-seq and ChIP-seq protocols. To additionally leverage sequence conservation for regulatory element annotations, we generated chromosome-scale genome assemblies and chromatin accessibility data for the three closest relatives of S. mediterranea: S. polychroa, S. nova, and S. lugubris. We find substantial divergence in protein-coding sequences and regulatory regions, yet reveal remarkable conservation in ChIP-mark bearing open chromatin regions identified as promoters and enhancers in S. mediterranea. The resulting high-confidence set of evolutionary conserved enhancers and promoters provides a valuable resource for the analysis of gene regulatory circuits and their evolution within the taxon. In addition, our four chromosome-scale genome assemblies provide a first comparative perspective on planarian genome evolution. Our analyses reveal frequent retrotransposon-associated chromosomal inversions and inter-chromosomal translocations that lead to a degradation of synteny across the genus. Interestingly, we further find independent and near-complete losses of the ancestral metazoan synteny across Schmidtea and two other flatworm groups, indicating that platyhelminth genomes largely evolve without syntenic constraints. Our work provides valuable genome resources for the planarian research community and sets a foundation for the comparative genomics of planarians. We reveal a contrast between the fast structural evolution of planarian genomes and the conservation of their regulatory elements, suggesting a unique genome evolution in flatworms where gene positioning may not be essential.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572342">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572342" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572342">
        <p class="paperTitle">Single-nucleus RNA-sequencing reveals novel potential mechanisms of ovarian insufficiency in 45,X Turner Syndrome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572342" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572342" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McGlacken-Byrne, S. M.; Del Valle, I.; Xenakis, T.; Suntharalingham, J. P.; Nel, L.; Liptrot, D.; Crespo, B.; Ogunbiyi, O. K.; Niola, P.; Brooks, T.; Solanky, N.; Conway, G. S.; Achermann, J. C.</p>
        <p class="info">Score: 4.9, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572342' target='https://doi.org/10.1101/2023.12.22.572342'> 10.1101/2023.12.22.572342</a></p>
        <p class="abstract">Study questionCan single-nuclei and bulk RNA sequencing technologies be used to elucidate novel mechanisms of ovarian insufficiency in Turner Syndrome (TS)?

Summary answerUsing single-nucleus and bulk RNA sequencing approaches, we identified novel potential pathogenic mechanisms underlying ovarian insufficiency in TS including and beyond X chromosome haploinsufficiency.

What is known alreadyTurner syndrome (TS) is the most common genetic cause of Primary Ovarian Insufficiency (POI) in humans. Morphological analyses of human fetal 45,X ovaries have demonstrated fewer germ cells and marked apoptosis established by 15-20 weeks post conception (wpc); however, we do not understand why POI develops mechanistically in the first instance.

Study design, size, durationSingle-nucleus RNA sequencing (snRNA-seq): two 46,XX and two 45,X (TS) human fetal ovaries at 12-13 wpc. Bulk RNA sequencing: 19 human fetal ovary, 20 fetal testis, and 8 fetal control tissue (n=47 total samples; Carnegie Stage 22-16wpc).

Participants/materials, setting, methodsTo identify novel potential mechanisms of ovarian insufficiency in TS and to characterise X chromosome gene expression in the 45,X ovary, we performed snRNA-seq of peri-meiotic 46,XX (n=2) and 45,X (n=2) fetal ovaries at 12-13 weeks post conception (wpc); and 2) a bulk RNA sequencing time-series analysis of fetal ovary, testis, and control samples across four developmental timepoints.

Main results and the role of chanceGerm and somatic cell subpopulations were mostly shared across 46,XX and 45,X ovaries, aside from a 46XX-specific/45,X-depleted cluster of oogonia (&#34;synaptic oogonia&#34;) containing genes with functions relating to sex chromosome synapsis; histone modification; intracellular protein regulation and chaperone systems. snRNA-seq enabled accurate cell counting localised to individual cell clusters; the 45,X ovary has fewer germ cells than the 46,XX ovary in every germ cell subpopulation, confirmed by histopathological analysis. The normal sequence of X-chromosome inactivation and reactivation is disrupted in 45,X ovaries; XIST was not expressed in 45,X somatic cells but was present in germ cell clusters, albeit with lower expression than in corresponding 46,XX clusters. The 45,X ovary has a globally abnormal transcriptome, with low expression of genes with proteostasis functions (RSP4X); cell cycle progression (BUB1B); and OXPHOS mitochondrial energy production (COX6C, ATP11C). Genes with higher expression in 45,X cell populations were enriched for apoptotic functions (e.g., NR4A1).

Limitations, reasons for cautionLimitations include the relatively small sample size of the snRNA-seq analysis and the focus on a fixed meiotic timepoint which may overlook a dynamic process over time.

Wider implications of the findingsWe characterise the human fetal peri-meiotic 45,X ovary at single-cell resolution and offer insights into novel pathogenic mechanisms underlying ovarian insufficiency in TS. Although asynapsis due to X chromosome haploinsufficiency likely plays a significant role, these data suggest meiotic failure and ovarian insufficiency may be a combinatorial process characterised by periods of vulnerability throughout early 45,X germ cell development

Study funding/competing interest(s)This research was funded in whole, or in part, by the Wellcome Trust Grants 216362/Z/19/Z to SMcG-B and 209328/Z/17/Z to JCA. Human fetal material was provided by the Joint MRC/Wellcome Trust (Grant MR/R006237/1) Human Developmental Biology Resource (http://www.hdbr.org). Research at UCL Great Ormond Street Institute of Child Health is supported by the National Institute for Health Research, Great Ormond Street Hospital Biomedical Research Centre (grant IS-BRC-1215-20012).</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
