<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Sources of gene expression variation in a globally diverse human cohort
Authors: Taylor, D. J.; Chhetri, S. B.; Biddanda, A.; Tassia, M. G.; Battle, A.; McCoy, R. C.
Score: 45.2, Published: 2023-11-08 DOI: 10.1101/2023.11.04.565639
Abstract/SummaryGenetic variation influencing gene expression and splicing is a key source of phenotypic diversity. Though invaluable, studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering evolutionary research.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Sources of gene expression variation in a globally diverse human cohort
Authors: Taylor, D. J.; Chhetri, S. B.; Biddanda, A.; Tassia, M. G.; Battle, A.; McCoy, R. C.
Score: 45.2, Published: 2023-11-08 DOI: 10.1101/2023.11.04.565639
Abstract/SummaryGenetic variation influencing gene expression and splicing is a key source of phenotypic diversity. Though invaluable, studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering evolutionary research." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-12T10:37:43+00:00" />
<meta property="article:modified_time" content="2023-11-12T10:37:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Sources of gene expression variation in a globally diverse human cohort
Authors: Taylor, D. J.; Chhetri, S. B.; Biddanda, A.; Tassia, M. G.; Battle, A.; McCoy, R. C.
Score: 45.2, Published: 2023-11-08 DOI: 10.1101/2023.11.04.565639
Abstract/SummaryGenetic variation influencing gene expression and splicing is a key source of phenotypic diversity. Though invaluable, studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering evolutionary research."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Sources of gene expression variation in a globally diverse human cohort\nAuthors: Taylor, D. J.; Chhetri, S. B.; Biddanda, A.; Tassia, M. G.; Battle, A.; McCoy, R. C.\nScore: 45.2, Published: 2023-11-08 DOI: 10.1101/2023.11.04.565639\nAbstract/SummaryGenetic variation influencing gene expression and splicing is a key source of phenotypic diversity. Though invaluable, studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering evolutionary research.",
  "keywords": [
    
  ],
  "articleBody": " Sources of gene expression variation in a globally diverse human cohort\nAuthors: Taylor, D. J.; Chhetri, S. B.; Biddanda, A.; Tassia, M. G.; Battle, A.; McCoy, R. C.\nScore: 45.2, Published: 2023-11-08 DOI: 10.1101/2023.11.04.565639\nAbstract/SummaryGenetic variation influencing gene expression and splicing is a key source of phenotypic diversity. Though invaluable, studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering evolutionary research. To address these limitations, we developed MAGE, an open-access RNA-seq data set of lymphoblastoid cell lines from 731 individuals from the 1000 Genomes Project spread across 5 continental groups and 26 populations. Most variation in gene expression (92%) and splicing (95%) was distributed within versus between populations, mirroring variation in DNA sequence. We mapped associations between genetic variants and expression and splicing of nearby genes (cis-eQTLs and cis-sQTLs, respective), identifying \u003e15,000 putatively causal eQTLs and \u003e16,000 putatively causal sQTLs that are enriched for relevant epigenomic signatures. These include 1310 eQTLs and 1657 sQTLs that are largely private to previously underrepresented populations. Our data further indicate that the magnitude and direction of causal eQTL effects are highly consistent across populations and that apparent \"population-specific\" effects observed in previous studies were largely driven by low resolution or additional independent eQTLs of the same genes that were not detected. Together, our study expands understanding of gene expression diversity across human populations and provides an inclusive resource for studying the evolution and function of human genomes.\nIntragenomic conflicts with plasmids and chromosomal mobile genetic elements drive the evolution of natural transformation within species\nAuthors: Mazzamurro, F.; Chirakadavil, J. B.; Durieux, I.; Poire, L.; Plantade, J.; Ginevra, C.; Jarraud, S.; Wilharm, G.; Charpentier, X.; Rocha, E. P. C.\nScore: 27.8, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565790\nNatural transformation is the only mechanism of genetic exchange controlled by the recipient bacteria. We quantified its rates in 1282 strains of the human pathogens Legionella pneumophila (Lp) and Acinetobacter baumannii (Ab) and found that transformation rates evolve by large quick changes as a jump process across six orders of magnitude. Close to half of the strains are non-transformable in standard conditions. Transitions to non-transformability were frequent and recent, suggesting that they are deleterious and subsequently purged by natural selection. Accordingly, we find that transformation decreases genetic linkage in both species, which often accelerates adaptation. Intragenomic conflicts with chromosomal mobile genetic elements (MGEs) and plasmids could explain these transitions and a GWAS confirmed systematic negative associations between transformation and MGEs: plasmids and other conjugative elements in Lp, prophages in Ab, and transposable elements in both. In accordance with the modulation of transformation rates by genetic conflicts, transformable strains have fewer MGEs. Defense systems against the latter are associated with lower transformation except the adaptive CRISPR-Cas systems which show the inverse trend. The two species have different lifestyles and gene repertoires, but they exhibit very similar trends in terms of variation of transformation rates and its determinants, suggesting that genetic conflicts could drive the evolution of natural transformation in many bacteria.\nSingle-cell transcriptomic and genomic changes in the aging human brain\nAuthors: Jeffries, A. M.; Yu, T.; Ziegenfuss, J. S.; Tolles, A. K.; Kim, Y.; Weng, Z.; Lodato, M. A.\nScore: 21.6, Published: 2023-11-07 DOI: 10.1101/2023.11.07.566050\nAging brings dysregulation of various processes across organs and tissues, often stemming from stochastic damage to individual cells over time. Here, we used a combination of single-nucleus RNA-sequencing and single-cell whole-genome sequencing to identify transcriptomic and genomic changes in the prefrontal cortex of the human brain across life span, from infancy to centenarian. We identified infant-specific cell clusters enriched for the expression of neurodevelopmental genes, and a common down-regulation of cell-essential homeostatic genes that function in ribosomes, transport, and metabolism during aging across cell types. Conversely, expression of neuron-specific genes generally remains stable throughout life. We observed a decrease in specific DNA repair genes in aging, including genes implicated in generating brain somatic mutations as indicated by mutation signature analysis. Furthermore, we detected gene-length-specific somatic mutation rates that shape the transcriptomic landscape of the aged human brain. These findings elucidate critical aspects of human brain aging, shedding light on transcriptomic and genomics dynamics.\nModelling the demographic history of human North African genomes points to a recent soft split divergence between populations\nAuthors: Serradell, J. M.; Lorenzo-Salazar, J.; Flores, C.; Lao, O.; Comas, D.\nScore: 18.6, Published: 2023-11-09 DOI: 10.1101/2023.11.07.565966\nBackground: North African human populations present a complex demographic scenario. The presence of an autochthonous genetic component and population substructure, plus extensive gene flow from the Middle East, Europe, and sub-Saharan Africa, have shaped the genetic composition of its people through time. Results: We conducted a comprehensive analysis of 364 genomes to construct detailed demographic models for the North African region, encompassing its two primary ethnic groups, the Arab and Amazigh populations. This was achieved through the utilization of the Approximate Bayesian Computation with Deep Learning (ABC-DL) framework and a novel algorithm called Genetic Programming for Population Genetics (GP4PG). This innovative approach enabled us to effectively model intricate demographic scenarios, utilizing a subset of 16 whole-genomes at \u003e30X coverage. The demographic model suggested by GP4PG exhibited a closer alignment with the observed data compared to the ABC-DL model. Both methods point to a back-to-Africa origin of North African individuals and a close relationship of North African with Eurasian populations. Results support different origins for Amazigh and Arab populations, with Amazigh populations originating back in Epipaleolithic times, as early as 22.3 Kya. GP4PG model supports Arabization as the main source of Middle Eastern ancestry in North Africa. The GP4PG model better explaining the observed data includes population substructure in surrounding populations (sub-Saharan Africa and Middle East) with continuous gene flow after the split between populations (migration decay). In contrast to what we observed in the ABC-DL, the best GP4PG model does not require pulses of admixture from surrounding populations into North Africa pointing to soft splits as drivers of divergence in North Africa. Conclusions: We have built a demographic model on North Africa that points to a back-to-Africa expansion and a differential origin between Arab and Amazigh populations, emphasizing the complex demographic history at a population level.\nlink-ancestors: Fast simulation of local ancestry with tree sequence software\nAuthors: Tsambos, G.; Kelleher, J.; Ralph, P. L.; Leslie, S. J.; Vukcevic, D.\nScore: 10.6, Published: 2023-11-05 DOI: 10.1101/2023.11.03.564476\nSummaryIt is challenging to simulate realistic tracts of genetic ancestry on a scale suitable for simulation based inference. We present an algorithm that enables this information to be extracted efficiently from tree sequences produced by simulations run with msprime and SLiM. Availability and ImplementationA C-based implementation of the link-ancestors algorithm is in tskit (https://tskit.dev/tskit/docs/stable/). We also provide a user-friendly wrapper for link-ancestors in tspop, a Python-based utility package. Contactgtsambos@uw.edu. Supplementary informationTechnical details of link-ancestors are in the Supplementary Information. Documentation for the tspop package is at http://tspop.readthedocs.io/.\nIn vivo silencing of regulatory elements using a single AAV-CRISPRi vector\nAuthors: Laurette, P.; Cao, C.; Ramanujam, D.; Schwaderer, M.; Lueneburg, T.; Kuss, S.; Weiss, L.; Dilshat, R.; Furlong, E. E.; Rezende, F.; Engelhardt, S.; Gilsbach, R.\nScore: 9.1, Published: 2023-11-06 DOI: 10.1101/2023.11.05.565512\nCRISPR-Cas9 based transcriptional repressors (CRISPRi) have emerged as specific and robust tools for functional epigenetic silencing of regulatory elements. Adeno-associated viruses (AAVs) are promising CRISPRi delivery vectors for cardiovascular research and therapy. However, compact vectors enabling codelivery of all CRISPRi components by a single AAV are needed for an enhanced and consistent performance. We engineered a 4.7kb all-in-one CRISPRi construct compatible with AAV-mediated delivery and produced cardiotropic AAVi 6 and 9 particles for in vitro and in vivo application, respectively. AAVi vectors targeting the Nppa promoter (AAViNppa) reduced gene expression in cultivated cardiomyocytes (HL-1 cells) in a dose-dependent manner. The maximum effect was a \u003e95% reduction as measured by qPCR and RNA-seq. This effect was orchestrated by loss of chromatin accessibility (ATAC-seq) and establishment of heterochromatin (H3K9me3 ChIP-seq) specifically at the target promoter region. We confirmed the broad applicability of AAVi to different cardiomyocyte cell culture systems by silencing several genes in primary neonatal rat ventricular cardiomyocytes (NRVMs), human iPSC-derived cardioids and iPSC-CMs. To demonstrate the efficacy of AAVi in vivo we injected 8-week-old C57Bl/6 WT mice with a single dose of AAViNppa and implanted osmotic minipumps releasing Phenylephrine (50 mg/kg/d) and Angiotensin II (0.45 mg/kg/d) to induce Nppa transcription. AAViNppa silenced Nppa transcription as revealed by qPCR and single nuclei RNA-seq even under stress conditions. On the epigenome layer AAViNppa induced closed chromatin at the Nppa promoter site comparable to the in vitro effect. Here, we present an efficient AAV-based method for CRISPRi-mediated epigenetic silencing of gene expression in cardiac myocytes in vivo and in vitro. This functional epigenetic approach provides an efficient way to modulate gene expression in the heart and could become a standard method for cardiovascular disease modelling and translational research.\nGenomic structures and regulation patterns at HPV integration sites in cervical cancer\nAuthors: Porter, V. L.; O'Neill, K.; MacLennan, S.; Corbett, R. D.; Ng, M.; Culibrk, L.; Hamadeh, Z.; Iden, M.; Schmidt, R.; Tsaih, S.-W.; Chang, G.; Fan, J.; Nip, K. M.; Akbari, V.; Chan, S. K.; Hopkins, J.; Moore, R. A.; Chuah, E.; Mungall, K. L.; Mungall, A. J.; Birol, I.; Jones, S. J. M.; Rader, J. S.; Marra, M. A.\nScore: 8.8, Published: 2023-11-05 DOI: 10.1101/2023.11.04.564800\nHuman papillomavirus (HPV) integration has been implicated in transforming HPV infection into cancer, but its genomic consequences have been difficult to study using short-read technologies. To resolve the dysregulation associated with HPV integration, we performed long-read sequencing on 63 cervical cancer genomes. We identified six categories of integration events based on HPV-human genomic structures. Of all HPV integrants, defined as two HPV-human breakpoints bridged by an HPV sequence, 24% contained variable copies of HPV between the breakpoints, a phenomenon we termed heterologous integration. Analysis of DNA methylation within and in proximity to the HPV genome at individual integration events revealed relationships between methylation status of the integrant and its orientation and structure. Dysregulation of the human epigenome and neighboring gene expression in cis with the HPV-integrated allele was observed over megabase-ranges of the genome. By elucidating the structural, epigenetic, and allele-specific impacts of HPV integration, we provide insight into the role of integrated HPV in cervical cancer.\nGenetic Influences on Educational Attainment Through the Lens of the Evolving Swedish Welfare State: A cross-level gene-environment interaction study based on polygenic indices and longitudinal register data\nAuthors: Pettersson, O.\nScore: 9.1, Published: 2023-11-03 DOI: 10.1101/2023.11.02.565287\nGene-environment interaction with regards to educational attainment has received increasing attention during the last few years. However, the potential interdependence between different types of environments in gene-environment interaction models has mostly been neglected. Using high-quality register data for an extensive panel of Swedish twins, born during most of the twentieth century, this study explores how genetic propensities for educational attainment, as measured by a polygenic index, interact with both macro-level institutional and sociopolitical context, and with socioeconomic background. The analyses, which combine between-family and causally robust within-family models, suggest that the average association between genetic propensities and educational attainment has increased in Sweden during the twentieth century, along with the expansion of the educational system and decreased economic inequality. There is also evidence of a positive interaction between genetic propensities and socioeconomic background, but only in the oldest cohorts in the sample, and that were born before the Swedish welfare state had been fully established. This implies that micro-level gene-environment interactions can be significantly dependent on macro-level context, an insight that has arguably not yet been given sufficient attention in the literature. Acknowledging limitations of polygenic indices, and the arbitrariness of the genetic lottery, the results may nevertheless indicate a development towards higher equality of opportunity in Sweden during the twentieth century.\nNAD+ controls circadian rhythmicity during cardiac aging\nAuthors: Carpenter, B. J.; Lecacheur, M.; Mangold, Y. N.; Cui, K.; Guenther, S.; Dierickx, P.\nScore: 8.0, Published: 2023-11-05 DOI: 10.1101/2023.11.02.565150\nDisruption of the circadian clock as well as reduced NAD+ levels are both hallmarks of aging. While circadian rhythms and NAD+ metabolism have been linked in heart disease, their relationship during cardiac aging is less clear. Here, we show that aging leads to disruption of diurnal gene expression in the heart. Long-term supplementation with the NAD+ precursor nicotinamide riboside (NR) boosts NAD+ levels, reprograms the diurnal transcriptome and reverses naturally occurring cardiac enlargement in aged female mice. In addition, complete abolishment of NAD+ levels in CMs impairs PER2::luc oscillations, which is rescued by NR supplementation. These findings reveal an essential role for NAD+ in regulation of the cardiac circadian clock upon aging, which opens up new avenues to counteract age-related cardiac disorders.\nAntibiosis helps Aquificae surmount competition by archaeal thermophiles, and crowding by bacterial mesophiles, to dominate the boiling vent-water of a Trans-Himalayan sulfur-borax spring\nAuthors: Mondal, N.; Dutta, S.; Chatterjee, S.; Sarkar, J.; Mondal, M.; Roy, C.; Chakraborty, R.; Ghosh, W.\nScore: 7.0, Published: 2023-11-06 DOI: 10.1101/2023.07.10.548480\nTrans-Himalayan hot spring waters rich in sulfur, boron, sodium and chlorine (but poor in calcium and silicon) are known based on PCR-amplified 16S rRNA gene sequence data to harbor high diversities of infiltrating bacterial mesophiles. Yet, little is known about the population dynamics, primary productivity, mutual interactions, and thermal adaptations of the microorganisms present in the steaming waters discharged by these geochemically peculiar spring systems. We revealed these aspects of a bacteria-dominated microbiome thriving in the boiling (85{degrees}C) fluid vented by a sulfur-borax spring called Lotus Pond, situated at 4436 m above the mean sea-level, in the Puga valley of eastern Ladakh, on the Changthang plateau. Assembly, annotation, and population-binning of \u003e15-GB metagenomic sequence illuminated the numeral predominance of Aquificae (particularly, the genera Hydrogenobacter and Sulfurihydrogenibium), even as Proteobacteria, and Firmicutes, Cyanobacteria, Chloroflexi and Bacteroidetes, contributed most of the genetic diversity of the microbiome; prevalence and diversity of Deinococcus-Thermus, Thermotogae, Thermodesulfobacteria, Crenarchaeota and Euryarchaeota were far less. Perpetual presence of bacterial mesophiles in Lotus Ponds vent-water was evident from the detection of 51 such genera that had been discovered in previous 16S-amplicon-based investigations of the habitat, but have no member with laboratory growth at \u003e45{degrees}C. The metagenome was replete with genes crucial for hydrothermal, and other stress, adaptations, and a majority of them were attributed to phylogenetic relatives of mesophilic bacteria; examples included genes encoding heat shock proteins, molecular chaperones, and chaperonin complexes; proteins controlling/modulating/inhibiting DNA gyrase; universal stress proteins; methionine sulfoxide reductases; fatty acid desaturases; different toxin-antitoxin systems; enzymes protecting against oxidative damage; proteins conferring flagellar structure/function, chemotaxis, cell adhesion/aggregation, biofilm formation, and quorum sensing. The Lotus Pond Aquificae not only dominated the microbiome numerically but also acted potentially as the main primary producers of the ecosystem, with chemolithotrophic sulfur oxidation (Sox) being the fundamental bioenergetic mechanism, and reductive tricarboxylic acid (rTCA) cycle the predominant carbon fixation pathway. Furthermore, the Aquificae members present in situ contained copious genes for virulence functions, and biosynthesis of the antibiotics monobactams, streptomycin, prodigiosin, acarbose and validamycin, novobiocin, carbapenems, and vancomycin (a few genes were there for phenazine, and neomycin/kanamycin/gentamicin as well). The Lotus Pond Aquificae also contained copious genes for resistance against beta-lactam antibiotics (penicillins, cephalosporins, monobactams, and carbapenems) cationic antimicrobial peptides, vancomycin, aminoglycosides (streptomycin and neomycin/kanamycin/gentamicin), and fluoroquinolones; they also had several genes for multidrug resistance efflux pumps. Among the other thermophiles present, only Thermodesulfobacteria and Deinococcus-Thermus had genes for all these functions, albeit at much lesser frequencies; the Archaea and Thermotogae were either devoid of, or had very few genes for, the aforesaid functions. The Lotus Pond mesophiles, led by Alphaproteobacteria and Gammaproteobacteria, possessed a slightly wider variety, and higher abundance, of antibiotic synthesis/resistance genes, compared with the Aquificae, besides having a more uniform gene count across the antibiotics synthesized/resisted. We hypothesize that antibiosis helped Aquificae avoid niche overlap with other thermophiles/hyperthermophiles, especially the Archaea, besides exacerbating the bioenergetic costs of thermal endurance for the mesophiles infiltrating the system, thereby keeping the potential proliferation of these intruders at bay.\n",
  "wordCount" : "2641",
  "inLanguage": "en",
  "datePublished": "2023-11-12T10:37:43Z",
  "dateModified": "2023-11-12T10:37:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 12, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.04.565639">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.04.565639" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.04.565639">
        <p class="paperTitle">Sources of gene expression variation in a globally diverse human cohort</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.04.565639" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.04.565639" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, D. J.; Chhetri, S. B.; Biddanda, A.; Tassia, M. G.; Battle, A.; McCoy, R. C.</p>
        <p class="info">Score: 45.2, Published: 2023-11-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.04.565639' target='https://doi.org/10.1101/2023.11.04.565639'> 10.1101/2023.11.04.565639</a></p>
        <p class="abstract">Abstract/SummaryGenetic variation influencing gene expression and splicing is a key source of phenotypic diversity. Though invaluable, studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering evolutionary research. To address these limitations, we developed MAGE, an open-access RNA-seq data set of lymphoblastoid cell lines from 731 individuals from the 1000 Genomes Project spread across 5 continental groups and 26 populations. Most variation in gene expression (92%) and splicing (95%) was distributed within versus between populations, mirroring variation in DNA sequence. We mapped associations between genetic variants and expression and splicing of nearby genes (cis-eQTLs and cis-sQTLs, respective), identifying &gt;15,000 putatively causal eQTLs and &gt;16,000 putatively causal sQTLs that are enriched for relevant epigenomic signatures. These include 1310 eQTLs and 1657 sQTLs that are largely private to previously underrepresented populations. Our data further indicate that the magnitude and direction of causal eQTL effects are highly consistent across populations and that apparent &#34;population-specific&#34; effects observed in previous studies were largely driven by low resolution or additional independent eQTLs of the same genes that were not detected. Together, our study expands understanding of gene expression diversity across human populations and provides an inclusive resource for studying the evolution and function of human genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565790">
        <p class="paperTitle">Intragenomic conflicts with plasmids and chromosomal mobile genetic elements drive the evolution of natural transformation within species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mazzamurro, F.; Chirakadavil, J. B.; Durieux, I.; Poire, L.; Plantade, J.; Ginevra, C.; Jarraud, S.; Wilharm, G.; Charpentier, X.; Rocha, E. P. C.</p>
        <p class="info">Score: 27.8, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565790' target='https://doi.org/10.1101/2023.11.06.565790'> 10.1101/2023.11.06.565790</a></p>
        <p class="abstract">Natural transformation is the only mechanism of genetic exchange controlled by the recipient bacteria. We quantified its rates in 1282 strains of the human pathogens Legionella pneumophila (Lp) and Acinetobacter baumannii (Ab) and found that transformation rates evolve by large quick changes as a jump process across six orders of magnitude. Close to half of the strains are non-transformable in standard conditions. Transitions to non-transformability were frequent and recent, suggesting that they are deleterious and subsequently purged by natural selection. Accordingly, we find that transformation decreases genetic linkage in both species, which often accelerates adaptation. Intragenomic conflicts with chromosomal mobile genetic elements (MGEs) and plasmids could explain these transitions and a GWAS confirmed systematic negative associations between transformation and MGEs: plasmids and other conjugative elements in Lp, prophages in Ab, and transposable elements in both. In accordance with the modulation of transformation rates by genetic conflicts, transformable strains have fewer MGEs. Defense systems against the latter are associated with lower transformation except the adaptive CRISPR-Cas systems which show the inverse trend. The two species have different lifestyles and gene repertoires, but they exhibit very similar trends in terms of variation of transformation rates and its determinants, suggesting that genetic conflicts could drive the evolution of natural transformation in many bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566050">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566050" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566050">
        <p class="paperTitle">Single-cell transcriptomic and genomic changes in the aging human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566050" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566050" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jeffries, A. M.; Yu, T.; Ziegenfuss, J. S.; Tolles, A. K.; Kim, Y.; Weng, Z.; Lodato, M. A.</p>
        <p class="info">Score: 21.6, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566050' target='https://doi.org/10.1101/2023.11.07.566050'> 10.1101/2023.11.07.566050</a></p>
        <p class="abstract">Aging brings dysregulation of various processes across organs and tissues, often stemming from stochastic damage to individual cells over time. Here, we used a combination of single-nucleus RNA-sequencing and single-cell whole-genome sequencing to identify transcriptomic and genomic changes in the prefrontal cortex of the human brain across life span, from infancy to centenarian. We identified infant-specific cell clusters enriched for the expression of neurodevelopmental genes, and a common down-regulation of cell-essential homeostatic genes that function in ribosomes, transport, and metabolism during aging across cell types. Conversely, expression of neuron-specific genes generally remains stable throughout life. We observed a decrease in specific DNA repair genes in aging, including genes implicated in generating brain somatic mutations as indicated by mutation signature analysis. Furthermore, we detected gene-length-specific somatic mutation rates that shape the transcriptomic landscape of the aged human brain. These findings elucidate critical aspects of human brain aging, shedding light on transcriptomic and genomics dynamics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565966" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565966">
        <p class="paperTitle">Modelling the demographic history of human North African genomes points to a recent soft split divergence between populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Serradell, J. M.; Lorenzo-Salazar, J.; Flores, C.; Lao, O.; Comas, D.</p>
        <p class="info">Score: 18.6, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565966' target='https://doi.org/10.1101/2023.11.07.565966'> 10.1101/2023.11.07.565966</a></p>
        <p class="abstract">Background: North African human populations present a complex demographic scenario. The presence of an autochthonous genetic component and population substructure, plus extensive gene flow from the Middle East, Europe, and sub-Saharan Africa, have shaped the genetic composition of its people through time. Results: We conducted a comprehensive analysis of 364 genomes to construct detailed demographic models for the North African region, encompassing its two primary ethnic groups, the Arab and Amazigh populations. This was achieved through the utilization of the Approximate Bayesian Computation with Deep Learning (ABC-DL) framework and a novel algorithm called Genetic Programming for Population Genetics (GP4PG). This innovative approach enabled us to effectively model intricate demographic scenarios, utilizing a subset of 16 whole-genomes at &gt;30X coverage. The demographic model suggested by GP4PG exhibited a closer alignment with the observed data compared to the ABC-DL model. Both methods point to a back-to-Africa origin of North African individuals and a close relationship of North African with Eurasian populations. Results support different origins for Amazigh and Arab populations, with Amazigh populations originating back in Epipaleolithic times, as early as 22.3 Kya. GP4PG model supports Arabization as the main source of Middle Eastern ancestry in North Africa. The GP4PG model better explaining the observed data includes population substructure in surrounding populations (sub-Saharan Africa and Middle East) with continuous gene flow after the split between populations (migration decay). In contrast to what we observed in the ABC-DL, the best GP4PG model does not require pulses of admixture from surrounding populations into North Africa pointing to soft splits as drivers of divergence in North Africa. Conclusions: We have built a demographic model on North Africa that points to a back-to-Africa expansion and a differential origin between Arab and Amazigh populations, emphasizing the complex demographic history at a population level.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.564476">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.564476" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.564476">
        <p class="paperTitle">link-ancestors: Fast simulation of local ancestry with tree sequence software</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.564476" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.564476" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsambos, G.; Kelleher, J.; Ralph, P. L.; Leslie, S. J.; Vukcevic, D.</p>
        <p class="info">Score: 10.6, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.564476' target='https://doi.org/10.1101/2023.11.03.564476'> 10.1101/2023.11.03.564476</a></p>
        <p class="abstract">SummaryIt is challenging to simulate realistic tracts of genetic ancestry on a scale suitable for simulation based inference. We present an algorithm that enables this information to be extracted efficiently from tree sequences produced by simulations run with msprime and SLiM.

Availability and ImplementationA C-based implementation of the link-ancestors algorithm is in tskit (https://tskit.dev/tskit/docs/stable/). We also provide a user-friendly wrapper for link-ancestors in tspop, a Python-based utility package.

Contactgtsambos@uw.edu.

Supplementary informationTechnical details of link-ancestors are in the Supplementary Information. Documentation for the tspop package is at http://tspop.readthedocs.io/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.05.565512">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.05.565512" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.05.565512">
        <p class="paperTitle">In vivo silencing of regulatory elements using a single AAV-CRISPRi vector</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.05.565512" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.05.565512" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Laurette, P.; Cao, C.; Ramanujam, D.; Schwaderer, M.; Lueneburg, T.; Kuss, S.; Weiss, L.; Dilshat, R.; Furlong, E. E.; Rezende, F.; Engelhardt, S.; Gilsbach, R.</p>
        <p class="info">Score: 9.1, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.05.565512' target='https://doi.org/10.1101/2023.11.05.565512'> 10.1101/2023.11.05.565512</a></p>
        <p class="abstract">CRISPR-Cas9 based transcriptional repressors (CRISPRi) have emerged as specific and robust tools for functional epigenetic silencing of regulatory elements. Adeno-associated viruses (AAVs) are promising CRISPRi delivery vectors for cardiovascular research and therapy. However, compact vectors enabling codelivery of all CRISPRi components by a single AAV are needed for an enhanced and consistent performance.

We engineered a 4.7kb all-in-one CRISPRi construct compatible with AAV-mediated delivery and produced cardiotropic AAVi 6 and 9 particles for in vitro and in vivo application, respectively.

AAVi vectors targeting the Nppa promoter (AAViNppa) reduced gene expression in cultivated cardiomyocytes (HL-1 cells) in a dose-dependent manner. The maximum effect was a &gt;95% reduction as measured by qPCR and RNA-seq. This effect was orchestrated by loss of chromatin accessibility (ATAC-seq) and establishment of heterochromatin (H3K9me3 ChIP-seq) specifically at the target promoter region. We confirmed the broad applicability of AAVi to different cardiomyocyte cell culture systems by silencing several genes in primary neonatal rat ventricular cardiomyocytes (NRVMs), human iPSC-derived cardioids and iPSC-CMs.

To demonstrate the efficacy of AAVi in vivo we injected 8-week-old C57Bl/6 WT mice with a single dose of AAViNppa and implanted osmotic minipumps releasing Phenylephrine (50 mg/kg/d) and Angiotensin II (0.45 mg/kg/d) to induce Nppa transcription. AAViNppa silenced Nppa transcription as revealed by qPCR and single nuclei RNA-seq even under stress conditions. On the epigenome layer AAViNppa induced closed chromatin at the Nppa promoter site comparable to the in vitro effect. Here, we present an efficient AAV-based method for CRISPRi-mediated epigenetic silencing of gene expression in cardiac myocytes in vivo and in vitro. This functional epigenetic approach provides an efficient way to modulate gene expression in the heart and could become a standard method for cardiovascular disease modelling and translational research.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.04.564800">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.04.564800" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.04.564800">
        <p class="paperTitle">Genomic structures and regulation patterns at HPV integration sites in cervical cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.04.564800" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.04.564800" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Porter, V. L.; O&#39;Neill, K.; MacLennan, S.; Corbett, R. D.; Ng, M.; Culibrk, L.; Hamadeh, Z.; Iden, M.; Schmidt, R.; Tsaih, S.-W.; Chang, G.; Fan, J.; Nip, K. M.; Akbari, V.; Chan, S. K.; Hopkins, J.; Moore, R. A.; Chuah, E.; Mungall, K. L.; Mungall, A. J.; Birol, I.; Jones, S. J. M.; Rader, J. S.; Marra, M. A.</p>
        <p class="info">Score: 8.8, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.04.564800' target='https://doi.org/10.1101/2023.11.04.564800'> 10.1101/2023.11.04.564800</a></p>
        <p class="abstract">Human papillomavirus (HPV) integration has been implicated in transforming HPV infection into cancer, but its genomic consequences have been difficult to study using short-read technologies. To resolve the dysregulation associated with HPV integration, we performed long-read sequencing on 63 cervical cancer genomes. We identified six categories of integration events based on HPV-human genomic structures. Of all HPV integrants, defined as two HPV-human breakpoints bridged by an HPV sequence, 24% contained variable copies of HPV between the breakpoints, a phenomenon we termed heterologous integration. Analysis of DNA methylation within and in proximity to the HPV genome at individual integration events revealed relationships between methylation status of the integrant and its orientation and structure. Dysregulation of the human epigenome and neighboring gene expression in cis with the HPV-integrated allele was observed over megabase-ranges of the genome. By elucidating the structural, epigenetic, and allele-specific impacts of HPV integration, we provide insight into the role of integrated HPV in cervical cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565287">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565287" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565287">
        <p class="paperTitle">Genetic Influences on Educational Attainment Through the Lens of the Evolving Swedish Welfare State: A cross-level gene-environment interaction study based on polygenic indices and longitudinal register data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565287" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565287" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pettersson, O.</p>
        <p class="info">Score: 9.1, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565287' target='https://doi.org/10.1101/2023.11.02.565287'> 10.1101/2023.11.02.565287</a></p>
        <p class="abstract">Gene-environment interaction with regards to educational attainment has received increasing attention during the last few years. However, the potential interdependence between different types of environments in gene-environment interaction models has mostly been neglected. Using high-quality register data for an extensive panel of Swedish twins, born during most of the twentieth century, this study explores how genetic propensities for educational attainment, as measured by a polygenic index, interact with both macro-level institutional and sociopolitical context, and with socioeconomic background. The analyses, which combine between-family and causally robust within-family models, suggest that the average association between genetic propensities and educational attainment has increased in Sweden during the twentieth century, along with the expansion of the educational system and decreased economic inequality. There is also evidence of a positive interaction between genetic propensities and socioeconomic background, but only in the oldest cohorts in the sample, and that were born before the Swedish welfare state had been fully established. This implies that micro-level gene-environment interactions can be significantly dependent on macro-level context, an insight that has arguably not yet been given sufficient attention in the literature. Acknowledging limitations of polygenic indices, and the arbitrariness of the genetic lottery, the results may nevertheless indicate a development towards higher equality of opportunity in Sweden during the twentieth century.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565150">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565150" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565150">
        <p class="paperTitle">NAD&#43; controls circadian rhythmicity during cardiac aging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565150" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565150" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carpenter, B. J.; Lecacheur, M.; Mangold, Y. N.; Cui, K.; Guenther, S.; Dierickx, P.</p>
        <p class="info">Score: 8.0, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565150' target='https://doi.org/10.1101/2023.11.02.565150'> 10.1101/2023.11.02.565150</a></p>
        <p class="abstract">Disruption of the circadian clock as well as reduced NAD&#43; levels are both hallmarks of aging. While circadian rhythms and NAD&#43; metabolism have been linked in heart disease, their relationship during cardiac aging is less clear. Here, we show that aging leads to disruption of diurnal gene expression in the heart. Long-term supplementation with the NAD&#43; precursor nicotinamide riboside (NR) boosts NAD&#43; levels, reprograms the diurnal transcriptome and reverses naturally occurring cardiac enlargement in aged female mice. In addition, complete abolishment of NAD&#43; levels in CMs impairs PER2::luc oscillations, which is rescued by NR supplementation. These findings reveal an essential role for NAD&#43; in regulation of the cardiac circadian clock upon aging, which opens up new avenues to counteract age-related cardiac disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548480">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548480" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548480">
        <p class="paperTitle">Antibiosis helps Aquificae surmount competition by archaeal thermophiles, and crowding by bacterial mesophiles, to dominate the boiling vent-water of a Trans-Himalayan sulfur-borax spring</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548480" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548480" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mondal, N.; Dutta, S.; Chatterjee, S.; Sarkar, J.; Mondal, M.; Roy, C.; Chakraborty, R.; Ghosh, W.</p>
        <p class="info">Score: 7.0, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548480' target='https://doi.org/10.1101/2023.07.10.548480'> 10.1101/2023.07.10.548480</a></p>
        <p class="abstract">Trans-Himalayan hot spring waters rich in sulfur, boron, sodium and chlorine (but poor in calcium and silicon) are known based on PCR-amplified 16S rRNA gene sequence data to harbor high diversities of infiltrating bacterial mesophiles. Yet, little is known about the population dynamics, primary productivity, mutual interactions, and thermal adaptations of the microorganisms present in the steaming waters discharged by these geochemically peculiar spring systems. We revealed these aspects of a bacteria-dominated microbiome thriving in the boiling (85{degrees}C) fluid vented by a sulfur-borax spring called Lotus Pond, situated at 4436 m above the mean sea-level, in the Puga valley of eastern Ladakh, on the Changthang plateau. Assembly, annotation, and population-binning of &gt;15-GB metagenomic sequence illuminated the numeral predominance of Aquificae (particularly, the genera Hydrogenobacter and Sulfurihydrogenibium), even as Proteobacteria, and Firmicutes, Cyanobacteria, Chloroflexi and Bacteroidetes, contributed most of the genetic diversity of the microbiome; prevalence and diversity of Deinococcus-Thermus, Thermotogae, Thermodesulfobacteria, Crenarchaeota and Euryarchaeota were far less. Perpetual presence of bacterial mesophiles in Lotus Ponds vent-water was evident from the detection of 51 such genera that had been discovered in previous 16S-amplicon-based investigations of the habitat, but have no member with laboratory growth at &gt;45{degrees}C. The metagenome was replete with genes crucial for hydrothermal, and other stress, adaptations, and a majority of them were attributed to phylogenetic relatives of mesophilic bacteria; examples included genes encoding heat shock proteins, molecular chaperones, and chaperonin complexes; proteins controlling/modulating/inhibiting DNA gyrase; universal stress proteins; methionine sulfoxide reductases; fatty acid desaturases; different toxin-antitoxin systems; enzymes protecting against oxidative damage; proteins conferring flagellar structure/function, chemotaxis, cell adhesion/aggregation, biofilm formation, and quorum sensing. The Lotus Pond Aquificae not only dominated the microbiome numerically but also acted potentially as the main primary producers of the ecosystem, with chemolithotrophic sulfur oxidation (Sox) being the fundamental bioenergetic mechanism, and reductive tricarboxylic acid (rTCA) cycle the predominant carbon fixation pathway. Furthermore, the Aquificae members present in situ contained copious genes for virulence functions, and biosynthesis of the antibiotics monobactams, streptomycin, prodigiosin, acarbose and validamycin, novobiocin, carbapenems, and vancomycin (a few genes were there for phenazine, and neomycin/kanamycin/gentamicin as well). The Lotus Pond Aquificae also contained copious genes for resistance against beta-lactam antibiotics (penicillins, cephalosporins, monobactams, and carbapenems) cationic antimicrobial peptides, vancomycin, aminoglycosides (streptomycin and neomycin/kanamycin/gentamicin), and fluoroquinolones; they also had several genes for multidrug resistance efflux pumps. Among the other thermophiles present, only Thermodesulfobacteria and Deinococcus-Thermus had genes for all these functions, albeit at much lesser frequencies; the Archaea and Thermotogae were either devoid of, or had very few genes for, the aforesaid functions. The Lotus Pond mesophiles, led by Alphaproteobacteria and Gammaproteobacteria, possessed a slightly wider variety, and higher abundance, of antibiotic synthesis/resistance genes, compared with the Aquificae, besides having a more uniform gene count across the antibiotics synthesized/resisted. We hypothesize that antibiosis helped Aquificae avoid niche overlap with other thermophiles/hyperthermophiles, especially the Archaea, besides exacerbating the bioenergetic costs of thermal endurance for the mesophiles infiltrating the system, thereby keeping the potential proliferation of these intruders at bay.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
