<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Sources of gene expression variation in a globally diverse human cohort
Authors: Taylor, D. J.; Chhetri, S. B.; Biddanda, A.; Tassia, M. G.; Battle, A.; McCoy, R. C.
Score: 34.1, Published: 2023-11-05 DOI: 10.1101/2023.11.04.565639
Genetic variation influencing gene expression and splicing is a key source of phenotypic diversity. Though invaluable, studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering evolutionary research.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Sources of gene expression variation in a globally diverse human cohort
Authors: Taylor, D. J.; Chhetri, S. B.; Biddanda, A.; Tassia, M. G.; Battle, A.; McCoy, R. C.
Score: 34.1, Published: 2023-11-05 DOI: 10.1101/2023.11.04.565639
Genetic variation influencing gene expression and splicing is a key source of phenotypic diversity. Though invaluable, studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering evolutionary research." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Sources of gene expression variation in a globally diverse human cohort
Authors: Taylor, D. J.; Chhetri, S. B.; Biddanda, A.; Tassia, M. G.; Battle, A.; McCoy, R. C.
Score: 34.1, Published: 2023-11-05 DOI: 10.1101/2023.11.04.565639
Genetic variation influencing gene expression and splicing is a key source of phenotypic diversity. Though invaluable, studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering evolutionary research."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Sources of gene expression variation in a globally diverse human cohort\nAuthors: Taylor, D. J.; Chhetri, S. B.; Biddanda, A.; Tassia, M. G.; Battle, A.; McCoy, R. C.\nScore: 34.1, Published: 2023-11-05 DOI: 10.1101/2023.11.04.565639\nGenetic variation influencing gene expression and splicing is a key source of phenotypic diversity. Though invaluable, studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering evolutionary research.",
  "keywords": [
    
  ],
  "articleBody": " Sources of gene expression variation in a globally diverse human cohort\nAuthors: Taylor, D. J.; Chhetri, S. B.; Biddanda, A.; Tassia, M. G.; Battle, A.; McCoy, R. C.\nScore: 34.1, Published: 2023-11-05 DOI: 10.1101/2023.11.04.565639\nGenetic variation influencing gene expression and splicing is a key source of phenotypic diversity. Though invaluable, studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering evolutionary research. To address these limitations, we developed MAGE, an open-access RNA-seq data set of lymphoblastoid cell lines from 731 individuals from the 1000 Genomes Project spread across 5 continental groups and 26 populations. Most variation in gene expression (92%) and splicing (95%) was distributed within versus between populations, mirroring variation in DNA sequence. We mapped associations between genetic variants and expression and splicing of nearby genes (cis-eQTLs and cis-sQTLs, respective), identifying \u003e15,000 putatively causal eQTLs and \u003e16,000 putatively causal sQTLs that are enriched for relevant epigenomic signatures. These include 1310 eQTLs and 1657 sQTLs that are largely private to previously underrepresented populations. Our data further indicate that the magnitude and direction of causal eQTL effects are highly consistent across populations and that apparent \"population-specific\" effects observed in previous studies were largely driven by low resolution or additional independent eQTLs of the same genes that were not detected. Together, our study expands understanding of gene expression diversity across human populations and provides an inclusive resource for studying the evolution and function of human genomes.\nIntragenomic conflicts with plasmids and chromosomal mobile genetic elements drive the evolution of natural transformation within species\nAuthors: Mazzamurro, F.; Chirakadavil, J. B.; Durieux, I.; Poire, L.; Plantade, J.; Ginevra, C.; Jarraud, S.; Wilharm, G.; Charpentier, X.; Rocha, E. P. C.\nScore: 23.2, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565790\nNatural transformation is the only mechanism of genetic exchange controlled by the recipient bacteria. We quantified its rates in 1282 strains of the human pathogens Legionella pneumophila (Lp) and Acinetobacter baumannii (Ab) and found that transformation rates evolve by large quick changes as a jump process across six orders of magnitude. Close to half of the strains are non-transformable in standard conditions. Transitions to non-transformability were frequent and recent, suggesting that they are deleterious and subsequently purged by natural selection. Accordingly, we find that transformation decreases genetic linkage in both species, which often accelerates adaptation. Intragenomic conflicts with chromosomal mobile genetic elements (MGEs) and plasmids could explain these transitions and a GWAS confirmed systematic negative associations between transformation and MGEs: plasmids and other conjugative elements in Lp, prophages in Ab, and transposable elements in both. In accordance with the modulation of transformation rates by genetic conflicts, transformable strains have fewer MGEs. Defense systems against the latter are associated with lower transformation except the adaptive CRISPR-Cas systems which show the inverse trend. The two species have different lifestyles and gene repertoires, but they exhibit very similar trends in terms of variation of transformation rates and its determinants, suggesting that genetic conflicts could drive the evolution of natural transformation in many bacteria.\nFrom sequence to molecules: Feature sequence-based genome mining uncovers the hidden diversity of bacterial siderophore pathways\nAuthors: Gu, S.; Shao, Y.; Rehm, K.; Bigler, L.; Zhang, D.; He, R.; Shao, J.; Jousset, A.; Friman, V.-P.; Wei, Z.; Kummerli, R.; Li, Z.\nScore: 19.2, Published: 2023-11-06 DOI: 10.1101/2023.10.30.564663\nMicrobial secondary metabolites have long been recognized as a rich source for pharmaceutical compound discovery and to have crucial ecological functions. However, the sequence-to-function mapping in microbial secondary metabolism pathways remains challenging because neither protein function nor substrate specificity can accurately be predicted from genome data. Here we focus on the iron-scavenging pyoverdines, siderophores of Pseudomonas bacteria, as model system to develop a knowledge-guided bioinformatic pipeline that extracts functional information of both the pyoverdine synthesis machinery and uptake receptors from 1928 draft genomes. For pyoverdine synthesis, our approach predicts the chemical structure of 188 different pyoverdines with nearly 100% accuracy. For pyoverdine uptake, our pipeline uncovers 94 different pyoverdine receptor groups. Our results demonstrate that combining feature sequence and phylogenetic approaches is a powerful way to reconstruct bacterial secondary metabolism pathways based on sequence data, unveiling an enormous yet overlooked diversity of siderophores and their receptors.\nNon-canonical ORFs are an important source of tumor-specific antigens in a liver cancer meta-cohort\nAuthors: E. Camarena, M.; Theunissen, P.; Ruiz Egozcue, M.; Ruiz-Orera, J.; Calvo-Serra, B.; Castelo, R.; Sarobe, P.; Fortes, P.; Perera-Bel, J.; Alba, M. M.\nScore: 18.5, Published: 2023-11-01 DOI: 10.1101/2023.10.30.564375\nThe expression of tumor-specific antigens during cancer progression can trigger an immune response against the tumor. Antigens that have been used as cancer vaccines are those originated by non- synonymous mutations and those derived from cancer/testis antigens. However, the first class is predominantly patient-specific, preventing the development of therapies than can benefit multiple patients, and the second one offers a limited set of actionable targets. A possible alternative is the use of peptides derived from non-canonical ORFs (ncORFs). While many ncORFs have been shown to be translated in cancer cells, their tumor-specificity and patient distribution remains to be determined. Here we analyze RNA sequencing data 117 hepatocellular carcinoma (HCC) tumors and matched healthy tissue, together with ribosome profiling data from an additional 10 HCC tumors, to answer these open questions. Combining HLA-epitope binding predictions and experimental validation experiments we conclude that around 40% of the tumor-specific antigens in HCC are likely to be derived from ncORFs in lncRNAs, including two peptides that can trigger an immune response in mice. We identify a subset of 33 tumor-specific lncRNAs expressing novel cancer antigens shared by more than 10% of the HCC analyzed, which could be combined to target a large proportion of the patients. The results of the study open new avenues for extending the range of anti-cancer vaccines.\nThe complete human diploid reference genome of RPE-1 identifies the phased epigenetic landscapes from multi-omics data\nAuthors: Volpe, E.; Corda, L.; Di Tommaso, E.; Pelliccia, F.; Ottalevi, R.; Licastro, D.; Formenti, G.; Capulli, M.; Guarracino, A.; Tassone, E.; Giunta, S.\nScore: 15.8, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565049\nComparative analysis of recent human genome assemblies highlights profound sequence divergence that peaks within polymorphic loci such as centromeres. This raises the question about the adequacy of relying on human reference genomes to accurately analyze sequencing data derived from experimental cell lines. Here we propose a novel approach, referred to as isogenomic reference, that leverages a matched reference genome to perform multi-omics analyses. We have generated a new diploid genome assembly for the human retinal epithelial cells (RPE-1), a widely used non-cancer laboratory cell line with a stable diploid karyotype, that presents phased haplotypes and chromosome-level scaffolds that completely span centromeres. Using this assembly, we have characterized haplotype-resolved genomic variation unique to RPE-1, including a stable marker chromosome X with a 73.18 Mb segmental duplication of chromosome 10 translocated onto the microdeleted telomere t(Xq;10q), specific to this cell line. Comparative analyses revealed sequence polymorphism within centromeric regions, including unexpected genetic and epigenetic diversity among haplotypes for all chromosomes. Using our assembly as reference, we re-analyzed both our own and publicly available sequencing, methylation and epigenetic data generated in RPE-1 which had previously been analyzed with non-matched and non-diploid reference genomes. Our results show that the isogenomic reference improves alignments with an increased mapping quality up to 85% while halving mismatches, resulting in significant changes in peaks calling related to centromeres. Our work represents a proof-of-concept, showcasing the use of matched reference genomes for multiomics analyses and, at scale, serves as the foundation for a call to comprehensively assemble experimentally relevant cell lines for a widespread application of isogenomic reference genomes. HighlightsO_LITo improve the standard practice of aligning sequencing reads originating from laboratory human diploid cell lines against a reference, we present a novel approach that leverages the matched, or isogenomic reference genome for multi-omics analyses. C_LIO_LITo achieve this, we have generated a phased human diploid genome assembly of one of the most widely used non-cancer cell lines (RPE-1) with a stable diploid karyotype. We used this genome as a matched reference to analyze sequencing data from RPE-1. C_LIO_LIWe demonstrate how the isogenomic reference genome improves read alignment quality, peak calling accuracy and faithful assignment of sequencing data to each haplotype, uncovering for the first time inter-haplotype variation within the same diploid genome at polymorphic loci such as centromeres. C_LIO_LIThe RPE-1 genome represents a new telomere-to-telomere human diploid reference for the scientific community that will advance genetic and epigenetic research across fields using this cell line. C_LI\nThe GIAB genomic stratifications resource for human reference genomes\nAuthors: Dwarshuis, N.; Kalra, D.; McDaniel, J.; Sanio, P.; Alvarez Jerez, P.; Jadhav, B.; Huang, W.; Mondal, R.; Busby, B.; Olson, N. D.; Sedlazeck, F. J.; Wagner, J.; Majidian, S.; Zook, J. M.\nScore: 17.1, Published: 2023-10-29 DOI: 10.1101/2023.10.27.563846\nStratification of the genome into different genomic contexts is useful when developing bioinformatics software like variant callers, to assess performance in difficult regions in the human genome. Here we describe a set of genomic stratifications for the human reference genomes GRCh37, GRCh38, and T2T-CHM13v2.0. Generating stratifications for the new complete CHM13 reference genome is critical to understanding improvements in variant caller performance when using this new complete reference. The GIAB stratifications can be used when benchmarking variant calls to analyze difficult regions of the human genome in a standardized way. Here we present stratifications in the CHM13 genome in comparison to GRCh37 and GRCh38, highlighting expansions in hard-to-map and GC-rich stratifications which provide useful insight for accuracy of variants in these newly-added regions. To evaluate the reliability and utility of the new stratifications, we used the stratifications of the three references to assess accuracy of variant calls in diverse, challenging genomic regions. The means to generate these stratifications are available as a snakemake pipeline at https://github.com/ndwarshuis/giab-stratifications.\nOligo-barcodes illuminate holocentric karyotype evolution in Rhynchospora (Cyperaceae)\nAuthors: Mata-Sucre, Y.; Parteka, L. M.; Ritz, C.; Gatica-Arias, A.; Felix, L. P.; Thomas, W. W.; Souza, G.; Vanzela, A. L. L.; Pedrosa-Harand, A.; Marques, A.\nScore: 11.8, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564898\nHolocentric karyotypes are assumed to rapidly evolve through chromosome fusions and fissions due to the diffuse nature of their centromeres. Here, we took advantage of the recent availability of a chromosome-scale reference genome for Rhynchospora breviuscula, a model species of this holocentric genus, and developed the first set of oligo-based barcode probes for a holocentric plant. These probes were applied to 13 additional species of the genus, aiming to investigate the evolutionary dynamics driving the karyotype evolution in Rhynchospora. The two sets of probes were composed of 27,392 (green) and 23,968 (magenta) oligonucleotides, and generated 15 distinct FISH signals as a unique barcode pattern for the identification of all five chromosome pairs of the R. breviuscula karyotype. Oligo-FISH comparative analyzes revealed different types of rearrangements, such as fusions, fissions, putative inversions and translocations, as well as genomic duplications among the analyzed species. Two rounds of whole genome duplication (WGD) were demonstrated in R. pubera, but both analyzed accessions differed in the complex chain of events that gave rise to its large, structurally diploidized karyotypes with 2n = 10 or 12. Considering the phylogenetic relationships and divergence time of the species, the specificity and synteny of the probes were maintained up to species with a divergence time of [~]25 My. However, karyotype divergence in more distant species hindered chromosome mapping and the inference of specific events. This barcoding system is a powerful tool to study chromosomal variations and genomic evolution in holocentric chromosomes of Rhynchospora species.\nDeciphering histone mark-specific fine-scale chromatin organization at high resolution with Micro-C-ChIP\nAuthors: Metelova, M.; Krietenstein, N.\nScore: 11.6, Published: 2023-11-02 DOI: 10.1101/2023.10.30.563558\nThe regulation of cell-type-specific transcription programs is a highly controlled and complex process that needs to be fully understood. The gene regulation is often influenced by distal regulatory elements and their interactions with promoters in three-dimensional space. Although proximity ligation techniques like Hi-C have revolutionized our understanding of genome organization, the genomic resolution for many of these methods is limited by both experimental and financial constraints. Here, we introduce Micro-C-ChIP to provide extremely high-resolution views of chromosome architecture at genomic loci marked by specific covalent histone modifications. This is achieved by chromatin immunoprecipitation of specific chromatin states to target chromosome folding libraries to focus on chromatin domains (regulatory elements, heterochromatin, etc.) of interest, yielding extremely high sequencing depth at these loci. We applied Micro-C-ChIP to mouse embryonic stem cells (mESC) and hTERT-immortalized human retinal epithelial cells (hTERT-RPE1), revealing architectural features of genome organization with comparable or higher resolution than Micro-C datasets sequenced with higher depth. We discovered extensive promoter-promoter networks in both cell types and characterized the specific architecture of bivalently marked promoters in mESC. Together, these data highlight Micro-C-ChIP as a cost-effective approach to exploring the landscape of genome folding at extraordinarily high resolution.\nThe shared genetic architecture and evolution of human language and musical rhythm\nAuthors: Alagoez, G.; Eising, E.; Mekki, Y.; Bignardi, G.; Fontanillas, P.; 23andMe Research Team, ; Nivard, M. G.; Luciano, M.; Cox, N. J.; Fisher, S. E.; Gordon, R. L.\nScore: 11.4, Published: 2023-11-02 DOI: 10.1101/2023.11.01.564908\nRhythm and language-related traits are phenotypically correlated, but their genetic overlap is largely unknown. Here, we leveraged two large-scale genome-wide association studies performed to shed light on the shared genetics of rhythm (N=606,825) and dyslexia (N=1,138,870). Our results reveal an intricate shared genetic and neurobiological architecture, and lay groundwork for resolving longstanding debates about the potential co-evolution of human language and musical traits.\nlink-ancestors: Fast simulation of local ancestry with tree sequence software\nAuthors: Tsambos, G.; Kelleher, J.; Ralph, P. L.; Leslie, S. J.; Vukcevic, D.\nScore: 10.6, Published: 2023-11-05 DOI: 10.1101/2023.11.03.564476\nIt is challenging to simulate realistic tracts of genetic ancestry on a scale suitable for simulation based inference. We present an algorithm that enables this information to be extracted efficiently from tree sequences produced by simulations run with msprime and SLiM.\n",
  "wordCount" : "2302",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.04.565639">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.04.565639" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.04.565639">
        <p class="paperTitle">Sources of gene expression variation in a globally diverse human cohort</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.04.565639" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.04.565639" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, D. J.; Chhetri, S. B.; Biddanda, A.; Tassia, M. G.; Battle, A.; McCoy, R. C.</p>
        <p class="info">Score: 34.1, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.04.565639' target='https://doi.org/10.1101/2023.11.04.565639'> 10.1101/2023.11.04.565639</a></p>
        <p class="abstract">Genetic variation influencing gene expression and splicing is a key source of phenotypic diversity. Though invaluable, studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering evolutionary research. To address these limitations, we developed MAGE, an open-access RNA-seq data set of lymphoblastoid cell lines from 731 individuals from the 1000 Genomes Project spread across 5 continental groups and 26 populations. Most variation in gene expression (92%) and splicing (95%) was distributed within versus between populations, mirroring variation in DNA sequence. We mapped associations between genetic variants and expression and splicing of nearby genes (cis-eQTLs and cis-sQTLs, respective), identifying &gt;15,000 putatively causal eQTLs and &gt;16,000 putatively causal sQTLs that are enriched for relevant epigenomic signatures. These include 1310 eQTLs and 1657 sQTLs that are largely private to previously underrepresented populations. Our data further indicate that the magnitude and direction of causal eQTL effects are highly consistent across populations and that apparent &#34;population-specific&#34; effects observed in previous studies were largely driven by low resolution or additional independent eQTLs of the same genes that were not detected. Together, our study expands understanding of gene expression diversity across human populations and provides an inclusive resource for studying the evolution and function of human genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565790">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565790" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565790">
        <p class="paperTitle">Intragenomic conflicts with plasmids and chromosomal mobile genetic elements drive the evolution of natural transformation within species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565790" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565790" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mazzamurro, F.; Chirakadavil, J. B.; Durieux, I.; Poire, L.; Plantade, J.; Ginevra, C.; Jarraud, S.; Wilharm, G.; Charpentier, X.; Rocha, E. P. C.</p>
        <p class="info">Score: 23.2, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565790' target='https://doi.org/10.1101/2023.11.06.565790'> 10.1101/2023.11.06.565790</a></p>
        <p class="abstract">Natural transformation is the only mechanism of genetic exchange controlled by the recipient bacteria. We quantified its rates in 1282 strains of the human pathogens Legionella pneumophila (Lp) and Acinetobacter baumannii (Ab) and found that transformation rates evolve by large quick changes as a jump process across six orders of magnitude. Close to half of the strains are non-transformable in standard conditions. Transitions to non-transformability were frequent and recent, suggesting that they are deleterious and subsequently purged by natural selection. Accordingly, we find that transformation decreases genetic linkage in both species, which often accelerates adaptation. Intragenomic conflicts with chromosomal mobile genetic elements (MGEs) and plasmids could explain these transitions and a GWAS confirmed systematic negative associations between transformation and MGEs: plasmids and other conjugative elements in Lp, prophages in Ab, and transposable elements in both. In accordance with the modulation of transformation rates by genetic conflicts, transformable strains have fewer MGEs. Defense systems against the latter are associated with lower transformation except the adaptive CRISPR-Cas systems which show the inverse trend. The two species have different lifestyles and gene repertoires, but they exhibit very similar trends in terms of variation of transformation rates and its determinants, suggesting that genetic conflicts could drive the evolution of natural transformation in many bacteria.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564663">
        <p class="paperTitle">From sequence to molecules: Feature sequence-based genome mining uncovers the hidden diversity of bacterial siderophore pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gu, S.; Shao, Y.; Rehm, K.; Bigler, L.; Zhang, D.; He, R.; Shao, J.; Jousset, A.; Friman, V.-P.; Wei, Z.; Kummerli, R.; Li, Z.</p>
        <p class="info">Score: 19.2, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564663' target='https://doi.org/10.1101/2023.10.30.564663'> 10.1101/2023.10.30.564663</a></p>
        <p class="abstract">Microbial secondary metabolites have long been recognized as a rich source for pharmaceutical compound discovery and to have crucial ecological functions. However, the sequence-to-function mapping in microbial secondary metabolism pathways remains challenging because neither protein function nor substrate specificity can accurately be predicted from genome data. Here we focus on the iron-scavenging pyoverdines, siderophores of Pseudomonas bacteria, as model system to develop a knowledge-guided bioinformatic pipeline that extracts functional information of both the pyoverdine synthesis machinery and uptake receptors from 1928 draft genomes. For pyoverdine synthesis, our approach predicts the chemical structure of 188 different pyoverdines with nearly 100% accuracy. For pyoverdine uptake, our pipeline uncovers 94 different pyoverdine receptor groups. Our results demonstrate that combining feature sequence and phylogenetic approaches is a powerful way to reconstruct bacterial secondary metabolism pathways based on sequence data, unveiling an enormous yet overlooked diversity of siderophores and their receptors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564375">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564375" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564375">
        <p class="paperTitle">Non-canonical ORFs are an important source of tumor-specific antigens in a liver cancer meta-cohort</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564375" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564375" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: E. Camarena, M.; Theunissen, P.; Ruiz Egozcue, M.; Ruiz-Orera, J.; Calvo-Serra, B.; Castelo, R.; Sarobe, P.; Fortes, P.; Perera-Bel, J.; Alba, M. M.</p>
        <p class="info">Score: 18.5, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564375' target='https://doi.org/10.1101/2023.10.30.564375'> 10.1101/2023.10.30.564375</a></p>
        <p class="abstract">The expression of tumor-specific antigens during cancer progression can trigger an immune response against the tumor. Antigens that have been used as cancer vaccines are those originated by non- synonymous mutations and those derived from cancer/testis antigens. However, the first class is predominantly patient-specific, preventing the development of therapies than can benefit multiple patients, and the second one offers a limited set of actionable targets. A possible alternative is the use of peptides derived from non-canonical ORFs (ncORFs). While many ncORFs have been shown to be translated in cancer cells, their tumor-specificity and patient distribution remains to be determined. Here we analyze RNA sequencing data 117 hepatocellular carcinoma (HCC) tumors and matched healthy tissue, together with ribosome profiling data from an additional 10 HCC tumors, to answer these open questions. Combining HLA-epitope binding predictions and experimental validation experiments we conclude that around 40% of the tumor-specific antigens in HCC are likely to be derived from ncORFs in lncRNAs, including two peptides that can trigger an immune response in mice. We identify a subset of 33 tumor-specific lncRNAs expressing novel cancer antigens shared by more than 10% of the HCC analyzed, which could be combined to target a large proportion of the patients. The results of the study open new avenues for extending the range of anti-cancer vaccines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565049" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565049">
        <p class="paperTitle">The complete human diploid reference genome of RPE-1 identifies the phased epigenetic landscapes from multi-omics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Volpe, E.; Corda, L.; Di Tommaso, E.; Pelliccia, F.; Ottalevi, R.; Licastro, D.; Formenti, G.; Capulli, M.; Guarracino, A.; Tassone, E.; Giunta, S.</p>
        <p class="info">Score: 15.8, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565049' target='https://doi.org/10.1101/2023.11.01.565049'> 10.1101/2023.11.01.565049</a></p>
        <p class="abstract">Comparative analysis of recent human genome assemblies highlights profound sequence divergence that peaks within polymorphic loci such as centromeres. This raises the question about the adequacy of relying on human reference genomes to accurately analyze sequencing data derived from experimental cell lines. Here we propose a novel approach, referred to as  isogenomic reference, that leverages a matched reference genome to perform multi-omics analyses. We have generated a new diploid genome assembly for the human retinal epithelial cells (RPE-1), a widely used non-cancer laboratory cell line with a stable diploid karyotype, that presents phased haplotypes and chromosome-level scaffolds that completely span centromeres. Using this assembly, we have characterized haplotype-resolved genomic variation unique to RPE-1, including a stable marker chromosome X with a 73.18 Mb segmental duplication of chromosome 10 translocated onto the microdeleted telomere t(Xq;10q), specific to this cell line. Comparative analyses revealed sequence polymorphism within centromeric regions, including unexpected genetic and epigenetic diversity among haplotypes for all chromosomes. Using our assembly as reference, we re-analyzed both our own and publicly available sequencing, methylation and epigenetic data generated in RPE-1 which had previously been analyzed with non-matched and non-diploid reference genomes. Our results show that the isogenomic reference improves alignments with an increased mapping quality up to 85% while halving mismatches, resulting in significant changes in peaks calling related to centromeres. Our work represents a proof-of-concept, showcasing the use of matched reference genomes for multiomics analyses and, at scale, serves as the foundation for a call to comprehensively assemble experimentally relevant cell lines for a widespread application of isogenomic reference genomes.

HighlightsO_LITo improve the standard practice of aligning sequencing reads originating from laboratory human diploid cell lines against a reference, we present a novel approach that leverages the matched, or  isogenomic reference genome for multi-omics analyses.
C_LIO_LITo achieve this, we have generated a phased human diploid genome assembly of one of the most widely used non-cancer cell lines (RPE-1) with a stable diploid karyotype. We used this genome as a matched reference to analyze sequencing data from RPE-1.
C_LIO_LIWe demonstrate how the isogenomic reference genome improves read alignment quality, peak calling accuracy and faithful assignment of sequencing data to each haplotype, uncovering for the first time inter-haplotype variation within the same diploid genome at polymorphic loci such as centromeres.
C_LIO_LIThe RPE-1 genome represents a new telomere-to-telomere human diploid reference for the scientific community that will advance genetic and epigenetic research across fields using this cell line.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.563846">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.563846" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.563846">
        <p class="paperTitle">The GIAB genomic stratifications resource for human reference genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.563846" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.563846" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dwarshuis, N.; Kalra, D.; McDaniel, J.; Sanio, P.; Alvarez Jerez, P.; Jadhav, B.; Huang, W.; Mondal, R.; Busby, B.; Olson, N. D.; Sedlazeck, F. J.; Wagner, J.; Majidian, S.; Zook, J. M.</p>
        <p class="info">Score: 17.1, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.563846' target='https://doi.org/10.1101/2023.10.27.563846'> 10.1101/2023.10.27.563846</a></p>
        <p class="abstract">Stratification of the genome into different genomic contexts is useful when developing bioinformatics software like variant callers, to assess performance in difficult regions in the human genome. Here we describe a set of genomic stratifications for the human reference genomes GRCh37, GRCh38, and T2T-CHM13v2.0. Generating stratifications for the new complete CHM13 reference genome is critical to understanding improvements in variant caller performance when using this new complete reference. The GIAB stratifications can be used when benchmarking variant calls to analyze difficult regions of the human genome in a standardized way. Here we present stratifications in the CHM13 genome in comparison to GRCh37 and GRCh38, highlighting expansions in hard-to-map and GC-rich stratifications which provide useful insight for accuracy of variants in these newly-added regions. To evaluate the reliability and utility of the new stratifications, we used the stratifications of the three references to assess accuracy of variant calls in diverse, challenging genomic regions. The means to generate these stratifications are available as a snakemake pipeline at https://github.com/ndwarshuis/giab-stratifications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564898">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564898" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564898">
        <p class="paperTitle">Oligo-barcodes illuminate holocentric karyotype evolution in Rhynchospora (Cyperaceae)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564898" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564898" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mata-Sucre, Y.; Parteka, L. M.; Ritz, C.; Gatica-Arias, A.; Felix, L. P.; Thomas, W. W.; Souza, G.; Vanzela, A. L. L.; Pedrosa-Harand, A.; Marques, A.</p>
        <p class="info">Score: 11.8, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564898' target='https://doi.org/10.1101/2023.10.31.564898'> 10.1101/2023.10.31.564898</a></p>
        <p class="abstract">Holocentric karyotypes are assumed to rapidly evolve through chromosome fusions and fissions due to the diffuse nature of their centromeres. Here, we took advantage of the recent availability of a chromosome-scale reference genome for Rhynchospora breviuscula, a model species of this holocentric genus, and developed the first set of oligo-based barcode probes for a holocentric plant. These probes were applied to 13 additional species of the genus, aiming to investigate the evolutionary dynamics driving the karyotype evolution in Rhynchospora. The two sets of probes were composed of 27,392 (green) and 23,968 (magenta) oligonucleotides, and generated 15 distinct FISH signals as a unique barcode pattern for the identification of all five chromosome pairs of the R. breviuscula karyotype. Oligo-FISH comparative analyzes revealed different types of rearrangements, such as fusions, fissions, putative inversions and translocations, as well as genomic duplications among the analyzed species. Two rounds of whole genome duplication (WGD) were demonstrated in R. pubera, but both analyzed accessions differed in the complex chain of events that gave rise to its large, structurally diploidized karyotypes with 2n = 10 or 12. Considering the phylogenetic relationships and divergence time of the species, the specificity and synteny of the probes were maintained up to species with a divergence time of [~]25 My. However, karyotype divergence in more distant species hindered chromosome mapping and the inference of specific events. This barcoding system is a powerful tool to study chromosomal variations and genomic evolution in holocentric chromosomes of Rhynchospora species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.563558">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.563558" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.563558">
        <p class="paperTitle">Deciphering histone mark-specific fine-scale chromatin organization at high resolution with Micro-C-ChIP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.563558" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.563558" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Metelova, M.; Krietenstein, N.</p>
        <p class="info">Score: 11.6, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.563558' target='https://doi.org/10.1101/2023.10.30.563558'> 10.1101/2023.10.30.563558</a></p>
        <p class="abstract">The regulation of cell-type-specific transcription programs is a highly controlled and complex process that needs to be fully understood. The gene regulation is often influenced by distal regulatory elements and their interactions with promoters in three-dimensional space. Although proximity ligation techniques like Hi-C have revolutionized our understanding of genome organization, the genomic resolution for many of these methods is limited by both experimental and financial constraints. Here, we introduce Micro-C-ChIP to provide extremely high-resolution views of chromosome architecture at genomic loci marked by specific covalent histone modifications. This is achieved by chromatin immunoprecipitation of specific chromatin states to target chromosome folding libraries to focus on chromatin domains (regulatory elements, heterochromatin, etc.) of interest, yielding extremely high sequencing depth at these loci. We applied Micro-C-ChIP to mouse embryonic stem cells (mESC) and hTERT-immortalized human retinal epithelial cells (hTERT-RPE1), revealing architectural features of genome organization with comparable or higher resolution than Micro-C datasets sequenced with higher depth. We discovered extensive promoter-promoter networks in both cell types and characterized the specific architecture of bivalently marked promoters in mESC. Together, these data highlight Micro-C-ChIP as a cost-effective approach to exploring the landscape of genome folding at extraordinarily high resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.564908">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.564908" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.564908">
        <p class="paperTitle">The shared genetic architecture and evolution of human language and musical rhythm</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.564908" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.564908" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alagoez, G.; Eising, E.; Mekki, Y.; Bignardi, G.; Fontanillas, P.; 23andMe Research Team,  ; Nivard, M. G.; Luciano, M.; Cox, N. J.; Fisher, S. E.; Gordon, R. L.</p>
        <p class="info">Score: 11.4, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.564908' target='https://doi.org/10.1101/2023.11.01.564908'> 10.1101/2023.11.01.564908</a></p>
        <p class="abstract">Rhythm and language-related traits are phenotypically correlated, but their genetic overlap is largely unknown. Here, we leveraged two large-scale genome-wide association studies performed to shed light on the shared genetics of rhythm (N=606,825) and dyslexia (N=1,138,870). Our results reveal an intricate shared genetic and neurobiological architecture, and lay groundwork for resolving longstanding debates about the potential co-evolution of human language and musical traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.564476">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.564476" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.564476">
        <p class="paperTitle">link-ancestors: Fast simulation of local ancestry with tree sequence software</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.564476" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.564476" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tsambos, G.; Kelleher, J.; Ralph, P. L.; Leslie, S. J.; Vukcevic, D.</p>
        <p class="info">Score: 10.6, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.564476' target='https://doi.org/10.1101/2023.11.03.564476'> 10.1101/2023.11.03.564476</a></p>
        <p class="abstract">It is challenging to simulate realistic tracts of genetic ancestry on a scale suitable for simulation based inference. We present an algorithm that enables this information to be extracted efficiently from tree sequences produced by simulations run with msprime and SLiM.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
