<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Regulatory Transposable Elements in the Encyclopedia of DNA Elements
Authors: Du, A. Y.; Chobirko, J. D.; Zhuo, X.; Feschotte, C.; Wang, T.
Score: 51.5, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556380
Transposable elements (TEs) make up about half of the human genome and many have the biochemical hallmarks of tissue- or cell type-specific cis-regulatory elements. While some TEs have been rigorously documented to contribute directly to host gene regulation, we still have a very partial view of their regulatory landscape.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Regulatory Transposable Elements in the Encyclopedia of DNA Elements
Authors: Du, A. Y.; Chobirko, J. D.; Zhuo, X.; Feschotte, C.; Wang, T.
Score: 51.5, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556380
Transposable elements (TEs) make up about half of the human genome and many have the biochemical hallmarks of tissue- or cell type-specific cis-regulatory elements. While some TEs have been rigorously documented to contribute directly to host gene regulation, we still have a very partial view of their regulatory landscape." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-13T10:39:03+00:00" />
<meta property="article:modified_time" content="2023-09-13T10:39:03+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Regulatory Transposable Elements in the Encyclopedia of DNA Elements
Authors: Du, A. Y.; Chobirko, J. D.; Zhuo, X.; Feschotte, C.; Wang, T.
Score: 51.5, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556380
Transposable elements (TEs) make up about half of the human genome and many have the biochemical hallmarks of tissue- or cell type-specific cis-regulatory elements. While some TEs have been rigorously documented to contribute directly to host gene regulation, we still have a very partial view of their regulatory landscape."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Regulatory Transposable Elements in the Encyclopedia of DNA Elements\nAuthors: Du, A. Y.; Chobirko, J. D.; Zhuo, X.; Feschotte, C.; Wang, T.\nScore: 51.5, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556380\nTransposable elements (TEs) make up about half of the human genome and many have the biochemical hallmarks of tissue- or cell type-specific cis-regulatory elements. While some TEs have been rigorously documented to contribute directly to host gene regulation, we still have a very partial view of their regulatory landscape.",
  "keywords": [
    
  ],
  "articleBody": " Regulatory Transposable Elements in the Encyclopedia of DNA Elements\nAuthors: Du, A. Y.; Chobirko, J. D.; Zhuo, X.; Feschotte, C.; Wang, T.\nScore: 51.5, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556380\nTransposable elements (TEs) make up about half of the human genome and many have the biochemical hallmarks of tissue- or cell type-specific cis-regulatory elements. While some TEs have been rigorously documented to contribute directly to host gene regulation, we still have a very partial view of their regulatory landscape. Leveraging Phase 4 ENCODE data, we carried out the most comprehensive study to date of TE contributions to the regulatory genome. Here we investigated the sequence origins of candidate cis-regulatory elements (cCREs), showing that [~]25% of human cCREs comprising 236,181 elements are derived from TEs. Human-mouse comparisons indicate that over 90% of TE-derived cCREs are lineage-specific, accounting for 8-36% of lineage-specific cCREs across cCRE types. Next, we found that cCRE-associated transcription factor (TF) binding motifs in TEs originated from TE ancestral sequences significantly more than expected in all TE classes except for SINEs. Using both cCRE and TF binding data, we discovered that TEs providing cCREs and TF binding sites are closer in genomic distance to non-TE sites compared to other TEs, suggesting that TE integration site influences their later co-option as regulatory elements. We show that TEs have promoted TF binding site turnover events since human-mouse divergence, accounting for 3-56% of turnover events across 30 TFs examined. Finally, we demonstrate that TE-derived cCREs share similar features with non-TE cCREs, including massively parallel reporter assay activity and GWAS variant enrichment. Overall, our results substantiate the notion that TEs have played an important role in shaping the human regulatory genome.\nCrossing host boundaries: the evolutionary drivers and correlates of viral host jumps\nAuthors: Tan, C. C. S.; van Dorp, L.; Balloux, F.\nScore: 23.6, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555953\nMost emerging and re-emerging infectious diseases stem from viruses that naturally circulate in non-human vertebrates. When these viruses cross over into humans, they can cause disease outbreaks and epidemics. While zoonotic host jumps have been extensively studied from an ecological perspective, little attention has gone into characterising the evolutionary drivers and correlates underlying these events. To address this gap, we harnessed the entirety of publicly available viral genomic data, employing a comprehensive suite of network and phylogenetic analyses. We address a series of questions concerning the evolutionary mechanisms underpinning viral host jumps. Notably, we challenge conventional assumptions about the directionality of host jumps, demonstrating that humans are as much a source as a sink for viral spillover events, insofar we infer more viruses to have jumped from humans to other animals, than from animals to humans. Moreover, we demonstrate heightened evolution in viral lineages that involve putative host jumps. We further observe that the mutational threshold associated with a host jump is lower for viruses with broad host ranges. Finally, we show that the genomic targets of natural selection upon a successful host jump vary across different viral families with either structural or auxiliary genes being the prime targets of selection. Collectively, our results illuminate some of the evolutionary drivers underlying viral host jumps that may contribute to mitigating viral threats across species boundaries.\nA General Approach to Adjusting Genetic Studies for Assortative Mating\nAuthors: Bilghese, M.; Manansala, R.; Jaishankar, D.; Jala, J.; Benjamin, D. J.; Kimball, M.; Auer, P. L.; Livermore, M. A.; Turley, P.\nScore: 20.2, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555983\nThe effects of assortative mating (AM) on estimates from genetic studies has been receiving increasing attention in recent years. We extend existing AM theory to more general models of sorting and conclude that correct theory-based AM adjustments require knowledge of complicated, unknown historical sorting patterns. We propose a simple, general-purpose approach using polygenic indexes (PGIs). Our approach can estimate the fraction of genetic variance and genetic correlation that is driven by AM. Our approach is less effective when applied to Mendelian randomization (MR) studies for two reasons: AM can induce a form of selection bias in MR studies that remains after our adjustment; and, in the MR context, the adjustment is particularly sensitive to PGI estimation error. Using data from the UK Biobank, we find that AM inflates genetic correlation estimates between health traits and education by 14% on average. Our results suggest caution in interpreting genetic correlations or MR estimates for traits subject to AM.\nAIRR-C Human IG Reference Sets: curated sets of immunoglobulin heavy and light chain germline genes\nAuthors: Collins, A. M.; Ohlin, M.; Corcoran, M.; Heather, J. M.; Ralph, D.; Law, M.; Martinez-Barnetche, J.; Ye, J.; Richardson, E.; Gibson, W. S.; Rodriguez, O. L.; Peres, A.; Yaari, G.; Watson, C. T.; Lees, W. D.\nScore: 21.1, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555348\nAnalysis of an individuals immunoglobulin (IG) gene repertoire requires the use of high-quality germline gene Reference Sets. The Adaptive Immune Receptor Repertoire-Community (AIRR-C) Reference Sets have been developed to include only human IG heavy and light chain alleles that have been confirmed by evidence from multiple high-quality sources. By including only those alleles with a high level of support, including some new sequences that currently lack official names, AIRR-seq analysis will have greater accuracy and studies of the evolution of immunoglobulin genes, their allelic variants and the expressed immune repertoire will be facilitated. Although containing less than half the previously recognised IG alleles (e.g. just 198 IGHV sequences), the Reference Sets eliminated erroneous calls and provided excellent coverage when tested on a set of repertoires from 99 individuals comprising over 4 million V(D)J rearrangements. To improve AIRR-seq analysis, some alleles have been extended to deal with short 3 or 5 truncations that can lead them to be overlooked by alignment utilities. To avoid other challenges for analysis programs, exact paralogs (e.g. IGHV1-69*01 and IGHV1-69D*01) are only represented once in each set, though alternative sequence names are noted in accompanying metadata. The Reference Sets also include novel alleles: 8 IGHV alleles, 2 IGKV alleles and 5 IGLV alleles. The version-tracked AIRR-C Reference Sets are freely available at the OGRDB website (https://ogrdb.airr-community.org/germline_sets/Human) and will be regularly updated to include newly-observed and previously-reported sequences that can be confirmed by new high-quality data.\nGenetics of cell-type-specific post-transcriptional gene regulation during human neurogenesis\nAuthors: Aygün, N.; Krupa, O.; Mory, J.; Le, B.; Valone, J.; Liang, D.; Love, M. I.; Stein, J. L.\nScore: 14.2, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555019\nThe function of some genetic variants associated with brain-relevant traits has been explained through colocalization with expression quantitative trait loci (eQTL) conducted in bulk post-mortem adult brain tissue. However, many brain-trait associated loci have unknown cellular or molecular function. These genetic variants may exert context-specific function on different molecular phenotypes including post-transcriptional changes. Here, we identified genetic regulation of RNA-editing and alternative polyadenylation (APA), within a cell-type-specific population of human neural progenitors and neurons. More RNA-editing and isoforms utilizing longer polyadenylation sequences were observed in neurons, likely due to higher expression of genes encoding the proteins mediating these post-transcriptional events. We also detected hundreds of cell-type-specific editing quantitative trait loci (edQTLs) and alternative polyadenylation QTLs (apaQTLs). We found colocalizations of a neuron edQTL in CCDC88A with educational attainment and a progenitor apaQTL in EP300 with schizophrenia, suggesting genetically mediated post-transcriptional regulation during brain development lead to differences in brain function.\nGenomic Insights into the Population History and Adaptive Traits of Latin American Criollo Cattle\nAuthors: Ward, J. A.; Ng'ang'a, S. I.; Randhawa, I. A. S.; McHugo, G. P.; O'Grady, J. F.; Florez, J. M.; Browne, J. A.; Perez O'Brien, A. M.; Landaeta-Hernandez, A. J.; Garcia, J. F.; Sonstegard, T. S.; Frantz, L. A. F.; Salter-Townshend, M.; MacHugh, D. E.\nScore: 12.2, Published: 2023-09-10 DOI: 10.1101/2023.09.10.556512\nCriollo cattle, descendants of animals brought by Iberian colonists to the Americas, have been subject to centuries of natural and human-mediated selection in novel tropical agroecological zones. Consequently, these breeds have evolved distinct characteristics such as resistance to disease and exceptional heat tolerance. In addition to European taurine (Bos taurus) ancestry, it has been proposed that gene flow from African taurine and Asian zebu (Bos indicus) cattle has shaped the ancestry of Criollo cattle. In this study, we analysed Criollo breeds from Colombia and Venezuela using whole genome sequencing data to examine population structure and admixture at very high resolution. We did not detect evidence of African taurine admixture in Criollo cattle; however, there was evidence of longstanding zebu ancestry in one breed (Harton del Valle). In addition, we detected selection signatures associated with a myriad of adaptive traits, revealing genes linked to thermotolerance, reproduction, fertility, immunity, and distinct coat and skin coloration traits. This study underscores the remarkable adaptability of Criollo cattle and highlights the genetic richness and potential of these breeds in the face of climate change, habitat flux, and disease challenges. Further research is warranted to leverage these findings for more effective and sustainable cattle breeding programmes.\nThe genome and population genomics of allopolyploid Coffea arabica reveal the diversification history of modern coffee cultivars\nAuthors: Salojarvi, J.; Rambani, A.; Yu, Z.; Guyot, R.; Strickler, S.; Lepelley, M.; Wang, C.; Rajaraman, S.; Rastas, P.; Zheng, C.; Munoz, D. S.; Meidanis, J.; Paschoal, A. R.; Bawin, Y.; Krabbenhoft, T.; Wang, Z. Q.; Fleck, S.; Aussel, R.; Bellanger, L.; Charpagne, A.; Fournier, C.; Kassam, M.; Lefebvre, G.; Metairon, S.; Moine, D.; Rigoreau, M.; Stolte, J.; Hamon, P.; Couturon, E.; Tranchant-Dubreuil, C.; Mukherjee, M.; Lan, T.; Engelhardt, J.; Stadler, P.; DeLemos, S. C.; Suzuki, S. I.; Sumirat, U.; ChingMan, W.; Dauchot, N.; Orozco-Arias, S.; Garavito, A.; Kiwuka, C.; Musoli, P.; Nalukenge, A.; Gu\nScore: 11.7, Published: 2023-09-06 DOI: 10.1101/2023.09.06.556570\nCoffea arabica, an allotetraploid hybrid of C. eugenioides and C. canephora, is the source of approximately 60% of coffee products worldwide. Cultivated accessions have undergone several population bottlenecks resulting in low genetic diversity. We present chromosome-level assemblies of a di-haploid C. arabica accession and modern representatives of its diploid progenitors, C. eugenioides and C. canephora. The three species exhibit largely conserved genome structures between diploid parents and descendant subgenomes, which show a mosaic pattern of dominance, similar to other polyploid crop species. Resequencing of 39 wild and cultivated accessions suggests a founding polyploidy event [~]610,000 years ago, followed by several subsequent bottlenecks, including a population split [~]30.5 kya and a period of migration between Arabica populations until [~]8.9 kya. Analysis of lines historically introgressed with C. canephora highlights loci that may contribute to their superior pathogen resistance and lay the groundwork for future genomics-based breeding of C. arabica.\nTelomere-to-telomere Schizosaccharomyces japonicus genome assembly reveals hitherto unknown genome features\nAuthors: Etherington, G. J.; Nieduszynski, C.; Oliferenko, S.; Wu, P.-S.; Uhlmann, F.\nScore: 9.8, Published: 2023-09-06 DOI: 10.1101/2023.09.04.556195\nSchizosaccharomyces japonicus belongs to the single-genus class Schizosaccharomycetes, otherwise known as fission yeasts. As part of a composite model system with its widely studied S. pombe sister species, S. japonicus has provided critical insights into the workings and the evolution of cell biological mechanisms. Furthermore, its divergent biology makes S. japonicus a valuable model organism in its own right. However, the currently available short-read genome assembly contains gaps and has been unable to resolve centromeres and other repeat-rich chromosomal regions. Here we present a telomere-to-telomere long-read genome assembly of the S japonicus genome. This includes the three megabase-length chromosomes, with centromeres hundreds of kilobases long, rich in 5S ribosomal RNAs, transfer RNAs, long terminal repeats, and short repeats. We identify a gene-sparse region on chromosome 2 that resembles a 331 kb centromeric duplication. We revise the genome size of S. japonicus to at least 16.6 Mb and possibly up to 18.12 Mb, at least 30% larger than previous estimates. Our whole genome assembly will support the growing S. japonicus research community and facilitate research in new directions, including centromere and DNA repeat evolution, and yeast comparative genomics. Take-awayO_LIA telomere-to-telomere genome assembly of the fission yeast S. japonicus C_LIO_LIChromosome 2 harbours a previously unknown second centromere-like region C_LIO_LIThe estimated genome size of S. japonicus may be up to 18.12 Mb C_LI\nEstrogen receptor activation remodels TEAD1 gene expression to alleviate nonalcoholic fatty liver disease\nAuthors: Sommerauer, C.; Gallardo-Dodd, C. J.; Savva, C.; Hases, L.; Birgersson, M.; Indukuri, R.; Shen, J. X.; Carravilla, P.; Geng, K.; Norskov Sondergaard, J.; Ferrer-Aumatell, C.; Mercier, G.; Sezgin, E.; Korach-Andre, M.; Petersson, C.; Hagstrom, H.; Lauschke, V. M.; Archer, A.; Williams, C.; Kutter, C.\nScore: 9.7, Published: 2023-09-08 DOI: 10.1101/2023.09.07.556687\nIntroduction: The occurrence of obesity-related hepatic malignancies differs between sexes, suggesting the involvement of sex hormones. Female sex hormones maintain cell homeostasis through estrogen receptor (ER) signaling and protect from developing nonalcoholic fatty liver disease (NAFLD) in mice and humans. Rationale: To understand recovery from high-fat diet (HFD)-induced liver disease in males upon estrogen treatment, we comprehensively characterized molecular changes in the liver upon selective activation of estrogen receptors (ERs) to identify novel therapeutic targets downstream of estrogen signaling. Methods: To dissect hepatic ER isoform-driven responses, we integrated liver transcriptomes from female and male HFD mice treated with or without four different estrogen agonists, along with multi-omics data, including bulk, single-cell and spatial transcriptomics, chromatin profiling, machine learning models and advanced microscopy. Patient cohorts and primary human hepatocyte spheroids datasets were included. Results: Only males developed liver steatosis. We found that selective activation of either ERa or ERb reduced HFD-induced hepatic steatosis in male mice. Systemic ER activation restored HFD-induced aberrant gene expression of cellular processes across liver cell types, including hepatocytes. Profiling of marked histones revealed that ER activation modulated promoter and enhancer sites and identified 68 estrogen-sensitive enhancer-gene pairs. Most of these genes were similarly deregulated in human nonalcoholic fatty liver disease (NAFLD) patients, including the transcription factor TEAD1. TEAD1 expression increased in NAFLD patients, and inhibiting TEAD ameliorated steatosis in spheroids by suppressing lipogenic pathways. Conclusions: Systemic activation of ERa or ERb modulates molecular pathways in the liver to counteract NAFLD. Our study identified TEAD1 as a key ER-sensitive gene and suggested that its inhibition poses a therapeutic strategy to combat NAFLD without the undesired side effects elicited by estrogen signaling. Clinical research relevance: We identified drug targets downstream of estrogen signaling, including TEAD1, and demonstrate that TEAD inhibition improves steatosis by suppressing lipogenic pathways. Basic research relevance: The targeted activation of nuclear ERs recovers high-fat diet-induced molecular and physiological liver phenotypes by remodeling core pathways beyond lipid metabolism. ER-responsive enhancers regulate central metabolic genes of clinical significance in NAFLD patients, highlighting the potential impact of this research on understanding liver cell plasticity.\nPredicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation\nAuthors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.\nScore: 121.0, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582\nSequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence. Using statistics derived from Borzois predicted coverage, we isolate and accurately score variant effects across multiple layers of regulation, including transcription, splicing, and polyadenylation. Evaluated on QTLs, Borzoi is competitive with, and often outperforms, state-of-the-art models trained on individual regulatory functions. By applying attribution methods to the derived statistics, we extract cis-regulatory patterns driving RNA expression and post-transcriptional regulation in normal tissues. The wide availability of RNA-seq data across species, conditions, and assays profiling specific aspects of regulation emphasizes the potential of this approach to decipher the mapping from DNA sequence to regulatory function.\n",
  "wordCount" : "2563",
  "inLanguage": "en",
  "datePublished": "2023-09-13T10:39:03Z",
  "dateModified": "2023-09-13T10:39:03Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on September 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556380">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556380" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556380">
        <p class="paperTitle">Regulatory Transposable Elements in the Encyclopedia of DNA Elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556380" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556380" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Du, A. Y.; Chobirko, J. D.; Zhuo, X.; Feschotte, C.; Wang, T.</p>
        <p class="info">Score: 51.5, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556380' target='https://doi.org/10.1101/2023.09.05.556380'> 10.1101/2023.09.05.556380</a></p>
        <p class="abstract">Transposable elements (TEs) make up about half of the human genome and many have the biochemical hallmarks of tissue- or cell type-specific cis-regulatory elements. While some TEs have been rigorously documented to contribute directly to host gene regulation, we still have a very partial view of their regulatory landscape. Leveraging Phase 4 ENCODE data, we carried out the most comprehensive study to date of TE contributions to the regulatory genome. Here we investigated the sequence origins of candidate cis-regulatory elements (cCREs), showing that [~]25% of human cCREs comprising 236,181 elements are derived from TEs. Human-mouse comparisons indicate that over 90% of TE-derived cCREs are lineage-specific, accounting for 8-36% of lineage-specific cCREs across cCRE types. Next, we found that cCRE-associated transcription factor (TF) binding motifs in TEs originated from TE ancestral sequences significantly more than expected in all TE classes except for SINEs. Using both cCRE and TF binding data, we discovered that TEs providing cCREs and TF binding sites are closer in genomic distance to non-TE sites compared to other TEs, suggesting that TE integration site influences their later co-option as regulatory elements. We show that TEs have promoted TF binding site turnover events since human-mouse divergence, accounting for 3-56% of turnover events across 30 TFs examined. Finally, we demonstrate that TE-derived cCREs share similar features with non-TE cCREs, including massively parallel reporter assay activity and GWAS variant enrichment. Overall, our results substantiate the notion that TEs have played an important role in shaping the human regulatory genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555953">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555953" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555953">
        <p class="paperTitle">Crossing host boundaries: the evolutionary drivers and correlates of viral host jumps</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555953" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555953" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tan, C. C. S.; van Dorp, L.; Balloux, F.</p>
        <p class="info">Score: 23.6, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555953' target='https://doi.org/10.1101/2023.09.01.555953'> 10.1101/2023.09.01.555953</a></p>
        <p class="abstract">Most emerging and re-emerging infectious diseases stem from viruses that naturally circulate in non-human vertebrates. When these viruses cross over into humans, they can cause disease outbreaks and epidemics. While zoonotic host jumps have been extensively studied from an ecological perspective, little attention has gone into characterising the evolutionary drivers and correlates underlying these events. To address this gap, we harnessed the entirety of publicly available viral genomic data, employing a comprehensive suite of network and phylogenetic analyses. We address a series of questions concerning the evolutionary mechanisms underpinning viral host jumps. Notably, we challenge conventional assumptions about the directionality of host jumps, demonstrating that humans are as much a source as a sink for viral spillover events, insofar we infer more viruses to have jumped from humans to other animals, than from animals to humans. Moreover, we demonstrate heightened evolution in viral lineages that involve putative host jumps. We further observe that the mutational threshold associated with a host jump is lower for viruses with broad host ranges. Finally, we show that the genomic targets of natural selection upon a successful host jump vary across different viral families with either structural or auxiliary genes being the prime targets of selection. Collectively, our results illuminate some of the evolutionary drivers underlying viral host jumps that may contribute to mitigating viral threats across species boundaries.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555983">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555983" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555983">
        <p class="paperTitle">A General Approach to Adjusting Genetic Studies for Assortative Mating</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555983" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555983" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bilghese, M.; Manansala, R.; Jaishankar, D.; Jala, J.; Benjamin, D. J.; Kimball, M.; Auer, P. L.; Livermore, M. A.; Turley, P.</p>
        <p class="info">Score: 20.2, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555983' target='https://doi.org/10.1101/2023.09.01.555983'> 10.1101/2023.09.01.555983</a></p>
        <p class="abstract">The effects of assortative mating (AM) on estimates from genetic studies has been receiving increasing attention in recent years. We extend existing AM theory to more general models of sorting and conclude that correct theory-based AM adjustments require knowledge of complicated, unknown historical sorting patterns. We propose a simple, general-purpose approach using polygenic indexes (PGIs). Our approach can estimate the fraction of genetic variance and genetic correlation that is driven by AM. Our approach is less effective when applied to Mendelian randomization (MR) studies for two reasons: AM can induce a form of selection bias in MR studies that remains after our adjustment; and, in the MR context, the adjustment is particularly sensitive to PGI estimation error. Using data from the UK Biobank, we find that AM inflates genetic correlation estimates between health traits and education by 14% on average. Our results suggest caution in interpreting genetic correlations or MR estimates for traits subject to AM.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555348">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555348" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555348">
        <p class="paperTitle">AIRR-C Human IG Reference Sets: curated sets of immunoglobulin heavy and light chain germline genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555348" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555348" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Collins, A. M.; Ohlin, M.; Corcoran, M.; Heather, J. M.; Ralph, D.; Law, M.; Martinez-Barnetche, J.; Ye, J.; Richardson, E.; Gibson, W. S.; Rodriguez, O. L.; Peres, A.; Yaari, G.; Watson, C. T.; Lees, W. D.</p>
        <p class="info">Score: 21.1, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555348' target='https://doi.org/10.1101/2023.09.01.555348'> 10.1101/2023.09.01.555348</a></p>
        <p class="abstract">Analysis of an individuals immunoglobulin (IG) gene repertoire requires the use of high-quality germline gene Reference Sets. The Adaptive Immune Receptor Repertoire-Community (AIRR-C) Reference Sets have been developed to include only human IG heavy and light chain alleles that have been confirmed by evidence from multiple high-quality sources. By including only those alleles with a high level of support, including some new sequences that currently lack official names, AIRR-seq analysis will have greater accuracy and studies of the evolution of immunoglobulin genes, their allelic variants and the expressed immune repertoire will be facilitated. Although containing less than half the previously recognised IG alleles (e.g. just 198 IGHV sequences), the Reference Sets eliminated erroneous calls and provided excellent coverage when tested on a set of repertoires from 99 individuals comprising over 4 million V(D)J rearrangements. To improve AIRR-seq analysis, some alleles have been extended to deal with short 3 or 5 truncations that can lead them to be overlooked by alignment utilities. To avoid other challenges for analysis programs, exact paralogs (e.g. IGHV1-69*01 and IGHV1-69D*01) are only represented once in each set, though alternative sequence names are noted in accompanying metadata. The Reference Sets also include novel alleles: 8 IGHV alleles, 2 IGKV alleles and 5 IGLV alleles. The version-tracked AIRR-C Reference Sets are freely available at the OGRDB website (https://ogrdb.airr-community.org/germline_sets/Human) and will be regularly updated to include newly-observed and previously-reported sequences that can be confirmed by new high-quality data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555019">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555019" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555019">
        <p class="paperTitle">Genetics of cell-type-specific post-transcriptional gene regulation during human neurogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555019" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555019" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aygün, N.; Krupa, O.; Mory, J.; Le, B.; Valone, J.; Liang, D.; Love, M. I.; Stein, J. L.</p>
        <p class="info">Score: 14.2, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555019' target='https://doi.org/10.1101/2023.08.30.555019'> 10.1101/2023.08.30.555019</a></p>
        <p class="abstract">The function of some genetic variants associated with brain-relevant traits has been explained through colocalization with expression quantitative trait loci (eQTL) conducted in bulk post-mortem adult brain tissue. However, many brain-trait associated loci have unknown cellular or molecular function. These genetic variants may exert context-specific function on different molecular phenotypes including post-transcriptional changes. Here, we identified genetic regulation of RNA-editing and alternative polyadenylation (APA), within a cell-type-specific population of human neural progenitors and neurons. More RNA-editing and isoforms utilizing longer polyadenylation sequences were observed in neurons, likely due to higher expression of genes encoding the proteins mediating these post-transcriptional events. We also detected hundreds of cell-type-specific editing quantitative trait loci (edQTLs) and alternative polyadenylation QTLs (apaQTLs). We found colocalizations of a neuron edQTL in CCDC88A with educational attainment and a progenitor apaQTL in EP300 with schizophrenia, suggesting genetically mediated post-transcriptional regulation during brain development lead to differences in brain function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.10.556512">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.10.556512" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.10.556512">
        <p class="paperTitle">Genomic Insights into the Population History and Adaptive Traits of Latin American Criollo Cattle</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.10.556512" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.10.556512" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ward, J. A.; Ng&#39;ang&#39;a, S. I.; Randhawa, I. A. S.; McHugo, G. P.; O&#39;Grady, J. F.; Florez, J. M.; Browne, J. A.; Perez O&#39;Brien, A. M.; Landaeta-Hernandez, A. J.; Garcia, J. F.; Sonstegard, T. S.; Frantz, L. A. F.; Salter-Townshend, M.; MacHugh, D. E.</p>
        <p class="info">Score: 12.2, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.10.556512' target='https://doi.org/10.1101/2023.09.10.556512'> 10.1101/2023.09.10.556512</a></p>
        <p class="abstract">Criollo cattle, descendants of animals brought by Iberian colonists to the Americas, have been subject to centuries of natural and human-mediated selection in novel tropical agroecological zones. Consequently, these breeds have evolved distinct characteristics such as resistance to disease and exceptional heat tolerance. In addition to European taurine (Bos taurus) ancestry, it has been proposed that gene flow from African taurine and Asian zebu (Bos indicus) cattle has shaped the ancestry of Criollo cattle. In this study, we analysed Criollo breeds from Colombia and Venezuela using whole genome sequencing data to examine population structure and admixture at very high resolution. We did not detect evidence of African taurine admixture in Criollo cattle; however, there was evidence of longstanding zebu ancestry in one breed (Harton del Valle). In addition, we detected selection signatures associated with a myriad of adaptive traits, revealing genes linked to thermotolerance, reproduction, fertility, immunity, and distinct coat and skin coloration traits. This study underscores the remarkable adaptability of Criollo cattle and highlights the genetic richness and potential of these breeds in the face of climate change, habitat flux, and disease challenges. Further research is warranted to leverage these findings for more effective and sustainable cattle breeding programmes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556570" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556570">
        <p class="paperTitle">The genome and population genomics of allopolyploid Coffea arabica reveal the diversification history of modern coffee cultivars</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Salojarvi, J.; Rambani, A.; Yu, Z.; Guyot, R.; Strickler, S.; Lepelley, M.; Wang, C.; Rajaraman, S.; Rastas, P.; Zheng, C.; Munoz, D. S.; Meidanis, J.; Paschoal, A. R.; Bawin, Y.; Krabbenhoft, T.; Wang, Z. Q.; Fleck, S.; Aussel, R.; Bellanger, L.; Charpagne, A.; Fournier, C.; Kassam, M.; Lefebvre, G.; Metairon, S.; Moine, D.; Rigoreau, M.; Stolte, J.; Hamon, P.; Couturon, E.; Tranchant-Dubreuil, C.; Mukherjee, M.; Lan, T.; Engelhardt, J.; Stadler, P.; DeLemos, S. C.; Suzuki, S. I.; Sumirat, U.; ChingMan, W.; Dauchot, N.; Orozco-Arias, S.; Garavito, A.; Kiwuka, C.; Musoli, P.; Nalukenge, A.; Gu</p>
        <p class="info">Score: 11.7, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556570' target='https://doi.org/10.1101/2023.09.06.556570'> 10.1101/2023.09.06.556570</a></p>
        <p class="abstract">Coffea arabica, an allotetraploid hybrid of C. eugenioides and C. canephora, is the source of approximately 60% of coffee products worldwide. Cultivated accessions have undergone several population bottlenecks resulting in low genetic diversity. We present chromosome-level assemblies of a di-haploid C. arabica accession and modern representatives of its diploid progenitors, C. eugenioides and C. canephora. The three species exhibit largely conserved genome structures between diploid parents and descendant subgenomes, which show a mosaic pattern of dominance, similar to other polyploid crop species. Resequencing of 39 wild and cultivated accessions suggests a founding polyploidy event [~]610,000 years ago, followed by several subsequent bottlenecks, including a population split [~]30.5 kya and a period of migration between Arabica populations until [~]8.9 kya. Analysis of lines historically introgressed with C. canephora highlights loci that may contribute to their superior pathogen resistance and lay the groundwork for future genomics-based breeding of C. arabica.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556195">
        <p class="paperTitle">Telomere-to-telomere Schizosaccharomyces japonicus genome assembly reveals hitherto unknown genome features</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Etherington, G. J.; Nieduszynski, C.; Oliferenko, S.; Wu, P.-S.; Uhlmann, F.</p>
        <p class="info">Score: 9.8, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556195' target='https://doi.org/10.1101/2023.09.04.556195'> 10.1101/2023.09.04.556195</a></p>
        <p class="abstract">Schizosaccharomyces japonicus belongs to the single-genus class Schizosaccharomycetes, otherwise known as  fission yeasts. As part of a composite model system with its widely studied S. pombe sister species, S. japonicus has provided critical insights into the workings and the evolution of cell biological mechanisms. Furthermore, its divergent biology makes S. japonicus a valuable model organism in its own right. However, the currently available short-read genome assembly contains gaps and has been unable to resolve centromeres and other repeat-rich chromosomal regions. Here we present a telomere-to-telomere long-read genome assembly of the S japonicus genome. This includes the three megabase-length chromosomes, with centromeres hundreds of kilobases long, rich in 5S ribosomal RNAs, transfer RNAs, long terminal repeats, and short repeats. We identify a gene-sparse region on chromosome 2 that resembles a 331 kb centromeric duplication. We revise the genome size of S. japonicus to at least 16.6 Mb and possibly up to 18.12 Mb, at least 30% larger than previous estimates. Our whole genome assembly will support the growing S. japonicus research community and facilitate research in new directions, including centromere and DNA repeat evolution, and yeast comparative genomics.

Take-awayO_LIA telomere-to-telomere genome assembly of the fission yeast S. japonicus
C_LIO_LIChromosome 2 harbours a previously unknown second centromere-like region
C_LIO_LIThe estimated genome size of S. japonicus may be up to 18.12 Mb
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.07.556687">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.07.556687" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.07.556687">
        <p class="paperTitle">Estrogen receptor activation remodels TEAD1 gene expression to alleviate nonalcoholic fatty liver disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.07.556687" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.07.556687" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sommerauer, C.; Gallardo-Dodd, C. J.; Savva, C.; Hases, L.; Birgersson, M.; Indukuri, R.; Shen, J. X.; Carravilla, P.; Geng, K.; Norskov Sondergaard, J.; Ferrer-Aumatell, C.; Mercier, G.; Sezgin, E.; Korach-Andre, M.; Petersson, C.; Hagstrom, H.; Lauschke, V. M.; Archer, A.; Williams, C.; Kutter, C.</p>
        <p class="info">Score: 9.7, Published: 2023-09-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.07.556687' target='https://doi.org/10.1101/2023.09.07.556687'> 10.1101/2023.09.07.556687</a></p>
        <p class="abstract">Introduction: The occurrence of obesity-related hepatic malignancies differs between sexes, suggesting the involvement of sex hormones. Female sex hormones maintain cell homeostasis through estrogen receptor (ER) signaling and protect from developing nonalcoholic fatty liver disease (NAFLD) in mice and humans. Rationale: To understand recovery from high-fat diet (HFD)-induced liver disease in males upon estrogen treatment, we comprehensively characterized molecular changes in the liver upon selective activation of estrogen receptors (ERs) to identify novel therapeutic targets downstream of estrogen signaling. Methods: To dissect hepatic ER isoform-driven responses, we integrated liver transcriptomes from female and male HFD mice treated with or without four different estrogen agonists, along with multi-omics data, including bulk, single-cell and spatial transcriptomics, chromatin profiling, machine learning models and advanced microscopy. Patient cohorts and primary human hepatocyte spheroids datasets were included. Results: Only males developed liver steatosis. We found that selective activation of either ERa or ERb reduced HFD-induced hepatic steatosis in male mice. Systemic ER activation restored HFD-induced aberrant gene expression of cellular processes across liver cell types, including hepatocytes. Profiling of marked histones revealed that ER activation modulated promoter and enhancer sites and identified 68 estrogen-sensitive enhancer-gene pairs. Most of these genes were similarly deregulated in human nonalcoholic fatty liver disease (NAFLD) patients, including the transcription factor TEAD1. TEAD1 expression increased in NAFLD patients, and inhibiting TEAD ameliorated steatosis in spheroids by suppressing lipogenic pathways. Conclusions: Systemic activation of ERa or ERb modulates molecular pathways in the liver to counteract NAFLD. Our study identified TEAD1 as a key ER-sensitive gene and suggested that its inhibition poses a therapeutic strategy to combat NAFLD without the undesired side effects elicited by estrogen signaling. Clinical research relevance: We identified drug targets downstream of estrogen signaling, including TEAD1, and demonstrate that TEAD inhibition improves steatosis by suppressing lipogenic pathways. Basic research relevance: The targeted activation of nuclear ERs recovers high-fat diet-induced molecular and physiological liver phenotypes by remodeling core pathways beyond lipid metabolism. ER-responsive enhancers regulate central metabolic genes of clinical significance in NAFLD patients, highlighting the potential impact of this research on understanding liver cell plasticity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555582">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555582" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555582">
        <p class="paperTitle">Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555582" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555582" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.</p>
        <p class="info">Score: 121.0, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555582' target='https://doi.org/10.1101/2023.08.30.555582'> 10.1101/2023.08.30.555582</a></p>
        <p class="abstract">Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence. Using statistics derived from Borzois predicted coverage, we isolate and accurately score variant effects across multiple layers of regulation, including transcription, splicing, and polyadenylation. Evaluated on QTLs, Borzoi is competitive with, and often outperforms, state-of-the-art models trained on individual regulatory functions. By applying attribution methods to the derived statistics, we extract cis-regulatory patterns driving RNA expression and post-transcriptional regulation in normal tissues. The wide availability of RNA-seq data across species, conditions, and assays profiling specific aspects of regulation emphasizes the potential of this approach to decipher the mapping from DNA sequence to regulatory function.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
