<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Regulatory Transposable Elements in the Encyclopedia of DNA Elements
Authors: Du, A. Y.; Chobirko, J. D.; Zhuo, X.; Feschotte, C.; Wang, T.
Score: 54.4, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556380
Transposable elements (TEs) make up about half of the human genome and many have the biochemical hallmarks of tissue- or cell type-specific cis-regulatory elements. While some TEs have been rigorously documented to contribute directly to host gene regulation, we still have a very partial view of their regulatory landscape.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Regulatory Transposable Elements in the Encyclopedia of DNA Elements
Authors: Du, A. Y.; Chobirko, J. D.; Zhuo, X.; Feschotte, C.; Wang, T.
Score: 54.4, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556380
Transposable elements (TEs) make up about half of the human genome and many have the biochemical hallmarks of tissue- or cell type-specific cis-regulatory elements. While some TEs have been rigorously documented to contribute directly to host gene regulation, we still have a very partial view of their regulatory landscape." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-17T10:36:50+00:00" />
<meta property="article:modified_time" content="2023-09-17T10:36:50+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Regulatory Transposable Elements in the Encyclopedia of DNA Elements
Authors: Du, A. Y.; Chobirko, J. D.; Zhuo, X.; Feschotte, C.; Wang, T.
Score: 54.4, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556380
Transposable elements (TEs) make up about half of the human genome and many have the biochemical hallmarks of tissue- or cell type-specific cis-regulatory elements. While some TEs have been rigorously documented to contribute directly to host gene regulation, we still have a very partial view of their regulatory landscape."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Regulatory Transposable Elements in the Encyclopedia of DNA Elements\nAuthors: Du, A. Y.; Chobirko, J. D.; Zhuo, X.; Feschotte, C.; Wang, T.\nScore: 54.4, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556380\nTransposable elements (TEs) make up about half of the human genome and many have the biochemical hallmarks of tissue- or cell type-specific cis-regulatory elements. While some TEs have been rigorously documented to contribute directly to host gene regulation, we still have a very partial view of their regulatory landscape.",
  "keywords": [
    
  ],
  "articleBody": " Regulatory Transposable Elements in the Encyclopedia of DNA Elements\nAuthors: Du, A. Y.; Chobirko, J. D.; Zhuo, X.; Feschotte, C.; Wang, T.\nScore: 54.4, Published: 2023-09-06 DOI: 10.1101/2023.09.05.556380\nTransposable elements (TEs) make up about half of the human genome and many have the biochemical hallmarks of tissue- or cell type-specific cis-regulatory elements. While some TEs have been rigorously documented to contribute directly to host gene regulation, we still have a very partial view of their regulatory landscape. Leveraging Phase 4 ENCODE data, we carried out the most comprehensive study to date of TE contributions to the regulatory genome. Here we investigated the sequence origins of candidate cis-regulatory elements (cCREs), showing that [~]25% of human cCREs comprising 236,181 elements are derived from TEs. Human-mouse comparisons indicate that over 90% of TE-derived cCREs are lineage-specific, accounting for 8-36% of lineage-specific cCREs across cCRE types. Next, we found that cCRE-associated transcription factor (TF) binding motifs in TEs originated from TE ancestral sequences significantly more than expected in all TE classes except for SINEs. Using both cCRE and TF binding data, we discovered that TEs providing cCREs and TF binding sites are closer in genomic distance to non-TE sites compared to other TEs, suggesting that TE integration site influences their later co-option as regulatory elements. We show that TEs have promoted TF binding site turnover events since human-mouse divergence, accounting for 3-56% of turnover events across 30 TFs examined. Finally, we demonstrate that TE-derived cCREs share similar features with non-TE cCREs, including massively parallel reporter assay activity and GWAS variant enrichment. Overall, our results substantiate the notion that TEs have played an important role in shaping the human regulatory genome.\nGenetic tracing of market wildlife and viruses at the epicenter of the COVID-19 pandemic\nAuthors: Crits-Christoph, A.; Levy, J. I.; Pekar, J.; Goldstein, S. A.; Singh, R.; Hensel, Z.; Gangavarapu, K.; Rogers, M. B.; Moshiri, N.; Garry, R. f.; Holmes, E. C.; Koopmans, M. P. G.; Lemey, P.; Popescu, S.; Rambaut, A.; Robertson, D. L.; Suchard, M. A.; Wertheim, J.; Rasmussen, A. L.; Andersen, K. G.; Worobey, M.; Debarre, F.\nScore: 44.4, Published: 2023-09-14 DOI: 10.1101/2023.09.13.557637\nZoonotic spillovers of viruses have occurred through the animal trade worldwide. The start of the COVID-19 pandemic was traced epidemiologically to the Huanan Wholesale Seafood Market, the site with the most reported wildlife vendors in the city of Wuhan, China. Here, we analyze publicly available qPCR and sequencing data from environmental samples collected in the Huanan market in early 2020. We demonstrate that the SARS-CoV-2 genetic diversity linked to this market is consistent with market emergence, and find increased SARS-CoV-2 positivity near and within a particular wildlife stall. We identify wildlife DNA in all SARS-CoV-2 positive samples from this stall. This includes species such as civets, bamboo rats, porcupines, hedgehogs, and one species, raccoon dogs, known to be capable of SARS-CoV-2 transmission. We also detect other animal viruses that infect raccoon dogs, civets, and bamboo rats. Combining metagenomic and phylogenetic approaches, we recover genotypes of market animals and compare them to those from other markets. This analysis provides the genetic basis for a short list of potential intermediate hosts of SARS-CoV-2 to prioritize for retrospective serological testing and viral sampling.\nRegulation of interferon signaling by transposon exonization\nAuthors: Pasquesi, G. I. M.; Allen, H.; Ivancevic, A.; Barbachano-Guerrero, A.; Joyner, O.; Guo, K.; Simpson, D. M.; Gapin, K.; Horton, I.; Nguyen, L.; Yang, Q.; Warren, C. J.; Florea, L. D.; Bitler, B. G.; Santiago, M. L.; Sawyer, S. L.; Chuong, E. B.\nScore: 33.5, Published: 2023-09-15 DOI: 10.1101/2023.09.11.557241\nInnate immune signaling is essential for clearing pathogens and damaged cells, and must be tightly regulated to avoid excessive inflammation or autoimmunity. Here, we found that the alternative splicing of exons derived from transposable elements is a key mechanism controlling immune signaling in human cells. By analyzing long-read transcriptome datasets, we identified numerous transposon exonization events predicted to generate functional protein variants of immune genes, including the type I interferon receptor IFNAR2. We demonstrated that the transposon-derived isoform of IFNAR2 is more highly expressed than the canonical isoform in almost all tissues, and functions as a decoy receptor that potently inhibits interferon signaling including in cells infected with SARS-CoV-2. Our findings uncover a primate-specific axis controlling interferon signaling and show how a transposon exonization event can be co-opted for immune regulation.\nCas13d-mediated isoform-specific RNA knockdown with a unified computational and experimental toolbox\nAuthors: Schertzer, M. D.; Stirn, A.; Isaev, K.; Pereira, L.; Das, A.; Harbison, C.; Park, S. H.; Wessels, H.-H.; Sanjana, N. E.; Knowles, D. A.\nScore: 20.6, Published: 2023-09-13 DOI: 10.1101/2023.09.12.557474\nAlternative splicing is an essential mechanism for diversifying proteins, in which mature RNA isoforms produce proteins with potentially distinct functions. Two major challenges in characterizing the cellular function of isoforms are the lack of experimental methods to specifically and efficiently modulate isoform expression and computational tools for complex experimental design. To address these gaps, we developed and methodically tested a strategy which pairs the RNA-targeting CRISPR/Cas13d system with guide RNAs that span exon-exon junctions in the mature RNA. We performed a high-throughput essentiality screen, quantitative RT-PCR assays, and PacBio long read sequencing to affirm our ability to specifically target and robustly knockdown individual RNA isoforms. In parallel, we provide computational tools for experimental design and screen analysis. Considering all possible splice junctions annotated in GENCODE for multi-isoform genes and our gRNA efficacy predictions, we estimate that our junction-centric strategy can uniquely target up to 89% of human RNA isoforms, including 50,066 protein-coding and 11,415 lncRNA isoforms. Importantly, this specificity spans all splicing and transcriptional events, including exon skipping and inclusion, alternative 5 and 3 splice sites, and alternative starts and ends.\nThe genome of the Wollemi pine, a critically endangered living fossil unchanged since the Cretaceous, reveals extensive ancient transposon activity.\nAuthors: Stevenson, D. W.; Ramakrishnan, S.; Alves, C. d. S.; Coelho, L. A.; Kramer, M.; Goodwin, S.; Ramos, O. M.; Eshel, G.; Sondervan, V. M.; Frangos, S.; Zumajo-Cardona, C.; Jenike, K.; Ou, S.; Wang, X.; Lee, Y. P.; Loke, S.; Rossetto, M.; McPherson, H.; Nigris, S.; Moschin, S.; Little, D. P.; Katari, M. S.; Varala, K.; Kolokotronis, S.-O.; Ambrose, B.; Croft, L. J.; Coruzzi, G. M.; Schatz, M.; McCombie, W. R.; Martienssen, R. A.\nScore: 26.2, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554647\nWe present the genome of the living fossil, Wollemia nobilis, a southern hemisphere conifer morphologically unchanged since the Cretaceous. Presumed extinct until rediscovery in 1994, the Wollemi pine is critically endangered with less than 60 wild adults threatened by intensifying bushfires in the Blue Mountains of Australia. The 12 Gb genome is among the most contiguous large plant genomes assembled, with extremely low heterozygosity and unusual abundance of DNA transposons. Reduced representation and genome re-sequencing of individuals confirms a relictual population since the last major glacial/drying period in Australia, 120 ky BP. Small RNA and methylome sequencing reveal conservation of ancient silencing mechanisms despite the presence of thousands of active and abundant transposons, including some transferred horizontally to conifers from arthropods in the Jurassic. A retrotransposon burst 8-6 my BP coincided with population decline, possibly as an adaptation enhancing epigenetic diversity. Wollemia, like other conifers, is susceptible to Phytophthora, and a suite of defense genes, similar to those in loblolly pine, are targeted for silencing by sRNAs in leaves. The genome provides insight into the earliest seed plants, while enabling conservation efforts.\nGenomic Insights into the Population History and Adaptive Traits of Latin American Criollo Cattle\nAuthors: Ward, J. A.; Ng'ang'a, S. I.; Randhawa, I. A. S.; McHugo, G. P.; O'Grady, J. F.; Florez, J. M.; Browne, J. A.; Perez O'Brien, A. M.; Landaeta-Hernandez, A. J.; Garcia, J. F.; Sonstegard, T. S.; Frantz, L. A. F.; Salter-Townshend, M.; MacHugh, D. E.\nScore: 11.3, Published: 2023-09-10 DOI: 10.1101/2023.09.10.556512\nCriollo cattle, descendants of animals brought by Iberian colonists to the Americas, have been subject to centuries of natural and human-mediated selection in novel tropical agroecological zones. Consequently, these breeds have evolved distinct characteristics such as resistance to disease and exceptional heat tolerance. In addition to European taurine (Bos taurus) ancestry, it has been proposed that gene flow from African taurine and Asian zebu (Bos indicus) cattle has shaped the ancestry of Criollo cattle. In this study, we analysed Criollo breeds from Colombia and Venezuela using whole genome sequencing data to examine population structure and admixture at very high resolution. We did not detect evidence of African taurine admixture in Criollo cattle; however, there was evidence of longstanding zebu ancestry in one breed (Harton del Valle). In addition, we detected selection signatures associated with a myriad of adaptive traits, revealing genes linked to thermotolerance, reproduction, fertility, immunity, and distinct coat and skin coloration traits. This study underscores the remarkable adaptability of Criollo cattle and highlights the genetic richness and potential of these breeds in the face of climate change, habitat flux, and disease challenges. Further research is warranted to leverage these findings for more effective and sustainable cattle breeding programmes.\nPredicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation\nAuthors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.\nScore: 124.5, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582\nSequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence. Using statistics derived from Borzois predicted coverage, we isolate and accurately score variant effects across multiple layers of regulation, including transcription, splicing, and polyadenylation. Evaluated on QTLs, Borzoi is competitive with, and often outperforms, state-of-the-art models trained on individual regulatory functions. By applying attribution methods to the derived statistics, we extract cis-regulatory patterns driving RNA expression and post-transcriptional regulation in normal tissues. The wide availability of RNA-seq data across species, conditions, and assays profiling specific aspects of regulation emphasizes the potential of this approach to decipher the mapping from DNA sequence to regulatory function.\nRetracing Human Genetic Histories and Natural Selection Using Precise Local Ancestry Inference\nAuthors: Lerga-Jaso, J.; Novkovic, B.; Unnikrishnan, D.; Bamunusinghe, V.; Hatorangan, M.; Manson, C.; Pedersen, H.; Osama, A.; Terpolovsky, A.; Bohn, S.; De Marino, A.; Mahmoud, A.; Bircan, K.; Khan, U.; Grabherr, M.; Yazdi, P.\nScore: 5.1, Published: 2023-09-13 DOI: 10.1101/2023.09.11.557177\nIn an increasingly diverse world, including admixed individuals in genomic studies is imperative for equity and portability. A crucial first step is precise local ancestry inference (LAI). We have developed Orchestra, a LAI model with unprecedented accuracy, and trained on over 10,000 single-origin individuals from 35 worldwide populations. We employed Orchestra to delve into genetic relationships and demographic histories, with a focus on Latin Americans, a prime example of admixture, and the Ashkenazi Jewish, whose origins have long been debated. Finally, Orchestra enabled us to map signatures of selection, notably identifying trace Scandinavian ancestry in British samples and unveiling an immune-rich region linked to respiratory infections. Our work advances the field of LAI and holds promise for improvements in future applications for admixed populations. One-Sentence SummaryOrchestra unveils Latino and Ashkenazi ancestral roots and a candidate Viking locus under selection in the British population\nA General Approach to Adjusting Genetic Studies for Assortative Mating\nAuthors: Bilghese, M.; Manansala, R.; Jaishankar, D.; Jala, J.; Benjamin, D. J.; Kimball, M.; Auer, P. L.; Livermore, M. A.; Turley, P.\nScore: 23.9, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555983\nThe effects of assortative mating (AM) on estimates from genetic studies has been receiving increasing attention in recent years. We extend existing AM theory to more general models of sorting and conclude that correct theory-based AM adjustments require knowledge of complicated, unknown historical sorting patterns. We propose a simple, general-purpose approach using polygenic indexes (PGIs). Our approach can estimate the fraction of genetic variance and genetic correlation that is driven by AM. Our approach is less effective when applied to Mendelian randomization (MR) studies for two reasons: AM can induce a form of selection bias in MR studies that remains after our adjustment; and, in the MR context, the adjustment is particularly sensitive to PGI estimation error. Using data from the UK Biobank, we find that AM inflates genetic correlation estimates between health traits and education by 14% on average. Our results suggest caution in interpreting genetic correlations or MR estimates for traits subject to AM.\nMycotools: An Automated and Scalable Platform for Comparative Genomics\nAuthors: Konkel, Z.; Slot, J.\nScore: 5.0, Published: 2023-09-12 DOI: 10.1101/2023.09.08.556886\nComparative genomics comprises analyses that investigate the genetic basis of organismal biology and ecology, which have also been applied to high throughput trait screening for applied purposes. The number of fungal genomes deposited in publicly available databases are currently in exponential growth. Due to the limited cutting-edge software availability and size or efficiency constraints of web-based analyses, comparative genomics research is often conducted on local computing environments. There is thus a need for an efficient standardized framework for locally assimilating, curating, and interfacing with genomic data. We present Mycotools as a comparative genomics database software suite that automatically curates, updates, and standardizes local comparative genomics. Mycotools incorporates novel analysis pipelines that are built on a suite of modules that streamline routine-to-complex comparative genomic tasks. The Mycotools software suite serves as a foundation for accessible and reproducible large-scale comparative genomics on local compute systems.\n",
  "wordCount" : "2213",
  "inLanguage": "en",
  "datePublished": "2023-09-17T10:36:50Z",
  "dateModified": "2023-09-17T10:36:50Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on September 17, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.05.556380">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.05.556380" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.05.556380">
        <p class="paperTitle">Regulatory Transposable Elements in the Encyclopedia of DNA Elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.05.556380" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.05.556380" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Du, A. Y.; Chobirko, J. D.; Zhuo, X.; Feschotte, C.; Wang, T.</p>
        <p class="info">Score: 54.4, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.05.556380' target='https://doi.org/10.1101/2023.09.05.556380'> 10.1101/2023.09.05.556380</a></p>
        <p class="abstract">Transposable elements (TEs) make up about half of the human genome and many have the biochemical hallmarks of tissue- or cell type-specific cis-regulatory elements. While some TEs have been rigorously documented to contribute directly to host gene regulation, we still have a very partial view of their regulatory landscape. Leveraging Phase 4 ENCODE data, we carried out the most comprehensive study to date of TE contributions to the regulatory genome. Here we investigated the sequence origins of candidate cis-regulatory elements (cCREs), showing that [~]25% of human cCREs comprising 236,181 elements are derived from TEs. Human-mouse comparisons indicate that over 90% of TE-derived cCREs are lineage-specific, accounting for 8-36% of lineage-specific cCREs across cCRE types. Next, we found that cCRE-associated transcription factor (TF) binding motifs in TEs originated from TE ancestral sequences significantly more than expected in all TE classes except for SINEs. Using both cCRE and TF binding data, we discovered that TEs providing cCREs and TF binding sites are closer in genomic distance to non-TE sites compared to other TEs, suggesting that TE integration site influences their later co-option as regulatory elements. We show that TEs have promoted TF binding site turnover events since human-mouse divergence, accounting for 3-56% of turnover events across 30 TFs examined. Finally, we demonstrate that TE-derived cCREs share similar features with non-TE cCREs, including massively parallel reporter assay activity and GWAS variant enrichment. Overall, our results substantiate the notion that TEs have played an important role in shaping the human regulatory genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557637">
        <p class="paperTitle">Genetic tracing of market wildlife and viruses at the epicenter of the COVID-19 pandemic</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Crits-Christoph, A.; Levy, J. I.; Pekar, J.; Goldstein, S. A.; Singh, R.; Hensel, Z.; Gangavarapu, K.; Rogers, M. B.; Moshiri, N.; Garry, R. f.; Holmes, E. C.; Koopmans, M. P. G.; Lemey, P.; Popescu, S.; Rambaut, A.; Robertson, D. L.; Suchard, M. A.; Wertheim, J.; Rasmussen, A. L.; Andersen, K. G.; Worobey, M.; Debarre, F.</p>
        <p class="info">Score: 44.4, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557637' target='https://doi.org/10.1101/2023.09.13.557637'> 10.1101/2023.09.13.557637</a></p>
        <p class="abstract">Zoonotic spillovers of viruses have occurred through the animal trade worldwide. The start of the COVID-19 pandemic was traced epidemiologically to the Huanan Wholesale Seafood Market, the site with the most reported wildlife vendors in the city of Wuhan, China. Here, we analyze publicly available qPCR and sequencing data from environmental samples collected in the Huanan market in early 2020. We demonstrate that the SARS-CoV-2 genetic diversity linked to this market is consistent with market emergence, and find increased SARS-CoV-2 positivity near and within a particular wildlife stall. We identify wildlife DNA in all SARS-CoV-2 positive samples from this stall. This includes species such as civets, bamboo rats, porcupines, hedgehogs, and one species, raccoon dogs, known to be capable of SARS-CoV-2 transmission. We also detect other animal viruses that infect raccoon dogs, civets, and bamboo rats. Combining metagenomic and phylogenetic approaches, we recover genotypes of market animals and compare them to those from other markets. This analysis provides the genetic basis for a short list of potential intermediate hosts of SARS-CoV-2 to prioritize for retrospective serological testing and viral sampling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557241">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557241" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557241">
        <p class="paperTitle">Regulation of interferon signaling by transposon exonization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557241" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557241" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pasquesi, G. I. M.; Allen, H.; Ivancevic, A.; Barbachano-Guerrero, A.; Joyner, O.; Guo, K.; Simpson, D. M.; Gapin, K.; Horton, I.; Nguyen, L.; Yang, Q.; Warren, C. J.; Florea, L. D.; Bitler, B. G.; Santiago, M. L.; Sawyer, S. L.; Chuong, E. B.</p>
        <p class="info">Score: 33.5, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557241' target='https://doi.org/10.1101/2023.09.11.557241'> 10.1101/2023.09.11.557241</a></p>
        <p class="abstract">Innate immune signaling is essential for clearing pathogens and damaged cells, and must be tightly regulated to avoid excessive inflammation or autoimmunity. Here, we found that the alternative splicing of exons derived from transposable elements is a key mechanism controlling immune signaling in human cells. By analyzing long-read transcriptome datasets, we identified numerous transposon exonization events predicted to generate functional protein variants of immune genes, including the type I interferon receptor IFNAR2. We demonstrated that the transposon-derived isoform of IFNAR2 is more highly expressed than the canonical isoform in almost all tissues, and functions as a decoy receptor that potently inhibits interferon signaling including in cells infected with SARS-CoV-2. Our findings uncover a primate-specific axis controlling interferon signaling and show how a transposon exonization event can be co-opted for immune regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557474">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557474" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557474">
        <p class="paperTitle">Cas13d-mediated isoform-specific RNA knockdown with a unified computational and experimental toolbox</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557474" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557474" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schertzer, M. D.; Stirn, A.; Isaev, K.; Pereira, L.; Das, A.; Harbison, C.; Park, S. H.; Wessels, H.-H.; Sanjana, N. E.; Knowles, D. A.</p>
        <p class="info">Score: 20.6, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557474' target='https://doi.org/10.1101/2023.09.12.557474'> 10.1101/2023.09.12.557474</a></p>
        <p class="abstract">Alternative splicing is an essential mechanism for diversifying proteins, in which mature RNA isoforms produce proteins with potentially distinct functions. Two major challenges in characterizing the cellular function of isoforms are the lack of experimental methods to specifically and efficiently modulate isoform expression and computational tools for complex experimental design. To address these gaps, we developed and methodically tested a strategy which pairs the RNA-targeting CRISPR/Cas13d system with guide RNAs that span exon-exon junctions in the mature RNA. We performed a high-throughput essentiality screen, quantitative RT-PCR assays, and PacBio long read sequencing to affirm our ability to specifically target and robustly knockdown individual RNA isoforms. In parallel, we provide computational tools for experimental design and screen analysis. Considering all possible splice junctions annotated in GENCODE for multi-isoform genes and our gRNA efficacy predictions, we estimate that our junction-centric strategy can uniquely target up to 89% of human RNA isoforms, including 50,066 protein-coding and 11,415 lncRNA isoforms. Importantly, this specificity spans all splicing and transcriptional events, including exon skipping and inclusion, alternative 5 and 3 splice sites, and alternative starts and ends.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554647">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554647" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554647">
        <p class="paperTitle">The genome of the Wollemi pine, a critically endangered living fossil unchanged since the Cretaceous, reveals extensive ancient transposon activity.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554647" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554647" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stevenson, D. W.; Ramakrishnan, S.; Alves, C. d. S.; Coelho, L. A.; Kramer, M.; Goodwin, S.; Ramos, O. M.; Eshel, G.; Sondervan, V. M.; Frangos, S.; Zumajo-Cardona, C.; Jenike, K.; Ou, S.; Wang, X.; Lee, Y. P.; Loke, S.; Rossetto, M.; McPherson, H.; Nigris, S.; Moschin, S.; Little, D. P.; Katari, M. S.; Varala, K.; Kolokotronis, S.-O.; Ambrose, B.; Croft, L. J.; Coruzzi, G. M.; Schatz, M.; McCombie, W. R.; Martienssen, R. A.</p>
        <p class="info">Score: 26.2, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554647' target='https://doi.org/10.1101/2023.08.24.554647'> 10.1101/2023.08.24.554647</a></p>
        <p class="abstract">We present the genome of the living fossil, Wollemia nobilis, a southern hemisphere conifer morphologically unchanged since the Cretaceous. Presumed extinct until rediscovery in 1994, the Wollemi pine is critically endangered with less than 60 wild adults threatened by intensifying bushfires in the Blue Mountains of Australia. The 12 Gb genome is among the most contiguous large plant genomes assembled, with extremely low heterozygosity and unusual abundance of DNA transposons. Reduced representation and genome re-sequencing of individuals confirms a relictual population since the last major glacial/drying period in Australia, 120 ky BP. Small RNA and methylome sequencing reveal conservation of ancient silencing mechanisms despite the presence of thousands of active and abundant transposons, including some transferred horizontally to conifers from arthropods in the Jurassic. A retrotransposon burst 8-6 my BP coincided with population decline, possibly as an adaptation enhancing epigenetic diversity. Wollemia, like other conifers, is susceptible to Phytophthora, and a suite of defense genes, similar to those in loblolly pine, are targeted for silencing by sRNAs in leaves. The genome provides insight into the earliest seed plants, while enabling conservation efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.10.556512">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.10.556512" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.10.556512">
        <p class="paperTitle">Genomic Insights into the Population History and Adaptive Traits of Latin American Criollo Cattle</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.10.556512" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.10.556512" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ward, J. A.; Ng&#39;ang&#39;a, S. I.; Randhawa, I. A. S.; McHugo, G. P.; O&#39;Grady, J. F.; Florez, J. M.; Browne, J. A.; Perez O&#39;Brien, A. M.; Landaeta-Hernandez, A. J.; Garcia, J. F.; Sonstegard, T. S.; Frantz, L. A. F.; Salter-Townshend, M.; MacHugh, D. E.</p>
        <p class="info">Score: 11.3, Published: 2023-09-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.10.556512' target='https://doi.org/10.1101/2023.09.10.556512'> 10.1101/2023.09.10.556512</a></p>
        <p class="abstract">Criollo cattle, descendants of animals brought by Iberian colonists to the Americas, have been subject to centuries of natural and human-mediated selection in novel tropical agroecological zones. Consequently, these breeds have evolved distinct characteristics such as resistance to disease and exceptional heat tolerance. In addition to European taurine (Bos taurus) ancestry, it has been proposed that gene flow from African taurine and Asian zebu (Bos indicus) cattle has shaped the ancestry of Criollo cattle. In this study, we analysed Criollo breeds from Colombia and Venezuela using whole genome sequencing data to examine population structure and admixture at very high resolution. We did not detect evidence of African taurine admixture in Criollo cattle; however, there was evidence of longstanding zebu ancestry in one breed (Harton del Valle). In addition, we detected selection signatures associated with a myriad of adaptive traits, revealing genes linked to thermotolerance, reproduction, fertility, immunity, and distinct coat and skin coloration traits. This study underscores the remarkable adaptability of Criollo cattle and highlights the genetic richness and potential of these breeds in the face of climate change, habitat flux, and disease challenges. Further research is warranted to leverage these findings for more effective and sustainable cattle breeding programmes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555582">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555582" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555582">
        <p class="paperTitle">Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555582" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555582" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.</p>
        <p class="info">Score: 124.5, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555582' target='https://doi.org/10.1101/2023.08.30.555582'> 10.1101/2023.08.30.555582</a></p>
        <p class="abstract">Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence. Using statistics derived from Borzois predicted coverage, we isolate and accurately score variant effects across multiple layers of regulation, including transcription, splicing, and polyadenylation. Evaluated on QTLs, Borzoi is competitive with, and often outperforms, state-of-the-art models trained on individual regulatory functions. By applying attribution methods to the derived statistics, we extract cis-regulatory patterns driving RNA expression and post-transcriptional regulation in normal tissues. The wide availability of RNA-seq data across species, conditions, and assays profiling specific aspects of regulation emphasizes the potential of this approach to decipher the mapping from DNA sequence to regulatory function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.11.557177">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.11.557177" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.11.557177">
        <p class="paperTitle">Retracing Human Genetic Histories and Natural Selection Using Precise Local Ancestry Inference</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.11.557177" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.11.557177" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lerga-Jaso, J.; Novkovic, B.; Unnikrishnan, D.; Bamunusinghe, V.; Hatorangan, M.; Manson, C.; Pedersen, H.; Osama, A.; Terpolovsky, A.; Bohn, S.; De Marino, A.; Mahmoud, A.; Bircan, K.; Khan, U.; Grabherr, M.; Yazdi, P.</p>
        <p class="info">Score: 5.1, Published: 2023-09-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.11.557177' target='https://doi.org/10.1101/2023.09.11.557177'> 10.1101/2023.09.11.557177</a></p>
        <p class="abstract">In an increasingly diverse world, including admixed individuals in genomic studies is imperative for equity and portability. A crucial first step is precise local ancestry inference (LAI). We have developed Orchestra, a LAI model with unprecedented accuracy, and trained on over 10,000 single-origin individuals from 35 worldwide populations. We employed Orchestra to delve into genetic relationships and demographic histories, with a focus on Latin Americans, a prime example of admixture, and the Ashkenazi Jewish, whose origins have long been debated. Finally, Orchestra enabled us to map signatures of selection, notably identifying trace Scandinavian ancestry in British samples and unveiling an immune-rich region linked to respiratory infections. Our work advances the field of LAI and holds promise for improvements in future applications for admixed populations.

One-Sentence SummaryOrchestra unveils Latino and Ashkenazi ancestral roots and a candidate Viking locus under selection in the British population</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555983">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555983" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555983">
        <p class="paperTitle">A General Approach to Adjusting Genetic Studies for Assortative Mating</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555983" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555983" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bilghese, M.; Manansala, R.; Jaishankar, D.; Jala, J.; Benjamin, D. J.; Kimball, M.; Auer, P. L.; Livermore, M. A.; Turley, P.</p>
        <p class="info">Score: 23.9, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555983' target='https://doi.org/10.1101/2023.09.01.555983'> 10.1101/2023.09.01.555983</a></p>
        <p class="abstract">The effects of assortative mating (AM) on estimates from genetic studies has been receiving increasing attention in recent years. We extend existing AM theory to more general models of sorting and conclude that correct theory-based AM adjustments require knowledge of complicated, unknown historical sorting patterns. We propose a simple, general-purpose approach using polygenic indexes (PGIs). Our approach can estimate the fraction of genetic variance and genetic correlation that is driven by AM. Our approach is less effective when applied to Mendelian randomization (MR) studies for two reasons: AM can induce a form of selection bias in MR studies that remains after our adjustment; and, in the MR context, the adjustment is particularly sensitive to PGI estimation error. Using data from the UK Biobank, we find that AM inflates genetic correlation estimates between health traits and education by 14% on average. Our results suggest caution in interpreting genetic correlations or MR estimates for traits subject to AM.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.08.556886">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.08.556886" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.08.556886">
        <p class="paperTitle">Mycotools: An Automated and Scalable Platform for Comparative Genomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.08.556886" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.08.556886" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Konkel, Z.; Slot, J.</p>
        <p class="info">Score: 5.0, Published: 2023-09-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.08.556886' target='https://doi.org/10.1101/2023.09.08.556886'> 10.1101/2023.09.08.556886</a></p>
        <p class="abstract">Comparative genomics comprises analyses that investigate the genetic basis of organismal biology and ecology, which have also been applied to high throughput trait screening for applied purposes. The number of fungal genomes deposited in publicly available databases are currently in exponential growth. Due to the limited cutting-edge software availability and size or efficiency constraints of web-based analyses, comparative genomics research is often conducted on local computing environments. There is thus a need for an efficient standardized framework for locally assimilating, curating, and interfacing with genomic data. We present Mycotools as a comparative genomics database software suite that automatically curates, updates, and standardizes local comparative genomics. Mycotools incorporates novel analysis pipelines that are built on a suite of modules that streamline routine-to-complex comparative genomic tasks. The Mycotools software suite serves as a foundation for accessible and reproducible large-scale comparative genomics on local compute systems.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
