<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Comparative analysis of multiplexed in situ gene expression profiling technologies
Authors: Hartman, A.; Satija, R.
Score: 104.1, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135
The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Comparative analysis of multiplexed in situ gene expression profiling technologies
Authors: Hartman, A.; Satija, R.
Score: 104.1, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135
The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-17T10:40:22+00:00" />
<meta property="article:modified_time" content="2024-01-17T10:40:22+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Comparative analysis of multiplexed in situ gene expression profiling technologies
Authors: Hartman, A.; Satija, R.
Score: 104.1, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135
The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Comparative analysis of multiplexed in situ gene expression profiling technologies\nAuthors: Hartman, A.; Satija, R.\nScore: 104.1, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135\nThe burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets.",
  "keywords": [
    
  ],
  "articleBody": " Comparative analysis of multiplexed in situ gene expression profiling technologies\nAuthors: Hartman, A.; Satija, R.\nScore: 104.1, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135\nThe burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets. We find that standard sensitivity metrics, such as the number of unique molecules detected per cell, are not directly comparable across datasets due to substantial differences in the incidence of off-target molecular artifacts impacting specificity. To address these challenges, we explored various potential sources of molecular artifacts, developed novel metrics to control for them, and utilized these metrics to evaluate and compare different in situ technologies. Finally, we demonstrate how molecular false positives can seriously confound spatially-aware differential expression analysis, requiring caution in the interpretation of downstream results. Our analysis provides guidance for the selection, processing, and interpretation of in situ spatial technologies.\nDe novo annotation of the wheat pangenome reveals complexity and diversity of the hexaploid wheat pan-transcriptome\nAuthors: White, B.; LUX, T.; Rusholme-Pilcher, R.; Juhasz, A.; Kaithakottil, G.; Duncan, S.; Simmonds, J.; Rees, H.; Wright, J.; Colmer, J.; Ward, S.; joynson, R.; Coombes, B.; Irish, N.; Henderson, S.; Baker, T.; Chapman, H.; Catchpole, L.; Gharbi, K.; Okada, M.; Handa, H.; Nasuda, S.; Shimizu, K.; Gundlach, H.; Lang, D.; Naamati, G.; Legg, E.; Bharti, A.; Colgrave, M.; Haerty, W. `; Uauy, C. `; Swarbreck, D.; Borrill, P.; Poland, J.; Krattinger, S.; Stein, N.; Mayer, K. F. X.; Pozniak, C.; Spannagl, M.; Hall, A.\nScore: 26.6, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574802\nWheat is the most widely cultivated crop in the world with over 215 million hectares grown annually. However, to meet the demands of a growing global population, breeders face the challenge of increasing wheat production by approximately 60% within the next 40 years. The 10+ Wheat Genomes Project recently sequenced and assembled the genomes of 15 wheat cultivars to develop our understanding of genetic diversity and selection within the pan-genome of wheat. Here, we provide a wheat pan-transcriptome with de novo annotation and differential expression analysis for nine of these wheat cultivars, across multiple different tissues and whole seedlings sampled at dusk/dawn. Analysis of these de novo annotations facilitated the discovery of genes absent from the Chinese Spring reference, identified genes specific to particular cultivars and defined the core and dispensable genomes. Expression analysis across cultivars and tissues revealed conservation in expression between a large core set of homoeologous genes, but also widespread changes in sub-genome homoeolog expression bias between cultivars. Co-expression network analysis revealed the impact of divergence of sub-genome homoeolog expression and identified tissue-associated cultivar-specific expression profiles. In summary, this work provides both a valuable resource for the wider wheat community and reveals diversity in gene content and expression patterns between global wheat cultivars.\nA Phased, Chromosome-scale Genome for Malus domestica 'WA 38'\nAuthors: Zhang, H.; Eaker, A.; Ko, I.; Haney, S.; Khuu, N.; Ryan, K.; Appleby, A.; Hoffmann, B.; Landis, H.; Pierro, K.; Willsea, N.; Hargarten, H.; Yocca, A.; Harkess, A.; Honaas, L.; Ficklin, S.\nScore: 16.3, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574953\nGenome sequencing for agriculturally important Rosaceous crops has made rapid progress both in completeness and annotation quality. Whole genome sequence and annotation gives breeders, researchers, and growers information about cultivar specific traits such as fruit quality, disease resistance, and informs strategies to enhance postharvest storage. Here we present a haplotype-phased, chromosomal level, genome of Malus domestica, WA 38, a new cultivar of apple released to market in 2017 as Cosmic Crisp (R). Using both short and long read sequencing our fully phased genome of WA 38 is about 650 Megabases, with a BUSCO score of 98.7% complete. The genome contains two haplomes (HapA and HapB) which are composed of 17 chromosomes each. A total of 48,227 and 54,479 genes were annotated from HapA and HapB, respectively. Additionally, we provide genome-scale comparisons to Gala, Honeycrisp, and other relevant cultivars highlighting major structural and quality differences. This assembly and annotation was done in collaboration with the American Campus Tree Genomes project that included WA 38 (WSU) and the dAnjou pear (Auburn University). To ensure transparency, reproducibility, and applicability for any genome project, our genome assembly and annotation workflow is recorded in detail and shared under a public GitHub repository. All software is containerized, offering a simple implementation of the workflow.\nLong-read RNA sequencing identifies region- and sex-specific C57BL/6J mouse brain mRNA isoform expression and usage\nAuthors: Jones, E. F.; Howton, T. C.; Flanary, V. L.; Clark, A. D.; Lasseigne, B. N.\nScore: 14.1, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575219\nAlternative splicing (AS) contributes to the biological heterogeneity between species, sexes, tissues, and cell types. Many diseases are either caused by alterations in AS or by alterations to AS. Therefore, measuring AS accurately and efficiently is critical for assessing molecular phenotypes, including those associated with disease. Long-read sequencing enables more accurate quantification of differentially spliced isoform expression than short-read sequencing approaches, and third-generation platforms facilitate high-throughput experiments. To assess differences in AS across the cerebellum, cortex, hippocampus, and striatum by sex, we generated and analyzed Oxford Nanopore Technologies (ONT) long-read RNA sequencing (lrRNA-Seq) C57BL/6J mouse brain cDNA libraries. From \u003e85 million reads that passed quality control metrics, we calculated differential gene expression (DGE), differential transcript expression (DTE), and differential transcript usage (DTU) across brain regions and by sex. We found significant DGE, DTE, and DTU across brain regions and that the cerebellum had the most differences compared to the other three regions. Additionally, we found region-specific differential splicing between sexes, with the most sex differences in DTU in the cortex and no DTU in the hippocampus. We also report on two distinct patterns of sex DTU we observed, sex-divergent and sex-specific, that could potentially help explain sex differences in the prevalence and prognosis of various neurological and psychiatric disorders in future studies. Finally, we built a Shiny web application for researchers to explore the data further. Our study provides a resource for the community; it underscores the importance of AS in biological heterogeneity and the utility of long-read sequencing to better understand AS in the brain.\nDual DNA/RNA-binding factor regulates dynamics of hnRNP splicing condensates\nAuthors: Ray, M.; Zaborowsky, J.; Mahableshwarkar, P.; Vaidyanathan, S.; Shum, J.; Viswanathan, R.; Huang, A.; Wang, S.-H.; Johnson, V.; Wake, N.; Conard, A. M.; Conicella, A. E.; Puterbaugh, R.; Fawzi, N. L.; Larschan, E.\nScore: 7.8, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575216\nDespite decades of research, mechanisms by which co-transcriptional alternative splicing events are targeted to the correct genomic locations to drive cell fate decisions remain unknown. By combining structural and molecular approaches, we define a new mechanism by which an essential transcription factor (TF) targets co-transcriptional splicing through physical and functional interaction with RNA and RNA binding proteins (RBPs). We show that an essential TF co-transcriptionally regulates sex-specific alternative splicing by directly interacting with a subset of target RNAs on chromatin and modulating the dynamics of hnRNPA2 homolog nuclear splicing condensates.\nDORA: an interactive map for the visualization and analysis of ancient human DNA and associated data\nAuthors: Harris, K. D.; Greenbaum, G.\nScore: 5.5, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575663\nThe ability to sequence ancient genomes has revolutionized the way in which we study evolutionary history by providing access to the most important aspect of evolution - time. Until recently, studying human demography, ecology, biology, and history using population genomic inference relied on contemporary genomic datasets. Over the past decade, the availability of human ancient DNA (aDNA) has increased rapidly, almost doubling every year, opening the way for spatiotemporal studies of ancient human populations. However, the multidimensionality of aDNA, with genotypes having temporal, spatial and genomic coordinates, and the need to integrate multiple sources of data, poses a challenge for developing meta-analyses pipelines. To address this challenge, we developed a publicly-available interactive tool, DORA, which integrates multiple data types, genomic and non-genomic, in a unified interface. This web-based tool allows users to browse sample metadata along with additional layers of information, such as population structure, climatic data, and unpublished samples. Users can then perform analyses on genotypes of these samples, or export sample subsets for external analyses. DORA integrates analyses and visualizations in a single intuitive interface, resolving the technical issues of combining datasets from different sources and formats, and allowing researchers to focus on analysis and the scientific questions that can be addressed through analysis of aDNA datasets.\nscRNA-seq reveals persistent aberrant differentiation of nasal epithelium driven by TNFα and TGFβ in post-COVID syndrome\nAuthors: Faehnrich, A.; Reddy, K.; Ott, F.; Maluje, Y.; Saurabh, R.; Schaaf, A.-C.; Winkelmann, S.; Voss, B.; Laudien, M.; Bahmer, T.; Heyckendorf, J.; Brinkmann, F.; Schreiber, S.; Lieb, W.; Weckmann, M.; Busch, H.\nScore: 5.3, Published: 2024-01-10 DOI: 10.1101/2024.01.10.574801\nPost-COVID syndrome (PCS) currently affects approximately 3-17% of people following severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection and has the potential to become a significant global health burden. PCS presents with various symptoms, and methods for improved PCS assessment are presently developed to guide therapy. Nevertheless, there are few mechanistic insights and treatment options. Here, we performed single-cell RNA transcriptomics on nasal biopsies from 33 patients suffering from PCS with mild, moderate, or severe symptoms. We identified 17 different cell clusters representing 12 unique cell populations, including all major epithelial cell types of the conducting airways and basal, secretory, and ciliated cells. Severe PCS was associated with decreased numbers of ciliated cells and the presence of immune cells. Ensuing inflammatory signaling upregulated TGF{beta} and induced an epithelial-mesenchymal transition, which led to the high abundance of basal cells and a mis-stratified epithelium. We confirmed the results in vitro using an air-liquid interface culture and validated TNF as the causal inflammatory cytokine. In summary, our results show that one mechanism for sustained PCS is not through continued viral load, but through the presence of immune cells in nasal tissue leading to impaired mucosal barrier function and repeated infections. These findings could be further explored as a therapeutic option akin to other chronic inflammatory diseases by inhibiting the TNF-TGF{beta} axis, restoring the nasal epithelium, and reducing respiratory tract-related infections.\nPangenome reconstruction in rats enhances genotype-phenotype mapping and novel variant discovery\nAuthors: Villani, F.; Guarracino, A.; Ward, R. R.; Green, T.; Emms, M.; Pravenec, M.; Prins, P.; Garrison, E.; Williams, R. W.; Chen, H.; Colonna, V.\nScore: 5.3, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575041\nThe HXB/BXH family of recombinant inbred rat strains is a unique genetic resource that has been extensively phenotyped over 25 years, resulting in a vast dataset of quantitative molecular and physiological phenotypes. We built a pangenome graph from 10x Genomics linked-read data for 31 recombinant inbred rats to study genetic variation and association mapping. The pangenome length was on average 2.4 times greater than the corresponding length of the reference mRatBN7.2, confirming the capture of substantial additional variation. We validated variants in challenging regions, including complex structural variants resolving into multiple haplotypes. Phenome-wide association analysis of validated SNPs uncovered variants associated with glucose/insulin levels and hippocampal gene expression. We propose an interaction between Pirl1l1, Cromogranine expression, TNF- levels, and insulin regulation. This study demonstrates the utility of linked-read pangenomes for comprehensive variant detection and mapping phenotypic diversity in a widely used rat genetic reference panel.\nConvergence and global epidemiology of Klebsiella pneumoniae plasmids harbouring the iuc3 virulence locus.\nAuthors: Gibbon, M. J.; Couto, N.; Cozens, K.; Habib, S.; Cowley, L.; Aanensen, D.; Corander, J.; Thorpe, H.; Hetland, M. A.; Sassera, D.; Merla, C.; Corbella, M.; Ferrari, C.; Turner, K. M.; Sirikancha, K.; Dulyayangkul, P.; Alhusein, N.; Charoenlap, N.; Thamlikitikul, V.; Avison, M. B.; Feil, E. J.\nScore: 14.8, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574329\nBackgroundKlebsiella pneumoniae (Kp) is an important pathogen of humans and animals, and recent reports of convergent strains that carry both virulence and antimicrobial resistance genes (ARGs) have raised serious public health concern. The plasmid-borne iuc locus, encoding the siderophore aerobactin, is a key virulence factor in this species. The variant iuc3 is associated with porcine and human clinical isolates and is carried by mostly uncharacterised IncF plasmids. MethodsWe used a combination of short-read and long-read sequencing to characterise IncFIB(K)/IncFII iuc3-carrying plasmids harboured by 79 Kp isolates and one K. oxytoca isolate recovered as part of two large One-Health studies in Italy (SpARK) and Thailand (OH-DART). Adding data from public repositories gave a combined dataset of 517 iuc3 isolates, and the plasmids were analysed using both clustering and phylogenetic methods. FindingsWe note seven large, convergent, plasmids from Thailand that have emerged through the hybridisation of co-circulating plasmids harbouring iuc3 and antimicrobial resistance genes (ARGs) encoding extended-spectrum beta-lactamases (ESBLs). We were also able to identify putative parental plasmids which were mostly associated with two neighbouring meat markets, as were the hybrid plasmids. Clustering and global phylogenetic analyses resolved an iuc3 plasmid sub-group circulating throughout Asia, with occasional examples in Europe and elsewhere. This variant carries multiple ARGs and is commonly harboured by clinical isolates, thus warranting targeted plasmid surveillance. InterpretationOur study reveals that plasmid hybridisation leading to the convergence of resistance and virulence traits may be very common, even in non-clinical ( One-Health) settings. Population-scale plasmid genomics makes it possible to identify putative parental plasmids, which will help to identify plasmid types that are most likely to hybridise, and what the selective consequences may be for the plasmid and host. A distinct iuc3 plasmid sub-variant is associated with clinical isolates in Asia which requires close monitoring. Research In ContextMultiple reports of convergent clones of Klebsiella pneumoniae that combine both hypervirulence and multidrug resistance (MDR-hvKp) have been published recently; a PubMed search in November 2023 using the key words convergence Klebsiella pneumoniae returned 143 papers, 99 of which were published from 2020 onwards. Our study demonstrates that the hybridisation of plasmids carrying AMR and virulence genes is a frequent, ongoing, process in natural populations. The subsequent transfer of plasmids conferring both traits is thus likely to be a key driver behind the spread of convergent strains. Our study also provides an exemplar of how hybrid assemblies can facilitate large-scale global genomic plasmid epidemiology. Evidence before the studyAlthough multiple recent reports highlight the emergence and spread of convergent Kp strains, the confluence of resistance and virulence genes within the same plasmid has not been studied at a population level, and putative parental plasmids are rarely identified. Moreover, there have been few high-resolution genomic epidemiology studies on closely related plasmids using both long and short-read data on a global scale. Added valueWe more than double the number of complete sequences available for plasmids harbouring iuc3 from 58 to 139 and provide evidence on the host lineages most likely to harbour these plasmids (e.g., ST35), and epidemiological source (e.g., pig, wild animal, human). Our comparative analysis of phylogenetic and clustering approaches will help to inform future plasmid epidemiological studies. ImplicationsThe hybridisation of plasmids harbouring virulence and resistance genes occurs frequently in natural populations, even within One-Health settings. However, the selective drivers (if any) and evolutionary consequences of this phenomenon are unclear. There is clear utility in generating closed plasmid genomes on a population scale, and targeted plasmid surveillance on a clinical sub-variant of iuc3 plasmids is warranted.\nGenomic epidemiology of Candida auris introduction and outbreaks in the United Kingdom\nAuthors: Kappel, D.; Gifford, H.; Brackin, A. P.; Abdolrasouli, A.; Eyre, D. W.; Jeffery, K.; Schelenz, S.; Aanensen, D.; Brown, C.; Borman, A.; Johnson, E.; Holmes, A.; Armstrong-James, D.; Fisher, M.; Rhodes, J.\nScore: 4.8, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575049\nBackground: Candida auris is a globally emerging fungal pathogen that causes nosocomial invasive infections, particularly in intensive care units (ICU). Four prominent genetic clades originating from distinct geographic regions have been identified: South Asia (Clade I), East Asia (Clade II), Africa (Clade III) and South America (Clade IV) with each clade presenting differing antifungal resistance profiles. We aimed to elucidate the epidemiology of this infection in the United Kingdom (UK) 2014 - 2019 using genomic approaches. Methods: Genome sequences from 24 isolates from six patients in four London hospitals were combined with genome sequences from 183 additional outbreak isolates from nine UK hospitals. These data were used to assess the numbers and timings of C. auris lineage introductions into the UK and to infer subsequent patterns of inter- and intra-hospital transmission. Findings: We found evidence for at least three introductions of C. auris into the UK, one from Clade I and one from Clade III. The evolutionary rate of C. auris Clade I and Clade III were estimated at 2.764 x 10-4 and 3.186 x 10-4 substitutions per polymorphic site per year, respectively, with entry into the UK as 2013 and 2016 for Clades I and III respectively. We observed temporal and geographical evidence for multiple transmission events between hospitals and identified local within-hospital patient-to-patient transmission events. Interpretation: These data confirm that C. auris is a newly emerged infection in the UK with at least three original introductions into this country. Our analysis shows that hospital outbreaks are linked and confirms that transmission amongst patients has occurred, explaining local hospital outbreaks. Our findings argue that enhanced surveillance of C. auris infection within the UK is necessary in order to protect healthcare and to curtail expansion of this emerging infection.\n",
  "wordCount" : "2871",
  "inLanguage": "en",
  "datePublished": "2024-01-17T10:40:22Z",
  "dateModified": "2024-01-17T10:40:22Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 17, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575135" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575135">
        <p class="paperTitle">Comparative analysis of multiplexed in situ gene expression profiling technologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hartman, A.; Satija, R.</p>
        <p class="info">Score: 104.1, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575135' target='https://doi.org/10.1101/2024.01.11.575135'> 10.1101/2024.01.11.575135</a></p>
        <p class="abstract">The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets. We find that standard sensitivity metrics, such as the number of unique molecules detected per cell, are not directly comparable across datasets due to substantial differences in the incidence of off-target molecular artifacts impacting specificity. To address these challenges, we explored various potential sources of molecular artifacts, developed novel metrics to control for them, and utilized these metrics to evaluate and compare different in situ technologies. Finally, we demonstrate how molecular false positives can seriously confound spatially-aware differential expression analysis, requiring caution in the interpretation of downstream results. Our analysis provides guidance for the selection, processing, and interpretation of in situ spatial technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574802">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574802" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574802">
        <p class="paperTitle">De novo annotation of the wheat pangenome reveals complexity and diversity of the hexaploid wheat pan-transcriptome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574802" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574802" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: White, B.; LUX, T.; Rusholme-Pilcher, R.; Juhasz, A.; Kaithakottil, G.; Duncan, S.; Simmonds, J.; Rees, H.; Wright, J.; Colmer, J.; Ward, S.; joynson, R.; Coombes, B.; Irish, N.; Henderson, S.; Baker, T.; Chapman, H.; Catchpole, L.; Gharbi, K.; Okada, M.; Handa, H.; Nasuda, S.; Shimizu, K.; Gundlach, H.; Lang, D.; Naamati, G.; Legg, E.; Bharti, A.; Colgrave, M.; Haerty, W. `; Uauy, C. `; Swarbreck, D.; Borrill, P.; Poland, J.; Krattinger, S.; Stein, N.; Mayer, K. F. X.; Pozniak, C.; Spannagl, M.; Hall, A.</p>
        <p class="info">Score: 26.6, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574802' target='https://doi.org/10.1101/2024.01.09.574802'> 10.1101/2024.01.09.574802</a></p>
        <p class="abstract">Wheat is the most widely cultivated crop in the world with over 215 million hectares grown annually. However, to meet the demands of a growing global population, breeders face the challenge of increasing wheat production by approximately 60% within the next 40 years. The 10&#43; Wheat Genomes Project recently sequenced and assembled the genomes of 15 wheat cultivars to develop our understanding of genetic diversity and selection within the pan-genome of wheat. Here, we provide a wheat pan-transcriptome with de novo annotation and differential expression analysis for nine of these wheat cultivars, across multiple different tissues and whole seedlings sampled at dusk/dawn. Analysis of these de novo annotations facilitated the discovery of genes absent from the Chinese Spring reference, identified genes specific to particular cultivars and defined the core and dispensable genomes. Expression analysis across cultivars and tissues revealed conservation in expression between a large core set of homoeologous genes, but also widespread changes in sub-genome homoeolog expression bias between cultivars. Co-expression network analysis revealed the impact of divergence of sub-genome homoeolog expression and identified tissue-associated cultivar-specific expression profiles. In summary, this work provides both a valuable resource for the wider wheat community and reveals diversity in gene content and expression patterns between global wheat cultivars.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574953">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574953" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574953">
        <p class="paperTitle">A Phased, Chromosome-scale Genome for Malus domestica &#39;WA 38&#39;</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574953" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574953" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, H.; Eaker, A.; Ko, I.; Haney, S.; Khuu, N.; Ryan, K.; Appleby, A.; Hoffmann, B.; Landis, H.; Pierro, K.; Willsea, N.; Hargarten, H.; Yocca, A.; Harkess, A.; Honaas, L.; Ficklin, S.</p>
        <p class="info">Score: 16.3, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574953' target='https://doi.org/10.1101/2024.01.10.574953'> 10.1101/2024.01.10.574953</a></p>
        <p class="abstract">Genome sequencing for agriculturally important Rosaceous crops has made rapid progress both in completeness and annotation quality. Whole genome sequence and annotation gives breeders, researchers, and growers information about cultivar specific traits such as fruit quality, disease resistance, and informs strategies to enhance postharvest storage. Here we present a haplotype-phased, chromosomal level, genome of Malus domestica,  WA 38, a new cultivar of apple released to market in 2017 as Cosmic Crisp (R). Using both short and long read sequencing our fully phased genome of  WA 38 is about 650 Megabases, with a BUSCO score of 98.7% complete. The genome contains two haplomes (HapA and HapB) which are composed of 17 chromosomes each. A total of 48,227 and 54,479 genes were annotated from HapA and HapB, respectively. Additionally, we provide genome-scale comparisons to  Gala,  Honeycrisp, and other relevant cultivars highlighting major structural and quality differences. This assembly and annotation was done in collaboration with the American Campus Tree Genomes project that included  WA 38 (WSU) and the  dAnjou pear (Auburn University). To ensure transparency, reproducibility, and applicability for any genome project, our genome assembly and annotation workflow is recorded in detail and shared under a public GitHub repository. All software is containerized, offering a simple implementation of the workflow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575219" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575219">
        <p class="paperTitle">Long-read RNA sequencing identifies region- and sex-specific C57BL/6J mouse brain mRNA isoform expression and usage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jones, E. F.; Howton, T. C.; Flanary, V. L.; Clark, A. D.; Lasseigne, B. N.</p>
        <p class="info">Score: 14.1, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575219' target='https://doi.org/10.1101/2024.01.11.575219'> 10.1101/2024.01.11.575219</a></p>
        <p class="abstract">Alternative splicing (AS) contributes to the biological heterogeneity between species, sexes, tissues, and cell types. Many diseases are either caused by alterations in AS or by alterations to AS. Therefore, measuring AS accurately and efficiently is critical for assessing molecular phenotypes, including those associated with disease. Long-read sequencing enables more accurate quantification of differentially spliced isoform expression than short-read sequencing approaches, and third-generation platforms facilitate high-throughput experiments. To assess differences in AS across the cerebellum, cortex, hippocampus, and striatum by sex, we generated and analyzed Oxford Nanopore Technologies (ONT) long-read RNA sequencing (lrRNA-Seq) C57BL/6J mouse brain cDNA libraries. From &gt;85 million reads that passed quality control metrics, we calculated differential gene expression (DGE), differential transcript expression (DTE), and differential transcript usage (DTU) across brain regions and by sex. We found significant DGE, DTE, and DTU across brain regions and that the cerebellum had the most differences compared to the other three regions. Additionally, we found region-specific differential splicing between sexes, with the most sex differences in DTU in the cortex and no DTU in the hippocampus. We also report on two distinct patterns of sex DTU we observed, sex-divergent and sex-specific, that could potentially help explain sex differences in the prevalence and prognosis of various neurological and psychiatric disorders in future studies. Finally, we built a Shiny web application for researchers to explore the data further. Our study provides a resource for the community; it underscores the importance of AS in biological heterogeneity and the utility of long-read sequencing to better understand AS in the brain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575216">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575216" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575216">
        <p class="paperTitle">Dual DNA/RNA-binding factor regulates dynamics of hnRNP splicing condensates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575216" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575216" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ray, M.; Zaborowsky, J.; Mahableshwarkar, P.; Vaidyanathan, S.; Shum, J.; Viswanathan, R.; Huang, A.; Wang, S.-H.; Johnson, V.; Wake, N.; Conard, A. M.; Conicella, A. E.; Puterbaugh, R.; Fawzi, N. L.; Larschan, E.</p>
        <p class="info">Score: 7.8, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575216' target='https://doi.org/10.1101/2024.01.11.575216'> 10.1101/2024.01.11.575216</a></p>
        <p class="abstract">Despite decades of research, mechanisms by which co-transcriptional alternative splicing events are targeted to the correct genomic locations to drive cell fate decisions remain unknown. By combining structural and molecular approaches, we define a new mechanism by which an essential transcription factor (TF) targets co-transcriptional splicing through physical and functional interaction with RNA and RNA binding proteins (RBPs). We show that an essential TF co-transcriptionally regulates sex-specific alternative splicing by directly interacting with a subset of target RNAs on chromatin and modulating the dynamics of hnRNPA2 homolog nuclear splicing condensates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575663" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575663">
        <p class="paperTitle">DORA: an interactive map for the visualization and analysis of ancient human DNA and associated data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harris, K. D.; Greenbaum, G.</p>
        <p class="info">Score: 5.5, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575663' target='https://doi.org/10.1101/2024.01.15.575663'> 10.1101/2024.01.15.575663</a></p>
        <p class="abstract">The ability to sequence ancient genomes has revolutionized the way in which we study evolutionary history by providing access to the most important aspect of evolution - time. Until recently, studying human demography, ecology, biology, and history using population genomic inference relied on contemporary genomic datasets. Over the past decade, the availability of human ancient DNA (aDNA) has increased rapidly, almost doubling every year, opening the way for spatiotemporal studies of ancient human populations. However, the multidimensionality of aDNA, with genotypes having temporal, spatial and genomic coordinates, and the need to integrate multiple sources of data, poses a challenge for developing meta-analyses pipelines. To address this challenge, we developed a publicly-available interactive tool, DORA, which integrates multiple data types, genomic and non-genomic, in a unified interface. This web-based tool allows users to browse sample metadata along with additional layers of information, such as population structure, climatic data, and unpublished samples. Users can then perform analyses on genotypes of these samples, or export sample subsets for external analyses. DORA integrates analyses and visualizations in a single intuitive interface, resolving the technical issues of combining datasets from different sources and formats, and allowing researchers to focus on analysis and the scientific questions that can be addressed through analysis of aDNA datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574801">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574801" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574801">
        <p class="paperTitle">scRNA-seq reveals persistent aberrant differentiation of nasal epithelium driven by TNFα and TGFβ in post-COVID syndrome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574801" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574801" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faehnrich, A.; Reddy, K.; Ott, F.; Maluje, Y.; Saurabh, R.; Schaaf, A.-C.; Winkelmann, S.; Voss, B.; Laudien, M.; Bahmer, T.; Heyckendorf, J.; Brinkmann, F.; Schreiber, S.; Lieb, W.; Weckmann, M.; Busch, H.</p>
        <p class="info">Score: 5.3, Published: 2024-01-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574801' target='https://doi.org/10.1101/2024.01.10.574801'> 10.1101/2024.01.10.574801</a></p>
        <p class="abstract">Post-COVID syndrome (PCS) currently affects approximately 3-17% of people following severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection and has the potential to become a significant global health burden. PCS presents with various symptoms, and methods for improved PCS assessment are presently developed to guide therapy. Nevertheless, there are few mechanistic insights and treatment options. Here, we performed single-cell RNA transcriptomics on nasal biopsies from 33 patients suffering from PCS with mild, moderate, or severe symptoms. We identified 17 different cell clusters representing 12 unique cell populations, including all major epithelial cell types of the conducting airways and basal, secretory, and ciliated cells. Severe PCS was associated with decreased numbers of ciliated cells and the presence of immune cells. Ensuing inflammatory signaling upregulated TGF{beta} and induced an epithelial-mesenchymal transition, which led to the high abundance of basal cells and a mis-stratified epithelium. We confirmed the results in vitro using an air-liquid interface culture and validated TNF as the causal inflammatory cytokine. In summary, our results show that one mechanism for sustained PCS is not through continued viral load, but through the presence of immune cells in nasal tissue leading to impaired mucosal barrier function and repeated infections. These findings could be further explored as a therapeutic option akin to other chronic inflammatory diseases by inhibiting the TNF-TGF{beta} axis, restoring the nasal epithelium, and reducing respiratory tract-related infections.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575041">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575041" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575041">
        <p class="paperTitle">Pangenome reconstruction in rats enhances genotype-phenotype mapping and novel variant discovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575041" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575041" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Villani, F.; Guarracino, A.; Ward, R. R.; Green, T.; Emms, M.; Pravenec, M.; Prins, P.; Garrison, E.; Williams, R. W.; Chen, H.; Colonna, V.</p>
        <p class="info">Score: 5.3, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575041' target='https://doi.org/10.1101/2024.01.10.575041'> 10.1101/2024.01.10.575041</a></p>
        <p class="abstract">The HXB/BXH family of recombinant inbred rat strains is a unique genetic resource that has been extensively phenotyped over 25 years, resulting in a vast dataset of quantitative molecular and physiological phenotypes. We built a pangenome graph from 10x Genomics linked-read data for 31 recombinant inbred rats to study genetic variation and association mapping. The pangenome length was on average 2.4 times greater than the corresponding length of the reference mRatBN7.2, confirming the capture of substantial additional variation. We validated variants in challenging regions, including complex structural variants resolving into multiple haplotypes. Phenome-wide association analysis of validated SNPs uncovered variants associated with glucose/insulin levels and hippocampal gene expression. We propose an interaction between Pirl1l1, Cromogranine expression, TNF- levels, and insulin regulation. This study demonstrates the utility of linked-read pangenomes for comprehensive variant detection and mapping phenotypic diversity in a widely used rat genetic reference panel.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574329">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574329" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574329">
        <p class="paperTitle">Convergence and global epidemiology of Klebsiella pneumoniae plasmids harbouring the iuc3 virulence locus.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574329" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574329" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gibbon, M. J.; Couto, N.; Cozens, K.; Habib, S.; Cowley, L.; Aanensen, D.; Corander, J.; Thorpe, H.; Hetland, M. A.; Sassera, D.; Merla, C.; Corbella, M.; Ferrari, C.; Turner, K. M.; Sirikancha, K.; Dulyayangkul, P.; Alhusein, N.; Charoenlap, N.; Thamlikitikul, V.; Avison, M. B.; Feil, E. J.</p>
        <p class="info">Score: 14.8, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574329' target='https://doi.org/10.1101/2024.01.05.574329'> 10.1101/2024.01.05.574329</a></p>
        <p class="abstract">BackgroundKlebsiella pneumoniae (Kp) is an important pathogen of humans and animals, and recent reports of  convergent strains that carry both virulence and antimicrobial resistance genes (ARGs) have raised serious public health concern. The plasmid-borne iuc locus, encoding the siderophore aerobactin, is a key virulence factor in this species. The variant iuc3 is associated with porcine and human clinical isolates and is carried by mostly uncharacterised IncF plasmids.

MethodsWe used a combination of short-read and long-read sequencing to characterise IncFIB(K)/IncFII iuc3-carrying plasmids harboured by 79 Kp isolates and one K. oxytoca isolate recovered as part of two large  One-Health studies in Italy (SpARK) and Thailand (OH-DART). Adding data from public repositories gave a combined dataset of 517 iuc3 isolates, and the plasmids were analysed using both clustering and phylogenetic methods.

FindingsWe note seven large, convergent, plasmids from Thailand that have emerged through the hybridisation of co-circulating plasmids harbouring iuc3 and antimicrobial resistance genes (ARGs) encoding extended-spectrum beta-lactamases (ESBLs). We were also able to identify putative parental plasmids which were mostly associated with two neighbouring meat markets, as were the hybrid plasmids. Clustering and global phylogenetic analyses resolved an iuc3 plasmid sub-group circulating throughout Asia, with occasional examples in Europe and elsewhere. This variant carries multiple ARGs and is commonly harboured by clinical isolates, thus warranting targeted plasmid surveillance.

InterpretationOur study reveals that plasmid hybridisation leading to the convergence of resistance and virulence traits may be very common, even in non-clinical ( One-Health) settings. Population-scale plasmid genomics makes it possible to identify putative parental plasmids, which will help to identify plasmid types that are most likely to hybridise, and what the selective consequences may be for the plasmid and host. A distinct iuc3 plasmid sub-variant is associated with clinical isolates in Asia which requires close monitoring.

Research In ContextMultiple reports of  convergent clones of Klebsiella pneumoniae that combine both hypervirulence and multidrug resistance (MDR-hvKp) have been published recently; a PubMed search in November 2023 using the key words  convergence Klebsiella pneumoniae returned 143 papers, 99 of which were published from 2020 onwards. Our study demonstrates that the hybridisation of plasmids carrying AMR and virulence genes is a frequent, ongoing, process in natural populations. The subsequent transfer of plasmids conferring both traits is thus likely to be a key driver behind the spread of convergent strains. Our study also provides an exemplar of how hybrid assemblies can facilitate large-scale global genomic plasmid epidemiology.

Evidence before the studyAlthough multiple recent reports highlight the emergence and spread of convergent Kp strains, the confluence of resistance and virulence genes within the same plasmid has not been studied at a population level, and putative parental plasmids are rarely identified. Moreover, there have been few high-resolution genomic epidemiology studies on closely related plasmids using both long and short-read data on a global scale.

Added valueWe more than double the number of complete sequences available for plasmids harbouring iuc3 from 58 to 139 and provide evidence on the host lineages most likely to harbour these plasmids (e.g., ST35), and epidemiological source (e.g., pig, wild animal, human). Our comparative analysis of phylogenetic and clustering approaches will help to inform future plasmid epidemiological studies.

ImplicationsThe hybridisation of plasmids harbouring virulence and resistance genes occurs frequently in natural populations, even within  One-Health settings. However, the selective drivers (if any) and evolutionary consequences of this phenomenon are unclear. There is clear utility in generating closed plasmid genomes on a population scale, and targeted plasmid surveillance on a clinical sub-variant of iuc3 plasmids is warranted.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575049" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575049">
        <p class="paperTitle">Genomic epidemiology of Candida auris introduction and outbreaks in the United Kingdom</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kappel, D.; Gifford, H.; Brackin, A. P.; Abdolrasouli, A.; Eyre, D. W.; Jeffery, K.; Schelenz, S.; Aanensen, D.; Brown, C.; Borman, A.; Johnson, E.; Holmes, A.; Armstrong-James, D.; Fisher, M.; Rhodes, J.</p>
        <p class="info">Score: 4.8, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575049' target='https://doi.org/10.1101/2024.01.15.575049'> 10.1101/2024.01.15.575049</a></p>
        <p class="abstract">Background: Candida auris is a globally emerging fungal pathogen that causes nosocomial invasive infections, particularly in intensive care units (ICU). Four prominent genetic clades originating from distinct geographic regions have been identified: South Asia (Clade I), East Asia (Clade II), Africa (Clade III) and South America (Clade IV) with each clade presenting differing antifungal resistance profiles. We aimed to elucidate the epidemiology of this infection in the United Kingdom (UK) 2014 - 2019 using genomic approaches. Methods: Genome sequences from 24 isolates from six patients in four London hospitals were combined with genome sequences from 183 additional outbreak isolates from nine UK hospitals. These data were used to assess the numbers and timings of C. auris lineage introductions into the UK and to infer subsequent patterns of inter- and intra-hospital transmission. Findings: We found evidence for at least three introductions of C. auris into the UK, one from Clade I and one from Clade III. The evolutionary rate of C. auris Clade I and Clade III were estimated at 2.764 x 10-4 and 3.186 x 10-4 substitutions per polymorphic site per year, respectively, with entry into the UK as 2013 and 2016 for Clades I and III respectively. We observed temporal and geographical evidence for multiple transmission events between hospitals and identified local within-hospital patient-to-patient transmission events. Interpretation: These data confirm that C. auris is a newly emerged infection in the UK with at least three original introductions into this country. Our analysis shows that hospital outbreaks are linked and confirms that transmission amongst patients has occurred, explaining local hospital outbreaks. Our findings argue that enhanced surveillance of C. auris infection within the UK is necessary in order to protect healthcare and to curtail expansion of this emerging infection.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
