<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A novel approach to exploring the dark genome and its application to mapping of the vertebrate virus &#39;fossil record&#39;.
Authors: Blanco-Melo, D.; Campbell, M. A.; Zhu, H.; Dennis, T.; Mohda, S.; Lytras, S.; Hughes, J. J.; Gatseva, A.; Gifford, R. J.
Score: 27.6, Published: 2023-10-19 DOI: 10.1101/2023.10.17.562709
Background: Genomic regions that remain poorly understood, often referred to as the &#34;dark genome,&#34; contain a variety of functionally relevant and biologically informative genome features.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="A novel approach to exploring the dark genome and its application to mapping of the vertebrate virus &#39;fossil record&#39;.
Authors: Blanco-Melo, D.; Campbell, M. A.; Zhu, H.; Dennis, T.; Mohda, S.; Lytras, S.; Hughes, J. J.; Gatseva, A.; Gifford, R. J.
Score: 27.6, Published: 2023-10-19 DOI: 10.1101/2023.10.17.562709
Background: Genomic regions that remain poorly understood, often referred to as the &#34;dark genome,&#34; contain a variety of functionally relevant and biologically informative genome features." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-22T10:37:23+00:00" />
<meta property="article:modified_time" content="2023-10-22T10:37:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="A novel approach to exploring the dark genome and its application to mapping of the vertebrate virus &#39;fossil record&#39;.
Authors: Blanco-Melo, D.; Campbell, M. A.; Zhu, H.; Dennis, T.; Mohda, S.; Lytras, S.; Hughes, J. J.; Gatseva, A.; Gifford, R. J.
Score: 27.6, Published: 2023-10-19 DOI: 10.1101/2023.10.17.562709
Background: Genomic regions that remain poorly understood, often referred to as the &#34;dark genome,&#34; contain a variety of functionally relevant and biologically informative genome features."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "A novel approach to exploring the dark genome and its application to mapping of the vertebrate virus \u0026#39;fossil record\u0026#39;.\nAuthors: Blanco-Melo, D.; Campbell, M. A.; Zhu, H.; Dennis, T.; Mohda, S.; Lytras, S.; Hughes, J. J.; Gatseva, A.; Gifford, R. J.\nScore: 27.6, Published: 2023-10-19 DOI: 10.1101/2023.10.17.562709\nBackground: Genomic regions that remain poorly understood, often referred to as the \u0026#34;dark genome,\u0026#34; contain a variety of functionally relevant and biologically informative genome features.",
  "keywords": [
    
  ],
  "articleBody": " A novel approach to exploring the dark genome and its application to mapping of the vertebrate virus 'fossil record'.\nAuthors: Blanco-Melo, D.; Campbell, M. A.; Zhu, H.; Dennis, T.; Mohda, S.; Lytras, S.; Hughes, J. J.; Gatseva, A.; Gifford, R. J.\nScore: 27.6, Published: 2023-10-19 DOI: 10.1101/2023.10.17.562709\nBackground: Genomic regions that remain poorly understood, often referred to as the \"dark genome,\" contain a variety of functionally relevant and biologically informative genome features. These include endogenous viral elements (EVEs) - virus-derived sequences that can dramatically impact host biology and serve as a virus \"fossil record\". In this study, we introduce a database-integrated genome screening (DIGS) approach to investigating the dark genome in silico, focusing on EVEs found within vertebrate genomes. Results: Using DIGS on 874 vertebrate species genomes, we uncovered approximately 1.1 million EVE sequences, with over 99% originating from endogenous retroviruses or transposable elements that contain EVE DNA. We show that the remaining 6038 sequences represent over a thousand distinct horizontal gene transfer events across ten virus families, including some that have not previously been reported as EVEs. We explore the genomic and phylogenetic characteristics of non-retroviral EVEs and determine their rates of acquisition during vertebrate evolution. Our study uncovers novel virus diversity and broadens our knowledge of virus distribution among vertebrate hosts. It also provides new insights into the long-term evolution of highly pathogenic filoviruses. Conclusions: We comprehensively catalogue and analyse EVEs within 874 vertebrate genomes, shedding light on the distribution, diversity and long-term evolution of viruses, and revealing their extensive impact on vertebrate genome evolution. Our results demonstrate the power of linking a relational database management system to a similarity search-based screening pipeline for in silico exploration of the dark genome.\nActive regulatory elements recruit cohesin to establish cell-specific chromatin domains.\nAuthors: Georgiades, E.; Harrold, C. L.; Roberts, N.; Kassouf, M.; Riva, S. G.; Sanders, E. L.; Francis, H. S.; Blayney, J.; Oudelaar, M.; Milne, T. A.; Higgs, D.; Hughes, J. R.\nScore: 18.2, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562171\nAs the structure of the genome is analysed at ever increasing resolution it is becoming clear that there is considerable variation in the 3D chromatin architecture across different cell types. It has been proposed that this may, in part, be due to increased recruitment of cohesin to activated cis-elements (enhancers and promoters) leading to cell-type specific loop extrusion underlying the formation of new subTADs. Here we show that cohesin correlates well with the presence of active enhancers and this varies in an allele-specific manner with the presence or absence of polymorphic enhancers which vary from one individual to another. Using the alpha globin cluster as a model, we show that when all enhancers are removed, peaks of cohesin disappear from these regions and the erythroid specific subTAD is no longer formed. Re-insertion of the major alpha globin enhancer (R2) is associated with the appearance of a new peak of cohesin at the site of insertion. In complementary experiments insertion of R2 into a \"neutral\" region of the genome recruits cohesin, induces transcription and creates a new large (75kb) erythroid specific domain. Together these findings support the proposal that active enhancers recruit cohesin, stimulate loop extrusion and promote the formation of cell specific subTADs.\np53 rapidly restructures 3D chromatin organization to trigger a transcriptional response\nAuthors: Serra, F.; Nieto-Aliseda, A.; Fanlo, L.; Rovirosa, L.; Cabrera-Pasadas, M.; Lazarenkov, A.; Urmeneta, B.; Alcalde, A.; Nola, E. M.; Okorokov, A. L.; Fraser, P.; Graupera, M.; Castillo, S. D.; Sardina, J. L.; Valencia, A.; Javierre, B. M.\nScore: 18.8, Published: 2023-10-13 DOI: 10.1101/2023.10.10.561663\nActivation of the p53 tumor suppressor triggers a transcriptional program to control cellular response to stress. However, the molecular mechanisms by which p53 controls gene transcription are not completely understood. Here, using a multi-omics integration framework, we uncover the critical role of spatio-temporal genome architecture in this process. We demonstrate that p53 drives direct and indirect changes in genome compartments, topologically associating domains and DNA loops within minutes of its activation, which escort the p53 transcriptional program along time. Focused on p53-bound enhancers, we report a core transcriptional program of 340 genes directly regulated by p53 over distance, most of these not previously identified. Finally, we showcase that p53 controls transcription of distal genes through newly formed and pre-existing enhancer-promoter loops in a cohesin dependent manner. Taken together, our findings demonstrate a previously unappreciated architectural role of p53 as regulator at distinct topological layers and provide a reliable set of new p53 direct target genes that may help future designs of p53-based cancer therapies.\nQuantification of the escape from X chromosome inactivation with the million cell-scale human single-cell omics datasets reveals heterogeneity of escape across cell types and tissues\nAuthors: Tomofuji, Y.; Edahiro, R.; Shirai, Y.; Kock, K. H.; Sonehara, K.; Wang, Q. S.; Namba, S.; Moody, J.; Ando, Y.; Suzuki, A.; Yata, T.; Ogawa, K.; Namkoong, H.; Lin, Q. X. X.; Buyamin, E. V.; Tan, L. M.; Sonthalia, R.; Han, K. Y.; Tanaka, H.; Lee, H.; Asian Immune Diversity Atlas Network, ; Japan COVID-19 Task Force, ; The BioBank Japan Project, ; Okuno, T.; Liu, B.; Matsuda, K.; Fukunaga, K.; Mochizuki, H.; Park, W.-Y.; Yamamoto, K.; Hon, C.-C.; Shin, J. W.; Prabhakar, S.; Kumanogoh, A.; Okada, Y.\nScore: 12.5, Published: 2023-10-18 DOI: 10.1101/2023.10.14.561800\nOne of the two X chromosomes of females is silenced through X chromosome inactivation (XCI) to compensate for the difference in the dosage between sexes. Among the X-linked genes, several genes escape from XCI, which could contribute to the differential gene expression between the sexes. However, the differences in the escape across cell types and tissues are still poorly characterized because no methods could directly evaluate the escape under a physiological condition at the cell-cluster resolution with versatile technology. Here, we developed a method, single-cell Level inactivated X chromosome mapping (scLinaX), which directly quantifies relative gene expression from the inactivated X chromosome with droplet-based single-cell RNA-sequencing (scRNA-seq) data. The scLinaX and differentially expressed genes analyses with the scRNA-seq datasets of [~]1,000,000 blood cells consistently identified the relatively strong degree of escape in lymphocytes compared to myeloid cells. An extension of scLinaX for multi-modal datasets, scLinaX-multi, suggested a stronger degree of escape in lymphocytes than myeloid cells at the chromatin-accessibility level with a 10X multiome dataset. The scLinaX analysis with the human multiple-organ scRNA-seq datasets also identified the relatively strong degree of escape from XCI in lymphoid tissues and lymphocytes. Finally, effect size comparisons of genome-wide association studies between sexes identified the larger effect sizes of the PRKX gene locus-lymphocyte counts association in females than males. This could suggest evidence of the underlying impact of escape on the genotype-phenotype association in humans. Overall, scLinaX and the quantified catalog of escape identified the heterogeneity of escape across cell types and tissues and would contribute to expanding the current understanding of the XCI, escape, and sex differences in gene regulation.\nKeratinocytes Drive the Epithelial Hyperplasia Key to Sea Lice Resistance in Coho Salmon\nAuthors: Salisbury, S. J.; Ruiz Daniels, R.; Monaghan, S. J.; Bron, J. E.; Villamayor, P. R.; Gervais, O.; Fast, M. D.; Sveen, L.; Houston, R. D.; Robinson, N.; Robledo, D.\nScore: 12.0, Published: 2023-10-17 DOI: 10.1101/2023.10.15.562030\nBackgroundSalmonid species have followed markedly divergent evolutionary trajectories in their interactions with sea lice. While sea lice parasitism poses significant economic, environmental, and animal welfare challenges for Atlantic salmon (Salmo salar) aquaculture, coho salmon (Oncorhynchus kisutch) exhibit near-complete resistance to sea lice, achieved through a potent epithelial hyperplasia response leading to rapid louse detachment. The molecular mechanisms underlying these divergent responses to sea lice are unknown. ResultsWe characterised the cellular and molecular responses of Atlantic salmon and coho salmon to sea lice using single-nuclei RNA sequencing. Juvenile fish were exposed to copepodid sea lice (Lepeophtheirus salmonis), and lice-attached pelvic fin and skin samples were collected 12h, 24h, 36h, 48h, and 60h after exposure, along with control samples. Comparative analysis of control and treatment samples revealed an immune and wound-healing response that was common to both species, but attenuated in Atlantic salmon, potentially reflecting greater sea louse immunomodulation. Our results revealed unique but complementary roles of three layers of keratinocytes in the epithelial hyperplasia response leading to rapid sea lice rejection in coho salmon. Our results suggest that basal keratinocytes direct the expansion and mobility of intermediate and, especially, superficial keratinocytes, which eventually encapsulate the parasite. ConclusionOur results highlight the key role of keratinocytes to coho salmons sea lice resistance, and the diverged biological response of the two salmonid host species when interacting with this parasite. This study has identified key pathways and candidate genes that could be manipulated using various biotechnological solutions to improve Atlantic salmon sea lice resistance.\nMaximizing Efficiency in SedaDNA Analysis: A Novel Extract Pooling Approach\nAuthors: Oberreiter, V.; Gelabert, P.; Brueck, F.; Franz, S.; Zelger, E.; Szedlacsek, S.; Cheronet, O.; Tenorio Cano, F.; Exler, F.; Zagorc, B.; Karavanic, I.; Banda, M.; Gasparyan, B.; Straus, L. G.; Gonzalez Morales, M. R.; Kappelman, J.; Stahlschmidt, M.; Rattei, T.; Kraemer, S. M.; Sawyer, S.; Pinhasi, R.\nScore: 8.5, Published: 2023-10-20 DOI: 10.1101/2023.10.17.562718\nIn recent years, the field of ancient DNA (aDNA) has taken a new direction toward studying human population dynamics through sedimentary DNA (sedaDNA), enabling the study of past ecosystems. However, the screening of numerous sediment samples from archaeological sites remains a time-consuming and costly endeavor, particularly when targeting hominin DNA. Here, we present a novel high-throughput method that facilitates the fast and efficient analysis of sediment samples by applying a pooled testing method. Our approach involves combining multiple extracts, allowing users to parallelize laboratory procedures early in the sample preparation pipeline while effectively screening for the presence of aDNA. Pooled samples that exhibit aDNA signals can then undergo detailed analysis, while empty pools are discarded. We have successfully applied our extract pooling method to various sediment samples from Middle and Upper Paleolithic sites in Europe, Asia, and Africa. Notably, our results reveal that an aDNA signal remains discernible even when pooled with four negative samples. We also demonstrate that the DNA yield of double-stranded libraries increases significantly when reducing the extract input, potentially mitigating the effects of inhibition. By embracing this innovative approach, researchers can analyze large numbers of sediment samples for aDNA preservation, achieving significant cost reductions of up to 70% and reducing hands-on laboratory time to one-fifth.\nChromosomal DNA sequences of the Pacific saury genome: versatile resources for fishery science and comparative biology\nAuthors: Sato, M.; Fukuda, K.; Kadota, M.; Makino-Itou, H.; Tatsumi, K.; Yamauchi, S.; Kuraku, S.\nScore: 8.4, Published: 2023-10-18 DOI: 10.1101/2023.10.16.562003\nPacific saury (Cololabis saira) is a commercially important small pelagic fish species in Asian. In this study, we conducted the first-ever whole genome sequencing of this species, with single molecule, real-time (SMRT) sequencing technology. The obtained high-fidelity (HiFi) long-read sequence data, which amount to approximately 30 folds of its haploid genome size that was measured with quantitative PCR (1.17 Gb), were assembled into contigs. Scaffolding with Hi-C reads yielded a whole genome assembly containing 24 chromosome-scale sequences, with a scaffold N50 length of 47.7 Mb. Screening of repetitive elements including telomeric repeats was performed to characterize possible factors that need to be resolved towards telomere-to-telomere sequencing. The larger genome size than in medaka, a close relative in Beloniformes, is at least partly explained by larger repetitive element quantity, which is reflected in more abundant tRNAs, in the Pacific saury genome. Protein-coding regions was predicted using transcriptome data, which resulted in 22,274 components. Retrieval of Pacific saury homologs of aquaporin (AQP) genes known from other teleost fishes validated high completeness and continuity of the genome assembly. These resources are available at https://treethinkers.nig.ac.jp/saira/ and will assist various molecular-level studies in fishery science and comparative biology.\nEpigenetic context predicts gene expression variation and reproductive traits across genetically identical individuals\nAuthors: Webster, A. K.; Willis, J. H.; Johnson, E.; Sarkies, P.; Phillips, P. C.\nScore: 10.7, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562270\nIn recent decades, genome-wide association studies (GWAS) have been the major approach to understand the biological basis of individual differences in traits and diseases. However, GWAS approaches have proven to have limited predictive power to explain individual differences, particularly for complex traits and diseases in which environmental factors play a substantial role in their etiology. Indeed, individual differences persist even in genetically identical individuals, although fully separating genetic and environmental causation is difficult or impossible in most organisms. To understand the basis of individual differences in the absence of genetic differences, we measured two quantitative reproductive traits in 180 genetically identical young adult Caenorhabditis elegans roundworms in a shared environment and performed single-individual transcriptomics on each worm. We identified hundreds of genes for which expression variation was strongly associated with reproductive traits, some of which depended on prior environmental experience and some of which was random. Multiple small sets of genes together were highly predictive of reproductive traits across individuals, explaining on average over half and over a quarter of variation in the two traits. We manipulated mRNA levels of predictive genes using RNA interference to identify a set of causal genes, demonstrating the utility of this approach for both prediction and understanding underlying biology. Finally, we found that the chromatin environment of predictive genes was enriched for H3K27 trimethylation, suggesting that individual gene expression differences underlying critical traits may be driven in part by chromatin structure. Together, this work shows that individual differences in gene expression that arise independently of underlying genetic differences are both predictive and causal in shaping reproductive traits at levels that equal or exceed genetic variation.\nProteasome inhibition reprograms chromatin landscape in breast cancer\nAuthors: Kinyamu, H. K.; Bennett, B. D.; Ward, J. M.; Archer, T. K.\nScore: 5.1, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562284\nThe 26S proteasome is the major protein degradation machinery in cells. Cancer cells use the proteasome to modulate gene expression networks that promote tumor growth. Proteasome inhibitors have emerged as effective cancer therapeutics, but how they work mechanistically remains unclear. Here, using integrative genomic analysis, we discovered unexpected reprogramming of the chromatin landscape and RNAPII transcription initiation in breast cancer cells treated with the proteasome inhibitor MG132. The cells acquired dynamic changes in chromatin accessibility at specific genomic loci termed Differentially Open Chromatin Regions (DOCRs). DOCRs with decreased accessibility were promoter proximal and exhibited unique chromatin architecture associated with divergent RNAPII transcription. Conversely, DOCRs with increased accessibility were primarily distal to transcription start sites and enriched in oncogenic super enhancers predominantly accessible in non-basal breast tumor subtypes. These findings describe the mechanisms by which the proteasome modulates the expression of gene networks intrinsic to breast cancer biology. HighlightsO_LIProteasome inhibition uncovers de novo Differential Open Chromatin Regions (DOCRs) in breast cancer cells. C_LIO_LIProteasome inhibitor sensitive promoters exhibit a distinctive chromatin architecture with discrete transcription initiation patterns. C_LIO_LIProteasome inhibition reprograms accessibility of super enhancers. C_LIO_LIProteasome inhibitor sensitive super enhancers distinguish basal from non-basal breast cancer subtypes. C_LI\nNon-coding variants impact cis-regulatory coordination in a cell type-specific manner\nAuthors: Pushkarev, O.; van Mierlo, G.; Kribelbauer, J. F.; Saelens, W.; Gardeux, V.; Deplancke, B.\nScore: 12.8, Published: 2023-10-11 DOI: 10.1101/2023.10.11.561870\nBACKGROUNDInteractions among cis-regulatory elements (CREs) play a crucial role in gene regulation. Various approaches have been developed to map these interactions genome-wide, including those relying on interindividual epigenomic variation to identify groups of covariable regulatory elements, referred to as chromatin modules (CMs). While CM mapping allows to investigate the relationship between chromatin modularity and gene expression, the computational principles used for CM identification vary in their application and outcomes. RESULTSWe comprehensively evaluate and streamline existing CM mapping tools and present guidelines for optimal utilization of epigenome data from a diverse population of individuals to assess regulatory coordination across the human genome. We showcase the effectiveness of our recommended practices by analysing distinct cell types and demonstrate cell type-specificity of CRE interactions in CMs and their relevance for gene expression. Integration of genotype information revealed that many non-coding disease-associated variants affect the activity of CMs in a cell type-specific manner by affecting the binding of cell type-specific transcription factors. We provide example cases that illustrate in detail how CMs can be used to deconstruct GWAS loci, understand variable expression of cell surface receptors in immune cells and reveal how genetic variation can impact the expression of prognostic markers in chronic lymphocytic leukaemia. CONCLUSIONSOur study presents an optimal strategy for CM mapping, and reveals how CMs capture the coordination of CREs and its impact on gene expression. Non-coding genetic variants can disrupt this coordination, and we highlight how this may lead to disease predisposition in a cell type-specific manner.\n",
  "wordCount" : "2744",
  "inLanguage": "en",
  "datePublished": "2023-10-22T10:37:23Z",
  "dateModified": "2023-10-22T10:37:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on October 22, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562709">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562709" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562709">
        <p class="paperTitle">A novel approach to exploring the dark genome and its application to mapping of the vertebrate virus &#39;fossil record&#39;.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562709" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562709" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blanco-Melo, D.; Campbell, M. A.; Zhu, H.; Dennis, T.; Mohda, S.; Lytras, S.; Hughes, J. J.; Gatseva, A.; Gifford, R. J.</p>
        <p class="info">Score: 27.6, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562709' target='https://doi.org/10.1101/2023.10.17.562709'> 10.1101/2023.10.17.562709</a></p>
        <p class="abstract">Background: Genomic regions that remain poorly understood, often referred to as the &#34;dark genome,&#34; contain a variety of functionally relevant and biologically informative genome features. These include endogenous viral elements (EVEs) - virus-derived sequences that can dramatically impact host biology and serve as a virus &#34;fossil record&#34;. In this study, we introduce a database-integrated genome screening (DIGS) approach to investigating the dark genome in silico, focusing on EVEs found within vertebrate genomes. Results: Using DIGS on 874 vertebrate species genomes, we uncovered approximately 1.1 million EVE sequences, with over 99% originating from endogenous retroviruses or transposable elements that contain EVE DNA. We show that the remaining 6038 sequences represent over a thousand distinct horizontal gene transfer events across ten virus families, including some that have not previously been reported as EVEs. We explore the genomic and phylogenetic characteristics of non-retroviral EVEs and determine their rates of acquisition during vertebrate evolution. Our study uncovers novel virus diversity and broadens our knowledge of virus distribution among vertebrate hosts. It also provides new insights into the long-term evolution of highly pathogenic filoviruses. Conclusions: We comprehensively catalogue and analyse EVEs within 874 vertebrate genomes, shedding light on the distribution, diversity and long-term evolution of viruses, and revealing their extensive impact on vertebrate genome evolution. Our results demonstrate the power of linking a relational database management system to a similarity search-based screening pipeline for in silico exploration of the dark genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562171">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562171" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562171">
        <p class="paperTitle">Active regulatory elements recruit cohesin to establish cell-specific chromatin domains.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562171" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562171" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Georgiades, E.; Harrold, C. L.; Roberts, N.; Kassouf, M.; Riva, S. G.; Sanders, E. L.; Francis, H. S.; Blayney, J.; Oudelaar, M.; Milne, T. A.; Higgs, D.; Hughes, J. R.</p>
        <p class="info">Score: 18.2, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562171' target='https://doi.org/10.1101/2023.10.13.562171'> 10.1101/2023.10.13.562171</a></p>
        <p class="abstract">As the structure of the genome is analysed at ever increasing resolution it is becoming clear that there is considerable variation in the 3D chromatin architecture across different cell types. It has been proposed that this may, in part, be due to increased recruitment of cohesin to activated cis-elements (enhancers and promoters) leading to cell-type specific loop extrusion underlying the formation of new subTADs. Here we show that cohesin correlates well with the presence of active enhancers and this varies in an allele-specific manner with the presence or absence of polymorphic enhancers which vary from one individual to another. Using the alpha globin cluster as a model, we show that when all enhancers are removed, peaks of cohesin disappear from these regions and the erythroid specific subTAD is no longer formed. Re-insertion of the major alpha globin enhancer (R2) is associated with the appearance of a new peak of cohesin at the site of insertion. In complementary experiments insertion of R2 into a &#34;neutral&#34; region of the genome recruits cohesin, induces transcription and creates a new large (75kb) erythroid specific domain. Together these findings support the proposal that active enhancers recruit cohesin, stimulate loop extrusion and promote the formation of cell specific subTADs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561663">
        <p class="paperTitle">p53 rapidly restructures 3D chromatin organization to trigger a transcriptional response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Serra, F.; Nieto-Aliseda, A.; Fanlo, L.; Rovirosa, L.; Cabrera-Pasadas, M.; Lazarenkov, A.; Urmeneta, B.; Alcalde, A.; Nola, E. M.; Okorokov, A. L.; Fraser, P.; Graupera, M.; Castillo, S. D.; Sardina, J. L.; Valencia, A.; Javierre, B. M.</p>
        <p class="info">Score: 18.8, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561663' target='https://doi.org/10.1101/2023.10.10.561663'> 10.1101/2023.10.10.561663</a></p>
        <p class="abstract">Activation of the p53 tumor suppressor triggers a transcriptional program to control cellular response to stress. However, the molecular mechanisms by which p53 controls gene transcription are not completely understood. Here, using a multi-omics integration framework, we uncover the critical role of spatio-temporal genome architecture in this process. We demonstrate that p53 drives direct and indirect changes in genome compartments, topologically associating domains and DNA loops within minutes of its activation, which escort the p53 transcriptional program along time. Focused on p53-bound enhancers, we report a core transcriptional program of 340 genes directly regulated by p53 over distance, most of these not previously identified. Finally, we showcase that p53 controls transcription of distal genes through newly formed and pre-existing enhancer-promoter loops in a cohesin dependent manner. Taken together, our findings demonstrate a previously unappreciated architectural role of p53 as regulator at distinct topological layers and provide a reliable set of new p53 direct target genes that may help future designs of p53-based cancer therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.14.561800">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.14.561800" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.14.561800">
        <p class="paperTitle">Quantification of the escape from X chromosome inactivation with the million cell-scale human single-cell omics datasets reveals heterogeneity of escape across cell types and tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.14.561800" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.14.561800" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tomofuji, Y.; Edahiro, R.; Shirai, Y.; Kock, K. H.; Sonehara, K.; Wang, Q. S.; Namba, S.; Moody, J.; Ando, Y.; Suzuki, A.; Yata, T.; Ogawa, K.; Namkoong, H.; Lin, Q. X. X.; Buyamin, E. V.; Tan, L. M.; Sonthalia, R.; Han, K. Y.; Tanaka, H.; Lee, H.; Asian Immune Diversity Atlas Network,  ; Japan COVID-19 Task Force,  ; The BioBank Japan Project,  ; Okuno, T.; Liu, B.; Matsuda, K.; Fukunaga, K.; Mochizuki, H.; Park, W.-Y.; Yamamoto, K.; Hon, C.-C.; Shin, J. W.; Prabhakar, S.; Kumanogoh, A.; Okada, Y.</p>
        <p class="info">Score: 12.5, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.14.561800' target='https://doi.org/10.1101/2023.10.14.561800'> 10.1101/2023.10.14.561800</a></p>
        <p class="abstract">One of the two X chromosomes of females is silenced through X chromosome inactivation (XCI) to compensate for the difference in the dosage between sexes. Among the X-linked genes, several genes escape from XCI, which could contribute to the differential gene expression between the sexes. However, the differences in the escape across cell types and tissues are still poorly characterized because no methods could directly evaluate the escape under a physiological condition at the cell-cluster resolution with versatile technology. Here, we developed a method, single-cell Level inactivated X chromosome mapping (scLinaX), which directly quantifies relative gene expression from the inactivated X chromosome with droplet-based single-cell RNA-sequencing (scRNA-seq) data. The scLinaX and differentially expressed genes analyses with the scRNA-seq datasets of [~]1,000,000 blood cells consistently identified the relatively strong degree of escape in lymphocytes compared to myeloid cells. An extension of scLinaX for multi-modal datasets, scLinaX-multi, suggested a stronger degree of escape in lymphocytes than myeloid cells at the chromatin-accessibility level with a 10X multiome dataset. The scLinaX analysis with the human multiple-organ scRNA-seq datasets also identified the relatively strong degree of escape from XCI in lymphoid tissues and lymphocytes. Finally, effect size comparisons of genome-wide association studies between sexes identified the larger effect sizes of the PRKX gene locus-lymphocyte counts association in females than males. This could suggest evidence of the underlying impact of escape on the genotype-phenotype association in humans. Overall, scLinaX and the quantified catalog of escape identified the heterogeneity of escape across cell types and tissues and would contribute to expanding the current understanding of the XCI, escape, and sex differences in gene regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.15.562030">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.15.562030" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.15.562030">
        <p class="paperTitle">Keratinocytes Drive the Epithelial Hyperplasia Key to Sea Lice Resistance in Coho Salmon</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.15.562030" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.15.562030" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Salisbury, S. J.; Ruiz Daniels, R.; Monaghan, S. J.; Bron, J. E.; Villamayor, P. R.; Gervais, O.; Fast, M. D.; Sveen, L.; Houston, R. D.; Robinson, N.; Robledo, D.</p>
        <p class="info">Score: 12.0, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.15.562030' target='https://doi.org/10.1101/2023.10.15.562030'> 10.1101/2023.10.15.562030</a></p>
        <p class="abstract">BackgroundSalmonid species have followed markedly divergent evolutionary trajectories in their interactions with sea lice. While sea lice parasitism poses significant economic, environmental, and animal welfare challenges for Atlantic salmon (Salmo salar) aquaculture, coho salmon (Oncorhynchus kisutch) exhibit near-complete resistance to sea lice, achieved through a potent epithelial hyperplasia response leading to rapid louse detachment. The molecular mechanisms underlying these divergent responses to sea lice are unknown.

ResultsWe characterised the cellular and molecular responses of Atlantic salmon and coho salmon to sea lice using single-nuclei RNA sequencing. Juvenile fish were exposed to copepodid sea lice (Lepeophtheirus salmonis), and lice-attached pelvic fin and skin samples were collected 12h, 24h, 36h, 48h, and 60h after exposure, along with control samples. Comparative analysis of control and treatment samples revealed an immune and wound-healing response that was common to both species, but attenuated in Atlantic salmon, potentially reflecting greater sea louse immunomodulation. Our results revealed unique but complementary roles of three layers of keratinocytes in the epithelial hyperplasia response leading to rapid sea lice rejection in coho salmon. Our results suggest that basal keratinocytes direct the expansion and mobility of intermediate and, especially, superficial keratinocytes, which eventually encapsulate the parasite.

ConclusionOur results highlight the key role of keratinocytes to coho salmons sea lice resistance, and the diverged biological response of the two salmonid host species when interacting with this parasite. This study has identified key pathways and candidate genes that could be manipulated using various biotechnological solutions to improve Atlantic salmon sea lice resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562718">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562718" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562718">
        <p class="paperTitle">Maximizing Efficiency in SedaDNA Analysis: A Novel Extract Pooling Approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562718" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562718" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oberreiter, V.; Gelabert, P.; Brueck, F.; Franz, S.; Zelger, E.; Szedlacsek, S.; Cheronet, O.; Tenorio Cano, F.; Exler, F.; Zagorc, B.; Karavanic, I.; Banda, M.; Gasparyan, B.; Straus, L. G.; Gonzalez Morales, M. R.; Kappelman, J.; Stahlschmidt, M.; Rattei, T.; Kraemer, S. M.; Sawyer, S.; Pinhasi, R.</p>
        <p class="info">Score: 8.5, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562718' target='https://doi.org/10.1101/2023.10.17.562718'> 10.1101/2023.10.17.562718</a></p>
        <p class="abstract">In recent years, the field of ancient DNA (aDNA) has taken a new direction toward studying human population dynamics through sedimentary DNA (sedaDNA), enabling the study of past ecosystems. However, the screening of numerous sediment samples from archaeological sites remains a time-consuming and costly endeavor, particularly when targeting hominin DNA. Here, we present a novel high-throughput method that facilitates the fast and efficient analysis of sediment samples by applying a pooled testing method. Our approach involves combining multiple extracts, allowing users to parallelize laboratory procedures early in the sample preparation pipeline while effectively screening for the presence of aDNA. Pooled samples that exhibit aDNA signals can then undergo detailed analysis, while empty pools are discarded. We have successfully applied our extract pooling method to various sediment samples from Middle and Upper Paleolithic sites in Europe, Asia, and Africa. Notably, our results reveal that an aDNA signal remains discernible even when pooled with four negative samples. We also demonstrate that the DNA yield of double-stranded libraries increases significantly when reducing the extract input, potentially mitigating the effects of inhibition. By embracing this innovative approach, researchers can analyze large numbers of sediment samples for aDNA preservation, achieving significant cost reductions of up to 70% and reducing hands-on laboratory time to one-fifth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562003">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562003" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562003">
        <p class="paperTitle">Chromosomal DNA sequences of the Pacific saury genome: versatile resources for fishery science and comparative biology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562003" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562003" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sato, M.; Fukuda, K.; Kadota, M.; Makino-Itou, H.; Tatsumi, K.; Yamauchi, S.; Kuraku, S.</p>
        <p class="info">Score: 8.4, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562003' target='https://doi.org/10.1101/2023.10.16.562003'> 10.1101/2023.10.16.562003</a></p>
        <p class="abstract">Pacific saury (Cololabis saira) is a commercially important small pelagic fish species in Asian. In this study, we conducted the first-ever whole genome sequencing of this species, with single molecule, real-time (SMRT) sequencing technology. The obtained high-fidelity (HiFi) long-read sequence data, which amount to approximately 30 folds of its haploid genome size that was measured with quantitative PCR (1.17 Gb), were assembled into contigs. Scaffolding with Hi-C reads yielded a whole genome assembly containing 24 chromosome-scale sequences, with a scaffold N50 length of 47.7 Mb. Screening of repetitive elements including telomeric repeats was performed to characterize possible factors that need to be resolved towards telomere-to-telomere sequencing. The larger genome size than in medaka, a close relative in Beloniformes, is at least partly explained by larger repetitive element quantity, which is reflected in more abundant tRNAs, in the Pacific saury genome. Protein-coding regions was predicted using transcriptome data, which resulted in 22,274 components. Retrieval of Pacific saury homologs of aquaporin (AQP) genes known from other teleost fishes validated high completeness and continuity of the genome assembly. These resources are available at https://treethinkers.nig.ac.jp/saira/ and will assist various molecular-level studies in fishery science and comparative biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562270">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562270" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562270">
        <p class="paperTitle">Epigenetic context predicts gene expression variation and reproductive traits across genetically identical individuals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562270" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562270" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Webster, A. K.; Willis, J. H.; Johnson, E.; Sarkies, P.; Phillips, P. C.</p>
        <p class="info">Score: 10.7, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562270' target='https://doi.org/10.1101/2023.10.13.562270'> 10.1101/2023.10.13.562270</a></p>
        <p class="abstract">In recent decades, genome-wide association studies (GWAS) have been the major approach to understand the biological basis of individual differences in traits and diseases. However, GWAS approaches have proven to have limited predictive power to explain individual differences, particularly for complex traits and diseases in which environmental factors play a substantial role in their etiology. Indeed, individual differences persist even in genetically identical individuals, although fully separating genetic and environmental causation is difficult or impossible in most organisms. To understand the basis of individual differences in the absence of genetic differences, we measured two quantitative reproductive traits in 180 genetically identical young adult Caenorhabditis elegans roundworms in a shared environment and performed single-individual transcriptomics on each worm. We identified hundreds of genes for which expression variation was strongly associated with reproductive traits, some of which depended on prior environmental experience and some of which was random. Multiple small sets of genes together were highly predictive of reproductive traits across individuals, explaining on average over half and over a quarter of variation in the two traits. We manipulated mRNA levels of predictive genes using RNA interference to identify a set of causal genes, demonstrating the utility of this approach for both prediction and understanding underlying biology. Finally, we found that the chromatin environment of predictive genes was enriched for H3K27 trimethylation, suggesting that individual gene expression differences underlying critical traits may be driven in part by chromatin structure. Together, this work shows that individual differences in gene expression that arise independently of underlying genetic differences are both predictive and causal in shaping reproductive traits at levels that equal or exceed genetic variation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562284">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562284" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562284">
        <p class="paperTitle">Proteasome inhibition reprograms chromatin landscape in breast cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562284" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562284" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kinyamu, H. K.; Bennett, B. D.; Ward, J. M.; Archer, T. K.</p>
        <p class="info">Score: 5.1, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562284' target='https://doi.org/10.1101/2023.10.13.562284'> 10.1101/2023.10.13.562284</a></p>
        <p class="abstract">The 26S proteasome is the major protein degradation machinery in cells. Cancer cells use the proteasome to modulate gene expression networks that promote tumor growth. Proteasome inhibitors have emerged as effective cancer therapeutics, but how they work mechanistically remains unclear. Here, using integrative genomic analysis, we discovered unexpected reprogramming of the chromatin landscape and RNAPII transcription initiation in breast cancer cells treated with the proteasome inhibitor MG132. The cells acquired dynamic changes in chromatin accessibility at specific genomic loci termed Differentially Open Chromatin Regions (DOCRs). DOCRs with decreased accessibility were promoter proximal and exhibited unique chromatin architecture associated with divergent RNAPII transcription. Conversely, DOCRs with increased accessibility were primarily distal to transcription start sites and enriched in oncogenic super enhancers predominantly accessible in non-basal breast tumor subtypes. These findings describe the mechanisms by which the proteasome modulates the expression of gene networks intrinsic to breast cancer biology.

HighlightsO_LIProteasome inhibition uncovers de novo Differential Open Chromatin Regions (DOCRs) in breast cancer cells.
C_LIO_LIProteasome inhibitor sensitive promoters exhibit a distinctive chromatin architecture with discrete transcription initiation patterns.
C_LIO_LIProteasome inhibition reprograms accessibility of super enhancers.
C_LIO_LIProteasome inhibitor sensitive super enhancers distinguish basal from non-basal breast cancer subtypes.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561870">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561870" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561870">
        <p class="paperTitle">Non-coding variants impact cis-regulatory coordination in a cell type-specific manner</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561870" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561870" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pushkarev, O.; van Mierlo, G.; Kribelbauer, J. F.; Saelens, W.; Gardeux, V.; Deplancke, B.</p>
        <p class="info">Score: 12.8, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561870' target='https://doi.org/10.1101/2023.10.11.561870'> 10.1101/2023.10.11.561870</a></p>
        <p class="abstract">BACKGROUNDInteractions among cis-regulatory elements (CREs) play a crucial role in gene regulation. Various approaches have been developed to map these interactions genome-wide, including those relying on interindividual epigenomic variation to identify groups of covariable regulatory elements, referred to as chromatin modules (CMs). While CM mapping allows to investigate the relationship between chromatin modularity and gene expression, the computational principles used for CM identification vary in their application and outcomes.

RESULTSWe comprehensively evaluate and streamline existing CM mapping tools and present guidelines for optimal utilization of epigenome data from a diverse population of individuals to assess regulatory coordination across the human genome. We showcase the effectiveness of our recommended practices by analysing distinct cell types and demonstrate cell type-specificity of CRE interactions in CMs and their relevance for gene expression. Integration of genotype information revealed that many non-coding disease-associated variants affect the activity of CMs in a cell type-specific manner by affecting the binding of cell type-specific transcription factors. We provide example cases that illustrate in detail how CMs can be used to deconstruct GWAS loci, understand variable expression of cell surface receptors in immune cells and reveal how genetic variation can impact the expression of prognostic markers in chronic lymphocytic leukaemia.

CONCLUSIONSOur study presents an optimal strategy for CM mapping, and reveals how CMs capture the coordination of CREs and its impact on gene expression. Non-coding genetic variants can disrupt this coordination, and we highlight how this may lead to disease predisposition in a cell type-specific manner.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
