<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Comparative analysis of multiplexed in situ gene expression profiling technologies
Authors: Hartman, A.; Satija, R.
Score: 137.8, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135
The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Comparative analysis of multiplexed in situ gene expression profiling technologies
Authors: Hartman, A.; Satija, R.
Score: 137.8, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135
The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-24T10:40:04+00:00" />
<meta property="article:modified_time" content="2024-01-24T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Comparative analysis of multiplexed in situ gene expression profiling technologies
Authors: Hartman, A.; Satija, R.
Score: 137.8, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135
The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Comparative analysis of multiplexed in situ gene expression profiling technologies\nAuthors: Hartman, A.; Satija, R.\nScore: 137.8, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135\nThe burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets.",
  "keywords": [
    
  ],
  "articleBody": " Comparative analysis of multiplexed in situ gene expression profiling technologies\nAuthors: Hartman, A.; Satija, R.\nScore: 137.8, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135\nThe burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets. We find that standard sensitivity metrics, such as the number of unique molecules detected per cell, are not directly comparable across datasets due to substantial differences in the incidence of off-target molecular artifacts impacting specificity. To address these challenges, we explored various potential sources of molecular artifacts, developed novel metrics to control for them, and utilized these metrics to evaluate and compare different in situ technologies. Finally, we demonstrate how molecular false positives can seriously confound spatially-aware differential expression analysis, requiring caution in the interpretation of downstream results. Our analysis provides guidance for the selection, processing, and interpretation of in situ spatial technologies.\nAggregation of recount3 RNA-seq data improves the inference of consensus and tissue-specific gene co-expression networks.\nAuthors: Battle, A.; Ravichandran, P.; Parsana, P.; Keener, R.; Hansen, K.\nScore: 12.4, Published: 2024-01-22 DOI: 10.1101/2024.01.20.576447\nBackground: Gene co-expression networks (GCNs) describe relationships among expressed genes key to maintaining cellular identity and homeostasis. However, the small sample size of typical RNA-seq experiments which is several orders of magnitude fewer than the number of genes is too low to infer GCNs reliably. recount3, a publicly available dataset comprised of 316,443 uniformly processed human RNA-seq samples, provides an opportunity to improve power for accurate network reconstruction and obtain biological insight from the resulting networks. Results: We compared alternate aggregation strategies to identify an optimal workflow for GCN inference by data aggregation and inferred three consensus networks: a universal network, a non-cancer network, and a cancer network in addition to 27 tissue context-specific networks. Central network genes from our consensus networks were enriched for evolutionarily constrained genes and ubiquitous biological pathways, whereas central context-specific network genes included tissue-specific transcription factors and factorization based on the hubs led to clustering of related tissue contexts. We discovered that annotations corresponding to context-specific networks inferred from aggregated data were enriched for trait heritability beyond known functional genomic annotations and were significantly more enriched when we aggregated over a larger number of samples. Conclusion: This study outlines best practices for network GCN inference and evaluation by data aggregation. We recommend estimating and regressing confounders in each data set before aggregation and prioritizing large sample size studies for GCN reconstruction. Increased statistical power in inferring context-specific networks enabled the derivation of variant annotations that were enriched for concordant trait heritability independent of functional genomic annotations that are context-agnostic. While we observed strictly increasing held-out log-likelihood with data aggregation, we noted diminishing marginal improvements. Future directions aimed at alternate methods for estimating confounders and integrating orthogonal information from modalities such as Hi-C and ChIP-seq can further improve GCN inference.\nScalable ultra-high-throughput single-cell chromatin and RNA sequencing reveals gene regulatory dynamics linking macrophage polarization to autoimmune disease\nAuthors: Lobato-Moreno, S.; Yildiz, U.; Claringbould, A.; Servaas, N. H.; Vlachou, E. P.; Arnold, C.; Bauersachs, H. G.; Campos-Fornes, V.; Prummel, K. D.; Noh, K. M.; Marttinen, M.; Zaugg, J. B.\nScore: 47.9, Published: 2024-01-23 DOI: 10.1101/2023.12.26.573253\nEnhancers and transcription factors (TFs) are crucial in regulating cellular processes, including disease-associated cell states. Current multiomic technologies to study these elements in gene regulatory mechanisms lack multiplexing capability and scalability. Here, we present SUM-seq, a cost-effective, scalable Single-cell Ultra-high-throughput Multiomic sequencing method for co-assaying chromatin accessibility and gene expression in single nuclei. SUM-seq enables profiling hundreds of samples at the million cell scale and outperforms current high-throughput single-cell methods. We applied SUM-seq to dissect the gene regulatory mechanisms governing macrophage polarization and explored their link to traits from genome-wide association studies (GWAS). Our analyses confirmed known TFs orchestrating M1 and M2 macrophage programs, unveiled key regulators, and demonstrated extensive enhancer rewiring. Integration with GWAS data further pinpointed the impact of specific TFs on a set of immune traits. Notably, inferred enhancers regulated by the STAT1/STAT2/IRF9 (ISGF3) complex were enriched for genetic variants associated with Crohn's disease, ulcerative colitis and multiple sclerosis, and their target genes included known drug targets. This highlights the potential of SUM-seq for dissecting molecular disease mechanisms. SUM-seq offers a cost-effective, scalable solution for ultra-high-throughput single-cell multiomic sequencing, excelling in unraveling complex gene regulatory networks in cell differentiation, responses to perturbations, and disease studies.\nGenomic diversity and antimicrobial resistance in clinical Klebsiella pneumoniae isolates from tertiary hospitals in Southern Ghana\nAuthors: Mills, R. O.; Dadzie, I.; Le-Viet, T.; Baker, D. J.; Addy, H. P. K.; Akwetey, S. A.; Donkoh, I. E.; Quansah, E.; Semanshia, P. S.; Morgan, J.; Mensah, A.; Adade, N. E.; Ampah, E. O.; Owusu, E.; Mwintige, P.; Amoako, E. O.; Spadar, A.; Holt, K. E.; Foster-Nyarko, E.\nScore: 11.8, Published: 2024-01-22 DOI: 10.1101/2024.01.20.576413\nComprehensive data on the genomic epidemiology of hospital-associated Klebsiella pneumoniae in Ghana is scarce. This study sequenced 103 clinical K. pneumoniae isolates from five tertiary hospitals in Southern Ghana, predominantly from paediatric patients under five years (67/103, 65%), with the majority collected from urine (32/103, 31%) and blood (25/103, 24%) cultures. We employed Pathogenwatch for genotyping via Kaptive (K/O antigens) and Kleborate (antimicrobial resistance and hypervirulence) and determined clonal relationships using core-genome multilocus sequence typing (cgMLST). Among the 44 distinct sequence types (STs) detected, ST133 was the most common, comprising 23% of isolates (n=23/103). We discovered 27 different capsular (K) locus antigens and seven lipopolysaccharide (O) types; KL116 (28/103, 27%) and O1 (66/103, 64%) were the most prevalent. Single-linkage clustering highlighted the global spread of multidrug-resistant clones such as ST15, ST307, ST17, ST11, ST101, and ST48, with minimal allele differences (1-5) from publicly available genomes worldwide. Conversely, several isolates (n=17) constituted novel clonal groups and lacked close relatives among publicly available genomes, displaying unique genetic diversity within our study population. A significant proportion of isolates (88/103, 85%) carried resistance genes for three or more antibiotic classes, with the blaCTXM-15 gene present in 78% (n=80/103). Carbapenem resistance, predominantly due to blaOXA-181 and blaNDM-1 genes, was found in 10% (n=10/103) of the isolates. Yersiniabactin was the predominant acquired virulence trait, identified in 70% (n=72/103) of the isolates. Our findings reveal a complex genomic landscape of K. pneumoniae in Southern Ghana, underscoring the critical need for ongoing genomic surveillance to manage the substantial burden of antimicrobial resistance.\nNatural diversity of heat-induced transcription of retrotransposons in Arabidopsis thaliana\nAuthors: Xu, W.; Thieme, M.; Roulin, A.\nScore: 11.7, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575637\nTransposable elements (TEs) are major components of plant genomes, profoundly impacting the fitness of their hosts. However, technical bottlenecks have long hindered our mechanistic understanding of TEs. Using RNA-Seq and long-read sequencing with Oxford Nanopore Technologies direct cDNA sequencing, we analyzed the heat-induced transcription of TEs in three natural accessions of Arabidopsis thaliana (Cvi-0, Col-0, and Ler-1). In addition to the well- studied ONSEN retrotransposon family, we identified Copia-35 as a second heat-responsive retrotransposon family with particularly high activity in the relict accession Cvi-0. Our analysis revealed distinct expression patterns of individual TE copies and suggest different mechanisms regulating the GAG protein production in the ONSEN versus Copia-35 families. In addition, analogously to ONSEN, Copia-35 activation led to the upregulation of flanking genes such as AMUP9 and potentially to the quantitative modulation of flowering time. Unexpectedly, our results indicate that for both families, the upregulation of flanking genes is not directly initiated by transcription from their 3 LTRs. These findings highlight the inter- and intraspecific expressional diversity linked to retrotransposon activation under stress, providing insights into their potential roles in plant adaptation and evolution at elevated temperatures.\nA complex Plasmodium falciparum cryptotype circulating at low frequency across the African continent\nAuthors: Miotto, O.; Amambua-Ngwa, A.; Amenga-Etego, L. N.; Abdel Hamid, M. M.; Adam, I.; Aninagyei, E.; Apinjoh, T.; Awandare, G. A.; Bejon, P.; Bertin, G.; Bouyou-Akotet, M. K.; Claessens, A.; Conway, D.; D'Alessandro, U.; Diakite, M.; Djimde, A.; Dondorp, A. M.; Duffy, P.; Fairhurst, R. M.; Fanello, C. I.; Ghansah, A.; Ishengoma, D.; Lawniczak, M.; Maiga-Ascofare, O.; Auburn, S.; Rosanas-Urgell, A.; Wasakul, V.; White, N.; Almagro-Garcia, J.; Pearson, R.; Goncalves, S.; Ariani, C.; Bozdech, Z.; Hamilton, W. L.; Simpson, V.; Kwiatkowski, D. P.\nScore: 9.2, Published: 2024-01-22 DOI: 10.1101/2024.01.20.576496\nThe population structure of the malaria parasite Plasmodium falciparum can reveal underlying demographic and adaptive evolutionary processes. Here, we analyse population structure in 4,376 P. falciparum genomes from 21 countries across Africa. We identified a strongly differentiated cluster of parasites, comprising ~1.2% of samples analysed, geographically distributed over 13 countries across the continent. Members of this cluster, named AF1, carry a genetic background consisting of a large number of highly differentiated variants, rarely observed outside this cluster, at a multitude of genomic loci distributed across most chromosomes. At these loci, the AF1 haplotypes appear to have common ancestry, irrespective of the sampling location; outside the shared loci, however, AF1 members are genetically similar to their sympatric parasites. AF1 parasites sharing up to 23 genomic co-inherited regions were found in all major regions of Africa, at locations over 7,000 km apart. We coined the term cryptotype to describe a complex common background which is geographically widespread, but concealed by genomic regions of local origin. Most AF1 differentiated variants are functionally related, comprising structural variations and single nucleotide polymorphisms in components of the MSP1 complex and several other genes involved in interactions with red blood cells, including invasion and erythrocyte antigen export. We propose that AF1 parasites have adapted to some as yet unidentified evolutionary niche, by acquiring a complex compendium of interacting variants that rarely circulate separately in Africa. As the cryptotype spread across the continent, it appears to have been maintained mostly intact in spite of recombination events, suggesting a selective advantage. It is possible that other cryptotypes circulate in Africa, and new analysis methods may be needed to identify them.\nDORA: an interactive map for the visualization and analysis of ancient human DNA and associated data\nAuthors: Harris, K. D.; Greenbaum, G.\nScore: 13.9, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575663\nThe ability to sequence ancient genomes has revolutionized the way in which we study evolutionary history by providing access to the most important aspect of evolution -- time. Until recently, studying human demography, ecology, biology, and history using population genomic inference relied on contemporary genomic datasets. Over the past decade, the availability of human ancient DNA (aDNA) has increased rapidly, almost doubling every year, opening the way for spatiotemporal studies of ancient human populations. However, the multidimensionality of aDNA, with genotypes having temporal, spatial and genomic coordinates, and the need to integrate multiple sources of data, poses a challenge for developing meta-analyses pipelines. To address this challenge, we developed a publicly-available interactive tool, DORA, which integrates multiple data types, genomic and non-genomic, in a unified interface. This web-based tool allows users to browse sample metadata along with additional layers of information, such as population structure, climatic data, and unpublished samples. Users can then perform analyses on genotypes of these samples, or export sample subsets for external analyses. DORA integrates analyses and visualizations in a single intuitive interface, resolving the technical issues of combining datasets from different sources and formats, and allowing researchers to focus on analysis and the scientific questions that can be addressed through analysis of aDNA datasets.\nEpigenetic reprogramming driving successful and failed repair in acute kidney injury\nAuthors: Muto, Y.; Dixon, E. E.; Yoshimura, Y.; Ledru, N.; Kirita, Y.; Wu, H.; Humphreys, B. D.\nScore: 7.6, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576421\nAcute kidney injury (AKI) causes epithelial damage followed by subsequent repair. While successful repair restores kidney function, this process is often incomplete and can lead to chronic kidney disease (CKD) in a process called failed repair. To better understand the epigenetic reprogramming driving this AKI-to-CKD transition we generated a single nucleus multiomic atlas for the full mouse AKI time course, consisting of ~280,000 single nucleus transcriptomes and epigenomes. We reveal cell-specific dynamic alterations in gene regulatory landscapes reflecting especially activation of proinflammatory pathways. We further generated single nucleus multiomic data from four human AKI samples including validation by genome-wide identification of NF-kB binding sites. A regularized regression analysis identifies key regulators involved in both successful and failed repair cell fate, identifying the transcription factor CREB5 as a regulator of both successful and failed tubular repair that also drives proximal tubule cell proliferation after injury. Our interspecies multiomic approach provides a foundation to comprehensively understand cell states in AKI.\nMultiplex generation and single cell analysis of structural variants in a mammalian genome\nAuthors: Pinglay, S.; Lalanne, J.-B.; Daza, R. M.; Koeppel, J.; Li, X.; Lee, D. S.; Shendure, J.\nScore: 4.0, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576756\nThe functional consequences of structural variants (SVs) in mammalian genomes are challenging to study. This is due to several factors, including: 1) their numerical paucity relative to other forms of standing genetic variation such as single nucleotide variants (SNVs) and short insertions or deletions (indels); 2) the fact that a single SV can involve and potentially impact the function of more than one gene and/or cis regulatory element; and 3) the relative immaturity of methods to generate SVs, either randomly or in targeted fashion, in in vitro or in vivo model systems. Towards addressing some of these challenges, we developed Genome-Shuffle-seq, a straightforward method that enables the multiplex generation of several major forms of SVs (deletions, inversions, translocations) throughout a mammalian genome. Genome-Shuffle-seq is based on the random integration of 'shuffle cassettes' to the genome, wherein each shuffle cassette contains components that facilitate its site-specific recombination (SSR) with other integrated shuffle cassettes (via Cre-loxP), its mapping to a specific genomic location (via T7-mediated in vitro transcription or IVT), and its identification in single-cell RNA-seq (scRNA-seq) data (via T7-mediated in situ transcription or IST). In this proof-of-concept, we apply Genome-Shuffle-seq to induce and map thousands of genomic SVs in mouse embryonic stem cells (mESCs) in a single experiment. Induced SVs are rapidly depleted from the cellular population over time, possibly due to some combination of Cre-mediated toxicity and negative selection on the rearrangements themselves. We demonstrate that we can efficiently genotype which SVs are present in association with each of many single cell transcriptomes in scRNA-seq data. Finally, preliminary evidence suggests our method may be a powerful means of generating extrachromosomal circular DNAs (ecDNAs). Looking forward, we anticipate that Genome-Shuffle-seq may be broadly useful for the systematic exploration of the functional consequences of SVs on gene expression, the chromatin landscape, and 3D nuclear architecture. We further anticipate potential uses for in vitro modeling of ecDNAs, as well as in paving the path to a minimal mammalian genome.\nCellDemux: coherent genetic demultiplexing in single-cell and single-nuclei experiments.\nAuthors: Zoodsma, M.; Zhan, Q.; Kumar, S.; Botey-Bataller, J.; Li, W.; Zhou, L.; Alaswad, A.; Liu, Z.; Zhang, Z.; Zhang, B.; Xu, C.-J.; Li, Y.\nScore: 4.0, Published: 2024-01-20 DOI: 10.1101/2024.01.18.576186\nMultiplexed single-cell experiment designs are superior in terms of reduced batch effects, increased cost-effectiveness, throughput and statistical power. However, current computational strategies using genetics to demultiplex single-cell (sc) libraries are limited when applied to single-nuclei (sn) sequencing data (e.g., snATAC-seq and snMultiome). Here, we present CellDemux: a computational framework for genetic demultiplexing within and across data modalities, including single-cell, single-nuclei and paired snMultiome measurements. CellDemux uses a consensus approach, leveraging modality-specific tools to robustly identify non-empty oil droplets and singlets, which are subsequently demultiplexed to donors. Notable, CellDemux demonstrates good performance in demultiplexing snMultiome data and is generalizable to single modalities, i.e. snATAC-seq and sc/snRNA-seq libraries. We benchmark CellDemux on 187 genetically multiplexed libraries from 800 samples (scRNA-seq, snATAC-seq, CITE-seq and snMultiome), confidently identifying and assigning cells to 88% of donors. In paired snMultiome libraries, CellDemux achieves consistent demultiplexing across data modalities. Moreover, analysis of 38 snATAC libraries from 149 samples shows that CellDemux retains more genetically demultiplexed nuclei for downstream analyses compared to existing methods. In summary, CellDemux is a modular and robust framework that deconvolves donors from genetically multiplexed single-cell and single-nuclei RNA/ATAC/Multiome libraries.\n",
  "wordCount" : "2680",
  "inLanguage": "en",
  "datePublished": "2024-01-24T10:40:04Z",
  "dateModified": "2024-01-24T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 24, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575135" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575135">
        <p class="paperTitle">Comparative analysis of multiplexed in situ gene expression profiling technologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hartman, A.; Satija, R.</p>
        <p class="info">Score: 137.8, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575135' target='https://doi.org/10.1101/2024.01.11.575135'> 10.1101/2024.01.11.575135</a></p>
        <p class="abstract">The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets. We find that standard sensitivity metrics, such as the number of unique molecules detected per cell, are not directly comparable across datasets due to substantial differences in the incidence of off-target molecular artifacts impacting specificity. To address these challenges, we explored various potential sources of molecular artifacts, developed novel metrics to control for them, and utilized these metrics to evaluate and compare different in situ technologies. Finally, we demonstrate how molecular false positives can seriously confound spatially-aware differential expression analysis, requiring caution in the interpretation of downstream results. Our analysis provides guidance for the selection, processing, and interpretation of in situ spatial technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576447">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576447" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576447">
        <p class="paperTitle">Aggregation of recount3 RNA-seq data improves the inference of consensus and tissue-specific gene co-expression networks.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576447" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576447" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Battle, A.; Ravichandran, P.; Parsana, P.; Keener, R.; Hansen, K.</p>
        <p class="info">Score: 12.4, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576447' target='https://doi.org/10.1101/2024.01.20.576447'> 10.1101/2024.01.20.576447</a></p>
        <p class="abstract">Background: Gene co-expression networks (GCNs) describe relationships among expressed genes key to maintaining cellular identity and homeostasis. However, the small sample size of typical RNA-seq experiments which is several orders of magnitude fewer than the number of genes is too low to infer GCNs reliably. recount3, a publicly available dataset comprised of 316,443 uniformly processed human RNA-seq samples, provides an opportunity to improve power for accurate network reconstruction and obtain biological insight from the resulting networks. Results: We compared alternate aggregation strategies to identify an optimal workflow for GCN inference by data aggregation and inferred three consensus networks: a universal network, a non-cancer network, and a cancer network in addition to 27 tissue context-specific networks. Central network genes from our consensus networks were enriched for evolutionarily constrained genes and ubiquitous biological pathways, whereas central context-specific network genes included tissue-specific transcription factors and factorization based on the hubs led to clustering of related tissue contexts. We discovered that annotations corresponding to context-specific networks inferred from aggregated data were enriched for trait heritability beyond known functional genomic annotations and were significantly more enriched when we aggregated over a larger number of samples. Conclusion: This study outlines best practices for network GCN inference and evaluation by data aggregation. We recommend estimating and regressing confounders in each data set before aggregation and prioritizing large sample size studies for GCN reconstruction. Increased statistical power in inferring context-specific networks enabled the derivation of variant annotations that were enriched for concordant trait heritability independent of functional genomic annotations that are context-agnostic. While we observed strictly increasing held-out log-likelihood with data aggregation, we noted diminishing marginal improvements. Future directions aimed at alternate methods for estimating confounders and integrating orthogonal information from modalities such as Hi-C and ChIP-seq can further improve GCN inference.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573253">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573253" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573253">
        <p class="paperTitle">Scalable ultra-high-throughput single-cell chromatin and RNA sequencing reveals gene regulatory dynamics linking macrophage polarization to autoimmune disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573253" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573253" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lobato-Moreno, S.; Yildiz, U.; Claringbould, A.; Servaas, N. H.; Vlachou, E. P.; Arnold, C.; Bauersachs, H. G.; Campos-Fornes, V.; Prummel, K. D.; Noh, K. M.; Marttinen, M.; Zaugg, J. B.</p>
        <p class="info">Score: 47.9, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573253' target='https://doi.org/10.1101/2023.12.26.573253'> 10.1101/2023.12.26.573253</a></p>
        <p class="abstract">Enhancers and transcription factors (TFs) are crucial in regulating cellular processes, including disease-associated cell states. Current multiomic technologies to study these elements in gene regulatory mechanisms lack multiplexing capability and scalability. Here, we present SUM-seq, a cost-effective, scalable Single-cell Ultra-high-throughput Multiomic sequencing method for co-assaying chromatin accessibility and gene expression in single nuclei. SUM-seq enables profiling hundreds of samples at the million cell scale and outperforms current high-throughput single-cell methods. We applied SUM-seq to dissect the gene regulatory mechanisms governing macrophage polarization and explored their link to traits from genome-wide association studies (GWAS). Our analyses confirmed known TFs orchestrating M1 and M2 macrophage programs, unveiled key regulators, and demonstrated extensive enhancer rewiring. Integration with GWAS data further pinpointed the impact of specific TFs on a set of immune traits. Notably, inferred enhancers regulated by the STAT1/STAT2/IRF9 (ISGF3) complex were enriched for genetic variants associated with Crohn&#39;s disease, ulcerative colitis and multiple sclerosis, and their target genes included known drug targets. This highlights the potential of SUM-seq for dissecting molecular disease mechanisms. SUM-seq offers a cost-effective, scalable solution for ultra-high-throughput single-cell multiomic sequencing, excelling in unraveling complex gene regulatory networks in cell differentiation, responses to perturbations, and disease studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576413">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576413" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576413">
        <p class="paperTitle">Genomic diversity and antimicrobial resistance in clinical Klebsiella pneumoniae isolates from tertiary hospitals in Southern Ghana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576413" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576413" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mills, R. O.; Dadzie, I.; Le-Viet, T.; Baker, D. J.; Addy, H. P. K.; Akwetey, S. A.; Donkoh, I. E.; Quansah, E.; Semanshia, P. S.; Morgan, J.; Mensah, A.; Adade, N. E.; Ampah, E. O.; Owusu, E.; Mwintige, P.; Amoako, E. O.; Spadar, A.; Holt, K. E.; Foster-Nyarko, E.</p>
        <p class="info">Score: 11.8, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576413' target='https://doi.org/10.1101/2024.01.20.576413'> 10.1101/2024.01.20.576413</a></p>
        <p class="abstract">Comprehensive data on the genomic epidemiology of hospital-associated Klebsiella pneumoniae in Ghana is scarce. This study sequenced 103 clinical K. pneumoniae isolates from five tertiary hospitals in Southern Ghana, predominantly from paediatric patients under five years (67/103, 65%), with the majority collected from urine (32/103, 31%) and blood (25/103, 24%) cultures. We employed Pathogenwatch for genotyping via Kaptive (K/O antigens) and Kleborate (antimicrobial resistance and hypervirulence) and determined clonal relationships using core-genome multilocus sequence typing (cgMLST). Among the 44 distinct sequence types (STs) detected, ST133 was the most common, comprising 23% of isolates (n=23/103). We discovered 27 different capsular (K) locus antigens and seven lipopolysaccharide (O) types; KL116 (28/103, 27%) and O1 (66/103, 64%) were the most prevalent. Single-linkage clustering highlighted the global spread of multidrug-resistant clones such as ST15, ST307, ST17, ST11, ST101, and ST48, with minimal allele differences (1-5) from publicly available genomes worldwide. Conversely, several isolates (n=17) constituted novel clonal groups and lacked close relatives among publicly available genomes, displaying unique genetic diversity within our study population. A significant proportion of isolates (88/103, 85%) carried resistance genes for three or more antibiotic classes, with the blaCTXM-15 gene present in 78% (n=80/103). Carbapenem resistance, predominantly due to blaOXA-181 and blaNDM-1 genes, was found in 10% (n=10/103) of the isolates. Yersiniabactin was the predominant acquired virulence trait, identified in 70% (n=72/103) of the isolates. Our findings reveal a complex genomic landscape of K. pneumoniae in Southern Ghana, underscoring the critical need for ongoing genomic surveillance to manage the substantial burden of antimicrobial resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575637" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575637">
        <p class="paperTitle">Natural diversity of heat-induced transcription of retrotransposons in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, W.; Thieme, M.; Roulin, A.</p>
        <p class="info">Score: 11.7, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575637' target='https://doi.org/10.1101/2024.01.15.575637'> 10.1101/2024.01.15.575637</a></p>
        <p class="abstract">Transposable elements (TEs) are major components of plant genomes, profoundly impacting the fitness of their hosts. However, technical bottlenecks have long hindered our mechanistic understanding of TEs. Using RNA-Seq and long-read sequencing with Oxford Nanopore Technologies direct cDNA sequencing, we analyzed the heat-induced transcription of TEs in three natural accessions of Arabidopsis thaliana (Cvi-0, Col-0, and Ler-1). In addition to the well- studied ONSEN retrotransposon family, we identified Copia-35 as a second heat-responsive retrotransposon family with particularly high activity in the relict accession Cvi-0. Our analysis revealed distinct expression patterns of individual TE copies and suggest different mechanisms regulating the GAG protein production in the ONSEN versus Copia-35 families. In addition, analogously to ONSEN, Copia-35 activation led to the upregulation of flanking genes such as AMUP9 and potentially to the quantitative modulation of flowering time. Unexpectedly, our results indicate that for both families, the upregulation of flanking genes is not directly initiated by transcription from their 3 LTRs. These findings highlight the inter- and intraspecific expressional diversity linked to retrotransposon activation under stress, providing insights into their potential roles in plant adaptation and evolution at elevated temperatures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576496" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576496">
        <p class="paperTitle">A complex Plasmodium falciparum cryptotype circulating at low frequency across the African continent</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miotto, O.; Amambua-Ngwa, A.; Amenga-Etego, L. N.; Abdel Hamid, M. M.; Adam, I.; Aninagyei, E.; Apinjoh, T.; Awandare, G. A.; Bejon, P.; Bertin, G.; Bouyou-Akotet, M. K.; Claessens, A.; Conway, D.; D&#39;Alessandro, U.; Diakite, M.; Djimde, A.; Dondorp, A. M.; Duffy, P.; Fairhurst, R. M.; Fanello, C. I.; Ghansah, A.; Ishengoma, D.; Lawniczak, M.; Maiga-Ascofare, O.; Auburn, S.; Rosanas-Urgell, A.; Wasakul, V.; White, N.; Almagro-Garcia, J.; Pearson, R.; Goncalves, S.; Ariani, C.; Bozdech, Z.; Hamilton, W. L.; Simpson, V.; Kwiatkowski, D. P.</p>
        <p class="info">Score: 9.2, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576496' target='https://doi.org/10.1101/2024.01.20.576496'> 10.1101/2024.01.20.576496</a></p>
        <p class="abstract">The population structure of the malaria parasite Plasmodium falciparum can reveal underlying demographic and adaptive evolutionary processes. Here, we analyse population structure in 4,376 P. falciparum genomes from 21 countries across Africa. We identified a strongly differentiated cluster of parasites, comprising ~1.2% of samples analysed, geographically distributed over 13 countries across the continent. Members of this cluster, named AF1, carry a genetic background consisting of a large number of highly differentiated variants, rarely observed outside this cluster, at a multitude of genomic loci distributed across most chromosomes. At these loci, the AF1 haplotypes appear to have common ancestry, irrespective of the sampling location; outside the shared loci, however, AF1 members are genetically similar to their sympatric parasites. AF1 parasites sharing up to 23 genomic co-inherited regions were found in all major regions of Africa, at locations over 7,000 km apart. We coined the term cryptotype to describe a complex common background which is geographically widespread, but concealed by genomic regions of local origin. Most AF1 differentiated variants are functionally related, comprising structural variations and single nucleotide polymorphisms in components of the MSP1 complex and several other genes involved in interactions with red blood cells, including invasion and erythrocyte antigen export. We propose that AF1 parasites have adapted to some as yet unidentified evolutionary niche, by acquiring a complex compendium of interacting variants that rarely circulate separately in Africa. As the cryptotype spread across the continent, it appears to have been maintained mostly intact in spite of recombination events, suggesting a selective advantage. It is possible that other cryptotypes circulate in Africa, and new analysis methods may be needed to identify them.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575663" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575663">
        <p class="paperTitle">DORA: an interactive map for the visualization and analysis of ancient human DNA and associated data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harris, K. D.; Greenbaum, G.</p>
        <p class="info">Score: 13.9, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575663' target='https://doi.org/10.1101/2024.01.15.575663'> 10.1101/2024.01.15.575663</a></p>
        <p class="abstract">The ability to sequence ancient genomes has revolutionized the way in which we study evolutionary history by providing access to the most important aspect of evolution -- time. Until recently, studying human demography, ecology, biology, and history using population genomic inference relied on contemporary genomic datasets. Over the past decade, the availability of human ancient DNA (aDNA) has increased rapidly, almost doubling every year, opening the way for spatiotemporal studies of ancient human populations. However, the multidimensionality of aDNA, with genotypes having temporal, spatial and genomic coordinates, and the need to integrate multiple sources of data, poses a challenge for developing meta-analyses pipelines. To address this challenge, we developed a publicly-available interactive tool, DORA, which integrates multiple data types, genomic and non-genomic, in a unified interface. This web-based tool allows users to browse sample metadata along with additional layers of information, such as population structure, climatic data, and unpublished samples. Users can then perform analyses on genotypes of these samples, or export sample subsets for external analyses. DORA integrates analyses and visualizations in a single intuitive interface, resolving the technical issues of combining datasets from different sources and formats, and allowing researchers to focus on analysis and the scientific questions that can be addressed through analysis of aDNA datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576421">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576421" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576421">
        <p class="paperTitle">Epigenetic reprogramming driving successful and failed repair in acute kidney injury</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576421" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576421" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Muto, Y.; Dixon, E. E.; Yoshimura, Y.; Ledru, N.; Kirita, Y.; Wu, H.; Humphreys, B. D.</p>
        <p class="info">Score: 7.6, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576421' target='https://doi.org/10.1101/2024.01.20.576421'> 10.1101/2024.01.20.576421</a></p>
        <p class="abstract">Acute kidney injury (AKI) causes epithelial damage followed by subsequent repair. While successful repair restores kidney function, this process is often incomplete and can lead to chronic kidney disease (CKD) in a process called failed repair. To better understand the epigenetic reprogramming driving this AKI-to-CKD transition we generated a single nucleus multiomic atlas for the full mouse AKI time course, consisting of ~280,000 single nucleus transcriptomes and epigenomes. We reveal cell-specific dynamic alterations in gene regulatory landscapes reflecting especially activation of proinflammatory pathways. We further generated single nucleus multiomic data from four human AKI samples including validation by genome-wide identification of NF-kB binding sites. A regularized regression analysis identifies key regulators involved in both successful and failed repair cell fate, identifying the transcription factor CREB5 as a regulator of both successful and failed tubular repair that also drives proximal tubule cell proliferation after injury. Our interspecies multiomic approach provides a foundation to comprehensively understand cell states in AKI.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576756" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576756">
        <p class="paperTitle">Multiplex generation and single cell analysis of structural variants in a mammalian genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pinglay, S.; Lalanne, J.-B.; Daza, R. M.; Koeppel, J.; Li, X.; Lee, D. S.; Shendure, J.</p>
        <p class="info">Score: 4.0, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576756' target='https://doi.org/10.1101/2024.01.22.576756'> 10.1101/2024.01.22.576756</a></p>
        <p class="abstract">The functional consequences of structural variants (SVs) in mammalian genomes are challenging to study. This is due to several factors, including: 1) their numerical paucity relative to other forms of standing genetic variation such as single nucleotide variants (SNVs) and short insertions or deletions (indels); 2) the fact that a single SV can involve and potentially impact the function of more than one gene and/or cis regulatory element; and 3) the relative immaturity of methods to generate SVs, either randomly or in targeted fashion, in in vitro or in vivo model systems. Towards addressing some of these challenges, we developed Genome-Shuffle-seq, a straightforward method that enables the multiplex generation of several major forms of SVs (deletions, inversions, translocations) throughout a mammalian genome. Genome-Shuffle-seq is based on the random integration of &#39;shuffle cassettes&#39; to the genome, wherein each shuffle cassette contains components that facilitate its site-specific recombination (SSR) with other integrated shuffle cassettes (via Cre-loxP), its mapping to a specific genomic location (via T7-mediated in vitro transcription or IVT), and its identification in single-cell RNA-seq (scRNA-seq) data (via T7-mediated in situ transcription or IST). In this proof-of-concept, we apply Genome-Shuffle-seq to induce and map thousands of genomic SVs in mouse embryonic stem cells (mESCs) in a single experiment. Induced SVs are rapidly depleted from the cellular population over time, possibly due to some combination of Cre-mediated toxicity and negative selection on the rearrangements themselves. We demonstrate that we can efficiently genotype which SVs are present in association with each of many single cell transcriptomes in scRNA-seq data. Finally, preliminary evidence suggests our method may be a powerful means of generating extrachromosomal circular DNAs (ecDNAs). Looking forward, we anticipate that Genome-Shuffle-seq may be broadly useful for the systematic exploration of the functional consequences of SVs on gene expression, the chromatin landscape, and 3D nuclear architecture. We further anticipate potential uses for in vitro modeling of ecDNAs, as well as in paving the path to a minimal mammalian genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.18.576186">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.18.576186" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.18.576186">
        <p class="paperTitle">CellDemux: coherent genetic demultiplexing in single-cell and single-nuclei experiments.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.18.576186" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.18.576186" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zoodsma, M.; Zhan, Q.; Kumar, S.; Botey-Bataller, J.; Li, W.; Zhou, L.; Alaswad, A.; Liu, Z.; Zhang, Z.; Zhang, B.; Xu, C.-J.; Li, Y.</p>
        <p class="info">Score: 4.0, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.18.576186' target='https://doi.org/10.1101/2024.01.18.576186'> 10.1101/2024.01.18.576186</a></p>
        <p class="abstract">Multiplexed single-cell experiment designs are superior in terms of reduced batch effects, increased cost-effectiveness, throughput and statistical power. However, current computational strategies using genetics to demultiplex single-cell (sc) libraries are limited when applied to single-nuclei (sn) sequencing data (e.g., snATAC-seq and snMultiome). Here, we present CellDemux: a computational framework for genetic demultiplexing within and across data modalities, including single-cell, single-nuclei and paired snMultiome measurements. CellDemux uses a consensus approach, leveraging modality-specific tools to robustly identify non-empty oil droplets and singlets, which are subsequently demultiplexed to donors. Notable, CellDemux demonstrates good performance in demultiplexing snMultiome data and is generalizable to single modalities, i.e. snATAC-seq and sc/snRNA-seq libraries. We benchmark CellDemux on 187 genetically multiplexed libraries from 800 samples (scRNA-seq, snATAC-seq, CITE-seq and snMultiome), confidently identifying and assigning cells to 88% of donors. In paired snMultiome libraries, CellDemux achieves consistent demultiplexing across data modalities. Moreover, analysis of 38 snATAC libraries from 149 samples shows that CellDemux retains more genetically demultiplexed nuclei for downstream analyses compared to existing methods. In summary, CellDemux is a modular and robust framework that deconvolves donors from genetically multiplexed single-cell and single-nuclei RNA/ATAC/Multiome libraries.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
