<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Systematic reconstruction of molecular pathway signatures using scalable single-cell perturbation screens
Authors: Jiang, L.; Dalgarno, C.; Papalexi, E.; Mascio, I.; Wessels, H.-H.; Yun, H.; Iremadze, N.; Lithwick-Yanai, G.; Lipson, D.; Satija, R.
Score: 48.9, Published: 2024-01-30 DOI: 10.1101/2024.01.29.576933
Recent advancements in functional genomics have provided an unprecedented ability to measure diverse molecular modalities, but learning causal regulatory relationships from observational data remains challenging. Here, we leverage pooled genetic screens and single cell sequencing (i.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Systematic reconstruction of molecular pathway signatures using scalable single-cell perturbation screens
Authors: Jiang, L.; Dalgarno, C.; Papalexi, E.; Mascio, I.; Wessels, H.-H.; Yun, H.; Iremadze, N.; Lithwick-Yanai, G.; Lipson, D.; Satija, R.
Score: 48.9, Published: 2024-01-30 DOI: 10.1101/2024.01.29.576933
Recent advancements in functional genomics have provided an unprecedented ability to measure diverse molecular modalities, but learning causal regulatory relationships from observational data remains challenging. Here, we leverage pooled genetic screens and single cell sequencing (i." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-04T10:38:39+00:00" />
<meta property="article:modified_time" content="2024-02-04T10:38:39+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Systematic reconstruction of molecular pathway signatures using scalable single-cell perturbation screens
Authors: Jiang, L.; Dalgarno, C.; Papalexi, E.; Mascio, I.; Wessels, H.-H.; Yun, H.; Iremadze, N.; Lithwick-Yanai, G.; Lipson, D.; Satija, R.
Score: 48.9, Published: 2024-01-30 DOI: 10.1101/2024.01.29.576933
Recent advancements in functional genomics have provided an unprecedented ability to measure diverse molecular modalities, but learning causal regulatory relationships from observational data remains challenging. Here, we leverage pooled genetic screens and single cell sequencing (i."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Systematic reconstruction of molecular pathway signatures using scalable single-cell perturbation screens\nAuthors: Jiang, L.; Dalgarno, C.; Papalexi, E.; Mascio, I.; Wessels, H.-H.; Yun, H.; Iremadze, N.; Lithwick-Yanai, G.; Lipson, D.; Satija, R.\nScore: 48.9, Published: 2024-01-30 DOI: 10.1101/2024.01.29.576933\nRecent advancements in functional genomics have provided an unprecedented ability to measure diverse molecular modalities, but learning causal regulatory relationships from observational data remains challenging. Here, we leverage pooled genetic screens and single cell sequencing (i.",
  "keywords": [
    
  ],
  "articleBody": " Systematic reconstruction of molecular pathway signatures using scalable single-cell perturbation screens\nAuthors: Jiang, L.; Dalgarno, C.; Papalexi, E.; Mascio, I.; Wessels, H.-H.; Yun, H.; Iremadze, N.; Lithwick-Yanai, G.; Lipson, D.; Satija, R.\nScore: 48.9, Published: 2024-01-30 DOI: 10.1101/2024.01.29.576933\nRecent advancements in functional genomics have provided an unprecedented ability to measure diverse molecular modalities, but learning causal regulatory relationships from observational data remains challenging. Here, we leverage pooled genetic screens and single cell sequencing (i.e. Perturb-seq) to systematically identify the targets of signaling regulators in diverse biological contexts. We demonstrate how Perturb-seq is compatible with recent and commercially available advances in combinatorial indexing and next-generation sequencing, and perform more than 1,500 perturbations split across six cell lines and five biological signaling contexts. We introduce an improved computational framework (Mixscale) to address cellular variation in perturbation efficiency, alongside optimized statistical methods to learn differentially expressed gene lists and conserved molecular signatures. Finally, we demonstrate how our Perturb-seq derived gene lists can be used to precisely infer changes in signaling pathway activation for in-vivo and in-situ samples. Our work enhances our understanding of signaling regulators and their targets, and lays a computational framework towards the data-driven inference of an atlas of perturbation signatures.\nGiant polyketide synthase enzymes biosynthesize a giant marine polyether biotoxin.\nAuthors: Fallon, T. R.; Shende, V. V.; Wierzbicki, I. H.; Auber, R. P.; Gonzalez, D. J.; Wisecaver, J. H.; Moore, B. S.\nScore: 31.8, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577497\nPrymnesium parvum are harmful haptophyte algae that cause massive environmental fish-kills. Their polyketide polyether toxins, the prymnesins, are amongst the largest nonpolymeric compounds in nature, alongside structurally-related health-impacting \"red-tide\" polyether toxins whose biosynthetic origins have been an enigma for over 40 years. Here we report the PKZILLAs, massive P. parvum polyketide synthase (PKS) genes, whose existence and challenging genomic structure evaded prior detection. PKZILLA-1 and -2 encode giant protein products of 4.7 and 3.2 MDa with 140 and 99 enzyme domains, exceeding the largest known protein titin and all other known PKS systems. Their predicted polyene product matches the proposed pre-prymnesin precursor of the 90-carbon-backbone A-type prymnesins. This discovery establishes a model system for microalgal polyether biosynthesis and expands expectations of genetic and enzymatic size limits in biology.\nA Comprehensive Resource for Exploring Antiphage Defense: DefenseFinder Webservice, Wiki and Databases.\nAuthors: Tesson, F.; Planel, R.; Egorov, A. A.; Georjon, H.; Vaysset, H.; Brancotte, B.; Neron, B.; Mordret, E.; Atkinson, G. C.; Bernheim, A.; Cury, J.\nScore: 56.0, Published: 2024-01-26 DOI: 10.1101/2024.01.25.577194\nIn recent years, a vast number of novel antiphage defense mechanisms were uncovered. To facilitate the exploration of mechanistic, ecological, and evolutionary aspects related to antiphage defense systems, we released DefenseFinder in 2021 (Tesson et al., 2022). DefenseFinder is a bioinformatic program designed for the systematic identification of all known antiphage defense mechanisms. The initial release of DefenseFinder v1.0.0 included 60 systems. Over the past three years, the number of antiphage systems incorporated into DefenseFinder has grown to 152. The increasing number of known systems makes it a challenge to enter the field and makes the interpretation of detections of antiphage systems difficult. Moreover, the rapid development of sequence-based predictions of structures offers novel possibilities of analysis and should be easily available. To overcome these challenges, we present a hub of resources on defense systems including 1) a search tool (DefenseFinder as a web service) 2) a knowledge base, and 3) precomputed databases (results of DefenseFinder ran on RefSeq database, predicted structure database computed with AlphaFold). These pages can be freely accessed for users as a starting point on their journey to better understand a given system. We anticipate that these resources will foster the use of bioinformatics in the study of antiphage systems and will serve the community of researchers who study antiphage systems. This resource is available at: https://defensefinder.mdmlab.fr.\nGenetic diversity within diagnostic sputum samples is mirrored in the culture of Mycobacterium tuberculosis\nAuthors: Mariner-Llicer, C.; Goig, G. A.; Torres-Puente, M.; Vashakidze, S.; Villamayor, L.; Saavedra-Cervera, B.; Mambuque, E.; Khurtsilava, I.; Avaliani, Z.; Rosenthal, A.; Gabrielian, A.; Shurgaia, M.; Shubladze, N.; Garcia-Basteiro, A. L.; Lopez, M.; Comas, I.\nScore: 15.6, Published: 2024-01-31 DOI: 10.1101/2024.01.30.577772\nCulturing Mycobacterium tuberculosis (MTB) from tuberculosis cases is the basis for many research and clinical applications. Paradoxically, it is assumed to impose a diversity bottleneck, which, if true, would entail unexplored consequences. The alternative, culture-free sequencing from diagnostic samples, is a promising but challenging approach both to obtain and analyse the MTB genome from the complex sample. This study obtains high-quality genomes of sputum-culture pairs from two different settings after developing a workflow for sequencing from sputum and a tailored bioinformatics pipeline. Our approach reveals that 88% of variants called in culture-free sequencing analysis are false positives due to supplementary alignments, mostly in enriched-sputa samples. Overall, contrary to the bottleneck dogma, we identify a 97% variant agreement within sputum-culture pairs, with a high correlation also in the variants frequency (0.98). Our findings extrapolate to all publicly available data, thus demonstrating that in most cases culture accurately mirrors clinical samples.\nA comparison between low-cost library preparation kits for low coverage sequencing\nAuthors: Stewart, C. M.; Gibson, M. J.; Parsa, J.-Y.; Li, J. H.\nScore: 15.5, Published: 2024-01-31 DOI: 10.1101/2024.01.30.578044\nIn the fields of human health and agricultural research, low coverage whole-genome sequencing followed by imputation to a large haplotype reference panel has emerged as a cost-effective alternative to genotyping arrays for assaying large numbers of samples. However, a systematic comparison of library preparation methods tailored for low coverage sequencing remains absent in the existing literature. In this study, we evaluated one full sized kit from IDT and miniaturized and evaluated three Illumina-compatible library preparation kits--the KAPA HyperPlus kit (Roche), the DNA Prep kit (Illumina), and an IDT kit--using 96 human DNA samples. Metrics evaluated included imputation concordance with high-depth genotypes, coverage, duplication rates, time for library preparation, and additional optimization requirements. Despite slightly elevated duplication rates in IDT kits, we find that all four kits perform well in terms of imputation accuracy, with IDT kits being only marginally less performant than Illumina and Roche kits. Laboratory handling of the kits was similar: thus, the choice of a kit will largely depend on (1) existing or planned infrastructure, such as liquid handling capabilities, (2) whether a specific characteristic is desired, such as the use of full-length adapters, shorter processing times, or (3) use case, for instance, long vs short read sequencing. Our findings offer a comprehensive resource for both commercial and research workflows of low-cost library preparation methods suitable for high-throughput low coverage whole genome sequencing.\nIsoform-level profiling of m6A epitranscriptomic signatures in human brain\nAuthors: Gleeson, J.; Madugalle, S. U.; McLean, C.; Bredy, T. W.; De Paoli-Iseppi, R.; Clark, M. B.\nScore: 11.3, Published: 2024-01-31 DOI: 10.1101/2024.01.31.578088\nThe RNA modification N6-methyladenosine (m6A) is highly abundant in the human brain and implicated in neuropsychiatric and neurodegenerative disorders. However, most techniques for studying m6A cannot resolve modifications within RNA isoforms and we lack an isoform-level map of m6A sites in the brain. Profiling m6A within isoforms is therefore a critical step towards understanding the complex mechanisms that underpin brain function and disease. Oxford Nanopore direct RNA sequencing (DRS) can quantify isoform expression, modifications and polyA tail lengths, enabling simultaneous investigation of the transcriptome and epitranscriptome. We applied DRS to three post-mortem human brain regions: prefrontal cortex, caudate nucleus and cerebellum. We identified 57,000 m6A sites within 15,000 isoforms and estimated that \u003e27% of mRNA molecules contained an m6A modification. Our results revealed both isoform- and brain-region-specific patterning of m6A modifications and polyA tail lengths. The prefrontal cortex exhibited a distinctive profile of specifically modified isoforms enriched in excitatory neuron cell types and also had the highest proportion of previously unannotated m6A sites. A population of isoforms were hypermodified with m6A and were associated with excitatory neuron cell types in all three brain regions. We also discovered \u003e15k differentially expressed isoforms, \u003e2k differentially modified m6A sites and 566 isoforms with differential polyA lengths between brain regions. Our study demonstrates the utility of DRS for investigating multiple features of RNA isoforms in the brain and provides new insights into brain region specificity and functioning with implications for neurological development and disease.\n10x Genomics Gene Expression Flex is a powerful tool for single-cell transcriptomics of xenograft models\nAuthors: Llora-Batlle, O.; Farcas, A.; Fransen, D.; Floch, N.; Talbot, S.; Schwiening, A.; Bojko, L.; Calver, J.; Josipovic, N.; Lashuk, K.; Schueler, J.; Prodan, A.; Mooijman, D.; McDermott, U.; PERSIST-SEQ Consortium, Score: 11.0, Published: 2024-01-29 DOI: 10.1101/2024.01.25.577066\nThe 10x Genomics Gene Expression Flex protocol allows profiling of fixed or frozen material, greatly simplifying the logistics of sample collection, storage and transfer prior to single -cell sequencing. The method makes single-cell transcriptomics possible for existing fresh-frozen or FFPE tissue samples, but also facilitates the logistics of the sampling process, allowing instant preservation of samples. The technology relies on species-specific probes available for human and mouse. Nevertheless, processing of patient-derived (PDX) or cell line (CDX) xenografts, which contain mixed human and mouse cells, is currently not supported by this protocol due to the high degree of homology between the probe sets. Here we show that it is feasible to simultaneously profile populations containing both human and mouse cells by mixing the transcriptome probe sets of both species. Cellranger outputs a count table for each of the species allowing evaluation of the performance of the different probe sets. Cross-reactive probes are greatly outperformed by the specific probe hybridizations leading to a clear difference in the recovery of UMIs and unique genes per cell. Furthermore, we developed a pipeline that removes cross-reactive signal from the data and provides species-specific count tables for further downstream analysis. Hence, the 10x Genomics Gene Expression Flex protocol can be used to process xenograft samples without the need for separation of human and mouse cells by flow sorting and allows analysis of the human and mouse single-cell transcriptome from each sample. We anticipate it will be increasingly used for single-cell sequencing of cancer cell line and patient-derived xenografts, facilitating the preservation of the samples and allowing the interrogation of both the (human) xenograft and the (mouse) tumor microenvironment at single-cell resolution.\nRandomizing the human genome by engineering recombination between repeat elements\nAuthors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.\nScore: 103.2, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576745\nWhile protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent. To bridge this gap, we have developed a toolbox to create deletions, inversions, translocations, and extrachromosomal circular DNA at scale by highly multiplexed insertion of recombinase recognition sites into repetitive sequences with CRISPR prime editing. Using this strategy, we derived stable human cell lines with several thousand clonal insertions, the highest number of novel sequences inserted into single human genomes. Subsequent recombinase induction generated an average of more than one hundred megabase-sized rearrangements per cell, and thousands across the whole population. The ability to detect rearrangements as they are generated and to track their abundance over time allowed us to measure the selection pressures acting on different types of structural changes. We observed a consolidation towards shorter variants that preferentially delete growth-inhibiting genes and a depletion of translocations. We isolated and characterized 21 clones with multiple recombinase-induced rearrangements. These included viable haploid clones with deletions that span hundreds of kilobases as well as triploid HEK293T clones with aneuploidies and fold back chromosomes. We mapped the impact of these genetic changes on gene expression to decipher how structural variants affect gene regulation. The genome scrambling strategy developed here makes it possible to delete megabases of sequence, move sequences between and within chromosomes, and implant regulatory elements into new contexts which will shed light on the genome organization principles of humans and other species.\nGene-specific RNA homeostasis revealed by perturbation of the NuA4/Tip60 acetyltransferase complex\nAuthors: Forouzanfar, F.; Plassard, D.; Furst, A.; Moreno, D.; Oliveira, K. A.; Reina-San-Martin, B.; Tora, L.; Molina, N.; Mendoza, M.\nScore: 7.9, Published: 2024-01-31 DOI: 10.1101/2024.01.30.577960\nTranscript buffering entails the reciprocal modulation of mRNA synthesis and degradation rates, ensuring a constant RNA concentration amidst changes in cellular conditions. While an increasing body of research supports a global, non-sequence-specific linkage between mRNA synthesis and degradation, the underlying mechanisms remain elusive. To explore this, we investigated alterations in RNA metabolism following the acute depletion of TIP60/KAT5, the transcriptional coactivator and acetyltransferase subunit of the NuA4 complex, in mouse embryonic stem cells. By combining RNA sequencing of nuclear, cytoplasmic, and newly synthesised transcript fractions with biophysical modelling, we show that TIP60 activates transcription of numerous genes, with substantially fewer genes undergoing transcriptional repression. Surprisingly, specific RNA species transcription changes triggered by TIP60 depletion were counterbalanced by compensatory adjustments in RNA export and/or stability within the nucleus and in RNA stability within the cytoplasm. These discoveries imply that transcript buffering operates on a gene-specific level and suggest that cells continually monitor RNA molecule counts in nuclear and cytoplasmic compartments to maintain cellular homeostasis.\nEfficient CRISPR genome editing and integrative genomic analyses reveal the mosaicism of Cas-induced mutations and pleiotropic effects of scarlet gene in an emerging model system\nAuthors: Xu, S.; Neupane, S.; Wang, H.; Pham, T.; Snyman, M.; Huynh, T.; Wang, L.\nScore: 6.5, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577787\nDespite the revolutionary impacts of CRISPR-Cas gene editing systems, the effective and widespread use of CRISPR technologies in emerging model organisms still faces significant challenges. These include the inefficiency in generating heritable mutations at the organismal level, limited knowledge about the genomic consequences of gene editing, and an inadequate understanding of the inheritance patterns of CRISPR-Cas-induced mutations. This study addresses these issues by 1) developing an efficient microinjection delivery method for CRISPR editing in the microcrustacean Daphnia pulex; 2) assessing the editing efficiency of Cas9 and Cas12a nucleases, examining mutation inheritance patterns, and analyzing the local and global mutation spectrum in the scarlet mutants; and 3) investigating the transcriptomes of scarlet mutants to understand the pleiotropic effects of scarlet underlying their swimming behavior changes. Our reengineered CRISPR microinjection method results in efficient biallelic editing with both nucleases. While indels are dominant in Cas-induced mutations, a few on-site large deletions (\u003e1kb) are observed, most likely caused by microhomology-mediated end joining repair. Knock-in of a stop codon cassette to the scarlet locus was successful, despite complex induced mutations surrounding the target site. Moreover, extensive germline mosaicism exists in some mutants, which unexpectedly produce different phenotypes/genotypes in their asexual progenies. Lastly, our transcriptomic analyses unveil significant gene expression changes associated with scarlet knock-out and altered swimming behavior in mutants, including several genes (e.g., NMDA1, ABAT, CNTNAP2) involved in human neurodegenerative diseases. This study expands our understanding of the dynamics of gene editing in the tractable model organism Daphnia and highlights its promising potential as a neurological disease model.\n",
  "wordCount" : "2490",
  "inLanguage": "en",
  "datePublished": "2024-02-04T10:38:39Z",
  "dateModified": "2024-02-04T10:38:39Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 4, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.576933">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.576933" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.576933">
        <p class="paperTitle">Systematic reconstruction of molecular pathway signatures using scalable single-cell perturbation screens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.576933" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.576933" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, L.; Dalgarno, C.; Papalexi, E.; Mascio, I.; Wessels, H.-H.; Yun, H.; Iremadze, N.; Lithwick-Yanai, G.; Lipson, D.; Satija, R.</p>
        <p class="info">Score: 48.9, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.576933' target='https://doi.org/10.1101/2024.01.29.576933'> 10.1101/2024.01.29.576933</a></p>
        <p class="abstract">Recent advancements in functional genomics have provided an unprecedented ability to measure diverse molecular modalities, but learning causal regulatory relationships from observational data remains challenging. Here, we leverage pooled genetic screens and single cell sequencing (i.e. Perturb-seq) to systematically identify the targets of signaling regulators in diverse biological contexts. We demonstrate how Perturb-seq is compatible with recent and commercially available advances in combinatorial indexing and next-generation sequencing, and perform more than 1,500 perturbations split across six cell lines and five biological signaling contexts. We introduce an improved computational framework (Mixscale) to address cellular variation in perturbation efficiency, alongside optimized statistical methods to learn differentially expressed gene lists and conserved molecular signatures. Finally, we demonstrate how our Perturb-seq derived gene lists can be used to precisely infer changes in signaling pathway activation for in-vivo and in-situ samples. Our work enhances our understanding of signaling regulators and their targets, and lays a computational framework towards the data-driven inference of an  atlas of perturbation signatures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577497">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577497" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577497">
        <p class="paperTitle">Giant polyketide synthase enzymes biosynthesize a giant marine polyether biotoxin.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577497" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577497" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fallon, T. R.; Shende, V. V.; Wierzbicki, I. H.; Auber, R. P.; Gonzalez, D. J.; Wisecaver, J. H.; Moore, B. S.</p>
        <p class="info">Score: 31.8, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577497' target='https://doi.org/10.1101/2024.01.29.577497'> 10.1101/2024.01.29.577497</a></p>
        <p class="abstract">Prymnesium parvum are harmful haptophyte algae that cause massive environmental fish-kills. Their polyketide polyether toxins, the prymnesins, are amongst the largest nonpolymeric compounds in nature, alongside structurally-related health-impacting &#34;red-tide&#34; polyether toxins whose biosynthetic origins have been an enigma for over 40 years. Here we report the  PKZILLAs, massive P. parvum polyketide synthase (PKS) genes, whose existence and challenging genomic structure evaded prior detection. PKZILLA-1 and -2 encode giant protein products of 4.7 and 3.2 MDa with 140 and 99 enzyme domains, exceeding the largest known protein titin and all other known PKS systems. Their predicted polyene product matches the proposed pre-prymnesin precursor of the 90-carbon-backbone A-type prymnesins. This discovery establishes a model system for microalgal polyether biosynthesis and expands expectations of genetic and enzymatic size limits in biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577194">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577194" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577194">
        <p class="paperTitle">A Comprehensive Resource for Exploring Antiphage Defense: DefenseFinder Webservice, Wiki and Databases.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577194" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577194" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tesson, F.; Planel, R.; Egorov, A. A.; Georjon, H.; Vaysset, H.; Brancotte, B.; Neron, B.; Mordret, E.; Atkinson, G. C.; Bernheim, A.; Cury, J.</p>
        <p class="info">Score: 56.0, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577194' target='https://doi.org/10.1101/2024.01.25.577194'> 10.1101/2024.01.25.577194</a></p>
        <p class="abstract">In recent years, a vast number of novel antiphage defense mechanisms were uncovered. To facilitate the exploration of mechanistic, ecological, and evolutionary aspects related to antiphage defense systems, we released DefenseFinder in 2021 (Tesson et al., 2022). DefenseFinder is a bioinformatic program designed for the systematic identification of all known antiphage defense mechanisms. The initial release of DefenseFinder v1.0.0 included 60 systems. Over the past three years, the number of antiphage systems incorporated into DefenseFinder has grown to 152. The increasing number of known systems makes it a challenge to enter the field and makes the interpretation of detections of antiphage systems difficult. Moreover, the rapid development of sequence-based predictions of structures offers novel possibilities of analysis and should be easily available. To overcome these challenges, we present a hub of resources on defense systems including 1) a search tool (DefenseFinder as a web service) 2) a knowledge base, and 3) precomputed databases (results of DefenseFinder ran on RefSeq database, predicted structure database computed with AlphaFold). These pages can be freely accessed for users as a starting point on their journey to better understand a given system. We anticipate that these resources will foster the use of bioinformatics in the study of antiphage systems and will serve the community of researchers who study antiphage systems. This resource is available at: https://defensefinder.mdmlab.fr.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577772">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577772" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577772">
        <p class="paperTitle">Genetic diversity within diagnostic sputum samples is mirrored in the culture of Mycobacterium tuberculosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577772" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577772" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mariner-Llicer, C.; Goig, G. A.; Torres-Puente, M.; Vashakidze, S.; Villamayor, L.; Saavedra-Cervera, B.; Mambuque, E.; Khurtsilava, I.; Avaliani, Z.; Rosenthal, A.; Gabrielian, A.; Shurgaia, M.; Shubladze, N.; Garcia-Basteiro, A. L.; Lopez, M.; Comas, I.</p>
        <p class="info">Score: 15.6, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577772' target='https://doi.org/10.1101/2024.01.30.577772'> 10.1101/2024.01.30.577772</a></p>
        <p class="abstract">Culturing Mycobacterium tuberculosis (MTB) from tuberculosis cases is the basis for many research and clinical applications. Paradoxically, it is assumed to impose a diversity bottleneck, which, if true, would entail unexplored consequences. The alternative, culture-free sequencing from diagnostic samples, is a promising but challenging approach both to obtain and analyse the MTB genome from the complex sample. This study obtains high-quality genomes of sputum-culture pairs from two different settings after developing a workflow for sequencing from sputum and a tailored bioinformatics pipeline. Our approach reveals that 88% of variants called in culture-free sequencing analysis are false positives due to supplementary alignments, mostly in enriched-sputa samples. Overall, contrary to the bottleneck dogma, we identify a 97% variant agreement within sputum-culture pairs, with a high correlation also in the variants frequency (0.98). Our findings extrapolate to all publicly available data, thus demonstrating that in most cases culture accurately mirrors clinical samples.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.578044">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.578044" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.578044">
        <p class="paperTitle">A comparison between low-cost library preparation kits for low coverage sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.578044" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.578044" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stewart, C. M.; Gibson, M. J.; Parsa, J.-Y.; Li, J. H.</p>
        <p class="info">Score: 15.5, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.578044' target='https://doi.org/10.1101/2024.01.30.578044'> 10.1101/2024.01.30.578044</a></p>
        <p class="abstract">In the fields of human health and agricultural research, low coverage whole-genome sequencing followed by imputation to a large haplotype reference panel has emerged as a cost-effective alternative to genotyping arrays for assaying large numbers of samples. However, a systematic comparison of library preparation methods tailored for low coverage sequencing remains absent in the existing literature. In this study, we evaluated one full sized kit from IDT and miniaturized and evaluated three Illumina-compatible library preparation kits--the KAPA HyperPlus kit (Roche), the DNA Prep kit (Illumina), and an IDT kit--using 96 human DNA samples. Metrics evaluated included imputation concordance with high-depth genotypes, coverage, duplication rates, time for library preparation, and additional optimization requirements. Despite slightly elevated duplication rates in IDT kits, we find that all four kits perform well in terms of imputation accuracy, with IDT kits being only marginally less performant than Illumina and Roche kits. Laboratory handling of the kits was similar: thus, the choice of a kit will largely depend on (1) existing or planned infrastructure, such as liquid handling capabilities, (2) whether a specific characteristic is desired, such as the use of full-length adapters, shorter processing times, or (3) use case, for instance, long vs short read sequencing. Our findings offer a comprehensive resource for both commercial and research workflows of low-cost library preparation methods suitable for high-throughput low coverage whole genome sequencing.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578088">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578088" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578088">
        <p class="paperTitle">Isoform-level profiling of m6A epitranscriptomic signatures in human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578088" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578088" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gleeson, J.; Madugalle, S. U.; McLean, C.; Bredy, T. W.; De Paoli-Iseppi, R.; Clark, M. B.</p>
        <p class="info">Score: 11.3, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578088' target='https://doi.org/10.1101/2024.01.31.578088'> 10.1101/2024.01.31.578088</a></p>
        <p class="abstract">The RNA modification N6-methyladenosine (m6A) is highly abundant in the human brain and implicated in neuropsychiatric and neurodegenerative disorders. However, most techniques for studying m6A cannot resolve modifications within RNA isoforms and we lack an isoform-level map of m6A sites in the brain. Profiling m6A within isoforms is therefore a critical step towards understanding the complex mechanisms that underpin brain function and disease. Oxford Nanopore direct RNA sequencing (DRS) can quantify isoform expression, modifications and polyA tail lengths, enabling simultaneous investigation of the transcriptome and epitranscriptome. We applied DRS to three post-mortem human brain regions: prefrontal cortex, caudate nucleus and cerebellum. We identified 57,000 m6A sites within 15,000 isoforms and estimated that &gt;27% of mRNA molecules contained an m6A modification. Our results revealed both isoform- and brain-region-specific patterning of m6A modifications and polyA tail lengths. The prefrontal cortex exhibited a distinctive profile of specifically modified isoforms enriched in excitatory neuron cell types and also had the highest proportion of previously unannotated m6A sites. A population of isoforms were hypermodified with m6A and were associated with excitatory neuron cell types in all three brain regions. We also discovered &gt;15k differentially expressed isoforms, &gt;2k differentially modified m6A sites and 566 isoforms with differential polyA lengths between brain regions. Our study demonstrates the utility of DRS for investigating multiple features of RNA isoforms in the brain and provides new insights into brain region specificity and functioning with implications for neurological development and disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577066">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577066" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577066">
        <p class="paperTitle">10x Genomics Gene Expression Flex is a powerful tool for single-cell transcriptomics of xenograft models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577066" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577066" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Llora-Batlle, O.; Farcas, A.; Fransen, D.; Floch, N.; Talbot, S.; Schwiening, A.; Bojko, L.; Calver, J.; Josipovic, N.; Lashuk, K.; Schueler, J.; Prodan, A.; Mooijman, D.; McDermott, U.; PERSIST-SEQ Consortium,  </p>
        <p class="info">Score: 11.0, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577066' target='https://doi.org/10.1101/2024.01.25.577066'> 10.1101/2024.01.25.577066</a></p>
        <p class="abstract">The 10x Genomics Gene Expression Flex protocol allows profiling of fixed or frozen material, greatly simplifying the logistics of sample collection, storage and transfer prior to single -cell sequencing. The method makes single-cell transcriptomics possible for existing fresh-frozen or FFPE tissue samples, but also facilitates the logistics of the sampling process, allowing instant preservation of samples. The technology relies on species-specific probes available for human and mouse. Nevertheless, processing of patient-derived (PDX) or cell line (CDX) xenografts, which contain mixed human and mouse cells, is currently not supported by this protocol due to the high degree of homology between the probe sets. Here we show that it is feasible to simultaneously profile populations containing both human and mouse cells by mixing the transcriptome probe sets of both species. Cellranger outputs a count table for each of the species allowing evaluation of the performance of the different probe sets. Cross-reactive probes are greatly outperformed by the specific probe hybridizations leading to a clear difference in the recovery of UMIs and unique genes per cell. Furthermore, we developed a pipeline that removes cross-reactive signal from the data and provides species-specific count tables for further downstream analysis. Hence, the 10x Genomics Gene Expression Flex protocol can be used to process xenograft samples without the need for separation of human and mouse cells by flow sorting and allows analysis of the human and mouse single-cell transcriptome from each sample. We anticipate it will be increasingly used for single-cell sequencing of cancer cell line and patient-derived xenografts, facilitating the preservation of the samples and allowing the interrogation of both the (human) xenograft and the (mouse) tumor microenvironment at single-cell resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576745">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576745" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576745">
        <p class="paperTitle">Randomizing the human genome by engineering recombination between repeat elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576745" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576745" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.</p>
        <p class="info">Score: 103.2, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576745' target='https://doi.org/10.1101/2024.01.22.576745'> 10.1101/2024.01.22.576745</a></p>
        <p class="abstract">While protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent. To bridge this gap, we have developed a toolbox to create deletions, inversions, translocations, and extrachromosomal circular DNA at scale by highly multiplexed insertion of recombinase recognition sites into repetitive sequences with CRISPR prime editing. Using this strategy, we derived stable human cell lines with several thousand clonal insertions, the highest number of novel sequences inserted into single human genomes. Subsequent recombinase induction generated an average of more than one hundred megabase-sized rearrangements per cell, and thousands across the whole population. The ability to detect rearrangements as they are generated and to track their abundance over time allowed us to measure the selection pressures acting on different types of structural changes. We observed a consolidation towards shorter variants that preferentially delete growth-inhibiting genes and a depletion of translocations. We isolated and characterized 21 clones with multiple recombinase-induced rearrangements. These included viable haploid clones with deletions that span hundreds of kilobases as well as triploid HEK293T clones with aneuploidies and fold back chromosomes. We mapped the impact of these genetic changes on gene expression to decipher how structural variants affect gene regulation. The genome scrambling strategy developed here makes it possible to delete megabases of sequence, move sequences between and within chromosomes, and implant regulatory elements into new contexts which will shed light on the genome organization principles of humans and other species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577960">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577960" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577960">
        <p class="paperTitle">Gene-specific RNA homeostasis revealed by perturbation of the NuA4/Tip60 acetyltransferase complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577960" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577960" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Forouzanfar, F.; Plassard, D.; Furst, A.; Moreno, D.; Oliveira, K. A.; Reina-San-Martin, B.; Tora, L.; Molina, N.; Mendoza, M.</p>
        <p class="info">Score: 7.9, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577960' target='https://doi.org/10.1101/2024.01.30.577960'> 10.1101/2024.01.30.577960</a></p>
        <p class="abstract">Transcript buffering entails the reciprocal modulation of mRNA synthesis and degradation rates, ensuring a constant RNA concentration amidst changes in cellular conditions. While an increasing body of research supports a global, non-sequence-specific linkage between mRNA synthesis and degradation, the underlying mechanisms remain elusive. To explore this, we investigated alterations in RNA metabolism following the acute depletion of TIP60/KAT5, the transcriptional coactivator and acetyltransferase subunit of the NuA4 complex, in mouse embryonic stem cells. By combining RNA sequencing of nuclear, cytoplasmic, and newly synthesised transcript fractions with biophysical modelling, we show that TIP60 activates transcription of numerous genes, with substantially fewer genes undergoing transcriptional repression. Surprisingly, specific RNA species transcription changes triggered by TIP60 depletion were counterbalanced by compensatory adjustments in RNA export and/or stability within the nucleus and in RNA stability within the cytoplasm. These discoveries imply that transcript buffering operates on a gene-specific level and suggest that cells continually monitor RNA molecule counts in nuclear and cytoplasmic compartments to maintain cellular homeostasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577787">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577787" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577787">
        <p class="paperTitle">Efficient CRISPR genome editing and integrative genomic analyses reveal the mosaicism of Cas-induced mutations and pleiotropic effects of scarlet gene in an emerging model system</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577787" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577787" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, S.; Neupane, S.; Wang, H.; Pham, T.; Snyman, M.; Huynh, T.; Wang, L.</p>
        <p class="info">Score: 6.5, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577787' target='https://doi.org/10.1101/2024.01.29.577787'> 10.1101/2024.01.29.577787</a></p>
        <p class="abstract">Despite the revolutionary impacts of CRISPR-Cas gene editing systems, the effective and widespread use of CRISPR technologies in emerging model organisms still faces significant challenges. These include the inefficiency in generating heritable mutations at the organismal level, limited knowledge about the genomic consequences of gene editing, and an inadequate understanding of the inheritance patterns of CRISPR-Cas-induced mutations. This study addresses these issues by 1) developing an efficient microinjection delivery method for CRISPR editing in the microcrustacean Daphnia pulex; 2) assessing the editing efficiency of Cas9 and Cas12a nucleases, examining mutation inheritance patterns, and analyzing the local and global mutation spectrum in the scarlet mutants; and 3) investigating the transcriptomes of scarlet mutants to understand the pleiotropic effects of scarlet underlying their swimming behavior changes. Our reengineered CRISPR microinjection method results in efficient biallelic editing with both nucleases. While indels are dominant in Cas-induced mutations, a few on-site large deletions (&gt;1kb) are observed, most likely caused by microhomology-mediated end joining repair. Knock-in of a stop codon cassette to the scarlet locus was successful, despite complex induced mutations surrounding the target site. Moreover, extensive germline mosaicism exists in some mutants, which unexpectedly produce different phenotypes/genotypes in their asexual progenies. Lastly, our transcriptomic analyses unveil significant gene expression changes associated with scarlet knock-out and altered swimming behavior in mutants, including several genes (e.g., NMDA1, ABAT, CNTNAP2) involved in human neurodegenerative diseases. This study expands our understanding of the dynamics of gene editing in the tractable model organism Daphnia and highlights its promising potential as a neurological disease model.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
