<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation
Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.
Score: 120.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582
Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation
Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.
Score: 120.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582
Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-10T10:36:49+00:00" />
<meta property="article:modified_time" content="2023-09-10T10:36:49+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation
Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.
Score: 120.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582
Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation\nAuthors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.\nScore: 120.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582\nSequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence.",
  "keywords": [
    
  ],
  "articleBody": " Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation\nAuthors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.\nScore: 120.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582\nSequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence. Using statistics derived from Borzois predicted coverage, we isolate and accurately score variant effects across multiple layers of regulation, including transcription, splicing, and polyadenylation. Evaluated on QTLs, Borzoi is competitive with, and often outperforms, state-of-the-art models trained on individual regulatory functions. By applying attribution methods to the derived statistics, we extract cis-regulatory patterns driving RNA expression and post-transcriptional regulation in normal tissues. The wide availability of RNA-seq data across species, conditions, and assays profiling specific aspects of regulation emphasizes the potential of this approach to decipher the mapping from DNA sequence to regulatory function.\nIn vivo perturb-seq of cancer and immune cells dissects oncologic drivers and therapy response\nAuthors: Liu, S. J.; Pak, J.; Zou, C.; Casey-Clyde, T.; Borah, A.; Wu, D.; Seo, K.; O'Loughlin, T.; Lim, D.; Ozawa, T.; Berger, M.; Weiss, W.; Raleigh, D. R.; Gilbert, L. A.\nScore: 27.9, Published: 2023-09-03 DOI: 10.1101/2023.09.01.555831\nGenetic perturbation screens with single cell readouts have enabled rich phenotyping of gene function and regulatory networks. These approaches have been challenging in vivo, especially in adult disease models such as cancer, which include mixtures of malignant and microenvironment cells. Here, we develop a multiplex in vivo perturb-seq CRISPRi platform for single cell genetic screens in cancer and immune cells that leverages intracranial convection enhanced delivery of sgRNA libraries into models of glioblastoma (GBM), a fatal brain cancer. Our results reveal in vivo specific rewiring of genetic dependencies in response to radiotherapy, a potent yet non-curative therapy for GBM, as well as alterations of ligand-receptor interactions between microenvironment cells. In sum, we demonstrate the utility of multiplexed perturb-seq for in vivo single cell dissection of adult tissue biology across multiple cell types in the context of therapeutic stimuli.\nRepeat turnover meets stable chromosomes: repetitive DNA sequences mark speciation and gene pool boundaries in sugar beet and wild beets\nAuthors: Schmidt, N.; Sielemann, K.; Breitenbach, S.; Fuchs, J.; Pucker, B.; Weisshaar, B.; HoltgraÌˆwe, D.; Heitkam, T.\nScore: 27.5, Published: 2023-09-06 DOI: 10.1101/2023.09.01.555723\nBackgroundSugar beet (Beta vulgaris subsp. vulgaris) and its crop wild relatives share a base chromosome number of nine and similar chromosome morphologies. Yet, interspecific breeding is impeded by chromosome and sequence divergence that is still not fully understood. Since repetitive DNA sequences represent the fastest evolving parts of the genome, they likely impact genomic variability and contribute to the separation of beet gene pools. Hence, we investigated if innovations and losses in the repeatome can be linked to chromosomal differentiation and speciation. ResultsWe traced genome- and chromosome-wide evolution across sugar beet and twelve wild beets comprising all sections of the beet genera Beta and Patellifolia. For this, we combined data from short and long read sequencing, flow cytometry, and cytogenetics to build a comprehensive data framework for our beet panel that spans the complete scale from DNA sequence to chromosome up to the genome. Genome sizes and repeat profiles reflect the separation of the beet species into three gene pools. These gene pools harbor repeats with contrasting evolutionary patterns: We identified section- and species-specific repeat emergences and losses, e.g. of the retrotransposons causal for genome expansions in the section Corollinae/Nanae. Since most genomic variability was found in the satellite DNAs, we focused on tracing the 19 beetSat families across the three beet sections/genera. These taxa harbor evidence for contrasting strategies in repeat evolution, leading to contrasting satellite DNA profiles and fundamentally different centromere architectures, ranging from chromosomal uniformity in Beta and Patellifolia species to the formation of patchwork chromosomes in Corollinae/Nanae species. ConclusionsWe show that repetitive DNA sequences are causal for genome size expansion and contraction across the beet genera, providing insights into the genomic underpinnings of beet speciation. Satellite DNAs in particular vary considerably among beet taxa, leading to the evolution of distinct chromosomal setups. These differences likely contribute to the barriers in beet breeding between the three gene pools. Thus, with their isokaryotypic chromosome sets, beet genomes present an ideal system for studying the link between repeats, genome variability, and chromosomal differentiation/evolution and provide a theoretical basis for understanding barriers in crop breeding.\nCrossing host boundaries: the evolutionary drivers and correlates of viral host jumps\nAuthors: Tan, C. C. S.; van Dorp, L.; Balloux, F.\nScore: 22.7, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555953\nMost emerging and re-emerging infectious diseases stem from viruses that naturally circulate in non-human vertebrates. When these viruses cross over into humans, they can cause disease outbreaks and epidemics. While zoonotic host jumps have been extensively studied from an ecological perspective, little attention has gone into characterising the evolutionary drivers and correlates underlying these events. To address this gap, we harnessed the entirety of publicly available viral genomic data, employing a comprehensive suite of network and phylogenetic analyses. We address a series of questions concerning the evolutionary mechanisms underpinning viral host jumps. Notably, we challenge conventional assumptions about the directionality of host jumps, demonstrating that humans are as much a source as a sink for viral spillover events, insofar we infer more viruses to have jumped from humans to other animals, than from animals to humans. Moreover, we demonstrate heightened evolution in viral lineages that involve putative host jumps. We further observe that the mutational threshold associated with a host jump is lower for viruses with broad host ranges. Finally, we show that the genomic targets of natural selection upon a successful host jump vary across different viral families with either structural or auxiliary genes being the prime targets of selection. Collectively, our results illuminate some of the evolutionary drivers underlying viral host jumps that may contribute to mitigating viral threats across species boundaries.\nAIRR-C Human IG Reference Sets: curated sets of immunoglobulin heavy and light chain germline genes\nAuthors: Collins, A. M.; Ohlin, M.; Corcoran, M.; Heather, J. M.; Ralph, D.; Law, M.; Martinez-Barnetche, J.; Ye, J.; Richardson, E.; Gibson, W. S.; Rodriguez, O. L.; Peres, A.; Yaari, G.; Watson, C. T.; Lees, W. D.\nScore: 21.1, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555348\nAnalysis of an individuals immunoglobulin (IG) gene repertoire requires the use of high-quality germline gene Reference Sets. The Adaptive Immune Receptor Repertoire-Community (AIRR-C) Reference Sets have been developed to include only human IG heavy and light chain alleles that have been confirmed by evidence from multiple high-quality sources. By including only those alleles with a high level of support, including some new sequences that currently lack official names, AIRR-seq analysis will have greater accuracy and studies of the evolution of immunoglobulin genes, their allelic variants and the expressed immune repertoire will be facilitated. Although containing less than half the previously recognised IG alleles (e.g. just 198 IGHV sequences), the Reference Sets eliminated erroneous calls and provided excellent coverage when tested on a set of repertoires from 99 individuals comprising over 4 million V(D)J rearrangements. To improve AIRR-seq analysis, some alleles have been extended to deal with short 3 or 5 truncations that can lead them to be overlooked by alignment utilities. To avoid other challenges for analysis programs, exact paralogs (e.g. IGHV1-69*01 and IGHV1-69D*01) are only represented once in each set, though alternative sequence names are noted in accompanying metadata. The Reference Sets also include novel alleles: 8 IGHV alleles, 2 IGKV alleles and 5 IGLV alleles. The version-tracked AIRR-C Reference Sets are freely available at the OGRDB website (https://ogrdb.airr-community.org/germline_sets/Human) and will be regularly updated to include newly-observed and previously-reported sequences that can be confirmed by new high-quality data.\nA General Approach to Adjusting Genetic Studies for Assortative Mating\nAuthors: Bilghese, M.; Manansala, R.; Jaishankar, D.; Jala, J.; Benjamin, D. J.; Kimball, M.; Auer, P. L.; Livermore, M. A.; Turley, P.\nScore: 19.3, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555983\nThe effects of assortative mating (AM) on estimates from genetic studies has been receiving increasing attention in recent years. We extend existing AM theory to more general models of sorting and conclude that correct theory-based AM adjustments require knowledge of complicated, unknown historical sorting patterns. We propose a simple, general-purpose approach using polygenic indexes (PGIs). Our approach can estimate the fraction of genetic variance and genetic correlation that is driven by AM. Our approach is less effective when applied to Mendelian randomization (MR) studies for two reasons: AM can induce a form of selection bias in MR studies that remains after our adjustment; and, in the MR context, the adjustment is particularly sensitive to PGI estimation error. Using data from the UK Biobank, we find that AM inflates genetic correlation estimates between health traits and education by 14% on average. Our results suggest caution in interpreting genetic correlations or MR estimates for traits subject to AM.\nGenetics of cell-type-specific post-transcriptional gene regulation during human neurogenesis\nAuthors: AyguÌˆn, N.; Krupa, O.; Mory, J.; Le, B.; Valone, J.; Liang, D.; Love, M. I.; Stein, J. L.\nScore: 14.2, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555019\nThe function of some genetic variants associated with brain-relevant traits has been explained through colocalization with expression quantitative trait loci (eQTL) conducted in bulk post-mortem adult brain tissue. However, many brain-trait associated loci have unknown cellular or molecular function. These genetic variants may exert context-specific function on different molecular phenotypes including post-transcriptional changes. Here, we identified genetic regulation of RNA-editing and alternative polyadenylation (APA), within a cell-type-specific population of human neural progenitors and neurons. More RNA-editing and isoforms utilizing longer polyadenylation sequences were observed in neurons, likely due to higher expression of genes encoding the proteins mediating these post-transcriptional events. We also detected hundreds of cell-type-specific editing quantitative trait loci (edQTLs) and alternative polyadenylation QTLs (apaQTLs). We found colocalizations of a neuron edQTL in CCDC88A with educational attainment and a progenitor apaQTL in EP300 with schizophrenia, suggesting genetically mediated post-transcriptional regulation during brain development lead to differences in brain function.\nComparative pangenomic insights into the distinct evolution of virulence factors among grapevine trunk pathogens\nAuthors: Garcia, J.; Morales-Cruz, A.; Cochetel, N.; Minio, A.; Figueroa-Balderas, R.; Rolshausen, P.; Baumgartner, K.; Cantu, D.\nScore: 11.3, Published: 2023-09-04 DOI: 10.1101/2023.09.03.555958\nThe permanent organs of grapevines (V. vinifera L.), like other woody perennials, are colonized by various unrelated pathogenic ascomycete fungi secreting cell wall-degrading enzymes and phytotoxic secondary metabolites that contribute to host damage and disease symptoms. Trunk pathogens differ in the symptoms they induce and the extent and speed of damage. Isolates of the same species often display a wide virulence range, even within the same vineyard. This study focuses on Eutypa lata, Neofusicoccum parvum, and Phaeoacremonium minimum, causal agents of Eutypa dieback, Botryosphaeria dieback, and Esca, respectively. We sequenced fifty isolates from viticulture regions worldwide and built nucleotide-level, reference-free pangenomes for each species. Through examining genomic diversity and pangenome structure, we analyzed intraspecific conservation and variability of putative virulence factors, focusing on functions under positive selection, and recent gene-family dynamics of contraction and expansion. Our findings reveal contrasting distributions of putative virulence factors in the core, dispensable, and private genomes of each pangenome. For example, CAZymes were prevalent in the core genomes of each pangenome, whereas biosynthetic gene clusters were prevalent in the dispensable genomes of E. lata and P. minimum. The dispensable fractions were also enriched in Gypsy transposable elements and virulence factors under positive selection (polyketide synthases genes in E. lata and P. minimum glycosyltransferases in N. parvum). Our findings underscore the complexity of the genomic architecture in each species and provide insights into their adaptive strategies, enhancing our understanding of the underlying mechanisms of virulence.\nThe genome and population genomics of allopolyploid Coffea arabica reveal the diversification history of modern coffee cultivars\nAuthors: Salojarvi, J.; Rambani, A.; Yu, Z.; Guyot, R.; Strickler, S.; Lepelley, M.; Wang, C.; Rajaraman, S.; Rastas, P.; Zheng, C.; Munoz, D. S.; Meidanis, J.; Paschoal, A. R.; Bawin, Y.; Krabbenhoft, T.; Wang, Z. Q.; Fleck, S.; Aussel, R.; Bellanger, L.; Charpagne, A.; Fournier, C.; Kassam, M.; Lefebvre, G.; Metairon, S.; Moine, D.; Rigoreau, M.; Stolte, J.; Hamon, P.; Couturon, E.; Tranchant-Dubreuil, C.; Mukherjee, M.; Lan, T.; Engelhardt, J.; Stadler, P.; DeLemos, S. C.; Suzuki, S. I.; Sumirat, U.; ChingMan, W.; Dauchot, N.; Orozco-Arias, S.; Garavito, A.; Kiwuka, C.; Musoli, P.; Nalukenge, A.; Gu\nScore: 11.2, Published: 2023-09-06 DOI: 10.1101/2023.09.06.556570\nCoffea arabica, an allotetraploid hybrid of C. eugenioides and C. canephora, is the source of approximately 60% of coffee products worldwide. Cultivated accessions have undergone several population bottlenecks resulting in low genetic diversity. We present chromosome-level assemblies of a di-haploid C. arabica accession and modern representatives of its diploid progenitors, C. eugenioides and C. canephora. The three species exhibit largely conserved genome structures between diploid parents and descendant subgenomes, which show a mosaic pattern of dominance, similar to other polyploid crop species. Resequencing of 39 wild and cultivated accessions suggests a founding polyploidy event [~]610,000 years ago, followed by several subsequent bottlenecks, including a population split [~]30.5 kya and a period of migration between Arabica populations until [~]8.9 kya. Analysis of lines historically introgressed with C. canephora highlights loci that may contribute to their superior pathogen resistance and lay the groundwork for future genomics-based breeding of C. arabica.\nTelomere-to-telomere Schizosaccharomyces japonicus genome assembly reveals hitherto unknown genome features\nAuthors: Etherington, G. J.; Nieduszynski, C.; Oliferenko, S.; Wu, P.-S.; Uhlmann, F.\nScore: 9.2, Published: 2023-09-06 DOI: 10.1101/2023.09.04.556195\nSchizosaccharomyces japonicus belongs to the single-genus class Schizosaccharomycetes, otherwise known as fission yeasts. As part of a composite model system with its widely studied S. pombe sister species, S. japonicus has provided critical insights into the workings and the evolution of cell biological mechanisms. Furthermore, its divergent biology makes S. japonicus a valuable model organism in its own right. However, the currently available short-read genome assembly contains gaps and has been unable to resolve centromeres and other repeat-rich chromosomal regions. Here we present a telomere-to-telomere long-read genome assembly of the S japonicus genome. This includes the three megabase-length chromosomes, with centromeres hundreds of kilobases long, rich in 5S ribosomal RNAs, transfer RNAs, long terminal repeats, and short repeats. We identify a gene-sparse region on chromosome 2 that resembles a 331 kb centromeric duplication. We revise the genome size of S. japonicus to at least 16.6 Mb and possibly up to 18.12 Mb, at least 30% larger than previous estimates. Our whole genome assembly will support the growing S. japonicus research community and facilitate research in new directions, including centromere and DNA repeat evolution, and yeast comparative genomics.\n",
  "wordCount" : "2433",
  "inLanguage": "en",
  "datePublished": "2023-09-10T10:36:49Z",
  "dateModified": "2023-09-10T10:36:49Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on September 10, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555582">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555582" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555582">
        <p class="paperTitle">Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555582" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555582" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.</p>
        <p class="info">Score: 120.3, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555582' target='https://doi.org/10.1101/2023.08.30.555582'> 10.1101/2023.08.30.555582</a></p>
        <p class="abstract">Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence. Using statistics derived from Borzois predicted coverage, we isolate and accurately score variant effects across multiple layers of regulation, including transcription, splicing, and polyadenylation. Evaluated on QTLs, Borzoi is competitive with, and often outperforms, state-of-the-art models trained on individual regulatory functions. By applying attribution methods to the derived statistics, we extract cis-regulatory patterns driving RNA expression and post-transcriptional regulation in normal tissues. The wide availability of RNA-seq data across species, conditions, and assays profiling specific aspects of regulation emphasizes the potential of this approach to decipher the mapping from DNA sequence to regulatory function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555831">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555831" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555831">
        <p class="paperTitle">In vivo perturb-seq of cancer and immune cells dissects oncologic drivers and therapy response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555831" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555831" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, S. J.; Pak, J.; Zou, C.; Casey-Clyde, T.; Borah, A.; Wu, D.; Seo, K.; O&#39;Loughlin, T.; Lim, D.; Ozawa, T.; Berger, M.; Weiss, W.; Raleigh, D. R.; Gilbert, L. A.</p>
        <p class="info">Score: 27.9, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555831' target='https://doi.org/10.1101/2023.09.01.555831'> 10.1101/2023.09.01.555831</a></p>
        <p class="abstract">Genetic perturbation screens with single cell readouts have enabled rich phenotyping of gene function and regulatory networks. These approaches have been challenging in vivo, especially in adult disease models such as cancer, which include mixtures of malignant and microenvironment cells. Here, we develop a multiplex in vivo perturb-seq CRISPRi platform for single cell genetic screens in cancer and immune cells that leverages intracranial convection enhanced delivery of sgRNA libraries into models of glioblastoma (GBM), a fatal brain cancer. Our results reveal in vivo specific rewiring of genetic dependencies in response to radiotherapy, a potent yet non-curative therapy for GBM, as well as alterations of ligand-receptor interactions between microenvironment cells. In sum, we demonstrate the utility of multiplexed perturb-seq for in vivo single cell dissection of adult tissue biology across multiple cell types in the context of therapeutic stimuli.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555723">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555723" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555723">
        <p class="paperTitle">Repeat turnover meets stable chromosomes: repetitive DNA sequences mark speciation and gene pool boundaries in sugar beet and wild beets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555723" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555723" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmidt, N.; Sielemann, K.; Breitenbach, S.; Fuchs, J.; Pucker, B.; Weisshaar, B.; HoltgraÌˆwe, D.; Heitkam, T.</p>
        <p class="info">Score: 27.5, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555723' target='https://doi.org/10.1101/2023.09.01.555723'> 10.1101/2023.09.01.555723</a></p>
        <p class="abstract">BackgroundSugar beet (Beta vulgaris subsp. vulgaris) and its crop wild relatives share a base chromosome number of nine and similar chromosome morphologies. Yet, interspecific breeding is impeded by chromosome and sequence divergence that is still not fully understood. Since repetitive DNA sequences represent the fastest evolving parts of the genome, they likely impact genomic variability and contribute to the separation of beet gene pools. Hence, we investigated if innovations and losses in the repeatome can be linked to chromosomal differentiation and speciation.

ResultsWe traced genome- and chromosome-wide evolution across sugar beet and twelve wild beets comprising all sections of the beet genera Beta and Patellifolia. For this, we combined data from short and long read sequencing, flow cytometry, and cytogenetics to build a comprehensive data framework for our beet panel that spans the complete scale from DNA sequence to chromosome up to the genome.

Genome sizes and repeat profiles reflect the separation of the beet species into three gene pools. These gene pools harbor repeats with contrasting evolutionary patterns: We identified section- and species-specific repeat emergences and losses, e.g. of the retrotransposons causal for genome expansions in the section Corollinae/Nanae. Since most genomic variability was found in the satellite DNAs, we focused on tracing the 19 beetSat families across the three beet sections/genera. These taxa harbor evidence for contrasting strategies in repeat evolution, leading to contrasting satellite DNA profiles and fundamentally different centromere architectures, ranging from chromosomal uniformity in Beta and Patellifolia species to the formation of patchwork chromosomes in Corollinae/Nanae species.

ConclusionsWe show that repetitive DNA sequences are causal for genome size expansion and contraction across the beet genera, providing insights into the genomic underpinnings of beet speciation. Satellite DNAs in particular vary considerably among beet taxa, leading to the evolution of distinct chromosomal setups. These differences likely contribute to the barriers in beet breeding between the three gene pools. Thus, with their isokaryotypic chromosome sets, beet genomes present an ideal system for studying the link between repeats, genome variability, and chromosomal differentiation/evolution and provide a theoretical basis for understanding barriers in crop breeding.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555953">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555953" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555953">
        <p class="paperTitle">Crossing host boundaries: the evolutionary drivers and correlates of viral host jumps</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555953" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555953" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tan, C. C. S.; van Dorp, L.; Balloux, F.</p>
        <p class="info">Score: 22.7, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555953' target='https://doi.org/10.1101/2023.09.01.555953'> 10.1101/2023.09.01.555953</a></p>
        <p class="abstract">Most emerging and re-emerging infectious diseases stem from viruses that naturally circulate in non-human vertebrates. When these viruses cross over into humans, they can cause disease outbreaks and epidemics. While zoonotic host jumps have been extensively studied from an ecological perspective, little attention has gone into characterising the evolutionary drivers and correlates underlying these events. To address this gap, we harnessed the entirety of publicly available viral genomic data, employing a comprehensive suite of network and phylogenetic analyses. We address a series of questions concerning the evolutionary mechanisms underpinning viral host jumps. Notably, we challenge conventional assumptions about the directionality of host jumps, demonstrating that humans are as much a source as a sink for viral spillover events, insofar we infer more viruses to have jumped from humans to other animals, than from animals to humans. Moreover, we demonstrate heightened evolution in viral lineages that involve putative host jumps. We further observe that the mutational threshold associated with a host jump is lower for viruses with broad host ranges. Finally, we show that the genomic targets of natural selection upon a successful host jump vary across different viral families with either structural or auxiliary genes being the prime targets of selection. Collectively, our results illuminate some of the evolutionary drivers underlying viral host jumps that may contribute to mitigating viral threats across species boundaries.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555348">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555348" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555348">
        <p class="paperTitle">AIRR-C Human IG Reference Sets: curated sets of immunoglobulin heavy and light chain germline genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555348" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555348" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Collins, A. M.; Ohlin, M.; Corcoran, M.; Heather, J. M.; Ralph, D.; Law, M.; Martinez-Barnetche, J.; Ye, J.; Richardson, E.; Gibson, W. S.; Rodriguez, O. L.; Peres, A.; Yaari, G.; Watson, C. T.; Lees, W. D.</p>
        <p class="info">Score: 21.1, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555348' target='https://doi.org/10.1101/2023.09.01.555348'> 10.1101/2023.09.01.555348</a></p>
        <p class="abstract">Analysis of an individuals immunoglobulin (IG) gene repertoire requires the use of high-quality germline gene Reference Sets. The Adaptive Immune Receptor Repertoire-Community (AIRR-C) Reference Sets have been developed to include only human IG heavy and light chain alleles that have been confirmed by evidence from multiple high-quality sources. By including only those alleles with a high level of support, including some new sequences that currently lack official names, AIRR-seq analysis will have greater accuracy and studies of the evolution of immunoglobulin genes, their allelic variants and the expressed immune repertoire will be facilitated. Although containing less than half the previously recognised IG alleles (e.g. just 198 IGHV sequences), the Reference Sets eliminated erroneous calls and provided excellent coverage when tested on a set of repertoires from 99 individuals comprising over 4 million V(D)J rearrangements. To improve AIRR-seq analysis, some alleles have been extended to deal with short 3 or 5 truncations that can lead them to be overlooked by alignment utilities. To avoid other challenges for analysis programs, exact paralogs (e.g. IGHV1-69*01 and IGHV1-69D*01) are only represented once in each set, though alternative sequence names are noted in accompanying metadata. The Reference Sets also include novel alleles: 8 IGHV alleles, 2 IGKV alleles and 5 IGLV alleles. The version-tracked AIRR-C Reference Sets are freely available at the OGRDB website (https://ogrdb.airr-community.org/germline_sets/Human) and will be regularly updated to include newly-observed and previously-reported sequences that can be confirmed by new high-quality data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555983">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555983" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555983">
        <p class="paperTitle">A General Approach to Adjusting Genetic Studies for Assortative Mating</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555983" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555983" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bilghese, M.; Manansala, R.; Jaishankar, D.; Jala, J.; Benjamin, D. J.; Kimball, M.; Auer, P. L.; Livermore, M. A.; Turley, P.</p>
        <p class="info">Score: 19.3, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555983' target='https://doi.org/10.1101/2023.09.01.555983'> 10.1101/2023.09.01.555983</a></p>
        <p class="abstract">The effects of assortative mating (AM) on estimates from genetic studies has been receiving increasing attention in recent years. We extend existing AM theory to more general models of sorting and conclude that correct theory-based AM adjustments require knowledge of complicated, unknown historical sorting patterns. We propose a simple, general-purpose approach using polygenic indexes (PGIs). Our approach can estimate the fraction of genetic variance and genetic correlation that is driven by AM. Our approach is less effective when applied to Mendelian randomization (MR) studies for two reasons: AM can induce a form of selection bias in MR studies that remains after our adjustment; and, in the MR context, the adjustment is particularly sensitive to PGI estimation error. Using data from the UK Biobank, we find that AM inflates genetic correlation estimates between health traits and education by 14% on average. Our results suggest caution in interpreting genetic correlations or MR estimates for traits subject to AM.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555019">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555019" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555019">
        <p class="paperTitle">Genetics of cell-type-specific post-transcriptional gene regulation during human neurogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555019" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555019" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: AyguÌˆn, N.; Krupa, O.; Mory, J.; Le, B.; Valone, J.; Liang, D.; Love, M. I.; Stein, J. L.</p>
        <p class="info">Score: 14.2, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555019' target='https://doi.org/10.1101/2023.08.30.555019'> 10.1101/2023.08.30.555019</a></p>
        <p class="abstract">The function of some genetic variants associated with brain-relevant traits has been explained through colocalization with expression quantitative trait loci (eQTL) conducted in bulk post-mortem adult brain tissue. However, many brain-trait associated loci have unknown cellular or molecular function. These genetic variants may exert context-specific function on different molecular phenotypes including post-transcriptional changes. Here, we identified genetic regulation of RNA-editing and alternative polyadenylation (APA), within a cell-type-specific population of human neural progenitors and neurons. More RNA-editing and isoforms utilizing longer polyadenylation sequences were observed in neurons, likely due to higher expression of genes encoding the proteins mediating these post-transcriptional events. We also detected hundreds of cell-type-specific editing quantitative trait loci (edQTLs) and alternative polyadenylation QTLs (apaQTLs). We found colocalizations of a neuron edQTL in CCDC88A with educational attainment and a progenitor apaQTL in EP300 with schizophrenia, suggesting genetically mediated post-transcriptional regulation during brain development lead to differences in brain function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.03.555958">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.03.555958" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.03.555958">
        <p class="paperTitle">Comparative pangenomic insights into the distinct evolution of virulence factors among grapevine trunk pathogens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.03.555958" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.03.555958" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garcia, J.; Morales-Cruz, A.; Cochetel, N.; Minio, A.; Figueroa-Balderas, R.; Rolshausen, P.; Baumgartner, K.; Cantu, D.</p>
        <p class="info">Score: 11.3, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.03.555958' target='https://doi.org/10.1101/2023.09.03.555958'> 10.1101/2023.09.03.555958</a></p>
        <p class="abstract">The permanent organs of grapevines (V. vinifera L.), like other woody perennials, are colonized by various unrelated pathogenic ascomycete fungi secreting cell wall-degrading enzymes and phytotoxic secondary metabolites that contribute to host damage and disease symptoms. Trunk pathogens differ in the symptoms they induce and the extent and speed of damage. Isolates of the same species often display a wide virulence range, even within the same vineyard. This study focuses on Eutypa lata, Neofusicoccum parvum, and Phaeoacremonium minimum, causal agents of Eutypa dieback, Botryosphaeria dieback, and Esca, respectively. We sequenced fifty isolates from viticulture regions worldwide and built nucleotide-level, reference-free pangenomes for each species. Through examining genomic diversity and pangenome structure, we analyzed intraspecific conservation and variability of putative virulence factors, focusing on functions under positive selection, and recent gene-family dynamics of contraction and expansion. Our findings reveal contrasting distributions of putative virulence factors in the core, dispensable, and private genomes of each pangenome. For example, CAZymes were prevalent in the core genomes of each pangenome, whereas biosynthetic gene clusters were prevalent in the dispensable genomes of E. lata and P. minimum. The dispensable fractions were also enriched in Gypsy transposable elements and virulence factors under positive selection (polyketide synthases genes in E. lata and P. minimum glycosyltransferases in N. parvum). Our findings underscore the complexity of the genomic architecture in each species and provide insights into their adaptive strategies, enhancing our understanding of the underlying mechanisms of virulence.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.06.556570">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.06.556570" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.06.556570">
        <p class="paperTitle">The genome and population genomics of allopolyploid Coffea arabica reveal the diversification history of modern coffee cultivars</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.06.556570" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.06.556570" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Salojarvi, J.; Rambani, A.; Yu, Z.; Guyot, R.; Strickler, S.; Lepelley, M.; Wang, C.; Rajaraman, S.; Rastas, P.; Zheng, C.; Munoz, D. S.; Meidanis, J.; Paschoal, A. R.; Bawin, Y.; Krabbenhoft, T.; Wang, Z. Q.; Fleck, S.; Aussel, R.; Bellanger, L.; Charpagne, A.; Fournier, C.; Kassam, M.; Lefebvre, G.; Metairon, S.; Moine, D.; Rigoreau, M.; Stolte, J.; Hamon, P.; Couturon, E.; Tranchant-Dubreuil, C.; Mukherjee, M.; Lan, T.; Engelhardt, J.; Stadler, P.; DeLemos, S. C.; Suzuki, S. I.; Sumirat, U.; ChingMan, W.; Dauchot, N.; Orozco-Arias, S.; Garavito, A.; Kiwuka, C.; Musoli, P.; Nalukenge, A.; Gu</p>
        <p class="info">Score: 11.2, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.06.556570' target='https://doi.org/10.1101/2023.09.06.556570'> 10.1101/2023.09.06.556570</a></p>
        <p class="abstract">Coffea arabica, an allotetraploid hybrid of C. eugenioides and C. canephora, is the source of approximately 60% of coffee products worldwide. Cultivated accessions have undergone several population bottlenecks resulting in low genetic diversity. We present chromosome-level assemblies of a di-haploid C. arabica accession and modern representatives of its diploid progenitors, C. eugenioides and C. canephora. The three species exhibit largely conserved genome structures between diploid parents and descendant subgenomes, which show a mosaic pattern of dominance, similar to other polyploid crop species. Resequencing of 39 wild and cultivated accessions suggests a founding polyploidy event [~]610,000 years ago, followed by several subsequent bottlenecks, including a population split [~]30.5 kya and a period of migration between Arabica populations until [~]8.9 kya. Analysis of lines historically introgressed with C. canephora highlights loci that may contribute to their superior pathogen resistance and lay the groundwork for future genomics-based breeding of C. arabica.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.04.556195">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.04.556195" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.04.556195">
        <p class="paperTitle">Telomere-to-telomere Schizosaccharomyces japonicus genome assembly reveals hitherto unknown genome features</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.04.556195" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.04.556195" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Etherington, G. J.; Nieduszynski, C.; Oliferenko, S.; Wu, P.-S.; Uhlmann, F.</p>
        <p class="info">Score: 9.2, Published: 2023-09-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.04.556195' target='https://doi.org/10.1101/2023.09.04.556195'> 10.1101/2023.09.04.556195</a></p>
        <p class="abstract">Schizosaccharomyces japonicus belongs to the single-genus class Schizosaccharomycetes, otherwise known as fission yeasts. As part of a composite model system with its widely studied S. pombe sister species, S. japonicus has provided critical insights into the workings and the evolution of cell biological mechanisms. Furthermore, its divergent biology makes S. japonicus a valuable model organism in its own right. However, the currently available short-read genome assembly contains gaps and has been unable to resolve centromeres and other repeat-rich chromosomal regions. Here we present a telomere-to-telomere long-read genome assembly of the S japonicus genome. This includes the three megabase-length chromosomes, with centromeres hundreds of kilobases long, rich in 5S ribosomal RNAs, transfer RNAs, long terminal repeats, and short repeats. We identify a gene-sparse region on chromosome 2 that resembles a 331 kb centromeric duplication. We revise the genome size of S. japonicus to at least 16.6 Mb and possibly up to 18.12 Mb, at least 30% larger than previous estimates. Our whole genome assembly will support the growing S. japonicus research community and facilitate research in new directions, including centromere and DNA repeat evolution, and yeast comparative genomics.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
