<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation
Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.
Score: 117.6, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582
Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation
Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.
Score: 117.6, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582
Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-06T10:39:07+00:00" />
<meta property="article:modified_time" content="2023-09-06T10:39:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation
Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.
Score: 117.6, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582
Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation\nAuthors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.\nScore: 117.6, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582\nSequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence.",
  "keywords": [
    
  ],
  "articleBody": " Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation\nAuthors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.\nScore: 117.6, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582\nSequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence. Using statistics derived from Borzois predicted coverage, we isolate and accurately score variant effects across multiple layers of regulation, including transcription, splicing, and polyadenylation. Evaluated on QTLs, Borzoi is competitive with, and often outperforms, state-of-the-art models trained on individual regulatory functions. By applying attribution methods to the derived statistics, we extract cis-regulatory patterns driving RNA expression and post-transcriptional regulation in normal tissues. The wide availability of RNA-seq data across species, conditions, and assays profiling specific aspects of regulation emphasizes the potential of this approach to decipher the mapping from DNA sequence to regulatory function.\nIn vivo perturb-seq of cancer and immune cells dissects oncologic drivers and therapy response\nAuthors: Liu, S. J.; Pak, J.; Zou, C.; Casey-Clyde, T.; Borah, A.; Wu, D.; Seo, K.; O'Loughlin, T.; Lim, D.; Ozawa, T.; Berger, M.; Weiss, W.; Raleigh, D. R.; Gilbert, L. A.\nScore: 27.1, Published: 2023-09-03 DOI: 10.1101/2023.09.01.555831\nGenetic perturbation screens with single cell readouts have enabled rich phenotyping of gene function and regulatory networks. These approaches have been challenging in vivo, especially in adult disease models such as cancer, which include mixtures of malignant and microenvironment cells. Here, we develop a multiplex in vivo perturb-seq CRISPRi platform for single cell genetic screens in cancer and immune cells that leverages intracranial convection enhanced delivery of sgRNA libraries into models of glioblastoma (GBM), a fatal brain cancer. Our results reveal in vivo specific rewiring of genetic dependencies in response to radiotherapy, a potent yet non-curative therapy for GBM, as well as alterations of ligand-receptor interactions between microenvironment cells. In sum, we demonstrate the utility of multiplexed perturb-seq for in vivo single cell dissection of adult tissue biology across multiple cell types in the context of therapeutic stimuli.\nRepeat turnover meets stable chromosomes: repetitive DNA sequences mark speciation and gene pool boundaries in sugar beet and wild beets\nAuthors: Schmidt, N.; Sielemann, K.; Breitenbach, S.; Fuchs, J.; Pucker, B.; Weisshaar, B.; HoltgraÌˆwe, D.; Heitkam, T.\nScore: 25.5, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555723\nBackgroundSugar beet (Beta vulgaris subsp. vulgaris) and its crop wild relatives share a base chromosome number of nine and similar chromosome morphologies. Yet, interspecific breeding is impeded by chromosome and sequence divergence that is still not fully understood. Since repetitive DNA sequences represent the fastest evolving parts of the genome, they likely impact genomic variability and contribute to the separation of beet gene pools. Hence, we investigated if innovations and losses in the repeatome can be linked to chromosomal differentiation and speciation. ResultsWe traced genome- and chromosome-wide evolution across sugar beet and twelve wild beets comprising all sections of the beet genera Beta and Patellifolia. For this, we combined data from short and long read sequencing, flow cytometry, and cytogenetics to build a comprehensive data framework for our beet panel that spans the complete scale from DNA sequence to chromosome up to the genome. Genome sizes and repeat profiles reflect the separation of the beet species into three gene pools. These gene pools harbor repeats with contrasting evolutionary patterns: We identified section- and species-specific repeat emergences and losses, e.g. of the retrotransposons causal for genome expansions in the section Corollinae/Nanae. Since most genomic variability was found in the satellite DNAs, we focused on tracing the 19 beetSat families across the three beet sections/genera. These taxa harbor evidence for contrasting strategies in repeat evolution, leading to contrasting satellite DNA profiles and fundamentally different centromere architectures, ranging from chromosomal uniformity in Beta and Patellifolia species to the formation of patchwork chromosomes in Corollinae/Nanae species. ConclusionsWe show that repetitive DNA sequences are causal for genome size expansion and contraction across the beet genera, providing insights into the genomic underpinnings of beet speciation. Satellite DNAs in particular vary considerably among beet taxa, leading to the evolution of distinct chromosomal setups. These differences likely contribute to the barriers in beet breeding between the three gene pools. Thus, with their isokaryotypic chromosome sets, beet genomes present an ideal system for studying the link between repeats, genome variability, and chromosomal differentiation/evolution and provide a theoretical basis for understanding barriers in crop breeding.\nCrossing host boundaries: the evolutionary drivers and correlates of viral host jumps\nAuthors: Tan, C. C. S.; van Dorp, L.; Balloux, F.\nScore: 9.3, Published: 2023-09-05 DOI: 10.1101/2023.09.01.555953\nMost emerging and re-emerging infectious diseases stem from viruses that naturally circulate in non-human vertebrates. When these viruses cross over into humans, they can cause disease outbreaks and epidemics. While zoonotic host jumps have been extensively studied from an ecological perspective, little attention has gone into characterising the evolutionary drivers and correlates underlying these events. To address this gap, we harnessed the entirety of publicly available viral genomic data, employing a comprehensive suite of network and phylogenetic analyses. We address a series of questions concerning the evolutionary mechanisms underpinning viral host jumps. Notably, we challenge conventional assumptions about the directionality of host jumps, demonstrating that humans are as much a source as a sink for viral spillover events, insofar we infer more viruses to have jumped from humans to other animals, than from animals to humans. Moreover, we demonstrate heightened evolution in viral lineages that involve putative host jumps. We further observe that the mutational threshold associated with a host jump is lower for viruses with broad host ranges. Finally, we show that the genomic targets of natural selection upon a successful host jump vary across different viral families with either structural or auxiliary genes being the prime targets of selection. Collectively, our results illuminate some of the evolutionary drivers underlying viral host jumps that may contribute to mitigating viral threats across species boundaries.\nMetagenomic analysis of the honey bee queen microbiome reveals low bacterial diversity and Caudoviricetes phages\nAuthors: Caesar, L.; Rice, D. W.; McAfee, A.; Underwood, R.; Tarpy, D. R.; J Foster, L.; Newton, I. L. G.\nScore: 8.9, Published: 2023-09-01 DOI: 10.1101/2023.08.29.555332\nIn eusocial insects, the health of the queens - the colony founders and sole reproductive females - is a primary determinant for colony success. Queen failure in the honey bee Apis mellifera, for example, is a major concern of beekeepers that annually suffer with colony losses, necessitating a greater knowledge of queen health. Several studies on the microbiome of honey bees have characterized its diversity and shown its importance for the health of worker bees, the female non-reproductive caste. However, the microbiome of workers differs from that of queens, which in comparison is still poorly studied. Thus, direct investigations of the queen microbiome are required to understand colony-level microbiome assembly, functional roles, and evolution. Here we used metagenomics to comprehensively characterize the honey bee queen microbiome. Comparing samples from different geographic locations and breeder sources, we show that the microbiome of queens is mostly shaped by the environment experienced since early life, and is predicted to play roles in breakdown of the diet, and protection from pathogens and xenobiotics. We also reveal the microbiome of queens comprises only four core bacterial species, Apilactobacillus kunkeei, Lactobacillus apis, Bombella apis and Commensalibacter sp. Interestingly, in addition to bacteria, we show that bacteriophages infect the queen microbiome, for which Lactobacillaceae are predicted to be the main reservoirs. Together, our results provide the basis to understand the honey bee colony microbiome assemblage, can guide improvements in queen rearing processes, and highlight the importance of bacteriophages for queen microbiome health and microbiome homeostasis in eusocial insects. ImportanceThe queen caste plays a central role for colony success in eusocial insects, as queens lay eggs, and regulate colony behavior and development. Queen failure can cause colonies to collapse, which is one of the major concerns of beekeepers. Thus, understanding of the biology behind the queens health is a pressing issue. Previous studies have shown the bee microbiome plays an important role in worker bee health, but little is known about the queen microbiome and its function in vivo. Here, we characterized the queen microbiome identifying for the first time present species and their putative functions. We show that the queen microbiome have predicted nutritional and protective roles in queen association, and comprises only four consistently present bacterial species. Additionally, we bring to attention the spread of phages in the queen microbiome, which increased in abundance in failing queens and may impact the fate of the colony.\nFunctional annotation of a divergent genome using sequence and structure-based homology.\nAuthors: Svedberg, D.; Winiger, R. R.; Berg, A.; Sharma, H.; Debrunner-Vossbrinck, B. A.; Vossbrinck, C. R.; Barandun, J.\nScore: 9.2, Published: 2023-08-28 DOI: 10.1101/2023.08.27.554996\nBackgroundMicrosporidia are a large taxon of intracellular pathogens characterized by extraordinarily streamlined genomes with unusually high sequence divergence and many species-specific adaptations. These unique factors pose challenges for traditional genome annotation methods based on sequence homology. As a result, many of the microsporidian genomes sequenced to date contain numerous genes of unknown function. Recent innovations in rapid and accurate structure prediction and comparison, together with the growing amount of data in structural databases, provide new opportunities to assist in the functional annotation of newly sequenced genomes. ResultsIn this study, we established a workflow that combines sequence and structure-based functional gene annotation approaches employing a ChimeraX plugin, allowing for visual inspection and manual curation. We employed this workflow on a high-quality telomere-to-telomere sequenced tetraploid genome of Vairimorpha necatrix. First, the 3080 predicted open reading frames, of which 89 % were confirmed with RNA sequencing data, were used as input. Next, ColabFold was used to create protein structure predictions, followed by a Foldseek search for structural matching to the PDB and AlphaFold databases. The subsequent manual curation, using sequence and structure-based hits, increased the accuracy and quality of the functional genome annotation compared to results using only traditional annotation tools. Our workflow resulted in a comprehensive description of the V. necatrix genome, along with a structural summary of the most prevalent protein groups, such as the ricin B lectin family. In addition, and to test our tool, we identified the functions of several previously uncharacterized Encephalitozoon cuniculi genes. ConclusionWe provide a new functional annotation tool for divergent organisms and employ it on a newly sequenced, high-quality microsporidian genome to shed light on this uncharacterized intracellular pathogen of Lepidoptera. The addition of a structure-based annotation approach can serve as a valuable template for studying other microsporidian or similarly divergent species.\nTranscriptomic profiling of Schlemm's canal cells reveals a lymphatic-biased identity and three major cell states\nAuthors: Balasubramanian, R.; Kizhatil, K.; Li, T.; Tolman, N.; Bhandari, A.; Clark, G.; Bupp-Chickering, V.; Zhou, S.; Peregrin, J.; Simon, M.; Montgomery, C.; Qian, J.; John, S.\nScore: 7.8, Published: 2023-09-03 DOI: 10.1101/2023.08.31.555823\nSchlemm's canal (SC) is central in intraocular pressure regulation but requires much characterization. It has distinct inner and outer walls, each composed of Schlemm's canal endothelial cells (SECs) with different morphologies and functions. Recent transcriptomic studies of the anterior segment added important knowledge, but were limited in power by SEC numbers or did not focus on SC. To gain a more comprehensive understanding of SC biology, we performed bulk RNA sequencing on C57BL/6J SC, blood vessel and lymphatic endothelial cells from limbal tissue (~4500 SECs). We also analyzed mouse limbal tissues by single-cell and single-nucleus RNA sequencing (C57BL/6J and 129/Sj strains), successfully sequencing 903 individual SECs. Together, these datasets confirm that SC has molecular characteristics of both blood and lymphatic endothelia with a lymphatic phenotype predominating. SECs are enriched in pathways that regulate cell-cell junction formation pointing to the importance of junctions in determining SC fluid permeability. Importantly, and for the first time, our analyses characterize 3 molecular classes of SECs, molecularly distinguishing inner wall from outer wall SECs and discovering two inner wall cell states that likely result from local environmental differences. Further, and based on ligand and receptor expression patterns, we document key interactions between SECs and cells of the adjacent trabecular meshwork (TM) drainage tissue. Also, we present cell type expression for a collection of human glaucoma genes. These data provide a new molecular foundation that will enable the functional dissection of key homeostatic processes mediated by SECs as well as the development of new glaucoma therapeutics.\nThree Accessory Gene Clusters Drive Host-Adaptation in Group B Streptococcus\nAuthors: Crestani, C.; Forde, T. L.; Bell, J.; Lycett, S. J.; Oliveira, L. M.; Pinto, T. C.; Cobo-Angel, C. G.; Ceballos-Marquez, A.; Phuoc, N. N.; Sirimanapong, W.; Chen, S. L.; Jamrozy, D.; Bentley, S. D.; Fontaine, M.; Zadoks, R. N.\nScore: 10.0, Published: 2023-08-13 DOI: 10.1101/2023.08.10.552778\nStreptococcus agalactiae (Group B Streptococcus, GBS) is a major pathogen of humans and animals, posing a threat to human health as well as food security. Here, we investigate the role of genomic mechanisms, including homologous recombination and horizontal gene transfer, in shaping the population structure of GBS and its adaptation to three major host groups (humans, cattle, fishes). We demonstrate that the GBS population comprises host-specialist, host-adapted lineages as well as host generalists, and that these categories differ in their level or recombination. Although the accessory genome at large varies by lineage rather than host, genome wide association studies show that host association is driven by three accessory genome clusters, regardless of lineage or breadth of the host spectrum. These genomic clusters (scpB in human GBS, lactose operon in bovine GBS, Locus 3 in fish GBS) are known (scpB, Lac.2) or shown here (Locus 3) to be functionally relevant and are shared with other streptococcal species occupying the same host niche. These findings demonstrate the importance of considering the role of non-human host species in the evolution of GBS, including high risk clones that may lead to interspecies transmission and affect efficacy of future GBS vaccines.\nAIRR-C Human IG Reference Sets: curated sets of immunoglobulin heavy and light chain germline genes\nAuthors: Collins, A. M.; Ohlin, M.; Corcoran, M.; Heather, J. M.; Ralph, D.; Law, M.; Martinez-Barnetche, J.; Ye, J.; Richardson, E.; Gibson, W. S.; Rodriguez, O. L.; Peres, A.; Yaari, G.; Watson, C. T.; Lees, W. D.\nScore: 5.7, Published: 2023-09-01 DOI: 10.1101/2023.09.01.555348\nAnalysis of an individual's immunoglobulin (IG) gene repertoire requires the use of high-quality germline gene Reference Sets. The Adaptive Immune Receptor Repertoire-Community (AIRR-C) Reference Sets have been developed to include only human IG heavy and light chain alleles that have been confirmed by evidence from multiple high-quality sources. By including only those alleles with a high level of support, including some new sequences that currently lack official names, AIRR-seq analysis will have greater accuracy and studies of the evolution of immunoglobulin genes, their allelic variants and the expressed immune repertoire will be facilitated. Although containing less than half the previously recognised IG alleles (e.g. just 198 IGHV sequences), the Reference Sets eliminated erroneous calls and provided excellent coverage when tested on a set of repertoires from 99 individuals comprising over 4 million V(D)J rearrangements. To improve AIRR-seq analysis, some alleles have been extended to deal with short 3' or 5' truncations that can lead them to be overlooked by alignment utilities. To avoid other challenges for analysis programs, exact paralogs (e.g. IGHV1-69*01 and IGHV1-69D*01) are only represented once in each set, though alternative sequence names are noted in accompanying metadata. The Reference Sets also include novel alleles: 8 IGHV alleles, 2 IGKV alleles and 5 IGLV alleles. The version-tracked AIRR-C Reference Sets are freely available at the OGRDB website (https://ogrdb.airr-community.org/germline_sets/Human) and will be regularly updated to include newly-observed and previously-reported sequences that can be confirmed by new high-quality data.\nProgrammable multi-kilobase RNA editing using CRISPR-mediated trans-splicing\nAuthors: Borrajo, J.; Javanmardi, K.; Griffin, J.; St. Martin, S. J.; Yao, D.; Hill, K.; Blainey, P.; Al-Shayeb, B.\nScore: 108.2, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553620\nCurrent gene editing approaches in eukaryotic cells are limited to single base edits or small DNA insertions and deletions, and remain encumbered by unintended permanent effects and significant challenges in the delivery of large DNA cargo. Here we describe Splice Editing, a generalizable platform to correct gene transcripts in situ by programmable insertion or replacement of large RNA segments. By combining CRISPR-mediated RNA targeting with endogenous cellular RNA-splicing machinery, Splice Editing enables efficient, precise, and programmable large-scale editing of gene targets without DNA cleavage or mutagenesis. RNA sequencing and measurement of spliced protein products confirm that Splice Editing achieves efficient and specific targeted RNA and protein correction. We show that Splice Editors based on novel miniature RNA-targeting CRISPR-Cas systems discovered and characterized in this work can be packaged for effective delivery to human cells and affect different types of edits across multiple targets and cell lines. By editing thousands of bases simultaneously in a single reversible step, Splice Editing could expand the treatable disease population for monogenic diseases with large allelic diversity without the permanent unintended effects of DNA editing. One-sentence summaryCRISPR-guided trans-splicing enables efficient and specific recombination of large RNA molecules in mammalian cells, with broad applications in therapeutic development for genetic diseases and as a research tool for the study of basic RNA biology.\n",
  "wordCount" : "2868",
  "inLanguage": "en",
  "datePublished": "2023-09-06T10:39:07Z",
  "dateModified": "2023-09-06T10:39:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on September 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555582">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555582" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555582">
        <p class="paperTitle">Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555582" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555582" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.</p>
        <p class="info">Score: 117.6, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555582' target='https://doi.org/10.1101/2023.08.30.555582'> 10.1101/2023.08.30.555582</a></p>
        <p class="abstract">Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence. Using statistics derived from Borzois predicted coverage, we isolate and accurately score variant effects across multiple layers of regulation, including transcription, splicing, and polyadenylation. Evaluated on QTLs, Borzoi is competitive with, and often outperforms, state-of-the-art models trained on individual regulatory functions. By applying attribution methods to the derived statistics, we extract cis-regulatory patterns driving RNA expression and post-transcriptional regulation in normal tissues. The wide availability of RNA-seq data across species, conditions, and assays profiling specific aspects of regulation emphasizes the potential of this approach to decipher the mapping from DNA sequence to regulatory function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555831">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555831" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555831">
        <p class="paperTitle">In vivo perturb-seq of cancer and immune cells dissects oncologic drivers and therapy response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555831" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555831" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liu, S. J.; Pak, J.; Zou, C.; Casey-Clyde, T.; Borah, A.; Wu, D.; Seo, K.; O&#39;Loughlin, T.; Lim, D.; Ozawa, T.; Berger, M.; Weiss, W.; Raleigh, D. R.; Gilbert, L. A.</p>
        <p class="info">Score: 27.1, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555831' target='https://doi.org/10.1101/2023.09.01.555831'> 10.1101/2023.09.01.555831</a></p>
        <p class="abstract">Genetic perturbation screens with single cell readouts have enabled rich phenotyping of gene function and regulatory networks. These approaches have been challenging in vivo, especially in adult disease models such as cancer, which include mixtures of malignant and microenvironment cells. Here, we develop a multiplex in vivo perturb-seq CRISPRi platform for single cell genetic screens in cancer and immune cells that leverages intracranial convection enhanced delivery of sgRNA libraries into models of glioblastoma (GBM), a fatal brain cancer. Our results reveal in vivo specific rewiring of genetic dependencies in response to radiotherapy, a potent yet non-curative therapy for GBM, as well as alterations of ligand-receptor interactions between microenvironment cells. In sum, we demonstrate the utility of multiplexed perturb-seq for in vivo single cell dissection of adult tissue biology across multiple cell types in the context of therapeutic stimuli.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555723">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555723" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555723">
        <p class="paperTitle">Repeat turnover meets stable chromosomes: repetitive DNA sequences mark speciation and gene pool boundaries in sugar beet and wild beets</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555723" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555723" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schmidt, N.; Sielemann, K.; Breitenbach, S.; Fuchs, J.; Pucker, B.; Weisshaar, B.; HoltgraÌˆwe, D.; Heitkam, T.</p>
        <p class="info">Score: 25.5, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555723' target='https://doi.org/10.1101/2023.09.01.555723'> 10.1101/2023.09.01.555723</a></p>
        <p class="abstract">BackgroundSugar beet (Beta vulgaris subsp. vulgaris) and its crop wild relatives share a base chromosome number of nine and similar chromosome morphologies. Yet, interspecific breeding is impeded by chromosome and sequence divergence that is still not fully understood. Since repetitive DNA sequences represent the fastest evolving parts of the genome, they likely impact genomic variability and contribute to the separation of beet gene pools. Hence, we investigated if innovations and losses in the repeatome can be linked to chromosomal differentiation and speciation.

ResultsWe traced genome- and chromosome-wide evolution across sugar beet and twelve wild beets comprising all sections of the beet genera Beta and Patellifolia. For this, we combined data from short and long read sequencing, flow cytometry, and cytogenetics to build a comprehensive data framework for our beet panel that spans the complete scale from DNA sequence to chromosome up to the genome.

Genome sizes and repeat profiles reflect the separation of the beet species into three gene pools. These gene pools harbor repeats with contrasting evolutionary patterns: We identified section- and species-specific repeat emergences and losses, e.g. of the retrotransposons causal for genome expansions in the section Corollinae/Nanae. Since most genomic variability was found in the satellite DNAs, we focused on tracing the 19 beetSat families across the three beet sections/genera. These taxa harbor evidence for contrasting strategies in repeat evolution, leading to contrasting satellite DNA profiles and fundamentally different centromere architectures, ranging from chromosomal uniformity in Beta and Patellifolia species to the formation of patchwork chromosomes in Corollinae/Nanae species.

ConclusionsWe show that repetitive DNA sequences are causal for genome size expansion and contraction across the beet genera, providing insights into the genomic underpinnings of beet speciation. Satellite DNAs in particular vary considerably among beet taxa, leading to the evolution of distinct chromosomal setups. These differences likely contribute to the barriers in beet breeding between the three gene pools. Thus, with their isokaryotypic chromosome sets, beet genomes present an ideal system for studying the link between repeats, genome variability, and chromosomal differentiation/evolution and provide a theoretical basis for understanding barriers in crop breeding.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555953">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555953" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555953">
        <p class="paperTitle">Crossing host boundaries: the evolutionary drivers and correlates of viral host jumps</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555953" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555953" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tan, C. C. S.; van Dorp, L.; Balloux, F.</p>
        <p class="info">Score: 9.3, Published: 2023-09-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555953' target='https://doi.org/10.1101/2023.09.01.555953'> 10.1101/2023.09.01.555953</a></p>
        <p class="abstract">Most emerging and re-emerging infectious diseases stem from viruses that naturally circulate in non-human vertebrates. When these viruses cross over into humans, they can cause disease outbreaks and epidemics. While zoonotic host jumps have been extensively studied from an ecological perspective, little attention has gone into characterising the evolutionary drivers and correlates underlying these events. To address this gap, we harnessed the entirety of publicly available viral genomic data, employing a comprehensive suite of network and phylogenetic analyses. We address a series of questions concerning the evolutionary mechanisms underpinning viral host jumps. Notably, we challenge conventional assumptions about the directionality of host jumps, demonstrating that humans are as much a source as a sink for viral spillover events, insofar we infer more viruses to have jumped from humans to other animals, than from animals to humans. Moreover, we demonstrate heightened evolution in viral lineages that involve putative host jumps. We further observe that the mutational threshold associated with a host jump is lower for viruses with broad host ranges. Finally, we show that the genomic targets of natural selection upon a successful host jump vary across different viral families with either structural or auxiliary genes being the prime targets of selection. Collectively, our results illuminate some of the evolutionary drivers underlying viral host jumps that may contribute to mitigating viral threats across species boundaries.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555332">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555332" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555332">
        <p class="paperTitle">Metagenomic analysis of the honey bee queen microbiome reveals low bacterial diversity and Caudoviricetes phages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555332" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555332" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caesar, L.; Rice, D. W.; McAfee, A.; Underwood, R.; Tarpy, D. R.; J Foster, L.; Newton, I. L. G.</p>
        <p class="info">Score: 8.9, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555332' target='https://doi.org/10.1101/2023.08.29.555332'> 10.1101/2023.08.29.555332</a></p>
        <p class="abstract">In eusocial insects, the health of the queens - the colony founders and sole reproductive females - is a primary determinant for colony success. Queen failure in the honey bee Apis mellifera, for example, is a major concern of beekeepers that annually suffer with colony losses, necessitating a greater knowledge of queen health. Several studies on the microbiome of honey bees have characterized its diversity and shown its importance for the health of worker bees, the female non-reproductive caste. However, the microbiome of workers differs from that of queens, which in comparison is still poorly studied. Thus, direct investigations of the queen microbiome are required to understand colony-level microbiome assembly, functional roles, and evolution. Here we used metagenomics to comprehensively characterize the honey bee queen microbiome. Comparing samples from different geographic locations and breeder sources, we show that the microbiome of queens is mostly shaped by the environment experienced since early life, and is predicted to play roles in breakdown of the diet, and protection from pathogens and xenobiotics. We also reveal the microbiome of queens comprises only four core bacterial species, Apilactobacillus kunkeei, Lactobacillus apis, Bombella apis and Commensalibacter sp. Interestingly, in addition to bacteria, we show that bacteriophages infect the queen microbiome, for which Lactobacillaceae are predicted to be the main reservoirs. Together, our results provide the basis to understand the honey bee colony microbiome assemblage, can guide improvements in queen rearing processes, and highlight the importance of bacteriophages for queen microbiome health and microbiome homeostasis in eusocial insects.

ImportanceThe queen caste plays a central role for colony success in eusocial insects, as queens lay eggs, and regulate colony behavior and development. Queen failure can cause colonies to collapse, which is one of the major concerns of beekeepers. Thus, understanding of the biology behind the queens health is a pressing issue. Previous studies have shown the bee microbiome plays an important role in worker bee health, but little is known about the queen microbiome and its function in vivo. Here, we characterized the queen microbiome identifying for the first time present species and their putative functions. We show that the queen microbiome have predicted nutritional and protective roles in queen association, and comprises only four consistently present bacterial species. Additionally, we bring to attention the spread of phages in the queen microbiome, which increased in abundance in failing queens and may impact the fate of the colony.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.554996">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.554996" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.554996">
        <p class="paperTitle">Functional annotation of a divergent genome using sequence and structure-based homology.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.554996" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.554996" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Svedberg, D.; Winiger, R. R.; Berg, A.; Sharma, H.; Debrunner-Vossbrinck, B. A.; Vossbrinck, C. R.; Barandun, J.</p>
        <p class="info">Score: 9.2, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.554996' target='https://doi.org/10.1101/2023.08.27.554996'> 10.1101/2023.08.27.554996</a></p>
        <p class="abstract">BackgroundMicrosporidia are a large taxon of intracellular pathogens characterized by extraordinarily streamlined genomes with unusually high sequence divergence and many species-specific adaptations. These unique factors pose challenges for traditional genome annotation methods based on sequence homology. As a result, many of the microsporidian genomes sequenced to date contain numerous genes of unknown function. Recent innovations in rapid and accurate structure prediction and comparison, together with the growing amount of data in structural databases, provide new opportunities to assist in the functional annotation of newly sequenced genomes.

ResultsIn this study, we established a workflow that combines sequence and structure-based functional gene annotation approaches employing a ChimeraX plugin, allowing for visual inspection and manual curation. We employed this workflow on a high-quality telomere-to-telomere sequenced tetraploid genome of Vairimorpha necatrix. First, the 3080 predicted open reading frames, of which 89 % were confirmed with RNA sequencing data, were used as input. Next, ColabFold was used to create protein structure predictions, followed by a Foldseek search for structural matching to the PDB and AlphaFold databases. The subsequent manual curation, using sequence and structure-based hits, increased the accuracy and quality of the functional genome annotation compared to results using only traditional annotation tools. Our workflow resulted in a comprehensive description of the V. necatrix genome, along with a structural summary of the most prevalent protein groups, such as the ricin B lectin family. In addition, and to test our tool, we identified the functions of several previously uncharacterized Encephalitozoon cuniculi genes.

ConclusionWe provide a new functional annotation tool for divergent organisms and employ it on a newly sequenced, high-quality microsporidian genome to shed light on this uncharacterized intracellular pathogen of Lepidoptera. The addition of a structure-based annotation approach can serve as a valuable template for studying other microsporidian or similarly divergent species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.31.555823">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.31.555823" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.31.555823">
        <p class="paperTitle">Transcriptomic profiling of Schlemm&#39;s canal cells reveals a lymphatic-biased identity and three major cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.31.555823" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.31.555823" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Balasubramanian, R.; Kizhatil, K.; Li, T.; Tolman, N.; Bhandari, A.; Clark, G.; Bupp-Chickering, V.; Zhou, S.; Peregrin, J.; Simon, M.; Montgomery, C.; Qian, J.; John, S.</p>
        <p class="info">Score: 7.8, Published: 2023-09-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.31.555823' target='https://doi.org/10.1101/2023.08.31.555823'> 10.1101/2023.08.31.555823</a></p>
        <p class="abstract">Schlemm&#39;s canal (SC) is central in intraocular pressure regulation but requires much characterization. It has distinct inner and outer walls, each composed of Schlemm&#39;s canal endothelial cells (SECs) with different morphologies and functions. Recent transcriptomic studies of the anterior segment added important knowledge, but were limited in power by SEC numbers or did not focus on SC. To gain a more comprehensive understanding of SC biology, we performed bulk RNA sequencing on C57BL/6J SC, blood vessel and lymphatic endothelial cells from limbal tissue (~4500 SECs). We also analyzed mouse limbal tissues by single-cell and single-nucleus RNA sequencing (C57BL/6J and 129/Sj strains), successfully sequencing 903 individual SECs. Together, these datasets confirm that SC has molecular characteristics of both blood and lymphatic endothelia with a lymphatic phenotype predominating. SECs are enriched in pathways that regulate cell-cell junction formation pointing to the importance of junctions in determining SC fluid permeability. Importantly, and for the first time, our analyses characterize 3 molecular classes of SECs, molecularly distinguishing inner wall from outer wall SECs and discovering two inner wall cell states that likely result from local environmental differences. Further, and based on ligand and receptor expression patterns, we document key interactions between SECs and cells of the adjacent trabecular meshwork (TM) drainage tissue. Also, we present cell type expression for a collection of human glaucoma genes. These data provide a new molecular foundation that will enable the functional dissection of key homeostatic processes mediated by SECs as well as the development of new glaucoma therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552778" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552778">
        <p class="paperTitle">Three Accessory Gene Clusters Drive Host-Adaptation in Group B Streptococcus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Crestani, C.; Forde, T. L.; Bell, J.; Lycett, S. J.; Oliveira, L. M.; Pinto, T. C.; Cobo-Angel, C. G.; Ceballos-Marquez, A.; Phuoc, N. N.; Sirimanapong, W.; Chen, S. L.; Jamrozy, D.; Bentley, S. D.; Fontaine, M.; Zadoks, R. N.</p>
        <p class="info">Score: 10.0, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552778' target='https://doi.org/10.1101/2023.08.10.552778'> 10.1101/2023.08.10.552778</a></p>
        <p class="abstract">Streptococcus agalactiae (Group B Streptococcus, GBS) is a major pathogen of humans and animals, posing a threat to human health as well as food security. Here, we investigate the role of genomic mechanisms, including homologous recombination and horizontal gene transfer, in shaping the population structure of GBS and its adaptation to three major host groups (humans, cattle, fishes). We demonstrate that the GBS population comprises host-specialist, host-adapted lineages as well as host generalists, and that these categories differ in their level or recombination. Although the accessory genome at large varies by lineage rather than host, genome wide association studies show that host association is driven by three accessory genome clusters, regardless of lineage or breadth of the host spectrum. These genomic clusters (scpB in human GBS, lactose operon in bovine GBS, Locus 3 in fish GBS) are known (scpB, Lac.2) or shown here (Locus 3) to be functionally relevant and are shared with other streptococcal species occupying the same host niche. These findings demonstrate the importance of considering the role of non-human host species in the evolution of GBS, including high risk clones that may lead to interspecies transmission and affect efficacy of future GBS vaccines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555348">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555348" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555348">
        <p class="paperTitle">AIRR-C Human IG Reference Sets: curated sets of immunoglobulin heavy and light chain germline genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555348" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555348" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Collins, A. M.; Ohlin, M.; Corcoran, M.; Heather, J. M.; Ralph, D.; Law, M.; Martinez-Barnetche, J.; Ye, J.; Richardson, E.; Gibson, W. S.; Rodriguez, O. L.; Peres, A.; Yaari, G.; Watson, C. T.; Lees, W. D.</p>
        <p class="info">Score: 5.7, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555348' target='https://doi.org/10.1101/2023.09.01.555348'> 10.1101/2023.09.01.555348</a></p>
        <p class="abstract">Analysis of an individual&#39;s immunoglobulin (IG) gene repertoire requires the use of high-quality germline gene Reference Sets. The Adaptive Immune Receptor Repertoire-Community (AIRR-C) Reference Sets have been developed to include only human IG heavy and light chain alleles that have been confirmed by evidence from multiple high-quality sources. By including only those alleles with a high level of support, including some new sequences that currently lack official names, AIRR-seq analysis will have greater accuracy and studies of the evolution of immunoglobulin genes, their allelic variants and the expressed immune repertoire will be facilitated. Although containing less than half the previously recognised IG alleles (e.g. just 198 IGHV sequences), the Reference Sets eliminated erroneous calls and provided excellent coverage when tested on a set of repertoires from 99 individuals comprising over 4 million V(D)J rearrangements. To improve AIRR-seq analysis, some alleles have been extended to deal with short 3&#39; or 5&#39; truncations that can lead them to be overlooked by alignment utilities. To avoid other challenges for analysis programs, exact paralogs (e.g. IGHV1-69*01 and IGHV1-69D*01) are only represented once in each set, though alternative sequence names are noted in accompanying metadata. The Reference Sets also include novel alleles: 8 IGHV alleles, 2 IGKV alleles and 5 IGLV alleles. The version-tracked AIRR-C Reference Sets are freely available at the OGRDB website (https://ogrdb.airr-community.org/germline_sets/Human) and will be regularly updated to include newly-observed and previously-reported sequences that can be confirmed by new high-quality data.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553620">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553620" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553620">
        <p class="paperTitle">Programmable multi-kilobase RNA editing using CRISPR-mediated trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553620" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553620" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Borrajo, J.; Javanmardi, K.; Griffin, J.; St. Martin, S. J.; Yao, D.; Hill, K.; Blainey, P.; Al-Shayeb, B.</p>
        <p class="info">Score: 108.2, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553620' target='https://doi.org/10.1101/2023.08.18.553620'> 10.1101/2023.08.18.553620</a></p>
        <p class="abstract">Current gene editing approaches in eukaryotic cells are limited to single base edits or small DNA insertions and deletions, and remain encumbered by unintended permanent effects and significant challenges in the delivery of large DNA cargo. Here we describe Splice Editing, a generalizable platform to correct gene transcripts in situ by programmable insertion or replacement of large RNA segments. By combining CRISPR-mediated RNA targeting with endogenous cellular RNA-splicing machinery, Splice Editing enables efficient, precise, and programmable large-scale editing of gene targets without DNA cleavage or mutagenesis. RNA sequencing and measurement of spliced protein products confirm that Splice Editing achieves efficient and specific targeted RNA and protein correction. We show that Splice Editors based on novel miniature RNA-targeting CRISPR-Cas systems discovered and characterized in this work can be packaged for effective delivery to human cells and affect different types of edits across multiple targets and cell lines. By editing thousands of bases simultaneously in a single reversible step, Splice Editing could expand the treatable disease population for monogenic diseases with large allelic diversity without the permanent unintended effects of DNA editing.

One-sentence summaryCRISPR-guided trans-splicing enables efficient and specific recombination of large RNA molecules in mammalian cells, with broad applications in therapeutic development for genetic diseases and as a research tool for the study of basic RNA biology.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
