<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Phage-plasmids promote genetic exchanges between phages and plasmids and create novel ones.
Authors: Pfeifer, E.; Rocha, E. P. C.
Score: 27.5, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552325
Phages and plasmids have key roles in bacterial evolution and are usually very different. Yet, they must recombine, since they sometimes carry nearly identical accessory genes. Phage-plasmids are both plasmids and phages and we found they connect them by being at their interface in the network of gene repertoire relatedness.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Phage-plasmids promote genetic exchanges between phages and plasmids and create novel ones.
Authors: Pfeifer, E.; Rocha, E. P. C.
Score: 27.5, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552325
Phages and plasmids have key roles in bacterial evolution and are usually very different. Yet, they must recombine, since they sometimes carry nearly identical accessory genes. Phage-plasmids are both plasmids and phages and we found they connect them by being at their interface in the network of gene repertoire relatedness." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-13T10:36:01+00:00" />
<meta property="article:modified_time" content="2023-08-13T10:36:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Phage-plasmids promote genetic exchanges between phages and plasmids and create novel ones.
Authors: Pfeifer, E.; Rocha, E. P. C.
Score: 27.5, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552325
Phages and plasmids have key roles in bacterial evolution and are usually very different. Yet, they must recombine, since they sometimes carry nearly identical accessory genes. Phage-plasmids are both plasmids and phages and we found they connect them by being at their interface in the network of gene repertoire relatedness."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Phage-plasmids promote genetic exchanges between phages and plasmids and create novel ones.\nAuthors: Pfeifer, E.; Rocha, E. P. C.\nScore: 27.5, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552325\nPhages and plasmids have key roles in bacterial evolution and are usually very different. Yet, they must recombine, since they sometimes carry nearly identical accessory genes. Phage-plasmids are both plasmids and phages and we found they connect them by being at their interface in the network of gene repertoire relatedness.",
  "keywords": [
    
  ],
  "articleBody": " Phage-plasmids promote genetic exchanges between phages and plasmids and create novel ones.\nAuthors: Pfeifer, E.; Rocha, E. P. C.\nScore: 27.5, Published: 2023-08-11 DOI: 10.1101/2023.08.08.552325\nPhages and plasmids have key roles in bacterial evolution and are usually very different. Yet, they must recombine, since they sometimes carry nearly identical accessory genes. Phage-plasmids are both plasmids and phages and we found they connect them by being at their interface in the network of gene repertoire relatedness. We searched for recombining genes in all three types of elements and found that phage-plasmid exchange rates are between those of plasmids (highest) and phages (lowest). Phage-plasmids drive gene flow between phages and plasmids, including that of antibiotic resistance and defense systems. Genetic exchanges and gene inactivation resulted in P1-like phage-plasmids turning into integrative prophages or just plasmids. The latter acquired conjugation-related functions thereby transitioning from viral to conjugation-driven horizontal transfer. Phage-plasmids have thus a key role in spreading traits across mobile genetic elements and their hosts, while facilitating conversions of one type of element into another.\nA harmonized public resource of deeply sequenced diverse human genomes\nAuthors: Koenig, Z.; Yohannes, M. T.; Nkambule, L. L.; Goodrich, J. K.; Kim, H. A.; Zhao, X.; Wilson, M. W.; Tiao, G.; Hao, S. P.; Sahakian, N.; Chao, K. R.; gnomAD Project Consortium, ; Rehm, H.; Neale, B.; Talkowski, M. E.; Daly, M. J.; Brand, H.; Karczewski, K.; Atkinson, E. G.; Martin, A.\nScore: 153.3, Published: 2023-08-10 DOI: 10.1101/2023.01.23.525248\nUnderrepresented populations are often excluded from genomic studies due in part to a lack of resources supporting their analyses. The 1000 Genomes Project (1kGP) and Human Genome Diversity Project (HGDP), which have recently been sequenced to high coverage, are valuable genomic resources because of the global diversity they capture and their open data sharing policies. Here, we harmonized a high quality set of 4,096 whole genomes from HGDP and 1kGP with data from gnomAD and identified over 159 million high-quality SNVs, indels, and SVs. We performed a detailed ancestry analysis of this cohort, characterizing population structure and patterns of admixture across populations, analyzing site frequency spectra, and measuring variant counts at global and subcontinental levels. We also demonstrate substantial added value from this dataset compared to the prior versions of the component resources, typically combined via liftover and variant intersection; for example, we catalog millions of new genetic variants, mostly rare, compared to previous releases. In addition to unrestricted individual-level public release, we provide detailed tutorials for conducting many of the most common quality control steps and analyses with these data in a scalable cloud-computing environment and publicly release this new phased joint callset for use as a haplotype resource in phasing and imputation pipelines. This jointly called reference panel will serve as a key resource to support research of diverse ancestry populations.\nUsing deep long-read RNAseq in Alzheimer's disease brain to assess clinical relevance of RNA isoform diversity\nAuthors: Aguzzoli Heberle, B.; Brandon, J. A.; Page, M. L.; Nations, K. A.; Dikobe, K. I.; White, B. J.; Gordon, L. A.; Fox, G. A.; Wadsworth, M. E.; Doyle, P. H.; Fox, E. J.; Shantaraman, A.; Ryten, M.; Goodwin, S.; Ghiban, E.; Wappel, R.; Mavruk-Eskpelivan, S.; Miller, J. B.; Seyfried, N.; Nelson, P. T.; Fryer, J. D.; Ebbert, M. T. W.\nScore: 25.5, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552162\nDue to alternative splicing, human protein-coding genes average over eight RNA isoforms, resulting in nearly four distinct protein coding sequences per gene. Long-read RNAseq (IsoSeq) enables more accurate quantification of isoforms, shedding light on their specific roles. To assess the clinical relevance of measuring RNA isoform expression, we sequenced 12 aged human frontal cortices (6 Alzheimers disease cases and 6 controls, 50% female) using one Oxford Nanopore PromethION flow cell per sample. Our study uncovered 53 new high-confidence RNA isoforms in clinically relevant genes, including several where the new isoform was one of the most highly expressed for that gene. Specific examples include WDR4 (61%; microcephaly), MYL3 (44%; hypertrophic cardiomyopathy), and MTHFS (25%; major depression, schizophrenia, bipolar disorder). Other notable genes with new high-confidence isoforms include CPLX2 (10%; schizophrenia, epilepsy) and MAOB (9%; targeted for Parkinsons disease treatment). We identified 1,917 clinically relevant genes expressing multiple isoforms in human frontal cortex, where 1,018 had multiple isoforms with different protein coding sequences, demonstrating the need to better understand how individual isoforms from a \"single\" gene are involved in human health and disease. Exactly 98 of the 1,917 genes are implicated in brain-related diseases, including Alzheimers disease genes such as APP (A{beta} precursor protein; five), MAPT (tau protein; four), and BIN1 (eight). We also found 99 differentially expressed RNA isoforms between Alzheimers cases and controls, despite the genes themselves not exhibiting differential expression. Our findings highlight the significant knowledge gaps in RNA isoform diversity and their clinical relevance. Deep long-read RNA sequencing will be necessary going forward to fully comprehend the clinical relevance of individual isoforms for a \"single\" gene.\nPhenotypic variation in maize can be largely explained by genetic variation at transcription factor binding sites\nAuthors: Engelhorn, J.; Snodgrass, S. J.; Kok, A.; Seetharam, A. S.; Schneider, M.; Kiwit, T.; Singh, A.; Banf, M.; Khaipho-Burch, M.; Runcie, D. E.; Camargo, V. S.; Torres-Rodriguez, J. V.; Sun, G.; Stam, M.; Fiorani, F.; Schnable, J. C.; Bass, H. W.; Hufford, M. B.; Stich, B.; Frommer, W. B.; Ross-Ibarra, J.; Hartwig, T.\nScore: 23.2, Published: 2023-08-09 DOI: 10.1101/2023.08.08.551183\nComprehensive maps of functional variation at transcription factor (TF) binding sites (cis-elements) are crucial for elucidating how genotype shapes phenotype. Here we report the construction of a pan-cistrome of the maize leaf under well-watered and drought conditions. We quantified haplotype-specific TF footprints across a pan-genome of 25 maize hybrids and mapped nearly two-hundred thousand genetic variants (termed binding-QTLs) linked to cis-element occupancy. The functional significance of binding-QTLs is supported by three lines of evidence: i) they coincide with known causative loci that regulate traits, including novel alleles of Upright Plant Architecture2, Trehalase1, and the MITE transposon near ZmNAC111 under drought; ii) their footprint bias is mirrored between inbred parents and by ChIP-seq; iii) partitioning genetic variation across genomic regions demonstrates that binding-QTLs capture the majority of heritable trait variation across [~]70% of 143 phenotypes. Our study provides a promising approach to make previously hidden cis-variation more accessible for genetic studies and multi-target engineering of complex traits.\nGenome-scale quantification and prediction of pathogenic stop codon readthrough by small molecules\nAuthors: Toledano, I.; Supek, F.; Lehner, B.\nScore: 22.5, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552350\nPremature termination codons (PTCs) cause [~]10-20% of Mendelian diseases and are the major mechanism of tumor suppressor gene inactivation in cancer. A general strategy to alleviate the effects of PTCs would be to promote translational readthrough. Nonsense suppression by small molecules has proven effective in diverse disease models, but translation into the clinic is hampered by ineffective readthrough of many PTCs. Here we directly tackle the challenge of defining drug efficacy by quantifying readthrough of [~]5,800 human pathogenic stop codons by 8 drugs. We find that different drugs promote readthrough of complementary subsets of PTCs defined by local sequence context. This allows us to build interpretable models that accurately predict drug-induced readthrough genome-wide. Accurate readthrough quantification and prediction will empower clinical trial design and the development of personalized nonsense suppression therapies.\nA computational pipeline for spatial mechano-transcriptomics\nAuthors: Hallou, A.; He, R.; Simons, B. D.; Dumitrascu, B.\nScore: 22.7, Published: 2023-08-05 DOI: 10.1101/2023.08.03.551894\nAdvances in spatial profiling technologies are providing insights into how molecular programs are influenced by local signaling and environmental cues. However, cell fate specification and tissue patterning involve the interplay of biochemical and mechanical feedback. Here, we develop a computational framework that enables the joint statistical analysis of transcriptional and mechanical signals in the context of spatial transcriptomics. To illustrate the application and utility of the approach, we use spatial transcriptomics data from the developing mouse embryo to infer the forces acting on individual cells, and use these results to identify mechanical, morphometric, and gene expression signatures that are predictive of tissue compartment boundaries. In addition, we use geoadditive structural equation modeling to identify gene modules that predict the mechanical behavior of cells in an unbiased manner. This computational framework is easily generalized to other spatial profiling contexts, providing a generic scheme for exploring the interplay of biomolecular and mechanical cues in tissues.\nA genome-wide atlas of human cell morphology\nAuthors: Ramezani, M.; Bauman, J.; Singh, A.; Weisbart, E.; Yong, J.; Lozada, M. E.; Way, G. P.; Kavari, S. L.; Diaz, C.; Haghighi, M.; Perez-Schindler, J.; Batista, T. M.; Claussnitzer, M.; Singh, S.; Cimini, B. A.; Blainey, P. C.; Carpenter, A. E.; Jan, C. H.; Neal, J. T.\nScore: 20.4, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552164\nA key challenge of the modern genomics era is developing data-driven representations of gene function. Here, we present the first unbiased morphology-based genome-wide perturbation atlas in human cells, containing three genome-scale genotype-phenotype maps comprising \u003e20,000 single-gene CRISPR-Cas9-based knockout experiments in \u003e30 million cells. Our optical pooled cell profiling approach (PERISCOPE) combines a de-stainable high-dimensional phenotyping panel (based on Cell Painting1,2) with optical sequencing of molecular barcodes and a scalable open-source analysis pipeline to facilitate massively parallel screening of pooled perturbation libraries. This approach provides high-dimensional phenotypic profiles of individual cells, while simultaneously enabling interrogation of subcellular processes. Our atlas reconstructs known pathways and protein-protein interaction networks, identifies culture media-specific responses to gene knockout, and clusters thousands of human genes by phenotypic similarity. Using this atlas, we identify the poorly-characterized disease-associated transmembrane protein TMEM251/LYSET as a Golgi-resident protein essential for mannose-6-phosphate-dependent trafficking of lysosomal enzymes, showing the power of these representations. In sum, our atlas and screening technology represent a rich and accessible resource for connecting genes to cellular functions at scale.\nCopy number losses of oncogenes and gains of tumor suppressor genes generate common driver events of human cancer\nAuthors: Besedina, E.; Supek, F.\nScore: 12.4, Published: 2023-08-07 DOI: 10.1101/2023.08.05.552104\nCancer driver genes can be under positive selection for various types of genetic alterations, including gain-of-function or loss-of-function point mutations (single-nucleotide variants, SNV), small indels, copy number alterations (CNA) and other structural variants. We studied the landscape of interactions between these different types of alterations affecting the same gene by a statistical method, MutMatch, which can test for significant differences in selection, while accounting for various causes of mutation risk heterogeneity. Analyzing [~]18,000 cancer exomes and genomes, we found that known oncogenes simultaneously exhibit signatures of positive selection and also negative selection, where the latter can mask the former. Consistently, focussing on known positively selected regions identifies additional tumor types where an oncogene is relevant. Next, we characterized the landscape of CNA-dependent selection effects, revealing a general trend of increased positive selection on oncogene mutations not only upon CNA gains but also upon CNA deletions. Conversely, we observe a positive interaction between mutations and CNA gains in tumor suppressor genes. Thus, two-hit events involving point mutations and CNA are universally observed on driver genes regardless of the type of CNA, and may signal new therapeutic opportunities that have been overlooked. An explicit focus on the somatic CNA two-hit events can identify additional driver genes relevant to a tumor type. By a global analysis of CNA-selection effects across many driver genes and tissues, we identified at least four independently varying signatures, and thus generated a comprehensive, data-driven classification of cancer genes by mechanisms of (in)activation by genetic alterations.\nRibosomal DNA copy number is associated with body mass in humans and other mammals\nAuthors: Law, P. P.; Rodriguez-Algarra, F.; Asenius, F.; Gregori, M.; Seabourne, R. A.; Yildizoglu, S.; Miller, J. R.; Mesnage, R.; Antoniou, M.; Li, W.; Tan, Q.; Hillman, S.; Rakyan, V. K.; Williams, D. J.; Holland, M. L.\nScore: 11.6, Published: 2023-08-06 DOI: 10.1101/2023.08.03.551775\nBody mass results from a complex interplay between genetics and environment. The contribution of genetic variation to body mass has been extensively studied, but due to the technical limitations of platforms used for population scale studies, repetitive parts of the genome have not previously been considered. Here we applied genome-wide approaches to identify an association between adult body mass and the copy number (CN) of 45S-ribosomal DNA (rDNA). rDNA codes for the rRNA components of the ribosome and exists in hundreds of copies/cell in mammals. Inter-individual variation in rDNA CN has not previously been documented but has not been associated with a mammalian phenotype. Here, we show that germ-line inherited rDNA CN is associated with post-pubertal growth rate in rats and BMI in adult humans. rDNA CN variation is not associated with rRNA transcription rates in adult tissues, suggesting the mechanistic link occurs earlier in development. This is supported by our observation that it is growth rate and body mass, rather than obesity that are associated with copy number and that these phenotypes emerge prior to adulthood. We present hypotheses for future investigation into the mechanistic basis of this association.\nModel-based inference of RNA velocity modules improves cell fate prediction\nAuthors: Aivazidis, A.; Memi, F.; Kleshchevnikov, V.; Clarke, B.; Stegle, O.; Bayraktar, O.\nScore: 8.8, Published: 2023-08-05 DOI: 10.1101/2023.08.03.551650\nRNA velocity is a powerful paradigm that exploits the temporal information contained in spliced and unspliced RNA counts to infer transcriptional dynamics. Existing velocity models either rely on coarse biophysical simplifications or require extensive numerical approximations to solve the underlying differential equations. This results in loss of accuracy in challenging settings, such as complex or weak transcription rate changes across cellular trajectories. Here, we present cell2fate, a formulation of RNA velocity based on a linearization of the velocity ODE, which allows solving a biophysically accurate model in a fully Bayesian fashion. As a result, cell2fate decomposes the RNA velocity solutions into modules, which provides a new biophysical connection between RNA velocity and statistical dimensionality reduction. We comprehensively benchmark cell2fate in real-world settings, demonstrating enhanced interpretability and increased power to reconstruct complex dynamics and weak dynamical signals in rare and mature cell types. Finally, we apply cell2fate to a newly generated dataset from the developing human brain, where we spatially map RNA velocity modules onto the tissue architecture, thereby connecting the spatial organisation of tissues with temporal dynamics of transcription.\n",
  "wordCount" : "2341",
  "inLanguage": "en",
  "datePublished": "2023-08-13T10:36:01Z",
  "dateModified": "2023-08-13T10:36:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on August 13, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.552325">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.552325" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.552325">
        <p class="paperTitle">Phage-plasmids promote genetic exchanges between phages and plasmids and create novel ones.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.552325" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.552325" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pfeifer, E.; Rocha, E. P. C.</p>
        <p class="info">Score: 27.5, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.552325' target='https://doi.org/10.1101/2023.08.08.552325'> 10.1101/2023.08.08.552325</a></p>
        <p class="abstract">Phages and plasmids have key roles in bacterial evolution and are usually very different. Yet, they must recombine, since they sometimes carry nearly identical accessory genes. Phage-plasmids are both plasmids and phages and we found they connect them by being at their interface in the network of gene repertoire relatedness. We searched for recombining genes in all three types of elements and found that phage-plasmid exchange rates are between those of plasmids (highest) and phages (lowest). Phage-plasmids drive gene flow between phages and plasmids, including that of antibiotic resistance and defense systems. Genetic exchanges and gene inactivation resulted in P1-like phage-plasmids turning into integrative prophages or just plasmids. The latter acquired conjugation-related functions thereby transitioning from viral to conjugation-driven horizontal transfer. Phage-plasmids have thus a key role in spreading traits across mobile genetic elements and their hosts, while facilitating conversions of one type of element into another.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.23.525248">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.23.525248" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.23.525248">
        <p class="paperTitle">A harmonized public resource of deeply sequenced diverse human genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.23.525248" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.23.525248" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koenig, Z.; Yohannes, M. T.; Nkambule, L. L.; Goodrich, J. K.; Kim, H. A.; Zhao, X.; Wilson, M. W.; Tiao, G.; Hao, S. P.; Sahakian, N.; Chao, K. R.; gnomAD Project Consortium,  ; Rehm, H.; Neale, B.; Talkowski, M. E.; Daly, M. J.; Brand, H.; Karczewski, K.; Atkinson, E. G.; Martin, A.</p>
        <p class="info">Score: 153.3, Published: 2023-08-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.23.525248' target='https://doi.org/10.1101/2023.01.23.525248'> 10.1101/2023.01.23.525248</a></p>
        <p class="abstract">Underrepresented populations are often excluded from genomic studies due in part to a lack of resources supporting their analyses. The 1000 Genomes Project (1kGP) and Human Genome Diversity Project (HGDP), which have recently been sequenced to high coverage, are valuable genomic resources because of the global diversity they capture and their open data sharing policies. Here, we harmonized a high quality set of 4,096 whole genomes from HGDP and 1kGP with data from gnomAD and identified over 159 million high-quality SNVs, indels, and SVs. We performed a detailed ancestry analysis of this cohort, characterizing population structure and patterns of admixture across populations, analyzing site frequency spectra, and measuring variant counts at global and subcontinental levels. We also demonstrate substantial added value from this dataset compared to the prior versions of the component resources, typically combined via liftover and variant intersection; for example, we catalog millions of new genetic variants, mostly rare, compared to previous releases. In addition to unrestricted individual-level public release, we provide detailed tutorials for conducting many of the most common quality control steps and analyses with these data in a scalable cloud-computing environment and publicly release this new phased joint callset for use as a haplotype resource in phasing and imputation pipelines. This jointly called reference panel will serve as a key resource to support research of diverse ancestry populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552162">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552162" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552162">
        <p class="paperTitle">Using deep long-read RNAseq in Alzheimer&#39;s disease brain to assess clinical relevance of RNA isoform diversity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552162" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552162" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aguzzoli Heberle, B.; Brandon, J. A.; Page, M. L.; Nations, K. A.; Dikobe, K. I.; White, B. J.; Gordon, L. A.; Fox, G. A.; Wadsworth, M. E.; Doyle, P. H.; Fox, E. J.; Shantaraman, A.; Ryten, M.; Goodwin, S.; Ghiban, E.; Wappel, R.; Mavruk-Eskpelivan, S.; Miller, J. B.; Seyfried, N.; Nelson, P. T.; Fryer, J. D.; Ebbert, M. T. W.</p>
        <p class="info">Score: 25.5, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552162' target='https://doi.org/10.1101/2023.08.06.552162'> 10.1101/2023.08.06.552162</a></p>
        <p class="abstract">Due to alternative splicing, human protein-coding genes average over eight RNA isoforms, resulting in nearly four distinct protein coding sequences per gene. Long-read RNAseq (IsoSeq) enables more accurate quantification of isoforms, shedding light on their specific roles. To assess the clinical relevance of measuring RNA isoform expression, we sequenced 12 aged human frontal cortices (6 Alzheimers disease cases and 6 controls, 50% female) using one Oxford Nanopore PromethION flow cell per sample. Our study uncovered 53 new high-confidence RNA isoforms in clinically relevant genes, including several where the new isoform was one of the most highly expressed for that gene. Specific examples include WDR4 (61%; microcephaly), MYL3 (44%; hypertrophic cardiomyopathy), and MTHFS (25%; major depression, schizophrenia, bipolar disorder). Other notable genes with new high-confidence isoforms include CPLX2 (10%; schizophrenia, epilepsy) and MAOB (9%; targeted for Parkinsons disease treatment). We identified 1,917 clinically relevant genes expressing multiple isoforms in human frontal cortex, where 1,018 had multiple isoforms with different protein coding sequences, demonstrating the need to better understand how individual isoforms from a &#34;single&#34; gene are involved in human health and disease. Exactly 98 of the 1,917 genes are implicated in brain-related diseases, including Alzheimers disease genes such as APP (A{beta} precursor protein; five), MAPT (tau protein; four), and BIN1 (eight). We also found 99 differentially expressed RNA isoforms between Alzheimers cases and controls, despite the genes themselves not exhibiting differential expression. Our findings highlight the significant knowledge gaps in RNA isoform diversity and their clinical relevance. Deep long-read RNA sequencing will be necessary going forward to fully comprehend the clinical relevance of individual isoforms for a &#34;single&#34; gene.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.08.551183">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.08.551183" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.08.551183">
        <p class="paperTitle">Phenotypic variation in maize can be largely explained by genetic variation at transcription factor binding sites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.08.551183" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.08.551183" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Engelhorn, J.; Snodgrass, S. J.; Kok, A.; Seetharam, A. S.; Schneider, M.; Kiwit, T.; Singh, A.; Banf, M.; Khaipho-Burch, M.; Runcie, D. E.; Camargo, V. S.; Torres-Rodriguez, J. V.; Sun, G.; Stam, M.; Fiorani, F.; Schnable, J. C.; Bass, H. W.; Hufford, M. B.; Stich, B.; Frommer, W. B.; Ross-Ibarra, J.; Hartwig, T.</p>
        <p class="info">Score: 23.2, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.08.551183' target='https://doi.org/10.1101/2023.08.08.551183'> 10.1101/2023.08.08.551183</a></p>
        <p class="abstract">Comprehensive maps of functional variation at transcription factor (TF) binding sites (cis-elements) are crucial for elucidating how genotype shapes phenotype. Here we report the construction of a pan-cistrome of the maize leaf under well-watered and drought conditions. We quantified haplotype-specific TF footprints across a pan-genome of 25 maize hybrids and mapped nearly two-hundred thousand genetic variants (termed binding-QTLs) linked to cis-element occupancy. The functional significance of binding-QTLs is supported by three lines of evidence: i) they coincide with known causative loci that regulate traits, including novel alleles of Upright Plant Architecture2, Trehalase1, and the MITE transposon near ZmNAC111 under drought; ii) their footprint bias is mirrored between inbred parents and by ChIP-seq; iii) partitioning genetic variation across genomic regions demonstrates that binding-QTLs capture the majority of heritable trait variation across [~]70% of 143 phenotypes. Our study provides a promising approach to make previously hidden cis-variation more accessible for genetic studies and multi-target engineering of complex traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552350">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552350" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552350">
        <p class="paperTitle">Genome-scale quantification and prediction of pathogenic stop codon readthrough by small molecules</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552350" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552350" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Toledano, I.; Supek, F.; Lehner, B.</p>
        <p class="info">Score: 22.5, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552350' target='https://doi.org/10.1101/2023.08.07.552350'> 10.1101/2023.08.07.552350</a></p>
        <p class="abstract">Premature termination codons (PTCs) cause [~]10-20% of Mendelian diseases and are the major mechanism of tumor suppressor gene inactivation in cancer. A general strategy to alleviate the effects of PTCs would be to promote translational readthrough. Nonsense suppression by small molecules has proven effective in diverse disease models, but translation into the clinic is hampered by ineffective readthrough of many PTCs. Here we directly tackle the challenge of defining drug efficacy by quantifying readthrough of [~]5,800 human pathogenic stop codons by 8 drugs. We find that different drugs promote readthrough of complementary subsets of PTCs defined by local sequence context. This allows us to build interpretable models that accurately predict drug-induced readthrough genome-wide. Accurate readthrough quantification and prediction will empower clinical trial design and the development of personalized nonsense suppression therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551894">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551894" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551894">
        <p class="paperTitle">A computational pipeline for spatial mechano-transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551894" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551894" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hallou, A.; He, R.; Simons, B. D.; Dumitrascu, B.</p>
        <p class="info">Score: 22.7, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551894' target='https://doi.org/10.1101/2023.08.03.551894'> 10.1101/2023.08.03.551894</a></p>
        <p class="abstract">Advances in spatial profiling technologies are providing insights into how molecular programs are influenced by local signaling and environmental cues. However, cell fate specification and tissue patterning involve the interplay of biochemical and mechanical feedback. Here, we develop a computational framework that enables the joint statistical analysis of transcriptional and mechanical signals in the context of spatial transcriptomics. To illustrate the application and utility of the approach, we use spatial transcriptomics data from the developing mouse embryo to infer the forces acting on individual cells, and use these results to identify mechanical, morphometric, and gene expression signatures that are predictive of tissue compartment boundaries. In addition, we use geoadditive structural equation modeling to identify gene modules that predict the mechanical behavior of cells in an unbiased manner. This computational framework is easily generalized to other spatial profiling contexts, providing a generic scheme for exploring the interplay of biomolecular and mechanical cues in tissues.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552164">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552164" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552164">
        <p class="paperTitle">A genome-wide atlas of human cell morphology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552164" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552164" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramezani, M.; Bauman, J.; Singh, A.; Weisbart, E.; Yong, J.; Lozada, M. E.; Way, G. P.; Kavari, S. L.; Diaz, C.; Haghighi, M.; Perez-Schindler, J.; Batista, T. M.; Claussnitzer, M.; Singh, S.; Cimini, B. A.; Blainey, P. C.; Carpenter, A. E.; Jan, C. H.; Neal, J. T.</p>
        <p class="info">Score: 20.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552164' target='https://doi.org/10.1101/2023.08.06.552164'> 10.1101/2023.08.06.552164</a></p>
        <p class="abstract">A key challenge of the modern genomics era is developing data-driven representations of gene function. Here, we present the first unbiased morphology-based genome-wide perturbation atlas in human cells, containing three genome-scale genotype-phenotype maps comprising &gt;20,000 single-gene CRISPR-Cas9-based knockout experiments in &gt;30 million cells. Our optical pooled cell profiling approach (PERISCOPE) combines a de-stainable high-dimensional phenotyping panel (based on Cell Painting1,2) with optical sequencing of molecular barcodes and a scalable open-source analysis pipeline to facilitate massively parallel screening of pooled perturbation libraries. This approach provides high-dimensional phenotypic profiles of individual cells, while simultaneously enabling interrogation of subcellular processes. Our atlas reconstructs known pathways and protein-protein interaction networks, identifies culture media-specific responses to gene knockout, and clusters thousands of human genes by phenotypic similarity. Using this atlas, we identify the poorly-characterized disease-associated transmembrane protein TMEM251/LYSET as a Golgi-resident protein essential for mannose-6-phosphate-dependent trafficking of lysosomal enzymes, showing the power of these representations. In sum, our atlas and screening technology represent a rich and accessible resource for connecting genes to cellular functions at scale.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552104">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552104" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552104">
        <p class="paperTitle">Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver events of human cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552104" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552104" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Besedina, E.; Supek, F.</p>
        <p class="info">Score: 12.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552104' target='https://doi.org/10.1101/2023.08.05.552104'> 10.1101/2023.08.05.552104</a></p>
        <p class="abstract">Cancer driver genes can be under positive selection for various types of genetic alterations, including gain-of-function or loss-of-function point mutations (single-nucleotide variants, SNV), small indels, copy number alterations (CNA) and other structural variants. We studied the landscape of interactions between these different types of alterations affecting the same gene by a statistical method, MutMatch, which can test for significant differences in selection, while accounting for various causes of mutation risk heterogeneity. Analyzing [~]18,000 cancer exomes and genomes, we found that known oncogenes simultaneously exhibit signatures of positive selection and also negative selection, where the latter can mask the former. Consistently, focussing on known positively selected regions identifies additional tumor types where an oncogene is relevant. Next, we characterized the landscape of CNA-dependent selection effects, revealing a general trend of increased positive selection on oncogene mutations not only upon CNA gains but also upon CNA deletions. Conversely, we observe a positive interaction between mutations and CNA gains in tumor suppressor genes. Thus, two-hit events involving point mutations and CNA are universally observed on driver genes regardless of the type of CNA, and may signal new therapeutic opportunities that have been overlooked. An explicit focus on the somatic CNA two-hit events can identify additional driver genes relevant to a tumor type. By a global analysis of CNA-selection effects across many driver genes and tissues, we identified at least four independently varying signatures, and thus generated a comprehensive, data-driven classification of cancer genes by mechanisms of (in)activation by genetic alterations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551775">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551775" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551775">
        <p class="paperTitle">Ribosomal DNA copy number is associated with body mass in humans and other mammals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551775" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551775" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Law, P. P.; Rodriguez-Algarra, F.; Asenius, F.; Gregori, M.; Seabourne, R. A.; Yildizoglu, S.; Miller, J. R.; Mesnage, R.; Antoniou, M.; Li, W.; Tan, Q.; Hillman, S.; Rakyan, V. K.; Williams, D. J.; Holland, M. L.</p>
        <p class="info">Score: 11.6, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551775' target='https://doi.org/10.1101/2023.08.03.551775'> 10.1101/2023.08.03.551775</a></p>
        <p class="abstract">Body mass results from a complex interplay between genetics and environment. The contribution of genetic variation to body mass has been extensively studied, but due to the technical limitations of platforms used for population scale studies, repetitive parts of the genome have not previously been considered. Here we applied genome-wide approaches to identify an association between adult body mass and the copy number (CN) of 45S-ribosomal DNA (rDNA). rDNA codes for the rRNA components of the ribosome and exists in hundreds of copies/cell in mammals. Inter-individual variation in rDNA CN has not previously been documented but has not been associated with a mammalian phenotype. Here, we show that germ-line inherited rDNA CN is associated with post-pubertal growth rate in rats and BMI in adult humans. rDNA CN variation is not associated with rRNA transcription rates in adult tissues, suggesting the mechanistic link occurs earlier in development. This is supported by our observation that it is growth rate and body mass, rather than obesity that are associated with copy number and that these phenotypes emerge prior to adulthood. We present hypotheses for future investigation into the mechanistic basis of this association.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.03.551650">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.03.551650" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.03.551650">
        <p class="paperTitle">Model-based inference of RNA velocity modules improves cell fate prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.03.551650" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.03.551650" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Aivazidis, A.; Memi, F.; Kleshchevnikov, V.; Clarke, B.; Stegle, O.; Bayraktar, O.</p>
        <p class="info">Score: 8.8, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.03.551650' target='https://doi.org/10.1101/2023.08.03.551650'> 10.1101/2023.08.03.551650</a></p>
        <p class="abstract">RNA velocity is a powerful paradigm that exploits the temporal information contained in spliced and unspliced RNA counts to infer transcriptional dynamics. Existing velocity models either rely on coarse biophysical simplifications or require extensive numerical approximations to solve the underlying differential equations. This results in loss of accuracy in challenging settings, such as complex or weak transcription rate changes across cellular trajectories. Here, we present cell2fate, a formulation of RNA velocity based on a linearization of the velocity ODE, which allows solving a biophysically accurate model in a fully Bayesian fashion. As a result, cell2fate decomposes the RNA velocity solutions into modules, which provides a new biophysical connection between RNA velocity and statistical dimensionality reduction. We comprehensively benchmark cell2fate in real-world settings, demonstrating enhanced interpretability and increased power to reconstruct complex dynamics and weak dynamical signals in rare and mature cell types. Finally, we apply cell2fate to a newly generated dataset from the developing human brain, where we spatially map RNA velocity modules onto the tissue architecture, thereby connecting the spatial organisation of tissues with temporal dynamics of transcription.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
