<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms
Authors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.
Score: 54.0, Published: 2024-01-06 DOI: 10.1101/2024.01.02.573821
Research and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms
Authors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.
Score: 54.0, Published: 2024-01-06 DOI: 10.1101/2024.01.02.573821
Research and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms
Authors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.
Score: 54.0, Published: 2024-01-06 DOI: 10.1101/2024.01.02.573821
Research and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms\nAuthors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.\nScore: 54.0, Published: 2024-01-06 DOI: 10.1101/2024.01.02.573821\nResearch and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance.",
  "keywords": [
    
  ],
  "articleBody": " Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms\nAuthors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.\nScore: 54.0, Published: 2024-01-06 DOI: 10.1101/2024.01.02.573821\nResearch and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance. This necessitates a framework to identify all types of variants independent of their size (e.g., SNV/SV) or location (e.g., repeats). Here we present DRAGEN that utilizes novel methods based on multigenomes, hardware acceleration, and machine learning based variant detection to provide novel insights into individual genomes with [~]30min computation time (from raw reads to variant detection). DRAGEN outperforms all other state-of-the-art methods in speed and accuracy across all variant types (SNV, indel, STR, SV, CNV) and further incorporates specialized methods to obtain key insights in medically relevant genes (e.g., HLA, SMN, GBA). We showcase DRAGEN across 3,202 genomes and demonstrate its scalability, accuracy, and innovations to further advance the integration of comprehensive genomics for research and medical applications.\nRNA polymerases reshape chromatin and coordinate transcription on individual fibers\nAuthors: Tullius, T. W.; Isaac, R. S.; Ranchalis, J.; Dubocanin, D.; Churchman, L. S.; Stergachis, A. B.\nScore: 37.5, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573133\nDuring eukaryotic transcription, RNA polymerases must initiate and pause within a crowded, complex environment, surrounded by nucleosomes and other transcriptional activity. This environment creates a spatial arrangement along individual chromatin fibers ripe for both competition and coordination, yet these relationships remain largely unknown owing to the inherent limitations of traditional structural and sequencing methodologies. To address these limitations, we employed long-read chromatin fiber sequencing (Fiber-seq) to visualize RNA polymerases within their native chromatin context at single-molecule and near single-nucleotide resolution along up to 30 kb fibers. We demonstrate that Fiber-seq enables the identification of single-molecule RNA Polymerase (Pol) II and III transcription associated foot-prints, which, in aggregate, mirror bulk short-read sequencing-based measurements of transcription. We show that Pol II pausing destabilizes downstream nucleosomes, with frequently paused genes maintaining a short-term memory of these destabilized nucleosomes. Furthermore, we demonstrate pervasive direct coordination and anti-coordination between nearby Pol II genes, Pol III genes, transcribed enhancers, and insulator elements. This coordination is largely limited to spatially organized elements within 5 kb of each other, implicating short-range chromatin environments as a predominant determinant of coordinated polymerase initiation. Overall, transcription initiation reshapes surrounding nucleosome architecture and coordinates nearby transcriptional machinery along individual chromatin fibers.\nThe majority of somatic mutations in fruit trees are layer-specific\nAuthors: Goel, M.; Campoy, J. A.; Krause, K.; Baus, L. C.; Sahu, A.; Sun, H.; Walkemeier, B.; Marek, M.; Beaudry, R.; Ruiz, D.; Huettel, B.; Schneeberger, K.\nScore: 33.3, Published: 2024-01-04 DOI: 10.1101/2024.01.04.573414\nPlant meristems consist of three distinct stem cells layers. The identities of these layers are maintained in developing organs. Somatic mutations in the meristem thus propagate only into the respective layers of the differentiated organs. We analysed the genomes of individual cell layers in an apricot tree and found an unexpectedly high mutation load, where \u003e90% of the somatic mutations were layer-specific. The layers had similar mutation spectra but layer 1 (epidermis) had substantially more mutations than layer 2 (mesocarp). Most somatic mutations were shared between fruits and neighbouring leaves corroborating their meristematic origin. Single-cell transcriptomics revealed their layer-specific effects forming the basis for layer-specific phenotypes. Our work unveils the hidden abundance of layer-specific meristematic mutations and provides insights into their identification and understanding. One sentence summarySomatic mutations in fruit trees occur in specific cell layers.\nmRNA initiation and termination are spatially coordinated\nAuthors: Calvo-Roitberg, E.; Carroll, C. L.; Venev, S. V.; Kim, G.; Mick, S. T.; Dekker, J.; Fiszbein, A.; Pai, A. A.\nScore: 29.6, Published: 2024-01-07 DOI: 10.1101/2024.01.05.574404\nThe expression of a precise mRNA transcriptome is crucial for establishing cell identity and function, with dozens of alternative isoforms produced for a single gene sequence. The regulation of mRNA isoform usage occurs by the coordination of co-transcriptional mRNA processing mechanisms across a gene. Decisions involved in mRNA initiation and termination underlie the largest extent of mRNA isoform diversity, but little is known about any relationships between decisions at both ends of mRNA molecules. Here, we systematically profile the joint usage of mRNA transcription start sites (TSSs) and polyadenylation sites (PASs) across tissues and species. Using both short and long read RNA-seq data, we observe that mRNAs preferentially using upstream TSSs also tend to use upstream PASs, and congruently, the usage of downstream sites is similarly paired. This observation suggests that mRNA 5 end choice may directly influence mRNA 3 ends. Our results suggest a novel \"Positional Initiation-Termination Axis\" (PITA), in which the usage of alternative terminal sites are coupled based on the order in which they appear in the genome. PITA isoforms are more likely to encode alternative protein domains and use conserved sites. PITA is strongly associated with the length of genomic features, such that PITA is enriched in longer genes with more area devoted to regions that regulate alternative 5 or 3 ends. Strikingly, we found that PITA genes are more likely than non-PITA genes to have multiple, overlapping chromatin structural domains related to pairing of ordinally coupled start and end sites. In turn, PITA coupling is also associated with fast RNA Polymerase II (RNAPII) trafficking across these long gene regions. Our findings indicate that a combination of spatial and kinetic mechanisms couple transcription initiation and mRNA 3 end decisions based on ordinal position to define the expression mRNA isoforms.\nSomatic cancer driver mutations are enriched and associated with inflammatory states in Alzheimer's disease microglia\nAuthors: Huang, A. Y.; Zhou, Z.; Talukdar, M.; Miller, M. B.; Chhouk, B.; Enyenihi, L.; Rosen, I.; Stronge, E.; Zhao, B.; Kim, D.; Choi, J.; Khoshkhoo, S.; Kim, J.; Ganz, J.; Travaglini, K. J.; Gabitto, M.; Hodge, R. D.; Kaplan, E.; Lein, E.; De Jager, P.; Bennett, D. A.; Lee, E. A.; Walsh, C. A.\nScore: 13.1, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574078\nAlzheimers disease (AD) is an age-associated neurodegenerative disorder characterized by progressive neuronal loss and pathological accumulation of the misfolded proteins amyloid-{beta} and tau1,2. Neuroinflammation mediated by microglia and brain-resident macrophages plays a crucial role in AD pathogenesis1-5, though the mechanisms by which age, genes, and other risk factors interact remain largely unknown. Somatic mutations accumulate with age and lead to clonal expansion of many cell types, contributing to cancer and many non-cancer diseases6,7. Here we studied somatic mutation in normal aged and AD brains by three orthogonal methods and in three independent AD cohorts. Analysis of bulk RNA sequencing data from 866 samples from different brain regions revealed significantly higher ([~]two-fold) overall burdens of somatic single-nucleotide variants (sSNVs) in AD brains compared to age-matched controls. Molecular-barcoded deep (\u003e1000X) gene panel sequencing of 311 prefrontal cortex samples showed enrichment of sSNVs and somatic insertions and deletions (sIndels) in cancer driver genes in AD brain compared to control, with recurrent, and often multiple, mutations in genes implicated in clonal hematopoiesis (CH)8,9. Pathogenic sSNVs were enriched in CSF1R+ microglia of AD brains, and the high proportion of microglia (up to 40%) carrying some sSNVs in cancer driver genes suggests mutation-driven microglial clonal expansion (MiCE). Analysis of single-nucleus RNA sequencing (snRNAseq) from temporal neocortex of 62 additional AD cases and controls exhibited nominally increased mosaic chromosomal alterations (mCAs) associated with CH10,11. Microglia carrying mCA showed upregulated pro-inflammatory genes, resembling the transcriptomic features of disease-associated microglia (DAM) in AD. Our results suggest that somatic driver mutations in microglia are common with normal aging but further enriched in AD brain, driving MiCE with inflammatory and DAM signatures. Our findings provide the first insights into microglial clonal dynamics in AD and identify potential new approaches to AD diagnosis and therapy.\nCell-cycle dependent DNA repair and replication unifies patterns of chromosome instability\nAuthors: Lu, B.; Winnall, S.; Cross, W.; Barnes, C. P.\nScore: 12.1, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574048\nChromosomal instability (CIN) is pervasive in human tumours and often leads to structural or numerical chromosomal aberrations. Somatic structural variants (SVs) are intimately related to copy number alterations but the two types of variant are often studied independently. In addition, despite numerous studies on detecting various SV patterns, there are still no general quantitative models of SV generation. To address this issue, we develop a computational cell-cycle model for the generation of SVs from end-joining repair and replication after double strand break formation. Our model provides quantitative information on the relationship between breakage fusion bridge cycle, chromothripsis, seismic amplification, and extra-chromosomal circular DNA. Given single-cell whole-genome sequencing data, the model also allows us to infer important parameters in SV generation with Bayesian inference. Our quantitative framework unifies disparate genomic patterns resulted from CIN, provides a null mutational model for SV, and reveals new insights into the impact of genome rearrangement on tumour evolution.\nConvergence and global epidemiology of Klebsiella pneumoniae plasmids harbouring the iuc3 virulence locus.\nAuthors: Gibbon, M. J.; Couto, N.; Cozens, K.; Habib, S.; Cowley, L.; Aanensen, D.; Corander, J.; Thorpe, H.; Hetland, M. A.; Sassera, D.; Merla, C.; Corbella, M.; Ferrari, C.; Turner, K. M.; Sirikancha, K.; Dulyayangkul, P.; Alhusein, N.; Charoenlap, N.; Thamlikitikul, V.; Avison, M. B.; Feil, E. J.\nScore: 11.2, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574329\nBackgroundKlebsiella pneumoniae (Kp) is an important pathogen of humans and animals, and recent reports of convergent strains that carry both virulence and antimicrobial resistance genes (ARGs) have raised serious public health concern. The plasmid-borne iuc locus, encoding the siderophore aerobactin, is a key virulence factor in this species. The variant iuc3 is associated with porcine and human clinical isolates and is carried by mostly uncharacterised IncF plasmids. MethodsWe used a combination of short-read and long-read sequencing to characterise IncFIB(K)/IncFII iuc3-carrying plasmids harboured by 79 Kp isolates and one K. oxytoca isolate recovered as part of two large One-Health studies in Italy (SpARK) and Thailand (OH-DART). Adding data from public repositories gave a combined dataset of 517 iuc3 isolates, and the plasmids were analysed using both clustering and phylogenetic methods. FindingsWe note seven large, convergent, plasmids from Thailand that have emerged through the hybridisation of co-circulating plasmids harbouring iuc3 and antimicrobial resistance genes (ARGs) encoding extended-spectrum beta-lactamases (ESBLs). We were also able to identify putative parental plasmids which were mostly associated with two neighbouring meat markets, as were the hybrid plasmids. Clustering and global phylogenetic analyses resolved an iuc3 plasmid sub-group circulating throughout Asia, with occasional examples in Europe and elsewhere. This variant carries multiple ARGs and is commonly harboured by clinical isolates, thus warranting targeted plasmid surveillance. InterpretationOur study reveals that plasmid hybridisation leading to the convergence of resistance and virulence traits may be very common, even in non-clinical ( One-Health) settings. Population-scale plasmid genomics makes it possible to identify putative parental plasmids, which will help to identify plasmid types that are most likely to hybridise, and what the selective consequences may be for the plasmid and host. A distinct iuc3 plasmid sub-variant is associated with clinical isolates in Asia which requires close monitoring. Research In ContextMultiple reports of convergent clones of Klebsiella pneumoniae that combine both hypervirulence and multidrug resistance (MDR-hvKp) have been published recently; a PubMed search in November 2023 using the key words convergence Klebsiella pneumoniae returned 143 papers, 99 of which were published from 2020 onwards. Our study demonstrates that the hybridisation of plasmids carrying AMR and virulence genes is a frequent, ongoing, process in natural populations. The subsequent transfer of plasmids conferring both traits is thus likely to be a key driver behind the spread of convergent strains. Our study also provides an exemplar of how hybrid assemblies can facilitate large-scale global genomic plasmid epidemiology. Evidence before the studyAlthough multiple recent reports highlight the emergence and spread of convergent Kp strains, the confluence of resistance and virulence genes within the same plasmid has not been studied at a population level, and putative parental plasmids are rarely identified. Moreover, there have been few high-resolution genomic epidemiology studies on closely related plasmids using both long and short-read data on a global scale. Added valueWe more than double the number of complete sequences available for plasmids harbouring iuc3 from 58 to 139 and provide evidence on the host lineages most likely to harbour these plasmids (e.g., ST35), and epidemiological source (e.g., pig, wild animal, human). Our comparative analysis of phylogenetic and clustering approaches will help to inform future plasmid epidemiological studies. ImplicationsThe hybridisation of plasmids harbouring virulence and resistance genes occurs frequently in natural populations, even within One-Health settings. However, the selective drivers (if any) and evolutionary consequences of this phenomenon are unclear. There is clear utility in generating closed plasmid genomes on a population scale, and targeted plasmid surveillance on a clinical sub-variant of iuc3 plasmids is warranted.\nComparative genomics of Cryptococcus and Kwoniella reveals pathogenesis evolution and contrasting modes of karyotype evolution via chromosome fusion or intercentromeric recombination\nAuthors: Coelho, M. A.; David-Palma, M.; Shea, T.; Bowers, K.; McGinley-Smith, S.; Mohammad, A. W.; Gnirke, A.; Yurkov, A. M.; Nowrousian, M.; Sun, S.; Cuomo, C. A.; Heitman, J.\nScore: 23.0, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573464\nA large-scale comparative genomic analysis was conducted for the global human fungal pathogens within the Cryptococcus genus, compared to non-pathogenic Cryptococcus species, and related species from the sister genus Kwoniella. Chromosome-level genome assemblies were generated for multiple species of both genera, resulting in a dataset encompassing virtually all of their known diversity. Although Cryptococcus and Kwoniella have comparable genome sizes (about 19.2 and 22.9 Mb) and similar gene content, hinting at pre-adaptive pathogenic potential, our analysis found evidence in pathogenic Cryptococcus species of specific examples of gene gain (via horizontal gene transfer) and gene loss, which might represent evolutionary signatures of pathogenic development. Genome analysis also revealed a significant variation in chromosome number and structure between the two genera. By combining synteny analysis and experimental centromere validation, we found that most Cryptococcus species have 14 chromosomes, whereas most Kwoniella species have fewer (11, 8, 5 or even as few as 3). Reduced chromosome number in Kwoniella is associated with formation of giant chromosomes (up to 18 Mb) through repeated chromosome fusion events, each marked by a pericentric inversion and centromere loss. While similar chromosome inversion-fusion patterns were observed in all Kwoniella species with fewer than 14 chromosomes, no such pattern was detected in Cryptococcus. Instead, Cryptococcus species with less than 14 chromosomes, underwent chromosome reductions primarily through rearrangements associated with the loss of repeat-rich centromeres. Additionally, Cryptococcus genomes exhibited frequent interchromosomal translocations, including intercentromeric recombination facilitated by transposons shared between centromeres. Taken together, our findings advance our understanding of genomic changes possibly associated with pathogenicity in Cryptococcus and provide a foundation to elucidate mechanisms of centromere loss and chromosome fusion driving distinct karyotypes in closely related fungal species, including prominent global human pathogens.\nStable chromosome configuration and loop-based polarization in animal symbionts\nAuthors: Viehboeck, T.; Weber, P. M.; Krause, N.; Varoquaux, N.; Boccard, F.; Junier, I.; Bulgheresi, S.; Lioy, V. S.\nScore: 7.6, Published: 2024-01-03 DOI: 10.1101/2023.12.21.572873\nChromosome partitioning precedes the division of the cytoplasm, and its evolution is linked with the positioning of the division plane. So far, bacterial chromosome biology has heavily focused on transversally dividing, free-living ones. Here, we determined the chromosome organization of three longitudinally dividing Neisseriaceae exclusively inhabiting the oral cavity of mammals. We showed that in all three multicellular bacteria the origin of DNA replication is invariably located at the host-attached (proximal) pole. Next, 3C-seq revealed loop-based folding of the ori region in Alysiella filiformis and Simonsiella muelleri. Moreover, genes involved in cell motility, piliation and signal transduction mechanisms were specifically looped when transcriptionally and translationally active cells adhered to a substrate, but not when cultured in liquid. Overall, we propose that proximal positioning of the ori and loop-based folding of its surrounding DNA may mediate localized translation of proteins involved in host colonization.\nA validated pangenome-scale metabolic model for the Klebsiella pneumoniae species complex\nAuthors: Cooper, H. B.; Vezina, B.; Hawkey, J.; Passet, V.; Lopez-Fernandez, S.; Monk, J. M.; Brisse, S.; Holt, K. E.; Wyres, K. L.\nScore: 8.3, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572682\nThe Klebsiella pneumoniae Species Complex (KpSC) is a major source of nosocomial infections globally with high rates of resistance to antimicrobials. Consequently, there is growing interest in understanding virulence factors and their association with cellular metabolic processes for developing novel anti-KpSC therapeutics. Phenotypic assays have revealed metabolic diversity within the KpSC, but metabolism research has been neglected due to experiments being difficult and cost-intensive. Genome-scale metabolic models (GSMMs) represent a rapid and scalable in silico approach for exploring metabolic diversity, which compiles genomic and biochemical data to reconstruct the metabolic network of an organism. Here we use a diverse collection of 507 KpSC isolates, including representatives of globally distributed clinically-relevant lineages, to construct the most comprehensive KpSC pan-metabolic model to-date, KpSC pan v2. Candidate metabolic reactions were identified using gene orthology to known metabolic genes, prior to manual curation via extensive literature and database searches. The final model comprised a total of 3,550 reactions, 2,403 genes and can simulate growth on 360 unique substrates. We used KpSC pan v2 as a reference to derive strain-specific GSMMs for all 507 KpSC isolates, and compared these to GSMMs generated using a prior KpSC pan-reference (KpSC pan v1) and two single-strain references. We show that KpSC pan v2 includes a greater proportion of accessory reactions (8.8%) than KpSC pan v1 (2.5%). GSMMs derived from KpSC pan v2 also result in more accuracy growth predictions than those derived from other references in both aerobic (median accuracy = 95.4%) and anaerobic (median accuracy = 78.8%). KpSC pan v2 also generates more accurate growth predictions, with high median accuracies of 95.4% (aerobic, n=37 isolates) and 78.8% (anaerobic, n=36 isolates) for 124 matched carbon substrates. KpSC pan v2 is freely available at https://github.com/kelwyres/KpSC-pan-metabolic-model, representing a valuable resource for the scientific community, both as a source of curated metabolic information and as a reference to derive accurate strain-specific GSMMs. The latter can be used to investigate the relationship between KpSC metabolism and traits of interest, such as reservoirs, epidemiology, drug resistance or virulence, and ultimately to inform novel KpSC control strategies. Significance as a BioResource to the communityKlebsiella pneumoniae and its close relatives in the K. pneumoniae Species Complex (KpSC) are priority antimicrobial resistant pathogens that exhibit extensive genomic diversity. There is growing interest in understanding KpSC metabolism, and genome scale metabolic models (GSMMs) provide a rapid, scalable option for exploration of whole cell metabolism plus phenotype prediction. Here we present a KpSC pan-metabolic model representing the cellular metabolism of 507 diverse KpSC isolates. Our model is the largest and most comprehensive of its kind, comprising \u003e2,400 genes associated with \u003e3,500 metabolic reactions, plus manually curated evidence annotations. These data alone represent a key knowledge resource for the Klebsiella research community; however, our models greatest impact lies in its potential for use as a reference from which highly accurate strain-specific GSMMs can be derived to inform in depth strain-specific and/or large-scale comparative analyses. Data summaryO_LIKlebsiella pneumoniae species complex (KpSC) pan v2 metabolic model available at https://github.com/kelwyres/KpSC-pan-metabolic-model. C_LIO_LIAll KpSC isolate whole genome sequences used in this work were reported previously and are available under Bioprojects PRJEB6891, PRJNA351909, PRJNA493667, PRJNA768294, PRJNA253462, PRJNA292902 and PRJNA391323. Individual accessions listed in Table S1. C_LIO_LIStrain-specific GSMMs used for comparative analyses (deposited in Figshare - 10.6084/m9.figshare.24871914), plus their associated MEMOTE reports (indicates completeness and annotation quality), reaction and gene presence-absence matrices across all isolates. C_LIO_LIGrowth phenotype predictions derived from strain-specific GSMMs are available in Table S4. C_LIO_LIBinarised Biolog growth phenotype data for n=37 isolates (plates PM1 and PM2, aerobic and anaerobic conditions) are available in Tables S6 \u0026 S7. C_LIO_LIAdditional growth assay data for six substrates not included on Biolog plates PM1 and PM2 (deposited in Figshare - 10.6084/m9.figshare.24871914). C_LI\n",
  "wordCount" : "3301",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573821">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573821" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573821">
        <p class="paperTitle">Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573821" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573821" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.</p>
        <p class="info">Score: 54.0, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573821' target='https://doi.org/10.1101/2024.01.02.573821'> 10.1101/2024.01.02.573821</a></p>
        <p class="abstract">Research and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance. This necessitates a framework to identify all types of variants independent of their size (e.g., SNV/SV) or location (e.g., repeats). Here we present DRAGEN that utilizes novel methods based on multigenomes, hardware acceleration, and machine learning based variant detection to provide novel insights into individual genomes with [~]30min computation time (from raw reads to variant detection). DRAGEN outperforms all other state-of-the-art methods in speed and accuracy across all variant types (SNV, indel, STR, SV, CNV) and further incorporates specialized methods to obtain key insights in medically relevant genes (e.g., HLA, SMN, GBA). We showcase DRAGEN across 3,202 genomes and demonstrate its scalability, accuracy, and innovations to further advance the integration of comprehensive genomics for research and medical applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573133" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573133">
        <p class="paperTitle">RNA polymerases reshape chromatin and coordinate transcription on individual fibers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tullius, T. W.; Isaac, R. S.; Ranchalis, J.; Dubocanin, D.; Churchman, L. S.; Stergachis, A. B.</p>
        <p class="info">Score: 37.5, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573133' target='https://doi.org/10.1101/2023.12.22.573133'> 10.1101/2023.12.22.573133</a></p>
        <p class="abstract">During eukaryotic transcription, RNA polymerases must initiate and pause within a crowded, complex environment, surrounded by nucleosomes and other transcriptional activity. This environment creates a spatial arrangement along individual chromatin fibers ripe for both competition and coordination, yet these relationships remain largely unknown owing to the inherent limitations of traditional structural and sequencing methodologies. To address these limitations, we employed long-read chromatin fiber sequencing (Fiber-seq) to visualize RNA polymerases within their native chromatin context at single-molecule and near single-nucleotide resolution along up to 30 kb fibers. We demonstrate that Fiber-seq enables the identification of single-molecule RNA Polymerase (Pol) II and III transcription associated foot-prints, which, in aggregate, mirror bulk short-read sequencing-based measurements of transcription. We show that Pol II pausing destabilizes downstream nucleosomes, with frequently paused genes maintaining a short-term memory of these destabilized nucleosomes. Furthermore, we demonstrate pervasive direct coordination and anti-coordination between nearby Pol II genes, Pol III genes, transcribed enhancers, and insulator elements. This coordination is largely limited to spatially organized elements within 5 kb of each other, implicating short-range chromatin environments as a predominant determinant of coordinated polymerase initiation. Overall, transcription initiation reshapes surrounding nucleosome architecture and coordinates nearby transcriptional machinery along individual chromatin fibers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.573414">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.573414" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.573414">
        <p class="paperTitle">The majority of somatic mutations in fruit trees are layer-specific</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.573414" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.573414" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goel, M.; Campoy, J. A.; Krause, K.; Baus, L. C.; Sahu, A.; Sun, H.; Walkemeier, B.; Marek, M.; Beaudry, R.; Ruiz, D.; Huettel, B.; Schneeberger, K.</p>
        <p class="info">Score: 33.3, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.573414' target='https://doi.org/10.1101/2024.01.04.573414'> 10.1101/2024.01.04.573414</a></p>
        <p class="abstract">Plant meristems consist of three distinct stem cells layers. The identities of these layers are maintained in developing organs. Somatic mutations in the meristem thus propagate only into the respective layers of the differentiated organs. We analysed the genomes of individual cell layers in an apricot tree and found an unexpectedly high mutation load, where &gt;90% of the somatic mutations were layer-specific. The layers had similar mutation spectra but layer 1 (epidermis) had substantially more mutations than layer 2 (mesocarp). Most somatic mutations were shared between fruits and neighbouring leaves corroborating their meristematic origin. Single-cell transcriptomics revealed their layer-specific effects forming the basis for layer-specific phenotypes. Our work unveils the hidden abundance of layer-specific meristematic mutations and provides insights into their identification and understanding.

One sentence summarySomatic mutations in fruit trees occur in specific cell layers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574404">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574404" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574404">
        <p class="paperTitle">mRNA initiation and termination are spatially coordinated</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574404" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574404" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Calvo-Roitberg, E.; Carroll, C. L.; Venev, S. V.; Kim, G.; Mick, S. T.; Dekker, J.; Fiszbein, A.; Pai, A. A.</p>
        <p class="info">Score: 29.6, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574404' target='https://doi.org/10.1101/2024.01.05.574404'> 10.1101/2024.01.05.574404</a></p>
        <p class="abstract">The expression of a precise mRNA transcriptome is crucial for establishing cell identity and function, with dozens of alternative isoforms produced for a single gene sequence. The regulation of mRNA isoform usage occurs by the coordination of co-transcriptional mRNA processing mechanisms across a gene. Decisions involved in mRNA initiation and termination underlie the largest extent of mRNA isoform diversity, but little is known about any relationships between decisions at both ends of mRNA molecules. Here, we systematically profile the joint usage of mRNA transcription start sites (TSSs) and polyadenylation sites (PASs) across tissues and species. Using both short and long read RNA-seq data, we observe that mRNAs preferentially using upstream TSSs also tend to use upstream PASs, and congruently, the usage of downstream sites is similarly paired. This observation suggests that mRNA 5 end choice may directly influence mRNA 3 ends. Our results suggest a novel &#34;Positional Initiation-Termination Axis&#34; (PITA), in which the usage of alternative terminal sites are coupled based on the order in which they appear in the genome. PITA isoforms are more likely to encode alternative protein domains and use conserved sites. PITA is strongly associated with the length of genomic features, such that PITA is enriched in longer genes with more area devoted to regions that regulate alternative 5 or 3 ends. Strikingly, we found that PITA genes are more likely than non-PITA genes to have multiple, overlapping chromatin structural domains related to pairing of ordinally coupled start and end sites. In turn, PITA coupling is also associated with fast RNA Polymerase II (RNAPII) trafficking across these long gene regions. Our findings indicate that a combination of spatial and kinetic mechanisms couple transcription initiation and mRNA 3 end decisions based on ordinal position to define the expression mRNA isoforms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574078">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574078" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574078">
        <p class="paperTitle">Somatic cancer driver mutations are enriched and associated with inflammatory states in Alzheimer&#39;s disease microglia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574078" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574078" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, A. Y.; Zhou, Z.; Talukdar, M.; Miller, M. B.; Chhouk, B.; Enyenihi, L.; Rosen, I.; Stronge, E.; Zhao, B.; Kim, D.; Choi, J.; Khoshkhoo, S.; Kim, J.; Ganz, J.; Travaglini, K. J.; Gabitto, M.; Hodge, R. D.; Kaplan, E.; Lein, E.; De Jager, P.; Bennett, D. A.; Lee, E. A.; Walsh, C. A.</p>
        <p class="info">Score: 13.1, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574078' target='https://doi.org/10.1101/2024.01.03.574078'> 10.1101/2024.01.03.574078</a></p>
        <p class="abstract">Alzheimers disease (AD) is an age-associated neurodegenerative disorder characterized by progressive neuronal loss and pathological accumulation of the misfolded proteins amyloid-{beta} and tau1,2. Neuroinflammation mediated by microglia and brain-resident macrophages plays a crucial role in AD pathogenesis1-5, though the mechanisms by which age, genes, and other risk factors interact remain largely unknown. Somatic mutations accumulate with age and lead to clonal expansion of many cell types, contributing to cancer and many non-cancer diseases6,7. Here we studied somatic mutation in normal aged and AD brains by three orthogonal methods and in three independent AD cohorts. Analysis of bulk RNA sequencing data from 866 samples from different brain regions revealed significantly higher ([~]two-fold) overall burdens of somatic single-nucleotide variants (sSNVs) in AD brains compared to age-matched controls. Molecular-barcoded deep (&gt;1000X) gene panel sequencing of 311 prefrontal cortex samples showed enrichment of sSNVs and somatic insertions and deletions (sIndels) in cancer driver genes in AD brain compared to control, with recurrent, and often multiple, mutations in genes implicated in clonal hematopoiesis (CH)8,9. Pathogenic sSNVs were enriched in CSF1R&#43; microglia of AD brains, and the high proportion of microglia (up to 40%) carrying some sSNVs in cancer driver genes suggests mutation-driven microglial clonal expansion (MiCE). Analysis of single-nucleus RNA sequencing (snRNAseq) from temporal neocortex of 62 additional AD cases and controls exhibited nominally increased mosaic chromosomal alterations (mCAs) associated with CH10,11. Microglia carrying mCA showed upregulated pro-inflammatory genes, resembling the transcriptomic features of disease-associated microglia (DAM) in AD. Our results suggest that somatic driver mutations in microglia are common with normal aging but further enriched in AD brain, driving MiCE with inflammatory and DAM signatures. Our findings provide the first insights into microglial clonal dynamics in AD and identify potential new approaches to AD diagnosis and therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574048">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574048" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574048">
        <p class="paperTitle">Cell-cycle dependent DNA repair and replication unifies patterns of chromosome instability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574048" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574048" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, B.; Winnall, S.; Cross, W.; Barnes, C. P.</p>
        <p class="info">Score: 12.1, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574048' target='https://doi.org/10.1101/2024.01.03.574048'> 10.1101/2024.01.03.574048</a></p>
        <p class="abstract">Chromosomal instability (CIN) is pervasive in human tumours and often leads to structural or numerical chromosomal aberrations. Somatic structural variants (SVs) are intimately related to copy number alterations but the two types of variant are often studied independently. In addition, despite numerous studies on detecting various SV patterns, there are still no general quantitative models of SV generation. To address this issue, we develop a computational cell-cycle model for the generation of SVs from end-joining repair and replication after double strand break formation. Our model provides quantitative information on the relationship between breakage fusion bridge cycle, chromothripsis, seismic amplification, and extra-chromosomal circular DNA. Given single-cell whole-genome sequencing data, the model also allows us to infer important parameters in SV generation with Bayesian inference. Our quantitative framework unifies disparate genomic patterns resulted from CIN, provides a null mutational model for SV, and reveals new insights into the impact of genome rearrangement on tumour evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574329">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574329" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574329">
        <p class="paperTitle">Convergence and global epidemiology of Klebsiella pneumoniae plasmids harbouring the iuc3 virulence locus.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574329" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574329" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gibbon, M. J.; Couto, N.; Cozens, K.; Habib, S.; Cowley, L.; Aanensen, D.; Corander, J.; Thorpe, H.; Hetland, M. A.; Sassera, D.; Merla, C.; Corbella, M.; Ferrari, C.; Turner, K. M.; Sirikancha, K.; Dulyayangkul, P.; Alhusein, N.; Charoenlap, N.; Thamlikitikul, V.; Avison, M. B.; Feil, E. J.</p>
        <p class="info">Score: 11.2, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574329' target='https://doi.org/10.1101/2024.01.05.574329'> 10.1101/2024.01.05.574329</a></p>
        <p class="abstract">BackgroundKlebsiella pneumoniae (Kp) is an important pathogen of humans and animals, and recent reports of  convergent strains that carry both virulence and antimicrobial resistance genes (ARGs) have raised serious public health concern. The plasmid-borne iuc locus, encoding the siderophore aerobactin, is a key virulence factor in this species. The variant iuc3 is associated with porcine and human clinical isolates and is carried by mostly uncharacterised IncF plasmids.

MethodsWe used a combination of short-read and long-read sequencing to characterise IncFIB(K)/IncFII iuc3-carrying plasmids harboured by 79 Kp isolates and one K. oxytoca isolate recovered as part of two large  One-Health studies in Italy (SpARK) and Thailand (OH-DART). Adding data from public repositories gave a combined dataset of 517 iuc3 isolates, and the plasmids were analysed using both clustering and phylogenetic methods.

FindingsWe note seven large, convergent, plasmids from Thailand that have emerged through the hybridisation of co-circulating plasmids harbouring iuc3 and antimicrobial resistance genes (ARGs) encoding extended-spectrum beta-lactamases (ESBLs). We were also able to identify putative parental plasmids which were mostly associated with two neighbouring meat markets, as were the hybrid plasmids. Clustering and global phylogenetic analyses resolved an iuc3 plasmid sub-group circulating throughout Asia, with occasional examples in Europe and elsewhere. This variant carries multiple ARGs and is commonly harboured by clinical isolates, thus warranting targeted plasmid surveillance.

InterpretationOur study reveals that plasmid hybridisation leading to the convergence of resistance and virulence traits may be very common, even in non-clinical ( One-Health) settings. Population-scale plasmid genomics makes it possible to identify putative parental plasmids, which will help to identify plasmid types that are most likely to hybridise, and what the selective consequences may be for the plasmid and host. A distinct iuc3 plasmid sub-variant is associated with clinical isolates in Asia which requires close monitoring.

Research In ContextMultiple reports of  convergent clones of Klebsiella pneumoniae that combine both hypervirulence and multidrug resistance (MDR-hvKp) have been published recently; a PubMed search in November 2023 using the key words  convergence Klebsiella pneumoniae returned 143 papers, 99 of which were published from 2020 onwards. Our study demonstrates that the hybridisation of plasmids carrying AMR and virulence genes is a frequent, ongoing, process in natural populations. The subsequent transfer of plasmids conferring both traits is thus likely to be a key driver behind the spread of convergent strains. Our study also provides an exemplar of how hybrid assemblies can facilitate large-scale global genomic plasmid epidemiology.

Evidence before the studyAlthough multiple recent reports highlight the emergence and spread of convergent Kp strains, the confluence of resistance and virulence genes within the same plasmid has not been studied at a population level, and putative parental plasmids are rarely identified. Moreover, there have been few high-resolution genomic epidemiology studies on closely related plasmids using both long and short-read data on a global scale.

Added valueWe more than double the number of complete sequences available for plasmids harbouring iuc3 from 58 to 139 and provide evidence on the host lineages most likely to harbour these plasmids (e.g., ST35), and epidemiological source (e.g., pig, wild animal, human). Our comparative analysis of phylogenetic and clustering approaches will help to inform future plasmid epidemiological studies.

ImplicationsThe hybridisation of plasmids harbouring virulence and resistance genes occurs frequently in natural populations, even within  One-Health settings. However, the selective drivers (if any) and evolutionary consequences of this phenomenon are unclear. There is clear utility in generating closed plasmid genomes on a population scale, and targeted plasmid surveillance on a clinical sub-variant of iuc3 plasmids is warranted.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573464">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573464" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573464">
        <p class="paperTitle">Comparative genomics of Cryptococcus and Kwoniella reveals pathogenesis evolution and contrasting modes of karyotype evolution via chromosome fusion or intercentromeric recombination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573464" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573464" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coelho, M. A.; David-Palma, M.; Shea, T.; Bowers, K.; McGinley-Smith, S.; Mohammad, A. W.; Gnirke, A.; Yurkov, A. M.; Nowrousian, M.; Sun, S.; Cuomo, C. A.; Heitman, J.</p>
        <p class="info">Score: 23.0, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573464' target='https://doi.org/10.1101/2023.12.27.573464'> 10.1101/2023.12.27.573464</a></p>
        <p class="abstract">A large-scale comparative genomic analysis was conducted for the global human fungal pathogens within the Cryptococcus genus, compared to non-pathogenic Cryptococcus species, and related species from the sister genus Kwoniella. Chromosome-level genome assemblies were generated for multiple species of both genera, resulting in a dataset encompassing virtually all of their known diversity. Although Cryptococcus and Kwoniella have comparable genome sizes (about 19.2 and 22.9 Mb) and similar gene content, hinting at pre-adaptive pathogenic potential, our analysis found evidence in pathogenic Cryptococcus species of specific examples of gene gain (via horizontal gene transfer) and gene loss, which might represent evolutionary signatures of pathogenic development. Genome analysis also revealed a significant variation in chromosome number and structure between the two genera. By combining synteny analysis and experimental centromere validation, we found that most Cryptococcus species have 14 chromosomes, whereas most Kwoniella species have fewer (11, 8, 5 or even as few as 3). Reduced chromosome number in Kwoniella is associated with formation of giant chromosomes (up to 18 Mb) through repeated chromosome fusion events, each marked by a pericentric inversion and centromere loss. While similar chromosome inversion-fusion patterns were observed in all Kwoniella species with fewer than 14 chromosomes, no such pattern was detected in Cryptococcus. Instead, Cryptococcus species with less than 14 chromosomes, underwent chromosome reductions primarily through rearrangements associated with the loss of repeat-rich centromeres. Additionally, Cryptococcus genomes exhibited frequent interchromosomal translocations, including intercentromeric recombination facilitated by transposons shared between centromeres. Taken together, our findings advance our understanding of genomic changes possibly associated with pathogenicity in Cryptococcus and provide a foundation to elucidate mechanisms of centromere loss and chromosome fusion driving distinct karyotypes in closely related fungal species, including prominent global human pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572873">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572873" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572873">
        <p class="paperTitle">Stable chromosome configuration and loop-based polarization in animal symbionts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572873" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572873" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Viehboeck, T.; Weber, P. M.; Krause, N.; Varoquaux, N.; Boccard, F.; Junier, I.; Bulgheresi, S.; Lioy, V. S.</p>
        <p class="info">Score: 7.6, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572873' target='https://doi.org/10.1101/2023.12.21.572873'> 10.1101/2023.12.21.572873</a></p>
        <p class="abstract">Chromosome partitioning precedes the division of the cytoplasm, and its evolution is linked with the positioning of the division plane. So far, bacterial chromosome biology has heavily focused on transversally dividing, free-living ones. Here, we determined the chromosome organization of three longitudinally dividing Neisseriaceae exclusively inhabiting the oral cavity of mammals. We showed that in all three multicellular bacteria the origin of DNA replication is invariably located at the host-attached (proximal) pole. Next, 3C-seq revealed loop-based folding of the ori region in Alysiella filiformis and Simonsiella muelleri. Moreover, genes involved in cell motility, piliation and signal transduction mechanisms were specifically looped when transcriptionally and translationally active cells adhered to a substrate, but not when cultured in liquid. Overall, we propose that proximal positioning of the ori and loop-based folding of its surrounding DNA may mediate localized translation of proteins involved in host colonization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572682">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572682" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572682">
        <p class="paperTitle">A validated pangenome-scale metabolic model for the Klebsiella pneumoniae species complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572682" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572682" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cooper, H. B.; Vezina, B.; Hawkey, J.; Passet, V.; Lopez-Fernandez, S.; Monk, J. M.; Brisse, S.; Holt, K. E.; Wyres, K. L.</p>
        <p class="info">Score: 8.3, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572682' target='https://doi.org/10.1101/2023.12.20.572682'> 10.1101/2023.12.20.572682</a></p>
        <p class="abstract">The Klebsiella pneumoniae Species Complex (KpSC) is a major source of nosocomial infections globally with high rates of resistance to antimicrobials. Consequently, there is growing interest in understanding virulence factors and their association with cellular metabolic processes for developing novel anti-KpSC therapeutics. Phenotypic assays have revealed metabolic diversity within the KpSC, but metabolism research has been neglected due to experiments being difficult and cost-intensive.

Genome-scale metabolic models (GSMMs) represent a rapid and scalable in silico approach for exploring metabolic diversity, which compiles genomic and biochemical data to reconstruct the metabolic network of an organism. Here we use a diverse collection of 507 KpSC isolates, including representatives of globally distributed clinically-relevant lineages, to construct the most comprehensive KpSC pan-metabolic model to-date, KpSC pan v2. Candidate metabolic reactions were identified using gene orthology to known metabolic genes, prior to manual curation via extensive literature and database searches. The final model comprised a total of 3,550 reactions, 2,403 genes and can simulate growth on 360 unique substrates. We used KpSC pan v2 as a reference to derive strain-specific GSMMs for all 507 KpSC isolates, and compared these to GSMMs generated using a prior KpSC pan-reference (KpSC pan v1) and two single-strain references. We show that KpSC pan v2 includes a greater proportion of accessory reactions (8.8%) than KpSC pan v1 (2.5%). GSMMs derived from KpSC pan v2 also result in more accuracy growth predictions than those derived from other references in both aerobic (median accuracy = 95.4%) and anaerobic (median accuracy = 78.8%). KpSC pan v2 also generates more accurate growth predictions, with high median accuracies of 95.4% (aerobic, n=37 isolates) and 78.8% (anaerobic, n=36 isolates) for 124 matched carbon substrates.

KpSC pan v2 is freely available at https://github.com/kelwyres/KpSC-pan-metabolic-model, representing a valuable resource for the scientific community, both as a source of curated metabolic information and as a reference to derive accurate strain-specific GSMMs. The latter can be used to investigate the relationship between KpSC metabolism and traits of interest, such as reservoirs, epidemiology, drug resistance or virulence, and ultimately to inform novel KpSC control strategies.

Significance as a BioResource to the communityKlebsiella pneumoniae and its close relatives in the K. pneumoniae Species Complex (KpSC) are priority antimicrobial resistant pathogens that exhibit extensive genomic diversity. There is growing interest in understanding KpSC metabolism, and genome scale metabolic models (GSMMs) provide a rapid, scalable option for exploration of whole cell metabolism plus phenotype prediction. Here we present a KpSC pan-metabolic model representing the cellular metabolism of 507 diverse KpSC isolates. Our model is the largest and most comprehensive of its kind, comprising &gt;2,400 genes associated with &gt;3,500 metabolic reactions, plus manually curated evidence annotations. These data alone represent a key knowledge resource for the Klebsiella research community; however, our models greatest impact lies in its potential for use as a reference from which highly accurate strain-specific GSMMs can be derived to inform in depth strain-specific and/or large-scale comparative analyses.

Data summaryO_LIKlebsiella pneumoniae species complex (KpSC) pan v2 metabolic model available at https://github.com/kelwyres/KpSC-pan-metabolic-model.
C_LIO_LIAll KpSC isolate whole genome sequences used in this work were reported previously and are available under Bioprojects PRJEB6891, PRJNA351909, PRJNA493667, PRJNA768294, PRJNA253462, PRJNA292902 and PRJNA391323. Individual accessions listed in Table S1.
C_LIO_LIStrain-specific GSMMs used for comparative analyses (deposited in Figshare - 10.6084/m9.figshare.24871914), plus their associated MEMOTE reports (indicates completeness and annotation quality), reaction and gene presence-absence matrices across all isolates.
C_LIO_LIGrowth phenotype predictions derived from strain-specific GSMMs are available in Table S4.
C_LIO_LIBinarised Biolog growth phenotype data for n=37 isolates (plates PM1 and PM2, aerobic and anaerobic conditions) are available in Tables S6 &amp; S7.
C_LIO_LIAdditional growth assay data for six substrates not included on Biolog plates PM1 and PM2 (deposited in Figshare - 10.6084/m9.figshare.24871914).
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
