<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Rapid and Quantitative Functional Interrogation of Human Enhancer Variant Activity in Live Mice
Authors: Hollingsworth, E. W.; Liu, T. A.; Jacinto, S. H.; Chen, C. X.; Alcantara, J. A.; Kvon, E. Z.
Score: 25.3, Published: 2023-12-10 DOI: 10.1101/2023.12.10.570890
Functional analysis of non-coding variants associated with human congenital disorders remains challenging due to the lack of efficient in vivo models. Here we introduce dual-enSERT, a robust Cas9-based two-color fluorescent reporter system which enables rapid, quantitative comparison of enhancer allele activities in live mice of any genetic background.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Rapid and Quantitative Functional Interrogation of Human Enhancer Variant Activity in Live Mice
Authors: Hollingsworth, E. W.; Liu, T. A.; Jacinto, S. H.; Chen, C. X.; Alcantara, J. A.; Kvon, E. Z.
Score: 25.3, Published: 2023-12-10 DOI: 10.1101/2023.12.10.570890
Functional analysis of non-coding variants associated with human congenital disorders remains challenging due to the lack of efficient in vivo models. Here we introduce dual-enSERT, a robust Cas9-based two-color fluorescent reporter system which enables rapid, quantitative comparison of enhancer allele activities in live mice of any genetic background." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Rapid and Quantitative Functional Interrogation of Human Enhancer Variant Activity in Live Mice
Authors: Hollingsworth, E. W.; Liu, T. A.; Jacinto, S. H.; Chen, C. X.; Alcantara, J. A.; Kvon, E. Z.
Score: 25.3, Published: 2023-12-10 DOI: 10.1101/2023.12.10.570890
Functional analysis of non-coding variants associated with human congenital disorders remains challenging due to the lack of efficient in vivo models. Here we introduce dual-enSERT, a robust Cas9-based two-color fluorescent reporter system which enables rapid, quantitative comparison of enhancer allele activities in live mice of any genetic background."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Rapid and Quantitative Functional Interrogation of Human Enhancer Variant Activity in Live Mice\nAuthors: Hollingsworth, E. W.; Liu, T. A.; Jacinto, S. H.; Chen, C. X.; Alcantara, J. A.; Kvon, E. Z.\nScore: 25.3, Published: 2023-12-10 DOI: 10.1101/2023.12.10.570890\nFunctional analysis of non-coding variants associated with human congenital disorders remains challenging due to the lack of efficient in vivo models. Here we introduce dual-enSERT, a robust Cas9-based two-color fluorescent reporter system which enables rapid, quantitative comparison of enhancer allele activities in live mice of any genetic background.",
  "keywords": [
    
  ],
  "articleBody": " Rapid and Quantitative Functional Interrogation of Human Enhancer Variant Activity in Live Mice\nAuthors: Hollingsworth, E. W.; Liu, T. A.; Jacinto, S. H.; Chen, C. X.; Alcantara, J. A.; Kvon, E. Z.\nScore: 25.3, Published: 2023-12-10 DOI: 10.1101/2023.12.10.570890\nFunctional analysis of non-coding variants associated with human congenital disorders remains challenging due to the lack of efficient in vivo models. Here we introduce dual-enSERT, a robust Cas9-based two-color fluorescent reporter system which enables rapid, quantitative comparison of enhancer allele activities in live mice of any genetic background. We use this new technology to examine and measure the gain- and loss-of-function effects of enhancer variants linked to limb polydactyly, autism, and craniofacial malformation. By combining dual-enSERT with single-cell transcriptomics, we characterize variant enhancer alleles at cellular resolution, thereby implicating candidate molecular pathways in pathogenic enhancer misregulation. We further show that independent, polydactyly-linked enhancer variants lead to ectopic expression in the same cell populations, indicating shared genetic mechanisms underlying non-coding variant pathogenesis. Finally, we streamline dual-enSERT for analysis in F0 animals by placing both reporters on the same transgene separated by a synthetic insulator. Dual-enSERT allows researchers to go from identifying candidate enhancer variants to analysis of comparative enhancer activity in live embryos in under two weeks.\nSystematic comparison of sequencing-based spatial transcriptomic methods\nAuthors: You, Y.; Fu, Y.; Li, L.; Zhang, Z.; Jia, S.; Lu, S.; Ren, W.; Liu, Y.; Xu, Y.; Liu, X.; Jiang, F.; Peng, G.; Kumar, A. S.; Ritchie, M. E.; Liu, X.; Tian, L.\nScore: 30.6, Published: 2023-12-04 DOI: 10.1101/2023.12.03.569744\nRecent advancements of sequencing-based spatial transcriptomics (sST) have catalyzed significant advancements by facilitating transcriptome-scale spatial gene expression measurement. Despite this progress, efforts to comprehensively benchmark different platforms are currently lacking. The extant variability across technologies and datasets poses challenges in formulating standardized evaluation metrics. In this study, we established a collection of reference tissues and regions characterized by well-defined histological architectures, and used them to generate data to compare six sST methods. We highlighted molecular diffusion as a variable parameter across different methods and tissues, significantly impacting the effective resolutions. Furthermore, we observed that spatial transcriptomic data demonstrate unique attributes beyond merely adding a spatial axis to single-cell data, including an enhanced ability to capture patterned rare cell states along with specific markers, albeit being influenced by multiple factors including sequencing depth and resolution. Our study assists biologists in sST platform selection, and helps foster a consensus on evaluation standards and establish a framework for future benchmarking efforts that can be used as a gold standard for the development and benchmarking of computational tools for spatial transcriptomic analysis.\nStrain-specific evolution and host-specific regulation of transposable elements in the model plant symbiont Rhizophagus irregularis\nAuthors: Oliveira, J. I. N.; Corradi, N.\nScore: 20.0, Published: 2023-12-05 DOI: 10.1101/2023.12.01.569630\nTransposable elements (TE) are repetitive DNA that can create variability in genome structure and regulation. The genome of Rhizophagus irregularis, a widely studied arbuscular mycorrhizal fungus (AMF), comprises approximately 50% repetitive sequences that include transposable elements (TE). Despite their abundance, two-thirds of TE remain unclassified, and their regulation among AMF life-stages remains unknown. Here, we aimed to improve our understanding of TE diversity and regulation in this model species by curating repeat datasets obtained from chromosome-level assemblies and by investigating their expression across multiple conditions. Our analyses uncovered new TE superfamilies and families in this model symbiont and revealed significant differences in how these sequences evolve both within and between R. irregularis genomes. With this curated TE annotation, we detected that the number of upregulated TE families in colonized roots is four times higher than extraradical mycelium, and their overall expression differs depending on their host. This work provides a fine-scale view of TE diversity and evolution in model plant symbionts and highlights their transcriptional dynamism and specificity during host-microbe interactions. We also provide Hidden Markov Model profiles of TE domains are now available for future manual curation of uncharacterized sequences (https://github.com/jordana-olive/TE-manual-curation/tree/main).\nDevelopment of a Next Generation SNP Genotyping Array for Wheat\nAuthors: Burridge, A. J.; Winfield, M.; Przewieslik-Allen, A.; Edwards, K. J.; Siddique, I.; Barral-Arca, R.; Griffiths, S.; Cheng, S.; Huang, Z.; Feng, C.; Dreisigacker, S.; Bentley, A.; Brown-Guedira, G.; Barker, G.\nScore: 16.9, Published: 2023-11-27 DOI: 10.1101/2023.11.27.568448\nHigh throughput genotyping arrays have provided a cost effective, reliable and interoperable system for genotyping hexaploid wheat and its related germplasm pool. Existing, highly cited arrays including our 35K Axiom Wheat Breeders genotyping array and the Illumina 90K iSelect array were designed based on a limited amount of varietal sequence diversity and with imperfect knowledge of SNP positions. Recent progress in sequencing wheat varieties and landraces has given us access to a vast pool of SNP diversity, whilst technological improvements in array design has allowed us to fit significantly more probes onto a 384-well format Axiom array than was previously possible. Here we describe a novel High Density Axiom genotyping array, the Triticum aestivum Next Generation array (TaNG), largely derived from whole genome skim sequencing of 204 elite wheat lines and 111 wheat landraces taken from the Watkins \"Core Collection\". We use a novel \"minimal marker\" optimisation approach to select up to six SNPs in each 1.5 MB region of the wheat genome with the highest combined varietal discrimination potential. A design iteration step allowed us to test and replace skim-sequence derived SNPs which failed to convert to reliable Axiom markers, resulting in a final design, designated TaNG1.1 with 43,372 SNPs derived from a haplotype-optimised combination of novel SNPs, DArTAG-derived and legacy wheat Axiom markers. We show that this design has an even distribution of SNPs across chromosomes and sub-genomes compared to previous arrays and can be used to generate genetic maps with a significantly higher number of distinct bins than our previous Axiom array. We also demonstrate the improved performance of TaNG1.1 for Genome Wide Association Studies (GWAS) and its utility for Copy Number Variation (CNV) analysis. The array is commercially available, and the marker annotations, initial genotyping results and software used to generate the optimised marker sets are freely available.\nImproving population scale statistical phasing with whole-genome sequencing data\nAuthors: Wertenbroek, R.; Hofmeister, R.; Xenarios, I.; Thoma, Y.; Delaneau, O.\nScore: 15.3, Published: 2023-12-07 DOI: 10.1101/2023.12.07.570528\nHaplotype estimation, or phasing, has gained significant traction in large-scale projects due to its valuable contributions to population genetics, variant analysis, and the creation of reference panels for imputation and phasing of new samples. To scale with the growing number of samples, haplotype estimation methods designed for population scale rely on highly optimized statistical models to phase genotype data, and usually ignore read-level information. Statistical methods excel in resolving common variants, however, they still struggle at rare variants due to the lack of statistical information. In this study we introduce SAPPHIRE, a new method that leverages whole-genome sequencing data to enhance the precision of haplotype calls produced by statistical phasing. SAPPHIRE achieves this by refining haplotype estimates through the realignment of sequencing reads, particularly targeting low-confidence phase calls. Our findings demonstrate that SAPPHIRE significantly enhances the accuracy of haplotypes obtained from state of the art methods and also provides the subset of phase calls that are validated by sequencing reads. Finally, we show that our method scales to large data sets by its successful application to the extensive 3.6 Petabytes of sequencing data of the last UK Biobank 200,031 sample release.\nMitotic chromosomes harbor cell type and species-specific structural features within a universal looping architecture\nAuthors: Oomen, M. E.; Fox, A. N.; Gonzalez, I.; Molliex, A.; Papadopoulou, T.; Navarro, P.; Dekker, J.\nScore: 12.7, Published: 2023-12-09 DOI: 10.1101/2023.12.08.570796\nThe architecture of mammalian mitotic chromosomes is considered to be universal across species and cell types. However, some studies suggest that features of mitotic chromosomes might be cell type or species specific. We previously reported that CTCF binding in human differentiated cell lines is lost in mitosis, whereas mouse embryonic stem cells (mESC) display prominent binding at a subset of CTCF sites in mitosis. Here, we perform parallel footprint ATAC-seq data analyses of mESCs and somatic mouse and human cells to further explore these differences. We then investigate roles of mitotically bound (bookmarked) CTCF in prometaphase chromosome organization by Hi-C. We do not find any remaining interphase structures such as TADs or CTCF loops at mitotically bookmarked CTCF sites in mESCs. This suggests that mitotic loop extruders condensin I and II are not blocked by bound CTCF, and thus that any remaining CTCF binding does not alter mitotic chromosome folding. Lastly, we compare mitotic Hi-C data generated in this study in mouse with publicly available data from human and chicken cell lines. We do not find any cell type specific differences; however, we do find a difference between species. The average genomic size of mitotic loops is much smaller in chicken (200-350 kb), compared to human (500-750 kb) and mouse (1-2 mb). Interestingly, we find that this difference in loop size is correlated with the average genomic length of the q-arm in these species, a finding we confirm by microscopy measurements of chromosome compaction. This suggests that the dimensions of mitotic chromosomes can be modulated through control of sizes of loops generated by condensins to facilitate species-appropriate shortening of chromosome arms.\nCharacterization of cortical neurodevelopment in vitro using gene expression and morphology profiles from single cells\nAuthors: Sundaresh, A.; Meistermann, D.; Lampela, R.; Yang, Z.; Woldegebriel, R.; Ganna, A.; Costa, P. P.; Kilpinen, H.\nScore: 12.9, Published: 2023-12-05 DOI: 10.1101/2023.12.03.569583\nDifferentiation of induced pluripotent stem cells (iPSC) towards different neuronal lineages has enabled diverse cellular models of human neurodevelopment and related disorders. However, in vitro differentiation is a variable process that frequently leads to heterogeneous cell populations that may confound disease-relevant phenotypes. To characterize the baseline and diversity of cortical neurodevelopment in vitro, we differentiated iPSC lines from multiple healthy donors to cortical neurons and profiled the transcriptomes of 60,000 single cells across three timepoints spanning 70 days. We compared the cell types observed in vitro to those seen in vivo and in organoid cultures to assess how well iPSC-derived cells recapitulate neurodevelopment in vivo. We found that over 60% of the cells resembled those seen in the fetal brain with high confidence, while 28% represented metabolically abnormal cell states and broader neuronal classes observed in organoids. Further, we used high-content imaging to quantify morphological phenotypes of the differentiating neurons across the same time points using Cell Painting. By modeling the relationship between image-based features and gene expression, we compared cell type- and donor-specific effects across the two modalities at single cell resolution. We found that while morphological features capture broader neuronal classes than scRNA-seq, they enhance our ability to quantify the biological processes that drive neuronal differentiation over time, such as mitochondrial function and cell cycle. Finally, we show that iPSC-derived cortical neurons are a relevant model for a range of brain-related complex traits. Taken together, we provide a comprehensive molecular atlas of human cortical neuron development in vitro that introduces a relevant framework for disease modeling.\nFunctional identification of cis-regulatory long noncoding RNAs at controlled false-discovery rates\nAuthors: Dhaka, B.; Zimmerli, M.; Hanhart, D.; Moser, M.; Guillen-Ramirez, H. A.; Mishra, S.; Esposito, R.; Polidori, T.; Widmer, M.; Garcia-Perez, R.; Kruithof-de Julio, M.; Pervouchine, D. D.; Mele, M.; Chouvardas, P.; Johnson, R.\nScore: 19.1, Published: 2023-12-09 DOI: 10.1101/2022.09.18.508380\nA key attribute of some long noncoding RNAs (lncRNAs) is their ability to regulate expression of neighbouring genes in cis. However, such cis-lncRNAs are presently defined using ad hoc criteria that, we show, are prone to false-positive predictions. The resulting lack of cis-lncRNA catalogues hinders our understanding of their extent, characteristics and mechanisms. Here, we introduce TransCistor, a framework for defining and identifying cis-lncRNAs based on enrichment of targets amongst proximal genes. TransCistors simple and conservative statistical models are compatible with functionally-defined target gene maps generated by existing and future technologies. Using transcriptome-wide perturbation experiments for 268 human and 134 mouse lncRNAs, we provide the first large-scale survey of cis-lncRNAs. Known cis-lncRNAs are correctly identified, including XIST, LINC00240 and UMLILO, and predictions are consistent across analysis methods, perturbation types and independent experiments. Our results indicate that cis-activity is detected in a minority of lncRNAs, primarily involving activators over repressors. Cis-lncRNAs are detected by both RNA interference and antisense oligonucleotide perturbations. Mechanistically, cis-lncRNA transcripts are observed to physically associate with their target-genes, and are weakly enriched with enhancer-elements. In summary, TransCistor establishes a quantitative foundation for cis-lncRNAs, opening a path to elucidating their molecular mechanisms and biological significance.\nMutational signature decomposition with deep neural networks reveals origins of clock-like processes and hypoxia dependencies\nAuthors: Serrano Colome, C.; Canal Anton, O.; Seplyarskiy, V.; Weghorn, D.\nScore: 10.7, Published: 2023-12-08 DOI: 10.1101/2023.12.06.570467\nDNA mutational processes generate patterns of somatic and germline mutations. A multitude of such mutational processes has been identified and linked to biochemical mechanisms of DNA damage and repair. Cancer genomics relies on these so-called mutational signatures to classify tumours into subtypes, navigate treatment, determine exposure to mutagens, and characterise the origin of individual mutations. Yet, state-of-the-art methods to quantify the contributions of different mutational signatures to a tumour sample frequently fail to detect certain mutational signatures, work well only for a relatively high number of mutations, and do not provide comprehensive error estimates of signature contributions. Here, we present a novel approach to signature decomposition using artificial neural networks that addresses these problems. We show that our approach, SigNet, outperforms existing methods by learning the prior frequencies of signatures and their correlations present in real data. Unlike any other method we tested, SigNet achieves high prediction accuracy even with few mutations. We used this to generate estimates of signature weights for more than 7500 tumours for which only whole-exome sequencing data are available. We then identified systematic differences in signature activity both as a function of epigenetic covariates and over the course of tumour evolution. This allowed us to decipher the origins of signatures SBS3, SBS5 and SBS40. We further discovered novel associations of mutational signatures with hypoxia, including strong positive correlations with the activities of clock-like and defective DNA repair mutational processes. These results provide new insights into the interplay between tumour biology and mutational processes and demonstrate the utility of our novel approach to mutational signature decomposition, a crucial part of cancer genomics studies.\nCryptic endogenous retrovirus subfamilies in the primate lineage\nAuthors: Chen, X.; Zhang, Z.; Yan, Y.; Goubert, C.; Bourque, G.; Inoue, F.\nScore: 10.1, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570592\nMany endogenous retroviruses (ERVs) in the human genome are primate-specific and have contributed novel cis-regulatory elements and transcripts. However, current approaches for classifying and annotating ERVs and their long terminal repeats (LTRs) have limited resolution and are inaccurate. Here, we developed a new annotation based on phylogenetic analysis and cross-species conservation. Focusing on the evolutionary young MER11A/B/C subfamilies, we revealed the presence of 4 phyletic groups, that better explained the epigenetic heterogeneity observed within these subfamilies, suggesting a new annotation for 412 (19.8%) of the MER11 instances. Furthermore, we functionally validated the regulatory potential of these four phyletic groups using a massively parallel reporter assay (MPRA), which also identified motifs associated with their differential activities. Combining MPRA with phyletic groups across primates revealed an apes-specific gain of SOX related motifs through a single-nucleotide deletion. Lastly, by applying our approach across 53 primate-specific LTR subfamilies, we determined the presence of 75 phyletic groups and found that 3,807 (30.0%) instances from 26 LTR subfamilies could be categorized into a novel phyletic group, many of which with a distinct epigenetic profile. Thus, with our refined annotation of primate-specific LTRs, it will be possible to better understand the evolution in primate genomes and potentially identify new roles for ERV/LTRs in their hosts.\n",
  "wordCount" : "2603",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.10.570890">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.10.570890" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.10.570890">
        <p class="paperTitle">Rapid and Quantitative Functional Interrogation of Human Enhancer Variant Activity in Live Mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.10.570890" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.10.570890" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hollingsworth, E. W.; Liu, T. A.; Jacinto, S. H.; Chen, C. X.; Alcantara, J. A.; Kvon, E. Z.</p>
        <p class="info">Score: 25.3, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.10.570890' target='https://doi.org/10.1101/2023.12.10.570890'> 10.1101/2023.12.10.570890</a></p>
        <p class="abstract">Functional analysis of non-coding variants associated with human congenital disorders remains challenging due to the lack of efficient in vivo models. Here we introduce dual-enSERT, a robust Cas9-based two-color fluorescent reporter system which enables rapid, quantitative comparison of enhancer allele activities in live mice of any genetic background. We use this new technology to examine and measure the gain- and loss-of-function effects of enhancer variants linked to limb polydactyly, autism, and craniofacial malformation. By combining dual-enSERT with single-cell transcriptomics, we characterize variant enhancer alleles at cellular resolution, thereby implicating candidate molecular pathways in pathogenic enhancer misregulation. We further show that independent, polydactyly-linked enhancer variants lead to ectopic expression in the same cell populations, indicating shared genetic mechanisms underlying non-coding variant pathogenesis. Finally, we streamline dual-enSERT for analysis in F0 animals by placing both reporters on the same transgene separated by a synthetic insulator. Dual-enSERT allows researchers to go from identifying candidate enhancer variants to analysis of comparative enhancer activity in live embryos in under two weeks.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569744">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569744" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569744">
        <p class="paperTitle">Systematic comparison of sequencing-based spatial transcriptomic methods</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569744" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569744" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: You, Y.; Fu, Y.; Li, L.; Zhang, Z.; Jia, S.; Lu, S.; Ren, W.; Liu, Y.; Xu, Y.; Liu, X.; Jiang, F.; Peng, G.; Kumar, A. S.; Ritchie, M. E.; Liu, X.; Tian, L.</p>
        <p class="info">Score: 30.6, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569744' target='https://doi.org/10.1101/2023.12.03.569744'> 10.1101/2023.12.03.569744</a></p>
        <p class="abstract">Recent advancements of sequencing-based spatial transcriptomics (sST) have catalyzed significant advancements by facilitating transcriptome-scale spatial gene expression measurement. Despite this progress, efforts to comprehensively benchmark different platforms are currently lacking. The extant variability across technologies and datasets poses challenges in formulating standardized evaluation metrics. In this study, we established a collection of reference tissues and regions characterized by well-defined histological architectures, and used them to generate data to compare six sST methods. We highlighted molecular diffusion as a variable parameter across different methods and tissues, significantly impacting the effective resolutions. Furthermore, we observed that spatial transcriptomic data demonstrate unique attributes beyond merely adding a spatial axis to single-cell data, including an enhanced ability to capture patterned rare cell states along with specific markers, albeit being influenced by multiple factors including sequencing depth and resolution. Our study assists biologists in sST platform selection, and helps foster a consensus on evaluation standards and establish a framework for future benchmarking efforts that can be used as a gold standard for the development and benchmarking of computational tools for spatial transcriptomic analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569630">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569630" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569630">
        <p class="paperTitle">Strain-specific evolution and host-specific regulation of transposable elements in the model plant symbiont Rhizophagus irregularis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569630" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569630" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oliveira, J. I. N.; Corradi, N.</p>
        <p class="info">Score: 20.0, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569630' target='https://doi.org/10.1101/2023.12.01.569630'> 10.1101/2023.12.01.569630</a></p>
        <p class="abstract">Transposable elements (TE) are repetitive DNA that can create variability in genome structure and regulation. The genome of Rhizophagus irregularis, a widely studied arbuscular mycorrhizal fungus (AMF), comprises approximately 50% repetitive sequences that include transposable elements (TE). Despite their abundance, two-thirds of TE remain unclassified, and their regulation among AMF life-stages remains unknown. Here, we aimed to improve our understanding of TE diversity and regulation in this model species by curating repeat datasets obtained from chromosome-level assemblies and by investigating their expression across multiple conditions. Our analyses uncovered new TE superfamilies and families in this model symbiont and revealed significant differences in how these sequences evolve both within and between R. irregularis genomes. With this curated TE annotation, we detected that the number of upregulated TE families in colonized roots is four times higher than extraradical mycelium, and their overall expression differs depending on their host. This work provides a fine-scale view of TE diversity and evolution in model plant symbionts and highlights their transcriptional dynamism and specificity during host-microbe interactions. We also provide Hidden Markov Model profiles of TE domains are now available for future manual curation of uncharacterized sequences (https://github.com/jordana-olive/TE-manual-curation/tree/main).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.568448">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.568448" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.568448">
        <p class="paperTitle">Development of a Next Generation SNP Genotyping Array for Wheat</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.568448" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.568448" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Burridge, A. J.; Winfield, M.; Przewieslik-Allen, A.; Edwards, K. J.; Siddique, I.; Barral-Arca, R.; Griffiths, S.; Cheng, S.; Huang, Z.; Feng, C.; Dreisigacker, S.; Bentley, A.; Brown-Guedira, G.; Barker, G.</p>
        <p class="info">Score: 16.9, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.568448' target='https://doi.org/10.1101/2023.11.27.568448'> 10.1101/2023.11.27.568448</a></p>
        <p class="abstract">High throughput genotyping arrays have provided a cost effective, reliable and interoperable system for genotyping hexaploid wheat and its related germplasm pool. Existing, highly cited arrays including our 35K Axiom Wheat Breeders genotyping array and the Illumina 90K iSelect array were designed based on a limited amount of varietal sequence diversity and with imperfect knowledge of SNP positions. Recent progress in sequencing wheat varieties and landraces has given us access to a vast pool of SNP diversity, whilst technological improvements in array design has allowed us to fit significantly more probes onto a 384-well format Axiom array than was previously possible. Here we describe a novel High Density Axiom genotyping array, the Triticum aestivum Next Generation array (TaNG), largely derived from whole genome skim sequencing of 204 elite wheat lines and 111 wheat landraces taken from the Watkins &#34;Core Collection&#34;. We use a novel &#34;minimal marker&#34; optimisation approach to select up to six SNPs in each 1.5 MB region of the wheat genome with the highest combined varietal discrimination potential. A design iteration step allowed us to test and replace skim-sequence derived SNPs which failed to convert to reliable Axiom markers, resulting in a final design, designated TaNG1.1 with 43,372 SNPs derived from a haplotype-optimised combination of novel SNPs, DArTAG-derived and legacy wheat Axiom markers. We show that this design has an even distribution of SNPs across chromosomes and sub-genomes compared to previous arrays and can be used to generate genetic maps with a significantly higher number of distinct bins than our previous Axiom array. We also demonstrate the improved performance of TaNG1.1 for Genome Wide Association Studies (GWAS) and its utility for Copy Number Variation (CNV) analysis. The array is commercially available, and the marker annotations, initial genotyping results and software used to generate the optimised marker sets are freely available.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570528">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570528" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570528">
        <p class="paperTitle">Improving population scale statistical phasing with whole-genome sequencing data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570528" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570528" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wertenbroek, R.; Hofmeister, R.; Xenarios, I.; Thoma, Y.; Delaneau, O.</p>
        <p class="info">Score: 15.3, Published: 2023-12-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570528' target='https://doi.org/10.1101/2023.12.07.570528'> 10.1101/2023.12.07.570528</a></p>
        <p class="abstract">Haplotype estimation, or phasing, has gained significant traction in large-scale projects due to its valuable contributions to population genetics, variant analysis, and the creation of reference panels for imputation and phasing of new samples. To scale with the growing number of samples, haplotype estimation methods designed for population scale rely on highly optimized statistical models to phase genotype data, and usually ignore read-level information. Statistical methods excel in resolving common variants, however, they still struggle at rare variants due to the lack of statistical information. In this study we introduce SAPPHIRE, a new method that leverages whole-genome sequencing data to enhance the precision of haplotype calls produced by statistical phasing. SAPPHIRE achieves this by refining haplotype estimates through the realignment of sequencing reads, particularly targeting low-confidence phase calls. Our findings demonstrate that SAPPHIRE significantly enhances the accuracy of haplotypes obtained from state of the art methods and also provides the subset of phase calls that are validated by sequencing reads. Finally, we show that our method scales to large data sets by its successful application to the extensive 3.6 Petabytes of sequencing data of the last UK Biobank 200,031 sample release.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570796">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570796" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570796">
        <p class="paperTitle">Mitotic chromosomes harbor cell type and species-specific structural features within a universal looping architecture</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570796" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570796" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oomen, M. E.; Fox, A. N.; Gonzalez, I.; Molliex, A.; Papadopoulou, T.; Navarro, P.; Dekker, J.</p>
        <p class="info">Score: 12.7, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570796' target='https://doi.org/10.1101/2023.12.08.570796'> 10.1101/2023.12.08.570796</a></p>
        <p class="abstract">The architecture of mammalian mitotic chromosomes is considered to be universal across species and cell types. However, some studies suggest that features of mitotic chromosomes might be cell type or species specific. We previously reported that CTCF binding in human differentiated cell lines is lost in mitosis, whereas mouse embryonic stem cells (mESC) display prominent binding at a subset of CTCF sites in mitosis. Here, we perform parallel footprint ATAC-seq data analyses of mESCs and somatic mouse and human cells to further explore these differences. We then investigate roles of mitotically bound (bookmarked) CTCF in prometaphase chromosome organization by Hi-C. We do not find any remaining interphase structures such as TADs or CTCF loops at mitotically bookmarked CTCF sites in mESCs. This suggests that mitotic loop extruders condensin I and II are not blocked by bound CTCF, and thus that any remaining CTCF binding does not alter mitotic chromosome folding. Lastly, we compare mitotic Hi-C data generated in this study in mouse with publicly available data from human and chicken cell lines. We do not find any cell type specific differences; however, we do find a difference between species. The average genomic size of mitotic loops is much smaller in chicken (200-350 kb), compared to human (500-750 kb) and mouse (1-2 mb). Interestingly, we find that this difference in loop size is correlated with the average genomic length of the q-arm in these species, a finding we confirm by microscopy measurements of chromosome compaction. This suggests that the dimensions of mitotic chromosomes can be modulated through control of sizes of loops generated by condensins to facilitate species-appropriate shortening of chromosome arms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569583">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569583" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569583">
        <p class="paperTitle">Characterization of cortical neurodevelopment in vitro using gene expression and morphology profiles from single cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569583" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569583" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sundaresh, A.; Meistermann, D.; Lampela, R.; Yang, Z.; Woldegebriel, R.; Ganna, A.; Costa, P. P.; Kilpinen, H.</p>
        <p class="info">Score: 12.9, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569583' target='https://doi.org/10.1101/2023.12.03.569583'> 10.1101/2023.12.03.569583</a></p>
        <p class="abstract">Differentiation of induced pluripotent stem cells (iPSC) towards different neuronal lineages has enabled diverse cellular models of human neurodevelopment and related disorders. However, in vitro differentiation is a variable process that frequently leads to heterogeneous cell populations that may confound disease-relevant phenotypes. To characterize the baseline and diversity of cortical neurodevelopment in vitro, we differentiated iPSC lines from multiple healthy donors to cortical neurons and profiled the transcriptomes of 60,000 single cells across three timepoints spanning 70 days. We compared the cell types observed in vitro to those seen in vivo and in organoid cultures to assess how well iPSC-derived cells recapitulate neurodevelopment in vivo. We found that over 60% of the cells resembled those seen in the fetal brain with high confidence, while 28% represented metabolically abnormal cell states and broader neuronal classes observed in organoids. Further, we used high-content imaging to quantify morphological phenotypes of the differentiating neurons across the same time points using Cell Painting. By modeling the relationship between image-based features and gene expression, we compared cell type- and donor-specific effects across the two modalities at single cell resolution. We found that while morphological features capture broader neuronal classes than scRNA-seq, they enhance our ability to quantify the biological processes that drive neuronal differentiation over time, such as mitochondrial function and cell cycle. Finally, we show that iPSC-derived cortical neurons are a relevant model for a range of brain-related complex traits. Taken together, we provide a comprehensive molecular atlas of human cortical neuron development in vitro that introduces a relevant framework for disease modeling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.09.18.508380">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.09.18.508380" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.09.18.508380">
        <p class="paperTitle">Functional identification of cis-regulatory long noncoding RNAs at controlled false-discovery rates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.09.18.508380" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.09.18.508380" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dhaka, B.; Zimmerli, M.; Hanhart, D.; Moser, M.; Guillen-Ramirez, H. A.; Mishra, S.; Esposito, R.; Polidori, T.; Widmer, M.; Garcia-Perez, R.; Kruithof-de Julio, M.; Pervouchine, D. D.; Mele, M.; Chouvardas, P.; Johnson, R.</p>
        <p class="info">Score: 19.1, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.09.18.508380' target='https://doi.org/10.1101/2022.09.18.508380'> 10.1101/2022.09.18.508380</a></p>
        <p class="abstract">A key attribute of some long noncoding RNAs (lncRNAs) is their ability to regulate expression of neighbouring genes in cis. However, such  cis-lncRNAs are presently defined using ad hoc criteria that, we show, are prone to false-positive predictions. The resulting lack of cis-lncRNA catalogues hinders our understanding of their extent, characteristics and mechanisms. Here, we introduce TransCistor, a framework for defining and identifying cis-lncRNAs based on enrichment of targets amongst proximal genes. TransCistors simple and conservative statistical models are compatible with functionally-defined target gene maps generated by existing and future technologies. Using transcriptome-wide perturbation experiments for 268 human and 134 mouse lncRNAs, we provide the first large-scale survey of cis-lncRNAs. Known cis-lncRNAs are correctly identified, including XIST, LINC00240 and UMLILO, and predictions are consistent across analysis methods, perturbation types and independent experiments. Our results indicate that cis-activity is detected in a minority of lncRNAs, primarily involving activators over repressors. Cis-lncRNAs are detected by both RNA interference and antisense oligonucleotide perturbations. Mechanistically, cis-lncRNA transcripts are observed to physically associate with their target-genes, and are weakly enriched with enhancer-elements. In summary, TransCistor establishes a quantitative foundation for cis-lncRNAs, opening a path to elucidating their molecular mechanisms and biological significance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570467">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570467" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570467">
        <p class="paperTitle">Mutational signature decomposition with deep neural networks reveals origins of clock-like processes and hypoxia dependencies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570467" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570467" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Serrano Colome, C.; Canal Anton, O.; Seplyarskiy, V.; Weghorn, D.</p>
        <p class="info">Score: 10.7, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570467' target='https://doi.org/10.1101/2023.12.06.570467'> 10.1101/2023.12.06.570467</a></p>
        <p class="abstract">DNA mutational processes generate patterns of somatic and germline mutations. A multitude of such mutational processes has been identified and linked to biochemical mechanisms of DNA damage and repair. Cancer genomics relies on these so-called mutational signatures to classify tumours into subtypes, navigate treatment, determine exposure to mutagens, and characterise the origin of individual mutations. Yet, state-of-the-art methods to quantify the contributions of different mutational signatures to a tumour sample frequently fail to detect certain mutational signatures, work well only for a relatively high number of mutations, and do not provide comprehensive error estimates of signature contributions. Here, we present a novel approach to signature decomposition using artificial neural networks that addresses these problems. We show that our approach, SigNet, outperforms existing methods by learning the prior frequencies of signatures and their correlations present in real data. Unlike any other method we tested, SigNet achieves high prediction accuracy even with few mutations. We used this to generate estimates of signature weights for more than 7500 tumours for which only whole-exome sequencing data are available. We then identified systematic differences in signature activity both as a function of epigenetic covariates and over the course of tumour evolution. This allowed us to decipher the origins of signatures SBS3, SBS5 and SBS40. We further discovered novel associations of mutational signatures with hypoxia, including strong positive correlations with the activities of clock-like and defective DNA repair mutational processes. These results provide new insights into the interplay between tumour biology and mutational processes and demonstrate the utility of our novel approach to mutational signature decomposition, a crucial part of cancer genomics studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570592">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570592" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570592">
        <p class="paperTitle">Cryptic endogenous retrovirus subfamilies in the primate lineage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570592" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570592" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chen, X.; Zhang, Z.; Yan, Y.; Goubert, C.; Bourque, G.; Inoue, F.</p>
        <p class="info">Score: 10.1, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570592' target='https://doi.org/10.1101/2023.12.07.570592'> 10.1101/2023.12.07.570592</a></p>
        <p class="abstract">Many endogenous retroviruses (ERVs) in the human genome are primate-specific and have contributed novel cis-regulatory elements and transcripts. However, current approaches for classifying and annotating ERVs and their long terminal repeats (LTRs) have limited resolution and are inaccurate. Here, we developed a new annotation based on phylogenetic analysis and cross-species conservation. Focusing on the evolutionary young MER11A/B/C subfamilies, we revealed the presence of 4  phyletic groups, that better explained the epigenetic heterogeneity observed within these subfamilies, suggesting a new annotation for 412 (19.8%) of the MER11 instances. Furthermore, we functionally validated the regulatory potential of these four phyletic groups using a massively parallel reporter assay (MPRA), which also identified motifs associated with their differential activities. Combining MPRA with phyletic groups across primates revealed an apes-specific gain of SOX related motifs through a single-nucleotide deletion. Lastly, by applying our approach across 53 primate-specific LTR subfamilies, we determined the presence of 75 phyletic groups and found that 3,807 (30.0%) instances from 26 LTR subfamilies could be categorized into a novel phyletic group, many of which with a distinct epigenetic profile. Thus, with our refined annotation of primate-specific LTRs, it will be possible to better understand the evolution in primate genomes and potentially identify new roles for ERV/LTRs in their hosts.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
