<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep indel mutagenesis reveals the impact of insertions and deletions on protein stability and function
Authors: Topolska, M.; Beltran, T.; Lehner, B.
Score: 32.8, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561180
Amino acid insertions and deletions (indels) are an abundant class of genetic variants. However, compared to substitutions, the effects of indels are not well understood and poorly predicted. Here we address this shortcoming by performing deep indel mutagenesis (DIM) of structurally diverse proteins.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Deep indel mutagenesis reveals the impact of insertions and deletions on protein stability and function
Authors: Topolska, M.; Beltran, T.; Lehner, B.
Score: 32.8, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561180
Amino acid insertions and deletions (indels) are an abundant class of genetic variants. However, compared to substitutions, the effects of indels are not well understood and poorly predicted. Here we address this shortcoming by performing deep indel mutagenesis (DIM) of structurally diverse proteins." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-15T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-10-15T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Deep indel mutagenesis reveals the impact of insertions and deletions on protein stability and function
Authors: Topolska, M.; Beltran, T.; Lehner, B.
Score: 32.8, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561180
Amino acid insertions and deletions (indels) are an abundant class of genetic variants. However, compared to substitutions, the effects of indels are not well understood and poorly predicted. Here we address this shortcoming by performing deep indel mutagenesis (DIM) of structurally diverse proteins."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Deep indel mutagenesis reveals the impact of insertions and deletions on protein stability and function\nAuthors: Topolska, M.; Beltran, T.; Lehner, B.\nScore: 32.8, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561180\nAmino acid insertions and deletions (indels) are an abundant class of genetic variants. However, compared to substitutions, the effects of indels are not well understood and poorly predicted. Here we address this shortcoming by performing deep indel mutagenesis (DIM) of structurally diverse proteins.",
  "keywords": [
    
  ],
  "articleBody": " Deep indel mutagenesis reveals the impact of insertions and deletions on protein stability and function\nAuthors: Topolska, M.; Beltran, T.; Lehner, B.\nScore: 32.8, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561180\nAmino acid insertions and deletions (indels) are an abundant class of genetic variants. However, compared to substitutions, the effects of indels are not well understood and poorly predicted. Here we address this shortcoming by performing deep indel mutagenesis (DIM) of structurally diverse proteins. Indel tolerance is strikingly different to substitution tolerance and varies extensively both between different proteins and within different regions of the same protein. Although state of the art variant effect predictors perform poorly on indels, we show that both experimentally-measured and computationally-predicted substitution scores can be repurposed as good indel variant effect predictors by incorporating information on protein secondary structures. Quantifying the effects of indels on protein-protein interactions reveals that insertions can be an important class of gain-of-function variants. Our results provide an overview of the impact of indels on proteins and a method to predict their effects genome-wide.\nThe single-molecule accessibility landscape of newly replicated mammalian chromatin\nAuthors: Ostrowski, M. S.; Yang, M. G.; McNally, C. P.; Abdulhay, N. J.; Wang, S.; Nora, E. P.; Goodarzi, H.; Ramani, V.\nScore: 16.0, Published: 2023-10-11 DOI: 10.1101/2023.10.09.561582\nThe higher-order structure of newly replicated (i.e. nascent) chromatin fibers remains poorly-resolved, limiting our understanding of how epigenomes are maintained across cell divisions. To address this, we present Replication-Aware Single-molecule Accessibility Mapping (RASAM), a long-read sequencing method that nondestructively measures genome-wide replication-status and protein-DNA interactions simultaneously on intact chromatin templates. We report that individual human and mouse nascent chromatin fibers are hyperaccessible compared to steady-state chromatin. This hyperaccessibility occurs at two, coupled length-scales: first, individual nucleosome core particles on nascent DNA exist as a mixture of partially-unwrapped nucleosomes and other subnucleosomal species; second, newly-replicated chromatin fibers are significantly enriched for irregularly-spaced nucleosomes on individual DNA molecules. Focusing on specific cis-regulatory elements (e.g. transcription factor binding sites; active transcription start sites [TSSs]), we discover unique modes by which nascent chromatin hyperaccessibility is resolved at the single-molecule level: at CCCTC-binding factor (CTCF) binding sites, CTCF and nascent nucleosomes compete for motifs on nascent chromatin fibers, resulting in quantitatively-reduced CTCF occupancy and motif accessibility post-replication; at active TSSs, high levels of steady-state chromatin accessibility are preserved, implying that nucleosome free regions (NFRs) are rapidly re-established behind the fork. Our study introduces a new paradigm for studying higher-order chromatin fiber organization behind the replication fork. More broadly, we uncover a unique organization of newly replicated chromatin that must be reset by active processes, providing a substrate for epigenetic reprogramming.\nDevelopmental and housekeeping transcriptional programs display distinct modes of enhancer-enhancer cooperativity in Drosophila\nAuthors: LOUBIERE, V.; de Almeida, B. P.; Pagani, M.; Stark, A.\nScore: 15.4, Published: 2023-10-12 DOI: 10.1101/2023.10.10.561770\nGenomic enhancers are key transcriptional regulators which, upon the binding of sequence-specific transcription factors, activate their cognate target promoters. Although enhancers have been extensively studied in isolation, a substantial number of genes have more than one simultaneously active enhancer, and it remains unclear how these cooperate to regulate transcription. Using Drosophila melanogaster S2 cells as a model, we assay the activities of more than a thousand individual enhancers and a million enhancer pairs towards housekeeping and developmental core promoters with STARR-seq. We report that housekeeping and developmental enhancers show distinct modes of enhancer-enhancer cooperativity: while housekeeping enhancers are additive such that their combined activity mirrors the sum of their individual activities, developmental enhancers are synergistic and follow a multiplicative model of cooperativity. This developmental enhancer synergy is promiscuous and neither depends on the enhancers' endogenous genomic contexts nor on specific transcription factor motif signatures, but it saturates for the highest levels of enhancer activity. These results have important implications for our understanding of gene-regulation in complex multi-enhancer loci and genomically clustered housekeeping genes, providing a rationale for strong and mild transcriptional effects of mutations within enhancer regions.\nInsertion sequence driven evolution of the microbiota revealed by a novel metagenomics pipeline\nAuthors: Kirsch, J. M.; Hryckowian, A. J.; Duerkop, B. A.\nScore: 15.4, Published: 2023-10-09 DOI: 10.1101/2023.10.06.561241\nInsertion sequence (IS) elements are mobile genetic elements in bacterial genomes that support adaptation. We developed a database of IS elements coupled to a computational pipeline that identifies IS element insertions in the microbiota. We discovered that diverse IS elements insert into the genomes of intestinal bacteria regardless of human host lifestyle. These insertions targeted bacterial accessory genes that aid in their adaptation to unique environmental conditions. Using IS element expansion in Bacteroides, we show that IS element activity leads to insertion \"hot spots\" in accessory genes. We show that IS element insertions are stable and can be transferred between humans. Extreme environmental perturbations force IS element insertions to fall out of the microbiota and many fail to rebound following homeostasis. Our work shows that IS elements drive bacterial genome diversification within the microbiota and establishes a framework for understanding how strain level variation within the microbiota impacts human health.\nThe European Reference Genome Atlas: piloting a decentralised approach to equitable biodiversity genomics\nAuthors: Mc Cartney, A. M.; Formenti, G.; Mouton, A.; De Panis, D.; Marins, L. S.; Leitao, H. G.; Diedericks, G.; Kirangwa, J.; Morselli, M.; Salces, J.; Escudero, N.; Iannucci, A.; Natali, C.; Svardal, H.; Fernandez, R.; De Pooter, T.; Joris, G.; Strazisar, M.; Wood, J.; Herron, K. E.; Seehausen, O.; Watts, P. C.; Shaw, F.; Davey, R. P.; Minotto, A.; Fernandez Gonzalez, J. M.; Bohne, A.; Alegria, C.; Alioto, T.; Alves, P. C.; Amorim, I. R.; Aury, J.-M.; Backstrom, N.; Baldrian, P.; Ballarin, L.; Baltrunaite, L.; Barta, E.; BedHom, B.; Belser, C.; Bergsten, J.; Bertrand, L.; Bilandija, H.; Binzer-Panch\nScore: 34.8, Published: 2023-09-30 DOI: 10.1101/2023.09.25.559365\nA global genome database of all of Earths species diversity could be a treasure trove of scientific discoveries. However, regardless of the major advances in genome sequencing technologies, only a tiny fraction of species have genomic information available. To contribute to a more complete planetary genomic database, scientists and institutions across the world have united under the Earth BioGenome Project (EBP), which plans to sequence and assemble high-quality reference genomes for all [~]1.5 million recognized eukaryotic species through a stepwise phased approach. As the initiative transitions into Phase II, where 150,000 species are to be sequenced in just four years, worldwide participation in the project will be fundamental to success. As the European node of the EBP, the European Reference Genome Atlas (ERGA) seeks to implement a new decentralised, accessible, equitable and inclusive model for producing high-quality reference genomes, which will inform EBP as it scales. To embark on this mission, ERGA launched a Pilot Project to establish a network across Europe to develop and test the first infrastructure of its kind for the coordinated and distributed reference genome production on 98 European eukaryotic species from sample providers across 33 European countries. Here we outline the process and challenges faced during the development of a pilot infrastructure for the production of reference genome resources, and explore the effectiveness of this approach in terms of high-quality reference genome production, considering also equity and inclusion. The outcomes and lessons learned during this pilot provide a solid foundation for ERGA while offering key learnings to other transnational and national genomic resource projects.\nPostglacial genomes from foragers across Northern Eurasia reveal prehistoric mobility associated with the spread of the Uralic and Yeniseian languages\nAuthors: Zeng, T. C.; Vyazov, L. A.; Kim, A.; Flegontov, P.; Sirak, K.; Maier, R.; Lazaridis, I.; Akbari, A.; Frachetti, M.; Tishkin, A. A.; Ryabogina, N. E.; Agapov, S. A.; Agapov, D. S.; Alekseev, A. N.; Boeskorov, G. G.; Chizhevsky, A. A.; Derevianko, A. P.; Dyakonov, V. M.; Enshin, D. N.; Fribus, A. V.; Frolov, Y. V.; Grushin, S. P.; Khokhlov, A. A.; Kitov, E. P.; Kosintsev, P.; Kovtun, I. V.; Kiryushin, K. Y.; Kiryushin, Y. F.; Makarov, N. P.; Morozov, V. V.; Nikolaev, E. N.; Rykun, M. P.; Savenkova, T. M.; Shchelchkova, M. V.; Skochina, S. N.; Shirokov, V.; Sherstobitova, O. S.; Slepchenko, S. M.\nScore: 46.7, Published: 2023-10-04 DOI: 10.1101/2023.10.01.560332\nThe North Eurasian forest and forest-steppe zones have sustained millennia of sociocultural connections among northern peoples. We present genome-wide ancient DNA data for 181 individuals from this region spanning the Mesolithic, Neolithic and Bronze Age. We find that Early to Mid-Holocene hunter-gatherer populations from across the southern forest and forest-steppes of Northern Eurasia can be characterized by a continuous gradient of ancestry that remained stable for millennia, ranging from fully West Eurasian in the Baltic region to fully East Asian in the Transbaikal region. In contrast, cotemporaneous groups in far Northeast Siberia were genetically distinct, retaining high levels of continuity from a population that was the primary source of ancestry for Native Americans. By the mid-Holocene, admixture between this early Northeastern Siberian population and groups from Inland East Asia and the Amur River Basin produced two distinctive populations in eastern Siberia that played an important role in the genetic formation of later people. Ancestry from the first population, Cis-Baikal Late Neolithic-Bronze Age (Cisbaikal_LNBA), is found substantially only among Yeniseian-speaking groups and those known to have admixed with them. Ancestry from the second, Yakutian Late Neolithic-Bronze Age (Yakutia_LNBA), is strongly associated with present-day Uralic speakers. We show how Yakutia_LNBA ancestry spread from an east Siberian origin [~]4.5kya, along with subclades of Y-chromosome haplogroup N occurring at high frequencies among present-day Uralic speakers, into Western and Central Siberia in communities associated with Seima-Turbino metallurgy: a suite of advanced bronze casting techniques that spread explosively across an enormous region of Northern Eurasia [~]4.0kya. However, the ancestry of the 16 Seima-Turbino-period individuals--the first reported from sites with this metallurgy--was otherwise extraordinarily diverse, with partial descent from Indo-Iranian-speaking pastoralists and multiple hunter-gatherer populations from widely separated regions of Eurasia. Our results provide support for theories suggesting that early Uralic speakers at the beginning of their westward dispersal where involved in the expansion of Seima-Turbino metallurgical traditions, and suggests that both cultural transmission and migration were important in the spread of Seima-Turbino material culture.\nNon-coding variants impact cis-regulatory coordination in a cell type-specific manner\nAuthors: Pushkarev, O.; van Mierlo, G.; Kribelbauer, J. F.; Saelens, W.; Gardeux, V.; Deplancke, B.\nScore: 9.2, Published: 2023-10-11 DOI: 10.1101/2023.10.11.561870\nInteractions among cis-regulatory elements (CREs) play a crucial role in gene regulation. Various approaches have been developed to map these interactions genome-wide, including those relying on interindividual epigenomic variation to identify groups of covariable regulatory elements, referred to as chromatin modules (CMs). While CM mapping allows to investigate the relationship between chromatin modularity and gene expression, the computational principles used for CM identification vary in their application and outcomes. Here, we comprehensively evaluate and streamline existing CM mapping tools and present guidelines for optimal utilization of epigenome data from a diverse population of individuals to assess regulatory coordination across the human genome. We showcase the effectiveness of our recommended practices by analysing distinct cell types and demonstrate cell type-specificity of CRE interactions in CMs and their relevance for gene expression. Integration of genotype information revealed that many non-coding disease-associated variants affect the activity of CMs in a cell type-specific manner by affecting the binding of cell type-specific transcription factors. We provide example cases that illustrate in detail how CMs can be used to deconstruct GWAS loci, understand variable expression of cell surface receptors in immune cells and reveal how genetic variation can impact the expression of prognostic markers in chronic lymphocytic leukaemia. Our study presents an optimal strategy for CM mapping, and reveals how CMs capture the coordination of CREs and its impact on gene expression. Non-coding genetic variants can disrupt this coordination, and we highlight how this may lead to disease predisposition in a cell type-specific manner.\nEpigenetic context predicts gene expression variation and reproductive traits across genetically identical individuals\nAuthors: Webster, A. K.; Willis, J. H.; Johnson, E.; Sarkies, P.; Phillips, P. C.\nScore: 8.3, Published: 2023-10-13 DOI: 10.1101/2023.10.13.562270\nIn recent decades, genome-wide association studies (GWAS) have been the major approach to understand the biological basis of individual differences in traits and diseases. However, GWAS approaches have proven to have limited predictive power to explain individual differences, particularly for complex traits and diseases in which environmental factors play a substantial role in their etiology. Indeed, individual differences persist even in genetically identical individuals, although fully separating genetic and environmental causation is difficult or impossible in most organisms. To understand the basis of individual differences in the absence of genetic differences, we measured two quantitative reproductive traits in 180 genetically identical young adult Caenorhabditis elegans roundworms in a shared environment and performed single-individual transcriptomics on each worm. We identified hundreds of genes for which expression variation was strongly associated with reproductive traits, some of which depended on prior environmental experience and some of which was random. Multiple small sets of genes together were highly predictive of reproductive traits across individuals, explaining on average over half and over a quarter of variation in the two traits. We manipulated mRNA levels of predictive genes using RNA interference to identify a set of causal genes, demonstrating the utility of this approach for both prediction and understanding underlying biology. Finally, we found that the chromatin environment of predictive genes was enriched for H3K27 trimethylation, suggesting that individual gene expression differences underlying critical traits may be driven in part by chromatin structure. Together, this work shows that individual differences in gene expression that arise independently of underlying genetic differences are both predictive and causal in shaping reproductive traits at levels that equal or exceed genetic variation.\nRevisiting genomes of non-model species with long reads yields new insights into their biology and evolution\nAuthors: Guiglielmoni, N.; Villegas, L.; Kirangwa, J.; Schiffer, P.\nScore: 8.6, Published: 2023-10-07 DOI: 10.1101/2023.10.06.561169\nHigh-quality genomes obtained using long-read data allow not only for a better understanding of heterozygosity levels, repeat content, and more accurate gene annotation, and prediction when compared to those obtained with short-read technologies, but also allow to understand haplotype divergence. Advances in long-read sequencing technologies in the last years have made it possible to produce such high-quality assemblies for non-model organisms. This allows us to revisit genomes, which have been problematic to scaffold to chromosome-scale with previous generations of data, and assembly software. Nematoda, one of the most diverse, and speciose animal phyla within metazoans, remains poorly studied, and many previously assembled genomes are fragmented. Using long reads obtained with Nanopore R10.4.1 and PacBio HiFi, we generated highly contiguous assemblies of a diploid nematode of the Mermithidae family, for which no closely related genomes are available to date, as well as a collapsed assembly and a phased assembly for a triploid nematode from the Panagrolaimidae family. Both genomes had been analysed before, but the fragmented assemblies had scaffold sizes comparable to the length of long reads prior to assembly. Our new assemblies illustrate how long-read technologies allow for a much better representation of species genomes. We are now able to conduct more accurate downstream assays based on more complete gene and transposable element predictions.\nDeepurify: a multi-modal deep language model to remove contamination from metagenome-assembled genomes\nAuthors: ZOU, B.; Wang, J.; Ding, Y.; Zhang, Z.; Yufen, H.; Fang, X.; Cheung, K. C.; See, S.; Zhang, L.\nScore: 21.1, Published: 2023-09-29 DOI: 10.1101/2023.09.27.559668\nMetagenome-assembled genomes (MAGs) offer valuable insights into the exploration of microbial dark matter using metagenomic sequencing data. However, there is a growing concern that contamination in MAGs may significantly impact the downstream analysis results. Existing MAG decontamination methods heavily rely on marker genes but do not fully leverage genomic sequences. To address the limitations, we have introduced a novel decontamination approach named Deepurify, which utilizes a multi-modal deep language model employing contrastive learning to learn taxonomic similarities of genomic sequences. Deepurify utilizes inferred taxonomic lineages to guide the allocation of contigs into a MAG-separated tree and employs a tree traversal strategy for maximizing the total number of medium- and high-quality MAGs. Extensive experiments were conducted on two simulated datasets, CAMI I, and human gut metagenomic sequencing data. These results demonstrate that Deepurify significantly outperforms other decontamination methods.\n",
  "wordCount" : "2668",
  "inLanguage": "en",
  "datePublished": "2023-10-15T10:38:01Z",
  "dateModified": "2023-10-15T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on October 15, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561180">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561180" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561180">
        <p class="paperTitle">Deep indel mutagenesis reveals the impact of insertions and deletions on protein stability and function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561180" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561180" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Topolska, M.; Beltran, T.; Lehner, B.</p>
        <p class="info">Score: 32.8, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561180' target='https://doi.org/10.1101/2023.10.06.561180'> 10.1101/2023.10.06.561180</a></p>
        <p class="abstract">Amino acid insertions and deletions (indels) are an abundant class of genetic variants. However, compared to substitutions, the effects of indels are not well understood and poorly predicted. Here we address this shortcoming by performing deep indel mutagenesis (DIM) of structurally diverse proteins. Indel tolerance is strikingly different to substitution tolerance and varies extensively both between different proteins and within different regions of the same protein. Although state of the art variant effect predictors perform poorly on indels, we show that both experimentally-measured and computationally-predicted substitution scores can be repurposed as good indel variant effect predictors by incorporating information on protein secondary structures. Quantifying the effects of indels on protein-protein interactions reveals that insertions can be an important class of gain-of-function variants. Our results provide an overview of the impact of indels on proteins and a method to predict their effects genome-wide.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561582">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561582" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561582">
        <p class="paperTitle">The single-molecule accessibility landscape of newly replicated mammalian chromatin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561582" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561582" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ostrowski, M. S.; Yang, M. G.; McNally, C. P.; Abdulhay, N. J.; Wang, S.; Nora, E. P.; Goodarzi, H.; Ramani, V.</p>
        <p class="info">Score: 16.0, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561582' target='https://doi.org/10.1101/2023.10.09.561582'> 10.1101/2023.10.09.561582</a></p>
        <p class="abstract">The higher-order structure of newly replicated (i.e. nascent) chromatin fibers remains poorly-resolved, limiting our understanding of how epigenomes are maintained across cell divisions. To address this, we present Replication-Aware Single-molecule Accessibility Mapping (RASAM), a long-read sequencing method that nondestructively measures genome-wide replication-status and protein-DNA interactions simultaneously on intact chromatin templates. We report that individual human and mouse nascent chromatin fibers are hyperaccessible compared to steady-state chromatin. This hyperaccessibility occurs at two, coupled length-scales: first, individual nucleosome core particles on nascent DNA exist as a mixture of partially-unwrapped nucleosomes and other subnucleosomal species; second, newly-replicated chromatin fibers are significantly enriched for irregularly-spaced nucleosomes on individual DNA molecules. Focusing on specific cis-regulatory elements (e.g. transcription factor binding sites; active transcription start sites [TSSs]), we discover unique modes by which nascent chromatin hyperaccessibility is resolved at the single-molecule level: at CCCTC-binding factor (CTCF) binding sites, CTCF and nascent nucleosomes compete for motifs on nascent chromatin fibers, resulting in quantitatively-reduced CTCF occupancy and motif accessibility post-replication; at active TSSs, high levels of steady-state chromatin accessibility are preserved, implying that nucleosome free regions (NFRs) are rapidly re-established behind the fork. Our study introduces a new paradigm for studying higher-order chromatin fiber organization behind the replication fork. More broadly, we uncover a unique organization of newly replicated chromatin that must be reset by active processes, providing a substrate for epigenetic reprogramming.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.10.561770">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.10.561770" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.10.561770">
        <p class="paperTitle">Developmental and housekeeping transcriptional programs display distinct modes of enhancer-enhancer cooperativity in Drosophila</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.10.561770" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.10.561770" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: LOUBIERE, V.; de Almeida, B. P.; Pagani, M.; Stark, A.</p>
        <p class="info">Score: 15.4, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.10.561770' target='https://doi.org/10.1101/2023.10.10.561770'> 10.1101/2023.10.10.561770</a></p>
        <p class="abstract">Genomic enhancers are key transcriptional regulators which, upon the binding of sequence-specific transcription factors, activate their cognate target promoters. Although enhancers have been extensively studied in isolation, a substantial number of genes have more than one simultaneously active enhancer, and it remains unclear how these cooperate to regulate transcription. Using Drosophila melanogaster S2 cells as a model, we assay the activities of more than a thousand individual enhancers and a million enhancer pairs towards housekeeping and developmental core promoters with STARR-seq. We report that housekeeping and developmental enhancers show distinct modes of enhancer-enhancer cooperativity: while housekeeping enhancers are additive such that their combined activity mirrors the sum of their individual activities, developmental enhancers are synergistic and follow a multiplicative model of cooperativity. This developmental enhancer synergy is promiscuous and neither depends on the enhancers&#39; endogenous genomic contexts nor on specific transcription factor motif signatures, but it saturates for the highest levels of enhancer activity. These results have important implications for our understanding of gene-regulation in complex multi-enhancer loci and genomically clustered housekeeping genes, providing a rationale for strong and mild transcriptional effects of mutations within enhancer regions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561241">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561241" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561241">
        <p class="paperTitle">Insertion sequence driven evolution of the microbiota revealed by a novel metagenomics pipeline</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561241" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561241" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kirsch, J. M.; Hryckowian, A. J.; Duerkop, B. A.</p>
        <p class="info">Score: 15.4, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561241' target='https://doi.org/10.1101/2023.10.06.561241'> 10.1101/2023.10.06.561241</a></p>
        <p class="abstract">Insertion sequence (IS) elements are mobile genetic elements in bacterial genomes that support adaptation. We developed a database of IS elements coupled to a computational pipeline that identifies IS element insertions in the microbiota. We discovered that diverse IS elements insert into the genomes of intestinal bacteria regardless of human host lifestyle. These insertions targeted bacterial accessory genes that aid in their adaptation to unique environmental conditions. Using IS element expansion in Bacteroides, we show that IS element activity leads to insertion &#34;hot spots&#34; in accessory genes. We show that IS element insertions are stable and can be transferred between humans. Extreme environmental perturbations force IS element insertions to fall out of the microbiota and many fail to rebound following homeostasis. Our work shows that IS elements drive bacterial genome diversification within the microbiota and establishes a framework for understanding how strain level variation within the microbiota impacts human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559365" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559365">
        <p class="paperTitle">The European Reference Genome Atlas: piloting a decentralised approach to equitable biodiversity genomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mc Cartney, A. M.; Formenti, G.; Mouton, A.; De Panis, D.; Marins, L. S.; Leitao, H. G.; Diedericks, G.; Kirangwa, J.; Morselli, M.; Salces, J.; Escudero, N.; Iannucci, A.; Natali, C.; Svardal, H.; Fernandez, R.; De Pooter, T.; Joris, G.; Strazisar, M.; Wood, J.; Herron, K. E.; Seehausen, O.; Watts, P. C.; Shaw, F.; Davey, R. P.; Minotto, A.; Fernandez Gonzalez, J. M.; Bohne, A.; Alegria, C.; Alioto, T.; Alves, P. C.; Amorim, I. R.; Aury, J.-M.; Backstrom, N.; Baldrian, P.; Ballarin, L.; Baltrunaite, L.; Barta, E.; BedHom, B.; Belser, C.; Bergsten, J.; Bertrand, L.; Bilandija, H.; Binzer-Panch</p>
        <p class="info">Score: 34.8, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559365' target='https://doi.org/10.1101/2023.09.25.559365'> 10.1101/2023.09.25.559365</a></p>
        <p class="abstract">A global genome database of all of Earths species diversity could be a treasure trove of scientific discoveries. However, regardless of the major advances in genome sequencing technologies, only a tiny fraction of species have genomic information available. To contribute to a more complete planetary genomic database, scientists and institutions across the world have united under the Earth BioGenome Project (EBP), which plans to sequence and assemble high-quality reference genomes for all [~]1.5 million recognized eukaryotic species through a stepwise phased approach. As the initiative transitions into Phase II, where 150,000 species are to be sequenced in just four years, worldwide participation in the project will be fundamental to success. As the European node of the EBP, the European Reference Genome Atlas (ERGA) seeks to implement a new decentralised, accessible, equitable and inclusive model for producing high-quality reference genomes, which will inform EBP as it scales. To embark on this mission, ERGA launched a Pilot Project to establish a network across Europe to develop and test the first infrastructure of its kind for the coordinated and distributed reference genome production on 98 European eukaryotic species from sample providers across 33 European countries. Here we outline the process and challenges faced during the development of a pilot infrastructure for the production of reference genome resources, and explore the effectiveness of this approach in terms of high-quality reference genome production, considering also equity and inclusion. The outcomes and lessons learned during this pilot provide a solid foundation for ERGA while offering key learnings to other transnational and national genomic resource projects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560332">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560332" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560332">
        <p class="paperTitle">Postglacial genomes from foragers across Northern Eurasia reveal prehistoric mobility associated with the spread of the Uralic and Yeniseian languages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560332" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560332" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zeng, T. C.; Vyazov, L. A.; Kim, A.; Flegontov, P.; Sirak, K.; Maier, R.; Lazaridis, I.; Akbari, A.; Frachetti, M.; Tishkin, A. A.; Ryabogina, N. E.; Agapov, S. A.; Agapov, D. S.; Alekseev, A. N.; Boeskorov, G. G.; Chizhevsky, A. A.; Derevianko, A. P.; Dyakonov, V. M.; Enshin, D. N.; Fribus, A. V.; Frolov, Y. V.; Grushin, S. P.; Khokhlov, A. A.; Kitov, E. P.; Kosintsev, P.; Kovtun, I. V.; Kiryushin, K. Y.; Kiryushin, Y. F.; Makarov, N. P.; Morozov, V. V.; Nikolaev, E. N.; Rykun, M. P.; Savenkova, T. M.; Shchelchkova, M. V.; Skochina, S. N.; Shirokov, V.; Sherstobitova, O. S.; Slepchenko, S. M.</p>
        <p class="info">Score: 46.7, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560332' target='https://doi.org/10.1101/2023.10.01.560332'> 10.1101/2023.10.01.560332</a></p>
        <p class="abstract">The North Eurasian forest and forest-steppe zones have sustained millennia of sociocultural connections among northern peoples. We present genome-wide ancient DNA data for 181 individuals from this region spanning the Mesolithic, Neolithic and Bronze Age. We find that Early to Mid-Holocene hunter-gatherer populations from across the southern forest and forest-steppes of Northern Eurasia can be characterized by a continuous gradient of ancestry that remained stable for millennia, ranging from fully West Eurasian in the Baltic region to fully East Asian in the Transbaikal region. In contrast, cotemporaneous groups in far Northeast Siberia were genetically distinct, retaining high levels of continuity from a population that was the primary source of ancestry for Native Americans. By the mid-Holocene, admixture between this early Northeastern Siberian population and groups from Inland East Asia and the Amur River Basin produced two distinctive populations in eastern Siberia that played an important role in the genetic formation of later people. Ancestry from the first population, Cis-Baikal Late Neolithic-Bronze Age (Cisbaikal_LNBA), is found substantially only among Yeniseian-speaking groups and those known to have admixed with them. Ancestry from the second, Yakutian Late Neolithic-Bronze Age (Yakutia_LNBA), is strongly associated with present-day Uralic speakers. We show how Yakutia_LNBA ancestry spread from an east Siberian origin [~]4.5kya, along with subclades of Y-chromosome haplogroup N occurring at high frequencies among present-day Uralic speakers, into Western and Central Siberia in communities associated with Seima-Turbino metallurgy: a suite of advanced bronze casting techniques that spread explosively across an enormous region of Northern Eurasia [~]4.0kya. However, the ancestry of the 16 Seima-Turbino-period individuals--the first reported from sites with this metallurgy--was otherwise extraordinarily diverse, with partial descent from Indo-Iranian-speaking pastoralists and multiple hunter-gatherer populations from widely separated regions of Eurasia. Our results provide support for theories suggesting that early Uralic speakers at the beginning of their westward dispersal where involved in the expansion of Seima-Turbino metallurgical traditions, and suggests that both cultural transmission and migration were important in the spread of Seima-Turbino material culture.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561870">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561870" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561870">
        <p class="paperTitle">Non-coding variants impact cis-regulatory coordination in a cell type-specific manner</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561870" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561870" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pushkarev, O.; van Mierlo, G.; Kribelbauer, J. F.; Saelens, W.; Gardeux, V.; Deplancke, B.</p>
        <p class="info">Score: 9.2, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561870' target='https://doi.org/10.1101/2023.10.11.561870'> 10.1101/2023.10.11.561870</a></p>
        <p class="abstract">Interactions among cis-regulatory elements (CREs) play a crucial role in gene regulation. Various approaches have been developed to map these interactions genome-wide, including those relying on interindividual epigenomic variation to identify groups of covariable regulatory elements, referred to as chromatin modules (CMs). While CM mapping allows to investigate the relationship between chromatin modularity and gene expression, the computational principles used for CM identification vary in their application and outcomes. Here, we comprehensively evaluate and streamline existing CM mapping tools and present guidelines for optimal utilization of epigenome data from a diverse population of individuals to assess regulatory coordination across the human genome. We showcase the effectiveness of our recommended practices by analysing distinct cell types and demonstrate cell type-specificity of CRE interactions in CMs and their relevance for gene expression. Integration of genotype information revealed that many non-coding disease-associated variants affect the activity of CMs in a cell type-specific manner by affecting the binding of cell type-specific transcription factors. We provide example cases that illustrate in detail how CMs can be used to deconstruct GWAS loci, understand variable expression of cell surface receptors in immune cells and reveal how genetic variation can impact the expression of prognostic markers in chronic lymphocytic leukaemia. Our study presents an optimal strategy for CM mapping, and reveals how CMs capture the coordination of CREs and its impact on gene expression. Non-coding genetic variants can disrupt this coordination, and we highlight how this may lead to disease predisposition in a cell type-specific manner.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562270">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562270" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562270">
        <p class="paperTitle">Epigenetic context predicts gene expression variation and reproductive traits across genetically identical individuals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562270" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562270" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Webster, A. K.; Willis, J. H.; Johnson, E.; Sarkies, P.; Phillips, P. C.</p>
        <p class="info">Score: 8.3, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562270' target='https://doi.org/10.1101/2023.10.13.562270'> 10.1101/2023.10.13.562270</a></p>
        <p class="abstract">In recent decades, genome-wide association studies (GWAS) have been the major approach to understand the biological basis of individual differences in traits and diseases. However, GWAS approaches have proven to have limited predictive power to explain individual differences, particularly for complex traits and diseases in which environmental factors play a substantial role in their etiology. Indeed, individual differences persist even in genetically identical individuals, although fully separating genetic and environmental causation is difficult or impossible in most organisms. To understand the basis of individual differences in the absence of genetic differences, we measured two quantitative reproductive traits in 180 genetically identical young adult Caenorhabditis elegans roundworms in a shared environment and performed single-individual transcriptomics on each worm. We identified hundreds of genes for which expression variation was strongly associated with reproductive traits, some of which depended on prior environmental experience and some of which was random. Multiple small sets of genes together were highly predictive of reproductive traits across individuals, explaining on average over half and over a quarter of variation in the two traits. We manipulated mRNA levels of predictive genes using RNA interference to identify a set of causal genes, demonstrating the utility of this approach for both prediction and understanding underlying biology. Finally, we found that the chromatin environment of predictive genes was enriched for H3K27 trimethylation, suggesting that individual gene expression differences underlying critical traits may be driven in part by chromatin structure. Together, this work shows that individual differences in gene expression that arise independently of underlying genetic differences are both predictive and causal in shaping reproductive traits at levels that equal or exceed genetic variation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561169">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561169" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561169">
        <p class="paperTitle">Revisiting genomes of non-model species with long reads yields new insights into their biology and evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561169" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561169" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guiglielmoni, N.; Villegas, L.; Kirangwa, J.; Schiffer, P.</p>
        <p class="info">Score: 8.6, Published: 2023-10-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561169' target='https://doi.org/10.1101/2023.10.06.561169'> 10.1101/2023.10.06.561169</a></p>
        <p class="abstract">High-quality genomes obtained using long-read data allow not only for a better understanding of heterozygosity levels, repeat content, and more accurate gene annotation, and prediction when compared to those obtained with short-read technologies, but also allow to understand haplotype divergence. Advances in long-read sequencing technologies in the last years have made it possible to produce such high-quality assemblies for non-model organisms. This allows us to revisit genomes, which have been problematic to scaffold to chromosome-scale with previous generations of data, and assembly software. Nematoda, one of the most diverse, and speciose animal phyla within metazoans, remains poorly studied, and many previously assembled genomes are fragmented. Using long reads obtained with Nanopore R10.4.1 and PacBio HiFi, we generated highly contiguous assemblies of a diploid nematode of the Mermithidae family, for which no closely related genomes are available to date, as well as a collapsed assembly and a phased assembly for a triploid nematode from the Panagrolaimidae family. Both genomes had been analysed before, but the fragmented assemblies had scaffold sizes comparable to the length of long reads prior to assembly. Our new assemblies illustrate how long-read technologies allow for a much better representation of species genomes. We are now able to conduct more accurate downstream assays based on more complete gene and transposable element predictions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559668">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559668" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559668">
        <p class="paperTitle">Deepurify: a multi-modal deep language model to remove contamination from metagenome-assembled genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559668" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559668" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: ZOU, B.; Wang, J.; Ding, Y.; Zhang, Z.; Yufen, H.; Fang, X.; Cheung, K. C.; See, S.; Zhang, L.</p>
        <p class="info">Score: 21.1, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559668' target='https://doi.org/10.1101/2023.09.27.559668'> 10.1101/2023.09.27.559668</a></p>
        <p class="abstract">Metagenome-assembled genomes (MAGs) offer valuable insights into the exploration of microbial dark matter using metagenomic sequencing data. However, there is a growing concern that contamination in MAGs may significantly impact the downstream analysis results. Existing MAG decontamination methods heavily rely on marker genes but do not fully leverage genomic sequences. To address the limitations, we have introduced a novel decontamination approach named Deepurify, which utilizes a multi-modal deep language model employing contrastive learning to learn taxonomic similarities of genomic sequences. Deepurify utilizes inferred taxonomic lineages to guide the allocation of contigs into a MAG-separated tree and employs a tree traversal strategy for maximizing the total number of medium- and high-quality MAGs. Extensive experiments were conducted on two simulated datasets, CAMI I, and human gut metagenomic sequencing data. These results demonstrate that Deepurify significantly outperforms other decontamination methods.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
