<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="mRNA initiation and termination are spatially coordinated
Authors: Calvo-Roitberg, E.; Carroll, C. L.; Venev, S. V.; Kim, G.; Mick, S. T.; Dekker, J.; Fiszbein, A.; Pai, A. A.
Score: 30.9, Published: 2024-01-07 DOI: 10.1101/2024.01.05.574404
The expression of a precise mRNA transcriptome is crucial for establishing cell identity and function, with dozens of alternative isoforms produced for a single gene sequence. The regulation of mRNA isoform usage occurs by the coordination of co-transcriptional mRNA processing mechanisms across a gene.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="mRNA initiation and termination are spatially coordinated
Authors: Calvo-Roitberg, E.; Carroll, C. L.; Venev, S. V.; Kim, G.; Mick, S. T.; Dekker, J.; Fiszbein, A.; Pai, A. A.
Score: 30.9, Published: 2024-01-07 DOI: 10.1101/2024.01.05.574404
The expression of a precise mRNA transcriptome is crucial for establishing cell identity and function, with dozens of alternative isoforms produced for a single gene sequence. The regulation of mRNA isoform usage occurs by the coordination of co-transcriptional mRNA processing mechanisms across a gene." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-14T10:38:40+00:00" />
<meta property="article:modified_time" content="2024-01-14T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="mRNA initiation and termination are spatially coordinated
Authors: Calvo-Roitberg, E.; Carroll, C. L.; Venev, S. V.; Kim, G.; Mick, S. T.; Dekker, J.; Fiszbein, A.; Pai, A. A.
Score: 30.9, Published: 2024-01-07 DOI: 10.1101/2024.01.05.574404
The expression of a precise mRNA transcriptome is crucial for establishing cell identity and function, with dozens of alternative isoforms produced for a single gene sequence. The regulation of mRNA isoform usage occurs by the coordination of co-transcriptional mRNA processing mechanisms across a gene."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "mRNA initiation and termination are spatially coordinated\nAuthors: Calvo-Roitberg, E.; Carroll, C. L.; Venev, S. V.; Kim, G.; Mick, S. T.; Dekker, J.; Fiszbein, A.; Pai, A. A.\nScore: 30.9, Published: 2024-01-07 DOI: 10.1101/2024.01.05.574404\nThe expression of a precise mRNA transcriptome is crucial for establishing cell identity and function, with dozens of alternative isoforms produced for a single gene sequence. The regulation of mRNA isoform usage occurs by the coordination of co-transcriptional mRNA processing mechanisms across a gene.",
  "keywords": [
    
  ],
  "articleBody": " mRNA initiation and termination are spatially coordinated\nAuthors: Calvo-Roitberg, E.; Carroll, C. L.; Venev, S. V.; Kim, G.; Mick, S. T.; Dekker, J.; Fiszbein, A.; Pai, A. A.\nScore: 30.9, Published: 2024-01-07 DOI: 10.1101/2024.01.05.574404\nThe expression of a precise mRNA transcriptome is crucial for establishing cell identity and function, with dozens of alternative isoforms produced for a single gene sequence. The regulation of mRNA isoform usage occurs by the coordination of co-transcriptional mRNA processing mechanisms across a gene. Decisions involved in mRNA initiation and termination underlie the largest extent of mRNA isoform diversity, but little is known about any relationships between decisions at both ends of mRNA molecules. Here, we systematically profile the joint usage of mRNA transcription start sites (TSSs) and polyadenylation sites (PASs) across tissues and species. Using both short and long read RNA-seq data, we observe that mRNAs preferentially using upstream TSSs also tend to use upstream PASs, and congruently, the usage of downstream sites is similarly paired. This observation suggests that mRNA 5 end choice may directly influence mRNA 3 ends. Our results suggest a novel \"Positional Initiation-Termination Axis\" (PITA), in which the usage of alternative terminal sites are coupled based on the order in which they appear in the genome. PITA isoforms are more likely to encode alternative protein domains and use conserved sites. PITA is strongly associated with the length of genomic features, such that PITA is enriched in longer genes with more area devoted to regions that regulate alternative 5 or 3 ends. Strikingly, we found that PITA genes are more likely than non-PITA genes to have multiple, overlapping chromatin structural domains related to pairing of ordinally coupled start and end sites. In turn, PITA coupling is also associated with fast RNA Polymerase II (RNAPII) trafficking across these long gene regions. Our findings indicate that a combination of spatial and kinetic mechanisms couple transcription initiation and mRNA 3 end decisions based on ordinal position to define the expression mRNA isoforms.\nComparative analysis of multiplexed in situ gene expression profiling technologies\nAuthors: Hartman, A.; Satija, R.\nScore: 25.8, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135\nThe burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets. We find that standard sensitivity metrics, such as the number of unique molecules detected per cell, are not directly comparable across datasets due to substantial differences in the incidence of off-target molecular artifacts impacting specificity. To address these challenges, we explored various potential sources of molecular artifacts, developed novel metrics to control for them, and utilized these metrics to evaluate and compare different in situ technologies. Finally, we demonstrate how molecular false positives can seriously confound spatially-aware differential expression analysis, requiring caution in the interpretation of downstream results. Our analysis provides guidance for the selection, processing, and interpretation of in situ spatial technologies.\nDe novo annotation of the wheat pangenome reveals complexity and diversity of the hexaploid wheat pan-transcriptome\nAuthors: White, B.; LUX, T.; Rusholme-Pilcher, R.; Juhasz, A.; Kaithakottil, G.; Duncan, S.; Simmonds, J.; Rees, H.; Wright, J.; Colmer, J.; Ward, S.; joynson, R.; Coombes, B.; Irish, N.; Henderson, S.; Baker, T.; Chapman, H.; Catchpole, L.; Gharbi, K.; Okada, M.; Handa, H.; Nasuda, S.; Shimizu, K.; Gundlach, H.; Lang, D.; Naamati, G.; Legg, E.; Bharti, A.; Colgrave, M.; Haerty, W. `; Uauy, C. `; Swarbreck, D.; Borrill, P.; Poland, J.; Krattinger, S.; Stein, N.; Mayer, K. F. X.; Pozniak, C.; Spannagl, M.; Hall, A.\nScore: 25.7, Published: 2024-01-09 DOI: 10.1101/2024.01.09.574802\nWheat is the most widely cultivated crop in the world with over 215 million hectares grown annually. However, to meet the demands of a growing global population, breeders face the challenge of increasing wheat production by approximately 60% within the next 40 years. The 10+ Wheat Genomes Project recently sequenced and assembled the genomes of 15 wheat cultivars to develop our understanding of genetic diversity and selection within the pan-genome of wheat. Here, we provide a wheat pan-transcriptome with de novo annotation and differential expression analysis for nine of these wheat cultivars, across multiple different tissues and whole seedlings sampled at dusk/dawn. Analysis of these de novo annotations facilitated the discovery of genes absent from the Chinese Spring reference, identified genes specific to particular cultivars and defined the core and dispensable genomes. Expression analysis across cultivars and tissues revealed conservation in expression between a large core set of homoeologous genes, but also widespread changes in sub-genome homoeolog expression bias between cultivars. Co-expression network analysis revealed the impact of divergence of sub-genome homoeolog expression and identified tissue-associated cultivar-specific expression profiles. In summary, this work provides both a valuable resource for the wider wheat community and reveals diversity in gene content and expression patterns between global wheat cultivars.\nThe majority of somatic mutations in fruit trees are layer-specific\nAuthors: Goel, M.; Campoy, J. A.; Krause, K.; Baus, L. C.; Sahu, A.; Sun, H.; Walkemeier, B.; Marek, M.; Beaudry, R.; Ruiz, D.; Huettel, B.; Schneeberger, K.\nScore: 34.0, Published: 2024-01-04 DOI: 10.1101/2024.01.04.573414\nPlant meristems consist of three distinct stem cells layers. The identities of these layers are maintained in developing organs. Somatic mutations in the meristem thus propagate only into the respective layers of the differentiated organs. We analysed the genomes of individual cell layers in an apricot tree and found an unexpectedly high mutation load, where \u003e90% of the somatic mutations were layer-specific. The layers had similar mutation spectra but layer 1 (epidermis) had substantially more mutations than layer 2 (mesocarp). Most somatic mutations were shared between fruits and neighbouring leaves corroborating their meristematic origin. Single-cell transcriptomics revealed their layer-specific effects forming the basis for layer-specific phenotypes. Our work unveils the hidden abundance of layer-specific meristematic mutations and provides insights into their identification and understanding. One sentence summarySomatic mutations in fruit trees occur in specific cell layers.\nA Phased, Chromosome-scale Genome for Malus domestica 'WA 38'\nAuthors: Zhang, H.; Eaker, A.; Ko, I.; Haney, S.; Khuu, N.; Ryan, K.; Appleby, A.; Hoffmann, B.; Landis, H.; Pierro, K.; Willsea, N.; Hargarten, H.; Yocca, A.; Harkess, A.; Honaas, L.; Ficklin, S.\nScore: 16.8, Published: 2024-01-11 DOI: 10.1101/2024.01.10.574953\nGenome sequencing for agriculturally important Rosaceous crops has made rapid progress both in completeness and annotation quality. Whole genome sequence and annotation gives breeders, researchers, and growers information about cultivar specific traits such as fruit quality, disease resistance, and informs strategies to enhance postharvest storage. Here we present a haplotype-phased, chromosomal level, genome of Malus domestica, WA 38, a new cultivar of apple released to market in 2017 as Cosmic Crisp (R). Using both short and long read sequencing our fully phased genome of WA 38 is about 650 Megabases, with a BUSCO score of 98.7% complete. The genome contains two haplomes (HapA and HapB) which are composed of 17 chromosomes each. A total of 48,227 and 54,479 genes were annotated from HapA and HapB, respectively. Additionally, we provide genome-scale comparisons to Gala, Honeycrisp, and other relevant cultivars highlighting major structural and quality differences. This assembly and annotation was done in collaboration with the American Campus Tree Genomes project that included WA 38 (WSU) and the dAnjou pear (Auburn University). To ensure transparency, reproducibility, and applicability for any genome project, our genome assembly and annotation workflow is recorded in detail and shared under a public GitHub repository. All software is containerized, offering a simple implementation of the workflow.\nLong-read RNA sequencing identifies region- and sex-specific C57BL/6J mouse brain mRNA isoform expression and usage\nAuthors: Jones, E. F.; Howton, T. C.; Flanary, V. L.; Clark, A. D.; Lasseigne, B. N.\nScore: 13.6, Published: 2024-01-11 DOI: 10.1101/2024.01.11.575219\nAlternative splicing (AS) contributes to the biological heterogeneity between species, sexes, tissues, and cell types. Many diseases are either caused by alterations in AS or by alterations to AS. Therefore, measuring AS accurately and efficiently is critical for assessing molecular phenotypes, including those associated with disease. Long-read sequencing enables more accurate quantification of differentially spliced isoform expression than short-read sequencing approaches, and third-generation platforms facilitate high-throughput experiments. To assess differences in AS across the cerebellum, cortex, hippocampus, and striatum by sex, we generated and analyzed Oxford Nanopore Technologies (ONT) long-read RNA sequencing (lrRNA-Seq) C57BL/6J mouse brain cDNA libraries. From \u003e85 million reads that passed quality control metrics, we calculated differential gene expression (DGE), differential transcript expression (DTE), and differential transcript usage (DTU) across brain regions and by sex. We found significant DGE, DTE, and DTU across brain regions and that the cerebellum had the most differences compared to the other three regions. Additionally, we found region-specific differential splicing between sexes, with the most sex differences in DTU in the cortex and no DTU in the hippocampus. We also report on two distinct patterns of sex DTU we observed, sex-divergent and sex-specific, that could potentially help explain sex differences in the prevalence and prognosis of various neurological and psychiatric disorders in future studies. Finally, we built a Shiny web application for researchers to explore the data further. Our study provides a resource for the community; it underscores the importance of AS in biological heterogeneity and the utility of long-read sequencing to better understand AS in the brain.\nConvergence and global epidemiology of Klebsiella pneumoniae plasmids harbouring the iuc3 virulence locus.\nAuthors: Gibbon, M. J.; Couto, N.; Cozens, K.; Habib, S.; Cowley, L.; Aanensen, D.; Corander, J.; Thorpe, H.; Hetland, M. A.; Sassera, D.; Merla, C.; Corbella, M.; Ferrari, C.; Turner, K. M.; Sirikancha, K.; Dulyayangkul, P.; Alhusein, N.; Charoenlap, N.; Thamlikitikul, V.; Avison, M. B.; Feil, E. J.\nScore: 14.8, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574329\nBackgroundKlebsiella pneumoniae (Kp) is an important pathogen of humans and animals, and recent reports of convergent strains that carry both virulence and antimicrobial resistance genes (ARGs) have raised serious public health concern. The plasmid-borne iuc locus, encoding the siderophore aerobactin, is a key virulence factor in this species. The variant iuc3 is associated with porcine and human clinical isolates and is carried by mostly uncharacterised IncF plasmids. MethodsWe used a combination of short-read and long-read sequencing to characterise IncFIB(K)/IncFII iuc3-carrying plasmids harboured by 79 Kp isolates and one K. oxytoca isolate recovered as part of two large One-Health studies in Italy (SpARK) and Thailand (OH-DART). Adding data from public repositories gave a combined dataset of 517 iuc3 isolates, and the plasmids were analysed using both clustering and phylogenetic methods. FindingsWe note seven large, convergent, plasmids from Thailand that have emerged through the hybridisation of co-circulating plasmids harbouring iuc3 and antimicrobial resistance genes (ARGs) encoding extended-spectrum beta-lactamases (ESBLs). We were also able to identify putative parental plasmids which were mostly associated with two neighbouring meat markets, as were the hybrid plasmids. Clustering and global phylogenetic analyses resolved an iuc3 plasmid sub-group circulating throughout Asia, with occasional examples in Europe and elsewhere. This variant carries multiple ARGs and is commonly harboured by clinical isolates, thus warranting targeted plasmid surveillance. InterpretationOur study reveals that plasmid hybridisation leading to the convergence of resistance and virulence traits may be very common, even in non-clinical ( One-Health) settings. Population-scale plasmid genomics makes it possible to identify putative parental plasmids, which will help to identify plasmid types that are most likely to hybridise, and what the selective consequences may be for the plasmid and host. A distinct iuc3 plasmid sub-variant is associated with clinical isolates in Asia which requires close monitoring. Research In ContextMultiple reports of convergent clones of Klebsiella pneumoniae that combine both hypervirulence and multidrug resistance (MDR-hvKp) have been published recently; a PubMed search in November 2023 using the key words convergence Klebsiella pneumoniae returned 143 papers, 99 of which were published from 2020 onwards. Our study demonstrates that the hybridisation of plasmids carrying AMR and virulence genes is a frequent, ongoing, process in natural populations. The subsequent transfer of plasmids conferring both traits is thus likely to be a key driver behind the spread of convergent strains. Our study also provides an exemplar of how hybrid assemblies can facilitate large-scale global genomic plasmid epidemiology. Evidence before the studyAlthough multiple recent reports highlight the emergence and spread of convergent Kp strains, the confluence of resistance and virulence genes within the same plasmid has not been studied at a population level, and putative parental plasmids are rarely identified. Moreover, there have been few high-resolution genomic epidemiology studies on closely related plasmids using both long and short-read data on a global scale. Added valueWe more than double the number of complete sequences available for plasmids harbouring iuc3 from 58 to 139 and provide evidence on the host lineages most likely to harbour these plasmids (e.g., ST35), and epidemiological source (e.g., pig, wild animal, human). Our comparative analysis of phylogenetic and clustering approaches will help to inform future plasmid epidemiological studies. ImplicationsThe hybridisation of plasmids harbouring virulence and resistance genes occurs frequently in natural populations, even within One-Health settings. However, the selective drivers (if any) and evolutionary consequences of this phenomenon are unclear. There is clear utility in generating closed plasmid genomes on a population scale, and targeted plasmid surveillance on a clinical sub-variant of iuc3 plasmids is warranted.\nComprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms\nAuthors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.\nScore: 55.5, Published: 2024-01-06 DOI: 10.1101/2024.01.02.573821\nResearch and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance. This necessitates a framework to identify all types of variants independent of their size (e.g., SNV/SV) or location (e.g., repeats). Here we present DRAGEN that utilizes novel methods based on multigenomes, hardware acceleration, and machine learning based variant detection to provide novel insights into individual genomes with [~]30min computation time (from raw reads to variant detection). DRAGEN outperforms all other state-of-the-art methods in speed and accuracy across all variant types (SNV, indel, STR, SV, CNV) and further incorporates specialized methods to obtain key insights in medically relevant genes (e.g., HLA, SMN, GBA). We showcase DRAGEN across 3,202 genomes and demonstrate its scalability, accuracy, and innovations to further advance the integration of comprehensive genomics for research and medical applications.\nA validated pangenome-scale metabolic model for the Klebsiella pneumoniae species complex\nAuthors: Cooper, H. B.; Vezina, B.; Hawkey, J.; Passet, V.; Lopez-Fernandez, S.; Monk, J. M.; Brisse, S.; Holt, K. E.; Wyres, K. L.\nScore: 9.3, Published: 2023-12-21 DOI: 10.1101/2023.12.20.572682\nThe Klebsiella pneumoniae Species Complex (KpSC) is a major source of nosocomial infections globally with high rates of resistance to antimicrobials. Consequently, there is growing interest in understanding virulence factors and their association with cellular metabolic processes for developing novel anti-KpSC therapeutics. Phenotypic assays have revealed metabolic diversity within the KpSC, but metabolism research has been neglected due to experiments being difficult and cost-intensive. Genome-scale metabolic models (GSMMs) represent a rapid and scalable in silico approach for exploring metabolic diversity, which compiles genomic and biochemical data to reconstruct the metabolic network of an organism. Here we use a diverse collection of 507 KpSC isolates, including representatives of globally distributed clinically-relevant lineages, to construct the most comprehensive KpSC pan-metabolic model to-date, KpSC pan v2. Candidate metabolic reactions were identified using gene orthology to known metabolic genes, prior to manual curation via extensive literature and database searches. The final model comprised a total of 3,550 reactions, 2,403 genes and can simulate growth on 360 unique substrates. We used KpSC pan v2 as a reference to derive strain-specific GSMMs for all 507 KpSC isolates, and compared these to GSMMs generated using a prior KpSC pan-reference (KpSC pan v1) and two single-strain references. We show that KpSC pan v2 includes a greater proportion of accessory reactions (8.8%) than KpSC pan v1 (2.5%). GSMMs derived from KpSC pan v2 also result in more accuracy growth predictions than those derived from other references in both aerobic (median accuracy = 95.4%) and anaerobic (median accuracy = 78.8%). KpSC pan v2 also generates more accurate growth predictions, with high median accuracies of 95.4% (aerobic, n=37 isolates) and 78.8% (anaerobic, n=36 isolates) for 124 matched carbon substrates. KpSC pan v2 is freely available at https://github.com/kelwyres/KpSC-pan-metabolic-model, representing a valuable resource for the scientific community, both as a source of curated metabolic information and as a reference to derive accurate strain-specific GSMMs. The latter can be used to investigate the relationship between KpSC metabolism and traits of interest, such as reservoirs, epidemiology, drug resistance or virulence, and ultimately to inform novel KpSC control strategies. Significance as a BioResource to the communityKlebsiella pneumoniae and its close relatives in the K. pneumoniae Species Complex (KpSC) are priority antimicrobial resistant pathogens that exhibit extensive genomic diversity. There is growing interest in understanding KpSC metabolism, and genome scale metabolic models (GSMMs) provide a rapid, scalable option for exploration of whole cell metabolism plus phenotype prediction. Here we present a KpSC pan-metabolic model representing the cellular metabolism of 507 diverse KpSC isolates. Our model is the largest and most comprehensive of its kind, comprising \u003e2,400 genes associated with \u003e3,500 metabolic reactions, plus manually curated evidence annotations. These data alone represent a key knowledge resource for the Klebsiella research community; however, our models greatest impact lies in its potential for use as a reference from which highly accurate strain-specific GSMMs can be derived to inform in depth strain-specific and/or large-scale comparative analyses. Data summaryO_LIKlebsiella pneumoniae species complex (KpSC) pan v2 metabolic model available at https://github.com/kelwyres/KpSC-pan-metabolic-model. C_LIO_LIAll KpSC isolate whole genome sequences used in this work were reported previously and are available under Bioprojects PRJEB6891, PRJNA351909, PRJNA493667, PRJNA768294, PRJNA253462, PRJNA292902 and PRJNA391323. Individual accessions listed in Table S1. C_LIO_LIStrain-specific GSMMs used for comparative analyses (deposited in Figshare - 10.6084/m9.figshare.24871914), plus their associated MEMOTE reports (indicates completeness and annotation quality), reaction and gene presence-absence matrices across all isolates. C_LIO_LIGrowth phenotype predictions derived from strain-specific GSMMs are available in Table S4. C_LIO_LIBinarised Biolog growth phenotype data for n=37 isolates (plates PM1 and PM2, aerobic and anaerobic conditions) are available in Tables S6 \u0026 S7. C_LIO_LIAdditional growth assay data for six substrates not included on Biolog plates PM1 and PM2 (deposited in Figshare - 10.6084/m9.figshare.24871914). C_LI\nPangenome reconstruction in rats enhances genotype-phenotype mapping and novel variant discovery\nAuthors: Villani, F.; Guarracino, A.; Ward, R. R.; Green, T.; Emms, M.; Pravenec, M.; Prins, P.; Garrison, E.; Williams, R. W.; Chen, H.; Colonna, V.\nScore: 5.3, Published: 2024-01-11 DOI: 10.1101/2024.01.10.575041\nThe HXB/BXH family of recombinant inbred rat strains is a unique genetic resource that has been extensively phenotyped over 25 years, resulting in a vast dataset of quantitative molecular and physiological phenotypes. We built a pangenome graph from 10x Genomics linked-read data for 31 recombinant inbred rats to study genetic variation and association mapping. The pangenome length was on average 2.4 times greater than the corresponding length of the reference mRatBN7.2, confirming the capture of substantial additional variation. We validated variants in challenging regions, including complex structural variants resolving into multiple haplotypes. Phenome-wide association analysis of validated SNPs uncovered variants associated with glucose/insulin levels and hippocampal gene expression. We propose an interaction between Pirl1l1, Cromogranine expression, TNF- levels, and insulin regulation. This study demonstrates the utility of linked-read pangenomes for comprehensive variant detection and mapping phenotypic diversity in a widely used rat genetic reference panel.\n",
  "wordCount" : "3229",
  "inLanguage": "en",
  "datePublished": "2024-01-14T10:38:40Z",
  "dateModified": "2024-01-14T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574404">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574404" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574404">
        <p class="paperTitle">mRNA initiation and termination are spatially coordinated</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574404" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574404" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Calvo-Roitberg, E.; Carroll, C. L.; Venev, S. V.; Kim, G.; Mick, S. T.; Dekker, J.; Fiszbein, A.; Pai, A. A.</p>
        <p class="info">Score: 30.9, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574404' target='https://doi.org/10.1101/2024.01.05.574404'> 10.1101/2024.01.05.574404</a></p>
        <p class="abstract">The expression of a precise mRNA transcriptome is crucial for establishing cell identity and function, with dozens of alternative isoforms produced for a single gene sequence. The regulation of mRNA isoform usage occurs by the coordination of co-transcriptional mRNA processing mechanisms across a gene. Decisions involved in mRNA initiation and termination underlie the largest extent of mRNA isoform diversity, but little is known about any relationships between decisions at both ends of mRNA molecules. Here, we systematically profile the joint usage of mRNA transcription start sites (TSSs) and polyadenylation sites (PASs) across tissues and species. Using both short and long read RNA-seq data, we observe that mRNAs preferentially using upstream TSSs also tend to use upstream PASs, and congruently, the usage of downstream sites is similarly paired. This observation suggests that mRNA 5 end choice may directly influence mRNA 3 ends. Our results suggest a novel &#34;Positional Initiation-Termination Axis&#34; (PITA), in which the usage of alternative terminal sites are coupled based on the order in which they appear in the genome. PITA isoforms are more likely to encode alternative protein domains and use conserved sites. PITA is strongly associated with the length of genomic features, such that PITA is enriched in longer genes with more area devoted to regions that regulate alternative 5 or 3 ends. Strikingly, we found that PITA genes are more likely than non-PITA genes to have multiple, overlapping chromatin structural domains related to pairing of ordinally coupled start and end sites. In turn, PITA coupling is also associated with fast RNA Polymerase II (RNAPII) trafficking across these long gene regions. Our findings indicate that a combination of spatial and kinetic mechanisms couple transcription initiation and mRNA 3 end decisions based on ordinal position to define the expression mRNA isoforms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575135" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575135">
        <p class="paperTitle">Comparative analysis of multiplexed in situ gene expression profiling technologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hartman, A.; Satija, R.</p>
        <p class="info">Score: 25.8, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575135' target='https://doi.org/10.1101/2024.01.11.575135'> 10.1101/2024.01.11.575135</a></p>
        <p class="abstract">The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets. We find that standard sensitivity metrics, such as the number of unique molecules detected per cell, are not directly comparable across datasets due to substantial differences in the incidence of off-target molecular artifacts impacting specificity. To address these challenges, we explored various potential sources of molecular artifacts, developed novel metrics to control for them, and utilized these metrics to evaluate and compare different in situ technologies. Finally, we demonstrate how molecular false positives can seriously confound spatially-aware differential expression analysis, requiring caution in the interpretation of downstream results. Our analysis provides guidance for the selection, processing, and interpretation of in situ spatial technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.09.574802">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.09.574802" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.09.574802">
        <p class="paperTitle">De novo annotation of the wheat pangenome reveals complexity and diversity of the hexaploid wheat pan-transcriptome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.09.574802" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.09.574802" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: White, B.; LUX, T.; Rusholme-Pilcher, R.; Juhasz, A.; Kaithakottil, G.; Duncan, S.; Simmonds, J.; Rees, H.; Wright, J.; Colmer, J.; Ward, S.; joynson, R.; Coombes, B.; Irish, N.; Henderson, S.; Baker, T.; Chapman, H.; Catchpole, L.; Gharbi, K.; Okada, M.; Handa, H.; Nasuda, S.; Shimizu, K.; Gundlach, H.; Lang, D.; Naamati, G.; Legg, E.; Bharti, A.; Colgrave, M.; Haerty, W. `; Uauy, C. `; Swarbreck, D.; Borrill, P.; Poland, J.; Krattinger, S.; Stein, N.; Mayer, K. F. X.; Pozniak, C.; Spannagl, M.; Hall, A.</p>
        <p class="info">Score: 25.7, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.09.574802' target='https://doi.org/10.1101/2024.01.09.574802'> 10.1101/2024.01.09.574802</a></p>
        <p class="abstract">Wheat is the most widely cultivated crop in the world with over 215 million hectares grown annually. However, to meet the demands of a growing global population, breeders face the challenge of increasing wheat production by approximately 60% within the next 40 years. The 10&#43; Wheat Genomes Project recently sequenced and assembled the genomes of 15 wheat cultivars to develop our understanding of genetic diversity and selection within the pan-genome of wheat. Here, we provide a wheat pan-transcriptome with de novo annotation and differential expression analysis for nine of these wheat cultivars, across multiple different tissues and whole seedlings sampled at dusk/dawn. Analysis of these de novo annotations facilitated the discovery of genes absent from the Chinese Spring reference, identified genes specific to particular cultivars and defined the core and dispensable genomes. Expression analysis across cultivars and tissues revealed conservation in expression between a large core set of homoeologous genes, but also widespread changes in sub-genome homoeolog expression bias between cultivars. Co-expression network analysis revealed the impact of divergence of sub-genome homoeolog expression and identified tissue-associated cultivar-specific expression profiles. In summary, this work provides both a valuable resource for the wider wheat community and reveals diversity in gene content and expression patterns between global wheat cultivars.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.573414">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.573414" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.573414">
        <p class="paperTitle">The majority of somatic mutations in fruit trees are layer-specific</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.573414" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.573414" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goel, M.; Campoy, J. A.; Krause, K.; Baus, L. C.; Sahu, A.; Sun, H.; Walkemeier, B.; Marek, M.; Beaudry, R.; Ruiz, D.; Huettel, B.; Schneeberger, K.</p>
        <p class="info">Score: 34.0, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.573414' target='https://doi.org/10.1101/2024.01.04.573414'> 10.1101/2024.01.04.573414</a></p>
        <p class="abstract">Plant meristems consist of three distinct stem cells layers. The identities of these layers are maintained in developing organs. Somatic mutations in the meristem thus propagate only into the respective layers of the differentiated organs. We analysed the genomes of individual cell layers in an apricot tree and found an unexpectedly high mutation load, where &gt;90% of the somatic mutations were layer-specific. The layers had similar mutation spectra but layer 1 (epidermis) had substantially more mutations than layer 2 (mesocarp). Most somatic mutations were shared between fruits and neighbouring leaves corroborating their meristematic origin. Single-cell transcriptomics revealed their layer-specific effects forming the basis for layer-specific phenotypes. Our work unveils the hidden abundance of layer-specific meristematic mutations and provides insights into their identification and understanding.

One sentence summarySomatic mutations in fruit trees occur in specific cell layers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.574953">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.574953" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.574953">
        <p class="paperTitle">A Phased, Chromosome-scale Genome for Malus domestica &#39;WA 38&#39;</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.574953" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.574953" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, H.; Eaker, A.; Ko, I.; Haney, S.; Khuu, N.; Ryan, K.; Appleby, A.; Hoffmann, B.; Landis, H.; Pierro, K.; Willsea, N.; Hargarten, H.; Yocca, A.; Harkess, A.; Honaas, L.; Ficklin, S.</p>
        <p class="info">Score: 16.8, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.574953' target='https://doi.org/10.1101/2024.01.10.574953'> 10.1101/2024.01.10.574953</a></p>
        <p class="abstract">Genome sequencing for agriculturally important Rosaceous crops has made rapid progress both in completeness and annotation quality. Whole genome sequence and annotation gives breeders, researchers, and growers information about cultivar specific traits such as fruit quality, disease resistance, and informs strategies to enhance postharvest storage. Here we present a haplotype-phased, chromosomal level, genome of Malus domestica,  WA 38, a new cultivar of apple released to market in 2017 as Cosmic Crisp (R). Using both short and long read sequencing our fully phased genome of  WA 38 is about 650 Megabases, with a BUSCO score of 98.7% complete. The genome contains two haplomes (HapA and HapB) which are composed of 17 chromosomes each. A total of 48,227 and 54,479 genes were annotated from HapA and HapB, respectively. Additionally, we provide genome-scale comparisons to  Gala,  Honeycrisp, and other relevant cultivars highlighting major structural and quality differences. This assembly and annotation was done in collaboration with the American Campus Tree Genomes project that included  WA 38 (WSU) and the  dAnjou pear (Auburn University). To ensure transparency, reproducibility, and applicability for any genome project, our genome assembly and annotation workflow is recorded in detail and shared under a public GitHub repository. All software is containerized, offering a simple implementation of the workflow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575219">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575219" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575219">
        <p class="paperTitle">Long-read RNA sequencing identifies region- and sex-specific C57BL/6J mouse brain mRNA isoform expression and usage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575219" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575219" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jones, E. F.; Howton, T. C.; Flanary, V. L.; Clark, A. D.; Lasseigne, B. N.</p>
        <p class="info">Score: 13.6, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575219' target='https://doi.org/10.1101/2024.01.11.575219'> 10.1101/2024.01.11.575219</a></p>
        <p class="abstract">Alternative splicing (AS) contributes to the biological heterogeneity between species, sexes, tissues, and cell types. Many diseases are either caused by alterations in AS or by alterations to AS. Therefore, measuring AS accurately and efficiently is critical for assessing molecular phenotypes, including those associated with disease. Long-read sequencing enables more accurate quantification of differentially spliced isoform expression than short-read sequencing approaches, and third-generation platforms facilitate high-throughput experiments. To assess differences in AS across the cerebellum, cortex, hippocampus, and striatum by sex, we generated and analyzed Oxford Nanopore Technologies (ONT) long-read RNA sequencing (lrRNA-Seq) C57BL/6J mouse brain cDNA libraries. From &gt;85 million reads that passed quality control metrics, we calculated differential gene expression (DGE), differential transcript expression (DTE), and differential transcript usage (DTU) across brain regions and by sex. We found significant DGE, DTE, and DTU across brain regions and that the cerebellum had the most differences compared to the other three regions. Additionally, we found region-specific differential splicing between sexes, with the most sex differences in DTU in the cortex and no DTU in the hippocampus. We also report on two distinct patterns of sex DTU we observed, sex-divergent and sex-specific, that could potentially help explain sex differences in the prevalence and prognosis of various neurological and psychiatric disorders in future studies. Finally, we built a Shiny web application for researchers to explore the data further. Our study provides a resource for the community; it underscores the importance of AS in biological heterogeneity and the utility of long-read sequencing to better understand AS in the brain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574329">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574329" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574329">
        <p class="paperTitle">Convergence and global epidemiology of Klebsiella pneumoniae plasmids harbouring the iuc3 virulence locus.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574329" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574329" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gibbon, M. J.; Couto, N.; Cozens, K.; Habib, S.; Cowley, L.; Aanensen, D.; Corander, J.; Thorpe, H.; Hetland, M. A.; Sassera, D.; Merla, C.; Corbella, M.; Ferrari, C.; Turner, K. M.; Sirikancha, K.; Dulyayangkul, P.; Alhusein, N.; Charoenlap, N.; Thamlikitikul, V.; Avison, M. B.; Feil, E. J.</p>
        <p class="info">Score: 14.8, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574329' target='https://doi.org/10.1101/2024.01.05.574329'> 10.1101/2024.01.05.574329</a></p>
        <p class="abstract">BackgroundKlebsiella pneumoniae (Kp) is an important pathogen of humans and animals, and recent reports of  convergent strains that carry both virulence and antimicrobial resistance genes (ARGs) have raised serious public health concern. The plasmid-borne iuc locus, encoding the siderophore aerobactin, is a key virulence factor in this species. The variant iuc3 is associated with porcine and human clinical isolates and is carried by mostly uncharacterised IncF plasmids.

MethodsWe used a combination of short-read and long-read sequencing to characterise IncFIB(K)/IncFII iuc3-carrying plasmids harboured by 79 Kp isolates and one K. oxytoca isolate recovered as part of two large  One-Health studies in Italy (SpARK) and Thailand (OH-DART). Adding data from public repositories gave a combined dataset of 517 iuc3 isolates, and the plasmids were analysed using both clustering and phylogenetic methods.

FindingsWe note seven large, convergent, plasmids from Thailand that have emerged through the hybridisation of co-circulating plasmids harbouring iuc3 and antimicrobial resistance genes (ARGs) encoding extended-spectrum beta-lactamases (ESBLs). We were also able to identify putative parental plasmids which were mostly associated with two neighbouring meat markets, as were the hybrid plasmids. Clustering and global phylogenetic analyses resolved an iuc3 plasmid sub-group circulating throughout Asia, with occasional examples in Europe and elsewhere. This variant carries multiple ARGs and is commonly harboured by clinical isolates, thus warranting targeted plasmid surveillance.

InterpretationOur study reveals that plasmid hybridisation leading to the convergence of resistance and virulence traits may be very common, even in non-clinical ( One-Health) settings. Population-scale plasmid genomics makes it possible to identify putative parental plasmids, which will help to identify plasmid types that are most likely to hybridise, and what the selective consequences may be for the plasmid and host. A distinct iuc3 plasmid sub-variant is associated with clinical isolates in Asia which requires close monitoring.

Research In ContextMultiple reports of  convergent clones of Klebsiella pneumoniae that combine both hypervirulence and multidrug resistance (MDR-hvKp) have been published recently; a PubMed search in November 2023 using the key words  convergence Klebsiella pneumoniae returned 143 papers, 99 of which were published from 2020 onwards. Our study demonstrates that the hybridisation of plasmids carrying AMR and virulence genes is a frequent, ongoing, process in natural populations. The subsequent transfer of plasmids conferring both traits is thus likely to be a key driver behind the spread of convergent strains. Our study also provides an exemplar of how hybrid assemblies can facilitate large-scale global genomic plasmid epidemiology.

Evidence before the studyAlthough multiple recent reports highlight the emergence and spread of convergent Kp strains, the confluence of resistance and virulence genes within the same plasmid has not been studied at a population level, and putative parental plasmids are rarely identified. Moreover, there have been few high-resolution genomic epidemiology studies on closely related plasmids using both long and short-read data on a global scale.

Added valueWe more than double the number of complete sequences available for plasmids harbouring iuc3 from 58 to 139 and provide evidence on the host lineages most likely to harbour these plasmids (e.g., ST35), and epidemiological source (e.g., pig, wild animal, human). Our comparative analysis of phylogenetic and clustering approaches will help to inform future plasmid epidemiological studies.

ImplicationsThe hybridisation of plasmids harbouring virulence and resistance genes occurs frequently in natural populations, even within  One-Health settings. However, the selective drivers (if any) and evolutionary consequences of this phenomenon are unclear. There is clear utility in generating closed plasmid genomes on a population scale, and targeted plasmid surveillance on a clinical sub-variant of iuc3 plasmids is warranted.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573821">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573821" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573821">
        <p class="paperTitle">Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573821" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573821" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.</p>
        <p class="info">Score: 55.5, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573821' target='https://doi.org/10.1101/2024.01.02.573821'> 10.1101/2024.01.02.573821</a></p>
        <p class="abstract">Research and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance. This necessitates a framework to identify all types of variants independent of their size (e.g., SNV/SV) or location (e.g., repeats). Here we present DRAGEN that utilizes novel methods based on multigenomes, hardware acceleration, and machine learning based variant detection to provide novel insights into individual genomes with [~]30min computation time (from raw reads to variant detection). DRAGEN outperforms all other state-of-the-art methods in speed and accuracy across all variant types (SNV, indel, STR, SV, CNV) and further incorporates specialized methods to obtain key insights in medically relevant genes (e.g., HLA, SMN, GBA). We showcase DRAGEN across 3,202 genomes and demonstrate its scalability, accuracy, and innovations to further advance the integration of comprehensive genomics for research and medical applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572682">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572682" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572682">
        <p class="paperTitle">A validated pangenome-scale metabolic model for the Klebsiella pneumoniae species complex</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572682" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572682" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cooper, H. B.; Vezina, B.; Hawkey, J.; Passet, V.; Lopez-Fernandez, S.; Monk, J. M.; Brisse, S.; Holt, K. E.; Wyres, K. L.</p>
        <p class="info">Score: 9.3, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572682' target='https://doi.org/10.1101/2023.12.20.572682'> 10.1101/2023.12.20.572682</a></p>
        <p class="abstract">The Klebsiella pneumoniae Species Complex (KpSC) is a major source of nosocomial infections globally with high rates of resistance to antimicrobials. Consequently, there is growing interest in understanding virulence factors and their association with cellular metabolic processes for developing novel anti-KpSC therapeutics. Phenotypic assays have revealed metabolic diversity within the KpSC, but metabolism research has been neglected due to experiments being difficult and cost-intensive.

Genome-scale metabolic models (GSMMs) represent a rapid and scalable in silico approach for exploring metabolic diversity, which compiles genomic and biochemical data to reconstruct the metabolic network of an organism. Here we use a diverse collection of 507 KpSC isolates, including representatives of globally distributed clinically-relevant lineages, to construct the most comprehensive KpSC pan-metabolic model to-date, KpSC pan v2. Candidate metabolic reactions were identified using gene orthology to known metabolic genes, prior to manual curation via extensive literature and database searches. The final model comprised a total of 3,550 reactions, 2,403 genes and can simulate growth on 360 unique substrates. We used KpSC pan v2 as a reference to derive strain-specific GSMMs for all 507 KpSC isolates, and compared these to GSMMs generated using a prior KpSC pan-reference (KpSC pan v1) and two single-strain references. We show that KpSC pan v2 includes a greater proportion of accessory reactions (8.8%) than KpSC pan v1 (2.5%). GSMMs derived from KpSC pan v2 also result in more accuracy growth predictions than those derived from other references in both aerobic (median accuracy = 95.4%) and anaerobic (median accuracy = 78.8%). KpSC pan v2 also generates more accurate growth predictions, with high median accuracies of 95.4% (aerobic, n=37 isolates) and 78.8% (anaerobic, n=36 isolates) for 124 matched carbon substrates.

KpSC pan v2 is freely available at https://github.com/kelwyres/KpSC-pan-metabolic-model, representing a valuable resource for the scientific community, both as a source of curated metabolic information and as a reference to derive accurate strain-specific GSMMs. The latter can be used to investigate the relationship between KpSC metabolism and traits of interest, such as reservoirs, epidemiology, drug resistance or virulence, and ultimately to inform novel KpSC control strategies.

Significance as a BioResource to the communityKlebsiella pneumoniae and its close relatives in the K. pneumoniae Species Complex (KpSC) are priority antimicrobial resistant pathogens that exhibit extensive genomic diversity. There is growing interest in understanding KpSC metabolism, and genome scale metabolic models (GSMMs) provide a rapid, scalable option for exploration of whole cell metabolism plus phenotype prediction. Here we present a KpSC pan-metabolic model representing the cellular metabolism of 507 diverse KpSC isolates. Our model is the largest and most comprehensive of its kind, comprising &gt;2,400 genes associated with &gt;3,500 metabolic reactions, plus manually curated evidence annotations. These data alone represent a key knowledge resource for the Klebsiella research community; however, our models greatest impact lies in its potential for use as a reference from which highly accurate strain-specific GSMMs can be derived to inform in depth strain-specific and/or large-scale comparative analyses.

Data summaryO_LIKlebsiella pneumoniae species complex (KpSC) pan v2 metabolic model available at https://github.com/kelwyres/KpSC-pan-metabolic-model.
C_LIO_LIAll KpSC isolate whole genome sequences used in this work were reported previously and are available under Bioprojects PRJEB6891, PRJNA351909, PRJNA493667, PRJNA768294, PRJNA253462, PRJNA292902 and PRJNA391323. Individual accessions listed in Table S1.
C_LIO_LIStrain-specific GSMMs used for comparative analyses (deposited in Figshare - 10.6084/m9.figshare.24871914), plus their associated MEMOTE reports (indicates completeness and annotation quality), reaction and gene presence-absence matrices across all isolates.
C_LIO_LIGrowth phenotype predictions derived from strain-specific GSMMs are available in Table S4.
C_LIO_LIBinarised Biolog growth phenotype data for n=37 isolates (plates PM1 and PM2, aerobic and anaerobic conditions) are available in Tables S6 &amp; S7.
C_LIO_LIAdditional growth assay data for six substrates not included on Biolog plates PM1 and PM2 (deposited in Figshare - 10.6084/m9.figshare.24871914).
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.10.575041">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.10.575041" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.10.575041">
        <p class="paperTitle">Pangenome reconstruction in rats enhances genotype-phenotype mapping and novel variant discovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.10.575041" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.10.575041" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Villani, F.; Guarracino, A.; Ward, R. R.; Green, T.; Emms, M.; Pravenec, M.; Prins, P.; Garrison, E.; Williams, R. W.; Chen, H.; Colonna, V.</p>
        <p class="info">Score: 5.3, Published: 2024-01-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.10.575041' target='https://doi.org/10.1101/2024.01.10.575041'> 10.1101/2024.01.10.575041</a></p>
        <p class="abstract">The HXB/BXH family of recombinant inbred rat strains is a unique genetic resource that has been extensively phenotyped over 25 years, resulting in a vast dataset of quantitative molecular and physiological phenotypes. We built a pangenome graph from 10x Genomics linked-read data for 31 recombinant inbred rats to study genetic variation and association mapping. The pangenome length was on average 2.4 times greater than the corresponding length of the reference mRatBN7.2, confirming the capture of substantial additional variation. We validated variants in challenging regions, including complex structural variants resolving into multiple haplotypes. Phenome-wide association analysis of validated SNPs uncovered variants associated with glucose/insulin levels and hippocampal gene expression. We propose an interaction between Pirl1l1, Cromogranine expression, TNF- levels, and insulin regulation. This study demonstrates the utility of linked-read pangenomes for comprehensive variant detection and mapping phenotypic diversity in a widely used rat genetic reference panel.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
