<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Natural selection exerted by historical coronavirus epidemic(s): comparative genetic analysis in China Kadoorie Biobank and UK Biobank
Authors: Morris, S. C.; Lin, K.; Millwood, I. Y.; Yu, C.; Lv, J.; Pei, P.; Li, L.; Sun, D.; Davey Smith, G.; Chen, Z.; Walters, R. G.
Score: 23.0, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579075
BackgroundPathogens have been one of the primary sources of natural selection affecting modern humans. The footprints of historical selection events - &#34;selective sweeps&#34; - can be detected in the genomes of present-day individuals.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Natural selection exerted by historical coronavirus epidemic(s): comparative genetic analysis in China Kadoorie Biobank and UK Biobank
Authors: Morris, S. C.; Lin, K.; Millwood, I. Y.; Yu, C.; Lv, J.; Pei, P.; Li, L.; Sun, D.; Davey Smith, G.; Chen, Z.; Walters, R. G.
Score: 23.0, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579075
BackgroundPathogens have been one of the primary sources of natural selection affecting modern humans. The footprints of historical selection events - &#34;selective sweeps&#34; - can be detected in the genomes of present-day individuals." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-11T10:38:55+00:00" />
<meta property="article:modified_time" content="2024-02-11T10:38:55+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Natural selection exerted by historical coronavirus epidemic(s): comparative genetic analysis in China Kadoorie Biobank and UK Biobank
Authors: Morris, S. C.; Lin, K.; Millwood, I. Y.; Yu, C.; Lv, J.; Pei, P.; Li, L.; Sun, D.; Davey Smith, G.; Chen, Z.; Walters, R. G.
Score: 23.0, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579075
BackgroundPathogens have been one of the primary sources of natural selection affecting modern humans. The footprints of historical selection events - &#34;selective sweeps&#34; - can be detected in the genomes of present-day individuals."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Natural selection exerted by historical coronavirus epidemic(s): comparative genetic analysis in China Kadoorie Biobank and UK Biobank\nAuthors: Morris, S. C.; Lin, K.; Millwood, I. Y.; Yu, C.; Lv, J.; Pei, P.; Li, L.; Sun, D.; Davey Smith, G.; Chen, Z.; Walters, R. G.\nScore: 23.0, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579075\nBackgroundPathogens have been one of the primary sources of natural selection affecting modern humans. The footprints of historical selection events - \u0026#34;selective sweeps\u0026#34; - can be detected in the genomes of present-day individuals.",
  "keywords": [
    
  ],
  "articleBody": " Natural selection exerted by historical coronavirus epidemic(s): comparative genetic analysis in China Kadoorie Biobank and UK Biobank\nAuthors: Morris, S. C.; Lin, K.; Millwood, I. Y.; Yu, C.; Lv, J.; Pei, P.; Li, L.; Sun, D.; Davey Smith, G.; Chen, Z.; Walters, R. G.\nScore: 23.0, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579075\nBackgroundPathogens have been one of the primary sources of natural selection affecting modern humans. The footprints of historical selection events - \"selective sweeps\" - can be detected in the genomes of present-day individuals. Previous analyses of 629 samples from the 1000 Genomes Project suggested that an ancient coronavirus epidemic [~]20,000 years ago drove multiple selective sweeps in the ancestors of present-day East Asians, but not in other worldwide populations. ResultsUsing a much larger genetic dataset of 76,719 unrelated individuals from each of the China Kadoorie Biobank (CKB) and UK Biobank (UKB) to identify regions of long-range linkage disequilibrium, we further investigated signatures of past selective sweeps and how they reflect previous viral epidemics. Using independently-curated lists of human host proteins which interact physically or functionally with viruses (virus-interacting proteins; VIPs), we found enrichment in CKB for regions of long-range linkage disequilibrium at genes encoding VIPs for coronaviruses, but not DNA viruses. By contrast, we found no clear evidence for any VIP enrichment in UKB. These findings were supported by additional analyses using saltiLASSi, a selection-scan method robust to false positives caused by demographic events. By contrast, for GWAS signals for SARS-Cov2 susceptibility (critical illness, hospitalisation, and reported infection), there was no difference between UKB and CKB in the number located at or near signals of selection, as expected for a novel virus which has had no opportunity to impact the CKB/UKB study populations. ConclusionsTogether, these results provide evidence of selection events consistent with historical coronavirus epidemic(s) originating in East Asia. These results show how biobank-scale datasets and evolutionary genomics theory can provide insight into the study of past epidemics. The results also highlights how historic infectious diseases epidemics can shape the genetic architecture of present-day human populations.\nNanopore-based DNA long-read sequencing analysis of the aged human brain\nAuthors: Ramirez, P.; Sun, W.; Kazempour Dehkordi, S.; Zare, H.; Fongang, B.; Bieniek, K. F.; Frost, B.\nScore: 21.2, Published: 2024-02-05 DOI: 10.1101/2024.02.01.578450\nAging disrupts cellular processes such as DNA repair and epigenetic control, leading to a gradual buildup of genomic alterations that can have detrimental effects in post-mitotic cells. Genomic alterations in regions of the genome that are rich in repetitive sequences, often termed \"dark loci,\" are difficult to resolve using traditional sequencing approaches. New long-read technologies offer promising avenues for exploration of previously inaccessible regions of the genome. Using nanopore-based long-read whole-genome sequencing of DNA extracted from aged 18 human brains, we identify previously unreported structural variants and methylation patterns within repetitive DNA, focusing on transposable elements (\"jumping genes\") as crucial sources of variation, particularly in dark loci. Our analyses reveal potential somatic insertion variants and provides DNA methylation frequencies for many retrotransposon families. We further demonstrate the utility of this technology for the study of these challenging genomic regions in brains affected by Alzheimers disease and identify significant differences in DNA methylation in pathologically normal brains versus those affected by Alzheimers disease. Highlighting the power of this approach, we discover specific polymorphic retrotransposons with altered DNA methylation patterns. These retrotransposon loci have the potential to contribute to pathology, warranting further investigation in Alzheimers disease research. Taken together, our study provides the first long-read DNA sequencing-based analysis of retrotransposon sequences, structural variants, and DNA methylation in the aging brain affected with Alzheimers disease neuropathology.\nA complex Plasmodium falciparum cryptotype circulating at low frequency across the African continent\nAuthors: Miotto, O.; Amambua-Ngwa, A.; Amenga-Etego, L. N.; Abdel Hamid, M. M.; Adam, I.; Aninagyei, E.; Apinjoh, T.; Awandare, G. A.; Bejon, P.; Bertin, G.; Bouyou-Akotet, M. K.; Claessens, A.; Conway, D.; D'Alessandro, U.; Diakite, M.; Djimde, A.; Dondorp, A. M.; Duffy, P.; Fairhurst, R. M.; Fanello, C. I.; Ghansah, A.; Ishengoma, D.; Lawniczak, M.; Maiga-Ascofare, O.; Auburn, S.; Rosanas-Urgell, A.; Wasakul, V.; White, N.; Almagro-Garcia, J.; Pearson, R.; Goncalves, S.; Ariani, C.; Bozdech, Z.; Hamilton, W. L.; Simpson, V.; Kwiatkowski, D. P.\nScore: 23.6, Published: 2024-01-22 DOI: 10.1101/2024.01.20.576496\nThe population structure of the malaria parasite Plasmodium falciparum can reveal underlying demographic and adaptive evolutionary processes. Here, we analyse population structure in 4,376 P. falciparum genomes from 21 countries across Africa. We identified a strongly differentiated cluster of parasites, comprising [~]1.2% of samples analysed, geographically distributed over 13 countries across the continent. Members of this cluster, named AF1, carry a genetic background consisting of a large number of highly differentiated variants, rarely observed outside this cluster, at a multitude of genomic loci distributed across most chromosomes. At these loci, the AF1 haplotypes appear to have common ancestry, irrespective of the sampling location; outside the shared loci, however, AF1 members are genetically similar to their sympatric parasites. AF1 parasites sharing up to 23 genomic co-inherited regions were found in all major regions of Africa, at locations over 7,000 km apart. We coined the term cryptotype to describe a complex common background which is geographically widespread, but concealed by genomic regions of local origin. Most AF1 differentiated variants are functionally related, comprising structural variations and single nucleotide polymorphisms in components of the MSP1 complex and several other genes involved in interactions with red blood cells, including invasion and erythrocyte antigen export. We propose that AF1 parasites have adapted to some as yet unidentified evolutionary niche, by acquiring a complex compendium of interacting variants that rarely circulate separately in Africa. As the cryptotype spread across the continent, it appears to have been maintained mostly intact in spite of recombination events, suggesting a selective advantage. It is possible that other cryptotypes circulate in Africa, and new analysis methods may be needed to identify them.\nDE Novo emerged gene SEarch in Eukaryotes with DENSE\nAuthors: Roginski, P.; Grandchamp, A.; Quignot, C.; Lopes, A.\nScore: 12.2, Published: 2024-01-31 DOI: 10.1101/2024.01.30.578014\nThe discovery of de novo emerged genes, originating from previously noncoding DNA regions, challenges traditional views of species evolution. Indeed, the hypothesis of neutrally evolving sequences giving rise to functional proteins is highly unlikely. This conundrum has sparked numerous studies to quantify and characterize these genes, aiming to understand their functional roles and contributions to genome evolution. Yet, no fully automated pipeline for their identification is available. Therefore, we introduce DENSE (DE Novo emerged gene Search), an automated Nextflow pipeline based on two distinct steps: detection of Taxonomically Restricted Genes (TRGs) through phylostratigraphy, and filtering of TRGs for de novo emerged genes via genome comparisons and synteny search. DENSE is available as a user-friendly command-line tool, while the second step is accessible through a web server upon providing a list of TRGs. Highly flexible, DENSE provides various strategy and parameter combinations, enabling users to adapt to specific configurations or define their own strategy through a rational framework, facilitating protocol communication, and study interoperability. We apply DENSE to seven model organisms, exploring the impact of its strategies and parameters on de novo gene predictions. This thorough analysis across species with different evolutionary rates reveals useful metrics for users to define input datasets, identify favorable/unfavorable conditions for de novo gene detection, and control potential biases in genome annotations. Additionally, predictions made for the seven model organisms are compiled into a requestable database, that we hope will serve as a reference for de novo emerged gene lists generated with specific criteria combinations. Significance StatementThe identification and classification of de novo genes, which originate from noncoding regions of DNA, remain an ongoing challenge in genomics research. While various approaches have been employed for their identification, the lack of a standardized protocol has resulted in varying lists of de novo genes across studies. This study introduces a novel tool: DENSE, that formalizes the common practices used in the field into a comprehensive and automated pipeline. DENSE streamlines the identification of taxonomically restricted genes, homology searches, and synteny analysis. This standardized methodology aims to enhance the accuracy and reliability of de novo gene identification, fostering a deeper understanding of the evolutionary mechanisms that drive gene birth and shape the genetic diversity of organisms.\nQuantitative omnigenic model discovers interpretable genome-wide associations\nAuthors: Ruzickova, N.; Hledik, M.; Tkacik, G.\nScore: 11.6, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578486\nAs their statistical power grows, genome-wide association studies (GWAS) have identified an increasing number of loci underlying quantitative traits of interest. These loci are scattered throughout the genome and are individually responsible only for small fractions of the total heritable trait variance. The recently proposed omnigenic model provides a conceptual framework to explain these observations by postulating that numerous distant loci contribute to each complex trait via effect propagation through intracellular regulatory networks. We formalize this conceptual framework by proposing the \"quantitative omnigenic model\" (QOM), a statistical model that combines prior knowledge of the regulatory network topology with genomic data. By applying our model to gene expression traits in yeast, we demonstrate that QOM achieves similar gene expression prediction performance to traditional GWAS with hundreds of times less parameters, while simultaneously extracting candidate causal and quantitative chains of effect propagation through the regulatory network for every individual gene. We estimate the fraction of heritable trait variance in cis- and in trans-, break the latter down by effect propagation order, assess the trans- variance not attributable to transcriptional regulation, and show that QOM correctly accounts for the low-dimensional structure of gene expression covariance. We furthermore demonstrate the relevance of QOM for systems biology, by employing it as a statistical test for the quality of regulatory network reconstructions, and linking it to the propagation of non-genetic, environmental effects. Significance statementGenetic variation leads to differences in traits implicated in health and disease. Identifying genetic variants associated with these traits is spearheaded by \"genome-wide association studies\" (GWAS) - statistically rigorous procedures whose power has grown with the number of genotyped samples. Nevertheless, GWAS have a substantial shortcoming: they are ill-equipped to detect the causal basis and reveal the complex systemic mechanisms of polygenic traits. Even a single genetic change can propagate throughout the entire genetic regulatory network causing a myriad of spurious detections, thereby significantly limiting GWAS usefulness. We tackle this challenge with a novel statistical approach that incorporates known regulatory network information to substantially boost the interpretability of state-of-the-art genomic analyses while simultaneously extracting novel systems biology insights.\nTIPP_plastid: A user-friendly tool for de novo assembly of plastid genomes\nAuthors: Xian, W.; Weigel, D.\nScore: 11.8, Published: 2024-01-31 DOI: 10.1101/2024.01.29.577798\nMotivationThe chloroplast is the primary site of photosynthesis in plants, converting solar energy into chemical energy. The chloroplast genome is typically a circular DNA composed of four sections: a large single copy (LSC), a small single copy (SSC), and two intervening repeats (IRs). Long-read sequencing data now allow for assembly of chloroplast genomes along with nuclear genomes, but this requires that reads originating from the chloroplast genome are distinguished from reads that represent nuclear insertions of chloroplast DNA or related sequences from the mitochondrial genomes and that chloroplast heteroplasmy, i.e., the presence of multiple distinct chloroplast genomes is taken into account. ResultsWe introduce TIPP_plastid, an efficient tool for the assembly of plastid genomes based on the exclusion of reads originating from the mitochondrial and the nuclear genome, followed by the construction of assembly graphs and extraction of distinct chloroplast genomes. We demonstrate the usefulness of TIPP_plastid by assembling the chloroplast genomes of 45 phylogenetically diverse species. Availability and ImplementationTIPP_plastid is available at Github (https://github.com/Wenfei-Xian/TIPP). Code to reproduce results from this paper can be found at Github (https://github.com/Wenfei-Xian/Reproducible_for_TIPP_paper). Chloroplast genomes and assembly graphs of 45 phylogenetically diverse species can be downloaded at figshare (https://doi.org/10.6084/m9.figshare.24715017.v1).\nNAVIP: Unraveling the Influence of Neighboring Small Sequence Variants on Functional Impact Prediction\nAuthors: Baasner, J.-S.; Rempel, A.; Howard, D.; Pucker, B.\nScore: 50.6, Published: 2024-02-07 DOI: 10.1101/596718\nOnce a suitable reference sequence has been generated, intraspecific variation is often assessed by re-sequencing. Variant calling processes can reveal all differences between strains, accessions, genotypes, or individuals. These variants can be enriched with predictions about their functional implications based on available structural annotations, i.e. gene models. Although these functional impact predictions on a per-variant basis are often accurate, some challenging cases require the simultaneous incorporation of multiple adjacent variants into this prediction process. Examples include neighboring variants which modify each others functional impact. The Neighborhood-Aware Variant Impact Predictor (NAVIP) considers all variants within a given protein coding sequence when predicting the functional consequences. As a proof of concept, variants between the Arabidopsis thaliana accessions Columbia-0 and Niederzenz-1 were annotated. NAVIP is freely available on GitHub (https://github.com/bpucker/NAVIP) and accessible through a web server (https://pbb-tools.de). Author SummaryIntraspecific variation gains increasing relevance as reference genome sequences are available for many investigated (plant) species. Understanding the functional consequences of sequence variants between individuals of a population is a challenge. SnpEff (Cigolnai et al., 2012) is the current standard tool to predict the functional impact of sequence variants, but does only consider one sequence variant at the time. We developed NAVIP to properly handle cases in which multiple sequence variants are clustering together and influence each others functional impact. A comparison of two Arabidopsis thaliana accessions demonstrates the relevance of considering multiple sequence variants simultaneously for the prediction of changes in encoded proteins. NAVIP is universally applicable to non-model organisms. All underlying code is freely available from GitHub and we operate a web server for users convenience.\nIntegrative analysis of Hydra head regeneration reveals actions of conserved signaling pathways and transposable elements\nAuthors: Macias-Munoz, A.; Mesrop, L. Y.; Liang, H. Y.; Mortazavi, A.\nScore: 16.1, Published: 2024-01-15 DOI: 10.1101/2022.05.17.492329\nThe extent to which animals can regenerate cells, tissues, or body parts varies largely. Hydra has a remarkable ability to undergo full body regeneration. Bisected polyps can regenerate the head and foot, and whole polyps can form from aggregates of cells. This capability is made possible by a cluster of cells known as the head organizer. Previous studies have found Wnt3 and other developmental genes associated with head organizer function. Yet, the genetic and molecular mechanisms of regenerating Hydra heads remain unresolved. In this study, we used bulk RNA-seq, bulk ATAC-seq, and single-cell RNA-seq from a regeneration time course to characterize the gene expression, gene regulation, molecular, and cellular features of Hydra head regeneration. WGCNA modules and cis-regulation of candidate head organizer genes identified genes co-expressed with Wnt3 and transcription factors important for regeneration. At the single-cell level, we identified at least six distinct cell types in the regenerating head tissue and characterized the expression of candidate genes. With these combined data sets, we identify and clarify some of the interactions between JNK, Wnt3, and EGRF signaling during Hydra head regeneration. Our study reveals coordination of early wound response, developmental transcription factors, and transposable elements during Hydra tissue regeneration and provides insight into the evolution of regeneration programs.\nCell Cycle-Dependent TICRR/TRESLIN and MTBP Chromatin Binding Mechanisms and Patterns\nAuthors: Noble, T. D.; Sansam, C. G.; Wittig, K. A.; Majchrzycka, B.; Sansam, C. L.\nScore: 4.3, Published: 2024-02-07 DOI: 10.1101/2024.02.02.578516\nThe selection of replication origins is a defining characteristic of DNA replication in eukaryotes, yet its mechanism in humans has not been well-defined. In this study, we use Cut\u0026Run to examine genomic binding locations for TICRR/TRESLIN and MTBP, the human orthologs for the yeast DNA replication initiation factors Sld3 and Sld7. We mapped TRESLIN and MTBP binding in HCT116 colorectal cancer cells using asynchronous and G1 synchronized populations. Our data show that TRESLIN and MTBP binding patterns are more defined in a G1 synchronized population compared to asynchronously cycling cells. We also examined whether TRESLIN and MTBP are dependent on one another for binding. Our data suggest MTBP is dependent on TRESLIN for proper association with chromatin during G1 but not S phase. Finally, we asked whether TRESLIN and MTBP binding to chromatin requires licensed origins. Using cell lines with a non-degradable inducible Geminin to inhibit licensing, we show TRESLIN and MTBP binding does not require loaded MCMs. Altogether, our Cut\u0026Run data provides evidence for a chromatin binding mechanism of TRESLIN-MTBP during G1 that is dependent on TRESLIN and does not require interactions with licensed origins.\nWhole Genome Sequencing of DENV-2 isolated from Aedes aegypti mosquitoes in Esmeraldas, Ecuador. Genomic epidemiology of genotype III Southern Asian-American in the country.\nAuthors: Carrazco-Montalvo, A.; Gutierrez-Pallo, D.; Arevalo, V.; Ponce, P.; Rodriguez-Polit, C.; Alarcon, D.; Echeverria-Garces, G.; Coloma, J.; Nipaz, V.; Cevallos, V.\nScore: 4.1, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579255\nEcuador is a tropical country reporting Dengue virus (DENV) outbreaks, with areas of hyperendemic viral transmission. Entomo-virological surveillance and monitoring effort conducted in the Northwestern border province of Esmeraldas in April 2022, five pools of female Aedes aegypti mosquitoes from a rural community tested positive for DENV serotype 2 by RT-qPCR. One pool was sequenced by Illumina MiSeq, and it corresponded to genotype III Southern Asian-American. Comparison with other genomes revealed genetic similarity to a human DENV genome sequenced in 2021, also from Esmeraldas. Potential introduction events to the country could have originated from Colombia, considering the vicinity of the collection sites to the neighboring country and high human movement. The inclusion of genomic information complements entomo-virological surveillance, providing valuable insights into genetic variants. This contribution enhances our understanding of Dengue virus (DENV) epidemiology in rural areas and guides evidence-based decisions for surveillance and interventions.\n",
  "wordCount" : "2880",
  "inLanguage": "en",
  "datePublished": "2024-02-11T10:38:55Z",
  "dateModified": "2024-02-11T10:38:55Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 11, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579075">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579075" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579075">
        <p class="paperTitle">Natural selection exerted by historical coronavirus epidemic(s): comparative genetic analysis in China Kadoorie Biobank and UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579075" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579075" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Morris, S. C.; Lin, K.; Millwood, I. Y.; Yu, C.; Lv, J.; Pei, P.; Li, L.; Sun, D.; Davey Smith, G.; Chen, Z.; Walters, R. G.</p>
        <p class="info">Score: 23.0, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579075' target='https://doi.org/10.1101/2024.02.06.579075'> 10.1101/2024.02.06.579075</a></p>
        <p class="abstract">BackgroundPathogens have been one of the primary sources of natural selection affecting modern humans. The footprints of historical selection events - &#34;selective sweeps&#34; - can be detected in the genomes of present-day individuals. Previous analyses of 629 samples from the 1000 Genomes Project suggested that an ancient coronavirus epidemic [~]20,000 years ago drove multiple selective sweeps in the ancestors of present-day East Asians, but not in other worldwide populations.

ResultsUsing a much larger genetic dataset of 76,719 unrelated individuals from each of the China Kadoorie Biobank (CKB) and UK Biobank (UKB) to identify regions of long-range linkage disequilibrium, we further investigated signatures of past selective sweeps and how they reflect previous viral epidemics. Using independently-curated lists of human host proteins which interact physically or functionally with viruses (virus-interacting proteins; VIPs), we found enrichment in CKB for regions of long-range linkage disequilibrium at genes encoding VIPs for coronaviruses, but not DNA viruses. By contrast, we found no clear evidence for any VIP enrichment in UKB. These findings were supported by additional analyses using saltiLASSi, a selection-scan method robust to false positives caused by demographic events. By contrast, for GWAS signals for SARS-Cov2 susceptibility (critical illness, hospitalisation, and reported infection), there was no difference between UKB and CKB in the number located at or near signals of selection, as expected for a novel virus which has had no opportunity to impact the CKB/UKB study populations.

ConclusionsTogether, these results provide evidence of selection events consistent with historical coronavirus epidemic(s) originating in East Asia. These results show how biobank-scale datasets and evolutionary genomics theory can provide insight into the study of past epidemics. The results also highlights how historic infectious diseases epidemics can shape the genetic architecture of present-day human populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578450">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578450" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578450">
        <p class="paperTitle">Nanopore-based DNA long-read sequencing analysis of the aged human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578450" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578450" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramirez, P.; Sun, W.; Kazempour Dehkordi, S.; Zare, H.; Fongang, B.; Bieniek, K. F.; Frost, B.</p>
        <p class="info">Score: 21.2, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578450' target='https://doi.org/10.1101/2024.02.01.578450'> 10.1101/2024.02.01.578450</a></p>
        <p class="abstract">Aging disrupts cellular processes such as DNA repair and epigenetic control, leading to a gradual buildup of genomic alterations that can have detrimental effects in post-mitotic cells. Genomic alterations in regions of the genome that are rich in repetitive sequences, often termed &#34;dark loci,&#34; are difficult to resolve using traditional sequencing approaches. New long-read technologies offer promising avenues for exploration of previously inaccessible regions of the genome. Using nanopore-based long-read whole-genome sequencing of DNA extracted from aged 18 human brains, we identify previously unreported structural variants and methylation patterns within repetitive DNA, focusing on transposable elements (&#34;jumping genes&#34;) as crucial sources of variation, particularly in dark loci. Our analyses reveal potential somatic insertion variants and provides DNA methylation frequencies for many retrotransposon families. We further demonstrate the utility of this technology for the study of these challenging genomic regions in brains affected by Alzheimers disease and identify significant differences in DNA methylation in pathologically normal brains versus those affected by Alzheimers disease. Highlighting the power of this approach, we discover specific polymorphic retrotransposons with altered DNA methylation patterns. These retrotransposon loci have the potential to contribute to pathology, warranting further investigation in Alzheimers disease research. Taken together, our study provides the first long-read DNA sequencing-based analysis of retrotransposon sequences, structural variants, and DNA methylation in the aging brain affected with Alzheimers disease neuropathology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576496" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576496">
        <p class="paperTitle">A complex Plasmodium falciparum cryptotype circulating at low frequency across the African continent</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miotto, O.; Amambua-Ngwa, A.; Amenga-Etego, L. N.; Abdel Hamid, M. M.; Adam, I.; Aninagyei, E.; Apinjoh, T.; Awandare, G. A.; Bejon, P.; Bertin, G.; Bouyou-Akotet, M. K.; Claessens, A.; Conway, D.; D&#39;Alessandro, U.; Diakite, M.; Djimde, A.; Dondorp, A. M.; Duffy, P.; Fairhurst, R. M.; Fanello, C. I.; Ghansah, A.; Ishengoma, D.; Lawniczak, M.; Maiga-Ascofare, O.; Auburn, S.; Rosanas-Urgell, A.; Wasakul, V.; White, N.; Almagro-Garcia, J.; Pearson, R.; Goncalves, S.; Ariani, C.; Bozdech, Z.; Hamilton, W. L.; Simpson, V.; Kwiatkowski, D. P.</p>
        <p class="info">Score: 23.6, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576496' target='https://doi.org/10.1101/2024.01.20.576496'> 10.1101/2024.01.20.576496</a></p>
        <p class="abstract">The population structure of the malaria parasite Plasmodium falciparum can reveal underlying demographic and adaptive evolutionary processes. Here, we analyse population structure in 4,376 P. falciparum genomes from 21 countries across Africa. We identified a strongly differentiated cluster of parasites, comprising [~]1.2% of samples analysed, geographically distributed over 13 countries across the continent. Members of this cluster, named AF1, carry a genetic background consisting of a large number of highly differentiated variants, rarely observed outside this cluster, at a multitude of genomic loci distributed across most chromosomes. At these loci, the AF1 haplotypes appear to have common ancestry, irrespective of the sampling location; outside the shared loci, however, AF1 members are genetically similar to their sympatric parasites. AF1 parasites sharing up to 23 genomic co-inherited regions were found in all major regions of Africa, at locations over 7,000 km apart. We coined the term cryptotype to describe a complex common background which is geographically widespread, but concealed by genomic regions of local origin. Most AF1 differentiated variants are functionally related, comprising structural variations and single nucleotide polymorphisms in components of the MSP1 complex and several other genes involved in interactions with red blood cells, including invasion and erythrocyte antigen export. We propose that AF1 parasites have adapted to some as yet unidentified evolutionary niche, by acquiring a complex compendium of interacting variants that rarely circulate separately in Africa. As the cryptotype spread across the continent, it appears to have been maintained mostly intact in spite of recombination events, suggesting a selective advantage. It is possible that other cryptotypes circulate in Africa, and new analysis methods may be needed to identify them.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.578014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.578014" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.578014">
        <p class="paperTitle">DE Novo emerged gene SEarch in Eukaryotes with DENSE</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.578014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.578014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roginski, P.; Grandchamp, A.; Quignot, C.; Lopes, A.</p>
        <p class="info">Score: 12.2, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.578014' target='https://doi.org/10.1101/2024.01.30.578014'> 10.1101/2024.01.30.578014</a></p>
        <p class="abstract">The discovery of de novo emerged genes, originating from previously noncoding DNA regions, challenges traditional views of species evolution. Indeed, the hypothesis of neutrally evolving sequences giving rise to functional proteins is highly unlikely. This conundrum has sparked numerous studies to quantify and characterize these genes, aiming to understand their functional roles and contributions to genome evolution. Yet, no fully automated pipeline for their identification is available. Therefore, we introduce DENSE (DE Novo emerged gene Search), an automated Nextflow pipeline based on two distinct steps: detection of Taxonomically Restricted Genes (TRGs) through phylostratigraphy, and filtering of TRGs for de novo emerged genes via genome comparisons and synteny search. DENSE is available as a user-friendly command-line tool, while the second step is accessible through a web server upon providing a list of TRGs. Highly flexible, DENSE provides various strategy and parameter combinations, enabling users to adapt to specific configurations or define their own strategy through a rational framework, facilitating protocol communication, and study interoperability. We apply DENSE to seven model organisms, exploring the impact of its strategies and parameters on de novo gene predictions. This thorough analysis across species with different evolutionary rates reveals useful metrics for users to define input datasets, identify favorable/unfavorable conditions for de novo gene detection, and control potential biases in genome annotations. Additionally, predictions made for the seven model organisms are compiled into a requestable database, that we hope will serve as a reference for de novo emerged gene lists generated with specific criteria combinations.

Significance StatementThe identification and classification of de novo genes, which originate from noncoding regions of DNA, remain an ongoing challenge in genomics research. While various approaches have been employed for their identification, the lack of a standardized protocol has resulted in varying lists of de novo genes across studies. This study introduces a novel tool: DENSE, that formalizes the common practices used in the field into a comprehensive and automated pipeline. DENSE streamlines the identification of taxonomically restricted genes, homology searches, and synteny analysis. This standardized methodology aims to enhance the accuracy and reliability of de novo gene identification, fostering a deeper understanding of the evolutionary mechanisms that drive gene birth and shape the genetic diversity of organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578486">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578486" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578486">
        <p class="paperTitle">Quantitative omnigenic model discovers interpretable genome-wide associations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578486" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578486" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ruzickova, N.; Hledik, M.; Tkacik, G.</p>
        <p class="info">Score: 11.6, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578486' target='https://doi.org/10.1101/2024.02.01.578486'> 10.1101/2024.02.01.578486</a></p>
        <p class="abstract">As their statistical power grows, genome-wide association studies (GWAS) have identified an increasing number of loci underlying quantitative traits of interest. These loci are scattered throughout the genome and are individually responsible only for small fractions of the total heritable trait variance. The recently proposed omnigenic model provides a conceptual framework to explain these observations by postulating that numerous distant loci contribute to each complex trait via effect propagation through intracellular regulatory networks. We formalize this conceptual framework by proposing the &#34;quantitative omnigenic model&#34; (QOM), a statistical model that combines prior knowledge of the regulatory network topology with genomic data. By applying our model to gene expression traits in yeast, we demonstrate that QOM achieves similar gene expression prediction performance to traditional GWAS with hundreds of times less parameters, while simultaneously extracting candidate causal and quantitative chains of effect propagation through the regulatory network for every individual gene. We estimate the fraction of heritable trait variance in cis- and in trans-, break the latter down by effect propagation order, assess the trans- variance not attributable to transcriptional regulation, and show that QOM correctly accounts for the low-dimensional structure of gene expression covariance. We furthermore demonstrate the relevance of QOM for systems biology, by employing it as a statistical test for the quality of regulatory network reconstructions, and linking it to the propagation of non-genetic, environmental effects.

Significance statementGenetic variation leads to differences in traits implicated in health and disease. Identifying genetic variants associated with these traits is spearheaded by &#34;genome-wide association studies&#34; (GWAS) - statistically rigorous procedures whose power has grown with the number of genotyped samples. Nevertheless, GWAS have a substantial shortcoming: they are ill-equipped to detect the causal basis and reveal the complex systemic mechanisms of polygenic traits. Even a single genetic change can propagate throughout the entire genetic regulatory network causing a myriad of spurious detections, thereby significantly limiting GWAS usefulness. We tackle this challenge with a novel statistical approach that incorporates known regulatory network information to substantially boost the interpretability of state-of-the-art genomic analyses while simultaneously extracting novel systems biology insights.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.577798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.577798" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.577798">
        <p class="paperTitle">TIPP_plastid: A user-friendly tool for de novo assembly of plastid genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.577798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.577798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xian, W.; Weigel, D.</p>
        <p class="info">Score: 11.8, Published: 2024-01-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.577798' target='https://doi.org/10.1101/2024.01.29.577798'> 10.1101/2024.01.29.577798</a></p>
        <p class="abstract">MotivationThe chloroplast is the primary site of photosynthesis in plants, converting solar energy into chemical energy. The chloroplast genome is typically a circular DNA composed of four sections: a large single copy (LSC), a small single copy (SSC), and two intervening repeats (IRs). Long-read sequencing data now allow for assembly of chloroplast genomes along with nuclear genomes, but this requires that reads originating from the chloroplast genome are distinguished from reads that represent nuclear insertions of chloroplast DNA or related sequences from the mitochondrial genomes and that chloroplast heteroplasmy, i.e., the presence of multiple distinct chloroplast genomes is taken into account.

ResultsWe introduce TIPP_plastid, an efficient tool for the assembly of plastid genomes based on the exclusion of reads originating from the mitochondrial and the nuclear genome, followed by the construction of assembly graphs and extraction of distinct chloroplast genomes. We demonstrate the usefulness of TIPP_plastid by assembling the chloroplast genomes of 45 phylogenetically diverse species.

Availability and ImplementationTIPP_plastid is available at Github (https://github.com/Wenfei-Xian/TIPP). Code to reproduce results from this paper can be found at Github (https://github.com/Wenfei-Xian/Reproducible_for_TIPP_paper). Chloroplast genomes and assembly graphs of 45 phylogenetically diverse species can be downloaded at figshare (https://doi.org/10.6084/m9.figshare.24715017.v1).</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/596718">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/596718" aria-expanded="false" aria-controls="flush-collapse10.1101/596718">
        <p class="paperTitle">NAVIP: Unraveling the Influence of Neighboring Small Sequence Variants on Functional Impact Prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/596718" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/596718" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baasner, J.-S.; Rempel, A.; Howard, D.; Pucker, B.</p>
        <p class="info">Score: 50.6, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/596718' target='https://doi.org/10.1101/596718'> 10.1101/596718</a></p>
        <p class="abstract">Once a suitable reference sequence has been generated, intraspecific variation is often assessed by re-sequencing. Variant calling processes can reveal all differences between strains, accessions, genotypes, or individuals. These variants can be enriched with predictions about their functional implications based on available structural annotations, i.e. gene models. Although these functional impact predictions on a per-variant basis are often accurate, some challenging cases require the simultaneous incorporation of multiple adjacent variants into this prediction process. Examples include neighboring variants which modify each others functional impact. The Neighborhood-Aware Variant Impact Predictor (NAVIP) considers all variants within a given protein coding sequence when predicting the functional consequences. As a proof of concept, variants between the Arabidopsis thaliana accessions Columbia-0 and Niederzenz-1 were annotated. NAVIP is freely available on GitHub (https://github.com/bpucker/NAVIP) and accessible through a web server (https://pbb-tools.de).

Author SummaryIntraspecific variation gains increasing relevance as reference genome sequences are available for many investigated (plant) species. Understanding the functional consequences of sequence variants between individuals of a population is a challenge. SnpEff (Cigolnai et al., 2012) is the current standard tool to predict the functional impact of sequence variants, but does only consider one sequence variant at the time. We developed NAVIP to properly handle cases in which multiple sequence variants are clustering together and influence each others functional impact. A comparison of two Arabidopsis thaliana accessions demonstrates the relevance of considering multiple sequence variants simultaneously for the prediction of changes in encoded proteins. NAVIP is universally applicable to non-model organisms. All underlying code is freely available from GitHub and we operate a web server for users convenience.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.05.17.492329">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.05.17.492329" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.05.17.492329">
        <p class="paperTitle">Integrative analysis of Hydra head regeneration reveals actions of conserved signaling pathways and transposable elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.05.17.492329" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.05.17.492329" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Macias-Munoz, A.; Mesrop, L. Y.; Liang, H. Y.; Mortazavi, A.</p>
        <p class="info">Score: 16.1, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.05.17.492329' target='https://doi.org/10.1101/2022.05.17.492329'> 10.1101/2022.05.17.492329</a></p>
        <p class="abstract">The extent to which animals can regenerate cells, tissues, or body parts varies largely. Hydra has a remarkable ability to undergo full body regeneration. Bisected polyps can regenerate the head and foot, and whole polyps can form from aggregates of cells. This capability is made possible by a cluster of cells known as the head organizer. Previous studies have found Wnt3 and other developmental genes associated with head organizer function. Yet, the genetic and molecular mechanisms of regenerating Hydra heads remain unresolved. In this study, we used bulk RNA-seq, bulk ATAC-seq, and single-cell RNA-seq from a regeneration time course to characterize the gene expression, gene regulation, molecular, and cellular features of Hydra head regeneration. WGCNA modules and cis-regulation of candidate head organizer genes identified genes co-expressed with Wnt3 and transcription factors important for regeneration. At the single-cell level, we identified at least six distinct cell types in the regenerating head tissue and characterized the expression of candidate genes. With these combined data sets, we identify and clarify some of the interactions between JNK, Wnt3, and EGRF signaling during Hydra head regeneration. Our study reveals coordination of early wound response, developmental transcription factors, and transposable elements during Hydra tissue regeneration and provides insight into the evolution of regeneration programs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.02.578516">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.02.578516" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.02.578516">
        <p class="paperTitle">Cell Cycle-Dependent TICRR/TRESLIN and MTBP Chromatin Binding Mechanisms and Patterns</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.02.578516" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.02.578516" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Noble, T. D.; Sansam, C. G.; Wittig, K. A.; Majchrzycka, B.; Sansam, C. L.</p>
        <p class="info">Score: 4.3, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.02.578516' target='https://doi.org/10.1101/2024.02.02.578516'> 10.1101/2024.02.02.578516</a></p>
        <p class="abstract">The selection of replication origins is a defining characteristic of DNA replication in eukaryotes, yet its mechanism in humans has not been well-defined. In this study, we use Cut&amp;Run to examine genomic binding locations for TICRR/TRESLIN and MTBP, the human orthologs for the yeast DNA replication initiation factors Sld3 and Sld7. We mapped TRESLIN and MTBP binding in HCT116 colorectal cancer cells using asynchronous and G1 synchronized populations. Our data show that TRESLIN and MTBP binding patterns are more defined in a G1 synchronized population compared to asynchronously cycling cells. We also examined whether TRESLIN and MTBP are dependent on one another for binding. Our data suggest MTBP is dependent on TRESLIN for proper association with chromatin during G1 but not S phase. Finally, we asked whether TRESLIN and MTBP binding to chromatin requires licensed origins. Using cell lines with a non-degradable inducible Geminin to inhibit licensing, we show TRESLIN and MTBP binding does not require loaded MCMs. Altogether, our Cut&amp;Run data provides evidence for a chromatin binding mechanism of TRESLIN-MTBP during G1 that is dependent on TRESLIN and does not require interactions with licensed origins.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579255">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579255" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579255">
        <p class="paperTitle">Whole Genome Sequencing of DENV-2 isolated from Aedes aegypti mosquitoes in Esmeraldas, Ecuador. Genomic epidemiology of genotype III Southern Asian-American in the country.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579255" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579255" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carrazco-Montalvo, A.; Gutierrez-Pallo, D.; Arevalo, V.; Ponce, P.; Rodriguez-Polit, C.; Alarcon, D.; Echeverria-Garces, G.; Coloma, J.; Nipaz, V.; Cevallos, V.</p>
        <p class="info">Score: 4.1, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579255' target='https://doi.org/10.1101/2024.02.06.579255'> 10.1101/2024.02.06.579255</a></p>
        <p class="abstract">Ecuador is a tropical country reporting Dengue virus (DENV) outbreaks, with areas of hyperendemic viral transmission. Entomo-virological surveillance and monitoring effort conducted in the Northwestern border province of Esmeraldas in April 2022, five pools of female Aedes aegypti mosquitoes from a rural community tested positive for DENV serotype 2 by RT-qPCR. One pool was sequenced by Illumina MiSeq, and it corresponded to genotype III Southern Asian-American. Comparison with other genomes revealed genetic similarity to a human DENV genome sequenced in 2021, also from Esmeraldas. Potential introduction events to the country could have originated from Colombia, considering the vicinity of the collection sites to the neighboring country and high human movement. The inclusion of genomic information complements entomo-virological surveillance, providing valuable insights into genetic variants. This contribution enhances our understanding of Dengue virus (DENV) epidemiology in rural areas and guides evidence-based decisions for surveillance and interventions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
