<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Programmable multi-kilobase RNA editing using CRISPR-mediated trans-splicing
Authors: Borrajo, J.; Javanmardi, K.; Griffin, J.; St. Martin, S. J.; Yao, D.; Hill, K.; Blainey, P.; Al-Shayeb, B.
Score: 102.2, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553620
Current gene editing approaches in eukaryotic cells are limited to single base edits or small DNA insertions and deletions, and remain encumbered by unintended permanent effects and significant challenges in the delivery of large DNA cargo. Here we describe Splice Editing, a generalizable platform to correct gene transcripts in situ by programmable insertion or replacement of large RNA segments.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Programmable multi-kilobase RNA editing using CRISPR-mediated trans-splicing
Authors: Borrajo, J.; Javanmardi, K.; Griffin, J.; St. Martin, S. J.; Yao, D.; Hill, K.; Blainey, P.; Al-Shayeb, B.
Score: 102.2, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553620
Current gene editing approaches in eukaryotic cells are limited to single base edits or small DNA insertions and deletions, and remain encumbered by unintended permanent effects and significant challenges in the delivery of large DNA cargo. Here we describe Splice Editing, a generalizable platform to correct gene transcripts in situ by programmable insertion or replacement of large RNA segments." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Programmable multi-kilobase RNA editing using CRISPR-mediated trans-splicing
Authors: Borrajo, J.; Javanmardi, K.; Griffin, J.; St. Martin, S. J.; Yao, D.; Hill, K.; Blainey, P.; Al-Shayeb, B.
Score: 102.2, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553620
Current gene editing approaches in eukaryotic cells are limited to single base edits or small DNA insertions and deletions, and remain encumbered by unintended permanent effects and significant challenges in the delivery of large DNA cargo. Here we describe Splice Editing, a generalizable platform to correct gene transcripts in situ by programmable insertion or replacement of large RNA segments."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Programmable multi-kilobase RNA editing using CRISPR-mediated trans-splicing\nAuthors: Borrajo, J.; Javanmardi, K.; Griffin, J.; St. Martin, S. J.; Yao, D.; Hill, K.; Blainey, P.; Al-Shayeb, B.\nScore: 102.2, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553620\nCurrent gene editing approaches in eukaryotic cells are limited to single base edits or small DNA insertions and deletions, and remain encumbered by unintended permanent effects and significant challenges in the delivery of large DNA cargo. Here we describe Splice Editing, a generalizable platform to correct gene transcripts in situ by programmable insertion or replacement of large RNA segments.",
  "keywords": [
    
  ],
  "articleBody": " Programmable multi-kilobase RNA editing using CRISPR-mediated trans-splicing\nAuthors: Borrajo, J.; Javanmardi, K.; Griffin, J.; St. Martin, S. J.; Yao, D.; Hill, K.; Blainey, P.; Al-Shayeb, B.\nScore: 102.2, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553620\nCurrent gene editing approaches in eukaryotic cells are limited to single base edits or small DNA insertions and deletions, and remain encumbered by unintended permanent effects and significant challenges in the delivery of large DNA cargo. Here we describe Splice Editing, a generalizable platform to correct gene transcripts in situ by programmable insertion or replacement of large RNA segments. By combining CRISPR-mediated RNA targeting with endogenous cellular RNA-splicing machinery, Splice Editing enables efficient, precise, and programmable large-scale editing of gene targets without DNA cleavage or mutagenesis. RNA sequencing and measurement of spliced protein products confirm that Splice Editing achieves efficient and specific targeted RNA and protein correction. We show that Splice Editors based on novel miniature RNA-targeting CRISPR-Cas systems discovered and characterized in this work can be packaged for effective delivery to human cells and affect different types of edits across multiple targets and cell lines. By editing thousands of bases simultaneously in a single reversible step, Splice Editing could expand the treatable disease population for monogenic diseases with large allelic diversity without the permanent unintended effects of DNA editing. One-sentence summaryCRISPR-guided trans-splicing enables efficient and specific recombination of large RNA molecules in mammalian cells, with broad applications in therapeutic development for genetic diseases and as a research tool for the study of basic RNA biology.\nMapping the Genetic Interaction Network of PARP inhibitor Response\nAuthors: Simpson, D.; Ling, J.; Jing, Y.; Adamson, B.\nScore: 35.7, Published: 2023-08-20 DOI: 10.1101/2023.08.19.553986\nGenetic interactions have long informed our understanding of the coordinated proteins and pathways that respond to DNA damage in mammalian cells, but systematic interrogation of the genetic network underlying that system has yet to be achieved. Towards this goal, we measured 147,153 pairwise interactions among genes implicated in PARP inhibitor (PARPi) response. Evaluating genetic interactions at this scale, with and without exposure to PARPi, revealed hierarchical organization of the pathways and complexes that maintain genome stability during normal growth and defined changes that occur upon accumulation of DNA lesions due to cytotoxic doses of PARPi. We uncovered unexpected relationships among DNA repair genes, including context-specific buffering interactions between the minimally characterized AUNIP and BRCA1-A complex genes. Our work thus establishes a foundation for mapping differential genetic interactions in mammalian cells and provides a comprehensive resource for future studies of DNA repair and PARP inhibitors.\nA genome-wide atlas of human cell morphology\nAuthors: Ramezani, M.; Bauman, J.; Singh, A.; Weisbart, E.; Yong, J.; Lozada, M. E.; Way, G. P.; Kavari, S. L.; Diaz, C.; Haghighi, M.; Perez-Schindler, J.; Batista, T. M.; Claussnitzer, M.; Singh, S.; Cimini, B. A.; Blainey, P. C.; Carpenter, A. E.; Jan, C. H.; Neal, J. T.\nScore: 65.4, Published: 2023-08-07 DOI: 10.1101/2023.08.06.552164\nA key challenge of the modern genomics era is developing data-driven representations of gene function. Here, we present the first unbiased morphology-based genome-wide perturbation atlas in human cells, containing three genome-scale genotype-phenotype maps comprising \u003e20,000 single-gene CRISPR-Cas9-based knockout experiments in \u003e30 million cells. Our optical pooled cell profiling approach (PERISCOPE) combines a de-stainable high-dimensional phenotyping panel (based on Cell Painting1,2) with optical sequencing of molecular barcodes and a scalable open-source analysis pipeline to facilitate massively parallel screening of pooled perturbation libraries. This approach provides high-dimensional phenotypic profiles of individual cells, while simultaneously enabling interrogation of subcellular processes. Our atlas reconstructs known pathways and protein-protein interaction networks, identifies culture media-specific responses to gene knockout, and clusters thousands of human genes by phenotypic similarity. Using this atlas, we identify the poorly-characterized disease-associated transmembrane protein TMEM251/LYSET as a Golgi-resident protein essential for mannose-6-phosphate-dependent trafficking of lysosomal enzymes, showing the power of these representations. In sum, our atlas and screening technology represent a rich and accessible resource for connecting genes to cellular functions at scale.\nEnvironmental challenge rewires functional connections among human genes\nAuthors: Herken, B. W.; Wong, G.; Norman, T.; Gilbert, L.\nScore: 23.7, Published: 2023-08-09 DOI: 10.1101/2023.08.09.552346\nA fundamental question in biology is how a limited number of genes combinatorially govern cellular responses to environmental changes. While the prevailing hypothesis is that relationships between genes, processes, and ontologies could be plastic to achieve this adaptability, quantitatively comparing human gene functional connections between specific environmental conditions at scale is very challenging. Therefore, it remains unclear whether and how human genetic interaction networks are rewired in response to changing environmental conditions. Here, we developed a framework for mapping context-specific genetic interactions, enabling us to measure the plasticity of human genetic architecture upon environmental challenge for [~]250,000 interactions, using cell cycle interruption, genotoxic perturbation, and nutrient deprivation as archetypes. We discover large-scale rewiring of human gene relationships across conditions, highlighted by dramatic shifts in the functional connections of epigenetic regulators (TIP60), cell cycle regulators (PP2A), and glycolysis metabolism. Our study demonstrates that upon environmental perturbation, intra-complex genetic rewiring is rare while inter-complex rewiring is common, suggesting a modular and flexible evolutionary genetic strategy that allows a limited number of human genes to enable adaptation to a large number of environmental conditions. One Sentence SummaryFive human genetic interaction maps reveal how the landscape of genes functional relationships is rewired as cells experience environmental stress to DNA integrity, cell cycle regulation, and metabolism.\n16S rRNA phylogeny and clustering is not a reliable proxy for genome-based taxonomy in Streptomyces\nAuthors: Kiepas, A. B.; Hoskisson, P. A.; Pritchard, L.\nScore: 16.3, Published: 2023-08-15 DOI: 10.1101/2023.08.15.553377\nAlthough Streptomyces is one of the most extensively studied genera of bacteria, their taxonomy remains contested and is suspected to contain significant species-level misclassification. Resolving the classification of Streptomyces would benefit many areas of study and applied microbiology that rely heavily on having an accurate ground truth classification of similar and dissimilar organisms, including comparative genomics-based searches for novel antimicrobials in the fight against the ongoing antimicrobial resistance (AMR) crisis. To attempt a resolution, we investigate taxonomic conflicts between 16S rRNA and whole genome classifications using all available 48,981 full-length 16S rRNA Streptomyces sequences from the combined SILVA, Greengenes, Ribosomal Database Project (RDP) and NCBI (National Center for Biotechnology Information) databases, and 2,276 publicly available Streptomyces genome assemblies. We construct a 16S gene tree for 14,239 distinct Streptomyces 16S rRNA sequences, identifying three major lineages of Streptomyces, and find that existing taxonomic classifications are inconsistent with the tree topology. We also use these data to delineate 16S and whole genome landscapes for Streptomyces, finding that 16S and whole-genome classifications of Streptomyces strains are frequently in disagreement, and in particular that 16S zero-radius Operational Taxonomic Units (zOTUs) are often inconsistent with Average Nucleotide Identity (ANI)-based taxonomy. Our results strongly imply that 16S rRNA sequence data does not map to taxonomy sufficiently well to delineate Streptomyces species reliably, and we propose that alternative markers should instead be adopted by the community for classification and metabarcoding. As much of current Streptomyces taxonomy has been determined or supported by historical 16S sequence data and may in parts be in error, we also propose that reclassification of the genus by alternative approaches is required. Impact StatementAccurate classification of microbes, usually in the form of taxonomic assignments, provides a fundamental ground truth or reference point for many aspects of applied microbiology including comparative genomics, identification of strains for natural product discovery, and dereplication of strains. Bacteria belonging to the genus Streptomyces are an important source of bioactive metabolites and enzymes in biotechnology, and proper understanding of their phylogeny aids understanding of the evolution of industrially important gene products and metabolites, and prioritization of strains for industrial exploitation. Taxonomic classification in the genus Streptomyces is complex and contested, and there are clear conflicts between taxonomies inferred from 16S rRNA and from whole genome sequences. Despite this, 16S sequence-based classifications are still widely used to infer taxonomic identity, to determine community composition, and to prioritise strains for study. We investigate a diverse and comprehensive set of Streptomyces genomes using whole-genome Average Nucleotide Identity (ANI) and 16S sequence analysis to delineate and compare classifications made using these approaches. We outline the genomic and 16S sequence landscape of Streptomyces, demonstrating that (i) distinct taxonomic species may share identical full-length 16S sequences, and (ii) in some instances, isolates representing the same taxonomic species do not share any common 16S rRNA sequence. Our results strongly imply that 16S rRNA sequence variation does not map to taxonomy sufficiently well to delineate Streptomyces species reliably, and that alternative markers should instead be adopted by the community. Much of current Streptomyces taxonomy has been determined or supported by historical 16S sequence data, and we therefore propose that reclassification within this group by alternative approaches is required. Data summaryAll code, raw and supporting data are publicly available from GitHub (https://github.com/kiepczi/Kiepas_et_al_2023_16S) and Zenodo (https://doi.org/10.5281/zenodo.8223787). The flowchart provided in Supplementary File 28 provides an overview of analysis steps and serves as a guide through Supplementary Files generated during reconstruction of the 16S phylogeny. The flowchart in Supplementary File 29 outlines the workflow processes and supplementary materials used for analysis of 16S rRNA sequences from Streptomyces genomes. Supplementary DataSupplementary File 1: Generate figures using Python and R. ZIP file containing all data, Python and R scripts to generate figures for this manuscript. (ZIP 40.9MB) Supplementary File 2: Raw 16S rRNA public databases. Zip file containing four separate txt files with sequence IDs for public 16S rRNA databases used in this manuscript, and an additional txt file with Greengenes sequence taxonomy information, and a Python script used to map taxonomy information to sequences found in Greengenes v13.5. (ZIP 34.8MB) Supplementary File 3: Filtration of 16S rRNA public databases. Zip file containing Python script used for filtration of the raw databases, and generated outputs. (ZIP 7.2MB) Supplementary File 4: Cleaning of the filtrated 16S rRNA local. Zip file containing all bash and Python scripts used to clean the local full-length 16S rRNA local databases by removing redundant and poor quality 16S rRNA sequences. (ZIP 9MB) Supplementary File 5: Sequence Clustering. Zip file containing a bash script used to cluster the full-length cleaned local 16S rRNA Streptomyces local databases at various thresholds, and provides txt files with accessions for representative sequences, and cluster members for each clustering threshold. (ZIP 40.8MB) Supplementary File 6: Analysis of taxonomic composition for each clustering threshold. Zip file containing Python scripts, NCBI taxonomy input and all outputs generated used to determine the taxonomic composition for each clustering threshold. (ZIP 49.6) Supplementary File 7. Cluster sizes. Empirical cumulative frequency plot showing cluster sizes generated for all clustering thresholds. (PDF 44KB) Supplementary File 8. Cluster taxID abundance. Empirical cumulative frequency plot for unique numbers of taxID present at all clustering thresholds. (PDF 9KB) Supplementary File 9. MSA. Zip file containing all Python and bash scripts, and additional data needed to generate and clean MSA for phylogenetic analysis. (ZIP 4.2MB) Supplementary File 10. Phylogenetic reconstruction. ZIP file containing bash scripts used for phylogenetic reconstruction, and all generated outputs and log files. (ZIP 16.8MB). Supplementary File 11. Collapse branches. ZIP file containing jupyter notebook used for collapsing branches with the same species names, and the collapsed tree in newick format. (ZIP 385KB) Supplementary File 12. Phylogenetic tree. PDF file showing collapsed phylogenetic tree with marked branches with transfer bootstrap expectation support of \u003e= 50%. (PDF 224KB) Supplementary File 13. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of Streptomyces albus and Streptomyces griseus. (PDF 229KB) Supplementary File 14. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of Streptomyces albulus, Streptomyces lydicus and Streptomyces venezuelae. (PDF 228KB) Supplementary File 15. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of Streptomyces clavuligerus and Streptomyces coelicolor. (PDF 227KB) Supplementary File 16. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of Streptomyces lavendulae, Streptomyces rimosus and Streptomyces scabiei. (PDF 228KB) Supplementary File 17. Streptomyces genomes. Zip file containing bash scripts used to download Streptomyces genomes, and Python scripts used to check assembly status. The ZIP file also contains two separate txt files with Streptomyces genomes used in this manuscript: one file with all initial candidates, and a second file with replaced genomes. (ZIP 2.6MB) Supplementary File 18. Extraction of full-length and ambiguity free 16S rRNA sequences from Streptomyces genomes. Zip file containing all Python and bash scripts used to extract full-length sequences from the filtered Streptomyces genomes. A single FASTA file with all extracted 16S rRNA sequences, and a single FASTA file with filtered sequences. A txt file with accession of genomes retained in the analysis. (ZIP 742KB) Supplementary File 19. ANI analysis among Streptomyces genomes with identical 16S rRNA sequences. ZIP file containing all Bash and Python scripts used to determine taxonomic boundaries among Streptomyces genomes sharing identical full-length 16S rRNA sequences. All output and pyANI log files. (ZIP 37.1MB) Supplementary File 20. Network analysis of genomes based on shared 16S sequences. ZIP file containing jupyter notebook with NetworkX analysis and all associated output files including. bash script for pyANI analysis runs on each connected component and all associated matrices, heatmaps and log files. (ZIP 29.3MB) Supplementary File 21. Interactive network graph. HTML file containing interactive network graph of genomes sharing common full-length 16S sequences with each node colour corresponding to the number of connections/degrees. (HTML 4.7MB) Supplementary File 22. Interactive network graph. HTML file containing interactive network graph of genomes sharing common full-length 16S sequences showing clique (blue) and non-clique (green) components. (HTML 4.7MB) Supplementary File 23. Interactive network graph. HTML file containing interactive network graph of genomes sharing common full-length 16S sequences showing number of unique genera within each connected component. Each candidate genus is represented as a single node colour within a connected component. (HTML 4.7MB) Supplementary File 24. Interactive network graph. HTML file containing interactive network graph of genomes sharing common full-length 16S rRNA sequences showing number of unique species within each connected component. Each candidate species is represented as a single node colour within a connected component. (HTML 4.7MB) Supplementary File 25 Interactive network graph. HTML file containing interactive network graph of genomes sharing common full-length 16S rRNA sequences showing number of unique NCBI names within each connected component. Each NCBI assigned name is represented as a single node colour within a connected component. Gray nodes represent genomes currently lacking assigned species names. (HTML 4.7MB) Supplementary file 26. Intragenomic 16S rRNA heterogeneity within 1,369 Streptomyces genomes which exclusively contain only full-length and ambiguity symbol-free 16S rRNA sequences. A total of 811 genomes containing single 16S rRNA sequences are not shown. (PDF 8KB) Supplementary File 27. Distribution of 16S copies per genome with a distinction between unique and total copies for genomes at assembly level complete and chromosome. (PDF 7KB) Supplementary File 28. Schematic workflow for construction of the full-length 16S rRNA Streptomyces phylogeny. Each arrow represents a process and is annotated with script used and corresponding supplementary file. Output/data files, and the number of remaining sequences after each step, are indicated by rectangles. The green shading represents a single processing step of collecting and collating 16S database sequences. (PDF 91KB) Supplementary File 29. Schematic representation of the pipeline used to filter publicly available Streptomyces genomes. (PDF 59KB) Supplementary File 30. Sankey plot showing counts of taxonomic names in source databases, assigned at ranks from phylum to genus, to sequences identified with a key word Streptomyces in the taxonomy field. Note that Actinobacteria and Actinobacteriota are synonyms in LPSN for the correct Phylum name Actinomycetota, but that Actinomycetales and Streptomycetales are not taxonomic synonyms for each other. Streptomycetales is synonymous in LPSN with the correct name Kitasatosporales; Actinomycetales is a distinct taxonomic Order. The parent order of the Family Streptomycetaceae in LPSN is Kitasatosporales. (PDF 64KB) Supplementary File 31. Rectangular phylogram of the comprehensive maximum-likelihood tree of the genus Streptomyces based on the 16S sequence diversity of all 5,064 full-length 16S rRNA sequences with 100 TBE values. (PDF 194KB) Supplementary file 32. Genomes sharing identical 16S rRNA sequences are assigned different names in NCBI. A total of 1,030 singleton clusters are not shown. (PDF 8KB) Supplementary File 33. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of members of the novel Acintacidiphila genus. (PDF 228KB) Supplementary File 34. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of members of the novel Phaeacidiphilus genus. (PDF 228KB) Supplementary File 35. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of members of the novel Mangrovactinospora genus. (PDF 228KB) Supplementary File 36. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of members of the novel Wenjunlia genus. (PDF 228KB) Supplementary File 37. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of members of the novel Streptantibioticus genus. (PDF 228KB)\nActive learning of enhancer and silencer regulatory grammar in photoreceptors\nAuthors: Friedman, R. Z.; Ramu, A.; Lichtarge, S.; Myers, C. A.; Granas, D. M.; Gause, M.; Corbo, J. C.; Cohen, B. A.; White, M. A.\nScore: 15.8, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554146\nCis-regulatory elements (CREs) direct gene expression in health and disease, and models that can accurately predict their activities from DNA sequences are crucial for biomedicine. Deep learning represents one emerging strategy to model the regulatory grammar that relates CRE sequence to function. However, these models require training data on a scale that exceeds the number of CREs in the genome. We address this problem using active machine learning to iteratively train models on multiple rounds of synthetic DNA sequences assayed in live mammalian retinas. During each round of training the model actively selects sequence perturbations to assay, thereby efficiently generating informative training data. We iteratively trained a model that predicts the activities of sequences containing binding motifs for the photoreceptor transcription factor Cone-rod homeobox (CRX) using an order of magnitude less training data than current approaches. The model's internal confidence estimates of its predictions are reliable guides for designing sequences with high activity. The model correctly identified critical sequence differences between active and inactive sequences with nearly identical transcription factor binding sites, and revealed order and spacing preferences for combinations of motifs. Our results establish active learning as an effective method to train accurate deep learning models of cis-regulatory function after exhausting naturally occurring training examples in the genome.\nMapping Human Tissues with Highly Multiplexed RNA in situ Hybridization\nAuthors: Kalhor, K.; Chen, C.-j.; Lee, H. S.; Cai, M.; Nafisi, M.; Que, R.; Palmer, C.; Yuan, Y.; Zhang, Y.; Song, J.; Knoten, A.; Lake, B.; Gaut, J.; Keene, D.; Lein, E.; Kharchenko, P.; Chun, J.; Jain, S.; Fan, J.-B.; Zhang, K.\nScore: 13.8, Published: 2023-08-19 DOI: 10.1101/2023.08.16.553610\nIn situ transcriptomic techniques promise a holistic view of tissue organization and cell-cell interactions. Recently there has been a surge of multiplexed RNA in situ techniques but their application to human tissues and clinical biopsies has been limited due to their large size, general lower tissue quality and high background autofluorescence. Here we report DART-FISH, a versatile padlock probe-based technology capable of profiling hundreds to thousands of genes in centimeter-sized human tissue sections at cellular resolution. We introduced an omni-cell type cytoplasmic stain, dubbed RiboSoma that substantially improves the segmentation of cell bodies. We developed a computational decoding-by-deconvolution workflow to extract gene spots even in the presence of optical crowding. Our enzyme-free isothermal decoding procedure allowed us to image 121 genes in a large section from the human neocortex in less than 10 hours, where we successfully recapitulated the cytoarchitecture of 20 neuronal and non-neuronal subclasses. Additionally, we demonstrated the detection of transcripts as short as 461 nucleotides, including neuropeptides and discovered new cortical layer markers. We further performed in situ mapping of 300 genes on a diseased human kidney, profiled \u003e20 healthy and pathological cell states, and identified diseased niches enriched in transcriptionally altered epithelial cells and myofibroblasts.\nReconstructing the genetic relationship between ancient and present-day Siberian populations\nAuthors: Gill, H.; Lee, J.; Jeong, C.\nScore: 11.8, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554074\nHuman populations across a vast area in northern Eurasia, from Fennoscandia to Chukotka, share a distinct genetic component often referred to as the Siberian ancestry. Most enriched in present-day Samoyedic-speaking populations such as Nganasans, its origins and history still remain elusive despite the growing list of ancient and present-day genomes from Siberia. Here we reanalyze published ancient and present-day Siberian genomes focusing on the Baikal and Yakutia, resolving key questions regarding their genetic history. First, we show a long-term presence of a unique genetic profile in southern Siberia, up to 6,000 years ago, which distinctly shares a deep ancestral connection with Native Americans. Second, in the Baikal we find no direct contribution of the Early Neolithic Kitoi people to Late Neolithic and Early Bronze Age Serovo-Glazkovo ones. Third, the Middle Neolithic individual from Yakutia, belonging to the Belkachi culture, serves as the best source so far available for the spread of the Siberian ancestry into Fennoscandia and Greenland. These findings shed light on the genetic legacy of the Siberian ancestry and provide insights into the complex interplay between different populations in northern Eurasia throughout history.\nR2C2+UMI: Combining concatemeric consensus sequencing with unique molecular identifiers enables ultra-accurate sequencing of amplicons on Oxford Nanopore Technologies sequencers\nAuthors: Deng, D. Z. Q.; Verhage, J.; Neudorf, C.; Corbett-Detig, R.; Mekonen, H.; Castaldi, P. J.; Vollmers, C.\nScore: 11.5, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553937\nThe sequencing of PCR amplicons is a core application of high-throughput sequencing technology. Using unique molecular identifiers (UMIs), individual amplified molecules can be sequenced to very high accuracy on an Illumina sequencer. However, Illumina sequencers have limited read length and are therefore restricted to sequencing amplicons shorter than 600bp unless using inefficient synthetic long-read approaches. Native long-read sequencers from Pacific Biosciences and Oxford Nanopore Technologies can, using consensus read approaches, match or exceed Illumina quality while achieving much longer read lengths. Using a circularization-based concatemeric consensus sequencing approach (R2C2) paired with UMIs (R2C2+UMI) we show that we can sequence ~550nt antibody heavy-chain (IGH) and ~1500nt 16S amplicons at accuracies up to and exceeding Q50 (\u003c1 error in 100,0000 sequenced bases), which exceeds accuracies of UMI-supported Illumina paired sequencing as well as synthetic long-read approaches.\nA Pooled Cell Painting CRISPR Screening Platform Enables de novo Inference of Gene Function by Self-supervised Deep Learning\nAuthors: Sivanandan, S.; Leitmann, B.; Lubeck, E.; Sultan, M. M.; Stanitsas, P.; Ranu, N.; Ewer, A.; Mancuso, J. E.; Phillips, Z. F.; Kim, A.; Bisognano, J. W.; Cesarek, J.; Ruggiu, F.; Feldman, D.; Koller, D.; Sharon, E.; Kaykas, A.; Salick, M. R.; Chu, C.\nScore: 9.3, Published: 2023-08-16 DOI: 10.1101/2023.08.13.553051\nPooled CRISPR screening has emerged as a powerful method of mapping gene functions thanks to its scalability, affordability, and robustness against well or plate-specific confounders present in array-based screening 1-6. Most pooled CRISPR screens assay for low dimensional phenotypes (e.g. fitness, fluorescent markers). Higher-dimensional assays such as perturb-seq are available but costly and only applicable to transcriptomics readouts 7-11. Recently, pooled optical screening, which combines pooled CRISPR screening and microscopy-based assays, has been demonstrated in the studies of the NFkB pathway, essential human genes, cytoskeletal organization and antiviral response 12-15. While the pooled optical screening methodology is scalable and information-rich, the applications thus far employ hypothesis-specific assays. Here, we enable hypothesis-free reverse genetic screening for generic morphological phenotypes by re-engineering the Cell Painting 16 technique to provide compatibility with pooled optical screening. We validated this technique using well-defined morphological genesets (124 genes), compared classical image analysis and self-supervised learning methods using a mechanism-of-action (MoA) library (300 genes), and performed discovery screening with a druggable genome library (1640 genes) 17. Across these three experiments we show that the combination of rich morphological data and deep learning allows gene networks to emerge without the need for target-specific biomarkers, leading to better discovery of gene functions.\n",
  "wordCount" : "3939",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553620">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553620" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553620">
        <p class="paperTitle">Programmable multi-kilobase RNA editing using CRISPR-mediated trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553620" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553620" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Borrajo, J.; Javanmardi, K.; Griffin, J.; St. Martin, S. J.; Yao, D.; Hill, K.; Blainey, P.; Al-Shayeb, B.</p>
        <p class="info">Score: 102.2, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553620' target='https://doi.org/10.1101/2023.08.18.553620'> 10.1101/2023.08.18.553620</a></p>
        <p class="abstract">Current gene editing approaches in eukaryotic cells are limited to single base edits or small DNA insertions and deletions, and remain encumbered by unintended permanent effects and significant challenges in the delivery of large DNA cargo. Here we describe Splice Editing, a generalizable platform to correct gene transcripts in situ by programmable insertion or replacement of large RNA segments. By combining CRISPR-mediated RNA targeting with endogenous cellular RNA-splicing machinery, Splice Editing enables efficient, precise, and programmable large-scale editing of gene targets without DNA cleavage or mutagenesis. RNA sequencing and measurement of spliced protein products confirm that Splice Editing achieves efficient and specific targeted RNA and protein correction. We show that Splice Editors based on novel miniature RNA-targeting CRISPR-Cas systems discovered and characterized in this work can be packaged for effective delivery to human cells and affect different types of edits across multiple targets and cell lines. By editing thousands of bases simultaneously in a single reversible step, Splice Editing could expand the treatable disease population for monogenic diseases with large allelic diversity without the permanent unintended effects of DNA editing.

One-sentence summaryCRISPR-guided trans-splicing enables efficient and specific recombination of large RNA molecules in mammalian cells, with broad applications in therapeutic development for genetic diseases and as a research tool for the study of basic RNA biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.553986">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.553986" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.553986">
        <p class="paperTitle">Mapping the Genetic Interaction Network of PARP inhibitor Response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.553986" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.553986" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Simpson, D.; Ling, J.; Jing, Y.; Adamson, B.</p>
        <p class="info">Score: 35.7, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.553986' target='https://doi.org/10.1101/2023.08.19.553986'> 10.1101/2023.08.19.553986</a></p>
        <p class="abstract">Genetic interactions have long informed our understanding of the coordinated proteins and pathways that respond to DNA damage in mammalian cells, but systematic interrogation of the genetic network underlying that system has yet to be achieved. Towards this goal, we measured 147,153 pairwise interactions among genes implicated in PARP inhibitor (PARPi) response. Evaluating genetic interactions at this scale, with and without exposure to PARPi, revealed hierarchical organization of the pathways and complexes that maintain genome stability during normal growth and defined changes that occur upon accumulation of DNA lesions due to cytotoxic doses of PARPi. We uncovered unexpected relationships among DNA repair genes, including context-specific buffering interactions between the minimally characterized AUNIP and BRCA1-A complex genes. Our work thus establishes a foundation for mapping differential genetic interactions in mammalian cells and provides a comprehensive resource for future studies of DNA repair and PARP inhibitors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.552164">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.552164" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.552164">
        <p class="paperTitle">A genome-wide atlas of human cell morphology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.552164" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.552164" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramezani, M.; Bauman, J.; Singh, A.; Weisbart, E.; Yong, J.; Lozada, M. E.; Way, G. P.; Kavari, S. L.; Diaz, C.; Haghighi, M.; Perez-Schindler, J.; Batista, T. M.; Claussnitzer, M.; Singh, S.; Cimini, B. A.; Blainey, P. C.; Carpenter, A. E.; Jan, C. H.; Neal, J. T.</p>
        <p class="info">Score: 65.4, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.552164' target='https://doi.org/10.1101/2023.08.06.552164'> 10.1101/2023.08.06.552164</a></p>
        <p class="abstract">A key challenge of the modern genomics era is developing data-driven representations of gene function. Here, we present the first unbiased morphology-based genome-wide perturbation atlas in human cells, containing three genome-scale genotype-phenotype maps comprising &gt;20,000 single-gene CRISPR-Cas9-based knockout experiments in &gt;30 million cells. Our optical pooled cell profiling approach (PERISCOPE) combines a de-stainable high-dimensional phenotyping panel (based on Cell Painting1,2) with optical sequencing of molecular barcodes and a scalable open-source analysis pipeline to facilitate massively parallel screening of pooled perturbation libraries. This approach provides high-dimensional phenotypic profiles of individual cells, while simultaneously enabling interrogation of subcellular processes. Our atlas reconstructs known pathways and protein-protein interaction networks, identifies culture media-specific responses to gene knockout, and clusters thousands of human genes by phenotypic similarity. Using this atlas, we identify the poorly-characterized disease-associated transmembrane protein TMEM251/LYSET as a Golgi-resident protein essential for mannose-6-phosphate-dependent trafficking of lysosomal enzymes, showing the power of these representations. In sum, our atlas and screening technology represent a rich and accessible resource for connecting genes to cellular functions at scale.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552346">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552346" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552346">
        <p class="paperTitle">Environmental challenge rewires functional connections among human genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552346" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552346" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Herken, B. W.; Wong, G.; Norman, T.; Gilbert, L.</p>
        <p class="info">Score: 23.7, Published: 2023-08-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552346' target='https://doi.org/10.1101/2023.08.09.552346'> 10.1101/2023.08.09.552346</a></p>
        <p class="abstract">A fundamental question in biology is how a limited number of genes combinatorially govern cellular responses to environmental changes. While the prevailing hypothesis is that relationships between genes, processes, and ontologies could be plastic to achieve this adaptability, quantitatively comparing human gene functional connections between specific environmental conditions at scale is very challenging. Therefore, it remains unclear whether and how human genetic interaction networks are rewired in response to changing environmental conditions. Here, we developed a framework for mapping context-specific genetic interactions, enabling us to measure the plasticity of human genetic architecture upon environmental challenge for [~]250,000 interactions, using cell cycle interruption, genotoxic perturbation, and nutrient deprivation as archetypes. We discover large-scale rewiring of human gene relationships across conditions, highlighted by dramatic shifts in the functional connections of epigenetic regulators (TIP60), cell cycle regulators (PP2A), and glycolysis metabolism. Our study demonstrates that upon environmental perturbation, intra-complex genetic rewiring is rare while inter-complex rewiring is common, suggesting a modular and flexible evolutionary genetic strategy that allows a limited number of human genes to enable adaptation to a large number of environmental conditions.

One Sentence SummaryFive human genetic interaction maps reveal how the landscape of genes functional relationships is rewired as cells experience environmental stress to DNA integrity, cell cycle regulation, and metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553377">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553377" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553377">
        <p class="paperTitle">16S rRNA phylogeny and clustering is not a reliable proxy for genome-based taxonomy in Streptomyces</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553377" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553377" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kiepas, A. B.; Hoskisson, P. A.; Pritchard, L.</p>
        <p class="info">Score: 16.3, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553377' target='https://doi.org/10.1101/2023.08.15.553377'> 10.1101/2023.08.15.553377</a></p>
        <p class="abstract">Although Streptomyces is one of the most extensively studied genera of bacteria, their taxonomy remains contested and is suspected to contain significant species-level misclassification. Resolving the classification of Streptomyces would benefit many areas of study and applied microbiology that rely heavily on having an accurate ground truth classification of similar and dissimilar organisms, including comparative genomics-based searches for novel antimicrobials in the fight against the ongoing antimicrobial resistance (AMR) crisis. To attempt a resolution, we investigate taxonomic conflicts between 16S rRNA and whole genome classifications using all available 48,981 full-length 16S rRNA Streptomyces sequences from the combined SILVA, Greengenes, Ribosomal Database Project (RDP) and NCBI (National Center for Biotechnology Information) databases, and 2,276 publicly available Streptomyces genome assemblies. We construct a 16S gene tree for 14,239 distinct Streptomyces 16S rRNA sequences, identifying three major lineages of Streptomyces, and find that existing taxonomic classifications are inconsistent with the tree topology. We also use these data to delineate 16S and whole genome landscapes for Streptomyces, finding that 16S and whole-genome classifications of Streptomyces strains are frequently in disagreement, and in particular that 16S zero-radius Operational Taxonomic Units (zOTUs) are often inconsistent with Average Nucleotide Identity (ANI)-based taxonomy. Our results strongly imply that 16S rRNA sequence data does not map to taxonomy sufficiently well to delineate Streptomyces species reliably, and we propose that alternative markers should instead be adopted by the community for classification and metabarcoding. As much of current Streptomyces taxonomy has been determined or supported by historical 16S sequence data and may in parts be in error, we also propose that reclassification of the genus by alternative approaches is required.

Impact StatementAccurate classification of microbes, usually in the form of taxonomic assignments, provides a fundamental ground truth or reference point for many aspects of applied microbiology including comparative genomics, identification of strains for natural product discovery, and dereplication of strains. Bacteria belonging to the genus Streptomyces are an important source of bioactive metabolites and enzymes in biotechnology, and proper understanding of their phylogeny aids understanding of the evolution of industrially important gene products and metabolites, and prioritization of strains for industrial exploitation. Taxonomic classification in the genus Streptomyces is complex and contested, and there are clear conflicts between taxonomies inferred from 16S rRNA and from whole genome sequences. Despite this, 16S sequence-based classifications are still widely used to infer taxonomic identity, to determine community composition, and to prioritise strains for study. We investigate a diverse and comprehensive set of Streptomyces genomes using whole-genome Average Nucleotide Identity (ANI) and 16S sequence analysis to delineate and compare classifications made using these approaches. We outline the genomic and 16S sequence landscape of Streptomyces, demonstrating that (i) distinct taxonomic species may share identical full-length 16S sequences, and (ii) in some instances, isolates representing the same taxonomic species do not share any common 16S rRNA sequence. Our results strongly imply that 16S rRNA sequence variation does not map to taxonomy sufficiently well to delineate Streptomyces species reliably, and that alternative markers should instead be adopted by the community. Much of current Streptomyces taxonomy has been determined or supported by historical 16S sequence data, and we therefore propose that reclassification within this group by alternative approaches is required.

Data summaryAll code, raw and supporting data are publicly available from GitHub (https://github.com/kiepczi/Kiepas_et_al_2023_16S) and Zenodo (https://doi.org/10.5281/zenodo.8223787). The flowchart provided in Supplementary File 28 provides an overview of analysis steps and serves as a guide through Supplementary Files generated during reconstruction of the 16S phylogeny. The flowchart in Supplementary File 29 outlines the workflow processes and supplementary materials used for analysis of 16S rRNA sequences from Streptomyces genomes.

Supplementary DataSupplementary File 1: Generate figures using Python and R. ZIP file containing all data, Python and R scripts to generate figures for this manuscript. (ZIP 40.9MB)

Supplementary File 2: Raw 16S rRNA public databases. Zip file containing four separate txt files with sequence IDs for public 16S rRNA databases used in this manuscript, and an additional txt file with Greengenes sequence taxonomy information, and a Python script used to map taxonomy information to sequences found in Greengenes v13.5. (ZIP 34.8MB)

Supplementary File 3: Filtration of 16S rRNA public databases. Zip file containing Python script used for filtration of the raw databases, and generated outputs. (ZIP 7.2MB)

Supplementary File 4: Cleaning of the filtrated 16S rRNA local. Zip file containing all bash and Python scripts used to clean the local full-length 16S rRNA local databases by removing redundant and poor quality 16S rRNA sequences. (ZIP 9MB)

Supplementary File 5: Sequence Clustering. Zip file containing a bash script used to cluster the full-length cleaned local 16S rRNA Streptomyces local databases at various thresholds, and provides txt files with accessions for representative sequences, and cluster members for each clustering threshold. (ZIP 40.8MB)

Supplementary File 6: Analysis of taxonomic composition for each clustering threshold. Zip file containing Python scripts, NCBI taxonomy input and all outputs generated used to determine the taxonomic composition for each clustering threshold. (ZIP 49.6)

Supplementary File 7. Cluster sizes. Empirical cumulative frequency plot showing cluster sizes generated for all clustering thresholds. (PDF 44KB)

Supplementary File 8. Cluster taxID abundance. Empirical cumulative frequency plot for unique numbers of taxID present at all clustering thresholds. (PDF 9KB)

Supplementary File 9. MSA. Zip file containing all Python and bash scripts, and additional data needed to generate and clean MSA for phylogenetic analysis. (ZIP 4.2MB)

Supplementary File 10. Phylogenetic reconstruction. ZIP file containing bash scripts used for phylogenetic reconstruction, and all generated outputs and log files. (ZIP 16.8MB).

Supplementary File 11. Collapse branches. ZIP file containing jupyter notebook used for collapsing branches with the same species names, and the collapsed tree in newick format. (ZIP 385KB)

Supplementary File 12. Phylogenetic tree. PDF file showing collapsed phylogenetic tree with marked branches with transfer bootstrap expectation support of &gt;= 50%. (PDF 224KB)

Supplementary File 13. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of Streptomyces albus and Streptomyces griseus. (PDF 229KB)

Supplementary File 14. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of Streptomyces albulus, Streptomyces lydicus and Streptomyces venezuelae. (PDF 228KB)

Supplementary File 15. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of Streptomyces clavuligerus and Streptomyces coelicolor. (PDF 227KB)

Supplementary File 16. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of Streptomyces lavendulae, Streptomyces rimosus and Streptomyces scabiei. (PDF 228KB)

Supplementary File 17. Streptomyces genomes. Zip file containing bash scripts used to download Streptomyces genomes, and Python scripts used to check assembly status. The ZIP file also contains two separate txt files with Streptomyces genomes used in this manuscript: one file with all initial candidates, and a second file with replaced genomes. (ZIP 2.6MB)

Supplementary File 18. Extraction of full-length and ambiguity free 16S rRNA sequences from Streptomyces genomes. Zip file containing all Python and bash scripts used to extract full-length sequences from the filtered Streptomyces genomes. A single FASTA file with all extracted 16S rRNA sequences, and a single FASTA file with filtered sequences. A txt file with accession of genomes retained in the analysis. (ZIP 742KB)

Supplementary File 19. ANI analysis among Streptomyces genomes with identical 16S rRNA sequences. ZIP file containing all Bash and Python scripts used to determine taxonomic boundaries among Streptomyces genomes sharing identical full-length 16S rRNA sequences. All output and pyANI log files. (ZIP 37.1MB)

Supplementary File 20. Network analysis of genomes based on shared 16S sequences. ZIP file containing jupyter notebook with NetworkX analysis and all associated output files including. bash script for pyANI analysis runs on each connected component and all associated matrices, heatmaps and log files. (ZIP 29.3MB)

Supplementary File 21. Interactive network graph. HTML file containing interactive network graph of genomes sharing common full-length 16S sequences with each node colour corresponding to the number of connections/degrees. (HTML 4.7MB)

Supplementary File 22. Interactive network graph. HTML file containing interactive network graph of genomes sharing common full-length 16S sequences showing clique (blue) and non-clique (green) components. (HTML 4.7MB)

Supplementary File 23. Interactive network graph. HTML file containing interactive network graph of genomes sharing common full-length 16S sequences showing number of unique genera within each connected component. Each candidate genus is represented as a single node colour within a connected component. (HTML 4.7MB)

Supplementary File 24. Interactive network graph. HTML file containing interactive network graph of genomes sharing common full-length 16S rRNA sequences showing number of unique species within each connected component. Each candidate species is represented as a single node colour within a connected component. (HTML 4.7MB)

Supplementary File 25 Interactive network graph. HTML file containing interactive network graph of genomes sharing common full-length 16S rRNA sequences showing number of unique NCBI names within each connected component. Each NCBI assigned name is represented as a single node colour within a connected component. Gray nodes represent genomes currently lacking assigned species names. (HTML 4.7MB)

Supplementary file 26. Intragenomic 16S rRNA heterogeneity within 1,369 Streptomyces genomes which exclusively contain only full-length and ambiguity symbol-free 16S rRNA sequences. A total of 811 genomes containing single 16S rRNA sequences are not shown. (PDF 8KB)

Supplementary File 27. Distribution of 16S copies per genome with a distinction between unique and total copies for genomes at assembly level complete and chromosome. (PDF 7KB)

Supplementary File 28. Schematic workflow for construction of the full-length 16S rRNA Streptomyces phylogeny. Each arrow represents a process and is annotated with script used and corresponding supplementary file. Output/data files, and the number of remaining sequences after each step, are indicated by rectangles. The green shading represents a single processing step of collecting and collating 16S database sequences. (PDF 91KB)

Supplementary File 29. Schematic representation of the pipeline used to filter publicly available Streptomyces genomes. (PDF 59KB)

Supplementary File 30. Sankey plot showing counts of taxonomic names in source databases, assigned at ranks from phylum to genus, to sequences identified with a key word  Streptomyces in the taxonomy field. Note that Actinobacteria and Actinobacteriota are synonyms in LPSN for the correct Phylum name Actinomycetota, but that Actinomycetales and Streptomycetales are not taxonomic synonyms for each other. Streptomycetales is synonymous in LPSN with the correct name Kitasatosporales; Actinomycetales is a distinct taxonomic Order. The parent order of the Family Streptomycetaceae in LPSN is Kitasatosporales. (PDF 64KB)

Supplementary File 31. Rectangular phylogram of the comprehensive maximum-likelihood tree of the genus Streptomyces based on the 16S sequence diversity of all 5,064 full-length 16S rRNA sequences with 100 TBE values. (PDF 194KB)

Supplementary file 32. Genomes sharing identical 16S rRNA sequences are assigned different names in NCBI. A total of 1,030 singleton clusters are not shown. (PDF 8KB)

Supplementary File 33. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of members of the novel Acintacidiphila genus. (PDF 228KB)

Supplementary File 34. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of members of the novel Phaeacidiphilus genus. (PDF 228KB)

Supplementary File 35. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of members of the novel Mangrovactinospora genus. (PDF 228KB)

Supplementary File 36. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of members of the novel Wenjunlia genus. (PDF 228KB)

Supplementary File 37. Phylogenetic tree. PDF file showing collapsed phylogenetic tree showing distribution of members of the novel Streptantibioticus genus. (PDF 228KB)</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554146">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554146" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554146">
        <p class="paperTitle">Active learning of enhancer and silencer regulatory grammar in photoreceptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554146" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554146" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Friedman, R. Z.; Ramu, A.; Lichtarge, S.; Myers, C. A.; Granas, D. M.; Gause, M.; Corbo, J. C.; Cohen, B. A.; White, M. A.</p>
        <p class="info">Score: 15.8, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554146' target='https://doi.org/10.1101/2023.08.21.554146'> 10.1101/2023.08.21.554146</a></p>
        <p class="abstract">Cis-regulatory elements (CREs) direct gene expression in health and disease, and models that can accurately predict their activities from DNA sequences are crucial for biomedicine. Deep learning represents one emerging strategy to model the regulatory grammar that relates CRE sequence to function. However, these models require training data on a scale that exceeds the number of CREs in the genome. We address this problem using active machine learning to iteratively train models on multiple rounds of synthetic DNA sequences assayed in live mammalian retinas. During each round of training the model actively selects sequence perturbations to assay, thereby efficiently generating informative training data. We iteratively trained a model that predicts the activities of sequences containing binding motifs for the photoreceptor transcription factor Cone-rod homeobox (CRX) using an order of magnitude less training data than current approaches. The model&#39;s internal confidence estimates of its predictions are reliable guides for designing sequences with high activity. The model correctly identified critical sequence differences between active and inactive sequences with nearly identical transcription factor binding sites, and revealed order and spacing preferences for combinations of motifs. Our results establish active learning as an effective method to train accurate deep learning models of cis-regulatory function after exhausting naturally occurring training examples in the genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553610">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553610" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553610">
        <p class="paperTitle">Mapping Human Tissues with Highly Multiplexed RNA in situ Hybridization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553610" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553610" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kalhor, K.; Chen, C.-j.; Lee, H. S.; Cai, M.; Nafisi, M.; Que, R.; Palmer, C.; Yuan, Y.; Zhang, Y.; Song, J.; Knoten, A.; Lake, B.; Gaut, J.; Keene, D.; Lein, E.; Kharchenko, P.; Chun, J.; Jain, S.; Fan, J.-B.; Zhang, K.</p>
        <p class="info">Score: 13.8, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553610' target='https://doi.org/10.1101/2023.08.16.553610'> 10.1101/2023.08.16.553610</a></p>
        <p class="abstract">In situ transcriptomic techniques promise a holistic view of tissue organization and cell-cell interactions. Recently there has been a surge of multiplexed RNA in situ techniques but their application to human tissues and clinical biopsies has been limited due to their large size, general lower tissue quality and high background autofluorescence. Here we report DART-FISH, a versatile padlock probe-based technology capable of profiling hundreds to thousands of genes in centimeter-sized human tissue sections at cellular resolution. We introduced an omni-cell type cytoplasmic stain, dubbed RiboSoma that substantially improves the segmentation of cell bodies. We developed a computational decoding-by-deconvolution workflow to extract gene spots even in the presence of optical crowding. Our enzyme-free isothermal decoding procedure allowed us to image 121 genes in a large section from the human neocortex in less than 10 hours, where we successfully recapitulated the cytoarchitecture of 20 neuronal and non-neuronal subclasses. Additionally, we demonstrated the detection of transcripts as short as 461 nucleotides, including neuropeptides and discovered new cortical layer markers. We further performed in situ mapping of 300 genes on a diseased human kidney, profiled &gt;20 healthy and pathological cell states, and identified diseased niches enriched in transcriptionally altered epithelial cells and myofibroblasts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554074">
        <p class="paperTitle">Reconstructing the genetic relationship between ancient and present-day Siberian populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gill, H.; Lee, J.; Jeong, C.</p>
        <p class="info">Score: 11.8, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554074' target='https://doi.org/10.1101/2023.08.21.554074'> 10.1101/2023.08.21.554074</a></p>
        <p class="abstract">Human populations across a vast area in northern Eurasia, from Fennoscandia to Chukotka, share a distinct genetic component often referred to as the Siberian ancestry. Most enriched in present-day Samoyedic-speaking populations such as Nganasans, its origins and history still remain elusive despite the growing list of ancient and present-day genomes from Siberia. Here we reanalyze published ancient and present-day Siberian genomes focusing on the Baikal and Yakutia, resolving key questions regarding their genetic history. First, we show a long-term presence of a unique genetic profile in southern Siberia, up to 6,000 years ago, which distinctly shares a deep ancestral connection with Native Americans. Second, in the Baikal we find no direct contribution of the Early Neolithic Kitoi people to Late Neolithic and Early Bronze Age Serovo-Glazkovo ones. Third, the Middle Neolithic individual from Yakutia, belonging to the Belkachi culture, serves as the best source so far available for the spread of the Siberian ancestry into Fennoscandia and Greenland. These findings shed light on the genetic legacy of the Siberian ancestry and provide insights into the complex interplay between different populations in northern Eurasia throughout history.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.553937">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.553937" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.553937">
        <p class="paperTitle">R2C2&#43;UMI: Combining concatemeric consensus sequencing with unique molecular identifiers enables ultra-accurate sequencing of amplicons on Oxford Nanopore Technologies sequencers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.553937" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.553937" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Deng, D. Z. Q.; Verhage, J.; Neudorf, C.; Corbett-Detig, R.; Mekonen, H.; Castaldi, P. J.; Vollmers, C.</p>
        <p class="info">Score: 11.5, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.553937' target='https://doi.org/10.1101/2023.08.19.553937'> 10.1101/2023.08.19.553937</a></p>
        <p class="abstract">The sequencing of PCR amplicons is a core application of high-throughput sequencing technology. Using unique molecular identifiers (UMIs), individual amplified molecules can be sequenced to very high accuracy on an Illumina sequencer. However, Illumina sequencers have limited read length and are therefore restricted to sequencing amplicons shorter than 600bp unless using inefficient synthetic long-read approaches. Native long-read sequencers from Pacific Biosciences and Oxford Nanopore Technologies can, using consensus read approaches, match or exceed Illumina quality while achieving much longer read lengths. Using a circularization-based concatemeric consensus sequencing approach (R2C2) paired with UMIs (R2C2&#43;UMI) we show that we can sequence ~550nt antibody heavy-chain (IGH) and ~1500nt 16S amplicons at accuracies up to and exceeding Q50 (&lt;1 error in 100,0000 sequenced bases), which exceeds accuracies of UMI-supported Illumina paired sequencing as well as synthetic long-read approaches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553051">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553051" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553051">
        <p class="paperTitle">A Pooled Cell Painting CRISPR Screening Platform Enables de novo Inference of Gene Function by Self-supervised Deep Learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553051" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553051" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sivanandan, S.; Leitmann, B.; Lubeck, E.; Sultan, M. M.; Stanitsas, P.; Ranu, N.; Ewer, A.; Mancuso, J. E.; Phillips, Z. F.; Kim, A.; Bisognano, J. W.; Cesarek, J.; Ruggiu, F.; Feldman, D.; Koller, D.; Sharon, E.; Kaykas, A.; Salick, M. R.; Chu, C.</p>
        <p class="info">Score: 9.3, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553051' target='https://doi.org/10.1101/2023.08.13.553051'> 10.1101/2023.08.13.553051</a></p>
        <p class="abstract">Pooled CRISPR screening has emerged as a powerful method of mapping gene functions thanks to its scalability, affordability, and robustness against well or plate-specific confounders present in array-based screening 1-6. Most pooled CRISPR screens assay for low dimensional phenotypes (e.g. fitness, fluorescent markers). Higher-dimensional assays such as perturb-seq are available but costly and only applicable to transcriptomics readouts 7-11. Recently, pooled optical screening, which combines pooled CRISPR screening and microscopy-based assays, has been demonstrated in the studies of the NFkB pathway, essential human genes, cytoskeletal organization and antiviral response 12-15. While the pooled optical screening methodology is scalable and information-rich, the applications thus far employ hypothesis-specific assays. Here, we enable hypothesis-free reverse genetic screening for generic morphological phenotypes by re-engineering the Cell Painting 16 technique to provide compatibility with pooled optical screening. We validated this technique using well-defined morphological genesets (124 genes), compared classical image analysis and self-supervised learning methods using a mechanism-of-action (MoA) library (300 genes), and performed discovery screening with a druggable genome library (1640 genes) 17. Across these three experiments we show that the combination of rich morphological data and deep learning allows gene networks to emerge without the need for target-specific biomarkers, leading to better discovery of gene functions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
