<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Concurrent profiling of multiscale 3D genome organization and gene expression in single mammalian cells
Authors: Zhou, T.; Zhang, R.; Jia, D.; Doty, R. T.; Munday, A. D.; Gao, D.; Xin, L.; Abkowitz, J. L.; Duan, Z.; Ma, J.
Score: 49.0, Published: 2023-07-25 DOI: 10.1101/2023.07.20.549578
The organization of mammalian genomes within the nucleus features a complex, multiscale three-dimensional (3D) architecture. The functional significance of these 3D genome features, however, remains largely elusive due to limited single-cell technologies that can concurrently profile genome organization and transcriptional activities.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Concurrent profiling of multiscale 3D genome organization and gene expression in single mammalian cells
Authors: Zhou, T.; Zhang, R.; Jia, D.; Doty, R. T.; Munday, A. D.; Gao, D.; Xin, L.; Abkowitz, J. L.; Duan, Z.; Ma, J.
Score: 49.0, Published: 2023-07-25 DOI: 10.1101/2023.07.20.549578
The organization of mammalian genomes within the nucleus features a complex, multiscale three-dimensional (3D) architecture. The functional significance of these 3D genome features, however, remains largely elusive due to limited single-cell technologies that can concurrently profile genome organization and transcriptional activities." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-02T10:38:09+00:00" />
<meta property="article:modified_time" content="2023-08-02T10:38:09+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Concurrent profiling of multiscale 3D genome organization and gene expression in single mammalian cells
Authors: Zhou, T.; Zhang, R.; Jia, D.; Doty, R. T.; Munday, A. D.; Gao, D.; Xin, L.; Abkowitz, J. L.; Duan, Z.; Ma, J.
Score: 49.0, Published: 2023-07-25 DOI: 10.1101/2023.07.20.549578
The organization of mammalian genomes within the nucleus features a complex, multiscale three-dimensional (3D) architecture. The functional significance of these 3D genome features, however, remains largely elusive due to limited single-cell technologies that can concurrently profile genome organization and transcriptional activities."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Concurrent profiling of multiscale 3D genome organization and gene expression in single mammalian cells\nAuthors: Zhou, T.; Zhang, R.; Jia, D.; Doty, R. T.; Munday, A. D.; Gao, D.; Xin, L.; Abkowitz, J. L.; Duan, Z.; Ma, J.\nScore: 49.0, Published: 2023-07-25 DOI: 10.1101/2023.07.20.549578\nThe organization of mammalian genomes within the nucleus features a complex, multiscale three-dimensional (3D) architecture. The functional significance of these 3D genome features, however, remains largely elusive due to limited single-cell technologies that can concurrently profile genome organization and transcriptional activities.",
  "keywords": [
    
  ],
  "articleBody": " Concurrent profiling of multiscale 3D genome organization and gene expression in single mammalian cells\nAuthors: Zhou, T.; Zhang, R.; Jia, D.; Doty, R. T.; Munday, A. D.; Gao, D.; Xin, L.; Abkowitz, J. L.; Duan, Z.; Ma, J.\nScore: 49.0, Published: 2023-07-25 DOI: 10.1101/2023.07.20.549578\nThe organization of mammalian genomes within the nucleus features a complex, multiscale three-dimensional (3D) architecture. The functional significance of these 3D genome features, however, remains largely elusive due to limited single-cell technologies that can concurrently profile genome organization and transcriptional activities. Here, we report GAGE-seq, a highly scalable, robust single-cell co-assay that simultaneously measures 3D genome structure and transcriptome within the same cell. Employing GAGE-seq on mouse brain cortex and human bone marrow CD34+ cells, we comprehensively characterized the intricate relationships between 3D genome and gene expression. We found that these multiscale 3D genome features collectively inform cell type-specific gene expressions, hence contributing to defining cell identity at the single-cell level. Integration of GAGE-seq data with spatial transcriptomic data revealed in situ variations of the 3D genome in mouse cortex. Moreover, our observations of lineage commitment in normal human hematopoiesis unveiled notable discordant changes between 3D genome organization and gene expression, underscoring a complex, temporal interplay at the single-cell level that is more nuanced than previously appreciated. Together, GAGE-seq provides a powerful, cost-effective approach for interrogating genome structure and gene expression relationships at the single-cell level across diverse biological contexts.\nSystematic assessment of long-read RNA-seq methods for transcript identification and quantification\nAuthors: Pardo-Palacios, F. J.; Wang, D.; Reese, F.; Diekhans, M.; Carbonell-Sala, S.; Williams, B.; Loveland, J. E.; De Maria, M.; Adams, M. S.; Balderrama-Gutierrez, G.; Behera, A. K.; Gonzalez, J. M.; Hunt, T.; Lagarde, J.; Liang, C. E.; Li, H.; Meade, M. J.; Amador, D. A. M.; Prjibelski, A. D.; Birol, I.; Bostan, H.; Brooks, A. M.; Celik, M. H.; Chen, Y.; Du, M. R. M.; Felton, C.; Goke, J.; Hafezqorani, S.; Herwig, R.; Kawaji, H.; Lee, J.; Li, J.-L.; Lienhard, M.; Mikheenko, A.; Mulligan, D.; Nip, K. M.; Pertea, M.; Ritchie, M. E.; Sim, A. D.; Tang, A. D.; Wan, Y. K.; Wang, C.; Wong, B. Y.; Yang, C\nScore: 41.1, Published: 2023-07-27 DOI: 10.1101/2023.07.25.550582\nAbstractThe Long-read RNA-Seq Genome Annotation Assessment Project (LRGASP) Consortium was formed to evaluate the effectiveness of long-read approaches for transcriptome analysis. The consortium generated over 427 million long-read sequences from cDNA and direct RNA datasets, encompassing human, mouse, and manatee species, using different protocols and sequencing platforms. These data were utilized by developers to address challenges in transcript isoform detection and quantification, as well as de novo transcript isoform identification. The study revealed that libraries with longer, more accurate sequences produce more accurate transcripts than those with increased read depth, whereas greater read depth improved quantification accuracy. In well-annotated genomes, tools based on reference sequences demonstrated the best performance. When aiming to detect rare and novel transcripts or when using reference-free approaches, incorporating additional orthogonal data and replicate samples are advised. This collaborative study offers a benchmark for current practices and provides direction for future method development in transcriptome analysis.\nA multiplex, prime editing framework for identifying drug resistance variants at scale\nAuthors: Chardon, F. M.; Suiter, C.; Daza, R.; Smith, N. T.; Parrish, P.; McDiarmid, T.; Lalanne, J.-B.; Martin, B. K.; Calderon, D.; Ellison, A.; Berger, A.; Shendure, J.; Starita, L. M.\nScore: 36.8, Published: 2023-07-30 DOI: 10.1101/2023.07.27.550902\nCRISPR-based genome editing has revolutionized functional genomics, enabling screens in which thousands of perturbations of either gene expression or primary genome sequence can be competitively assayed in single experiments. However, for libraries of specific mutations, a challenge of CRISPR-based screening methods such as saturation genome editing is that only one region (e.g. one exon) can be studied per experiment. Here we describe prime-SGE (\"prime saturation genome editing\"), a new framework based on prime editing, in which libraries of specific mutations can be installed into genes throughout the genome and functionally assessed in a single, multiplex experiment. Prime-SGE is based on quantifying the abundance of prime editing guide RNAs (pegRNAs) in the context of a functional selection, rather than quantifying the mutations themselves. We apply prime-SGE to assay thousands of single nucleotide changes in eight oncogenes for their ability to confer drug resistance to three EGFR tyrosine kinase inhibitors. Although currently restricted to positive selection screens by the limited efficiency of prime editing, our strategy opens the door to the possibility of functionally assaying vast numbers of precise mutations at locations throughout the genome.\nSingle-cell gene expression prediction from DNA sequence at large contexts\nAuthors: Schwessinger, R.; Deasy, J.; Woodruff, R. T.; Young, S.; Branson, K. M.\nScore: 28.1, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550634\nHuman genetic variants impacting traits such as disease susceptibility frequently act through modulation of gene expression in a highly cell-type-specific manner. Computational models capable of predicting gene expression directly from DNA sequence can assist in the interpretation of expression-modulating variants, and machine learning models now operate at the large sequence contexts required for capturing long-range human transcriptional regulation. However, existing predictors have focused on bulk transcriptional measurements where gene expression heterogeneity can be drowned out in broadly defined cell types. Here, we use a transfer learning framework, seq2cells, leveraging a pre-trained epigenome model for gene expression prediction from large sequence contexts at single-cell resolution. We show that seq2cells captures cell-specific gene expression beyond the resolution of pseudo-bulked data. Using seq2cells for variant effect prediction reveals heterogeneity within annotated cell types and enables in silico transfer of variant effects between cell populations. We demonstrate the challenges and value of gene expression and variant effect prediction at single-cell resolution, and offer a path to the interpretation of genomic variation at uncompromising resolution and scale.\nSurprising connections between DNA binding and function for the near-complete set of yeast transcription factors\nAuthors: Mahendrawada, L.; Warfield, L.; Donczew, R.; hahn, s.\nScore: 20.7, Published: 2023-08-01 DOI: 10.1101/2023.07.25.550593\nDNA sequence-specific transcrip-on factors (TFs) modulate transcrip-on and chroma-n architecture, ac-ng from regulatory sites in enhancers and promoters of eukaryo-c genes. How TFs locate their DNA targets and how mul-ple TFs cooperate to regulate individual genes is s-ll unclear. Most yeast TFs are thought to regulate transcrip-on via binding to upstream ac-va-ng sequences, situated within a few hundred base pairs upstream of the regulated gene. While this model has been validated for individual TFs and specific genes, it has not been tested in a systema-c way with the large set of yeast TFs. Here, we have integrated informa-on on the binding and expression targets for the near-complete set of yeast TFs. While we found many instances of func-onal TF binding sites in upstream regulatory regions, we found many more instances that do not fit this model. In many cases, rapid TF deple-on affects gene expression where there is no detectable binding of that TF to the upstream region of the affected gene. In addi-on, for most TFs, only a small frac-on of bound TFs regulates the nearby gene, showing that TF binding does not automa-cally correspond to regula-on of the linked gene. Finally, we found that only a small percentage of TFs are exclusively strong ac-vators or repressors with most TFs having dual func-on. Overall, our comprehensive mapping of TF binding and regulatory targets have both confirmed known TF rela-onships and revealed surprising proper-es of TF func-on.\nKoNA: Korean Nucleotide Archive as a New Data Repository for Nucleotide Sequence Data\nAuthors: Ko, G.; Lee, J.; Sim, Y. M.; Song, W.; Yoon, B.-H.; Byeon, I.; Lee, B. H.; Kim, S.-O.; Choi, J.; Jang, I.; Kim, H.; Yang, J. O.; Jang, K.; Kim, S.; Kim, J.-H.; Jeon, J.; Jung, J.; Hwang, S.; Park, J.-H.; Kim, P.-G.; Kim, S.-Y.; Lee, B.\nScore: 11.7, Published: 2023-07-29 DOI: 10.1101/2023.07.27.550450\nDuring the last decade, generation and accumulation of petabase-scale high-throughput sequencing data have resulted in ethical and technical challenges, including access to human data, and transfer, storage, and sharing of enormous amount of data. To promote data-driven research in biology, the Korean government announced that all the biological data generated from government-funded research projects should be deposited in the Korea BioData Station (K-BDS), which consists of multiple databases for individual data types. We introduce the Korean Nucleotide Archive (KoNA), a repository for nucleotide sequence data. As of July 2022, the Korean Read Archive in KoNA has collected over 477 TB of raw next generation sequencing data from several national genome projects. To ensure data quality and prepare for international alignment, a standard operating procedure (SOP) was adopted, which is similar to the International Nucleotide Sequence Database Collaboration. The SOP includes quality control processes for submitted data and metadata using an automated pipeline followed by manual examination. To ensure fast and stable data transfer, a high-speed transmission system called GBox is used in KoNA. Furthermore, the data uploaded to or downloaded from KoNA through GBox can be readily processed in a cloud-computing service for genomic data analysis called Bio-Express. This seamless coupling of KoNA, GBox, and Bio-Express enhances data experience including submission, access, and analysis of raw nucleotide sequences. KoNA not only satisfies the unmet needs for a national sequence repository in Korea, but also provides datasets to researchers globally and contribute to advances in genomics. KoNA is available at https://www.kobic.re.kr/kona/.\nSplam: a deep-learning-based splice site predictor that improves spliced alignments\nAuthors: Chao, K.-H.; Mao, A.; Salzberg, S. L.; Pertea, M.\nScore: 10.8, Published: 2023-07-29 DOI: 10.1101/2023.07.27.550754\nThe process of splicing messenger RNA to remove introns plays a central role in creating genes and gene variants. Here we describe Splam, a novel method for predicting splice junctions in DNA based on deep residual convolutional neural networks. Unlike some previous models, Splam looks at a relatively limited window of 400 base pairs flanking each splice site, motivated by the observation that the biological process of splicing relies primarily on signals within this window. Additionally, Splam introduces the idea of training the network on donor and acceptor pairs together, based on the principal that the splicing machinery recognizes both ends of each intron at once. We compare Splams accuracy to recent state-of-the-art splice site prediction methods, particularly SpliceAI, another method that uses deep neural networks. Our results show that Splam is consistently more accurate than SpliceAI, with an overall accuracy of 96% at predicting human splice junctions. Splam generalizes even to non- human species, including distant ones like the flowering plant Arabidopsis thaliana. Finally, we demonstrate the use of Splam on a novel application: processing the spliced alignments of RNA-seq data in order to identify and eliminate errors. We show that when used in this manner, Splam yields substantial improvements in the accuracy of downstream transcriptome analysis on both poly(A) and ribo-depleted RNA-seq libraries. Overall, Splam offers a faster and more accurate approach to detecting splice junctions, while also providing a reliable and efficient solution for cleaning up erroneous spliced alignments.\nDifferential expansion and retention patterns of LRR-RLK genes across plant evolution\nAuthors: Kileeg, Z.; Haldar, A.; Khan, H.; Qamar, A.; Mott, A.\nScore: 7.9, Published: 2023-07-26 DOI: 10.1101/2023.07.26.549740\nTo maximize overall fitness, plants must accurately respond to a host of growth, developmental, and environmental signals throughout their life. Many of these internal and external signals are perceived by the leucine-rich repeat receptor-like kinases, which play roles in regulating growth, development, and immunity. This largest family of receptor kinases in plants can be divided into subfamilies based on conservation of the kinase domain, which demonstrates that shared evolutionary history often indicates shared molecular function. Here we investigate the evolutionary history of this family across the evolution of 112 plant species. We identify lineage-specific expansions of the malectin-domain containing subfamily LRR subfamily I primarily in the Brassicales and bryophytes. Most other plant lineages instead show a large expansion in LRR subfamily XII, which in Arabidopsis is known to contain key receptors in pathogen perception. This striking asymmetric expansion may reveal a dichotomy in the evolutionary history and adaptation strategies employed by plants. A greater understanding of the evolutionary pressures and adaptation strategies acting on members of this receptor family offers a way to improve functional predictions for orphan receptors and simplify identification of novel stress related receptors.\nCell type-specific consequences of mosaic structural variants in hematopoietic stem and progenitor cells\nAuthors: Grimes, K.; Jeong, H.; Amoah, A.; Xu, N.; Niemann, J.; Raeder, B.; Hasenfeld, P.; Stober, C.; Rausch, T.; Benito, E.; Jann, J.-C.; Nowak, D.; Emini, R.; Hoenicka, M.; Liebold, A.; Ho, A. D.; Shuai, S.; Geiger, H.; Sanders, A. D.; Korbel, J. O.\nScore: 7.8, Published: 2023-07-28 DOI: 10.1101/2023.07.25.550502\nThe functional impact and cellular context of mosaic structural variants (mSVs) in normal tissues is understudied. Utilizing Strand-seq, we sequenced 1,133 single cell genomes from 19 human donors of increasing age, revealing a heterogeneous mSV landscape in hematopoietic stem and progenitor cells (HSPCs). While mSV clonal expansions are confined to individuals over 60, de novo mSV formation occurs consistently across age, frequently leading to megabase-scale segmental aneuploidies. Cells harboring subclonal mosaicism show evidence for increased mSV formation. To enable high-resolution cell-typing of each Strand-seq library, we generated single-cell MNase-seq reference datasets for eight distinct HSPCs. Subclonal mSVs frequently exhibit enrichment in myeloid progenitors, and single-cell multiomic analysis suggests that these mSVs result in recurrent dysregulation of pathways related to proliferation and metabolism, including Ras signaling and lipid metabolism. The comprehensive mSV landscape identified in this study implicates mSVs in cell type-specific molecular phenotypes, establishing a foundation for deciphering links between mSVs, aging, and disease risk.\nCRISPR-Based Genome Editing in Harmonia Axyridis\nAuthors: Partosh, T.; Davidovitz, M.; Firer, N.; Pines, G.\nScore: 7.2, Published: 2023-07-27 DOI: 10.1101/2023.07.27.550814\nHarmonia axyridis (Pallas), commonly known as the Asian lady beetle, is a native insect species of Asia that has been intentionally introduced to various regions for biocontrol purposes. However, its widespread presence beyond its original release sites suggests a high degree of invasiveness. Previous studies have employed double-stranded RNA techniques to investigate the functional genomics of this species. In this study, we utilized the CRISPR-Cas9 approach to achieve precise genome editing in H. axyridis. Specifically, we targeted two distinct genes known to exhibit visible phenotypic effects. While the knockout of the laccase2 gene resulted in an early-detectable phenotype but also in lethality, we successfully established a viable and genetically stable mutant colony by disrupting the scarlet gene. Our findings contribute to the expanding knowledge of genetic manipulation in H. axyridis and provide insights into its potential for future research and practical applications for biocontrol and invasive species management.\n",
  "wordCount" : "2398",
  "inLanguage": "en",
  "datePublished": "2023-08-02T10:38:09Z",
  "dateModified": "2023-08-02T10:38:09Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on August 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549578">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549578" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549578">
        <p class="paperTitle">Concurrent profiling of multiscale 3D genome organization and gene expression in single mammalian cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549578" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549578" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, T.; Zhang, R.; Jia, D.; Doty, R. T.; Munday, A. D.; Gao, D.; Xin, L.; Abkowitz, J. L.; Duan, Z.; Ma, J.</p>
        <p class="info">Score: 49.0, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549578' target='https://doi.org/10.1101/2023.07.20.549578'> 10.1101/2023.07.20.549578</a></p>
        <p class="abstract">The organization of mammalian genomes within the nucleus features a complex, multiscale three-dimensional (3D) architecture. The functional significance of these 3D genome features, however, remains largely elusive due to limited single-cell technologies that can concurrently profile genome organization and transcriptional activities. Here, we report GAGE-seq, a highly scalable, robust single-cell co-assay that simultaneously measures 3D genome structure and transcriptome within the same cell. Employing GAGE-seq on mouse brain cortex and human bone marrow CD34&#43; cells, we comprehensively characterized the intricate relationships between 3D genome and gene expression. We found that these multiscale 3D genome features collectively inform cell type-specific gene expressions, hence contributing to defining cell identity at the single-cell level. Integration of GAGE-seq data with spatial transcriptomic data revealed in situ variations of the 3D genome in mouse cortex. Moreover, our observations of lineage commitment in normal human hematopoiesis unveiled notable discordant changes between 3D genome organization and gene expression, underscoring a complex, temporal interplay at the single-cell level that is more nuanced than previously appreciated. Together, GAGE-seq provides a powerful, cost-effective approach for interrogating genome structure and gene expression relationships at the single-cell level across diverse biological contexts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550582">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550582" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550582">
        <p class="paperTitle">Systematic assessment of long-read RNA-seq methods for transcript identification and quantification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550582" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550582" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pardo-Palacios, F. J.; Wang, D.; Reese, F.; Diekhans, M.; Carbonell-Sala, S.; Williams, B.; Loveland, J. E.; De Maria, M.; Adams, M. S.; Balderrama-Gutierrez, G.; Behera, A. K.; Gonzalez, J. M.; Hunt, T.; Lagarde, J.; Liang, C. E.; Li, H.; Meade, M. J.; Amador, D. A. M.; Prjibelski, A. D.; Birol, I.; Bostan, H.; Brooks, A. M.; Celik, M. H.; Chen, Y.; Du, M. R. M.; Felton, C.; Goke, J.; Hafezqorani, S.; Herwig, R.; Kawaji, H.; Lee, J.; Li, J.-L.; Lienhard, M.; Mikheenko, A.; Mulligan, D.; Nip, K. M.; Pertea, M.; Ritchie, M. E.; Sim, A. D.; Tang, A. D.; Wan, Y. K.; Wang, C.; Wong, B. Y.; Yang, C</p>
        <p class="info">Score: 41.1, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550582' target='https://doi.org/10.1101/2023.07.25.550582'> 10.1101/2023.07.25.550582</a></p>
        <p class="abstract">AbstractThe Long-read RNA-Seq Genome Annotation Assessment Project (LRGASP) Consortium was formed to evaluate the effectiveness of long-read approaches for transcriptome analysis. The consortium generated over 427 million long-read sequences from cDNA and direct RNA datasets, encompassing human, mouse, and manatee species, using different protocols and sequencing platforms. These data were utilized by developers to address challenges in transcript isoform detection and quantification, as well as de novo transcript isoform identification. The study revealed that libraries with longer, more accurate sequences produce more accurate transcripts than those with increased read depth, whereas greater read depth improved quantification accuracy. In well-annotated genomes, tools based on reference sequences demonstrated the best performance. When aiming to detect rare and novel transcripts or when using reference-free approaches, incorporating additional orthogonal data and replicate samples are advised. This collaborative study offers a benchmark for current practices and provides direction for future method development in transcriptome analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550902">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550902" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550902">
        <p class="paperTitle">A multiplex, prime editing framework for identifying drug resistance variants at scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550902" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550902" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chardon, F. M.; Suiter, C.; Daza, R.; Smith, N. T.; Parrish, P.; McDiarmid, T.; Lalanne, J.-B.; Martin, B. K.; Calderon, D.; Ellison, A.; Berger, A.; Shendure, J.; Starita, L. M.</p>
        <p class="info">Score: 36.8, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550902' target='https://doi.org/10.1101/2023.07.27.550902'> 10.1101/2023.07.27.550902</a></p>
        <p class="abstract">CRISPR-based genome editing has revolutionized functional genomics, enabling screens in which thousands of perturbations of either gene expression or primary genome sequence can be competitively assayed in single experiments. However, for libraries of specific mutations, a challenge of CRISPR-based screening methods such as saturation genome editing is that only one region (e.g. one exon) can be studied per experiment. Here we describe prime-SGE (&#34;prime saturation genome editing&#34;), a new framework based on prime editing, in which libraries of specific mutations can be installed into genes throughout the genome and functionally assessed in a single, multiplex experiment. Prime-SGE is based on quantifying the abundance of prime editing guide RNAs (pegRNAs) in the context of a functional selection, rather than quantifying the mutations themselves. We apply prime-SGE to assay thousands of single nucleotide changes in eight oncogenes for their ability to confer drug resistance to three EGFR tyrosine kinase inhibitors. Although currently restricted to positive selection screens by the limited efficiency of prime editing, our strategy opens the door to the possibility of functionally assaying vast numbers of precise mutations at locations throughout the genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550634">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550634" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550634">
        <p class="paperTitle">Single-cell gene expression prediction from DNA sequence at large contexts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550634" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550634" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schwessinger, R.; Deasy, J.; Woodruff, R. T.; Young, S.; Branson, K. M.</p>
        <p class="info">Score: 28.1, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550634' target='https://doi.org/10.1101/2023.07.26.550634'> 10.1101/2023.07.26.550634</a></p>
        <p class="abstract">Human genetic variants impacting traits such as disease susceptibility frequently act through modulation of gene expression in a highly cell-type-specific manner. Computational models capable of predicting gene expression directly from DNA sequence can assist in the interpretation of expression-modulating variants, and machine learning models now operate at the large sequence contexts required for capturing long-range human transcriptional regulation. However, existing predictors have focused on bulk transcriptional measurements where gene expression heterogeneity can be drowned out in broadly defined cell types. Here, we use a transfer learning framework, seq2cells, leveraging a pre-trained epigenome model for gene expression prediction from large sequence contexts at single-cell resolution. We show that seq2cells captures cell-specific gene expression beyond the resolution of pseudo-bulked data. Using seq2cells for variant effect prediction reveals heterogeneity within annotated cell types and enables in silico transfer of variant effects between cell populations. We demonstrate the challenges and value of gene expression and variant effect prediction at single-cell resolution, and offer a path to the interpretation of genomic variation at uncompromising resolution and scale.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550593">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550593" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550593">
        <p class="paperTitle">Surprising connections between DNA binding and function for the near-complete set of yeast transcription factors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550593" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550593" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mahendrawada, L.; Warfield, L.; Donczew, R.; hahn, s.</p>
        <p class="info">Score: 20.7, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550593' target='https://doi.org/10.1101/2023.07.25.550593'> 10.1101/2023.07.25.550593</a></p>
        <p class="abstract">DNA sequence-specific transcrip-on factors (TFs) modulate transcrip-on and chroma-n architecture, ac-ng from regulatory sites in enhancers and promoters of eukaryo-c genes. How TFs locate their DNA targets and how mul-ple TFs cooperate to regulate individual genes is s-ll unclear. Most yeast TFs are thought to regulate transcrip-on via binding to upstream ac-va-ng sequences, situated within a few hundred base pairs upstream of the regulated gene. While this model has been validated for individual TFs and specific genes, it has not been tested in a systema-c way with the large set of yeast TFs. Here, we have integrated informa-on on the binding and expression targets for the near-complete set of yeast TFs. While we found many instances of func-onal TF binding sites in upstream regulatory regions, we found many more instances that do not fit this model. In many cases, rapid TF deple-on affects gene expression where there is no detectable binding of that TF to the upstream region of the affected gene. In addi-on, for most TFs, only a small frac-on of bound TFs regulates the nearby gene, showing that TF binding does not automa-cally correspond to regula-on of the linked gene. Finally, we found that only a small percentage of TFs are exclusively strong ac-vators or repressors with most TFs having dual func-on. Overall, our comprehensive mapping of TF binding and regulatory targets have both confirmed known TF rela-onships and revealed surprising proper-es of TF func-on.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550450">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550450" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550450">
        <p class="paperTitle">KoNA: Korean Nucleotide Archive as a New Data Repository for Nucleotide Sequence Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550450" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550450" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ko, G.; Lee, J.; Sim, Y. M.; Song, W.; Yoon, B.-H.; Byeon, I.; Lee, B. H.; Kim, S.-O.; Choi, J.; Jang, I.; Kim, H.; Yang, J. O.; Jang, K.; Kim, S.; Kim, J.-H.; Jeon, J.; Jung, J.; Hwang, S.; Park, J.-H.; Kim, P.-G.; Kim, S.-Y.; Lee, B.</p>
        <p class="info">Score: 11.7, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550450' target='https://doi.org/10.1101/2023.07.27.550450'> 10.1101/2023.07.27.550450</a></p>
        <p class="abstract">During the last decade, generation and accumulation of petabase-scale high-throughput sequencing data have resulted in ethical and technical challenges, including access to human data, and transfer, storage, and sharing of enormous amount of data. To promote data-driven research in biology, the Korean government announced that all the biological data generated from government-funded research projects should be deposited in the Korea BioData Station (K-BDS), which consists of multiple databases for individual data types. We introduce the Korean Nucleotide Archive (KoNA), a repository for nucleotide sequence data. As of July 2022, the Korean Read Archive in KoNA has collected over 477 TB of raw next generation sequencing data from several national genome projects. To ensure data quality and prepare for international alignment, a standard operating procedure (SOP) was adopted, which is similar to the International Nucleotide Sequence Database Collaboration. The SOP includes quality control processes for submitted data and metadata using an automated pipeline followed by manual examination. To ensure fast and stable data transfer, a high-speed transmission system called GBox is used in KoNA. Furthermore, the data uploaded to or downloaded from KoNA through GBox can be readily processed in a cloud-computing service for genomic data analysis called Bio-Express. This seamless coupling of KoNA, GBox, and Bio-Express enhances data experience including submission, access, and analysis of raw nucleotide sequences. KoNA not only satisfies the unmet needs for a national sequence repository in Korea, but also provides datasets to researchers globally and contribute to advances in genomics. KoNA is available at https://www.kobic.re.kr/kona/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550754">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550754" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550754">
        <p class="paperTitle">Splam: a deep-learning-based splice site predictor that improves spliced alignments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550754" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550754" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chao, K.-H.; Mao, A.; Salzberg, S. L.; Pertea, M.</p>
        <p class="info">Score: 10.8, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550754' target='https://doi.org/10.1101/2023.07.27.550754'> 10.1101/2023.07.27.550754</a></p>
        <p class="abstract">The process of splicing messenger RNA to remove introns plays a central role in creating genes and gene variants. Here we describe Splam, a novel method for predicting splice junctions in DNA based on deep residual convolutional neural networks. Unlike some previous models, Splam looks at a relatively limited window of 400 base pairs flanking each splice site, motivated by the observation that the biological process of splicing relies primarily on signals within this window. Additionally, Splam introduces the idea of training the network on donor and acceptor pairs together, based on the principal that the splicing machinery recognizes both ends of each intron at once. We compare Splams accuracy to recent state-of-the-art splice site prediction methods, particularly SpliceAI, another method that uses deep neural networks. Our results show that Splam is consistently more accurate than SpliceAI, with an overall accuracy of 96% at predicting human splice junctions. Splam generalizes even to non- human species, including distant ones like the flowering plant Arabidopsis thaliana. Finally, we demonstrate the use of Splam on a novel application: processing the spliced alignments of RNA-seq data in order to identify and eliminate errors. We show that when used in this manner, Splam yields substantial improvements in the accuracy of downstream transcriptome analysis on both poly(A) and ribo-depleted RNA-seq libraries. Overall, Splam offers a faster and more accurate approach to detecting splice junctions, while also providing a reliable and efficient solution for cleaning up erroneous spliced alignments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.549740">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.549740" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.549740">
        <p class="paperTitle">Differential expansion and retention patterns of LRR-RLK genes across plant evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.549740" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.549740" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kileeg, Z.; Haldar, A.; Khan, H.; Qamar, A.; Mott, A.</p>
        <p class="info">Score: 7.9, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.549740' target='https://doi.org/10.1101/2023.07.26.549740'> 10.1101/2023.07.26.549740</a></p>
        <p class="abstract">To maximize overall fitness, plants must accurately respond to a host of growth, developmental, and environmental signals throughout their life. Many of these internal and external signals are perceived by the leucine-rich repeat receptor-like kinases, which play roles in regulating growth, development, and immunity. This largest family of receptor kinases in plants can be divided into subfamilies based on conservation of the kinase domain, which demonstrates that shared evolutionary history often indicates shared molecular function. Here we investigate the evolutionary history of this family across the evolution of 112 plant species. We identify lineage-specific expansions of the malectin-domain containing subfamily LRR subfamily I primarily in the Brassicales and bryophytes. Most other plant lineages instead show a large expansion in LRR subfamily XII, which in Arabidopsis is known to contain key receptors in pathogen perception. This striking asymmetric expansion may reveal a dichotomy in the evolutionary history and adaptation strategies employed by plants. A greater understanding of the evolutionary pressures and adaptation strategies acting on members of this receptor family offers a way to improve functional predictions for orphan receptors and simplify identification of novel stress related receptors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550502">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550502" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550502">
        <p class="paperTitle">Cell type-specific consequences of mosaic structural variants in hematopoietic stem and progenitor cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550502" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550502" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Grimes, K.; Jeong, H.; Amoah, A.; Xu, N.; Niemann, J.; Raeder, B.; Hasenfeld, P.; Stober, C.; Rausch, T.; Benito, E.; Jann, J.-C.; Nowak, D.; Emini, R.; Hoenicka, M.; Liebold, A.; Ho, A. D.; Shuai, S.; Geiger, H.; Sanders, A. D.; Korbel, J. O.</p>
        <p class="info">Score: 7.8, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550502' target='https://doi.org/10.1101/2023.07.25.550502'> 10.1101/2023.07.25.550502</a></p>
        <p class="abstract">The functional impact and cellular context of mosaic structural variants (mSVs) in normal tissues is understudied. Utilizing Strand-seq, we sequenced 1,133 single cell genomes from 19 human donors of increasing age, revealing a heterogeneous mSV landscape in hematopoietic stem and progenitor cells (HSPCs). While mSV clonal expansions are confined to individuals over 60, de novo mSV formation occurs consistently across age, frequently leading to megabase-scale segmental aneuploidies. Cells harboring subclonal mosaicism show evidence for increased mSV formation. To enable high-resolution cell-typing of each Strand-seq library, we generated single-cell MNase-seq reference datasets for eight distinct HSPCs. Subclonal mSVs frequently exhibit enrichment in myeloid progenitors, and single-cell multiomic analysis suggests that these mSVs result in recurrent dysregulation of pathways related to proliferation and metabolism, including Ras signaling and lipid metabolism. The comprehensive mSV landscape identified in this study implicates mSVs in cell type-specific molecular phenotypes, establishing a foundation for deciphering links between mSVs, aging, and disease risk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550814">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550814" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550814">
        <p class="paperTitle">CRISPR-Based Genome Editing in Harmonia Axyridis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550814" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550814" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Partosh, T.; Davidovitz, M.; Firer, N.; Pines, G.</p>
        <p class="info">Score: 7.2, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550814' target='https://doi.org/10.1101/2023.07.27.550814'> 10.1101/2023.07.27.550814</a></p>
        <p class="abstract">Harmonia axyridis (Pallas), commonly known as the Asian lady beetle, is a native insect species of Asia that has been intentionally introduced to various regions for biocontrol purposes. However, its widespread presence beyond its original release sites suggests a high degree of invasiveness. Previous studies have employed double-stranded RNA techniques to investigate the functional genomics of this species. In this study, we utilized the CRISPR-Cas9 approach to achieve precise genome editing in H. axyridis. Specifically, we targeted two distinct genes known to exhibit visible phenotypic effects. While the knockout of the laccase2 gene resulted in an early-detectable phenotype but also in lethality, we successfully established a viable and genetically stable mutant colony by disrupting the scarlet gene. Our findings contribute to the expanding knowledge of genetic manipulation in H. axyridis and provide insights into its potential for future research and practical applications for biocontrol and invasive species management.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
