<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://yorks0n.github.io/TRxiv2" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/TRxiv2/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Identification of a novel HKU4-related coronavirus in single-cell datasets and clade viral host analysis
Authors: Jones, A.; Massey, S. E.; Nemzer, L. R.; Zhang, D.; Deigin, Y.; Quay, S. C.
Score: 49.4, Published: 2023-06-21 DOI: 10.1101/2023.06.18.545480
HKU4-related coronaviruses (CoVs) are merbecoviruses related to Middle Eastern Respiratory Syndrome coronavirus (MERS-CoV). In 2022 and 2023, two HKU4-related CoV strains were discovered in Manis javanica (Malayan pangolin) metagenomic datasets derived from organ samples: HKU4-P251T and MjHKU4r-CoV-1.">
<meta name="author" content="">
<link rel="canonical" href="https://yorks0n.github.io/TRxiv2/posts/genomics/">
<link crossorigin="anonymous" href="/TRxiv2/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/TRxiv2/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://yorks0n.github.io/TRxiv2/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://yorks0n.github.io/TRxiv2/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://yorks0n.github.io/TRxiv2/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://yorks0n.github.io/TRxiv2/apple-touch-icon.png">
<link rel="mask-icon" href="https://yorks0n.github.io/TRxiv2/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Identification of a novel HKU4-related coronavirus in single-cell datasets and clade viral host analysis
Authors: Jones, A.; Massey, S. E.; Nemzer, L. R.; Zhang, D.; Deigin, Y.; Quay, S. C.
Score: 49.4, Published: 2023-06-21 DOI: 10.1101/2023.06.18.545480
HKU4-related coronaviruses (CoVs) are merbecoviruses related to Middle Eastern Respiratory Syndrome coronavirus (MERS-CoV). In 2022 and 2023, two HKU4-related CoV strains were discovered in Manis javanica (Malayan pangolin) metagenomic datasets derived from organ samples: HKU4-P251T and MjHKU4r-CoV-1." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://yorks0n.github.io/TRxiv2/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T09:51:07+00:00" />
<meta property="article:modified_time" content="2023-06-28T09:51:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Identification of a novel HKU4-related coronavirus in single-cell datasets and clade viral host analysis
Authors: Jones, A.; Massey, S. E.; Nemzer, L. R.; Zhang, D.; Deigin, Y.; Quay, S. C.
Score: 49.4, Published: 2023-06-21 DOI: 10.1101/2023.06.18.545480
HKU4-related coronaviruses (CoVs) are merbecoviruses related to Middle Eastern Respiratory Syndrome coronavirus (MERS-CoV). In 2022 and 2023, two HKU4-related CoV strains were discovered in Manis javanica (Malayan pangolin) metagenomic datasets derived from organ samples: HKU4-P251T and MjHKU4r-CoV-1."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://yorks0n.github.io/TRxiv2/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://yorks0n.github.io/TRxiv2/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Identification of a novel HKU4-related coronavirus in single-cell datasets and clade viral host analysis\nAuthors: Jones, A.; Massey, S. E.; Nemzer, L. R.; Zhang, D.; Deigin, Y.; Quay, S. C.\nScore: 49.4, Published: 2023-06-21 DOI: 10.1101/2023.06.18.545480\nHKU4-related coronaviruses (CoVs) are merbecoviruses related to Middle Eastern Respiratory Syndrome coronavirus (MERS-CoV). In 2022 and 2023, two HKU4-related CoV strains were discovered in Manis javanica (Malayan pangolin) metagenomic datasets derived from organ samples: HKU4-P251T and MjHKU4r-CoV-1.",
  "keywords": [
    
  ],
  "articleBody": " Identification of a novel HKU4-related coronavirus in single-cell datasets and clade viral host analysis\nAuthors: Jones, A.; Massey, S. E.; Nemzer, L. R.; Zhang, D.; Deigin, Y.; Quay, S. C.\nScore: 49.4, Published: 2023-06-21 DOI: 10.1101/2023.06.18.545480\nHKU4-related coronaviruses (CoVs) are merbecoviruses related to Middle Eastern Respiratory Syndrome coronavirus (MERS-CoV). In 2022 and 2023, two HKU4-related CoV strains were discovered in Manis javanica (Malayan pangolin) metagenomic datasets derived from organ samples: HKU4-P251T and MjHKU4r-CoV-1. Together with the Tylonycteris robustula bat CoV 162275, which was discovered in 2022, pangolin CoVs HKU4-P251T and MjHKU4r-CoV-1 form a novel phylogenetic clade distinct from all previously documented HKU4-related CoVs. In this study, we identified a novel HKU4-related CoV in a pangolin single-cell sequencing dataset generated by BGI-Shenzhen in Shenzhen, Guangdong, China in 2020. The CoV phylogenetically belongs to the same newly identified clade. The single cell datasets were reported as generated from organ samples of a single pangolin that died of natural causes. 98% of the HKU4-related CoV reads were found in only one of the seven single cell datasets -- a large intestine cell dataset, cells of which exhibit low expression of DPP4. Bacterial contamination was found to be moderately correlated with HKU4-related CoV presence. We further identified with high confidence that the RNA-Seq dataset supporting one of four near identical variants of MjHKU4r-CoV-1 is a Sus scrofa (wild pig) metagenomic dataset, with only a trace level of Manis javanica genomic content. The presence of HKU4-related CoV reads in the dataset are almost certainly laboratory research-related and not from a premortal pangolin or pig infection. Our findings raise concerns about the provenance of the novel HKU4-related CoV we identify here, MjHKU4r-CoV-1 and its four near-identical variants.\nDark side of the honeymoon: reconstructing the Asian x European rose breeding history through the lens of genomics\nAuthors: Leroy, T.; Albert, E.; Thouroude, T.; Baudino, S.; Caissard, J.-C.; Chastellier, A.; Chameau, J.; Jeauffre, J.; Loubert, T.; Paramita, S. N.; Pernet, A.; Soufflet-Freslon, V.; Oghina-Pavie, C.; Foucher, F.; Oyant, L. H.-S.; Clotault, J.\nScore: 33.7, Published: 2023-06-26 DOI: 10.1101/2023.06.22.546162\nRoses have a specific symbolic charge in Western cultural heritage, often used as a metaphor for love and romance. Despite its ancient cultivation, the passion for the phenotypic diversity of roses is relatively recent, dating back to the early 19th century. During that century, the number of rose varieties has increased exponentially from roughly 100 to 8,000, in such a way that this period can be considered as the golden age for rose breeding. To retrace the history of rose breeding in Europe and shed new light on genetic changes during this period, we collected large phenotypic and genetic data from 204 accessions, including botanical roses and varieties bred between 1800 and 1910. We also used whole-genome sequences from 32 accessions as an extra resource. The genetic makeup of roses is remarkably consistent with a temporal shift from a historical European to a near-Asian genetic background within a few generations. Our analyses are consistent with a substantial erosion of the genetic diversity during this period, both because of the greater contribution of the Asian genepool - a less genetically diverse group - and of specific genomic footprints of selection, in particular regarding the extension of the blooming period. Thanks to this study, we have generated the largest GWAS catalog for roses to date, which can be used as a tool for future rose breeding programs. We particularly discuss the crucial importance of preserving ancient rose collections to safeguard genetic diversity and ensure a sustainable breeding for the long-term.\nAccounting for statistical non-additive interactions enables the recovery of missing heritability from GWAS summary statistics\nAuthors: Smith, S. P.; Darnell, G.; Udwin, D.; Harpak, A.; Ramachandran, S.; Crawford, L.\nScore: 30.2, Published: 2023-06-23 DOI: 10.1101/2022.07.21.501001\nLD score regression (LDSC) is a method to estimate narrow-sense heritability from genome-wide association study (GWAS) summary statistics alone, making it a fast and popular approach. The key concept underlying the LDSC framework is that there is a positive linear relationship between the magnitude of GWAS allelic effect estimates and linkage disequilibrium (LD) when complex traits are generated under the infinitesimal model -- that is, causal variants are uniformly distributed along the genome and each have the same expected contribution to phenotypic variation. We present interaction-LD score (i-LDSC) regression: an extension of the original LDSC framework that accounts for non-additive genetic effects. By studying a wide range of generative models in simulations, and by re-analyzing 25 well-studied quantitative phenotypes from 349,468 individuals in the UK Biobank and up to 159,095 individuals in BioBank Japan, we show that the inclusion of a cis-interaction score (i.e., interactions between a focal variant and nearby variants) significantly recovers substantial non-additive heritability that is not captured by LDSC. For each of the 25 traits analyzed in the UK Biobank and 23 of the 25 traits analyzed in BioBank Japan, i-LDSC detects a significant amount of variation contributed by genetic interactions. The i-LDSC software and its application to these biobanks represent a step towards resolving further genetic contributions of sources of non-additive genetic effects to complex trait variation.\nA Risk-reward Examination of Sample Multiplexing Reagents for Single Cell RNA-Seq\nAuthors: Brown, D. V.; Anttila, C. J. A.; Ling, L.; Grave, P.; Baldwin, T. M.; Munnings, R.; Farchione, A. J.; Bryant, V. L.; Dunstone, A.; Biben, C.; Taoudi, S.; Weber, T. S.; Naik, S. H.; Hadla, A.; Barker, H. E.; Vandenberg, C. J.; Dall, G.; Scott, C. L.; Moore, Z.; Whittle, J. R.; Freytag, S.; Best, S. A.; Papenfuss, A. T.; Olechnowicz, S. W. Z.; MacRaild, S. E.; Wilcox, S.; Hickey, P. F.; Amann-Zalcenstein, D.; Bowden, R.\nScore: 24.4, Published: 2023-06-23 DOI: 10.1101/2023.06.20.544880\nSingle-cell RNA sequencing (scRNA-Seq) has emerged as a powerful tool for understanding cellular heterogeneity and function. However the choice of sample multiplexing reagents can impact data quality and experimental outcomes. In this study, we compared various multiplexing reagents, including MULTI-Seq, Hashtag antibody, and CellPlex, across diverse sample types such as human peripheral blood mononuclear cells (PBMCs), mouse embryonic brain and patient-derived xenografts (PDXs). We found that all multiplexing reagents worked well in cell types robust to ex vivo manipulation but suffered from signal-to-noise issues in more delicate sample types. We compared multiple demultiplexing algorithms which differed in performance depending on data quality. We find that minor improvements to laboratory workflows such as titration and rapid processing are critical to optimal performance. We also compared the performance of fixed scRNA-Seq kits and highlight the advantages of the Parse Biosciences kit for fragile samples. Highly multiplexed scRNA-Seq experiments require more sequencing resources, therefore we evaluated CRISPR-based destruction of non-informative genes to enhance sequencing value. Our comprehensive analysis provides insights into the selection of appropriate sample multiplexing reagents and protocols for scRNASeq experiments, facilitating more accurate and cost-effective studies.\nCapTrap-Seq: A platform-agnostic and quantitative approach for high-fidelity full-length RNA transcript sequencing\nAuthors: Carbonell-Sala, S.; Lagarde, J.; Nishiyori, H.; Palumbo, E.; Arnan, C.; Takahashi, H.; Carninci, P.; Uszczynska-Ratajczak, B.; Guigo, R.\nScore: 25.6, Published: 2023-06-18 DOI: 10.1101/2023.06.16.543444\nLong-read RNA sequencing is essential to produce accurate and exhaustive annotation of eukaryotic genomes. Despite advancements in throughput and accuracy, achieving reliable end-to-end identification of RNA transcripts remains a challenge for long-read sequencing methods. To address this limitation, we developed CapTrap-seq, a cDNA library preparation method, which combines the Cap-trapping strategy with oligo(dT) priming to detect 5capped, full-length transcripts, together with the data processing pipeline LyRic. We benchmarked CapTrap-seq and other popular RNA-seq library preparation protocols in a number of human tissues using both ONT and PacBio sequencing. To assess the accuracy of the transcript models produced, we introduced a capping strategy for synthetic RNA spike-in sequences that mimics the natural 5cap formation in RNA spike-in molecules. We found that the vast majority (up to 90%) of transcript models that LyRic derives from CapTrap-seq reads are full-length. This makes it possible to produce highly accurate annotations with minimal human intervention.\nComparative 3D genome analysis between neural retina and RPE reveals differential cis-regulatory interactions at retinal disease loci\nAuthors: D'haene, E.; Soriano, V. L.; Martinez-Garcia, P. M.; Kalayanamontri, S.; Rey, A. D.; Sousa-Ortega, A.; Naranjo, S.; Van de Sompele, S.; Vantomme, L.; Mahieu, Q.; Vergult, S.; Neto, A. B.; Gomez-Skarmeta, J. L.; Martinez-Morales, J. R.; Bauwens, M.; Tena, J. J.; De Baere, E.\nScore: 17.5, Published: 2023-06-21 DOI: 10.1101/2023.06.20.543842\nVision depends on the functional interplay between the photoreceptor cells of the neural retina and the supporting cells of the underlying retinal pigment epithelium (RPE). Most genes involved in inherited retinal diseases (IRD) display highly specific spatiotemporal expression within these interconnected retinal components through the local recruitment of cis-regulatory elements (CREs) in 3D nuclear space. To understand the role of differential chromatin architecture in establishing tissue-specific expression patterns at IRD loci in the human neural retina and the RPE, we mapped genome-wide chromatin interactions by applying in situ Hi-C and H3K4me3 HiChIP to human adult post-mortem donor retinas. A comparative 3D genome analysis between neural retina and RPE/choroid revealed that almost 60% of 290 known IRD genes were marked by differential 3D genome structure and/or cis-regulatory interactions. One of these genes was ABCA4, which is implicated in the most common autosomal recessive IRD. We zoomed in on tissue-specific chromatin interactions at the ABCA4 locus using high-resolution UMI-4C assays. Upon integration with bulk and single-cell epigenomic datasets and in vivo enhancer assays in zebrafish, we revealed tissue-specific CREs interacting with ABCA4. In summary, through extensive comparative 3D genome mapping, based on genome-wide (Hi-C), promoter-centric (HiChIP) and locus-specific (UMI-4C) assays of human neural retina and RPE, we have shown that gene regulation at key IRD loci is likely mediated by tissue-specific chromatin interactions. These findings do not only provide insight into tissue-specific regulatory landscapes of IRD genes, but also delineate the search space for non-coding genomic variation underlying unsolved IRD. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=75 SRC=\"FIGDIR/small/543842v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (22K): org.highwire.dtl.DTLVardef@647869org.highwire.dtl.DTLVardef@1d4515org.highwire.dtl.DTLVardef@1a291deorg.highwire.dtl.DTLVardef@1693fe_HPS_FORMAT_FIGEXP M_FIG C_FIG\nTradeoffs in Modeling Context Dependency in Complex Trait Genetics\nAuthors: Weine, E.; Smith, S. P.; Knowlton, R. K.; Harpak, A.\nScore: 15.2, Published: 2023-06-23 DOI: 10.1101/2023.06.21.545998\nGenetic effects on complex traits may depend on context, such as age, sex, environmental exposures or social settings. However, it is often unclear if the extent of context dependency, or Gene-by-Environment interaction (GxE), merits more involved models than the additive model typically used to analyze data from genomewide association studies (GWAS). Here, we derive governing principles for when modeling GxE in GWAS is useful. We derive a decision rule for choosing between competing models for the estimation of allelic effects. The rule weighs the increased noise of context-specific effect estimation against the potential bias when context dependency is ignored. In human GWAS, the increased noise of context-specific estimation often outweighs the bias reduction, rendering GxE models less useful when variants are considered independently. However, for complex traits, considering patterns of context dependency across many variants jointly can substantially improve estimation and trait prediction. Finally, we demonstrate via example that considering polygenic trends of GxE may also be important for interpretation, as analyses based on independently ascertained \"top hits\" alone can be misleading.\nGenome-wide kinetic profiling of pre-mRNA 3' end cleavage\nAuthors: Torres-Ulloa, L.; Calvo-Roitberg, E.; Pai, A. A.\nScore: 13.2, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545926\nCleavage and polyadenylation is necessary for the formation of mature mRNA molecules. The rate at which this process occurs can determine the temporal availability of mRNA for subsequent function throughout the cell and is likely tightly regulated. Despite advances in high-throughput approaches for global kinetic profiling of RNA maturation, genome-wide 3 end cleavage rates have never been measured. Here, we describe a novel approach to estimate the rates of cleavage, using metabolic labeling of nascent RNA, high-throughput sequencing, and mathematical modeling. Using in-silico simulations of nascent RNA-seq data, we show that our approach can accurately and precisely estimate cleavage half-lives for both constitutive and alternative sites. We find that 3 end cleavage is fast on average, with half-lives under a minute, but highly variable across individual sites. Rapid cleavage is promoted by the presence of canonical sequence elements and an increased density of polyadenylation signals near a cleavage site. Finally, we find that cleavage rates are associated with the localization of RNA Polymerase II at the end of a gene and faster cleavage leads to quicker degradation of downstream read-through RNA. Our findings shed light on the features important for efficient 3 end cleavage and the regulation of transcription termination.\nTranscriptional profiling of aging tissues from female and male African turquoise killifish\nAuthors: Xu, A.; Teefy, B. B.; Lu, R. J.; Nozownik, S.; Tyers, A. M.; Valenzano, D. R.; Benayoun, B. A.\nScore: 11.3, Published: 2023-06-24 DOI: 10.1101/2023.06.20.545766\nThe African turquoise killifish is an emerging vertebrate model organism with great potential for aging research due to its naturally short lifespan. Thus far, turquoise killifish aging omic studies using RNA-seq have examined a single organ, single sex and/or evaluated samples from non-reference strains. Here, we describe a resource dataset of ribosomal RNA depleted RNA-seq libraries generated from the brain, heart, muscle, and spleen from both sexes, as well as young and old animals, in the reference GRZ turquoise killifish strain. We provide basic quality control steps and demonstrate the utility of our dataset by performing differential gene expression and gene ontology analyses by age and sex. Importantly, we show that age has a greater impact than sex on transcriptional landscapes across probed tissues. Finally, we confirm transcription of transposable elements (TEs), which are highly abundant and increase in expression with age in brain tissue. This dataset will be a useful resource for exploring gene and TE expression as a function of both age and sex in a powerful naturally short-lived vertebrate model.\nThe Tabulae Paralytica: Multimodal single-cell and spatial atlases of spinal cord injury\nAuthors: Skinnider, M. A.; Gautier, M.; Teo, A. Y. Y.; Kathe, C.; Hutson, T.; Laskaratos, A.; de Coucy, A.; Regazzi, N.; Aureli, V.; James, N. D.; Schneider, B.; Sofroniew, M.; Barraud, Q.; Bloch, J.; Anderson, M. A.; Squair, J.; Courtine, G.\nScore: 10.8, Published: 2023-06-24 DOI: 10.1101/2023.06.23.544348\nHere, we introduce the Tabulae Paralytica - a compilation of four atlases of spinal cord injury (SCI) comprising a single-nucleus transcriptome atlas of half a million cells; a multiome atlas pairing transcriptomic and epigenomic measurements within the same nuclei; and two spatial transcriptomic atlases of the injured spinal cord spanning four spatial and temporal dimensions. We integrated these atlases into a common framework to dissect the molecular logic that governs the responses to injury within the spinal cord. The Tabulae Paralytica exposed new biological principles that dictate the consequences of SCI, including conserved and divergent neuronal responses to injury; the priming of specific neuronal subpopulations to become circuit-reorganizing neurons after injury; an inherent trade-off between neuronal stress responses and the activation of circuit reorganization programs; the necessity of reestablishing a tripartite neuroprotective barrier between immune-privileged and extra-neural environments after SCI; and a catastrophic failure to form this barrier in old mice. We leveraged the Tabulae Paralytica to develop a rejuvenative gene therapy that reestablished this tripartite barrier, and restored the natural recovery of walking after paralysis in old mice. The Tabulae Paralytica provides an unprecedented window into the pathobiology of SCI, while establishing a framework for integrating multimodal, genome-scale measurements in four dimensions to study biology and medicine.\n",
  "wordCount" : "2540",
  "inLanguage": "en",
  "datePublished": "2023-06-28T09:51:07Z",
  "dateModified": "2023-06-28T09:51:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://yorks0n.github.io/TRxiv2/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://yorks0n.github.io/TRxiv2/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://yorks0n.github.io/TRxiv2" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.18.545480">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.18.545480" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.18.545480">
        <p class="paperTitle">Identification of a novel HKU4-related coronavirus in single-cell datasets and clade viral host analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.18.545480" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.18.545480" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jones, A.; Massey, S. E.; Nemzer, L. R.; Zhang, D.; Deigin, Y.; Quay, S. C.</p>
        <p class="info">Score: 49.4, Published: 2023-06-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.18.545480' target='https://doi.org/10.1101/2023.06.18.545480'> 10.1101/2023.06.18.545480</a></p>
        <p class="abstract">HKU4-related coronaviruses (CoVs) are merbecoviruses related to Middle Eastern Respiratory Syndrome coronavirus (MERS-CoV). In 2022 and 2023, two HKU4-related CoV strains were discovered in Manis javanica (Malayan pangolin) metagenomic datasets derived from organ samples: HKU4-P251T and MjHKU4r-CoV-1. Together with the Tylonycteris robustula bat CoV 162275, which was discovered in 2022, pangolin CoVs HKU4-P251T and MjHKU4r-CoV-1 form a novel phylogenetic clade distinct from all previously documented HKU4-related CoVs. In this study, we identified a novel HKU4-related CoV in a pangolin single-cell sequencing dataset generated by BGI-Shenzhen in Shenzhen, Guangdong, China in 2020. The CoV phylogenetically belongs to the same newly identified clade. The single cell datasets were reported as generated from organ samples of a single pangolin that died of natural causes. 98% of the HKU4-related CoV reads were found in only one of the seven single cell datasets -- a large intestine cell dataset, cells of which exhibit low expression of DPP4. Bacterial contamination was found to be moderately correlated with HKU4-related CoV presence. We further identified with high confidence that the RNA-Seq dataset supporting one of four near identical variants of MjHKU4r-CoV-1 is a Sus scrofa (wild pig) metagenomic dataset, with only a trace level of Manis javanica genomic content. The presence of HKU4-related CoV reads in the dataset are almost certainly laboratory research-related and not from a premortal pangolin or pig infection. Our findings raise concerns about the provenance of the novel HKU4-related CoV we identify here, MjHKU4r-CoV-1 and its four near-identical variants.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.22.546162">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.22.546162" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.22.546162">
        <p class="paperTitle">Dark side of the honeymoon: reconstructing the Asian x European rose breeding history through the lens of genomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.22.546162" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.22.546162" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leroy, T.; Albert, E.; Thouroude, T.; Baudino, S.; Caissard, J.-C.; Chastellier, A.; Chameau, J.; Jeauffre, J.; Loubert, T.; Paramita, S. N.; Pernet, A.; Soufflet-Freslon, V.; Oghina-Pavie, C.; Foucher, F.; Oyant, L. H.-S.; Clotault, J.</p>
        <p class="info">Score: 33.7, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.22.546162' target='https://doi.org/10.1101/2023.06.22.546162'> 10.1101/2023.06.22.546162</a></p>
        <p class="abstract">Roses have a specific symbolic charge in Western cultural heritage, often used as a metaphor for love and romance. Despite its ancient cultivation, the passion for the phenotypic diversity of roses is relatively recent, dating back to the early 19th century. During that century, the number of rose varieties has increased exponentially from roughly 100 to 8,000, in such a way that this period can be considered as the golden age for rose breeding. To retrace the history of rose breeding in Europe and shed new light on genetic changes during this period, we collected large phenotypic and genetic data from 204 accessions, including botanical roses and varieties bred between 1800 and 1910. We also used whole-genome sequences from 32 accessions as an extra resource. The genetic makeup of roses is remarkably consistent with a temporal shift from a historical European to a near-Asian genetic background within a few generations. Our analyses are consistent with a substantial erosion of the genetic diversity during this period, both because of the greater contribution of the Asian genepool - a less genetically diverse group - and of specific genomic footprints of selection, in particular regarding the extension of the blooming period. Thanks to this study, we have generated the largest GWAS catalog for roses to date, which can be used as a tool for future rose breeding programs. We particularly discuss the crucial importance of preserving ancient rose collections to safeguard genetic diversity and ensure a sustainable breeding for the long-term.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.07.21.501001">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.07.21.501001" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.07.21.501001">
        <p class="paperTitle">Accounting for statistical non-additive interactions enables the recovery of missing heritability from GWAS summary statistics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.07.21.501001" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.07.21.501001" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smith, S. P.; Darnell, G.; Udwin, D.; Harpak, A.; Ramachandran, S.; Crawford, L.</p>
        <p class="info">Score: 30.2, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.07.21.501001' target='https://doi.org/10.1101/2022.07.21.501001'> 10.1101/2022.07.21.501001</a></p>
        <p class="abstract">LD score regression (LDSC) is a method to estimate narrow-sense heritability from genome-wide association study (GWAS) summary statistics alone, making it a fast and popular approach. The key concept underlying the LDSC framework is that there is a positive linear relationship between the magnitude of GWAS allelic effect estimates and linkage disequilibrium (LD) when complex traits are generated under the infinitesimal model -- that is, causal variants are uniformly distributed along the genome and each have the same expected contribution to phenotypic variation. We present interaction-LD score (i-LDSC) regression: an extension of the original LDSC framework that accounts for non-additive genetic effects. By studying a wide range of generative models in simulations, and by re-analyzing 25 well-studied quantitative phenotypes from 349,468 individuals in the UK Biobank and up to 159,095 individuals in BioBank Japan, we show that the inclusion of a cis-interaction score (i.e., interactions between a focal variant and nearby variants) significantly recovers substantial non-additive heritability that is not captured by LDSC. For each of the 25 traits analyzed in the UK Biobank and 23 of the 25 traits analyzed in BioBank Japan, i-LDSC detects a significant amount of variation contributed by genetic interactions. The i-LDSC software and its application to these biobanks represent a step towards resolving further genetic contributions of sources of non-additive genetic effects to complex trait variation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.544880">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.544880" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.544880">
        <p class="paperTitle">A Risk-reward Examination of Sample Multiplexing Reagents for Single Cell RNA-Seq</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.544880" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.544880" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brown, D. V.; Anttila, C. J. A.; Ling, L.; Grave, P.; Baldwin, T. M.; Munnings, R.; Farchione, A. J.; Bryant, V. L.; Dunstone, A.; Biben, C.; Taoudi, S.; Weber, T. S.; Naik, S. H.; Hadla, A.; Barker, H. E.; Vandenberg, C. J.; Dall, G.; Scott, C. L.; Moore, Z.; Whittle, J. R.; Freytag, S.; Best, S. A.; Papenfuss, A. T.; Olechnowicz, S. W. Z.; MacRaild, S. E.; Wilcox, S.; Hickey, P. F.; Amann-Zalcenstein, D.; Bowden, R.</p>
        <p class="info">Score: 24.4, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.544880' target='https://doi.org/10.1101/2023.06.20.544880'> 10.1101/2023.06.20.544880</a></p>
        <p class="abstract">Single-cell RNA sequencing (scRNA-Seq) has emerged as a powerful tool for understanding cellular heterogeneity and function. However the choice of sample multiplexing reagents can impact data quality and experimental outcomes. In this study, we compared various multiplexing reagents, including MULTI-Seq, Hashtag antibody, and CellPlex, across diverse sample types such as human peripheral blood mononuclear cells (PBMCs), mouse embryonic brain and patient-derived xenografts (PDXs). We found that all multiplexing reagents worked well in cell types robust to ex vivo manipulation but suffered from signal-to-noise issues in more delicate sample types. We compared multiple demultiplexing algorithms which differed in performance depending on data quality. We find that minor improvements to laboratory workflows such as titration and rapid processing are critical to optimal performance. We also compared the performance of fixed scRNA-Seq kits and highlight the advantages of the Parse Biosciences kit for fragile samples. Highly multiplexed scRNA-Seq experiments require more sequencing resources, therefore we evaluated CRISPR-based destruction of non-informative genes to enhance sequencing value. Our comprehensive analysis provides insights into the selection of appropriate sample multiplexing reagents and protocols for scRNASeq experiments, facilitating more accurate and cost-effective studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.16.543444">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.16.543444" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.16.543444">
        <p class="paperTitle">CapTrap-Seq: A platform-agnostic and quantitative approach for high-fidelity full-length RNA transcript sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.16.543444" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.16.543444" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Carbonell-Sala, S.; Lagarde, J.; Nishiyori, H.; Palumbo, E.; Arnan, C.; Takahashi, H.; Carninci, P.; Uszczynska-Ratajczak, B.; Guigo, R.</p>
        <p class="info">Score: 25.6, Published: 2023-06-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.16.543444' target='https://doi.org/10.1101/2023.06.16.543444'> 10.1101/2023.06.16.543444</a></p>
        <p class="abstract">Long-read RNA sequencing is essential to produce accurate and exhaustive annotation of eukaryotic genomes. Despite advancements in throughput and accuracy, achieving reliable end-to-end identification of RNA transcripts remains a challenge for long-read sequencing methods. To address this limitation, we developed CapTrap-seq, a cDNA library preparation method, which combines the Cap-trapping strategy with oligo(dT) priming to detect 5capped, full-length transcripts, together with the data processing pipeline LyRic. We benchmarked CapTrap-seq and other popular RNA-seq library preparation protocols in a number of human tissues using both ONT and PacBio sequencing. To assess the accuracy of the transcript models produced, we introduced a capping strategy for synthetic RNA spike-in sequences that mimics the natural 5cap formation in RNA spike-in molecules. We found that the vast majority (up to 90%) of transcript models that LyRic derives from CapTrap-seq reads are full-length. This makes it possible to produce highly accurate annotations with minimal human intervention.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.543842">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.543842" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.543842">
        <p class="paperTitle">Comparative 3D genome analysis between neural retina and RPE reveals differential cis-regulatory interactions at retinal disease loci</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.543842" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.543842" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: D&#39;haene, E.; Soriano, V. L.; Martinez-Garcia, P. M.; Kalayanamontri, S.; Rey, A. D.; Sousa-Ortega, A.; Naranjo, S.; Van de Sompele, S.; Vantomme, L.; Mahieu, Q.; Vergult, S.; Neto, A. B.; Gomez-Skarmeta, J. L.; Martinez-Morales, J. R.; Bauwens, M.; Tena, J. J.; De Baere, E.</p>
        <p class="info">Score: 17.5, Published: 2023-06-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.543842' target='https://doi.org/10.1101/2023.06.20.543842'> 10.1101/2023.06.20.543842</a></p>
        <p class="abstract">Vision depends on the functional interplay between the photoreceptor cells of the neural retina and the supporting cells of the underlying retinal pigment epithelium (RPE). Most genes involved in inherited retinal diseases (IRD) display highly specific spatiotemporal expression within these interconnected retinal components through the local recruitment of cis-regulatory elements (CREs) in 3D nuclear space.

To understand the role of differential chromatin architecture in establishing tissue-specific expression patterns at IRD loci in the human neural retina and the RPE, we mapped genome-wide chromatin interactions by applying in situ Hi-C and H3K4me3 HiChIP to human adult post-mortem donor retinas. A comparative 3D genome analysis between neural retina and RPE/choroid revealed that almost 60% of 290 known IRD genes were marked by differential 3D genome structure and/or cis-regulatory interactions. One of these genes was ABCA4, which is implicated in the most common autosomal recessive IRD. We zoomed in on tissue-specific chromatin interactions at the ABCA4 locus using high-resolution UMI-4C assays. Upon integration with bulk and single-cell epigenomic datasets and in vivo enhancer assays in zebrafish, we revealed tissue-specific CREs interacting with ABCA4.

In summary, through extensive comparative 3D genome mapping, based on genome-wide (Hi-C), promoter-centric (HiChIP) and locus-specific (UMI-4C) assays of human neural retina and RPE, we have shown that gene regulation at key IRD loci is likely mediated by tissue-specific chromatin interactions. These findings do not only provide insight into tissue-specific regulatory landscapes of IRD genes, but also delineate the search space for non-coding genomic variation underlying unsolved IRD.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=75 SRC=&#34;FIGDIR/small/543842v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (22K):
org.highwire.dtl.DTLVardef@647869org.highwire.dtl.DTLVardef@1d4515org.highwire.dtl.DTLVardef@1a291deorg.highwire.dtl.DTLVardef@1693fe_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545998">
        <p class="paperTitle">Tradeoffs in Modeling Context Dependency in Complex Trait Genetics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weine, E.; Smith, S. P.; Knowlton, R. K.; Harpak, A.</p>
        <p class="info">Score: 15.2, Published: 2023-06-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545998' target='https://doi.org/10.1101/2023.06.21.545998'> 10.1101/2023.06.21.545998</a></p>
        <p class="abstract">Genetic effects on complex traits may depend on context, such as age, sex, environmental exposures or social settings. However, it is often unclear if the extent of context dependency, or Gene-by-Environment interaction (GxE), merits more involved models than the additive model typically used to analyze data from genomewide association studies (GWAS). Here, we derive governing principles for when modeling GxE in GWAS is useful. We derive a decision rule for choosing between competing models for the estimation of allelic effects. The rule weighs the increased noise of context-specific effect estimation against the potential bias when context dependency is ignored. In human GWAS, the increased noise of context-specific estimation often outweighs the bias reduction, rendering GxE models less useful when variants are considered independently. However, for complex traits, considering patterns of context dependency across many variants jointly can substantially improve estimation and trait prediction. Finally, we demonstrate via example that considering polygenic trends of GxE may also be important for interpretation, as analyses based on independently ascertained &#34;top hits&#34; alone can be misleading.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545926">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545926" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545926">
        <p class="paperTitle">Genome-wide kinetic profiling of pre-mRNA 3&#39; end cleavage</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545926" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545926" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Torres-Ulloa, L.; Calvo-Roitberg, E.; Pai, A. A.</p>
        <p class="info">Score: 13.2, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545926' target='https://doi.org/10.1101/2023.06.21.545926'> 10.1101/2023.06.21.545926</a></p>
        <p class="abstract">Cleavage and polyadenylation is necessary for the formation of mature mRNA molecules. The rate at which this process occurs can determine the temporal availability of mRNA for subsequent function throughout the cell and is likely tightly regulated. Despite advances in high-throughput approaches for global kinetic profiling of RNA maturation, genome-wide 3 end cleavage rates have never been measured. Here, we describe a novel approach to estimate the rates of cleavage, using metabolic labeling of nascent RNA, high-throughput sequencing, and mathematical modeling. Using in-silico simulations of nascent RNA-seq data, we show that our approach can accurately and precisely estimate cleavage half-lives for both constitutive and alternative sites. We find that 3 end cleavage is fast on average, with half-lives under a minute, but highly variable across individual sites. Rapid cleavage is promoted by the presence of canonical sequence elements and an increased density of polyadenylation signals near a cleavage site. Finally, we find that cleavage rates are associated with the localization of RNA Polymerase II at the end of a gene and faster cleavage leads to quicker degradation of downstream read-through RNA. Our findings shed light on the features important for efficient 3 end cleavage and the regulation of transcription termination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545766">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545766" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545766">
        <p class="paperTitle">Transcriptional profiling of aging tissues from female and male African turquoise killifish</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545766" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545766" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, A.; Teefy, B. B.; Lu, R. J.; Nozownik, S.; Tyers, A. M.; Valenzano, D. R.; Benayoun, B. A.</p>
        <p class="info">Score: 11.3, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545766' target='https://doi.org/10.1101/2023.06.20.545766'> 10.1101/2023.06.20.545766</a></p>
        <p class="abstract">The African turquoise killifish is an emerging vertebrate model organism with great potential for aging research due to its naturally short lifespan. Thus far, turquoise killifish aging  omic studies using RNA-seq have examined a single organ, single sex and/or evaluated samples from non-reference strains. Here, we describe a resource dataset of ribosomal RNA depleted RNA-seq libraries generated from the brain, heart, muscle, and spleen from both sexes, as well as young and old animals, in the reference GRZ turquoise killifish strain. We provide basic quality control steps and demonstrate the utility of our dataset by performing differential gene expression and gene ontology analyses by age and sex. Importantly, we show that age has a greater impact than sex on transcriptional landscapes across probed tissues. Finally, we confirm transcription of transposable elements (TEs), which are highly abundant and increase in expression with age in brain tissue. This dataset will be a useful resource for exploring gene and TE expression as a function of both age and sex in a powerful naturally short-lived vertebrate model.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.23.544348">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.23.544348" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.23.544348">
        <p class="paperTitle">The Tabulae Paralytica: Multimodal single-cell and spatial atlases of spinal cord injury</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.23.544348" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.23.544348" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Skinnider, M. A.; Gautier, M.; Teo, A. Y. Y.; Kathe, C.; Hutson, T.; Laskaratos, A.; de Coucy, A.; Regazzi, N.; Aureli, V.; James, N. D.; Schneider, B.; Sofroniew, M.; Barraud, Q.; Bloch, J.; Anderson, M. A.; Squair, J.; Courtine, G.</p>
        <p class="info">Score: 10.8, Published: 2023-06-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.23.544348' target='https://doi.org/10.1101/2023.06.23.544348'> 10.1101/2023.06.23.544348</a></p>
        <p class="abstract">Here, we introduce the Tabulae Paralytica - a compilation of four atlases of spinal cord injury (SCI) comprising a single-nucleus transcriptome atlas of half a million cells; a multiome atlas pairing transcriptomic and epigenomic measurements within the same nuclei; and two spatial transcriptomic atlases of the injured spinal cord spanning four spatial and temporal dimensions. We integrated these atlases into a common framework to dissect the molecular logic that governs the responses to injury within the spinal cord. The Tabulae Paralytica exposed new biological principles that dictate the consequences of SCI, including conserved and divergent neuronal responses to injury; the priming of specific neuronal subpopulations to become circuit-reorganizing neurons after injury; an inherent trade-off between neuronal stress responses and the activation of circuit reorganization programs; the necessity of reestablishing a tripartite neuroprotective barrier between immune-privileged and extra-neural environments after SCI; and a catastrophic failure to form this barrier in old mice. We leveraged the Tabulae Paralytica to develop a rejuvenative gene therapy that reestablished this tripartite barrier, and restored the natural recovery of walking after paralysis in old mice. The Tabulae Paralytica provides an unprecedented window into the pathobiology of SCI, while establishing a framework for integrating multimodal, genome-scale measurements in four dimensions to study biology and medicine.</p>
      </div>
    </div>
  </div>
</div>









<script src="/TRxiv2/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://yorks0n.github.io/TRxiv2">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
