<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms
Authors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.
Score: 51.3, Published: 2024-01-06 DOI: 10.1101/2024.01.02.573821
Research and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms
Authors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.
Score: 51.3, Published: 2024-01-06 DOI: 10.1101/2024.01.02.573821
Research and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-07T10:38:59+00:00" />
<meta property="article:modified_time" content="2024-01-07T10:38:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms
Authors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.
Score: 51.3, Published: 2024-01-06 DOI: 10.1101/2024.01.02.573821
Research and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms\nAuthors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.\nScore: 51.3, Published: 2024-01-06 DOI: 10.1101/2024.01.02.573821\nResearch and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance.",
  "keywords": [
    
  ],
  "articleBody": " Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms\nAuthors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.\nScore: 51.3, Published: 2024-01-06 DOI: 10.1101/2024.01.02.573821\nResearch and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance. This necessitates a framework to identify all types of variants independent of their size (e.g., SNV/SV) or location (e.g., repeats). Here we present DRAGEN that utilizes novel methods based on multigenomes, hardware acceleration, and machine learning based variant detection to provide novel insights into individual genomes with [~]30min computation time (from raw reads to variant detection). DRAGEN outperforms all other state-of-the-art methods in speed and accuracy across all variant types (SNV, indel, STR, SV, CNV) and further incorporates specialized methods to obtain key insights in medically relevant genes (e.g., HLA, SMN, GBA). We showcase DRAGEN across 3,202 genomes and demonstrate its scalability, accuracy, and innovations to further advance the integration of comprehensive genomics for research and medical applications.\nRNA polymerases reshape chromatin and coordinate transcription on individual fibers\nAuthors: Tullius, T. W.; Isaac, R. S.; Ranchalis, J.; Dubocanin, D.; Churchman, L. S.; Stergachis, A. B.\nScore: 37.3, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573133\nDuring eukaryotic transcription, RNA polymerases must initiate and pause within a crowded, complex environment, surrounded by nucleosomes and other transcriptional activity. This environment creates a spatial arrangement along individual chromatin fibers ripe for both competition and coordination, yet these relationships remain largely unknown owing to the inherent limitations of traditional structural and sequencing methodologies. To address these limitations, we employed long-read chromatin fiber sequencing (Fiber-seq) to visualize RNA polymerases within their native chromatin context at single-molecule and near single-nucleotide resolution along up to 30 kb fibers. We demonstrate that Fiber-seq enables the identification of single-molecule RNA Polymerase (Pol) II and III transcription associated foot-prints, which, in aggregate, mirror bulk short-read sequencing-based measurements of transcription. We show that Pol II pausing destabilizes downstream nucleosomes, with frequently paused genes maintaining a short-term memory of these destabilized nucleosomes. Furthermore, we demonstrate pervasive direct coordination and anti-coordination between nearby Pol II genes, Pol III genes, transcribed enhancers, and insulator elements. This coordination is largely limited to spatially organized elements within 5 kb of each other, implicating short-range chromatin environments as a predominant determinant of coordinated polymerase initiation. Overall, transcription initiation reshapes surrounding nucleosome architecture and coordinates nearby transcriptional machinery along individual chromatin fibers.\nThe majority of somatic mutations in fruit trees are layer-specific\nAuthors: Goel, M.; Campoy, J. A.; Krause, K.; Baus, L. C.; Sahu, A.; Sun, H.; Walkemeier, B.; Marek, M.; Beaudry, R.; Ruiz, D.; Huettel, B.; Schneeberger, K.\nScore: 23.6, Published: 2024-01-04 DOI: 10.1101/2024.01.04.573414\nPlant meristems consist of three distinct stem cells layers. The identities of these layers are maintained in developing organs. Somatic mutations in the meristem thus propagate only into the respective layers of the differentiated organs. We analysed the genomes of individual cell layers in an apricot tree and found an unexpectedly high mutation load, where \u003e90% of the somatic mutations were layer-specific. The layers had similar mutation spectra but layer 1 (epidermis) had substantially more mutations than layer 2 (mesocarp). Most somatic mutations were shared between fruits and neighbouring leaves corroborating their meristematic origin. Single-cell transcriptomics revealed their layer-specific effects forming the basis for layer-specific phenotypes. Our work unveils the hidden abundance of layer-specific meristematic mutations and provides insights into their identification and understanding. One sentence summarySomatic mutations in fruit trees occur in specific cell layers.\nSomatic cancer driver mutations are enriched and associated with inflammatory states in Alzheimer's disease microglia\nAuthors: Huang, A. Y.; Zhou, Z.; Talukdar, M.; Miller, M. B.; Chhouk, B.; Enyenihi, L.; Rosen, I.; Stronge, E.; Zhao, B.; Kim, D.; Choi, J.; Khoshkhoo, S.; Kim, J.; Ganz, J.; Travaglini, K. J.; Gabitto, M.; Hodge, R. D.; Kaplan, E.; Lein, E.; De Jager, P.; Bennett, D. A.; Lee, E. A.; Walsh, C. A.\nScore: 13.1, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574078\nAlzheimers disease (AD) is an age-associated neurodegenerative disorder characterized by progressive neuronal loss and pathological accumulation of the misfolded proteins amyloid-{beta} and tau1,2. Neuroinflammation mediated by microglia and brain-resident macrophages plays a crucial role in AD pathogenesis1-5, though the mechanisms by which age, genes, and other risk factors interact remain largely unknown. Somatic mutations accumulate with age and lead to clonal expansion of many cell types, contributing to cancer and many non-cancer diseases6,7. Here we studied somatic mutation in normal aged and AD brains by three orthogonal methods and in three independent AD cohorts. Analysis of bulk RNA sequencing data from 866 samples from different brain regions revealed significantly higher ([~]two-fold) overall burdens of somatic single-nucleotide variants (sSNVs) in AD brains compared to age-matched controls. Molecular-barcoded deep (\u003e1000X) gene panel sequencing of 311 prefrontal cortex samples showed enrichment of sSNVs and somatic insertions and deletions (sIndels) in cancer driver genes in AD brain compared to control, with recurrent, and often multiple, mutations in genes implicated in clonal hematopoiesis (CH)8,9. Pathogenic sSNVs were enriched in CSF1R+ microglia of AD brains, and the high proportion of microglia (up to 40%) carrying some sSNVs in cancer driver genes suggests mutation-driven microglial clonal expansion (MiCE). Analysis of single-nucleus RNA sequencing (snRNAseq) from temporal neocortex of 62 additional AD cases and controls exhibited nominally increased mosaic chromosomal alterations (mCAs) associated with CH10,11. Microglia carrying mCA showed upregulated pro-inflammatory genes, resembling the transcriptomic features of disease-associated microglia (DAM) in AD. Our results suggest that somatic driver mutations in microglia are common with normal aging but further enriched in AD brain, driving MiCE with inflammatory and DAM signatures. Our findings provide the first insights into microglial clonal dynamics in AD and identify potential new approaches to AD diagnosis and therapy.\nCell-cycle dependent DNA repair and replication unifies patterns of chromosome instability\nAuthors: Lu, B.; Winnall, S.; Cross, W.; Barnes, C. P.\nScore: 11.6, Published: 2024-01-04 DOI: 10.1101/2024.01.03.574048\nChromosomal instability (CIN) is pervasive in human tumours and often leads to structural or numerical chromosomal aberrations. Somatic structural variants (SVs) are intimately related to copy number alterations but the two types of variant are often studied independently. In addition, despite numerous studies on detecting various SV patterns, there are still no general quantitative models of SV generation. To address this issue, we develop a computational cell-cycle model for the generation of SVs from end-joining repair and replication after double strand break formation. Our model provides quantitative information on the relationship between breakage fusion bridge cycle, chromothripsis, seismic amplification, and extra-chromosomal circular DNA. Given single-cell whole-genome sequencing data, the model also allows us to infer important parameters in SV generation with Bayesian inference. Our quantitative framework unifies disparate genomic patterns resulted from CIN, provides a null mutational model for SV, and reveals new insights into the impact of genome rearrangement on tumour evolution.\nComparison of software packages for detecting unannotated translated small open reading frames by Ribo-seq\nAuthors: Martinez, T. F.; Tong, G.; Hah, N.\nScore: 11.0, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573709\nAccurate and comprehensive annotation of microprotein-coding small open reading frames (smORFs) is critical to our understanding of normal physiology and disease. Empirical identification of translated smORFs is carried out primarily using ribosome profiling (Ribo-seq). While effective, published Ribo-seq datasets can vary drastically in quality and different analysis tools are frequently employed. Here, we examine the impact of these factors on identifying translated smORFs. We compared five commonly used software tools that assess ORF translation from Ribo-seq (RibORFv0.1, RibORFv1.0, RiboCode, ORFquant, and Ribo-TISH), and found surprisingly low agreement across all tools. Only [~]2% of smORFs were called translated by all five tools and [~]15% by three or more tools when assessing the same high- resolution Ribo-seq dataset. For larger annotated genes, the same analysis showed [~]72% agreement across all five tools. We also found that some tools are strongly biased against low- resolution Ribo-seq data, while others are more tolerant. Analyzing Ribo-seq coverage as a proxy for translation levels revealed that highly translated smORFs are more likely to be detected by more than one tool. Together these results support employing multiple tools to identify the most confident microprotein-coding smORFs, and choosing the tools based on the quality of the dataset and planned downstream characterization experiments of predicted smORFs.\nComparative genomics of Cryptococcus and Kwoniella reveals pathogenesis evolution and contrasting modes of karyotype evolution via chromosome fusion or intercentromeric recombination\nAuthors: Coelho, M. A.; David-Palma, M.; Shea, T.; Bowers, K.; McGinley-Smith, S.; Mohammad, A. W.; Gnirke, A.; Yurkov, A. M.; Nowrousian, M.; Sun, S.; Cuomo, C. A.; Heitman, J.\nScore: 23.0, Published: 2023-12-27 DOI: 10.1101/2023.12.27.573464\nA large-scale comparative genomic analysis was conducted for the global human fungal pathogens within the Cryptococcus genus, compared to non-pathogenic Cryptococcus species, and related species from the sister genus Kwoniella. Chromosome-level genome assemblies were generated for multiple species of both genera, resulting in a dataset encompassing virtually all of their known diversity. Although Cryptococcus and Kwoniella have comparable genome sizes (about 19.2 and 22.9 Mb) and similar gene content, hinting at pre-adaptive pathogenic potential, our analysis found evidence in pathogenic Cryptococcus species of specific examples of gene gain (via horizontal gene transfer) and gene loss, which might represent evolutionary signatures of pathogenic development. Genome analysis also revealed a significant variation in chromosome number and structure between the two genera. By combining synteny analysis and experimental centromere validation, we found that most Cryptococcus species have 14 chromosomes, whereas most Kwoniella species have fewer (11, 8, 5 or even as few as 3). Reduced chromosome number in Kwoniella is associated with formation of giant chromosomes (up to 18 Mb) through repeated chromosome fusion events, each marked by a pericentric inversion and centromere loss. While similar chromosome inversion-fusion patterns were observed in all Kwoniella species with fewer than 14 chromosomes, no such pattern was detected in Cryptococcus. Instead, Cryptococcus species with less than 14 chromosomes, underwent chromosome reductions primarily through rearrangements associated with the loss of repeat-rich centromeres. Additionally, Cryptococcus genomes exhibited frequent interchromosomal translocations, including intercentromeric recombination facilitated by transposons shared between centromeres. Taken together, our findings advance our understanding of genomic changes possibly associated with pathogenicity in Cryptococcus and provide a foundation to elucidate mechanisms of centromere loss and chromosome fusion driving distinct karyotypes in closely related fungal species, including prominent global human pathogens.\nStable chromosome configuration and loop-based polarization in animal symbionts\nAuthors: Viehboeck, T.; Weber, P. M.; Krause, N.; Varoquaux, N.; Boccard, F.; Junier, I.; Bulgheresi, S.; Lioy, V. S.\nScore: 7.6, Published: 2024-01-03 DOI: 10.1101/2023.12.21.572873\nChromosome partitioning precedes the division of the cytoplasm, and its evolution is linked with the positioning of the division plane. So far, bacterial chromosome biology has heavily focused on transversally dividing, free-living ones. Here, we determined the chromosome organization of three longitudinally dividing Neisseriaceae exclusively inhabiting the oral cavity of mammals. We showed that in all three multicellular bacteria the origin of DNA replication is invariably located at the host-attached (proximal) pole. Next, 3C-seq revealed loop-based folding of the ori region in Alysiella filiformis and Simonsiella muelleri. Moreover, genes involved in cell motility, piliation and signal transduction mechanisms were specifically looped when transcriptionally and translationally active cells adhered to a substrate, but not when cultured in liquid. Overall, we propose that proximal positioning of the ori and loop-based folding of its surrounding DNA may mediate localized translation of proteins involved in host colonization.\nA single-cell transposable element atlas of human cell identity\nAuthors: Reyes-Gopar, H.; Marston, J. L.; Singh, B.; Greenig, M.; Lin, J.; Ostrowski, M. A.; Randall, K. N.; Sandoval-Motta, S.; Dopkins, N.; Lawrence, E.; O'Mara, M. M.; Fei, T.; Duarte, R. R. R.; Powell, T. R.; Hernandez-Lemus, E.; Iniguez, L. P.; Nixon, D. F.; Bendall, M. L.\nScore: 21.4, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573568\nSingle cell RNA sequencing (scRNA-seq) is revolutionizing the study of complex biological systems. However, most sequencing studies overlook the contribution of transposable element (TE) expression to the transcriptome. In both scRNA-seq and bulk tissue RNA sequencing (RNA-seq), quantification of TE expression is challenging due to repetitive sequence content and poorly characterized TE gene models. Here, we developed a tool and analysis pipeline for Single cell Transposable Element Locus Level Analysis of scRNA Sequencing (Stellarscope) that reassigns multi-mapped reads to specific genomic loci using an expectation-maximization algorithm. Using Stellarscope, we built an atlas of TE expression in human PBMCs. We found that locus-specific TEs delineate cell types and define new cell subsets not identified by standard mRNA expression profiles. Altogether, this study provides comprehensive insights into the influence of transposable elements in human biology.\nGenome-wide characterisation of pathogenicity-related proteins in Mycena citricolor, the causal agent of the American Leaf Spot in coffee\nAuthors: Larke-Mejia, N. L.; Arciniegas, N.; Di Palma, F.; Angel C, C. A.; De Vega, J. J.\nScore: 5.7, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573698\nMycena citricolor is a fungus that causes the American Leaf Spot (ALS) disease in multiple hosts, including coffee and avocado. This hemibiotroph penetrates the plant through damage induced by oxalic acid. This can cause 20-90% crop losses in coffee depending on the environmental and production conditions. M. citricolor is the only known pathogenic species in the Mycena genus, a large group of saprophytic mushrooms. Comparing the saprophytic and pathogenic genomes can allow us to identify genetic machinery associated with the pathogens genome-wide functional acquisitions to cause disease. To identify pathogenicity-related genes in M. citricolor, we analysed protein family copy-number variation, secretome prediction, and homology to known virulence factors in two M. citricolor assemblies, including a newly assembled and annotated long-read genome. We found that the pathogenic M. citricolor had a higher proportion of secreted genes expanded in copy-number, and expanded gene copies homologous to known virulence factors than the saprophytic Mycena. We shortlisted over 300 candidate genes in each M. citricolor assembly. Focusing on genes strongly regulated during plant interaction, we found over 100 candidates, primarily from multiple copies (up to 4-3 times) of 42 well-known virulence factors (e.g. MFS1, CUTA, NoxA/B, OLE1, NorA), plus a few clade-specific uncharacterised genes. M. citricolor transition to a pathogenic lifestyle reflected genome-wide functional changes. M. citricolor seems to primarily depend on well-known virulence factors in large copy numbers, suggesting the molecular plant-interaction processes involved are like those of better-studied fungi. Hypothetically, the development of ALS resistance could mirror studied responses to these virulence factors.\n",
  "wordCount" : "2393",
  "inLanguage": "en",
  "datePublished": "2024-01-07T10:38:59Z",
  "dateModified": "2024-01-07T10:38:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573821">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573821" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573821">
        <p class="paperTitle">Comprehensive and accurate genome analysis at scale using DRAGEN accelerated algorithms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573821" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573821" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Behera, S.; Catreux, S.; Rossi, M.; Truong, S.; Huang, Z.; Ruehle, M.; Visvanath, A.; Parnaby, G.; Roddey, C.; Onuchic, V.; Cameron, D. L.; English, A.; Mehtalia, S.; Han, J.; Mehio, R.; Sedlazeck, F. J.</p>
        <p class="info">Score: 51.3, Published: 2024-01-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573821' target='https://doi.org/10.1101/2024.01.02.573821'> 10.1101/2024.01.02.573821</a></p>
        <p class="abstract">Research and medical genomics require comprehensive and scalable solutions to drive the discovery of novel disease targets, evolutionary drivers, and genetic markers with clinical significance. This necessitates a framework to identify all types of variants independent of their size (e.g., SNV/SV) or location (e.g., repeats). Here we present DRAGEN that utilizes novel methods based on multigenomes, hardware acceleration, and machine learning based variant detection to provide novel insights into individual genomes with [~]30min computation time (from raw reads to variant detection). DRAGEN outperforms all other state-of-the-art methods in speed and accuracy across all variant types (SNV, indel, STR, SV, CNV) and further incorporates specialized methods to obtain key insights in medically relevant genes (e.g., HLA, SMN, GBA). We showcase DRAGEN across 3,202 genomes and demonstrate its scalability, accuracy, and innovations to further advance the integration of comprehensive genomics for research and medical applications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573133" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573133">
        <p class="paperTitle">RNA polymerases reshape chromatin and coordinate transcription on individual fibers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tullius, T. W.; Isaac, R. S.; Ranchalis, J.; Dubocanin, D.; Churchman, L. S.; Stergachis, A. B.</p>
        <p class="info">Score: 37.3, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573133' target='https://doi.org/10.1101/2023.12.22.573133'> 10.1101/2023.12.22.573133</a></p>
        <p class="abstract">During eukaryotic transcription, RNA polymerases must initiate and pause within a crowded, complex environment, surrounded by nucleosomes and other transcriptional activity. This environment creates a spatial arrangement along individual chromatin fibers ripe for both competition and coordination, yet these relationships remain largely unknown owing to the inherent limitations of traditional structural and sequencing methodologies. To address these limitations, we employed long-read chromatin fiber sequencing (Fiber-seq) to visualize RNA polymerases within their native chromatin context at single-molecule and near single-nucleotide resolution along up to 30 kb fibers. We demonstrate that Fiber-seq enables the identification of single-molecule RNA Polymerase (Pol) II and III transcription associated foot-prints, which, in aggregate, mirror bulk short-read sequencing-based measurements of transcription. We show that Pol II pausing destabilizes downstream nucleosomes, with frequently paused genes maintaining a short-term memory of these destabilized nucleosomes. Furthermore, we demonstrate pervasive direct coordination and anti-coordination between nearby Pol II genes, Pol III genes, transcribed enhancers, and insulator elements. This coordination is largely limited to spatially organized elements within 5 kb of each other, implicating short-range chromatin environments as a predominant determinant of coordinated polymerase initiation. Overall, transcription initiation reshapes surrounding nucleosome architecture and coordinates nearby transcriptional machinery along individual chromatin fibers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.04.573414">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.04.573414" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.04.573414">
        <p class="paperTitle">The majority of somatic mutations in fruit trees are layer-specific</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.04.573414" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.04.573414" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goel, M.; Campoy, J. A.; Krause, K.; Baus, L. C.; Sahu, A.; Sun, H.; Walkemeier, B.; Marek, M.; Beaudry, R.; Ruiz, D.; Huettel, B.; Schneeberger, K.</p>
        <p class="info">Score: 23.6, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.04.573414' target='https://doi.org/10.1101/2024.01.04.573414'> 10.1101/2024.01.04.573414</a></p>
        <p class="abstract">Plant meristems consist of three distinct stem cells layers. The identities of these layers are maintained in developing organs. Somatic mutations in the meristem thus propagate only into the respective layers of the differentiated organs. We analysed the genomes of individual cell layers in an apricot tree and found an unexpectedly high mutation load, where &gt;90% of the somatic mutations were layer-specific. The layers had similar mutation spectra but layer 1 (epidermis) had substantially more mutations than layer 2 (mesocarp). Most somatic mutations were shared between fruits and neighbouring leaves corroborating their meristematic origin. Single-cell transcriptomics revealed their layer-specific effects forming the basis for layer-specific phenotypes. Our work unveils the hidden abundance of layer-specific meristematic mutations and provides insights into their identification and understanding.

One sentence summarySomatic mutations in fruit trees occur in specific cell layers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574078">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574078" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574078">
        <p class="paperTitle">Somatic cancer driver mutations are enriched and associated with inflammatory states in Alzheimer&#39;s disease microglia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574078" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574078" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, A. Y.; Zhou, Z.; Talukdar, M.; Miller, M. B.; Chhouk, B.; Enyenihi, L.; Rosen, I.; Stronge, E.; Zhao, B.; Kim, D.; Choi, J.; Khoshkhoo, S.; Kim, J.; Ganz, J.; Travaglini, K. J.; Gabitto, M.; Hodge, R. D.; Kaplan, E.; Lein, E.; De Jager, P.; Bennett, D. A.; Lee, E. A.; Walsh, C. A.</p>
        <p class="info">Score: 13.1, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574078' target='https://doi.org/10.1101/2024.01.03.574078'> 10.1101/2024.01.03.574078</a></p>
        <p class="abstract">Alzheimers disease (AD) is an age-associated neurodegenerative disorder characterized by progressive neuronal loss and pathological accumulation of the misfolded proteins amyloid-{beta} and tau1,2. Neuroinflammation mediated by microglia and brain-resident macrophages plays a crucial role in AD pathogenesis1-5, though the mechanisms by which age, genes, and other risk factors interact remain largely unknown. Somatic mutations accumulate with age and lead to clonal expansion of many cell types, contributing to cancer and many non-cancer diseases6,7. Here we studied somatic mutation in normal aged and AD brains by three orthogonal methods and in three independent AD cohorts. Analysis of bulk RNA sequencing data from 866 samples from different brain regions revealed significantly higher ([~]two-fold) overall burdens of somatic single-nucleotide variants (sSNVs) in AD brains compared to age-matched controls. Molecular-barcoded deep (&gt;1000X) gene panel sequencing of 311 prefrontal cortex samples showed enrichment of sSNVs and somatic insertions and deletions (sIndels) in cancer driver genes in AD brain compared to control, with recurrent, and often multiple, mutations in genes implicated in clonal hematopoiesis (CH)8,9. Pathogenic sSNVs were enriched in CSF1R&#43; microglia of AD brains, and the high proportion of microglia (up to 40%) carrying some sSNVs in cancer driver genes suggests mutation-driven microglial clonal expansion (MiCE). Analysis of single-nucleus RNA sequencing (snRNAseq) from temporal neocortex of 62 additional AD cases and controls exhibited nominally increased mosaic chromosomal alterations (mCAs) associated with CH10,11. Microglia carrying mCA showed upregulated pro-inflammatory genes, resembling the transcriptomic features of disease-associated microglia (DAM) in AD. Our results suggest that somatic driver mutations in microglia are common with normal aging but further enriched in AD brain, driving MiCE with inflammatory and DAM signatures. Our findings provide the first insights into microglial clonal dynamics in AD and identify potential new approaches to AD diagnosis and therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.03.574048">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.03.574048" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.03.574048">
        <p class="paperTitle">Cell-cycle dependent DNA repair and replication unifies patterns of chromosome instability</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.03.574048" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.03.574048" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lu, B.; Winnall, S.; Cross, W.; Barnes, C. P.</p>
        <p class="info">Score: 11.6, Published: 2024-01-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.03.574048' target='https://doi.org/10.1101/2024.01.03.574048'> 10.1101/2024.01.03.574048</a></p>
        <p class="abstract">Chromosomal instability (CIN) is pervasive in human tumours and often leads to structural or numerical chromosomal aberrations. Somatic structural variants (SVs) are intimately related to copy number alterations but the two types of variant are often studied independently. In addition, despite numerous studies on detecting various SV patterns, there are still no general quantitative models of SV generation. To address this issue, we develop a computational cell-cycle model for the generation of SVs from end-joining repair and replication after double strand break formation. Our model provides quantitative information on the relationship between breakage fusion bridge cycle, chromothripsis, seismic amplification, and extra-chromosomal circular DNA. Given single-cell whole-genome sequencing data, the model also allows us to infer important parameters in SV generation with Bayesian inference. Our quantitative framework unifies disparate genomic patterns resulted from CIN, provides a null mutational model for SV, and reveals new insights into the impact of genome rearrangement on tumour evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573709">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573709" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573709">
        <p class="paperTitle">Comparison of software packages for detecting unannotated translated small open reading frames by Ribo-seq</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573709" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573709" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martinez, T. F.; Tong, G.; Hah, N.</p>
        <p class="info">Score: 11.0, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573709' target='https://doi.org/10.1101/2023.12.30.573709'> 10.1101/2023.12.30.573709</a></p>
        <p class="abstract">Accurate and comprehensive annotation of microprotein-coding small open reading frames (smORFs) is critical to our understanding of normal physiology and disease. Empirical identification of translated smORFs is carried out primarily using ribosome profiling (Ribo-seq). While effective, published Ribo-seq datasets can vary drastically in quality and different analysis tools are frequently employed. Here, we examine the impact of these factors on identifying translated smORFs. We compared five commonly used software tools that assess ORF translation from Ribo-seq (RibORFv0.1, RibORFv1.0, RiboCode, ORFquant, and Ribo-TISH), and found surprisingly low agreement across all tools. Only [~]2% of smORFs were called translated by all five tools and [~]15% by three or more tools when assessing the same high- resolution Ribo-seq dataset. For larger annotated genes, the same analysis showed [~]72% agreement across all five tools. We also found that some tools are strongly biased against low- resolution Ribo-seq data, while others are more tolerant. Analyzing Ribo-seq coverage as a proxy for translation levels revealed that highly translated smORFs are more likely to be detected by more than one tool. Together these results support employing multiple tools to identify the most confident microprotein-coding smORFs, and choosing the tools based on the quality of the dataset and planned downstream characterization experiments of predicted smORFs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.27.573464">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.27.573464" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.27.573464">
        <p class="paperTitle">Comparative genomics of Cryptococcus and Kwoniella reveals pathogenesis evolution and contrasting modes of karyotype evolution via chromosome fusion or intercentromeric recombination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.27.573464" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.27.573464" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coelho, M. A.; David-Palma, M.; Shea, T.; Bowers, K.; McGinley-Smith, S.; Mohammad, A. W.; Gnirke, A.; Yurkov, A. M.; Nowrousian, M.; Sun, S.; Cuomo, C. A.; Heitman, J.</p>
        <p class="info">Score: 23.0, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.27.573464' target='https://doi.org/10.1101/2023.12.27.573464'> 10.1101/2023.12.27.573464</a></p>
        <p class="abstract">A large-scale comparative genomic analysis was conducted for the global human fungal pathogens within the Cryptococcus genus, compared to non-pathogenic Cryptococcus species, and related species from the sister genus Kwoniella. Chromosome-level genome assemblies were generated for multiple species of both genera, resulting in a dataset encompassing virtually all of their known diversity. Although Cryptococcus and Kwoniella have comparable genome sizes (about 19.2 and 22.9 Mb) and similar gene content, hinting at pre-adaptive pathogenic potential, our analysis found evidence in pathogenic Cryptococcus species of specific examples of gene gain (via horizontal gene transfer) and gene loss, which might represent evolutionary signatures of pathogenic development. Genome analysis also revealed a significant variation in chromosome number and structure between the two genera. By combining synteny analysis and experimental centromere validation, we found that most Cryptococcus species have 14 chromosomes, whereas most Kwoniella species have fewer (11, 8, 5 or even as few as 3). Reduced chromosome number in Kwoniella is associated with formation of giant chromosomes (up to 18 Mb) through repeated chromosome fusion events, each marked by a pericentric inversion and centromere loss. While similar chromosome inversion-fusion patterns were observed in all Kwoniella species with fewer than 14 chromosomes, no such pattern was detected in Cryptococcus. Instead, Cryptococcus species with less than 14 chromosomes, underwent chromosome reductions primarily through rearrangements associated with the loss of repeat-rich centromeres. Additionally, Cryptococcus genomes exhibited frequent interchromosomal translocations, including intercentromeric recombination facilitated by transposons shared between centromeres. Taken together, our findings advance our understanding of genomic changes possibly associated with pathogenicity in Cryptococcus and provide a foundation to elucidate mechanisms of centromere loss and chromosome fusion driving distinct karyotypes in closely related fungal species, including prominent global human pathogens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572873">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572873" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572873">
        <p class="paperTitle">Stable chromosome configuration and loop-based polarization in animal symbionts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572873" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572873" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Viehboeck, T.; Weber, P. M.; Krause, N.; Varoquaux, N.; Boccard, F.; Junier, I.; Bulgheresi, S.; Lioy, V. S.</p>
        <p class="info">Score: 7.6, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572873' target='https://doi.org/10.1101/2023.12.21.572873'> 10.1101/2023.12.21.572873</a></p>
        <p class="abstract">Chromosome partitioning precedes the division of the cytoplasm, and its evolution is linked with the positioning of the division plane. So far, bacterial chromosome biology has heavily focused on transversally dividing, free-living ones. Here, we determined the chromosome organization of three longitudinally dividing Neisseriaceae exclusively inhabiting the oral cavity of mammals. We showed that in all three multicellular bacteria the origin of DNA replication is invariably located at the host-attached (proximal) pole. Next, 3C-seq revealed loop-based folding of the ori region in Alysiella filiformis and Simonsiella muelleri. Moreover, genes involved in cell motility, piliation and signal transduction mechanisms were specifically looped when transcriptionally and translationally active cells adhered to a substrate, but not when cultured in liquid. Overall, we propose that proximal positioning of the ori and loop-based folding of its surrounding DNA may mediate localized translation of proteins involved in host colonization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573568">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573568" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573568">
        <p class="paperTitle">A single-cell transposable element atlas of human cell identity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573568" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573568" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Reyes-Gopar, H.; Marston, J. L.; Singh, B.; Greenig, M.; Lin, J.; Ostrowski, M. A.; Randall, K. N.; Sandoval-Motta, S.; Dopkins, N.; Lawrence, E.; O&#39;Mara, M. M.; Fei, T.; Duarte, R. R. R.; Powell, T. R.; Hernandez-Lemus, E.; Iniguez, L. P.; Nixon, D. F.; Bendall, M. L.</p>
        <p class="info">Score: 21.4, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573568' target='https://doi.org/10.1101/2023.12.28.573568'> 10.1101/2023.12.28.573568</a></p>
        <p class="abstract">Single cell RNA sequencing (scRNA-seq) is revolutionizing the study of complex biological systems. However, most sequencing studies overlook the contribution of transposable element (TE) expression to the transcriptome. In both scRNA-seq and bulk tissue RNA sequencing (RNA-seq), quantification of TE expression is challenging due to repetitive sequence content and poorly characterized TE gene models. Here, we developed a tool and analysis pipeline for Single cell Transposable Element Locus Level Analysis of scRNA Sequencing (Stellarscope) that reassigns multi-mapped reads to specific genomic loci using an expectation-maximization algorithm. Using Stellarscope, we built an atlas of TE expression in human PBMCs. We found that locus-specific TEs delineate cell types and define new cell subsets not identified by standard mRNA expression profiles. Altogether, this study provides comprehensive insights into the influence of transposable elements in human biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573698">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573698" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573698">
        <p class="paperTitle">Genome-wide characterisation of pathogenicity-related proteins in Mycena citricolor, the causal agent of the American Leaf Spot in coffee</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573698" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573698" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Larke-Mejia, N. L.; Arciniegas, N.; Di Palma, F.; Angel C, C. A.; De Vega, J. J.</p>
        <p class="info">Score: 5.7, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573698' target='https://doi.org/10.1101/2023.12.30.573698'> 10.1101/2023.12.30.573698</a></p>
        <p class="abstract">Mycena citricolor is a fungus that causes the American Leaf Spot (ALS) disease in multiple hosts, including coffee and avocado. This hemibiotroph penetrates the plant through damage induced by oxalic acid. This can cause 20-90% crop losses in coffee depending on the environmental and production conditions. M. citricolor is the only known pathogenic species in the Mycena genus, a large group of saprophytic mushrooms. Comparing the saprophytic and pathogenic genomes can allow us to identify genetic machinery associated with the pathogens genome-wide functional acquisitions to cause disease.

To identify pathogenicity-related genes in M. citricolor, we analysed protein family copy-number variation, secretome prediction, and homology to known virulence factors in two M. citricolor assemblies, including a newly assembled and annotated long-read genome. We found that the pathogenic M. citricolor had a higher proportion of secreted genes expanded in copy-number, and expanded gene copies homologous to known virulence factors than the saprophytic Mycena. We shortlisted over 300 candidate genes in each M. citricolor assembly. Focusing on genes strongly regulated during plant interaction, we found over 100 candidates, primarily from multiple copies (up to 4-3 times) of 42 well-known virulence factors (e.g. MFS1, CUTA, NoxA/B, OLE1, NorA), plus a few clade-specific uncharacterised genes.

M. citricolor transition to a pathogenic lifestyle reflected genome-wide functional changes. M. citricolor seems to primarily depend on well-known virulence factors in large copy numbers, suggesting the molecular plant-interaction processes involved are like those of better-studied fungi. Hypothetically, the development of ALS resistance could mirror studied responses to these virulence factors.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
