<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Concurrent profiling of multiscale 3D genome organization and gene expression in single mammalian cells
Authors: Zhou, T.; Zhang, R.; Jia, D.; Doty, R. T.; Munday, A. D.; Gao, D.; Xin, L.; Abkowitz, J. L.; Duan, Z.; Ma, J.
Score: 24.6, Published: 2023-07-25 DOI: 10.1101/2023.07.20.549578
The organization of mammalian genomes within the nucleus features a complex, multiscale three-dimensional (3D) architecture. The functional significance of these 3D genome features, however, remains largely elusive due to limited single-cell technologies that can concurrently profile genome organization and transcriptional activities.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Concurrent profiling of multiscale 3D genome organization and gene expression in single mammalian cells
Authors: Zhou, T.; Zhang, R.; Jia, D.; Doty, R. T.; Munday, A. D.; Gao, D.; Xin, L.; Abkowitz, J. L.; Duan, Z.; Ma, J.
Score: 24.6, Published: 2023-07-25 DOI: 10.1101/2023.07.20.549578
The organization of mammalian genomes within the nucleus features a complex, multiscale three-dimensional (3D) architecture. The functional significance of these 3D genome features, however, remains largely elusive due to limited single-cell technologies that can concurrently profile genome organization and transcriptional activities." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Concurrent profiling of multiscale 3D genome organization and gene expression in single mammalian cells
Authors: Zhou, T.; Zhang, R.; Jia, D.; Doty, R. T.; Munday, A. D.; Gao, D.; Xin, L.; Abkowitz, J. L.; Duan, Z.; Ma, J.
Score: 24.6, Published: 2023-07-25 DOI: 10.1101/2023.07.20.549578
The organization of mammalian genomes within the nucleus features a complex, multiscale three-dimensional (3D) architecture. The functional significance of these 3D genome features, however, remains largely elusive due to limited single-cell technologies that can concurrently profile genome organization and transcriptional activities."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Concurrent profiling of multiscale 3D genome organization and gene expression in single mammalian cells\nAuthors: Zhou, T.; Zhang, R.; Jia, D.; Doty, R. T.; Munday, A. D.; Gao, D.; Xin, L.; Abkowitz, J. L.; Duan, Z.; Ma, J.\nScore: 24.6, Published: 2023-07-25 DOI: 10.1101/2023.07.20.549578\nThe organization of mammalian genomes within the nucleus features a complex, multiscale three-dimensional (3D) architecture. The functional significance of these 3D genome features, however, remains largely elusive due to limited single-cell technologies that can concurrently profile genome organization and transcriptional activities.",
  "keywords": [
    
  ],
  "articleBody": " Concurrent profiling of multiscale 3D genome organization and gene expression in single mammalian cells\nAuthors: Zhou, T.; Zhang, R.; Jia, D.; Doty, R. T.; Munday, A. D.; Gao, D.; Xin, L.; Abkowitz, J. L.; Duan, Z.; Ma, J.\nScore: 24.6, Published: 2023-07-25 DOI: 10.1101/2023.07.20.549578\nThe organization of mammalian genomes within the nucleus features a complex, multiscale three-dimensional (3D) architecture. The functional significance of these 3D genome features, however, remains largely elusive due to limited single-cell technologies that can concurrently profile genome organization and transcriptional activities. Here, we report GAGE-seq, a highly scalable, robust single-cell co-assay that simultaneously measures 3D genome structure and transcriptome within the same cell. Employing GAGE-seq on mouse brain cortex and human bone marrow CD34+ cells, we comprehensively characterized the intricate relationships between 3D genome and gene expression. We found that these multiscale 3D genome features collectively inform cell type-specific gene expressions, hence contributing to defining cell identity at the single-cell level. Integration of GAGE-seq data with spatial transcriptomic data revealed in situ variations of the 3D genome in mouse cortex. Moreover, our observations of lineage commitment in normal human hematopoiesis unveiled notable discordant changes between 3D genome organization and gene expression, underscoring a complex, temporal interplay at the single-cell level that is more nuanced than previously appreciated. Together, GAGE-seq provides a powerful, cost-effective approach for interrogating genome structure and gene expression relationships at the single-cell level across diverse biological contexts.\nSequencing 4.3 million mutations in wheat promoters to understand and modify gene expression\nAuthors: Zhang, J.; Xiong, H.; Burguener, G. F.; Vasquez-Gross, H.; Liu, Q.; Debernardi, J. M.; Akhunova, A.; Campbell-Garland, K.; Kianian, S.; Brown-Guedira, G.; Pozniak, C.; Faris, J.; Akhunov, E.; Dubcovsky, J.\nScore: 20.3, Published: 2023-07-22 DOI: 10.1101/2023.07.21.550110\nWheat is an important contributor to global food security, and further improvements are required to feed a growing human population. New functional genetics and genomics tools can help us to understand the function of different genes and to engineer beneficial changes. In this study, we used a promoter capture assay to sequence 2-kb regions upstream of all high-confidence annotated genes from 1,513 mutagenized plants from the tetraploid wheat variety Kronos. We identified 4.3 million induced mutations with an accuracy of 99.8%, resulting in a mutation density of 41.9 mutations per kb. We also remapped Kronos exome capture reads to Chinese Spring RefSeq v1.1, identified 4.7 million mutations, and predicted their effects on annotated genes. Using these predictions, we identified 59% more non-synonymous substitutions and 49% more truncation mutations than in the original study. To show the biological value of the new promoter dataset, we selected two mutations within the promoter of the VRN-A1 vernalization gene. Both mutations, located within transcription factor binding sites, significantly altered VRN-A1 expression, and one reduced the number of spikelets per spike. These publicly available sequenced mutant datasets provide rapid and inexpensive access to induced variation in the promoters and coding regions of most wheat genes. These mutations can be used to understand and modulate gene expression and phenotypes for both basic and commercial applications, where limited governmental regulations can facilitate deployment. These mutant collections, together with gene editing, provide valuable tools to accelerate functional genetic studies in this economically important crop. Significance StatementWe sequenced 4.3 million induced mutations in the promoters and 4.7 million in the coding regions of most wheat genes. We also show how this public resource can be used to understand gene function, modulate gene expression, and generate changes in valuable wheat agronomic traits.\nRecurrent chromosome destabilization through repeat-mediated rearrangements in a fungal pathogen\nAuthors: Fouche, S.; Oggenfuss, U.; McDonald, B. A.; Croll, D.\nScore: 19.8, Published: 2023-07-15 DOI: 10.1101/2023.07.14.549097\nGenomic instability caused by chromosomal rearrangements has severe consequences for organismal fitness and progression of cancerous cell lines. The triggers of destabilized chromosomes remain poorly understood but are often assumed to be associated with fragile sites. Here, we retrace a runaway chromosomal degeneration process observed in a fungal pathogen using telomere-to-telomere assemblies across an experimental pedigree. We show that the same fragile sites triggered reproducible, large-scale rearrangements through non-allelic recombination. Across the four-generation pedigree, chromosomal rearrangements were accompanied by non-disjunction events and caused aneuploid progeny to carry up to four chromosomal copies. We identify a specific transposable element as the most likely trigger for the repeated chromosomal degeneration. The element is associated with higher virulence of the pathogen and has undergone a burst increasing copy numbers across the genome. Chromosome sequences are also targeted by a genome defense mechanism active on duplicated sequences, which may contribute to decay. Our work identifies the exact sequence triggers initiating chromosome instability and perpetuating degenerative cycles. Dissecting proximate causes leading to run-away chromosomal degeneration expands our understanding of chromosomal evolution beyond cancer lines.\nThe International Weed Genomics Consortium: Community Resources for Weed Genomics Research\nAuthors: Montgomery, J. S.; Morran, S.; MacGregor, D. R.; McElroy, J. S.; Neve, P.; Neto, C.; Vila-Aiub, M. M.; Sandoval, M. V.; Menendez, A. I.; Kreiner, J.; Fan, L.; Caicedo, A. L.; Maughan, P. J.; Martins, B. A. B.; Mika, J.; Collavo, A.; Merotto, A.; Subramanian, N. K.; Bagavathiannan, M. V.; Cutti, L.; Islam, M. M.; Gill, B. S.; Cicchillo, R.; Gast, R.; Soni, N.; Wright, T. R.; Zastrow-Hayes, G.; May, G.; Malone, J. M.; Sehgal, D.; Kaundun, S. S.; Dale, R. P.; Vorster, B. J.; Peters, B.; Lerchl, J.; Tranel, P. J.; Beffa, R.; Jugulam, M.; Fengler, K.; Llaca, V.; Patterson, E. L.; Gaines, T. A.\nScore: 12.4, Published: 2023-07-24 DOI: 10.1101/2023.07.19.549613\nThe International Weed Genomics Consortium is a collaborative group of researchers focused on developing genomic resources for the study of weedy plants. Weeds are attractive systems for basic and applied research due to their impacts on agricultural systems and capacity to swiftly adapt in response to anthropogenic selection pressures. Our goal is to use genomic information to develop sustainable and effective weed control methods and to provide insights about biotic and abiotic stress tolerance to assist crop breeding. Here, we outline resources under development by the consortium and highlight areas of research that will be impacted by these enabling resources.\nHuman Cytomegalovirus in breast milk is associated with milk composition, the infant gut microbiome, and infant growth\nAuthors: Johnson, K. E.; Heisel, T.; Fields, D. A.; Isganaitis, E.; Jacobs, K. M.; Knights, D.; Lock, E. F.; Rudolph, M. C.; Gale, C. A.; Schleiss, M. R.; Albert, F. W.; Demerath, E. W.; Blekhman, R.\nScore: 12.4, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549370\nHuman cytomegalovirus (CMV) is a highly prevalent herpesvirus that is often transmitted to the neonate via breast milk. Postnatal CMV transmission can have negative health consequences for preterm and immunocompromised infants, but any effects on healthy term infants are thought to be benign. Furthermore, the impact of CMV on the composition of the hundreds of bioactive factors in human milk has not been tested. Here, we utilize a cohort of exclusively breastfeeding full term mother-infant pairs to test for differences in the milk transcriptome and metabolome associated with CMV, and the impact of CMV in breast milk on the infant gut microbiome and infant growth. We find upregulation of the indoleamine 2,3-dioxygenase (IDO) tryptophan-to-kynurenine metabolic pathway in CMV+ milk samples, and that CMV+ milk is associated with decreased Bifidobacterium in the infant gut. Our data indicate a complex relationship between milk CMV, milk kynurenine, and infant growth; with kynurenine positively correlated, and CMV viral load negatively correlated, with infant weight-for-length at 1 month of age. These results suggest CMV transmission, CMV-related changes in milk composition, or both may be modulators of full term infant development.\nThe SWI/SNF nucleosome remodeler constrains enhancer activity during Drosophila wing development\nAuthors: Niederhuber, M. J.; Leatham-Jensen, M.; McKay, D. J.\nScore: 11.8, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549384\nChromatin remodeling is central to the dynamic changes in gene expression that drive cell fate determination. During development, the sets of enhancers that are accessible for use change globally as cells transition between stages. While transcription factors and nucleosome remodeling complexes are known to work together to control access to enhancers, it is unclear how the short stretches of DNA that they individually unmask yield the kilobase-sized accessible regions that are characteristic of active enhancers. Here, we performed a genetic screen to investigate the role of ATP-dependent nucleosome remodeling complexes in the control of dynamic enhancer activity. We find that the Drosophila BAP complex, a member of the SWI/SNF family of nucleosome remodelers, is required for repression of a temporally dynamic enhancer, brdisc. Contrary to expectations, we find that the BAP-specific subunit, Osa, is dispensable for mediating changes in chromatin accessibility between early and late stages of wing development. Instead, we find that Osa is required to constrain the levels of brdisc activity in imaginal wing discs when the enhancer is normally active. Genome-wide profiling reveals that Osa binds directly to the brdisc enhancer as well as thousands of other developmentally dynamic regulatory sites, including multiple genes encoding components and targets of the Notch signaling pathway. We find that Osa loss of function results in development of ectopic sensory structures that are normally patterned by Notch signaling early in wing development. Moreover, we find that Osa loss of function results in hyperactivation of the Delta gene, which encodes the Notch ligand. Together, these findings indicate that proper constraint of enhancer activity is necessary for regulation of dose-dependent developmental events.\nThe human genome's vocabulary as proposed by the DNA language model GROVER\nAuthors: Sanabria, M.; Hirsch, J.; Poetsch, A. R.\nScore: 11.0, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549677\nLarge Language Models (LLMs) on natural language have achieved a level of performance that allows the generation of coherent and syntactically correct text. DNA sequence of genomes follows rules similar to natural language, but a distinguishing factor is the absence of a concept analogous to words. We established byte-pair tokenization on the human genome and trained a foundation language model called GROVER (\"Genome Rules Obtained Via Extracted Representations\") to select the optimal vocabulary with a custom fine-tuning task of next-k-mer prediction. We thus defined a dictionary of words/tokens in the human genome that best carries the information content for DNA language models. Analyzing GROVERs learned representations, we observed that token embeddings primarily encode information related to their frequency, sequence content, and length. Some tokens are almost exclusively localized in repeats, while the vast majority widely distributes over the genome. The model also learns context and lexical ambiguity. Average embeddings of genomic regions relate to functional genomics annotation and thus indicate that GROVER has learned these structures purely from the contextual relationships of tokens. That we can extract functional annotations from the genome, purely based on sequence representation to the trained model, highlights the extent of information content encoded by the sequence. This is supported by fine-tuning tasks on genome biology with questions of promoter identity and protein-DNA binding. GROVER learns sequence context, a sense for grammatical structures and language rules in the genome. This knowledge can be extracted and used to compose a grammar book for the code of life.\nRobust and Versatile Arrayed Libraries for Human Genome-Wide CRISPR Activation, Deletion and Silencing\nAuthors: Yin, J.-A.; Frick, L.; Scheidmann, M. C.; Liu, T.; Trevisan, C.; Dhingra, A.; Spinelli, A.; Wu, Y.; Yao, L.; Vena, D. L.; Knapp, B.; De Cecco, E.; Ging, K.; Armani, A.; Oakeley, E.; Nigsch, F.; Jenzer, J.; Haegele, J.; Pikusa, M.; Tager, J.; Rodriguez-Nieto, S.; Guo, J.; Bouris, V.; Ribeiro, R.; Baroni, F.; Bedi, M. S.; Berry, S.; Losa, M.; Hornemann, S.; Kampmann, M.; Pelkmans, L.; Hoepfner, D.; Heutink, P.; Aguzzi, A.\nScore: 43.9, Published: 2023-07-20 DOI: 10.1101/2022.05.25.493370\nArrayed CRISPR libraries extend the scope of gene-perturbation screens but require large numbers of efficacious sgRNA-expressing vectors. Using a newly invented liquid-phase plasmid cloning methodology, we constructed genome-wide arrayed libraries for human gene ablation (19,936 plasmids), activation, and epigenetic silencing (22,442 plasmids). At least 76% of each plasmid preparation encoded an intact array of 4 non-overlapping sgRNAs designed to tolerate most human DNA polymorphisms. We achieved perturbation efficacies of 75-99%, 76-92% and up to 10,000x in deletion, silencing and activation experiments, respectively. Upon conversion into massively parallel lentiviral vectors, an arrayed activation screen of 1,634 human transcription factors yielded 11 novel regulators of the cellular prion protein PrPC. Furthermore, a screen using a pooled version of the ablation library identified 5 novel modifiers of autophagy that went undetected with either of two 1sgRNA libraries. The CRISPR libraries described here represent a powerful resource for the targeted perturbation of human protein-coding genes.\nDNA satellite and chromatin organization at house mouse centromeres and pericentromeres\nAuthors: Packiaraj, J.; Thakur, J.\nScore: 8.1, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549612\nCentromeres are essential for faithful chromosome segregation during mitosis and meiosis. However, the organization of satellite DNA and chromatin at mouse centromeres and pericentromeres is poorly understood due to the challenges of sequencing and assembling repetitive genomic regions. Using recently available PacBio long-read sequencing data from the C57BL/6 strain and chromatin profiling, we found that contrary to the previous reports of their highly homogeneous nature, centromeric and pericentromeric satellites display varied sequences and organization. We find that both centromeric minor satellites and pericentromeric major satellites exhibited sequence variations within and between arrays. While most arrays are continuous, a significant fraction is interspersed with non-satellite sequences, including transposable elements. Additionally, we investigated CENP-A and H3K9me3 chromatin organization at centromeres and pericentromeres using Chromatin immunoprecipitation sequencing (ChIP-seq). We found that the occupancy of CENP-A and H3K9me3 chromatin at centromeric and pericentric regions, respectively, is associated with increased sequence abundance and homogeneity at these regions. Furthermore, the transposable elements at centromeric regions are not part of functional centromeres as they lack CENP-A enrichment. Finally, we found that while H3K9me3 nucleosomes display a well-phased organization on major satellite arrays, CENP-A nucleosomes on minor satellite arrays lack phased organization. Interestingly, the homogeneous class of major satellites phase CENP-A and H3K27me3 nucleosomes as well, indicating that the nucleosome phasing is an inherent property of homogeneous major satellites. Overall, our findings reveal that house mouse centromeres and pericentromeres, which were previously thought to be highly homogenous, display significant diversity in satellite sequence, organization, and chromatin structure.\nA single cell atlas of sexual development in Plasmodium falciparum\nAuthors: Dogga, S. K.; Rop, J. C.; Cudini, J.; Farr, E.; Dara, A.; Ouologuem, D.; Djimde, A. A.; Talman, A. M.; Lawniczak, M. K. N.\nScore: 16.8, Published: 2023-07-17 DOI: 10.1101/2023.07.16.547819\nThe developmental decision made by malaria parasites to become sexual underlies all malaria transmission. Here, we describe a rich atlas of short and long-read single-cell transcriptomes of over 37,000 Plasmodium falciparum cells across intraerythrocytic asexual and sexual development. We used the atlas to explore transcriptional modules and exon usage along sexual development, and expanded it to include malaria parasites collected from a Malian individual naturally infected with multiple P. falciparum strains. We investigated genotypic and transcriptional heterogeneity within and among these wild strains at a single-cell level for the first time, finding considerable differential expression between different strains even within the same host. This work is a key addition to the Malaria Cell Atlas, enabling a deeper understanding of the biology and diversity of transmission stages. One sentence summaryThis addition to the Malaria Cell Atlas presents an analysis of sexual development and uses it to explore a natural infection.\n",
  "wordCount" : "2524",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549578">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549578" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549578">
        <p class="paperTitle">Concurrent profiling of multiscale 3D genome organization and gene expression in single mammalian cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549578" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549578" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhou, T.; Zhang, R.; Jia, D.; Doty, R. T.; Munday, A. D.; Gao, D.; Xin, L.; Abkowitz, J. L.; Duan, Z.; Ma, J.</p>
        <p class="info">Score: 24.6, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549578' target='https://doi.org/10.1101/2023.07.20.549578'> 10.1101/2023.07.20.549578</a></p>
        <p class="abstract">The organization of mammalian genomes within the nucleus features a complex, multiscale three-dimensional (3D) architecture. The functional significance of these 3D genome features, however, remains largely elusive due to limited single-cell technologies that can concurrently profile genome organization and transcriptional activities. Here, we report GAGE-seq, a highly scalable, robust single-cell co-assay that simultaneously measures 3D genome structure and transcriptome within the same cell. Employing GAGE-seq on mouse brain cortex and human bone marrow CD34&#43; cells, we comprehensively characterized the intricate relationships between 3D genome and gene expression. We found that these multiscale 3D genome features collectively inform cell type-specific gene expressions, hence contributing to defining cell identity at the single-cell level. Integration of GAGE-seq data with spatial transcriptomic data revealed in situ variations of the 3D genome in mouse cortex. Moreover, our observations of lineage commitment in normal human hematopoiesis unveiled notable discordant changes between 3D genome organization and gene expression, underscoring a complex, temporal interplay at the single-cell level that is more nuanced than previously appreciated. Together, GAGE-seq provides a powerful, cost-effective approach for interrogating genome structure and gene expression relationships at the single-cell level across diverse biological contexts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.550110">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.550110" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.550110">
        <p class="paperTitle">Sequencing 4.3 million mutations in wheat promoters to understand and modify gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.550110" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.550110" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, J.; Xiong, H.; Burguener, G. F.; Vasquez-Gross, H.; Liu, Q.; Debernardi, J. M.; Akhunova, A.; Campbell-Garland, K.; Kianian, S.; Brown-Guedira, G.; Pozniak, C.; Faris, J.; Akhunov, E.; Dubcovsky, J.</p>
        <p class="info">Score: 20.3, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.550110' target='https://doi.org/10.1101/2023.07.21.550110'> 10.1101/2023.07.21.550110</a></p>
        <p class="abstract">Wheat is an important contributor to global food security, and further improvements are required to feed a growing human population. New functional genetics and genomics tools can help us to understand the function of different genes and to engineer beneficial changes. In this study, we used a promoter capture assay to sequence 2-kb regions upstream of all high-confidence annotated genes from 1,513 mutagenized plants from the tetraploid wheat variety Kronos. We identified 4.3 million induced mutations with an accuracy of 99.8%, resulting in a mutation density of 41.9 mutations per kb. We also remapped Kronos exome capture reads to Chinese Spring RefSeq v1.1, identified 4.7 million mutations, and predicted their effects on annotated genes. Using these predictions, we identified 59% more non-synonymous substitutions and 49% more truncation mutations than in the original study. To show the biological value of the new promoter dataset, we selected two mutations within the promoter of the VRN-A1 vernalization gene. Both mutations, located within transcription factor binding sites, significantly altered VRN-A1 expression, and one reduced the number of spikelets per spike. These publicly available sequenced mutant datasets provide rapid and inexpensive access to induced variation in the promoters and coding regions of most wheat genes. These mutations can be used to understand and modulate gene expression and phenotypes for both basic and commercial applications, where limited governmental regulations can facilitate deployment. These mutant collections, together with gene editing, provide valuable tools to accelerate functional genetic studies in this economically important crop.

Significance StatementWe sequenced 4.3 million induced mutations in the promoters and 4.7 million in the coding regions of most wheat genes. We also show how this public resource can be used to understand gene function, modulate gene expression, and generate changes in valuable wheat agronomic traits.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549097" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549097">
        <p class="paperTitle">Recurrent chromosome destabilization through repeat-mediated rearrangements in a fungal pathogen</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fouche, S.; Oggenfuss, U.; McDonald, B. A.; Croll, D.</p>
        <p class="info">Score: 19.8, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549097' target='https://doi.org/10.1101/2023.07.14.549097'> 10.1101/2023.07.14.549097</a></p>
        <p class="abstract">Genomic instability caused by chromosomal rearrangements has severe consequences for organismal fitness and progression of cancerous cell lines. The triggers of destabilized chromosomes remain poorly understood but are often assumed to be associated with fragile sites. Here, we retrace a runaway chromosomal degeneration process observed in a fungal pathogen using telomere-to-telomere assemblies across an experimental pedigree. We show that the same fragile sites triggered reproducible, large-scale rearrangements through non-allelic recombination. Across the four-generation pedigree, chromosomal rearrangements were accompanied by non-disjunction events and caused aneuploid progeny to carry up to four chromosomal copies. We identify a specific transposable element as the most likely trigger for the repeated chromosomal degeneration. The element is associated with higher virulence of the pathogen and has undergone a burst increasing copy numbers across the genome. Chromosome sequences are also targeted by a genome defense mechanism active on duplicated sequences, which may contribute to decay. Our work identifies the exact sequence triggers initiating chromosome instability and perpetuating degenerative cycles. Dissecting proximate causes leading to run-away chromosomal degeneration expands our understanding of chromosomal evolution beyond cancer lines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549613">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549613" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549613">
        <p class="paperTitle">The International Weed Genomics Consortium: Community Resources for Weed Genomics Research</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549613" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549613" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Montgomery, J. S.; Morran, S.; MacGregor, D. R.; McElroy, J. S.; Neve, P.; Neto, C.; Vila-Aiub, M. M.; Sandoval, M. V.; Menendez, A. I.; Kreiner, J.; Fan, L.; Caicedo, A. L.; Maughan, P. J.; Martins, B. A. B.; Mika, J.; Collavo, A.; Merotto, A.; Subramanian, N. K.; Bagavathiannan, M. V.; Cutti, L.; Islam, M. M.; Gill, B. S.; Cicchillo, R.; Gast, R.; Soni, N.; Wright, T. R.; Zastrow-Hayes, G.; May, G.; Malone, J. M.; Sehgal, D.; Kaundun, S. S.; Dale, R. P.; Vorster, B. J.; Peters, B.; Lerchl, J.; Tranel, P. J.; Beffa, R.; Jugulam, M.; Fengler, K.; Llaca, V.; Patterson, E. L.; Gaines, T. A.</p>
        <p class="info">Score: 12.4, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549613' target='https://doi.org/10.1101/2023.07.19.549613'> 10.1101/2023.07.19.549613</a></p>
        <p class="abstract">The International Weed Genomics Consortium is a collaborative group of researchers focused on developing genomic resources for the study of weedy plants. Weeds are attractive systems for basic and applied research due to their impacts on agricultural systems and capacity to swiftly adapt in response to anthropogenic selection pressures. Our goal is to use genomic information to develop sustainable and effective weed control methods and to provide insights about biotic and abiotic stress tolerance to assist crop breeding. Here, we outline resources under development by the consortium and highlight areas of research that will be impacted by these enabling resources.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549370">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549370" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549370">
        <p class="paperTitle">Human Cytomegalovirus in breast milk is associated with milk composition, the infant gut microbiome, and infant growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549370" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549370" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Johnson, K. E.; Heisel, T.; Fields, D. A.; Isganaitis, E.; Jacobs, K. M.; Knights, D.; Lock, E. F.; Rudolph, M. C.; Gale, C. A.; Schleiss, M. R.; Albert, F. W.; Demerath, E. W.; Blekhman, R.</p>
        <p class="info">Score: 12.4, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549370' target='https://doi.org/10.1101/2023.07.19.549370'> 10.1101/2023.07.19.549370</a></p>
        <p class="abstract">Human cytomegalovirus (CMV) is a highly prevalent herpesvirus that is often transmitted to the neonate via breast milk. Postnatal CMV transmission can have negative health consequences for preterm and immunocompromised infants, but any effects on healthy term infants are thought to be benign. Furthermore, the impact of CMV on the composition of the hundreds of bioactive factors in human milk has not been tested. Here, we utilize a cohort of exclusively breastfeeding full term mother-infant pairs to test for differences in the milk transcriptome and metabolome associated with CMV, and the impact of CMV in breast milk on the infant gut microbiome and infant growth. We find upregulation of the indoleamine 2,3-dioxygenase (IDO) tryptophan-to-kynurenine metabolic pathway in CMV&#43; milk samples, and that CMV&#43; milk is associated with decreased Bifidobacterium in the infant gut. Our data indicate a complex relationship between milk CMV, milk kynurenine, and infant growth; with kynurenine positively correlated, and CMV viral load negatively correlated, with infant weight-for-length at 1 month of age. These results suggest CMV transmission, CMV-related changes in milk composition, or both may be modulators of full term infant development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549384">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549384" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549384">
        <p class="paperTitle">The SWI/SNF nucleosome remodeler constrains enhancer activity during Drosophila wing development</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549384" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549384" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Niederhuber, M. J.; Leatham-Jensen, M.; McKay, D. J.</p>
        <p class="info">Score: 11.8, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549384' target='https://doi.org/10.1101/2023.07.17.549384'> 10.1101/2023.07.17.549384</a></p>
        <p class="abstract">Chromatin remodeling is central to the dynamic changes in gene expression that drive cell fate determination. During development, the sets of enhancers that are accessible for use change globally as cells transition between stages. While transcription factors and nucleosome remodeling complexes are known to work together to control access to enhancers, it is unclear how the short stretches of DNA that they individually unmask yield the kilobase-sized accessible regions that are characteristic of active enhancers. Here, we performed a genetic screen to investigate the role of ATP-dependent nucleosome remodeling complexes in the control of dynamic enhancer activity. We find that the Drosophila BAP complex, a member of the SWI/SNF family of nucleosome remodelers, is required for repression of a temporally dynamic enhancer, brdisc. Contrary to expectations, we find that the BAP-specific subunit, Osa, is dispensable for mediating changes in chromatin accessibility between early and late stages of wing development. Instead, we find that Osa is required to constrain the levels of brdisc activity in imaginal wing discs when the enhancer is normally active. Genome-wide profiling reveals that Osa binds directly to the brdisc enhancer as well as thousands of other developmentally dynamic regulatory sites, including multiple genes encoding components and targets of the Notch signaling pathway. We find that Osa loss of function results in development of ectopic sensory structures that are normally patterned by Notch signaling early in wing development. Moreover, we find that Osa loss of function results in hyperactivation of the Delta gene, which encodes the Notch ligand. Together, these findings indicate that proper constraint of enhancer activity is necessary for regulation of dose-dependent developmental events.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549677">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549677" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549677">
        <p class="paperTitle">The human genome&#39;s vocabulary as proposed by the DNA language model GROVER</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549677" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549677" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanabria, M.; Hirsch, J.; Poetsch, A. R.</p>
        <p class="info">Score: 11.0, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549677' target='https://doi.org/10.1101/2023.07.19.549677'> 10.1101/2023.07.19.549677</a></p>
        <p class="abstract">Large Language Models (LLMs) on natural language have achieved a level of performance that allows the generation of coherent and syntactically correct text. DNA sequence of genomes follows rules similar to natural language, but a distinguishing factor is the absence of a concept analogous to words. We established byte-pair tokenization on the human genome and trained a foundation language model called GROVER (&#34;Genome Rules Obtained Via Extracted Representations&#34;) to select the optimal vocabulary with a custom fine-tuning task of next-k-mer prediction. We thus defined a dictionary of words/tokens in the human genome that best carries the information content for DNA language models. Analyzing GROVERs learned representations, we observed that token embeddings primarily encode information related to their frequency, sequence content, and length. Some tokens are almost exclusively localized in repeats, while the vast majority widely distributes over the genome. The model also learns context and lexical ambiguity. Average embeddings of genomic regions relate to functional genomics annotation and thus indicate that GROVER has learned these structures purely from the contextual relationships of tokens. That we can extract functional annotations from the genome, purely based on sequence representation to the trained model, highlights the extent of information content encoded by the sequence. This is supported by fine-tuning tasks on genome biology with questions of promoter identity and protein-DNA binding. GROVER learns sequence context, a sense for grammatical structures and language rules in the genome. This knowledge can be extracted and used to compose a grammar book for the code of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.05.25.493370">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.05.25.493370" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.05.25.493370">
        <p class="paperTitle">Robust and Versatile Arrayed Libraries for Human Genome-Wide CRISPR Activation, Deletion and Silencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.05.25.493370" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.05.25.493370" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yin, J.-A.; Frick, L.; Scheidmann, M. C.; Liu, T.; Trevisan, C.; Dhingra, A.; Spinelli, A.; Wu, Y.; Yao, L.; Vena, D. L.; Knapp, B.; De Cecco, E.; Ging, K.; Armani, A.; Oakeley, E.; Nigsch, F.; Jenzer, J.; Haegele, J.; Pikusa, M.; Tager, J.; Rodriguez-Nieto, S.; Guo, J.; Bouris, V.; Ribeiro, R.; Baroni, F.; Bedi, M. S.; Berry, S.; Losa, M.; Hornemann, S.; Kampmann, M.; Pelkmans, L.; Hoepfner, D.; Heutink, P.; Aguzzi, A.</p>
        <p class="info">Score: 43.9, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.05.25.493370' target='https://doi.org/10.1101/2022.05.25.493370'> 10.1101/2022.05.25.493370</a></p>
        <p class="abstract">Arrayed CRISPR libraries extend the scope of gene-perturbation screens but require large numbers of efficacious sgRNA-expressing vectors. Using a newly invented liquid-phase plasmid cloning methodology, we constructed genome-wide arrayed libraries for human gene ablation (19,936 plasmids), activation, and epigenetic silencing (22,442 plasmids). At least 76% of each plasmid preparation encoded an intact array of 4 non-overlapping sgRNAs designed to tolerate most human DNA polymorphisms. We achieved perturbation efficacies of 75-99%, 76-92% and up to 10,000x in deletion, silencing and activation experiments, respectively. Upon conversion into massively parallel lentiviral vectors, an arrayed activation screen of 1,634 human transcription factors yielded 11 novel regulators of the cellular prion protein PrPC. Furthermore, a screen using a pooled version of the ablation library identified 5 novel modifiers of autophagy that went undetected with either of two 1sgRNA libraries. The CRISPR libraries described here represent a powerful resource for the targeted perturbation of human protein-coding genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549612">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549612" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549612">
        <p class="paperTitle">DNA satellite and chromatin organization at house mouse centromeres and pericentromeres</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549612" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549612" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Packiaraj, J.; Thakur, J.</p>
        <p class="info">Score: 8.1, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549612' target='https://doi.org/10.1101/2023.07.18.549612'> 10.1101/2023.07.18.549612</a></p>
        <p class="abstract">Centromeres are essential for faithful chromosome segregation during mitosis and meiosis. However, the organization of satellite DNA and chromatin at mouse centromeres and pericentromeres is poorly understood due to the challenges of sequencing and assembling repetitive genomic regions. Using recently available PacBio long-read sequencing data from the C57BL/6 strain and chromatin profiling, we found that contrary to the previous reports of their highly homogeneous nature, centromeric and pericentromeric satellites display varied sequences and organization. We find that both centromeric minor satellites and pericentromeric major satellites exhibited sequence variations within and between arrays. While most arrays are continuous, a significant fraction is interspersed with non-satellite sequences, including transposable elements. Additionally, we investigated CENP-A and H3K9me3 chromatin organization at centromeres and pericentromeres using Chromatin immunoprecipitation sequencing (ChIP-seq). We found that the occupancy of CENP-A and H3K9me3 chromatin at centromeric and pericentric regions, respectively, is associated with increased sequence abundance and homogeneity at these regions. Furthermore, the transposable elements at centromeric regions are not part of functional centromeres as they lack CENP-A enrichment. Finally, we found that while H3K9me3 nucleosomes display a well-phased organization on major satellite arrays, CENP-A nucleosomes on minor satellite arrays lack phased organization. Interestingly, the homogeneous class of major satellites phase CENP-A and H3K27me3 nucleosomes as well, indicating that the nucleosome phasing is an inherent property of homogeneous major satellites. Overall, our findings reveal that house mouse centromeres and pericentromeres, which were previously thought to be highly homogenous, display significant diversity in satellite sequence, organization, and chromatin structure.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.547819">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.547819" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.547819">
        <p class="paperTitle">A single cell atlas of sexual development in Plasmodium falciparum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.547819" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.547819" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dogga, S. K.; Rop, J. C.; Cudini, J.; Farr, E.; Dara, A.; Ouologuem, D.; Djimde, A. A.; Talman, A. M.; Lawniczak, M. K. N.</p>
        <p class="info">Score: 16.8, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.547819' target='https://doi.org/10.1101/2023.07.16.547819'> 10.1101/2023.07.16.547819</a></p>
        <p class="abstract">The developmental decision made by malaria parasites to become sexual underlies all malaria transmission. Here, we describe a rich atlas of short and long-read single-cell transcriptomes of over 37,000 Plasmodium falciparum cells across intraerythrocytic asexual and sexual development. We used the atlas to explore transcriptional modules and exon usage along sexual development, and expanded it to include malaria parasites collected from a Malian individual naturally infected with multiple P. falciparum strains. We investigated genotypic and transcriptional heterogeneity within and among these wild strains at a single-cell level for the first time, finding considerable differential expression between different strains even within the same host. This work is a key addition to the Malaria Cell Atlas, enabling a deeper understanding of the biology and diversity of transmission stages.

One sentence summaryThis addition to the Malaria Cell Atlas presents an analysis of sexual development and uses it to explore a natural infection.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
