<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Ancient genomes illuminate Eastern Arabian population history and adaptation against malaria
Authors: Martiniano, R.; Haber, M.; Almarri, M. A.; Mattiangeli, V.; Kuijpers, M. C. M.; Chamel, B.; Breslin, E. M.; Littleton, J.; Almahari, S.; Aloraifi, F.; Bradley, D. G.; Lombard, P.; Durbin, R.
Score: 120.3, Published: 2023-10-01 DOI: 10.1101/2023.10.01.559299
The harsh climate of Arabia has posed challenges in generating ancient DNA from the region, hindering the direct examination of ancient genomes for understanding the demographic processes that shaped Arabian populations.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Ancient genomes illuminate Eastern Arabian population history and adaptation against malaria
Authors: Martiniano, R.; Haber, M.; Almarri, M. A.; Mattiangeli, V.; Kuijpers, M. C. M.; Chamel, B.; Breslin, E. M.; Littleton, J.; Almahari, S.; Aloraifi, F.; Bradley, D. G.; Lombard, P.; Durbin, R.
Score: 120.3, Published: 2023-10-01 DOI: 10.1101/2023.10.01.559299
The harsh climate of Arabia has posed challenges in generating ancient DNA from the region, hindering the direct examination of ancient genomes for understanding the demographic processes that shaped Arabian populations." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-08T10:37:16+00:00" />
<meta property="article:modified_time" content="2023-10-08T10:37:16+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Ancient genomes illuminate Eastern Arabian population history and adaptation against malaria
Authors: Martiniano, R.; Haber, M.; Almarri, M. A.; Mattiangeli, V.; Kuijpers, M. C. M.; Chamel, B.; Breslin, E. M.; Littleton, J.; Almahari, S.; Aloraifi, F.; Bradley, D. G.; Lombard, P.; Durbin, R.
Score: 120.3, Published: 2023-10-01 DOI: 10.1101/2023.10.01.559299
The harsh climate of Arabia has posed challenges in generating ancient DNA from the region, hindering the direct examination of ancient genomes for understanding the demographic processes that shaped Arabian populations."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Ancient genomes illuminate Eastern Arabian population history and adaptation against malaria\nAuthors: Martiniano, R.; Haber, M.; Almarri, M. A.; Mattiangeli, V.; Kuijpers, M. C. M.; Chamel, B.; Breslin, E. M.; Littleton, J.; Almahari, S.; Aloraifi, F.; Bradley, D. G.; Lombard, P.; Durbin, R.\nScore: 120.3, Published: 2023-10-01 DOI: 10.1101/2023.10.01.559299\nThe harsh climate of Arabia has posed challenges in generating ancient DNA from the region, hindering the direct examination of ancient genomes for understanding the demographic processes that shaped Arabian populations.",
  "keywords": [
    
  ],
  "articleBody": " Ancient genomes illuminate Eastern Arabian population history and adaptation against malaria\nAuthors: Martiniano, R.; Haber, M.; Almarri, M. A.; Mattiangeli, V.; Kuijpers, M. C. M.; Chamel, B.; Breslin, E. M.; Littleton, J.; Almahari, S.; Aloraifi, F.; Bradley, D. G.; Lombard, P.; Durbin, R.\nScore: 120.3, Published: 2023-10-01 DOI: 10.1101/2023.10.01.559299\nThe harsh climate of Arabia has posed challenges in generating ancient DNA from the region, hindering the direct examination of ancient genomes for understanding the demographic processes that shaped Arabian populations. In this study, we report whole genome sequence data obtained from four Tylos-period individuals from Bahrain. Their genetic ancestry can be modelled as a mixture of sources from ancient Anatolia, Levant and Iran/Caucasus, with variation between individuals suggesting population heterogeneity in Bahrain before the onset of Islam. We identify the G6PD Mediterranean mutation associated with malaria-resistance in three out of four ancient Bahraini samples and estimate that it rose in frequency in Eastern Arabia from 5-6 kya onwards, around the time agriculture appeared in the region. Our study provides the first characterisation of the genetic composition of ancient Arabians, shedding light on the population history of Bahrain and demonstrating the feasibility of aDNA studies in the region.\nEpithelial zonation along the mouse and human small intestine defines five discrete metabolic domains\nAuthors: Zwick, R. K.; Kasparek, P.; Palikuqi, B.; Viragova, S.; Weichselbaum, L.; McGinnis, C. S.; McKinley, K. L.; Rathnayake, A.; Vaka, D.; Nguyen, V.; Trentesaux, C.; Reyes, E.; Gupta, A.; Gartner, Z. J.; Locksley, R. M.; Gardner, J. M.; Itzkovitz, S.; Boffelli, D.; Klein, O. D.\nScore: 63.9, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558726\nA key aspect of nutrient absorption is the exquisite division of labor across the length of the small intestine, with individual classes of micronutrients taken up at different positions. For millennia, the small intestine was thought to comprise three segments with indefinite borders: the duodenum, jejunum, and ileum. By examining fine-scale longitudinal segmentation of the mouse and human small intestines, we identified transcriptional signatures and upstream regulatory factors that define five domains of nutrient absorption, distinct from the three traditional sections. Spatially restricted expression programs were most prominent in nutrient-absorbing enterocytes but initially arose in intestinal stem cells residing in three regional populations. While a core signature was maintained across mice and humans with different diets and environments, domain properties were influenced by dietary changes. We established the functions of Ppar- and Cdx1 in patterning lipid metabolism in distal domains and generated a predictive model of additional transcription factors that direct domain identity. Molecular domain identity can be detected with machine learning, representing the first systematic method to computationally identify specific intestinal regions in mice. These findings provide a foundational framework for the identity and control of longitudinal zonation of absorption along the proximal:distal small intestinal axis.\nPostglacial genomes from foragers across Northern Eurasia reveal prehistoric mobility associated with the spread of the Uralic and Yeniseian languages\nAuthors: Zeng, T. C.; Vyazov, L. A.; Kim, A.; Flegontov, P.; Sirak, K.; Maier, R.; Lazaridis, I.; Akbari, A.; Frachetti, M.; Tishkin, A. A.; Ryabogina, N. E.; Agapov, S. A.; Agapov, D. S.; Alekseev, A. N.; Boeskorov, G. G.; Chizhevsky, A. A.; Derevianko, A. P.; Dyakonov, V. M.; Enshin, D. N.; Fribus, A. V.; Frolov, Y. V.; Grushin, S. P.; Khokhlov, A. A.; Kitov, E. P.; Kosintsev, P.; Kovtun, I. V.; Kiryushin, K. Y.; Kiryushin, Y. F.; Makarov, N. P.; Morozov, V. V.; Nikolaev, E. N.; Rykun, M. P.; Savenkova, T. M.; Shchelchkova, M. V.; Skochina, S. N.; Shirokov, V.; Sherstobitova, O. S.; Slepchenko, S. M.\nScore: 38.0, Published: 2023-10-04 DOI: 10.1101/2023.10.01.560332\nThe North Eurasian forest and forest-steppe zones have sustained millennia of sociocultural connections among northern peoples. We present genome-wide ancient DNA data for 181 individuals from this region spanning the Mesolithic, Neolithic and Bronze Age. We find that Early to Mid-Holocene hunter-gatherer populations from across the southern forest and forest-steppes of Northern Eurasia can be characterized by a continuous gradient of ancestry that remained stable for millennia, ranging from fully West Eurasian in the Baltic region to fully East Asian in the Transbaikal region. In contrast, cotemporaneous groups in far Northeast Siberia were genetically distinct, retaining high levels of continuity from a population that was the primary source of ancestry for Native Americans. By the mid-Holocene, admixture between this early Northeastern Siberian population and groups from Inland East Asia and the Amur River Basin produced two distinctive populations in eastern Siberia that played an important role in the genetic formation of later people. Ancestry from the first population, Cis-Baikal Late Neolithic-Bronze Age (Cisbaikal_LNBA), is found substantially only among Yeniseian-speaking groups and those known to have admixed with them. Ancestry from the second, Yakutian Late Neolithic-Bronze Age (Yakutia_LNBA), is strongly associated with present-day Uralic speakers. We show how Yakutia_LNBA ancestry spread from an east Siberian origin ~4.5kya, along with subclades of Y-chromosome haplogroup N occurring at high frequencies among present-day Uralic speakers, into Western and Central Siberia in communities associated with Seima-Turbino metallurgy: a suite of advanced bronze casting techniques that spread explosively across an enormous region of Northern Eurasia ~4.0kya. However, the ancestry of the 16 Seima-Turbino-period individuals--the first reported from sites with this metallurgy--was otherwise extraordinarily diverse, with partial descent from Indo-Iranian-speaking pastoralists and multiple hunter-gatherer populations from widely separated regions of Eurasia. Our results provide support for theories suggesting that early Uralic speakers at the beginning of their westward dispersal where involved in the expansion of Seima-Turbino metallurgical traditions, and suggests that both cultural transmission and migration were important in the spread of Seima-Turbino material culture.\nPeripheral Blood Single-Cell Sequencing Uncovers Common and Specific Immune Aberrations in Fibrotic Lung Diseases\nAuthors: Zhao, A. Y.; Unterman, A.; Abu Hussein, N.; Sharma, P.; Flint, J.; Yan, X.; Adams, T. S.; Justet, A.; Sumida, T. S.; Zhao, J.; Schupp, J. C.; Raredon, M. S. B.; Ahangari, F. C.; Zhang, Y.; Buendia-Roldan, I.; Adegunsoye, A.; Sperling, A. I.; Prasse, A.; Ryu, C.; Herzog, E.; Selman, M.; Pardo, A.; Kaminski, N.\nScore: 20.7, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558301\nRationale and ObjectivesThe extent and commonality of peripheral blood immune aberrations in fibrotic interstitial lung diseases are not well characterized. In this study, we aimed to identify common and distinct immune aberrations in patients with idiopathic pulmonary fibrosis (IPF) and fibrotic hypersensitivity pneumonitis (FHP) using cutting-edge single-cell profiling technologies. MethodsSingle-cell RNA sequencing was performed on patients and healthy controls peripheral blood and bronchoalveolar lavage samples using 10X Genomics 5 gene expression and V(D)J profiling. Cell type composition, transcriptional profiles, cellular trajectories and signaling, and T and B cell receptor repertoires were studied. The standard Seurat R pipeline was followed for cell type composition and differential gene expression analyses. Transcription factor activity was imputed using the DoRothEA-VIPER algorithm. Pseudotime analyses were conducted using Monocle3, while RNA velocity analyses were performed with Velocyto, scVelo, and CellRank. Cell-cell connectomics were assessed using the Connectome R package. V(D)J analyses were conducted using CellRanger and Immcantation frameworks. Across all analyses, disease group differences were assessed using the Wilcoxon rank-sum test. Measurements and Main Results327,990 cells from 83 samples were profiled. Overall, changes in monocytes were common to IPF and FHP, whereas lymphocytes exhibited disease-specific aberrations. Both diseases displayed enrichment of CCL3hi/CCL4hi CD14+ monocytes (p\u003c2.2e-16) and S100Ahi CD14+ monocytes (p\u003c2.2e-16) versus controls. Trajectory and RNA velocity analysis suggested that pro-fibrotic macrophages observed in BAL originated from peripheral blood monocytes. Lymphocytes exhibited disease-specific aberrations, with CD8+ GZMKhi T cells and activated B cells primarily enriched in FHP patients. V(D)J analyses revealed unique T and B cell receptor complementarity-determining region 3 (CDR3) amino acid compositions (p\u003c0.05) in FHP and significant IgA enrichment in IPF (p\u003c5.2e-7). ConclusionsWe identified common and disease-specific immune mechanisms in IPF and FHP; S100Ahi monocytes and SPP1hi macrophages are common to IPF and FHP, whereas GMZKhi T lymphocytes and T and B cell receptor repertoires were unique in FHP. Our findings open novel strategies for the diagnosis and treatment of IPF and FHP.\nAncestral genetic components are consistently associated with the complex trait landscape in European biobanks\nAuthors: Pankratov, V.; Mezzavilla, M.; Aneli, S.; Fusco, D.; Wilson, J. F.; Metspalu, M.; Provero, P.; Pagani, L.; Marnetto, D.\nScore: 14.9, Published: 2023-10-05 DOI: 10.1101/2023.10.04.560881\nThe genetic structure in Europe was mostly shaped by admixture between the Western Hunter-Gatherer, Anatolian Neolithic and Steppe's Yamnaya ancestral components. Such structure is regarded as a confounder in GWAS and follow-up studies, and gold-standard methods exist to correct for it. However, it is still poorly understood to which extent these ancestral components contribute to complex trait variation in present-day Europe. In this work we harness the UK Biobank to address this question. By extensive demographic simulations and incorporating previous results obtained using the Estonian Biobank, we carefully evaluate the significance and scope of our findings. Heart rate, platelet count, monocyte percentage and many other traits show stratification similar to height and pigmentation traits, likely targets of selection and divergence across ancestral groups. The consistency of our results across biobanks indicates that these ancestry-specific genetic predispositions act as a source of variability and as potential confounders in Europe as a whole.\nImproved Resolution of Highly Pathogenic Avian Influenza Virus Haemagglutinin Cleavage Site Using Oxford Nanopore R10 Sequencing Chemistry\nAuthors: Ratcliff, J.; Merritt, B.; Gooden, H.; Siegers, J.; Srikanth, A.; Yann, S.; Kol, S.; Sin, S.; Tok, S.; Karlsson, E. A.; Thielen, P. M.\nScore: 14.3, Published: 2023-10-02 DOI: 10.1101/2023.09.30.560331\nHighly pathogenic avian influenza viruses continue to pose global risks to One Health, including agriculture, public, and animal health. Rapid and accurate genomic surveillance is critical for monitoring viral mutations, tracing transmission, and guiding interventions in near real-time. Oxford Nanopore sequencing holds promise for real-time influenza genotyping, but data quality from R9 chemistry has limited its adoption due to challenges resolving low-complexity regions such as the biologically critical hemagglutinin cleavage site, a homopolymer of basic amino acids that distinguish highly pathogenic strains. In this study, human and avian influenza isolates (n=45) from Cambodia were sequenced using both R9.4.1 and R10.4.1 flow cells and chemistries to evaluate performance between approaches. Overall, R10.4.1 yielded increased data output with higher average quality compared to R9.4.1, producing improved consensus sequences using a reference-based bioinformatics approach. R10.4.1 had significantly lower minor population insertion and deletion frequencies, driven by improved performance in low sequence complexity regions prone to insertion and deletion errors, such as homopolymers. Within the hemagglutinin cleavage site, R10.4.1 resolved the correct motif in 90% of genomes compared to only 60% with R9.4.1. Further examination showed reduced frameshift mutations in consensus sequences generated with R10.4.1 that could result in incorrectly classified virulence. Improved consensus genome quality from nanopore sequencing approaches, especially across biologically important low-complexity regions, is critical to reduce subjective hand-curation and will improve local and global genomic surveillance responses.\nDeepurify: a multi-modal deep language model to remove contamination from metagenome-assembled genomes\nAuthors: ZOU, B.; Wang, J.; Ding, Y.; Zhang, Z.; Yufen, H.; Fang, X.; Cheung, K. C.; See, S.; Zhang, L.\nScore: 14.9, Published: 2023-09-29 DOI: 10.1101/2023.09.27.559668\nMetagenome-assembled genomes (MAGs) offer valuable insights into the exploration of microbial dark matter using metagenomic sequencing data. However, there is a growing concern that contamination in MAGs may significantly impact the downstream analysis results. Existing MAG decontamination methods heavily rely on marker genes but do not fully leverage genomic sequences. To address the limitations, we have introduced a novel decontamination approach named Deepurify, which utilizes a multi-modal deep language model employing contrastive learning to learn taxonomic similarities of genomic sequences. Deepurify utilizes inferred taxonomic lineages to guide the allocation of contigs into a MAG-separated tree and employs a tree traversal strategy for maximizing the total number of medium- and high-quality MAGs. Extensive experiments were conducted on two simulated datasets, CAMI I, and human gut metagenomic sequencing data. These results demonstrate that Deepurify significantly outperforms other decontamination methods.\nHidden in the sand: phylogenomics unravel an unexpected evolutionary history on the desert-adapted vipers of the genus Cerastes\nAuthors: Mochales-Riano, G.; Burriel-Carranza, B.; Barros, M. I.; Velo-Anton, G.; Talavera, A.; Spilani, L.; Tejero-Cicuendez, H.; Andre-Crochet, P.; Piris, A.; Garcia-Cardenete, L.; Busais, S.; Els, J.; Shobrak, M.; Carlos-Brito, J.; Smid, J.; Carranza, S.; Martinez-Freiria, F.\nScore: 11.9, Published: 2023-10-04 DOI: 10.1101/2023.10.03.560686\nThe desert vipers of the genus Cerastes are a small clade of medically important venomous snakes within the family Viperidae. According to published morphological and molecular studies, the group is comprised by four species: two morphologically similar and phylogenetically sister taxa, the African horned viper (Cerastes cerastes) and the Arabian horned viper (Cerastes gasperettii); a more distantly related species, the Saharan sand viper (Cerastes vipera), and the enigmatic Bohme's sand viper (Cerastes boehmei), only known from a single specimen in captivity allegedly captured in Central Tunisia. In this study, we analyzed one mitochondrial marker (COI) as well as genome-wide data (ddRAD sequencing) from 28 and 41 samples, respectively, covering the entire distribution range of the genus to explore the population genomics, phylogenomic relationships and introgression patterns within the genus Cerastes. Additionally, and to provide insights into the mode of diversification of the group, we carried out niche overlap analyses considering climatic and habitat variables. Both nuclear phylogenomic reconstructions and population structure analyses have unveiled an unexpected evolutionary history for the genus Cerastes, which sharply contradicts the morphological similarities and previously published mitochondrial approaches. Cerastes cerastes and C. vipera are recovered as sister taxa whilst C. gasperettii is a sister taxon to the clade formed by these two species. We found a relatively high niche overlap (OI \u003e 0.7) in both climatic and habitat variables between C. cerastes and C. vipera, contradicting a potential scenario of sympatric speciation. These results are in line with the introgression found between the northwestern African populations of C. cerastes and C. vipera. Finally, our genomic data confirms the existence of a lineage of C. cerastes in Arabia. All these results highlight the importance of genome-wide data over few genetic markers to study the evolutionary history of species.\nGenomic comparison of the temperate coral Astrangia poculata with tropical corals yields insights into winter quiescence, innate immunity, and sexual reproduction\nAuthors: Stankiewicz, K. H.; Guiglielmoni, N.; Kitchen, S. A.; Flot, J.-F.; Barott, K. L.; Davies, S. W.; Finnerty, J. R.; Grace, S. P.; Kaufman, L. S.; Putnam, H. M.; Rotjan, R. D.; Sharp, K. H.; Baums, I. B.\nScore: 11.9, Published: 2023-09-23 DOI: 10.1101/2023.09.22.558704\nFacultatively symbiotic corals provide important experimental models to explore the establishment, maintenance, and breakdown of the mutualism between corals and members of the algal family Symbiodiniaceae. The temperate coral Astrangia poculata is one such model as it is not only facultatively symbiotic, but also occurs across a broad temperature and latitudinal gradient. Here, we report the de novo chromosome-scale assembly and annotation of the A. poculata genome. Though widespread segmental/tandem duplications of genomic regions were detected, we did not find strong evidence of a whole genome duplication (WGD) event. Comparison of the gene arrangement between A. poculata and the tropical coral Acropora millepora revealed 56.38% of the orthologous genes were conserved in syntenic blocks despite [~]415 million years of divergence. Gene families related to sperm hyperactivation and innate immunity, including lectins, were found to contain more genes in A. millepora relative to A. poculata. Sperm hyperactivation in A. millepora is expected given the extreme requirements of gamete competition during mass spawning events in tropical corals, while lectins are important in the establishment of coral-algal symbiosis. By contrast, gene families involved in sleep promotion, feeding suppression, and circadian sleep/wake cycle processes were expanded in A. poculata. These expanded gene families may play a role in A. poculatas ability to enter a dormancy-like state (\"winter quiescence\") to survive freezing temperatures at the northern edges of the species range.\nIntegrated characterization of cell types, states and molecular programs in disseminated appendiceal neoplasms\nAuthors: Bui, L. T.; Cao, X.; Wang, J. T.; Meng, F.; Feng, M.; Arvanitis, L.; Mannan, R.; Woo, Y.; Idrees, K.; Banovich, N.; Raoof, M.\nScore: 10.7, Published: 2023-10-01 DOI: 10.1101/2023.09.28.559817\nAppendiceal neoplasms include a heterogeneous group of epithelial and non-epithelial tumors with varying malignant potential. Despite the rise in incidence of appendiceal neoplasms in recent years, limited progress has been made in the understanding, management and therapeutic treatment. To comprehensively characterize the cell types and molecular mechanisms driving cellular remodeling in epithelial appendiceal neoplasms, we performed an integrated scRNA-seq study. We analyzed 126,998 cells from 16 appendix samples (11 peritoneal metastases samples, 5 healthy controls) and identified 33 distinct cell types/cell states with seven being cancer-specific. Highlights of our study include the characterization of tumor cells across the histologic spectrum, the identification of a novel cancer-associated-fibroblast (CAF) subtypes (fiCAFs) and the identification of pathologic-specific cellular crosstalk between tumor cells and the tumor microenvironment (TME). Together, our study provides a high-resolution insight into the complexity and heterogeneity of epithelial appendiceal neoplasms and a valuable resource for therapeutic strategies.\n",
  "wordCount" : "2789",
  "inLanguage": "en",
  "datePublished": "2023-10-08T10:37:16Z",
  "dateModified": "2023-10-08T10:37:16Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on October 8, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.559299">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.559299" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.559299">
        <p class="paperTitle">Ancient genomes illuminate Eastern Arabian population history and adaptation against malaria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.559299" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.559299" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martiniano, R.; Haber, M.; Almarri, M. A.; Mattiangeli, V.; Kuijpers, M. C. M.; Chamel, B.; Breslin, E. M.; Littleton, J.; Almahari, S.; Aloraifi, F.; Bradley, D. G.; Lombard, P.; Durbin, R.</p>
        <p class="info">Score: 120.3, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.559299' target='https://doi.org/10.1101/2023.10.01.559299'> 10.1101/2023.10.01.559299</a></p>
        <p class="abstract">The harsh climate of Arabia has posed challenges in generating ancient DNA from the region, hindering the direct examination of ancient genomes for understanding the demographic processes that shaped Arabian populations. In this study, we report whole genome sequence data obtained from four Tylos-period individuals from Bahrain. Their genetic ancestry can be modelled as a mixture of sources from ancient Anatolia, Levant and Iran/Caucasus, with variation between individuals suggesting population heterogeneity in Bahrain before the onset of Islam. We identify the G6PD Mediterranean mutation associated with malaria-resistance in three out of four ancient Bahraini samples and estimate that it rose in frequency in Eastern Arabia from 5-6 kya onwards, around the time agriculture appeared in the region. Our study provides the first characterisation of the genetic composition of ancient Arabians, shedding light on the population history of Bahrain and demonstrating the feasibility of aDNA studies in the region.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558726">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558726" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558726">
        <p class="paperTitle">Epithelial zonation along the mouse and human small intestine defines five discrete metabolic domains</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558726" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558726" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zwick, R. K.; Kasparek, P.; Palikuqi, B.; Viragova, S.; Weichselbaum, L.; McGinnis, C. S.; McKinley, K. L.; Rathnayake, A.; Vaka, D.; Nguyen, V.; Trentesaux, C.; Reyes, E.; Gupta, A.; Gartner, Z. J.; Locksley, R. M.; Gardner, J. M.; Itzkovitz, S.; Boffelli, D.; Klein, O. D.</p>
        <p class="info">Score: 63.9, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558726' target='https://doi.org/10.1101/2023.09.20.558726'> 10.1101/2023.09.20.558726</a></p>
        <p class="abstract">A key aspect of nutrient absorption is the exquisite division of labor across the length of the small intestine, with individual classes of micronutrients taken up at different positions. For millennia, the small intestine was thought to comprise three segments with indefinite borders: the duodenum, jejunum, and ileum. By examining fine-scale longitudinal segmentation of the mouse and human small intestines, we identified transcriptional signatures and upstream regulatory factors that define five domains of nutrient absorption, distinct from the three traditional sections. Spatially restricted expression programs were most prominent in nutrient-absorbing enterocytes but initially arose in intestinal stem cells residing in three regional populations. While a core signature was maintained across mice and humans with different diets and environments, domain properties were influenced by dietary changes. We established the functions of Ppar- and Cdx1 in patterning lipid metabolism in distal domains and generated a predictive model of additional transcription factors that direct domain identity. Molecular domain identity can be detected with machine learning, representing the first systematic method to computationally identify specific intestinal regions in mice. These findings provide a foundational framework for the identity and control of longitudinal zonation of absorption along the proximal:distal small intestinal axis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560332">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560332" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560332">
        <p class="paperTitle">Postglacial genomes from foragers across Northern Eurasia reveal prehistoric mobility associated with the spread of the Uralic and Yeniseian languages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560332" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560332" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zeng, T. C.; Vyazov, L. A.; Kim, A.; Flegontov, P.; Sirak, K.; Maier, R.; Lazaridis, I.; Akbari, A.; Frachetti, M.; Tishkin, A. A.; Ryabogina, N. E.; Agapov, S. A.; Agapov, D. S.; Alekseev, A. N.; Boeskorov, G. G.; Chizhevsky, A. A.; Derevianko, A. P.; Dyakonov, V. M.; Enshin, D. N.; Fribus, A. V.; Frolov, Y. V.; Grushin, S. P.; Khokhlov, A. A.; Kitov, E. P.; Kosintsev, P.; Kovtun, I. V.; Kiryushin, K. Y.; Kiryushin, Y. F.; Makarov, N. P.; Morozov, V. V.; Nikolaev, E. N.; Rykun, M. P.; Savenkova, T. M.; Shchelchkova, M. V.; Skochina, S. N.; Shirokov, V.; Sherstobitova, O. S.; Slepchenko, S. M.</p>
        <p class="info">Score: 38.0, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560332' target='https://doi.org/10.1101/2023.10.01.560332'> 10.1101/2023.10.01.560332</a></p>
        <p class="abstract">The North Eurasian forest and forest-steppe zones have sustained millennia of sociocultural connections among northern peoples. We present genome-wide ancient DNA data for 181 individuals from this region spanning the Mesolithic, Neolithic and Bronze Age. We find that Early to Mid-Holocene hunter-gatherer populations from across the southern forest and forest-steppes of Northern Eurasia can be characterized by a continuous gradient of ancestry that remained stable for millennia, ranging from fully West Eurasian in the Baltic region to fully East Asian in the Transbaikal region. In contrast, cotemporaneous groups in far Northeast Siberia were genetically distinct, retaining high levels of continuity from a population that was the primary source of ancestry for Native Americans. By the mid-Holocene, admixture between this early Northeastern Siberian population and groups from Inland East Asia and the Amur River Basin produced two distinctive populations in eastern Siberia that played an important role in the genetic formation of later people. Ancestry from the first population, Cis-Baikal Late Neolithic-Bronze Age (Cisbaikal_LNBA), is found substantially only among Yeniseian-speaking groups and those known to have admixed with them. Ancestry from the second, Yakutian Late Neolithic-Bronze Age (Yakutia_LNBA), is strongly associated with present-day Uralic speakers. We show how Yakutia_LNBA ancestry spread from an east Siberian origin ~4.5kya, along with subclades of Y-chromosome haplogroup N occurring at high frequencies among present-day Uralic speakers, into Western and Central Siberia in communities associated with Seima-Turbino metallurgy: a suite of advanced bronze casting techniques that spread explosively across an enormous region of Northern Eurasia ~4.0kya. However, the ancestry of the 16 Seima-Turbino-period individuals--the first reported from sites with this metallurgy--was otherwise extraordinarily diverse, with partial descent from Indo-Iranian-speaking pastoralists and multiple hunter-gatherer populations from widely separated regions of Eurasia. Our results provide support for theories suggesting that early Uralic speakers at the beginning of their westward dispersal where involved in the expansion of Seima-Turbino metallurgical traditions, and suggests that both cultural transmission and migration were important in the spread of Seima-Turbino material culture.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558301">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558301" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558301">
        <p class="paperTitle">Peripheral Blood Single-Cell Sequencing Uncovers Common and Specific Immune Aberrations in Fibrotic Lung Diseases</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558301" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558301" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, A. Y.; Unterman, A.; Abu Hussein, N.; Sharma, P.; Flint, J.; Yan, X.; Adams, T. S.; Justet, A.; Sumida, T. S.; Zhao, J.; Schupp, J. C.; Raredon, M. S. B.; Ahangari, F. C.; Zhang, Y.; Buendia-Roldan, I.; Adegunsoye, A.; Sperling, A. I.; Prasse, A.; Ryu, C.; Herzog, E.; Selman, M.; Pardo, A.; Kaminski, N.</p>
        <p class="info">Score: 20.7, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558301' target='https://doi.org/10.1101/2023.09.20.558301'> 10.1101/2023.09.20.558301</a></p>
        <p class="abstract">Rationale and ObjectivesThe extent and commonality of peripheral blood immune aberrations in fibrotic interstitial lung diseases are not well characterized. In this study, we aimed to identify common and distinct immune aberrations in patients with idiopathic pulmonary fibrosis (IPF) and fibrotic hypersensitivity pneumonitis (FHP) using cutting-edge single-cell profiling technologies.

MethodsSingle-cell RNA sequencing was performed on patients and healthy controls peripheral blood and bronchoalveolar lavage samples using 10X Genomics 5 gene expression and V(D)J profiling. Cell type composition, transcriptional profiles, cellular trajectories and signaling, and T and B cell receptor repertoires were studied. The standard Seurat R pipeline was followed for cell type composition and differential gene expression analyses. Transcription factor activity was imputed using the DoRothEA-VIPER algorithm. Pseudotime analyses were conducted using Monocle3, while RNA velocity analyses were performed with Velocyto, scVelo, and CellRank. Cell-cell connectomics were assessed using the Connectome R package. V(D)J analyses were conducted using CellRanger and Immcantation frameworks. Across all analyses, disease group differences were assessed using the Wilcoxon rank-sum test.

Measurements and Main Results327,990 cells from 83 samples were profiled. Overall, changes in monocytes were common to IPF and FHP, whereas lymphocytes exhibited disease-specific aberrations. Both diseases displayed enrichment of CCL3hi/CCL4hi CD14&#43; monocytes (p&lt;2.2e-16) and S100Ahi CD14&#43; monocytes (p&lt;2.2e-16) versus controls. Trajectory and RNA velocity analysis suggested that pro-fibrotic macrophages observed in BAL originated from peripheral blood monocytes. Lymphocytes exhibited disease-specific aberrations, with CD8&#43; GZMKhi T cells and activated B cells primarily enriched in FHP patients. V(D)J analyses revealed unique T and B cell receptor complementarity-determining region 3 (CDR3) amino acid compositions (p&lt;0.05) in FHP and significant IgA enrichment in IPF (p&lt;5.2e-7).

ConclusionsWe identified common and disease-specific immune mechanisms in IPF and FHP; S100Ahi monocytes and SPP1hi macrophages are common to IPF and FHP, whereas GMZKhi T lymphocytes and T and B cell receptor repertoires were unique in FHP. Our findings open novel strategies for the diagnosis and treatment of IPF and FHP.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560881">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560881" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560881">
        <p class="paperTitle">Ancestral genetic components are consistently associated with the complex trait landscape in European biobanks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560881" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560881" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pankratov, V.; Mezzavilla, M.; Aneli, S.; Fusco, D.; Wilson, J. F.; Metspalu, M.; Provero, P.; Pagani, L.; Marnetto, D.</p>
        <p class="info">Score: 14.9, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560881' target='https://doi.org/10.1101/2023.10.04.560881'> 10.1101/2023.10.04.560881</a></p>
        <p class="abstract">The genetic structure in Europe was mostly shaped by admixture between the Western Hunter-Gatherer, Anatolian Neolithic and Steppe&#39;s Yamnaya ancestral components. Such structure is regarded as a confounder in GWAS and follow-up studies, and gold-standard methods exist to correct for it. However, it is still poorly understood to which extent these ancestral components contribute to complex trait variation in present-day Europe. In this work we harness the UK Biobank to address this question. By extensive demographic simulations and incorporating previous results obtained using the Estonian Biobank, we carefully evaluate the significance and scope of our findings. Heart rate, platelet count, monocyte percentage and many other traits show stratification similar to height and pigmentation traits, likely targets of selection and divergence across ancestral groups. The consistency of our results across biobanks indicates that these ancestry-specific genetic predispositions act as a source of variability and as potential confounders in Europe as a whole.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.30.560331">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.30.560331" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.30.560331">
        <p class="paperTitle">Improved Resolution of Highly Pathogenic Avian Influenza Virus Haemagglutinin Cleavage Site Using Oxford Nanopore R10 Sequencing Chemistry</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.30.560331" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.30.560331" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ratcliff, J.; Merritt, B.; Gooden, H.; Siegers, J.; Srikanth, A.; Yann, S.; Kol, S.; Sin, S.; Tok, S.; Karlsson, E. A.; Thielen, P. M.</p>
        <p class="info">Score: 14.3, Published: 2023-10-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.30.560331' target='https://doi.org/10.1101/2023.09.30.560331'> 10.1101/2023.09.30.560331</a></p>
        <p class="abstract">Highly pathogenic avian influenza viruses continue to pose global risks to One Health, including agriculture, public, and animal health. Rapid and accurate genomic surveillance is critical for monitoring viral mutations, tracing transmission, and guiding interventions in near real-time. Oxford Nanopore sequencing holds promise for real-time influenza genotyping, but data quality from R9 chemistry has limited its adoption due to challenges resolving low-complexity regions such as the biologically critical hemagglutinin cleavage site, a homopolymer of basic amino acids that distinguish highly pathogenic strains. In this study, human and avian influenza isolates (n=45) from Cambodia were sequenced using both R9.4.1 and R10.4.1 flow cells and chemistries to evaluate performance between approaches. Overall, R10.4.1 yielded increased data output with higher average quality compared to R9.4.1, producing improved consensus sequences using a reference-based bioinformatics approach. R10.4.1 had significantly lower minor population insertion and deletion frequencies, driven by improved performance in low sequence complexity regions prone to insertion and deletion errors, such as homopolymers. Within the hemagglutinin cleavage site, R10.4.1 resolved the correct motif in 90% of genomes compared to only 60% with R9.4.1. Further examination showed reduced frameshift mutations in consensus sequences generated with R10.4.1 that could result in incorrectly classified virulence. Improved consensus genome quality from nanopore sequencing approaches, especially across biologically important low-complexity regions, is critical to reduce subjective hand-curation and will improve local and global genomic surveillance responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559668">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559668" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559668">
        <p class="paperTitle">Deepurify: a multi-modal deep language model to remove contamination from metagenome-assembled genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559668" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559668" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: ZOU, B.; Wang, J.; Ding, Y.; Zhang, Z.; Yufen, H.; Fang, X.; Cheung, K. C.; See, S.; Zhang, L.</p>
        <p class="info">Score: 14.9, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559668' target='https://doi.org/10.1101/2023.09.27.559668'> 10.1101/2023.09.27.559668</a></p>
        <p class="abstract">Metagenome-assembled genomes (MAGs) offer valuable insights into the exploration of microbial dark matter using metagenomic sequencing data. However, there is a growing concern that contamination in MAGs may significantly impact the downstream analysis results. Existing MAG decontamination methods heavily rely on marker genes but do not fully leverage genomic sequences. To address the limitations, we have introduced a novel decontamination approach named Deepurify, which utilizes a multi-modal deep language model employing contrastive learning to learn taxonomic similarities of genomic sequences. Deepurify utilizes inferred taxonomic lineages to guide the allocation of contigs into a MAG-separated tree and employs a tree traversal strategy for maximizing the total number of medium- and high-quality MAGs. Extensive experiments were conducted on two simulated datasets, CAMI I, and human gut metagenomic sequencing data. These results demonstrate that Deepurify significantly outperforms other decontamination methods.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560686">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560686" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560686">
        <p class="paperTitle">Hidden in the sand: phylogenomics unravel an unexpected evolutionary history on the desert-adapted vipers of the genus Cerastes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560686" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560686" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mochales-Riano, G.; Burriel-Carranza, B.; Barros, M. I.; Velo-Anton, G.; Talavera, A.; Spilani, L.; Tejero-Cicuendez, H.; Andre-Crochet, P.; Piris, A.; Garcia-Cardenete, L.; Busais, S.; Els, J.; Shobrak, M.; Carlos-Brito, J.; Smid, J.; Carranza, S.; Martinez-Freiria, F.</p>
        <p class="info">Score: 11.9, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560686' target='https://doi.org/10.1101/2023.10.03.560686'> 10.1101/2023.10.03.560686</a></p>
        <p class="abstract">The desert vipers of the genus Cerastes are a small clade of medically important venomous snakes within the family Viperidae. According to published morphological and molecular studies, the group is comprised by four species: two morphologically similar and phylogenetically sister taxa, the African horned viper (Cerastes cerastes) and the Arabian horned viper (Cerastes gasperettii); a more distantly related species, the Saharan sand viper (Cerastes vipera), and the enigmatic Bohme&#39;s sand viper (Cerastes boehmei), only known from a single specimen in captivity allegedly captured in Central Tunisia. In this study, we analyzed one mitochondrial marker (COI) as well as genome-wide data (ddRAD sequencing) from 28 and 41 samples, respectively, covering the entire distribution range of the genus to explore the population genomics, phylogenomic relationships and introgression patterns within the genus Cerastes. Additionally, and to provide insights into the mode of diversification of the group, we carried out niche overlap analyses considering climatic and habitat variables. Both nuclear phylogenomic reconstructions and population structure analyses have unveiled an unexpected evolutionary history for the genus Cerastes, which sharply contradicts the morphological similarities and previously published mitochondrial approaches. Cerastes cerastes and C. vipera are recovered as sister taxa whilst C. gasperettii is a sister taxon to the clade formed by these two species. We found a relatively high niche overlap (OI &gt; 0.7) in both climatic and habitat variables between C. cerastes and C. vipera, contradicting a potential scenario of sympatric speciation. These results are in line with the introgression found between the northwestern African populations of C. cerastes and C. vipera. Finally, our genomic data confirms the existence of a lineage of C. cerastes in Arabia. All these results highlight the importance of genome-wide data over few genetic markers to study the evolutionary history of species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.22.558704">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.22.558704" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.22.558704">
        <p class="paperTitle">Genomic comparison of the temperate coral Astrangia poculata with tropical corals yields insights into winter quiescence, innate immunity, and sexual reproduction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.22.558704" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.22.558704" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stankiewicz, K. H.; Guiglielmoni, N.; Kitchen, S. A.; Flot, J.-F.; Barott, K. L.; Davies, S. W.; Finnerty, J. R.; Grace, S. P.; Kaufman, L. S.; Putnam, H. M.; Rotjan, R. D.; Sharp, K. H.; Baums, I. B.</p>
        <p class="info">Score: 11.9, Published: 2023-09-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.22.558704' target='https://doi.org/10.1101/2023.09.22.558704'> 10.1101/2023.09.22.558704</a></p>
        <p class="abstract">Facultatively symbiotic corals provide important experimental models to explore the establishment, maintenance, and breakdown of the mutualism between corals and members of the algal family Symbiodiniaceae. The temperate coral Astrangia poculata is one such model as it is not only facultatively symbiotic, but also occurs across a broad temperature and latitudinal gradient. Here, we report the de novo chromosome-scale assembly and annotation of the A. poculata genome. Though widespread segmental/tandem duplications of genomic regions were detected, we did not find strong evidence of a whole genome duplication (WGD) event. Comparison of the gene arrangement between A. poculata and the tropical coral Acropora millepora revealed 56.38% of the orthologous genes were conserved in syntenic blocks despite [~]415 million years of divergence. Gene families related to sperm hyperactivation and innate immunity, including lectins, were found to contain more genes in A. millepora relative to A. poculata. Sperm hyperactivation in A. millepora is expected given the extreme requirements of gamete competition during mass spawning events in tropical corals, while lectins are important in the establishment of coral-algal symbiosis. By contrast, gene families involved in sleep promotion, feeding suppression, and circadian sleep/wake cycle processes were expanded in A. poculata. These expanded gene families may play a role in A. poculatas ability to enter a dormancy-like state (&#34;winter quiescence&#34;) to survive freezing temperatures at the northern edges of the species range.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.559817">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.559817" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.559817">
        <p class="paperTitle">Integrated characterization of cell types, states and molecular programs in disseminated appendiceal neoplasms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.559817" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.559817" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bui, L. T.; Cao, X.; Wang, J. T.; Meng, F.; Feng, M.; Arvanitis, L.; Mannan, R.; Woo, Y.; Idrees, K.; Banovich, N.; Raoof, M.</p>
        <p class="info">Score: 10.7, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.559817' target='https://doi.org/10.1101/2023.09.28.559817'> 10.1101/2023.09.28.559817</a></p>
        <p class="abstract">Appendiceal neoplasms include a heterogeneous group of epithelial and non-epithelial tumors with varying malignant potential. Despite the rise in incidence of appendiceal neoplasms in recent years, limited progress has been made in the understanding, management and therapeutic treatment. To comprehensively characterize the cell types and molecular mechanisms driving cellular remodeling in epithelial appendiceal neoplasms, we performed an integrated scRNA-seq study. We analyzed 126,998 cells from 16 appendix samples (11 peritoneal metastases samples, 5 healthy controls) and identified 33 distinct cell types/cell states with seven being cancer-specific. Highlights of our study include the characterization of tumor cells across the histologic spectrum, the identification of a novel cancer-associated-fibroblast (CAF) subtypes (fiCAFs) and the identification of pathologic-specific cellular crosstalk between tumor cells and the tumor microenvironment (TME). Together, our study provides a high-resolution insight into the complexity and heterogeneity of epithelial appendiceal neoplasms and a valuable resource for therapeutic strategies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
