<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Systematic identification of cargo-carrying genetic elements reveals new dimensions of eukaryotic diversity
Authors: Gluck-Thaler, E.; Vogan, A. A.
Score: 33.4, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563810
Cargo-carrying mobile elements (CCEs) are genetic entities that transpose diverse protein coding sequences. Although common in bacteria, we know little about the biology of eukaryotic CCEs because no appropriate tools exist for their annotation. For example, Starships are fungal CCEs whose functions are largely unknown because they require time-intensive manual curation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Systematic identification of cargo-carrying genetic elements reveals new dimensions of eukaryotic diversity
Authors: Gluck-Thaler, E.; Vogan, A. A.
Score: 33.4, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563810
Cargo-carrying mobile elements (CCEs) are genetic entities that transpose diverse protein coding sequences. Although common in bacteria, we know little about the biology of eukaryotic CCEs because no appropriate tools exist for their annotation. For example, Starships are fungal CCEs whose functions are largely unknown because they require time-intensive manual curation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-01T10:38:40+00:00" />
<meta property="article:modified_time" content="2023-11-01T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Systematic identification of cargo-carrying genetic elements reveals new dimensions of eukaryotic diversity
Authors: Gluck-Thaler, E.; Vogan, A. A.
Score: 33.4, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563810
Cargo-carrying mobile elements (CCEs) are genetic entities that transpose diverse protein coding sequences. Although common in bacteria, we know little about the biology of eukaryotic CCEs because no appropriate tools exist for their annotation. For example, Starships are fungal CCEs whose functions are largely unknown because they require time-intensive manual curation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Systematic identification of cargo-carrying genetic elements reveals new dimensions of eukaryotic diversity\nAuthors: Gluck-Thaler, E.; Vogan, A. A.\nScore: 33.4, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563810\nCargo-carrying mobile elements (CCEs) are genetic entities that transpose diverse protein coding sequences. Although common in bacteria, we know little about the biology of eukaryotic CCEs because no appropriate tools exist for their annotation. For example, Starships are fungal CCEs whose functions are largely unknown because they require time-intensive manual curation.",
  "keywords": [
    
  ],
  "articleBody": " Systematic identification of cargo-carrying genetic elements reveals new dimensions of eukaryotic diversity\nAuthors: Gluck-Thaler, E.; Vogan, A. A.\nScore: 33.4, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563810\nCargo-carrying mobile elements (CCEs) are genetic entities that transpose diverse protein coding sequences. Although common in bacteria, we know little about the biology of eukaryotic CCEs because no appropriate tools exist for their annotation. For example, Starships are fungal CCEs whose functions are largely unknown because they require time-intensive manual curation. To address this knowledge gap, we developed starfish, a computational workflow for high-throughput eukaryotic CCE annotation. We applied starfish to 2, 899 genomes of 1, 649 fungal species and found that starfish recovers known Starships with \u003e95% precision and accuracy while expanding the number of annotated elements ten-fold. Extant Starship diversity is partitioned into 11 families that differ in their enrichment patterns across fungal classes. Starship cargo changes rapidly such that elements from the same family differ substantially in their functional repertoires, which are predicted to contribute to diverse biological processes such as metabolism. Many elements have convergently evolved to insert into 5S rDNA and AT-rich sequence while others integrate into random locations, revealing both specialist and generalist strategies for persistence. Our work establishes a framework for advancing mobile element biology and provides the means to investigate an emerging dimension of eukaryotic genetic diversity, that of genomes within genomes.\nModular vector assembly enables rapid assessment of emerging CRISPR technologies\nAuthors: McGee, A. V.; Liu, Y. V.; Griffith, A. L.; Szegletes, Z. M.; Wen, B.; Kraus, C.; Miller, N. W.; Steger, R. J.; Escude Velasco, B.; Bosch, J. A.; Zirin, J. D.; Viswanatha, R.; Sontheimer, E. J.; Goodale, A.; Greene, M. A.; Green, T. M.; Doench, J. G.\nScore: 30.1, Published: 2023-10-27 DOI: 10.1101/2023.10.25.564061\nThe diversity of CRISPR systems, coupled with scientific ingenuity, has led to an explosion of applications; however, to test newly-described innovations in their model systems, researchers typically embark on cumbersome, one-off cloning projects to generate custom reagents that are optimized for their biological questions. Here, we leverage Golden Gate cloning to create the Fragmid toolkit, a modular set of CRISPR cassettes and delivery technologies, along with a web portal, resulting in a combinatorial platform that enables scalable vector assembly within days. We further demonstrate that multiple CRISPR technologies can be assessed in parallel in a pooled screening format using this resource, enabling the rapid optimization of both novel technologies and cellular models. These results establish Fragmid as a robust system for the rapid design of CRISPR vectors, and we anticipate that this assembly approach will be broadly useful for systematic development, comparison, and dissemination of CRISPR technologies.\nThe history of chromosomal instability in genome doubled tumors\nAuthors: Baker, T. M.; Lai, S.; Lesluyes, T.; Yan, H.; Verfaillie, A.; Dentro, S.; Lynch, A. R.; Bowes, A. L.; Pillay, N.; Flanagan, A. M.; Swanton, C.; Tarabichi, M.; Van Loo, P.\nScore: 20.5, Published: 2023-10-25 DOI: 10.1101/2023.10.22.563273\nTumors frequently display high chromosomal instability (CIN) and contain multiple copies of genomic regions. Here, we describe GRITIC, a generic method for timing genomic gains leading to complex copy number states, using single-sample bulk whole-genome sequencing data. By applying GRITIC to 5,656 tumors, we found that non-parsimonious evolution is frequent in the formation of complex copy number states in genome-duplicated tumors. We measured CIN before and after genome duplication in human tumors and found that late genome doubling was followed by an increase in the rate of copy number gain. Copy number gains often accumulate as punctuated bursts, commonly after genome duplication. We infer that genome duplications typically affect the selection landscape of copy number losses, while only minimally impacting copy number gains. In summary, GRITIC is a novel copy number gain timing framework that permits the analysis of copy number evolution in chromosomally unstable tumors. Statement of significanceComplex genomic gains are associated with whole-genome duplications, which are frequent across tumors, span a large fraction of their genomes, and are linked to poorer outcomes. GRITIC infers when these gains occur during tumor development, which will help to identify the genetic events that drive tumor evolution.\nJoint sequence \u0026 chromatin neural networks characterize the differential abilities of Forkhead transcription factors to engage inaccessible chromatin\nAuthors: Arora, S.; Yang, J.; Akiyama, T.; James, D. Q.; Morrissey, A.; Blanda, T.; Badjatia, N.; Lai, W. K. M.; Ko, M. S. H.; Pugh, B. F.; Mahony, S.\nScore: 18.2, Published: 2023-10-31 DOI: 10.1101/2023.10.06.561228\nThe DNA-binding activities of transcription factors (TFs) are influenced by both intrinsic sequence preferences and extrinsic interactions with cell-specific chromatin landscapes and other regulatory proteins. Disentangling the roles of these binding determinants remains challenging. For example, the FoxA subfamily of Forkhead domain (Fox) TFs are known pioneer factors that can bind to relatively inaccessible sites during development. Yet FoxA TF binding also varies across cell types, pointing to a combination of intrinsic and extrinsic forces guiding their binding. While other Forkhead domain TFs are often assumed to have pioneering abilities, how sequence and chromatin features influence the binding of related Fox TFs has not been systematically characterized. Here, we present a principled approach to compare the relative contributions of intrinsic DNA sequence preference and cell-specific chromatin environments to a TF's DNA-binding activities. We apply our approach to investigate how a selection of Fox TFs (FoxA1, FoxC1, FoxG1, FoxL2, and FoxP3) vary in their binding specificity. We over-express the selected Fox TFs in mouse embryonic stem cells, which offer a platform to contrast each TF's binding activity within the same preexisting chromatin background. By applying a convolutional neural network to interpret the Fox TF binding patterns, we evaluate how sequence and preexisting chromatin features jointly contribute to induced TF binding. We demonstrate that Fox TFs bind different DNA targets, and drive differential gene expression patterns, even when induced in identical chromatin settings. Despite the association between Forkhead domains and pioneering activities, the selected Fox TFs display a wide range of affinities for preexiting chromatin states. Using sequence and chromatin feature attribution techniques to interpret the neural network predictions, we show that differential sequence preferences combined with differential abilities to engage relatively inaccessible chromatin together explain Fox TF binding patterns at individual sites and genome-wide.\nAdipose tissue eQTL meta-analysis reveals the contribution of allelic heterogeneity to gene expression regulation and cardiometabolic traits\nAuthors: Brotman, S. M.; El-Sayed Moustafa, J. S.; Guan, L.; Broadaway, K. A.; Wang, D.; Jackson, A. U.; Welch, R.; Currin, K. W.; Tomlinson, M.; Vadlamudi, S.; Stringham, H. M.; Roberts, A. L.; Lakka, T. A.; Oravilahti, A.; Fernandes Silva, L.; Narisu, N.; Erdos, M. R.; Yan, T.; Bonnycastle, L. L.; Raulerson, C. K.; Raza, Y.; Yan, X.; Parker, S. C. J.; Kuusisto, J.; Pajukanta, P.; Tuomilehto, J.; Collins, F. S.; Boehnke, M.; Love, M. I.; Koistinen, H. A.; Laakso, M.; Mohlke, K. L.; Small, K. S.; Scott, L. J.\nScore: 16.1, Published: 2023-10-27 DOI: 10.1101/2023.10.26.563798\nComplete characterization of the genetic effects on gene expression is needed to elucidate tissue biology and the etiology of complex traits. Here, we analyzed 2,344 subcutaneous adipose tissue samples and identified 34K conditionally distinct expression quantitative trait locus (eQTL) signals in 18K genes. Over half of eQTL genes exhibited at least two eQTL signals. Compared to primary signals, non-primary signals had lower effect sizes, lower minor allele frequencies, and less promoter enrichment; they corresponded to genes with higher heritability and higher tolerance for loss of function. Colocalization of eQTL with conditionally distinct genome-wide association study signals for 28 cardiometabolic traits identified 3,605 eQTL signals for 1,861 genes. Inclusion of non-primary eQTL signals increased colocalized signals by 46%. Among 30 genes with [\u0026ge;]2 pairs of colocalized signals, 21 showed a mediating gene dosage effect on the trait. Thus, expanded eQTL identification reveals more mechanisms underlying complex traits and improves understanding of the complexity of gene expression regulation.\nHigh-throughput single-nucleus hybrid sequencing reveals genome-transcriptome correlations in cancer\nAuthors: Li, S.; Alexander, J.; Kendall, J.; Andrews, P.; Rose, E.; Orjuela, H.; Park, S.; Podszus, C.; Shanley, L.; Ma, R.; Ranade, N.; Ronemus, M.; Rishi, A.; Donoho, D. L.; Goldberg, G. L.; Levy, D.; Wigler, M.\nScore: 18.0, Published: 2023-10-06 DOI: 10.1101/2023.10.04.560973\nSingle-cell genomic analyses can provide information on cellular mutation and tumor heterogeneity, whereas single-cell transcriptomic analyses can distinguish cell types and states. However, the disconnect between genomic and transcriptomic spaces limits our understanding of cancer development. To address this, we developed a novel high-throughput method that simultaneously captures both DNA and RNA from single nuclei and new algorithms for the quantitative clustering and filtering of single-cell data. We applied this hybrid protocol to 65,499 single nuclei extracted from frozen biopsies of five different endometrial cancer patients and separately clustered the genome and expression data. We also analyzed 34,651 and 21,432 nuclei using RNA-only and DNA-only protocols, respectively, from the same samples to verify the clustering. Multiple tumor genome and/or expression clusters were often present within an individual patient, and different tumor clones could project into distinct or shared expression states. Almost all possible genome-transcriptome correlations were observed in the cohort. Stromal clusters were largely shared between patients, but some patients possessed unique stromal components, or mutant stroma with a significant loss of the X chromosome. This study reveals the complex landscape involving genome and transcriptome interactions at single-cell level, and provides new insights into mutant stroma as a potential clinical biomarker.\nHigh-resolution CTCF footprinting reveals impact of chromatin state on cohesin extrusion dynamics\nAuthors: Sept, C. E.; Tak, Y. E.; Cerda-Smith, C. G.; Hutchinson, H. M.; Goel, V.; Blanchette, M.; Bhakta, M. S.; Hansen, A. S.; Joung, J. K.; Johnstone, S.; Eyler, C. E.; Aryee, M. J.\nScore: 18.6, Published: 2023-10-28 DOI: 10.1101/2023.10.20.563340\nDNA looping is vital for establishing many enhancer-promoter interactions. While CTCF is known to anchor many cohesin-mediated loops, the looped chromatin fiber appears to predominantly exist in a poorly characterized actively extruding state. To better characterize extruding chromatin loop structures, we used CTCF MNase HiChIP data to determine both CTCF binding at high resolution and 3D contact information. Here we present FactorFinder, a tool that identifies CTCF binding sites at near base-pair resolution. We leverage this substantial advance in resolution to determine that the fully extruded (CTCF-CTCF) state is rare genome-wide with locus-specific variation from [~]1-10%. We further investigate the impact of chromatin state on loop extrusion dynamics, and find that active enhancers and RNA Pol II impede cohesin extrusion, facilitating an enrichment of enhancer-promoter contacts in the partially extruded loop state. We propose a model of topological regulation whereby the transient, partially extruded states play active roles in transcription.\nReplication stress underlies genomic instability at CTCF/cohesin-binding sites in cancer\nAuthors: Faseela, E. E.; Notani, D.; Sabarinathan, R.\nScore: 13.3, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563697\nCCCTC-binding factor (CTCF) and cohesin play a significant role in the formation of chromatin loops and topologically associating domains (TADs), which influence gene expression and DNA replication. CTCF/cohesin-binding sites (CBSs) present at the loop anchors and TAD boundaries are frequently mutated in cancer; however, the molecular mechanisms underlying this remain unclear. Here, we investigate whether the binding of CTCF/cohesin on DNA imposes constraints on DNA replication, leading to replication stress and genomic instability. Our results reveal that CTCF and cohesin remain bound to DNA during replication (S phase) in cancer cells (HeLa). Further, examination of replication stress through imaging revealed colocalization of DNA damage response proteins (MRE11 and STN1) with CTCF/cohesin in the S phase. In addition, ChIP-seq of the DNA damage response/repair proteins (MRE11, STN1, {gamma}H2AX and RAD51) showed high enrichment of these proteins at CBSs (as compared to their immediate flanking regions and control sites) and positively correlated with the binding strength of CTCF/cohesin at CBSs in the S phase. Moreover, analysis of somatic mutations from cancer genomes supports that the enrichment of mutations at CBSs is significantly higher in samples harbouring somatic copy number deletion in MRE11 and STN1 compared to wild-type samples. Together, these results demonstrate that the binding of CTCF/cohesin on the DNA during the S phase causes replication stress and DNA strand breaks, and this could lead to genome instability at CBSs observed in cancer.\nA public resource of single cell transcriptomes and multiscale networks from persons with and without Alzheimer's disease\nAuthors: Wang, Q.; Antone, J.; Alsop, E.; Reiman, R.; Funk, C.; Bendl, J.; Dudley, J. T.; Liang, W. S.; Karr, T. L.; Roussos, P.; Bennett, D. A.; De Jager, P. L.; Serrano, G. E.; Beach, T. G.; Van Keuren-Jensen, K.; Mastroeni, D.; Reiman, E. M.; Readhead, B. P.\nScore: 14.6, Published: 2023-10-24 DOI: 10.1101/2023.10.20.563319\nThe emergence of technologies that can support high-throughput profiling of single cell transcriptomes offers to revolutionize the study of brain tissue from persons with and without Alzheimers disease (AD). Integration of these data with additional complementary multiomics data such as genetics, proteomics and clinical data provides powerful opportunities to link observed cell subpopulations and molecular network features within a broader disease-relevant context. We report here single nucleus RNA sequencing (snRNA-seq) profiles generated from superior frontal gyrus cortical tissue samples from 101 exceptionally well characterized, aged subjects from the Banner Brain and Body Donation Program in combination with whole genome sequences. We report findings that link common AD risk variants with CR1 expression in oligodendrocytes as well as alterations in peripheral hematological lab parameters, with these observations replicated in an independent, prospective cohort study of ageing and dementia. We also observed an AD-associated CD83(+) microglial subtype with unique molecular networks that encompass many known regulators of AD-relevant microglial biology, and which are associated with immunoglobulin IgG4 production in the transverse colon. These findings illustrate the power of multi-tissue molecular profiling to contextualize snRNA-seq brain transcriptomics and reveal novel disease biology. The transcriptomic, genetic, phenotypic, and network data resources described within this study are available for access and utilization by the scientific community.\nHigh-fidelity (repeat) consensus sequences from short reads using combined read clustering and assembly\nAuthors: Mann, L.; Balasch, K.; Schmidt, N.; Heitkam, T.\nScore: 10.5, Published: 2023-10-30 DOI: 10.1101/2023.10.26.564123\nBackground: Despite the many cheap and fast ways to generate genomic data, good and exact genome assembly is still a problem, with especially the repeats being vastly underrepresented and often misassembled. As short reads in low coverage are already sufficient to represent the repeat landscape of any given genome, many read cluster algorithms were brought forward that provide repeat identification and classification. But how can trustworthy, reliable and representative full-length repeat consensuses be derived from unassembled genomes? Results: Here, we combine methods from repeat identification and genome assembly to derive these robust consensuses. We test several use cases, such as (1) consensus building from clustered short reads of non-model genomes, (2) from genome-wide amplification setups, and (3) specific repeat-centred questions, such as the linked vs. unlinked arrangement of ribosomal genes. In all our use-cases, the derived consensuses are robust and representative. To evaluate overall performance, we compare our high-fidelity repeat consensuses to RepeatExplorer2-derived contigs and check, if they represent real transposable elements as found in long reads. Our results demonstrate that it is possible to generate useful, reliable and trustworthy consensuses from short reads by a combination from read cluster and genome assembly methods in an automatable way. Conclusion: We anticipate that our workflow opens the way towards more efficient and less manual repeat characterization and annotation, benefitting all genome studies, but especially those of non-model organisms.\n",
  "wordCount" : "2539",
  "inLanguage": "en",
  "datePublished": "2023-11-01T10:38:40Z",
  "dateModified": "2023-11-01T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 1, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563810">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563810" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563810">
        <p class="paperTitle">Systematic identification of cargo-carrying genetic elements reveals new dimensions of eukaryotic diversity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563810" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563810" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gluck-Thaler, E.; Vogan, A. A.</p>
        <p class="info">Score: 33.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563810' target='https://doi.org/10.1101/2023.10.24.563810'> 10.1101/2023.10.24.563810</a></p>
        <p class="abstract">Cargo-carrying mobile elements (CCEs) are genetic entities that transpose diverse protein coding sequences. Although common in bacteria, we know little about the biology of eukaryotic CCEs because no appropriate tools exist for their annotation. For example, Starships are fungal CCEs whose functions are largely unknown because they require time-intensive manual curation. To address this knowledge gap, we developed starfish, a computational workflow for high-throughput eukaryotic CCE annotation. We applied starfish to 2, 899 genomes of 1, 649 fungal species and found that starfish recovers known Starships with &gt;95% precision and accuracy while expanding the number of annotated elements ten-fold. Extant Starship diversity is partitioned into 11 families that differ in their enrichment patterns across fungal classes. Starship cargo changes rapidly such that elements from the same family differ substantially in their functional repertoires, which are predicted to contribute to diverse biological processes such as metabolism. Many elements have convergently evolved to insert into 5S rDNA and AT-rich sequence while others integrate into random locations, revealing both specialist and generalist strategies for persistence. Our work establishes a framework for advancing mobile element biology and provides the means to investigate an emerging dimension of eukaryotic genetic diversity, that of genomes within genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564061">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564061" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564061">
        <p class="paperTitle">Modular vector assembly enables rapid assessment of emerging CRISPR technologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564061" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564061" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McGee, A. V.; Liu, Y. V.; Griffith, A. L.; Szegletes, Z. M.; Wen, B.; Kraus, C.; Miller, N. W.; Steger, R. J.; Escude Velasco, B.; Bosch, J. A.; Zirin, J. D.; Viswanatha, R.; Sontheimer, E. J.; Goodale, A.; Greene, M. A.; Green, T. M.; Doench, J. G.</p>
        <p class="info">Score: 30.1, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564061' target='https://doi.org/10.1101/2023.10.25.564061'> 10.1101/2023.10.25.564061</a></p>
        <p class="abstract">The diversity of CRISPR systems, coupled with scientific ingenuity, has led to an explosion of applications; however, to test newly-described innovations in their model systems, researchers typically embark on cumbersome, one-off cloning projects to generate custom reagents that are optimized for their biological questions. Here, we leverage Golden Gate cloning to create the Fragmid toolkit, a modular set of CRISPR cassettes and delivery technologies, along with a web portal, resulting in a combinatorial platform that enables scalable vector assembly within days. We further demonstrate that multiple CRISPR technologies can be assessed in parallel in a pooled screening format using this resource, enabling the rapid optimization of both novel technologies and cellular models. These results establish Fragmid as a robust system for the rapid design of CRISPR vectors, and we anticipate that this assembly approach will be broadly useful for systematic development, comparison, and dissemination of CRISPR technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.22.563273">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.22.563273" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.22.563273">
        <p class="paperTitle">The history of chromosomal instability in genome doubled tumors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.22.563273" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.22.563273" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baker, T. M.; Lai, S.; Lesluyes, T.; Yan, H.; Verfaillie, A.; Dentro, S.; Lynch, A. R.; Bowes, A. L.; Pillay, N.; Flanagan, A. M.; Swanton, C.; Tarabichi, M.; Van Loo, P.</p>
        <p class="info">Score: 20.5, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.22.563273' target='https://doi.org/10.1101/2023.10.22.563273'> 10.1101/2023.10.22.563273</a></p>
        <p class="abstract">Tumors frequently display high chromosomal instability (CIN) and contain multiple copies of genomic regions. Here, we describe GRITIC, a generic method for timing genomic gains leading to complex copy number states, using single-sample bulk whole-genome sequencing data. By applying GRITIC to 5,656 tumors, we found that non-parsimonious evolution is frequent in the formation of complex copy number states in genome-duplicated tumors. We measured CIN before and after genome duplication in human tumors and found that late genome doubling was followed by an increase in the rate of copy number gain. Copy number gains often accumulate as punctuated bursts, commonly after genome duplication. We infer that genome duplications typically affect the selection landscape of copy number losses, while only minimally impacting copy number gains. In summary, GRITIC is a novel copy number gain timing framework that permits the analysis of copy number evolution in chromosomally unstable tumors.

Statement of significanceComplex genomic gains are associated with whole-genome duplications, which are frequent across tumors, span a large fraction of their genomes, and are linked to poorer outcomes. GRITIC infers when these gains occur during tumor development, which will help to identify the genetic events that drive tumor evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561228">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561228" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561228">
        <p class="paperTitle">Joint sequence &amp; chromatin neural networks characterize the differential abilities of Forkhead transcription factors to engage inaccessible chromatin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561228" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561228" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arora, S.; Yang, J.; Akiyama, T.; James, D. Q.; Morrissey, A.; Blanda, T.; Badjatia, N.; Lai, W. K. M.; Ko, M. S. H.; Pugh, B. F.; Mahony, S.</p>
        <p class="info">Score: 18.2, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561228' target='https://doi.org/10.1101/2023.10.06.561228'> 10.1101/2023.10.06.561228</a></p>
        <p class="abstract">The DNA-binding activities of transcription factors (TFs) are influenced by both intrinsic sequence preferences and extrinsic interactions with cell-specific chromatin landscapes and other regulatory proteins. Disentangling the roles of these binding determinants remains challenging. For example, the FoxA subfamily of Forkhead domain (Fox) TFs are known pioneer factors that can bind to relatively inaccessible sites during development. Yet FoxA TF binding also varies across cell types, pointing to a combination of intrinsic and extrinsic forces guiding their binding. While other Forkhead domain TFs are often assumed to have pioneering abilities, how sequence and chromatin features influence the binding of related Fox TFs has not been systematically characterized. Here, we present a principled approach to compare the relative contributions of intrinsic DNA sequence preference and cell-specific chromatin environments to a TF&#39;s DNA-binding activities. We apply our approach to investigate how a selection of Fox TFs (FoxA1, FoxC1, FoxG1, FoxL2, and FoxP3) vary in their binding specificity. We over-express the selected Fox TFs in mouse embryonic stem cells, which offer a platform to contrast each TF&#39;s binding activity within the same preexisting chromatin background. By applying a convolutional neural network to interpret the Fox TF binding patterns, we evaluate how sequence and preexisting chromatin features jointly contribute to induced TF binding. We demonstrate that Fox TFs bind different DNA targets, and drive differential gene expression patterns, even when induced in identical chromatin settings. Despite the association between Forkhead domains and pioneering activities, the selected Fox TFs display a wide range of affinities for preexiting chromatin states. Using sequence and chromatin feature attribution techniques to interpret the neural network predictions, we show that differential sequence preferences combined with differential abilities to engage relatively inaccessible chromatin together explain Fox TF binding patterns at individual sites and genome-wide.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.563798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.563798" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.563798">
        <p class="paperTitle">Adipose tissue eQTL meta-analysis reveals the contribution of allelic heterogeneity to gene expression regulation and cardiometabolic traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.563798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.563798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brotman, S. M.; El-Sayed Moustafa, J. S.; Guan, L.; Broadaway, K. A.; Wang, D.; Jackson, A. U.; Welch, R.; Currin, K. W.; Tomlinson, M.; Vadlamudi, S.; Stringham, H. M.; Roberts, A. L.; Lakka, T. A.; Oravilahti, A.; Fernandes Silva, L.; Narisu, N.; Erdos, M. R.; Yan, T.; Bonnycastle, L. L.; Raulerson, C. K.; Raza, Y.; Yan, X.; Parker, S. C. J.; Kuusisto, J.; Pajukanta, P.; Tuomilehto, J.; Collins, F. S.; Boehnke, M.; Love, M. I.; Koistinen, H. A.; Laakso, M.; Mohlke, K. L.; Small, K. S.; Scott, L. J.</p>
        <p class="info">Score: 16.1, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.563798' target='https://doi.org/10.1101/2023.10.26.563798'> 10.1101/2023.10.26.563798</a></p>
        <p class="abstract">Complete characterization of the genetic effects on gene expression is needed to elucidate tissue biology and the etiology of complex traits. Here, we analyzed 2,344 subcutaneous adipose tissue samples and identified 34K conditionally distinct expression quantitative trait locus (eQTL) signals in 18K genes. Over half of eQTL genes exhibited at least two eQTL signals. Compared to primary signals, non-primary signals had lower effect sizes, lower minor allele frequencies, and less promoter enrichment; they corresponded to genes with higher heritability and higher tolerance for loss of function. Colocalization of eQTL with conditionally distinct genome-wide association study signals for 28 cardiometabolic traits identified 3,605 eQTL signals for 1,861 genes. Inclusion of non-primary eQTL signals increased colocalized signals by 46%. Among 30 genes with [&amp;ge;]2 pairs of colocalized signals, 21 showed a mediating gene dosage effect on the trait. Thus, expanded eQTL identification reveals more mechanisms underlying complex traits and improves understanding of the complexity of gene expression regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560973">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560973" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560973">
        <p class="paperTitle">High-throughput single-nucleus hybrid sequencing reveals genome-transcriptome correlations in cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560973" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560973" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, S.; Alexander, J.; Kendall, J.; Andrews, P.; Rose, E.; Orjuela, H.; Park, S.; Podszus, C.; Shanley, L.; Ma, R.; Ranade, N.; Ronemus, M.; Rishi, A.; Donoho, D. L.; Goldberg, G. L.; Levy, D.; Wigler, M.</p>
        <p class="info">Score: 18.0, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560973' target='https://doi.org/10.1101/2023.10.04.560973'> 10.1101/2023.10.04.560973</a></p>
        <p class="abstract">Single-cell genomic analyses can provide information on cellular mutation and tumor heterogeneity, whereas single-cell transcriptomic analyses can distinguish cell types and states. However, the disconnect between genomic and transcriptomic spaces limits our understanding of cancer development. To address this, we developed a novel high-throughput method that simultaneously captures both DNA and RNA from single nuclei and new algorithms for the quantitative clustering and filtering of single-cell data. We applied this hybrid protocol to 65,499 single nuclei extracted from frozen biopsies of five different endometrial cancer patients and separately clustered the genome and expression data. We also analyzed 34,651 and 21,432 nuclei using RNA-only and DNA-only protocols, respectively, from the same samples to verify the clustering. Multiple tumor genome and/or expression clusters were often present within an individual patient, and different tumor clones could project into distinct or shared expression states. Almost all possible genome-transcriptome correlations were observed in the cohort. Stromal clusters were largely shared between patients, but some patients possessed unique stromal components, or mutant stroma with a significant loss of the X chromosome. This study reveals the complex landscape involving genome and transcriptome interactions at single-cell level, and provides new insights into mutant stroma as a potential clinical biomarker.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563340">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563340" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563340">
        <p class="paperTitle">High-resolution CTCF footprinting reveals impact of chromatin state on cohesin extrusion dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563340" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563340" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sept, C. E.; Tak, Y. E.; Cerda-Smith, C. G.; Hutchinson, H. M.; Goel, V.; Blanchette, M.; Bhakta, M. S.; Hansen, A. S.; Joung, J. K.; Johnstone, S.; Eyler, C. E.; Aryee, M. J.</p>
        <p class="info">Score: 18.6, Published: 2023-10-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563340' target='https://doi.org/10.1101/2023.10.20.563340'> 10.1101/2023.10.20.563340</a></p>
        <p class="abstract">DNA looping is vital for establishing many enhancer-promoter interactions. While CTCF is known to anchor many cohesin-mediated loops, the looped chromatin fiber appears to predominantly exist in a poorly characterized actively extruding state. To better characterize extruding chromatin loop structures, we used CTCF MNase HiChIP data to determine both CTCF binding at high resolution and 3D contact information. Here we present FactorFinder, a tool that identifies CTCF binding sites at near base-pair resolution. We leverage this substantial advance in resolution to determine that the fully extruded (CTCF-CTCF) state is rare genome-wide with locus-specific variation from [~]1-10%. We further investigate the impact of chromatin state on loop extrusion dynamics, and find that active enhancers and RNA Pol II impede cohesin extrusion, facilitating an enrichment of enhancer-promoter contacts in the partially extruded loop state. We propose a model of topological regulation whereby the transient, partially extruded states play active roles in transcription.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563697">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563697" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563697">
        <p class="paperTitle">Replication stress underlies genomic instability at CTCF/cohesin-binding sites in cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563697" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563697" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faseela, E. E.; Notani, D.; Sabarinathan, R.</p>
        <p class="info">Score: 13.3, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563697' target='https://doi.org/10.1101/2023.10.24.563697'> 10.1101/2023.10.24.563697</a></p>
        <p class="abstract">CCCTC-binding factor (CTCF) and cohesin play a significant role in the formation of chromatin loops and topologically associating domains (TADs), which influence gene expression and DNA replication. CTCF/cohesin-binding sites (CBSs) present at the loop anchors and TAD boundaries are frequently mutated in cancer; however, the molecular mechanisms underlying this remain unclear. Here, we investigate whether the binding of CTCF/cohesin on DNA imposes constraints on DNA replication, leading to replication stress and genomic instability. Our results reveal that CTCF and cohesin remain bound to DNA during replication (S phase) in cancer cells (HeLa). Further, examination of replication stress through imaging revealed colocalization of DNA damage response proteins (MRE11 and STN1) with CTCF/cohesin in the S phase. In addition, ChIP-seq of the DNA damage response/repair proteins (MRE11, STN1, {gamma}H2AX and RAD51) showed high enrichment of these proteins at CBSs (as compared to their immediate flanking regions and control sites) and positively correlated with the binding strength of CTCF/cohesin at CBSs in the S phase. Moreover, analysis of somatic mutations from cancer genomes supports that the enrichment of mutations at CBSs is significantly higher in samples harbouring somatic copy number deletion in MRE11 and STN1 compared to wild-type samples. Together, these results demonstrate that the binding of CTCF/cohesin on the DNA during the S phase causes replication stress and DNA strand breaks, and this could lead to genome instability at CBSs observed in cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563319">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563319" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563319">
        <p class="paperTitle">A public resource of single cell transcriptomes and multiscale networks from persons with and without Alzheimer&#39;s disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563319" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563319" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, Q.; Antone, J.; Alsop, E.; Reiman, R.; Funk, C.; Bendl, J.; Dudley, J. T.; Liang, W. S.; Karr, T. L.; Roussos, P.; Bennett, D. A.; De Jager, P. L.; Serrano, G. E.; Beach, T. G.; Van Keuren-Jensen, K.; Mastroeni, D.; Reiman, E. M.; Readhead, B. P.</p>
        <p class="info">Score: 14.6, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563319' target='https://doi.org/10.1101/2023.10.20.563319'> 10.1101/2023.10.20.563319</a></p>
        <p class="abstract">The emergence of technologies that can support high-throughput profiling of single cell transcriptomes offers to revolutionize the study of brain tissue from persons with and without Alzheimers disease (AD). Integration of these data with additional complementary multiomics data such as genetics, proteomics and clinical data provides powerful opportunities to link observed cell subpopulations and molecular network features within a broader disease-relevant context. We report here single nucleus RNA sequencing (snRNA-seq) profiles generated from superior frontal gyrus cortical tissue samples from 101 exceptionally well characterized, aged subjects from the Banner Brain and Body Donation Program in combination with whole genome sequences. We report findings that link common AD risk variants with CR1 expression in oligodendrocytes as well as alterations in peripheral hematological lab parameters, with these observations replicated in an independent, prospective cohort study of ageing and dementia. We also observed an AD-associated CD83(&#43;) microglial subtype with unique molecular networks that encompass many known regulators of AD-relevant microglial biology, and which are associated with immunoglobulin IgG4 production in the transverse colon. These findings illustrate the power of multi-tissue molecular profiling to contextualize snRNA-seq brain transcriptomics and reveal novel disease biology. The transcriptomic, genetic, phenotypic, and network data resources described within this study are available for access and utilization by the scientific community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564123" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564123">
        <p class="paperTitle">High-fidelity (repeat) consensus sequences from short reads using combined read clustering and assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mann, L.; Balasch, K.; Schmidt, N.; Heitkam, T.</p>
        <p class="info">Score: 10.5, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564123' target='https://doi.org/10.1101/2023.10.26.564123'> 10.1101/2023.10.26.564123</a></p>
        <p class="abstract">Background: Despite the many cheap and fast ways to generate genomic data, good and exact genome assembly is still a problem, with especially the repeats being vastly underrepresented and often misassembled. As short reads in low coverage are already sufficient to represent the repeat landscape of any given genome, many read cluster algorithms were brought forward that provide repeat identification and classification. But how can trustworthy, reliable and representative full-length repeat consensuses be derived from unassembled genomes? Results: Here, we combine methods from repeat identification and genome assembly to derive these robust consensuses. We test several use cases, such as (1) consensus building from clustered short reads of non-model genomes, (2) from genome-wide amplification setups, and (3) specific repeat-centred questions, such as the linked vs. unlinked arrangement of ribosomal genes. In all our use-cases, the derived consensuses are robust and representative. To evaluate overall performance, we compare our high-fidelity repeat consensuses to RepeatExplorer2-derived contigs and check, if they represent real transposable elements as found in long reads. Our results demonstrate that it is possible to generate useful, reliable and trustworthy consensuses from short reads by a combination from read cluster and genome assembly methods in an automatable way. Conclusion: We anticipate that our workflow opens the way towards more efficient and less manual repeat characterization and annotation, benefitting all genome studies, but especially those of non-model organisms.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
