<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Enhanced detection of RNA modifications and mappability with high-accuracy nanopore RNA basecalling models
Authors: Diensthuber, G.; Pryszcz, L.; Llovera, L.; Lucas, M. C.; Delgado-Tejedor, A.; Cruciani, S.; Roignant, J.-Y.; Begik, O.; Novoa, E. M.
Score: 35.5, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568965
In recent years, nanopore direct RNA sequencing (DRS) has established itself as a valuable tool for studying the epitranscriptome, due to its ability to detect multiple modifications within the same full-length native RNA molecules.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Enhanced detection of RNA modifications and mappability with high-accuracy nanopore RNA basecalling models
Authors: Diensthuber, G.; Pryszcz, L.; Llovera, L.; Lucas, M. C.; Delgado-Tejedor, A.; Cruciani, S.; Roignant, J.-Y.; Begik, O.; Novoa, E. M.
Score: 35.5, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568965
In recent years, nanopore direct RNA sequencing (DRS) has established itself as a valuable tool for studying the epitranscriptome, due to its ability to detect multiple modifications within the same full-length native RNA molecules." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-06T10:40:04+00:00" />
<meta property="article:modified_time" content="2023-12-06T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Enhanced detection of RNA modifications and mappability with high-accuracy nanopore RNA basecalling models
Authors: Diensthuber, G.; Pryszcz, L.; Llovera, L.; Lucas, M. C.; Delgado-Tejedor, A.; Cruciani, S.; Roignant, J.-Y.; Begik, O.; Novoa, E. M.
Score: 35.5, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568965
In recent years, nanopore direct RNA sequencing (DRS) has established itself as a valuable tool for studying the epitranscriptome, due to its ability to detect multiple modifications within the same full-length native RNA molecules."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Enhanced detection of RNA modifications and mappability with high-accuracy nanopore RNA basecalling models\nAuthors: Diensthuber, G.; Pryszcz, L.; Llovera, L.; Lucas, M. C.; Delgado-Tejedor, A.; Cruciani, S.; Roignant, J.-Y.; Begik, O.; Novoa, E. M.\nScore: 35.5, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568965\nIn recent years, nanopore direct RNA sequencing (DRS) has established itself as a valuable tool for studying the epitranscriptome, due to its ability to detect multiple modifications within the same full-length native RNA molecules.",
  "keywords": [
    
  ],
  "articleBody": " Enhanced detection of RNA modifications and mappability with high-accuracy nanopore RNA basecalling models\nAuthors: Diensthuber, G.; Pryszcz, L.; Llovera, L.; Lucas, M. C.; Delgado-Tejedor, A.; Cruciani, S.; Roignant, J.-Y.; Begik, O.; Novoa, E. M.\nScore: 35.5, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568965\nIn recent years, nanopore direct RNA sequencing (DRS) has established itself as a valuable tool for studying the epitranscriptome, due to its ability to detect multiple modifications within the same full-length native RNA molecules. While RNA modifications can be identified in the form of systematic basecalling errors in DRS datasets, N6-methyladenosine (m6A) modifications produce relatively low errors compared to other RNA modifications, limiting the applicability of this approach to m6A sites that are modified at high stoichiometries. Here, we demonstrate that the use of alternative RNA basecalling models, trained with fully unmodified sequences, increases the error signal of m6A, leading to enhanced detection and improved sensitivity even at low stoichiometries. Moreover, we find that high-accuracy alternative RNA basecalling models can show up to 97% median basecalling accuracy, outperforming currently available RNA basecalling models, which show 91% median basecalling accuracy. Notably, the use of high-accuracy basecalling models is accompanied by a significant increase in the number of mapped reads -especially in shorter RNA fractions- and increased basecalling error signatures at pseudouridine ({Psi}) and N1-methylpseudouridine (m1{Psi}) modified sites. Overall, our work demonstrates that alternative RNA basecalling models can be used to improve the detection of RNA modifications, read mappability and basecalling accuracy in nanopore DRS datasets.\nThe Complete Sequence and Comparative Analysis of Ape Sex Chromosomes\nAuthors: Makova, K. D.; Pickett, B. D.; Harris, R. S.; Hartley, G. A.; Cechova, M.; Pal, K.; Nurk, S.; Yoo, D.; Li, Q.; Hebbar, P.; McGrath, B. C.; Antonacci, F.; Aubel, M.; Biddanda, A.; Borchers, M.; Bornberg, E.; Bouffard, G. G.; Brooks, S. Y.; Carbone, L.; Carrel, L.; Carroll, A.; Chang, P.-C.; Chin, C.-S.; Cook, D. E.; Craig, S. J. C.; de Gennaro, L.; Diekhans, M.; Dutra, A.; Garcia, G. H.; Grady, P. G. S.; Green, R. E.; Haddad, D.; Hallast, P.; Harvey, W. T.; Hickey, G.; Hillis, D. A.; Hoyt, S. J.; Jeong, H.; Kamali, K.; Pond, S. L. K.; LaPolice, T. M.; Lee, C.; Lewis, A. P.; Loh, Y.-H. E.; Maste\nScore: 28.9, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569198\nApes possess two sex chromosomes--the male-specific Y and the X shared by males and females. The Y chromosome is crucial for male reproduction, with deletions linked to infertility. The X chromosome carries genes vital for reproduction and cognition. Variation in mating patterns and brain function among great apes suggests corresponding differences in their sex chromosome structure and evolution. However, due to their highly repetitive nature and incomplete reference assemblies, ape sex chromosomes have been challenging to study. Here, using the state-of-the-art experimental and computational methods developed for the telomere-to-telomere (T2T) human genome, we produced gapless, complete assemblies of the X and Y chromosomes for five great apes (chimpanzee, bonobo, gorilla, Bornean and Sumatran orangutans) and a lesser ape, the siamang gibbon. These assemblies completely resolved ampliconic, palindromic, and satellite sequences, including the entire centromeres, allowing us to untangle the intricacies of ape sex chromosome evolution. We found that, compared to the X, ape Y chromosomes vary greatly in size and have low alignability and high levels of structural rearrangements. This divergence on the Y arises from the accumulation of lineage-specific ampliconic regions and palindromes (which are shared more broadly among species on the X) and from the abundance of transposable elements and satellites (which have a lower representation on the X). Our analysis of Y chromosome genes revealed lineage-specific expansions of multi-copy gene families and signatures of purifying selection. In summary, the Y exhibits dynamic evolution, while the X is more stable. Finally, mapping short-read sequencing data from \u003e100 great ape individuals revealed the patterns of diversity and selection on their sex chromosomes, demonstrating the utility of these reference assemblies for studies of great ape evolution. These complete sex chromosome assemblies are expected to further inform conservation genetics of nonhuman apes, all of which are endangered species.\nOrigin and evolution of the bread wheat D genome\nAuthors: Cavalet-Giorsa, E.; Gonzalez-Munoz, A.; Athiyannan, N.; Holden, S.; Salhi, A.; Gardener, C.; Quiroz-Chavez, J.; Rustamova, S. M.; Elkot, A. F.; Patpour, M.; Rasheed, A.; Mao, L.; Lagudah, E. S.; Periyannan, S. K.; Sharon, A.; Himmelbach, A.; Reif, J. C.; Knauft, M.; Mascher, M.; Stein, N.; Chayut, N.; Ghosh, S.; Perovic, D.; Putra, A.; Perera, A. B.; Hu, C.-Y.; Yu, G.; Ahmed, H. I.; Laquai, K. D.; Rivera, L. F.; Chen, R.; Wang, Y.; Gao, X.; Liu, S.; Raupp, W. J.; Olson, E. L.; Lee, J.-Y.; Chhuneja, P.; Kaur, S.; Zhang, P.; Park, R. F.; Ding, Y.; Liu, D.-C.; Li, W.; Nasyrova, F. Y.; Dvorak, J.;\nScore: 28.1, Published: 2023-11-29 DOI: 10.1101/2023.11.29.568958\nBread wheat (Triticum aestivum) is a globally dominant crop and major source of calories and proteins for the human diet. Compared to its wild ancestors, modern bread wheat shows lower genetic diversity caused by polyploidisation, domestication, and breeding bottlenecks1,2. Wild wheat relatives represent genetic reservoirs, harbouring diversity and beneficial alleles that have not been incorporated into bread wheat. Here, we establish and analyse pangenome resources for Tauschs goatgrass, Aegilops tauschii, the donor of the bread wheat D genome. This new pangenome facilitated the cloning of a disease resistance gene and haplotype analysis across a complex disease resistance locus, allowing us to discern alleles from paralogous gene copies. We also reveal the complex genetic composition and history of the bread wheat D genome, involving previously unreported contributions from genetically and geographically discrete Ae. tauschii subpopulations. Together, our results reveal the complex history of the bread wheat D genome and demonstrate the potential of wild relatives in crop improvement.\nSystematic comparison of sequencing-based spatial transcriptomic methods\nAuthors: You, Y.; Fu, Y.; Li, L.; Zhang, Z.; Jia, S.; Lu, S.; Ren, W.; Liu, Y.; Xu, Y.; Liu, X.; Jiang, F.; Peng, G.; Kumar, A. S.; Ritchie, M. E.; Liu, X.; Tian, L.\nScore: 20.9, Published: 2023-12-04 DOI: 10.1101/2023.12.03.569744\nRecent advancements of sequencing-based spatial transcriptomics (sST) have catalyzed significant advancements by facilitating transcriptome-scale spatial gene expression measurement. Despite this progress, efforts to comprehensively benchmark different platforms are currently lacking. The extant variability across technologies and datasets poses challenges in formulating standardized evaluation metrics. In this study, we established a collection of reference tissues and regions characterized by well-defined histological architectures, and used them to generate data to compare six sST methods. We highlighted molecular diffusion as a variable parameter across different methods and tissues, significantly impacting the effective resolutions. Furthermore, we observed that spatial transcriptomic data demonstrate unique attributes beyond merely adding a spatial axis to single-cell data, including an enhanced ability to capture patterned rare cell states along with specific markers, albeit being influenced by multiple factors including sequencing depth and resolution. Our study assists biologists in sST platform selection, and helps foster a consensus on evaluation standards and establish a framework for future benchmarking efforts that can be used as a gold standard for the development and benchmarking of computational tools for spatial transcriptomic analysis.\nDiploid genome assembly of the Malbec grapevine cultivar enables haplotype-aware analysis of transcriptomic differences underlying clonal phenotypic variation\nAuthors: Calderon, L.; Carbonell-Bejerano, P.; Munoz, C.; Bree, L.; Sola, C.; Bergamin, D.; Tulle, W.; Gomez-Talquenca, S.; Lanz, C.; Royo, C.; Ibanez, J.; Martinez-Zapater, J. M.; Weigel, D.; Lijavetzky, D.\nScore: 13.3, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569420\nGrapevine cultivars (Vitis vinifera L. ssp. vinifera) must be clonally propagated because of their highly heterozygous genomes. Malbec, a France-originated cultivar appreciated for the production of high-quality wines, is the offspring of cultivars Prunelard and Magdeleine Noire des Charentes. Here, we have built a diploid genome assembly of Malbec, after trio binning of PacBio long reads into the two haploid complements inherited from either parent. After haplotype-aware deduplication and corrections, complete assemblies for the two haplophases were obtained with very low haplotype switch-error rate (\u003c0.025). The haplophases alignment identified \u003e25% of polymorphic regions. Gene annotation including RNA-seq transcriptome assembly and ab initio prediction evidence resulted in similar gene model numbers for both haplophases. The annotated diploid assembly was exploited in the transcriptomic comparison of four clonal accessions of Malbec that exhibited variation in berry composition traits. Analysis of the ripening pericarp transcriptome using either haplophases as reference yielded similar results, although some differences were observed. Particularly, among the differentially expressed genes in the Magdeleine-inherited haplotype, we observed an over-representation of hypothetically hemizygous genes. The higher berry anthocyanin content of clonal accession 595 was associated with increased abscisic acid responses, leading to overexpression of secondary metabolism genes and deregulation of genes associated to abiotic stress response. Overall, the results highlight the importance of producing diploid assemblies to fully represent the genomic diversity of highly heterozygous woody crop cultivars and to unveil the molecular bases of clonal phenotypic variation.\nEarly Pastoralism in Central European Forests: Insights from Ancient Environmental Genomics\nAuthors: Zampirolo, G.; Holman, L. E.; Sawafuji, R.; Ptakova, M.; Kovacikova, L.; Sida, P.; Pokorny, P.; Pedersen, M. W.; Walls, M.\nScore: 13.1, Published: 2023-12-03 DOI: 10.1101/2023.12.01.569562\nCentral European forests have been shaped by complex human interactions throughout the Holocene, with significant changes following the introduction of domesticated animals in the Neolithic (~7.5 - 6.0 kyr BP). However, understanding early pastoral practices and their impact on forests is limited by methods for detecting animal movement across past landscapes. Here we examine ancient sedimentary DNA (sedaDNA) preserved at the Velky Mamut'ak rock shelter, in northern Bohemia (Czech Republic), which has been a forested enclave since the early Holocene. We find that domesticated animals, their associated microbiomes, and plants potentially gathered for fodder, have clear representation by the Late Neolithic, around 6.0 kyr BP, and persist throughout the Bronze Age into recent times. We identify a change in dominant grazing species from sheep to pigs in the Bronze Age (~4.1 - 3.0 kyr BP) and interpret the impact this had in the mid-Holocene retrogressions that still define the structure of Central European forests today. This study highlights the ability of ancient metagenomics to bridge archaeological and paleoecological methods and provide an enhanced perspective on the roots of the Anthropocene.\nClimate change introduces threatened killer whale populations and conservation challenges to the Arctic\nAuthors: Garroway, C. J.; de Greef, E.; Lefort, K. J.; Thorstensen, M. J.; Foote, A.; Matthews, C. J.; Higdon, J. W.; Kucheravy, C. E.; Petersen, S. D.; Rosing-Asvid, A.; Ugarte, F.; Dietz, R.; Ferguson, S.\nScore: 13.0, Published: 2023-11-27 DOI: 10.1101/2023.11.25.568606\nThe Arctic is the fastest-warming region on the planet, and sea ice loss has opened new habitat for sub-Arctic species such as the killer whale (Orcinus orca). As apex predators, killer whales can cause significant ecosystem-scale changes, however, we know very little about killer whales in the Arctic. Setting conservation priorities for killer whales and their Arctic prey species requires knowledge of their evolutionary history and demography. We found that there are two highly genetically distinct, non-interbreeding populations of killer whales using the eastern Canadian Arctic--one population is newly identified as globally distinct. The effective sizes of both populations recently declined, and both are vulnerable to inbreeding and reduced adaptive potential. Furthermore, we present evidence that human-caused mortalities, particularly ongoing harvest, pose an ongoing threat to these populations. The certainty of substantial environmental change in the Arctic complicates conservation and management significantly. Killer whales bring top-down pressure to Arctic food webs, however, they also merit conservation concern. The opening of the Arctic to killer whales exemplifies the magnitude of complex decisions surrounding local peoples, wildlife conservation, and resource management as the effects of climate change are realized.\nExtensive remodelling of XIST regulatory networks during primate evolution\nAuthors: Cazottes, E.; Alfeghaly, C.; Rognard, C.; Loda, A.; Castel, G.; Villacorta, L.; Dong, M.; Heard, E.; Aksoy, I.; Savatier, P.; Morey, C.; Rougeulle, C.\nScore: 9.4, Published: 2023-12-04 DOI: 10.1101/2023.12.04.569904\nUnravelling how gene regulatory networks are remodelled during evolution is crucial to understand how species adapt to environmental changes. We addressed this question for X-chromosome inactivation, a process essential to female development that is governed, in eutherians, by the XIST lncRNA and its cis-regulators. To reach high resolution, we studied closely related primate species, spanning 55 million years of evolution. We show that the XIST regulatory circuitry has diversified extensively over such an evolutionary timeframe. The insertion of a HERVK transposon has reshuffled XIST 3D interaction network in macaque embryonic stem cells (ESC) and XIST expression is maintained by the additive effects of the JPX lncRNA gene and a macaque specific enhancer. In contrast, JPX is the main contributor to XIST expression in human ESCs but is not significantly involved in XIST regulation in marmoset ESCs. None of these entities are however under purifying selection, which suggests that neutrally evolving non-coding elements harbour high adaptive potentials.\nSimultaneous CRISPR screening and spatial transcriptomics reveals intracellular, intercellular, and functional transcriptional circuits.\nAuthors: Binan, L.; Danquah, S.; Valakh, V.; Simonton, B.; Bezney, J.; Nehme, R.; Cleary, B.; Farhi, S. L.\nScore: 9.1, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569494\nPooled optical screens have enabled the study of cellular interactions, morphology, or dynamics at massive scale, but have not yet leveraged the power of highly-plexed single-cell resolved transcriptomic readouts to inform molecular pathways. Here, we present Perturb-FISH, which bridges these approaches by combining imaging spatial transcriptomics with parallel optical detection of in situ amplified guide RNAs. We show that Perturb-FISH recovers intracellular effects that are consistent with Perturb-seq results in a screen of lipopolysaccharide response in cultured monocytes, and uncover new intercellular and density-dependent regulation of the innate immune response. We further pair Perturb-FISH with a functional readout in a screen of autism spectrum disorder risk genes, showing common calcium activity phenotypes in induced pluripotent stem cell derived astrocytes and their associated genetic interactions and dysregulated molecular pathways. Perturb-FISH is thus a generally applicable method for studying the genetic and molecular associations of spatial and functional biology at single-cell resolution.\nAn optimised protocol for quality control of gene therapy vectors using Nanopore direct RNA sequencing\nAuthors: Zeglinski, K.; Montellese, C.; Ritchie, M. E.; Alhamdoosh, M.; Vonarburg, C.; Bowden, R.; Jordi, M.; Gouil, Q.; Aeschimann, F.; Hsu, A.\nScore: 9.0, Published: 2023-12-03 DOI: 10.1101/2023.12.03.569756\nDespite recent advances made towards improving the efficacy of lentiviral gene therapies, a sizeable proportion of produced vector contains an incomplete and thus potentially non-functional RNA genome. This can undermine gene delivery by the lentivirus as well as increase manufacturing costs and must be improved to facilitate the widespread clinical implementation of lentiviral gene therapies. Here, we compare three long-read sequencing technologies for their ability to detect issues in vector design and determine Nanopore direct RNA sequencing to be the most powerful. We show how this approach identifies and quantifies incomplete RNA caused by cryptic splicing and polyadenylation sites, including a potential cryptic polyadenylation site in the widely used Woodchuck Hepatitis Virus Posttranscriptional Regulatory Element (WPRE). Using artificial polyadenylation of the lentiviral RNA, we also identify multiple hairpin-associated truncations in the analysed lentiviral vectors, which account for most of the detected RNA fragments. Finally, we show that these insights can be used for optimization of lentiviral vector design. In summary, Nanopore direct RNA sequencing is a powerful tool for the quality control and optimisation of lentiviral vectors, which may help to improve lentivirus manufacturing and thus the development of higher quality lentiviral gene therapies.\n",
  "wordCount" : "2544",
  "inLanguage": "en",
  "datePublished": "2023-12-06T10:40:04Z",
  "dateModified": "2023-12-06T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 6, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568965">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568965" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568965">
        <p class="paperTitle">Enhanced detection of RNA modifications and mappability with high-accuracy nanopore RNA basecalling models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568965" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568965" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Diensthuber, G.; Pryszcz, L.; Llovera, L.; Lucas, M. C.; Delgado-Tejedor, A.; Cruciani, S.; Roignant, J.-Y.; Begik, O.; Novoa, E. M.</p>
        <p class="info">Score: 35.5, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568965' target='https://doi.org/10.1101/2023.11.28.568965'> 10.1101/2023.11.28.568965</a></p>
        <p class="abstract">In recent years, nanopore direct RNA sequencing (DRS) has established itself as a valuable tool for studying the epitranscriptome, due to its ability to detect multiple modifications within the same full-length native RNA molecules. While RNA modifications can be identified in the form of systematic basecalling  errors in DRS datasets, N6-methyladenosine (m6A) modifications produce relatively low  errors compared to other RNA modifications, limiting the applicability of this approach to m6A sites that are modified at high stoichiometries. Here, we demonstrate that the use of alternative RNA basecalling models, trained with fully unmodified sequences, increases the  error signal of m6A, leading to enhanced detection and improved sensitivity even at low stoichiometries. Moreover, we find that high-accuracy alternative RNA basecalling models can show up to 97% median basecalling accuracy, outperforming currently available RNA basecalling models, which show 91% median basecalling accuracy. Notably, the use of high-accuracy basecalling models is accompanied by a significant increase in the number of mapped reads -especially in shorter RNA fractions- and increased basecalling error signatures at pseudouridine ({Psi}) and N1-methylpseudouridine (m1{Psi}) modified sites. Overall, our work demonstrates that alternative RNA basecalling models can be used to improve the detection of RNA modifications, read mappability and basecalling accuracy in nanopore DRS datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569198">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569198" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569198">
        <p class="paperTitle">The Complete Sequence and Comparative Analysis of Ape Sex Chromosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569198" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569198" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Makova, K. D.; Pickett, B. D.; Harris, R. S.; Hartley, G. A.; Cechova, M.; Pal, K.; Nurk, S.; Yoo, D.; Li, Q.; Hebbar, P.; McGrath, B. C.; Antonacci, F.; Aubel, M.; Biddanda, A.; Borchers, M.; Bornberg, E.; Bouffard, G. G.; Brooks, S. Y.; Carbone, L.; Carrel, L.; Carroll, A.; Chang, P.-C.; Chin, C.-S.; Cook, D. E.; Craig, S. J. C.; de Gennaro, L.; Diekhans, M.; Dutra, A.; Garcia, G. H.; Grady, P. G. S.; Green, R. E.; Haddad, D.; Hallast, P.; Harvey, W. T.; Hickey, G.; Hillis, D. A.; Hoyt, S. J.; Jeong, H.; Kamali, K.; Pond, S. L. K.; LaPolice, T. M.; Lee, C.; Lewis, A. P.; Loh, Y.-H. E.; Maste</p>
        <p class="info">Score: 28.9, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569198' target='https://doi.org/10.1101/2023.11.30.569198'> 10.1101/2023.11.30.569198</a></p>
        <p class="abstract">Apes possess two sex chromosomes--the male-specific Y and the X shared by males and females. The Y chromosome is crucial for male reproduction, with deletions linked to infertility. The X chromosome carries genes vital for reproduction and cognition. Variation in mating patterns and brain function among great apes suggests corresponding differences in their sex chromosome structure and evolution. However, due to their highly repetitive nature and incomplete reference assemblies, ape sex chromosomes have been challenging to study. Here, using the state-of-the-art experimental and computational methods developed for the telomere-to-telomere (T2T) human genome, we produced gapless, complete assemblies of the X and Y chromosomes for five great apes (chimpanzee, bonobo, gorilla, Bornean and Sumatran orangutans) and a lesser ape, the siamang gibbon. These assemblies completely resolved ampliconic, palindromic, and satellite sequences, including the entire centromeres, allowing us to untangle the intricacies of ape sex chromosome evolution. We found that, compared to the X, ape Y chromosomes vary greatly in size and have low alignability and high levels of structural rearrangements. This divergence on the Y arises from the accumulation of lineage-specific ampliconic regions and palindromes (which are shared more broadly among species on the X) and from the abundance of transposable elements and satellites (which have a lower representation on the X). Our analysis of Y chromosome genes revealed lineage-specific expansions of multi-copy gene families and signatures of purifying selection. In summary, the Y exhibits dynamic evolution, while the X is more stable. Finally, mapping short-read sequencing data from &gt;100 great ape individuals revealed the patterns of diversity and selection on their sex chromosomes, demonstrating the utility of these reference assemblies for studies of great ape evolution. These complete sex chromosome assemblies are expected to further inform conservation genetics of nonhuman apes, all of which are endangered species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.568958">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.568958" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.568958">
        <p class="paperTitle">Origin and evolution of the bread wheat D genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.568958" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.568958" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cavalet-Giorsa, E.; Gonzalez-Munoz, A.; Athiyannan, N.; Holden, S.; Salhi, A.; Gardener, C.; Quiroz-Chavez, J.; Rustamova, S. M.; Elkot, A. F.; Patpour, M.; Rasheed, A.; Mao, L.; Lagudah, E. S.; Periyannan, S. K.; Sharon, A.; Himmelbach, A.; Reif, J. C.; Knauft, M.; Mascher, M.; Stein, N.; Chayut, N.; Ghosh, S.; Perovic, D.; Putra, A.; Perera, A. B.; Hu, C.-Y.; Yu, G.; Ahmed, H. I.; Laquai, K. D.; Rivera, L. F.; Chen, R.; Wang, Y.; Gao, X.; Liu, S.; Raupp, W. J.; Olson, E. L.; Lee, J.-Y.; Chhuneja, P.; Kaur, S.; Zhang, P.; Park, R. F.; Ding, Y.; Liu, D.-C.; Li, W.; Nasyrova, F. Y.; Dvorak, J.;</p>
        <p class="info">Score: 28.1, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.568958' target='https://doi.org/10.1101/2023.11.29.568958'> 10.1101/2023.11.29.568958</a></p>
        <p class="abstract">Bread wheat (Triticum aestivum) is a globally dominant crop and major source of calories and proteins for the human diet. Compared to its wild ancestors, modern bread wheat shows lower genetic diversity caused by polyploidisation, domestication, and breeding bottlenecks1,2. Wild wheat relatives represent genetic reservoirs, harbouring diversity and beneficial alleles that have not been incorporated into bread wheat. Here, we establish and analyse pangenome resources for Tauschs goatgrass, Aegilops tauschii, the donor of the bread wheat D genome. This new pangenome facilitated the cloning of a disease resistance gene and haplotype analysis across a complex disease resistance locus, allowing us to discern alleles from paralogous gene copies. We also reveal the complex genetic composition and history of the bread wheat D genome, involving previously unreported contributions from genetically and geographically discrete Ae. tauschii subpopulations. Together, our results reveal the complex history of the bread wheat D genome and demonstrate the potential of wild relatives in crop improvement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569744">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569744" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569744">
        <p class="paperTitle">Systematic comparison of sequencing-based spatial transcriptomic methods</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569744" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569744" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: You, Y.; Fu, Y.; Li, L.; Zhang, Z.; Jia, S.; Lu, S.; Ren, W.; Liu, Y.; Xu, Y.; Liu, X.; Jiang, F.; Peng, G.; Kumar, A. S.; Ritchie, M. E.; Liu, X.; Tian, L.</p>
        <p class="info">Score: 20.9, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569744' target='https://doi.org/10.1101/2023.12.03.569744'> 10.1101/2023.12.03.569744</a></p>
        <p class="abstract">Recent advancements of sequencing-based spatial transcriptomics (sST) have catalyzed significant advancements by facilitating transcriptome-scale spatial gene expression measurement. Despite this progress, efforts to comprehensively benchmark different platforms are currently lacking. The extant variability across technologies and datasets poses challenges in formulating standardized evaluation metrics. In this study, we established a collection of reference tissues and regions characterized by well-defined histological architectures, and used them to generate data to compare six sST methods. We highlighted molecular diffusion as a variable parameter across different methods and tissues, significantly impacting the effective resolutions. Furthermore, we observed that spatial transcriptomic data demonstrate unique attributes beyond merely adding a spatial axis to single-cell data, including an enhanced ability to capture patterned rare cell states along with specific markers, albeit being influenced by multiple factors including sequencing depth and resolution. Our study assists biologists in sST platform selection, and helps foster a consensus on evaluation standards and establish a framework for future benchmarking efforts that can be used as a gold standard for the development and benchmarking of computational tools for spatial transcriptomic analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569420">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569420" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569420">
        <p class="paperTitle">Diploid genome assembly of the Malbec grapevine cultivar enables haplotype-aware analysis of transcriptomic differences underlying clonal phenotypic variation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569420" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569420" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Calderon, L.; Carbonell-Bejerano, P.; Munoz, C.; Bree, L.; Sola, C.; Bergamin, D.; Tulle, W.; Gomez-Talquenca, S.; Lanz, C.; Royo, C.; Ibanez, J.; Martinez-Zapater, J. M.; Weigel, D.; Lijavetzky, D.</p>
        <p class="info">Score: 13.3, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569420' target='https://doi.org/10.1101/2023.11.30.569420'> 10.1101/2023.11.30.569420</a></p>
        <p class="abstract">Grapevine cultivars (Vitis vinifera L. ssp. vinifera) must be clonally propagated because of their highly heterozygous genomes. Malbec, a France-originated cultivar appreciated for the production of high-quality wines, is the offspring of cultivars Prunelard and Magdeleine Noire des Charentes. Here, we have built a diploid genome assembly of Malbec, after trio binning of PacBio long reads into the two haploid complements inherited from either parent. After haplotype-aware deduplication and corrections, complete assemblies for the two haplophases were obtained with very low haplotype switch-error rate (&lt;0.025). The haplophases alignment identified &gt;25% of polymorphic regions. Gene annotation including RNA-seq transcriptome assembly and ab initio prediction evidence resulted in similar gene model numbers for both haplophases. The annotated diploid assembly was exploited in the transcriptomic comparison of four clonal accessions of Malbec that exhibited variation in berry composition traits. Analysis of the ripening pericarp transcriptome using either haplophases as reference yielded similar results, although some differences were observed. Particularly, among the differentially expressed genes in the Magdeleine-inherited haplotype, we observed an over-representation of hypothetically hemizygous genes. The higher berry anthocyanin content of clonal accession 595 was associated with increased abscisic acid responses, leading to overexpression of secondary metabolism genes and deregulation of genes associated to abiotic stress response. Overall, the results highlight the importance of producing diploid assemblies to fully represent the genomic diversity of highly heterozygous woody crop cultivars and to unveil the molecular bases of clonal phenotypic variation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569562">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569562" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569562">
        <p class="paperTitle">Early Pastoralism in Central European Forests: Insights from Ancient Environmental Genomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569562" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569562" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zampirolo, G.; Holman, L. E.; Sawafuji, R.; Ptakova, M.; Kovacikova, L.; Sida, P.; Pokorny, P.; Pedersen, M. W.; Walls, M.</p>
        <p class="info">Score: 13.1, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569562' target='https://doi.org/10.1101/2023.12.01.569562'> 10.1101/2023.12.01.569562</a></p>
        <p class="abstract">Central European forests have been shaped by complex human interactions throughout the Holocene, with significant changes following the introduction of domesticated animals in the Neolithic (~7.5 - 6.0 kyr BP). However, understanding early pastoral practices and their impact on forests is limited by methods for detecting animal movement across past landscapes. Here we examine ancient sedimentary DNA (sedaDNA) preserved at the Velky Mamut&#39;ak rock shelter, in northern Bohemia (Czech Republic), which has been a forested enclave since the early Holocene. We find that domesticated animals, their associated microbiomes, and plants potentially gathered for fodder, have clear representation by the Late Neolithic, around 6.0 kyr BP, and persist throughout the Bronze Age into recent times. We identify a change in dominant grazing species from sheep to pigs in the Bronze Age (~4.1 - 3.0 kyr BP) and interpret the impact this had in the mid-Holocene retrogressions that still define the structure of Central European forests today. This study highlights the ability of ancient metagenomics to bridge archaeological and paleoecological methods and provide an enhanced perspective on the roots of the Anthropocene.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.568606">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.568606" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.568606">
        <p class="paperTitle">Climate change introduces threatened killer whale populations and conservation challenges to the Arctic</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.568606" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.568606" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garroway, C. J.; de Greef, E.; Lefort, K. J.; Thorstensen, M. J.; Foote, A.; Matthews, C. J.; Higdon, J. W.; Kucheravy, C. E.; Petersen, S. D.; Rosing-Asvid, A.; Ugarte, F.; Dietz, R.; Ferguson, S.</p>
        <p class="info">Score: 13.0, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.568606' target='https://doi.org/10.1101/2023.11.25.568606'> 10.1101/2023.11.25.568606</a></p>
        <p class="abstract">The Arctic is the fastest-warming region on the planet, and sea ice loss has opened new habitat for sub-Arctic species such as the killer whale (Orcinus orca). As apex predators, killer whales can cause significant ecosystem-scale changes, however, we know very little about killer whales in the Arctic. Setting conservation priorities for killer whales and their Arctic prey species requires knowledge of their evolutionary history and demography. We found that there are two highly genetically distinct, non-interbreeding populations of killer whales using the eastern Canadian Arctic--one population is newly identified as globally distinct. The effective sizes of both populations recently declined, and both are vulnerable to inbreeding and reduced adaptive potential. Furthermore, we present evidence that human-caused mortalities, particularly ongoing harvest, pose an ongoing threat to these populations. The certainty of substantial environmental change in the Arctic complicates conservation and management significantly. Killer whales bring top-down pressure to Arctic food webs, however, they also merit conservation concern. The opening of the Arctic to killer whales exemplifies the magnitude of complex decisions surrounding local peoples, wildlife conservation, and resource management as the effects of climate change are realized.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569904">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569904" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569904">
        <p class="paperTitle">Extensive remodelling of XIST regulatory networks during primate evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569904" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569904" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cazottes, E.; Alfeghaly, C.; Rognard, C.; Loda, A.; Castel, G.; Villacorta, L.; Dong, M.; Heard, E.; Aksoy, I.; Savatier, P.; Morey, C.; Rougeulle, C.</p>
        <p class="info">Score: 9.4, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569904' target='https://doi.org/10.1101/2023.12.04.569904'> 10.1101/2023.12.04.569904</a></p>
        <p class="abstract">Unravelling how gene regulatory networks are remodelled during evolution is crucial to understand how species adapt to environmental changes. We addressed this question for X-chromosome inactivation, a process essential to female development that is governed, in eutherians, by the XIST lncRNA and its cis-regulators. To reach high resolution, we studied closely related primate species, spanning 55 million years of evolution. We show that the XIST regulatory circuitry has diversified extensively over such an evolutionary timeframe. The insertion of a HERVK transposon has reshuffled XIST 3D interaction network in macaque embryonic stem cells (ESC) and XIST expression is maintained by the additive effects of the JPX lncRNA gene and a macaque specific enhancer. In contrast, JPX is the main contributor to XIST expression in human ESCs but is not significantly involved in XIST regulation in marmoset ESCs. None of these entities are however under purifying selection, which suggests that neutrally evolving non-coding elements harbour high adaptive potentials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569494">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569494" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569494">
        <p class="paperTitle">Simultaneous CRISPR screening and spatial transcriptomics reveals intracellular, intercellular, and functional transcriptional circuits.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569494" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569494" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Binan, L.; Danquah, S.; Valakh, V.; Simonton, B.; Bezney, J.; Nehme, R.; Cleary, B.; Farhi, S. L.</p>
        <p class="info">Score: 9.1, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569494' target='https://doi.org/10.1101/2023.11.30.569494'> 10.1101/2023.11.30.569494</a></p>
        <p class="abstract">Pooled optical screens have enabled the study of cellular interactions, morphology, or dynamics at massive scale, but have not yet leveraged the power of highly-plexed single-cell resolved transcriptomic readouts to inform molecular pathways. Here, we present Perturb-FISH, which bridges these approaches by combining imaging spatial transcriptomics with parallel optical detection of in situ amplified guide RNAs. We show that Perturb-FISH recovers intracellular effects that are consistent with Perturb-seq results in a screen of lipopolysaccharide response in cultured monocytes, and uncover new intercellular and density-dependent regulation of the innate immune response. We further pair Perturb-FISH with a functional readout in a screen of autism spectrum disorder risk genes, showing common calcium activity phenotypes in induced pluripotent stem cell derived astrocytes and their associated genetic interactions and dysregulated molecular pathways. Perturb-FISH is thus a generally applicable method for studying the genetic and molecular associations of spatial and functional biology at single-cell resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569756" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569756">
        <p class="paperTitle">An optimised protocol for quality control of gene therapy vectors using Nanopore direct RNA sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zeglinski, K.; Montellese, C.; Ritchie, M. E.; Alhamdoosh, M.; Vonarburg, C.; Bowden, R.; Jordi, M.; Gouil, Q.; Aeschimann, F.; Hsu, A.</p>
        <p class="info">Score: 9.0, Published: 2023-12-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569756' target='https://doi.org/10.1101/2023.12.03.569756'> 10.1101/2023.12.03.569756</a></p>
        <p class="abstract">Despite recent advances made towards improving the efficacy of lentiviral gene therapies, a sizeable proportion of produced vector contains an incomplete and thus potentially non-functional RNA genome. This can undermine gene delivery by the lentivirus as well as increase manufacturing costs and must be improved to facilitate the widespread clinical implementation of lentiviral gene therapies. Here, we compare three long-read sequencing technologies for their ability to detect issues in vector design and determine Nanopore direct RNA sequencing to be the most powerful. We show how this approach identifies and quantifies incomplete RNA caused by cryptic splicing and polyadenylation sites, including a potential cryptic polyadenylation site in the widely used Woodchuck Hepatitis Virus Posttranscriptional Regulatory Element (WPRE). Using artificial polyadenylation of the lentiviral RNA, we also identify multiple hairpin-associated truncations in the analysed lentiviral vectors, which account for most of the detected RNA fragments. Finally, we show that these insights can be used for optimization of lentiviral vector design. In summary, Nanopore direct RNA sequencing is a powerful tool for the quality control and optimisation of lentiviral vectors, which may help to improve lentivirus manufacturing and thus the development of higher quality lentiviral gene therapies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
