<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Reconstructing the genetic relationship between ancient and present-day Siberian populations
Authors: Gill, H.; Lee, J.; Jeong, C.
Score: 22.6, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554074
Human populations across a vast area in northern Eurasia, from Fennoscandia to Chukotka, share a distinct genetic component often referred to as the Siberian ancestry. Most enriched in present-day Samoyedic-speaking populations such as Nganasans, its origins and history still remain elusive despite the growing list of ancient and present-day genomes from Siberia.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Reconstructing the genetic relationship between ancient and present-day Siberian populations
Authors: Gill, H.; Lee, J.; Jeong, C.
Score: 22.6, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554074
Human populations across a vast area in northern Eurasia, from Fennoscandia to Chukotka, share a distinct genetic component often referred to as the Siberian ancestry. Most enriched in present-day Samoyedic-speaking populations such as Nganasans, its origins and history still remain elusive despite the growing list of ancient and present-day genomes from Siberia." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-30T10:38:23+00:00" />
<meta property="article:modified_time" content="2023-08-30T10:38:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Reconstructing the genetic relationship between ancient and present-day Siberian populations
Authors: Gill, H.; Lee, J.; Jeong, C.
Score: 22.6, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554074
Human populations across a vast area in northern Eurasia, from Fennoscandia to Chukotka, share a distinct genetic component often referred to as the Siberian ancestry. Most enriched in present-day Samoyedic-speaking populations such as Nganasans, its origins and history still remain elusive despite the growing list of ancient and present-day genomes from Siberia."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Reconstructing the genetic relationship between ancient and present-day Siberian populations\nAuthors: Gill, H.; Lee, J.; Jeong, C.\nScore: 22.6, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554074\nHuman populations across a vast area in northern Eurasia, from Fennoscandia to Chukotka, share a distinct genetic component often referred to as the Siberian ancestry. Most enriched in present-day Samoyedic-speaking populations such as Nganasans, its origins and history still remain elusive despite the growing list of ancient and present-day genomes from Siberia.",
  "keywords": [
    
  ],
  "articleBody": " Reconstructing the genetic relationship between ancient and present-day Siberian populations\nAuthors: Gill, H.; Lee, J.; Jeong, C.\nScore: 22.6, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554074\nHuman populations across a vast area in northern Eurasia, from Fennoscandia to Chukotka, share a distinct genetic component often referred to as the Siberian ancestry. Most enriched in present-day Samoyedic-speaking populations such as Nganasans, its origins and history still remain elusive despite the growing list of ancient and present-day genomes from Siberia. Here we reanalyze published ancient and present-day Siberian genomes focusing on the Baikal and Yakutia, resolving key questions regarding their genetic history. First, we show a long-term presence of a unique genetic profile in southern Siberia, up to 6,000 years ago, which distinctly shares a deep ancestral connection with Native Americans. Second, in the Baikal we find no direct contribution of the Early Neolithic Kitoi people to Late Neolithic and Early Bronze Age Serovo-Glazkovo ones. Third, the Middle Neolithic individual from Yakutia, belonging to the Belkachi culture, serves as the best source so far available for the spread of the Siberian ancestry into Fennoscandia and Greenland. These findings shed light on the genetic legacy of the Siberian ancestry and provide insights into the complex interplay between different populations in northern Eurasia throughout history.\nThe genome of the colonial hydroid Hydractinia reveals their stem cells use a toolkit of genes shared with all animals\nAuthors: Schnitzler, C. E.; Chang, E. S.; Waletich, J.; Wong, W. Y.; Nguyen, A.-D.; Barreira, S.; Doonan, L.; Gonzalez, P.; Koren, S.; Gahan, J.; Quiroga-Artigas, G.; Sanders, S.; Bradshaw, B.; DuBuc, T.; Febrimarsa, F.; de Jong, D.; Nawrocki, E.; Larson, A.; Klasfeld, S.; Gornik, S.; Moreland, R. T.; Wolfsberg, T.; Phillippy, A. M.; Mullikin, J.; Simakov, O.; Cartwright, P.; Nicotra, M.; Frank, U.; Baxevanis, A. D.\nScore: 18.1, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554815\nHydractinia is a colonial marine hydroid that exhibits remarkable biological properties, including the capacity to regenerate its entire body throughout its lifetime, a process made possible by its adult migratory stem cells, known as i-cells. Here, we provide an in-depth characterization of the genomic structure and gene content of two Hydractinia species, H. symbiolongicarpus and H. echinata, placing them in a comparative evolutionary framework with other cnidarian genomes. We also generated and annotated a single-cell transcriptomic atlas for adult male H. symbiolongicarpus and identified cell type markers for all major cell types, including key i-cell markers. Orthology analyses based on the markers revealed that Hydractinia's i-cells are highly enriched in genes that are widely shared amongst animals, a striking finding given that Hydractinia has a higher proportion of phylum-specific genes than any of the other 41 animals in our orthology analysis. These results indicate that Hydractinia's stem cells and early progenitor cells may use a toolkit shared with all animals, making it a promising model organism for future exploration of stem cell biology and regenerative medicine. The genomic and transcriptomic resources for Hydractinia presented here will enable further studies of their regenerative capacity, colonial morphology, and ability to distinguish self from non-self.\nBoth phenotypic and genotypic sex influence sex chromosome dosage compensation in a sex reversing lizard\nAuthors: Hanrahan, B. J.; Chang, J. K.; Lister, N. C.; Dissanayake, D. S. B.; Hammond, J. M.; Reis, A. L. M.; Deveson, I. W.; Ruiz-Herrera, A.; Patel, H. R.; Marshall Graves, J. A.; Georges, A.; Waters, P. D.\nScore: 17.3, Published: 2023-08-26 DOI: 10.1101/2023.08.24.554710\nStudies of sex chromosome dosage compensation have historically focussed on therian mammals which have a conserved XY sex determination system. In contrast, lizards have sex determination systems that can differ between even closely related species that include XY and ZW systems and thermolabile systems where genetic and temperature interact to various degrees to determine sex. The eastern three-lined skink (Bassiana duperreyi) has a differentiated XY sex determination system, in which low temperature incubation during development can cause female to male sex reversal, producing XX males. This provides a unique opportunity to investigate how genotype and phenotype affect dosage compensation. We generated transcriptomes from brain and heart tissue of normal adult males and females, along with brain tissue of sex-reversed XX males. We observed partial dosage compensation between XX females and XY males in both brain and heart, with median gene expression from the X in normal males being 0.7 times that of normal females. Surprisingly, in brain of sex reversed XX males the median X chromosome output did not match that of either normal males or females, but instead was 0.89 times that of the normal XX female level. This suggests that not just genotype, but also sexual phenotype, influences gene dosage of the X chromosome. This has profound implications for our understanding of the evolution of dosage compensation.\nA Pooled Cell Painting CRISPR Screening Platform Enables de novo Inference of Gene Function by Self-supervised Deep Learning\nAuthors: Sivanandan, S.; Leitmann, B.; Lubeck, E.; Sultan, M. M.; Stanitsas, P.; Ranu, N.; Ewer, A.; Mancuso, J. E.; Phillips, Z. F.; Kim, A.; Bisognano, J. W.; Cesarek, J.; Ruggiu, F.; Feldman, D.; Koller, D.; Sharon, E.; Kaykas, A.; Salick, M. R.; Chu, C.\nScore: 24.1, Published: 2023-08-27 DOI: 10.1101/2023.08.13.553051\nPooled CRISPR screening has emerged as a powerful method of mapping gene functions thanks to its scalability, affordability, and robustness against well or plate-specific confounders present in array-based screening. Most pooled CRISPR screens assay for low dimensional phenotypes (e.g. fitness, fluorescent markers). Higher-dimensional assays such as perturb-seq are available but costly and only applicable to transcriptomics readouts. Recently, pooled optical screening, which combines pooled CRISPR screening and microscopy-based assays, has been demonstrated in the studies of the NFkB pathway, essential human genes, cytoskeletal organization and antiviral response. While the pooled optical screening methodology is scalable and information-rich, the applications thus far employ hypothesis-specific assays. Here, we enable hypothesis-free reverse genetic screening for generic morphological phenotypes by re-engineering the Cell Painting technique to provide compatibility with pooled optical screening. We validated this technique using well-defined morphological genesets (124 genes), compared classical image analysis and self-supervised learning methods using a mechanism-of-action (MoA) library (300 genes), and performed discovery screening with a druggable genome library (1640 genes). Across these three experiments we show that the combination of rich morphological data and deep learning allows gene networks to emerge without the need for target-specific biomarkers, leading to better discovery of gene functions.\nA novel, N2-fixing cyanobacterium present and active in the global oceans\nAuthors: Cleveland, C. S.; Turk-Kubo, K. A.; Zhao, Y.; Zehr, J. P.; Webb, E. A.\nScore: 13.8, Published: 2023-08-28 DOI: 10.1101/2023.08.27.555023\nMarine N2-fixing cyanobacteria, including the unicellular genus Crocosphaera, are considered keystone species in marine food webs. Crocosphaera are globally distributed and provide new sources of nitrogen (N) and carbon (C), which fuel oligotrophic microbial communities and upper trophic levels. Despite their ecosystem importance, only one species, Crocosphaera watsonii, has ever been identified and characterized as widespread in the oligotrophic oceans. Herein, we present a novel species, candidatus Crocosphaera waterburyi (C. waterburyi hereafter), enriched from the North Pacific Ocean, active in situ, and globally distributed in environmental datasets. C. waterburyi is morphologically, phylogenetically, and physiologically distinct from C. watsonii; therefore, description of this novel species provides a new window into previously uncharacterized diversity and ecology of unicellular N2-fixing cyanobacterial taxa and further highlights their importance in the global N cycle.\nActive learning of enhancer and silencer regulatory grammar in photoreceptors\nAuthors: Friedman, R. Z.; Ramu, A.; Lichtarge, S.; Myers, C. A.; Granas, D. M.; Gause, M.; Corbo, J. C.; Cohen, B. A.; White, M. A.\nScore: 21.6, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554146\nCis-regulatory elements (CREs) direct gene expression in health and disease, and models that can accurately predict their activities from DNA sequences are crucial for biomedicine. Deep learning represents one emerging strategy to model the regulatory grammar that relates CRE sequence to function. However, these models require training data on a scale that exceeds the number of CREs in the genome. We address this problem using active machine learning to iteratively train models on multiple rounds of synthetic DNA sequences assayed in live mammalian retinas. During each round of training the model actively selects sequence perturbations to assay, thereby efficiently generating informative training data. We iteratively trained a model that predicts the activities of sequences containing binding motifs for the photoreceptor transcription factor Cone-rod homeobox (CRX) using an order of magnitude less training data than current approaches. The models internal confidence estimates of its predictions are reliable guides for designing sequences with high activity. The model correctly identified critical sequence differences between active and inactive sequences with nearly identical transcription factor binding sites, and revealed order and spacing preferences for combinations of motifs. Our results establish active learning as an effective method to train accurate deep learning models of cis-regulatory function after exhausting naturally occurring training examples in the genome.\nA Spatiotemporal Molecular Atlas of the Ovulating Mouse Ovary\nAuthors: Mantri, M.; Zhang, H. H.; Spanos, E.; Ren, Y. A.; De Vlaminck, I.\nScore: 9.4, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554210\nOvulation is essential for reproductive success, yet the underlying cellular and molecular mechanisms are far from clear. Here, we applied high-resolution spatiotemporal transcriptomics to map out cell-type- and ovulation-stage-specific molecular programs as function of time during follicle maturation and ovulation in mice. Our analysis revealed dynamic molecular transitions within granulosa cell types that occur in tight coordination with mesenchymal cell proliferation. We identified new molecular markers for the emerging cumulus cell fate during the preantral-to-antral transition. We describe transcriptional programs that respond rapidly to ovulation stimulation and those associated with follicle rupture, highlighting the prominent roles of apoptotic and metabolic pathways during the final stages of follicle maturation. We further report stage-specific oocyte-cumulus cell interactions and diverging molecular differentiation in follicles approaching ovulation. Collectively, this study provides insights into the cellular and molecular processes that regulate mouse ovarian follicle maturation and ovulation with important implications for advancing therapeutic strategies in reproductive medicine.\nThe genome of the Wollemi pine, a critically endangered living fossil unchanged since the Cretaceous, reveals extensive ancient transposon activity.\nAuthors: Stevenson, D. W.; Ramakrishnan, S.; Alves, C. d. S.; Coelho, L. A.; Kramer, M.; Goodwin, S.; Ramos, O. M.; Eshel, G.; Sondervan, V. M.; Frangos, S.; Zumajo-Cardona, C.; Jenike, K.; Ou, S.; Wang, X.; Lee, Y. P.; Loke, S.; Rossetto, M.; McPherson, H.; Nigris, S.; Moschin, S.; Little, D. P.; Katari, M. S.; Varala, K.; Kolokotronis, S.-O.; Ambrose, B.; Croft, L. J.; Coruzzi, G. M.; Schatz, M.; McCombie, W. R.; Martienssen, R. A.\nScore: 8.2, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554647\nWe present the genome of the living fossil, Wollemia nobilis, a southern hemisphere conifer morphologically unchanged since the Cretaceous. Presumed extinct until rediscovery in 1994, the Wollemi pine is critically endangered with less than 60 wild adults threatened by intensifying bushfires in the Blue Mountains of Australia. The 12 Gb genome is among the most contiguous large plant genomes assembled, with extremely low heterozygosity and unusual abundance of DNA transposons. Reduced representation and genome re-sequencing of individuals confirms a relictual population since the last major glacial/drying period in Australia, 120 ky BP. Small RNA and methylome sequencing reveal conservation of ancient silencing mechanisms despite the presence of thousands of active and abundant transposons, including some transferred horizontally to conifers from arthropods in the Jurassic. A retrotransposon burst 8-6 my BP coincided with population decline, possibly as an adaptation enhancing epigenetic diversity. Wollemia, like other conifers, is susceptible to Phytophthora, and a suite of defense genes, similar to those in loblolly pine, are targeted for silencing by sRNAs in leaves. The genome provides insight into the earliest seed plants, while enabling conservation efforts.\nProgrammable multi-kilobase RNA editing using CRISPR-mediated trans-splicing\nAuthors: Borrajo, J.; Javanmardi, K.; Griffin, J.; St. Martin, S. J.; Yao, D.; Hill, K.; Blainey, P.; Al-Shayeb, B.\nScore: 103.4, Published: 2023-08-18 DOI: 10.1101/2023.08.18.553620\nCurrent gene editing approaches in eukaryotic cells are limited to single base edits or small DNA insertions and deletions, and remain encumbered by unintended permanent effects and significant challenges in the delivery of large DNA cargo. Here we describe Splice Editing, a generalizable platform to correct gene transcripts in situ by programmable insertion or replacement of large RNA segments. By combining CRISPR-mediated RNA targeting with endogenous cellular RNA-splicing machinery, Splice Editing enables efficient, precise, and programmable large-scale editing of gene targets without DNA cleavage or mutagenesis. RNA sequencing and measurement of spliced protein products confirm that Splice Editing achieves efficient and specific targeted RNA and protein correction. We show that Splice Editors based on novel miniature RNA-targeting CRISPR-Cas systems discovered and characterized in this work can be packaged for effective delivery to human cells and affect different types of edits across multiple targets and cell lines. By editing thousands of bases simultaneously in a single reversible step, Splice Editing could expand the treatable disease population for monogenic diseases with large allelic diversity without the permanent unintended effects of DNA editing. One-sentence summaryCRISPR-guided trans-splicing enables efficient and specific recombination of large RNA molecules in mammalian cells, with broad applications in therapeutic development for genetic diseases and as a research tool for the study of basic RNA biology.\nMapping the Genetic Interaction Network of PARP inhibitor Response\nAuthors: Simpson, D.; Ling, J.; Jing, Y.; Adamson, B.\nScore: 40.6, Published: 2023-08-20 DOI: 10.1101/2023.08.19.553986\nGenetic interactions have long informed our understanding of the coordinated proteins and pathways that respond to DNA damage in mammalian cells, but systematic interrogation of the genetic network underlying that system has yet to be achieved. Towards this goal, we measured 147,153 pairwise interactions among genes implicated in PARP inhibitor (PARPi) response. Evaluating genetic interactions at this scale, with and without exposure to PARPi, revealed hierarchical organization of the pathways and complexes that maintain genome stability during normal growth and defined changes that occur upon accumulation of DNA lesions due to cytotoxic doses of PARPi. We uncovered unexpected relationships among DNA repair genes, including context-specific buffering interactions between the minimally characterized AUNIP and BRCA1-A complex genes. Our work thus establishes a foundation for mapping differential genetic interactions in mammalian cells and provides a comprehensive resource for future studies of DNA repair and PARP inhibitors.\n",
  "wordCount" : "2336",
  "inLanguage": "en",
  "datePublished": "2023-08-30T10:38:23Z",
  "dateModified": "2023-08-30T10:38:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on August 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554074">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554074" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554074">
        <p class="paperTitle">Reconstructing the genetic relationship between ancient and present-day Siberian populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554074" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554074" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gill, H.; Lee, J.; Jeong, C.</p>
        <p class="info">Score: 22.6, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554074' target='https://doi.org/10.1101/2023.08.21.554074'> 10.1101/2023.08.21.554074</a></p>
        <p class="abstract">Human populations across a vast area in northern Eurasia, from Fennoscandia to Chukotka, share a distinct genetic component often referred to as the Siberian ancestry. Most enriched in present-day Samoyedic-speaking populations such as Nganasans, its origins and history still remain elusive despite the growing list of ancient and present-day genomes from Siberia. Here we reanalyze published ancient and present-day Siberian genomes focusing on the Baikal and Yakutia, resolving key questions regarding their genetic history. First, we show a long-term presence of a unique genetic profile in southern Siberia, up to 6,000 years ago, which distinctly shares a deep ancestral connection with Native Americans. Second, in the Baikal we find no direct contribution of the Early Neolithic Kitoi people to Late Neolithic and Early Bronze Age Serovo-Glazkovo ones. Third, the Middle Neolithic individual from Yakutia, belonging to the Belkachi culture, serves as the best source so far available for the spread of the Siberian ancestry into Fennoscandia and Greenland. These findings shed light on the genetic legacy of the Siberian ancestry and provide insights into the complex interplay between different populations in northern Eurasia throughout history.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554815">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554815" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554815">
        <p class="paperTitle">The genome of the colonial hydroid Hydractinia reveals their stem cells use a toolkit of genes shared with all animals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554815" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554815" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schnitzler, C. E.; Chang, E. S.; Waletich, J.; Wong, W. Y.; Nguyen, A.-D.; Barreira, S.; Doonan, L.; Gonzalez, P.; Koren, S.; Gahan, J.; Quiroga-Artigas, G.; Sanders, S.; Bradshaw, B.; DuBuc, T.; Febrimarsa, F.; de Jong, D.; Nawrocki, E.; Larson, A.; Klasfeld, S.; Gornik, S.; Moreland, R. T.; Wolfsberg, T.; Phillippy, A. M.; Mullikin, J.; Simakov, O.; Cartwright, P.; Nicotra, M.; Frank, U.; Baxevanis, A. D.</p>
        <p class="info">Score: 18.1, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554815' target='https://doi.org/10.1101/2023.08.25.554815'> 10.1101/2023.08.25.554815</a></p>
        <p class="abstract">Hydractinia is a colonial marine hydroid that exhibits remarkable biological properties, including the capacity to regenerate its entire body throughout its lifetime, a process made possible by its adult migratory stem cells, known as i-cells. Here, we provide an in-depth characterization of the genomic structure and gene content of two Hydractinia species, H. symbiolongicarpus and H. echinata, placing them in a comparative evolutionary framework with other cnidarian genomes. We also generated and annotated a single-cell transcriptomic atlas for adult male H. symbiolongicarpus and identified cell type markers for all major cell types, including key i-cell markers. Orthology analyses based on the markers revealed that Hydractinia&#39;s i-cells are highly enriched in genes that are widely shared amongst animals, a striking finding given that Hydractinia has a higher proportion of phylum-specific genes than any of the other 41 animals in our orthology analysis. These results indicate that Hydractinia&#39;s stem cells and early progenitor cells may use a toolkit shared with all animals, making it a promising model organism for future exploration of stem cell biology and regenerative medicine. The genomic and transcriptomic resources for Hydractinia presented here will enable further studies of their regenerative capacity, colonial morphology, and ability to distinguish self from non-self.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554710" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554710">
        <p class="paperTitle">Both phenotypic and genotypic sex influence sex chromosome dosage compensation in a sex reversing lizard</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hanrahan, B. J.; Chang, J. K.; Lister, N. C.; Dissanayake, D. S. B.; Hammond, J. M.; Reis, A. L. M.; Deveson, I. W.; Ruiz-Herrera, A.; Patel, H. R.; Marshall Graves, J. A.; Georges, A.; Waters, P. D.</p>
        <p class="info">Score: 17.3, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554710' target='https://doi.org/10.1101/2023.08.24.554710'> 10.1101/2023.08.24.554710</a></p>
        <p class="abstract">Studies of sex chromosome dosage compensation have historically focussed on therian mammals which have a conserved XY sex determination system. In contrast, lizards have sex determination systems that can differ between even closely related species that include XY and ZW systems and thermolabile systems where genetic and temperature interact to various degrees to determine sex. The eastern three-lined skink (Bassiana duperreyi) has a differentiated XY sex determination system, in which low temperature incubation during development can cause female to male sex reversal, producing XX males. This provides a unique opportunity to investigate how genotype and phenotype affect dosage compensation. We generated transcriptomes from brain and heart tissue of normal adult males and females, along with brain tissue of sex-reversed XX males. We observed partial dosage compensation between XX females and XY males in both brain and heart, with median gene expression from the X in normal males being 0.7 times that of normal females. Surprisingly, in brain of sex reversed XX males the median X chromosome output did not match that of either normal males or females, but instead was 0.89 times that of the normal XX female level. This suggests that not just genotype, but also sexual phenotype, influences gene dosage of the X chromosome. This has profound implications for our understanding of the evolution of dosage compensation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553051">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553051" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553051">
        <p class="paperTitle">A Pooled Cell Painting CRISPR Screening Platform Enables de novo Inference of Gene Function by Self-supervised Deep Learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553051" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553051" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sivanandan, S.; Leitmann, B.; Lubeck, E.; Sultan, M. M.; Stanitsas, P.; Ranu, N.; Ewer, A.; Mancuso, J. E.; Phillips, Z. F.; Kim, A.; Bisognano, J. W.; Cesarek, J.; Ruggiu, F.; Feldman, D.; Koller, D.; Sharon, E.; Kaykas, A.; Salick, M. R.; Chu, C.</p>
        <p class="info">Score: 24.1, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553051' target='https://doi.org/10.1101/2023.08.13.553051'> 10.1101/2023.08.13.553051</a></p>
        <p class="abstract">Pooled CRISPR screening has emerged as a powerful method of mapping gene functions thanks to its scalability, affordability, and robustness against well or plate-specific confounders present in array-based screening. Most pooled CRISPR screens assay for low dimensional phenotypes (e.g. fitness, fluorescent markers). Higher-dimensional assays such as perturb-seq are available but costly and only applicable to transcriptomics readouts. Recently, pooled optical screening, which combines pooled CRISPR screening and microscopy-based assays, has been demonstrated in the studies of the NFkB pathway, essential human genes, cytoskeletal organization and antiviral response. While the pooled optical screening methodology is scalable and information-rich, the applications thus far employ hypothesis-specific assays. Here, we enable hypothesis-free reverse genetic screening for generic morphological phenotypes by re-engineering the Cell Painting technique to provide compatibility with pooled optical screening. We validated this technique using well-defined morphological genesets (124 genes), compared classical image analysis and self-supervised learning methods using a mechanism-of-action (MoA) library (300 genes), and performed discovery screening with a druggable genome library (1640 genes). Across these three experiments we show that the combination of rich morphological data and deep learning allows gene networks to emerge without the need for target-specific biomarkers, leading to better discovery of gene functions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.555023">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.555023" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.555023">
        <p class="paperTitle">A novel, N2-fixing cyanobacterium present and active in the global oceans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.555023" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.555023" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cleveland, C. S.; Turk-Kubo, K. A.; Zhao, Y.; Zehr, J. P.; Webb, E. A.</p>
        <p class="info">Score: 13.8, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.555023' target='https://doi.org/10.1101/2023.08.27.555023'> 10.1101/2023.08.27.555023</a></p>
        <p class="abstract">Marine N2-fixing cyanobacteria, including the unicellular genus Crocosphaera, are considered keystone species in marine food webs. Crocosphaera are globally distributed and provide new sources of nitrogen (N) and carbon (C), which fuel oligotrophic microbial communities and upper trophic levels. Despite their ecosystem importance, only one species, Crocosphaera watsonii, has ever been identified and characterized as widespread in the oligotrophic oceans. Herein, we present a novel species, candidatus Crocosphaera waterburyi (C. waterburyi hereafter), enriched from the North Pacific Ocean, active in situ, and globally distributed in environmental datasets. C. waterburyi is morphologically, phylogenetically, and physiologically distinct from C. watsonii; therefore, description of this novel species provides a new window into previously uncharacterized diversity and ecology of unicellular N2-fixing cyanobacterial taxa and further highlights their importance in the global N cycle.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554146">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554146" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554146">
        <p class="paperTitle">Active learning of enhancer and silencer regulatory grammar in photoreceptors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554146" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554146" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Friedman, R. Z.; Ramu, A.; Lichtarge, S.; Myers, C. A.; Granas, D. M.; Gause, M.; Corbo, J. C.; Cohen, B. A.; White, M. A.</p>
        <p class="info">Score: 21.6, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554146' target='https://doi.org/10.1101/2023.08.21.554146'> 10.1101/2023.08.21.554146</a></p>
        <p class="abstract">Cis-regulatory elements (CREs) direct gene expression in health and disease, and models that can accurately predict their activities from DNA sequences are crucial for biomedicine. Deep learning represents one emerging strategy to model the regulatory grammar that relates CRE sequence to function. However, these models require training data on a scale that exceeds the number of CREs in the genome. We address this problem using active machine learning to iteratively train models on multiple rounds of synthetic DNA sequences assayed in live mammalian retinas. During each round of training the model actively selects sequence perturbations to assay, thereby efficiently generating informative training data. We iteratively trained a model that predicts the activities of sequences containing binding motifs for the photoreceptor transcription factor Cone-rod homeobox (CRX) using an order of magnitude less training data than current approaches. The models internal confidence estimates of its predictions are reliable guides for designing sequences with high activity. The model correctly identified critical sequence differences between active and inactive sequences with nearly identical transcription factor binding sites, and revealed order and spacing preferences for combinations of motifs. Our results establish active learning as an effective method to train accurate deep learning models of cis-regulatory function after exhausting naturally occurring training examples in the genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554210">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554210" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554210">
        <p class="paperTitle">A Spatiotemporal Molecular Atlas of the Ovulating Mouse Ovary</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554210" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554210" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mantri, M.; Zhang, H. H.; Spanos, E.; Ren, Y. A.; De Vlaminck, I.</p>
        <p class="info">Score: 9.4, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554210' target='https://doi.org/10.1101/2023.08.21.554210'> 10.1101/2023.08.21.554210</a></p>
        <p class="abstract">Ovulation is essential for reproductive success, yet the underlying cellular and molecular mechanisms are far from clear. Here, we applied high-resolution spatiotemporal transcriptomics to map out cell-type- and ovulation-stage-specific molecular programs as function of time during follicle maturation and ovulation in mice. Our analysis revealed dynamic molecular transitions within granulosa cell types that occur in tight coordination with mesenchymal cell proliferation. We identified new molecular markers for the emerging cumulus cell fate during the preantral-to-antral transition. We describe transcriptional programs that respond rapidly to ovulation stimulation and those associated with follicle rupture, highlighting the prominent roles of apoptotic and metabolic pathways during the final stages of follicle maturation. We further report stage-specific oocyte-cumulus cell interactions and diverging molecular differentiation in follicles approaching ovulation. Collectively, this study provides insights into the cellular and molecular processes that regulate mouse ovarian follicle maturation and ovulation with important implications for advancing therapeutic strategies in reproductive medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554647">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554647" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554647">
        <p class="paperTitle">The genome of the Wollemi pine, a critically endangered living fossil unchanged since the Cretaceous, reveals extensive ancient transposon activity.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554647" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554647" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stevenson, D. W.; Ramakrishnan, S.; Alves, C. d. S.; Coelho, L. A.; Kramer, M.; Goodwin, S.; Ramos, O. M.; Eshel, G.; Sondervan, V. M.; Frangos, S.; Zumajo-Cardona, C.; Jenike, K.; Ou, S.; Wang, X.; Lee, Y. P.; Loke, S.; Rossetto, M.; McPherson, H.; Nigris, S.; Moschin, S.; Little, D. P.; Katari, M. S.; Varala, K.; Kolokotronis, S.-O.; Ambrose, B.; Croft, L. J.; Coruzzi, G. M.; Schatz, M.; McCombie, W. R.; Martienssen, R. A.</p>
        <p class="info">Score: 8.2, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554647' target='https://doi.org/10.1101/2023.08.24.554647'> 10.1101/2023.08.24.554647</a></p>
        <p class="abstract">We present the genome of the living fossil, Wollemia nobilis, a southern hemisphere conifer morphologically unchanged since the Cretaceous. Presumed extinct until rediscovery in 1994, the Wollemi pine is critically endangered with less than 60 wild adults threatened by intensifying bushfires in the Blue Mountains of Australia. The 12 Gb genome is among the most contiguous large plant genomes assembled, with extremely low heterozygosity and unusual abundance of DNA transposons. Reduced representation and genome re-sequencing of individuals confirms a relictual population since the last major glacial/drying period in Australia, 120 ky BP. Small RNA and methylome sequencing reveal conservation of ancient silencing mechanisms despite the presence of thousands of active and abundant transposons, including some transferred horizontally to conifers from arthropods in the Jurassic. A retrotransposon burst 8-6 my BP coincided with population decline, possibly as an adaptation enhancing epigenetic diversity. Wollemia, like other conifers, is susceptible to Phytophthora, and a suite of defense genes, similar to those in loblolly pine, are targeted for silencing by sRNAs in leaves. The genome provides insight into the earliest seed plants, while enabling conservation efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553620">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553620" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553620">
        <p class="paperTitle">Programmable multi-kilobase RNA editing using CRISPR-mediated trans-splicing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553620" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553620" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Borrajo, J.; Javanmardi, K.; Griffin, J.; St. Martin, S. J.; Yao, D.; Hill, K.; Blainey, P.; Al-Shayeb, B.</p>
        <p class="info">Score: 103.4, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553620' target='https://doi.org/10.1101/2023.08.18.553620'> 10.1101/2023.08.18.553620</a></p>
        <p class="abstract">Current gene editing approaches in eukaryotic cells are limited to single base edits or small DNA insertions and deletions, and remain encumbered by unintended permanent effects and significant challenges in the delivery of large DNA cargo. Here we describe Splice Editing, a generalizable platform to correct gene transcripts in situ by programmable insertion or replacement of large RNA segments. By combining CRISPR-mediated RNA targeting with endogenous cellular RNA-splicing machinery, Splice Editing enables efficient, precise, and programmable large-scale editing of gene targets without DNA cleavage or mutagenesis. RNA sequencing and measurement of spliced protein products confirm that Splice Editing achieves efficient and specific targeted RNA and protein correction. We show that Splice Editors based on novel miniature RNA-targeting CRISPR-Cas systems discovered and characterized in this work can be packaged for effective delivery to human cells and affect different types of edits across multiple targets and cell lines. By editing thousands of bases simultaneously in a single reversible step, Splice Editing could expand the treatable disease population for monogenic diseases with large allelic diversity without the permanent unintended effects of DNA editing.

One-sentence summaryCRISPR-guided trans-splicing enables efficient and specific recombination of large RNA molecules in mammalian cells, with broad applications in therapeutic development for genetic diseases and as a research tool for the study of basic RNA biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.553986">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.553986" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.553986">
        <p class="paperTitle">Mapping the Genetic Interaction Network of PARP inhibitor Response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.553986" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.553986" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Simpson, D.; Ling, J.; Jing, Y.; Adamson, B.</p>
        <p class="info">Score: 40.6, Published: 2023-08-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.553986' target='https://doi.org/10.1101/2023.08.19.553986'> 10.1101/2023.08.19.553986</a></p>
        <p class="abstract">Genetic interactions have long informed our understanding of the coordinated proteins and pathways that respond to DNA damage in mammalian cells, but systematic interrogation of the genetic network underlying that system has yet to be achieved. Towards this goal, we measured 147,153 pairwise interactions among genes implicated in PARP inhibitor (PARPi) response. Evaluating genetic interactions at this scale, with and without exposure to PARPi, revealed hierarchical organization of the pathways and complexes that maintain genome stability during normal growth and defined changes that occur upon accumulation of DNA lesions due to cytotoxic doses of PARPi. We uncovered unexpected relationships among DNA repair genes, including context-specific buffering interactions between the minimally characterized AUNIP and BRCA1-A complex genes. Our work thus establishes a foundation for mapping differential genetic interactions in mammalian cells and provides a comprehensive resource for future studies of DNA repair and PARP inhibitors.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
