<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Epithelial zonation along the mouse and human small intestine defines five discrete metabolic domains
Authors: Zwick, R. K.; Kasparek, P.; Palikuqi, B.; Viragova, S.; Weichselbaum, L.; McGinnis, C. S.; McKinley, K. L.; Rathnayake, A.; Vaka, D.; Nguyen, V.; Trentesaux, C.; Reyes, E.; Gupta, A.; Gartner, Z. J.; Locksley, R. M.; Gardner, J. M.; Itzkovitz, S.; Boffelli, D.; Klein, O. D.
Score: 68.4, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558726
A key aspect of nutrient absorption is the exquisite division of labor across the length of the small intestine, with individual classes of micronutrients taken up at different positions.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Epithelial zonation along the mouse and human small intestine defines five discrete metabolic domains
Authors: Zwick, R. K.; Kasparek, P.; Palikuqi, B.; Viragova, S.; Weichselbaum, L.; McGinnis, C. S.; McKinley, K. L.; Rathnayake, A.; Vaka, D.; Nguyen, V.; Trentesaux, C.; Reyes, E.; Gupta, A.; Gartner, Z. J.; Locksley, R. M.; Gardner, J. M.; Itzkovitz, S.; Boffelli, D.; Klein, O. D.
Score: 68.4, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558726
A key aspect of nutrient absorption is the exquisite division of labor across the length of the small intestine, with individual classes of micronutrients taken up at different positions." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-11T10:39:46+00:00" />
<meta property="article:modified_time" content="2023-10-11T10:39:46+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Epithelial zonation along the mouse and human small intestine defines five discrete metabolic domains
Authors: Zwick, R. K.; Kasparek, P.; Palikuqi, B.; Viragova, S.; Weichselbaum, L.; McGinnis, C. S.; McKinley, K. L.; Rathnayake, A.; Vaka, D.; Nguyen, V.; Trentesaux, C.; Reyes, E.; Gupta, A.; Gartner, Z. J.; Locksley, R. M.; Gardner, J. M.; Itzkovitz, S.; Boffelli, D.; Klein, O. D.
Score: 68.4, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558726
A key aspect of nutrient absorption is the exquisite division of labor across the length of the small intestine, with individual classes of micronutrients taken up at different positions."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Epithelial zonation along the mouse and human small intestine defines five discrete metabolic domains\nAuthors: Zwick, R. K.; Kasparek, P.; Palikuqi, B.; Viragova, S.; Weichselbaum, L.; McGinnis, C. S.; McKinley, K. L.; Rathnayake, A.; Vaka, D.; Nguyen, V.; Trentesaux, C.; Reyes, E.; Gupta, A.; Gartner, Z. J.; Locksley, R. M.; Gardner, J. M.; Itzkovitz, S.; Boffelli, D.; Klein, O. D.\nScore: 68.4, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558726\nA key aspect of nutrient absorption is the exquisite division of labor across the length of the small intestine, with individual classes of micronutrients taken up at different positions.",
  "keywords": [
    
  ],
  "articleBody": " Epithelial zonation along the mouse and human small intestine defines five discrete metabolic domains\nAuthors: Zwick, R. K.; Kasparek, P.; Palikuqi, B.; Viragova, S.; Weichselbaum, L.; McGinnis, C. S.; McKinley, K. L.; Rathnayake, A.; Vaka, D.; Nguyen, V.; Trentesaux, C.; Reyes, E.; Gupta, A.; Gartner, Z. J.; Locksley, R. M.; Gardner, J. M.; Itzkovitz, S.; Boffelli, D.; Klein, O. D.\nScore: 68.4, Published: 2023-09-22 DOI: 10.1101/2023.09.20.558726\nA key aspect of nutrient absorption is the exquisite division of labor across the length of the small intestine, with individual classes of micronutrients taken up at different positions. For millennia, the small intestine was thought to comprise three segments with indefinite borders: the duodenum, jejunum, and ileum. By examining fine-scale longitudinal segmentation of the mouse and human small intestines, we identified transcriptional signatures and upstream regulatory factors that define five domains of nutrient absorption, distinct from the three traditional sections. Spatially restricted expression programs were most prominent in nutrient-absorbing enterocytes but initially arose in intestinal stem cells residing in three regional populations. While a core signature was maintained across mice and humans with different diets and environments, domain properties were influenced by dietary changes. We established the functions of Ppar- and Cdx1 in patterning lipid metabolism in distal domains and generated a predictive model of additional transcription factors that direct domain identity. Molecular domain identity can be detected with machine learning, representing the first systematic method to computationally identify specific intestinal regions in mice. These findings provide a foundational framework for the identity and control of longitudinal zonation of absorption along the proximal:distal small intestinal axis.\nDeep indel mutagenesis reveals the impact of insertions and deletions on protein stability and function\nAuthors: Topolska, M.; Beltran, T.; Lehner, B.\nScore: 31.1, Published: 2023-10-06 DOI: 10.1101/2023.10.06.561180\nAmino acid insertions and deletions (indels) are an abundant class of genetic variants. However, compared to substitutions, the effects of indels are not well understood and poorly predicted. Here we address this shortcoming by performing deep indel mutagenesis (DIM) of structurally diverse proteins. Indel tolerance is strikingly different to substitution tolerance and varies extensively both between different proteins and within different regions of the same protein. Although state of the art variant effect predictors perform poorly on indels, we show that both experimentally-measured and computationally-predicted substitution scores can be repurposed as good indel variant effect predictors by incorporating information on protein secondary structures. Quantifying the effects of indels on protein-protein interactions reveals that insertions can be an important class of gain-of-function variants. Our results provide an overview of the impact of indels on proteins and a method to predict their effects genome-wide.\nPostglacial genomes from foragers across Northern Eurasia reveal prehistoric mobility associated with the spread of the Uralic and Yeniseian languages\nAuthors: Zeng, T. C.; Vyazov, L. A.; Kim, A.; Flegontov, P.; Sirak, K.; Maier, R.; Lazaridis, I.; Akbari, A.; Frachetti, M.; Tishkin, A. A.; Ryabogina, N. E.; Agapov, S. A.; Agapov, D. S.; Alekseev, A. N.; Boeskorov, G. G.; Chizhevsky, A. A.; Derevianko, A. P.; Dyakonov, V. M.; Enshin, D. N.; Fribus, A. V.; Frolov, Y. V.; Grushin, S. P.; Khokhlov, A. A.; Kitov, E. P.; Kosintsev, P.; Kovtun, I. V.; Kiryushin, K. Y.; Kiryushin, Y. F.; Makarov, N. P.; Morozov, V. V.; Nikolaev, E. N.; Rykun, M. P.; Savenkova, T. M.; Shchelchkova, M. V.; Skochina, S. N.; Shirokov, V.; Sherstobitova, O. S.; Slepchenko, S. M.\nScore: 41.0, Published: 2023-10-04 DOI: 10.1101/2023.10.01.560332\nThe North Eurasian forest and forest-steppe zones have sustained millennia of sociocultural connections among northern peoples. We present genome-wide ancient DNA data for 181 individuals from this region spanning the Mesolithic, Neolithic and Bronze Age. We find that Early to Mid-Holocene hunter-gatherer populations from across the southern forest and forest-steppes of Northern Eurasia can be characterized by a continuous gradient of ancestry that remained stable for millennia, ranging from fully West Eurasian in the Baltic region to fully East Asian in the Transbaikal region. In contrast, cotemporaneous groups in far Northeast Siberia were genetically distinct, retaining high levels of continuity from a population that was the primary source of ancestry for Native Americans. By the mid-Holocene, admixture between this early Northeastern Siberian population and groups from Inland East Asia and the Amur River Basin produced two distinctive populations in eastern Siberia that played an important role in the genetic formation of later people. Ancestry from the first population, Cis-Baikal Late Neolithic-Bronze Age (Cisbaikal_LNBA), is found substantially only among Yeniseian-speaking groups and those known to have admixed with them. Ancestry from the second, Yakutian Late Neolithic-Bronze Age (Yakutia_LNBA), is strongly associated with present-day Uralic speakers. We show how Yakutia_LNBA ancestry spread from an east Siberian origin [~]4.5kya, along with subclades of Y-chromosome haplogroup N occurring at high frequencies among present-day Uralic speakers, into Western and Central Siberia in communities associated with Seima-Turbino metallurgy: a suite of advanced bronze casting techniques that spread explosively across an enormous region of Northern Eurasia [~]4.0kya. However, the ancestry of the 16 Seima-Turbino-period individuals--the first reported from sites with this metallurgy--was otherwise extraordinarily diverse, with partial descent from Indo-Iranian-speaking pastoralists and multiple hunter-gatherer populations from widely separated regions of Eurasia. Our results provide support for theories suggesting that early Uralic speakers at the beginning of their westward dispersal where involved in the expansion of Seima-Turbino metallurgical traditions, and suggests that both cultural transmission and migration were important in the spread of Seima-Turbino material culture.\nAncestral genetic components are consistently associated with the complex trait landscape in European biobanks\nAuthors: Pankratov, V.; Mezzavilla, M.; Aneli, S.; Fusco, D.; Wilson, J. F.; Metspalu, M.; Provero, P.; Pagani, L.; Marnetto, D.\nScore: 14.9, Published: 2023-10-05 DOI: 10.1101/2023.10.04.560881\nThe genetic structure in Europe was mostly shaped by admixture between the Western Hunter-Gatherer, Anatolian Neolithic and Steppes Yamnaya ancestral components. Such structure is regarded as a confounder in GWAS and follow-up studies, and gold-standard methods exist to correct for it. However, it is still poorly understood to which extent these ancestral components contribute to complex trait variation in present-day Europe. In this work we harness the UK Biobank to address this question. By extensive demographic simulations and incorporating previous results obtained using the Estonian Biobank, we carefully evaluate the significance and scope of our findings. Heart rate, platelet count, monocyte percentage and many other traits show stratification similar to height and pigmentation traits, likely targets of selection and divergence across ancestral groups. The consistency of our results across biobanks indicates that these ancestry-specific genetic predispositions act as a source of variability and as potential confounders in Europe as a whole.\nInsertion sequence driven evolution of the microbiota revealed by a novel metagenomics pipeline\nAuthors: Kirsch, J. M.; Hryckowian, A. J.; Duerkop, B. A.\nScore: 13.7, Published: 2023-10-09 DOI: 10.1101/2023.10.06.561241\nInsertion sequence (IS) elements are mobile genetic elements in bacterial genomes that support adaptation. We developed a database of IS elements coupled to a computational pipeline that identifies IS element insertions in the microbiota. We discovered that diverse IS elements insert into the genomes of intestinal bacteria regardless of human host lifestyle. These insertions targeted bacterial accessory genes that aid in their adaptation to unique environmental conditions. Using IS element expansion in Bacteroides, we show that IS element activity leads to insertion hot spots in accessory genes. We show that IS element insertions are stable and can be transferred between humans. Extreme environmental perturbations force IS element insertions to fall out of the microbiota and many fail to rebound following homeostasis. Our work shows that IS elements drive bacterial genome diversification within the microbiota and establishes a framework for understanding how strain level variation within the microbiota impacts human health.\nHidden in the sand: phylogenomics unravel an unexpected evolutionary history on the desert-adapted vipers of the genus Cerastes\nAuthors: Mochales-Riano, G.; Burriel-Carranza, B.; Barros, M. I.; Velo-Anton, G.; Talavera, A.; Spilani, L.; Tejero-Cicuendez, H.; Andre-Crochet, P.; Piris, A.; Garcia-Cardenete, L.; Busais, S.; Els, J.; Shobrak, M.; Carlos-Brito, J.; Smid, J.; Carranza, S.; Martinez-Freiria, F.\nScore: 12.8, Published: 2023-10-04 DOI: 10.1101/2023.10.03.560686\nThe desert vipers of the genus Cerastes are a small clade of medically important venomous snakes within the family Viperidae. According to published morphological and molecular studies, the group is comprised by four species: two morphologically similar and phylogenetically sister taxa, the African horned viper (Cerastes cerastes) and the Arabian horned viper (Cerastes gasperettii); a more distantly related species, the Saharan sand viper (Cerastes vipera), and the enigmatic Bohmes sand viper (Cerastes boehmei), only known from a single specimen in captivity allegedly captured in Central Tunisia. In this study, we analyzed one mitochondrial marker (COI) as well as genome-wide data (ddRAD sequencing) from 28 and 41 samples, respectively, covering the entire distribution range of the genus to explore the population genomics, phylogenomic relationships and introgression patterns within the genus Cerastes. Additionally, and to provide insights into the mode of diversification of the group, we carried out niche overlap analyses considering climatic and habitat variables. Both nuclear phylogenomic reconstructions and population structure analyses have unveiled an unexpected evolutionary history for the genus Cerastes, which sharply contradicts the morphological similarities and previously published mitochondrial approaches. Cerastes cerastes and C. vipera are recovered as sister taxa whilst C. gasperettii is a sister taxon to the clade formed by these two species. We found a relatively high niche overlap (OI \u003e 0.7) in both climatic and habitat variables between C. cerastes and C. vipera, contradicting a potential scenario of sympatric speciation. These results are in line with the introgression found between the northwestern African populations of C. cerastes and C. vipera. Finally, our genomic data confirms the existence of a lineage of C. cerastes in Arabia. All these results highlight the importance of genome-wide data over few genetic markers to study the evolutionary history of species.\nAncient genomes illuminate Eastern Arabian population history and adaptation against malaria\nAuthors: Martiniano, R.; Haber, M.; Almarri, M. A.; Mattiangeli, V.; Kuijpers, M. C. M.; Chamel, B.; Breslin, E. M.; Littleton, J.; Almahari, S.; Aloraifi, F.; Bradley, D. G.; Lombard, P.; Durbin, R.\nScore: 119.9, Published: 2023-10-01 DOI: 10.1101/2023.10.01.559299\nThe harsh climate of Arabia has posed challenges in generating ancient DNA from the region, hindering the direct examination of ancient genomes for understanding the demographic processes that shaped Arabian populations. In this study, we report whole genome sequence data obtained from four Tylos-period individuals from Bahrain. Their genetic ancestry can be modelled as a mixture of sources from ancient Anatolia, Levant and Iran/Caucasus, with variation between individuals suggesting population heterogeneity in Bahrain before the onset of Islam. We identify the G6PD Mediterranean mutation associated with malaria-resistance in three out of four ancient Bahraini samples and estimate that it rose in frequency in Eastern Arabia from 5-6 kya onwards, around the time agriculture appeared in the region. Our study provides the first characterisation of the genetic composition of ancient Arabians, shedding light on the population history of Bahrain and demonstrating the feasibility of aDNA studies in the region.\nTransposition of HOPPLA in siRNA-deficient plants suggests a limited effect of the environment on retrotransposon mobility in Brachypodium distachyon\nAuthors: Thieme, M.; Minadakis, N.; Himber, C.; Keller, B.; Xu, W.; rutowicz, K.; Matteoli, C.; Boehrer, M.; Rymen, B.; Laudencia-Chingcuanco, D.; Vogel, J. P.; Sibout, R.; Stritt, C.; Blevins, T.; Roulin, A. C.\nScore: 12.4, Published: 2023-09-27 DOI: 10.1101/2023.09.25.559196\nLong terminal repeat retrotransposons (LTR-RTs) are powerful mutagens regarded as a major source of genetic novelty and important drivers of evolution. Yet, the uncontrolled and potentially selfish proliferation of LTR-RTs can lead to deleterious mutations and genome instability, with large fitness costs for their host. While population genomics data suggest that an ongoing LTR-RT mobility is common in many species, the understanding of their dual roles in evolution is limited. Here, we harness the genetic diversity of 320 sequenced natural accessions of the Mediterranean grass Brachypodium distachyon to characterize how genetic and environmental factors influence plant LTR-RT dynamics in the wild. When combining a coverage-based approach to estimate global LTR-RT copy number variations with mobilome-sequencing of nine accessions exposed to eight different stresses, we find little evidence for a major role of environmental factors in LTR-RT accumulations in B. distachyon natural accessions. Instead, we show that loss of RNA polymerase IV (Pol IV), which mediates RNA-directed DNA methylation in plants, results in high transcriptional and transposition activities of RLC_BdisC024 (HOPPLA) LTR-RT family elements, and that these effects are not stress-specific. This work supports findings indicating an ongoing mobility in B. distachyon and reveals that host RNA-directed DNA methylation rather than environmental factors controls their mobility in this wild grass model.\nThe European Reference Genome Atlas: piloting a decentralised approach to equitable biodiversity genomics\nAuthors: Mc Cartney, A. M.; Formenti, G.; Mouton, A.; De Panis, D.; Marins, L. S.; Leitao, H. G.; Diedericks, G.; Kirangwa, J.; Morselli, M.; Salces, J.; Escudero, N.; Iannucci, A.; Natali, C.; Svardal, H.; Fernandez, R.; De Pooter, T.; Joris, G.; Strazisar, M.; Wood, J.; Herron, K. E.; Seehausen, O.; Watts, P. C.; Shaw, F.; Davey, R. P.; Minotto, A.; Fernandez Gonzalez, J. M.; Bohne, A.; Alegria, C.; Alioto, T.; Alves, P. C.; Amorim, I. R.; Aury, J.-M.; Backstrom, N.; Baldrian, P.; Ballarin, L.; Baltrunaite, L.; Barta, E.; BedHom, B.; Belser, C.; Bergsten, J.; Bertrand, L.; Bilandija, H.; Binzer-Panch\nScore: 31.2, Published: 2023-09-30 DOI: 10.1101/2023.09.25.559365\nA global genome database of all of Earths species diversity could be a treasure trove of scientific discoveries. However, regardless of the major advances in genome sequencing technologies, only a tiny fraction of species have genomic information available. To contribute to a more complete planetary genomic database, scientists and institutions across the world have united under the Earth BioGenome Project (EBP), which plans to sequence and assemble high-quality reference genomes for all [~]1.5 million recognized eukaryotic species through a stepwise phased approach. As the initiative transitions into Phase II, where 150,000 species are to be sequenced in just four years, worldwide participation in the project will be fundamental to success. As the European node of the EBP, the European Reference Genome Atlas (ERGA) seeks to implement a new decentralised, accessible, equitable and inclusive model for producing high-quality reference genomes, which will inform EBP as it scales. To embark on this mission, ERGA launched a Pilot Project to establish a network across Europe to develop and test the first infrastructure of its kind for the coordinated and distributed reference genome production on 98 European eukaryotic species from sample providers across 33 European countries. Here we outline the process and challenges faced during the development of a pilot infrastructure for the production of reference genome resources, and explore the effectiveness of this approach in terms of high-quality reference genome production, considering also equity and inclusion. The outcomes and lessons learned during this pilot provide a solid foundation for ERGA while offering key learnings to other transnational and national genomic resource projects.\nRevisiting genomes of non-model species with long reads yields new insights into their biology and evolution\nAuthors: Guiglielmoni, N.; Villegas, L.; Kirangwa, J.; Schiffer, P.\nScore: 8.8, Published: 2023-10-07 DOI: 10.1101/2023.10.06.561169\nHigh-quality genomes obtained using long-read data allow not only for a better understanding of heterozygosity levels, repeat content, and more accurate gene annotation, and prediction when compared to those obtained with short-read technologies, but also allow to understand haplotype divergence. Advances in long-read sequencing technologies in the last years have made it possible to produce such high-quality assemblies for non-model organisms. This allows us to revisit genomes, which have been problematic to scaffold to chromosome-scale with previous generations of data, and assembly software. Nematoda, one of the most diverse, and speciose animal phyla within metazoans, remains poorly studied, and many previously assembled genomes are fragmented. Using long reads obtained with Nanopore R10.4.1 and PacBio HiFi, we generated highly contiguous assemblies of a diploid nematode of the Mermithidae family, for which no closely related genomes are available to date, as well as a collapsed assembly and a phased assembly for a triploid nematode from the Panagrolaimidae family. Both genomes had been analysed before, but the fragmented assemblies had scaffold sizes comparable to the length of long reads prior to assembly. Our new assemblies illustrate how long-read technologies allow for a much better representation of species genomes. We are now able to conduct more accurate downstream assays based on more complete gene and transposable element predictions.\n",
  "wordCount" : "2702",
  "inLanguage": "en",
  "datePublished": "2023-10-11T10:39:46Z",
  "dateModified": "2023-10-11T10:39:46Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on October 11, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558726">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558726" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558726">
        <p class="paperTitle">Epithelial zonation along the mouse and human small intestine defines five discrete metabolic domains</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558726" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558726" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zwick, R. K.; Kasparek, P.; Palikuqi, B.; Viragova, S.; Weichselbaum, L.; McGinnis, C. S.; McKinley, K. L.; Rathnayake, A.; Vaka, D.; Nguyen, V.; Trentesaux, C.; Reyes, E.; Gupta, A.; Gartner, Z. J.; Locksley, R. M.; Gardner, J. M.; Itzkovitz, S.; Boffelli, D.; Klein, O. D.</p>
        <p class="info">Score: 68.4, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558726' target='https://doi.org/10.1101/2023.09.20.558726'> 10.1101/2023.09.20.558726</a></p>
        <p class="abstract">A key aspect of nutrient absorption is the exquisite division of labor across the length of the small intestine, with individual classes of micronutrients taken up at different positions. For millennia, the small intestine was thought to comprise three segments with indefinite borders: the duodenum, jejunum, and ileum. By examining fine-scale longitudinal segmentation of the mouse and human small intestines, we identified transcriptional signatures and upstream regulatory factors that define five domains of nutrient absorption, distinct from the three traditional sections. Spatially restricted expression programs were most prominent in nutrient-absorbing enterocytes but initially arose in intestinal stem cells residing in three regional populations. While a core signature was maintained across mice and humans with different diets and environments, domain properties were influenced by dietary changes. We established the functions of Ppar- and Cdx1 in patterning lipid metabolism in distal domains and generated a predictive model of additional transcription factors that direct domain identity. Molecular domain identity can be detected with machine learning, representing the first systematic method to computationally identify specific intestinal regions in mice. These findings provide a foundational framework for the identity and control of longitudinal zonation of absorption along the proximal:distal small intestinal axis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561180">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561180" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561180">
        <p class="paperTitle">Deep indel mutagenesis reveals the impact of insertions and deletions on protein stability and function</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561180" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561180" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Topolska, M.; Beltran, T.; Lehner, B.</p>
        <p class="info">Score: 31.1, Published: 2023-10-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561180' target='https://doi.org/10.1101/2023.10.06.561180'> 10.1101/2023.10.06.561180</a></p>
        <p class="abstract">Amino acid insertions and deletions (indels) are an abundant class of genetic variants. However, compared to substitutions, the effects of indels are not well understood and poorly predicted. Here we address this shortcoming by performing deep indel mutagenesis (DIM) of structurally diverse proteins. Indel tolerance is strikingly different to substitution tolerance and varies extensively both between different proteins and within different regions of the same protein. Although state of the art variant effect predictors perform poorly on indels, we show that both experimentally-measured and computationally-predicted substitution scores can be repurposed as good indel variant effect predictors by incorporating information on protein secondary structures. Quantifying the effects of indels on protein-protein interactions reveals that insertions can be an important class of gain-of-function variants. Our results provide an overview of the impact of indels on proteins and a method to predict their effects genome-wide.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.560332">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.560332" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.560332">
        <p class="paperTitle">Postglacial genomes from foragers across Northern Eurasia reveal prehistoric mobility associated with the spread of the Uralic and Yeniseian languages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.560332" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.560332" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zeng, T. C.; Vyazov, L. A.; Kim, A.; Flegontov, P.; Sirak, K.; Maier, R.; Lazaridis, I.; Akbari, A.; Frachetti, M.; Tishkin, A. A.; Ryabogina, N. E.; Agapov, S. A.; Agapov, D. S.; Alekseev, A. N.; Boeskorov, G. G.; Chizhevsky, A. A.; Derevianko, A. P.; Dyakonov, V. M.; Enshin, D. N.; Fribus, A. V.; Frolov, Y. V.; Grushin, S. P.; Khokhlov, A. A.; Kitov, E. P.; Kosintsev, P.; Kovtun, I. V.; Kiryushin, K. Y.; Kiryushin, Y. F.; Makarov, N. P.; Morozov, V. V.; Nikolaev, E. N.; Rykun, M. P.; Savenkova, T. M.; Shchelchkova, M. V.; Skochina, S. N.; Shirokov, V.; Sherstobitova, O. S.; Slepchenko, S. M.</p>
        <p class="info">Score: 41.0, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.560332' target='https://doi.org/10.1101/2023.10.01.560332'> 10.1101/2023.10.01.560332</a></p>
        <p class="abstract">The North Eurasian forest and forest-steppe zones have sustained millennia of sociocultural connections among northern peoples. We present genome-wide ancient DNA data for 181 individuals from this region spanning the Mesolithic, Neolithic and Bronze Age. We find that Early to Mid-Holocene hunter-gatherer populations from across the southern forest and forest-steppes of Northern Eurasia can be characterized by a continuous gradient of ancestry that remained stable for millennia, ranging from fully West Eurasian in the Baltic region to fully East Asian in the Transbaikal region. In contrast, cotemporaneous groups in far Northeast Siberia were genetically distinct, retaining high levels of continuity from a population that was the primary source of ancestry for Native Americans. By the mid-Holocene, admixture between this early Northeastern Siberian population and groups from Inland East Asia and the Amur River Basin produced two distinctive populations in eastern Siberia that played an important role in the genetic formation of later people. Ancestry from the first population, Cis-Baikal Late Neolithic-Bronze Age (Cisbaikal_LNBA), is found substantially only among Yeniseian-speaking groups and those known to have admixed with them.

Ancestry from the second, Yakutian Late Neolithic-Bronze Age (Yakutia_LNBA), is strongly associated with present-day Uralic speakers. We show how Yakutia_LNBA ancestry spread from an east Siberian origin [~]4.5kya, along with subclades of Y-chromosome haplogroup N occurring at high frequencies among present-day Uralic speakers, into Western and Central Siberia in communities associated with Seima-Turbino metallurgy: a suite of advanced bronze casting techniques that spread explosively across an enormous region of Northern Eurasia [~]4.0kya. However, the ancestry of the 16 Seima-Turbino-period individuals--the first reported from sites with this metallurgy--was otherwise extraordinarily diverse, with partial descent from Indo-Iranian-speaking pastoralists and multiple hunter-gatherer populations from widely separated regions of Eurasia. Our results provide support for theories suggesting that early Uralic speakers at the beginning of their westward dispersal where involved in the expansion of Seima-Turbino metallurgical traditions, and suggests that both cultural transmission and migration were important in the spread of Seima-Turbino material culture.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.04.560881">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.04.560881" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.04.560881">
        <p class="paperTitle">Ancestral genetic components are consistently associated with the complex trait landscape in European biobanks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.04.560881" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.04.560881" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pankratov, V.; Mezzavilla, M.; Aneli, S.; Fusco, D.; Wilson, J. F.; Metspalu, M.; Provero, P.; Pagani, L.; Marnetto, D.</p>
        <p class="info">Score: 14.9, Published: 2023-10-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.04.560881' target='https://doi.org/10.1101/2023.10.04.560881'> 10.1101/2023.10.04.560881</a></p>
        <p class="abstract">The genetic structure in Europe was mostly shaped by admixture between the Western Hunter-Gatherer, Anatolian Neolithic and Steppes Yamnaya ancestral components. Such structure is regarded as a confounder in GWAS and follow-up studies, and gold-standard methods exist to correct for it. However, it is still poorly understood to which extent these ancestral components contribute to complex trait variation in present-day Europe.

In this work we harness the UK Biobank to address this question. By extensive demographic simulations and incorporating previous results obtained using the Estonian Biobank, we carefully evaluate the significance and scope of our findings.

Heart rate, platelet count, monocyte percentage and many other traits show stratification similar to height and pigmentation traits, likely targets of selection and divergence across ancestral groups. The consistency of our results across biobanks indicates that these ancestry-specific genetic predispositions act as a source of variability and as potential confounders in Europe as a whole.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561241">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561241" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561241">
        <p class="paperTitle">Insertion sequence driven evolution of the microbiota revealed by a novel metagenomics pipeline</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561241" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561241" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kirsch, J. M.; Hryckowian, A. J.; Duerkop, B. A.</p>
        <p class="info">Score: 13.7, Published: 2023-10-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561241' target='https://doi.org/10.1101/2023.10.06.561241'> 10.1101/2023.10.06.561241</a></p>
        <p class="abstract">Insertion sequence (IS) elements are mobile genetic elements in bacterial genomes that support adaptation. We developed a database of IS elements coupled to a computational pipeline that identifies IS element insertions in the microbiota. We discovered that diverse IS elements insert into the genomes of intestinal bacteria regardless of human host lifestyle. These insertions targeted bacterial accessory genes that aid in their adaptation to unique environmental conditions. Using IS element expansion in Bacteroides, we show that IS element activity leads to insertion hot spots in accessory genes. We show that IS element insertions are stable and can be transferred between humans. Extreme environmental perturbations force IS element insertions to fall out of the microbiota and many fail to rebound following homeostasis. Our work shows that IS elements drive bacterial genome diversification within the microbiota and establishes a framework for understanding how strain level variation within the microbiota impacts human health.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.03.560686">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.03.560686" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.03.560686">
        <p class="paperTitle">Hidden in the sand: phylogenomics unravel an unexpected evolutionary history on the desert-adapted vipers of the genus Cerastes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.03.560686" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.03.560686" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mochales-Riano, G.; Burriel-Carranza, B.; Barros, M. I.; Velo-Anton, G.; Talavera, A.; Spilani, L.; Tejero-Cicuendez, H.; Andre-Crochet, P.; Piris, A.; Garcia-Cardenete, L.; Busais, S.; Els, J.; Shobrak, M.; Carlos-Brito, J.; Smid, J.; Carranza, S.; Martinez-Freiria, F.</p>
        <p class="info">Score: 12.8, Published: 2023-10-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.03.560686' target='https://doi.org/10.1101/2023.10.03.560686'> 10.1101/2023.10.03.560686</a></p>
        <p class="abstract">The desert vipers of the genus Cerastes are a small clade of medically important venomous snakes within the family Viperidae. According to published morphological and molecular studies, the group is comprised by four species: two morphologically similar and phylogenetically sister taxa, the African horned viper (Cerastes cerastes) and the Arabian horned viper (Cerastes gasperettii); a more distantly related species, the Saharan sand viper (Cerastes vipera), and the enigmatic Bohmes sand viper (Cerastes boehmei), only known from a single specimen in captivity allegedly captured in Central Tunisia. In this study, we analyzed one mitochondrial marker (COI) as well as genome-wide data (ddRAD sequencing) from 28 and 41 samples, respectively, covering the entire distribution range of the genus to explore the population genomics, phylogenomic relationships and introgression patterns within the genus Cerastes. Additionally, and to provide insights into the mode of diversification of the group, we carried out niche overlap analyses considering climatic and habitat variables. Both nuclear phylogenomic reconstructions and population structure analyses have unveiled an unexpected evolutionary history for the genus Cerastes, which sharply contradicts the morphological similarities and previously published mitochondrial approaches. Cerastes cerastes and C. vipera are recovered as sister taxa whilst C. gasperettii is a sister taxon to the clade formed by these two species. We found a relatively high niche overlap (OI &gt; 0.7) in both climatic and habitat variables between C. cerastes and C. vipera, contradicting a potential scenario of sympatric speciation. These results are in line with the introgression found between the northwestern African populations of C. cerastes and C. vipera. Finally, our genomic data confirms the existence of a lineage of C. cerastes in Arabia. All these results highlight the importance of genome-wide data over few genetic markers to study the evolutionary history of species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.01.559299">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.01.559299" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.01.559299">
        <p class="paperTitle">Ancient genomes illuminate Eastern Arabian population history and adaptation against malaria</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.01.559299" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.01.559299" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Martiniano, R.; Haber, M.; Almarri, M. A.; Mattiangeli, V.; Kuijpers, M. C. M.; Chamel, B.; Breslin, E. M.; Littleton, J.; Almahari, S.; Aloraifi, F.; Bradley, D. G.; Lombard, P.; Durbin, R.</p>
        <p class="info">Score: 119.9, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.01.559299' target='https://doi.org/10.1101/2023.10.01.559299'> 10.1101/2023.10.01.559299</a></p>
        <p class="abstract">The harsh climate of Arabia has posed challenges in generating ancient DNA from the region, hindering the direct examination of ancient genomes for understanding the demographic processes that shaped Arabian populations. In this study, we report whole genome sequence data obtained from four Tylos-period individuals from Bahrain. Their genetic ancestry can be modelled as a mixture of sources from ancient Anatolia, Levant and Iran/Caucasus, with variation between individuals suggesting population heterogeneity in Bahrain before the onset of Islam. We identify the G6PD Mediterranean mutation associated with malaria-resistance in three out of four ancient Bahraini samples and estimate that it rose in frequency in Eastern Arabia from 5-6 kya onwards, around the time agriculture appeared in the region. Our study provides the first characterisation of the genetic composition of ancient Arabians, shedding light on the population history of Bahrain and demonstrating the feasibility of aDNA studies in the region.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559196">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559196" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559196">
        <p class="paperTitle">Transposition of HOPPLA in siRNA-deficient plants suggests a limited effect of the environment on retrotransposon mobility in Brachypodium distachyon</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559196" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559196" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thieme, M.; Minadakis, N.; Himber, C.; Keller, B.; Xu, W.; rutowicz, K.; Matteoli, C.; Boehrer, M.; Rymen, B.; Laudencia-Chingcuanco, D.; Vogel, J. P.; Sibout, R.; Stritt, C.; Blevins, T.; Roulin, A. C.</p>
        <p class="info">Score: 12.4, Published: 2023-09-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559196' target='https://doi.org/10.1101/2023.09.25.559196'> 10.1101/2023.09.25.559196</a></p>
        <p class="abstract">Long terminal repeat retrotransposons (LTR-RTs) are powerful mutagens regarded as a major source of genetic novelty and important drivers of evolution. Yet, the uncontrolled and potentially selfish proliferation of LTR-RTs can lead to deleterious mutations and genome instability, with large fitness costs for their host. While population genomics data suggest that an ongoing LTR-RT mobility is common in many species, the understanding of their dual roles in evolution is limited. Here, we harness the genetic diversity of 320 sequenced natural accessions of the Mediterranean grass Brachypodium distachyon to characterize how genetic and environmental factors influence plant LTR-RT dynamics in the wild. When combining a coverage-based approach to estimate global LTR-RT copy number variations with mobilome-sequencing of nine accessions exposed to eight different stresses, we find little evidence for a major role of environmental factors in LTR-RT accumulations in B. distachyon natural accessions. Instead, we show that loss of RNA polymerase IV (Pol IV), which mediates RNA-directed DNA methylation in plants, results in high transcriptional and transposition activities of RLC_BdisC024 (HOPPLA) LTR-RT family elements, and that these effects are not stress-specific. This work supports findings indicating an ongoing mobility in B. distachyon and reveals that host RNA-directed DNA methylation rather than environmental factors controls their mobility in this wild grass model.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.25.559365">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.25.559365" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.25.559365">
        <p class="paperTitle">The European Reference Genome Atlas: piloting a decentralised approach to equitable biodiversity genomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.25.559365" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.25.559365" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mc Cartney, A. M.; Formenti, G.; Mouton, A.; De Panis, D.; Marins, L. S.; Leitao, H. G.; Diedericks, G.; Kirangwa, J.; Morselli, M.; Salces, J.; Escudero, N.; Iannucci, A.; Natali, C.; Svardal, H.; Fernandez, R.; De Pooter, T.; Joris, G.; Strazisar, M.; Wood, J.; Herron, K. E.; Seehausen, O.; Watts, P. C.; Shaw, F.; Davey, R. P.; Minotto, A.; Fernandez Gonzalez, J. M.; Bohne, A.; Alegria, C.; Alioto, T.; Alves, P. C.; Amorim, I. R.; Aury, J.-M.; Backstrom, N.; Baldrian, P.; Ballarin, L.; Baltrunaite, L.; Barta, E.; BedHom, B.; Belser, C.; Bergsten, J.; Bertrand, L.; Bilandija, H.; Binzer-Panch</p>
        <p class="info">Score: 31.2, Published: 2023-09-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.25.559365' target='https://doi.org/10.1101/2023.09.25.559365'> 10.1101/2023.09.25.559365</a></p>
        <p class="abstract">A global genome database of all of Earths species diversity could be a treasure trove of scientific discoveries. However, regardless of the major advances in genome sequencing technologies, only a tiny fraction of species have genomic information available. To contribute to a more complete planetary genomic database, scientists and institutions across the world have united under the Earth BioGenome Project (EBP), which plans to sequence and assemble high-quality reference genomes for all [~]1.5 million recognized eukaryotic species through a stepwise phased approach. As the initiative transitions into Phase II, where 150,000 species are to be sequenced in just four years, worldwide participation in the project will be fundamental to success. As the European node of the EBP, the European Reference Genome Atlas (ERGA) seeks to implement a new decentralised, accessible, equitable and inclusive model for producing high-quality reference genomes, which will inform EBP as it scales. To embark on this mission, ERGA launched a Pilot Project to establish a network across Europe to develop and test the first infrastructure of its kind for the coordinated and distributed reference genome production on 98 European eukaryotic species from sample providers across 33 European countries. Here we outline the process and challenges faced during the development of a pilot infrastructure for the production of reference genome resources, and explore the effectiveness of this approach in terms of high-quality reference genome production, considering also equity and inclusion. The outcomes and lessons learned during this pilot provide a solid foundation for ERGA while offering key learnings to other transnational and national genomic resource projects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561169">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561169" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561169">
        <p class="paperTitle">Revisiting genomes of non-model species with long reads yields new insights into their biology and evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561169" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561169" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guiglielmoni, N.; Villegas, L.; Kirangwa, J.; Schiffer, P.</p>
        <p class="info">Score: 8.8, Published: 2023-10-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561169' target='https://doi.org/10.1101/2023.10.06.561169'> 10.1101/2023.10.06.561169</a></p>
        <p class="abstract">High-quality genomes obtained using long-read data allow not only for a better understanding of heterozygosity levels, repeat content, and more accurate gene annotation, and prediction when compared to those obtained with short-read technologies, but also allow to understand haplotype divergence. Advances in long-read sequencing technologies in the last years have made it possible to produce such high-quality assemblies for non-model organisms. This allows us to revisit genomes, which have been problematic to scaffold to chromosome-scale with previous generations of data, and assembly software. Nematoda, one of the most diverse, and speciose animal phyla within metazoans, remains poorly studied, and many previously assembled genomes are fragmented. Using long reads obtained with Nanopore R10.4.1 and PacBio HiFi, we generated highly contiguous assemblies of a diploid nematode of the Mermithidae family, for which no closely related genomes are available to date, as well as a collapsed assembly and a phased assembly for a triploid nematode from the Panagrolaimidae family. Both genomes had been analysed before, but the fragmented assemblies had scaffold sizes comparable to the length of long reads prior to assembly. Our new assemblies illustrate how long-read technologies allow for a much better representation of species genomes. We are now able to conduct more accurate downstream assays based on more complete gene and transposable element predictions.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
