<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Unifying approaches from statistical genetics and phylogenetics for mapping phenotypes in structured populations
Authors: Schraiber, J. G.; Edge, M. D.; Pennell, M.
Score: 32.9, Published: 2024-02-12 DOI: 10.1101/2024.02.10.579721
In both statistical genetics and phylogenetics, a major goal is to identify correlations between genetic loci or other aspects of the phenotype or environment and a focal trait. In these two fields, there are sophisticated but disparate statistical traditions aimed at these tasks.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Unifying approaches from statistical genetics and phylogenetics for mapping phenotypes in structured populations
Authors: Schraiber, J. G.; Edge, M. D.; Pennell, M.
Score: 32.9, Published: 2024-02-12 DOI: 10.1101/2024.02.10.579721
In both statistical genetics and phylogenetics, a major goal is to identify correlations between genetic loci or other aspects of the phenotype or environment and a focal trait. In these two fields, there are sophisticated but disparate statistical traditions aimed at these tasks." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-14T10:38:48+00:00" />
<meta property="article:modified_time" content="2024-02-14T10:38:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Unifying approaches from statistical genetics and phylogenetics for mapping phenotypes in structured populations
Authors: Schraiber, J. G.; Edge, M. D.; Pennell, M.
Score: 32.9, Published: 2024-02-12 DOI: 10.1101/2024.02.10.579721
In both statistical genetics and phylogenetics, a major goal is to identify correlations between genetic loci or other aspects of the phenotype or environment and a focal trait. In these two fields, there are sophisticated but disparate statistical traditions aimed at these tasks."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Unifying approaches from statistical genetics and phylogenetics for mapping phenotypes in structured populations\nAuthors: Schraiber, J. G.; Edge, M. D.; Pennell, M.\nScore: 32.9, Published: 2024-02-12 DOI: 10.1101/2024.02.10.579721\nIn both statistical genetics and phylogenetics, a major goal is to identify correlations between genetic loci or other aspects of the phenotype or environment and a focal trait. In these two fields, there are sophisticated but disparate statistical traditions aimed at these tasks.",
  "keywords": [
    
  ],
  "articleBody": " Unifying approaches from statistical genetics and phylogenetics for mapping phenotypes in structured populations\nAuthors: Schraiber, J. G.; Edge, M. D.; Pennell, M.\nScore: 32.9, Published: 2024-02-12 DOI: 10.1101/2024.02.10.579721\nIn both statistical genetics and phylogenetics, a major goal is to identify correlations between genetic loci or other aspects of the phenotype or environment and a focal trait. In these two fields, there are sophisticated but disparate statistical traditions aimed at these tasks. The disconnect between their respective approaches is becoming untenable as questions in medicine, conservation biology, and evolutionary biology increasingly rely on integrating data from within and among species and once clear conceptual divisions are becoming increasingly blurred. To help bridge this divide, we derive a general model describing the covariance between the genetic contributions to the quantitative phenotypes of different individuals. Taking this approach shows that standard models in both statistical genetics (e.g., Genome-Wide Association Studies; GWAS) and phylogenetic comparative biology (e.g., phylogenetic regression) can be interpreted as special cases of this more general quantitative-genetic model. The fact that these models share the same core architecture means that we can build a unified understanding of the strengths and limitations of different methods for controlling for genetic structure when testing for associations. We develop intuition for why and when spurious correlations may occur using analytical theory and conduct population genetic and phylogenetic simulations of quantitative traits to provide to a detailed analysis of the general regression model, in which both standard GWAS methods and phylogenetic regression are subsumed. This enables us to take methodological advances from one field and apply them in the other. We demonstrate this by showing how a standard GWAS technique---including both the genetic relatedness matrix (GRM) as well as its leading eigenvectors, corresponding the principal components (PCs) of the genotype matrix, in a regression model---can mitigate spurious correlations in phylogenetic analyses. As a case study of this, we re-examine an analysis testing for co-evolution of expression levels between genes across a fungal phylogeny, and show that including principal components as covariates decreases the false positive rate while simultaneously increasing the true positive rate. More generally, this work provides a foundation for more integrative approaches for understanding the genetic architecture of phenotypes and how evolutionary processes shape it.\nNatural selection exerted by historical coronavirus epidemic(s): comparative genetic analysis in China Kadoorie Biobank and UK Biobank\nAuthors: Morris, S. C.; Lin, K.; Millwood, I. Y.; Yu, C.; Lv, J.; Pei, P.; Li, L.; Sun, D.; Davey Smith, G.; Chen, Z.; Walters, R. G.\nScore: 25.0, Published: 2024-02-06 DOI: 10.1101/2024.02.06.579075\nBackgroundPathogens have been one of the primary sources of natural selection affecting modern humans. The footprints of historical selection events - \"selective sweeps\" - can be detected in the genomes of present-day individuals. Previous analyses of 629 samples from the 1000 Genomes Project suggested that an ancient coronavirus epidemic [~]20,000 years ago drove multiple selective sweeps in the ancestors of present-day East Asians, but not in other worldwide populations. ResultsUsing a much larger genetic dataset of 76,719 unrelated individuals from each of the China Kadoorie Biobank (CKB) and UK Biobank (UKB) to identify regions of long-range linkage disequilibrium, we further investigated signatures of past selective sweeps and how they reflect previous viral epidemics. Using independently-curated lists of human host proteins which interact physically or functionally with viruses (virus-interacting proteins; VIPs), we found enrichment in CKB for regions of long-range linkage disequilibrium at genes encoding VIPs for coronaviruses, but not DNA viruses. By contrast, we found no clear evidence for any VIP enrichment in UKB. These findings were supported by additional analyses using saltiLASSi, a selection-scan method robust to false positives caused by demographic events. By contrast, for GWAS signals for SARS-Cov2 susceptibility (critical illness, hospitalisation, and reported infection), there was no difference between UKB and CKB in the number located at or near signals of selection, as expected for a novel virus which has had no opportunity to impact the CKB/UKB study populations. ConclusionsTogether, these results provide evidence of selection events consistent with historical coronavirus epidemic(s) originating in East Asia. These results show how biobank-scale datasets and evolutionary genomics theory can provide insight into the study of past epidemics. The results also highlights how historic infectious diseases epidemics can shape the genetic architecture of present-day human populations.\nNanopore-based DNA long-read sequencing analysis of the aged human brain\nAuthors: Ramirez, P.; Sun, W.; Kazempour Dehkordi, S.; Zare, H.; Fongang, B.; Bieniek, K. F.; Frost, B.\nScore: 22.9, Published: 2024-02-05 DOI: 10.1101/2024.02.01.578450\nAging disrupts cellular processes such as DNA repair and epigenetic control, leading to a gradual buildup of genomic alterations that can have detrimental effects in post-mitotic cells. Genomic alterations in regions of the genome that are rich in repetitive sequences, often termed \"dark loci,\" are difficult to resolve using traditional sequencing approaches. New long-read technologies offer promising avenues for exploration of previously inaccessible regions of the genome. Using nanopore-based long-read whole-genome sequencing of DNA extracted from aged 18 human brains, we identify previously unreported structural variants and methylation patterns within repetitive DNA, focusing on transposable elements (\"jumping genes\") as crucial sources of variation, particularly in dark loci. Our analyses reveal potential somatic insertion variants and provides DNA methylation frequencies for many retrotransposon families. We further demonstrate the utility of this technology for the study of these challenging genomic regions in brains affected by Alzheimers disease and identify significant differences in DNA methylation in pathologically normal brains versus those affected by Alzheimers disease. Highlighting the power of this approach, we discover specific polymorphic retrotransposons with altered DNA methylation patterns. These retrotransposon loci have the potential to contribute to pathology, warranting further investigation in Alzheimers disease research. Taken together, our study provides the first long-read DNA sequencing-based analysis of retrotransposon sequences, structural variants, and DNA methylation in the aging brain affected with Alzheimers disease neuropathology.\nA complex Plasmodium falciparum cryptotype circulating at low frequency across the African continent\nAuthors: Miotto, O.; Amambua-Ngwa, A.; Amenga-Etego, L. N.; Abdel Hamid, M. M.; Adam, I.; Aninagyei, E.; Apinjoh, T.; Awandare, G. A.; Bejon, P.; Bertin, G.; Bouyou-Akotet, M. K.; Claessens, A.; Conway, D.; D'Alessandro, U.; Diakite, M.; Djimde, A.; Dondorp, A. M.; Duffy, P.; Fairhurst, R. M.; Fanello, C. I.; Ghansah, A.; Ishengoma, D.; Lawniczak, M.; Maiga-Ascofare, O.; Auburn, S.; Rosanas-Urgell, A.; Wasakul, V.; White, N.; Almagro-Garcia, J.; Pearson, R.; Goncalves, S.; Ariani, C.; Bozdech, Z.; Hamilton, W. L.; Simpson, V.; Kwiatkowski, D. P.\nScore: 30.5, Published: 2024-01-22 DOI: 10.1101/2024.01.20.576496\nThe population structure of the malaria parasite Plasmodium falciparum can reveal underlying demographic and adaptive evolutionary processes. Here, we analyse population structure in 4,376 P. falciparum genomes from 21 countries across Africa. We identified a strongly differentiated cluster of parasites, comprising [~]1.2% of samples analysed, geographically distributed over 13 countries across the continent. Members of this cluster, named AF1, carry a genetic background consisting of a large number of highly differentiated variants, rarely observed outside this cluster, at a multitude of genomic loci distributed across most chromosomes. At these loci, the AF1 haplotypes appear to have common ancestry, irrespective of the sampling location; outside the shared loci, however, AF1 members are genetically similar to their sympatric parasites. AF1 parasites sharing up to 23 genomic co-inherited regions were found in all major regions of Africa, at locations over 7,000 km apart. We coined the term cryptotype to describe a complex common background which is geographically widespread, but concealed by genomic regions of local origin. Most AF1 differentiated variants are functionally related, comprising structural variations and single nucleotide polymorphisms in components of the MSP1 complex and several other genes involved in interactions with red blood cells, including invasion and erythrocyte antigen export. We propose that AF1 parasites have adapted to some as yet unidentified evolutionary niche, by acquiring a complex compendium of interacting variants that rarely circulate separately in Africa. As the cryptotype spread across the continent, it appears to have been maintained mostly intact in spite of recombination events, suggesting a selective advantage. It is possible that other cryptotypes circulate in Africa, and new analysis methods may be needed to identify them.\nHiCrayon reveals distinct layers of multi-state 3D chromatin organization\nAuthors: Nolan, B.; Harris, H. L.; Kalluchi, A.; Reznicek, T. E.; Cummings, C. T.; Rowley, M. J.\nScore: 15.3, Published: 2024-02-12 DOI: 10.1101/2024.02.11.579821\nThe co-visualization of chromatin conformation with 1D 'omics data is key to the multi-omics driven data analysis of 3D genome organization. Chromatin contact maps are often shown as 2D heatmaps and visually compared to 1D genomic data by simple juxtaposition. While common, this strategy is imprecise, placing the onus on the reader to align features with each other. To remedy this, we developed HiCrayon, an interactive tool that facilitates the integration of 3D chromatin organization maps and 1D datasets. This visualization method integrates data from genomic assays directly into the chromatin contact map by coloring interactions according to 1D signal. HiCrayon is implemented using R shiny and python to create a graphical user interface (GUI) application, available in both web or containerized format to promote accessibility. HiCrayon is implemented in R, and includes a graphical user interface (GUI), as well as a slimmed-down web-based version that lets users quickly produce publication-ready images. We demonstrate the utility of HiCrayon in visualizing the effectiveness of compartment calling and the relationship between ChIP-seq and various features of chromatin organization. We also demonstrate the improved visualization of other 3D genomic phenomena, such as differences between loops associated with CTCF/cohesin vs. those associated with H3K27ac. We then demonstrate HiCrayon's visualization of organizational changes that occur during differentiation and use HiCrayon to detect compartment patterns that cannot be assigned to either A or B compartments, revealing a distinct 3rd chromatin compartment. Overall, we demonstrate the utility of co-visualizing 2D chromatin conformation with 1D genomic signals within the same matrix to reveal fundamental aspects of genome organization.\nDifferential Conservation and Loss of CR1 Retrotransposons in Squamates Reveals Lineage-Specific Genome Dynamics across Reptiles\nAuthors: Gable, S. M.; Bushroe, N. A.; Mendez, J. M.; Wilson, A. A.; Pinto, B. J.; Gamble, T.; Tollis, M.\nScore: 11.1, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579686\nTransposable elements (TEs) are repetitive DNA sequences which create mutations and generate genetic diversity across the tree of life. In amniotic vertebrates, TEs have been mainly studied in mammals and birds, whose genomes generally display low TE diversity. Squamates (Order Squamata; ~11,000 extant species of lizards and snakes) show as much variation in TE abundance and activity as they do in species and phenotypes. Despite this high TE activity, squamate genomes are remarkably uniform in size. We hypothesize that novel, lineage-specific dynamics have evolved over the course of squamate evolution to constrain genome size across the order. Thus, squamates may represent a prime model for investigations into TE diversity and evolution. To understand the interplay between TEs and host genomes, we analyzed the evolutionary history of the CR1 retrotransposon, a TE family found in most tetrapod genomes. We compared 113 squamate genomes to the genomes of turtles, crocodilians, and birds, and used ancestral state reconstruction to identify shifts in the rate of CR1 copy number evolution across reptiles. We analyzed the repeat landscapes of CR1 in squamate genomes and determined that shifts in the rate of CR1 copy number evolution are associated with lineage-specific variation in CR1 activity. We then used phylogenetic reconstruction of CR1 subfamilies across amniotes to reveal both recent and ancient CR1 subclades across the squamate tree of life. The patterns of CR1 evolution in squamates contrast other amniotes, suggesting key differences in how TEs interact with different host genomes and at different points across evolutionary history.\nNAVIP: Unraveling the Influence of Neighboring Small Sequence Variants on Functional Impact Prediction\nAuthors: Baasner, J.-S.; Rempel, A.; Howard, D.; Pucker, B.\nScore: 52.0, Published: 2024-02-07 DOI: 10.1101/596718\nOnce a suitable reference sequence has been generated, intraspecific variation is often assessed by re-sequencing. Variant calling processes can reveal all differences between strains, accessions, genotypes, or individuals. These variants can be enriched with predictions about their functional implications based on available structural annotations, i.e. gene models. Although these functional impact predictions on a per-variant basis are often accurate, some challenging cases require the simultaneous incorporation of multiple adjacent variants into this prediction process. Examples include neighboring variants which modify each others functional impact. The Neighborhood-Aware Variant Impact Predictor (NAVIP) considers all variants within a given protein coding sequence when predicting the functional consequences. As a proof of concept, variants between the Arabidopsis thaliana accessions Columbia-0 and Niederzenz-1 were annotated. NAVIP is freely available on GitHub (https://github.com/bpucker/NAVIP) and accessible through a web server (https://pbb-tools.de). Author SummaryIntraspecific variation gains increasing relevance as reference genome sequences are available for many investigated (plant) species. Understanding the functional consequences of sequence variants between individuals of a population is a challenge. SnpEff (Cigolnai et al., 2012) is the current standard tool to predict the functional impact of sequence variants, but does only consider one sequence variant at the time. We developed NAVIP to properly handle cases in which multiple sequence variants are clustering together and influence each others functional impact. A comparison of two Arabidopsis thaliana accessions demonstrates the relevance of considering multiple sequence variants simultaneously for the prediction of changes in encoded proteins. NAVIP is universally applicable to non-model organisms. All underlying code is freely available from GitHub and we operate a web server for users convenience.\nQuantitative omnigenic model discovers interpretable genome-wide associations\nAuthors: Ruzickova, N.; Hledik, M.; Tkacik, G.\nScore: 11.2, Published: 2024-02-01 DOI: 10.1101/2024.02.01.578486\nAs their statistical power grows, genome-wide association studies (GWAS) have identified an increasing number of loci underlying quantitative traits of interest. These loci are scattered throughout the genome and are individually responsible only for small fractions of the total heritable trait variance. The recently proposed omnigenic model provides a conceptual framework to explain these observations by postulating that numerous distant loci contribute to each complex trait via effect propagation through intracellular regulatory networks. We formalize this conceptual framework by proposing the \"quantitative omnigenic model\" (QOM), a statistical model that combines prior knowledge of the regulatory network topology with genomic data. By applying our model to gene expression traits in yeast, we demonstrate that QOM achieves similar gene expression prediction performance to traditional GWAS with hundreds of times less parameters, while simultaneously extracting candidate causal and quantitative chains of effect propagation through the regulatory network for every individual gene. We estimate the fraction of heritable trait variance in cis- and in trans-, break the latter down by effect propagation order, assess the trans- variance not attributable to transcriptional regulation, and show that QOM correctly accounts for the low-dimensional structure of gene expression covariance. We furthermore demonstrate the relevance of QOM for systems biology, by employing it as a statistical test for the quality of regulatory network reconstructions, and linking it to the propagation of non-genetic, environmental effects. Significance statementGenetic variation leads to differences in traits implicated in health and disease. Identifying genetic variants associated with these traits is spearheaded by \"genome-wide association studies\" (GWAS) - statistically rigorous procedures whose power has grown with the number of genotyped samples. Nevertheless, GWAS have a substantial shortcoming: they are ill-equipped to detect the causal basis and reveal the complex systemic mechanisms of polygenic traits. Even a single genetic change can propagate throughout the entire genetic regulatory network causing a myriad of spurious detections, thereby significantly limiting GWAS usefulness. We tackle this challenge with a novel statistical approach that incorporates known regulatory network information to substantially boost the interpretability of state-of-the-art genomic analyses while simultaneously extracting novel systems biology insights.\nGlobal genomic epidemiology of blaGES-5 carbapenemase-associated integrons\nAuthors: Matlock, W.; Shaw, L. P.; Stoesser, N.\nScore: 8.6, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579496\nAntimicrobial resistance (AMR) gene cassettes comprise an AMR gene flanked by short recombination sites (attI x attC or attC x attC). Integrons are genetic elements able to capture, excise, and shuffle these cassettes, providing adaptation on demand, and can be found on both chromosomes and plasmids. Understanding the patterns of integron diversity may help to understand the epidemiology of AMR genes. As a case study, we examined the clinical resistance gene blaGES-5, an integron-associated class A carbapenemase first reported in Greece in 2004 and since observed worldwide, which to our knowledge has not been the subject of a previous global analysis. Using a dataset comprising all NCBI contigs containing blaGES-5 (n = 431), we developed a pangenome graph-based workflow to characterise and cluster the diversity of blaGES-5 -associated integrons. We demonstrate that blaGES-5-associated integrons on plasmids are different to those on chromosomes. Chromosomal integrons were almost all identified in P. aeruginosa ST235, with a consistent gene cassette content and order. We observed instances where insertion sequence IS110 disrupted attC sites, which might immobilise the gene cassettes and explain the conserved integron structure despite the presence of intI1 integrase promoters, which would typically facilitate capture or excision and rearrangement. The plasmid-associated integrons were more diverse in their gene cassette content and order, which could be an indication of greater integrase activity and shuffling of integrons on plasmids.\nGraph-based Nanopore Adaptive Sampling with GNASTy enables sensitive pneumococcal serotyping in complex samples\nAuthors: Horsfield, S. T.; Fok, B.; Fu, Y.; Turner, P.; Lees, J. A.; Croucher, N. J.\nScore: 8.4, Published: 2024-02-12 DOI: 10.1101/2024.02.11.579857\nSerotype surveillance of Streptococcus pneumoniae (the pneumococcus) is critical for understanding the effectiveness of current vaccination strategies. However, existing methods for serotyping are limited in their ability to identify co-carriage of multiple pneumococci and detect novel serotypes. To develop a scalable and portable serotyping method that overcomes these challenges, we employed Nanopore Adaptive Sampling (NAS), an on-sequencer enrichment method which selects for target DNA in real-time, for direct detection of S. pneumoniae in complex samples. Whereas NAS targeting the whole S. pneumoniae genome was ineffective in the presence of non-pathogenic streptococci, the method was both specific and sensitive when targeting the capsular biosynthetic locus (CBL), the operon that determines S. pneumoniae serotype. NAS significantly improved coverage and yield of the CBL relative to sequencing without NAS, and accurately quantified the relative prevalence of serotypes in samples representing co-carriage. To maximise the sensitivity of NAS to detect novel serotypes, we developed and benchmarked a new pangenome-graph algorithm, named GNASTy. We show that GNASTy outperforms the current NAS implementation, which is based on linear genome alignment, when a sample contains a serotype absent from the database of targeted sequences. The methods developed in this work provide an improved approach for novel serotype discovery and routine S. pneumoniae surveillance that is fast, accurate and feasible in low resource settings. GNASTy therefore has the potential to increase the density and coverage of global pneumococcal surveillance.\n",
  "wordCount" : "3089",
  "inLanguage": "en",
  "datePublished": "2024-02-14T10:38:48Z",
  "dateModified": "2024-02-14T10:38:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.10.579721">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.10.579721" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.10.579721">
        <p class="paperTitle">Unifying approaches from statistical genetics and phylogenetics for mapping phenotypes in structured populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.10.579721" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.10.579721" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schraiber, J. G.; Edge, M. D.; Pennell, M.</p>
        <p class="info">Score: 32.9, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.10.579721' target='https://doi.org/10.1101/2024.02.10.579721'> 10.1101/2024.02.10.579721</a></p>
        <p class="abstract">In both statistical genetics and phylogenetics, a major goal is to identify correlations between genetic loci or other aspects of the phenotype or environment and a focal trait. In these two fields, there are sophisticated but disparate statistical traditions aimed at these tasks. The disconnect between their respective approaches is becoming untenable as questions in medicine, conservation biology, and evolutionary biology increasingly rely on integrating data from within and among species and once clear conceptual divisions are becoming increasingly blurred. To help bridge this divide, we derive a general model describing the covariance between the genetic contributions to the quantitative phenotypes of different individuals. Taking this approach shows that standard models in both statistical genetics (e.g., Genome-Wide Association Studies; GWAS) and phylogenetic comparative biology (e.g., phylogenetic regression) can be interpreted as special cases of this more general quantitative-genetic model. The fact that these models share the same core architecture means that we can build a unified understanding of the strengths and limitations of different methods for controlling for genetic structure when testing for associations. We develop intuition for why and when spurious correlations may occur using analytical theory and conduct population genetic and phylogenetic simulations of quantitative traits to provide to a detailed analysis of the general regression model, in which both standard GWAS methods and phylogenetic regression are subsumed. This enables us to take methodological advances from one field and apply them in the other. We demonstrate this by showing how a standard GWAS technique---including both the genetic relatedness matrix (GRM) as well as its leading eigenvectors, corresponding the principal components (PCs) of the genotype matrix, in a regression model---can mitigate spurious correlations in phylogenetic analyses. As a case study of this, we re-examine an analysis testing for co-evolution of expression levels between genes across a fungal phylogeny, and show that including principal components as covariates decreases the false positive rate while simultaneously increasing the true positive rate. More generally, this work provides a foundation for more integrative approaches for understanding the genetic architecture of phenotypes and how evolutionary processes shape it.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579075">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579075" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579075">
        <p class="paperTitle">Natural selection exerted by historical coronavirus epidemic(s): comparative genetic analysis in China Kadoorie Biobank and UK Biobank</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579075" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579075" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Morris, S. C.; Lin, K.; Millwood, I. Y.; Yu, C.; Lv, J.; Pei, P.; Li, L.; Sun, D.; Davey Smith, G.; Chen, Z.; Walters, R. G.</p>
        <p class="info">Score: 25.0, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579075' target='https://doi.org/10.1101/2024.02.06.579075'> 10.1101/2024.02.06.579075</a></p>
        <p class="abstract">BackgroundPathogens have been one of the primary sources of natural selection affecting modern humans. The footprints of historical selection events - &#34;selective sweeps&#34; - can be detected in the genomes of present-day individuals. Previous analyses of 629 samples from the 1000 Genomes Project suggested that an ancient coronavirus epidemic [~]20,000 years ago drove multiple selective sweeps in the ancestors of present-day East Asians, but not in other worldwide populations.

ResultsUsing a much larger genetic dataset of 76,719 unrelated individuals from each of the China Kadoorie Biobank (CKB) and UK Biobank (UKB) to identify regions of long-range linkage disequilibrium, we further investigated signatures of past selective sweeps and how they reflect previous viral epidemics. Using independently-curated lists of human host proteins which interact physically or functionally with viruses (virus-interacting proteins; VIPs), we found enrichment in CKB for regions of long-range linkage disequilibrium at genes encoding VIPs for coronaviruses, but not DNA viruses. By contrast, we found no clear evidence for any VIP enrichment in UKB. These findings were supported by additional analyses using saltiLASSi, a selection-scan method robust to false positives caused by demographic events. By contrast, for GWAS signals for SARS-Cov2 susceptibility (critical illness, hospitalisation, and reported infection), there was no difference between UKB and CKB in the number located at or near signals of selection, as expected for a novel virus which has had no opportunity to impact the CKB/UKB study populations.

ConclusionsTogether, these results provide evidence of selection events consistent with historical coronavirus epidemic(s) originating in East Asia. These results show how biobank-scale datasets and evolutionary genomics theory can provide insight into the study of past epidemics. The results also highlights how historic infectious diseases epidemics can shape the genetic architecture of present-day human populations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578450">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578450" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578450">
        <p class="paperTitle">Nanopore-based DNA long-read sequencing analysis of the aged human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578450" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578450" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramirez, P.; Sun, W.; Kazempour Dehkordi, S.; Zare, H.; Fongang, B.; Bieniek, K. F.; Frost, B.</p>
        <p class="info">Score: 22.9, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578450' target='https://doi.org/10.1101/2024.02.01.578450'> 10.1101/2024.02.01.578450</a></p>
        <p class="abstract">Aging disrupts cellular processes such as DNA repair and epigenetic control, leading to a gradual buildup of genomic alterations that can have detrimental effects in post-mitotic cells. Genomic alterations in regions of the genome that are rich in repetitive sequences, often termed &#34;dark loci,&#34; are difficult to resolve using traditional sequencing approaches. New long-read technologies offer promising avenues for exploration of previously inaccessible regions of the genome. Using nanopore-based long-read whole-genome sequencing of DNA extracted from aged 18 human brains, we identify previously unreported structural variants and methylation patterns within repetitive DNA, focusing on transposable elements (&#34;jumping genes&#34;) as crucial sources of variation, particularly in dark loci. Our analyses reveal potential somatic insertion variants and provides DNA methylation frequencies for many retrotransposon families. We further demonstrate the utility of this technology for the study of these challenging genomic regions in brains affected by Alzheimers disease and identify significant differences in DNA methylation in pathologically normal brains versus those affected by Alzheimers disease. Highlighting the power of this approach, we discover specific polymorphic retrotransposons with altered DNA methylation patterns. These retrotransposon loci have the potential to contribute to pathology, warranting further investigation in Alzheimers disease research. Taken together, our study provides the first long-read DNA sequencing-based analysis of retrotransposon sequences, structural variants, and DNA methylation in the aging brain affected with Alzheimers disease neuropathology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576496" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576496">
        <p class="paperTitle">A complex Plasmodium falciparum cryptotype circulating at low frequency across the African continent</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miotto, O.; Amambua-Ngwa, A.; Amenga-Etego, L. N.; Abdel Hamid, M. M.; Adam, I.; Aninagyei, E.; Apinjoh, T.; Awandare, G. A.; Bejon, P.; Bertin, G.; Bouyou-Akotet, M. K.; Claessens, A.; Conway, D.; D&#39;Alessandro, U.; Diakite, M.; Djimde, A.; Dondorp, A. M.; Duffy, P.; Fairhurst, R. M.; Fanello, C. I.; Ghansah, A.; Ishengoma, D.; Lawniczak, M.; Maiga-Ascofare, O.; Auburn, S.; Rosanas-Urgell, A.; Wasakul, V.; White, N.; Almagro-Garcia, J.; Pearson, R.; Goncalves, S.; Ariani, C.; Bozdech, Z.; Hamilton, W. L.; Simpson, V.; Kwiatkowski, D. P.</p>
        <p class="info">Score: 30.5, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576496' target='https://doi.org/10.1101/2024.01.20.576496'> 10.1101/2024.01.20.576496</a></p>
        <p class="abstract">The population structure of the malaria parasite Plasmodium falciparum can reveal underlying demographic and adaptive evolutionary processes. Here, we analyse population structure in 4,376 P. falciparum genomes from 21 countries across Africa. We identified a strongly differentiated cluster of parasites, comprising [~]1.2% of samples analysed, geographically distributed over 13 countries across the continent. Members of this cluster, named AF1, carry a genetic background consisting of a large number of highly differentiated variants, rarely observed outside this cluster, at a multitude of genomic loci distributed across most chromosomes. At these loci, the AF1 haplotypes appear to have common ancestry, irrespective of the sampling location; outside the shared loci, however, AF1 members are genetically similar to their sympatric parasites. AF1 parasites sharing up to 23 genomic co-inherited regions were found in all major regions of Africa, at locations over 7,000 km apart. We coined the term cryptotype to describe a complex common background which is geographically widespread, but concealed by genomic regions of local origin. Most AF1 differentiated variants are functionally related, comprising structural variations and single nucleotide polymorphisms in components of the MSP1 complex and several other genes involved in interactions with red blood cells, including invasion and erythrocyte antigen export. We propose that AF1 parasites have adapted to some as yet unidentified evolutionary niche, by acquiring a complex compendium of interacting variants that rarely circulate separately in Africa. As the cryptotype spread across the continent, it appears to have been maintained mostly intact in spite of recombination events, suggesting a selective advantage. It is possible that other cryptotypes circulate in Africa, and new analysis methods may be needed to identify them.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.579821">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.579821" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.579821">
        <p class="paperTitle">HiCrayon reveals distinct layers of multi-state 3D chromatin organization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.579821" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.579821" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nolan, B.; Harris, H. L.; Kalluchi, A.; Reznicek, T. E.; Cummings, C. T.; Rowley, M. J.</p>
        <p class="info">Score: 15.3, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.579821' target='https://doi.org/10.1101/2024.02.11.579821'> 10.1101/2024.02.11.579821</a></p>
        <p class="abstract">The co-visualization of chromatin conformation with 1D &#39;omics data is key to the multi-omics driven data analysis of 3D genome organization. Chromatin contact maps are often shown as 2D heatmaps and visually compared to 1D genomic data by simple juxtaposition. While common, this strategy is imprecise, placing the onus on the reader to align features with each other. To remedy this, we developed HiCrayon, an interactive tool that facilitates the integration of 3D chromatin organization maps and 1D datasets. This visualization method integrates data from genomic assays directly into the chromatin contact map by coloring interactions according to 1D signal. HiCrayon is implemented using R shiny and python to create a graphical user interface (GUI) application, available in both web or containerized format to promote accessibility. HiCrayon is implemented in R, and includes a graphical user interface (GUI), as well as a slimmed-down web-based version that lets users quickly produce publication-ready images. We demonstrate the utility of HiCrayon in visualizing the effectiveness of compartment calling and the relationship between ChIP-seq and various features of chromatin organization. We also demonstrate the improved visualization of other 3D genomic phenomena, such as differences between loops associated with CTCF/cohesin vs. those associated with H3K27ac. We then demonstrate HiCrayon&#39;s visualization of organizational changes that occur during differentiation and use HiCrayon to detect compartment patterns that cannot be assigned to either A or B compartments, revealing a distinct 3rd chromatin compartment. Overall, we demonstrate the utility of co-visualizing 2D chromatin conformation with 1D genomic signals within the same matrix to reveal fundamental aspects of genome organization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579686">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579686" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579686">
        <p class="paperTitle">Differential Conservation and Loss of CR1 Retrotransposons in Squamates Reveals Lineage-Specific Genome Dynamics across Reptiles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579686" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579686" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gable, S. M.; Bushroe, N. A.; Mendez, J. M.; Wilson, A. A.; Pinto, B. J.; Gamble, T.; Tollis, M.</p>
        <p class="info">Score: 11.1, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579686' target='https://doi.org/10.1101/2024.02.09.579686'> 10.1101/2024.02.09.579686</a></p>
        <p class="abstract">Transposable elements (TEs) are repetitive DNA sequences which create mutations and generate genetic diversity across the tree of life. In amniotic vertebrates, TEs have been mainly studied in mammals and birds, whose genomes generally display low TE diversity. Squamates (Order Squamata; ~11,000 extant species of lizards and snakes) show as much variation in TE abundance and activity as they do in species and phenotypes. Despite this high TE activity, squamate genomes are remarkably uniform in size. We hypothesize that novel, lineage-specific dynamics have evolved over the course of squamate evolution to constrain genome size across the order. Thus, squamates may represent a prime model for investigations into TE diversity and evolution. To understand the interplay between TEs and host genomes, we analyzed the evolutionary history of the CR1 retrotransposon, a TE family found in most tetrapod genomes. We compared 113 squamate genomes to the genomes of turtles, crocodilians, and birds, and used ancestral state reconstruction to identify shifts in the rate of CR1 copy number evolution across reptiles. We analyzed the repeat landscapes of CR1 in squamate genomes and determined that shifts in the rate of CR1 copy number evolution are associated with lineage-specific variation in CR1 activity. We then used phylogenetic reconstruction of CR1 subfamilies across amniotes to reveal both recent and ancient CR1 subclades across the squamate tree of life. The patterns of CR1 evolution in squamates contrast other amniotes, suggesting key differences in how TEs interact with different host genomes and at different points across evolutionary history.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/596718">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/596718" aria-expanded="false" aria-controls="flush-collapse10.1101/596718">
        <p class="paperTitle">NAVIP: Unraveling the Influence of Neighboring Small Sequence Variants on Functional Impact Prediction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/596718" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/596718" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baasner, J.-S.; Rempel, A.; Howard, D.; Pucker, B.</p>
        <p class="info">Score: 52.0, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/596718' target='https://doi.org/10.1101/596718'> 10.1101/596718</a></p>
        <p class="abstract">Once a suitable reference sequence has been generated, intraspecific variation is often assessed by re-sequencing. Variant calling processes can reveal all differences between strains, accessions, genotypes, or individuals. These variants can be enriched with predictions about their functional implications based on available structural annotations, i.e. gene models. Although these functional impact predictions on a per-variant basis are often accurate, some challenging cases require the simultaneous incorporation of multiple adjacent variants into this prediction process. Examples include neighboring variants which modify each others functional impact. The Neighborhood-Aware Variant Impact Predictor (NAVIP) considers all variants within a given protein coding sequence when predicting the functional consequences. As a proof of concept, variants between the Arabidopsis thaliana accessions Columbia-0 and Niederzenz-1 were annotated. NAVIP is freely available on GitHub (https://github.com/bpucker/NAVIP) and accessible through a web server (https://pbb-tools.de).

Author SummaryIntraspecific variation gains increasing relevance as reference genome sequences are available for many investigated (plant) species. Understanding the functional consequences of sequence variants between individuals of a population is a challenge. SnpEff (Cigolnai et al., 2012) is the current standard tool to predict the functional impact of sequence variants, but does only consider one sequence variant at the time. We developed NAVIP to properly handle cases in which multiple sequence variants are clustering together and influence each others functional impact. A comparison of two Arabidopsis thaliana accessions demonstrates the relevance of considering multiple sequence variants simultaneously for the prediction of changes in encoded proteins. NAVIP is universally applicable to non-model organisms. All underlying code is freely available from GitHub and we operate a web server for users convenience.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578486">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578486" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578486">
        <p class="paperTitle">Quantitative omnigenic model discovers interpretable genome-wide associations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578486" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578486" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ruzickova, N.; Hledik, M.; Tkacik, G.</p>
        <p class="info">Score: 11.2, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578486' target='https://doi.org/10.1101/2024.02.01.578486'> 10.1101/2024.02.01.578486</a></p>
        <p class="abstract">As their statistical power grows, genome-wide association studies (GWAS) have identified an increasing number of loci underlying quantitative traits of interest. These loci are scattered throughout the genome and are individually responsible only for small fractions of the total heritable trait variance. The recently proposed omnigenic model provides a conceptual framework to explain these observations by postulating that numerous distant loci contribute to each complex trait via effect propagation through intracellular regulatory networks. We formalize this conceptual framework by proposing the &#34;quantitative omnigenic model&#34; (QOM), a statistical model that combines prior knowledge of the regulatory network topology with genomic data. By applying our model to gene expression traits in yeast, we demonstrate that QOM achieves similar gene expression prediction performance to traditional GWAS with hundreds of times less parameters, while simultaneously extracting candidate causal and quantitative chains of effect propagation through the regulatory network for every individual gene. We estimate the fraction of heritable trait variance in cis- and in trans-, break the latter down by effect propagation order, assess the trans- variance not attributable to transcriptional regulation, and show that QOM correctly accounts for the low-dimensional structure of gene expression covariance. We furthermore demonstrate the relevance of QOM for systems biology, by employing it as a statistical test for the quality of regulatory network reconstructions, and linking it to the propagation of non-genetic, environmental effects.

Significance statementGenetic variation leads to differences in traits implicated in health and disease. Identifying genetic variants associated with these traits is spearheaded by &#34;genome-wide association studies&#34; (GWAS) - statistically rigorous procedures whose power has grown with the number of genotyped samples. Nevertheless, GWAS have a substantial shortcoming: they are ill-equipped to detect the causal basis and reveal the complex systemic mechanisms of polygenic traits. Even a single genetic change can propagate throughout the entire genetic regulatory network causing a myriad of spurious detections, thereby significantly limiting GWAS usefulness. We tackle this challenge with a novel statistical approach that incorporates known regulatory network information to substantially boost the interpretability of state-of-the-art genomic analyses while simultaneously extracting novel systems biology insights.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579496">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579496" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579496">
        <p class="paperTitle">Global genomic epidemiology of blaGES-5 carbapenemase-associated integrons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579496" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579496" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Matlock, W.; Shaw, L. P.; Stoesser, N.</p>
        <p class="info">Score: 8.6, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579496' target='https://doi.org/10.1101/2024.02.09.579496'> 10.1101/2024.02.09.579496</a></p>
        <p class="abstract">Antimicrobial resistance (AMR) gene cassettes comprise an AMR gene flanked by short recombination sites (attI x attC or attC x attC). Integrons are genetic elements able to capture, excise, and shuffle these cassettes, providing  adaptation on demand, and can be found on both chromosomes and plasmids. Understanding the patterns of integron diversity may help to understand the epidemiology of AMR genes. As a case study, we examined the clinical resistance gene blaGES-5, an integron-associated class A carbapenemase first reported in Greece in 2004 and since observed worldwide, which to our knowledge has not been the subject of a previous global analysis. Using a dataset comprising all NCBI contigs containing blaGES-5 (n = 431), we developed a pangenome graph-based workflow to characterise and cluster the diversity of blaGES-5 -associated integrons. We demonstrate that blaGES-5-associated integrons on plasmids are different to those on chromosomes. Chromosomal integrons were almost all identified in P. aeruginosa ST235, with a consistent gene cassette content and order. We observed instances where insertion sequence IS110 disrupted attC sites, which might immobilise the gene cassettes and explain the conserved integron structure despite the presence of intI1 integrase promoters, which would typically facilitate capture or excision and rearrangement. The plasmid-associated integrons were more diverse in their gene cassette content and order, which could be an indication of greater integrase activity and  shuffling of integrons on plasmids.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.11.579857">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.11.579857" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.11.579857">
        <p class="paperTitle">Graph-based Nanopore Adaptive Sampling with GNASTy enables sensitive pneumococcal serotyping in complex samples</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.11.579857" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.11.579857" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Horsfield, S. T.; Fok, B.; Fu, Y.; Turner, P.; Lees, J. A.; Croucher, N. J.</p>
        <p class="info">Score: 8.4, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.11.579857' target='https://doi.org/10.1101/2024.02.11.579857'> 10.1101/2024.02.11.579857</a></p>
        <p class="abstract">Serotype surveillance of Streptococcus pneumoniae (the pneumococcus) is critical for understanding the effectiveness of current vaccination strategies. However, existing methods for serotyping are limited in their ability to identify co-carriage of multiple pneumococci and detect novel serotypes. To develop a scalable and portable serotyping method that overcomes these challenges, we employed Nanopore Adaptive Sampling (NAS), an on-sequencer enrichment method which selects for target DNA in real-time, for direct detection of S. pneumoniae in complex samples. Whereas NAS targeting the whole S. pneumoniae genome was ineffective in the presence of non-pathogenic streptococci, the method was both specific and sensitive when targeting the capsular biosynthetic locus (CBL), the operon that determines S. pneumoniae serotype. NAS significantly improved coverage and yield of the CBL relative to sequencing without NAS, and accurately quantified the relative prevalence of serotypes in samples representing co-carriage. To maximise the sensitivity of NAS to detect novel serotypes, we developed and benchmarked a new pangenome-graph algorithm, named GNASTy. We show that GNASTy outperforms the current NAS implementation, which is based on linear genome alignment, when a sample contains a serotype absent from the database of targeted sequences. The methods developed in this work provide an improved approach for novel serotype discovery and routine S. pneumoniae surveillance that is fast, accurate and feasible in low resource settings. GNASTy therefore has the potential to increase the density and coverage of global pneumococcal surveillance.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
