<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="An encyclopedia of enhancer-gene regulatory interactions in the human genome
Authors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="An encyclopedia of enhancer-gene regulatory interactions in the human genome
Authors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-22T10:41:31+00:00" />
<meta property="article:modified_time" content="2023-11-22T10:41:31+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="An encyclopedia of enhancer-gene regulatory interactions in the human genome
Authors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "An encyclopedia of enhancer-gene regulatory interactions in the human genome\nAuthors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K.",
  "keywords": [
    
  ],
  "articleBody": " An encyclopedia of enhancer-gene regulatory interactions in the human genome\nAuthors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K.; Perez, E. M.; Nguyen, T. C.; Kelley, D. R.; Finucane, H. K.; Moore, J. E.; Weng, Z.; Kellis, M.; Bassik, M. C.; Price, A. L.; Beer, M. A.; Guigo, R.; Stamatoyannopoulos, J. A.; Aiden, E. L.; Greenleaf, W. J.; Leslie, C. S.; Steinmetz, L. M.; Kundaje, A.; Engreitz, J. M.\nScore: 113.2, Published: 2023-11-13 DOI: 10.1101/2023.11.09.563812\nIdentifying transcriptional enhancers and their target genes is essential for understanding gene regulation and the impact of human genetic variation on disease1-6. Here we create and evaluate a resource of \u003e13 million enhancer-gene regulatory interactions across 352 cell types and tissues, by integrating predictive models, measurements of chromatin state and 3D contacts, and large-scale genetic perturbations generated by the ENCODE Consortium7. We first create a systematic benchmarking pipeline to compare predictive models, assembling a dataset of 10,411 element-gene pairs measured in CRISPR perturbation experiments, \u003e30,000 fine-mapped eQTLs, and 569 fine-mapped GWAS variants linked to a likely causal gene. Using this framework, we develop a new predictive model, ENCODE-rE2G, that achieves state-of-the-art performance across multiple prediction tasks, demonstrating a strategy involving iterative perturbations and supervised machine learning to build increasingly accurate predictive models of enhancer regulation. Using the ENCODE-rE2G model, we build an encyclopedia of enhancer-gene regulatory interactions in the human genome, which reveals global properties of enhancer networks, identifies differences in the functions of genes that have more or less complex regulatory landscapes, and improves analyses to link noncoding variants to target genes and cell types for common, complex diseases. By interpreting the model, we find evidence that, beyond enhancer activity and 3D enhancer-promoter contacts, additional features guide enhancer-promoter communication including promoter class and enhancer-enhancer synergy. Altogether, these genome-wide maps of enhancer-gene regulatory interactions, benchmarking software, predictive models, and insights about enhancer function provide a valuable resource for future studies of gene regulation and human genetics.\nA single-cell atlas of transcribed cis-regulatory elements in the human genome\nAuthors: Moody, J.; Kouno, T.; Kojima, M.; Koya, I.; Leon, J.; Suzuki, A.; Hasegawa, A.; Akiyama, T.; Akiyama, N.; Amagai, M.; Chang, J.-C.; Fukushima-Nomura, A.; Handa, M.; Hino, K.; Hino, M.; Hirata, T.; Imai, Y.; Inoue, K.; Kawasaki, H.; Kimura, T.; Kinoshita, T.; Kubo, K.-i.; Kunii, Y.; Lopez-Redondo, F.; Manabe, R.-i.; Miyai, T.; Morimoto, S.; Nagaoka, A.; Nakajima, J.; Noma, S.; Okazaki, Y.; Ozaki, K.; Saeki, N.; Sakai, H.; Seyama, K.; Shibayama, Y.; Sujino, T.; Tagami, M.; Takahashi, H.; Takao, M.; Takeshita, M.; Takiuchi, T.; Terao, C.; Yip, C. W.; Yoshinaga, S.; Okano, H.; Yamamoto, K.; Kasuka\nScore: 50.1, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566791\nTranscribed cis-regulatory elements (tCREs), such as promoters and enhancers, are fundamental to modulate gene expression and define cell identity. The detailed mapping of tCREs at single-cell resolution is essential for understanding the regulatory mechanisms that govern cellular functions. Prior tCRE catalogs, limited by bulk analysis, have often overlooked cellular heterogeneity. We have constructed a tCRE atlas using single-cell 5-RNA-seq, capturing over 340,000 single-cells from 23 human tissues and annotating more than 175,000 tCREs, substantially enhancing the scope and granularity of existing cis-regulatory element annotations in the human genome. This atlas unveils patterns of gene regulation, revealing connections between broadly expressed promoters and cell type-specific distal tCREs. Assessing trait heritability at single-cell resolution with a novel tCRE module-based approach, we uncovered the nuanced trait-gene regulatory relationships across a continuum of cell populations, offering insights beyond traditional gene-level and bulk-sample analyses. Our study bridges the gap between gene regulation and trait heritability, underscoring the potential of single-cell analysis to elucidate the genetic foundations of complex traits. These insights set the stage for future research to investigate the impact of genetic variations on diseases at the individual level, advancing the understanding of cellular and molecular basis of trait heritability.\nInterpreting cis-regulatory mechanisms from genomic deep neural networks using surrogate models\nAuthors: Seitz, E.; McCandlish, D. M.; Kinney, J. B.; Koo, P. K.\nScore: 27.4, Published: 2023-11-16 DOI: 10.1101/2023.11.14.567120\nDeep neural networks (DNNs) have greatly advanced the ability to predict genome function from sequence. Interpreting genomic DNNs in terms of biological mechanisms, however, remains difficult. Here we introduce SQUID, a genomic DNN interpretability framework based on surrogate modeling. SQUID approximates genomic DNNs in user-specified regions of sequence space using surrogate models, i.e., simpler models that are mechanistically interpretable. Importantly, SQUID removes the confounding effects that nonlinearities and heteroscedastic noise in functional genomics data can have on model interpretation. Benchmarking analysis on multiple genomic DNNs shows that SQUID, when compared to established interpretability methods, identifies motifs that are more consistent across genomic loci and yields improved single-nucleotide variant-effect predictions. SQUID also supports surrogate models that quantify epistatic interactions within and between cis-regulatory elements. SQUID thus advances the ability to mechanistically interpret genomic DNNs.\nTesting Times: Challenges in Disentangling Admixture Histories in Recent and Complex Demographies\nAuthors: Williams, M. P.; Flegontov, P.; Maier, R.; Huber, C. D.\nScore: 15.8, Published: 2023-11-15 DOI: 10.1101/2023.11.13.566841\nPaleogenomics has expanded our knowledge of human evolutionary history. Since the 2020s, the study of ancient DNA has increased its focus on reconstructing the recent past. However, the accuracy of paleogenomic methods in answering questions of historical and archaeological importance amidst the increased demographic complexity and decreased genetic differentiation within the historical period remains an open question. We used two simulation approaches to evaluate the limitations and behavior of commonly used methods, qpAdm and the f3-statistic, on admixture inference. The first is based on branch-length data simulated from four simple demographic models of varying complexities and configurations. The second, an analysis of Eurasian history composed of 59 populations using whole-genome data modified with ancient DNA conditions such as SNP ascertainment, data missingness, and pseudo-haploidization. We show that under conditions resembling historical populations, qpAdm can identify a small candidate set of true sources and populations closely related to them. However, in typical ancient DNA conditions, qpAdm is unable to further distinguish between them, limiting its utility for resolving fine-scaled hypotheses. Notably, we find that complex gene-flow histories generally lead to improvements in the performance of qpAdm and observe no bias in the estimation of admixture weights. We offer a heuristic for admixture inference that incorporates admixture weight estimate and P-values of qpAdm models, and f3-statistics to enhance the power to distinguish between multiple plausible candidates. Finally, we highlight the future potential of qpAdm through whole-genome branch-length f2-statistics, demonstrating the improved demographic inference that could be achieved with advancements in f-statistic estimations.\nkmerDB: A Database Encompassing the Set of Genomic and Proteomic Sequence Information for Each Species\nAuthors: Mouratidis, I.; Baltoumas, F. A.; Chantzi, N.; Chan, C.; Montgomery, A.; Konnaris, M. A.; Georgakopoulos, G. C.; Das, A.; Chartoumpekis, D.; Kovac, J.; Pavlopoulos, G.; Georgakopoulos-Soares, I.\nScore: 11.9, Published: 2023-11-16 DOI: 10.1101/2023.11.13.566926\nThe rapid decline in sequencing cost has enabled the generation of reference genomes and proteomes for a growing number of organisms. However, at the present time, there is no established repository that provides information about organism-specific genomic and proteomic sequences of certain lengths, also known as kmers, that are either present or absent in each genome or proteome. In this article, we present kmerDB, a database accessible through an interactive web interface that provides kmer based information from genomic and proteomic sequences in a systematic way. kmerDB currently contains 202,340,859,107 base pairs and 19,304,903,356 amino acids, spanning 45,785 and 22,386 reference genomes and proteomes, respectively, as well as 14,658,776 and 149,264,442 genomic and proteomic species-specific sequences, termed quasi-primes. Additionally, we provide access to 5,186,757 nucleic and 214,904,089 peptide sequences that are absent from every genome and proteome, termed primes. kmerDB features a user-friendly interface offering various search options and filters for easy parsing and searching. The service is available at: www.kmerdb.com.\nAn appropriate differential expression analysis in single-cell data depends on the study design\nAuthors: Lee, H.; Han, B.\nScore: 15.1, Published: 2023-11-20 DOI: 10.1101/2023.01.09.523212\nLarge-scale multi-subject single-cell data have become very common. However, these data have high heterogeneity in study designs, leading to confusion in differential gene expression (DGE) analysis. In this work, we show that pseudobulk produces a substantial amount of type 2 error when the group label being compared varies within a subject. When the comparison label is constant within a subject, type 1 error is inflated if pseudoreplication is not accounted for. As a general principle, we show that an appropriate method depends on the design of the study. We propose solutions to the inflated error rates and provide practical guidelines for researchers.\nBacteriocin Distribution Patterns in Enterococcus faecium and Enterococcus lactis: Bioinformatic Analysis Using a Tailored Genomics Framework\nAuthors: Tedim, A. P.; Almeida-Santos, A. C.; Lanza, V. F.; Novais, C.; Coque, T. M.; Freitas, A. R.; Peixe, L.\nScore: 17.2, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566347\nMultidrug-resistant Enterococcus faecium represent a major concern due to their ability to thrive in diverse environments and cause life-threatening infections. While antimicrobial resistance and virulence mechanisms have been extensively studied, the contribution of bacteriocins to E. faeciums adaptability remains poorly explored. E. faecium, within the Bacillota phylum, is a prominent bacteriocin producer. Here, we developed a tailored database of 76 Bacillota bacteriocins (217 sequences, including 40 novel bacteriocins) and applied it to uncover bacteriocin distribution patterns in 997 quality-filtered E. faecium and Enterococcus lactis (former E. faecium clade B) genomes. Curated using computational pipelines and literature mining, our database demonstrates superior precision versus leading public tools in identifying diverse bacteriocins. Distinct bacteriocin profiles emerged between E. faecium and E. lactis, highlighting species-specific adaptations. E. faecium strains from hospitalized patients were significantly enriched for bacteriocins as entA, bac43, bacAS5 and bacAS11. These bacteriocins strongly associated with antibiotic resistance, particularly vancomycin and ampicillin, and Inc18 rep2_pRE25-derivative plasmids, classically associated with vancomycin resistance transposons. Our integrated genomic and epidemiological analysis elucidates meaningful connections between bacteriocin determinants, antimicrobial resistance, mobile genetic elements, and ecological origins in E. faecium. This work significantly expands the knowledge on the understudied bacteriocin diversity in opportunistic enterococci, revealing their contribution to environmental adaptation. Further characterization of strain-level bacteriocin landscapes could inform strategies to combat high-risk clones. Overall, these insights provide a framework for unravelling bacteriocins therapeutic and biotechnological potential.\nContinent-wide genomic analysis of the African buffalo (Syncerus caffer).\nAuthors: Talenti, A.; Wilkinson, T.; Cook, E. A.; Hemmink, J. D.; Paxton, E.; Mutinda, M.; Ngulu, S. D.; Jayaraman, S.; Bishop, R. P.; Obara, I.; Hourlier, T.; Garcia Giron, C.; Martin, F. J.; Labuschagne, M.; Atimnedi, P.; Nanteza, A.; Keyyu, J. D.; Mramba, F.; Caron, A.; Cornelis, D.; Chardonnet, P.; Fyumagwa, R.; Lembo, T.; Auty, H. K.; Michaux, J.; Smitz, N.; Toye, P.; Robert, C.; Prendergast, J. G. D.; Morrison, L. J.\nScore: 9.6, Published: 2023-11-15 DOI: 10.1101/2023.11.12.566748\nThe African buffalo (Syncerus caffer) is a wild bovid with a historical distribution across much of sub-Saharan Africa. Genomic analysis can provide insights into the evolutionary history of the species, and the key selective pressures shaping populations, including assessment of population level differentiation, population fragmentation, and population genetic structure. In this study we generated the highest quality de novo genome assembly (2.65 Gb, scaffold N50 69.17 Mb) of African buffalo to date, and sequenced a further 195 genomes from across the species distribution. Principal component and admixture analyses provided surprisingly little support for the currently described four subspecies, but indicated three main lineages, in Western/Central, Eastern and Southern Africa, respectively. Estimating Effective Migration Surfaces analysis suggested that geographical barriers have played a significant role in shaping gene flow and the population structure. Estimated effective population sizes indicated a substantial drop occurring in all populations 5-10,000 years ago, coinciding with the increase in human populations. Finally, signatures of selection were enriched for key genes associated with the immune response, suggesting infectious disease exert a substantial selective pressure upon the African buffalo. These findings have important implications for understanding bovid evolution, buffalo conservation and population management.\nNon-canonical ORFs are an important source of tumor-specific antigens in a liver cancer meta-cohort\nAuthors: E. Camarena, M.; Theunissen, P.; Ruiz Egozcue, M.; Ruiz-Orera, J.; Calvo-Serra, B.; Castelo, R.; Sarobe, P.; Fortes, P.; Perera-Bel, J.; Alba, M. M.\nScore: 21.2, Published: 2023-11-01 DOI: 10.1101/2023.10.30.564375\nThe expression of tumor-specific antigens during cancer progression can trigger an immune response against the tumor. Antigens that have been used as cancer vaccines are those originated by non- synonymous mutations and those derived from cancer/testis antigens. However, the first class is predominantly patient-specific, preventing the development of therapies than can benefit multiple patients, and the second one offers a limited set of actionable targets. A possible alternative is the use of peptides derived from non-canonical ORFs (ncORFs). While many ncORFs have been shown to be translated in cancer cells, their tumor-specificity and patient distribution remains to be determined. Here we analyze RNA sequencing data 117 hepatocellular carcinoma (HCC) tumors and matched healthy tissue, together with ribosome profiling data from an additional 10 HCC tumors, to answer these open questions. Combining HLA-epitope binding predictions and experimental validation experiments we conclude that around 40% of the tumor-specific antigens in HCC are likely to be derived from ncORFs in lncRNAs, including two peptides that can trigger an immune response in mice. We identify a subset of 33 tumor-specific lncRNAs expressing novel cancer antigens shared by more than 10% of the HCC analyzed, which could be combined to target a large proportion of the patients. The results of the study open new avenues for extending the range of anti-cancer vaccines.\nTranscriptomic reprogramming screen identifies SRSF1 as rejuvenation factor\nAuthors: Plesa, A. M.; Jung, S.; Wang, H. H.; Omar, F.; Shadpour, M.; Choy Buentello, D.; Perez-Matos, M. C.; Horwitz, N.; Cai, G.; Ngian, Z.-K.; de Magalhaes, C. V.; Wagers, A. J.; Mair, W. B.; del Sol Mesa, A.; Church, G. M.\nScore: 7.8, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566787\nAging is a complex process that manifests through the time-dependent functional decline of a biological system. Age-related changes in epigenetic and transcriptomic profiles have been successfully used to measure the aging process1,2. Moreover, modulating gene regulatory networks through interventions such as the induction of the Yamanaka factors has been shown to reverse aging signatures and improve cell function3,4. However, this intervention has safety and efficacy limitations for in vivo rejuvenation5,6, underscoring the need for identifying novel age reversal factors. Here, we discovered SRSF1 as a new rejuvenation factor that can improve cellular function in vitro and in vivo. Using a cDNA overexpression screen with a transcriptomic readout we identified that SRSF1 induction reprograms the cell transcriptome towards a younger state. Furthermore, we observed beneficial changes in senescence, proteasome function, collagen production, and ROS stress upon SRSF1 overexpression. Lastly, we showed that SRSF1 can improve wound healing in vitro and in vivo and is linked to organismal longevity. Our study provides a proof of concept for using transcriptomic reprogramming screens in the discovery of age reversal interventions and identifies SRSF1 as a promising target for cellular rejuvenation.\n",
  "wordCount" : "2516",
  "inLanguage": "en",
  "datePublished": "2023-11-22T10:41:31Z",
  "dateModified": "2023-11-22T10:41:31Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 22, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.563812">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.563812" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.563812">
        <p class="paperTitle">An encyclopedia of enhancer-gene regulatory interactions in the human genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.563812" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.563812" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K.; Perez, E. M.; Nguyen, T. C.; Kelley, D. R.; Finucane, H. K.; Moore, J. E.; Weng, Z.; Kellis, M.; Bassik, M. C.; Price, A. L.; Beer, M. A.; Guigo, R.; Stamatoyannopoulos, J. A.; Aiden, E. L.; Greenleaf, W. J.; Leslie, C. S.; Steinmetz, L. M.; Kundaje, A.; Engreitz, J. M.</p>
        <p class="info">Score: 113.2, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.563812' target='https://doi.org/10.1101/2023.11.09.563812'> 10.1101/2023.11.09.563812</a></p>
        <p class="abstract">Identifying transcriptional enhancers and their target genes is essential for understanding gene regulation and the impact of human genetic variation on disease1-6. Here we create and evaluate a resource of &gt;13 million enhancer-gene regulatory interactions across 352 cell types and tissues, by integrating predictive models, measurements of chromatin state and 3D contacts, and large-scale genetic perturbations generated by the ENCODE Consortium7. We first create a systematic benchmarking pipeline to compare predictive models, assembling a dataset of 10,411 element-gene pairs measured in CRISPR perturbation experiments, &gt;30,000 fine-mapped eQTLs, and 569 fine-mapped GWAS variants linked to a likely causal gene. Using this framework, we develop a new predictive model, ENCODE-rE2G, that achieves state-of-the-art performance across multiple prediction tasks, demonstrating a strategy involving iterative perturbations and supervised machine learning to build increasingly accurate predictive models of enhancer regulation. Using the ENCODE-rE2G model, we build an encyclopedia of enhancer-gene regulatory interactions in the human genome, which reveals global properties of enhancer networks, identifies differences in the functions of genes that have more or less complex regulatory landscapes, and improves analyses to link noncoding variants to target genes and cell types for common, complex diseases. By interpreting the model, we find evidence that, beyond enhancer activity and 3D enhancer-promoter contacts, additional features guide enhancer-promoter communication including promoter class and enhancer-enhancer synergy. Altogether, these genome-wide maps of enhancer-gene regulatory interactions, benchmarking software, predictive models, and insights about enhancer function provide a valuable resource for future studies of gene regulation and human genetics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566791" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566791">
        <p class="paperTitle">A single-cell atlas of transcribed cis-regulatory elements in the human genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moody, J.; Kouno, T.; Kojima, M.; Koya, I.; Leon, J.; Suzuki, A.; Hasegawa, A.; Akiyama, T.; Akiyama, N.; Amagai, M.; Chang, J.-C.; Fukushima-Nomura, A.; Handa, M.; Hino, K.; Hino, M.; Hirata, T.; Imai, Y.; Inoue, K.; Kawasaki, H.; Kimura, T.; Kinoshita, T.; Kubo, K.-i.; Kunii, Y.; Lopez-Redondo, F.; Manabe, R.-i.; Miyai, T.; Morimoto, S.; Nagaoka, A.; Nakajima, J.; Noma, S.; Okazaki, Y.; Ozaki, K.; Saeki, N.; Sakai, H.; Seyama, K.; Shibayama, Y.; Sujino, T.; Tagami, M.; Takahashi, H.; Takao, M.; Takeshita, M.; Takiuchi, T.; Terao, C.; Yip, C. W.; Yoshinaga, S.; Okano, H.; Yamamoto, K.; Kasuka</p>
        <p class="info">Score: 50.1, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566791' target='https://doi.org/10.1101/2023.11.13.566791'> 10.1101/2023.11.13.566791</a></p>
        <p class="abstract">Transcribed cis-regulatory elements (tCREs), such as promoters and enhancers, are fundamental to modulate gene expression and define cell identity. The detailed mapping of tCREs at single-cell resolution is essential for understanding the regulatory mechanisms that govern cellular functions. Prior tCRE catalogs, limited by bulk analysis, have often overlooked cellular heterogeneity. We have constructed a tCRE atlas using single-cell 5-RNA-seq, capturing over 340,000 single-cells from 23 human tissues and annotating more than 175,000 tCREs, substantially enhancing the scope and granularity of existing cis-regulatory element annotations in the human genome. This atlas unveils patterns of gene regulation, revealing connections between broadly expressed promoters and cell type-specific distal tCREs. Assessing trait heritability at single-cell resolution with a novel tCRE module-based approach, we uncovered the nuanced trait-gene regulatory relationships across a continuum of cell populations, offering insights beyond traditional gene-level and bulk-sample analyses. Our study bridges the gap between gene regulation and trait heritability, underscoring the potential of single-cell analysis to elucidate the genetic foundations of complex traits. These insights set the stage for future research to investigate the impact of genetic variations on diseases at the individual level, advancing the understanding of cellular and molecular basis of trait heritability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567120" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567120">
        <p class="paperTitle">Interpreting cis-regulatory mechanisms from genomic deep neural networks using surrogate models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seitz, E.; McCandlish, D. M.; Kinney, J. B.; Koo, P. K.</p>
        <p class="info">Score: 27.4, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567120' target='https://doi.org/10.1101/2023.11.14.567120'> 10.1101/2023.11.14.567120</a></p>
        <p class="abstract">Deep neural networks (DNNs) have greatly advanced the ability to predict genome function from sequence. Interpreting genomic DNNs in terms of biological mechanisms, however, remains difficult. Here we introduce SQUID, a genomic DNN interpretability framework based on surrogate modeling. SQUID approximates genomic DNNs in user-specified regions of sequence space using surrogate models, i.e., simpler models that are mechanistically interpretable. Importantly, SQUID removes the confounding effects that nonlinearities and heteroscedastic noise in functional genomics data can have on model interpretation. Benchmarking analysis on multiple genomic DNNs shows that SQUID, when compared to established interpretability methods, identifies motifs that are more consistent across genomic loci and yields improved single-nucleotide variant-effect predictions. SQUID also supports surrogate models that quantify epistatic interactions within and between cis-regulatory elements. SQUID thus advances the ability to mechanistically interpret genomic DNNs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566841">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566841" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566841">
        <p class="paperTitle">Testing Times: Challenges in Disentangling Admixture Histories in Recent and Complex Demographies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566841" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566841" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Williams, M. P.; Flegontov, P.; Maier, R.; Huber, C. D.</p>
        <p class="info">Score: 15.8, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566841' target='https://doi.org/10.1101/2023.11.13.566841'> 10.1101/2023.11.13.566841</a></p>
        <p class="abstract">Paleogenomics has expanded our knowledge of human evolutionary history. Since the 2020s, the study of ancient DNA has increased its focus on reconstructing the recent past. However, the accuracy of paleogenomic methods in answering questions of historical and archaeological importance amidst the increased demographic complexity and decreased genetic differentiation within the historical period remains an open question. We used two simulation approaches to evaluate the limitations and behavior of commonly used methods, qpAdm and the f3-statistic, on admixture inference. The first is based on branch-length data simulated from four simple demographic models of varying complexities and configurations. The second, an analysis of Eurasian history composed of 59 populations using whole-genome data modified with ancient DNA conditions such as SNP ascertainment, data missingness, and pseudo-haploidization. We show that under conditions resembling historical populations, qpAdm can identify a small candidate set of true sources and populations closely related to them. However, in typical ancient DNA conditions, qpAdm is unable to further distinguish between them, limiting its utility for resolving fine-scaled hypotheses. Notably, we find that complex gene-flow histories generally lead to improvements in the performance of qpAdm and observe no bias in the estimation of admixture weights. We offer a heuristic for admixture inference that incorporates admixture weight estimate and P-values of qpAdm models, and f3-statistics to enhance the power to distinguish between multiple plausible candidates. Finally, we highlight the future potential of qpAdm through whole-genome branch-length f2-statistics, demonstrating the improved demographic inference that could be achieved with advancements in f-statistic estimations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566926">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566926" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566926">
        <p class="paperTitle">kmerDB: A Database Encompassing the Set of Genomic and Proteomic Sequence Information for Each Species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566926" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566926" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mouratidis, I.; Baltoumas, F. A.; Chantzi, N.; Chan, C.; Montgomery, A.; Konnaris, M. A.; Georgakopoulos, G. C.; Das, A.; Chartoumpekis, D.; Kovac, J.; Pavlopoulos, G.; Georgakopoulos-Soares, I.</p>
        <p class="info">Score: 11.9, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566926' target='https://doi.org/10.1101/2023.11.13.566926'> 10.1101/2023.11.13.566926</a></p>
        <p class="abstract">The rapid decline in sequencing cost has enabled the generation of reference genomes and proteomes for a growing number of organisms. However, at the present time, there is no established repository that provides information about organism-specific genomic and proteomic sequences of certain lengths, also known as kmers, that are either present or absent in each genome or proteome. In this article, we present kmerDB, a database accessible through an interactive web interface that provides kmer based information from genomic and proteomic sequences in a systematic way. kmerDB currently contains 202,340,859,107 base pairs and 19,304,903,356 amino acids, spanning 45,785 and 22,386 reference genomes and proteomes, respectively, as well as 14,658,776 and 149,264,442 genomic and proteomic species-specific sequences, termed quasi-primes. Additionally, we provide access to 5,186,757 nucleic and 214,904,089 peptide sequences that are absent from every genome and proteome, termed primes. kmerDB features a user-friendly interface offering various search options and filters for easy parsing and searching. The service is available at: www.kmerdb.com.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.09.523212">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.09.523212" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.09.523212">
        <p class="paperTitle">An appropriate differential expression analysis in single-cell data depends on the study design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.09.523212" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.09.523212" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, H.; Han, B.</p>
        <p class="info">Score: 15.1, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.09.523212' target='https://doi.org/10.1101/2023.01.09.523212'> 10.1101/2023.01.09.523212</a></p>
        <p class="abstract">Large-scale multi-subject single-cell data have become very common. However, these data have high heterogeneity in study designs, leading to confusion in differential gene expression (DGE) analysis. In this work, we show that pseudobulk produces a substantial amount of type 2 error when the group label being compared varies within a subject. When the comparison label is constant within a subject, type 1 error is inflated if pseudoreplication is not accounted for. As a general principle, we show that an appropriate method depends on the design of the study. We propose solutions to the inflated error rates and provide practical guidelines for researchers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566347">
        <p class="paperTitle">Bacteriocin Distribution Patterns in Enterococcus faecium and Enterococcus lactis: Bioinformatic Analysis Using a Tailored Genomics Framework</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tedim, A. P.; Almeida-Santos, A. C.; Lanza, V. F.; Novais, C.; Coque, T. M.; Freitas, A. R.; Peixe, L.</p>
        <p class="info">Score: 17.2, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566347' target='https://doi.org/10.1101/2023.11.13.566347'> 10.1101/2023.11.13.566347</a></p>
        <p class="abstract">Multidrug-resistant Enterococcus faecium represent a major concern due to their ability to thrive in diverse environments and cause life-threatening infections. While antimicrobial resistance and virulence mechanisms have been extensively studied, the contribution of bacteriocins to E. faeciums adaptability remains poorly explored. E. faecium, within the Bacillota phylum, is a prominent bacteriocin producer. Here, we developed a tailored database of 76 Bacillota bacteriocins (217 sequences, including 40 novel bacteriocins) and applied it to uncover bacteriocin distribution patterns in 997 quality-filtered E. faecium and Enterococcus lactis (former E. faecium clade B) genomes. Curated using computational pipelines and literature mining, our database demonstrates superior precision versus leading public tools in identifying diverse bacteriocins. Distinct bacteriocin profiles emerged between E. faecium and E. lactis, highlighting species-specific adaptations. E. faecium strains from hospitalized patients were significantly enriched for bacteriocins as entA, bac43, bacAS5 and bacAS11. These bacteriocins strongly associated with antibiotic resistance, particularly vancomycin and ampicillin, and Inc18 rep2_pRE25-derivative plasmids, classically associated with vancomycin resistance transposons. Our integrated genomic and epidemiological analysis elucidates meaningful connections between bacteriocin determinants, antimicrobial resistance, mobile genetic elements, and ecological origins in E. faecium. This work significantly expands the knowledge on the understudied bacteriocin diversity in opportunistic enterococci, revealing their contribution to environmental adaptation. Further characterization of strain-level bacteriocin landscapes could inform strategies to combat high-risk clones. Overall, these insights provide a framework for unravelling bacteriocins therapeutic and biotechnological potential.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.12.566748">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.12.566748" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.12.566748">
        <p class="paperTitle">Continent-wide genomic analysis of the African buffalo (Syncerus caffer).</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.12.566748" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.12.566748" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Talenti, A.; Wilkinson, T.; Cook, E. A.; Hemmink, J. D.; Paxton, E.; Mutinda, M.; Ngulu, S. D.; Jayaraman, S.; Bishop, R. P.; Obara, I.; Hourlier, T.; Garcia Giron, C.; Martin, F. J.; Labuschagne, M.; Atimnedi, P.; Nanteza, A.; Keyyu, J. D.; Mramba, F.; Caron, A.; Cornelis, D.; Chardonnet, P.; Fyumagwa, R.; Lembo, T.; Auty, H. K.; Michaux, J.; Smitz, N.; Toye, P.; Robert, C.; Prendergast, J. G. D.; Morrison, L. J.</p>
        <p class="info">Score: 9.6, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.12.566748' target='https://doi.org/10.1101/2023.11.12.566748'> 10.1101/2023.11.12.566748</a></p>
        <p class="abstract">The African buffalo (Syncerus caffer) is a wild bovid with a historical distribution across much of sub-Saharan Africa. Genomic analysis can provide insights into the evolutionary history of the species, and the key selective pressures shaping populations, including assessment of population level differentiation, population fragmentation, and population genetic structure. In this study we generated the highest quality de novo genome assembly (2.65 Gb, scaffold N50 69.17 Mb) of African buffalo to date, and sequenced a further 195 genomes from across the species distribution. Principal component and admixture analyses provided surprisingly little support for the currently described four subspecies, but indicated three main lineages, in Western/Central, Eastern and Southern Africa, respectively. Estimating Effective Migration Surfaces analysis suggested that geographical barriers have played a significant role in shaping gene flow and the population structure. Estimated effective population sizes indicated a substantial drop occurring in all populations 5-10,000 years ago, coinciding with the increase in human populations. Finally, signatures of selection were enriched for key genes associated with the immune response, suggesting infectious disease exert a substantial selective pressure upon the African buffalo. These findings have important implications for understanding bovid evolution, buffalo conservation and population management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564375">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564375" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564375">
        <p class="paperTitle">Non-canonical ORFs are an important source of tumor-specific antigens in a liver cancer meta-cohort</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564375" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564375" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: E. Camarena, M.; Theunissen, P.; Ruiz Egozcue, M.; Ruiz-Orera, J.; Calvo-Serra, B.; Castelo, R.; Sarobe, P.; Fortes, P.; Perera-Bel, J.; Alba, M. M.</p>
        <p class="info">Score: 21.2, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564375' target='https://doi.org/10.1101/2023.10.30.564375'> 10.1101/2023.10.30.564375</a></p>
        <p class="abstract">The expression of tumor-specific antigens during cancer progression can trigger an immune response against the tumor. Antigens that have been used as cancer vaccines are those originated by non- synonymous mutations and those derived from cancer/testis antigens. However, the first class is predominantly patient-specific, preventing the development of therapies than can benefit multiple patients, and the second one offers a limited set of actionable targets. A possible alternative is the use of peptides derived from non-canonical ORFs (ncORFs). While many ncORFs have been shown to be translated in cancer cells, their tumor-specificity and patient distribution remains to be determined. Here we analyze RNA sequencing data 117 hepatocellular carcinoma (HCC) tumors and matched healthy tissue, together with ribosome profiling data from an additional 10 HCC tumors, to answer these open questions. Combining HLA-epitope binding predictions and experimental validation experiments we conclude that around 40% of the tumor-specific antigens in HCC are likely to be derived from ncORFs in lncRNAs, including two peptides that can trigger an immune response in mice. We identify a subset of 33 tumor-specific lncRNAs expressing novel cancer antigens shared by more than 10% of the HCC analyzed, which could be combined to target a large proportion of the patients. The results of the study open new avenues for extending the range of anti-cancer vaccines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566787">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566787" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566787">
        <p class="paperTitle">Transcriptomic reprogramming screen identifies SRSF1 as rejuvenation factor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566787" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566787" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Plesa, A. M.; Jung, S.; Wang, H. H.; Omar, F.; Shadpour, M.; Choy Buentello, D.; Perez-Matos, M. C.; Horwitz, N.; Cai, G.; Ngian, Z.-K.; de Magalhaes, C. V.; Wagers, A. J.; Mair, W. B.; del Sol Mesa, A.; Church, G. M.</p>
        <p class="info">Score: 7.8, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566787' target='https://doi.org/10.1101/2023.11.13.566787'> 10.1101/2023.11.13.566787</a></p>
        <p class="abstract">Aging is a complex process that manifests through the time-dependent functional decline of a biological system. Age-related changes in epigenetic and transcriptomic profiles have been successfully used to measure the aging process1,2. Moreover, modulating gene regulatory networks through interventions such as the induction of the Yamanaka factors has been shown to reverse aging signatures and improve cell function3,4. However, this intervention has safety and efficacy limitations for in vivo rejuvenation5,6, underscoring the need for identifying novel age reversal factors. Here, we discovered SRSF1 as a new rejuvenation factor that can improve cellular function in vitro and in vivo. Using a cDNA overexpression screen with a transcriptomic readout we identified that SRSF1 induction reprograms the cell transcriptome towards a younger state. Furthermore, we observed beneficial changes in senescence, proteasome function, collagen production, and ROS stress upon SRSF1 overexpression. Lastly, we showed that SRSF1 can improve wound healing in vitro and in vivo and is linked to organismal longevity. Our study provides a proof of concept for using transcriptomic reprogramming screens in the discovery of age reversal interventions and identifies SRSF1 as a promising target for cellular rejuvenation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
