<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Randomizing the human genome by engineering recombination between repeat elements
Authors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.
Score: 98.4, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576745
While protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Randomizing the human genome by engineering recombination between repeat elements
Authors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.
Score: 98.4, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576745
While protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Randomizing the human genome by engineering recombination between repeat elements
Authors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.
Score: 98.4, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576745
While protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Randomizing the human genome by engineering recombination between repeat elements\nAuthors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.\nScore: 98.4, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576745\nWhile protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent.",
  "keywords": [
    
  ],
  "articleBody": " Randomizing the human genome by engineering recombination between repeat elements\nAuthors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.\nScore: 98.4, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576745\nWhile protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent. To bridge this gap, we have developed a toolbox to create deletions, inversions, translocations, and extrachromosomal circular DNA at scale by highly multiplexed insertion of recombinase recognition sites into repetitive sequences with CRISPR prime editing. Using this strategy, we derived stable human cell lines with several thousand clonal insertions, the highest number of novel sequences inserted into single human genomes. Subsequent recombinase induction generated an average of more than one hundred megabase-sized rearrangements per cell, and thousands across the whole population. The ability to detect rearrangements as they are generated and to track their abundance over time allowed us to measure the selection pressures acting on different types of structural changes. We observed a consolidation towards shorter variants that preferentially delete growth-inhibiting genes and a depletion of translocations. We isolated and characterized 21 clones with multiple recombinase-induced rearrangements. These included viable haploid clones with deletions that span hundreds of kilobases as well as triploid HEK293T clones with aneuploidies and fold back chromosomes. We mapped the impact of these genetic changes on gene expression to decipher how structural variants affect gene regulation. The genome scrambling strategy developed here makes it possible to delete megabases of sequence, move sequences between and within chromosomes, and implant regulatory elements into new contexts which will shed light on the genome organization principles of humans and other species.\nMultiplex generation and single cell analysis of structural variants in a mammalian genome\nAuthors: Pinglay, S.; Lalanne, J.-B.; Daza, R. M.; Koeppel, J.; Li, X.; Lee, D. S.; Shendure, J.\nScore: 59.3, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576756\nThe functional consequences of structural variants (SVs) in mammalian genomes are challenging to study. This is due to several factors, including: 1) their numerical paucity relative to other forms of standing genetic variation such as single nucleotide variants (SNVs) and short insertions or deletions (indels); 2) the fact that a single SV can involve and potentially impact the function of more than one gene and/or cis regulatory element; and 3) the relative immaturity of methods to generate SVs, either randomly or in targeted fashion, in in vitro or in vivo model systems. Towards addressing some of these challenges, we developed Genome-Shuffle-seq, a straightforward method that enables the multiplex generation of several major forms of SVs (deletions, inversions, translocations) throughout a mammalian genome. Genome-Shuffle-seq is based on the random integration of \"shuffle cassettes\" to the genome, wherein each shuffle cassette contains components that facilitate its site-specific recombination (SSR) with other integrated shuffle cassettes (via Cre-loxP), its mapping to a specific genomic location (via T7-mediated in vitro transcription or IVT), and its identification in single-cell RNA-seq (scRNA-seq) data (via T7-mediated in situ transcription or IST). In this proof-of-concept, we apply Genome-Shuffle-seq to induce and map thousands of genomic SVs in mouse embryonic stem cells (mESCs) in a single experiment. Induced SVs are rapidly depleted from the cellular population over time, possibly due to some combination of Cre-mediated toxicity and negative selection on the rearrangements themselves. We demonstrate that we can efficiently genotype which SVs are present in association with each of many single cell transcriptomes in scRNA-seq data. Finally, preliminary evidence suggests our method may be a powerful means of generating extrachromosomal circular DNAs (ecDNAs). Looking forward, we anticipate that Genome-Shuffle-seq may be broadly useful for the systematic exploration of the functional consequences of SVs on gene expression, the chromatin landscape, and 3D nuclear architecture. We further anticipate potential uses for in vitro modeling of ecDNAs, as well as in paving the path to a minimal mammalian genome.\nA Comprehensive Resource for Exploring Antiphage Defense: DefenseFinder Webservice, Wiki and Databases.\nAuthors: Tesson, F.; Planel, R.; Egorov, A. A.; Georjon, H.; Vaysset, H.; Brancotte, B.; Neron, B.; Mordret, E.; Atkinson, G. C.; Bernheim, A.; Cury, J.\nScore: 56.0, Published: 2024-01-26 DOI: 10.1101/2024.01.25.577194\nIn recent years, a vast number of novel antiphage defense mechanisms were uncovered. To facilitate the exploration of mechanistic, ecological, and evolutionary aspects related to antiphage defense systems, we released DefenseFinder in 2021 (Tesson et al., 2022). DefenseFinder is a bioinformatic program designed for the systematic identification of all known antiphage defense mechanisms. The initial release of DefenseFinder v1.0.0 included 60 systems. Over the past three years, the number of antiphage systems incorporated into DefenseFinder has grown to 152. The increasing number of known systems makes it a challenge to enter the field and makes the interpretation of detections of antiphage systems difficult. Moreover, the rapid development of sequence-based predictions of structures offers novel possibilities of analysis and should be easily available. To overcome these challenges, we present a hub of resources on defense systems including 1) a search tool (DefenseFinder as a web service) 2) a knowledge base, and 3) precomputed databases (results of DefenseFinder ran on RefSeq database, predicted structure database computed with AlphaFold). These pages can be freely accessed for users as a starting point on their journey to better understand a given system. We anticipate that these resources will foster the use of bioinformatics in the study of antiphage systems and will serve the community of researchers who study antiphage systems. This resource is available at: https://defensefinder.mdmlab.fr.\nSystematic reconstruction of molecular pathway signatures using scalable single-cell perturbation screens\nAuthors: Jiang, L.; Dalgarno, C.; Papalexi, E.; Mascio, I.; Wessels, H.-H.; Yun, H.; Iremadze, N.; Lithwick-Yanai, G.; Lipson, D.; Satija, R.\nScore: 38.7, Published: 2024-01-30 DOI: 10.1101/2024.01.29.576933\nRecent advancements in functional genomics have provided an unprecedented ability to measure diverse molecular modalities, but learning causal regulatory relationships from observational data remains challenging. Here, we leverage pooled genetic screens and single cell sequencing (i.e. Perturb-seq) to systematically identify the targets of signaling regulators in diverse biological contexts. We demonstrate how Perturb-seq is compatible with recent and commercially available advances in combinatorial indexing and next-generation sequencing, and perform more than 1,500 perturbations split across six cell lines and five biological signaling contexts. We introduce an improved computational framework (Mixscale) to address cellular variation in perturbation efficiency, alongside optimized statistical methods to learn differentially expressed gene lists and conserved molecular signatures. Finally, we demonstrate how our Perturb-seq derived gene lists can be used to precisely infer changes in signaling pathway activation for in-vivo and in-situ samples. Our work enhances our understanding of signaling regulators and their targets, and lays a computational framework towards the data-driven inference of an 'atlas' of perturbation signatures.\nA high coverage Mesolithic aurochs genome and effective leveraging of ancient cattle genomes using whole genome imputation.\nAuthors: Erven, J. A. M.; Scheu, A.; Verdugo, M. P.; Cassidy, L.; Chen, N.; Gehlen, B.; Street, M.; Madsen, O.; Mullin, V. E.\nScore: 16.4, Published: 2024-01-24 DOI: 10.1101/2024.01.23.576850\nAncient genomic analyses are often restricted to utilising pseudo-haploid data due to low genome coverage. Leveraging low coverage data by imputation to calculate phased diploid genotypes that enable haplotype-based interrogation and SNP calling at unsequenced positions is highly desirable. This has not been investigated for ancient cattle genomes despite these being compelling subjects for archaeological, evolutionary and economic reasons. Here we test this approach by sequencing a Mesolithic European aurochs (18.49x; 9852-9376 calBC), an Early Medieval European cow (18.69x; 427-580 calAD), and combine these with published individuals; two ancient and three modern. We downsample these genomes (0.25x, 0.5x, 1.0x, 2.0x) and impute diploid genotypes, utilising a reference panel of 171 published modern cattle genomes that we curated for 21.7 million (Mn) phased single-nucleotide polymorphisms (SNPs). We recover high densities of correct calls with an accuracy of \u003e99.1% at variant sites for the lowest downsample depth of 0.25x, increasing to \u003e99.5% for 2.0x (transversions only, minor allele frequency (MAF) [\u0026ge;]2.5%). The recovery of SNPs correlates with coverage, on average 58% of sites are recovered for 0.25x increasing to 87% for 2.0x, utilising an average of 3.5 million (Mn) transversions (MAF [\u0026ge;]2.5%), even in the aurochs which is temporally and morphologically distinct from the reference panel. Our imputed genomes behave similarly to directly called data in allele-frequency-based analyses; for example consistently identifying runs of homozygosity \u003e2mb, including a long homozygous region in the Mesolithic European aurochs.\nWhole-genome phylogenomics of the tinamous (Aves: Tinamidae): comparing gene tree estimation error between BUSCOs and UCEs illuminates rapid divergence with introgression\nAuthors: Musher, L. J.; Catanach, T. A.; Valqui, T.; Brumfield, R. T.; Aleixo, A.; Johnson, K. P.; Weckstein, J. D.\nScore: 15.6, Published: 2024-01-23 DOI: 10.1101/2024.01.22.576737\nIncomplete lineage sorting (ILS) and introgression increase genealogical discordance across the genome, which complicates phylogenetic inference. In such cases, identifying orthologs that result in gene trees with low estimation error is crucial because phylogenomic methods rely on accurate gene histories. We sequenced whole genomes for the tinamous (Aves: Tinamidae) to dissect the sources of gene and species-tree discordance and reconstruct their interrelationships. We compared results based on four ortholog sets: (1) coding genes (BUSCOs), (2) ultraconserved elements (UCEs) with short flanking regions, (3) UCEs with intermediate flanks, and (4) UCEs with long flanks. We hypothesized that orthologs with more phylogenetically informative sites would result in more accurate species trees because the resulting gene trees contain lower error. Consistent with our hypothesis, we found that long UCEs had the most informative sites and lowest rates of error. However, despite having many informative sites, BUSCO gene trees contained high error compared to long UCEs. Unlike UCEs, BUSCO gene sequences showed a positive association between the proportion of parsimony informative sites and gene tree error. Thus, BUSCO and UCE datasets have different underlying properties of molecular evolution, and these differences should be considered when selecting loci for phylogenomic analysis. Still, species trees from different datasets were mostly congruent. Only one clade, with a history of ILS and introgression, exhibited substantial species-tree discordance across the different data sets. Overall, we present the most complete phylogeny for tinamous to date, identify a new species, and provide a case study for species-level phylogenomic analysis using whole-genomes.\n10x Genomics Gene Expression Flex is a powerful tool for single-cell transcriptomics of xenograft models\nAuthors: Llora-Batlle, O.; Farcas, A.; Fransen, D.; Floch, N.; Talbot, S.; Schwiening, A.; Bojko, L.; Calver, J.; Josipovic, N.; Lashuk, K.; Schueler, J.; Prodan, A.; Mooijman, D.; McDermott, U.; PERSIST-SEQ Consortium, Score: 11.0, Published: 2024-01-29 DOI: 10.1101/2024.01.25.577066\nThe 10x Genomics Gene Expression Flex protocol allows profiling of fixed or frozen material, greatly simplifying the logistics of sample collection, storage and transfer prior to single -cell sequencing. The method makes single-cell transcriptomics possible for existing fresh-frozen or FFPE tissue samples, but also facilitates the logistics of the sampling process, allowing instant preservation of samples. The technology relies on species-specific probes available for human and mouse. Nevertheless, processing of patient-derived (PDX) or cell line (CDX) xenografts, which contain mixed human and mouse cells, is currently not supported by this protocol due to the high degree of homology between the probe sets. Here we show that it is feasible to simultaneously profile populations containing both human and mouse cells by mixing the transcriptome probe sets of both species. Cellranger outputs a count table for each of the species allowing evaluation of the performance of the different probe sets. Cross-reactive probes are greatly outperformed by the specific probe hybridizations leading to a clear difference in the recovery of UMIs and unique genes per cell. Furthermore, we developed a pipeline that removes cross-reactive signal from the data and provides species-specific count tables for further downstream analysis. Hence, the 10x Genomics Gene Expression Flex protocol can be used to process xenograft samples without the need for separation of human and mouse cells by flow sorting and allows analysis of the human and mouse single-cell transcriptome from each sample. We anticipate it will be increasingly used for single-cell sequencing of cancer cell line and patient-derived xenografts, facilitating the preservation of the samples and allowing the interrogation of both the (human) xenograft and the (mouse) tumor microenvironment at single-cell resolution.\nMultiple Displacement Amplification Facilitates SMRT Sequencing of Microscopic Animals and the Genome of the Gastrotrich Lepidodermella squamata (Dujardin, 1841)\nAuthors: Roberts, N. G.; Gilmore, M. J.; Struck, T. H.; Kocot, K.\nScore: 11.2, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576123\nBackgroundObtaining adequate DNA for long-read genome sequencing remains a roadblock to producing contiguous genomes from small-bodied organisms. Multiple displacement amplification (MDA) leverages Phi29 DNA polymerase to produce micrograms of DNA from picograms of input. Few genomes have been generated using this approach, due to concerns over biases in amplification related to GC and repeat content and chimera production. Here, we explored the utility of MDA for generating template DNA for PacBio HiFi sequencing using Caenorhabditis elegans (Nematoda) and Lepidodermella squamata (Gastrotricha). ResultsHiFi sequencing of libraries prepared from MDA DNA produced highly contiguous and complete genomes for both C. elegans (102 Mbp assembly; 336 contigs; N50 = 868 Kbp; L50 = 39; BUSCO_nematoda: S:92.2%, D:2.7%) and L. squamata (122 Mbp assembly; 157 contigs; N50 = 3.9 Mb; L50 = 13; BUSCO_metazoa: S: 78.0%, D: 2.8%). Amplified C. elegans reads mapped to the reference genome with a rate of 99.92% and coverage of 99.75% with just one read (of 708,811) inferred to be chimeric. Coverage uniformity was nearly identical for reads from MDA DNA and reads from pooled worm DNA when mapped to the reference genome. The genome of Lepidodermella squamata, the first of its phylum, was leveraged to infer the phylogenetic position of Gastrotricha, which has long been debated, as the sister taxon of Platyhelminthes. ConclusionsThis methodology will help generate contiguous genomes of microscopic taxa whose body size precludes standard long-read sequencing. L. squamata is an emerging model in evolutionary developmental biology and this genome will facilitate further work on this species.\nRepeated horizontal acquisition of lagriamide-producing symbionts in Lagriinae beetles\nAuthors: Uppal, S.; Waterworth, S. C.; Nick, A.; Vogel, H.; Florez, L.; Kaltenpoth, M.; Kwan, J. C.\nScore: 8.4, Published: 2024-01-25 DOI: 10.1101/2024.01.23.576914\nMicrobial symbionts associate with multicellular organisms on a continuum from facultative associations to mutual codependency. In the oldest symbioses there is exclusive vertical symbiont transmission, and extensive co-diversification of symbiotic partners over millions of years and speciation events. Such symbionts often undergo genome reduction due to low effective populations, frequent population bottlenecks and weakened purifying selection. We describe here a group of Lagriinae beetle symbionts, which produce defensive lagriamide molecules to protect beetle eggs and larvae from fungal infection. These beetles likely evolved specialised symbiont-storage structures between 55 and 82 million years ago. Previous work found, despite the lagriamide-producing bacterium not being genetically isolated and potentially surviving in the environment, it underwent genome reduction in Lagria villosa hosts. Here, we use shotgun metagenomics to assemble 11 additional lagriamide-producing symbionts from diverse hosts within Lagriinae from five countries, to clarify the evolution of this symbiotic relationship. Surprisingly, we did not find evidence of co-evolution between symbionts and hosts, and although the lagriamide gene cluster (lga) had been horizontally transferred to the L. villosa symbiont, it likely was not transferred between symbionts. Instead, lga was likely obtained only once in the common ancestor of all lagriamide-producing symbionts and subsequently lost in related free-living strains. Our results suggest that beetles acquired lagriamide-producing symbionts specifically multiple times independently, which was followed by genome erosion. We also show that these symbionts have been replaced in the beetle host multiple times. The system therefore offers a unique opportunity to study the process of genome reduction.\nHaplotype phased genome of 'Fairchild' mandarin highlights influence of local chromatin state on gene expression\nAuthors: Diaz, I. A.; Ostovar, T.; Chen, J.; Saddoris, S.; Schmitz, R. J.; Wessler, S. R.; Stajich, J.; Seymour, D. K.\nScore: 6.9, Published: 2024-01-24 DOI: 10.1101/2024.01.20.575729\nBackgroundCis-regulatory sequences control gene expression through the coordinated action of transcription factors and their associated partners. Both genetic and epigenetic perturbation of cis-regulatory sequences can lead to novel patterns of gene expression. Phased genome assemblies now enable the local dissection of linkages between cis-regulatory sequences, including their epigenetic state, and gene expression to further characterize gene regulation in heterozygous genomes. ResultsWe assembled a locally phased genome for a mandarin hybrid named Fairchild to explore the molecular signatures of allele-specific gene expression. With genome phasing, genes with allele-specific expression were paired with haplotype-specific chromatin states, including levels of chromatin accessibility, histone modifications, and DNA methylation. We found that 30% of variation in allele-specific expression could be attributed to haplotype associated factors, with allelic levels of chromatin accessibility and three histone modifications in gene bodies having the most influence. Structural variants in promoter regions were also associated with allele-specific expression, including specific enrichments of hAT and MULE-MuDR DNA transposon sequences. Mining of cis-regulatory sequences underlying regions with allelic variation in chromatin accessibility revealed a paternally-associated sequence motif bound by ERF48, a target of the Polycomb repressive complex 2 (PRC2), and sequence similarity of this motif corresponded to local levels of H3K27me3, a signature of PRC2 activity. ConclusionsUsing a locally phased assembly of a heterozygous citrus cultivar, we dissected the interplay between genetic variants and molecular phenotypes with the goal of revealing functional cis-regulatory sequences and exploring the evolution of gene regulation.\n",
  "wordCount" : "2881",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576745">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576745" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576745">
        <p class="paperTitle">Randomizing the human genome by engineering recombination between repeat elements</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576745" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576745" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koeppel, J.; Ferreira, R.; Vanderstichele, T.; Riedmayr, L. M.; Peets, E. M.; Girling, G.; Weller, J.; Liberante, F. G.; Ellis, T.; Church, G. M.; Parts, L.</p>
        <p class="info">Score: 98.4, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576745' target='https://doi.org/10.1101/2024.01.22.576745'> 10.1101/2024.01.22.576745</a></p>
        <p class="abstract">While protein-coding genes are characterized increasingly well, 99% of the human genome is non-coding and poorly understood. This gap is due to a lack of tools for engineering variants that affect sequence to the necessary extent. To bridge this gap, we have developed a toolbox to create deletions, inversions, translocations, and extrachromosomal circular DNA at scale by highly multiplexed insertion of recombinase recognition sites into repetitive sequences with CRISPR prime editing. Using this strategy, we derived stable human cell lines with several thousand clonal insertions, the highest number of novel sequences inserted into single human genomes. Subsequent recombinase induction generated an average of more than one hundred megabase-sized rearrangements per cell, and thousands across the whole population. The ability to detect rearrangements as they are generated and to track their abundance over time allowed us to measure the selection pressures acting on different types of structural changes. We observed a consolidation towards shorter variants that preferentially delete growth-inhibiting genes and a depletion of translocations. We isolated and characterized 21 clones with multiple recombinase-induced rearrangements. These included viable haploid clones with deletions that span hundreds of kilobases as well as triploid HEK293T clones with aneuploidies and fold back chromosomes. We mapped the impact of these genetic changes on gene expression to decipher how structural variants affect gene regulation. The genome scrambling strategy developed here makes it possible to delete megabases of sequence, move sequences between and within chromosomes, and implant regulatory elements into new contexts which will shed light on the genome organization principles of humans and other species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576756" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576756">
        <p class="paperTitle">Multiplex generation and single cell analysis of structural variants in a mammalian genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pinglay, S.; Lalanne, J.-B.; Daza, R. M.; Koeppel, J.; Li, X.; Lee, D. S.; Shendure, J.</p>
        <p class="info">Score: 59.3, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576756' target='https://doi.org/10.1101/2024.01.22.576756'> 10.1101/2024.01.22.576756</a></p>
        <p class="abstract">The functional consequences of structural variants (SVs) in mammalian genomes are challenging to study. This is due to several factors, including: 1) their numerical paucity relative to other forms of standing genetic variation such as single nucleotide variants (SNVs) and short insertions or deletions (indels); 2) the fact that a single SV can involve and potentially impact the function of more than one gene and/or cis regulatory element; and 3) the relative immaturity of methods to generate SVs, either randomly or in targeted fashion, in in vitro or in vivo model systems. Towards addressing some of these challenges, we developed Genome-Shuffle-seq, a straightforward method that enables the multiplex generation of several major forms of SVs (deletions, inversions, translocations) throughout a mammalian genome. Genome-Shuffle-seq is based on the random integration of &#34;shuffle cassettes&#34; to the genome, wherein each shuffle cassette contains components that facilitate its site-specific recombination (SSR) with other integrated shuffle cassettes (via Cre-loxP), its mapping to a specific genomic location (via T7-mediated in vitro transcription or IVT), and its identification in single-cell RNA-seq (scRNA-seq) data (via T7-mediated in situ transcription or IST). In this proof-of-concept, we apply Genome-Shuffle-seq to induce and map thousands of genomic SVs in mouse embryonic stem cells (mESCs) in a single experiment. Induced SVs are rapidly depleted from the cellular population over time, possibly due to some combination of Cre-mediated toxicity and negative selection on the rearrangements themselves. We demonstrate that we can efficiently genotype which SVs are present in association with each of many single cell transcriptomes in scRNA-seq data. Finally, preliminary evidence suggests our method may be a powerful means of generating extrachromosomal circular DNAs (ecDNAs). Looking forward, we anticipate that Genome-Shuffle-seq may be broadly useful for the systematic exploration of the functional consequences of SVs on gene expression, the chromatin landscape, and 3D nuclear architecture. We further anticipate potential uses for in vitro modeling of ecDNAs, as well as in paving the path to a minimal mammalian genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577194">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577194" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577194">
        <p class="paperTitle">A Comprehensive Resource for Exploring Antiphage Defense: DefenseFinder Webservice, Wiki and Databases.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577194" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577194" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tesson, F.; Planel, R.; Egorov, A. A.; Georjon, H.; Vaysset, H.; Brancotte, B.; Neron, B.; Mordret, E.; Atkinson, G. C.; Bernheim, A.; Cury, J.</p>
        <p class="info">Score: 56.0, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577194' target='https://doi.org/10.1101/2024.01.25.577194'> 10.1101/2024.01.25.577194</a></p>
        <p class="abstract">In recent years, a vast number of novel antiphage defense mechanisms were uncovered. To facilitate the exploration of mechanistic, ecological, and evolutionary aspects related to antiphage defense systems, we released DefenseFinder in 2021 (Tesson et al., 2022). DefenseFinder is a bioinformatic program designed for the systematic identification of all known antiphage defense mechanisms. The initial release of DefenseFinder v1.0.0 included 60 systems. Over the past three years, the number of antiphage systems incorporated into DefenseFinder has grown to 152. The increasing number of known systems makes it a challenge to enter the field and makes the interpretation of detections of antiphage systems difficult. Moreover, the rapid development of sequence-based predictions of structures offers novel possibilities of analysis and should be easily available. To overcome these challenges, we present a hub of resources on defense systems including 1) a search tool (DefenseFinder as a web service) 2) a knowledge base, and 3) precomputed databases (results of DefenseFinder ran on RefSeq database, predicted structure database computed with AlphaFold). These pages can be freely accessed for users as a starting point on their journey to better understand a given system. We anticipate that these resources will foster the use of bioinformatics in the study of antiphage systems and will serve the community of researchers who study antiphage systems. This resource is available at: https://defensefinder.mdmlab.fr.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.29.576933">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.29.576933" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.29.576933">
        <p class="paperTitle">Systematic reconstruction of molecular pathway signatures using scalable single-cell perturbation screens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.29.576933" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.29.576933" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jiang, L.; Dalgarno, C.; Papalexi, E.; Mascio, I.; Wessels, H.-H.; Yun, H.; Iremadze, N.; Lithwick-Yanai, G.; Lipson, D.; Satija, R.</p>
        <p class="info">Score: 38.7, Published: 2024-01-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.29.576933' target='https://doi.org/10.1101/2024.01.29.576933'> 10.1101/2024.01.29.576933</a></p>
        <p class="abstract">Recent advancements in functional genomics have provided an unprecedented ability to measure diverse molecular modalities, but learning causal regulatory relationships from observational data remains challenging. Here, we leverage pooled genetic screens and single cell sequencing (i.e. Perturb-seq) to systematically identify the targets of signaling regulators in diverse biological contexts. We demonstrate how Perturb-seq is compatible with recent and commercially available advances in combinatorial indexing and next-generation sequencing, and perform more than 1,500 perturbations split across six cell lines and five biological signaling contexts. We introduce an improved computational framework (Mixscale) to address cellular variation in perturbation efficiency, alongside optimized statistical methods to learn differentially expressed gene lists and conserved molecular signatures. Finally, we demonstrate how our Perturb-seq derived gene lists can be used to precisely infer changes in signaling pathway activation for in-vivo and in-situ samples. Our work enhances our understanding of signaling regulators and their targets, and lays a computational framework towards the data-driven inference of an &#39;atlas&#39; of perturbation signatures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576850">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576850" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576850">
        <p class="paperTitle">A high coverage Mesolithic aurochs genome and effective leveraging of ancient cattle genomes using whole genome imputation.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576850" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576850" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Erven, J. A. M.; Scheu, A.; Verdugo, M. P.; Cassidy, L.; Chen, N.; Gehlen, B.; Street, M.; Madsen, O.; Mullin, V. E.</p>
        <p class="info">Score: 16.4, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576850' target='https://doi.org/10.1101/2024.01.23.576850'> 10.1101/2024.01.23.576850</a></p>
        <p class="abstract">Ancient genomic analyses are often restricted to utilising pseudo-haploid data due to low genome coverage. Leveraging low coverage data by imputation to calculate phased diploid genotypes that enable haplotype-based interrogation and SNP calling at unsequenced positions is highly desirable. This has not been investigated for ancient cattle genomes despite these being compelling subjects for archaeological, evolutionary and economic reasons. Here we test this approach by sequencing a Mesolithic European aurochs (18.49x; 9852-9376 calBC), an Early Medieval European cow (18.69x; 427-580 calAD), and combine these with published individuals; two ancient and three modern. We downsample these genomes (0.25x, 0.5x, 1.0x, 2.0x) and impute diploid genotypes, utilising a reference panel of 171 published modern cattle genomes that we curated for 21.7 million (Mn) phased single-nucleotide polymorphisms (SNPs). We recover high densities of correct calls with an accuracy of &gt;99.1% at variant sites for the lowest downsample depth of 0.25x, increasing to &gt;99.5% for 2.0x (transversions only, minor allele frequency (MAF) [&amp;ge;]2.5%). The recovery of SNPs correlates with coverage, on average 58% of sites are recovered for 0.25x increasing to 87% for 2.0x, utilising an average of 3.5 million (Mn) transversions (MAF [&amp;ge;]2.5%), even in the aurochs which is temporally and morphologically distinct from the reference panel. Our imputed genomes behave similarly to directly called data in allele-frequency-based analyses; for example consistently identifying runs of homozygosity &gt;2mb, including a long homozygous region in the Mesolithic European aurochs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.22.576737">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.22.576737" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.22.576737">
        <p class="paperTitle">Whole-genome phylogenomics of the tinamous (Aves: Tinamidae): comparing gene tree estimation error between BUSCOs and UCEs illuminates rapid divergence with introgression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.22.576737" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.22.576737" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Musher, L. J.; Catanach, T. A.; Valqui, T.; Brumfield, R. T.; Aleixo, A.; Johnson, K. P.; Weckstein, J. D.</p>
        <p class="info">Score: 15.6, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.22.576737' target='https://doi.org/10.1101/2024.01.22.576737'> 10.1101/2024.01.22.576737</a></p>
        <p class="abstract">Incomplete lineage sorting (ILS) and introgression increase genealogical discordance across the genome, which complicates phylogenetic inference. In such cases, identifying orthologs that result in gene trees with low estimation error is crucial because phylogenomic methods rely on accurate gene histories. We sequenced whole genomes for the tinamous (Aves: Tinamidae) to dissect the sources of gene and species-tree discordance and reconstruct their interrelationships. We compared results based on four ortholog sets: (1) coding genes (BUSCOs), (2) ultraconserved elements (UCEs) with short flanking regions, (3) UCEs with intermediate flanks, and (4) UCEs with long flanks. We hypothesized that orthologs with more phylogenetically informative sites would result in more accurate species trees because the resulting gene trees contain lower error. Consistent with our hypothesis, we found that long UCEs had the most informative sites and lowest rates of error. However, despite having many informative sites, BUSCO gene trees contained high error compared to long UCEs. Unlike UCEs, BUSCO gene sequences showed a positive association between the proportion of parsimony informative sites and gene tree error. Thus, BUSCO and UCE datasets have different underlying properties of molecular evolution, and these differences should be considered when selecting loci for phylogenomic analysis. Still, species trees from different datasets were mostly congruent. Only one clade, with a history of ILS and introgression, exhibited substantial species-tree discordance across the different data sets. Overall, we present the most complete phylogeny for tinamous to date, identify a new species, and provide a case study for species-level phylogenomic analysis using whole-genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.25.577066">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.25.577066" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.25.577066">
        <p class="paperTitle">10x Genomics Gene Expression Flex is a powerful tool for single-cell transcriptomics of xenograft models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.25.577066" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.25.577066" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Llora-Batlle, O.; Farcas, A.; Fransen, D.; Floch, N.; Talbot, S.; Schwiening, A.; Bojko, L.; Calver, J.; Josipovic, N.; Lashuk, K.; Schueler, J.; Prodan, A.; Mooijman, D.; McDermott, U.; PERSIST-SEQ Consortium,  </p>
        <p class="info">Score: 11.0, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.25.577066' target='https://doi.org/10.1101/2024.01.25.577066'> 10.1101/2024.01.25.577066</a></p>
        <p class="abstract">The 10x Genomics Gene Expression Flex protocol allows profiling of fixed or frozen material, greatly simplifying the logistics of sample collection, storage and transfer prior to single -cell sequencing. The method makes single-cell transcriptomics possible for existing fresh-frozen or FFPE tissue samples, but also facilitates the logistics of the sampling process, allowing instant preservation of samples. The technology relies on species-specific probes available for human and mouse. Nevertheless, processing of patient-derived (PDX) or cell line (CDX) xenografts, which contain mixed human and mouse cells, is currently not supported by this protocol due to the high degree of homology between the probe sets. Here we show that it is feasible to simultaneously profile populations containing both human and mouse cells by mixing the transcriptome probe sets of both species. Cellranger outputs a count table for each of the species allowing evaluation of the performance of the different probe sets. Cross-reactive probes are greatly outperformed by the specific probe hybridizations leading to a clear difference in the recovery of UMIs and unique genes per cell. Furthermore, we developed a pipeline that removes cross-reactive signal from the data and provides species-specific count tables for further downstream analysis. Hence, the 10x Genomics Gene Expression Flex protocol can be used to process xenograft samples without the need for separation of human and mouse cells by flow sorting and allows analysis of the human and mouse single-cell transcriptome from each sample. We anticipate it will be increasingly used for single-cell sequencing of cancer cell line and patient-derived xenografts, facilitating the preservation of the samples and allowing the interrogation of both the (human) xenograft and the (mouse) tumor microenvironment at single-cell resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576123" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576123">
        <p class="paperTitle">Multiple Displacement Amplification Facilitates SMRT Sequencing of Microscopic Animals and the Genome of the Gastrotrich Lepidodermella squamata (Dujardin, 1841)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roberts, N. G.; Gilmore, M. J.; Struck, T. H.; Kocot, K.</p>
        <p class="info">Score: 11.2, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576123' target='https://doi.org/10.1101/2024.01.17.576123'> 10.1101/2024.01.17.576123</a></p>
        <p class="abstract">BackgroundObtaining adequate DNA for long-read genome sequencing remains a roadblock to producing contiguous genomes from small-bodied organisms. Multiple displacement amplification (MDA) leverages Phi29 DNA polymerase to produce micrograms of DNA from picograms of input. Few genomes have been generated using this approach, due to concerns over biases in amplification related to GC and repeat content and chimera production. Here, we explored the utility of MDA for generating template DNA for PacBio HiFi sequencing using Caenorhabditis elegans (Nematoda) and Lepidodermella squamata (Gastrotricha).

ResultsHiFi sequencing of libraries prepared from MDA DNA produced highly contiguous and complete genomes for both C. elegans (102 Mbp assembly; 336 contigs; N50 = 868 Kbp; L50 = 39; BUSCO_nematoda: S:92.2%, D:2.7%) and L. squamata (122 Mbp assembly; 157 contigs; N50 = 3.9 Mb; L50 = 13; BUSCO_metazoa: S: 78.0%, D: 2.8%). Amplified C. elegans reads mapped to the reference genome with a rate of 99.92% and coverage of 99.75% with just one read (of 708,811) inferred to be chimeric. Coverage uniformity was nearly identical for reads from MDA DNA and reads from pooled worm DNA when mapped to the reference genome. The genome of Lepidodermella squamata, the first of its phylum, was leveraged to infer the phylogenetic position of Gastrotricha, which has long been debated, as the sister taxon of Platyhelminthes.

ConclusionsThis methodology will help generate contiguous genomes of microscopic taxa whose body size precludes standard long-read sequencing. L. squamata is an emerging model in evolutionary developmental biology and this genome will facilitate further work on this species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.23.576914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.23.576914" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.23.576914">
        <p class="paperTitle">Repeated horizontal acquisition of lagriamide-producing symbionts in Lagriinae beetles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.23.576914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.23.576914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uppal, S.; Waterworth, S. C.; Nick, A.; Vogel, H.; Florez, L.; Kaltenpoth, M.; Kwan, J. C.</p>
        <p class="info">Score: 8.4, Published: 2024-01-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.23.576914' target='https://doi.org/10.1101/2024.01.23.576914'> 10.1101/2024.01.23.576914</a></p>
        <p class="abstract">Microbial symbionts associate with multicellular organisms on a continuum from facultative associations to mutual codependency. In the oldest symbioses there is exclusive vertical symbiont transmission, and extensive co-diversification of symbiotic partners over millions of years and speciation events. Such symbionts often undergo genome reduction due to low effective populations, frequent population bottlenecks and weakened purifying selection. We describe here a group of Lagriinae beetle symbionts, which produce defensive lagriamide molecules to protect beetle eggs and larvae from fungal infection. These beetles likely evolved specialised symbiont-storage structures between 55 and 82 million years ago. Previous work found, despite the lagriamide-producing bacterium not being genetically isolated and potentially surviving in the environment, it underwent genome reduction in Lagria villosa hosts. Here, we use shotgun metagenomics to assemble 11 additional lagriamide-producing symbionts from diverse hosts within Lagriinae from five countries, to clarify the evolution of this symbiotic relationship. Surprisingly, we did not find evidence of co-evolution between symbionts and hosts, and although the lagriamide gene cluster (lga) had been horizontally transferred to the L. villosa symbiont, it likely was not transferred between symbionts. Instead, lga was likely obtained only once in the common ancestor of all lagriamide-producing symbionts and subsequently lost in related free-living strains. Our results suggest that beetles acquired lagriamide-producing symbionts specifically multiple times independently, which was followed by genome erosion. We also show that these symbionts have been replaced in the beetle host multiple times. The system therefore offers a unique opportunity to study the process of genome reduction.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.575729">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.575729" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.575729">
        <p class="paperTitle">Haplotype phased genome of &#39;Fairchild&#39; mandarin highlights influence of local chromatin state on gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.575729" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.575729" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Diaz, I. A.; Ostovar, T.; Chen, J.; Saddoris, S.; Schmitz, R. J.; Wessler, S. R.; Stajich, J.; Seymour, D. K.</p>
        <p class="info">Score: 6.9, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.575729' target='https://doi.org/10.1101/2024.01.20.575729'> 10.1101/2024.01.20.575729</a></p>
        <p class="abstract">BackgroundCis-regulatory sequences control gene expression through the coordinated action of transcription factors and their associated partners. Both genetic and epigenetic perturbation of cis-regulatory sequences can lead to novel patterns of gene expression. Phased genome assemblies now enable the local dissection of linkages between cis-regulatory sequences, including their epigenetic state, and gene expression to further characterize gene regulation in heterozygous genomes.

ResultsWe assembled a locally phased genome for a mandarin hybrid named  Fairchild to explore the molecular signatures of allele-specific gene expression. With genome phasing, genes with allele-specific expression were paired with haplotype-specific chromatin states, including levels of chromatin accessibility, histone modifications, and DNA methylation. We found that 30% of variation in allele-specific expression could be attributed to haplotype associated factors, with allelic levels of chromatin accessibility and three histone modifications in gene bodies having the most influence. Structural variants in promoter regions were also associated with allele-specific expression, including specific enrichments of hAT and MULE-MuDR DNA transposon sequences. Mining of cis-regulatory sequences underlying regions with allelic variation in chromatin accessibility revealed a paternally-associated sequence motif bound by ERF48, a target of the Polycomb repressive complex 2 (PRC2), and sequence similarity of this motif corresponded to local levels of H3K27me3, a signature of PRC2 activity.

ConclusionsUsing a locally phased assembly of a heterozygous citrus cultivar, we dissected the interplay between genetic variants and molecular phenotypes with the goal of revealing functional cis-regulatory sequences and exploring the evolution of gene regulation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
