<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Comparative analysis of multiplexed in situ gene expression profiling technologies
Authors: Hartman, A.; Satija, R.
Score: 138.0, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135
The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Comparative analysis of multiplexed in situ gene expression profiling technologies
Authors: Hartman, A.; Satija, R.
Score: 138.0, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135
The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-21T10:39:08+00:00" />
<meta property="article:modified_time" content="2024-01-21T10:39:08+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Comparative analysis of multiplexed in situ gene expression profiling technologies
Authors: Hartman, A.; Satija, R.
Score: 138.0, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135
The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Comparative analysis of multiplexed in situ gene expression profiling technologies\nAuthors: Hartman, A.; Satija, R.\nScore: 138.0, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135\nThe burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets.",
  "keywords": [
    
  ],
  "articleBody": " Comparative analysis of multiplexed in situ gene expression profiling technologies\nAuthors: Hartman, A.; Satija, R.\nScore: 138.0, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575135\nThe burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets. We find that standard sensitivity metrics, such as the number of unique molecules detected per cell, are not directly comparable across datasets due to substantial differences in the incidence of off-target molecular artifacts impacting specificity. To address these challenges, we explored various potential sources of molecular artifacts, developed novel metrics to control for them, and utilized these metrics to evaluate and compare different in situ technologies. Finally, we demonstrate how molecular false positives can seriously confound spatially-aware differential expression analysis, requiring caution in the interpretation of downstream results. Our analysis provides guidance for the selection, processing, and interpretation of in situ spatial technologies.\nScalable ultra-high-throughput single-cell chromatin and RNA sequencing reveals gene regulatory dynamics linking macrophage polarization to autoimmune disease\nAuthors: Lobato-Moreno, S.; Yildiz, U.; Claringbould, A.; Servaas, N. H.; Vlachou, E. P.; Arnold, C.; Bauersachs, H. G.; Campos-Fornes, V.; Prummel, K. D.; Noh, K. M.; Marttinen, M.; Zaugg, J. B.\nScore: 48.5, Published: 2023-12-26 DOI: 10.1101/2023.12.26.573253\nEnhancers and transcription factors (TFs) are crucial in regulating cellular processes, including disease-associated cell states. Current multiomic technologies to study these elements in gene regulatory mechanisms lack multiplexing capability and scalability. Here, we present SUM-seq, a cost-effective, scalable Single-cell Ultra-high-throughput Multiomic sequencing method for co-assaying chromatin accessibility and gene expression in single nuclei. SUM-seq enables profiling hundreds of samples at the million cell scale and outperforms current high-throughput single-cell methods. We applied SUM-seq to dissect the gene regulatory mechanisms governing macrophage polarization and explored their link to traits from genome-wide association studies (GWAS). Our analyses confirmed known TFs orchestrating M1 and M2 macrophage programs, unveiled key regulators, and demonstrated extensive enhancer rewiring. Integration with GWAS data further pinpointed the impact of specific TFs on a set of immune traits. Notably, inferred enhancers regulated by the STAT1/STAT2/IRF9 (ISGF3) complex were enriched for rheumatoid arthritis-associated genetic variants, and their target genes included known drug targets. This highlights the potential of SUM-seq for dissecting molecular disease mechanisms. SUM-seq offers a cost-effective, scalable solution for ultra-high-throughput single-cell multiomic sequencing, excelling in unraveling complex gene regulatory networks in cell differentiation, responses to perturbations, and disease studies.\nNatural diversity of heat-induced transcription of retrotransposons in Arabidopsis thaliana\nAuthors: Xu, W.; Thieme, M.; Roulin, A.\nScore: 11.7, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575637\nTransposable elements (TEs) are major components of plant genomes, profoundly impacting the fitness of their hosts. However, technical bottlenecks have long hindered our mechanistic understanding of TEs. Using RNA-Seq and long-read sequencing with Oxford Nanopore Technologies direct cDNA sequencing, we analyzed the heat-induced transcription of TEs in three natural accessions of Arabidopsis thaliana (Cvi-0, Col-0, and Ler-1). In addition to the well- studied ONSEN retrotransposon family, we identified Copia-35 as a second heat-responsive retrotransposon family with particularly high activity in the relict accession Cvi-0. Our analysis revealed distinct expression patterns of individual TE copies and suggest different mechanisms regulating the GAG protein production in the ONSEN versus Copia-35 families. In addition, analogously to ONSEN, Copia-35 activation led to the upregulation of flanking genes such as AMUP9 and potentially to the quantitative modulation of flowering time. Unexpectedly, our results indicate that for both families, the upregulation of flanking genes is not directly initiated by transcription from their 3 LTRs. These findings highlight the inter- and intraspecific expressional diversity linked to retrotransposon activation under stress, providing insights into their potential roles in plant adaptation and evolution at elevated temperatures.\nDORA: an interactive map for the visualization and analysis of ancient human DNA and associated data\nAuthors: Harris, K. D.; Greenbaum, G.\nScore: 7.5, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575663\nThe ability to sequence ancient genomes has revolutionized the way in which we study evolutionary history by providing access to the most important aspect of evolution -- time. Until recently, studying human demography, ecology, biology, and history using population genomic inference relied on contemporary genomic datasets. Over the past decade, the availability of human ancient DNA (aDNA) has increased rapidly, almost doubling every year, opening the way for spatiotemporal studies of ancient human populations. However, the multidimensionality of aDNA, with genotypes having temporal, spatial and genomic coordinates, and the need to integrate multiple sources of data, poses a challenge for developing meta-analyses pipelines. To address this challenge, we developed a publicly-available interactive tool, DORA, which integrates multiple data types, genomic and non-genomic, in a unified interface. This web-based tool allows users to browse sample metadata along with additional layers of information, such as population structure, climatic data, and unpublished samples. Users can then perform analyses on genotypes of these samples, or export sample subsets for external analyses. DORA integrates analyses and visualizations in a single intuitive interface, resolving the technical issues of combining datasets from different sources and formats, and allowing researchers to focus on analysis and the scientific questions that can be addressed through analysis of aDNA datasets.\nGenomic epidemiology of Candida auris introduction and outbreaks in the United Kingdom\nAuthors: Kappel, D.; Gifford, H.; Brackin, A. P.; Abdolrasouli, A.; Eyre, D. W.; Jeffery, K.; Schelenz, S.; Aanensen, D.; Brown, C.; Borman, A.; Johnson, E.; Holmes, A.; Armstrong-James, D.; Fisher, M.; Rhodes, J.\nScore: 4.8, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575049\nBackgroundCandida auris is a globally emerging fungal pathogen that causes nosocomial invasive infections, particularly in intensive care units (ICU). Four prominent genetic clades originating from distinct geographic regions have been identified: South Asia (Clade I), East Asia (Clade II), Africa (Clade III) and South America (Clade IV) with each clade presenting differing antifungal resistance profiles. We aimed to elucidate the epidemiology of this infection in the United Kingdom (UK) 2014 - 2019 using genomic approaches. MethodsGenome sequences from 24 isolates from six patients in four London hospitals were combined with genome sequences from 183 additional outbreak isolates from nine UK hospitals. These data were used to assess the numbers and timings of C. auris lineage introductions into the UK and to infer subsequent patterns of inter- and intra-hospital transmission. FindingsWe found evidence for at least three introductions of C. auris into the UK, one from Clade I and one from Clade III. The evolutionary rate of C. auris Clade I and Clade III were estimated at 2.764 x 10-4 and 3.186 x 10-4 substitutions per polymorphic site per year, respectively, with entry into the UK as 2013 and 2016 for Clades I and III respectively. We observed temporal and geographical evidence for multiple transmission events between hospitals and identified local within-hospital patient-to-patient transmission events. InterpretationThese data confirm that C. auris is a newly emerged infection in the UK with at least three original introductions into this country. Our analysis shows that hospital outbreaks are linked and confirms that transmission amongst patients has occurred, explaining local hospital outbreaks. Our findings argue that enhanced surveillance of C. auris infection within the UK is necessary in order to protect healthcare and to curtail expansion of this emerging infection. FundingThis work was supported by a Wellcome Trust Institutional Strategic Support Fund Springboard Fellowship, and by the Medical Research Council (MR/R015600/1), the Natural Environment Research Council (NE/P001165/1) and the Wellcome Trust (219551/Z/19/Z). Research in contextSince its first description in 2009, Candida auris has spread across the globe. For this study, authors considered all publications describing whole genome sequences of isolates within the UK up until 2020 to assess the genomic epidemiology of this fungal pathogen. This study adds value to the current understanding of C. auris epidemiology by undertaking the first analysis to assess transmission between and within hospitals using genomic data. We also provide the first in-depth temporal analysis to estimate likely times of introduction into the UK. These results have clinical implications, encouraging hospitals to identify yeast upon admission and to assess multiple isolates from single patients, which may harbour much diversity in terms of genetics and drug resistance.\nDual DNA/RNA-binding factor regulates dynamics of hnRNP splicing condensates\nAuthors: Ray, M.; Zaborowsky, J.; Mahableshwarkar, P.; Vaidyanathan, S.; Shum, J.; Viswanathan, R.; Huang, A.; Wang, S.-H.; Johnson, V.; Wake, N.; Conard, A. M.; Conicella, A. E.; Puterbaugh, R.; Fawzi, N. L.; Larschan, E.\nScore: 7.8, Published: 2024-01-12 DOI: 10.1101/2024.01.11.575216\nDespite decades of research, mechanisms by which co-transcriptional alternative splicing events are targeted to the correct genomic locations to drive cell fate decisions remain unknown. By combining structural and molecular approaches, we define a new mechanism by which an essential transcription factor (TF) targets co-transcriptional splicing through physical and functional interaction with RNA and RNA binding proteins (RBPs). We show that an essential TF co-transcriptionally regulates sex-specific alternative splicing by directly interacting with a subset of target RNAs on chromatin and modulating the dynamics of hnRNPA2 homolog nuclear splicing condensates.\nUncovering the genetic architecture and evolutionary roots of androgenetic alopecia in African men\nAuthors: Janivara, R.; Hazra, U.; Pfennig, A.; Harlemon, M.; Kim, M. S.; Eaaswarkhanth, M.; Chen, W. C.; Ogunbiyi, A.; Kachambwa, P.; Petersen, L. N.; Jalloh, M.; Mensah, J. E.; Adjei, A. A.; Adusei, B.; Joffe, M.; Gueye, S. M.; Aisuodionoe-Shadrach, O. I.; Fernandez, P. W.; Rohan, T. E.; Andrews, C.; Rebbeck, T. R.; Adebiyi, A. O.; Agalliu, I.; Lachance, J.\nScore: 4.1, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575396\nAndrogenetic alopecia is a highly heritable trait. However, much of our understanding about the genetics of male pattern baldness comes from individuals of European descent. Here, we examined a novel dataset comprising 2,136 men from Ghana, Nigeria, Senegal, and South Africa that were genotyped using a custom array. We first tested how genetic predictions of baldness generalize from Europe to Africa, finding that polygenic scores from European GWAS yielded AUC statistics that ranged from 0.513 to 0.546, indicating that genetic predictions of baldness in African populations performed notably worse than in European populations. Subsequently, we conducted the first African GWAS of androgenetic alopecia, focusing on self-reported baldness patterns at age 45. After correcting for present age, population structure, and study site, we identified 266 moderately significant associations, 51 of which were independent (p-value \u003c 10-5, r2 \u003c 0.2). Most baldness associations were autosomal, and the X chromosomes does not appear to have a large impact on baldness in African men. Finally, we examined the evolutionary causes of continental differences in genetic architecture. Although Neanderthal alleles have previously been associated with skin and hair phenotypes, we did not find evidence that European-ascertained baldness hits were enriched for signatures of ancient introgression. Most loci that are associated with androgenetic alopecia are evolving neutrally. However, multiple baldness-associated SNPs near the EDA2R and AR genes have large allele frequency differences between continents. Collectively, our findings illustrate how evolutionary history contributes to the limited portability of genetic predictions across ancestries.\nFAK and p130Cas modulate stiffness-mediated early transcription and cellular metabolism\nAuthors: Tumenbayar, B.-I.; Tutino, V. M.; Brazzo, J. A.; Yao, P.; Bae, Y.\nScore: 3.8, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575789\nCellular metabolism is influenced by the stiffness of the extracellular matrix. Focal adhesion kinase (FAK) and its binding partner, p130Cas, transmit biomechanical signals about substrate stiffness to the cell to regulate a variety of cellular responses, but their roles in early transcriptional and metabolic responses remain largely unexplored. We cultured mouse embryonic fibroblasts with or without siRNA-mediated FAK or p130Cas knockdown and assessed the early transcriptional responses of these cells to placement on soft and stiff substrates by RNA sequencing and bioinformatics analyses. Exposure to the stiff ECM altered the expression of genes important for metabolic and biosynthetic processes, and these responses were influenced by knockdown of FAK and p130Cas. Our findings reveal that FAK-p130Cas signaling mechanotransduces ECM stiffness to early transcriptional changes that alter cellular metabolism and biosynthesis.\nGenome assembly of the milky mangrove Excoecaria agallocha\nAuthors: Hong Kong Biodiversity Genomics Consortium, ; Hui, J. H. L.; Chan, T.-F.; Chan, L.; Cheung, S. G.; Cheang, C. C.; Fang, J.; Gaitan-Espitia, J. D.; Lau, S.; Sung, Y. H.; Wong, C.; Yip, K.; Wei, Y.; law, S. T. S.; So, W. L.; Nong, W.; Lau, D.; Lee, S. Y.; Yip, H. Y.\nScore: 3.8, Published: 2024-01-15 DOI: 10.1101/2024.01.13.575302\nThe milky mangrove Excoecaria agallocha is a latex-secreting mangrove that are distributed in tropical and subtropical regions. While its poisonous latex is regarded as a potential source of phytochemicals for biomedical applications, the genomic resources of E. agallocha remains limited. Here, we present a chromosomal level genome of E. agallocha, assembled from the combination of PacBio long-read sequencing and Omni-C data. The resulting assembly size is 1,332.45 Mb and has high contiguity and completeness with a scaffold N50 of 58.9 Mb and a BUSCO score of 98.4 %. 73,740 protein-coding genes were also predicted. The milky mangrove genome provides a useful resource for further understanding the biosynthesis of phytochemical compounds in E. agallocha.\nDenisovan admixture facilitated environmental adaptation in Papua New Guinean populations\nAuthors: Yermakovich, D.; Andre, M.; Brucato, N.; Kariwiga, J.; Leavesley, M.; Pankratov, V.; Mondal, M.; Ricaut, F.-X.; Dannemann, M.\nScore: 3.5, Published: 2024-01-14 DOI: 10.1101/2024.01.14.575483\nNeandertals and Denisovans, having inhabited distinct regions in Eurasia and possibly Oceania for over 200,000 years, experienced ample time to adapt to diverse environmental challenges these regions presented. Among present-day human populations, Papua New Guineans (PNG) stand out as one of the few carrying substantial amounts of both Neandertal and Denisovan DNA, a result of past admixture events with these archaic human groups. This study investigates the distribution of introgressed Denisovan and Neandertal DNA within two distinct PNG populations, residing in the highlands of Mt Wilhelm and the lowlands of Daru Island. These locations exhibit unique environmental features, some of which may parallel the challenges that archaic humans once confronted and adapted to. Our results show that Denisovan-like haplotypes exhibit increased levels of population differentiation between PNG highlanders and lowlanders. The highly differentiated haplotypes, more common among highlanders, reside in genomic areas linked to brain development genes. Conversely, those more frequent in lowlanders overlap with genes enriched in immune response processes. Furthermore, Denisovan-like haplotypes displayed pronounced signatures of diversification within the major histocompatibility complex. Our findings suggest that Denisovan DNA has provided a valuable source of genetic variation to PNG genomes that facilitated adaptive responses to environmental challenges.\n",
  "wordCount" : "2390",
  "inLanguage": "en",
  "datePublished": "2024-01-21T10:39:08Z",
  "dateModified": "2024-01-21T10:39:08Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 21, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575135" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575135">
        <p class="paperTitle">Comparative analysis of multiplexed in situ gene expression profiling technologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hartman, A.; Satija, R.</p>
        <p class="info">Score: 138.0, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575135' target='https://doi.org/10.1101/2024.01.11.575135'> 10.1101/2024.01.11.575135</a></p>
        <p class="abstract">The burgeoning interest in in situ multiplexed gene expression profiling technologies has opened new avenues for understanding cellular behavior and interactions. In this study, we present a comparative benchmark analysis of six in situ gene expression profiling methods, including both commercially available and academically developed methods, using publicly accessible mouse brain datasets. We find that standard sensitivity metrics, such as the number of unique molecules detected per cell, are not directly comparable across datasets due to substantial differences in the incidence of off-target molecular artifacts impacting specificity. To address these challenges, we explored various potential sources of molecular artifacts, developed novel metrics to control for them, and utilized these metrics to evaluate and compare different in situ technologies. Finally, we demonstrate how molecular false positives can seriously confound spatially-aware differential expression analysis, requiring caution in the interpretation of downstream results. Our analysis provides guidance for the selection, processing, and interpretation of in situ spatial technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573253">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573253" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573253">
        <p class="paperTitle">Scalable ultra-high-throughput single-cell chromatin and RNA sequencing reveals gene regulatory dynamics linking macrophage polarization to autoimmune disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573253" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573253" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lobato-Moreno, S.; Yildiz, U.; Claringbould, A.; Servaas, N. H.; Vlachou, E. P.; Arnold, C.; Bauersachs, H. G.; Campos-Fornes, V.; Prummel, K. D.; Noh, K. M.; Marttinen, M.; Zaugg, J. B.</p>
        <p class="info">Score: 48.5, Published: 2023-12-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573253' target='https://doi.org/10.1101/2023.12.26.573253'> 10.1101/2023.12.26.573253</a></p>
        <p class="abstract">Enhancers and transcription factors (TFs) are crucial in regulating cellular processes, including disease-associated cell states. Current multiomic technologies to study these elements in gene regulatory mechanisms lack multiplexing capability and scalability. Here, we present SUM-seq, a cost-effective, scalable Single-cell Ultra-high-throughput Multiomic sequencing method for co-assaying chromatin accessibility and gene expression in single nuclei. SUM-seq enables profiling hundreds of samples at the million cell scale and outperforms current high-throughput single-cell methods. We applied SUM-seq to dissect the gene regulatory mechanisms governing macrophage polarization and explored their link to traits from genome-wide association studies (GWAS). Our analyses confirmed known TFs orchestrating M1 and M2 macrophage programs, unveiled key regulators, and demonstrated extensive enhancer rewiring. Integration with GWAS data further pinpointed the impact of specific TFs on a set of immune traits. Notably, inferred enhancers regulated by the STAT1/STAT2/IRF9 (ISGF3) complex were enriched for rheumatoid arthritis-associated genetic variants, and their target genes included known drug targets. This highlights the potential of SUM-seq for dissecting molecular disease mechanisms. SUM-seq offers a cost-effective, scalable solution for ultra-high-throughput single-cell multiomic sequencing, excelling in unraveling complex gene regulatory networks in cell differentiation, responses to perturbations, and disease studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575637" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575637">
        <p class="paperTitle">Natural diversity of heat-induced transcription of retrotransposons in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Xu, W.; Thieme, M.; Roulin, A.</p>
        <p class="info">Score: 11.7, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575637' target='https://doi.org/10.1101/2024.01.15.575637'> 10.1101/2024.01.15.575637</a></p>
        <p class="abstract">Transposable elements (TEs) are major components of plant genomes, profoundly impacting the fitness of their hosts. However, technical bottlenecks have long hindered our mechanistic understanding of TEs. Using RNA-Seq and long-read sequencing with Oxford Nanopore Technologies direct cDNA sequencing, we analyzed the heat-induced transcription of TEs in three natural accessions of Arabidopsis thaliana (Cvi-0, Col-0, and Ler-1). In addition to the well- studied ONSEN retrotransposon family, we identified Copia-35 as a second heat-responsive retrotransposon family with particularly high activity in the relict accession Cvi-0. Our analysis revealed distinct expression patterns of individual TE copies and suggest different mechanisms regulating the GAG protein production in the ONSEN versus Copia-35 families. In addition, analogously to ONSEN, Copia-35 activation led to the upregulation of flanking genes such as AMUP9 and potentially to the quantitative modulation of flowering time. Unexpectedly, our results indicate that for both families, the upregulation of flanking genes is not directly initiated by transcription from their 3 LTRs. These findings highlight the inter- and intraspecific expressional diversity linked to retrotransposon activation under stress, providing insights into their potential roles in plant adaptation and evolution at elevated temperatures.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575663" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575663">
        <p class="paperTitle">DORA: an interactive map for the visualization and analysis of ancient human DNA and associated data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Harris, K. D.; Greenbaum, G.</p>
        <p class="info">Score: 7.5, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575663' target='https://doi.org/10.1101/2024.01.15.575663'> 10.1101/2024.01.15.575663</a></p>
        <p class="abstract">The ability to sequence ancient genomes has revolutionized the way in which we study evolutionary history by providing access to the most important aspect of evolution -- time. Until recently, studying human demography, ecology, biology, and history using population genomic inference relied on contemporary genomic datasets. Over the past decade, the availability of human ancient DNA (aDNA) has increased rapidly, almost doubling every year, opening the way for spatiotemporal studies of ancient human populations. However, the multidimensionality of aDNA, with genotypes having temporal, spatial and genomic coordinates, and the need to integrate multiple sources of data, poses a challenge for developing meta-analyses pipelines. To address this challenge, we developed a publicly-available interactive tool, DORA, which integrates multiple data types, genomic and non-genomic, in a unified interface. This web-based tool allows users to browse sample metadata along with additional layers of information, such as population structure, climatic data, and unpublished samples. Users can then perform analyses on genotypes of these samples, or export sample subsets for external analyses. DORA integrates analyses and visualizations in a single intuitive interface, resolving the technical issues of combining datasets from different sources and formats, and allowing researchers to focus on analysis and the scientific questions that can be addressed through analysis of aDNA datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575049" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575049">
        <p class="paperTitle">Genomic epidemiology of Candida auris introduction and outbreaks in the United Kingdom</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kappel, D.; Gifford, H.; Brackin, A. P.; Abdolrasouli, A.; Eyre, D. W.; Jeffery, K.; Schelenz, S.; Aanensen, D.; Brown, C.; Borman, A.; Johnson, E.; Holmes, A.; Armstrong-James, D.; Fisher, M.; Rhodes, J.</p>
        <p class="info">Score: 4.8, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575049' target='https://doi.org/10.1101/2024.01.15.575049'> 10.1101/2024.01.15.575049</a></p>
        <p class="abstract">BackgroundCandida auris is a globally emerging fungal pathogen that causes nosocomial invasive infections, particularly in intensive care units (ICU). Four prominent genetic clades originating from distinct geographic regions have been identified: South Asia (Clade I), East Asia (Clade II), Africa (Clade III) and South America (Clade IV) with each clade presenting differing antifungal resistance profiles. We aimed to elucidate the epidemiology of this infection in the United Kingdom (UK) 2014 - 2019 using genomic approaches.

MethodsGenome sequences from 24 isolates from six patients in four London hospitals were combined with genome sequences from 183 additional outbreak isolates from nine UK hospitals. These data were used to assess the numbers and timings of C. auris lineage introductions into the UK and to infer subsequent patterns of inter- and intra-hospital transmission.

FindingsWe found evidence for at least three introductions of C. auris into the UK, one from Clade I and one from Clade III. The evolutionary rate of C. auris Clade I and Clade III were estimated at 2.764 x 10-4 and 3.186 x 10-4 substitutions per polymorphic site per year, respectively, with entry into the UK as 2013 and 2016 for Clades I and III respectively. We observed temporal and geographical evidence for multiple transmission events between hospitals and identified local within-hospital patient-to-patient transmission events.

InterpretationThese data confirm that C. auris is a newly emerged infection in the UK with at least three original introductions into this country. Our analysis shows that hospital outbreaks are linked and confirms that transmission amongst patients has occurred, explaining local hospital outbreaks. Our findings argue that enhanced surveillance of C. auris infection within the UK is necessary in order to protect healthcare and to curtail expansion of this emerging infection.

FundingThis work was supported by a Wellcome Trust Institutional Strategic Support Fund Springboard Fellowship, and by the Medical Research Council (MR/R015600/1), the Natural Environment Research Council (NE/P001165/1) and the Wellcome Trust (219551/Z/19/Z).

Research in contextSince its first description in 2009, Candida auris has spread across the globe. For this study, authors considered all publications describing whole genome sequences of isolates within the UK up until 2020 to assess the genomic epidemiology of this fungal pathogen. This study adds value to the current understanding of C. auris epidemiology by undertaking the first analysis to assess transmission between and within hospitals using genomic data. We also provide the first in-depth temporal analysis to estimate likely times of introduction into the UK. These results have clinical implications, encouraging hospitals to identify yeast upon admission and to assess multiple isolates from single patients, which may harbour much diversity in terms of genetics and drug resistance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.11.575216">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.11.575216" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.11.575216">
        <p class="paperTitle">Dual DNA/RNA-binding factor regulates dynamics of hnRNP splicing condensates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.11.575216" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.11.575216" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ray, M.; Zaborowsky, J.; Mahableshwarkar, P.; Vaidyanathan, S.; Shum, J.; Viswanathan, R.; Huang, A.; Wang, S.-H.; Johnson, V.; Wake, N.; Conard, A. M.; Conicella, A. E.; Puterbaugh, R.; Fawzi, N. L.; Larschan, E.</p>
        <p class="info">Score: 7.8, Published: 2024-01-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.11.575216' target='https://doi.org/10.1101/2024.01.11.575216'> 10.1101/2024.01.11.575216</a></p>
        <p class="abstract">Despite decades of research, mechanisms by which co-transcriptional alternative splicing events are targeted to the correct genomic locations to drive cell fate decisions remain unknown. By combining structural and molecular approaches, we define a new mechanism by which an essential transcription factor (TF) targets co-transcriptional splicing through physical and functional interaction with RNA and RNA binding proteins (RBPs). We show that an essential TF co-transcriptionally regulates sex-specific alternative splicing by directly interacting with a subset of target RNAs on chromatin and modulating the dynamics of hnRNPA2 homolog nuclear splicing condensates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575396">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575396" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575396">
        <p class="paperTitle">Uncovering the genetic architecture and evolutionary roots of androgenetic alopecia in African men</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575396" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575396" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janivara, R.; Hazra, U.; Pfennig, A.; Harlemon, M.; Kim, M. S.; Eaaswarkhanth, M.; Chen, W. C.; Ogunbiyi, A.; Kachambwa, P.; Petersen, L. N.; Jalloh, M.; Mensah, J. E.; Adjei, A. A.; Adusei, B.; Joffe, M.; Gueye, S. M.; Aisuodionoe-Shadrach, O. I.; Fernandez, P. W.; Rohan, T. E.; Andrews, C.; Rebbeck, T. R.; Adebiyi, A. O.; Agalliu, I.; Lachance, J.</p>
        <p class="info">Score: 4.1, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575396' target='https://doi.org/10.1101/2024.01.12.575396'> 10.1101/2024.01.12.575396</a></p>
        <p class="abstract">Androgenetic alopecia is a highly heritable trait. However, much of our understanding about the genetics of male pattern baldness comes from individuals of European descent. Here, we examined a novel dataset comprising 2,136 men from Ghana, Nigeria, Senegal, and South Africa that were genotyped using a custom array. We first tested how genetic predictions of baldness generalize from Europe to Africa, finding that polygenic scores from European GWAS yielded AUC statistics that ranged from 0.513 to 0.546, indicating that genetic predictions of baldness in African populations performed notably worse than in European populations. Subsequently, we conducted the first African GWAS of androgenetic alopecia, focusing on self-reported baldness patterns at age 45. After correcting for present age, population structure, and study site, we identified 266 moderately significant associations, 51 of which were independent (p-value &lt; 10-5, r2 &lt; 0.2). Most baldness associations were autosomal, and the X chromosomes does not appear to have a large impact on baldness in African men. Finally, we examined the evolutionary causes of continental differences in genetic architecture. Although Neanderthal alleles have previously been associated with skin and hair phenotypes, we did not find evidence that European-ascertained baldness hits were enriched for signatures of ancient introgression. Most loci that are associated with androgenetic alopecia are evolving neutrally. However, multiple baldness-associated SNPs near the EDA2R and AR genes have large allele frequency differences between continents. Collectively, our findings illustrate how evolutionary history contributes to the limited portability of genetic predictions across ancestries.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575789">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575789" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575789">
        <p class="paperTitle">FAK and p130Cas modulate stiffness-mediated early transcription and cellular metabolism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575789" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575789" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tumenbayar, B.-I.; Tutino, V. M.; Brazzo, J. A.; Yao, P.; Bae, Y.</p>
        <p class="info">Score: 3.8, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575789' target='https://doi.org/10.1101/2024.01.15.575789'> 10.1101/2024.01.15.575789</a></p>
        <p class="abstract">Cellular metabolism is influenced by the stiffness of the extracellular matrix. Focal adhesion kinase (FAK) and its binding partner, p130Cas, transmit biomechanical signals about substrate stiffness to the cell to regulate a variety of cellular responses, but their roles in early transcriptional and metabolic responses remain largely unexplored. We cultured mouse embryonic fibroblasts with or without siRNA-mediated FAK or p130Cas knockdown and assessed the early transcriptional responses of these cells to placement on soft and stiff substrates by RNA sequencing and bioinformatics analyses. Exposure to the stiff ECM altered the expression of genes important for metabolic and biosynthetic processes, and these responses were influenced by knockdown of FAK and p130Cas. Our findings reveal that FAK-p130Cas signaling mechanotransduces ECM stiffness to early transcriptional changes that alter cellular metabolism and biosynthesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.13.575302">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.13.575302" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.13.575302">
        <p class="paperTitle">Genome assembly of the milky mangrove Excoecaria agallocha</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.13.575302" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.13.575302" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hong Kong Biodiversity Genomics Consortium,  ; Hui, J. H. L.; Chan, T.-F.; Chan, L.; Cheung, S. G.; Cheang, C. C.; Fang, J.; Gaitan-Espitia, J. D.; Lau, S.; Sung, Y. H.; Wong, C.; Yip, K.; Wei, Y.; law, S. T. S.; So, W. L.; Nong, W.; Lau, D.; Lee, S. Y.; Yip, H. Y.</p>
        <p class="info">Score: 3.8, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.13.575302' target='https://doi.org/10.1101/2024.01.13.575302'> 10.1101/2024.01.13.575302</a></p>
        <p class="abstract">The milky mangrove Excoecaria agallocha is a latex-secreting mangrove that are distributed in tropical and subtropical regions. While its poisonous latex is regarded as a potential source of phytochemicals for biomedical applications, the genomic resources of E. agallocha remains limited. Here, we present a chromosomal level genome of E. agallocha, assembled from the combination of PacBio long-read sequencing and Omni-C data. The resulting assembly size is 1,332.45 Mb and has high contiguity and completeness with a scaffold N50 of 58.9 Mb and a BUSCO score of 98.4 %. 73,740 protein-coding genes were also predicted. The milky mangrove genome provides a useful resource for further understanding the biosynthesis of phytochemical compounds in E. agallocha.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.14.575483">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.14.575483" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.14.575483">
        <p class="paperTitle">Denisovan admixture facilitated environmental adaptation in Papua New Guinean populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.14.575483" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.14.575483" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yermakovich, D.; Andre, M.; Brucato, N.; Kariwiga, J.; Leavesley, M.; Pankratov, V.; Mondal, M.; Ricaut, F.-X.; Dannemann, M.</p>
        <p class="info">Score: 3.5, Published: 2024-01-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.14.575483' target='https://doi.org/10.1101/2024.01.14.575483'> 10.1101/2024.01.14.575483</a></p>
        <p class="abstract">Neandertals and Denisovans, having inhabited distinct regions in Eurasia and possibly Oceania for over 200,000 years, experienced ample time to adapt to diverse environmental challenges these regions presented. Among present-day human populations, Papua New Guineans (PNG) stand out as one of the few carrying substantial amounts of both Neandertal and Denisovan DNA, a result of past admixture events with these archaic human groups. This study investigates the distribution of introgressed Denisovan and Neandertal DNA within two distinct PNG populations, residing in the highlands of Mt Wilhelm and the lowlands of Daru Island. These locations exhibit unique environmental features, some of which may parallel the challenges that archaic humans once confronted and adapted to. Our results show that Denisovan-like haplotypes exhibit increased levels of population differentiation between PNG highlanders and lowlanders. The highly differentiated haplotypes, more common among highlanders, reside in genomic areas linked to brain development genes. Conversely, those more frequent in lowlanders overlap with genes enriched in immune response processes. Furthermore, Denisovan-like haplotypes displayed pronounced signatures of diversification within the major histocompatibility complex. Our findings suggest that Denisovan DNA has provided a valuable source of genetic variation to PNG genomes that facilitated adaptive responses to environmental challenges.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
