<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Enhanced detection of RNA modifications and mappability with high-accuracy nanopore RNA basecalling models
Authors: Diensthuber, G.; Pryszcz, L.; Llovera, L.; Lucas, M. C.; Delgado-Tejedor, A.; Cruciani, S.; Roignant, J.-Y.; Begik, O.; Novoa, E. M.
Score: 34.5, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568965
In recent years, nanopore direct RNA sequencing (DRS) has established itself as a valuable tool for studying the epitranscriptome, due to its ability to detect multiple modifications within the same full-length native RNA molecules.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Enhanced detection of RNA modifications and mappability with high-accuracy nanopore RNA basecalling models
Authors: Diensthuber, G.; Pryszcz, L.; Llovera, L.; Lucas, M. C.; Delgado-Tejedor, A.; Cruciani, S.; Roignant, J.-Y.; Begik, O.; Novoa, E. M.
Score: 34.5, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568965
In recent years, nanopore direct RNA sequencing (DRS) has established itself as a valuable tool for studying the epitranscriptome, due to its ability to detect multiple modifications within the same full-length native RNA molecules." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-03T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-12-03T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Enhanced detection of RNA modifications and mappability with high-accuracy nanopore RNA basecalling models
Authors: Diensthuber, G.; Pryszcz, L.; Llovera, L.; Lucas, M. C.; Delgado-Tejedor, A.; Cruciani, S.; Roignant, J.-Y.; Begik, O.; Novoa, E. M.
Score: 34.5, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568965
In recent years, nanopore direct RNA sequencing (DRS) has established itself as a valuable tool for studying the epitranscriptome, due to its ability to detect multiple modifications within the same full-length native RNA molecules."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Enhanced detection of RNA modifications and mappability with high-accuracy nanopore RNA basecalling models\nAuthors: Diensthuber, G.; Pryszcz, L.; Llovera, L.; Lucas, M. C.; Delgado-Tejedor, A.; Cruciani, S.; Roignant, J.-Y.; Begik, O.; Novoa, E. M.\nScore: 34.5, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568965\nIn recent years, nanopore direct RNA sequencing (DRS) has established itself as a valuable tool for studying the epitranscriptome, due to its ability to detect multiple modifications within the same full-length native RNA molecules.",
  "keywords": [
    
  ],
  "articleBody": " Enhanced detection of RNA modifications and mappability with high-accuracy nanopore RNA basecalling models\nAuthors: Diensthuber, G.; Pryszcz, L.; Llovera, L.; Lucas, M. C.; Delgado-Tejedor, A.; Cruciani, S.; Roignant, J.-Y.; Begik, O.; Novoa, E. M.\nScore: 34.5, Published: 2023-11-28 DOI: 10.1101/2023.11.28.568965\nIn recent years, nanopore direct RNA sequencing (DRS) has established itself as a valuable tool for studying the epitranscriptome, due to its ability to detect multiple modifications within the same full-length native RNA molecules. While RNA modifications can be identified in the form of systematic basecalling errors in DRS datasets, N6-methyladenosine (m6A) modifications produce relatively low errors compared to other RNA modifications, limiting the applicability of this approach to m6A sites that are modified at high stoichiometries. Here, we demonstrate that the use of alternative RNA basecalling models, trained with fully unmodified sequences, increases the error signal of m6A, leading to enhanced detection and improved sensitivity even at low stoichiometries. Moreover, we find that high-accuracy alternative RNA basecalling models can show up to 97% median basecalling accuracy, outperforming currently available RNA basecalling models, which show 91% median basecalling accuracy. Notably, the use of high-accuracy basecalling models is accompanied by a significant increase in the number of mapped reads -especially in shorter RNA fractions- and increased basecalling error signatures at pseudouridine ({Psi}) and N1-methylpseudouridine (m1{Psi}) modified sites. Overall, our work demonstrates that alternative RNA basecalling models can be used to improve the detection of RNA modifications, read mappability and basecalling accuracy in nanopore DRS datasets.\nOrigin and evolution of the bread wheat D genome\nAuthors: Cavalet-Giorsa, E.; Gonzalez-Munoz, A.; Athiyannan, N.; Holden, S.; Salhi, A.; Gardener, C.; Quiroz-Chavez, J.; Rustamova, S. M.; Elkot, A. F.; Patpour, M.; Rasheed, A.; Mao, L.; Lagudah, E. S.; Periyannan, S. K.; Sharon, A.; Himmelbach, A.; Reif, J. C.; Knauft, M.; Mascher, M.; Stein, N.; Chayut, N.; Ghosh, S.; Perovic, D.; Putra, A.; Perera, A. B.; Hu, C.-Y.; Yu, G.; Ahmed, H. I.; Laquai, K. D.; Rivera, L. F.; Chen, R.; Wang, Y.; Gao, X.; Liu, S.; Raupp, W. J.; Olson, E. L.; Lee, J.-Y.; Chhuneja, P.; Kaur, S.; Zhang, P.; Park, R. F.; Ding, Y.; Liu, D.-C.; Li, W.; Nasyrova, F. Y.; Dvorak, J.;\nScore: 24.1, Published: 2023-11-29 DOI: 10.1101/2023.11.29.568958\nBread wheat (Triticum aestivum) is a globally dominant crop and major source of calories and proteins for the human diet. Compared to its wild ancestors, modern bread wheat shows lower genetic diversity caused by polyploidisation, domestication, and breeding bottlenecks1,2. Wild wheat relatives represent genetic reservoirs, harbouring diversity and beneficial alleles that have not been incorporated into bread wheat. Here, we establish and analyse pangenome resources for Tauschs goatgrass, Aegilops tauschii, the donor of the bread wheat D genome. This new pangenome facilitated the cloning of a disease resistance gene and haplotype analysis across a complex disease resistance locus, allowing us to discern alleles from paralogous gene copies. We also reveal the complex genetic composition and history of the bread wheat D genome, involving previously unreported contributions from genetically and geographically discrete Ae. tauschii subpopulations. Together, our results reveal the complex history of the bread wheat D genome and demonstrate the potential of wild relatives in crop improvement.\nClimate change introduces threatened killer whale populations and conservation challenges to the Arctic\nAuthors: Garroway, C. J.; de Greef, E.; Lefort, K. J.; Thorstensen, M. J.; Foote, A.; Matthews, C. J.; Higdon, J. W.; Kucheravy, C. E.; Petersen, S. D.; Rosing-Asvid, A.; Ugarte, F.; Dietz, R.; Ferguson, S.\nScore: 13.0, Published: 2023-11-27 DOI: 10.1101/2023.11.25.568606\nThe Arctic is the fastest-warming region on the planet, and sea ice loss has opened new habitat for sub-Arctic species such as the killer whale (Orcinus orca). As apex predators, killer whales can cause significant ecosystem-scale changes, however, we know very little about killer whales in the Arctic. Setting conservation priorities for killer whales and their Arctic prey species requires knowledge of their evolutionary history and demography. We found that there are two highly genetically distinct, non-interbreeding populations of killer whales using the eastern Canadian Arctic--one population is newly identified as globally distinct. The effective sizes of both populations recently declined, and both are vulnerable to inbreeding and reduced adaptive potential. Furthermore, we present evidence that human-caused mortalities, particularly ongoing harvest, pose an ongoing threat to these populations. The certainty of substantial environmental change in the Arctic complicates conservation and management significantly. Killer whales bring top-down pressure to Arctic food webs, however, they also merit conservation concern. The opening of the Arctic to killer whales exemplifies the magnitude of complex decisions surrounding local peoples, wildlife conservation, and resource management as the effects of climate change are realized.\nInterpreting cis-regulatory mechanisms from genomic deep neural networks using surrogate models\nAuthors: Seitz, E.; McCandlish, D. M.; Kinney, J. B.; Koo, P. K.\nScore: 29.8, Published: 2023-11-16 DOI: 10.1101/2023.11.14.567120\nDeep neural networks (DNNs) have greatly advanced the ability to predict genome function from sequence. Interpreting genomic DNNs in terms of biological mechanisms, however, remains difficult. Here we introduce SQUID, a genomic DNN interpretability framework based on surrogate modeling. SQUID approximates genomic DNNs in user-specified regions of sequence space using surrogate models, i.e., simpler models that are mechanistically interpretable. Importantly, SQUID removes the confounding effects that nonlinearities and heteroscedastic noise in functional genomics data can have on model interpretation. Benchmarking analysis on multiple genomic DNNs shows that SQUID, when compared to established interpretability methods, identifies motifs that are more consistent across genomic loci and yields improved single-nucleotide variant-effect predictions. SQUID also supports surrogate models that quantify epistatic interactions within and between cis-regulatory elements. SQUID thus advances the ability to mechanistically interpret genomic DNNs.\nSimultaneous CRISPR screening and spatial transcriptomics reveals intracellular, intercellular, and functional transcriptional circuits.\nAuthors: Binan, L.; Danquah, S.; Valakh, V.; Simonton, B.; Bezney, J.; Nehme, R.; Cleary, B.; Farhi, S. L.\nScore: 5.6, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569494\nPooled optical screens have enabled the study of cellular interactions, morphology, or dynamics at massive scale, but have not yet leveraged the power of highly-plexed single-cell resolved transcriptomic readouts to inform molecular pathways. Here, we present Perturb-FISH, which bridges these approaches by combining imaging spatial transcriptomics with parallel optical detection of in situ amplified guide RNAs. We show that Perturb-FISH recovers intracellular effects that are consistent with Perturb-seq results in a screen of lipopolysaccharide response in cultured monocytes, and uncover new intercellular and density-dependent regulation of the innate immune response. We further pair Perturb-FISH with a functional readout in a screen of autism spectrum disorder risk genes, showing common calcium activity phenotypes in induced pluripotent stem cell derived astrocytes and their associated genetic interactions and dysregulated molecular pathways. Perturb-FISH is thus a generally applicable method for studying the genetic and molecular associations of spatial and functional biology at single-cell resolution.\nHigh-fidelity, Large-scale Targeted Profiling of Microsatellites\nAuthors: Loh, C. A.; Shields, D. A.; Schwing, A.; Evrony, G. D.\nScore: 5.6, Published: 2023-11-29 DOI: 10.1101/2023.11.28.569106\nMicrosatellites are highly mutable sequences that can serve as markers for relationships among individuals or cells within a population. The accuracy and resolution of reconstructing these relationships depends on the fidelity of microsatellite profiling and the number of microsatellites profiled. However, current methods for targeted profiling of microsatellites incur significant \"stutter\" artifacts that interfere with accurate genotyping, and sequencing costs preclude whole-genome microsatellite profiling of a large number of samples. We developed a novel method for accurate and cost-effective targeted profiling of a panel of \u003e 150,000 microsatellites per sample, along with a computational tool for designing large-scale microsatellite panels. Our method addresses the greatest challenge for microsatellite profiling -- \"stutter\" artifacts -- with a low-temperature hybridization capture that significantly reduces these artifacts. We also developed a computational tool for accurate genotyping of the resulting microsatellite sequencing data that uses an ensemble approach integrating three microsatellite genotyping tools, which we optimize by analysis of de novo microsatellite mutations in human trios. Altogether, our suite of experimental and computational tools enables high-fidelity, large-scale profiling of microsatellites, which may find utility in diverse applications such as lineage tracing, population genetics, ecology, and forensics.\nDiploid genome assembly of the Malbec grapevine cultivar enables haplotype-aware analysis of transcriptomic differences underlying clonal phenotypic variation\nAuthors: Calderon, L.; Carbonell-Bejerano, P.; Munoz, C.; Bree, L.; Sola, C.; Bergamin, D.; Tulle, W.; Gomez-Talquenca, S.; Lanz, C.; Royo, C.; Ibanez, J.; Martinez-Zapater, J. M.; Weigel, D.; Lijavetzky, D.\nScore: 4.2, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569420\nGrapevine cultivars (Vitis vinifera L. ssp. vinifera) must be clonally propagated because of their highly heterozygous genomes. Malbec, a France-originated cultivar appreciated for the production of high-quality wines, is the offspring of cultivars Prunelard and Magdeleine Noire des Charentes. Here, we have built a diploid genome assembly of Malbec, after trio binning of PacBio long reads into the two haploid complements inherited from either parent. After haplotype-aware deduplication and corrections, complete assemblies for the two haplophases were obtained with very low haplotype switch-error rate (\u003c0.025). The haplophases alignment identified \u003e25% of polymorphic regions. Gene annotation including RNA-seq transcriptome assembly and ab initio prediction evidence resulted in similar gene model numbers for both haplophases. The annotated diploid assembly was exploited in the transcriptomic comparison of four clonal accessions of Malbec that exhibited variation in berry composition traits. Analysis of the ripening pericarp transcriptome using either haplophases as reference yielded similar results, although some differences were observed. Particularly, among the differentially expressed genes in the Magdeleine-inherited haplotype, we observed an over-representation of hypothetically hemizygous genes. The higher berry anthocyanin content of clonal accession 595 was associated with increased abscisic acid responses, leading to overexpression of secondary metabolism genes and deregulation of genes associated to abiotic stress response. Overall, the results highlight the importance of producing diploid assemblies to fully represent the genomic diversity of highly heterozygous woody crop cultivars and to unveil the molecular bases of clonal phenotypic variation.\nAn encyclopedia of enhancer-gene regulatory interactions in the human genome\nAuthors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K.; Perez, E. M.; Nguyen, T. C.; Kelley, D. R.; Finucane, H. K.; Moore, J. E.; Weng, Z.; Kellis, M.; Bassik, M. C.; Price, A. L.; Beer, M. A.; Guigo, R.; Stamatoyannopoulos, J. A.; Aiden, E. L.; Greenleaf, W. J.; Leslie, C. S.; Steinmetz, L. M.; Kundaje, A.; Engreitz, J. M.\nScore: 115.0, Published: 2023-11-13 DOI: 10.1101/2023.11.09.563812\nIdentifying transcriptional enhancers and their target genes is essential for understanding gene regulation and the impact of human genetic variation on disease1-6. Here we create and evaluate a resource of \u003e13 million enhancer-gene regulatory interactions across 352 cell types and tissues, by integrating predictive models, measurements of chromatin state and 3D contacts, and large-scale genetic perturbations generated by the ENCODE Consortium7. We first create a systematic benchmarking pipeline to compare predictive models, assembling a dataset of 10,411 element-gene pairs measured in CRISPR perturbation experiments, \u003e30,000 fine-mapped eQTLs, and 569 fine-mapped GWAS variants linked to a likely causal gene. Using this framework, we develop a new predictive model, ENCODE-rE2G, that achieves state-of-the-art performance across multiple prediction tasks, demonstrating a strategy involving iterative perturbations and supervised machine learning to build increasingly accurate predictive models of enhancer regulation. Using the ENCODE-rE2G model, we build an encyclopedia of enhancer-gene regulatory interactions in the human genome, which reveals global properties of enhancer networks, identifies differences in the functions of genes that have more or less complex regulatory landscapes, and improves analyses to link noncoding variants to target genes and cell types for common, complex diseases. By interpreting the model, we find evidence that, beyond enhancer activity and 3D enhancer-promoter contacts, additional features guide enhancer-promoter communication including promoter class and enhancer-enhancer synergy. Altogether, these genome-wide maps of enhancer-gene regulatory interactions, benchmarking software, predictive models, and insights about enhancer function provide a valuable resource for future studies of gene regulation and human genetics.\nThe Complete Sequence and Comparative Analysis of Ape Sex Chromosomes\nAuthors: Makova, K. D.; Pickett, B. D.; Harris, R. S.; Hartley, G. A.; Cechova, M.; Pal, K.; Nurk, S.; Yoo, D.; Li, Q.; Hebbar, P.; McGrath, B. C.; Antonacci, F.; Aubel, M.; Biddanda, A.; Borchers, M.; Bornberg, E.; Bouffard, G. G.; Brooks, S. Y.; Carbone, L.; Carrel, L.; Carroll, A.; Chang, P.-C.; Chin, C.-S.; Cook, D. E.; Craig, S. J. C.; de Gennaro, L.; Diekhans, M.; Dutra, A.; Garcia, G. H.; Grady, P. G. S.; Green, R. E.; Haddad, D.; Hallast, P.; Harvey, W. T.; Hickey, G.; Hillis, D. A.; Hoyt, S. J.; Jeong, H.; Kamali, K.; Pond, S. L. K.; LaPolice, T. M.; Lee, C.; Lewis, A. P.; Loh, Y.-H. E.; Maste\nScore: 2.9, Published: 2023-12-01 DOI: 10.1101/2023.11.30.569198\nApes possess two sex chromosomes--the male-specific Y and the X shared by males and females. The Y chromosome is crucial for male reproduction, with deletions linked to infertility. The X chromosome carries genes vital for reproduction and cognition. Variation in mating patterns and brain function among great apes suggests corresponding differences in their sex chromosome structure and evolution. However, due to their highly repetitive nature and incomplete reference assemblies, ape sex chromosomes have been challenging to study. Here, using the state-of-the-art experimental and computational methods developed for the telomere-to-telomere (T2T) human genome, we produced gapless, complete assemblies of the X and Y chromosomes for five great apes (chimpanzee, bonobo, gorilla, Bornean and Sumatran orangutans) and a lesser ape, the siamang gibbon. These assemblies completely resolved ampliconic, palindromic, and satellite sequences, including the entire centromeres, allowing us to untangle the intricacies of ape sex chromosome evolution. We found that, compared to the X, ape Y chromosomes vary greatly in size and have low alignability and high levels of structural rearrangements. This divergence on the Y arises from the accumulation of lineage-specific ampliconic regions and palindromes (which are shared more broadly among species on the X) and from the abundance of transposable elements and satellites (which have a lower representation on the X). Our analysis of Y chromosome genes revealed lineage-specific expansions of multi-copy gene families and signatures of purifying selection. In summary, the Y exhibits dynamic evolution, while the X is more stable. Finally, mapping short-read sequencing data from \u003e100 great ape individuals revealed the patterns of diversity and selection on their sex chromosomes, demonstrating the utility of these reference assemblies for studies of great ape evolution. These complete sex chromosome assemblies are expected to further inform conservation genetics of nonhuman apes, all of which are endangered species.\nSingle-cell transcriptomic and genomic changes in the aging human brain\nAuthors: Jeffries, A. M.; Yu, T.; Ziegenfuss, J. S.; Tolles, A. K.; Kim, Y.; Weng, Z.; Lodato, M. A.\nScore: 24.0, Published: 2023-11-07 DOI: 10.1101/2023.11.07.566050\nAging brings dysregulation of various processes across organs and tissues, often stemming from stochastic damage to individual cells over time. Here, we used a combination of single-nucleus RNA-sequencing and single-cell whole-genome sequencing to identify transcriptomic and genomic changes in the prefrontal cortex of the human brain across life span, from infancy to centenarian. We identified infant-specific cell clusters enriched for the expression of neurodevelopmental genes, and a common down-regulation of cell-essential homeostatic genes that function in ribosomes, transport, and metabolism during aging across cell types. Conversely, expression of neuron-specific genes generally remains stable throughout life. We observed a decrease in specific DNA repair genes in aging, including genes implicated in generating brain somatic mutations as indicated by mutation signature analysis. Furthermore, we detected gene-length-specific somatic mutation rates that shape the transcriptomic landscape of the aged human brain. These findings elucidate critical aspects of human brain aging, shedding light on transcriptomic and genomics dynamics.\n",
  "wordCount" : "2605",
  "inLanguage": "en",
  "datePublished": "2023-12-03T10:38:01Z",
  "dateModified": "2023-12-03T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 3, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.568965">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.568965" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.568965">
        <p class="paperTitle">Enhanced detection of RNA modifications and mappability with high-accuracy nanopore RNA basecalling models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.568965" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.568965" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Diensthuber, G.; Pryszcz, L.; Llovera, L.; Lucas, M. C.; Delgado-Tejedor, A.; Cruciani, S.; Roignant, J.-Y.; Begik, O.; Novoa, E. M.</p>
        <p class="info">Score: 34.5, Published: 2023-11-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.568965' target='https://doi.org/10.1101/2023.11.28.568965'> 10.1101/2023.11.28.568965</a></p>
        <p class="abstract">In recent years, nanopore direct RNA sequencing (DRS) has established itself as a valuable tool for studying the epitranscriptome, due to its ability to detect multiple modifications within the same full-length native RNA molecules. While RNA modifications can be identified in the form of systematic basecalling  errors in DRS datasets, N6-methyladenosine (m6A) modifications produce relatively low  errors compared to other RNA modifications, limiting the applicability of this approach to m6A sites that are modified at high stoichiometries. Here, we demonstrate that the use of alternative RNA basecalling models, trained with fully unmodified sequences, increases the  error signal of m6A, leading to enhanced detection and improved sensitivity even at low stoichiometries. Moreover, we find that high-accuracy alternative RNA basecalling models can show up to 97% median basecalling accuracy, outperforming currently available RNA basecalling models, which show 91% median basecalling accuracy. Notably, the use of high-accuracy basecalling models is accompanied by a significant increase in the number of mapped reads -especially in shorter RNA fractions- and increased basecalling error signatures at pseudouridine ({Psi}) and N1-methylpseudouridine (m1{Psi}) modified sites. Overall, our work demonstrates that alternative RNA basecalling models can be used to improve the detection of RNA modifications, read mappability and basecalling accuracy in nanopore DRS datasets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.568958">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.568958" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.568958">
        <p class="paperTitle">Origin and evolution of the bread wheat D genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.568958" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.568958" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cavalet-Giorsa, E.; Gonzalez-Munoz, A.; Athiyannan, N.; Holden, S.; Salhi, A.; Gardener, C.; Quiroz-Chavez, J.; Rustamova, S. M.; Elkot, A. F.; Patpour, M.; Rasheed, A.; Mao, L.; Lagudah, E. S.; Periyannan, S. K.; Sharon, A.; Himmelbach, A.; Reif, J. C.; Knauft, M.; Mascher, M.; Stein, N.; Chayut, N.; Ghosh, S.; Perovic, D.; Putra, A.; Perera, A. B.; Hu, C.-Y.; Yu, G.; Ahmed, H. I.; Laquai, K. D.; Rivera, L. F.; Chen, R.; Wang, Y.; Gao, X.; Liu, S.; Raupp, W. J.; Olson, E. L.; Lee, J.-Y.; Chhuneja, P.; Kaur, S.; Zhang, P.; Park, R. F.; Ding, Y.; Liu, D.-C.; Li, W.; Nasyrova, F. Y.; Dvorak, J.;</p>
        <p class="info">Score: 24.1, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.568958' target='https://doi.org/10.1101/2023.11.29.568958'> 10.1101/2023.11.29.568958</a></p>
        <p class="abstract">Bread wheat (Triticum aestivum) is a globally dominant crop and major source of calories and proteins for the human diet. Compared to its wild ancestors, modern bread wheat shows lower genetic diversity caused by polyploidisation, domestication, and breeding bottlenecks1,2. Wild wheat relatives represent genetic reservoirs, harbouring diversity and beneficial alleles that have not been incorporated into bread wheat. Here, we establish and analyse pangenome resources for Tauschs goatgrass, Aegilops tauschii, the donor of the bread wheat D genome. This new pangenome facilitated the cloning of a disease resistance gene and haplotype analysis across a complex disease resistance locus, allowing us to discern alleles from paralogous gene copies. We also reveal the complex genetic composition and history of the bread wheat D genome, involving previously unreported contributions from genetically and geographically discrete Ae. tauschii subpopulations. Together, our results reveal the complex history of the bread wheat D genome and demonstrate the potential of wild relatives in crop improvement.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.25.568606">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.25.568606" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.25.568606">
        <p class="paperTitle">Climate change introduces threatened killer whale populations and conservation challenges to the Arctic</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.25.568606" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.25.568606" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garroway, C. J.; de Greef, E.; Lefort, K. J.; Thorstensen, M. J.; Foote, A.; Matthews, C. J.; Higdon, J. W.; Kucheravy, C. E.; Petersen, S. D.; Rosing-Asvid, A.; Ugarte, F.; Dietz, R.; Ferguson, S.</p>
        <p class="info">Score: 13.0, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.25.568606' target='https://doi.org/10.1101/2023.11.25.568606'> 10.1101/2023.11.25.568606</a></p>
        <p class="abstract">The Arctic is the fastest-warming region on the planet, and sea ice loss has opened new habitat for sub-Arctic species such as the killer whale (Orcinus orca). As apex predators, killer whales can cause significant ecosystem-scale changes, however, we know very little about killer whales in the Arctic. Setting conservation priorities for killer whales and their Arctic prey species requires knowledge of their evolutionary history and demography. We found that there are two highly genetically distinct, non-interbreeding populations of killer whales using the eastern Canadian Arctic--one population is newly identified as globally distinct. The effective sizes of both populations recently declined, and both are vulnerable to inbreeding and reduced adaptive potential. Furthermore, we present evidence that human-caused mortalities, particularly ongoing harvest, pose an ongoing threat to these populations. The certainty of substantial environmental change in the Arctic complicates conservation and management significantly. Killer whales bring top-down pressure to Arctic food webs, however, they also merit conservation concern. The opening of the Arctic to killer whales exemplifies the magnitude of complex decisions surrounding local peoples, wildlife conservation, and resource management as the effects of climate change are realized.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567120" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567120">
        <p class="paperTitle">Interpreting cis-regulatory mechanisms from genomic deep neural networks using surrogate models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seitz, E.; McCandlish, D. M.; Kinney, J. B.; Koo, P. K.</p>
        <p class="info">Score: 29.8, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567120' target='https://doi.org/10.1101/2023.11.14.567120'> 10.1101/2023.11.14.567120</a></p>
        <p class="abstract">Deep neural networks (DNNs) have greatly advanced the ability to predict genome function from sequence. Interpreting genomic DNNs in terms of biological mechanisms, however, remains difficult. Here we introduce SQUID, a genomic DNN interpretability framework based on surrogate modeling. SQUID approximates genomic DNNs in user-specified regions of sequence space using surrogate models, i.e., simpler models that are mechanistically interpretable. Importantly, SQUID removes the confounding effects that nonlinearities and heteroscedastic noise in functional genomics data can have on model interpretation. Benchmarking analysis on multiple genomic DNNs shows that SQUID, when compared to established interpretability methods, identifies motifs that are more consistent across genomic loci and yields improved single-nucleotide variant-effect predictions. SQUID also supports surrogate models that quantify epistatic interactions within and between cis-regulatory elements. SQUID thus advances the ability to mechanistically interpret genomic DNNs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569494">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569494" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569494">
        <p class="paperTitle">Simultaneous CRISPR screening and spatial transcriptomics reveals intracellular, intercellular, and functional transcriptional circuits.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569494" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569494" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Binan, L.; Danquah, S.; Valakh, V.; Simonton, B.; Bezney, J.; Nehme, R.; Cleary, B.; Farhi, S. L.</p>
        <p class="info">Score: 5.6, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569494' target='https://doi.org/10.1101/2023.11.30.569494'> 10.1101/2023.11.30.569494</a></p>
        <p class="abstract">Pooled optical screens have enabled the study of cellular interactions, morphology, or dynamics at massive scale, but have not yet leveraged the power of highly-plexed single-cell resolved transcriptomic readouts to inform molecular pathways. Here, we present Perturb-FISH, which bridges these approaches by combining imaging spatial transcriptomics with parallel optical detection of in situ amplified guide RNAs. We show that Perturb-FISH recovers intracellular effects that are consistent with Perturb-seq results in a screen of lipopolysaccharide response in cultured monocytes, and uncover new intercellular and density-dependent regulation of the innate immune response. We further pair Perturb-FISH with a functional readout in a screen of autism spectrum disorder risk genes, showing common calcium activity phenotypes in induced pluripotent stem cell derived astrocytes and their associated genetic interactions and dysregulated molecular pathways. Perturb-FISH is thus a generally applicable method for studying the genetic and molecular associations of spatial and functional biology at single-cell resolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569106">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569106" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569106">
        <p class="paperTitle">High-fidelity, Large-scale Targeted Profiling of Microsatellites</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569106" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569106" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Loh, C. A.; Shields, D. A.; Schwing, A.; Evrony, G. D.</p>
        <p class="info">Score: 5.6, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569106' target='https://doi.org/10.1101/2023.11.28.569106'> 10.1101/2023.11.28.569106</a></p>
        <p class="abstract">Microsatellites are highly mutable sequences that can serve as markers for relationships among individuals or cells within a population. The accuracy and resolution of reconstructing these relationships depends on the fidelity of microsatellite profiling and the number of microsatellites profiled. However, current methods for targeted profiling of microsatellites incur significant &#34;stutter&#34; artifacts that interfere with accurate genotyping, and sequencing costs preclude whole-genome microsatellite profiling of a large number of samples. We developed a novel method for accurate and cost-effective targeted profiling of a panel of &gt; 150,000 microsatellites per sample, along with a computational tool for designing large-scale microsatellite panels. Our method addresses the greatest challenge for microsatellite profiling -- &#34;stutter&#34; artifacts -- with a low-temperature hybridization capture that significantly reduces these artifacts. We also developed a computational tool for accurate genotyping of the resulting microsatellite sequencing data that uses an ensemble approach integrating three microsatellite genotyping tools, which we optimize by analysis of de novo microsatellite mutations in human trios. Altogether, our suite of experimental and computational tools enables high-fidelity, large-scale profiling of microsatellites, which may find utility in diverse applications such as lineage tracing, population genetics, ecology, and forensics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569420">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569420" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569420">
        <p class="paperTitle">Diploid genome assembly of the Malbec grapevine cultivar enables haplotype-aware analysis of transcriptomic differences underlying clonal phenotypic variation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569420" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569420" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Calderon, L.; Carbonell-Bejerano, P.; Munoz, C.; Bree, L.; Sola, C.; Bergamin, D.; Tulle, W.; Gomez-Talquenca, S.; Lanz, C.; Royo, C.; Ibanez, J.; Martinez-Zapater, J. M.; Weigel, D.; Lijavetzky, D.</p>
        <p class="info">Score: 4.2, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569420' target='https://doi.org/10.1101/2023.11.30.569420'> 10.1101/2023.11.30.569420</a></p>
        <p class="abstract">Grapevine cultivars (Vitis vinifera L. ssp. vinifera) must be clonally propagated because of their highly heterozygous genomes. Malbec, a France-originated cultivar appreciated for the production of high-quality wines, is the offspring of cultivars Prunelard and Magdeleine Noire des Charentes. Here, we have built a diploid genome assembly of Malbec, after trio binning of PacBio long reads into the two haploid complements inherited from either parent. After haplotype-aware deduplication and corrections, complete assemblies for the two haplophases were obtained with very low haplotype switch-error rate (&lt;0.025). The haplophases alignment identified &gt;25% of polymorphic regions. Gene annotation including RNA-seq transcriptome assembly and ab initio prediction evidence resulted in similar gene model numbers for both haplophases. The annotated diploid assembly was exploited in the transcriptomic comparison of four clonal accessions of Malbec that exhibited variation in berry composition traits. Analysis of the ripening pericarp transcriptome using either haplophases as reference yielded similar results, although some differences were observed. Particularly, among the differentially expressed genes in the Magdeleine-inherited haplotype, we observed an over-representation of hypothetically hemizygous genes. The higher berry anthocyanin content of clonal accession 595 was associated with increased abscisic acid responses, leading to overexpression of secondary metabolism genes and deregulation of genes associated to abiotic stress response. Overall, the results highlight the importance of producing diploid assemblies to fully represent the genomic diversity of highly heterozygous woody crop cultivars and to unveil the molecular bases of clonal phenotypic variation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.563812">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.563812" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.563812">
        <p class="paperTitle">An encyclopedia of enhancer-gene regulatory interactions in the human genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.563812" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.563812" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K.; Perez, E. M.; Nguyen, T. C.; Kelley, D. R.; Finucane, H. K.; Moore, J. E.; Weng, Z.; Kellis, M.; Bassik, M. C.; Price, A. L.; Beer, M. A.; Guigo, R.; Stamatoyannopoulos, J. A.; Aiden, E. L.; Greenleaf, W. J.; Leslie, C. S.; Steinmetz, L. M.; Kundaje, A.; Engreitz, J. M.</p>
        <p class="info">Score: 115.0, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.563812' target='https://doi.org/10.1101/2023.11.09.563812'> 10.1101/2023.11.09.563812</a></p>
        <p class="abstract">Identifying transcriptional enhancers and their target genes is essential for understanding gene regulation and the impact of human genetic variation on disease1-6. Here we create and evaluate a resource of &gt;13 million enhancer-gene regulatory interactions across 352 cell types and tissues, by integrating predictive models, measurements of chromatin state and 3D contacts, and large-scale genetic perturbations generated by the ENCODE Consortium7. We first create a systematic benchmarking pipeline to compare predictive models, assembling a dataset of 10,411 element-gene pairs measured in CRISPR perturbation experiments, &gt;30,000 fine-mapped eQTLs, and 569 fine-mapped GWAS variants linked to a likely causal gene. Using this framework, we develop a new predictive model, ENCODE-rE2G, that achieves state-of-the-art performance across multiple prediction tasks, demonstrating a strategy involving iterative perturbations and supervised machine learning to build increasingly accurate predictive models of enhancer regulation. Using the ENCODE-rE2G model, we build an encyclopedia of enhancer-gene regulatory interactions in the human genome, which reveals global properties of enhancer networks, identifies differences in the functions of genes that have more or less complex regulatory landscapes, and improves analyses to link noncoding variants to target genes and cell types for common, complex diseases. By interpreting the model, we find evidence that, beyond enhancer activity and 3D enhancer-promoter contacts, additional features guide enhancer-promoter communication including promoter class and enhancer-enhancer synergy. Altogether, these genome-wide maps of enhancer-gene regulatory interactions, benchmarking software, predictive models, and insights about enhancer function provide a valuable resource for future studies of gene regulation and human genetics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569198">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569198" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569198">
        <p class="paperTitle">The Complete Sequence and Comparative Analysis of Ape Sex Chromosomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569198" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569198" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Makova, K. D.; Pickett, B. D.; Harris, R. S.; Hartley, G. A.; Cechova, M.; Pal, K.; Nurk, S.; Yoo, D.; Li, Q.; Hebbar, P.; McGrath, B. C.; Antonacci, F.; Aubel, M.; Biddanda, A.; Borchers, M.; Bornberg, E.; Bouffard, G. G.; Brooks, S. Y.; Carbone, L.; Carrel, L.; Carroll, A.; Chang, P.-C.; Chin, C.-S.; Cook, D. E.; Craig, S. J. C.; de Gennaro, L.; Diekhans, M.; Dutra, A.; Garcia, G. H.; Grady, P. G. S.; Green, R. E.; Haddad, D.; Hallast, P.; Harvey, W. T.; Hickey, G.; Hillis, D. A.; Hoyt, S. J.; Jeong, H.; Kamali, K.; Pond, S. L. K.; LaPolice, T. M.; Lee, C.; Lewis, A. P.; Loh, Y.-H. E.; Maste</p>
        <p class="info">Score: 2.9, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569198' target='https://doi.org/10.1101/2023.11.30.569198'> 10.1101/2023.11.30.569198</a></p>
        <p class="abstract">Apes possess two sex chromosomes--the male-specific Y and the X shared by males and females. The Y chromosome is crucial for male reproduction, with deletions linked to infertility. The X chromosome carries genes vital for reproduction and cognition. Variation in mating patterns and brain function among great apes suggests corresponding differences in their sex chromosome structure and evolution. However, due to their highly repetitive nature and incomplete reference assemblies, ape sex chromosomes have been challenging to study. Here, using the state-of-the-art experimental and computational methods developed for the telomere-to-telomere (T2T) human genome, we produced gapless, complete assemblies of the X and Y chromosomes for five great apes (chimpanzee, bonobo, gorilla, Bornean and Sumatran orangutans) and a lesser ape, the siamang gibbon. These assemblies completely resolved ampliconic, palindromic, and satellite sequences, including the entire centromeres, allowing us to untangle the intricacies of ape sex chromosome evolution. We found that, compared to the X, ape Y chromosomes vary greatly in size and have low alignability and high levels of structural rearrangements. This divergence on the Y arises from the accumulation of lineage-specific ampliconic regions and palindromes (which are shared more broadly among species on the X) and from the abundance of transposable elements and satellites (which have a lower representation on the X). Our analysis of Y chromosome genes revealed lineage-specific expansions of multi-copy gene families and signatures of purifying selection. In summary, the Y exhibits dynamic evolution, while the X is more stable. Finally, mapping short-read sequencing data from &gt;100 great ape individuals revealed the patterns of diversity and selection on their sex chromosomes, demonstrating the utility of these reference assemblies for studies of great ape evolution. These complete sex chromosome assemblies are expected to further inform conservation genetics of nonhuman apes, all of which are endangered species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566050">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566050" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566050">
        <p class="paperTitle">Single-cell transcriptomic and genomic changes in the aging human brain</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566050" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566050" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jeffries, A. M.; Yu, T.; Ziegenfuss, J. S.; Tolles, A. K.; Kim, Y.; Weng, Z.; Lodato, M. A.</p>
        <p class="info">Score: 24.0, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566050' target='https://doi.org/10.1101/2023.11.07.566050'> 10.1101/2023.11.07.566050</a></p>
        <p class="abstract">Aging brings dysregulation of various processes across organs and tissues, often stemming from stochastic damage to individual cells over time. Here, we used a combination of single-nucleus RNA-sequencing and single-cell whole-genome sequencing to identify transcriptomic and genomic changes in the prefrontal cortex of the human brain across life span, from infancy to centenarian. We identified infant-specific cell clusters enriched for the expression of neurodevelopmental genes, and a common down-regulation of cell-essential homeostatic genes that function in ribosomes, transport, and metabolism during aging across cell types. Conversely, expression of neuron-specific genes generally remains stable throughout life. We observed a decrease in specific DNA repair genes in aging, including genes implicated in generating brain somatic mutations as indicated by mutation signature analysis. Furthermore, we detected gene-length-specific somatic mutation rates that shape the transcriptomic landscape of the aged human brain. These findings elucidate critical aspects of human brain aging, shedding light on transcriptomic and genomics dynamics.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
