<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="High throughput PRIME editing screens identify functional DNA variants in the human genome
Authors: Ren, X.; Yang, H.; Nierenberg, J. L.; Sun, Y.; Chen, J.; Beaman, C.; Pham, T.; Nobuhara, M.; Takagi, M. A.; Narayan, V.; Li, Y.; Ziv, E.; Shen, Y.
Score: 15.0, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548736
Despite tremendous progress in detecting DNA variants associated with human disease, interpreting their functional impact in a high-throughput and base-pair resolution manner remains challenging.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="High throughput PRIME editing screens identify functional DNA variants in the human genome
Authors: Ren, X.; Yang, H.; Nierenberg, J. L.; Sun, Y.; Chen, J.; Beaman, C.; Pham, T.; Nobuhara, M.; Takagi, M. A.; Narayan, V.; Li, Y.; Ziv, E.; Shen, Y.
Score: 15.0, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548736
Despite tremendous progress in detecting DNA variants associated with human disease, interpreting their functional impact in a high-throughput and base-pair resolution manner remains challenging." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-19T10:38:24+00:00" />
<meta property="article:modified_time" content="2023-07-19T10:38:24+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="High throughput PRIME editing screens identify functional DNA variants in the human genome
Authors: Ren, X.; Yang, H.; Nierenberg, J. L.; Sun, Y.; Chen, J.; Beaman, C.; Pham, T.; Nobuhara, M.; Takagi, M. A.; Narayan, V.; Li, Y.; Ziv, E.; Shen, Y.
Score: 15.0, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548736
Despite tremendous progress in detecting DNA variants associated with human disease, interpreting their functional impact in a high-throughput and base-pair resolution manner remains challenging."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "High throughput PRIME editing screens identify functional DNA variants in the human genome\nAuthors: Ren, X.; Yang, H.; Nierenberg, J. L.; Sun, Y.; Chen, J.; Beaman, C.; Pham, T.; Nobuhara, M.; Takagi, M. A.; Narayan, V.; Li, Y.; Ziv, E.; Shen, Y.\nScore: 15.0, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548736\nDespite tremendous progress in detecting DNA variants associated with human disease, interpreting their functional impact in a high-throughput and base-pair resolution manner remains challenging.",
  "keywords": [
    
  ],
  "articleBody": " High throughput PRIME editing screens identify functional DNA variants in the human genome\nAuthors: Ren, X.; Yang, H.; Nierenberg, J. L.; Sun, Y.; Chen, J.; Beaman, C.; Pham, T.; Nobuhara, M.; Takagi, M. A.; Narayan, V.; Li, Y.; Ziv, E.; Shen, Y.\nScore: 15.0, Published: 2023-07-12 DOI: 10.1101/2023.07.12.548736\nDespite tremendous progress in detecting DNA variants associated with human disease, interpreting their functional impact in a high-throughput and base-pair resolution manner remains challenging. Here, we develop a novel pooled prime editing screen method, PRIME, which can be applied to characterize thousands of coding and non-coding variants in a single experiment with high reproducibility. To showcase its applications, we first identified essential nucleotides for a 716 bp MYC enhancer via PRIME-mediated saturation mutagenesis. Next, we applied PRIME to functionally characterize 1,304 non-coding variants associated with breast cancer and 3,699 variants from ClinVar. We discovered that 103 non-coding variants and 156 variants of uncertain significance are functional via affecting cell fitness. Collectively, we demonstrate PRIME capable of characterizing genetic variants at base-pair resolution and scale, advancing accurate genome annotation for disease risk prediction, diagnosis, and therapeutic target identification.\nA single cell atlas of sexual development in Plasmodium falciparum\nAuthors: Dogga, S. K.; Rop, J. C.; Cudini, J.; Farr, E.; Dara, A.; Ouologuem, D.; Djimde, A. A.; Talman, A. M.; Lawniczak, M. K. N.\nScore: 12.3, Published: 2023-07-17 DOI: 10.1101/2023.07.16.547819\nThe developmental decision made by malaria parasites to become sexual underlies all malaria transmission. Here, we describe a rich atlas of short and long-read single-cell transcriptomes of over 37,000 Plasmodium falciparum cells across intraerythrocytic asexual and sexual development. We used the atlas to explore transcriptional modules and exon usage along sexual development, and expanded it to include malaria parasites collected from a Malian individual naturally infected with multiple P. falciparum strains. We investigated genotypic and transcriptional heterogeneity within and among these wild strains at a single-cell level for the first time, finding considerable differential expression between different strains even within the same host. This work is a key addition to the Malaria Cell Atlas, enabling a deeper understanding of the biology and diversity of transmission stages.\nRecurrent chromosome destabilization through repeat-mediated rearrangements in a fungal pathogen\nAuthors: Fouche, S.; Oggenfuss, U.; McDonald, B. A.; Croll, D.\nScore: 11.4, Published: 2023-07-15 DOI: 10.1101/2023.07.14.549097\nGenomic instability caused by chromosomal rearrangements has severe consequences for organismal fitness and progression of cancerous cell lines. The triggers of destabilized chromosomes remain poorly understood but are often assumed to be associated with fragile sites. Here, we retrace a runaway chromosomal degeneration process observed in a fungal pathogen using telomere-to-telomere assemblies across an experimental pedigree. We show that the same fragile sites triggered reproducible, large-scale rearrangements through non-allelic recombination. Across the four-generation pedigree, chromosomal rearrangements were accompanied by non-disjunction events and caused aneuploid progeny to carry up to four chromosomal copies. We identify a specific transposable element as the most likely trigger for the repeated chromosomal degeneration. The element is associated with higher virulence of the pathogen and has undergone a burst increasing copy numbers across the genome. Chromosome sequences are also targeted by a genome defense mechanism active on duplicated sequences, which may contribute to decay. Our work identifies the exact sequence triggers initiating chromosome instability and perpetuating degenerative cycles. Dissecting proximate causes leading to run-away chromosomal degeneration expands our understanding of chromosomal evolution beyond cancer lines.\nComprehensive Assessment of Eleven de novo HiFi Assemblers on Complex Eukaryotic Genomes and Metagenomes\nAuthors: Yu, W.; Luo, H.; Yang, J.; Zhang, S.; Jiang, H.; Zhao, X.; Hui, X.; Sun, D.; Li, L.; Wei, X.-q.; Lonardi, S.; Pan, W.\nScore: 27.4, Published: 2023-06-29 DOI: 10.1101/2023.06.29.546998\nBackgroundPacific Bioscience HiFi sequencing technology generates long reads (\u003e10 kbp) with very high accuracy (less than 0.01% sequencing error). While several de novo assembly tools are available for HiFi reads, there are no comprehensive studies on the evaluation of these assemblers. ResultsWe evaluated the performance of eleven de novo HiFi assemblers on (i) real data for three eukaryotic genomes, (ii) 34 synthetic datasets with different ploidy, sequencing coverage levels, heterozygosity rates and sequencing error rates, (iii) one real metagenomic dataset, and (iv) five synthetic metagenomic datasets with different composition abundance and heterozygosity rates. The nine assemblers were evaluated using QUAST (Quality Assessment Tool) and BUSCO (Benchmarking Universal Single-Copy Ortholog). We also used several additional criteria, namely, completion rate, single-copy completion rate, duplicated completion rate, average proportion of largest category, average distance difference, quality value, run-time and memory utilization. On complex eukaryotic genomes, Hifiasm had a clear advantage over the other assemblers in all tested experiments. On synthetic datasets, Hifiasm, HiCanu, and HiFlye performed equally well. Shasta and Peregrine had good performance across varying ploidy, but required high computational resources. On metagenomic datasets, Hifiasm-meta demonstrated a clear advantage over other assemblers. ConclusionWe carried out a comprehensive benchmarking study of commonly used assemblers on complex eukaryotic genomes and metagenomes. Our study will help the research community to choose the most appropriate assembler for their data and identify possible improvements in assembly algorithms.\nHigh quality genome assembly and annotation (v1) of the eukaryotic terrestrial microalga Coccomyxa viridis SAG 216-4\nAuthors: Kraege, A.; Chavarro-Carrero, E.; Guiglielmoni, N.; Schnell, E.; Kirangwa, J.; Heilmann-Heimbach, S.; Becker, K.; Kohrer, K.; WGGC Team, ; DeRGA Community, ; Schiffer, P.; Thomma, B. P.; Rovenich, H.\nScore: 9.0, Published: 2023-07-12 DOI: 10.1101/2023.07.11.548521\nUnicellular green algae of the genus Coccomyxa are recognized for their worldwide distribution and ecological versatility. Most species described to date live in close association with various host species, such as in lichen associations. However, little is known about the molecular mechanisms that drive such symbiotic lifestyles. We generated a high-quality genome assembly for the lichen photobiont Coccomyxa viridis SAG 216-4 (formerly C. mucigena). Using long-read PacBio HiFi and Oxford Nanopore Technologies in combination with chromatin conformation capture (Hi-C) sequencing, we assembled the genome into 21 scaffolds with a total length of 50.9 Mb, an N50 of 2.7 Mb and a BUSCO score of 98.6%. While nineteen scaffolds represent full-length nuclear chromosomes, two additional scaffolds represent the mitochondrial and plastid genomes. Transcriptome-guided gene annotation resulted in the identification of 13,557 protein-coding genes, of which 68% have annotated PFAM domains and 962 are predicted to be secreted.\nDetecting differential transcript usage in complex diseases with SPIT\nAuthors: Erdogdu, B.; Varabyou, A.; Hicks, S. C.; Salzberg, S. L.; Pertea, M.\nScore: 10.7, Published: 2023-07-10 DOI: 10.1101/2023.07.10.548289\nDifferential transcript usage (DTU) plays a crucial role in determining how gene expression differs among cells, tissues, and different developmental stages, thereby contributing to the complexity and diversity of biological systems. In abnormal cells, it can also lead to deficiencies in protein function, potentially leading to pathogenesis of diseases. Detecting such events for single-gene genetic traits is relatively uncomplicated; however, the heterogeneity of populations with complex diseases presents an intricate challenge due to the presence of diverse causal events and undetermined subtypes. SPIT is the first statistical tool that quantifies the heterogeneity in transcript usage within a population and identifies predominant subgroups along with their distinctive sets of DTU events. We provide comprehensive assessments of SPITs methodology in both single-gene and complex traits and report the results of applying SPIT to analyze brain samples from individuals with schizophrenia. Our analysis reveals previously unreported DTU events in six candidate genes.\nTopological stratification of continuous genetic variation in large biobanks\nAuthors: Diaz-Papkovich, A.; Zabad, S.; Ben-Eghan, C.; Anderson-Trocme, L.; Femerling, G.; Nathan, V.; Patel, J.; Gravel, S.\nScore: 30.3, Published: 2023-07-07 DOI: 10.1101/2023.07.06.548007\nBiobanks now contain genetic data from millions of individuals. Dimension-ality reduction, visualization and stratification are standard when exploring data at these scales; while efficient and tractable methods exist for the first two, stratification remains challenging because of uncertainty about sources of population structure. In practice, stratification is commonly performed by drawing shapes around dimensionally reduced data or assuming populations have a \"type\" genome. We propose a method of stratifying data with topo-logical analysis that is fast, easy to implement, and integrates with existing pipelines. The approach is robust to the presence of sub-populations of vary-ing sizes and wide ranges of population structure patterns. We demonstrate its effectiveness on genotypes from three biobanks and illustrate how topolog-ical genetic strata can help us understand structure within biobanks, evaluate distributions of genotypic and phenotypic data, examine polygenic score trans-ferability, identify potential influential alleles, and perform quality control.\nThe expression landscape and pangenome of long non-coding RNA in the fungal wheat pathogen Zymoseptoria tritici\nAuthors: Glad, H. M.; Moser Tralamazza, S.; Croll, D.\nScore: 7.6, Published: 2023-07-17 DOI: 10.1101/2023.07.16.549241\nLong non-coding RNAs (lncRNAs) are regulatory molecules interacting in a wide array of biological processes. LncRNAs in fungal pathogens can be responsive to stress and play roles in regulating growth and nutrient acquisition. Recent evidence suggests that lncRNAs may also play roles in virulence, such as regulating pathogenicity-associated enzymes and on-host reproductive cycles. Despite the importance of lncRNAs, only few model fungi have well-documented inventories of lncRNA. In this study, we apply a machine-learning based pipeline to predict high-confidence lncRNA candidates in Zymoseptoria tritici, an important global pathogen of wheat impacting global food production. We analyzed genomic features of lncRNAs and the most likely associated processes through analyses of expression over a host infection cycle. We find that lncRNAs are frequently expressed during early infection, before the switch to necrotrophic growth. They are mostly located in facultative heterochromatic regions, which are known to contain many genes associated with pathogenicity. Furthermore, we find that lncRNAs are frequently co-expressed with genes that may be involved in responding to host signals, such as those responses to oxidative stress. Finally, we assess pangenome features of lncRNAs using four additional reference-quality genomes. We find evidence that the repertoire of expressed lncRNAs varies substantially between individuals, even though lncRNA loci tend to be shared at the genomic level. Overall, this study provides a repertoire and putative functions of lncRNAs in Z. tritici enabling molecular genetics and functional analyses in an important pathogen.\nGenome-wide expression QTL mapping reveals the highly dynamic regulatory landscape of a major wheat pathogen\nAuthors: Abraham, L. N.; Croll, D.\nScore: 6.5, Published: 2023-07-16 DOI: 10.1101/2023.07.14.549109\nIn agricultural ecosystems, outbreaks of diseases are frequent and pose a significant threat to food security. A successful pathogen undergoes a complex and well-timed sequence of regulatory changes to avoid detection by the host immune system, hence well-tuned gene regulation is essential for survival. However, the extent to which the regulatory polymorphisms in a pathogen population provide an adaptive advantage is poorly understood. We used Zymoseptoria tritici, one of the most important pathogens of wheat, to generate a genome-wide map of regulatory polymorphism governing gene expression. We investigated genome-wide transcription levels of 146 strains grown under nutrient starvation and performed expression quantitative trait loci (eQTL) mapping. We identified cis-eQTLs for 65.3% of all genes and the majority of all eQTL loci are within 2kb upstream and downstream of the transcription start site (TSS). We also show that polymorphism in different gene elements contributes disproportionally to gene expression variation. Investigating regulatory polymorphism in gene categories, we found an enrichment of regulatory variants for genes predicted to be important for fungal pathogenesis but with comparatively low effect size, suggesting a separate layer of gene regulation involving epigenetics. We also show that previously reported trait-associated SNPs in pathogen populations are frequently cis-regulatory variants of neighboring genes with implications for the trait architecture. Overall, our study provides extensive evidence that single populations segregate large-scale regulatory variation and are likely to fuel rapid adaptation to resistant hosts and environmental change.\nCell type and condition specific functional annotation of schizophrenia associated non-coding genetic variants\nAuthors: Rummel, C. K.; Gagliardi, M.; Herholt, A.; Ahmad, R.; Murek, V.; Weigert, L.; Hausruckinger, A.; Maidl, S.; Jimenez-Barron, L.; Trastulla, L.; Hauger, B.; Eder, M.; Rossner, M. J.; Ziller, M. J.\nScore: 8.6, Published: 2023-06-27 DOI: 10.1101/2023.06.27.545266\nSchizophrenia (SCZ) is a highly polygenic disease and genome wide association studies have identified thousands of genetic variants that are statistically associated with this psychiatric disorder. However, our ability to translate these associations into insights on the disease mechanisms has been challenging since the causal genetic variants, their molecular function and their target genes remain largely unknown. In order to address these questions, we established a functional genomics pipeline in combination with induced pluripotent stem cell technology to functionally characterize ~35,000 non-coding genetic variants associated with schizophrenia along with their target genes. This analysis identified a set of 620 (1.7%) single nucleotide polymorphisms as functional on a molecular level in a highly cell type and condition specific fashion. These results provide a high-resolution map of functional variant-gene combinations and offer comprehensive biological insights into the developmental context and stimulation dependent molecular processes modulated by SCZ associated genetic variation.\n",
  "wordCount" : "2111",
  "inLanguage": "en",
  "datePublished": "2023-07-19T10:38:24Z",
  "dateModified": "2023-07-19T10:38:24Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on July 19, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548736">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548736" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548736">
        <p class="paperTitle">High throughput PRIME editing screens identify functional DNA variants in the human genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548736" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548736" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ren, X.; Yang, H.; Nierenberg, J. L.; Sun, Y.; Chen, J.; Beaman, C.; Pham, T.; Nobuhara, M.; Takagi, M. A.; Narayan, V.; Li, Y.; Ziv, E.; Shen, Y.</p>
        <p class="info">Score: 15.0, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548736' target='https://doi.org/10.1101/2023.07.12.548736'> 10.1101/2023.07.12.548736</a></p>
        <p class="abstract">Despite tremendous progress in detecting DNA variants associated with human disease, interpreting their functional impact in a high-throughput and base-pair resolution manner remains challenging. Here, we develop a novel pooled prime editing screen method, PRIME, which can be applied to characterize thousands of coding and non-coding variants in a single experiment with high reproducibility. To showcase its applications, we first identified essential nucleotides for a 716 bp MYC enhancer via PRIME-mediated saturation mutagenesis. Next, we applied PRIME to functionally characterize 1,304 non-coding variants associated with breast cancer and 3,699 variants from ClinVar. We discovered that 103 non-coding variants and 156 variants of uncertain significance are functional via affecting cell fitness. Collectively, we demonstrate PRIME capable of characterizing genetic variants at base-pair resolution and scale, advancing accurate genome annotation for disease risk prediction, diagnosis, and therapeutic target identification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.547819">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.547819" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.547819">
        <p class="paperTitle">A single cell atlas of sexual development in Plasmodium falciparum</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.547819" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.547819" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dogga, S. K.; Rop, J. C.; Cudini, J.; Farr, E.; Dara, A.; Ouologuem, D.; Djimde, A. A.; Talman, A. M.; Lawniczak, M. K. N.</p>
        <p class="info">Score: 12.3, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.547819' target='https://doi.org/10.1101/2023.07.16.547819'> 10.1101/2023.07.16.547819</a></p>
        <p class="abstract">The developmental decision made by malaria parasites to become sexual underlies all malaria transmission. Here, we describe a rich atlas of short and long-read single-cell transcriptomes of over 37,000 Plasmodium falciparum cells across intraerythrocytic asexual and sexual development. We used the atlas to explore transcriptional modules and exon usage along sexual development, and expanded it to include malaria parasites collected from a Malian individual naturally infected with multiple P. falciparum strains. We investigated genotypic and transcriptional heterogeneity within and among these wild strains at a single-cell level for the first time, finding considerable differential expression between different strains even within the same host. This work is a key addition to the Malaria Cell Atlas, enabling a deeper understanding of the biology and diversity of transmission stages.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549097">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549097" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549097">
        <p class="paperTitle">Recurrent chromosome destabilization through repeat-mediated rearrangements in a fungal pathogen</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549097" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549097" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fouche, S.; Oggenfuss, U.; McDonald, B. A.; Croll, D.</p>
        <p class="info">Score: 11.4, Published: 2023-07-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549097' target='https://doi.org/10.1101/2023.07.14.549097'> 10.1101/2023.07.14.549097</a></p>
        <p class="abstract">Genomic instability caused by chromosomal rearrangements has severe consequences for organismal fitness and progression of cancerous cell lines. The triggers of destabilized chromosomes remain poorly understood but are often assumed to be associated with fragile sites. Here, we retrace a runaway chromosomal degeneration process observed in a fungal pathogen using telomere-to-telomere assemblies across an experimental pedigree. We show that the same fragile sites triggered reproducible, large-scale rearrangements through non-allelic recombination. Across the four-generation pedigree, chromosomal rearrangements were accompanied by non-disjunction events and caused aneuploid progeny to carry up to four chromosomal copies. We identify a specific transposable element as the most likely trigger for the repeated chromosomal degeneration. The element is associated with higher virulence of the pathogen and has undergone a burst increasing copy numbers across the genome. Chromosome sequences are also targeted by a genome defense mechanism active on duplicated sequences, which may contribute to decay. Our work identifies the exact sequence triggers initiating chromosome instability and perpetuating degenerative cycles. Dissecting proximate causes leading to run-away chromosomal degeneration expands our understanding of chromosomal evolution beyond cancer lines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546998">
        <p class="paperTitle">Comprehensive Assessment of Eleven de novo HiFi Assemblers on Complex Eukaryotic Genomes and Metagenomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, W.; Luo, H.; Yang, J.; Zhang, S.; Jiang, H.; Zhao, X.; Hui, X.; Sun, D.; Li, L.; Wei, X.-q.; Lonardi, S.; Pan, W.</p>
        <p class="info">Score: 27.4, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546998' target='https://doi.org/10.1101/2023.06.29.546998'> 10.1101/2023.06.29.546998</a></p>
        <p class="abstract">BackgroundPacific Bioscience HiFi sequencing technology generates long reads (&gt;10 kbp) with very high accuracy (less than 0.01% sequencing error). While several de novo assembly tools are available for HiFi reads, there are no comprehensive studies on the evaluation of these assemblers.

ResultsWe evaluated the performance of eleven de novo HiFi assemblers on (i) real data for three eukaryotic genomes, (ii) 34 synthetic datasets with different ploidy, sequencing coverage levels, heterozygosity rates and sequencing error rates, (iii) one real metagenomic dataset, and (iv) five synthetic metagenomic datasets with different composition abundance and heterozygosity rates. The nine assemblers were evaluated using QUAST (Quality Assessment Tool) and BUSCO (Benchmarking Universal Single-Copy Ortholog). We also used several additional criteria, namely, completion rate, single-copy completion rate, duplicated completion rate, average proportion of largest category, average distance difference, quality value, run-time and memory utilization. On complex eukaryotic genomes, Hifiasm had a clear advantage over the other assemblers in all tested experiments. On synthetic datasets, Hifiasm, HiCanu, and HiFlye performed equally well. Shasta and Peregrine had good performance across varying ploidy, but required high computational resources. On metagenomic datasets, Hifiasm-meta demonstrated a clear advantage over other assemblers.

ConclusionWe carried out a comprehensive benchmarking study of commonly used assemblers on complex eukaryotic genomes and metagenomes. Our study will help the research community to choose the most appropriate assembler for their data and identify possible improvements in assembly algorithms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548521">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548521" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548521">
        <p class="paperTitle">High quality genome assembly and annotation (v1) of the eukaryotic terrestrial microalga Coccomyxa viridis SAG 216-4</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548521" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548521" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kraege, A.; Chavarro-Carrero, E.; Guiglielmoni, N.; Schnell, E.; Kirangwa, J.; Heilmann-Heimbach, S.; Becker, K.; Kohrer, K.; WGGC Team,  ; DeRGA Community,  ; Schiffer, P.; Thomma, B. P.; Rovenich, H.</p>
        <p class="info">Score: 9.0, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548521' target='https://doi.org/10.1101/2023.07.11.548521'> 10.1101/2023.07.11.548521</a></p>
        <p class="abstract">Unicellular green algae of the genus Coccomyxa are recognized for their worldwide distribution and ecological versatility. Most species described to date live in close association with various host species, such as in lichen associations. However, little is known about the molecular mechanisms that drive such symbiotic lifestyles. We generated a high-quality genome assembly for the lichen photobiont Coccomyxa viridis SAG 216-4 (formerly C. mucigena). Using long-read PacBio HiFi and Oxford Nanopore Technologies in combination with chromatin conformation capture (Hi-C) sequencing, we assembled the genome into 21 scaffolds with a total length of 50.9 Mb, an N50 of 2.7 Mb and a BUSCO score of 98.6%. While nineteen scaffolds represent full-length nuclear chromosomes, two additional scaffolds represent the mitochondrial and plastid genomes. Transcriptome-guided gene annotation resulted in the identification of 13,557 protein-coding genes, of which 68% have annotated PFAM domains and 962 are predicted to be secreted.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.10.548289">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.10.548289" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.10.548289">
        <p class="paperTitle">Detecting differential transcript usage in complex diseases with SPIT</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.10.548289" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.10.548289" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Erdogdu, B.; Varabyou, A.; Hicks, S. C.; Salzberg, S. L.; Pertea, M.</p>
        <p class="info">Score: 10.7, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.10.548289' target='https://doi.org/10.1101/2023.07.10.548289'> 10.1101/2023.07.10.548289</a></p>
        <p class="abstract">Differential transcript usage (DTU) plays a crucial role in determining how gene expression differs among cells, tissues, and different developmental stages, thereby contributing to the complexity and diversity of biological systems. In abnormal cells, it can also lead to deficiencies in protein function, potentially leading to pathogenesis of diseases. Detecting such events for single-gene genetic traits is relatively uncomplicated; however, the heterogeneity of populations with complex diseases presents an intricate challenge due to the presence of diverse causal events and undetermined subtypes. SPIT is the first statistical tool that quantifies the heterogeneity in transcript usage within a population and identifies predominant subgroups along with their distinctive sets of DTU events. We provide comprehensive assessments of SPITs methodology in both single-gene and complex traits and report the results of applying SPIT to analyze brain samples from individuals with schizophrenia. Our analysis reveals previously unreported DTU events in six candidate genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.548007">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.548007" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.548007">
        <p class="paperTitle">Topological stratification of continuous genetic variation in large biobanks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.548007" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.548007" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Diaz-Papkovich, A.; Zabad, S.; Ben-Eghan, C.; Anderson-Trocme, L.; Femerling, G.; Nathan, V.; Patel, J.; Gravel, S.</p>
        <p class="info">Score: 30.3, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.548007' target='https://doi.org/10.1101/2023.07.06.548007'> 10.1101/2023.07.06.548007</a></p>
        <p class="abstract">Biobanks now contain genetic data from millions of individuals. Dimension-ality reduction, visualization and stratification are standard when exploring data at these scales; while efficient and tractable methods exist for the first two, stratification remains challenging because of uncertainty about sources of population structure. In practice, stratification is commonly performed by drawing shapes around dimensionally reduced data or assuming populations have a &#34;type&#34; genome. We propose a method of stratifying data with topo-logical analysis that is fast, easy to implement, and integrates with existing pipelines. The approach is robust to the presence of sub-populations of vary-ing sizes and wide ranges of population structure patterns. We demonstrate its effectiveness on genotypes from three biobanks and illustrate how topolog-ical genetic strata can help us understand structure within biobanks, evaluate distributions of genotypic and phenotypic data, examine polygenic score trans-ferability, identify potential influential alleles, and perform quality control.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.16.549241">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.16.549241" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.16.549241">
        <p class="paperTitle">The expression landscape and pangenome of long non-coding RNA in the fungal wheat pathogen Zymoseptoria tritici</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.16.549241" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.16.549241" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Glad, H. M.; Moser Tralamazza, S.; Croll, D.</p>
        <p class="info">Score: 7.6, Published: 2023-07-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.16.549241' target='https://doi.org/10.1101/2023.07.16.549241'> 10.1101/2023.07.16.549241</a></p>
        <p class="abstract">Long non-coding RNAs (lncRNAs) are regulatory molecules interacting in a wide array of biological processes. LncRNAs in fungal pathogens can be responsive to stress and play roles in regulating growth and nutrient acquisition. Recent evidence suggests that lncRNAs may also play roles in virulence, such as regulating pathogenicity-associated enzymes and on-host reproductive cycles. Despite the importance of lncRNAs, only few model fungi have well-documented inventories of lncRNA. In this study, we apply a machine-learning based pipeline to predict high-confidence lncRNA candidates in Zymoseptoria tritici, an important global pathogen of wheat impacting global food production. We analyzed genomic features of lncRNAs and the most likely associated processes through analyses of expression over a host infection cycle. We find that lncRNAs are frequently expressed during early infection, before the switch to necrotrophic growth. They are mostly located in facultative heterochromatic regions, which are known to contain many genes associated with pathogenicity. Furthermore, we find that lncRNAs are frequently co-expressed with genes that may be involved in responding to host signals, such as those responses to oxidative stress. Finally, we assess pangenome features of lncRNAs using four additional reference-quality genomes. We find evidence that the repertoire of expressed lncRNAs varies substantially between individuals, even though lncRNA loci tend to be shared at the genomic level. Overall, this study provides a repertoire and putative functions of lncRNAs in Z. tritici enabling molecular genetics and functional analyses in an important pathogen.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549109">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549109" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549109">
        <p class="paperTitle">Genome-wide expression QTL mapping reveals the highly dynamic regulatory landscape of a major wheat pathogen</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549109" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549109" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abraham, L. N.; Croll, D.</p>
        <p class="info">Score: 6.5, Published: 2023-07-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549109' target='https://doi.org/10.1101/2023.07.14.549109'> 10.1101/2023.07.14.549109</a></p>
        <p class="abstract">In agricultural ecosystems, outbreaks of diseases are frequent and pose a significant threat to food security. A successful pathogen undergoes a complex and well-timed sequence of regulatory changes to avoid detection by the host immune system, hence well-tuned gene regulation is essential for survival. However, the extent to which the regulatory polymorphisms in a pathogen population provide an adaptive advantage is poorly understood. We used Zymoseptoria tritici, one of the most important pathogens of wheat, to generate a genome-wide map of regulatory polymorphism governing gene expression. We investigated genome-wide transcription levels of 146 strains grown under nutrient starvation and performed expression quantitative trait loci (eQTL) mapping. We identified cis-eQTLs for 65.3% of all genes and the majority of all eQTL loci are within 2kb upstream and downstream of the transcription start site (TSS). We also show that polymorphism in different gene elements contributes disproportionally to gene expression variation. Investigating regulatory polymorphism in gene categories, we found an enrichment of regulatory variants for genes predicted to be important for fungal pathogenesis but with comparatively low effect size, suggesting a separate layer of gene regulation involving epigenetics. We also show that previously reported trait-associated SNPs in pathogen populations are frequently cis-regulatory variants of neighboring genes with implications for the trait architecture. Overall, our study provides extensive evidence that single populations segregate large-scale regulatory variation and are likely to fuel rapid adaptation to resistant hosts and environmental change.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.545266">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.545266" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.545266">
        <p class="paperTitle">Cell type and condition specific functional annotation of schizophrenia associated non-coding genetic variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.545266" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.545266" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rummel, C. K.; Gagliardi, M.; Herholt, A.; Ahmad, R.; Murek, V.; Weigert, L.; Hausruckinger, A.; Maidl, S.; Jimenez-Barron, L.; Trastulla, L.; Hauger, B.; Eder, M.; Rossner, M. J.; Ziller, M. J.</p>
        <p class="info">Score: 8.6, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.545266' target='https://doi.org/10.1101/2023.06.27.545266'> 10.1101/2023.06.27.545266</a></p>
        <p class="abstract">Schizophrenia (SCZ) is a highly polygenic disease and genome wide association studies have identified thousands of genetic variants that are statistically associated with this psychiatric disorder. However, our ability to translate these associations into insights on the disease mechanisms has been challenging since the causal genetic variants, their molecular function and their target genes remain largely unknown. In order to address these questions, we established a functional genomics pipeline in combination with induced pluripotent stem cell technology to functionally characterize ~35,000 non-coding genetic variants associated with schizophrenia along with their target genes. This analysis identified a set of 620 (1.7%) single nucleotide polymorphisms as functional on a molecular level in a highly cell type and condition specific fashion. These results provide a high-resolution map of functional variant-gene combinations and offer comprehensive biological insights into the developmental context and stimulation dependent molecular processes modulated by SCZ associated genetic variation.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
