<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Exhaustive identification of genome-wide binding events of transcriptional regulators with ICEBERG
Authors: Nordin, A.; Pagella, P.; Zambanini, G.; Cantu, C.
Score: 42.4, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547050
Genome-wide protein interaction assays aspire to map the complete binding pattern of gene regulators. How-ever, common practice relies on replication and high stringency statistics which favor false negatives over false positives, thereby excluding portions of signal which may represent biologically relevant events. Here, we present ICEBERG (Increased Capture of Enrichment By Exhaustive Replicate aGgregation), an experimental and analytical pipeline that harnesses large numbers of CUT&amp;RUN replicates to discover the full set of binding events and chart the line between false positives and false negatives.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Exhaustive identification of genome-wide binding events of transcriptional regulators with ICEBERG
Authors: Nordin, A.; Pagella, P.; Zambanini, G.; Cantu, C.
Score: 42.4, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547050
Genome-wide protein interaction assays aspire to map the complete binding pattern of gene regulators. How-ever, common practice relies on replication and high stringency statistics which favor false negatives over false positives, thereby excluding portions of signal which may represent biologically relevant events. Here, we present ICEBERG (Increased Capture of Enrichment By Exhaustive Replicate aGgregation), an experimental and analytical pipeline that harnesses large numbers of CUT&amp;RUN replicates to discover the full set of binding events and chart the line between false positives and false negatives." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-09T10:40:17+00:00" />
<meta property="article:modified_time" content="2023-07-09T10:40:17+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Exhaustive identification of genome-wide binding events of transcriptional regulators with ICEBERG
Authors: Nordin, A.; Pagella, P.; Zambanini, G.; Cantu, C.
Score: 42.4, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547050
Genome-wide protein interaction assays aspire to map the complete binding pattern of gene regulators. How-ever, common practice relies on replication and high stringency statistics which favor false negatives over false positives, thereby excluding portions of signal which may represent biologically relevant events. Here, we present ICEBERG (Increased Capture of Enrichment By Exhaustive Replicate aGgregation), an experimental and analytical pipeline that harnesses large numbers of CUT&amp;RUN replicates to discover the full set of binding events and chart the line between false positives and false negatives."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Exhaustive identification of genome-wide binding events of transcriptional regulators with ICEBERG\nAuthors: Nordin, A.; Pagella, P.; Zambanini, G.; Cantu, C.\nScore: 42.4, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547050\nGenome-wide protein interaction assays aspire to map the complete binding pattern of gene regulators. How-ever, common practice relies on replication and high stringency statistics which favor false negatives over false positives, thereby excluding portions of signal which may represent biologically relevant events. Here, we present ICEBERG (Increased Capture of Enrichment By Exhaustive Replicate aGgregation), an experimental and analytical pipeline that harnesses large numbers of CUT\u0026amp;RUN replicates to discover the full set of binding events and chart the line between false positives and false negatives.",
  "keywords": [
    
  ],
  "articleBody": " Exhaustive identification of genome-wide binding events of transcriptional regulators with ICEBERG\nAuthors: Nordin, A.; Pagella, P.; Zambanini, G.; Cantu, C.\nScore: 42.4, Published: 2023-07-01 DOI: 10.1101/2023.06.29.547050\nGenome-wide protein interaction assays aspire to map the complete binding pattern of gene regulators. How-ever, common practice relies on replication and high stringency statistics which favor false negatives over false positives, thereby excluding portions of signal which may represent biologically relevant events. Here, we present ICEBERG (Increased Capture of Enrichment By Exhaustive Replicate aGgregation), an experimental and analytical pipeline that harnesses large numbers of CUT\u0026RUN replicates to discover the full set of binding events and chart the line between false positives and false negatives. We employed ICEBERG to map the full set of H3K4me3-marked regulatory regions and {beta}-catenin targets in human colorectal cancer cells. The ICE-BERG datasets allow benchmarking of individual replicates, comparison of the performance of peak calling and replication approaches and expose the arbitrary nature of other strategies to identify reproducible peaks. Instead of a static view of genomic targets, ICEBERG established a spectrum of detection probabilities across the genome for a given factor, underlying the intrinsic dynamicity of its mechanism of action, and permitting to distinguish frequent from rare regulation events. Finally, ICEBERG discovered instances, undetectable with other approaches, that might underlie novel mechanisms of colorectal cancer progression.\nThree-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample\nAuthors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O'Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J. P.; Pletch, K.; Ruiz-Herrera, A.; Lander, E. S.; Rowley, M. J.; Onuchic, J. N.; Dalen, L.; Marti-Renom, M. A.; Gilbert, M. T. P.; Aiden, E. L.\nScore: 46.6, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547175\nAncient DNA (aDNA) sequencing analysis typically involves alignment to a modern reference genome assembly from a related species. Since aDNA molecules are fragmentary, these alignments yield information about small-scale differences, but provide no information about larger features such as the chromosome structure of ancient species. We report the genome assembly of a female Late Pleistocene woolly mammoth (Mammuthus primigenius) with twenty-eight chromosome-length scaffolds, generated using mammoth skin preserved in permafrost for roughly 52,000 years. We began by creating a modified Hi-C protocol, dubbed PaleoHi-C, optimized for ancient samples, and using it to map chromatin contacts in a woolly mammoth. Next, we developed \"reference-assisted 3D genome assembly,\" which begins with a reference genome assembly from a related species, and uses Hi-C and DNA-Seq data from a target species to split, order, orient, and correct sequences on the basis of their 3D proximity, yielding accurate chromosome-length scaffolds for the target species. By means of this reference-assisted 3D genome assembly, PaleoHi-C data reveals the 3D architecture of a woolly mammoth genome, including chromosome territories, compartments, domains, and loops. The active (A) and inactive (B) genome compartments in mammoth skin more closely resemble those observed in Asian elephant skin than the compartmentalization patterns seen in other Asian elephant tissues. Differences in compartmentalization between these skin samples reveal sequences whose transcription was potentially altered in mammoth. We observe a tetradic structure for the inactive X chromosome in mammoth, distinct from the bipartite architecture seen in human and mouse. Generating chromosome-length genome assemblies for two other elephantids (Asian and African elephant), we find that the overall karyotype, and this tetradic Xi structure, are conserved throughout the clade. These results illustrate that cell-type specific epigenetic information can be preserved in ancient samples, in the form of DNA geometry, and that it may be feasible to perform de novo genome assembly of some extinct species.\nPersonal transcriptome variation is poorly explained by current genomic deep learning models\nAuthors: Huang, C.; Shuai, R.; Baokar, P.; Chung, R.; Rastogi, R.; Kathail, P.; Ioannidis, N. M.\nScore: 19.9, Published: 2023-06-30 DOI: 10.1101/2023.06.30.547100\nGenomic deep learning models can predict genome-wide epigenetic features and gene expression levels directly from DNA sequence. While current models perform well at predicting gene expression levels across genes in different cell types from the reference genome, their ability to explain expression variation between individuals due to cis-regulatory genetic variants remains largely unexplored. Here we evaluate four state-of-the-art models on paired personal genome and transcriptome data and find limited performance when explaining variation in expression across individuals.\nAllelic bias when performing in-solution enrichment of ancient human DNA\nAuthors: Davidson, R.; Williams, M. P.; Roca-Rada, X.; Kassadjikova, K.; Tobler, R.; Fehren-Schmitz, L.; Llamas, B.\nScore: 18.1, Published: 2023-07-05 DOI: 10.1101/2023.07.04.547445\nIn-solution hybridisation enrichment of genetic variation is a valuable methodology in human paleogenomics. It allows enrichment of endogenous DNA by targeting genetic markers that are comparable between sequencing libraries. Many studies have used the 1240k reagent-which enriches 1,237,207 genome-wide SNPs-since 2015, though access was restricted. In 2021, Twist Biosciences and Daicel Arbor Biosciences independently released commercial kits that enabled all researchers to perform enrichments for the same 1240k SNPs. We used the Daicel Arbor Biosciences Prime Plus kit to enrich 268 ancient samples from four continents. We identified a systematic assay bias that increases genetic similarity between enriched samples and that cannot be explained by batch effects. We present the impact of the bias on population genetics inferences (e.g., Principal Components Analysis, f-statistics) and genetic relatedness (READ). We compare the Prime Plus bias to that previously reported of the legacy 1240k enrichment assay. In f-statistics, we find that all Prime-Plus-generated data exhibit artefactual excess shared drift, such that within-continent relationships cannot be correctly determined. In PCA, the first PCs may capture the assay bias rather than the underlying genetic structure. The bias is more subtle in READ, though interpretation of the results can still be misleading in specific contexts. We expect the bias may affect analyses we have not yet tested. Our observations support previously reported concerns for the integration of different data types in paleogenomics. We also caution that technological solutions to generate 1240k data necessitate a thorough validation process before their adoption in the paleogenomic community.\nPangenome of cultivated beet and crop wild relatives reveals parental relationships of a tetraploid wild beet\nAuthors: Sielemann, K.; Schmidt, N.; Guzik, J.; Kalina, N.; Pucker, B.; Viehöver, P.; Breitenbach, S.; Weisshaar, B.; Heitkam, T.; Holtgräwe, D.\nScore: 24.0, Published: 2023-06-30 DOI: 10.1101/2023.06.28.546919\nMost crop plants, including sugar beet (Beta vulgaris subsp. vulgaris), suffer from domestication bottlenecks and low genetic diversity caused by extensive selection for few traits. However, crop wild relatives (CWRs) harbour useful traits relevant for crop improvement, including enhanced adaptation to biotic and abiotic stresses. Especially polyploids are interesting from an evolutionary perspective as genes undergo reorganisation after the polyploidisation event. Through neo-and subfunctionalisation, novel functions emerge, which enable plants to cope with changing environments and extreme/harsh conditions. Particularly in the face of climate change, specific stress and pathogen resistances or tolerances gain importance. To introduce such traits into breeding material, CWRs have already been identified as an important source for sustainable breeding. The identification of genes underlying traits of interest is crucial for crop improvement. For beets, the section Corollinae contains the tetraploid species Beta corolliflora (2n=4x=36) that harbours salt and frost tolerances as well as a wealth of pathogen resistances. The number of beneficial traits of B. corolliflora is increased compared to those of the known diploids in this section (all 2n=2x=18). Nevertheless, neither the parental relationships of B. corolliflora have been resolved, nor are genomic resources available to steer sustainable, genomics-informed breeding. To benefit from the resources offered by polyploid beet wild relatives, we generated a comprehensive pangenome dataset including B. corolliflora, Beta lomatogona, and Beta macrorhiza, as well as a more distant wild beet Patellifolia procumbens (2n=2x=18). Joined analyses with publicly available genome sequences of two additional wild beets allowed the identification of genomic regions absent from cultivated beet, providing a sequence database harbouring traits relevant for future breeding endeavours. In addition, we present strong evidence for the parental relationship of the B. corolliflora wild beet as an autotetraploid emerging from B. macrorhiza.\nSequence basis of transcription initiation in human genome\nAuthors: Dudnyk, K.; Shi, C.; Zhou, J.\nScore: 75.1, Published: 2023-06-29 DOI: 10.1101/2023.06.27.546584\nTranscription initiation is an essential process for ensuring proper function of any gene, however, a unified understanding of sequence patterns and rules that determine transcription initiation sites in human genome remains elusive. By explaining transcription initiation at basepair resolution from sequence with a deep learning-inspired explainable modeling approach, here we show that simple rules can explain the vast majority of human promoters. We identified key sequence patterns that contribute to human promoter function, each activating transcription with a distinct position-specific effect curve that likely reflects its mechanism of promoting transcription initiation. Most of these position-specific effects have not been previously characterized, and we verified them using experimental perturbations of transcription factors and sequences. We revealed the sequence basis of bidirectional transcription at promoters and links between promoter selectivity and gene expression variation across cell types. Additionally, by analyzing 241 mammalian genomes and mouse transcription initiation site data, we showed that the sequence determinants are conserved across mammalian species. Taken together, we provide a unified model of the sequence basis of transcription initiation at the basepair level that is broadly applicable across mammalian species, and shed new light on basic questions related to promoter sequence and function.\nInterpreting Cis-Regulatory Interactions from Large-Scale Deep Neural Networks for Genomics\nAuthors: Toneyan, S.; Koo, P. K.\nScore: 15.1, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547592\nThe rise of large-scale, sequence-based deep neural networks (DNNs) for predicting gene expression has introduced challenges in their evaluation and interpretation. Current evaluations align DNN predictions with experimental perturbation assays, offering a limited perspective of the DNN's capabilities within the studied loci. Moreover, existing model explainability tools mainly focus on motif analysis, which becomes complex to interpret for longer sequences. Here we introduce CREME, an in silico perturbation toolkit that interrogates large-scale DNNs to uncover rules of gene regulation that it has learned. Using CREME, we investigate Enformer, a prominent DNN in gene expression prediction, revealing cis-regulatory elements (CREs) that directly enhance or silence target genes. We explore the relationship between CRE distance from transcription start sites and gene expression, as well as the intricate complexity of higher-order CRE interactions. This work advances the ability to translate the powerful predictions of large-scale DNNs to study open questions in gene regulation.\nContextualising samples: Supporting reference genomes for European biodiversity through sample and associated metadata collection\nAuthors: Boehne, A.; Fernandez, R.; Leonard, J. A.; McCartney, A. M.; McTaggart, S.; Melo-Ferreira, J.; Monteiro, R.; Oomen, R. A.; Vinnere Pettersson, O.; Struck, T. H.\nScore: 15.6, Published: 2023-06-30 DOI: 10.1101/2023.06.28.546652\nThe European Reference Genome Atlas (ERGA) consortium aims to generate a reference genome catalogue for all of Europes eukaryotic biodiversity. The biological material underlying this mission, the specimens and their derived samples, are provided through ERGAs pan-European network. To demonstrate the communitys capability and capacity to realise ERGAs ambitious mission, the ERGA Pilot project was initiated. In support of the ERGA Pilot effort to generate reference genomes for European biodiversity, the ERGA Sampling and Sample Processing committee (SSP) was formed by volunteer experts from ERGAs member base. SSP aims to aid participating researchers through i) establishing standards for and collecting of sample/ specimen metadata; ii) prioritisation of species for genome sequencing; and iii) development of taxon-specific collection guidelines including logistics support. SSP serves as the sample providers entry point to the ERGA genomic resource production infrastructure and guarantees that ERGAs high-quality standards are upheld throughout sample collection and processing. With the volume of researchers, projects, consortia, and organisations with interests in genomics resources expanding, this manuscript shares important experiences and lessons learned during the development of standardised operational procedures and sample provider support. The manuscript details our experiences in incorporating the FAIR and CARE principles, species prioritisation, and workflow development, which could be useful to individuals as well as other initiatives.\nNon-catalytic function of PRC2 in the control of small RNA dynamics during programmed genome elimination in Paramecium\nAuthors: Miro-Pina, C.; Arnaiz, O.; de Vanssay, A.; Frapporti, A.; Charmant, O.; Humbert, A.; Lhuillier-Akakpo, M.; Duharcourt, S.\nScore: 13.7, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547679\nTo limit transposable element (TE) mobilization, most eukaryotes have evolved small RNAs to silence TE activity via homology-dependent mechanisms. Small RNAs, 20-30 nucleotides in length, bind to PIWI proteins and guide them to nascent transcripts by sequence complementarity, triggering the recruitment of histone methyltransferase enzymes on chromatin to repress the transcriptional activity of TEs and other repeats. In the ciliate Paramecium tetraurelia, 25-nt scnRNAs corresponding to TEs recruit Polycomb Repressive Complex 2 (PRC2), and trigger their elimination during the formation of the somatic nucleus. Here, we sequenced sRNAs during the entire sexual cycle with unprecedented precision. Our data confirmed that scnRNAs are produced from the entire germline genome, from TEs and non-TE sequences, during meiosis. Non-TE scnRNAs are selectively degraded, which results in the specific selection of TE-scnRNAs. We provide important mechanistic insight into the scnRNA selection pathway by identifying PRC2 and its cofactors as essential for the selective degradation of non-TE-scnRNAs. Our findings reveal a new mechanism for PRC2 that involves a non-methyltransferase function for regulating small RNA dynamics during development.\nComprehensive Assessment of Eleven de novo HiFi Assemblers on Complex Eukaryotic Genomes and Metagenomes\nAuthors: Yu, W.; Luo, H.; Yang, J.; Zhang, S.; Jiang, H.; Zhao, X.; Hui, X.; Sun, D.; Li, L.; Wei, X.-q.; Lonardi, S.; Pan, W.\nScore: 16.3, Published: 2023-06-29 DOI: 10.1101/2023.06.29.546998\nBackgroundPacific Bioscience HiFi sequencing technology generates long reads (\u003e10 kbp) with very high accuracy (less than 0.01% sequencing error). While several de novo assembly tools are available for HiFi reads, there are no comprehensive studies on the evaluation of these assemblers. ResultsWe evaluated the performance of eleven de novo HiFi assemblers on (i) real data for three eukaryotic genomes, (ii) 34 synthetic datasets with different ploidy, sequencing coverage levels, heterozygosity rates and sequencing error rates, (iii) one real metagenomic dataset, and (iv) five synthetic metagenomic datasets with different composition abundance and heterozygosity rates. The nine assemblers were evaluated using QUAST (Quality Assessment Tool) and BUSCO (Benchmarking Universal Single-Copy Ortholog). We also used several additional criteria, namely, completion rate, single-copy completion rate, duplicated completion rate, average proportion of largest category, average distance difference, quality value, run-time and memory utilization. On complex eukaryotic genomes, Hifiasm had a clear advantage over the other assemblers in all tested experiments. On synthetic datasets, Hifiasm, HiCanu, and HiFlye performed equally well. Shasta and Peregrine had good performance across varying ploidy, but required high computational resources. On metagenomic datasets, Hifiasm-meta demonstrated a clear advantage over other assemblers. ConclusionWe carried out a comprehensive benchmarking study of commonly used assemblers on complex eukaryotic genomes and metagenomes. Our study will help the research community to choose the most appropriate assembler for their data and identify possible improvements in assembly algorithms.\n",
  "wordCount" : "2481",
  "inLanguage": "en",
  "datePublished": "2023-07-09T10:40:17Z",
  "dateModified": "2023-07-09T10:40:17Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on July 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.547050">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.547050" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.547050">
        <p class="paperTitle">Exhaustive identification of genome-wide binding events of transcriptional regulators with ICEBERG</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.547050" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.547050" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nordin, A.; Pagella, P.; Zambanini, G.; Cantu, C.</p>
        <p class="info">Score: 42.4, Published: 2023-07-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.547050' target='https://doi.org/10.1101/2023.06.29.547050'> 10.1101/2023.06.29.547050</a></p>
        <p class="abstract">Genome-wide protein interaction assays aspire to map the complete binding pattern of gene regulators. How-ever, common practice relies on replication and high stringency statistics which favor false negatives over false positives, thereby excluding portions of signal which may represent biologically relevant events. Here, we present ICEBERG (Increased Capture of Enrichment By Exhaustive Replicate aGgregation), an experimental and analytical pipeline that harnesses large numbers of CUT&amp;RUN replicates to discover the full set of binding events and chart the line between false positives and false negatives. We employed ICEBERG to map the full set of H3K4me3-marked regulatory regions and {beta}-catenin targets in human colorectal cancer cells. The ICE-BERG datasets allow benchmarking of individual replicates, comparison of the performance of peak calling and replication approaches and expose the arbitrary nature of other strategies to identify reproducible peaks. Instead of a static view of genomic targets, ICEBERG established a spectrum of detection probabilities across the genome for a given factor, underlying the intrinsic dynamicity of its mechanism of action, and permitting to distinguish frequent from rare regulation events. Finally, ICEBERG discovered instances, undetectable with other approaches, that might underlie novel mechanisms of colorectal cancer progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547175">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547175" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547175">
        <p class="paperTitle">Three-dimensional genome architecture persists in a 52,000-year-old woolly mammoth skin sample</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547175" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547175" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sandoval-Velasco, M.; Dudchenko, O.; Antonio Rodriguez, J.; Perez Estrada, C.; Dehasque, M.; Fontsere, C.; Mak, S. S. T.; Plotnikov, V.; Khan, R.; Weisz, D.; Contessoto, V. G.; Oliveira Junior, A. B.; Kalluchi, A.; Omer, A. D.; Batra, S. S.; Shamim, M. S.; Durand, N. C.; O&#39;Connell, B.; Roca, A. L.; Gnirke, A.; Garcia-Trevino, I.; Coke, R.; Flanagan, J. P.; Pletch, K.; Ruiz-Herrera, A.; Lander, E. S.; Rowley, M. J.; Onuchic, J. N.; Dalen, L.; Marti-Renom, M. A.; Gilbert, M. T. P.; Aiden, E. L.</p>
        <p class="info">Score: 46.6, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547175' target='https://doi.org/10.1101/2023.06.30.547175'> 10.1101/2023.06.30.547175</a></p>
        <p class="abstract">Ancient DNA (aDNA) sequencing analysis typically involves alignment to a modern reference genome assembly from a related species. Since aDNA molecules are fragmentary, these alignments yield information about small-scale differences, but provide no information about larger features such as the chromosome structure of ancient species. We report the genome assembly of a female Late Pleistocene woolly mammoth (Mammuthus primigenius) with twenty-eight chromosome-length scaffolds, generated using mammoth skin preserved in permafrost for roughly 52,000 years. We began by creating a modified Hi-C protocol, dubbed PaleoHi-C, optimized for ancient samples, and using it to map chromatin contacts in a woolly mammoth. Next, we developed &#34;reference-assisted 3D genome assembly,&#34; which begins with a reference genome assembly from a related species, and uses Hi-C and DNA-Seq data from a target species to split, order, orient, and correct sequences on the basis of their 3D proximity, yielding accurate chromosome-length scaffolds for the target species. By means of this reference-assisted 3D genome assembly, PaleoHi-C data reveals the 3D architecture of a woolly mammoth genome, including chromosome territories, compartments, domains, and loops. The active (A) and inactive (B) genome compartments in mammoth skin more closely resemble those observed in Asian elephant skin than the compartmentalization patterns seen in other Asian elephant tissues. Differences in compartmentalization between these skin samples reveal sequences whose transcription was potentially altered in mammoth. We observe a tetradic structure for the inactive X chromosome in mammoth, distinct from the bipartite architecture seen in human and mouse. Generating chromosome-length genome assemblies for two other elephantids (Asian and African elephant), we find that the overall karyotype, and this tetradic Xi structure, are conserved throughout the clade. These results illustrate that cell-type specific epigenetic information can be preserved in ancient samples, in the form of DNA geometry, and that it may be feasible to perform de novo genome assembly of some extinct species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.30.547100">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.30.547100" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.30.547100">
        <p class="paperTitle">Personal transcriptome variation is poorly explained by current genomic deep learning models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.30.547100" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.30.547100" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, C.; Shuai, R.; Baokar, P.; Chung, R.; Rastogi, R.; Kathail, P.; Ioannidis, N. M.</p>
        <p class="info">Score: 19.9, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.30.547100' target='https://doi.org/10.1101/2023.06.30.547100'> 10.1101/2023.06.30.547100</a></p>
        <p class="abstract">Genomic deep learning models can predict genome-wide epigenetic features and gene expression levels directly from DNA sequence. While current models perform well at predicting gene expression levels across genes in different cell types from the reference genome, their ability to explain expression variation between individuals due to cis-regulatory genetic variants remains largely unexplored. Here we evaluate four state-of-the-art models on paired personal genome and transcriptome data and find limited performance when explaining variation in expression across individuals.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547445">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547445" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547445">
        <p class="paperTitle">Allelic bias when performing in-solution enrichment of ancient human DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547445" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547445" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davidson, R.; Williams, M. P.; Roca-Rada, X.; Kassadjikova, K.; Tobler, R.; Fehren-Schmitz, L.; Llamas, B.</p>
        <p class="info">Score: 18.1, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547445' target='https://doi.org/10.1101/2023.07.04.547445'> 10.1101/2023.07.04.547445</a></p>
        <p class="abstract">In-solution hybridisation enrichment of genetic variation is a valuable methodology in human paleogenomics. It allows enrichment of endogenous DNA by targeting genetic markers that are comparable between sequencing libraries. Many studies have used the 1240k reagent-which enriches 1,237,207 genome-wide SNPs-since 2015, though access was restricted. In 2021, Twist Biosciences and Daicel Arbor Biosciences independently released commercial kits that enabled all researchers to perform enrichments for the same 1240k SNPs. We used the Daicel Arbor Biosciences Prime Plus kit to enrich 268 ancient samples from four continents. We identified a systematic assay bias that increases genetic similarity between enriched samples and that cannot be explained by batch effects. We present the impact of the bias on population genetics inferences (e.g., Principal Components Analysis, f-statistics) and genetic relatedness (READ). We compare the Prime Plus bias to that previously reported of the legacy 1240k enrichment assay. In f-statistics, we find that all Prime-Plus-generated data exhibit artefactual excess shared drift, such that within-continent relationships cannot be correctly determined. In PCA, the first PCs may capture the assay bias rather than the underlying genetic structure. The bias is more subtle in READ, though interpretation of the results can still be misleading in specific contexts. We expect the bias may affect analyses we have not yet tested. Our observations support previously reported concerns for the integration of different data types in paleogenomics. We also caution that technological solutions to generate 1240k data necessitate a thorough validation process before their adoption in the paleogenomic community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546919">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546919" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546919">
        <p class="paperTitle">Pangenome of cultivated beet and crop wild relatives reveals parental relationships of a tetraploid wild beet</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546919" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546919" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sielemann, K.; Schmidt, N.; Guzik, J.; Kalina, N.; Pucker, B.; Viehöver, P.; Breitenbach, S.; Weisshaar, B.; Heitkam, T.; Holtgräwe, D.</p>
        <p class="info">Score: 24.0, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546919' target='https://doi.org/10.1101/2023.06.28.546919'> 10.1101/2023.06.28.546919</a></p>
        <p class="abstract">Most crop plants, including sugar beet (Beta vulgaris subsp. vulgaris), suffer from domestication bottlenecks and low genetic diversity caused by extensive selection for few traits. However, crop wild relatives (CWRs) harbour useful traits relevant for crop improvement, including enhanced adaptation to biotic and abiotic stresses.

Especially polyploids are interesting from an evolutionary perspective as genes undergo reorganisation after the polyploidisation event. Through neo-and subfunctionalisation, novel functions emerge, which enable plants to cope with changing environments and extreme/harsh conditions. Particularly in the face of climate change, specific stress and pathogen resistances or tolerances gain importance. To introduce such traits into breeding material, CWRs have already been identified as an important source for sustainable breeding. The identification of genes underlying traits of interest is crucial for crop improvement.

For beets, the section Corollinae contains the tetraploid species Beta corolliflora (2n=4x=36) that harbours salt and frost tolerances as well as a wealth of pathogen resistances. The number of beneficial traits of B. corolliflora is increased compared to those of the known diploids in this section (all 2n=2x=18). Nevertheless, neither the parental relationships of B. corolliflora have been resolved, nor are genomic resources available to steer sustainable, genomics-informed breeding.

To benefit from the resources offered by polyploid beet wild relatives, we generated a comprehensive pangenome dataset including B. corolliflora, Beta lomatogona, and Beta macrorhiza, as well as a more distant wild beet Patellifolia procumbens (2n=2x=18). Joined analyses with publicly available genome sequences of two additional wild beets allowed the identification of genomic regions absent from cultivated beet, providing a sequence database harbouring traits relevant for future breeding endeavours. In addition, we present strong evidence for the parental relationship of the B. corolliflora wild beet as an autotetraploid emerging from B. macrorhiza.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.27.546584">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.27.546584" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.27.546584">
        <p class="paperTitle">Sequence basis of transcription initiation in human genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.27.546584" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.27.546584" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dudnyk, K.; Shi, C.; Zhou, J.</p>
        <p class="info">Score: 75.1, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.27.546584' target='https://doi.org/10.1101/2023.06.27.546584'> 10.1101/2023.06.27.546584</a></p>
        <p class="abstract">Transcription initiation is an essential process for ensuring proper function of any gene, however, a unified understanding of sequence patterns and rules that determine transcription initiation sites in human genome remains elusive. By explaining transcription initiation at basepair resolution from sequence with a deep learning-inspired explainable modeling approach, here we show that simple rules can explain the vast majority of human promoters. We identified key sequence patterns that contribute to human promoter function, each activating transcription with a distinct position-specific effect curve that likely reflects its mechanism of promoting transcription initiation. Most of these position-specific effects have not been previously characterized, and we verified them using experimental perturbations of transcription factors and sequences. We revealed the sequence basis of bidirectional transcription at promoters and links between promoter selectivity and gene expression variation across cell types. Additionally, by analyzing 241 mammalian genomes and mouse transcription initiation site data, we showed that the sequence determinants are conserved across mammalian species. Taken together, we provide a unified model of the sequence basis of transcription initiation at the basepair level that is broadly applicable across mammalian species, and shed new light on basic questions related to promoter sequence and function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547592">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547592" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547592">
        <p class="paperTitle">Interpreting Cis-Regulatory Interactions from Large-Scale Deep Neural Networks for Genomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547592" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547592" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Toneyan, S.; Koo, P. K.</p>
        <p class="info">Score: 15.1, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547592' target='https://doi.org/10.1101/2023.07.03.547592'> 10.1101/2023.07.03.547592</a></p>
        <p class="abstract">The rise of large-scale, sequence-based deep neural networks (DNNs) for predicting gene expression has introduced challenges in their evaluation and interpretation. Current evaluations align DNN predictions with experimental perturbation assays, offering a limited perspective of the DNN&#39;s capabilities within the studied loci. Moreover, existing model explainability tools mainly focus on motif analysis, which becomes complex to interpret for longer sequences. Here we introduce CREME, an in silico perturbation toolkit that interrogates large-scale DNNs to uncover rules of gene regulation that it has learned. Using CREME, we investigate Enformer, a prominent DNN in gene expression prediction, revealing cis-regulatory elements (CREs) that directly enhance or silence target genes. We explore the relationship between CRE distance from transcription start sites and gene expression, as well as the intricate complexity of higher-order CRE interactions. This work advances the ability to translate the powerful predictions of large-scale DNNs to study open questions in gene regulation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546652">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546652" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546652">
        <p class="paperTitle">Contextualising samples: Supporting reference genomes for European biodiversity through sample and associated metadata collection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546652" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546652" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Boehne, A.; Fernandez, R.; Leonard, J. A.; McCartney, A. M.; McTaggart, S.; Melo-Ferreira, J.; Monteiro, R.; Oomen, R. A.; Vinnere Pettersson, O.; Struck, T. H.</p>
        <p class="info">Score: 15.6, Published: 2023-06-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546652' target='https://doi.org/10.1101/2023.06.28.546652'> 10.1101/2023.06.28.546652</a></p>
        <p class="abstract">The European Reference Genome Atlas (ERGA) consortium aims to generate a reference genome catalogue for all of Europes eukaryotic biodiversity. The biological material underlying this mission, the specimens and their derived samples, are provided through ERGAs pan-European network. To demonstrate the communitys capability and capacity to realise ERGAs ambitious mission, the ERGA Pilot project was initiated. In support of the ERGA Pilot effort to generate reference genomes for European biodiversity, the ERGA Sampling and Sample Processing committee (SSP) was formed by volunteer experts from ERGAs member base. SSP aims to aid participating researchers through i) establishing standards for and collecting of sample/ specimen metadata; ii) prioritisation of species for genome sequencing; and iii) development of taxon-specific collection guidelines including logistics support. SSP serves as the sample providers entry point to the ERGA genomic resource production infrastructure and guarantees that ERGAs high-quality standards are upheld throughout sample collection and processing. With the volume of researchers, projects, consortia, and organisations with interests in genomics resources expanding, this manuscript shares important experiences and lessons learned during the development of standardised operational procedures and sample provider support. The manuscript details our experiences in incorporating the FAIR and CARE principles, species prioritisation, and workflow development, which could be useful to individuals as well as other initiatives.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547679">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547679" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547679">
        <p class="paperTitle">Non-catalytic function of PRC2 in the control of small RNA dynamics during programmed genome elimination in Paramecium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547679" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547679" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miro-Pina, C.; Arnaiz, O.; de Vanssay, A.; Frapporti, A.; Charmant, O.; Humbert, A.; Lhuillier-Akakpo, M.; Duharcourt, S.</p>
        <p class="info">Score: 13.7, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547679' target='https://doi.org/10.1101/2023.07.04.547679'> 10.1101/2023.07.04.547679</a></p>
        <p class="abstract">To limit transposable element (TE) mobilization, most eukaryotes have evolved small RNAs to silence TE activity via homology-dependent mechanisms. Small RNAs, 20-30 nucleotides in length, bind to PIWI proteins and guide them to nascent transcripts by sequence complementarity, triggering the recruitment of histone methyltransferase enzymes on chromatin to repress the transcriptional activity of TEs and other repeats. In the ciliate Paramecium tetraurelia, 25-nt scnRNAs corresponding to TEs recruit Polycomb Repressive Complex 2 (PRC2), and trigger their elimination during the formation of the somatic nucleus. Here, we sequenced sRNAs during the entire sexual cycle with unprecedented precision. Our data confirmed that scnRNAs are produced from the entire germline genome, from TEs and non-TE sequences, during meiosis. Non-TE scnRNAs are selectively degraded, which results in the specific selection of TE-scnRNAs. We provide important mechanistic insight into the scnRNA selection pathway by identifying PRC2 and its cofactors as essential for the selective degradation of non-TE-scnRNAs. Our findings reveal a new mechanism for PRC2 that involves a non-methyltransferase function for regulating small RNA dynamics during development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.29.546998">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.29.546998" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.29.546998">
        <p class="paperTitle">Comprehensive Assessment of Eleven de novo HiFi Assemblers on Complex Eukaryotic Genomes and Metagenomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.29.546998" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.29.546998" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yu, W.; Luo, H.; Yang, J.; Zhang, S.; Jiang, H.; Zhao, X.; Hui, X.; Sun, D.; Li, L.; Wei, X.-q.; Lonardi, S.; Pan, W.</p>
        <p class="info">Score: 16.3, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.29.546998' target='https://doi.org/10.1101/2023.06.29.546998'> 10.1101/2023.06.29.546998</a></p>
        <p class="abstract">BackgroundPacific Bioscience HiFi sequencing technology generates long reads (&gt;10 kbp) with very high accuracy (less than 0.01% sequencing error). While several de novo assembly tools are available for HiFi reads, there are no comprehensive studies on the evaluation of these assemblers.

ResultsWe evaluated the performance of eleven de novo HiFi assemblers on (i) real data for three eukaryotic genomes, (ii) 34 synthetic datasets with different ploidy, sequencing coverage levels, heterozygosity rates and sequencing error rates, (iii) one real metagenomic dataset, and (iv) five synthetic metagenomic datasets with different composition abundance and heterozygosity rates. The nine assemblers were evaluated using QUAST (Quality Assessment Tool) and BUSCO (Benchmarking Universal Single-Copy Ortholog). We also used several additional criteria, namely, completion rate, single-copy completion rate, duplicated completion rate, average proportion of largest category, average distance difference, quality value, run-time and memory utilization. On complex eukaryotic genomes, Hifiasm had a clear advantage over the other assemblers in all tested experiments. On synthetic datasets, Hifiasm, HiCanu, and HiFlye performed equally well. Shasta and Peregrine had good performance across varying ploidy, but required high computational resources. On metagenomic datasets, Hifiasm-meta demonstrated a clear advantage over other assemblers.

ConclusionWe carried out a comprehensive benchmarking study of commonly used assemblers on complex eukaryotic genomes and metagenomes. Our study will help the research community to choose the most appropriate assembler for their data and identify possible improvements in assembly algorithms.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
