<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Simultaneous Profiling of Chromatin Accessibility and DNA Methylation in Complete Plant Genomes Using Long-Read Sequencing
Authors: Leduque, B.; Edera, A.; Vitte, C.; Quadrana, L.
Score: 21.6, Published: 2023-11-21 DOI: 10.1101/2023.11.15.567180
Epigenetic regulations, including chromatin accessibility, nucleosome positioning, and DNA methylation intricately shape genome function. However, current chromatin profiling techniques relying on short-read sequencing technologies fail to characterise highly repetitive genomic regions and cannot detect multiple chromatin features simultaneously. Here, we performed Simultaneous Accessibility and DNA Methylation Sequencing (SAM-seq) of purified plant nuclei.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Simultaneous Profiling of Chromatin Accessibility and DNA Methylation in Complete Plant Genomes Using Long-Read Sequencing
Authors: Leduque, B.; Edera, A.; Vitte, C.; Quadrana, L.
Score: 21.6, Published: 2023-11-21 DOI: 10.1101/2023.11.15.567180
Epigenetic regulations, including chromatin accessibility, nucleosome positioning, and DNA methylation intricately shape genome function. However, current chromatin profiling techniques relying on short-read sequencing technologies fail to characterise highly repetitive genomic regions and cannot detect multiple chromatin features simultaneously. Here, we performed Simultaneous Accessibility and DNA Methylation Sequencing (SAM-seq) of purified plant nuclei." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-26T10:37:33+00:00" />
<meta property="article:modified_time" content="2023-11-26T10:37:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Simultaneous Profiling of Chromatin Accessibility and DNA Methylation in Complete Plant Genomes Using Long-Read Sequencing
Authors: Leduque, B.; Edera, A.; Vitte, C.; Quadrana, L.
Score: 21.6, Published: 2023-11-21 DOI: 10.1101/2023.11.15.567180
Epigenetic regulations, including chromatin accessibility, nucleosome positioning, and DNA methylation intricately shape genome function. However, current chromatin profiling techniques relying on short-read sequencing technologies fail to characterise highly repetitive genomic regions and cannot detect multiple chromatin features simultaneously. Here, we performed Simultaneous Accessibility and DNA Methylation Sequencing (SAM-seq) of purified plant nuclei."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Simultaneous Profiling of Chromatin Accessibility and DNA Methylation in Complete Plant Genomes Using Long-Read Sequencing\nAuthors: Leduque, B.; Edera, A.; Vitte, C.; Quadrana, L.\nScore: 21.6, Published: 2023-11-21 DOI: 10.1101/2023.11.15.567180\nEpigenetic regulations, including chromatin accessibility, nucleosome positioning, and DNA methylation intricately shape genome function. However, current chromatin profiling techniques relying on short-read sequencing technologies fail to characterise highly repetitive genomic regions and cannot detect multiple chromatin features simultaneously. Here, we performed Simultaneous Accessibility and DNA Methylation Sequencing (SAM-seq) of purified plant nuclei.",
  "keywords": [
    
  ],
  "articleBody": " Simultaneous Profiling of Chromatin Accessibility and DNA Methylation in Complete Plant Genomes Using Long-Read Sequencing\nAuthors: Leduque, B.; Edera, A.; Vitte, C.; Quadrana, L.\nScore: 21.6, Published: 2023-11-21 DOI: 10.1101/2023.11.15.567180\nEpigenetic regulations, including chromatin accessibility, nucleosome positioning, and DNA methylation intricately shape genome function. However, current chromatin profiling techniques relying on short-read sequencing technologies fail to characterise highly repetitive genomic regions and cannot detect multiple chromatin features simultaneously. Here, we performed Simultaneous Accessibility and DNA Methylation Sequencing (SAM-seq) of purified plant nuclei. Thanks to the use of long-read nanopore sequencing, SAM-seq enables high-resolution profiling of m6A-tagged chromatin accessibility together with endogenous cytosine methylation in plants. Analysis of naked genomic DNA revealed significant sequence preference biases of m6A-MTases, controllable through a normalisation step. By applying SAM-seq to Arabidopsis and maize nuclei we obtained fine-grained accessibility and DNA methylation landscapes genome-wide. In addition, we uncovered crosstalk between chromatin accessibility and DNA methylation within nucleosomes of genes, TEs, and centromeric repeats. SAM-seq also facilitated the identification of DNA footprints over cis-regulatory regions. Furthermore, using the single-molecule information provided by SAM-seq we identified extensive cellular heterogeneity at chromatin domains harbouring antagonistic chromatin marks, suggesting that bivalency reflects cell-specific regulations of gene activity. SAM-seq is a powerful approach to simultaneously study multiple epigenetic features over unique and repetitive sequences, opening new opportunities for the investigation of epigenetic mechanisms in model as well non-model plant species with limited genomic and epigenomic information.\nMPRAbase: A Massively Parallel Reporter Assay Database\nAuthors: Zhao, J.; Baltoumas, F. A.; Konnaris, M. A.; Mouratidis, I.; Liu, Z.; Sims, J.; Agarwal, V.; Pavlopoulos, G. A.; Georgakopoulos-Soares, I.; Ahituv, N.\nScore: 18.2, Published: 2023-11-22 DOI: 10.1101/2023.11.19.567742\nMassively parallel reporter assays (MPRAs) represent a set of high-throughput technologies that measure the functional effects of thousands of sequences/variants on gene regulatory activity. There are several different variations of MPRA technology and they are used for numerous applications, including regulatory element discovery, variant effect measurement, saturation mutagenesis, synthetic regulatory element generation or characterization of evolutionary gene regulatory differences. Despite their many designs and uses, there is no comprehensive database that incorporates the results of these experiments. To address this, we developed MPRAbase, a manually curated database that currently harbors 129 experiments, encompassing 17,718,677 elements tested across 35 cell types and 4 organisms. The MPRAbase web interface (http://www.mprabase.com) serves as a centralized user-friendly repository to download existing MPRA data for independent analysis and is designed with the ability to allow researchers to share their published data for rapid dissemination to the community.\nDNA-binding factor footprints and enhancer RNAs identify functional non-coding genetic variants\nAuthors: Biddie, S. C.; Hird, E. F.; Weykopf, G.; Friman, E. T.; Bickmore, W. A.\nScore: 16.0, Published: 2023-11-20 DOI: 10.1101/2023.11.20.567860\nGenome-wide association studies (GWAS) have revealed a multitude of candidate genetic variants affecting the risk of developing complex traits and diseases. However, these highlighted regions are typically in the non-coding genome, and uncovering the functional causative single nucleotide variants (SNVs) is challenging. Prioritisation of variants is commonly based on functional genomic annotation with markers of active regulatory elements, but current approaches still poorly predict functional variants. To address this, we systematically analyse six markers of active regulatory elements for their ability to identify functional variants. We benchmark against molecular quantitative trait loci (molQTL) from assays of regulatory element activity that identify allelic effects on DNA-binding factor occupancy, reporter assay expression, and chromatin accessibility. We identify the combination of DNase footprints and divergent enhancer RNA as markers for functional variants. This signature provides high precision, trading-off low recall, thus substantially reducing candidate variant sets to prioritise variants for functional validation. We present this as a framework called FINDER - Functional SNV IdeNtification using DNase footprints and Enhancer RNA, and demonstrate its utility to prioritise variants using leukocyte count trait and analyse variants in linkage disequilibrium with a lead variant to predict a functional variant in asthma. Our findings have implications for prioritising variants from GWAS, in development of predictive scoring algorithms, and for functionally informed fine mapping approaches.\nAn appropriate differential expression analysis in single-cell data depends on the study design\nAuthors: Lee, H.; Han, B.\nScore: 15.1, Published: 2023-11-20 DOI: 10.1101/2023.01.09.523212\nLarge-scale multi-subject single-cell data have become very common. However, these data have high heterogeneity in study designs, leading to confusion in differential gene expression (DGE) analysis. In this work, we show that pseudobulk produces a substantial amount of type 2 error when the group label being compared varies within a subject. When the comparison label is constant within a subject, type 1 error is inflated if pseudoreplication is not accounted for. As a general principle, we show that an appropriate method depends on the design of the study. We propose solutions to the inflated error rates and provide practical guidelines for researchers.\nkmerDB: A Database Encompassing the Set of Genomic and Proteomic Sequence Information for Each Species\nAuthors: Mouratidis, I.; Baltoumas, F. A.; Chantzi, N.; Chan, C.; Montgomery, A.; Konnaris, M. A.; Georgakopoulos, G. C.; Das, A.; Chartoumpekis, D.; Kovac, J.; Pavlopoulos, G.; Georgakopoulos-Soares, I.\nScore: 12.1, Published: 2023-11-16 DOI: 10.1101/2023.11.13.566926\nThe rapid decline in sequencing cost has enabled the generation of reference genomes and proteomes for a growing number of organisms. However, at the present time, there is no established repository that provides information about organism-specific genomic and proteomic sequences of certain lengths, also known as kmers, that are either present or absent in each genome or proteome. In this article, we present kmerDB, a database accessible through an interactive web interface that provides kmer based information from genomic and proteomic sequences in a systematic way. kmerDB currently contains 202,340,859,107 base pairs and 19,304,903,356 amino acids, spanning 45,785 and 22,386 reference genomes and proteomes, respectively, as well as 14,658,776 and 149,264,442 genomic and proteomic species-specific sequences, termed quasi-primes. Additionally, we provide access to 5,186,757 nucleic and 214,904,089 peptide sequences that are absent from every genome and proteome, termed primes. kmerDB features a user-friendly interface offering various search options and filters for easy parsing and searching. The service is available at: www.kmerdb.com.\nTranscriptomic reprogramming screen identifies SRSF1 as rejuvenation factor\nAuthors: Plesa, A. M.; Jung, S.; Wang, H. H.; Omar, F.; Shadpour, M.; Choy Buentello, D.; Perez-Matos, M. C.; Horwitz, N.; Cai, G.; Ngian, Z.-K.; de Magalhaes, C. V.; Wagers, A. J.; Mair, W. B.; del Sol Mesa, A.; Church, G. M.\nScore: 9.4, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566787\nAging is a complex process that manifests through the time-dependent functional decline of a biological system. Age-related changes in epigenetic and transcriptomic profiles have been successfully used to measure the aging process1,2. Moreover, modulating gene regulatory networks through interventions such as the induction of the Yamanaka factors has been shown to reverse aging signatures and improve cell function3,4. However, this intervention has safety and efficacy limitations for in vivo rejuvenation5,6, underscoring the need for identifying novel age reversal factors. Here, we discovered SRSF1 as a new rejuvenation factor that can improve cellular function in vitro and in vivo. Using a cDNA overexpression screen with a transcriptomic readout we identified that SRSF1 induction reprograms the cell transcriptome towards a younger state. Furthermore, we observed beneficial changes in senescence, proteasome function, collagen production, and ROS stress upon SRSF1 overexpression. Lastly, we showed that SRSF1 can improve wound healing in vitro and in vivo and is linked to organismal longevity. Our study provides a proof of concept for using transcriptomic reprogramming screens in the discovery of age reversal interventions and identifies SRSF1 as a promising target for cellular rejuvenation.\nYY1 binding is a gene-intrinsic barrier to Xist-mediated gene silencing\nAuthors: Bowness, J.; Almeida, M.; Nesterova, T.; Brockdorff, N.\nScore: 8.2, Published: 2023-11-23 DOI: 10.1101/2023.11.22.568288\nX chromosome inactivation (XCI) in mammals is mediated by Xist RNA which functions in cis to silence genes on a single X chromosome in XX female cells, thereby equalising levels of X-linked gene expression relative to XY males. XCI progresses over a period of several days, with some X-linked genes silencing faster than others. Chromosomal location of a gene is an important determinant of silencing rate, but uncharacterised gene-intrinsic features also mediate resistance or susceptibility to silencing. In this study, we integrate time-course data of gene silencing and decreasing inactive X (Xi) chromatin accessibility in mouse embryonic stem cell lines with an inducible Xist allele (iXist-ChrX mESCs). Our analysis reveals that motifs bound by the transcription factor YY1 are associated with persistently accessible regulatory elements, including many promoters and enhancers of slow-silencing genes. We further show that YY1 is evicted relatively slowly from target sites on Xi, and that silencing of X-linked genes is increased upon YY1 degradation. Together our results indicate that YY1 acts as barrier to Xist-mediated silencing that is removed only at late stages of the XCI process.\nDNA methylation carries signatures of sublethal effects under thermal stress in loggerhead sea turtles\nAuthors: Yen, E. C.; Gilbert, J. D.; Balard, A.; Afonso, I. O.; Fairweather, K.; Newlands, D.; Lopes, A.; Correia, S. M.; Taxonera, A.; Rossiter, S. J.; Martin-Duran, J. M.; Eizaguirre, C.\nScore: 7.5, Published: 2023-11-22 DOI: 10.1101/2023.11.22.568239\nRising global temperatures are a major threat to biodiversity. Whilst research generally focuses on thermal tolerance and mortality, sublethal effects may alter population dynamics and subsequently the adaptive potential of species. However, detecting such effects in the wild can be challenging, particularly for endangered and long-lived species with cryptic life histories. This necessitates the development of molecular tools to identify their signatures. In a split-clutch design experiment, we relocated clutches of wild, nesting loggerhead sea turtles (Caretta caretta) to a protected, in-situ hatchery. Eggs were then split into two sub-clutches incubated under shallow (35cm) or deep (55cm) conditions, with those in the shallow treatment experiencing significantly higher temperatures. Whilst no difference in hatching success was observed between treatments, hatchlings that emerged from the shallow, warmer treatment had altered length-mass relationships, and were weaker at fitness tests of locomotion capacity than their siblings incubated in the deep, cooler treatment. To characterise the molecular signatures of these thermal effects, we performed whole genome bisulfite sequencing on blood samples collected upon emergence. This identified 714 differentially methylated sites between treatments, including on genes with neuronal development, cytoskeleton, and sex determination functions. Taken together, our results show that higher incubation temperatures can induce sublethal effects in hatchlings, which are reflected in their DNA methylation status at identified sites. Such sites could be used as biomarkers of thermal stress, especially if they are retained across life stages. Overall, this study suggests that global warming may have population-level consequences for loggerhead sea turtles, by reducing hatchling quality, dispersal capacity and the adaptive potential of this species. Conservation efforts for climate-threatened taxa like endangered sea turtles will therefore benefit from strategies that monitor and mitigate exposure to incubation temperatures that lead to sublethal effects.\nImaging genetics of language network functional connectivity reveals links with language-related abilities, dyslexia and handedness\nAuthors: Amelink, J. S.; Postema, M. C.; Kong, X.-Z.; Schijven, D.; Carrion-Castillo, A.; Soheili-Nezhad, S.; Sha, Z.; Molz, B.; Joliot, M.; Fisher, S. E.; Francks, C.\nScore: 6.6, Published: 2023-11-23 DOI: 10.1101/2023.11.22.568256\nLanguage is supported by a distributed network of brain regions with a particular contribution from the left hemisphere. A multi-level understanding of this network requires studying the genetic architecture of its functional connectivity and hemispheric asymmetry. We used resting state functional imaging data from 29,681 participants from the UK Biobank to measure functional connectivity between 18 left-hemisphere regions implicated in multimodal sentence-level processing, as well as their homotopic regions in the right-hemisphere, and interhemispheric connections. Multivariate genome-wide association analysis of this total network, based on common genetic variants (with population frequencies above 1%), identified 14 loci associated with network functional connectivity. Three of these loci were also associated with hemispheric differences of intrahemispheric connectivity. Polygenic dispositions to lower language-related abilities, dyslexia and left-handedness were associated with generally reduced leftward asymmetry of functional connectivity, but with some trait- and connection-specific exceptions. Exome-wide association analysis based on rare, protein-altering variants (frequencies \u003c 1%) suggested 7 additional genes. These findings shed new light on the genetic contributions to language network connectivity and its asymmetry based on both common and rare genetic variants, and reveal genetic links to language-related traits and hemispheric dominance for hand preference.\nDevelopment and validation of a reliable DNA copy-number-based machine learning algorithm (CopyClust) for breast cancer integrative cluster classification\nAuthors: Young, C. C.; Eason, K.; Manzano Garcia, R.; Moulange, R.; Mukherjee, S.; Chin, S.-F. C.; Caldas, C.; Rueda, O. M.\nScore: 5.9, Published: 2023-11-22 DOI: 10.1101/2023.11.21.568129\nThe Integrative Clusters (IntClusts) provide a framework for the classification of breast cancer tumors into 10 distinct genomic subtypes based on DNA copy number and gene expression. Current classifiers achieve only low accuracy without gene expression data, warranting the development of new approaches to copy-number-only-based IntClust classification. A novel XGBoost-driven classification algorithm, CopyClust, was trained using genomic features from METABRIC and validated on TCGA achieving a nine-percentage point or greater improvement in overall IntClust subtype classification accuracy.\n",
  "wordCount" : "2147",
  "inLanguage": "en",
  "datePublished": "2023-11-26T10:37:33Z",
  "dateModified": "2023-11-26T10:37:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 26, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567180">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567180" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567180">
        <p class="paperTitle">Simultaneous Profiling of Chromatin Accessibility and DNA Methylation in Complete Plant Genomes Using Long-Read Sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567180" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567180" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Leduque, B.; Edera, A.; Vitte, C.; Quadrana, L.</p>
        <p class="info">Score: 21.6, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567180' target='https://doi.org/10.1101/2023.11.15.567180'> 10.1101/2023.11.15.567180</a></p>
        <p class="abstract">Epigenetic regulations, including chromatin accessibility, nucleosome positioning, and DNA methylation intricately shape genome function. However, current chromatin profiling techniques relying on short-read sequencing technologies fail to characterise highly repetitive genomic regions and cannot detect multiple chromatin features simultaneously. Here, we performed Simultaneous Accessibility and DNA Methylation Sequencing (SAM-seq) of purified plant nuclei. Thanks to the use of long-read nanopore sequencing, SAM-seq enables high-resolution profiling of m6A-tagged chromatin accessibility together with endogenous cytosine methylation in plants. Analysis of naked genomic DNA revealed significant sequence preference biases of m6A-MTases, controllable through a normalisation step. By applying SAM-seq to Arabidopsis and maize nuclei we obtained fine-grained accessibility and DNA methylation landscapes genome-wide. In addition, we uncovered crosstalk between chromatin accessibility and DNA methylation within nucleosomes of genes, TEs, and centromeric repeats. SAM-seq also facilitated the identification of DNA footprints over cis-regulatory regions. Furthermore, using the single-molecule information provided by SAM-seq we identified extensive cellular heterogeneity at chromatin domains harbouring antagonistic chromatin marks, suggesting that bivalency reflects cell-specific regulations of gene activity. SAM-seq is a powerful approach to simultaneously study multiple epigenetic features over unique and repetitive sequences, opening new opportunities for the investigation of epigenetic mechanisms in model as well non-model plant species with limited genomic and epigenomic information.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.19.567742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.19.567742" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.19.567742">
        <p class="paperTitle">MPRAbase: A Massively Parallel Reporter Assay Database</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.19.567742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.19.567742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhao, J.; Baltoumas, F. A.; Konnaris, M. A.; Mouratidis, I.; Liu, Z.; Sims, J.; Agarwal, V.; Pavlopoulos, G. A.; Georgakopoulos-Soares, I.; Ahituv, N.</p>
        <p class="info">Score: 18.2, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.19.567742' target='https://doi.org/10.1101/2023.11.19.567742'> 10.1101/2023.11.19.567742</a></p>
        <p class="abstract">Massively parallel reporter assays (MPRAs) represent a set of high-throughput technologies that measure the functional effects of thousands of sequences/variants on gene regulatory activity. There are several different variations of MPRA technology and they are used for numerous applications, including regulatory element discovery, variant effect measurement, saturation mutagenesis, synthetic regulatory element generation or characterization of evolutionary gene regulatory differences. Despite their many designs and uses, there is no comprehensive database that incorporates the results of these experiments. To address this, we developed MPRAbase, a manually curated database that currently harbors 129 experiments, encompassing 17,718,677 elements tested across 35 cell types and 4 organisms. The MPRAbase web interface (http://www.mprabase.com) serves as a centralized user-friendly repository to download existing MPRA data for independent analysis and is designed with the ability to allow researchers to share their published data for rapid dissemination to the community.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567860">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567860" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567860">
        <p class="paperTitle">DNA-binding factor footprints and enhancer RNAs identify functional non-coding genetic variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567860" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567860" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Biddie, S. C.; Hird, E. F.; Weykopf, G.; Friman, E. T.; Bickmore, W. A.</p>
        <p class="info">Score: 16.0, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567860' target='https://doi.org/10.1101/2023.11.20.567860'> 10.1101/2023.11.20.567860</a></p>
        <p class="abstract">Genome-wide association studies (GWAS) have revealed a multitude of candidate genetic variants affecting the risk of developing complex traits and diseases. However, these highlighted regions are typically in the non-coding genome, and uncovering the functional causative single nucleotide variants (SNVs) is challenging. Prioritisation of variants is commonly based on functional genomic annotation with markers of active regulatory elements, but current approaches still poorly predict functional variants. To address this, we systematically analyse six markers of active regulatory elements for their ability to identify functional variants. We benchmark against molecular quantitative trait loci (molQTL) from assays of regulatory element activity that identify allelic effects on DNA-binding factor occupancy, reporter assay expression, and chromatin accessibility. We identify the combination of DNase footprints and divergent enhancer RNA as markers for functional variants. This signature provides high precision, trading-off low recall, thus substantially reducing candidate variant sets to prioritise variants for functional validation. We present this as a framework called FINDER - Functional SNV IdeNtification using DNase footprints and Enhancer RNA, and demonstrate its utility to prioritise variants using leukocyte count trait and analyse variants in linkage disequilibrium with a lead variant to predict a functional variant in asthma. Our findings have implications for prioritising variants from GWAS, in development of predictive scoring algorithms, and for functionally informed fine mapping approaches.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.09.523212">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.09.523212" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.09.523212">
        <p class="paperTitle">An appropriate differential expression analysis in single-cell data depends on the study design</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.09.523212" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.09.523212" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, H.; Han, B.</p>
        <p class="info">Score: 15.1, Published: 2023-11-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.09.523212' target='https://doi.org/10.1101/2023.01.09.523212'> 10.1101/2023.01.09.523212</a></p>
        <p class="abstract">Large-scale multi-subject single-cell data have become very common. However, these data have high heterogeneity in study designs, leading to confusion in differential gene expression (DGE) analysis. In this work, we show that pseudobulk produces a substantial amount of type 2 error when the group label being compared varies within a subject. When the comparison label is constant within a subject, type 1 error is inflated if pseudoreplication is not accounted for. As a general principle, we show that an appropriate method depends on the design of the study. We propose solutions to the inflated error rates and provide practical guidelines for researchers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566926">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566926" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566926">
        <p class="paperTitle">kmerDB: A Database Encompassing the Set of Genomic and Proteomic Sequence Information for Each Species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566926" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566926" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mouratidis, I.; Baltoumas, F. A.; Chantzi, N.; Chan, C.; Montgomery, A.; Konnaris, M. A.; Georgakopoulos, G. C.; Das, A.; Chartoumpekis, D.; Kovac, J.; Pavlopoulos, G.; Georgakopoulos-Soares, I.</p>
        <p class="info">Score: 12.1, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566926' target='https://doi.org/10.1101/2023.11.13.566926'> 10.1101/2023.11.13.566926</a></p>
        <p class="abstract">The rapid decline in sequencing cost has enabled the generation of reference genomes and proteomes for a growing number of organisms. However, at the present time, there is no established repository that provides information about organism-specific genomic and proteomic sequences of certain lengths, also known as kmers, that are either present or absent in each genome or proteome. In this article, we present kmerDB, a database accessible through an interactive web interface that provides kmer based information from genomic and proteomic sequences in a systematic way. kmerDB currently contains 202,340,859,107 base pairs and 19,304,903,356 amino acids, spanning 45,785 and 22,386 reference genomes and proteomes, respectively, as well as 14,658,776 and 149,264,442 genomic and proteomic species-specific sequences, termed quasi-primes. Additionally, we provide access to 5,186,757 nucleic and 214,904,089 peptide sequences that are absent from every genome and proteome, termed primes. kmerDB features a user-friendly interface offering various search options and filters for easy parsing and searching. The service is available at: www.kmerdb.com.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566787">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566787" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566787">
        <p class="paperTitle">Transcriptomic reprogramming screen identifies SRSF1 as rejuvenation factor</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566787" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566787" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Plesa, A. M.; Jung, S.; Wang, H. H.; Omar, F.; Shadpour, M.; Choy Buentello, D.; Perez-Matos, M. C.; Horwitz, N.; Cai, G.; Ngian, Z.-K.; de Magalhaes, C. V.; Wagers, A. J.; Mair, W. B.; del Sol Mesa, A.; Church, G. M.</p>
        <p class="info">Score: 9.4, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566787' target='https://doi.org/10.1101/2023.11.13.566787'> 10.1101/2023.11.13.566787</a></p>
        <p class="abstract">Aging is a complex process that manifests through the time-dependent functional decline of a biological system. Age-related changes in epigenetic and transcriptomic profiles have been successfully used to measure the aging process1,2. Moreover, modulating gene regulatory networks through interventions such as the induction of the Yamanaka factors has been shown to reverse aging signatures and improve cell function3,4. However, this intervention has safety and efficacy limitations for in vivo rejuvenation5,6, underscoring the need for identifying novel age reversal factors. Here, we discovered SRSF1 as a new rejuvenation factor that can improve cellular function in vitro and in vivo. Using a cDNA overexpression screen with a transcriptomic readout we identified that SRSF1 induction reprograms the cell transcriptome towards a younger state. Furthermore, we observed beneficial changes in senescence, proteasome function, collagen production, and ROS stress upon SRSF1 overexpression. Lastly, we showed that SRSF1 can improve wound healing in vitro and in vivo and is linked to organismal longevity. Our study provides a proof of concept for using transcriptomic reprogramming screens in the discovery of age reversal interventions and identifies SRSF1 as a promising target for cellular rejuvenation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568288">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568288" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568288">
        <p class="paperTitle">YY1 binding is a gene-intrinsic barrier to Xist-mediated gene silencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568288" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568288" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bowness, J.; Almeida, M.; Nesterova, T.; Brockdorff, N.</p>
        <p class="info">Score: 8.2, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568288' target='https://doi.org/10.1101/2023.11.22.568288'> 10.1101/2023.11.22.568288</a></p>
        <p class="abstract">X chromosome inactivation (XCI) in mammals is mediated by Xist RNA which functions in cis to silence genes on a single X chromosome in XX female cells, thereby equalising levels of X-linked gene expression relative to XY males. XCI progresses over a period of several days, with some X-linked genes silencing faster than others. Chromosomal location of a gene is an important determinant of silencing rate, but uncharacterised gene-intrinsic features also mediate resistance or susceptibility to silencing. In this study, we integrate time-course data of gene silencing and decreasing inactive X (Xi) chromatin accessibility in mouse embryonic stem cell lines with an inducible Xist allele (iXist-ChrX mESCs). Our analysis reveals that motifs bound by the transcription factor YY1 are associated with persistently accessible regulatory elements, including many promoters and enhancers of slow-silencing genes. We further show that YY1 is evicted relatively slowly from target sites on Xi, and that silencing of X-linked genes is increased upon YY1 degradation. Together our results indicate that YY1 acts as barrier to Xist-mediated silencing that is removed only at late stages of the XCI process.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568239">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568239" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568239">
        <p class="paperTitle">DNA methylation carries signatures of sublethal effects under thermal stress in loggerhead sea turtles</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568239" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568239" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yen, E. C.; Gilbert, J. D.; Balard, A.; Afonso, I. O.; Fairweather, K.; Newlands, D.; Lopes, A.; Correia, S. M.; Taxonera, A.; Rossiter, S. J.; Martin-Duran, J. M.; Eizaguirre, C.</p>
        <p class="info">Score: 7.5, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568239' target='https://doi.org/10.1101/2023.11.22.568239'> 10.1101/2023.11.22.568239</a></p>
        <p class="abstract">Rising global temperatures are a major threat to biodiversity. Whilst research generally focuses on thermal tolerance and mortality, sublethal effects may alter population dynamics and subsequently the adaptive potential of species. However, detecting such effects in the wild can be challenging, particularly for endangered and long-lived species with cryptic life histories. This necessitates the development of molecular tools to identify their signatures. In a split-clutch design experiment, we relocated clutches of wild, nesting loggerhead sea turtles (Caretta caretta) to a protected, in-situ hatchery. Eggs were then split into two sub-clutches incubated under shallow (35cm) or deep (55cm) conditions, with those in the shallow treatment experiencing significantly higher temperatures. Whilst no difference in hatching success was observed between treatments, hatchlings that emerged from the shallow, warmer treatment had altered length-mass relationships, and were weaker at fitness tests of locomotion capacity than their siblings incubated in the deep, cooler treatment. To characterise the molecular signatures of these thermal effects, we performed whole genome bisulfite sequencing on blood samples collected upon emergence. This identified 714 differentially methylated sites between treatments, including on genes with neuronal development, cytoskeleton, and sex determination functions. Taken together, our results show that higher incubation temperatures can induce sublethal effects in hatchlings, which are reflected in their DNA methylation status at identified sites. Such sites could be used as biomarkers of thermal stress, especially if they are retained across life stages. Overall, this study suggests that global warming may have population-level consequences for loggerhead sea turtles, by reducing hatchling quality, dispersal capacity and the adaptive potential of this species. Conservation efforts for climate-threatened taxa like endangered sea turtles will therefore benefit from strategies that monitor and mitigate exposure to incubation temperatures that lead to sublethal effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.568256">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.568256" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.568256">
        <p class="paperTitle">Imaging genetics of language network functional connectivity reveals links with language-related abilities, dyslexia and handedness</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.568256" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.568256" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Amelink, J. S.; Postema, M. C.; Kong, X.-Z.; Schijven, D.; Carrion-Castillo, A.; Soheili-Nezhad, S.; Sha, Z.; Molz, B.; Joliot, M.; Fisher, S. E.; Francks, C.</p>
        <p class="info">Score: 6.6, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.568256' target='https://doi.org/10.1101/2023.11.22.568256'> 10.1101/2023.11.22.568256</a></p>
        <p class="abstract">Language is supported by a distributed network of brain regions with a particular contribution from the left hemisphere. A multi-level understanding of this network requires studying the genetic architecture of its functional connectivity and hemispheric asymmetry. We used resting state functional imaging data from 29,681 participants from the UK Biobank to measure functional connectivity between 18 left-hemisphere regions implicated in multimodal sentence-level processing, as well as their homotopic regions in the right-hemisphere, and interhemispheric connections. Multivariate genome-wide association analysis of this total network, based on common genetic variants (with population frequencies above 1%), identified 14 loci associated with network functional connectivity. Three of these loci were also associated with hemispheric differences of intrahemispheric connectivity. Polygenic dispositions to lower language-related abilities, dyslexia and left-handedness were associated with generally reduced leftward asymmetry of functional connectivity, but with some trait- and connection-specific exceptions. Exome-wide association analysis based on rare, protein-altering variants (frequencies &lt; 1%) suggested 7 additional genes. These findings shed new light on the genetic contributions to language network connectivity and its asymmetry based on both common and rare genetic variants, and reveal genetic links to language-related traits and hemispheric dominance for hand preference.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568129">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568129" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568129">
        <p class="paperTitle">Development and validation of a reliable DNA copy-number-based machine learning algorithm (CopyClust) for breast cancer integrative cluster classification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568129" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568129" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Young, C. C.; Eason, K.; Manzano Garcia, R.; Moulange, R.; Mukherjee, S.; Chin, S.-F. C.; Caldas, C.; Rueda, O. M.</p>
        <p class="info">Score: 5.9, Published: 2023-11-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568129' target='https://doi.org/10.1101/2023.11.21.568129'> 10.1101/2023.11.21.568129</a></p>
        <p class="abstract">The Integrative Clusters (IntClusts) provide a framework for the classification of breast cancer tumors into 10 distinct genomic subtypes based on DNA copy number and gene expression. Current classifiers achieve only low accuracy without gene expression data, warranting the development of new approaches to copy-number-only-based IntClust classification. A novel XGBoost-driven classification algorithm, CopyClust, was trained using genomic features from METABRIC and validated on TCGA achieving a nine-percentage point or greater improvement in overall IntClust subtype classification accuracy.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
