<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Systematic identification of cargo-carrying genetic elements reveals new dimensions of eukaryotic diversity
Authors: Gluck-Thaler, E.; Vogan, A. A.
Score: 34.8, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563810
Cargo-carrying mobile elements (CCEs) are genetic entities that transpose diverse protein coding sequences. Although common in bacteria, we know little about the biology of eukaryotic CCEs because no appropriate tools exist for their annotation. For example, Starships are fungal CCEs whose functions are largely unknown because they require time-intensive manual curation.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Systematic identification of cargo-carrying genetic elements reveals new dimensions of eukaryotic diversity
Authors: Gluck-Thaler, E.; Vogan, A. A.
Score: 34.8, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563810
Cargo-carrying mobile elements (CCEs) are genetic entities that transpose diverse protein coding sequences. Although common in bacteria, we know little about the biology of eukaryotic CCEs because no appropriate tools exist for their annotation. For example, Starships are fungal CCEs whose functions are largely unknown because they require time-intensive manual curation." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Systematic identification of cargo-carrying genetic elements reveals new dimensions of eukaryotic diversity
Authors: Gluck-Thaler, E.; Vogan, A. A.
Score: 34.8, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563810
Cargo-carrying mobile elements (CCEs) are genetic entities that transpose diverse protein coding sequences. Although common in bacteria, we know little about the biology of eukaryotic CCEs because no appropriate tools exist for their annotation. For example, Starships are fungal CCEs whose functions are largely unknown because they require time-intensive manual curation."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Systematic identification of cargo-carrying genetic elements reveals new dimensions of eukaryotic diversity\nAuthors: Gluck-Thaler, E.; Vogan, A. A.\nScore: 34.8, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563810\nCargo-carrying mobile elements (CCEs) are genetic entities that transpose diverse protein coding sequences. Although common in bacteria, we know little about the biology of eukaryotic CCEs because no appropriate tools exist for their annotation. For example, Starships are fungal CCEs whose functions are largely unknown because they require time-intensive manual curation.",
  "keywords": [
    
  ],
  "articleBody": " Systematic identification of cargo-carrying genetic elements reveals new dimensions of eukaryotic diversity\nAuthors: Gluck-Thaler, E.; Vogan, A. A.\nScore: 34.8, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563810\nCargo-carrying mobile elements (CCEs) are genetic entities that transpose diverse protein coding sequences. Although common in bacteria, we know little about the biology of eukaryotic CCEs because no appropriate tools exist for their annotation. For example, Starships are fungal CCEs whose functions are largely unknown because they require time-intensive manual curation. To address this knowledge gap, we developed starfish, a computational workflow for high-throughput eukaryotic CCE annotation. We applied starfish to 2, 899 genomes of 1, 649 fungal species and found that starfish recovers known Starships with \u003e95% precision and accuracy while expanding the number of annotated elements ten-fold. Extant Starship diversity is partitioned into 11 families that differ in their enrichment patterns across fungal classes. Starship cargo changes rapidly such that elements from the same family differ substantially in their functional repertoires, which are predicted to contribute to diverse biological processes such as metabolism. Many elements have convergently evolved to insert into 5S rDNA and AT-rich sequence while others integrate into random locations, revealing both specialist and generalist strategies for persistence. Our work establishes a framework for advancing mobile element biology and provides the means to investigate an emerging dimension of eukaryotic genetic diversity, that of genomes within genomes.\nFrom sequence to molecules: Feature sequence-based genome mining uncovers the hidden diversity of bacterial siderophore pathways\nAuthors: Gu, S.; Shao, Y.; Rehm, K.; Bigler, L.; Zhang, D.; He, R.; Shao, J.; Jousset, A.; Friman, V.-P.; Wei, Z.; Kummerli, R.; Li, Z.\nScore: 19.4, Published: 2023-11-01 DOI: 10.1101/2023.10.30.564663\nMicrobial secondary metabolites have long been recognized as a rich source for pharmaceutical compound discovery and to have crucial ecological functions. However, the sequence-to-function mapping in microbial secondary metabolism pathways remains challenging because neither protein function nor substrate specificity can accurately be predicted from genome data. Here we focus on the iron-scavenging pyoverdines, siderophores of Pseudomonas bacteria, as model system to develop a knowledge-guided bioinformatic pipeline that extracts functional information of both the pyoverdine synthesis machinery and uptake receptors from 1928 draft genomes. For pyoverdine synthesis, our approach predicts the chemical structure of 188 different pyoverdines with nearly 100% accuracy. For pyoverdine uptake, our pipeline uncovers 94 different pyoverdine receptor groups. Our results demonstrate that combining feature sequence and phylogenetic approaches is a powerful way to reconstruct bacterial secondary metabolism pathways based on sequence data, unveiling an enormous yet overlooked diversity of siderophores and their receptors.\nNon-canonical ORFs are an important source of tumor-specific antigens in a liver cancer meta-cohort\nAuthors: E. Camarena, M.; Theunissen, P.; Ruiz Egozcue, M.; Ruiz-Orera, J.; Calvo-Serra, B.; Castelo, R.; Sarobe, P.; Fortes, P.; Perera-Bel, J.; Alba, M. M.\nScore: 17.6, Published: 2023-11-01 DOI: 10.1101/2023.10.30.564375\nThe expression of tumor-specific antigens during cancer progression can trigger an immune response against the tumor. Antigens that have been used as cancer vaccines are those originated by non-synonymous mutations and those derived from cancer/testis antigens. However, the first class is predominantly patient-specific, preventing the development of therapies than can benefit multiple patients, and the second one offers a limited set of actionable targets. A possible alternative is the use of peptides derived from non-canonical ORFs (ncORFs). While many ncORFs have been shown to be translated in cancer cells, their tumor-specificity and patient distribution remains to be determined. Here we analyze RNA sequencing data 117 hepatocellular carcinoma (HCC) tumors and matched healthy tissue, together with ribosome profiling data from an additional 10 HCC tumors, to answer these open questions. Combining HLA-epitope binding predictions and experimental validation experiments we conclude that around 40% of the tumor-specific antigens in HCC are likely to be derived from ncORFs in lncRNAs, including two peptides that can trigger an immune response in mice. We identify a subset of 33 tumor-specific lncRNAs expressing novel cancer antigens shared by more than 10% of the HCC analyzed, which could be combined to target a large proportion of the patients. The results of the study open new avenues for extending the range of anti-cancer vaccines.\nJoint sequence \u0026 chromatin neural networks characterize the differential abilities of Forkhead transcription factors to engage inaccessible chromatin\nAuthors: Arora, S.; Yang, J.; Akiyama, T.; James, D. Q.; Morrissey, A.; Blanda, T.; Badjatia, N.; Lai, W. K. M.; Ko, M. S. H.; Pugh, B. F.; Mahony, S.\nScore: 18.7, Published: 2023-10-31 DOI: 10.1101/2023.10.06.561228\nThe DNA-binding activities of transcription factors (TFs) are influenced by both intrinsic sequence preferences and extrinsic interactions with cell-specific chromatin landscapes and other regulatory proteins. Disentangling the roles of these binding determinants remains challenging. For example, the FoxA subfamily of Forkhead domain (Fox) TFs are known pioneer factors that can bind to relatively inaccessible sites during development. Yet FoxA TF binding also varies across cell types, pointing to a combination of intrinsic and extrinsic forces guiding their binding. While other Forkhead domain TFs are often assumed to have pioneering abilities, how sequence and chromatin features influence the binding of related Fox TFs has not been systematically characterized. Here, we present a principled approach to compare the relative contributions of intrinsic DNA sequence preference and cell-specific chromatin environments to a TFs DNA-binding activities. We apply our approach to investigate how a selection of Fox TFs (FoxA1, FoxC1, FoxG1, FoxL2, and FoxP3) vary in their binding specificity. We over-express the selected Fox TFs in mouse embryonic stem cells, which offer a platform to contrast each TFs binding activity within the same preexisting chromatin background. By applying a convolutional neural network to interpret the Fox TF binding patterns, we evaluate how sequence and preexisting chromatin features jointly contribute to induced TF binding. We demonstrate that Fox TFs bind different DNA targets, and drive differential gene expression patterns, even when induced in identical chromatin settings. Despite the association between Forkhead domains and pioneering activities, the selected Fox TFs display a wide range of affinities for preexiting chromatin states. Using sequence and chromatin feature attribution techniques to interpret the neural network predictions, we show that differential sequence preferences combined with differential abilities to engage relatively inaccessible chromatin together explain Fox TF binding patterns at individual sites and genome-wide.\nThe GIAB genomic stratifications resource for human reference genomes\nAuthors: Dwarshuis, N.; Kalra, D.; McDaniel, J.; Sanio, P.; Alvarez Jerez, P.; Jadhav, B.; Huang, W.; Mondal, R.; Busby, B.; Olson, N. D.; Sedlazeck, F. J.; Wagner, J.; Majidian, S.; Zook, J. M.\nScore: 15.7, Published: 2023-10-29 DOI: 10.1101/2023.10.27.563846\nStratification of the genome into different genomic contexts is useful when developing bioinformatics software like variant callers, to assess performance in difficult regions in the human genome. Here we describe a set of genomic stratifications for the human reference genomes GRCh37, GRCh38, and T2T-CHM13v2.0. Generating stratifications for the new complete CHM13 reference genome is critical to understanding improvements in variant caller performance when using this new complete reference. The GIAB stratifications can be used when benchmarking variant calls to analyze difficult regions of the human genome in a standardized way. Here we present stratifications in the CHM13 genome in comparison to GRCh37 and GRCh38, highlighting expansions in hard-to-map and GC-rich stratifications which provide useful insight for accuracy of variants in these newly-added regions. To evaluate the reliability and utility of the new stratifications, we used the stratifications of the three references to assess accuracy of variant calls in diverse, challenging genomic regions. The means to generate these stratifications are available as a snakemake pipeline at https://github.com/ndwarshuis/giab-stratifications.\nThe complete human diploid reference genome of RPE-1 identifies the phased epigenetic landscapes from multi-omics data\nAuthors: Volpe, E.; Corda, L.; Di Tommaso, E.; Pelliccia, F.; Ottalevi, R.; Licastro, D.; Formenti, G.; Capulli, M.; Guarracino, A.; Tassone, E.; Giunta, S.\nScore: 15.3, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565049\nComparative analysis of recent human genome assemblies highlights profound sequence divergence that peaks within polymorphic loci such as centromeres. This raises the question about the adequacy of relying on human reference genomes to accurately analyze sequencing data derived from experimental cell lines. Here we propose a novel approach, referred to as \"isogenomic reference\", that leverages a matched reference genome to perform multi-omics analyses. We have generated a new diploid genome assembly for the human retinal epithelial cells (RPE-1), a widely used non-cancer laboratory cell line with a stable diploid karyotype, that presents phased haplotypes and chromosome-level scaffolds that completely span centromeres. Using this assembly, we have characterized haplotype-resolved genomic variation unique to RPE- 1, including a stable marker chromosome X with a 73.18 Mb segmental duplication of chromosome 10 translocated onto the microdeleted telomere t(Xq;10q), specific to this cell line. Comparative analyses revealed sequence polymorphism within centromeric regions, including unexpected genetic and epigenetic diversity among haplotypes for all chromosomes. Using our assembly as reference, we re-analyzed both our own and publicly available sequencing, methylation and epigenetic data generated in RPE-1 which had previously been analyzed with non-matched and non-diploid reference genomes. Our results show that the isogenomic reference improves alignments with an increased mapping quality up to 85% while halving mismatches, resulting in significant changes in peaks calling related to centromeres. Our work represents a proof-of-concept, showcasing the use of matched reference genomes for multiomics analyses and, at scale, serves as the foundation for a call to comprehensively assemble experimentally relevant cell lines for a widespread application of isogenomic reference genomes.\nModular vector assembly enables rapid assessment of emerging CRISPR technologies\nAuthors: McGee, A. V.; Liu, Y. V.; Griffith, A. L.; Szegletes, Z. M.; Wen, B.; Kraus, C.; Miller, N. W.; Steger, R. J.; Escude Velasco, B.; Bosch, J. A.; Zirin, J. D.; Viswanatha, R.; Sontheimer, E. J.; Goodale, A.; Greene, M. A.; Green, T. M.; Doench, J. G.\nScore: 31.0, Published: 2023-10-27 DOI: 10.1101/2023.10.25.564061\nThe diversity of CRISPR systems, coupled with scientific ingenuity, has led to an explosion of applications; however, to test newly-described innovations in their model systems, researchers typically embark on cumbersome, one-off cloning projects to generate custom reagents that are optimized for their biological questions. Here, we leverage Golden Gate cloning to create the Fragmid toolkit, a modular set of CRISPR cassettes and delivery technologies, along with a web portal, resulting in a combinatorial platform that enables scalable vector assembly within days. We further demonstrate that multiple CRISPR technologies can be assessed in parallel in a pooled screening format using this resource, enabling the rapid optimization of both novel technologies and cellular models. These results establish Fragmid as a robust system for the rapid design of CRISPR vectors, and we anticipate that this assembly approach will be broadly useful for systematic development, comparison, and dissemination of CRISPR technologies.\nHigh-fidelity (repeat) consensus sequences from short reads using combined read clustering and assembly\nAuthors: Mann, L.; Balasch, K.; Schmidt, N.; Heitkam, T.\nScore: 13.8, Published: 2023-10-30 DOI: 10.1101/2023.10.26.564123\nBackgroundDespite the many cheap and fast ways to generate genomic data, good and exact genome assembly is still a problem, with especially the repeats being vastly underrepresented and often misassembled. As short reads in low coverage are already sufficient to represent the repeat landscape of any given genome, many read cluster algorithms were brought forward that provide repeat identification and classification. But how can trustworthy, reliable and representative full-length repeat consensuses be derived from unassembled genomes? ResultsHere, we combine methods from repeat identification and genome assembly to derive these robust consensuses. We test several use cases, such as (1) consensus building from clustered short reads of non-model genomes, (2) from genome-wide amplification setups, and (3) specific repeat-centred questions, such as the linked vs. unlinked arrangement of ribosomal genes. In all our use-cases, the derived consensuses are robust and representative. To evaluate overall performance, we compare our high-fidelity repeat consensuses to RepeatExplorer2-derived contigs and check, if they represent real transposable elements as found in long reads. Our results demonstrate that it is possible to generate useful, reliable and trustworthy consensuses from short reads by a combination from read cluster and genome assembly methods in an automatable way. ConclusionWe anticipate that our workflow opens the way towards more efficient and less manual repeat characterization and annotation, benefitting all genome studies, but especially those of non-model organisms.\nReplication stress underlies genomic instability at CTCF/cohesin-binding sites in cancer\nAuthors: Faseela, E. E.; Notani, D.; Sabarinathan, R.\nScore: 13.8, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563697\nCCCTC-binding factor (CTCF) and cohesin play a significant role in the formation of chromatin loops and topologically associating domains (TADs), which influence gene expression and DNA replication. CTCF/cohesin-binding sites (CBSs) present at the loop anchors and TAD boundaries are frequently mutated in cancer; however, the molecular mechanisms underlying this remain unclear. Here, we investigate whether the binding of CTCF/cohesin on DNA imposes constraints on DNA replication, leading to replication stress and genomic instability. Our results reveal that CTCF and cohesin remain bound to DNA during replication (S phase) in cancer cells (HeLa). Further, examination of replication stress through imaging revealed colocalization of DNA damage response proteins (MRE11 and STN1) with CTCF/cohesin in the S phase. In addition, ChIP-seq of the DNA damage response/repair proteins (MRE11, STN1, {gamma}H2AX and RAD51) showed high enrichment of these proteins at CBSs (as compared to their immediate flanking regions and control sites) and positively correlated with the binding strength of CTCF/cohesin at CBSs in the S phase. Moreover, analysis of somatic mutations from cancer genomes supports that the enrichment of mutations at CBSs is significantly higher in samples harbouring somatic copy number deletion in MRE11 and STN1 compared to wild-type samples. Together, these results demonstrate that the binding of CTCF/cohesin on the DNA during the S phase causes replication stress and DNA strand breaks, and this could lead to genome instability at CBSs observed in cancer.\nOligo-barcodes illuminate holocentric karyotype evolution in Rhynchospora (Cyperaceae)\nAuthors: Mata-Sucre, Y.; Parteka, L. M.; Ritz, C.; Gatica-Arias, A.; Felix, L. P.; Thomas, W. W.; Souza, G.; Vanzela, A. L. L.; Pedrosa-Harand, A.; Marques, A.\nScore: 12.0, Published: 2023-11-02 DOI: 10.1101/2023.10.31.564898\nHolocentric karyotypes are assumed to rapidly evolve through chromosome fusions and fissions due to the diffuse nature of their centromeres. Here, we took advantage of the recent availability of a chromosome-scale reference genome for Rhynchospora breviuscula, a model species of this holocentric genus, and developed the first set of oligo-based barcode probes for a holocentric plant. These probes were applied to 13 additional species of the genus, aiming to investigate the evolutionary dynamics driving the karyotype evolution in Rhynchospora. The two sets of probes were composed of 27,392 (green) and 23,968 (magenta) oligonucleotides, and generated 15 distinct FISH signals as a unique barcode pattern for the identification of all five chromosome pairs of the R. breviuscula karyotype. Oligo-FISH comparative analyzes revealed different types of rearrangements, such as fusions, fissions, putative inversions and translocations, as well as genomic duplications among the analyzed species. Two rounds of whole genome duplication (WGD) were demonstrated in R. pubera, but both analyzed accessions differed in the complex chain of events that gave rise to its large, structurally diploidized karyotypes with 2n = 10 or 12. Considering the phylogenetic relationships and divergence time of the species, the specificity and synteny of the probes were maintained up to species with a divergence time of ~25 My. However, karyotype divergence in more distant species hindered chromosome mapping and the inference of specific events. This barcoding system is a powerful tool to study chromosomal variations and genomic evolution in holocentric chromosomes of Rhynchospora species.\n",
  "wordCount" : "2547",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563810">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563810" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563810">
        <p class="paperTitle">Systematic identification of cargo-carrying genetic elements reveals new dimensions of eukaryotic diversity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563810" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563810" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gluck-Thaler, E.; Vogan, A. A.</p>
        <p class="info">Score: 34.8, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563810' target='https://doi.org/10.1101/2023.10.24.563810'> 10.1101/2023.10.24.563810</a></p>
        <p class="abstract">Cargo-carrying mobile elements (CCEs) are genetic entities that transpose diverse protein coding sequences. Although common in bacteria, we know little about the biology of eukaryotic CCEs because no appropriate tools exist for their annotation. For example, Starships are fungal CCEs whose functions are largely unknown because they require time-intensive manual curation. To address this knowledge gap, we developed starfish, a computational workflow for high-throughput eukaryotic CCE annotation. We applied starfish to 2, 899 genomes of 1, 649 fungal species and found that starfish recovers known Starships with &gt;95% precision and accuracy while expanding the number of annotated elements ten-fold. Extant Starship diversity is partitioned into 11 families that differ in their enrichment patterns across fungal classes. Starship cargo changes rapidly such that elements from the same family differ substantially in their functional repertoires, which are predicted to contribute to diverse biological processes such as metabolism. Many elements have convergently evolved to insert into 5S rDNA and AT-rich sequence while others integrate into random locations, revealing both specialist and generalist strategies for persistence. Our work establishes a framework for advancing mobile element biology and provides the means to investigate an emerging dimension of eukaryotic genetic diversity, that of genomes within genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564663">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564663" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564663">
        <p class="paperTitle">From sequence to molecules: Feature sequence-based genome mining uncovers the hidden diversity of bacterial siderophore pathways</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564663" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564663" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gu, S.; Shao, Y.; Rehm, K.; Bigler, L.; Zhang, D.; He, R.; Shao, J.; Jousset, A.; Friman, V.-P.; Wei, Z.; Kummerli, R.; Li, Z.</p>
        <p class="info">Score: 19.4, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564663' target='https://doi.org/10.1101/2023.10.30.564663'> 10.1101/2023.10.30.564663</a></p>
        <p class="abstract">Microbial secondary metabolites have long been recognized as a rich source for pharmaceutical compound discovery and to have crucial ecological functions. However, the sequence-to-function mapping in microbial secondary metabolism pathways remains challenging because neither protein function nor substrate specificity can accurately be predicted from genome data. Here we focus on the iron-scavenging pyoverdines, siderophores of Pseudomonas bacteria, as model system to develop a knowledge-guided bioinformatic pipeline that extracts functional information of both the pyoverdine synthesis machinery and uptake receptors from 1928 draft genomes. For pyoverdine synthesis, our approach predicts the chemical structure of 188 different pyoverdines with nearly 100% accuracy. For pyoverdine uptake, our pipeline uncovers 94 different pyoverdine receptor groups. Our results demonstrate that combining feature sequence and phylogenetic approaches is a powerful way to reconstruct bacterial secondary metabolism pathways based on sequence data, unveiling an enormous yet overlooked diversity of siderophores and their receptors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564375">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564375" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564375">
        <p class="paperTitle">Non-canonical ORFs are an important source of tumor-specific antigens in a liver cancer meta-cohort</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564375" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564375" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: E. Camarena, M.; Theunissen, P.; Ruiz Egozcue, M.; Ruiz-Orera, J.; Calvo-Serra, B.; Castelo, R.; Sarobe, P.; Fortes, P.; Perera-Bel, J.; Alba, M. M.</p>
        <p class="info">Score: 17.6, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564375' target='https://doi.org/10.1101/2023.10.30.564375'> 10.1101/2023.10.30.564375</a></p>
        <p class="abstract">The expression of tumor-specific antigens during cancer progression can trigger an immune response against the tumor. Antigens that have been used as cancer vaccines are those originated by non-synonymous mutations and those derived from cancer/testis antigens. However, the first class is predominantly patient-specific, preventing the development of therapies than can benefit multiple patients, and the second one offers a limited set of actionable targets. A possible alternative is the use of peptides derived from non-canonical ORFs (ncORFs). While many ncORFs have been shown to be translated in cancer cells, their tumor-specificity and patient distribution remains to be determined. Here we analyze RNA sequencing data 117 hepatocellular carcinoma (HCC) tumors and matched healthy tissue, together with ribosome profiling data from an additional 10 HCC tumors, to answer these open questions. Combining HLA-epitope binding predictions and experimental validation experiments we conclude that around 40% of the tumor-specific antigens in HCC are likely to be derived from ncORFs in lncRNAs, including two peptides that can trigger an immune response in mice. We identify a subset of 33 tumor-specific lncRNAs expressing novel cancer antigens shared by more than 10% of the HCC analyzed, which could be combined to target a large proportion of the patients. The results of the study open new avenues for extending the range of anti-cancer vaccines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.06.561228">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.06.561228" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.06.561228">
        <p class="paperTitle">Joint sequence &amp; chromatin neural networks characterize the differential abilities of Forkhead transcription factors to engage inaccessible chromatin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.06.561228" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.06.561228" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arora, S.; Yang, J.; Akiyama, T.; James, D. Q.; Morrissey, A.; Blanda, T.; Badjatia, N.; Lai, W. K. M.; Ko, M. S. H.; Pugh, B. F.; Mahony, S.</p>
        <p class="info">Score: 18.7, Published: 2023-10-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.06.561228' target='https://doi.org/10.1101/2023.10.06.561228'> 10.1101/2023.10.06.561228</a></p>
        <p class="abstract">The DNA-binding activities of transcription factors (TFs) are influenced by both intrinsic sequence preferences and extrinsic interactions with cell-specific chromatin landscapes and other regulatory proteins. Disentangling the roles of these binding determinants remains challenging. For example, the FoxA subfamily of Forkhead domain (Fox) TFs are known pioneer factors that can bind to relatively inaccessible sites during development. Yet FoxA TF binding also varies across cell types, pointing to a combination of intrinsic and extrinsic forces guiding their binding. While other Forkhead domain TFs are often assumed to have pioneering abilities, how sequence and chromatin features influence the binding of related Fox TFs has not been systematically characterized.

Here, we present a principled approach to compare the relative contributions of intrinsic DNA sequence preference and cell-specific chromatin environments to a TFs DNA-binding activities. We apply our approach to investigate how a selection of Fox TFs (FoxA1, FoxC1, FoxG1, FoxL2, and FoxP3) vary in their binding specificity. We over-express the selected Fox TFs in mouse embryonic stem cells, which offer a platform to contrast each TFs binding activity within the same preexisting chromatin background. By applying a convolutional neural network to interpret the Fox TF binding patterns, we evaluate how sequence and preexisting chromatin features jointly contribute to induced TF binding.

We demonstrate that Fox TFs bind different DNA targets, and drive differential gene expression patterns, even when induced in identical chromatin settings. Despite the association between Forkhead domains and pioneering activities, the selected Fox TFs display a wide range of affinities for preexiting chromatin states. Using sequence and chromatin feature attribution techniques to interpret the neural network predictions, we show that differential sequence preferences combined with differential abilities to engage relatively inaccessible chromatin together explain Fox TF binding patterns at individual sites and genome-wide.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.563846">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.563846" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.563846">
        <p class="paperTitle">The GIAB genomic stratifications resource for human reference genomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.563846" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.563846" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dwarshuis, N.; Kalra, D.; McDaniel, J.; Sanio, P.; Alvarez Jerez, P.; Jadhav, B.; Huang, W.; Mondal, R.; Busby, B.; Olson, N. D.; Sedlazeck, F. J.; Wagner, J.; Majidian, S.; Zook, J. M.</p>
        <p class="info">Score: 15.7, Published: 2023-10-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.563846' target='https://doi.org/10.1101/2023.10.27.563846'> 10.1101/2023.10.27.563846</a></p>
        <p class="abstract">Stratification of the genome into different genomic contexts is useful when developing bioinformatics software like variant callers, to assess performance in difficult regions in the human genome. Here we describe a set of genomic stratifications for the human reference genomes GRCh37, GRCh38, and T2T-CHM13v2.0. Generating stratifications for the new complete CHM13 reference genome is critical to understanding improvements in variant caller performance when using this new complete reference. The GIAB stratifications can be used when benchmarking variant calls to analyze difficult regions of the human genome in a standardized way. Here we present stratifications in the CHM13 genome in comparison to GRCh37 and GRCh38, highlighting expansions in hard-to-map and GC-rich stratifications which provide useful insight for accuracy of variants in these newly-added regions. To evaluate the reliability and utility of the new stratifications, we used the stratifications of the three references to assess accuracy of variant calls in diverse, challenging genomic regions. The means to generate these stratifications are available as a snakemake pipeline at https://github.com/ndwarshuis/giab-stratifications.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565049" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565049">
        <p class="paperTitle">The complete human diploid reference genome of RPE-1 identifies the phased epigenetic landscapes from multi-omics data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Volpe, E.; Corda, L.; Di Tommaso, E.; Pelliccia, F.; Ottalevi, R.; Licastro, D.; Formenti, G.; Capulli, M.; Guarracino, A.; Tassone, E.; Giunta, S.</p>
        <p class="info">Score: 15.3, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565049' target='https://doi.org/10.1101/2023.11.01.565049'> 10.1101/2023.11.01.565049</a></p>
        <p class="abstract">Comparative analysis of recent human genome assemblies highlights profound sequence divergence that peaks within polymorphic loci such as centromeres. This raises the question about the adequacy of relying on human reference genomes to accurately analyze sequencing data derived from experimental cell lines. Here we propose a novel approach, referred to as &#34;isogenomic reference&#34;, that leverages a matched reference genome to perform multi-omics analyses. We have generated a new diploid genome assembly for the human retinal epithelial cells (RPE-1), a widely used non-cancer laboratory cell line with a stable diploid karyotype, that presents phased haplotypes and chromosome-level scaffolds that completely span centromeres. Using this assembly, we have characterized haplotype-resolved genomic variation unique to RPE- 1, including a stable marker chromosome X with a 73.18 Mb segmental duplication of chromosome 10 translocated onto the microdeleted telomere t(Xq;10q), specific to this cell line. Comparative analyses revealed sequence polymorphism within centromeric regions, including unexpected genetic and epigenetic diversity among haplotypes for all chromosomes. Using our assembly as reference, we re-analyzed both our own and publicly available sequencing, methylation and epigenetic data generated in RPE-1 which had previously been analyzed with non-matched and non-diploid reference genomes. Our results show that the isogenomic reference improves alignments with an increased mapping quality up to 85% while halving mismatches, resulting in significant changes in peaks calling related to centromeres. Our work represents a proof-of-concept, showcasing the use of matched reference genomes for multiomics analyses and, at scale, serves as the foundation for a call to comprehensively assemble experimentally relevant cell lines for a widespread application of isogenomic reference genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.25.564061">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.25.564061" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.25.564061">
        <p class="paperTitle">Modular vector assembly enables rapid assessment of emerging CRISPR technologies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.25.564061" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.25.564061" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: McGee, A. V.; Liu, Y. V.; Griffith, A. L.; Szegletes, Z. M.; Wen, B.; Kraus, C.; Miller, N. W.; Steger, R. J.; Escude Velasco, B.; Bosch, J. A.; Zirin, J. D.; Viswanatha, R.; Sontheimer, E. J.; Goodale, A.; Greene, M. A.; Green, T. M.; Doench, J. G.</p>
        <p class="info">Score: 31.0, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.25.564061' target='https://doi.org/10.1101/2023.10.25.564061'> 10.1101/2023.10.25.564061</a></p>
        <p class="abstract">The diversity of CRISPR systems, coupled with scientific ingenuity, has led to an explosion of applications; however, to test newly-described innovations in their model systems, researchers typically embark on cumbersome, one-off cloning projects to generate custom reagents that are optimized for their biological questions. Here, we leverage Golden Gate cloning to create the Fragmid toolkit, a modular set of CRISPR cassettes and delivery technologies, along with a web portal, resulting in a combinatorial platform that enables scalable vector assembly within days. We further demonstrate that multiple CRISPR technologies can be assessed in parallel in a pooled screening format using this resource, enabling the rapid optimization of both novel technologies and cellular models. These results establish Fragmid as a robust system for the rapid design of CRISPR vectors, and we anticipate that this assembly approach will be broadly useful for systematic development, comparison, and dissemination of CRISPR technologies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564123" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564123">
        <p class="paperTitle">High-fidelity (repeat) consensus sequences from short reads using combined read clustering and assembly</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mann, L.; Balasch, K.; Schmidt, N.; Heitkam, T.</p>
        <p class="info">Score: 13.8, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564123' target='https://doi.org/10.1101/2023.10.26.564123'> 10.1101/2023.10.26.564123</a></p>
        <p class="abstract">BackgroundDespite the many cheap and fast ways to generate genomic data, good and exact genome assembly is still a problem, with especially the repeats being vastly underrepresented and often misassembled. As short reads in low coverage are already sufficient to represent the repeat landscape of any given genome, many read cluster algorithms were brought forward that provide repeat identification and classification. But how can trustworthy, reliable and representative full-length repeat consensuses be derived from unassembled genomes?

ResultsHere, we combine methods from repeat identification and genome assembly to derive these robust consensuses. We test several use cases, such as (1) consensus building from clustered short reads of non-model genomes, (2) from genome-wide amplification setups, and (3) specific repeat-centred questions, such as the linked vs. unlinked arrangement of ribosomal genes. In all our use-cases, the derived consensuses are robust and representative. To evaluate overall performance, we compare our high-fidelity repeat consensuses to RepeatExplorer2-derived contigs and check, if they represent real transposable elements as found in long reads. Our results demonstrate that it is possible to generate useful, reliable and trustworthy consensuses from short reads by a combination from read cluster and genome assembly methods in an automatable way.

ConclusionWe anticipate that our workflow opens the way towards more efficient and less manual repeat characterization and annotation, benefitting all genome studies, but especially those of non-model organisms.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563697">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563697" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563697">
        <p class="paperTitle">Replication stress underlies genomic instability at CTCF/cohesin-binding sites in cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563697" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563697" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Faseela, E. E.; Notani, D.; Sabarinathan, R.</p>
        <p class="info">Score: 13.8, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563697' target='https://doi.org/10.1101/2023.10.24.563697'> 10.1101/2023.10.24.563697</a></p>
        <p class="abstract">CCCTC-binding factor (CTCF) and cohesin play a significant role in the formation of chromatin loops and topologically associating domains (TADs), which influence gene expression and DNA replication. CTCF/cohesin-binding sites (CBSs) present at the loop anchors and TAD boundaries are frequently mutated in cancer; however, the molecular mechanisms underlying this remain unclear. Here, we investigate whether the binding of CTCF/cohesin on DNA imposes constraints on DNA replication, leading to replication stress and genomic instability. Our results reveal that CTCF and cohesin remain bound to DNA during replication (S phase) in cancer cells (HeLa). Further, examination of replication stress through imaging revealed colocalization of DNA damage response proteins (MRE11 and STN1) with CTCF/cohesin in the S phase. In addition, ChIP-seq of the DNA damage response/repair proteins (MRE11, STN1, {gamma}H2AX and RAD51) showed high enrichment of these proteins at CBSs (as compared to their immediate flanking regions and control sites) and positively correlated with the binding strength of CTCF/cohesin at CBSs in the S phase. Moreover, analysis of somatic mutations from cancer genomes supports that the enrichment of mutations at CBSs is significantly higher in samples harbouring somatic copy number deletion in MRE11 and STN1 compared to wild-type samples. Together, these results demonstrate that the binding of CTCF/cohesin on the DNA during the S phase causes replication stress and DNA strand breaks, and this could lead to genome instability at CBSs observed in cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.564898">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.564898" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.564898">
        <p class="paperTitle">Oligo-barcodes illuminate holocentric karyotype evolution in Rhynchospora (Cyperaceae)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.564898" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.564898" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mata-Sucre, Y.; Parteka, L. M.; Ritz, C.; Gatica-Arias, A.; Felix, L. P.; Thomas, W. W.; Souza, G.; Vanzela, A. L. L.; Pedrosa-Harand, A.; Marques, A.</p>
        <p class="info">Score: 12.0, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.564898' target='https://doi.org/10.1101/2023.10.31.564898'> 10.1101/2023.10.31.564898</a></p>
        <p class="abstract">Holocentric karyotypes are assumed to rapidly evolve through chromosome fusions and fissions due to the diffuse nature of their centromeres. Here, we took advantage of the recent availability of a chromosome-scale reference genome for Rhynchospora breviuscula, a model species of this holocentric genus, and developed the first set of oligo-based barcode probes for a holocentric plant. These probes were applied to 13 additional species of the genus, aiming to investigate the evolutionary dynamics driving the karyotype evolution in Rhynchospora. The two sets of probes were composed of 27,392 (green) and 23,968 (magenta) oligonucleotides, and generated 15 distinct FISH signals as a unique barcode pattern for the identification of all five chromosome pairs of the R. breviuscula karyotype. Oligo-FISH comparative analyzes revealed different types of rearrangements, such as fusions, fissions, putative inversions and translocations, as well as genomic duplications among the analyzed species. Two rounds of whole genome duplication (WGD) were demonstrated in R. pubera, but both analyzed accessions differed in the complex chain of events that gave rise to its large, structurally diploidized karyotypes with 2n = 10 or 12. Considering the phylogenetic relationships and divergence time of the species, the specificity and synteny of the probes were maintained up to species with a divergence time of ~25 My. However, karyotype divergence in more distant species hindered chromosome mapping and the inference of specific events. This barcoding system is a powerful tool to study chromosomal variations and genomic evolution in holocentric chromosomes of Rhynchospora species.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
