<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Genomic analysis of Staphylococcus aureus isolates from bacteremia reveals genetic features associated with the COVID-19 pandemic
Authors: Sanchez-Osuna, M.; Pedrosa, M.; Bierge, P.; Gomez-Sanchez, I.; Alguacil-Guillen, M.; Espasa, M.; Erill, I.; Gasch, O.; Pich, O. Q.
Score: 151.1, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572975
Genomic analyses of bacterial isolates are necessary to monitor the prevalence of antibiotic resistance genes and virulence determinants. Herein, we provide a comprehensive genomic description of a collection of 339 Staphylococcus aureus strains isolated from patients with bacteremia between 2014 and 2022.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Genomic analysis of Staphylococcus aureus isolates from bacteremia reveals genetic features associated with the COVID-19 pandemic
Authors: Sanchez-Osuna, M.; Pedrosa, M.; Bierge, P.; Gomez-Sanchez, I.; Alguacil-Guillen, M.; Espasa, M.; Erill, I.; Gasch, O.; Pich, O. Q.
Score: 151.1, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572975
Genomic analyses of bacterial isolates are necessary to monitor the prevalence of antibiotic resistance genes and virulence determinants. Herein, we provide a comprehensive genomic description of a collection of 339 Staphylococcus aureus strains isolated from patients with bacteremia between 2014 and 2022." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-24T10:38:02+00:00" />
<meta property="article:modified_time" content="2023-12-24T10:38:02+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Genomic analysis of Staphylococcus aureus isolates from bacteremia reveals genetic features associated with the COVID-19 pandemic
Authors: Sanchez-Osuna, M.; Pedrosa, M.; Bierge, P.; Gomez-Sanchez, I.; Alguacil-Guillen, M.; Espasa, M.; Erill, I.; Gasch, O.; Pich, O. Q.
Score: 151.1, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572975
Genomic analyses of bacterial isolates are necessary to monitor the prevalence of antibiotic resistance genes and virulence determinants. Herein, we provide a comprehensive genomic description of a collection of 339 Staphylococcus aureus strains isolated from patients with bacteremia between 2014 and 2022."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Genomic analysis of Staphylococcus aureus isolates from bacteremia reveals genetic features associated with the COVID-19 pandemic\nAuthors: Sanchez-Osuna, M.; Pedrosa, M.; Bierge, P.; Gomez-Sanchez, I.; Alguacil-Guillen, M.; Espasa, M.; Erill, I.; Gasch, O.; Pich, O. Q.\nScore: 151.1, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572975\nGenomic analyses of bacterial isolates are necessary to monitor the prevalence of antibiotic resistance genes and virulence determinants. Herein, we provide a comprehensive genomic description of a collection of 339 Staphylococcus aureus strains isolated from patients with bacteremia between 2014 and 2022.",
  "keywords": [
    
  ],
  "articleBody": " Genomic analysis of Staphylococcus aureus isolates from bacteremia reveals genetic features associated with the COVID-19 pandemic\nAuthors: Sanchez-Osuna, M.; Pedrosa, M.; Bierge, P.; Gomez-Sanchez, I.; Alguacil-Guillen, M.; Espasa, M.; Erill, I.; Gasch, O.; Pich, O. Q.\nScore: 151.1, Published: 2023-12-22 DOI: 10.1101/2023.12.22.572975\nGenomic analyses of bacterial isolates are necessary to monitor the prevalence of antibiotic resistance genes and virulence determinants. Herein, we provide a comprehensive genomic description of a collection of 339 Staphylococcus aureus strains isolated from patients with bacteremia between 2014 and 2022. Nosocomial acquisition accounted for 56.6% of episodes, with vascular catheters being the predominant source of infection (31.8%). Cases of fatality (27.4%), persistent bacteremia (19.5%) and diagnosis of septic emboli (24.2%) were documented. During the COVID-19 pandemic, we observed a 140% increase of the episodes of S. aureus bacteremia per year, with a concomitant increase of the cases from nosocomial origin. This prompted us to investigate the existence of genetic features associated with S. aureus isolates from the COVID-19 pandemic. While genes conferring resistance to {beta}-lactams (blaI-blaR-blaZ), macrolides (ermA, ermC, ermT, mphC, msrA) and aminoglycosides (ant(4')-Ia, ant(9)-Ia, aph(3')-IIIa, aph(2'')-Ih) were prevalent in our collection, detection of the msrA and mphC genes increased significantly in pandemic S. aureus isolates. Similarly, we observed a higher prevalence of isolates carrying the genes encoding the Clumping Factors A and B, involved in fibrinogen binding. Of note, macrolides were extensively used as accessory therapy for COVID-19 and fibrinogen levels were usually elevated upon SARS-CoV-2 infection. Therefore, our results reveal a remarkable adaptation of the S. aureus isolates to the COVID-19 pandemic context and demonstrates the potential of whole-genome sequencing to conduct molecular epidemiology studies.\nChIP-DIP: A multiplexed method for mapping hundreds of proteins to DNA uncovers diverse regulatory elements controlling gene expression\nAuthors: Perez, A. A.; Goronzy, I. N.; Blanco, M. R.; Guo, J. K.; Guttman, M.\nScore: 145.5, Published: 2023-12-18 DOI: 10.1101/2023.12.14.571730\nGene expression is controlled by the dynamic localization of thousands of distinct regulatory proteins to precise regions of DNA. Understanding this cell-type specific process has been a goal of molecular biology for decades yet remains challenging because most current DNA-protein mapping methods study one protein at a time. To overcome this, we developed ChIP-DIP (ChIP Done In Parallel), a split-pool based method that enables simultaneous, genome-wide mapping of hundreds of diverse regulatory proteins in a single experiment. We demonstrate that ChIP-DIP generates highly accurate maps for all classes of DNA-associated proteins, including histone modifications, chromatin regulators, transcription factors, and RNA Polymerases. Using these data, we explore quantitative combinations of protein localization on genomic DNA to define distinct classes of regulatory elements and their functional activity. Our data demonstrate that ChIP-DIP enables the generation of consortium level, context-specific protein localization maps within any molecular biology lab.\nImage-based spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis\nAuthors: Vannan, A.; Lyu, R.; Williams, A. L.; Negretti, N. M.; Mee, E. D.; Hirsh, J.; Hirsh, S.; Nichols, D. S.; Calvi, C. L.; Talor, C. J.; Polosukhin, V. V.; Serezani, A. P.; McCall, A. S.; Gokey, J. J.; Shim, H.; Ware, L. B.; Bacchetta, M. J.; Shaver, C. M.; Blackwell, T. S.; Walia, R.; Sucre, J. M.; Kropski, J. A.; McCarthy, D. J.; Banovich, N. E.\nScore: 35.1, Published: 2023-12-17 DOI: 10.1101/2023.12.15.571954\nThe human lung is structurally complex, with a diversity of specialized epithelial, stromal and immune cells playing specific functional roles in anatomically distinct locations, and large-scale changes in the structure and cellular makeup of this distal lung is a hallmark of pulmonary fibrosis (PF) and other progressive chronic lung diseases. Single-cell transcriptomic studies have revealed numerous disease-emergent/enriched cell types/states in PF lungs, but the spatial contexts wherein these cells contribute to disease pathogenesis has remained uncertain. Using sub-cellular resolution image-based spatial transcriptomics, we analyzed the gene expression of more than 1 million cells from 19 unique lungs. Through complementary cell-based and innovative cell-agnostic analyses, we characterized the localization of PF-emergent cell-types, established the cellular and molecular basis of classical PF histopathologic disease features, and identified a diversity of distinct molecularly-defined spatial niches in control and PF lungs. Using machine-learning and trajectory analysis methods to segment and rank airspaces on a gradient from normal to most severely remodeled, we identified a sequence of compositional and molecular changes that associate with progressive distal lung pathology, beginning with alveolar epithelial dysregulation and culminating with changes in macrophage polarization. Together, these results provide a unique, spatially-resolved characterization of the cellular and molecular programs of PF and control lungs, provide new insights into the heterogeneous pathobiology of PF, and establish analytical approaches which should be broadly applicable to other imaging-based spatial transcriptomic studies.\nAtlas of telomeric repeat diversity in Arabidopsis thaliana\nAuthors: Tao, Y.; Xian, W.; Rabanal, F.; Movilli, A.; Lanz, C.; Shirsekar, G.; Weigel, D.\nScore: 19.8, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572118\nTelomeric repeat arrays at the ends of chromosomes are highly dynamic but their repetitive nature and technological limitations have made it difficult to assess the variation in genome diversity surveys. Here we present a comprehensive characterization of the sequence variation immediately adjacent to the canonical telomeric repeat arrays at the very ends of chromosomes in 49 genetically diverse Arabidopsis thaliana accessions. We reveal several types of distinct telomeric repeat units and identify evolutionary processes such as local homogenization and higher-order repeat formation that shape diversity of chromosome ends. The identification of segmental duplications and at least one recombination event suggests a plausible history of telomerase-independent maintenance generation. By comparing largely isogenic samples, we are able to determine variant telomeric repeat number variation at both the germline and somatic levels. Analysis of haplotype structure uncovers chromosome end-specific as well as genetic group-specific patterns in telomeric repeat diversity and provides evidence for linkage disequilibrium between repeat arrays and their adjacent non-coding regions. Together, our findings illustrate the fine-scale telomeric repeat spectrum in A. thaliana, expanding our knowledge of the evolution of chromosome ends.\nSomatic mutation as an explanation for epigenetic aging\nAuthors: Koch, Z.; Li, A. Y.; Evans, D.; Cummings, S.; Ideker, T.\nScore: 16.6, Published: 2023-12-09 DOI: 10.1101/2023.12.08.569638\nDNA methylation marks have recently been used to build models known as \"epigenetic clocks\" which predict calendar age. As methylation of cytosine promotes C-to-T mutations, we hypothesized that the methylation changes observed with age should reflect the accrual of somatic mutations, and the two should yield analogous aging estimates. In analysis of multimodal data from 9,331 human individuals, we find that CpG mutations indeed coincide with changes in methylation, not only at the mutated site but also with pervasive remodeling of the methylome out to {+/-}10 kilobases. This one-to-many mapping enables mutation-based predictions of age that agree with epigenetic clocks, including which individuals are aging faster or slower than expected. Moreover, genomic loci where mutations accumulate with age also tend to have methylation patterns that are especially predictive of age. These results suggest a close coupling between the accumulation of sporadic somatic mutations and the widespread changes in methylation observed over the course of life.\nReconstructing oral cavity tumor evolution through brush biopsy\nAuthors: John, E.; Lesluyes, T.; Baker, T.; Tarabichi, M.; Gillenwater, A.; Wang, J. R.; Van Loo, P.; Zhao, X.\nScore: 12.4, Published: 2023-12-01 DOI: 10.1101/2023.11.29.569216\nOral premalignant lesions (OPLs) with genomic alterations have a heightened risk of evolving into oral squamous cell carcinoma (OSCC). Currently, genomic data are obtained through invasive tissue biopsy. Brush biopsy has been utilized for diagnosing dysplasia but its effectiveness in reflecting the complete genomic landscape of OPLs remains uncertain. This study investigates the potential of brush biopsy samples in accurately reconstructing the genomic profile of OPLs. We analyzed single nucleotide variants (SNVs), copy number aberrations (CNAs), and subclonal structures in paired tissue and brush biopsy samples from a patient with both OPL and OSCC lesions. The results showed that brush biopsy can effectively reflect about 90% of SNVs and similar CNA profiles as those found in tissue biopsies. It was specific, as normal oral epithelium didnt share these genomic alterations. Interestingly, brush biopsy revealed shared SNVs and CNAs between the distinct OPL and OSCC lesions, indicating a common ancestral origin. Subclonal reconstruction confirmed this shared ancestry, followed by divergent evolution of the lesions. These findings highlight the potential of brush biopsies in accurately representing the genomic profile of OPL and OSCC, proving useful in understanding tumor evolution.\nComplete genome of the Medicago anthracnose fungus, Colletotrichum destructivum, reveals a mini-chromosome-like region within a core chromosome.\nAuthors: LAPALU, N.; SIMON, A.; Lu, A.; Plaumann, P.-L.; Amselem, J.; Pigne, S.; Auger, A.; Koch, C.; Dallery, J.-F.; O'Connell, R. J.\nScore: 9.1, Published: 2023-12-16 DOI: 10.1101/2023.12.16.571984\nColletotrichum destructivum (Cd) is a phytopathogenic fungus causing significant economic losses on forage legume crops (Medicago and Trifolium species) worldwide. To gain insights into the genetic basis of fungal virulence and host specificity, we sequenced the genome of an isolate from M. sativa using long-read (PacBio) technology. The resulting genome assembly has a total length of 51.7 Mb and comprises 10 core chromosomes and two accessory chromosomes, all of which were sequenced from telomere to telomere. A total of 15,631 gene models were predicted, including genes encoding potentially pathogenicity-related proteins such as candidate secreted effectors (484), secondary metabolism key enzymes (110) and carbohydrate-active enzymes (619). Synteny analysis revealed extensive structural rearrangements in the genome of Cd relative to the closely-related Brassicaceae pathogen, C. higginsianum. In addition, a 1.2 Mb species-specific region was detected within the largest core chromosome of Cd that has all the characteristics of fungal accessory chromosomes (transposon-rich, gene-poor, distinct codon usage), providing evidence for exchange between these two genomic compartments. This region was also unique in having undergone extensive intra-chromosomal segmental duplications. Our findings provide insights into the evolution of accessory regions and possible mechanisms for generating genetic diversity in this asexual fungal pathogen. Data summaryAll RNA-seq data were submitted to the NCBI GEO portal under the GEO accession GSE246592. C. destructivum genome assembly and annotation are available under the NCBI BioProject PRJNA1029933 with sequence accessions CP137305-CP137317. Supplementary data (genomic and annotation files, genome browser) are available from the INRAE BIOGER Bioinformatics platform (https://bioinfo.bioger.inrae.fr/). Transposable Elements consensus sequences are also available from the French national data repository, research.data.gouv.fr with doi 10.57745/TOO1JS. Impact statementColletotrichum is a large genus of fungal phytopathogens that cause major economic losses on a wide range of crop plants throughout the world. These pathogens vary widely in their host specificity and may have either broad or narrow host ranges. Here, we report the first complete genome of the alfalfa (Medicago sativa) pathogen, Colletotrichum destructivum, which will facilitate the genomic analysis of host adaptation and comparison with other members of the Destructivum clade. We identified a species-specific 1.2 Mb region within chromosome 1 displaying all the hallmarks of fungal accessory chromosomes, which may have arisen through the integration of a mini-chromosome into a core chromosome and could be linked to the pathogenicity of this fungus. We show this region is also a focus for segmental duplications, which may contribute to generating genetic diversity for adaptive evolution. Finally, we report infection by this fungus of the model legume, Medicago truncatula, providing a novel pathosystem for studying fungal-plant interactions.\nAssessing the impact of post-mortem damage and contamination on imputation performance in ancient DNA\nAuthors: Garrido Marques, A.; Rubinacci, S.; Malaspinas, A.-S.; Delaneau, O.; Sousa da Mota, B.\nScore: 9.1, Published: 2023-12-17 DOI: 10.1101/2023.12.17.572049\nLow-coverage imputation is becoming ever more present in ancient DNA (aDNA) studies. Imputation pipelines commonly used for present-day genomes have been shown to yield accurate results when applied to ancient genomes. However, post-mortem damage (PMD), in the form of C-to-T substitutions at the reads termini, and contamination with DNA from closely related species can potentially affect imputation performance in aDNA. In this study, we evaluated imputation performance i) when using a genotype caller designed for aDNA, ATLAS, compared to bcftools, and ii) when contamination is present. We evaluated imputation performance with principal component analyses (PCA) and by calculating imputation error rates. With a particular focus on differently imputed sites, we found that using ATLAS prior to imputation substantially improved imputed genotypes for a very damaged ancient genome (42% PMD). For the remaining genomes, ATLAS brought limited gains. Finally, to examine the effect of contamination on imputation, we added various amounts of reads from two present-day genomes to a previously downsampled high-coverage ancient genome. We observed that imputation accuracy drastically decreased for contamination rates above 5%. In conclusion, we recommend i) accounting for PMD by using a genotype caller such as ATLAS before imputing highly damaged genomes and ii) only imputing genomes containing up to 5% of contamination.\nInteractive visualization of whole eukaryote genome alignments using NCBI's Comparative Genome Viewer (CGV)\nAuthors: Rangwala, S. H.; Rudnev, D. V.; Ananiev, V. V.; Asztalos, A.; Benica, B.; Borodin, E. A.; Bouk, N.; Evgeniev, V. I.; Kodali, V. K.; Lotov, V.; Mozes, E.; Oh, D.-H.; Omelchenko, M. V.; Savkina, S.; Sukharnikov, E.; Virothaisakun, J.; Murphy, T. D.; Pruitt, K. D.; Schneider, V. A.\nScore: 14.7, Published: 2023-11-29 DOI: 10.1101/2023.10.30.564672\nWe report a new visualization tool for analysis of whole genome assembly-assembly alignments, the Comparative Genome Viewer (CGV) (https://ncbi.nlm.nih.gov/genome/cgv/). CGV visualizes pairwise same-species and cross-species alignments provided by NCBI using assembly alignment algorithms developed by us and others. Researchers can examine the alignments between the two assemblies using two alternate views: a chromosome ideogram- based view or a 2D genome dotplot. Whole genome alignment views expose large structural differences spanning chromosomes, such as inversions or translocations. Users can also navigate to regions of interest, where they can detect and analyze smaller-scale deletions and rearrangements within specific chromosome or gene regions. RefSeq or user-provided gene annotation is displayed in the ideogram view where available. CGV currently provides approximately 700 alignments from over 300 animal, plant, and fungal species. CGV and related NCBI viewers are undergoing active development to further meet needs of the research community in comparative genome visualization.\nA Comparative Analysis of Imaging-Based Spatial Transcriptomics Platforms\nAuthors: Cook, D. P.; Jensen, K. B.; Wise, K.; Roache, M. J.; Segato Dezem, F.; Ryan, N. K.; Zamojski, M.; Vlachos, I.; Knott, S. R.; Butler, L. M.; Wrana, J. L.; Banovich, N.; Plummer, J. T.; Martelotto, L. G.\nScore: 42.5, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571385\nSpatial transcriptomics is a rapidly evolving field, overwhelmed by a multitude of technologies. This study aims to offer a comparative analysis of datasets generated from leading in situ imaging platforms. We have generated spatial transcriptomics data from serial sections of prostate adenocarcinoma using the 10x Genomics Xenium and NanoString CosMx SMI platforms. Additionally, orthogonal single-nucleus RNA sequencing (snRNA-seq) was performed on the same FFPE tissue to establish a reference for the tumors transcriptional profiles. We assessed various technical aspects, such as reproducibility, sensitivity, dynamic range, cell segmentation, cell type annotation, and congruence with single-cell profiling. The practicality of assessing cellular organization and biomarker localization was evaluated. Although fewer genes are measured (CosMx: 960, Xenium: 377, with an overlap of 125), Xenium consistently demonstrates higher sensitivity, a broader dynamic range, and better alignment with single-cell reference profiles. Conversely, CosMxs out-of-the-box segmentation outperformed Xeniums, resulting in noticeable transcript misassignment in Xenium within certain tissue areas. However, the impact of this on the cells transcriptional profile was minimal. Together, this comprehensive comparison of two leading commercial platforms for spatial transcriptomics provides essential metrics for assessing their performance, offering invaluable insights for future research and technological advancements in this dynamic field.\n",
  "wordCount" : "2557",
  "inLanguage": "en",
  "datePublished": "2023-12-24T10:38:02Z",
  "dateModified": "2023-12-24T10:38:02Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 24, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.572975">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.572975" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.572975">
        <p class="paperTitle">Genomic analysis of Staphylococcus aureus isolates from bacteremia reveals genetic features associated with the COVID-19 pandemic</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.572975" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.572975" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanchez-Osuna, M.; Pedrosa, M.; Bierge, P.; Gomez-Sanchez, I.; Alguacil-Guillen, M.; Espasa, M.; Erill, I.; Gasch, O.; Pich, O. Q.</p>
        <p class="info">Score: 151.1, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.572975' target='https://doi.org/10.1101/2023.12.22.572975'> 10.1101/2023.12.22.572975</a></p>
        <p class="abstract">Genomic analyses of bacterial isolates are necessary to monitor the prevalence of antibiotic resistance genes and virulence determinants. Herein, we provide a comprehensive genomic description of a collection of 339 Staphylococcus aureus strains isolated from patients with bacteremia between 2014 and 2022. Nosocomial acquisition accounted for 56.6% of episodes, with vascular catheters being the predominant source of infection (31.8%). Cases of fatality (27.4%), persistent bacteremia (19.5%) and diagnosis of septic emboli (24.2%) were documented. During the COVID-19 pandemic, we observed a 140% increase of the episodes of S. aureus bacteremia per year, with a concomitant increase of the cases from nosocomial origin. This prompted us to investigate the existence of genetic features associated with S. aureus isolates from the COVID-19 pandemic. While genes conferring resistance to {beta}-lactams (blaI-blaR-blaZ), macrolides (ermA, ermC, ermT, mphC, msrA) and aminoglycosides (ant(4&#39;)-Ia, ant(9)-Ia, aph(3&#39;)-IIIa, aph(2&#39;&#39;)-Ih) were prevalent in our collection, detection of the msrA and mphC genes increased significantly in pandemic S. aureus isolates. Similarly, we observed a higher prevalence of isolates carrying the genes encoding the Clumping Factors A and B, involved in fibrinogen binding. Of note, macrolides were extensively used as accessory therapy for COVID-19 and fibrinogen levels were usually elevated upon SARS-CoV-2 infection. Therefore, our results reveal a remarkable adaptation of the S. aureus isolates to the COVID-19 pandemic context and demonstrates the potential of whole-genome sequencing to conduct molecular epidemiology studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571730">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571730" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571730">
        <p class="paperTitle">ChIP-DIP: A multiplexed method for mapping hundreds of proteins to DNA uncovers diverse regulatory elements controlling gene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571730" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571730" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perez, A. A.; Goronzy, I. N.; Blanco, M. R.; Guo, J. K.; Guttman, M.</p>
        <p class="info">Score: 145.5, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571730' target='https://doi.org/10.1101/2023.12.14.571730'> 10.1101/2023.12.14.571730</a></p>
        <p class="abstract">Gene expression is controlled by the dynamic localization of thousands of distinct regulatory proteins to precise regions of DNA. Understanding this cell-type specific process has been a goal of molecular biology for decades yet remains challenging because most current DNA-protein mapping methods study one protein at a time. To overcome this, we developed ChIP-DIP (ChIP Done In Parallel), a split-pool based method that enables simultaneous, genome-wide mapping of hundreds of diverse regulatory proteins in a single experiment. We demonstrate that ChIP-DIP generates highly accurate maps for all classes of DNA-associated proteins, including histone modifications, chromatin regulators, transcription factors, and RNA Polymerases. Using these data, we explore quantitative combinations of protein localization on genomic DNA to define distinct classes of regulatory elements and their functional activity. Our data demonstrate that ChIP-DIP enables the generation of  consortium level, context-specific protein localization maps within any molecular biology lab.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571954">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571954" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571954">
        <p class="paperTitle">Image-based spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571954" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571954" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Vannan, A.; Lyu, R.; Williams, A. L.; Negretti, N. M.; Mee, E. D.; Hirsh, J.; Hirsh, S.; Nichols, D. S.; Calvi, C. L.; Talor, C. J.; Polosukhin, V. V.; Serezani, A. P.; McCall, A. S.; Gokey, J. J.; Shim, H.; Ware, L. B.; Bacchetta, M. J.; Shaver, C. M.; Blackwell, T. S.; Walia, R.; Sucre, J. M.; Kropski, J. A.; McCarthy, D. J.; Banovich, N. E.</p>
        <p class="info">Score: 35.1, Published: 2023-12-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571954' target='https://doi.org/10.1101/2023.12.15.571954'> 10.1101/2023.12.15.571954</a></p>
        <p class="abstract">The human lung is structurally complex, with a diversity of specialized epithelial, stromal and immune cells playing specific functional roles in anatomically distinct locations, and large-scale changes in the structure and cellular makeup of this distal lung is a hallmark of pulmonary fibrosis (PF) and other progressive chronic lung diseases. Single-cell transcriptomic studies have revealed numerous disease-emergent/enriched cell types/states in PF lungs, but the spatial contexts wherein these cells contribute to disease pathogenesis has remained uncertain. Using sub-cellular resolution image-based spatial transcriptomics, we analyzed the gene expression of more than 1 million cells from 19 unique lungs. Through complementary cell-based and innovative cell-agnostic analyses, we characterized the localization of PF-emergent cell-types, established the cellular and molecular basis of classical PF histopathologic disease features, and identified a diversity of distinct molecularly-defined spatial niches in control and PF lungs. Using machine-learning and trajectory analysis methods to segment and rank airspaces on a gradient from normal to most severely remodeled, we identified a sequence of compositional and molecular changes that associate with progressive distal lung pathology, beginning with alveolar epithelial dysregulation and culminating with changes in macrophage polarization. Together, these results provide a unique, spatially-resolved characterization of the cellular and molecular programs of PF and control lungs, provide new insights into the heterogeneous pathobiology of PF, and establish analytical approaches which should be broadly applicable to other imaging-based spatial transcriptomic studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572118">
        <p class="paperTitle">Atlas of telomeric repeat diversity in Arabidopsis thaliana</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tao, Y.; Xian, W.; Rabanal, F.; Movilli, A.; Lanz, C.; Shirsekar, G.; Weigel, D.</p>
        <p class="info">Score: 19.8, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572118' target='https://doi.org/10.1101/2023.12.18.572118'> 10.1101/2023.12.18.572118</a></p>
        <p class="abstract">Telomeric repeat arrays at the ends of chromosomes are highly dynamic but their repetitive nature and technological limitations have made it difficult to assess the variation in genome diversity surveys. Here we present a comprehensive characterization of the sequence variation immediately adjacent to the canonical telomeric repeat arrays at the very ends of chromosomes in 49 genetically diverse Arabidopsis thaliana accessions. We reveal several types of distinct telomeric repeat units and identify evolutionary processes such as local homogenization and higher-order repeat formation that shape diversity of chromosome ends. The identification of segmental duplications and at least one recombination event suggests a plausible history of telomerase-independent maintenance generation. By comparing largely isogenic samples, we are able to determine variant telomeric repeat number variation at both the germline and somatic levels. Analysis of haplotype structure uncovers chromosome end-specific as well as genetic group-specific patterns in telomeric repeat diversity and provides evidence for linkage disequilibrium between repeat arrays and their adjacent non-coding regions. Together, our findings illustrate the fine-scale telomeric repeat spectrum in A. thaliana, expanding our knowledge of the evolution of chromosome ends.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.569638">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.569638" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.569638">
        <p class="paperTitle">Somatic mutation as an explanation for epigenetic aging</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.569638" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.569638" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Koch, Z.; Li, A. Y.; Evans, D.; Cummings, S.; Ideker, T.</p>
        <p class="info">Score: 16.6, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.569638' target='https://doi.org/10.1101/2023.12.08.569638'> 10.1101/2023.12.08.569638</a></p>
        <p class="abstract">DNA methylation marks have recently been used to build models known as &#34;epigenetic clocks&#34; which predict calendar age. As methylation of cytosine promotes C-to-T mutations, we hypothesized that the methylation changes observed with age should reflect the accrual of somatic mutations, and the two should yield analogous aging estimates. In analysis of multimodal data from 9,331 human individuals, we find that CpG mutations indeed coincide with changes in methylation, not only at the mutated site but also with pervasive remodeling of the methylome out to {&#43;/-}10 kilobases. This one-to-many mapping enables mutation-based predictions of age that agree with epigenetic clocks, including which individuals are aging faster or slower than expected. Moreover, genomic loci where mutations accumulate with age also tend to have methylation patterns that are especially predictive of age. These results suggest a close coupling between the accumulation of sporadic somatic mutations and the widespread changes in methylation observed over the course of life.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569216">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569216" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569216">
        <p class="paperTitle">Reconstructing oral cavity tumor evolution through brush biopsy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569216" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569216" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: John, E.; Lesluyes, T.; Baker, T.; Tarabichi, M.; Gillenwater, A.; Wang, J. R.; Van Loo, P.; Zhao, X.</p>
        <p class="info">Score: 12.4, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569216' target='https://doi.org/10.1101/2023.11.29.569216'> 10.1101/2023.11.29.569216</a></p>
        <p class="abstract">Oral premalignant lesions (OPLs) with genomic alterations have a heightened risk of evolving into oral squamous cell carcinoma (OSCC). Currently, genomic data are obtained through invasive tissue biopsy. Brush biopsy has been utilized for diagnosing dysplasia but its effectiveness in reflecting the complete genomic landscape of OPLs remains uncertain. This study investigates the potential of brush biopsy samples in accurately reconstructing the genomic profile of OPLs. We analyzed single nucleotide variants (SNVs), copy number aberrations (CNAs), and subclonal structures in paired tissue and brush biopsy samples from a patient with both OPL and OSCC lesions. The results showed that brush biopsy can effectively reflect about 90% of SNVs and similar CNA profiles as those found in tissue biopsies. It was specific, as normal oral epithelium didnt share these genomic alterations. Interestingly, brush biopsy revealed shared SNVs and CNAs between the distinct OPL and OSCC lesions, indicating a common ancestral origin. Subclonal reconstruction confirmed this shared ancestry, followed by divergent evolution of the lesions. These findings highlight the potential of brush biopsies in accurately representing the genomic profile of OPL and OSCC, proving useful in understanding tumor evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.571984">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.571984" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.571984">
        <p class="paperTitle">Complete genome of the Medicago anthracnose fungus, Colletotrichum destructivum, reveals a mini-chromosome-like region within a core chromosome.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.571984" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.571984" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: LAPALU, N.; SIMON, A.; Lu, A.; Plaumann, P.-L.; Amselem, J.; Pigne, S.; Auger, A.; Koch, C.; Dallery, J.-F.; O&#39;Connell, R. J.</p>
        <p class="info">Score: 9.1, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.571984' target='https://doi.org/10.1101/2023.12.16.571984'> 10.1101/2023.12.16.571984</a></p>
        <p class="abstract">Colletotrichum destructivum (Cd) is a phytopathogenic fungus causing significant economic losses on forage legume crops (Medicago and Trifolium species) worldwide. To gain insights into the genetic basis of fungal virulence and host specificity, we sequenced the genome of an isolate from M. sativa using long-read (PacBio) technology. The resulting genome assembly has a total length of 51.7 Mb and comprises 10 core chromosomes and two accessory chromosomes, all of which were sequenced from telomere to telomere. A total of 15,631 gene models were predicted, including genes encoding potentially pathogenicity-related proteins such as candidate secreted effectors (484), secondary metabolism key enzymes (110) and carbohydrate-active enzymes (619). Synteny analysis revealed extensive structural rearrangements in the genome of Cd relative to the closely-related Brassicaceae pathogen, C. higginsianum. In addition, a 1.2 Mb species-specific region was detected within the largest core chromosome of Cd that has all the characteristics of fungal accessory chromosomes (transposon-rich, gene-poor, distinct codon usage), providing evidence for exchange between these two genomic compartments. This region was also unique in having undergone extensive intra-chromosomal segmental duplications. Our findings provide insights into the evolution of accessory regions and possible mechanisms for generating genetic diversity in this asexual fungal pathogen.

Data summaryAll RNA-seq data were submitted to the NCBI GEO portal under the GEO accession GSE246592. C. destructivum genome assembly and annotation are available under the NCBI BioProject PRJNA1029933 with sequence accessions CP137305-CP137317.

Supplementary data (genomic and annotation files, genome browser) are available from the INRAE BIOGER Bioinformatics platform (https://bioinfo.bioger.inrae.fr/). Transposable Elements consensus sequences are also available from the French national data repository, research.data.gouv.fr with doi 10.57745/TOO1JS.

Impact statementColletotrichum is a large genus of fungal phytopathogens that cause major economic losses on a wide range of crop plants throughout the world. These pathogens vary widely in their host specificity and may have either broad or narrow host ranges. Here, we report the first complete genome of the alfalfa (Medicago sativa) pathogen, Colletotrichum destructivum, which will facilitate the genomic analysis of host adaptation and comparison with other members of the Destructivum clade. We identified a species-specific 1.2 Mb region within chromosome 1 displaying all the hallmarks of fungal accessory chromosomes, which may have arisen through the integration of a mini-chromosome into a core chromosome and could be linked to the pathogenicity of this fungus. We show this region is also a focus for segmental duplications, which may contribute to generating genetic diversity for adaptive evolution. Finally, we report infection by this fungus of the model legume, Medicago truncatula, providing a novel pathosystem for studying fungal-plant interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.17.572049">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.17.572049" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.17.572049">
        <p class="paperTitle">Assessing the impact of post-mortem damage and contamination on imputation performance in ancient DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.17.572049" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.17.572049" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Garrido Marques, A.; Rubinacci, S.; Malaspinas, A.-S.; Delaneau, O.; Sousa da Mota, B.</p>
        <p class="info">Score: 9.1, Published: 2023-12-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.17.572049' target='https://doi.org/10.1101/2023.12.17.572049'> 10.1101/2023.12.17.572049</a></p>
        <p class="abstract">Low-coverage imputation is becoming ever more present in ancient DNA (aDNA) studies. Imputation pipelines commonly used for present-day genomes have been shown to yield accurate results when applied to ancient genomes. However, post-mortem damage (PMD), in the form of C-to-T substitutions at the reads termini, and contamination with DNA from closely related species can potentially affect imputation performance in aDNA. In this study, we evaluated imputation performance i) when using a genotype caller designed for aDNA, ATLAS, compared to bcftools, and ii) when contamination is present. We evaluated imputation performance with principal component analyses (PCA) and by calculating imputation error rates. With a particular focus on differently imputed sites, we found that using ATLAS prior to imputation substantially improved imputed genotypes for a very damaged ancient genome (42% PMD). For the remaining genomes, ATLAS brought limited gains. Finally, to examine the effect of contamination on imputation, we added various amounts of reads from two present-day genomes to a previously downsampled high-coverage ancient genome. We observed that imputation accuracy drastically decreased for contamination rates above 5%. In conclusion, we recommend i) accounting for PMD by using a genotype caller such as ATLAS before imputing highly damaged genomes and ii) only imputing genomes containing up to 5% of contamination.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564672">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564672" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564672">
        <p class="paperTitle">Interactive visualization of whole eukaryote genome alignments using NCBI&#39;s Comparative Genome Viewer (CGV)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564672" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564672" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rangwala, S. H.; Rudnev, D. V.; Ananiev, V. V.; Asztalos, A.; Benica, B.; Borodin, E. A.; Bouk, N.; Evgeniev, V. I.; Kodali, V. K.; Lotov, V.; Mozes, E.; Oh, D.-H.; Omelchenko, M. V.; Savkina, S.; Sukharnikov, E.; Virothaisakun, J.; Murphy, T. D.; Pruitt, K. D.; Schneider, V. A.</p>
        <p class="info">Score: 14.7, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564672' target='https://doi.org/10.1101/2023.10.30.564672'> 10.1101/2023.10.30.564672</a></p>
        <p class="abstract">We report a new visualization tool for analysis of whole genome assembly-assembly alignments, the Comparative Genome Viewer (CGV) (https://ncbi.nlm.nih.gov/genome/cgv/). CGV visualizes pairwise same-species and cross-species alignments provided by NCBI using assembly alignment algorithms developed by us and others. Researchers can examine the alignments between the two assemblies using two alternate views: a chromosome ideogram- based view or a 2D genome dotplot. Whole genome alignment views expose large structural differences spanning chromosomes, such as inversions or translocations. Users can also navigate to regions of interest, where they can detect and analyze smaller-scale deletions and rearrangements within specific chromosome or gene regions. RefSeq or user-provided gene annotation is displayed in the ideogram view where available. CGV currently provides approximately 700 alignments from over 300 animal, plant, and fungal species. CGV and related NCBI viewers are undergoing active development to further meet needs of the research community in comparative genome visualization.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571385">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571385" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571385">
        <p class="paperTitle">A Comparative Analysis of Imaging-Based Spatial Transcriptomics Platforms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571385" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571385" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cook, D. P.; Jensen, K. B.; Wise, K.; Roache, M. J.; Segato Dezem, F.; Ryan, N. K.; Zamojski, M.; Vlachos, I.; Knott, S. R.; Butler, L. M.; Wrana, J. L.; Banovich, N.; Plummer, J. T.; Martelotto, L. G.</p>
        <p class="info">Score: 42.5, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571385' target='https://doi.org/10.1101/2023.12.13.571385'> 10.1101/2023.12.13.571385</a></p>
        <p class="abstract">Spatial transcriptomics is a rapidly evolving field, overwhelmed by a multitude of technologies. This study aims to offer a comparative analysis of datasets generated from leading in situ imaging platforms. We have generated spatial transcriptomics data from serial sections of prostate adenocarcinoma using the 10x Genomics Xenium and NanoString CosMx SMI platforms. Additionally, orthogonal single-nucleus RNA sequencing (snRNA-seq) was performed on the same FFPE tissue to establish a reference for the tumors transcriptional profiles. We assessed various technical aspects, such as reproducibility, sensitivity, dynamic range, cell segmentation, cell type annotation, and congruence with single-cell profiling. The practicality of assessing cellular organization and biomarker localization was evaluated. Although fewer genes are measured (CosMx: 960, Xenium: 377, with an overlap of 125), Xenium consistently demonstrates higher sensitivity, a broader dynamic range, and better alignment with single-cell reference profiles. Conversely, CosMxs out-of-the-box segmentation outperformed Xeniums, resulting in noticeable transcript misassignment in Xenium within certain tissue areas. However, the impact of this on the cells transcriptional profile was minimal. Together, this comprehensive comparison of two leading commercial platforms for spatial transcriptomics provides essential metrics for assessing their performance, offering invaluable insights for future research and technological advancements in this dynamic field.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
