<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation
Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.
Score: 77.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582
Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation
Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.
Score: 77.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582
Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-03T10:37:18+00:00" />
<meta property="article:modified_time" content="2023-09-03T10:37:18+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation
Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.
Score: 77.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582
Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation\nAuthors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.\nScore: 77.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582\nSequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence.",
  "keywords": [
    
  ],
  "articleBody": " Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation\nAuthors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.\nScore: 77.3, Published: 2023-09-01 DOI: 10.1101/2023.08.30.555582\nSequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence. Using statistics derived from Borzoi's predicted coverage, we isolate and accurately score variant effects across multiple layers of regulation, including transcription, splicing, and polyadenylation. Evaluated on QTLs, Borzoi is competitive with, and often outperforms, state-of-the-art models trained on individual regulatory functions. By applying attribution methods to the derived statistics, we extract cis-regulatory patterns driving RNA expression and post-transcriptional regulation in normal tissues. The wide availability of RNA-seq data across species, conditions, and assays profiling specific aspects of regulation emphasizes the potential of this approach to decipher the mapping from DNA sequence to regulatory function.\nThe genome of the colonial hydroid Hydractinia reveals their stem cells use a toolkit of genes shared with all animals\nAuthors: Schnitzler, C. E.; Chang, E. S.; Waletich, J.; Wong, W. Y.; Nguyen, A.-D.; Barreira, S.; Doonan, L.; Gonzalez, P.; Koren, S.; Gahan, J.; Quiroga-Artigas, G.; Sanders, S.; Bradshaw, B.; DuBuc, T.; Febrimarsa, F.; de Jong, D.; Nawrocki, E.; Larson, A.; Klasfeld, S.; Gornik, S.; Moreland, R. T.; Wolfsberg, T.; Phillippy, A. M.; Mullikin, J.; Simakov, O.; Cartwright, P.; Nicotra, M.; Frank, U.; Baxevanis, A. D.\nScore: 18.9, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554815\nHydractinia is a colonial marine hydroid that exhibits remarkable biological properties, including the capacity to regenerate its entire body throughout its lifetime, a process made possible by its adult migratory stem cells, known as i-cells. Here, we provide an in-depth characterization of the genomic structure and gene content of two Hydractinia species, H. symbiolongicarpus and H. echinata, placing them in a comparative evolutionary framework with other cnidarian genomes. We also generated and annotated a single-cell transcriptomic atlas for adult male H. symbiolongicarpus and identified cell type markers for all major cell types, including key i-cell markers. Orthology analyses based on the markers revealed that Hydractinias i-cells are highly enriched in genes that are widely shared amongst animals, a striking finding given that Hydractinia has a higher proportion of phylum-specific genes than any of the other 41 animals in our orthology analysis. These results indicate that Hydractinias stem cells and early progenitor cells may use a toolkit shared with all animals, making it a promising model organism for future exploration of stem cell biology and regenerative medicine. The genomic and transcriptomic resources for Hydractinia presented here will enable further studies of their regenerative capacity, colonial morphology, and ability to distinguish self from non-self.\nBoth phenotypic and genotypic sex influence sex chromosome dosage compensation in a sex reversing lizard\nAuthors: Hanrahan, B. J.; Chang, J. K.; Lister, N. C.; Dissanayake, D. S. B.; Hammond, J. M.; Reis, A. L. M.; Deveson, I. W.; Ruiz-Herrera, A.; Patel, H. R.; Marshall Graves, J. A.; Georges, A.; Waters, P. D.\nScore: 17.8, Published: 2023-08-26 DOI: 10.1101/2023.08.24.554710\nStudies of sex chromosome dosage compensation have historically focussed on therian mammals which have a conserved XY sex determination system. In contrast, lizards have sex determination systems that can differ between even closely related species that include XY and ZW systems and thermolabile systems where genetic and temperature interact to various degrees to determine sex. The eastern three-lined skink (Bassiana duperreyi) has a differentiated XY sex determination system, in which low temperature incubation during development can cause female to male sex reversal, producing XX males. This provides a unique opportunity to investigate how genotype and phenotype affect dosage compensation. We generated transcriptomes from brain and heart tissue of normal adult males and females, along with brain tissue of sex-reversed XX males. We observed partial dosage compensation between XX females and XY males in both brain and heart, with median gene expression from the X in normal males being 0.7 times that of normal females. Surprisingly, in brain of sex reversed XX males the median X chromosome output did not match that of either normal males or females, but instead was 0.89 times that of the normal XX female level. This suggests that not just genotype, but also sexual phenotype, influences gene dosage of the X chromosome. This has profound implications for our understanding of the evolution of dosage compensation.\nA novel, N2-fixing cyanobacterium present and active in the global oceans\nAuthors: Cleveland, C. S.; Turk-Kubo, K. A.; Zhao, Y.; Zehr, J. P.; Webb, E. A.\nScore: 14.9, Published: 2023-08-28 DOI: 10.1101/2023.08.27.555023\nMarine N2-fixing cyanobacteria, including the unicellular genus Crocosphaera, are considered keystone species in marine food webs. Crocosphaera are globally distributed and provide new sources of nitrogen (N) and carbon (C), which fuel oligotrophic microbial communities and upper trophic levels. Despite their ecosystem importance, only one species, Crocosphaera watsonii, has ever been identified and characterized as widespread in the oligotrophic oceans. Herein, we present a novel species, candidatus Crocosphaera waterburyi (C. waterburyi hereafter), enriched from the North Pacific Ocean, active in situ, and globally distributed in environmental datasets. C. waterburyi is morphologically, phylogenetically, and physiologically distinct from C. watsonii; therefore, description of this novel species provides a new window into previously uncharacterized diversity and ecology of unicellular N2-fixing cyanobacterial taxa and further highlights their importance in the global N cycle.\nThree Accessory Gene Clusters Drive Host-Adaptation in Group B Streptococcus\nAuthors: Crestani, C.; Forde, T. L.; Bell, J.; Lycett, S. J.; Oliveira, L. M.; Pinto, T. C.; Cobo-Angel, C. G.; Ceballos-Marquez, A.; Phuoc, N. N.; Sirimanapong, W.; Chen, S. L.; Jamrozy, D.; Bentley, S. D.; Fontaine, M.; Zadoks, R. N.\nScore: 10.0, Published: 2023-08-13 DOI: 10.1101/2023.08.10.552778\nStreptococcus agalactiae (Group B Streptococcus, GBS) is a major pathogen of humans and animals, posing a threat to human health as well as food security. Here, we investigate the role of genomic mechanisms, including homologous recombination and horizontal gene transfer, in shaping the population structure of GBS and its adaptation to three major host groups (humans, cattle, fishes). We demonstrate that the GBS population comprises host-specialist, host-adapted lineages as well as host generalists, and that these categories differ in their level or recombination. Although the accessory genome at large varies by lineage rather than host, genome wide association studies show that host association is driven by three accessory genome clusters, regardless of lineage or breadth of the host spectrum. These genomic clusters (scpB in human GBS, lactose operon in bovine GBS, Locus 3 in fish GBS) are known (scpB, Lac.2) or shown here (Locus 3) to be functionally relevant and are shared with other streptococcal species occupying the same host niche. These findings demonstrate the importance of considering the role of non-human host species in the evolution of GBS, including high risk clones that may lead to interspecies transmission and affect efficacy of future GBS vaccines.\nFunctional annotation of a divergent genome using sequence and structure-based homology.\nAuthors: Svedberg, D.; Winiger, R. R.; Berg, A.; Sharma, H.; Debrunner-Vossbrinck, B. A.; Vossbrinck, C. R.; Barandun, J.\nScore: 9.2, Published: 2023-08-28 DOI: 10.1101/2023.08.27.554996\nBackgroundMicrosporidia are a large taxon of intracellular pathogens characterized by extraordinarily streamlined genomes with unusually high sequence divergence and many species-specific adaptations. These unique factors pose challenges for traditional genome annotation methods based on sequence homology. As a result, many of the microsporidian genomes sequenced to date contain numerous genes of unknown function. Recent innovations in rapid and accurate structure prediction and comparison, together with the growing amount of data in structural databases, provide new opportunities to assist in the functional annotation of newly sequenced genomes. ResultsIn this study, we established a workflow that combines sequence and structure-based functional gene annotation approaches employing a ChimeraX plugin, allowing for visual inspection and manual curation. We employed this workflow on a high-quality telomere-to-telomere sequenced tetraploid genome of Vairimorpha necatrix. First, the 3080 predicted open reading frames, of which 89 % were confirmed with RNA sequencing data, were used as input. Next, ColabFold was used to create protein structure predictions, followed by a Foldseek search for structural matching to the PDB and AlphaFold databases. The subsequent manual curation, using sequence and structure-based hits, increased the accuracy and quality of the functional genome annotation compared to results using only traditional annotation tools. Our workflow resulted in a comprehensive description of the V. necatrix genome, along with a structural summary of the most prevalent protein groups, such as the ricin B lectin family. In addition, and to test our tool, we identified the functions of several previously uncharacterized Encephalitozoon cuniculi genes. ConclusionWe provide a new functional annotation tool for divergent organisms and employ it on a newly sequenced, high-quality microsporidian genome to shed light on this uncharacterized intracellular pathogen of Lepidoptera. The addition of a structure-based annotation approach can serve as a valuable template for studying other microsporidian or similarly divergent species.\nMetagenomic analysis of the honey bee queen microbiome reveals low bacterial diversity and Caudoviricetes phages\nAuthors: Caesar, L.; Rice, D. W.; McAfee, A.; Underwood, R.; Tarpy, D. R.; J Foster, L.; Newton, I. L. G.\nScore: 8.9, Published: 2023-09-01 DOI: 10.1101/2023.08.29.555332\nIn eusocial insects, the health of the queens - the colony founders and sole reproductive females - is a primary determinant for colony success. Queen failure in the honey bee Apis mellifera, for example, is a major concern of beekeepers that annually suffer colony losses, necessitating a greater knowledge of queen health. Several studies on the microbiome of honey bees have characterized its diversity and shown its importance for the health of worker bees, the female non-reproductive caste. However, the microbiome of workers differs from that of queens, which in comparison is still poorly studied. Thus, direct investigations of the queen microbiome are required to understand colony-level microbiome assembly, functional roles, and evolution. Here we used metagenomics to comprehensively characterize the honey bee queen microbiome. Comparing samples from different geographic locations and breeder sources, we show that the microbiome of queens is mostly shaped by the environment experienced since early life, and is predicted to play roles in breakdown of the diet, and protection from pathogens and xenobiotics. We also reveal the microbiome of queens comprises only four core bacterial species, Apilactobacillus kunkeei, Lactobacillus apis, Bombella apis and Commensalibacter sp. Interestingly, in addition to bacteria, we show that bacteriophages infect the queen microbiome, for which Lactobacillaceae are predicted to be the main reservoirs. Together, our results provide the basis to understand the honey bee colony microbiome assemblage, can guide improvements in queen rearing processes, and highlight the importance of bacteriophages for queen microbiome health and microbiome homeostasis in eusocial insects.\nThe genome of the Wollemi pine, a critically endangered living fossil unchanged since the Cretaceous, reveals extensive ancient transposon activity.\nAuthors: Stevenson, D. W.; Ramakrishnan, S.; Alves, C. d. S.; Coelho, L. A.; Kramer, M.; Goodwin, S.; Ramos, O. M.; Eshel, G.; Sondervan, V. M.; Frangos, S.; Zumajo-Cardona, C.; Jenike, K.; Ou, S.; Wang, X.; Lee, Y. P.; Loke, S.; Rossetto, M.; McPherson, H.; Nigris, S.; Moschin, S.; Little, D. P.; Katari, M. S.; Varala, K.; Kolokotronis, S.-O.; Ambrose, B.; Croft, L. J.; Coruzzi, G. M.; Schatz, M.; McCombie, W. R.; Martienssen, R. A.\nScore: 8.2, Published: 2023-08-24 DOI: 10.1101/2023.08.24.554647\nWe present the genome of the living fossil, Wollemia nobilis, a southern hemisphere conifer morphologically unchanged since the Cretaceous. Presumed extinct until rediscovery in 1994, the Wollemi pine is critically endangered with less than 60 wild adults threatened by intensifying bushfires in the Blue Mountains of Australia. The 12 Gb genome is among the most contiguous large plant genomes assembled, with extremely low heterozygosity and unusual abundance of DNA transposons. Reduced representation and genome re-sequencing of individuals confirms a relictual population since the last major glacial/drying period in Australia, 120 ky BP. Small RNA and methylome sequencing reveal conservation of ancient silencing mechanisms despite the presence of thousands of active and abundant transposons, including some transferred horizontally to conifers from arthropods in the Jurassic. A retrotransposon burst 8-6 my BP coincided with population decline, possibly as an adaptation enhancing epigenetic diversity. Wollemia, like other conifers, is susceptible to Phytophthora, and a suite of defense genes, similar to those in loblolly pine, are targeted for silencing by sRNAs in leaves. The genome provides insight into the earliest seed plants, while enabling conservation efforts.\nActivity of MukBEF for chromosome management in E. coli and its inhibition by MatP\nAuthors: Seba, M.; Boccard, F.; Duigou, S.\nScore: 4.7, Published: 2023-08-31 DOI: 10.1101/2023.08.30.555477\nWhile different features for the activity of the bacterial canonical SMC complex, Smc-ScpAB, have been described in different bacteria, not much is known about the way chromosomes in enterobacteria interact with their SMC complex, MukBEF. Here we used a number of in vivo assays in E. coli to reveal how MukBEF controls chromosome conformation and how the MatP/matS system prevents MukBEF activity. Our results indicate that the loading of MukBEF occurs preferentially in newly replicated DNA, at multiple loci on the chromosome where it can promote long-range contacts in cis even though MukBEF can promote long-range contacts in the absence of replication. Using HiC and ChIP-seq analyses in strains with rearranged chromosomes, the prevention of MukBEF activity increases with the number of matS sites and this effect likely results from the unloading of MukBEF by MatP. Altogether, our results reveal how MukBEF operates to control chromosome folding and segregation in E. coli.\nHigh-quality genome of the zoophytophagous stinkbug, Nesidiocoris tenuis, informs their food habitadaptation\nAuthors: Shibata, T.; Shimoda, M.; Kobayashi, T.; Arai, H.; Owashi, Y.; Uehara, T.\nScore: 3.9, Published: 2023-08-31 DOI: 10.1101/2023.08.29.555439\nThe zoophytophagous stink bug, Nesidiocoris tenuis, is a promising natural enemy of micro-pests such as whiteflies and thrips. This bug possesses both phytophagous and entomophagous food habits, enabling it to obtain nutrition from both plants and insects. This trait allows us to maintain its population density in agricultural fields by introducing insectary plants, even when the pest prey density is extremely low. However, if the bugs' population becomes too dense, they can sometimes damage crop plants. This dual character seems to arise from the food preferences and chemosensation of this predator. To understand the genomic landscape of N. tenuis, we examined the whole genome sequence of a commercially available Japanese strain. We used long-read sequencing and Hi-C analysis to assemble the genome at the chromosomal level. We then conducted a comparative analysis of the genome with previously reported genomes of phytophagous and hematophagous stink bugs to focus on the genetic factors contributing to this species' herbivorous and carnivorous tendencies. Our findings suggest that the gustatory gene set plays a pivotal role in adapting to food habits, making it a promising target for selective breeding. Furthermore, we identified the whole genomes of microorganisms symbiotic with this species through genomic analysis. We believe that our results shed light on the food habit adaptations of N. tenuis and will accelerate breeding efforts based on new breeding techniques for natural enemy insects, including genomics and genome editing.\n",
  "wordCount" : "2504",
  "inLanguage": "en",
  "datePublished": "2023-09-03T10:37:18Z",
  "dateModified": "2023-09-03T10:37:18Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on September 3, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555582">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555582" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555582">
        <p class="paperTitle">Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555582" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555582" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Linder, J.; Srivastava, D.; Yuan, H.; Agarwal, V.; Kelley, D. R.</p>
        <p class="info">Score: 77.3, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555582' target='https://doi.org/10.1101/2023.08.30.555582'> 10.1101/2023.08.30.555582</a></p>
        <p class="abstract">Sequence-based machine learning models trained on genome-scale biochemical assays improve our ability to interpret genetic variants by providing functional predictions describing their impact on the cis-regulatory code. Here, we introduce a new model, Borzoi, which learns to predict cell- and tissue-specific RNA-seq coverage from DNA sequence. Using statistics derived from Borzoi&#39;s predicted coverage, we isolate and accurately score variant effects across multiple layers of regulation, including transcription, splicing, and polyadenylation. Evaluated on QTLs, Borzoi is competitive with, and often outperforms, state-of-the-art models trained on individual regulatory functions. By applying attribution methods to the derived statistics, we extract cis-regulatory patterns driving RNA expression and post-transcriptional regulation in normal tissues. The wide availability of RNA-seq data across species, conditions, and assays profiling specific aspects of regulation emphasizes the potential of this approach to decipher the mapping from DNA sequence to regulatory function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554815">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554815" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554815">
        <p class="paperTitle">The genome of the colonial hydroid Hydractinia reveals their stem cells use a toolkit of genes shared with all animals</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554815" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554815" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schnitzler, C. E.; Chang, E. S.; Waletich, J.; Wong, W. Y.; Nguyen, A.-D.; Barreira, S.; Doonan, L.; Gonzalez, P.; Koren, S.; Gahan, J.; Quiroga-Artigas, G.; Sanders, S.; Bradshaw, B.; DuBuc, T.; Febrimarsa, F.; de Jong, D.; Nawrocki, E.; Larson, A.; Klasfeld, S.; Gornik, S.; Moreland, R. T.; Wolfsberg, T.; Phillippy, A. M.; Mullikin, J.; Simakov, O.; Cartwright, P.; Nicotra, M.; Frank, U.; Baxevanis, A. D.</p>
        <p class="info">Score: 18.9, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554815' target='https://doi.org/10.1101/2023.08.25.554815'> 10.1101/2023.08.25.554815</a></p>
        <p class="abstract">Hydractinia is a colonial marine hydroid that exhibits remarkable biological properties, including the capacity to regenerate its entire body throughout its lifetime, a process made possible by its adult migratory stem cells, known as i-cells. Here, we provide an in-depth characterization of the genomic structure and gene content of two Hydractinia species, H. symbiolongicarpus and H. echinata, placing them in a comparative evolutionary framework with other cnidarian genomes. We also generated and annotated a single-cell transcriptomic atlas for adult male H. symbiolongicarpus and identified cell type markers for all major cell types, including key i-cell markers. Orthology analyses based on the markers revealed that Hydractinias i-cells are highly enriched in genes that are widely shared amongst animals, a striking finding given that Hydractinia has a higher proportion of phylum-specific genes than any of the other 41 animals in our orthology analysis. These results indicate that Hydractinias stem cells and early progenitor cells may use a toolkit shared with all animals, making it a promising model organism for future exploration of stem cell biology and regenerative medicine. The genomic and transcriptomic resources for Hydractinia presented here will enable further studies of their regenerative capacity, colonial morphology, and ability to distinguish self from non-self.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554710">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554710" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554710">
        <p class="paperTitle">Both phenotypic and genotypic sex influence sex chromosome dosage compensation in a sex reversing lizard</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554710" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554710" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hanrahan, B. J.; Chang, J. K.; Lister, N. C.; Dissanayake, D. S. B.; Hammond, J. M.; Reis, A. L. M.; Deveson, I. W.; Ruiz-Herrera, A.; Patel, H. R.; Marshall Graves, J. A.; Georges, A.; Waters, P. D.</p>
        <p class="info">Score: 17.8, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554710' target='https://doi.org/10.1101/2023.08.24.554710'> 10.1101/2023.08.24.554710</a></p>
        <p class="abstract">Studies of sex chromosome dosage compensation have historically focussed on therian mammals which have a conserved XY sex determination system. In contrast, lizards have sex determination systems that can differ between even closely related species that include XY and ZW systems and thermolabile systems where genetic and temperature interact to various degrees to determine sex. The eastern three-lined skink (Bassiana duperreyi) has a differentiated XY sex determination system, in which low temperature incubation during development can cause female to male sex reversal, producing XX males. This provides a unique opportunity to investigate how genotype and phenotype affect dosage compensation. We generated transcriptomes from brain and heart tissue of normal adult males and females, along with brain tissue of sex-reversed XX males. We observed partial dosage compensation between XX females and XY males in both brain and heart, with median gene expression from the X in normal males being 0.7 times that of normal females. Surprisingly, in brain of sex reversed XX males the median X chromosome output did not match that of either normal males or females, but instead was 0.89 times that of the normal XX female level. This suggests that not just genotype, but also sexual phenotype, influences gene dosage of the X chromosome. This has profound implications for our understanding of the evolution of dosage compensation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.555023">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.555023" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.555023">
        <p class="paperTitle">A novel, N2-fixing cyanobacterium present and active in the global oceans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.555023" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.555023" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cleveland, C. S.; Turk-Kubo, K. A.; Zhao, Y.; Zehr, J. P.; Webb, E. A.</p>
        <p class="info">Score: 14.9, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.555023' target='https://doi.org/10.1101/2023.08.27.555023'> 10.1101/2023.08.27.555023</a></p>
        <p class="abstract">Marine N2-fixing cyanobacteria, including the unicellular genus Crocosphaera, are considered keystone species in marine food webs. Crocosphaera are globally distributed and provide new sources of nitrogen (N) and carbon (C), which fuel oligotrophic microbial communities and upper trophic levels. Despite their ecosystem importance, only one species, Crocosphaera watsonii, has ever been identified and characterized as widespread in the oligotrophic oceans. Herein, we present a novel species, candidatus Crocosphaera waterburyi (C. waterburyi hereafter), enriched from the North Pacific Ocean, active in situ, and globally distributed in environmental datasets. C. waterburyi is morphologically, phylogenetically, and physiologically distinct from C. watsonii; therefore, description of this novel species provides a new window into previously uncharacterized diversity and ecology of unicellular N2-fixing cyanobacterial taxa and further highlights their importance in the global N cycle.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.10.552778">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.10.552778" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.10.552778">
        <p class="paperTitle">Three Accessory Gene Clusters Drive Host-Adaptation in Group B Streptococcus</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.10.552778" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.10.552778" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Crestani, C.; Forde, T. L.; Bell, J.; Lycett, S. J.; Oliveira, L. M.; Pinto, T. C.; Cobo-Angel, C. G.; Ceballos-Marquez, A.; Phuoc, N. N.; Sirimanapong, W.; Chen, S. L.; Jamrozy, D.; Bentley, S. D.; Fontaine, M.; Zadoks, R. N.</p>
        <p class="info">Score: 10.0, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.10.552778' target='https://doi.org/10.1101/2023.08.10.552778'> 10.1101/2023.08.10.552778</a></p>
        <p class="abstract">Streptococcus agalactiae (Group B Streptococcus, GBS) is a major pathogen of humans and animals, posing a threat to human health as well as food security. Here, we investigate the role of genomic mechanisms, including homologous recombination and horizontal gene transfer, in shaping the population structure of GBS and its adaptation to three major host groups (humans, cattle, fishes). We demonstrate that the GBS population comprises host-specialist, host-adapted lineages as well as host generalists, and that these categories differ in their level or recombination. Although the accessory genome at large varies by lineage rather than host, genome wide association studies show that host association is driven by three accessory genome clusters, regardless of lineage or breadth of the host spectrum. These genomic clusters (scpB in human GBS, lactose operon in bovine GBS, Locus 3 in fish GBS) are known (scpB, Lac.2) or shown here (Locus 3) to be functionally relevant and are shared with other streptococcal species occupying the same host niche. These findings demonstrate the importance of considering the role of non-human host species in the evolution of GBS, including high risk clones that may lead to interspecies transmission and affect efficacy of future GBS vaccines.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.554996">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.554996" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.554996">
        <p class="paperTitle">Functional annotation of a divergent genome using sequence and structure-based homology.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.554996" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.554996" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Svedberg, D.; Winiger, R. R.; Berg, A.; Sharma, H.; Debrunner-Vossbrinck, B. A.; Vossbrinck, C. R.; Barandun, J.</p>
        <p class="info">Score: 9.2, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.554996' target='https://doi.org/10.1101/2023.08.27.554996'> 10.1101/2023.08.27.554996</a></p>
        <p class="abstract">BackgroundMicrosporidia are a large taxon of intracellular pathogens characterized by extraordinarily streamlined genomes with unusually high sequence divergence and many species-specific adaptations. These unique factors pose challenges for traditional genome annotation methods based on sequence homology. As a result, many of the microsporidian genomes sequenced to date contain numerous genes of unknown function. Recent innovations in rapid and accurate structure prediction and comparison, together with the growing amount of data in structural databases, provide new opportunities to assist in the functional annotation of newly sequenced genomes.

ResultsIn this study, we established a workflow that combines sequence and structure-based functional gene annotation approaches employing a ChimeraX plugin, allowing for visual inspection and manual curation. We employed this workflow on a high-quality telomere-to-telomere sequenced tetraploid genome of Vairimorpha necatrix. First, the 3080 predicted open reading frames, of which 89 % were confirmed with RNA sequencing data, were used as input. Next, ColabFold was used to create protein structure predictions, followed by a Foldseek search for structural matching to the PDB and AlphaFold databases. The subsequent manual curation, using sequence and structure-based hits, increased the accuracy and quality of the functional genome annotation compared to results using only traditional annotation tools. Our workflow resulted in a comprehensive description of the V. necatrix genome, along with a structural summary of the most prevalent protein groups, such as the ricin B lectin family. In addition, and to test our tool, we identified the functions of several previously uncharacterized Encephalitozoon cuniculi genes.

ConclusionWe provide a new functional annotation tool for divergent organisms and employ it on a newly sequenced, high-quality microsporidian genome to shed light on this uncharacterized intracellular pathogen of Lepidoptera. The addition of a structure-based annotation approach can serve as a valuable template for studying other microsporidian or similarly divergent species.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555332">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555332" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555332">
        <p class="paperTitle">Metagenomic analysis of the honey bee queen microbiome reveals low bacterial diversity and Caudoviricetes phages</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555332" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555332" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caesar, L.; Rice, D. W.; McAfee, A.; Underwood, R.; Tarpy, D. R.; J Foster, L.; Newton, I. L. G.</p>
        <p class="info">Score: 8.9, Published: 2023-09-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555332' target='https://doi.org/10.1101/2023.08.29.555332'> 10.1101/2023.08.29.555332</a></p>
        <p class="abstract">In eusocial insects, the health of the queens - the colony founders and sole reproductive females - is a primary determinant for colony success. Queen failure in the honey bee Apis mellifera, for example, is a major concern of beekeepers that annually suffer colony losses, necessitating a greater knowledge of queen health. Several studies on the microbiome of honey bees have characterized its diversity and shown its importance for the health of worker bees, the female non-reproductive caste. However, the microbiome of workers differs from that of queens, which in comparison is still poorly studied. Thus, direct investigations of the queen microbiome are required to understand colony-level microbiome assembly, functional roles, and evolution. Here we used metagenomics to comprehensively characterize the honey bee queen microbiome. Comparing samples from different geographic locations and breeder sources, we show that the microbiome of queens is mostly shaped by the environment experienced since early life, and is predicted to play roles in breakdown of the diet, and protection from pathogens and xenobiotics. We also reveal the microbiome of queens comprises only four core bacterial species, Apilactobacillus kunkeei, Lactobacillus apis, Bombella apis and Commensalibacter sp. Interestingly, in addition to bacteria, we show that bacteriophages infect the queen microbiome, for which Lactobacillaceae are predicted to be the main reservoirs. Together, our results provide the basis to understand the honey bee colony microbiome assemblage, can guide improvements in queen rearing processes, and highlight the importance of bacteriophages for queen microbiome health and microbiome homeostasis in eusocial insects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554647">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554647" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554647">
        <p class="paperTitle">The genome of the Wollemi pine, a critically endangered living fossil unchanged since the Cretaceous, reveals extensive ancient transposon activity.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554647" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554647" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stevenson, D. W.; Ramakrishnan, S.; Alves, C. d. S.; Coelho, L. A.; Kramer, M.; Goodwin, S.; Ramos, O. M.; Eshel, G.; Sondervan, V. M.; Frangos, S.; Zumajo-Cardona, C.; Jenike, K.; Ou, S.; Wang, X.; Lee, Y. P.; Loke, S.; Rossetto, M.; McPherson, H.; Nigris, S.; Moschin, S.; Little, D. P.; Katari, M. S.; Varala, K.; Kolokotronis, S.-O.; Ambrose, B.; Croft, L. J.; Coruzzi, G. M.; Schatz, M.; McCombie, W. R.; Martienssen, R. A.</p>
        <p class="info">Score: 8.2, Published: 2023-08-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554647' target='https://doi.org/10.1101/2023.08.24.554647'> 10.1101/2023.08.24.554647</a></p>
        <p class="abstract">We present the genome of the living fossil, Wollemia nobilis, a southern hemisphere conifer morphologically unchanged since the Cretaceous. Presumed extinct until rediscovery in 1994, the Wollemi pine is critically endangered with less than 60 wild adults threatened by intensifying bushfires in the Blue Mountains of Australia. The 12 Gb genome is among the most contiguous large plant genomes assembled, with extremely low heterozygosity and unusual abundance of DNA transposons. Reduced representation and genome re-sequencing of individuals confirms a relictual population since the last major glacial/drying period in Australia, 120 ky BP. Small RNA and methylome sequencing reveal conservation of ancient silencing mechanisms despite the presence of thousands of active and abundant transposons, including some transferred horizontally to conifers from arthropods in the Jurassic. A retrotransposon burst 8-6 my BP coincided with population decline, possibly as an adaptation enhancing epigenetic diversity. Wollemia, like other conifers, is susceptible to Phytophthora, and a suite of defense genes, similar to those in loblolly pine, are targeted for silencing by sRNAs in leaves. The genome provides insight into the earliest seed plants, while enabling conservation efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555477">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555477" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555477">
        <p class="paperTitle">Activity of MukBEF for chromosome management in E. coli and its inhibition by MatP</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555477" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555477" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seba, M.; Boccard, F.; Duigou, S.</p>
        <p class="info">Score: 4.7, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555477' target='https://doi.org/10.1101/2023.08.30.555477'> 10.1101/2023.08.30.555477</a></p>
        <p class="abstract">While different features for the activity of the bacterial canonical SMC complex, Smc-ScpAB, have been described in different bacteria, not much is known about the way chromosomes in enterobacteria interact with their SMC complex, MukBEF. Here we used a number of in vivo assays in E. coli to reveal how MukBEF controls chromosome conformation and how the MatP/matS system prevents MukBEF activity. Our results indicate that the loading of MukBEF occurs preferentially in newly replicated DNA, at multiple loci on the chromosome where it can promote long-range contacts in cis even though MukBEF can promote long-range contacts in the absence of replication. Using HiC and ChIP-seq analyses in strains with rearranged chromosomes, the prevention of MukBEF activity increases with the number of matS sites and this effect likely results from the unloading of MukBEF by MatP. Altogether, our results reveal how MukBEF operates to control chromosome folding and segregation in E. coli.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555439">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555439" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555439">
        <p class="paperTitle">High-quality genome of the zoophytophagous stinkbug, Nesidiocoris tenuis, informs their food habitadaptation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555439" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555439" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shibata, T.; Shimoda, M.; Kobayashi, T.; Arai, H.; Owashi, Y.; Uehara, T.</p>
        <p class="info">Score: 3.9, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555439' target='https://doi.org/10.1101/2023.08.29.555439'> 10.1101/2023.08.29.555439</a></p>
        <p class="abstract">The zoophytophagous stink bug, Nesidiocoris tenuis, is a promising natural enemy of micro-pests such as whiteflies and thrips. This bug possesses both phytophagous and entomophagous food habits, enabling it to obtain nutrition from both plants and insects. This trait allows us to maintain its population density in agricultural fields by introducing insectary plants, even when the pest prey density is extremely low. However, if the bugs&#39; population becomes too dense, they can sometimes damage crop plants. This dual character seems to arise from the food preferences and chemosensation of this predator. To understand the genomic landscape of N. tenuis, we examined the whole genome sequence of a commercially available Japanese strain. We used long-read sequencing and Hi-C analysis to assemble the genome at the chromosomal level. We then conducted a comparative analysis of the genome with previously reported genomes of phytophagous and hematophagous stink bugs to focus on the genetic factors contributing to this species&#39; herbivorous and carnivorous tendencies. Our findings suggest that the gustatory gene set plays a pivotal role in adapting to food habits, making it a promising target for selective breeding. Furthermore, we identified the whole genomes of microorganisms symbiotic with this species through genomic analysis. We believe that our results shed light on the food habit adaptations of N. tenuis and will accelerate breeding efforts based on new breeding techniques for natural enemy insects, including genomics and genome editing.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
