<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Single-cell nascent RNA sequencing using click-chemistry unveils coordinated transcription
Authors: Mahat, D. B.; Tippens, N. D.; Martin-Rufino, J. D.; Waterton, S. K.; Fu, J.; Blatt, S. E.; Sharp, P. A.
Score: 43.4, Published: 2023-09-19 DOI: 10.1101/2023.09.15.558015
Transcription is the primary regulatory step in gene expression. Divergent transcription initiation from promoters and enhancers produces stable RNAs from genes and unstable RNAs from enhancers1-5. Nascent RNA capture and sequencing assays simultaneously measure gene and enhancer activity in cell populations6-9.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="Single-cell nascent RNA sequencing using click-chemistry unveils coordinated transcription
Authors: Mahat, D. B.; Tippens, N. D.; Martin-Rufino, J. D.; Waterton, S. K.; Fu, J.; Blatt, S. E.; Sharp, P. A.
Score: 43.4, Published: 2023-09-19 DOI: 10.1101/2023.09.15.558015
Transcription is the primary regulatory step in gene expression. Divergent transcription initiation from promoters and enhancers produces stable RNAs from genes and unstable RNAs from enhancers1-5. Nascent RNA capture and sequencing assays simultaneously measure gene and enhancer activity in cell populations6-9." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-24T10:37:06+00:00" />
<meta property="article:modified_time" content="2023-09-24T10:37:06+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="Single-cell nascent RNA sequencing using click-chemistry unveils coordinated transcription
Authors: Mahat, D. B.; Tippens, N. D.; Martin-Rufino, J. D.; Waterton, S. K.; Fu, J.; Blatt, S. E.; Sharp, P. A.
Score: 43.4, Published: 2023-09-19 DOI: 10.1101/2023.09.15.558015
Transcription is the primary regulatory step in gene expression. Divergent transcription initiation from promoters and enhancers produces stable RNAs from genes and unstable RNAs from enhancers1-5. Nascent RNA capture and sequencing assays simultaneously measure gene and enhancer activity in cell populations6-9."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "Single-cell nascent RNA sequencing using click-chemistry unveils coordinated transcription\nAuthors: Mahat, D. B.; Tippens, N. D.; Martin-Rufino, J. D.; Waterton, S. K.; Fu, J.; Blatt, S. E.; Sharp, P. A.\nScore: 43.4, Published: 2023-09-19 DOI: 10.1101/2023.09.15.558015\nTranscription is the primary regulatory step in gene expression. Divergent transcription initiation from promoters and enhancers produces stable RNAs from genes and unstable RNAs from enhancers1-5. Nascent RNA capture and sequencing assays simultaneously measure gene and enhancer activity in cell populations6-9.",
  "keywords": [
    
  ],
  "articleBody": " Single-cell nascent RNA sequencing using click-chemistry unveils coordinated transcription\nAuthors: Mahat, D. B.; Tippens, N. D.; Martin-Rufino, J. D.; Waterton, S. K.; Fu, J.; Blatt, S. E.; Sharp, P. A.\nScore: 43.4, Published: 2023-09-19 DOI: 10.1101/2023.09.15.558015\nTranscription is the primary regulatory step in gene expression. Divergent transcription initiation from promoters and enhancers produces stable RNAs from genes and unstable RNAs from enhancers1-5. Nascent RNA capture and sequencing assays simultaneously measure gene and enhancer activity in cell populations6-9. However, fundamental questions in the temporal regulation of transcription and enhancer-gene synchrony remain unanswered primarily due to the absence of a single-cell perspective on active transcription. In this study, we present scGRO-seq - a novel single-cell nascent RNA sequencing assay using click-chemistry - and unveil the coordinated transcription throughout the genome. scGRO-seq demonstrates the episodic nature of transcription, and estimates burst size and frequency by directly quantifying transcribing RNA polymerases in individual cells. It reveals the co-transcription of functionally related genes and leverages the replication-dependent non-polyadenylated histone genes transcription to elucidate cell-cycle dynamics. The single-nucleotide spatial and temporal resolution of scGRO-seq identifies networks of enhancers and genes and indicates that the bursting of transcription at super-enhancers precedes the burst from associated genes. By imparting insights into the dynamic nature of transcription and the origin and propagation of transcription signals, scGRO-seq demonstrates its unique ability to investigate the mechanisms of transcription regulation and the role of enhancers in gene expression.\nBiophysically Interpretable Inference of Cell Types from Multimodal Sequencing Data\nAuthors: Chari, T.; Gorin, G.; Pachter, L.\nScore: 37.1, Published: 2023-09-19 DOI: 10.1101/2023.09.17.558131\nMultimodal, single-cell genomics technologies enable simultaneous capture of multiple facets of DNA and RNA processing in the cell. This creates opportunities for transcriptome-wide, mechanistic studies of cellular processing in heterogeneous cell types, with applications ranging from inferring kinetic differences between cells, to the role of stochasticity in driving heterogeneity. However, current methods for determining cell types or clusters present in multimodal data often rely on ad hoc or independent treatment of modalities, and assumptions ignoring inherent properties of the count data. To enable interpretable and consistent cell cluster determination from multimodal data, we present meK-Means (mechanistic K-Means) which integrates modalities and learns underlying, shared biophysical states through a unifying model of transcription. In particular, we demonstrate how meK-Means can be used to cluster cells from unspliced and spliced mRNA count modalities. By utilizing the causal, physical relationships underlying these modalities, we identify shared transcriptional kinetics across cells, which induce the observed gene expression profiles, and provide an alternative definition for clusters through the governing parameters of cellular processes.\nTuning gene expression to music: the compensatory effect of music on age-related cognitive disorders\nAuthors: Gomez-Carballa, A.; Navarro, L.; Pardo-Seco, J.; Bello, X.; Pischedda, S.; Viz-Lasheras, S.; Camino-Mera, A.; Curras, M. J.; Ferreiros, I.; Mallah, N.; Rey-Vazquez, S.; Redondo, L.; Dacosta-Urbieta, A.; Caamanho-Vinha, F.; Rivero-Calle, I.; Rodriguez-Tenrreiro, C.; Martinon-Torres, F.; Salas, A.\nScore: 35.2, Published: 2023-09-15 DOI: 10.1101/2023.09.12.557408\nExtensive literature has explored the beneficial effects of music in age-related cognitive disorders (ACD), but limited knowledge exists regarding its impact on gene expression. We analyzed transcriptomes of ACD patients and healthy controls, pre-post a music session (n=60), and main genes/pathways were compared to those dysregulated in mild cognitive impairment (MCI) and Alzheimers disease (AD) as revealed by a multi-cohort study (n=1269 MCI/AD and controls). Music was associated with 2.3 times more whole-genome gene expression, particularly on neurodegeneration-related genes, in ACD than controls. Co-expressed gene-modules and pathways analysis demonstrated that music impacted autophagy, vesicle and endosome organization, biological processes commonly dysregulated in MCI/AD. Notably, the data indicated a strong negative correlation between musically-modified genes/pathways in ACD and those dysregulated in MCI/AD. These findings highlight the compensatory effect of music on genes/biological processes affected in MCI/AD, providing insights into the molecular mechanisms underlying the benefits of music on these disorders.\nMassive-scale single-cell chromatin accessibility sequencing using combinatorial fluidic indexing\nAuthors: Zhang, X.; Marand, A. P.; Yan, H.; Schmitz, R. J.\nScore: 29.0, Published: 2023-09-20 DOI: 10.1101/2023.09.17.558155\nSingle-cell ATAC-seq has emerged as a powerful approach for revealing candidate cis-regulatory elements genome-wide at cell-type resolution. However, current single-cell methods suffer from limited throughput and high costs. Here, we present a novel technique called single-cell combinatorial fluidic indexing ATAC-sequencing (\"scifi-ATAC-seq\"), which combines a barcoded Tn5 pre-indexing step with droplet-based single-cell ATAC-seq using a widely commercialized microfluidics platform (10X Genomics). With scifi-ATAC-seq, up to 200,000 nuclei across multiple samples in a single emulsion reaction can be indexed, representing a [~]20-fold increase in throughput compared to the standard 10X Genomics workflow.\nGene regulatory network inference from CRISPR perturbations in primary CD4+ T cells elucidates the genomic basis of immune disease\nAuthors: Weinstock, J. S.; Arce, M. M.; Freimer, J. W.; Ota, M.; Marson, A.; Battle, A.; Pritchard, J. K.\nScore: 28.2, Published: 2023-09-17 DOI: 10.1101/2023.09.17.557749\nThe effects of genetic variation on complex traits act mainly through changes in gene regulation. Although many genetic variants have been linked to target genes in cis, the trans-regulatory cascade mediating their effects remains largely uncharacterized. Mapping trans-regulators based on natural genetic variation, including eQTL mapping, has been challenging due to small effects. Experimental perturbation approaches offer a complementary and powerful approach to mapping trans-regulators. We used CRISPR knockouts of 84 genes in primary CD4+ T cells to perturb an immune cell gene network, targeting both inborn error of immunity (IEI) disease transcription factors (TFs) and background TFs matched in constraint and expression level, but without a known immune disease association. We developed a novel Bayesian structure learning method called Linear Latent Causal Bayes (LLCB) to estimate the gene regulatory network from perturbation data and observed 211 directed edges among the genes which could not be detected in existing CD4+ trans-eQTL data. We used LLCB to characterize the differences between the IEI and background TFs, finding that the gene groups were highly interconnected, but that IEI TFs were much more likely to regulate immune cell specific pathways and immune GWAS genes. We further characterized nine coherent gene programs based on downstream effects of the TFs and linked these modules to regulation of GWAS genes, finding that canonical JAK-STAT family members are regulated by KMT2A, a global epigenetic regulator. These analyses reveal the trans-regulatory cascade from upstream epigenetic regulator to intermediate TFs to downstream effector cytokines and elucidate the logic linking immune GWAS genes to key signaling pathways.\nCryptic splicing mediates genetic and therapeutic perturbation of human gene expression levels\nAuthors: Fair, B. J.; Abad, C. B.; Zhao, J.; Lozano, S.; Reilly, A. K.; Staley, J. P.; Wang, J.; Li, Y. I.\nScore: 27.1, Published: 2023-09-16 DOI: 10.1101/2023.09.13.557588\nNonsense-mediated decay (NMD) is an RNA surveillance mechanism that degrades transcripts containing premature termination codons. Alternative splicing (AS) is known to affect the expression levels of some human genes by producing transcripts rapidly degraded by NMD. However, the importance of AS in determining expression levels of most genes is unclear because rapidly degraded mRNA isoforms are difficult to quantify. To assess the impact of AS on gene expression levels, we analyzed population-scale data across eight molecular assays that capture gene regulation before, during, and after transcription and mRNA decay. Sequencing nascent RNA revealed that [~]15% of all mRNA molecules are NMD targets, of which most result from low-usage cryptic splicing. Leveraging genetic variation across cell lines, we find that trait-associated loci explained by AS are similarly likely to associate with NMD-induced expression level differences as with differences in protein isoform usage, suggesting that much of the impact of AS is mediated by NMD. Finally, we used the splice-switching drug risdiplam to perturb AS at hundreds of genes, finding that [~]3/4 of the splicing perturbations induce NMD. The remarkable prevalence of cryptic splice sites across the genome presents a rich opportunity for natural selection and therapeutics to shape the expression levels of nearly all human genes.\nA low-input high resolution sequential chromatin immunoprecipitation method captures genome-wide dynamics of bivalent chromatin\nAuthors: Ho, W.; Seneviratne, J. A.; Glancy, E.; Eckersley-Maslin, M. A.\nScore: 25.3, Published: 2023-09-18 DOI: 10.1101/2023.09.18.558170\nBackgroundBivalent chromatin is an exemplar of epigenetic plasticity. This co-occurrence of active-associated H3K4me3 and inactive-associated H3K27me3 histone modifications on opposite tails of the same nucleosome occurs predominantly at promoters where it poises them for future transcriptional upregulation or terminal silencing. We know little of the dynamics, resolution, and regulation of this chromatin state outside of embryonic stem cells where it was first described. This is partly due to the technical challenges distinguishing bone-fide bivalent chromatin, where both marks are on the same nucleosome, from allelic or sample heterogeneity where there is a mix of H3K4me3-only and H3K27me3-only mononucleosomes. ResultsHere, we present a robust and sensitive method to accurately map genome-wide bivalent chromatin along with all necessary controls from as little as 2 million cells. We optimised and refined the sequential ChIP protocol which uses two sequential overnight immunoprecipitation reactions to robustly purify nucleosomes that are truly bivalent and contain both H3K4me3 and H3K27me3 modifications. Our method generates high quality genome-wide maps with strong peak enrichment and low background which can be analysed using standard bioinformatic packages. Using this method, we detect twice as many bivalent regions in mouse embryonic stem cells as previously identified, bringing the total number of bivalently marked gene promoters to 8,373. Furthermore, profiling Dppa2/4 knockout mouse embryonic stem cells which lose both H3K4me3 and H3K27me3 at approximately 10% of bivalent promoters, demonstrated the ability of our method to capture bivalent chromatin dynamics. ConclusionsOur optimised sequential reChIP method enables high-resolution genome-wide assessment of bivalent chromatin together with all required controls in as little as 2 million cells. We share a detailed protocol and guidelines that will enable bivalent chromatin landscapes to be generated in a range of cellular contexts, greatly enhancing our understanding of bivalent chromatin and epigenetic plasticity beyond embryonic stem cells.\nGenetic tracing of market wildlife and viruses at the epicenter of the COVID-19 pandemic\nAuthors: Crits-Christoph, A.; Levy, J. I.; Pekar, J.; Goldstein, S. A.; Singh, R.; Hensel, Z.; Gangavarapu, K.; Rogers, M. B.; Moshiri, N.; Garry, R. f.; Holmes, E. C.; Koopmans, M. P. G.; Lemey, P.; Popescu, S.; Rambaut, A.; Robertson, D. L.; Suchard, M. A.; Wertheim, J.; Rasmussen, A. L.; Andersen, K. G.; Worobey, M.; Debarre, F.\nScore: 53.9, Published: 2023-09-14 DOI: 10.1101/2023.09.13.557637\nZoonotic spillovers of viruses have occurred through the animal trade worldwide. The start of the COVID-19 pandemic was traced epidemiologically to the Huanan Wholesale Seafood Market, the site with the most reported wildlife vendors in the city of Wuhan, China. Here, we analyze publicly available qPCR and sequencing data from environmental samples collected in the Huanan market in early 2020. We demonstrate that the SARS-CoV-2 genetic diversity linked to this market is consistent with market emergence, and find increased SARS-CoV-2 positivity near and within a particular wildlife stall. We identify wildlife DNA in all SARS-CoV-2 positive samples from this stall. This includes species such as civets, bamboo rats, porcupines, hedgehogs, and one species, raccoon dogs, known to be capable of SARS-CoV-2 transmission. We also detect other animal viruses that infect raccoon dogs, civets, and bamboo rats. Combining metagenomic and phylogenetic approaches, we recover genotypes of market animals and compare them to those from other markets. This analysis provides the genetic basis for a short list of potential intermediate hosts of SARS-CoV-2 to prioritize for retrospective serological testing and viral sampling.\nCohesin mediated loop extrusion from active enhancers form chromatin jets in C. elegans\nAuthors: Kim, J.; Ercan, S.\nScore: 22.7, Published: 2023-09-20 DOI: 10.1101/2023.09.18.558239\nIn mammals, cohesin and CTCF organize the 3D genome into topologically associated domains (TADs) to regulate communication between cis-regulatory elements. However, many organisms, including S. cerevisiae, C. elegans, and A. thaliana lack CTCF. Here, we use C. elegans as a model to investigate the function of cohesin in 3D genome organization in an animal without CTCF. We use auxin-inducible degradation to acutely deplete SMC-3 or its negative regulator WAPL-1 from somatic cells. Using Hi-C data, we identify a cohesin-dependent 3D genome feature termed chromatin jets (aka fountains), also observed in zebrafish and mammalian genomes. The jets emerge from NIPBL occupied segments, and the trajectory of the jets coincides with cohesin binding. The spreading of cohesin from jet origins depends on a fully intact cohesin complex and is extended upon WAPL-1 depletion. These results support the idea that cohesin is preferentially loaded at NIPBL occupied sites, from which cohesin loop extrudes in an effectively two-sided manner. The location of putative loading sites coincides with active enhancers and the pattern of chromatin jets correlates with transcription. We propose that in the absence of CTCF, preferential loading of cohesin at enhancers is a conserved mechanism of genome organization that regulates the interaction of gene regulatory elements in 3D.\nSignatures of transposon-mediated genome inflation, host specialization, and photoentrainment in Entomophthora muscae and allied entomophthoralean fungi\nAuthors: Stajich, J. E.; Lovett, B.; Lee, E.; Macias, A. M.; Hajek, A. E.; de Bivort, B. L.; Kasson, M. T.; De Fine Licht, H. H.; Elya, C.\nScore: 16.0, Published: 2023-09-16 DOI: 10.1101/2023.09.13.557621\nDespite over a century of observations, the obligate insect parasites within the order Entomophthorales remain poorly characterized at the genetic level. This is in part due to their large genome sizes and difficulty in obtaining sequenceable material. In this manuscript, we leveraged a recently-isolated, laboratory-tractable Entomophthora muscae isolate and improved long-read sequencing to obtain a largely-complete entomophthoralean genome. Our E. muscae assembly is 1.03 Gb, consists of 7,810 contigs and contains 81.3% complete fungal BUSCOs. Using a comparative approach with other available (transcriptomic and genomic) datasets from entomophthoralean fungi, we provide new insight into the biology of these understudied pathogens. We offer a head-to-head comparison of morphological and molecular data for species within the E. muscae species complex. Our findings suggest that substantial taxonomic revision is needed to define species within this group and we provide recommendations for differentiating strains and species in the context of the existing body of E. muscae scientific literature. We show that giant genomes are the norm within Entomophthoraceae owing to extensive, but not recent, Ty3 retrotransposon activity, despite the presence of anti-transposable element defense machinery (RNAi). In addition, we find that E. muscae and its closest allies are enriched for M16A peptidases and possess genes that are likely homologs to the blue-light sensor white-collar 1, a Neurospora crassa gene that has a well-established role in maintaining circadian rhythms. We find that E. muscae has an expanded group of acid-trehalases, consistent with trehalose being the primary sugar component of fly (and insect) hemolymph. We uncover evidence that E. muscae diverged from other entomophthoralean fungi by expansion of existing families, rather than loss of particular domains, and possesses a potentially unique suite of secreted catabolic enzymes, consistent with E. muscaes species-specific, biotrophic lifestyle. Altogether, we provide a genetic and molecular foundation that we hope will provide a platform for the continued study of the unique biology of entomophthoralean fungi.\n",
  "wordCount" : "2471",
  "inLanguage": "en",
  "datePublished": "2023-09-24T10:37:06Z",
  "dateModified": "2023-09-24T10:37:06Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on September 24, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.558015">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.558015" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.558015">
        <p class="paperTitle">Single-cell nascent RNA sequencing using click-chemistry unveils coordinated transcription</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.558015" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.558015" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mahat, D. B.; Tippens, N. D.; Martin-Rufino, J. D.; Waterton, S. K.; Fu, J.; Blatt, S. E.; Sharp, P. A.</p>
        <p class="info">Score: 43.4, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.558015' target='https://doi.org/10.1101/2023.09.15.558015'> 10.1101/2023.09.15.558015</a></p>
        <p class="abstract">Transcription is the primary regulatory step in gene expression. Divergent transcription initiation from promoters and enhancers produces stable RNAs from genes and unstable RNAs from enhancers1-5. Nascent RNA capture and sequencing assays simultaneously measure gene and enhancer activity in cell populations6-9. However, fundamental questions in the temporal regulation of transcription and enhancer-gene synchrony remain unanswered primarily due to the absence of a single-cell perspective on active transcription. In this study, we present scGRO-seq - a novel single-cell nascent RNA sequencing assay using click-chemistry - and unveil the coordinated transcription throughout the genome. scGRO-seq demonstrates the episodic nature of transcription, and estimates burst size and frequency by directly quantifying transcribing RNA polymerases in individual cells. It reveals the co-transcription of functionally related genes and leverages the replication-dependent non-polyadenylated histone genes transcription to elucidate cell-cycle dynamics. The single-nucleotide spatial and temporal resolution of scGRO-seq identifies networks of enhancers and genes and indicates that the bursting of transcription at super-enhancers precedes the burst from associated genes. By imparting insights into the dynamic nature of transcription and the origin and propagation of transcription signals, scGRO-seq demonstrates its unique ability to investigate the mechanisms of transcription regulation and the role of enhancers in gene expression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.558131">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.558131" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.558131">
        <p class="paperTitle">Biophysically Interpretable Inference of Cell Types from Multimodal Sequencing Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.558131" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.558131" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chari, T.; Gorin, G.; Pachter, L.</p>
        <p class="info">Score: 37.1, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.558131' target='https://doi.org/10.1101/2023.09.17.558131'> 10.1101/2023.09.17.558131</a></p>
        <p class="abstract">Multimodal, single-cell genomics technologies enable simultaneous capture of multiple facets of DNA and RNA processing in the cell. This creates opportunities for transcriptome-wide, mechanistic studies of cellular processing in heterogeneous cell types, with applications ranging from inferring kinetic differences between cells, to the role of stochasticity in driving heterogeneity. However, current methods for determining cell types or  clusters present in multimodal data often rely on ad hoc or independent treatment of modalities, and assumptions ignoring inherent properties of the count data. To enable interpretable and consistent cell cluster determination from multimodal data, we present meK-Means (mechanistic K-Means) which integrates modalities and learns underlying, shared biophysical states through a unifying model of transcription. In particular, we demonstrate how meK-Means can be used to cluster cells from unspliced and spliced mRNA count modalities. By utilizing the causal, physical relationships underlying these modalities, we identify shared transcriptional kinetics across cells, which induce the observed gene expression profiles, and provide an alternative definition for  clusters through the governing parameters of cellular processes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.12.557408">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.12.557408" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.12.557408">
        <p class="paperTitle">Tuning gene expression to music: the compensatory effect of music on age-related cognitive disorders</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.12.557408" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.12.557408" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gomez-Carballa, A.; Navarro, L.; Pardo-Seco, J.; Bello, X.; Pischedda, S.; Viz-Lasheras, S.; Camino-Mera, A.; Curras, M. J.; Ferreiros, I.; Mallah, N.; Rey-Vazquez, S.; Redondo, L.; Dacosta-Urbieta, A.; Caamanho-Vinha, F.; Rivero-Calle, I.; Rodriguez-Tenrreiro, C.; Martinon-Torres, F.; Salas, A.</p>
        <p class="info">Score: 35.2, Published: 2023-09-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.12.557408' target='https://doi.org/10.1101/2023.09.12.557408'> 10.1101/2023.09.12.557408</a></p>
        <p class="abstract">Extensive literature has explored the beneficial effects of music in age-related cognitive disorders (ACD), but limited knowledge exists regarding its impact on gene expression. We analyzed transcriptomes of ACD patients and healthy controls, pre-post a music session (n=60), and main genes/pathways were compared to those dysregulated in mild cognitive impairment (MCI) and Alzheimers disease (AD) as revealed by a multi-cohort study (n=1269 MCI/AD and controls). Music was associated with 2.3 times more whole-genome gene expression, particularly on neurodegeneration-related genes, in ACD than controls. Co-expressed gene-modules and pathways analysis demonstrated that music impacted autophagy, vesicle and endosome organization, biological processes commonly dysregulated in MCI/AD. Notably, the data indicated a strong negative correlation between musically-modified genes/pathways in ACD and those dysregulated in MCI/AD. These findings highlight the compensatory effect of music on genes/biological processes affected in MCI/AD, providing insights into the molecular mechanisms underlying the benefits of music on these disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.558155">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.558155" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.558155">
        <p class="paperTitle">Massive-scale single-cell chromatin accessibility sequencing using combinatorial fluidic indexing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.558155" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.558155" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, X.; Marand, A. P.; Yan, H.; Schmitz, R. J.</p>
        <p class="info">Score: 29.0, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.558155' target='https://doi.org/10.1101/2023.09.17.558155'> 10.1101/2023.09.17.558155</a></p>
        <p class="abstract">Single-cell ATAC-seq has emerged as a powerful approach for revealing candidate cis-regulatory elements genome-wide at cell-type resolution. However, current single-cell methods suffer from limited throughput and high costs. Here, we present a novel technique called single-cell combinatorial fluidic indexing ATAC-sequencing (&#34;scifi-ATAC-seq&#34;), which combines a barcoded Tn5 pre-indexing step with droplet-based single-cell ATAC-seq using a widely commercialized microfluidics platform (10X Genomics). With scifi-ATAC-seq, up to 200,000 nuclei across multiple samples in a single emulsion reaction can be indexed, representing a [~]20-fold increase in throughput compared to the standard 10X Genomics workflow.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.17.557749">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.17.557749" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.17.557749">
        <p class="paperTitle">Gene regulatory network inference from CRISPR perturbations in primary CD4&#43; T cells elucidates the genomic basis of immune disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.17.557749" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.17.557749" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Weinstock, J. S.; Arce, M. M.; Freimer, J. W.; Ota, M.; Marson, A.; Battle, A.; Pritchard, J. K.</p>
        <p class="info">Score: 28.2, Published: 2023-09-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.17.557749' target='https://doi.org/10.1101/2023.09.17.557749'> 10.1101/2023.09.17.557749</a></p>
        <p class="abstract">The effects of genetic variation on complex traits act mainly through changes in gene regulation. Although many genetic variants have been linked to target genes in cis, the trans-regulatory cascade mediating their effects remains largely uncharacterized. Mapping trans-regulators based on natural genetic variation, including eQTL mapping, has been challenging due to small effects. Experimental perturbation approaches offer a complementary and powerful approach to mapping trans-regulators. We used CRISPR knockouts of 84 genes in primary CD4&#43; T cells to perturb an immune cell gene network, targeting both inborn error of immunity (IEI) disease transcription factors (TFs) and background TFs matched in constraint and expression level, but without a known immune disease association. We developed a novel Bayesian structure learning method called Linear Latent Causal Bayes (LLCB) to estimate the gene regulatory network from perturbation data and observed 211 directed edges among the genes which could not be detected in existing CD4&#43; trans-eQTL data. We used LLCB to characterize the differences between the IEI and background TFs, finding that the gene groups were highly interconnected, but that IEI TFs were much more likely to regulate immune cell specific pathways and immune GWAS genes. We further characterized nine coherent gene programs based on downstream effects of the TFs and linked these modules to regulation of GWAS genes, finding that canonical JAK-STAT family members are regulated by KMT2A, a global epigenetic regulator. These analyses reveal the trans-regulatory cascade from upstream epigenetic regulator to intermediate TFs to downstream effector cytokines and elucidate the logic linking immune GWAS genes to key signaling pathways.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557588">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557588" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557588">
        <p class="paperTitle">Cryptic splicing mediates genetic and therapeutic perturbation of human gene expression levels</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557588" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557588" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fair, B. J.; Abad, C. B.; Zhao, J.; Lozano, S.; Reilly, A. K.; Staley, J. P.; Wang, J.; Li, Y. I.</p>
        <p class="info">Score: 27.1, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557588' target='https://doi.org/10.1101/2023.09.13.557588'> 10.1101/2023.09.13.557588</a></p>
        <p class="abstract">Nonsense-mediated decay (NMD) is an RNA surveillance mechanism that degrades transcripts containing premature termination codons. Alternative splicing (AS) is known to affect the expression levels of some human genes by producing transcripts rapidly degraded by NMD. However, the importance of AS in determining expression levels of most genes is unclear because rapidly degraded mRNA isoforms are difficult to quantify. To assess the impact of AS on gene expression levels, we analyzed population-scale data across eight molecular assays that capture gene regulation before, during, and after transcription and mRNA decay. Sequencing nascent RNA revealed that [~]15% of all mRNA molecules are NMD targets, of which most result from low-usage cryptic splicing. Leveraging genetic variation across cell lines, we find that trait-associated loci explained by AS are similarly likely to associate with NMD-induced expression level differences as with differences in protein isoform usage, suggesting that much of the impact of AS is mediated by NMD. Finally, we used the splice-switching drug risdiplam to perturb AS at hundreds of genes, finding that [~]3/4 of the splicing perturbations induce NMD. The remarkable prevalence of cryptic splice sites across the genome presents a rich opportunity for natural selection and therapeutics to shape the expression levels of nearly all human genes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558170">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558170" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558170">
        <p class="paperTitle">A low-input high resolution sequential chromatin immunoprecipitation method captures genome-wide dynamics of bivalent chromatin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558170" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558170" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ho, W.; Seneviratne, J. A.; Glancy, E.; Eckersley-Maslin, M. A.</p>
        <p class="info">Score: 25.3, Published: 2023-09-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558170' target='https://doi.org/10.1101/2023.09.18.558170'> 10.1101/2023.09.18.558170</a></p>
        <p class="abstract">BackgroundBivalent chromatin is an exemplar of epigenetic plasticity. This co-occurrence of active-associated H3K4me3 and inactive-associated H3K27me3 histone modifications on opposite tails of the same nucleosome occurs predominantly at promoters where it poises them for future transcriptional upregulation or terminal silencing. We know little of the dynamics, resolution, and regulation of this chromatin state outside of embryonic stem cells where it was first described. This is partly due to the technical challenges distinguishing bone-fide bivalent chromatin, where both marks are on the same nucleosome, from allelic or sample heterogeneity where there is a mix of H3K4me3-only and H3K27me3-only mononucleosomes.

ResultsHere, we present a robust and sensitive method to accurately map genome-wide bivalent chromatin along with all necessary controls from as little as 2 million cells. We optimised and refined the sequential ChIP protocol which uses two sequential overnight immunoprecipitation reactions to robustly purify nucleosomes that are truly bivalent and contain both H3K4me3 and H3K27me3 modifications. Our method generates high quality genome-wide maps with strong peak enrichment and low background which can be analysed using standard bioinformatic packages. Using this method, we detect twice as many bivalent regions in mouse embryonic stem cells as previously identified, bringing the total number of bivalently marked gene promoters to 8,373. Furthermore, profiling Dppa2/4 knockout mouse embryonic stem cells which lose both H3K4me3 and H3K27me3 at approximately 10% of bivalent promoters, demonstrated the ability of our method to capture bivalent chromatin dynamics.

ConclusionsOur optimised sequential reChIP method enables high-resolution genome-wide assessment of bivalent chromatin together with all required controls in as little as 2 million cells. We share a detailed protocol and guidelines that will enable bivalent chromatin landscapes to be generated in a range of cellular contexts, greatly enhancing our understanding of bivalent chromatin and epigenetic plasticity beyond embryonic stem cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557637">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557637" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557637">
        <p class="paperTitle">Genetic tracing of market wildlife and viruses at the epicenter of the COVID-19 pandemic</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557637" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557637" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Crits-Christoph, A.; Levy, J. I.; Pekar, J.; Goldstein, S. A.; Singh, R.; Hensel, Z.; Gangavarapu, K.; Rogers, M. B.; Moshiri, N.; Garry, R. f.; Holmes, E. C.; Koopmans, M. P. G.; Lemey, P.; Popescu, S.; Rambaut, A.; Robertson, D. L.; Suchard, M. A.; Wertheim, J.; Rasmussen, A. L.; Andersen, K. G.; Worobey, M.; Debarre, F.</p>
        <p class="info">Score: 53.9, Published: 2023-09-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557637' target='https://doi.org/10.1101/2023.09.13.557637'> 10.1101/2023.09.13.557637</a></p>
        <p class="abstract">Zoonotic spillovers of viruses have occurred through the animal trade worldwide. The start of the COVID-19 pandemic was traced epidemiologically to the Huanan Wholesale Seafood Market, the site with the most reported wildlife vendors in the city of Wuhan, China. Here, we analyze publicly available qPCR and sequencing data from environmental samples collected in the Huanan market in early 2020. We demonstrate that the SARS-CoV-2 genetic diversity linked to this market is consistent with market emergence, and find increased SARS-CoV-2 positivity near and within a particular wildlife stall. We identify wildlife DNA in all SARS-CoV-2 positive samples from this stall. This includes species such as civets, bamboo rats, porcupines, hedgehogs, and one species, raccoon dogs, known to be capable of SARS-CoV-2 transmission. We also detect other animal viruses that infect raccoon dogs, civets, and bamboo rats. Combining metagenomic and phylogenetic approaches, we recover genotypes of market animals and compare them to those from other markets. This analysis provides the genetic basis for a short list of potential intermediate hosts of SARS-CoV-2 to prioritize for retrospective serological testing and viral sampling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558239">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558239" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558239">
        <p class="paperTitle">Cohesin mediated loop extrusion from active enhancers form chromatin jets in C. elegans</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558239" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558239" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, J.; Ercan, S.</p>
        <p class="info">Score: 22.7, Published: 2023-09-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558239' target='https://doi.org/10.1101/2023.09.18.558239'> 10.1101/2023.09.18.558239</a></p>
        <p class="abstract">In mammals, cohesin and CTCF organize the 3D genome into topologically associated domains (TADs) to regulate communication between cis-regulatory elements. However, many organisms, including S. cerevisiae, C. elegans, and A. thaliana lack CTCF. Here, we use C. elegans as a model to investigate the function of cohesin in 3D genome organization in an animal without CTCF. We use auxin-inducible degradation to acutely deplete SMC-3 or its negative regulator WAPL-1 from somatic cells. Using Hi-C data, we identify a cohesin-dependent 3D genome feature termed chromatin jets (aka fountains), also observed in zebrafish and mammalian genomes. The jets emerge from NIPBL occupied segments, and the trajectory of the jets coincides with cohesin binding. The spreading of cohesin from jet origins depends on a fully intact cohesin complex and is extended upon WAPL-1 depletion. These results support the idea that cohesin is preferentially loaded at NIPBL occupied sites, from which cohesin loop extrudes in an effectively two-sided manner. The location of putative loading sites coincides with active enhancers and the pattern of chromatin jets correlates with transcription. We propose that in the absence of CTCF, preferential loading of cohesin at enhancers is a conserved mechanism of genome organization that regulates the interaction of gene regulatory elements in 3D.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.13.557621">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.13.557621" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.13.557621">
        <p class="paperTitle">Signatures of transposon-mediated genome inflation, host specialization, and photoentrainment in Entomophthora muscae and allied entomophthoralean fungi</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.13.557621" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.13.557621" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Stajich, J. E.; Lovett, B.; Lee, E.; Macias, A. M.; Hajek, A. E.; de Bivort, B. L.; Kasson, M. T.; De Fine Licht, H. H.; Elya, C.</p>
        <p class="info">Score: 16.0, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.13.557621' target='https://doi.org/10.1101/2023.09.13.557621'> 10.1101/2023.09.13.557621</a></p>
        <p class="abstract">Despite over a century of observations, the obligate insect parasites within the order Entomophthorales remain poorly characterized at the genetic level. This is in part due to their large genome sizes and difficulty in obtaining sequenceable material. In this manuscript, we leveraged a recently-isolated, laboratory-tractable Entomophthora muscae isolate and improved long-read sequencing to obtain a largely-complete entomophthoralean genome. Our E. muscae assembly is 1.03 Gb, consists of 7,810 contigs and contains 81.3% complete fungal BUSCOs. Using a comparative approach with other available (transcriptomic and genomic) datasets from entomophthoralean fungi, we provide new insight into the biology of these understudied pathogens. We offer a head-to-head comparison of morphological and molecular data for species within the E. muscae species complex. Our findings suggest that substantial taxonomic revision is needed to define species within this group and we provide recommendations for differentiating strains and species in the context of the existing body of E. muscae scientific literature. We show that giant genomes are the norm within Entomophthoraceae owing to extensive, but not recent, Ty3 retrotransposon activity, despite the presence of anti-transposable element defense machinery (RNAi). In addition, we find that E. muscae and its closest allies are enriched for M16A peptidases and possess genes that are likely homologs to the blue-light sensor white-collar 1, a Neurospora crassa gene that has a well-established role in maintaining circadian rhythms. We find that E. muscae has an expanded group of acid-trehalases, consistent with trehalose being the primary sugar component of fly (and insect) hemolymph. We uncover evidence that E. muscae diverged from other entomophthoralean fungi by expansion of existing families, rather than loss of particular domains, and possesses a potentially unique suite of secreted catabolic enzymes, consistent with E. muscaes species-specific, biotrophic lifestyle. Altogether, we provide a genetic and molecular foundation that we hope will provide a platform for the continued study of the unique biology of entomophthoralean fungi.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
